Britta Strehl  
Ulrike Seifert  
Elke Krüger  
Sylvia Heink  
Ulrike Kuckelkorn  
Peter-M. Kloetzel  

Interferon-$\gamma$, the functional plasticity  
of the ubiquitin–proteasome system,  
and MHC class I antigen processing  

Authors' address  
Britta Strehl, Ulrike Seifert, Elke Krüger, Sylvia Heink,  
Ulrike Kuckelkorn, Peter-M. Kloetzel,  
Institut für Biochemie, Charité, Berlin University  
Berlin, Germany.  

Correspondence to:  
Peter-M. Kloetzel  
Institut für Biochemie, Charité, Berlin University  
Monbijoustr. 2, 10117 Berlin, Germany  
Tel.: +49 30 450 528071  
Fax: +49 30 450 528921  
E-mail: p-m.kloetzel@charite.de  

Summary: The proteasome system is a central component of a cascade  
of proteolytic processing steps required to generate antigenic peptides  
presented at the cell surface to cytotoxic T lymphocytes by major histo-  
compatibility complex (MHC) class I molecules. The nascent protein pool  
or DRiPs (defective ribosomal products) appear to represent an important  
source for MHC class I epitopes. Owing to the destructive activities of  
aminopeptidases in the cytosol, at most 1% of the peptides generated by  
the ubiquitin–proteasome system seems to be made available to the  
immune system. Interferon-$\gamma$ (IFN-$\gamma$) helps to override these limitations  
by the formation of immunoproteasomes, the activator complex PA28,  
and the induction of several aminopeptidases. Both immunoproteasomes  
and PA28 use cleavage sites already used by constitutive proteasomes but  
with altered and in some cases dramatically enhanced frequency.  
Therefore, two proteolytic cascades appear to have evolved to provide  
MHC class I epitopes. The ‘constitutive proteolytic cascade’ is designed to  
efficiently degrade proteins to single amino acid residues, allowing only a  
small percentage of peptides to be presented at the cell surface. In  
contrast, the IFN-$\gamma$-controlled proteolytic cascade generates larger  
amounts of appropriate antigenic peptides, assuring more peptides to  
overcome the proteolytic restrictions of the constitutive system, thereby  
enhancing MHC class I antigen presentation.  

Introduction  
As part of the vertebrate immune surveillance system, cyto-  
toxic T lymphocytes (CTLs) recognize antigens that are bound  
by major histocompatibility complex (MHC) class I proteins  
(1–3). To allow binding to MHC class I molecules, an antigen  
has to be proteolytically processed into peptides. The peptides  
of appropriate length that harbor a specific MHC class I-binding  
motif dock onto the transporter associated with antigen  
processing (TAP) to be translocated into the endoplasmic  
reticulum (ER), where they bind to the respective MHC class  
I molecules (4). An individual cell can express up to six  
different MHC class I alleles that have very different require-  
ments for peptide binding (5). The interaction of a peptide  
with the MHC class I peptide-binding groove obeys very  
Strehl et al · The proteasome and MHC class I antigen processing

stringent structural requirements, and only peptides with high affinity stabilize the MHC class I complex for transport to the plasma membrane (6, 7). The recognition of the MHC–peptide complex on the plasma membrane by a T-cell receptor, which is specific for a given antigenic peptide bound to a specific MHC class I molecule, eventually leads to T-cell activation (Fig. 1).

This sophisticated recognition system depends not only on the generation of antigenic peptides of the right quality, i.e., the correct size of eight to ten residues and correct sequence allowing binding into the groove of the MHC class I molecules, but also on the production of a sufficient number of peptides to trigger a successful immune response. This review focuses on the present understanding of how the cellular activity of the proteasome system can be modulated to allow a peptide-specific CTL response against all odds. Since the aminopeptidases involved in antigen processing are reviewed in other contributions to this issue, they are only referred to as needed for the discussion here.

The source and quantity of MHC class I peptides

The source of antigenic peptides, the amount of peptides being made available for antigen presentation, and the

proteolytic systems involved in the production of antigenic peptides are central issues in MHC class I antigen processing. An important part of research has centered on questions regarding the primary source of antigenic peptides and how the proteasome system can gain sufficient access to metabolically stable, lowly abundant, and in particular compartmentalized proteins. A processing mechanism linked to translation would guarantee that the cellular immune system has access to newly synthesized proteins before they reach a final functional state (8).

Early work on protein breakdown had shown that about one-third of intracellular proteolysis in mammalian cells is directed to nascent proteins and that these newly synthesized proteins are readily degraded within 1 or 2 min after their synthesis. This finding was considered to reflect the physiological necessity to sustain a high flux of proteins required for maintenance, growth, and adaptability of cells, and as being part of normal protein homeostasis regulating the cellular protein pool (9–11). More recent work revealed that the surprisingly large pool of nascent proteins was ubiquitinated and it was suggested, though not formally shown, that it is composed mostly of defective proteins, hence called defective ribosomal products (DRiPs) (12, 13). This nascent protein pool or DRiPs appear to represent an important (yet not the

![Diagram](attachment:diagram.png)

Fig. 1. Major histocompatibility complex (MHC) class I antigen processing. Short-lived nascent proteins or DRiPs, the major pool for the generation of antigenic peptides, are conjugated with a multiubiquitin chain, which targets them to the 26S proteasome for degradation. Protein processing results in the generation of peptides that carry the correct C-terminus for efficient TAP transport and an N-terminal extension (epitope precursor). The vast majority of the generated peptides (99%) will be destroyed by cytosol resident aminopeptidases. Antigenic peptides that have survived are transported into the endoplasmic reticulum (ER) by the TAP complex. Within the ER, peptides are trimmed to appropriate size by ER-

resident aminopeptidases in order to bind and stabilize the MHC class I heterodimers, which comprise a heavy chain and a β2-microglobulin molecule. The assembly of MHC molecules is coordinated by molecular chaperones. Fully assembled, peptide-loaded MHC molecules are translocated, via the Golgi apparatus, to the cell surface. By binding specifically to a given MHC allomorph loaded with a unique peptide, cytotoxic T cells with complementary T-cell receptors are stimulated to proliferate and destroy the infected target cell. DriPs, defective ribosomal proteins; TAP, transporter associated with antigen processing [modified from Kloetzel (22)].
sole) source from which MHC class I epitopes are derived (14, 15) and help to overcome potential restrictions for substrate accessibility due to compartmentalization and long half-lives of proteins (16).

In discussing the amount of peptides made available for antigen presentation and the proteolytic systems involved in their production, it is important to acknowledge that in order to satisfy its needs for peptides, the MHC class I antigen-presentation pathway is taking advantage of existing proteolytic systems and thus has not invented its own protein-degradation machinery (17). The MHC class I pathway, the cellular protein-degradation machineries involving the adenosine triphosphate (ATP)-dependent ubiquitin-proteasome system (UPS), and other proteases including aminopeptidases are first of all designed to eliminate regulatory, nonfunctional, and potentially toxic proteins (18) or to discard proteins that have been synthesized in excess and may therefore interfere with protein homeostasis. Proteins then are degraded to single amino acid residues, allowing their reutilization for protein synthesis.

Underscoring the difficulty of the immune system’s task to take advantage of the cellular protein-processing products, it has been estimated that at most only 1% of the peptide pool generated during protein breakdown is available to the immune system (16, 17, 19). The size of the peptide pool appears to be limited and tightly controlled by the activity of various aminopeptidases that degrade putative antigenic peptides. It was recently shown that overexpression of the cytosolic thimet oligopeptidase alone is sufficient to drastically downregulate MHC class I cell-surface expression (20).

The enhancement of MHC class I antigen presentation demands an increase in the available pool of antigenic peptides against the destructive forces of aminopeptidases, in order to raise the number of antigenic peptides with the required quality to stimulate an effective CTL response. One possible way to raise the quantity of antigenic peptides would be to increase the proteasome’s antigen-processing efficiency. This review therefore discusses how the functional plasticity of the UPS can be influenced by interferon-γ (IFN-γ) and how cytokine-induced components of the proteasome contribute to raise the antigen-generation productivity.

### Ubiquitin–proteasome system

The UPS is a key player within a cascade of proteolytic events that supply the cellular immune system with peptides of the appropriate quality and quantity. As part of this proteolytic cascade that also involves several aminopeptidases (21, 22),

the 26S proteasome complex is responsible for the initial selective degradation of polyubiquitinated protein substrates. It binds and degrades intracellular proteins that become tagged with a polyubiquitin chain by a three-step process involving the sequential action the ubiquitin-activating enzyme E1, several E2 ubiquitin-conjugating enzymes, and multiple E3 ubiquitin ligases that recognize specific substrates. The 26S proteasome is formed by the catalytic 20S core complex and two multisubunit 19S-regulator complexes that are responsible for the binding and unfolding of ubiquitinated substrates (23, 24).

The 20S proteasome is composed of 14 nonidentical subunits building four stacked rings of seven subunits each. Seven different but related α-subunits (α1–α7) form the two outer α-rings, whereas the two inner rings contain seven different β-subunits (β1–β7). The hydrolyzing activities of the 20S core are conferred by three of the seven β-subunits, i.e. subunits β1, β2, and β5, located in both of the two inner heptameric β-rings (25). The three active sites in one inner ring differ in their proteolytic activities and in their ability to hydrolyze short fluorogenic peptide substrates. Accordingly, the activities are referred to as chymotrypsin-like (β5), trypsin-like (β2), or caspase-like (β1) (1, 22, 26, 27). Despite some preference in cleavage site usage, the enzyme complex nevertheless possesses a high degree of flexibility concerning the quality of the generated peptides. Within a protein, almost every amino acid residue can serve as cleavage site, albeit at different efficiencies depending on the residues that flank the cleavage sites (28).

The 20S proteasome α-subunits are arranged to form a central gate that controls the access of substrates to the catalytic chamber. This gate is normally closed by the N-termini of the seven α-subunits, thus preventing uncontrolled access of substrates (29). Substrates enter the gate by binding of the 19S regulator (30), which is composed of two functionally different entities, i.e. the base and the lid (31). The base is formed by six ATPase subunits as well as by two non-ATPase subunits and directly interacts with the two outer α-rings of the 20S proteasome. The base is responsible for the opening of the gate and unfolding a protein substrate (24), while the lid components play a role in substrate binding and deubiquitination, which are essential for selective protein degradation.

Another protein that can activate the 20S proteasome is PA28 (32, 33). This complex is composed of the two IFN-γ-inducible subunits PA28α and PA28β, forming a ring-like structure built of three PA28α and four PA28β-subunits (34). Expression of PA28 however is not only induced by IFN-γ,
since most tissues reveal a varying constitutive and IFN-
γ-independent expression of the two PA28-subunits. PA 28
does not appear to activate the proteasome by directly affecting
the hydrolyzing activities of the different active sites
buried in the catalytic cavity. In contrast, structural and biochemical
evidence suggests that by binding to the outer
α-rings of the 20S proteasome, PA28 causes the N-terminal
tails of the α-subunits to flip upwards, thereby facilitating
substrate entry and product exit through the otherwise closed
central 'gate' of the 20S proteasome (35–37). On the basis of
these observations, it was suggested that such an open con-
formation might support the release of longer N-terminally
extended peptides, which were assumed to be adapted to the
requirements of antigen presentation. However, binding of
the 19S regulator also results in the complete opening of the
gates, and so-called hybrid proteasomes formed by the 19S
regulator–20S proteasome–PA28 exhibit the same peptide
hydrolyzing activity as 26S proteasome (38, 39). Thus, it is
unlikely that observed biological effects of PA28 function are
primarily the result of a gated 20S proteasome configuration.

The proteasome and antigen processing

On the basis of the proteasome's ability to cleave preferentially
after hydrophobic or basic residues, to generate peptides in the
range of three to 22 residues with a mean length of eight to nine
residues, and results from experiments using proteasome-
specific inhibitors, proteasomes emerged as the main protease
thought to be involved in the generation of MHC class I antigens
(40, 41). Since then, many studies have shown that one of the
intrinsic proteolytic properties of 20S proteasomes is their ability
to generate the correct C-terminus of epitopes with high efficiency,
whereas residues used as cleavage sites N-terminal of the
epitope appeared to be less well defined (3, 42). However,
while generation of the C-terminal anchor residues of MHC
class I epitopes seems to be a basic functional feature of the
proteasome, correct C-terminal cleavage site usage is not as
robust as one might expect. Presumably as part of an immune
escape mechanism, mutations have naturally evolved affecting
residues that directly flank the correct C-terminal cleavage site,
which strongly reduced epitope-generation efficiency. Thus, the
recognition of the Moloney murine leukemia virus (MuLV)–
derived epitope SSWDFTIV and the recognition of the p53–
derived epitope spanning residues 264–272 by specific CTLs
was abolished due to a single amino acid substitution immediately
flanking the C-terminal anchor residue of these epitopes
(28, 43). The most recently shown example for such an
immune escape mutation is a conservative Y/F residue exchange

flanking the immunodominant hepatitis C virus (HCV)
NS3<sub>1073–1081</sub> epitope that was identified in a HCV strain isolated
from chronically infected patients. The Y/F mutation impaired
the proteasome's ability to generate the immunodominant
NS3<sub>1073–1081</sub> epitope in any significant amount in vitro.
Furthermore, infection of human leukocyte antigen (HLA)–A2
transgenic mice with a vaccinia virus expressing the mutant NS3
protein demonstrated the inability of the proteasome to generate
this epitope. These studies therefore presented also the first
direct experimental proof that in vitro processing reactions perfectly resemble the in vivo situation (44).

Generation of epitope precursor peptides

Most of the MHC class I epitopes are derived from precursor
peptides that already possess the correct C-terminal anchor
residue of the epitope required for MHC class I binding
and that carry an N-terminal extension requiring trimming
both in the cytosol and in the ER by a set of specific
aminopeptidases.

To establish epitope prediction programs based on proteasomal cleavage properties, emphasis was primarily laid on
deciphering the sequence pattern surrounding the C-terminus
of an epitope that allows effective cleavage by the proteasome
(45–47). No study has focused in detail on the identification
of possibly existing predominant proteasomal cleavage sites
within the N-terminal flanking regions of an epitope. It is
apparent however that effective excision of an epitope–
harboring precursor peptide from a protein will not be
achieved solely by facilitating the generation of the correct
C-terminus of an epitope but at the same time requires efficient and coordinated cleavages within the N-terminal flanking
region. Thus, in the absence of detailed analyses, one may
predict that the cleavage sites used within the sequence flanking the N-terminus of an epitope will be the next less dominant but still preferred proteasomal cleavage sites. Since for a
single specific epitope several precursor peptides of different
length and with different N-terminal residues will exist,
the sum of the several less well-defined cleavages in the
N-terminal flanking region of an epitope has therefore to
equalize the number of the more specific C-terminal cleavages
to permit productive liberation of epitope precursor peptides.

Immunoproteasomes

The immune-modulatory cytokine IFN-γ, produced by activated T-helper 1 lymphocytes, CD8<sup>+</sup> CTLs, and natural killer
(NK) cells, is a pivotal player in the immune system. IFN-γ

enhances antigen presentation and triggers an antiviral response in the infected organism. IFN-γ binding to IFN-γR1 initiates an activation cascade involving the two IFN-γR2 chains and the respective Janus kinases (Jak), Jak1 and Jak2, which leads to the homodimerization of phospho-signal transducer and activator of transcription-1 (STAT1) molecules, their translocation into the nucleus, and the transcriptional upregulation of IFN-γ-inducible genes (48).

In vertebrates, specific catalytically active proteasome subunits, collectively referred to as immunosubunits (49), have evolved that greatly improve proteasome-dependent antigen processing. IFN-γ induces the synthesis of the immunosubunits β1i [also known as low-molecular-weight protein 2 (LMP2)], β2i [multicatalytic endopeptidase complex-like-1 (MECL1)], and β5i (LMP7) (50–54). The proforms of these subunits are cooperatively incorporated into nascent 20S proteasomes, thereby replacing their constitutively expressed homologs β1 (δ), β2 (z), and β5 (MB1) (55, 56). Thus, two types of proteasomes exist, i.e., constitutive proteasomes that are constitutively expressed in all cells of an organism and immunoproteasomes that are formed upon exposure of cells to IFN-γ (Fig. 2). Nevertheless, a certain tissue-specific constitutive expression can be observed for immunoproteasomes in immuno-relevant tissues like spleen and small intestine (57). Whether this expression is due to a local cytokine-dependent induction or the result of an alternative tissue-specific transcriptional regulation of the immunosubunits is not known.

Both constitutive proteasomes and immunoproteasomes are formed upon de novo assembly in a complex multistep biogenesis program. The β-subunits are synthesized as inactive proteins that mature within the proteasome precursor complexes. Final autocatalytic activation requires the formation of a so-called preholoproteasome assembly intermediate

Strehl et al. · The proteasome and MHC class I antigen processing that allows the cis- and trans-autocatalytic removal of the propeptides, thus liberating the N-terminal active site threonine of the now fully active 20S proteasome (58, 59).

Proteasome biogenesis is facilitated by several additional factors, one of them being the proteasome maturation protein (POMP) (60, 61). Interestingly, this protein, which directly interacts with proteasome precursor complexes of both constitutive and immunoproteasomes, becomes the first substrate of the matured active 20S proteasome. The observation that POMP mRNA levels are upregulated upon INF-γ stimulation suggested that POMP may also play an important role in immunoproteasome biogenesis (62). In support of this observation, experiments recently performed in our laboratory showed a functional interdependence of POMP and the immunosubunit β5i, in that the kinetics of immunoproteasome assembly is determined by the presence of active β5i/LMP7 subunits in combination with sufficient availability of POMP. As a result, immunoproteasome assembly is accelerated by a factor of three to four when compared with that of constitutive proteasomes (63). Thereby, both β5i and POMP seem to be of equal importance. In humans, two splice variants of the β5i mRNA exist, i.e., E1-LMP7 and E2-LMP7 (64, 65), which encode different prosequences. The E1-LMP7 protein is poorly translated and inefficiently incorporated into immunoproteasomes (64). Interestingly, we recently observed that tumor cells that strongly expressed the E1-LMP7 mRNA and hence lacked functional β5I-subunit revealed an impaired immunoproteasome assembly, even in the presence of the two other immunosubunits and increased POMP levels. Similarly, the presence of a proteolytically inactive E2-LMP7-subunit slowed down proteasome maturation (Heink et al., unpublished data). The strongly accelerated biogenesis of immunoproteasome formation providing cells rapidly with immune-adapted proteasome complexes

**Constitutive proteasome**                                                                 **Immunoproteasome**                                                                 **PA28**

![Diagram](attachment:diagram.png)

Fig. 2. Formation of immunoproteasomes. The 20S proteasome (constitutive as well as immuno) consists of 28 (14 different) subunits arranged as four heptameric, staggered rings. The outer rings contain the α-subunits (α1–α7); the two inner rings contain the β-subunits (β1–β7). Three of the seven constitutive β-subunits (β1/δ, β2/z, β5/MB1) in each β-ring contain the active sites. Therefore, the eukaryotic 20S proteasome contains six active sites. Interferon-γ (IFN-γ) induces the synthesis of three active sites containing β-subunits, β1i/LMP2, β2i/

MECL1, β5i/LMP7, and the synthesis of the proteasome maturation protein (POMP). During de novo proteasome formation, the immunosubunits are preferentially incorporated into the enzyme complex to form the immunoproteasome. IFN-γ also induces the synthesis of the proteasome activator PA28 subunits PA28α and PA28β. The two subunits form a heptameric ring with a PA28α₃/PA28β₄ stoichiometry. PA28 binds to the outer α-rings of the 20S proteasome thereby opening the central gate and facilitating substrate entry [modified from Kloetzel (22)].
correlates well with its proposed role in mounting an early and successful cytotoxic immune response. Remarkably, immunoproteasomes appear also to be special in that their half-life of only 21h is considerably shorter than that observed for constitutive proteasomes (120h). The rapid turnover of immunoproteasome is independent of the presence of cytokines and thus appears to reflect an intrinsic structural property of immunoproteasomes (63).

IFN-γ-induced rapid upregulation of immunoproteasomes therefore appears to be designed as a transient response during the early phases of an infection. It allows cells to adapt quickly to the immunological requirements and to return rapidly to a normal situation once immunoproteasome function is not needed anymore. As such immunoproteasomes exhibit features typical of cytokine-regulated proteins.

These new insights into the regulation of immunoproteasome formation and turnover indicate that prolonged high immunoproteasome levels might be harmful and that immunoproteasomes may not be designed to execute all the functions normally performed by constitutive proteasomes. In fact, the observation that the level of immunoproteasome expression in tissues after infection rarely rises above 60% of the total proteasome population (Strehl et al., unpublished observation) and that even in professional antigen-presenting cells immunoproteasomes do not replace constitutive proteasomes completely (66) may be taken in support of the above argument. Another interesting aspect of immunoproteasome regulation is the recent observation that due to dephosphorylation of the α7-subunit in response to IFN-γ stimulation, 26S proteasomes are destabilized, resulting in an increase of free 20S immunoproteasomes (67). The potential functional consequences of increased amounts of free 20S immunoproteasomes have previously been discussed (22).

Antigen processing in the presence of immunoproteasomes and PA28

Up to 99% of the generated peptides are readily degraded by amino peptidases (19), raising the questions of what mechanisms guarantee the sufficient and appropriate supply of MHC class I epitopes and whether immunoproteasomes and PA28 are involved in this process. In other words, do immunoproteasomes/PA28 generate antigenic peptides by alternative cleavage site usage that results in the generation of a qualitatively different set of peptides being more adapted to TAP-dependent transport and the requirements of MHC class I binding? Or, do immunoproteasomes/PA28 generate a similar set of peptides as constitutive proteasomes but by increased numbers and more efficiently to overcome the high peptide destruction rate within the cell to satisfy the needs of increased peptide supply?

Possible answers to the question whether the cytokine-regulated, modulating components of the UPS affect peptide quality or peptide quantity strongly depend on whether one approaches these questions from a more biochemical or from a more immunological point of view. We therefore discuss experiments explaining why it is important to distinguish between the two different approaches and why regulating epitope quantity may be a more complicated issue than previously thought.

Experiments in which HeLa cells were infected with vaccinia virus expressing the hepatitis B virus (HBV) core antigen showed that the efficient liberation and presentation of the HBVcAg<sub>141–151</sub> epitope essentially required the stimulation of HeLa cells with IFN-γ (68). No measurable peptide-specific CTL response was triggered in the absence of immunoproteasomes. Accepting the ability to stimulate CTLs as the most sensitive immunological read-out to assess cellular proteasome function, one would have to conclude from these experiments that immunoproteasomes determine the quality of epitope generation and as such also the immunological phenotype of a cell or tissue. In full support of the above finding, in vitro processing experiments with a synthetic polypeptide containing the epitope and using purified constitutive proteasomes from T2 cells and immunoproteasomes from T2 cells transfected with immunosubunits also demonstrated the essential requirements for immunoproteasomes.

However, highly sensitive analysis by mass spectrometry of the generated peptide products revealed that the HBVcAg<sub>141–151</sub> epitope is also produced by constitutive proteasomes, albeit with a dramatically reduced efficiency. In consequence, the biochemical approach reveals that the functional difference between the two types of proteasomes at the molecular level does not necessarily reside in the complete inability of constitutive proteasomes to generate an epitope or epitope precursor peptide but rather in its enormous inefficiency to do so. On the basis of these and other similar observations (69), it must be inferred that at the biochemical level immunoproteasomes strongly influence the available amount and therefore the quantity of a given epitope. As a result, effects at the immunological level become detectable once a certain quantitative threshold is reached.

The influence of immunoproteasomes on antigen presentation has mostly been studied on viral epitopes. From these studies in which the generation of a large number of different
viral epitopes of various origins was analyzed in great detail, one has to conclude that in the majority of cases, the presence of immunoproteasomes has a positive effect on MHC class I antigen presentation (3, 70–75). So far there exists no published example in which immunoproteasomes exerted a negative effect on the generation and presentation of a viral epitope. Applying cell lines for the analysis of viral epitope generation that either constitutively expressed immunoproteasome or allowed the regulated tetracycline (TET)-dependent expression of immunosubunits revealed that in many cases in which constitutive proteasomes already supported low or moderate CTL stimulation, immunoproteasome expression resulted in an accelerated and improved antigen presentation (69). Analysis of the processing of the adenovirus E1B-derived epitope E1B<sub>192-20</sub> showed that TET-regulated immunoproteasome expression increased E1B<sub>192-20</sub> epitope-presentation efficiency. The experiments also demonstrated that optimal CTL activation after Adeno 5 infection was achieved very early and at extremely low cellular immunoproteasome concentrations, indicating a certain functional dominance of immunoproteasomes within the antigen-processing pathway. In support of the above experiments, MHC class I presentation of an immunodominant epitope of the MuLV gag protein was also enhanced after immunoproteasome induction, whereas the presentation of a subdominant MuLV env epitope was not obviously influenced by immunoproteasomes (70).

In vitro digestion experiments using a cell-free system composed of purified 20S proteasomes (constitutive and immunoproteasomes) with synthetic polypeptides as substrates encompassing the epitopes and the natural flanking sequences reproduced the observations made in intact cellular systems with surprisingly high fidelity. For example, experiments using peptide-specific CTL assays as a ‘read-out’ system showed that the amount of the adenovirus E1B-derived epitope, referred to above (69), generated in vitro by 20S immunoproteasomes closely correlated the effect of immunoproteasomes in in vivo experiments.

A large number of quantitative analyses by mass spectrometry combined with high-performance liquid chromatography of peptide fragments produced in vitro from different virus-derived polypeptides harboring MHC class I epitopes revealed that the presence of immunosubunits affects the cleavage site preference of the proteasome and hence changes the relative amounts at which an epitope or its precursor peptide is generated. However, the observation that in vitro substrate turnover by 20S immunoproteasomes in many cases was accelerated compared with constitutive proteasomes complicated the quantitative analysis of in vitro epitope generation.

Therefore, in evaluating such studies and drawing conclusions for the in vivo situation, one has to discriminate between peptide fragments that are more abundant due to accelerated substrate turnover and those that are more prevalent due to altered proteasomal cleavage site usage. We recently overcame this problem by using the so-called antitopes, i.e. peptide fragments generated by cleavage within an epitope sequence but containing the correct N-terminus of the epitope or the epitope precursor, as internal reference for the quantification of relative amounts of epitopes that are generated in vitro by either constitutive or immunoproteasomes (Strehl et al., unpublished observation).

The main conclusion that can be drawn from these experiments is that immunoproteasomes, when compared with constitutive proteasomes, possess an altered cleavage site preference with a strong predominance to cleave behind residues that represent the correct C-terminus of an MHC class I epitope.

Analysis of larger substrates also revealed that immunoproteasomes preferably generate epitope precursor peptides with a more extended N-terminal sequence that will facilitate TAP transport and require trimming by amino peptidases (76, 77).

Depending on the amino acid sequence that surrounds an epitope in a protein, the relative frequency with which certain cleavage sites are used to generate an epitope can differ dramatically between constitutive and immunoproteasomes. As a consequence, immunoproteasome activity will influence the immunological phenotype and hence affect T-cell selection by changing the relative prevalence of epitopes.

In this context, studies using immunoproteasome-deficient mice are of particular interest. Since in each case these mice lack only one of the immunosubunit-encoding genes, they are only devoid of complete immunoproteasomes. Nevertheless, they will express proteasomes that do contain βi1 and β2i in β5i-deficient mice or β5i and some of β2i in β1i-deficient mice. Analyses of different influenza epitopes as well as self-epitopes in these mice showed that immunoproteasomes can influence immunodominance hierarchies of antiviral CD8<sup>+</sup> T cells by generating different amounts of a specific epitope (78). In few cases, the lack of fully functional immunoproteasomes however may also shape the available T-cell repertoire, in that certain viral epitopes or self-epitopes will not be efficiently generated while others are (78, 79). In addition, mutagenesis studies performed in cell systems had shown before that immunoproteasomes containing an inactive β1i/LMP2 immunosubunit exhibit altered antigen-processing characteristics (80). As shown by mass spectrometry, the relative prevalence of a given peptide within a peptide

Immunological Reviews 207/2005 25

population generated by proteasomes will be altered and thus
can influence the peptide-specific CTL response.
    IFN-γ modulates the activity of the cellular antigen-
processing machinery not only through induction of immuno-
proteasome formation but also through the induced synthesis
of the proteasome activator PA28. Both in vitro and in vivo
studies have linked PA28 to MHC class I antigen processing.
Early in vitro studies have shown that purified and recombinant
PA28 augments proteasome-dependent liberation of epitopes
apparently by facilitating coordinated dual cleavage events in
epitope containing polypeptide substrates derived from the
mouse cytomegalovirus (MCMV) IE pp89, the human tyrosine
kinase JAK1, and human p53 (81–84). Overexpression of
PA28 in mouse fibroblasts also was shown to increase the
sensitivity for lysis by CTLs directed against an MCMV-pp89-
derived epitope. In addition, TET-regulated expression of
PA28αβ substantially increased the presentation of a MuLV
gagL-derived CTL epitope. The investigation of a larger num-
ber of viral antigens revealed that PA28 enhances the presen-
tation of some viral epitopes without apparently increasing
overall protein turnover or the turnover of viral protein sub-
strates, while the presentation of other virus-derived epitopes
was not affected (70). Thus, similar to immunoproteasomes,
PA28 does not affect presentation of all MHC class I epitopes
equally. However, somewhat in contrast to immunoprotea-
somes, PA28 seems to affect the generation and presentation
of only a selective and minor fraction of epitopes. It has to be
kept in mind, however, that PA28 function has been analyzed
in considerably fewer cases and that the experimental condi-
tions and read-out systems suitable for analysis of immuno-
proteasome function may not be fully appropriate for PA28
analysis.
    Detailed in vitro analysis of the effects of PA28 on proteaso-
mal cleavage site usage revealed that PA28 does not confer
new cleavage specificities. Instead, PA28 markedly enhances
the frequency of usage of specific preferred or minor cleavage
sites, resulting in an immediate liberation of the intervening
peptide fragments (82, unpublished observation). Similar to
immunoproteasomes, PA28 also induces a drastically acceler-
ated turnover of synthetic polypeptide substrates that harbor
MHC class I epitopes.
    The most striking example for PA28-dependent epitope
presentation was recently obtained, when the processing and
presentation of two CTL epitopes derived from a melanoma
differentiation antigen, i.e. tyrosinase 2 (TRP2), were ana-
lyzed in the mouse as well as in the human system (85, 86).
The investigations of human cells revealed that melanoma 18a
cells failed to present TRP2<sub>360–368</sub>, while the presentation of
the TRP2<sub>288–296</sub> was not affected. The inability of melanoma
18a cells to present the TRP2<sub>360–368</sub> epitope correlated with
an impaired expression of PA28. Epitope presentation and the
ability to activate TRP2<sub>360–368</sub> epitope-specific CTLs by mel-
anoma 18a cells, however, could be fully restored by transfec-
tion with cDNAs encoding the two PA28-subunits. Furthermore, in vitro studies performed with a TRP2-derived
polypeptide encompassing the TRP2<sub>360–368</sub> epitope also
demonstrated that the presence of PA28 was essential for the
20S proteasome’s ability to liberate significant amounts of the
TRP2<sub>360–368</sub> epitope. Our mass spectrometric analyses did not
reveal major changes in cleavage site quality in the presence of
PA28. Instead, the specific cleavage sites that were not domi-
nant in the absence of PA28 appeared to be dramatically
enhanced at early time points of the processing reaction
(86). Since in vivo the generation of the second epitope
(TRP2<sub>288–296</sub>) was not impaired, the dramatic effects of
PA28 on TRP2<sub>360–368</sub> epitope liberation are unlikely to be
solely explained by the opening of the 20S proteasome's
catalytic channel or altered protein turnover rates. Instead,
these data and recent mutational analysis of PA28 suggest
that PA28 binding induces subtle conformational changes
within the 20S complex that may alter the accessibility of
the active site pockets or their binding affinity for certain
epitope-harboring protein stretches (86).
    Combining the data obtained by the analysis of immuno-
proteasomes and PA28 under different experimental condi-
tions and in different experimental systems, it can be
concluded that both the incorporation of immunosubunits,
which has been shown to be independent of PA28 expression
(85, 88), and binding of PA28 to the outer α-rings of con-
stitutive proteasomes induce subtle structural changes within
the 20S catalytic core unit. As a consequence, proteasomes
exhibit a weighted, altered cleavage site preference to the
effect that the relative and absolute numbers of an antigenic
peptide generated by the proteasome as well as the immuno-
logical phenotype of an antigen-presenting cell can be drama-
tically altered.
    Using the ability to stimulate CTLs as a read-out system, only
epitopes whose generation is affected by either immunoprotea-
some or by PA28 have been identified. No experimental evi-
dence exists that immunoproteasomes and PA28 exert
cooperative or additive effects with regard to the cellular
immune response. This lack would suggest that PA28 and the
immunosubunits differ in their molecular effect on the protea-
some structure and possibly complement each other function-
ally to regulate the relative amounts in a wide spectrum of
antigenic peptides. Interestingly, our own data also suggest that
PA28 exerts stronger effects on constitutive proteasomes than on immunoproteasomes (Seifert, unpublished observation).

### Mixed proteasome complexes

A critical point, which for experimental simplicity in most investigations has so far been neglected and which has not been part of the arguments here, is that within an organism there are no ideal cells or tissues that either express immunoproteasomes or constitutive proteasomes only (57), and almost all tissues constitutively express at least low amounts of PA28 (35). The situation is even more complex since IFN-γ not only induces the formation of immunoproteasomes but also that of intermediate 20S proteasome subtypes in which immunosubunits and active site constitutive subunits coexist (89, 90). In particular, soon after cytokine stimulation, when PA28 and large amounts of immunoproteasomes are formed *de novo*, the cellular proteasome population is shaped in its subtype composition and consequently in its functional properties.

Biochemical analysis of mixed proteasome populations showed that they exhibit slightly altered proteolytic properties. The dynamic of the changes in proteasome activity during cytokine exposure becomes even more evident when the antigen-processing capacity of tissue-derived proteasome populations is analyzed at different time intervals after infection and systemic IFN-γ release (Strehl et al., unpublished observation). Investigating the antigen-processing capacity of proteasome populations in combination with PA28 at different time points during the very early phases of IFN-γ stimulation also showed, depending on proteasome subtype composition, that the relative amounts of specific antigenic peptides generated can vary dramatically over time (Seifert et al., unpublished observation). From these studies, it can be inferred that different antigenic peptides may be generated with different relative efficiencies during a cytokine response and that cytokine induction increases the functional diversity of the proteasome pool in cells, allowing them to produce a broad range of peptides above the immunological detection limit.

Accepting that IFN-γ enhances MHC class I antigen presentation and that short-lived nascent proteins or DRiPs represent an important source for the generation of antigenic peptides raises an unresolved question: does IFN-γ modulate the proteasome system and the downstream proteolytic cascade in a way that a constant number of nascent short-lived proteins is used more efficiently to increase the number of peptides? Or, does IFN-γ also influence the size of the pool of short-lived proteins and consequently provide larger amounts of ubiquitinated short-lived substrates?

The existing experimental evidence is not in support of the idea that upregulation of PA28 or immunoproteasomes has any significant influence on the overall half-life of cellular proteins. However, there exist no detailed experiments published studying the influence of IFN-γ on the size of the pool of short-lived proteins. If IFN-γ were to significantly increase the pool of short-lived proteins or DRiPs, one would expect an increase in the amounts of free ubiquitin and in the enzymes responsible for polyubiquitination and deubiquitination. However, the INF-α-induced upregulation of the two ubiquitin-conjugating enzymes UbcH8 and UbcH5 has been reported (91).

#### An important role for INF-γ-induced aminopeptidases

The sextet of IFN-γ-responsive upregulated components of the proteasome systems, i.e. POMP, PA28αβ, and the three immunoproteasome subunits, is complemented by IFN-γ-induced TAP1 and TAP2, responsible for efficient transport of peptides into the ER, and by a small set of aminopeptidases that mediate the productive N-terminal trimming of antigen-precursor peptides (91). The first IFN-γ-inducible aminopeptidase identified was leucine aminopeptidase (LAP), which is localized in the cytosol. On the basis of in vitro experiments, it was suggested that LAP can contribute to the trimming process of antigenic peptides (92).

For the mouse ER-resident IFN-γ-inducible aminopeptidase, ERAP1 or ERAAP, it was shown that it plays an essential role in the trimming of antigenic precursor peptide (93, 94) and that its overexpression enhanced the presentation of an ovalbumin-derived epitope SIINFKL from N-terminally extended precursors in the ER of intact cells. Furthermore, silencing of mRNA expression by RNA interference prevented the MHC class I presentation of the epitope (94). In humans, a second IFN-γ-inducible ER-resident aminopeptidase, L-RAP, was identified. L-RAP can trim several antigenic precursor peptides including a human immunodeficiency virus-Nef antigen (95). Interestingly, no homolog of L-RAP appears to exist in mice, suggesting that in humans the ER is equipped with several more specialized trim peptidases. A more in-depth discussion can be found by van Endert in this issue.

#### Two functionally different antigen-processing cascades

Accumulating evidence suggests that in order to optimize the regulation of the cellular immune response, two proteolytic cascades have evolved that centered on the UPS control of MHC class I antigen presentation. The ‘constitutive proteolytic cascade’ is designed to eliminate and efficiently degrade cellular short-lived regulatory, potentially toxic, or falsely folded

Immunological Reviews 207/2005   27
Strehl et al · The proteasome and MHC class I antigen processing

Fig. 3. Immunoproteasomes affect the size of the antigenic peptide pool. (A). The 'constitutive proteolytic cascade' is designed to efficiently eliminate proteins and to degrade them to single amino acid residues. Only a small percentage of peptides generated by the proteasome system survive to be presented at the cell surface. (B). The 'immune-adaptive proteolytic' cascade is designed to generate larger amounts of appropriate antigenic peptides to allow more peptides to overcome the proteolytic restrictions of the constitutive system and to provide larger numbers of peptides for major histocompatibility complex class I antigen presentation [modified from Kloetzel (22)].

proteins to single amino acid residues. This constitutive system, controlling cellular protein homeostasis, works with a high efficiency, allowing only a small percentage of peptides generated by the proteasome system to survive the proteolytic attack of aminopeptidases and to be presented at the cell surface. In contrast, the immune-adaptive proteolytic cascade, which is built up very rapidly as an early cellular response following IFN-γ stimulation, is designed to generate larger amounts of appropriate antigenic peptides to allow more peptides to overcome the proteolytic restrictions of the constitutive system and to provide larger numbers of peptides for MHC class I antigen presentation (Fig. 3). During an immune response, the balance between the two coexisting proteolytic cascades gradually changes in favor of the immune-adapted cascade. Since the constitutive and the immune-adapted cascade share a large number of components, a functional cross-talk exists between them, thus giving rise to an optimized generation and presentation of antigenic peptides.

### References

1. Rock K, Goldberg AL. Degradation of cell proteins and the generation of MHC class I presented peptides. Annu Rev Immunol 1999;17:737–779.
2. York LA, Goldberg AL, Mo XY, Rock KL. Proteolysis and class I major histocompatibility complex antigen presentation. Immunol Rev 1999;172:29–48.
3. Kloetzel PM. Antigen processing by the proteasome. Nat Rev Mol Cell Biol 2001;2:179–187.
4. Momburg F, Hämmerling GJ. Generation of TAP mediated transport of peptides for major histocompatibility complex class I molecules. Adv Immunol 1998;68:191–256.
5. Ortmann B, et al. A critical role for tapasin in the assembly and function of multimeric MHC class I-TAT complex. Science 1997;277:1306–1309.
6. Falk K, et al. Identification of naturally processed viral nonapeptides allows their quantification in infected cells and suggests an allele-specific T cell epitope forecast. J Exp Med 1991;174:425–434.
7. Rammensee HG, Friede T, Stevanovic S. MHC ligands and peptide motifs: first listing. Immunogenetics 1995;41:178–228.
8. Reits EAJ, Vos JC, Grommé M, Neefjes J. The major substrates for TAP in vivo are derived from newly synthesized proteins. Nature 2000;404:744–748.
9. Wheatley DN, Giddings MR, Inglis MS. Kinetics of degradation of ‘short-’ and ‘long-lived’ proteins in cultured mammalian cells. Cell Biol Int Rep 1980;4:1081–1091.
10. Wheatley DN, Grisola S, Hernandez-Yago J. Significance of rapid degradation of newly synthesised proteins in mammalian cells: a working hypothesis. J Theor Biol 1982;98:283–300.
11. Wheatly DN. Intracellular protein degradation: basis of a self-regulating mechanism for the proteolysis of endogenous proteins. J Theor Biol 1984;107:127–149.
12. Schubert U, Antón LC, Gibbs J, Norbury CC, Yewdell JW, Bennink JR. Rapid degradation of a large fraction of newly synthesised proteins by proteasomes. Nature 2000;404:770–774.
13. Turner CC, Varshavsky A. Detecting and measuring cotranslational protein degradation. Science 2000;289:2117–2120.
14. Princiotta MF, et al. Quantitating protein synthesis, degradation, and endogenous antigen processing. Immunity 2003;18:342–354.
15. Yewdell JW, Princiotta MF. Proteasomes get by with lots of help from their friends. Immunity 2004;20:362–363.
16. Lehner PJ. The calculus of immunity: quantitating antigen processing. Immunity 2003;18:315–317.
17. Yewdell JW, Reits E, Neefjes J. Making sense of mass destruction: quantitating MHC class I antigen presentation. Nat Rev Immunol 2003;3:952–961.
18. Ciechanover A. Proteolysis: from the lysosome to ubiquitin and the proteasome. Nat Rev Mol Cell Biol 2005;6:79–87.
19. Reits E, et al. Peptide diffusion, protection, and degradation in nuclear and cytoplasmic compartments before antigen presentation by MHC class I. Immunity 2003;19:97–108.

28 Immunological Reviews 207/2005

20. York IA, et al. The cytosolic endopeptidase, thimet oligopeptidase, destroys antigenic peptides and limits the extent of MHC class I antigen presentation. Immunity 2003; **3**:429–440.
21. Rock KL, York IA, Goldberg AL. Post-proteasomal antigen processing for major histocompatibility complex class I presentation. Nat Immunol 2004; **5**:670–677.
22. Kloetzel PM. Generation of major histocompatibility complex class I antigens: functional interplay between proteasomes and TPPII. Nat Immunol 2004; **5**:661–669.
23. Voges D, Zwickl P, Baumeister W. The 26S proteasome: a molecular machine designed for controlled proteolysis. Annu Rev Biochem 1999; **68**:1015–1068.
24. Braun BC, et al. The base of the proteasome regulatory particle exhibits ATP-dependent chaperone-like activity. Nat Cell Biol 1999; **1**:221–226.
25. Groll M, Ditzel L, Löwe J, Stock D, Bochtler M, Bartunik HD, Huber R. Structure of 20S proteasome from yeast at 2.4 Å resolution. Nature 1997; **386**:463–471.
26. Kisselev AF, et al. The caspase like site of proteasomes, their substrate specificities, new inhibitors and substrates and allosteric interaction with the trypsin like site. J Biol Chem 2003; **278**:35869–35877.
27. Eleuteri AM, Kohanski RA, Cardozo C, Orlowski M. Bovine spleen multicatalytic proteinase complex (proteasome). Replacement of X, Y, and Z subunits by LMP7, LMP2 and MECL1 and changes in properties and specificities. J Biol Chem 1997; **272**:11824–11831.
28. Beekman NJ, et al. Abrogation of CTL epitope processing by single amino acid substitution flanking the C-terminal proteasome cleavage site. J Immunol 2000; **164**:1898–1905.
29. Groll M, et al. A gated channel into the proteasome core particle. Nat Struct Biol 2000; **7**:1062–1067.
30. Ferrell K, Wilkinson CRM, Dubiel W, Gordon C. Regulatory subunit interactions of the 26S proteasome, a complex problem. Trends Biochem Sci 2000; **25**:83–88.
31. Glickman MH, et al. A subcomplex of the proteasome regulatory particle required for ubiquitin-conjugate degradation and related to the COP9-signalosome and eIF3. Cell 1998; **94**:615–623.
32. Dubiel W, Pratt G, Ferrell K, Rechsteiner M. Purification of an 11S regulator of the multicatalytic proteinase. J Biol Chem 1992; **267**:22369–22377.
33. Ma CP, Slaugther CA, DeMartino GN. Identification, purification and characterization of a protein activator (PA28) of the 20S proteasome (macropain). J Biol Chem 1992; **267**:10115–10123.
34. Soza A, Knuehl C, Groettrup M, Henklein P, Tanaka K, Kloetzel PM. Expression and subcellular localization of mouse 20S proteasome activator PA28. FEBS Lett 1997; **413**:27–34.
35. Stohwasser R, Salzmann U, Giesebracht J, Kloetzel PM, Holzhütter HG. Kinetic evidence for facilitation of peptide channelling by the proteasome activator PA28. Eur J Biochem 2000; **276**:6221–6229.
36. Knowlton JR, Johnston SC, Whitby FG, Realini C, Zhang Z, Rechsteiner M, Hill CP. Structure of the proteasome activator REGalpha (PA28alpha). Nature 1997; **390**:639–643.
37. Hill CP, Masters EI, Whitby FG. The 11S regulators of 20S proteasome activity. Curr Top Microbiol Immunol 2002; **268**:73–89.
38. Hendil KB, Khan S, Tanaka K. Simultaneous binding of PA28 and PA700 activators to 20S proteasomes. Biochem J 1998; **332**:749–754.
39. Kopp F, Dahlmann B, Kuehn L. Reconstitution of hybrid proteasomes from purified PA700-20S complexes and PA28alphabeta activator: ultrastructure and peptidase activities. J Mol Biol 2001; **313**:465–471.
40. Kisselev AF, Akopian TN, Woo KM, Goldberg AF. The sizes of peptides generated from protein by mammalian 26S and 20S proteasomes. Implications for understanding the degradative mechanism and antigen presentation. J Biol Chem 2002; **274**:3363–3371.
41. Rock KL, Gramm C, Rothstein L, Clark K, Stein R, Dick L, Hwang D, Goldberg AL. Inhibitors of the proteasome block the degradation of most cell proteins and the generation of peptides presented on MHC class I molecules. Cell 1994; **78**:761–771.
42. Rock LK, York IA, Saric T, Goldberg AL. Protein degradation and the generation of MHC class I-presented peptides. Adv Immunol 2002; **80**:1–59.
43. Theobald M, et al. A mutational hotspot in p53 protects cells from lysis by CTL specific for a flanking epitope. J Exp Med 1998; **11**:1017–1020.
44. Seifert U, et al. Hepatitis C virus mutation affects proteasomal epitope processing. J Clin Invest 2004; **114**:250–259.
45. Nielsen M, Lundegaard C, Lund O, Kesmir C. The role of the proteasome in generating cytotoxic T-cell epitopes: insights obtained from improved predictions of proteasomal cleavage. Immunogenetics 2005; **57**:33–41.
46. Hakenberg J, et al. MAPPPP: MHC class I antigenic peptide prediction. Appl Bioinformatics 2003; **2**:155–158.
47. Holzhuetter HG, Froemmel C, Kloetzel PM. A theoretical approach towards the identification of cleavage determining amino acid motifs of the 20S proteasome. J Mol Biol 1999; **286**:1251–1265.
48. Rosenzweig SD, Holland SM. Defects in the interferon-γ and interleukin-12 pathways. Immunol Rev 2005; **203**:38–47.
49. Aki M, et al. Interferon-gamma induces different subunit organizations and functional diversity of proteasomes. J Biochem 1994; **115**:257–269.
50. Driscoll J, Brown MG, Finley D, Monaco JJ. MHC-linked LMP gene products specifically after peptidase activities of the proteasome. Nature 1993; **365**:262–264.
51. Ortiz-Naravete V, Seelig A, Gernold M, Frentzel S, Kloetzel PM, Hammerling GJ. Subunit of the 20S proteasome (multicatalytic proteinase) encoded by the major histocompatibility complex. Nature 1991; **353**:662–762.
52. Glynne R, Powis S, Beck S, Kelly A, Kerr LA, Trowsdale J. A proteasome related gene between the two abc transporter loci in the calls II region of the human MHC. Nature 1991; **353**:357–360.
53. Cerundolo V, Kelly A, Elliott T, Trowsdale J, Townsend A. Genes encoded in the major histocompatibility complex affecting the generation of peptides for TAP transport. Eur J Immunol 1995; **25**:554–562.
54. Cruz M, Elenich LA, Smolarek TA, Menono AG, Monaco JJ. DNA sequence, chromosomal localization and tissue expression of the mouse proteasome subunits LMP10 (Psmb10) gene. Genomics 1997; **45**:618–622.
55. Griffin TA, Nandi D, Cruz M, Fehling HJ, Kaer LV, Monaco JJ, Colbert RA. Immunoproteasome assembly: cooperative incorporation of interferon gamma (IFN-gamma)-inducible subunits. J Exp Med 1998; **187**:97–104.
56. Groettrup M, Standera S, Stohwasser R, Kloetzel PM. The subunits MECL-1 and LMP2 are mutually required for incorporation into the 20S proteasome. Proc Natl Acad Sci USA 1997; **94**:8970–8975.
57. Kuckelkorn U, et al. Link between organ-specific antigen processing by 20S proteasomes and CD8(+) T cell-mediated autoimmunity. J Exp Med 2002; **195**:983–990.
58. Schmidtke G, et al. Analysis of mammalian 20S proteasome biogenesis: The maturation of β-subunits is an ordered two-step mechanism involving autocatalysis. EMBO J 1996; **15**:6887–6898.
59. Nandi D, Woodward E, Ginsburg DB, Monaco JJ. Intermediates in the formation of mouse 20S proteasomes: implications for the assembly of precursor subunits. EMBO J 1997; **16**:5363–5375.
60. Ramos PC, Höckendorff J, Johnson ES, Varshavsky A, Dohmen RJ. Ump1p is required for proper maturation of the 20S proteasome and becomes its substrate upon completion of the assembly. Cell 1998; **92**:489–499.

61. Witt E, Zantopf D, Schmidt M, Kraft R, Kloetzel PM, Krüger E. Characterization of the newly identified human Ump1 homologue POMP and analysis of LMP7 (β5i) incorporation into 20S proteasomes. J Mol Biol 2000;301:1–9.

62. Burri L, Höckendorff J, Boehm U, Klamp T, Dohmen JR, Lévy F. Identification and characterization of a mammalian protein interacting with 20S proteasome precursors. Proc Natl Acad Sci USA 2000;97:10348–10353.

63. Heink S, Ludwig D, Kloe PM, Kruger E. IFN-γ-induced immune adaption of the proteasome system is an accelerated and transient response. Proc Natl Acad Sci USA 2005;102:9241–9246.

64. Kingsbury DJ, Griffin TA, Colbert RA. Novel propeptide function in 20S proteasome assembly influences beta subunit composition. J Biol Chem 2000;275:24156–24162.

65. Yang Y, Fruh K, Ahn K, Peterson PA. In vivo assembly of the proteasomal complexes, implications for antigen processing. J Biol Chem 1995;270:27687–27694.

66. Macagno A, Gillet M, Sallust F, Lanzavecchia A, Nestle FO, Groettrup M. Dendritic cells upregulate immunoproteasomes and the proteasome regulator PA28 during maturation. Eur J Immunol 1999;29:4037–4042.

67. Bose S, Stratford FL, Broeadfoot KI, Mason GG, Rivett AJ. Phosphorylation of 20S proteasome alpha subunit C8 (alpha 7) stabilizes the 26S proteasome and plays a role in the regulation of proteasome complexes by gamma interferon. Biochem J 2004;378:177–184.

68. Sijts A, Ruppert T, Rehermann B, Schmidt M, Koszinowski U, Kloetzel PM. Structural features of immunoproteasomes determine the efficient generation of a viral CTL epitope. J Exp Med 2000;191:503–513.

69. Sijts A, Ruppert T, Rehermann B, Schmidt M., Koszinowski U, Kloetzel PM. MHC class I antigen processing of an Adenovirus CTL epitope is linked to the levels of immunoproteasomes in infected cells. J Immunol 2000;164:450–456.

70. van Hall T, Sijts A, Camps M, Offringa R, Melief C, Kloetzel PM, Ossendorp F. Differential influence on CTL epitope presentation by controlled expression of either proteasome immuno-subunits or PA28. J Exp Med 2000;192:483–492.

71. Schwarz K, et al. Overexpression of the proteasome subunits LMP2, LMP7 and MECL-1 but not PA28αβ enhances the presentation of an immunodominant lymphocyte choriomeningitis virus T cell epitope. J Immunol 1994;165:768–778.

72. Van Kaer L, et al. Altered peptide and viral-specific T cell response in LMP2 mutant mice. Immunity 1994;1:533–541.

73. Boes B, Hengel H, Ruppert T, Multhaup G, Koszinowski U, Kloetzel PM. IFN-γ stimulation modulates the proteolytic activity and cleavage site preference of 20S mouse proteasomes. J Exp Med 1994;179:901–909.

74. Kuckelkorn U, Frentzel S, Kraft R, Kostka S, Groettrup M, Kloetzel PM. Incorporation of major histocompatibility complex encoded subunits LMP2 and LMP7 changes the quality of the 20S proteasome polypeptide processing products independent of interferon-γ. Eur J Immunol 1995;25:2605–2611.

75. Sibille C, et al. LMP2 + proteasomes are required for the presentation of specific antigens to cytotoxic T lymphocytes. Curr Biol 1995;5:923–930.

76. Cascio P, Hilton C, Kisselev AF, Rock KL, Goldberg AL. 26S proteasomes and immunoproteasomes produce mainly N-extended versions of an antigenic peptide. EMBO J 2001;20:2357–2366.

77. Knuehl C, Spee P, Ruppert T, Kuckelkorn U, Henklein P, Neefjes J, Kloetzel PM. The murine cytomegalovirus pp89 immunodominant H-2Ld epitope is generated and translocated into the endoplasmic reticulum as an 11-mer precursor peptide. J Immunol 2001;167:1515–1521.

78. Chen W, Norbury CC, Cho Y, Yewdell JW, Bennink JR. Immuno-proteasomes shape immunodominance hierarchies of antiviral CD8(+) T cells at the levels of T cell repertoire and presentation of viral antigens. J Exp Med 2001;193:1319–1326.

79. Toes RE, et al. Discrete cleavage motifs of constitutive and immuno-proteasomes revealed by quantitative analysis of cleavage products. J Exp Med 2001;194:1–12.

80. Schmidtke G, Eggers M, Ruppert T, Groettrup M, Koszinowski U, Kloetzel PM. Inactivation of a defined active site in the mouse 20S proteasome complexes enhances MHC class I presentation of a murine cytomegalovirus protein. J Exp Med 1998;187:1641–1646.

81. Groettrup M, et al. A role for the proteasome regulator PA28α in antigen presentation. Nature 1996;381:166–168.

82. Dick TP, et al. Coordinated dual cleavages induced by the proteasome regulator PA28 lead to dominant MHC ligands. Cell 1996;86:253–262.

83. Shimbara N, Nakajima H, Tanahashi N, Ogawa K, Niwa S, Uenaka A, Nakayama E, Tanaka K. Double-cleavage production of the CTL epitope by proteasomes and PA28: role of the flanking region. Genes Cells 1997;2:785–800.

84. Kuckelkom U, Ferreira EA, Drung I, Liewer U, Kloetzel PM, Theobald M. The effect of the interferon-gamma-inducible processing machinery on the generation of a naturally tumor-associated human cytotoxic T-lymphocyte epitope within a wild-type and mutant p53 sequence context. Eur J Immunol 2002;32:1368–1375.

85. Murata S, et al. Immunoproteasome assembly and antigen presentation in mice lacking both PA28alpha and PA28beta. EMBO J 2001;20:5898–5907.

86. Sun Y, et al. Expression of the proteasome activator PA28 rescues the presentation of a cytotoxic T lymphocyte epitope on melanoma cells. Cancer Res 2002;62:2875–2882.

87. Li J, et al. Lysine 188 substitutions convert the pattern of proteasome activation by REGgamma to that of REGs alpha and beta. EMBO J 2001;20:3359–3369.

88. Schwarz K, Eggers M, Soza A, Koszinowski UH, Kloetzel PM, Groettrup M. The proteasome regulator PA28αβ can enhance antigen presentation without affecting 20S proteasome subunit composition. Eur J Immunol 2000;165:768–778.

89. Belich MP, Glynne RJ, Senger G, Sheer D, Trowsdale J. Proteasome components with reciprocal expression to that of the MHC-encoded LMP proteins. Curr Biol 1994;4:769–776.

90. Dahlmann B, Ruppert T, Kuehn L, Merforth S, Kloetzel PM. Different proteasome subtypes in a single tissue exhibit different enzymatic properties. J Mol Biol 2000;303:643–653.

91. Nyman TA, Matikainen S, Sareneva T, Julkunen I, Kalkkinen N. Proteome analysis reveals ubiquitin-conjugating enzymes to be a new family of interferon-alpha regulated genes. Eur J Biochem 2000;267:4011–4019.

92. Beninga J, Rock KL, Goldberg AL. Interferon-gamma can stimulate post-proteasomal trimming of the N terminus of an antigenic peptide by inducing leucine aminopeptidase. J Biol Chem 1998;273:18734–18742.

93. Serwold T, Gonzalez F, Kim J, Jacob R, Shastri N. ERAAP customizes peptides for MHC class I molecules in the endoplasmic reticulum. Nature 2002;419:480–483.

94. Saric T, Chang SC, Hattori A, York IA, Markant S, Rock KL, Tsujimoto M, Goldberg AL. An IFN-gamma induced aminopeptidase in the ER, ERAP1, trims precursors to MHC class I-presented peptides. Nat Immunol 2002;12:1169–1176.

95. Tanioka T, et al. Human leukocyte-derived arginine aminopeptidase. The third member of the oxyocinase subfamily of aminopeptidases. J Biol Chem 2003;278:32275–32283.
