
Molecular Microbiology (2005) 58(4), 1186–1202

doi:10.1111/j.1365-2958.2005.04902.x  
First published online 14 October 2005

# CsrA and three redundant small RNAs regulate quorum sensing in Vibrio cholerae

Derrick H. Lenz,$^{1}$ Melissa B. Miller,$^{1\dagger}$ Jun Zhu,$^{2}$ Rahul V. Kulkarni$^{3}$ and Bonnie L. Bassler$^{1*}$

$^{1}$ Department of Molecular Biology, Howard Hughes Medical Institute, Princeton University, Princeton, NJ 08544-1014, USA.  
$^{2}$ Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, PA 19104, USA.  
$^{3}$ Department of Physics, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061, USA.

## Summary

Bacteria communicate using a process called quorum sensing which involves production, secretion and detection of signalling molecules called autoinducers. Quorum sensing allows populations of bacteria to simultaneously regulate gene expression in response to changes in cell density. The human pathogen, *Vibrio cholerae*, uses a quorum-sensing circuit composed of parallel systems that transduce information through four redundant regulatory small RNAs (sRNAs) called quorum regulatory RNAs (Qrr) to control the expression of numerous genes, most notably those required for virulence. We show that the VarS/VarA two-component sensory system comprises an additional regulatory input controlling quorum-sensing-dependent gene expression in *V. cholerae*. VarS/VarA controls transcription of three previously unidentified small regulatory RNAs (sRNAs) that are similar to the sRNAs CsrB and CsrC of *Escherichia coli*. The three *V. cholerae* sRNAs, which we name CsrB, CsrC and CsrD, act redundantly to control the activity of the global regulatory protein, CsrA. The VarS/VarA-CsrA/BCD system converges with the *V. cholerae* quorum-sensing systems to regulate the expression of the Qrr sRNAs, and thus, the entire quorum-sensing regulon.

## Introduction

Quorum sensing is a process of chemical communication that bacteria use to monitor cell population density and synchronize the gene expression of the community (Miller and Bassler, 2001). Quorum sensing involves production, release and detection of signalling molecules called autoinducers. Quorum-sensing-controlled behaviours are typically ones that are only productive when carried out in unison by a group of cells. Examples include DNA exchange, sporulation, biofilm formation, virulence factor production and bioluminescence (Miller and Bassler, 2001; Zhu *et al.*, 2002; Hammer and Bassler, 2003; Zhu and Mekalanos, 2003; Yildiz *et al.*, 2004).

The human pathogen *Vibrio cholerae* has multiple quorum-sensing systems that act in parallel (Fig. 1) (Miller *et al.*, 2002). System 1 is composed of the CAI-1 autoinducer of unknown structure and CqsS, its two-component sensor. System 2 is made up of AI-2, a furanosyl borate diester, the periplasmic binding protein, LuxP, and the two-component sensor, LuxQ (Bassler *et al.*, 1994; Chen *et al.*, 2002). At low-cell density, when concentrations of autoinducers are low, the sensors are kinases that autophosphorylate on conserved histidines (H1) and transfer phosphate to conserved aspartates (D1) on their response regulator modules (Freeman *et al.*, 2000). Phosphate is passed to a histidine (H2) on the phosphotransfer protein LuxU and next to an aspartate (D2) on the response regulator LuxO (Freeman and Bassler, 1999; 2000). Phospho-LuxO, with the alternative sigma factor σ⁵⁴, activates expression of four genes encoding small RNAs (sRNAs) (Lilley and Bassler, 2000; Lenz *et al.*, 2004). The sRNAs (called Qrr sRNAs for quorum regulatory RNA) with the sRNA chaperone Hfq, bind to the *hapR* mRNA encoding the master regulator of the quorum-sensing cascade, HapR, and destabilize it. Thus, little HapR protein is made at low-cell density. At high cell density, in response to autoinducers, the sensors switch from kinases to phosphatases. Phosphate flows backward through the circuit rendering LuxO inactive. The *qrr* genes are not transcribed, *hapR* mRNA is stabilized, HapR is translated, and HapR activates or represses a variety of genes. For example, when HapR is produced, it activates protease production and represses virulence factor production and biofilm formation (Fig. 1) (Jobling and Holmes, 1997; Zhu *et al.*, 2002; Hammer and Bassler, 2003; Zhu and Mekalanos, 2003).

A previous study predicted that a third sensory system exists in *V. cholerae* and functions through LuxO but independently of LuxU (Miller *et al.*, 2002). Here we report experiments that identify a sensor kinase-response regu-

Accepted 29 August, 2005. *For* correspondence. E-mail bbassler@molbio.princeton.edu; Tel. (+1) 609 258 2857; Fax (+1) 609 258 2957. $^\dagger$Present address: University of North Carolina School of Medicine, Department of Pathology and Laboratory Medicine, Chapel Hill, NC 27599, USA.

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd

Regulation of quorum sensing in Vibrio cholerae 1187

VarS/VarA-CsrA/BCD System
CAI-1-CqsS System
AI-2-LuxPQ System

OM

VarS
H1
D1
H2
↓
VarA
D2
↓
CsrB, C, D
↓
CsrA

CqsS
H1
D1
P
LuxU
H2
LuxO
D2
+ σ⁵⁴
sRNAs + Hfq
HapR

LuxP
AL-2
LuxQ

IM

virulence
biofilms
other genes

lux
protease
other genes

Fig. 1. Model of the V. cholerae quorum-sensing circuit. The VarS/VarA-CsrA/BCD system functions with the CAI-1-CqsS and AI-2-LuxPQ systems. Pentagons and triangles denote CAI-1 and AI-2 respectively. The flow of phosphate indicated by the arrows depicts the low-cell density state. Flow is reversed at high-cell density. Dotted lines denote hypothetical interactions.

lator pair that we call VarS/VarA as part of this third system. The response regulator, VarA had previously been identified as a regulator of virulence in V. cholerae (Wong et al., 1998). VarS/VarA homologues exist in a variety of Gram-negative bacteria, including Escherichia coli (BarA/UvrY), Salmonella typhimurium (BarA/SirA), members of the genus Pseudomonas (GacS/GacA) and Legionella pneumophila (LetS/LetA) (Heeb and Haas, 2001; Molofsky and Swanson, 2004). While the signal(s) for these systems are not known, it is established that VarS-type proteins act as sensor kinases for VarA-type response regulators (Rich et al., 1994; Pernestig et al., 2001). In E. coli, the VarS/VarA homologues BarA/UvrY activate transcription of genes encoding the sRNAs CsrB and CsrC which bind to and inhibit the activity of the CsrA protein (Liu et al., 1997; Weilbacher et al., 2003). CsrA is a post-transcriptional regulator of diverse processes including carbon metabolism and biofilm formation (Romeo et al., 1993; Romeo, 1998). In Pseudomonas species, the VarS/VarA homologues, called GacS/GacA, activate transcription of rsm genes encoding sRNAs that inhibit the CsrA homologue RsmA (Aarons et al., 2000; Heeb et al., 2002; Valverde et al., 2003; Burrowes et al., 2005). Here we show that, in V. cholerae, VarS/VarA activate expression of three previously unidentified genes encoding sRNAs similar to CsrB of E. coli. We name these sRNAs CsrB, CsrC and CsrD, and demonstrate that they play redundant

roles in the inhibition of V. cholerae CsrA. We further show that VarS/VarA-CsrA/BCD act through the quorum-sensing network to regulate the expression of hapR. Our findings suggest that the VarS/VarA-CsrA/BCD system could function as part of the third sensory system in V. cholerae.

Results

A screen for V. cholerae quorum-sensing regulators reveals VarS and VarA

The V. cholerae quorum-sensing network is similar to that of the bioluminescent marine bacterium Vibrio harveyi (Miller et al., 2002). In V. harveyi, quorum sensing controls the expression of the luciferase operon (luxCDABE) (Nealson and Hastings, 1979; Bassler et al., 1993; 1994). We showed that luxCDABE from V. harveyi is a faithful reporter of quorum-sensing gene expression in V. cholerae. Using luxCDABE as the reporter, Tn5 mutagenesis of V. cholerae was performed, and followed by a screen for decreased light production. This led to identification of the CAI-1-CqsS quorum-sensing system and we showed it functions in parallel with the AI-2-LuxPQ system (Miller et al., 2002).

The above screen yielded mutants with dim/dark phenotypes that did not have insertions in genes encoding known quorum-sensing regulators. We have sequenced

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, Molecular Microbiology, 58, 1186–1202

these insertions, and several are located in two unlinked genes, annotated VC2453 and VC1213. VC2453 and VC1213 encode proteins homologous to GacS and GacA, respectively, which were first identified as global regulators of secondary metabolite production in *Pseudomonas syringae* and *Pseudomonas fluorescens* (Hrabak and Willis, 1992; Laville et al., 1992). VC1213 has previously been identified in *V. cholerae* as the response regulator VarA (Virulence Associated Regulator) and a *varA* mutant has reduced virulence due to decreased cholerae toxin (CT) and toxin-coregulated pilus (TCP) production (Wong et al., 1998). Our *varA* mutant has a virulence defect identical to that previously reported (not shown). VC2453 is predicted to encode a two-component sensor kinase with three conserved domains (H1, D1 and H2). This domain organization is identical to other GacS homologues, and VC2453 is 34% identical to GacS of *Pseudomonas* spp., and 52% and 51% identical to BarA of *E. coli* and *S. typhimurium* respectively. No cognate sensor kinase has been identified for VarA; we suggest VC2453 fulfils this function. Therefore, we name the VC2453 gene product VarS.

To investigate the roles of VarS and VarA in quorum sensing, we measured cell density-dependent light production from *luxCDABE* in various *V. cholerae* strains. All strains were grown to high-cell density and diluted 1000-fold. The characteristic quorum-sensing behaviour is displayed by the wild type (Fig. 2A, open squares). After dilution, light production declines precipitously in response to dilution of the autoinducers to a concentration below the level required for detection. As the culture grows, autoinducers are produced and accumulate. The cells respond by commencing light production. *varS* and *varA* mutants are impaired in production of light. At high-cell density, the *varS* and *varA* mutants produce 100-fold and 10 000-fold less light than the wild type respectively. Mutation of *varA* is more severe than mutation of *varS*. A likely interpretation is that, similar to *E. coli*, mutation of the *barA* homologue, *varS*, partially disrupts signalling through VarA, whereas mutation of the downstream *uvrY* homologue, *varA*, produces a null phenotype (Suzuki et al., 2002).

mutants produce low levels of light (closed circles and closed triangles respectively). The *varS*, *luxO* and *varA*, *luxO* double mutants display constitutive bright *lux* phenotypes (open circles and open triangles respectively). Thus, VarS and VarA require LuxO to transduce information through the circuit, so we place them upstream of LuxO in the regulatory hierarchy (Fig. 1). We note that the *luxO* mutant is 10-fold brighter than the *varS*, *luxO* and *varA*, *luxO* double mutants, however, the double mutants have severe growth defects, and we suspect that this contributes to their inability to produce maximal levels of light. We have observed similar effects in other mutants that show impaired growth (Lenz et al., 2004).

We also examined VarS/VarA function relative to LuxU. Unlike a *luxO* mutant, a *luxU* mutant retains density-dependent expression of *lux* (Fig. 2C, open diamonds). We have argued previously that this phenotype is due to an unidentified System 3 channelling information into the circuit through LuxO but independently of LuxU. The *varS*, *luxU* and *varA*, *luxU* double mutants do not give a clear epistasis result, but instead produce levels of light intermediate between the single *varS* and *varA* mutants and the single *luxU* mutant (Fig. 2C open circles and open triangles respectively). Mutation of *luxU* has the opposite effect on LuxO-phosphate, and in turn on bioluminescence, than does mutation of *varS* or *varA*. In the absence of LuxU, all input from the CAI-1-CqsS and AI-2-LuxPQ systems is eliminated, reducing the level of phospho-LuxO, so a *luxU* strain is brighter than wild type at low-cell density. In contrast, *varS* and *varA* mutants are darker than wild type suggesting that increased phosphate is shuttled to LuxO in these mutants. When we combine the *varS* or *varA* mutation with the *luxU* mutation, we believe we observe an intermediate phenotype because of the competing effects each mutation has on the luminescence readout. Additional evidence follows to support this hypothesis.

We wondered if the decreased Lux production of *varS* and *varA* mutants depended on the kinase input from the CqsS and LuxQ sensors. To investigate this, we constructed *cqsS*, *luxQ*, *varS* and *cqsS*, *luxQ*, *varA* triple mutants. Mutation of *cqsS* and *luxQ* does not alter the *varS* or *varA* phenotype (Fig. 2D; compare closed and open circles and closed and open triangles, respectively), indicating that VarS/VarA operate independently of CqsS and LuxQ. We suggest that the VarS/VarA system acts in parallel to the CAI-1-CqsS and AI-2-LuxPQ systems and channels information into the circuit at the level of LuxO (Fig. 1).

VarS/VarA function in parallel to CAI-1-CqsS and AI-2-LuxPQ

To determine where VarS/VarA channel information into the quorum-sensing pathway, we performed genetic epistasis tests. First we analysed whether LuxO is required for the VarS/VarA effect on bioluminescence (Fig. 2B). *V. cholerae* *luxO* mutants carrying *luxCDABE* are constitutively bright because, in the absence of LuxO, HapR is always present and constitutively activates *lux* expression (Fig. 2B; open diamonds). *varS* and *varA*

VarS/VarA require CsrA to control quorum sensing

Based on homologous systems, we predicted that an additional regulatory component(s) must exist

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **58**, 1186–1202

Fig. 2. The VarS/VarA two-component system controls quorum sensing in *V. cholerae*.

Density-dependent *lux* assays for *V. cholerae* strains: A. MM227 (WT, open squares), MM609 (∆varS, closed circles), MM560 (∆varA, closed triangles). Relative light units (indicated as RLU) are counts min⁻¹ ml⁻¹/OD₆₀₀.

B. MM227 (WT, open squares), MM349 (∆luxO, open diamonds), MM609 (∆varS, closed circles), MM560 (∆varA, closed triangles), DL3698 (∆luxO, ∆varS, open circles), DL3696 (∆luxO, ∆varA, open triangles).

C. MM227 (WT, open squares), MM359 (∆luxU, open diamonds), MM609 (∆varS, closed circles), MM560 (∆varA, closed triangles), MM634 (∆luxU, ∆varS, open circles), MBM32 (∆luxU, ∆varA, open triangles).

D. MM227 (WT, open squares), DL2820 (∆cqsS, luxQ, open diamonds), MM609 (∆varS, closed circles), MM560 (∆varA, closed triangles), DL3282 (∆cqsS, ∆luxQ, ∆varS, open circles), DL3268 (∆cqsS, ∆luxQ, ∆varA, open triangles).

downstream of VarS/VarA and link it to the quorum-
sensing system. To identify this component(s), we per-
formed a varA bypass suppressor screen by subjecting
the dim V. cholerae varA mutant carrying luxCDABE to
Tn5 mutagenesis and screening for mutants restored to
a bright phenotype. Tn5 insertions were located in luxO,
rpoN (encoding σ⁵⁴) and VC0548. Mutations in luxO
and rpoN were anticipated because their inactivation
results in constitutive light production. VC0548 encodes
a protein that is 89% identical to CsrA, a sRNA-binding,
post-transcriptional regulator in E. coli (Romeo et al.,
1993).

The phenotypes of the varS, varA and csrA::Tn5 single
mutants and the varS, csrA::Tn5 and varA, csrA::Tn5 dou-
ble mutants are shown in Fig. 3A. Mutation of csrA in the
wild type causes no significant change in quorum-sensing
phenotype (compare open squares to closed diamonds).
However, in the varS or varA dim mutants, Tn5 insertion
in csrA restores the wild type profile (open circles and
open triangles respectively). Introduction of the wild-type
V. cholerae csrA gene into the varA, csrA::Tn5 mutant
complements the defect and restores the dark varA phe-
notype (data not shown). These results indicate that VarS/
VarA control quorum sensing through CsrA.

We defined the epistatic relationships between csrA
and the genes encoding the quorum-sensing regulators.
Analogous to the results with the varS and varA mutants,
the phenotype of the csrA::Tn5 mutant is converted to the
constitutive bright phenotype of the luxO mutant in the
double csrA::Tn5, luxO mutant showing that input from
CsrA depends on LuxO (data not shown). Similar to
above, the csrA::Tn5 mutation does not alter the pheno-
type of the double cqsS, luxQ mutant. Thus, input from
CsrA does not depend on the known sensors (data not
shown). Combining the csrA::Tn5 mutation with the luxU
null mutation increases light production from the luxU
mutant 50-fold at low-cell density (Fig. 3B; compare open
squares and open diamonds). Mutation of luxU eliminates
significant phospho-flow to LuxO, making the strain
brighter than wild type at low-cell density. Combining the
luxU mutation with the csrA::Tn5 mutation results in a
strain with an even brighter phenotype at low-cell density,
indicating that phosphate flow to LuxO has been further
reduced by mutation of csrA. These results are in contrast

Fig. 3. CsrA operates downstream of VarS/
VarA. Density-dependent lux assays for: A
MM227 (WT, open squares), MM609 (∆varS,
closed circles), MM560 (∆varA, closed trian-
gles), DL2493 (csrA::Tn5, closed diamonds),
DL3404 (∆varS, csrA::Tn5, open circles),
DL2499 (∆varA, csrA::Tn5, open triangles) and
B MM359 (∆luxU, open squares), DL2493
(csrA::Tn5, closed diamonds), DL2489 (∆luxU,
csrA::Tn5, open diamonds).
to the epistasis results with varS and varA and luxU (Fig. 2B). Our interpretation is that, in the case of luxU and varS/varA, the effects of the mutations oppose each other so combining the mutations results in an intermediate phenotype. In the case of luxU and csrA, the effects of the two mutations reinforce one another, and a synergistic effect is observed when the mutations are combined. All of the results are consistent with VarS/VarA-CsrA being components of a third sensory system functioning as depicted in Fig. 1.

### Identification and analysis of V. cholerae CsrB, CsrC and CsrD

In the E. coli BarA/UvrY-CsrA system, the sRNAs CsrB and CsrC control the activity of CsrA (Liu et al., 1997; Weilbacher et al., 2003). Likewise, in Pseudomonas GacS/GacA control RsmA activity through the Rsm sRNAs (Aarons et al., 2000; Heeb et al., 2002; Valverde et al., 2003; Burrowes et al., 2005). In these cases, the sRNAs bind and titrate the activity of CsrA and RsmA. Thus, the sRNAs have a negative effect on their sRNA-binding proteins.

To identify putative sRNAs that could act in conjunction with CsrA in V. cholerae, we performed a BLAST search of the V. cholerae genomic contigs using the E. coli csrB sequence. This analysis identified one candidate csrB gene. We suspected multiple sRNAs act downstream of VarA, so we used the V. cholerae csrB sequence we had identified to perform a second BLAST search against the V. cholerae genome. Two additional sRNAs were revealed. We name the three genes csrB (located between VC0190/VC0191), csrC (located between VC0882/VC0883) and csrD (located between VCA0839/VCA0840). While the genome contexts of the E. coli csrB and csrC genes are conserved at least partially in closely related species, none of the genes flanking the csr sRNAs genes in V. cholerae are orthologues of the genes flanking csrB and csrC in E. coli. The three V. cholerae csr sRNA genes most likely arose from gene duplications because they are more homologous to each other than to the csr sRNA genes from other organisms.

An alignment of the V. cholerae csrB, csrC and csrD genes is shown in Fig. 4A and the predicted secondary structures of the corresponding sRNAs are shown in Fig. 4B. In E. coli, secondary structures for CsrB and CsrC show 18 and nine repeats, respectively, containing AGGA/AGGGA sequences located primarily in the loop regions of stem-loops or in other single-stranded regions which are the predicted sites of interaction with CsrA (Liu et al., 1997; Weilbacher et al., 2003). The V. cholerae Csr sRNA predictions are consistent with the proposed binding motif for CsrA (Liu et al., 1997). In V. cholerae, CsrB has 15 AGGA and three AGGGA motifs, CsrC has

Regulation of quorum sensing in Vibrio cholerae 1191

17 AGGA and two AGGGA motifs, and CsrD has 13 AGGA and two AGGGA motifs. These results suggest that CsrBCD of V. cholerae likely act in a manner similar to their homologues in E. coli, and control CsrA by binding multiple copies of the protein at the AGGA loops. To test if these components are equivalent to the analogous E. coli components, we examined their roles in glycogen production because CsrA is a repressor of glycogen production in E. coli. Introduction of V. cholerae csrA into an E. coli csrA mutant reduced glycogen production as judged by iodine staining of glycogen on M63-minimal glucose plates (Romeo et al., 1993). Consistent with this, overexpression of the V. cholerae csr sRNA genes in wild-type E. coli significantly increased glycogen production. These results show that the V. cholerae CsrA and CsrBCD components are equivalent to their counterparts in E. coli and justify the naming convention we have chosen.

To test whether CsrB, CsrC and CsrD regulate V. cholerae quorum sensing we measured bioluminescence in csrB, csrC and csrD single, double and triple mutants. Figure 4C shows that each single mutant and all the combinations of double sRNA mutants are wild type with respect to density-dependent bioluminescence. However, the triple csrBCD mutant is dark (closed black circles), and thus quorum sensing is eliminated in this mutant. Importantly, the dark csrBCD phenotype differs from the density-dependent bright Lux phenotype of the csrA mutant (Fig. 3A) suggesting that CsrBCD act oppositely to CsrA with respect to quorum sensing. An epistasis test demonstrated that the csrA, csrBCD quadruple mutant phenotype is identical to that of the single csrA mutant (not shown) showing that CsrBCD act upstream of CsrA (Fig. 1). As in other systems, the CsrBCD sRNAs most likely work by inhibiting the action of the CsrA binding protein.

To verify that CsrBCD function in the VarS/VarA pathway, we overexpressed each in the varA mutant and measured light production (Fig. 4D). Expression of any one of the sRNAs causes over a 1000-fold increase in light production. Thus, each sRNA can bypass the varA mutation suggesting that these sRNAs act downstream of VarA (Fig. 1). Because deletion of all three csr sRNA genes is required to obtain the varA phenotype, we suggest that CsrB, CsrC and CsrD play redundant roles in quorum-sensing regulation. Deletion of all three is sufficient to simulate the varA phenotype, so we conclude that no additional sRNAs function at this step.

VarS/VarA activate expression of csrB, csrC and csrD

In other bacteria, VarS/VarA homologues are proposed to activate expression of genes encoding the CsrB-type sRNAs via conserved DNA regions speculated to bind the

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **58**, 1186–1202
1192 D. H. Lenz et al.

A

| CsrB | GTCG3AAGGATACTGACACCGGAACAGGAAAGCACCAAGGATTGGTCACTTCAGGAAGAACAT CGAAACATTCAGGATGAATGCT...CGGGAGAAAAGGTAAATTGGACATTGAAACGGACGC |
| --- | --- |
| CsrC | .........GTCGAAGGAAATGACCCTAGGAACGGTATCTCAGGATGAGAAT.........GGTAGGGATGCCACAAGCGGAAAGCGCATAGGAATGCGTCAGCGC |
| CsrD | GTCA GCCAGGAAGCGGACACGGAAACAGGAAAGAQCQAAGGATTGGTCACTTCAGGACGAACATTTCGAT TGTTCAGGATGAATGT....CGGGAGAGAGCA...AAGGACGTTTAGCTGGA CGC |

| CsrB | AATCGTTGCTAGGATGGTAACGAAQAGGATGGGAAATGGAAACCTC...TGGA C.GAGGCAAGGACTGAACATCAGGATGATGGGAGGGGACACCGGCTCAGGGAACAAAGTGATGTGAACTCGCTAG |
| --- | --- |
| CsrC | .....ACAGTGAGG TGAC..CTGCAAAGGATGATTOA.GGATACTCTA.GGGATGTGAGAGGAAAC...CCATCAGGAGGATGGCAAGGACACCGGCTCAAGGAA CAAGTGATGAAGGCTGAATAG |
| CsrD | T.TAATAACTAGGATGCGTTATAA.ACGGAGAGTTAATGGACA ACTTAATGGAC.TAAGCGAGTTCTG......TCAGGAT................. |

| CsrB | GATTGTGAGTTGTATCGAAAGACACAGGACACCCGCTAGGA TGGCGATGCAA.GGATTGAGCTGACGGATGCAAGCTACTATC AAGGATAGATGCAAGGAGCACCTTTTAGTAGCTGGAAATGC |
| --- | --- |
| CsrC | GATTATTCG..GGATCAGGATAAATAGGACACCCGCTAGGATGGCGATGCTC.GGAGTQQCTGA AAGGAT.CAGG...CCGTTAAGGAT..GATGCATGGAGCACATTC AAGTAGCCCGGAATGC |
| CsrD | ............AGAGGAA G.....GGACACCGCTAGGAAAGGCGATGAAACGGATTGTGCTGAAGGAC.ACGGAGACTATCAAGGATAGATGCAAGGAGCACCTATTAGTGCGGGATTCGC |

| CsrB | TGC GAGTAAAGACTCTAAACCCC GGTGGGCTT.GCCCC..GGGGTTTTTTT |
| --- | --- |
| CsrC | TGC GAGTAAAGA.TTAAGCCCC.GACACCTCTGTG...GGGGCTTT....... |
| CsrD | TGC GAGT GAGAAACTGAAACCCC ACTAAGCTTAGTGGGGTTTTTCTTTTTT |

B

![Image of CsrB](image1.png)
![Image of CsrC](image2.png)
![Image of CsrD](image3.png)

C

![Graph with data points](graph1.png)

D

![Bar chart](bar_chart.png)

Strain

Fig. 4. Identification and activity of the CsrB, CsrC and CsrD sRNAs.
A. Alignment of the DNA sequences encoding the CsrB, CsrC and CsrD sRNAs. The sequences are shown from the predicted transcription start to termination sites.
B. Lowest-energy secondary-structural predictions for V. cholerae CsrB, CsrC and CsrD sRNAs. AGGA and AGGGA motifs are in bold.
C. Density-dependent lux assays for: MM227 (WT, open squares), MM349 (∆luxO, open diamonds), MM560 (∆varA, closed black diamonds), DL3232 (∆csrBCD, closed black circles), DL2479 (∆csrB, dark grey squares), DL3227 (∆csrC, dark grey diamonds), DL3228 (∆csrD, dark grey circles), DL3230 (∆csrBC, light grey squares), DL3229 (∆csrBD, light grey diamonds), DL3231 (∆csrCD, light grey circles).
D. Single time point lux assays following overexpression of csrB, csrC and csrD in a V. cholerae ∆varA mutant: DL3610 (∆varA/pKK177-3RI), DL3611 (∆varA/pKK177-3RI-csrB), DL3612 (∆varA/pKK177-3RI-csrC), DL3613 (∆varA/pKK177-3RI-csrD).

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **58**, 1186–1202

VarA-type proteins (Valverde *et al.*, 2003). Alignment of the regions upstream of the three sRNA genes in *V. cholerae* shows a conserved 18 bp sequence similar to the putative GacA upstream activating sequence identified in the promoter region of *rsmY* in *P. fluorescens* (Valverde *et al.*, 2003). In *V. cholerae*, the consensus binding site is TGTGCGAGATCTCTTACA. The binding site upstream of *V. cholerae csrB* is identical to the consensus binding site, whereas the sites for *csrC* and *csrD* contain two and one mismatches from the consensus respectively. Moreover, the above binding site is nearly identical (1 mismatch) to the sequence (TGTGAGAGAT CTCTTACA) upstream of *csrB* in *E. coli*.

To test whether VarS/VarA regulate *csrB*, *csrC* and *csrD* expression, we constructed transcriptional reporter fusions to each. Figure 5A shows that transcription of *csrB*, *csrC* and *csrD* is 200-fold higher in the wild type than in the *varS* mutant. Transcription of *csrC* is 200-fold higher, and transcription of *csrB* and *csrD* is 2000-fold higher in the wild type than in the *varA* mutant (Fig. 5A). Thus, VarS/VarA activate transcription of *csrB*, *csrC* and *csrD*, and we hypothesize this regulation is direct.

VarS and VarA regulate *csrC* identically but have different effects on *csrB* and *csrD*. The former result signifies that all VarA regulation of *csrC* is due to input from the sensor VarS. However, in the case of *csrB* and *csrD*, VarA has a 10-fold stronger regulatory effect than does VarS indicating that a portion of the VarA regulation of *csrB* and *csrD* is independent of VarS. This latter result suggests that VarA receives input in the form of phosphate from cellular component(s) in addition to VarS, and further, that the effect of this additional input is manifested in the regulation of *csrB* and *csrD*. One explanation is that the *csrB* and *csrD* promoters have a higher affinity for phospho-VarA than does the *csrC* promoter. This predicted hierarchy of promoter affinity (*csrB ~ csrD > csrC*) follows the conservation of the respective binding sites upstream of these genes (*csrB*: no mismatches, *csrD* one mismatch, *csrC* two mismatches). Thus, we propose that the phospho-VarA present in the absence of VarS acts at *csrB* and *csrD*, but because there is a limited supply, not at *csrC*.

To verify that the *csrBCD* genes indeed encode sRNAs, and to examine whether the differences in transcription we observed with the reporter fusions are reflected in the levels of the Csr sRNAs, the Csr sRNAs were measured at high-cell density via steady-state Northern blots (Fig. 5B). All three Csr RNAs are produced by the wild type and significantly less of each is produced by the *varS* and *varA* mutants. Consistent with the transcriptional reporter data, the same level of CsrC RNA is produced by the *varS* and *varA* mutants, whereas the *varA* mutant produces less CsrB and CsrD RNA than does the *varS* mutant.


Regulation of quorum sensing in Vibrio cholerae 1193

We wondered if the difference in the levels of VarS and VarA regulation of *csrB* and *csrD* accounted for the distinct quorum-sensing phenotypes of the *varS* and *varA* mutants (Fig. 2A). If so, we predict that, in a *varS* background, deletion of *csrB* and *csrD* would confer the *varA* phenotype. Only this specific deletion combination and none of the other single or combinations of *csrB*, *csrC*, or *csrD* double mutations in conjunction with *varS* should produce the phenotype. Figure 5C shows this is indeed the case. The *varS* single mutant is dim and the *csrBD* double mutant displays density-dependent *lux* expression (closed squares and closed triangles respectively). In contrast, the *varA* single mutant and the triple *varS*, *csrBD* mutant have dark phenotypes (closed diamonds and stars). Importantly, Fig. 5C also shows that deletion of any single *csr* sRNA gene, or double deletion of *csrBC* or *csrCD* in the *varS* background does not alter the phenotype. We propose that without VarS, the remaining VarA-mediated regulation of *csrB* and *csrD* leads to residual density-dependent *lux* expression. Taken together, the results in Fig. 5 suggest that the original quorum-sensing defect observed in the *varA* mutant is due to misregulation of *csrB*, *csrC* and *csrD*.

VarS/VarA-CsrA/BCD controls *qrr* expression and HapR production

The present work shows that the VarS/VarA-CsrA/BCD system operates through LuxO which controls expression of genes encoding the four Qrr sRNAs. The sRNAs control the stability of *hapR* mRNA (Lenz *et al.*, 2004). It follows therefore, that *qrr* gene expression and HapR protein production should be controlled by VarS/VarA-CsrA/BCD. To verify this, we measured levels of Qrr1, Qrr2, Qrr3, Qrr4 and HapR protein in selected mutants (Fig. 6A). For reference, we have included the wild type, *luxO* null strain and *luxO* D47E strain. In wild-type cells, at high-cell density, LuxO is dephosphorylated, the *qrr* genes are not transcribed and *hapR* mRNA is stabilized (Fig. 1). Thus, under this condition very little of the Qrr sRNAs can be detected while high levels of HapR protein are present (Fig. 6A; WT). The *luxO* null strain is locked in a state mimicking high-cell density (Fig. 2B), and the *luxO* null profile is similar to the wild type with respect to Qrr sRNAs and HapR protein (Fig. 6A; *luxO*). The opposite pattern is observed for the *luxO* D47E strain (Fig. 6A; *luxO* D47E). The LuxO D47E allele of LuxO locks LuxO into a form simulating the phosphorylated, low-cell density state. *V. cholerae* strains carrying LuxO D47E constitutively transcribe the Qrr sRNAs, and thus *hapR* mRNA is always destabilized (Lenz *et al.*, 2004). Figure 6A shows that this strain over-produces the Qrr sRNAs and no HapR protein is detected.

Fig. 5. VarS/VarA positively regulate csrB, csrC and csrD expression.

A. Single time point lux assay for csrB, csrC and csrD-lux transcriptional fusions in: WT, ΔvarS and ΔvarA strains: DL3273 (WT + csrB-lux), DL3274 (WT + csrC-lux), DL3275 (WT + csrD-lux), DL3279 (ΔvarS + csrB-lux), DL3280 (ΔvarS + csrC-lux), DL3281 (ΔvarS + csrD-lux), DL3276 (ΔvarA + csrB-lux), DL3277 (ΔvarA + csrC-lux), DL3278 (ΔvarA + csrD-lux).

B. Northern blots of RNA isolated from V. cholerae C6706str2 (WT), MM533 (ΔvarA) and MM597 (ΔvarS) probed for the sRNAs CsrB, CsrC and CsrD. Gels stained for ribosomal RNA are shown as loading controls.

C. Density-dependent lux assays for: MM609 (ΔvarS, closed squares), MM560 (ΔvarA, closed diamonds), DL3229 (ΔcsrBD, closed triangles), DL3704 (ΔvarS, ΔcsrB, open squares), DL3705 (ΔvarS, ΔcsrC, open diamonds), DL3706 (ΔvarS, ΔcsrD, open circles), DL3707 (ΔvarS, ΔcsrBC, open triangles), DL3406 (ΔvarS, ΔcsrBD, stars), DL3708 (ΔvarS, ΔcsrCD, plus symbols).

Regulation of quorum sensing in Vibrio cholerae 1195

Fig. 6. Qrr1–4 and HapR protein levels are regulated by VarS/VarA and CsrA/BCD.

A. RNA isolated from V. cholerae C6706str2 (WT), MM307 (∆luxO), BH38 (luxO D47E), MM533 (∆varA), MM597 (∆varS), DL3183 (∆csrBCD) and DL2413 (csrA::Tn5) was probed for the sRNAs Qrr1, Qrr2, Qrr3 and Qrr4. Gels stained for ribosomal RNA are shown as loading controls. The bottom panel shows a Western blot prepared for the same strains probed with antibody to HapR.

B. Single time point lux assay following IPTG induction of hapR in: DL3336 (WT/pJZ146), DL3338 (∆hapR/pJZ146), DL3340 (luxO D47E/pJZ146), DL3379 (∆varS/pJZ146), DL3342 (∆varA/pJZ146), DL3381 (∆csrBCD/pJZ146).

Figure 6A shows that, compared with the wild type, increased levels of Qrr1, Qrr2 and Qrr3 are present in varA and varS single and csrBCD triple mutants, consistent with mutations in these genes promoting the low-cell density state. HapR protein is also not detected in the varA and varS mutants. A minor amount of HapR is present in the csrBCD mutant suggesting that mutation of csrBCD does not completely mimic mutation of varA. In contrast, the csrA single mutant produces low (wild type) levels of Qrr1, Qrr2 and Qrr3 and a high level of HapR protein. This is expected given that CsrA acts inversely to and downstream of VarS, VarA and CsrBCD. Unexpectedly, Qrr4 levels appear minimally altered in all of the single mutants. qrr4 is the most highly expressed of the four qrr sRNA genes (Lenz et al., 2004). Most likely, LuxOP has the highest affinity for the qrr4 promoter and, at low-cell density, phospho-LuxO more fully occupies the qrr4 promoter than those of qrr1–3. Thus, alterations in levels of phospho-LuxO, either up or down, would have more dramatic effects at the latter three promoters than at qrr4. VarS/VarA-CsrA/BCD input into LuxO is transduced into effects on qrr expression and HapR production. We therefore place the entire VarS/VarA-CsrA/BCD system upstream of LuxO in the model in Fig. 1. Consistent with

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, Molecular Microbiology, **58**, 1186–1202

this arrangement, Fig. 6B shows that overexpression of hapR in the hapR, luxO D47E, varS, varA and csrBCD mutants restores light production, demonstrating that HapR is the most downstream component in the circuit.

VarS/VarA-CsrA/BCD functions through LuxO

When VarS/VarA is functioning, CsrBCD are expressed, and inhibit CsrA. Under this condition, the Lux quorum-sensing system functions normally. When VarS/VarA is inactivated, CsrBCD are not made, and CsrA is active. In this mode, the cells are dark, implying that there is increased active LuxO. Two ways CsrA could accomplish this are by increasing LuxO levels or increasing LuxO activity. In both cases, the effect would be to inhibit the transition of the cells from the low to the high-cell density state.

In an initial attempt to distinguish between whether CsrA influences LuxO levels or phospho-LuxO activity, we examined the lux phenotype of the luxO D47E, csrA::Tn5 double mutant (Fig. 7A). The luxO D47E mutation, as mentioned, causes a dark Lux phenotype. If active CsrA increases the level of LuxO, then inactivation of csrA in the luxO D47E background should result in an increase in bioluminescence because reduced LuxO would be present. Alternatively, if active CsrA increases phospho-LuxO activity, in the absence of CsrA, no change in luminescence output should occur in the luxO D47E mutant. The figure shows that the luxO D47E, csrA::Tn5 double mutant has a dark phenotype indistinguishable from the luxO D47E single mutant. While we do not know the molecular mechanism linking active CsrA to LuxO activity, this result provides preliminary evidence supporting a model in which the VarS/VarA-CsrA/BCD system influences the activity of LuxO but not the absolute levels of the LuxO protein in the cell. Reporter fusions to luxO and real-time polymerase chain reaction (PCR) were used to determine the levels of luxO mRNA in the wild type varA, varS and csrBCD mutants. No significant differences in luxO transcript levels were observed supporting the above model (data not shown). In addition to functioning upstream of LuxO, CsrA could also control hapR expres-

Fig. 7. CsrA affects the activity of LuxO.
A. Density-dependent lux assays for: MM227 (WT, open squares), BH48 (luxO D47E, open circles), DL2493 (csrA::Tn5, open diamonds) and DL3265 (luxO D47E, csrA::Tn5, closed diamonds).
B. Western blot of HapR in whole cell lysates prepared from high-cell density cultures of: MM307 (∆luxO), DL2953 (∆qrr1, 2, 3, 4), MM533 (∆varA), DL2413 (csrA::Tn5), MM851 (∆luxO, ∆varA), DL2417 (∆luxO, csrA::Tn5), DL3743 (∆qrr1, 2, 3, 4, ∆varA), DL3747 (∆qrr1, 2, 3, 4, csrA::Tn5).

A

![Graph](https://i.imgur.com/placeholder.png)

B

![Western Blot](https://i.imgur.com/placeholder.png)

these genes that occurred at low-cell density. To verify that VarS/VarA-CsrA/BCD control of *vpsL* and *vpsT* expression is mediated through LuxO and the downstream components of the quorum-sensing cascade, we performed an epistasis experiment by combining the *luxO* D47E and *csrA::Tn5* mutations and the *luxO* and *varA* mutations. In both cases, the *luxO* mutations were epistatic, showing that indeed VarS/VarA-CsrA/BCD control of exopolysaccharide production functions through the quorum-sensing circuit (data not shown). The *csrA::Tn5* mutation strongly reduces *vpsL* and *vpsT* expression even though this mutation has no discernible effect on *lux* regulation. Thus, CsrA could have additional effects on *vpsL* and *vpsT* expression that are independent of quorum sensing. Finally, the *hapA* gene, encoding the H/A protease, is activated by quorum sensing at high-cell density (Jobling and Holmes, 1997). Mutations in the VarS/VarA-CsrA/BCD pathway affected *hapA* expression, and the effect of each mutation was opposite to that on *vpsT* or *vpsL* (not shown). Taken together, it is likely that VarS/VarA-CsrA/BCD control, through the circuit shown in Fig. 1, extends to all quorum-sensing-regulated genes in *V. cholerae*.

### Discussion

The VarS/VarA phosphorelay system, originally identified for its role in *V. cholerae* pathogenesis (Wong *et al.*, 1998), acts in parallel to the CAI-1-CqsS and AI-2-LuxPQ quorum-sensing systems. VarS/VarA activate the expression of genes encoding three sRNAs, CsrB, CsrC and CsrD which, in turn, inhibit the activity of the sRNA-binding protein CsrA. Together the VarS/VarA-CsrA/BCD regulatory components relay information, in a LuxO-dependent manner, to control the expression of the genes encoding the four Qrr sRNAs. The Qrr sRNAs, with the sRNA chaperone Hfq, control the stability of the mRNA encoding the master regulator of the quorum-sensing cascade, HapR (Fig. 1).

Previously we suggested that an additional sensory system transduces information into the *V. cholerae* Lux circuit in a manner that bypasses LuxU but requires LuxO. This supposition was based on findings that *V. cholerae* strains lacking the two known autoinducer detectors (CqsS and LuxPQ) or lacking LuxU retain density-dependent gene expression while *V. cholerae luxO* null mutants are incapable of density-dependent gene expression (Miller *et al.*, 2002). The VarS/VarA-CsrA/BCD could be part of the unidentified sensory apparatus. Our evidence shows that VarS/VarA-CsrA/BCD system acts independently of the known autoinducer sensors (Fig. 2D), and signalling through VarS/VarA-CsrA/BCD requires LuxO (Fig. 2B).

CsrA is an important post-transcriptional regulator in many bacteria. A model has been proposed for *E. coli* in

Fig. 8. *vpsT* and *vpsL* are regulated by the VarSA-CsrA/BCD System.

Single time point *lux* assay for *vpsR-lux* transcriptional fusions in: A. DL3572 (WT), DL3573 (∆*luxO*), DL3574 (*luxO* D47E), DL3582 (∆*hapR*), DL3575 (∆*varA*), DL3576 (∆*varS*), DL3577 (∆*csrBCD*), DL3578 (*csrA::Tn5*).

B. *vpsT-lux* transcriptional fusions in: DL3479 (WT), DL3480 (∆*luxO*), DL3481 (*luxO* D47E), DL3482 (∆*hapR*), DL3483 (∆*varA*), DL3484 (∆*varS*), DL3485 (∆*csrBCD*), DL3498 (*csrA::Tn5*).

C. *vpsL-lux* transcriptional fusions in: DL3438 (WT), DL3543 (∆*luxO*), BH931 (*luxO* D47E), BH633 (∆*hapR*), DL3440 (∆*varA*), DL3442 (∆*varS*), DL3443 (∆*csrBCD*), DL3491 (*csrA::Tn5*).

which CsrA promotes exponential phase and prevents stationary phase gene expression (Romeo, 1998). CsrA functions by binding to specific sequences in the 5′ untranslated regions of target mRNAs and altering their stability and/or translation. Inhibition of CsrA requires the VarS/VarA homologues BarA/UvrY to activate the expression of the genes encoding the sRNAs CsrB and CsrC

(Suzuki et al., 2002; Weilbacher et al., 2003). These two sRNAs have numerous AGGA-containing loops to which CsrA binds, which titrates CsrA away from target mRNAs. We suggest a role for CsrA in controlling *V*. *cholerae* quorum sensing, and our results show that CsrA acts upstream of LuxO. There is no CsrA consensus binding site upstream of the *luxO* gene and mutation of *csrA* did
not alter the levels of $luxO$ transcript, suggesting that CsrA does not act directly on $luxO$ expression. Rather, we predict that an additional component (denoted X in Fig. 1) exists and couples signalling from VarS/VarA-CsrA/BCD to LuxO. Because we do not know the molecular mechanism underlying this event, we have denoted this interaction with a dotted line. Under conditions where VarS/VarA do not induce expression of $csrBCD$, CsrA is active and promotes enhanced LuxO functioning. This could be accomplished through CsrA-mediated regulation of some factor, X, whose function is to increase the concentration or activity of phospho-LuxO. The component X could be a kinase, a small molecule phospho-donor, or a regulatory component that controls the expression or activity of a phospho-donor. An obvious candidate for the phospho-donor is acetyl-phosphate. However, inactivation of the genes responsible for acetyl-phosphate production in *V. cholerae* (*pta*, *ack1* and *ack2*) in the *varA* background did not have any effect on light production (data not shown) suggesting that if some small molecule phospho-donor is responsible, it is not acetyl-phosphate. Although we have preliminary evidence indicating that VarS/VarA-CsrA/BCD primarily functions through LuxO to control *hapR* (Fig. 7B), we have not fully eliminated the possibility that, in addition, VarS/VarA-CsrA/BCD inputs information into the quorum-sensing cascade at other points.

Bioinformatics, overexpression and deletion analysis revealed that the link between VarS/VarA and CsrA in *V. cholerae* is three sRNAs we name CsrB, CsrC and CsrD. Finding three CsrB-type sRNAs is surprising because this level of redundancy has not been observed in other bacterial systems in which CsrA regulation has been investigated. We suspect that VarA binds upstream of each of the *csr* sRNA genes based on transcriptional fusions showing that VarS/VarA activate their expression 200- to 2000-fold combined with an analysis of their upstream DNA sequences showing that a site very similar to the putative upstream activating sequence in *Pseudomonas* spp. (Valverde et al., 2003) exists upstream of each. We believe that our analysis enabled identification of the entire suite of sRNAs involved in controlling CsrA activity in *V. cholerae* because the triple *csrBCD* mutant has a phenotype indistinguishable from the *varA* mutant (Fig. 4C). We also know that *csrBCD* act redundantly because the triple *csrBCD* mutant affects HapR protein production (Fig. 6A), whereas the single mutants and all combinations of double mutants do not (data not shown). While redundancy in sRNA function has been observed in other CsrA-type systems, unlike in *V. cholerae*, none of the previously described systems is known to have greater than two sRNAs that act on CsrA. However, in some cases, the complete set of sRNAs have probably not yet been identified, preventing the absolute definition of their regulatory roles.

---

Regulation of quorum sensing in Vibrio cholerae 1199

Although the VarS/A-CsrA/BCD system operates in parallel to the two *V. cholerae* quorum-sensing systems, we do not know if VarS responds to an extracellular molecule. While several reports connect VarS/VarA-CsrA-sRNA homologues to quorum sensing (Reimmann et al., 1997; Whitehead et al., 2002), no signal has been identified that initiates the phosphorelay. Recently, organic extracts of conditioned medium prepared from stationary phase cultures of *Pseudomonas fluorescens* were shown to contain an activity capable of modest activation of GacS/GacA-regulated targets (Valverde et al., 2003; Zuber et al., 2003). This finding suggests that GacS could respond to an extracellular molecule that is produced or accumulates in stationary phase. If so, and if this holds true for VarS, it would be especially exciting because the remainder of the quorum-sensing regulators in vibrios and in pseudomonads share no similarity. We found that *csrB*, *csrC* and *csrD* transcription is induced approximately 50-fold from low- to high-cell density. Thus, CsrB, CsrC and CsrD levels are growth phase regulated. However, we were unable to induce their expression at low-cell density by the addition of cell-free culture fluids collected from wild-type *V. cholerae* grown to high-cell density.

A final, but important point regarding the complexity of the circuit shown in Fig. 1 comes from examination of the phenotypes of some of the mutants shown in Fig. 2C and D. We observe that the phenotype of a *luxU* mutant is not identical to that of the double *cqsS*, *luxQ* mutant. We find that *luxU* mutants are consistently 100-times brighter than *cqsS*, *luxQ* double mutants. This result implies that an additional sensory input(s) is funnelled into LuxU. This input cannot be attributed to VarS/A-CsrA/BCD because that system feeds information into the circuit at the level of LuxO. We therefore conclude that, Al-1-CqsS is System 1, Al-2-LuxPQ is System 2, VarS/A-CsrA/BCD is part of System 3, and there must also exist a System 4 that relays information through LuxU, and thus must influence the activity of LuxO, the levels of the Qrrs and HapR, and, in turn, the expression of all the genes that lie downstream of HapR in the hierarchy.

The molecular architecture controlling *V. cholerae* quorum sensing requires at least three signalling channels, and at a minimum, seven small regulatory RNAs (four Qrr and three Csr sRNAs). The circuit is highly redundant in terms of the extracellular signals and the signal transduction machinery. This degree of overlap probably exists because high fidelity information flow is required, and the mechanism *V. cholerae* has evolved for ensuring accuracy relies on extreme functional redundancy. If the signal detected by VarS/VarA accumulates in stationary phase, this could be the link connecting growth phase to community number and composition. Funneling all the information into one integrated circuit could provide

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **58**, 1186–1202
1200 D. H. Lenz et al.

V. cholerae a combinatorial mechanism for broadly probing the environment for fluctuations and responding accurately.

### Experimental procedures

#### Bacterial strains and media

*Vibrio cholerae* strains are derived from the El Tor strain C6706str2 (Thelin and Taylor, 1996). Details are provided in figure legends. *V. cholerae* strains were grown with aeration at 30°C in SOC broth (Sambrook *et al.*, 1989) for *lux* assays and in Luria–Bertani (LB) broth for other experiments. Plasmids were maintained at 37°C in LB in *E. coli* JM109 (Yanisch-Perron *et al.*, 1985) or *E. coli* S17-1λ*pir* (de Lorenzo and Timmis, 1994). Antibiotics concentrations used are: ampicillin 100 μg ml⁻¹; tetracycline 10 μg ml⁻¹; kanamycin 100 μg ml⁻¹; chloramphenicol 10 μg ml⁻¹; streptomycin 1–5 mg ml⁻¹; and polymixin B 50 units ml⁻¹.

#### DNA manipulations

DNA manipulations were performed according to Sambrook *et al.* (1989). Deletions were constructed according to Skorupski and Taylor (1996). *csrA::Tn5* strains were constructed by PCR amplification of the *csrA::Tn5* region, cloning into pKAS32 vector (Skorupski and Taylor, 1996) and gene replacement onto the chromosome with kanamycin selection. PCR reactions for cloning used PFU turbo polymerase (Stratagene); other PCR reactions used *Taq* polymerase (Roche). dNTPs, restriction endonucleases and T4 ligase were purchased from New England Biolabs. DNA purification was performed by QIAGEN methods. Primer sequences are available by request. The *csrB*, *csrC* and *csrD* overexpression plasmids were constructed in pKK177-3RI as in Lenz *et al.* (2004), and were introduced into *V. cholerae* by electroporation. The *lux* transcriptional fusion plasmids were constructed as in Lenz *et al.* (2004) and introduced into *V. cholerae* by conjugation. The *hapR* overexpression construct pJZ146 came from Zhu *et al.* (2002) and was induced with 0.5 mM IPTG.

#### Bioinformatics

The locations of *csrBCD* in the *V. cholerae* genome were identified using BLAST at The Institute for Genomic Research (TIGR, http://www.tigr.org). Multiple alignments were carried out with TCOFFEE (Poirot *et al.*, 2003). MFOLD was used for secondary structure predictions (Jacobson and Zuker, 1993; Zuker, 2003). Default parameters were used for the folding predictions, and the structure with the lowest free energy is shown. To avoid spurious loop formation in the secondary structure, we set a cut-off of 40 for the maximum distance between paired bases.

#### Bioluminescence assays

*Vibrio cholerae* bioluminescence assays were performed as in Lenz *et al.* (2004). pBB1, containing *V. harveyi luxCDABE*,

was introduced into *V. cholerae* strains by conjugation. Single time point bioluminescence assays were performed at high-cell density in triplicate.

#### Western blot analysis

Western blots to monitor HapR protein were performed as in Lenz *et al.* (2004). Strains were grown to high-cell density.

#### Northern blot analysis

Steady-state Northern blots were as reported (Lenz *et al.*, 2004) except that cultures were grown to OD~600~ of 2 prior to RNA preparation and twice as much RNA was loaded. Gels containing ribosomal RNA stained with ethidium bromide prior to transfer are shown as loading controls.

#### Genetic screen to identify *varS/varA* and *csrA*

The Tn5 mutagenesis and analysis yielding insertions in *varS* and *varA* has been described (Miller *et al.*, 2002). Bypass suppressors were obtained by introduction of the pRL27c Tn5 suicide vector (Larsen *et al.*, 2002) into the *varA* in-frame deletion strain MM533. Kanamycin resistant colonies were pooled, and pBB1 conjugated into them. Exconjugants were screened for a bright phenotype. The Tn5-chromosome junctions were characterized as described (Miller *et al.*, 2002).

### Acknowledgements

This work was supported by HHMI and NSF Grant MCB-0343821, NIH Grant 5RO1 G065859, ONR Grant N00014-03-0183 and an HHMI Predoctoral Fellowship (D.H.L.), The Jeffress Foundation (R.V.K.). We thank the Bassler Lab, Dr T. Silhavy and Dr N. Wingreen for insightful discussions.

### References

Aaron, S., Abbas, A., Adams, C., Fenton, A., and O’Gara, F. (2000) A regulatory RNA (PrrB RNA) modulates expression of secondary metabolite genes in *Pseudomonas fluorescens* F113. *J. Bacteriol.* **182**: 3913–3919.

Bassler, B.L., Wright, M., Showalter, R.E., and Silverman, M.R. (1993) Intercellular signalling in *Vibrio harveyi*: sequence and function of genes regulating expression of luminescence. *Mol. Microbiol.* **9**: 773–786.

Bassler, B.L., Wright, M., and Silverman, M.R. (1994) Multiple signalling systems controlling expression of luminescence in *Vibrio harveyi*: sequence and function of genes encoding a second sensory pathway. *Mol. Microbiol.* **13**: 273–286.

Burrowes, E., Abbas, A., O’Neill, A., Adams, C., and O’Gara, F. (2005) Characterisation of the regulatory RNA RsmB from *Pseudomonas aeruginosa* PAO1. *Res. Microbiol.* **156**: 7–16.

Casper-Lindley, C., and Yildiz, F.H. (2004) VpsT is a transcriptional regulator required for expression of *vps* biosynthesis genes and the development of rugose colonial

morphology in *Vibrio cholerae* O1 El Tor. *J Bacteriol* 186: 1574–1578.

Chen, X., Schauder, S., Potier, N., Van Dorsselaer, A., Pelczer, I., Bassler, B.L., and Hughson, F.M. (2002) Structural identification of a bacterial quorum-sensing signal containing boron. *Nature* 415: 545–549.

Freeman, J.A., and Bassler, B.L. (1999) A genetic analysis of the function of LuxO, a two-component response regulator involved in quorum sensing in *Vibrio harveyi*. *Mol Microbiol* 31: 665–677.

Freeman, J.A., and Bassler, B.L. (2000) Sequence and function of LuxU: a two-component phosphorelay protein that regulates quorum sensing in *Vibrio harveyi*. *J Bacteriol* 181: 899–906.

Freeman, J.A., Lilley, B.N., and Bassler, B.L. (2000) A genetic analysis of the functions of LuxN: a two-component hybrid sensor kinase that regulates quorum sensing in *Vibrio harveyi*. *Mol Microbiol* 35: 139–149.

Hammer, B.K., and Bassler, B.L. (2003) Quorum sensing controls biofilm formation in *Vibrio cholerae*. *Mol Microbiol* 50: 101–104.

Heeb, S., and Haas, D. (2001) Regulatory roles of the GacS/GacA two-component system in plant-associated and other gram-negative bacteria. *Mol Plant Microbe Interact* 14: 1351–1363.

Heeb, S., Blumer, C., and Haas, D. (2002) Regulatory RNA as mediator in GacA/RsmA-dependent global control of exoprotein formation in *Pseudomonas fluorescens* CHA0. *J Bacteriol* 184: 1046–1056.

Hrabak, E.M., and Willis, D.K. (1992) The *lemA* gene required for pathogenicity of *Pseudomonas syringae* pv. syringae on bean is a member of a family of two-component regulators. *J Bacteriol* 174: 3011–3020.

Jacobson, A.B., and Zuker, M. (1993) Structural analysis by energy dot plot of a large mRNA. *J Mol Biol* 233: 261–269.

Jobling, M.G., and Holmes, R.K. (1997) Characterization of *hapR*, a positive regulator of the *Vibrio cholerae* HA/protease gene *hap*, and its identification as a functional homologue of the *Vibrio harveyi luxR* gene. *Mol Microbiol* 26: 1023–1034.

Larsen, R.A., Wilson, M.M., Guss, A.M., and Metcalf, W.W. (2002) Genetic analysis of pigment biosynthesis in *Xanthobacter autotrophicus* Py2 using a new, highly efficient transposon mutagenesis system that is functional in a wide variety of bacteria. *Arch Microbiol* 178: 193–201.

Laville, J., Voisard, C., Keel, C., Maurhofer, M., Defago, G., and Haas, D. (1992) Global control in *Pseudomonas fluorescens* mediating antibiotic synthesis and suppression of black root rot of tobacco. *Proc Natl Acad Sci USA* 89: 1562–1566.

Lenz, D.H., Mok, K.C., Lilley, B.N., Kulkarni, R.V., Wingreen, N.S., and Bassler, B.L. (2004) The small RNA chaperone Hfq and multiple small RNAs control quorum sensing in *Vibrio harveyi* and *Vibrio cholerae*. *Cell* 118: 69–82.

Lilley, B.N., and Bassler, B.L. (2000) Regulation of quorum sensing in *Vibrio harveyi* by LuxO and sigma-54. *Mol Microbiol* 36: 940–954.

Liu, M.Y., Gui, G., Wei, B., Preston, J.F., 3rd, Oakford, L., Yuksel, U., *et al.* (1997) The RNA molecule CsrB binds to the global regulatory protein CsrA and antagonizes its activity in *Escherichia coli*. *J Biol Chem* 272: 17502–17510.


Regulation of quorum sensing in Vibrio cholerae 1201

de Lorenzo, V., and Timmis, K.N. (1994) Analysis and construction of stable phenotypes in Gram-negative bacteria with Tn5- and Tn10-derived minitransposons. *Methods Enzymol* 235: 386–405.

Miller, M.B., and Bassler, B.L. (2001) Quorum sensing in bacteria. *Annu Rev Microbiol* 55: 165–199.

Miller, M.B., Skorupski, K., Lenz, D.H., Taylor, R.K., and Bassler, B.L. (2002) Parallel quorum sensing systems converge to regulate virulence in *Vibrio cholerae*. *Cell* 110: 303–314.

Molofsky, A.B., and Swanson, M.S. (2004) Differentiate to thrive: lessons from the *Legionella pneumophila* life cycle. *Mol Microbiol* 53: 29–40.

Nealson, K.H., and Hastings, J.W. (1979) Bacterial bioluminescence: its control and ecological significance. *Microbiol Rev* 43: 496–518.

Pernestig, A.K., Melefors, O., and Georgellis, D. (2001) Identification of UvrY as the cognate response regulator for the BarA sensor kinase in *Escherichia coli*. *J Biol Chem* 276: 225–231.

Poirot, O., O’Toole, E., and Notredame, C. (2003) Tcoffee@igs: a web server for computing, evaluating and combining multiple sequence alignments. *Nucleic Acids Res* 31: 3503–3506.

Reimmann, C., Beyeler, M., Latifi, A., Winteler, H., Foglino, M., Lazdunski, A., and Haas, D. (1997) The global activator GacA of *Pseudomonas aeruginosa* PAO positively controls the production of the autoinducer N-butyryl-homoserine lactone and the formation of the virulence factors pyocyanin, cyanide, and lipase. *Mol Microbiol* 24: 309–319.

Rich, J.J., Kinscherf, T.G., Kitten, T., and Willis, D.K. (1994) Genetic evidence that the *gacA* gene encodes the cognate response regulator for the *lemA* sensor in *Pseudomonas syringae*. *J Bacteriol* 176: 7468–7475.

Romeo, T. (1998) Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB. *Mol Microbiol* 29: 1321–1330.

Romeo, T., Gong, M., Liu, M.Y., and Brun-Zinkernagel, A.M. (1993) Identification and molecular characterization of *csrA*, a pleiotropic gene from *Escherichia coli* that affects glycogen biosynthesis, gluconeogenesis, cell size, and surface properties. *J Bacteriol* 175: 4744–4755.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular Cloning: a Laboratory Manual*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Skorupski, K., and Taylor, R.K. (1996) Positive selection vectors for allelic exchange. *Gene* 169: 47–52.

Suzuki, K., Wang, X., Weilbacher, T., Pernestig, A.K., Melefors, O., Georgellis, D., *et al.* (2002) Regulatory circuitry of the CsrA/CsrB and BarA/UvrY systems of *Escherichia coli*. *J Bacteriol* 184: 5130–5140.

Thelin, K.H., and Taylor, R.K. (1996) Toxin-coregulated pilus, but not mannose-sensitive hemagglutinin, is required for colonization by *Vibrio cholerae* O1 El Tor biotype and O139 strains. *Infect Immun* 64: 2853–2856.

Valverde, C., Heeb, S., Keel, C., and Haas, D. (2003) RsmY, a small regulatory RNA, is required in concert with RsmZ for GacA-dependent expression of biocontrol traits in *Pseudomonas fluorescens* CHA0. *Mol Microbiol* 50: 1361–1379.

Weilbacher, T., Suzuki, K., Dubey, A.K., Wang, X., Gudapaty, S., Morozov, I., *et al.* (2003) A novel sRNA component of the carbon storage regulatory system of *Escherichia coli*. *Mol Microbiol* 48: 657–670.

Whitehead, N.A., Byers, J.T., Commander, P., Corbett, M.J., Coulthurst, S.J., Everson, L., *et al.* (2002) The regulation of virulence in phytopathogenic *Erwinia* species: quorum sensing, antibiotics and ecological considerations. *Antonie Van Leeuwenhoek* 81: 223–231.

Wong, S.M., Carroll, P.A., Rahme, L.G., Ausubel, F.M., and Calderwood, S.B. (1998) Modulation of expression of the ToxR regulon in *Vibrio cholerae* by a member of the two-component family of response regulators. *Infect Immun* 66: 5854–5861.

Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. *Gene* 33: 103–119.

Yildiz, F.H., and Schoolnik, G.K. (1999) *Vibrio cholerae* O1 El Tor: identification of a gene cluster required for the rugose colony type, exopolysaccharide production, chlorine resistance, and biofilm formation. *Proc Natl Acad Sci USA* 96: 4028–4033.

Yildiz, F.H., Dolganov, N.A., and Schoolnik, G.K. (2001)

VpsR, a Member of the Response Regulators of the Two-Component Regulatory Systems, Is Required for Expression of vps Biosynthesis Genes and EPS (ETr)-Associated Phenotypes in *Vibrio cholerae* O1 El Tor. *J Bacteriol* 183: 1716–1726.

Yildiz, F.H., Liu, X.S., Heydorn, A., and Schoolnik, G.K. (2004) Molecular analysis of rugosity in a *Vibrio cholerae* O1 El Tor phase variant. *Mol Microbiol* 53: 497–515.

Zhu, J., and Mekalanos, J.J. (2003) Quorum-sensing-dependent biofilms enhance colonization in *Vibrio cholerae*. *Dev Cell* 5: 647–656.

Zhu, J., Miller, M.B., Vance, R.E., Dziejman, M., Bassler, B.L., and Mekalanos, J.J. (2002) Quorum-sensing regulators control virulence gene expression in *Vibrio cholerae*. *Proc Natl Acad Sci USA* 99: 3129–3134.

Zuber, S., Carruthers, F., Keel, C., Mattart, A., Blumer, C., Pessi, G., *et al.* (2003) GacS sensor domains pertinent to the regulation of exoproduct formation and to the biocontrol potential of *Pseudomonas fluorescens* CHA0. *Mol Plant Microbe Interact* 16: 634–644.

Zuker, M. (2003) Mfold web server for nucleic acid folding and hybridization prediction. *Nucleic Acids Res* 31: 3406–3415.
