
Journal of Bacteriology

An α/β-Fold C—C Bond Hydrolase Is Involved in a Central Step of Nicotine Catabolism by *Arthrobacter nicotinovorans*

Paula Sachelaru, Emile Schiltz, Gabor L. Igloi and Roderich Brandsch

*J. Bacteriol.* 2005, 187(24):8516. DOI: 10.1128/JB.187.24.8516-8519.2005.

---

Updated information and services can be found at:  
<http://jb.asm.org/content/187/24/8516>

These include:

**REFERENCES**  
This article cites 20 articles, 7 of which can be accessed free at:  
<http://jb.asm.org/content/187/24/8516#ref-list-1>

**CONTENT ALERTS**  
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more](https://jb.asm.org/content/187/24/8516)

---

Information about commercial reprint orders: <http://journals.asm.org/site/misc/reprints.xhtml>  
To subscribe to to another ASM Journal go to: <http://journals.asm.org/site/subscriptions/>

---

Journals.ASM.org

JOURNAL OF BACTERIOLOGY, Dec. 2005, p. 8516–8519
0021-9193/05/$08.00+0 doi:10.1128/JB.187.24.8516–8519.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

An α/β-Fold C—C Bond Hydrolase Is Involved in a Central Step
of Nicotine Catabolism by *Arthrobacter nicotinovorans*

Paula Sachelaru,$^{1}$ Emile Schiltz,$^{2}$ Gabor L. Igloi,$^{3}$ and Roderich Brandsch$^{1*}$
*Institut für Biochemie und Molekularbiologie,*$^{1}$ *Institut für Organische Chemie und Biochemie,*$^{2}$
and *Institut für Biologie III,*$^{3}$ *Universität Freiburg, Freiburg, Germany*

Received 15 September 2005/Accepted 22 September 2005

The enzyme catalyzing the hydrolytic cleavage of 2,6-dihydroxypseudooxynicotine to 2,6-dihydroxypyridine and γ-N-methylaminobutyrate was found to be encoded on pAO1 of *Arthrobacter nicotinovorans*. The new enzyme answers an old question about nicotine catabolism and may be the first C—C bond hydrolase that is involved in the biodegradation of a heterocyclic compound.

L-Nicotine, the main alkaloid produced by the tobacco plant, is used by the gram-positive soil bacterium *Arthrobacter nicotinovorans* (formerly *Arthrobacter oxidans*) as a growth substrate. Intermediates formed during nicotine degradation were identified almost 50 years ago by Rittenberg and coworkers (9, 18) and Decker and coworkers (5, 10) by using resting cells and crude extracts of *A. nicotinovorans* grown in the presence of nicotine. This work established that the hydroxylation of the pyridine ring with formation of 6-hydroxy-L-nicotine (6HLNO) is the first step in the degradation of nicotine that follows the pathway depicted in Fig. 1. The nicotine-hydroxylating enzymes, known as nicotine dehydrogenase (NDH) and ketone dehydrogenase (KDH) (8, 20), belong to the family of molybdenum cofactor, Fe-S cluster, and flavin adenine dinucleotide (FAD)-dependent hydroxylases (11). 6-Hydroxynicotine oxidases—one specific for 6HLNO and one specific for 6-hydroxy-D-nicotine, a minor side product of nicotine metabolism—are FAD-containing enzymes (5). The major unelucidated aspect of nicotine catabolism was the enzymatic turnover of 2,6-dihydroxypseudooxynicotine (DHPON) (Fig. 1). It spontaneously rearranges into 2,6-dihydroxymethylmyosmine (DHMM) (Fig. 1), a metabolically inactive side product formed in vitro (9, 10, 18). The identification of 2,6-dihydroxypyridine (DHP) and γ-N-methylaminobutyrate indicated, however, that DHPON was the physiological intermediate in nicotine degradation. The subsequent enzymatic steps involved in the turnover of DHP into 2,3,6-trihydroxypyridine by an FAD-dependent hydroxylase (2, 13) and the novel flavoenzyme γ-N-methylaminobutyrate oxidase (MABO) responsible for the oxidative demethylation of γ-N-methylaminobutyrate (4) have been described.

Genes known or presumed to be involved in nicotine catabolism are clustered on the megaplasmid pAO1 of *A. nicotinovorans* (2, 15). They are expressed in the presence of nicotine in the growth medium and form a nicotine regulon. Here we show that the pAO1 genes carrying open reading frame 367 (ORF367) and the adjacent ORF116 belong to this nicotine regulon and that the protein encoded by ORF367, predicted to be related to peptidases of the α/β-fold hydrolase family (14), is responsible for the cleavage of DHPON. The hydrolase catalyzing DHPON cleavage is, to our knowledge, the first enzyme of this family involved in the degradation of a heteroaromatic compound.

Nicotine-dependent transcription of pAO1 ORFs addressed in this study. As part of the effort to identify and characterize the as yet unknown enzymes involved in nicotine degradation, we analyzed in this work the product of the gene carrying ORF367 (15). This ORF is situated on pAO1 adjacent to the gene of the large subunit of KDH (*kdhl*) and is transcribed divergently to it (Fig. 2A). Its 3′ end overlaps with ORF116, an indication that these ORFs may be translationally coupled. Internal DNA fragments of *kdhl*, ORF367, and ORF116 were PCR amplified as previously described (19), which resulted in 361-bp (I), 1,024-bp (II), and 301 bp (III) products, as shown in Fig. 2A. As in the case of *kdhl* (see legend to Fig. 2B, I, lanes 1 to 3), a gene known to belong to the nicotine regulon of pAO1 (20), transcripts of the genes carrying ORF367 and ORF116 were detected only in bacteria grown in the presence of nicotine in the growth medium (see legend to Fig. 2B, II and III, lanes 4 to 10). No transcripts were detected in bacteria grown in the absence of nicotine (see legend to Fig. 2B, I, II, and III, lanes 13 to 21). Thus the ORF367 and ORF116 genes are part of the nicotine regulon of pAO1 and are cotranscribed into one RNA transcript (see legend to Fig. 2B, II + III, lanes 11 to 12). The function of ORF116 protein was not addressed in this study.

Cloning of the ORF367 gene and purification of the protein. The DNA of ORF367 was cloned into pH6EX3 (3) as an EcoRI-XhoI endonuclease restriction fragment and expressed in *Escherichia coli* XL-1 Blue. It generated a fusion protein with the N-terminal extension MSPIH_LVPRGSEASNSGM (one-letter amino acid code), including a six-histidine tag and a thrombin cleavage site (underlined). The translational start methionine of ORF367 is written in boldface. The protein was purified with Ni-chelating Sepharose, and the recombinant protein migrated on sodium dodecyl sulfate-polyacrylamide gel electrophoresis as expected for its calculated size of 43,555 Da. Gel permeation chromatography indicated a molecular mass of 43,000 Da, which suggested that the protein is a monomer in solution (not shown).

* Corresponding author. Mailing address: Institut für Biochemie und Molekularbiologie, Hermann-Herder-Str. 7, 79104 Freiburg, Germany. Phone: 49-761-203-5230. Fax: 49-761-203-5253. E-mail: roderich.brandsch@biochemie.uni-freiburg.de.

Vol. 187, 2005

NOTES 8517

![Diagram](attachment:nicotine_degradation_pathway.png)

FIG. 1. Enzymatic steps in nicotine degradation by *A. nicotinovorans*. The enzymes are NDH, 6HLNO, and KDH. The intermediates are HPON [γ-N-methylaminopropyl-(6-hydroxypyridyl)-ketone], DHPON [γ-N-methylaminopropyl-(2,6-dihydroxypyridyl)-ketone], DHMM, and DHP. Shown in brackets are two possible tautomeric forms of 2,6-dihydroxypseudooxynicotine that may be cleaved, at the position marked by the bar, by a 1,3-β-ketolase, or a 1,5-β-ketolase activity, as indicated by the numbering. See the text for a discussion.

Demonstration of DHPON-cleaving activity of ORF367 protein. Since DHPON, the substrate of the cleaving enzyme (Fig. 1), with time rearranges to the metabolically inactive compound DHMM (10, 18) and efforts to isolate DHPON by high-performance liquid chromatography (HPLC) were unsuccessful, a coupled enzyme assay was employed. This assay included HPON, purified recombinant KDH (P. Sachelaru and R. Brandsch, unpublished), and the ORF367 protein. HPON was prepared enzymatically from 1 ml 10 mM 6-hydroxy-L-nicotine in 75 mM phosphate buffer, pH 7.5, 68 mM NaCl (PBS) with 6-hydroxy-L-nicotine oxidase, prepared as described in reference 12. Following incubation overnight at room temperature, the reaction assays were applied to an HPLC C₁₈ reverse-phase column (250 by 4.6 mm, 5 μm [Aqua; Phenomenex, Aschaffenburg, Germany]) and eluted with 0.1% trifluoroacetic acid in water. A rapid-scanning detector was used (model 206PHD from Linear Instruments Corp., Sykam, Gilching, Germany), and the HPLC peak of HPON, with an absorption maximum at 289 nm (9), was collected, dried in an Eppendorf Concentrator 5231, and dissolved in PBS. The HPLC-purified HPON fraction was then used as substrate in the coupled assay with KDH and ORF367 protein. A reaction was set up in 1 ml PBS–0.15 mM DCPIP (dichlorophenolindophenol) and with various amounts of HPON preparation in the presence of 100 μg KDH. The reaction was allowed to proceed until the absorption at 345 nm of the DHPON product reached its maximum (3 min). Then 10 μg of ORF367 protein was added to the assay, and the linear decrease in absorption

8518 NOTES

A

B

FIG. 2. pAO1 ORFs addressed in this study and their nicotine-dependent transcription. (A) Transcriptional orientation of *kdhl*, ORF367, and ORF116. The fragment number and fragment size amplified by PCR from each ORF are shown. Primers are indicated schematically by arrows, and nucleotide positions of ORFs on pAO1 DNA are given. (B) RT-PCR was performed as described in reference 19 with RNA extracted from nicotine-induced (lanes 1 to 12) and uninduced (lanes 13 to 21) *A. nicotinovorans* pAO1. pAO1 DNA as positive control, RNA as negative control, and RNA-derived cDNA were employed as templates in PCR performed with primer pairs derived from each of the ORFs. Nicotine-induced: I, *kdhl* transcripts (lane 1, pAO1 DNA; lane 2, RNA; lane 3, cDNA); II, ORF367 transcripts (lane 4, pAO1; lane 5, RNA; lane 6, cDNA); III, ORF116 transcripts (lane 7, pAO1 DNA; lane 8, RNA; lane 9, cDNA); II + III, transcripts obtained with a forward primer from ORF367 and a reverse primer from ORF116 (lane 10, pAO1 DNA; lane 11, RNA; lane 12, cDNA). Nicotine uninduced: I (lanes 13 to 15), II (lanes 16 to 18), III (lanes 19 to 21) (same order as for induced). M, molecular size markers.

at 345 nm was monitored for 100 s in an Ultrospec 3100 UV/visible spectrophotometer (Amersham Biosciences). The DHPON concentration generated by KDH during the first step of the coupled assay was calculated from the absorption at 345 nm and the molar extinction coefficient of DHPON of 29 mM⁻¹ cm⁻¹ in 40 mM phosphate buffer, pH 7 (10).

The 2,6-dihydroxypyridine formed in the coupled assay was isolated by HPLC on a Synergi Polar-RP column (Phenomenex, Aschaffenburg, Germany) using 0.1% trifluoroacetic acid in water as the eluent at room temperature and a flow rate of 1 ml/min. The retention time was 7.75 min, and the absorbance maximum at the acidic pH was 302 nm (Fig. 3A). In the absence of ORF367 protein, no material with the characteristics of 2,6-dihydroxypyridine was formed (not shown). Analysis of the peak material by mass spectrometry with a mass spectrometer LCQ (Finnigan, Dreieich, Germany) identified it as 2,6DHP with a molecular mass of 112. Identification of γ-N-methylaminobutyrate in the coupled assay was performed with the aid of a protein sequencer. If γ-N-methylaminobutyrate was present, it should be derivatized by phenylisothiocyanate

FIG. 3. Identification of 2,6-dihydroxypyridine and γ-N-methylaminobutyrate as products of the reaction catalyzed by 2,6-dihydroxypseudooxynicotine hydrolase. (A, a) HPLC analysis of 5 μl of a 10 mM 2,6-dihydroxypyridine standard. (A, b) HPLC analysis of 10 μl of coupled assay. (B, a) Ten microliters of 1 mM phenylisothiocyanate-derivatized γ-N-methylaminobutyrate standard, which eluted directly after phenylthiohydantoin-glutamate. (P, b) Ten microliters of phenylisothiocyanate-derivatized coupled assay analyzed in a protein sequencer.

to the corresponding phenylthiourea and identifiable as such in the on-line HPLC system. Only in the presence of ORF367 protein could a product be detected which coeluted with derivatized authentic γ-N-methylaminobutyrate (Fig. 3B). In the absence of KDH, but the presence of ORF367 protein, no formation of γ-N-methylaminobutyrate was detected (not shown), indicating that HPON was not a substrate in this reaction. From these results, we concluded that ORF367 protein was the DHPON-cleaving enzyme, and it was named 2,6-dihydroxypseudooxynicotine hydrolase (DHPONH).

With increasing amounts of HPON added to the coupled assay, a \(K_m\) of 6 ± 0.5 μM and a \(k_{cat}\) of 792 ± 12 s⁻¹ for the enzyme were determined. The pH optimum of the enzyme, determined in the range from pH 5 to pH 10, was found at pH 7.5.

The hydrolytic cleavage of C—C bonds is a relatively rare enzymatic reaction. It is, however, generally found in the degradation pathway of various hydroxylated aromatic compounds, as the first step following ring opening by an extradiol dioxygenase (*meta*-cleavage pathway). These enzymes belong

to the α/β-fold hydrolase family (14). Members of the family show little amino acid sequence homology, yet share common features of three-dimensional structure. They cover a wide range of enzymatic specificities, but the underlying catalytic mechanism involves a similar arrangement of active-site residues formed by a catalytic triad of serine, aspartate, and histidine (16). Although predicted to be an α/β-fold hydrolase by BLAST analysis, the overall similarity of DHPONH to C—C bond α/β-fold hydrolase family members of the *meta*-cleavage pathway of aromatic compounds (1, 6, 7) was low. However, the sequence GRS217LGG, corresponding to the conserved fingerprint sequence GXSXGG of the “nucleophile elbow” containing the nucleophile serine of the hydrolases, was detected by multiple amino acid sequence alignment of DHPONH with these enzymes. Identification of additional catalytically relevant residues of DHPONH, such as those corresponding to the postulated catalytic triad Ser-Asp-His, will require the correlation of mutagenesis with structural studies of the enzyme.

The α/β-fold hydrolase family enzymes belong to the larger group of β-ketolases, which by a common catalytic reaction mechanism cleave the carbon—carbon bonds of 1,3-diketones and 1,5-dioxyvinyls (17). Gherna et al. (9) pointed out that the carbonyl function attached to C3 of the pyridine ring is β to the 2-pyridone (the 2-hydroxy tautomer) and the structure is that of a β-keto acid amide. Hydroxylated heterocyclic compounds tend to adopt the keto tautomeric form in solution. Depending on which tautomeric form is imposed on the bound substrate (see Fig. 1) by the catalytic center of the enzyme, DHPONH may function as a 1,3-β-ketolase or a 1,5-β-ketolase. Since 6-hydroxypseudooxynicotine is not a substrate for DHPONH, it may be suggested that the 2-keto tautomer of 2,6-dihydroxy pseudooxynicotine is the substrate of the enzyme. Because of the ease of enol-keto tautomerization with loss of the aromatic nature of the pyridine ring, no ring opening is required for the cleaving reaction to proceed. Thus, hydrolysis takes place between C3 and the C carbonyl of the side chain, leading to 2,6-dihydroxypyridine and γ-N-methylaminobutyrate.

We thank Karl Decker (Freiburg, Germany) for enzyme substrates and valuable discussions.

This work was supported by a grant from the Deutsche Forschungsgemeinschaft to R.B.

---

**REFERENCES**

1. Ahmad, D., J. Fraser, M. Sylvestre, A. Larose, A. Khan, J. Bergeron, J. M. Juteau, and M. Sondossi. 1995. Sequence of the *bphD* gene encoding 2-hydroxy-6-oxo-(phenyl/chlorophenyl)hexa-2,4-dienoic acid (HOP/cPDA) hydrolase involved in the biphenyl/polychlorinated biphenyl degradation path-
way in *Comamonas testosteroni*: evidence suggesting involvement of Ser112 in catalytic activity. Gene **156**:69–74.
2. Baitsch, D., C. Sandu, and R. Brandsch. 2001. A gene cluster on pAO1 of *Arthrobacter nicotinovorans* involved in degradation of the plant alkaloid nicotine: cloning, purification, and characterization of 2,6-dihydroxypyridine 3-hydroxylase. J. Bacteriol. **183**:5262–5267.
3. Berthold, H., M. Scanarini, C. C. Abney, B. Frorath, and W. Northemann. 1992. Purification of recombinant antigenic epitopes of the human 68-kDa (U1) ribonucleoprotein antigen using the expression system pH6EX3 followed by metal chelating affinity chromatography. Protein Expr. Purif. **3**: 50–56.
4. Chiribau, C.-B., C. Sandu, M. Fraaije, E. Schiltz, and R. Brandsch. 2004. A novel γ-N-methylaminobutyrate demethylating oxidase involved in catabolism of the tobacco alkaloid nicotine by *Arthrobacter nicotinovorans* pAO1. Eur. J. Biochem. **271**:4677–4684.
5. Decker, K., and H. Bleeg. 1965. Induction and purification of stereospecific nicotine oxidizing enzymes from *Arthrobacter oxidans*. Biochim. Biophys. Acta **105**:313–334.
6. Diaz, E., and K. N. Timmis. 1995. Identification of functional residues in a 2-hydroxymuconic semialdehyde hydrolase. A new member of the alpha/beta hydrolase-fold family of enzymes which cleaves carbon-carbon bonds. J. Biol. Chem. **270**:6403–6411.
7. Dunn, G., M. G. Montgomery, F. Mohammed, A. Coker, J. B. Cooper, T. Robertson, J. L. Garcia, T. D. Bugg, and S. P. Wood. 2005. The structure of the C—C bond hydrolase MhpC provides insights into its catalytic mechanism. J. Mol. Biol. **346**:253–265.
8. Freudenberg, W., K. König, and J. R. Andreasen. 1988. Nicotine dehydrogenase from *Arthrobacter oxidans*: a molybdenum-containing hydroxylase. FEMS Microbiol. Lett. **52**:13–18.
9. Gherna, R. L., S. H. Richardson, and S. C. Rittenberg. 1965. The bacterial oxidation of nicotine. VI. The metabolism of 2,6-dihydroxypseudoxy-nicotine. J. Biol. Chem. **240**:3669–3674.
10. Gries, F. A., K. Decker, H. Eberwein, and M. Brühmüller. 1961. Über den Abbau des Nicotins durch Bakterienenzyme. VI. Die enzymatische Umwandlung des (γ-Methylamino-propyl)-[6-hydroxy-pyridyl-(3)]-ketons. Biochem. Z. **335**:285–302.
11. Hille, R. 1996. The mononuclear molybdenum enzymes. Chem. Rev. **96**: 2757–2816.
12. Hinkkanen, A., E.-M. Lilius, J. Nowack, R. Maas, and K. Decker. 1983. Purification of the flavoproteins 6-hydroxy-D- and 6-hydroxy-L-nicotine oxidase using hydrophobic affinity chromatography. Hoppe-Seyler’s Z. Physiol. Chem. **364**:801–806.
13. Holmes, P. E., and S. Rittenberg. 1972. The bacterial oxidation of nicotine. VII. Partial purification and properties of 2,6-dihydroxypyridine oxidase. J. Biol. Chem. **247**:7622–7627.
14. Holmquist, M. 2000. Alpha/beta-hydrolase fold enzymes: structures, functions and mechanisms. Curr. Protocol Peptide Sci. **1**:209–235.
15. Igloi, G. L., and R. Brandsch. 2003. Sequence of the 165-kilobase catabolic plasmid pAO1 from *Arthrobacter nicotinovorans* and identification of a pAO1-dependent nicotine uptake system. J. Bacteriol. **185**:1976–1986.
16. Li, C., M. G. Montgomery, F. Li, J. J. Mohammed, S. P. Wood, and T. D. Bugg. 2005. Catalytic mechanism of C—C hydrolase MhpC from Escherichia coli: kinetic analysis of His263 and Ser110 site-directed mutants. J. Mol. Biol. **346**:241–251.
17. Pokorny, D., L. Brecker, W. Steiner, and D. W. Ribbons. 1999. β-Ketolases—forgotten hydrolytic enzymes? Trends Biotechnol. **15**:291–296.
18. Richardson, S. H., and S. C. Rittenberg. 1961. The bacterial oxidation of nicotine. IV. The isolation and identification of 2,6-dihydroxy-N-methylmyosmine. J. Biol. Chem. **236**:959–963.
19. Sandu, C., C.-B. Chiribau, and R. Brandsch. 2003. Characterization of HdnoR, the transcriptional repressor of the 6-hydroxy-D-nicotine oxidase gene of *Arthrobacter nicotinovorans* pAO1, and its DNA-binding activity in response to L- and D-nicotine derivatives. J. Biol. Chem. **278**:51307–51315.
20. Schenk, S., A. Hoelz, B. Krauß, and K. Decker. 1998. Gene structure and properties of enzymes of the plasmid-encoded nicotine catabolism of *Arthrobacter nicotinovorans*. J. Mol. Biol. **284**:1322–1339.
