
Sone, T., Saeki, Y., Toh-e, A., and Yokosawa, H. (2004). Sem1p is a novel subunit of the 26 S proteasome from *Saccharomyces cerevisiae*. J. Biol. Chem. **279**, 28807–28816.

Sudol, M., and Hunter, T. (2000). NeW wrinkles for an old domain. Cell **103**, 1001–1004.

Thrower, J. S., Hoffman, L., Rechsteiner, M., and Pickart, C. M. (2000). Recognition of the polyubiquitin proteolytic signal. EMBO J. **19**, 94–102.

Verma, R., Aravind, L., Oania, R., McDonald, W. H., Yates, J. R., 3rd, Koonin, E. V., and Deshaies, R. J. (2002). Role of Rpn11 metalloprotease in deubiquitination and degradation by the 26S proteasome. Science **298**, 611–615.

Verma, R., Chen, S., Feldman, R., Schieltz, D., Yates, J., Dohmen, J., and Deshaies, R. J. (2000). Proteasomal proteomics: identification of nucleotide-sensitive proteasome-interacting proteins by mass spectrometric analysis of affinity-purified proteasomes. Mol. Biol. Cell **11**, 3425–3439.

Verma, R., McDonald, H., Yates, J. R., 3rd, and Deshaies, R. J. (2001). Selective degradation of ubiquitinated Sic1 by purified 26S proteasome yields active S phase cyclin-Cdk. Mol. Cell **8**, 439–448.

Verma, R., Oania, R., Graumann, J., and Deshaies, R. J. (2004). Multiubiquitin chain receptors define a layer of substrate selectivity in the ubiquitin-proteasome system. Cell **118**, 99–110.


[15] Large- and Small-Scale Purification of Mammalian 26S Proteasomes

By YUKO HIRANO, SHIGEO MURATA, and KEIJI TANAKA

Abstract

The 26S proteasome is an ATP-dependent protease known to collaborate with ubiquitin, whose polymerization acts as a marker for regulated and enforced destruction of unnecessary proteins in eukaryotic cells. It is an unusually large multi-subunit protein complex, consisting of a central catalytic machine (called the 20S proteasome or CP/core particle) and two terminal regulatory subcomplexes, termed PA700 or RP/regulatory particle, that are attached to both ends of the central portion in opposite orientations to form an enzymatically active proteasome. To date, proteolysis driven by the ubiquitin-proteasome system has been shown to be involved in a diverse array of biologically important processes, such as the cell cycle, immune response, signaling cascades, and developmental programs; and the field continues to expand rapidly. Whereas the proteasome complex has been highly conserved during evolution because of its fundamental roles in cells, it has also acquired considerable diversity in multicellular organisms, particularly in mammals, such as immunoproteasomes, PA28, S5b, and various alternative splicing forms of S5a (Rpm 10). However, the details of the ultimate pathophysiological roles


METHODS IN ENZYMOLOGY, VOL. 399
Copyright 2005, Elsevier Inc. All rights reserved.
DOI: 10.1016/S0076-6879(05)99015-0

of mammalian proteasomes have remained elusive. This article focuses on methods for assay and purification of 26S proteasomes from mammalian cells and tissues.

# Introduction

The 26S proteasome is a protein-destroying apparatus capable of degrading a variety of cellular proteins in a rapid and timely fashion. Most, if not all, substrates are modified by ubiquitin before their degradation by the 26S proteasome. The covalent attachment of multiple ubiquitins on target proteins is catalyzed by a multienzyme cascade, consisting of the E1 (Ub-activating), E2 (Ub-conjugating), and E3 (Ub-ligating) enzymes ([Hershko and Ciechanover, 1998; Pickart, 2001]). The resulting polyubiquitin chain serves as a signal for trapping the target protein, and, consequently, the substrate is destroyed after proteolytic attack by the 26S proteasome ([Baumeister et al., 1998; Coux et al., 1996]). The 26S proteasome is a dumbbell-shaped particle, consisting of a centrally located, cylindrical 20S proteasome (alias core particle, CP) that functions as a catalytic machine and two large terminal PA700 modules (alias 19S complex, or regulatory particle, RP) attached to the 20S core particle in opposite orientations.

The 20S proteasome/CP is a complex with a sedimentation coefficient of 20S and a molecular mass of approximately 750 kDa (see a model of Fig. 1). It is a barrel-like particle formed by the axial stacking of four rings made up of two outer $\alpha$-rings and two inner $\beta$-rings, which are each made up of seven structurally similar $\alpha$- and $\beta$-subunits, respectively, being associated in the order of $\alpha_{1-7}\beta_{1-7}\beta_{1-7}\alpha_{1-7}$. The overall architectures of the highly ordered structures of yeast (*Saccharomyces cerevisiae*) and mammalian (bovine) 20S proteasomes are indistinguishable, as demonstrated by x-ray crystallography ([Groll et al., 1997; Unno et al., 2002]). Three of the $\beta$-type subunits of each inner ring have catalytically active threonine residues at their N-terminus, all of which show N-terminal nucleophile (Ntn) hydrolase activity, indicating that the proteasome is a novel threonine protease, differing from the known protease families categorized as seryl-, thiol-, carboxyl-, and metalloproteases. The catalytic $\beta 1$, $\beta 2$, and $\beta 5$ subunits correspond to caspase-like/PGPH (peptidylglutamyl-peptide hydrolyzing), trypsin-like, and chymotrypsin-like activities, respectively, which are capable of cleaving peptide bonds at the C-terminal side of acidic, basic, and hydrophobic amino acid residues, respectively. Two copies of these three active sites face the interior of the cylinder and reside in a chamber formed by the centers of the abutting $\beta$ rings.

PA700/RP contains approximately 20 distinct subunits of 25–110 kDa, which can be classified into two subgroups: a subgroup of six ATPases,

[15] PURIFICATION OF MAMMALIAN 26S PROTEASOMES 229

FIG. 1. Molecular organization of 26S proteasomes. (Left panel) Averaged image of the 26S proteasome complex of rat based on electron micrographs. The $\alpha$ and $\beta$ rings of the 20S proteasome are indicated. Photograph kindly provided by W. Baumeister. (Right panel) Schematic drawing of the subunit structure. CP, core particle (alias 20S proteasome); RP, 19S regulatory particle (alias PA700) consisting of the base and lid subcomplexes; Rpn, RP non-ATPase; Rpt, RP triple-ATPase. Note that relative positions of 19S subunits have not been established.

numbered from Rpt1 to Rpt6 (i.e., **RP** triple ATPases 1–6), that are structurally similar and have been highly conserved during evolution, and a subgroup of more than 15 heterogeneous subunits, numbered from Rpn1 to Rpn15 (i.e., **RP** non-ATPases 1–15), that are structurally unrelated to the members of the ATPase family (*Tanaka et al.*, 2005). The PA700/RP structurally consists of two subcomplexes, known as “base” and “lid,” which, in the 26S proteasome, correspond to the portions of PA700 proximal and distal, respectively, to the 20S proteasome (*Glickman et al.*, 1998). The base is made up of six ATPases (Rpt1–Rpt6) and two large regulatory components Rpn1 and Rpn2, functioning as presumptive receptor(s) of ubiquitin-like proteins, and the lid contains multiple non-ATPase subunits (Rpn3–Rpn15). The base-complex is thought to bind in an ATP-dependent manner to the outer $\alpha$-ring of the central 20S proteasome. The six ATPases in this base-complex are assembled into one ring complex. The main role of the ATPase ring is to supply energy continuously for the degradation

of target proteins. In fact, the metabolic energy liberated by ATP consumption is probably used for unfolding target proteins, gate opening of the 20S proteasome, and substrate translocation so that they can penetrate into the channel formed by the $\alpha$- and $\beta$-rings of the 20S proteasome (Ogura and Tanaka, 2003). On the other hand, the lid-complex is thought to be involved in the recognition of polyubiquitylated target proteins, deubiquitylation of substrates for reutilization of ubiquitin, and physical interactions with various other proteins that influence proteasome activity. The details of molecular bases for functions of individual subunits, however, are largely unknown to date.

## Assay of Proteasome Activity

### Assay of Suc-LLVY-MCA Degrading Activity

The 26S proteasome is incubated at $37^\circ$ for 10 min in $50 \mathrm{mM}$ Tris-HCl buffer (pH 8.5) containing $1 \mathrm{mM}$ dithiothreitol (DTT) and a $0.1 \mathrm{mM}$ concentration of a fluorogenic substrate, the synthetic peptide succinyl-Leu-Leu-Val-Tyr-4-methyl-coumaryl-7-amide (Suc-LLVY-MCA) (Peptide Institute). This substrate is added to the assay mixture at a final concentration of DMSO of 1% (v/v). The reaction was stopped by adding 10% SDS at final concentration of 1%, and the reaction product is measured fluorometrically (excitation 380 nm, emission 460 nm).

The 26S proteasome can be visualized on electrophoretic gels as a Suc-LLVY-MCA-degrading enzyme. Samples are subjected to nondenaturing polyacrylamide gel electrophoresis (PAGE) at $4^\circ$ before the gels are overlaid with $0.1 \mathrm{mM}$ Suc-LLVY-MCA for 10 min at room temperature. Fluorescence was then detected under ultraviolet light.

### Assay of Polyubiquitylated ${}^{125}\mathrm{I}$-Lysozyme Degrading Activity

Preparation of polyubiquitylated ${}^{125}\mathrm{I}$-lysozyme can be prepared by using purified E1, E2, and E3 enzymes, as described previously (Fujimuro et al., 1994; Tamura et al., 1991). For assay of degradation of polyubiquitylated ${}^{125}\mathrm{I}$-lysozyme, samples of ${}^{125}\mathrm{I}$-lysozyme-ubiquitin conjugates (5000–10,000 cpm) are incubated at $37^\circ$ for 15–60 min in a total volume of $100 \mu\mathrm{l}$ of reaction mixture containing $50 \mathrm{mM}$ Tris-HCl buffer (pH 8.5) with $5 \mathrm{mM} \mathrm{MgCl}_2$, $2 \mathrm{mM}$ ATP, an ATP-regeneration system ($10 \mu\mathrm{g}$ of creatine phosphokinase and $10 \mathrm{mM}$ phosphocreatine), $1 \mathrm{mM}$ DTT, and a suitable amount of the 26S proteasome. After the reaction is stopped by adding SDS-PAGE sample buffer, the proteins are subjected to

SDS-PAGE and autoradiographed. The gels are dried and exposed to x-ray film at $-70^\circ$ with an intensifying screen. For measuring the degradation of ${}^{125}$I-lysozyme-ubiquitin conjugates into acid-soluble fragments by the 26S proteasome, the reaction is terminated by addition of 575 $\mu$l of 10% trichloroacetic acid (TCA) with 125 $\mu$l of 4% bovine serum albumin (BSA) as a carrier, and the radioactivity recovered in the acid-soluble fraction after centrifugation is determined in a $\gamma$-counter.

### Assay of Polyubiquitylated Sic1 Degrading Activity

Deshais and his colleagues devised an *in vitro* assay method of 26S proteasomes using polyubiquitylated Sic1, a CDK inhibitor in the budding yeast, as a substrate. Polyubiquitylation of Sic1 phosphorylated by CDK is catalyzed by E1, E2 (Cdc34), and E3 ($\mathrm{SCF}^{\mathrm{Cdc4}}$). The details of the methods were described previously (*Verma and Deshaies*, 2005; *Verma et al.*, 2001).

Saeki *et al.* devised an improved method by preparing PY motif-inserted Sic1 (Sic1${}^{\mathrm{PY}}$) that is effectively polyubiquitylated by Rsp5 E3-ligase and rapidly degraded by 26S proteasomes in an ATP-dependent fashion (for details, see Chapter 14 [*Saeki et al.*, 2005]). It is of note that all components used in this assay system can be easily expressed and purified using bacterial cells.

### Assay of ${}^{35}$S-ODC Degrading Activity

For quantitative and sensitive measurement of ATP-dependent proteolysis activity of mammalian proteasomes *in vitro*, ornithine decarboxylase (ODC) is a useful substrate. ODC is the best-known natural substrate of the proteasome whose recognition and degradation are independent of ubiquitilation (*Murakami et al.*, 1992). Antizyme (AZ), an ODC inhibitory protein that is needed for this *in vitro* degradation assay, is prepared as a recombinant protein (*Murakami et al.*, 1999). Rat AZ cDNA Z1 is expressed in *Escherichia coli*, and an extract of the *E. coli* (800 mg protein) is applied to a monoclonal anti-AZ antibody (HZ-2E9)-AffiGel 10 column (1 ml); the column is washed with 25 mM Tris-HCl buffer (pH 7.5) containing 1 mM EDTA, 1 mM DTT, and 0.01% Tween 80, supplemented with 4 M NaCl. AZ is eluted with 4 ml of 3 M MgCl${}_2$, and the eluate is dialyzed against the same buffer. ${}^{35}$S-labeled ODC is produced by an *in vitro* translation system using rabbit reticulocyte lysate containing rat ODC mRNA, ${}^{35}$S-labeled methionine, and ${}^{35}$S-labeled cysteine (Du Pont NEN). The reaction is applied to a monoclonal anti-ODC antibody (HO101)-AffiGel 10 column (0.15 ml). The procedures for wash and elution are the same as AZ purification.
The degradation of the recombinant $^{35}$S-labeled-ODC (2000–3000 cpm) is assayed in the presence of ATP, an ATP-regenerating system, and AZ (Murakami *et al.*, 1999). After incubation for 60 min at 37°C, the amount of TCA-soluble radioactivity in the reaction mixture is measured, and the activity is expressed as a percent of total ODC added.

### Comments for Assays

1. Suc-LLVY-MCA (i.e., a substrate of chymotrypsin-like activity) is recommended as a sensitive substrate. Various other fluorogenic peptides, such as Boc (*t*-Butyloxycarbonyl)-Leu-Arg-Arg-MCA and Z (benzyloxy-carbonyl)-Leu-Leu-Glu-MCA for monitoring trypsin-like and caspase-like/PGPH (peptidylglutamyl-peptide hydrolyzing) activity, respectively, are suitable for measurement of 20S and 26S proteasomal activity, because proteasomes show broad substrate specificity. The hydrolytic activities toward various fluorogenic substrates are determined by measuring the fluorescence of groups liberated from these peptides. Latent 20S proteasomes can be activated in various ways. We recommend the use of SDS at low concentrations of 0.02–0.08% for activation of Suc-LLVY-MCA breakdown; the optimal concentration depends on the enzyme source and the protein concentration used. The fluorogenic peptide (e.g., Suc-LLVY-MCA) can be used for assay of the 26S proteasome, because it is active without any treatment unlike the latent 20S proteasome. MCA (4-methyl-coumaryl-7-amide) is used as a reference compound for analysis with peptidyl-MCAs.

2. Various fluorogenic peptides are suitable for measurement of 20S and 26S proteasomal activity, but note that all of them are not specific substrates for these proteasomes. For specific assay, ATP-dependent degradation of polyubiquitinated $^{125}$I-lysozyme, or Sic1/Sic1$^{\text{PY}}$ should be measured, although such assay is not easy, because three kinds of enzymes, E1, E2, and E3, must be purified for *in vitro* preparation of ubiquitinated substrates. Therefore, for quantitative and sensitive measurement of ATP-dependent proteolysis activity of mammalian proteasomes *in vitro*, ODC is a useful substrate. Note that AZ is not present in lower organisms such as yeasts, and thus this assay is not fit for proteasomes isolated from these cells.

3. The purification of the 26S proteasome is monitored by measuring ATPase activity at later steps of its purification, because the 26S proteasome has intrinsic ATPase activity (Ugai *et al.*, 1993). Note that this assay is not sensitive and cannot be used in crude extracts because of the existence of numerous other ATPases in cells.
Large-Scale Purification of 20S and 26S Proteasomes from Rat Liver

**Purification Procedure of 20S Proteasomes**

Step 1. Homogenize 200–400 g samples of rat liver in 3 vol of 25 mM Tris-HCl buffer (pH 7.5) containing 1 mM DTT and 0.25 M sucrose in a Potter-Elvehjem homogenizer. Centrifuge the homogenate for 1 h at 70,100g, and use the resulting supernatant as the crude extract.

Step 2. Add glycerol at a final concentration of 20% to the crude extract. Then mix the extract with 500 g of Q-Sepharose (Amersham) that has been equilibrated with buffer A (25 mM Tris-HCl [pH 7.5] containing 1 mM DTT [or 10 mM 2-mercaptoethanol] and 20% glycerol). Wash the Q-Sepharose with the buffer A on a Büchner funnel and transfer to a column (5 × 60 cm). Wash the column with buffer A and elute the material with 2 liters of a linear gradient of 0–0.8 M NaCl in buffer A, and measure the activity of proteasomes using Suc-LLVY-MCA as a substrate.

Step 3. Pool fractions containing 20S proteasomes from the Q-Sepharose column and add 50% polyethylene glycol 6000 (Sigma) (adjust to pH 7.4) to a final concentration of 15% with gentle stirring. After 15 min, centrifuge the mixture at 10,000g for 20 min, dissolve the resulting pellet in a minimum volume (approximately 50 ml) of buffer A, and centrifuge at 20,000g for 10 min to remove insoluble material.

Step 4. Fractionate the material precipitated with polyethylene glycol on a Bio-Gel A-1.5m column (5 × 90 cm) in buffer A. Collect fractions of 10 ml and assay their proteasome activity. Pool fractions of 20S proteasomes.

Step 5. Apply the active fractions from the Bio-Gel A-1.5m (Bio-Rad) column directly to a column of hydroxylapatite equilibrated with buffer B (10 mM phosphate buffer [pH 6.8] containing 1 mM DTT and 20% glycerol). Wash the column with the same buffer and elute the material with 400 ml of a linear gradient of 10–300 mM phosphate. Collect fractions of 4 ml. 20S proteasomes are eluted with approximately 150 mM phosphate.

Step 6. Combine the active fractions from the hydroxylapatite (Bio-Rad), dialyze against buffer A, and apply to a column of heparin-Sepharose CL-6B (Amersham) equilibrated with buffer A. Wash the column with the same buffer until the absorbance of the eluate at 280 nm returns to baseline. Then elute with 200 ml of a linear gradient of 0–0.4 M NaCl in buffer A, and collect fractions of 2 ml. 20S proteasomes are eluted with approximately 75 mM NaCl.

Step 7. Pool the fractions with high proteasomal activity, dialyze against buffer A, and concentrate to about 5 mg/ml protein by ultrafiltration in an Amicon cell with a PM-10 membrane (Millipore). The enzyme can be stored at –80° for at least 2–3 years. The SDS-PAGE analysis of purified enzyme

revealed that it consists of a set of proteins, displaying the molecular weights of 20–32 kDa (see left panel of Fig. 2).

# Purification Procedure of 26 Proteasomes

Step 1. Homogenize 200–400-g samples of rat liver in 3 vol of 25 mM Tris-HCl buffer (pH 7.5) containing 1 mM DTT, 2 mM ATP, and 0.25 M sucrose in a Potter-Elvehjem homogenizer. Centrifuge the homogenate for 1 h at 70,100g and use the resulting supernatant as the starting material.

Step 2. Recentrifuge the crude supernatant for 5 h at 70,100g to obtain 26S proteasomes, which precipitate almost completely. Dissolve the precipitate in a suitable volume (40–50 ml) of buffer C (buffer A containing 0.5 mM ATP) and centrifuge at 20,000g for 30 min to remove insoluble material.

Step 3. Apply samples of the preparation from step 2 to a Bio-Gel A-1.5m column (5 × 90 cm) in buffer C. Collect fractions of 10 ml and assay the 26S proteasome activity in the fractions. Pool fractions of 26S proteasomes.

Step 4. Add ATP at a final concentration of 5 mM to the pooled fractions of 26S proteasomes from the Bio-Gel A-1.5m column. Apply the sample directly to a hydroxylapatite column with a 50-ml bed volume that has been equilibrated with buffer D (buffer B containing 5 mM ATP). Recover the 26S proteasome in the flow-through fraction, because they do

| KDa | 20S | 26S | 9.7 | 8.3 | 6.4 | 4.9 | 4.1 | pl |
|------|-----|-----|-----|-----|-----|-----|-----|----|
|      |     |     |     |     |     |     |     |    |

SDS-PAGE  
2D-PAGE (26S Proteasomes)

FIG. 2. Electrophoretic analyses of 20S and 26S proteasomes from rat liver. (Left panel) SDS-PAGE pattern of purified 20S and 26S proteasomes. (Right panel) 2D-PAGE pattern of purified 26S proteasomes. Proteins were stained with Coomassie Brilliant Blue (CBB).

not associate with this column in the presence of 5 mM ATP. Approximately 70% of the proteins, including free 20S proteasomes, bind to the hydroxylapatite resin.

Step 5. Apply the flow-through fraction from the hydroxylapatite column to a Q-Sepharose column that has been equilibrated with buffer C without ATP and washed with 1 bed volume of buffer C. Wash the column with 5 bed volumes of buffer C, and elute the adsorbed materials with 300 ml of a linear gradient of 0–0.8 M NaCl in buffer C. Collect fractions of 3.0 ml. Proteins with ability to degrade Suc-LLVY-MCA with or without 0.05% SDS are eluted with approximately 0.4 M NaCl as a single symmetrical peak. ATPase activity and the ATP-dependent activity necessary to degrade ${}^{125}$I-lysozyme-Ub conjugates are observed at the same position as the peptidase activity and are eluted as superimposable symmetrical peaks, which suggests a specific association of ATPase with the 26S proteasome complex. Collect the protein in fractions exhibiting high activity.

Step 6. Concentrate the 26S proteasome fraction obtained by Q-Sepharose chromatography to 2.0 mg/ml by ultrafiltration with an Amicon PM-30 membrane, and subject samples of 2.0 mg of protein to 10–40% glycerol density-gradient centrifugation (30 ml in buffer C containing 2 mM ATP). Centrifuge for 22 h at 82,200g in a SW rotor, and collect fractions of 1 ml from the bottom of the centrifuge tube. A single major peak of peptidase activity, measured in the absence of SDS, is eluted around fraction 15, but when the activity is assayed with 0.05% SDS, another small peak is observed around fraction 20. The latter peak corresponds to the elution position of 20S proteasomes. ATPase activity is observed at the same position as peptidase activity. Activity for ATP-dependent degradation of ${}^{125}$I-lysozyme-Ub conjugates is also observed as a single symmetrical peak, coinciding in position with the ATPase and peptidase activities in the absence of SDS. No significant ${}^{125}$I-lysozyme-Ub conjugate degrading activity is detected in fractions of 20S proteasomes. Pool fractions 12–16 and store at $-80^\circ$. Two-dimensional (2D) PAGE revealed that the purified enzyme consists of a set of approximately 40 proteins displaying the molecular weights of 20–110 kDa and isoelectric points (pIs) of 3–10 (see right panel of Fig. 2).

## Small-Scale Purification of 26S Proteasomes

### Conventional Chromatographic Purification of Nuclear 26S Proteasomes

Preparation of Nuclear Extracts. The nuclei from rat liver were prepared as described previously (Tanaka et al., 1989).

Step 1. Homogenize animal tissues (mouse or rat) (50 g) in 4 volumes (200 ml) of 50 mM Tris-HCl (pH 8.0) buffer containing 1 mM DTT, 15 mM KCl, 1 mM EDTA, 5 % glycerol, 2.2 M sucrose, and Complete protease inhibitor cocktail (Roche Molecular Biochemical). The resulting homogenates are layered on a cushion of 50 mM Tris-HCl (pH 8.0) buffer containing 1 mM DTT, 15 mM KCl, 1 mM EDTA, 10% glycerol, and 2 M sucrose occupying one third the volume of centrifuge tubes and are centrifuged at 83,000g for 60 min in aSW rotor to pellet the nuclei.

Step 2. Disrupt the isolated nuclei by sonication in 50 mM Tris-HCl (pH 8.0) buffer containing 1 mM DTT, 2 mM ATP, and Complete protease inhibitor cocktail. The nuclear extracts were obtained by centrifugation at 10,000g for 20 min as the resulting supernatants (approximately 40 mg). The purity of the nuclear extracts should be examined by Western blot analysis. Histone H1, a marker of nucleus (detected with antibodies from Upstate Biotechnology), but not LDH, a marker of cytosol (detected with antibodies from Abcam), should be detected in the nuclear extracts without obvious cross-contamination.

### Purification of Nuclear 26S Proteasomes

Step 1. Load the nuclear extracts on a RESOURCE Q column (Amershah Biosciences) equilibrated with buffer E (50 mM Tris-HCl [pH 8.0] buffer containing 1 mM DTT, 2 mM ATP, and 10% glycerol), wash the column with buffer E, and elute bound proteins with a gradient of 0–0.8 M NaCl in buffer E. Pool the fractions with Suc-LLVY-MCA degrading activity. 26S proteasomes are eluted with 450–500 mM NaCl.

Step 2. Add ATP at a final concentration of 5 mM to the pooled fractions of 26S proteasomes from RESOURCE Q column. Load the fractions on a Hydroxylapatite column (Bio-Rad) equilibrated with buffer D. Recover 26S proteasomes in the flow-through fractions. (Check Suc-LLVY-MCA degrading activity).

Step 3. Load the flow-through fractions on a Mono Q column (Amershah Biosciences) equilibrated with buffer E, wash the column with buffer E, and elute bound proteins with a gradient of 0–0.8 M NaCl in buffer E (0.5 ml/fraction). Pool the fraction exhibiting peak activity and the adjacent fractions. 26S proteasomes are eluted with 450–500 mM NaCl. This step helps to concentrate 26S proteasomes for the next step.

Step 4. Subject the pooled fractions (approximately 2.0 mg protein/1.5 ml) to 10–40% glycerol density-gradient centrifugation (30 ml in buffer F [50 mM Tris-HCl {pH 8.0} buffer containing 1 mM DTT and 2 mM ATP]). Centrifuge for 22 h at 82,200g in SW28 (Beckman) or P28S (HITACHI) rotor, collect fractions of 1 ml from the top of the centrifuge tube, and check

Suc-LLVY-MCA degrading activity. A single major peak of peptidase activity, measured in the absence of SDS, corresponds to 26S proteasomes sedimented around fraction 20 (approximately 0.1 mg protein). Pool fractions with high Suc-LLVY-MCA degrading activity and store at $-80^\circ$.

**Affinity Purification**

Conventional biochemical techniques for purification of 26S proteasomes use chromatographic columns as described previously. During the purification steps, 26S proteasomes are exposed to high ionic strength buffers, which cause dissociation of proteins bound to proteasomes transiently or with low affinity. In yeast, tagging of certain subunits of 26S proteasomes that are driven by their own promoters and purification by the tag in milder conditions has enabled identification of many novel proteasome-interacting proteins (PIPs). Mammalian proteasomes are expected to have a more complicated network and it is essential to clarify mammalian PIPs to fully understand the roles of proteasomes. To solve this problem, we developed an ES cell line that has one allele of the human Rpn11 gene tagged with a C-terminal flag epitope ($\text{Rpn11}^{\text{FLAG/+}}$ ES cells) by a homologous recombination technique. The method for establishing the ES cell line will be described elsewhere.

**Procedure**

Step 1. Grow $\text{Rpn11}^{\text{FLAG/+}}$ ES cells on six 10-cm dishes on which mitomycin C-treated murine embryonic fibroblasts were laid.

Step 2. Collect cells using an appropriate scraper with PBS in a conical tube, centrifuge at 1500g for 10 min. Wash cells once more with PBS.

Step 3. The cell pellet was resuspended in 6 ml of buffer G (20 mM HEPES-NaOH [pH 7.5], 0.2% NP-40, 2 mM ATP, 1 mM DTT) by gentle pipetting and placed on ice for 10 min.

Step 4. Centrifuge at 10,000g for 10 min to remove cell debris.

Step 5. To preclear the lysate, pass the lysate through a column packed with 0.5 ml (bed volume) of Sepharose CL-4B (Sigma).

Step 6. Apply the flow-through onto the column packed with 50 $\mu$l (bed volume) of M2-agarose (Sigma). Pass the flow-through through the column five times.

Step 7. Wash the column 10 times with 5 ml of buffer G supplemented with 50 mM NaCl.

Step 8. Incubate the column with 50 $\mu$l of FLAG peptide (Sigma; dissolved at 100 $\mu$g/ml in buffer G) on ice for 3 min.

Step 9. Recover the eluted proteins by centrifugation at 1000 rpm for 1 min.

Step 10. Repeat step 8 and step 9 three more times, and collect all the eluted materials in one tube. We usually obtained about 60 μg of 26S proteasome in this procedure. The 2D PAGE pattern of 26S proteasomes purified by this method is shown in Fig. 3.

## Discussion

Proteasomes have been purified from a variety of eukaryotic cells by many investigators. Many purification methods have been reported, but no special techniques are necessary, because 20S proteasomes are very stable and abundant in cells, constituting 0.5–1.0% of the total cellular proteins. The procedures used for purification of 20S proteasomes obviously differ, depending on whether they are small or large operations. For their isolation from small amounts of biological materials, such as cultured cells, 10–40% glycerol density gradient centrifugation is very effective. 20S proteasomes are present in a latent form in cells and can be isolated in this form in the presence of 20% glycerol. For their isolation in high yield, a key point is to keep them in their latent form, because their activation results in autolytic loss of a certain subunit(s) and marked reduction of enzymatic activities, particularly their hydrolysis of various proteins. Accordingly, all buffers used contain 10–20% glycerol as a stabilizer. Furthermore, a reducing agent is required, because 20S proteasomes precipitate in its absence.

![Two-dimensional PAGE pattern of 26S proteasomes purified from Rpn11<sup>FLAG/+</sup> ES cells. Proteins were stained with CBB.](image.png)

FIG. 3. Two-dimensional PAGE pattern of 26S proteasomes purified from Rpn11<sup>FLAG/+</sup> ES cells. Proteins were stained with CBB.

All purification procedures are performed at $4^{\circ}$, but operations in a high-performance liquid chromatography (HPLC) apparatus can be carried out within a few hours at room temperature.

For purification of the 26S proteasome, ATP ($0.5 \mathrm{mM}$ or $2 \mathrm{mM}$) together with $20 \%$ glycerol and $1 \mathrm{mM} \mathrm{DTT}$ should be added to all solutions used, because they strongly stabilize the 26S proteasome complex: the purified enzyme is stable during storage at $-70^{\circ}$ for at least 6 months in the presence of $2 \mathrm{mM} \mathrm{ATP}$ and $20 \%$ glycerol. Chromatographic steps that require high salt concentrations or extremes of pH should be avoided, because these operations may result in dissociation of the 26S complex into its constituents.

## References

Baumeister, W., Walz, J., Zuhl, F., and Seemuller, E. (1998). The proteasome: Paradigm of a self-compartmentalizing protease. *Cell* **92**, 367–380.

Coux, O., Tanaka, K., and Goldberg, A. L. (1996). Structure and functions of the 20S and 26S proteasomes. *Annu. Rev. Biochem.* **65**, 801–847.

Fujimuro, M., Sawada, H., and Yokosawa, H. (1994). Production and characterization of monoclonal antibodies specific to multi-ubiquitin chains of polyubiquitinated proteins. *FEBS Lett.* **349**, 173–180.

Glickman, M. H., Rubin, D. M., Coux, O., Wefes, I., Pfeifer, G., Cjeka, Z., Baumeister, W., Fried, V. A., and Finley, D. (1998). A subcomplex of the proteasome regulatory particle required for ubiquitin-conjugate degradation and related to the COP9-signalosome and eIF3. *Cell* **94**, 615–623.

Groll, M., Ditzel, L., Lowe, J., Stock, D., Bochtler, M., Bartunik, H. D., and Huber, R. (1997). Structure of 20S proteasome from yeast at 2.4 Å resolution. *Nature* **386**, 463–471.

Hershko, A., and Ciechanover, A. (1998). The ubiquitin system. *Annu. Rev. Biochem.* **67**, 425–479.

Murakami, Y., Matsufuji, S., Hayashi, S. I., Tanahashi, N., and Tanaka, K. (1999). ATP-dependent inactivation and sequestration of ornithine decarboxylase by the 26S proteasome are prerequisites for degradation. *Mol. Cell Biol.* **19**, 7216–7227.

Murakami, Y., Matsufuji, S., Kameji, T., Hayashi, S., Igarashi, K., Tamura, T., Tanaka, K., and Ichihara, A. (1992). Ornithine decarboxylase is degraded by the 26S proteasome without ubiquitination. *Nature* **360**, 597–599.

Ogura, T., and Tanaka, K. (2003). Dissecting various ATP-dependent steps involved in proteasomal degradation. *Mol. Cell* **11**, 3–5.

Pickart, C. M. (2001). Ubiquitin enters the new millennium. *Mol. Cell* **8**, 499–504.

Saeki, Y., Isono, E., and Toh-e, A. (2005). Preparation of ubiquitinated substrates by the PY motif-insertion method for monitoring 26S proteasome activity. *Methods Enzymol.* **399**, 215–227.

Tamura, T., Tanaka, K., Tanahashi, N., and Ichihara, A. (1991). Improved method for preparation of ubiquitin-ligated lysozyme as substrate of ATP-dependent proteolysis. *FEBS Lett.* **292**, 154–158.

Tanaka, K., Kumatori, A., Ii, K., and Ichihara, A. (1989). Direct evidence for nuclear and cytoplasmic colocalization of proteasomes (multiprotease complexes) in liver. *J. Cell Physiol.* **139**, 34–41.

Tanaka, K., Yashiroda, H., and Murata, S. (2005). Ubiquitin and diversity of the proteasome system. *In* “Protein Degradation” (R. J. Mayer, A. Ciechanover, and M. Rechsteiner, eds.). Wiley-VCH Verlag, Weinheim (in press).

Ugai, S., Tamura, T., Tanahashi, N., Takai, S., Komi, N., Chung, C. H., Tanaka, K., and Ichihara, A. (1993). Purification and characterization of the 26S proteasome complex catalyzing ATP-dependent breakdown of ubiquitin-ligated proteins from rat liver. *J. Biochem.* (Tokyo) **113**, 754–768.

Unno, M., Mizushima, T., Morimoto, Y., Tomisugi, Y., Tanaka, K., Yasuoka, N., and Tsukihara, T. (2002). The structure of the mammalian 20S proteasome at 2.75 Å resolution. *Structure* (Camb). **10**, 609–618.

Verma, R., and Deshaies, R. J. (2005). Assaying degradation and deubiquitination of a ubiquitinated substrate by purified 26S proteasomes. *Methods Enzymol.* **398**, 391–399.

Verma, R., McDonald, H., Yates, J. R., 3rd, and Deshaies, R. J. (2001). Selective degradation of ubiquitinated Sic1 by purified 26S proteasome yields active S phase cyclin-Cdk. *Mol. Cell* **8**, 439–448.
