
The Plant Journal (2006) 45, 133–143

doi: 10.1111/j.1365-313X.2005.02625.x

# Localization of a flavonoid biosynthetic polyphenol oxidase in vacuoles

Eiichiro Ono${}^{1}$, Masayoshi Hatayama${}^{2}$, Yuri Isono${}^{2}$, Takuya Sato${}^{2}$, Ryoko Watanabe${}^{2}$, Keiko Yonekura-Sakakibara${}^{1,\dagger}$, Masako Fukuchi-Mizutani${}^{1}$, Yoshikazu Tanaka${}^{1}$, Takaaki Kusumi${}^{1}$, Tokuzo Nishino${}^{2}$ and Toru Nakayama${}^{2,*}$

${}^{1}$Suntory Research Center, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan, and  
${}^{2}$Graduate School of Engineering, Tohoku University, Aramaki, Aoba 6-6-11-504, Sendai, Miyagi 980-8579, Japan

Received 26 July 2005; revised 27 September 2005; accepted 4 October 2005.  
*For correspondence (fax +81 22 795 7270; e-mail nakayama@seika.che.tohoku.ac.jp).  
$\dagger$Present address: Plant Science Center, Institute of Physical and Chemical Research, Tsurumi-ku, Yokohama-shi, Kanagawa 230-0045, Japan.

---

## Summary

Aureusidin synthase, a polyphenol oxidase (PPO), specifically catalyzes the oxidative formation of aurones from chalcones, which are plant flavonoids, and is responsible for the yellow coloration of snapdragon (*Antirrhinum majus*) flowers. All known PPOs have been found to be localized in plastids, whereas flavonoid biosynthesis is thought to take place in the cytoplasm [or on the cytoplasmic surface of the endoplasmic reticulum (ER)]. However, the primary structural characteristics of aureusidin synthase and some of its molecular properties argue against localization of the enzyme in plastids and the cytoplasm. In this study, the subcellular localization of the enzyme in petal cells of the yellow snapdragon was investigated. Sucrose-density gradient and differential centrifugation analyses suggested that the enzyme (the 39-kDa mature form) is not located in plastids or on the ER. Transient assays using a green fluorescent protein (GFP) chimera fused with the putative propeptide of the PPO precursor suggested that the enzyme was localized within the vacuole lumen. We also found that the necessary information for vacuolar targeting of the PPO was encoded within the 53-residue N-terminal sequence (NTPP), but not in the C-terminal sequence of the precursor. NTPP-mediated ER-to-Golgi trafficking to vacuoles was confirmed by means of the co-expression of an NTPP-GFP chimera with a dominant negative mutant of the Arabidopsis GTPase Sar1 or with a monomeric red fluorescent protein (mRFP)-fused Golgi marker (an H${}^{+}$-translocating inorganic pyrophosphatase of Arabidopsis). We identified a sequence-specific vacuolar sorting determinant in the NTPP of the precursor. We have demonstrated the biosynthesis of a flavonoid skeleton in vacuoles. The findings of this metabolic compartmentation may provide a strategy for overcoming the biochemical instability of the precursor chalcones in the cytoplasm, thus leading to the efficient accumulation of aurones in the flower.

Keywords: polyphenol oxidase, vacuole, biosynthesis of flavonoids, subcellular localization, flower coloration, aurone.

---

## Introduction

The yellow coloration of snapdragon (*Antirrhinum majus*) flowers is mainly caused by the presence of aurones, a class of plant flavonoids (Nakayama, 2002). Aurones (6-O-glucosides of aureusidin and bracteatin; Figure 1) are present in the vacuoles of petal cells along with small amounts of 4′-O-glucosides of chalcones [i.e. 2′,4′,6′-tetrahydroxychalcone (THC) and 2′,3,4′,6′-peptahydroxychalcone (PHC); Figure 1] (Sato *et al.*, 2001). Previously, we identified the gene encoding aureusidin synthase (AmAS1; EC 1.21.3.6) which is responsible for the biosynthesis of aurones from chalcones in yellow snapdragon flowers (Nakayama *et al.*, 2000). The aureusidin synthase gene represents a promising tool for creating, using genetic engineering approaches, novel yellow flowers in ornamental plants, yellow varieties of which are missing (e.g. cyclamen (*Cyclamen persicum*), geranium (*Pelargonium graveolens*), saintpaulia (*Saintpaulia ionantha*) and morning glory (*Ipomoea nil*); Tanaka *et al.*, 2005). This 39-kDa enzyme is a binuclear copper-requiring enzyme with sugar chain(s). Analyses of the primary structure and substrate specificities of this enzyme revealed that it is a chalcone-specific homolog of plant polyphenol oxidase (PPO; Nakayama *et al.*, 2000, 2001). PPOs, which occur

134 Eiichiro Ono et al.

![Chalcones](https://i.imgur.com/1234567.png)
**Chalcones**
- R₁=H, R₂=H: THC
- R₁=H, R₂=Glc: THC 4′-O-glucoside
- R₁=OH, R₂=H: PHC
- R₁=OH, R₂=Glc: PHC 4′-O-glucoside

![Aurones](https://i.imgur.com/ABCDEFG.png)
**Aurones**
- R₁=H, R₂=H: Aureusidin
- R₁=H, R₂=Glc: Aureusidin 6-O-glucoside
- R₁=OH, R₂=H: Bracteatin
- R₁=OH, R₂=Glc: Bracteatin 6-O-glucoside

![Flavanones](https://i.imgur.com/HJKLMNO.png)
**Flavanones**
- R₁=H: Naringenin
- R₁=OH: Eriodictyol

Figure 1. Structures of chalcones, aurones, and flavanones. Key positional numberings are also shown on the structures. THC, 2′,4,4′,6′-tetrahydroxychalcone; PHC, 2′,3,4,4′,6′-peptahydroxychalcone; Glc, β-D-glucopyranosyl. The glucosylated forms of chalcones and aurones are present in the vacuoles (Nakayama, 2002). The conversion of chalcones to aurones is catalyzed by aureusidin synthase (Nakayama et al., 2000). For details of the aureusidin synthase reaction, see Figure 8.

nearly ubiquitously in higher plants, generally catalyze the oxidation of monophenols and o-diphenols to o-quinones. Although plant PPO is known to be responsible for the browning of plant tissues that are exposed to air, the physiological functions of PPO in plants are not completely understood (Strack and Schliemann, 2001; Vaughn et al., 1988). PPOs have frequently been proposed to be involved in plant defense against pests and pathogens (Li and Steffens, 2002). Some biosynthetic roles of PPOs have also recently been identified, and aureusidin synthase represents one such unequivocal example (Strack and Schliemann, 2001). Other biosynthetic PPOs include the larreatricin hydroxylase of the creosote bush (*Larrea tridentata*), which is involved in the biosynthesis of nordihydroguaiaretic acid (Cho et al., 2003) and the tyrosine hydroxylase of *Portulaca grandiflora*, which is involved in betacyanin biosynthesis (Strack and Schliemann, 2001).

Flavonoid biosynthesis has been proposed to occur generally in the cytoplasm or on the cytoplasmic surface of the endoplasmic reticulum (ER; Burbulis and Winkel-Shirley, 1999). However, plant PPOs have been shown to be exclusively localized in plastids (or chloroplasts; Vaughn et al., 1988). The primary structures of plant PPO precursors generally consist of an N-terminal plastid transit peptide of approximately 60–80 amino acid residues, the mature domain, and a C-terminal propeptide of approximately 150 amino acid residues. N-terminal plastid transit peptides generally contain very few negatively charged amino acids and are highly enriched in hydroxylated amino acids. Moreover, many thylakoid proteins, including PPOs, have composite targeting signals with a typical plastid transit peptide followed by a thylakoid transfer domain (Figure 2; Emanuelsson and von Heijne, 2001; Joy et al., 1995). AmAS1 is also encoded as a precursor protein of 562 amino acids with N-terminal and C-terminal propeptides of approximately 60 and 150 amino acid residues, respectively, similar to other plant PPO precursors. However, the deduced amino acid sequence of the N-terminal propeptide (NTPP) of AmAS1 does not share the known features of the plastid transit sequences of the PPOs described above (Figure 2). Moreover, several other observations, such as the low pH optimum (5.4) for catalytic activity and the occurrence of sugar chain(s) in the AmAS1 molecule, also argue against the subcellular localization of AmAS1 in plastids (Nakayama et al., 2000). Thus, the subcellular localization of AmAS1 is an important issue to be clarified, given the unique biological role of this PPO homolog in the flower.

The findings herein establish that AmAS1 is localized in the lumen of the vacuole. This paper presents evidence for the biosynthesis of a flavonoid skeleton in the vacuole lumen, providing important information relevant to the use of the AmAS1 gene for metabolic engineering to create a novel yellow flower.

### Results

#### AmAS1 is not localized in plastids or associated with the ER

The subcellular localization of AmAS1 in the petal cells of the yellow snapdragon was examined by sucrose-density gradient centrifugation analysis. The organelles were separated from each other on the basis of their isopycnic densities and the correlation between the distribution of AmAS1 and those of organelle markers was examined by enzyme assay and Western blot analysis (Figure 3). AmAS1 activity (Figure 3b) and the 39-kDa mature protein (Figure 3a) of AmAS1 were detected in the lowest isopycnic-density fractions, with good correlation with a vacuole marker (flavonoids). Specifically, the mature AmAS1 and its enzyme activity were not correlated with markers of plastids (chlorophylls and carotenoids) and mitochondria (cytochrome c oxidase activity), indicating that AmAS1 (the mature form) was not localized in these organelles. In the present analysis, one large and one small peak, corresponding to nucleoside diphosphatase (a Golgi

Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), **45**, 133–143

AmAS1 M--FKNP--- -NIRYHKLSS ---KSND--- --NDQESSHR C---KHILLF I---I---TL 37
Grape MASL-PWSLT TSTAIANTTN ISAFPPSPLF QRASHVPVAR NRSRRFAPSK VSCNSANGDP 59
Pokeweed MASLSSPPTS TTTSATTLAT ---NISNPLF PKTSQFSLPG CRNRRHVMPK ISCKVDKDEL 57
Tomato MSSSSSITTT LPLCTNKLSL SSFTTTNSSL L-SKPSQLFL HGRRNQS-FK VSCN-ANNVD 57
Potato MASVCNSSST -TTTLKTPFI SSNTSLGST- --PKPSQLFL HGKRNKTFK VSCKVSN SGD 55

AmAS1 F--------- --------- -LL-IVGLYI ANSLAYAR-F ASTSTGPIAA PDVTKCGQP- 74
Grape NSDSTSDVRE TSSGKLDRRN VLLGTGGLYG AAGGLGATKP LAF-GAPIQA PDISKCGT-A 117
Pokeweed YPKTTN--DD GNTNKLDRRN MLIGLGSIYG A-GTTLGGGP A-TLAAPLIT PNPT-EFGAS 112
Tomato KN--------- --PDAVDRRN VLLGLGGLYG AA-NLAPLAT A----aPIPP PDLKSCGTAH 102
Potato QN-------QN IETNSVDRRN VLLGLGGLYG VA-NAIPLAA SA----THIPS PDLKT CGRAT 105

AmAS1 DLPPGTAP-- INCCPPIPAK IIDFELPPPS ----TTMRVR RAAHLVDDAY IAKFKKAVEL 128
Grape TVPDGVTP-- TNCCPPVTTK TIDFQLPSSG --SP--MTRR PAAHLVSKEY LAKYKKAIEL 171
Pokeweed DVG-------- LNCCPPTARN IVDFQLPPQP ----TTLRVR PPAHLVDDQY LQKFSRAISL 161
Tomato VKEGVDVIY- -SCCPPVPDD -IDS-VPYYK FPSMTKLRIR PPAHAADEEY VAKYQLATS R 158
Potato ISDGPLVPY- -TCCPPPMPPT NFDT-IPYYK FPSMTKLRIR PPAHAVDEEY IAKYNLAISR 162

Figure 2. Alignment of the N-terminal amino acid sequence of Antirrhinum majus aureusidin synthase (AmAS1; DDBJ AB044884) with those of plastid polyphenol oxidase (PPO) precursors [grape, PPO of Vitis vinifera, Genbank accession number S52629; Pokeweed, PPO of Phytolacca americana (PAP1), BAA08234; tomato, PPO of Lycopersicon esculentum, JQ1672; potato, PPO of Solanum tuberosum, AAA85122].

Hydroxylated amino acid residues in the 30-residue N-terminal portion are shown in blue. Amino acid residues identical to those of AmAS1 are shown in red. The N-terminal processing site identified in the PPO precursors is indicated by a vertical line in the sequence. Sequences representing the 'thylakoid transfer domain' (Joy et al., 1995) of plastid PPOs are shown using a yellow box. The underline below the AmAS1 sequence indicates the identification of a purified peptide derived from the purified AmAS1 (Nakayama et al., 2000). The putative sequence-specific vacuolar sorting determinant (Matsuoka and Neuhaus, 1999) of AmAS1 is double-underlined. It should be noted that the validity of the identification of the translation initiation codon (methionine 1) of AmAS1 cDNA was corroborated by the occurrence of an inframe TAA triplet immediately upstream of it (not shown).

marker) and NADPH/cytochrome c reductase (an ER marker) activities, were identified (Figure 3d). Although the activity and the 39-kDa mature protein of AmAS1 were seemingly correlated with large peaks of these organelle markers, the localization of the mature form of this PPO in these organelles is highly unlikely, judging from the results of differential centrifugation analyses and transient assays (see below). A Western blot analysis revealed the presence of an immunopositive protein of approximately 67 kDa in fractions 7–9 (Figure 3a), which appeared to correspond to small peaks of the marker activities of Golgi bodies and ERs. This observation is consistent with the suggested intracellular trafficking of the AmAS1 precursor (approximately 67 kDa) through these organelles in petal cells (see below).

To further confirm that AmAS1 activity (i.e. the 39-kDa mature form) is not associated with the ER, we carried out differential centrifugation analyses, where the distribution of activities of AmAS1 and NADPH-cytochrome c reductase (an ER marker) in fractions obtained by centrifugation were compared in the presence and absence of 1 mM MgCl₂ (Table S1). Inclusion of magnesium ions in the centrifugation buffer is known to prevent the dissociation of membrane-bound ribosomes, resulting in an increase in the isopycnic density of the microsomes with microsome-associated enzymes (Lord et al., 1973; Saunders et al., 1977). The results showed that most of the NADPH-cytochrome c reductase activities were recovered in the 5000 and 20 000 g pellets in the presence of MgCl₂, whereas approximately 1/3 of the total recovered NADPH-cytochrome c reductase activity was found in the 100 000 g pellet in the absence of MgCl₂. By contrast, the per cent activity of total recovered AmAS1 was approximately 95% in the 100 000 g supernatant, irrespective of the presence or absence of MgCl₂ (Table S1). These results suggest that AmAS1 activity is not associated with the ER.

Transient assays showed vacuolar localization of AmAS1

To further determine whether AmAS1 is localized in the cytoplasm or vacuoles and to examine the role of the putative NTPP and C-terminal propeptide (CTPP) of AmAS1 in the subcellular localization of AmAS1, we constructed expression vectors that encode green fluorescent protein (GFP) chimeras with the putative NTPP and CTPP of AmAS1 (Figure 4). The processing sites in the AmAS1 sequence remain to be unambiguously determined because of the possible modification of the N-terminus of the purified AmAS1. However, the identification of tryptic peptides derived from the purified enzyme revealed that the N- and C-termini of the mature form should lie upstream of Ile⁹² (Figure 2) and downstream of Trp⁴⁰⁰, respectively, in the 562-residue precursor (Nakayama et al., 2000). Thus, we

© 2005 The Authors
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), **45**, 133–143

136 Eiichiro Ono et al.

constructed a series of expression vectors encoding A92-GFP (a GFP chimera containing a 92-residue N-terminal sequence of AmAS1 at the N-terminus of GFP; Figure 4, entry 2), A60-GFP (entry 3), A53-GFP (entry 4), A47-GFP (entry 5), and A36-GFP (entry 6) as well as that encoding GFP-AC (a GFP chimera containing a 162-residue C-terminal sequence

of AmAS1 at the C-terminus of GFP; Figure 4, entry 7). These GFP chimeras were expressed in the epidermal cells of onion (*Allium cepa*) bulbs and the subcellular localizations of the expressed products were observed by means of confocal laser microscopy 48 h after bombardment. The petal cells of snapdragon were not used as the host in routine transient assays because (i) the bombardment assays often resulted in necrosis of snapdragon petal cells; (ii) the large sizes of onion bulb epidermal cells permitted easier microscopic observations; and (iii) it is generally accepted that intracellular sorting mechanisms are widely conserved among plant species (Baldwin *et al.*, 2001; Holm *et al.*, 2002). When A92-GFP, A60-GFP, and A53-GFP were expressed, the fluorescence signal was consistently observed in vacuoles (Figure 5a), indicating that the expressed GFPs were exclusively localized in the vacuole lumen. In contrast, when A47-GFP, A36-GFP, and GFP-AC were expressed, the fluorescence signals appeared to be localized in the cytoplasm and nuclei of the cells (Figure 5b,c). In the absence of the fused NTPP or CTPP (entry 1), the expressed GFP was also localized in the cytoplasm and nuclei of the cells (Figure 5d), indicating that the observed localization of A92-GFP, A60-GFP, and A53-GFP in the vacuole lumen did not arise from an autophagy process. When P109-GFP [a GFP chimera with an N-terminal 109-residue propeptide of pokeweed (*Phytolacca americana*) PPO (PAP1; a plastid enzyme; Joy *et al.*, 1995)] at the N-terminus of GFP, entry 8] was expressed as a control, the fluorescence signal was found to be localized in granular organelles (probably plastids) distributed in the cytoplasm (Figure 5e). These results, along with the results obtained from the centrifugal analyses, clearly show that, unlike known PPOs that are localized in plastids, AmAS1 is localized in the vacuole lumen, and the necessary information for the vacuolar targeting of this PPO homolog is encoded within the 53-residue N-terminal sequence, but not in the CTPP, of the precursor AmAS1 protein.

Similar transient assays were also carried out using cells from petals of the yellow snapdragon as host cells. In these

**Figure 3. Analysis of the correlation of *Antirrhinum majus* aureusidin synthase (AmAS1) with organelle markers by sucrose density gradient centrifugation.**

(a) Western blotting analysis of centrifugal fractions, probed with the primary anti-mature AmAS1 immunoglobulin G (IgG; rabbit) and the secondary anti-rabbit IgG-horseradish peroxidase conjugate. Molecular sizes are also indicated. Stains marked with an asterisk indicate experimental artifacts.

(b) Aureusidin synthase (AS) activity (open circles), the content of flavonoids (a vacuole marker; open triangles) and sucrose concentrations (weight/volume %, closed circles) in each fraction are plotted.

(c) The contents of plastid markers (carotenoids, closed squares; chlorophylls, open squares).

(d) Apparent activities of nucleoside diphosphatase (IDPase, a Golgi marker; open diamonds) and NADH/cytochrome c reductase (NADH/cyt.c red, an endoplasmic reticulum marker; closed diamonds).

(e) The apparent cytochrome c oxidase activity (cyt.c-ox, a mitochondrial marker; closed triangles). For (b) to (e), the highest values for the enzyme activity and the pigment contents are taken as 100%.

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), 45, 133–143

| Entry | Structure                                                                 | Designation    | Subcellular localization determined |
|-------|--------------------------------------------------------------------------|----------------|-------------------------------------|
| 1     | sGFP(S65T)                                                              | GFP            | C/N                               |
| 2     | NTPP                                                                     | A92-GFP        | V                                 |
| 3     |                                                                          | A60-GFP        | V                                 |
| 4     |                                                                          | A53-GFP        | V                                 |
| 5     |                                                                          | A47-GFP        | C/N                               |
| 6     |                                                                          | A36-GFP        | C/N                               |
| 7     | CTPP                                                                     | GFP-AC         | C/N                               |
| 8     | PAP                                                                      | P109-GFP       | P                                 |
| 9     | mRFP                                                                     | A60-mRFP       | V                                 |
| 10    |                                                                          | GFP-A60        | V                                 |

Figure 4. Structures of green fluorescent protein (GFP) chimeras. Pentagonal boxes represent the N-terminal sequence (NTPP) of Antirrhinum majus aureusidin synthase (AmAS1; yellow), sGFP(S65T; green), the C-terminal sequence (CTPP) of AmAS1 (gray), the NTPP of polyphenol oxidase (PPO) precursor of Phytolacca Americana (PAP1; purple), and monomeric red fluorescent protein (mRFP; red), each shown in an N-terminus (left) → C-terminus (right) orientation. The portion of NTPP and CTPP utilized is shown, above the boxes, by amino acid numbering of the corresponding PPO precursors (see Figure 2). The results of localization of the expressed fluorogenic reporter in these transient assays are also summarized: C/N, cytoplasm and nucleus; V, vacuoles; P, plastid-like granular organelles.

Figure 5. Signal for sorting of the cargo protein to the vacuole lumen is encoded within the 53-residue N-terminal sequence (NTPP) of Antirrhinum majus aureusidin synthase (AmAS1). At 48 h after the particle-gun bombardment of plasmid encoding the following protein on epidermal cells of onion, transformed cells were observed by confocal-laser fluorescence microscopy: (a) A60-green fluorescent protein (GFP), (b) A47-GFP, (c) GFP-AC, (d) GFP, and (e) P109-GFP. (f) Shows staining of the vacuole with 2′,7′-bis(carboxyethyl)-5-carboxyfluorescein acetoxy-methyl ester (BCECF-AM). (g) Shows the co-expression of A60-monomeric red fluorescent protein (mRFP) and GFP in A. majus petal cells, and (h) shows the co-expression of A60-mRFP and GFP in the onion epidermal petal cells. (i) Shows a schematic representation of epidermal cells of onion. Bars, 100 μm for (a)-(f) and (h), and 20 μm for (g).

cases, chimeras of the monomeric red fluorescent protein (mRFP; Tamura et al., 2005; e.g. Figure 4, entry 9) were used because the strong endogenous yellow-green fluorescence of aurones in the petal vacuoles hampered the fluorometric identification of GFP signals. The results were analogous to those obtained with onion epidermal cells (Figure 5g,h).

138 Eiichiro Ono et al.

(a) Expression of  
A60-GFP  

Expression of  
P109-GFP  

Figure 6. Evidence for endoplasmic reticulum (ER)-to-Golgi trafficking of *Antirrhinum majus* aureusidin synthase (AmAS1).  
(a) Co-expression of green fluorescent protein (GFP) chimeras with the wild-type or a dominant negative mutant of Arabidopsis Sar1 [(WT)AtSar1 and (DN)AtSar1, respectively]. The transformed cells were observed by confocal-laser fluorescence microscopy 48 h after bombardment. Panels show: (a) expression of A60-GFP; (b) co-expression of A60-GFP with (WT)ATSar1; (c, d) co-expression of A60-GFP with (DN)ATSar1; (e) expression of P109-GFP; (f) co-expression of P109-GFP with (WT)ATSar1; (g, h) co-expression of P109-GFP with (DN)ATSar1. Bars, 100 μm for (a)-(c) and (e)-(g), and 20 μm for (d) and (h).  
(b) Transient co-localization of GFP chimeras with AVP2-mRFP, which is a fusion of AVP2 (an H⁺-translocating inorganic pyrophosphatase of Arabidopsis, a marker protein for the Golgi body) and monomeric red fluorescent protein (mRFP). Onion epidermal cells were transformed with the plasmids encoding (a) A60-GFP (Figure 4, entry 3) and AVP2-mRFP or (b) those encoding GFP-A60 (Figure 4, entry 10) and AVP2-mRFP. The transformed cells were observed by confocal-laser fluorescence and differential interference contrast (DIC) microscopy at 24 h (upper panels) and 48 h (lower panels) after bombardment. Bars, 50 μm for all panels.

Evidence for sorting of AmAS1 to the vacuole via ER-to-Golgi trafficking

To determine whether vacuolar sorting mediated by the NTPP of AmAS1 passes through the Golgi body, A60-GFP was co-expressed with the wild-type or a dominant negative mutant of Arabidopsis Sar1 [termed (WT)AtSar1 and (DN)AtSar1, respectively; Takeuchi et al., 2000], and the subcellular localization of the fluorescence signal was determined. The AtSar1, a plant ortholog of yeast Sar1 GTPase, specifically functions in ER-to-Golgi trafficking by regulating the recruitment of coatmer protein (COP II) coat complexes onto the ER membranes and promoting vesicle formation from the ER. It has been shown that simultaneous transient expression with (DN)AtSar1 causes dominant negative inhibition of protein transport from the ER to the Golgi body, and this should provide a useful strategy for identifying of ER-to-Golgi trafficking during sorting to the vacuole (Takeuchi et al., 2000).

The co-expression of A60-GFP with (WT)AtSar1 resulted in the localization of the fluorescence signal in the vacuole lumen (Figure 6a, panel b), as was observed when A60-GFP was solely expressed (Figure 6a, panel a). By contrast, when A60-GFP was co-expressed with (DN)AtSar1, a network of fluorescence signals was exclusively localized in the cytoplasmic space (Figure 6a, panels c,d); this fluorescence-localization pattern was consistent with the localization of the cargo protein in the ER and was also observed when a GFP chimera with AtErd2 (an *Arabidopsis thaliana* HDEL receptor protein similar to yeast Erd2, which is an integral membrane protein localized in the cis region of the Golgi body; Takeuchi et al., 2000) was co-expressed with (DN)AtSar1 (data not shown). However, when P109-GFP (Figure 1, entry 8) was co-expressed with (DN)AtSar1, the fluorescence

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), 45, 133–143

Figure 6. (Continued).

(a) A60-GFP + AVP2-mRFP

(b) GFP-A60 + AVP2-mRFP

signals were localized in granular organelles distributed in the cytoplasm (Figure 6a, panels g,h), and this was also the case when this GFP chimera was co-expressed with (WT)AtSar1 (Figure 6a, panel f). Thus, these results suggest that AmAS1 was transported from the ER, via the Golgi body, to the vacuole, as is the case for most vacuolar proteins in plants.

The ER-to-Golgi trafficking governed by the NTPP of AmAS1 was further examined by co-transformation of A60-GFP and AVP2-mRFP, a fusion of AVP2 (a H⁺-translocating inorganic pyrophosphatase of Arabidopsis, a marker protein for the Golgi body; Mitsuda *et al.*, 2001) and mRFP (Figure 6b). At 24 h after the co-transformation of these constructs, GFP signals were identified in the Golgi body, where mRFP is specifically localized [Figure 6b(a), upper panels], whereas they were exclusively identified in the vacuole lumen 48 h after the co-transformation [Figure 6b(a), lower panels]. These results are also consistent with the ER-to-Golgi trafficking of AmAS1.

### Identification of a sequence-specific vacuolar sorting determinant (ssVSD) of AmAS1

The above findings show that the expression of A53-GFP resulted in the vacuolar localization of the expressed product, whereas that of A47-GFP did not. This suggests that vacuolar sorting information should, at least in part, be localized between Asn⁴⁸ and Ala⁵³ in AmAS1. Indeed, the sequence -Asn⁴⁸-Ser-Leu-Ala-Tyr⁵²- (NSLAY) of AmAS1 appears to satisfy the characteristics of an N-terminal ssVSD in plants, which is referred to as the NPIXL motif: [preferably Asn]-[not acidic]-[Ile or Leu]-[any amino acid]-[large hydrophobic], where the specific nature of the amino acids at the first, third, and fifth positions of this motif has been shown to be of particular importance for its function (Matsuoka and Neuhaus, 1999). To examine the functional significance of this NSLAY sequence in the vacuolar sorting of AmAS1, the third and fifth positions of this sequence were replaced, respectively, by Phe and Ser, and the GFP chimeras with the mutated signal sequences A60(L50F)-GFP and A60(Y52S)-GFP were expressed for use in transient assays. The replacement amino acid residues (i.e., Phe and Ser) were selected on the basis of the previous observation that replacement with these residues had the most significant effects on the function of the NPIXL motif of the precursor of sweet potato (*Ipomoea batatas*) sporamin, a vacuolar protein (Matsuoka and Nakamura, 1999). The results showed that, for both of these mutants, the expressed product was mislocalized in the cytoplasm and nucleus (Figure 7b,c) — these single amino acid substitutions in the NSLAY sequence inhibited vacuolar sorting (the strong endogenous fluorescence from the intercellular space hampered examination of the possible localization of the fluorogenic reporter therein). It has been shown that the NPIXL motif of the precursor of sweet potato sporamin may also serve as a vacuolar sorting signal when it is located at the C-terminus

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), **45**, 133–143

biosynthetic pathway in petal cells of snapdragon flowers, as shown in Figure 8.

To date, flavonoid biosynthesis has been proposed to generally occur in the cytoplasm or on the cytoplasmic surface of ERs (Burbulis and Winkel-Shirley, 1999). In cells of yellow snapdragon flowers, the biosynthesis of chalcones (THC and PHC and their 4′-O-glucosides) should also take place in the cytoplasm (Figure 8; M. Hatayama and T. Nakayama, unpublished results). It is important to note, in this regard, that the cytoplasmic milieu (pH approximately 7) entails a very rapid enzymatic and non-enzymatic isomerization of chalcones to produce flavanones (Figure 1; Sato et al., 2001; see also Figure 8), which do not serve as substrates for AmAS1 (Nakayama et al., 2001). Thus, for the efficient accumulation of aurones in the flower, a mechanism that circumvents such non-productive pathways must be operative and the compartmentation of AmAS1 in vacuoles could represent one such strategy—in the vacuole, the isomerization of chalcones would be very slow because of its acidic milieu (pH 4–5) and the absence of chalcone isomerase (CHI). It is generally accepted that flavonoids are glycosylated before translocation from the cytoplasm to the vacuole. Thus, in petal cells of snapdragon flowers, chalcones, once produced, should be 4′-O-glucosylated in the cytoplasm and immediately incorporated into vacuoles. It is worth noting here that that the 4′-O-glucosylation of chalcone likely inhibits its CHI-catalyzed isomerization, because the crystal structure of alfalfa (*Medicago sativa*) CHI shows that the 7-hydroxy group of naringenin (corresponding to the 4′-hydroxyl group of chalcones; see Figure 1) is hydrogen bonded with Thr170 in the substrate pocket of CHI (Jez et al., 2000), and hence the 4′-O-glucosylation of chalcone should hamper the binding of the substrate to CHI. The chalcone 4′-O-glucosides subsequently would be oxidized in vacuoles by the action of AmAS1, resulting in the accumulation of aurone 6-O-glucosides therein (Figure 8). Therefore, the actual *in vivo* substrates for AmAS1 appear to be 4′-O-glucosides of chalcones, and not the unglycosylated forms. The 4′-O-glucosides of THC and PHC were reported to be very good substrates for the enzyme, showing 2-fold higher specific activities than the corresponding aglycons even in the absence of added H₂O₂, an activator of the enzyme (Nakayama et al., 2001; Sato et al., 2001). Thus, the present finding provides an example of the biosynthesis of a flavonoid skeleton not only in the vacuole lumen, but also after glycosylation. Moreover, the findings of this study may promote the successful use of the *AmAS1* gene to genetically engineer a novel yellow flower. Namely, the removal of chalcones from the cytoplasm and their incorporation into vacuoles require the activity of UDP-glucose:THC 4′-O-glucosyltransferase (C4′GT), which, however, has not been identified. Thus, to create a yellow flower by means of *AmAS1*, its co-expression with the *C4′GT* gene would be essential. Attempts to isolate *C4′GT* cDNA from yellow

Figure 7. Mutational analysis of a putative sequence-specific vacuolar sorting determinant (ssVSD) in the N-terminal sequence (NTPP) of *Antirrhinum majus* aureusidin synthase (AmAS1). At 48 h after particle-gun bombardment of plasmid encoding the following protein on epidermal cells of onion, transformed cells were observed by confocal-laser fluorescence microscopy: (a) A53-green fluorescent protein (GFP; Figure 4, entry 4), (b) A53(L50F)-GFP, (c) A53(Y52S)-GFP, and (d) GFP-A60 (Figure 4, entry 10). Bars, 50 μm for (a)–(d).

of a cargo protein (Matsuoka and Neuhaus, 1999). To further determine whether this is also the case for the NSLAY motif of AmAS1, GFP-A60 (a GFP chimera having the 60-residue N-terminal sequence of AmAS1 at the C-terminus of GFP; Figure 4, entry 10) was expressed, and the subcellular localization of fluorescence signal was monitored. The results showed that the fluorescence signal was localized in the vacuole lumen when GFP-A60 was expressed (Figure 7d). Moreover, the ER-to-Golgi trafficking of GFP-A60 during its translocation to the vacuole was confirmed [Figure 6b(b)]. Thus, the NSLAY sequence may function as an ssVSD when it is located either in the N- or C-terminus of a cargo protein, as was observed for the NPIXL motif of sweet potato sporamin and barley (*Hordeum vulgare*) aleurain (Matsuoka and Neuhaus, 1999). These results strongly suggest that the NSLAY sequence serves as an ssVSD, which is structurally and functionally related to the NPIXL motif.

Discussion

The findings herein clearly establish that AmAS1 is localized in the vacuole lumen of petal cells of the yellow snapdragon, providing evidence for a vacuolar PPO. The reported molecular and catalytic properties of this enzyme, such as its low optimum pH (5.4) for catalytic activity and the presence of a sugar chain(s) in the molecule, are all consistent with the vacuolar localization of this enzyme. Moreover, the enzyme precursor is transported via ER-to-Golgi trafficking, and the sorting signal is encoded within the 53-residue NTPP, where an ssVSD that is related to the NPIXL motif was identified. All of these findings corroborate the vacuolar localization of the enzyme. An interesting issue to be addressed in future studies is how this PPO has acquired its unique subcellular localization and biological role (flower coloration) during its evolution. On the basis of the results presented above, we propose the subcellular compartmentation of the aurone

Vacuolar polyphenol oxidase 141

![Diagram](attachment:diagram.png)

Figure 8. The proposed intracellular compartmentation of the aurone biosynthetic pathway in petal cells of snapdragon (*Antirrhinum majus*) flowers. Synthesis, glycosylation, and isomerization of chalcones may take place on the cytoplasmic surface of the endoplasmic reticulum (ER). We previously showed that aureusidin can be produced from either 2′,4′,6′-tetrahydroxychalcone (THC) or 2′,3,4,4′,6′-pentahydroxychalcone (PHC), whereas bracteatin arises solely from PHC. Moreover, the 4′-O-glucosides of these chalcones serve as very good substrates for enzymatic aurone synthesis (Nakayama et al., 2001; Sato et al., 2001). AS, aureusidin synthase; CHI, chalcone isomerase; C4′GT, chalcone 4′-O-glucosyltransferase; Glc, β-D-glucopyranosyl.

snapdragon flowers and the creation of novel yellow flowers by metabolic engineering on the basis of this strategy will be reported elsewhere.

### Experimental procedures

#### Plasmids and materials

A commercial variety of the ‘Yellow Butterfly’ snapdragon was purchased from a local market in Sendai, Japan and used throughout this study. The AmAS1 cDNA was obtained as described previously (Nakayama et al., 2000). The psGFP(S65T) plasmid (Niwa et al., 1999), *CaM35S-sGFP(S65T)-NOS3′*, was the generous gift of Dr Yasuo Niwa (Graduate School of Nutritional and Environmental Sciences, University of Shizuoka, Shizuoka, Japan). Isolation of the *PAP1* cDNA (Joy et al., 1995) and the construction of the AtSar1 plasmids [plasmid 1, *P35S-sporamin-GFP-Tnos-P35S-(WT)AtSar1-Tnos* in pUC18; plasmid 2, *P35S-sporamin-GFP-Tnos-P35S-(DN)AtSar1-Tnos* in pUC18] have been described elsewhere (Takeuchi et al., 2000). The construct AVP2-mRFP was generously provided by Dr Masa H. Sato (Faculty of Integrated Human Studies, Kyoto University, Kyoto, Japan; Mitsuda et al., 2001). BCECF-AM [2′,7′-bis-(carboxyethyl)-5-carboxyfluorescein acetoxymethyl ester] was obtained from Molecular Probes (Eugene, OR, USA).

#### Sucrose density gradient centrifugation

The petals of fresh yellow snapdragon flowers at stages 3–5 (Sato et al., 2001) were chopped into small pieces (approximately 1 mm × 1 mm) using a razor blade in ice-cold buffer A [150 mM Tricine-NaOH, pH 7.5, containing 1 mM EDTA, 1 mM MgCl₂, and 13% (weight/volume; w/v) sucrose] at a ratio of 1 g of fresh weight per 2 ml of buffer. The chopped material was further disrupted with a Tefron-glass homogenizer at 4°C for 10 min, followed by centrifugation at 500 g for several seconds. The supernatant was then filtered first through a 180-μm filter (Millipore, Billerica, MA, USA), then through a 100-μm filter, and the filtrate was then subjected to sucrose gradient density centrifugation.

Continuous sucrose gradients (9.0 ml) were prepared from 6% and 55% (weight/volume) sucrose stock solutions each containing 1 mM EDTA in ultracentrifuge tubes (14 × 89 mm; Beckman, Coulter, Fullerton, California, USA). The filtrate (0.5 ml) was layered onto the gradient and centrifuged at 119 000 g and 4°C for 4 h in a Beckman SW41 Ti rotor. After puncturing the bottoms of the tubes, 0.5-ml fractions of the gradients were collected and were assayed for AmAS1 activity and the following organelle markers: plastids, contents of chlorophylls and carotenoids (Hurkman et al., 1979); mitochondria, cytochrome *c* oxidase activity; Golgi body, nucleoside diphosphatase activity (Ray et al., 1969); ER, NADPH-cytochrome *c* reductase activity (Lord et al., 1973; Saunders et al., 1977); vacuoles, flavonoid contents. AmAS1 activity was assayed

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), **45**, 133–143

using PHC as a substrate by reversed-phase high-performance liquid chromatography (HPLC) as previously described (Sato *et al.*, 2001).

### Differential centrifugation

The petals of fresh yellow snapdragon flowers were chopped into small pieces using a razor blade in buffer B [150 mm Tricine-NaOH, pH 7.5, containing 0.5 M sucrose, 1 mM DTT, 1 mM MgCl₂, and 13% (w/v) sucrose] or buffer B containing 1 mM EDTA instead of MgCl₂ (Lord *et al.*, 1973; Saunders *et al.*, 1977). The chopped material was centrifuged at 500 g for several seconds. Supernatant was then filtered through a 180-μm filter, then a 100-μm filter, and the filtrate was then subjected to differential centrifugation at 5000 g for 10 min, 20 000 g for 30 min, and 100 000 g for 2 h.

### Sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE) and Western blotting

SDS-PAGE was performed as described by Laemmli (1970). The proteins on the gels were visualized by staining with Coomassie brilliant blue. For Western blotting analyses, proteins in the SDS-PAGE gel were transferred to an Immobilon-P membrane (Millipore), which was then blocked with the recommended dilution of a blocking reagent (Amersham Bioscience, Piscataway, New Jersey, USA) at room temperature for 2 h. The blots were probed with a 1:1000 dilution of primary anti-mature AmAS1 immunoglobulin G (IgG; rabbit) and a 1:5000 dilution of the secondary donkey anti-rabbit IgG-horseradish peroxidase conjugate. The immune complexes were visualized by means of a peroxidase-catalyzed chemiluminescence reaction using an ECL Western blotting kit (Amersham Bioscience) following the manufacturer’s recommendations.

### Construction of expression vectors

A series of expression vectors for GFP chimeric proteins (Figure 4, entries 2–6) where the NTPP (whose length varied) of AmAS1 was introduced into the N-terminus of the sGFP(S65T) molecule was produced as follows (pA92-GFP, for example, denotes a vector that expresses A92-GFP). An NTPP cDNA fragment was amplified from the template, pAmAS1 (Nakayama *et al.*, 2000), by PCR using the forward primer fsal1 (see Table S2) and the corresponding reverse primer (rnco92 for pA92-GFP, rnco60 for pA60-GFP, rnco53 for pA53-GFP, rnco47 for pA47-GFP, and rnco36 for pA36-GFP; Table S2), followed by digestion with the restriction enzymes SalI and Ncol. The resulting DNA fragment was ligated with SalI/Ncol-treated psGFP. DNA sequences were verified by sequencing, in both orientations, using an automated DNA sequencer, the CEQ2000 DNA analysis system (Beckman Coulter, Fullerton, CA, USA). The expression vector pP109-GFP (Figure 4, entry 8) was produced essentially as described above, except that the primers Npf1 and Npr1 (Table S2) and the template PAP1 (Joy *et al.*, 1995) were used for PCR amplification of the NTPP portion of PAP1 cDNA.

For construction of the vector, pGFP-AC, expressing the GFP-AC protein (Figure 4, entry 7), a CTPP cDNA fragment was amplified from pAmAS1 by PCR using the forward primers fbsrg1 and rgcrq1 (see Table S2), followed by digestion with the restriction enzyme BsrGI. The resulting DNA fragment was ligated with BsrGI-treated psGFP. It should be noted that the guanine at position 711 of psGFP(S65T) was replaced by cytosine to create a SacI site and to inactivate the restriction BsrGI site of psGFP(S65T), which also is present in the NTPP of AmAS1. The orientation of the inserted DNA was confirmed by sequencing. pGFP-A60 was constructed essentially as described above, using fnco1 and rnco1 as primers (Table S2).

For construction of the vector, pA60-mRFP, expressing A60-mRFP (entry 9), the 5′-terminal region of AmAS1 cDNA (AmAS1₁₋₆₀) was amplified by PCR using the primers SalI-AmAS1-FW and BglII-AmAS1-RV (Table S2). The mRFP1 subcloned in pRESETB (Invitrogen, Carlsbad, CA, USA; Campbell *et al.*, 2002) was amplified with the primers BamHI-mRFP-FW and NotI-mRFP-RV (Table S2). These two fragments were fused at the BglII site of the 3′-terminus of the amplified AmAS1₁₋₆₀ and at the BamHI site of the 5′-terminus of mRFP1, resulting in A60-mRFP. The A60-mRFP was inserted at the SalI and NotI sites in the sGFP/pUC vector (Figure 4, entry 1).

### Transient assays

Particle-gun bombardment of the plasmids in plant cells was carried out with the Helios Gene Gun System (Bio-Rad, Hercules, CA, USA) following the standard protocol provided by the supplier. Gold particles (diameter 1.0 μm), a microcarrier loading quantity of 0.5 mg per target and a DNA loading ratio of 50 μg DNA/25 mg of gold were used in the particle bombardment, and onion epidermal cells were bombarded at a pressure 120 psi per shot. After bombardment, the transformed onion epidermal cells were placed in a sealed Petri dish (90 mm in diameter) containing a small amount of water (approximately 1.0 ml) to prevent drying and incubated at 22°C for 48 h under conditions of continuous darkness (Noji *et al.*, 1998). Under light conditions, the expressed GFP chimeras rapidly underwent proteolytic degradation in the vacuole (Tamura *et al.*, 2003). Transformed cells were visualized with a fluorescent microscope (LSM510; Carl Zeiss, Jena, Germany) according to a previous report (Matsushima *et al.*, 2004).

### Co-expression GFP chimeric proteins with (WT)AtSar1 or (DN)AtSar1

For the construction of the vector co-expressing (WT)AtSar1 and A60-GFP (Figure 4, entry 3) or P109-GFP (Figure 4, entry 8), a region containing 35S promoter-sporamin-GFP in the AtSar1 plasmid 1 (see above) was replaced by 35S promoter-A60-GFP or 35S promoter-P109-GFP using HindIII and NotI sites, resulting in two co-expression vectors, A60-GFP with (WT)AtSar1, and P109-GFP with (WT)AtSar1, respectively. A co-expression vector with (DN)AtSar1 was constructed using the AtSar1 plasmid 2 in the same manner as above.

### Co-expression of GFP chimeric proteins with AVP2-mRFP

For co-expression of the A60-GFP chimeric protein (Figure 4, entry 3) and AVP2-mRFP, gold particles were simultaneously incubated with equal amounts of the two plasmids (pA60-GFP and AVP2-mRFP; see above) at a concentration of 2 μg plasmid mg⁻¹ of gold particles. The gold particles fixed with the two plasmids were used for particle bombardment using a Helios Gene Gun (Bio-Rad) as described (Noji *et al.*, 1998).

### Acknowledgements

We would like to thank Professor Ikuko Hara-Nishimura, Kyoto University, and Professor Akihiko Nakano, Tokyo University, for their critical reading of the manuscript. We are grateful to Dr Masayuki Nozue, Shinshu University, for fruitful discussions. We are also indebted to Professor Akihiko Nakano, Tokyo University,
Dr Masa H. Sato, Kyoto University, and Professor Yasuo Niwa, University of Shizuoka, for providing us with the AtSar1 plasmids, the AVP2-mRFP constructs, and the psGFP(S65T) plasmid, respectively. This work was supported in part by Grants-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology of Japan (to TN), and by grants from The Asahi Glass Foundation (to TN), Shorai Foundation (to TN), and Novartis Foundation (to TN).

## References

Baldwin, T.C., Handford, M.G., Yuseff, M.I., Orellana, A. and Dupree, P. (2001) Identification and characterization of GONST1, a Golgi-localized GDP-mannose transporter in Arabidopsis. Plant Cell, 13, 2283–2295.

Burbulis, I.E. and Winkel-Shirley, B. (1999) Interactions among enzymes of the Arabidopsis flavonoid biosynthetic pathway. Proc. Natl Acad. Sci. USA 96, 12929–12934.

Campbell, R.E., Tour, O., Palmer, A.E., Steinbach, P.A., Baird, G.S., Zacharias, D.A. and Tsien, R.Y. (2002) A monomeric red fluorescent protein. Proc. Natl Acad. Sci. USA 99, 7877–7882.

Cho, M.-H., Moinuddin, S.G.A., Hishiyama, S., Eichinger, D., Davin, L.B. and Lewis, N.G. (2003) (+)-Larreatricin hydroxylase, an enantio-specific polyphenol oxidase from the creosote bush (Larrea tridentate). Proc. Natl Acad. Sci. USA 100, 10643–10646.

Emanuelsson, O. and von Heijne, G. (2001) Prediction of organellar targeting signals. Biochim. Biophys. Acta 1541, 114–119.

Holm, M., Ma, L.G., Qu, L.J. and Deng, X.W. (2002) Two interacting bZIP proteins are direct targets of COP1-mediated control of light-dependent gene expression in Arabidopsis. Genes Dev. 16, 1247–1259.

Hurkman, W.J., Morré, D.J., Bracker, C.E. and Mollenhauer, H.H. (1979) Identification of etioplast membranes in fractions from soybean hypocotyls. Plant Physiol. 64, 398–403.

Jez, M.J., Bowman, M.E., Dixon, R.A. and Noel, J.P. (2000) Structure and mechanism of the evolutionarily unique plant enzyme chalcone isomerase. Nat. Struct. Biol. 7, 786–791.

Joy, R.W. IV, Sugiyama, M., Fukuda, H. and Komamine, A. (1995) Cloning and characterization of polyphenol oxidase cDNAs of Phytolacca americana. Plant Physiol. 107, 1083–1089.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature, 227, 680–685.

Li, L. and Steffens, J.C. (2002) Overexpression of polyphenol oxidase in transgenic tomato plants results in enhanced bacterial disease resistance. Planta, 215, 239–247.

Lord, J.M., Kagawa, T., Moore, T.S. and Beevers, H. (1973) Endoplasmic reticulum as the site of lecithin formation in castor bean endosperm. J. Cell Biol. 57, 659–667.

Matsuoka, K. and Nakamura, K. (1999) Large alkyl side-chains of isoleucine and leucine in the NPRL region constitute the core of the vacuolar sorting determinant of sporamin precursor. Plant Mol. Biol. 41, 825–835.

Matsuoka, K. and Neuhaus, J.M. (1999) Cis-elements of protein transport to the plant vacuoles. J. Exp. Bot. 50, 165–174.

Matsushima, R., Fukao, Y., Nishimura, M. and Hara-Nishimura, I. (2004) NAI gene encodes a basic-helix-loop-helix-type putative transcription factor that regulates the formation of an endoplasmic reticulum-derived structure, the ER body. Plant Cell, 16, 1536–1549.

Mitsuda, N., Enami, K., Nakata, M., Takeyasu, K., Masa, H. and Sato, M.H. (2001) Novel type Arabidopsis thaliana H⁺-PPase is localized to the Golgi apparatus. FEBS Lett. 488, 29–33.

Nakayama, T. (2002) Enzymology of aurone biosynthesis. J. Biosci. Bioengin. 94, 487–491.

Nakayama, T., Yonekura-Sakakibara, K., Sato, T. *et al.* (2000) Aureusidin synthase: a polyphenol oxidase homolog responsible for flower coloration. Science, 290, 1163–1166.

Nakayama, T., Sato, T., Fukui, Y., Yonekura-Sakakibara, K., Hayashi, H., Tanaka, Y., Kusumi, T. and Nishino, T. (2001) Specificity analysis and mechanism of aurone synthesis catalyzed by aureusidin synthase, a polyphenol oxidase homolog responsible for flower coloration. FEBS Lett. 499, 107–111.

Niwa, Y., Hirano, T., Yoshimoto, K., Shimizu, M. and Kobayashi, H. (1999) Non-invasive quantitative detection and applications of non-toxic, S65T-type green fluorescent protein in living plants. Plant J. 18, 455–463.

Noji, M., Inoue, K., Kimura, N., Gouda, A. and Saito, K. (1998) Isoform-dependent differences in feedback regulation and subcellular localization of serine acetyltransferase involved in cysteine biosynthesis from Arabidopsis thaliana. J Biol. Chem. 273, 32739–32745.

Ray, P.M., Shininger, T.L. and Ray, M.M. (1969) Isolation of β-glucan synthetase particles from plant cells and identification with Golgi membranes. Proc. Natl Acad. Sci. USA 64, 605–612.

Sato, T., Nakayama, T., Kikuchi, S., Fukui, Y., Yonekura-Sakakibara, K., Ueda, T., Nishino, T., Tanaka, Y. and Kusumi, T. (2001) Enzymatic formation of aurones in the extracts of yellow snapdragon flowers. Plant Sci. 160, 229–236.

Saunders, J.A., Conn, E.E., Lin, C.H. and Shimada, M. (1977) Localization of cinnamic acid 4-monooxygenase and the membrane-bound enzyme system for dhurrin biosynthesis in Sorghum seedlings. Plant Physiol. 60, 629–634.

Strack, D. and Schliemann, W. (2001) Bifunctional polyphenol oxidases: new functions in plant pigment biosynthesis. Angew. Chem. Int. Ed. Engl. 40, 3791–3794.

Takeuchi, M., Ueda, T., Sato, K., Abe, H., Nagata, T. and Nakano, A. (2000) A dominant negative mutant of Sar1 GTPase inhibits protein transport from the endoplasmic reticulum to the Golgi body in tobacco and Arabidopsis cultures cells. Plant J. 23, 517–525.

Tamura, K., Shimada, T., Ono, E., Tanaka, Y., Nagatani, A., Higashi, S., Watanabe, M., Nishimura, M. and Hara-Nishimura, I. (2003) Why green fluorescent fusion proteins have not been observed in the vacuoles of higher plants. Plant J. 35, 545–555.

Tamura, K., Shimada, T., Kondo, M., Nishimura, M. and Hara-Nishimura, I. (2005) KATAMARI1 is a novel Golgi membrane protein that is required for endomembrane organization in Arabidopsis. Plant Cell, 17, 1764–1776.

Tanaka, Y., Katsumoto, Y., Brugliera, F. and Mason, J. (2005) Genetic engineering in floriculture. Plant Cell Tissue Org. Cult. 80, 1–24.

Vaughn, K.C., Lax, A.R. and Duke, S.O. (1988) Polyphenol oxidase: the chloroplast enzyme with no established function. Physiol. Plant. 72, 659–665.

© 2005 The Authors  
Journal compilation © 2005 Blackwell Publishing Ltd, *The Plant Journal*, (2006), 45, 133–143