
Biosci. Biotechnol. Biochem., 70 (1), 10–21, 2006

**Review**

# The Physiological Role of the Ribulose Monophosphate Pathway in Bacteria and Archaea

Nobuo KATO,$^{1,\dagger}$ Hiroya YURIMOTO,$^{2}$ and Rudolf K. THAUER$^{1}$

$^{1}$Max-Planck-Institut für terrestrische Mikrobiologie, Karl-von-Frisch-Strasse, D-35043 Marburg, Germany  
$^{2}$Division of Applied Life Sciences, Graduate School of Agriculture, Kyoto University, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan

---

3-Hexulose-6-phosphate synthase (HPS) and 6-phospho-3-hexuloisomerase (PHI) are the key enzymes of the ribulose monophosphate pathway. This pathway, which was originally found in methylotrophic bacteria, is now recognized as a widespread prokaryotic pathway involved in formaldehyde fixation and detoxification. Recent progress, involving biochemical and genetic approaches in elucidating the physiological functions of HPS and PHI in methylotrophic as well as non-methylotrophic bacteria are described in this review. HPS and PHI orthologs are also found in a variety of archaeal strains. Some archaeal HPS orthologs are fused with other genes to form single ORF (e.g., the *hps-phi* gene of *Pyrococcus* spp. and the *faeB-hpsB* gene of *Methanosarcina* spp.). These fused gene products exhibit functions corresponding to the individual enzyme activities, and are more efficient than equivalent systems made up of discrete enzymes. Recently, a novel metabolic function for HPS and PHI has been proposed in which these enzymes catalyze the reverse reaction for the biosynthesis of pentose phosphate in some archaeal strains. Thus the enzyme system plays a different role in bacteria and archaea by catalyzing the forward and reverse reactions respectively.

Key words: ribulose monophosphate pathway; methylotrophic bacteria; formaldehyde fixation; orotidine 5′-monophosphate decarboxylase (OMPDC) superfamiliy; pentose phosphate synthesis

In the biosphere, CO₂ is fixed into organic compounds by autotrophic microorganisms as well as by green plants in aerobic or anaerobic environments. CO₂ is reduced to methane by methanogens under anaerobic conditions. Methylotrophs, which are microbes that can utilize C₁ compounds of a more reduced form than that of CO₂ as a sole source of energy and carbon, are situated between autotrophs and methanogens. Among the reduced C₁ compounds, methane is very abundant in the natural environment, and methanol and formaldehyde are also ubiquitous in nature in the form of methyl esters of pectin and of the methoxyl group of lignin. Thus C₁ compounds at several oxidation levels, from methane to CO₂, constitute the principal components of the carbon cycle in the biosphere. These C₁ compounds are utilized by the cooperative action of autotrophs, methanogens, and methylotrophs.

The reduced C₁ compounds are first oxidized to formaldehyde, which is situated at the branch point between further oxidation to CO₂ for energy formation and assimilation for biosynthesis. Therefore the fate of formaldehyde is central to understanding methylotrophic metabolism.¹⁾ Because formaldehyde is highly cytotoxic, methylotrophic microorganisms have adopted several metabolic strategies to deal with the intracellular accumulation of this compound.¹⁻²⁾ One such strategy is to trap formaldehyde with a cofactor, such as glutathione, mycothiol, tetrahydrofolate, or tetrahydro-methanopterin, before it is oxidized to CO₂. Alternatively, free formaldehyde can be trapped by sugar phosphates as the first reaction in the C₁ assimilation pathways, the xylulose monophosphate pathway for yeasts and the ribulose monophosphate (RuMP) pathway for methylotrophic bacteria.

3-Hexulose-6-phosphate synthase (HPS) and 6-phospho-3-hexuloisomerase (PHI) are key enzymes in the RuMP pathway, which is involved in formaldehyde fixation in many methylotrophic bacteria. HPS catalyzes the aldol condensation of formaldehyde with ribulose 5-phosphate. The subsequent product, D-arabino-3-hexulose 6-phosphate, is isomerized by PHI to form fructose 6-phosphate. The genes encoding these two enzymes were first cloned from an obligate methylotroph, *Methylomonas aminofaciens* 77a.³⁻⁴⁾ These genes have also been cloned from a facultative methylotroph, *Mycobacterium gastri* MB19,⁵⁾ and a thermotolerant methylotroph, *Bacillus brevis* S1.⁶⁾ Initially the RuMP

---

$^\dagger$ To whom correspondence should be addressed. Present address: Kyotogakuen University, Kameoka, Kyoto 621-8555, Japan; Tel: +81-771-29-2440; Fax: +81-771-29-3641; E-mail: nkato@kyotogakuen.ac.jp

Abbreviations: HPS, 3-hexulose-6-phosphate synthase; PHI, 6-phospho-3-hexuloisomerase; RuMP, ribulose monophosphate; RuBP, ribulose bisphosphate; OMPDC, orotidine 5′-monophosphate decarboxylase; KGPDC, 3-keto-L-gulonate 6-phosphate decarboxylase; RPE, d-ribulose 5-phosphate 3-epimerase; Fae, formaldehyde-activating enzyme; H₄MPT, tetrahydromethanopterin; PPP, pentose phosphate pathway; OPPP, oxidative PPP; NOPPP, non-oxidative PPP; FOR, formaldehyde ferredoxin oxidoreductase; XPE, L-xylulose 5-phosphate 3-epimerase

The Ribulose Monophosphate Pathway

Cell constituents

HC*HO
CH₂OH
CO
HCOH
HCOH
CH₂O-P
Ribulose 5-P

C*H₂OH
HOCH
CO
HCOH
HCOH
CH₂O-P
D-arabino-3-Hexulose 6-P

C*H₂OH
HOCH
CO
HCOH
HCOH
CH₂O-P
Fructose 6-P

ATP
ADP
C*H₂O-P
HOCH
CO
HCOH
HCOH
CH₂O-P
Fructose 1,6-bisP

C*H₂OH
CO
CH₂O-P
Dihydroxyacetone P
CHO
HCOH
CH₂O-P
Glyceraldehyde 3-P

Rearrangement reactions

Fig. 1. A Variant of the Ribulose Monophosphate Pathway.
The carbon from formaldehyde is asterisked. Enzymes: HPS, 3-hexulose-6-phosphate synthase; PHI, 6-phospho-3-hexuloisomerase.

pathway was thought to be confined to methylotrophs, ${}^{7)}$ but recent data show that the formaldehyde-fixing reaction is more widely distributed. ${}^{5,8)}$ As well as methylotrophic bacteria, genes homologous to HPS and PHI have been found in a variety of non-methylotrophic organisms, including bacteria and archaea.

These findings have raised issues concerning the physiological role of the RuMP pathway, which was thought to be restricted to the fixation of formaldehyde into cellular constituents in methylotrophic bacteria. Indeed, recent biochemical and genetic approaches have shown that the RuMP pathway functions in the detoxification of formaldehyde in some non-methylotrophic bacteria. ${}^{9,10)}$ Moreover, several reports suggest that HPS and PHI catalyze the reverse reactions and may be involved in pentose phosphate synthesis in some archaea whose pentose phosphate pathway is imperfect. ${}^{11-15)}$ In this review, we focus on the microbial and biochemical aspects of the RuMP pathway, and discuss the physiological functions of the pathway in bacteria and archaea.

I. The RuMP Pathway for Formaldehyde Fixation in Bacteria

Methylotrophic bacteria exhibit one of three assimilation pathways, the RuMP pathway, the serine pathway, or the ribulose bisphosphate (RuBP) pathway. ${}^{16)}$ The RuMP pathway functions as a highly efficient system for trapping free formaldehyde at relatively low concentrations. ${}^{7,17)}$ The pathway performs the net synthesis of a C₃ compound (pyruvate) from 3 moles of formaldehyde (Fig. 1). Formaldehyde is fixed with ribulose 5-phosphate to form D-arabino-3-hexulose 6-phosphate, which is then isomerized to fructose 6-phosphate. These two reactions are catalyzed by HPS and PHI respectively. The fructose 6-phosphate formed is then converted to fructose 1,6-bisphosphate, which is subsequently metabolized into pyruvate via the glycolytic pathway (cleavage stage). Ribulose 5-phosphate, the acceptor of formaldehyde, is regenerated from fructose 6-phosphate and glyceraldehyde 3-phosphate through a combination of transketolase, transaldolase, and several reactions for pentose phosphate isomerizations (regeneration stage). There are two alternative stages for the cleavage of fructose 6-phosphate to C₃ compounds and two stages for the regeneration of ribulose 5-phosphate. The combination of two cleavage and two regeneration stages thus leads to a total of four possible variants of the RuMP pathway. ${}^{18)}$ The variant of the RuMP pathway shown in Fig. 1 is actually exergonic on the basis of the following equation:

$$
3 \text{ HCHO } + \text{ ADP } + \text{ NAD}^+ \rightarrow \text{ Pyruvate } + \text{ ATP } + \text{ NADH } + \text{ H}^+
$$

The other variants are also predicted to be mainly exergonic, being much more efficient than the serine pathway or the RuBP pathway. ${}^{7,18)}$

II. Properties of HPS and PHI

HPS has been purified from several methylotrophic bacteria. Some salient properties of this enzyme are

Table 1. Properties of 3-Hexulose-6-phosphate Synthases from Several Methylotrophic Bacteria

| Source                     | Specific activity (U/mg) | $K_m$        | Molecular mass      | Activator       | Reference |
|----------------------------|--------------------------|--------------|---------------------|------------------|-----------|
|                            |                          | Ru5P (mM)    | holo-enzyme (kDa)   | subunit          |           |
| Methylococcus capsulatus   | 69                       | 0.083        | 310                 | 49               | Mg²⁺/Mn²⁺ | 17)      |
| Methylomonas M 15          | 66.5                     | 1.6          | 40–43               | 22               | Mg²⁺/Mn²⁺ | 20)      |
| Methylomonas aminofaciens  | 53                       | 0.059        | 45–47               | 23               | Mg²⁺/Mn²⁺ | 21)      |
| Methylophilus methylotrophus | 97.4                    | 0.136        | 40                  | 22.5             | Mg²⁺/Mn²⁺ | 22)      |
| Mycobacterium gastri       | 74.2                     | —            | 43                  | 24               | Mg²⁺/Mn²⁺ | 23)      |
| Bacillus C1                | 64                       | 0.45         | 32                  | 27               | Co²⁺/Ni²⁺ | 24)      |

given in Table 1. HPS catalyzes the Mg²⁺-dependent aldol condensation between formaldehyde and ribulose 5-phosphate to form D-arabino-3-hexulose 6-phosphate. In most cases, the $K_m$ values for formaldehyde fall within the range of concentrations for tolerance to bacterial growth (1 to 2 mM). The enzyme is usually homodimeric and in each case the subunit molecular mass (22 to 27 kDa) is close to the anticipated value based on the corresponding gene sequence, but HPS from *Methylococcus capsulatus* is a hexameric enzyme with an unusually large subunit size. Based on the genome sequence, the molecular mass appears to correspond to the product of the *hps-phi*-fused gene.

By comparison with HPS, the enzymology of PHI is rather poorly documented. This paucity of information arises because the substrate for PHI, D-arabino-3-hexulose 6-phosphate, is very unstable. Consequently, the activity of PHI can be determined only by coupling the reaction with HPS. The *phi* gene from *Methylomonas aminofaciens 77a* has been engineered for heterologous expression in *Escherichia coli*. The apparent subunit molecular mass (20 kDa) of the recombinant PHI corresponded to the anticipated value, and the enzyme was shown to exist as a homodimer. The purified enzyme exhibited PHI activity of 15,400 U·mg⁻¹ protein. Comparison of the relative activities of HPS and PHI indicates that the rate of the sequential reaction is limited by HPS. Recently, the crystal structure of a PHI enzyme (hypothetical protein MJ1247) from the methanogen *Methanocaldococcus jannaschii* was determined. The PHI protein has an α/β structure consisting of a five-stranded parallel β sheet flanked on both sides by α helices, forming a three-layered α-β sandwich.

III. Reaction Mechanism of Formaldehyde Fixation by HPS

HPS is a member of the orotidine 5′-monophosphate decarboxylase (OMPDC) superfamily. Members of the OMPDC superfamily, such as OMPDC, 3-keto-L-gulonate 6-phosphate decarboxylase (KGPDPC), D-ribulose 5-phosphate 3-epimerase (RPE), and HPS, share a conserved active site architecture although the mechanism of catalysis can vary significantly. X-ray structures

The Ribulose Monophosphate Pathway

- KGPDC


CO₂⁻
HO
H
=O
H
OH
HO
H
CH₂OPO₃²⁻


$\xrightarrow{\text{ }}$


H
OH
O⁻
Mg²⁺
H
O⁻
H
HO
H
CH₂OPO₃²⁻


$\xrightarrow{\text{ }}$


CH₂OH
=O
H
HO
HO
H
CH₂OPO₃²⁻


3-Keto-L-gulonate 6-P
L-Xylulose 5-P

- HPS


CH₂OH
=O
H
OH
H
OH
CH₂OPO₃²⁻


$\xrightarrow{\text{ }}$


H
OH
O⁻
Mg²⁺
H
O⁻
OH
H
CH₂OPO₃²⁻


$\xrightarrow{\text{ }}$


O
C
H
H⁺
CH₂OH
HO
=O
H
OH
H
OH
CH₂OPO₃²⁻


D-Ribulose 5-P
D-arabino-3-Hexulose 6-P

Fig. 2. Reaction Mechanisms for 3-Keto-L-gulonate 6-Phosphate Decarboxylase (KGPDC) and 3-Hexulose-6-phosphate Synthase (HPS).

A

- Methylomonas aminofaciens 77a
  (obligate methylotroph)


hps → IS10-R → phi


Down-regulation
Up-regulation

- Mycobacterium gastri MB19
  (facultative methylotroph)


rmpR → phi → hps


RuMP operon

- Bacillus brevis S1
  (thermotolerant methylotroph)


hps → phi


B

- Pyrococcus spp.
  (hyperthermophilic anaerobic archaea)


hps → phi


- Methanosarcina spp.
  (methanogenic archaea)


fae → hps → phi


Fig. 3. Genomic Arrangement of HPS and PHI from Methylotrophic Bacteria (A) and Archaea (B).

IV. Organization and Expression of Genes for HPS and PHI in Methylotrophic Bacteria

Methylotrophic bacteria possessing the RuMP pathway are found in three groups: Gram-negative obligate methylotrophs, Gram-positive facultative methylotrophs, and thermotolerant *Bacillus* species. We have cloned the genes for HPS and PHI from all three types of methylotrophic bacteria (Fig. 3A).³⁻⁶⁾ Although the primary amino acid sequence of HPS and PHI showed a high degree of similarity among these organisms, the gene organization and regulatory mechanism were quite different for the obligate methylotrophs. In the obligate methylotroph, *Methylomonas aminofaciens* 77a, a transposase (IS10-R), located between *hps* and *phi*, is involved in the regulation of the adjacent gene expression.⁴⁾ Both *hps* and *phi* are expressed monocistronically, and the inside promoters of the IS10-R gene down-regulate the expression of the *hps* gene and up-

regulate the *phi* gene, respectively. This suggests that the ratio of HPS to PHI is precisely controlled for formaldehyde fixation. This finely balanced system of regulation appears to be present in those organisms that are capable of growing only on C₁ compounds.

The *hps* and *phi* genes of facultative methylotrophs are transcribed as a polycistronic operon, and expression is induced by formaldehyde.⁵⁶) Induction by a key intermediate of methylotrophic metabolism is indispensable for facultative methylotrophs that grow on diverse carbon sources including C₁ compounds. In *Mycobacterium gastri* MB19, a putative GntR-family transcriptional regulator, *rmpR*, is located upstream of the operon (Fig. 3A), but the molecular mechanism of formaldehyde-induced gene expression in this methylotrophic bacterium is still unclear. Recently, a novel DNA-binding protein was found to be involved in formaldehyde-induced expression of a HPS- and PHI-encoding operon in the non-methylotrophic *Bacillus subtilis* (see below).²⁷)

### V. Detoxification of Formaldehyde by the RuMP Pathway in Non-Methylotrophic Bacteria

Genome sequencing projects of many kinds of bacterial strains have revealed that some non-methylotrophic bacteria possess a RuMP operon. In *Bacillus subtilis*, the RuMP pathway has been shown to function during growth on a formaldehyde-containing medium.⁹) The *hxlA* and *hxlB* genes, coding for HPS and PHI respectively, make up the *hxlAB* operon. The *hxlR* gene is located upstream of the *hxlAB* operon and is transcribed in the opposite direction.⁹,²⁷) Disruption of either the *hxlR* gene or the *hxlAB* operon results in a strain that is sensitive to formaldehyde, suggesting that the RuMP pathway acts as a detoxification system for this compound.⁹) Recently, the product of the *hxlR* gene, HxlR, has been shown to act as a transcriptional activator during formaldehyde-induced expression of the *hxlAB* operon by binding to specific sequences upstream of the *hxlAB* operon.²⁷) HxlR is now designated as a transcriptional activator of the “helix-turn-helix, HxlR type” (Fig. 4A, http://www.ebi.ac.uk/interpro/IEntry?ac=IPR002577). There are currently more than 270 members of this protein family, which belongs to the superfamily of winged helix DNA-binding proteins (IPR011991). Two 25 bp regions (BRH1 and BRH2) of DNA upstream of the *hxlAB* operon have been identified as the binding site for HxlR.²⁷) Surface plasmon resonance analyses indicate that two HxlR dimers bind to BRH1 and BRH2 in two steps (Fig. 4B). However, the mechanism by which formaldehyde modulates expression of the *hxlAB* operon remains unclear.

In the course of vanillin degradation by *Burkholderia cepacia* TM1, formaldehyde is released from the methoxyl group of vanillic acid by oxidative demethylation (Fig. 5). Engineering the *hps* and *phi* genes from *Methylomonas aminofaciens* 77a for heterologous expression in *Burkholderia cepacia* significantly improved the rate of vanillin degradation.¹⁰) Furthermore, incorporation of ¹⁴C-labelled formaldehyde into the cell constituents increased after the introduction of the *hps* and *phi* genes into *Burkholderia cepacia* TM1. These results indicate that formaldehyde utilization is rate-limiting during the degradation of vanillic acid. In the engineered strain, the RuMP pathway plays a significant role in the detoxification and assimilation of formaldehyde.

### VI. HPS and PHI of Archaeal Strains

Genome sequence analysis of some archaeal strains has revealed the presence of HPS and PHI homologs, although the gene organization is quite different from that in bacteria (Fig. 3B). In a hyperthermophilic anaerobic archaeon, *Pyrococcus horikoshii*, *hps* and *phi* are encoded in a single open reading frame. In the methanogens, *Methanosarcina barkeri* and *Methanosarcina mazei*, a *hps*-like sequence is fused with the sequence for formaldehyde-activating enzyme (Fae). In these organisms, a gene homologous to *phi* is present at a different locus from these *hps* sequences. Each gene has been heterologously expressed in *E. coli*, and in every case the purified recombinant protein was found to exhibit the corresponding activity.¹⁴,¹⁵)

The deduced amino acid sequence of the PH1938 gene from *Pyrococcus horikoshii* OT3 revealed that the N- and C-terminal halves showed significant similarity to HPS and PHI respectively. Strain OT3 constitutively produced the HPS-PHI fused protein, which exhibited the activity of the sequential HPS- and PHI-catalyzed reactions. This was the first example of an archaeal strain displaying HPS and PHI activities.¹⁴) Unlike most of the bacterial HPS and PHI enzymes, which were soluble and formaldehyde-inducible, the hybrid enzyme from *Pyrococcus horikoshii* OT3 was found primarily in the particulate fraction and the expression level was unaffected by the addition of formaldehyde to the culture medium. The full-length gene encoding the hybrid enzyme and truncated versions corresponding to the HPS and PHI regions were engineered for expression in *E. coli* to give active recombinant enzymes, rHps-Phi, rHps, and rPhi respectively. Purified rHps-Phi and rHps were found to be stable at 90°C, whereas rPhi was highly susceptible to heat. This surprising result suggests that the thermostability of the PHI domain of the hybrid enzyme results from fusion with HPS. The activity of the fused protein at 80°C was about 50-fold greater than a mixture of rHps and rPhi on an equimolar subunit basis. Fusion of the two enzymes to form a hybrid presumably increases the efficiency of the net reaction because of a localized increase in the concentration of the product (D-arabino-3-hexulose 6-phosphate) of the HPS-catalyzed reaction within the microenvironment

The Ribulose Monophosphate Pathway

A
20aa

| HXLR_BACSU (P42406) |  |
| YYBR_BACSU (P37486) |  |
| YDEP_BACSU (P96673) |  |
| MALR_FUSMR (Q93K57) |  |
| YC85_METTH (O27346) |  |
| YTFH_ECOLI (P39316) |  |
| YDZF_BACSU (O31494) |  |
| YKVN_BACSU (O31679) |  |
| YODB_BACSU (O34844) |  |
| YTCD_BACSU (O34533) |  |
| Y655_BACHD (Q9Z9W1) |  |
| YF12_SYNY3 (P72979) |  |
| YU95_MYCTU (O05774) |  |
| Y925_METJA (Q58335) |  |

B


     BRH1          BRH2
      ↓             ↓
    hxlR ---------- hxlA
         \_________/
               HxIR


Fig. 4. Predicted Domain Structure of HxIR and Model for Binding to the hxlAB Promoter.

A, Predicted domain structure of HxIR and similar proteins from the Interpro server (http://www.ebi.ac.uk/interpro/). Winged helix DNA-binding domain (InterPro no. IPR009058) and Helix-turn-helix, HxIR type (InterPro no. IPR002577) are indicated. Accession numbers to the protein database are shown in parentheses. HXLR_BACSU, *Bacillus subtilis* HxIR; YYBR_BACSU, *B. subtilis* hypothetical protein YybR; YDEP_BACSU, *Bacillus subtilis* hypothetical protein YdeP; MALR_FUSMR, *Fusobacterium mortiferum* putative regulatory protein MalR; YC85_METTH, *Methanothermobacter thermautotrophicus* hypothetical protein MTH1285; YTFH_ECOLI, *E. coli* hypothetical protein YtfH; YDZF_BACSU, *Bacillus subtilis* hypothetical protein YdzF; YKVN_BACSU, *Bacillus subtilis* hypothetical protein YkvN; YODB_BACSU, *Bacillus subtilis* hypothetical protein YodB; YTCD_BACSU, *Bacillus subtilis* hypothetical protein YtcD; Y655_BACHD, *Bacillus halodurans* hypothetical protein BH0655; YF12_SYNY3, *Synechocystis sp.* hypothetical protein Sll1512; YU95_MYCTU, *Mycobacterium tuberculosis* hypothetical protein Rv3095; Y925_METJA, *Methanocaldococcus jannaschii* hypothetical protein MJ0925. B, Model for binding of HxIR to the hxlAB promoter. First a dimeric form of HxIR binds to BRH1. When the amount of HxIR protein reaches a threshold level, the second dimeric form of HxIR binds to BRH1+BRH2.

![Chemical Reaction Diagram](attachment:chemical_reaction_diagram.png)

Fig. 5. Degradation of Vanillin by *Burkholderia cepacia* TM1.

Formaldehyde is incorporated into cell constituents through the RuMP pathway.

around the PHI component of the hybrid enzyme. This fused enzyme system is particularly advantageous to the hyperthermophile, because the product of the first enzyme, D-arabino-3-hexulose 6-phosphate, is highly unstable at elevated temperatures.²⁸

The natural fusion of HPS and PHI increases the efficiency of the two sequential reactions and the stability of the individual enzymes against thermal

Fig. 6. Pentose Phosphate Synthesis through HPS and PHI in an Archaeon, Which Is Imperfect in the Pentose Phosphate Pathway.

Pathways: PPP, pentose phosphate pathway; OPPP, oxidative PPP; NOPPP, non-oxidative PPP. Intermediates: G6P, glucose 6-phosphate; 6PGL, 6-phosphoglucono-δ-lactone; 6PG, 6-phosphogluconate; Ru5P, ribulose 5-phosphate; Xu5P, xylulose 5-phosphate; Ri5P, ribose 5-phosphate; S7P, sedoheptulose 7-phosphate; GAP, glyceraldehyde 3-phosphate; E4P, erythrose 4-phosphate; F6P, fructose 6-phosphate. Enzymes: G6PDH, glucose-6-phosphate dehydrogenase; 6PGL, 6-phosphoglucono-δ-lactonase; 6PGDH, 6-phosphogluconate dehydrogenase; Ru5PE, ribulose-5-phosphate 3-epimerase; RPI, ribose-5-phosphate isomerase; TK, transketolase; TA, transaldolase.

denaturation in *Pyrococcus horikoshii* OT3. This suggests that it might be possible artificially to generate active HPS-PHI enzymes by engineering the *hps* and *phi* genes from some methylotrophic bacteria. From the viewpoint of biotechnology, such artificial gene fusions are attractive because they simplify the problem of regulating gene expression, can increase the catalytic efficiency of the sequential reaction, and permit enzyme purification in a single process. The existence of natural fused systems also raises some interesting questions concerning the evolution of methylotrophy.

The methanogenic archaea *Methanosarcina barkeri* and *Methanosarcina mazei* have a fused gene (*faeB-hpsB*) consisting of Fae and HPS. Fae catalyzes the condensation of formaldehyde and tetrahydromethanopterin (H₄MPT) to methylene-H₄MPT, and is involved in the oxidation of formaldehyde to CO₂ and in formaldehyde detoxification. The *faeB-hpsB* genes of *Methanosarcina barkeri* and *Methanosarcina mazei* were engineered for expression in *E. coli*, and the purified recombinant enzymes exhibited Fae and HPS activities. Interestingly, HPS activity of the fused enzyme was not stimulated by the substrate of Fae, H₄MPT. The physiological role of the fused enzyme is discussed below.

### VII. Physiological Role of HPS and PHI in Archaeal Strains

The activity of formaldehyde fixation in *Pyrococcus horikoshii* OT3 and the kinetic parameters of rHps-Phi are comparable with those of HPS from the thermotolerant methylotroph, *Bacillus* strain C1. Although the presence of formaldehyde has not been directly determined in a marine hydrothermal vent, from which

Pyrococcus horikoshii OT3 was isolated, its presence would not be surprising. C₁ compounds are found at a variety of redox levels in marine hydrothermal vents (e.g., methane, CO, and CO₂). If the original habitat of the archaeon is rich in formaldehyde, an enzyme system capable of fixing this toxic compound is essential for survival of the organism.

Several recent papers suggest that the reverse “RuMP pathway” (i.e., sequential reactions by HPS and PHI, but not the whole pathway) might be employed for pentose phosphate synthesis in some archaea, which lack the enzymes of the pentose phosphate pathway. In bacteria and eukarya, pentose phosphates are provided through the pentose phosphate pathway (PPP), but the biosynthesis of pentose phosphates of some archaeal strains is poorly understood. The PPP consists of an oxidative (OPPP) and a non-oxidative branch (NOPPP) (Fig. 6). Ribulose 5-phosphate, which is the original source of the pentose moiety of nucleotides, is generated from glucose 6-phosphate through the PPP, involving the reduction of two moles of NADP. In the NOPPP, ribulose 5-phosphate is converted to ribose 5-phosphate, a component of nucleic acid, and erythrose 4-phosphate, a biosynthetic precursor of aromatic amino acids. Based on phylogenetic analysis of the genes encoding enzymes of the pentose phosphate pathway and “the RuMP pathway” from 13 complete archaeal genomes, Soderberg proposed that the “reverse RuMP pathway” might be employed for pentose phosphate synthesis in archaea lacking a NOPPP. In the genome of *Pyrococcus horikoshii*, genes for OPPP (NAD(P)-dependent glucose 6-phosphate dehydrogenase and NAD(P)-dependent 6-phosphogluconate) and for NOPPP (transketolase, transaldolase, and ribulose-5-phosphate epimerase) are missing. As described above, it is evident that *Pyrococcus*

horikoshii produces HPS-PHI protein even in the absence of formaldehyde, suggesting that this bifunctional enzyme is involved in the constitutive metabolic activity. Because HPS catalyzes the reverse reaction, ${ }^{17)}$ HPS-PHI of *Pyrococcus horikoshii* OT3 might mediate the formation of ribulose 5-phosphate and formaldehyde from fructose 6-phosphate. Indeed, the specific activity of the purified rHps-Phi was 1/7 of the forward reaction. ${ }^{14)}$ This value appears to be comparable to that of enzymes participating in constitutive metabolic activities. Combined with the biochemical results for rHps-Phi and the phylogenetic analysis of the PPP in *Pyrococcus horikoshii*, the evidence clearly indicates that the reverse reaction of HPS-PHI is involved in pentose phosphate synthesis.

The biosynthesis of ribulose 5-phosphate by the reverse reaction of HPS and PHI results in the concomitant production of formaldehyde, which must be detoxified. One reliable mechanism for formaldehyde detoxification is the system involving Fae (Fig. 7). The reaction catalyzed by Fae (condensation of formaldehyde and H${}_{4}$MPT to form methylene H${}_{4}$MPT) also proceeds spontaneously, although the rate of this spontaneous reaction is insufficient to keep up with the rate of C${}_{1}$ metabolism. ${ }^{29)}$ In methylotrophic bacteria, formaldehyde is oxidized (or detoxified) to CO${}_{2}$ via methylene H${}_{4}$PMT, N${}^{5}$-formylmethanofuran, and formate. ${ }^{2)}$ Fae has been found not only in methylotrophic bacteria, but also in methanogenic and sulfate-reducing archaea. ${ }^{2)}$ As with *Pyrococcus horikoshii*, the genome sequence of *Methanosarcina barkeri* shows it to be deficient in enzymes for OPPP and NOPPP. Here again, ribose 5-phosphate is assumed to be formed in the reverse reaction of HPS and PHI. ${ }^{15)}$ *Methanosarcina barkeri* possesses two genes encoding Fae, *faeA* and *faeB-hpsB*. The latter gene encodes an enzyme with Fae and HPS activities. Kinetic analysis suggests that Fae in *Methanosarcina barkeri* (also in *Methanosarcina mazei*) might be involved in the detoxification of formaldehyde generated during the biosynthesis of ribose 5-phosphate. The gene product of *faeB-hpsB* might catalyze the following reaction, where formaldehyde is formed only as the intermediate within the total reaction: ${ }^{15)}$

\[
\begin{aligned}
& \text { D-arabino-3-hexulose 6-phosphate + H${}_{4}$MPT } \\
& \quad = \text { ribulose 5-phosphate + methylene-H${}_{4}$MPT } \\
& \Delta G^{0^{\prime}} = -12 \, \text{kJ/mol}
\end{aligned}
\]

Database analysis shows that the following methanogens and acetogen are analogous to *Methanosarcina barkeri* and *Methanosarcina mazei* in lacking PPP and possessing HPS, PHI, and Fae: *Methanosarcina acetivorans*, *Methanopyrus kandleri*, *Methanothermobacter thermautotrophicus*, *Methanocaldococcus jannaschii*, and *Archaeoglobus fulgidus*. ${ }^{13,15)}$

Interestingly, *Pyrococcus horikoshii* lacks the gene for Fae. One early candidate enzyme for formaldehyde detoxification in this archaeon was the gene product of formaldehyde ferredoxin oxidoreductase (FOR, Fig. 7). FOR from *Pyrococcus furiosus* oxidizes a variety of aldehydes, including formaldehyde, ${ }^{34,35)}$ but the apparent $K_{m}$ for formaldehyde of the purified enzyme is 25 mM, implying that formaldehyde is unlikely to be the physiological substrate. Xavier *et al.* reported that glucose 6-phosphate was cleaved into a C${}_{5}$ intermediate (xylulose 5-phosphate) and formate in *Thermococcus zilligii*. ${ }^{11)}$ The cleavage of hexose phosphate might be explained by the reverse reaction of HPS, and formaldehyde might be oxidized to formate. ${ }^{12)}$ More detailed investigation is required to elucidate the formaldehyde detoxification of archaeal strains that do not contain Fae.

### VIII. Phylogenetic Analysis of HPS and PHI

There are several different putative functions assigned to the coding sequence for the *hps* homologs, and some OMPDC enzymes share sequence similarity with HPS. ${ }^{36)}$ More detailed phylogenetic analysis of the sequences of putative HPS enzymes reveal that members of the OMPDC superfamily (KGPDC, RPE, OMPDC, and HPS) are clearly clustered together (Fig. 8). HPS enzymes are divided into six clusters. The methylotrophic origins of HPS enzymes converge in one cluster. Bacterial HPSs are found in clusters HPS (2) and HPS (3), with most methylotrophic enzymes present in HPS (2). Archaeal HPSs are found in clusters HPS (1) and (4). Each *hps* sequence of a methanogen (*Methanosarcina barkeri*, *Methanosarcina mazei*, *Methanosarcina acetivorans*, *Methanothermobacter thermoautotrophicum*, or *Methanocaldococcus jannaschii*) is fused with an *fae* sequence. HPS-PHI orthologs (cluster HPS-PHI) appear in the genomes of hyperthermophilic anaerobic archaea, such as *Pyrococcus horikoshii*, *Pyrococcus furiosus*, *Thermococcus kodakaraensis*, and *Pyrococcus abyssi*.

The enzymes, which have been assigned to PHI, are branched into two clusters, L-xylulose 5-phosphate 3-

N. KATO et al.

HPS-PHI

HPS (2)

Myc. gastri(BAA 90546)
Mem. aminofaciens(Q 48907)
Am. aminovorus(AAG 29505)
Mec. capsulatus(AAU 91180)
Mec. capsulatus(AAU 90889)
Arc. fulgidus(E 69357)
Mets. acetivorans(AAM 07844)
Mets. mazei(AAM 30911)
Su. solfataricus(E 90161)
Su. tokodaii(BAB 65199)
C. acetobutylicum(E 96948)
Metc. jannaschii(O 58842)
Mett. thermautotrophicus(D 69063)
Mets. barkeri(ZP_00297671)
Mets. acetivorans(AAM 07947)
Mets. mazei(AAM 30975)
Mett. thermautotrophicus(O 26232)
Tp. acidophilum(O 74110)
T. kodakarensis(BAD 86465)
Pyc. furiosus(AAL 81238)
Pyc. horikoshii(O 58462)
Pyc. abyssi(CAB 50224)
Pla. falciparum(AAL 86703)
B. subtilis(P 25971)
E. coli(P 08244)
Tp. acidophilum(CAC 12436)
Metc. jannaschii(Q 58093)
B. subtilis(CAB 13452)
E. coli(NP_417845)
P. putida(AAN 66045)

RPE

Fig. 8. An Unrooted Tree of HPS and Other Members of the OMPDC Suprafamily.
OMPDC, orotidine-5'-monophosphate decarboxylase; KGPDPC, 3-keto-L-gulonate-6-phosphate decarboxylase; RPE, D-ribose-5-phosphate-3-epimerase. Accession nos. to the protein database are shown in parentheses. Am., Aminomonas; Ap., Aeropyrum; Arc., Archaeoglobus; B., Bacillus; Bb., Brevibacillus; C., Clostridium; E., Escherichia; H., Haemophilus; K., Klebsiella; Mec., Methylococcus; Mem., Methylomonas; Metc., Methanocaldococcus; Metp., Methanopyrus; Mets., Methanosarcina; Mett., Methanothermobacter; Myc., Mycobacterium; Myp., Mycoplasma; P., Pseudomonas; Pa., Pasteurella; Pla., Plasmodium; Pyb., Pyrobaculum; Pyc., Pyrococcus; Sal., Salmonella; Sta., Staphylococcus; Str., Streptococcus; Su., Sulfolobus; T., Thermococcus; Tp., Thermoplasma; V., Vibrio.

epimerase (XPE) and PHI enzymes (Fig. 9). XPE catalyzes the epimerization of L-xylulose 5-phosphate to L-ribulose 5-phosphate after the reaction catalyzed by KGPDPC in the utilization of L-ascorbate.²⁴) XPE, listed in Fig. 9, and KGPDPC, listed in Fig. 8, make up operons like HPS and PHI. The PHI enzymes are clearly divided into the bacterial enzymes and archaeal enzymes. These organisms, which appear in the clusters of HPS and PHI, have homologous genes for both HPS and PHI, implying that the two genes evolved as a pair. Although not every HPS or PHI ortholog has been checked for activity, at least one of the orthologs in each cluster has been functionally determined. Thus an organism in the clusters of HPS or PHI is regarded as possessing the complete or partial RuMP pathway.

IX. Conclusion and Outlook

In bacteria, HPS and PHI are induced by form-
aldehyde and function in C₁ compound assimilation of methylotrophs or formaldehyde detoxification of non-methylotrophs. In the archaeal stains, HPS and PHI activities are expressed constitutively. Phylogenetic analysis of archaeal genomes has revealed a novel function of the enzyme system in which the reverse reaction is employed for the biosynthesis of pentose phosphates. This finding is consistent with the enzymology of archaeal HPS and PHI.

The discovery of reactions dependent on H₄MPT in *Methylobacterium extorquens* AM1 brought about the important realization that the metabolism of aerobic methylothropy and anaerobic methanogenesis involve common genes that cross bacterial/archaeal boundaries.³⁷) In this case, knowledge gained from archaea was used to explain bacterial metabolism. By contrast, genetic information concerning HPS and PHI from aerobic bacteria has served to elucidate a novel glycolic pathway in Archaea involving the reverse reactions of

The Ribulose Monophosphate Pathway

Myp. pneumoniae (P75294)
Myp. pulmonis (F90586)
Metp. kandleri (AAM 02876)
B. subtilis (P42404)
Bb. brevis (CAH 33894)
Sal. typhimurium (AAL 21642)
Myc. gastri (BAA 90545)
Sta. aureus (BAB 94391)
Mec. capsulatus (AAU 90888)
Mec. capsulatus (AAU 91180)
Mem. aminofaciens (BAA 83098)
Mets. mazei PHI (AAM 32063)
Mets. acetivorans PHI (AAM 04800)
Mets. barkeri PHI (ZP_00542652)

V. cholerae (G82483)
Str. pyogenes (BAC 63237)
E. coli (AAC 77154)
Sal. enterica (AAX 68167)
E. coli (AAC 76606)
K. oxytoca (AAK 69527)
H. influenzae (P44990)
Pa. multocida (AAK 03329)
C. acetobutylicum (F96948)
Metc. jannaschii (Q58644)
Mett. thermautotrophicus (C69073)
Pyb. aerophilum (AAL 63515)
Ap. pernix (BAA 80949)
Su. solfataricus (C90155)
Su. tokodaii (BAB 65155)
Pyc. furiosus (AAL 80344)
T. kodakarensis (BAD 64664)
Pyc. horikoshii (B71209)
Pyc. abyssi (C75022)

Fig. 9. An Unrooted Tree of PHI and L-Xylulose-5-phosphate 3-Epimerase (XPE).

Accession nos. to the protein database are shown in parentheses. Abbreviations are the same as in Fig. 8.

both enzymes. In order to confirm the proposed pathway in archaea, more detailed biochemical and microbiological studies are required.

Acknowledgments

Research by N.K. was supported by the “Forschungsstipendiums im Rahmen der Förderung der wissenschaftlichen Zusammenarbeit mit dem Ausland” of the Max-Planck-Institut für terrestrische Mikrobiologie. This work was supported in part by the 21st Century COE Program of the Japanese Ministry of Education, Culture, Sports, Science and Technology. We are grateful to Professor Yasuyoshi Sakai and Ms. Izumi Orita (Kyoto University), and Dr. Seigo Shima and Dr. Meike Goenrich (Max-Planck-Institut, Marburg) for stimulating discussion during the preparation of this manuscript and for commenting on the text.

References

1) Yurimoto, H., Kato, N., and Sakai, Y., Assimilation, dissimilation, and detoxification of formaldehyde, a central metabolic intermediate of methylotrophic metabolism. *Chem. Rec.*, **5**, 367–375 (2005).
2) Vorholt, J. A., Cofactor-dependent pathway of formalddehyde oxidation in methylotrophic bacteria. *Arch. Microbiol.*, **178**, 239–249 (2002).
3) Yanase, H., Ikeyama, K., Mitsui, R., Ra, S., Kita, K., Sakai, Y., and Kato, N., Cloning and sequence of the gene encoding 3-hexulose-6-phosphate synthase from the methylotrophic bacterium, *Methylomonas aminofaciens* 77a, and its expression in *Escherichia coli*. *FEMS Microbiol. Lett.*, **135**, 201–205 (1996).
4) Sakai, Y., Mitsui, R., Katayama, Y., Yanase, H., and Kato, N., Organization of the genes involved in the ribulose monophosphate pathway in an obligate methylotrophic bacterium, *Methylomonas aminofaciens* 77a. *FEMS Microbiol. Lett.*, **176**, 125–130 (1999).
5) Mitsui, R., Sakai, Y., Yasueda, H., and Kato, N., A novel operon encoding formaldehyde fixation: the ribulose monophosphate pathway in the Gram-positive facultative methylotrophic bacterium *Mycobacterium gastri* MB19. *J. Bacteriol.*, **184**, 944–948 (2000).
6) Yurimoto, H., Hirai, R., Yasueda, H., Mitsui, R., Sakai, Y., and Kato, N., The ribulose monophosphate pathway operon encoding formaldehyde fixation in a thermotolerant methylotroph, *Bacillus brevis* S1. *FEMS Micro-

biol. Lett., 214, 189–193 (2002).

7) Dijkhuizen, L., Levering, P. R., and Vries, D., The physiology and biochemistry of aerobic methanol-utilizing gram-negative and gram-positive bacteria. In “Biotechnology Handbooks”, eds. Murrell, J. C., and Dalton, H., Plenum Publishing Co., New York, pp. 149–181 (1992).

8) Vorholt, J. A., Marx, C. J., Lidstrom, M. E., and Thauer, R. K., Novel formaldehyde-activating enzyme in *Methylobacterium extorquens* AM1 required for growth on methanol. *J. Bacteriol.*, 182, 6645–6650 (2000).

9) Yasueda, H., Kawahara, K., and Sugimoto, S., *Bacillus subtilis yckG* and *yckF* encode two key enzymes of the ribulose monophosphate pathway used by methylotrophs, and *yckH* is required for their expression. *J. Bacteriol.*, 181, 7154–7160 (1999).

10) Mitsui, R., Kusano, Y., Yurimoto, H., Sakai, Y., Kato, N., and Tanaka, M., Formaldehyde fixation contributes to detoxification for growth of a nonmethylotroph, *Burkholderia cepacia* TM1, on vanillic acid. *Appl. Environ. Microbiol.*, 69, 6128–6132 (2003).

11) Xavier, K. B., Da Costa, M. S., and Santos, H., Demonstration of a novel glycolic pathway in the hyperthermophilic archaeon *Thermococcus zilligii* by ¹³C-labeling experiments and nuclear magnetic resonance analysis. *J. Bacteriol.*, 182, 4632–4636 (2000).

12) Verhees, C. H., Kengen, W. M., Tuninga, J. E., Schut, G. J., Adams, M. W. W., De Vos, W. M., and Van Der Oost, J., The unique features of glycolytic pathways in Archaea. *Biochem. J.*, 375, 231–246 (2000).

13) Soderberg, T., Biosynthesis of ribose-5-phosphate and erythrose-4-phosphate in archaea: a phylogenetic analysis of archaeal genomes. *Archaea*, 1, 347–352 (2005).

14) Orita, I., Yurimoto, H., Hirai, R., Kawarabayasi, Y., Sakai, Y., and Katou, N., The archaeon *Pyrococcus horikoshii* possesses a bifunctional enzyme for formaldehyde fixation via the ribulose monophosphate pathway. *J. Bacteriol.*, 187, 3636–3642 (2005).

15) Goenrich, M., Thauer, R. K., Yurimoto, H., and Kato, N., Formaldehyde activating enzyme (Fae) and hexose-6-phosphate synthase (Hps) in *Methanosarcina barkeri*: a possible function in ribose-5-phosphate biosynthesis. *Arch. Microbiol.*, 184, 41–48 (2005).

16) Murrell, J. C., and Mcdonald, I. R., Methylophy. In “Encyclopedia of Microbiology” Vol. 3, ed. Lederberg, J., Academic Press, London, pp. 245–255 (2000).

17) Ferenci, T., Strome, T., and Quayle, J. R., Purification and properties of 3-hexulose phosphate synthase and phospho-3-hexuloisomerase from *Methylococcus capsulatus*. *Biochem. J.*, 144, 477–486 (1974).

18) Quayle, J. R., and Ferenci, T., Evolutionary aspects of autotrophy. *Microbiol. Rev.*, 42, 251–273 (1978).

19) Ward, N., Larsen, O., Sakwa, J., Bruseth, L., Khouri, H., Durkin, A. S., Dimitrov, G., Jiang, L., Scanlan, D., Kang, K. H., Lewis, M., Nelson, K. E., Methe, B., Wu, M., Heidelberg, J. F., Paulsen, I. T., Fouts, D., Ravel, J., Tettelin, H., Ren, Q., Read, T., Deboy, R. T., Seshadri, R., Salzberg, S. L., Jensen, H. B., Birkeland, N. K., Nelson, W. C., Dodson, R. J., Grindhaug, S. H., Holt, I., Eidhammer, I., Jonassen, I., Vanaken, S., Utterback, T., Feldblyum, T. V., Fraser, C. M., Lillehaug, J. R., and Eisen, J. A., Genomic insights into methanotrophy: the complete genome sequence of *Methylococcus capsulatus*

(Bath). *PLoS Biol.*, 2, 1616–1628 (2004).

20) Martinez-Cruz, L. A., Dreyer, M. K., Boisvert, D. C., Yokota, H., Martinez-Chanter, M. L., Kim, R., and Kim, S. H., Crystal structure of MJ1247 protein from *M. jannaschii* at 2.0 Å resolution infers a molecular function of 3-hexulose-6-phosphate isomerase. *Structure*, 10, 195–204 (2002).

21) Gerlt, J. A., and Raushel, F. M., Evolution of function in (β/α)₈-barrel enzyme. *Curr. Opin. Chem. Biol.*, 7, 252–264 (2003).

22) Wise, E. L., Yew, W. S., Gerlt, J. A., and Rayment, I., Evolution of enzymatic activities in the orotidine 5′-monophosphate decarboxylase superfamily: crystallographic evidence for a proton relay system in the active site of 3-keto-L-gulonate 6-phosphate decarboxylase. *Biochemistry*, 43, 6438–6446 (2004).

23) Yaoi, T., Laksanalamai, P., Jiemjit, A., Kagawa, H. K., Alton, T., and Trent, J. D., Cloning and characterization of *ftsZ* and *pyrF* from the archaeon *Thermoplasma acidophilum*. *Biochem. Biophys. Res. Commun.*, 275, 936–945 (2000).

24) Yew, W. S., and Gerlt, J. A., Utilization of L-ascorbate by *Escherichia coli* K-12: assignments of functions to products of the *yif-sga* and *yia-sgb* operons. *J. Bacteriol.*, 184, 302–306 (2002).

25) Yew, W. S., Wise, E. L., Rayment, I., and Gerlt, J. A., Evolution of enzymatic activities in the orotidine 5′-monophosphate decarboxylase superfamily: mechanistic evidence for a proton relay system in the active site of 3-keto-L-gulonate 6-phosphate decarboxylase. *Biochemistry*, 43, 6427–6437 (2004).

26) Yew, W. S., Akana, J., Wise, E. L., Rayment, I., and Gerlt, J. A., Evolution of enzymatic activities in the orotidine 5′-monophosphate decarboxylase superfamily: enhancing the promiscuous D-arabino-hex-3-ulose 6-phosphate synthase reaction catalyzed by 3-keto-L-gulonate 6-phosphate decarboxylase. *Biochemistry*, 44, 1807–1815 (2005).

27) Yurimoto, H., Hirai, R., Matsuno, N., Yasueda, H., Kato, N., and Sakai, Y., HxlR, a member of the DUF24 protein family, is a DNA-binding protein that acts as a positive regulator of the formaldehyde-inducible *hxlAB* operon in *Bacillus subtilis*. *Mol. Microbiol.*, 57, 511–519 (2005).

28) Kemp, M. B., Hexose phosphate synthase from *Methylococcus capsulatus* makes D-arabino-3-hexulose phosphate. *Biochem. J.*, 139, 129–134 (1974).

29) Marx, C. J., Chistoserdova, L., and Lidstrom, M. E., Formaldehyde-detoxifying role of the tetrahydromethanopterin-linked pathway in *Methylobacterium extorquens* AM1. *J. Bacteriol.*, 185, 7160–7168 (2003).

30) Gonzalez, J. M., Masuchi, Y., Robb, F. T., Ammerman, J. W., Maeder, D. L., Yanagibayashi, M., Tamaoka, J., and Kato, C., *Pyrococcus horikoshii* sp. nov., a hyperthermophilic archaeon isolated from a hydrothermal vent at the Okinawa trough. *Extremophiles*, 2, 123–130 (1998).

31) Canet, C., Prol-Ledesma, R. M., Melgarejo, J.-C., and Reyes, A., Methane-related carbonates formed at submarine hydrothermal springs: a new setting for microbially-derived carbonates? *Marine Geol.*, 199, 245–261 (2003).

32) Damm, E., and Budeus, G., Fate of vent-derived methane in seawater above the Hakon Mosby mud

volcano (Norwegian Sea). *Marine Chem.*, **82**, 1–11 (2003).

33) Kotelnikova, S., Microbial production and oxidation of methane in deep subsurface. *Earth-Sci. Rev.*, **58**, 367–395 (2002).

34) Hu, Y., Faham, S., Roy, R., Adams, M. W. W., and Rees, D. C., Formaldehyde ferredoxin oxidoreductase from *Pyrococcus furiosus*: the 1.85 Å resolution crystal structure and its mechanistic implications. *J. Mol. Biol.*, **286**, 899–914 (1998).

35) Roy, R., Mukund, S., Schut, G. J., Dunn, D. M., Weiss, R., and Adams, M. W. W., Purification and molecular characterization of the tungsten-containing formaldehyde ferredoxin oxidoreductase from the hyperthermo-

philic archaeon *Pyrococcus furiosus*: the third of a putative five-member tungstoenzyme family. *J. Bacteriol.*, **181**, 1171–1180 (1999).

36) Taylor, E. J., Smith, N. L., Colby, J., Charnock, S. J., and Black, G. W., The gene encoding the ribulose monophosphate pathway enzyme, 3-hexulose-6-phosphate synthase, from *Aminomonas aminovorans* C2A1 is adjacent to coding sequences that exhibit similarity to histidine biosynthesis enzymes. *Antonie Van Leeuwenhoek*, **86**, 167–172 (2004).

37) Chistoserdova, L., Vorholt, J. A., Thauer, R. K., and Lidstrom, M. E., C₁ transfer enzymes and coenzymes linking methylotrophic bacteria and methanogenic archaea. *Science*, **281**, 99–102 (1998).
