
FEBS Letters 580 (2006) 1198–1204

# Glyceraldehyde dehydrogenases from the thermoacidophilic euryarchaeota *Picrophilus torridus* and *Thermoplasma acidophilum*, key enzymes of the non-phosphorylative Entner–Doudoroff pathway, constitute a novel enzyme family within the aldehyde dehydrogenase superfamily

**Matthias Reher, Peter Schönheit\***

Institut für Allgemeine Mikrobiologie, Christian-Albrechts-Universität Kiel, Am Botanischen Garten 1-9, D-24118 Kiel, Germany

Received 9 November 2005; revised 21 December 2005; accepted 11 January 2006

Available online 19 January 2006

Edited by Judit Ovádi

---

### Abstract  
Cells of *Picrophilus torridus*, grown on glucose, contained all enzyme activities of a non-phosphorylative Entner–Doudoroff pathway, including glucose dehydrogenase, gluconate dehydratase, 2-keto-3-deoxygluconate aldolase, glyceraldehyde dehydrogenase (GADH), glycerate kinase (2-phosphoglycerate forming), enolase and pyruvate kinase. GADH was purified to homogeneity. The 115-kDa homodimeric protein catalyzed the oxidation of glyceraldehyde with NADP⁺ at highest catalytic efficiency. NAD⁺ was not used. By MALDI-TOF analysis, open reading frame (ORF) Pto0332 was identified in the genome of *P. torridus* as the encoding gene, designated *gadh*, and the recombinant GADH was characterized. In *Thermoplasma acidophilum* ORF Ta0809 represents a *gadh* homolog with highest sequence identity; the gene was expressed and the recombinant protein was characterized as functional GADH with properties very similar to the *P. torridus* enzyme. Sequence comparison and phylogenetic analysis define both GADHs as members of novel enzyme family within the aldehyde dehydrogenase superfamily.

© 2006 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

#### Keywords: Non-phosphorylative Entner–Doudoroff pathway; Glyceraldehyde dehydrogenase; Aldehyde dehydrogenase superfamily; *Picrophilus torridus*; *Thermoplasma acidophilum*

---

### 1. Introduction

The thermoacidophilic euryarchaeota *Thermoplasma acidophilum* and *Picrophilus torridus* grow optimally at about 60 °C and at pH values between 1–4 and 0–2, respectively. Thus, *P. torridus* represents the most acidophilic organism described so far [1,2]. The internal pH of *Picrophilus* has been determined to be 4.6 [3]. *T. acidophilum* and *P. torridus* utilize a variety of sugars, including glucose and starch, as growth substrates [4,5]. Analysis of the glucose degradation pathway revealed that in *T. acidophilum* a non-phosphorylative version of the Entner–Doudoroff (ED) pathway is operative [6]. Accordingly, glucose is oxidized to gluconate via glucose dehydrogenase, subsequent dehydration via gluconate dehydratase yields 2-keto-3-deoxygluconate, which is cleaved to pyruvate and glyceraldehyde. Glyceraldehyde is then oxidized to glycerate via a NADP⁺ reducing glyceraldehyde dehydrogenase (GADH). A specific kinase phosphorylates glycerate to 2-phosphoglycerate, which is then converted to phosphoenolpyruvate by enolase and further to pyruvate by pyruvate kinase (Fig. 1) [6,7]. Based on genome data, a non-phosphorylative ED pathway was also postulated for *P. torridus* [2]. However, with exception of glucose dehydrogenase [8], no enzyme activities of that pathways had yet been demonstrated. Here we report that extracts of glucose-grown *P. torridus* contained all enzyme activities of a non-phosphorylative ED pathway, as reported for *Thermoplasma*, including a NADP⁺ reducing GADH. Further we describe the purification and characterization of GADH from *P. torridus* and the identification of its encoding gene, designated *gadh*, by MALDI-TOF analysis. In *T. acidophilum* the *gadh* homolog was identified, expressed and the recombinant protein characterized as GADH with properties very similar as the *Picrophilus* enzyme. Sequence comparison, phylogenetic analysis and catalytic function define both GADHs as members of a novel enzyme family within the aldehyde dehydrogenase (ALDH) superfamily.

---

### 2. Materials and methods

#### 2.1. Determination of enzyme activities

Unless otherwise stated, enzyme activities were assayed spectrophotometrically at 55 °C in 50 mM HEPES, pH 6.5, with 1 ml of assay mixture. One unit of enzyme activity is defined a 1 μmol substrate consumed or product formed per min. It was ensured that in coupled assays the auxiliary enzymes were not rate limiting. Glucose dehydrogenase was tested by the reduction of NADP⁺. The assay mixture contained 1 mM NADP⁺, 25 mM glucose, and extract. Gluconate dehydratase was determined by measuring the formation of KDG,

---

*Corresponding author. Fax: +49 431 880 2194. E-mail address: peter.schoenheit@ifam.uni-kiel.de (P. Schönheit).

0014-5793/$32.00 © 2006 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

doi:10.1016/j.febslet.2006.01.029

M. Reher, P. Schönheit / FEBS Letters 580 (2006) 1198–1204

Glucose  
NAD(P)+  
NAD(P)H + H+  

Gluconate  

Gluconate  
Dehydratase  

KDG  

KDGAldolase  

Pyruvate Glyceraldehyde  

Glyceraldehyde  
Dehydrogenase  

Glycerate  

ATP  
ADP  

2-Phosphoglycerate  

Enolase  

H₂O  

Phosphoenolpyruvate  

ADP  

ATP  

Pyruvate  

Pyruvate  
kinase  

Fig. 1. Proposed non-phosphorylative ED pathway in the thermoacidophilic euryarchaea *Picrophilus torridus* and *Thermoplasma acidophilum* [6] involving glyceraldehyde dehydrogenase as a key enzyme.

which was quantified as described by Hurwitz and Weissbach [9]. The assay mixture contained 5 mM gluconate and extract at 60 °C. KDG aldolase was tested by measuring KDG formation from glyceraldehyde and pyruvate. The assay mixture contained 100 mM pyruvate, 15 mM glyceraldehyde and extract. The formation of KDG was followed at 60 °C over a period of 30 min and quantitated in the thiobarbituric acid assay at 546 nm [9]. KDPG aldolase was determined as described above for KDG aldolase. The assay mixture contained 100 mM pyruvate, 15 mM glyceraldehyde-3-phosphate and extract. Glycerate kinase was tested by coupling the ATP-dependent formation of 2-phosphoglycerate to the oxidation of NADH via enolase, pyruvate kinase and lactate dehydrogenase. The assay mixture contained 0.3 mM NADH, 5 mM glycerate, 2 mM ATP, 2 mM ADP, 10 mM MgCl₂, 5 mM EGTA, 5 U enolase, 5 U pyruvate kinase, 5.5 U lactate dehydrogenase and extract. Enolase was tested by coupling the formation of phosphoenolpyruvate to the oxidation of NADH via pyruvate kinase and lactate dehydrogenase. The assay mixture contained 0.3 mM NADH, 1 mM 2-phosphoglycerate, 2 mM ADP, 10 mM MgCl₂, 5 U pyruvate kinase, 5.5 U lactate dehydrogenase and extract. Pyruvate kinase was determined by coupling the ADP-dependent formation of pyruvate and ATP with the oxidation of NADH by lactate dehydrogenase. The assay mixture contained 1 mM phosphoenolpyruvate, 2 mM ADP, 10 mM MnCl₂, 0.3 mM NADH, 5 U lactate dehydrogenase, and extract.

### 2.1.1. GADH assays.
GADH activity was assayed at 58 °C by measuring the rate of reduction of NADP⁺ at 365 nm. One unit of activity was defined as reduction of 1 μmol NADP⁺ per min. The standard assay mixture contained 100 mM HEPES, pH 6.9 and 7.9, for the enzymes of *P. torridus* and *T. acidophilum*, respectively, 1 mM NADP⁺, 1 mM glyceraldehyde, and protein. For the determination of apparent \( K_m \) and \( k_{cat} \) values for aldehydes and NADP⁺, the following concentrations were used: glyceraldehyde (0–2 mM) with 1 mM NADP⁺; NADP⁺ (0–3 mM) with 1 mM glyceraldehyde; glyceraldehyde-3-phosphate (0–10 mM) with 1 mM NADP⁺; glycolaldehyde (0–30 mM), formaldehyde (0–7 mM), benzaldehyde (0–12 mM), acetaldehyde (0–12 mM), crotonaldehyde (1–10 mM), dihydroxyacetone (0–12 mM) with 1 mM NADP⁺ and NAD⁺ (0–5 mM) with 1 mM glyceraldehyde.

### 2.2. Purification of GADH from *P. torridus* and identification of the encoding gene
For GADH purification, fermentor-grown cells (20 g wet weight) were harvested in the late-exponential growth phase by centrifugation, suspended in 50 mM Na-acetate, pH 6, and disrupted by passage through a French pressure cell at \( 1.3 \times 10^8 \) Pa. Cell debris was removed by centrifugation for 90 min at 100000×g. The supernatant was adjusted to a pH of 4.9 and applied to a SP-Sepharose HiLoad 26/10 column equilibrated with 50 mM Na-acetate, pH 4.9. Protein was desorbed with 50 mM Na-acetate, pH 5.4, followed by a linear gradient from 0 to 2 M NaCl. The fractions with the highest GADH activity (0.1–0.3 M NaCl) were pooled, adjusted to pH 6 with 100 mM BisTris, pH 6 and applied to a UNO-Q1 column equilibrated with the same buffer. The protein was eluted with a increasing gradient from 0 to 2 M NaCl. The fraction containing the highest GADH activity (0.1 M NaCl) was applied to a Superdex 200 HiLoad 16/60 column equilibrated with 50 mM Tris/HCl, pH 7.4, and 150 mM NaCl. The protein was eluted at a flow rate of 1 ml/min. At this stage GADH was essentially pure as judged by SDS-PAGE using 12% gels. The protein spot was excised from the Coomassie-stained SDS gel and subjected to digestion in the gel with trypsin as described [10]. The tryptic peptides were extracted and analyzed using a MALDI-TOF mass spectrometer (Voyager STR, Applied Biosystem, Weiterstadt, Germany) as reported previously [10].

### 2.3. Cloning and expression of glyceraldehyde dehydrogenase from *P. torridus* and *T. acidophilum* in *E. coli* and purification of the recombinant enzymes
On the basis of MALDI-TOF analysis, a single open reading frame (ORF), Pto0332, was identified in the sequenced genome of *P. torridus* [2]. By BLASTP search a homologous ORF, Ta0809, was found in the genome of *T. acidophilum* [11]. The ORFs Pto0332 and Ta0809 were amplified by PCR. The PCR products were both cloned into pET vectors via two restriction sites (pET17b: EcoR V, XhoI and pET19b: NdeI, BamH1) created with the primers 5′ AATAGA-TCCGATGAAAATTACATAGATG-3′ and 5′ TTTCTCGA-GTTTTAAAG-ATTTTGAAATAGGA-3′ for pET17b-Pt00332, and 5′ GCTAACGTT-ACATATGGATACGAAGTTATAC-3′ and 5′ ATGGATCCTTAA-GATCACTGAAACA-GATCGTC-3′ for pET-19b-Ta0809. Both vectors were transformed into *E. coli* BL21 codon plus (DE3)-RIL cells. For expression of pET17b-Pto0332, cells were grown in Luria-Bertani medium at 20 °C. The expression was started by inducing the promoter with 0.4 mM isopropyl-β-D-thiogalactopyranoside (IPTG). After 16 h of further growth, cells were harvested by centrifugation followed by resuspension in 100 mM Na-acetate, pH 6. Transformed *E. coli* cells were disrupted by passing through a French pressure cell. After centrifugation, the supernatant was heat-precipitated at 60 °C for 30 min and centrifuged. The *P. torridus* enzyme purified as described for the native GADH. Recombinant GADH from *T. acidophilum* was obtained as a His-tagged protein after expression of pET19b-Ta0809. Cells were grown in Luria-Bertani medium at 37 °C and expression was started by inducing the promoter with 0.4 mM IPTG. After 4 h of further growth, cells were harvested by centrifugation followed by resuspension in 100 mM Tris-HCl, 300 mM NaCl, and 4 mM imidazole, pH 7.4. Transformed *E. coli* cells were disrupted by passing through a French pressure cell. After centrifugation, the supernatant was applied to a Ni-NTA column equilibrated with the same buffer. The protein was eluted stepwise with increasing imidazole concentration (20, 100, 400 mM). The fractions containing GADH activity (400 mM imidazole) were incubated 1 h at 58 °C and centrifuged. At this stage *T. acidophilum* GADH was essentially pure as judged by SDS-PAGE using 12% gels. Protein concentrations were determined by the method of Bradford with bovine serum albumin as standard.

The apparent molecular mass of GADH was determined by gel filtration on a Superdex 200 HiLoad 16/60 column (see above). The molecular mass of the subunit was determined by SDS-PAGE.

### 2.4. pH dependence, temperature dependence and thermal stability
The pH dependence of the GADHs from *P. torridus* and *T. acidophilum* was measured at 58 °C between pH 4.0 and pH 8.8 using different buffers, Na-acetate 0.1 M (pH 4.0–6.0), HEPES 0.1 M (pH 6.0–8.2) or CHES 0.2 M (pH 8.3–8.8). The temperature dependence was determined between 40 and 77 °C. The long term thermostability

(6 μg protein in 60 μl HEPES buffer, pH 7.0) was tested in sealed vials, which were incubated at 50, 60 and 70 °C up to 120 min. The vials were cooled for 10 min and the remaining activity was tested.

### 3. Results

#### 3.1. Enzymes of the non-phosphorylative ED pathway in *P. torridus*

Cell extracts of *P. torridus*, grown on glucose as substrate, contain all enzyme activities (given in U/mg) of a non-phosphorylative ED pathway, including glucose dehydrogenase (2.6), gluconate dehydratase (0.07), 2-keto-3-deoxygluconate aldolase (0.05), GADH (0.11), glycerate kinase (0.08), enolase (0.18) and pyruvate kinase (0.32). 2-Keto-3-deoxy-6-phosphogluconate aldolase activity could not be detected. The data indicate that in *P. torridus* a non-phosphorylative ED pathway is operative as reported for *T. acidophilum* [6].

#### 3.2. Purification of GADH from *P. torridus* and identification of the encoding gene

GADH was purified 223-fold to apparent homogeneity (Table 1). SDS-PAGE of the purified enzyme revealed one subunit of 56 kDa (Fig. 2). The molecular mass, as estimated by gel filtration on Superdex 200, was 115 kDa indicating a homodimeric structure of the native enzyme. By MALDI-TOF analysis a single ORF, Pto0332, was identified in the genome of *P. torridus*. The matched peptides cover 23% of the protein and the coverage map (MS-Digest index) is 332. Pto0332, previously annotated as putative succinate semialdehyde dehydrogenase (SSDH) [2], consists of 1482 bp coding for a polypeptide of 493 amino acids with a calculated molecular mass of 55.2 kDa. The coding function of Pto0332 was proven by cloning and heterologous overexpression in *E. coli*. The recombinant protein was purified to homogeneity and characterized as functional GADH, catalyzing the glyceraldehyde dependent reduction of NADP⁺ at 60°C with a specific activity of 5.5 U/mg (1 mM glyceraldehyde and 1 mM NADP⁺). Thus, Pto0332 was assigned as *gadh* gene. Detailed kinetic characterization of GADH was carried out using the recombinant enzyme (Table 2).

#### 3.3. Properties of recombinant GADH from *P. torridus*

The recombinant enzyme catalyzed the oxidation of glyceraldehyde with NADP⁺ as electron acceptor. Rate dependence on glyceraldehyde (up to 1.5 mM) and on NADP⁺ followed Michaelis-Menten kinetics (*K*ₘ, 0.06 mM). The enzyme showed slight substrate inhibition with respect to glyceraldehyde (25% inhibition at 2 mM glyceraldehyde). In addition

**Fig. 2.** Purified glyceraldehyde dehydrogenase from *P. torridus* as analyzed by SDS-PAGE. Lanes: (1) molecular mass standards; (2) enzyme from *P. torridus*.

**Table 2**

Molecular and catalytic properties of recombinant GADH from *P. torridus* and *T. acidophilum*

| Property | *P. torridus* | *T. acidophilum* |
|----------|---------------|------------------|
| Apparent molecular mass of enzyme (kDa) | Native | 120 (115)<sup>a</sup> | 120 |
| | Subunit | 56 (55)<sup>a</sup> | 56 |
| | Calculated | 55.2 | 54.7 |
| Oligomeric structure | α₂ | α₂ |
| *K*ₘ (mM) | Glyceraldehyde | 0.06 | 0.2 |
| | GAP | 5.1 | 18.3 |
| | Glycolaldehyde | 11.1 | 12.3 |
| | NADP⁺ | 0.42 | 0.36 |
| *K*<sub>cat</sub> (s⁻¹) | Glyceraldehyde | 10 | 26 |
| | GAP | 12 | 20 |
| | Glycolaldehyde | 20 | 37 |
| | NADP⁺ | 10 | 25 |
| *K*<sub>cat</sub>/*K*ₘ (s⁻¹ mM⁻¹) | Glyceraldehyde | 167 | 130 |
| | GAP | 2.4 | 1.1 |
| | Glycolaldehyde | 1.8 | 3 |
| | NADP⁺ | 24 | 70 |
| pH optimum | 6.6 | 7.6 |
| Temperature optimum (°C) | 63 | 63 |

The molecular mass of native enzyme was determined by gel filtration, of subunit by SDS-PAGE.

<sup>a</sup>GADH purified from *P. torridus*.

to glyceraldehyde (100%), the enzyme accepted glyceraldehyde-3-phosphate and glycolaldehyde as substrates, however at significant lower catalytic efficiencies (*k*<sub>cat</sub>/*K*ₘ), 1.5% and 1%, respectively (Table 2). Other aldehydes such as acetaldehyde, crotonaldehyde, benzaldehyde and succinate semialdehyde, and dihydroxyacetone were not used by the enzyme. GADH was specific for NADP⁺ as electron acceptor; NAD⁺ was not accepted. The pH optimum of GADH was at pH 6.6 and showed still 50% of activity at pH 4.3 and at pH 8.1. GADH showed moderate thermophilic properties; the enzyme had a temperature optimum at 63°C and did not lose activity

**Table 1**

Purification of GADH from *P. torridus*

| Purification step | Total protein (mg) | Total activity (U) | Specific activity (U mg⁻¹) | Enrichment factor | Yield (%) |
|--------------------|--------------------|--------------------|----------------------------|--------------------|-----------|
| Cell extract       | 2853               | 90                 | 0.031                      |                    |           |
| Cation exchange    | 6                  | 18                 | 3                          | 96                 | 20        |
| Anion exchange     | 1.9                | 12.3               | 6.5                        | 209                | 7.2       |
| Gel filtration     | 0.145              | 1                  | 6.9                        | 223                | 1.1       |

upon incubation at 60° for 2 h. The half time at 70 °C was about 15 min.

### 3.4. Identification and characterization of GADH from T. acidophilum

Using the *P. torridus* GADH sequence homologs with high sequence identity (>70%) were found by BlastP searches only in genomes of other Thermoplasmales, including *Ferroplasma acidarmanus*, *T. acidophilum* and *Thermoplasma volcanium*. One homolog, ORF TA0809 of *T. acidophilum*, which was previously annotated as putative ALDH, was cloned and expressed in *E. coli*. The recombinant, His-tagged protein was purified to homogeneity using Ni-NTA chromatography and a heat treatment step. The purified enzyme was characterized as NADP⁺ reducing GADH showing similar molecular properties and substrate specificities as the *P. torridus* GADH (Table 2). The enzyme was highly specific for both glyceraldehyde and NADP⁺. Glyceraldehyde-3-phosphate and glycolaldehyde were used only at 1–2% of the catalytic efficiency found for glyceraldehyde. NAD⁺ was not accepted as an electron acceptor. Also, the thermophilic properties of the *T. acidophilum* enzyme were similar, it had a temperature optimum at 63 °C, did not lose activity at 60 °C for 2 h and showed a halflife at 70 °C of 40 min. The pH optimum of *T. acidophilum* GADH was pH 7.6; 50% of activity was found at pH 5.8 and 8.7. Thus, the optimal pH of the *T. acidophilum* enzyme was by more than one pH unit more alkaline than that of the *P. torridus* enzyme.

## 4. Discussion

In this paper we report the presence of all enzyme activities of the non-phosphorylative ED pathway in the *P. torridus*, indicating this modification of the ED pathway is also operative in this organism as reported for *T. acidophilum* [6]. GADH, a key enzyme of this pathway, was characterized from both *P. torridus* and *T. acidophilum*. The genes encoding GADH, were identified in *P. torridus* (Pto0332) by MALDI-TOF analysis of the purified enzyme and in *T. acidophilum* (Ta0809) by homology search. Both genes were expressed in *E. coli* as functional GADH, and were thus assigned as *gadh* genes.

**Biochemical features:** GADHs from both organisms are dimeric enzymes that were highly specific for glyceraldehyde and for the electron acceptor NADP⁺. NAD⁺ was not reduced. Besides glyceraldehyde only glyceraldehyde-3-phosphate and glycolaldehyde were used, however at catalytic efficiencies less than 2% as compared to glyceraldehyde. Any other aldehyde tested, including aliphatic and aromatic aldehydes or succinate semialdehyde did not serve as substrates. Thus, the annotation of Pto0332 as putative SSDH [2] can be ruled out. GADHs showed moderate thermophilic properties, i.e. a temperature optimum and pronounced thermostability around 60 °C, which is in accordance with the optimal growth temperature (about 60 °C) of *Picrophilus* and *Thermoplasma*. The pH-optima from GADHs of *P. torridus* and *T. acidophilum* were 6.6 and 7.6, respectively, and thus are significantly higher than the internal pH in these organisms, pH 4.6 and 6 [3], respectively. However, GADH of *P. torridus* still retained a high (almost 60%) activity at pH 4.6 suggesting a significant adaptation of the enzyme to this extremely low pH value.

### 4.1. GADHs constitute a novel enzyme family of the ALDH superfamily

**Sequence alignment:** GADHs from *P. torridus* and *T. acidophilum* showed high sequence identity (>70%) to each other and to putative homologs of other members of Thermoplasmales, i.e. of *Thermoplasma vulcanium* and *F. acidarmanus*. A significant sequence identity of GADHs, of 25–35%, were found to be members of various ALDH families, which belong to the ALDH superfamily, suggesting that GADH are members of that superfamily. This was confirmed by the sequences alignment shown in Fig. 3, which include sequences of characterized and putative GADHs from Thermoplasmales, and of two members of different ALDH families, the non-phosphorylating glyceraldehyde-3-phosphate dehydrogenase (GAPN) from the archaeon *Thermoproteus tenax* and the eukaryal ALDH1A1 from sheep. All sequences share the conserved motifs typical of the ALDH superfamily including the three sequences shown in boxes A–C. The two consensus patterns, consensus 1 (PROSITE pattern PS00687, Box B) and consensus 2 (PROSITE pattern PS00070, Box C), indicative for the ALDH superfamily, contain the typical conserved active site glutamate and the active site cysteine, respectively, as concluded from crystal structures of various ALDHs, including ALDH 1A1. Further, the sequence of a modified Rossmann fold (GxTxxG), a typical feature of the ALDH superfamily [12], is conserved in all sequences (Box A). On the basis of the modified Rossmann fold, members of the ALDH superfamily can be discriminated from dehydrogenases such as the phosphorylating glyceraldehyde-3-phosphate dehydrogenases (GAPDH), which contain the classical Rossmann fold and which belong to the glyceraldehyde-3-phosphate superfamily. The presence of the conserved features is indicative for ALDH superfamily and define GADHs as members of that superfamily. The ALDH superfamily comprises a large numbers of different ALDHs families from the three domains of life, Eukarya, Bacteria and Archaea; 555 sequences of ALDHs are known (July 2002) [13] and several crystal structures have been solved [14–16].

**Phylogenetic affiliation:** The phylogenetic relationship between GADHs from Thermoplasmales and members of a selected number of ALDH families is shown in Fig. 4. All these families of the ALDH form distinct phylogenetic clusters and display a characteristic catalytic function. The human ALDH1 family is primarily involved in the oxidation of retinal to retinoic, which is a modulator of gene expression and differentiation. ALDH3 oxidizes a variety of aromatic and medium chain fatty acid aldehydes: SSDHs are involved in the degradation of 4-aminobutyrate [16–18], and members of the aldA family, e.g., of *E. coli*, catalyze the oxidation of lactaldehyde as part of the fucose metabolism [19]. Sequences of GAPN form two distinct enzyme families, one comprising eukaryal and bacterial sequences and a second containing all archaeal sequences. GAPN catalyzes the NAD(P)⁺ dependent oxidation of glyceraldehyde-3-phosphate to 3-phosphoglycerate. In accordance with the distinct phylogenetic clustering, the two different GAPN families display different physiological functions: the bacterial and eukaryal GAPN are involved in the generation of NADPH for anabolic purposes [20,21], whereas the archaeal GAPN characterized from *T. tenax* and *Sulfolobus solfataricus* have a catabolic function in glucose degradation as part of a modified Embden Meyerhof pathway or semi-phosphorylative branch of ED pathway, respectively [7,22].

10 20 30 40 50 60 70 80
P. torridus 1 -MKIYIDGEWRDSSGETIKKYNESTGEVLDTFPAATRNDVDAAIDSAEDAFKR---WSDM 57
T. acidophilum 1 -MDTKLYIDGQWVNSSSGKTVDKYSPTGQVIGRFEAATRDDVDRAIDAAEDAFWA---WNDL 59
T. volcanium 1 -MDGKLYIDGQWVDS SNGDTLTKYNPTGEVIGKFAAASRDDVDRAFDAEADAFQK---WMDL 59
F. acidarmanus 1 -MKIYIDGTWKES SNGNTLNKYNESTGEVLDTFPAATKDDIDAAMDSAESAFPS---WSDM 57
T. tenax_GAPN 1 --MRAGLLEGVIKEKGVPVYPSYLAGEWGS--GQEIEVKSPIDLATIAKVISPSREEVERTLDVLFKGRWS---ARDM 74
AL1A1 SHEEP 1 MSSSAMPDVPAPLTNLQFKYTKIFINNEWHSSVSGKKFPVFNRATEEKLCVEVEEGDKEDVDKAVKAARQAFQIGSPWRTM 80

90 100 110 120 130 140 150 160
P. torridus 58 TSERSKILYKALELISKDKDQLTDLLIKENGKVKR-EAMDTEGVIDQLQYYTEFERKLTDIVEGTSNKRKIFQY--- 133
T. acidophilum 60 GSVERSKIIYRAKELIEKNRAELENIIMEENGPVK-EAKEEVGDVIDQIQYYAEWARKLNGEVEGTSSHRKIFQY--- 135
T. volcanium 60 GSVERSKIIYKAKELIEENRKELENTIQENGKPVK-EAAEEVDGVLDQMYYAEWARKLNGEVEGTTNHRKIFQY--- 135
F. acidarmanus 58 GTQERAKILYKAVELISQKRGELENLLTLEVGIKR-EAIDEVDGVIDQIQYYAEFERKLTDIVEGSSSKRKIFQY--- 133
T. tenax_GAPN 75 PGTERLAVLRKAADIERNLDVFAEVLVMNAGKPKS-AAVGEVKAADVRLRLAELDLKKIGGDYIPGDWTYDTLETEGLV 153
AL1A1 SHEEP 81 DASERGRLLNKLADLIERDRLLLLTEAMNGGKLFSNAYLMDLGCCIKLRYCAGWADKIQGRTIPMDGN-FFTYTR--- 156

170 180 190 200 210 220 230 240
P. torridus 133 -KVBYGIVVAITPWNNFAAMVIRKLAPALLTGNTVILKPSSSDTPLTAEWLVKKFVDAGIPKGVLNLITGKGSEIGDYIVE 212
T. acidophilum 135 -KVBYGIVVALTPWNNFAGMVARKLAPALLTGNTVVLKPSSSDTPGSAEWIVRKFVEAGVPKGVLNFITGRGSEIGDYIVE 214
T. volcanium 135 -KVBYGTVVALTPWNNFAGMVARKLAPALLTGNTVILKPSSSDTPGSAEWIVKKFIQAGVPKGALNYITGRGSVIGDYIVE 214
F. acidarmanus 133 -KVBYGIVVAITPWNNFAAMVVRKLAPALLTGNTVILKPSSSDTPLTANWIVNKFIEAGIPKGVLNFVTGRGSEIGDYIVN 212
T. tenax_GAPN 154 RREPLGVVAAITPFNYPFLDAVNKITYSFIYGNAVVVKPSISDBLPAMAAMVKALLDAGPPPDAIALLLNLPG-KEAEKIVA 232
AL1A1 SHEEP 156 -SEPVGVCGQIIIPWNFBLLMFLWKIGPALSCGNTVVVKPAEQTPLTALHMGSLIKEAGPPPGVVNIVPGYGPPTAGAAISS 235

250 A 260 270 B 280 290 300 C 310 320
P. torridus 213 HKKVSILITMTGSTSTGQRIMEKAS-KNMAKLIIELGGKAPFMVWKDANIERALKSLVWAKYLNVGQSCIAAERLIYIHEDI 291
T. acidophilum 215 HKKVNLLITMTGSTATGQRIMQKAS-ANMAKLIIELGGKAPFMVWKDADMNALKTLLWAKYWNAAGQSCIAAERLIYVHEDI 293
T. volcanium 215 HKKVALITMTGSTSTGQRIMQKAS-ANMAKLMIELGGKAPFMVWKDADINNALKTLLWAKYWNAAGQSCIAAERLIYVHEDI 293
F. acidarmanus 213 HKKVGILVTMTGSTSTGQKIMQNAS-SNMAKLIIELGGKAPFIVWKDADMIRALKSLIWAKYLNVGQSCIAAERLIYIHEDI 291
T. tenax_GAPN 233 DDRVAAVSF TGSTEVEGERRVKVGG---VKQYVMELGGGDPAILVEDADLDIAADKIARGIYSYAGQRCDAIKLVLAERP 309
AL1A1 SHEEP 236 HMDVDKVAFTGSTEEVGKLIKEAAGKSNLKRVSELGGKSPCIVFADADLDNAVEFAHQGVFYHQGQQCIAASRLFVEESI 315

330 340 350 360 370 380 390 400
P. torridus 292 YDEFMKKFIEVSKRIKLGDPES--DMGPLINKSAVETTEKYVDIARSSGYKILLGGKKPELSGKYKN-GYFYEPTTIEN 368
T. acidophilum 294 YDTFMSRFVELSRKLAGDPKNA--DMGPLINKGALQATSEIVEEAKESGAKILFGGSQPSLSGPYRN-GYFFLPTTIGN 370
T. volcanium 294 YDTFINKFVELSKKIVVGDP SRA--DMGPLINTTALKGTEEEVVEEAKESGARILYGGSKPDLSGPYKN-GYFFLPTTIDN 370
F. acidarmanus 292 YDKFMSKFVAITKKLTTGDP DTS--DVGPLINKNALAGMVQTNNAKDSGYKILTGGNTPDLQGKYKN-GYFFAPTIIEN 368
T. tenax_GAPN 310 YGKLVEEVAKRLSSLRVGDP RDPTVDVGPLISPAVDEMMAAIEDAVEKGGRVLAGG---------RRLGPTYVQPTLVEA 381
AL1A1 SHEEP 316 YDEFVRRSVERAKKYVLGNELTPGVSQGPQIDKEQYEKILDLLIESGKKEGAKLECGG---------GPWGNKGYFIQPTVFS 389

410 420 430 440 450 460 470 480
P. torridus 369 VPQ---DSPLFQEEIREGPVIGAESVSSVDELYEAKANDSKYGLASYLFTEDPELIFEASEKIRFGELEYVN-MPGPEASQGY 444
T. acidophilum 371 ADQ---KSKIFQEEIFAPVIGARKISSVEEMYDLANDSKYGLASYLFTKDPNIIFEASERIRFGELEYVN-MPGPEASQGY 446
T. volcanium 371 VDQ---KSKIFQEEIFAPVIGARKVSDVEEMYSLANDSKYGLASYLFTKDPNLIFEAAERIRFGELEYIN-MPGPEASQGY 446
F. acidarmanus 369 VPQ---DSGLFQDEIREGPPIIGTEKISGVDEMYRKAANDSKYGLASYLFTEDPELMLEASEKIRFGELEYIN-MPGPEASQGY 444
T. tenax_GAPN 382 PADRVKDMVLYKREVAPVASAVEVKDLDQAIELANGRPYGGLDAAVEGRDVVVKIRRAVRLLEVGAIYINDMPRHGIGYYP 461
AL1A1 SHEEP 390 VTD---DMRIAKEEIFRGPVQQIMKFKSLDDVIKRANNTFYGLSAGIFTNDIDKAITVSSALQS GTVWVN-CYSVVSAQCP 465

490 500 510 520
P. torridus 445 HTGFRLTGQAGEGSKYGISEYLKLKNVYVDYSRGNLSISTVNDDLFKNL 493
T. acidophilum 447 HTGFRMTGQAGEGSKYGISEYLKLKNIVVDYSGKPLHINTVRDDLFQ-- 493
T. volcanium 447 HTGFRLTGQAGEGSKYGILEYLKLKNIVVDYSGKPLQINTVRDDLFQ-- 493
F. acidarmanus 445 HTGFRMTGQAGEGSKYGINEYLKLKNVYVDYSKKALYINTINNSLFDNL 493
T. tenax_GAPN 462 FGGRKKS GVFREGIGYAVEAVTAYKTIVFNYKGKGVWKYE-------- 501
AL1A1 SHEEP 466 FGFKMSGNGRELGEYGFFEYTEVKTVTIKISQKNS---------- 501

Fig. 3. Multiple sequence alignment of amino acid sequences of GADHs from Thermoplasmales, GAPN from *Thermoproteus tenax* and ALDH1A1 from sheep. The alignment was generated with ClustalX using the gonnet matrix. Box A: essential residues of the modified Rossmann fold (GxTxxG); Box B: Aldehyde dehydrogenase glutamic acid active site (PROSITE Pattern PS00687); Box C: Aldehyde dehydrogenase cysteine active site (PROSITE Pattern PS00070). NCBI Accession numbers: *Picrophilus torridus* DSM 9790 AAT42917.1; *Thermoplasma acidophilum* CAC11938.1; *Thermoplasma volcanium* GSS1NP_111540.1; *Ferroplasma acidarmanus* Ferl EAM94231.1; *Thermoproteus tenax* CAA71651.1; AL1A1 SHEEP P51977.

The following features characterize GADHs from Thermoplasmales as a novel, distinct enzyme family within the ALDH superfamily. (i) GADHs show a high sequence identity to each other (70–85%) and a significant lower identity (25–35%) to members of all other ALDH families. (ii) GADHs display a unique substrate specificity being highly specific for glyceraldehyde as aldehyde substrate and for NADP⁺ as electron acceptor. In contrast, ALDH families from eukaryotes, e.g., ALDH1 and ALDH3 use a wide range of aldehydes and are specific for NAD⁺ (ALDH1) or use both, NAD⁺ and NADP⁺ (ALDH3) as cofactors [23]. Members of the SSDH family accept both NADP⁺ (*E. coli*) [17] or NAD⁺ (human) [18], and members of the aldA family catalyze the NAD⁺ specific oxidation of lactaldehyde and few other aldehydes. Members of the two GAPN families are highly specific for glyceraldehyde-3-phosphate as aldehyde substrate and use either NADP⁺ (Bacteria and Eukarya) or NAD⁺ (the archaeon *T. tenax*) as cofactors. (iii) GADHs form a distinct phylogenetic

M. Reher, P. Schönheit / FEBS Letters 580 (2006) 1198–1204

![Diagram](attachment:diagram.png)

Fig. 4. Phylogenetic relationship of GADH with selected members of the ALDH superfamily from Eukarya, Bacteria and Archaea. Characterized enzymes are underlined. The numbers at the nodes are bootstrapping values according to neighbor-joining (generated by using the neighbor-joining algorithm of ClustalX [25]). NCBI accession numbers: aldA: *Escherichia coli* K12 AAC74497.1; *Klebsiella pneumoniae* ABA00458.1; *Neisseria meningitidis* MC58 AAF42297.1; GADH: *Picrophilus torridus* DSM 9790 AAT42917.1; *Ferroplasma acidarmanus* Fer1 EAM94231.1; *Thermoplasma acidophilum* CAC11938.1; *Thermoplasma volcanium* GSS1 NP_111540.1; SSDH: *Escherichia coli* K12 AAC75708.1; *Pseudomonas aeruginosa* PAO1 NP_248956.1; *Azotobacter vinelandii* AvOP ZP_00417203.1; *Homo sapiens* CAD20883.2; ALDH1: *Homo sapiens* AAH69274.1; SHEEP P51977; *Mus musculus* AAH46315.1; GAPN: *Streptococcus pneumoniae* R6 AAK99832.1; *Streptococcus pyogenes* MGAS5005 AAZ51737.1; *Streptococcus agalactiae* A909 YP_329569.1; *Streptococcus mutans* Q59931; PEA, *Pisum sativum* P81406; NICPL *Nicotiana plumbaginifolia* P93338; *Clostridium perfringens* str. 13 BAB82144.1; *Sulfolobus solfataricus* P2 NP_344502.1; *Pyrococcus furiosus* DSM3638 NP_578484.1; *Thermococcus kodakarensis* KOD1 YP_183118.1; *Aeropyrum pernix* K1 NP_148165.1; *Sulfolobus tokodaii* str. 7 NP_378478.1; *Thermoproteus tenax* CAA71651.1; ALDH3: ALDH3A1 *Homo sapiens* NP_000682.3; ALDH3B1 *Homo sapiens* P43353; ALDH3B2 *Homo sapiens* P48448. The scale bar corresponds to 0.1 substitutions per site. Characterized enzymes are underlined.

cluster, clearly separated from all ALDH families as indicated by high bootstrap support.

In summary, the GADHs from Thermoplasmatales are the first characterized functional glyceraldehyde dehydrogenases in a sugar degradation pathway i.e. as key enzymes of the non-phosphorylative ED pathway in Archaea. Besides Thermoplasmatales, *Sulfolobus* species have also been proposed to involve a non-phosphorylative route of the ED pathway for glucose degradation (see in [7]). Since no homologs of GADH are found in these species the oxidation of glyceraldehyde has to be catalyzed by a different enzyme; a molybdenum-containing glyceraldehyde oxidoreductase has been proposed for *Sulfolobus acidocaldarius* [24].

Acknowledgement: The authors thank Michael Bott (Jülich, Germany) for performing MALDI-TOF analysis of *P. torridus* GADH and Garo Antranikian (TU Hamburg-Harburg, Germany) for providing a growing culture of *Picrophilus torridus*.

References

[1] Schleper, C., Puehler, G., Holz, I., Gambacorta, A., Janekovic, D., Santarius, U., Klenk, H.P. and Zillig, W. (1995) *Picrophilus* gen. nov., fam. nov.: a novel aerobic, heterotrophic, thermoacidophilic genus and family comprising archaea capable of growth around pH 0. J. Bacteriol. 177, 7050–7059.

[2] Fütterer, O., Angelov, A., Liesegang, H., Gottschalk, G., Schleper, C., Schepers, B., Dock, C., Antranikian, G. and Liebl, W. (2004) Genome sequence of *Picrophilus torridus* and its implications for life around pH 0. Proc. Natl. Acad. Sci. USA 101, 9091–9096.

[3] van de Vossenberg, J.L.C.M., Driessen, A.J., Zillig, W. and Konings, W.N. (1998) Bioenergetics and cytoplasmic membrane stability of the extremely acidophilic, thermophilic archaeon *Picrophilus oshimae*. Extremophiles 2, 67–74.

[4] Serour, E. and Antranikian, G. (2002) Novel thermoactive glucoamylases from the thermoacidophilic Archaea *Thermoplasma acidophilum*, *Picrophilus torridus* and *Picrophilus oshimae*. Antonie Van Leeuwenhoek 81, 73–83.

[5] Segerer, A. and Stetter, K.O. (1992) The genus Thermoplasma in: The Prokaryotes (Balows, A., Trüper, H.G., Dworkin, M., Harder, W. and Schleifer, K.H., Eds.), pp. 712–718, Springer-Verlag, New York.

[6] Budgen, N. and Danson, M.J. (1986) Metabolism of glucose via a modified Entner-Doudoroff pathway in the thermoacidophilic archaeabacterium *Thermoplasma acidophilum*. FEBS Lett. 196, 207–210.

[7] Siebers, B. and Schönheit, P. (2005) Unusual pathways and enzymes of central carbohydrate metabolism in Archaea. Curr. Opin. Microbiol. 8, 695–705.

[8] Angelov, A., Futterer, O., Valerius, O., Braus, G.H. and Liebl, W. (2005) Properties of the recombinant glucose/galactose dehydrogenase from the extreme thermoacidophile, *Picrophilus torridus*. FEBS J 272, 1054–1062.

[9] Hurwitz, J. and Weissbach, A. (1959) The Formation of 2-keto-3-deoxyheptonic acid in extracts of *Escherichia coli* B; II. Enzymic studies. J. Biol. Chem. 234, 710–712.

[10] Schaffer, S., Weil, B., Nguyen, V.D., Dongmann, G., Gunther, K., Nickolaus, M., Hermann, T. and Bott, M. (2001) A high-resolution reference map for cytoplasmic and membrane-associated proteins of *Corynebacterium glutamicum*. Electrophoresis 22, 4404–4422.

[11] Ruepp, A., Graml, W., Santos-Martinez, M.L., Koretke, K.K., Volker, C., Mewes, H.W., Frishman, D., Stocker, S., Lupas, A.N. and Baumeister, W. (2000) The genome sequence of the thermoacidophilic scavenger *Thermoplasma acidophilum*. Nature 407, 508–513.

[12] Vasiliou, V., Bairoch, A., Tipton, K.F. and Nebert, D.W. (1999) Eukaryotic aldehyde dehydrogenase (ALDH) genes: human polymorphisms, and recommended nomenclature based on divergent evolution and chromosomal mapping. Pharmacogenetics 9, 421–434.

[13] Sophos, N.A. and Vasiliou, V. (2003) Aldehyde dehydrogenase gene superfamily: the 2002 update. Chem. Biol. Interact. 143–144, 5–22.

[14] Moore, S.A., Baker, H.M., Blythe, T.J., Kitson, K.E., Kitson, T.M. and Baker, E.N. (1998) Sheep liver cytosolic aldehyde dehydrogenase: the structure reveals the basis for the retinal specificity of class 1 aldehyde dehydrogenases. Structure 6, 1541–1551.

[15] Pohl, E., Brunner, N., Wilmanns, M. and Hensel, R. (2002) The crystal structure of the allosteric non-phosphorylating glyceraldehyde-3-phosphate dehydrogenase from the hyperthermophilic archaeum *Thermoproteus tenax*. J. Biol. Chem. 277, 19938–19945.

[16] Lutke-Eversloh, T. and Steinbüchel, A. (1999) Biochemical and molecular characterization of a succinate semialdehyde dehydrogenase involved in the catabolism of 4-hydroxybutyric acid in *Ralstonia eutropha*. FEMS Microbiol. Lett. 181, 63–71.

[17] Bartsch, K., von Johnn-Marteville, A. and Schulz, A. (1990) Molecular analysis of two genes of the *Escherichia coli gab* cluster: nucleotide sequence of the glutamate:succinic semialdehyde transaminase gene (*gabT*) and characterization of the succinic semialdehyde dehydrogenase gene (*gabD*). J. Bacteriol. 172, 7035–7042.

[18] Chambliss, K.L., Caudle, D.L., Hinson, D.D., Moomaw, C.R., Slaughter, C.A., Jakobs, C. and Gibson, K.M. (1995) Molecular cloning of the mature NAD⁺-dependent succinic semialdehyde dehydrogenase from rat and human. cDNA isolation, evolutionary homology, and tissue expression. J. Biol. Chem. 270, 461–467.

[19] Baldomà, L. and Aguilar, J. (1987) Involvement of lactaldehyde dehydrogenase in several metabolic pathways of *Escherichia coli* K12. J. Biol. Chem. 262, 13991–13996.

[20] Iddar, A., Valverde, F., Serrano, A. and Soukri, A. (2003) Purification of recombinant non-phosphorylating NADP-dependent glyceraldehyde-3-phosphate dehydrogenase from *Streptococcus pyogenes* expressed in *E. coli*. Mol. Cell Biochem. 247, 195–203.

[21] Habenicht, A. (1997) The non-phosphorylating glyceraldehyde-3-phosphate dehydrogenase: biochemistry, structure, occurrence and evolution. Biol. Chem. 378, 1413–1419.

[22] Ahmed, H., Ettema, T.J., Tjaden, B., Geerling, A.C., van der O.J. and Siebers, B. (2005) The semi-phosphorylative Entner-Doudoroff pathway in hyperthermophilic archaea – a re-evaluation. Biochem. J..

[23] Perozich, J., Kuo, I., Lindahl, R. and Hempel, J. (2001) Coenzyme specificity in aldehyde dehydrogenase. Chem. Biol. Interact. 130–132, 115–124.

[24] Kardinahl, S., Schmidt, C.L., Hansen, T., Anemüller, S., Petersen, A. and Schäfer, G. (1999) The strict molybdate-dependence of glucose-degradation by the thermoacidophile *Sulfolobus acidocaldarius* reveals the first crenarchaeotic molybdenum containing enzyme – an aldehyde oxidoreductase. Eur. J. Biochem. 260, 540–548.

[25] Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F. and Higgins, D.G. (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res. 25, 4876–4882.
