Drug Metab. Pharmacokin. 21(1): 1-18 (2006).

Review

# Multiplicity of Mammalian Reductases for Xenobiotic Carbonyl Compounds

Toshiyuki MATSUNAGA, Shinichi SHINTANI and Akira HARA*
Laboratory of Biochemistry, Gifu Pharmaceutical University, Gifu, Japan

Full text of this paper is available at http://www.jstage.jst.go.jp/browse/dmpk

Summary: A variety of carbonyl compounds are present in foods, environmental pollutants, and drugs. These xenobiotic carbonyl compounds are metabolized into the corresponding alcohols by many mammalian NAD(P)H-dependent reductases, which belong to the short-chain dehydrogenase/reductase (SDR) and aldo-keto reductase superfamilies. Recent genomic analysis, cDNA isolation and characterization of the recombinant enzymes suggested that, in humans, the six members of each of the two superfamilies, i.e., total of 12 enzymes, are involved in the reductive metabolism of xenobiotic carbonyl compounds. They comprise three types of carbonyl reductase, dehydrogenase/reductase (SDR family) member 4, 11β-hydroxysteroid dehydrogenase type 1, L-xylulose reductase, two types of aflatoxin B₁ aldehyde reductase, 20α-hydroxysteroid dehydrogenase, and three types of 3α-hydroxysteroid dehydrogenase. Accumulating data on the human enzymes provide new insights into their roles in cellular and molecular reactions including xenobiotic metabolism. On the other hand, mice and rats lack the gene for a protein corresponding to human 3α-hydroxysteroid dehydrogenase type 3, but instead possess additional five or six genes encoding proteins that are structurally related to human hydroxy-steroid dehydrogenases. Characterization of the additional enzymes suggested their involvement in species-specific biological events and species differences in the metabolism of xenobiotic carbonyl compounds.

Key words: Carbonyl reduction; short-chain dehydrogenase/reductase superfamily; aldo-keto reductase superfamily; carbonyl reductase; hydroxysteroid dehydrogenase; species difference

## Introduction

Aldehydes, ketones and quinones are present in a diverse range of natural and synthetic compounds to which living organisms are exposed. In addition, carbonyl compounds are formed through biological transformation of endogenous components and xenobiotics that are ingested. Aldehydes are chemically reactive and interact with the nucleophilic centers of nucleic acids and proteins. α-Dicarbonyl compounds, such as methyl glyoxal and diacetyl, are more reactive. Ketones are less reactive, and many drugs contain keto group(s). Quinones have a toxic effect, i.e., quinone-induced oxidative stress, when they are reduced through single-electron transfer to the corresponding semiquinones. Organisms have evolved several enzyme systems for detoxifying reactive carbonyl compounds. Such well established pathways include the oxidation of aldehydes to the corresponding carboxylic acids by aldehyde dehydrogenases and aldehyde oxidases, and the reduction of aldehydes and ketones into the corresponding alcohols by NADPH-dependent reductases with broad substrate specificity. Some NADPH-dependent reductases reduce quinones through two-electron transfer to the corresponding hydroquinones, which is also mediated by NA(D)PH:quinone reductase. This group of reductases with broad substrate specificity for xenobiotic carbonyl compounds was originally called ‘aldo-keto reductases’¹⁾ and/or ‘carbonyl reductases’.²⁾ Subsequently, according to the accumulated knowledge on the functions and structures of the reductases, most carbonyl-reducing enzymes have been grouped into two distinct protein families, the short-chain dehydrogenase (SDR)³⁾ and aldo-keto reductase (AKR)⁴⁾ superfamilies. There had been three excellent reviews on the carbonyl-reducing enzymes by 2000.⁵⁻⁷⁾ Over the recent five years, several new enzymes that may reduce xenobiotic carbonyl compounds have been found on genomic analysis, and thus the enzyme names have been changed. For example, well-known carbonyl reductase (CBR), a member of the SDR superfamily, is now named CBR1 according to the Human Gene Nomenclature.

---

Received; September 30, 2005, Accepted; October 24, 2005  
*To whom correspondence should be addressed: Akira HARA, Laboratory of Biochemistry, Gifu Pharmaceutical University, Mitahora-higashi, Gifu, Gifu 502-8585, Japan. Tel. & Fax. +81-58-237-8586, E-mail: hara@gifu-pu.ac.jp
Table 1. Human enzymes involved in the reduction of xenobiotic carbonyl compounds.

| Enzymes                | Gene name (location)       | Other names                                                                 | Endogenous substrates                                      | Subcellular localization | Accession number^a^ |
|-----------------------|---------------------------|-----------------------------------------------------------------------------|------------------------------------------------------------|--------------------------|---------------------|
| SDR family            |                           |                                                                             |                                                            |                          |                     |
| CBR 1                 | CBR 1 (21q22.13)          | Carbonyl reductase, PG 9-ketoreductase                                       | PG, isatin, ketosteroids                                   | Cytoplasm                | P16152              |
| CBR 3                 | CBR 3 (21q22.2)           |                                                                             | Unknown                                                    | Cytoplasm                | O75828              |
| 11βHSD 1              | HSD11B 1 (1q32-q41)       | Corticosteroid 11β-dehydrogenase isozyme 1                                  | 11-Ketoglucocorticoids                                     | Microsomes               | P28845              |
| DHRS 4                | Dhrs 4 (14q11.2)          | Peroxisomal short-chain alcohol dehydrogenase, NADPH-dependent retinol dehydrogenase/reductase (NDRD) | Retinal                                                     | Peroxisomes              | Q9BTZ2              |
| L-Xylulose reductase  | DXCR (17q25.3)            | Dicarbonyl/L-xylulose reductase, diacetyl reductase                          | L-Xylulose, diacetyl                                       | Cytoplasm                | Q7Z4W1              |
| AKR family            |                           |                                                                             |                                                            |                          |                     |
| AKR7A 2               | AKR7A 2 (1p35.1-p36.23)   | Aflatoxin B₁ aldehyde reductase member 2, AFAR 2                            | Succinic semialdehyde                                      | Golgi                    | O43488              |
| AKR7A 3               | AKR7A 3 (1p35.1-p36.23)   | Aflatoxin B₁ aldehyde reductase member 3, AFAR 1                            | Unknown                                                    | Cytoplasm                | O95154              |
| AKR1C 1               | AKR1C 1 (10q15-q14)       | 20α-HSD, 3(20)α-HSD, DD 1                                                  | 3-and 20-ketosteroids                                      | Cytoplasm                | Q04828              |
| AKR1C 2               | AKR1C 2 (10q15-q14)       | 3α-HSD type 3, DD 2, bile acid-binding protein                              | 3-Ketosteroids                                             | Cytoplasm                | P52895              |
| AKR1C 3               | AKR1C 3 (10q15-q14)       | 3α-HSD type 2, 17β-HSD type 5, PGF synthase, DDX                             | 3-, 17-and 20-keto-steroids, PGD₂                          | Cytoplasm                | P42330              |
| AKR1C 4               | AKR1C 4 (10q15-q14)       | 3α-HSD type 1, DD 4, chlordecone reductase                                  | 3-ketosteroids                                             | Cytoplasm                | P17516              |

^a^ The structural and functional information is available under the accession numbers of UniprotKB/Swiss-Prot (http://kr.expasy.org/sprot/).

Mammalian Carbonyl-Reducing Enzymes

Table 2. Typical xenobiotic substrates of human reductases.

| Enzyme       | Drugs                                                                                                      | Others                                                                                                                                         |
|--------------|-----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
| CBR 1        | Daunorubicin, doxorubicin, haloperidol, bromperidol, metyrapone, loxoprofen, wortmannin, dolasetron              | Quinones, aromatic aldehydes, aromatic ketones, NNK                                                                                         |
| CBR 3        |                                                                                                           | Menadione                                                                                                                                      |
| 11βHSD 1     | Ketoprofen, metyrapone, insecticidal metyrapone analogues, oracin                                              | NNK, menadione, aromatic aldehydes and ketones, 7-ketocholesterol                                                                            |
| DHRS 4       |                                                                                                           | Dicarbonyl compounds with aromatic rings, alkyl phenyl ketones, some aromatic aldehydes and ketones                                             |
| L-Xylulose   |                                                                                                           | Dicarbonyl compounds, some aromatic aldehydes and ketones                                                                                    |
| reductase    |                                                                                                           |                                                                                                                                                |
| AKR7A 2      | Daunorubicin, ethacrynic acid                                                                              | Aflatoxin B₁ dialdehyde, dicarbonyl compounds, aromatic aldehydes                                                                            |
| AKR7A 3      |                                                                                                           | Aflatoxin B₁ dialdehyde, 9, 10-phenanthrenequinone, 4-nitrobenzaldehyde                                                                       |
| AKR1C 1      | Dolasetron, naloxone, naltrexone, oxycodone, oracin, befunolol, ketotifen, 10-oxonortriptyline, haloperidol,     | Aromatic aldehydes and ketones, quinones, dicarbonyl compounds, NNK, trans-dihydrodiols of aromatic hydrocarbons, alicyclic alcohols           |
|              | loxoprofen, acetohexamide, daunorubicin, ethacrynic acid                                                    |                                                                                                                                                |
| AKR1C 2      | Dolasetron, naloxone, naltrexone, oxycodone, oracin, befunolol, ketoprofen, ketotifen, 10-oxonortriptyline,     | Aromatic aldehydes and ketones, quinones, dicarbonyl compounds, NNK, trans-dihydrodiols of aromatic hydrocarbons, alicyclic alcohols           |
|              | haloperidol, loxoprofen, acetohexamide, daunorubicin                                                        |                                                                                                                                                |
| AKR1C 3      | Naloxone, naltrexone                                                                                      | 9, 10-Phenanthrenequinone, 4-nitrobenzaldehyde, trans-dihydrodiols of aromatic hydrocarbons, alicyclic alcohols                                 |
| AKR1C 4      | Dolasetron, naloxone, naltrexone, oxycodone, oracin, ketoprofen, loxoprofen, acetohexamide, ethacrynic acid,     | Aromatic aldehydes and ketones, quinones, dicarbonyl compounds, NNK, trans-dihydrodiols of aromatic hydrocarbons, alicyclic alcohols           |
|              | metyrapone, chlordecone                                                                                  |                                                                                                                                                |

Table 3. Homologs of human enzymes in other mammalian species.

| Human enzyme | Mouse          | Rat            |
|--------------|----------------|----------------|
|              | Gene name (CH) | Gene name (CH) |
|              | Enzyme name    | Enzyme name    |
|              | Accession (SI) | Accession (SI) |
| CBR 1        | Cbr1(16)       | Cbr1(11)       |
|              | CBR 1          | CBR 1          |
|              | P48758(86%)    | P47727(85%)    |
| *c)          | Cbr2(11)       | Unknown        |
|              | CBR 2          | (Non-inducible CBR) |
|              | (Mouse lung CBR) |                |
| CBR 3        | Cbr3(16)       | Cbr3_predicted |
|              | CBR 3          | CBR 3          |
|              | Q8K354(85%)    | XP_22164(85%)  |
|              |                | (11)           |
| CBR 4        | BC009118(8)    | Cbr4(16)       |
|              | CBR 4          | CBR 4          |
|              | NP_663570(86%) | Q7TS56(84%)    |
| 11βHSD 1     | Hsd11b1(1)     | Hsd11b1(13)    |
|              | 11βHSD 1       | 11βHSD 1       |
|              | P50172(78%)    | P16232(76%)    |
| DHRS 4       | Dhrs4(14)      | Dhr4(15)       |
|              | DHRS 4         | DHRS 4         |
|              | Q99LB2(81%)    | Q8VID1(81%)    |
|              | (NDRD)         | (NDRD)         |
| L-Xylulose   | Dcxr(11)       | Dcxr(Unknown)  |
| reductase    | L-Xylulose reductase | L-Xylulose reductase |
|              | Q91X52(84%)    | Q920P0(84%)    |
| AKR7A 2      | Akr7a5(4)      | RGD:620311(5)  |
|              | AKR7A 5        | AKR7A 4        |
|              | (AFAR 2)       | (AFAR 2)       |
|              | Q8CG76(88%)    | Q8CG45(87%)    |
| AKR7A 3      | Not exist      | Akr7a3(5)      |
|              |                | AKR7A 1        |
|              |                | (AFAR 1)       |
|              |                | P38918(81%)    |
| AKR1C 1      | Akrlc18(13)    | LOC171516(17)  |
|              | AKR1C 18       | AKR1C 8        |
|              | (20α-HSD)      | (20α-HSD)      |
|              | Q8K023(68%)    | P51652(71%)    |
| AKR1C 3      | Akrlc6(13)     | LOC364773(17)  |
|              | AKR1C 6        | RAKh           |
|              | (17β-HSD type 5) | (17β-HSD type 5) |
|              | P70694(69%)    | NP_001014240   |
|              |                | (73%)          |
| AKR1C 4      | Akrlc14(13)    | LOC191574(17)  |
|              | AKR1C 14       | AKR1C 9        |
|              | (3α-HSD)       | (3α-HSD)       |
|              | Q91WT7(67%)    | P23457(69%)    |

a) Most rat genes are named tentatively. CH, chromosome.
b) Accession: The number in UniprotKB/Swiss-Prot or amino acid sequence in NCBI database. SI: sequence identity with the relevant human enzyme.
c) The CBR 2 gene does not exist in the human genome.

aromatic hydrocarbons. ${ }^{9}, { }^{15}$ In human liver and placenta, CBR 1 acts as a major quinone reductase, but the enzymatic reduction of several o-quinones results in redox cycling of the quinones, leading to the generation of semiquinones and the superoxide anion. ${ }^{16}$ Thus,

CBR 1 seems to be involved in both the detoxification and intoxication of quinones. The endogenous substrates of CBR 1 are suggested to be PGs, some 3-ketosteroids and isatin, of which isatin is the best one, showing a high affinity and turnover number that are
compatible with those of xenobiotic o-quinones. ${ }^{17)}$ In addition, CBR1 catalyzes the reduction of the 4-keto, aldehyde and C=C of 4-oxonon-2-enal, a product of lipid peroxidation. ${ }^{18)}$ CBR1 homozygous null mice are nonviable, ${ }^{19)}$ suggesting that the enzyme plays a non-redundant role in cell signaling during embryogenesis and development. A study on the biological function of CBR1 in A549 adenocarcinoma cells suggested that the enzyme is involved in serum-free-induced apoptosis in the cells. ${ }^{13)}$ On the other hand, several studies involving tumor tissue specimens have indicated that a decrease in CBR1 expression is correlated with the degree of dedifferentiation in hepatocellular carcinomas, ${ }^{20)}$ poor survival and lymph node metastasis in epithelial ovarian cancer, ${ }^{21)}$ and tumor progression and angiogenesis in lung cancer. ${ }^{22)}$ Therefore, elucidation of the mechanisms underlying the suggested roles of the enzyme in cell signaling, apoptosis and cancer progression is an important future goal.

Human CBR1 reduces several therapeutic agents and toxicologically important compounds (Table 2). The drug substrates include daunorubicin and doxorubicin, ${ }^{6)}$ loxoprofen, ${ }^{23)}$ metyrapone, ${ }^{23)}$ haloperidol, ${ }^{24)}$ bromperidol, ${ }^{25)}$ timiperone, ${ }^{26)}$ and wortmannin, ${ }^{27)}$ which has been proposed to be a potential antineoplastic agent. Since daunorubicinol, the reduced product of antiproliferative daunorubicin, is cardiotoxic, CBR1 is thought to be responsible for the severe cardiotoxicity associated with daunorubicin treatment. ${ }^{6)}$ This is supported by the finding that mice heterozygous for a null allele of CBR1 show reduced sensitivity to anthracycline-induced cardiotoxicity. ${ }^{19)}$ The enzyme also reduces 4-methylnitrosamino-1-(3-pyridyl)-1-butanone (NNK), a tobacco-specific potent carcinogen, into (S)-4-methylnitrosamino-1-(3-pyridyl)-1-butanol, which is more tumorigenic. The significance of the NNK metabolism by CBR1 and other reductases was described in a recent review by Maser and Breyer-Pfaff. ${ }^{28)}$

Other animal enzymes that have similar sequences to human CBR1 are classified as CBR1s. The genes for CBR1s of mouse and rat have been identified through genomic analyses (Table 3). In the mouse genome, an additional gene (accession no.: LOC435489) encoding a protein exhibiting 87% sequence identity with mouse CBR1 is predicted. Similarly, two cDNAs for gonadotropin-inducible and non-inducible CBR1 isoforms (86% sequence identity) have been isolated from rat ovary. ${ }^{29)}$ The inducible rat CBR is encoded by the genomic CBR1 gene, and identical to a testicular CBR that was previously characterized. ${ }^{30)}$ The non-inducible isoform might correspond to another form of the two rat testicular CBRs, ${ }^{31)}$ although the gene for this isoform was not found on the present genomic sequencing. Two cDNAs for such CBR1 isoforms have also

been isolated from rabbit liver ${ }^{32)}$ and Chinese hamster ovary cells. ${ }^{33)}$ Thus, these animal species probably have two genes for CBR1 isoforms, in contrast to the existence of only one gene for human CBR1. The substrate specificities of the CBR1s of these animals and pig ${ }^{34)}$ are essentially identical to that of the human enzyme, except that pig CBR1 exhibits $20 \beta$-HSD activity. Mouse and Chinese hamster CBR1s, similar to the human enzyme, are distributed in many tissues, ${ }^{33,35)}$ whereas the rat enzyme is specifically expressed in reproductive tissues and the adrenal gland. ${ }^{31)}$

CBR2: This enzyme is not present in human tissues, as its gene has not been found in the human genome. However, CBR2 is highly expressed in the mitochondria of epithelial cells of mouse, guinea-pig and pig lungs. ${ }^{36-38)}$ In addition, mitochondrial CBR2, which was previously known as a sperm protein, P26h, is present in testis and epididymis of hamster. ${ }^{39)}$ As a gene for CBR2 is not found in the rat genome (Table 3), CBR2 protein has not been detected in rat tissues. CBR2s are homotetramers composed of 26 kDa-subunits, and exhibit low sequence identity with monomeric CBR1. The enzymes that reduce various aliphatic, alicyclic and aromatic carbonyl compounds with lower $K_{\mathrm{m}}$ values are activated by fatty acids and inhibited by pyrazole, a known inhibitor of alcohol dehydrogenase. The differences between hamster CBR2 and the lung enzymes of other species are in the cofactor specificity and the reversibility of the reaction. The hamster enzyme shows a cofactor preference for NAD(H) and efficiently catalyzes the oxidation of 5$\alpha$-dihydrosterosterone, whereas the lung enzymes utilize NADP(H) as a preferred cofactor and do not exhibit significant dehydrogenase activity. The tissue distribution and catalytic properties suggest that lung CBR2s function in the detoxication of carbonyl compounds derived through lipid peroxidation and xenobiotic carbonyl compounds ingested from the airways and blood. ${ }^{36-38)}$ The hamster CBR2 is thought to act as a $3 \alpha$-HSD to control the intracellular concentration of a potent androgen, 5$\alpha$-dihydrosterosterone, during spermatogenesis. ${ }^{39)}$ A crystallographic study of mouse CBR2 has provided insights into its unique tetrameric structure and cofactor binding mode. ${ }^{40)}$ The key residues in the cofactor specificity were also identified by site-directed mutagenesis studies of mouse and hamster CBR2s. ${ }^{39,41)}$

CBR3: The gene for this enzyme was first identified 62 kilobases apart from the CBR1 gene on chromosome 21. ${ }^{42)}$ The encoded protein is composed of 277 amino acids, and its sequence identity with human CBR1 is 71% and that with animal CBR2s is less than 23%. Recently, a natural allelic variant of CBR3 (V244M) was found, and the kinetic constants for NADPH and menadione of the recombinant CBR3 and V244M variant were determined. ${ }^{43)}$ The mRNA for CBR3 is
suggested to be expressed in various tissues, 33) although
the expression level is much lower than that of CBR1
(personal communication from Dr. T. Terada). cDNA
microarray analysis in patients with keloid showed that
the CBR3 gene, together with eight other genes, in
keloid tissue is consistently upregulated.44) The CBR3
gene has been identified or predicted in the mouse and
rat genomes (Table 3). In Chinese hamster, CBR3 (86%
sequence identity with human CBR3) exhibits high
activity toward daunorubicin and isatin compared to
CBR1, but conversely lacks oxidoreductase activity
toward PGs, which are substrates for CBR1.33) The
mRNA for CBR3 is detected in the order of kidney > brain > liver in Chinese hamster, which differs
from the expression patterns of the mRNAs for CBR1
and its isoform. Thus, reports on CBR3 have been few,
and the concentrations of CBR3 proteins in human and
animal tissues are unknown. Further studies on the
enzymology and gene regulation of CBR3 are necessary
to elucidate its roles in the metabolism of endogenous
and exogenous compounds, and in the pathogenesis of
diseases.

CBR4: The gene for this enzyme was predicted on
recent genomic analyses, and is located on human chromosome 4 (4q32.3). The encoded protein (accession no.
Q8N4T8) is composed of 237 amino acids, and exhibits
low sequence identity (<25%) with other types of CBR.
The CBR4 gene is present in laboratory animals
(Table 3) and dog (accession no.: XP_534547), suggesting a role in the metabolism of endogenous compounds
for CBR4. Although the enzyme is named carbonyl or
carbonic reductase 4, its enzymatic properties and tissue
distribution remain unknown.

2. 11β-HSD type 1 (HSD11B1, EC 1.1.1.146)
11β-HSD is a microsomal protein that catalyzes the
interconversion of active cortisol into inactive cortisone.
Two isoforms, NADP(H)-dependent HSD11B1 and
NAD(H)-dependent 11β-HSD type 2 (HSD11B2), in
mammalian tissues have been cloned and characterized.
HSD11B1 is expressed in a wide range of tissues, acts
predominantly as a reductase in intact cells and tissues
by regenerating active cortisol from cortisone, and
regulates glucocorticoid access to the glucocorticoid
receptor. HSD11B2 is mainly expressed in mineralocorticoid target tissues such as kidney and colon, acts only
as a dehydrogenase by producing inactive cortisone, and
protects the mineralocorticoid receptor from high levels
of receptor-active cortisol. The roles and enzymology of
the two isoforms were described in recent reviews.45,46)
Increases in the activity and expression of HSD11B1
have been implicated in the pathogenesis of many
common conditions including obesity, insulin
resistance, the metabolic syndrome, the polycystic
ovarian syndrome, osteoporosis and glaucoma.47)
Therefore, selective HSD11B1 inhibition has been

proposed as a novel therapeutic strategy for many of
these conditions. Mammalian HSD11B1s exhibit high
sequence identity, and the crystal structures of guinea-
pig and mouse HSD11B1s were recently solved.48,49)
Selective and potent HSD11B1 inhibitors will be
developed in the near future through structure-based
drug design using the crystal structures.
HSD11B1 also plays a role in the metabolism of
non-glucocorticoid carbonyl compounds, i.e., oracin,
ketoprofen, metyrapone, insecticidal metyrapone
analogues, NNK, menadione, and some aromatic
aldehydes and ketones.50-52) On the reduction of NNK,
the human enzyme forms both the (S)- and (R)-enantiomers of 4-methylaminosamino-1-(3-pyridyl)-1-butanol.
This enantioselectivity is in contrast to those of human
CBR1 and other AKR1C isoforms, which form the
(S)-enantioemer on the reduction.28) Recently, HSD11B1
was reported to efficiently catalyze the reduction of 7-
ketocholesterol, the major dietary oxysterol, into 7-
hydroxycholesterol, species-specific differences in the
stereospecific reduction being observed between the rat,
human, and hamster enzymes.53) The human and rat
enzymes reduce 7-ketocholesterol into 7β-hydroxy-
cholesterol, whereas the hamster enzyme catalyzes the
interconversion of 7-ketocholesterol into both 7α-
and 7β-hydroxycholesterol. Although mammalian
HSD11B1s show similar substrate specificities for
xenobiotic compounds, the stereoselective reduction
may differ depending with the species and/or the
reduced compound.

3. Dehydrogenase/reductase (SDR family) member
4 (DHRS4, EC 1.1.1.-)
The cDNA for human DHRS4 was first cloned as a
peroxisomal 2,4-dienoyl-CoA reductase-related protein.54) DHRS4 belongs to the SDR family, and has a
C-terminal peroxisomal targeting signal 1 (Ser-Arg-
Leu). It has also been called SDR-SRL, peroxisomal
short-chain dehydrogenase and NADPH-dependent
retinol dehydrogenase/reductase (NDRD), because the
rabbit and pig homologues of human DHRS4 exhibit
NADPH-linked retinal reductase activity.55) In addition,
two alternative splicing forms of human DHRS4, SDR-
SRL1 (lacking residues 118–204) and SDR-SRL-2
(lacking 85–204), have been isolated.54) DHRS4, SDR-
SRL1 and SDR-SRL-2 are now designated as DHRS4
isoforms 1, 2 and 3, respectively. Since there has been
no literature on the properties of human DHRS4, we
characterized the recombinant DHRS4 isoform 1. It is
an NADPH-dependent tetrameric reductase for
xenobiotic carbonyl compounds, but differs from the
enzymes of other species (described below) in its low
activity toward retinoids and instability at low temperature. The human enzyme reduces all-trans-retinal with a
low Vmax/Km (0.8 units/mg/mM), and does not exhibit
significant 9-cis-retinol reductase or all-trans-retinol
dehydrogenase activity. The enzyme is gradually inactivated at 4°C, in contrast to the stable enzymes of other species. The cold inactivation might have prevented detection of the enzyme activity in human tissues in previous studies on CBRs. The mRNA for DHRS4 isoform 1 is ubiquitously expressed in human tissues, in which expression of the mRNA for the isoform 2 is low. No enzymatic activity is observed for recombinant isoform 2. Thus, the physiological role of human DHRS4 is not clear, but the enzyme reduces alkyl phenyl ketones and might be involved in the metabolism of the butyrophenone type of drugs.

The cDNAs for other mammalian DHRS4s have been cloned and their genes are shown in Table 3. The rabbit and pig DHRS4s are peroxisomal homotetramers, being composed of 27-kDa subunits.⁵⁵) The enzymes reduce alkyl phenyl ketones, α-dicarbonyl compounds, retinals and some aromatic aldehydes using NADPH as a cofactor, and are identical to a tetrameric CBR that was previously purified from rabbit heart.⁵⁶) Of these xenobiotic substrates, 9,10-phenanthrenequinone is efficiently reduced by the enzymes, and the reduction results in the formation of reactive oxygen species (ROS).⁵⁷) The enzymes show high catalytic efficiency for all-trans-retinal and 9-cis-retinal (the $V_{\text{max}}/K_{\text{m}}$ values of the pig enzyme are 618 and 30 units/mg/mM, respectively, at 25°C and pH 7.4), and oxidize all-trans-retinol with a low $V_{\text{max}}/K_{\text{m}}$ value of 1.5 units/mg/mM.⁵⁵) Mouse DHRS4 also exhibits high reactivity towards retinoids, and its content in the liver increases with clofibrate feeding.⁵⁸) In the three animals, mRNA for DHRS4 is expressed in many tissues, of which liver and kidney exhibit the highest expression. We have cloned a cDNA for DHRS4 from dog liver, and characterization of the recombinant enzyme revealed that it is identical to a high-molecular weight CBR that is highly expressed in the liver.⁵⁹)

### 4. L-Xylulose reductase (EC 1.1.1.10)
L-Xylulose reductase catalyzes the NADPH-dependent reduction of L-xylulose to xylitol in the uronate cycle of glucose metabolism. The human, mouse, rat, guinea pig and hamster enzymes have been cloned and shown to be identical with diacetyl reductase (EC 1.1.1.5) that reduces various α-dicarbonyl compounds including o-quinones, and aromatic aldehydes and ketones.⁶⁰,⁶¹) Because of its broad substrate specificity, L-xylulose reductase is thought to be involved in the metabolism of xenobiotic carbonyl compounds, although drugs containing carbonyl groups have not been examined as substrates. The enzyme is a homotetramer, and exhibits high sequence identity of 65% with mouse CBR2. The crystal structure of human L-xylulose reductase is similar to that of mouse CBR2,⁶²) but there is a significant difference in their substrate binding clefts. L-Xylulose reductase also differs from

CBR2 in inhibitor sensitivity and tissue distribution. L-Xylulose reductase is inhibited by n-butyric acid, but not by pyrazole. The enzyme is distributed in many tissues, of which the liver and kidney show high expression of the enzyme. Furthermore, the L-xylulose reductases of mouse, rat, guinea pig and hamster are rapidly dissociated into their inactive dimeric forms at low temperature.⁶¹,⁶³)

#### Carbonyl-reducing Enzymes in the AKR Superfamily

The AKR superfamily is a rapidly growing group of NAD(P)(H)-dependent oxidoreductases that metabolize carbohydrates, steroids, prostaglandins, and other endogenous aldehydes and ketones, as well as xenobiotic compounds.⁴) Currently there are more than a hundred known members of this superfamily that is classified into 14 families. The nomenclature system is similar to that for the cytochrome P450 superfamily, but, unlike that system, it involves amino acid sequence comparisons. Within a given family, subfamilies are defined according to >60% identity in amino acid sequence among subfamily members. The largest family, AKR1, is subdivided into five subfamilies: AKR1A, mammalian aldehyde reductases; AKR1B, mammalian aldose reductases; AKR1C, HSDs and PGF synthases; AKR1D, Δ⁴-3-ketosteroid-5β-reductases; and AKR1E, mouse keto-reductase. More information on this family is available on the AKR superfamily homepage (www.med.upenn.edu/akr). Among the enzymes in this superfamily, aldose reductases, aldehyde reductases, HSDs, PGF synthases and aflatoxin B₁ reductases (AFARs, belonging to the AKR7A subfamily) exhibit broad substrate specificities for xenobiotic carbonyl compounds. In this section, we describe recent studies on the characteristics of HSDs, PGF synthases and AFARs, focusing mainly on their roles in xenobiotic metabolism. Aldehyde and aldose reductases are not included in this section, because they are not involved in the reduction of drug ketones, except for daunorubicin and acetohexamide.²³) A recent review described the properties and functions of aldose reductase and closely related aldo-keto reductases,⁶⁴) of which AKR1B10, i.e., human aldose reductase-like protein, has been recognized as a new diagnostic marker of hepatocellular carcinomas⁶⁵) and smokers’ non-small cell lung carcinomas.⁶⁶) Since the mouse and rat genomes contain many genes for the enzymes belonging to the AKR1C subfamily compared with the human genome, we divide the enzymes into human and rodent ones in order to avoid more confusion.

##### 1. Enzymes in the AKR7A subfamily
AFAR catalyzes the NADPH-dependent reduction of aflatoxin B₁ dialdehyde, a toxic metabolite produced from aflatoxin B₁ by CYP3A, into unreactive mono- and di-alcohol derivatives.⁶⁷) The human, mouse and ratenzymes have been cloned and characterized, and their genes are shown in Tables 1 and 3. The enzymes are dimers, and show similar broad substrate specificities for various aromatic aldehydes and dicarbonyl compounds. Of these substrates, 2-carboxybenzaldehyde is useful as a diagnostic model substrate for AFAR, because it is an inactive or poor substrate for other reductases in the SDR and AKR families.⁶⁸) In human and rat tissues, AFAR is present in two isoforms, which have been classified into two classes, AFAR1 and AFAR2, with respect to sequence similarity, affinity for succinic semialdehyde, intracellular localization and inducibility by xenobiotics.

AFAR1: This class of AFAR includes human AKR7A3⁶⁹) and rat AKR7A1,⁷⁰) the subunits of which are composed of 327 amino acids and exhibit 81% sequence identity. The rat enzyme reduces various aldehydes, but its \(K_m\) value for succinic semialdehyde, an endogenous substrate, is high.⁷¹) The enzyme is expressed in liver and several extrahepatic tissues, and is induced by phenolic antioxidants, ethoxyquin, coumarin, and dietary indoles and isothiocyanates.⁷⁰,⁷²,⁷³) Modulation of the level of AKR7A1 is also suggested to correlate with the carcinogenesis caused by aflatoxin B₁.⁷⁰,⁷²) In rat liver, AKR7A1 exists as both a homodimer and a heterodimer with a subunit of AKR7A4 belonging to another class of AFAR. The crystal structure of AKR7A1 revealed the details of the dimeric structure and substrate recognition.⁷⁴,⁷⁵) Little is known about the catalytic activity and inducibility of human AKR7A3, which is reported to reduce some substrates (Table 2).⁶⁹)

AFAR2: This class includes human AKR7A2,⁷⁶) mouse AKR7A5,⁷⁷) and rat AKR7A4,⁷⁸) the subunits of which are composed of 359–367 amino acids and exhibit high sequence identity (>87%) with each other. The enzymes are constitutively expressed in many tissues, in which they are thought to be associated with the Golgi apparatus.⁷⁹) Their substrate specificities are similar to that of AFAR1, but AFAR2 shows a low \(K_m\) value for succinic semialdehyde.⁶⁸,⁷⁶⁻⁷⁹) Human AKR7A2 is indeed identical to succinic semialdehyde reductase,⁸⁰) indicating its physiological role in γ-hydroxybutyrate synthesis. Human AKR7A2 reduces various xenobiotic aromatic aldehydes and α-dicarbonyl compounds, but is essentially inactive towards aliphatic and aromatic ketones.⁶⁸,⁷⁶) Although the human enzyme reduces daunorubicin and ethacrynic acid at low rates,⁶⁸) other drugs have not been examined as substrates.

2. Human enzymes in the AKR1C subfamily

Four human enzymes, AKR1C1-AKR1C4, have been cloned and characterized. The genes exist in a cluster on chromosome 10 (Table 1). Although two genes, AKR1CL1 and AKR1CL2, for AKR1C-like proteins have been found near the cluster, their gene functions are unknown: The AKR1CL1 gene codes for a hypothetical polypeptide of 129-amino acids (accession no.: Q6ZN81), and the AKR1CL2 gene encodes a monomeric protein of 320-amino acids (accession no.: Q9BU71) that exhibits low reductase activity only toward 9,10-phenanthrenequinone.⁸¹) AKR1C1-AKR1C4 are NADP(H)-dependent monomeric dehydrogenases composed of 323 amino acids and exhibit high sequence identity (>83%), but their biochemical properties are different. The enzymes are 3α-, 17β-and/or 20α-HSDs, and play roles in the pre-receptor regulation of steroid receptors, nuclear orphan receptors and/or membrane-bound ligand-gated ion channels. The enzymes are suggested to be targets for the treatment of prostate cancer, breast cancer, endometriosis and endometrial cancer.⁸²) The other characteristic of the enzymes differing from those of the members of the AKR superfamily is that they show high dihydrodiol dehydrogenase activity (DD, EC 1.3.1.20), which oxidizes the trans-dihydrodiols of aromatic hydrocarbons into the corresponding catechols. Therefore, AKR1C1, AKR1C2, AKR1C3 and AKR1C4 are called DD1, DD2, DDX and DD4, respectively. DD was first shown to be a detoxification enzyme in the metabolism of carcinogenic polycyclic aromatic hydrocarbons, but has been regarded as a toxification enzyme, because oxidation by the enzyme yields reactive and redox-active o-quinones and ROS.⁸³) In addition, AKR1C1-AKR1C4 metabolize PGs and a number of xenobiotic compounds. In this section, we summarize the main characteristics of the individual enzymes and describe their roles in xenobiotic carbonyl metabolism.

AKR1C1: This enzyme is known as 20α-HSD (EC 1.1.1.149), and is distributed in many tissues.⁸⁴⁻⁸⁶) AKR1C1 exhibits additional 3α- and 3β-HSD activities depending on the steroid structure: It reduces 3-keto-5β-dihydrosteroids into the 3α-hydroxy derivatives, whereas 3β-hydroxy products are formed on the reduction of 3-keto-5α-dihydrosteroids.⁸⁷,⁸⁸) The 5α-androstane-3β,17β-diol and 5α,3β-tetrahydroxypr-egnanes produced are ligands of estrogen receptor β⁸⁹) and antagonists for the γ-aminobutyric acid type A (GABA_A) receptor,⁹⁰,⁹¹) respectively. Thus, the enzyme plays important roles in the steroid metabolism through the formation of the above steroids as well as inactivation of progesterone, neuroactive 5α/β-pregnan-3α-ol-20-ones (positive modulators of the GABA_A receptor), and 5α-dihydrotestosterone.⁸²) AKR1C1 differs from AKR1C2 (3α-HSD type 3) by only seven amino acids, of which the residue at position 54 (Leu in AKR1C1 and Val in AKR1C2) has been demonstrated to be a determinant of the steroid specificity of the two enzymes in site-directed mutagenesis and crystallographic studies.⁸⁸,⁹²,⁹³)

AKR1C1 reduces a variety of non-steroid carbonyl compounds including PGs and NNK. The enzyme
Table 4. Comparison of kinetic constants for xenobiotic substrates among human enzymes.

| Substrate    | CBR1       | HSD11B1      | AKR1C1      | AKR1C2      | AKR1C4      |
|--------------|------------|--------------|-------------|-------------|-------------|
|              | $K_{\mathrm{m}}$ | $V_{\max } / K_{\mathrm{m}}$ | $K_{\mathrm{m}}$ | $V_{\max } / K_{\mathrm{m}}$ | $K_{\mathrm{m}}$ | $V_{\max } / K_{\mathrm{m}}$ | $K_{\mathrm{m}}$ | $V_{\max } / K_{\mathrm{m}}$ | $K_{\mathrm{m}}$ | $V_{\max } / K_{\mathrm{m}}$ |
| NNK          | 7          | 400          | 12          | 20          | 0.2         | 90          | 0.3        | 57          | 8          | 10         |
| Metyrapone   | 0.9        | 534          | ND          | NS          | NS          | 19          | 35         |
| R-Ketotifen  | NS         | NT           | 0.01        | 500         | 0.008       | 7250        | NS         |
| S-Ketotifen  | NS         | NT           | 0.05        | 1840        | 0.004       | 25000       | NS         |
| Dolasetron   | NS         | NT           | 0.06        | 3280        | 0.03        | 900         | 0.2        | 2570        |
| Naloxone     | NS         | NT           | 0.4         | 41          | 0.02        | 570         | 0.006      | 12400       |
| Naltrexone   | NS         | NT           | 0.4         | 59          | 0.1         | 210         | 0.01       | 6440        |
| Oxycodone    | NS         | NT           | 1.8         | 17          | NS          | 0.4         | 79         |

$K_{\mathrm{m}}=$ mM and $V_{\max } / K_{\mathrm{m}}=$ units/mg/M (1 unit = nmol of reduced product or NADPH/min).

ND: the activity was detected, but the kinetic constants have not been determined.
NS: no significant activity was detected. NT: not tested as a substrate.

The values for naloxone, naltrexone and oxycodone were determined at pH 7.4 and 37°C.
Thus, the enzyme is involved in the metabolism of androgens and PGs, and is suggested to be involved in the development of prostate cancer by forming testosterone and the differentiation of leukemia cells caused by metabolizing PGD₂. AKR1C13 is significantly up-regulated during the differentiation of HL-60 cells by all-trans-retinoic acid and vitamin D₃, and its mRNA expression in human fibroblasts is also induced by thyroid hormone. Recently, three crystal structures of the enzyme were determined to elucidate the mechanism underlying the multiple specificity and to develop specific inhibitors. Such several inhibitors selective for AKR1C1, AKR1C2 and AKR1C3 have been reported.

Only 9,10-phenanthrenequinone and 4-nitrobenzaldehyde have been reported to be xenobiotic substrates of AKR1C3. We tested several drugs as substrates, and found that naloxone and naltrexone are reduced by the enzyme (unpublished results). The Kₘ and Vₘₐₓ/Kₘ values for naloxone are 1.3 mM and 2.6 units/mg/M, respectively, the respective values for naltrexone being 0.6 mM and 54 units/mg/M at 37°C and pH 7.4. Comparison of these values with those for other enzymes (Table 4) suggests that AKR1C3 is not a major enzyme in drug metabolism.

AKR1C4: This enzyme is 3α-HSD type 1, and exhibits higher affinity and velocities for most 3-keto and 3α-hydroxysteroids than the other types of the enzyme do. It also exhibits low 20α-HSD and 17β-HSD activities, and is inhibited by phenolphthalein and steroidal anti-inflammatory agents. Crystallization of AKR1C4 has not been achieved. This enzyme is expressed specifically in the liver, and is involved in the catabolism of circulating steroid hormones and the metabolism of bile acids. The liver-specific expression of AKR1C4 mRNA is regulated by transcription factors, hepatocyte nuclear factor (HNF)-4α/γ, HNF-1α and variant HNF-1.

As AKR1C4 was first cloned as a cDNA for chlordecone (an insecticide) reductase, it reduces various xenobiotic carbonyl compounds and drugs (Table 2). AKR1C4 efficiently reduces naloxone, naltrexone and oxycodone, although it is almost inactive towards befunolol, daunorubicin, haroperidol and ketotifen (Table 4). The enzyme probably plays an important role in the reduction of these three drugs and chlordecone. On the other hand, several drugs such as sulfofomophthalein, clofibric acid derivatives and anti-inflammatory 2-arylpropionic acids enhance the enzyme activity of AKR1C4. While there has been no report on the induction of AKR1C4 by xenobiotics so far, an interindividual difference in hepatic chlordecone reductase activity has been reported. A large interindividual difference has also been noted in the hepatic expression of AKR1C4 mRNA, which is related to the

differences in the amounts of HNF-1α, HNF-4α and HNF-4γ. In addition, a variant of AKR1C4 (S145C/L311V) that decreases the enzyme activity has been identified. Such studies on interindividual differences are needed for the other AKR1C enzymes, because they are involved not only in the metabolism of other drugs, but also in several diseases.

3. Rodent enzymes in the AKR1C subfamily NADP(H)-dependent 20α-, 17β- and 3α-HSDs, which correspond to human AKR1C1, AKR1C3 and AKR1C4, respectively, have been cloned and characterized from mouse and rat tissues (Table 3). The rodent enzymes exhibit the following differences in substrate specificity and tissue distribution from the relevant human enzymes, although data on rat aldo-ketoreductase h (RAKh) have not been published. (1) Mouse AKR1C18 and rat AKR1C8 are 20α-HSDs, exhibiting additional 3α/3β-HSD activity that is observed with human AKR1C1. In contrast to the high DD activity of human AKR1C1, the rodent enzymes show low DD activity, and significantly high reductase activity toward nonsteroidal substrates such as α-dicarboxyl compounds, aromatic aldehydes and ketones. In addition, the tissue distributions of the rodent enzymes differ from that of the ubiquitous human enzyme. Mouse AKR1C18 is highly expressed in the ovary and kidney, and at lower levels in many other tissues. AKR1C8 is ovary-specific in female rats and its expression in male rat tissues is quite low. (2) Mouse AKR1C6 and rat RAKh are 17β-HSDs that are associated with low 20α-HSD activity, and high DD and reductase activities toward a variety of nonsteroidal carbonyl compounds. Thus, they appear to be type 5 isozymes of 17β-HSD that correspond to human AKR1C3. However, the rodent enzymes do not exhibit 3α-HSD and PGF synthase activities, which are high for AKR1C3. AKR1C6 is a liver-specific enzyme, and RAKh is expressed in rat liver and kidney. This also differs from the ubiquitous tissue distribution of AKR1C3. (3) Mouse AKR1C14 and rat AKR1C9 are 3α-HSDs, exhibiting high reductase activities toward various nonsteroidal carbonyl compounds. In this respect, the rodent enzymes are counterparts of human AKR1C4, but they do not exhibit 20α-or 17β-HSD activity that is detected at low levels for the human enzyme. The rodent enzymes exhibit a ubiquitous tissue distribution, which is also different from the liver-specific expression of human AKR1C4. (4) There has been no report on a mouse or rat NADP(H)-dependent 3α-HSD that has similar properties to human AKR1C2.

Recent genomic analyses have shown that eight and nine genes for enzymes belonging to the AKR1C subfamily are present in the mouse and rat genomes, respectively. The proteins include the above rodent HSDs and partially characterized mouse aldo-keto reductases
Table 5. AKR1C enzymes specifically expressed in mouse and rat tissues.
The genes exist as a cluster on chromosomes 13 and 17 for the genomes of mouse and rat, respectively.

| Mouse AKR1C enzymes | Rat AKR1C enzymes |
| --- | --- |
| **Gene (product)** | **Characteristics** | **Gene (product)** | **Characteristics** |
| Akr1c12 (AKR1C12) | NAD(H)-preferring 17β-HSD with moderate DD activity and low 3(20)α-HSD activity. | LOC 36773 (AKR1C24) | Specificity for cofactors and substrates is essentially the same as that of AKR1C12. |
| Akr1c13 (AKR1C13) | NAD(H)-preferring DD, oxidizing alicyclic alcohols, cis-and trans-dihydrodiols, and nerol. It reduces dicarbonyl compounds and some aromatic aldehydes. | Akr1c12_predicted (AKR1C16) | Specificity for cofactors and substrates is essentially the same as that of AKR1C13. It slowly oxidizes some 3α-, 17β- and 20α-hydroxysteroids. |
| Akr1c19 (AKR1C19) | NADH-preferring reductase specific for dicarbonyl compounds. It slowly oxidizes 3-hydroxyhexobarbital. | LOC 307096 (RAKi) | NADH-preferring reductase for dicarbonyl compounds. |
| Akr1c21 (AKR1C21) | NADP(H)-preferring 3(17)α-HSD with DD activity towards cis-dihydrodiols. It reduces dicarbonyl compounds, and some aromatic aldehydes and ketones. | Akr1c21_predicted (RAKg) | Specificity for cofactors and substrates is essentially the same as that of AKR1C21. |
| Akr1c20 (AKR1C20) | NADPH-preferring reductase for dicarbonyl compounds with aromatic ring(s). It exhibits low 3α(17β)-HSD activity. | LOC 498790 (AKR1C17) | NAD(H)-preferring 3α-HSD for bile aids, 5β-prognanes, and 4-androstenes. It reduces dicarbonyl compounds and some aromatic aldehydes. |
|  |  | Akr1c11_predicted (AKR1C15) | NADPH-preferring reductase with broad specificity for various aliphatic and aromatic carbonyl compounds, quinones, and 17-ketoandrostanes. |

a) Products are named according to the nomenclature of the AKR superfamily. RAKi and RAKg are tentative names, because they are not assigned in this superfamily.

(AKR1C12 and AKR1C13), but the functions of the other enzymes have not been studied. We have isolated the cDNAs for AKR1C12, AKR1C13 and other functionally unknown enzymes from mouse and rat tissues, and have characterized the recombinant enzymes. Table 5 summarizes the genes for the enzymes and their characteristics, which include our unpublished results. The tissue distributions of the enzymes, and their relationship with previously isolated enzymes and proteins are presented below.

(1) Mouse AKR1C12 has the same sequence as AKRa, which is encoded by the interleukin-3-regulated gene. The rat counterpart exhibiting high sequence identity (90%) is AKR1C24, one form (TBER1) of the two reductases for 6-tert-butyl-2,3-epoxy-5-cyclohexene-1,4-dione, and may be identical to RAKf, which was previously isolated as a partial cDNA. The recombinant AKR1C12 and AKR1C24 utilize NAD(H) as a preferred cofactor, and efficiently oxidize various 17β-hydroxysteroids. In addition, they exhibit moderate DD activity, and low 3α- and 20α-HSD activities, as well as high reductase activity toward α-dicarbonyl compounds and aromatic aldehydes. As AKR1C12 was found as a stomach aldo-keto reductase, both AKR1C12 and AKR1C24 are highly expressed in the gastrointestinal tract and liver, and at low levels in other tissues of the two animals. (2) Mouse AKR1C13 is probably the same enzyme as NAD+-preferring DD, which is designated as AKR1C22 in the AKR superfamily, because they differ only by four amino acids. Its amino acid sequence is also highly similar to that of mouse morphine 6-dehydrogenase (EC 1.1.1.218), suggesting the identity of the two enzymes. The rat counterpart (92% sequence identity) is AKR1C16, which is encoded in the cDNA for RAKb. The properties of the recombinant AKR1C16 are essentially identical to those of mouse AKR1C13, except that the rat enzyme shows low dehydrogenase activity toward some 3α-, 17β- and 20α-hydroxysteroids. The tissue expression patterns of the mRNAs for AKR1C13 and AKR1C16 are similar to those of AKR1C12 and AKR1C24. In rat liver, the activity of AKR1C16 is lower than that of AKR1C24. Thus, the physiological roles of AKR1C13 and AKR1C16 remain unknown, although the enzymes may be involved in the oxidative metabolism of morphine. (3) Mouse AKR1C19 shows high NADH-linked reductase activity only toward dicarbonyl compounds, but its NAD+-linked dehydrogenase activity is low and directed toward some alcohol substrates. The enzyme is thought to act as a reductase based on the low inhibition of its reductase activity by NAD(P)+. The rat counterpart is RAKi (91% sequence identity), and the recombinant RAKi shows similar NADH-linked reductase activity and tissue distribution to AKR1C19. (4) Mouse AKR1C21 is a NADP(H)-dependent 3(17)α-HSD that is identical to DD purified from kidney. Although the enzyme is highly expressed in the kidney, its mRNA is detected in other tissues. The rat counterpart of AKR1C21 is RAKg, the sequence identity being 86%. NADP+-linked 3(17)α-HSD activity was also observed for recombinant RAKg, but the mRNA for RAKg is only expressed in liver and kidney, in which its alternative splicing form is detected. (5) AKR1C17 and AKR1C15 are rat-specific

enzymes. AKR1C17 has been identified as a kidney-
specific 3α-HSD that differs from ubiquitous NADP(H)-
dependent 3α-HSD (AKR1C9) with respect to its cofac-
tor preference for NAD(H) compared to NADP(H),
and substrate specificity for bile acids, 5β-pregnanes and
5α-androstanes. AKR1C15 is a NADPH-dependent
reductase, which is highly expressed in rat lung. It
reduces a variety of carbonyl compounds with low K_m
values (0.3–20 μM). In particular, the ability of the
enzyme to reduce aliphatic ketones and aldehydes is a
unique characteristic that is not observed for the other
human and rodent enzymes in the AKR7A and AKR1C
subfamilies. The broad substrate specificity of the
enzyme resembles that of mouse lung CBR2, and the
only difference being in the steroid specificity:
AKR1C15 reduces the 17-keto groups of 5α/5β-
androstanes, whereas mouse CBR2 reduces 3-
ketosteroids. Since the CBR2 gene has not been
identified in the rat genome and such a tetrameric CBR
is indeed absent in rat lung, AKR1C15 may play an
important role in the pulmonary metabolism of car-
bonyl compounds derived through lipid peroxidation,
and androgens and xenobiotics. (6) Mouse AKR1C20
shows 89% sequence identity with mouse 17β-HSD type
5 (AKR1C6), and its mRNA is expressed only in the
liver.¹³¹ The recombinant AKR1C20 exhibits low 17β-
HSD and moderate 3α-HSD activities using NADP(H)
as the preferred cofactors. However, the V_max/K_m values
for 3α-hydroxy- and 3-keto-steroids of AKR1C20
(0.01–20 units/mg/mM) are much lower than those of
mouse 3α-HSD, AKR1C14 (200–4860 units/mg/mM).
AKR1C20 shows high reductase activity towards α-
dicarbonyl compounds with aromatic ring(s), but does
not reduce aromatic and aliphatic ketones.

The genes for all the enzymes belonging to the
AKR1C subfamily exist as a cluster on chromosomes
10, 13 and 17 for the genomes of humans, mouse and
rat, respectively. This genomic organization suggests
that these genes arose from ancient duplication events
followed by divergence. As shown in a phylogenetic
analysis of the results of the multiple sequence align-
ment (Fig. 1), the enzymes form a monophyletic gene
family comprised of three separate clades: the rodent
3(17)α-HSD (AKR1C21 and RAKg) clade, the rodent
3α-HSDs and 20α-HSD clade, and the clade of other
enzymes. The phylogenetic tree reveals that initial
duplication of the ancestral gene common to the
AKR1C enzymes shared among the these clades, and the
ancestral gene was subsequently duplicated also, giving
rise to three clusters: the human enzymes, rodent
enzymes (AKR1C15, RAKh, AKR1C6 and AKR1C20)
and other rodent enzymes (AKR1C12–RAKi).

As described above, there are significant differences in
the multiplicity and tissue distribution of the enzymes
belonging to the AKR1C subfamily between humans

HsAKR1C4
HsAKR1C3
HsAKR1C1
HsAKR1C2
RnAKR1C15
RnRAKh
MmAKR1C6
MmAKR1C20
MmAKR1C12
MmAKR1C13
RnAKR1C24
RnAKR1C16
RnAKR1C17
MmAKR1C19
RnRAKi
RnAKR1C8
MmAKR1C18
RnAKR1C9
MmAKR1C14
MmAKR1C21
RnRAKg

Fig. 1. Phylogenetic relationship among the human, rat and mouse
members of the AKR1C subfamily. Cluster analysis of the human
(Hs), rat (Rn) and mouse (Mm) enzymes were performed using Clustal
W¹⁴⁵ and the tree was drawn with the TreeView program.¹⁴⁶

and laboratory animals. Although the substrate specific-
ities for ketone-containing drugs of all the enzymes have
not been studied comparatively, such a difference
between the human and rodent enzymes may be respon-
sible for the species difference in drug metabolism. In
addition, the occurrence of many HSDs belonging to
the AKR1C subfamily in mouse and rat tissues suggests
that the enzyme system to regulate cellular concentra-
tions of active steroids in laboratory animals is more
complex than that in humans. Human AKR1C1–
AKR1C4 are potential targets for the treatment of
several cancers, endometriosis and anxiety, and selective
inhibitors of the individual enzymes have been suggest-
ed to be new pharmaceutical agents for the treatment of
these diseases.⁸⁶ The multiplicity of the mouse and rat
enzymes should be considered when the effects of inhibi-
tors are examined using laboratory animals.

Conclusion

Recent genomic and cDNA analyses suggested that
the reductive metabolism of xenobiotic carbonyl com-
pounds is mediated by many enzymes that differ in
intracellular localization. The number of the human
enzymes is 12, which include uncharacterized CBR4.
Most enzymes function in the cellular metabolism of
their natural components, and convert toxic and lipid-soluble carbonyl compounds into less-reactive alcohols because of their broad substrate specificity. Thus, the omnipresent reductive potential must play an important part not only in normal cellular physiology, but also in the metabolism of ketone-containing drugs and xenobiotic carbonyl compounds ingested. The literature indicates that human CBR1, HSD11B1, AKR1C1, AKR1C2 and AKR1C4 are major reductases of ketone-containing drugs, but suggests that their specificity and catalytic efficiency for several therapeutic drugs are significantly different. In future studies on the metabolic fate of ketone-containing drugs, it will be necessary to comparatively examine the substrate specificities of all the enzymes, expecting roles of the newly-found enzymes, CBR3, CBR4, DHRS4 and L-xylulose reductase, in the metabolism.

In rats and mice, additional five or six reductases belonging to the AKR1C subfamily are suggested to be involved in the metabolism of xenobiotic carbonyl compounds. In addition, the tissue distributions of rodent 20α-HSDs, 17β-HSD type 5 isozymes and 3α-HSDs are significantly different from those of the corresponding human enzymes, AKR1C1, AKR1C3 and AKR1C4, which play important roles in pre-receptor regulation of steroid actions. Such additional enzymes are perhaps involved in the animal-specific cellular metabolism. This, together with the differences in the tissue distributions of the other HSDs, may be responsible for the difference in the metabolism of ketone-containing drugs between humans and laboratory animals.

### References

1) Bachur, N. R. Cytoplasmic aldo-keto reductases: A class of drug metabolizing enzymes. *Science*, **193**: 595–597 (1976).

2) Felsted, R. L. and Bachur, N. R.: Mammalian carbonyl reductases. *Drug Metab. Rev.*, **11**: 1–60 (1980).

3) Oppermann, U., Filling, C., Hult, M., Shafqat, N., Wu, X., Lindh, M., Shafqat, J., Nordling, E. Kallberg, Y., Persson, B. and Jörnvall, H.: Short-chain dehydrogenases/reductases (SDR): the 2002 update. *Chem. Biol. Interact.*, **143–144**: 247–253 (2003).

4) Hyndman, D., Bauman, D. R., Heredia, V. V., and Penning, T. M.: The aldo-keto reductase superfamily homepage. *Chem. Biol. Interact.*, **143–144**: 499–525 (2003).

5) Maser, E.: Xenobiotic carbonyl reduction and physiological steroid oxidoreduction. The pluripotency of several hydroxysteroid dehydrogenases. *Biochem. Pharmacol.*, **49**: 421–440 (1995).

6) Forrest, G. L. and Gonzalez, B.: Carbonyl reductase. *Chem. Biol. Interact.*. **129**: 21–40 (2000).

7) Oppermann, U. C. and Maser, E.: Molecular and structural aspects of xenobiotic carbonyl metabolizing enzymes. Role of reductases and dehydrogenases in xenobiotic phase I reactions. *Toxicology*, **144**: 71–81

8) Wermuth, B., Bohren, K. M., Heinemann, G., von Wartburg, J.-P. and Gabbay, K. H.: Human carbonyl reductase: nucleotide sequence analysis of a cDNA and amino acid sequence of the encoded protein. *J. Biol. Chem.*, **263**: 16185–16188 (1988).

9) Wermuth, B.: Purification and properties of an NADPH-dependent carbonyl reductase from human brain: Relationship to prostaglandin 9-ketoreductase and xenobiotic ketone reductase. *J. Biol. Chem.*, **256**: 1206–1213 (1981).

10) Wirth, H. and Wermuth, B.: Immunohistochemical localization of carbonyl reductase in human tissues. *J. Histochem. Cytochem.*, **40**: 1857–1863 (1992).

11) Lemieux, N., Malfoy, B. and Forrest, G. L.: Human carbonyl reductase (CBR) localized to band 21q22.1 by high-resolution fluorescence in situ hybridization displays gene dosage effects in trisomy 21 cells. *Genomics*, **15**: 169–172 (1993).

12) Forrest, G. L., Akman, S., Krutzik, S., Paxton, R. J., Sparkes, R. S., Doroshow, J., Felsted, R. L., Glover, C. J., Mohandas, T. and Bachur, N. R.: Induction of a human carbonyl reductase gene located on chromosome 21. *Biochim. Biophys. Acta*, **1048**: 149–155 (1990).

13) Tanaka, M., Bateman, R., Rauh, D., Vaisberg, E., Ramachandani, S., Zhang, C., Hansen, K. C., Burlingame, A. L., Trautman, J. K., Shokat, K. M. and Adams, C. L.: An unbiased cell morphology-based screen for new, biologically active small molecules. *PLoS Biol.*, **3**: e128 (2005).

14) Ghosh, D., Sawicki, M., Pletnev, V., Erman, M., Ohno, S., Nakajin, S. and Duax, W. L.: Porcine carbonyl reductase. Structural basis for a functional monomer in short chain dehydrogenases/reductases. *J. Biol. Chem.*, **276**: 18457–18463 (2001).

15) Wermuth, B., Platt, K. L., Seidel, A. and Oesch, F.: Carbonyl reductase provides the enzymatic basis of quinine detoxication in man. *Biochem. Pharmacol.*, **33**: 1277–1282 (1986).

16) Jarabak, J. and Harvey, R. G.: Studies on three reductases which have polycyclic aromatic hydrocarbon quinines as substrates. *Arch. Biochem. Biophys.*, **303**: 394–401 (1993).

17) Usami, N., Kitahara, K., Ishikura, S., Nagano, M., Sakai, S. and Hara, A.: Characterization of a major form of human isatin reductase and the reduced metabolite. *Eur. J. Biochem.*, **268**: 5755–5763 (2001).

18) Doorn, J. A., Maser, E., Blum, A., Claffey, D. J. and Petersen, D. R.: Human carbonyl reductase catalyzes reduction of 4-oxonon-2-enal. *Biochemistry*, **43**: 13106–13114 (2004).

19) Olson, L. E., Bedja, D., Alvey, S. J., Cardounel, A. J., Gabrielson, K. L. and Reeves, R. H.: Protection from doxorubicin-induced cardiac toxicity in mice with a null allele of carbonyl reductase 1. *Cancer Res.*, **63**: 6602–6606 (2003).

20) Suto, K., Kajihara-Kano, H., Yokoyama, Y., Hayashi, M., Kimura, J., Kumano, T., Takahata, T., Kudo, H. and Tsuchida, S.: Decreased expression of the perox-
isomal bifunctional enzyme and carbonyl reductase in
human hepatocellular carcinomas. J. Cancer Res. Clin.
Oncol., 125: 83–88 (1999).

21) Umemoto, M., Yokoyama, Y., Sato, S., Tsuchida, S.,
    Al-Mulla, F. and Saito, Y.: Carbonyl reductase as a
    significant predictor of survival and lymph node
    metastasis in epithelial ovarian cancer. Br. J. Cancer.,
    85: 1032–1036 (2001).

22) Takenaka, K., Ogawa, E., Oyanagi, H., Wada, H. and
    Tanaka, F.: Carbonyl reductase expression and its
    clinical significance in non-small-cell lung cancer.
    Cancer. Epidemiol. Biomarkers Prev., 14: 1972–1975
    (2005).

23) Ohara, H., Miyabe, Y., Deyashiki, Y., Matsuura, K.
    and Hara, A.: Reduction of drug ketones by
    dihydrodiol dehydrogenases, carbonyl reductase and
    aldehyde reductase of human liver. Biochem. Phar-
    macol., 50: 221–227 (1995).

24) Breyer-Pfaff, U. and Nill, K.: High-affinity stereoselec-
    tive reduction of the enantiomers of ketotifen and of
    ketonic nortriptyline metabolites by aldo-keto reduc-
    tases from human liver. Biochem. Pharmacol., 59:
    249–260 (2000).

25) Someya, T., Muratake, T., Hirokane, G., Shibasaki,
    M., Shimoda, K. and Takahashi, S.: Interindividual
    variation in bromperidol metabolism and relationship
    to therapeutic effects. J. Clin. Psychopharmacol., 20:
    175–180 (2000).

26) Shimoda, K., Shibasaki, M., Inaba, T., Cheung, S. W.,
    Someya, T. and Takahashi, S.: Carbonyl reduction of
    timiperone in human liver cytosol. Pharmacol.
    Toxicol., 83: 164–168 (1998).

27) Holleran, J. L., Fourcade, J., Egorin, M. J., Eiseman,
    J. L., Parise, R. A., Musser, S. M., White, K. D.,
    Covey, J. M., Forrest, G. L. and Pan, S. S.: In vitro
    metabolism of the phosphatidylinositol 3-kinase inhibi-
    tor, wortmannin, by carbonyl reductase. Drug Metab.
    Dispos., 32: 490–496 (2004).

28) Maser, E. and Breyer-Pfaff, U.: Competing roles of
    reductase in the detoxification of the tobacco-specific
    nitrosamine ketone NNK. In Penning T. M. and
    Petrash, J. M. (ed.): Aldo-keto Reductases and Tox-
    icant Metabolism, Washington DC, American Chemi-
    cal Society, 2004, pp. 67–82.

29) Aoki, H., Okada, T., Mizutani, T., Numata, Y.,
    Minegishi, T. and Miyamoto, K.: Identification of two
    closely related genes, inducible and noninducible
    carbonyl reductases in the rat ovary. Biochem.
    Biophys. Res. Commun., 230; 518–523 (1997).

30) Wermuth, B., Maeder-Heinemann, G. and Ernst, E.:
    Cloning and expression of carbonyl reductase from rat
    testis. Eur. J. Biochem., 228, 473–479 (1995).

31) Iwata, N., Inazu, N. and Satoh, T.: The purification
    and properties of NADPH-dependent reductases from
    rat ovary. J. Biochem. (Tokyo), 105: 556–564 (1989).

32) Gonzalez, B., Sapra, A., Rivera, H., Kaplan, W. D.,
    Yam, B. and Forrest, G. L.: Cloning and expression of
    the cDNA encoding rabbit liver carbonyl reductase.
    Gene, 154: 297–298 (1995).

33) Terada, T., Sugihara, Y., Nakamura, K., Mizobuchi,

H. and Maeda, M.: Further characterization of Chinese
hamster carbonyl reductases (CHCRs). Chem. Biol.
Interact., 143–144: 373–381 (2003).

34) Nakjin, S., Tamura, F., Takase, N. and Toyoshima,
S.: Carbonyl reductase activity exhibited by pig testicu-
lar 20β-hydroxysteroid dehydrogenase. Biol. Pharm.
Bull., 20: 1215–1218 (1997).

35) Ishikura, S., Yamamoto, Y., Matsuura, K., Wei, J.,
Hodes, M. E. and Hara, A.: Properties and tissue
distribution of mouse monomeric carbonyl reductase.
Biol. Pharm. Bull., 21: 879–881 (1998).

36) Nakayama, T., Yashiro, K., Inoue, Y., Matsuura, K.,
Ichikawa, H., Hara, A. and Sawada, H.: Characteriza-
tion of pulmonary carbonyl reductase of mouse and
guinea pig. Biochim. Biophys. Acta, 882: 220–227
(1986).

37) Oritani, H., Deyashiki, Y., Nakayama, T., Hara, A.,
Sawada, H., Bunai, Y. and Ohya, I.: Purification and
characterization of pig lung carbonyl reductase. Arch.
Biochem. Biophys., 292: 539–547 (1992).

38) Matsuura, K., Hara, A., Sawada, H., Bunai, Y. and
Ohya, I.: Localization of pulmonary carbonyl
reductase in guinea-pig and mouse: Enzyme histochem-
ical and immunochemical studies. J. Histochem.
Cytochem., 38: 217–223 (1990).

39) Ishikura, S., Usami, N., Kitahara, K., Isaji, T., Oda,
K., Nakagawa, J. and Hara, A.: Enzymatic characteri-
zation and subcellular distribution of a short-chain
dehydrogenase/reductase family protein, P26h, in
hamster testis and epididymis. Biochemistry, 40:
214–224 (2001).

40) Tanaka, N., Nonaka, T., Nakanishi, M., Deyashiki,
Y., Hara, A. and Mitsui, Y.: Crystal structure of the
ternary complex of mouse lung carbonyl reductase at
1.8 Å resolution: The structural origin of coenzyme
specificity in the short-chain dehydrogenase/reductase
family. Structure, 4: 33–45 (1996).

41) Nakanishi, M., Matsuura, K., Kaibe, H., Tanaka, N.,
Nonaka, T., Mitsui, Y. and Hara, A.: Switch of
coenzyme specificity of mouse lung carbonyl reductase
by substitution of threonine 38 with aspartic acid. J.
Biol. Chem., 272: 2218–2222 (1997).

42) Watanabe, K., Sugawara, C., Ono, A., Fukuzumi, Y.,
Itakura, S., Yamazaki, M., Tashiro, H., Osoegawa,
K., Soeda, E. and Nomura, T.: Mapping of a novel
human carbonyl reductase, CBR3, and ribosomal
pseudogenes to human chromosome 21q22.2. Genom-
ics, 52: 95–100 (1998).

43) Lakham, S. S., Ghosh, D. and Blanco, J. G.:
Functional significance of a natural allelic variant of
human carbonyl reductase 3 (CBR3). Drug Metab.
Dispos., 33: 254–257 (2005).

44) Na, G. Y., Seo, S. K., Lee, S. J., Kim, D. W., Kim, M.
K. and Kim, J. C.: Upregulation of the NNP-1 (novel
nuclear protein-1, D21S2056E) gene in keloid tissue
determined by cDNA microarray and in situ hybridiza-
tion. Br. J. Dermatol., 151: 1143–1149 (2004).

45) Tomlinson, J. W., Walker, E. A., Bujalska, I. J.,
Draper, N., Lavery, G. G., Cooper, M. S., Hewison,
M. and Stewart, P. M.: 11β-hydroxysteroid de-

hydrogenase type 1: a tissue-specific regulator of glucocorticoid response. *Endocr. Rev.*, **25**: 831–866 (2004).

46) Blum, A. and Maser, E.: Enzymology and molecular biology of glucocorticoid metabolism in humans. *Prog. Nucleic Acid Res. Mol. Biol.*, **75**: 173–216 (2003).

47) Tomlinson, J. W.: 11β-Hydroxysteroid dehydrogenase type 1 in human disease: a novel therapeutic target. *Minerva. Endocrinol.*, **30**: 37–46 (2005).

48) Ogg, D., Elleby, B., Norstrom, C., Stefansson, K., Abrahmsen, L., Oppermann, U. and Svensson, S.: The crystal structure of guinea pig 11β-hydroxysteroid dehydrogenase type 1 provides a model for enzyme-lipid bilayer interactions. *J. Biol. Chem.*, **280**: 3789–3794 (2005).

49) Zhang, J., Osslund, T. D., Plant, M. H., Clogston, C. L., Nybo, R. E., Xiong, F., Delaney, J. M. and Jordan, S. R.: Crystal structure of murine 11β-hydroxysteroid dehydrogenase 1: an important therapeutic target for diabetes. *Biochemistry*, **44**: 6948–6957 (2005).

50) Maser, E. and Oppermann, U. C.: Role of type-1 11β-hydroxysteroid dehydrogenase in detoxification processes. *Eur. J. Biochem.*, **249**: 365–369 (1997).

51) Wsol, V., Szotakova, B., Skalova, L. and Maser, E.: The novel anticancer drug oracin: different stereospecificity and cooperativity for carbonyl reduction by purified human liver 11β-hydroxysteroid dehydrogenase type 1. *Toxicology*, **197**: 253–261 (2004).

52) Hult, M., Nobel, C. S., Abrahmsen, L., Nicoll-Griffith, D. A., Jörnvall, H. and Oppermann, U. C.: Novel enzymological profiles of human 11β-hydroxysteroid dehydrogenase type 1. *Chem. Biol. Interact.*, **130–132**: 805–814 (2001).

53) Schweizer, R. A., Zurcher, M., Balazs, Z., Dick, B. and Odermatt, A.: Rapid hepatic metabolism of 7-ketocholesterol by 11β-hydroxysteroid dehydrogenase type 1: species-specific differences between the rat, human, and hamster enzyme. *J. Biol. Chem.*, **279**: 18415–18424 (2004).

54) Fransen, M., Van Veldhoven, P. P. and Subramani, S.: Identification of peroxisomal proteins by using M13 phage protein VI phage display: molecular evidence that mammalian peroxisomes contain a 2,4-dienoyl-CoA reductase. *Biochem. J.*, **340**: 561–568 (1999).

55) Usami, N., Ishikura, S., Abe, H., Nagano, M., Uebuchi, M., Kuniyasu, A., Otagiri, M., Nakayama, H., Imamura, Y. and Hara, A.: Cloning, expression and tissue distribution of a tetrameric form of pig carbonyl reductase. *Chem. Biol. Interact.*, **143–144**: 353–361 (2003).

56) Imamura, Y., Migita, T., Otagiri, M., Choshi, T. and Hibino, S.: Purification and catalytic properties of a tetrameric carbonyl reductase from rabbit heart, *J. Biochem.* (Tokyo), **125**: 41–47 (1999).

57) Shimada, H., Oginuma, M., Hara, A. and Imamura Y.: 9,10-Phenanthrenequinone, a component of diesel exhaust particles, inhibits the reduction of 4-benzoylpyridine and all-trans-retinal and mediates superoxide formation through its redox cycling in pig heart. *Chem.*

Res. Toxicol., **17**: 1145–1150 (2004).

58) Lei, Z., Chen, W., Zhang, M. and Napoli, J. L.: Reduction of all-trans-retinal in the mouse liver peroxisome fraction by the short-chain dehydrogenase/reductase RRD: induction by the PPAR α ligand clofibrate. *Biochemistry*, **42**: 4190–4196 (2003).

59) Hara, A., Nakayama, T., Deyashiki, Y., Kariya, K. and Sawada, H.: Carbonyl reductase of dog liver: Purification, properties, and kinetic mechanism. *Arch. Biochem. Biophys.*, **244**: 238–247 (1986).

60) Ishikura, S., Isaji, T., Usami, N., Kitahara, K., Nakagawa, J. and Hara, A.: Molecular cloning of a cDNA for hamster diacetyl reductase that is identical with L-xylulose reductase. *Chem. Biol. Interact.*, **130–132**: 879–889 (2001).

61) Nakagawa, J., Ishikura, S., Asami, J., Isaji, T., Usami, N., Hara, A., Sakurai, T., Tsuritani, K., Oda, K., Takahashi, M., Yoshimoto, M., Otsuka, N. and Kitamura, K.: Molecular characterization of mammalian dicarbonyl/L-xylulose reductase and its localization in kidney. *J. Biol. Chem.*, **277**: 17883–17891 (2002).

62) El-Kabbani, O., Carbone, V., Darmanin, C., Ishikura, S. and Hara, A.: Structure of the tetrameric form of human L-xylulose reductase: Probing the inhibitor-binding site with molecular modeling and site-directed mutagenesis. *Proteins*, **61**: 688–690 (2005).

63) Ishikura, S., Usami, N., El-Kabbani, O. and Hara, A.: Structural determinant for cold inactivation of rodent L-xylulose reductase. *Biochem. Biophys. Res. Commun.*, **308**: 68–72 (2003).

64) Petrush, J. M.: All in the family: aldose reductase and closely related aldo-keto reductases. *Cell Mol. Life Sci.*, **61**: 737–749 (2004).

65) Zeindl-Eberhart, E., Haraida, S., Liebmann, S., Jungblut, P. R., Lamer, S., Mayer, D., Jager, G., Chung, S. and Rabes, H. M.: Detection and identification of tumor-associated protein variants in human hepatocellular carcinomas. *Hepatology*, **39**: 540–549 (2004).

66) Fukumoto, S., Yamauchi, N., Moriguchi, H., Hippo, Y., Watanabe, A., Shibahara, J., Taniguchi, H., Ishikawa, S., Ito, H., Yamamoto, S., Iwanari, H., Hironaka, M., Ishikawa, Y., Niki, T., Sohara, Y., Kodama, T., Nishimura, M., Fukayama, M., Dosaka-Akita, H. and Aburatani, H.: Overexpression of the aldo-keto reductase family protein AKR1B10 is highly correlated with smokers’ non-small cell lung carcinomas. *Clin. Cancer Res.*, **11**: 1776–1785, (2005).

67) Guengerich, F. P., Cai, H., McMahon, M., Hayes, J. D., Sutter, T. R., Groopman, J. D., Deng, Z. and Harris, T. M.: Reduction of aflatoxin B₁ dialdehyde by rat and human aldo-keto reductases. *Chem. Res. Toxicol.*, **14**: 727–737 (2001).

68) O’connor, T., Ireland, L. S., Harrison, D. J. and Hayes, J. D.: Major differences exist in the function and tissue-specific expression of human aflatoxin B₁ aldehyde reductase and the principal human aldo-keto reductase AKR1 family members. *Biochem. J.*, **343**: 487–504 (1999).

69) Knight, L. P., Primiano, T., Groopman, J. D., Kensler, T. W. and Sutter, T. R.: cDNA cloning, expression and activity of a second human aflatoxin B₁-metabolizing member of the aldo-keto reductase superfamily, AKR7A3. *Carcinogenesis*, 20: 1215–1223 (1999).

70) Hayes, J. D., Judah, D. J. and Neal, G. E.: Resistance to aflatoxin B₁ is associated with the expression of a novel aldo-keto reductase which has catalytic activity towards a cytotoxic aldehyde-containing metabolite of the toxin. *Cancer Res.*, 53: 3887–3894 (1993).

71) Ellis, E. M. and Hayes, J. D.: Substrate specificity of an aflatoxin-metabolizing aldehyde reductase. *Biochem. J.*, 312 (Pt 2): 535–541 (1995).

72) Kelly, V. P., Ellis, E. M., Manson, M. M., Chanas, S. A., Moffat, G. J., McLeod, R., Judah, D. J., Neal, G. E. and Hayes, J. D.: Chemoprevention of aflatoxin B₁ hepatocarcinogenesis by coumarin, a natural benzopyrone that is a potent inducer of aflatoxin B₁-aldehyde reductase, the glutathione S-transferase A5 and P1 subunits, and NAD(P)H:quinone oxidoreductase in rat liver. *Cancer Res.*, 60: 957–969 (2000).

73) Bonnesen, C., Eggleston, I. M. and Hayes, J. D.: Dietary indoles and isothiocyanates that are generated from cruciferous vegetables can both stimulate apoptosis and confer protection against DNA damage in human colon cell lines. *Cancer Res.*, 61: 6120–6130 (2001).

74) Kozma, E., Brown, E., Ellis, E. M. and Lapthorn, A. J.: The Crystal structure of rat liver Akr7A1: A dimeric member of the aldo-keto reductase superfamily. *J. Biol. Chem.*, 277: 16285–16293 (2002).

75) Kozma, E., Brown, E., Ellis, E. M. and Lapthorn, A. J.: The high resolution crystal structure of rat liver AKR7A1: understanding the substrate specificities of the AKR7 family. *Chem. Biol. Interact.*, 143–144: 289–297 (2003).

76) Ireland, L. S., Harrison, D. J., Neal, G. E. and Hayes, J. D.: Molecular cloning, expression and catalytic activity of a human AKR7 member of the aldo-keto reductase superfamily: evidence that the major 2-carboxybenzaldehyde reductase from human liver is a homologue of rat aflatoxin B₁-aldehyde reductase. *Biochem. J.*, 332: 21–34 (1998).

77) Hinshelwood, A., McGarvie, G. and Ellis, E.: Characterization of a novel mouse liver aldo-keto reductase AKR7A5. *FEBS Lett.*, 523: 213–218 (2002).

78) Kelly, V. P., Ireland, L. S., Ellis, E. M. and Hayes, J. D.: Purification from rat liver of a novel constitutively expressed member of the aldo-keto reductase 7 family that is widely distributed in extrahepatic tissues. *Biochem. J.*, 348: 389–400 (2000).

79) Kelly, V. P., Sherratt, P. J., Crouch, D. H. and Hayes, J. D.: Novel homodimeric and heterodimeric rat γ-hydroxybutyrate synthases that associate with the Golgi apparatus define a distinct subclass of aldo-keto reductase 7 family proteins. *Biochem. J.*, 366: 847–861 (2002).

80) Schaller, M., Schaffhauser, M., Sans, N. and Wermuth, B.: Cloning and expression of succinic

81) Nishinaka, T., Azuma, Y., Ushijima, S., Miki, T. and Yabe-Nishimura, C.: Human testis specific protein: a new member of aldo-keto reductase superfamily. *Chem. Biol. Interact.*, 143–144: 299–305 (2003).

82) Bauman, D. R., Steckelbroeck, S., Penning, T. M.: The roles of aldo-keto reductases in steroid hormone action. *Drug News Perspect.*, 17: 563–578 (2004).

83) Penning, T. M.: Aldo-keto reductases and formation of polycyclic aromatic hydrocarbon o-quinones. *Method Enzymol.*, 378: 31–67 (2004).

84) Shiraishi, H., Matsuura, K., Kume, T. and Hara, A.: Expression of mRNA for dihydrodiol dehydrogenase isoforms in human tissues. *Adv. Exp. Biol.*, 463: 539–544 (1999).

85) Nishizawa, M., Nakajima, T., Yasuda, K., Kanzaki, H., Sasaguri, Y., Watanabe, K. and Ito, S.: Close kinship of human 20α-hydroxysteroid dehydrogenase gene with three aldo-keto reductase genes. *Genes Cells*, 5: 111–125 (2000).

86) Penning, T. M., Burczynski, M. E., Jez, J. M., Hung, C. F., Lin, H. K., Ma, H., Moore, M., Palackal, N. and Ratnam, K.: Human 3α-hydroxysteroid dehydrogenase isoforms (AKR1C1-AKR1C4) of the aldo-keto reductase superfamily: functional plasticity and tissue distribution reveals roles in the inactivation and formation of male and female sex hormones. *Biochem. J.*, 351: 67–77 (2000).

87) Steckelbroeck, S., Jin, Y., Gopishetty, S., Oyesanmi, B. and Penning, T. M.: Human cytosolic 3α-hydroxysteroid dehydrogenases of the aldo-keto reductase superfamily display significant 3β-hydroxysteroid dehydrogenase activity. *J. Biol. Chem.*, 279: 10784–10795 (2004).

88) Ishikura, S., Nakajima, S., Kaneko, T., Shinitani, S., Usami, N., Yamamoto, I., Carbone, V., El-Kabbani, O. and Hara A.: Comparison of stereoselective reduction of 3- and 20-oxosteroids among mouse and primate 20α-hydroxysteroid dehydrogenases. In Weiner H., Plapp, B., Lindahl, R. and Maser, E. (ed.): *Enzymology and Molecular Biology of Carbonyl Metabolism 12*, West Lafayette, Purdue University Press, 2005, pp. 341–351.

89) Weihua, Z., Lathe, R., Warner, M. and Gustafsson, J.-Å.: An endocrine pathway in the prostate, ERβ, AR, 5α-androstane-3β,17β-diol, and CYP7B1, regulates prostate growth. *Proc. Natl. Acad. Sci. USA*, 99: 13589–13594 (2001).

90) Wang, M., He, Y., Eisenman, L. N., Fields, C., Zeng, C. M., Mathews, J., Benz, A. and Fu, T.: 3β-Hydroxypregnane steroids are pregnenolone sulfate-like GABA<sub>A</sub> receptor antagonists. *J. Neurosci.*, 22: 3366–3375 (2002).

91) Turkmen, S., Lundgren, P., Birzniece, V., Zingmark, E., Backstrom, T. and Johansson, I. M.: 3β,20β-Dihydroxy-5α-pregnane (UC1011) antagonism of the GABA potentiating and the learning impairment induced in rats by allopregnanolone. *Eur. J. Neurosci.*,

20: 1604–1612 (2004).

92) Matsuura, K., Deyashiki, Y., Sato, K., Miwa, G., Ishida, N. and Hara, A.: Identification of amino acid residues responsible for differences in substrate specificity and inhibitor sensitivity between two human liver dihydrodiol dehydrogenase isoenzymes by site-directed mutagenesis. *Biochem. J.*, 323: 61–64 (1997).

93) Couture, J.-F., Legrand, P., Cantin, L., Luu-The, V., Labrie, F. and Breton, R.: Human 20α-hydroxysteroid dehydrogenase: Crystallographic and site-directed mutagenesis studies lead to the identification of an alternative binding site for C21-steroids. *J. Mol. Biol.*, 331: 593–604 (2003).

94) Ohara, H., Nakayama, T., Deyashiki, Y., Hara, A., Miyabe, Y. and Tsukada, F.: Reduction of prostaglandin D₂ to 9α,11β-prostaglandin F₂ by a human liver 3α-hydroxysteroid dehydrogenase isoenzyme. *Biochim. Biophys. Acta*, 1215: 59–65 (1994).

95) Breyer-Pfaff, U. and Nill, K.: Carbonyl reduction of naltrexone and dolasetron by oxidoreductases isolated from human liver cytosol. *J. Pharm. Pharmacol.*, 56: 1601–1606 (2004).

96) Wsol, V., Szotakova, B., Skalova, L. and Maser, E.: Contribution of aldo-keto reductases to the metabolism of the novel anticancer drugs oracin in man. In Weiner H., Plapp, B., Lindahl, R. and Maser, E. (ed.): Enzymology and Molecular Biology of Carbonyl Metabolism 12, West Lafayette, Purdue University Press, 2005, pp. 249–255.

97) Higaki, Y., Usami, N., Shintani, S., Ishikura, S., El-Kabbani, O. and Hara, A.: Selective and potent inhibitors of human 20α-hydroxysteroid dehydrogenase (AKR1C1) that metabolizes neurosteroids derived from progesterone. *Chem. Biol. Interact.*, 143–144: 503–513 (2003).

98) Ciaccio, P. J., Jaiswal, A. K. and Tew, K. D.: Regulation of human dihydrodiol dehydrogenase by Michael acceptor xenobiotics. *J. Biol. Chem.*, 269: 15558–15562 (1994).

99) Burczynski, M. E., Lin, H. K. and Penning, T. M.: Isoform-specific induction of a human aldo-keto reductase by polycyclic aromatic hydrocarbons (PAHs), electrophiles, and oxidative stress: implications for the alternative pathway of PAH activation catalyzed by human dihydrodiol dehydrogenase. *Cancer Res.*, 59: 607–614 (1999).

100) Burczynski, M. E., Sridhar, G. R., Palackal, N. T. and Penning, T. M.: The reactive oxygen species and Michael acceptor-inducible human aldo-keto reductase AKR1C1 reduces the α,β-unsaturated aldehyde 4-hydroxy-2-nonenal to 1,4-dihydroxy-2-nonenal. *J. Biol. Chem.*, 276: 2890–2897 (2001).

101) Agapova, O. A., Yang, P., Wang, W. H., Lane, D. A., Clark, A. F., Weinstein, B. I. and Hernandez, M. R.: Altered expression of 3α-hydroxysteroid dehydrogenases in human glaucomatous optic nerve head astrocytes. *Neurobiol. Dis.*, 14: 63–73 (2003).

102) Blouin, K., Blanchette, S., Richard, C., Dupont, P., Luu-The, V. and Tchernof, A.: Expression and activity of steroid aldoketoreductases 1C in omental adipose tissue are positive correlates of adiposity in women. *Am. J. Physiol.*, 288: E398–E404 (2005).

103) Deyashiki, Y., Taniguchi, H., Amano, T., Nakayama, T., Hara, A. and Sawada, H.: Structural and functional comparison of two human liver dihydrodiol dehydrogenases associated with 3α-hydroxysteroid dehydrogenase activity. *Biochem. J.*, 282: 741–746 (1992).

104) Hara, A., Matsuura, K., Tamada, Y., Sato, K., Miyabe, Y., Deyashiki, Y. and Ishida, N.: Relationship of human liver dihydrodiol dehydrogenases to hepatic bile-acid-binding protein and an oxidoreductase of human colon cells. *Biochem. J.*, 313: 373–376 (1996).

105) Lin, H. K., Steckelbroeck, S., Fung, K. M., Jones, A. N. and Penning, T. M.: Characterization of a monoclonal antibody for human aldo-keto reductase AKR1C3 (type 2 3α-hydroxysteroid dehydrogenase/type 5 17β-hydroxysteroid dehydrogenase); immunohistochemical detection in breast and prostate. *Steroids*, 69: 795–801 (2004).

106) Hara, A., Taniguchi, H., Nakayama, T. and Sawada, H.: Purification and properties of multiple forms of dihydrodiol dehydrogenase from human liver. *J. Biochem. (Tokyo)*, 108: 250–254 (1990).

107) Dufort, I., Rheault, P., Huang, X. F., Soucy, P. and Luu-The, V.: Characteristics of a highly labile human type 5 17β-hydroxysteroid dehydrogenase. *Endocrinology*, 140: 568–574 (1999).

108) Khanna, M., Qin, K. N., Wang, R. W. and Cheng, K. C.: Substrate specificity, gene structure, and tissue-specific distribution of multiple human 3α-hydroxysteroid dehydrogenases. *J. Biol. Chem.*, 270: 20162–20168 (1995).

109) Peltoketo, H., Luu-The, V., Simard, J. and Adamski, J.: 17β-Hydroxysteroid dehydrogenase (HSD)/17-ketosteroid reductase (KSR) family; nomenclature and main characteristics of the 17HSD/KSR enzymes. *J. Mol. Endocrinol.*, 23: 1–11 (1999).

110) Matsuura, K., Shiraishi, H., Hara, A., Sato, K., Deyashiki, Y., Ninomiya, M. and Sakai, S.: Identification of a principal mRNA species for human 3α-hydroxysteroid dehydrogenase isoform (AKR1C3) that exhibits high prostaglandin D₂ 11-ketoreductase activity. *J. Biochem. (Tokyo)*, 124: 940–946 (1998).

111) Suzuki-Yamamoto, T., Nishizawa, M., Fukui, M., Okuda-Ashitaka, E. and Nakajima, T.: cDNA cloning, expression and characterization of human prostaglandin F synthase. *FEBS Lett.*, 462: 335–340 (1999).

112) Desmond, J. C., Mountford, J. C., Drayson, M. T., Walker, E. A., Hewison, M., Ride, J. P., Luong, Q. T., Hayden, R. E., Vanin, E. F. and Bunce, C. M.: The aldo-keto reductase AKR1C3 is a novel suppressor of cell differentiation that provides a plausible target for the non-cyclooxygenase-dependent antineoplastic actions of nonsteroidal anti-inflammatory drugs. *Cancer Res.*, 63: 505–512 (2003).

113) Mills, K. I., Gilkes, A. F., Sweeney, M., Choudhry, M. A., Woodgate, L. J., Bunce, C. M., Brown, G. and Burnett, A. K.: Identification of a retinoic acid responsive aldoketoreductase expressed in HL60 leukaemic

cells. FEBS Lett., 440: 158–162 (1998).

114) Moeller, L. C., Dumitrescu, A. M., Walker, R. L., Meltzer, P. S. and Refetoff, S.: Thyroid hormone responsive genes in cultured human fibroblasts. J. Clin. Endocrinol. Metab., 90: 936–943 (2005).
115) Lovering, A. L., Ride, J. P., Bunce, C. M., Desmond, J. C., Cummings, S. M. and White, S. A.: Crystal structures of prostaglandin D₂ 11-ketoreductase (AKR1C3) in complex with the nonsteroidal anti-inflammatory drugs flufenamic acid and indomethacin. Cancer Res., 64: 1802–1810 (2004).
116) Komoto, J., Yamada, T., Watanabe, K. and Takusagawa, F.: Crystal structure of human prostaglandin F synthase (AKR1C3). Biochemistry, 43: 2188–2198 (2004).
117) Qiu, W., Zhou, M., Labrie, F. and Lin, S. X.: Crystal structures of the multispecific 17β-hydroxysteroid dehydrogenase type 5: critical androgen regulation in human peripheral tissues. Mol. Endocrinol., 18: 1798–1807 (2004).
118) Bauman, D. R., Rudnick, S. I., Szewczuk, L. M., Jin, Y., Gopishetty, S., Penning, T. M.: Development of nonsteroidal anti-inflammatory drug analogs and steroid carboxylates selective for human aldo-keto reductase isoforms: potential antineoplastic agents that work independently of cyclooxygenase isozymes. Mol. Pharmacol., 67: 60–68 (2005).
119) Russell, D. W.: The enzymes, regulation, and genetics of bile acid synthesis. Ann. Rev. Biochem., 72: 137–174 (2003).
120) Ozeki, T., Takahashi, Y., Kume, T., Nakayama, K., Yokoi, T., Nunoya, K., Hara, A. and Kamataki, T.: Co-operative regulation of the transcription of human dihydrodiol dehydrogenase (DD) 4/aldo-keto reductase (AKR) 1C4 gene by hepatocyte nuclear factor (HNF) 4α/γ and HNF-1α. Biochem. J., 355: 537–544 (2001).
121) Ozeki, T., Takahashi, Y., Nakayama, K. and Kamataki, T.: Hepatocyte nuclear factor (HNF)-4α/γ, HNF-1α, and vHNF-1 regulate the cell-specific expression of the human dihydrodiol dehydrogenase (DD) 4/AKR1C4 gene. Arch. Biochem. Biophys., 405: 185–190 (2002).
122) Winters, C. J., Molowa, D. T. and Guzelian, P. S.: Isolation and characterization of cloned cDNAs encoding human liver chlordecone reductase. Biochemistry, 29: 1080–1087 (1990).
123) Matsuura, K., Tamada, Y., Deyashiki, Y., Miyabe, Y., Nakanishi, M., Ohya, I. and Hara, A.: Activation of human liver 3α-hydroxysteroid dehydrogenase by sulphobromophthalein. Biochem. J., 313: 179–184 (1996).
124) Matsuura, K., Hara, A., Kato, M., Deyashiki, Y., Miyabe, Y., Ishikura, S., Sugiyama, T. and Katagiri, Y.: Activation of human liver 3α-hydroxysteroid dehydrogenase by clofibrate derivatives. J. Pharmacol. Exp. Ther., 285: 1096–1103 (1998).
125) Yamamoto, T., Matsuura, K., Shintani, S., Hara, A., Miyabe, Y., Sugiyama, T. and Katagiri, Y.: Dual effects of anti-inflammatory 2-arylpropionic acid derivatives on a major isoform of human liver 3α-hydroxysteroid dehydrogenase. Biol. Pharm. Bull., 21:

1148–1153 (1998).
126) Molowa, D. T., Shayne, A. G. and Guzelian, P. S.: Purification and characterization of chlordecone reductase from human liver. J. Biol. Chem., 261: 12624–12627 (1986).
127) Ozeki, T., Takahashi, Y., Nakayama, K., Funayama, M., Nagashima, K., Kodama, T. and Kamataki, T.: Hepatocyte nuclear factor-4α/γ and hepatocyte nuclear factor-1α as causal factors of interindividual difference in the expression of human dihydrodiol dehydrogenase 4 mRNA in human livers. Pharmacogenetics, 13: 49–53 (2003).
128) Kume, T., Iwasa, H., Shiraishi, H., Yokoi, T., Nagashima, K., Otsuka, M., Terada, T., Takagi, T., Hara, A. and Kamataki, T.: Characterization of a novel variant (S145C/L311V) of 3α-hydroxysteroid dihydrodiol dehydrogenase in human liver. Pharmacogenetics, 9: 763–771 (1999).
129) Ishida, M., Chang, K.-T., Hirabayashi, K., Nishihara, M. and Takahashi, M.: Cloning of mouse 20α-hydroxysteroid dehydrogenase. J. Reprod. Dev., 45: 321–329 (1999).
130) Ma, H. and Penning, T. M.: Characterization of homogeneous recombinant 20α-hydroxysteroid dehydrogenase: fluorescent properties and inhibition profile. Biochem. J., 341: 853–859 (1999).
131) Vergnes, L., Phan, J., Stolz, A. and Reue, K.: A cluster of eight hydroxysteroid dehydrogenase genes belonging to the aldo-keto reductase supergene family on mouse chromosome 13. J. Lipid Res., 44: 503–511 (2003).
132) Deyashiki, Y., Ohshima, K., Nakanishi, M., Sato, K., Matsuura, K. and Hara, A.: Molecular cloning and characterization of mouse estradiol 17β-dehydrogenase (A-specific), a member of the aldoketo reductase family. J. Biol. Chem., 270: 10461–10467 (1995).
133) Ishikura, S., Usami, N., Nakajima, S., Carbone, V., El-Kabbani, O. and Hara, A.: Site-directed mutagenesis of the coenzyme-binding residues of mouse 3α- and 3(17)α-hydroxysteroid dehydrogenases. Their roles in the specificity for coenzymes and substrates. In Weiner H., Plapp, B., Lindahl, R. and Maser, E. (ed.): Enzymology and Molecular Biology of Carbonyl Metabolism 12, West Lafayette, Purdue University Press, 2005, pp. 324–332.
134) Pawlowski, J. E. and Penning, T. M.: Overexpression and mutagenesis of the cDNA for rat liver 3α-hydroxysteroid dehydrogenase. Role of cysteines and tyrosines in catalysis. J. Biol. Chem., 269: 13502–13510 (1994).
135) Ikeda, S., Okuda-Ashitaka, E., Masu, Y., Suzuki, T., Watanabe, K., Nakano, M., Shingu, K. and Ito, S.: Cloning and characterization of two novel aldo-keto reductases (AKR1C12 and AKR1C13) from mouse stomach. FEBS Lett., 459: 433–437 (1999).
136) Du, Y., Tsai, S., Keller, J. R. and Williams, S. C.: Identification of an interleukin-3-regulated aldoketo reductase gene in myeloid cells which may function in autocrine regulation of myelopoiesis. J. Biol. Chem., 275: 6724–6732 (2000).
137) Tajima, K., Hashizaki, M., Yamamoto, K.,

Narimatsu, S. and Mizutani, T.: Purification and some properties of two enzymes from rat liver cytosol that catalyze carbonyl reduction of 6-tert-butyl-2,3-epoxy-5-cyclohexene-1,4-dione, a metabolite of 3-tert-butyl-4-hydroxyanisole. *Arch. Biochem. Biophys.*, **361**: 207–214 (1999).

Matsuura, K., Hara, A. and Sawada, H.: Identification of two dihydrodiol dehydrogenases associated with 3(17)α-hydroxysteroid dehydrogenase activity in mouse kidney. *J. Biochem.* (Tokyo), **106**: 633–638 (1989).

Qin, K.-N. and Cheng, K.-C.: Structure and tissue-specific expression of the aldo-keto reductase superfamily. *Biochemistry*, **33**: 3223–3228 (1994).

Ishikura, S., Usami, N., Nakajima, S., Kameyama, A., Shiraichi, H., Carbone, V., El-Kabbani, O. and Hara, A.: Characterization of two isoforms of mouse 3(17)α-hydroxysteroid dehydrogenases of the aldo-keto reductase family. *Biol. Pharm. Bull.*, **27**: 1939–1945 (2004).

Deyashiki, Y., Takatsuji, T. and Hara, A.: Molecular cloning and characterization of dihydrodiol dehydrogenase from the mouse. In Penning T. M. and Petrash, J. M. (ed.): *Aldo-keto Reductases and Toxicant Metabolism*, Washington, DC, American Chemical Society, 2004, pp. 101–114.

Bellemare, V., Faucher, F., Breton, R. and Luu-The, V.: Characterization of 17α-hydroxysteroid dehydrogenase activity (17α-HSD) and its involvement in the biosynthesis of epitestosterone. *BMC-Biochem.*, **6**: 12 (2005).

Toki, S. and Yamano, S.: Production of morphinone as a metabolite of morphine and its physiological role. *Yakugaku-Zasshi*, **119**: 249–267 (1999).

Thompson, J. D., Higgins, D. G., Gibson, T. J.: CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence-weighting, positions-specific gap penalties and weight matrix choice. *Nucleic Acids Res.*, **22**: 4673–4680 (1994).

Ishikura, S., Horie, K., Sanai, M., Matsumoto, K. and Hara, A.: Enzymatic properties of a member (AKR1C19) of the aldo-keto reductase family. *Biol. Pharm. Bull.*, **28**: 1075–1078 (2005).

Page, R. D. M.: TREEVIEW: An application to display phylogenetic trees on personal computers. *Comput. Appl. Biosci.*, **12**: 357–358 (1996).

Nakagawa, M., Tsukada, F., Nakayama, T.,
