
Identification of a Novel Arabino-furanosyltransferase (AftA) Involved in Cell Wall Arabinan Biosynthesis in *Mycobacterium tuberculosis*

Received for publication, January 3, 2006, and in revised form, March 31, 2006 Published, JBC Papers in Press, April 4, 2006, DOI 10.1074/jbc.M600045200

Luke J. Alderwick ${}^{1,2}$, Mathias Seidel ${}^{1}$, Hermann Sahm ${}^{3}$, Gurdyal S. Besra ${}^{4}$, and Lothar Eggeling ${}^{5}$

From the ${}^{\dagger}$School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, United Kingdom and ${}^{\S}$Institute for Biotechnology 1, Research Centre Juelich, D-52425 Juelich, Germany

---

The cell wall mycolyl-arabinogalactan-peptidoglycan complex is essential in mycobacterial species, such as *Mycobacterium tuberculosis*, and is the target of several anti-tubercular drugs. For instance, ethambutol targets arabinogalactan biosynthesis through inhibition of the arabino-furanosyltransferases Mt-EmbA and Mt-EmbB. Following a detailed bioinformatics analysis of genes surrounding the conserved *emb* locus, we present the identification and characterization of a novel arabino-furanosyltransferase AftA (Rv3792). The enzyme catalyzes the addition of the first key arabino-furanosyl residue from the sugar donor $\beta$-D-arabino-furanosyl-1-monophosphoryldecaprenol to the galactan domain of the cell wall, thus “priming” the galactan for further elaboration by the arabino-furanosyltransferases. Because *aftA* is an essential gene in *M. tuberculosis*, we deleted its orthologue in *Corynebacterium glutamicum* to produce a slow growing but viable mutant. Analysis of its cell wall revealed the complete absence of arabinose resulting in a truncated cell wall structure possessing only a galactan core with a concomitant loss of cell wall-bound mycolates. Complementation of the mutant was fully restored to the wild type phenotype by Cg-*aftA*. In addition, by developing an *in vitro* assay using recombinant *Escherichia coli* expressing Mt-*aftA* and use of cell wall galactan as an acceptor, we demonstrated the transfer of arabinose from $\beta$-D-arabino-furanosyl-1-monophosphoryldecaprenol to galactan, and unlike the Mt-Emb proteins, Mt-AftA was not inhibited by ethambutol. This newly discovered glycosyltransferase represents an attractive drug target for further exploitation by chemotherapeutic intervention.

---

The Corynebacterianeae represent a distinct group within Gram-positive bacteria, with prominent members being the human pathogens *Mycobacterium tuberculosis*, *Mycobacterium leprae*, and *Corynebacterium diphtheriae* (1). In addition, nonpathogenic bacteria belonging to this taxon, such as *Corynebacterium glutamicum* and *Corynebacterium efficiens*, are used in the industrial production of amino acids (2). A common feature of the Corynebacterianeae is that they possess an unusual cell wall architecture (3–5). The cell wall is dominated by an essential heteropolysaccharide, arabinogalactan (AG), linked to both peptidoglycan and mycolic acids, forming the mycolyl-arabinogalactan-peptidoglycan (mAGP) complex (3–6). The biosynthesis of the arabinan domain of AG, which is made up of $\alpha$1→5-, $\alpha$1→3-, and $\beta$1→2-glycosyl linkages, results from the sequential addition of arabino-furanose (Araf) residues from the sugar donor $\beta$-D-arabino-furanosyl-1-monophosphorylde-caprenol (DPA) (7–9), by a set of unique arabino-furanosyltransferases termed the Emb proteins, of which three paralogues exist in *Mycobacterium avium* (10) and *M. tuberculosis* (11).

The anti-tuberculosis drug ethambutol (EMB) specifically inhibits AG biosynthesis (12), and the molecular target of EMB occupies the *embCAB* locus in *M. tuberculosis* (11). Upon individual disruption of *embC*, *embA*, and *embB* in *Mycobacterium smegmatis*, the resultant mutants are viable (13, 14). However, the crucial terminal Ara${}_{6}$ motif, which is the structural motif for mycolylation in AG (5), is altered in both the Ms-*embA* and Ms-*embB* mutants (13). These results suggest that EmbA and EmbB are involved in the formation of the terminal Ara${}_{6}$ motif in AG and also presumably compensated for each other in the respective Ms-*embA* and Ms-*embB* mutants, whereas Ms-*embC* is probably involved in the formation of the arabinan domains of lipoarabinomannan (LAM) (14). This is in agreement with the initial studies of the Ms-*embC* mutant (14) and recent findings that indicate when point mutations were re-introduced into the Ms-*embC* mutant on a multicopy plasmid expressing the mutant allele, a truncated LAM was synthesized, which retained the basic glycosyl linkage profile of LAM (15). However, attempts to obtain deletion mutants of *embA* and *embB* in *M. tuberculosis* or *embAB* in *M. smegmatis* have proved unsuccessful, presumably because of the essentiality of the cell wall mAGP in these bacteria (16–19). In contrast, *C. glutamicum* has proven useful in the study of orthologous *M. tuberculosis* genes essential for cell viability. For instance, Cg-*pks* has been shown to be the key Claisen condensation enzyme involved in mycolic acid biosynthesis through the construction of a deletion mutant of *C. glutamicum* and its complementation with the Mt-*pks13* orthologue (19, 20).

In a recent study (6), deletion of the single Cg-*emb* orthologue in *C. glutamicum* resulted in a slow growing yet viable mutant that synthesized a novel truncated AG structure possessing a galactan core and only $t$-Araf residues. Moreover, partial acid hydrolysis and matrix-assisted laser desorption ionization time-of-flight analysis identified the precise location of the three singular $t$-Araf residues attached to the galactan core on the 8th, 10th, and 12th galactofuranose (Galf) residue, thus representing the anchor points of the intact arabinan domains of AG (6). Treatment of *C. glutamicum* with EMB resulted in an identical phenotype comparable with the *C. glutamicum Δemb* mutant (6). Thus,

---

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* Both authors contributed equally to this work.
* Biotechnology and Biological Sciences Research Council Quota Student.
* Supported by the Fonds der Chemischen Industrie.
* Supported as a Lister Institute-Jenner Research Fellow and the Medical Research Council (UK). To whom correspondence should be addressed. Tel.: 121-415-8125; Fax: 121-414-5925; E-mail: g.besra@bham.ac.uk.
* The abbreviations used are: AG, arabinogalactan; Ara, arabinose; CMAME, corynomonic acid methyl ester; DPA, decaprenyl phosphoarabinose; DPPR, decaprenylphosphoryl-5-phosphoribose; EMB, ethambutol; GC, gas chromatography; GC/MS, gas chromatography/mass spectrometry; mAGP, mycolyl-arabinogalactan-peptidoglycan; Rha, rhamnose; Araf, arabino-furanose; MOPS, 4-morpholinepropanesulfonic acid.
* G. S. Besra, unpublished results.

Identification of a Novel Arabinofuranosyltransferase

in contrast to the disruption of Ms-embA and Ms-embB, deletion of Cg-emb leads to an almost entire absence of arabinose in the cell wall, apart from specific t-Araf residues that are directly linked to the galactan backbone. This suggested the presence of a novel enzyme responsible for “priming” the galactan domain for further elaboration by the Emb proteins, resulting in the final maturation of the native AG polysaccharide. It is the aim of this study to identify this novel arabinofuranosyltransferase, which catalyzes the addition of the first key Araf residues to the galactan domain of AG.

**EXPERIMENTAL PROCEDURES**

Strains and Culture Conditions—M. tuberculosis H37Rv DNA was obtained from the Tuberculosis Research Material Contract (National Institutes of Health) at Colorado State University. C. glutamicum ATCC 13032 (the wild type strain, and in the remainder of the text it is referred to as C. glutamicum) and Escherichia coli DH5αMCR were grown in Luria-Bertani broth (LB; Difco) at 30 and 37 °C, respectively. The mutants generated in this study were grown on complex medium BHIS (21). Kanamycin and ampicillin were used at a concentration of 50 μg/ml. The minimal medium CGXII was used for C. glutamicum (21). Samples for lipid analyses were prepared by harvesting cells at an absorbance of 10–15, followed by a saline wash and freeze drying. Cultivation of C. glutamicum ΔaftA for lipid and cell wall analysis required two pre-cultures. First, a 5-ml BHIS culture was grown for 8 h, which was then used to inoculate a 50-ml BHIS culture for 15 h. This was then used to inoculate a 100-ml BHIS culture to an absorbance of 1, which was harvested after reaching absorbance of 3.

Construction of Plasmids and Strains—The vectors made were pET23b-Mt-aftA (Rv3792), pEKEx2Cg-aftA (NCgl0185), pEKEx2Mt-aftA, and pK19mobsacBΔaftA, with the gene number of the M. tuberculosis and C. glutamicum aftA orthologue added in parentheses. To construct the E. coli expression vector pET23b-Mt-aftA, the primer pair 5′-GATCGATCCCATATGCCGAGCAGACGCAAAAGCCCC-CAATTC-3′ and 5′-GATCGATCAAGCTTCGGCGCTCTCCTGCGG-CTTGC GGATGGC-3′ was used with the restriction sites NdeI and HindIII underlined, with M. tuberculosis H37Rv chromosomal DNA as a template. The purified PCR fragment was ligated with accordingly digested pET23b (Novagen). To overexpress C. glutamicum aftA, the primer pair 5′-TCCCCCGGGGAAGGAGATATAGATATGATTAACACCTCTGA-AGATGAAG-3′ and 5′-TCCCCCCGGGTACTCAT TGTGCGTTAC-CACCAC-3′ was used to amplify C. glutamicum aftA, which was ligated with SmaI-cleaved pEKEx2 to generate pEKEx2Cg-aftA. Similarly, primer pairs 5′-CAGGATCCAAGGAGATATAGATATGCCGAGCAGACG-CAAAAG-3′ and 5′-CAGGATCCCCCATCCGCGCTCTCCTGCGGCT-TGC-3′ were used to clone M. tuberculosis aftA into the BamHI site of pEKEx2. To construct the deletion vector pK19mobsacBΔaftA, crossover PCR was applied with primer pairs AB (A, 5′-CGTGGATCCGGT-GCC-3′; B, 5′-CCCATCCCACTAAACTAACATT CAGAGGTTGT-T-AATCAT-3′) and CD (C, 5′-TGTTTAAGTTTAGTGGATGGGGGTGG-GACCTTTCGTGGTGGTAACG-3′; D, 5′-GGCGTCCCGTACTGTC-CAG-3′) and C. glutamicum genomic DNA as template. Both PCR products were used in a second PCR with primer pairs AD to generate a fragment consisting of sequences adjacent to Cg-aftA, which was blunt end-ligated with SmaI-cleaved pK19mobsacB. All plasmids were finally confirmed by sequencing. The chromosomal deletion of Cg-aftA was performed as described using two rounds of positive selection (22), and its successful deletion was verified by use of different primer pairs. Plasmid pET23b-Mt-aftA was used to transform chemically competent cells of E. coli C43 (DE3) to ampicillin resistance (100 µg/ml), and

FIGURE 1. Comparison of the aftA locus within the Corynebacterianeae. A, the locus in M. tuberculosis (M. tub.) consists of aftA with its two upstream genes Rv3790 and Rv3791 leading to the formation of DPA (24, 25). Downstream of aftA the genes embC, embA, and embB are located encoding known arabinofuranosyltransferases (11). The organization of Rv3790, Rv3791, aftA, and embC is retained in a number of Corynebacterianeae indicative of a basic functional unit. In N. farcinica (N. far.), a glycosyltransferase of unknown function is located between aftA and embC. In C. jeikeium (C. jek.), Rv3790 and Rv3791 are clustered but are located at another locus. The abbreviations used are as follows: M. bov., M. bovis; M. av. p., M. avium paratuberculosis; M. lep., M. leprae; C. eff., C. efficiens; C. dip., C. diphteriae; and C. glu., C. glutamicum. B, topology of AftA and EmbC of M. tuberculosis. The topology is predicted using dense alignment surface (30). AftA spans the membrane 11 times and EmbC 13 times, and both have a C-terminal extension located in the periplasm covering about one-third of the protein. The star indicates a highly conserved region that resides in the periplasmic loop and is probably concerned with glycosyltransferase activity. C, partial sequence comparison of region I of AftA proteins (star in B), indicating their high degree of identity. The conserved negative residues possibly involved in glycosyltransferase activity are shaded in gray. The abbreviations are as above including Mycobacterium marinum (M. mar.).

pEKEx2Cg-aftA was introduced into C. glutamicum ΔaftA by electroporation with selection to kanamycin resistance (25 µg/ml).

Extraction and Analysis of Cell Wall-bound Mycolic Acids—Cells were grown as described above, harvested, washed, and freeze-dried. Cells (100 mg) were extracted by two consecutive extractions with 2 ml of CHCl₃/CH₃OH/H₂O (10:10:3, v/v) for 3 h at 50 °C. The bound lipids from the delipidated extracts or purified cell walls (see below) were released by the addition of 5% aqueous solution of tetrabutylammonium hydroxide, followed by overnight incubation at 100 °C, methylated as described previously (6), and analyzed by TLC using known standards (6).

Isolation of the mAGP Complex—The thawed cells were resuspended in phosphate-buffered saline containing 2% Triton X-100 (pH 7.2), disrupted by sonication, and centrifuged at 27,000 × g (4, 6, 23). The pelleted material was extracted three times with 2% SDS in phosphate-

Identification of a Novel Arabino-furanosyltransferase

**A**

![Diagram](https://i.imgur.com/1234567.png)

**B**

![Graph](https://i.imgur.com/8901234.png)

**C**

![Graph](https://i.imgur.com/5678901.png)

FIGURE 2. Construction and characteristics of *C. glutamicum ΔaftA*. A, illustrated is Cg-aftA with its adjacent genes NCgl0186 and Cg-emb and the strategy to delete Cg-aftA by using the deletion vector pk19mobsacBΔaftA. This vector carries 18 nucleotides of the 5′-end of Cg-aftA and 36 nucleotides of its 3′-end, thereby enabling the in-frame deletion of almost the entire Cg-aftA gene. The arrows marked P2 locate the primers used for the PCR analysis to confirm the absence of Cg-aftA. Primers P1 were used to detect NCgl0187 and NCgl10186, the orthologue of Rv3790 and Rv3791, and P3 to detect Cg-emb. Distances are not drawn to scale. The results of the PCR analysis are shown on the right, where NCgl0187 and NCgl10186 mark the result obtained with primers P1, Cg-aftA with P2, and Cg-emb with P3. Samples were applied pairwise with the PCR product obtained from the wild type applied in the left lane and that of the deletion mutant in the right lane. St marks the standard, where the arrowheads are located at 10, 3, 2, 1, and 0.5 kb, respectively. B, the consequences of Cg-aftA deletion on growth in rich medium BHI. Growth of *C. glutamicum* (●), *C. glutamicum ΔaftA* (□), as well as the same strain expressing plasmid encoded Cg-aftA (■), Mt-aftA (▲), and Cg-emb (◆) is shown. C, consequences of Cg-aftA deletion on the same medium as in B but supplemented with 0.5 M sorbitol for osmotic stabilization. Symbols are same as in B.

**Solvent**
Front

![TLC Image](https://i.imgur.com/abcdefg.png)

CMAMEs

Origin

1   2   3

FIGURE 3. Analysis of cell wall-bound CMAMEs from delipidated cells of *C. glutamicum*, *C. glutamicum ΔaftA*, and *C. glutamicum ΔaftA pEKEx2aftA*. Lane 1, *C. glutamicum*; lane 2, *C. glutamicum ΔaftA*; lane 3, *C. glutamicum ΔaftA pEKEx2aftA*. The bound coronymycolic acids from the delipidated extracts or purified cell walls were released by the addition of tetrabutylammonium hydroxide at 100 °C overnight and methylated as described under “Experimental Procedures.” An aliquot from each strain was subjected to TLC using silica gel plates (5735 Silica Gel 60F<sub>254</sub>, Merck), and developed in petroleum ether/acetone (95:5, v/v) and charred using 5% molybdophosphoric acid in ethanol at 100 °C to reveal CMAMEs and compared with known standards (6, 19).

buffered saline at 95 °C for 1 h to remove associated proteins, successively washed with water, 80% (v/v) acetone in water, and acetone, and finally lyophilized to yield a highly purified cell wall preparation (4, 6, 23).

Glycosyl Composition and Linkage Analysis of Cell Walls by Alditol Acetates—Cell wall preparations were hydrolyzed using 2 M trifluoroacetic acid, reduced with NaB<sup>2</sup>H<sub>4</sub>, and the resultant alditols per-O-acetylated and examined by gas chromatography (GC) as described previously (4, 6, 23). Cell wall preparations were per-O-methylated using dimethyl sulfinyl carbanion as described previously (4, 6, 23). The per-O-methylated cell walls were hydrolyzed using 2 M trifluoroacetic acid, reduced with NaB<sup>2</sup>H<sub>4</sub>, per-O-acetylated, and examined by gas chromatography/mass spectrometry (GC/MS) as described previously (4, 6, 23). Analysis of alditol acetate sugar derivatives was performed on a CE Instruments ThermoQuest Trace GC 2000. Samples were injected in the splitless mode. The column used was a DB225 (Supelco). The oven was programmed to hold at an isothermal temperature of 275 °C for a run time of 15 min (6). GC/MS was carried out on a Finnigan Polaris/GCQ Plus™. The column used was a BPX5 (Supelco).

DPA and Cg-Emb Biosynthetic Activity within Membrane Preparations of *C. glutamicum* and *C. glutamicum ΔaftA*—Membranes from *C. glutamicum* and *C. glutamicum ΔaftA* were prepared as described previously to determine DPA biosynthetic activity (24, 25). Membrane protein (1 mg) was added to 5-phospho[^14^C]ribofuranose pyrophosphate (2 × 10<sup>6</sup> cpm), 50 μg of decaprenol monophosphate, 60 μM ATP, 0.5 mM NADP in 50 mM MOPS (pH 7.9), 5 mM β-mercaptoethanol, and 10 mM MgCl<sub>2</sub> (buffer A) to a final volume of 160 μl. The reaction mixture was incubated for 1 h at 37 °C and stopped by the addition of 3 ml of CHCl<sub>3</sub>/CH<sub>3</sub>OH (2:1, v/v). Radiolabeled lipid linked sugars were extracted, as described previously, prior to scintillation counting and subjected to TLC using silica gel plates (5735 Silica Gel 60F<sub>254</sub>, Merck) in CHCl<sub>3</sub>/CH<sub>3</sub>OH/H<sub>2</sub>O/NH<sub>4</sub>OH (65:25:3.6:0.5, v/v) with reaction products visualized by autoradiography (24, 25). Analysis of Cg-Emb activity was determined using the synthetic α-D-Araf-(1→5)-α-D-Araf-O-C<sub>10:1</sub> acceptor in a cell-free assay as described previously (8).

Expression and Analysis of Mt-aftA Gene Product—For expression studies, *E. coli* (C43) was transformed with pET23b-Mt-aftA and cultured in Terrific Broth at 37 °C until an absorbance of 0.5, followed by the addition of 1 mM isopropyl 1-thio-β-D-galactopyranoside and further incubation for 12 h at 16 °C. Cells were harvested by centrifugation

Identification of a Novel Arabinofuranosyltransferase

**GC**

Ara  
↓  

Rha  
↓  

Gal  
↓  

Retention Time (Min)  
5     10     15  

**GC/MS**

Relative abundance (%)  
100  
80  
60  
40  
20  
0  

5-Ara  
t-Ara  
t-Rha  
2-Ara  
3,5-Ara  
4-Rha  
t-Gal  
2,5-Ara  
5-Gal  
6-Gal  
5,6-Gal  

Retention Time (Min)  
5     10     15  

B  

Rha  
↓  

Gal →  

Retention Time (Min)  
5     10     15  

Relative abundance (%)  
100  
80  
60  
40  
20  
0  

5-Gal  
4-Rha t-Gal  
6-Gal  

Retention Time (Min)  
5     10     15  

C  

Ara  
↓  

Rha  
↓  

Gal  
↓  

Retention Time (Min)  
5     10     15  

Relative abundance (%)  
100  
80  
60  
40  
20  
0  

5-Ara  
t-Ara  
t-Rha  
2-Ara  
3,5-Ara  
4-Rha  
t-Gal  
2,5-Ara  
5-Gal  
6-Gal  
5,6-Gal  

Retention Time (Min)  
5     10     15  

FIGURE 4. Glycosyl compositional and glycosyl linkage analysis of cell walls of C. glutamicum (A), C. glutamicum ΔaftA (B), and C. glutamicum ΔaftA pEKEx2aftA (C). Samples of purified cell walls were hydrolyzed using 2 M trifluoroacetic acid, reduced, per-O-acetylated, and subjected to GC as described under “Experimental Procedures.” Alditol acetate standards (Supelco) of Rha, Ara, and Gal were analyzed with retention times of 6, 7 and 10.1 min, respectively. Cell walls were per-O-methylated, hydrolyzed using 2 M trifluoroacetic acid, reduced, and per-O-acetylated. The resulting partially per-O-methylated and per-O-acetylated glycosyl derivatives were analyzed by GC/MS as described previously (4, 6, 23).

at 5000 rpm, and the resulting pellet was resuspended in buffer A. Resuspended cells were sonicated and centrifuged at 23,000 × g for 20 min at 4°C, and the resulting supernatant was recentrifuged at 100,000 × g for 90 min at 4°C to isolate cell membranes that were collected and concentrated to a protein concentration of 15–20 mg/ml.

Decaprenol phospho[^14^C]arabinose (100,000 cpm (45 μM) prepared as described previously (9, 26) and stored in CHCl₃/CH₃OH, 2:1 (v/v))

was dried under a stream of argon in a microcentrifuge tube (1.5 ml) and placed in a vacuum desiccator for 15 min to remove any residual solvent. The dried decaprenol phospho[^14^C]arabinose was then resuspended in 30 μl of buffer A supplemented with 10% IgePal CA-630 (Sigma). An aliquot of this decaprenol phospho[^14^C]arabinose solution (10,000 cpm, 4.5 μM, 3 μl) was added to the remaining constituents of the assay, which included 1 mg of membranes containing Mt-AftA and increasing

15656 JOURNAL OF BIOLOGICAL CHEMISTRY  
VOLUME 281 • NUMBER 23 • JUNE 9, 2006

amounts of purified cell wall galactan polymer (0.1–1.0 mg, which represents ~0.015–0.15 mM galactan acceptor (6, 27, 28)) from *C. glutamicum ΔaftA* in buffer A to a final volume of 300 μl. The reaction mixture was incubated for 1 h at 37 °C, and the pellet was recovered following centrifugation at 18,000 × g. The pellet was washed twice with 1 ml of buffer A containing 1% IgePal CA-630 and further washed with 1 ml of CHCl₃/CH₃OH (2:1, v/v) until the radioactivity was recorded as background in the final supernatant washes (less than 30 cpm). The pellet was finally resuspended in buffer A and transferred to a scintillation vial, and 10 ml of EcoScint was added, and the amount of [¹⁴C]arabinose incorporation into cell wall galactan was measured by scintillation counting.

**Analysis of Mt-AftA Assay Product—**The basic assay was repeated five times as described above, but using nonradiolabeled DPA (200 μg, 0.75 mM) (9, 26), and 1 mg of cell wall galactan polymer prepared from *C. glutamicum ΔaftA*. Following an initial incubation at 37 °C for 1 h, the assay was replenished with fresh membranes containing Mt-AftA (1 mg) and re-incubated for 1 h at 37 °C with the entire process repeated three times with the addition of fresh membranes. The reaction mixture was processed as described above using two 1% IgePal CA-630 detergent washes in buffer A (1 ml) and five CHCl₃/CH₃OH (1 ml, 2:1, v/v) extractions followed by centrifugation at 18,000 × g. The recovered pellets were pooled, per-O-methylated, hydrolyzed using 2 M trifluoroacetic acid, reduced with NaB²H₄, per-O-acetylated, and analyzed by GC/MS as described previously (6).

**RESULTS**

**Genome Comparison of the Emb Locus—**Based on our previous observation that *C. glutamicum Δemb* possessed no known arabinofuranosyltransferase activity yet contained single t-Araf units attached to the galactan core (6), we analyzed a 14-kb chromosomal region of *M. tuberculosis* encompassing *embC*, *embA*, and *embB* in detail and compared it with that of other Corynebacterianeae (Fig. 1A). This region includes the recently discovered decaprenylphosphoryl-5-phosphoribose (DPPR) epimerizing enzymes encoded by Rv3790 and Rv3791, which eventually provide Araf units from the sugar donor DPA (8, 24, 25, 29). These genes are followed by *Rv3792* which is adjacent to *embC*. This particular region consisting of four genes is syntenic with regions in *Mycobacterium bovis*, *M. avium* subsp. *paratuberculosis*, and *M. leprae* as well as with those of *C. efficiens*, *C. diphtheriae*, and *C. glutamicum*. In addition, *Nocardia farcinica* also shows ample synteny to the *M. tuberculosis* gene locus, as well as *Corynebacterium jeikeium*, indicating a fundamental function of Rv3792. Based on the results described below, the *Rv3792* gene and its orthologues was designated *aftA* (acronym for arabinofuranosyltransferase A).

*M. tuberculosis aftA* is predicted to encode a membrane protein of 643 amino acid residues. The predicted topology of the N-terminal region (residues 1–459) contains several hydrophobic segments based on dense alignment surface analysis (30), which probably form 11 transmembrane-spanning helices, whereas the C-terminal region (residues 460–643) is predicted to be directed toward the periplasm. This domain organization and localization somewhat resemble that of EmbC (Fig. 1B). Nevertheless, the AftA proteins show no significant sequence similarity to the Emb proteins, and unlike the Emb proteins, they are not included in the CAZy data base of glycosyltransferases (31). However, the similarity of the AftA proteins among each other is very high over their entire sequence. Even for the most distant pairs, *M. tuberculosis* and *C. diphtheriae*, there is still 35.1% identity spanning 555 amino acid residues. The three regions of maximal similarity extend from 111 to 191 (I), 474 to 498 (II), and 516 to 551 (III) (amino acid residues of *M.*

**FIGURE 5.** Production of DPA (*A*) and Cg-Emb activity (*B*) within membrane preparations of *C. glutamicum* and *C. glutamicum ΔaftA*. Radiolabeled lipid-linked sugars (*A*) were extracted following incubation with membranes and 5-phospho[¹⁴C]ribofuranose pyrophosphate, counted, and subjected to TLC using silica gel plates (5735 Silica Gel 60F₂₅₄, Merck) in CHCl₃/CH₃OH/H₂O/NH₄OH (65:25:3.6:0.5, v/v) with reaction products (DPA and DPPR) visualized by autoradiography. Lane 1, *C. glutamicum*; lane 2, *C. glutamicum ΔaftA*. Cg-Emb activity (*B*) was determined using the synthetic α-D-Araf-(1→5)-α-D-Araf-O-C₁₀₁ acceptor in a cell-free assay as described previously (8). The product X (α-D-[¹⁴C]Araf-(1→2/5)-α-D-Araf-(1→5)-α-D-Araf-O-C₁₀₁) was resuspended prior to scintillation counting and subjected to TLC using silica gel plates (5735 silica gel 60F₂₅₄, Merck) in CHCl₃:CH₃OH:H₂O (65:25:4, v/v) with the reaction products visualized by autoradiography. Lane 1, control, no membranes; lane 2, *C. glutamicum*; lane 3, *C. glutamicum ΔaftA*.

tuberculosis AftA). These regions contain several strictly conserved acidic and polar side chains as exemplified for part of region I of AftA (Fig. 1C), which are also known to play roles of general base and nucleophilic residues in glycosyl hydrolysis (31). Interestingly, region I (marked by a star in Fig. 1B) contains a conserved stretch of amino acids similar to that of the GT-C motif in EmbC, which is located in a periplasmic loop between membrane-spanning domains 3 and 4 (15). Taken together, the structural features of AftA as well as the localization of *aftA* within the gene cluster involved in arabinan biosynthesis suggest that AftA represents a putative glycosyltransferase involved in arabinan polymerization.

**Construction and Growth of *C. glutamicum ΔaftA*—**Despite the fact that Rv3792 is essential for *M. tuberculosis* (32), we attempted to delete its orthologue in *C. glutamicum*. The nonreplicative plasmid pK19mobsacBΔ*aftA* was constructed carrying sequences adjacent to *Cg-aftA*. The vector was introduced into *C. glutamicum*, and in several electroporation assays kanamycin-resistant clones were obtained, indicating integration of pK19mobsacBΔ*aftA* into the genome by homologous recombination (Fig. 2A). The *sacB* gene enables positive selection of a second homologous recombination event that can result either in the original wild type genomic organization or in clones deleted of *aftA* (20). More than 200 clones were obtained after 2–4 days and analyzed by PCR, but all were wild type, illustrating a strong disadvantage of *aftA* deletion. We continued with this analysis to eventually obtain in three independent approaches three rough textured colonies appearing after about 15 days. These were shown by PCR to have *aftA* deleted, whereas inspection of the adjacent *C. glutamicum* open reading frames resulted in identical PCR products to that of controls derived from the wild type (see Fig. 2A).

Growth of *C. glutamicum ΔaftA* in liquid brain-heart-infusion medium is shown in Fig. 2B. The mutant was almost unable to grow on this medium, whereas the presence of plasmid-encoded *Cg-aftA* (pEKEx2Cg-*aftA*) almost fully restored growth. Also, upon complementation of the mutant with Mt-*aftA* (pEKEx2Mt-*aftA*), growth restoration was obtained, albeit somewhat reduced, which might be due to the biased codon usage of *M. tuberculosis*. Transformation of *C. glutamicum ΔaftA* with pEKEx2emb (6) did not restore the growth defect, showing that overexpression of *Cg-emb* is unable to substitute *Cg-aftA*, thus confirming the unique specificity of both AftA and Emb. The iden-

Identification of a Novel Arabinofuranosyltransferase

tical strains were also grown on the same medium as before (brain-heart-infusion) but osmotically stabilized with 0.5 M sorbitol (Fig. 2C). Surprisingly, under these conditions, substantial growth was possible for *C. glutamicum ΔaftA*, which is presumably indicative of sorbitol stabilizing the cell wall mutant. On this medium the growth rate obtained for the wild type was 0.66 h⁻¹ with a final absorbance of 17 and that for the deletion mutant was 0.31 h⁻¹ with a final absorbance of 15.

Cell Wall-bound Corynomycolic Acid and Glycosyl Compositional and Linkage Analysis of Cell Walls—To relate the above growth phenotypic changes of *C. glutamicum ΔaftA* to its cellular composition, *C. glutamicum ΔaftA* and its complemented strain along with *C. glutamicum* were analyzed for arabinogalactan-esterified corynomycolic acids. *C. glutamicum* exhibited the known profile of corynomycolic acid methyl esters (CMAMEs) (Fig. 3, lane 1), whereas cell wall-bound corynomycolic acids were absent in *C. glutamicum ΔaftA* (Fig. 3, lane 2). The complementation of *C. glutamicum ΔaftA* with Cg-*aftA* (Fig. 3, lane 3) led to the restoration of cell wall-bound corynomycolic acids. These results suggest that Cg-*aftA* was involved in a key aspect of arabinan biosynthesis, whereby deletion perturbs tethering of corynomycolic acids to AG. Analysis of alditol acetate derivatives prepared from purified cell walls of *C. glutamicum* by GC revealed the sugar composition rhamnose (Rha), Ara, and Gal (Fig. 4A) (6). GC analysis of alditol acetates prepared from *C. glutamicum ΔaftA* (Fig. 4B) revealed a total loss of cell wall arabinose, which was restored upon complementation with plasmid pEKEx2Cg-*aftA* (Fig. 4C). GC/MS of per-O-methylated alditol acetate derivatives of *C. glutamicum*, *C. glutamicum ΔaftA*, and *C. glutamicum ΔaftA* pEKEx2Cg-*aftA* is shown in Fig. 4, A–C. Apart from the presence of 2,5-Araf and *t*-Rhap associated with the arabinan domain of AG in *M. tuberculosis* (4, 33) and *C. glutamicum* (6, 27, 33) and include *t*-Araf, 2-Araf, 5-Araf, 3,5-Araf, 4-Rhap, *t*-Galf, 5-Galf, 6-Galf, and 5,6-Galf (Fig. 4A). As expected, *C. glutamicum ΔaftA* was devoid of *t*-Araf, *t*-Rhap, 2-Araf, 5-Araf, 3,5-Araf, and 2,5-Araf, whereas the galactan domain (apart from the 5,6-Galf branching residues resulting from arabinan side chains) was completely unaffected by the deletion of Cg-*aftA* and contained 4-Rhap, *t*-Galf, 5-Galf, and 6-Galf (Fig. 4B). Complementation of *C. glutamicum ΔaftA* with plasmid pEKEx2Cg-*aftA* restored the glycosyl linkage profile to that of *C. glutamicum* (Fig. 4, A and C). The previous deletion of *emb* in *C. glutamicum* and chemical analysis of the cell wall revealed a drastically truncated AG structure possessing only *t*-Araf residues and an unaltered galactan domain (6). Thus, the results indicate that *aftA* represents a putative arabinofuranosyltransferase responsible for priming the galactan domain with Araf residues for subsequent elaboration by Emb proteins.

Analysis of DPA Synthesis and Cg-Emb Activity—In order to ensure that in *C. glutamicum ΔaftA* the biosynthesis of DPA is not reduced thereby disabling Araf delivery to the cell wall, we examined DPA and DPPr biosynthesis. Based on previous studies (24, 25) using 5-phospho[^14^C]ribofuranose pyrophosphate and decaprenol phosphate to monitor DPA formation, membranes were prepared from *C. glutamicum* and *C. glutamicum ΔaftA*. Both preparations afforded DPA synthesis (Fig. 5A), demonstrating that there was no reduced ability of *C. glutamicum ΔaftA* to synthesize DPA. In fact, an accumulation of DPA and DPPr was

FIGURE 6. Expression and functional characterization of recombinant Mt-*aftA*. A, expression of Mt-*aftA* within *E. coli* (C43) membranes was confirmed by SDS-PAGE analysis. Lane 1, molecular mass standards (kDa); lane 2, *E. coli* (C43) uninduced membranes; lane 3, *E. coli* (C43) membranes expressing Mt-AftA. B, the data illustrate an increase in [^14^C]arabinose incorporation into cell wall galactan from decaprenol phospho[^14^C]arabinose with a fixed amount of *E. coli* (C43) membranes containing recombinant Mt-AftA (1 mg/ml). Background counts (less than 30 cpm) have been subtracted to give a final value for [^14^C]arabinose incorporation with 0.1 mg (lane 1), 0.25 mg (lane 2), 0.5 mg (lane 3), and 1.0 mg (lane 4) of cell wall galactan. Lane 5 represents the same reaction as lane 4 supplemented with

100 μg/ml ethambutol. In addition, control assays were performed with membranes prepared from uninduced *E. coli* or *E. coli* harboring empty pET23b also resulted in background counts (less than 30 cpm) for the [^14^C]Araf incorporation from decaprenol phospho[^14^C]arabinose in the presence of increasing cell wall galactan acceptor. C, the reaction identical to that described above (*B*, lane 4) was performed but using nonradio-labeled DPA. After incubation the reaction product was extracted, per-O-methylated, hydrolyzed, reduced with NaB²H₄, per-O-acetylated, and analyzed by GC/MS as described under “Experimental Procedures.”

Identification of a Novel Arabino-furanosyltransferase

Galactan Core

OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
OH
Identification of a Novel Arabinofuranosyltransferase

branes were prepared from *E. coli* (C43) pET23b-Mt-*aftA* and analyzed by SDS-PAGE revealing a weakly staining protein band with an apparent molecular mass of 69.5 kDa, which was absent in uninduced cultures, and a size predicted for Mt-His₆AftA (Fig. 6A). According to the current data, the reaction catalyzed by Mt-AftA is the transfer of Araf from DPA to galactan. Therefore, an arabinose-free cell wall galactan acceptor was prepared from *C. glutamicum ΔaftA* and incubated with the *E. coli* membrane preparation expressing His₆-tagged Mt-*aftA*. Following incubation, the residual decaprenol phospho[^14^C]arabinose substrate was removed from the cell wall galactan by detergent and several repeated extractions using CHCl₃/CH₃OH (2:1, v/v). The remaining insoluble cell wall core was then subjected to scintillation counting revealing an increased amount of [^14^C]Araf incorporation in the presence of increasing amounts of cell wall galactan acceptor (Fig. 6B). Surprisingly, a 10-fold increase in cell wall galactan acceptor resulted in only a 3-fold increase in transferase activity. This poor turnover is presumably because of the inefficiency of the assay, which utilizes insoluble cell wall galactan as an acceptor. In addition, the activity of Mt-AftA remained unaffected in the presence of 100 μg/ml EMB (Fig. 6B), a known inhibitor of Cg-Emb (6), Mt-EmbA, and Mt-EmbB (11).

The conversion of the galactan acceptor to a sugar polymer containing *t*-Araf units was further confirmed by glycosyl linkage analysis of the enzymatic reaction product (Fig. 6C). The newly synthesized product of several scaled up nonradiolabeled reactions was recovered, per-O-methylated, and derivatized to alditol acetates, which were analyzed by GC/MS. The linkages identified included those associated with the original cell wall galactan acceptor (Fig. 4B) plus the appearance of *t*-Araf, and as a result the branched 5,6-Galf residues (Fig. 6C).

Thus, our results describe the first report of a novel EMB-resistant arabinofuranosyltransferase, AftA, as the key initial glycosyltransferase involved in cell wall arabinan biosynthesis in Corynebacterianeae (Fig. 7) like *M. tuberculosis* and *C. glutamicum*.

## DISCUSSION

The mAGP represents one of the most important cell wall components of the Corynebacterianeae and is essential for the viability of *M. tuberculosis* (16–19). It is therefore not surprising that one of the most effective anti-mycobacterial drugs, EMB, targets its biosynthesis. However, the emergence of multidrug-resistant tuberculosis has accelerated the need to discover new drug targets (34). We previously hypothesized the presence of a new priming enzyme that would link the initial Araf unit with the C-5 OH of a β(1→6)-linked Galf of a presynthesized galactan core (6). This was derived from a thorough analysis of a *C. glutamicum* mutant deleted of its single arabinofuranosyltransferase Emb (6), which still synthesizes a linear galactan extending from the reducing Rha with single *t*-Araf residues attached to the 8th, 10th, and 12th Galf residue. Apparently, these specific Araf residues serve for recognition and extension by the known Emb proteins resulting in the formation of the mature arabinan chains (6, 10, 11).

The *in vivo* analysis of *C. glutamicum ΔaftA*, as well as the *in vitro* study with the AftA protein of *M. tuberculosis*, identifies a *bona fide* arabinofuranosyltransferase. In principle, the absence of Araf residues in *C. glutamicum ΔaftA* could be due to the unavailability of precursors, as we demonstrated previously with a *C. glutamicum* mutant devoid of the polyprenyl transferase (UbiA) activity involved in the synthesis of DPA (6). However, we established that DPA biosynthesis was maintained in *C. glutamicum ΔaftA*, as well as Emb-catalyzed arabinan biosynthesis *in vitro*. This, together with our previous study on *C. glutamicum Δemb*, shows that AftA functions to link the first Araf residue to the cell wall galactan core. More importantly, the recombinant Mt-AftA transferred Araf units from DPA to a cell wall galactan core acceptor *in vitro*, thus further confirming the unique activity of the enzyme. Although both Emb and AftA are arabinofuranosyltransferases, the proteins cannot functionally replace each other. Thus, despite some functional relationship, both glycosyltransferases have inherent specific features as is also evident from the insensitivity of Mt-AftA and Cg-AftA toward EMB, whereas the single Cg-Emb (6, 35) and Mt-Emb proteins are sensitive toward EMB (11).

The discovery of AftA sheds new light on the key arabinofuranosyltransferases that build the arabinan domain, which is typical for Corynebacterianeae. An elementary structure of this sugar polymer is apparent in *C. glutamicum*, and this bacterium has proven useful for a number of studies on mAGP biosynthesis (6, 19, 20). It represents the archetype of the Corynebacterianeae and has a low frequency of structural alterations as manifested, for instance in a low number of gene duplications (36). *Corynebacterium* species have only one *emb* gene, whereas *Mycobacterium* species have up to three. Nevertheless, the glycosidic linkage analysis of *C. glutamicum* AG shows that 2-Araf, 5-Araf, and 3,5-Araf linkages are present, which are analogous to those found in the AG of *M. tuberculosis* (4). Furthermore, in *C. diphtheriae* and *C. glutamicum* 2,5-Araf linkages are also evident (6, 37). This suggests the possibility that one single Emb glycosyltransferase enables the formation of different linkage types, a feature reminiscent of the bi-functional galactofuranosyltransferase (GlfT) of *M. tuberculosis* that produces alternating 5-Galf and 6-Galf linkages within the galactan core of AG (38, 39). The high synteny of the *M. tuberculosis* Rv3790, Rv3791, *aftA*, and *embC* to the maps of all other *Mycobacterium* and *Corynebacterium* species are in agreement with this view (Fig. 1A). Also in *N. farcinica* this general organization is retained with an additional membrane protein-encoding gene. This largely retained organization, as well as the separation of the paralogous *embAB* genes in *M. leprae* and *M. avium* subsp. *paratuberculosis* is indicative of an ancient core function of Rv3790, Rv3791, *aftA*, and *embC* within the Corynebacterianeae involved in the synthesis of Araf donors and their use to assemble a basic periplasmic arabinan domain that serves as a scaffold to tie mycolic acids. The identification of new cell wall biosynthetic drug targets is of great importance, especially with the emergence of multidrug-resistant tuberculosis. This newly discovered DPA-dependent arabinofuranosyltransferase represents a promising candidate for further exploitation as a potential drug target to disrupt the essential mycolyl-arabinogalactan-peptidoglycan complex in mycobacterial species, such as *M. tuberculosis*.

---

Acknowledgments—*M. tuberculosis* H37Rv DNA was obtained from the Tuberculosis Research Materials Contract (National Institutes of Health) at Colorado State University. We thank Graham Burns for technical assistance.

---

## REFERENCES

1. Bloom, B. R., and Murray, C. J. (1992) *Science* **257**, 1055–1064
2. Sahm, H., Eggeling, L., and de Graaf, A. A. (2000) *Biol. Chem.* **381**, 899–910
3. McNeil, M., Daffe, M., and Brennan, P. J. (1990) *J. Biol. Chem.* **265**, 18200–18206
4. Besra, G. S., Khoo, K. H., McNeil, M. R., Dell, A., Morris, H. R., and Brennan, P. J. (1995) *Biochemistry* **34**, 4257–4266
5. McNeil, M., Daffe, M., and Brennan, P. J. (1991) *J. Biol. Chem.* **266**, 13217–13223
6. Alderwick, L. J., Radmacher, E., Seidel, M., Gande, R., Hitchen, P. G., Morris, H. R., Dell, A., Sahm, H., Eggeling, L., and Besra, G. S. (2005) *J. Biol. Chem.* **280**, 32362–32371
7. Wolucka, B. A., McNeil, M. R., de Hoffmann, E., Chojnacki, T., and Brennan, P. J. (1994) *J. Biol. Chem.* **269**, 23328–23335
8. Lee, R. E., Brennan, P. J., and Besra, G. S. (1997) *Glycobiology* **7**, 1121–1128
9. Lee, R. E., Mikusova, K., Brennan, P. J., and Besra, G. S. (1995) *J. Am. Chem. Soc.* **117**, 11829–11832
10. Belanger, A. E., Besra, G. S., Ford, M. E., Mikusova, K., Belisle, J. T., Brennan, P. J., and Inamine, J. M. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 11919–11924

15660 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 281 • NUMBER 23 • JUNE 9, 2006

11. Telenti, A., Philipp, W. J., Sreevatsan, S., Bernasconi, C., Stockbauer, K. E., Wieles, B., Musser, J. M., and Jacobs, W. R., Jr. (1997) *Nat. Med.* **3**, 567–570

12. Takayama, K., and Kilburn, J. O. (1989) *Antimicrob. Agents Chemother.* **33**, 1493–1499

13. Escuyer, V. E., Lety, M. A., Torrelles, J. B., Khoo, K. H., Tang, J. B., Rithner, C. D., Frehel, C., McNeil, M. R., Brennan, P. J., and Chatterjee, D. (2001) *J. Biol. Chem.* **276**, 48854–48862

14. Zhang, N., Torrelles, J. B., McNeil, M. R., Escuyer, V. E., Khoo, K. H., Brennan, P. J., and Chatterjee, D. (2003) *Mol. Microbiol.* **50**, 69–76

15. Berg, S., Starbeck, J., Torrelles, J. B., Vissa, V. D., Crick, D. C., Chatterjee, D., and Brennan, P. J. (2005) *J. Biol. Chem.* **280**, 5651–5663

16. Pan, F., Jackson, M., Ma, Y., and McNeil, M. (2001) *J. Bacteriol.* **183**, 3991–3998

17. Mills, J. A., Motichka, K., Jucker, M., Wu, H. P., Uhlik, B. C., Stern, R. J., Scherman, M. S., Vissa, V. D., Pan, F., Kundu, M., Ma, Y. F., and McNeil, M. (2004) *J. Biol. Chem.* **279**, 43540–43546

18. Vilcheze, C., Morbidoni, H. R., Weisbrod, T. R., Iwamoto, H., Kuo, M., Sacchettini, J. C., and Jacobs, W. R., Jr. (2000) *J. Bacteriol.* **182**, 4059–4067

19. Gande, R., Gibson, K. J., Brown, A. K., Krumbach, K., Dover, L. G., Sahm, H., Shioyama, S., Oikawa, T., Besra, G. S., and Eggeling, L. (2004) *J. Biol. Chem.* **279**, 44847–44857

20. Portevin, D., De Sousa-D’Auria, C., Houssin, C., Grimaldi, C., Chami, M., Daffe, M., and Guilhot, C. (2004) *Proc. Natl. Acad. Sci. U. S. A.* **101**, 314–319

21. Eggeling, L., and Reyes, O. (2005) in *Handbook of Corynebacterium glutamicum*, (Eggeling, L. and Botts, M., eds) pp. 535–566, CRC Press, Inc., Taylor Francis Group, Boca Raton, FL

22. Schafer, A., Tauch, A., Jager, W., Kalinowski, J., Thierbach, G., and Puhler, A. (1994) *Gene (Amst.)* **145**, 69–73

23. Daffe, M., Brennan, P. J., and McNeil, M. (1990) *J. Biol. Chem.* **265**, 6734–6743

24. Mikusova, K., Huang, H., Yagi, T., Holsters, M., Vereecke, D., D’Haeze, W., Sherman, M. S., Brennan, P. J., McNeil, M. R., and Crick, D. C. (2005) *J. Bacteriol.* **187**, 8020–8025

25. Huang, H., Scherman, M. S., D’Haeze, W., Vereecke, D., Holsters, M., Crick, D. C., and McNeil, M. R. (2005) *J. Biol. Chem.* **280**, 24539–24543

26. Lee, R. E., Brennan, P. J., and Besra, G. S. (1998) *Bioorg. Med. Chem. Lett.* **8**, 951–954

27. Puech, V., Chami, M., Lemassu, A., Laneelle, M. A., Schiffler, B., Gounon, P., Bayan, N., Benz, R., and Daffe, M. (2001) *Microbiology* **147**, 1365–1382

28. Acharya, P. V., and Goldman, D. S. (1970) *J. Bacteriol.* **102**, 733–739

29. Cole, S. T., Brosch, R., Parkhill, J., Garnier, T., Churcher, C., Harris, D., Gordon, S. V., Eiglmeier, K., Gas, S., Barry, C. E., III, Tekaia, F., Badcock, K., Basham, D., Brown, D., Chillingworth, T., Connor, R., Davies, R., Devlin, K., Feltwell, T., Gentles, S., Hamlin, N., Holroyd, S., Hornsby, T., Jagels, K., Krogh, A., McLean, J., Moule, S., Murphy, L., Oliver, K., Osborne, J., Quail, M. A., Rajandream, M.-A., Rogers, J., Rutter, S., Seeger, K., Skelton, J., Squares, S., Squares, R., Sulston, J. E., Taylor, K., Whitehead, S., and Barrell, B. G. (1998) *Nature* **393**, 537–544

30. Cserzo, M., Wallin, E., Simon, I., von Heijne, G., and Elofsson, A. (1997) *Protein Eng.* **10**, 673–676

31. Liu, J., and Mushegian, A. (2003) *Protein Sci.* **12**, 1418–1431

32. Sassetti, C. M., Boyd, D. H., and Rubin, E. J. (2003) *Mol. Microbiol.* **48**, 77–84

33. Daffe, M., McNeil, M., and Brennan, P. J. (1993) *Carbohydr. Res.* **249**, 383–398

34. Brennan, P. J., and Nikaido, H. (1995) *Annu. Rev. Biochem.* **64**, 29–63

35. Radmachera, E., Stansen, K. C., Besra, G. S., Alderwick, L. J., Maughan, W. N., Hollweg, G., Sahm, H., Wendisch, V. F., and Eggeling, L. (2005) *Microbiology* **151**, 1359–1368

36. Nakamura, Y., Nishio, Y., Ikeo, K., and Gojobori, T. (2003) *Gene (Amst.)* **317**, 149–155

37. Dover, L. G., Cerdeno-Tarraga, A. M., Pallen, M. J., Parkhill, J., and Besra, G. S. (2004) *FEMS Microbiol. Rev.* **28**, 225–250

38. Kremer, L., Dover, L. G., Morehouse, C., Hitchin, P., Everett, M., Morris, H. R., Dell, A., Brennan, P. J., McNeil, M. R., Flaherty, C., Duncan, K., and Besra, G. S. (2001) *J. Biol. Chem.* **276**, 26430–26440

39. Mikusova, K., Yagi, T., Stern, R., McNeil, M. R., Besra, G. S., Crick, D. C., and Brennan, P. J. (2000) *J. Biol. Chem.* **275**, 33890–33897
