
CURRENT MICROBIOLOGY Vol. 53 (2006), pp. 30–36  
DOI: 10.1007/s00284-005-0180-1  

---

**Biodegradation of Dibenzofuran by Janibacter terrae Strain XJ-1**

Shiwei Jin,${}^{1,2,3}$ Tao Zhu,${}^{1}$ Xudong Xu,${}^{1}$ Ying Xu${}^{1}$

${}^{1}$State Key Laboratory of Freshwater Ecology and Biotechnology, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, Hubei 430072, China  
${}^{2}$School of Chemical Engineering and Pharmacy, Wuhan Institute of Chemical Technology, Wuhan, Hubei 430073, China  
${}^{3}$Graduate School, Chinese Academy of Sciences, 19A Yuquan Road, Beijing 100039, China  

Received: 24 June 2005 / Accepted: 2 December 2005  

---

**Abstract.** The dibenzofuran (DF)-degrading bacterium, *Janibacter terrae* strain XJ-1, was isolated from sediment from East Lake in Wuhan, China. This strain grows aerobically on DF as the sole source of carbon and energy; it has a doubling time of 12 hours at 30°C; and it almost completely degraded 100 mg/L ${}^{-1}$ DF in 5 days, producing 2,2′,3-trihydroxybiphenyl, salicylic acid, gentisic acid, and other metabolites. The *dbdA* (DF dioxygenase) gene cluster in the strain is almost identical to that on a large plasmid in *Terrabacter* sp. YK3. Unlike *Janibacter* sp. strain YY-1, XJ-1 accumulates gentisic acid rather than catechol as a final product of DF degradation.

Polychlorinated dibenzo-p-dioxins (PCDDs) and polychlorinated DF (PCDFs) are ubiquitous environmental pollutants [3]. Large quantities of dioxins are released into the environment as unintentional contaminants in pesticides and herbicides and from incineration processes. These hazardous compounds are highly toxic and persistent in the environment and tend to accumulate in the body fat of animals. They are widely distributed but usually at very low concentrations. Physical and chemical treatment procedures seem to be not feasible in the case of large masses of contaminated soil. Recently, bioremediation methods have become widely accepted as an alternative to physical and chemical methods because they can be applied in situ at a relatively low cost [9, 12, 22]. There have been no reports on microorganisms using PCDDs and PCDFs as the sole source of carbon and energy for growth. However, after reductive dehalogenation, some PCDDs and PCDFs can be transformed into less-halogenated congeners, which can be used as carbon and energy sources for the growth of certain microbes. Nonhalogenated dibenzofuran (DF) has been used as a model compound to study the biodegradation of PCDDs and PCDFs. Some bacterial strains capable of metabolizing DF have been isolated,

and in most cases, the ring cleavage reactions have been elucidated [2, 5, 6, 11, 23, 24, 28, 29, 32]. Some of these—such as *Sphingomonas* sp. strain RW1 [13, 17, 31], *Sphingomonas* sp. HL7 [6], *Rhodococcus opacus* SAO101 [18], and *Terrabacter* sp. DBF63 [10]—can co-oxidize slightly chlorinated dibenzodioxin (DD) and DF. When grown on DF, *Terrabacter* sp. strains DBF63, DPO 360, and DPO1361—the latter a heterotypic synonym to *Janibacter terrae*—produced intermediates 2,2′,3-trihydroxybiphenyl and salicylic acid by way of initial dioxygenation by angular dioxygenase [15, 16, 20, 26]. Salicylic acid can be converted to catechol and gentisic acid [16]. Unlike these strains, *Janibacter* sp. strain YY-1 [33] metabolizes DF by way of both angular and lateral dioxygenation and produces catechol as its final product.

In this article, we report the isolation and characterization of another DF-degrading *J. terrae* strain. It is similar to strain YY-1 in that it metabolizes DF by way of both angular and lateral dioxygenation, but it produces gentisic acid instead of catechol as the final product.

---

**Materials and Methods**

Chemicals. DF, salicylic acid, gentisic acid, and the derivatization reagent, N-methyl-N-trimethylsilyl trifluoroacetamide, were purchased

Correspondence to: Y. Xu; email: xuying@ihb.ac.cn

S. Jin et al.: DF Metabolism in *Janibacter terrae*

![Graph 1](#)

(I)

![Graph 2](#)

(II)

Fig. 1. Production of salicylic acid and gentisic acid during degradation of DF by *J. terrae* XJ-1. (A) Degradation of DF. Closed diamond = *J. terrae* XJ-1; open triangle = control (autoclaved cells). (B) Dynamics of salicylic acid and gentisic acid. Closed square = salicylic acid; open circle = gentisic acid.

from Sigma-Aldrich. All other chemicals were obtained from Beijing Chemicals.

**Media and growth conditions.** The mineral salts medium (MSM) contained 7 g Na₂HPO₄·12H₂O; 2 g KH₂PO₄; 0.5 g (NH₄)₂SO₄; 0.2 g MgSO₄·7H₂O; 0.005 g ZnSO₄·7H₂O; 0.0025 g Na₂MoO₄·2H₂O; 0.014 g Ca(H₂PO₄)₂; 0.01 g FeSO₄·6H₂O (pH 7.0); and 1 L ddH₂O. Solid DF was added to the sterilized medium at 1 g/L⁻¹. To prepare solid plates, 18 g agar was added to 1 L liquid MSM and autoclaved. The enriched medium contained 5 g yeast extracts, 10 g tryptone, 10 g NaCl, and 1 L ddH₂O. The bacterium was cultured with 100 mL liquid medium in 500-mL flasks and vigorously shaken (150 to 180 rpm) at 30°C.

**Isolation of DF-degrading bacterium.** Sediment samples were collected from East Lake in Wuhan, China. Five grams lake sediment were added to 50 mL MSM liquid medium supplemented with DF crystals in a 250-mL flask and shaken at 30°C. Aliquots were transferred weekly to fresh medium and cultured under the same conditions until certain microbe(s) grew in the medium (five rounds). The microbes were then purified by streaking on plates.

**Growth rate and degradation of DF.** Bacterial growth was monitored as increasing turbidity at 585 nm after removal of crystal DF from the culture by filtration with Schleicher sand Schuell (Germany) grade 597 filter paper. Degradation of DF was tested in sterilized tubes containing 3 mL cell suspension (optical density₅₈₅ = 0.94). DF dissolved in dimethylsulfoxide, 100 mgL⁻¹, was added to the medium. The test tubes were vigorously shaken at 180 rpm at 30°C. At each sampling point, 3 of the test tubes were taken from the shaker and stored at –20°C. At the end of the experiment, all frozen samples were thawed and extracted three times with an equal volume of methylene chloride, and the DF concentration was

determined by gas chromatography (GC). The data were reported as the means measured from three repeats. The SD of the experiment was <8%.

Analysis of metabolites derived from DF. Cells were cultured in 100 mL MSM containing 1 g/L⁻¹ DF and spun down by centrifugation. Metabolites were extracted from the supernatant twice with an equal volume of ethyl acetate (pH 7.0) (neutral fraction) and once again after acidification of the aqueous residue to pH 2.0 with 2 mol·L⁻¹ N hydrochloric acid. The organic phases were dried with anhydrous sodium sulfate, and the solvent was removed by gently blowing with a stream of nitrogen gas. The extracts obtained were derivatized by N-methyl-N-trimethylsilyl trifluoroacetamide at 60°C for 1 hour and analyzed by GC–mass spectrometry (MS).

GC was carried out using an Agilent 6890 GC equipped with an HP-5 column (length 30 m, inner diameter 0.32 mm, film thickness 0.25 μm) with a flame ionization detector at 300°C. The following temperature program was used for the detection of DF: 60°C to 180°C at a rate of 10°C/min⁻¹, then at a rate of 15°C/min⁻¹ to 280°C. The extracts were measured using an Agilent GC coupled with a 5973 mass selective detector with HP-5 MS column (length 30 m, inner diameter 0.25 mm, film thickness 0.25 μm). Helium as carrier gas was controlled at a flow rate of 1.0 mL/min⁻¹. The injector was operated in pulsed splitless mode at a temperature of 250°C. Electron impact ionization was set at 70 eV. The temperatures of interface, source, and quadrupole were 280°C, 230°C, and 150°C, respectively. The following temperature program was used for the identification of metabolites: 80°C for 3 minutes, then 80°C to 290°C at a rate of 5°C·min⁻¹.

Polymerase chain reaction and sequencing. Polymerase chain reaction (PCR) consisted of initial denaturation at 94°C for 5 minutes and 30 cycles of the following: 94°C for 1 minute, 60°C for 1 minute, 72°C for 1 minute, and a final extension at 72°C for 5 minutes. The 16S rDNA was amplified by PCR using primers EubA (5′ AAGGAGGTGATCCANCCRC A 3′) and EubB (5′ AGAGTTTGA TCMTGGCTCAG 3′) [21], A 1.06-kb fragment of the *dbdA* region was amplified using primers DBF-seq1 (5′ GCAGTCTGTACC GACGCT 3′) and DBF-28 (5′ GAGTGCGACGGGATGGAC 3′). Sequencing of these DNA fragments was performed at BioAsia Biotech (Shanghai). The PCR products were purified with a glass milk kit (MBI, Lithuania). The NCBI GenBank accession numbers for 16S rDNA and the *dbdA* region are AY769994 and DQ086214, respectively.

Construction of the phylogenetic tree. Multiple alignments of 16S rDNA sequences were performed using ClustalX [30] and SeaView alignment editor [8]. Aligned sequences were analyzed using the neighbor-joining method [25] assisted by MEGA version 2.1 [19].

Fig. 2. Phylogenetic relations of *J. terrae* XJ-1 and related strains inferred from 16S rDNA sequences. The phylogenetic tree was constructed using neighbor-joining method. Numbers above the branches are the percentage of 1,000 bootstrap replicates (>50%). XJ-1 = *J. terrae* XJ-1; DSM 13953<sup>T</sup> = *J. brevis* DSM 13953<sup>T</sup>; CS12<sup>T</sup> = *J. terrae* CS12<sup>T</sup>; DPO 1361 = *Terrabacter* sp. DPO 1361; YY-1 = *Janibacter* sp. YY-1; YK3 = *Terrabacter* sp. YK3; DSM 46097 = *Oerskovia jenensis* DSM 46097 (outgroup).

Sequence divergences between species pairs were calculated using the uncorrected p-distance model. Branch support was assessed using 1,000 bootstrap replicates [4]. 16S rDNA sequences of *J. brevis* DSM13953<sup>T</sup>, *J. terrae* CS12<sup>T</sup>, *Terrabacter* sp. DPO1361, *Janibacter* sp. YY-1, *Terrabacter* sp. YK3, and *Oerskovia jenensis* DSM 46097 were retrieved from the National Center for Biotechnology Information GenBank under accession numbers AJ310085, AF176948, Y08853, AB089480, AB070459, and AJ314850, respectively.

### Results

#### Isolation and characterization of DF-degrading bacterium. The bacterium XJ-1 was isolated from lake sediment by enrichment cultures with DF as the sole source of carbon and energy and purified by picking up single colonies on agar plates. In MSM containing 1 g L⁻¹ DF, this strain grew rapidly at 30°C, with a doubling time of 12.0 hours; it released certain orange substance(s) with maximal absorption at 324, 462, and 583 nm. After 120 hours, 100 mg L⁻¹ DF was almost completely degraded by the bacterium (Fig. 1). The sequence of 16S rDNA showed that this bacterium is closely related to *J. brevis* strain DSM13953<sup>T</sup>, *J. terrae* CS12<sup>T</sup>, and *Terrabacter* sp. strain DPO1361, differing by only 1 or 2 of <1480 bases. *J. brevis* and *Terrabacter* sp. strain DPO1361 have been proposed to be the heterotypic synonym to *J. terrae* [20]. The similarity of 16S rDNA sequence supports the assignment of the strain XJ-1 to the genus *Janibacter* as a strain of the species *J. terrae*. *Janibacter* sp. YY-1 is a DF-degrading strain that produces catechol as the final product. Compared with YY-1, there are 36 substitutions or deletions and insertions of 1516 bases in the 16S rDNA of strain XJ-1. Fig. 2 shows a phylogenetic analysis of strain XJ-1 and other DF-degrading *Janibacter* and *Terrabacter* strains.

Because *dbdA* genes are found in many DF-degrading actinomycetes, we used a pair of PCR primers, DBF-seq1 and DBF-28, to detect the presence of the *dbdA* gene cluster, resulting in a 1.06-kb DNA fragment. Sequencing of this fragment showed almost identical

S. Jin et al.: DF Metabolism in *Janibacter terrae*

Abundance

| 2600000 |
| 2200000 |
| 1800000 |
| 1400000 |
| 1000000 |
| 600000  |
| 200000  |

I

H

E

D

C

12.0    16.0    20.0    24.0    28.0    32.0

Time(min)

(I)

Abundance

| 260000 |
| 220000 |
| 180000 |
| 140000 |
| 100000 |
| 60000  |
| 20000  |

A

G

B

F

12.0    16.0    20.0    24.0    28.0

Time(min)

(II)

Fig. 3. GC-MS chromatogram. (I) TMS derivatives of acidified ethyl acetate extracts from growing cell cultures incubated with DF. C = 2-hydroxy-6-(2′-hydroxyphenyl)-6-oxo-2,4-hexadienoic acid; D = 2-oxo-6-(2′-hydroxyphenyl)-6-hydroxy-3,5-hexadienoic acid; E = 3-(chroman-4-on-2-yl) pyruvate; H = salicylic acid; I = gentisic acid. (II) Neutral ethyl acetate extracts from cells grown on DF. A = DF; B = 2,2′,3-trihydroxybiphenyl; F = 1,2-dihydroxy-1,2-dihydrobenzofuran; G = 2-hydroxy-dibenzofuran.

angular dioxygenation
A

F
lateral dioxygenation
1,2-dihydroxy-1,2-
dihydrodibenzofuran

B
2,2',3-trihydroxybiphenyl

C
2-hydroxy-6-(2'-hydroxyphenyl)-
6-oxo-2,4-hexadienoic acid

D
2-oxo-6-(2'-hydroxy
phenyl)-6-hydroxy-3,5-
hexadienoic acid

E
3-(chroman-4-on-2-yl)pyruvate

G
OH
O
2-hydroxydibenzofuran

H
salicylic acid

I
gentisic acid

Fig. 4. Proposed metabolic pathways of DF by *J. terrae* XJ-1.

sequence to *dbdA1* (3' end)-*dbdA2-dbdA3* (5' end) on a large plasmid of *Terrabacter* YK3 [14], except for a G > C substitution at nt.103 of *dbdA3*. By growing it in a rich medium at high temperature, the DF-degrading capability and the *dbdA* cluster could be removed from the strain XJ-1 (T. Zhu, unpublished data), which suggests that it should also be plasmid borne.

Metabolites produced on degradation of DF by the bacterium. GC-MS analysis of the extracts showed many peaks of metabolites (Fig. 3). We identified some of them by comparing their mass spectra with those of authentic compounds and documented data from the National Institute of Standards and Technology library. Metabolites C, D, E, H, and I in the acidic extract of the growing cell culture were identified as (1) 2-hydroxy-6-(2'-hydroxyphenyl)-6-oxo-2,4-hexadienoic acid (trimethylsilyl [TMS] derivative mass spectrum [m/z]: 450, 435, 407, 333, 257, 147, 73), (2) 2-oxo-6-(2'-hydroxyphenyl)-6-hydroxy-3,5-hexadienoic acid (TMS derivative mass spectrum [m/z]: 450, 435, 407, 333, 317, 147, 73), (3) 3-(chroman-4-on-2-yl) pyruvate (TMS derivative mass spectrum [m/z]: 378, 363, 293, 261, 245, 193, 171, 147, 73), (4) salicylic acid, and (5) gentisic acid, respectively (Fig. 3I). Metabolites B, G, and F in the neutral extract were identified as (1) 2,2',3-trihydroxybiphenyl, (2) 2-hydroxydibenzofuran, and (3) 1,2-dihydroxy-1,2-dihydrobenzofuran, respectively (Fig. 3II). Salicylic acid and gentisic acid were accumulated during incubation of strain XJ-1 with DF.

S. Jin et al.: DF Metabolism in *Janibacter terrae*

(Fig. 1). The metabolites produced from degradation of DF by the bacterium were not found in the control using autoclaved cells. Based on the identified metabolites and the previously reported DF-degrading pathway [33], we propose DF-metabolic pathways in the strain XJ-1 (Fig. 4).

### Discussion

In the present study, DF-utilizing strain XJ-1 was isolated from lake sediment and identified as *J. terrae* based on its 16S rDNA sequence. Like other DF-degrading bacteria, *J. terrae* XJ-1 may be used to enhance the degradation or cometabolisation of PCDDs and PCDFs and other related compounds. The strain XJ-1, growing on DF as the sole source of carbon and energy, produced 2,2′,3-trihydroxybiphenyl, 2-hydroxy-6-(2′-hydroxyphenyl)-6-oxo-2,4-hexadienoic acid, 3-(chroman-4-on-2-yl) pyruvate, and salicylic acid as intermediate and gentisic acid as final products. In this route, the degradation starts with 4,4a-dioxygenation of DF to produce 2,2′,3-trihydroxybiphenyl, which is metacleaved and converted to 2-hydroxy-6-(2′-hydroxyphenyl)-6-oxo-2,4-hexadienoic acid. The latter product is then converted into salicylic acid by spontaneous fission of the hemiacetal bond, metafission, and hydrolysis. Salicylic acid is further converted to gentisic acid.

However, on the basis of detection of 1,2-dihydroxy-1,2-dihydrodibenzofuran, we postulate that there is another subordinate route by way of lateral dioxygenation at the 1,2-carbon atom position: 1,2-dihydroxy-1,2-dihydribenzofuran is converted to 2-hydroxydibenzofuran and 1,2-dihydroxydibenzofuran by way of spontaneous conversion, and 1,2-dihydroxydibenzofuran is further converted to salicylic acid. This pathway is analogous to that of cometabolic degradation of DF by biphenyl-utilizing bacteria [1, 27]. Because only traces of metabolites of the lateral dioxygenation pathway were detected, this route could be either minor in the metabolism of DF, or it may proceed much more rapidly so that intermediates do not accumulate. There have been a few reports on transformation of DF by both angular and lateral dioxygenation in other bacteria [7]. *Janibacter* strains YY-1 [33] and XJ-1 also use both pathways. However, strain XJ-1 highly accumulates gentisic acid instead of catechol in the final products. Compared with YY-1 and those DF-degrading *Janibacter* and *Terrabacter* strains isolated from Europe [20, 26], XJ-1 is more similar to YY-1 in the DF-metabolizing pathway and to European strains in the 16S rDNA sequence. It has been found that DF-degrading genes are dispersed on both chromosomal and extrachromosomal DNA. The *dbdA* gene cluster is located on a large plasmid in *Terrabacter* sp. YK3 [14]. Between XJ-1 and YK3, there are 26 substitutions or deletions and insertions of 1486 bases in 16S rDNA, but only 1 of 1061 bases in the *dbdA* region, which strongly suggests a lateral transfer, probably by way of transmission of the large plasmid, between bacteria.

### ACKNOWLEDGMENTS

This study was supported by the Environmental Protection Program of the National High Tech Development Project (863) of China under Grants No. 2002AA601170 and 2003AA640601.

### Literature Cited

1. Becher D, Specht MC, Hammer E, Francke W, Schauer F (2000) Cometabolic degradation of dibenzofuran by biphenyl-cultivated *Ralstonia* sp. strain SBUG 290. Appl Environ Microbiol 66:4528–4531
2. Bressler DC, Fedorak PM (2000) Bacterial metabolism of fluorene, dibenzofuran, dibenzothiophene, and carbazole. Can J Microbiol 46:397–409
3. Brzuzy LP, Hites RA (1996) Global mass balance for polychlorinated dibenzo-p-dioxins and dibenzofurans. Environ Sci Technol 30:1797–1804
4. Felsenstein J (1985) Confidence limits on phylogenies: An approach using the bootstrap. Evolution 39:783–791
5. Fortnagel P, Harms H, Wittich RM, Krohn S, Meyer H, Sinnwell V, et al. (1990) Metabolism of dibenzofuran by *Pseudomonas* sp. strain HH69 and the mixed culture HH27. Appl Environ Microbiol 56:1148–1156
6. Fukuda K, Nagata S, Taniguchi H (2002) Isolation and characterization of dibenzofuran-degrading bacteria. FEMS Microbiol Lett 208:179–185
7. Fuse H, Takimura O, Murakami K, Inoue H, Yamaoka Y (2003) Degradation of chlorinated biphenyl, dibenzofuran, and dibenzo-p-dioxin by marine bacteria that degrade biphenyl, carbazole, or dibenzofuran. Biosci Biotechnol Biochem 67:1121–1125
8. Galtier N, Gouy M, Gautier C (1996) SEAVIEW and PHYLOWIN: Two graphic tools for sequence alignment and molecular phylogeny. Comput Appl Biosci 12:543–548
9. Habe H, Ide K, Yotsumoto M, Tsuji H, Hirano H, Widada J, et al. (2001) Preliminary examinations for applying a carbazole-degrader *Pseudomonas* sp. strain CA10 to dioxin-contaminated soil remediation. Appl Microbiol Biotechnol 56:788–795
10. Habe H, Chung JS, Lee JH, Kasuga K, Yoshida T, Nojiri H, et al. (2001) Degradation of chlorinated dibenzofurans and dibenzo-p-dioxins by two types of bacteria having angular dioxygenases with different features. Appl Environ Microbiol 67:3610–3617
11. Halden RU, Dwyer DF (1997) Biodegradation of dioxin-related compounds: A review. Bioremed J 1:11–25
12. Halden RU, Halden BG, Dwyer DF (1999) Removal of dibenzofuran, dibenzo-p-dioxin, and 2-chlorodibenzo-p-dioxin from soils inoculated with *Sphingomonas* sp. strain RW1. Appl Environ Microbiol 65:2246–2249
13. Hong HB, Chang YS, Nam IH, Fortnagel P, Schmidt S (2002) Biotransformation of 2,7-dichloro- and 1,2,3,4-tetrachlorodibenzop-dioxin by *Sphingomonas wittichii* RW1. Appl Environ Microbiol 68:2584–2588
14. Ida T, Mukouzaka Y, Nakamura K, Kudo T (2002) Plasmid-borne genes code for an angular dioxygenase involved in dibenzofuran

degradation by Terrabacter sp. strain YK3. Appl Environ Microbiol 68:3716–3723

15. Kasuga K, Nojiri H, Yamane H, Kodama T (1997) Cloning and characterization of the genes involved in the degradation of dibenzofuran by Terrabacter sp. strain DBF63. J Ferment Bioeng 84:387–399

16. Kasuga K, Habe H, Chung JS, Yoshida T, Nojiri H, Yamane H, et al. (2001) Isolation and characterization of the genes encoding a novel oxygenase component of angular dioxygenase from the gram-positive dibenzofuran-degrader Terrabacter sp. strain DBF63. Biochem Biophy Res Commun 283:195–204

17. Keim T, Francke W, Schmidt S, Fortnagel P (1999) Catabolism of 2,7-dichloro- and 2,4,8-trichlorodibenzofuran by Sphingomonas sp. strain RW1. J Ind Microbiol Biotechnol 23:359–363

18. Kimura N, Urushigawa Y (2001) Metabolism of dibenzo-p-dioxin and chlorinated dibenzo-p-dioxin by a gram-positive bacterium, Rhodococcus opacus SAO 101. J Biosci Bioeng 92:138–143

19. Kumar S, Tamura K, Jakobsen IB, Nei M (2001) MEGA2: Molecular Evolutionary Genetics Analysis software. Bioinformatics 17:1244–1245

20. Lang E, Kroppenstedt RM, Swiderski J, Schumann P, Ludwig W, Schmid A, et al. (2003) Emended description of Janibacter terrae, including ten dibenzofuran-degrading strains and Janibacter brevis as its later heterotypic synonym. Int J Syst Evol Microbiol 53:1999–2005

21. Mauel MJ, Giovannoni SJ, Fryer JL (1996) Development of polymerase chain reaction assays for detection, identification, and differentiation of Piscirickettsia salmonis. Dis Aquat Org 26:189–195

22. Megharaj M, Wittich RM, Blasco R, Pieper DH, Timmis KN (1997) Superior survival and degradation of dibenzo-p-dioxin and dibenzofuran in soil by soil-adapted and non-adapted Sphingomonas sp. strain RW1. Appl Microbiol Biotech 48:109–114

23. Monna L, Omori T, Kodama T (1993) Microbial degradation of dibenzofuran, fluorene, and dibenzo-p-dioxin by Staphylococcus auricularis DBF63. Appl Environ Microbiol 59:285–289

24. Resnick SM, Gibson DT (1996) Regio- and stereospecific oxidation of fluorene, dibenzofuran, and dibenzothiophene by naphthalene dioxygenase from *Pseudomonas* sp. strain NCIB 9816-4. Appl Environ Microbiol 62:4073–4080

25. Saitou N, Nei M (1987) The neighbor-joining method: A new method for reconstructing phylogenetic trees. Mol Biol Evol 4:406–425

26. Schmid A, Rothe B, Altenbuchner J, Ludwig W, Engesser KH (1997) Characterization of three distinct extradiol dioxygenases involved in mineralization of dibenzofuran by *Terrabacter* sp. strain DPO360. J Bacteriol 179:53–62

27. Stope MB, Becher D, Hammer E, Schauer F (2002) Cometabolic ring fission of dibenzofuran by gram-negative and gram-positive biphenyl-utilizing bacteria. Appl Microbiol Biotechnol 59:62–67

28. Strubel V, Rast HG, Fietz W, Knackmuss HJ, Engesser KH (1989) Enrichment of dibenzofuran utilizing bacteria with high co-metabolic potential towards dibenzodioxin and other anellated aromatics. FEMS Microbiol Lett 58:233–238

29. Strubel V, Engesser KH, Fischer P, Knackmuss HJ (1991) 3-(2-hydroxyphenyl) catechol as substrate for proximal meta ring cleavage in dibenzofuran degradation by *Brevibacterium* sp. strain DPO 1361. J Bacteriol 173:1932–1937

30. Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The Clustal X windows interface: Flexible strategies for multiple sequences alignment aided by quality analysis tools. Nucleic Acids Res 25:4876–4882

31. Wilkes H, Wittich RM, Timmis KN, Fortnagel P, Francke W (1996) Degradation of chlorinated dibenzofurans and dibenzo-p-dioxins by *Sphingomonas* sp. strain RW1. Appl Environ Microbiol 62:367–371

32. Wittich RM (1998) Degradation of dioxin-like compounds by microorganisms. Appl Microbiol Biotechnol 49:489–499

33. Yamazoe A, Yagi O, Oyaizu H (2004) Degradation of polycyclic aromatic hydrocarbons by a newly isolated dibenzofuran-utilizing Janibacter sp. strain YY-1. Appl Microbiol Biotechnol 65:211–218
