
ADVANCES IN VIRUS RESEARCH, VOL 66

IN A NUTSHELL: STRUCTURE AND ASSEMBLY OF THE VACCINIA VIRION

Richard C. Condit, * Nissin Moussatche, *, † and Paula Traktman ‡

*Department of Molecular Genetics and Microbiology  
University of Florida, Gainesville, Florida 32610  

†Laboratório de Biologia Molecular de Vírus, Instituto de Biofísica  
Carlos Chagas Filho, CCS, UFRJ, Rio de Janeiro, RJ 21941-590, Brazil  

‡Department of Microbiology and Molecular Genetics  
Medical College of Wisconsin, Milwaukee, Wisconsin 53226  

I. Preface  
II. Introduction  
    A. Poxvirus Biology and Replication: An Overview  
    B. Genetic Nomenclature  
    C. Poxvirus Genetics: Special Considerations  
III. Vaccinia Virion Structure  
    A. Imaging Studies of MV Structure  
    B. Chemical Composition of MV  
    C. Controlled Degradation of MV  
    D. Structure of the Genome in the Core  
    E. MV Proteins  
    F. Model for MV Structure  
IV. Vaccinia Virus Morphogenesis: An Overview  
V. Formation of Factories  
VI. Crescent Formation  
    A. One Membrane or More?  
    B. Source of the Virion Membrane  
    C. Regulatory Proteins in Viral Membrane Formation  
    D. Membrane Proteins Essential for Membrane Biogenesis: A14 and A17  
    E. D13: The Crescent Scaffold Protein  
VII. Immature Virion Formation: A Complex Process  
    A. Seven-Protein Complex  
    B. A10: p4a/4a  
VIII. IV→IVN Transition: Genome Encapsulation  
    A. Membrane Proteins  
    B. Core Proteins  
IX. MV Formation  
    A. Assembly of the Transcription Apparatus Within the Core  
    B. Core Proteins Involved in the IV to MV Transition  
    C. Proteolysis of Virion Protein Precursors; Involvement of Genes I7L and G1L  
    D. Membrane Proteins Involved in the IV to MV Transition  
    E. Restructuring the Particle Surface  
    F. Core Proteins that are not Essential for Virion Morphogenesis but are Required for Transcriptional Competence of the Virion Core

RICHARD C. CONDIT ET AL.

G. Membrane Proteins Affecting Virus Binding, Entry, and Fusion  
X. Transport, Occlusion, and Secondary Wrapping of MV  
A. A27  
B. A26  
XI. Virion Proteins of Unknown Function  
XII. Summary and Perspectives  
References  

---

**ABSTRACT**

Poxviruses comprise a large family of viruses characterized by a large, linear dsDNA genome, a cytoplasmic site of replication and a complex virion morphology. The most notorious member of the poxvirus family is variola, the causative agent of smallpox. The laboratory prototype virus used for the study of poxviruses is vaccinia, the virus that was used as a live, naturally attenuated vaccine for the eradication of smallpox.

Both the morphogenesis and structure of poxvirus virions are unique among viruses. Poxvirus virions apparently lack any of the symmetry features common to other viruses such as helical or icosahedral capsids or nucleocapsids. Instead poxvirus virions appear as “brick shaped” or “ovoid” membrane-bound particles with a complex internal structure featuring a walled, biconcave core flanked by “lateral bodies.” The virion assembly pathway involves a remarkable fabrication of membrane-containing crescents and immature virions, which evolve into mature virions in a process that is unparalleled in virology. As a result of significant advances in poxvirus genetics and molecular biology during the past 15 years, we can now positively identify over 70 specific gene products contained in poxvirus virions, and we can describe the effects of mutations in over 50 specific genes on poxvirus assembly. This review summarizes these advances and attempts to assemble them into a comprehensible and thoughtful picture of poxvirus structure and assembly.

---

I. PREFACE

Both the morphogenesis and structure of poxvirus virions are unique among viruses. Poxvirus virions apparently lack any of the symmetry features common to other viruses such as helical or icosahedral capsids or nucleocapsids. Instead poxvirus virions appear as “brick shaped” or

"ovoid" membrane-bound particles with a complex internal structure featuring a walled, biconcave core flanked by "lateral bodies." The virion assembly pathway involves a remarkable fabrication of membrane-containing crescents and immature virions, which evolve into mature virions in a process that is unparalleled in virology.

Poxvirus structure and assembly has been the subject of intense scrutiny since the virus was first subjected to detailed examination by electron microscopy in 1950s and 1960s, yet the most recent comprehensive reviews of the subject are at least 15 years old (Dales and Pogo, 1981; Fenner *et al.*, 1989; see also Moss, 2001; Sodeik and Krijnse-Locker, 2002). During these 15 years, key advances in poxvirology have brought us to a point where a comprehensive review of virus structure and assembly is appropriate. First, advances in genome and protein sequencing now readily permit the unambiguous assignment of protein products to specific genes on the poxvirus genome (Chung *et al.*, 2006; Goebel *et al.*, 1990; Jensen *et al.*, 1996; Takahashi *et al.*, 1994). Second, advances in poxvirus genetics have yielded both well-characterized collections of temperature-sensitive mutants and techniques for targeted construction of conditional lethal mutants with alterations in specific poxvirus genes (Condit and Niles, 1990; Hasset and Condit, 1994; Lackner *et al.*, 2003; Rodriguez and Smith, 1990b; Zhang and Moss, 1991a). As a result of these advances, we can now positively identify over 70 specific gene products contained in poxvirus virions, and we can describe the effects of mutations in over 50 specific genes on poxvirus assembly. This review summarizes these advances and attempts to assemble them into a comprehensible and thoughtful picture of poxvirus structure and assembly.

II. INTRODUCTION

A. Poxvirus Biology and Replication: An Overview

Poxviruses comprise a large family of viruses that infect a wide variety of vertebrate and invertebrate hosts. Members of the family Poxviridae are characterized by a large, linear dsDNA genome, a cytoplasmic site of replication and a complex virion morphology. The most notorious member of the poxvirus family is variola, the causative agent of smallpox. The laboratory prototype virus used for the study of poxviruses is vaccinia, the virus that was used as a live, naturally attenuated vaccine for the eradication of smallpox. Because the vast majority of genetic and biochemical studies on poxvirus structure and

morphogenesis have been performed with vaccinia, this review will confine itself exclusively to this virus. Members of the poxvirus family are sufficiently similar such that lessons learned from vaccinia can easily be applied to other poxviruses.

The vaccinia virus genome is approximately 200 kb in length and encodes approximately 200 genes (there is some strain variation in DNA size and coding capacity) (Goebel et al., 1990; Johnson et al., 1993; Lefkowitz et al., 2005). Because of the cytoplasmic site of virus replication, vaccinia viral mRNAs are not spliced, and therefore vaccinia genes do not contain introns, greatly simplifying interpretation of genomic sequence. Genes are closely spaced on the genome, and each gene appears to be controlled by its own transcriptional promoter. Comparison of poxvirus genomes reveals a set of 91 genes that are conserved throughout the Chordopoxvirinae subfamily (vertebrate poxviruses); a subset of 49 of these genes are conserved throughout the entire Poxviridae family (including both vertebrate and invertebrate poxviruses) (Upton et al., 2003). The remaining nonconserved genes, many of which are nonessential for replication in cell culture, dictate individual virus characteristics of host range and pathogenicity.

Poxvirus virions exist in three infectious forms: mature virions (MV), wrapped virions (WV), and extracellular virions (EV).<sup>1</sup> MV, the simplest form of the virus, are membraned particles containing a biconcave, DNA-containing core flanked by lateral bodies, which fill the concavities of the core. MV are normally found exclusively inside cells and are liberated only by cell lysis. WV consist of MV which are surrounded by two additional lipid bilayers derived from trans-Golgi cisternae. WV, whose outer membranes contain characteristic viral proteins, are precursors of EV and are also found within the cell. EV consist of WV which have been exocytosed via fusion of the outermost WV membrane with the plasma membrane, leaving an MV wrapped in one additional membrane. A fraction of EV are found attached to the cell surface, while some are found free in the extracellular medium. EV are thought to be important for spread of the virus within an organism,

<sup>1</sup> This nomenclature follows a recent proposal by Moss (2005). Previously, MV was called intracellular mature virus (IMV), WV was called intracellular enveloped virus (IEV), and EV included both cell-associated extracellular enveloped virus (CEV) and extracellular enveloped virus (EEV). In older literature, MV was called “intracellular naked virus” (INV) (Sodeik et al., 1993), thus MV is identical to both IMV and INV.

while MV are thought to be important for long-term stability and transmission of the virus between hosts in the environment. It should be noted, however, that the EV membrane is disrupted when EV bind to cells, enabling MV to make direct contact with the plasma membrane and enter cells (Law, M., Carter, G. C., Roberts, K. L., Hollinshead, M., and Smith, G. L., personal communication). The extra membrane composition, assembly, and transport of WV and EV have been reviewed (*Smith et al.*, 2002); therefore, this review will confine itself exclusively to the structure and assembly of MV.

MV enter cells by fusion of the MV membrane with the plasma membrane of the host cell, releasing the core (and lateral bodies) into the cytoplasm and activating the virus' transcriptional program (*Armstrong et al.*, 1973; *Carter et al.*, 2005). The virion cores contain the full complement of virus-coded enzymes required for synthesis and modification of early mRNA, including a nine subunit RNA polymerase, a virus early transcription factor (VETF), a capping enzyme, and a poly(A) polymerase. Early genes encode enzymes required for DNA replication, and thus as early gene expression peaks, viral DNA replication ensues in cytoplasmic sites termed "factories." Early genes also encode intermediate transcription factors, and intermediate genes, in turn, encode late transcription factors, so that intermediate and late genes are expressed in succession after the prerequisite initiation of viral DNA replication. Thus, the full complement of viral genes are transcribed in a temporal cascade, with the early, intermediate and late classes being distinguished by class-specific transcriptional promoters and virally encoded transcription factors. Furthermore, only replicated genomes are competent templates for intermediate and late transcription. These two classes of genes together encode virion structural proteins, virion enzymes, and assembly factors required for assembly of new virus particles (for a review of poxvirus replication, see *Moss*, 2001).

## B. Genetic Nomenclature

The first complete poxvirus genome sequence to be published was that of the Copenhagen strain of vaccinia virus (*Goebel et al.*, 1990). With the publication of this sequence, a genetic nomenclature was adopted in which genes were named according to their position relative to the left end of individual Hind III restriction fragments of the genome and also according to their transcriptional orientation. Thus, for example, the virus DNA polymerase is encoded by gene *E9L*, the

ninth gene from the left end of the Hind III E fragment, and the gene is transcribed in a leftward direction relative to the standard orientation of the genome.

While Copenhagen was the first vaccinia strain sequenced, most of the genetic and biochemical analysis of vaccinia has been performed using the WR strain, a strain that is not identical in genetic content to Copenhagen. Initially, the Copenhagen system of nomenclature was applied to WR; however, strain variation, sequencing glitches, and the absence of complete genome sequence made this usage occasionally confusing. For example, the vaccinia strain WR genes initially dubbed \( A5L \) and \( A8L \) are orthologous to the Copenhagen genes \( A4L \) and \( A7L \), while further to the right in the Hind III A fragment, WR genes orthologous to Copenhagen genes were given the precise Copenhagen name. Despite this initial confusion, over the years the Copenhagen nomenclature has become the most familiar among poxvirologists, and a custom has emerged which is to refer to genes in WR using the orthologous Copenhagen gene identification. Accordingly, the WR genes previously called \( A5L \) and \( A8L \) are now referred to as \( A4L \) and \( A7L \).

More recently, faced with an explosion of poxvirus genome sequences available and the impossibility of maintaining the Copenhagen nomenclature across such a diversity of sequences, a new nomenclature has emerged in which genes are simply numbered in sequence from left to right on the genome, along with an identifier specifying the virus strain. Thus while the \( E9L \) gene in Copenhagen is now officially “VV-Cop-E9L,” in WR it is “VACWR065.”

This review deals almost exclusively with experiments done with the WR strain of vaccinia. For the sake of clarity, we have maintained the custom of referring to WR genes using the Hind III designation of the Copenhagen ortholog. Following established convention, genes are referred to using the entire designation, for example, “\( E9L \)” for the DNA polymerase gene, while proteins are specified by omitting the “L” or “R,” for example, “E9” for the DNA polymerase enzyme. For the sake of precision, in **Table I** we have included a column specifying the official WR gene name for each protein.

### C. Poxvirus Genetics: Special Considerations

Conditional lethal mutants have been isolated which encode alterations in over half of the essential genes in vaccinia virus. These mutants are of two different types, inducible and temperature sensitive. The two different types of mutants function by fundamentally different

TABLE I  
VACCINIA VIRION PROTEINS  

| Gene* | WR designation | MW† | Location | Mutants‡ | Comment§ |
|-------|-----------------|-----|----------|----------|----------|
| D13L  | VACWR 118      | 62  | Absent   | ind, ts  | Rifampicin target, IV scaffold protein |
| A11R  | VACWR 130      | 36  | Absent   | ind      | Hydrophobic, phosphorylated (independent of F10), interacts with A32, itself |
| A2.5L | VACWR 121      | 9   | Virion enzyme | ind     | Thiol oxidoreductase |
| A32L  | VACWR 155      | 31  | Virion    | ind      | Putative ATPase, DNA encapsidation |
| A17L  | VACWR 137      | 23  | Membrane  | ind      | TM(2), F10-dependent phosphorylation, cleaved, complex with A14, A27 |
| A14L  | VACWR 133      | 10  | Membrane  | ind      | TM(2), F10-dependent phosphorylation, dimerizes, complex with A17, A27 |
| A13L  | VACWR 132      | 8   | Membrane  | ind, ts  | TM(1), phosphorylated (independent of F10), DNA encapsidation |
| A9L   | VACWR 128      | 12  | Membrane  | ind      | TM(1) |
| L1R   | VACWR 088      | 27  | Membrane  | ind      | TM(1), myristylated, redox substrate |
| H3L   | VACWR 101      | 37  | Membrane  | ind, ko  | TM(1), heparin binding |
| A21L  | VACWR 140      | 14  | Membrane  | ind      | TM(1), redox substrate, fusion complex |
| L5R   | VACWR 092      | 15  | Membrane  | ind      | TM(1), redox substrate, fusion complex |
| A28L  | VACWR 151      | 16  | Membrane  | ind, ts  | TM(1), redox substrate, fusion complex |
| H2R   | VACWR 100      | 22  | Membrane  | ind      | TM(1), redox substrate, fusion complex |
| G3L   | VACWR 079      | 13  | Membrane  | ts       | TM(1), fusion complex |
| G9R   | VACWR 087      | 39  | Membrane  | ind      | TM(1), myristylated, fusion complex |
| J5L   | VACWR 097      | 15  | Membrane  | ind      | TM(1), myristylated, fusion complex |
| A16L  | VACWR 136      | 43  | Membrane  | ind      | TM(1), myristylated, fusion complex |
| I2L   | VACWR 071      | 8   | Membrane  | ind      | TM(1) |
| D8L   | VACWR 113      | 35  | Membrane  | ko       | TM(1), chondroitin binding |

| Gene* | WR designation | MW† | Location | Mutants‡ | Comment§ |
|-------|----------------|-----|----------|----------|----------|
| A27L  | VACWR 150      | 13  | Membrane | ind, ko  | No TM, complex with A17, A14, heparin binding |
| A26L  | VACWR 149      | 58  | Membrane | ko       | p4c, TM(?) |
| A14.5L | VACWR 134     | 6   | Membrane | ko       | TM(2) |
| I5L   | VACWR 074      | 9   | Membrane |          | TM(2) |
| E10R  | VACWR 066      | 11  | Membrane enzyme | ind    | Thiol oxidoreductase, no TM |
| G4L   | VACWR 081      | 14  | Membrane enzyme | ind    | Thiol oxidoreductase, no TM |
| H5R   | VACWR 103      | 22  | Core     | ts       | F10, B1-dependent phosphorylation, roles in DNA replication, transcription, morphogenesis |
| G5R   | VACWR 082      | 50  | Core     | ind      |         |
| A30L  | VACWR 153      | 9   | Core     | ind, ts  | 7 complex, F10-dependent phosphorylation, partially NP40 extractable, in viroplasm, IV's |
| G7L   | VACWR 085      | 42  | Core     | ind, ts  | 7 complex, F10-dependent phosphorylation, cleaved |
| J1R   | VACWR 093      | 18  | Core     | ind, ts  | 7 complex, partially NP40 extractable |
| A15L  | VACWR 135      | 11  | Core     | ind      | 7 complex |
| D2L   | VACWR 107      | 17  | Core     | ind, ts  | 7 complex |
| D3R   | VACWR 108      | 28  | Core     | ts       | 7 complex |
| A10L  | VACWR 129      | 102 | Core     | ind, ts  | p4a/4a, cleaved, complex A4, core wall? |
| I6L   | VACWR 075      | 43  | Core     | ts       | Telomere binding, DNA encapsidation |
| I1L   | VACWR 070      | 36  | Core     | ind      | DNA binding |
| F17R  | VACWR 056      | 11  | Core     | ind      | H1 dependent, partially F10 dependent phosphorylation, DNA binding |

A4L    VACWR 123   31    Core    ind     Core wall spike protein(?), membrane associated(?), F10, H1-dependent phosphorylation, complex with A10
A3L    VACWR 122   73    Core    ts      p4b/4b, cleaved, core wall
L4R    VACWR 091   28    Core    ind     VP8, cleaved
E8R    VACWR 064   32    Core    ts      TM(2), phosphorylated
E11L   VACWR 067   15    Core    ts      
L3L    VACWR 090   41    Core    ind     
A12L   VACWR 131   20    Core            Cleaved
J6R    VACWR 098   147   Core enzyme     ts      RNA polymerase subunit, rpo 147
A24R   VACWR 144   133   Core enzyme     ts      RNA polymerase subunit, rpo 132
H4L    VACWR 102   94    Core enzyme     ind, ts RNA polymerase subunit, rap 94, early gene specificity factor, required for early transcription termination
A29L   VACWR 152   35    Core enzyme     ts      RNA polymerase subunit, rpo 35
E4L    VACWR 060   30    Core enzyme     ts      RNA polymerase subunit, rpo 30
J4R    VACWR 096   21    Core enzyme     ts      RNA polymerase subunit, rpo 22
A5R    VACWR 124   19    Core enzyme     ts      RNA polymerase subunit, rpo 19
D7R    VACWR 112   18    Core enzyme     ts      RNA polymerase subunit, rpo 18
G5.5R  VACWR 083   7     Core enzyme     ts      RNA polymerase subunit, rpo 7
A7L    VACWR 126   82    Core enzyme     ind     Early gene transcription initiation factor, VETF
D6R    VACWR 111   74    Core enzyme     ind, ts Early gene transcription initiation factor, VETF
D1R    VACWR 106   97    Core enzyme     ts      Capping enzyme, also early gene termination factor, intermediate gene initiation factor
D12L   VACWR 117   33    Core enzyme     ts      Capping enzyme, also early gene termination factor, intermediate gene initiation factor


(continues)

TABLE I (continued)

| Gene* | WR designation | MW† | Location | Mutants‡ | Comment§ |
|-------|-----------------|-----|----------|----------|----------|
| E1L   | VACWR 057      | 55  | Core enzyme |          | Poly(A) polymerase catalytic subunit |
| J3R   | VACWR 095      | 39  | Core enzyme | ko       | Poly(A) polymerase elongation subunit, also 2′O methyltransferase, transcription elongation factor |
| D11L  | VACWR 116      | 72  | Core enzyme | ts       | Early gene termination factor, NPHI |
| I8R   | VACWR 077      | 78  | Core enzyme | ts       | RNA helicase |
| A18R  | VACWR 138      | 57  | Core enzyme | ts       | Postreplicative transcription termination factor, RNA release factor |
| A22R  | VACWR 142      | 22  | Core enzyme | ind      | Holliday resolvase, concatemer resolution, palmityl protein |
| H6R   | VACWR 104      | 37  | Core enzyme | ko       | Topoisomerase |
| K4L   | VACWR 035      | 49  | Core enzyme | ko       | DNA nicking and joining enzyme, nonessential |
| G1L   | VACWR 078      | 68  | Core enzyme | ind      | Metalloproteinase(?) |
| I7L   | VACWR 076      | 49  | Core enzyme | ind, ts  | Cysteine proteinase(?), virion protein processing |
| F10L  | VACWR 049      | 52  | Core enzyme | ts, ind  | Ser/Thr/Tyr kinase, 7 complex |
| B1R   | VACWR 183      | 34  | Core enzyme | ts       | Ser/Thr kinase, essential for DNA replication |
| H1L   | VACWR 099      | 20  | Core enzyme | ind      | Tyr/Ser/Thr phosphatase |
| O2L   | VACWR 069      | 12  | Core enzyme | ko       | Glutaredoxin, nonessential, not conserved |
| A45R  | VACWR 171      | 14  | Core enzyme | ko       | Inactive superoxide dismutase, nonessential |

* Copenhagen designation.
† predicted, apparent MW may differ.
‡ ind, inducible; ts, temperature sensitive; ko, knockout.
§ TM, transmembrane domain, number of domains in parentheses.

mechanisms; an understanding of these mechanisms is important for proper evaluation of mutant phenotypes.

In inducible mutants, the gene in question is engineered such that its expression is controlled by the regulatory elements of a bacterial operon, commonly either lac or tet (Rodriguez and Smith, 1990a; Traktman et al., 2000; Zhang and Moss, 1991a). Thus, in the presence of inducer, the wild-type gene and protein in question are expressed, a permissive condition exists, and virus can be grown for study. In the absence of inducer, the gene is not expressed, a nonpermissive condition exists, and the mutant phenotype can be studied. Importantly, under nonpermissive conditions in an inducible mutant, the gene product in question is absent or present at only low levels.

In temperature-sensitive mutants, the gene in question contains a missense mutation that renders the protein product either unstable or nonfunctional at high temperature (Condit and Niles, 1990). Thus, virus is grown for study at low temperature, the permissive condition, and the mutant phenotype studied using infections at high temperature, the nonpermissive condition. In mutants where the protein in question is unstable at high temperature, the protein is effectively absent from the infection and thus mechanistically the mutant is similar to an inducible mutant. An important caveat to this statement, however, is that any residual protein is mutant in nature, whereas the residual protein present in the case of inducible mutants is wild type. For temperature-sensitive mutants in which the target protein is nonfunctional at the high temperature but nevertheless physically present, the mechanism of temperature sensitivity is significantly different than in an inducible mutant. Most temperature-sensitive mutants extant have been isolated by classical mutagenesis and random screening methods;², however, methods exist whereby temperature-sensitive mutants can be engineered into specific genes in a targeted fashion (DeMasi and Traktman, 2000; Grubisha and Traktman, 2003; Hassett and Condit, 1994; Hassett et al., 1997; Ishii and Moss, 2001; Punjabi et al., 2001).

² Temperature-sensitive mutants are referred to using either a mutant-specific designation or a generic protein-specific designation. The mutant-specific designation consists of a letter, either C, D, or E, signifying the original mutant collection, (Condit, Dales, or Ensinger) followed by “ts” and a number specific to the mutant, for example, Dts46 (Lackner et al., 2003). The generic, protein-specific notation consists of “ts” followed by the name of the protein, for example, tsA30.
Each mutant type has advantages and disadvantages. Advantages of inducible mutants include the relative ease with which a mutant may reliably be targeted to a specific gene and the fact that all experiments can be done at one temperature, thus avoiding the possibility of temperature-induced abnormalities in the infection. Disadvantages include the inability to use this system effectively for viral genes expressed early in infection and the inability to engender a tight phenotype for some late genes, due to some intrinsic leakiness in the system. Advantages of temperature-sensitive mutants include the ability sometimes to perform temperature shift-up and shift-down experiments to test for the requirement for a gene at different times during the life cycle and the existence of some mutants which synthesize stable but nonfunctional protein at the nonpermissive temperature. Disadvantages of temperature-sensitive mutants include the difficulty of isolating mutants in specific genes in a targeted fashion and the necessity of performing experiments at high temperature, which may induce mutant-independent abnormalities in the infection. In the end, the two types of mutants have complementary advantages and provide powerful tools for the study of gene function.

### III. VACCINIA VIRION STRUCTURE

#### A. Imaging Studies of MV Structure

Electron microscopic investigation of vaccinia virus structure has involved analysis of both whole mount preparations of purified virus and thin sections of purified virions or virus-infected cells. Each approach has strengths and limitations that are important to consider in deriving an overall model for the structure of the virus. In overall dimensions, the virion is highly asymmetric, described variously as ellipsoidal, brick shaped, or barrel shaped (see the following paragraph). As a result of this asymmetry, in most whole mount preparations of virus, the majority of particles appear to be oriented in a nonrandom fashion, presenting the broadest surface of the virion perpendicular to the electron beam, revealing clearly only the two widest dimensions. Whole mount preparations also provide detail relevant to the surface of the particle but usually reveal less information about the internal structure. By contrast, thin sections present all possible orientations and sections of the virion for viewing; however,
reconstructing an accurate three-dimensional (3D) view of a relatively complex and asymmetric structure can be daunting. While thin sections provide minimal information about the surface of the particle, they reveal significant information about internal structure.

Three studies provide estimates of the external dimensions of the virion that are consistent with historical measurements ([Dales and Pogo, 1981; Fenner et al., 1989]) and at the same time reflect some continuing uncertainty regarding the ultimate shape of the virion. Measurements of cryo-electron micrographs by Griffiths and coworkers suggest a brick- or pillow-shaped particle, with dimensions of \(310 \times 240 \times 140\) nm ([Griffiths et al., 2001b; Roos et al., 1996; Sodeik and Krijnse-Locker, 2002]). A study using atomic force microscopy suggests a slightly flattened ellipsoid with dimensions of \(350 \times 300 \times 265\) nm ([Malkin et al., 2003]). Lastly, recent cryo-electron tomography of the virus provides evidence that MV are barrel shaped, with dimensions of \(360 \times 270 \times 250\) nm ([Cyrklaff et al., 2005]). Malkin et al. (2003) note that if particles are air dried and examined by atomic force microscopy, they shrink more than twofold along the shortest axis but only slightly along the longer two axes, yielding final dimensions of approximately \(330 \times 260 \times 125\) nm, provocatively similar to the measurements reported by Griffiths and coworkers using cryo-electron microscopy ([Griffiths et al., 2001b; Roos et al., 1996]). In summary all three of these studies, using different approaches, suggest a flattened ellipsoid or barrel but leave some lingering uncertainty concerning the ratio of the length of the shortest axis to the longer axes.

The surface of purified MV particles, as viewed in whole mount preparations, assumes two fundamentally different appearances depending on the integrity of the particles and the method of preparation. Particles observed using negative staining, metal shadowing, freeze etch or deep etch electron microscopy, or atomic force microscopy possess randomly arranged surface ridges called “surface tubule elements” (STEs) (Fig. 1A–E) ([Dales, 1962; Heuser, 2005; Malkin et al., 2003; Medzon and Bauer, 1970; Noyes, 1962a,b; Stern and Dales, 1976; Westwood et al., 1964; Wilton et al., 1995]). This form of virus was named the “M” form by Westwood et al. (1964) for its mulberry like appearance. Most negatively stained, purified preparations of MV also contain virions which lack obvious STEs but instead possess a clearly delineated boundary surrounding the particle (Fig. 1A). These latter forms were dubbed the “C” form by Westwood et al. (1964), because the boundary looks like a capsule surrounding the particle.

44 RICHARD C. CONDIT ET AL.

Visualization of purified MV in whole mount using cryo-electron microscopy reveals particles that closely resemble the C form of virus, that is, lacking evidence of STEs and possessing a boundary layer or “surface domain” (Fig. 1F) (Dubochet et al., 1994; Griffiths et al., 2001b). In negatively stained preparations, treatments that are likely to disturb the integrity of the particle convert the M form of the virus to the C form (Dubochet et al., 1994; Westwood et al., 1964). Dubochet et al. (1994) interpreted these results to mean that the STEs are preparation artifacts induced by osmotic stress imposed during

![Figure 1](#fig1)

FIG 1. Vaccinia virus surface features. (A) Electron micrograph of purified virus, whole mount, negative stain. “Mulberry” (M) and “capsule” (C) forms are shown (Westwood et al., 1964). (B) Electron micrograph of a purified virion, whole mount, negative stain (Wilton et al., 1995). (C) Freeze etch electron micrograph of a purified virion (Nermut, 1973). (D) Deep etch electron micrograph of a purified virion (Heuser, 2005). (E) Atomic force micrograph of a purified virion (Malkin et al., 2003). (F) Cryo-electron micrograph of a whole mount preparation of purified virions (Griffiths et al., 2001b). Arrows denote the limits of the surface domain; arrowhead denotes the palisade layer. (A) Reprinted from Westwood et al. (1964) with permission. (B) Reprinted from Wilton et al. (1995) with permission. (C) Reprinted from Nermut (1973) with permission. (D) Reprinted from Heuser (2005) with permission. (E) Reprinted from Malkin et al. (2003) with permission. (F) Reprinted from Griffiths et al. (2001b) with permission.

negative staining and that the smooth surfaced C form of virus revealed by cryo-electron microscopy most accurately represents the surface of the virus. However, the persistence of STEs in metal shadowing, freeze etch and deep etch electron microscopy, and atomic force microscopy (Heuser, 2005; Malkin *et al.*, 2003; Medzon and Bauer, 1970), plus the biochemical evidence for a structure resembling STEs (Wilton *et al.*, 1995) (Sections III.C and IX.E.3) suggests that STEs are not artificial. As originally proposed by Westwood *et al.* (1964), the C form of virus present in negatively stained preparations most likely results from penetration of stain into damaged virions, which destroys the contrast that ordinarily highlights STEs. The C form appearance of particles viewed by cryo-electron microscopy probably results from an insensitivity of cryo-electron microscopy to fine surface detail (Malkin *et al.*, 2003).

Insights into the internal architecture of MV have been gained primarily through the use of thin sections, although some information can be obtained from whole mount preparations. Thin sections reveal two distinct boundaries, the membrane,³ which surrounds the entire MV particle, and the core wall, which surrounds an internal core. Cryosections, such as that shown in Fig. 2A, offer the clearest definition of the substructure of the membrane and the core wall, each of which comprises two layers or domains. Estimates of the thicknesses of each of these domains vary, and the numbers bear significance to the biochemical makeup of each domain (discussed in Section VI.A.3) (Cyrklaff *et al.*, 2005; Dubochet *et al.*, 1994; Fenner *et al.*, 1989; Hollinshead *et al.*, 1999; Ichihashi *et al.*, 1984). The outer domain of the membrane (outermost dark layer in Fig. 2A) is between 3 and 9 nm thick, the inner domain (light layer underneath the outer domain and surrounding the entire particle in Fig. 2A) 5–6 nm thick, and estimates of the thickness of the membrane in aggregate range from 10 to 20 nm. The outer core wall (dark layer surrounding the core in Fig. 2A) is clearly thicker than any of the four domains comprising

³ Dales and coworkers proposed the term “envelope” to describe the MV outer boundary, based on its lipid content and its function analogous to envelopes of other viruses (Dales and Pogo, 1981). However, the common usage of the term envelope in referring to WV and EV (heretofore called IEV, CEV, and EEV) invites confusion over the origin, structure, and function of the MV outer boundary. We use the term “membrane” to describe the MV outer boundary; however, it is critical to understand that this two-domain structure is clearly significantly more complex than a simple single lipid membrane bilayer. Details of the substructure, lipid and protein composition of the MV membrane are provided in subsequent sections.

RICHARD C. CONDIT ET AL.

FIG 2. Vaccinia virus internal features. (A) Cryosection (*Hollinshead et al.*, 1999). (B) Whole mount cryo-electron micrograph using uranyl acetate staining (*Griffiths et al.*, 2001b). The authors interpreted images of isolated particles as revealing significant asymmetry in the virion structure, two lipid bilayer membranes, and infoldings of the membranes (Section VI, Fig. 8, Model 4). Hence, the “R” indicates the right hand side of the particle. The authors state that “The particle has attached to the grid support in a manner such that a classical side view is evident. The two membrane layers are indicated with large arrows, and the spikes are indicated by an arrowhead. The star indicates the projection of one of the lateral bodies that we believe is due to the overlapping of the peripheral lobes of the virus (top and bottom). The small arrow indicates an inward groove in the membrane.” The image is one of the best available showing the internal structure of the virion in a whole mount preparation of virus. (C, F, and I) Three different projections of a whole mount preparation of virus, phosphotungstic acid stained (*Peters and Mueller*, 1963). (D, G, and J) Sections of purified virus, uranyl acetate stained, in three different perpendicular planes (*Peters and Mueller*, 1963). (E, H, and K) Sections of MV in three different perpendicular planes (*Kato et al.*, 2004; Condit, unpublished). (E) and (H) are virions in infected cells; (K) was from a preparation of purified virus. (A) Reprinted from *Hollinshead et al.* (1999) with permission. (B) Reprinted from *Griffiths et al.* (2001b) with permission. (C, D, F, G, I, and J) Reprinted from *Peters and Mueller* (1963) with permission. (K) Reprinted from *Kato et al.* (2004) with permission.

the membrane and the core wall; estimates of the width of the outer core wall range from 8 to 17 nm. The outer core wall has a striated appearance and therefore has been called the “palisade layer.” The inner core wall or “smooth layer” (innermost light layer in Fig. 2A), has been estimated at 5–8 nm in thickness. While biochemical evidence suggests
that the layers of the core wall are in fact biochemically distinct, the layered appearance of the membrane may simply reflect a structural asymmetry within an otherwise unitary structure. Over much of its surface, the outer core wall appears to be closely apposed to the inner domain of the membrane, as is clearly seen at the top and bottom of the particle in Fig. 2A. Appropriate sections of the particle reveal the central bilateral concavities in the core, formed by a separation of the core wall from the membrane. Filling these concavities between the core wall and the membrane are two apparently amorphous masses called “lateral bodies,” as shown clearly in the particles in Fig. 2A and K.

Importantly, different sections through the virion yield very different impressions of the relationship between the core and the membrane. Figure 2C–K shows sections in all three possible spatial planes. Sections through the broadest plane (Fig. 2C–E) closely resemble whole mount cryo-EM images of purified virus or the C form of virus viewed by negative staining. Thus, the capsule of the C form and the surface domain of the cryo-EM images most likely represent the four domains of the membrane and the core wall, tightly apposed with no separation between the core wall and the membrane and no evidence of lateral bodies. While most whole mount preparations of virus seem to be nonrandomly oriented in this fashion with the broadest plane perpendicular to the electron beam, particles are occasionally oriented on their sides, revealing the biconcave shape of the core and hinting at lateral bodies (Fig. 2B) (Griffiths et al., 2001b). Sections through the other two planes perpendicular to the broadest plane (Fig. 2F–K) often reveal the lateral bodies but may not if they are cut close to the ends or the sides of the particle.

The core contains a tube-like structure first detailed by Peters and Mueller (1963) (Fig. 2C–K). Images that clearly reveal the length of the tube are rare, possibly because this structure is relatively labile (Peters and Mueller, 1963) and also because sections must be cut precisely parallel to, and must bisect, the broadest plane of the virus in order to clearly visualize the tube. Images of the tube in cross section are more common, although still relatively rare (Fig. 2F–H) (Heuser, 2005).

### B. Chemical Composition of MV

Whole MV particles contain approximately 3.2% (of dry weight) DNA and 5% lipid, the balance of the mass presumably being composed of protein (Zwartouw, 1964). There is general agreement that the particle does not contain significant amounts of RNA. In addition, the presence of spermine and spermidine in MV has been reported;
as described in Section VIII, these polyamines may play a role in condensation of the viral DNA (Lanzer and Holowczak, 1975).

The lipid composition of MV is generally similar to that of the host cell, with two significant differences. First, the fraction of phosphatidyl ethanolamine is about one-third less than the host cell and second, MV lipid contains approximately 25% of a biphosphatidic acid analog, acyl bis(monoacylglycero)phosphate, three to four times the amount found in host cells (Hiller et al., 1981; Stern and Dales, 1974). These differences carry implications for the origin of the lipid in MV, discussed in a later section.

C. Controlled Degradation of MV

Additional detail regarding the fine structure of MV has been obtained through controlled degradation of the particle, followed by electron microscopic analysis of subparticle fractions. Easterbrook (1966) first demonstrated that treatment of MV with a neutral detergent (NP40) and a reducing agent (2-mercaptoethanol or dithiothreitol (DTT)) solubilizes the virion membrane, leaving behind an intact core with lateral bodies attached. This observation, coupled with the observation that MV can enter cells by fusion (Armstrong et al., 1973; Carter et al., 2005), demonstrates that the MV membrane contains a lipid bilayer, which probably corresponds to the innermost of the two visible domains comprising the MV membrane (Section III.A) (Hollinshead et al., 1999). Ichihashi et al. (1984) observed that treatment of MV with SDS alone yielded a “ghost” particle, which remained bounded by a structure that resembles the membrane but is reduced in thickness from 20 to 5–7 nm. This finding implies that the protein component(s) of the membrane comprise an extensive disulfide bonded network (Section VI.D.2).

Careful titration of NP40 and 2-mercaptoethanol into suspensions of MV yields cores mixed with an abundance of tubular structures (Fig. 3A) (Wilton et al., 1995). Wilton et al. (1995) equated these structures with STEs. While the isolated tubules are not identical in dimension and structure to STEs, the overall similarity in appearance is compelling. After complete solubilization of the MV membrane with saturating amounts of NP40 and 2-mercaptoethanol, the core, with lateral bodies attached, can be purified from the solubilized membrane fraction by differential centrifugation (Fig. 3B). (It is noteworthy that most investigators refer to this NP40 and 2-mercaptoethanol insoluble fraction as the “core” fraction, when it in fact contains both the virion core and the lateral bodies.) Isolated cores no longer possess concavities and reveal clearly the details of the outer palisade layer of the core wall (Fig. 3C and D)

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 49

![Image](image.png)

FIG 3. Vaccinia virion substructures. (A) Negatively stained preparation of MV after mild treatment with NP40 and 2-mercaptoethanol, so that surface tubule elements remain intact (Wilton *et al.*, 1995). Bar, 0.2 μm. (B) Isolated core in thin section. Note lateral bodies still attached (Ichihashi *et al.*, 1984). (C) Isolated core visualized by cryo-electron microscopy (Dubochet *et al.*, 1994). (D) Isolated core visualized by negative staining (Wilton *et al.*, 1995). (A and D) Reprinted from Wilton *et al.* (1995) with permission. (B) Reprinted from Ichihashi *et al.* (1984) with permission. (C) Reprinted from Dubochet *et al.* (1994) with permission.

(Dubochet *et al.*, 1994; Easterbrook, 1966; Wilton *et al.*, 1995). The palisade is composed of a lattice, sometimes hexagonal, of cylindrical pegs measuring approximately 5 nm in diameter and 10 nm in length (Dubochet *et al.*, 1994; Easterbrook, 1966; Westwood *et al.*, 1964). The lateral bodies and palisade layer sometimes spontaneously dissociate from a fraction of isolated cores, revealing the inner smooth layer and thus substantiating the two-layer character of the core wall. Treatment of the NP40, 2-mercaptoethanol insoluble fraction with trypsin actively removes the lateral bodies, suggesting that the lateral bodies are proteinaceous in nature (Easterbrook, 1966). Within the core resides the viral DNA genome.

### D. Structure of the Genome in the Core

Investigation of the core substructure suggests that the viral genome exists in the core complexed with viral protein. Rupture of MV by various methods can release 30–40-nm diameter tubules that

possess an apparently helical substructure (Griffiths *et al.*, 2001b; Malkin *et al.*, 2003). The relationship, if any, between these tubules and the 50-nm diameter tube contained in the core (Peters and Mueller, 1963) is unclear. Treatment with protease reduces the 30-nm tubes to 16-nm fibers, also helical in substructure, which coat the viral DNA (Malkin *et al.*, 2003). Holowczak and coworkers, using various treatments with detergents and denaturing agents, isolated from MV DNA-containing substructures of varying complexity they termed “nucleoids” and “subnucleoids” (Holowczak *et al.*, 1975; Soloski and Holowczak, 1980, 1981; Soloski *et al.*, 1979; reviewed in Holowczak, 1982). Subnucleoids, the simplest of these structures, contained four proteins apparently complexed with supercoiled DNA in 30–60-nm spherical structures interconnected with DNA fibers, roughly reminiscent of a very loose nucleosome structure. In summary, these studies suggest that the vaccinia genome exists within the core complexed in an organized fashion with viral proteins. However, this conclusion is still controversial. As described later in Section VIII.B, repression or inactivation of two different proteins required for genome encapsidation leads to the production of MV that lack the viral genome. These virions contain what appears to be the full complement of viral proteins: the absence of the encapsidated genome is not accompanied by the absence of an abundant DNA-wrapping protein. It may be that association of the genome with viral proteins to form a nucleoprotein complex occurs after genome encapsidation, within the context of the nascent virion core.

### E. MV Proteins

Historically, the protein composition of MV has been analyzed using three fundamentally distinct approaches. First, virion proteins have been cataloged by electrophoretic resolution of whole virions or virion subfractions (Essani and Dales, 1979; Ichihashi *et al.*, 1984; Jensen *et al.*, 1996; Sarov and Joklik, 1972; Takahashi *et al.*, 1994; Wilton *et al.*, 1995). Second, numerous proteins have been purified from MV by virtue of their enzymatic activities and subsequently characterized (Martin *et al.*, 1975; Morgan *et al.*, 1984; Niles *et al.*, 1989). Finally, antibodies have been generated for individual gene products in a targeted fashion, allowing proteins to be subsequently localized to virions by immunoblot analysis (da Fonseca *et al.*, 2000a, 2004). Early attempts at cataloging the protein content of virions by gel electrophoresis relied on apparent molecular weights as a means of identifying proteins. Because of inconsistencies in molecular weight determinations, these

studies are often difficult to compare with each other, and an unambiguous assignment of a specific protein from the older literature to a discrete vaccinia gene is prone to error. More recently, the combination of genetics and protein sequencing techniques has made possible the unambiguous correlation of genes with virion proteins. We have confined ourselves in this review exclusively to those virion proteins for which a positive gene identification has been made.

A complete list of the virion proteins for which a specific gene has been identified is given in **Table I**. These virion proteins can be sorted into four groups based on whether or not they are solubilized by NP40 and a reducing agent and whether or not they possess enzymatic activity: (1) proteins that are solubilized by NP40 and a reducing agent but have not been assigned an enzymatic activity are presumed to be *membrane structural proteins*; (2) proteins that are solubilized by NP40 and a reducing agent and also possess an enzymatic activity are classified as *membrane enzymes*; (3) proteins that are not solubilized by NP40 and a reducing agent and have not been assigned an enzymatic activity are presumed to be *core structural proteins*; and (4) proteins that are not solubilized by NP40 and a reducing agent and also possess an enzymatic activity are classified as *core enzymes*. For completeness, **Table I** also includes two proteins (D13 and A11) that affect virion morphogenesis but are not encapsidated and two proteins (A32 and A2.5) whose subvirion localization remains to be determined. The list given in **Table I** is undoubtedly incomplete but is probably approaching saturation. Indeed, a comprehensive catalog of specific vaccinia gene products contained in MV has been compiled based on gel-free liquid chromatography and tandem mass spectroscopy of whole virions (*Chung et al.*, 2006). These new data correlate remarkably well with the data in **Table I** and identify an additional 10 proteins (C6, F8, F9, E6, I3, A6, A25, A31, A42, and A46), whose virion localization is currently either disputed or unconfirmed by other techniques. Because of the possibility of nonspecific association of proteins with virions (*Franke and Hruby*, 1987), we have not included in **Table I** these 10 new candidate virion proteins and have instead focused attention on proteins whose presence in the virion has been confirmed by additional experiments. A few general features of the catalog of virion proteins are highlighted here, and otherwise the details of the contributions of individual proteins to structure and morphogenesis of the virions are detailed in the remainder of the review.

To date, a total of 22 proteins have been localized to the virion membrane. Of these, 2 are enzymes associated with a virus-coded redox pathway responsible for maintenance of some of the disulfide bonds in

the membrane, while the remaining 20 proteins can be thought of as structural proteins. Eighteen of the 20 structural proteins are integral membrane proteins containing at least one transmembrane domain. Interestingly, none of these proteins are glycosylated in the MV membrane, reflecting the unusual origin and structure of the membrane.

To date, 47 proteins have been localized to the virion “core,” that is, the NP40, reducing agent insoluble fraction, which includes both the core and the lateral bodies. Of these 47 proteins, 19 have no known enzymatic function and are presumed to be structural proteins. Sixteen of the enzyme proteins have well-characterized roles in early viral mRNA synthesis, including initiation, elongation and termination of transcription, mRNA capping and polyadenylation. The remaining enzymes play less clearly defined roles in the viral life cycle but have in some cases been shown to interact with and/or modify both proteins and nucleic acids. It is important to note that virion enzymes may have structural as well as enzymatic roles in the virion. While attempts have been made to sublocalize proteins to core nucleoprotein, inner core wall, outer core wall, and lateral bodies (Ichihashi *et al.*, 1984; Sarov and Joklik, 1972), these studies date from the pregenomic era and few if any unambiguous assignments of core proteins to core substructures have been made.

Finally, several cellular proteins have been found associated with purified particles (Castro *et al.*, 2003; Chung *et al.*, 2006; Jensen *et al.*, 1996; Webb *et al.*, 1999). It is possible that some of these proteins adhere to the MV during purification or are incorporated nonspecifically during virus maturation (Franke and Hruby, 1987). However, cyclophilin A has been clearly demonstrated to be packaged into virus cores (Castro *et al.*, 2003). The role of this protein in virus formation is not known, although it could participate in the trafficking of virus proteins to the factories or in the proper folding of viral proteins during the maturation process.

### F. Model for MV Structure

Based on the evidence presented previously, we have constructed a schematic 3D model of the vaccinia virion. The model is not intended to be definitive in its details, especially given the continuing uncertainty regarding dimensions of the virion and its substructures. Nevertheless, we feel that such a schematic will clarify some of the microscopic images available and the interrelationships of the virion components. For external dimensions we have used the measurements determined by Cyrklaff *et al.* (2005) using cryo-electron tomography, and for

internal dimensions we have used the measurements determined by Hollinshead *et al.* (1999) from cryosections (Fig. 2A). Thus the virus appears as a slightly flattened barrel (Fig. 4A) with overall dimensions of approximately $360 \times 270 \times 250$ nm. The particle is encased in an outer membrane, which itself consists of two component domains. The outermost membrane domain is 9 nm thick, and the innermost membrane domain is 5 nm thick (Fig. 4C–F). The membrane clearly contains lipid; however, the details of the lipid bilayer content and structure of MV are the subject of some debate and will be therefore be discussed at length later in the review (Section VI.A). Within the membrane is the core, which is also barrel shaped but contains two indentations, one on each

![Figure 4](#fig-4)

FIG 4. A model for vaccinia virion structure. (A) The intact MV. No attempt has been made to represent surface tubule elements. (B) The virion core. (C) Cutaway view. The membrane has been removed from the upper half of the virion, the near end has been removed, and the core wall has been rendered transparent, thus revealing the multiple layers, the concavities in the core, the lateral bodies, and the tubular internal structure. (D and E) Sections through the virion in three different perpendicular planes. The sections correspond to sections shown in Fig. 2. See text for details. (Model courtesy of Michel Moussatche. A dynamic 3D model is available online at [http://www.vacciniamodel.com/](http://www.vacciniamodel.com/).)

of the largest surfaces (Fig. 4B). The core is defined by a core wall, which is also composed of two layers, an outer, thicker, striated “palisade” layer 17 nm thick, and an inner smooth layer 8 nm thick (Fig. 4C–F). Filling the spaces between the core wall and the membrane that are created by the indentations in the core are “lateral bodies.” Within the core is a tube-like structure, 50 nm in diameter with a 10 nm diameter core. This tube is folded upon itself into three continuous segments, with a total length of approximately 250 nm. Importantly, sections through the center of the particle in each of the three possible spatial planes yield three significantly different impressions of the overall shape and internal structure of the particle (Fig. 4D–F). For comparison, the electron micrographic images in Fig. 2C–E correspond to the model section shown in Fig. 4D; Fig. 2F–H correspond to Fig. 4E; Fig. 2I–K corresponds to Fig. 4F. A dynamic 3D rendering of the model can be found online at http://www.vacciniamodel.com/.

IV. VACCINIA VIRUS MORPHOGENESIS: AN OVERVIEW

The balance of this review focuses on the details of the assembly of vaccinia, with particular emphasis on lessons learned from the study of mutants affected in individual virus genes that influence virus morphogenesis. To establish a context for this discussion, we offer here a bird’s-eye view of the entire assembly process.

The temporal sequence of events comprising vaccinia assembly was first deciphered experimentally by electron microscopic examination of cultured cells synchronously infected with virus (Dales, 1963; Dales and Siminovitch, 1961). At very early times following uptake of virus and dissolution of the core particle, infection-specific cytoplasmic domains are observed that are uniform in density, contain few if any cellular organelles, and are sometimes surrounded by ER derived cisternae (Fig. 5A and B) (Tolonen *et al.*, 2001). These domains represent sites of viral DNA replication, sometimes called “factories,” “viral factories,” or “DNA factories,” and they increase in size with time. The earliest evidence of virus assembly is the appearance within factories of rigid crescent-shaped structures, cupules in three dimensions, 10–15 nm in thickness (Figs. 5C and 6). In most electron micrographs crescents comprise two distinct layers, an inner, smooth layer that at high resolution in fixed embedded sections has the trilamellar appearance of a lipid bilayer and an outer layer composed of regularly spaced projections termed “spicules” (Dales and Mosbach, 1968; Risco *et al.*, 2002). It is universally accepted that crescents contain at least one

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 55

FIG 5. Vaccinia factories. (A) An early factory, 4 hpi. The factory is represented by the central cleared area surrounded by cytoplasmic organelles. Bar, 1 μm. (Kato and Condit, unpublished) (B) A factory 2 h, 45 min pi (*Tolonen et al.*, 2001). The star indicates the factory. G, Golgi stack. M, mitochondria. Bar, 0.2 μm. (C) A factory late during infection (*Rodriguez et al.*, 1998). N, nucleus. IV, immature virions. Arrows indicate IVN. Arrowheads indicate MV. Note that the MV are removed from the factory. (B) Reprinted from *Tolonen et al.* (2001) with permission. (C) Reprinted from *Rodriguez et al.* (1998) with permission.

lipid bilayer, although the precise membrane makeup of crescents has been the subject of some debate, discussed in more detail in Section VI.A. Crescents apparently grow in length while maintaining the same curvature until they become closed circles, spheres in three dimensions, called immature virions (IV) (Figs. 5C and 6). IV are approximately 350 nm in diameter and are filled with “viroplasm,” material that is uniform in density but discernibly more electron dense than the surrounding factory. Viroplasm appears not only in IV but it also often appears to fill the concavities defined by crescents. Viroplasm is also often present as relatively large subdomains, “virosomes,” within the surrounding factory, sometimes unbounded, sometimes surrounded

RICHARD C. CONDIT ET AL.

FIG 6. Vaccinia morphogenesis intermediates. (1) Crescents. (2) IV. (3) IVN. (4) Structures intermediate between IV and MV. (5) MV. Reprinted from *Moss* (2001) with permission.

by multiple crescents that appear to “bite off” portions of viroplasm. Appearing at approximately the same time as IV are IV which contain an electron dense, round or ovoid subdomain called a “nucleoid” (Figs. 5C and 6). We refer to IVs that contain nucleoids as “IVN.” Nucleoids contain DNA (*Dales*, 1963; *Ericsson et al.*, 1995). Nucleoid material is frequently observed spanning a small gap in a nearly complete IV, so that it lies partially inside and partially outside of the particle (*Morgan*, 1976a). Under some conditions, larger inclusions of DNA-containing nucleoid material are found within factories, and may appear as either large, spherical, granular inclusions or alternatively paracrystalline arrays. Importantly, serial sections reveal that most if not all IVs contain nucleoids, so that in fact in a normal infection, there may be no real distinction between IVs and IVNs, rather, an IV is merely an IVN that has been sectioned through a plane that does not include the nucleoid (*Morgan et al.*, 1955). Following the appearance of IVNs, MV appear (Figs. 5C and 6). Proteolytic cleavage of several virion protein precursors to a mature form accompanies, and in fact is required for, morphogenesis from IVN to MV (*Ansarah-Sobrinho and Moss*, 2004b; *Katz and Moss*, 1970b). The majority of MV are found outside factories and may exist in clusters either at the periphery of a factory or apparently separated by a

significant distance from the nearest factory. Particles which have a structure that appears to be intermediate between IVN and MV have been described, but they are rare, suggesting that the transition from IV to MV is a rapid and concerted process (Fig. 6).

For the purposes of this review, we have artificially divided vaccinia morphogenesis into stages, including formation of factories, crescents, IV, IVN, and MV. A summary of the entire process is shown in Fig. 7. We now consider the details of each of these stages separately. Critical to the understanding of the assembly process is the phenotypic analysis of infections performed with conditional lethal mutants; a complete list of the currently available mutants affected in genes that encode proteins required for virion morphogenesis or infectivity is shown in Table II.

![Diagram](attachment:diagram.png)

FIG 7. Vaccinia morphogenesis summary. Following the arrows in sequence from the upper left corner: Several integral viral membrane proteins (black Xs) are made in the ER (red lines) and transported to viral factories (gray area) along with ER derived lipid to be assembled into crescents which contain a lipid bilayer (red line) and the membrane proteins, scaffolded on a honeycomb structure composed of the D13 protein (yellow circles). Crescent formation is controlled by phosphorylation. The crescents mature to IV and IVN, accompanied by encapsidation of the genome. The dotted line surrounding IV, IVN, and encapsidation signifies that the order of these events is uncertain. Metamorphosis to MV is accompanied by loss of the D13 scaffold, proteolysis, further addition of membrane proteins (blue line) and movement of particles outside of factories. MV acquire Golgi derived membranes (green lines) to become WV and are exocytosed through the plasma membrane (black line) to become EV.

TABLE II  
VACCINIA MORPHOGENESIS GENETICS  

| Gene   | Identity               | Location         | Mutants  | Cres | IV | MV | EV | Telo | Cleave | DNA | Notes on phenotype                          |
|--------|------------------------|------------------|----------|------|----|----|----|------|--------|-----|--------------------------------------------|
| J4R    | rpo 22                | Core             | ts       | –    | –  | –  |     |      |        |     | Factories only                             |
| J6R    | rpo 147               | Core             | ts       | –    | –  | –  |     |      |        |     | Factories only                             |
| F10L   | Kinase, 7 complex     | Core             | ts, ind  | –    | –  | –  |     |      |        |     | Factories only                             |
| H5R    |                       | Core             | ts       | –    | –  | –  | +   |      |        |     | Curdled virosomes or DNA                   |
| G5R    |                       | Core             | ts       | –    | –  | –  | +   |      |        |     | Curdled virosomes                          |
| A11R   |                       | Absent           | ind      | –    | –  | –  |     |      |        |     | Some normal and aberrant virosomes          |
| A17L   | A17-A14-A27 complex   | Membrane         | ind      | –    | –  | –  |     |      |        |     | Virosomes, vesicles                       |
| A14L   | A17-A14-A27 complex   | Membrane         | ind, ts  | –    | –  | –  |     |      |        |     | Virosomes, vesicles                       |
| D13L   | IV scaffold protein   | Absent           | ind, ts  | –    | –  | –  |     |      |        |     | Aberrant crescents, rif resistance locus   |
| A30L   | 7 complex             | Membrane/core    | ind, ts  | +    | –  | –  |     |      |        |     | Empty IV                                   |
| G7L    | 7 complex             | Core             | ind, ts  | +    | –  | –  |     |      |        |     | Virosomes + crescents (ts), or empty IV(ind) |
| J1R    | 7 complex             | Membrane/core    | ind, ts  | +    | –  | –  | +   |      |        |     | Aberrant, partial, empty IV                |
| A15L   | 7 complex             | Core             | ind      | +    | –  | –  |     |      |        |     | Empty IV                                   |
| D2L    | 7 complex             | Core             | ind, ts  | +    | –  | –  |     |      |        |     | Empty IV                                   |
| D3R    | 7 complex             | Core             | ts       | +    | –  | –  |     |      |        |     | Empty IV                                   |

A10L    p4a         Core      ind, ts   +   -   -   -   Aberrant, empty IV's, DNA nucleoid aggregates
A13L    Membrane    ind, ts   +   +   -   +   -   No IVN, membraned DNA crystalloids
A32L    NTP motif   Virion    ind     +   +   -   +   -   Dense spherical particles lacking DNA
I6L     Telomere    Core      ts      +   +   -   +   -   Dense spherical particles lacking DNA
        binding
A22R    Holliday    Core      ind     +   +   -   -   -   Dense spherical particles
        resolvase
A7L     VETF        Core      ind     +   +   -   -   +   IVN, dense spherical particles, viroplasm
D6R     VETF        Core      ind, ts +   +   -   -   -   IVN, dense spherical particles, viroplasm
I1L     DNA binding Core      ind     +   +   -   +   -   IVN, DNA crystalloids
        protein
F17R    11k         Core      ind     +   +   -   +   -   Some aberrant, unstable IV
A4L     Membrane/   Core      ind     +   +   -   +   +   IVN
        core
A3L     p4b         Core      ts      +   +   -   +   +   Aberrant MV
G1L     Metallo-    Core      ind     +   +   -   +   +   Aberrant MV
        proteinase(?)
I7L     Cysteine    Core      ind, ts +   +   -   +   +   Aberrant MV
        proteinase(?)
A9L     Membrane    ind       +   +   -   -   -   Some aberrant IV
L1R     Myristylated Membrane ind     +   +   -   -   -   Unstable IV, IVN
        protein
H3L     Heparin     Membrane  ind, ko +   +   -   -   -   Some MV, defective, viroplasm
        binding
(continues)

| Gene | Identity | Location | Mutants | Cres | IV | MV | EV | Telo | Cleave | DNA | Notes on phenotype |
|------|----------|----------|---------|------|----|----|----|------|-------|-----|-------------------|
| E10R | Thiol oxidoreductase | Membrane | ind | + | + | - | - | - | - | - | - |
| G4L | Thiol oxidoreductase | Membrane | ind | + | + | - | - | - | - | - | - |
| A2.5L | Thiol oxidoreductase | Virion | ind | + | + | - | - | - | - | - | - |
| H4L | rap 94 | Core | ind, ts | + | + | + | - | - | - | - | Noninfectious MV, transcription enzymes missing |
| I8R | RNA helicase | Core | ts | + | + | + | + | - | - | - | Transcription defective MV |
| H6R | Topoisomerase | Core | ko | + | + | + | + | + | - | - | MV show reduced infectivity, reduced early txn |
| L4R | VP8 | Core | ind | + | + | + | + | + | + | + | Noninfectious, unstable, transcription defective MV |
| H1L | Phosphatase | Core | ind | + | + | + | - | - | - | - | Noninfectious, transcription defective MV |
| E8R | Core | Core | ts | + | + | + | + | - | - | - | Noninfectious, transcription defective MV |
| L3L | Core | Core | ind | + | + | + | + | + | + | + | Noninfectious, transcription defective MV |
| L5R | Membrane | ind | + | + | + | + | - | - | - | - | Noninfectious MV, fusion/entry defective |

| A21L | Membrane | ind | + | + | + | + | Noninfectious MV, fusion/entry defective |
| --- | --- | --- | --- | --- | --- | --- | --- |
| A28L | Membrane | ind, ts | + | + | + | + | Noninfectious MV, fusion/entry defective |
| H2R | Membrane | ind | + | + | + | + | Noninfectious MV, fusion/entry defective |
| A16L | Membrane | ind | + | + | + | + | Noninfectious MV, fusion/entry defective |
| G3L | Membrane | ts | + | + | + | + | Noninfectious MV |
| I2L | Membrane | ind | + | + | + | + | Noninfectious MV, fusion/entry defective |
| D8L | Membrane | ko | + | + | + |  | Reduced infectivity MV, cell binding |
| A27L | A17-A14-A27 complex | Membrane | ind, ko | + | + | + | - | Infectious MV, defective in EV formation |
| A26L | p4c | Membrane | ko | + | + | + | + | Infectious MV; empty ATI |
| A14.5L |  | Membrane | ko | + | + | + |  | Infectious MV, attenuated in animals |
| E11L |  | Core | ts | + | + | + |  |  |
| A45R | Inactive SOD | Core | ko | + | + | + | + | Nonessential |
| K4L | Nicking/joining enzyme | Core | ko | + | + | + | + | Nonessential |

* Mutant designations: ind, inducible; ts, temperature sensitive; ko, knockout; Cres, crescents; Telo, telomere resolution; Cleave, virion protein proteolyzed; DNA, purified particles contain DNA; +, normal structure formed; –, normal structure not formed; blank, not reported.

RICHARD C. CONDIT ET AL.

V. FORMATION OF FACTORIES

Early studies by Cairns (1960) demonstrated that the number of factories observed early during vaccinia infection was proportional to the multiplicity of infection and concluded that each factory observed arose from a single infecting particle. Factories represent sites of viral DNA replication: factories become labeled with DNA precursors as assessed by EM autoradiography, factory formation is absent or severely restricted following infection with mutants defective in DNA replication, and infection with mutants that are DNA replication competent but defective in late viral protein synthesis results in formation of large factories that are lacking in crescents or any other evidence of virus morphogenesis (Cairns, 1960; Hooda-Dhingra *et al.*, 1989). Most reports described early factories as unbounded in the cytoplasm (Fig. 5A) (Dales and Kajioka, 1964); however, recent reports suggest that at early times, before the appearance of any specific viral structures, factories are transiently surrounded by cellular ER-derived membrane cisternae (Fig. 5B) (Doglio *et al.*, 2002; Tolonen *et al.*, 2001; Traktman, unpublished).

VI. CRESCENT FORMATION

A. One Membrane or More?

The earliest detailed descriptions of the substructure of vaccinia viral crescents concluded that they consisted of a single lipid bilayer coated on its outer surface with a layer of spicules, which conferred the rigid arched shape to the crescents. Furthermore, these studies concluded that the lipid bilayer of the crescent formed within factories without any direct connection to other existing cellular membranes (Fig. 8, Model 1) (Dales and Mosbach, 1968). This *de novo* formation model, although unique in biology, nevertheless stood as the accepted model until it was challenged in 1993 by Griffiths and coworkers (Sodeik *et al.*, 1993). Central to this challenge was the commonly held

4 A significant body of literature has made use of inhibitors of DNA replication to study morphogenesis. For example, hydroxyurea has been used to segregate early and late stages of morphogenesis (Morgan, 1976a; Pogo and Dales, 1971). It is now clear that in the absence of DNA replication, factory formation and therefore morphogenesis is inhibited. It seems probable that in reports where factory formation and some morphogenesis were observed in the presence of DNA replication inhibitors, the inhibitors were used under conditions that were not completely inhibitory, and thus some DNA replication and late viral protein synthesis occurred. These studies should be interpreted with this caveat in mind.

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 63

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG 8. Models for membrane processing during vaccinia morphogenesis. In each case, a single solid red line represents a single lipid bilayer. DNA/nucleoid material is represented in gray; lateral bodies are represented in blue. Four models are shown in four columns. Each row represents and compares a discrete state of morphogenesis, with the top row representing crescents, the middle row representing immature virions with nucleoids (IVN) and the bottom row representing mature virions (MV). Model 1: Crescents and IVN contain a single lipid bilayer, which becomes a single lipid bilayer of the structure surrounding MV (Dales and Mosbach, 1968; Hollinshead et al., 1999). In this case the core wall does not contain lipid and is therefore represented in black. Model 2: Crescents are derived from cisternae and comprise two tightly apposed lipid bilayers. The ends of cisternae fuse or are sealed to form IVN containing two tightly apposed bilayers. During MV formation from IVN, the inner bilayer separates from the outer bilayer so that the inner bilayer becomes a component of the core wall while the outer bilayer remains as a component of the MV membrane (Sodeik et al., 1993). Model 3: Crescent and IVN formation is similar to Model 2; however, the cisternae do not fuse but are “sealed.” During morphogenesis to MV the two bilayers do not separate, and thus the MV membrane contains two tightly apposed bilayers (Roos et al., 1996). As in Model 1, the core wall does not contain lipid and is therefore represented in black. Model 4: Crescents are derived from cisternae as in Models 2 and 3; emphasis is placed on the continuity of the rigid crescent domain of this cisternae with extensive flaccid cisternal domains. DNA in a prenucleoid structure is associated with the flaccid cisternal domain. IVN is formed by the infolding of the flaccid domain within the rigid crescent domain. The cisterna of IVN are sealed so that the resulting MV contains four lipid bilayers, two forming the virion envelope and two forming the core wall (Griffiths et al., 2001a; Sodeik and Krijnse-Locker, 2002).

view that all cellular membranes are derived from preexisting organelles (Palade, 1983). Sodeik et al. (1993) proposed that crescents in fact were derived from preexisting cisternae of the cellular secretory machinery, which had collapsed on themselves through the action of

virus proteins. Crescents formed in this manner would consist of two tightly apposed bilayers rather than a single bilayer (Fig. 8, Model 2). (Crescents would therefore be analogous to the membranes involved in the development of spherical autophagosomes, which are delimited by a double bilayer [Reggiori and Klionsky, 2005].) Investigations into the consequences of this cisternae-based model for crescent formation ultimately resulted in three different new models for IV and MV formation, shown in Fig. 8, Models 2–4. In the balance of this section, we describe and evaluate these models.

Model 1. Single membrane bilayer (Dales and Mosbach, 1968). As described previously, this model states that crescents arise *de novo*, that is, without apparent continuity with other preexisting membrane organelles, and that crescents contain a single lipid bilayer coated on its outer surface with a layer of spicules. During morphogenesis from IVN to MV, the spicules are lost, the surface undergoes some additional modifications, and the contents of IVN are reorganized to yield the core and lateral bodies (Essani *et al.*, 1982; Stern and Dales, 1976). In this model, the core is not surrounded by a lipid membrane (Wilton *et al.*, 1995). The primary objections to this model are that it seems to violate a principle which dictates that all membranes in the cell are derived from preexisting organelles and that at the edges of a crescent, the hydrophobic core of the bilayer is left exposed to the cytoplasm, a theoretically unstable condition. The model is supported by the fact that it is consistent with some existing microscopic data and that it yields a single membraned MV that can enter cells by a simple fusion event at the plasma membrane.

Model 2. Double membrane bilayer; membrane bound core (Sodeik *et al.*, 1993). As described previously, this model states that crescents actually contain two tightly apposed lipid bilayers, formed by collapse of preexisting cellular membrane cisternae, modified by virus proteins. The model states further that during morphogenesis from IVN to MV, the two membranes separate and the innermost membrane collapses around the core, so that both the MV surface and the core are each now bounded by a single lipid bilayer. The primary theoretical objections to this model are that it introduces a problem of how the material making up the lateral bodies is transported across the innermost membrane during morphogenesis from IV to MV and that it would result in delivery of a membrane-bound core to the cytoplasm if MV were to enter cells by a simple fusion event at the plasma membrane. The model is supported by numerous electron micrographs which seem to show continuity between crescents and cisternae (Ericsson *et al.*, 1997; Sodeik *et al.*, 1993, 1994), and by electron microscopic images of MV

which seem to show a membrane surrounding the core (Cyrklaff *et al.*, 2005; Sodeik *et al.*, 1993).

Model 3. Double membrane bilayer; naked core (Roos *et al.*, 1996). This model also proposes that crescents are composed of two tightly apposed lipid bilayers; however, it differs from Model 2 with respect to the fate of the innermost bilayer during the morphogenesis from IV to MV. Specifically, Model 3 states that the two bilayers do not separate during MV morphogenesis, so that the resulting MV contains two tightly apposed bilayers on its surface, and the core is not bound by a membrane. The model also states that the membranes surrounding the particle are not fused but rather joined in an overlap by a proteinaceous plug. Relative to Model 2, this model resolves the theoretical problem of transport of material across the innermost membrane during formation of lateral bodies but complicates further the problem of virus entry; in this case, a simple fusion event between the virion and the plasma membrane would leave the entire contents of the particle outside of the cell. In support of this model are the same micrographs that seem to show continuity between crescents and cisternae (Ericsson *et al.*, 1997; Sodeik *et al.*, 1993, 1994), coupled with experiments that indicate that core surface antigens are exposed when MV are disrupted with a reducing agent (Roos *et al.*, 1996).

Model 4. Folded cisternae (Griffiths *et al.*, 2001a; Rodriguez *et al.*, 2006). In this model collapsed cisternae evolve distinct virus-specific domains, including a rigid crescent domain that is continuous with a DNA binding domain. IV and MV are formed by folding of these domains upon themselves, resulting in a particle which now contains, in effect, four lipid bilayers, two on the surface of MV and two surrounding the core. Like Model 3, Model 4 proposes that the membranes are not fused but rather joined in an overlap by a proteinaceous plug. Like Model 3, this model resolves the theoretical problem of transport of material across the innermost membrane during formation of lateral bodies, but leaves the same problem of virus entry described previously for Model 3. The model is supported by electron micrographs which seem to show the continuity of the cisternal domains, and by electron micrographs of isolated MV which are interpreted to reveal a complexity consistent with the model (Griffiths *et al.*, 2001a,b).

The distinction among these four models is embodied by four questions relating to crescent, IV and MV structure and assembly: (1) Do the crescents contain two membranes? (2) Are the crescents continuous with cisternae, and is this continuity theoretically necessary? (3) Are nucleoids or cores bounded by membranes? (4) How does the virus enter cells? Each of these questions is addressed separately below.

RICHARD C. CONDIT ET AL.

1. **Do Crescents Contain Two Membranes?**

The structure of viral crescents has been investigated by various methods of electron microscopy, with conflicting interpretations of the results. The overall thickness of crescents including the inner smooth layer and the outer “spicule” layer is 10–15 nm, at least enough to accommodate two tightly apposed bilayers. Occasional micrographs show that these layers are physically distinct in that they sometimes appear separated (Dales and Mosbach, 1968; Essani et al., 1982; Hollinshead et al., 1999; Sodeik et al., 1993). Some investigators interpret the outermost layer to be devoid of lipid membrane and conclude that crescents contain only one membrane corresponding to the innermost layer (Dales and Mosbach, 1968; Essani et al., 1982; Hollinshead et al., 1999). Other investigators conclude that both layers represent lipid bilayers and that the spicules can obscure the outermost lipid bilayer in conventional electron micrographs (Risco et al., 2002; Sodeik et al., 1993). Freeze fracture of multiply membraned organelles often clearly reveals the individual membranes as steps in the fracture pattern. While one freeze fracture study of vaccinia IVs suggests a double membraned particle (Risco et al., 2002), most freeze fracture studies fail to reveal evidence of more than one membrane in IV (Heuser, 2005).

Electron microscopy of IVs formed after reversal of a rifampicin block provides unique insight into the formation of crescents. As detailed in Section VI.E, it is now clearly established that the major component of the spicule layer is a lattice of the D13 protein. In the presence of the antibiotic rifampicin, the association of D13 protein with viral membranes is prevented, so that flaccid viral membranes accumulate at the periphery of virosomes while D13 protein accumulates within viroplasmic inclusions distant from the membranes. While some investigators describe only single lipid bilayers accumulating in the presence of rifampicin (Grimley et al., 1970), others describe both single membranes and cisternae (Sodeik et al., 1994), or “dense membranes (around 18 nm thick), twisted double membranes with vesicular ends, and tubular elements (30 nm thick), some of them with vesicles at one end” (Risco et al., 2002). Removal of rifampicin results in formation of rigid crescents within minutes, presumably via association of accumulated D13 with viral membranes. Under conditions of rifampicin reversal, multiple crescents can form on one continuous membrane, providing a direct comparison of crescents with spicule-free viral membranes. Under these circumstances, clear images have been obtained which suggest that the crescent membrane is a single lipid bilayer and that the spicule layer is attached in patches along this unit membrane (Fig. 9) (Grimley et al., 1970).

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 67

FIG 9. Evidence that crescents contain a single lipid bilayer. Electron micrograph of an infected cell 10 min following reversal of a rifampicin block (Grimley et al., 1970). Two crescents are connected by a continuous membrane, which has the dimensions and appearance of a single lipid bilayer. Reprinted from Grimley et al. (1970) with permission.

In summary, while electron micrographs are subject to different interpretations, the bulk of the evidence, in particular from freeze fracture microscopy and rifampicin reversal, favors the model that crescents contain a single lipid bilayer formed beneath a scaffold composed predominantly of D13 protein.

2. Are the Crescents Continuous with Cisternae, and is this Continuity Theoretically Necessary?

Much has been made of the use of the term *de novo* to describe vaccinia viral membrane biogenesis. To some the appearance of membranes without continuity with other intracellular membranes is a heretical violation of biological principle, while to others it is simply another amusing example of the inventiveness of nature. Numerous electron micrographic studies have drawn conclusions on either side of this debate. On the one hand, for example, serial sections of crescents have been published which show no continuity between crescents and other membranes, and in the same study tilt series electron micrographs reveal that crescents which are in fact not continuous with cisternae, may appear to be in continuity when viewed at an appropriate angle, raising the possibility of artifact (Hollinshead et al., 1999). On the other hand, numerous, highly convincing electron micrographs exist which seem to show crescents in continuity with membranes or cisternae, or crescents with membranous extensions or loops at the ends (Ericsson et al., 1997; Griffiths et al., 2001a; Risco et al., 2002; Sodeik et al., 1993, 1994). In fact, the *de novo* argument may be irrelevant. Based on evidence cited in Section VI.B, a reasonable model for viral

membrane biogenesis is that lipid is delivered to factories from the ER as vesicles complexed with viral protein and that crescents may form and grow via addition of lipid and viral protein from these vesicles. Assuming the vesicles are relatively small and short lived, any number of structures may be visible in viral factories, including vesicles, tubules, crescents with no apparent continuity with other membranes, or crescents with long, short, or looped extensions at their ends. Furthermore, all these structures may be subject to variation in sample preparation and interpretation. Formation of a crescent, which contains a double bilayer, would be the obvious outcome of vesicle fusion. However, if crescents contain a single bilayer, then some mechanism must exist for building a single bilayer from a vesicle, tubule, or cisterna, and some “capping” mechanism must exist to sequester the hydrophobic core of the single membrane bilayer from the aqueous environment of the cytoplasm. Heuser (2005) suggests one possible mechanism for formation of a single bilayer from virtually any other membrane, based on a current “Tjunction” model for lipid droplet formation (Fig. 10). Capping of the crescent ends might be accomplished by a novel, yet to be discovered viral gene product. Alternatively, the lateral edges of the crescent membrane may contain phospholipids which assume a curved, micellar formation rather than a true lipid bilayer, such that only polar head groups are in contact with the cytoplasm.

In summary, it is possible that growing crescents may appear either continuous or discontinuous with other membranes; however, the presence of absence of such continuities does not necessarily dictate whether the crescent and IV membrane is a single bilayer or two tightly apposed bilayers.

3. Are Nucleoids or Cores Bounded by Membranes?

As described previously, two of the double membrane models require that the core be surrounded by at least one membrane; therefore, the question of whether or not the core is bounded by a membrane is relevant to distinguish between one and two membrane models for crescent and IV assembly. In common with other aspects of the debate, much of this discussion rests on electron microscopic data, which is subject to varied interpretation. For example, in cryo-electron micrographs, the inner layer of the core wall appears electron lucent or “white,” consistent with a lipid bilayer (Fig. 2A). While some authors interpret this layer as lipid (Cyrklaff et al., 2005; Sodeik et al., 1993), others conclude that its dimensions are inconsistent with a lipid bilayer (Hollinshead et al., 1999). Similarly, while some images are interpreted to reveal a complex folding of membranes in developing

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 69

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig 10. Model for poxvirus membrane growth. (Top) Model for lipid droplet formation from ER. (Bottom) Model for vaccinia crescent formation based on the lipid droplet formation model (Heuser, 2005). A crescent membrane, represented by a bilayer studded with lollipops, is extruded in a T-junction from another bilayer, which could be part of any membrane structure. The lollipops represent a viral protein that directs the process. Reprinted from Heuser (2005) with permission.

or mature particles consistent with the folded cisternae model (Model 4) (Griffiths et al., 2001a,b; Sodeik et al., 1993), numerous other images show no evidence of membrane boundaries within IV, IVN, or MV and no surface folds or continuity between internal and external boundaries in MV (Dales and Mosbach, 1968; Dales and Siminovitch, 1961; Essani et al., 1982; Heuser, 2005; Hollinshead et al., 1999; Mohandas and Dales, 1995; Morgan, 1976a,b; Risco et al., 2002). Perhaps the most compelling data relevant to the lipid content of cores consists of examination of isolated cores. As described previously, cores are typically isolated by dissociating the MV membrane with a neutral detergent (NP40) and DTT. This treatment completely dissolves the MV membrane and should therefore extract any lipid from the cores, and yet the structure of the isolated core remains essentially the same as the core found in intact MV (Dubochet et al., 1994; Easterbrook, 1966; Ichihashi et al., 1984; Wilton et al., 1995). Isolated cores lose the concavities present in mature MV and assume a more regular barrel shape but nevertheless maintain a bilaminar structure with an inner, smooth layer and an outer palisade layer, virtually identical to the core present in mature MV. If the core were to be encased in one or two lipid bilayers, then this lipid would have to be resistant to detergent extraction sufficient to dissolve the membrane, or the lipid would have to contribute very little to the structure of the core, so that its absence

would not affect the gross structure of the core. Both of these latter possibilities seem unlikely.

In summary, the weight of the evidence seems to favor the conclusion that the core is not bounded by a lipid bilayer.

4. How Does the Virus Enter Cells?

As noted previously, any of the two membrane models present problems for entry of virus into cells. If Model 2 were correct, a simple fusion of the outermost viral membrane with the plasma membrane would deliver a membraned subparticle to the cell cytoplasm, which would then require further uncoating. If Model 3 or 4 were correct, then fusion of the outermost lipid bilayer of MV with the plasma membrane would leave the entire contents of the particle outside of the cell. While novel models for entry of a multiply membraned particle have been proposed, backed by some electron microscopy (Griffiths et al., 2001a; Locker et al., 2000; Pedersen et al., 2000), the best evidence to date shows that MV enters cells by a simple fusion event between the virus membrane and the plasma membrane, releasing the core and lateral bodies into the cell cytoplasm (Carter et al., 2005). Therefore, the mechanism of entry of the virus particle favors the single membrane model (Model 1) for vaccinia crescent, IV, and MV formation.

In summary, while no single experiment necessarily provides an incontrovertible distinction among the models proposed for the membrane structure of maturing and mature vaccinia virus, taken together the data strongly favor the original one membrane model. This leaves open several interesting issues to be resolved, foremost being the mechanism by which a single bilayer may be fabricated and stabilized using virus-modified cellular precursors in the viroplasm. For the balance of this review, we will use the single membrane model in discussing vaccinia structure and assembly.

B. Source of the Virion Membrane

Few topics in vaccinia morphogenesis have been as controversial as the source of the virion membrane (Sodeik and Krijnse-Locker, 2002). As described previously, *de novo* biogenesis was initially favored, first because no ultrastructural continuity with intracellular membranes could be observed and second, because the lipid content of the virion membrane was distinct from known cellular membranes (Dales and Mosbach, 1968). However, no plausible mechanism for the *de novo* assembly of a membrane within the cytosol has been proposed. The second model for membrane biogenesis was diversion of membranes

from a component of the secretory apparatus. Although none of the membrane proteins within MV were glycosylated, it was assumed that they were synthesized in the ER. This assumption followed first from the findings that several of the major membrane proteins (A14, A17) (Section VI.D) could be cotranslationally inserted into microsomal membranes *in vitro*, and second from ultrastructural observations. Indeed, the ultrastructural observations initially seemed to support a colocalization of the A17 and A14 proteins with the cellular Rab1 protein, which was thought to be a specific marker of the ER/Golgi intermediate compartment (ERGIC) (*Sodeik et al.*, 1993). Thus, derivation of the virion membrane from the ERGIC was assumed for a number of years. However, experiments involving pharmacological inhibitors (brefeldin A, nordihydroguaiaretic acid) or dominant negative variants of cellular proteins (Sar1p) have argued against the role of either the COPI or COPII trafficking pathways in MV biogenesis (*Husain and Moss*, 2003; *Ulaeto et al.*, 1995; Traktman, unpublished). These two vesicular trafficking pathways are the crux of the ERGIC system. Consequently, it seems more likely that a specialized compartment of the ER is the initial source of the virion membrane. Consistent with this idea, H89, a known inhibitor of ER exit site formation, and cerulenin, an inhibitor of *de novo* lipid biosynthesis, have been shown to block the recovery of virion morphogenesis in cultures released from an early morphogenesis block (*Punjabi and Traktman*, 2005).

If the MV membrane were composed of two lipid bilayers, it would be relatively straightforward to envision how vesicular elements could fuse to form growing crescents and remain stable in the cytoplasm. However, diversion of a single lipid bilayer from the ER in order to form nascent crescents, stabilization of the exposed fatty acid chains in these crescents within the cytoplasm, and enlargement of these crescents to form the IV membrane, remains a conundrum. Lipid synthesis is normally centered in the smooth endoplasmic reticulum, and in recent studies delivery of lipids to existing organelles have been focused on the subcellular compartments, plasma membrane associated membranes (PAM), and mitochondrial associated membranes (MAM) (*Gaigg et al.*, 1995; *Pichler et al.*, 2001). Gaining an understanding of how the virus accomplishes membrane biogenesis will be a fascinating challenge.

C. Regulatory Proteins in Viral Membrane Formation

The mechanism by which crescents form remains largely unknown. However, cumulative genetic data implicate four viral proteins in

regulating the earliest steps of this process. These are encoded by the $F10L, H5R, G5R$, and $A11R$ genes.

### 1. F10

The F10 protein was first identified as the major protein kinase encapsidated in the virion core (VPK2) (Lin and Broyles, 1994). Recombinant F10 has enzymatic activity *in vitro*, indicating that the protein possesses intrinsic kinase activity and does not need additional protein cofactors (Lin and Broyles, 1994; Traktman et al., 1995; Wang and Shuman, 1995). The predicted amino acid sequence of F10 retains some of the conserved sequence motifs associated with ATP binding or phosphotransfer, but other motifs cannot be recognized due to the divergence of the F10 sequence from that of the majority of protein kinases. F10 has been shown to direct phosphorylation of serine, threonine, and tyrosine residues, indicating that it is a dual specificity kinase (DSP) (Derrien et al., 1999). The 50-kDa protein is expressed late during infection and undergoes autophosphorylation but does not appear to be proteolytically processed despite having the canonical AG↓X motif associated with the virally encoded proteolytic processing machinery (Section IX.C) (Punjabi and Traktman, 2005). F10 is known to be part of a seven-protein complex implicated in the filling of growing crescents with viroplasmic proteins (Section VII.A), but it has also been shown to be tightly associated with membranes during infection and to bind to some lipid species (phosphoinositide phosphates) *in vitro* (Punjabi and Traktman, 2005; Szajner et al., 2004a). F10 is itself encapsidated in nascent virions, with an average abundance of ~300 copies/core.

The importance of F10 during infection was foreshadowed by the inability of investigators to generate a viable virus in which the $F10L$ gene had been deleted. Subsequent studies identified four temperature-sensitive mutants as encoding mutant alleles of $F10L$; cells infected with these viruses show a profound defect in virion morphogenesis at the nonpermissive temperature (Traktman et al., 1995; Wang and Shuman, 1995). This phenotype has been recapitulated by engineering viruses in which F10 expression is dependent on inclusion of IPTG or tetracycline in the culture medium (Punjabi and Traktman, 2005; Szajner et al., 2004b). The latter studies also confirmed that expression of a catalytically active F10 protein is essential for virus morphogenesis. The first arrest observed on inactivation of F10 occurs prior to the formation of any signs of membrane biogenesis. Although the late viral proteins accumulate normally and factories are formed, virosomes are either absent or malformed, and no crescents,
IV, IVN, or MV are observed (Fig. 11A). Under these circumstances, there is also no phosphorylation of the major membrane proteins A14 or A17 (Section VI.D). When cells infected with tsF10 mutants under nonpermissive conditions were shifted to the permissive temperature at 12 hour postinfection (hpi), membrane biogenesis resumed, and crescents were observed within 45 min. Execution point studies have shown that the F10 kinase is also required for later steps in morphogenesis; under certain experimental protocols, tsF10-infected cells show a defect in crescent filling that it is nearly indistinguishable from that observed when other proteins found within the seven-protein complex are repressed or inactivated (Section VII.A). Identification of viral and cellular proteins whose phosphorylation by F10 is important in morphogenesis is an area of intense interest. Although the phosphorylation of a number of viral proteins either fails to occur during nonpermissive infections with tsF10 mutants (e.g., A14, A17, A30, G7) or is diminished (H5, F17), the phosphorylation of at least some of these proteins has been shown to be blocked when morphogenesis is arrested by other means. Thus, it is not clear which phosphorylation events are directly mediated by F10, and which may be mediated by other viral or cellular kinases in the context of productive morphogenesis. Phosphorylation of at least H5 and A30 has been verified *in vitro*.

In summary, studies with mutants in $F10L$ suggest that virus-mediated phosphorylation plays a critical role throughout MV assembly, starting from the very earliest events in virus membrane formation. It is important to reiterate that there is no evidence of membrane diversion or remodeling when F10 is repressed or inactivated.

### 2. H5

The H5 protein has a predicted MW of 22 kDa but has an electrophoretic mobility consistent with a MW of 35 kDa, due to a proline-rich domain within the N-terminus (Gordon et al., 1991). The protein is expressed constitutively and is encapsidated at ~2500 copies/core (Traktman, unpublished). Some of the early literature on the H5 protein is misleading because an antiserum purported to show specificity for H5 was in fact quite reactive against the comigrating H3 protein. The bona fide role of H5 during the viral life cycle has been confusing because genetic and biochemical studies have implicated H5 in several processes. The protein has an amphipathic helix and is phosphorylated *in vivo*; H5 can be phosphorylated by both vaccinia-encoded kinases (B1 and F10) as well as by cellular kinases (Beaud and Beaud, 2000; Beaud et al., 1995; Brown et al., 2000; Traktman, unpublished).
RICHARD C. CONDIT ET AL.

FIG 11. Representative morphogenesis arrests seen during nonpermissive infections with mutants affected in genes involved in MV formation. (A–C) Mutants affecting membrane formation. (D–F) Mutants affecting IV formation and encapsidation. (G–I) Mutants affecting core formation. (A) Formation of factories, but no virosomes and no evidence of morphogenesis: tsF10 mutant (*Traktman et al.*, 1995); (B) accumulation of virosomes, membrane vesicles, and a few aberrant crescents: inducible A14 mutant (*Traktman et al.*, 2000); (C) formation of virosomes and accumulation of membrane vesicles: inducible A17 mutant (Punjabi, Mercer, and Traktman, unpublished); (D) accumulation of empty IV and virosomes: tsG7 mutant (2nd arrest) (*Mercer and Traktman*, 2005); (E) formation of IV and DNA crystalloids due to arrest prior to genome encapsidation: inducible A13 mutant (*Unger and Traktman*, 2004); (F) accumulation of IV, DNA crystalloids and aberrant MV due to lack of genome encapsidation: tsI6 mutant (*Grubisha and Traktman*, 2003); (G) accumulation of aberrant IV due to absence of nucleoid and/or core formation: inducible F17 mutant (Mercer and Traktman, unpublished); (H) accumulation of IV and aberrant particles due to lack of core formation; tsI7

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 75

An interaction between H5 and A20, a stoichiometric component of the processive DNA polymerase, has been found by yeast two hybrid analysis, thus implicating H5 in DNA replication (Ishii and Moss, 2002; McCraith *et al.*, 2000). Consistent with this finding, one of the temperature-sensitive mutants from the Dales collection, which has been the subject of recent systematic analysis, has been shown to have a lesion within the $H5$ gene, and preliminary analysis of this mutant reveals a defect in DNA replication (D’Costa, Prins, and Condit, unpublished; Lackner *et al.*, 2003). Two lines of evidence also implicate H5 in the process of viral transcription: H5 stimulates late gene transcription *in vitro*, and H5 has been shown to interact with proteins which regulate late gene transcription (G2, G8, A2) (Black *et al.*, 1998; Dellis *et al.*, 2004; Kovacs and Moss, 1996). However, a different temperature-sensitive mutant, generated by targeted alanine-scanning mutagenesis of the $H5R$ gene, implicates H5 in virus assembly. This mutant, which has a dominant phenotype, is unimpaired in its ability to direct viral gene expression but shows a profound defect in virion morphogenesis (DeMasi and Traktman, 2000). The phenotype is quite similar to that seen in tsF10 mutants: virosomes are either absent or malformed, and no signs of membrane biogenesis are seen. Whether H5 plays a direct role in morphogenesis, or whether the mutant H5 protein somehow sequesters F10 and hence arrests morphogenesis, is not known. Because H5 is so abundant, it is also possible that it serves as a scaffold for transcription, replication, and virion assembly, thus accounting for the apparent diversity of roles for the H5 protein during infection.

3. G5

The 50-kDa G5 protein is expressed at early times postinfection and is encapsidated in the virion core, although the abundance of the protein within virions has not been reported (da Fonseca *et al.*, 2004). The essentiality of the $G5R$ gene is suggested by its conservation

mutant (Kane and Shuman, 1993); (I) accumulation of aberrant MV due to defect in core wall formation: tsA3 mutant (insets are wrapped aberrant MV [lower left] and extracellular aberrant MV [mid right]) (Kato *et al.*, 2004). (A) Reprinted from Traktman *et al.* (1995) with permission. (B) Reprinted from Traktman *et al.* (2000) with permission. (D) Reprinted from Mercer and Traktman (2005) with permission. (E) Reprinted from Unger and Traktman (2004) with permission. (F) Reprinted from Grubisha and Traktman (2003) with permission. (H) Reprinted from Kane and Shuman (1993) with permission. (I) Reprinted from Kato *et al.* (2004) with permission.

within most if not all poxviral genomes; attempts to isolate a virus deleted for the $G5R$ ORF have been unsuccessful. Temperature-sensitive mutants carrying $G5R$ lesions were isolated by targeted alanine scanning mutagenesis, and phenotypic analysis of two such mutants revealed a role for $G5R$ in the earliest stages of virion morphogenesis (da Fonseca et al., 2004). As was seen in tsF10 and tsH5 infections, factories were formed during tsG5 infections; however, electron-dense virosomes were either absent or malformed, and, most importantly, no signs of membrane biogenesis were seen.

### 4. A11

The 40-kDa A11 protein is expressed at late times postinfection and is phosphorylated in an F10-independent manner (Resch et al., 2005). An inducible recombinant in which A11 expression is dependent on inclusion of IPTG in the culture medium has been generated; repression of $A11R$ transcription has a profound impact on virus production and leads to an early arrest in virion morphogenesis. This is a striking observation, since A11 itself does not appear to be encapsidated and is therefore the first nonstructural protein to be identified as playing an essential role in the biogenesis of the virion membrane. In the absence of A11, factories were formed; however, no crescents, IV, IVN, or MV were seen. Electron-dense virosomes were observed, as were unusual areas of intermediate electron-density that were associated with membranous structures containing markers of the endoplasmic reticulum. These areas of intermediate density also contained the D13 crescent scaffold protein (Section VI.E).

#### D. Membrane Proteins Essential for Membrane Biogenesis: A14 and A17

The proteins encoded by the $A14L$ and $A17L$ genes are abundant components of the MV membrane, and both are essential for its biogenesis.

### 1. A17

The 203-aa A17 protein is expressed at late times of infection. The protein has been shown to undergo cotranslational insertion into microsomal membranes *in vitro* and to associate with ER membranes *in vivo* (Krijnse-Locker et al., 1996). In the context of its association with microsomal membranes, A17 spans the membrane twice, with

the N- and C-termini extending into the solution and an internal hydrophilic loop extending into the lumenal space (Fig. 12) (Betakova *et al.*, 1999a). An intramolecular disulfide bond is formed within this central loop (Betakova and Moss, 2000). A fraction of A17 proteins appear to form intermolecular disulfide bonds through a Cys residue found in the cytosolic C-terminal tail.

The N- and C-termini of A17 are modified by proteolytic cleavage events, which occur at AG16↓A and AG185↓A motifs (Section IX.C) (Fig. 13). These cleavages occur earlier than those associated with maturation of the major core proteins and do not depend on the IV to MV transition, since they occur in rifampicin-arrested cultures in which morphogenesis is blocked prior to IV formation (Section VI.E). However, during the normal morphogenetic pathway, immunoEM analysis suggests that loss of the C-terminal tail, at least, occurs largely after IV are formed. The N-terminal hydrophilic tail of the mature form of A17 (aa 17–60) is accessible to antibodies and to protease in intact MV, confirming the surface localization of the protein; the hydrophilic tail at the C-terminus is only predicted to be 23 aa long after proteolytic cleavage, which may limit its accessibility to these reagents (Betakova *et al.*, 1999b; Wallengren *et al.*, 2001).

A17 also undergoes phosphorylation on Ser, Thr, and Tyr residues in an F10-dependent manner (Betakova *et al.*, 1999b; Derrien *et al.*, 1999). Phosphorylation of A17 fails to occur if morphogenesis is arrested prior to normal crescent formation. Tyr phosphorylation has been shown to occur on Tyr203, the terminal amino acid in the protein; conflicting results obtained with different pTyr antibodies has obscured the question of whether other Tyr residues also undergo modification. The Ser/Thr phosphorylation appears to occur solely in the C-terminal hydrophilic tail of the protein that is removed during morphogenesis.

Inducible recombinants in which A17 expression is dependent on inclusion of IPTG in the culture medium have been generated; A17 is essential for virion production (Rodriguez *et al.*, 1995, 1996; Wolff *et al.*, 1996). In the absence of A17, vast numbers of membrane vesicles accumulate at the periphery of electron dense virosomes (Fig. 11C). Thus, A17 does not appear to be essential for the F10-dependent process of membrane diversion but is essential for crescent formation. The A17 protein has been shown by coimmunoprecipitation analysis to interact with the A14 protein during productive membrane biogenesis (Betakova *et al.*, 1999b; Mercer and Traktman, 2003; Rodriguez *et al.*, 1997); this interaction is mediated by the N-terminus of A17 (Unger, Mercer, and Traktman, submitted for publication).

External
Internal

A14
A17

N'
C'
P
N'
C'
N'
C'
N'
C'
N'

A13
A9
L1
H3
D8

a.a.
90
203
70
108
250
324
308

External
Internal

C'
C'
C'
C'
C'
C'
C'
C'
C'

N'
N'
N'
N'
N'
N'
N'
N'
N'

G3
A21
L5
J5
A28
H2
G9
A16
I2
A14.5
I5

a.a.
111
117
128
133
146
189
309
378
78
53
79

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 79

2. A14

The 90-aa A14 protein, which interacts with the A17 protein, is another major component of the MV membrane. A14 undergoes cotranslational insertion into microsomal membranes *in vitro* and has been shown to associate with ER membranes *in vivo* (Rodriguez et al., 1997; Salmons et al., 1997). The protein appears to span the membrane twice, assuming a topology in which the N- and C-terminal tails are luminal and the central hydrophilic loop is exposed (Fig. 12). The C-terminal tail contains an N83HS motif, which is readily glycosylated *in vitro* but poorly glycosylated *in vivo* during wt infections (Mercer and Traktman, 2003). Moreover, none of the glycosylated A14 protein appears to be encapsidated, suggesting that interactions of A14 with other viral proteins, or further posttranslational modifications (see later) may prevent glycosylation of A14. Alternatively, the majority of A14 may be synthesized in, or rapidly diverted to, a membrane domain that does not support glycosylation. This latter hypothesis is supported by observations that the fraction of A14 that undergoes glycosylation is significantly higher when membrane biogenesis is prevented at an early stage by repression or inactivation of F10, H5, A17, or G7.

A14 undergoes F10-dependent phosphorylation on Ser85, which is also part of the glycosylation motif discussed previously (Mercer and Traktman, 2003). Disruption of the glycosylation motif increases the level of A14 phosphorylation, suggesting that the two posttranslational modification systems are competing for access to the protein. Finally, the vast majority of A14 dimerizes by forming intermolecular disulfide bonds through Cys71. A mutant allele of A14 in which Cys71 has been altered to Ser is viable, although viral yields are reduced approximately fivefold. Virions containing this altered A14

FIG 12. Schematic view of integral MV membrane proteins. The overall topology of known transmembrane proteins contained within the MV membrane is shown; external and internal refers to the orientation within MV. A14 and A17 are essential for biogenesis; A13, A9, and L1 are required for virion maturation; H3 stimulates virion maturation and, along with D8, binds to glycosaminoglycans on the surface of target cells; G3, A21, L5, J5, A28, H2, G9, and A16 comprise an eight-member fusion/entry complex; 12 is also required for virion entry; A14.5 affects virulence in mice, and the function of I5 has not been determined. The number of amino acid residues is shown beneath each protein. Sites of phosphorylation, P0 myristylation (chemical structure), proteolytic cleavage (dotted line), and disulfide bond formation (-S-) are shown. The symbol shown adjacent to the H2 and A16 proteins ♦ indicates that, although these proteins are known to contain intramolecular disulfide bonds, the number and position of these linkages are not known.

protein are exquisitely sensitive to nonionic detergents. Under conditions in which wt virions remain intact and infectious, NP40 treatment of these A14-Cys71 virions leads to the extraction of both A14 and A17 and causes a 1000-fold decrease in specific infectivity. These findings support a model in which dimerized A14, interacting noncovalently with A17, forms a lattice within the MV membrane that sustains virion integrity, consistent with previous results from controlled degradation of MV described in Section III.C.

Recombinant viruses in which expression of A14 is dependent on inclusion of either IPTG or tetracycline in the culture medium have been generated; expression of A14 is essential for virion production ([Rodriguez *et al.*, 1998; Traktman *et al.*, 2000]). In the absence of A14, vast numbers of vesicles (and/or tubules) accumulate within the cytoplasm (Fig. 11B). Some aberrant crescents are seen which are adjacent to, but fail to make contact with, electron-dense virosomes that accumulate in factories. Importantly, the vesicles formed in the absence of A14 contain A17 and another membrane protein, D8, while the virosomes formed contain the core proteins L4 and F17 ([Traktman *et al.*, 2000]). Thus, the A14 mutant phenotype is similar but not identical to that seen upon repression of A17. Most importantly, the presence of both of A14 and A17 is required to form bona fide crescents, which can progress to form the IV membrane. Whether the vesicles seen upon their repression represent stalled intermediates, or aberrant structures formed by the collapse of unstable membranes, has not been determined. This is a question of considerable relevance to a fuller appreciation of the mechanism of IV membrane biogenesis.

Transient complementation has been used to probe which features of the A14 protein are essential for membrane biogenesis. Sequences that flank the first transmembrane domain, and those within the external, hydrophilic loop are most sensitive to mutation. The latter region has been found to interact with the N-terminus of A17 *in vitro*; moreover, mutations that disrupt complementation activity *in vivo* disrupt interaction *in vitro* (Unger, Mercer, and Traktman, submitted for publication).

In summary, the A17 and A14 membrane proteins are critical players in diversion of ER membranes to viral factories and formation of the viral membrane. The proteins are synthesized in the ER and trafficked to viral factories to form membrane-containing crescents. Within crescents and the mature virion membrane, A14 and A17 appear to form a lattice that is stabilized by disulfide bonds, probably contributing to the sensitivity of the virion membrane to reducing agents. Finally, the A14–A17 lattice serves as an anchor within the

viral membrane to which several other proteins important in virion structure and morphogenesis attach (see in a later section).

### E. D13: The Crescent Scaffold Protein

The “spicule” layer that comprises the convex surface of crescents, IV and IVN was first described in detail by Dales and Mosbach (1968). Based on studies with metabolic inhibitors that partially disrupted crescent formation these authors concluded that the spicule layer conferred curvature and rigidity to the crescents. This conclusion was confirmed and extended through studies with the antibiotic inhibitor rifampicin. Rifampicin, better known as an inhibitor of prokaryotic transcription, inhibits vaccinia virus morphogenesis (Grimley et al., 1970; Moss et al., 1969a,b; Nagayama et al., 1970). In the presence of this drug, irregularly shaped membranes that lack a spicule layer accumulate around areas of electron-dense viroplasm. These membranes can be labeled with anti-A17 antibody using immune electron microscopy, confirming a relatedness to virus crescent membranes (Wolffe et al., 1996). Inhibition with rifampicin is rapidly reversible; normal crescents form within minutes after the removal of drug.

Several lines of evidence demonstrate that the spicule protein is the product of gene *D13L*. First, immune electron microscopy shows that in a normal infection, antibody specific for the D13 protein decorates crescents and IV (Miner and Hruby, 1989; Mohandas and Dales, 1995; Sodeik et al., 1994; Szajner et al., 2005). In the presence of rifampicin, virus membranes do not label with D13 antibody, and instead D13 antigen accumulates in inclusions within factories, removed from membranes. When the rifampicin block is reversed, newly formed crescents are now labeled with D13 antibody and the D13 inclusions disappear. Second, mutations that confer resistance to rifampicin map to the *D13L* gene (Baldick and Moss, 1987; Tartaglia and Paoletti, 1985). Third, conditional lethal mutations in the *D13L* gene display a phenotype identical to the effects of rifampicin (Lackner et al., 2003; Stern et al., 1977; Zhang and Moss, 1992).

Experiments demonstrate that the “spicule” layer is in fact a spherical honeycomb lattice composed of D13 trimers (Heuser, 2005; Szajner et al., 2005). The D13 protein itself can associate with two integral virus membrane proteins, A17 and H3, thus providing a mechanism for scaffolding of the virus membrane onto the D13 lattice. As discussed in Section IX.E.1, D13 is lost from virions during maturation from IVN to MV (Essani et al., 1982; Mohandas and Dales, 1995; Sodeik et al., 1994); how the completion of the IV membrane sends a

signal that activates dissolution of the D13 lattice, and how this lattice is dismantled, are intriguing questions that remain to be answered.

In summary, the “spicule” layer that appears on the convex surface of crescents and the outside of IV and IVN is a temporary scaffold for assembly of the virus membrane and is composed primarily of a honeycomb lattice built from D13 protein trimers. It follows that the curvature of the IV membrane, and therefore the final size of IV, is largely determined by the structure of the D13 lattice. Parallels between the D13 lattice and the clathrin coats of endocytic vesicles are evident.

## VII. IMMATURE VIRION FORMATION: A COMPLEX PROCESS

Mutants in seven different vaccinia genes produce a similar phenotype which suggests a role for the gene products in the association of viral crescents with viroplasm. One of these seven proteins, A10 or 4a, is one of the most abundant virion core proteins. The other six proteins, A15, A30, D2, D3, G7, and J1, are found in a seven-protein complex along with the F10 protein kinase, which, as described previously, phosphorylates several membrane and core proteins and plays a critical role throughout virus assembly. The role of each of these proteins in IV formation is summarized in the following sections.

### A. Seven-Protein Complex

As membrane crescents appear, they are often seen at the periphery of electron-dense regions of viroplasm, which is known to contain viral proteins destined for encapsidation into the virion core. It is not known what protein–protein or protein–lipid interactions are involved in mediating the association of viroplasm with growing crescents: there are no known interactions of individual membrane proteins with core proteins. However, a group of seven core proteins, F10, A30, G7, J1, D2, D3, and A15, has been shown to form a multimeric complex, and in some cases the proteins are mutually dependent on one another for stability (*Szajner et al.*, 2004a). Several members of the complex are phosphorylated *in vivo* in an F10-dependent manner. The stoichiometry of the individual components within the complex has not been determined. In addition to being physically linked, these proteins show a genetic interaction, since comparable phenotypes are observed upon mutation or repression of individual members of the complex. As detailed in a later section, the phenotypes suggest a role for these proteins in the association of viroplasm with crescents.

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 83

1. $F10$

    F10 has been described previously in significant detail (Section VI.C.1). With regard to its involvement as a member of the seven-protein complex which mediates the association of viroplasm with growing crescents, it is worth recalling the fact that F10 has been shown to be tightly associated with membranes *in vivo* (Punjabi and Traktman, 2005). Direct evidence for the functional involvement of F10 in the association of viroplasm with crescents was obtained from ultrastructural analyses of tsF10 infections that were initiated at the permissive temperature in the presence of rifampicin, and then shifted to the nonpermissive in the absence of rifampicin at 12 hpi. The phenotype seen here was nearly identical to that observed upon disruption of A30 or G7 expression (see later). Indeed, the phosphorylation of A30 and G7 is dependent on F10 *in vivo* and also appears to be dependent on the assembly of the multimeric complex and the normal progression of morphogenesis (Mercer and Traktman, 2005; Szajner et al., 2004c). It seems likely that F10 mediates phosphorylation of these proteins in a context-dependent manner.

2. $A30$

    Disruption of the $A30L$ gene was shown to lead to a phenotype in which the association of nascent membranes with viroplasm was disrupted. This disruption led to the appearance of large virosomes and “empty” IV and/or pseudo-IV with multiple membrane wrappings. This phenotype was observed when an inducible recombinant was generated permitting the repression of A30, or when a temperature-sensitive mutant, Dts46, was analyzed at the nonpermissive temperature (Szajner et al., 2001a,b). The tsA30 protein is extremely thermolabile: effectively, at the nonpermissive temperature, repression of the A30 protein is mimicked. A30 is a 9-kDa protein that is found in viroplasm and encapsidated within virions; fractionation studies have demonstrated that A30 is loosely associated with the virion core. A30 is phosphorylated on Ser residues *in vivo* in an F10-dependent manner, and it has been shown to be a direct substrate of F10 *in vitro*. A30 has been shown to interact directly with the G7 protein (discussed in a later section), and these proteins appear to depend on one another for their mutual stability.

3. $G7$

    G7 was first implicated in virion morphogenesis because of its association with the A30 protein (Szajner et al., 2003). The 42-kDa G7 protein is expressed at late times postinfection, encapsidated into

virion cores, and processed by proteolytic cleavage at two AG↓X motifs during virion maturation (Fig. 13). The G7 protein is phosphorylated in vivo in an F10-dependent manner, but F10 does not appear to phosphorylate G7 in vitro, either when presented alone or in the presence of its binding partner A30 (Mercer and Traktman, 2005; Szajner et al., 2004c). The function of G7 in vivo has been addressed both by the generation of an inducible recombinant in which G7 expression is IPTG-dependent and by characterization of Cts11, which encodes a defective G7 protein (Mercer and Traktman, 2005; Szajner et al., 2003). These two approaches have yielded somewhat different answers regarding G7’s function, as detailed below.

During infections with Cts11, virosomes accumulate as do short membrane crescents that are embedded within a distinct matrix of medium electron density. Both the G7 and A30 proteins are stable during these infections, permitting the examination of the impact of a specific loss of G7 function. When various G7 and A30 proteins were synthesized in vitro and subjected to coimmunoprecipitation protocols, the tsG7 protein had lost the ability to interact with A30. When Cts11

![Diagram](attachment:diagram.png)

FIG 13. Proteolysis of vaccinia core and membrane proteins. The proteolysis of five core proteins (A10, A3, G7, A12, and L4) and one membrane protein (A17) is shown. Black lines represent the length of the precursor polypeptide, drawn to scale. Red arrowheads represent known cleavage sites. Information concerning the precursor (MW, any pseudonyms in brackets, and localization if relevant) is given above each line. Information concerning each product (MW, pseudonyms in brackets, localization if known) is given below each line. ? = fate of the product is unknown.

infections were performed at permissive temperature in the presence of rifampicin for 12 h and then removed from the rifampicin block and shifted to the nonpermissive temperature, the phenotype observed mimicked what had been observed on repression of A30 (Fig. 11D). This phenotype, characterized by the accumulation of empty IV, is also observed when G7 is repressed. Interestingly, tsG7 mutants are noncomplementing with tsA30 mutants, providing additional genetic evidence for an interaction between G7 and A30 (*Lackner et al.*, 2003). In summary, G7 appears to play crucial roles at two stages of morphogenesis (*Mercer and Traktman*, 2005). First, G7 is required for the movement of crescents to the periphery of virosomes, where they enlarge; later, G7 is required for the filling of the crescents with viroplasmic material. The small amount of wt G7 that is likely to accumulate when infections with the inducible recombinant are performed in the absence of the IPTG inducer may be sufficient for the initial G7-dependent step, whereas the tsG7 protein must be unable to mediate this effect.

4. J1

The 21-kDa J1 protein is expressed at late times and encapsidated; like A30, although J1 is associated with virion cores, a fraction of the protein can be extracted with nonionic detergents. Purified J1 has been shown to form oligomers (*Chiu et al.*, 2005), and interactions between the N- and C-termini of J1 have been observed in yeast two-hybrid analyses. The role of J1 *in vivo* has been analyzed using both an inducible recombinant in which J1 expression is IPTG-dependent and through the analysis of Cts45, which encodes a mutant J1 protein (*Chiu and Chang*, 2002; *Chiu et al.*, 2005). Repression or inactivation of J1 phenocopies the repression of A30 or G7, and indeed both A30 and G7 become unstable in the absence of J1 at 40°C. J1 is part of the seven-protein complex involved in viroplasm/membrane association, although the proteins with which J1 associates directly have not been identified (*Szajner et al.*, 2004a). The tsJ1 protein remains stable at the nonpermissive temperature but does not associate with the multiprotein complex.

5. A15

The 10-kDa A15 protein was shown to be a component of the seven-protein complex by mass spectroscopy (*Szajner et al.*, 2004a). This protein is expressed at late times postinfection and encapsidated in the virion core. An inducible recombinant in which A15 expression is IPTG-dependent has been generated; in the absence of A15, the characteristic accumulation of empty IV was observed (*Szajner et al.*, 2004a).

RICHARD C. CONDIT ET AL.

6. D2

D2 was identified as being a component of the seven-protein complex by mass spectroscopy (Szajner et al., 2004a). The 16.9-kDa D2 protein had been known to be a component of the virion core (Dyster and Niles, 1991). The temperature sensitivity of the Ets52 mutant is due to a mutation in D2 that renders the protein extremely thermolabile; at the nonpermissive temperature, few signs of morphogenesis are seen. Crescents and empty IV are observed only occasionally. The phenotype is similar to that seen for the leakier tsF10 mutants (Traktman et al., 1995; Wang and Shuman, 1995). An inducible recombinant has been generated in which expression of D2 is dependent on IPTG. Even in the presence of IPTG, the virus forms small plaques, suggesting that the epitope tag appended to D2 is deleterious. Nevertheless, repression of D2 does severely impair virion maturation, leading to an accumulation of empty IV (Szajner et al., 2004a).

7. D3

The 27-kDa D3 protein is also a component of the virion core and seven-protein complex (Dyster and Niles, 1991; Szajner et al., 2004a). The phenotype of Cts35, which encodes a defective D3 protein that is extremely thermolabile, is virtually indistinguishable from that seen for tsD2. Consistent with their presence in the seven-protein complex, stability of the D2 and D3 proteins is mutually interdependent: both D3 and D2 are destabilized during tsD3 and tsD2 infections.

B. A10: p4a/4a

The A10L gene encodes the virion precursor protein p4a (Witteke et al., 1984). During morphogenesis from IV to MV, p4a is processed by two proteolytic cleavages into an N-terminal 62-kDa polypeptide, an internal 9-kDa polypeptide, and a C-terminal 23-kDa polypeptide (Fig. 13) (Vanslyke et al., 1991b). Both the 62- and the 23-kDa polypeptides are packaged into virions; the fate of the 9-kDa polypeptide is unknown (Vanslyke and Hruby, 1994; Vanslyke et al., 1991b). The 62-kDa polypeptide is also commonly called 4a. 4a localizes to the virion core (Sarov and Joklik, 1972); Ichihashi et al. (1984) suggest that in fact it is a component of the outer core wall. 4a comprises an impressive 14% of the mass of the mature virion, or 28% of the mass of the virion core (Sarov and Joklik, 1972). Thus, p4a and its proteolytic products are major determinants of vaccinia virion structure. A10 function has been examined using an inducible recombinant in which A10 expression is IPTG-dependent. In the absence of

inducer, normal crescents are seen, but the IV that form are aberrant ([Heljasvaara *et al.*, 2001](https://doi.org/10.1016/S0042-6822(00)00375-9)). Many of the abnormal IV particles appear empty, but others contain dense patches of DNA-containing material. DNA aggregates also accumulate in the cytoplasm. The membranes surrounding the IV are normal in appearance and contain the A14 and A17 membrane proteins as well as the D13 scaffold protein, as judged by immunoelectron microscopy. Thus, A10 appears to be dispensable for the formation of the IV membrane but is required for the proper organization and packaging of the viroplasmic matrix that is normally included in IV. The phenotype observed on repression of A10 mimics to a great degree the phenotype of mutants affected in components of the seven-protein complex described previously.

The A10 precursor and the 60-kDa cleavage product form stable complexes with another abundant virion core protein, A4 ([Risco *et al.*, 1999](https://doi.org/10.1016/S0022-2836(99)80027-4)). The implications of this interaction are discussed in Section IX.B.3 along with the role of A4 in MV formation.

---

## VIII. IV→IVN TRANSITION: GENOME ENCAPSIDATION

The process of viral DNA replication produces progeny genomes that will be encapsidated into nascent virions. Concatemeric intermediates are formed during the replication process, and these are processed to monomeric genomes by a virally encoded resolvase ([Garcia *et al.*, 2000](https://doi.org/10.1016/S0022-2836(00)96968-0)) (Section VIII.B.3). Genome maturation occurs normally when virion morphogenesis is blocked in the very earliest stages of morphogenesis, for example, in H5, G5, and J1 mutant infections ([Table II](#table:II)), indicating that this maturation is a prerequisite for, but is not coupled to, genome encapsidation ([Chiu and Chang, 2002](https://doi.org/10.1016/S0022-2836(02)00003-5); [da Fonseca *et al.*, 2004](https://doi.org/10.1016/j.virol.2004.03.020); [DeMasi and Traktman, 2000](https://doi.org/10.1016/S0042-6822(00)00375-9)). Much remains to be discovered about the encapsidation of the viral genome, although it appears to be distinct from the process described previously for the filling of maturing crescents with viroplasmic proteins destined to form the virion core. One of the unresolved questions is whether genome entry precedes closure of the IV membrane. Serial section analysis shows that most fully closed IV do indeed have nucleoids ([Morgan *et al.*, 1955](https://doi.org/10.1093/aob/mcmaa01)), suggesting that genome encapsidation is likely to occur prior to the closure of the IV ([Morgan, 1976a,b](https://doi.org/10.1093/aob/mcmaa01)). If this model is not correct, then the encapsidation machinery must either pass through a proteinaceous plug that has been proposed to mark the site of IV membrane closure, or it must pass through the IV membrane itself. Some investigators have proposed that genome entry causes membrane invagination, leading to the

presence of a membrane around the nucleoid and, consequently, the core. However, this model is not widely accepted (Section VI.A) (Fig. 8).

It is not known exactly how replicated genomes are distributed throughout the cytoplasm, although sites of replication have been visualized by autoradiography or with antisera directed against BrdU, DNA, or the single-stranded binding protein, I3. When morphogenesis is arrested prior to the appearance of IVN, DNA crystalloids accumulate in the cytoplasm (Ericsson *et al.*, 1995; Grimley *et al.*, 1970; Grubisha and Traktman, 2003; Unger and Traktman, 2004). These crystalloids are often surrounded by membranes, consistent with the proposal that DNA replication occurs within ER-delimited domains of the cytoplasm (Schramm and Locker, 2005; Tolonen *et al.*, 2001) (Section V) (Fig. 5).

The literature contains a number of ultrastructural images that have been interpreted as representing the ongoing process of DNA encapsidation (Morgan, 1976a,b). Although this process is commonly referred to as encapsidation of the viral "nucleoprotein," there is in fact nothing known about how the genome is condensed and whether it is indeed bound to proteins. As described later, when either of the two proteins that have been specifically associated with genome encapsidation are repressed or inactivated, no other proteins appear to be missing from the virions (Cassetti *et al.*, 1998; Grubisha and Traktman, 2003). These data do not support a model in which the genome is assembled into a conventional nucleoprotein complex. In fact, it may be that condensation of the genome is effected by interactions with polyamines; encapsidation of spermidine has been reported (Lanzer and Holowczak, 1975; Saminathan *et al.*, 2002). In any case, the genome has a predicted dimension of 2 nm × 68 μm, which must fit into the viral core. The approximate dimensions of the nucleoid, as observed in ultra thin sections, are 0.1 × 0.15 μm, indicating that the genome must undergo massive condensation before encapsidation.

To date, four viral proteins have been implicated in maturation and processing of the viral genome. These include the core proteins A22 (a Holliday resolvase), I6 and A32, and the membrane protein A13.

### A. Membrane Proteins

#### 1. A13

The 70-amino acid A13 protein has a hydrophobic N-terminus, which is predicted to be the membrane-spanning domain; indeed A13 undergoes cotranslational insertion into microsomal membranes *in vitro* (Salmons *et al.*, 1997) and is a component of the virion

membrane (Fig. 12). The hydrophilic tail of A13 undergoes phosphorylation *in vivo* on Ser residues; mutation of Ser40→Ala prevents phosphorylation (Unger and Traktman, 2004). This modification appears to be independent of the F10 kinase; Ser40 is found within the motif NSPPP, which is predicted to be a high-affinity site for the cellular kinase ERK. The function of A13 has been elucidated both by the analysis of a TET-inducible recombinant and a temperature-sensitive mutant (Cts40) (Unger and Traktman, 2004); the A13 protein in Cts40 has a Thr48→Ile substitution and is thermostable but apparently inactive. When A13 is inactivated, a morphogenesis arrest is seen; IV are formed, but IVN are rare and MV are not seen (Fig. 11E). Numerous DNA crystalloids, which appear to be surrounded by membranes, accumulate in the cytoplasm. Thus, A13 appears to be essential for the encapsidation of DNA into IV and the subsequent maturation of IV→IVN→MV.

### B. Core Proteins

#### 1. A32

Bioinformatic analysis was first used to identify the A32 protein as a putative ATPase with limited homology to the encapsidation proteins encoded by bacteriophages and adenovirus (Koonin et al., 1993). The 34-kDa protein is expressed at late times postinfection and is encapsidated; to date, there has been no experimental data confirming that the protein indeed possesses the predicted ATPase activity. An inducible recombinant in which expression of A32 is IPTG-dependent has been generated; in the absence of A32, the production of infectious virus is severely diminished (Casetti et al., 1998). Morphogenesis appears normal through the IV stage, but then aberrant forms are seen. Spherical virions that appear dense or half-dense/half-empty are numerous, as are cytoplasmic DNA crystalloids. The aberrant virions have been purified and shown to contain the full-protein complement but to be devoid of viral DNA.

#### 2. I6

The 44-kDa I6 protein was identified as binding with great specificity and stability to the telomeric hairpins of the viral genome (Demasi et al., 2001b). The telomere–I6 interaction was shown to be dependent on the presence of the extrahelical bases, which are so characteristic of the viral telomeres. Very low levels of I6 are made at early times postinfection; the majority of the protein is synthesized after the onset of DNA replication. An average of ~35 copies of I6 is encapsidated in the virion core. A series of mutants with lesions in the *I6L* gene have been

generated by clustered charge-to-alanine mutagenesis, and one of these has a tight temperature-sensitive phenotype (Grubisha and Traktman, 2003). At the nonpermissive temperature, the phenotype observed is almost indistinguishable from that seen on repression of A32: IV appear normal, cytoplasmic DNA crystalloids are abundant, and aberrant, spherical virions are produced that contain the full-protein complement but no viral DNA (Fig. 11F). A32 remains stable during these infections and is encapsidated into the aberrant virions, indicating that its encapsidation is independent of DNA encapsidation and I6 function. Conversely, no I6 is encapsidated into A32-deficient virions.

These cumulative data support a model in which the presence of A32 within virions enables the encapsidation of a DNA genome on which I6 protein is bound to the telomeric hairpins. As described previously, repression of certain membrane proteins, such as A13, leads to the accumulation of IV and DNA crystalloids; it is possible that the presence of these proteins in the membrane is also important for A32 and/or I6-DNA encapsidation.

### 3. A22, a Holliday Resolvase

As described previously, concatemeric replication intermediates are processed to mature genomes by a virally encoded Holliday junction resolvase, the A22 protein (Garcia et al., 2000). The product of the A22R gene is a palmityl protein that is expressed late during virus infection and packaged into the virion core (Garcia and Moss, 2001; Grosenbach et al., 2000). Analysis of an inducible mutant demonstrates that in the absence of A22, concatemer resolution is defective, and virion morphogenesis is arrested at an intermediate stage (Garcia and Moss, 2001). In these A22-deficient infections, MV formation is inhibited and IV accumulate, a very few of which have nucleoids. In addition, there is an accumulation of dense spherical particles similar to those seen in other encapsidation defective mutants. Thus, while genome maturation can proceed in the absence of virion morphogenesis, the converse is not true. If concatemer resolution is inhibited, virion morphogenesis is arrested and the nature of the intermediate particles that accumulate suggests that the defect lies in DNA packaging.

---

### IX. MV FORMATION

Perhaps the most striking event during morphogenesis of vaccinia is the transition from IVN to MV. The precise protein composition of immature particles is difficult to assess because normal IV and IVN

are difficult to isolate in sufficient quantity or purity to permit rigorous biochemical study (Vanslyke *et al.*, 1993). Nevertheless, as detailed later, a synthesis of the evidence to date suggests that IVN contain the full complement of proteins contained in MV, with the exception of a few membrane proteins that are added during the IV/IVN to MV transition. Furthermore, since both the IV membrane and the MV membrane contain the major viral membrane proteins A14 and A17, the membrane of IVN is the precursor for the MV membrane (Griffiths *et al.*, 2001a; Rodriguez *et al.*, 1997; Traktman *et al.*, 2000). Thus the IV/IVN to MV transition involves a rearrangement of the contents of IVN to form a distinct core and lateral bodies, and in the process the particle is transformed from a spherical to a barrel shape. Indicative of the complexity of this process, a significant majority of virus mutants that affect virion morphogenesis affect the IV/IVN to MV transition. These mutants fall into two general classes: those that fail to assemble morphologically normal looking MV particles and those that assemble normal looking MV particles, which are nevertheless not infectious. Analysis of these mutants informs us that the IV/IVN to MV transition involves or requires: (1) assembly of the transcription apparatus within the core, (2) restructuring of the particle surface, including loss of the D13 protein, formation of numerous disulfide bond linkages by the virally encoded redox system and addition of several additional surface proteins, (3) proteolysis of several virion protein precursors and formation of an enzymatically active core, in part through the action of the viral phosphatase, and (4) movement of the finished MV from viroplasm to the periphery of the cell. Each of these events is discussed in more detail in later sections.

### A. Assembly of the Transcription Apparatus Within the Core

The phenotype of virus mutants affecting two different virus coded transcription factors, VETF and rap94, reveals an intriguing relationship between assembly of the early transcription machinery within the virion core and formation of MV. Vaccinia RNA polymerase exists in two different forms, one that contains the subunit rap94 (the product of the $H4L$ gene), and one that lacks rap94 (Broyles, 2003). The rap94-containing enzyme transcribes early viral genes exclusively, while RNA polymerase lacking rap94 transcribes exclusively intermediate and late genes. VETF, composed of the D6 and A7 proteins, binds specifically to early virus promoters and is required for early gene transcription *in vitro* (Broyles, 2003). Thus, specificity of the RNA polymerase for early promoters is thought to result from a specific

interaction between rap94 and VETF, although such an association has never been proven directly.

### 1. D6 and A7: VETF

Genetic analysis of VETF is facilitated by the existence of inducible recombinants and temperature-sensitive mutants. There exist multiple temperature-sensitive mutants with lesions in the $D6R$ gene, and inducible recombinants have been generated which allow IPTG-dependent expression of both subunits of VETF, D6 and A7 (Christen *et al.*, 1992; Hu *et al.*, 1996, 1998). Ultrastructural analysis of cells infected with the temperature-sensitive mutants has not been reported, but such studies have been published for the inducible recombinants. When either VETF subunit is repressed, a “normal” gene expression phenotype is observed; that is, all three classes of viral proteins, early, intermediate, and late, are synthesized in normal amounts and with normal kinetics. However, electron microscopic analysis of the mutant infections reveals a block in morphogenesis at the IV/IVN stage. MV are either absent (D6) or significantly reduced in number (A7) in the mutant infections. Instead, an abundance of morphologically normal IV and IVN accumulate and occasional abnormal, dense, spherical particles are seen in viral factories. Confocal microscopy of A7-deficient infections stained for DNA and a virion membrane protein suggests that at least a fraction of the immature particles

5 A general and seemingly peculiar feature of mutants in vaccinia genes affecting virion transcription enzymes is that they confer a phenotype that is expressed late rather than early during infection. Offhand, one might expect a mutation in a virion transcription enzyme, in particular a temperature-sensitive mutant, to affect early viral transcription and thus disrupt the infection at an early stage. However, all mutants in transcription enzymes reported to date proceed through the early phase of infection normally and display a phenotype that impacts the late phase of infection. In the case of inducible mutants, this phenomenon is not difficult to understand. The inducible mutant must be grown under permissive conditions in order to study the phenotype under nonpermissive conditions. During growth under permissive conditions, the wild type gene product is synthesized and packaged into virions, which are therefore competent for early transcription under nonpermissive conditions. The nonpermissive infection therefore only tests for any requirement for the protein during the later phases of infection. In the case of temperature-sensitive mutations the same phenomenon is observed; however, in this case the virions synthesized under permissive conditions package mutant enzyme. One must therefore argue that the mutant proteins are not thermolabile in virions, but rather that they are temperature sensitive for synthesis, that is, newly synthesized mutant protein is either rapidly degraded or cannot form an active conformation at the nonpermissive temperature, and thus the phenotype observed represents the requirement for newly synthesized enzyme late during infection (Hooda-Dhingra *et al.*, 1989).

contain DNA. Both mutants also accumulate abnormally large masses of viroplasm within viral factories. The protein composition of the accumulated IV and IVN is unknown. Thus in the absence of VETF, virus assembly arrests at the IV/INV stage, implying a pivotal role for VETF in formation of the viral core and transition from IVN to MV.

2. $H4$: rap94

Genetic analysis of the $H4L$ gene, which encodes rap94, is also enabled by the existence of both inducible and temperature-sensitive mutants affected in this gene (Kane and Shuman, 1992; Zhang et al., 1994). As was described for the VETF mutants, inactivation or repression of rap94 does not impair the profile of viral gene expression. Unlike the VETF mutants however, both the temperature sensitive and the inducible mutants in rap94 assemble particles that are morphologically indistinguishable from wt MV as judged by electron microscopy. These MV made under nonpermissive conditions are deficient in their ability to direct *in situ* transcription *in vitro*. Moreover, they lack not only rap94 as expected but also specifically lack all of the major known or suspected enzymes involved in early viral transcription, including the viral RNA polymerase, the capping enzyme/termination factor, the poly(A) polymerase large subunit, DNA-dependent ATPase I, RNA helicase, and topoisomerase. These abnormal MV contain normal amounts of VETF. These results are interpreted to mean that a presumed rap94–VETF interaction seeds assembly of a transcription complex onto the viral genome and thus coordinates packaging of the transcription machinery during virus assembly. Notably, the result also implies that the morphological transformation from IV to MV, including formation of the core and lateral bodies, occurs independently of the assembly of the transcription apparatus within the core.

Together, studies with mutants in VETF and rap94 suggest that the early viral transcription factor VETF, perhaps in conjunction with or through interaction with other DNA binding or structural proteins, plays a critical role in the morphological transition from IVN to MV, and also, perhaps through an interaction with rap94, plays a critical role in recruitment and assembly of a transcription complex in the developing virion. The recruitment and assembly of the transcription complex itself, however, is not a prerequisite for virion maturation, in that normal appearing MV can be assembled in the absence of the transcription complex. Finally, it is worth recalling that encapsidation of VETF and the early transcriptional machinery does not appear to require their association with the viral genome, since DNA-deficient

particles assembled in the absence of the A32 protein do contain the early transcriptional machinery (Cassetti *et al.*, 1998).

### B. Core Proteins Involved in the IV to MV Transition

A number of core proteins are required for virion maturation, as revealed by the phenotypes exhibited by temperature-sensitive mutants that encode mutant forms of these proteins, or inducible recombinants which allow the expression of the proteins of interest to be experimentally repressed. In some cases, repression or inactivation of the protein simply arrests morphogenesis at the IV or IVN stage; in other cases, aberrant particles are seen. Since not all publications are equally thorough in their ultrastructural analysis, implication of a given protein in a precise step of virion assembly is not always possible. It would appear that when the D13 scaffold remains intact, there is an accumulation of IV and/or IVN. In contrast, when the D13 scaffold is removed, then further virion maturation is attempted unsuccessfully, leading to the production of aberrant particles.

#### 1. I1

The 36-kDa I1 protein is expressed at late times postinfection and encapsidated into virion cores (~700 copies/virion) (Klemperer *et al.*, 1997). The protein has nonspecific DNA binding activity but shows some preference for binding to the viral telomeres (Demasi *et al.*, 2001a; Klemperer *et al.*, 1997). An inducible recombinant in which I1 expression is IPTG-dependent has been generated; upon the repression of I1, morphogenesis arrests with the accumulation of numerous IV. IVN were also seen, although some DNA crystalloids were observed in the cytoplasm (Klemperer *et al.*, 1997). The I1 protein is highly conserved among poxviral genomes, in keeping with its essentiality for virion maturation.

#### 2. F17

The 11-kDa F17 protein is expressed at late times postinfection and is one of the major components of the core; ~27,000 copies of the protein are encapsidated per virion. This protein was initially referred to as p11 or VP11 (Kao and Bauer, 1987; Kao *et al.*, 1981; Nowakowski *et al.*, 1978a,b). The protein is phosphorylated *in vivo* on Ser residues; phosphorylation is only minimally dependent on the F10 kinase but is modulated by expression of the viral H1 phosphatase. It is likely that the protein is phosphorylated on two PSSP motifs which are target sites for cellular proline-directed kinases such as ERK1 and CDK1 (Reddy, unpublished; Traktman, unpublished). The F17 protein is

quite basic and was originally described as a DNA-binding protein that could coat DNA stoichiometrically. However, normal amounts of F17 are encapsidated even when the genomic DNA is not, suggesting that F17 is certainly not a classical “nucleoprotein” in the sense that was originally proposed (Cassetti *et al.*, 1998; Grubisha and Traktman, 2003). An inducible recombinant has been generated in which expression of the *F17R* gene is IPTG-dependent; in the absence of F17, morphogenesis arrests with the appearance of aberrant IV (Fig. 11G) (Zhang and Moss, 1991a,b). In some of these particles, there is an internal nucleoid that resembles the DNA crystalloids described earlier. In other cases, the IV appear almost empty or contain small patches of condensed material and/or membrane. Because DNA crystalloids do not accumulate in the cytoplasm, however (Traktman, unpublished), it is likely that the absence of F17 does not impair genome encapsidation per se but impairs nucleoid formation and disrupts the internal organization of the IV. These aberrant IV do not form a light scattering band upon sucrose gradient ultracentrifugation, indicating that they may be unstable.

### 3. A4

The *A4L* gene encodes a 39-kDa protein that is expressed late and encapsidated into the virion. This protein was first identified as an immunodominant antigen and was referred to as p39 (Cudmore *et al.*, 1996; Demkowicz *et al.*, 1992; Maa and Esteban, 1987; Roos *et al.*, 1996). A4 appears to be phosphorylated *in vivo* and is a substrate for the F10 kinase *in vitro*; A4 phosphorylation appears to be modulated by the expression of the viral H1 phosphatase (Traktman, unpublished). Immunoelectron microscopy has revealed that antibodies to A4 decorate the interior of IV but not the nucleoid, and within MV, appear to localize to a region between the core and membrane. On extraction of purified MV with neutral detergent and DTT, A4 partitions into both the core (insoluble) and the membrane (soluble) fractions. Consistent with these results, anti-A4 stains the surface of isolated cores, leading Roos *et al.* (1996) and Cudmore *et al.* (1996) to conclude that A4 comprises, at least in part, the “spike” or palisade layer of the outer core wall. The A4 protein forms a complex with both the A10 precursor protein, p4a, and the 62-kDa proteolysis product, 4a (Risco *et al.*, 1999). Nevertheless, inducible A10 mutants present a significantly different phenotype than A4 mutants. A10 mutants are defective in the association of viroplasm with crescents that is required to form IV (Section VII.B). In contrast, A4 mutants are defective in formation of MV but form both normal IV and IVN; some abnormal

structures consisting of IV which appear to be surrounded by an additional membrane are also seen (Williams *et al.*, 1999). In this case the authors were able to purify virions from the A4-deficient infections. The mutant particles were seemingly normal in protein content, although they contained slightly more unprocessed A10 protein, consistent with an assembly block at the IV/IVN to MV transition. Thus, A4 appears to be necessary for core assembly during the IV to MV transition.

### 4. A3

The A3L gene encodes the p4b/4b protein, a major component of the virion core that undergoes proteolytic processing during the IV to MV transition. The 73-kDa precursor is cleaved to a mature protein of 60 kDa, called 4b (Fig. 13), which accounts for 11% of the dry mass of the virion (Sarov and Joklik, 1972). Several independent studies localize 4b to the core wall (Ichihashi *et al.*, 1984; Sarov and Joklik, 1972; Wilton *et al.*, 1995). Analysis of temperature-sensitive mutants that encode defective A3 proteins (Cts8, Cts26) has revealed that A3 is essential for the formation of a structurally normal core (Kato *et al.*, 2004). Under nonpermissive conditions, tsA3 mutants form normal IV and IVN but do not form normal MV. Instead, spherical or irregularly shaped particles are formed which accumulate at a distance from viral factories and can be wrapped in Golgi-derived membranes and released by exocytosis, as if they were normal MV (Fig. 11I). The aberrant tsA3 particles can be purified for analysis; they contain the normal complement of virion proteins and DNA; however, their internal structures are malformed relative to wild-type MV. In the tsA3 mutants, mutant 4b protein is made, processed, and packaged in normal amounts in aberrant virions, suggesting that aberrant particles observed in tsA3 mutants result from the inability of the mutant 4b protein to function in formation of a normal core wall. Importantly, the aberrant A3 mutant particles are defective in virion transcription despite containing viral DNA and an active transcriptional apparatus, demonstrating that proper core formation is required for transcription *in situ*.

#### C. Proteolysis of Virion Protein Precursors; Involvement of Genes I7L and G1L

An important role for proteolytic processing in virion maturation is a common theme in numerous systems, and vaccinia is no exception (Hellen and Wimmer, 1992). Early studies revealed that the major structural protein 4a, the product of the A10L gene, is derived from the p4a precursor by proteolytic processing during the late stages of

virion assembly (Katz and Moss, 1970a,b). Subsequent studies have shown that in addition to A10 (p4a/4a), the virion core proteins A3 (p4b/4b), L4 (VP8), A12, and G7, and the membrane protein A17 are also derived via proteolytic processing of precursor polypeptides (Betakova et al., 1999b; Mercer and Traktman, 2005; Rodriguez et al., 1997; Rosel and Moss, 1985; Sarov and Joklik, 1972; Szajner et al., 2003; Takahashi et al., 1994; Vanslyke et al., 1991a,b; Weir and Moss, 1985; Whitehead and Hruby, 1994b; Yang et al., 1988). The processing of each of the proteolyzed virion proteins is understood in significant detail, as summarized in Fig. 13.

Alignment of all of the known cleavage sites for proteolytically processed vaccinia proteins yields the consensus sequence AG↓X, where cleavage occurs between the G and the X (Vanslyke et al., 1991b). A survey of the vaccinia genome reveals 82 potential AG↓X cleavage sites, although it has been shown experimentally that not all of these sites are cleaved (Whitehead and Hruby, 1994b). Nevertheless, given the abundance of potential cleavage sites, it seems likely that additional targets for morphogenesis-associated proteolysis will be discovered.

Vaccinia contains two genes, *I7L* and *G1L*, which encode proteins with significant homology to known proteases. The G1 protein, a component of the virion core, contains the motif HXXEH that is common to a subset of metalloproteases (Ansarah-Sobrinho and Moss, 2004a; Hedengren-Olcott et al., 2004; Whitehead and Hruby, 1994a). In transient infection–transfection assays, the *G1* gene product can cleave the L4 precursor protein in a manner that depends on an intact HXXEH motif. G1-dependent cleavage of L4 does occur at an AG↓X motif, although cleavage occurs at a cryptic AG↓S site rather than the natural AG↓A site (Whitehead and Hruby, 1994a). An inducible recombinant in which expression of G1 can be experimentally regulated has been constructed; repression of G1 blocks the production of infectious virus at the IV to MV transition (see later) (Ansarah-Sobrinho and Moss, 2004a). This phenotype cannot be complemented by expression of a *G1L* allele containing mutations in the HXXEH motif, implying that the predicted metalloprotease activity of G1 is essential for virus growth. However, although normal MV do not form when G1 is repressed, cleavage of the A3, A10, L4, G7, and A17 precursors occurs normally, showing that the G1 protein is not required for their proteolytic maturation. Thus, although G1 is clearly required for virion maturation, its role as a protease remains unresolved.

The I7 protein, which is also a component of the virion core, has homology to known cysteine proteinases encoded by African swine fever virus and yeast (Andres et al., 2001; Kane and Shuman, 1993;

Li and Hochstrasser, 1999). Nonpermissive infections performed with temperature-sensitive mutants or an inducible recombinant affected in the *I7L* gene are characterized by defective processing of the A3, A10, L4, and A17 precursors. Neither this proteolysis defect nor the accompanying morphogenesis arrest at the IV to MV transition can be complemented by mutants containing alterations in residues predicted to comprise the active site of the I7 protease (Ansarah-Sobrinho and Moss, 2004b; Byrd et al., 2002, 2003; Ericsson et al., 1995). Direct biochemical evidence that purified I7 protein can cleave appropriate substrates *in vitro* is still lacking, and it is therefore formally possible that the impact of repression or inactivation of I7 on proteolysis *in vivo* is indirect. Nevertheless, it seems highly probable that the I7 cysteine protease is directly responsible for processing most or all of the viral proteins known to undergo morphogenesis-associated proteolysis.

Proteolysis of the virion core proteins, but not the A17 membrane protein, is coupled to and probably required for maturation of IV/IVN to MV. Rifampicin treatment prevents proteolysis of core proteins but not the A17 membrane protein, suggesting that membrane protein maturation is an early event in virus assembly while core protein proteolysis is a late event (Betakova et al., 1999b; Katz and Moss, 1970a; Rodriguez et al., 1993). Inspection of Table II reveals that where tested, all mutants that are blocked in the IV/IVN to MV transition are also blocked in core protein proteolysis, with two notable exceptions, namely G1 mutants and A3 mutants, discussed later. Conversely, all mutants that accumulate morphologically normal MV show normal proteolysis of core proteins. Thus, there exists a correlation between proteolysis and the IV/IVN to MV transition. The core protein precursors are almost certainly contained within completed IVN prior to the conversion to MV. Because IV and IVN appear to be unstable in the standard hypotonic buffers used for virus purifications, the precise protein composition of immature virions is difficult to assess (Hu et al., 1996; Klemperer et al., 1997; Ravanello and Hruby, 1994b; Vanslyke et al., 1993). Nevertheless, the presence of the A3, A10, L4, and G7 proteins, or more likely their precursors, within IV/IVN has been demonstrated by immunoelectron microscopy (Szajner et al., 2003; Vanslyke and Hruby, 1994). Importantly, mutation of the I7 protease not only prevents proteolysis of core proteins but also prevents formation of MV (Ansarah-Sobrinho and Moss, 2004b; Byrd and Hruby, 2005; Ericsson et al., 1995; Kane and Shuman, 1993) (Fig. 11H). Instead, IV and IVN accumulate, and in addition dense, spherical aberrant particles (not visible in Fig. 11H) are formed (Ansarah-Sobrinho and Moss, 2004b; Ericsson et al., 1995).

The aberrant particles contain viral DNA and are identical to MV in protein composition, with the exception that the normally processed core proteins remain in their precursor form. The aberrant I7 particles also become wrapped to form aberrant WV and EV, suggesting that the membrane has been properly restructured (Section IX.E). In sum, the data indicate that proteolysis of one or more core proteins is required for morphogenesis from IVN to MV, although the presence of the I7 protein itself may also be required for the internal restructuring of the virion. The phenotype of I7 mutants also suggests that core formation and restructuring of the particle surface can occur independently of each other.

The phenotypes of A3 mutants (described previously in Section IX.B.4) and G1 mutants provide interesting exceptions to the correlation between proteolysis of core proteins and the IV/IVN to MV transition. These mutants display a phenotype that is virtually identical to the phenotype of mutants affected in the I7 cysteine proteinase: accumulation of IV and IVN, inhibition of MV formation, and accumulation of dense, spherical, or abnormally shaped particles that are indistinguishable from MV in protein and DNA content and are wrapped to form aberrant WV and released by exocytosis to form aberrant EV (Fig. 11I) (Ansarah-Sobrinho and Moss, 2004a; Hedengren-Olcott *et al.*, 2004; Kato *et al.*, 2004). However, the core proteins contained in the aberrant virions that form when A3 or G1 are mutated have been proteolytically processed to the mature forms. In the case of the A3 mutant, the mutant phenotype likely results directly from an inability of the mutant A3 protein to form a good core wall (Section IX.B.4). The similarity between the I7 and A3 mutant phenotypes therefore suggests that failure to process the core proteins has a significant impact on formation of the core wall. The similarity of the G1 metalloprotease mutant phenotype to the I7 cysteine protease mutant phenotype is provocative, in that it suggests that the G1 protein does indeed act as a protease during infection, targeting substrates whose processing is also critical for core formation and the IV/IVN to MV transition.

### D. Membrane Proteins Involved in the IV to MV Transition

There are a number of additional membrane proteins that are dispensable for formation of the IV membrane but play key roles in the IV→MV transition. How and when these proteins are added to the membrane, and how they assist in genome encapsidation and/or core formation, is a question of significant interest.

RICHARD C. CONDIT ET AL.

1. **A9**

The 12-kDa A9 protein is expressed at late times of infection and incorporated into the virion membrane (Yeh et al., 2000). It is oriented with its C-terminus exposed on the surface of the virion and it appears to be present in both IV and MV (Fig. 12). The predicted transmembrane domain is located in the middle of the protein; although the N-terminus of the protein contains a predicted signal sequence, cotranslational insertion of A9 into membranes has not been documented. An inducible recombinant in which A9 expression is IPTG-dependent has been generated, and A9 has been shown to be essential for virion production. In the absence of A9, IV are formed, as well as aberrant IV that appear to be “empty” or only partially filled with viroplasmic material (Yeh et al., 2000).

2. **L1**

The L1 protein is a myristylated component of the virion membrane; a C-terminal hydrophobic domain is responsible for anchoring the protein in the membrane, although the timing and mode of insertion are not known (Fig. 12) (Franke et al., 1990; Ravanello and Hruby, 1994a). L1 was recognized early on as containing immunodominant epitopes, and antisera to the L1 protein have potent neutralizing activity (Wolffe et al., 1995). The ectodomain of the L1 protein contains three intramolecular disulfide bonds that are formed by the vaccinia-encoded cytoplasmic thiol oxyreductase (Section IX.E.2) (Senkevich et al., 2002a); the crystal structure of the ectodomain of L1 has been solved (Su et al., 2005). An inducible recombinant in which expression of the L1R gene is IPTG-dependent has been generated (Ravanello and Hruby, 1994b). L1 is essential for virion production; in the absence of L1, crescents are numerous and IV do form, but maturation of IV to MV does not occur. This defect can be complemented in trans by expression of wt L1, but expression of a derivative lacking the target site for myristylation, Gly2, fails to complement. Likewise, complementation is lost when the Cys residues implicated in disulfide bond formation are altered (Blouch et al., 2005).

3. **H3**

The *H3L* gene encodes an immunodominant protein which is exposed on the surface of MV (Chertov et al., 1991; Zinoviev et al., 1994). Expressed at late times postinfection, the H3 protein has a C-terminal hydrophobic tail which mediates posttranslational insertion of H3 into membranes; immunoelectron microscopy suggests that
H3 is not present in membrane crescents but associates with virion membranes during IV maturation (Fig. 12) (Section IX.E) (da Fonseca *et al.*, 2000a). Antisera generated against H3 have been shown to neutralize viral infectivity (Lin *et al.*, 2000). A soluble form of H3 binds to heparin sulfate on the cell surface, and indeed this protein competes with virions when present during adsorption (Lin *et al.*, 2000). Together, these data support the conclusion that H3 may mediate initial, low-affinity interactions between MV and the cell surface.

The $H3L$ gene is not essential in tissue culture, since deletion mutants in which $H3L$ has been insertionaly inactivated are viable (da Fonseca *et al.*, 2000b; Lin *et al.*, 2000). To aid in the analysis of the function of H3, an inducible recombinant in which H3 expression is IPTG-dependent has also been constructed (da Fonseca *et al.*, 2000b). Deletion or repression of H3 causes a distinct phenotype: plaque size is diminished, and viral yields are reduced by approximately 10-fold. Electron microscopic analysis revealed that, in the absence of H3, production of MV is reduced, and virosomes, crescents, and IV accumulate to higher than normal levels. The absence of H3 does not prevent the incorporation of L1 (see in an earlier section) or D8 (another membrane protein associated with binding to chondroitin sulfate) into virion membranes. In sum, H3 appears to enhance the efficiency of the IV→MV transition, as well as mediating interactions between virions and heparin sulfate moieties found on the surface of target cells.

### E. Restructuring the Particle Surface

#### 1. Loss of D13

As described previously, IV and IVN are coated on their outer surface with a “spicule layer” composed of a honeycomb lattice of the D13 protein (Section VI.E) (Dales and Mosbach, 1968; Szajner *et al.*, 2005). Electron microscopic observations led Dales and coworkers to conclude that during the transition from IVN to MV, the spicule layer was lost and replaced by a morphologically distinct structure, “surface tubule elements” (Dales and Pogo, 1981). Although low levels of the D13 protein have been found within purified MV by immunoblot analysis and mass spectroscopy (Chung *et al.*, 2006; Miner and Hruby, 1989), this association is probably fortuitous (Franke and Hruby, 1987); immunoelectron microscopy reveals that anti-D13 antibodies decorate IV but not MV within infected cells (Mohandas and Dales, 1995; Sodeik *et al.*, 1994). It is now accepted that, during the transition from
IV to MV, the D13 spicules are indeed lost from the maturing virus particles. Understanding the signal that initiates this disassociation will be of significant interest.

## 2. Disulfide Bond Formation

Numerous vaccinia virion structural proteins contain intra or intermolecular disulfide bonds. These include the core proteins A4, L4, and A10 (4a), and the membrane proteins A14, A17, A27, D8, L1, A28, A21, L5, and H2 (Ichihashi and Oie, 1996; Ichihashi *et al.*, 1984; Locker and Griffiths, 1999; Oie and Ichihashi, 1981; Rodriguez *et al.*, 1987, 1993; Senkevich and Moss, 2005; Senkevich *et al.*, 2000b, 2004a; Townsley *et al.*, 2005a,b; Senkevich, T. G., personal communication; Wolfe *et al.*, 1995). Disulfide bond formation in a subset of the membrane proteins, L1, A28, A21, L5, and H2, is controlled by a virally encoded redox system, and disruption of this redox system disrupts virus morphogenesis, as described in the following paragraphs.

The vaccinia coded redox system was uncovered through investigation of vaccinia genes which are conserved throughout poxviruses and which also contained sequence motifs suggestive of redox capability (Senkevich *et al.*, 2000a,b; White *et al.*, 2000). The complete pathway consists of the E10, A2.5, and G4 proteins, which interact with each other to catalyze a cascade of disulfide oxidation-reduction reactions shown in Fig. 14, sequentially oxidizing and reducing intramolecular disulfide bonds within each enzyme and ultimately resulting in formation of intramolecular disulfide bonds in a number of target proteins (Senkevich *et al.*, 2002a). All three redox enzymes are associated with the virion; E10 and G4 both seem to be concentrated in the virion membrane. The target proteins include L1, F9, A28, A21, L5, and H2 (Senkevich *et al.*, 2000b, 2004a; Townsley *et al.*, 2005a,b). L1 is described in Section IX.D.2, and A28, A21, L5, and H2, all components of

![Figure 14](attachment://image.png)

FIG 14. Vaccinia virus disulfide bond formation pathway. The coupled thiol-disulfide exchange reactions from E10R to the substrate proteins L1R and F9L are depicted. Electron transfers to FAD and oxygen indicated in the dashed rectangle are hypothetical (Senkevich *et al.*, 2002a). Reprinted from Senkevich *et al.* (2002a) with permission.

a membrane-anchored complex controlling virus entry, are discussed in Section IX.G.3. The F9 protein shows significant homology to L1, containing the same six conserved cysteines, although the role of F9 in infection is currently unknown (Senkevich *et al.*, 2000b).

Repression of any of the three enzymes of the redox pathway, E10, A2.5, or G4, interrupts virion morphogenesis at the IVN to MV transition (Ravanello and Hruby, 1994b; Senkevich *et al.*, 2000a, 2002b; White *et al.*, 2000). Specifically, in these mutants, MV are significantly decreased in number or absent from infection while IV accumulate. This does not imply that the redox pathway is active specifically during the IVN to MV transition; on the contrary, the redox substrates L1 and F9 are oxidized when morphogenesis is blocked at earlier stages by repression of the A17 membrane protein or treatment with rifampicin (Senkevich *et al.*, 2000b). However, the fact that redox pathway mutants and L1 mutants (Section IX.D.2) produce the same phenotype suggests that the phenotype of the redox pathway mutants is, at least in part, a manifestation of their inability to oxidize the L1 disulfide bonds, thereby presumably rendering L1 nonfunctional (Blouch *et al.*, 2005).

### 3. Addition of New Surface Proteins

In 1962, in a brief, prescient, and elegant report, Morgan and coworkers showed that when ferritin-labeled antibodies prepared from rabbits immunized with vaccinia virus were used in immunoelectron microscopic examination of vaccinia-infected cells, MV were stained but IV were not stained. The authors concluded “that maturation of vaccinia virus into the stable, infectious form is accompanied not only by an alteration in structure but also by the acquisition of a surface coating of antigen not present on the immature particles” (Morgan *et al.*, 1962). Forty-four years later, we understand with some precision the nature of a few of these modifications. Specifically, immunoelectron microscopy of infected cells demonstrates that antibody against either of the surface proteins A27 (Section X) or H3 shows significantly more staining of MV than IV, suggesting that both proteins are added to particles during the IV to MV transition (da Fonseca *et al.*, 2000a; Sodeik *et al.*, 1995). As described previously (Section IX.D.3), H3 is a transmembrane protein that undergoes posttranslational insertion into virion membranes. In contrast, the peripheral protein A27 is anchored to the viral membrane via its interactions with the A17 transmembrane protein. A27 is not required for MV formation but rather for wrapping of MV with Golgi membranes to form WV. Thus, surface proteins can be incorporated during the later stages of virion

maturation by distinct mechanisms. We speculate that these additions occur after the departure of the D13 lattice from the IV membrane. Other candidates for surface proteins added during the IV to MV transition include the L1 protein (Section IX.D.2). Immunoelectron microscopy of virus-infected cells using anti-L1 antibody shows a higher density of L1 staining in regions containing MV than regions containing IV (Wolffe *et al.*, 1995). As described previously, L1 is required for the IV to MV transition, a process that is also stimulated by H3. It seems likely that additional surface modifications during the IV to MV transition will be revealed in the future.

Dales and coworkers have suggested that the spicule layer surrounding IV, now known to be composed of D13 protein, is replaced by the STEs appearing on MV, raising the question of the relationship between STE and the proteins added late during MV maturation (Essani *et al.*, 1982; Stern and Dales, 1976). Wilton *et al.* (1995) examined the protein composition of isolated STE, and while they did not unambiguously identify each of the STE proteins, the results suggest that STE contain a mixture of proteins that appear in the crescents and MV membrane both early and late during morphogenesis, such as A17 and H3. Thus, while the removal of D13 from IV and addition of several immunodominant MV membrane antigens to the developing MV may restructure the particle surface, it is likely that STE represent a complex of several of the full complement of MV membrane proteins, rather than exclusively proteins added after the removal of D13.

### F. Core Proteins that are not Essential for Virion Morphogenesis but are Required for Transcriptional Competence of the Virion Core

Mutation of the genes encoding certain core proteins results, under nonpermissive conditions, in the formation of MV that are normal in appearance but not infectious. Two of these proteins, the I8 RNA helicase and the H6 topoisomerase, are virion enzymes. The phenotypes of I8 and H6 mutants are similar: permeabilized virions purified from nonpermissive infections display defects in early transcription. These data imply that the RNA helicase and DNA topoisomerase are both required to deal with the topological constraints associated with the virion transcription reaction (da Fonseca and Moss, 2003; Gross and Shuman, 1996). There also exist at least four virion proteins that are not thought to function in transcription per se but rather seem to affect the assembly of a core that is competent for *in situ* transcription. These include L4, L3, H1, and E8, described in more detail in the following sections.

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 105

1. $L4$

The $L4R$ gene encodes a 29-kDa protein that is proteolytically processed to the mature 25-kDa form during the IV to MV transition (Fig. 13). L4 was originally referred to as VP8 and has been shown to have both DNA and RNA binding activity (Bayliss and Smith, 1997; Yang and Bauer, 1988; Yang et al., 1988). An inducible recombinant has been generated in which L4 expression is IPTG-dependent (Wilcock and Smith, 1994, 1996). When L4 is repressed, formation of both IV and MV is observed by electron microscopy. The IV, however, appear slightly abnormal: they possess a space between the viroplasm and the inner surface of the membrane, as if bonding of the viroplasm to the membrane is weaker than normal. L4-deficient MV contain an otherwise normal complement of virion proteins, but they are defective in transcription as judged by both *in vivo* and *in vitro* assays, and hence they are not infectious. Furthermore, when these virions are extracted with deoxycholate to solubilize virion enzymes, more proteins appear to be more easily released than when L4-containing virions are examined in parallel. Thus, the internal structure of MV assembled in the absence of L4 appears to be somewhat fragile. These findings suggest that L4 plays a structural role in the assembly of a transcriptionally-competent core; in this regard, it is worth noting that L4 was originally reported to be encapsidated at ~7000 copies/virion.

2. $L3$

The $L3L$ gene encodes a highly conserved 41-kDa protein that is packaged into virion cores. Repression of L3 expression results in the assembly of MV that are ultrastructurally normal in appearance but are reduced in infectivity (Resch and Moss, 2005). L3-deficient virions contain the normal complement of active transcription enzymes, but they are nevertheless defective for transcription *in situ*. Consistent with these observations, the L3-deficient MV can bind and enter cells, however, only very small amounts of early viral mRNA are synthesized. Thus, similar to L4 (above) and E8 (below), L3 deficiency may produce a subtle defect in MV structure that ultimately impacts on the transcriptional competency of the mutant virions.

3. $H1$

The $H1L$ gene encodes a dual specificity protein phosphatase. This 20-kDa protein is encapsidated at ~200 copies/virion (Guan et al., 1991; Liu et al., 1995). It is known to regulate the phosphorylation status of a number of viral and cellular proteins *in vivo* and has

been shown to dephosphorylate A17, A14, and F17 in vitro. An inducible recombinant has been generated in which H1 expression is TET-dependent. In the absence of H1, F17, A14, and A17 are hyperphosphorylated. Moreover, the virions purified from H1-deficient infections show a phenotype much like that engendered by repression of L4: the cores show a reduced stability and are transcriptionally incompetent. Hence, H1 is not required for morphogenesis per se but rather affects the assembly of a transcriptionally competent core and hence is required for virion infectivity.

### 4. E8

E8 initially drew attention during a search for proteins which might associate with the ER membranes surrounding virus factories early during infection (Doglio *et al.*, 2002; Schramm and Locker, 2005; Tolonen *et al.*, 2001). The E8 amino acid sequence predicts the presence of two possible transmembrane domains, and both immunoelectron microscopy and immunofluorescence suggests an association with viral and cellular membranes during infection. These observations led Locker and coworkers to propose a role for E8 in mediating the assembly of ER-delimited cytoplasmic domains within which viral replication would occur (Schramm and Locker, 2005) (Fig. 5) (Section V). However, other observations seem inconsistent with this model. First, the E8 protein clearly partitions to the virion core on extraction with NP40 and DTT. Second, while Locker and coworkers reported that E8 was expressed early during infection, more recent experiments demonstrate that *E8R* is a late gene, making it unlikely that the protein plays a role early during infection (Kato, Condit, and Moussatche, unpublished; Nichols and Traktman, unpublished). Lastly, infections with a temperature-sensitive mutant containing a lesion in the *E8R* gene result in the assembly of morphologically normal MV, WV, and EV; the MV that form show a defect in transcription *in situ* and a significant reduction in infectivity (Kato, Condit, and Moussatche, unpublished). Thus, in many respects E8 behaves like a structural protein that contributes to the assembly of a transcriptionally active core, similar to L4, L3, or H1.

### G. Membrane Proteins Affecting Virus Binding, Entry, and Fusion

Several additional MV membrane proteins have been identified which, when mutated, result in formation of MV that are morphologically normal in appearance. Ten of these proteins have roles in virus entry and/or fusion as detailed in the following section.

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 107

1. **Entry/Fusion Complex**

   A complex of eight interacting proteins has been described, which is anchored in the virion membrane and whose components appear to mediate fusion of the virion membrane with the plasma membrane during virus entry (Ojeda *et al.*, 2006; Senkevich and Moss, 2005; Senkevich *et al.*, 2004b, 2005; Townsley *et al.*, 2005a,b). The complex contains the proteins A21, L5, A28, H2, G3, G9, A16, and J5. At least four of the proteins contain intramolecular disulfide bonds catalyzed by the virus coded redox system (Section IX.E.2), and three are myristylated. Inducible mutants have been constructed for five of the genes encoding fusion complex proteins, and in all cases the mutants produce MV that are morphologically normal and transcriptionally active and which can undergo wrapping and export from cells as EV. These mutant MV can bind to cells but are defective in penetration. Repression of individual components of the complex does not appear to block the insertion of the remaining proteins into the virion membrane, although it does disrupt their ability to mediate virion entry. In sum, the membrane fusion complex is dispensable for virion morphogenesis but essential for virion entry.

2. **I2**

   The I2L gene encodes a 78-amino acid protein with a hydrophobic C-terminus predicted to serve as a transmembrane anchor. Indeed (Nichols, Stanitsa, Traktman, submitted for publication), I2L is expressed late, behaves as an integral membrane protein, and is encapsidated into the virion membrane. I2 appears to be an essential protein, and an inducible recombinant in which I2 expression is TET dependent has been generated. In the absence of I2, the viral life cycle progresses normally, but the MV that are produced show an ~400-fold reduction in specific infectivity. These virions are transcriptionally competent and can bind to target cells but are defective in fusion-mediated entry. Interactions between I2 and the entry complex described previously have not been investigated.

3. **D8**

   The D8 protein is expressed late and incorporated into the virion membrane; the transmembrane domain is found at the N-terminus of the protein, and the C-terminus is exposed on the surface of the virions (Niles and Seto, 1988). D8 is one of three proteins, including A27 and H3, which bind cell surface glycosaminoglycans and are therefore

implicated in initial events that facilitate the binding of virus to cells (Chung et al., 1998; Hsiao et al., 1998, 1999; Lin et al., 2000). A soluble form of D8 has been shown to bind to chondroitin sulfate on the cell surface and to compete with virions for binding to the cell (Hsiao et al., 1999). Viruses in which D8 has been insertionally inactivated are viable and direct normal morphogenesis; however, the virions that are produced show a 10-fold reduction in specific infectivity due to an impaired ability to bind to target cells.

X. TRANSPORT, OCCLUSION, AND SECONDARY WRAPPING OF MV

As noted previously (Section III), most MV localize in clusters within the cytoplasm that are distal from the virus factories in which IV and IVN are assembled. Treatment of infected cells with drugs that inhibit microtubule formation prevents this characteristic trafficking, implying that MV dissemination is a microtubule-mediated process (Sanderson et al., 2000). The mechanism by which MV interacts with microtubules has been investigated but remains unresolved (Sanderson et al., 2000; Ward, 2005).

While the majority of MV remain intracellular and represent a functional end stage to virus assembly, a fraction of MV may undergo further processing to be either exported as EV or embedded within A-type inclusions (ATIs). Two MV membrane proteins are implicated in these processes, A27 and A26.

A. A27

A27 is unusual among MV membrane proteins in that it does not contain a transmembrane domain. Instead, the A27 protein forms a disulfide-bonded trimer, which is anchored to the MV membrane via a strong interaction with the integral A17 membrane protein (Lai et al., 1990; Rodriguez et al., 1987, 1993). A27 is an immunodominant antigen, and anti-A27 antibodies possess virus neutralizing activity (Rodriguez et al., 1985). As described previously, A27 is added to virus particles late during maturation; it appears on MV but not IV (Section IX.E.3). When A27 is repressed, morphologically normal MV form that are fully infectious, although they are unable to undergo secondary wrapping by Golgi-derived membranes to form WV and, ultimately, to be released by exocytosis as EV (Rodriguez and Smith, 1990b; Ward, 2005). The A27 protein has also been implicated in attachment of virus to cells and in intracellular transport of MV (Chung et al., 1998; Sanderson et al., 2000).

VACCINIA VIRUS: STRUCTURE AND ASSEMBLY 109

B. A26

ATIs are large, cytoplasmic, electron-dense regions within which numerous MV are embedded (McKelvey *et al.*, 2002). Formation of ATIs requires two viral genes, the ATI gene and the $A26L$ gene. The $ATI$ gene encodes the protein that forms the inclusion; the $ATI$ gene is not essential, and indeed the WR strain of vaccinia does not express an intact ATI protein and hence does not form inclusions. The $A26L$ gene, on the other hand, encodes an MV membrane protein, initially named p4c, which controls whether or not MV become embedded in inclusions. A26 is nonessential: if the $p4c$ gene is inactivated, infectious MV, WV, and EV are produced. However, any inclusions that are formed do not contain embedded virus (McKelvey *et al.*, 2002). The A26 protein is present on the surface of MV but not EEV, and it is therefore possible that the presence or absence of p4c in the virion membrane may influence whether MV undergo secondary wrapping to become WV and EV, or whether they remain unwrapped in the cytoplasm (Ulaeto *et al.*, 1996). It is not yet known how A26 associates with the MV membrane; although some computer algorithms have predicted the presence of a transmembrane region, others do not, and the protein does not contain any regions of significant hydrophobicity. However, the A26 protein does contain a C-terminal region that is 44% identical to the region of the A27 protein that interacts with the membrane-bound A17 protein (see earlier). This observation raises the interesting possibility that A26 and A27 may compete for binding to A17 and thus determine the balance between MV that remain unwrapped and those that continue on to form WV and EV.

XI. VIRION PROTEINS OF UNKNOWN FUNCTION

Four additional virion proteins of unknown function have been discussed in the literature: the E11 and A12 core proteins and the I5 and A14.5 membrane proteins (Fig. 12). Little is known about E11 except that temperature-sensitive mutants with lesions in this gene produce noninfectious virions at the nonpermissive temperature (Wang and Shuman, 1996). An inducible recombinant in which expression of A14.5 is IPTG-dependent has been generated; repression of this protein has no impact on the viral life cycle in culture but leads to an attenuated phenotype in mice (Betakova *et al.*, 2000). I5 (VP13) and A12 were identified during proteomic analyses of the vaccinia virion (Chung *et al.*, 2006; Takahashi *et al.*, 1994). No function has been ascribed to

either protein as neither gene has been subjected to genetic analysis, although both genes are highly conserved among poxviral genomes and as such the proteins are likely to play significant roles(s) during some facet of the viral life cycle. Encapsulation of I5 into the virion membrane has been verified by analysis of a virus encoding an epitope-tagged I5 allele at the endogenous locus (Nichols, Stanitsa, and Traktman, unpublished). The A12 core protein undergoes proteolytic maturation during virion morphogenesis (Whitehead and Hruby, 1994b).

XII. SUMMARY AND PERSPECTIVES

Investigation into poxvirus structure and assembly is now at a technical and conceptual crossroads. Identification of proteins in the virion and basic characterization of mutants affecting assembly is approaching saturation. A detailed picture of the contribution of individual genes to the structure and stepwise assembly of the virion is therefore beginning to emerge. The genetic, biochemical and cell biological tools are in place to bring this task to completion. Fascinating and significant questions remain to be answered, including:

A. What are the precise localizations within the virion of each of the encapsidated proteins? What is the composition and structure of the tube-like structure within the core? What proteins make up the individual layers of the core wall? What proteins are contained in the lateral bodies and what are their functions? What are the structural relationships among the proteins in the membrane and how does this contribute to the bilaminar appearance of this structure?

B. What is the initial source of the membrane used to form crescents, how is the single bilayer of the crescents and IV assembled and stabilized in the virus factories, and what are the contributions of cellular components and individual viral proteins to this process?

C. How are viral factories organized so that assembly components are delivered to maturing virions in an organized fashion?

D. What are the details of the molecular interactions that bind crescents to viroplasm, and how does this relate to the ultimate maturation of the virion core?

E. What is the precise mechanism of DNA encapsidation into virions?

F. What triggers the transition from IVN to MV, and how are the various events in this transition co-coordinated, including loss of D13, restructuring of the virion surface, proteolysis of structural proteins,

maturation of the core, and migration of maturing virions from factories to the periphery of the cell?

A thorough understanding of poxvirus morphogenesis will provide invaluable insights into this complex and biomedically significant virus. Moreover, these studies should lead to the elucidation and discovery of cell biological strategies that have deep relevance to such diverse processes as organelle biogenesis, intracellular protein trafficking, and membrane remodeling.

# ACKNOWLEDGMENTS

We thank Bernie Moss and John Heuser for helpful discussions while the manuscript was being written. We thank Bernie Moss and Wen Chang for communicating data in advance of publication. We thank Jason Mercer for help in figure preparation. We thank Michel Moussatche for his expert rendering of the vaccinia virion model. Work described herein was supported by a grant to RCC from the NIH (5R01-AI055560) and by grants to PT from the NIH (9R01 AI 063620-11) and the GLRCE (sub-contract, 1-U54-AI057153).

# REFERENCES

Andres, G., Alejo, A., Simon-Mateo, C., and Salas, M. L. (2001). African swine fever virus protease, a new viral member of the SUMO-1-specific protease family. *J. Biol. Chem.* **276**:780–787.

Ansarah-Sobrinho, C., and Moss, B. (2004a). Vaccinia virus G1 protein, a predicted metalloprotease, is essential for morphogenesis of infectious virions but not for cleavage of major core proteins. *J. Virol.* **78**:6855–6863.

Ansarah-Sobrinho, C., and Moss, B. (2004b). Role of the I7 protein in proteolytic processing of vaccinia virus membrane and core components. *J. Virol.* **78**:6335–6343.

Armstrong, J. A., Metz, D. H., and Young, M. R. (1973). The mode of entry of vaccinia virus into L cells. *J. Gen. Virol.* **21**:533–537.

Baldick, C. J., Jr., and Moss, B. (1987). Resistance of vaccinia virus to rifampicin conferred by a single nucleotide substitution near the predicted NH2 terminus of a gene encoding an Mr 62,000 polypeptide. *Virology* **156**:138–145.

Bayliss, C. D., and Smith, G. L. (1997). Vaccinia virion protein VP8, the 25 kDa product of the L4R gene, binds single-stranded DNA and RNA with similar affinity. *Nucleic Acids Res.* **25**:3984–3990.

Beaud, G., and Beaud, R. (2000). Temperature-dependent phosphorylation state of the H5R protein synthesized at the early stage of infection in cells infected with vaccinia virus ts mutants of the B1R and F10L protein kinases. *Intervirology* **43**:67–70.

Beaud, G., Beaud, R., and Leader, D. P. (1995). Vaccinia virus gene H5R encodes a protein that is phosphorylated by the multisubstrate vaccinia virus B1R protein kinase. *J. Virol.* **69**:1819–1826.

Betakova, T., and Moss, B. (2000). Disulfide bonds and membrane topology of the vaccinia virus A17L envelope protein. *J. Virol.* 74:2438–2442.

Betakova, T., Wolffe, E. J., and Moss, B. (1999a). Membrane topology of the vaccinia virus A17L envelope protein. *Virology* 261:347–356.

Betakova, T., Wolffe, E. J., and Moss, B. (1999b). Regulation of vaccinia virus morphogenesis: Phosphorylation of the A14L and A17L membrane proteins and C-terminal truncation of the A17L protein are dependent on the F10L kinase. *J. Virol.* 73:3534–3543.

Betakova, T., Wolffe, E. J., and Moss, B. (2000). The vaccinia virus A14.5L gene encodes a hydrophobic 53-amino-acid virion membrane protein that enhances virulence in mice and is conserved among vertebrate poxviruses. *J. Virol.* 74:4085–4092.

Black, E. P., Moussatche, N., and Condit, R. C. (1998). Characterization of the interactions among vaccinia virus transcription factors G2R, A18R, and H5R. *Virology* 245:313–322.

Blouch, R. E., Byrd, C. M., and Hruby, D. E. (2005). Importance of disulphide bonds for vaccinia virus L1R protein function. *Virology J.* 2:91.

Brown, N. G., Nick, M. D., Beaud, G., Hardie, G., and Leader, D. P. (2000). Identification of sites phosphorylated by the vaccinia virus B1R kinase in viral protein H5R. *BMC Biochem.* 1:2.

Broyles, S. S. (2003). Vaccinia virus transcription. *J. Gen. Virol.* (on line. 2003).

Byrd, C. M., Bolken, T. C., and Hruby, D. E. (2002). The vaccinia virus I7L gene product is the core protein proteinase. *J. Virol.* 76:8973–8976.

Byrd, C. M., Bolken, T. C., and Hruby, D. E. (2003). Molecular dissection of the vaccinia virus I7L core protein proteinase. *J. Virol.* 77:11279–11283.

Byrd, C. M., and Hruby, D. E. (2005). A conditional-lethal vaccinia virus mutant demonstrates that the I7L gene product is required for virion morphogenesis. *Virology J.* 2:4.

Cairns, J. (1960). The initiation of vaccinia infection. *Virology* 11:603–623.

Carter, G. C., Law, M., Hollinshead, M., and Smith, G. L. (2005). Entry of the vaccinia virus intracellular mature virion and its interactions with glycosaminoglycans. *J. Gen. Virol.* 86:1279–1290.

Cassetti, M. C., Merchlinsky, M., Wolffe, E. J., Weisberg, A. S., and Moss, B. (1998). DNA packaging mutant: Repression of the vaccinia virus A32 gene results in noninfectious, DNA-deficient, spherical, enveloped particles. *J. Virol.* 72:5769–5780.

Castro, A. P., Carvalho, T. M., Moussatche, N., and Damaso, C. R. (2003). Redistribution of cyclophilin A to viral factories during vaccinia virus infection and its incorporation into mature particles. *J. Virol.* 77:9052–9068.

Chertov, O. Y., Telezhinskaya, I. N., Zaitseva, E. V., Golubeva, T. B., Zinov'ev, V. V., Ovechkina, L. G., Mazkova, L. B., and Malygin, E. G. (1991). Amino acid sequence determination of vaccinia virus immunodominant protein p35 and identification of the gene. *Biomed. Sci.* 2:151–154.

Chiu, W. L., and Chang, W. (2002). Vaccinia virus J1R protein: A viral membrane protein that is essential for virion morphogenesis. *J. Virol.* 76:9575–9587.

Chiu, W. L., Szajner, P., Moss, B., and Chang, W. (2005). Effects of a temperature sensitivity mutation in the J1R protein component of a complex required for vaccinia virus assembly. *J. Virol.* 79:8046–8056.

Christen, L., Higman, M. A., and Niles, E. G. (1992). Phenotypic characterization of three temperature-sensitive mutations in the vaccinia virus early gene transcription initiation factor. *J. Gen. Virol.* 73(Pt. 12):3155–3167.

Chung, C. S., Hsiao, J. C., Chang, Y. S., and Chang, W. (1998). A27L protein mediates vaccinia virus interaction with cell surface heparan sulfate. *J. Virol.* 72:1577–1585.

Chung, C. S., Chen, C. H., Ho, M. Y., Hwang, C. Y., Liao, C. L., and Chang, W. (2006). Vaccinia virus proteome: Identification of proteins in vaccinia virus intracellular mature virion particles. *J. Virol.* 80: 2127–2140.

Condit, R. C., and Niles, E. G. (1990). Orthopoxvirus genetics. *Curr. Top. Microbiol. Immunol.* 163:1–39.

Cudmore, S., Blasco, R., Vincentelli, R., Esteban, M., Sodeik, B., Griffiths, G., and Krijnse, L. J. (1996). A vaccinia virus core protein, p39, is membrane associated. *J. Virol.* 70:6909–6921.

Cyrklaff, M., Risco, C., Fernandez, J. J., Jimenez, M. V., Esteban, M., Baumeister, W., and Carrascosa, J. L. (2005). Cryo-electron tomography of vaccinia virus. *Proc. Natl. Acad. Sci. USA* 102:2772–2777.

da Fonseca, F., and Moss, B. (2003). Poxvirus DNA topoisomerase knockout mutant exhibits decreased infectivity associated with reduced early transcription. *Proc. Natl. Acad. Sci. USA* 100:11291–11296.

da Fonseca, F. G., Wolff, E. J., Weisberg, A., and Moss, B. (2000a). Characterization of the vaccinia virus H3L envelope protein: Topology and posttranslational membrane insertion via the C-terminal hydrophobic tail. *J. Virol.* 74:7508–7517.

da Fonseca, F. G., Wolff, E. J., Weisberg, A., and Moss, B. (2000b). Effects of deletion or stringent repression of the H3L envelope gene on vaccinia virus replication. *J. Virol.* 74:7518–7528.

da Fonseca, F. G., Weisberg, A. S., Caeiro, M. F., and Moss, B. (2004). Vaccinia virus mutants with alanine substitutions in the conserved G5R gene fail to initiate morphogenesis at the nonpermissive temperature. *J. Virol.* 78:10238–10248.

Dales, S. (1962). An electron microscope study of the early association between two mammalian viruses and their hosts. *J. Cell Biol.* 13:303–322.

Dales, S. (1963). The uptake and development of vaccinia virus in strain L cells followed with labeled viral deoxyribonucleic acid. *J. Cell Biol.* 18:51–72.

Dales, S., and Kajioka, R. (1964). The cycle of multiplication of vaccinia virus in Earle’s strain L cells. I. Uptake and penetration. *Virology* 24:278–294.

Dales, S., and Mosbach, E. H. (1968). Vaccinia as a model for membrane biogenesis. *Virology* 35:564–583.

Dales, S., and Pogo, B. G. (1981). Biology of poxviruses. *Virol. Monogr.* 18:1–109.

Dales, S., and Siminovitch, L. (1961). The development of vaccinia virus in Earle’s L strain cells as examined by electron microscopy. *J. Biophys. Biochem. Cytol.* 10:475–503.

Dellis, S., Strickland, K. C., McCrary, W. J., Patel, A., Stocum, E., and Wright, C. F. (2004). Protein interactions among the vaccinia virus late transcription factors. *Virology* 329:328–336.

Demasi, J., and Traktman, P. (2000). Clustered charge-to-alanine mutagenesis of the vaccinia virus H5 gene: Isolation of a dominant, temperature-sensitive mutant with a profound defect in morphogenesis. *J. Virol.* 74:2393–2405.

Demasi, J., Du, S., Lennon, D., and Traktman, P. (2001a). Vaccinia virus telomeres: Interaction with the viral I1, I6, and K4 proteins. *J. Virol.* 75:10090–10105.

Demasi, J., Du, S., Lennon, D., and Traktman, P. (2001b). Vaccinia virus telomeres: Interaction with the viral I1, I6, and K4 proteins. *J. Virol.* 75:10090–10105.

Demkowicz, W. E., Maa, J. S., and Esteban, M. (1992). Identification and characterization of vaccinia virus genes encoding proteins that are highly antigenic in animals and are immunodominant in vaccinated humans. *J. Virol.* 66:386–398.

Derrien, M., Punjabi, A., Khanna, M., Grubisha, O., and Traktman, P. (1999). Tyrosine phosphorylation of A17 during vaccinia virus infection: Involvement of the H1 phosphatase and the F10 kinase. *J. Virol.* 73:7287–7296.

Doglio, L., De, M. A., Schleich, S., Roos, N., and Krijnse, L. J. (2002). The vaccinia virus E8R gene product: A viral membrane protein that is made early in infection and packaged into the virions' core. *J. Virol.* 76:9773–9786.

Dubochet, J., Adrian, M., Richter, K., Garces, J., and Wittek, R. (1994). Structure of intracellular mature vaccinia virus observed by cryoelectron microscopy. *J. Virol.* 68:1935–1941.

Dyster, L. M., and Niles, E. G. (1991). Genetic and biochemical characterization of vaccinia virus genes D2L and D3R which encode virion structural proteins. *Virology* 182:455–467.

Easterbrook, K. B. (1966). Controlled degradation of vaccinia virions *in vitro*: An electron microscopic study. *J. Ultrastruct. Res.* 14:484–496.

Ericsson, M., Cudmore, S., Shuman, S., Condit, R. C., Griffiths, G., and Locker, J. K. (1995). Characterization of ts 16, a temperature-sensitive mutant of vaccinia virus. *J. Virol.* 69:7072–7086.

Ericsson, M., Sodeik, B., Locker, J. K., and Griffiths, G. (1997). *In vitro* reconstitution of an intermediate assembly stage of vaccinia virus. *Virology* 235:218–227.

Essani, K., and Dales, S. (1979). Biogenesis of vaccinia: Evidence for more than 100 polypeptides in the virion. *Virology* 95:385–394.

Essani, K., Dugre, R., and Dales, S. (1982). Biogenesis of vaccinia: Involvement of spicules of the envelope during virion assembly examined by means of conditional lethal mutants and serology. *Virology* 118:279–292.

Fenner, F., Wittek, R., and Dumbell, K. R. (1989). “The orthopoxviruses.” Academic Press, San Diego.

Franke, C. A., and Hruby, D. E. (1987). Association of non-viral proteins with recombinant vaccinia virus virions. *Arch. Virol.* 94:347–351.

Franke, C. A., Wilson, E. M., and Hruby, D. E. (1990). Use of a cell-free system to identify the vaccinia virus L1R gene product as the major late myristylated virion protein M25. *J. Virol.* 64:5988–5996.

Gaigg, B., Simbeni, R., Hrastnik, C., Paltauf, F., and Daum, G. (1995). Characterization of a microsomal subfraction associated with mitochondria of the yeast, *Saccharomyces cerevisiae*. Involvement in synthesis and import of phospholipids into mitochondria. *Biochim. Biophys. Acta* 1234:214–220.

Garcia, A. D., and Moss, B. (2001). Repression of vaccinia virus holliday junction resolvase inhibits processing of viral DNA into unit-length genomes. *J. Virol.* 75:6460–6471.

Garcia, A. D., Aravind, L., Koonin, E. V., and Moss, B. (2000). Bacterial-type DNA holliday junction resolvases in eukaryotic viruses. *Proc. Natl. Acad. Sci. USA* 97:8926–8931.

Goebel, S. J., Johnson, G. P., Perkus, M. E., Davis, S. W., Winslow, J. P., and Paoletti, E. (1990). The complete DNA sequence of vaccinia virus. *Virology* 179:247–263.

Gordon, J., Mohandas, A., Wilton, S., and Dales, S. (1991). A prominent antigenic surface polypeptide involved in the biogenesis and function of the vaccinia virus envelope. *Virology* 181:671–686.

Griffiths, G., Roos, N., Schleich, S., and Locker, J. K. (2001a). Structure and assembly of intracellular mature vaccinia virus: Thin-section analyses. *J. Virol.* 75:11056–11070.

Griffiths, G., Wepf, R., Wendt, T., Locker, J. K., Cyrklaff, M., and Roos, N. (2001b). Structure and assembly of intracellular mature vaccinia virus: Isolated-particle analysis. *J. Virol.* 75:11034–11055.

Grimley, P. M., Rosenblum, E. N., Mims, S. J., and Moss, B. (1970). Interruption by Rifampin of an early stage in vaccinia virus morphogenesis: Accumulation of membranes which are precursors of virus envelopes. *J. Virol.* 6:519–533.

Grosenbach, D. W., Hansen, S. G., and Hruby, D. E. (2000). Identification and analysis of vaccinia virus palmitoylproteins. *Virology* **275**:193–206.

Gross, C. H., and Shuman, S. (1996). Vaccinia virions lacking the RNA helicase nucleoside triphosphate phosphohydrolase II are defective in early transcription. *J. Virol.* **70**:8549–8557.

Grubisha, O., and Traktman, P. (2003). Genetic analysis of the vaccinia virus I6 telomere-binding protein uncovers a key role in genome encapsidation. *J. Virol.* **77**:10929–10942.

Guan, K. L., Broyles, S. S., and Dixon, J. E. (1991). A Tyr/Ser protein phosphatase encoded by vaccinia virus. *Nature* **350**:359–362.

Hassett, D. E., and Condit, R. C. (1994). Targeted construction of temperature-sensitive mutations in vaccinia virus by replacing clustered charged residues with alanine. *Proc. Natl. Acad. Sci. USA* **91**:4554–4558.

Hassett, D. E., Lewis, J. I., Xing, X., DeLange, L., and Condit, R. C. (1997). Analysis of a temperature-sensitive vaccinia virus mutant in the viral mRNA capping enzyme isolated by clustered charge-to-alanine mutagenesis and transient dominant selection. *Virology* **238**:391–409.

Hedengren-Olcott, M., Byrd, C. M., Watson, J., and Hruby, D. E. (2004). The vaccinia virus G1L putative metalloproteinase is essential for viral replication *in vivo*. *J. Virol.* **78**:9947–9953.

Heljasvaara, R., Rodriguez, D., Risco, C., Carrascosa, J. L., Esteban, M., and Rodriguez, J. R. (2001). The major core protein P4a (A10L gene) of vaccinia virus is essential for correct assembly of viral DNA into the nucleoprotein complex to form immature viral particles. *J. Virol.* **75**:5778–5795.

Hellen, C. U., and Wimmer, E. (1992). The role of proteolytic processing in the morphogenesis of virus particles. *Experientia* **48**:201–215.

Heuser, J. (2005). Deep-etch EM reveals that the early poxvirus envelope is a single membrane bilayer stabilized by a geodetic “honeycomb” surface coat. *J. Cell Biol.* **169**:269–283.

Hiller, G., Eibl, H., and Weber, K. (1981). Acyl bis(monoacylglycerol)phosphate, assumed to be a marker for lysosomes, is a major phospholipid of vaccinia virions. *Virology* **113**:761–764.

Hollinshead, M., Vanderplasschen, A., Smith, G. L., and Vaux, D. J. (1999). Vaccinia virus intracellular mature virions contain only one lipid membrane. *J. Virol.* **73**:1503–1517.

Holowczak, J. A. (1982). Poxvirus DNA. *Curr. Top. Microbiol. Immunol.* **97**:27–79.

Holowczak, J. A., Thomas, V. I., and Flores, L. (1975). Isolation and characterization of vaccinia virus “nucleoids.” *Virology* **67**:506–519.

Hooda-Dhingra, U., Thompson, C. L., and Condit, R. C. (1989). Detailed phenotypic characterization of five temperature-sensitive mutants in the 22- and 147-kilodalton subunits of vaccinia virus DNA-dependent RNA polymerase. *J. Virol.* **63**:714–729.

Hsiao, J. C., Chung, C. S., and Chang, W. (1998). Cell surface proteoglycans are necessary for A27L protein-mediated cell fusion: Identification of the N-terminal region of A27L protein as the glycosaminoglycan-binding domain. *J. Virol.* **72**:8374–8379.

Hsiao, J. C., Chung, C. S., and Chang, W. (1999). Vaccinia virus envelope D8L protein binds to cell surface chondroitin sulfate and mediates the adsorption of intracellular mature virions to cells. *J. Virol.* **73**:8750–8761.

Hu, X., Carroll, L. J., Wolffe, E. J., and Moss, B. (1996). *De novo* synthesis of the early transcription factor 70-kilodalton subunit is required for morphogenesis of vaccinia virions. *J. Virol.* **70**:7669–7677.

Hu, X., Wolffe, E. J., Weisberg, A. S., Carroll, L. J., and Moss, B. (1998). Repression of the A8L gene, encoding the early transcription factor 82-kilodalton subunit, inhibits morphogenesis of vaccinia virions. *J. Virol.* **72**:104–112.

Husain, M., and Moss, B. (2003). Evidence against an essential role of COPII-mediated cargo transport to the endoplasmic reticulum-Golgi intermediate compartment in the formation of the primary membrane of vaccinia virus. *J. Virol.* **77**:11754–11766.

Ichihashi, Y., and Oie, M. (1996). Neutralizing epitope on penetration protein of vaccinia virus. *Virology* **220**:491–494.

Ichihashi, Y., Oie, M., and Tsuruhara, T. (1984). Location of DNA-binding proteins and disulfide-linked proteins in vaccinia virus structural elements. *J. Virol.* **50**:929–938.

Ishii, K., and Moss, B. (2001). Role of vaccinia virus A20R protein in DNA replication: Construction and characterization of temperature-sensitive mutants. *J. Virol.* **75**:1656–1663.

Ishii, K., and Moss, B. (2002). Mapping interaction sites of the A20R protein component of the vaccinia virus DNA replication complex. *Virology* **303**:232–239.

Jensen, O. N., Houthaeve, T., Shevchenko, A., Cudmore, S., Ashford, T., Mann, M., Griffiths, G., and Krijnse, L. J. (1996). Identification of the major membrane and core proteins of vaccinia virus by two-dimensional electrophoresis. *J. Virol.* **70**:7485–7497.

Johnson, G. P., Goebel, S. J., and Paoletti, E. (1993). An update on the vaccinia virus genome. *Virology* **196**:381–401.

Kane, E. M., and Shuman, S. (1992). Temperature-sensitive mutations in the vaccinia virus H4 gene encoding a component of the virion RNA polymerase. *J. Virol.* **66**:5752–5762.

Kane, E. M., and Shuman, S. (1993). Vaccinia virus morphogenesis is blocked by a temperature-sensitive mutation in the I7 gene that encodes a virion component. *J. Virol.* **67**:2689–2698.

Kao, S. Y., and Bauer, W. R. (1987). Biosynthesis and phosphorylation of vaccinia virus structural protein VP11. *Virology* **159**:399–407.

Kao, S. Y., Ressner, E., Kates, J., and Bauer, W. R. (1981). Purification and characterization of a superhelix binding protein from vaccinia virus. *Virology* **111**:500–508.

Kato, S. E., Strahl, A. L., Moussatche, N., and Condit, R. C. (2004). Temperature-sensitive mutants in the vaccinia virus 4b virion structural protein assemble malformed, transcriptionally inactive intracellular mature virions. *Virology* **330**:127–146.

Katz, E., and Moss, B. (1970a). Formation of a vaccinia virus structural polypeptide from a higher molecular weight precursor: Inhibition by rifampicin. *Proc. Natl. Acad. Sci. USA* **66**:677–684.

Katz, E., and Moss, B. (1970b). Vaccinia virus structural polypeptide derived from a high-molecular-weight precursor: Formation and integration into virus particles. *J. Virol.* **6**:717–726.

Klemperer, N., Ward, J., Evans, E., and Traktman, P. (1997). The vaccinia virus I1 protein is essential for the assembly of mature virions. *J. Virol.* **71**:9285–9294.

Koonin, E. V., Senkevich, T. G., and Chernos, V. I. (1993). Gene A32 product of vaccinia virus may be an ATPase involved in viral DNA packaging as indicated by sequence comparisons with other putative viral ATPases. *Virus Genes* **7**:89–94.

Kovacs, G. R., and Moss, B. (1996). The vaccinia virus H5R gene encodes late gene transcription factor 4: Purification, cloning, and overexpression. *J. Virol.* **70**:6796–6802.

Krijnse-Locker, J., Schleich, S., Rodriguez, D., Goud, B., Snijder, E. J., and Griffiths, G. (1996). The role of a 21-kDa viral membrane protein in the assembly of vaccinia virus from the intermediate compartment. *J. Biol. Chem.* **271**:14950–14958.

Lackner, C. A., D'Costa, S. M., Buck, C., and Condit, R. C. (2003). Complementation analysis of the Dales collection of vaccinia virus temperature-sensitive mutants. *Virology* 305:240–259.

Lai, C. F., Gong, S. C., and Esteban, M. (1990). Structural and functional properties of the 14-kDa envelope protein of vaccinia virus synthesized in *Escherichia coli*. *J. Biol. Chem.* 265:22174–22180.

Lanzer, W., and Holowczak, J. A. (1975). Polyamines in vaccinia virions and polypeptides released from viral cores by acid extraction. *J. Virol.* 16:1254–1264.

Law, M., Carter, G. C., Roberts, K. L., Hollinshead, M., and Smith, G. L. (2006). Ligand-induced and nonfusogenic dissolution of a viral membrane. *Proc. Natl. Acad. Sci.* 103:5989–5994.

Lefkowitz, E. J., Upton, C., Changayil, S. S., Buck, C., Traktman, P., and Buller, R. M. (2005). Poxvirus bioinformatics resource center: A comprehensive poxviridae informational and analytical resource. *Nucleic Acids Res.* 33:D311–D316.

Li, S. J., and Hochstrasser, M. (1999). A new protease required for cell-cycle progression in yeast. *Nature* 398:246–251.

Lin, C. L., Chung, C. S., Heine, H. G., and Chang, W. (2000). Vaccinia virus envelope H3L protein binds to cell surface heparan sulfate and is important for intracellular mature virion morphogenesis and virus infection *in vitro* and *in vivo*. *J. Virol.* 74:3353–3365.

Lin, S., and Broyles, S. S. (1994). Vaccinia protein kinase 2: A second essential serine/threonine protein kinase encoded by vaccinia virus. *Proc. Natl. Acad. Sci. USA* 91:7653–7657.

Liu, K., Lemon, B., and Traktman, P. (1995). The dual-specificity phosphatase encoded by vaccinia virus, VH1, is essential for viral transcription *in vivo* and *in vitro*. *J. Virol.* 69:7823–7834.

Locker, J. K., and Griffiths, G. (1999). An unconventional role for cytoplasmic disulfide bonds in vaccinia virus proteins. *J. Cell Biol.* 144:267–279.

Locker, J. K., Kuehn, A., Schleich, S., Rutter, G., Hohenberg, H., Wepf, R., and Griffiths, G. (2000). Entry of the two infectious forms of vaccinia virus at the plasma membrane is signaling-dependent for the IMV but not the EEV. *Mol. Biol. Cell* 11:2497–2511.

Maa, J. S., and Esteban, M. (1987). Structural and functional studies of a 39,000-Mr immunodominant protein of vaccinia virus. *J. Virol.* 61:3910–3919.

Malkin, A. J., McPherson, A., and Gershon, P. D. (2003). Structure of intracellular mature vaccinia virus visualized by in situ atomic force microscopy. *J. Virol.* 77:6332–6340.

Martin, S. A., Paoletti, E., and Moss, B. (1975). Purification of mRNA guanylyltransferase and mRNA (guanine-7-) methyltransferase from vaccinia virions. *J. Biol. Chem.* 250:9322–9329.

McCraith, S., Holtzman, T., Moss, B., and Fields, S. (2000). Genome-wide analysis of vaccinia virus protein-protein interactions. *Proc. Natl. Acad. Sci. USA* 97:4879–4884.

McKelvey, T. A., Andrews, S. C., Miller, S. E., Ray, C. A., and Pickup, D. J. (2002). Identification of the orthopoxvirus p4c gene, which encodes a structural protein that directs intracellular mature virus particles into A-type inclusions. *J. Virol.* 76:11216–11225.

Medzon, E. L., and Bauer, H. (1970). Structural features of vaccinia virus revealed by negative staining, sectioning, and freeze-etching. *Virology* 40:860–867.

Mercer, J., and Traktman, P. (2003). Investigation of structural and functional motifs within the vaccinia virus A14 phosphoprotein, an essential component of the virion membrane. *J. Virol.* 77:8857–8871.

Mercer, J., and Traktman, P. (2005). Genetic and cell biological characterization of the vaccinia virus A30 and G7 phosphoproteins. *J. Virol.* 79:7146–7161.

Miner, J. N., and Hruby, D. E. (1989). Rifampicin prevents virosome localization of L65, an essential vaccinia virus polypeptide. *Virology* 170:227–237.

Mohandas, A. R., and Dales, S. (1995). Involvement of spicules in the formation of vaccinia virus envelopes elucidated by a conditional lethal mutant. *Virology* 214:494–502.

Morgan, C. (1976a). The insertion of DNA into vaccinia virus. *Science* 193:591–592.

Morgan, C. (1976b). Vaccinia virus reexamined: Development and release. *Virology* 73:43–58.

Morgan, C., Ellison, S. A., Rose, H. M., and Moore, D. H. (1955). Serial sections of vaccinia virus examined at one stage of development in the electron microscope. *Exp. Cell Res.* 9:572–578.

Morgan, C., Rose, H. M., and Rifkind, R. A. (1962). Use of ferritin-conjugated antibodies in electron microscopic studies of influenza and vaccinia viruses. *Cold Spring Harb. Symp. Quant. Biol.* 27:57–65.

Morgan, J. R., Cohen, L. K., and Roberts, B. E. (1984). Identification of the DNA sequences encoding the large subunit of the mRNA-capping enzyme of vaccinia virus. *J. Virol.* 52:206–214.

Moss, B. (2001). Poxviridae: The viruses and their replication. In “Fields Virology” (D. M. Knipe and P. M. Howley, eds.), pp. 2849–2884. Lippincott Williams & Wilkins, Philadelphia.

Moss, B. (2006). Poxvirus entry and membrane fusion. *Virology* 344:48–54.

Moss, B., Katz, E., and Rosenblum, E. N. (1969a). Vaccinia virus directed RNA and protein synthesis in the presence of rifampicin. *Biochem. Biophys. Res. Commun.* 36:858–865.

Moss, B., Rosenblum, E. N., Katz, E., and Grimley, P. M. (1969b). Rifampicin: A specific inhibitor of vaccinia virus assembly. *Nature* 224:1280–1284.

Nagayama, A., Pogo, B. G., and Dales, S. (1970). Biogenesis of vaccinia: Separation of early stages from maturation by means of rifampicin. *Virology* 40:1039–1051.

Nermut, M. V. (1973). Freeze-drying and freeze-etching of viruses. In “Freeze-Etching Techniques and Applications” (E. L. Benedetti and P. Favard, eds.), pp. 135–150. Societe Francaise de Microscopie Electronique, Paris.

Niles, E. G., and Seto, J. (1988). Vaccinia virus gene D8 encodes a virion transmembrane protein. *J. Virol.* 62:3772–3778.

Niles, E. G., Lee-Chen, G. J., Shuman, S., Moss, B., and Broyles, S. S. (1989). Vaccinia virus gene D12L encodes the small subunit of the viral mRNA capping enzyme. *Virology* 172:513–522.

Nowakowski, M., Bauer, W., and Kates, J. (1978a). Characterization of a DNA-binding phosphoprotein from vaccinia virus replication complex. *Virology* 86:217–225.

Nowakowski, M., Kates, J., and Bauer, W. (1978b). Isolation of two DNA-binding proteins from the intracellular replication complex of vaccinia virus. *Virology* 84:260–267.

Noyes, W. F. (1962a). Further studies on the structure of vaccinia virus. *Virology* 18:511–516.

Noyes, W. F. (1962b). The surface fine structure of vaccinia virus. *Virology* 17:282–287.

Oie, M., and Ichihashi, Y. (1981). Characterization of vaccinia polypeptides. *Virology* 113:263–276.

Ojeda, S., Senkevich, T. G., and Moss, B. (2006). Entry of vaccinia virus and cell-cell fusion require a highly conserved cysteine-rich membrane protein encoded by the A16L gene. *J. Virol.* 80:51–61.

Palade, G. E. (1983). Membrane biogenesis: An overview. *Methods Enzymol.* 96:xxix–xxLv.

Pedersen, K., Snijder, E. J., Schleich, S., Roos, N., Griffiths, G., and Locker, J. K. (2000). Characterization of vaccinia virus intracellular cores: Implications for viral uncoating and core structure. *J. Virol.* 74:3525–3536.

Peters, D., and Mueller, G. (1963). The fine structure of the DNA-containing core of vaccinia virus. *Virology* 21:267–269.

Pichler, H., Gaigg, B., Hrastnik, C., Achleitner, G., Kohlwein, S. D., Zellnig, G., Perktold, A., and Daum, G. (2001). A subfraction of the yeast endoplasmic reticulum associates with the plasma membrane and has a high capacity to synthesize lipids. *Eur. J. Biochem.* 268:2351–2361.

Pogo, B. G., and Dales, S. (1971). Biogenesis of vaccinia: Separation of early stages from maturation by means of hydroxyurea. *Virology* 43:144–151.

Punjabi, A., and Traktman, P. (2005). Cell biological and functional characterization of the vaccinia virus f10 kinase: Implications for the mechanism of virion morphogenesis. *J. Virol.* 79:2171–2190.

Punjabi, A., Boyle, K., DeMasi, J., Grubisha, O., Unger, B., Khanna, M., and Traktman, P. (2001). Clustered charge-to-alanine mutagenesis of the vaccinia virus A20 gene: Temperature-sensitive mutants have a DNA-minus phenotype and are defective in the production of processive DNA polymerase activity. *J. Virol.* 75:12308–12318.

Ravanello, M. P., and Hruby, D. E. (1994a). Characterization of the vaccinia virus L1R myristylprotein as a component of the intracellular virion envelope. *J. Gen. Virol.* 75 (Pt. 6):1479–1483.

Ravanello, M. P., and Hruby, D. E. (1994b). Conditional lethal expression of the vaccinia virus L1R myristylated protein reveals a role in virion assembly. *J. Virol.* 68:6401–6410.

Reggiori, F., and Klionsky, D. J. (2005). Autophagosomes: Biogenesis from scratch? *Curr. Opin. Cell Biol.* 17:415–422.

Resch, W., and Moss, B. (2005). The conserved poxvirus L3 virion protein is required for transcription of vaccinia virus early genes. *J. Virol.* 79:14719–14729.

Resch, W., Weisberg, A. S., and Moss, B. (2005). Vaccinia virus nonstructural protein encoded by the A11R gene is required for formation of the virion membrane. *J. Virol.* 79:6598–6609.

Risco, C., Rodriguez, J. R., Demkowicz, W., Heljasvaara, R., Carrascosa, J. L., Esteban, M., and Rodriguez, D. (1999). The vaccinia virus 39-kDa protein forms a stable complex with the p4a/4a major core protein early in morphogenesis. *Virology* 265:375–386.

Risco, C., Rodriguez, J. R., Lopez-Iglesias, C., Carrascosa, J. L., Esteban, M., and Rodriguez, D. (2002). Endoplasmic reticulum-Golgi intermediate compartment membranes and vimentin filaments participate in vaccinia virus assembly. *J. Virol.* 76:1839–1855.

Rodriguez, D., Rodriguez, J. R., and Esteban, M. (1993). The vaccinia virus 14-kilodalton fusion protein forms a stable complex with the processed protein encoded by the vaccinia virus A17L gene. *J. Virol.* 67:3435–3440.

Rodriguez, D., Esteban, M., and Rodriguez, J. R. (1995). Vaccinia virus A17L gene product is essential for an early step in virion morphogenesis. *J. Virol.* 69:4640–4648.

Rodriguez, D., Risco, C., Rodriguez, J. R., Carrascosa, J. L., and Esteban, M. (1996). Inducible expression of the vaccinia virus A17L gene provides a synchronized system to monitor sorting of viral proteins during morphogenesis. *J. Virol.* 70:7641–7653.

Rodriguez, D., Barcena, M., Mobius, W., Schleich, S., Esteban, M., Geerts, W. J. C., Koster, A. J., Griffiths, G., and Locker, J. K. (2006). A vaccinia virus lacking A10L: Viral core proteins accumulate on structures derived from the endoplasmic reticulum. *Cellular Microbiology* 8:427–437.

Rodriguez, J. F., and Smith, G. L. (1990a). Inducible gene expression from vaccinia virus vectors. *Virology* **177**:239–250.

Rodriguez, J. F., and Smith, G. L. (1990b). IPTG-dependent vaccinia virus: Identification of a virus protein enabling virion envelopment by Golgi membrane and egress. *Nucleic Acids Res.* **18**:5347–5351.

Rodriguez, J. F., Janeczko, R., and Esteban, M. (1985). Isolation and characterization of neutralizing monoclonal antibodies to vaccinia virus. *J. Virol.* **56**:482–488.

Rodriguez, J. F., Paez, E., and Esteban, M. (1987). A 14,000-Mr envelope protein of vaccinia virus is involved in cell fusion and forms covalently linked trimers. *J. Virol.* **61**:395–404.

Rodriguez, J. R., Risco, C., Carrascosa, J. L., Esteban, M., and Rodriguez, D. (1997). Characterization of early stages in vaccinia virus membrane biogenesis: Implications of the 21-kilodalton protein and a newly identified 15-kilodalton envelope protein. *J. Virol.* **71**:1821–1833.

Rodriguez, J. R., Risco, C., Carrascosa, J. L., Esteban, M., and Rodriguez, D. (1998). Vaccinia virus 15-kilodalton (A14L) protein is essential for assembly and attachment of viral crescents to virosomes. *J. Virol.* **72**:1287–1296.

Roos, N., Cyrklaff, M., Cudmore, S., Blasco, R., Krijnse-Locker, J., and Griffiths, G. (1996). A novel immunogold cryoelectron microscopic approach to investigate the structure of the intracellular and extracellular forms of vaccinia virus. *EMBO J.* **15**:2343–2355.

Rosel, J., and Moss, B. (1985). Transcriptional and translational mapping and nucleotide sequence analysis of a vaccinia virus gene encoding the precursor of the major core polypeptide 4b. *J. Virol.* **56**:830–838.

Salmons, T., Kuhn, A., Wylie, F., Schleich, S., Rodriguez, J. R., Rodriguez, D., Esteban, M., Griffiths, G., and Locker, J. K. (1997). Vaccinia virus membrane proteins p8 and p16 are cotranslationally inserted into the rough endoplasmic reticulum and retained in the intermediate compartment. *J. Virol.* **71**:7404–7420.

Saminathan, M., Thomas, T., Shirahata, A., Pillai, C. K., and Thomas, T. J. (2002). Polyamine structural effects on the induction and stabilization of liquid crystalline DNA: Potential applications to DNA packaging, gene therapy and polyamine therapeutics. *Nucleic Acids Res.* **30**:3722–3731.

Sanderson, C. M., Hollinshead, M., and Smith, G. L. (2000). The vaccinia virus A27L protein is needed for the microtubule-dependent transport of intracellular mature virus particles. *J. Gen. Virol.* **81**(Pt. 1):47–58.

Sarov, I., and Joklik, W. K. (1972). Studies on the nature and location of the capsid polypeptides of vaccinia virions. *Virology* **50**:579–592.

Schramm, B., and Locker, J. K. (2005). Cytoplasmic organization of POXvirus DNA replication. *Traffic* **6**:839–846.

Senkevich, T. G., and Moss, B. (2005). Vaccinia virus H2 protein is an essential component involved in virus entry and cell-cell fusion. *J. Virol.* **79**:4744–4754.

Senkevich, T. G., Weisberg, A. S., and Moss, B. (2000a). Vaccinia virus E10R protein is associated with the membranes of intracellular mature virions and has a role in morphogenesis. *Virology* **278**:244–252.

Senkevich, T. G., White, C. L., Koonin, E. V., and Moss, B. (2000b). A viral member of the ERV1/ALR protein family participates in a cytoplasmic pathway of disulfide bond formation. *Proc. Natl. Acad. Sci. USA* **97**:12068–12073.

Senkevich, T. G., White, C. L., Koonin, E. V., and Moss, B. (2002a). Complete pathway for protein disulfide bond formation encoded by poxviruses. *Proc. Natl. Acad. Sci. USA* **99**:6667–6672.

Senkevich, T. G., White, C. L., Weisberg, A., Granek, J. A., Wolff, E. J., Koonin, E. V., and Moss, B. (2002b). Expression of the vaccinia virus A2.5L redox protein is required for virion morphogenesis. *Virology* 300:296–303.

Senkevich, T. G., Ward, B. M., and Moss, B. (2004a). Vaccinia virus A28L gene encodes an essential protein component of the virion membrane with intramolecular disulfide bonds formed by the viral cytoplasmic redox pathway. *J. Virol.* 78:2348–2356.

Senkevich, T. G., Ward, B. M., and Moss, B. (2004b). Vaccinia virus entry into cells is dependent on a virion surface protein encoded by the A28L gene. *J. Virol.* 78:2357–2366.

Senkevich, T. G., Ojeda, S., Townsley, A. C., Nelson, G. E., and Moss, B. (2005). Poxvirus multiprotein entry-fusion complex. *Proc. Natl. Acad. Sci. USA* 102:18572–18577.

Smith, G. L., Vanderplasschen, A., and Law, M. (2002). The formation and function of extracellular enveloped vaccinia virus. *J. Gen. Virol.* 83:2915–2931.

Sodeik, B., and Krijnse-Locker, J. (2002). Assembly of vaccinia virus revisited: De novo membrane synthesis or acquisition from the host? *Trends Microbiol.* 10:15–24.

Sodeik, B., Doms, R. W., Ericsson, M., Hiller, G., Machamer, C. E., van’t, G., vanMeer, H. W., Moss, B., and Griffiths, G. (1993). Assembly of vaccinia virus: Role of the intermediate compartment between the endoplasmic reticulum and the Golgi stacks. *J. Cell Biol.* 121:521–541.

Sodeik, B., Griffiths, G., Ericsson, M., Moss, B., and Doms, R. W. (1994). Assembly of vaccinia virus: Effects of rifampin on the intracellular distribution of viral protein p65. *J. Virol.* 68:1103–1114.

Sodeik, B., Cudmore, S., Ericsson, M., Esteban, M., Niles, E. G., and Griffiths, G. (1995). Assembly of vaccinia virus: Incorporation of p14 and p32 into the membrane of the intracellular mature virus. *J. Virol.* 69:3560–3574.

Soloski, M., and Holowczak, J. A. (1980). Preparation of subviral particles from vaccinia virions irradiated with ultraviolet light. *J. Virol. Methods* 1:185–195.

Soloski, M. J., and Holowczak, J. A. (1981). Characterization of supercoiled nucleoprotein complexes released from detergent-treated vaccinia virions. *J. Virol.* 37:770–783.

Soloski, M. J., Cabrera, C. V., Esteban, M., and Holowczak, J. A. (1979). Studies concerning the structure and organization of the vaccinia virus nucleoid. I. Isolation and characterization of subviral particles prepared by treating virions with guanidine-HCL, nonidet-P40, and 2-mercaptoethanol. *Virology* 99:209–217.

Stern, W., and Dales, S. (1974). Biogenesis of vaccinia: Concerning the origin of the envelope phospholipids. *Virology* 62:293–306.

Stern, W., and Dales, S. (1976). Biogenesis of vaccinia: Isolation and characterization of a surface component that elicits antibody suppressing infectivity and cell-cell fusion. *Virology* 75:232–241.

Stern, W., Pogo, B. G., and Dales, S. (1977). Biogenesis of poxviruses: Analysis of the morphogenetic sequence using a conditional lethal mutant defective in envelope self-assembly. *Proc. Natl. Acad. Sci. USA* 74:2162–2166.

Su, H. P., Garman, S. C., Allison, T. J., Fogg, C., Moss, B., and Garboczi, D. N. (2005). The 1.51-angstrom structure of the poxvirus L1 protein, a target of potent neutralizing antibodies. *Proc. Natl. Acad. Sci. USA* 102:4240–4245.

Szajner, P., Weisberg, A. S., and Moss, B. (2001a). Unique temperature-sensitive defect in vaccinia virus morphogenesis maps to a single nucleotide substitution in the A30L gene. *J. Virol.* 75:11222–11226.

Szajner, P., Weisberg, A. S., Wolff, E. J., and Moss, B. (2001b). Vaccinia virus a30l protein is required for association of viral membranes with dense viroplasm to form immature virions. *J. Virol.* 75:5752–5761.

Szajner, P., Jaffe, H., Weisberg, A. S., and Moss, B. (2003). Vaccinia virus G7L protein Interacts with the A30L protein and is required for association of viral membranes with dense viroplasm to form immature virions. *J. Virol.* **77**:3418–3429.

Szajner, P., Jaffe, H., Weisberg, A. S., and Moss, B. (2004a). A complex of seven vaccinia virus proteins conserved in all chordopoxviruses is required for the association of membranes and viroplasm to form immature virions. *Virology* **330**:447–459.

Szajner, P., Weisberg, A. S., and Moss, B. (2004b). Evidence for an essential catalytic role of the F10 protein kinase in vaccinia virus morphogenesis. *J. Virol.* **78**:257–265.

Szajner, P., Weisberg, A. S., and Moss, B. (2004c). Physical and functional interactions between vaccinia virus F10 protein kinase and virion assembly proteins A30 and G7. *J. Virol.* **78**:266–274.

Szajner, P., Weisberg, A. S., Lebowitz, J., Heuser, J., and Moss, B. (2005). External scaffold of spherical immature poxvirus particles is made of protein trimers forming a honeycomb lattice. *J. Cell Biol.* **170**:971–981.

Takahashi, T., Oie, M., and Ichihashi, Y. (1994). N-terminal amino acid sequences of vaccinia virus structural proteins. *Virology* **202**:844–852.

Tartaglia, J., and Paoletti, E. (1985). Physical mapping and DNA sequence analysis of the rifampicin resistance locus in vaccinia virus. *Virology* **147**:394–404.

Tolonen, N., Doglio, L., Schleich, S., and Krijnse, L. J. (2001). Vaccinia virus DNA replication occurs in endoplasmic reticulum-enclosed cytoplasmic mini-nuclei. *Mol. Cell* **12**:2031–2046.

Townsley, A. C., Senkevich, T. G., and Moss, B. (2005a). The product of the vaccinia virus L5R gene is a fourth membrane protein encoded by all poxviruses that is required for cell entry and cell-cell fusion. *J. Virol.* **79**:10988–10998.

Townsley, A. C., Senkevich, T. G., and Moss, B. (2005b). Vaccinia virus a21 virion membrane protein is required for cell entry and fusion. *J. Virol.* **79**:9458–9469.

Traktman, P., Caligiuri, A., Jesty, S. A., Liu, K., and Sankar, U. (1995). Temperature-sensitive mutants with lesions in the vaccinia virus F10 kinase undergo arrest at the earliest stage of virion morphogenesis. *J. Virol.* **69**:6581–6587.

Traktman, P., Liu, K., DeMasi, J., Rollins, R., Jesty, S., and Unger, B. (2000). Elucidating the essential role of the A14 phosphoprotein in vaccinia virus morphogenesis: Construction and characterization of a tetracycline-inducible recombinant. *J. Virol.* **74**:3682–3695.

Ulaeto, D., Grosenbach, D., and Hruby, D. E. (1995). Brefeldin A inhibits vaccinia virus envelope but does not prevent normal processing and localization of the putative envelope receptor P37. *J. Gen. Virol.* **76**(Pt. 1):103–111.

Ulaeto, D., Grosenbach, D., and Hruby, D. E. (1996). The vaccinia virus 4c and A-type inclusion proteins are specific markers for the intracellular mature virus particle. *J. Virol.* **70**:3372–3377.

Unger, B., and Traktman, P. (2004). Vaccinia virus morphogenesis: A13 phosphoprotein is required for assembly of mature virions. *J. Virol.* **78**:8885–8901.

Upton, C., Slack, S., Hunter, A. L., Ehlers, A., and Roper, R. L. (2003). Poxvirus orthologous clusters: Toward defining the minimum essential poxvirus genome. *J. Virol.* **77**:7590–7600.

Vanslyke, J. K., and Hruby, D. E. (1994). Immunolocalization of vaccinia virus structural proteins during virion formation. *Virology* **198**:624–635.

Vanslyke, J. K., Franke, C. A., and Hruby, D. E. (1991a). Proteolytic maturation of vaccinia virus core proteins: Identification of a conserved motif at the N termini of the 4b and 25K virion proteins. *J. Gen. Virol.* **72**(Pt. 2):411–416.

Vanslyke, J. K., Whitehead, S. S., Wilson, E. M., and Hruby, D. E. (1991b). The multistep proteolytic maturation pathway utilized by vaccinia virus P4a protein: A degenerate conserved cleavage motif within core proteins. *Virology* **183**:467–478.

Vanslyke, J. K., Lee, P., Wilson, E. M., and Hruby, D. E. (1993). Isolation and analysis of vaccinia virus previrions. *Virus Genes* **7**:311–324.

Wallengren, K., Risco, C., Krijnse-Locker, J., Esteban, M., and Rodriguez, D. (2001). The A17L gene product of vaccinia virus is exposed on the surface of IMV. *Virology* **290**:143–152.

Wang, S., and Shuman, S. (1995). Vaccinia virus morphogenesis is blocked by temperature-sensitive mutations in the F10 gene, which encodes protein kinase 2. *J. Virol.* **69**:6376–6388.

Wang, S. P., and Shuman, S. (1996). A temperature-sensitive mutation of the vaccinia virus E11 gene encoding a 15-kDa virion component. *Virology* **216**:252–257.

Ward, B. M. (2005). Visualization and characterization of the intracellular movement of vaccinia virus intracellular mature virions. *J. Virol.* **79**:4755–4763.

Webb, J. H., Mayer, R. J., and Dixon, L. K. (1999). A lipid modified ubiquitin is packaged into particles of several enveloped viruses. *FEBS Lett.* **444**:136–139.

Weir, J. P., and Moss, B. (1985). Use of a bacterial expression vector to identify the gene encoding a major core protein of vaccinia virus. *J. Virol.* **56**:534–540.

Westwood, J. C., Harris, W. J., Zwartouw, H. T., Titmuss, D. H., and Appleyard, G. (1964). Studies on the structure of vaccinia virus. *J. Gen. Microbiol.* **34**:67–78.

White, C. L., Weisberg, A. S., and Moss, B. (2000). A glutaredoxin, encoded by the G4L gene of vaccinia virus, is essential for virion morphogenesis. *J. Virol.* **74**:9175–9183.

Whitehead, S. S., and Hruby, D. E. (1994a). A transcriptionally controlled trans-processing assay: Putative identification of a vaccinia virus-encoded proteinase which cleaves precursor protein P25K. *J. Virol.* **68**:7603–7608.

Whitehead, S. S., and Hruby, D. E. (1994b). Differential utilization of a conserved motif for the proteolytic maturation of vaccinia virus proteins. *Virology* **200**:154–161.

Wilcock, D., and Smith, G. L. (1994). Vaccinia virus core protein VP8 is required for virus infectivity, but not for core protein processing or for INV and EEV formation. *Virology* **202**:294–304.

Wilcock, D., and Smith, G. L. (1996). Vaccinia virions lacking core protein VP8 are deficient in early transcription. *J. Virol.* **70**:934–943.

Williams, O., Wolff, E. J., Weisberg, A. S., and Merchlinsky, M. (1999). Vaccinia virus WR gene A5L is required for morphogenesis of mature virions. *J. Virol.* **73**:4590–4599.

Wilton, S., Mohandas, A. R., and Dales, S. (1995). Organization of the vaccinia envelope and relationship to the structure of intracellular mature virions. *Virolology* **214**:503–511.

Wittek, R., Richner, B., and Hiller, G. (1984). Mapping of the genes coding for the two major vaccinia virus core polypeptides. *Nucleic Acids Res.* **12**:4835–4848.

Wolffe, E. J., Vijaya, S., and Moss, B. (1995). A myristylated membrane protein encoded by the vaccinia virus L1R open reading frame is the target of potent neutralizing monoclonal antibodies. *Virology* **211**:53–63.

Wolffe, E. J., Moore, D. M., Peters, P. J., and Moss, B. (1996). Vaccinia virus A17L open reading frame encodes an essential component of nascent viral membranes that is required to initiate morphogenesis. *J. Virol.* **70**:2797–2808.

Yang, W. P., and Bauer, W. R. (1988). Purification and characterization of vaccinia virus structural protein VP8. *Virology* **167**:578–584.

Yang, W. P., Kao, S. Y., and Bauer, W. R. (1988). Biosynthesis and post-translational cleavage of vaccinia virus structural protein VP8. *Virology* 167:585–590.

Yeh, W. W., Moss, B., and Wolff, E. J. (2000). The vaccinia virus A9L gene encodes a membrane protein required for an early step in virion morphogenesis. *J. Virol.* 74:9701–9711.

Zhang, Y., and Moss, B. (1992). Immature viral envelope formation is interrupted at the same stage by lac operator-mediated repression of the vaccinia virus D13L gene and by the drug rifampicin. *Virology* 187:643–653.

Zhang, Y. F., and Moss, B. (1991a). Inducer-dependent conditional-lethal mutant animal viruses. *Proc. Natl. Acad. Sci. USA* 88:1511–1515.

Zhang, Y. F., and Moss, B. (1991b). Vaccinia virus morphogenesis is interrupted when expression of the gene encoding an 11-kilodalton phosphorylated protein is prevented by the *Escherichia coli* lac repressor. *J. Virol.* 65:6101–6110.

Zhang, Y., Ahn, B. Y., and Moss, B. (1994). Targeting of a multicomponent transcription apparatus into assembling vaccinia virus particles requires RAP94, an RNA polymerase-associated protein. *J. Virol.* 68:1360–1370.

Zinoviev, V. V., Tchikaev, N. A., Chertov, O. Y., and Malygin, E. G. (1994). Identification of the gene encoding vaccinia virus immunodominant protein p35. *Gene* 147:209–214.

Zwartouw, H. T. (1964). The chemical composition of vaccinia virus. *J. Gen. Microbiol.* 34:115–123.
