
Cell and Molecular Biology of the Exosome: How to Make or Break an RNA

Geurt Schilders, Erwin van Dijk, Reinout Raijmakers, and Ger J. M. Pruijn  
Department of Biochemistry, Nijmegen Center for Molecular Life Sciences  
Radboud University Nijmegen, Nijmegen, The Netherlands  

---

The identification and characterization of the exosome complex has shown that the exosome is a complex of 3′ → 5′ exoribonucleases that plays a key role in the processing and degradation of a wide variety of RNA substrates. Advances in the understanding of exosome function have led to the identification of numerous cofactors that are required for a selective recruitment of the exosome to substrate RNAs, for their structural alterations to facilitate degradation, and to aid in their complete degradation/processing. Structural data obtained by two-hybrid interaction analyses and X-ray crystallography show that the core of the exosome adopts a doughnut-like structure and demonstrates that probably not all exosome subunits are active exoribonucleases. Despite all data obtained on the structure and function of the exosome during the last decade, there are still a lot of unanswered questions. What is the molecular mechanism by which cofactors select and target substrate RNAs to the exosome and modulate its function for correct processing or degradation? How can the exosome discriminate between processing or degradation of a specific substrate RNA? What is the precise structure of exosome subunits and how do they contribute to its function? Here we discuss studies that provide some insight to these questions and speculate on the mechanisms that control the exosome.

KEY WORDS: Exosome, PM/Scl complex, RNA processing, RNA degradation, 3′ → 5′ exoribonuclease. © 2006 Elsevier Inc.
---

International Review of Cytology, Vol. 251   159   0074-7696/06 $35.00  
Copyright 2006, Elsevier Inc. All rights reserved.   DOI: 10.1016/S0074-7696(06)51005-8

I. Introduction

A. The History of the Exosome

The human exosome was originally identified as an autoantigen in patients with the overlap syndrome of the systemic autoimmune disorders polymyositis (PM) and scleroderma (Scl). For this reason, it was designated the “PM/Scl complex.” In 1977, autoantibodies were described in the serum of patients suffering from polymyositis which could precipitate an antigen (called “PM-1” at first) from calf thymus extract in immunodiffusion assays ([Wolfe et al., 1977](https://doi.org/10.1002/art.1780100506)). These autoantibodies were shown to be most prevalent in patients with the overlap syndrome of PM and Scl ([Reichlin et al., 1984](https://doi.org/10.1002/art.1780270306); [Treadwell et al., 1984](https://doi.org/10.1002/art.1780270307)) and the antigen was called the PM/Scl antigen. By indirect immunofluorescence using patient autoantibodies, the PM/Scl antigen was shown to accumulate in the nucleolus of human cells ([Targoff and Reichlin, 1985](https://doi.org/10.1002/art.1780280113)). The first insight into the structure and function of the complex came when it was discovered that the PM/Scl antigen represented a complex of proteins, with 8 to 9 of these proteins having apparent molecular weight between 18 kDa and 39 kDa (which would later turn out to be the core exosome components) and 3 to 4 proteins with apparent molecular masses between 80 kDa and 110 kDa. Some of these proteins were suggested to be phosphorylated and for the first time a function for the complex in ribosome synthesis was suggested, mainly based upon its nuclear localization, which was now confirmed using purified antibodies ([Gelpi et al., 1990](https://doi.org/10.1002/art.1780330116); [Reimer et al., 1986](https://doi.org/10.1002/art.1780290116)).

Two of the autoantigenic protein components of the complex were identified by screening human cDNA expression libraries and anti-PM/Scl patient sera. The first of these was the PM/Scl-75 protein, which turned out to be a 40-kDa polypeptide with an aberrant migration at 75 kDa in SDS-PAGE due to a highly charged C-terminal region. The protein was found to be localized to the nucleolus and was suggested to contain a nuclear localization signal in the highly charged domain ([Alderuccio et al., 1991](https://doi.org/10.1002/art.1780340116)). The originally described amino acid sequence turned out to lack a part of the N-terminus; nevertheless, the calculated molecular mass of the full-length protein (50 kDa) is still significantly less than 75 kDa ([Raijmakers et al., 2003](https://doi.org/10.1002/art.11104)). The second protein to be identified was PM/Scl-100, which was independently cloned by two different research groups ([Bluthner and Bautz, 1992](https://doi.org/10.1002/art.1780350116); [Ge et al., 1992](https://doi.org/10.1002/art.1780350117)). The identified protein was found to be a 100-kDa polypeptide with no clear homology to other proteins at that time. Antibodies raised against the N-terminal part of the protein showed that this PM/Scl protein also was localized to the nucleolus.

After the identification of these proteins it would still take another 5 years before new insight into the function of the PM/Scl complex was obtained.

STRUCTURE AND FUNCTION OF THE EXOSOME 161

Both PM/Scl-75 and PM/Scl-100 were shown to contain domains with a high level of similarity to the bacterial exoribonucleases RNase PH and RNase D, respectively (Mian, 1997). In the same year, a novel complex involved in the maturation of pre-ribosomal RNA was identified in the yeast *Saccharomyces cerevisiae*; the complex appeared to consist of a large number of exoribonucleases, and was therefore called the *exosome* (Mitchell *et al.*, 1997). Soon thereafter it was realized that the PM/Scl complex and the exosome were in fact the same complex, but in different species (Allmang *et al.*, 1999b). This facilitated the identification of the other components of the human PM/Scl complex (Brouwer *et al.*, 2001b; Chen *et al.*, 2001), most of which share homology with yeast exosome proteins and bacterial ribonucleases. Because of its structural and functional homology to the yeast exosome, the PM/Scl complex is nowadays most often referred to as the *human exosome* complex (Brouwer *et al.*, 2001a).

## II. General Properties of the Exosome

### A. Composition and Structure

#### 1. Composition of the Core

The exosome complex consists of a core of nine polypeptides and a number of exosome-associated proteins. Six of the core components contain an RNase PH domain (RPD) and the other three contain S1 and/or KH RNA-binding domains (Fig. 1A). The RPD-containing proteins are members of the PDX family of exoribonucleases, whereas the other three belong to the RRP4 exoribonuclease family (Zuo and Deutscher, 2001). In eukaryotes each of the nine core subunits is encoded by a different gene, whereas in archaeal exosomes multiple copies of two RPD-type subunits and two RRP4-type subunits are present (see also below).

The assembly of six RPD domains and three S1/KH domains in the core of the exosome is very similar to the eubacterial polynucleotide phosphorylase (PNPase) complex. PNPase was originally identified as the first RNA synthesizing enzyme (Grunberg-Manago and Ochoa, 1955), for which Severo Ochoa was awarded the Nobel Prize for Physiology or Medicine in 1959. This enzyme appeared to synthesize RNA using nucleoside diphosphates (NDPs) in the absence of templates and (thus) not sequence specifically. Later it became clear that not RNA synthesis, but RNA degradation is the main function of PNPase, and that this enzymatic reaction is fully reversible. In the presence of an excess of NDPs, the enzyme elongates RNA strands, whereas in the presence of an excess of free phosphate it degrades RNA

FIG. 1 The composition and structure of the exosome complex. Models for the human exosome core (panel A) and yeast exosome core (panel B) with RPD1-type subunits colored in different shades of green, RPD2-type subunits in shades of blue, and S1 domain containing subunits in shades of orange. Note that yeast and human orthologues have the same color code, and that the suggested order of the subunits in the two models is different. Panel C shows the crystal structure of an archaeal exosome core complex (from *Archaeoglobus fulgidus*), consisting of alternating Rrp42 (green) and Rrp41 (blue) subunits in the ring, two Rrp4 subunits (dark orange), and one Csl4 subunit (light orange). In panel D, an Rrp41-Rrp42 dimer of the archaeal exosome crystal structure is shown, bound to a four-nucleotide RNA molecule. The most 3′ nucleotide (n₁) is located very close to the catalytic site of Rrp41 (residue Asp182 is marked yellow) and the phosphate binding site (residues Ser138 and Arg139 are marked red). Note that a small part of the Rrp42 protein has been removed in the inset image, to allow a closeup of the inside of the RNA binding pocket.

(Yehudai-Resheff et al., 2001). Although the protein was at first found only in eubacteria, it became clear that in eukaryotes PNPase proteins are often present in chloroplasts and mitochondria (Baginsky et al., 2001; Piwowarski et al., 2003; Rajmakers, 2004; Rajmakers et al., 2002a).

Both bacterial and organellar PNPase polypeptides contain two individual RPDs, and two RNA-binding domains, one KH domain, and one S1 domain. PNPase proteins are known to assemble into a homotrimeric complex, resulting in a total of six RPDs and three KH/S1 domains per PNPase trimer (Symmons *et al.*, 2000). The nine exosome core components, which were originally described in the yeast *S. cerevisiae*, are Rrp41p, Rrp42p, Rrp43p, Rrp45p, Rrp46p, and Mtr3p, the six proteins homologous to the bacterial RNase PH, and Rrp4p, Rrp40p, and Csl4p, which contain the S1 and KH RNA-binding domains (Table I) (Allmang *et al.*, 1999b; Mitchell and Tollervey, 2000; Mitchell *et al.*, 1997). Archaeal exosomes contain two RNase PH-type subunits, designated Rrp41 and Rrp42, and also two S1/KH-type subunits, Csl4 and Rrp4 (Evguenieva-Hackenberg *et al.*, 2003).

## Evolutionary Aspects of Exosome Core Proteins

Because of the remarkable similarity between PNPase and exosome complexes it is very likely that these proteins are evolutionarily related. A protein containing a single RPD is probably the earliest common ancestor of PNPase and the exosome. Following gene duplication of the RPD encoding gene, the

TABLE I  
Components of the Core Exosome Complex  

| Human protein | Yeast orthologue | Conserved domains | Activity | Exoribonuclease family |
|---------------|-------------------|--------------------|----------|------------------------|
| hCsl4p        | Csl4p/Ski4p       | S1                 | RNA binding<sup>a</sup> | RRP4                  |
| hRrp4p        | Rrp4p             | S1/KH              | 3′ → 5′ hydrolytic exoribonuclease<sup>a</sup> | RRP4                  |
| hRrp40p       | Rrp40p            | S1/KH              | 3′ → 5′ hydrolytic exoribonuclease | RRP4                  |
| hRrp42p       | Rrp42p            | RPD1               | 3′ → 5′ phosphorolytic exoribonuclease | PDX                   |
| OIP2          | Rrp43p            | RPD1               | 3′ → 5′ phosphorolytic exoribonuclease | PDX                   |
| PM/Scl-75     | Rrp45p            | RPD1               | 3′ → 5′ phosphorolytic exoribonuclease<sup>a</sup> | PDX                   |
| hMtr3p        | Mtr3p             | RPD2               | 3′ → 5′ phosphorolytic exoribonuclease | PDX                   |
| hRrp41p       | Rrp41p/Ski6p      | RPD2               | 3′ → 5′ phosphorolytic exoribonuclease<sup>a</sup> | PDX                   |
| hRrp46p       | Rrp46p            | RPD2               | 3′ → 5′ phosphorolytic exoribonuclease | PDX                   |

<sup>a</sup>Hypothetical activity.

acquirement of an interaction with proteins containing KH and/or S1 domains might have resulted in a complex containing all protein domains found in present day exosomes and PNPsases. After the RPD duplication, both copies of this domain apparently have diverged with a concomitant loss of exoribonuclease activity in one of the two RPDs. The N-terminal domains (RPD1) of PNPsases display less homology to the PDX family consensus sequence than their RPD2 domains ([Symmons et al., 2000](https://doi.org/10.1073/pnas.250496897)). In eubacteria, the genes encoding the individual subunits probably have fused to yield the gene encoding the PNPsase protein, a phenomenon that is not uncommon to prokaryotic organisms ([Snel et al., 2000](https://doi.org/10.1073/pnas.250496897)). In addition, the relative positioning of the S1 and KH domains has switched, most likely in the eubacterial PNPsase gene, because domain swapping most often occurs in large multi-domain proteins ([Fliess et al., 2002](https://doi.org/10.1073/pnas.250496897)). Finally, eubacteria often also express an RNase PH protein, which possibly originated from a duplication of the RPD2 of eubacterial PNPsase. In Archaea and eukaryotes, apparently only the S1 and KH domains have fused together. Because many Archaea have a Csl4 homologue, containing only the S1 domain, this duplication also must have occurred in the archaeal lineage. Interestingly, three exosome subunit genes (all except the Csl4 homologue) display a tandem arrangement in the genomes of many archaebacteria ([Koonin et al., 2001](https://doi.org/10.1073/pnas.250496897)), reminiscent of the early clustering of these genes which probably also allowed the genes to be fused in eubacteria. In eukaryotes, a number of gene duplications led to the presence of six RPD containing exosome subunits and three RRP4-type exosome proteins. Finally, PNPsase was introduced into eukaryotes concomitant with the generation of mitochondria and chloroplasts, although some eukaryotes (e.g., *Saccharomyces cerevisiae*) seem to have lost the PNPsase gene during evolution ([Fig. 2](#fig:2)).

As previously described, the domain structures of PNPsase and exosome core subunits display extensive similarities. To identify with which of the two RPDs of PNPsase the exosome proteins containing an RPD share the highest degree of homology, the RPDs of all exosomal RNase PH-type subunits was compared with the RPDs of a number of PNPsases from different organisms ([Raijmakers et al., 2002a](https://doi.org/10.1073/pnas.250496897); [Yehudai-Resheff et al., 2003](https://doi.org/10.1073/pnas.250496897)). A multiple alignment of the segment of these RPDs that is most highly conserved among PNPsases and exosome components showed the second RPD of PNPsase is more highly conserved than the first and a clear distinction can be made between the two RPDs of each PNPsase (see also earlier mention). Strikingly, the RPD of three human (hRrp41p, hRrp46p, and hMtr3p) and the corresponding three yeast exosome proteins displayed the highest degree of sequence homology with the second RPD of PNPsases, whereas the other three human RNase PH-type exosome subunits (hRrp42, PM/Scl-75, and OIP2) and the corresponding yeast counterparts appeared to be more homologous to the first RPD of PNPsases. This classification of RPD-containing exosome subunits was

STRUCTURE AND FUNCTION OF THE EXOSOME 165

Eukaryotes
TP
PNPase
Endosymbiosis leading
to chloroplasts and
mitochondria
Exosome
[ RPD1-like
RPD2-like
RRP4 family ]

Eubacteria
PNPase
RNase PH
Duplication of
RPD2 domain
Fusion of all
the domains
Swap of the KH
and S1 domains

Archaea
Duplications of
exosome proteins
Exosome
Duplication of S1
domain
Fusion of the KH
and S1 domains

Ancient PN Pase/ exosome
RPD1 - RPD2 - S1 - KH -
↑ Addition of KH and S1 domains
and divergence of RPDs
RPD - RPD -
↑ RPD duplication
RPD

FIG. 2 Model for the evolution of the exosome and related proteins. A schematic overview of the evolutionary events leading from a single RPD domain ancestor to the exosome complex and the related protein PNPase in present day multicellular eukaryotes. All RPD1 domains are colored in shades of green, RPD2 domains in shades of blue, and S1 and KH domains in shades of orange and yellow, respectively. Note that the PNPase proteins have acquired an α-helical domain linking the two RPDs (not highlighted in this scheme) and that the eukaryotic PNPase proteins contain N-terminal transition peptides (TP), which allow their import into the chloroplasts and/or mitochondria.

confirmed by a comparison of the complete RPD sequences, as well as by a comparison of archaeal and human RPD containing subunits (Lorentzen *et al.*, 2005; Rajmakers *et al.*, 2002a).

166                                                                 SCHILDERS ET AL.

3. Proteins and Complexes Interacting with the Core

Today, quite a number of additional proteins or protein complexes have been reported to have the capacity to associate with the exosome core complex, although differences are observed between organisms and most of these proteins are only bound to a subset of exosome core complexes in the cell. These proteins include Rrp6p (PM/Scl-100); Dis3p (Rrp44p); Rrp47p (Lrp1p, C1D); Mtr4p, which is associated with the TRAMP complex; Ski2p, which is associated with the Ski-complex; and MPP6 (Table II). Homologues of most though not all of these proteins have been identified in the eukaryotes analyzed and the association with the exosome for some of them might be species-specific. MPP6, for instance, seems to be specific for the human exosome and in contrast to the yeast Dis3p protein, the human hDis3p protein does not seem to be stably associated with the exosome (Allmang et al., 1999b; Brouwer et al., 2001b; Chen et al., 2001; Lim et al., 1997; Shiomi et al., 1998). In yeast, the RNase D homologue Rrp6p was found only in the nuclear exosome complex and Dis3p/Rrp44p (homologous to bacterial RNase R) in both nuclear and cytoplasmic exosome complexes (Mitchell and Tollervey, 2000). In contrast to the exclusively nuclear localization reported for Rrp6p in yeast, its human homologue PM/Scl-100 was found in both the nuclear and cytoplasmic exosome (Brouwer et al., 2001b; Lejeune et al., 2003). The exosome-associated protein termed Rrp47p/Lrp1p in yeast, TbEAP3 in *Trypanosoma brucei*, and C1D in humans most likely binds to the exosome via interactions with Rrp6p, TbRRP6, and PM/Scl-100, respectively (Estevez et al., 2003; Mitchell et al., 2003; Peng et al., 2003) (our unpublished observations). MPP6, for which no yeast counterpart has been identified so far, binds to PM/Scl-100 and probably also mediates the association of hMtr4p (Lehner and Sanderson, 2004). In yeast the Mtr4p protein has been reported to be part of the TRAMP complex, which also contains the Trf4p/Trf5p and Air1p/Air2p proteins (LaCava et al., 2005; Wyers et al., 2005). The Ski2p protein, which like Mtr4p belongs to the family of DEAD box RNA helicases, is a component of the Ski-complex, containing also Ski3p, Ski7p, and Ski8p. The interaction of the Ski-complex with the exosome is probably mediated by Ski7p in yeast (Araki et al., 2001). An overview of exosome proteins identified in various organisms is given in Table III. To reduce the complexity of the nomenclature of exosome subunits we propose a general name for each of these in the first column of the table, which is mainly based on the nomenclature in *S. cerevisiae*, the organism in which most subunits were first identified. The compositional variation of the exosome complexes is probably further increased (in higher eukaryotes) by the expression of splice variants of both core and associated proteins. At least four possible splice variants exist for PM/Scl-75, the human homologue of the yeast Rrp45p protein. Because no functional differences were found so far.

TABLE II  
Exosome-Associated Proteins  

| Human protein | Yeast orthologue | Conserved domains | Activity | Comments |
|---------------|------------------|--------------------|----------|----------|
| PM/Scl-100    | Rrp6p            | RNaseD            | $3' \rightarrow 5'$ hydrolytic exoribonuclease$^{b}$ | in yeast only associated with the nuclear exosome |
| hDis3p$^{a}$  | Rrp44p/Dis3p     | RNaseR            | $3' \rightarrow 5'$ hydrolytic exoribonuclease$^{b}$ | - |
| hSki2$^{c}$   | Ski2p            | DEADbox           | RNA helicase$^{b}$ | part of Ski complex |
| hSki3$^{c}$   | Ski3p            | TPR repeats       | unknown   | part of Ski complex |
| hSki8$^{c}$   | Ski8p            | WD 40 repeats     | unknown   | part of Ski and hPAF complex |
| eRFS$^{a}$    | Ski7p            | GTPase            | unknown   | part of Ski complex, is associated with the cytoplasmic exosome |
| KIAA 0052     | Mtr4p/Dob1p      | DEADbox           | RNA helicase$^{b}$ | part of TRAMP complex |
| TRF4-1$^{a}$  | Trf4p            | PAP-associated NTP transferase | Poly(A) polymerase | part of TRAMP complex |
| TRF4-2$^{a}$  | Trf5p            | PAP-associated NTP transferase | Poly(A) polymerase | interacts with Mtr4p |
| -             | Air1p            | RING-finger       | RNA binding$^{b}$ | can functionally replace Air2p |
| -             | Air2p            | RING-finger       | RNA binding$^{b}$ | part of TRAMP complex |
| MPP 6         | -                | -                 | RNA binding        | associated with the nuclear exosome |
| RHAU          | -                | DExH/D box        | RNA helicase$^{b}$ | longer isoform is located to the nucleus, shorter isoform to the cytoplasm |
| C1D$^{a}$     | Rrp47p           | C1D               | nucleotide binding$^{b}$ | associated with the nuclear exosome |

(continued)

TABLE II (continued)

| Human protein | Yeast orthologue | Conserved domains | Activity | Comments |
|---------------|------------------|--------------------|----------|----------|
| KSRP          | –                | KH                 | binds to the AU-rich element of an mRNA | – |
| AUF1/         | Hrp1p<sup>a</sup> | RRM                | binds to the AU-rich element of an mRNA | – |
| hnRNP D       | –                | –                  | –        | – |
| TTP           | Tis11p<sup>a</sup> | Zinc finger-CCCH   | binds to the AU-rich element of an mRNA | also its homologue BRF-1 can interact with the exosome |
| ISG20<sup>d</sup> | Rex4p<sup>a</sup> | ExoI/II/III        | 3' → 5'exonuclease<sup>b</sup> | – |
| Ran<sup>a</sup> | Gsp1p            | GTPase             | GTPase   | interacts with Dis3p |
| CGI-37<sup>a</sup> | Nip7p            | –                  | unknown  | interacts with Dis3p |
| Rpp14         | –                | –                  | 3' → 5' phosphorolytic exoribonuclease | interacts with OIP2 |

<sup>a</sup> Association is based on homology only.  
<sup>b</sup> Hypothetical activity.  
<sup>c</sup> Our unpublished observations.  
<sup>d</sup> Dr. N. Mechti, personal communication.

General name | Drosophila melanogaster | Arabidopsis thaliana | Trypanosoma brucei | Saccharomyces cerevisiae | Archaea<sup>c</sup> | Calculated MW(kD) | HUGO gene symbol
--- | --- | --- | --- | --- | --- | --- | ---
Csl4 | hCsl4 | dCsl4 | TbCSL 4 | Csl4p/Ski4p | Csl4 | 21-32 | EXOSC1
Rrp4 | hRrp4 | dRrp4 | AtRrp4 | TbRRP 4 | Rrp4 | 28-39 | EXOSC2
Rrp40 | hRrp40 | dRrp40 | TbRRP 40 | Rrp40p | b | 27-32 | EXOSC3
Rrp42 | hRrp42 | dRrp42 | TbEAP1 | Rrp42p | Rrp42 | 29-32 | EXOSC7
Rrp43 | OIP2 | b | TbEAP2 | Rrp43p | b | 30-44 | EXOSC8
Rrp45 | PM/ScI-75 | dRrp45<sup>a</sup> | TbRRP 45 | Rrp45p | b | 34-49 | EXOSC9
Mtr3 | hMtr3 | dMtr3 | TbEAP4 | Mtr3p | b | 24-37 | EXOSC6
Rrp41 | hRrp41 | dSki6 | AtRrp41 | TbRRP41A | Rrp41p/Ski6p | Rrp41 | 26-28 | EXOSC4
Rrp46 | hRrp46 | dRrp46 | TbRRP41B | Rrp46p | b | 25-28 | EXOSC5
Rrp6 | PM/ScI-100 | dRrp6 | TbRRP 6 | Rrp6p | 84-100 | EXOSC10
Rrp44 | hDis3<sup>a</sup> | dDis3 | TbRRP 44<sup>a</sup> | Rrp44p/Dis3p | 105-113 | KIAA1008

* No experimental evidence for exosome association available (based on homology).
* No homologues in genome.
* Exosome components identified in Sulfolobus solfataricus and Archaeoglobus fulgidus.

between the isoforms with different N-termini or with a sequence element encoded by an optional exon, the function of this multiplicity of PM/Scl-75 splice variants remains to be determined (Raijmakers *et al.*, 2003). For the human PM/Scl-100 two splice variants have been annotated, which may at least in part correspond to the multiple PM/Scl-100 bands appearing in SDS-PAGE gels containing the affinity-purified human exosome (Chen *et al.*, 2001).

# 4. Structure of the Exosome Core Complex

The striking similarity of protein domains present in exosome subunits on the one hand and PNPase on the other hand, as well as their functional relationship, suggested that their three-dimensional structure might also be similar. The structure of *S.* *antibioticus* PNPase as obtained by X-ray crystallography has revealed that this protein exists as a homo-trimeric complex, in which the six RPDs form a ring-shaped hexameric structure (Symmons *et al.*, 2000). Each pair of RPDs present within a single PNPase molecule pack closely together to give an extended dumbbell-shaped intramolecular dimer. In the crystal structure the S1 domains of PNPase form a crown of mobile RNA-binding domains around the space in the center of the trimer. Based upon the PNPase structure and taking into account the interactions observed between different exosome subunits in a mammalian two-hybrid system, a model for the human exosome was generated in which the six RNase PH-like subunits form a hexameric ring and the three proteins with S1 (and KH) RNA-binding domains are positioned at the outer surface of this ring (Fig. 1A) (Raijmakers *et al.*, 2002a; Symmons *et al.*, 2000). In agreement with the *S.* *antibioticus* PNPase structure, the ring-shaped hexamer is predicted to be composed of three pairs of anti-parallel RPDs. The analysis of mutants of hRrp46, lacking either the N- or the C-terminus of the protein, demonstrated that the C-terminal part of hRrp46 interacts with OIP2 (our unpublished observations). In PNPase the C-terminal regions of the RPDs mediate the intramolecular interaction between these RPDs (Raijmakers *et al.*, 2002a). This suggests that hRrp46 and OIP2 correspond to one PNPase-like RPD pair in the ring structure, and extrapolation would mean that PM/Scl-75—hRrp41 and hRrp42—hMtr3 represent the other pairs. The ring-shaped model for the exosome core complex is expected to be applicable to the exosome of many species, including yeast. The first biophysical evidence for this architecture of the exosome core was provided by the EM-structure of the yeast exosome (Aloy *et al.*, 2002). Based upon this structure the relative positioning of the RPD-type subunits in the ring of the yeast exosome was predicted using bioinformatics data (Fig. 1B). This prediction only partially agrees with the model for the human exosome core proposed by us (Raijmakers *et al.*, 2002a). Although the discrepancy might

reflect species differences (our model is based on interactions between human exosome subunits), it is more likely due to an alternative interpretation of the homologies among the exosome subunit RPDs (based on a predicted functional site of only five amino acids [which is in fact located just outside the approximately 200-amino-acid-long RPD as predicted by Pfam]). Aloy and collaborators classified the yeast exosomal RPD proteins as homologues of either RPD1 (Rrp43p, Rrp46p, Mtr3p) or RPD2 (Rrp41p, Rrp42p, Rrp45p) of PNPase. When the complete RPDs (as predicted by Pfam) from many exosome and PNPase proteins or the most highly conserved segment of these domains are used for the analysis, Rrp42p, Rrp43p, and Rrp45p appear to be more closely related to RPD1 and Rrp41p, Rrp46p, and Mtr3p appear to be more closely related to RPD2 (*Raijmakers et al.*, 2002a). Importantly, the arrangement of the RPD subunits in our model is strongly supported by the available protein–protein interaction data for the yeast, human, and *T. brucei* exosome (*Estevez et al.*, 2003; *Lehner and Sanderson*, 2004; *Lorentzen et al.*, 2005; *Raijmakers et al.*, 2002a). Further biophysical support for the doughnut-like structure was provided by the crystal structure of the exosome core of the archaeon *Sulfolobus solfataricus*, which consists of a trimer of Rrp41-Rrp42 heterodimers (*Lorentzen et al.*, 2005; *Pruijn*, 2005). A comparison with the phosphate-binding site of PNPase and RNase PH facilitated the identification of the active sites of the archaeal exosome and mutational studies confirmed that its RNA degradation activity resides within the Rrp41 subunit.

These studies were confirmed and extended by the determination of the crystal structures of two nine-subunit archaeal exosome core isoforms, which contain either three copies of Rrp4 or three copies of Csl4 in addition to the three Rrp41-Rrp42 heterodimers. These *Archaeoglobus fulgidus* exosome core structures revealed that Rrp4 or Csl4 forms flat, trimeric caps that cover one face of the hexameric Rrp41-Rrp42 ring (*Buttner et al.*, 2005). Based on the analysis of the purified *A. fulgidus* exosome, the native archaeal exosome core is suggested to contain two copies of the Rrp4 protein and one copy of the Csl4 protein (*Fig.* 1C). The S1 domains of both Rrp4 and Csl4 appeared to be located near the center of the trimer, whereas the remaining domains of these proteins are situated on the outside of the trimer. In this structure, the Csl4 protein was shown to contain a C-terminal zinc-ribbon domain, which might have a structural role in the complex. The Csl4 and Rrp4 proteins were shown to mainly interact with the exosome core via their N- and C-terminal domains and not via the S1 domain, consistent with our early observation that human Csl4 does not require the S1 domain for its interaction with other exosome subunits (*Raijmakers et al.*, 2002b). The stacking of the Rrp4/Csl4 trimer on the Rrp41-Rrp42 hexamer extends the central channel, the diameter of which varies between about 10 Å and 18 Å depending on the position in the channel. The pore formed by the S1

domains is about 18 Å wide. Directly beneath the S1 pore, an Rrp41 loop forms a neck of about 10 Å in diameter, which is followed by a central chamber in the Rrp41-Rrp42 ring. The active sites are located in this chamber. The second end of the channel is formed by a pore of about 18 Å in diameter ([Buttner *et al.*, 2005](https://doi.org/10.1038/nsb1276)). Based upon these structural data, a mechanism was proposed for the recognition and degradation of substrate RNAs by the exosome core. Rrp4 and Csl4 are proposed to recruit substrate RNAs and/or cofactors and lead the RNA to the entrance of the channel formed by their S1 domains. The S1 domains guide the RNA into the neck, which allows only single-stranded RNA to enter the central processing chamber, where, in the archaeal exosome, three active sites are present on the surface of Rrp41 to catalyze the phosphorolytic removal of 3′ nucleotides. Interestingly, the crystal structures of the *S. solfataricus* Rrp41-Rrp42 hexameric core bound to A₅ or U₈ RNA or to ADP are completely consistent with this mechanism ([Lorentzen and Conti, 2005](https://doi.org/10.1038/nsb1276)). The four most 3′ nucleotides of these RNAs interact with the interface of the Rrp41 and Rrp42 subunits in a region facing the processing chamber (Fig. 1D). The 3′ end of the RNA is located near the phosphate binding site and the active site of Rrp41 (Fig. 1D, inset). Moreover, the binding mode of ADP appeared to be very similar to that of the most 3′ nucleotide of the RNAs.

## 5. Interaction of Cofactors with the Exosome Core

The mode of binding of the exosome-associated proteins Rrp6p/PM/Scl-100, Dis3p, Rrp47p/C1D/TbEAP3, MPP6, and the Ski and TRAMP complexes to the exosome core is still largely unknown. Two-hybrid analyses did not reveal interactions with core exosome subunits. Although Rrp6p/PM/Scl-100 was suggested to be a core exosome component by the original studies, the exclusive association of Rrp6p with the nuclear exosome supports the notion that this protein indeed cannot be considered a core component of the exosome ([Allmang *et al.*, 1999b](https://doi.org/10.1038/nsb1276)). Moreover, the results of size-exclusion chromatography experiments demonstrated that PM/Scl-100 is only associated with the higher molecular mass exosome complexes (approx. 700 kDa) and not with lower molecular mass complexes (250–600 kDa), in which all core components were detected ([Brouwer *et al.*, 2001b](https://doi.org/10.1038/nsb1276)). Also in glycerol gradient sedimentation PM/Scl-100 showed a distinct fractionation pattern in comparison with exosome core components ([Schilders *et al.*, 2005](https://doi.org/10.1038/nsb1276)). Taken together, these results strongly suggest that PM/Scl-100 is associated with only a subset of exosome core complexes in the cell. The failure of Rrp6p/PM/Scl-100/TbRRP6 to interact with exosome core components in two-hybrid systems might be due to the fact that this protein requires more

than one of the core components for efficient binding. In view of the fact that Rrp6p is relatively stably associated with (a subset of) yeast exosomes and that the core model fitted in the EM density left a lump of extra density, it is tempting to speculate that this represents the position of Rrp6p (*Aloy et al.*, 2002, 2004). This would mean that Rrp6p binds to the surface of the cap formed by the RRP4-type subunits and that at least two of these subunits are required for its association. Interestingly, the orientation of protein domains of these subunits in the nine-subunit archaeal crystal structure suggests that the caps form a macromolecular interaction surface with two zones. One of these is formed by the S1 domains and would be suited for the regulation of RNA entry into the channel. The other, peripheral zone, which contains several putative macromolecular interaction surfaces, may be involved in the recognition of cofactors like Rrp6p (*Buttner et al.*, 2005).

Interestingly, data from yeast, *T. brucei*, and human suggest Rrp6p/TbRRP6/PM/Scl-100 directly interacts with Rrp47p/TbEAP3/C1D (*Estevez et al.*, 2003; *Mitchell et al.*, 2003; our unpublished observations) as well as with MPP6 (*Lehner and Sanderson*, 2004). Also a two-hybrid interaction between hMtr4p and MPP6 has been reported (*Lehner and Sanderson*, 2004). The *in vivo* function of the exosome is expected to require the activity of additional cofactors. This is supported by the results of genetic experiments in yeast, which show that all characterized nuclear functions of the exosome require the putative RNA helicase Mtr4p/Dob1p (*de la Cruz et al.*, 1998), whereas exosome-mediated cytoplasmic mRNA turnover pathways are dependent upon Ski7p and the Ski-complex, comprising the putative RNA helicase Ski2p and the proteins Ski3p and Ski8p (*Brown et al.*, 2000; *Jacobs Anderson and Parker*, 1998; *Van Hoof et al.*, 2000b).

The ring-shaped structure of the exosome core contains a single copy of all RNase PH- and S1-like subunits. The results of co-immunoprecipitation experiments suggest that at least two copies of hRrp4 and hRrp41 might be present in the assembled human exosome (*Rajmakers et al.*, 2002a), suggesting that two or more of these rings may assemble into a single complex. These observations raise the question: What is the overall structure of the human exosome? One option is that the exosome can form a proteasome- or Hsp60-like structure (which consists of 4 or 2 stacked heptameric rings, respectively [*Groll et al.*, 1997; *Xu et al.*, 1997]) by direct ring-ring interactions. The potential similarity between exosome-mediated RNA degradation and proteasome-mediated protein degradation has been long suspected and the elucidated crystal structures of the archaeal exosome core strongly support this view (*Buttner et al.*, 2005; *Lorentzen and Conti*, 2005). Another possibility is that PM/Scl-100 and/or hDis3 are able to associate with more than one ring simultaneously, thereby combining two or more rings into one

complex. Interestingly, the calculated molecular mass of a complex containing one copy of each of the “core” subunits of the human exosome is approximately 260 kDa, which means that a complex consisting of two rings and one copy of PM/Scl-100 and hDis3 (both approx. 100 kDa) would indeed be around 700 kDa, in good agreement with the size-exclusion chromatography data ([Brouwer *et al.*, 2001b](https://doi.org/10.1093/nar/gkg548)). The appearance of 250–600 kDa complexes may represent disassembled or only partially assembled exosomes. Based upon its EM structure, the yeast exosome does not appear to consist of multiple rings ([Aloy *et al.*, 2002](https://doi.org/10.1016/s0092-8674(02)00636-9)), which is in agreement with its molecular mass of 300–400 kDa ([Mitchell *et al.*, 1997](https://doi.org/10.1016/s0092-8674(00)80370-2)). Also the crystal structures of the archaeal exosome core complexes did not reveal any interaction between two hexameric RPD-domain rings.

## B. Exosome Complexes in Various Organisms

### 1. Saccharomyces cerevisiae

The first exosome component was discovered in the yeast *S. cerevisiae* when it was noticed that a mutation in Rrp4p leads to 5.8S rRNA precursors which are extended at their 3′ ends and recombinant Rrp4p turned out to have 3′ → 5′ exoribonuclease activity *in vitro* ([Mitchell *et al.*, 1996](https://doi.org/10.1016/s0092-8674(00)80370-2)). Soon after, it was found that Rrp4p was part of a complex containing at least four other proteins. The proteins belonging to this complex were named Rrp4p, Rrp41p, Rrp42p, Rrp43p, and Rrp44p, where Rrp stands for Ribosomal RNA Processing. Three of these proteins contain an RNase PH domain, whereas Rrp44p is homologous to the exoribonuclease RNase R ([Mitchell *et al.*, 1997](https://doi.org/10.1016/s0092-8674(00)80370-2)). Because almost all of these subunits appeared to have exoribonuclease domains, the complex was termed *the exosome*.

Screening of the yeast genome for more proteins with a putative RNase PH domain revealed three more ORFs, one of which was found to encode for the previously identified protein Mtr3p and the other two coded for novel proteins that were now called Rrp45p and Rrp46p. Like the other identified exosome components, these three proteins appeared to be involved in 5.8S rRNA maturation. Subsequent mass spectroscopy analysis of the purified exosome identified two additional exosome subunits, Rrp40p and Csl4p. Combined data from co-immunoprecipitation, biochemical purification, and genetic depletion experiments indicated that these proteins might be part of a single complex. One additional component, Rrp6p, appeared to be present in sub-stoichiometric quantities in comparison to the other exosome components. Moreover, depletion of Rrp6p led to different effects on 5.8S rRNA processing and appeared to be a nuclear-specific exosome component ([Allmang *et al.*, 1999b](https://doi.org/10.1016/s0092-8674(00)80370-2)).

STRUCTURE AND FUNCTION OF THE EXOSOME 175

2. *Homo sapiens*

Soon after the identification of the yeast exosome, human homologues were identified for most of the yeast exosome components. Expression of the human homologues of Rrp4p, Rrp44p, and Csl4p suppressed pre-rRNA processing defects in yeast strains carrying mutations in the corresponding yeast genes indicating functional complementation. Additionally, the yeast exosome component Rrp6p was found to be the homologue of the human PM/Scl-100 protein, whereas Rrp45p is homologous to PM/Scl-75. Both proteins are part of the human PM/Scl complex indicating that the yeast exosome and the PM/Scl complex are similar complexes (see also Chapter I) (*Allmang et al.*, 1999b).

Other human exosome components were identified by database homology searches that revealed human ESTs with homology to yeast exosome components. The cDNAs encoding hRrp40p, hRrp41p, and hRrp46p were isolated and co-immunoprecipitation experiments confirmed the association of hRrp40p, hRrp41p, and hRrp46p with the PM/Scl complex (*Brouwer et al.*, 2001b). The hRrp41p protein was shown to be able to restore exosome function in yeast strains lacking the yeast Rrp41p protein. Further identification of the remaining human exosome components was performed by purification of the complex using a TAP-tagged exosome component, hRrp4p, and mass spectrometry. In this study, all known human homologues of yeast exosome components were found and one new human exosome component, hMtr3p, was identified. Remarkably, the only protein that was not present in the purified human exosome was hDis3p, the homologue of the yeast core exosome component Dis3p. In addition to core exosome components, the putative RNA helicase KIAA0052 (hMtr4p) and the RNA-binding protein MPP6 co-purified with the human exosome in this study (*Chen et al.*, 2001).

3. *Trypanosoma brucei*

The exosome of *Trypanosoma brucei*, an extracellular eukaryotic flagellate parasite, has been identified in a similar way as the human exosome. Initially, the purification of the *T*. *brucei* exosome from a cytosolic extract revealed the subunits TbRRP6, TbRRP4, TbRRP45, and TbCSL4. As for the human exosome TbRRP44 (Dis3p) is not associated with the exosome complex, but in contrast to the yeast and human exosome TbRRP6 appears to be present in stoichiometric amounts in the cytoplasmic exosome (*Estevez et al.*, 2001). Further analysis revealed the other core exosome components of the *T*. *brucei* exosome which appeared to be very similar to the yeast exosome. Six *T*. *brucei* components contain an RNase PH domain: TbRRP41A, TbRRP41B, TbRRP45, TbEAP1, TbEAP2, and TbEAP4, and are presumably the

homologues of Rrp41p, Rrp46p, Rrp45p, Rrp42p, Rrp43p, and Mtr3p, respectively, whereas *TbRRP4*, *TbRRP40*, and *TbCSL4* are related to the S1 RNA-binding proteins in yeast. Nevertheless, it should be noted that due to poor sequence conservation the assignment of orthologues is difficult (*Estevez et al.*, 2003).

### 4. Drosophila melanogaster

In *Drosophila* the exosome complex was identified because of its association with dSpt6, a transcription elongation factor that is recruited to sites of active transcription. Nine core exosome components could be identified and, like in yeast, Dis3p/Rrp44p was found to be stably associated to the exosome in *Drosophila*. In contrast to the exosome complexes previously described, no homologue of Rrp43p and Rrp45p were co-purified along with the other exosome components. Screening of the *Drosophila* genome revealed a putative homologue of Rrp45, but no clear homologue of Rrp43 could be identified. Either this means that *Drosophila* has an unrecognizable homologue of Rrp43 or that Rrp43 is not present in the core exosome of this organism (*Andrulis et al.*, 2002).

### 5. Arabidopsis thaliana

Also in *Arabidopsis thaliana*, a small flowering plant, several exosome components have been identified. *AtRrp41* is able to complement the lethal phenotype of depletion of Rrp41p in yeast showing functional conservation of the complex between the different organisms. Protein-protein interaction studies showed that *AtRrp41* can interact with the yeast proteins Rrp4p, Rrp44p, and Rrp45p *in vitro* suggesting that in contrast to human and *T. brucei*, Rrp44p/Dis3p is an integral part of the *A. thaliana* exosome. So far, the only other exosome component identified in this organism is *AtRrp4*. However, *AtRrp41p* is found in high-molecular-weight complexes, which are similar in size to the human and yeast exosome complexes. Most likely, these large complexes represent the complete plant exosome (*Chekanova et al.*, 2000, 2002).

### 6. Archaea

The existence of exosome complexes in archaea was first predicted by clusters of genes showing similarity to exosome subunits from other organisms in completely sequenced archaeal genomes. Most Archaea were found to have genes encoding homologues of Rrp41p, Rrp42p, Rrp4p, and Csl4p. Except for the *Csl4p* gene, these are located in a single gene cluster or superoperon, which also contains the genes for other RNA-processing

proteins and some proteasome subunits (Koonin *et al.*, 2001). The only two Archaeal families that do not appear to have an exosome complex are the *Methanococci* and the halophilic Archaea (Portnoy *et al.*, 2005).

Biochemical purification of the *S.* *solfataricus* exosome confirmed that it consists of only four subunits that are the orthologues of the yeast proteins, Rrp41p, Rrp42p, and Rrp4p, which are present in similar amounts, whereas the fourth orthologue Csl4p appeared to be present in smaller amounts. Besides the core exosome components, the protein DnaG was identified as an exosome cofactor and was proposed to function as an RNA helicase (Evguenieva-Hackenberg *et al.*, 2003). In the purified exosome of *Methanothermobacter thermautotrophicus* no Csl4p could be identified, but the DnaG protein was found at similar levels as the core exosome components and one additional protein, an tRNA intron endonuclease, was found to be associated with the complex (Farhoud *et al.*, 2005). The crystal structures obtained from the exosome complexes of *S.* *solfataricus* and *Archaeoglobus fulgidus* have greatly increased our understanding of the functioning and structure of exosome complexes from all organisms. Because these experiments were performed with Archaeabacteria overexpressing different exosome components, the composition of the complex could be controlled. As previously described, high-resolution crystal structures are now available for the core structure of several Archaea and with various protein compositions. (Buttner *et al.*, 2005; Lorentzen and Conti, 2005; Lorentzen *et al.*, 2005).

In summary, the observation that not all organisms, most notably the Archaeabacteria, have six different RNase PH-like proteins and three different S1 RBD proteins indicates that organisms can have several copies of one specific protein in an assembled exosome. If also in higher organisms (a subset of) the exosome complexes can contain multiple copies of the same protein remains to be seen, but there is some evidence that at least in humans more copies of hRrp4p and hRrp41p can be present in the exosome complex. This could, however, also be explained by multimerization of complete exosome complexes in a manner similar to the proteasome (Rajmakers *et al.*, 2002a). For a complete overview of all identified exosome subunits and associated proteins in different organisms see Table III.

### C. Subcellular Localization

#### 1. Exosome Core

The exosome has been described to function in a wide variety of processes, such as processing of ribosomal RNA, small nuclear RNA, and mRNA turnover (see Chapter III for details). Therefore, the exosome is expected to be localized in the cytoplasm, the nucleoplasm, as well as the nucleolus.

Indeed, immuno-fluorescence experiments in yeast have demonstrated that several core exosome components localize to the nucleoplasm, nucleolus, and to a lesser extent in the cytoplasm (Allmang *et al.*, 1999b; Van Hoof *et al.*, 2000b; Zanchin and Goldfarb, 1999a). Although the human exosome is believed to have a similar subcellular localization, fluorescence microscopy with human cells expressing GFP- and VSV-tagged exosome components indicated that the core exosome is almost exclusively found in the nucleus with an enrichment in the nucleoli (Brouwer *et al.*, 2001b; Rajmakers *et al.*, 2002b, 2003). Immunolocalization studies using PM/Scl patient sera are consistent with this observation (Alderuccio *et al.*, 1991; Gelpi *et al.*, 1990; Reimer *et al.*, 1986). It should, however, be noted that in the majority of PM/ Scl patient sera contain reactivity to PM/Scl-100, which is an exosome cofactor enriched in the nucleus. To some extent, the subcellular distribution of the exosome reflects the number of substrate RNAs present in the different cellular compartments. The amount of ribosomal RNAs, which needs to be processed by the exosome, exceeds the amount of small nuclear and mRNAs, whereas the volume of the nucleoli is much smaller than that of the nucleoplasm and cytoplasm. Therefore, the relatively low concentration of substrate RNAs in the nucleoplasm and cytoplasm may in part explain why these cellular compartments are only poorly visible in immuno-fluorescent staining of exosome component comparison to the nucleolus. Nevertheless, several biochemical fractionation experiments have shown that, although hardly visible in immuno-fluorescence, there is a significant cytoplasmic pool of exosome complexes. Both the nuclear and the cytoplasmic pool of exosomes can be fractionated in different complexes, most likely reflecting the exosome in its different cytoplasmic functions (Brouwer *et al.*, 2001b; Schilders *et al.*, 2005).

Currently, it is not exactly known which elements determine nuclear import of the exosome (components). Most likely, the core of the exosome assembles in the cytoplasm and is subsequently transported to the nucleus, where it accumulates in the nucleolus. Putative nuclear localization signals have been identified in the sequences of PM/Scl-75 and hRrp41p. Although the putative nuclear localization signal of PM/Scl-75 was shown to be able to direct a reporter protein to the nucleus, this element appeared to be particularly important for its accumulation in the nucleolus (Rajmakers *et al.*, 2003). An alternative possibility is that partially assembled parts of the exosome core enter the nucleus and assemble into a complete exosome core in the nucleoplasm or nucleolus. Nuclear exosome cofactors are probably independently imported into the nucleus and associate with the exosome core in nucleoplasm or nucleolus. Indeed, putative nuclear localization signals are found in the amino acid sequences of most of these proteins.

Several proteins involved in $5' \rightarrow 3'$ mRNA decay were found to localize to so-called “processing bodies” or “GW bodies.” Therefore, the question was

raised whether these cytoplasmic foci are related to mRNA decay pathways such as nonsense-mediated decay (NMD), ARE-mediated decay, and RNA interference. Indeed, several studies in both yeast and human cells have suggested that processing bodies are directly involved in RNA interference, microRNA function, and mRNA turnover (Fillman and Lykke-Andersen, 2005). The discovery of these cytoplasmic foci also raised the question of where the cytoplasmic exosome exerts its function because it does not appear to localize to the processing bodies. Most interestingly, it was demonstrated in *Drosophila melanogaster* cells that both the Ski-complex and exosome are required for the degradation of the 5′ intermediates generated by the RNA Induced Silencing Complex (RISC), whereas the 3′ mRNA fragments are degraded by Xrn1 which is known to localize to the processing bodies (Orban and Izaurralde, 2005). Although this study provides a link between 5′ → 3′ and 3′ → 5′ turnover of mRNAs generated by RISC, it is unclear whether exosome-dependent decay of the 5′ mRNA fragments occurs all over the cytosol or in a subset of alternative mRNA processing bodies. A detailed study of the precise subcellular localization of the exosome core might provide more insight into these mechanisms.

## Exosome Cofactors

### a. Human PM/Scl-100 versus Yeast Rrp6p

In yeast, localization studies with the exosome associated 3′ → 5′ exonuclease Rrp6p resulted in a punctate nucleoplasmic staining with a nuclear enrichment. Also biochemical fractionation experiments have demonstrated that Rrp6p is a nuclear protein (Allmang et al., 1999b). For the human homologue of Rrp6p, PM/Scl-100, there is still some debate whether this protein is restricted to the nucleus or also occurs in the cytoplasm. One study detected PM/Scl-100 solely in a nuclear extract (Allmang et al., 1999b), but there are several other reports which describe that PM/Scl-100 is present in the cytoplasm as well (Brouwer et al., 2001b; Chen et al., 2001). The most convincing evidence is an indirect immuno-fluorescence experiment using anti-PM/Scl-100 antibodies showing a fraction of PM/Scl-100 resides in the cytoplasm and the observation that knock-down of PM/Scl-100 results in a defect in cytoplasmic NMD (Lejeune et al., 2003). In conclusion, it is very likely that there is a subset of cytoplasmic PM/Scl-100 containing exosome complexes involved in human mRNA surveillance.

### b. The Ski-Complex in Human and Yeast

In addition to the exosome, cytoplasmic 3′ → 5′ mRNA decay requires the Ski-complex consisting of Ski2p, Ski3p, Ski7p, and Ski8p (Jacobs Anderson and Parker, 1998; Van Hoof et al., 2000b). Three of these proteins, Ski2p, Ski3p, and Ski8p, are known to form a trimeric complex, which resides exclusively in the cytoplasm

of the yeast *S. cerevisiae* (Brown *et al.*, 2000; Van Hoof *et al.*, 2000b). The Ski-complex has also been described in human cells. In contrast to the yeast Ski-complex, the human Ski-complex localizes to both nucleus and cytoplasm and is associated with the nuclear PAF complex. The PAF complex plays a key role in setting histone marks for active transcription, and is also required for maintaining proper poly(A) tail length. These data suggest that, in contrast to the yeast complex, the human Ski-complex might have an additional nuclear role in mRNA surveillance (Zhu *et al.*, 2005).

## III. Functions of the Exosome

In recent years is has become clear that the exosome has an important role in a wide variety of biological processes, all concerning the maturation or degradation of RNA. The important biological role of the exosome is substantiated by the observation that all yeast core exosome subunits are essential for viability. As previously described, the exosome resides in the cytoplasm, the nucleoplasm, and the nucleolus of eukaryotic cells and functions in the processing, surveillance, or complete degradation of a variety of different RNA species. The yeast exosome components, Rrp4p, Rrp41p, Rrp44p/Dis3p, and Rrp6p, the human exosome proteins OIP2 and hRrp4p, as well as the *Arabidopsis thaliana* exosome subunit AtRrp4p have been shown to display 3′ → 5′ exoribonuclease activity *in vitro* (Burkard and Butler, 2000; Jiang and Altman, 2002; Mitchell *et al.*, 1997). If all of these proteins also contain this activity *in vivo*, this implies that the exonucleolytic degradation of substrate RNAs is not restricted to the processing chamber formed by the six RNase PH-type subunits (see earlier discussion), but is also a property of the S1-type core subunits and associated factors. Several studies have provided more insight into the biological functions of the exosome. Most of these studies have been performed in yeast, but similar functions are believed to apply to the exosome complexes of other organisms due to the high level of homology between the exosome complexes and the essential nature of most substrate RNAs.

### A. rRNA Processing

The biogenesis of ribosomes has been relatively well studied in yeast and requires the processing of precursor-ribosomal RNAs (pre-rRNAs) that are transcribed by RNA polymerase I from the ribosomal DNA repeat genes. Three of the four rRNAs are transcribed as a single precursor RNA, which contains two internally transcribed spacers (ITS1 and ITS2) and two
externally transcribed spacers (5′-ETS and 3′-ETS), which are removed by a complex series of endonucleolytic and exonucleolytic processing steps to produce the mature 18S, 5.8S, and 25S rRNAs (Fig. 3A). Processing initiates with endonucleolytic cleavage events at sites A0, A1, and B2, leaving excised fragments of the 5′ ETS and 3′ ETS. Subsequently, cleavage at site A2 results in the separation of 20S pre-rRNA and 27SA2 pre-rRNAs. Cleavage at A3 or B1L provides distinct pathways for the formation 5.8S<sub>S</sub> rRNA or 5.8S<sub>L</sub> rRNA, respectively. In both of these pathways, cleavage at C2 results in a 7S precursor, which is processed to the mature 5.8S rRNA.

Yeast strains lacking either exosome core subunits or specific exosome associated proteins (Dis3p, Dblp/Mtr4p, Nop53p, Gsp1p/Ran) accumulate 5.8S rRNAs that are extended at the 3′ end ([Allmang et al., 1999a; de la Cruz et al., 1998; Suzuki et al., 2001; Zanchin and Goldfarb, 1999b]). However, the 3′ end of these 5.8S rRNA precursors varies between different mutants indicating that they do not have identical functions. Especially mutants of the yeast exosome-associated components Rrp6 and Rrp47p revealed a distinct defect in pre-rRNA processing; with these mutants, accumulation of 5.8S rRNA molecules that are 3′ extended by only ~30 nucleotides was observed, while mutations in core exosome components led to the accumulation of larger processing intermediates. This suggests that the core exosome components normally act upstream of Rrp6p and Rrp47p. Based upon these observations, it was proposed that for efficient 5.8S rRNA processing is characterized by a hand-over mechanism in which the core of the exosome degrades most of the 3′ end of the 7S pre-rRNA, after which the remaining nucleotides are removed by Rrp6p, possibly assisted by Rrp47p ([Briggs et al., 1998; Mitchell et al., 2003; Phillips and Butler, 2003]). In other organisms, such as *T. brucei* and human, the exosome has also been shown to be implicated in 5.8S rRNA processing ([Estevez et al., 2001; Stoecklin et al., 2005]).

Surprisingly, upon knock-down of the human PM/Scl-100 and hRrp41p proteins the same 5.8S rRNA precursors accumulated, suggesting that either the hand-over mechanism is not relevant in the human system or that their functions are more tightly coupled ([Schilders et al., 2005]). The expression of the *A. thaliana* Rrp41 protein in yeast strains lacking Rrp41p rescued 5.8S rRNA processing defects, suggesting that also in this organism the exosome is involved in rRNA processing ([Chekanova et al., 2000]).

Besides its involvement in 5.8S rRNA maturation it has been shown in yeast that exosome depletion leads to the accumulation of a 5′ ETS fragment and accumulation of incompletely or incorrectly processed rRNA precursors. Upon depletion of one of the core exosome components, inhibition of rRNA cleavage at sites A<sub>0</sub>, A<sub>1</sub>, A<sub>2</sub>, and A<sub>3</sub> leads to the accumulation of 35S and 27S pre-rRNA and reduced levels of the mature 18S and 25S rRNA. Additionally, the processing inhibition results in the formation of the
182

SCHILDERS ET AL.

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 3 Pre-rRNA processing by the exosome in *S. cerevisiae*. The 35S pre-rRNA is transcribed by RNA polymerase I as a single precursor and contains two external transcribed spacers (5′-ETS and 3′-ETS) and two internal transcribed spacers (ITS1 and ITS2), which are removed by several processing steps yielding the mature 18S, 5.8S, and 25S rRNAs. The exosome is

aberrant pre-rRNAs termed 23S*, 21S*, and A₂-C* , which are in wild-type cells degraded by the exosome complex (Fig. 3B) (Allmang *et al.*, 2000; Zanchin and Goldfarb, 1999b). Taken together, these observations suggest that the exosome may participate in a quality control mechanism for pre-rRNA processing, which only allows correctly processed pre-rRNAs to be incorporated into ribosomal subunits.

A yeast strain lacking Rrp6p was found to accumulate polyadenylated rRNAs which reside mostly in the nucleus. Interestingly, this polyadenylation occurred preferentially on pre-rRNAs. This observation led to the identification of the TRAMP complex, consisting of the poly(A) polymerase Trf4p, one of the putative RNA binding proteins Air1p or Air2p, and the RNA helicase Mtr4p, which was already known to function in rRNA processing in concert with the exosome. This complex is able to stimulate exosome-dependent RNA processing by adding a poly(A) tail to its substrate RNAs (LaCava *et al.*, 2005; Wyers *et al.*, 2005).

## B. Small RNA Processing

In the nucleoplasm, a number of small nuclear RNAs (snRNAs) are involved in the splicing of pre-mRNA. The primary transcripts of these sn RNAs are post-transcriptionally processed to produce functional snRNAs (Perumal

involved in several of these steps either directly or indirectly. (A) Endonucleolytic cleavage at site A₀ in the 5′ ETS results in the 33S pre-rRNA and the 5′ part of the 5′ ETS which is completely degraded by the exosome. Cleavage at the A₁ site generates the 32S pre-rRNA, which is followed by the cleavage at the A₂ site leading to the 20S and 27SA₂ pre-rRNAs. The 20S pre-rRNA is further processed by cleavage at site D to generate mature 18S rRNA. The 5′ end of the 27SA₂ pre-rRNA can be processed by two alternative pathways concerning ITS1 processing, which results in either a short or a long form of 5.8S rRNA. Because the exosome is involved in processing of this rRNA precursor in ITS2 and this is the same for both ITS1 processing pathways, only the major pathway, leading to maturation of the short form of 5.8S rRNA, is included here. The 27SA₂ pre-rRNA is cleaved at site A₃, which yields the 27SA₃ pre-rRNA. This molecule is then processed by a 5′ → 3′ exonuclease to site B₁(S) concomitant with cleavage at site B₂ at the 3′ end of 25S rRNA, followed by cleavages at sites C₁ and C₂ resulting in the mature 25S rRNA and the 7S precursor rRNA. 3′ end processing of the 7S pre-rRNA by the exosome, generates the mature 5.8S rRNA. (B) Mutation of one of the exosome components results in the inhibition of cleavage at either sites A₀, A₁, and A₂ (1) or at site A₃ (2). Note that after exosome depletion these processing pathways are not completely blocked. (1) Inhibition of cleavage sites A₀–A₂ results in the production of 23S pre-rRNA, which is subsequently converted to 23S* pre-rRNA and 21S* pre-rRNA. The latter is formed by cleavage of the 23S* pre-rRNA at site A₁. (2) Inhibition of cleavage at site A₃ results in a pre-rRNA that extends from site A₂ to C₂. In the absence of a functional exosome complex A₂-* pre-rRNAs accumulate, which have heterogeneous 3′ ends between C₂ and the 3′ end of the 5.8S rRNA.

and Reddy, 2002). For the U1, U4, and U5 snRNAs it has been shown that their 3′ end processing can be performed by the exosome and the associated factors Mtr4p/Dob1p and Rrp47p (Mitchell et al., 2003; Peng et al., 2003). However, processing is not blocked if individual components of the exosome are mutated (Allmang et al., 1999a; Van Hoof et al., 2000a). In addition, the exoribonucleases Rex1p, Rex2p, and Rex3p have been shown to affect the 3′ end processing of the U4 snRNA, which was initially thought to be performed exclusively by the exosome. This observation can be explained in two ways. One possibility is that the exoribonucleases function in different, parallel processing pathways and the second possibility is that multiple exonucleases are involved in the same pathway of 3′ end processing.

The small nuclear RNAs (snoRNAs) can be divided in two major classes: the box C/D snoRNAs, functioning in the methylation of 2′-hydroxyl groups in substrate RNAs, and the box H/ACA snoRNAs, which are responsible for the conversion of uridine into pseudouridine in substrate RNAs. In eukaryotes, both 5′ and 3′ processing of pre-snoRNAs involves exonuclease activities (Kiss, 2002). In yeast, initial processing of the 3′ end of a wide variety of snoRNAs requires the core exosome as well as Rrp6p, although processing is not completely blocked if either the core exosome or Rrp6p is mutated and the effect is enhanced if both are deleted. In addition, depletion of Rrp6p leads to the accumulation of several snoRNAs that are 3′ extended by three nucleotides, whereas depletion of other exosome components does not affect the trimming of these last three nucleotides. This suggests that only Rrp6p is required for the final trimming of these snoRNAs (Allmang et al., 1999a; Kufel et al., 2000). As for snRNAs, 3′ end processing of intron-derived, polycistronic and independently transcribed snoRNAs appeared to require not only the core exosome but also the nuclear exosome specific cofactors Mtr4p/Dob1p and Rrp47p. Although the abrogation of Rrp47p function leads to similar processing defects as Rrp6p depletion with regard to sn(o)RNAs, Rrp47p is not essential for the Rrp6p-dependent degradation of 3′-extended nuclear pre-mRNAs. Therefore, Rrp47p was proposed to function as a substrate-specific nuclear cofactor for exosome activity in the processing of small nuclear RNAs and rRNAs (Mitchell et al., 2003; Peng et al., 2003).

In the absence of Rrp6p, Mtr4p, or core exosome components, polyadenylation of at least some small nuclear RNAs occurs. Some of these polyadenylated RNA species were also observed at low levels in wild-type strains suggesting that they are normal intermediates in the biogenesis of small nuclear RNAs. Like for rRNA processing, it was found that the TRAMP complex is required for normal processing of the U14 snoRNA and the U5 snRNA by the exosome (LaCava et al., 2005). In addition to this role of the exosome and the TRAMP complex in the maturation of small nuclear RNAs, both complexes are involved in a surveillance pathway in which

hypomodified tRNAs are polyadenylated by the TRAMP complex and subsequently degraded by the exosome. However, the manner in which the TRAMP and exosome complexes can discriminate between normal tRNAs and tRNAs that are not properly processed is not yet fully understood, but most likely involves proper folding of the processed tRNAs (*Kadaba et al.*, 2004; *Vanacova et al.*, 2005). Another link between the exosome and tRNA processing has been suggested by a yeast two-hybrid interaction between the core exosome component OIP2 and Rpp14, a component of the RNase P complex. Both proteins were shown to possess exonuclease activity on precursor tRNA substrates, which was also present in immunoprecipitated RNase P complexes, suggesting that exonucleolytic activity may be one of the mechanisms to generate mature tRNA 3′ ends (*Jiang and Altman*, 2002).

Yet another structural RNA species that appears to be generated through exosome-mediated processing is the RNA component of the signal recognition particle (SRP). The SRP consists of the SRP RNA and four SRP proteins and is assembled in the nucleolus of eukaryotic cells. The yeast exosome component Rrp44p/Dis3p has been shown to contribute to the 3′ end-trimming of SRP RNAs because mutations in Rrp44p led to aberrant 3′ end processing and nucleolar accumulation of the SRP. Whether other components of the exosome are also involved in the processing of the SRP RNA is not known (*Grosshans et al.*, 2001).

## C. mRNA

### 1. mRNA Turnover in the Nucleus and Cytoplasm

Steady-state mRNA levels are determined by the balance between the rates of synthesis (transcription) and degradation of mRNA molecules. While the process of transcription has been extensively studied, relatively little is known about the mechanisms and regulation of mRNA degradation. Multiple pathways of mRNA degradation were uncovered, and several factors involved have been identified. For the degradation of mature, wild type mRNAs, two main cytoplasmic pathways have been identified, one of which involves the exosome as the main degradation machinery. In addition, the exosome has been implicated in several nuclear and cytoplasmic pathways that have been shown to exist for the decay of aberrant messengers (Fig. 4).

### 2. Nuclear mRNA Turnover

In 1998, the first evidence was presented for a nuclear mRNA degradation pathway in yeast; mutations in Rrp6p were identified as suppressors of polyadenylation defects due to a mutation in Pap1p (*Briggs et al.*, 1998).

Rrp6  
3′-5′ Degradation by  
nuclear exosome  

Nuclear retention  
Incorrect  
splicing  

Export to cytoplasm  
Failure to assemble  
complete spliceosome  

RNA polymerase  
Transcription  

Extended 3′ end  

m7G Capping  
m7G Splicing  
m7G AAA Polyadenylation  

Rescue of mRNA by  
nuclear exosome core  

Hand-over to Rrp6  
and 3′-5′ degradation  

Retention at  
transcription site  

Export to cytoplasm  

Nucleus  

Cytoplasm  

First round of translation  

Premature  
stop codons No stop codons  

m7G STP STP STP AAA  

m7G STP AAA Correct  
messenger  

Nonsense mediated  
decay (NMD)  

Decapping Deadenylation  

5′-3′ Degradation 3′-5′ Degradation by  
cytoplasmic exosome  
and Ski-complex  

Ski3  
Ski8  
Ski2  
Ski7  

Multiple rounds of translation  

AU-rich elements (AREs)  

Regular turnover  

Nonstop  
decay  

Deadenylation and degradation  
by the cytoplasmic exosome  

Recruitment of exosome  
via AU binding proteins  

Degradation by the  
cytoplasmic exosome  

FIG. 4 The role of the exosome in mRNA surveillance and turnover. An overview of mRNA surveillance by the exosome. When errors occur during mRNA biogenesis, different pathways can be activated in which the exosome degrades the aberrant messenger. Aberrant 3′ end

STRUCTURE AND FUNCTION OF THE EXOSOME 187

Under nonpermissive conditions, the temperature-sensitive mutation *pap1-1* causes a defect in polyadenylation, which results in the inability of the cells to produce significant quantities of most mRNAs. Mutations that inactivate Rrp6p partially suppress this defect and result in increased levels of poly(A)+ mRNA (Burkard and Butler, 2000). This finding led to the hypothesis that Rrp6p, probably as a component of the nuclear exosome, competes with mRNA 3′ end processing and destroys slowly or improperly processed pre-mRNAs (Burkard and Butler, 2000). Indeed, analysis of the fate of unspliced pre-mRNAs that accumulate in strains with *cis*- or *trans*-acting splicing defects revealed enhanced levels of unspliced transcripts in the presence of defective core exosome components or in the absence of Rrp6p (Bousquet-Antonelli et al., 2000). In agreement with these observations, Hilleren and co-workers have shown that properly polyadenylated transcripts are sequestered at or near transcription sites in an exosome-dependent manner (Hilleren et al., 2001) (Fig. 4).

Interestingly, not all aberrant pre-mRNAs appear to be retained in the nucleus and degraded in this compartment. Another study (Hilleren and Parker, 2003) indicates that yeast pre-mRNAs containing specific *cis*-acting mutations that either prevent spliceosome assembly or trap the pre-mRNA at the second catalytic step in splicing are targeted to a cytoplasmic mRNA decay pathway. In this pathway, splice-defective lariat intermediates are debranched by an endoribonuclease called Dbr1p and subsequently degraded 5′ → 3′ by the cytoplasmic exonuclease Xrn1p. Alternatively, when debranching is blocked, these splicing intermediates can be degraded in a 3′ → 5′ direction in a manner dependent on the cytoplasmic exosome.

While the bulk of mature wild-type mRNAs appear to be degraded in the cytoplasm (see later), it has been shown that normal mRNAs can also be degraded in the nucleus. A yeast strain with a defect in mRNA export from the nucleus showed nuclear degradation of wild-type mRNAs. This degradation was dependent on Rrp6p (which is not required for cytoplasmic mRNA decay), Rat1p, a nuclear 5′ → 3′ exoribonuclease, and the nuclear cap-binding complex (CBC).

formation will lead either to rescue of the mRNA by the nuclear exosome or to complete degradation in an Rrp6-dependent manner. Errors in splicing can lead to degradation by the nuclear exosome, but the mRNA can also be exported to the cytoplasm where it is then degraded by the regular mRNA turnover machinery. After transport to the cytoplasm, the mRNA is subjected to nonsense-mediated decay (NMD) if translation terminates prematurely. Alternatively, if the normal stop codon is lacking, translation proceeds and ribosomes stall at the 3′ end of the mRNA, which will activate non-stop decay. If none of these errors are encountered, mRNAs will be translated properly and eventually degraded through regular turnover pathways. The lifespan of the latter mRNAs differs and is in part dependent on the presence of (de)stabilizing elements, such as AU-rich elements (AREs), which recruit the mRNA degradation machinery including the exosome, thereby enhancing their turnover rates.

3. Cytoplasmic mRNA Turnover

In yeast, two main pathways have been identified for the degradation of mature wild-type mRNA molecules; in both of these pathways, the initial decay event is the shortening of the poly(A) tail to an oligo(A) length of 10 to 12 nucleotides (Decker and Parker, 1993; Muhlrad *et al.*, 1995). After poly (A) shortening, transcripts can be further degraded by either the $5' \rightarrow 3'$ pathway or the $3' \rightarrow 5'$ decay pathway. In the $5' \rightarrow 3'$ pathway, deadenylation triggers mRNA decapping by the Dcp1p/Dcp2p decapping enzyme complex, after which the mRNA is exonucleolytically digested by the $5' \rightarrow 3'$ exoribonuclease Xrn1p (Beelman *et al.*, 1996; Dunckley and Parker, 1999; Hsu and Stevens, 1993; Muhlrad *et al.*, 1994). In the $3' \rightarrow 5'$ mRNA decay pathway, deadenylation is followed by degradation of the mRNA body by the exosome up till a residual capped oligonucleotide of less than ten nucleotides is left, which is then hydrolyzed by a decapping enzyme, DcpS (for scavenger decapping enzyme) (Wang and Kiledjian, 2001).

The first step in mammalian mRNA decay also appears to be poly(A) shortening (Shyu *et al.*, 1991). The first evidence for the existence of a subsequent decapping-dependent $5' \rightarrow 3'$ mRNA decay pathway in mammalian cells came from the detection of deadenylated and decapped mRNAs in mammalian cell extracts (Couttet *et al.*, 1997). Human homologues of the yeast mRNA decapping factors Dcp1 and Dcp2 have been identified (Lykke-Andersen, 2002; Van Dijk *et al.*, 2002; Wang *et al.*, 2002). The use of an *in vitro* mRNA decay system provided evidence that $3' \rightarrow 5'$ degradation is mediated by the mammalian exosome (Chen *et al.*, 2001; Mukherjee *et al.*, 2002; Wang and Kiledjian, 2001). Lejeune and co-workers provided evidence that both the $5' \rightarrow 3'$ and the $3' \rightarrow 5'$ decay pathways are involved in the degradation of mRNAs that contain premature translation codons in the specialized surveillance pathway called nonsense-mediated decay (NMD) (Lejeune *et al.*, 2003).

Having identified the main pathways of mammalian mRNA turnover, an interesting question that remains is how these pathways may be regulated. It is known that mRNA half-lives can vary from minutes to days and it has been established that many mRNAs contain sequence elements in their $3'$-untranslated regions ($3'$-UTR) that influence their stability (Parker and Song, 2004). Among these, a widespread class of AU-rich elements (AREs) are responsible for targeting the transcripts for rapid decay (Chen and Shyu, 1995). Numerous proteins have been identified that specifically interact with AREs, some of which appear to stabilize (Ford *et al.*, 1999) or destabilize mRNAs (see Table II) (Loflin *et al.*, 1999). A clue to how these RNA-binding proteins may communicate with the mRNA turnover machinery was provided by a study which indicated that the ARE-binding protein tristetraprolin (TTP) and its homologue BRF-1 co-immunopurify with factors from

the $5^{\prime} \rightarrow 3^{\prime}$ mRNA decay pathway as well as with the exosome (Lykke-Andersen and Wagner, 2005). These data suggest that ARE-binding proteins may activate both $5^{\prime} \rightarrow 3^{\prime}$ mRNA decay as well as $3^{\prime} \rightarrow 5^{\prime}$ mRNA decay. Interestingly, AREs may also recruit the exosome directly, because the exosome subunit PM/Scl-75 has been reported to directly interact with AREs (Mukherjee *et al.*, 2002).

In yeast, it has been shown that $3^{\prime} \rightarrow 5^{\prime}$ mRNA decay requires, besides the exosome, also the Ski-complex, consisting of Ski2p, Ski3p, Ski7p, and Ski8p (Jacobs Anderson and Parker, 1998; Van Hoof *et al.*, 2000b). Mutations in the SKI genes are synthetically lethal with loss of Xrn1p, an essential component of the $5^{\prime} \rightarrow 3^{\prime}$ mRNA decay pathway. This suggested that the SKI genes have a general function in regulating gene expression (Johnson and Kolodner, 1995). Importantly, Ski4p (Csl4p) and Ski6p (Rrp41p) are core components of the exosome. As mentioned previously, Ski2p, Ski3p, and Ski8p assemble into a trimeric complex (Brown *et al.*, 2000), which interacts with the exosome through Ski7p (Van Hoof *et al.*, 2000b). Homologues of yeast Ski2p, Ski3p, and Ski8p have been identified in *Drosophila melanogaster* and human cells, and for the human Ski homologues, it has been shown that they also assemble into a complex (Orban and Izaurralde, 2005; Zhu *et al.*, 2005). No homologues of yeast Ski7p have been identified so far in other organisms. Furthermore, it remains to be established whether the human Ski-complex plays a role in mRNA degradation.

## Translation-Dependent mRNA Quality Control

In addition to the aforementioned nuclear pathways to degrade improperly processed pre-mRNAs, eukaryotes have evolved surveillance mechanisms that are intimately linked to translation to eliminate errors in mRNA biogenesis. One of these pathways serves to eliminate mRNAs with premature termination codons that might otherwise encode truncated proteins that interfere with normal cellular processes, thus acting as natural “dominant negatives” (Weischenfeldt *et al.*, 2005). This process is termed nonsense-mediated decay (NMD), and its mechanism has been studied in some detail. Three conserved proteins, Upf1, Upf2, and Upf3, play critical roles in recognizing mRNAs with premature termination codons. The Upf proteins, which form a complex in yeast, are believed to identify the premature termination codon in conjunction with the translational release factors eRF1 and eRF3 (Czaplinski *et al.*, 1998; Ishigaki *et al.*, 2001). This process is also splicing-dependent, as the Upf proteins recognize exon junction complexes (present at the site where exons are spliced together) that normally are removed during the initial round of mRNA translation. Upon recognizing a premature termination codon, the NMD machinery can directly activate
mRNA decapping, thereby bypassing the requirement for prior deadenylation; alternatively NMD can lead to accelerated deadenylation and degradation by the exosome ([Mitchell and Tollervey, 2003](https://doi.org/10.1093/nar/gkg587)). In human cells, NMD also requires the exosome, as evidenced by the observation that downregulation of the PM/Scl-100 subunit of the human exosome leads to elevated levels of a nonsense transcript ([Lejeune et al., 2003](https://doi.org/10.1093/emboj/cdg468)). Furthermore, human homologues of the yeast Upf factors were found to co-immunopurify with the PM/Scl-100, hRrp4p, and hRrp41p subunits of the human exosome in this study. However, human Dcp2 also appears to be involved in NMD in human cells ([Lejeune et al., 2003](https://doi.org/10.1093/emboj/cdg468)). Thus, NMD appears to require both $5' \rightarrow 3'$ as well as $3' \rightarrow 5'$ mRNA degradation in yeast as well as in mammalian cells.

An additional translation-dependent pathway to degrade aberrant transcripts was discovered; this pathway serves to eliminate mRNAs lacking termination codons and is therefore called “nonstop decay” ([Frischmeyer et al., 2002](https://doi.org/10.1073/pnas.252628699); [Van Hoof et al., 2002](https://doi.org/10.1073/pnas.252628699)). In an mRNA that lacks a termination codon, the ribosome is predicted to continue translation into the 3′ UTR and the poly(A) tail and eventually to stall at the 3′ end of the mRNA, or until it encounters an in-frame termination codon in the UTR. In all cases, this would lead to proteins containing additional, aberrant sequences that might interfere with normal protein function and cellular processes. Interestingly, nonstop decay in both yeast and mammalian cells requires neither the major deadenylase nor players in either the $5' \rightarrow 3'$ or the NMD pathways. Instead, it involved cytoplasmic exosome-mediated decay that begins at the 3′ poly(A) tail. Thus, the exosome degrades both the poly(A) tail and the body of the messenger, which is surprising given that normal mRNAs cannot be deadenylated by the exosome ([Tucker et al., 2001](https://doi.org/10.1073/pnas.252628699)). Possibly, this can be explained if one assumes that poly(A) binding proteins that would normally protect the poly(A) tail from degradation are displaced by scanning ribosomes entering the 3′ UTR and poly(A) tail of the messenger.

### D. Degradation of Cryptic Unstable Transcripts

Cryptic unstable transcripts are RNA polymerase II transcripts mapping to intergenic regions of the genome. These transcripts are unstable and do not contain long or conserved reading frames and are thought to be present only at extremely low levels in wild-type cells. However, in yeast strains lacking Rrp6p, it was observed that polyadenylated forms of these RNAs accumulate. This suggests that the exosome together with the TRAMP complex participates in a surveillance pathway preventing the accumulation of inappropriate transcripts of intergenic regions, most likely by rapid degradation of these polyadenylated transcripts by the exosome ([Wyers et al., 2005](https://doi.org/10.1073/pnas.252628699)).
STRUCTURE AND FUNCTION OF THE EXOSOME

E. Anti-Viral Protection

In yeast, loss of function mutations in one of the eight SKI genes (see earlier) gives rise to the *super killer* phenotype, which is characterized by an enhanced replication of dsRNA viruses in infected *ski* strains compared to infected wild-type strains (*Ridley et al.*, 1984). It is now known that Ski6p (Rrp41p) and Ski4p (Csl4p) are core components of the yeast exosome, whereas Ski2p, Ski3p, Ski8p, and Ski7p form the Ski-complex which is functionally linked to the cytoplasmic exosome. Skilp is not associated with one of these complexes, but is identical to the $5' \rightarrow 3'$ exonuclease Xrn1p, and Ski5p proved to be identical to Ski3p (*Brown et al.*, 2000; *Hougan et al.*, 1989; *Wang et al.*, 2005). Two models have been proposed to explain the enhanced expression of dsRNA viruses. First, the Ski proteins may regulate the cellular machinery in a way that favors the translation of capped, adenylated mRNAs over uncapped, unadenylated viral transcripts. Therefore, loss of *SKI* function would lead to an increased production of virus particles (*Benard et al.*, 1998). A second and more likely explanation is that the Ski-complex recruits the viral RNA to the exosome complex for efficient degradation. In *SKI* mutants this degradation pathway will be less efficient causing the super killer phenotype. Finally, an interferon stimulated gene (ISG20, homologous to the yeast $3' \rightarrow 5'$ exoribonuclease Rex4p), capable of protecting cells from infection with RNA viruses, has been found to associate with the human exosome (Dr. N. Mechti, personal communication) (*Espert et al.*, 2003). Taken together, these data suggest the exosome complex plays an important role in the anti-viral response in eukaryotic cells by degrading virus-derived RNA species.

IV. Regulation of the Exosome

The involvement of the exosome in so many different processes raised the question how this complex can distinguish between RNA molecules which are to be degraded and those which have to become mature, functional RNAs. In principle every RNA molecule is a potential substrate for the exosome; therefore, additional factors most likely allow the complex to identify its proper RNA substrates. In **Figure 5**, several ways in which the activity of the complex can be regulated are shown. First of all, secondary and higher order structures present in RNA molecules might prevent their entry into the pore and/or neck of the ring (see earlier mention), inhibiting (further) degradation of the RNA. Additionally, other proteins can associate with the exosome core or the RNA, to either prevent or stimulate degradation of an RNA substrate. Indeed, several studies have now demonstrated

Inhibition by stable  
secondary structures  
in the RNA substrate  

Proteins that bind RNA and  
prevent further degradation  

Core exosome  
activity  

Proteins that actively contribute  
to substrate degradation or  
processing, e.g. Rrp6/PM/Scl-100  

Proteins that recruit or unwind  
RNA and promote degradation  
e.g. Ski and TRAMP complexes  
and specific ARE binding proteins  

FIG. 5 Regulation of the exoribonucleolytic activity of the core exosome. The $3'$ end of a  
substrate RNA most likely enters the central channel via the pore formed by the S1 domains  
and is exonucleolytically degraded at the active sites of RBD2 subunits in the processing  
chamber (central image). The S1 domain containing subunits also contain KH or zinc-ribbon  
(ZN) domains, which are positioned at the periphery of the ring formed by these subunits and  
are thus ideally situated for substrate and/or cofactor interactions. The various ways by which  
$3' \rightarrow 5'$ degradation of substrate RNAs by the core exosome can be regulated are schematically  
illustrated: inhibition of degradation by stable secondary structures in the RNA (upper left  
corner); inhibition of degradation by proteins bound to the substrate RNA (upper right corner);  
additional degradation/processing of (remaining) substrate RNA (fragments) by exosome-  
associated exoribonucleases (lower left corner); enhancement of the efficiency of degradation by  
proteins recruiting substrate RNA molecules to the exosome and/or by helicases unwinding  
secondary structures in the RNA that has to be degraded (lower right corner).

interactions of the exosome with cofactors which can facilitate and stimulate the degradation of RNA by the exosome. For example the putative RNA helicases Mtr4p, Ski2p, and RHAU might destabilize RNA-protein interactions or disrupt secondary structures making the RNA available for entry into the processing chamber of the exosome core. The TRAMP complex subunit Trf4p adds a poly(A) tail to nuclear RNA substrates making them suitable for efficient 3′ degradation by the exosome. The specificity of exosome-mediated RNA degradation might be provided by RNA-binding proteins, such as Air1p, MPP6, and Rrp47p. Protection of an RNA from degradation by the exosome could be mediated by the binding of proteins to its 3′ end. Alternatively, proteins binding to other regions of the RNA may interfere with the recruitment of the exosome, such as the ARE-binding protein HuR. RNA-binding proteins may also stabilize substrate RNAs, thereby preventing its degradation. The naked 7S pre-rRNA, for example, can be fully degraded by the exosome core, but is only partially processed *in vivo*, most likely because of ribosomal proteins bound to the RNA.

### A. Substrate Specificity Mediated by Exosome Core Components

#### 1. PM/Scl-75

By immunodepletion of the exosome component PM/Scl-75 it was first demonstrated that the human exosome complex is required for efficient 3′ → 5′ exonucleolytic decay of mRNAs containing AREs in the 3′ UTR. It is known that the PM/Scl-75 protein contains a domain which is homologous to the *Escherichia coli* RNase PH exonuclease, but no other known protein motifs can be discerned in its primary structure. Possibly the relatively large, highly charged C-terminal domain this protein, of which the function is currently unknown, is involved in this activity. Interestingly, PM/Scl-75 has been reported to interact directly and specifically with the AREs of such mRNAs *in vitro*. This led to the hypothesis that PM/Scl-75 might selectively interact with ARE-mRNAs providing substrate specificity to the exosome. Additional ARE-binding proteins might activate the exosome and/or assist in its recruitment to ARE-containing mRNAs via protein–protein interactions (Mukherjee et al., 2002).

#### 2. Csl4p

Another exosome component, Csl4p, contains a putative S1 RNA-binding domain and might play a role in distinguishing between different RNA substrates. Characterization of Csl4p in yeast revealed a point mutation in

the S1 domain that leads to defects in cytoplasmic $3' \rightarrow 5'$ mRNA degradation but not in 5.8S rRNA processing. Interestingly, another mutation in Csl4p did affect rRNA processing but not mRNA degradation. This observation supports the idea that the S1 domain of Csl4p plays a role in the selection of substrate RNAs. In another study, knock-down of the $TbCSL4$ and $TbRRP4$ proteins of the parasite *Trypanosoma brucei* demonstrated that these subunits are not required for mRNA degradation, whereas knock-down of the $TbRRP45$ did affect mRNA degradation (Haile et al., 2003; Van Hoof et al., 2000b).

## RNA Specificity of the Exosome

In archaeabacteria, the exosome has been shown to interact with the four terminal 3′ nucleotides of its substrate RNA via direct interactions with both the Rrp41p and Rrp42p subunits (the only two RNase PH subunits in the archaeal exosome). The catalytic Rrp41p subunits interact mainly with the terminal two nucleotides, whereas the non-catalytic Rrp42p protein interacts primarily with the nucleotides at the third and fourth position from the end of the RNA. Most of these interactions involve either the phosphate moiety or the 2′ hydroxyl group of these nucleotides, but the bases do not seem to be important, explaining the preference of the exosome for RNA over DNA and its lack of sequence specificity (Lorentzen and Conti, 2005). The three different active sites present within one archaeal exosome complex do not all interact with the substrate RNA at the same time, but may contribute to the highly processive activity on unstructured RNA molecules. In addition, the RNA-binding proteins Rrp4p and Csl4p, and in eukaryotes probably also Rrp40p, have their S1 RNA-binding domains located toward the central channel, assisting the core of the complex in binding RNA, whereas the KH domains are located at more peripheral positions, and are possibly involved in the recruitment of substrate RNAs (Buttner et al., 2005).

### B. Regulation by Exosome-Associated Components

#### 1. tRNA Processing

The Trf4 (TRAMP) complex, which cooperates with the exosome in nuclear RNA processing, can polyadenylate immature, unmodified tRNAs more efficiently than correctly processed tRNAs, suggesting that it can distinguish between these two classes of RNA. It was proposed that the TRAMP complex might recognize such tRNAs due to an altered tertiary structure of the tRNA. Remarkably, both the unmodified and wild-type alanine-tRNA are poor substrates for the TRAMP complex, probably because

alanine-tRNAs can fold correctly also without further modification. A mutant alanine-tRNA, however, with an altered tertiary structure was effectively polyadenylated. This shows that the TRAMP complex can recognize and polyadenylate aberrant tRNAs, which are subsequently handed over to the exosome. The poly(A) tail present on the tRNA then allows the exosome to start degrading these RNA molecules more easily, due to lack of secondary structure in the poly(A) tail (Vanacova *et al.*, 2005).

## mRNA Turnover

### a. General mRNA Turnover
The degradation of mRNAs can proceed via two main pathways, $5' \rightarrow 3'$ and $3' \rightarrow 5'$ decay, of which the latter is mediated by the exosome. An interesting question is, how is mRNA turnover regulated? The degradation of mRNA is normally initiated by removal of the poly(A) tail by a deadenylase complex. In mammalian cells the deadenylase PARN has been shown to be required for exosome-mediated mRNA decay suggesting that the exosome can only access mRNAs with shortened poly(A) tails (Chen *et al.*, 2001). In part the regulation can be explained by regulatory proteins that can bind to the poly(A) tail thereby either protecting against or stimulating the degradation of the mRNA. Such proteins might interact with the exosome in order to recruit it to the mRNA. Besides the core exosome $3' \rightarrow 5'$ mRNA decay requires the Ski-complex containing the RNA helicase Ski2, which is most likely required for remodeling the mRNA structure or stimulating the disassociation of proteins from the $3'$ UTR. In this way the Ski-complex stimulates exosomal degradation by making the $3'$ UTR accessible for the exosome (see Chapter III) (Anderson and Parker, 1998; Van Hoof *et al.*, 2000b). In addition, $3' \rightarrow 5'$ mRNA decay can be regulated by *cis*-acting elements in the $3'$ UTR, of which the AU-rich element (ARE) is the most well-known element. The regulation of the exosome in ARE-mediated decay is discussed below. The alternative pathway, $5' \rightarrow 3'$ decay, is characterized by decapping of the $5'$ end, followed by $5' \rightarrow 3'$ exoribonucleolytic degradation catalyzed by Xrn1 (Fillman and Lykke-Andersen, 2005).

### b. Recruiting the Exosome to ARE-mRNAs
In addition to PM/Scl-75, the mechanisms that regulate ARE-mRNA degradation involve the binding of specific AU-rich element binding proteins (AUBPs). The exosome has been shown to physically interact with several proteins of this family, including the proteins KSRP, AUF-1, and TTP, all of which can stimulate exosome mediated ARE-mRNA degradation *in vitro*. In contrast, the ARE-mRNA stabilizing factor HuR does not interact with the exosome (Chen *et al.*, 2001). It is known that the KH motifs of KSRP can directly mediate RNA-binding,

mRNA decay, and interactions with the exosome ([Gherzi *et al.*, 2004]). Another study shows that KSRP and HuR can compete for binding to the AU-rich element (ARE) of the human iNOS mRNA, whereas HuR and AUF1 can also bind to distinct, non-overlapping sites ([Lal *et al.*, 2004; Linker *et al.*, 2005]). Additionally, the putative DExH RNA helicase RHAU and poly(A) nuclease PARN are required for deadenylation and decay of ARE-mRNAs and have been shown to interact with the exosome ([Tran *et al.*, 2004]). These data suggest a mechanism in which the AUBPs KSRP and TTP may recruit PARN and RHAU to the RNA substrate. After deadenylation by PARN and the disruption of the interaction between HuR and the ARE by RHAU, the AUBPs can recruit the exosome to the ARE-RNAs to initiate degradation of the RNA body.

c. **The Role of the Exosome in NMD** Studies have revealed that Upf factors interact with mRNAs to distinguish between mRNAs that contain premature termination codons and mRNAs that do not. Initially it was found that the NMD pathway was invoked by decapping followed by $5' \rightarrow 3'$ decay. However, a role for the exosome in NMD was revealed by the observation that this mRNA turnover pathway required the exosome component Rrp4p as well as components of the Ski-complex showing that NMD can also take place in the $3' \rightarrow 5'$ direction ([Mitchell and Tollervey, 2003; Takahashi *et al.*, 2003]). It was found in human cells that the Upf factors Upf1, Upf2, Upf3X can associate with the exosome and PARN, although it is not known if this interaction is RNA-mediated or not. These data suggest that the Upf factors can recruit the deadenylation machinery as well as exonucleolytic proteins of the exosome to the target mRNA for efficient degradation ([Lejeune *et al.*, 2003]).

d. **mRNAs Lacking a Termination Codon** Also, messenger RNAs lacking a termination codon are rapidly degraded by a surveillance pathway called non-stop decay, which is initiated when the ribosome is stalled somewhere near the $3'$ end of the mRNA during the first “pioneer” round of translation. The non-stop pathway requires active translation as well as the exosome-associated protein Ski7p. The involvement of the exosome was demonstrated by co-purification of Ski7p with two core exosome components as well as the observation that a mutation which prevents Ski7p to associate with the exosome inhibits exosome-mediated mRNA decay. The protein Ski7p is related to a translation elongation and termination factor suggesting that recognition of non-stop mRNAs is mediated by Ski7p through binding to the ribosome, after which the Ski7p bound exosome can rapidly degrade the mRNA body ([Frischmeyer *et al.*, 2002; Van Hoof *et al.*, 2002]). Besides its role in non-stop decay Ski7p and the exosome have also been shown to be

required for regular $3' \rightarrow 5'$ mRNA decay (see earlier) (Jacobs Anderson and Parker, 1998; Van Hoof *et al.*, 2000b).

e. **Negative Feedback Regulation in an Exosome-Dependent Manner** Nab2p is a nuclear poly(A) mRNA-binding protein which can bind to the 3′ UTR of its own mRNA. In a study by Roth and co-workers it is shown that cleavage and polyadenylation of Nab2p mRNA can occur independently of Nab2p interaction, but that binding of Nab2p to the 3′ UTR accelerates Nab2p pre-mRNA degradation by the exosome. This data shows that the Nab2p protein can negatively control its own mRNA levels, most likely by recruiting the exosome to the 3′ UTR (Roth *et al.*, 2005).

Regulation of mRNA processing via Rrp6p has also been reported in a study describing a physical interaction of Rrp6p with the poly(A) mRNA-binding protein Npl3p. Because the Npl3p protein associates with Pol II transcripts and leaves the nucleus associated with the poly(A) mRNA, the degradation of mRNAs by the exosome is presumably inhibited because Npl3p is bound to the poly(A) mRNA. Degradation of aberrant transcripts might be observed when there is interference with the mRNA processing or export, which causes the release of Npl3p from the mRNA making the RNA available to the Rrp6p containing exosome (Burkard and Butler, 2000).

## Specificity and Activation of the Exosome in sn(o) and rRNA Processing

The Rrp47p protein is associated with the nuclear exosome component Rrp6p. In accordance with this, genetic depletion of Rrp47 in yeast leads to a similar sn(o)RNA and rRNA processing phenotype as depletion of Rrp6p. However, Rrp47p is not required for Rrp6p-dependent degradation of aberrant pre-mRNAs in the nucleus. This suggests that Rrp47p provides substrate specificity for the exosome in the processing of stable small RNA molecules (Mitchell *et al.*, 2003).

As mentioned before, the exosome is required for the processing of the 3′ end of the 5.8S rRNA, whereas processing of the 5′ end of the 5.8S rRNA is performed by the exoribonuclease Rat1p and its associated protein Railp. Surprisingly, depletion of Railp leads to accumulation of polyadenylated rRNA and inhibits Rrp6p-dependent processing of the 5.8S rRNA. This suggests that Railp provides a link for efficient rRNA processing at both sides of 5.8S rRNA and thus might regulate the role of the exosome in this process (Fang *et al.*, 2005).

The recently identified TRAMP complex can activate exosome degradation of RNA substrates by adding a single-stranded poly(A) tail, presumably making the RNA a better substrate for $3' \rightarrow 5'$ degradation. Proteomic

analysis of the purified TRAMP complex showed that Mtr4p can form complexes with Trf4p and Air2p as well as with Trf5p. Although another putative RNA binding protein, Air1p, did not co-purify with Trf4p or Mtr4p, *in vivo* analyses suggested that Air1p can replace Air2p in the TRAMP complex. Therefore, it is conceivable that different subsets of TRAMP complexes exist and that these play different roles in exosome-dependent RNA processing events. For example, Trf4p and Trf5p may polyadenylate different RNA substrates, whereas the putative RNA binding proteins Air1p and Air2p may recruit the exosome complex to different polyadenylated substrate RNAs (*LaCava et al.*, 2005).

# V. The Exosome in Disease

## A. Anti-Exosome Antibodies in Autoimmune Diseases

Autoantibodies that recognize the human exosome (anti-PM/Scl) were originally identified by double immunodiffusion of patient sera and calf thymus extract (*Wolfe et al.*, 1977). Although other ways to identify these autoantibodies were available, immunodiffusion would remain the most widely used method to test for anti-PM/Scl antibodies for almost 20 years. The main alternative methods were immunofluorescence, where anti-PM/Scl sera give a homogenous nucleolar staining, or immunoprecipitation, which yields the typical protein precipitation pattern of the exosome complex (8–9 proteins between 25 and 45 kD and additional proteins migrating at 75–80 kD and 100 kD). The immunofluorescence pattern, however, is not very specific for anti-PM/Scl, especially in the presence of other interfering autoantibody reactivities and immunoprecipitation is too laborious to be used for large-scale screening of patient cohorts. Also immunoblot detection of especially anti-PM/Scl-75 reactivity is notoriously difficult, leaving immunodiffusion as the main approach to detect anti-PM/Scl autoantibodies. Only much later it became more common to use recombinant proteins and peptides derived from these proteins to test for anti-PM/Scl antibodies, mainly using ELISA systems.

The main autoantigenic epitopes in the human exosome are located on the proteins PM/Scl-100 and PM/Scl-75 (*Bautz and Bluthner*, 1994; *Ge et al.*, 1994; *Raijmakers et al.*, 2004). Although sometimes other subunits of the exosome are recognized by patient sera, the prevalence of these reactivities is much lower. The most antigenic of the other proteins is hRrp4p, against which antibodies are found in 54% of the sera that are reactive with either

PM/Scl-75 or PM/Scl-100 (Brouwer *et al.*, 2002). In the PM/Scl-75 protein, the most often targeted areas are the highly charged C-terminal region of the protein (Alderuccio *et al.*, 1991) and the recently identified N-terminal part (Raijmakers *et al.*, 2004). The main autoantigenic epitope of PM/Scl-100 resides between amino acids 231 and 245 (sequence LDVPPALADFI-HQQR), although other regions of this protein can also be recognized by autoantibodies (Bluthner *et al.*, 1996, 2000; Ge *et al.*, 1996).

A great variation in the prevalence of anti-PM/Scl antibodies has been reported in the past. Using immunodiffusion assays, the occurrence of the antibodies has been reported to be anywhere between 12% and 88% in patients with the PM/Scl overlap syndrome, between 0% and 50% in patients with PM, and between 2% and 6% in Scl patients. All immunodiffusion studies combined, however, give an occurrence of 30% in PM/Scl overlap syndrome, 9% in PM patients, and 2% in Scl patients (Brouwer *et al.*, 2001a; Jablonska and Blaszczyk, 1998; Marguerie *et al.*, 1992; Oddis *et al.*, 1992; Reichlin and Arnett, 1984; Reimer *et al.*, 1988; Treadwell *et al.*, 1984; Wolfe *et al.*, 1977). These frequencies are supported by the results of other studies using recombinant PM/Scl-75 and PM/Scl-100 proteins (31% in PM/Scl, 2–6% in PM, and 10% in Scl) (Brouwer *et al.*, 2001c; Raijmakers *et al.*, 2004). Currently, the most sensitive way to detect these autoantibodies is a recently developed peptide-based assay, using the PM/Scl-100 epitope described previously. This assay detects the antibodies in 55% of PM/Scl patients, 8% of PM patients, and 13% of Scl patients (Mahler *et al.*, 2005).

It is not known why these patients develop autoantibodies against the exosome complex, but for quite a number of antigens it is known that they undergo post-translational modifications during cell death that might trigger the immune system to develop an immune response against these proteins (Utz *et al.*, 2000). Also for both main autoantigens in the exosome it has been suggested that they are being modified during apoptosis. PM/Scl-100 has been shown to be cleaved by granzyme B (Casciola-Rosen *et al.*, 1999) and also PM/Scl-75 might be cleaved by caspases during apoptosis (Raijmakers, 2004). If and how these antibodies contribute to the disease or whether they represent a nonpathogenic phenomenon remains to be investigated.

## B. The Exosome as a Target for Chemotherapy

During the last years, another association of the exosome with disease has been discovered. The antimetabolite 5-fluorouracil, which is widely used as chemotherapeutic to treat solid tumors, was shown to inhibit the function of the human exosome, thereby interfering with cell growth and survival

(Giaever *et al.*, 2004; Hughes *et al.*, 2004; Lum *et al.*, 2004). Treatment of yeast cells with this compound yields similar defects in RNA processing as exosome mutant strains, indicating that this drug acts at least in part by inhibiting the functioning of the exosome (Fang *et al.*, 2004).

Currently, no genetic defects are known related to exosome functioning. Most likely, the exosome complex is involved in essential cellular processes, causing even a small defect in any of the core components to be lethal, similar to the yeast exosome subunits (Allmang *et al.*, 1999b; Mitchell *et al.*, 1997).

## VI. Concluding Remarks

The identification and structural and functional characterization of the exosome has led to new insights into the RNA processing pathways as they exist in living cells. In recent years, the complex has been implicated in the 3′ end processing of 5.8S rRNA, numerous sn(o)RNAs, and rescue of mRNAs with an extended 3′ UTR. Besides its role in the maturation of RNA molecules, the exosome is also required for the complete degradation of (aberrant) mRNAs, incorrectly processed rRNAs, and aberrant tRNAs. This wide variety of functions has led to the identification of important exosome cofactors such as the TRAMP complex, the Ski-complex, and several RNA helicases and RNA-binding proteins, which are predicted to perform a regulatory role in exosome activity, in the recruitment of the exosome to substrate RNAs, and in the induction of structural alterations of substrate RNAs required for degradation by the exosome. Nevertheless, the precise mechanisms of how cofactors can target the exosome to its substrate RNAs, can change its activity, and cooperate with (other) 5′ → 3′ and 3′ → 5′ decay factors in similar processes is still largely unknown. Interestingly, 5′ → 3′ mRNA decay pathways, such as ARE-mediated decay and RNA interference, are believed to take place in specific cytoplasmic compartments called P- or GW-bodies. Although the exosome has been reported to be implicated in similar processes, it does not seem to localize to these cytoplasmic foci. Future research will be required to elucidate whether there are similar subcellular compartments for 3′ → 5′ decay and whether different types of mRNAs are targeted to different decay pathways. Published structural data on the Archaeal exosome indicates that the core of the exosome adopts a hexameric ring structure and demonstrates that at least in Archaea not all exosome subunits are active exoribonucleases. However, it is still unclear if this is also true for the exosomes of higher eukaryotes and why the composition of the exosome in the latter organisms is more complex. Taken together, the variety of functions, its complex structure, and its numerous associated proteins will

require extensive research to provide detailed insight in the functioning and regulation of the exosome.

# References

Alderuccio, F., Chan, E. K., and Tan, E. M. (1991). Molecular characterization of an autoantigen of PM-Scl in the polymyositis/scleroderma overlap syndrome: A unique and complete human cDNA encoding an apparent 75-kD acidic protein of the nuclear complex. *J. Exp. Med.* **173**, 941–952.

Allmang, C., Mitchell, P., Petfalski, E., and Tollervey, D. (2000). Degradation of ribosomal RNA precursors by the exosome. *Nucleic Acids Res.* **28**, 1684–1691.

Allmang, C., Kufel, J., Chanfreau, G., Mitchell, P., Petfalski, E., and Tollervey, D. (1999a). Functions of the exosome in rRNA, snoRNA and snRNA synthesis. *EMBO J.* **18**, 5399–5410.

Allmang, C., Petfalski, E., Podtelejnikov, A., Mann, M., Tollervey, D., and Mitchell, P. (1999b). The yeast exosome and human PM-Scl are related complexes of $3' \rightarrow 5'$ exonucleases. *Genes Dev.* **13**, 2148–2158.

Aloy, P., Ciccarelli, F. D., Leutwein, C., Gavin, A. C., Superti-Furga, G., Bork, P., Bottcher, B., and Russell, R. B. (2002). A complex prediction: Three-dimensional model of the yeast exosome. *EMBO Rep.* **3**, 628–635.

Aloy, P., Bottcher, B., Ceulemans, H., Leutwein, C., Mellwig, C., Fischer, S., Gavin, A. C., Bork, P., Superti-Furga, G., Serrano, L., and Russell, R. B. (2004). Structure-based assembly of protein complexes in yeast. *Science* **303**, 2026–2029.

Anderson, J. S., and Parker, R. P. (1998). The $3'$ to $5'$ degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and $3'$ to $5'$ exonucleases of the exosome complex. *EMBO J.* **17**, 1497–1506.

Andrulis, E. D., Werner, J., Nazarian, A., Erdjument-Bromage, H., Tempst, P., and Lis, J. T. (2002). The RNA processing exosome is linked to elongating RNA polymerase II in *Drosophila*. *Nature* **420**, 837–841.

Araki, Y., Takahashi, S., Kobayashi, T., Kajiho, H., Hoshino, S., and Katada, T. (2001). Ski7p G protein interacts with the exosome and the Ski complex for $3'$-to-$5'$ mRNA decay in yeast. *EMBO J.* **20**, 4684–4693.

Baginsky, S., Shteiman-Kotler, A., Liveanu, V., Yehudai-Resheff, S., Bellaoui, M., Settlage, R. E., Shabanowitz, J., Hunt, D. F., Schuster, G., and Gruissem, W. (2001). Chloroplast PNPNase exists as a homo-multimer enzyme complex that is distinct from the *Escherichia coli* degradosome. *RNA* **7**, 1464–1475.

Bautz, F. A., and Bluthner, M. (1994). The PM/Scl antigens. *Man. Biol. Markers Dis.* B5.4-1–B5.4-2.

Beelman, C. A., Stevens, A., Caponigro, G., LaGrandeur, T. E., Hatfield, L., Fortner, D. M., and Parker, R. (1996). An essential component of the decapping enzyme required for normal rates of mRNA turnover. *Nature* **382**, 642–646.

Benard, L., Carroll, K., Valle, R. C., and Wickner, R. B. (1998). Ski6p is a homolog of RNA-processing enzymes that affects translation of non-poly(A) mRNAs and 60S ribosomal subunit biogenesis. *Mol. Cell Biol.* **18**, 2688–2696.

Bluthner, M., and Bautz, F. A. (1992). Cloning and characterization of the cDNA coding for a polymyositis-scleroderma overlap syndrome-related nucleolar 100-kD protein. *J. Exp. Med.* **176**, 973–980.

Bluthner, M., Bautz, E. K., and Bautz, F. A. (1996). Mapping of epitopes recognized by PM/Scl autoantibodies with gene-fragment phage display libraries. *J. Immunol. Methods* **198**, 187–198.

Bluthner, M., Mahler, M., Muller, D. B., Dunzl, H., and Bautz, F. A. (2000). Identification of an alpha-helical epitope region on the PM/Scl-100 autoantigen with structural homology to a region on the heterochromatin p25beta autoantigen using immobilized overlapping synthetic peptides. *J. Mol. Med.* 78, 47–54.

Bousquet-Antonelli, C., Presutti, C., and Tollervey, D. (2000). Identification of a regulated pathway for nuclear pre-mRNA turnover. *Cell* 102, 765–775.

Briggs, M. W., Burkard, K. T., and Butler, J. S. (1998). Rrp6p, the yeast homologue of the human PM-Scl 100-kDa autoantigen, is essential for efficient 5.8 S rRNA 3′ end formation. *J. Biol. Chem.* 273, 13255–13263.

Brouwer, R., Hengstman, G. J., Vree, E. W., Ehrfeld, H., Bozic, B., Ghirardello, A., Grondal, G., Hietarinta, M., Isenberg, D., Kalden, J. R., Lundberg, I., Moutsopoulos, H., Roux-Lombard, P., Vencovsky, J., Wikman, A., Seelig, H. P., Van Engelen, B. G., and Van Venrooij, W. J. (2001a). Autoantibody profiles in the sera of European patients with myositis. *Ann. Rheum. Dis.* 60, 116–123.

Brouwer, R., Allmang, C., Rajmakers, R., Van Aarssen, Y., Egberts, W. V., Petfalski, E., Van Venrooij, W. J., Tollervey, D., and Pruijn, G. J. (2001b). Three novel components of the human exosome. *J. Biol. Chem.* 276, 6177–6184.

Brouwer, R., Pruijn, G. J., and Van Venrooij, W. J. (2001c). The human exosome: An autoantigenic complex of exoribonucleases in myositis and scleroderma. *Arthritis Res.* 3, 102–106.

Brouwer, R., Egberts, W. V., Hengstman, G. J. D., Rajmakers, R., Van Engelen, B. G. M., Seelig, H. P., Renz, M., Mierau, R., Gendt, E., Pruijn, G. J. M., and Van Venrooij, W. J. (2002). Autoantibodies directed to novel components of the PM/Scl complex, the human exosome. *Arthritis Res.* 4, 134–138.

Brown, J. T., Bai, X., and Johnson, A. W. (2000). The yeast antiviral proteins Ski2p, Ski3p, and Ski8p exist as a complex *in vivo*. *RNA* 6, 449–457.

Burkard, K. T., and Butler, J. S. (2000). A nuclear 3′-5′ exonuclease involved in mRNA degradation interacts with Poly(A) polymerase and the hnRNA protein Npl3p. *Mol. Cell. Biol.* 20, 604–616.

Buttner, K., Wenig, K., and Hopfner, K. P. (2005). Structural framework for the mechanism of archaeal exosomes in RNA processing. *Mol. Cell* 20, 461–471.

Casciola-Rosen, L., Andrade, F., Ulanet, D., Wong, W. B., and Rosen, A. (1999). Cleavage by granzyme B is strongly predictive of autoantigen status: Implications for initiation of autoimmunity. *J. Exp. Med.* 190, 815–826.

Chekanova, J. A., Dutko, J. A., Mian, I. S., and Belostotsky, D. A. (2002). Arabidopsis thaliana exosome subunit AtRrp4p is a hydrolytic 3′→5′ exonuclease containing S1 and KH RNA-binding domains. *Nucleic Acids Res.* 30, 695–700.

Chekanova, J. A., Shaw, R. J., Wills, M. A., and Belostotsky, D. A. (2000). Poly(A) tail-dependent exonuclease AtRrp41p from Arabidopsis thaliana rescues 5.8 S rRNA processing and mRNA decay defects of the yeast ski6 mutant and is found in an exosome-sized complex in plant and yeast cells. *J. Biol. Chem.* 275, 33158–33166.

Chen, C. Y., and Shyu, A. B. (1995). AU-rich elements: Characterization and importance in mRNA degradation. *Trends Biochem. Sci.* 20, 465–470.

Chen, C. Y., Gherzi, R., Ong, S. E., Chan, E. L., Rajmakers, R., Pruijn, G. J., Stoecklin, G., Moroni, C., Mann, M., and Karin, M. (2001). AU Binding Proteins Recruit the Exosome to Degrade ARE-Containing mRNAs. *Cell* 107, 451–464.

Couttet, P., Fromont-Racine, M., Steel, D., Pictet, R., and Grange, T. (1997). Messenger RNA deadenylation precedes decapping in mammalian cells. *Proc. Natl. Acad. Sci. USA* 94, 5628–5633.

Czaplinski, K., Ruiz-Echevarria, M. J., Paushkin, S. V., Han, X., Weng, Y., Perlick, H. A., Dietz, H. C., Ter Avanesyan, M. D., and Peltz, S. W. (1998). The surveillance complex

interacts with the translation release factors to enhance termination and degrade aberrant mRNAs. *Genes Dev.* **12**, 1665–1677.

De la Cruz, J., Kressler, D., Tollervey, D., and Linder, P. (1998). Doblp (Mtr4p) is a putative ATP-dependent RNA helicase required for the 3′ end formation of 5.8S rRNA in Saccharomyces cerevisiae. *EMBO J.* **17**, 1128–1140.

Decker, C. J., and Parker, R. (1993). A turnover pathway for both stable and unstable mRNAs in yeast: evidence for a requirement for deadenylation. *Genes Dev.* **7**, 1632–1643.

Dunckley, T., and Parker, R. (1999). The DCP2 protein is required for mRNA decapping in Saccharomyces cerevisiae and contains a functional MutT motif. *EMBO J.* **18**, 5411–5422.

Espert, L., Degols, G., Gongora, C., Blondel, D., Williams, B. R., Silverman, R. H., and Mechti, N. (2003). ISG20, a new interferon-induced RNase specific for single-stranded RNA, defines an alternative antiviral pathway against RNA genomic viruses. *J. Biol. Chem.* **278**, 16151–16158.

Estevez, A. M., Kempf, T., and Clayton, C. (2001). The exosome of Trypanosoma brucei. *EMBO J.* **20**, 3831–3839.

Estevez, A. M., Lehner, B., Sanderson, C. M., Ruppert, T., and Clayton, C. (2003). The roles of intersubunit interactions in exosome stability. *J. Biol. Chem.* **278**, 34943–34951.

Evguenieva-Hackenberg, E., Walter, P., Hochleitner, E., Lottspeich, F., and Klug, G. (2003). An exosome-like complex in Sulfolobus solfataricus. *EMBO Rep.* **4**, 889–893.

Fang, F., Hoskins, J., and Butler, J. S. (2004). 5-fluorouracil enhances exosome-dependent accumulation of polyadenylated rRNAs. *Mol. Cell. Biol.* **24**, 10766–10776.

Fang, F., Phillips, S., and Butler, J. S. (2005). Rat1p and Railp function with the nuclear exosome in the processing and degradation of rRNA precursors. *RNA* **11**, 1571–1578.

Farhoud, M. H., Wessels, H. J., Steenbakkers, P. J., Mattijssen, S., Wevers, R. A., Van Engelen, B. G., Jetten, M. S., Smeltink, J. A., Van den Heuvel, L. P., and Keltjens, J. T. (2005). Protein Complexes in the Archaeon Methanothermobacter thermautotrophicus Analyzed by Blue Native/SDS-PAGE and Mass Spectrometry. *Mol. Cell. Proteomics* **4**, 1653–1663.

Fillman, C., and Lykke-Andersen, J. (2005). RNA decapping inside and outside of processing bodies. *Curr. Opin. Cell Biol.* **17**, 326–331.

Fliess, A., Motro, B., and Unger, R. (2002). Swaps in protein sequences. *Proteins* **48**, 377–387.

Ford, L. P., Watson, J., Keene, J. D., and Wilusz, J. (1999). ELAV proteins stabilize deadenylated intermediates in a novel in vitro mRNA deadenylation/degradation system. *Genes Dev.* **13**, 188–201.

Frischmeyer, P. A., Van Hoof, A., O’Donnell, K., Guerrero, A. L., Parker, R., and Dietz, H. C. (2002). An mRNA surveillance mechanism that eliminates transcripts lacking termination codons. *Science* **295**, 2258–2261.

Ge, Q., Frank, M. B., O’Brien, C., and Targoff, I. N. (1992). Cloning of a complementary DNA coding for the 100-kD antigenic protein of the PM-Scl autoantigen. *J. Clin. Invest.* **90**, 559–570.

Ge, Q., Wu, Y., James, J. A., and Targoff, I. N. (1996). Epitope analysis of the major reactive region of the 100-kd protein of PM-Scl autoantigen. *Arthritis Rheum.* **39**, 1588–1595.

Ge, Q., Wu, Y., Trieu, E. P., and Targoff, I. N. (1994). Analysis of the specificity of anti-PM-Scl autoantibodies. *Arthritis Rheum.* **37**, 1445–1452.

Gelpi, C., Alguero, A., Angeles-Martinez, M., Vidal, S., Juarez, C., and Rodriguez-Sanchez, J. L. (1990). Identification of protein components reactive with anti-PM/Scl autoantibodies. *Clin. Exp. Immunol.* **81**, 59–64.

Gherzi, R., Lee, K. Y., Briata, P., Wegmuller, D., Moroni, C., Karin, M., and Chen, C. Y. (2004). A KH domain RNA binding protein, KSRP, promotes ARE-directed mRNA turnover by recruiting the degradation machinery. *Mol. Cell* **14**, 571–583.

Giaever, G., Flaherty, P., Kumm, J., Proctor, M., Nislow, C., Jaramillo, D. F., Chu, A. M., Jordan, M. I., Arkin, A. P., and Davis, R. W. (2004). Chemogenomic profiling: Identifying

the functional interactions of small molecules in yeast. *Proc. Natl. Acad. Sci. USA* 101, 793–798.

Groll, M., Ditzel, L., Lowe, J., Stock, D., Bochtler, M., Bartunik, H. D., and Huber, R. (1997). Structure of 20S proteasome from yeast at 2.4 Å resolution. *Nature* 386, 463–471.

Grosshans, H., Deinert, K., Hurt, E., and Simos, G. (2001). Biogenesis of the signal recognition particle (SRP) involves import of SRP proteins into the nucleolus, assembly with the SRP-RNA, and Xpo1p-mediated export. *J. Cell. Biol.* 153, 745–762.

Grunberg-Manago, M., and Ochoa, S. (1955). Enzymatic synthesis and breakdown of polynucleotides: Polynucleotide phosphorylase. *J. Am. Chem. Soc.* 77, 3165–3166.

Haile, S., Estevez, A. M., and Clayton, C. (2003). A role for the exosome in the *in vivo* degradation of unstable mRNAs. *RNA* 9, 1491–1501.

Hilleren, P., McCarthy, T., Rosbash, M., Parker, R., and Jensen, T. H. (2001). Quality control of mRNA 3′-end processing is linked to the nuclear exosome. *Nature* 413, 538–542.

Hilleren, P. J., and Parker, R. (2003). Cytoplasmic degradation of splice-defective pre-mRNAs and intermediates. *Mol. Cell* 12, 1453–1465.

Hougan, L., Thomas, D. Y., and Whiteway, M. (1989). Cloning and characterization of the SKI3 gene of Saccharomyces cerevisiae demonstrates allelism to SKI5. *Curr. Genet.* 16, 139–143.

Hsu, C. L., and Stevens, A. (1993). Yeast cells lacking 5′→3′ exoribonuclease 1 contain mRNA species that are poly(A) deficient and partially lack the 5′ cap structure. *Mol. Cell. Biol.* 13, 4826–4835.

Hughes, T., Andrews, B., and Boone, C. (2004). Old drugs, new tricks: Using genetically sensitized yeast to reveal drug targets. *Cell* 116, 5–7.

Ishigaki, Y., Li, X., Serin, G., and Maquat, L. E. (2001). Evidence for a pioneer round of mRNA translation: mRNAs subject to nonsense-mediated decay in mammalian cells are bound by CBP80 and CBP20. *Cell* 106, 607–617.

Jablonska, S., and Blaszczyk, M. (1998). Scleromyositis: A scleroderma/polymyositis overlap syndrome. *Clin. Rheumatol.* 17, 465–467.

Jacobs Anderson, J. S., and Parker, R. P. (1998). The 3′ to 5′ degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3′ to 5′ exonucleases of the exosome complex. *EMBO J.* 17, 1497–1506.

Jiang, T., and Altman, S. (2002). A protein subunit of human RNase P, Rpp14, and its interacting partner, OIP2, have 3′→5′ exoribonuclease activity. *Proc. Natl. Acad. Sci. USA* 8, 5295–5300.

Johnson, A. W., and Kolodner, R. D. (1995). Synthetic lethality of sep1 (xrn1) ski2 and sep1 (xrn1) ski3 mutants of *Saccharomyces cerevisiae* is independent of killer virus and suggests a general role for these genes in translation control. *Mol. Cell. Biol.* 15, 2719–2727.

Kadaba, S., Krueger, A., Trice, T., Krecic, A. M., Hinnebusch, A. G., and Anderson, J. (2004). Nuclear surveillance and degradation of hypomodified initiator tRNAMet in *S. cerevisiae*. *Genes Dev.* 18, 1227–1240.

Kiss, T. (2002). Small nucleolar RNAs: An abundant group of noncoding RNAs with diverse cellular functions. *Cell* 109, 145–148.

Koonin, E. V., Wolf, Y. I., and Aravind, L. (2001). Prediction of the archaeal exosome and its connections with the proteasome and the translation and transcription machineries by a comparative-genomic approach. *Genome Res.* 11, 240–252.

Kufel, J., Allmang, C., Chanfreau, G., Petfalski, E., Lafontaine, D., and Tollervey, D. (2000). Precursors to the U3 small nucleolar RNA lack small nucleolar RNP proteins but are stabilized by La binding. *Mol. Cell. Biol.* 20, 5415–5424.

LaCava, J., Houseley, J., Saveanu, C., Petfalski, E., Thompson, E., Jacquier, A., and Tollervey, D. (2005). RNA degradation by the exosome is promoted by a nuclear polyadenylation complex. *Cell* 121, 713–724.

Lal, A., Mazan-Mamczarz, K., Kawai, T., Yang, X., Martindale, J. L., and Gorospe, M. (2004). Concurrent versus individual binding of HuR and AUF1 to common labile target mRNAs. *EMBO J.* 23, 3092–3102.

Lehner, B., and Sanderson, C. M. (2004). A protein interaction framework for human mRNA degradation. *Genome Res.* 14, 1315–1323.

Lejeune, F., Li, X., and Maquat, L. E. (2003). Nonsense-mediated mRNA decay in mammalian cells involves decapping, deadenylating, and exoribonuclease activities. *Mol. Cell* 12, 675–687.

Lim, J., Kuroki, T., Ozaki, K., Kohsaki, H., Yamori, T., Tsuruo, T., Nakamori, S., Imaoka, S., Endo, M., and Nakamura, Y. (1997). Isolation of murine and human homologues of the fission-yeast dis3+ gene encoding a mitotic-control protein and its overexpression in cancer cells with progressive phenotype. *Cancer Res.* 57, 921–925.

Linker, K., Pautz, A., Fechir, M., Hubrich, T., Greeve, J., and Kleinert, H. (2005). Involvement of KSRP in the post-transcriptional regulation of human iNOS expression-complex interplay of KSRP with TTP and HuR. *Nucleic Acids Res.* 33, 4813–4827.

Loflin, P., Chen, C. Y., and Shyu, A. B. (1999). Unraveling a cytoplasmic role for hnRNP D in the in vivo mRNA destabilization directed by the AU-rich element. *Genes Dev.* 13, 1884–1897.

Lorentzen, E., and Conti, E. (2005). Structural Basis of 3′ End RNA Recognition and Exoribonucleolytic Cleavage by an Exosome RNase PH Core. *Mol. Cell* 20, 473–481.

Lorentzen, E., Walter, P., Fribourg, S., Evguenievva-Hackenberg, E., Klug, G., and Conti, E. (2005). The archaeal exosome core is a hexameric ring structure with three catalytic subunits. *Nat. Struct. Mol. Biol.* 12, 575–581.

Lum, P. Y., Armour, C. D., Stepaniants, S. B., Cavet, G., Wolf, M. K., Butler, J. S., Hinshaw, J. C., Garnier, P., Prestwich, G. D., Leonardson, A., Garrett-Engele, P., Rush, C. M., Bard, M., Schimmack, G., Phillips, J. W., Roberts, C. J., and Shoemaker, D. D. (2004). Discovering modes of action for therapeutic compounds using a genome-wide screen of yeast heterozygotes. *Cell* 116, 121–137.

Lykke-Andersen, J. (2002). Identification of a human decapping complex associated with hUpf proteins in nonsense-mediated decay. *Mol. Cell. Biol.* 22, 8114–8121.

Lykke-Andersen, J., and Wagner, E. (2005). Recruitment and activation of mRNA decay enzymes by two ARE-mediated decay activation domains in the proteins TTP and BRF-1. *Genes Dev.* 19, 351–361.

Mahler, M., Raijmakers, R., Dahnrich, C., Bluthner, M., and Fritzler, M. J. (2005). Clinical evaluation of autoantibodies to a novel PM/Scl peptide antigen. *Arthritis Res. Ther.* 7, R704–R713.

Marguerie, C., Bunn, C. C., Copier, J., Bernstein, R. M., Gilroy, J. M., Black, C. M., So, A. K., and Walport, M. J. (1992). The clinical and immunogenetic features of patients with autoantibodies to the nucleolar antigen PM-Scl. *Medicine (Baltimore)* 71, 327–336.

Mian, I. S. (1997). Comparative sequence analysis of ribonucleases HII, III, II PH and D. *Nucleic Acids Res.* 25, 3187–3195.

Mitchell, P., and Tollervey, D. (2000). Musing on the structural organization of the exosome complex. *Nat. Struct. Biol.* 7, 843–846.

Mitchell, P., and Tollervey, D. (2003). An NMD pathway in yeast involving accelerated deadenylation and exosome-mediated 3′→5′ degradation. *Mol. Cell* 11, 1405–1413.

Mitchell, P., Petfalski, E., Shevchenko, A., Mann, M., and Tollervey, D. (1997). The exosome: A conserved eukaryotic RNA processing complex containing multiple 3′→5′ exoribonucleases. *Cell* 91, 457–466.

Mitchell, P., Petfalski, E., Houalla, R., Podtelejnikov, A., Mann, M., and Tollervey, D. (2003). Rrp47p Is an Exosome-Associated Protein Required for the 3′ Processing of Stable RNAs. *Mol. Cell. Biol.* 23, 6982–6992.

Mitchell, P., Petfalski, E., and Tollervey, D. (1996). The 3′ end of yeast 5.8S rRNA is generated by an exonuclease processing mechanism. *Genes Dev.* 10, 502–513.

Muhlrad, D., Decker, C. J., and Parker, R. (1994). Deadenylation of the unstable mRNA encoded by the yeast MFA2 gene leads to decapping followed by 5′→3′ digestion of the transcript. *Genes Dev.* **8**, 855–866.

Muhlrad, D., Decker, C. J., and Parker, R. (1995). Turnover mechanisms of the stable yeast PGK1 mRNA. *Mol. Cell. Biol.* **15**, 2145–2156.

Mukherjee, D., Gao, M., O’Connor, J. P., Raijmakers, R., Pruijn, G., Lutz, C. S., and Wilusz, J. (2002). The mammalian exosome mediates the efficient degradation of mRNAs that contain AU-rich elements. *EMBO J.* **21**, 165–174.

Oddis, C. V., Okano, Y., Rudert, W. A., Trucco, M., Duquesnoy, R. J., and Medsger, T. A. (1992). Serum autoantibody to the nuclear antigen PM-Scl. Clinical and immunogenetic associations. *Arthritis Rheum.* **35**, 1211–1217.

Orban, T. I., and Izaurralde, E. (2005). Decay of mRNAs targeted by RISC requires XRN1, the Ski complex, and the exosome. *RNA.* **11**, 459–469.

Parker, R., and Song, H. (2004). The enzymes and control of eukaryotic mRNA turnover. *Nat. Struct. Mol. Biol.* **11**, 121–127.

Peng, W. T., Robinson, M. D., Mnaimneh, S., Krogan, N. J., Cagney, G., Morris, Q., Davierwala, A. P., Grigull, J., Yang, X., Zhang, W., Mitsakakis, N., Ryan, O. W., Datta, N., Jojic, V., Pal, C., Canadien, V., Richards, D., Beattie, B., Wu, L. F., Altschuler, S. J., Rowies, S., Frey, B. J., Emili, A., Greenblatt, J. F., and Hughes, T. R. (2003). A panoramic view of yeast noncoding RNA processing. *Cell* **113**, 919–933.

Perumal, K., and Reddy, R. (2002). The 3′ end formation in small RNAs. *Gene Expr.* **10**, 59–78.

Phillips, S., and Butler, J. S. (2003). Contribution of domain structure to the RNA 3′ end processing and degradation functions of the nuclear exosome subunit Rrp6p. *RNA* **9**, 1098–1107.

Piwowarski, J., Grzechnik, P., Dziembowski, A., Dmochowska, A., Minczuk, M., and Stepień, P. P. (2003). Human polynucleotide phosphorylase, hPNPase, is localized in mitochondria. *J. Mol. Biol.* **329**, 853–857.

Portnoy, V., Evguenieva-Hackenberg, E., Klein, F., Walter, P., Lorentzen, E., Klug, G., and Schuster, G. (2005). RNA polyadenylation in Archaea: Not observed in Haloferax while the exosome polyadenylates RNA in Sulfolobus. *EMBO Rep.* **6**, 1188–1193.

Pruijn, G. J. (2005). Doughnuts dealing with RNA. *Nat. Struct. Mol. Biol.* **12**, 562–564.

Raijmakers, R. (2004). The human exosome and PNase. *Thesis*, Nijmegen, The Netherlands.

Raijmakers, R., Egberts, W. V., Van Venrooij, W. J., and Pruijn, G. J. (2002a). Protein-Protein Interactions between Human Exosome Components Support the Assembly of RNase PH-type Subunits into a Six-membered PNase-like Ring. *J. Mol. Biol.* **323**, 653–663.

Raijmakers, R., Egberts, W. V., Van Venrooij, W. J., and Pruijn, G. J. (2003). The association of the human PM/Scl-75 autoantigen with the exosome is dependent on a newly identified N terminus. *J. Biol. Chem.* **278**, 30698–30704.

Raijmakers, R., Noordman, Y. E., Van Venrooij, W. J., and Pruijn, G. J. (2002b). Protein-protein Interactions of hCsl4p with other Human Exosome Subunits. *J. Mol. Biol.* **315**, 809–818.

Raijmakers, R., Renz, M., Wiemann, C., Egberts, W. V., Seelig, H. P., Van Venrooij, W. J., and Pruijn, G. J. (2004). PM-Scl-75 is the main autoantigen in patients with the polymyositis/scleroderma overlap syndrome. *Arthritis Rheum.* **50**, 565–569.

Reichlin, M., and Arnett, F. C. J. (1984). Multiplicity of antibodies in myositis sera. *Arthritis Rheum.* **27**, 1150–1156.

Reichlin, M., Maddison, P. J., Targoff, I., Bunch, T., Arnett, F., Sharp, G., Treadwell, E., and Tan, E. M. (1984). Antibodies to a nuclear/nucleolar antigen in patients with polymyositis overlap syndromes. *J. Clin. Immunol.* **4**, 40–44.

Reimer, G., Scheer, U., Peters, J. M., and Tan, E. M. (1986). Immunolocalization and partial characterization of a nuclear autoantigen (PM-Scl) associated with polymyositis/scleroderma overlap syndromes. *J. Immunol.* **137**, 3802–3808.

Reimer, G., Steen, V. D., Penning, C. A., Medsger, T. A., and Tan, E. M. (1988). Correlates between autoantibodies to nuclear antigens and clinical features in patients with systemic sclerosis (scleroderma). *Arthritis Rheum.* 31, 525–532.

Ridley, S. P., Sommer, S. S., and Wickner, R. B. (1984). Superkiller mutations in *Saccharomyces cerevisiae* suppress exclusion of M2 double-stranded RNA by L-A-HN and confer cold sensitivity in the presence of M and L-A-HN. *Mol. Cell. Biol.* 4, 761–770.

Roth, K. M., Wolf, M. K., Rossi, M., and Butler, J. S. (2005). The nuclear exosome contributes to autogenous control of NAB2 mRNA levels. *Mol. Cell. Biol.* 25, 1577–1585.

Schilders, G., Raijmakers, R., Raats, J. M. H., and Pruijn, G. J. M. (2005). MPP6 is an exosome-associated RNA-binding protein involved in 5.8S rRNA maturation. *Nucleic Acids Res.* 33, 6795–6804.

Shiomi, T., Fukushima, K., Suzuki, N., Nakashima, N., Noguchi, E., and Nishimoto, T. (1998). Human dis3p, which binds to either GTP- or GDP-Ran, complements *Saccharomyces cerevisiae* dis3. *J. Biochem.* 123, 883–890.

Shyu, A. B., Belasco, J. G., and Greenberg, M. E. (1991). Two distinct destabilizing elements in the c-fos message trigger deadenylation as a first step in rapid mRNA decay. *Genes Dev.* 5, 221–231.

Snel, B., Bork, P., and Huynen, M. (2000). Genome evolution. Gene fusion versus gene fission. *Trends Genet.* 16, 9–11.

Stoecklin, G., Mayo, T., and Anderson, P. (2005). ARE-mRNA degradation requires the 5′-3′ decay pathway. *EMBO Rep.* 7, 451–464.

Suzuki, N., Noguchi, E., Nakashima, N., Oki, M., Ohba, T., Tartakoff, A., Ohishi, M., and Nishimoto, T. (2001). The *Saccharomyces cerevisiae* Small GTPase, Gsp1p/Ran, Is Involved in 3′ Processing of 7S-to-5.8S rRNA and in Degradation of the Excised 5′-A0 Fragment of 35S Pre-rRNA, Both of Which Are Carried Out by the Exosome. *Genetics* 158, 613–625.

Symmons, M. F., Jones, G. H., and Luisi, B. F. (2000). A duplicated fold is the structural basis for polynucleotide phosphorylase catalytic activity, processivity, and regulation. *Structure Fold. Des.* 8, 1215–1226.

Takahashi, S., Araki, Y., Sakuno, T., and Katada, T. (2003). Interaction between Ski7p and Upf1p is required for nonsense-mediated 3′-to-5′ mRNA decay in yeast. *EMBO J.* 22, 3951–3959.

Targoff, I. N., and Reichlin, M. (1985). Nuclear localization of the PM-Scl antigen. *Arthritis Rheum.* 28, 226–230.

Tran, H., Schilling, M., Wirbelauer, C., Hess, D., and Nagamine, Y. (2004). Facilitation of mRNA deadenylation and decay by the exosome-bound, DExH protein RHAU. *Mol. Cell* 13, 101–111.

Treadwell, E. L., Alspaugh, M. A., Wolfe, J. F., and Sharp, G. C. (1984). Clinical relevance of PM-1 antibody and physiochemical characterization of PM-1 antigen. *J. Rheumatol.* 11, 658–662.

Tucker, M., Valencia-Sanchez, M. A., Staples, R. R., Chen, J., Denis, C. L., and Parker, R. (2001). The transcription factor associated Ccr4 and Caf1 proteins are components of the major cytoplasmic mRNA deadenylase in *Saccharomyces cerevisiae*. *Cell* 104, 377–386.

Utz, P. J., Gensler, T. J., and Anderson, P. (2000). Death, autoantigen modifications, and tolerance. *Arthritis Res.* 2, 101–114.

Van Dijk, E., Cougot, N., Meyer, S., Babajko, S., Wahle, E., and Seraphin, B. (2002). Human Dcp2: A catalytically active mRNA decapping enzyme located in specific cytoplasmic structures. *EMBO J.* 21, 6915–6924.

Van Hoof, A., Frischmeyer, P. A., Dietz, H. C., and Parker, R. (2002). Exosome-mediated recognition and degradation of mRNAs lacking a termination codon. *Science* 295, 2262–2264.

Van Hoof, A., Lennertz, P., and Parker, R. (2000a). Yeast exosome mutants accumulate 3′-extended polyadenylated forms of U4 small nuclear RNA and small nucleolar RNAs. *Mol. Cell. Biol.* **20**, 441–452.

Van Hoof, A., Staples, R. R., Baker, R. E., and Parker, R. (2000b). Function of the ski4p (Csl4p) and Ski7p proteins in 3′-to-5′ degradation of mRNA. *Mol. Cell. Biol.* **20**, 8230–8243.

Vanacova, S., Wolf, J., Martin, G., Blank, D., Dettwiler, S., Friedlein, A., Langen, H., Keith, G., and Keller, W. (2005). A new yeast poly(A) polymerase complex involved in RNA quality control. *PLoS Biol.* **3**, e189.

Wang, L., Lewis, M. S., and Johnson, A. W. (2005). Domain interactions within the Ski2/3/8 complex and between the Ski complex and Ski7p. *RNA* **11**, 1291–1302.

Wang, Z., Jiao, X., Carr-Schmid, A., and Kiledjian, M. (2002). The hDcp2 protein is a mammalian mRNA decapping enzyme. *Proc. Natl. Acad. Sci. USA* **99**, 12663–12668.

Wang, Z., and Kiledjian, M. (2001). Functional Link between the Mammalian Exosome and mRNA Decapping. *Cell* **107**, 751–762.

Weischenfeldt, J., Lykke-Andersen, J., and Porse, B. (2005). Messenger RNA surveillance: Neutralizing natural nonsense. *Curr. Biol.* **15**, R559–R562.

Wolfe, J. F., Adelstein, E., and Sharp, G. C. (1977). Antinuclear antibody with distinct specificity for polymyositis. *J. Clin. Invest.* **59**, 176–178.

Wyers, F., Rougemaille, M., Badis, G., Rousselle, J. C., Dufour, M. E., Boulay, J., Regnault, B., Devaux, F., Namane, A., Seraphin, B., Libri, D., and Jacquier, A. (2005). Cryptic pol II transcripts are degraded by a nuclear quality control pathway involving a new poly(A) polymerase. *Cell* **121**, 725–737.

Xu, Z., Horwich, A. L., and Sigler, P. B. (1997). The crystal structure of the asymmetric GroEL-GroES-(ADP)7 chaperonin complex. *Nature* **388**, 741–750.

Yehudai-Resheff, S., Hirsh, M., and Schuster, G. (2001). Polynucleotide phosphorylase functions as both an exonuclease and a poly(A) polymerase in spinach chloroplasts. *Mol. Cell. Biol.* **21**, 5408–5416.

Yehudai-Resheff, S., Portnoy, V., Yogev, S., Adir, N., and Schuster, G. (2003). Domain analysis of the chloroplast polynucleotide phosphorylase reveals discrete functions in RNA degradation, polyadenylation, and sequence homology with exosome proteins. *Plant Cell* **15**, 2003–2019.

Zanchin, N. I., and Goldfarb, D. S. (1999a). Nip7p interacts with Nop8p, an essential nucleolar protein required for 60S ribosome biogenesis, and the exosome subunit Rrp43p. *Mol. Cell. Biol.* **19**, 1518–1525.

Zanchin, N. I., and Goldfarb, D. S. (1999b). The exosome subunit Rrp43p is required for the efficient maturation of 5.8S, 18S and 25S rRNA. *Nucleic Acids Res.* **27**, 1283–1288.

Zhu, B., Mandal, S. S., Pham, A. D., Zheng, Y., Erdjument-Bromage, H., Batra, S. K., Tempst, P., and Reinberg, D. (2005). The human PAF complex coordinates transcription with events downstream of RNA synthesis. *Genes Dev.* **19**, 1668–1673.

Zuo, Y., and Deutscher, M. P. (2001). Exoribonuclease superfamilies: Structural analysis and phylogenetic distribution. *Nucleic Acids Res.* **29**, 1017–1026.
