
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 281, NO. 48, pp. 36944–36951, December 1, 2006  
© 2006 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Novel Benzene Ring Biosynthesis from C₃ and C₄ Primary Metabolites by Two Enzymes* §

Received for publication, August 23, 2006, and in revised form, September 26, 2006 Published, JBC Papers in Press, September 26, 2006, DOI 10.1074/jbc.M608103200

Hirokazu Suzuki†, Yasuo Ohnishi†, Yasuhide Furusho†, Shohei Sakuda§, and Sueharu Horinouchi¹

From the †Department of Biotechnology and §Applied Biological Chemistry, Graduate School of Agriculture and Life Sciences, University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan

---

The shikimate pathway, including seven enzymatic steps for production of chorismate via shikimate from phosphoenolpyruvate and erythrose-4-phosphate, is common in various organisms for the biosynthesis of not only aromatic amino acids but also most biogenic benzene derivatives. 3-Amino-4-hydroxybenzoic acid (3,4-AHBA) is a benzene derivative serving as a precursor for several secondary metabolites produced by Streptomycetes, including grixazone produced by *Streptomyces griseus*. Our study on the biosynthesis pathway of grixazone led to identification of the biosynthesis pathway of 3,4-AHBA from two primary metabolites. Two genes, *griI* and *griH*, within the grixazone biosynthesis gene cluster were found to be responsible for the biosynthesis of 3,4-AHBA; the two genes conferred the *in vivo* production of 3,4-AHBA even on *Escherichia coli*. In vitro analysis showed that GriI catalyzed aldol condensation between two primary metabolites, L-aspartate-4-semialdehyde and dihydroxyacetone phosphate, to form a 7-carbon product, 2-amino-4,5-dihydroxy-6-one-heptanoic acid-7-phosphate, which was subsequently converted to 3,4-AHBA by GriH. The latter reaction required Mn²⁺ ion but not any cofactors involved in reduction or oxidation. This pathway is independent of the shikimate pathway, representing a novel, simple enzyme system responsible for the synthesis of a benzene ring from the C₃ and C₄ primary metabolites.

established as the common pathway for the biosynthesis of aromatic amino acids in bacteria, fungi, algae, and higher plants. Not only aromatic amino acids but also most biogenic benzene derivatives, such as p-aminobenzoic acid, m-aminobenzoic acid, 2-amino-3-hydroxybenzoic acid, 2-amino-6-hydroxybenzoic acid, and many vitamins, are derived from chorismate (1, 2). The shikimate biosynthesis pathway is also employed by Archaea, although the genes encoding the first two enzymes involved in 3-dehydroquinate (DHQ) synthesis are missing in the genomic sequences of many Archaea (3). In one of Archaea, *Methanocaldococcus jannaschii*, DHQ is synthesized from aspartate 4-semialdehyde (ASA) and 6-deoxy-5-ketofructose-1-phosphate by two alternative enzymes and supplied to the shikimate pathway (4). Recent studies (5, 6) showed that 3-amino-5-hydroxybenzoic acid, a precursor for ansamycin antibiotics, is also synthesized through the aminoshikimate pathway, a variant of the shikimate pathway. Thus, the benzene ring as one of the primary chemical structures in nature is extensively formed through the shikimate pathway, although some benzene derivatives are formed from aliphatic acyl-CoA by polyketide synthases (7, 8).

We recently isolated grixazone (Fig. 1A), a mixture of yellow pigments grixazone A and grixazone B, containing a phenoxazinone chromophore, as secondary metabolites of *Streptomyces griseus* (9, 10). In the present study on the grixazone biosynthesis, we found that 3-amino-4-hydroxybenzoic acid (3,4-AHBA, Fig. 1A) is an intermediate of grixazone. 3,4-AHBA is a benzene derivative serving as a precursor for several secondary metabolites produced by Streptomycetes, such as 4-hydroxy-3-nitrosobenzamide of *Streptomyces murayamaensis*, asukamycin of *Streptomyces nodosus*, and manumycin of *Streptomyces parvulus* (11, 12). Although it has been proposed that 3,4-AHBA is derived from a pathway other than a shikimate-type pathway on the basis of incorporation experiments with ¹³C-labeled compounds in 4-hydroxy-3-nitrosobenzamide-producing *S. murayamaensis* (11), neither the whole picture of the 3,4-AHBA biosynthesis pathway nor the genes involved in 3,4-AHBA biosynthesis have been elucidated. Our study on the grixazone biosynthesis gene cluster has led to establishment of the biosynthesis pathway of 3,4-AHBA. Here we report a novel, simple pathway for 3,4-AHBA biosynthesis; only two enzymes are needed for the synthesis of 3,4-AHBA from C₃ and C₄ primary metabolites. The discovery of this pathway extends our

---

The shikimate pathway (Fig. 1B), involving seven enzymatic steps that produce chorismate via shikimate from phosphoenolpyruvate (PEP)² and erythrose-4-phosphate, is well

---

* This work was supported by Grant 03A07002 from the Industrial Technology Research Grant Program in 2003 of the New Energy and Industrial Technology Development Organization of Japan, by a grant-in-aid for scientific research on priority areas “Applied Genomics” from Monka sho, and by the BioDesign Program of the Ministry of Agriculture, Forestry, and Fisheries of Japan. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
♦ This article was selected as a Paper of the Week.
§ The on-line version of this article (available at http://www.jbc.org) contains supplemental Methods and Fig. 1s and 2s.
The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AB259663.
¹ To whom correspondence should be addressed. Tel.: 81-3-5841-5123; Fax: 81-3-5841-8021; E-mail: asuhori@mail.ecc.u-tokyo.ac.jp.
² The abbreviations used are: PEP, phosphoenolpyruvate; 3,4-AcAHBA, 3-acetylamino-4-hydroxybenzoic acid; 5,2-APC, 5-acetyl-1H-pyrrole-2-carboxylic acid; 3,4-AHBA, 3-amino-4-hydroxybenzoic acid; APOC, 2-amino-phenoxazin-3-one-8-carboxylic acid; DHQ, 3-dehydroquinate; HPLC, high-performance liquid chromatography; HRESI/TOF-MS, high resolution

electrospray ionization-time-of-flight mass spectrum; LC-ESIMS, liquid chromatography-electrospray ionization mass spectrometry; SMM, standard minimal medium.

A Novel System for Benzene Ring Biosynthesis

**A**

![Chemical Structure 1](chemical_structure_1.png)

**B**

![Chemical Structure 2](chemical_structure_2.png)

FIGURE 1. Biosynthesis pathways of a benzene ring. A, a proposed pathway for biosynthesis of 3,4-AHBA and its derivatives in *S. griseus*. 3,4-AHBA is formed from ASA and DHAP through two reactions catalyzed by Gril and GriH. ASA is derived from the biosynthetic pathway for amino acids of the aspartate group and DHAP is from the glycolytic pathway. The brackets for compound 1 denote that this compound is the probable product but has not been independently characterized. B, the shikimate pathway leading to chorismate (upper) (2) and an alternative DHQ synthesis pathway, involving MJ0400 and MJ1249, established for *M. jannaschii* (lower) (4). DKFP, 6-deoxy-5-ketofructose-1-phosphate; E4P, erythrose-4-phosphate.

knowledge on the biosynthesis of aromatic compounds in nature.

## EXPERIMENTAL PROCEDURES

### Bacterial Strains, Media, and Materials—S. griseus IFO13350 was obtained from the Institute of Fermentation, Osaka, Japan. A grixazone-nonproducing mutant strain, S. griseus M31, was derived from S. griseus IFO13350 by ultraviolet mutagenesis. S. lividans TK21 was obtained from D. A. Hopwood (13). S. griseus strains were grown at 30°C in YPD medium (0.2% yeast extract, 0.4% Bacto peptone, 0.5% NaCl, 0.2% MgSO₄·7 H₂O, 1% glucose, and 0.5% glycine, pH 7.2) or at 26.5°C in standard minimal medium (SMM), of which KH₂PO₄ was usually adjusted to 2.5 mM (10). S. lividans strains were grown at 30°C in YEME medium (0.3% yeast extract, 0.5% Bacto peptone, 0.3% Bacto malt extract, 1% glucose, 34% sucrose, 0.5% glycine, 5 mM MgCl₂, pH 7.2) (13). A thiostrepton resistance plasmid pIJ702 (14) (40–300 copies per chromosome) containing the melC promoter was used for DNA manipulation and gene expression in S. griseus. For production of Gril and GriH proteins in S. lividans TK21, pIJ4123 (15) containing the thiostrepton-inducible tipA promoter, was used. Escherichia coli strains JM109, JM110, and TOP10 (Invitrogen) and plasmids pUC19 and pCR4Blunt-TOPO (Invitrogen) were used for DNA manipulation. E. coli BL21(DE3) and pET-17b were used for in vivo production of 3,4-AHBA. DNA was manipulated in Streptomyces (13) and E. coli (16) as described earlier. 3,4-AHBA was purchased from Wako Chemicals. Stable isotope-labeled compounds were purchased from Nippon Sanso. ASA was synthesized according to the method described by Black and Wright (17, 18).

### Construction of Plasmids—Full details of the experimental methods are given in supplemental Methods; a summary is given below. A plasmid, pAYP20, conferring the production of a grixazone-like yellow pigment on strain M31, was isolated by shotgun cloning of a library of the chromosome of the wild-type strain. A 3.5-kb SphI fragment containing gril and griH was excised from pAYP20 and reintroduced into pIJ702 at the SphI
site in the same direction as that on pAYP20, resulting in pAYP25. The *griI-griH*, *griI*, and *griH* sequences were amplified by PCR and cloned in a pIJ702-derived plasmid, in which the *melC1-melC2* sequence under the *melC* promoter was replaced by a short linker, resulting in pAYP26, pAYP27, and pAYP28, respectively. On these plasmids, *griI* and *griH* were under the control of the *melC* promoter. For protein preparation, the *griI* and *griH* sequences were also cloned in pIJ4123, resulting in pIJ4123-*griI* and pIJ4123-*griH*, in which *griI* and *griH* were both under the control of the thiostrepton-inducible *tipA* promoter. The co-translational *griI-griH* sequence was generated by PCR and cloned into pET-17b for expression in *E. coli*, resulting in pET-*griIH*, in which the co-translational *griI-griH* sequence was under the control of the T7 promoter.

**3,4-AHBA Production in Vivo—S. griseus M31 [pAYP20, pAYP25, or pAYP26]** was precultured at 30 °C for 2 days in 100 ml of YPD medium supplemented with 10 μg/ml thiostrepton. The mycelium was harvested by centrifugation, washed two times, resuspended in 10 ml of SMM, and homogenized. A portion (100 μl) of the homogenized solution was inoculated to 100 ml of fresh SMM and cultured at 26.5 °C for 5 days. For 3,4-AHBA production in *E. coli* BL21(DE3) harboring pET-*griIH*, the cells were cultured at 26.5 °C for 24 h in M9 medium supplemented with 50 μg/ml ampicillin, 1% (v/v) glycerol, 1% (w/v) asparagine, and 1% (w/v) lactose. The culture broth was passed through a 0.2-μm membrane, and 3,4-AHBA in the broth was analyzed by reversed-phase high performance liquid chromatography (HPLC) and liquid chromatography-electrospray ionization mass spectrometry (LC-ESIMS).

**HPLC and LC-ESIMS Analysis—** HPLC analysis was carried out by using the Waters 600 HPLC system equipped with the Waters 996 photodiode array detector. Conditions for HPLC were as follows: column, Senshu Pak Docosil-B (4.6 × 250 mm, Senshu Kagaku); column temperature, 30 °C; flow rate, 1 ml/min. After 10 μl of the reaction mixture had been injected into the column equilibrated with 0.1% trifluoroacetic acid in water, the column was initially developed isocratically for 3 min, followed by development by a linear gradient from 0 to 90% acetonitrile in water containing 0.1% trifluoroacetic acid for 15 min. LC-ESIMS analysis was carried out by using a HPLC system (model 1100 series, Agilent Technologies) equipped with a mass spectrometer (Bruker HCT plus, Bruker Daltonics) with the ESI-positive/negative mode. HPLC was conducted using a Senshu Pak Docosil-B (2.6 × 250 mm, Senshu Kagaku) at a flow rate of 0.2 ml/min. After injection of the sample into the column equilibrated with 0.1% acetic acid in water, the column was initially developed isocratically for 10 min, followed by development by a linear gradient from 0 to 100% acetonitrile in water containing 0.1% acetic acid for 10 min.

**Structure Analyses—** Full details of purification of compounds are given in supplemental Methods. *S. griseus* M31 [pAYP20, pAYP25, or pAYP26] produced compounds, 2, 4, and 5 (see Fig. 2C, panel *a*). *S. griseus* M31 [pAYP20] was cultured at 26.5 °C in SMM with rotary shaking. The culture supernatant was adjusted to pH 2.5 and treated with ethyl acetate. By preparative HPLC, compound 2 was purified from the aqueous fraction and compound 4 was purified from the ethyl acetate fraction. Compound 5, which was formed from commercially available 3,4-AHBA by aerobic incubation of it at 26.5 °C for 5 days in aseptic SMM, was purified by ethyl acetate extraction and preparative HPLC. Compound 5 was confirmed to be identical to the yellow pigment produced by *S. griseus* M31 [pAYP20] by HPLC and LC-ESIMS. Compound 3, produced by *S. lividans* [pIJ4123-*griI*], was purified from the culture supernatant by ethyl acetate extraction, ion exchange chromatography on silica, and preparative HPLC. 3,4-AHBA (2), 3-acetylamino-4-hydroxybenzoic acid (4, 3,4-AcAHBA), 2-aminophenoxazin-3-one-8-carboxylic acid (5, APOC), and 5-acetyl-1H-pyrrole-2-carboxylic acid (3, 5,2-APC) were identified on the basis of the following spectroscopic parameters (see Fig. 1A).

- **3,4-AHBA (2):** ${}^{1}$H NMR (500 MHz, D$_{2}$O): δ 7.83 (d, $J = 2.5$ Hz, 1H, benzene 2-H), 7.83 (dd, $J = 9.0$, 2.0 Hz, 1H, benzene 6-H), 6.98 (d, $J = 9.0$ Hz, 1H, benzene 5-H); ${}^{13}$C NMR (125 MHz, D$_{2}$O): δ 170.2 (COOH), 155.4 (benzene 4-C), 132.8 (benzene 6-C), 126.4 (benzene 3-C), 122.8 (benzene 2-C), 119.1 (benzene 1-C), 117.0 (benzene 5-C); high resolution electrospray ionization-time-of-flight mass spectrum (HRESI/TOF-MS), $m/z$ 154.0522 [M + H]$^{+}$ (calculated for C$_{7}$H$_{8}$NO$_{3}$, 1.8 millimass units error).

- **5,2-APC (3):** ${}^{1}$H NMR (500 MHz, Me$_{2}$SO-$d_{6}$): δ 6.94 (d, $J = 4.0$, 2.0 Hz, 1H, pyrrole 3-H), 6.76 (dd, $J = 4.0$, 2.0 Hz, 1H, pyrrole 4-H), 2.42 (s, 3H, COCH$_{3}$); ${}^{13}$C NMR (125 MHz, Me$_{2}$SO-$d_{6}$) δ 188.4 (COCH$_{3}$), 161.5 (COOH), 134.9 (pyrrole 5-C), 128.2 (pyrrole 2-C), 116.1 (pyrrole 3-C), 115.1 (pyrrole 4-C), 26.5 (COCH$_{3}$); ESIMS, $m/z$ 152.2 [M − H]$^{-}$. They were in good agreement with those of 5,2-APC (19).

- **3,4-AcAHBA (4):** ${}^{1}$H NMR (500 MHz, Me$_{2}$SO-$d_{6}$): δ 10.69 (s, 1H, OH), 9.30 (s, 1H, NH), 8.43 (d, $J = 2.0$ Hz, 1H, benzene 2-H), 7.56 (dd, $J = 8.0$, 2.0 Hz, 1H, benzene 6-H), 6.92 (d, $J = 8.5$ Hz, 1H, benzene 5-H), 2.10 (s, 3H, COCH$_{3}$); ${}^{13}$C NMR (125 MHz, Me$_{2}$SO-$d_{6}$): δ 169.1 (COCH$_{3}$), 167.3 (COOH), 152.0 (benzene 4-C), 126.5 (benzene 6-C), 126.2 (benzene 3-C), 123.8 (benzene 2-C), 121.4 (benzene 1-C), 115.1 (benzene 5-C), 23.8 (COCH$_{3}$); HRESI/TOF-MS, $m/z$ 196.0596 [M + H]$^{+}$ (calculated for C$_{9}$H$_{9}$NO$_{4}$, 1.4 millimass units error).

- **APOC (5):** ${}^{1}$H NMR (500 MHz, Me$_{2}$SO-$d_{6}$): δ 8.16 (d, $J = 2.0$ Hz, 1H, 9-H), 7.96 (dd, $J = 8.5$, 2.5 Hz, 1H, 7-H), 7.57 (d, $J = 9.0$ Hz, 1H, 6-H), 6.41 (s, 1H, 1-H), 6.36 (s, 1H, 4-H); ${}^{13}$C NMR (125 MHz, Me$_{2}$SO-$d_{6}$): δ 180.5 (3-C), 166.4 (11-C), 149.0, 148.7, 147.7 (10a-C, 4a-C, and 2-C), 144.9 (5a-C), 133.4, 127.6 (9a-C and 8-C), 129.1 (7-C), 129.0 (9-C), 116.4 (6-C), 104.1 (1-C), 98.3 (4-C); HRESI/TOF-MS, $m/z$ 257.05540 [M + H]$^{+}$ (calculated for C$_{13}$H$_{9}$N$_{2}$O$_{4}$, 0.83 millimass unit error).

**Incorporation of ${}^{13}$C-Labeled Precursors into 3,4-AcAHBA—** *S. griseus* M31 [pAYP25] was cultured at 26.5 °C for about 2 days in 50 ml of SMM with reciprocal shaking until 3,4-AHBA had just started to be produced in the culture broth. The mycelium was harvested and washed three times with carbon source (0.9% glucose and 0.9% asparagine)-depleted SMM. The mycelium was suspended in 50 ml of the same fresh SMM containing a stable isotope-labeled compound (Table 1). After further cultivation at 26.5 °C for 3 days, 3,4-AcAHBA accumulated in the

³ The brackets denote the plasmid-carrier state.
A Novel System for Benzene Ring Biosynthesis

TABLE 1  
Incorporation of $^{13}$C-labeled compounds into 3,4-AcAHBA  

| Labeled compound          | Supplement                | Enrichment of carbon (%) |   |   |   |   |   |   |   |
|---------------------------|---------------------------|--------------------------|---|---|---|---|---|---|---|
|                           |                           | 2                        | 3 | 4 | 7 | 1 | 6 | 5 | 8 | 9 |
| $[4-{ }^{13} \mathrm{C}]$ Aspartate | Glucose                 | $a$                      | — | — | 6.9 | — | — | 23.4 | — | — |
| $[2-{ }^{13} \mathrm{C}]$ Aspartate | Glucose                 | —                        | — | — | 2.5 | 20.2 | 6.3 | 2.0 | — | — |
| $[3-{ }^{13} \mathrm{C}]$ Pyruvate    | Asparagine              | 3.3                      | 1.0 | — | — | — | — | — | — | 4.8 |
| $[3-{ }^{13} \mathrm{C}]$ Pyruvate    | Glucose and asparagine  | —                        | — | — | 2.1 | 2.2 | — | — | — | 4.4 |
| $[3-{ }^{13} \mathrm{C}]$ Pyruvate    | Glycerol and asparagine | —                        | — | — | 1.9 | 2.0 | — | — | — | 7.6 |
| $[2-{ }^{13} \mathrm{C}]$ Glycerol    | Asparagine              | —                        | 70.5 | — | 5.2 | 8.1 | 4.5 | 5.7 | 39.0 | 2.4 |
| $[2-{ }^{13} \mathrm{C}]$ Glycerol    | Pyruvate and asparagine | —                        | 33.8 | — | 1.6 | 3.5 | 2.5 | 1.7 | 6.5 | — |
| $[1-{ }^{13} \mathrm{C}]$ Glucose     | Asparagine              | 11.4                     | — | — | — | 1.9 | 1.8 | — | — | 5.5 |
| $[1-{ }^{13} \mathrm{C}]$ Glucose     | Pyruvate and asparagine | 7.1                      | — | — | — | — | — | — | — | 1.3 |
| $[6-{ }^{13} \mathrm{C}]$ Glucose     | Asparagine              | 7.0                      | — | — | — | 2.4 | 2.4 | — | — | 6.9 |
| $[6-{ }^{13} \mathrm{C}]$ Glucose     | Pyruvate and asparagine | 5.6                      | — | — | — | 1.1 | 1.2 | — | — | 1.9 |

$a$, indicates <1% enrichment.

culture broth was purified by preparative HPLC (see supplemental Methods). Incorporation of labeled precursors into 3,4-AcAHBA was determined by $^{13}$C NMR.

Preparation of GriI and GriH Proteins—S. lividans TK21 [pIJ4123-griI] and S. lividans TK21 [pIJ4123-griH] were cultured at 30°C for 2 days in YEME medium supplemented with 5 μg/ml kanamycin. The tipA promoter was induced by 5 μg/ml thiostrepton and the cultivation was further continued for 2 days. All operations described below were carried out at 4°C. Cells (5 ml wet volume) were harvested by centrifugation and resuspended in 5 ml of buffer A (50 mM sodium phosphate, pH 8.0, 0.5 M NaCl, and 20% glycerol) containing 2 mg/ml lysozyme. After incubation of the mixture on ice for 30 min, the cell suspension was sonicated for 3 min and then centrifuged at 10,000 × g for 10 min to remove cell debris. Polyethyleneimine was added to the supernatant to give a final concentration of 0.1% (w/v), followed by centrifugation of the mixture at 20,000 × g for 20 min. The supernatant, to which imidazole was added to give a final concentration of 10 mM, was applied to a 1-ml HiTrap chelating HP column (Amersham Biosciences) equilibrated with buffer A containing 10 mM imidazole on fast protein liquid chromatography. For GriI and GriH purification, the column was charged with Co²⁺ ions and Ni²⁺ ions, respectively. The column was washed successively with buffer A containing 10 mM imidazole and buffer A containing 50 mM imidazole, and proteins were then eluted with a 50–250 mM linear gradient of imidazole in buffer A. The enzyme solution was concentrated by ultrafiltration and applied to a gel filtration column (HiLoad Superdex 200 16/60 prep grade, Amersham Biosciences) on fast protein liquid chromatography with isocratic elution in buffer B (20 mM HEPES-NaOH, pH 7.2, 0.15 M NaCl, and 20% glycerol) at a flow rate of 1 ml/min. Proteins were quantified by measuring the absorbance of protein solution at 280 nm using the molar absorbance coefficients, 16,500 M⁻¹ cm⁻¹ for GriI and 23,400 M⁻¹ cm⁻¹ for GriH, calculated from their amino acid sequences.

In Vitro Assay of GriI and GriH—The standard reaction mixture (100 μl) consisted of buffer B containing 0.1 mM MnCl₂, 1 mM ASA, 1 mM dihydroxyacetone phosphate (DHAP, Fig. 1A), 0.1 mg/ml (3 μM for monomer) GriI, and 0.4 mg/ml (13 μM) GriH. After incubation at 30°C for 30 min, reaction products were analyzed by HPLC and LC-ESIMS. 3,4-AHBA produced was quantified by a colorimetric method using 4-dimethylaminobenzaldehyde (20), as follows. The reaction was stopped by the addition of 100 μl of 20% trichloroacetic acid (w/v) and then centrifuged. After 4-dimethylaminobenzaldehyde (5%, 400 μl) in acetonitrile/water (9:1 (v/v)) had been added to the supernatant, the absorbance at 450 nm was measured by a spectrometer (Spectra Max plus; Molecular Devices). The amount of 3,4-AHBA was estimated on the basis of the results of the control reactions with authentic 3,4-AHBA.

The GriI reaction was examined using the standard reaction mixture in the absence of GriH and MnCl₂. The GriI reaction product, from which GriI was removed by ultrafiltration, was used as a substrate for the GriH reaction. The concentration of the substrate for GriH in the GriI-removed reaction mixture was estimated on the basis of the amount of 3,4-AHBA produced by prolonged incubation of the mixture containing GriH and MnCl₂. The kinetics of the GriI reaction was determined by the rate of increase in the 3,4-AHBA concentration in the presence of an excess of GriH; the reaction mixture contained 0.1 mM MnCl₂, 20–200 μM ASA, 50–200 μM DHAP, 0.16 μM (for monomer) GriI, and 13 μM GriH in buffer B. Kinetic parameters were determined by non-linear least squares fitting on the equation for the double-displacement mechanism, i.e. 1/v = 1/V_max (1 + K_m,S₁/[S₁] + K_m,S₂/[S₂]) (21). The kinetics of the GriH reaction was determined by the rate of increase in the 3,4-AHBA concentration in the reaction mixtures containing various volumes of the GriI-removed reaction mixture, 0.1 mM MnCl₂, and 1.3 μM GriH in buffer B. Kinetic parameters were determined by linear least squares fitting on the Michaelis-Menten equation (21).

RESULTS AND DISCUSSION

Identification of Genes Responsible for 3,4-AHBA Biosynthesis—During subcloning of the grizaxone biosynthesis genes (Fig. 2A), we found that pAYP26 carrying griI and griH under the control of the melC promoter conferred the production of a grizaxone-like yellow pigment on a grizaxone-nonproducing mutant strain, S. griseus M31. Both griI and griH were required for pigmentation, because neither pAYP27 carrying griI alone nor pAYP28 carrying griH alone caused pigmentation (Fig. 2B). S. griseus M31 [pAYP26] produced three compounds, 2, 4, and 5 (Fig. 2C, panel a), of which the structures were determined as 3,4-AHBA (2), 3,4-AcAHBA (4), and APOC (5) (Fig. 1A). Addition of 3,4-AHBA to the culture of S. griseus M31

A Novel System for Benzene Ring Biosynthesis

**A**  
grixazone biosynthesis gene cluster  


T S R A B C D E F G H I J
pAYP20
pAYP25
pAYP26
pAYP27
pAYP28
pET-grilH


gril-griH sequence under the control of the T7 promoter produced 3,4-AHBA and 3,4-AcAHBA. An important implication from these findings was that no *Streptomyces*-specific genes other than gril and griH were necessary for the biosynthesis of 3,4-AHBA. In addition, the precursors that are converted to 3,4-AHBA by the actions of Gril and GriH were supposed to be present as primary metabolites commonly in bacterial cells. gril encodes a 274-amino acid protein showing sequence similarity (about 30–58% identity) to class I aldolases (supplemental Fig. 1s). griH encodes a 396-amino acid protein showing sequence similarity (23–62% identity) to hypothetical proteins belonging to the UPF0245 family (supplemental Fig. 2s).

Estimation of Primary Metabolites Serving as the Substrates for 3,4-AHBA Biosynthesis—Gould and co-workers (11, 22) and Floss and co-workers (23, 24) proposed that 3,4-AHBA was formed from a $C_4$ unit (corresponding to C-7, C-1, C-6, and C-5 of 3,4-AHBA) derived from the tricarboxylic acid cycle and a $C_3$ unit (C-2, C-3, and C-4 of 3,4-AHBA) derived from the glycolytic pathway. Gould and co-workers (22) also proposed the condensation between oxalacetate and either PEP or pyruvate for 3,4-AHBA biosynthesis. To determine the primary metabolite(s) serving as the substrates for the Gril-GriH system, we examined incorporation of $^{13}$C-labeled precursors into 3,4-AcAHBA by $^{13}$C NMR (Table 1). When [4-$^{13}$C]aspartate and [2-$^{13}$C]aspartate were fed, C-5 and C-1 of 3,4-AcAHBA, respectively, were mainly enriched. We therefore assumed that ASA, in addition to oxalacetate, was the most probable candidate for the $C_4$ unit of 3,4-AHBA in agreement with the proposal of Gould and co-workers (22), because these compounds were direct derivatives from aspartate and could be served as a substrate for aldol condensation.

When sodium [3-$^{13}$C]pyruvate was fed in the absence of glucose or glycerol, C-2 of 3,4-AcAHBA, in addition to C-9 in its acetyl moiety, was mainly enriched. The acetyl moiety was derived probably from acetyl-CoA. This finding is also consistent with the observation by Gould *et al.* (11). When sodium [3-$^{13}$C]pyruvate was fed in the presence of glucose or glycerol, however, C-1 and C-6 of 3,4-AcAHBA were enriched, indicating that pyruvate was incorporated into 3,4-AcAHBA after being converted to a precursor of the $C_4$ unit through the tricarboxylic acid pathway. On the other hand, when [1-$^{13}$C]glucose or [6-$^{13}$C]glucose was fed, C-2 of 3,4-AcAHBA was highly enriched even in the presence of pyruvate. Similarly, a very high level of enrichment at C-3 of 3,4-AcAHBA was observed when [2-$^{13}$C]glycerol was fed even in the presence of pyruvate. These results suggested that the $C_3$ unit moiety of 3,4-AHBA was derived from a metabolite upstream from pyruvate in the glycolytic pathway but not pyruvate itself.

In the glycolytic pathway, glucose is converted to fructose 1,6-bisphosphate and cleaved into two $C_3$ units, DHAP and glyceraldehyde 3-phosphate. The carbon at the phosphorylated position of DHAP and that of glyceraldehyde 3-phosphate are derived originally from the C-1 and C-6 carbons of glucose, respectively, although DHAP is reversibly converted into glyceraldehyde 3-phosphate. In our incorporation experiments with [1-$^{13}$C]glucose and [6-$^{13}$C]glucose, the level (11.4%) of enrichment at C-2 of 3,4-AcAHBA from [1-$^{13}$C]glucose was

FIGURE 2. Biosynthesis of 3,4-AHBA in *S*. griseus. A, organization of the grixazone biosynthesis gene cluster and plasmids used in this study. B, *S*. griseus M31 [pAYP26, pAYP27, pAYP28, or the vector plJ702] was grown at 28°C for 5 days on SMM agar medium. The yellow pigment produced by M31 [pAYP26] was determined to be APOC that was formed non-enzymatically via oxidative coupling of two 3,4-AHBA molecules. C, reversed-phased HPLC analysis of 3,4-AHBA produced in vivo (panel a) and in vitro (panel b). *S*. griseus M31 [pAYP26] was cultured at 26.5°C for 5 days in liquid SMM, and 10 μl of the culture broth was analyzed by HPLC (panel a). The standard Gril-GriH reaction (panel b) and the prolonged Gril reaction (panel b, inset) were analyzed by HPLC.

resulted in the conversion of 3,4-AHBA to 3,4-AcAHBA, suggesting that an acetyltransferase(s) of the host catalyzed the N-acetylation of 3,4-AHBA. APOC, a yellow pigment with visible absorption at 433 nm at pH 2.0, was perhaps produced non-enzymatically from 3,4-AHBA, because 3,4-AHBA was converted to APOC during aerobic incubation at 26.5°C in aseptic SMM. Therefore, pAYP26 carrying gril and griH directed the synthesis of 3,4-AHBA in *S*. griseus M31.

Simultaneous expression of gril and griH caused not only *S*. griseus M31 but also *E*. coli to produce 3,4-AHBA (data not shown). *E*. coli [pET-grilH] that contained a co-translational

A Novel System for Benzene Ring Biosynthesis

**FIGURE 3. Characteristics of Gril and GriH.** A, purified Gril and GriH proteins were separated by SDS-PAGE and stained with Coomassie Brilliant Blue. Myosin (200 kDa), β-galactosidase (116 kDa), phosphorylase b (97 kDa), bovine serum albumin (67 kDa), ovalbumin (45 kDa), carbonic anhydrase (31 kDa), soybean trypsin inhibitor (22 kDa), lysozyme (14 kDa), and bovine aprotinin (7 kDa) were used for the molecular mass standards. A minor band in the Gril preparation probably represents a degradation product of Gril. B, the plots of the initial velocities of the production of 3,4-AHBA versus concentrations of Mn²⁺. The initial reaction velocities were determined by using the standard reaction mixture containing various concentrations of MnCl₂. GriH concentrations were 13 (●), 1.3 (▲), and 0.13 (■) μM, and the 50% effective concentrations (EC₅₀, μM) of Mn²⁺ were estimated as 0.70, 1.41, and 20.3, respectively, by nonlinear least squares fitting on a sigmoidal function: log (V) = log (V_max) – log (EC₅₀/[Mn²⁺] + 1). C, the pH profile (panel a) and temperature profile (panel b) of the reaction producing 3,4-AHBA by Gril and GriH, pH stability (panel c), and temperature stability (panel d) of Gril (●) and GriH (■). Standard reaction mixtures were incubated at 30 °C for 30 min at various pH values (panel a) and at various temperatures at pH 7.2 (panel b), and the amounts of 3,4-AHBA formed were measured. The buffer used in panels a and b were buffer C (50 mM sodium phosphate, 50 mM sodium citrate, 0.15 M NaCl, and 20% glycerol, pH 4–11) and buffer B, respectively. pH stability (panel c) was determined by standard assays for the 3,4-AHBA formation, in which Gril or GriH had been kept at 30 °C for 1 h in buffer C adjusted at various pHs. Temperature stability (panel d) were determined by standard assays for the 3,4-AHBA formation, in which Gril or GriH had been kept for 1 h in buffer B at various temperatures. D, Lineweaver-Burke double-reciprocal plot of the Gril reaction (panel a) and GriH reaction (panel b). Concentrations of DHAP were 200 (●), 100 (▲), and 50 (■) μM in panel a.

higher than that (7.1%) from [6-¹³C]glucose. Furthermore, the ratio (the calculated ratio, 11.4/5.5 = 1.96) of enrichment at C-2 to C-9 of the labeled 3,4-AcAHBA derived from [1-¹³C]glucose was much higher than that (7.0/6.9 = 1.01) from [6-¹³C]glucose. These results showed that the carbon of C-1 of glucose had been incorporated into C-2 of 3,4-AcAHBA before it was incorporated into acetyl-CoA more efficiently than that of C-6 of glucose. Therefore, in disagreement with the proposal of Gould and co-workers (22), we assumed that DHAP, but not PEP or pyruvate, was the most probable candidate for the C₃ unit of 3,4-AHBA. This idea was supported by the finding that C-2 of glycerol, which enters into the glycolytic pathway via DHAP, was incorporated into C-3 of 3,4-AcAHBA very efficiently.

**3,4-AHBA Production in Vitro by Gril and GriH—Histidine-tagged Gril**, having the structure of MGSSH₆SSGLVPRGSH-Gril, was prepared from *S. lividans* [pIJ4123-gril]. Similarly, histidine-tagged GriH, having the structure of MGSSH₆SSGLVPRGSH-GriH, was prepared from *S. lividans* [pIJ4123-griH]. The molecular masses of Gril and GriH were estimated by gel filtration analysis as 340 and 54 kDa, respectively. Because purified Gril and GriH were apparently 33 and 46 kDa on SDS-PAGE, respectively (Fig. 3A), Gril was probably a decamer-type aldolase (25) and GriH was monomeric.

Using the Gril and GriH proteins, we examined various conditions for the synthesis of 3,4-AHBA. When ASA and DHAP were incubated with Gril and GriH in the presence of 0.1 mM MnCl₂ at 30 °C, 3,4-AHBA was produced at a constant rate of about 22 μM/min (2.2 nmol/min) for 30 min (Fig. 2C, panel b). As shown in Fig. 3C, this reaction proceeded over a pH range of 6.0–9.5 with a maximum rate at pH 8.0 at 40 °C. Gril was stable between pH 6.5–10.0 (at 30 °C for 1 h) and below 40 °C (at pH 7.2 for 1 h). GriH was stable between pH 6.5–10.5 (at 30 °C for 1 h) and below 30 °C (at pH 7.2 for 1 h). We examined all combinations of possible C₄ units (ASA, oxalacetate, L-aspartate, and L-homoserine) and C₃ units (DHAP, PEP, pyruvate, and dihydroxyacetone) as substrates and confirmed that 3,4-AHBA was synthesized only from the combination of ASA and DHAP. Addition of Mn²⁺ or some bivalent metal ions (0.1 mM) to the reaction mixture was essential for the synthesis of 3,4-AHBA (relative activity: MnCl₂, 100%; CoCl₂, 85%; FeSO₄, 81%; MgSO₄, 41%; NiSO₄, 6%; ZnSO₄, 4%; CaCl₂ and CuSO₄, <1%).

DECEMBER 1, 2006 • VOLUME 281 • NUMBER 48

A Novel System for Benzene Ring Biosynthesis

**A**

![Chemical Structure A](chemical_structure_a.png)

**B**

![Chemical Structure B](chemical_structure_b.png)

FIGURE 4. Proposed pathways from compound 1 to 5,2-APC (A) and 3,4-AHBA (B). A, a proposed mechanism for the non-enzymatic reaction leading to 5,2-APC from the product of the Gril reaction, 1. B, a proposed mechanism for the GriH reaction yielding 3,4-AHBA from the product of the Gril reaction, 1. Mn²⁺ is essentially required for the Gril-GriH reaction, although it is not illustrated.

No Cofactor Requirement for the GriH Reaction—Mn²⁺ was required for the GriH reaction but not for the Gril reaction, because addition of GriH and MnCl₂ to the Gril-removed mixture containing no Mn²⁺ and incubation of the mixture at 30 °C for 30 min yielded 3,4-AHBA. This experiment also showed that a compound produced as a result of aldol condensation of ASA and DHAP by Gril served as the substrate of GriH, resulting in 3,4-AHBA. The plots of the initial velocities of the production of 3,4-AHBA versus concentrations of Mn²⁺ showed sigmoidal curves (Fig. 3B), indicating that more Mn²⁺ was required for the reaction as the amount of GriH was decreased. This finding excluded the possibility that Mn²⁺ bound tightly to GriH functioned as a normal cofactor. Mn²⁺ may stabilize the substrate for GriH, i.e., the compound produced as a result of aldol condensation between ASA and DHAP by Gril, by chelating the compound, which was extremely unstable (see below).

Because MJ1249, a GriH homologue of *M. jannaschii*, requires NAD (4), we examined effects of cofactors on the formation of 3,4-AHBA by GriH. Addition of 0.1 mM each of NAD, NADH, NADP, NADPH, FAD, or FMN to the reaction mixture had negligible effects on the 3,4-AHBA formation (relative activity, 98–107%), although the reaction was partially inhibited by 0.1 mM pyridoxal phosphate (relative activity, 9.4%). Furthermore, HPLC and absorption spectrum analyses of the purified GriH protein showed the absence of any cofactors (data not shown).

Kinetics Analyses of Gril and GriH Reactions—We determined the kinetics of the Gril reaction by measuring the rate of increase in the 3,4-AHBA concentration in the presence of an excess of GriH. Because the reciprocal plots of the Gril reaction showed apparently parallel lines (Fig. 3D, panel a), the kinetic parameters were determined by fitting on the equation for the double-displacement mechanism (*k*~cat~, 0.20 ± 0.01 s⁻¹ for monomer; *K*~m~ for ASA, 5.6 ± 1.5 μM; *K*~m~ for DHAP, 140 ± 9 μM). The kinetics of the GriH reaction was determined by the rate of increase in the 3,4-AHBA concentration using the Gril-removed reaction mixture (see “Experimental Procedures”) as the substrate. The amount of the substrate for GriH (probably compound 1; see below) in the mixture was estimated on the basis of the amount of 3,4-AHBA produced by prolonged incubation of the mixture containing GriH and MnCl₂. The reciprocal plots of the GriH reaction followed the Michaelis-Menten kinetics (Fig. 3D, panel b). The *k*~cat~ and *K*~m~ values were calculated as 0.025 ± 0.002 s⁻¹ and 12 ± 2 μM, respectively. This *K*~m~ value for the GriH reaction might be underestimated because the concentration of the GriH substrate determined was perhaps lower than the actual concentration due to the instability of the substrate compound.

Proposed Reaction Pathway of 3,4-AHBA Formation—Although the product of the Gril reaction was extremely unstable, we detected a relatively stable product with absorption at 292 nm at pH 2.0 after prolonged Gril reaction (Fig. 2C, panel b, inset). Because the compound did not serve as a substrate for GriH, it must be a shunt product. The compound was also detected in the culture broth of *S. lividans* [pIJ4123-gril] and *S. griseus* [pAYP27] (data not shown). This shunt product was identified as 5,2-APC. We assumed that the actual product of

the Gril reaction was 2-amino-4,5-dihydroxy-6-one-heptanoic acid-7-phosphate (1, Fig. 1A), since 5,2-APC was presumably derived from 1 through a rationalized pathway (Fig. 4A). The pathway we propose is as follows. 1 is converted to a Schiff base 1b via formation of 1a by an intramolecular reaction. In 1b, the double bond migrates to produce 3a, which results in dephosphorylation to yield a product 3b. 3b is in equilibrium with 3c and 3d. Ring opening of 3d and subsequent ring closure afford 3f with a pyrrolidine ring, which produces 3g by dehydration. 5,2-APC is formed from 3g by migration of the double bonds. Gril thus catalyzed an aldolase reaction between the aldehyde carbon of ASA and the hydroxylated carbon of DHAP (Fig. 1A).

On the other hand, we detected no intermediates during the GriH reaction, suggesting that several steps in the GriH reaction proceeded consecutively in a substrate-binding pocket of GriH. Considering no requirement of GriH for cofactors involved in oxidative and reductive reactions, we propose a chemically rationalized pathway from the intermediate 1 to 3,4-AHBA (Fig. 4B). In the pathway, 1 is converted to a Schiff base 1b via formation of 1a by an intramolecular reaction. In 1b, the double bond migrates to produce a tautomeric Schiff base 1c, which is in equilibrium with the hydrated form 1d. A ketone 1e formed by dehydration of 1d facilitates dephosphorylation to give an enone 1f. The formation of an enone along with dephosphorylation is known for some enzyme reactions (26, 27). Ring opening of 1f and subsequent aldol condensation affords a carbocyclic compound 1h. A series of reactions from 1e to 1h seems to be reasonable, because similar reactions have been proposed for the DHQ synthase (27) (6 to DHQ in Fig. 1B). 3,4-AHBA is readily formed from the imino-ketone 1h by dehydration.

**Comparison of GriI-GriH System with an Alternative DHQ Synthesis Pathway of Archaea—Recently**, MJ0400 (a GriI homologue) and MJ1249 (a GriH homologue) of *M. jannaschii* were reported to be involved in an alternative pathway for DHQ biosynthesis (4). In this pathway, MJ0400 forms 2-amino-3,7-dideoxy-D-threo-hept-6-ulosonic acid (7, Fig. 1B) via a transaldol reaction between the dihydroxyacetone fragment of 6-deoxy-5-ketofructose-1-phosphate and ASA. DHQ is then formed by MJ1249 via NAD-dependent oxidative deamination of 7, producing 8, and subsequent cyclization. Because most Archaea lack the first two key enzymes involved in the synthesis of DHQ (3), MJ0400 and MJ1249 are thought to supply DHQ to the shikimate pathway (4). Interestingly, the reaction catalyzed by GriH is totally different from that catalyzed by MJ1249, despite sequence similarity between GriH and MJ1249. Much less conserved amino acid sequences in the N-terminal portions of GriH and MJ1249 may explain the difference of the reactions catalyzed by these enzymes. On the other hand, the reactions catalyzed by GriI and MJ0400 are analogous. Thus, the biosynthetic pathway of 3,4-AHBA may have evolved from the alternative DHQ synthesis pathway.

The genome data bases predict that griI and griH homologues are present in several bacteria (supplemental Figs. 1s and

2s). These griI and griH homologues are probably involved in the biosynthesis of 3,4-AHBA that serves as a building block of the respective secondary metabolites. Furthermore, griH homologues are found in the genomes of higher plants, such as *Arabidopsis thaliana* and *Oryza sativa*, giving the possibility that the benzene ring biosynthetic pathway involving GriH homologues is distributed widely in nature.

**REFERENCES**

1. Hillis, L. R., and Gould, S. J. (1985) *J. Am. Chem. Soc.* 107, 4593–4594
2. Knaggs, A. R. (2003) *Nat. Prod. Rep.* 20, 119–136
3. Woodard, R. W. (2004) *Bioorg. Chem.* 32, 309–315
4. White, R. H. (2004) *Biochemistry* 43, 7618–7627
5. Arakawa, K., Müller, R., Mahmud, T., Yu, T.-W., and Floss, H. G. (2002) *J. Am. Chem. Soc.* 124, 10644–10645
6. Rascher, A., Hu, Z., Buchanan, G. O., Reid, R., and Hutchinson, C. R. (2005) *Appl. Environ. Microbiol.* 71, 4862–4871
7. Funa, N., Ozawa, H., Hirata, A., and Horinouchi, S. (2006) *Proc. Natl. Acad. Sci. U. S. A.* 103, 6356–6361
8. Austin, M. B., and Noel, J. P. (2003) *Nat. Prod. Rep.* 20, 79–110
9. Ohnishi, Y., Furusho, Y., Higashi, T., Chun, H.-K., Furihata, K., Sakuda, S., and Horinouchi, S. (2004) *J. Antibiot.* 57, 218–223
10. Suzuki, H., Furusho, Y., Higashi, T., Ohnishi, Y., and Horinouchi, S. (2006) *J. Biol. Chem.* 281, 824–833
11. Gould, S. J., Melville, C. R., and Cone, M. C. (1996) *J. Am. Chem. Soc.* 118, 9228–9232
12. Hu, Y., and Floss, H. G. (2004) *J. Am. Chem. Soc.* 126, 3837–3844
13. Hopwood, D. A., Bibb, M. J., Chater, K. F., Kieser, T., Bruton, C. J., Kieser, H. M., Lydiate, D. J., Smith, C. P., Ward, J. M., and Schrempf, H. (1985) in *Genetic Manipulation in Streptomyces: A Laboratory Manual*, The John Innes Foundation, Norwich, UK
14. Katz, E., Thompson, C. J., and Hopwood, D. A. (1983) *J. Gen. Microbiol.* 129, 2703–2714
15. Takano, E., White, J., Thompson, C. J., and Bibb, M. J. (1995) *Gene (Amst.)* 166, 133–137
16. Maniatis, T., Fritsch, E. F., and Sambrook, J. (1982) *Molecular Cloning: A Laboratory Manual*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
17. Black, S., and Wright, N. G. (1955) *J. Biol. Chem.* 213, 39–50
18. Wang, X., Stumpf, D., and Larkins, B. A. (2001) *Plant Physiol.* 125, 1778–1787
19. Martyn, D. C., Vernall, A. J., Clark, B. M., and Abell, A. D. (2003) *Org. Biomol. Chem.* 1, 2103–2110
20. Westwood, I. M., Holton, S. J., Rodrigues-Lima, F., Dupret, J.-M., Bhakta, S., Noble, M. E. M., and Sim, E. (2005) *Biochem. J.* 385, 605–612
21. Copeland, R. A. (2000) *Enzymes. A Practical Introduction to Structure, Mechanism and Data Analysis*, 2nd Ed., pp. 109–123 and 355–357, Wiley-VCH, New York
22. Li, Y., Gould, S. J., and Proteau, P. J. (2000) *Tetrahedron Lett.* 41, 5181–5185
23. Hu, Y., Melville, C. R., Gould, S. J., and Floss, H. G. (1997) *J. Am. Chem. Soc.* 119, 4301–4302
24. Thiericke, R., Zeeck, A., Nakagawa, A., Omura, S., Herrold, R. E., Wu, S. T. S., Beale, J. M., and Floss, H. G. (1990) *J. Am. Chem. Soc.* 112, 3979–3987
25. Lorentzen, E., Pohl, E., Zwart, P., Stark, A., Russell, R. B., Knura, T., Hensel, R., and Siebers, B. (2003) *J. Biol. Chem.* 278, 47253–47260
26. Yeh, J. I., Du, S., Pohl, E., and Cane, D. E. (2002) *Biochemistry* 41, 11649–11657
27. Carpenter, E. P., Hawkins, A. R., Frost, J. W., and Brown, K. A. (1998) *Nature* 394, 299–302

Enzyme Catalysis and Regulation:  
Novel Benzene Ring Biosynthesis from C₃  
and C₄ Primary Metabolites by Two  
Enzymes  

Hirokazu Suzuki, Yasuo Ohnishi, Yasuhide  
Furusho, Shohei Sakuda and Sueharu  
Horinouchi  

J. Biol. Chem. 2006, 281:36944-36951.  
doi: 10.1074/jbc.M608103200 originally published online September 26, 2006  

Access the most updated version of this article at doi: [10.1074/jbc.M608103200](http://dx.doi.org/10.1074/jbc.M608103200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

Click here to choose from all of JBC's e-mail alerts  

Supplemental material:  
[http://www.jbc.org/content/suppl/2006/09/27/M608103200.DC1.html](http://www.jbc.org/content/suppl/2006/09/27/M608103200.DC1.html)  

This article cites 24 references, 7 of which can be accessed free at  
[http://www.jbc.org/content/281/48/36944.full.html#ref-list-1](http://www.jbc.org/content/281/48/36944.full.html#ref-list-1)
