
REVIEW

**Indolocarbazole natural products: occurrence, biosynthesis, and biological activity**

César Sánchez,$^{\dagger}$ Carmen Méndez and José A. Salas*

Received (in Cambridge, UK) 4th July 2006  
First published as an Advance Article on the web 19th October 2006  
DOI: 10.1039/b601930g  

Covering: up to June 2006  

The indolocarbazole family of natural products, including the biosynthetically related bisindolylmaleimides, is reviewed (with 316 references cited). The isolation of indolocarbazoles from natural sources and the biosynthesis of this class of compounds are thoroughly reviewed, including recent developments in molecular genetics, enzymology and metabolic engineering. The biological activities and underlying modes of action displayed by natural and synthetic indolocarbazoles is also presented, with an emphasis on the development of analogs that have entered clinical trials for its future use against cancer or other diseases.

---

### Table of Contents

1. Introduction
2. Indolocarbazoles from natural sources
   - 2.1 Actinomycetes
   - 2.2 Slime molds (myxomycetes)
   - 2.3 Cyanobacteria
   - 2.4 Marine invertebrates
   - 2.5 Other organisms
3. Biosynthesis of indolocarbazoles
   - 3.1 Studies made before the identification of biosynthetic genes
     - 3.1.1 Metabolic precursors
     - 3.1.2 Production of novel indolocarbazoles by altering the fermentation conditions
     - 3.1.3 Minor metabolites and blocked mutants
   - 3.2 Identification of genes involved in indolocarbazole biosynthesis
     - 3.2.1 Genes involved in rebeccamycin biosynthesis
     - 3.2.2 Genes involved in staurosporine biosynthesis
     - 3.2.3 Genes involved in biosynthesis of other indolocarbazoles
   - 3.3 Step-by-step description of indolocarbazole biosynthetic pathways
     - 3.3.1 Formation of the indolocarbazole ring system
       - 3.3.1.1 Tryptophan modification
       - 3.3.1.2 Formation of bisindole pyrrole (chromopyrrolic acid)
       - 3.3.1.3 Oxidative ring closure: formation of carbazole
     - 3.3.2 Formation of the sugar moiety
     - 3.3.2.1 Sugar moieties in rebeccamycin and AT 2433
     - 3.3.2.2 The staurosporine sugar moiety
     - 3.3.3 Regulation and self-resistance
     - 3.4 Implications for the biosyntheses of other natural products
4. Biological activities and modes of action
   - 4.1 Biological activities of indolocarbazoles
   - 4.2 Modes of action in mammalian cells
     - 4.2.1 Inhibition of protein kinases
     - 4.2.2 Inhibition of topoisomerases and interaction with DNA
   - 4.3 Modes of action in bacteria
5. Development of indolocarbazole analogs as clinical candidates
   - 5.1 Analogs targeting protein kinases
     - 5.1.1 Staurosporine derivatives
     - 5.1.2 K252a derivatives
     - 5.1.3 Indenopyrrolocarbazole derivatives
     - 5.1.4 Bisindolylmaleimide derivatives
   - 5.2 Analogs targeting DNA or DNA topoisomerases
     - 5.2.1 Rebeccamycin derivatives
     - 5.2.2 Analogs derived from BE-13793C
6. Perspectives and concluding remarks
7. Acknowledgements
8. References

---

### 1 Introduction

Since the isolation of the first indolocarbazole (ICZ) in 1977, this family of compounds has succeeded in attracting the attention of chemists, biologists, physicians, and pharmaceutical companies. The reason is the variety of chemical structures and, especially, the interesting biological activities exhibited by this group of compounds. Several ICZ analogs are currently being tested in the

---

Departmentamento de Biología Funcional & Instituto Universitario de Oncología del Principado de Asturias (I.U.O.P.A.), Universidad de Oviedo, 33006, Oviedo, Spain. E-mail: jasalas@uniovi.es; Fax: +34-985-103652; Tel: +34-985-103652  
$^{\dagger}$ Present address: EntreChem S.L., Edificio Científico Tecnológico, Campus El Cristo, 33006 Oviedo, Spain.

César Sánchez studied biology at the University of Oviedo, Spain, where he received his Ph.D. degree, in 1997, for his work on ribosomal protein genes in Streptomyces, under the guidance of Dr José A. Salas and Dr Gloria Blanco. He then worked as a postdoctoral researcher at the laboratory of Dr Ben Shen (University of California at Davis, US), studying the biosynthesis of bleomycin. In 2000 he was hired as a researcher at the Instituto Universitario de Oncología, University of Oviedo, working in the field of natural product biosynthesis.

Carmen Méndez graduated from the University of Oviedo, Spain, in 1984 with a Ph.D. based on the characterization of events that take place during the differentiation processes of colonies of Streptomyces antibioticus. After a post-doctoral stay in Keith Chater's laboratory in Norwich, UK, she returned to Oviedo, Spain, where she became Associate Professor in Microbiology in 1988. Her major interests are the cloning and characterization of biosynthetic pathways of antibiotics and antitumor compounds in actinomycetes, and the generation of novel compounds with potential antibiotic or antitumor activities through combinatorial biosynthesis.

José A. Salas graduated from the University of Oviedo, Spain, in 1980 with a Ph.D. on Streptomyces cell differentiation. After post-doctoral stays in David Ellar’s laboratory at Cambridge, UK, and Eric Cundliffes laboratory at Leicester, UK, he returned to Oviedo as Lecturer in Microbiology. Since 1999 he has been a Full Professor of Microbiology at the University of Oviedo. His major scientific interests are the isolation and characterization of biosynthetic pathways for bioactive compounds produced by actinomycetes and the generation of novel derivatives from natural products by combinatorial biosynthesis.


clinic for their future use against cancer or other diseases. In the past, several reviews have covered various aspects related to the isolation, synthesis, biological activities, or biosynthesis of ICZs (or wider groups of compounds which include ICZs). Most of the said reviews cover only some of these aspects, while the most complete ones are now quite outdated.

The present review deals with the ICZ family of natural products, including the biosynthetically related bisindolylmaleimides. We summarize the occurrence of ICZs obtained from diverse natural sources, from microorganisms to marine invertebrates. The biosynthesis of this class of compounds is thoroughly reviewed, including recent developments in the molecular genetics, enzymology and metabolic engineering of ICZ pathways in microorganisms. The biological activities and underlying modes of action displayed by natural and synthetic ICZs are also presented. Finally, we review the development of ICZ analogs that have already entered clinical trials.

Different arrangements are possible for indole and carbazole to yield the five isomeric ring systems available for ICZs: indolo[2,3- *a*]carbazole 1, indolo[2,3-*b*]carbazole 2, indolo[2,3-*c*]carbazole 3, indolo[3,2-*a*]carbazole 4, and indolo[3,2-*b*]carbazole 5. However, almost all of the ICZs isolated from nature are indolo[2,3-*a*]carbazoles, and in general, interest has been focused on this isomeric class. The remaining isomeric classes have been much less studied. Derivatives of bisindolylmaleimide 6 are considered to be a part of the family of ICZ compounds, due to their close biosynthetic relationships (which are reviewed here).

2 Indolocarbazoles from natural sources

During the last 30 years, a variety of ICZs have been isolated from different organisms, including bacteria, fungi, and invertebrates. In this section we review the occurrence of 129 compounds isolated either from field-collected samples (slime molds, marine invertebrates), or from microorganisms cultured under standard fermentation conditions (actinomycetes, cyanobacteria, and others). Table 1 summarizes the available information, by listing the organisms, their metabolites, and their diverse geographical origins. Other compounds, which were obtained under non-standard fermentation conditions (such as feeding non-natural precursors) or from mutated strains, are reviewed in Section 3.1. The biological activities that were associated with the discovery of the natural products are mentioned here, while additional activities and the mechanisms of action involved are reviewed in Section 4.

### 2.1 Actinomycetes

The isolation of an ICZ from a natural source was reported for the first time in 1977. The alkaloid, initially named AM-2282, was extracted from cultures of *Streptomyces staurosporeus* AM-2282 (ATCC 55006), which was collected from a soil sample. Soon afterwards, alkaloid AM-2282 was renamed staurosporine (STA), and its X-ray crystal structure 7 was determined. The structure consisted of a chromophore, derived from indolo[2,3-*a*]pyrrolo[3,4-*c*]carbazole, to which an aminopyranose was joined through two linkages, but the structure originally assigned (*ent*-STA 8) was incorrect. The correct relative stereochemistry 7 was not established until 1994, and it was later confirmed by total synthesis. The species epithet “*staurosporeus*” (from which the word “staurosporine” was derived) came from the Greek words “*stauros*” and “*spora*” meaning “stave” and “spore”, respectively. Later, the original producer strain was re-classified twice (Table 1), first as *Saccharothrix aerocolonigenes* subsp. *staurosporea* and, recently, as *Lentzea albida*. The latter name will be used for this strain hereinafter.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

STA showed inhibitory activity against fungi, but was generally inactive against most bacteria. The compound was found to possess strong hypotensive activity. STA started to attract greater attention in 1986, when it was reported that it was an extraordinarily potent inhibitor of protein kinase C (PKC) and that it showed a strong cytotoxic effect on cancer cells. Soon it was found that STA was a non-specific inhibitor of protein kinases. Since protein kinases are a large family of enzymes that mediate the response of eukaryotic cells to a wide variety of external stimuli, it is not surprising that STA has so many biological effects.

STA has been repeatedly isolated from organisms collected in remarkably different environments (Table 1). One reason for this frequent re-discovery is the wide variety of biological activities displayed by the compound; i.e., STA is a “frequent hitter”, which is found to be very active on different screening assays covering a wide range of targets. Nevertheless, it appears that production of STA (or very close derivatives) is widespread among actinomycetes, a bacterial group commonly found in environmental samples (especially from soil).

After screening over one thousand samples of culture broths from *Streptomyces* strains in an antitumor screening assay, a compound that turned out to be identical to STA was isolated from *Streptomyces actuosus*. The compound was also isolated from *Streptomyces* strain M-192, during a screening of platelet aggregation inhibitors. The same metabolite was later identified from the following strains: *Streptomyces hygroscopicus* C 39280-450-9 (ATCC 53730), *Streptomyces* sp. TP-A 0274, and *Streptomyces lividus* ATCC 21178 (Table 1). The isolation of further actinomycetes that produced STA together with a number of related compounds is reviewed below.

In 1985, a new ICZ was reported from cultures of *Nocardia aerocolonigenes* C-38383-RK-2 (ATCC 39243). The novel alkaloid was named “rebeccamycin” (REB) after the daughter of the scientist who isolated the compound. The structure and absolute configuration of REB 9 were determined by X-ray crystallography and total synthesis. The producer microorganism was soon renamed *Saccharothrix aerocolonigenes*, and it has been recently re-classified as *Lechevalieria aerocolonigenes* (this name will be used hereinafter). REB was found to be active against leukemia and melanoma implanted in mice, it inhibited the growth of human lung adenocarcinoma cells, and produced single-strand breaks in the DNA of these cells. Additionally, minor components of extracts from cultures of *L. aerocolonigenes* were identified as 11-deschloro-REB 10, 1,11-dideschloro-REB 11, and 4′-*O*-demethyl-1,11-dideschloro-REB (BMY-41219). These compounds showed antitumor activity. Additionally, 11-deschloro-REB 10 showed antibacterial activity.

In 1985, the isolation and structural elucidation of SF-2370 13 was reported from the culture broth of *Actinomadura* sp. SF-2370. SF-2370 was weakly active against some bacteria and fungi, and displayed protective effects against rice plant diseases caused by bacteria. A year later, the isolation of an ICZ, named K252a, was reported from culture broths of *Nocardiopsis* sp. K-252 (NRRL 15532). The producer microorganism was later

Table 1 Natural sources of compounds of the indolocarbazole family

| Organism                                                                 | Geographical origin                                                                                   | Compounds          | References       |
|--------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|---------------------|------------------|
| Actinomycetes                                                             |                                                                                                      |                     |                  |
| Streptomyces staurosporeus AM-2282 = Lentzea albida (ATCC 55006 = NRRL 11184) = Saccharothrix aerocolonigenes subsp. staurosporea | Iwate Prefecture, Japan (soil)                                         | 7, 27             | 14,15           |
| Streptomyces actuosus                                                    | Not reported                                                                                        | 7                   | 16               |
| Nocardia aerocolonigenes C-38383-RK-2 = Saccharothrix aerocolonigenes = Lechevalieria aerocolonigenes (ATCC 39243) | Panama (soil)                                                              | 9–12              | 17–20           |
| Actinomadura sp. SF-2370                                                 | Shimizu, Shizuoka Prefecture, Japan (soil)                                                           | 13                  | 21               |
| Nocardiopsis sp. K-252 (NRRL 15532) = Nonomuraea longicatena K-252T (NRRL15532T) | Ashahi-cho, Machida-shi, Tokyo, Japan (soil)                                                        | 13                  | 22,23            |
| Nocardiopsis sp. K-290                                                   | Tama-shi, Tokyo, Japan (soil)                                                                       | 13–16              | 23,24            |
| Streptomyces sp. M-193                                                   | Not reported (soil)                                                                                 | 7                   | 25               |
| Streptomyces sp. N-71 = Streptomyces sp. UCN-01 (FERM BP-990)              | Tsuno-gun, Yamaguchi Prefecture, Japan (soil)                                                       | 7, 17              | 26–28            |
| Streptomyces sp. N-115                                                   | Kyoto, Japan (soil)                                                                                | 7                   | 27               |
| Streptomyces sp. N-126                                                   | Okinawa, Japan (soil)                                                                              | 7, 17, 18          | 27,29            |
| Streptomyces sp. N-128                                                   | Okinawa, Japan (soil)                                                                              | 7                   | 27               |
| Streptomyces sp. N-139                                                   | Nagano, Japan (soil)                                                                               | 7                   | 27               |
| Actinomadura melliaura SCC 1655 (ATCC 39691)                             | Bristol Cove, California, US (soil)                                                                 | 19–22              | 30,31            |
| Nocardiopsis dassonvillei C-71425                                        | Tottori Prefecture, Japan (soil)                                                                   | 23                  | 32,33            |
| Streptomyces sp. C-71799                                                 | Okinawa Prefecture, Japan (soil)                                                                   | 7, 24              | 32,33            |
| Streptomyces hygroscopicus C 39280-450-9 (ATCC 53730)                    | Numazu Prefecture, Japan (soil)                                                                    | 7, 27              | 15,34            |
| Streptomyces sp. RK-286                                                  | Nanao-shi, Ishikawa Prefecture, Japan (soil)                                                       | 7, 15, 25, 26      | 35–37            |
| Saccharothrix aerocolonigenes subsp. copiosa SCC 1951 = Lechevalieria aerocolonigenes (ATCC 53856) | Spain (soil)                                                                                       | 7, 29–42           | 38               |
| Streptoverticillium mobaraense BA13793 = Streptomyces mobaraensis (FERM P-10489 = FERM BP-2785) | Seto, Aichi Prefecture, Japan (soil)                                                               | 43                 | 39               |
| Streptomyces platensis subsp. malvinus RK-1409 (FERM-P11952)              | Atou, Yamaguchi Prefecture, Japan (soil)                                                            | 7, 27, 28          | 40–42            |
| Streptomyces sp. AB1869R-359 (NRRL B-16735)                              | Dorado, Puerto Rico (soil)                                                                         | 7, 44              | 43               |
| Streptomyces longisporoflavus R-19 (DSM 10189)                            | Ellora, India (soil)                                                                               | 7, 13, 15, 24, 45–57 | 44–46            |
| Streptomyces sp. N96C-47                                                 | Holyrood, Newfoundland, Canada (marine sediment)                                                    | 7, 16, 58–60       | 47               |
| Streptomyces sp. TA-0403 (FERM P-17190)                                  | Aki-gun, Kochi Prefecture, Japan (soil)                                                             | 63, 64             | 48–50            |
| Micromonospora sp. L-31-CLCO-002 (CECT 3347)                             | Fuerteventura Island, Canary Islands, Spain (from a sponge homogenate)                               | 7, 61, 62          | 51               |
| Streptomyces sp. TP-A0274                                                | Kosugi-machi, Toyama, Japan (soil)                                                                  | 7                   | 52               |
| Streptomyces lividus ATCC 21178                                          | Not reported (soil)                                                                                 | 7                   | 53               |
| Actinomadura sp. strain 007                                              | Jiaozhou Bay, China (marine sediment)                                                               | 65                 | 54               |
| Strain NPS 012745 (ATCC PTA-5748)                                        | Mission Bay, California, US (marine sediment)                                                       | 66–71              | 55,56            |
| Slime molds (myxomycetes)                                                |                                                                                                      |                     |                  |
| Arcyria denudata, Arcyria nutans                                         | Not reported                                                                                        | 6, 72–76, 78–89    | 2,57             |
| Metatrichia vesparium                                                    | Not reported                                                                                        | 75, 76             | 58               |
| Dictydiaethalium plumbeum                                                | Not reported                                                                                        | 77                  | 2                |
| Lycogala epidendrum                                                      | Tokushima, Japan                                                                                    | 68, 90, 91         | 59,60            |
| Lycogala epidendrum                                                      | Not reported                                                                                        | 6, 15, 68, 74, 90, 91, 92 | 61               |
| Arcyria ferruginea                                                      | Hao, Yasucho, Kochi Prefecture, Japan                                                                | 73, 76, 93         | 62               |
| Tubifera casparyi                                                       | Mt. Mitsumine, Monobe-mura, Kochi Prefecture, Japan                                                 | 75, 76             | 62               |
| Lycogala epidendrum                                                      | Kochi Prefecture, Japan                                                                             | 15, 68, 90, 92, 96–98 | 63               |
| Arcyria cinerea                                                         | Kochi Prefecture, Japan                                                                             | 6, 94, 95          | 63               |
| Cyanobacteria                                                            |                                                                                                      |                     |                  |
| Nostoc sphaericum EX-5-1                                                 | Manoa, Hawaii, US (mud sample)                                                                      | 99–101             | 64               |
| Tolypothrix tijipanasensis DB-1-1A                                       | Vero Beach, Florida, US (soil)                                                                     | 102–116            | 65               |
| Fischerella ambigua strain 108b                                          | Mellingen, Switzerland (terrestrial sample)                                                          | 114                 | 66,67            |
| Marine invertebrates                                                     |                                                                                                      |                     |                  |
| Eudistoma sp. (tunicate)                                                 | Sapwale Bay, Pohnpei, Micronesia (shallow water)                                                    | 117, 118           | 68               |
| Eudistoma sp. (tunicate)                                                 | Santiago, Cape Verde (deep water)                                                                   | 15, 74             | 69               |
| Coriocella nigra (mollusk)                                               | Not reported                                                                                        | 118, 119           | 70               |
| Eudistoma toealensis (tunicate) and Pseudoceros sp. (flatworm)            | Truk Lagoon, Chuuk, Micronesia (shallow water)                                                      | 7, 15, 45, 58, 117, 119–125 | 71–73            |
| Ancorina sp. (sponge)                                                    | Chatham Island and Three Kings Islands, New Zealand                                                 | 126                 | 74               |
| Other organisms                                                          |                                                                                                      |                     |                  |
| Chromobacterium violaceum JCM 1249 (ATCC 12472)                          | Malaysia (fresh water)                                                                              | 92, 127            | 75–77            |
| Malassezia furfur CBS 1878                                               | (human skin)                                                                                       | 128–135            | 78–80            |

re-classified as *Nonomuraea longicatena.*<sup>93</sup> Structural elucidation of K252a proved that the metabolite was identical to the previously reported SF-2370 13,<sup>23</sup> and the name K252a has since been used for this compound.

![Chemical Structure](chemical_structure_1.png)

13K252a (R=Me)  
14K252b (R=H)

![Chemical Structure](chemical_structure_2.png)

16  
K252d

A different actinomycete strain, *Nocardiopsis* sp. K-290, was found to produce K252a 13 together with the related compounds K252b 14, K252c 15 and K252d 16.<sup>24</sup> The four metabolites shared the same STA chromophore, and K252c 15 was the aglycone itself.<sup>23</sup> K252a and K252b exhibited very potent inhibitory activities on PKC. K252c and K252d also inhibited PKC, although with less potency.<sup>22,24</sup> The four compounds showed cytotoxic properties, while lacking antimicrobial activity against the microorganisms tested.<sup>22,24</sup>

Fermentation broths from 5163 new soil isolates were tested during a screening program for selective inhibitors of PKC, and five streptomycetes (strains N-71, N-115, N-125, N-127 and N-138) were found to produce STA together with related compounds.<sup>27</sup> In other words, at least 0.1% of newly isolated actinomycetes were shown to produce STAs using only one culture condition. *Streptomyces* sp. N-71 additionally produced 7-hydroxy-STA, or UCN-01 17, which was a potent inhibitor of PKC, and showed *in vivo* antitumor activity<sup>26,28</sup> A second strain, *Streptomyces* sp. N-125, was found to produce UCN-01 17 and its stereoisomer, UCN-02 18, together with STA (major product).<sup>27,29</sup>

![Chemical Structure](chemical_structure_3.png)

17UCN-01 (R1=OH, R2=H)  
18UCN-02 (R1=H, R2=OH)

Culture broths of *Actinomadura melliaura* SCC 1655 (ATCC 39691) contained AT 2433-A1 19 (major product), AT 2433-A2 20, AT 2433-B1 21 and AT 2433-B2 22.<sup>30,31,94</sup> The compounds were similar to REB, but with an aminodisaccharide and an N-methylimide. The absolute stereochemistry of the aminosugar moiety was later revised by total synthesis of AT 2433-B1 21.<sup>95</sup>

![Chemical Structure](chemical_structure_4.png)

15  
K252c  
(STA aglycone)

19AT2433-A1 (R1=Cl, R2=Me)  
20AT2433-A2 (R1=Cl, R2=H)  
21AT2433-B1 (R1=H, R2=Me)  
22AT2433-B2 (R1=R2=H)

The isolation of two new STA derivatives with macrophage-activating properties, named TAN-999 23 and TAN-1030A 24, was reported in 1989.<sup>32,33</sup> TAN-999 23, or 10-methoxy-STA, was extracted from culture broths of *Nocardiposis dassonvillei* C-71425, while oxime TAN-1030A 24 was produced (in addition to STA) by cultivating *Streptomyces* sp. C-71799. TAN-999 inhibited PKC, and displayed antitumor properties.<sup>32</sup>

![Chemical Structure](chemical_structure_5.png)

23  
TAN-999  

![Chemical Structure](chemical_structure_6.png)

24  
TAN-1030A  

In the course of a screening program for PKC inhibitors, several strains were found to produce STA-related compounds. One of these strains, *Streptomyces* sp. RK-286, produced RK-286C 25 and RK-286D 26, together with STA (major product) and K252c 15.<sup>35–37</sup> The inhibitory activities of 25 and 26 against PKC were much weaker than that of STA (among the three compounds, RK-286D was the weakest inhibitor). However, the inhibition of platelet aggregation of RK-286C was as strong as that of STA.<sup>36</sup>

In 1990, a patent application disclosed a new STA derivative, BMY-41950 or 7-oxo-STA 27, which was isolated from two known STA producers (Lentzea albida ATCC 55006 and S. hygroscopicus ATCC 53730). ${ }^{15}$ 7-Oxo-STA 27 is a hybrid of a STA-type sugar moiety and a REB-type 7-oxo chromophore. Production of an identical metabolite, under the name RK-1409, was also reported from a new STA producer, Streptomyces platensis subsp. malvinus RK-1409. ${ }^{40,41}$ In comparison to STA, its 7-oxo analog was a more potent inhibitor of PKC but it was a weaker inhibitor of bleb formation and cell cycle progression, perhaps due to lower membrane permeability. The same strain, S. platensis subsp. malvinus RK-1409, produced RK-1409B 28, a C-3' epimer of RK-286C 25, as a minor component. ${ }^{42}$ While showing similar inhibitory activity to 25 on the bleb formation assay, RK-1409B was a weaker inhibitor of PKC than its epimer.

A new STA producer, Saccharothrix aerocolonigenes subsp. copiosa SCC 1951 (Lechevalieria aerocolonigenes ATCC 53856), was disclosed in a patent application in 1991. ${ }^{38}$ The strain produced small quantities of a complex mixture of about 20 ICZ components. From 1000 L of culture broth, 14 compounds (29–42) were purified and their structures determined, but their absolute stereochemistry was not reported (therefore, compound 34 might be identical to 25 or 28). It was claimed that the isolated compounds were inhibitors of PKC and MLCK (myosin light chain kinase), with higher selectivity than STA. In the same year, the isolation of BE-13793C (J-104303) 43 was reported from Streptoverticillium mobaraense BA 13793 (Streptomyces mobaraensis). ${ }^{39}$ BE-13793C inhibited topoisomerases I and II, and showed antitumor properties.

In a search for immunomodulators, it was found that culture extracts of Streptomyces sp. AB1869R-359 contained MLR-52, ${ }^{44}$ together with higher amounts of STA. ${ }^{43}$ The morphological and physiological characteristics of this microorganism were compared to those of other STA-producing Streptomyces species, confirming that it was a different strain. MLR-52 was nearly as potent as STA in inhibiting the mitogenic response in mixed lymphocyte cultures, and both metabolites showed in vitro immunosuppressive potencies similar to FK-506 or cyclosporine. MLR-52 also inhibited PKC.

STA-related metabolites were thoroughly studied in another STA producer, Streptomyces longisporoflavus R-19. ${ }^{44-46}$ By using 2000 L fermenters, 18 different compounds were isolated and

characterized (7, 13, 15, 17, 24, 45–57). The main product was STA, followed by much smaller amounts of K-252c 15 and TAN-1030A 24. Structural variations were mainly found at the 4′-position of the sugar moiety. Additionally, furanosylated derivatives, such as K252a 13 and its aminomethyl derivative 56, were found. All the tested compounds inhibited PKC. As compared to STA, the new derivatives were less potent inhibitors against several protein kinases, with the exception of 4′-N-methyl-STA 45, which showed a similar potency to that of STA.

Crude extracts from cultures of a marine actinomycete, Streptomyces sp. N96C-47, gave a promising profile of in vitro cytotoxicity against a panel of human cancer cell lines. Bioassay-guided fractionation of culture extracts yielded STA (major product), K252d 16, 3′-O-demethyl-STA 58, holyrine A 59 and holyrine B 60.⁴⁷ STA accounted for the interesting cytotoxicity profile observed for the crude extract. Holyrines A and B were only produced in significant amounts at 15 °C. Holyrine A exhibited potent inhibitory activity against some protein kinases. Another marine actinomycete, Micromonospora sp. L-31-CLCO-002, was found to produce STA (major product), 4′-N-methyl-5′-hydroxy-STA 61, and 5′-hydroxy-STA 62.⁵¹ The microorganism had been isolated from a homogenate of the sponge Clathrina coriacea. Compounds 61 and 62 showed very potent cytotoxic activities against tumor cell lines.

To search for natural products that could be useful to treat patients with neurological diseases, a screening was performed based on the differentiation of rat pheochromocytoma PC12 cells. Among 4671 extracts (2532 extracts from actinomycetes, 1958 extracts from fungi, and 179 extracts from plants), only those produced by a single microbial strain, *Streptomyces* sp. TA-0403, showed activity of interest. The active substances were indocarbazostatin 63 and indocarbazostatin B 64, which were related to K252a 13.⁴⁸⁻⁵⁰ The order of inhibitory activity on neurite outgrowth was indocarbazostatin > indocarbazostatin B > K252a > K252b, while the order of inhibitory activity against PKC was STA > indocarbazostatin > indocarbazostatin B > K252a ≫ K252b. The two indocarbazostatins also showed cytotoxicity at concentrations higher than those inhibiting neurite outgrowth.⁴⁹ It is noteworthy that, while K252a inhibited NGF-induced neurite outgrowth in PC12 cells,⁹⁶,⁹⁷ STA was an inducer of neurite outgrowth in the same cell line.⁹⁸

---

**63 indocarbazostatin (R=H)**  
**64 indocarbazostatin B (R=NH₂)**  

---

**2.2 Slime molds (myxomycetes)**

The myxomycetes (or plasmodial slime molds) are a group of fungus-like eukaryotic organisms usually present in terrestrial ecosystems. All the metabolites mentioned in this section were extracted from fruit bodies of myxomycetes directly collected from the field.

The main pigments of *Arcyria denudata* and *A. nutans* were found to be the bisindolylmaleimides arcyiarubins A, B, and C (6, 72, 73), and their derived indolopyrrolocarbazoles arcyiaflavins A, B, and C (74–76).²,⁵⁷ Arcyiaflavins B 75 and C 76 were also found in *Metatrichia vesparium*,⁵⁸ while a new derivative, arcyiaflavin D 77, was isolated from *Dictydiaethalium plumbeum*.²

A large number of additional bisindoles were isolated from *Arcyria denudata* and *A. nutans*, including arcyroxepins 78 and 79, arcyriacyanin A 80, dihydroarcyriacyanin A 81, arcyroxocins 82 and 83, dihydroarcyroxocin A 84, arcyroxindole A 85, dihydroarcyiarubin B 86, and arcyriaverdin C 87. All of these compounds might be biosynthetically derived from arcyiarubin A 6 by several oxygenations and oxidative cyclizations.² Arcyriacyanin A 80 was demonstrated to have unique inhibitory activity on a panel of human cancer cell lines and to inhibit PKC and protein tyrosine kinase.⁹⁹ Extracts of *Arcyria* species also contained the quinolopyrrolocarbazoles arcyrin A 88 and arcyrinin B 89, which might also be biogenetically related to the bisindolylmaleimides.²

Novel bisindole compounds have been reported from *Lycogala epidendrum*.⁵⁹⁻⁶¹ The three major compounds were all bisindolylpyrrole dicarboxylic acid derivatives (68, 90, 91), which were named (by two different laboratories) either lycogalic acid dimethyl esters A, B, and C, or lycogarubins C, B, and A, respectively. In addition, extracts from *L. epidendrum* also yielded

Recently, a group of bisindole pyroles 66–71, related to bisindolylmaleimide 6, were reported to be produced by actinomycete strain NPS012745.⁵⁵,⁵⁶ Some of the reported compounds showed antibacterial activity. Compound 68 was identical to lycogarubin C (or lycogalic acid dimethyl ester A), a natural product previously isolated from a slime mold⁵⁹⁻⁶¹ (see next section).

lycogalic acid A 92 and K252c 15, together with traces of 6 and 74.⁶¹ Compound 68 showed moderate antiviral activity against herpes simplex virus type I (HSV-I).⁵⁹ Lycogalic acid A 92 is identical to chromopyrrolic acid, previously reported to be isolated from the bacterium *Chromobacterium violaceum*⁷⁵ (Section 2.5). The name chromopyrrolic acid is used hereinafter for compound 92.

6 arcyiarubin A (R1=R2=H)
72 arcyiarubin B (R1=H, R2=OH)
73 arcyiarubin C (R1=R2=OH)

74

arcyiaflavin A

75 arcyiaflavin B (R1=R2=H, R3=OH)
76 arcyiaflavin C (R1=H, R2=R3=OH)
77 arcyiaflavin D (R1=R3=OH, R2=H)

68 lycogalic acid dimethyl ester A, lycogarubin C (R1=R2=H)
90 lycogalic acid dimethyl ester B, lycogarubin B (R1=H, R2=OH)
91 lycogalic acid dimethyl ester C, lycogarubin A (R1=R2=OH)

92

chromopyrrolic acid
(lycogalic acid A)

Extracts from *Arcyria ferruginea* contained dihydroarcyiarubin C 93, together with 73 and 76.⁶² The relative stereochemistry of 93 was not determined. Additionally, *Tubifera casparyi* contained

78 arcyroxepin A (R=H)
79 arcyroxepin B (R=OH)

80 arcyriacyanin A
81 dihydroarcyriacyanin A
82 arcyroxocin A (R=H)
83 arcyroxocin B (R=OH)

84 dihydroarcyroxocin A
85 arcyroxindole A
86 dihydroarcyiarubin B

87 arcyriaverdin C
88 arcyrin A (R1=H, R2=OH)
89 arcyrinin B (R1=OH, R2=H)

Arcyriaflavins B 75 and C 76.62 Arcyriaflavin C 76 induced cell-cycle arrest (at lower concentrations) and showed cytotoxicity (at higher concentrations). The cell cycle inhibition effect of arcyriarubin C 73 was weaker than for 76.62

![Structure](https://i.imgur.com/1234567.png)

**93 dihydroarcyriarubin C**

Cinereapyroles A and B 94 and 95 were isolated, together with 6, from *Arcyria cinerea*.63 Additionally, the authors reported the isolation of three new related alkaloids 96–98, together with known metabolites (15, 68, 74, 75, 90, 92), from *Lycogala epidendrum*.63 Arcyriaflavin B 75 was evaluated in a 39 cell-line panel assay at the Japanese Foundation for Cancer Research, and showed moderate antitumor activity.

![Structure](https://i.imgur.com/2345678.png)

**94 cinereapyrole A (R=OH)**  
**95 cinereapyrole B (R=H)**

![Structure](https://i.imgur.com/3456789.png)

**96 (R1=R2=H)**  
**97 (R1=H, R2=OH)**  
**98 (R1=OH, R2=H)**

### 2.3 Cyanobacteria

In 1990, ICZ metabolites were isolated for the first time from cyanobacteria (“blue-green algae”), a group of photosynthetic prokaryotes. Extracts from cultures of *Nostoc sphaericum* EX-5-1 possessed moderate antiviral activity against herpes simplex virus type 2 (HSV-2) and weak, non-selective cytotoxicity against tumor cell lines.64 The biological activity of the extracts was due to the presence of 6-cyano-5-methoxy-12-methylindolo[2,3-*a*]carbazole 99 (major product), its regioisomer 100, and demethyl analog 101.

![Structure](https://i.imgur.com/4567890.png)

**99**  
**100 (R=Me)**  
**101 (R=H)**

Unlike most ICZs, these compounds did not possess the annelated pyrrolo[3,4-*c*] unit.

Tjipanazoles 102–116 were extracted from culture extracts of *Tolypothrix tjipanasensis* DB-1-1A.65 Tjipanazole J 116 was the only compound possessing a pyrrolo[3,4-*c*] ring. Compounds 102 and 106 exhibited activity against phytopathogenic fungi. Tjipanazoles showed only weak cytotoxicity against tumor cell lines and were inactive as inhibitors of PKC.65 Tjipanazole D 114 was also isolated from cultures of *Fischerella ambigua*.66,67

### 2.4 Marine Invertebrates

Invertebrates such as sponges, tunicates, bryozoans, or mollusks are prolific sources of natural products from the seas, although, in many cases, associated microorganisms are known or suspected to be the biosynthetic source of the compounds. In 1991, it was reported that extracts from the tunicate *Eudistoma* sp. showed potent cytotoxic activity.68 Purification from these extracts yielded 11-hydroxy-STA 117 and 3,11-dihydroxy-STA 118 (a minor compound), which were the first ICZs isolated from a marine organism. Both compounds were active against cancer cells, and 11-hydroxy-STA was more potent than STA in PKC inhibition. K252c 15 and arcyriaflavin A 74 were later isolated from a specimen of *Eudistoma* sp. collected in a different ocean69 (Table 1). Additionally, 4′-N-demethyl-11-hydroxy-STA 119 was extracted from the prosobranch mollusk *Coriocella nigra*, together with 3,11-dihydroxy-STA 118.70

Twelve STA derivatives (7, 15, 45, 58, 117, 119–125) were isolated from the colonial tunicate *Eudistoma toealensis* and its predatory flatworm *Pseudoceros* sp. (which was feeding on the tunicates).71–73 The major products were STA 7 and 4′-N-demethyl-STA 120. The flatworms seemed to accumulate the metabolites from *E. toealensis*, since they did not show any STAs when collected from a different tunicate found nearby in the same habitat.71,72 Some of the derivatives were tested as inhibitors of cell proliferation with 12 human leukemia cell lines, and demonstrated powerful antiproliferative activities, with 3-hydroxy-STA 123 being the most potent (even more potent than STA itself).100 The co-occurrence of STA analogs in marine filter-feeders and their predators may serve as an indication for the origin of the compounds through the food chain, perhaps by filter-feeding of microorganisms. Production of STAs by marine actinomycetes47,51 supports this hypothesis.

An indolo[3,2-*a*]carbazole natural product (ancorinazole 126) was extracted from the sponge *Ancorina* sp.74 To the best of our knowledge, this is the only indolo[3,2-*a*]carbazole reported from a natural source.

### 2.5 Other Organisms

Here we review some bisindole natural products isolated from other groups of organisms. Although these compounds are not ICZs *sensu stricto*, they appear to be biosynthetically related. Violacein 127 is a bisindole pigment produced by several bacteria, including *Chromobacterium violaceum*.76,77 This microorganism was also the source of chromopyrrolic acid 92.75

102 tijpanazole A1 (R1=R2=Cl)
103 tijpanazole C1 (R1=Cl, R2=H)
104 tijpanazole C2 (R1=H, R2=Cl)
105 tijpanazole G1 (R1=R2=H)

106 tijpanazole A2 (R1=R2=Cl)
107 tijpanazole C3 (R1=Cl, R2=H)
108 tijpanazole C4 (R1=H, R2=Cl)
109 tijpanazole G2 (R1=R2=H)

110 tijpanazole B (R1=R2=Cl)
111 tijpanazole F1 (R1=Cl, R2=H)
112 tijpanazole F2 (R1=H, R2=Cl)

113
tijpanazole E

114 tijpanazole D (R=Cl)
115 tijpanazole I (R=H)
116
tijpanazole J

117 (R=H)
118 (R=OH)

119

120 (R=H)
121 (R=OH)

122

123 (R=Me)
124 (R=H)

125

126
ancorinazole

127
violacein

A number of indole alkaloids were isolated from tryptophan-enriched cultures of *Malassezia furfur*, a lipophilic yeast (basidiomycete) belonging to the resident flora of human skin. Pityri- anhydride 128, pityrialactone 129 and pityriarubins 130–132 were closely related to arcyiarubin A 6. These metabolites might form a broad photoprotection system, apparently related to some clinical features of pityriasis versicolor, a skin disease caused by *M. furfur*. In particular, pityriarubins might act *in vivo* as immunomodulators in the course of the disease. Additionally, among the metabolites isolated from *M. furfur*, malasseziazoles 133–135 were remarkable for containing an indolo[3,2-b]carbazole unit. Malasseziazoles appear to be the only indolo[3,2-b]carbazoles so far isolated from a natural source, if we exclude the formation of

indolo[3,2-b]carbazole 5 in the acidic environment of the stomach after intake of indole-3-carbinol. ${}^{101}$

$\begin{array}{cc}
\includegraphics{image.png} & \includegraphics{image.png} \\
128 & 129 \\
\text { pityrianhydride } & \text { pityrialactone }
\end{array}$

$\begin{array}{cc}
\includegraphics{image.png} & \includegraphics{image.png} \\
130 & 131 \text { pityriarubin B (R=NH) } \\
\text { pityriarubin A } & 132 \text { pityriarubin C (R=O) }
\end{array}$

$\begin{array}{c}
\includegraphics{image.png} \\
133 \text { malasseziazole A (R1=COOH, R2=H) } \\
134 \text { malasseziazole B (R1=COOH, R2=CHO) } \\
135 \text { malasseziazole C (R1=OH, R2=CHO) }
\end{array}$

$\begin{array}{ccc}
\includegraphics{image.png} & \rightarrow & \includegraphics{image.png} \\
2x & & 9 \\
\text { tryptophan } & & \text { REB }
\end{array}$

$\begin{array}{ccc}
\includegraphics{image.png} & \rightarrow & \includegraphics{image.png} \\
\text { glucose } & & 7 \\
& & \text { STA }
\end{array}$

$\begin{array}{ccc}
\includegraphics{image.png} & \rightarrow & \includegraphics{image.png} \\
\text { methionine } & &
\end{array}$

**Scheme 1** Metabolic precursors for the biosynthesis of rebeccamycin and staurosporine.

indolepyruvate was an intermediate in REB biosynthesis, while tryptamine was not incorporated.¹⁰⁸ On the other hand, feeding tryptamine to *Lentzea albida* resulted in efficient incorporation into STA, suggesting that tryptamine was a precursor in STA biosynthesis.¹⁰⁴ However, the results of the loss of the α-amino nitrogen on tryptophan and the loss of both α-protons in tryptamine-labeling experiments suggested that, in the major pathway, either tryptophan or tryptamine underwent oxidation on the α-carbon, leading to complete loss of the side chain nitrogen and the α-protons, respectively, before the incorporation of the indole units into STA.¹⁰⁴ The apparently conflicting results obtained with the two strains were perhaps due to the different experimental conditions employed, in addition to putative metabolic differences existing between the two organisms (e.g., transaminases and other enzymes not directly involved in ICZ biosynthesis). Recently, the identity of early intermediates has been established through *in vitro* experiments using purified biosynthetic enzymes (Section 3.3).

The biosynthesis of indocarbazostatin B 64 was also studied by feeding radiolabeled precursors. As expected, tryptophan was incorporated into the aglycone, while glucose was used for formation of the sugar moiety.¹⁰⁹ The labeling pattern found for the carbohydrate supported previous hypothesis on the biosynthesis of furanosylated ICZs, such as K252a (see Section 3.1.3).

Studies on the biosynthesis of violacein 127 in *C. violaceum* showed that all carbons, nitrogens and hydrogens were derived exclusively from L-tryptophan, while the oxygen atoms originated from molecular oxygen.¹¹⁰,¹¹¹ However, while intact incorporation of a decarboxylated tryptophan yielded the oxindole side, the other tryptophan unit suffered an intramolecular rearrangement (1,2-shift) of the indole ring to form the 5-hydroxyindole side¹¹² (Scheme 2). Additionally, the pyrrolidone nitrogen was derived from the α-amino group of the oxindole-side tryptophan. The biosynthesis of chromopyrrolic acid 92 was also analyzed in *C. violaceum*. The studies confirmed that 92 was derived from two tryptophan units.⁷⁵ The pyrrole dicarboxylic acid moiety was formed by condensation of the side chains of the two tryptophans (Scheme 2), in a similar way to that found for formation of the

![Chemical Structure](https://i.imgur.com/structure.png)

**Scheme 2** Incorporation of tryptophan into violacein and chromopyrrolic acid.

pyrrole[3,4-c] unit of REB and STA (Scheme 1). However, the origin of the pyrrole nitrogen could not be determined. Feeding radiolabeled 92 to *C. violaceum* did not result in incorporation into violacein, suggesting that the former was not an intermediate for the latter.

### 3.1.2 Production of novel indolocarbazoles by altering the fermentation conditions.

Several studies tested the ability of ICZ-producing microorganisms to produce novel analogs by accepting altered precursors. One of the first examples of this kind of study consisted of feeding 5-hydroxytryptophan to cells of *C. violaceum*, the producer of violacein. As a result, a new hydroxylated analog (oxyviolacein 136) was produced.¹¹³

![Chemical Structure](https://i.imgur.com/oxyviolacein.png)

**136**  
oxyviolacein

Addition of potassium bromide in the fermentation of *L. aerocolonigenes* resulted in production of bromorebeccamycin 137, with concomitant suppression of REB accumulation.¹¹⁴ This result strongly suggested that the REB chlorines originated from chloride ions. Addition of potassium fluoride or iodide to the medium did not lead to the production of the corresponding fluoro or iodo analogs. On the other hand, addition of 6-fluoro-, 5-fluoro-, or 4-fluorotryptophan to the fermentation of *L. aerocolonigenes* successfully produced fluoroindolocarbazoles 138–141.¹¹,¹¹⁵ Similarly, production of bromo, fluoro, or aza analogs 142–154 of bisindole pyrroles was achieved by cultivation of strain NPS012745 in media containing sodium bromide or modified tryptophan (5-fluoro-, 6-fluoro-, or 7-azatryptophan).⁵⁵,⁵⁶

In an attempt to perform precursor-directed biosynthesis with *Lentzea albida*, the following indole derivatives were utilized: indole-3-acetonitrile, indole-3-acetic acid, indole-3-acetamide, 5- and 6-fluorotryptophans, 5-hydroxytryptophan, indole-3-pyruvate, indole-3-propionic acid, tryptamine, 5- and 6-fluorotryptamines, and 5-hydroxytryptamine.¹¹⁶ No bisindole or STA derivatives were found in these experiments, suggesting that the enzymes responsible for STA biosynthesis in this strain were not capable of accepting the analogs tested as precursors. However, the tryptamine and fluorotryptamine feeding experiments produced some novel tryptamine derivatives, which were probably unrelated to STA formation.¹¹⁶,¹¹⁷ Several sugar derivatives were also evaluated for their efficacy in precursor-directed biosynthesis, but no STA derivatives with a modified glycone moiety were found.¹¹⁷

The bioconversion of more complex altered precursors was also tested. Novel D-glucosylated analogs 155–157 were produced by feeding the corresponding aglycones to cultures of *Microtetraspora* sp. A34549 and *L. aerocolonigenes* ATCC 39243.¹¹⁸

137  
bromorebeccamycin

138 (R=Me)  
139 (R=H)

140 (R1=F, R2=H)  
141 (R1=H, R2=F)

142 (R1=R2=Br)  
143 (R1=Br, R2=Cl)  
144 (R1=Cl, R2=Br)

145 (R=F)  
146 (R=H)  
147 (R=Cl)

148 (R1=F, R2=Cl)  
149 (R1=Cl, R2=F)

150 (R=Cl)  
151 (R=H)

152  

153 (R=H)  
154 (R=Cl)

155  
ED-110 (J-109384)

156  

157  

161, in addition to violacein 127. When the same inhibitor was added to cultures of *Lentzea albida*, STA biosynthesis was affected, and no bisindole derivative was produced. Additionally, several methyltransferase inhibitors, including sinefungin and (*S*)-adenosylhomocysteine, were evaluated. Only sinefungin affected

158  
deoxyviolacein

159 proviolacein (R=OH)  
160 prodeoxyviolacein (R=H)

Additionally, the use of enzyme inhibitors was evaluated for production of novel derivatives or intermediates. Addition of *N,N*-diethyldithiocarbamate (an inhibitor of copper-requiring enzymes) to *C. violaceum* resulted in accumulation of products 158–  

161 pseudodeoxyviolacein (R=H)  
162 pseudoviolacein (R=OH)

the biosynthesis of STA in *Lentzea albida*, by blocking the last step, *O*-methylation. An intermediate was efficiently accumulated in the medium, consisting of 3′-demethoxy-3′-hydroxy-STA 58.¹⁰⁶

### 3.1.3 Minor metabolites and blocked mutants.
Several mutants, unable to produce violacein, were obtained by treatment of *C. violaceum* with the mutagenic reagent *N*-methyl-*N*-nitrosoguanidine. When tryptophan was fed to their washed cells, the mutants produced various tryptophan metabolites, most of which could generally be detected in the parent strain, although in much lower amounts. Mutant no. 8 formed indolelactic acid, while mutant no. 9 accumulated chromopyrrolic acid 92 and indole-3-carboxylic acid.⁷⁵,¹²¹ Mutant no. 26 produced a number of green and red-orange pigments, including bisindole compounds (proviolacein 159, pseudoviolacein 162), and tetraindole compounds (chromoviridans 163, deoxychromoviridans 164).¹²⁰,¹²² Feeding radiolabeled 159 or 160 to the parent strain did not result in incorporation into violacein.¹²⁰

The STA-producing strain *Streptomyces longisporoflavus* R-19 was subjected to a mutagenic treatment by UV irradiation. This work led to the isolation of mutants M13 and M14, which accumulated K252c 15 and 3′-*O*-demethyl-STA 58, respectively.¹²³⁻¹²⁵ Compound 58 was converted into STA by cultures of the M13 mutant, and hence it was proposed as the last intermediate in STA biosynthesis.¹²⁶ Additionally, 58 could be converted *in vitro* into radiolabeled STA by using S-adenosyl-L-[methyl-¹⁴C]methionine and cell-free extracts of *S. longisporoflavus*. The *O*-methyltransferase was partially purified and characterized, and was found to be strongly inhibited by S-(5′-adenosyl)-L-homocysteine.¹²⁶

Based on the result that *S. longisporoflavus* produced K252a 13 as a minor metabolite, a pathway for the final steps in the biosynthesis of K252a was proposed, starting from aglycone 15⁴⁶ (Scheme 3). As K252d 16, containing an L-rhamnose moiety, was co-produced with K252a 13 in *Nonomuraea longicatena*,²³ it was suggested that L-rhamnose was the biosynthetic precursor for the sugar moiety of K252a and STA-related metabolites. A close biosynthetic relationship between STA, K252a and K252d was further supported by several facts. First, the absolute stereochemistry at positions C-4 and C-5 of L-rhamnose in K252d 16, and at positions C-3′ and C-2′ of STA 7 was identical. Second, STA (and TAN-1030A 24) had the same absolute stereochemistry as K252a 13 at the bridging atoms C-2′ and C-5′. Third, in the course of synthetic studies of ICZs, it was found that the sugar moiety of certain derivatives could undergo remarkably stereoselective reactions of either ring expansion or ring contraction reactions were carried out to produce 3′-amino-3′-dehydroxy-K252a 165 from

**163 chromoviridans (R=OH)**  
**164 deoxychromoviridans (R=H)**  

![Chemical Structures](https://i.imgur.com/XYZ.png)

**Scheme 3** A proposed pathway for the biosynthesis of the K252a sugar moiety.⁴⁶

64. The MUV-6-83 strain produced 64 as the major metabolite, and was used for feeding experiments with radiolabeled glucose and tryptophan. ${}^{109}$ Feeding with D-[U-${}^{13}$C]glucose resulted in labeling of all carbons at the carboxysugar moiety, with the exception of C-2′ and C-3′ (Scheme 6). This labeling pattern agreed with the proposed biosynthetic pathway for furanosylated ICZs, where the six-carbon carboxyfuranose is derived from a benzil → benzilic acid rearrangement of a six-carbon pyranose. The ethyl group (analogous to a corresponding methyl group in K252a 13) would be independently added from an unidentified precursor, either before or after the rearrangement.

169 indocarbazostatin C (R=H)
170 indocarbazostatin D (R=NH₂)

### 3.2 Identification of genes involved in indolocarbazole biosynthesis

#### 3.2.1 Genes involved in rebeccamycin biosynthesis.
Earlier experiments showed that the REB producer, *L. aerocolonigenes*, could convert BE-13793C 43 to its D-glucosylated derivative ED-110 155, which was a key intermediate for production of a clinical candidate. ${}^{118}$ Given that chemical production of 155 from 43 proceeded with low overall yield, it was decided to clone the gene responsible for glycosylation in order to facilitate efficient production of 155 by fermentation. ${}^{132}$ A genomic library of *L. aerocolonigenes* was generated in *Streptomyces lividans* using pIJ702 as vector. After testing more than 5000 transformants, one was found to convert 43 into its glucosylated derivative 155. The plasmid isolated from this transformant contained a 4.15 kb DNA insert, from which a 1.9 kb fragment that still expressed N-glycosylation activity was isolated. This fragment was sequenced, and a gene (*ngt*) possibly encoding a glycosyltransferase was found. The *ngt* gene was expressed in *Streptomyces mobaraensis*, which produced BE-13793C, and the resulting strain directly yielded the desired product 155. Additionally, the *ngt* gene was expressed in *S. lividans*, and the new strain was able to produce 13-(β-D-glucosyl)-6-N-methylaryciaflavin C 157 when fed with the corresponding aglycone. These results suggested that *ngt* was probably involved in REB biosynthesis, although no definite proof was reported at that time.

Given that biosynthetic genes for secondary metabolites usually occur as a cluster in actinomycetes, *ngt* served as the basis for the identification of the complete gene cluster responsible for REB biosynthesis in *L. aerocolonigenes*. The REB biosynthetic locus was independently isolated from the same strain by several laboratories. ${}^{52,133-135}$ In a first approach, ${}^{133}$ an *L. aerocolonigenes* genomic library was constructed in a shuttle cosmid vector (which was able to replicate in both *E. coli* and *Streptomyces*), and the library was screened using an internal fragment of *ngt* as

**Scheme 4** Chemical studies on ring-contraction and ring-expansion reactions.

oxime TAN-1030A 24 ${}^{127}$ or to synthesize N-protected K252a 166 from α-hydroxy ketone 167. ${}^{129,130}$ Additionally, the latter compound 167 could be synthesized from aldehyde 168 in a ring-expansion reaction. ${}^{129,130}$ Therefore, TAN-1030A 24 was proposed as a biosynthetic precursor for K252a 13, through a proposed intermediate 165 ${}^{46}$ (Scheme 3). Although compound 165 was not detected in the fermentation broth of *S. longisporoflavus*, its N-methyl derivative 56 was found. It was suggested that the free amino group of 165 could be converted to the hydroxy group of K252a. As an alternative hypothesis (Scheme 5), metabolites carrying a nitrogenated sugar (24, 56) might be viewed as shunt products in the pathway to STA. Formation of K252a 13 might then proceed through intermediates possessing a non-nitrogenated sugar.

Several mutants were derived from *Streptomyces* sp. TA-0403 by irradiation with UV light. ${}^{109,131}$ The mutants produced different levels of indocarbazostatins together with K252c 15 and K252d 16. High-producing mutant MUV-6-83 yielded new compounds indocarbazostatins C and D (169 and 170), in addition to 63 and

transfer of
non-nitrogenated
sugar

intermediates with
non-nitrogenated
sugar

15
K252c

transfer of
aminosugar

intermediates
with aminosugar

![Chemical Structure](https://i.imgur.com/structure1.png)

13
K252a

7
STA

56

TAN-1030A

Scheme 5 Hypothetical relationship between the biosyntheses of K252a and STA.

glucose

![Chemical Structure](https://i.imgur.com/structure2.png)

indocarbazostatin B

64

Scheme 6 Incorporation of glucose into indocarbazostatin B.

located at one end of the cluster (Fig. 1). A number of functions were proposed for the reb genes, participating in a hypothetical pathway for REB biosynthesis (Table 2). Some of these functions were supported by heterologous expression of subsets of reb genes in S. albus, which resulted in production of three non-chlorinated derivatives (11, 12, and 74).³³ These results showed that the heterologous expression of ICZ biosynthetic genes in an actinomycete host was a practicable solution for the production of derivatives.

Independently, another laboratory constructed an L. aerocoloni-genes genomic library using an E. coli cosmid vector, and the ngt gene was also used as a probe to isolate a cosmid clone containing the reb locus.⁵² In order to confirm the possible role of these genes in REB biosynthesis, they developed a genetic system for L. aerocolonigenes. (A genetic system consists of a procedure for introduction of foreign DNA into the cells of a microorganism, which is an essential first step towards its genetic manipulation.) They then inactivated several reb genes (by gene disruption experiments), and eight REB derivatives (12 and 171–177) were identified from the mutant strains. Remarkably, the authors identified a key intermediate in the pathway, 11,11′-dichlorochromopyrrolic acid 171.⁵² REB production was reconstituted by feeding metabolite 171 to a blocked mutant (unable to produce REB by itself), which confirmed 171 as an intermediate.

Table 2 Genes involved in the biosynthesis of rebeccamycin (reb),<sup>52,133,134</sup> staurosporine (sta),<sup>52,136,137</sup> and AT 2433 (atm)<sup>138</sup>

| Biosynthetic step                          | Genes   | reb     | sta     | atm     | Proposed function                                                                 |
|--------------------------------------------|---------|---------|---------|---------|----------------------------------------------------------------------------------|
| Tryptophan halogenation                    |         | `rebF`   | —       | —       | Flavin reductase                                                                |
|                                          |         | `rebH`   | —       | `atmH`  | Tryptophan halogenase                                                          |
| Formation of bisindole pyrrole             |         | `rebO`   | `staO`  | `atmO`  | Amino oxidase                                                                  |
|                                          |         | `rebD`   | `staD`  | `atmD`  | Chromopyrrolic acid synthase                                                   |
|                                          |         | —        | —       | `atmA`  | Amidotransferase                                                               |
| Formation of carbazole                    |         | `rebC`   | `staC`  | `atmC`  | Flavin-binding monooxygenase                                                   |
|                                          |         | `rebP`   | `staP`  | `atmP`  | Cytochrome P450 monooxygenase                                                  |
|                                          |         | —        | `staA`  | `atmS7` | Phosphosugar nucleotidyltransferase                                            |
|                                          |         | —        | `staB`  | —       | NDP-D-glucose 4,6-dehydratase                                                 |
|                                          |         | —        | —       | `atmS8` | NDP-hexose dehydrogenase                                                       |
|                                          |         | —        | —       | `atmS9` | NDP-hexose decarboxylase/4-epimerase                                          |
| Sugar formation                            |         | —        | `staJ`  | `atmS14`| NDP-hexose 2,3-dehydratase                                                    |
|                                          |         | —        | `staK`  | `atmS12`| NDP-hexose reductase                                                           |
|                                          |         | —        | `staI`  | `atmS13`| Aminotransferase                                                              |
|                                          |         | —        | `staE`  | —       | NDP-hexose 3,5-epimerase                                                       |
|                                          |         | `rebM`   | `staMA` | `atmM`  | Methyltransferases                                                             |
|                                          |         | —        | `staMB` | `atmS10`| —                                                                              |
| Sugar attachment                           |         | `rebG`   | `staG`  | `atmG`  | Glycosyltransferase                                                            |
|                                          |         | —        | —       | `atmGl` | Glycosyltransferase                                                            |
|                                          |         | —        | `staN`  | —       | Cytochrome P450 monooxygenase                                                  |
| Imide methylation                         |         | —        | —       | `atmM1` | Methyltransferase                                                              |
| Secretion or self-resistance               |         | `rebT`   | —       | `atmI`  | Major facilitator superfamily                                                  |
| Transcriptional regulation                 |         | `rebU`   | —       | `atmB`  | Antiporter                                                                    |
|                                          |         | `rebR`   | `staR`  | `atmR`  | LAL regulatory protein                                                         |
|                                          |         | `orfD12` | —       | `atmE`  | MarR regulatory protein                                                        |

![Diagram](#)

Fig. 1 Genetic loci responsible for the biosynthesis of rebeccamycin (reb genes), staurosporine (sta genes), AT2433 (atm genes), and violacein (vio genes).

This finding indicated that decarboxylation occurred after the coupling of two tryptophan-derived units and the formation of the pyrrolo[3,4-c] ring. Additionally, a new genomic library of *L. aerocolonigenes* was prepared, but this time using a shuttle cosmid vector, and a clone was selected that contained the complete gene cluster. When this cosmid clone was transformed into *S. lividans*, REB production was detected.<sup>139</sup>

Finally, another laboratory isolated the complete gene cluster in a single cosmid clone that conferred the ability to produce REB in *E. coli*, although at low levels.<sup>134</sup> An *E. coli* strain carrying this cosmid produced REB 9 (the major component),

REB aglycone 172, and a third metabolite 178 whose proposed structure lacked the pyrrolo[3,4-c] ring. The authors suggested that the first ring closure and aromatization in REB biosynthesis apparently occurred prior to forming the final pyrrolocarbazole ring. However, given that 11,11′-dichlorochromopyrrolic acid 171 has been confirmed as a true biosynthetic intermediate,<sup>52</sup> it is more likely that metabolite 178 might be produced by some unidentified enzyme from the *E. coli* host, acting on a REB intermediate. Interestingly, formation of metabolite 178 might show some resemblance to the biosynthesis of tijipazoles (compare to tijipazole D 114).

the aminotransferase involved in biosynthesis of the STA sugar moiety, but no evidence was reported.

The complete STA biosynthetic gene cluster was cloned from a different strain, *Streptomyces* sp. TP-A0274. ${ }^{52,136}$ A genomic library was constructed in a shuttle cosmid vector and was screened using a fragment of one of the reb genes (rebD) as a probe. One of the positive clones was confirmed to include all the required biosynthetic genes by heterologous expression in *S. lividans*, resulting in STA production. ${ }^{136}$ After the DNA region was sequenced (Fig. 1), putative functions were proposed for the *sta* genes, accounting for most of the structural differences between STA and REB (Table 2). Initially, a rebC homolog was not described in *Streptomyces* sp. TP-A0274, ${ }^{52,136}$ but it was later reported. ${ }^{137}$ Additionally, a rebD-like sequence was detected in a K252a producer, *Nonomuraea longicatena* NRRL 15532, by using a PCR screening for rebD-homolog sequences. ${ }^{136}$

Based upon the sequence information previously reported from *S. logisporoflavus*, ${ }^{140}$ the complete STA biosynthetic gene cluster from this strain was isolated in a single cosmid clone, which directed STA production upon introduction in *S. albus*. ${ }^{141}$ Partial sequencing of the DNA region showed that the *sta* gene cluster from *S. logisporoflavus* appeared to be virtually identical to the one described in *Streptomyces* sp. TP-A0274.

### 3.2.3 Genes involved in biosynthesis of other indolocarbazoles.
Genes encoding the biosynthesis of violacein 127 were isolated from *C. violaceum* UQM51 on a 14.5 kb fragment that was functionally expressed in *E. coli* and other Gram-negative bacteria. ${ }^{142}$ The cloned violacein locus (*vio*) was later characterized by DNA sequencing, transposon mutagenesis, and chemical analysis of pathway intermediates produced heterologously in *E. coli*. ${ }^{143}$ The *vio* gene cluster was reported to span 8 kb and to consist of four genes, *vioABCD* (Fig. 1). Independently, a different laboratory identified a virtually identical set of genes encoding violacein biosynthesis from *C. violaceum* JCM1249 (ATCC 12472), ${ }^{112}$ a strain whose genome was later completely sequenced. ${ }^{77}$ Additionally, violacein biosynthetic genes from an unidentified bacterium were isolated from an environmental DNA library. ${ }^{144}$ Later, a fifth gene (*vioE*) was found to be essential for violacein biosynthesis by heterologous expression of different combinations of *vio* genes in *E. coli* and *S. albus*. ${ }^{145}$

Recently, the gene cluster (*atm*) for the production of AT2433 complex 19–22 in *Actinomadura melliaura* has been described. ${ }^{138}$ The *atm* locus was identified by searching for genes similar to rebD and rebP in a genomic library of *A. melliaura*. The reported gene cluster spanned approx. 35 kb and consisted of 21 genes (Fig. 1). In agreement with the structural resemblance to REB 9, the gene cluster contained some genes that were likely homologs of their reb counterparts (Table 2). Additionally, there were other genes encoding proteins responsible for specific structural features, such as imide N-methylation and formation of the aminosugar moiety. The locus was confirmed by in vitro biochemical characterization of two methyltransferases (*atmM*, *atmMI*), as well as by heterologous expression and in vivo bioconversion experiments using the AtG glycosyltransferase. ${ }^{138}$

Additionally, a biosynthetic locus (*ink*) responsible for production of K252a 13 has been reported from *Nonomuraea longicatena* (nucleotide sequence with accession number DQ399653), ${ }^{146}$ although detailed information has not yet been published. Many

### 3.2.2 Genes involved in staurosporine biosynthesis.
A patent application, published in 1997, disclosed the identification of some genes needed for formation of the STA sugar moiety in *S. logisporoflavus*. ${ }^{140}$ A genomic library of *S. logisporoflavus* was constructed into plasmid pIJ486, and transformed into *S. logisporoflavus* mutant M14. This mutant was blocked in the final step of STA biosynthesis and produced 3′-O-demethyl-STA 58. ${ }^{123}$ One of the transformed colonies was found to have restored the ability to produce STA. This colony contained a plasmid carrying a DNA insert of approx. 20 kb. An internal 2.1 kb fragment from this plasmid still had the ability to complement STA production upon transformation into the M14 mutant. The 2.1 kb fragment was sequenced, and two genes (“gene 1” and “gene 2”) were identified, putatively coding for a SAM-dependent methyltransferase and a dTDP-4-keto-6-deoxyglucose 3,5-epimerase, respectively. Therefore, it was assumed that “gene 1” coded for the O-methyltransferase responsible for the last step in STA formation. The involvement of “gene 2” in biosynthesis of the sugar moiety of STA was supported by the fact that the M13 mutant, which accumulated K252c, could be complemented for STA production by the 2.1 kb fragment. A DNA region of about 10 kb, which included the 2.1 kb fragment, was sequenced. Five additional genes were identified, two of which coded for a putative SAM-dependent methyltransferase and an aminotransferase, respectively. It was suggested that these two genes would code, respectively, for the N-methyltransferase and

of the identified *ink* genes were likely homologs of corresponding *sta* genes. Apparently, only one methyltransferase gene was found, probably for methylation at the furanose. The reported sequence did not include an aminotransferase gene, which supports a K252a pathway proceeding through intermediates carrying a non-nitrogenated sugar. A number of *ink* genes had no counterpart in the *sta* locus, and some of them might be responsible for those steps specific for K252a formation.

### 3.3 Step-by-step description of indolocarbazole biosynthetic pathways

This section describes the biosynthesis of ICZ natural products, summarizing the available information regarding genes, enzymes, and intermediates involved. Unfortunately, only the biosyntheses of REB, STA, and violacein have been studied in detail. The data are mainly derived from two types of studies. The first type consists of *in vivo* studies made with gene-disrupted mutants of *L. aerocolonigenes,* or with recombinant strains of *S. albus* or *E. coli* (which expressed genes from ICZ producers). The *in vivo* studies not only contributed to a better understanding of the biosynthetic processes, but also provided genetically modified microorganisms that produced different ICZ derivatives. The second type of studies consisted of *in vitro* experiments made either with cell-free extracts, or with purified enzymes catalyzing biosynthetic reactions. Some genetic and biochemical aspects of the biosynthesis of REB and STA have been recently reviewed, with an emphasis on pathway engineering for the generation of novel derivatives.

#### 3.3.1 Formation of the indolocarbazole ring system.

##### 3.3.1.1 Tryptophan modification. Conversion of L-tryptophan to 7-chloro-L-tryptophan 179 is the first step in REB biosynthesis (Scheme 7) and serves to supply a unique monomer precursor

pool. The reaction is catalyzed by a two-component enzyme, RebF/RebH. In vitro halogenation by RebH, an FADH₂-dependent halogenase, requires molecular oxygen and flavin reductase RebF, which catalyzes the NADH-dependent reduction of FAD to provide FADH₂ for the halogenase. Chloride salts are the source of chlorine for RebH-catalyzed halogenation. Bromotryptophan could be produced by RebF/RebH in the presence of bromide ions, but neither fluoride nor iodide ions were competent for halogenation. These results were in agreement with previous reports on the production of bromorebeccamycin 137 by cultures of *L. aerocolonigenes.* Similarly, bromo analogs (137 and 180–185) were obtained by replacing chloride with bromide in the fermentation medium of *S. albus* strains expressing *reb* genes. However, in the absence of *rebH*, a full pathway for dideschloro-REB 11 could be reconstituted, indicating that the rest of the enzymes were able to use non-halogenated intermediates. Downstream enzymes could also utilize tryptophans carrying non-natural halogenations, as demonstrated by production of fluorindolocarbazoles 138–141 through addition of fluorotryptophans to fermentations of *L. aerocolonigenes.* Moreover, the heterologous expression of different tryptophan halogenases (*pyrH, thal*), in addition to *reb* genes, resulted in formation of new halogenated derivatives 186–190 in *S. albus*, presumably by incorporation of 5- and 6-chlorotryptophan.

Tryptophan hydroxylation, to yield 5-hydroxytryptophan, was proposed as an early step in violacein biosynthesis. The hypothesis was supported by the fact that radiolabeled 5-hydroxytryptophan could be incorporated into violacein 127 when fed to *C. violaceum*. However, the bulk of the added 5-hydroxytryptophan was not incorporated into violacein, but it was diverted into oxyviolacein 136, a novel product with an extra hydroxyl group that had not been detected in *C. violaceum* before. Recently, it was found that the hydroxylation may take place on bisindole intermediates, during the late steps

![Chemical Structures](attachment:chemical_structures.png)

**Scheme 7** Proposed pathway for biosynthesis of rebeccamycin.

191, with production of hydrogen peroxide ${}^{135,148}$ (Scheme 7). Imine 191 may be readily hydrolyzed in solution, existing in equilibrium with the predominant ketone form 192 (and its enol tautomer). In a survey of the 20 natural amino acids, RebO showed high oxidase activity only against L-tryptophan. ${}^{135}$ Nevertheless, l-methyl-L-tryptophan, 5-methyl-DL-tryptophan, and 5-fluoro-L-tryptophan were found to be substrates for RebO, while D-tryptophan, L-tryptophanamide and $N$-acetyltryptophan were not accepted by the enzyme. RebO showed significant preference for 7-chloro-L-tryptophan over L-tryptophan, further supporting the role of the former as the natural early pathway intermediate. ${}^{135}$ Presumably, StaO and VioA might have activities similar to that of RebO, but acting on L-tryptophan to yield indole-3-pyruvic acid imine 193 (in equilibrium with its ketone 194).

Studies made with recombinant strains of $S$. albus showed that formation of the simplest bisindole intermediate during ICZ biosynthesis required the co-expression of two genes: $r e b O$ and $r e b D .{ }^{137}$ The metabolite consisted of chromopyrrolic acid 92, which seemed to be an intermediate during production of non-chlorinated REB 11 ${}^{137}$ and STA 7. ${}^{151}$ The role of the chromopyrrolic scaffold as a central intermediate in ICZ biosynthesis was first discovered after the identification of 11,11′-dichlorochromopyrrolic acid 171, which was accumulated by a $r e b P$-disrupted mutant of $L$. aerocolonigenes. ${}^{52}$ This chlorinated metabolite was also produced by co-expression of $r e b H$, $r e b O$ and $r e b D$ in $S$. albus. ${}^{137}$

The RebD enzyme was characterized as the first member of a new subfamily of heme-containing oxidases. ${}^{148,150}$ In addition to the heme iron, RebD also contained one equivalent of non-heme iron. The enzyme acted as both a catalase and a chromopyrrolic acid synthase, apparently converting two molecules of 7-chloroindole-3-pyruvic acid imine 191 into 11,11′-dichlorochromopyrrolic acid 171. ${}^{148,150}$ The inherent instability of the putative imine intermediate might explain previous observations made from precursor incorporation experiments that strongly indicated that the $\alpha$-amino group from tryptophan did not significantly contribute to the pyrrole nitrogen of REB ${}^{107}$ or STA. ${}^{104}$ Formation of chromopyrrolic acid 92 by StaD, an enzyme homolog to RebD, was also confirmed for STA biosynthesis. ${}^{151}$

It has been recently suggested that formation of the asymmetrically halogenated aglycone in AT2433-A1 (19) and A2 (20) might require a dedicated activity, in addition to RebO and RebD homologs (AtmO, AtmD). ${}^{138}$ A gene, $a t m A$, was found at the

of violacein formation. ${}^{145}$ Therefore, it was suggested that 5-hydroxytryptophan is not a precursor in the major pathway for violacein biosynthesis, although it could be incorporated into violacein due to substrate flexibility of key enzymes.

### 3.3.1.2 Formation of bisindole pyrrole (chromopyrrolic acid)

Two genes appear to be characteristic and essential for ICZ production, at least for the biosyntheses of REB, STA and violacein. One of the genes encodes a tryptophan oxidase ($r e b O$, $s t a O$, $v i o A$), while the second one codes for a heme-containing oxidase ($r e b D$, $s t a D$, $v i o B$). In their absence, no bisindole intermediates can be formed. ${}^{52,137,147}$

The enzyme RebO, from $L$. aerocolonigenes, was characterized as an FAD-dependent L-tryptophan oxidase that converted 7-chloro-L-tryptophan 179 into 7-chloroindole-3-pyruvic acid imine

**Scheme 8** Incorporation of 5-hydroxy-L-[4,6-${}^{2}$H${}_{2}$]tryptophan into violacein and oxyviolacein. Incorporation rates are given in percentages for the hydroxyindole and the oxindole sides of each molecule.
border of the *atm* locus, which displayed similarities to glutamine-dependent amidotransferases. The authors postulated that AtmA (for which no homolog was reported in the *reb* or *sta* gene clusters) would be responsible for supplying the pyrrole nitrogen in AT2433 aglycone. Furthermore, they proposed that AtmA and AtmO might have unique substrate specificities, leading to the final formation of an asymmetrically halogenated aglycone.

As mentioned before, the biogenesis of violacein 127 showed a basic similarity to that of ICZs, with both types of pathways including a decarboxylative fusion of two tryptophan-derived units. However, in the case of violacein, one of the tryptophans suffered a peculiar 1,2-shift of the indole ring. Recently, the relationship between the biosyntheses of violacein and ICZs was re-evaluated. \({ }^{145}\) A pair of genes (*vioA*, *vioB*), responsible for the earliest steps in violacein formation, was found to be functionally equivalent to their homolog pair in the ICZ pathway (*rebO*, *rebD*), directing the formation of chromopyrrolic acid 92 (Scheme 9). However, in contrast to ICZ biosynthesis, chromopyrrolic acid appeared to be a shunt product during violacein formation. The

**vioE** gene was essential for production of the characteristic 1,2-shift of the indole to yield asymmetrical intermediates such as 160, instead of chromopyrrolic acid 92 (Scheme 9). \({ }^{145}\) After that, the violacein pathway appeared to be completed by the action of oxygenases VioC and VioD. \({ }^{143,145}\) In the absence of a functional VioC, green pigments of the chromoviridan type (163 and 164) were produced, probably as shunt products. \({ }^{122,145}\) Pigments 163 and 164 are tetraindole compounds that can be viewed as the result of linking two pro(deoxy)violaceins (159 or 160) together with an extra carbon, which is derived from serine (probably through \(N^{5}, N^{10}\)-methylene\-tetrahydrofolate). \({ }^{122}\)

### 3.3.1.3 Oxidative ring closure: formation of carbazole.
In order to complete the indolopyrrolocarbazole ring system, chromopyrrolic acid intermediates undergo a decarboxylative ring closure. During REB biosynthesis, the *rebC* and *rebP* genes are responsible for conversion of 171 into 172 (or, during production of dideschloro-REB 11, conversion of 92 into 74) \({ }^{52,137}\) (Scheme 7). For STA biosynthesis, the *staC* and *staP* genes are involved in conversion of 92 into 15 \({ }^{137}\) (Scheme 10). In

![Chemical Scheme](attachment:image.png)

**Scheme 9** Proposed pathway for violacein biosynthesis. \({ }^{145}\) The labeling pattern has been included to highlight the 1,2-shift of the indole (*) and the differences between the pattern of chromopyrrolic acid and that of the rest of the metabolites.
H₂N-COOH
     |
L-tryptophan

193 (R=NH) ↔ 194 (R=O)

StaO → COOH
         |
         | (x 2)
         v
HOOC-H-COOH
     |
chromopyrrolic acid 92

StaD → HOOC-H-COOH
         |
         | StaP
         v
H-N-COOH
     |
K252c 15

NDP-L-ristosamine

StaG

StaMB
     |
     v
H-N-COOH
     |
7 STA

StaMA
     |
     v
Me-N-O-Me
     |
58

StaMB
     |
     v
Me-N-O-Me
     |
120

StaMA
     |
     v
Me-N-O-Me
     |
122

StaN
     |
     v
H-N-COOH
     |
holyrine A 59

StaA → HO-OPO₃H₂
         |
         v
HO-OH

StaB → HO-OH
         |
         v
Me-OH

StaJ → Me-OH
         |
         v
Me-OH

StaI
     |
     v
Me-OH

StaK ← NH₂
     |
     v
Me-OH

NDP-L-ristosamine

StaE ← NH₂
     |
     v
Me-OH

Scheme 10 Proposed pathway for biosynthesis of staurosporine (modified from ref. 136 and 137).

addition to the chlorines (which are introduced by the halogenase RebH), the two aglycones (172, 15) differ in the C-7 oxidation state.

The involvement of rebP and rebC in the ring-closing reaction was first indicated by the fact that rebP- and rebC-disrupted mutants of L. aerocolonigenes accumulated, respectively, 11,11′-dichlorochromopyrrolic acid 171 and a mixture of REB derivatives differing at C-7.² Among the compounds accumulated by the rebC mutant, two of them were found to be 7-deoxo-7-hydroxy-REB 176 and its 4′-O-demethyl analog 177. NMR results showed that each peak consisted of a mixture of 12-N- and 13-N-glycosides not separable on HPLC. A rebG and rebC double disruptant produced three peaks, identified as REB aglycone 172, 7-deoxo-7-hydroxy-REB aglycone 173 and 7-deoxo-REB aglycone 174, respectively. The 7-deoxo or 7-hydroxy derivatives did not seem to be REB intermediates, as they were not converted into REB in bioconversion experiments.⁵²

In agreement with those observations, efficient production of arcyriaflavin A 74 in S. albus required the co-expression of four genes: rebO, rebD, rebC and rebP.¹³⁷ If the rebH gene was also included, aglycone 172 was produced. Although it was possible to obtain 74 in the absence of rebC (by co-expression of rebO, rebD and rebP), the result consisted of a mixture of three analogs differing at the C-7 position: 74, 7-hydroxy-K252c 195, and K252c 15.¹³⁷

The involvement of rebC, rebP, and their homologs staC and staP in the ring-closing reaction was tested using recombinant S. albus strains.¹³⁷ The rebO and rebD genes were co-expressed together with different combinations of rebC, rebP, staC and staP. The results showed that rebP and staP were functionally

equivalent, and that any of them was sufficient for processing chromopyrrolic acid 92 into a mixture of the three analogs (74, 195 and 15) differing at the C-7 position. When either rebC or staC was added to these gene combinations, a single product was obtained. Remarkably, the single product consisted of K252c 15 when using staC, or arcyriaflavin A 74 when using rebC. Therefore, it appears that a cytochrome P450 enzyme (RebP or StaP) is responsible for the decarboxylative oxidations needed to convert a chromopyrrolic intermediate into an indolopyrrolocarbazole. However, an additional monooxygenase (RebC or StaC) is needed for determination of the C-7 oxidation state of the ICZ. Interestingly, RebC and StaC determine different oxidation states in the final product. This finding allowed the generation of a series of 7-deoxo derivatives (15, 174 and 196–202) in S. albus strains expressing reb genes, by replacing rebC with staC. Glycosylated derivatives 197–202 were probably a mixture of 12- and 13-N-glycosides not separable on HPLC, although this was not confirmed.

![Chemical Structure 195](https://i.imgur.com/structure_195.png)
195

![Chemical Structure 196](https://i.imgur.com/structure_196.png)
196

![Chemical Structure 197(R=H)](https://i.imgur.com/structure_197_R_H.png)
197 (R=H)

![Chemical Structure 198(R=Me)](https://i.imgur.com/structure_198_R_Me.png)
198 (R=Me)

![Chemical Structure 199(R=H)](https://i.imgur.com/structure_199_R_H.png)
199 (R=H)

![Chemical Structure 200(R=Me)](https://i.imgur.com/structure_200_R_Me.png)
200 (R=Me)

![Chemical Structure 201(R=H)](https://i.imgur.com/structure_201_R_H.png)
201 (R=H)

![Chemical Structure 202(R=Me)](https://i.imgur.com/structure_202_R_Me.png)
202 (R=Me)

The atmM1 gene from A. melliaura has been recently proposed as encoding the enzyme responsible for N-methylation at the pyrrole during AT2433 biosynthesis. The enzyme was able to in vitro methylate some glycosylated and non-glycosylated indolopyrrolocarbazoles.

### 3.3.2 Formation of the sugar moiety.
#### 3.3.2.1 Sugar moieties in rebeccamycin and AT2433.
REB and AT2433 compounds contain a 4′-O-methyl-β-D-glucopyranosyl moiety. The rebG/atmG and rebM/atmM genes, respectively coding for a glycosyltransferase and a methyltransferase, appear to be responsible for formation of the carbohydrate (Table 1). RebG glycosylation takes place on a planar, aromatic ring system, but not on chromopyrrolic intermediates. A rebG mutant of L. aerocolinigenes accumulated REB aglycone 172, which is the likely substrate for glycosylation (Scheme 7). On the other hand, blocking the ring closure step by disrupting rebP in L. aerocolonigenes resulted in accumulation of 11,11′-dichloro-CPA 171, a non-glycosylated derivative. Glycosylation on chromopyrrolic acid 92 was not detected either by co-expression of rebO, rebD and rebG, or by feeding 92 to a rebG-expressing S. albus. Synthetic compounds structurally similar to chromopyrrolic acid also failed as RebG substrates when fed to rebG-expressing strains.

In vivo experiments made with L. aerocolonigenes, recombinant S. albus strains, or rebG-expressing S. lividans and E. coli indicated that the RebG glycosyltransferase was able to add a D-glucose moiety to a variety of indolo[2,3-a]pyrrolo[3,4-c]carbazoles. Additionally, indolo[2,3-a]carbazoles (lacking the pyrrole ring) were bioconverted by rebG-expressing strains to yield D-glucosyl derivatives 203–205. Structural elucidation of RebG conversion products confirmed that glycosylation of asymmetrical aglycones resulted in a mixture of two regioisomers (such as 204 and 205), not separable by HPLC, after indiscriminate N-glycosylation of either indole nitrogen. The AtmG glycosyltransferase appeared to behave in a similar manner to RebG.

So far, no sugar moieties other than D-glucose have been reported for glycosylations catalyzed by RebG. Presumably, the sugar substrate directly used by RebG might consist of a nucleotide-activated form, i.e., NDP-D-glucose. The biosynthesis of such an intermediate probably requires a glucose-1-phosphate nucleotidyltransferase, but a gene putatively encoding this activity was not found in the reb cluster. Such a gene is likely encoded in a different region of the L. aerocolonigenes genome. Production of RebG-glycosylated ICZs in S. albus, S. lividans, and E. coli indicated that these hosts possessed nucleotidyltransferases able to supply the required nucleotide-activated D-glucose.

Sugar methylation is the last step in REB biosynthesis. A rebM-disrupted mutant of L. aerocolonigenes accumulated 4′-O-demethyl-REB 175, which was converted to REB by a rebH disruptant. The RebM enzyme was characterized as the expected sugar 4′-O-methyltransferase. Several glycosylated ICZs, including both α- and β-glycosidic analogs, could be modified in vitro by RebM-catalyzed methylation, to yield derivatives 11, 198 and 206–212. Glycosides containing L-deoxysugars were not methylated. Additionally, RebM was able to use an N-mustard analog of S-adenosyl-L-methionine for in vitro modification of several substrates, resulting in formation of novel derivatives 213–216. Recently, the AtmM methyltransferase from A. melliaura has been partially characterized, showing an activity similar to that displayed by RebM.

In comparison to REB, the sugar moiety of AT2433 compounds 19–22 is remarkable as containing an aminodideoxypentose, in addition to the methylglucose. Accordingly, the atm locus contained 8 genes coding for enzymes putatively involved in formation and attachment of the aminopentose, which were not found in the reb gene cluster (Table 1). A proposed biosynthetic pathway for the aminodideoxypentose is shown in Scheme 11.

![Chemical Structure 1](image1.png)

Scheme 11 Proposed pathway for biosynthesis of the aminopentose in AT2433. The N-methylation step might alternatively occur after the sugar is transferred to the aglycone by glycosyltransferase AtmG1.

![Chemical Structures 203-216](image2.png)

3.3.2.2 The staurosporine sugar moiety. The carbohydrate present in the STA molecule is an aminodeoxysugar, which is linked to the aglycone through a pair of C–N bonds. Up to ten genes appear to be involved in formation of the sugar moiety (Table 2). Reconstitution of the STA pathway in *S.* albus, by heterologous expression of *sta* and *reb* genes, shed light on the nature of this process. ${}^{141}$ STA production in *S.* albus was achieved by co-expression of genes for K252c formation (*rebO*, *rebD*, *rebP*, *staC*) together with genes for sugar formation (*staG*, *staN*, *staMA*, *staJ*, *staK*, *staI*, *staE*, *staMB*). It was found that the *staA* and *staB* genes were not needed for STA formation in *S.* albus, suggesting that this host was providing the first two enzymatic activities required for deoxysugar biosynthesis. Deletion of *staG* abolished glycosylation, and only K252c 15 was produced. On the other hand, removal of *staN* (while keeping *staG*) resulted in production of hollyrine A 59, which had been previously isolated from a STA-producing marine actinomycete. ${}^{47}$ The carbohydrate in hollyrine A 59 was attached to the aglycone through a single glycosidic bond, and it lacked the two methylations found in STA 7. Hollyrine A could be converted into STA when fed to a strain that expressed *staN*, *staMA* and *staMB*. ${}^{141}$ In these experiments, independent removal of *staMA*, *staMB*, or both, caused hollyrine A to be converted into 4′-N-demethyl-STA 120, 3′-O-demethyl-STA 58, or 3′-O-demethyl-4′-N-demethyl-STA 122, respectively. Additionally, a *staN*-deleted mutant of *Streptomyces* sp. TP-A0274 accumulated hollyrine A, which could be converted into STA when fed to a *staD* mutant of the same strain. ${}^{154}$ Therefore, the first sugar-aglycone linkage seems to be made by StaG, while the second linkage would be catalyzed by StaN acting on hollyrine A (Scheme 10). Each one of the methylation steps can occur in the absence of the other, but only after the second linkage is formed. Presumably, most of the reactions needed for the biosynthesis of L-ristosamine might take place on nucleotide-activated sugar intermediates, before StaG-catalyzed attachment of L-ristosamine (or a previous intermediate) to the aglycone (Scheme 10).

In contrast to RebG, the StaG glycosyltransferase seems to possess a noteworthy regiospecificity towards its aglycone substrate: StaG appears to discriminate between the two indole nitrogens, and only one of them is chosen for the first (glycosidic) attachment. A natural product related to STA with the corresponding linkage made to the other indole nitrogen, N-12, has yet to be reported. Accordingly, StaG-glycosylated ICZs produced in recombinant strains appeared to exist as single regioisomers (glycosidic linkage at N-13), ${}^{141}$ and not as mixtures of two regioisomers (12-N- and 13-N-glycosides), as occurred with RebG-glycosylated compounds. ${}^{52,152}$

On the other hand, StaG accepts a variety of sugar derivatives, as has been shown through the production of novel glycosylated ICZs in recombinant *S.* albus strains. ${}^{141}$ This was done by replacing the *sta* genes involved in formation of the STA sugar with any one of various sets of genes, each set directing formation of a different sugar. As a result, a number of derivatives were produced, with a sugar moiety consisting of L-rhamnose (16 and 217), L-olivose (218 and 219), L-digitoxose (26 and 220), or D-olivose (221). With the exception of D-olivose, which yielded only the single-linkage derivative, the sugar could be attached through either one or two linkages, indicating that StaN also showed some substrate flexibility. The StaG/StaN “promiscuity” might have contributed to the abundance of minor metabolites with variations at the sugar moiety that were found in STA-producing organisms (Sections 2 and 3.1.3).

![Chemical Structures](chemical_structures.png)

3.3.3 Regulation and self-resistance. A gene (*rebR/staR/atmR/inkR*) coding for a putative LAL transcriptional regulatory protein has been found in the respective loci for the biosynthesis of REB, ${}^{52,133,134}$ STA, ${}^{136}$ AT2433, ${}^{138}$ and K252a. ${}^{146}$ Experiments made with recombinant *S.* albus strains suggested that *rebR* was needed for expression of at least some of the *reb* genes. ${}^{133}$ This was further supported by the fact that a *rebR*-truncated mutant of *L.* aerocolonigenes did not produce REB or related compounds. ${}^{52}$ Additionally, the *reb* and *atm* loci also included a gene (*orfD12/atmE*) putatively encoding a MarR regulatory protein (Table 1).

The *reb* gene cluster contained two genes (*rebT*, *rebU*) coding for putative transmembrane transporter proteins, which might be involved in self-resistance, *i.e.*, protection of the microorganism against its own toxic product. ${}^{52,133}$ Expression of *rebT* conferred resistance to exogenously added REB in *S.* albus (which was otherwise sensitive to REB). ${}^{133}$ Further studies made with recombinant *S.* albus strains showed that production of either REB 9 or 4′-O-demethyl-REB 175 (but not earlier intermediates or non-chlorinated analogs) was only possible if *rebT* was expressed. ${}^{137}$ However, disruption of *rebT* in *L.* aerocolonigenes did not decrease the production level of REB, nor did it affect growth of the microorganism. ${}^{52}$ Therefore, despite the proven role of *rebT* as a protecting system in the *S.* albus host, the gene was not essential for REB production in *L.* aerocolonigenes under the conditions tested. RebT might protect *L.* aerocolonigenes cells from an excess of REB accumulation in certain circumstances. Additionally, other resistance mechanisms may exist in *L.* aerocolonigenes that are not found in *S.* albus. For instance, the possible role of *rebU* in any aspect related to REB biosynthesis remains unknown. The *atm* locus from *A.* melliaura contained two genes, *atmI* and *atmB*, ${}^{138}$ which were putative homologs of *rebT* and *rebU*, respectively (Table 1).

While REB inhibits the growth of several bacteria, ${}^{18,19}$ STA is generally found to lack significant antibacterial activity. ${}^{14,155}$ This might explain the absence of candidate genes coding for specific self-resistance mechanisms in the *sta* locus. STA did not inhibit the growth of *S. albus*; therefore, STA production was achieved in this strain by co-expression of a defined set of *sta* and *reb* genes, in the absence of *rebT* or other dedicated resistance mechanisms. ${}^{141}$ However, STA appears to affect cell differentiation processes in streptomycetes (Section 5.3).

### 3.4 Implications for the biosyntheses of other natural products

In addition to the ICZ family, there exists a variety of bisindole natural products whose biosynthesis probably includes condensation of two tryptophan-derived units. Illustrative examples are nortopsentin A 222 (compare to 159), rhopaladin A 223, fellutanine A 224, dragmacidin A 225, and fascaplysin 226 (compare to 115 or 178). ${}^{12,156}$ Future research will tell if the biosynthetic pathways for these metabolites show some similarities to those of REB, STA, or violacein. Additionally, a noteworthy resemblance can be found between arcyiarubin 6 and didemnimid A 227, and also between the biogenetically related arcyiaflavin 74 and granulatimides 228 and 229. ${}^{156}$ It is tempting to speculate that some similarity might exist between the biosynthesis of 74, from two tryptophan units, and the formation of 227–229, which might be derived from condensation of one tryptophan and one histidine unit.

---

### 4.1 Biological activities of indolocarbazoles

Anticancer action is by far the most widely searched and reported biological activity, for both natural and synthetic ICZs. ${}^{8,10,11,157}$ Cytotoxic or antitumor properties have been widely reported for STA ${}^{16,86,88}$ and its natural derivatives, such as TAN-999 23, ${}^{32}$ 7-oxo-STA 27, ${}^{15}$ 4′-*N*-methyl-5′-hydroxy-STA 61 and 5′-hydroxy-STA 62, ${}^{51}$ ZHD-0501 65, ${}^{54}$ 11-hydroxy-STA 117 and 3,11-dihydroxy-STA 118, ${}^{68}$ 4′-*N*-demethyl-11-hydroxy-STA 119, ${}^{70}$ 3-hydroxy-STA 123 and others. ${}^{100}$ Additional natural products with antitumor activities include REB 9 and its deschloro, dideschloro and demethyldideschloro analogs (10–12), ${}^{11,18}$ AT2433-A1 19 and AT2433-B1 21, ${}^{30,158}$ BE-13793C 43, ${}^{39}$ K252c 15, ${}^{69}$ arcyiaflavins A–C 74–76, ${}^{62,63,69}$ and violacein 127 (see ref. 159 and references therein).

Different aspects concerning the antitumor activity of synthetic ICZs have been reviewed for a large series of REB derivatives ${}^{6-8,11}$ and for several STA analogs. ${}^{157}$ Compounds in the indolo[2,3-*a*]carbazole series are generally more active than similar compounds in the bisindolylmaleimide series. ${}^{6-8,63}$ However, bisindolylmaleimide derivatives with potent antitumor and antiangiogenic properties have been synthesized. ${}^{160-163}$ It has generally been found that ICZ glycosides are more potent antitumor agents than the corresponding aglycones. ${}^{6-8,164}$ Depending on the structural modifications, the analogs may or may not exhibit selectivity toward different tumor cell lines. ${}^{81,65,166}$ This cell line selectivity might be related to differences in cellular uptake of the compound, concentration of the target in the cell, and/or response due to downstream processes (e.g. resistance to DNA damage, variations in signal transduction machinery, etc.). ${}^{167,168}$ However, the anticancer properties of ICZs are not limited to their direct action on tumor cells. Some derivatives, used at non-cytotoxic concentrations, were found to potentiate the anticancer effect of other cytotoxic agents (such as DNA-damaging agents) or that of radiotherapy. ${}^{157,169-171}$

The following natural products have shown inhibitory activity against bacteria: REB 9 and 11-deschloro-REB 10, ${}^{18,19}$ AT2433 compounds 19–22, ${}^{30}$ arcyiarubins and arcyiaflavins (6 and 72–77), ${}^{2,57,172}$ violacein 127, ${}^{76,173}$ and bisindole pyrroles 66–71. ${}^{55}$ In general, the antibacterial activity was restricted to Gram-positive

---

### 4 Biological activities and modes of action

In the first subsection, we will review the biological activities observed for ICZs at the level of the cell or organism. Some of these biological activities could be correlated to specific mechanisms or modes of action at the molecular level, which will be reviewed in the second subsection.
bacteria; the tested Gram-negative strains were usually resistant. The exceptions, among the natural ICZs, were AT 2433-B121<sup>30</sup> and some bisindole pyrroles (66 and 69–71), <sup>55</sup> which showed activity against Gram-negative bacteria. Remarkably, STA 7<sup>14</sup> and K252a 13<sup>1</sup> lacked significant antibacterial activity, but both compounds interfered with cell differentiation processes in streptomycetes<sup>155</sup> (Section 5.3).

Antibacterial properties have been also reported for synthetic analogs, <sup>172,174–176</sup> some of which showed activity against Gram-negative bacteria.<sup>177</sup> Some synthetic derivatives also affected cell differentiation in streptomycetes.<sup>172,174,176</sup>

Additionally, the following ICZs have shown antifungal properties: STA 7, <sup>14</sup>RK-1409B 28 and RK-286C 25, <sup>42</sup> arcyiarubins and arcyriaflavins (6 and 72–77), <sup>2,57,172</sup> and tjipanazoles A1 102 and A2 106.<sup>65</sup> The inhibitory activities of violacein 127 against the tropical pathogenic protozoa *Trypanosoma cruzi* (the causative agent of Chagas' disease) and *Leishmania* sp. (which causes leishmaniasis) are also noteworthy.<sup>76,178</sup>

Antiviral properties have been reported for cyanobacterial ICZs 99–101 (active against HSV-2), <sup>64</sup> and lycogarubin C 68 (active against HSV-1).<sup>59</sup> Additionally, synthetic or semisynthetic analogs have shown activity against human immunodeficiency virus 1 (HIV-1), <sup>179–181</sup> human cytomegalovirus (HCMV), <sup>182–184</sup> and Epstein–Barr virus (EBV).<sup>185</sup>

ICZs, especially STA derivatives, have displayed many other biological activities (reviewed for STA in ref. 88), including hypotensive properties, inhibition of platelet aggregation, inhibition of smooth muscle contraction, activation of macrophages, <sup>32</sup> blocking of the proliferative response of T lymphoblasts to mitogens, <sup>186</sup> *in vitro* immunosuppression, <sup>43</sup> inhibition of the osteoclast proton pump, <sup>124</sup> insecticidal activity, <sup>187</sup> reversal of multidrug resistance, <sup>188,189</sup> and neuroprotection (promotion of neuronal survival). <sup>190–192</sup> Aza analogs of indolo[3,2-*a*]carbazole 4 were shown to be powerful benzodiazepine receptor ligands.<sup>193</sup> Additionally, derivatives of indolo[3,2-*b*]carbazole 5 displayed affinity to the 2,3,7,8-tetrachlorodibenzo-*p*-dioxin (TCDD) receptor (also referred to as the aryl hydrocarbon receptor protein).<sup>194</sup>

### 4.2 Modes of action in mammalian cells

At least three independent modes of action exist for ICZs: inhibition of protein kinases, inhibition of eukaryotic DNA topoisomerase I (Top1), and intercalative binding to DNA. Among the natural products, REB is a representative of a Top1 inhibitor, while STA and K252a are protein kinase inhibitors. Although the biological activity of a particular analog was often rationalized in terms of a single mechanism of action, it is now clear that at least some compounds appear to act through more than one mechanism. For instance, the antitumor activities of different REB derivatives were often not correlated to their anti-Top1 or DNA-binding activities.<sup>8,165,166,177</sup> Similarly, some K252a derivatives were found to be active against Top1, in addition to being non-selective protein kinase inhibitors.<sup>71,95</sup> Therefore, slight modifications to the ICZ structure may switch the preferred mode of action into an alternative one, substantially affecting the observed biological activity. Recently, results of antiproliferative activities against a panel of tumor cell lines have been used to identify potential biological targets of ICZs, by correlation with results obtained with compounds having a known mechanism.<sup>166</sup>

#### 4.2.1 Inhibition of protein kinases.

Protein kinases constitute a large family of enzymes catalyzing phosphorylation of amino acid residues in proteins. As a result, conformational changes generally occur in the target protein, affecting its function. Protein kinases are responsible for the control of a wide variety of signal transduction processes within the cell.<sup>196</sup> The number of different protein kinases present in the human genome might be larger than 500.<sup>197</sup> Deregulated protein kinase activity has been implicated in the pathophysiology of important human diseases, including cancer, disorders of the immune system, and neurodegenerative disorders. Since the discovery that STA was a nanomolar inhibitor of PKC in 1986, <sup>86</sup> the development of selective protein kinase inhibitors with potential therapeutic applications has generated great interest in the drug discovery community.<sup>198</sup> Kinase inhibitors are important not just for the treatment of disease, but also as reagents to help us understand more about the physiological roles of kinases.

STA 7 was found to inhibit other protein kinases in addition to PKC and, since then, the compound has been considered to be a highly potent but non-specific inhibitor of protein kinases.<sup>88</sup> Initially, several synthetic analogs such as GF109203X 230<sup>199</sup> were proposed to be selective inhibitors of PKC, and they were used in cell-based assays by many laboratories to propose a myriad of roles for this enzyme. However, this and other ICZs were later shown to lack specificity, as they inhibited other protein kinases<sup>200,201</sup> and, even, other unexpected types of proteins such as 5-hydroxytryptamine<sub>3</sub> receptor, <sup>202</sup> voltage-dependent ion channels, <sup>203,204</sup> P-glycoprotein, <sup>189</sup> adenosine kinase and quinone reductase type 2.<sup>205</sup> The ability of ICZ drugs to target different collections of protein kinases (and other cellular macromolecules), acting on multiple pathways, explains why very similar compounds may show apparently opposite activities: for example, one compound may display antitumor properties (by inducing death of cancer cells) while a similar compound may show neuroprotective activity (by inhibiting death of neurons).

![Chemical Structure](chemical_structure.png)

**Figure:** Chemical structure of GF109203X (Go 6850, bisindolylmaleimide I).

Protein kinase inhibition by STA was found to be competitive with respect to ATP, despite some initial debate due to conflicting kinetic data.<sup>86,88</sup> The interaction of STA with the ATP-binding site of kinases was confirmed by the determination of the crystal structures of STA bound to cyclin-dependent kinase 2 (CDK2) <sup>206</sup> and cAMP-dependent protein kinase (PKA).<sup>207,208</sup> Later, additional crystal structures have been determined for different kinases in interaction with STA.<sup>209–217</sup> Moreover, the structures of several kinases have been determined in interaction with other analogs such as UCN-01 17, <sup>212,213,218</sup> K252a 13, <sup>219</sup> 1,2,3,4-tetrahydro-STA (AFN 941) 231, <sup>220</sup> K252a-analog SB-218078 232, <sup>212</sup> aza-1,7-annulated derivative 233, <sup>221</sup> pyrrolocarbazole BDB 402 234, <sup>214</sup> and several bisindolylmaleimides (230 and 235–238).<sup>222–224</sup>
The crystal structures revealed that STA (or any related ICZ inhibitor) was located at the ATP binding site, in the cleft between the two lobes of the protein kinase. The indolo[2,3- $a$]carbazole moiety occupied the hydrophobic adenine-binding pocket, with the lactam group mimicking the hydrogen bonding pattern of the adenine base by usually forming two hydrogen bonds to the backbone of the hinge between the N-terminal and C-terminal domains of the kinase. The sugar moiety, in the boat conformation and perpendicular to the plane of the ICZ heterocycle, formed hydrophobic contacts and hydrogen bonds within the ribose-binding site. STA matched too well the shape of the ATP binding site, which made it a highly potent, but relatively unspecific inhibitor for protein kinases. Thus, an essential pharmacophore for kinase inhibition would consist of a bidentate hydrogen bond donating system (to mimic the cyclic amide of STA) flanked by various hydrophobic groups. Nevertheless, the binding mode of bisindolylmaleimides may be more complex than compounds possessing a planar ring system, due to their increased conformational flexibility. ${ }^{22}$

Although the ATP binding site is generally similar among protein kinases, it has been possible to obtain selectivity by exploiting differences in the mode of ligand interaction in the ATP-binding pocket. ${ }^{225,226}$ Several laboratories have synthesized series of ICZ analogs in a search for higher selectivity against particular kinases, including PKC isozymes, ${ }^{227,228}$ cyclin-dependent kinases, ${ }^{221,229-231}$ GSK-3 $\beta,{ }^{232}$ VEGF-R 2 tyrosine kinase, ${ }^{233}$ mixed lineage kinases (MLKs), ${ }^{234}$ and cytomegalovirus pUL97 protein kinase. ${ }^{183}$ A wide variety of structures are found among the kinase inhibitors of the ICZ family, from bisindolylmaleimides to glycosylated or non-glycosylated indolopyrrolocarbazoles (and their derived scaffolds). Specific structure–activity relationships for inhibitors seem to be reliable only for a particular protein kinase and a given ICZ type. Nevertheless, in head-to-head comparisons, indolopyrrolocarbazoles seemed to be more potent inhibitors than their bisindolylmaleimide counterparts. ${ }^{29}$ Attachment of a sugar moiety to the indolopyrrolocarbazole core appeared to enhance

kinase inhibitory activity. ${ }^{24,231}$ Additionally, one carbonyl group (amide) in the pyrrole ring seemed to be better for glycosidic inhibitors, ${ }^{7,8,41,235}$ while two carbonyl groups (imide) might be better for non-glycosylated analogs. ${ }^{229,236}$

### 4.2.2 Inhibition of topoisomerases and interaction with DNA.

DNA topoisomerases are ubiquitous enzymes found in all living organisms from archaeabacteria to humans. Topoisomerases participate in many cellular processes associated with separation of DNA strands such as replication, transcription, recombination and repair. ${ }^{237}$ They are classified into two classes: type I topoisomerases make a transient nick on a single strand of duplex DNA, passing another strand through the nick, while type II topoisomerases make a transient nick on both strands of the DNA, passing another double-stranded DNA segment through the gap. The common catalytic intermediates of DNA topoisomerases are enzyme-linked DNA breaks, or “cleavable complexes”. Upon completion of the strand passage reaction, topoisomerases religate the DNA without base loss or change in the DNA sequence. DNA topoisomerases are the targets for different classes of drugs with antitumor or antibacterial applications. ${ }^{238-242}$

Upon the discovery of REB, it was reported that its antitumor action was associated with extensive fragmentation in the DNA of cancer cells, but the underlying mechanism was unclear. ${ }^{18}$ Later, it was shown that REB was able to induce *in vitro* DNA cleavage mediated by DNA topoisomerase I. ${ }^{195}$ Actually, REB 9 was a weak inducer of DNA cleavage, while two semisynthetic derivatives of K252a (239 and 240) had more potent activity in the same assay. Compounds 239 and 240 acted on the topoisomerase by stabilizing the cleavable complexes. Furthermore, 240 behaved as a DNA intercalator in the absence of the enzyme. ${ }^{195}$ BE-13793C 43 showed inhibitory activity against topoisomerases I and II, as measured in DNA relaxation assays. ${ }^{39}$ After that, many antitumor ICZs have been synthesized to target DNA-related processes, through two primary mechanisms: DNA intercalation and topoisomerase

I poisoning. ${ }^{6-8,240,243-245}$ DNA intercalation can potentially affect, secondarily, many DNA-related processes including the action of topoisomerases, DNA replication, *etc.* A third mechanism might be the direct inhibition of DNA topoisomerase II: STA 7 inhibited the catalytic activity of this enzyme by blocking the transfer of phosphodiester bonds from DNA to the active tyrosine site. ${ }^{246}$ Additionally, the 7-oxo analogs of STA and K252a (27 and 241) appeared to be potent inhibitors of the catalytic activity of topoisomerase II without inducing any DNA cleavage or strong binding to DNA. ${ }^{11}$

![Chemical Structure](https://i.imgur.com/structure1.png)

So far, no inhibitors affecting bacterial type I topoisomerases have been reported. ${ }^{239}$ Inhibitors of eukaryotic topoisomerase I (Top1) are usually grouped into two classes: poisons (*e.g.* camptothecins, ICZs) and suppressors (*e.g.* doxorubicins). ${ }^{238,240,247}$ Top1 poisons trap cleavable complexes to form DNA–Top1–drug ternary complexes that prevent the religation step. Top1 suppressors inhibit the catalytic activity of the enzyme by preventing its binding to DNA; these drugs bind either to DNA or to the enzyme. Top1 poisons such as ICZs, therefore, convert the Top1 molecule into a DNA-damaging agent, resulting in cellular cytotoxicity and apoptosis. Each drug molecule has the potential of interacting with one Top1 molecule to cause a single DNA lesion. Therefore, sensitivity to Top1-targeting drugs should be dependent on high Top1 levels. The drugs appear to selectively target malignant cells over normal cells because enzyme levels are probably very low in normal non-proliferating cells. ${ }^{239}$ A common resistance mechanism to Top1 poisons in cancer cells is down-regulation of Top1. ${ }^{167,248}$ A second factor affecting the sensitivities of tumor cells to Top1-targeting ICZs is the cellular accumulation of the compounds. ${ }^{248,249}$ Cellular uptake of ICZ glycosides most likely occurs by passive diffusion through the plasma membrane; the glycosyl residue of the drug appears to play an essential role in the translocation process. ${ }^{250}$

Top1 inhibition can be demonstrated either by inhibition of Top1-mediated DNA relaxation or by induction of Top1-mediated DNA cleavage. ${ }^{251}$ Additionally, a bacterial screen based on an *E. coli* strain expressing the human Top1 gene ${ }^{252}$ has been successfully used to detect ICZ compounds with camptothecin-like activity. ${ }^{11}$ As a way to distinguish the cytotoxicity contributed by action on Top1 from cytotoxicity due to action on other targets, a pair of murine leukemia cell lines were used: a parental line that was sensitive to camptothecin, and a derived cell line showing resistance to camptothecin due to a diminished content of active Top1. ${ }^{11,181,245,250,253}$ In many cases, no direct correlation could be observed between Top1 inhibitory activities and cytotoxicities of ICZ derivatives, suggesting that other cellular targets were involved. ${ }^{180,181,254,255}$

Although many derivatives showed some degree of DNA-binding activity in addition to Top1 poisoning, no direct correlation existed between the strength of DNA interaction (in the absence of Top1) and the capacity of these derivatives to inhibit Top1. ${ }^{177,195,256}$ While the DNA-binding sequence selectivity of some of these drugs coincided with their Top1-mediated cleavage selectivity, ${ }^{257,258}$ the opposite was true for other analogs. ${ }^{256}$ Nevertheless, the two modes of action should not be fully independent because, during Top1 poisoning, the drugs actually interact with DNA (in an intercalative way) as well as with the Top1 enzyme (see below).

In contrast to the wide variety of different structural types found among the kinase inhibitors, ICZs with significant DNA binding or Top1 poisoning activity belong, almost exclusively, to the group of REB-like indolopyrrolocarbazole glycosides. Their structure–activity relationships have been thoroughly studied and reviewed. ${ }^{6-8,11,240,243,257}$ Three functional domains were proposed for REB-type structures: the Top1-interacting domain (the imide function on the pyrrole), the DNA-intercalative binding domain (the ICZ ring system), and the DNA-groove binding domain (the sugar moiety). ${ }^{259}$ The insertion of the planar chromophore between two consecutive base pairs would place the sugar residue into a groove of the double helix. ${ }^{257,260}$

The sugar residue plays an essential role for both DNA interaction and Top1 inhibition. ${ }^{164,175,257,258,261}$ Analogs for which the sugar residue is linked to the two indole nitrogens (as in STA) generally showed very weak interaction with DNA and produced minimal effects on Top1, with a few exceptions. ${ }^{180,195,255}$ With just a single $N$-glycosidic bond, REB-type glycosides are balanced between two conformations (Scheme 12): a closed conformation containing a

![Chemical Structure](https://i.imgur.com/structure2.png)

Scheme 12 Open and closed conformations in the rebeccamycin class of analogs.
hydrogen bond between the indole NH group and the pyranose oxygen, and an open conformation in which the indole NH is hydrogen-bonded to the solvent.²⁶²⁻²⁶⁴ The open conformation never has a commanding advantage, even in DMSO. Replacing the indole NH proton with an alkyl group reduced DNA interaction and abolished anti-Top1 activity, and this was accompanied by decreased cytotoxicity.²⁵⁴,²⁶¹,²⁶⁴ The presence of the methyl group induced a loss of the closed conformation, with reversion of the relative orientation of aglycone and sugar.²⁶⁴ In deschloro or dideschloro β-pyranosides with hydroxymethyl groups at the 5′-position of the pyranose, the closed conformation was maintained by a bifurcated hydrogen bond, where both the pyranose oxygen and the 6′-OH group were bonded to the indole nitrogen²⁶² (Scheme 12).

Despite their apparent chemical differences, camptothecin and REB derivatives were found to interfere similarly with the activity of wild-type and mutant Top1 enzymes, and a molecular model showing the common steric and electronic features of both classes of compounds was proposed.²⁶⁵ These aspects were recently elucidated by determining the X-ray crystal structures of the Top1–DNA complex bound with three classes of Top1 poisons: camptothecin, 3,9-dihydroxylated indolocarbazole SA315F 242, and an indenoisoquinoline.²⁶⁶,²⁶⁷ All three classes of inhibitors shared the same intercalative binding mode and mimicked a DNA base pair at the site of Top1-mediated cleavage. The three classes of molecules interacted with both the DNA and the enzyme but exploited different contacts to stabilize intercalation. The maleimide ring of SA315F was located on the minor groove side of the DNA, while the pyranosyl substituent was on the major groove. The glycosylated indole ring stacked with bases on the intact strand side of the duplex DNA, while the nonglycosylated indole ring stacked with bases on the cleaved strand side of DNA. Amino acid residues of the enzyme appeared to interact with a hydroxyl of the sugar moiety, with one of the carbonyls at the maleimide ring, and with both hydroxyl substituents of the ICZ ring system.

![Chemical Structure](https://i.imgur.com/1234567.png)

Finally, it should be mentioned that Top1 is a dual enzyme. Besides its DNA-relaxing action, it has an independent protein kinase activity that phosphorylates Ser/Arg-rich (SR) proteins, which are involved in spliceosome formation and expression of specific genes.²⁶⁸ Several antitumor ICZs (dechlorinated REB 11, R-3 243, NB-506 244, and dibromo analog 245) displayed an astonishing range of simultaneous modes of action: these compounds inhibited the DNA-relaxing activity of Top1, they were also DNA binding agents and, additionally, the drugs were potent inhibitors of the kinase activity of Top1.¹⁸¹,²⁴⁹,²⁵³,²⁶⁹ Inhibition of

---

Top1 kinase activity did not require DNA, implying that this action was independent of covalent Top1–DNA complexes. The inhibitory effect was also observed using a mutant Top1 enzyme that lacked DNA cleavage/religation activity, but did not affect phosphotransferase activity, indicating that the actions at both DNA cleavage and protein kinase sites were independent.²⁵³

![Chemical Structures](https://i.imgur.com/8901234.png)

4.3 Modes of action in bacteria

As far as we know, inhibition of bacterial DNA topoisomerases has not been reported for ICZ compounds. Therefore, the biological activities reported in bacteria might be due to DNA intercalation, inhibition of protein kinases, or other as-yet unknown modes of action. At least for some derivatives, the growth inhibitory activity might be related to their ability to interact directly with DNA.¹⁷⁷ However, the inhibition of cell differentiation in streptomycetes might be caused by direct action on specific protein kinases that participate in the regulation of developmental processes (see below). Nevertheless, a contribution of several modes of action cannot be excluded.

STA and K252a inhibited phosphorylation of *Streptomyces griseus* proteins in *in vitro* labeling studies.¹⁵⁵ Additionally, both compounds inhibited the formation of aerial hyphae and spores, pigment production, and production of the antibiotic streptomycin by *S. griseus*. Growth of vegetative mycelia was not affected. The compounds prevented sporulation of other *Streptomyces* species as well.¹⁵⁵ These data suggested that streptomycetes possessed several protein kinases of eukaryotic type which were essential for morphogenesis and secondary metabolism. Such observations inspired the development of an assay to screen for inhibitors of eukaryotic protein kinases by using a particular *Streptomyces* strain, which was found to be exceptionally sensitive to the effects of compounds that inhibited aerial hyphae formation.²⁷⁰ Additionally, *in vitro* phosphorylation of some proteins in a STA-producing *Streptomyces* sp. was also inhibited by STA and K252a, which suggested the need for a self-protecting mechanism in this

This journal is © The Royal Society of Chemistry 2006  
Nat. Prod. Rep., 2006, 23, 1007–1045 | 1037  
microorganism. Whether these metabolites play some role in controlling cell differentiation in the producing organisms is still an open question. On the other hand, given that STA and related compounds are potent inhibitors of eukaryotic protein kinases but, in general, do not show significant antibacterial activity, production of such metabolites might confer protection against surrounding eukaryotes. Recently, it was found that production of the related metabolite violacein in *C. violaceum* conferred protection against bacteria-eating protozoans.

### 5 Development of indolocarbazole analogs as clinical candidates

Several ICZ derivatives have entered clinical trials for the treatment of cancer or other diseases, although no analog has yet been launched onto the market. Because the status of compounds in clinical investigation can change rapidly, interested readers are encouraged to consult the recent literature and company web pages (included as references) for up-to-date information. Other sources of updated information can be found in web pages such as ClinicalTrials.gov (a service of the US National Institutes of Health), and *The Pharmaceutical Research and Manufacturers of America*. We give different designations (in brackets) for most of the clinical candidate drugs, because they have suffered successive changes of name over the years.

#### 5.1 Analogs targeting protein kinases

##### 5.1.1 Staurosporine derivatives. Among all the ICZ drugs in clinical trials, UCN-01 17 (7-hydroxy-STA, KW-2401, NSC-638850) is unique for being structurally identical to a natural product. UCN-01 has anticancer properties and inhibits several protein kinases, including various PKC isoforms, cyclin-dependent kinases, and PDK1. Additionally, UCN-01 enhances the cytotoxicity of other anticancer drugs such as DNA-damaging agents and antimetabolite drugs. The compound has entered clinical trials for treating various cancer types, and several phase I/II studies are currently focused on treating leukemias or lymphomas, advanced solid tumors, melanoma, and small-cell lung cancer. UCN-01 showed unusual pharmacokinetic features in patients with cancer, probably due to its high binding to human plasma proteins, which has hampered its clinical development.

Midostaurin 246 (PKC 412, CGP-41251, 4′-*N*-benzoyl-STA) is a potent inhibitor of several protein kinases, including PKC, VEGF, and FLT3. Midostaurin may suppress tumor growth by inhibiting tumor angiogenesis in addition to directly inhibiting tumor cell proliferation. Orally active midostaurin has entered phase I/II clinical trials for treating various cancer types, including leukemias such as acute myeloid leukemia (with or without mutated FLT3). The drug has also been tested in a phase I/II trial in patients with diabetic macular edema, based upon the ability of midostaurin to inhibit VEGF (which might contribute to the disease).

##### 5.1.2 K252a derivatives. Lestaurtinib 247 (CEP-701, KT-5555) and CEP-751 248 (KT-6587) have been shown to inhibit autophosphorylation and signaling of neurotrophin-specific Trk receptors and to display potent antitumor activity. However, these analogs also interfere with the platelet-derived growth factor,

epidermal growth factor receptor, FLT3, RET tyrosine kinase and probably other as-yet unidentified kinases. Orally active lestaurtinib has entered several clinical trials (up to phases II/III) for hematologic cancers, including acute myelogenous leukemia associated with FLT3 mutations. FLT3 mutations are associated with a poorer prognosis, with significantly lower cure rates from standard chemotherapy-based treatment. On April 2006, it was announced that the US Food and Drug Administration granted orphan drug designation for lestaurtinib for the treatment of acute myeloid leukemia. The drug has also entered a phase I trial for solid tumors in adults, phase II trials for multiple myeloma and prostate cancer, a phase I study for children with high-risk neuroblastoma, and a phase II trial for patients with severe psoriasis.

midostaurin  
(PKC 412, CGP 41251)

247 lestaurtinib (CEP-701, R=H)  
248 CEP-751 (R=Me)

A water-soluble prodrug of CEP-751 248 was developed, with the name CEP-2563 249 (KT-8391). A phase I clinical trial of CEP-2563 was conducted in patients with advanced solid tumors, but it appears that this drug is no longer being evaluated. CEP-1347 250 (KT-7515) is a 3,9-bis(ethylthiomethyl) derivative of K252a, which displays neuroprotective properties and

potent inhibitory activity against MLK kinases.¹⁹²,²³⁴ The MLKs are key participants in the activation of the JNK signaling cascade, which has been proposed to govern neuronal dysfunction and subsequent death. Orally active CEP-1347 entered a phase II/III clinical trial to determine its efficacy in delaying disability caused by the progression of Parkinson's disease.²⁷³,²⁹³ While existing drugs only offer symptomatic relief, it was expected that CEP-1347 might prevent disease progression by targeting the mechanisms involved in the pathogenesis of Parkinson's disease. However, after a review of interim results of the clinical trial, it was announced on May 2005 that the data were unlikely to provide evidence of any significant effect, and it seems that the trials have been discontinued.²⁹⁰,²⁹⁴

![Structure of CEP-1347](image)

**5.1.3 Indenopyrrolocarbazole derivatives.** CEP-7055 **251** is a 5-oxoindeno[2,1-*a*]pyrrolo[3,4-*c*]carbazole derivative, which shows antiangiogenic properties through targeting VEGF receptors.²³³ CEP-7055 is actually a water-soluble prodrug that converts *in vivo* to CEP-5214 **252**, the active metabolite. CEP-7055 entered phase I/II clinical trials for solid tumors²⁷⁴ but clinical development of the drug apparently ceased during 2005.²⁹⁰

![Structures of CEP-7055 and CEP-5214](image)

**5.1.4 Bisindolylmaleimide derivatives.** Ruboxistaurin **236** (LY333531), a macrocyclic derivative of bisindolylmaleimide, is a specific inhibitor of the β1 and β2 isoforms of PKC.¹⁶⁰ The drug has been developed for the treatment of microvascular complications in patients with diabetes mellitus.²⁹⁵,²⁹⁶ Hyperglycemia activates PKC-β, and this is associated with the development of microvascular complications in the retina, kidney, and nervous system. The mesylate salt of ruboxistaurin is being evaluated in phase II/III clinical studies for the treatment of diabetic complications including retinopathy, macular edema, nephropathy, and neuropathy.²⁷²,²⁷³,²⁹⁷–²⁹⁹ Recently, a phase III clinical trial has been completed, in which ruboxistaurin mesylate (Arxxant™) reduced the occurrence of vision loss in patients with diabetic retinopathy. A new drug application (NDA) for diabetic retinopathy was submitted by February 2006 to the US Food and Drug Administration.³⁰⁰ If ruboxistaurin is approved, it would be the first medication for the treatment of this serious complication of diabetes.

Enzastaurin **253** (LY317615), another modified bisindolylmaleimide, is a potent inhibitor of PKC-β and AKT/PI3 pathways. As a result, enzastaurin inhibits the growth of cancer cell lines (inducing apoptosis) and, additionally, inhibits VEGF-mediated angiogenesis.³⁰¹,³⁰² Orally administered enzastaurin has entered clinical trials for the treatment of cancer, including high-grade gliomas, glioblastoma (a brain tumor, phase III trials), various lymphomas, non-small-cell lung cancer, and colorectal cancer.²⁷³,³⁰¹,³⁰³

![Structure of Enzastaurin](image)

**5.2 Analogs targeting DNA or DNA topoisomerases**

**5.2.1 Rebeccamycin derivatives.** REB is essentially insoluble in water and, therefore, soon after its discovery, the design of analogs with improved water solubility was initiated. One of these compounds, 6-N-(diethylaminoethyl)-REB or becatecarin **254**, possessed improved pharmacokinetic and anticancer properties and was pursued as a clinical candidate.¹¹,³⁰⁴,³⁰⁵ Becatecarin has been named XL119, DEAE-REB, NSC-655649, BMY-27557-14, BMS 181176, rebeccamycin analog, and even just “rebeccamycin”. Unexpectedly, mechanism of action studies revealed that becatecarin was a strong DNA intercalating agent, whereas REB was not.¹¹ This strong DNA binding was most likely the primary cause for the strong inhibition of the catalytic activity of topoisomerase II exhibited by this compound, without inducing any Top1-mediated DNA cleavage.¹¹ Becatecarin has entered clinical trials for treating cancer, including biliary and hepatobiliary tumors (phase II and III trials), leukemias, renal cell carcinoma, colorectal cancer, breast cancer, ovarian epithelial cancer, and non-small-cell lung cancer.²⁷³,³⁰⁵ Additionally, becatecarin has entered clinical

![Structure of Becatecarin](image)

studies for treatment of children with solid tumors, non-Hodgkin's lymphoma, and neuroblastoma.²⁷³

As already mentioned, a series of fluoroindolocarbazoles 138–141 was obtained by feeding different fluorotryptophans to cultures of REB-producing *L. aerocolonigenes*.¹¹,¹¹⁵ Difluoro analog 140 was found to be a potent inducer of Top1-mediated DNA cleavage.¹¹ An analog program focused on this molecule culminated with the synthesis of a 4′-fluoro derivative (BMS-250749 255), and a 6′-amino analog with a thiophene core (BMS-251873 256), with improved antitumor activities.¹¹,²⁴⁴,²⁴⁵ It has been reported that BMS-250749 255 has entered human clinical trials.²⁴⁵

![Structure of BMS-250749](https://i.imgur.com/structure_bms_250749.png)

**255**

BMS-250749

by a more polar group, and the two hydroxyl groups on the chromophore were shifted from positions 1, 11 to positions 2, 10.³¹⁴ These changes increased the aqueous solubility and the plasma stability of the molecule, without penalizing the action on Top1, and reinforced the cytotoxicity of the compound.³¹⁴–³¹⁶ Edotecarin entered clinical trials for the treatment of cancer, including breast cancer, glioblastoma multiforme (phase III trial), and gastric cancer.²⁷³,³⁰⁷,³⁰⁸

![Structure of Edotecarin](https://i.imgur.com/structure_edotecarin.png)

**257**

edotecarin (J-107088)

## 6 Perspectives and concluding remarks

The ICZ family of natural products is a source of lead compounds with potential therapeutic applications in the treatment of cancer and other diseases. The compounds of natural origin have been isolated from diverse groups of organisms, from the prokaryote world (actinomycetes, cyanobacteria, β-proteobacteria) to eukaryotes (myxomycetes, basidiomycetes, marine invertebrates). So far, genetic and biochemical aspects of their biosynthesis have only been studied in actinomycetes and β-proteobacteria. Despite the structural differences existing between the final products (compare REB 9 and violacein 127), these studies revealed that the pathways shared some early steps responsible for the condensation of two tryptophan units. Given the high sequence similarity found between the enzymes involved (*e.g.*, 47% similarity between RebD and VioB), it is reasonable to speculate a common origin and, probably, a horizontal transfer of ancestor genes from an actinomycete to a β-proteobacterium. It will be exciting to see if these common steps are also found in the ICZ pathways of other organisms, or if novel biochemistry is involved.

It seems very likely that ICZs isolated from marine invertebrates are originally synthesized by associated microorganisms (probably actinomycetes or cyanobacteria), although the compounds might be further modified by the invertebrates themselves. If this is the case, the identification of the producing microorganisms may provide new sources for ICZ production by fermentation. Cyanobacterial indolo[2,3-*a*]carbazoles might be synthesized from indolopyrrolocarbazole intermediates, as suggested by co-occurrence of tjipanzole J 116, and by isolation of metabolite 178 from an *E. coli* strain producing REB due to expression of *reb* genes. The putative removal of the pyrrolo[3,4-*c*] ring from an indolopyrrolocarbazole would be a noteworthy reaction. On the other hand, production of arcyriaflavin A 74 and chromopyrrolic acid 92 from both slime molds and actinomycetes also suggests the possibility of common biosynthetic machinery.
As an addition to chemical synthesis, the introduction of biological processes might help in the production of useful derivatives in a cost-effective way and with lower environmental problems. The identification of novel producing organisms and the genes responsible for ICZ biosynthesis provide the necessary “toolkit” for such purposes. Combining fermentation of microorganisms (either wild-type or genetically manipulated) with *in vitro* enzymatic bioconversions and chemical synthesis will greatly expand the available repertory of analogs. The ICZ derivatives will continue to be useful as inhibitors of key enzymes in cell biology studies. Given the proven utility of targeting protein kinases and DNA topoisomerases for treating a range of important human diseases, it is hoped that some ICZ analogs will provide new medicines in the near future.

**Note added in proof**

Two relevant reports on the biochemistry of indolocarbazole biosynthesis have been recently published. \({ }^{317,318}\)

## 7 Acknowledgements

We are indebted to our co-workers Aaroa P. Salas and Alfredo F. Braña at the University of Oviedo, and to our collaborators Lili Zhu, Igor A. Butovich and Jürgen Rohr at the University of Kentucky, who contributed to our research on ICZ biosynthesis. The generous financial support by the Spanish Ministry of Education and Science (BIO2000-0274 and BMC2003-00478), Plan Regional de I+D+i del Principado de Asturias (Grant GEMED01-05), and Red Temática de Investigación Cooperativa de Centros de Cáncer (Ministerio de Sanidad y Consumo, Spain) is gratefully acknowledged. We would also like to thank Obra Social Cajastur for supporting C.S.

## 8 References

1. M. Gill and W. Steglich, in *Progress in the Chemistry of Organic Natural Products*, ed. W. Herz, H. Grisebach, G. W. Kirby and C. Tamm, Springer-Verlag, New York, 1987, vol. 51, pp. 1–317.
2. W. Steglich, *Pure Appl. Chem.*, 1989, **61**, 281–288.
3. J. Bergman, in *Studies in Natural Product Chemistry*, ed. Atta-ur-Rahman, Elsevier, Amsterdam, 1988, vol. 1 (Stereoselective synthesis, Part A), pp. 3–30.
4. J. Bergman, T. Janosik and N. Wahlstrom, in *Advances in Heterocyclic Chemistry*, ed. A. Katritzky, Academic Press, New York, 2001, vol. 80, pp. 1–71.
5. G. W. Gribble and S. J. Berthel, in *Studies in Natural Product Chemistry*, ed. Atta-ur-Rahman, Elsevier, Amsterdam, 1993, vol. 12 (Stereoselective synthesis, Part H), pp. 365–409.
6. M. Prudhomme, *Curr. Pharm. Des.*, 1997, **3**, 265–290.
7. M. Prudhomme, *Curr. Med. Chem.*, 2000, **7**, 1189–1212.
8. M. Prudhomme, *Eur. J. Med. Chem.*, 2003, **38**, 123–140.
9. U. Pindur, Y. S. Kim and F. Mehrabani, *Curr. Med. Chem.*, 1999, **6**, 29–69.
10. H. J. Knolker and K. R. Reddy, *Chem. Rev.*, 2002, **102**, 4303–4427.
11. B. H. Long, W. C. Rose, D. M. Vyas, J. A. Matson and S. Forenza, *Curr. Med. Chem.: Anti-Cancer Agents*, 2002, **2**, 255–266.
12. T. Kawasaki and K. Higuchi, *Nat. Prod. Rep.*, 2005, **22**, 761–793.
13. C. Sánchez, C. Méndez and J. A. Salas, *J. Ind. Microbiol. Biotechnol.*, 2006, **33**, 560–568.
14. S. Omura, Y. Iwai, A. Hirano, A. Nakagawa, J. Awaya, H. Tsuchiya, Y. Takahashi and R. Masuma, *J. Antibiot.*, 1977, **30**, 275–282.
15. D. Schroeder, K. S. Lam, J. Mattei and G. A. Hesler, *Eur. Pat.*, EP0388962, 1990.
16. H. Morioka, M. Ishihara, H. Shibai and T. Suzuki, *Agric. Biol. Chem.*, 1985, **49**, 1959–1963.
17. D. E. Nettleton, T. W. Doyle, B. Krishnan, G. K. Matsumoto and J. Clardy, *Tetrahedron Lett.*, 1985, **26**, 4011–4014.
18. J. A. Bush, B. H. Long, J. J. Catino, W. T. Bradner and K. Tomita, *J. Antibiot.*, 1987, **40**, 668–678.
19. J. A. Matson, *US Pat.*, US4567143, 1986.
20. K. S. Lam, L. McDonald, J. Mattei, S. Forenza and J. A. Matson, *US Pat.*, US5326754, 1994.
21. M. Sezaki, T. Sasaki, T. Nakazawa, U. Takeda, M. Iwata, T. Watanabe, M. Koyama, F. Kai, T. Shomura and M. Kojima, *J. Antibiot.*, 1985, **38**, 1437–1439.
22. H. Kase, K. Iwahashi and Y. Matsuda, *J. Antibiot.*, 1986, **39**, 1059–1065.
23. T. Yasuzawa, T. Iida, M. Yoshida, N. Hirayama, M. Takahashi, K. Shirahata and H. Sano, *J. Antibiot.*, 1986, **39**, 1072–1078.
24. S. Nakanishi, Y. Matsuda, K. Iwahashi and H. Kase, *J. Antibiot.*, 1986, **39**, 1066–1071.
25. S. Oka, M. Kodama, H. Takeda, N. Tomikuza and H. Suzuki, *Agric. Biol. Chem.*, 1986, **50**, 2723–2727.
26. I. Takahashi, E. Kobayashi, K. Asano, M. Yoshida and H. Nakano, *J. Antibiot.*, 1987, **40**, 1782–1784.
27. I. Takahashi, K. Asano, I. Kawamoto, T. Tamaoki and H. Nakano, *J. Antibiot.*, 1989, **42**, 564–570.
28. H. Nakano, I. Takahashi, T. Tamaoki, F. Tomita, I. Kawamoto, K. Asano, M. Morimoto and H. Nomoto, *US Pat.*, US4935415, 1990.
29. I. Takahashi, Y. Saitoh, M. Yoshida, H. Sano, H. Nakano, M. Morimoto and T. Tamaoki, *J. Antibiot.*, 1989, **42**, 571–576.
30. J. A. Matson, C. Claridge, J. A. Bush, J. Titus, W. T. Bradner, T. W. Doyle, A. C. Horan and M. Patel, *J. Antibiot.*, 1989, **42**, 1547–1555.
31. J. Golik, T. W. Doyle, B. Krishnan, G. Dubay and J. A. Matson, *J. Antibiot.*, 1989, **42**, 1784–1789.
32. S. Tanida, M. Takizawa, T. Takahashi, S. Tsubotani and S. Harada, *J. Antibiot.*, 1989, **42**, 1619–1630.
33. S. Tsubotani, S. Tanida and S. Harada, *Tetrahedron*, 1991, **47**, 3565–3574.
34. D. R. Schroeder, K. S. Lam, J. M. Mattei and G. A. Hesler, *US Pat.*, US4973552, 1990.
35. H. Takahashi, H. Osada, M. Uramoto and K. Isono, *J. Antibiot.*, 1990, **43**, 168–173.
36. H. Osada, H. Takahashi, K. Tsunoda, H. Kusakabe and K. Isono, *J. Antibiot.*, 1990, **43**, 163–167.
37. H. Osada, M. Satake, H. Koshino, R. Onose and K. Isono, *J. Antibiot.*, 1992, **45**, 278–279.
38. E. B. Barrabee, A. C. Horan, F. A. Gentile and M. G. Patel, *WIPO Pat.*, WO9109034, 1991.
39. K. Kojiri, H. Kondo, T. Yoshinari, H. Arakawa, S. Nakajima, F. Satoh, K. Kawamura, A. Okura, H. Suda and M. Okanishi, *J. Antibiot.*, 1991, **44**, 723–728.
40. H. Koshino, H. Osada and K. Isono, *J. Antibiot.*, 1992, **45**, 195–198.
41. H. Osada, H. Koshino, T. Kudo, R. Onose and K. Isono, *J. Antibiot.*, 1992, **45**, 189–194.
42. H. Koshino, H. Osada, S. Amano, R. Onose and K. Isono, *J. Antibiot.*, 1992, **45**, 1428–1432.
43. J. B. McAlpine, J. P. Karwowski, M. Jackson, M. M. Mullally, J. E. Hochlowski, U. Premachandran and N. S. Burres, *J. Antibiot.*, 1994, **47**, 281–288.
44. Y. Cai, A. Fredenhagen, P. Hug and H. H. Peter, *J. Antibiot.*, 1995, **48**, 143–148.
45. Y. Cai, A. Fredenhagen, P. Hug, T. Meyer and H. H. Peter, *J. Antibiot.*, 1996, **49**, 519–526.
46. Y. Cai, A. Fredenhagen, P. Hug and H. H. Peter, *J. Antibiot.*, 1996, **49**, 1060–1062.
47. D. E. Williams, V. S. Bernan, F. V. Ritacco, W. M. Maiese, M. Greenstein and R. J. Andersen, *Tetrahedron Lett.*, 1999, **40**, 7171–7174.
48. M. Ubukata, N. Tamehiro, N. Matsuura and N. Nakajima, *J. Antibiot.*, 1999, **52**, 921–924.
49. N. Matsuura, N. Tamehiro, T. Andoh, A. Kawashima and M. Ubukata, *J. Antibiot.*, 2002, **55**, 355–362.
50. N. Tamehiro, N. Matsuura, Y. Feng, N. Nakajima and M. Ubukata, *J. Antibiot.*, 2002, **55**, 363–370.
51. L. M. Hernandez, J. A. Blanco, J. P. Baz, J. L. Puentes, F. R. Millan, F. E. Vazquez, R. I. Fernandez-Chimeno and D. G. Gravalos, *J. Antibiot.*, 2000, **53**, 895–902.
52. H. Onaka, S. Taniguchi, Y. Igarashi and T. Furumai, *Biosci. Biotechnol. Biochem.*, 2003, **67**, 127–138.
53 T. C. Turek-Etienne, M. Lei, J. S. Terracciano, E. F. Langsdorf, R. W. Bryant, R. F. Hart and A. C. Horan, *J. Biomol. Screening*, 2004, **9**, 52–61.

54 X. Han, C. Cui, Q. Gu, W. Zhu, H. Liu, J. Gu and H. Osada, *Tetrahedron Lett.*, 2005, **46**, 6137–6140.

55 S. S. Mitchell, K. S. Lam, B. C. Potts, G. Tsueng, J. Grodberg, D. White and K. A. Reed, *WIPO Pat.*, WO2005070922, 2005.

56 V. P. Gullo, J. McAlpine, K. S. Lam, D. Baker and F. Petersen, *J. Ind. Microbiol. Biotechnol.*, 2006, **33**, 523–531.

57 W. Steglich, B. Steffan, L. Kopanski and G. Eckhardt, *Angew. Chem., Int. Ed. Engl.*, 1980, **19**, 459–460.

58 L. Kopanski, G.-R. Li, H. Besl and W. Steglich, *Liebigs Ann. Chem.*, 1982, 1722–1729.

59 T. Hashimoto, A. Yasuda, K. Akazawa, S. Takaoka, M. Tori and Y. Asakawa, *Tetrahedron Lett.*, 1994, **35**, 2559–2560.

60 M. S. Buchanan, T. Hashimoto and Y. Asakawa, *Phytochemistry*, 1996, **41**, 791–794.

61 R. Frode, C. Hinze, I. Josten, B. Schmidt, B. Steffan and W. Steglich, *Tetrahedron Lett.*, 1994, **35**, 1689–1690.

62 S. Nakatani, A. Naoe, Y. Yamamoto, T. Yamauchi, N. Yamaguchi and M. Ishibashi, *Bioorg. Med. Chem. Lett.*, 2003, **13**, 2879–2881.

63 K. Kamata, M. Kiyota, A. Naoe, S. Nakatani, Y. Yamamoto, M. Hayashi, K. Komiyama, T. Yamori and M. Ishibashi, *Chem. Pharm. Bull.*, 2005, **53**, 594–597.

64 G. Knubel, L. K. Larsen, R. E. Moore, I. A. Levine and G. M. Patterson, *J. Antibiot.*, 1990, **43**, 1236–1239.

65 R. Bonjouklian, T. A. Smitka, L. E. Doolin, R. M. Molloy, M. Debono, S. A. Shaffer, R. E. Moore, J. B. Stewart and G. M. L. Patterson, *Tetrahedron*, 1991, **47**, 7739–7750.

66 B. S. Falch, G. M. König, A. D. Wright, O. Sticher, H. Rüegger and G. Bernardinelli, *J. Org. Chem.*, 1993, **58**, 6570–6575.

67 A. D. Wright, O. Papendorf and G. M. König, *J. Nat. Prod.*, 2005, **68**, 459–461.

68 R. B. Kinnel and P. J. Scheuer, *J. Org. Chem.*, 1992, **57**, 6327–6329.

69 P. A. Horton, R. E. Longley, O. J. McConnell and L. M. Ballas, *Experientia*, 1994, **50**, 843–845.

70 C. L. Cantrell, A. Groweiss, K. R. Gustafson and M. R. Boyd, *Nat. Prod. Lett.*, 1999, **14**, 39–46.

71 P. Schupp, C. Eder, P. Proksch, V. Wray, B. Schneider, M. Herderich and V. Paul, *J. Nat. Prod.*, 1999, **62**, 959–962.

72 P. Schupp, P. Proksch and V. Wray, *J. Nat. Prod.*, 2002, **65**, 295–298.

73 P. Proksch, R. Ebel, R. A. Edrada, P. Schupp, W. H. Lin Sudarsono, V. Wray and K. Steube, *Pure Appl. Chem.*, 2003, **75**, 343–352.

74 K. M. Meragelman, L. M. West, P. T. Northcote, L. K. Pannell, T. C. McKee and M. R. Boyd, *J. Org. Chem.*, 2002, **67**, 6671–6677.

75 H. Hoshino, Y. Kojima, T. Hayashi, T. Uchiyama and K. Kaneko, *Biosci., Biochem.*, 1993, **57**, 775–781.

76 N. Duran and C. F. Menck, *Crit. Rev. Microbiol.*, 2001, **27**, 201–22.

77 Brazilian National Genome Project Consortium, *Proc. Natl. Acad. Sci. U. S. A.*, 2003, **100**, 11660–11665.

78 P. Mayser, H. Stapelkamp, H. J. Kramer, M. Podobinska, W. Wallbott, B. Irlinger and W. Steglich, *Antonie van Leeuwenhoek*, 2003, **84**, 185–191.

79 H. J. Kramer, D. Kessler, U. C. Hippler, B. Irlinger, W. Hort, R. H. Bodeker, W. Steglich and P. Mayser, *ChemBioChem*, 2005, **6**, 2290–2297.

80 B. Irlinger, A. Bartsch, H.-J. Krämer, P. Mayser and W. Steglich, *Helv. Chim. Acta*, 2005, **88**, 1472–1485.

81 A. Furusaki, N. Hashiba, T. Matsumoto, A. Hirano, Y. Iwai and S. Omura, *J. Chem. Soc.*, *Chem. Commun.*, 1978, 800–801.

82 N. Funato, H. Takayanagi, Y. Konda, Y. Toda, Y. Harigaya, Y. Iwai and S. Omura, *Tetrahedron Lett.*, 1994, **35**, 1251–1254.

83 J. T. Link, S. Raghavan, M. Gallant, S. J. Danishefsky, T. C. Chou and L. M. Ballas, *J. Am. Chem. Soc.*, 1996, **118**, 2825–2842.

84 Y. Takahashi, M. Shinose, A. Seino, Y. Iwai and S. Omura, *Actinomycetologica*, 1995, **9**, 19–26.

85 Q. Xie, Y. Wang, Y. Huang, Y. Wu, F. Ba and Z. Liu, *Int. J. Syst. Evol. Microbiol.*, 2002, **52**, 1815–1820.

86 T. Tamaoki, H. Nomoto, I. Takahashi, Y. Kato, M. Morimoto and F. Tomita, *Biochem. Biophys. Res. Commun.*, 1986, **135**, 397–402.

87 U. T. Ruegg and G. M. Burgess, *Trends Pharmacol. Sci.*, 1989, **10**, 218–220.

88 S. Omura, Y. Sasaki, Y. Iwai and H. Takeshima, *J. Antibiot.*, 1995, **48**, 535–548.

89 D. E. Nettleton, J. A. Bush, W. T. Bradner and T. W. Doyle, *US Pat.*, US4552842, 1985.

90 T. Kaneko, H. Wong, K. T. Okamoto and J. Clardy, *Tetrahedron Lett.*, 1985, **26**, 4015–4018.

91 L. P. Labeda, *Int. J. Syst. Bacteriol.*, 1986, **36**, 109–110.

92 D. P. Labeda, K. Hatano, R. M. Kroppenstedt and T. Tamura, *Int. J. Syst. Evol. Microbiol.*, 2001, **51**, 1045–1050.

93 S. Chiba, M. Suzuki and K. Ando, *Int. J. Syst. Bacteriol.*, 1999, **49**, 1623–1630.

94 A. C. Horan, J. Golik, J. A. Matson and M. G. Patel, *US Pat.*, US4743594, 1988.

95 J. D. Chisholm, J. Golik, B. Krishnan, J. A. Matson and D. L. Van Vranken, *J. Am. Chem. Soc.*, **121**, 3801–3802.

96 S. Hashimoto, *J. Cell Biol.*, 1988, **107**, 1531–1539.

97 S. Koizumi, M. L. Contreras, Y. Matsuda, T. Hama, P. Lazarovici and G. Guroff, *J. Neurosci.*, 1988, **8**, 715–721.

98 D. Rasouly, E. Rahamim, D. Lester, Y. Matsuda and P. Lazarovici, *Mol. Pharmacol.*, 1992, **42**, 35–43.

99 M. Murase, K. Watanabe, T. Yoshida and S. Tobinaga, *Chem. Pharm. Bull.*, 2000, **48**, 81–84.

100 P. Schupp, K. Steube, C. Meyer and P. Proksch, *Cancer Lett.*, 2001, **174**, 165–172.

101 L. F. Bjeldanes, J. Y. Kim, K. R. Grose, J. C. Bartholomew and C. A. Bradfield, *Proc. Natl. Acad. Sci. U. S. A.*, 1991, **88**, 9543–9547.

102 D. Meksuriyen and G. A. Cordell, *J. Nat. Prod.*, 1988, **51**, 893–899.

103 S.-W. Yang and G. A. Cordell, *J. Nat. Prod.*, 1996, **59**, 828–833.

104 S.-W. Yang and G. A. Cordell, *J. Nat. Prod.*, 1997, **60**, 788–790.

105 S.-W. Yang and G. A. Cordell, *J. Nat. Prod.*, 1997, **60**, 236–241.

106 S. W. Yang, L. J. Lin, G. A. Cordell, P. Wang and D. G. Corley, *J. Nat. Prod.*, 1999, **62**, 1551–1553.

107 C. J. Pearce, T. W. Doyle, S. Forenza, K. S. Lam and D. R. Schroeder, *J. Nat. Prod.*, 1988, **51**, 937–940.

108 K. S. Lam, S. Forenza, T. W. Doyle and C. J. Pearce, *J. Ind. Microbiol.*, 1990, **6**, 291–294.

109 Y. Feng, S. Mitsuhashi, T. Kishimoto and M. Ubukata, *J. Antibiot.*, 2005, **58**, 759–765.

110 T. Hoshino, T. Kondo, T. Uchiyama and N. Ogasawara, *Agric. Biol. Chem.*, 1987, **51**, 965–968.

111 T. Hoshino, T. Takano, S. Hori and N. Ogasawara, *Agric. Biol. Chem.*, 1987, **51**, 2733–2741.

112 A. Z. Momen and T. Hoshino, *Biosci., Biotechnol., Biochem.*, 2000, **64**, 539–549.

113 T. Hoshino and N. Ogasawara, *Agric. Biol. Chem.*, 1990, **54**, 2339–2346.

114 K. S. Lam, D. R. Schroeder, J. M. Veitch, J. A. Matson and S. Forenza, *J. Antibiot.*, 1991, **44**, 934–939.

115 K. S. Lam, D. R. Schroeder, J. M. Veitch, K. L. Colson, J. A. Matson, W. C. Rose, T. W. Doyle and S. Forenza, *J. Antibiot.*, 2001, **54**, 1–9.

116 S. W. Yang and G. A. Cordell, *J. Nat. Prod.*, 1997, **60**, 44–48.

117 S. W. Yang and G. A. Cordell, *J. Nat. Prod.*, 1997, **60**, 230–235.

118 K. Kojiri, H. Suzuki, H. Kondo and H. Suda, *US Pat.*, US5437996, 1995.

119 T. Hoshino, T. Hayashi and T. Uchiyama, *Biosci., Biotechnol., Biochem.*, 1994, **58**, 279–282.

120 T. Hoshino, T. Hayashi and T. Odajima, *J. Chem. Soc., Perkin Trans. I*, 1995, 1565–1571.

121 T. Hoshino, M. Yamamoto and T. Uchiyama, *Biosci., Biotechnol., Biochem.*, 1993, **57**, 1609–1610.

122 A. Z. M. Ruhul Momen, T. Mizuoka and T. Hoshino, *J. Chem. Soc., Perkin Trans. I*, 1998, 3087–3092.

123 P. Hoehn, O. Ghisalba, T. Moerker and H. H. Peter, *J. Antibiot.*, 1995, **48**, 300–305.

124 P. Hoehn-Thierry, O. Ghisalba, H. Peter and T. Moerker, *WIPO Pat.*, WO9500520, 1995.

125 K. Goeke, P. Hoehn and O. Ghisalba, *J. Antibiot.*, 1995, **48**, 428–430.

126 S. Weidner, M. Kittelmann, K. Goeke, O. Ghisalba and H. Zahner, *J. Antibiot.*, 1998, **51**, 679–682.

127 A. Fredenhagen and H. H. Peter, *Tetrahedron*, 1996, **52**, 1235–1238.

128 B. M. Stoltz and J. L. Wood, *Tetrahedron Lett.*, 1996, **37**, 3929–3930.

129 J. L. Wood, B. M. Stoltz and S. N. Goodman, *J. Am. Chem. Soc.*, 1996, **118**, 10656–10657.

130 J. L. Wood, B. M. Stoltz, S. N. Goodman and K. Onwueme, *J. Am. Chem. Soc.*, 1997, **119**, 9652–9661.

131 Y. Feng, N. Matsuura and M. Ubukata, *J. Antibiot.*, 2004, **57**, 627–633.

132 T. Ohuchi, A. Ikeda-Araki, A. Watanabe-Sakamoto, K. Kojiri, M. Nagashima, M. Okanishi and H. Suda, *J. Antibiot.*, 2000, **53**, 393–403.

133 C. Sánchez, I. A. Butovich, A. F. Braña, J. Rohr, C. Méndez and J. A. Salas, *Chem. Biol.*, 2002, **9**, 519–531.

134 C. G. Hyun, T. Bililign, J. Liao and J. S. Thorson, *ChemBioChem*, 2003, **4**, 114–117.

135 T. Nishizawa, C. C. Aldrich and D. H. Sherman, *J. Bacteriol.*, 2005, **187**, 2084–2092.

136 H. Onaka, S. Taniguchi, Y. Igarashi and T. Furumai, *J. Antibiot.*, 2002, **55**, 1063–1071.

137 C. Sánchez, L. Zhu, A. F. Braña, A. P. Salas, J. Rohr, C. Méndez and J. A. Salas, *Proc. Natl. Acad. Sci. U. S. A.*, 2005, **102**, 461–466.

138 Q. Gao, C. Zhang, S. Blanchard and J. S. Thorson, *Chem. Biol.*, 2006, **13**, 733–743.

139 H. Onaka, S. Taniguchi, H. Ikeda, Y. Igarashi and T. Furumai, *J. Antibiot.*, 2003, **56**, 950–956.

140 T. Schupp, N. Engel, J. Bietenhader, C. Toupet and A. Pospiech, WIPO Pat., WO9708323, 1997.

141 A. P. Salas, L. Zhu, C. Sánchez, A. F. Braña, J. Rohr, C. Méndez and J. A. Salas, *Mol. Microbiol.*, 2005, **58**, 17–27.

142 J. M. Pemberton, K. M. Vincent and R. J. Penfold, *Curr. Microbiol.*, 1991, **22**, 355–358.

143 P. R. August, T. H. Grossman, C. Minor, M. P. Draper, I. A. MacNeil, J. M. Pemberton, K. M. Call, D. Holt and M. S. Osburne, *J. Mol. Microbiol. Biotechnol.*, 2000, **2**, 513–519.

144 S. F. Brady, C. J. Chao, J. Handelsman and J. Clardy, *Org. Lett.*, 2001, **3**, 1981–1984.

145 C. Sánchez, A. F. Braña, C. Méndez and J. A. Salas, *ChemBioChem*, 2006, **7**, 1231–1240.

146 S. Y. Kim, J. S. Park, J. Y. Lee and K. B. Oh, presented at the International Meeting of the Federation of Korean Microbiological Societies, Seoul, October 2005.

147 T. Hoshino and M. Yamamoto, *Biosci., Biotechnol., Biochem.*, 1997, **61**, 2134–2136.

148 A. R. Howard-Jones and C. T. Walsh, *Biochemistry*, 2005, **44**, 15652–15663.

149 E. Yeh, S. Garneau and C. T. Walsh, *Proc. Natl. Acad. Sci. U. S. A.*, 2005, **102**, 3960–3965.

150 T. Nishizawa, S. Gruschow, D. H. Jayamaha, C. Nishizawa-Harada and D. H. Sherman, *J. Am. Chem. Soc.*, 2006, **128**, 724–725.

151 S. Asamizu, Y. Kato, Y. Igarashi, T. Furumai and H. Onaka, *Tetrahedron Lett.*, 2006, **47**, 473–475.

152 C. Zhang, C. Albermann, X. Fu, N. R. Peters, J. D. Chisholm, G. Zhang, E. J. Gilbert, P. G. Wang, D. L. Van Vranken and J. S. Thorson, *ChemBioChem*, 2006, **7**, 795–804.

153 C. Zhang, R. L. Weller, J. S. Thorson and S. R. Rajski, *J. Am. Chem. Soc.*, 2006, **128**, 2760–2761.

154 H. Onaka, S. Asamizu, Y. Igarashi, R. Yoshida and T. Furumai, *Biosci., Biotechnol., Biochem.*, 2005, **69**, 1753–1759.

155 S. K. Hong, A. Matsumoto, S. Horinouchi and T. Beppu, *Mol. Gen. Genet.*, 1993, **236**, 347–354.

156 U. Pindur and T. Lemster, *Curr. Med. Chem.*, 2001, **8**, 1681–198.

157 S. Akinaga, K. Sugiyama and T. Akiyama, *Anti-Cancer Drug Des.*, 2000, **15**, 43–52.

158 M. Facompre, C. Carrasco, P. Colson, C. Houssier, J. D. Chisholm, D. L. Van Vranken and C. Bailly, *Mol. Pharmacol.*, 2002, **62**, 1215–1227.

159 L. L. Kodach, C. L. Bos, N. Duran, M. P. Peppelenbosch, C. V. Ferreira and J. C. Hardwick, *Carcinogenesis*, 2006, **27**, 508–516.

160 M. R. Jirousek, J. R. Gillig, C. M. Gonzalez, W. F. Heath, J. H. McDonald, 3rd, D. A. Neel, C. J. Rito, U. Singh, L. E. Stramm, A. Melikian-Badalian, M. Baevsky, L. M. Ballas, S. E. Hall, L. L. Winneroski and M. M. Faul, *J. Med. Chem.*, 1996, **39**, 2664–2671.

161 M. F. Braña, L. Añorbe, G. Tarrasón, F. Mitjans and J. Piulats, *Bioorg. Med. Chem. Lett.*, 2001, **11**, 2701–2703.

162 E. Caballero, M. Adeva, S. Calderon, H. Sahagun, F. Tome, M. Medarde, J. L. Fernandez, M. Lopez-Lazaro and M. J. Ayuso, *Bioorg. Med. Chem.*, 2003, **11**, 3413–3421.

163 M. M. Faul, J. R. Gillig, M. R. Jirousek, L. M. Ballas, T. Schotten, A. Kahl and M. Mohr, *Bioorg. Med. Chem. Lett.*, 2003, **13**, 1857–1859.

164 G. Zhang, J. Shen, H. Cheng, L. Zhu, L. Fang, S. Luo, M. T. Muller, G. E. Lee, L. Wei, Y. Du, D. Sun and P. G. Wang, *J. Med. Chem.*, 2005, **48**, 2600–2611.

165 P. Moreau, N. Gaillard, C. Marminon, F. Anizon, N. Dias, B. Baldeyrou, C. Bailly, A. Pierre, J. Hickman, B. Pfeiffer, P. Renard and M. Prudhomme, *Bioorg. Med. Chem.*, 2003, **11**, 4871–4879.

166 P. Moreau, S. Holbeck, M. Prudhomme and E. A. Sausville, *Anti-Cancer Drugs*, 2005, **16**, 145–150.

167 Y. Pommier, P. Pourquier, Y. Urasaki, J. Wu and G. S. Laco, *Drug Resist. Updates*, 1999, **2**, 307–318.

168 J. Robert, A. Vekris, P. Pourquier and J. Bonnet, *Crit. Rev. Oncol. Hematol.*, 2004, **51**, 205–227.

169 J. R. Jackson, A. Gilmartin, C. Imburgia, J. D. Winkler, L. A. Marshall and A. Roshak, *Cancer Res.*, 2000, **60**, 566–572.

170 T. Kawabe, *Mol. Cancer Ther.*, 2004, **3**, 513–519.

171 A. Y. Chen, S. J. Shih, M. Hsiao, M. L. Rothenberg and M. Prudhomme, *Mol. Pharmacol.*, 2004, **66**, 553–560.

172 M. Sancelme, S. Fabre and M. Prudhomme, *J. Antibiot.*, 1994, **47**, 792–798.

173 Y. Nakamura, C. Asada and T. Sawada, *Biotechnol. Bioprocess Eng.*, 2003, **8**, 37–40.

174 E. R. Pereira, S. Fabre, M. Sancelme, M. Prudhomme and M. Rapp, *J. Antibiot.*, 1995, **48**, 863–868.

175 E. R. Pereira, L. Belin, M. Sancelme, M. Prudhomme, M. Ollier, M. Rapp, D. Severe, J. F. Riou, D. Fabbro and T. Meyer, *J. Med. Chem.*, 1996, **39**, 4471–4477.

176 P. Moreau, M. Sancelme, C. Bailly, S. Leonce, A. Pierre, J. Hickman, B. Pfeiffer and M. Prudhomme, *Eur. J. Med. Chem.*, 2001, **36**, 887–897.

177 F. Anizon, P. Moreau, M. Sancelme, W. Laine, C. Bailly and M. Prudhomme, *Bioorg. Med. Chem.*, 2003, **11**, 3709–3722.

178 L. L. Leon, C. C. Miranda, A. O. De Souza and N. Duran, *J. Antimicrob. Chemother.*, 2001, **48**, 449–450.

179 K. A. Qatsha, C. Rudolph, D. Marme, C. Schachtele and W. S. May, *Proc. Natl. Acad. Sci. U. S. A.*, 1993, **90**, 4674–4678.

180 P. Moreau, F. Anizon, M. Sancelme, M. Prudhomme, C. Bailly, D. Severe, J. F. Riou, D. Fabbro, T. Meyer and A. M. Aubertin, *J. Med. Chem.*, 1999, **42**, 584–592.

181 P. Moreau, F. Anizon, M. Sancelme, M. Prudhomme, D. Severe, J. F. Riou, J. F. Goossens, J. P. Henichart, C. Bailly, E. Labourier, J. Tazzi, D. Fabbro, T. Meyer and A. M. Aubertin, *J. Med. Chem.*, 1999, **42**, 1816–1822.

182 A. Zimmermann, H. Wilts, M. Lenhardt, M. Hahn and T. Mertens, *Antiviral Res.*, 2000, **48**, 49–60.

183 M. J. Slater, R. Baxter, R. W. Bonser, S. Cockerill, K. Gohil, N. Parry, E. Robinson, R. Randall, C. Yeates, W. Snowden and A. Walters, *Bioorg. Med. Chem. Lett.*, 2001, **11**, 1993–1995.

184 M. Marschall, M. Stein-Gerlach, M. Freitag, R. Kupfer, M. van den Bogaard and T. Stamminger, *J. Gen. Virol.*, 2002, **83**, 1013–1023.

185 E. Gershburg, K. Hong and J. S. Pagano, *Antimicrob. Agents Chemother.*, 2004, **48**, 1900–1903.

186 M. Kubbies, B. Goller, E. Russmann, H. Stockinger and W. Scheuer, *Eur. J. Immunol.*, 1989, **19**, 1393–1398.

187 K. M. McCoy and C. J. Hatton, US Pat., US4735939, 1988.

188 J. E. Merritt, J. A. Sullivan, L. Drew, A. Khan, K. Wilson, M. Mulqueen, W. Harris, D. Bradshaw, C. H. Hill, M. Rumsby and R. Warr, *Cancer Chemother. Pharmacol.*, 1999, **43**, 371–378.

189 G. Conseil, J. M. Perez-Victoria, J. M. Jault, F. Gamarro, A. Goffau, J. Hofmann and A. Di Pietro, *Biochemistry*, 2001, **40**, 2564–2571.

190 P. Lazarovici, D. Rasouly, L. Friedman, R. Tabekman, H. Ovadia and Y. Matsuda, *Adv. Exp. Med. Biol.*, 1996, **391**, 367–377.

191 G. H. Jeohn, C. L. Cooper, K. J. Jang, H. C. Kim and J. S. Hong, *Ann. N. Y. Acad. Sci.*, 2002, **962**, 347–359.

192 H. A. Mucke, *IDrugs*, 2003, **6**, 377–383.

193 M. L. Trudell, S. L. Lifer, Y. C. Tan, M. J. Martin, L. Deng, P. Skolnick and J. M. Cook, *J. Med. Chem.*, 1990, **33**, 2412–2420.

194 J. V. Schmidt and C. A. Bradfield, *Annu. Rev. Cell Dev. Biol.*, 1996, **12**, 55–89.

195 Y. Yamashita, N. Fujii, C. Murakata, T. Ashizawa, M. Okabe and H. Nakano, *Biochemistry*, 1992, **31**, 12069–12075.

196 T. Pawson and J. D. Scott, *Trends Biochem. Sci.*, 2005, **30**, 286–290.

197 L. Milanesi, M. Petrillo, L. Sepe, A. Boccia, N. D'Agostino, M. Passamano, S. Di Nardo, G. Tasco, R. Casadio and G. Paolella, *BMC Bioinf.*, 2005, **6**(suppl. 4), S20.

198 P. Cohen, *Nat. Rev. Drug Discovery*, 2002, **1**, 309–315.

199 D. Toullec, P. Pianetti, H. Coste, P. Bellevergue, T. Grand-Perret, M. Ajakane, V. Baudet, P. Boissin, E. Boursier, F. Loriolle, L. Duhamel, D. Charon and J. Kirilovsky, *J. Biol. Chem.*, 1991, **266**, 15771–15781.

200 D. R. Alessi, *FEBS Lett.*, 1997, **402**, 121–123.

201 S. P. Davies, H. Reddy, M. Caivano and P. Cohen, *Biochem. J.*, 2000, **351**, 95–105.

202 S. J. Coultrap, H. Sun, T. E. Tenner, Jr. and T. K. Machu, *J. Pharmacol. Exp. Ther.*, 1999, **290**, 76–82.

203 R. Lingameneni, T. N. Vysotskaya, D. S. Duch and H. C. Hemmings, Jr., *FEBS Lett.*, 2000, **473**, 265–268.

204 D. Thomas, B. C. Hammerling, A. B. Wimmer, K. Wu, E. Ficker, Y. A. Kuryshev, D. Scherer, J. Kiehn, H. A. Katus, W. Schoels and C. A. Karle, *Cardiovasc. Res.*, 2004, **64**, 467–476.

205 D. Brehmer, K. Godl, B. Zech, J. Wissing and H. Daub, *Mol. Cell Proteomics*, 2004, **3**, 490–500.

206 A. M. Lawrie, M. E. Noble, P. Tunnah, N. R. Brown, L. N. Johnson and J. A. Endicott, *Nat. Struct. Biol.*, 1997, **4**, 796–801.

207 L. Prade, R. A. Engh, A. Girod, V. Kinzel, R. Huber and D. Bossemeyer, *Structure (Cambridge, MA, U. S.)*, 1997, **5**, 1627–1637.

208 L. M. Toledo and N. B. Lydon, *Structure (Cambridge, MA, U. S.)*, 1997, **5**, 1551–1556.

209 M. B. Lamers, A. A. Antson, R. E. Hubbard, R. K. Scott and D. H. Williams, *J. Mol. Biol.*, 1999, **285**, 713–725.

210 X. Zhu, J. L. Kim, J. R. Newcomb, P. E. Rose, D. R. Stover, L. M. Toledo, H. Zhao and K. A. Morgenstern, *Structure (Cambridge, MA, U. S.)*, 1999, **7**, 651–661.

211 E. H. Walker, M. E. Pacold, O. Perisic, L. Stephens, P. T. Hawkins, M. P. Wymann and R. L. Williams, *Mol. Cell*, 2000, **6**, 909–919.

212 B. Zhao, M. J. Bower, P. J. McDevitt, H. Zhao, S. T. Davis, K. O. Johanson, S. M. Green, N. O. Concha and B. B. Zhou, *J. Biol. Chem.*, 2002, **277**, 46609–46615.

213 D. Komander, G. S. Kular, J. Bain, M. Elliott, D. R. Alessi and D. M. Van Aalten, *Biochem. J.*, 2003, **375**, 255–262.

214 A. Yamakawa, H. Ogata, K. Morita, N. Shibata, N. Andou, H. Sanuki, K. Yamada, T. Hioki, T. Ishii and Y. Higuchi, *Acta Crystallogr., Sect. D: Biol. Crystallogr.*, 2004, **60**, 764–766.

215 L. Jin, S. Pluskey, E. C. Petrella, S. M. Cantin, J. C. Gorga, M. J. Rynkiewicz, P. Pandey, J. E. Strickler, R. E. Babine, D. T. Weaver and K. J. Seidl, *J. Biol. Chem.*, 2004, **279**, 42818–42825.

216 Z. B. Xu, D. Chaudhary, S. Olland, S. Wolfrom, R. Czerwinski, K. Malakian, L. Lin, M. L. Stahl, D. Joseph-McCarthy, C. Benander, L. Fitz, R. Greco, W. S. Somers and L. Mosyak, *J. Biol. Chem.*, 2004, **279**, 50401–50409.

217 M. D. Jacobs, J. Black, O. Futer, L. Swenson, B. Hare, M. Fleming and K. Saxena, *J. Biol. Chem.*, 2005, **280**, 13728–13734.

218 L. N. Johnson, E. De Moliner, N. R. Brown, H. Song, D. Barford, J. A. Endicott and M. E. Noble, *Pharmacol. Ther.*, 2002, **93**, 113–124.

219 N. Schiering, S. Knapp, M. Marconi, M. M. Flocco, J. Cui, R. Perego, L. Rusconi and C. Cristiani, *Proc. Natl. Acad. Sci. U. S. A.*, 2003, **100**, 12654–12659.

220 T. J. Boggon, Y. Li, P. W. Manley and M. J. Eck, *Blood*, 2005, **106**, 996–1002.

221 R. S. Al-Awar, J. E. Ray, K. A. Hecker, S. Joseph, J. Huang, C. Shih, H. B. Brooks, C. D. Spencer, S. A. Watkins, R. M. Schultz, E. L. Considine, M. M. Faul, K. A. Sullivan, S. P. Kolis, M. A. Carr and F. Zhang, *Bioorg. Med. Chem. Lett.*, 2004, **14**, 3925–3928.

222 M. Gassel, C. B. Breitenlechner, N. König, R. Huber, R. A. Engh and D. Bossemeyer, *J. Biol. Chem.*, 2004, **279**, 23679–23690.

223 D. Komander, G. S. Kular, A. W. Schuttelkopf, M. Deak, K. R. Prakash, J. Bain, M. Elliott, M. Garrido-Franco, A. P. Kozikowski, D. R. Alessi and D. M. van Aalten, *Structure (Cambridge, MA, U. S.)*, 2004, **12**, 215–226.

224 A. N. Bullock, J. E. Debreczeni, O. Y. Fedorov, A. Nelson, B. D. Marsden and S. Knapp, *J. Med. Chem.*, 2005, **48**, 7604–7614.

225 M. E. Noble, J. A. Endicott and L. N. Johnson, *Science*, 2004, **303**, 1800–1805.

226 D. E. Gingrich, S. X. Yang, G. W. Gessner, T. S. Angeles and R. L. Hudkins, *J. Med. Chem.*, 2005, **48**, 3776–3783.

227 J. Kleinschroth, J. Hartenstein, C. Rudolph and C. Schachtele, *Bioorg. Med. Chem. Lett.*, 1995, **5**, 55–60.

228 H. C. Zhang, C. K. Derian, D. F. McComsey, K. B. White, H. Ye, L. R. Hecker, J. Li, M. F. Addo, D. Croll, A. J. Eckardt, C. E. Smith, Q. Li, W. M. Cheung, B. R. Conway, S. Emanuel, K. T. Demarest, P. Andrade-Gordon, B. P. Damiano and B. E. Maryanoff, *J. Med. Chem.*, 2005, **48**, 1725–1728.

229 C. Sanchez-Martinez, C. Shih, G. Zhu, T. Li, H. B. Brooks, B. K. Patel, R. M. Schultz, T. B. DeHahn, C. D. Spencer, S. A. Watkins, C. A. Ogg, E. Considine, J. A. Dempsey and F. Zhang, *Bioorg. Med. Chem. Lett.*, 2003, **13**, 3841–3846.

230 G. Zhu, S. E. Conner, X. Zhou, H. K. Chan, C. Shih, T. A. Engler, R. S. Al-Awar, H. B. Brooks, S. A. Watkins, C. D. Spencer, R. M. Schultz, J. A. Dempsey, E. L. Considine, B. R. Patel, C. A. Ogg, V. Vasudevan and M. L. Lytle, *Bioorg. Med. Chem. Lett.*, 2004, **14**, 3057–3061.

231 M. M. Faul, K. A. Sullivan, J. L. Grutsch, L. L. Winneroski, C. Shih, C. Sanchez-Martinez and J. T. Cooper, *Tetrahedron Lett.*, 2004, **45**, 1095–1098.

232 G. H. Kuo, C. Prouty, A. DeAngelis, L. Shen, D. J. O’Neill, C. Shah, P. J. Connolly, W. V. Murray, B. R. Conway, P. Cheung, L. Westover, J. Z. Xu, R. A. Look, K. T. Demarest, S. Emanuel, S. A. Middleton, L. Jolliffe, M. P. Beavers and X. Chen, *J. Med. Chem.*, 2003, **46**, 4021–4031.

233 D. E. Gingrich, D. R. Reddy, M. A. Iqbal, J. Singh, L. D. Aimone, T. S. Angeles, M. Albom, S. Yang, M. A. Ator, S. L. Meyer, C. Robinson, B. A. Ruggeri, C. A. Dionne, J. L. Vaught, J. P. Mallamo and R. L. Hudkins, *J. Med. Chem.*, 2003, **46**, 5375–5388.

234 C. Murakata, M. Kaneko, G. Gessner, T. S. Angeles, M. A. Ator, T. M. O’Kane, B. A. McKenna, B. A. Thomas, J. R. Mathiasen, M. S. Saporito, D. Bozyczko-Coyne and R. L. Hudkins, *Bioorg. Med. Chem. Lett.*, 2002, **12**, 147–150.

235 S. Fabre, M. Prudhomme, M. Sancelme and M. Rapp, *Bioorg. Med. Chem.*, 1994, **2**, 73–77.

236 S. Fabre, M. Prudhomme and M. Rapp, *Bioorg. Med. Chem.*, 1993, **1**, 193–196.

237 J. J. Champoux, *Annu. Rev. Biochem.*, 2001, **70**, 369–413.

238 Y. Pommier, *Biochimie*, 1998, **80**, 255–270.

239 J. A. Holden, *Curr. Med. Chem.: Anti-Cancer Agents*, 2001, **1**, 1–25.

240 L. H. Meng, Z. Y. Liao and Y. Pommier, *Curr. Top. Med. Chem.*, 2003, **3**, 305–320.

241 W. A. Denny, *Expert Opin. Emerging Drugs*, 2004, **9**, 105–133.

242 L. A. Mitscher, *Chem. Rev.*, 2005, **105**, 559–592.

243 C. Bailly, *DNA and RNA Binders: From Small Molecules to Drugs*, ed. M. Demeunynck, C. Bailly and W. D. E. Wilson, Wiley-VCH, Weinheim, 2003, vol. 2, pp. 538–575.

244 B. N. Balasubramanian, D. R. St Lauren, M. G. Saulnier, B. H. Long, C. Bachand, F. Beaulieu, W. Clarke, M. Deshpande, J. Eummer, C. R. Fairchild, D. B. Frennesson, R. Kramer, F. Y. Lee, M. Mahler, A. Martel, B. N. Naidu, W. C. Rose, J. Russell, E. Ruediger, C. Solomon, K. M. Stoffan, H. Wong, J. J. Wright, K. Zimmermann and D. M. Vyas, *J. Med. Chem.*, 2004, **47**, 1609–1612.

245 M. G. Saulnier, B. N. Balasubramanian, B. H. Long, D. B. Frennesson, E. Ruediger, K. Zimmermann, J. T. Eummer, D. R. St Lauren, K. M. Stoffan, B. N. Naidu, M. Mahler, F. Beaulieu, C. Bachand, F. Y. Lee, C. R. Fairchild, L. K. Stadnick, W. C. Rose, C. Solomon, H. Wong, A. Martel, J. J. Wright, R. Kramer, D. R. Langley and D. M. Vyas, *J. Med. Chem.*, 2005, **48**, 2258–2261.

246 P. Lassota, G. Singh and R. Kramer, *J. Biol. Chem.*, 1996, **271**, 26418–26423.

247 Y. Pommier, *Curr. Med. Chem.: Anti-Cancer Agents*, 2004, **4**, 429–434.

248 F. Kanzawa, K. Nishio, N. Kubota and N. Saijo, *Cancer Res.*, 1995, **55**, 2806–2813.

249 T. Yoshinari, M. Matsumoto, H. Arakawa, H. Okada, K. Noguchi, H. Suda, A. Okura and S. Nishimura, *Cancer Res.*, 1995, **55**, 1310–1315.

250 J. F. Goossens, J. P. Henichart, F. Anizon, M. Prudhomme, C. Dugave, J. Riou and C. Bailly, *Eur. J. Pharmacol.*, 2000, **389**, 141–146.

251 C. Bailly, in *Methods in Enzymology*, ed. J. B. Chaires and M. J. Waring, Academic Press, 2001, vol. 340 (Drug–nucleic acid interactions), pp. 610–623.

252 S. T. Taylor and R. Menzel, *Methods Mol. Biol.*, 2001, **95**, 195–204.

253 E. Labourier, J. F. Riou, M. Prudhomme, C. Carrasco, C. Bailly and J. Tazi, *Cancer Res.*, 1999, **59**, 52–55.

254 P. Moreau, F. Anizon, M. Sancelme, M. Prudhomme, C. Bailly, C. Carrasco, M. Ollier, D. Severe, J. F. Riou, D. Fabbro, T. Meyer and A. M. Aubertin, *J. Med. Chem.*, 1998, **41**, 1631–1640.

255 F. Anizon, P. Moreau, M. Sancelme, A. Voldoire, M. Prudhomme, M. Ollier, D. Severe, J. F. Riou, C. Bailly, D. Fabbro, T. Meyer and A. M. Aubertin, *Bioorg. Med. Chem.*, 1998, **6**, 1597–1604.

256 C. Bailly, L. Dassonneville, P. Colson, C. Houssier, K. Fukasawa, S. Nishimura and T. Yoshinari, *Cancer Res.*, 1999, **59**, 2853–2860.

257 C. Bailly, J. F. Riou, P. Colson, C. Houssier, E. Rodrigues-Pereira and M. Prudhomme, *Biochemistry*, 1997, **36**, 3917–3929.

258 C. Bailly, P. Colson, C. Houssier, E. Rodrigues-Pereira, M. Prudhomme and M. J. Waring, *Mol. Pharmacol.*, 1998, **53**, 77–87.

259 C. Bailly, X. Qu, F. Anizon, M. Prudhomme, J. F. Riou and J. B. Chaires, *Mol. Pharmacol.*, 1999, **55**, 377–3785.

260 C. Bailly, J. F. Goossens, W. Laine, F. Anizon, M. Prudhomme, J. Ren and J. B. Chaires, *J. Med. Chem.*, 2000, **43**, 4711–4720.

261 C. Bailly, X. Qu, D. E. Graves, M. Prudhomme and J. B. Chaires, *Chem. Biol.*, 1999, **6**, 277–286.

262 E. J. Gilbert, J. D. Chisholm and D. L. Van Vranken, *J. Org. Chem.*, 1999, **64**, 5670–5676.

263 A. Akao, S. Hiraga, T. Iida, A. Kamatani, M. Kawasaki, T. Mase, T. Nemoto, N. Satake, S. A. Weissman, D. M. Tschaen, K. Rossen, D. Petrillo, R. A. Reamer and R. P. Volante, *Tetrahedron*, 2001, **57**, 8917–8923.

264 M. Facompre, C. Carrasco, H. Vezin, J. D. Chisholm, J. C. Yoburn, D. L. Van Vranken and C. Bailly, *ChemBioChem*, 2003, **4**, 386–395.

265 C. Bailly, C. Carrasco, F. Hamy, H. Vezin, M. Prudhomme, A. Saleem and E. Rubin, *Biochemistry*, 1999, **38**, 8605–8611.

266 B. L. Staker, M. D. Feese, M. Cushman, Y. Pommier, D. Zembower, L. Stewart and A. B. Burgin, *J. Med. Chem.*, 2005, **48**, 2336–2345.

267 C. Marchand, S. Antony, K. W. Kohn, M. Cushman, A. Ioanoviciu, B. L. Staker, A. B. Burgin, L. Stewart and Y. Pommier, *Mol. Cancer Ther.*, 2006, **5**, 287–295.

268 J. Soret, M. Gabut, C. Dupon, G. Kohlhagen, J. Stevenin, Y. Pommier and J. Tazi, *Cancer Res.*, 2003, **63**, 8203–8211.

269 B. Pilch, E. Allemand, M. Facompre, C. Bailly, J. F. Riou, J. Soret and J. Tazi, *Cancer Res.*, 2001, **61**, 6876–6884.

270 B. Waters, G. Saxena, Y. Wanggui, D. Kau, S. Wrigley, R. Stokes and J. Davies, *J. Antibiot.*, 2002, **55**, 407–416.

271 C. Matz, P. Deines, J. Boenigk, H. Arndt, L. Eberl, S. Kjelleberg and K. Jurgens, *Appl. Environ. Microbiol.*, 2004, **70**, 1593–1599.

272 M. S. Butler, *Nat. Prod. Rep.*, 2005, **22**, 162–195.

273 ClinicalTrials.gov (a service of the US National Institutes of Health). Available at http://clinicaltrials.gov.

274 Pharmaceutical Research and Manufacturers of America. Available at http://www.phrma.org/new_medicines_in_development_for_cancer.

275 K. Grosios, *Curr. Opin. Investig. Drugs (Thomson Sci.)*, 2001, **2**, 287–297.

276 A. M. Senderowicz, *Oncologist*, 2002, **7(suppl. 3)**, 12–19.

277 E. Fuse, T. Kuwabara, A. Sparreboom, E. A. Sausville and W. D. Figg, *J. Clin. Pharmacol.*, 2005, **45**, 394–403.

278 E. A. Sausville, *J. Clin. Oncol.*, 2005, **23**, 1796–1798.

279 D. Fabbro, S. Ruetz, S. Bodis, M. Pruschy, K. Csermak, A. Man, P. Campochiaro, J. Wood, T. O’Reilly and T. Meyer, *Anti-Cancer Drug Des.*, 2000, **15**, 17–28.

280 E. Weisberg, C. Boulton, L. M. Kelly, P. Manley, D. Fabbro, T. Meyer, D. G. Gilliland and J. D. Griffin, *Cancer Cell*, 2002, **1**, 433–443.

281 A. Virchis, K. Ganesaguru, S. Hart, D. Jones, L. Fletcher, F. Wright, R. Wickremasinghe, A. Man, K. Csermak, T. Meyer, D. Fabbro, K. Champain, A. Yap, H. G. Prentice and A. Mehta, *Hematol. J.*, 2002, **3**, 131–136.

282 R. M. Stone, D. J. Deangelo, V. Klimek, I. Galinsky, E. Estey, S. D. Nimer, W. Grandin, D. Lebwohl, Y. Wang, P. Cohen, E. A. Fox, D. Neuberg, J. Clark, D. G. Gilliland and J. D. Griffin, *Blood*, 2005, **105**, 54–60.

283 P. A. Campochiaro and the C99-PKC412-003 Study Group, *Invest. Ophthalmol. Visual Sci.*, 2004, **45**, 922–931.

284 A. M. Camoratto, J. P. Jani, T. S. Angeles, A. C. Maroney, C. Y. Sanders, C. Murakata, N. T. Neff, J. L. Vaught, J. T. Isaacs and C. A. Dionne, *Int. J. Cancer*, 1997, **72**, 673–679.

285 B. A. Ruggeri, S. J. Miknyoczki, J. Singh and R. L. Hudkins, *Curr. Med. Chem.*, 1999, **6**, 845–857.

286 C. J. Strock, J. I. Park, D. M. Rosen, B. Ruggeri, S. R. Denmeade, D. W. Ball and B. D. Nelkin, *J. Clin. Endocrinol. Metab.*, 2006, **91**, 79–84.

287 C. J. Strock, J. I. Park, M. Rosen, C. Dionne, B. Ruggeri, S. Jones-Bolin, S. R. Denmeade, D. W. Ball and B. D. Nelkin, *Cancer Res.*, 2003, **63**, 5559–5563.

288 P. Brown, M. Levis, E. McIntyre, M. Griesemer and D. Small, *Leukemia*, 2006, **20**, 1368–1376.

289 B. D. Smith, M. Levis, M. Beran, F. Giles, H. Kantarjian, K. Berg, K. M. Murphy, T. Dauses, J. Allebach and D. Small, *Blood*, 2004, **103**, 3669–3676.

290 Further information available at http://www.cephalon.com.

291 S. D. Undevia, N. J. Vogelzang, A. M. Mauer, L. Janisch, S. Mani and M. J. Ratain, *Invest. New Drugs*, 2004, **22**, 449–458.

292 R. L. Hudkins, M. Iqbal, C. H. Park, J. Goldstein, J. L. Herman, E. Shek, C. Murakata and J. P. Mallamo, *Bioorg. Med. Chem. Lett.*, 1998, **8**, 1873–1876.

293 Parkinson Study Group, *Neurology*, 2004, **62**, 330–332.

294 T. H. Johnston and J. M. Brotchie, *Curr. Opin. Investig. Drugs*, 2006, **7**, 25–32.

295 G. D. Wheeler, *IDrugs*, 2003, **6**, 159–163.

296 S. V. Joy, A. C. Scates, S. Bearely, M. Dar, C. A. Taulien, J. A. Goebel and M. J. Cooney, *Ann. Pharmacother.*, 2005, **39**, 1693–1699.

297 K. R. Tuttle, G. L. Bakris, R. D. Toto, J. B. McGill, K. Hu and P. W. Anderson, *Diabetes Care*, 2005, **28**, 2686–2690.

298 A. I. Vinik, V. Bril, P. Kemplar, W. J. Litchy, S. Tesfaye, K. L. Price and E. J. Bastyr, 3rd, *Clin. Ther.*, 2005, **27**, 1164–1180.

299 L. P. Aiello, A. Clermont, V. Arora, M. D. Davis, M. J. Sheetz and S. E. Bursell, *Invest. Ophthalmol. Visual Sci.*, 2006, **47**, 86–92.

300 Further information available at http://www.lilly.com.

301 H. L. Pearce and M. Alice Miller, *Adv. Enzyme Regul.*, 2005, **45**, 229–255.

302 C. Querfeld, M. A. Rizvi, T. M. Kuzel, J. Guitart, A. Rademaker, S. S. Sabharwal, N. L. Krett and S. T. Rosen, *J. Invest. Dermatol.*, 2006, **126**, 1641–1647.

303 H. A. Fine, L. Kim, C. Royce, D. Draper, I. Haggarty, H. Ellinzano, P. Albert, P. Kinney, L. Musib and D. Thornton, *J. Clin. Oncol.*, 2005, **23(16S)**, 1504.

304 T. Kaneko, H. Wong, J. Utzig, J. Schurig and T. Doyle, *J. Antibiot.*, 1990, **43**, 125–127.

305 G. W. Rewcastle, *IDrugs*, 2005, **8**, 838–847.

306 B. H. Long and B. N. Balasubramanian, *Expert Opin. Ther. Pat.*, 2000, **10**, 635–666.

307 W. A. Denny, *IDrugs*, 2004, **7**, 173–177.

308 M. W. Saif and R. B. Diasio, *Clin. Colorectal Cancer*, 2005, **5**, 27–36.

309 S. Tanaka, M. Ohkubo, K. Kojiri, H. Suda, A. Yamada and D. Uemura, *J. Antibiot.*, 1992, **45**, 1797–1798.

310 T. Yoshinari, A. Yamada, D. Uemura, K. Nomura, H. Arakawa, K. Kojiri, E. Yoshida, H. Suda and A. Okura, *Cancer Res.*, 1993, **53**, 490–494.

311 H. Arakawa, T. Iguchi, T. Yoshinari, K. Kojiri, H. Suda and A. Okura, *Jpn. J. Cancer Res.*, 1993, **84**, 574–581.

312 H. Arakawa, T. Iguchi, M. Morita, T. Yoshinari, K. Kojiri, H. Suda, A. Okura and S. Nishimura, *Cancer Res.*, 1995, **55**, 1316–1320.

313 C. Bailly, X. Qu, J. B. Chaires, P. Colson, C. Houssier, M. Ohkubo, S. Nishimura and T. Yoshinari, *J. Med. Chem.*, 1999, **42**, 2927–2935.

314 M. Ohkubo, T. Nishimura, T. Honma, I. Nishimura, S. Ito, T. Yoshinari, H. Arakawa, H. Suda, H. Morishima and S. Nishimura, *Bioorg. Med. Chem. Lett.*, 1999, **9**, 3307–3312.

315 T. Yoshinari, M. Ohkubo, K. Fukasawa, S. Egashira, Y. Hara, M. Matsumoto, K. Nakai, H. Arakawa, H. Morishima and S. Nishimura, *Cancer Res.*, 1999, **59**, 4271–4275.

316 J. F. Goossens, J. Kluzá, H. Vezin, M. Kouach, G. Briand, B. Baldeyrou, N. Wattez and C. Bailly, *Biochem. Pharmacol.*, 2003, **65**, 25–34.

317 A. R. Howard-Jones and C. T. Walsh, *J. Am. Chem. Soc.*, 2006, **128**, 12289–12298.

318 E. Yeh, L. J. Cole, E. W. Barr, J. M. Bollinger Jr, D. P. Ballou and C. T. Walsh, *Biochemistry*, 2006, **45**, 7904–7912.
