
VOLUME 71, NUMBER 26                                                                                       DECEMBER 22, 2006

JOC Perspective

© Copyright 2006 by the American Chemical Society

Rings, Radicals, and Regeneration:  
The Early Years of a Bioorganic Laboratory  

Wilfred A. van der Donk  

Roger Adams Laboratory, Department of Chemistry, University of Illinois at Urbana-Champaign,  
600 South Mathews Avenue, Urbana, Illinois 61801  

vddonk@uiuc.edu  

Received July 8, 2006  

This Perspective provides an overview of the progress in two of the original programs in my research group focused on the biosynthesis of the antibiotics nisin, lacticin 481, fosfomycin, and bialaphos. The path from start-up funds to tenure and beyond offers insights into the opportunities realized and missed along the road.

A Perspective on the research in my laboratory provided an opportunity not only to reflect on the advances and progress made in our program but also to recognize missed chances. In this Perspective, I describe the often bumpy and certainly nonlinear path from start-up to tenure and beyond in the nine years since our research group was established at the University of Illinois. Hopefully, this journey will provide insights into the choices made at critical junctures along the road as well as into the serendipitous and unpredictable nature of research at the interface of organic chemistry and biology.

With a Ph.D. in organic chemistry and a subsequent postdoc that introduced me to enzymology, the central theme of my research proposals written in 1996/97 was to use the synthetic tools of organic chemistry to explore novel biochemical transformations in antibiotic biosynthesis and vitamin B₁₂ dependent reactions. At the time, the work of Khosla on polyketide biosynthesis¹ was instrumental in shaping my interest in natural product biosynthesis, which was further cemented by a paper from the Walsh group² on the post-translational maturation of microcin B17. With the advent of increasingly powerful molecular biology techniques and the first sequencing of whole bacterial genomes, it seemed that a wealth of novel chemistry and enzymology imbedded in natural products biosynthesis would soon be accessible to organic chemists. Since the fields of polyketide and nonribosomal peptide biosynthesis already appeared well populated by scientists applying chemical approaches, our laboratory focused on two classes of antibiotics that were relatively unexplored but yet of significant commercial and therapeutic importance, the lantibiotics and phosphonate antibiotics (Figure 1).

The choice of these systems for investigation was influenced by a number of factors. As described in more detail below, at the time good evidence existed that lantibiotic synthetases would have flexible substrate specificity, which we intended to explore by altering the natural substrates using organic synthesis. Our interest in phosphonate antibiotics was provoked by two unusual transformations that had been proposed in the literature for the biosynthesis of fosfomycin and phosphinthricin (Figure 1). The methyl groups highlighted in green had been reported to be derived from methylcobalamin.³,⁴ᵃ To date the natural vitamin B₁₂ derivatives adenosylcobalamin (AdoCbl) and methylcobalamin (MeCbl)⁵ are still the only confirmed compounds in nature containing a metal–carbon bond (Figure 2) and as such were intriguing to me based on my graduate studies in organometallic chemistry.⁶,⁷

After a difficult start, we made much progress in the past two years in the project on lantibiotics biosynthesis, whereas research on the phosphonate antibiotics is still in a more preliminary stage. This perspective will present our current understanding of the biosynthetic pathways of these two classes of compounds.

### Lantibiotic Biosynthesis

In the past 15 years, the biosynthesis of peptide natural products, whether gene encoded or of nonribosomal origin, has received much attention. Of special interest to us were the lantibiotics, which are extensively post-translationally modified. For some members, as many as 47% of the amino acids in the peptide undergo some form of modification.⁸ The unifying structural motifs, which also have given lantibiotics their name,⁹ are the lanthionine and methyllanthionine amino acids highlighted in red and blue, respectively, in Figure 1 for two representative members of the class I and class II lantibiotics,

10.1021/jo0614240 CCC: $33.50 © 2006 American Chemical Society  
Published on Web 11/08/2006                                                                                      J. Org. Chem. 2006, 71, 9561–9571 9561

JOC Perspective

intramolecular and stereoselective Michael-type addition of the thiols of cysteines to Dha or Dhb to produce lanthionine (Lan) or methyllanthionine (MeLan) thioether cross-links, respectively. The stereochemistry at the α-carbon of the former Ser/Thr residue is inverted to the D-configuration in this process. In a final step of maturation a protease (LanP) removes a leader peptide that has not been modified (Figure 3).

The best known lantibiotic family member is nisin, first recognized as an antimicrobial substance in 1928<sup>10</sup> but not assigned a structure until 1971.<sup>11</sup> Nisin has been used extensively in the food industry as a safe and highly effective preservative with minimal inhibitory concentrations in the low nanomolar range against many Gram-positive bacteria including food-borne pathogens such as *Clostridium botulinum* and *Listeria monocytogenes*<sup>8,12</sup> and multidrug-resistant bacteria.<sup>13</sup> Our interest in nisin and other lantibiotics stemmed from several in vivo site-directed mutagenesis experiments that suggested that the biosynthetic machinery was very forgiving with respect to the structure of its substrate.<sup>14</sup> Whereas site-directed mutagenesis at the time could only replace an amino acid with one of the other 20 proteinogenic amino acids, we envisioned that a combination of organic synthesis and solid-phase peptide synthesis with the biosynthetic enzymes could greatly increase the accessible structural and functional space to explore nisin analogues. This premise required us to obtain active biosynthetic enzymes, but that process proved extremely challenging. Although we ultimately succeeded,<sup>15,16</sup> our first success came after five years of unsuccessful and mostly unpublished attempts. The major culprits in our eyes were the dehydratase enzymes. We expressed and purified these proteins from *E. coli* or performed activity assays with crude cell preparations of lantibiotic producing bacterial strains without even once observing dehydration of the peptide substrates.<sup>17</sup> Without dehydration we had no substrate for the cyclase enzymes NisC (for nisin production) and SpaC (for production of subtilin, a close relative of nisin). At this time of our struggles, a high profile report showed that nisin was not only a pore former, as was generally accepted, but also interacted in a highly specific manner with the cell wall biosynthetic precursor lipid II,<sup>18</sup> which interestingly is also the target of vancomycin and ramoplanin.<sup>19,20</sup> By binding to lipid II, nisin inhibits cell wall biosynthesis, in itself a lethal activity, but it packs an even more powerful punch by assembling into a higher order aggregate consisting of four lipid II molecules and eight nisin molecules that form extremely stable pores in the cell membrane of target bacteria.<sup>19,21</sup> Although not directly relevant to our efforts on nisin biosynthesis, these findings strengthened our resolve to try to prepare nisin in vitro as the prospect of analogues was now even more appealing.

Since we had been unable to obtain the substrates for the cyclase enzymes through the enzymatic action of the dehydratases, we turned to organic synthesis. Our targets were the dehydrated prepeptide intermediates of nisin (box, Figure 3) and subtilin biosynthesis. Although a myriad of routes had been reported for preparation of Dha and Dhb residues, none of them were compatible with either established solid-phase peptide synthesis (SPPS) protocols or with the amino acids present in our targets.<sup>22</sup> We therefore developed new synthetic methodology that was suitable for our needs. The monomer 1, appropriately protected for Fmoc-based SPPS, was incorporated efficiently into peptides and after treatment with oxidizing agents (H₂O₂ or NaIO₄) afforded Dha-containing peptides (eg Scheme 1). Importantly, this methodology proved compatible with

nisin and lacticin 481. For class I, these structures are introduced by dehydration of Ser and Thr residues in a ribosomally produced prepeptide to the corresponding dehydroalanine (Dha, green Figure 3) and Z-dehydrobutyrine (Dhb, magenta) residues by a dehydratase LanB (Lan is used as a generic label for lantibiotic biosynthetic proteins; for instance, NisB is the LanB for nisin). This step is then followed by a LanC-catalyzed

FIGURE 1. Structures of the lantibiotics nisin A and lacticin 481, the phosphonate antibiotic fosfomycin, and the phosphinate phosphinothricin. Lanthionine (Lan) residues in the lantibiotics are shown in red and methyllanthionine (MeLan) residues in blue. The two methyl groups in fosfomycin and phosphinothricin that are considered to be derived from methylcobalamin are highlighted in green.

FIGURE 2. Structures of vitamin B₁₂ (X = cyanide), and the natural cofactors methylcobalamin (X = CH₃), and adenosylcobalamin (X = 5′-deoxyadenosine).

NisA

Leader-ITSISLCTPGCKTGALMGCNMKTATCHCSIHVSK

- 8 H₂O ← NisB (dehydratase)

Leader-IDhbDhaLDhaLC Dhb PGCK Dhb GALMGCNMK DhaADhb CHCSIHV DhaK

NisC (cyclase)

Leader-IDhbAlaIDhaL AlaAbu PGA laK Abu GALMG Ala NMK AbuA AbuAla HAla SIHV DhaK

Proteolysis

Nisin A

FIGURE 3. Post-translational maturation process of nisin as an example of a class I lantibiotic. The prepeptide NisA is ribosomally synthesized, followed by NisB-catalyzed dehydration of underlined Ser and Thr residues of NisA. NisC catalyzes the conjugate addition of Cys residues in a regio- and stereoselective manner to five of the Dha (green) and Dhb (magenta) residues to generate five cyclic thioethers: one lanthionine (red) and four methyllanthionines (blue). After dehydration/cyclization is complete, the unmodified leader peptide is proteolytically removed by the protease NisP. The sequence of the leader peptide is MSTKDFNLDLVSVSKKDSGASPR. Abu, S-2-aminobutyric acid.

unprotected side chains of the common amino acids, and hence, a range of different dehydropeptides could be prepared.²² An interesting modification of this strategy was recently used by the Szostak laboratory for ribosomal incorporation of 4-selenalysine (Scheme 1), which upon treatment with oxidizing agents also resulted in the formation of Dha.²³

The oxidative elimination of selenoxides is a stereospecific process, and therefore, use of monomer 2 provided convenient and stereospecific entry to Z-Dhb-containing peptides.²⁴,²⁵ With these tools in hand, we chemically prepared dehydropeptides as substrates for heterologously expressed and purified nisin and subtilin cyclases (NisC and SpaC). However, our attempts to distinguish enzymatic cyclization from spontaneous nonenzymatic cyclization of these peptides remained inconclusive.²⁶ Although disappointing at the time of these experiments, several years later we would show that the idea of using dehydrated peptides as substrates for NisC expressed in *E. coli* was sound but that the full length dehydropeptide is needed for a sensitive assay (see below).¹⁶

Although we did not succeed in applying our methodology of dehydropeptide synthesis to investigation of the biosynthetic enzymes, the difficulties we encountered in our efforts to distinguish enzymatic from nonenzymatic cyclization led us to study the latter process in more detail. Thus, Dha- and Dhb-containing peptides were prepared that also incorporated disulfide-protected Cys residues (Scheme 1). Upon release of the free Cys residues by using the reducing agent tris(carboxyethyl)-phosphate (TCEP), cyclization indeed occurred efficiently, and interestingly resulted in the stereoselective production of the naturally occurring diastereomers as the major and often only products (Scheme 1).²²,²⁴ This selectivity obtained in aqueous solution was quite unexpected as the protonation of the enolate intermediate was anticipated to be very fast and thus rather nonselective.

Mechanistic studies on these nonenzymatic cyclizations subsequently showed the selectivity to be due to kinetic and not thermodynamic control.²⁵ At the time, there was no conclusive evidence that the LanC proteins indeed were cyclases, which was assumed because disruption of LanC genes resulted in accumulation of dehydrated peptides.⁸,²⁷,²⁸ We therefore were curious as to whether the apparent inherent propensity of these short dehydropeptides to provide the same stereoselectivity as observed in the lantibiotics could be extended to the regioselectivity and chemoselectivity of cyclization. As a first test of this question, the dehydropeptide 3 was prepared using the methodology of Scheme 1. This peptide corresponds to the precursor to the A and B rings of nisin, the segment that interacts with lipid II.²⁹ When the dehydropeptide 3 was allowed to cyclize nonenzymatically, the inherently more reactive Dha residues were preferentially attacked by the Cys residues in the peptide; whereas the A ring was still formed, the B-ring (dashed arrow) was not (Scheme 2).²⁵ These results provided convincing

JOC Perspective

SCHEME 1. Preparation of Dehydropeptides by Oxidative Elimination

![Diagram 1]

LeuDhaProGlyCys(Trt)ValGly [LeuDhaAlaAsnCysLysIle]

| 1. TFA | 1. TCEP |
| --- | --- |
| 2. pH 8 | 2. pH 8 |
| 56% for 2 steps | 75% for 2 steps |

FmocHN$_{i}$COH  
SePh  
2  

1. SPPS  
2. H$_{2}$O$_{2}$  

Ala$_{i}$NH$_{i}$COH  
SePh  

4-selena-  
lysine  

1. TCEP  
2. pH 8  

additional evidence that a cyclase enzyme is needed to overcome the chemoselectivity of the nonenzymatic reaction and ensure the correct connectivity of the lanthionine and methyllanthionine rings.

A second project based on this methodology was the preparation of dehydropeptides for chemoselective ligations. The past decade has seen a resurgence in methods to achieve conjugations of two biomolecules or of a biomolecule with a biophysical probe or small molecule. By introducing a dehydroalanine into a peptide, a site-specific conjugation point is created for selective intermolecular reaction with nucleophiles. Indeed, we showed that prenyl thiols and complex carbohydrates containing a thiol functionality could be efficiently ligated to dehydropeptides by this approach (e.g., Scheme 3). A drawback to the methodology was that the diastereoselectivities of the protonation at the α-carbon were modest at best, in contrast to the intramolecular cyclizations in Scheme 1. To overcome this problem, a collaborative project with David Gin's laboratory was initiated to develop an efficient solid-phase synthesis of aziridine containing peptides in which the stereo-

SCHEME 2. Attempted Biomimetic Synthesis of the A- and B-Rings of Nisin

![Diagram 2]

1. P(CH$_{2}$CH$_{2}$CO$_{2}$H)$_{3}$  
2. pH 8  

B-ring  

SCHEME 3. Conjugate Addition of External Thiol Nucleophiles to Dha$^{32}$

![Diagram 3]

RSH  
75%  
dr = 1.2:1  

center at the α-carbon would be set. Subsequent ring opening of these aziridines with external nucleophiles allowed efficient ligation with a series of nucleophiles producing the desired bioconjugates with defined stereochemistry. Of particular interest are the ligations of thiol containing oligosaccharides in which both the stereochemistry at the anomeric center and at the α-carbon of the former aziridine are controlled (Scheme 4).

By the end of 2002, our lantibiotic biosynthesis project appeared destined for disappointing termination. Although our biomimetic and chemoselective ligation chemistry was fruitful, our initial goals of preparing lantibiotics in vitro using the biosynthetic machinery appeared unattainable. Focus within the group, which from 1997 to 2000 had been almost entirely on lantibiotics, was now diverted to two new projects on the mechanisms of phosphate dehydrogenase and prostaglandin synthase (also called cyclooxygenase or COX).

SCHEME 4. Stereoselective Synthesis of Thioglycopeptides<sup>33,34</sup>


HAla—○
     |
SPPS
     |
AcLys(Boc)Ser<sup>tBu</sup>Glu<sup>tBu</sup>GlyPhe—N—C—Ala—○
                |   |
                AcO OAc
                |   |
                AcO O
                |   |
                ACHN SH


AcLysSerGluGlyPheCys(GalNAcα1)AlaOH

19%, 14 steps

---

SCHEME 5. NAD<sup>+</sup>-Dependent Oxidation of Phosphite by PTDH


B:
    H—O—H
        |
        O—P—O—H<sup>a</sup>
        |   |
        AdO—P—O—P—O—
        |   |   |
        OH OH OH
        |
        phosphite
        dehydrogenase
        (PTDH)
        |
        HO—P—O—
        |   |
        AdO—P—O—P—O—
        |   |   |
        OH OH OH
        |
        NADH


dehydrogenase, discovered by Bill Metcalf in the Department of Microbiology at UIUC,<sup>36</sup> catalyzes the oxidation of phosphite to phosphate concomitant with the reduction of NAD<sup>+</sup> to NADH (Scheme 5).

We became initially interested in the enzyme because it promotes a highly unusual hydride transfer reaction.<sup>37</sup> Although catalyzed by a dehydrogenase, the reaction is formally a phosphoryl transfer from hydride to hydroxide. From a mechanistic perspective associative, dissociative (S<sub>N</sub>1) and concerted (S<sub>N</sub>2) mechanisms all appear to be improbable for this transformation. However, analysis of the solution thermodynamics of the transformation shows it to be 15 kcal/mol exergonic.<sup>37</sup> At present, we do not know specifics of the mechanism of this hydride transfer and discussion of our mechanistic studies are beyond the scope of this perspective,<sup>38</sup> but one practical application will be highlighted. It occurred to us that if this reaction was indeed exergonic, then phosphite dehydrogenase should be a good NADH regeneration enzyme that might be

---

SCHEME 6. Use of PTDH for Cofactor Regeneration<sup>39</sup>


R<sup>1</sup>C=OR<sup>2</sup> ——[oxidoreductase]—— R<sup>1</sup>CHOH<sub>R<sup>2</sup></sub>
                          |
                          NADH
                          |
                          PTDH
                          |
                          HO—P—O—
                          |   |
                          AdO—P—O—
                          |   |
                          OH OH
                          |
                          NADP+


useful in biocatalysis. We have been able to turn this idea into practice (Scheme 6),<sup>39</sup> and a subsequent joint investigation with Huimin Zhao’s laboratory employing rational design resulted in an engineered variant that uses both NAD<sup>+</sup> and NADP<sup>+</sup> effectively.<sup>40</sup> We then utilized directed evolution techniques to improve on its expression and activity,<sup>41</sup> and this latter mutant was further engineered in the Zhao laboratory to obtain a thermostable variant<sup>42</sup> that rivals or outperforms other enzymatic NAD(P)H regeneration systems, at least on a laboratory scale. Hence, what attracted us initially to this enzyme was purely mechanistic curiosity, but detailed analysis of the reaction ultimately resulted in an engineered analogue with great practical potential.

In the spring of 2003, one student was still working on the lantibiotic biosynthesis project with other students having switched to the phosphite dehydrogenase and ligation projects discussed in the previous section. In a last attempt to achieve in vitro lantibiotic biosynthesis, we had turned to the LanM proteins that combine the dehydratase and cyclase activities in one bifunctional enzyme in the biosynthesis of class II lantibiotics.<sup>8,43</sup> One such protein, LctM involved in the biosynthesis of lacticin 481, was heterologously overexpressed in *E. coli* and purified. The enzyme and its substrate peptide LctA were incubated with DTT, Zn<sup>2+</sup>, ATP, and Mg<sup>2+</sup>, and a MALDI-MS was recorded. For almost 5 years, we had always seen unmodified peptide no matter the ingredients of the assays. But this time, the assay product corresponded to a 4-fold dehydrated peptide (Figure 4)!<sup>15</sup> The subsequent determination of the location of those dehydrations and whether the cyclizations had also occurred was our long sought, but challenging, task. Proteolytic removal of the leader peptide showed that the four dehydrations had taken place in the propeptide region of the substrate peptide and not in the leader peptide. Moreover, individual mutagenesis of the Ser/Thr residues as well as Fourier transform MS/MS analysis in the laboratory of my colleague Neil Kelleher demonstrated that the Ser/Thr that were dehydrated corresponded with the anticipated sites of post-translational modification for lacticin 481 biosynthesis. Furthermore, the product after removal of the leader peptide showed the expected antimicrobial activity.<sup>15</sup> Thus, after more than 5 years of unsuccessful attempts, in vitro biosynthesis of a lantibiotic was finally achieved.<sup>44</sup>

With an active system in hand, the many questions that had initially attracted us to the lantibiotics could finally be investigated.<sup>46</sup> LctM does not have homology with proteins of known function except for enzymes involved in the biosynthesis of other lantibiotics. Its C-terminal domain has homology with the LanC proteins suggesting that this domain is responsible for the cyclization. However, the N-terminal domain of LctM does not have homology with the LanB dehydratases utilized in class I lantibiotic biosynthesis but is nevertheless still expected to

JOC Perspective

leader peptide—KGGSGVIHTSHECNMNSWQFVFTCCS LctA
propeptide
-4 H₂O
LctM
leader peptide—KGGSGVIHDhbIDhaHECNMNDhaWQFVFDhbCCS
↓
LctM
leader peptide—KGGSGVIHAbuAlaHEAlaNMNAlaWQFVFDhbAlaAlaS
↓ protease
lacticin 481
Gly-Lys
Ser
Gly
Val-Ile-His
Ile
D
NH
O
His
Gl
u
Met-Asn
Asn
O
Trp-Gln-Phe-Val-Phe
Ser
O
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
C
NH
O
NH
JOC Perspective

SCHEME 8. Several Members of the Phosphonate/Phosphinate Class of Natural Products$^{a}$

![Image of fosfomycin]

fosfomycin

![Image of fosmidomycin]

fosmidomycin

![Image of PTT or bialaphos]

PTT or bialaphos

![Image of phosphothrixin]

phosphothrixin

$^{a}$ The phosphothricin moiety in bialaphos is highlighted in red.

SCHEME 9. Proposed Biosynthetic Pathways of Fosfomycin and Bialaphos$^{3e}$

![Image of Scheme 9]

their biological action as mimics of either carboxylic acids or phosphate esters. Fosfomycin is currently clinically used under the name Monurol for the treatment of urinary tract infections.$^{59}$ It is produced by *Streptomyces wedmorensis*, *Streptomyces fradiae*, and several *Pseudomonas* species$^{3e,60}$ and inactivates UDP-N-acetylglucosamine-3-O-enolpyruvyltransferase (MurA), an essential enzyme in cell wall biosynthesis.$^{61}$ Phosphothricin (PT) is an intermediate in the biogenesis of bialaphos, which is commercially used as a herbicide under the name BASTA. Various formulations of phosphothricin are widely used in agriculture, with annual sales in excess of $200 million per year. Seminal pioneering studies by Seto and co-workers using genetic techniques complemented by feeding studies with isotopically labeled precursors suggested the biosynthetic pathway for fosfomycin shown in Scheme 9.$^{3e}$ Using stereoselectively labeled precursors, Hammerschmidt and co-workers worked out the stereochemistry of the overall process.$^{4}$ The methylation step that both groups proposed drew our initial interest. The methyl group in 2-hydroxypropylphosphonic acid (HPP) was shown by labeling studies to be derived from

FIGURE 5. Various post-translational modifications of proteinogenic amino acids found in lantibiotic family members.$^{8}$ The stereochemistry of the dihydroxyproline is currently unknown.$^{57}$ Thioviramide contains an aminovinylcysteine suggesting it may be a lantibiotic, but it has not yet been shown to be gene encoded. It also contains the thioamides shown in the figure.$^{58}$

analogues.$^{56}$ These biologically advantageous properties may therefore provide a framework for the design of a range of therapeutic peptides and lantibiotics. Certainly, the high tolerance of various lantibiotic modification enzymes suggests they are a class with much promise for biocatalysis. Coupled with the many other unique post-translational modifications found in lantibiotics (Figure 5), the prospects for diversity oriented biosynthesis are bright.

### Phosphonate Natural Products Biosynthesis

Despite their ubiquitous use in antibacterial, antiviral, and antiparasitic therapies, and in agriculture as herbicides and pesticides, biosynthetic pathways for P–C bond-containing natural products are relatively unexplored. A few of these compounds are shown in Scheme 8. Phosphonates and phosphinates are effective antibacterial and antifungal agents because the P–C bond is resistant to acid- and base-catalyzed hydrolysis, as well as phosphotransferase-catalyzed hydrolysis. They exert

JOC Perspective

A. "Conventional" methyltransfer mechanism

Nuc-Me

Nuc:

     Me
    / \
   Co  L


B. Proposed Mechanism


     R
    / \
   H   Me
    \ /
     Co
      |
      L


SCHEME 10. Possible Heterolytic Mechanism of P-Methyltransferase (Pmet).


R
|
C=O
|
NHAc
|
CH2
|
OH
|
P
|
OH
|
Me
|
Co(I)


R = Ala-Ala


R
|
C=O
|
NHAc
|
CH2
|
P+
|
OH
|
Me
|
Co(I)


H+


R
|
C=O
|
NHAc
|
CH2
|
P=O
|
HO
|
Me
|
N-AcPT


FIGURE 6. Conventional<sup>5</sup> and unusual mechanisms<sup>3e,4a</sup> of B<sub>12</sub>-dependent methyl transfer. The corrin ligand ring system (Figure 2) is schematically represented by a square. L may be the benzimidazole ligand of B<sub>12</sub> or an amino acid ligand from the protein.

methylcobalamin (MeCbl, Figure 2).<sup>3a</sup> What makes this methyl transfer very unusual is that it was proposed to take place as an anion (CH<sub>3</sub><sup>-</sup>) from MeCbl to an electrophile, the aldehyde of PnAA (Scheme 9).<sup>3e,4a</sup> This proposal is fundamentally different from the accepted mechanisms for all MeCbl-dependent methyltransferases in which methylation takes place via nucleophilic attack by the substrate at the methyl group of MeCbl (Figure 6).<sup>5</sup> This then produces the highly nucleophilic Co(I) form of the cofactor, which can accept a methyl cation equivalent from a cellular methyl donor like S-adenosylmethionine (AdoMet) or methyl tetrahydrofolate (CH<sub>3</sub>-H<sub>4</sub>folate).

Seto and co-workers also studied the biosynthesis of the phosphinate bialaphos (Scheme 9) and showed that its methyl group attached to phosphorus is likely also derived from MeCbl.<sup>3b,d</sup> The high degree of homology between the two proteins catalyzing the methyl transfer steps, Pmet and Fom3, suggests they catalyze their reactions using similar mechanisms. The substrate for Fom3 has a very different chemical reactivity, however, from the substrate for the P-methyltransferase. The latter, through its thermodynamically unfavored tricoordinate tautomer 4, could act as a nucleophile according to the conventional mechanism of methyl transfer from MeCbl (Scheme 10). However, umpolung with thiamin pyrophosphate of the electrophilic aldehyde group of the PnAA substrate has been ruled out for Fom3 by the labeling studies of Hammerschmidt.<sup>4b</sup>

Intrigued by the apparent mechanistic paradox in how these enzymes would utilize B<sub>12</sub> for their methyl-transfer reactions, the biosynthesis of fosfomycin and bialaphos was the focus of one of my initial research proposals in 1996. The genes for both enzymes contain the sequence CysX<sub>3</sub>CysX<sub>2</sub>Cys. In postdoctoral work, I had encountered this sequence before, which at the time was known in two enzymes, the anaerobic ribonucleotide reductase and pyruvate formate lyase.<sup>62</sup> In both enzymes, the cysteines are involved in an iron-sulfur cluster that was proposed to homolytically cleave the carbon sulfur bond of S-adenosylmethionine (SAM) to generate an adenosyl radical. This radical then abstracts a hydrogen atom from the substrate. From this information, we attempted to write a unifying mechanism for the two methyl-transfer reactions. For the Pmet transformation, the 5'-deoxyadenosyl radical 5 could abstract a hydrogen atom from the phosphinate breaking the relatively weak P-H bond (Scheme 11).

SCHEME 11. Possible Homolytic Mechanism of P-Methyltransferase (Pmet)


[Fe<sub>4</sub>S<sub>4</sub>]<sup>1+</sup>
        |
        v
NH<sub>3</sub>
|
COO-
|
CH<sub>2</sub>
|
S<sup>+</sup>
|
Me
|
CH<sub>2</sub>
|
O
|
A
|
HO
|
CH<sub>2</sub>
|
OH
|
AdoMet



[Fe<sub>4</sub>S<sub>4</sub>]<sup>2+</sup>
        |
        v
NH<sub>3</sub>
|
COO-
|
CH<sub>2</sub>
|
S-Me
|
[Fe<sub>4</sub>S<sub>4</sub>]<sup>2+</sup>



AcHN
|
R
|
O
|
P=O
|
H
|
HO
|
CH<sub>2</sub>
|
O
|
A
|
HO
|
CH<sub>2</sub>
|
OH
|
5



[Fe<sub>4</sub>S<sub>4</sub>]<sup>2+</sup>
        |
        v
Me
|
P-OH
|
CH<sub>2</sub>
|
NHAc
|
R
|
O
|
N-AcPTT


This step is supported by radical-chain reactions observed during addition of phosphinate radicals to alkenes in which the propagation step consists of hydrogen abstraction from the phosphinic acid by a carbon-based radical.<sup>63</sup> The resulting phosphorus-based radical could then potentially attack MeCbl to cleave the Co-C bond homolytically. Admittedly, this mechanism was (and is) highly speculative, but since we first formulated it, a number of reports in the literature provide support. In 1999, Sofia et al. published a highly influential paper in which the authors showed that the CysX<sub>3</sub>CysX<sub>2</sub>Cys motif is widespread in nature, and they termed this family the radical-SAM superfamily.<sup>64</sup> Indeed, many radical-SAM proteins have now been studied, and they all use an iron-sulfur cluster for the formation of the 5'-deoxyadenosyl radical that initiates catalysis of a wide range of transformations.<sup>65</sup> P-Methyltransferase and PnAA methyltransferase promote very different reactions than other confirmed radical-SAM proteins, but modern bioinformatics tools<sup>66</sup> predict both a B<sub>12</sub> and a SAM binding domain such that their inclusion in the family, although not proven, is likely. Indeed, Sofia et al. proposed a number of putative methyl transferases in the superfamily with the role of

9568 J. Org. Chem., Vol. 71, No. 26, 2006

SCHEME 12. Revised Biosynthetic Pathway of Fosfomycin

![Chemical Structure](https://i.imgur.com/1234567.png)

CloN6 involved in the biosynthesis of clorobiocin recently confirmed by gene inactivation.⁶⁷ In other developments supporting the model in Scheme 11, Kräutler and co-workers showed that carbon based free radicals react with MeCbl to give one-carbon homologated products (eq 1).⁶⁸

![Chemical Reaction](https://i.imgur.com/8901234.png)

However, the mechanism in Scheme 11 still poses problems for the reaction catalyzed by Fom3 because reaction of an acyl radical, if formed by abstraction of the hydrogen labeled H² in PnAA (Scheme 9), with MeCbl would produce a ketone, not an alcohol, and would be inconsistent with Hammerschmidt's labeling studies.⁴ᵇ Would Fom3 then indeed catalyze a methyl anion transfer or was there perhaps something missing in the proposed pathway?

Like our project on lantibiotics, our studies of the biosynthesis of fosfomycin and bialaphos initially met with many problems. Heterologously expressed proteins from *S. wedmorensis* were insoluble, and fusion proteins that were soluble were not active. Having learned from our experience with the lantibiotics that it is often advantageous to move to a different but related system rather than spend too much time on an initially intractable system, we decided to clone the fosfomycin biosynthetic cluster of *S. fradiae*.⁶⁹ Expression of the cluster in *S. lividans* resulted in the production of fosfomycin, the first successful example of heterologous expression. Gene inactivation analysis demonstrated that the gene *fomC* is absolutely essential to support fosfomycin production in *S. lividans*,⁷⁰ but this gene has no role in the previously proposed pathway in Scheme 9. FomC is predicted to be an iron-dependent dehydrogenase and we wondered whether perhaps PnAA must first be reduced to the corresponding alcohol before Fom3 carries out the methyl-transfer reaction (Scheme 12). Consistent with this hypothesis, fosfomycin production could be restored in the inactivated *fomC* mutant by adding exogenous hydroxyethyl phosphonate (HEP) to the growth medium. Collectively, these results strongly argue that HEP is a biosynthetic intermediate and suggests that it may be the true substrate for Fom3 (Scheme 12). Our revised pathway for fosfomycin biosynthesis is fully consistent with the studies of the Seto and Hammerschmidt groups. Both laboratories had previously shown that HEP can be converted to fosfomycin by wild-type producing organisms.³ᶠ,⁴ However, without compelling evidence HEP was considered to be an off-pathway intermediate. Recent cloning of the bialaphos biosynthetic cluster⁷¹ revealed an iron-dependent dehydrogenase with high sequence identity with FomC. Since PnAA is the last confirmed common intermediate in the biosynthetic pathways of these two compounds, it is likely that both dehydrogenases act on PnAA to give HEP, after which the two pathways diverge.

Hammerschmidt’s labeling studies are also consistent with Scheme 12. HEP labeled at C2 with deuterium resulted in label

---

JOC Perspective

SCHEME 13. Labeling Studies on Fosfomycin Biosynthesis⁴

![Labeling Studies](https://i.imgur.com/abcdefg.png)

SCHEME 14. Proposed Mechanism for Fom3

![Mechanism for Fom3](https://i.imgur.com/hijklmn.png)

incorporation in the fosfomycin product when fed to *S. fradiae*. In the context of the original pathway (Scheme 9), these findings were interpreted to result from oxidation of HEP to PnAA, subsequent methyl transfer, and finally epoxidation (Scheme 13). However, the data can be equally well accounted for by the revised pathway of Scheme 12. Furthermore, oxygen labeling studies are more readily explained with the pathway in Scheme 12 than that of Scheme 13. *S. fradiae* converted HEP labeled with ¹⁸O in the hydroxyl group into fosfomycin in which 50% of the label was retained in the epoxide oxygen.⁴ᵃ These elegant studies revealed for the first time the unusual nature of the epoxidation step,⁷² but as pointed out by Hammerschmidt, the retention of the label requires remarkably fast utilization of labeled PnAA since the half-life for exchange of oxygen in acetaldehyde is about 1 min at pH 7. The pathway in Scheme 12, however, does not require any kinetic limitations since the label would never be in an exchangeable position. The revised pathway also allows formulation of a unifying mechanism for the reactions catalyzed by Fom 3 and Pmet (Scheme 14). As proposed for Pmet (Scheme 11), Fom3 would generate a 5′-deoxyadenosyl radical, which then abstracts a hydrogen atom from C2 of HEP. Such hydrogen atom abstraction has ample precedent in adenosylcobalamin dependent enzymes.⁵ The ensuing carbon based radical could then react with the methyl group of MeCbl in similar fashion as the model studies of Kräutler (eq 1) to produce HPP. Studies to verify this new model are currently underway.

Retrospective

Looking back at the relatively short history of our laboratory, two of the projects we initiated in the first year did not come to

J. Org. Chem., Vol. 71, No. 26, 2006 9569

fruition until many years later and well after the tenure decision. The lantibiotics program ultimately delivered on the potential it contained, and the biosynthesis of phosphonate natural products now forms a theme at the new Institute for Genomic Biology at UIUC. Both projects initially met typical obstacles inherent with starting a new program that hinges on multiple objectives that need to be met simultaneously. One of the early comments from an NIH reviewer still rings true today; the lantibiotic biosynthesis was a high risk, high pay-off project. In 1998, the project was fundable despite only limited preliminary results, but in the current funding climate it might be difficult to support. Hopefully, the scientific community and funding agencies will not fall prey to relying on only supporting ongoing research or sure-bet proposals and will continue to give new young faculty the chance to take on a high risk, high impact problem.

A separate but equally important challenge that I suspect regularly faces assistant professors is to decide when to abandon a project when it is not looking promising. In our specific case, new projects on prostaglandin synthase and phosphite dehydrogenase were started in year 3, and along with a third initial project on B₁₂-catalyzed dechlorination,⁷ they carried the laboratory for the early years. Would the lantibiotics program have yielded results in a sufficiently timely manner if we had continued to put all our resources into it? Of course, one will never know. One frequently heard piece of advice for new faculty is to focus and not dilute their efforts, and I certainly endorse this recommendation. Our specific case allowed us to divert midflight because of the type of new projects that emerged with relatively short timelines and yet a high level of significance and because of the availability and capability of new students in our graduate program. Without either of these conditions, temporarily redirecting our research program would not have been advisable. Furthermore, over the years seminal contributions have been made by many outstanding new investigators that stayed focus with smaller groups.

Acknowledgment. I am grateful to all of my co-workers who have contributed to the success of our research and to many colleagues for their advice and support. Our work on lantibiotic biosynthesis was supported by the NIH (GM58822), the Burroughs Welcome Fund, and the Beckman Foundation. Research on phosphite dehydrogenase was supported by the NIH (GM63003) and the Biotechnology Research and Development Consortium (Project 2-4-121), and our work on fosfomycin and bialaphos was supported by the University of Illinois. Cover art by Dorothy Loudermilk, SCS Graphic Services, University of Illinois at Urbana—Champaign.

**References**

(1) (a) McDaniel, R.; Ebert-Khosla, S.; Hopwood, D. A.; Khosla, C. *Science* **1993**, *262*, 1546–1550. (b) Kao, C. M.; Katz, L.; Khosla, C. *Science* **1994**, *265*, 509–512.

(2) Li, Y. M.; Milne, J. C.; Madison, L. L.; Kolter, R.; Walsh, C. T. *Science* **1996**, *274*, 1188–1193.

(3) (a) Kuzuyama, T.; Hidaka, T.; Kamigiri, K.; Imai, S.; Seto, H. *J. Antibiot.* **1992**, *45*, 1812–1814. (b) Kamigiri, K.; Hidaka, T.; Imai, S.; Murakami, T.; Seto, H. *J. Antibiot.* **1992**, *45*, 781–787. (c) Hidaka, T.; Goda, M.; Kuzuyama, T.; Takei, N.; Kidaka, M.; Seto, H. *Mol. Gen. Genet.* **1995**, *249*, 274–280. (d) Hidaka, T.; Hidaka, M.; Kuzuyama, T.; Seto, H. *Gene* **1995**, *158*, 149–150. (e) Seto, H.; Kuzuyama, T. *Nat. Prod. Rep.* **1999**, *16*, 589–596. (f) Seto, H.; Hidaka, T.; Kuzuyama, T.; Shibahara, S.; Usui, T.; Sakanaka, O.; Imai, S. *J. Antibiot.* **1991**, *44*, 1286–1288.

(4) (a) Hammerschmidt, F. *Angew. Chem. Int. Ed. Engl.* **1994**, *33*, 341–342. (b) Hammerschmidt, F.; Kählig, H. *J. Org. Chem.* **1991**, *56*, 2364–2370.

(5) *The Chemistry and Biochemistry of B12*; Banerjee, R., Ed.; Wiley: New York, 1999.

(6) (a) Burgess, K.; van der Donk, W. A.; Jarstfer, M. B.; Ohlmeyer, M. J. *J. Am. Chem. Soc.* **1991**, *113*, 6139–6144. (b) Burgess, K.; van der Donk, W. A.; Westcott, S. A. *J. Am. Chem. Soc.* **1992**, *114*, 9350–9359. (c) Burgess, K.; van der Donk, W. A. *J. Am. Chem. Soc.* **1994**, *116*, 6561–6569.

(7) Completing the spectrum from antibiotic biosynthesis to vitamin B₁₂ chemistry, a third initial proposal focused on the B₁₂-catalyzed dehalogenation of the priority pollutants perchloroethylene and trichloroethylene. (a) Shey, J.; van der Donk, W. A. *J. Am. Chem. Soc.* **2000**, *122*, 12403–12404. (b) McCauley, K. M.; Wilson, S. R.; van der Donk, W. A. *Inorg. Chem.* **2002**, *41*, 393–404. (c) McCauley, K. M.; Wilson, S. R.; van der Donk, W. A. *Inorg. Chem.* **2002**, *41*, 5844–5848. (d) Shey, J.; McGinley, C. M.; McCauley, K. M.; Dearth, A.; Young, B.; van der Donk, W. A. *J. Org. Chem.* **2002**, *67*, 837–846. (e) McCauley, K. M.; Wilson, S. R.; van der Donk, W. A. *J. Am. Chem. Soc.* **2003**, *125*, 4410–4411. (f) McCauley, K. M.; Pratt, D. A.; Wilson, S. R.; Shey, J.; Burkey, T. J.; van der Donk, W. A. *J. Am. Chem. Soc.* **2005**, *127*, 1126–1136. (g) Pratt, D. A.; van der Donk, W. A. *J. Am. Chem. Soc.* **2005**, *127*, 384–396. (h) Pratt, D. A.; van der Donk, W. A. *Chem. Commun.* **2006**, 558–560. (i) McGinley, C. M.; Releya, H. A.; van der Donk, W. A. *Synlett* **2006**, 211–214.

(8) Chatterjee, C.; Paul, M.; Xie, L.; van der Donk, W. A. *Chem. Rev.* **2005**, *105*, 633–684.

(9) Schnell, N.; Entian, K.-D.; Schneider, U.; Götz, F.; Zahner, H.; Kellner, R.; Jung, G. *Nature (London)* **1988**, *333*, 276–278.

(10) Rogers, L. A. *J. Bacteriol.* **1928**, *16*, 321–325.

(11) Gross, E.; Morell, J. L. *J. Am. Chem. Soc.* **1971**, *93*, 4634–4635.

(12) Delves-Broughton, J.; Blackburn, P.; Evans, R. J.; Hugenholtz, J. *Antonie van Leeuwenhoek* **1996**, *69*, 193–202.

(13) Cotter, P. D.; Hill, C.; Ross, R. P. *Nat. Rev. Microbiol.* **2005**, *3*, 777–788.

(14) For a review, see: Kuipers, O. P.; Bierbaum, G.; Ottenwälder, B.; Dodd, H. M.; Horn, N.; Metzger, J.; Kupke, T.; Gnau, V.; Bongers, R.; van den Bogaard, P.; Kosters, H.; Rollema, H. S.; de Vos, W. M.; Siezen, R. J.; Jung, G.; Götz, F.; Sahl, H. G.; Gasson, M. J. *Antonie van Leeuwenhoek* **1996**, *69*, 161–169.

(15) Xie, L.; Miller, L. M.; Chatterjee, C.; Averin, O.; Kelleher, N. L.; van der Donk, W. A. *Science* **2004**, *303*, 679–681.

(16) Li, B.; Yu, J.-P. J.; Brunzelle, J. S.; Moll, G. N.; van der Donk, W. A.; Nair, S. K. *Science* **2006**, *311*, 1464–1467.

(17) Xie, L.; Chatterjee, C.; Balsara, R.; Okeley, N. M.; van der Donk, W. A. *Biochem. Biophys. Res. Commun.* **2002**, *295*, 952–957.

(18) Breukink, E.; Wiedemann, I.; van Kraaij, C.; Kuipers, O. P.; Sahl, H.; de Kruijff, B. *Science* **1999**, *286*, 2361–2364.

(19) Breukink, E.; de Kruijff, B. *Nat. Rev. Drug Discov.* **2006**, *5*, 321–332.

(20) Walker, S.; Chen, L.; Hu, Y.; Rew, Y.; Shin, D.; Boger, D. L. *Chem. Rev.* **2005**, *105*, 449–76.

(21) Hasper, H. E.; de Kruijff, B.; Breukink, E. *Biochemistry* **2004**, *43*, 11567–11575.

(22) Okeley, N. M.; Zhu, Y.; van der Donk, W. A. *Org. Lett.* **2000**, *2*, 3603–3606.

(23) Seebeck, F. P.; Szostak, J. W. *J. Am. Chem. Soc.* **2006**, *128*, 7150–7151.

(24) Zhou, H.; van der Donk, W. A. *Org. Lett.* **2002**, *4*, 1335–1338.

(25) Zhu, Y.; Gieselman, M.; Zhou, H.; Averin, O.; van der Donk, W. A. *Org. Biomol. Chem.* **2003**, *1*, 3304–3315.

(26) Okeley, N. M.; Paul, M.; Stasser, J. P.; Blackburn, N.; van der Donk, W. A. *Biochemistry* **2003**, *42*, 13613–13624.

(27) Meyer, C.; Bierbaum, G.; Heidrich, C.; Reis, M.; Süling, J.; Iglesias-Wind, M. I.; Kempter, C.; Molitor, E.; Sahl, H.-G. *Eur. J. Biochem.* **1995**, *232*, 478–489.

(28) Koponen, O.; Tolonen, M.; Qiao, M.; Wahlstrom, G.; Helin, J.; Saris Per, E. J. *Microbiology* **2002**, *148*, 3561–3568.

(29) Hsu, S. T.; Breukink, E.; Tischenko, E.; Lutters, M. A.; De Kruijff, B.; Kaptein, R.; Bonvin, A. M.; Van Nuland, N. A. *Nat. Struct. Mol. Biol.* **2004**, *11*, 963–967.

(30) For some important examples and reviews, see: (a) Dawson, P. E.; Muir, T. W.; Clark-Lewis, I.; Kent, S. B. *Science* **1994**, *266*, 776–779. (b) Wallace, C. J. *Curr. Opin. Biotechnol.* **1995**, *6*, 403–410. (c) Lemieux, G. A.; Bertozzi, C. R. *TIBTECH* **1998**, *16*, 506–513. (d) Muir, T. W.; Sondhi, D.; Cole, P. A. *Proc. Natl. Acad. Sci. U.S.A.* **1998**, *95*, 6705–6710. (e) Wang, Q.; Chan, T. R.; Hilgraf, R.; Fokin, V. V.; Sharpless, K. B.; Finn, M. G. *J. Am. Chem. Soc.* **2003**, *125*, 3192–3193. (f) Kimmerlin, T.; Seebach, D. *J. Pept. Res.* **2005**, *65*, 229–260.

(31) Zhu, Y.; van der Donk, W. A. *Org. Lett.* **2001**, *3*, 1189–1192.

(32) Galonić, D.; van der Donk, W. A.; Gin, D. Y. *Chem.—Eur. J.* **2003**, *24*, 5997–6006.

(33) Galonić, D.; van der Donk, W. A.; Gin, D. Y. *J. Am. Chem. Soc.* **2004**, *12712–12713*.

(34) Galonić, D. P.; Ide, N. D.; van der Donk, W. A.; Gin, D. Y. *J. Am. Chem. Soc.* **2005**, *127*, 7359–7369.

(35) For our work on PGHS and other systems involving amino acid radicals, see: (a) Peng, S.; Okeley, N. M.; Tsai, A.-L.; Wu, G.; Kulmacz, R. J.; van der Donk, W. A. *J. Am. Chem. Soc.* **2001**, *123*, 3609–3610. (b) Pesavento, R. P.; van der Donk, W. A. *Adv. Protein Chem.* **2001**, *58*, 317–385. (c) Peng, S.; Okeley, N. M.; Tsai, A.-L.; Wu, G.; Kulmacz, R.

J.; van der Donk, W. A. *J. Am. Chem. Soc.* **2002**, 124, 10785–10796.  
(d) van der Donk, W. A.; Tsai, A. L.; Kulmacz, R. J. *Biochemistry* **2002**, 41, 15451–15458.  
(e) Peng, S.; McGinley, C. M.; van der Donk, W. A. *Org. Lett.* **2004**, 6, 349–352.  
(f) McGinley, C. M.; van der Donk, W. A. *J. Labelled Compd. Radiopharm.* **2006**, 49, 545–558.  
(g) McCauley, K. M.; Vrtis, J. M.; Dupont, J.; van der Donk, W. A. *J. Am. Chem. Soc.* **2000**, 122, 2403–2404.  
(h) Pratt, D. A.; Pesavento, R. P.; van der Donk, W. A. *Org. Lett.* **2005**, 7, 2735–2738.  
(i) Pesavento, R. P.; Pratt, D. A.; Jeffers, J.; van der Donk, W. A. *Dalton Trans.* **2006**, 3326–3337.  

(36) Costas, A. M.; White, A. K.; Metcalf, W. W. *J. Biol. Chem.* **2001**, 276, 17429–17436.  
(37) Vrtis, J. M.; White, A.; Metcalf, W. W.; van der Donk, W. A. *J. Am. Chem. Soc.* **2001**, 123, 2672–2673.  
(38) (a) Relyea, H. A.; van der Donk, W. A. *Bioorg. Chem.* **2005**, 33, 171–189.  
(b) Relyea, H. A.; Vrtis, J. M.; Woodyer, R.; Rimkus, S. A.; van der Donk, W. A. *Biochemistry* **2005**, 44, 6640–6649.  
(c) Woodyer, R.; Wheatley, J.; Relyea, H.; Rimkus, S.; van der Donk, W. A. *Biochemistry* **2005**, 44, 4765–4774.  
(d) Woodyer, R.; Zhao, H.; van der Donk, W. A. *FEBS J.* **2005**, 272, 3816–27.  

(39) Vrtis, J. M.; White, A.; Metcalf, W. W.; van der Donk, W. A. *Angew. Chem., Int. Ed.* **2002**, 41, 3257–3259.  
(40) Woodyer, R.; van der Donk, W. A.; Zhao, H. *Biochemistry* **2003**, 42, 11604–11614.  
(41) Woodyer, R.; van der Donk, W. A.; Zhao, H. *Comb. Chem. High Throughput Screen.* **2006**, 9, 237–245.  
(42) Johannes, T. W.; Woodyer, R. W.; Zhao, H. *Appl. Environ. Microbiol.* **2005**, 71, 5728–5734.  
(43) Siezen, R. J.; Kuipers, O. P.; de Vos, W. M. *Antonie van Leeuwenhoek* **1996**, 69, 171–184.  
(44) Xie, L.; van der Donk, W. A. *Curr. Opin. Chem. Biol.* **2004**, 8, 498–507.  
(45) Ugen, P.; Hindré, T.; Didelot, S.; Marty, C.; Haras, D.; Le Pennec, J. P.; Valle-Rehel, K.; Dufour, A. *Appl. Environ. Microbiol.* **2005**, 71, 562–565.  
(46) Patton, G. C.; van der Donk, W. A. *Curr. Opin. Microbiol.* **2005**, 8, 543–551.  
(47) Miller, L. M.; Chatterjee, C.; van der Donk, W. A.; Kelleher, N. L. *J. Am. Chem. Soc.* **2006**, 128, 1420–1421.  
(48) Chatterjee, C.; Miller, L. M.; Leung, Y. L.; Xie, L.; Yi, M.; Kelleher, N. L.; van der Donk, W. A. *J. Am. Chem. Soc.* **2005**, 127, 15332–15333.  
(49) Chatterjee, C.; Patton, G. C.; Cooper, L.; Paul, M.; van der Donk, W. A. *Chem. Biol.* **2006**, 13, 1109–1117.  
(50) (a) Kuipers, A.; De Boef, E.; Rink, R.; Fekken, S.; Kluskens, L. D.; Driessen, A. J.; Leenhouts, K.; Kuipers, O. P.; Moll, G. N. *J. Biol. Chem.* **2004**, 279, 22176–22182.  
(b) Kluskens, L. D.; Kuipers, A.; Rink, R.; de Boef, E.; Fekken, S.; Driessen, A. J.; Kuipers, O. P.; Moll, G. N. *Biochemistry* **2005**, 44, 12827–12834.  
(c) Rink, R.; Kuipers, A.; de Boef, E.; Leenhouts, K. J.; Driessen, A. J.; Moll, G. N.; Kuipers, O. P. *Biochemistry* **2005**, 44, 8873–8882.  
(d) Dawson, M. J.; Cortes Bargallo, J.; Rudd, B. A. M.; Boakes, S.; Bierbaum, G.; Hoffmann, A.; Schmitz, S. p WO 2005/093069 A2, 2005.  

(51) Hightower, K. E.; Fierke, C. A. *Curr. Opin. Chem. Biol.* **1999**, 3, 176–181.  
(52) Cotter, P. D.; O’Connor, P. M.; Draper, L. A.; Lawton, E. M.; Deegan, L. H.; Hill, C.; Ross, R. P. *Proc. Natl. Acad. Sci. U.S.A.* **2005**, 102, 18584–18589.  
(53) Kupke, T.; Kempter, C.; Jung, G.; Götz, F. *J. Biol. Chem.* **1995**, 270, 11282–11289.  

(54) Paul, M.; van der Donk, W. A. *Minirev. Org. Chem.* **2005**, 2, 23–37.  
(55) (a) Kodani, S.; Hudson, M. E.; Durrant, M. C.; Buttner, M. J.; Nodwell, J. R.; Willey, J. M. *Proc. Natl. Acad. Sci. U.S.A.* **2004**, 101, 11448–11453.  
(b) Kodani, S.; Lodato, M. A.; Durrant, M. C.; Picart, F.; Willey, J. M. *Mol. Microbiol.* **2005**, 58, 1368–1380.  

(56) (a) van der Meer, J. R.; Polman, J.; Beerthuyzen, M. M.; Siezen, R. J.; Kuipers, O. P.; de Vos, W. M. *J. Bacteriol.* **1993**, 175, 2578.  
(b) Bierbaum, G.; Szekat, C.; Josten, M.; Heidrich, C.; Kempter, C.; Jung, G.; Sahl, H. G. *Appl. Environ. Microbiol.* **1996**, 62, 385–392.  
(c) Svensson, C. I.; Rew, Y.; Malkmus, S.; Schiller, P. W.; Taulane, J. P.; Goodman, M.; Yaksh, T. L. *J. Pharmacol. Exp. Ther.* **2003**, 304, 827–832.  

(57) Lazzarini, A.; Gastaldo, L.; Candiani, G.; Ciciliato, I.; Losi, D.; Marinelli, F.; Selva, E.; Parenti, F. p WO2005/014628 A1, 2005.  
(58) Hayakawa, Y.; Sasaki, K.; Nagai, K.; Shin-ya, K.; Furihata, K. *J. Antibiot. (Tokyo)* **2006**, 59, 6–10.  
(59) Reeves, D. S. *J. Antimicrob. Chemother.* **1994**, 34, 853–858.  

(60) (a) Hendlin, D.; Stapley, E. O.; Jackson, M.; Wallick, H.; Miller, A. K.; Wolf, F. J.; Miller, T. W.; Chaiet, L.; Kahan, F. M.; Foltz, E. L.; Woodruff, H. B.; Mata, J. M.; Hernandez, S.; Mochales, S. *Science* **1969**, 166, 122–123.  
(b) Christensen, B. G.; Leanza, W. J.; Beattie, T. R.; Patchett, A. A.; Arison, B. H.; Ormond, R. E.; Kuehl, F. A., Jr.; Albers-Schonberg, G.; Jardetzky, O. *Science* **1969**, 166, 123–125.  

(61) Marquardt, J. L.; Brown, E. D.; Lane, W. S.; Haley, T. M.; Ichikawa, Y.; Wong, C. H.; Walsh, C. T. *Biochemistry* **1994**, 33, 10646–10651.  
(62) Stubbe, J.; van der Donk, W. A. *Chem. Rev.* **1998**, 98, 705–762.  
(63) (a) Benschop, H. P.; Platenburg, D. H. J. *M. J. Chem. Soc., Chem. Commun.* **1970**, 1098–1099.  
(b) Farnham, W. B.; Murray, R. K., Jr.; Mislow, K. *J. Chem. Soc., Chem. Commun.* **1971**, 146–147.  

(64) Sofia, H. J.; Chen, G.; Hetzler, B. G.; Reyes-Spindola, J. F.; Miller, N. E. *Nucleic Acids Res.* **2001**, 29, 1097–1106.  
(65) Frey, P. A.; Magnusson, O. T. *Chem. Rev.* **2003**, 103, 2129–2148.  
(66) Marchler-Bauer, A.; Bryant, S. H. *Nucleic Acids Res.* **2004**, 32, W327–W331.  

(67) Westrich, L.; Heide, L.; Li, S. M. *Chembiochem* **2003**, 4, 768–773.  
(68) Mosimann, H.; Kräutler, B. *Angew. Chem. Int. Ed.* **2000**, 39, 393–395.  
(69) Woodyer, R. D.; Shao, Z.; Thomas, P.; Kelleher, N. L.; Metcalf, W. M.; van der Donk, W. A.; Zhao, H. **2006**, in press.  
(70) Woodyer, R. D.; Li, G.; Zhao, H.; van der Donk, W. A. *Chem. Comm.*, in press.  

(71) (a) Schwartz, D.; Berger, S.; Heinzelmann, E.; Muschko, K.; Welzel, K.; Wohlleben, W. *Appl. Environ. Microbiol.* **2004**, 70, 7093–7102.  
(b) Blodgett, J. A.; Zhang, J. K.; Metcalf, W. W. *Antimicrob. Agents Chemother.* **2005**, 49, 230–240.  

(72) For more recent mechanistic and structural studies of this fascinating process, see:  
(a) Liu, P.; Murakami, K.; Seki, T.; He, X.; Yeung, S. M.; Kuzuyama, T.; Seto, H.; Liu, H. W. *J. Am. Chem. Soc.* **2001**, 123, 4619–4620.  
(b) Zhao, Z.; Liu, P.; Murakami, K.; Kuzuyama, T.; Seto, H.; Liu, H. W. *Angew. Chem., Int. Ed.* **2002**, 41, 4529–4532.  
(c) Liu, P.; Liu, A.; Yan, F.; Wolfe, M. D.; Lipscomb, J. D.; Liu, H. W. *Biochemistry* **2003**, 42, 11577–11586.  
(d) Liu, P.; Mehn, M. P.; Yan, F.; Zhao, Z.; Que, L., Jr.; Liu, H. W. *J. Am. Chem. Soc.* **2004**, 126, 10306–10312.  
(e) Higgins, L. J.; Yan, F.; Liu, P.; Liu, H. W.; Drennan, C. L. *Nature (London)* **2005**, 437, 838–844.  
(f) McLuskey, K.; Cameron, S.; Hammerschmidt, F.; Hunter, W. N. *Proc. Natl. Acad. Sci. U.S.A.* **2005**, 102, 14221–14226.  

JO0614240
