
Available online at www.sciencedirect.com

ScienceDirect

Archives of Biochemistry and Biophysics 464 (2007) 122–129

**Rat NAD⁺-dependent 3α-hydroxysteroid dehydrogenase (AKR1C17): A member of the aldo-keto reductase family highly expressed in kidney cytosol**

Masaharu Sanai ${}^{a}$, Satoshi Endo ${}^{a}$, Toshiyuki Matsunaga ${}^{a}$, Shuhei Ishikura ${}^{a,1}$, Kazuo Tajima ${}^{b}$, Ossama El-Kabbani ${}^{c}$, Akira Hara ${}^{a,*}$

${}^{a}$ Laboratory of Biochemistry, Gifu Pharmaceutical University, Mitahora-higashi, Gifu 502-8585, Japan  
${}^{b}$ Faculty of Pharmaceutical Sciences, Hokuriku University, Kanazawa 920-1181, Japan  
${}^{c}$ Department of Medicinal Chemistry, Victorian College of Pharmacy, Monash University (Parkville campus), 381 Royal Parade, Parkville, Vic. 3052, Australia  

Received 13 March 2007, and in revised form 28 March 2007  
Available online 17 April 2007

---

### Abstract

Mammalian 3α-hydroxysteroid dehydrogenases (3α-HSDs) have been divided into two types: Cytosolic NADP(H)-dependent 3α-HSDs belonging to the aldo-keto reductase family, and mitochondrial and microsomal NAD⁺-dependent 3α-HSDs belonging to the short-chain dehydrogenase/reductase family. In this study, we characterized a rat aldo-keto reductase (AKR1C17), whose functions are unknown. The recombinant AKR1C17 efficiently oxidized 3α-hydroxysteroids and bile acids using NAD⁺ as the preferred coenzyme at an optimal pH of 7.4–9.5, and was inhibited by ketamine and organic anions. The mRNA for AKR1C17 was detected specifically in rat kidney, where the enzyme was more highly expressed as a cytosolic protein than NADP(H)-dependent 3α-HSD (AKR1C9). Thus, AKR1C17 represents a novel NAD⁺-dependent type of cytosolic 3α-HSD with unique inhibitor sensitivity and tissue distribution. In addition, the replacement of Gln270 and Glu276 of AKR1C17 with the corresponding residues of NADP(H)-dependent 3α-HSD resulted in a switch in favor of NADP⁺ specificity, suggesting their key roles in coenzyme specificity.

© 2007 Elsevier Inc. All rights reserved.

---

### Keywords: Aldo-keto reductase superfamily; AKR1C17; 3α-Hydroxysteroid dehydrogenase; Bile acid; Ketamine; Inhibitor sensitivity; Organic anion; Coenzyme specificity; Kidney-specific protein; Site-directed mutagenesis

---

3α-Hydroxysteroid dehydrogenases (3α-HSDs², EC 1.1.1.213) catalyze the conversion of 3-ketosteroids to 3α-hydroxy compounds, and play an important role in regulation of intracellular levels of biologically active steroid hormones. For example, 3α-HSD activity is responsible for maintaining the balance between the most potent androgen, 5α-dihydrotestosterone (5α-androstan-17β-ol-3-one), and its much less active form, 5α-androstane-3α,17β-diol [1]. In the brain, 3α-HSD activity controls the amounts of potent neuroactive steroids, 5α (and 5β)-pregnan-3α-ol-20-ones, which are positive allosteric regulators of the γ-aminobutyric acid type A (GABAₐ) receptor [1,2]. 3α-HSDs preferentially utilize either NADP(H) or NAD(H) as coenzyme. Because the predominant forms of pyridine nucleotide coenzymes in the cells are NAD⁺ and NADPH, it is generally believed that the NAD(H)-dependent hydroxysteroid dehydrogenases function mainly in the oxidative direction, whereas the NADP(H)-dependent enzymes function as reductases [3]. NAD⁺-dependent 3α-HSDs are present in the mitochondrial and microsomal

* Corresponding author.
E-mail address: hara@gifu-pu.ac.jp (A. Hara).
¹ Present address: Hospital for Sick Children, Toronto, Ont., Canada M5G 1X8.
² Abbreviations used: HSD, hydroxysteroid dehydrogenase; AKR, aldo-keto reductase; TBE, 6-tert-Butyl-2,3-epoxy-5-cyclohexene-1,4-dione; RT, reverse transcription.

0003-9861/$ - see front matter © 2007 Elsevier Inc. All rights reserved.  
doi:10.1016/j.abb.2007.04.003

fractions of mammalian tissues, and belong to the short-
chain dehydrogenase/reductase superfamily [4–7]. The
NADPH-dependent enzymes are found in the microsomal
and cytosolic fractions of tissue homogenates [8]. Although
the primary structures of the microsomal NADPH-depen-
dent enzymes have yet to be determined, the cytosolic
NADPH-dependent 3α-HSDs have been cloned from
human [1], monkey [9], mouse [10] and rat [11] tissues,
and are members of the aldo-keto reductase (AKR) super-
family. The NADPH-dependent 3α-HSDs show more than
60% amino acid sequence identity among them and with
other HSDs and prostaglandin F synthases, and are classi-
fied into an AKR1C subfamily of this superfamily [12].
A characteristic of the cytosolic 3α-HSD distinct from
the microsomal NAD⁺-dependent enzymes is that the cyto-
solic enzymes exhibit dihydrodiol dehydrogenase and car-
bonyl reductase activities for nonsteroidal compounds
[1,9,10,13,14]. In addition, several isozymes or multiple
forms of cytosolic NADPH-dependent 3α-HSD have been
found in mammalian tissues. In humans, the enzyme exists
in three isoforms, AKR1C2, AKR1C3, AKR1C4 [1,15], of
which AKR1C4 is liver-specific and the other isoforms are
ubiquitously distributed. In monkeys, cDNAs for two 3α-
HSD isoforms that show similar sequences to human
AKR1C3 and AKR1C4 have been cloned [9,16]. In mice,
3α-HSD (AKR1C14) [10] is ubiquitously expressed, and
the two enzymes with 17-HSD activity, 3(17)α-HSD
(AKR1C21) [17] and 3α(17β)-HSD (AKR1C20) [18], have
recently been identified in the kidney and liver, respectively.
Only one species of 3α-HSD (AKR1C9) has been identified
in rat tissues [11,13,14].

A recent rat genomic analysis has predicted a cluster of
nine genes for members of the AKR1C subfamily. Four of
the proteins encoded in the rat genes are AKR1C9, 20α-
HSD (AKR1C8) [19] plus two 17β-HSDs [20]. The functions
of the other five proteins remain unknown. In the course of
our study on the enzymatic characterization of the five func-
tionally unknown proteins [15], we found that one of the pro-
teins exhibited NAD⁺-linked 3α-HSD activity. This enzyme
is AKR1C17, which was first identified from a cDNA similar
to that for AKR1C9 and named rat aldo-keto reductase d
(RAKd) [21]. In this study, we examined the substrate spec-
ificity and inhibitor sensitivity of AKR1C17. We compared
its tissue distribution with that of AKR1C9 in order to
elucidate a physiological role for the novel cytosolic
NAD⁺-dependent 3α-HSD belonging to the AKR super-
family. Furthermore, the structural determinants for the
NAD(H) specificity of the enzyme were examined by
replacing Gln270 and Glu276 with the corresponding resi-
dues (Lys and Arg) of NADP(H)-dependent 3α-HSDs.

Materials and methods

**Materials**

Steroids were obtained from Steraloids (Newport, RI) and Sigma
Chemicals (Ann Arbor, MI); prostaglandins were from Cayman
Chemicals (Ann Arbor, MI); resins for column chromatography were
from Amersham Biosciences (Piscataway, NJ); and a pCR T7/CT-TOPO
TA expression kit and *Escherichia coli* BL21 (DE3) pLysS were from
Invitrogen (Carlsbad, CA). *Pfu* DNA polymerase and *Taq* DNA poly-
merase were purchased from Stratagene (La Jolla, CA) and Takara
(Kusatsu, Japan), respectively. 6-tert-Butyl-2,3-epoxy-5-cyclohexene-
1,4-dione (TBE) and *trans*-benzene dihydrol were synthesized by the
methods of Tajima et al. [22] and Platt and Osech [23]. Recombinant
AKR1C9 was prepared following published methodology in a previous
study [20]. All other chemicals were of the highest grade that could be
obtained commercially.

**cDNA isolation and site-directed mutagenesis**

The cDNA for AKR1C17 was amplified by reverse transcription (RT)-
PCR from total RNA of a 10-week-old male Wistar rat kidney. The
preparation of the total RNA and RT were carried out as described
previously [24]. PCR was performed with *Pfu* DNA polymerase and fol-
lowing primers. The sense primer (rakd-N; 5′-atgagttccaaacat-3′) and
antisense primer (rakd-C; 5′-agcatactgtccatat-3′) correspond to positions
1–15 and 1012–1028, respectively, of the transcript (Accession No.
XM_574075) predicted from the gene that is similar to RAKd. The
amplification consisted of an initial denaturation (94 °C/30 s), and 40
cycles of denaturation (94 °C/30 s), annealing (55 °C/30 s) and extension
(72 °C/3 min). The PCR products were ligated into pCR T7/CT-TOPO
vectors, and the expression constructs were transfected into *Escherichia*
coli BL21 (DE3) pLysS. The inserts of the cloned cDNAs were sequenced
by using a Beckman CEQ2000XL DNA sequencer.

Mutagenesis was performed using a QuikChange site directed muta-
genesis kit (Stratagene) and the pCR T7/CT-TOPO expression plasmid as
the template according to the protocol described by the manufacturer. The
primers for preparing Q270K and E276R were 5′-cgtgaggtggtgccctt
gccaagagtttcaaagag-3′ and 5′-gccagagtttcaaagaga tcggatgagagagaatttg-
cagg-3′, respectively, where the mutation codons are underlined. The
double mutation of Q270K/E276R was prepared using the former primer
and the expression plasmid harboring the cDNA for E276R as the tem-
plate. The complete coding regions of the cDNAs in the expression plas-
mids were sequenced to confirm the presence of the desired mutation and
to ensure that no other mutation had occurred.

**Expression and purification of recombinant enzymes**

The *E. coli* cells were cultured in a LB medium containing ampicillin
(50 μg/mL) for 24 h at 20 °C after the addition of isopropyl 1-thio-β-D-
galactopyranoside (1 mM). The cell extract was prepared as described
previously [24]. The recombinant AKR1C17 was purified at 4 °C by
subsequent three column chromatography steps in the buffers supple-
mented with 5 mM 2-mercaptoethanol, 1 mM EDTA and 20% glycerol to
stabilize the enzyme. In the purification of AKR1C17, the *E. coli* cell
extract was dialyzed against 20 mM Tris–HCl, pH 8.0, and applied to a Q-
Sepharose column (2 × 20 cm) equilibrated with the buffer. The enzyme
eluted with a linear gradient of 0–0.1 M NaCl was applied to a Sephadex
G-100 column (3 × 70 cm) that had been equilibrated with 10 mM
potassium phosphate, pH 7.0. The enzyme fraction was applied to a Blue-
Sepharose column (1.5 × 5 cm) equilibrated with the same buffer. The
enzyme was eluted with the buffer plus 0.5 mM NAD⁺, concentrated by
ultrafiltration, and stored at −20 °C. The mutant enzymes were purified
according to the methods for purification of the respective wild-type
enzymes, except that the enzymes were eluted with 20 mM Tris–HCl, pH
8.0, containing 0.5 mM NADP⁺ and 0.05 M NaCl from the Blue-
Sepharose column. Protein concentration was determined by the method
of Bradford [25] using bovine serum albumin as the standard.

**Assay of enzyme activity**

Dehydrogenase and reductase activities of AKR1C17 and AKR1C9
were assayed by measuring the rate of change in NAD(P)H fluorescence

(at 455 nm with an excitation wavelength of 340 nm) and its absorbance (at 340 nm), respectively. The standard reaction mixture for the dehydrogenase activity of AKR1C17 consisted of 0.1 M potassium phosphate buffer, pH 7.4, 1.0 mM NAD⁺, 5 μM 4-androsten-3α-ol-17-one and enzyme, in a total volume of 2.0 mL. The activity of AKR1C9 was determined with 0.25 mM NADP⁺ and 50 μM 5α-androstane-3α-ol-17-one as the coenzyme and substrate, respectively, in the reaction mixture. The reductase activity was assayed with 0.1 mM NADH or NADPH and an appropriate amount of carbonyl substrate in the reaction mixture. The steroids, other substrates and inhibitors, which are hardly soluble in water, were dissolved in methanol or 50% methanol, and added into the reaction mixture, in which the final concentration of methanol was less than 1.2%. The concentration of methanol did not affect the activity of the enzymes. One unit (U) of enzyme activity was defined as the amount of enzyme that catalyzes the formation or oxidation of 1 μmol NAD(P)H per minute at 25°C.

The pH dependency of the enzyme activity was determined with 0.1 M phosphate buffers (pH 5.5–11.0), which were prepared by mixing of tripotassium phosphate and phosphoric acid solutions. The apparent \( K_m \) and \( k_{cat} \) values were determined over a range of five substrate concentrations at a saturating concentration of coenzyme by fitting the initial velocities into the Michaelis–Menten equation. Kinetic studies in the presence of inhibitors were carried out in a similar manner, after IC₅₀ (concentration required for 50% inhibition) values for the inhibitors were determined. In addition to the kinetic constants, the inhibition constants (\( K_i \) and \( K'_i \)) [26] were calculated by using the appropriate programs of the ENZFITTER (Biosoft, Cambridge, UK). The kinetic constants and IC₅₀ values are expressed as the means of at least three determinations, in which all standard errors were less than 20%.

### Product identification

To identify reaction products, reduction was conducted in a 2.0 mL system containing 0.1 mM NADH, substrate (20–200 μM), enzyme (10 μg) and 0.1 M potassium phosphate, pH 7.4. The substrate and products were extracted into 6 mL ethyl acetate 30 min after the reaction was started at 37°C. The steroids were analyzed by liquid chromatography/mass spectrometry [18], and the products of TBE reduction were identified by the HPLC method [22].

### Analyses of expression of AKR1C17 and its mRNA in tissues

Kidneys (6 g), excised from 12 to 14 week-old female Wistar rats, were homogenized with 50 mM Tris–HCl, pH 7.5, containing 0.14 M KCl, 5 mM 2-mercaptoethanol, 0.5 mM EDTA and 20% glycerol. The mitochondrial, microsomal and cytosolic fractions of the homogenate were prepared by the methods described previously [4]. AKR1C17 and AKR1C9 in the cytosolic fraction were partially purified from the cytosolic fraction by the \((NH_4)_2SO_4\) fractionation (30–75% saturation) followed by two chromatography steps on Sephadex G-100 and Q-Sepharose columns, as described above. For analysis of expression of mRNAs for AKR1C17 and AKR1C9 in rat tissues, the total RNA samples were prepared from the rat tissues, and RT-PCR was carried out using Taq DNA polymerase and the gene-specific primers. The forward and reverse primers for the amplification of the cDNA for AKR1C17 (1028 bp) were rakd-N and rakd-C, respectively, and the respective primers for that of AKR1C9 cDNA were 5′-gcgatggattccatatctc and 5′-gacaaagtccaccatgttac. PCR amplification consisted of an initial denaturation step (94°C/5 min), 30 cycles of denaturation (94°C/30 s), annealing (50°C/30 s) and extension (72°C/1 min), and a final extension step (72°C/5 min). In the amplification of AKR1C9 cDNA (991 bp), the anneal temperature was changed to 55°C. The cDNA for rat β-actin was also amplified as an internal control with specific primers obtained from Toyobo (Osaka, Japan). The specificity of the primers was confirmed by sequencing the respective PCR products.

---

### Results

#### Purification of recombinant AKR1C17

The recombinant enzyme (37 kDa) was highly expressed in the *E. coli* cells transfected with the expression plasmid harboring the AKR1C17 cDNA (Fig. 1A). The cell extract exhibited NAD⁺-linked dehydrogenase activity toward 4-androsten-3α-ol-17-one (0.07 U/mg), which was about 10-fold higher than the NADP⁺-linked activity. The enzyme was purified from 500 mL of cultured cells and 42 mg of homogeneous enzyme with a specific activity of 0.23 U/mg was obtained in a yield of 44%. The low purification fold was due to the high expression of the recombinant enzyme in the *E. coli*. The enzyme was eluted at a low molecular weight of approximately 36 kDa on the Sephadex G-100 chromatography step, suggesting its monomeric nature. The purified AKR1C17 was unstable in solution without glycerol, and was almost completely inactivated within 6 h at 4°C when it was diluted to less than 20 μg/ml. Since the inactivation was prevented by the addition of 20% glycerol or 0.5 mM NAD⁺, 20% glycerol was added into the buffers used in the purification and dilution of the enzyme.

#### Substrate specificity

The optimum pH for the NAD⁺-linked dehydrogenase activity of the purified AKR1C17 was observed between

![Figure 1](#fig1)

**Fig. 1.** SDS-PAGE of *E. coli* cell extracts, and purified recombinant AKR1C17 and mutant enzymes. The gel (12.5%) was stained with Coomassie Brilliant Blue. (A) Lanes: a, molecular mass markers with 90, 66, 45, 30, 20.1 and 14.4 kDa from the top; b, the extract (10 μg) of *E. coli* cells transfected with the expression vector alone; c, the extract (10 μg) of *E. coli* cells transfected with the expression vector harboring AKR1C17 cDNA; and d, the purified recombinant AKR1C17 (3 μg). (B) Lanes: e, AKR1C17; f, Q270K; g, E276R; and h, Q270K/E276R (each 2 μg).
pH 7.4 and 9.5. The NADP⁺-linked activity was low with the maximum activity observed at pH 7.5. Therefore, the enzyme properties were examined with NAD(H) as the coenzymes at a physiological pH of 7.4. The enzyme oxidized 3α-hydroxy-C19- and C21-steroids and showed low \( K_{\mathrm{m}} \) values of 0.3–3.9 μM (Table 1). The \( k_{\mathrm{cat}} \) values for Δ⁴-unsaturated steroids and 5β-reduced steroids were higher than those for the 5α-reduced steroids. AKR1C17 also oxidized various bile acids including ursodeoxycholic acid and muricholic acid found in rat plasma [27]. The catalytic efficiencies for lithocholic acid, ursodeoxycholic acid, chenodeoxycholic acid and their conjugated forms were higher than those for other bile acids with 7α-OH, 12α-OH and 6α/β-OH groups. The enzyme did not oxidize 3β-hydroxysteroids (isolithocholic acid, 5α/β-androstan-3β-ol-17-ones and 5α/β-pregnan-3β-ol-20-ones), 17-hydroxysteroids (4-androsten-17α/β-ol-3-ones, 17α/β-estradiols and 5α/β-androstan-17β-ol-3-ones), and 20-hydroxysteroids (5α/β-pregnan-20α/β-ol-3-ones, 5β-pregnane-3β,20β-diol, 4-pregnen-20α/β-ol-3-ones and 4-pregenen-17α,20β-diol-3-one). When non-steroidal alcohols that are oxidized by AKR1C9 and other enzymes in the AKR1C subfamily [10,11,14,15,18,19,21] were tested as the substrates for AKR1C17, the enzyme exhibited low catalytic activity towards S-tetralol (\( K_{\mathrm{m}} = 1.7 \) mM, \( k_{\mathrm{cat}} = 1.0 \, \mathrm{min}^{-1} \)), cis-benzene dihydrodiol (\( K_{\mathrm{m}} = 2.5 \) mM, \( k_{\mathrm{cat}} = 2.6 \, \mathrm{min}^{-1} \)) and trans-benzene dihydrodiol (\( K_{\mathrm{m}} = 0.79 \) mM, \( k_{\mathrm{cat}} = 0.59 \, \mathrm{min}^{-1} \)), and did not show significant activity towards R-tetralol, cyclohex-2-en-1-ol, benzyl alcohol, prostaglandin F₂α and its 11β-isomer and aliphatic alcohols (all-trans-retinol, geranylgeraniol, farnesol and 1-nonanol).

In the reverse reaction in the presence of 0.1 mM NADH, AKR1C17 reduced 5β-reduced 3-ketosteroids such as 5β-androstane-3,17-dione (\( K_{\mathrm{m}} = 2.1 \) μM, \( k_{\mathrm{cat}} = 2.8 \, \mathrm{min}^{-1} \)), 5β-pregnane-3,20-dione (\( K_{\mathrm{m}} = 2.4 \) μM, \( k_{\mathrm{cat}} = 6.3 \, \mathrm{min}^{-1} \)), 5β-pregnan-20α-ol-3-one (\( K_{\mathrm{m}} = 2.7 \) μM, \( k_{\mathrm{cat}} = 6.9 \, \mathrm{min}^{-1} \)) and dehydrolithocholic acid (\( K_{\mathrm{m}} = 0.4 \) μM, \( k_{\mathrm{cat}} = 1.3 \, \mathrm{min}^{-1} \)). The enzyme exhibited low activity (less than 0.3 mU/mg) towards 20 μM testosterone, 4-androstene-3,17-dione, 5α-androstane-3,17-dione, 5α-dihydrotestosterone and 5α-pregnane-3,20-dione, and did not reduce progesterone, cortisol and aldosterone. The enzyme reduced non-steroidal α-dicarbonyl compounds such as 9,10-phenanthrenequinone (\( K_{\mathrm{m}} = 38 \) μM, \( k_{\mathrm{cat}} = 11 \, \mathrm{min}^{-1} \)), isatin (\( K_{\mathrm{m}} = 22 \) μM, \( k_{\mathrm{cat}} = 17 \, \mathrm{min}^{-1} \)), TBE (\( K_{\mathrm{m}} = 23 \) μM, \( k_{\mathrm{cat}} = 6.0 \, \mathrm{min}^{-1} \)) and diacetyl (\( K_{\mathrm{m}} = 0.42 \) mM, \( k_{\mathrm{cat}} = 10 \, \mathrm{min}^{-1} \)), but did not show significant activities towards 4-nitrobenzaldehyde and 4-nitroacetophenone that are substrates of AKR1C9 [13]. The reduced products of 5β-androstane-3,17-dione and 5β-pregnane-3,20-dione by the enzyme were identified as 5β-androstan-3α-ol-17-one and 5β-pregnan-3α-ol-20-one, respectively. In the case of TBE, both R- and S-forms of the reduced TBE (6-tert-butyl-2,3-epoxy-4-hydroxy-5-cyclohexen-1-one) were present in a ratio of 2.3 : 1, respectively.

### Table 1
#### Substrate specificity for hydroxysteroids in the oxidation reaction by AKR1C17

| Substrate                  | \( K_{\mathrm{m}} \) (μM) | \( k_{\mathrm{cat}} \) (min⁻¹) | \( k_{\mathrm{cat}} / K_{\mathrm{m}} \) (min⁻¹ μM⁻¹) |
|----------------------------|--------------------------|--------------------------------|--------------------------------------------------|
| **C19-steroids**           |                          |                                |                                                  |
| 4-Androsten-3α-ol-17-one   | 0.3                      | 11                             | 36                                               |
| 4-Androstene-3α,17β-diol   | 1.9                      | 7.6                            | 4.0                                              |
| 5β-Androstan-3α-ol-17-one  | 0.7                      | 1.7                            | 2.4                                              |
| 5β-Androstane-3α,17β-diol  | 1.0                      | 0.79                           | 0.79                                             |
| 5α-Androstan-3α-ol-17-one  | 3.9                      | 0.74                           | 0.19                                             |
| 5α-Androstane-3α,17β-diol  | 3.8                      | 0.21                           | 0.06                                             |
| **C21-steroids**           |                          |                                |                                                  |
| 5β-Pregnan-3α-ol-20-one    | 0.3                      | 4.8                            | 16                                               |
| 5β-Pregnane-3α,20α-diol    | 0.5                      | 7.9                            | 16                                               |
| 4-Pregnen-3α-ol-20-one     | 0.7                      | 8.0                            | 11                                               |
| 5α-Pregnane-3α,20α-diol    | 1.9                      | 0.23                           | 0.12                                             |
| 5α-Pregnan-3α-ol-20-one    | 1.2                      | 0.11                           | 0.09                                             |
| **Bile acids**             |                          |                                |                                                  |
| Glycolithocholic acid      | 0.2                      | 3.8                            | 19                                               |
| Taurolithocholic acid      | 0.3                      | 5.1                            | 17                                               |
| Ursodeoxycholic acid       | 0.3                      | 5.1                            | 17                                               |
| Tauroursodeoxycholic acid  | 1.0                      | 7.9                            | 7.9                                              |
| Lithocholic acid           | 0.3                      | 1.9                            | 6.3                                              |
| Glycochenodeoxycholic acid | 1.0                      | 6.1                            | 6.1                                              |
| Glycoursodeoxycholic acid  | 1.3                      | 7.1                            | 5.5                                              |
| Chenodeoxycholic acid      | 1.2                      | 4.1                            | 3.4                                              |
| Deoxycholic acid           | 1.4                      | 0.87                           | 0.62                                             |
| Cholic acid                | 15                       | 2.3                            | 0.15                                             |
| β-Muricholic acid          | 5.8                      | 0.58                           | 0.10                                             |
| Hyodeoxycholic acid        | 1.7                      | 0.12                           | 0.07                                             |
| Murocholic acid            | 3.4                      | 0.21                           | 0.06                                             |
tively, which is in contrast to the high stereospecific reduction into the S-form by AKR1C9 [20].

### Kinetic mechanism of the reaction and inhibition by drugs and organic anions

In the oxidation of 4-androsten-3α-ol-17-one by AKR1C17, double reciprocal plots of initial velocity versus concentration of the substrate at fixed levels of NAD⁺ yielded a series of intersecting lines (data not shown). The products, 4-androstene-3,17-dione and NADH, inhibited the activity uncompetitively \((K_{i}^{\prime}=18 \mu \mathrm{M})\) and competitively \(\left(K_{i}=1.7 \mu \mathrm{M}\right)\), respectively, with respect to NAD⁺ in the presence of a saturated concentration (4 μM) of the substrate. In the presence of a saturated concentration of the coenzyme (2 mM), 4-androstene-3,17-dione showed noncompetitive inhibition \(\left(K_{i}=2.2 \mu \mathrm{M}\right.\) and \(K_{i}^{\prime}=27 \mu \mathrm{M}\)) with respect to the substrate, while NADH did not show significant inhibition. The results are consistent with a sequential ordered mechanism [28], where NAD⁺ binds to the enzyme first and NADH leaves last.

AKR1C17 was inhibited by several drugs and organic anions, which were divided into three types based on the inhibition patterns (Table 2). Medroxyprogesterone acetate, glycyrrhetinic acid, dexamethasone, indomethacin, and ketamine were uncompetitive and competitive inhibitors with respect to NAD⁺ and the substrate, respectively. Among the organic anions, sulfobromophthalein and indigo carmine were competitive and noncompetitive inhibitors with respect to NAD⁺ and the substrate, respectively. Other organic anions showed noncompetitive inhibition with respect to both NAD⁺ and the substrate. Based on the above reaction mechanism, the inhibition patterns suggest that the drugs bind to the substrate binding site of the enzyme-coenzyme binary complex, while the organic anions bind to sites, different from the substrate-binding pocket, of both the free enzyme and its coenzyme complex. It should be noted that AKR1C9 was not inhibited by 1 mM ketamine.

#### Table 2
**Inhibition patterns and constants for drugs and organic anions**

| Inhibitor<sup>a</sup> | Varied substrate | Pattern<sup>b</sup> | Constant (μM) |
|-----------------------|------------------|--------------------|----------------|
|                       |                  |                    | \(K_{i}\)      | \(K_{i}^{\prime}\) |
| MPA                   | NAD⁺             | UC                 | –              | 3.1               |
|                       | TUDCA            | C                  | 0.4            | –                 |
| Glycyrrhetinic acid   | NAD⁺             | UC                 | –              | 5.6               |
|                       | TUDCA            | C                  | 1.0            | –                 |
| Dexamethasone         | NAD⁺             | UC                 | –              | 79                |
|                       | TUDCA            | C                  | 16             | –                 |
| Indomethacin          | NAD⁺             | UC                 | –              | 81                |
|                       | TUDCA            | C                  | 16             | –                 |
| Ketamine              | NAD⁺             | UC                 | –              | 133               |
|                       | TUDCA            | C                  | 44             | –                 |
| BSP                   | NAD⁺             | C                  | 0.6            | –                 |
|                       | TUDCA            | NC                 | 1.3            | 4.0               |
| Indigo carmine        | NAD⁺             | C                  | 3.0            | –                 |
|                       | TUDCA            | NC                 | 6.5            | 16                |
| Phenol red            | NAD⁺             | NC                 | 23             | 31                |
|                       | TUDCA            | NC                 | 19             | 32                |
| Amaranth              | NAD⁺             | NC                 | 13             | 22                |
|                       | TUDCA            | NC                 | 26             | 34                |
| ANS                   | NAD⁺             | NC                 | 30             | 34                |
|                       | TUDCA            | NC                 | 33             | 69                |

The activity was assayed with 10 μM tauroursodeoxycholic acid (TUDCA) or 1 mM NAD⁺ as the fixed substrate.

<sup>a</sup> Abbreviations: MPA, medroxyprogesterone acetate; BSP, sulfobromophthalein; and ANS, 1-anilino-8-naphthalene sulfonate.

<sup>b</sup> Inhibition patterns: UC, uncompetitive; C, competitive; and NC, noncompetitive.

one dehydrogenase activity (for AKR1C9) were partially purified from the renal cytosolic fraction. The two enzyme activities were eluted at the same fractions on the Sephadex G-100 filtration, but separated on Q-Sepharose chromatography. The NAD⁺-linked ursodeoxycholate dehydrogenase activity was separated into major and minor peaks, of which the activity in the minor peak was co-eluted with the NADP⁺-linked 5α-androstan-3α-ol-17-one dehydrogenase activity (Fig. 2B). The enzyme present in the major peak showed preference for 5β-reduced 3α-hydroxysteroids over the 5α-reduced isomers as the substrates (data not shown) and was almost completely inhibited by 1 mM ketamine, indicating that the separated enzyme is AKR1C17. The enzyme present in the minor peak oxidized both 5α- and 5β-reduced 3α-hydroxysteroids at similar rates, was not inhibited by ketamine, and showed high sensitivity to indomethacin \(\left(\mathrm{IC}_{50}=1.0 \mu \mathrm{M}\right)\). The properties of the enzyme are similar to those of AKR1C9 [13]. Based on the specific activities of the recombinant AKR1C17 and AKR1C9, the enzyme amounts eluted in the two peaks were calculated to be 5.6 mg and 0.015 mg, respectively.

### Tissue specific expression

Since AKR1C17 is NAD⁺-dependent 3α-HSD, its tissue distribution was compared with that of NADPH-dependent 3α-HSD (AKR1C9) by RT-PCR. The mRNA for AKR1C17 was detected only in kidneys of male and female rats, in contrast to ubiquitous expression of the AKR1C9 mRNA (Fig. 2A). The mRNAs for the two enzymes were expressed more highly in renal cortex than in the medulla. The dehydrogenase activity for ursodeoxycholic acid, a unique substrate of AKR1C17, was detected in the cytosolic fraction of the kidney homogenate, but not in the mitochondrial and microsomal fractions. To compare the amounts of the two enzymes in the kidney, the NAD⁺-linked ursodeoxycholate dehydrogenase activity (for AKR1C17) and NADP⁺-linked 5α-androstan-3α-ol-17-

### Alteration of coenzyme specificity by mutagenesis

One of the significant differences between AKR1C17 and AKR1C9 was the coenzyme specificity as described above. Crystallographic and site-directed mutagenesis studies of NADP(H)-dependent AKR1C9 and other

M. Sanai et al. | Archives of Biochemistry and Biophysics 464 (2007) 122–129

Q270K/E276R resulted in a drastic decrease in the \( K_{\mathrm{m}} \) value for NADP⁺ which is comparable to those reported for the NADPH-dependent 3α-HSDs [9,10,13,15], while the \( k_{\mathrm{cat}} \) value was low. NAD(P)H were competitive inhibitors with respect to NAD⁺, and the alteration of the \( K_{i} \) values for NAD(P)H by the mutagenesis was similar to that of the \( K_{\mathrm{m}} \) values for NAD(P)⁺, suggesting that the mutagenesis resulted in alterations of affinity for both oxidized and reduced forms of the coenzymes.

## Discussion

The enzymatic properties of AKR1C17 characterized in the present study indicate that the enzyme is a novel cytosolic NAD⁺-dependent 3α-HSD distinct from previously known mammalian 3α-HSDs with respect to the following terms. (1) The unique feature of AKR1C17 is the coenzyme specificity for NAD(H). This is in contrast to previous recognition that cytosolic 3α-HSDs belonging to the AKR superfamily are NADPH-dependent [1,9–11,13] while NAD⁺-dependent 3α-HSDs are microsomal or mitochondrial enzymes belonging to the short-chain dehydrogenase/reductase superfamily [4–7]. (2) The steroid specificity of AKR1C17 is similar to those of cytosolic NADPH-dependent 3α-HSDs [9,10,13,15]. However, AKR1C17's preference for 5β-reduced 3α-hydroxysteroids over the 5α-reduced isomers differs from those of the NADPH-dependent 3α-HSDs, which exhibit comparable catalytic efficiency for both 5β- and 5α-reduced steroids. In the reverse reaction AKR1C17 was inactive towards 4-nitroacetophenone and 4-nitrobenzaldehyde that are reduced by the NADPH-dependent 3α-HSDs and the enzymes with 17α-, 17β- or 20α-HSD activity [17,18,20]. In addition, AKR1C17 did not oxidize retinol, 3β-hydroxysteroids and 17β-hydroxysteroids that are substrates of microsomal 3α-HSDs [5–7]. (3) The inhibitor sensitivity of AKR1C17 is different from that of cytosolic NADPH-dependent 3α-HSDs, although no comparable data on the microsomal enzymes is available. AKR1C17 is less sensitive to medroxyprogesterone acetate and anti-inflammatory agents that potently inhibit the NADP(H)-preferring cytosolic 3α-HSDs [9,10,13]. Among the inhibitors identified from the present study, ketamine may be a specific inhibitor for AKR1C17, because it did not inhibit AKR1C9. This is supported by our preliminary results indicating that the compound did not inhibit other AKRs (1C1, 1C2, 1C4, 1C6, 1C8, 1C12, 1C14, 1C15, 1C16, 1C18, 1C19, 1C20 and 1C24). Thus, AKR1C17 is a novel NAD⁺-dependent 3α-HSD, which is found not only in rats, but also in other mammals.

As we have shown, AKR1C17 utilizes NAD(H) as the preferred coenzymes. The present mutagenesis study reveals important roles of both Gln270 and Glu276 of AKR1C17 in maintaining the coenzyme specificity. In the AKR1C subfamily, the two residues are strictly conserved in all enzymes which show their preference for NAD(H) over NADP(H) [20,32–36]. A possible structural factor

Table 3  
Alteration of kinetic constants for coenzymes and substrate by mutagenesis of Gln 270 and Glu 276 of AKR1C17  

| Coenzyme | Parameter                     | WT     | Q270K          | E276R          | Q270K/E276R       |
|----------|-------------------------------|--------|----------------|----------------|-------------------|
|          |                              | Value  | Ratio$^a$      | Value          | Ratio$^a$         |
| NAD$^+$  | $K_m$ for NAD$^+$ (mM)        | 0.08   | 0.30           | 1.1            | 14                |
|          | $k_{cat}$ (min$^{-1}$)        | 3.9    | 4.2            | 2.8            | 6.5               |
|          | $k_{cat}/K_m$ (min$^{-1}$mM$^{-1}$) | 49    | 14             | 2.5            | 5.9               |
|          | $K_m$ for THP ($\mu$M)        | 0.3    | 2.8            | 2.8            | 1.0               |
|          | $K_i$ for NADH ($\mu$M)       | 1.5    | 5.4            | 9.0            | 14                |
| NADP$^+$ | $K_m$ for NADP$^+$ (mM)       | 1.1    | 0.20           | 0.022          | 0.003             |
|          | $k_{cat}$ (min$^{-1}$)        | 2.5    | 3.1            | 3.5            | 1.2               |
|          | $k_{cat}/K_m$ (min$^{-1}$mM$^{-1}$) | 2.3    | 16             | 159            | 400               |
|          | $K_m$ for THP ($\mu$M)        | 2.5    | 0.5            | 0.4            | 0.1               |
|          | $K_i$ for NADPH ($\mu$M)      | 20     | 2.4            | 0.47           | 0.01              |

The kinetic constants for NAD(P)$^+$ were determined with 5$\beta$-pregnan-3$\alpha$-ol-20-one (THP) as the substrate. The concentrations in the NAD$^+$-linked activities of the wild-type (WT), Q270K, E276R and Q270K/E276R were 5, 20, 20 and 10 $\mu$M, respectively, and the respective concentrations in the NADP$^+$-linked activities were 20, 8, 5 and 1.0 $\mu$M. The $K_m$ values for the substrate were determined with 1 mM NAD(P)$^+$ as the coenzyme. The $K_i$ values for NAD(P)H were determined in the presence of the above concentrations of THP in the NAD$^+$-linked activities of the enzymes.

$^a$ Mutant/WT.

showed high affinity for various bile acids and was expressed at a much higher level in rat kidney than AKR1C9. Therefore, it is possible that AKR1C17 acts as a renal-specific bile acid-binding protein that facilitates the transport of the reabsorbed bile acids from the apical to basolateral membrane. In addition, AKR1C17 was inhibited by several organic anions with their \( K_i \) values being comparable to the dissociation constants of hepatic organic anion-binding proteins such as glutathione S-transferases [48,49]. The inhibition patterns suggested that the organic anions bind to sites other than the active site of the enzyme. AKR1C17 may also act as an intracellular binding protein for organic anions that are reabsorbed or excreted in the kidney.

### References

[1] T.M. Penning, Endocr. Rev. 18 (1997) 281–305.

[2] R. Rupprecht, F. Holsboer, Trends Neurosci. 22 (1999) 410–416.

[3] F. Labrie, V. Luu-The, S.-X. Lin, C. Labrie, J. Simard, R. Breton, A. Bélanger, Steroids 62 (1997) 148–158.

[4] S. Ishikura, N. Usami, K. Kitahara, T. Isaji, K. Oda, J. Nakagawa, A. Hara, Biochemistry 40 (2001) 214–224.

[5] M.G. Biswas, D.W. Russell, J. Biol. Chem. 272 (1997) 15959–15966.

[6] S.V. Chetyrkin, O.V. Belyaeva, W.H. Gough, N.Y. Kedishvili, J. Biol. Chem. 276 (2001) 22278–22286.

[7] S.V. Chetyrkin, J. Hu, W.H. Gough, N. Dumaual, N.Y. Kedishvili, Arch. Biochem. Biophys. 386 (2001) 1–10.

[8] W.G. Degtigar, N.E. Kushlinsky, Biochemistry (Moscow) 66 (2001) 318–331.

[9] Y. Higaki, T. Kamiya, N. Usami, S. Shintani, H. Shiraishi, S. Ishikura, I. Yamamoto, A. Hara, Drug Metab. Pharmacokin. 17 (2002) 348–356.

[10] S. Ishikura, S. Nakajima, V. Carbone, O. El-Kabbani, A. Hara, in: H. Weiner, B. Plapp, R. Lindahl, E. Maser (Eds.), Enzymology and Molecular Biology of Carbonyl Metabolism 12, Purdue University Press, West Lafayette, 2005, pp. 324–332.

[11] J.E. Pawlowski, M. Huizinga, T.M. Penning, J. Biol. Chem. 266 (1991) 8820–8825.

[12] D. Hyndman, D.R. Bauman, V.V. Heredia, T.M. Penning, Chem. Biol. Interact. 143–144 (2003) 499–525.

[13] T.M. Penning, I. Mukharji, S. Barrows, P. Talalay, Biochem. J. 222 (1984) 601–611.

[14] Y.-T. Hou, W. Xia, J.E. Pawlowski, T.M. Penning, Cancer Res. 54 (1994) 247–255.

[15] T. Matsunaga, S. Shintani, A. Hara, Drug Metab. Pharmacokin. 21 (2006) 1–18.

[16] T.M. Penning, M.E. Burczynski, J.M. Jez, C.-F. Hung, H.-K. Lin, H. Ma, M. Moore, N. Palackal, K. Patnam, Biochem. J. 351 (2000) 67–77.

[17] V. Luu-The, I. Dufort, G. Pelletier, F. Labrie, Mol. Cell. Endocrinol. 171 (2001) 77–81.

[18] S. Ishikura, N. Usami, S. Nakajima, A. Kameyama, H. Shiraishi, V. Carbone, O. El-Kabbani, A. Hara, Biol. Pharm. Bull. 27 (2004) 1939–1945.

[19] K. Matsumoto, S. Endo, S. Ishikura, T. Matsunaga, K. Tajima, O. El-Kabbani, A. Hara, Biol. Pharm. Bull. 29 (2006) 539–542.

[20] J. Mao, W.R. Duan, C.T. Albarracin, T.G. Parmer, G. Gibori, Biochem. Biophys. Res. Commun. 201 (1994) 1289–1295.

[21] S. Ishikura, K. Matsumoto, M. Sanai, K. Horie, T. Matsunaga, K. Tajima, O. El-Kabbani, A. Hara, J. Biochem. (Tokyo) 139 (2006) 1053–1063.

[22] K.-N. Qin, K.-C. Cheng, Biochemistry 33 (1994) 3223–3228.

[23] K. Tajima, M. Hashizaki, K. Yamamoto, T. Mizutani, Drug Metab. Dispos. 20 (1992) 816–820.

[24] K.L. Platt, F. Oesch, Synthesis 7 (1977) 449–450.

[25] M. Nakanishi, Y. Deyashiki, K. Ohshima, A. Hara, Eur. J. Biochem. 228 (1995) 381–387.

[26] M.M. Bradford, Anal. Biochem. 72 (1976) 248–254.

[27] A. Cornish-Bowden, in: Principles of Enzyme Kinetics, Butterworths, London, 1976, pp. 57–60.

[28] H. Sakakura, M. Suzuki, N. Kimura, H. Takeda, S. Nagata, M. Maeda, J. Chromatogr. 621 (1993) 123–131.

[29] I.H. Segel, in: Enzyme Kinetics, John Wiley and Sons, New York, 1975, pp. 474–656.

[30] M.J. Bennett, R.H. Albert, J.M. Jez, H. Ma, T.M. Penning, M. Lewis, Structure 5 (1997) 799–812.

[31] Y. Jin, D.E. Stayrook, R.H. Albert, N.T. Palackal, T.M. Penning, M. Lewis, Biochemistry 40 (2001) 10161–10168.

[32] J.-F. Couture, P. Legrand, L. Cantin, V. Luu-The, F. Labrie, R. Breton, J. Mol. Biol. 331 (2003) 593–604.

[33] S. Ishikura, K. Horie, M. Sanai, K. Matsumoto, A. Hara, Biol. Pharm. Bull. 28 (2005) 1075–1078.

[34] S. Endo, K. Matsumoto, T. Matsunaga, S. Ishikura, K. Tajima, O. El-Kabbani, A. Hara, Biol. Pharm. Bull. 29 (2006) 2488–2492.

[35] Y. Deyashiki, T. Takatsuji, A. Hara, in: T.M. Penning, J.M. Petrush (Eds.), Aldo-keto Reductases and Toxicant Metabolism, American Chemical Society, Washington, DC, 2004, pp. 101–114.

[36] T. Todaka, S. Yamano, S. Toki, Arch. Biochem. Biophys. 374 (2000) 189–197.

[37] C.R. Bellamacina, FASEB J. 10 (1996) 1257–1269.

[38] N. Tanaka, T. Nonaka, T. Tanabe, T. Yoshimoto, D. Tsuru, Y. Mitsui, Biochemistry 35 (1996) 7715–7730.

[39] J. Perozich, I. Kuo, B.C. Wang, J.S. Boesch, R. Lindahl, J. Hempel, Eur. J. Biochem. 267 (2000) 6197–6203.

[40] D. Krasowski, K. Yasuda, L.R. Hagey, E.G. Schuetz, Mol. Endocrinol. 19 (2005) 1720–1739.

[41] M. Makishima, A.Y. Okamoto, J.J. Repa, H. Tu, R.M. Learned, A. Luk, M.V. Hull, K.D. Lustig, D.J. Mangelsdorf, B. Shan, Science 284 (1999) 1362–1365.

[42] M. Souidi, Y. Gueguen, C. Linard, N. Dudoignon, S. Grison, C. Baudelin, C. Marquette, P. Gourmelon, J. Aigueperse, I. Dublineau, Toxicology 214 (2005) 113–122.

[43] D. Ackermann, B. Vogt, G. Escher, B. Dicj, J. Reichen, B.M. Frey, F.J. Frey, Hepatology 30 (1999) 623–629.

[44] F.A. Wilson, G. Burckhardt, H. Murer, G. Rumrich, K.J. Ullrich, J. Clin. Invest. 67 (1981) 1141–1150.

[45] D.M. Christie, P.A. Dawson, S. Thevananther, B.L. Schneider, Am. J. Physiol. 271 (1996) G377–G385.

[46] J.H. Schlattjan, C. Winter, J. Greven, Nephron Physiol. 95 (2003) 49–56.

[47] A. Stolz, H. Takikawa, Y. Sugiyama, J. Kuhlenkamp, N. Kaplowitz, J. Clin. Invest. 79 (1987) 427–434.

[48] Y. Sugiyama, T. Yamada, N. Kaplowitz, J. Biol. Chem. 258 (1983) 3602–3607.

[49] H. Takikawa, Y. Sugiyama, A. Stolz, A. Sugimoto, N. Kaplowitz, Biochem. Pharmacol. 35 (1986) 354–356.
