
A single amino acid in the PSPG-box plays an important role in the catalytic function of CaUGT2 (Curcumin glucosyltransferase), a Group D Family 1 glucosyltransferase from *Catharanthus roseus*

Sayaka Masada, Kazuyoshi Terasaka, Hajime Mizukami*

*Graduate School of Pharmaceutical Sciences, Nagoya City University, Tanabe-dori 3-1, Mizuho-ku, Nagoya 467-8603, Japan*

Received 18 April 2007; revised 1 May 2007; accepted 2 May 2007

Available online 11 May 2007

Edited by Richard Cogdell

---

**Abstract** Curcumin glucosyltransferase (CaUGT2) isolated from cell cultures of *Catharanthus roseus* exhibits unique substrate specificity. To identify amino acids involved in substrate recognition and catalytic activity of CaUGT2, a combination of domain swapping and site-directed mutagenesis was carried out. Exchange of the PSPG-box of CaUGT2 with that of NtGT1b (a phenolic glucosyltransferase from tobacco) led to complete loss of enzyme activity in the resulting recombinant protein. However, replacement of Arg378 of the NtGT1b PSPG-box with cysteine, the corresponding amino acid in CaUGT2, restored the catalytic activity of the chimeric enzyme. Further site-directed mutagenesis revealed that the size of the amino acid side-chain in that particular site is critical to the catalytic activity of CaUGT2.

© 2007 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

**Keywords:** Glucosyltransferase; Curcumin; PSPG-box; Catalytic function; Domain swapping; Site-directed mutagenesis

---

**1. Introduction**

Plants produce a huge number of low molecular weight natural products with diverse structures. The diversity is ensured by both a wide range of basic structures and multiple decorations of a common skeleton through processes such as hydroxylation, methylation, acylation and/or glycosylation. Glycosylation is one of the most widespread among these modifications, and is often the final step in the biosynthesis of plant natural products. The glucosyltransferases (UGTs) that transfer sugar moieties from UDP-activated sugars to low molecular weight acceptors are able to recognize both hydrophilic and lipophilic small molecules as sugar recipients [1–5]. Since conjugation of sugars to endogenous and exogenous lipophilic molecules leads to increased water solubility, improved chemical stability and altered biological activity, UGTs have attracted considerable attention as potential agents for production of glycosides with higher pharmacological values [6,7].

---

UGTs comprise a multigene superfamily with more than 75 families [8–10]; http://afmd.cnrs-mrs.fr/CAZY/acc.html. Plant UGTs capable of transferring sugars to a wide range of small molecules belong to Family 1, and are defined by the presence of a 44 amino acid C-terminal signature motif designated as the PSPG (plant secondary product glucosyltransferases)-box [11]. Biochemical and molecular characterization of plant Family 1 UGTs has shown that they have relatively broad substrate specificity but exert regioselectivity towards sugar acceptors [12,13].

Results from the phylogenetic analysis and comparison of substrate recognition patterns among *Arabidopsis* Family 1 UGTs have shown that there is greater variability within the N-terminal regions of these proteins than in their C-terminal regions, including the PSPG-box, and have indicated that amino acid residues in the N-terminal half of the proteins were responsible for acceptor binding, whereas those in the C-terminal half were involved mainly in interactions with donor substrates [8,10,14–16]. Investigation of the 3D-structures of betanidin 5-O-glucosyltransferase (B5GT) from *Dorotheanthus bellidiformis* [17] and cyanohydrin glucosyltransferase from *Sorghum bicolor* [18] by homology modeling, and of isoflavonoid 3′-O-glucosyltransferase from *Medicago truncatula* [19,20] and flavonoid 3-O-glucosyltransferase from *Vitis vinifera* [21] by X-ray crystallography, revealed the role of specific conserved amino acid residues in the PSPG-box that constitute the donor-sugar binding pockets. However, the roles of less well conserved amino acids within the motif that may determine the characteristics unique to particular enzymes such as substrate recognition and catalytic potential have been less closely examined.

Site-directed mutagenesis is a powerful method for determining the involvement of specific amino acid residues in substrate recognition or catalytic function, and has been applied previously to the UGT family [12,17,18,22]. Although the targets of site-directed mutagenesis are usually highly conserved residues, less conserved amino acids can also be of interest since these may decide the characteristics unique to particular enzymes.

We have recently cloned a cDNA encoding a novel glucosyltransferase, UDP-glucose: curcumin glucosyltransferase (CaUGT2), from *Catharanthus roseus* cultured cells [23]. Recombinant CaUGT2 was shown to catalyze glucose transfer from UDP-glucose to two phenolic hydroxyl groups of curcumin (Fig. 1). Phylogenetic analysis revealed that CaUGT2 belongs to group D of the Family 1 UGTs. In addition, we

---

*Corresponding author. Fax: +81 52 836 3415.  
E-mail address: hajimem@phar.nagoya-cu.ac.jp (H. Mizukami).*

0014-5793/$32.00 © 2007 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.  
doi:10.1016/j.febslet.2007.05.002

were performed with KOD-plus DNA polymerase (Toyobo), the resulting PCR products were mixed together and used as a template for a third PCR reaction, using Taq DNA polymerase (Roche) and the appropriate primers. The resulting fragments were cloned into the pCR2.1-TOPO vector (Invitrogen) and sequenced to ensure that no mutation had occurred during their construction.

### 2.4. Site-directed mutagenesis

Site-directed mutagenesis was performed by the Dpn I-mediated method [26] using plasmids harboring CaUGT2, NtGT1b and their chimeric proteins as templates. In brief, PCR was performed with KOD-plus DNA polymerase using two complementary primers containing the desired mutation. The PCR primers used for the construction of mutants are shown in Table S2 (see the Supplemental Data available with this article online). The amplified plasmids were treated with Dpn I (Roche) to digest the parental cDNAs and the mutated prodigy plasmids were subsequently used to transform into *E. coli* JM109. The presence of the desired point mutation was verified by sequencing.

### 2.5. Protein expression

The coding regions of the wild type, chimeric and mutant cDNAs were subcloned into the expression vector pQE30 (Qiagen) to create N-terminal fusion proteins with a His₆-tag, and transformed into *E. coli* JM109. Transformed bacteria were grown at 37°C in 200 ml LB medium containing 50 μg/ml carbenicillin, and protein expression was induced with 1 mM isopropyl β-D-galactoside (IPTG) at an OD₆₀₀ of 0.6. After incubation at 18°C overnight, the induced cells were harvested by centrifugation and stored at –80°C until use.

The crude enzyme was obtained by sonicating the cell pellet in 50 mM sodium phosphate buffer (pH 8.0) containing 300 mM NaCl and 10 mM imidazole. The recombinant protein was affinity-purified on a Ni-NTA agarose matrix (Qiagen) according to the manufacturer’s instruction. The purity of the enzyme was confirmed by SDS–PAGE on 10% (w/v) slab gels and the protein concentration was determined by the Bradford method [27] using bovine serum albumin as a standard.

### 2.6. Enzyme assay

Enzymatic activity studies were performed with a 50 μL standard reaction mixture containing the lysis buffer (50 mM sodium phosphate buffer [pH 8.0], 300 mM NaCl, and 10 mM imidazole), 0.25 mM curcumin or 1.25 mM 4-nitrophenol, and 5 mM UDP-glucose with 100 μg crude enzyme or 1–5 μg of the purified enzyme. The reaction was incubated at 30°C for 10 min and terminated by adding 100 μl methanol. After centrifugation at 12000 × g for 10 min, the glucosylation products were analyzed by HPLC and the elute monitored by absorbance at 423 nm for curcumin, or 280 nm for 4-nitrophenol [23]. Under these conditions the glucosylation product of curcumin was curcumin monoglucoside and formation of curcumin diglucoside was not detected.

### 2.7. Enzyme kinetics

For determining the kinetic parameters, enzyme assays were performed in duplicate at each substrate concentration with 1–5 μg of the purified enzyme at 30°C for 5 min or 10 min. The substrate concentrations used were 5–500 μM curcumin with UDP-glucose at 5 mM for acceptor kinetics, or 0.01–1 mM UDP-glucose with curcumin at 0.25 mM for donor kinetics. The initial velocity data were visualized by the Lineweaver-Burk plots and kinetic parameters were calculated based on linear regression analysis using Excel 2003 (Microsoft Japan). The experiments were repeated three times.

## 3. Results and discussion

### 3.1. Exchange of the PSPG-box

To examine how the PSPG-box structure affects substrate recognition and catalytic activity of UGTs, we performed domain swapping experiments between two UGTs, CaUGT2 and NtGT1b (Table 1). An initial set of chimeric enzymes was built

S. Masada et al. / FEBS Letters 581 (2007) 2605–2610

Table 1  
Schematic representation and enzyme activities of the chimeric proteins  

| Proteins           | PSPG-box          | Length (aa) | Activity (pkat/mg) $^a$ |
|--------------------|-------------------|-------------|-------------------------|
|                    |                   |             | Curcumin                | 4-Nitrophenol         |
| CaUGT2            | N—[■■■■■■■■]—C   | 488         | 238                     | 99.4                 |
| NtGT1b            | N—[///////]—C     | 480         | NA$^b$                  | 54.3                 |
| Ca-Nt             | N—[■■■■■■■■]—[///////]—C | 479         | NA                      | NA                    |
| Nt-Ca             | N—[///////]—[■■■■■■■■]—C | 489         | NA                      | NA                    |
| Ca-PSPG(Nt)-Ca    | N—[■■■■■■■■]—[///////]—C | 488         | NA                      | NA                    |
| Nt-PSPG(Ca)-Nt    | N—[///////]—[■■■■■■■■]—C | 480         | NA                      | NA                    |

$^a$ Enzyme activities were measured with 5 mM UDP-glucose and either 0.25 mM curcumin or 1.25 mM 4-nitrophenol at 30°C for 10 min.  
$^b$ NA, no detectable activity.

by exchanging homologous peptide fragments derived from  
the region located between the N-terminus and the PSPG-  
box in the two enzymes. Ca-Nt was constructed by combining  
the N-terminal portion from CaUGT2 and the C-terminal por-  
tion (encompassing PSPG-box) from NtGT1b, whereas Nt-Ca  
was constructed by combining the N-terminal portion of  
NtGT1b and the C-terminal portion of CaUGT2. The second  
set of chimeric enzymes was constructed by exchanging  
the PSPG-box-containing-fragments of each enzyme.  
Ca-PSPG(Nt)-Ca was constructed by exchanging PSPG-box  
region of CaUGT2 with that of NtGT1b, and Nt-PSPG(Ca)-  
Nt was constructed by exchanging the PSPG-box region of  
NtGT1b with that of CaUGT2.  
The exchange of PSPG-boxes between CaUGT2 and  
NtGT1b in different formats was a major focus of the present  
domain swapping experiments. The chimeric UGTs were ex-  
pressed at similar levels as the wild type UGTs in E. coli  
JM109 cells, indicating that the domain swapping did not  
cause significant changes in recombinant enzyme expression le-  
vel (data not shown). The wild type recombinant CaUGT2 cat-  
alyzed glucosylation of both curcumin and 4-nitrophenol  
whereas NtGT1b displayed very weak ability to glucosylate  
curcumin. However, none of the chimeric proteins exhibited  
glucosyl transfer activity for either of these two acceptor sub-  
strates. This complete loss of catalytic activity indicates that,  
despite their similarities, the two PSPG-boxes are not cross-  
compatible between the two parental UGTs. Since 30 of the  
44 amino acid residues in the PSPG-boxes were identical be-  
tween CaUGT2 and NtGT1b (Fig. 2a), we hypothesized that  
some of the 14 non-conserved residues were playing a pivotal  
role in defining the catalytic function of these UGTs, perhaps  
through interaction with amino acids in the N-terminal re-  
gions.

PSPG(Nt)-Ca, these eight amino acids were individually ex-  
changed with the corresponding amino acid in the PSPG-box  
of CaUGT2, and the glucosylation activities of the eight mu-  
tated chimeras were assayed using curcumin and 4-nitrophenol  
as substrates. Of the eight mutated chimeras, one with an  
Arg377Cys mutation displayed drastically increased glucosyla-  
tion activity towards both curcumin and 4-nitrophenol (Table  
2). The specific activity of the Arg377Cys chimera towards 4-  
nitrophenol was one-third of that towards curcumin, a ratio  
similar to that of wild type CaUGT2. None of the other single  
amino acid mutants exhibited measurable glucosylation activ-  
ity (Table 2). The result indicates that Cys377 in the PSPG-box  
of CaUGT2 plays an essential role in determining the catalytic  
function of CaUGT2.

Then, we analyzed the relationship between the identity, in  
some group D UGTs (Fig. 2b), of the amino acid residue cor-  
responding to Cys377 in CaUGT2 and the ability of these en-  
zymes to glycosylate curcumin. As shown in Table 3,  
flavonoid-7-O-glucosyltransferase (F7GT) from Scutellaria  
baicalensis [28] with Cys368, betanidin 5-O-glucosyltransferase  
(B5GT) from Dorotheanthus bellidiformis [29] with Ser381,  
UGT73F1 from Glycyrrhiza echinata [30] with Ser370, and  
UGT73C6 from Arabidopsis thaliana [31] with Thr384 as ami-  
no acid residues corresponding to Cys377 in CaUGT2 dis-  
played some enzymatic activity toward curcumin, whereas  
steroidal saponin synthase (SaGT4A) from Solanum aculeatiss-  
imum [32] with Ile380 was unable to catalyze the glucosylation  
of curcumin. Although the $K_m$ value for either curcumin or  
UDP-glucose fluctuated irrespective of the amino acid resi-  
dues, $k_{cat}$ values are higher for the enzymes with Cys than  
for those with Ser or Thr at that position. Based on these  
results, we hypothesized that a thiol or hydroxyl group in  
the amino acid residue corresponding to Cys377 in CaUGT2  
plays a pivotal role, not in the substrate recognition but rather  
in the ability of the enzyme to catalyze glucosyl transfer to cur-  
cumin.

3.2. Reciprocal site-directed mutagenesis  
We next carried out site-directed mutagenesis to identify  
which of the non-conserved amino acids within the PSPG-  
box of CaUGT2 plays an important role in controlling UGT  
function. Eight amino acids unique to either CaUGT2 or  
NtGT1b were selected (Fig. 2a). In the chimeric protein Ca-  

3.3. Site-directed mutagenesis of CaUGT2  
In order to confirm the functional importance of the Cys 377  
residue in CaUGT2, we constructed a series of mutant forms

a

CaUGT2 348WAPQTLILEHESVGAFTHCGWNSTLEGICAGVPLVTWPFFAEQ391  
NtGT1b 349WAPQLAILSHPSVGGFVSHCGWNSTLESVRSGVPIATWPLYAEQ392  

b

![Diagram](attachment:diagram.png)

Fig. 2. (a) Sequence alignment of PSPG-box domains of CaUGT2 and NtGT1b. The amino acids exchanged by site-directed mutagenesis are shown in bold. (b) Molecular phylogenetic relationship of UGTs used in the present investigation. The amino acid sequences were aligned using ClustalX [33]. The tree was constructed by the neighbor-joining method. GenBank accession numbers of the UGTs described in this study are: CaUGT2, AB159213; NtGT1a and 1b, AB052557 and AB052558; F7GT, AB031274; B5GT, Y18871; Glycyrrhiza UGT73F1, AB098614; UGT73C6, AC006282; SaGT4A, AB182385; CaUGT1, AB159212.

Table 2  
Enzymatic activities of the chimeric protein Ca-PSPG(Nt)-Ca and its site-directed mutants

| Proteins          | Specific activity (pkat/mg) * for Curcumin | 4-Nitrophenol |
|-------------------|-------------------------------------------|---------------|
| CaUGT2            | 238                                       | 99.4          |
| Ca-PSPG(Nt)-Ca    | NA b                                      | NA            |
| Leu352Thr         | NA                                        | NA            |
| Ser356Glu         | NA                                        | NA            |
| Ser375Gly         | NA                                        | NA            |
| Arg377Cys         | 124                                       | 41.1          |
| Ser378Ala         | NA                                        | NA            |
| Leu387Phe         | NA                                        | NA            |
| Tyr388Phe         | NA                                        | NA            |

* Enzyme activities were measured with 5mM UDP-glucose and either 0.25mM curcumin or 1.25mM 4-nitrophenol at 30°C for 10 min.
** NA, no detectable activity.

The replacement of Cys377 with either Gly or Ala did not significantly affect \( k_{\text{cat}} \), contrary to our hypothesis that a thiol or hydroxyl group on the amino acid residue corresponding to Cys377 plays an important role in the catalytic function of CaUGT2. Cys377Asp, Cys377Asn, and Cys377Tyr mutations led to a 3–6-fold decrease in \( k_{\text{cat}} \) values, whereas the \( K_{\text{m}} \) values for UDP-glucose were only slightly increased in these forms compared with that of the wild type CaUGT2. Cys377Lys and Cys377Arg mutations led to more drastic decreases in the \( k_{\text{cat}} \) values, resulting in about a 25-fold decrease in \( k_{\text{cat}}/K_{\text{m}} \) ratios, and the Cys377Trp mutant completely lost catalytic function.

These results clearly indicate that the size, rather than the physicochemical properties, of the side chain of the particular amino acid at the site corresponding to Cys377 plays an important role in catalytic activity of glucosyl transfer in CaUGT2. The more bulky the side chain was, the less active the enzyme was. The result that the amino acid displacement did not significantly affect the \( K_{\text{m}} \) value for curcumin is consistent with the idea that the side chain of the amino acid at position 377 does not directly interact with the aglycone substrate but is involved in the catalytic process.

of CaUGT2 in which Cys377 was replaced with various amino acids, and compared the kinetic parameters of the recombinant proteins in curcumin glucosylation (Table 4).

All the mutant CaUGT2 except the Cys377Trp form exhibited \( K_{\text{m}} \) values for curcumin similar to that of the wild type CaUGT2 (44 μM), ranging from 22 μM in the Cy-

S. Masada et al. / FEBS Letters 581 (2007) 2605–2610

Table 3  
Kinetic parameters for curcumin glucosylation by UGTs closely related to CaUGT2  

| UGT<sup>b</sup>       | Amino acid corresponding to Cys377 in CaUGT2 | Kinetic parameters<sup>a</sup>          |
|----------------------|-------------------------------------------|----------------------------------------|
|                      |                                           | Curcumin                              | UDP-glucose                          |
|                      |                                           | $K_{\mathrm{m}}(\mu \mathrm{M})$      | $k_{\mathrm{cat}}(s^{-1})$           | $k_{\mathrm{cat}}/K_{\mathrm{m}}\left(M^{-1} s^{-1}\right)$ | $K_{\mathrm{m}}(\mu \mathrm{M})$ |
| CaUGT2              | Cysteine                                 | 43.9                                  | 0.0165                               | 361                                   | 317                                |
| F7GT                | Cysteine                                 | 134                                   | 0.0500                               | 377                                   | 315                                |
| B5GT                | Serine                                   | 14.6                                  | 0.00055                              | 38.8                                  | 2573                               |
| UGT73F1             | Serine                                   | 22.3                                  | 0.00446                              | 212                                   | 1929                               |
| UGT73C6             | Threonine                                | 14.8                                  | 0.00160                              | 118                                   | 836                                |
| SaGT4A              | Isoleucine                               | NA<sup>c</sup>                        |                                      |                                       |                                    |

<sup>a</sup> Average values from triplicate measurements. Each measurement is comprised of duplicate reactions.  
<sup>b</sup> Enzyme names: F7GT, baicalein-7-O-glucosyltransferase from *Scutellaria baicalensis*; B5GT, betanidin-5-O-glucosyltransferase from *Dorotheanthus bellidiformis*; UGT73F1, isoflavonoid glucosyltransferase from *Glycyrrhiza echinata*; UGT73C6, flavonol-7-O-glucosyltransferase from *Arabidopsis thaliana*; SaGT4a, steroidal saponin-3-O-glucosyltransferase from *Solanum aculeatissimum*.  
<sup>c</sup> NA, no detectable activity.

---

Table 4  
Kinetic parameters of the wild type and site-directed mutants of CaUGT2  

| Proteins            | Kinetic parameters<sup>a</sup>          |
|---------------------|----------------------------------------|
|                     | Curcumin                              | UDP-glucose                          |
|                     | $K_{\mathrm{m}}(\mu \mathrm{M})$      | $k_{\mathrm{cat}}(s^{-1})$           | $k_{\mathrm{cat}}/K_{\mathrm{m}}\left(M^{-1} s^{-1}\right)$ | $K_{\mathrm{m}}(\mu \mathrm{M})$ |
| Wild type          | 43.9                                  | 0.0165                               | 361                                   | 317                                |
| Cys377Gly          | 40.5                                  | 0.0115                               | 312                                   | 586                                |
| Cys377Ala          | 30.1                                  | 0.0137                               | 485                                   | 521                                |
| Cys377Asn          | 66.7                                  | 0.0055                               | 83.4                                  | 731                                |
| Cys377Asp          | 45.2                                  | 0.0024                               | 57.1                                  | 668                                |
| Cys377Tyr          | 50.1                                  | 0.0022                               | 46.9                                  | 839                                |
| Cys377Lys          | 35.3                                  | 0.0005                               | 13.7                                  | >2500                              |
| Cys377Arg          | 22.2                                  | 0.0003                               | 15.5                                  | >2500                              |
| Cys377Trp          | NA<sup>b</sup>                        |                                      |                                       |                                    |

<sup>a</sup> Average values from triplicate measurements. Each measurement is comprised of duplicate reactions.  
<sup>b</sup> NA, no detectable activity.

References

[1] Vogt, T. and Jones, P. (2000) Glycosyltransferases in plant natural product synthesis: characterization of a supergene family. Trends Plant Sci. 5, 380–386.

[2] Jones, P. and Vogt, T. (2001) Glycosyltransferases in secondary plant metabolism: tranquilizers and stimulant controllers. Planta 213, 164–174.

[3] Lim, E.K. and Bowles, D.J. (2004) A class of plant glycosyltransferases involved in cellular homeostasis. Embo J. 23, 2915–2922.

[4] Gachon, C.M., Langlois-Meurinne, M. and Saindrenan, P. (2005) Plant secondary metabolism glycosyltransferases: the emerging functional analysis. Trends Plant Sci. 10, 542–549.

[5] Bowles, D., Lim, E.K., Poppenberger, B. and Vaistij, F.E. (2006) Glycosyltransferases of lipophilic small molecules. Annu. Rev. Plant Biol. 57, 567–597.

[6] Bowles, D., Isayenкова, J., Lim, E.K. and Poppenberger, B. (2005) Glycosyltransferases: managers of small molecules. Curr. Opin. Plant Biol. 8, 254–263.

[7] Lim, E.K. (2005) Plant glycosyltransferases: their potential as novel biocatalysts. Chemistry 11, 5486–5494.

[8] Li, Y., Baldauf, S., Lim, E.K. and Bowles, D.J. (2001) Phylogenetic analysis of the UDP-glycosyltransferase multigene family of *Arabidopsis thaliana*. J. Biol. Chem. 276, 4338–4343.

[9] Coutinho, P.M., Deleury, E., Davies, G.J. and Henrissat, B. (2003) An evolving hierarchical family classification for glycosyltransferases. J. Mol. Biol. 328, 307–317.

[10] Lim, E.K., Baldauf, S., Li, Y., Elias, L., Worrall, D., Spencer, S.P., Jackson, R.G., Taguchi, G., Ross, J. and Bowles, D.J. (2003) Evolution of substrate recognition across a multigene family of glycosyltransferases in *Arabidopsis*. Glycobiology 13, 139–145.

[11] Hughes, J. and Hughes, M.A. (1994) Multiple secondary plant product UDP-glucose glucosyltransferase genes expressed in cassava (*Manihot esculenta* Crantz) cotyledons. DNA Seq. 5, 41–49.

[12] Kubo, A., Arai, Y., Nagashima, S. and Yoshikawa, T. (2004) Alteration of sugar donor specificities of plant glycosyltransferases by a single point mutation. Arch. Biochem. Biophys. 429, 198–203.

[13] Isayenкова, J., Wray, V., Nimtz, M., Strack, D. and Vogt, T. (2006) Cloning and functional characterisation of two regioselective flavonoid glucosyltransferases from *Beta vulgaris*. Phytochemistry 67, 1598–1612.

[14] Ross, J., Li, Y., Lim, E. and Bowles, D.J. (2001) Higher plant glycosyltransferases. Genome Biol. 2, REVIEWS3004.

[15] Hu, Y. and Walker, S. (2002) Remarkable structural similarities between diverse glycosyltransferases. Chem. Biol. 9, 1287–1296.

[16] Paquette, S., Möller, B.L. and Bak, S. (2003) On the origin of family 1 plant glycosyltransferases. Phytochemistry 62, 399–413.

[17] Hans, J., Brandt, W. and Vogt, T. (2004) Site-directed mutagenesis and protein 3D-homology modelling suggest a catalytic mechanism for UDP-glucose-dependent betanidin 5-O-glucosyltransferase from *Dorotheanthus bellidiformis*. Plant J. 39, 319–333.

[18] Thorsøe, K.S., Bak, S., Olsen, C.E., Imberty, A., Breton, C. and Lindberg Møller, B. (2005) Determination of catalytic key amino acids and UDP sugar donor specificity of the cyanohydrin glycosyltransferase UGT85B1 from *Sorghum bicolor*. Molecular modeling substantiated by site-specific mutagenesis and biochemical analyses. Plant Physiol. 139, 664–673.

[19] Shao, H., He, X., Achnine, L., Blount, J.W., Dixon, R.A. and Wang, X. (2005) Crystal structures of a multifunctional triterpene/flavonoid glycosyltransferase from *Medicago truncatula*. Plant Cell. 17, 3141–3154.

[20] He, X.Z., Wang, X. and Dixon, R.A. (2006) Mutational analysis of the *Medicago* glycosyltransferase UGT71G1 reveals residues that control regioselectivity for (iso)flavonoid glycosylation. J. Biol. Chem. 281, 34441–34447.

[21] Offen, W., Martinez-Fleites, C., Yang, M., Kiat-Lim, E., Davis, B.G., Tarling, C.A., Ford, C.M., Bowles, D.J. and Davies, G.J. (2006) Structure of a flavonoid glucosyltransferase reveals the basis for plant natural product modification. Embo J. 25, 1396–1405.

[22] Hefner, T. and Stöckigt, J. (2003) Probing suggested catalytic domains of glycosyltransferases by site-directed mutagenesis. Eur. J. Biochem. 270, 533–538.

[23] Kaminaga, Y., Sahin, F.P. and Mizukami, H. (2004) Molecular cloning and characterization of a glucosyltransferase catalyzing glucosylation of curcumin in cultured *Catharanthus roseus* cells. FEBS Lett. 567, 197–202.

[24] Taguchi, G., Yazawa, T., Hayashida, N. and Okazaki, M. (2001) Molecular cloning and heterologous expression of novel glucosyltransferases from tobacco cultured cells that have broad substrate specificity and are induced by salicylic acid and auxin. Eur. J. Biochem. 268, 4086–4094.

[25] Higuchi, R., Krummel, B. and Saiki, R.K. (1988) A general method of in vitro preparation and specific mutagenesis of DNA fragments: study of protein and DNA interactions. Nucleic Acids Res. 16, 7351–7367.

[26] Shenoy, A.R. and Visweswariah, S.S. (2003) Site-directed mutagenesis using a single mutagenic oligonucleotide and *DpnI* digestion of template DNA. Anal Biochem. 319, 335–336.

[27] Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72, 248–254.

[28] Hirotani, M., Kuroda, R., Suzuki, H. and Yoshikawa, T. (2000) Cloning and expression of UDP-glucose: flavonoid 7-O-glucosyltransferase from hairy root cultures of *Scutellaria baicalensis*. Planta 210, 1006–1013.

[29] Vogt, T., Grimm, R. and Strack, D. (1999) Cloning and expression of a cDNA encoding betanidin 5-O-glucosyltransferase, a betanidin- and flavonoid-specific enzyme with high homology to inducible glucosyltransferases from the Solanaceae. Plant J. 19, 509–519.

[30] Nagashima, S., Inagaki, R., Kubo, A., Hirotani, M. and Yoshikawa, T. (2004) cDNA cloning and expression of isoflavonoid-specific glucosyltransferase from *Glycyrrhiza echinata* cell-suspension cultures. Planta 218, 456–459.

[31] Jones, P., Messner, B., Nakajima, J., Schaffner, A.R. and Saito, K. (2003) UGT73C6 and UGT78D1, glycosyltransferases involved in flavonol glycoside biosynthesis in *Arabidopsis thaliana*. J. Biol. Chem. 278, 43910–43918.

[32] Kohara, A., Nakajima, C., Hashimoto, K., Ikenaga, T., Tanaka, H., Shoyama, Y., Yoshida, S. and Muranaka, T. (2005) A novel glucosyltransferase involved in steroid saponin biosynthesis in *Solanum aculeatissimum*. Plant Mol. Biol. 57, 225–239.

[33] Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F. and Higgins, D.G. (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res. 25, 4876–4882.
