
Enzyme Catalysis and Regulation:  
Osmotically Induced Synthesis of the  
Compatible Solute Hydroxyectoine Is  
Mediated by an Evolutionarily Conserved  
Ectoine Hydroxylase  

Jan Bursy, Antonio J. Pierik, Nathalie Pica  
and Erhard Bremer  
J. Biol. Chem. 2007, 282:31147-31155.  
doi: 10.1074/jbc.M704023200 originally published online July 18, 2007  

Access the most updated version of this article at doi: [10.1074/jbc.M704023200](http://dx.doi.org/10.1074/jbc.M704023200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites.xhtml).  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

Click here to choose from all of JBC's e-mail alerts  

Supplemental material:  
[http://www.jbc.org/content/suppl/2007/07/20/M704023200.DC1.html](http://www.jbc.org/content/suppl/2007/07/20/M704023200.DC1.html)  

This article cites 54 references, 25 of which can be accessed free at  
[http://www.jbc.org/content/282/43/31147.full.html#ref-list-1](http://www.jbc.org/content/282/43/31147.full.html#ref-list-1)

Osmotically Induced Synthesis of the Compatible Solute Hydroxyectoine Is Mediated by an Evolutionarily Conserved Ectoine Hydroxylase*

Received for publication, May 16, 2007, and in revised form, July 18, 2007 Published, JBC Papers in Press, July 18, 2007, DOI 10.1074/jbc.M704023200

Jan Bursy, Antonio J. Pierik, Nathalie Pica, and Erhard Bremer¹

From the Laboratory for Microbiology, Department of Biology, Philipps-University Marburg, D-35032 Marburg, Germany

---

By using natural abundance ¹³C NMR spectroscopy, we investigated the types of compatible solutes synthesized in a variety of Bacilli under high salinity growth conditions. Glutamate, proline, and ectoine were the dominant compatible solutes synthesized by the various *Bacillus* species. The majority of the inspected *Bacilli* produced the tetrahydropyrimidine ectoine in response to high salinity stress, and a subset of these also synthesized a hydroxylation derivative of ectoine, 5-hydroxyectoine. In *Salibacillus salexigenes*, a representative of the ectoine- and 5-hydroxyectoine-producing species, ectoine production was linearly correlated with the salinity of the growth medium and dependent on an *ectABC* biosynthetic operon. The formation of 5-hydroxyectoine was primarily a stationary growth phase phenomenon. The enzyme responsible for ectoine hydroxylation (*EctD*) was purified from *S. salexigenes* to apparent homogeneity. The *EctD* protein was shown *in vitro* to directly hydroxylate ectoine in a reaction dependent on iron(II), molecular oxygen, and 2-oxoglutarate. We identified the structural gene (*ectD*) for the ectoine hydroxylase in *S. salexigenes*. Northern blot analysis showed that the transcript levels of the *ectABC* and *ectD* genes increased as a function of salinity. Many *EctD*-related proteins can be found in data base searches in various *Bacteria*. Each of these bacterial species also contains an *ectABC* ectoine biosynthetic gene cluster, suggesting that 5-hydroxyectoine biosynthesis strictly depends on the prior synthesis of ectoine. Our data base searches and the biochemical characterization of the *EctD* protein from *S. salexigenes* suggest that the *EctD*-related ectoine hydroxylases are members of a new subfamily within the non-heme-containing, iron(II)-and 2-oxoglutarate-dependent dioxygenase superfamily (EC 1.14.11).

To avoid excessive water efflux, plasmolysis, molecular crowding of the cytoplasm, and cessation of growth in high salinity environments (1, 2), many *Bacteria* (3–5) and *Archaea* (6, 7) amass large amounts of a particular class of highly soluble organic osmolytes, the so-called compatible solutes (4, 8–10). Important examples of compatible solutes within the domain of the *Bacteria* are the imino acid proline, the disaccharide trehalose, the trimethylammonium compound glycine betaine, and the tetrahydropyrimidine ectoine (4, 10, 11).

Ectoine ((S)-2-methyl-1,4,5,6-tetrahydropyrimidine-4-carboxylic acid) (Fig. 1) was originally found as a compatible solute in the extremely halophilic phototrophic purple sulfur bacterium *Ectothiorhodospira halochloris* (12). Its discovery was soon followed by the description of its hydroxylated derivative, 5-hydroxyectoine ((S,S)-2-methyl-5-hydroxy-1,4,5,6-tetrahydropyrimidine-4-carboxylic acid) (Fig. 1) in the Gram-positive soil bacterium *Streptomyces parvulus* (13). Ectoine was initially viewed as a rather uncommon compatible solute, but it is now known to be widely produced in response to high salinity within a taxonomically and physiologically diverse set of species within the domain of the *Bacteria* (4, 10, 14, 15).

Molecular analysis of the ectoine biosynthetic genes in various Gram-negative and Gram-positive *Bacteria* has shown that the ectoine biosynthetic enzymes are encoded by an evolutionarily conserved gene cluster, *ectABC*(14–16). Disruption of these genes in *Halomonas elongata*, *Chromohalobacter salexigenes*, or *Vibrio cholerae* results in the loss of the ectoine biosynthetic capacity (17–19). Ectoine biosynthesis is mediated by a three-step enzymatic reaction (16, 20) that converts the precursor L-aspartate-β-semialdehyde, an intermediate in amino acid metabolism, into ectoine; L-2,4-diaminobutyrate and *N*′-acetyl-L2,4-diaminobutyrate are the intermediates in this process (Fig. 1). The ectoine biosynthetic enzymes (*EctABC*) from *H. elongata* have been purified and characterized biochemically (21).

The ability to synthesize ectoine is widespread within the domain of the *Bacteria* (15), but only a subset of the ectoine producers also synthesizes the hydroxylation derivative, 5-hydroxyectoine (4, 14). Like ectoine, 5-hydroxyectoine serves as a compatible solute *in vivo*, functions as an osmoprotectant, and exhibits protein stabilizing properties *in vitro* (22–25). The ability of a given microorganism to synthesize 5-hydroxyectoine invariably depends on its ability to produce ectoine, suggesting that 5-hydroxyectoine formation occurs either directly from ectoine (26, 27) or from one of its biosynthetic intermediates (17). The most straightforward route to produce 5-hydroxyectoine would be the direct hydroxylation of ectoine via a

* This work was supported by Deutsche Forschungsgemeinschaft Grant SFB 395, the Graduiertenkolleg “Proteinfunktion auf atomarer Ebene,” the Bundesministerium für Bildung und Forschung through the “Genom Netzwerk Göttingen,” European Union Contract IAC-CT-2000-30041, the Fonds der Chemischen Industrie, and the Max Planck Institute for Terrestrial Microbiology (Marburg). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The on-line version of this article (available at http://www.jbc.org) contains supplemental “Experimental Procedures” and additional references.
¹ To whom correspondence should be addressed: Laboratory for Microbiology, Dept. of Biology, Philipps-University Marburg, Karl-von-Frisch-Str., D-35032 Marburg, Germany. Tel.: 49-6421-2821529; Fax: 49-6421-2828979; E-mail: bremer@staff.uni-marburg.de.

# Ectoine Hydroxylase from S. salexigenes

![Diagram](attachment:diagram.png)

FIGURE 1. Pathway for the biosynthesis of ectoine and 5-hydroxyectoine. The biosynthetic route for the formation of the compatible solute ectoine from L-aspartate-β-semialdehyde is shown. This pathway is based on proposals made by Louis and Galinski (16) and Ono *et al.* (21). The enzymes responsible for ectoine biosynthesis are the L-2,4-diaminobutyrat transaminase (EctB), the L-2,4-diaminobutyrate acetyltransferase (EctA), and the ectoine synthase (EctC). The suggested pathway for the enzymatic conversion of ectoine into 5-hydroxyectoine by the ectoine hydroxylase (EctD protein) from *S. salexigenes* is based on the data presented in this study. The proposed enzymatic reaction scheme for the EctD ectoine hydroxylase is shown in the enlarged part of the figure.

substrate-specific hydroxylase (Fig. 1). Here we report the biochemical characterization of such an enzyme from the moderate halophile *Salibacillus salexigenes*.

## EXPERIMENTAL PROCEDURES

### Chemicals—Ectoine ((S)-2-methyl-1,4,5,6-tetrahydropyrimidine-4-carboxylic acid) and 5-hydroxyectoine ((S,S)-2-methyl-5-hydroxy-1,4,5,6-tetrahydropyrimidine-4-carboxylic acid) were purchased from Biomol (Hamburg, Germany). L-Proline, 3-hydroxy-DL-proline, and 4-hydroxy-L-proline were obtained from Sigma-Aldrich. Two synthetic ectoine derivatives, homoectoine ((S)-4,5,6,7-tetrahydro-2-methyl-1H-(1, 3)-diazepine-4-carboxylic acid) (28) and DHMICA ((RS)-4,5-dihydro-2-methyl-imidazole-4-carboxylic acid) were kindly provided by E. A. Galinski (University of Bonn, Bonn, Germany). Anhydrotetracycline hydrochloride and d-desthiobiotin were purchased from IBA (Göttingen, Germany).

### High Pressure Liquid Chromatography Analysis of Ectoine and 5-Hydroxyectoine from Cell Extracts—Cultures (80 ml) of *S. salexigenes* (DSM 11483<sup>T</sup>) were grown in a shaking water bath (220 rpm) in mineral salt-based medium (29) (for details, see supplemental material) with the indicated NaCl concentrations until the cultures reached an \(A_{578}\) of approximately 1. The cells were harvested by centrifugation (10 min; 4 °C; 2800 × g) and lyophilized, the dry weight of the cells was determined, and the cells were then extracted using a modified Bligh and Dyer technique (30). The ectoine and 5-hydroxyectoine content of the cells was measured by HPLC<sup>2</sup> analysis using a GROM-SIL 100

---

<sup>2</sup>The abbreviations used are: HPLC, high pressure liquid chromatography; TES, N-tris(hydroxymethyl)methyl-2-aminoethanesulfonic acid.

**31148 JOURNAL OF BIOLOGICAL CHEMISTRY**  
VOLUME 282 • NUMBER 43 • OCTOBER 26, 2007

Ectoine Hydroxylase from S. salexitgens

**TABLE 1**

${}^{13}$C NMR determination of the organic osmolytes synthesized by Bacilli strains and strains from related genera

The strains were grown in Spizizen's minimal medium (SMM) with glucose as a unique source of carbon and energy and supplemented with trace elements, vitamins, and amino acids when necessary. For alkaliphilic strains, the medium was supplemented with 100 mM sodium sesquicarbonate.

| Strain | DSM number | Medium (NaCl molarity) | Glutamate | Proline | Ectoine | Hydroxyectoine | Others |
|--------|------------|-----------------------|-----------|---------|---------|----------------|---------|
| Aneurinibacillus aneurinilyticus$^a$ | DSM 5562$^T$ | SMM (0.5 M) | + | – | – | – | – |
| Bacillus cereus$^a$ | DSM 31$^T$ | SMM (0.5 M) | + | – | – | – | – |
| Bacillus circulans$^a$ | DSM 11$^T$ | SMM (0.5 M) | + | – | – | – | – |
| Bacillus thuringiensis$^a$ | DSM 2046$^T$ | SMM (0.5 M) | + | – | – | – | – |
| Sporosarcina globispora | DSM 4$^T$ | SMM (0.2 M) | + | – | – | – | – |
| Paenibacillus polymyxa$^a$ | DSM 36$^T$ | SMM (0.5 M) | + | – | – | – | trehalose |
| Bacillus amyloliquefaciens | FZB 42 | SMM (1.0 M) | + | + | – | – | – |
| Bacillus licheniformis$^a$ | DSM 13$^T$ | SMM (1.0 M) | + | + | – | – | – |
| Bacillus megaterium$^a$ | DSM 32$^T$ | SMM (1.0 M) | + | + | – | – | – |
| B. subtilis$^a$ | JH642 | SMM (1.0 M) | + | + | – | – | – |
| Bacillus vallismortis | DSM 11031$^T$ | SMM (1.8 M) | + | + | – | – | – |
| Bacillus agaradhaerens | DSM 8721$^T$ | SMM (1.3 M) | + | – | + | – | β-glutamate |
| Bacillus alcalophilus$^a$ | DSM 485$^T$ | SMM (1.0 M) | + | – | + | – | – |
| Bacillus halodurans | DSM 497$^T$ | SMM (1.6 M) | + | – | + | – | β-glutamate |
| Bacillus pseudalcaliphilus | DSM 8725$^T$ | SMM (1.0 M) | + | – | + | – | – |
| Sporosarcina psychrophila$^a$ | DSM 3$^T$ | SMM (1.0 M) | + | – | + | – | – |
| Sporosarcina pasteurii$^{a,b}$ | DSM 33$^T$ | SMM (1.0 M) | + | – | + | – | – |
| Bacillus clarkii | DSM 8720$^T$ | SMM (1.8 M) | + | – | + | + | β-glutamate |
| Gracilibacillus halotolerans | DSM 11805$^T$ | SMM (2.0 M) | + | – | + | + | – |
| Salibacillus marismortui | DSM 12325$^T$ | SMM (1.8 M) | + | – | + | + | – |
| Salibacillus salexitgens$^{a,c}$ | DSM 11483$^T$ | MM (3.4 M) | + | – | + | + | – |
| Bacillus mojavensis | DSM 9205$^T$ | SMM (1.5 M) | + | + | + | – | – |
| Bacillus pseudofirmus | DSM 8715$^T$ | SMM (1.6 M) | + | + | + | – | alanine |
| Halobacillus halophilus$^d$ | DSM 2266$^T$ | 802a (2.0 M) | + | + | + | – | N-ε-acetyl lysine |
| Halobacillus trueperi$^d$ | DSM 10404$^T$ | 802a (2.5 M) | + | + | + | – | N-ε-acetyl lysine |
| Virgibacillus pantothenticus$^a$ | DSM 26$^T$ | SMM (1.0 M) | + | + | + | – | – |

$a$ Compatible solute production of these strains were published by Kuhlmann and Bremer (14).

$b$ S. pasteurii was grown in SMM supplemented with 2% urea.

$c$ S. salexitgens was grown in a mineral salt-based medium (MM) supplemented with glucose, casamino acids, and vitamins.

$d$ H. halophilus and H. trueperi were grown in a minimal medium with glucose as source of carbon and energy and supplemented with trace elements, vitamins, and amino acids.

Ectoine Hydroxylase from S. salexigenes

![Figure 2](#fig2)

FIGURE 2. HPLC determination of ectoine and 5-hydroxyectoine content in salt-stressed *S. salexigenes* cells. Cells of *S. salexigenes* were grown at 37°C in a mineral salt-based medium supplemented with casamino acids, various vitamins, and glucose as the carbon source. Ectoine and 5-hydroxyectoine content of the cells was quantitated by HPLC analysis. A, cultures of *S. salexigenes* were grown in minimal medium with various salt concentrations. When the cultures reached the mid-exponential growth phase (*A*<sub>578</sub> = ~1), they were harvested by centrifugation and lyophilized. The soluble compounds were then extracted, and the ectoine and 5-hydroxyectoine content of the cells was determined by HPLC analysis. Closed diamonds, ectoine; open diamonds, 5-hydroxyectoine. B, a culture of *S. salexigenes* was grown in mineral salt-based medium with 3.2 M NaCl, and the growth of the culture was monitored by measuring the *A*<sub>578</sub> (gray triangles). The ectoine (closed diamonds) and 5-hydroxyectoine (open diamonds) content of the cells was quantitated by HPLC analysis.

The 15 ectoine producers can be further subgrouped into three different classes: (i) those species that produce ectoine alone (6 of 15); (ii) those species that produce ectoine and 5-hydroxyectoine (4 of 15); and (iii) those species that produce both ectoine and proline (5 of 15). Osmoregulatory ectoine production is therefore a widespread trait within the genus *Bacillus* and taxonomically closely related species.

Finely Tuned and Osmotically Controlled Ectoine and 5-Hydroxyectoine Biosynthesis in *S. salexigenes*—The main focus of this work is the elucidation of 5-hydroxyectoine formation in osmotically stressed *Bacilli*. We therefore chose *S. salexigenes* as a representative of the four ectoine and 5-hydroxyectoine producers discovered thus far within the *Bacilli* (Table 1). *S. salexigenes* (29) is a moderate halophile and soil bacterium that can grow over a wide range of salinities (between 0.1 and 3.4 M NaCl). This species has recently been reclassified, and the new name *Virgibacillus salexigenes* has been suggested (36).

To analyze the correlation between the salinity of the growth medium and the level of ectoine and 5-hydroxyectoine production in *S. salexigenes*, we grew this strain in minimal media of different salinities (0.1–2.0 M NaCl). We then quantitated the produced ectoine and 5-hydroxyectoine by HPLC analysis when the cultures had reached approximately the same optical density (*A*<sub>578</sub> = ~1). We found an essentially linear relationship between the ectoine content of the cells and the salinity of the growth medium over a wide range of osmotic conditions in these exponentially growing *S. salexigenes* cultures (Fig. 2A). It is thus apparent that the cells sensitively adjust their ectoine content to the prevalent salinity of the growth medium to maintain a physiological appropriate level of cell water and consequently turgor.

Although the exponentially growing cultures of *S. salexigenes* contained a considerable amount of ectoine when the cells were stressed with various concentrations of NaCl, we found that the amount of 5-hydroxyectoine was negligible in these cultures (Fig. 2A).

Our <sup>13</sup>C NMR experiments with *S. salexigenes* revealed that 5-hydroxyectoine production in osmotically challenged cells occurred primarily when the cultures entered the stationary growth phase. To investigate this in greater detail, we grew *S. salexigenes* in a minimal medium with 3.2 M NaCl and monitored ectoine and 5-hydroxyectoine production along the growth curve of the culture by HPLC analysis. Increased ectoine biosynthesis occurred as soon as the cells started to grow in this high osmolality medium, but there was no immediate 5-hydroxyectoine production. Appreciable amounts of 5-hydroxyectoine were made only when cell growth slowed and the culture entered stationary growth phase. Production of this compatible solute reached its maximum when the cells had stopped their growth (Fig. 2B). In these stationary growth phase cultures, we always found a mixture of both ectoine and 5-hydroxyectoine (Fig. 2B), indicating that the cell converted only part of the *de novo* synthesized ectoine into the hydroxylation product.

We examined a second ectoine and 5-hydroxyectoine producer, *Bacillus clarkii* (Table 1), for the pattern of ectoine and 5-hydroxyectoine synthesis in osmotically challenged exponential phase and stationary phase cultures. *B. clarkii* showed the same pattern of ectoine and 5-hydroxyectoine production as that described in Fig. 2 for *S. salexigenes.*<sup>3</sup> The other two 5-hydroxyectoine producers, *Gracilibacillus halotolerans* and *Salibacillus marismortui*, were not investigated for the pattern of ectoine and 5-hydroxyectoine production.

Purification and Characterization of the Ectoine Hydroxylase—We took a biochemical approach to characterize the ectoine hydroxylase from *S. salexigenes*. Kuhlmann and Bremer (14) have previously noted that the *ectABC* gene cluster in *Streptomyces coelicolor* A3 (2) (37) is immediately followed by a gene that encodes a protein with limited sequence identity (27%) to a L-proline 4-hydroxylase from *Dactylosporangium* sp (38). This observation suggested to us that the function of this gene product from *S. coelicolor* A3 (2) might have been incorrectly annotated during the genome sequencing project and may actually serve as an ectoine hydroxylase.

We developed an enzymatic assay for ectoine hydroxylase activity that used assay conditions similar to those employed for the biochemically characterized L-proline 4-hydroxylase from *Dactylosporangium* sp. strain RH1 (38), a L-proline 3-hydroxylase from the *Streptomyces* sp. strain TH1 (39), and a L-proline 4-hydroxylase from the *Streptomyces griseoviridis* strain P8648 (40). Each of these L-proline hydroxylases are members of the non-heme-containing, iron(II)- and 2-oxoglutarate-dependent dioxygenase superfamily (EC 1.14.11.2) (41–43). This ectoine hydroxylase activity assay enabled us to detect ectoine hydrox-

<sup>3</sup> N. Pica and E. Bremer, unpublished results.

ylase enzyme activity in total cell extracts prepared from stationary growth phase cultures of *S. salexigen* propagated in a minimal medium with 3.2 M NaCl (Table 2), growth conditions that allow the effective formation of 5-hydroxyectoine (Fig. 2B).

Using this ectoine hydroxylation assay, we purified the ectoine hydroxylase from *S. salexigen* to apparent homogeneity (Fig. 3A) via several chromatographic steps. Data from a typical purification of the *S. salexigen* ectoine hydroxylase are summarized in Table 2. The purified protein effectively converted ectoine into 5-hydroxyectoine in a time-dependent fashion (Fig. 3B). In the following, we refer to this ectoine hydroxylase from *S. salexigen* as the EctD protein.

**Properties of the Purified Ectoine Hydroxylase—** The ectoine hydroxylase from *S. salexigen* migrates on a SDS-12.5% polyacrylamide gel as a polypeptide with an apparent molecular mass of 34 kDa (Fig. 3A). It behaves in a gel filtration chromatography step on Superdex 75 as a protein species with an approximately molecular mass of 36 kDa, indicating that the EctD protein from *S. salexigen* is a monomer. The precise molecular mass of the purified EctD protein was determined by mass spectrometry to be 34.4 kDa. The NH₂-terminal end of the purified EctD protein was determined by sequential Edman degradation, and the following amino acid sequence was found: NH₂-M-E-D-L-Y-P-S-R-Q-N-N-Q-P-K-I.

Without the addition of the co-substrate 2-oxoglutarate, there was no hydroxylation of ectoine by the purified EctD protein. Likewise, molecular oxygen was absolutely required for the *in vitro* hydroxylation reaction, because the enzyme was completely inactive under anoxic assay conditions. The enzyme activity of EctD was stimulated by FeSO₄ up to a concentration of 1 mM, whereas higher FeSO₄ concentrations inhibited the ectoine hydroxylase enzyme activity of EctD. For enzymes of the superfamily of the non-heme-containing and iron(II)- and 2-oxoglutarate-dependent dioxygenases (EC 1.14.11), the addition of ascorbate and catalase is sometimes used to increase the enzymatic activity of these types of enzymes (40, 41). The addition of 1.3 kilounits catalase to the enzyme assay enhanced the hydroxylation activity of EctD by ~5-fold. In contrast, the addition of ascorbate inhibited the enzyme activity of EctD by ~70% when ascorbate was present in a concentration of 5 mM in the *in vitro* hydroxylation assay. The EctD enzyme activity of the purified ectoine hydroxylase had a pH optimum of 7.5 and a temperature optimum at 32 °C. Members of the superfamily of the non-heme-containing, iron(II)- and 2-oxoglutarate-dependent dioxygenases (EC 1.14.11) generally contain a mononuclear iron center (39, 41, 44). Consistent with our assignment of the EctD protein to this superfamily, we found that the purified EctD protein from *S. salexigen* contained 0.6 mol of iron/mol of protein. Taken together, these findings support the view that the *S. salexigen* EctD protein is actually an iron-containing enzyme with a mononuclear iron(II) center.

The kinetic parameters for the ectoine hydroxylase from *S. salexigen* were as follows: the \(K_m\) for the substrate ectoine was 3.5 ± 0.2 mM, and the \(K_m\) for the co-substrate 2-oxoglutarate was 5.2 ± 0.2 mM. The \(V_{\text{max}}\) of the EctD enzyme was 13.8 ± 0.1 units mg⁻¹ protein.

We suggest a reaction mechanism for the EctD ectoine hydroxylase from *S. salexigen* (Fig. 1) in which the co-substrate 2-oxoglutarate is stoichiometrically decarboxylated during the hydroxylation of the substrate ectoine. CO₂ is thereby liberated from 2-oxoglutarate to form succinate. During the hydroxylation reaction, one atom of the atmospheric oxygen molecule is incorporated into succinate, whereas the other atom is incorporated into the hydroxygroup formed on ectoine (Fig. 1).

The EctD enzyme requires molecular oxygen, Fe²⁺, and 2-oxoglutarate for its activity *in vitro* (Fig. 3B). EctD is a member of the non-heme-containing, iron(II)- and 2-oxoglutarate-dependent dioxygenases (41–44), an enzyme superfamily (EC 1.14.11) that carries out a diverse set of enzymatic reactions including hydroxylations, desaturations, and oxidative ring closures/rearrangements. The enzymatic reactions of members of this family are usually coupled to the oxidative decarboxylation of 2-oxoglutarate to succinate (41, 44), and this is also true for the EctD-mediated hydroxylation of ectoine (Fig. 1).

The conditions used by us for assaying the EctD-mediated ectoine hydroxylase activity are similar to those used previously to determine the enzyme activity of various pro-

**TABLE 2**

**Purification of the ectoine hydroxylase from *S. salexigen***

The enzyme was prepared from 15 g of wet cells grown in mineral salt-based medium with 3.2 M NaCl at 37°C.

| Purification step         | Total activity | Protein   | Specific activity | Yield |
|---------------------------|----------------|-----------|--------------------|-------|
|                           | units          | mg        | units·(mg protein)⁻¹ | %     |
| Cell-free extract         | 54.3           | 702.0     | 0.08               | 100   |
| Source 15 Q               | 41.1           | 35.7      | 1.15               | 76    |
| Ceramic hydroxylapatite   | 33.8           | 11.4      | 2.97               | 62    |
| Source 15 Phe             | 17.1           | 8.5       | 2.02               | 32    |
| Mono Q                    | 12.5           | 5.2       | 2.4                | 23    |
| Superdex 75 pg            | 11             | 3.6       | 3.06               | 20    |

**FIGURE 3.** **Purification of the EctD ectoine hydroxylase from *S. salexigen* and *in vitro* conversion of ectoine into 5-hydroxyectoine by the purified EctD protein.** A, the ectoine hydroxylase (EctD) from *S. salexigen* was purified through a series of chromatographic steps to apparent homogeneity and electrophoretically separated on an SDS-polyacrylamide gel (12.5% polyacrylamide). The proteins were stained with Coomassie Brilliant Blue. Lane 1, 0.6 μg of EctD protein was loaded onto the gel; lane 2, 2 μg of EctD protein was loaded onto the gel. B, chromatograms from the HPLC analysis of the reaction mixture of the *in vitro* conversion of ectoine to 5-hydroxyectoine by the purified EctD enzyme from *S. salexigen*. The retention times of ectoine and 5-hydroxyectoine were determined by using commercially available samples of these compounds.

Ectoine Hydroxylase from S. salexigenes

$\begin{array}{ll}
\text { H-5eq } & \text { H-4eq } \\
\text { H-6eq } & \text { H-6ax } \\
\text { CH}_3 &
\end{array}$

A

B

FIGURE 4. One-dimensional $^{1} \mathrm{H}$ NMR spectra of in vitro synthesized 5-hydroxyectoine by EctD. A, commercially available 5-hydroxyectoine was analyzed by 300 MHz $^{1} \mathrm{H}$ NMR spectroscopy. B, 5-hydroxyectoine was synthesized in vitro from ectoine by the purified ectoine hydroxylase (EctD) from S. salexigenes, and the reaction product was analyzed by 300 MHz $^{1} \mathrm{H}$ NMR spectroscopy. Above the spectra, the structural formula of 5-hydroxyectoine is shown and the hydrogens responsible for the recorded signals are highlighted.

line hydroxylases (38–40). To make certain that we had not isolated a proline hydroxylase from S. salexigenes that also exhibited ectoine hydroxylase activity, we tested whether the purified EctD protein would use L-proline as a substrate and convert it to a hydroxylated product. No EctD-mediated hydroxylation of L-proline was detected using a sensitive HPLC assay with either 3- or 4-hydroxy proline as reference standards for the HPLC analysis.

Recently, two synthetic ectoine derivatives, homoectoine ((S)-4,5,6,7-tetrahydro-2-methyl-1H-(1,3)-diazepine-4-carboxylic acid) and DHMICA ((RS)-4,5-dihydro-2-methyl-imidazole-4-carboxylic acid), have been described that function as effective PCR enhancers (28). We also tested whether these compounds would serve as substrates for an EctD-mediated hydroxylation reaction, but this was not the case. This finding indicates that the S. salexigenes EctD protein is rather substrate-specific and does not hydroxylate ectoine derivatives with either a five-membered ring structure (DHMICA) or a seven-membered ring structure (homoectoine); ectoine itself is a six-membered ring structure (Fig. 1).

One-dimensional $^{1} \mathrm{H}$ NMR Spectroscopy of the EctD Enzymatic Product—5-Hydroxyectoine produced in vivo by S. parvulus is known to have the (4S,5S) stereochemical configuration (45). To test whether the 5-hydroxyectoine produced by the purified ectoine hydroxylase from S. salexigenes yielded 5-hydroxyectoine in the same stereochemical configuration, we incubated (S)-ectoine with the EctD protein in vitro until the substrate was completely converted. The stereochemical configuration of the de novo synthesized 5-hydroxyectoine was then analyzed by 300 MHz $^{1} \mathrm{H}$ NMR spectroscopy. The result-

ing NMR spectrum of the in vitro synthesized 5-hydroxyectoine was compared with an $^{1} \mathrm{H}$ NMR spectrum prepared from commercially available 5-hydroxyectoine that was isolated by bacterial milking (46) from the Marinococcus sp. strain M52. Both $^{1} \mathrm{H}$ NMR spectra were identical with respect to chemical shifts and $^{1} \mathrm{H}-{ }^{1} \mathrm{H}$ coupling-patterns (Fig. 4). It thus can be inferred from this experiment that the stereochemical positioning of the hydroxyl-group into (S)-ectoine by the purified S. salexigenes EctD protein is the same as that observed in vivo for 5-hydroxyectoine isolated from S. parvulus (45) and Marinococcus sp. strain M52.

Cloning of the Ectoine Biosynthetic Genes from S. salexigenes—Genes for enzymes that operate in the same biosynthetic pathway often cluster together in bacterial genomes. We speculated that the gene encoding the ectoine hydroxylase from S. salexigenes might be positioned in the vicinity of the ectoine biosynthetic genes (ectABC), and we set out to clone and sequence the ectABC gene cluster from S. salexigenes. In total, the DNA sequence of a 2893-bp fragment covering the entire ectABC gene cluster was established and deposited in GenBank™ under accession number AY935521. The ectABC genes from various microorganisms have already been characterized in considerable detail at the molecular level (14–16, 18, 19, 47, 48), and the EctABC proteins from *H. elongata* have been studied biochemically (21). The features of the EctABC proteins from S. salexigenes closely correspond to those from other microorganisms and are therefore not further discussed here.

Cloning of the Structural Gene for the Ectoine Hydroxylase from S. salexigenes—We inspected the 5′ and 3′ regions of the sequenced ectABC gene cluster from S. salexigenes for the presence of an open reading frame (or an incomplete reading frame), which potentially could encode a hydroxylase. However, neither region could encode such a protein. Hence, our initial hypothesis that the structural gene for the ectoine hydroxylase from S. salexigenes might be located in the vicinity of the ectABC ectoine biosynthetic genes proved to be incorrect.

To clone the structural gene for the ectoine hydroxylase from S. salexigenes, we took advantage of the genome sequences of the Mycobacterium smegmatis strain MC2 155 (NC 008596), S. coelicolor A3 (2) (NC 003888), and Bordetella parapertussis (NC 002928). Each of these microbial species contains an ectABC gene cluster, which is immediately followed at the 3′ end by a gene currently annotated in the data bases as a potential proline hydroxylase. These observations raised the possibility that the function of the mentioned genes have been incorrectly

annotated in the various genome projects and actually operate as ectoine hydroxylases. Using a PCR approach with heterologous primers derived from the deduced amino acid sequence of the various potential ectoine hydroxylases, we initially recovered a 270-bp DNA fragment that encoded part of a protein with high sequence identity to the putative ectoine hydroxylases from *M. smegmatis*, *S. coelicolor* A3 (2), and *B. parapertussis*. This DNA fragment was then used as a hybridization probe to recover a 4.1-kb EcoRI restriction fragment from a λZAP-Express gene library. The recombinant λ phage was converted to a plasmid, and a 1977-bp DNA segment was sequenced (AY935522). Inspection of the DNA sequence (Fig. 5) revealed a 900-bp open reading frame that encodes a protein with a calculated molecular mass of 34.4 kDa (300 amino acids). This molecular mass matches exactly that of the purified ectoine hydroxylase from *S. salexigens* determined by us by mass spectrometry. Furthermore, the predicted 15 NH₂-terminal amino acids, as deduced from the DNA sequence of the cloned gene, perfectly match those that were determined experimentally for the NH₂ terminus of the purified EctD ectoine hydroxylase from *S. salexigens*.

Using the predicted amino acid sequence of EctD as template for data base searches, we identified 46 proteins from a variety

![Figure 5](#fig5)

**FIGURE 5.** Northern blot analysis of the osmotically controlled *S. salexigens* *ectABC* operon and the *ectD* region. Shown is the genetic organization of the *ectABC* gene cluster (*A*) and the *ectD* region (*B*) in *S. salexigens*. The cells of *S. salexigens* were grown in minimal medium with various NaCl concentrations, and total RNA was isolated from these cells when the cultures reached an optical density (AD₅₇₈) of ~0.5–1. Equal amounts of the obtained RNA (5 μg) were electrophoretically separated on an 1% agarose gel, blotted onto a nylon membrane, and reacted with a digoxigenin-labeled single-stranded antisense RNA-probe that covered *ectAB* (*A*), *ectD* (*B*), or a DNA segment downstream of *ectD* (*B*).

Ectoine Hydroxylase from *S. salexigens*

of both Gram-negative and Gram-positive *Bacteria* with a high degree (61–73%) of amino acid sequence identity to the EctD protein from *S. salexigens*. In 30 of the 46 *ectD*-containing microorganisms, the *ectD* gene is positioned immediately downstream of an *ectABC* gene cluster, whereas in 16 cases the *ectD* gene is not linked to the *ectABC* operon. However, these microorganisms contain *ectABC* gene clusters elsewhere in their genome. This situation is therefore identical to that occurring in *S. salexigens*.

The 46 putative ectoine hydroxylases are each of similar length (284–318 amino acids), and the amino acid sequence identity of these proteins with the EctD protein from *S. salexigens* extends over the entire length of the polypeptide chain. The various EctD proteins exhibit a limited amino acid sequence identity (26% to 32%) to the L-proline 4-hydroxylase from *Dactylosporangium* sp. strain RH1 (38). This latter protein is a member of the non-heme-containing, iron(II)- and 2-oxoglutarate-dependent dioxygenase superfamily (EC 1.14.11.2) (41–44), members of which contain a highly conserved iron-binding motif that is referred to in the literature as the 2-His-1-carboxylate facial triad (44). Such a putative iron-binding motif can also be found in the EctD protein from *S. salexigens*, and this motif is completely conserved within the amino acid sequence of the 46 EctD-related proteins. To document this fact, we show in Fig. 6 sequence alignment of an EctD protein segment from 14 ectoine hydroxylases that contains this putative iron-binding motif. This region also contains a block of 9 amino acid residues that is completely conserved within the 46 EctD related proteins currently deposited in the data base. These 9 amino acid residues (W-H-S-D-F-E-T-W-H) can serve as a signature sequence motif for ectoine hydroxylases because only these types of proteins are recovered in data base searches when this motif is used as a search template.

![Figure 6](#fig6)

**FIGURE 6.** Alignment of the EctD amino acid sequence from *S. salexigens* with various putative ectoine hydroxylases. The amino acid sequence of the EctD protein from *S. salexigens* is aligned with putative ectoine hydroxylases from *Acidiphilium cryptum* (NZ AAOO01000006), *Bacillus clausii* (AP006627), *Blastopirellula marina* (NZ AANZ01000014), *Bordetella avium* (AM167904), *Bordetella bronchiseptica* (BX640446), *B. parapertussis* (NC 002928), *Nitrobacter* sp. Nb-311A (NZ_AAMY01000005), *Sphingomonas* sp. SKA58 (NZ AAQG01000003), *Sphingopyxis alaskensis* (NC 008048), *Hahella chejuensis* (NC 007645), *Reinekea* sp. MED297 (NZ AAOE01000022), *Methylobacter alcaliphilus* (DQ016501), and *Oceanospirillum* sp. MED92 (ZP 01167714). The iron-binding motif (H-X-D-X<sub>98–100</sub>-H) conserved within the class of non-heme-containing, iron(II)- and 2-oxoglutarate-dependent dioxygenases is indicated by black inverted triangles. An arginine residue involved in the coordination of the co-substrate 2-oxoglutarate in the crystal structure of the TauD protein from *E. coli* (54) is indicated by a star.

Ectoine Hydroxylase from S. salixigenes

A
M NaCl
A C G T 3.4 0.4

B
GATTGGGTTTTATTTAAACTGTCAAGCTGACTACGTTTAATCAAGCTATATA
-35 17 bp spacer -10
AGGATTAACGCGTT **TTGACG** CTAGCTCTGATTCTTA **TAAGCT** AATAAAGTAG
ATTATTTACCAAACTTTGGGAGGTGAAAGCATCCAAAAACACTGTA
AGTACG ATG ACA AAA ACA AGT GTA ATT ... ectA
M T K T S V I

FIGURE 7. Mapping of the transcriptional start site of the *ectABC* operon from *S. salixigenes* by primer extension analysis. A, cultures of *S. salixigenes* were grown in minimal medium with either 0.4 M or 3.4 M NaCl until they reached the exponential growth phase (*A*<sub>578</sub> = 0.5–1). Total RNA was isolated from these cultures, and 20 μg of this RNA was used for primer extension reactions with a DNA primer labeled with the infrared dye IRD-800; the DNA primer was located within the *ectA* coding region. The reaction products of the primer extension reaction were analyzed on an automatic DNA sequencer, and a DNA sequencing reaction was run along the primer extension products to pinpoint the exact start site of the *ectABC* mRNA transcript. B, nucleotide sequence of the *ectABC* promoter region. The transcription initiation site is indicated by an arrow, and the putative –10 and –35 sequences are boxed. The part of the coding sequence of *ectA* shown here is printed in bold type, and the encoded amino acids of EctA are shown beneath the DNA sequence.

The detected *ectABC* transcript was 2.6 kb in size, thereby corresponding approximately to the calculated size (2232 bp) for the *ectABC* coding regions.

To identify the promoter of the *ectABC* operon, we mapped the *ectABC* promoter by primer extension analysis using total RNA from *S. salixigenes* as a template for the primer extension reaction. A single transcription initiation site was detected 56 bp upstream of the *ectA* gene (Fig. 7A). In its vicinity there are –35 and –10 promoter sequences that correspond closely to promoter sequences typically recognized by the housekeeping sigma factor of *B. subtilis*, SigA (51) (Fig. 7B). From the inspection of the *S. salixigenes* *ectABC* promoter sequence, it is not immediately obvious which DNA features make this promoter responsive to high salinity stress.

Heterologous Expression of the *ectD* Gene in Escherichia coli Results in an Enzymatically Active Protein—To provide further evidence that the *ectD*-encoded protein from *S. salixigenes* actually possesses ectoine hydroxylation activity, we cloned the *ectD* gene into the expression vector pASK-IBA3. In the resulting recombinant plasmid pBJ10, the *ectD* gene is positioned under the control of an anhydrotetracycline-inducible *tet* promoter, thereby allowing selective *ectD* gene expression in *E. coli*. *E. coli* is not capable of ectoine synthesis, but it can acquire this compatible solute from exogenous sources through the osmoregulated compatible solute transport systems ProU and ProP (49, 50). Plasmid pBJ10 was introduced into the *E. coli* strain DH5α by transformation, and the resulting strain was grown in a minimal medium with 250 mM NaCl that contained 5 mM ectoine. Expression of *ectD* was initiated by the addition of anhydrotetracycline to the culture, and after 1 h of further growth, the ectoine and 5-hydroxyectoine content of the cells was determined by HPLC analysis. Strain DH5α carrying the empty vector pASK-IBA3 had an ectoine pool of 213 nmol/mg dry weight, but as expected, there was no 5-hydroxyectoine detectable. In contrast, DH5α carrying the *ectD*<sup>+</sup> plasmid pBJ10 had an ectoine pool of 81 nmol/mg of dry weight and a 5-hydroxyectoine content of 138 nmol/mg of dry weight. Hence, 63% of the ectoine taken up by the *E. coli* cells was converted into 5-hydroxyectoine when the cells carried the *ectD* gene from *S. salixigenes*. Taken together, these data provide compelling evidence that the *ectD*-encoded protein from *S. salixigenes* actually possesses ectoine hydroxylase activity in vivo.

Transcription Levels of the *ectABC* Genes from *S. salixigenes* Increase as a Function of Salinity—Ectoine biosynthesis in *S. salixigenes* occurs in an osmotically controlled fashion (Fig. 2A). To test whether the osmotically stimulated ectoine biosynthesis was dependent on increased *ectABC* transcript level in high salinity-grown cells, we probed total RNA from *S. salixigenes* cultures grown at various salinities with an *ectAB*-specific probe in a Northern blot experiment. As shown in Fig. 5A, the level of the *ectABC* transcript in *S. salixigenes* was strongly increased when the salinity of the growth medium was raised.

Transcription Levels of the *ectD* Gene from *S. salixigenes* Increase as a Function of Salinity—To test whether the transcription of the ectoine hydroxylase structural gene *ectD* was increased when the cells were grown under high salinity, we probed total RNA from *S. salixigenes* cultures grown at various salinities with an *ectD*-specific antisense RNA probe. We found that the amount of the *ectD* mRNA was strongly increased in high salinity grown cultures of *S. salixigenes* (Fig. 5B). We detected a transcript with an approximate length of 1.5 kb (Fig. 5B). The size of the *ectD* coding region is 900 bp long; hence the *ectD* gene appeared to be co-transcribed with a DNA region located downstream of the *ectD* stop codon. This was confirmed by Northern blot analysis, using a hybridization probe derived from the DNA segment immediate following *ectD* (Fig. 5B).

Data base searches using the BLAST network service (52) revealed that the DNA region positioned downstream of *ectD* could encode a partial reading frame whose deduced gene product shows amino acid sequence similarity to MarR-type regulators of bacterial transcription (53). However, the reading frame for this putative regulatory protein was disrupted by two stop codons, and no start codon could be found. We do not have a satisfying explanation for the puzzling observation that the DNA segment positioned downstream of *ectD* is transcribed (Fig. 5B), but no appropriate open reading frame is detectable.

Acknowledgments—We thank W. Buckel for helpful discussions on the proposed EctD-mediated enzymatic reaction scheme and T. Selmer and D. Darley for help with the mass spectrometry and the <sup>1</sup>H NMR spectroscopy. We are grateful to E. A. Galinski for providing us with synthetic ectoine derivatives. We appreciate the help of V. Koogle in the editing of the manuscript.

REFERENCES

1. Record, M. T., Jr., Courtenay, E. S., Cayley, D. S., and Guttman, H. J. (1998) Trends Biochem. Sci. 23, 143–148
2. Record, M. T., Jr., Courtenay, E. S., Cayley, S., and Guttman, H. J. (1998) Trends Biochem. Sci. 23, 190–194
3. Kempf, B., and Bremer, E. (1998) Arch. Microbiol. 170, 319–330
4. Galinski, E. A., and Trüper, H. G. (1994) FEMS Microbiol. Rev. 15, 95–108
5. Wood, J. M., Bremer, E., Csonka, L. N., Kraemer, R., Poolman, B., van der Heide, T., and Smith, L. T. (2001) Comp. Biochem. Physiol. A Mol. Integr. Physiol. 130, 437–460
6. Roessler, M., and Müller, V. (2001) Environ. Microbiol. 3, 742–754
7. Roberts, M. F. (2000) Front. Biosci. 5, 796–812
8. Brown, A. D. (1976) Bacteriol. Rev. 40, 803–846
9. Le Rudulier, D., Strøm, A. R., Dandekar, A. M., Smith, L. T., and Valentine, R. C. (1984) Science 224, 1064–1068
10. da Costa, M. S., Santos, H., and Galinski, E. A. (1998) Adv. Biochem. Eng. Biotechnol. 61, 117–153
11. Strom, A. R., and Kaasen, I. (1993) Mol. Microbiol. 8, 205–210
12. Galinski, E. A., Pfeiffer, H. P., and Truper, H. G. (1985) Eur. J. Biochem. 149, 135–139
13. Inbar, L., and Lapidot, A. (1988) J. Biol. Chem. 263, 16014–16022
14. Kuhlmann, A. U., and Bremer, E. (2002) Appl. Environ. Microbiol. 68, 772–783
15. Reshetnikov, A. S., Khmelenina, V. N., and Trotsenko, Y. A. (2006) Arch. Microbiol. 184, 286–297
16. Louis, P., and Galinski, E. A. (1997) Microbiology 143, 1141–1149
17. Canovas, D., Vargas, C., Iglesias-Guerra, F., Csonka, L. N., Rhodes, D., Ventosa, A., and Nieto, J. J. (1997) J. Biol. Chem. 272, 25794–25801
18. Göller, K., Ofer, A., and Galinski, E. A. (1998) FEMS Microbiol. Lett. 161, 293–300
19. Pflughoeft, K. J., Kierek, K., and Watnick, P. I. (2003) Appl. Environ. Microbiol. 69, 5919–5927
20. Peters, P., Galinski, E. A., and Trüper, H. G. (1990) FEMS Microbiol. Lett. 71, 157–162
21. Ono, H., Sawada, K., Khunajakr, N., Tao, T., Yamamoto, M., Hiramoto, M., Shinmyo, A., Takano, M., and Murooka, Y. (1999) J. Bacteriol. 181, 91–99
22. Manzanera, M., Garcia de Castro, A., Tondervik, A., Rayner-Brandes, M., Strom, A. R., and Tunacliffe, A. (2002) Appl. Environ. Microbiol. 68, 4328–4333
23. Manzanera, M., Vilchez, S., and Tunacliffe, A. (2004) FEMS Microbiol. Lett. 233, 347–352
24. Lippert, K., and Galinski, E. A. (1992) Appl. Microbiol. Biotechnol. 37, 61–65
25. Borges, N., Ramos, A., Raven, N. D., Sharp, R. J., and Santos, H. (2002) Extremophiles 6, 209–216
26. Prabhu, J., Schauwecker, F., Grammel, N., Keller, U., and Bernhard, M. (2004) Appl. Environ. Microbiol. 70, 3130–3132
27. Garcia-Estepa, R., Argandona, M., Reina-Bueno, M., Capote, N., Iglesias-Guerra, F., Nieto, J. J., and Vargas, C. (2006) J. Bacteriol. 188, 3774–3784
28. Schnoor, M., Voss, P., Cullen, P., Boking, T., Galla, H. J., Galinski, E. A., and Lorkowski, S. (2004) Biochem. Biophys. Res. Commun. 322, 867–872
29. Garabito, M. J., Arahal, D. R., Mellado, E., Marquez, M. C., and Ventosa, A.

(1997) Int. J. Syst. Bacteriol. 47, 735–741
30. Kunte, H. J., Galinski, E. A., and Trüper, H. G. (1993) J. Microbiol. Methods 17, 129–136
31. Kappes, R., and Bremer, E. (1998) Microbiology 144, 83–90
32. Nau-Wagner, G., Boch, J., Le Good, J. A., and Bremer, E. (1999) Appl. Environ. Microbiol. 65, 560–568
33. Talibart, R., Jebbar, M., Gouesbet, G., Himdi-Kabbab, S., Wróblewski, H., Blanco, C., and Bernard, T. (1994) J. Bacteriol. 176, 5210–5217
34. Whatmore, A. M., Chudek, J. A., and Reed, R. H. (1990) J. Gen. Microbiol. 136, 2527–2535
35. Dulaney, E. L., Dulaney, D. D., and Rickes, E. L. (1968) Dev. Ind. Microbiol. 9, 260–269
36. Heyrman, J., Logan, N. A., Busse, H. J., Balcaen, A., Lebbe, L., Rodriguez-Diaz, M., Swings, J., and De Vos, P. (2003) Int. J. Syst. Evol. Microbiol 53, 501–511
37. Bentley, S. D., Chater, K. F., Cerdeno-Tarraga, A. M., Challis, G. L., Thompson, N. R., James, K. D., Harris, D. E., Quail, M. A., Kieser, H., Harper, D., Bateman, A., Brown, S., Chandra, G., Chen, C. W., Collins, M., Cronin, A., Fraser, A., Goble, A., Hidalgo, J., Hornsby, T., Howarth, S., Huang, C. H., Kieser, T., Larke, L., Murphy, L., Oliver, K., O’Neil, S., Rabbinowitsch, E., Rajandream, M. A., Rutherford, K., Rutter, S., Seeger, K., Saunders, D., Sharp, S., Squares, R., Squares, S., Taylor, K., Warren, T., Wietzorrek, A., Woodward, J., Barrell, B. G., Parkhill, J., and Hopwood, D. A. (2002) Nature 417, 141–147
38. Shibasaki, T., Mori, H., Chiba, S., and Ozaki, A. (1999) Appl. Environ. Microbiol. 65, 4028–4031
39. Mori, H., Shibasaki, T., Yano, K., and Ozaki, A. (1997) J. Bacteriol. 179, 5677–5683
40. Lawrence, C. C., Sobey, W. J., Field, R. A., Baldwin, J. E., and Schofield, C. J. (1996) Biochem. J. 313, 185–191
41. Prescott, A. G., and Lloyd, M. D. (2000) Nat. Prod. Rep. 17, 367–383
42. Costas, M., Mehn, M. P., Jensen, M. P., and Que, L., Jr. (2004) Chem. Rev. 104, 939–986
43. Solomon, E. I., Decker, A., and Lehnert, N. (2003) Proc. Natl. Acad. Sci. U. S. A. 100, 3589–3594
44. Hegg, E. L., and Que, L., Jr. (1997) Eur. J. Biochem. 250, 625–629
45. Inbar, L., Frolow, F., and Lapidot, A. (1993) Eur. J. Biochem. 214, 897–906
46. Sauer, T., and Galinski, E. A. (1998) Biotechnol. Bioeng. 57, 306–313
47. Canovas, D., Vargas, C., Calderon, M. I., Ventosa, A., and Nieto, J. J. (1998) Syst. Appl. Microbiol. 21, 487–497
48. Zhao, B., Lu, W., Yang, L., Zhang, B., Wang, L., and Yang, S. S. (2006) Curr. Microbiol. 53, 183–188
49. Jebbar, M., Talibart, R., Gloux, K., Bernard, T., and Blanco, C. (1992) J. Bacteriol. 174, 5027–5035
50. Lucht, J. M., and Bremer, E. (1994) FEMS Microbiol. Lett. 14, 3–20
51. Helmann, J. D. (1995) Nucleic Acids Res. 23, 2351–2360
52. Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D. J. (1997) Nucleic Acids Res. 25, 3389–3402
53. Alekshun, M. N., and Levy, S. B. (1999) Trends Microbiol. 7, 410–413
54. Elkins, J. M., Ryle, M. J., Clifton, I. J., Dunning Hotopp, J. C., Lloyd, J. S., Burzlaff, N. I., Baldwin, J. E., Hausinger, R. P., and Roach, P. L. (2002) Biochemistry 41, 5185–5192
