
JOURNAL OF BACTERIOLOGY, Oct. 2007, p. 7098–7104
0021-9193/07/$08.00+0 doi:10.1128/JB.00643-07
Copyright © 2007, American Society for Microbiology. All Rights Reserved.

# Origins of Flagellar Gene Operons and Secondary Flagellar Systems†

Renyi Liu¹ and Howard Ochman¹,²*

Department of Biochemistry and Molecular Biophysics¹ and Department of Ecology and Evolutionary Biology,²  
University of Arizona, Tucson, Arizona 85721

Received 24 April 2007/Accepted 12 July 2007

Forty-one flagellated species representing 11 bacterial phyla were used to investigate the origin of secondary flagellar systems and the structure and formation of flagellar gene operons over the course of bacterial evolution. Secondary (i.e., lateral) flagellar systems, which are harbored by five of the proteobacterial species considered, originated twice, once in the alphaproteobacterial lineage and again in the common ancestor of the Beta- and Gammaproteobacteria. The order and organization of flagellar genes have undergone extensive shuffling and rearrangement among lineages, and based on the phylogenetic distributions of flagellar gene complexes, the flagellar gene operons existed as small, usually two-gene units in the ancestor of *Bacteria* and have expanded through the recruitment of new genes and fusion of gene units. In contrast to the evolutionary trend towards larger flagellar gene complexes, operon structures have been highly disrupted through gene disassociation and rearrangements in the Epsilon- and Alphaproteobacteria. These results demonstrate that the genetic basis of this ancient and structurally conserved organelle has been subject to many lineage-specific modifications.

Bacterial flagella are complex organelles whose assembly is dependent on multiple cooperating components. In the well-studied systems, those of *Escherichia coli* and *Salmonella enterica* serovar Typhimurium, approximately 50 genes, distributed in at least 10 operons, contribute to the formation, regulation, and function of the flagella (19, 20). About one-half of these genes produce proteins that become part of the physical structure of flagella, whereas other genes have auxiliary or regulatory roles.

Although the basic structure of the flagellum is fairly well conserved across bacteria, lineages can vary with respect to the number of flagella per cell and the location of flagella on the cell surface, as well as to the overall number of genes devoted to the synthesis and regulation of flagella (3, 29). Among the more notable differences is the difference between the spirochetes, which possess periplasmic flagella whose filaments reside between the outer and cytoplasmic membranes, and other species whose filaments are situated outside the cells (8). In some bacteria, such as *Vibrio parahaemolyticus*, there are two flagellar systems (a polar system and a lateral system), which are encoded by distinct sets of genes and are responsible for different types of motility (22).

The structure, assembly, and function of flagella have been characterized in some detail by molecular, genetic, and biophysical analyses (1, 4, 19, 20, 24, 36); in contrast, relatively little is known about the evolutionary origins of the genes or gene clusters that specify these complex and diverse organelles (25). There is extensive similarity between flagellar genes and genes dedicated to protein secretion systems, leading to speculation that the flagellum arose from a primitive secretion system that was later adapted to cell motility (5, 6, 26, 27). In addition, some of the flagellar proteins resemble those of other systems—the flagellar motor protein MotB is related to the OmpA family of outer membrane proteins, FliI is related to the β-subunit of ATP synthase, FliA is related to RNA polymerase sigma-70 factor, and FliY is related to various ABC transporters—but in all cases the corresponding proteins are very highly diverged, making their ancestries somewhat difficult to reconstruct (28, 34).

The analysis of complete genomic sequences of flagellated bacteria in several distantly related phyla offers an opportunity to trace the history of each flagellar protein and flagellar gene complex through comparative and phylogenetic analyses. In a companion paper (17), we focused on an ancestral core set of genes—the genes whose products constitute much of the physical structure of the flagellum—that evolved through successive duplications of one or a few genes. In this paper, we turn our attention to the origins of the secondary flagellar systems that have been identified in several bacterial genomes and the formation, disruption, and transfer of flagellar gene operons.

## MATERIALS AND METHODS

Retrieving flagellar gene orthologs. The methods used to identify the homologs of each flagellar gene and protein are those described by us previously (17). In brief, orthologs and paralogs of flagellar proteins were identified in the genomes of 41 flagellated species by BLAST and PSI-BLAST searches (2) using the flagellar proteins of *E. coli* and of *Bacillus subtilis* as queries. For flagellar genes not present in *E. coli* (*flaK*, *flaM*, *flaL*, *flhF*, and *flhG*), the protein sequences in *V. parahaemolyticus* were used as queries. Orthology was confirmed by gene context information; high-scoring matches were considered orthologous if they had any other flagellar gene as a neighbor. Phylogenetic trees were built by using Muscle (10), Gblocks (7), and PhyML (12).

Construction of gene order phylogeny. The pairwise distance in gene order between two flagellar systems was calculated as one minus the ratio of the number of shared gene neighbors to the total number of gene neighbors. If a flagellar gene had a novel neighbor (i.e., a nonflagellar gene or a gene not included in the set of flagellar genes analyzed), it was counted as having a “gap” as its neighbor. Cases in which duplicated genes in a single flagellar system had

* Corresponding author. Mailing address: Department of Biochemistry & Molecular Biophysics, University of Arizona, Tucson, AZ 85721. Phone: (520) 626-8355. Fax: (520) 621-3709. E-mail: hochman@arizona.edu.
† Supplemental material for this article may be found at http://jb.asm.org/.
Published ahead of print on 20 July 2007.

the same gene neighbors were counted only once. Neighbor-joining and unweighted-pair group method using average linkage trees were constructed from a gene order distance matrix using MEGA (15). The organizations of the flagellar genes in the 41 species considered are presented in Fig. S1A to D in the supplemental material.

Formation of flagellar gene operons. To examine the origin and changes in flagellar gene operons, the constituent genes of 10 flagellar multigene operons in *E. coli* were retrieved from RegulonDB (35). For each operon, the gene order and transcription direction were examined in the 40 other flagellated species to trace how the operon formed or was disrupted in each lineage. Only when genes within an operon were in the same gene order as they were in *E. coli* were they considered part of the same operon. Based on operon structure, bacterial species were classified into groups. Although bacterial species in the same group have very similar operon organizations, there were some minor differences due to small-scale insertions or rearrangements. In cases where there were differences in gene order within a bacterial group, the species whose operon structure was most similar to that of *E. coli* was chosen to represent its group.

To quantify the integrity of operon structure in each group, we devised an operon clustering index (OCI), which is calculated as follows:

\[
OCI = 10 \times \left( 1 - \frac{\sum n_i}{\sum N_i} \right)
\]

where \( n_i \) is the number of actual separation points between neighbor genes and \( N_i \) is the number of potential separation points in operon \( i \). For example, because the *flgBCDEFGHIJ* operon in *Deltaproteobacteria* is separated into four units, *flgBC*, *flgDE*, *flgFG*, and *flgHII*, the values for determining the OCI are \( n_i = 3 \) and \( N_i = 8 \).

---

**RESULTS**

Origin of secondary flagellar systems. Five of the genomes that we considered encode two flagellar systems (Fig. 1). The two systems are encoded by physically nonoverlapping gene clusters, except that a probable translocation interspersed the two systems in *Chromobacterium violaceum* (see Fig. S1B in the supplemental material). In each of these genomes, there were typically two homologs of the majority of the *E. coli* flagellar gene queries. The polar system, which is analogous to the primary flagellar system in bacteria possessing only one system, is responsible for motility in liquids, whereas the lateral (secondary) system provides bacteria with the ability to adhere or to swarm on surfaces (3, 14, 23, 25). By examining the phylogeny and organization of genes encoding the secondary systems, we found that secondary flagellar systems originated twice from broadly different sources (Fig. 1).

The secondary flagellar systems of *C. violaceum* (*Betaproteobacteria*) and of *Photobacterium profundum*, *V. parahaemolyticus*, and *Yersinia pseudotuberculosis* (*Gammaproteobacteria*) are organized very similarly. These secondary systems differ from the primary systems within same genomes in that they lack *fliO* and the *fliEFGHIJKLMNPQR* gene cluster is rearranged: in the primary systems, these genes form a single cluster and are transcribed in the same direction (from *fliE* to *fliR*), whereas in the secondary systems, the gene cluster is split and inverted, resulting two gene clusters, *fliEFGHI* and *fliRQPNM*, transcribed in opposite directions (from *fliE* to *fliI* and from *fliM* to *fliR*). The three internal genes, *fliJ*, *fliK*, and *fliL*, are translocated to a separate region of the genome (see Fig. S1B and S1C in the supplemental material). Because these secondary systems are clustered in one branch in the phylogenetic tree (Fig. 1) and the branching order of the species containing secondary systems is not fully congruent with the

species or the primary flagellar system trees (Fig. 1), these secondary systems most likely originated from a single duplication in the nonenteric gammaproteobacterial lineage and were then horizontally transferred to the *Betaproteobacteria* and the enteric bacteria.

In the concatenated flagellar protein tree (Fig. 1), the secondary flagellar system of *Bradyrhizobium japonicum* is most closely related to the primary system present in other *Alphaproteobacteria* rather than to the secondary systems of *Beta-* or *Gammaproteobacteria* (Fig. 1), indicating that this secondary system had an independent origin. The gene order phylogeny (Fig. 2) provides a further hint to the origin of the secondary system in *B. japonicum*: this secondary system clusters with the primary (and only) system in one group of *Alphaproteobacteria* (designated *Alphaproteobacteria* group II [Fig. 1 and 2], including *Mesorhizobium loti*, *Sinorhizobium meliloti*, and *Agrobacterium tumefaciens*), whereas the primary system in *B. japonicum* has the same origin as the primary (and only) flagellar system in another group of *Alphaproteobacteria* (designated *Alphaproteobacteria* group I [Fig. 1 and 2], including *Gluconobacter oxydans*, *Caulobacter crescentus*, and *Rhodopseudomonas palustris*). There is a characteristic difference in gene order between the primary and secondary systems in *B. japonicum*: *flgA* is adjacent to *flgH* in the primary system, whereas it is adjacent to *flgL* in the secondary system (see Fig. S1B in the supplemental material).

Evolution of flagellar gene organization. Although flagellar genes typically reside in several gene clusters in bacterial genomes, the organization of these gene clusters, as well as the contents and numbers of gene clusters, differ greatly among bacterial lineages. A phylogenetic tree based on gene order distance captures these differences and shows that flagellar systems cluster into three broad groups (Fig. 2). Group 1 contains the primary flagellar systems of several *Alphaproteobacteria* and the secondary flagellar system of *B. japonicum*. These flagellar systems are very distinct from those of other groups in that many typical pairs of neighboring genes, such as *motAB*, *flhAB*, *fliQR*, *flgDE*, *fliFG*, and *flgFG*, are not adjacent. These systems also possess some unique gene neighbors, such as *fliNG-flhB*, *fliM-motA*, *flgAI*, *flgF-fliI*, and *flgD-fliQ*, that are not observed in any of the other systems in the other two groups (see Fig. S1B in the supplemental material).

Group 2 consists of the primary and secondary flagellar systems in *Beta-* and *Gammaproteobacteria* and the flagellar system in *Zymomonas mobilis*, which originated from lateral gene transfer from a gammaproteobacterial lineage (17). The characteristic gene organization of these flagellar systems consists of three gene clusters, *flgBCDEFGHIJKL*, *fliEFGHI*, and *fliMNOPQR*. As mentioned previously, the latter two gene clusters, together with *fliJ*, *fliK*, and *fliL*, form the large gene cluster *fliEFGHIJKLMNPQR* observed in the primary systems in this group, but *fliO* was lost in the secondary systems (see Fig. S1B and S1C in the supplemental material). Group 3 is also distinct in that some gene neighbors, such as the association of *fliK* with *flgD* and the association of *flgA* with *flgG* and *flgH*, are unique to this group (see Fig. S1 in the supplemental material).

Formation and disruption of flagellar operons. Operons facilitate the coordinated expression of functionally related genes by organizing multiple genes into a single transcription

A Species tree

- Thermoanaerobacter tengcongensis
  - Clostridium tetani
    - Listeria innocua
      - Bacillus subtilis
        - Salinibacter ruber
          - Leptospira interrogans
            - Rhodopirellula baltica
              - Borrelia burgdorferi
                - Treponema denticola
                  - Bdellovibrio bacteriovorus
                    - Aquifex aeolicus
                      - Thermotoga maritima
                        - Desulfotalea psychrophila
                          - Desulfovibrio vulgaris
                            - Campylobacter jejuni
                              - Helicobacter pylori
                                - Wolinella succinogenes
                                  - Gluconobacter oxydans
                                    - Zymomonas mobilis
                                      - Caulobacter crescentus
                                        - Rhodopseudomonas palustris
                                          - Bradyrhizobium japonicum
                                            - Mesorhizobium loti
                                              - Sinorhizobium meliloti
                                                - Agrobacterium tumefaciens
                                                  - Chromobacterium violaceum
                                                    - Bordetella bronchiseptica
                                                      - Burkholderia pseudomallei
                                                        - Xanthomonas axonopodis
                                                          - Legionella pneumophila
                                                            - Pseudomonas aeruginosa
                                                              - Idiomarina loihiensis
                                                                - Colwellia psychrerythraea
                                                                  - Shewanella oneidensis
                                                                    - Photobacterium profundum
                                                                      - Vibrio parahaemolyticus
                                                                        - Salmonella typhimurium
                                                                          - Escherichia coli
                                                                            - Photorhabdus luminescens
                                                                              - Erwinia carotovora
                                                                                - Yersinia pseudotuberculosis

B Flagellar protein tree

- Thermoanaerobacter tengcongensis
  - Bacillus subtilis
    - Clostridium tetani
      - Thermotoga maritima
        - Listeria innocua
          - Leptospira interrogans
            - Borrelia burgdorferi
              - Treponema denticola
                - Salinibacter ruber
                  - Rhodopirellula baltica
                    - Campylobacter jejuni
                      - Helicobacter pylori
                        - Wolinella succinogenes
                          - Aquifex aeolicus
                            - Bdellovibrio bacteriovorus
                              - Desulfotalea psychrophila
                                - Desulfovibrio vulgaris
                                  - Gluconobacter oxydans
                                    - Caulobacter crescentus
                                      - Rhodopseudomonas palustris
                                        - Bradyrhizobium japonicum 1
                                          - Mesorhizobium loti
                                            - Sinorhizobium meliloti
                                              - Agrobacterium tumefaciens
                                                - Zymomonas mobilis
                                                  - Xanthomonas axonopodis
                                                    - Legionella pneumophila
                                                      - Shewanella oneidensis
                                                        - Colwellia psychrerythraea
                                                          - Idiomarina loihiensis
                                                            - Vibrio parahaemolyticus 1
                                                              - Photobacterium profundum 1
                                                                - Pseudomonas aeruginosa
                                                                  - Photobacterium profundum 2
                                                                    - Vibrio parahaemolyticus 2
                                                                      - Yersinia pseudotuberculosis 2
                                                                        - Chromobacterium violaceum 2
                                                                          - Chromobacterium violaceum 1
                                                                            - Burkholderia pseudomallei
                                                                              - Bordetella bronchiseptica
                                                                                - Erwinia carotovora
                                                                                  - Escherichia coli
                                                                                    - Salmonella typhimurium
                                                                                      - Yersinia pseudotuberculosis 1
                                                                                        - Photorhabdus luminescens

FIG. 1. Phylogenetic trees comparing branching orders of flagellated bacteria based on universally conserved proteins and flagellar proteins.
(A) Species tree based on a concatenated alignment of 25 single-copy proteins selected from the proteins identified by Ciccarelli et al. (9).
(B) Phylogenetic tree based on a concatenated alignment of 14 flagellar proteins that are present in all primary and secondary flagellar systems considered. "I" and "2" after species names indicate primary and secondary systems, respectively. The secondary flagellar systems are red. Alpha I, Alphaproteobacteria group I; Alpha II, Alphaproteobacteria group II.

unit (31). To investigate how the flagellar gene operons originated and evolved, we started with 10 multigene operons of *E. coli* listed in RegulonDB (35). Examining the occurrence and distribution of the member genes from each operon within the major lineages of flagellated bacterial enabled us to identify the major events in the formation and disruption of gene clustering (Fig. 3).

The majority of the flagellar genes in the most ancient bacterial lineages, Firmicutes and Spirochaetes, are organized in two-gene units, suggesting that the ancestor contained many dispersed flagellar gene operons. Whereas several small operons, such as *flgKL* and *motAB*, are conserved in almost all bacterial lineages, there is a trend in the Beta- and Gammaproteobacteria towards increasing operon size by adding novel genes or combining small gene units. For example, the *flgBCDEFGHIJ* operon was formed by merging several small operons (Fig. 3). In contrast, both Alpha- and Epsilonproteobacteria show extensive disruption of operon structures, but for different reasons. In the Epsilonproteobacteria, flagellar gene operons became disrupted by the relocation of individual

FIG. 2. Gene order distance tree for the primary and secondary flagellar gene complexes present in 41 flagellated bacterial species. “1” and “2” after species names indicate primary and secondary systems, respectively. The secondary flagellar systems are red. The scale bar indicates gene order distance computed as described in Material and Methods. Alpha I, Alphaproteobacteria group I; Alpha II, Alphaproteobacteria group II.

DISCUSSION

To understand how the genes specifying the bacterial flagellar system evolved and diversified, we previously analyzed the origins of the set of structural genes that are ancestral to all flagellated bacteria (17). In this study, we focused on the origins of the secondary flagellar systems, which are present in several proteobacteria, as well as the formation and evolution

7102 LIU AND OCHMAN J. BACTERIOL.

Firmicutes

| flgBC | flgDE | flgFG | flgJ | flgKL | flhBA | motAB | fliA | fliY | fliDS | fliFGHIJ | fliK | fliLMN | fliOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Spirochaetes Addition of flgA, Addition of flgH and flgl to form flgHIJ

| flgA | flgBC | flgDE | flgFG | flgHIJ | flgKL | flhBA | motAB | fliA | fliY | fliDS | fliFGHIJ | fliK | fliL | fliM | fliNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Deltaproteobacteria Combination of fliLMNOPQR

| flgA | flgBC | flgDE | flgFG | flgHIJ | flgKL | flhBA | motAB | fliA | fliY | fliDS | fliFGHIJ | fliK | fliLMNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Epsilonproteobacteria Dissociation of flgHI, flgKL, flhBA, fliHI, and fliLMNOPQR; Loss of fliJ, fliO

| flgA | flgBC | flgDE | flgFG | flgH | flgL | flhB | flhA | motAB | fliA | fliY | fliDS | fliFGH | fliI | fliJ | fliK | fliL | fliMN | fliO | fliP | fliQ | fliR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Alphaproteobacteria I Dissociation of fliMN

| flgA | flgBC | flgDE | flgFG | flgH | flgL | flgKL | flhB | flhA | motAB | fliA | fliY | fliDS | fliFG | fliI | fliK | fliLM | fliN | fliP | fliQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Alphaproteobacteria II Dissociation of all except flgBC, flgKL and fliMN; Loss of fliDS

| flgA | flgBC | flgDE | flgFG | flgG | flgH | flgL | flgJ | flgKL | flhB | flhA | fota | motB | fliA | fliY | fliDS | fliF | fliG | fliI | fliL | fliMN | fliP | fliQ | fliR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Non-enteric Gammaproteobacteria Combination of flgBCDEFGHIJ; Combination of fliFGHIJK

| flgA | flgBCDEFGHIJ | flgKL | flhBA | motAB | fliA | fliY | fliDS | fliFGHIJK | fliLMNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Secondary system in Beta and Gamma Loss of fliO; Dissociation of fliK and fliL

| flgA | flgBCDEFGHIJ | flgKL | flhBA | motAB | fliA | fliDS | fliFGHI | fliKL | fliMNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Betaproteobacteria Formation of flgAMN and flhDC

| flgAMN | flgBCDEFGHIJ | flgKL | flhBA | motAB | flhDC | fliA | fliY | fliDS | fliFGHIJK | fliLMNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

Enteric bacteria Formation of flhBAE, fliAZY, and fliDST

| flgAMN | flgBCDEFGHIJ | flgKL | flhBAE | motAB | flhDC | fliAZY | fliDST | fliFGHIJK | fliLMNOPQR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

FIG. 3. Formation and disruption of flagellar operons in Bacteria. To enable comparisons across groups, genes are positioned according to their order in E. coli, with the result that the locations and transcription directions of genes that are not neighbors might not reflect their actual locations or directions in the corresponding genomes. The phylogenetic tree on the left is derived from the flagellar protein tree in Fig. 1. The numbers at the nodes are the OCI for the corresponding groups, calculated as described in Materials and Methods. Members of the core set of genes encoding flagellar proteins, as identified by us previously (17), are enclosed in boxes in the Firmicutes.

Because the secondary system of E. coli 042 clusters with those of the four Beta- and Gammaproteobacteria that we investigated (see Fig. S2 in the supplemental material) and is closest to that of Y. pseudotuberculosis, the secondary system in E. coli seems to have originated early in the enteric bacterial lineage and not by recent transfer events.

The primary flagellar system in B. japonicum is closely related to the only flagellar system in Alphaproteobacteria group I, whereas its secondary system is closely related to the flagellar system in Alphaproteobacteria group II (Fig. 1). Therefore, it is unlikely that this secondary system originated from an intragenomic duplication. The most likely scenario is that this

secondary system resulted from horizontal transfer from a species closely related to *Alphaproteobacteria* group II, but it is possible that the ancestor of all *Alphaproteobacteria* possessed a second flagellar system, like that in *B. japonicum*, and that one system was subsequently lost in both *Alphaproteobacteria* groups I and II. A recent study has shown that the alphaproteobacterium *Rhodobacter sphaeroides* contains two flagellar systems, which encode polar flagella (30). One of these systems is ancestral to *Alphaproteobacteria*, whereas the other is homologous to the primary system in *Z. mobilis*, which was shown previously to have originated by lateral gene transfer from *Gammaproteobacteria* (17).

Our results for the formation and disruption of flagellar operons are consistent with those of whole-genome analyses (13), which showed that operons and gene order are typically conserved among closely related organisms and that the destruction of operons may be selectively neutral. Our analyses revealed that the flagellar operons are most extensively disrupted in *Epsilon-* and *Alphaproteobacteria*, but whether other operons also show the most extensive disruption in these two lineages remains to be investigated. Because the flagellar gene operons of *E. coli* have been subject to extensive experimental verification, we performed this analysis using the operon structures of *E. coli* as our frame of reference. It is possible that in some organisms, the flagellar genes form operons containing member genes different from those in *E. coli* K-12 and that these flagellar operons are not as disassociated as they appear. However, based on their phylogenetic relationships, *E. coli* and the *Gammaproteobacteria* in general are the most recently derived organisms and also have the most clustered flagellar gene complexes, indicating a broad evolutionary trend toward increasing cluster size through both the inclusion of new genes and the fusion of existing operons.

Different models have been proposed to explain the driving forces and mechanisms in the formation and maintenance of bacterial operons. Under the “selfish operons” hypothesis (16), lateral gene transfer promotes the formation of operons because clustered genes have a better chance of being transferred together and functional in the new host. Whereas there are certainly instances of lateral gene transfer of flagellar systems, coamplification may also be a mechanism underlying the creation of flagellar operons. According to this model, genes in proximity to one another are more likely to be coamplified, and thus, the tandem duplication of small gene clusters can create new gene junctions and generate new regulatory schemes (32). As suggested by a recent study, coregulation seems to be a major driving force in operon formation and maintenance (31), in that the products of adjacent flagellar genes often form protein complexes (e.g., *flgBC*, *flgKL*, *flhBA*, *fliMN*, and *fliPQR*), and operons serve to regulate both the timing and the amounts of the interacting proteins.

Our data also confirm the rapid changes in flagellar gene regulators reported in previous studies (24, 36). Although details of flagellar gene regulation have not been fully elucidated, it is clear that species have widely different regulation networks, as shown by the variation in the numbers and types of the regulators harbored by different genomes. For example, the master regulators, which control the flagellar gene regulation cascade, differ among groups of bacteria. The master regulators in *E. coli*, FlhC and FlhD, are present only in enteric bacteria, and the master regulator FlaK in *V. parahaemolyticus* appears to be limited to nonenteric *Gammaproteobacteria*. Such diversity is consistent with the general pattern uncovered by genome-wide analyses: transcription factors evolve much more quickly than their target genes, and bacterial regulatory networks are extremely flexible (18, 21). Many of the genes with auxiliary roles in flagellar systems, such as the chaperone genes *fliJ*, *fliS*, *flgN*, and *flgJ*, have sporadic distributions or are lineage specific, although other genes, such as the chemotaxis genes, are shared by *Bacteria* and *Archaea* (11). Taken together, bacterial flagellar systems have been formed with genes having very different histories and have originated and evolved under a combination of different evolutionary forces, including duplication, gene loss, and lateral gene transfer.

### ACKNOWLEDGMENTS

We thank Nancy Moran for originally suggesting that we study the evolution of bacterial flagellar genes and Becky Nankivell for assistance with the preparation of figures.

This work was supported by grants GM56120 and GM74738 from the National Institute of Health to H.O.

### REFERENCES

1. Aldridge, P., and K. T. Hughes. 2002. Regulation of flagellar assembly. Curr. Opin. Microbiol. 5:160–165.
2. Altschul, S. F., T. L. Madden, A. A. Schaffer, J. H. Zhang, Z. Zhang, W. Miller, and D. J. Lipman. 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25: 3389–3402.
3. Bardy, S. L., S. Y. M. Ng, and K. F. Jarrell. 2003. Prokaryotic motility structures. Microbiology 149:295–304.
4. Berg, H. C. 2003. The rotary motor of bacterial flagella. Annu. Rev. Biochem. 72:19–54.
5. Blocker, A., K. Komoriya, and S. Aizawa. 2003. Type III secretion systems and bacterial flagella: insights into their function from structural similarities. Proc. Natl. Acad. Sci. USA 100:3027–3030.
6. Cascales, E., R. Lloubès, and J. N. Sturgis. 2001. The ToIQ-ToIR proteins energize TolA and share homologies with the flagellar motor proteins MotA-MotB. Mol. Microbiol. 42:795–807.
7. Castresana, J. 2000. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol. Biol. Evol. 17:540–552.
8. Charon, N. W., and S. F. Goldstein. 2002. Genetics of motility and chemotaxis of a fascinating group of bacteria: the spirochetes. Annu. Rev. Genet. 36:47–73.
9. Ciccarelli, F. D., T. Doerks, C. von Mering, C. J. Creevey, B. Snel, and P. Bork. 2006. Toward automatic reconstruction of a highly resolved tree of life. Science 311:1283–1287.
10. Edgar, R. C. 2004. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32:1792–1797.
11. Faguy, D. M., and K. F. Jarrell. 1999. A twisted tale: the origin and evolution of motility and chemotaxis in prokaryotes. Microbiology 145:279–281.
12. Guindon, S., and O. Gascuel. 2003. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst. Biol. 52:696–704.
13. Itoh, T., K. Takemoto, H. Mori, and T. Gojobori. 1999. Evolutionary instability of operon structures disclosed by sequence comparisons of complete microbial genomes. Mol. Biol. Evol. 16:332–346.
14. Kirov, S. M. 2003. Bacteria that express lateral flagella enable dissection of the multifunctional roles of flagella in pathogenesis. FEMS Microbiol. Lett. 224:151–159.
15. Kumar, S., K. Tamura, and M. Nei. 2004. MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment. Briefs Bioinformatics 5:150–163.
16. Lawrence, J. G., and J. R. Roth. 1996. Selfish operons: horizontal transfer may drive the evolution of gene clusters. Genetics 143:1843–1860.
17. Liu, R., and H. Ochman. 2007. Stepwise formation of the bacterial flagellar system. Proc. Natl. Acad. Sci. USA 104:7116–7121.
18. Lozada-Chavez, I., S. C. Janga, and J. Collado-Vides. 2006. Bacterial regulatory networks are extremely flexible in evolution. Nucleic Acids Res. 34: 3434–3445.
19. Macnab, R. M. 2003. How bacteria assemble flagella. Annu. Rev. Microbiol. 57:77–100.
20. Macnab, R. M. 2004. Type III flagellar protein export and flagellar assembly. Biochim. Biophys. Acta 1694:207–217.
21. Madan Babu, M., S. A. Teichmann, and L. Aravind. 2006. Evolutionary

dynamics of prokaryotic transcriptional regulatory networks. J. Mol. Biol. 358:614–633.

22. McCarter, L. L. 2004. Dual flagellar systems enable motility under different circumstances. J. Mol. Microbiol. Biotechnol. 7:18–29.

23. McCarter, L. L. 2001. Polar flagellar motility of the *Vibrionaceae*. Microbiol. Mol. Biol. Rev. 65:445–462.

24. McCarter, L. L. 2006. Regulation of flagella. Curr. Opin. Microbiol. 9:180–186.

25. Merino, S., J. G. Shaw, and J. M. Tomas. 2006. Bacterial lateral flagella: an inducible flagella system. FEMS Microbiol. Lett. 263:127–135.

26. Musgrave, I. 2004. Evolution of the bacterial flagellum, p. 72–84. *In* M. Young and T. Edis (ed.), Why intelligent design fails: a scientific critique of the new creationism. Rutgers University Press, New Brunswick, NJ.

27. Nguyen, L., I. T. Paulsen, J. Tchieu, C. J. Hueck, and M. H. Saier. 2000. Phylogenetic analyses of the constituents of type III protein secretion systems. J. Mol. Microbiol. Biotechnol. 2:125–144.

28. Pallen, M. J., and N. J. Matzke. 2006. From The Origin of Species to the origin of bacterial flagella. Nat. Rev. Microbiol. 4:784–790.

29. Penn, C. W., and C. J. Luke. 1992. Bacterial flagellar diversity and significance in pathogenesis. FEMS Microbiol. Lett. 100:331–336.

30. Poggio, S., C. Abreu-Goodger, S. Fabela, A. Osorio, G. Dreyfus, P. Vinuesa,

and L. Camarena. 2007. A complete set of flagellar genes acquired by horizontal transfer coexists with the endogenous flagellar system in *Rhodobacter sphaeroides*. J. Bacteriol. 189:3208–3216.

31. Price, M. N., K. H. Huang, A. P. Arkin, and E. J. Alm. 2005. Operon formation is driven by co-regulation and not by horizontal gene transfer. Genome Res. 15:809–819.

32. Reams, A. B., and E. L. Neidle. 2004. Selection for gene clustering by tandem duplication. Annu. Rev. Microbiol. 58:119–142.

33. Ren, C. P., S. A. Beatson, J. Parkhill, and M. J. Pallen. 2005. The Flag-2 locus, an ancestral gene cluster, is potentially associated with a novel flagellar system from *Escherichia coli*. J. Bacteriol. 187:1430–1440.

34. Rizzotti, M. 2000. Early evolution: from the appearance of the first cell to the first modern organisms, English edition. Birkhauser Verlag, Boston, MA.

35. Salgado, H., S. Gama-Castro, M. Peralta-Gil, E. Diaz-Peredo, F. Sanchez-Solano, A. Santos-Zavaleta, I. Martinez-Flores, V. Jimenez-Jacinto, C. Bonavides-Martinez, J. Segura-Salazar, A. Martinez-Antonio, and J. Collado-Vides. 2006. RegulonDB (version 5.0): *Escherichia coli* K-12 transcriptional regulatory network, operon organization, and growth conditions. Nucleic Acids Res. 34:D394–D397.

36. Soutourina, O. A., and P. N. Bertin. 2003. Regulation cascade of flagellar expression in Gram-negative bacteria. FEMS Microbiol. Rev. 27:505–523.
