
J. Nat. Prod. 2007, 70, 1384–1391

# Invited Review

## Biosynthesis of Unusual Aminocyclitol-Containing Natural Products

**Taifo Mahmud,**$^{*,†,‡}$ **Patricia M. Flatt,**$^{†}$ and **Xiumei Wu**$^{‡}$

**Department of Pharmaceutical Sciences and Genetics Program, Oregon State University, Corvallis, Oregon 97331-3507**

Received May 6, 2007

The aminocyclitol family of natural products is a class of sugar-derived microbial secondary metabolites that demonstrate significant biological activities. Within this class of natural products are the C<sub>7</sub>N-aminocyclitol-containing compounds, which were originally associated with potent sugar-hydrolase inhibition. However, recent discoveries indicate a broader array of chemical structures and biological activities of this class of compounds. Using both conventional feeding experiments and contemporary molecular genetic approaches, some progress has been made in understanding the biosynthesis of this class of natural products. Results of *in silico* investigation also suggest a wide distribution of this class of natural products or closely related compounds across different classes of microorganisms, including cyanobacteria and fungi. This review describes our recent progress in the biosynthetic studies of a number of C<sub>7</sub>N-aminocyclitol-containing compounds and the potential use of bioinformatic approaches to search for novel aminocyclitol-containing natural products.

---

### Introduction

Among the myriad of naturally occurring compounds are the aminocyclitol-containing natural products, which represent a large family of sugar-derived microbial secondary metabolites. These include the clinically important aminoglycoside antibiotics, e.g., kanamycin (1), neomycin (2), and streptomycin (3), the C<sub>7</sub>N-aminocyclitol-containing compounds, e.g., acarbose (4), validamycin A (5), and pyralomicin (6), and the cyclopentitol-derived antibiotics, e.g., pactamycin (7) and trehazolin (8) (Figure 1).<sup>1</sup> Many of these compounds are used widely for the treatment of diseases in humans, animals, and plants. For example, streptomycin, kanamycin, and neomycin have long been used in the clinic against bacterial infections,<sup>2</sup> whereas acarbose is a potent α-glucosidase inhibitor used for the treatment of insulin-independent type-II diabetes.<sup>3</sup> On the other hand, validamycin A is an important crop protectant, particularly against sheath blight disease of rice plants.<sup>4</sup>

On the basis of their origins, the aminocyclitols can be divided into four classes:<sup>1</sup> (1) the myo-inositol-derived aminocyclitols [e.g., streptomycin (3)], (2) the 2-deoxy-scyllo-inosose-derived aminocyclitols [e.g., kanamycin (1)], (3) the cyclopentitol-derived aminocyclitols [e.g., pactamycin (7)], and (4) the 2-epi-5-epi-valiolone-derived aminocyclitols [e.g., validamycin A (6)]. While the first three classes of aminocyclitols have been found to be originated from a hexose, i.e., glucose 6-phosphate, the fourth class is derived from a C<sub>7</sub>-sugar phosphate, sedoheptulose 7-phosphate, which is one of the key sugar phosphate intermediates in the pentose phosphate pathway.

Over the past several years, our group has been studying the biosynthesis of the C<sub>7</sub>N-aminocyclitol family of natural products. Using both conventional feeding experiments and contemporary molecular genetic approaches, progress has been made in understanding the formation of this class of natural products. In collaboration with the Deng group, we have recently identified and functionally analyzed the biosynthetic gene cluster of the antifungal agent validamycin in *Streptomyces hygroscopicus* var. *jinggangensis*.<sup>5</sup> Further studies have also been carried out in a number of different C<sub>7</sub>N-aminocyclitol-containing natural products, such as the antibacterial agents the pyralomics (6) and the antitumor agent cetoniacytone A (9), as well as in the human thioredoxin system inhibitor BE-40644 (10). Although the latter compound does not have nitrogen in its structure, the formation of the C<sub>7</sub>-cyclitol moiety appears to resemble that of the C<sub>7</sub>N-aminocyclitols. This finding is supported through the recent elucidation and characterization of the BE40644 biosynthetic gene cluster.<sup>6,7</sup> The present review describes our recent progress in the biosynthetic studies of this class of compounds. The potential use of bioinformatic approaches to search for novel aminocyclitol-containing natural products is also discussed.

---

### Biosynthesis of Pyralomicin in *Nonomuraea spiralis*

The pyralomics are a set of antibiotics isolated from the soil bacterium *Nonomuraea spiralis* by Takeuchi and co-workers in 1996.<sup>8</sup> Pyralomicin 1a (11), the major product, has a unique structure: a benzopyranopyrrole chromophore connected to a cyclitol, which is rarely seen in secondary metabolites (Figure 2). The same bacterium also produces pyralomics 2a (12) and its analogues, containing glucose as the glycon. The core benzopyranopyrrole unit in pyralomicin is similar to that of TAN-876A (13) and TAN-876B (15) isolated from a *Streptomyces* sp. by a group at the Takeda Company. It is also similar to pyoluteorin (14) and pyrrolomycin (16), which are produced by *Pseudomonas* spp. and *Actinosporangium/Streptomyces* spp., respectively.

The biosynthetic gene cluster of pyrrolomycin has been identified recently from two different producing organisms, *Actinosporangium vitaminophilum* ATCC 31673 and *Streptomyces* sp. strain UC 11065.<sup>9</sup> Some of the genes were found to be similar to those of the pyoluteorin cluster, which was identified earlier within a 24 kb genomic region of *P. fluorescens* Pf-5 by Loper and co-workers, revealing 10 genes involved in the formation of pyoluteorin.<sup>10</sup> These include genes that encode proteins homologous to type I polyketide synthases (PltB, PltC), an acyl-CoA dehydrogenase (PltE), an acyl-CoA synthase (PltF), a thioesterase (PltG), three halogenases (PltA,

---

**Based on a Matthew Suffness Award lecture presented at the 47th Annual Meeting of the American Society of Pharmacognosy, Arlington, VA (August 5–9, 2006).**

* To whom correspondence should be addressed. Tel: (541) 737-9679. Fax: (541) 737-3999. E-mail: Taifo.Mahmud@oregonstate.edu.
+ Department of Pharmaceutical Sciences.
‡ Genetics Program.

10.1021/np070210q CCC: $37.00 © 2007 American Chemical Society and American Society of Pharmacognosy  
Published on Web 07/28/2007

Reviews

Journal of Natural Products, 2007, Vol. 70, No. 8 1385

![Chemical structures of selected aminocyclitol (or cyclitol)-containing natural products. The myo-inositol-derived aminocyclitols is highlighted in blue, the 2-deoxy-scyllo-inosose-derived aminocyclitols are highlighted in green, the cyclopentitol-derived aminocyclitols are highlighted in pink, and the 2-epi-5-epi-valiolone-derived aminocyclitols are highlighted in gray.]

Figure 1. Chemical structures of selected aminocyclitol (or cyclitol)-containing natural products. The myo-inositol-derived aminocyclitol is highlighted in blue, the 2-deoxy-scyllo-inosose-derived aminocyclitols are highlighted in green, the cyclopentitol-derived aminocyclitols are highlighted in pink, and the 2-epi-5-epi-valiolone-derived aminocyclitols are highlighted in gray.

PltD, and PltM), and a peptidyl-carrier protein (PltL). More recently, Walsh and co-workers have characterized a number of enzymes involved in the biosynthesis of this compound, including PltE, PltF, and PltL, and found that these three proteins are sufficient for the conversion of L-proline to pyrrolyl-2-carboxyl-S-PCP, a carrier protein bound intermediate of pyoluteorin biosynthesis. Further characterization revealed that this intermediate serves as the substrate for the PltA halogenase.

The biosynthetic origin of pyralomicin has been studied by Takeuchi and co-workers through feeding experiments with a number of isotopically labeled compounds. The results revealed that the pyralomicin core structure is derived from proline, acetate, and propionate, whereas the cyclitol moiety is derived from D-glucose via the pentose phosphate pathway (Figure 3). Subsequently, in collaboration with the Naganawa group, we carried out feeding experiments with a number of synthetically prepared deuterium-labeled cyclitols, i.e., 2-epi-5-epi-[6-2H₂]valiolone ([6-²H₂]-17), 5-epi-[6-²H₂]valiolone ([6-²H₂]-18), [6-²H₂]valiolone ([6-²H₂]-19), [6-²H]valienone ([6-²H]-20), [6-²H]valienol ([6-²H]-21), 1-epi-[6-²H]valienol ([6-²H]-22), 5-epi-[6-²H₂]valiolol ([6-²H₂]-23), and 1-epi-5-epi-[6-²H₂]valiolol ([6-²H₂]-24) (Figure 4). It was found that only 2-epi-5-epi-valiolone (17) and 5-epi-valiolone (18) were incorporated into the cyclitol moiety of pyralomicin la with specific incorporation rates of 23% and 10%, respectively.

2-epi-5-epi-Valiolone (17) is known to be a common intermediate for many C₇N-aminocyclitol-derived natural products. However, it appears that downstream of this initial step different intermediates are utilized during the biosynthesis of various aminocyclitols. For example, during acarbose biosynthesis, 2-epi-5-epi-valiolone is activated by the kinase AcbM to give 2-epi-5-epi-valiolone 7-phosphate. Phosphorylated intermediates are then required for further processing. On the other hand, during validamycin biosynthesis, 2-epi-5-epi-valiolone undergoes epimerization and dehydration to yield the intermediate, valienone (20), prior to phosphorylation. ValC, a homologue of the acarbose kinase, AcbM, was found in the validamycin gene cluster; however, this enzyme was not able to phosphorylate 2-epi-5-epi-valiolone (17). Instead, it was shown to phosphorylate 20 and its saturated form validone to their corresponding 7-phosphate derivatives. It has been proposed that the divergence of the two pathways is due to the discrete substrate specificities of the kinases operating in the two systems. The incorporation of 2-epi-5-epi-valiolone, but not valienone, into pyralomicin suggests that the formation of its cyclitol moiety more likely resembles that of the core moiety of acarbose (Scheme 1). Biosynthetic feeding experiments in the pyralomicin producer support a biosynthetic pathway that includes the phosphorylation of 2-epi-5-epi-valiolone to its 7-phosphate derivative followed by epimerization at C-2 and dehydration at C-5/C-6 to give valienone 7-phosphate (Scheme 1). The modest incorporation of 5-epi-valiolone (only about 10%) into pyralomicin 1a could be due to the phosphorylation of 5-epi-valiolone by the kinase, albeit less efficiently than 2-epi-5-epi-valiolone, to generate the downstream intermediate 5-epi-valiolone 7-phosphate.

The mechanism required for the formation of the unique core structure benzopyranopyrrole during pyralomicin biosynthesis is unknown. The carbonyl group of the precursor proline was found to migrate to the β position of the pyrrole ring in pyralomicin 1a. However, on the basis of the results of feeding experiments and

Journal of Natural Products, 2007, Vol. 70, No. 8

Figure 2. Chemical structures of the pyralomicins and other related compounds.

pyralomicin 1a (11)

pyralomicin 2a (12)

TAN-876A (13)

pyoluteorin (14)

TAN-876B (15)

pyrrolomycin G (16)

Figure 3. Biosynthetic origin of pyralomicin 1a.

L-proline

D-glucose

L-methionine

acetate

propionate

Figure 4. Isotopically labeled compounds used in feeding experiments with the pyralomicin producer.

2-epi-5-epi-[6-<sup>2</sup>H<sub>2</sub>]-valiolone ([6-<sup>2</sup>H<sub>2</sub>]-17)

5-epi-[6-<sup>2</sup>H<sub>2</sub>]-valiolone ([6-<sup>2</sup>H<sub>2</sub>]-18)

[6-<sup>2</sup>H<sub>2</sub>]-valiolone ([6-<sup>2</sup>H<sub>2</sub>]-19)

[6-<sup>2</sup>H]-valienone ([6-<sup>2</sup>H]-20)

[6-<sup>2</sup>H]-valienol ([6-<sup>2</sup>H]-21)

1-epi-[6-<sup>2</sup>H]-valienol ([6-<sup>2</sup>H]-22)

5-epi-[6-<sup>2</sup>H<sub>2</sub>]-valiolol ([6-<sup>2</sup>H<sub>2</sub>]-23)

1-epi-5-epi-[6-<sup>2</sup>H<sub>2</sub>]-valiolol ([6-<sup>2</sup>H<sub>2</sub>]-24)

Walsh and co-workers' study on the pyoluteorin enzymes, it is predicted that the pyrrole ring is derived from L-proline, which is activated and loaded onto a peptidyl carrier protein. In the pyoluteorin system, the halogenation of the pyrrole ring occurs on the carrier protein bound substrate, yielding a dichloropyrrolyl derivative. A similar biosynthetic route is predicted for pyralomicin biosynthesis where a enzyme-bound 5-chloropyrrolyl derivative will be transferred to a polyketide synthase and extended to give a tetraketide intermediate (Scheme 2). Cyclization of the tetraketide and its release from the polyketide synthase would give compound 28, which is almost identical to pyoluteorin (14). However, in pyralomicin 1a biosynthesis, a second halogenation is expected to occur on the aromatic ring. The product would then be converted to the benzopyranopyrrole structure through an unknown rearrangement mechanism. This would be followed by a "cyclitolation" reaction, presumably by an N-glycosyltransferase that can transfer an activated cyclitol unit to the aglycon.

### Biosynthesis of Validamycin and Engineered Production of Validoxylamine A

Validamycin A (5) is a fungistatic compound isolated from Streptomyces hygroscopicus var. jinggangensis 5008 and S.

Scheme 1. Distinct Biosynthetic Pathways to Validamycin and Acarbose/Pyralomicin

![Diagram]

sedoheptulose 7-P

Scheme 2. Proposed Formation of the Aglycon Unit of Pyralomicin Involving the NRPS and the PKS Machineries

![Diagram]

L-proline

5, galactosylvalidoxylamine A (Minagawa, K.; Xu, H.; Mahmud, T., unpublished data).

## Biosynthesis of Cetoniacytone

Cetoniacytone A (9) is a moderate antitumor agent produced by an endosymbiotic *Actinomyces* sp. strain Lu9419, which was isolated from the intestines of the insect rose chafer (*Cetonia aurata*).²¹ It possesses an unusual C₇N-aminocyclitol moiety in its structure, in which the amino group is acetylated and located at the C-2 position. The same core structure is found in the antirheumatoid arthritis agents, the epoxyquinomycins (31), which were isolated from the culture broth of *Amycolatopsis* sp. strain MK 299-95 F4 (Figure 5).²²

The biosynthetic origin of cetoniacytone A (9) has been studied by feeding experiments using [U-¹³C]glycerol in the cetoniacytone producer. The results showed labeling and coupling patterns similar to those of the valienamine moiety of acarbose (4) and validamycin A (5).²¹ Further experiments with isotopically labeled 2-epi-5-epi-valiolone confirmed the involvement of this common precursor in the biosynthesis of cetoniacytone A (9).⁷ However, the uniqueness of the cetoniacytone core structure, which is in many aspects different from the valienamine moiety, has encouraged us to study their formation at the molecular level. A genomic library of *Actinomyces* sp. Lu 9419 has been constructed in the Streptomycetes shuttle vector pOJ446²³ and was screened using the acarbose 2-epi-5-epi-valiolone synthase gene *acbC* as a heterologous probe. Partial sequencing of the positive fragments revealed the presence of a 2-epi-5-epi-valiolone synthase gene (*cetA*) and three additional genes encoding a putative glyoxalase (*cetB*), a hypothetical protein (*cetC*), and an N-acetyltransferase (*cetD*).⁷ Sequence comparisons showed that CetA is highly similar to AcbC and ValA (the validamycin 2-epi-5-epi-valiolone synthase) with 50% and 55% identity, respectively. CetA is also similar (21% identity) with 3-dehydroquinate (DHQ) synthases, the shikimate pathway enzyme involved in the biosynthesis of aromatic amino acids. The actual function of CetA was confirmed by heterologous expression of the recombinant protein in *E. coli*, which was shown to catalyze the conversion of sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone.⁷ This enzyme requires NAD⁺ and Co²⁺ for its activity, and the reaction mechanism has been proposed to resemble that of 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAHP) to 3-dehydroquinate (DHQ) by the DHQ synthase family of enzymes.²⁴ While the functions of CetB and CetC in the biosynthesis of cetoniacytone are not clear, CetD is believed to catalyze the acetylation of cetoniacytone B (30) to cetoniacytone A (9). This reaction most likely takes place last in the biosynthesis. Feeding experiments with sodium [1-¹³C]acetate showed the incorporation of the acetate only into the N-acetyl moiety of cetoniacytone A. It has been reported

Scheme 3. Heterologous Expression of Seven Genes from the Validamycin Cluster in *S. lividans* Resulted in the Production of Validoxylamine A (A); Addition of the valG Gene into the Mutant Gave Rise to Validamycin A (B)

![Diagram](https://i.imgur.com/1234567.png)

that cultures of *Actinomyces* sp. strain Lu9419 without the addition of sodium acetate produced more unacetylated **30**.²¹

### Biosynthesis of BE-40466

BE-40644 (10) is a hybrid isoprenoid-cyclitol-derived inhibitor of the human thioredoxin system produced by *Actinoplanes* sp. A40644.²⁵ Feeding experiments with \[^{13}C\]acetate and \[^{13}C\]-glucose revealed that the isoprenoid portion of the compound is derived from the mevalonate pathway, whereas the cyclitol is derived from a pathway similar to those reported for some C₇N-aminocyclitol-containing natural products.²⁶

The gene cluster of BE-40466 has been isolated from *Actinoplanes* sp. A40644 by Dairi and co-workers.⁶ Included in this pathway are a set of genes (*orf2–orf7*) homologous to those of the mevalonate pathway. These are believed to be responsible for the biosynthesis of the isoprene units found in BE40644. Additional genes include *orf9–orf13* responsible for the formation of the cyclitol unit (Figure 6), *orf1* and *orf14* encoding a polyprenyl diphosphate synthase and prenyltransferase, respectively, and hypothetical and regulatory proteins (*orf15–orf18*). *BE-orf9* encodes a putative 2-epi-5-epi-valiolone synthase that shares homology with ValA (56% identity/68% similarity), CetA (54% identity/65% similarity), and AcbC (57% identity/70% similarity) from the validamycin, cetoniacytone, and acarbose pathways, respectively. Biochemical characterization of recombinant BE-Orf9 produced in *E. coli* revealed that the enzyme selectively converts sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone, confirming its function as 2-epi-5-epi-valiolone synthase.⁷ While no further biochemical data were available for the functions of the downstream enzymes, the lack of a cyclitol kinase in the cluster suggested that 2-epi-5-epi-valiolone may be transferred to the core cyclitol moiety without a phosphorylation step (Scheme 4).

### Scheme 4. Proposed Biosynthetic Pathway to BE-40644

![Diagram](https://i.imgur.com/8901234.png)

### Figure 5. Chemical structures of the cetoniacytones and epoxyquinomycin C.

![Chemical Structures](https://i.imgur.com/5678901.png)

### Bioinformatic Analysis of the 2-epi-5-epi-Valiolone Synthases within the Sugar Phosphate Cyclase (SPC) Superfamily

As described above, the amino acid sequences of 2-epi-5-epi-valiolone synthases are similar to those of the DHQ synthases of the shikimate pathway, as well as with the aminoDHQ synthases, which are involved in the biosynthesis of 3-amino-5-hydroxybenzoic acid (AHBA) (Scheme 5).²⁷ The latter compound is the precursor of many bioactive natural products, such as rifamycin, maytansine, and mitomycin. These enzymes all utilize C₇-sugar phosphates as

signature of altered binding-pocket residues when compared with DHQ synthases, which are the most well-studied subclass of SPC enzymes.⁷ This information can be used as a tool for screening novel aminocyclitol-containing natural products from other micro-organisms. Interestingly, results from the phylogenetic analysis have also revealed a number of hypothetical proteins and putative DHQS, mostly of cyanobacterial and fungal origin, that form two separate clades that are distinct from the other families of SPCs (Figure 7).⁷ All of these novel homologues have been identified from genome sequencing projects. Recently, the identification of the scytonemin biosynthetic gene cluster in *Nostoc punctiforme* ATCC 29133 identified Nos3 as a dedicated DHQS utilized in the biosynthesis of this UV-protective natural product.²⁸ Furthermore, our additional analysis of the genes neighboring *nos4* revealed their possible involvement in ubiquinone biosynthesis, suggesting that ubiquinone biosynthesis in *N. punctiforme* may also harness the activity of a dedicated DHQS during product formation. The function of the remaining homologues in the other clade is currently unknown, but may be involved in the formation of novel cyclitol-containing natural products in cyanobacteria and fungi. Efforts toward the identification and isolation of such molecules in cyanobacteria are currently being pursued in our laboratory.

### Summary and Perspectives

The wealth of knowledge obtained from studies on natural products biosynthesis has provided new opportunities to produce novel analogues of natural products via biosynthetic approaches. These include genetic manipulations of the biosynthetic gene clusters, application of recombinant proteins for chemoenzymatic synthesis, and incorporation of alternate precursors for mutasynthesis. Using these approaches, hundreds of structurally altered antibiotics have recently been generated, ranging from analogues of the antibacterial agent erythromycin²⁹ to the anthelmintic avermectins³⁰ and the antitumor indolocarbazoles.³¹ The same ap-

**Scheme 5. Biosynthetic Pathways That Parallel the Initial Steps in Shikimate Biosynthesis**$^{a}$

![Diagram](https://i.imgur.com/yourimageurl.png)

$a$ DOS = 2-deoxy-scyllo-inosose synthase, EVS = 2-epi-5-epi-valiolone synthase, DHQS = dehydroquinate synthase, aDHQS = aminodehydroquinate synthase.

1390 Journal of Natural Products, 2007, Vol. 70, No. 8

Figure 7. Phylogenetic analysis of the sugar phosphate cyclase superfamily involved in primary and secondary metabolism. The Phylip software package was used to generate an unrooted maximum likelihood tree. *E. coli* glycerol dehydrogenase was used as an out group. Species information and GenBank accession numbers are identified for each protein.

proaches may be employed for the production of novel analogues of aminocyclitol-containing compounds. Our recent work on validamycin biosynthesis has provided evidence that such approaches are feasible. The identification of the gene cluster in *S. hygroscopicus* var. *jinggangensis* has provided tools not only to study the biosynthesis of validamycin at the molecular level but also to produce the compound in a heterologous host and to generate novel analogues of validamycin using recombinant proteins. It may also be possible to engineer rice plants and other susceptible crops to produce validamycin A, which can protect them from the sheath blight disease directly.³²

Through our studies on the biosynthesis of the C₇N-aminocyclitols, we recognized that 2-epi-5-epi-valiolone synthase genes appear to be a predominant hallmark of the biosynthetic gene clusters of this class of natural products. The enzymes represent a subclass of the sugar phosphate cyclase superfamily, which catalyze the cyclization of sugar phosphates to produce a variety of cyclitol intermediates that serve as the building blocks of many primary and secondary metabolites. A comparative analysis of the amino acid sequences of the sugar phosphate cyclase superfamily revealed that the 2-epi-5-epi-valiolone synthase genes are distinctive enough so that class-specific degenerate PCR primers can be used to selectively amplify gene homologues from metabolically and genetically untapped organisms. This approach can potentially lead to the isolation of novel aminocyclitol-containing natural products, as well as the identification of their genetic codes. *In silico* studies of the sugar phosphate cyclase enzymes have also revealed a potentially new type(s) of enzymes distributed in fungi and cyanobacteria, which are subject to our current investigations.

Acknowledgment. The authors thank all former and current group members who have contributed to the projects and Drs. H. G. Floss, Z. Deng, L. Bai, H. Naganawa, A. Zeeck, and T. Dairi for fruitful collaborations. A grant from the National Institutes of Health (R01 AI061528) and the Oregon State University, College of Pharmacy, General Research Funds provided partial support for the experimental work described in this review.

References and Notes

(1) Flatt, P. M.; Mahmud, T. *Nat. Prod. Rep.* **2007**, 24, 358–392.

(2) Siegenthaler, W. E.; Bonetti, A.; Luthy, R. *Am. J. Med.* **1986**, 80, 2–14.

(3) Soonthornpun, S.; Rattarasarn, C.; Thamprasis, A.; Leetanaporn, K. *J. Med. Assoc. Thai.* **1998**, 81, 195–200.

(4) Asano, N.; Yamaguchi, T.; Kameda, Y.; Matsui, K. *J. Antibiot.* **1987**, 40, 526–532.

(5) Bai, L.; Li, L.; Xu, H.; Minagawa, K.; Yu, Y.; Zhang, Y.; Zhou, X.; Floss, H. G.; Mahmud, T.; Deng, Z. *Chem. Biol.* **2006**, 13, 387–397.

(6) Kawasaki, T.; Kuzuyama, T.; Furihata, K.; Itoh, N.; Seto, H.; Dairi, T. *J. Antibiot.* **2003**, 56, 957–966.

(7) Wu, X.; Flatt, P. M.; Schlorke, O.; Zeeck, A.; Dairi, T.; Mahmud, T. *ChemBioChem* **2007**, 8, 239–248.

(8) Kawamura, N.; Sawa, R.; Takahashi, Y.; Isshiki, K.; Sawa, T.; Naganawa, H.; Takeuchi, T. *J. Antibiot.* **1996**, 49, 651–656.

(9) Zhang, X.; Parry, R. *J. Antimicrob. Agents Chemother.* **2007**, 51, 946–957.

(10) Nowak-Thompson, B.; Chaney, N.; Wing, J. S.; Gould, S. J.; Loper, J. E. *J. Bacteriol.* **1999**, 181, 2166–2174.

(11) Thomas, M. G.; Burkart, M. D.; Walsh, C. T. *Chem. Biol.* **2002**, 9, 171–184.

(12) Dorrestein, P. C.; Yeh, E.; Garneau-Tsodikova, S.; Kelleher, N. L.; Walsh, C. T. *Proc. Natl. Acad. Sci. U.S.A.* **2005**, 102, 13843–13848.

(13) Kawamura, N.; Sawa, R.; Takahashi, Y.; Sawa, T.; Naganawa, H.; Takeuchi, T. *J. Antibiot.* 1996, 49, 657–660.

(14) Naganawa, H.; Hashizume, H.; Kubota, Y.; Sawa, R.; Takahashi, Y.; Arakawa, K.; Bowers, S. G.; Mahmud, T. *J. Antibiot.* 2002, 55, 578–584.

(15) Mahmud, T.; Tornus, I.; Egelkrout, E.; Wolf, E.; Uy, C.; Floss, H. G.; Lee, S. *J. Am. Chem. Soc.* 1999, 121, 6973–6983.

(16) Dong, H.; Mahmud, T.; Tornus, I.; Lee, S.; Floss, H. G. *J. Am. Chem. Soc.* 2001, 123, 2733–2742.

(17) Zhang, C. S.; Stratmann, A.; Block, O.; Bruckner, R.; Podeschwa, M.; Altenbach, H. J.; Wehmeier, U. F.; Piepersberg, W. *J. Biol. Chem.* 2002, 277, 22853–22862.

(18) Minagawa, K.; Zhang, Y.; Ito, T.; Bai, L.; Deng, Z.; Mahmud, T. *ChemBioChem* 2007, 8, 632–641.

(19) Iwasa, T.; Kameda, Y.; Asai, M.; Horii, S.; Mizuno, K. *J. Antibiot.* 1971, 24, 119–123.

(20) Kameda, Y.; Horii, S.; Yamano, T. *J. Antibiot.* 1975, 28, 298–306.

(21) Schlorke, O.; Krastel, P.; Muller, I.; Uson, I.; Dettner, K.; Zeeck, A. *J. Antibiot.* 2002, 55, 635–642.

(22) Tsuchida, T.; Umekita, M.; Kinoshita, N.; Inuma, H.; Nakamura, H.; Nakamura, K.; Naganawa, H.; Sawa, T.; Hamada, M.; Takeuchi, T. *J. Antibiot.* 1996, 49, 326–328.

(23) Bierman, M.; Logan, R.; O’Brien, K.; Seno, E. T.; Rao, R. N.; Schoner, B. E. *Gene* 1992, 116, 43–49.

(24) Stratmann, A.; Mahmud, T.; Lee, S.; Distler, J.; Floss, H. G.; Piepersberg, W. *J. Biol. Chem.* 1999, 274, 10889–10896.

(25) Torigoe, K.; Wakasugi, N.; Sakaizumi, N.; Ikejima, T.; Suzuki, H.; Kojiri, K.; Suda, H. *J. Antibiot.* 1996, 49, 314–317.

(26) Seto, H.; Orihara, N.; Furihata, K. *Tetrahedron Lett.* 1998, 39, 9497–9500.

(27) Floss, H. G. *Nat. Prod. Rep.* 1997, 14, 433–452.

(28) Soule, T.; Stout, V.; Swingley, W. D.; Meeks, J. C.; Garcia-Pichel, F. *J. Bacteriol.* 2007, 189, 4465–4472.

(29) McDaniel, R.; Thamchaipenet, A.; Gustafsson, C.; Fu, H.; Betlach, M.; Ashley, G. *Proc. Natl. Acad. Sci. U.S.A.* 1999, 96, 1846–1851.

(30) Cropp, T. A.; Wilson, D. J.; Reynolds, K. A. *Nat. Biotechnol.* 2000, 18, 980–983.

(31) Sanchez, C.; Mendez, C.; Salas, J. A. *J. Ind. Microbiol. Biotechnol.* 2006, 33, 560–568.

(32) Mahmud, T. *Nat. Prod. Rep.* 2003, 20, 137–166.
NP070210Q
