
REVIEW ARTICLE

# The biosynthesis of peptidoglycan lipid-linked intermediates

Ahmed Bouhss¹, Amy E. Trunkfield², Timothy D.H. Bugg² & Dominique Mengin-Lecreulx¹

¹Laboratoire des Enveloppes Bactériennes et Antibiotiques, Institut de Biochimie et Biophysique Moléculaire et Cellulaire, UMR 8619 CNRS, Univ Paris-Sud, Orsay, France; and ²Department of Chemistry, University of Warwick, Coventry, UK

---

**Correspondence:** Dominique Mengin-  
Lecreulx, Laboratoire des Enveloppes  
Bactériennes et Antibiotiques, IBBMC, UMR  
8619 CNRS, Université Paris-Sud, Bât. 430,  
91405 Orsay Cedex, France. Tel.: +33 1 69 15  
48 41; fax: +33 1 69 85 37 15; e-mail:  
dominique.mengin-lecreulx@u-psud.fr  

Received 10 July 2007; revised 20 September  
2007; accepted 24 September 2007.  
First published online 11 December 2007.  

DOI:10.1111/j.1574-6976.2007.00089.x  

Editor: Jacques Coyette  

**Keywords**  
peptidoglycan; undecaprenyl phosphate; UppS  
synthase; UppP phosphatases; MraY  
translocase; MurG transferase.

---

## Abstract

The biosynthesis of bacterial cell wall peptidoglycan is a complex process involving many different steps taking place in the cytoplasm (synthesis of the nucleotide precursors) and on the inner and outer sides of the cytoplasmic membrane (assembly and polymerization of the disaccharide-peptide monomer unit, respectively). This review summarizes the current knowledge on the membrane steps leading to the formation of the lipid II intermediate, i.e. the substrate of the polymerization reactions. It makes the point on past and recent data that have significantly contributed to the understanding of the biosynthesis of undecaprenyl phosphate, the carrier lipid required for the anchoring of the peptidoglycan hydrophilic units in the membrane, and to the characterization of the MraY and MurG enzymes which catalyze the successive transfers of the N-acetylmuramoyl-peptide and N-acetylglucosamine moieties onto the carrier lipid, respectively. Enzyme inhibitors and antibacterial compounds interfering with these essential metabolic steps and interesting targets are presented.

---

## Introduction

Peptidoglycan (murein) is a major heteropolymer of bacterial cell walls that consists in long glycan chains made of alternating units of N-acetylmuramoyl-peptides (MurNAc-peptides) and N-acetylglucosamine (GlcNAc) that are cross-linked together via the short peptide chains (Rogers *et al.*, 1980; Park, 1996; Vollmer *et al.*, 2008). The main essential function of this giant cell-sized macromolecule is to protect cells against the deleterious effects of the internal osmotic pressure. It also contributes to the maintenance of the characteristic cell shape and serves as a platform for the anchoring of other cell envelope components including proteins (Braun & Sieglin, 1970; Marraffini *et al.*, 2006; Dramsi *et al.*, 2008) and polysaccharides (Neuhaus & Baddiley, 2003).

The biosynthesis of peptidoglycan is a complex process involving many different cytoplasmic and membranes steps (van Heijenoort, 2001b). The first stage consists in the formation of the soluble nucleotide precursors, from UDP-GlcNAc to UDP-MurNAc-pentapeptide. In particular, the synthesis of the peptide moiety is performed by a series of enzymes designated as the Mur ligases (MurC, MurD, MurE and MurF) which are responsible for the respective additions of L-alanine, D-glutamic acid, meso-diaminopimelic acid (A₂pm) or L-lysine, and D-alanyl-D-alanine to UDP-MurNAc (Barreteau *et al.*, 2008). As reported by Schleifer & Kandler (1972) and exemplified in the accompanying reviews (Barreteau *et al.*, 2008; Vollmer *et al.*, 2008), the structure of this peptide and the substrate specificity of these enzymes exhibit some variations in the bacterial world. The membrane steps then begin with the transfer of the phospho-MurNAc-pentapeptide moiety from the cytoplasmic precursor to the membrane acceptor undecaprenyl phosphate (C₅₅-P), a reaction catalyzed by the transferase MraY (also named translocase) yielding undecaprenyl-pyrophosphoryl-Mur-NAc-pentapeptide (lipid I) (Fig. 1). Thereafter, the transferase MurG catalyzes the transfer of the GlcNAc moiety from UDP-GlcNAc to lipid I yielding undecaprenyl-pyrophosphoryl-MurNAc-(pentapeptide)-GlcNAc (lipid II) which, after its passage through the membrane by a yet unknown mechanism, will be used as the substrate for the polymerization reactions (van Heijenoort, 2001a,b) (Fig. 1). The C₅₅-P carrier lipid plays a central role in these steps and is also

Biosynthesis of peptidoglycan lipid intermediates

Fig. 1. Membrane steps of peptidoglycan biosynthesis. M, G and the five colored beads linked to M represent MurNAc, GlcNAc and the pentapeptide, respectively. C<sub>55</sub>-PP and C<sub>55</sub>-P are for undecaprenyl pyrophosphate and undecaprenyl phosphate, respectively.

required for the synthesis of other cell-wall polymers. This lipid as well as all the enzymes participating in peptidoglycan synthesis are essential, specific for the bacterial world and therefore constitute interesting potential targets to be exploited for the discovery of new antibacterials.

This review summarizes recent advances in the understanding of the membrane steps of peptidoglycan synthesis. In particular, it will focus on recent significant contributions to the knowledge of the metabolism of the carrier lipid and on progress made toward the biochemical and structural characterization of the MraY and MurG activities and the search of inhibitors of these enzymes.

## Undecaprenyl phosphate metabolism

Undecaprenyl phosphate (C<sub>55</sub>-P), also referred to as bactoprenol, is a key lipid involved in the biosynthesis of peptidoglycan and a variety of other cell-wall polysaccharide components such as lipopolysaccharides, the enterobacterial common antigen, capsule polysaccharides, and teichoic acids (Wright et al., 1967; Scher et al., 1968; Troy et al., 1971, 1975; Watkinson et al., 1971; Johnson & Wilson, 1977; Rohr et al., 1977; Reeves et al., 1996; Rick et al., 1998; van Heijenoort, 2001b; Raetz & Whitfield, 2002; Neuhaus & Baddiley, 2003). C<sub>55</sub>-P-linked saccharides are also used for N-linked protein glycosylation that occurs in certain prokaryotes (Glover et al., 2005; Szymanski & Wren, 2005). That a single lipid participates in the synthesis of various wall polymers has been earlier considered as a potential site of control that prevents an imbalance in the formation of the cell envelope as a whole (Anderson et al., 1972). Although C<sub>55</sub>-P remains the classical carrier lipid form encountered in bacterial world, the lower-size homologs decaprenyl phosphate (C<sub>50</sub>-P) and nonaprenyl phosphate

FEMS Microbiol Rev **32** (2008) 208–233

© 2007 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved

Biosynthesis and recycling of undecaprenyl pyrophosphate

The precursor for C₅₅-P, undecaprenyl pyrophosphate (C₅₅-PP), is synthesized by addition of eight C₅ isopentenyl units (cis[Z]-configuration) onto C₁₅ (all-trans[E])-farnesyl pyrophosphate (FPP) (Fig. 2). This reaction is catalyzed by the undecaprenyl pyrophosphate synthase UppS (di-trans,-poly-cis-decaprenylcistransferase; EC 2.5.1.31), which belongs to the family of cis-prenyltransferases of group IV (Ogura & Koyama, 1998). Farnesyl pyrophosphate itself results from head-to-tail condensation of isopentenyl pyrophosphate with dimethylallyl pyrophosphate, generating C₁₀ geranyl pyrophosphate, followed by a second condensation of isopentenyl pyrophosphate, a reaction catalyzed by the farnesyl pyrophosphate synthase that is a prototype of the trans-prenyltransferase family (Ogura & Koyama, 1998; Liang et al., 2002). The structure and mixed E,Z stereochemistry of the C₅₅-prenyl product (Fig. 3), as deduced from the recent knowledge of the reaction mechanism of the UppS synthase, confirmed earlier data of mass and nuclear magnetic resonance spectrometry of the undecaprenol isolated from bacterial membranes (Scher et al., 1968; Gough et al., 1970).

The UppS synthase had been partially purified and characterized from several bacteria including Salmonella newington, Micrococcus luteus, Lactobacillus plantarum, Bacillus subtilis and Escherichia coli (Christenson et al., 1969; Baba & Allen, 1978, 1980; Takahashi & Ogura, 1982; Baba et al., 1985; Fujisaki et al., 1986). Like the enzymes involved in fatty acid synthesis, UppS is a soluble cytoplasmic enzyme that generates a membrane embedded product (undecaprenyl pyrophosphate). The first *uppS* gene was identified much more recently, in 1998 by Ogura's group, using a genomic DNA library of *M. luteus* B-P 26 constructed in *E. coli* and a screening of recombinant clones for overexpression of the synthase activity (Shimizu et al., 1998). Orthologs were subsequently identified in various Gram-positive and Gram-negative bacterial species (Apfel et al., 1999) and this gene was demonstrated to be essential in *E. coli* (Kato et al., 1999) and *Streptococcus pneumoniae* (Apfel et al., 1999). Interestingly, these newly identified proteins did not exhibit significant sequence similarity with members of the trans-prenyltransferase family and in particular they did not carry the characteristic aspartate-rich DDXXD motif that is involved in substrate binding via a Mg²⁺ bridge in the latter enzyme family (Chen et al., 1994). The construction of expression vectors allowed the purification of mg quantities of various UppS in either wild-type or histidine-tagged form (Shimizu et al., 1998; Apfel et al., 1999; Pan et al., 2000). The enzyme activity showed both a detergent (Triton X-100) and MgCl₂ dependency (Apfel et al., 1999), the latter cation being required for the binding and subsequent condensation of isopentenyl pyrophosphate (Chen et al., 2002b). UppS has now been characterized in great detail, both biochemically and structurally, by different groups. The crystal structure of *M. luteus* and *E. coli* enzymes have been solved, either in apo form or in complex with Mg²⁺, isopentenyl phosphate, FPP, and product analogues (Fujihashi et al., 2001; Ko et al., 2001; Chang et al., 2004; Guo et al., 2005). The data showed that not only the primary but also the three-dimensional (3-D) structure of cis-prenyltransferases was totally different from that of trans-prenyltransferases (Takahashi & Koyama, 2006). The enzyme is a homodimer of 29-kDa subunits and each monomer is composed of six parallel β-strands forming a

![Structure of Undecaprenyl Phosphate](https://i.imgur.com/structure_undecaprenyl_phosphate.png)

**Fig. 3. Structure of the polyrenyl carrier lipid involved in cell wall biosynthesis.**

Biosynthesis of peptidoglycan lipid intermediates

central β-sheet core, which is surrounded by five of the seven α-helices (Fig. 4). A largely hydrophobic 30 Å depth cleft was observed at the protein surface whose entrance carries several positively charged residues as well as a ‘structural P loop’ motif that is characteristic of phosphate recognition enzymes. A flexible domain located in the vicinity of the H3 α-helix and cleft entrance was identified as an important region for the catalytic process and the two subunits were shown to alternate between two different ‘closed’ and ‘open’ conformations (Ko *et al.*, 2001; Chen *et al.*, 2002b), the ‘closed’ form being catalytically active (Chang *et al.*, 2003). Based on the 3-D structure and site-directed mutagenesis experiments, a model for substrate binding and a catalytic mechanism were proposed (Fujikura *et al.*, 2000, 2003; Pan *et al.*, 2000; Kharel *et al.*, 2001; Takahashi & Koyama, 2006). How the ultimate product chain length of cis-prenyltransferases is determined was also investigated in some detail (Ko *et al.*, 2001; Kharel *et al.*, 2006; Takahashi & Koyama, 2006). Site-directed mutagenesis of *E. coli* and *M. luteus* UppS and a sequence comparison with the C<sub>70–120</sub> product synthesizing eukaryotic cis-prenyltransferases highlighted some residues that play a critical role in the determination of the product chain length. It was proposed that charged residues present at the hinge region of the H3 α-helix might control the bending direction of the growing hydrophobic prenyl chain along the hydrophobic interior of the H3 helix so that the hydrophobic cleft could accommodate the bulk of the prenyl chain to fit a suitable size during enzymatic elongation. As the substrate specificity and catalytic properties of the FPP synthase and cis-prenyltransferases are varying to some extent in the bacterial world, the size and stereochemistry of the ultimate polyprenyl product, i.e. the carrier lipid, are bacterial species specific: undecaprenyl phosphate in most cases, rarely nonaprenyl phosphate (C<sub>45</sub>) (Ishii *et al.*, 1986), both with the classical ω,di-trans,poly-cis conformation, or decaprenyl phosphate (C<sub>50</sub>) with an unusual ω,trans,octa-cis conformation (Wolucka *et al.*, 1994; Kaur *et al.*, 2004; Mahapatra *et al.*, 2005) (Fig. 3). Trace amounts of nona-, deca-, and dodeca-prenyl alcohol derivatives were always detected together with the predominant C<sub>55</sub>-P lipid in membrane extracts, confirming the high but not absolute specificity of the corresponding UppS synthase activities observed during *in vitro* assays (Thorne & Kodicek, 1966; Higashi *et al.*, 1967, 1970b; Scher *et al.*, 1968; Gough *et al.*, 1970; Umbreit *et al.*, 1972; Umbreit & Strominger, 1972b; Apfel *et al.*, 1999).

The bacitracin antibiotic has been shown to inhibit bacterial cell wall biosynthesis through sequestration of C<sub>55</sub>-PP, the product of the UppS synthase, thereby provoking the loss of the cell integrity and lysis (Siewert & Strominger, 1967; Stone & Strominger, 1971; Storm & Strominger, 1973). This potent antibiotic produced as a mixture of related cyclic polypeptides by some strains of *Bacillus licheniformis* and *B. subtilis* is extensively used for prophylaxis and therapy in food animals. It is clinically used for treatments of surface tissue infections in combination with other antimicrobial drugs. Its oral use had been earlier suggested for the control of vancomycin.

Fig. 4. Crystal structure of undecaprenyl pyrophosphate synthase dimer (UppS) from *Escherichia coli*. The figure was prepared using PyMol and the atomic coordinates (1JP3) deposited by Ko *et al.* (2001). The seven α-helices and six β-strands are shown in red and green for subunit A and pink and blue for subunit B, respectively. The flexible loop without observable electron densities (residues 72–83) is represented as a dotted line in one subunit. The hydrophobic cleft at the molecular surface of each subunit is indicated by an arrow.

resistant enterococci (VRE) (O'Donovan *et al.*, 1994) although without evident success (Mondy *et al.*, 2001; Hachem & Raad, 2002).

### Formation of undecaprenyl phosphate

The dephosphorylation of C<sub>55</sub>-PP (step 2 in Fig. 2) is required before the lipid carrier becomes available for use in the various biosynthetic pathways. This reaction must also occur after each cycle of polymerization of cell wall components (e.g., of peptidoglycan) and the release of the linked saccharides, because the lipid carrier is in most cases liberated in the pyrophosphate form. However, some exceptions exist: for instance, the transfer of 4-amino-4-deoxy-L-arabinose (L-Ara4N) units to lipid A catalyzed by the ArnT membrane protein uses undecaprenyl-phosphoryl-L-Ara4N as the donor substrate and releases the lipid in the C<sub>55</sub>-P form (Trent *et al.*, 2001).

The membrane-bound phosphatase catalyzing this reaction had been partially purified from *M. luteus* by Goldman & Strominger (1972) about 30 years ago and some of its properties were investigated. Its optimal pH for activity was near 7.5, the enzyme did not require any cation, was stimulated by nonionic Triton detergents, and failed to hydrolyze isopentenyl pyrophosphate. The gene for this activity, however, remained to be identified.

In 1993, an *E. coli* gene whose overexpression resulted in a decreased susceptibility to bacitracin had been identified and the authors hypothesized that this gene should encode an undecaprenol phosphokinase (Cain *et al.*, 1993). This hypothesis was based on the assumption that a significant pool of free C<sub>55</sub>-OH may exist in *E. coli* membranes, as earlier demonstrated in some Gram-positive bacteria (Higashi *et al.*, 1970b), that would be directed towards the formation of C<sub>55</sub>-P by the overproduced kinase, thereby reducing cell requirements for C<sub>55</sub>-PP molecules and the cell sensitivity to bacitracin. In fact, this question was recently revisited and the *bacA* gene product was finally unambiguously proved to be a C<sub>55</sub>-PP phosphatase (El Ghachi *et al.*, 2004). The overproduction of the BacA protein, which allowed *E. coli* cells to resist to high concentrations of bacitracin, was correlated with a large (280-fold) increase of C<sub>55</sub>-PP phosphatase activity in membranes (El Ghachi *et al.*, 2004). The increased level of C<sub>55</sub>-PP phosphatase activity likely accelerated the conversion of the pool of C<sub>55</sub>-PP, the bacitracin target, to C<sub>55</sub>-P, resulting in an increased cell resistance to the antibiotic. The 30 kDa protein BacA was predicted to be an integral membrane protein with eight transmembrane segments. It was successfully extracted from cell membranes by the *n*-dodecyl-β-D-maltoside detergent and purified to near homogeneity in the histidine-tagged form (El Ghachi *et al.*, 2004). The *E. coli* enzyme exhibited a high C<sub>55</sub>-PP phosphatase activity of ca. 2200 nmol min<sup>−1</sup> mg<sup>−1</sup> of protein, a value about 7300-fold higher than the basal activity detected in wild-type cell membranes. It did not show any detectable C<sub>55</sub>-OH phosphokinase activity. Considering this newly identified function, it was proposed to rename the *bacA* gene *uppP* (El Ghachi *et al.*, 2004), for undecaprenyl pyrophosphate phosphatase (also named undecaprenyl-diphosphatase; EC 3.6.1.27), to follow the nomenclature previously adopted with *uppS* that encodes the C<sub>55</sub>-PP synthase (Apfel *et al.*, 1999).

The dephosphorylation of C<sub>55</sub>-PP was predicted to be an essential metabolic step. The finding that the *bacA* gene could be deleted from the chromosome of *E. coli* (El Ghachi *et al.*, 2004), *Mycobacterium smegmatis* (Rose *et al.*, 2004), *Staphylococcus aureus* and *Streptococcus pneumoniae* (Chalker *et al.*, 2000), without loss of viability or any apparent effect on growth rate or morphology, was therefore quite surprising. Its deletion however resulted in impaired biofilm and smegma formation in *M. smegmatis*, and attenuated virulence in mouse models of infection in *S. aureus* and *Streptococcus pneumoniae*. All the *bacA* deletion mutants showed enhanced susceptibility to bacitracin (Chalker *et al.*, 2000; El Ghachi *et al.*, 2004; Rose *et al.*, 2004). The unexpected viability of the *bacA* deletion mutants suggested the existence of other cell proteins with C<sub>55</sub>-PP phosphatase activity. The detection of a 25% residual phosphatase activity in the membranes of the *E. coli* mutant was consistent with this hypothesis (El Ghachi *et al.*, 2004). No *bacA* homologue was found in the genomes of the aforementioned species, indicating that these putative phosphatases belonged to a distinct protein family. Although a copy of *bacA* gene was found in most bacterial genomes sequenced to date (up to three putative *bacA* orthologs were found in some species, e.g., *Bacillus cereus*), it was apparently absent in some bacteria such as *Helicobacter pylori* (unpublished data).

A search in databases for putative membrane phosphatases activities identified three proteins of unknown function that formed the so-called BcrR family: BcrC from *B. licheniformis*, YbjG from *E. coli*, and YwoA from *B. subtilis* (El Ghachi *et al.*, 2005). Interestingly, the *bcrC* gene product was described as one of the three components of the ABC transporter system responsible for the protection of *B. licheniformis* against the antibiotic it produces, bacitracin (Podlesek *et al.*, 1995, 2000), and the overexpression of the *E. coli ybjG* and *B. subtilis ywoA* genes were reported to increase bacitracin resistance in the corresponding bacterial species (Harel *et al.*, 1999; Cao & Helmann, 2002; Bernard *et al.*, 2003). Two other *E. coli* genes displaying similarity with *ybjG* that encoded members of the PAP2 phosphatidic acid-phosphatase family were also identified: *yeiU*, of unknown function, and *pgpB*, encoding one of the two phosphatidylglycerolphosphate phosphatases (Icho & Raetz,

1983). All of these candidate proteins were predicted to be integral membrane proteins and contained a sequence identical or quite similar to the characteristic phosphatase signature KX₆RP-(X₁₂₋₅₄)-PSGH-(X₃₁₋₅₄)-SRX₅HX₃D (El Ghachi *et al.*, 2005) that had been identified previously by Stukey & Carman (1997) and Neuwald (1997). Interestingly, this conserved phosphatase motif was not detected in the sequence of BacA and the absence of significant sequence homology between BacA and the above mentioned proteins clearly indicated that they belonged to two distinct protein families. The chromosomal *ybjG*, *yeiU* and *pgpB* genes could be disrupted individually without apparent effect on cell growth but the coinactivation of the three genes *bacA*, *ybjG* and *pgpB* was shown to be lethal (El Ghachi *et al.*, 2005). A thermosensitive conditional triple mutant strain was generated which lysed at the restrictive temperature due to the depletion of C₅₅-PP phosphatase activity and arrest of peptidoglycan synthesis (El Ghachi *et al.*, 2005). It confirmed the implication of at least the three proteins BacA, YbjG and PgpB in the formation of the C₅₅-P carrier lipid *in vivo*. As observed with *bacA*, the overexpression of the individual *ybjG*, *yeiU* and *pgpB* genes was correlated to an increased resistance to bacitracin and an increased level of C₅₅-PP phosphatase activity in membranes. The *B. subtilis ywoA* gene product was also subsequently purified and proved to be a C₅₅-PP phosphatase (Bernard *et al.*, 2005).

Therefore, two different classes of integral membrane proteins which belong to the BacA and PAP2 phosphatase families, respectively, could catalyze the dephosphorylation of C₅₅-PP into C₅₅-P in bacteria. The number of these proteins could apparently vary from one bacterial species to another. The situation in *E. coli* is one BacA protein and at least two members of the PAP2 family. Most bacterial species have only one copy of *bacA* in their genome but some species seem to express several *bacA* orthologs (e.g. *Bacillus anthracis*, *B. cereus*) and some others apparently do not express any (e.g. *H. pylori*). The situation is similar for members of the PAP2 family but the precise number of those proteins that effectively exhibit C₅₅-PP phosphatase activity remains to be determined for each species.

Interestingly, genes belonging to either of these two classes were also found in gene clusters conferring antibiotic resistance in the bacitracin-producing and bacitracin-resistant species. For instance, *bcrC* and *ywoA* genes that encode phosphatases from the PAP2 family were located within clusters expressing bacitracin efflux systems in *B. licheniformis* and *B. subtilis* strains, respectively (Podlesek *et al.*, 1995; Cao & Helmann, 2002; Bernard *et al.*, 2003), and one *bacA* ortholog was recently found to play a similar essential role in acquired bacitracin resistance in *Enterococcus faecalis* (Manson *et al.*, 2004). C₅₅-PP phosphatase encoding genes are thus used by bacteria either for generating the essential carrier lipid C₅₅-P or for depleting cells of the pool of C₅₅-

PP, the bacitracin target, as a mechanism of resistance to this antibiotic.

The purification and further biochemical characterization of these different proteins is now required to discern phosphatase activities specifically involved in C₅₅-P metabolism from non- or less-specific phosphatases, of distinct metabolic function, that can also use C₅₅-PP as a substrate. C₅₅-PP is synthesized at the inner side of the cytoplasmic membrane but is released at the outer side of the membrane by the peptidoglycan polymerization machinery. Whether the dephosphorylation of C₅₅-PP occurs on both membrane sides or only on one side is at present unknown. A topological analysis of these different membrane proteins will help to answer this question. The involvement of either different phosphatases with catalytic sites orientated towards the cytoplasm and the periplasm or a single phosphatase could be envisaged, but in both cases a trans-bilayer movement of the carrier lipid should occur, suggesting here also the involvement of a putative flippase (see ‘The possible existence of a flippase’).

The reaction of dephosphorylation of C₅₅-PP was considered as an interesting potential target in a search for new antibiotics. The recent discovery that two classes of enzymes and multiple orthologs in each class could participate in this process could render this search more problematic. However, one efficient way to inhibit this step remains the sequestration of the C₅₅-PP substrate, as demonstrated with bacitracin (Siewert & Strominger, 1967; Stone & Strominger, 1971; Storm & Strominger, 1973). An attack of the pyrophosphate moiety of C₅₅-PP was also suggested to be part of the mechanism of action of nisin, a lantibiotic whose primary target is lipid II (Bonev *et al.*, 2004). It was earlier hypothesized that the reaction of dephosphorylation of C₅₅-PP could be the site of action of colicin M, a bacteriolytic toxin produced by some *E. coli* strains that kills sensitive *E. coli* strains and related species (Harkness & Braun, 1989a, b). This question was recently revisited and colicin M was in fact identified as an enzyme catalyzing the specific degradation of lipids I and II peptidoglycan intermediates (El Ghachi *et al.*, 2006).

## Undecaprenol: a storage form of lipid carrier?

The intriguing presence of free undecaprenol (C₅₅-OH) in bacterial membranes had been earlier reported in Gram-positive species. More than 90% of the endogenous C₅₅-isoprenyl lipid of *S. aureus* was found in this non-functional alcohol form (Higashi *et al.*, 1970b) and a similar situation was observed in *Enterococcus faecalis* (Umbreit *et al.*, 1972), *Listeria plantarum* (Thorne & Kodicek, 1966; Gough *et al.*, 1970), and *Listeria monocytogenes* (Vilim *et al.*, 1973). It could represent a reserve pool for the regulation of the C₅₅-P pool, an hypothesis that seemed to

be corroborated by the detection of two membrane-
associated enzyme activities, undecaprenol phosphate phosphatase, catalyzing
the interconversion of C<sub>55</sub>-OH and C<sub>55</sub>-P (steps 3 and 4
in Fig. 2), in some of the latter species (Higashi *et al.*, 1970a; Sandermann & Strominger, 1972; Willoughby *et al.*, 1972; Poxton *et al.*, 1974; Kalin & Allen, 1979). The corresponding genes however remained to be
identified.

Only one report on the characterization of a C<sub>55</sub>-P
phosphatase had been published to date (Willoughby *et al.*, 1972) but a potential involvement of nonspecific phosphatases that could explain the generation of the large pool of C<sub>55</sub>-OH was also suggested (Bohnenberger & Sandermann, 1976). The C<sub>55</sub>-P phosphatase activity detected in particulate fractions from S. aureus did not require any cation, was optimal at pH 5, and had an apparent K<sub>m</sub> for C<sub>55</sub>-P of 1.5 μM (Willoughby *et al.*, 1972). All attempts to extract it from membranes were unsuccessful and the enzyme was consequently not purified nor further characterized. In the same report, the authors mentioned the failure to detect a similar activity in particulate fractions from B. subtilis, Enterococcus faecalis, M. luteus and E. coli. This activity observed in S. aureus probably accounted for the stimulation by ATP of the overall rate of peptidoglycan synthesis observed *in vitro* with enzyme preparations from this organism (Anderson *et al.*, 1966).

A C<sub>55</sub>-OH phosphokinase activity was shown to be
extractable by butanol from S. aureus and Klebsiella aerogenes membranes (Higashi *et al.*, 1970a; Poxton *et al.*, 1974). Its activity was optimal at pH around 8.5 and was dependent on the presence of Mg<sup>2+</sup> (Higashi *et al.*, 1970a). The enzyme from S. aureus was purified to near homogeneity and its molecular weight (MW) was estimated at 14 kDa (Sander- mann & Strominger, 1972). The estimated K<sub>m</sub> value was 57 μM for both C<sub>55</sub>-OH and the nucleotide cosubstrate ATP, and ADP was confirmed as a reaction product. The phosphokinase from L. plantarum was partially solubilized by a variety of methods utilizing Triton X-100 and was characterized in some detail (Kalin & Allen, 1979). Its apparent K<sub>m</sub> values for C<sub>55</sub>-OH and ATP were estimated at 14 μM and 2 mM, respectively. No other nucleoside triphosphate was shown to substitute for ATP. Interestingly, it was very recently suggested that the diacylglycerol kinase (DGK) from Streptococcus mutans could also use C<sub>55</sub>-OH as an alternative substrate (Lis & Kuramitsu, 2003). Although this was not unambiguously established, the physiological significance of this putative undecaprenol kinase activity of DGK was further supported by an increased susceptibility to bacitracin of the dgk mutant strain as compared with that of the parental strain. It could thus be hypothesized that the C<sub>55</sub>-OH phosphokinase activity that had been purified from S. aureus membranes about 30 years ago was due, at least in part, to the DGK enzyme. The S. aureus dgk gene codes for a 114-residues protein with a MW of 12.97 kDa, a value close to that (14 kDa) earlier estimated for the C<sub>55</sub>-OH phospho- kinase from this organism (Sandermann & Strominger, 1972), but no evidence that this protein also exhibits C<sub>55</sub>- OH phosphokinase activity has been provided. It should be noted that the DGK from E. coli was proved to be inactive on C<sub>55</sub>-OH (Bohnenberger & Sandermann, 1979; Lis & Kuramitsu, 2003). Moreover, C<sub>55</sub>-OH has never been detected in E. coli cells membranes, except in nonphysiological conditions, e.g. following enzymatic degradation of lipid intermediates by colicin M (El Ghachi *et al.*, 2006). The existence of a pool of C<sub>55</sub>-OH and expression of the couple of kinase and phosphatase that catalyze its interconversion with C<sub>55</sub>-P have thus only been demonstrated in a restricted number of bacterial species. Their effective implication in the regulation of the pool of C<sub>55</sub>-P and of its use for the synthesis of the different cell-wall polymers remain to be demonstrated.

### Lipid I biosynthesis

The lipid I is an essential intermediate molecule in the peptidoglycan biosynthesis pathway (Fig. 1). Its existence was reported for the first time by Chatterjee & Park (1964) and Struve & Neuhaus (1965). Afterwards, it was identified as a lipid intermediate produced by transfer of the phospho-MurNAc-pentapeptide moiety from UDP-MurNAc-pentapeptide onto a lipid fraction (Anderson *et al.*, 1965) with concomitant release of UMP. The elucidation of the lipid structure had been achieved by Higashi *et al.* (1967), who found that the acceptor was a C<sub>55</sub> isoprenoid alcohol phosphate, undecaprenyl phosphate (C<sub>55</sub>-P). However, as detailed in ‘Undecaprenyl phosphate metabolism’, the length and stereochemistry of this carrier lipid appeared to be slightly different in certain bacterial species, such as M. smegmatis (Mahapatra *et al.*, 2005) and P. denitrificans (Ishii *et al.*, 1986). In E. coli, the pool of the lipid I was estimated at about 700 molecules cell<sup>−1</sup> (van Heijenoort *et al.*, 1992). Such an extremely low pool level was explained by the fact that this compound is an intermediate in the pathway whose synthesis and utilization reactions are efficiently coupled.

### Translocase I reaction and identification of the mraY gene

In 1965, Strominger and Neuhaus laboratories demonstrated for the first time the transfer of the phospho-MurNAc-pentapeptide moiety from the soluble UDP nucleotide precursor onto the C<sub>55</sub>-P carrier lipid, using membrane preparations from S. aureus and M. luteus

Biosynthesis of peptidoglycan lipid intermediates

(Anderson *et al.*, 1965; Struve & Neuhaus, 1965).

\[
C_{55}-P + \text{UDP-MurNAc-pentapeptide} \leftrightarrow \text{UMP}
\]
\[
+ C_{55}-PP-\text{MurNAc-pentapeptide (lipid I)}
\]

This reaction, which is generally referred to as the ‘transfer reaction’, does not lead to any modification of the basal MurNAc-pentapeptide structure synthesized by the Mur synthetases in the cytoplasm. It essentially consists in its translocation onto the $C_{55}$ carrier lipid present in the cytoplasmic membrane. This anchoring is required before subsequent steps could occur, i.e. the addition of the GlcNAc residues by the MurG transferase, the passage of the peptidoglycan monomeric structures through the membrane and finally their polymerization on the outer side of the cytoplasmic membrane. The enzyme catalyzing this first transfer/translocation reaction, the phospho-MurNAc-pentapeptide translocase or MraY (E.C. 2.7.8.13), therefore insures the link between the cytoplasmic and periplasmic steps of peptidoglycan biosynthesis (van Heijenoort, 2001b; Bugg *et al.*, 2006).

MraY was also found to be able to exchange radio-labelled UMP for the unlabelled UMP moiety of UDP-MurNAc-pentapeptide, consistent with the overall reaction referred to as the ‘exchange reaction’:

\[
[^{14}\text{C}] \text{UMP} + \text{UDP-MurNAc-pentapeptide} \leftrightarrow \text{UMP}
\]
\[
+ [^{14}\text{C}] \text{UDP-MurNAc-pentapeptide}
\]

The *mraY* gene had been identified by Ikeda *et al.* (1991) within a large cluster of genes, named *mra* for ‘murein region A’, located at 2 min on the chromosome map of *E. coli*. In this region the genes are tightly packed and appear in the order: *pbpB-murE-murF-mraY-murD-ftsW-murG-murC-ddlB-ftsQ-ftsA-ftsZ-envA*. They all code for proteins involved in peptidoglycan biosynthesis and cell division (Mur synthetases MurC/D/E/F, penicillin-binding protein PBP3; division proteins FtsW/Q/A/Z, MurG transferase, and D-Ala-D-Ala ligase DdlB). A quite similar organization of this cluster was found in other bacterial species. Ikeda *et al.* (1991) showed that the overexpression of the *E. coli mraY* gene resulted in an increase of the UDP-N-acetylmuramoyl-pentapeptide: undecaprenyl-phosphate N-acetylmuramoyl-pentapeptide transferase activity in membranes, demonstrating that this gene encoded the latter activity. Expression of the *mraY* gene was shown to be dependent on the $P_{mra}$ promoter (Hara *et al.*, 1997; Mengin-Lecreulx *et al.*, 1998). This gene is essential for the bacterial viability (Boyle & Donachie, 1998; Thanassi *et al.*, 2002) and a conditional *mraY* mutant strain was shown to accumulate peptidoglycan nucleotide precursors under restrictive growth conditions (Lara *et al.*, 2005). One copy of the *mraY* gene was found in all bacterial genomes sequenced to date but was not detectable in eukaryotic organisms and archaeabacteria which both lack peptidoglycan. Interestingly,

however, the presence of an *mraY* gene orthologue was recently demonstrated in the plant *Arabidopsis thaliana* (Mondego *et al.*, 2003). These authors postulated that this MraY-like product could participate in the biosynthesis of specific proteoglycan arabinogalactan proteins that reach peak expression during late flower bud development. It was also reported that the *mraY* gene product from *A. thaliana* had putative plastid-targeting signals (Machida *et al.*, 2006).

**MraY protein, a member of the polyphospho-N-acetyl hexosamine 1-phosphate transferase superfamily**

The alignment of MraY orthologue sequences from both Gram-negative and Gram-positive species available in databases allowed Bouhss *et al.* (1999) to identify a set of five well-conserved hydrophilic sequences (I–V) containing 34 invariant amino acid residues (Fig. 5). In the *E. coli* MraY sequence these domains were defined as H65-L80 (I), I111-K133 (II), N189-L200 (III), L251-G275 (IV) and V296-F342 (V). The size of the MraY protein appears to be fairly well conserved in the bacterial world, the sequences from Gram-negative species being generally longer than those found in Gram-positive species (~360 vs. 320 amino acid residues), owing mainly to an extension at the N-terminal extremity. Unusually large MraY proteins (~420 residues) are found, however, in some bacteria such as Bacteroides (Price & Momany, 2005). The presence of alternating hydrophobic and hydrophilic segments in its primary structure clearly suggested that MraY was an integral membrane protein spanning the cytoplasmic membrane several times (Ikeda *et al.*, 1991). Moreover, a lipid microenvironment was shown to be required for the MraY activity (Heydanek & Neuhaus, 1969; Umbreit & Strominger, 1972a; Geis & Plapp, 1978; Weppner & Neuhaus, 1979). More recently, Bouhss *et al.* (1999) determined the two-dimensional membrane topology of both the *E. coli* and *S. aureus* MraY translocases, using β-lactamase fusion experiments. A common topological model was proposed which contained ten transmembrane segments joining four periplasmic loops and five cytoplasmic sequences corresponding to the conserved hydrophilic patterns I–V (Fig. 5). Both the N- and C-terminal extremities were located in the periplasmic space. This model was in agreement with many structural features predicted from a sequence comparison of MraY orthologues, strongly suggesting its validity for all eubacterial MraY proteins (Bouhss *et al.*, 1999).

The cytoplasmic sequences II, III and IV of MraY were also found in other prokaryotic and eukaryotic proteins catalyzing the same kind of reaction, namely the transfer of a lipid phosphate to the β-phosphate of an UDP-linked hexosamine. These three conserved patterns thus defined a superfamily of enzymes termed polyphospho-

Fig. 5. Membrane topology of the MraY translocase (*Escherichia coli*). The topological model consists in ten transmembrane segments, four periplasmic loops and five cytoplasmic sequences corresponding to the conserved hydrophilic patterns I–V (Bouhss *et al.*, 1999). Conserved residues by identity (red) and by similarity (blue) are indicated.

Purification and biochemical characterization of MraY

Solubilization, expression, and purification of MraY

The availability of a pure, stable, soluble and active preparation of the integral membrane MraY protein was a prerequisite to the development of detailed biochemical investigations. Since 1969, many soluble and active preparations of MraY were described, extracted from membranes of various bacteria such as micrococci, *S*. *aureus* and *E*. *coli* using detergents and in particular Triton X-100 or CHAPS (Heydanek & Neuhaus, 1969; Umbreit & Strominger, 1972a; Pless & Neuhaus, 1973; Brandish *et al.*, 1996a; Breukink *et al.*, 2003; Lloyd *et al.*, 2004; Stachyra *et al.*, 2004). Upon detergent extraction, the *S*. *aureus* MraY was shown to require the presence of a phospholipid, either phosphatidylcholine, dioleoyl phosphatidyl choline or phosphatidylglycerol (Pless & Neuhaus, 1973). Brandish *et al.* (1996a) reported that overexpressed *E*. *coli* MraY protein was preferentially activated by phosphatidylglycerol. All previous attempts to overexpress significantly and purify any MraY protein had been unsuccessful. Thus, only partially purified enzyme preparations were generally used for enzymatic assays (Brandish *et al.*, 1996a; Zawadzke *et al.*, 2003; Lloyd *et al.*, 2004; Stachyra *et al.*, 2004). Recently, a comparative study performed with recombinant MraY proteins from *E*. *coli*, *S*. *aureus*, *B*. *subtilis* and *Thermotoga maritima*, expressed in the *E*. *coli* C43(DE3) host strain, identified *n*-dodecyl-β-ᴅ-maltoside and N-lauroyl-sarcosine as the most efficient detergents for the extraction of this protein from cell membranes (Bouhss *et al.*, 2004). In the

© 2007 Federation of European Microbiological Societies
Published by Blackwell Publishing Ltd. All rights reserved
same report, conditions allowing the high-level overexpression of a MraY protein and, for the first time, its purification to homogeneity in milligram quantities were described. The specific activity of the pure *B. subtilis* MraY protein was estimated at 1900 nmol min⁻¹ mg⁻¹ (Bouhss et al., 2004).

### Substrate specificity and kinetic properties

MraY has two substrates: C₅₅-P and UDP-MurNAc-pentapeptide. The structure of the lipid substrate is expected to be conserved in most bacterial species but a few exceptions exist, as mentioned in ‘Undecaprenyl phosphate metabolism’. The structure of the sugar nucleotide substrate shows important variations in the bacterial world, particularly in the peptide moiety (Schleifer & Kandler, 1972; Barreteau et al., 2008; Vollmer et al., 2008). This peptide is a pentapeptide that generally contains at the third position either a meso-A₂pm residue in Gram-negative bacteria (as *E. coli*) and *Bacillus* species or a lysine residue (more rarely an ornithine) in most Gram-positive bacteria (as *S. aureus*). In vivo complementation experiments have shown that the *S. aureus* MraY was functional in *E. coli* and restored growth of a *mraY* thermosensitive mutant, indicating that it accepts the A₂pm-containing sugar nucleotide (Bouhss et al., 1999). Similarly, the overexpression of the *S. aureus* MurE synthetase in *E. coli* (which introduces lysine instead of A₂pm at the third position of the nucleotide substrate) resulted in a massive incorporation of lysine into the peptidoglycan, demonstrating the efficient utilization of the lysine-containing UDP-MurNAc-pentapeptide by the *E. coli* MraY enzyme (Mengin-Lecreulx et al., 1999). The relatively low specificity of the MraY translocase and its tolerance toward the variability of the peptide chain was a well-known characteristic (Hammes & Neuhaus, 1974) that has been confirmed in various other circumstances both in vivo and in vitro. Shorter or longer peptides as well as modified peptides were shown to be accepted: dipeptides (Ornelas-Soares et al., 1994), tripeptides (Hammes & Neuhaus, 1974; Pisabarro et al., 1986; van Heijenoort et al., 1992), tetrapeptides (Hammes & Neuhaus, 1974), acetylated and dansylated pentapeptides (Ward & Perkins, 1974; Weppner & Neuhaus, 1977; Brandish et al., 1996a; Stachyra et al., 2004), as well as hexa- and heptapeptides (Billot-Klein et al., 1997). More recently, the MraY enzyme from *T. maritima* was shown to act on two different substrates in vivo: a D-lysine-containing UDP-MurNAc-tripeptide and an L-lysine-containing UDP-MurNAc-pentapeptide, with similar efficiencies (Boniface et al., 2006).

The replacement of the L-Ala and D-Ala residues at positions 1 and 4 of the pentapeptide chain, respectively, by glycine reduced the S. aureus MraY catalytic efficiency by 135-fold (Hammes & Neuhaus, 1974). The use of UDP-MurNAc-tetrapeptide (lacking one D-Ala) and UDP-Mur-

NAc-tripeptide (lacking two D-Ala) as substrates reduced the MraY catalytic efficiency by fourfold and 80-fold, respectively, as compared with the nucleotide pentapeptide. Stickgold & Neuhaus (1967) determined that 5-fluorouracil-substituted UDP-MurNAc-pentapeptide was utilized at < 2% of the rate of the native substrate and was a competitive inhibitor (*K*ᵢ = 0.12 mM) of the transfer reaction, just as 5-fluoro-UMP is a competitive inhibitor (*K*ᵢ = 50 μM) of the exchange reaction. However, 2′-deoxy UMP was accepted as a good substrate in the exchange reaction (Neuhaus, 1971).

Various MraY assays using radiolabeled or fluorescent substrates, resolution of reaction mixtures by paper or thin-layer chromatography, or direct measurement using microplates, have been developed for analyzing the kinetic properties of this enzyme and for the screening of inhibitors (Weppner & Neuhaus, 1977; Brandish et al., 1996a,b; Bouhss et al., 2004; Stachyra et al., 2004). Typical Michaelis–Menten kinetics were observed with both purified and partially purified enzyme preparations (Anderson et al., 1966; Struve et al., 1966; Stickgold & Neuhaus, 1967; Hammes & Neuhaus, 1974; Brandish et al., 1996a; Bouhss et al., 2004; Stachyra et al., 2004). *K*ₘ values observed for UDP-MurNAc-pentapeptide were generally in the 5–30 μM range with nonpurified enzyme preparations. However, a much higher value of 1 mM was determined with the purified MraY enzyme from *B. subtilis* (Bouhss et al., 2004).

The structure and size of the C₅₅-P lipid acceptor substrate present in the membranes is essentially determined by the substrate specificity and catalytic properties of the undecaprenyl pyrophosphate synthase UppS. However, the *E. coli* MraY translocase was shown to accept in vitro heptaprenyl (C₃₅) as well as dodecaprenyl (C₆₀) phosphate as alternative substrates, with *K*ₘ values (10–20 μM) similar to that of the natural substrate (Brandish et al., 1996a). Breukink et al. (2003) have reported that the *Micrococcus flavus* MraY was active in vitro on polyisoprenyl substrates containing from two to 25 isoprenyl units, with a maximal efficiency observed for substrates bigger than C₃₅. The purified *B. subtilis* MraY exhibited a *K*ₘ for C₅₅-P about 10-fold higher than that of the nonpurified MraYs from other bacterial species (Bouhss et al., 2004). It could thus be assumed that the membrane environment affects the recognition of the lipid substrate by the enzyme.

The MraY activity is dependent on the presence of both mono- and divalent metal ions. The activity of the *E. coli* and *B. subtilis* enzymes was stimulated (two- to fourfold) in presence of 10–100 mM of potassium or sodium (Brandish et al., 1996a; Bouhss et al., 2004). Heydanek et al. (1970) also observed that the *S. aureus* MraY was stimulated by many monovalent metal ions such as K⁺, Rb⁺, Cs⁺ and NH₄⁺. The MraY enzyme has an absolute requirement for a divalent metal ion, particularly Mg²⁺, at 5–40 mM concentrations

FEMS Microbiol Rev **32** (2008) 208–233

© 2007 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
Two-step catalytic mechanism

Uridine-O-P-O-P-O-MurNAc-peptide $\xleftrightarrow{\text{UMP}}$ MraY-O-P-O-MurNAc-peptide $-O-P-O-C_{55}$ MraY $C_{55}-O-P-O-P-O-$MurNAc-peptide

One-step catalytic mechanism

Uridine-O-P-O-P-O-MurNAc-peptide $-O-P-O-C_{55}$ MraY UMP $C_{55}-O-P-O-P-O-$MurNAc-peptide

Fig. 6. Proposed MraY translocase reaction mechanisms. The two-step mechanism involves a covalent enzyme-phospho-MurNAc-pentapeptide intermediate and the single-step mechanism consists in a direct attack of the phosphate oxyanion of $C_{55}$-P onto the β-phosphate of the nucleotide substrate.

(Heydanek *et al.*, 1970; Bouhss *et al.*, 2004). Mn²⁺ can replace Mg²⁺ but the activity is decreased by two orders of magnitude (Bouhss *et al.*, 2004). Lloyd *et al.* (2004) proposed that the D115 or D116 aspartate residues of *E. coli* MraY, which are located in the conserved hydrophilic sequence II (Fig. 5) and whose replacement by Asn leads to loss of catalytic activity, are involved in Mg²⁺ chelation.

### Catalytic mechanism of MraY

In 1969, on the basis of kinetic evidences, Heydanek *et al.* (1969) proposed a two-step catalytic mechanism for the MraY reaction (Fig. 6). It consisted in an attack by a nonidentified nucleophile residue of MraY on the β phosphate of UDP-MurNAc-pentapeptide, generating a covalent enzyme-phospho-MurNAc-pentapeptide intermediate with concomitant release of UMP. The second step corresponded to the attack by an oxyanion from $C_{55}$-P on the phosphate of the covalent intermediate, resulting in the formation of lipid I and regeneration of the native enzyme form. Recently, Lloyd *et al.* (2004) proposed that the D267 aspartate residue of *E. coli* MraY, which is located in the conserved hydrophilic sequence III, plays the role of the catalytic nucleophile. Three observations supported the two-step mechanism: (1) phospho-MurNAc-pentapeptide was formed during the transfer reaction that likely resulted from the hydrolysis of the enzyme-linked phospho-MurNAc-pentapeptide intermediate; (2) the enzyme could catalyze the exchange of UMP with the UMP moiety of UDP-MurNAc-pentapeptide in the presence or absence of the cosubstrate $C_{55}$-P (Pless & Neuhaus, 1973); (3) dodecylamine inhibited the synthesis of lipid I and caused the release of the phospho-MurNAc-pentapeptide.

A major criticism of the experiments performed by Heydanek *et al.* that could have resulted in a misinterpretation of the results was the use of a nonpurified MraY enzyme. The different observations they made could effectively be attributed to the presence of contaminating enzymes (phosphatases) and $C_{55}$-P lipid acceptor in the preparation used. An alternative mechanism consists in a direct attack of the phosphate oxyanion of $C_{55}$-P onto the β phosphate of UDP-MurNAc-pentapeptide. This would lead to the formation of lipid I and UMP in only one step (Fig. 6). The recent availability of pure MraY protein (Bouhss *et al.*, 2004) will allow to revisit this question and elucidate the catalytic mechanism.

### Recent progress on inhibitors of the MraY-catalysed reaction

MraY is the target for several classes of natural product antibiotics, which in some cases have been studied in detail through structure/function studies and/or detailed kinetic or mechanistic studies. The structures of the MraY inhibitors have been reviewed in detail (Kimura & Bugg, 2003; Dini, 2005; Bugg *et al.*, 2006). This section will briefly survey the different inhibitor classes, and small molecule inhibitors arising from structure/function studies.

There are five different classes of uridine-containing nucleoside natural antibiotic products that target MraY, illustrated in Fig. 7. The tunicamycins (related to the streptoviridins and corynetoxins) contain a uridine disaccharide, attached to a fatty-acyl chain. Tunicamycin is a reversible, competitive inhibitor of MraY, with $K_i$ 0.6 μM (Brandish *et al.*, 1996b), but is not useful as an antibacterial agent, since it is toxic to mammals, through potent inhibition of GlcNAc-1-P transferase in the dolichol cycle of N-linked glycoprotein biosynthesis (Heifetz *et al.*, 1979).

Biosynthesis of peptidoglycan lipid intermediates

Tunicamycin

![Chemical Structure 1](chemical_structure_1.png)

Liposidomycin B

![Chemical Structure 2](chemical_structure_2.png)

Mureidomycin A

![Chemical Structure 3](chemical_structure_3.png)

Muraymycin B1

![Chemical Structure 4](chemical_structure_4.png)

Capuramycin

![Chemical Structure 5](chemical_structure_5.png)

Fig. 7. Structure of MraY translocase inhibitors.

mureidomycins (related to the pacidamycins and napsamy- cins) are peptidyl nucleoside natural products, containing a 3′-deoxyuridine sugar attached via an enamide linkage to an unusual peptide chain. The peptide chain contains an N-methyl 2,3-diaminobutyric acid residue, and a urea linkage to a C-terminal aromatic amino acid, which can be meta-tyrosine, Trp, or Phe. Mureidomycin A is a slow-binding inhibitor of E. coli MraY, with \( K_{\mathrm{i}}^{*} = 2.2 \, \mathrm{nM} \) (Brandish et al., 1996a). The enamide functional group is not essential for inhibition, and a range of dihydro-pacidamycin analogues have been synthesized and tested for biological activity (Boojamra et al., 2001). The mechanism of action of mureidomycin A has been studied, and it was proposed that the amino-terminus binds to the \(\mathrm{Mg}^{2+}\) cofactor-binding site, and is positioned by an \(N\)-methyl amide cis-amide bond (Howard & Bugg, 2003). A uridyl dipeptide analogue of mureidomycin A retained biological activity against *Pseudomonas putida* (Howard & Bugg, 2003).

The liposidomycins are fatty acyl nucleosides, whose structures contain a sulfated aminoglycoside residue. Liposidomycin B is a slow-binding inhibitor (\(K_{\mathrm{i}}^{*} = 80 \, \mathrm{nM}\)) of solubilized E. coli MraY (Brandish et al., 1996b). A synthetic analogue containing the aminoribofuranose monosaccharide attached to the 5′ position of uridine showed moderate levels of inhibition (\(\mathrm{IC}_{50} = 50 \, \mu \mathrm{M}\)) against translocase I when assayed in toluene-permeabilized E. coli cells (Dini et al., 2000; Stachyra et al., 2004). Further structure–function studies in this series of compounds

(riburamycins) gave more active inhibitors, and a synthetic analogue containing a \(C_{12}\)-fatty-acyl chain showed antibacterial activity against *S. aureus* (Dini et al., 2002; Stachyra et al., 2004).

The muraymycins, reported by Wyeth Research in 2002, contain an aminoribofuranoside monosaccharide, attached to a short peptide chain, containing a urea linkage to a C-terminal amino acid (McDonald et al., 2002). Members of this family were potent inhibitors of MraY in vitro (\(\mathrm{IC}_{50} = 0.027 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)), showed antibacterial activity against *S. aureus* (\(\mathrm{MIC} = 2-16 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)) and enterococci (\(\mathrm{MIC} = 16-64 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)) and were able to protect mice against *S. aureus* infection (\(\mathrm{ED}_{50} = 1.1 \, \mathrm{mg} \, \mathrm{kg}^{-1}\)) (McDonald et al., 2002). Synthetic analogues lacking the aminoribofuranoside showed reduced activity in vitro, but retained some antibacterial activity (Yamashita et al., 2003).

Capuramycin, a further uridine nucleoside antibiotic, is a potent inhibitor of MraY in vitro (\(\mathrm{IC}_{50} = 0.017 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)), and a methylated derivative showed antibacterial activity against *M. smegmatis* (\(\mathrm{MIC} = 2-16 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)) (Muramatsu et al., 2003). Acylation derivatives of capuramycin showed very potent activity (\(\mathrm{MIC} = 0.06 \, \mu \mathrm{g} \, \mathrm{mL}^{-1}\)) against several *Mycobacterium* strains (Hotoda et al., 2003).

Genetic studies by Young and coworkers showed that MraY was also the target for the bacteriolytic E protein from bacteriophage φX174 (Bernhardt et al., 2000). The E protein is a 91-amino acid protein, containing a transmembrane domain. The killing action of E also required a host protein

SlyD, a peptidyl-prolyl isomerase. Recent studies on the mechanism of action of E showed that a 37-amino acid peptide containing the transmembrane domain of E was a potent inhibitor of membrane-bound MraY (IC₅₀ = 0.8 μM), but did not inhibit solubilized MraY, unlike the small molecule inhibitors (Mendel *et al.*, 2006). It has been proposed that E inhibits MraY via a protein–protein interaction, blocking the formation of protein–protein interactions between MraY and other cell wall assembly proteins in the cytoplasmic membrane at cell division (Mendel *et al.*, 2006).

Interestingly, it was recently demonstrated that colicin M was an enzyme acting by specifically targeting and destroying the peptidoglycan lipid intermediates, thereby provoking the arrest of peptidoglycan synthesis and cell lysis (El Ghachi *et al.*, 2006). The cleavage site was located between the C₅₅ and pyrophosphoryl groups, as demonstrated *in vitro* by appropriate assays and *in vivo* by the observation that colicin M-treated cells accumulated C₅₅-OH, a lipid form that is normally not detectable in *E. coli* cell membranes (El Ghachi *et al.*, 2006). To the authors’ knowledge this is the first example of a mechanism of inhibition of peptidoglycan biosynthesis occurring via enzymatic degradation of its precursors, in this case the MraY and MurG lipid reaction products.

### Lipid II biosynthesis

#### The translocase II reaction and identification of the murG gene

The translocase II (MurG) catalyses the second membrane associated step of peptidoglycan synthesis (Fig. 1). This glycosyl transferase of the GT-B superfamily transfers the GlcNAc moiety from UDP-GlcNAc to the C4 hydroxyl group of lipid I to form a β-linked disaccharide (lipid II). It was originally identified in *E. coli* but has since been recognized in all other bacteria that make peptidoglycan.

The *murG* gene was first discovered by Salmond *et al.* (1980) who identified, cloned and mapped the gene within the *mra* region of *E. coli*. The nucleotide sequence of the *E. coli murG* coding region (Mengin-Lecreulx *et al.*, 1990) revealed an ORF of 1065 nucleotides theoretically coding for a moderately hydrophobic 37.8 kDa protein. When cultures of a thermosensitive *murG* mutant strain (GS58) growing exponentially at 30 °C were shifted to the nonpermissive temperature of 42 °C, cells progressively lost their rod shape, became ovoid with a greatly increased volume and finally lysed (Mengin-Lecreulx *et al.*, 1991). This mutant accumulated significant amounts of UDP-MurNAc-pentapeptide, UDP-GlcNAc and to a lesser extent a lipid compound as labelled A₂pm-containing precursors, suggesting that the mutational block was in a membrane step. The fact that the ratio of the two lipid intermediates I and II, which had previously been reported to be between 0.3:1 and 0.6:1 in *E. coli* strains (Ramey & Ishiguro, 1978), was reversed and reached the considerably high value of 8.2:1 in this mutant clearly indicated that it was the second membrane step that was altered. It was consequently deduced that *murG* encoded the UDP-GlcNAc:MurNAc-(pentapeptide)-pyrophosphoryl-undecaprenol GlcNAc transferase (EC 2.4.1.227) which catalyses the formation of lipid II from lipid I. This protein was recently shown to interact with the MraY and MreB proteins and was suggested to participate in two multi-protein complexes involved in cell elongation and cell division, respectively (Aaron *et al.*, 2007; Mohammadi *et al.*, 2007; den Blaauwen *et al.*, 2008).

### Purification and biochemical characterization of MurG

Crouvoisier *et al.* (1999) first reported the overproduction, solubilization and purification of the protein. MurG was solubilized using salt or detergent from the cell membranes and purified using anion exchange for the wild-type protein or Ni²⁺ affinity for the His-tagged protein. Ha *et al.* (2000) also reported a similar procedure where the protein was solubilized with detergent directly from cell pellets of *E. coli* and purified using cation exchange followed by gel filtration. van den Brink-van der Laan *et al.* (2003) showed that the cardiolipin content of *E. coli* cell membranes was increased following overexpression of MurG and that lipid vesicles copurified with MurG. The activity of MurG was increased in the presence of cardiolipin, suggesting specific interactions of the protein with phospholipids and in particular with cardiolipin.

Using optimal conditions in HEPES buffer, pH 7.9, supplemented with 5 mM MgCl₂ or MnCl₂, Ha *et al.* (1999) determined the kinetic parameters for *E. coli* MurG. Because lipid I is almost impossible to isolate from bacterial cells and difficult to handle, a synthetic lipid I analogue was used. The assay used radiolabelled UDP-[¹⁴C]GlcNAc and a biotin-labelled lipid I analogue. A biotin capture technique was used to sequester products from the reaction which were then counted for bound radioactivity. The parameters were *K*<sub>UDP-GlcNAc</sub> = 58 ± 30 μM, *K*<sub>lipid I analogue</sub> = 37 ± 4 μM, *k*<sub>cat</sub> = 16 ± 2 min⁻¹ in the presence of MgCl₂. Using a different (C₃₅) lipid I analogue and a new assay based on HPLC, Auger *et al.* (2003) showed that a high concentration (35%) of dimethylsulfoxide was necessary for maximal enzyme activity. The kinetic constants they determined in these conditions were: *K*<sub>m</sub> UDP-GlcNAc = 150 ± 20 μM, *K*<sub>m</sub> lipid I analogue = 2.8 ± 1 μM, *k*<sub>cat</sub> = 56 ± 5 min⁻¹. Ha *et al.* (1999) also investigated the acceptor and donor substrate specificity for MurG. The results showed that a biotin-labelled UDP-MurNAc-pentapeptide

Biosynthesis of peptidoglycan lipid intermediates

was also a substrate for the enzyme with a relative rate of c. 20% when compared with the biotin-labelled lipid I analogue. The results also revealed that the enzyme was sensitive to the identity of the nucleotide, and required the presence of the diphosphate linkage. The enzyme was inhibited by UDP, but not by UMP, nor any other nucleotide diphosphate. The enzyme also showed high specificity for the equatorial stereochemistry at the C4 position of the donor. Therefore, UDP-N-acetylgalactosamine failed to show inhibitory activity even at millimolar concentrations and showed very little donor activity. Liu *et al.* (2003) reported the development of a continuous fluorescence coupled enzymatic assay in which the formation of UDP was coupled to a pyruvate kinase-lactate dehydrogenase assay and the fluorescence signal of NADH monitored. This group tested a variety of lipid I analogues, with various substituents replacing the undecaprenyl moiety, as substrates. MurG accepted all of them with \( K_{\mathrm{m}} \) values of around 20–50 μM but there were, however, large differences in the specific activity (\( k_{\mathrm{cat}} \)). Acceptor substrates with long saturated linear alkyl chains were better substrates than the natural lipid I demonstrating an increase in \( k_{\mathrm{cat}} \) from \( 11 \pm 2 \, \mathrm{min}^{-1} \) for the natural substrate to up to \( 180 \pm 13 \, \mathrm{min}^{-1} \) for an analogue with a \( \mathrm{C}_{14} \mathrm{H}_{29} \) chain.

Using synthetic substrate analogues and products containing different length lipid chains, as well as a synthetic dead-end acceptor analogue, Chen *et al.* (2002a) showed that MurG follows an ordered Bi-Bi mechanism in which UDP-GlcNAc binds first (Fig. 8). The enzyme likely utilizes a mechanism that involves partial participation of the lone pair on the sugar ring oxygen and therefore an oxocarbenium-ion-like transition state. Evidence exists that supports the procedure of glycosyltransferase-mediated reactions through an oxocarbenium-ion-like transition state, similar to that proposed for similar enzymes (Singh *et al.*, 1987; Kim *et al.*, 1988; Takayama *et al.*, 1999). Using a synthetic radiolabelled-analogue of lipid II, Auger *et al.* (2003) reported for the first time that the reaction catalyzed by MurG was reversible.

### 3-D structure of the MurG protein

The 1.9 Å crystal structure of *E. coli* MurG (Ha *et al.*, 2000) revealed that the free enzyme consists of two protein molecules in an asymmetric unit. Each protein chain has two domains separated by a cleft which is c. 20 Å deep and 18 Å across at its widest point. Each domain adopts a \( \alpha / \beta \) open sheet motif which is characteristic of domains that bind nucleotides. The N-terminal domain contains seven parallel \( \beta \)-strands and six \( \alpha \)-helices. The C-terminal domain contains six parallel \( \beta \)-strands and eight \( \alpha \)-helices including one irregular bipartite helix that connects the N-domain to the first \( \beta \)-strand of the C-domain. The structural homology between the two domains is high, despite low sequence homology. A subsequent, 2.5 Å crystal structure of *E. coli* MurG complexed with the donor substrate, UDP-GlcNAc, was reported in 2003 (Hu *et al.*, 2003a) (Fig. 9). Before this, no X-ray crystal structures containing intact substrates had been obtained for any of the NDP-glycosyltransferase superfamily. The substrate bound enzyme was reported to have only a 16 Å wide cleft, c. 2 Å narrower than the free enzyme. UDP-GlcNAc binds in both protein subunits along with four glycerol molecules and 121 water molecules. Only one of the UDP-GlcNAc molecules is in

![Chemical Structure](https://i.imgur.com/structure.png)

Fig. 8. Proposed MurG reaction mechanism. The model predicts a deprotonation of the C4 hydroxyl group of the MurNAc moiety of lipid I by a residue of MurG (probably His18). The oxyanion thus generated then attacks the C1 of GlcNAc of the nucleotide substrate to form the oxocarbenium-ion-like transition state. Finally, UDP is released generating lipid II. R = –C(CH₃)CO-ʟ-Ala-γ-ᴅ-Glu-meso-A₂pm-ᴅ-Ala-ᴅ-Ala, Un = undecaprenyl.

Fig. 9. Structure of *Escherichia coli* MurG complexed with UDP-GlcNAc. This structure, drawn with Swiss PDB Viewer, shows the interactions between UDP-GlcNAc and the enzyme active site. Binding of the lipid I substrate is presumed to take place in the cleft between the two domains of the MurG structure. From Hu *et al.* (2003a).

the correct orientation for catalytic activity and is presumed to be the Michaelis complex where the UDP is displaced by the incoming nucleophile. MurG contains a sequence motif that is found in most members of the GT-B superfamily. This α/β/α subunit has two α-helices located near to the cleft between the domains. The UDP-GlcNAc substrate makes several contacts to these helices and also to the loops that connect them to the adjacent β-strands. The GlcNAc moiety of the donor substrate makes contacts with the invariant residues of MurG through hydrogen bonding interactions between the backbone amide of the A263 residue, the side chain amides of N291 and N127 and the C4 hydroxyl group of GlcNAc. Interactions also exist between the side chain amide of Q287 to both the C3 and C4 hydroxyl groups of GlcNAc. The catalytic base appears to be histidine H18 situated 9.52 Å across the domain cleft in line with the anomeric bond. The contacts between the enzyme and the diphosphate are purely hydrogen bonding. S191 has been shown to be an important residue whose mutation to alanine affects all kinetic parameters including an increase in the $K_{\mathrm{m}}$ for lipid I binding from 0.053 to 0.179 mM. This serine residue is located on a GGS loop that is conserved in all MurG homologs which moves up towards the donor substrate as it binds. Because MurG is believed to utilise a sequential mechanism in which lipid I binds following UDP-GlcNAc (Chen *et al.*, 2002a), it is thought that conformational changes in the GGS loop may play a role in the adjustments required for lipid I binding, as well as directly contributing to UDP-GlcNAc binding. The uracil nucleotide is anchored in a pocket by hydrogen bonds from the backbone amide of I244 to the N₃H and O₄ atoms. There is also a possible interaction between the uracil O₂ atom and R163. The F243 residue also rotates, as the substrate binds, to cap the binding pocket. Contacts between the enzyme and the ribose 2′ and 3′ hydroxyl groups exist through hydrogen bonds from E268. To evaluate their role in the activity of the *E. coli* MurG enzyme, 13 residues that are invariant or highly conserved in the MurG enzyme family (T15, H18, Y105, H124, E125, N127, N134, S191, N198, R260, E268, Q288, N291) and located within or very close to the active site were recently submitted to site-directed mutagenesis (Hu *et al.*, 2003a; Crouvoisier *et al.*, 2007). Most of these mutations resulted in a great loss of enzyme activity, consistent with the interactions observed in the crystal structures of MurG alone or in complex with UDP-GlcNAc (Ha *et al.*, 2000; Hu *et al.*, 2003a). The important role of these residues either in substrate binding/catalysis or maintenance of the protein’s conformation was confirmed (Hu *et al.*, 2003a; Crouvoisier *et al.*, 2007). Interestingly, a hydrophobic patch has been identified in the MurG structure, consisting of I74, L78, F81, W84 and W115 residues, which appears to be the site of interaction between this extrinsic membrane protein and the phospholipid bilayer (Ha *et al.*, 2000).

Recent progress on inhibitors of the MurG reaction

The cyclic lipoglycodepsipeptide antibiotic ramoplanin has excellent activity against a wide range of Gram-positive bacteria. It was demonstrated that ramoplanin forms complexes with both lipids I and II (Reynolds & Somner, 1990;

Biosynthesis of peptidoglycan lipid intermediates

Lo *et al.*, 2000; Helm *et al.*, 2002; Hu *et al.*, 2003b), however, ramoplanin was also shown to directly interact with MurG although the exact mechanism of inhibition is not known (Fang *et al.*, 2006).

Using the most active lipid I analogue substrate they had identified, Liu *et al.* (2003) developed a screening for MurG inhibitors. The potential inhibitors screened were vancomycin, moenomycin and two chloro-biphenyl derivatives of vancomycin. All of these compounds were found to inhibit the enzyme with one of the vancomycin derivatives being particularly potent. These known inhibitors of the bacterial cell wall biosynthesis pathway, however, cannot penetrate the bacterial cell membrane and so they do not encounter MurG *in vivo* because the enzyme localizes on the cytoplasmic side of the cell membrane (Bupp & van Heijenoort, 1993).

A recent publication by Helm *et al.* (2003) detailed the high throughput screening of almost 50 000 compounds from a commercial library as inhibitors of MurG. The assay employed in the screening used an *N*-acyl fluoresceinated UDP-GlcNAc analogue instead of the natural substrate. As MurG was added to the labelled substrate, an increase in polarization occurred indicating that the substrate had bound. The compounds were then screened at 25 µg mL⁻¹ for *in vitro* activity as competitive inhibitors by looking for a decrease in polarization. Eleven compounds were found to reproducibly inhibit the enzyme by 50% or more. Seven of these compounds contained a five-membered, nitrogen containing heterocyclic core with an alkyl or aryl substituent at N-1 and an arylidine substituent at the 3 position. A later publication by Hu *et al.* (2004) utilized the same assay to screen a further 64 000 compounds from a variety of different compound libraries, including commercial and diversity-oriented synthesis (DOS) libraries. Fifty-five compounds were found to exhibit > 40% inhibition of MurG at a concentration of 2.5 µg mL⁻¹. Interestingly, 31 of the 55 compounds were found to contain one of four common cores (Fig. 10). It is not known exactly how these compounds bind to MurG and antibacterial activity has not yet been reported.

A direct continuous fluorescence assay for MurG based on fluorescence resonance energy transfer (FRET) was recently described (Li & Bugg, 2004). It used a dansyl-labelled lipid I (*λ*~ex~ = 500 nm, *λ*~em~ = 550 nm) and a fluorescent UDP-GlcNAc analogue (*λ*~ex~ = 290 nm, *λ*~em~ = 500 nm) as substrates for MurG. A linear relationship between enzyme concentration and the rate of increase in fluorescence was observed, indicating that an efficient energy transfer occurred as the substrates were converted to products by the enzyme. This assay is believed to be the first direct continuous fluorescence assay for MurG and may be very useful for determining the activity of potential inhibitors.


![Chemical Structures](chemical_structures.png)

Fig. 10. General structures of MurG inhibitors. From Hu *et al.* (2004). X = S, O.

### Enzymatic modification of lipid II in the bacterial world

The peptidoglycan structure of many bacteria, especially Gram-positive species, contains an additional peptide cross-link between the residue (*L*-Lys or *meso-A*₂pm in most cases) at position 3 of the pentapeptide chain and the *D*-alanine residue at position 4 of the cross-linked strand. There is significant variation in the type and structure of these cross-links across the bacterial kingdom (Schleifer & Kandler, 1972; Vollmer *et al.*, 2008). Some examples are shown in Fig. 11.

The addition of the cross-link amino acids occurs most commonly at the level of the lipid II intermediate (Fig. 11), although in *Weissella viridescens* it has been shown that it occurs at the level of the nucleotide precursor (Maillard *et al.*, 2005). Kamiryo & Matsuhashi (1972) showed that the formation of the penta-glycine cross-link in *S*. *aureus* required glycyl tRNA and lipid II. More recently, the *femABX* genes have been implicated in the addition of the penta-glycine bridge (Rohrer & Berger-Bächi, 2003): FemX is responsible for addition of the first glycine; FemA the next two glycines; and FemB the final two glycines (Rohrer & Berger-Bächi, 2003). The FemABX reactions have recently been reconstituted using purified recombinant proteins, with lipid II and glycyl tRNA as substrates (Schneider *et al.*, 2004). There is a crystal structure of *S*. *aureus* FemA, which contains a pair of extended α-helices, similar in structure to a motif found in seryl tRNA synthetase that interacts with the tRNA substrate (Benson *et al.*, 2002). A crystal structure of *W*. *viridescens* FemX complexed with its substrate UDP-MurNAc-pentapeptide has been determined, which reveals enzyme–substrate binding interactions (Biarrotte-Sorin *et al.*, 2004). Because aminoacyl tRNAs are found only in the cytoplasm, the lipid II modification reactions must occur on the cytoplasmic side of the membrane, after the

MurG-catalysed reaction, but before flipping to the cell surface.

Dipeptide Ala-Ala cross-links are found in *Enterococcus faecalis* (Bouhss et al., 2001), and in strains of highly penicillin-resistant *Streptococcus pneumoniae* (Garcia-Bustos & Tomasz, 1990), in the latter case linked to the *murMN* genes (Filipe & Tomasz, 2000). The *murMN* genes are found in all pneumococci, but specific sequences of *murM* are found in highly resistant strains (Filipe et al., 2000). Two transferases involved in the formation of the Ala–Ala cross-link in *Enterococcus faecalis* have also been identified, which are able to add Ala-tRNA to UDP-MurNAc-pentapeptide in vitro (Bouhss et al., 2002). The *Streptococcus pneumoniae* MurM reaction has recently been reconstituted in vitro using recombinant MurM, lipid II, and alanyl-tRNA (Lloyd et al., 2007). MurM from a penicillin-resistant strain was found to show 80-fold higher \(k_{cat}/K_m\) than MurM from a sensitive strain, providing an explanation for the higher level of cross-links in the resistant strains (Lloyd et al., 2007). The ATP-dependent enzyme responsible for addition of D-aspartic acid to L-lysine residue in *Enterococcus faecium* has also been identified (Bellais et al., 2006).

In a number of bacterial strains, the D-glutamic acid residue at position 2 of the pentapeptide chain is amidated, to form D-isoglutamine. In *S. aureus*, it is known that this

amidation is dependent upon ATP and ammonia, and that either lipid I or II can be amidated in vitro (Siewert & Strominger, 1968). A membrane-bound amidating activity has also been observed in *Bacillus stearothermophilus* that is able to amidate the nucleotide precursor (Linnett & Strominger, 1974). Lipid II isolated from *M. smegmatis* has been found to contain several modifications: the MurNAc residue is modified as N-glycolylmuramic acid, or as de-acylated muramic acid; the diaminopimelic acid residue is amidated; and the terminal D-alanine residue is methylated (Mahapatra et al., 2005). It appears that a number of modification reactions occur in particular bacterial strains, most commonly at the level of the lipid-linked intermediates (Fig. 11). In *S. aureus*, the acceptor substrate of sortase A that catalyzes anchoring of surface proteins to peptidoglycan was identified as lipid II (Perry et al., 2002). These different modifications should be taken into account when considering the lipid II as a potential target for new antibacterial agents (Breukink & de Kruijff, 2006).

## Chemical and enzymatic synthesis of lipids I and II and analogues

In order to reconstitute in vitro the lipid-linked steps of peptidoglycan biosynthesis, it was necessary to prepare the

![Structure of lipid II and its main modifications in bacterial world. R = H or COOH for Lys and A₂pm, respectively; D-Asx = D-Asp or D-Asn.]()

Fig. 11. Structure of lipid II and its main modifications in bacterial world. R = H or COOH for Lys and A₂pm, respectively; D-Asx = D-Asp or D-Asn.
lipid-linked intermediates. Because they are present naturally in very low abundance, it was not feasible to isolate them from bacterial cells in useful quantities (van Heijenoort *et al.*, 1992; Guan *et al.*, 2005). Chemical syntheses were published for lipid I (VanNieuwenhze *et al.*, 2001) and for lipid II (Ye *et al.*, 2001; VanNieuwenhze *et al.*, 2002), however the synthetic routes were long and proceeded in fairly low overall yield. Auger *et al.*, (1997, 2003) first succeeded for the hemi-synthesis of functional analogues of both lipids I and II. Chemical synthesis was used to prepare soluble analogues of lipid I containing 10-carbon (Ha *et al.*, 1999) and 20-carbon chains (Cudic *et al.*, 2001), which were used as soluble substrates for MurG in vitro (Lazar & Walker, 2002).

Alternatively, lipids I and II can be prepared enzymatically. The cytoplasmic precursor UDP-MurNAc-pentapeptide (containing *meso*-A₂pm at position 3) can be accumulated in *B. subtilis* using a cell wall synthesis medium (Lugtenberg *et al.*, 1972), and isolated in 50 mg quantities. High level expression and purification of enzymes MurA-F from *E. coli* or *Pseudomonas aeruginosa* and their use for the preparation of the cytoplasmic intermediates *in vitro* have been reported (Reddy *et al.*, 1999; El Zoeiby *et al.*, 2001; Bouhss *et al.*, 2004). An efficient procedure for the conversion of UDP-MurNAc-pentapeptide into lipid I or lipid II was published by Breukink *et al.* (2003), that used membranes prepared from *M. flavus*, a species which has elevated levels of MraY and MurG enzyme activities. Using this procedure, they reported the preparation of up to 50 mg quantities of lipids I or II. The availability of purified C₅₅-PP synthase UppS, C₅₅-PP phosphatase BacA, MraY and MurG proteins also facilitated the synthesis of different forms of lipids I and II, with a labelling in either the C₅₅ or the pentapeptide moiety (El Ghachi *et al.*, 2006).

The ε-amino group of *meso*-A₂pm or L-Lys at position 3 of the pentapeptide chain can be chemically modified by fluorophores such as dansyl. The N-dansyl analogue was used to develop a fluorescence assay for MraY, for enzyme kinetic studies (Brandish *et al.*, 1996a), and for high throughput inhibition assays (Stachyra *et al.*, 2004). The N-dansylated lipid II analogue was prepared and was shown to give rise to fluorescence changes upon treatment with *E. coli* PBP1b (Schwartz *et al.*, 2002). An alternative labelling strategy was recently published, in which the D-alanine residue at position 4 or 5 can be replaced by D-cysteine, by enzymatic synthesis from D-Ala-D-Cys or D-Cys-D-Ala, and the thiol side chain of D-Cys can be labelled by thiol-selective reagents such as pyrene maleimide (Schouten *et al.*, 2006). The S-labelled UDP-MurNAc-pentapeptide analogues prepared in this way were converted into fluorescent lipid I and II analogues, using *M. flavus* membranes.

# Possible existence of a flippase

Lipid II, biosynthesized on the cytoplasmic face of the membrane, must somehow be flipped onto the extracellular face of the membrane (Fig. 1). The possible existence of a ‘flippase’ protein capable of catalysing this process has been discussed for many years (Weppner & Neuhaus, 1978; Ehlert & Höltje, 1996) but no gene has been identified to date. Analysis of fluorescently labelled peptidoglycan precursors in *S. aureus* membranes by fluorescence energy transfer have earlier concluded that the rate of unassisted flipping was much too low to account for the rate of peptidoglycan synthesis required at cell division (Weppner & Neuhaus, 1978), therefore it was generally assumed that there must be some protein assistance for the flipping of lipid II across the membrane. In fact, the biochemical process was only recently demonstrated experimentally, in particular by analysis of the transmembrane transport of fluorescent lipid II through model and bacterial membranes (Breukink *et al.*, 1999; van Dam *et al.*, 2007). It was shown that lipid II flop did not occur spontaneously and was not obligatory coupled to lipid II synthesis. This suggested the involvement of one or more integral membrane proteins or membrane-associated proteins in this process. Based on their results, the authors further hypothesized a coupling of the translocation of lipid II with the subsequent reactions of polymerization (transglycosylation) catalyzed by the penicillin-binding proteins (van Dam *et al.*, 2007).

The existence of ABC transporters that are able to catalyse similar types of membrane transport events suggests that the putative flippase might be similar to this protein family, for which there is now structural information (Chang & Roth, 2001). The bacterial *wzx* gene was suggested as a candidate flippase gene in lipopolysaccharide biosynthesis, on the basis of gene knockout studies (Liu *et al.*, 1996). A putative flippase gene involved in N-linked protein glycosylation in eukaryotic cells was identified, based upon gene knockout studies (Helenius *et al.*, 2002). The *pglK* gene encoding an ABC transporter responsible for N-linked protein glycosylation in *Campylobacter jejuni* was shown to complement a *wzx* deficiency in *E. coli* O-antigen biosynthesis, although the corresponding gene products showed no sequence similarity (Alaimo *et al.*, 2006). In contrast, van Dam *et al.* (2007) found that the flipping of lipid II was not dependent upon ATP, which would argue against the involvement of an ABC transporter. These different studies suggest that there is a flippase protein for peptidoglycan biosynthesis, but its identity and precise mechanism of action are yet to be determined.

# Concluding remarks

In summary, the understanding of the biochemistry of the lipid-linked intermediates of bacterial peptidoglycan

FEMS Microbiol Rev **32** (2008) 208–233

© 2007 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
biosynthesis has increased considerably in the last few years. Several genes encoding the phosphatase and kinase enzymes involved in undecaprenyl phosphate metabolism have now been identified. The corresponding enzymes are potential new targets for antibacterial action. Translocase MraY has been successfully purified, and is the target for a number of natural product and synthetic inhibitors. There is an X-ray crystal structure for glycosyltransferase MurG, which will facilitate the development of novel inhibitors for this enzyme. Both MraY and MurG are attractive targets for antibacterial development, and methods for high-throughput screening of both enzymes have been developed. The putative ‘flippase’ activity responsible for transport of lipid intermediate II across the cytoplasmic membrane remains an interesting target which has not yet been identified. A number of enzymatic modifications of lipid II by the FemABX/MurMN proteins have now been reconstituted; their role in bacterial cell growth and antibiotic resistance mechanisms can now be studied. Further modifications of lipid-linked intermediates take place in certain microorganisms, and the ability to prepare lipids I and II in useful quantities will allow more detailed studies on new modification reactions. Availability of lipid II will also allow more detailed biochemical studies on the transglycosylase and transpeptidase reactions catalysed by the penicillin-binding proteins in future years. How all these different membrane proteins are organized in the membrane and possibly interact together in relation to the cell elongation and cell division processes also remains to be elucidated.

### Acknowledgements

The authors thank Didier Blanot and Thierry Touzé for critical reading of the manuscript. This work was supported by the Centre National de la Recherche Scientifique and by the European Community (FP6 projects COBRA, LSHM-CT-2003-503335, and EUR-INTAFAR, LSHM-CT-2004-512138). AET was supported by a PhD studentship from the Engineering and Physical Sciences Research Council.

### References

Aaron M, Charbon G, Lam H, Schwarz H, Vollmer W & Jacobs-Wagner C (2007) The tubulin homologue FtsZ contributes to cell elongation by guiding cell wall precursor synthesis in *Caulobacter crescentus*. Mol Microbiol 64: 938–952.

Alaimo C, Catrein I, Morf L, Marolda CL, Callewaert N, Valvano MA, Feldman MF & Aebi M (2006) Two distinct but interchangeable mechanisms for flipping of lipid-linked oligosaccharides. EMBO J 25: 967–976.

Anderson JS, Matsuhashi M, Haskin MA & Strominger JL (1965) Lipid-phosphoacetylmuramyl-pentapeptide and lipid-phosphodisaccharide-pentapeptide: presumed membrane transport intermediates in cell wall synthesis. Proc Natl Acad Sci USA 53: 881–889.

Anderson JS, Meadow PM, Haskin MA & Strominger JL (1966) Biosynthesis of the peptidoglycan of bacterial cell walls. I. Utilization of uridine diphosphate acetylmuramyl pentapeptide and uridine diphosphate acetylglucosamine for peptidoglycan synthesis by particulate enzymes from *Staphylococcus aureus* and *Micrococcus lysodeikticus*. Arch Biochem Biophys 116: 487–515.

Anderson RG, Hussey H & Baddiley J (1972) The mechanism of wall synthesis in bacteria. The organization of enzymes and isoprenoid phosphates in the membrane. Biochem J 127: 11–25.

Apfel CM, Takacs B, Fountoulakis M, Stieger M & Keck W (1999) Use of genomics to identify bacterial undecaprenyl pyrophosphate synthetase: cloning, expression, and characterization of the essential *uppS* gene. J Bacteriol 181: 483–492.

Auger G, Crouvoisier M, Caroff M, van Heijenoort J & Blanot D (1997) Synthesis of an analogue of the lipoglycopeptide membrane intermediate I of peptidoglycan biosynthesis. Lett Pept Sci 4: 371–376.

Auger G, van Heijenoort J, Mengin-Lecreulx D & Blanot D (2003) A MurG assay which utilises a synthetic analogue of lipid I. FEMS Microbiol Lett 219: 115–119.

Baba T & Allen CM Jr (1978) Substrate specificity of undecaprenyl pyrophosphate synthetase from *Lactobacillus plantarum*. Biochemistry 17: 5598–5604.

Baba T & Allen CM (1980) Prenyl transferases from *Micrococcus luteus*: characterization of undecaprenyl pyrophosphate synthetase. Arch Biochem Biophys 200: 474–484.

Baba T, Muth J & Allen CM (1985) Photoaffinity labeling of undecaprenyl pyrophosphate synthetase with a farnesyl pyrophosphate analogue. J Biol Chem 260: 10467–10473.

Barreteau H, Kovac A, Boniface A, Sova M, Gobec S & Blanot D (2008) Cytoplasmic steps of peptidoglycan biosynthesis. FEMS Microbiol Rev, in press.

Bellais S, Arthur M, Dubost L, Hugonnet JE, Gutmann L, van Heijenoort J, Legrand R, Brouard JP, Rice L & Mainardi JL (2006) Asl<sub>fm</sub>, the D-aspartate ligase responsible for the addition of D-aspartic acid onto the peptidoglycan precursor of *Enterococcus faecium*. J Biol Chem 281: 11586–11594.

Benson TE, Prince DB, Mutchler VT, Curry KA, Ho AM, Sarver RW, Hagadorn JC, Choi GH & Garlick RL (2002) X-ray crystal structure of *Staphylococcus aureus* FemA. Structure 10: 1107–1115.

Bernard R, Joseph P, Guiseppi A, Chippaux M & Denizot F (2003) YtsCD and YwoA, two independent systems that confer bacitracin resistance to *Bacillus subtilis*. FEMS Microbiol Lett 228: 93–97.

Bernard R, El Ghachi M, Mengin-Lecreulx D, Chippaux M & Denizot F (2005) BcrC from *Bacillus subtilis* acts as an undecaprenyl pyrophosphate phosphatase in bacitracin resistance. J Biol Chem 280: 28852–28857.

Bernhardt TG, Roof WD & Young R (2000) Genetic evidence that the bacteriophage φX174 lysis protein inhibits cell wall synthesis. *Proc Natl Acad Sci USA* 97: 4297–4302.

Biarrotte-Sorin S, Maillard AP, Delette J, Sougakoff W, Arthur M & Mayer C (2004) Crystal structures of Weissella viridescens FemX and its complex with UDP-MurNAc-pentapeptide: insights into FemABX family substrates recognition. *Structure* 12: 257–267.

Billot-Klein D, Shlaes D, Bryant D, Bell D, Legrand R, Gutmann L & van Heijenoort J (1997) Presence of UDP-N-acetylmuramyl-hexapeptides and -heptapeptides in enterococci and staphylococci after treatment with ramoplanin, tunicamycin, or vancomycin. *J Bacteriol* 179: 4684–4688.

Bohnenberger E & Sandermann H Jr (1976) Dephosphorylation of C<sub>55</sub>-isoprenyl-monophosphate by non-specific phosphatases. *FEBS Lett* 67: 85–89.

Bohnenberger E & Sandermann H Jr (1979) Diglyceride kinase from Escherichia coli. Purification in organic solvent and some properties of the enzyme. *Eur J Biochem* 94: 401–407.

Bonev BB, Breukink E, Swiezewska E, De Kruijff B & Watts A (2004) Targeting extracellular pyrophosphates underpins the high selectivity of nisin. *FASEB J* 18: 1862–1869.

Boniface A, Bouhss A, Mengin-Lecreulx D & Blanot D (2006) The MurE synthetase from Thermotoga maritima is endowed with an unusual D-lysine adding activity. *J Biol Chem* 281: 15680–15686.

Boojamra CG, Lemoine RC, Lee JC *et al.* (2001) Stereochemical elucidation and total synthesis of dihydropacifidamycin D, a semisynthetic pacidamycin. *J Am Chem Soc* 123: 870–874.

Bouhss A, Mengin-Lecreulx D, Le Beller D & van Heijenoort J (1999) Topological analysis of the MraY protein catalysing the first membrane step of peptidoglycan synthesis. *Mol Microbiol* 34: 576–585.

Bouhss A, Josseaume N, Allanic D, Crouvoisier M, Gutmann L, Mainardi JL, Mengin-Lecreulx D, van Heijenoort J & Arthur M (2001) Identification of the UDP-MurNAc-pentapeptide: L-alanine ligase for synthesis of branched peptidoglycan precursors in Enterococcus faecalis. *J Bacteriol* 183: 5122–5127.

Bouhss A, Josseaume N, Severin A, Tabei K, Hugonnet JE, Shlaes D, Mengin-Lecreulx D, van Heijenoort J & Arthur M (2002) Synthesis of the L-alanyl-L-alanine cross-bridge of Enterococcus faecalis peptidoglycan. *J Biol Chem* 277: 45935–45941.

Bouhss A, Crouvoisier M, Blanot D & Mengin-Lecreulx D (2004) Purification and characterization of the bacterial MraY translocase catalyzing the first membrane step of peptidoglycan biosynthesis. *J Biol Chem* 279: 29974–29980.

Boyle DS & Donachie WD (1998) *mraY* is an essential gene for cell growth in Escherichia coli. *J Bacteriol* 180: 6429–6432.

Brandish PE, Burnham MK, Lonsdale JT, Southgate R, Inukai M & Bugg TD (1996a) Slow binding inhibition of phospho-N-acetylmuramyl-pentapeptide-translocase (Escherichia coli) by mureidomycin A. *J Biol Chem* 271: 7609–7614.

Brandish PE, Kimura KI, Inukai M, Southgate R, Lonsdale JT & Bugg TD (1996b) Modes of action of tunicamycin,

liposidomycin B, and mureidomycin A: inhibition of phospho-N-acetylmuramyl-pentapeptide translocase from Escherichia coli. *Antimicrob Agents Chemother* 40: 1640–1644.

Braun V & Sieglin U (1970) The covalent murein-lipoprotein structure of the Escherichia coli cell wall. The attachment site of the lipoprotein on the murein. *Eur J Biochem* 13: 336–346.

Breukink E & de Kruijff B (2006) Lipid II as a target for antibiotics. *Nat Rev Drug Discov* 5: 321–332.

Breukink E, Wiedemann I, van Kraaij C, Kuipers OP, Sahl H & de Kruijff B (1999) Use of the cell wall precursor lipid II by a pore-forming peptide antibiotic. *Science* 286: 2361–2364.

Breukink E, van Heusden HE, Vollmerhaus PJ, Swiezewska E, Brunner L, Walker S, Heck AJ & de Kruijff B (2003) Lipid II is an intrinsic component of the pore induced by nisin in bacterial membranes. *J Biol Chem* 278: 19898–19903.

Bugg TD, Lloyd AJ & Roper DI (2006) Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins. *Infect Disord Drug Targets* 6: 85–106.

Bupp K & van Heijenoort J (1993) The final step of peptidoglycan subunit assembly in Escherichia coli occurs in the cytoplasm. *J Bacteriol* 175: 1841–1843.

Burda P & Aebi M (1999) The dolichol pathway of N-linked glycosylation. *Biochim Biophys Acta* 1426: 239–257.

Cain BD, Norton PJ, Eubanks W, Nick HS & Allen CM (1993) Amplification of the *bacA* gene confers bacitracin resistance to Escherichia coli. *J Bacteriol* 175: 3784–3789.

Cao M & Helmann JD (2002) Regulation of the Bacillus subtilis *bcrC* bacitracin resistance gene by two extracytoplasmic function sigma factors. *J Bacteriol* 184: 6123–6129.

Chalker AF, Ingram KA, Lunsford RD, Bryant AP, Bryant J, Wallis NG, Broskey JP, Pearson SC & Holmes DJ (2000) The *bacA* gene, which determines bacitracin susceptibility in Streptococcus pneumoniae and Staphylococcus aureus, is also required for virulence. *Microbiology* 146: 1547–1553.

Chang G & Roth CB (2001) Structure of MsbA from *E. coli*: a homolog of the multidrug resistance ATP binding cassette (ABC) transporters. *Science* 293: 1793–1800.

Chang SY, Chen YK, Wang AH & Liang PH (2003) Identification of the active conformation and the importance of length of the flexible loop 72–83 in regulating the conformational change of undecaprenyl pyrophosphate synthase. *Biochemistry* 42: 14452–14459.

Chang SY, Ko TP, Chen AP, Wang AH & Liang PH (2004) Substrate binding mode and reaction mechanism of undecaprenyl pyrophosphate synthase deduced from crystallographic studies. *Protein Sci* 13: 971–978.

Chatterjee AN & Park JT (1964) Biosynthesis of cell wall mucopeptide by a particulate fraction from Staphylococcus aureus. *Proc Natl Acad Sci USA* 51: 9–16.

Chen A, Kroon PA & Poulter CD (1994) Isoprenyl diphosphate synthases: protein sequence comparisons, a phylogenetic tree, and predictions of secondary structure. *Protein Sci* 3: 600–607.

Chen L, Men H, Ha S, Ye XY, Brunner L, Hu Y & Walker S (2002a) Intrinsic lipid preferences and kinetic mechanism of Escherichia coli MurG. *Biochemistry* 41: 6824–6833.

Chen YH, Chen AP, Chen CT, Wang AH & Liang PH (2002b) Probing the conformational change of *Escherichia coli* undecaprenyl pyrophosphate synthase during catalysis using an inhibitor and tryptophan mutants. *J Biol Chem* 277: 7369–7376.

Christenson JG, Gross SK & Robbins PW (1969) Enzymatic synthesis of the antigen carrier lipid. *J Biol Chem* 244: 5436–5439.

Crouvoisier M, Mengin-Lecreulx D & van Heijenoort J (1999) UDP-N-acetylglucosamine: N-acetylmuramoyl-(pentapeptide) pyrophosphoryl undecaprenol-N-acetylglucosamine transferase from *Escherichia coli*: overproduction, solubilization, and purification. *FEBS Lett* 449: 289–292.

Crouvoisier M, Auger G, Blanot D & Mengin-Lecreulx D (2007) Role of the amino acid invariants in the active site of MurG as evaluated by site-directed mutagenesis. *Biochimie* 89: 1498–1508.

Cudic P, Behenna DC, Yu MK, Kruger RG, Szewczuk LM & McCafferty DG (2001) Synthesis of P¹-citronellyl-P²-α-D-pyranosyl pyrophosphates as potential substrates for the *E. coli* undecaprenyl-pyrophosphoryl-N-acetylglucosaminyl transferase MurG. *Bioorg Med Chem Lett* 11: 3107–3110.

Dal Nogare AR, Dan N & Lehrman MA (1998) Conserved sequences in enzymes of the UDP-GlcNAc/MurNAc family are essential in hamster UDP-GlcNAc:dolichol-P GlcNAc-1-P transferase. *Glycobiology* 8: 625–632.

den Blaauwen T, Nguyen-Distèche M, de Pedro MA & Ayala JA (2008) Morphogenesis of rod shaped sacculi. *FEMS Microbiol Rev*, in press.

Dini C (2005) MraY inhibitors as novel antibacterial agents. *Curr Top Med Chem* 5: 1221–1236.

Dini C, Collette P, Drochon N, Guillot JC, Lemoine G, Mauvais P & Aszodi J (2000) Synthesis of the nucleoside moiety of liposidomycins: elucidation of the pharmacophore of this family of MraY inhibitors. *Bioorg Med Chem Lett* 10: 1839–1843.

Dini C, Didier-Laurent S, Drochon N, Feteanu S, Guillot JC, Monti F, Uridat E, Zhang J & Aszodi J (2002) Synthesis of submicromolar inhibitors of MraY by exploring the region originally occupied by the diazepanone ring in the liposidomycin structure. *Bioorg Med Chem Lett* 12: 1209–1213.

Dramsi S, Davison S, Magnet S & Arthur M (2008) Surface proteins covalently attached to peptidoglycan: examples from both Gram-positive and Gram-negative bacteria. *FEMS Microbiol Rev*, in press.

Ehlert K & Höltje JV (1996) Role of precursor translocation in coordination of murein and phospholipid synthesis in *Escherichia coli*. *J Bacteriol* 178: 6766–6771.

El Ghachi M, Bouhss A, Blanot D & Mengin-Lecreulx D (2004) The *bacA* gene of *Escherichia coli* encodes an undecaprenyl pyrophosphate phosphatase activity. *J Biol Chem* 279: 30106–30113.

El Ghachi M, Derbise A, Bouhss A & Mengin-Lecreulx D (2005) Identification of multiple genes encoding membrane proteins with undecaprenyl pyrophosphate phosphatase (UppP) activity in *Escherichia coli*. *J Biol Chem* 280: 18689–18695.

El Ghachi M, Bouhss A, Barreteau H, Touzé T, Auger G, Blanot D & Mengin-Lecreulx D (2006) Colicin M exerts its bacteriolytic effect via enzymatic degradation of undecaprenyl phosphate-linked peptidoglycan precursors. *J Biol Chem* 281: 22761–22772.

El Zoeiby A, Sanschagrin F, Havugimana PC, Garnier A & Levesque RC (2001) *In vitro* reconstruction of the biosynthetic pathway of peptidoglycan cytoplasmic precursor in *Pseudomonas aeruginosa*. *FEMS Microbiol Lett* 201: 229–235.

Fang X, Tiyanont K, Zhang Y, Wanner J, Boger D & Walker S (2006) The mechanism of action of ramoplanin and enduracidin. *Mol Biosyst* 2: 69–76.

Filipe SR & Tomasz A (2000) Inhibition of the expression of penicillin resistance in *Streptococcus pneumoniae* by inactivation of cell wall muropeptide branching genes. *Proc Natl Acad Sci USA* 97: 4891–4896.

Filipe SR, Pinho MG & Tomasz A (2000) Characterization of the *murMN* operon involved in the synthesis of branched peptidoglycan peptides in *Streptococcus pneumoniae*. *J Biol Chem* 275: 27768–27774.

Fujihashi M, Zhang YW, Higuchi Y, Li XY, Koyama T & Miki K (2001) Crystal structure of cis-prenyl chain elongating enzyme, undecaprenyl diphosphate synthase. *Proc Natl Acad Sci USA* 98: 4337–4342.

Fujikura K, Zhang YW, Yoshizaki H, Nishino T & Koyama T (2000) Significance of Asn-77 and Trp-78 in the catalytic function of undecaprenyl diphosphate synthase of *Micrococcus luteus* B-P 26. *J Biochem (Tokyo)* 128: 917–922.

Fujikura K, Zhang YW, Fujihashi M, Miki K & Koyama T (2003) Mutational analysis of allylic substrate binding site of *Micrococcus luteus* B-P 26 undecaprenyl diphosphate synthase. *Biochemistry* 42: 4035–4041.

Fujisaki S, Nishino T & Katsuki H (1986) Isoprenoid synthesis in *Escherichia coli*. Separation and partial purification of four enzymes involved in the synthesis. *J Biochem (Tokyo)* 99: 1327–1337.

Garcia-Bustos J & Tomasz A (1990) A biological price of antibiotic resistance: major changes in the peptidoglycan structure of penicillin-resistant pneumococci. *Proc Natl Acad Sci USA* 87: 5415–5419.

Geis A & Plapp R (1978) Phospho-N-acetylmuramoyl-pentapeptide-transferase of *Escherichia coli* K12. Properties of the membrane-bound and the extracted and partially purified enzyme. *Biochim Biophys Acta* 527: 414–424.

Glover KJ, Weerapana E & Imperiali B (2005) *In vitro* assembly of the undecaprenylpyrophosphate-linked heptasaccharide for prokaryotic N-linked glycosylation. *Proc Natl Acad Sci USA* 102: 14255–14259.

Goldman R & Strominger JL (1972) Purification and properties of C₅₅-isoprenylpyrophosphate phosphatase from *Micrococcus lysodeikticus*. *J Biol Chem* 247: 5116–5122.

Gough DP, Kirby AL, Richards JB & Hemming FW (1970) The characterization of undecaprenol of *Lactobacillus plantarum*. *Biochem J* 118: 167–170.

Guan Z, Breazeale SD & Raetz CR (2005) Extraction and identification by mass spectrometry of undecaprenyl diphosphate-MurNAc-pentapeptide-GlcNAc from *Escherichia coli*. *Anal Biochem* 345: 336–339.

Guo RT, Ko TP, Chen AP, Kuo CJ, Wang AH & Liang PH (2005) Crystal structures of undecaprenyl pyrophosphate synthase in complex with magnesium, isopentenyl pyrophosphate, and farnesyl thiopyrophosphate: roles of the metal ion and conserved residues in catalysis. *J Biol Chem* 280: 20762–20774.

Ha S, Chang E, Lo M-C, Men H, Park P, Ge M & Walker S (1999) The kinetic characterization of *Escherichia coli* MurG using synthetic substrate analogues. *J Am Chem Soc* 121: 8415–8426.

Ha S, Walker D, Shi Y & Walker S (2000) The 1.9 Å crystal structure of *Escherichia coli* MurG, a membrane-associated glycosyltransferase involved in peptidoglycan biosynthesis. *Protein Sci* 9: 1045–1052.

Hachem R & Raad I (2002) Failure of oral antimicrobial agents in eradicating gastrointestinal colonization with vancomycin-resistant enterococci. *Infect Control Hosp Epidemiol* 23: 43–44.

Hammes WP & Neuhaus FC (1974) On the specificity of phospho-N-acetylmuramyl-pentapeptide translocase. The peptide subunit of uridine diphosphate-N-acetylmuramyl-pentapeptide. *J Biol Chem* 249: 3140–3150.

Hara H, Yasuda S, Horiuchi K & Park JT (1997) A promoter for the first nine genes of the *Escherichia coli mra* cluster of cell division and cell envelope biosynthesis genes, including *ftsI* and *ftsW*. *J Bacteriol* 179: 5802–5811.

Harel YM, Bailone A & Bibi E (1999) Resistance to bacitracin as modulated by an *Escherichia coli* homologue of the bacitracin ABC transporter BcrC subunit from *Bacillus licheniformis*. *J Bacteriol* 181: 6176–6178.

Harkness RE & Braun V (1989a) Colicin M inhibits peptidoglycan biosynthesis by interfering with lipid carrier recycling. *J Biol Chem* 264: 6177–6182.

Harkness RE & Braun V (1989b) Inhibition of lipopolysaccharide O-antigen synthesis by colicin M. *J Biol Chem* 264: 14716–14722.

Heifetz A, Keenan RW & Elbein AD (1979) Mechanism of action of tunicamycin on the UDP-GlcNAc:dolichyl-phosphate GlcNAc-1-phosphate transferase. *Biochemistry* 18: 2186–2192.

Helenius J, Ng DT, Marolda CL, Walter P, Valvano MA & Aebi M (2002) Translocation of lipid-linked oligosaccharides across the ER membrane requires Rft1 protein. *Nature* 415: 447–450.

Helm JS, Chen L & Walker S (2002) Rethinking ramoplanin: the role of substrate binding in inhibition of peptidoglycan biosynthesis. *J Am Chem Soc* 124: 13970–13971.

Helm JS, Hu Y, Chen L, Gross B & Walker S (2003) Identification of active-site inhibitors of MurG using a generalizable, high-throughput glycosyltransferase screen. *J Am Chem Soc* 125: 11168–11169.

Heydanek MG Jr & Neuhaus FC (1969) The initial stage in peptidoglycan synthesis. IV. Solubilization of phospho-N-

acetylmuramyl-pentapeptide translocase. *Biochemistry* 8: 1474–1481.

Heydanek MG Jr, Struve WG & Neuhaus FC (1969) On the initial stage in peptidoglycan synthesis. III. Kinetics and uncoupling of phospho-N-acetylmuramyl-pentapeptide translocase (uridine 5′-phosphate). *Biochemistry* 8: 1214–1221.

Heydanek MG Jr, Linzer R, Pless DD & Neuhaus FC (1970) Initial stage in peptidoglycan synthesis. Mechanism of activation of phospho-N-acetylmuramyl-pentapeptide translocase by potassium ions. *Biochemistry* 9: 3618–3623.

Higashi Y, Strominger JL & Sweeley CC (1967) Structure of a lipid intermediate in cell wall peptidoglycan synthesis: a derivative of a C<sub>55</sub> isoprenoid alcohol. *Proc Natl Acad Sci USA* 57: 1878–1884.

Higashi Y, Siewert G & Strominger JL (1970a) Biosynthesis of the peptidoglycan of bacterial cell walls. XIX. Isoprenoid alcohol phosphokinase. *J Biol Chem* 245: 3683–3690.

Higashi Y, Strominger JL & Sweeley CC (1970b) Biosynthesis of the peptidoglycan of bacterial cell walls. XXI. Isolation of free C<sub>55</sub>-isoprenoid alcohol and of lipid intermediates in peptidoglycan synthesis from *Staphylococcus aureus*. *J Biol Chem* 245: 3697–3702.

Hotoda H, Furukawa M, Daigo M *et al.* (2003) Synthesis and antimycobacterial activity of capuramycin analogues Part 1: substitution of the azepan-2-one moiety of capuramycin. *Bioorg Med Chem Lett* 13: 2829–2832.

Howard NI & Bugg TD (2003) Synthesis and activity of 5′-uridinyl dipeptide analogues mimicking the amino terminal peptide chain of nucleoside antibiotic mureidomycin A. *Bioorg Med Chem* 11: 3083–3099.

Hu Y, Chen L, Ha S, Gross B, Falcone B, Walker D, Mokhtarzadeh M & Walker S (2003a) Crystal structure of the MurG:UDP-GlcNAc complex reveals common structural principles of a superfamily of glycosyltransferases. *Proc Natl Acad Sci USA* 100: 845–849.

Hu Y, Helm JS, Chen L, Ye XY & Walker S (2003b) Ramoplanin inhibits bacterial transglycosylases by binding as a dimer to lipid II. *J Am Chem Soc* 125: 8736–8737.

Hu Y, Helm JS, Chen L, Ginsberg C, Gross B, Kraybill B, Tiyanont K, Fang X, Wu T & Walker S (2004) Identification of selective inhibitors for the glycosyltransferase MurG via high-throughput screening. *Chem Biol* 11: 703–711.

Icho T & Raetz CR (1983) Multiple genes for membrane-bound phosphatases in *Escherichia coli* and their action on phospholipid precursors. *J Bacteriol* 153: 722–730.

Ikeda M, Wachi M, Jung HK, Ishino F & Matsuhashi M (1991) The *Escherichia coli mraY* gene encoding UDP-N-acetylmuramoyl-pentapeptide: undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide transferase. *J Bacteriol* 173: 1021–1026.

Ishii K, Sagami H & Ogura K (1986) A novel prenyltransferase from *Paracoccus denitrificans*. *Biochem J* 233: 773–777.

Johnson JG & Wilson DB (1977) Role of a sugar-lipid intermediate in colanic acid synthesis by *Escherichia coli*. *J Bacteriol* 129: 225–236.

Kalin JR & Allen CM Jr (1979) Characterization of undecaprenol kinase from *Lactobacillus plantarum*. *Biochim Biophys Acta* 574: 112–122.

Kamiryo T & Matsuhashi M (1972) The biosynthesis of the cross-linking peptides in the cell wall peptidoglycan of *Staphylococcus aureus*. *J Biol Chem* 247: 6306–6311.

Kato J, Fujisaki S, Nakajima K, Nishimura Y, Sato M & Nakano A (1999) The *Escherichia coli* homologue of yeast RER2, a key enzyme of dolichol synthesis, is essential for carrier lipid formation in bacterial cell wall synthesis. *J Bacteriol* 181: 2733–2738.

Kaur D, Brennan PJ & Crick DC (2004) Decaprenyl diphosphate synthesis in *Mycobacterium tuberculosis*. *J Bacteriol* 186: 7564–7570.

Kharel Y, Zhang YW, Fujihashi M, Miki K & Koyama T (2001) Identification of significant residues for homoallylic substrate binding of *Micrococcus luteus* B-P 26 undecaprenyl diphosphate synthase. *J Biol Chem* 276: 28459–28464.

Kharel Y, Takahashi S, Yamashita S & Koyama T (2006) Manipulation of prenyl chain length determination mechanism of *cis*-prenyltransferases. *FEBS J* 273: 647–657.

Kim SC, Singh AN & Raushel FM (1988) Analysis of the galactosyltransferase reaction by positional isotope exchange and secondary deuterium isotope effects. *Arch Biochem Biophys* 267: 54–59.

Kimura K & Bugg TD (2003) Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis. *Nat Prod Rep* 20: 252–273.

Ko TP, Chen YK, Robinson H, Tsai PC, Gao YG, Chen AP, Wang AH & Liang PH (2001) Mechanism of product chain length determination and the role of a flexible loop in *Escherichia coli* undecaprenyl-pyrophosphate synthase catalysis. *J Biol Chem* 276: 47474–47482.

Lara B, Mengin-Lecreulx D, Ayala JA & van Heijenoort J (2005) Peptidoglycan precursor pools associated with MraY and FtsW deficiencies or antibiotic treatments. *FEMS Microbiol Lett* 250: 195–200.

Lazar K & Walker S (2002) Substrate analogues to study cell-wall biosynthesis and its inhibition. *Curr Opin Chem Biol* 6: 786–793.

Lehrer J, Vigeant KA, Tatar LD & Valvano MA (2007) Functional characterization and membrane topology of *Escherichia coli* WecA, a sugar-phosphate transferase initiating the biosynthesis of enterobacterial common antigen and O-antigen lipopolysaccharide. *J Bacteriol* 189: 2618–2628.

Lehrman MA (1994) A family of UDP-GlcNAc/MurNAc: polyisoprenol-P GlcNAc/MurNAc-1-P transferases. *Glycobiology* 4: 768–771.

Li JJ & Bugg TD (2004) A fluorescent analogue of UDP-N-acetylglucosamine: application for FRET assay of peptidoglycan translocase II (MurG). *Chem Commun* (Cambridge) 182–183.

Liang PH, Ko TP & Wang AH (2002) Structure, mechanism and function of prenyltransferases. *Eur J Biochem* 269: 3339–3354.


Linnett PE & Strominger JL (1974) Amidation and cross-linking of the enzymatically synthesized peptidoglycan of *Bacillus stearothermophilus*. *J Biol Chem* 249: 2489–2496.

Lis M & Kuramitsu HK (2003) The stress-responsive *dgk* gene from *Streptococcus mutans* encodes a putative undecaprenol kinase activity. *Infect Immun* 71: 1938–1943.

Liu D, Cole RA & Reeves PR (1996) An O-antigen processing function for Wzx (RfbX): a promising candidate for O-unit flippase. *J Bacteriol* 178: 2102–2107.

Liu H, Ritter TK, Sadamoto R, Sears PS, Wu M & Wong CH (2003) Acceptor specificity and inhibition of the bacterial cell-wall glycosyltransferase MurG. *Chembiochem* 4: 603–609.

Lloyd AJ, Brandish PE, Gilbey AM & Bugg TDH (2004) Phospho-N-acetyl-muramyl-pentapeptide translocase from *Escherichia coli*: catalytic role of conserved aspartic acid residues. *J Bacteriol* 186: 1747–1757.

Lloyd AJ, Gilbey AM, Blewett AM *et al.* (2007) Characterization of tRNA-dependent peptide bond formation by MurM in the synthesis of *Streptococcus pneumoniae* peptidoglycan. *J Biol Chem*, in press.

Lo MC, Men H, Branstrom A, Helm JS, Yao N, Goldman R & Walker S (2000) A new mechanism of action proposed for ramoplanin. *J Am Chem Soc* 122: 3540–3541.

Lugtenberg EJ, De Haas-Menger L & Ruyters WH (1972) Murein synthesis and identification of cell wall precursors of temperature-sensitive lysis mutants of *Escherichia coli*. *J Bacteriol* 109: 326–335.

Machida M, Takechi K, Sato H *et al.* (2006) Genes for the peptidoglycan synthesis pathway are essential for chloroplast division in moss. *Proc Natl Acad Sci USA* 103: 6753–6758.

Mahapatra S, Yagi T, Belisle JT, Espinosa BJ, Hill PJ, McNeil MR, Brennan PJ & Crick DC (2005) Mycobacterial lipid II is composed of a complex mixture of modified muramyl and peptide moieties linked to decaprenyl phosphate. *J Bacteriol* 187: 2747–2757.

Maillard AP, Biarrotte-Sorin S, Villet R, Mesnage S, Bouhss A, Sougakoff W, Mayer C & Arthur M (2005) Structure-based site-directed mutagenesis of the UDP-MurNAc-pentapeptide-binding cavity of the FemX alanyl transferase from *Weissella viridescens*. *J Bacteriol* 187: 3833–3838.

Manson JM, Keis S, Smith JM & Cook GM (2004) Acquired bacitracin resistance in *Enterococcus faecalis* is mediated by an ABC transporter and a novel regulatory protein, BcrR. *Antimicrob Agents Chemother* 48: 3743–3748.

Marraffini LA, Dedent AC & Schneewind O (2006) Sortases and the art of anchoring proteins to the envelopes of gram-positive bacteria. *Microbiol Mol Biol Rev* 70: 192–221.

McDonald LA, Barbieri LR, Carter GT, Lenoy E, Lotvin J, Petersen PJ, Siegel MM, Singh G & Williamson RT (2002) Structures of the muraymycins, novel peptidoglycan biosynthesis inhibitors. *J Am Chem Soc* 124: 10260–10261.

Mendel S, Holbourn JM, Schouten JA & Bugg TD (2006) Interaction of the transmembrane domain of lysis protein E from bacteriophage φX174 with bacterial translocase MraY.

and peptidyl-prolyl isomerase SlyD. *Microbiology* 152: 2959–2967.

Mengin-Lecreulx D, Texier L & van Heijenoort J (1990) Nucleotide sequence of the cell-envelope *murG* gene of *Escherichia coli*. *Nucleic Acids Res* 18: 2810.

Mengin-Lecreulx D, Texier L, Rousseau M & van Heijenoort J (1991) The *murG* gene of *Escherichia coli* codes for the UDP-N-acetylglucosamine: N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase involved in the membrane steps of peptidoglycan synthesis. *J Bacteriol* 173: 4625–4636.

Mengin-Lecreulx D, Ayala J, Bouhss A, van Heijenoort J, Parquet C & Hara H (1998) Contribution of the *Pmra* promoter to expression of genes in the *Escherichia coli mra* cluster of cell envelope biosynthesis and cell division genes. *J Bacteriol* 180: 4406–4412.

Mengin-Lecreulx D, Falla T, Blanot D, van Heijenoort J, Adams DJ & Chopra I (1999) Expression of the *Staphylococcus aureus* UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:L-lysine ligase in *Escherichia coli* and effects on peptidoglycan biosynthesis and cell growth. *J Bacteriol* 181: 5909–5914.

Mohammadi T, Karczmarek A, Crouvoisier M, Bouhss A, Mengin-Lecreulx D & den Blaauwen T (2007) The essential peptidoglycan glycosyltransferase MurG forms a complex with proteins involved in lateral growth as well as with proteins involved in cell division in *Escherichia coli*. *Mol Microbiol* 65: 1106–1121.

Mondego JM, Simões-Araújo JL, de Oliveira DE & Alves-Ferreira M (2003) A gene similar to bacterial translocase I (*mraY*) identified by cDNA-AFLP is expressed during flower bud development of *Arabidopsis thaliana*. *Plant Science* 164: 323–331.

Mondy KE, Shannon W & Mundy LM (2001) Evaluation of zinc bacitracin capsules versus placebo for enteric eradication of vancomycin-resistant *Enterococcus faecium*. *Clin Infect Dis* 33: 473–476.

Muramatsu Y, Ishii MM & Inukai M (2003) Studies on novel bacterial translocase I inhibitors, A-500359s. II. Biological activities of A-500359 A, C, D and G. *J Antibiot (Tokyo)* 56: 253–258.

Neuhaus FC (1971) Initial translocation reaction in the biosynthesis of peptidoglycan by bacterial membranes. *Acc Chem Res* 4: 297–303.

Neuhaus FC & Baddiley J (2003) A continuum of anionic charge: structures and functions of D-alanyl-teichoic acids in gram-positive bacteria. *Microbiol Mol Biol Rev* 67: 686–723.

Neuwald AF (1997) An unexpected structural relationship between integral membrane phosphatases and soluble haloperoxidases. *Protein Sci* 6: 1764–1767.

O’Donovan CA, Fan-Havard P, Tecson-Tumang FT, Smith SM & Eng RH (1994) Enteric eradication of vancomycin-resistant *Enterococcus faecium* with oral bacitracin. *Diagn Microbiol Infect Dis* 18: 105–109.

Ogura K & Koyama T (1998) Enzymatic aspects of isoprenoid chain elongation. *Chem Rev* 98: 1263–1276.


Ornelas-Soares A, de Lencastre H, de Jonge BL & Tomasz A (1994) Reduced methicillin resistance in a new *Staphylococcus aureus* transposon mutant that incorporates muramyl dipeptides into the cell wall peptidoglycan. *J Biol Chem* 269: 27246–27250.

Pan JJ, Yang IW & Liang PH (2000) Effect of site-directed mutagenesis of the conserved aspartate and glutamate on *E. coli* undecaprenyl pyrophosphate synthase catalysis. *Biochemistry* 39: 13856–13861.

Park JT (1996) The murein sacculus. *Escherichia Coli and Salmonella: Cellular and Molecular Biology*, Vol. 1 (Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, Magasanik B, Reznikoff WS, Riley M, Schaechter M & Umbarger HE, eds), pp. 48–57. ASM Press, Washington, DC.

Perry AM, Ton-That H, Mazmanian SK & Schneewind O (2002) Anchoring of surface proteins to the cell wall of *Staphylococcus aureus*. III. Lipid II is an *in vivo* peptidoglycan substrate for sortase-catalyzed surface protein anchoring. *J Biol Chem* 277: 16241–16248.

Pisabarro AG, Prats R, Vazquez D & Rodriguez-Tébar A (1986) Activity of penicillin-binding protein 3 from *Escherichia coli*. *J Bacteriol* 168: 199–206.

Pless DD & Neuhaus FC (1973) Initial membrane reaction in peptidoglycan synthesis. Lipid dependence of phospho-N-acetylmuramyl-pentapeptide translocase (exchange reaction). *J Biol Chem* 248: 1568–1576.

Podlesek Z, Comino A, Herzog-Velikonja B, Zgur-Bertok D, Komel R & Grabnar M (1995) *Bacillus licheniformis* bacitracin-resistance ABC transporter: relationship to mammalian multidrug resistance. *Mol Microbiol* 16: 969–976.

Podlesek Z, Comino A, Herzog-Velikonja B & Grabnar M (2000) The role of the bacitracin ABC transporter in bacitracin resistance and collateral detergent sensitivity. *FEMS Microbiol Lett* 188: 103–106.

Poxton IR, Lomax JA & Sutherland IW (1974) Isoprenoid alcohol kinase: a third butanol-soluble enzyme in *Klebsiella aerogenes* membranes. *J Gen Microbiol* 84: 231–233.

Price NP & Momany FA (2005) Modeling bacterial UDP-HexNAc: polyprenol-P HexNAc-1-P transferases. *Glycobiology* 15: 29R–42R.

Raetz CR & Whitfield C (2002) Lipopolysaccharide endotoxins. *Annu Rev Biochem* 71: 635–700.

Ramey WD & Ishiguro EE (1978) Site of inhibition of peptidoglycan biosynthesis during the stringent response in *Escherichia coli*. *J Bacteriol* 135: 71–77.

Reddy SG, Waddell ST, Kuo DW, Wong KK & Pompliano DL (1999) Preparative enzymatic synthesis and characterization of the cytoplasmic intermediates of murein biosynthesis. *J Am Chem Soc* 121: 1175–1178.

Reeves PR, Hobbs M, Valvano MA *et al.* (1996) Bacterial polysaccharide synthesis and gene nomenclature. *Trends Microbiol* 4: 495–503.

Reynolds PE & Somner EA (1990) Comparison of the target sites and mechanisms of action of glycopeptide and

FEMS Microbiol Rev 32 (2008) 208–233

© 2007 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved
lipoglycodepsipeptide antibiotics. *Drugs Exp Clin Res* 16: 385–389.

Rick PD, Hubbard GL, Kitaoka M, Nagaki H, Kinoshita T, Dowd S, Simplaceanu V & Ho C (1998) Characterization of the lipid-carrier involved in the synthesis of enterobacterial common antigen (ECA) and identification of a novel phosphoglyceride in a mutant of *Salmonella typhimurium* defective in ECA synthesis. *Glycobiology* **8**: 557–567.

Rogers HJ, Perkins HR & Ward JB (1980) *Microbial Cell Walls and Membranes*. Chapman and Hall, London.

Rohr TE, Levy GN, Stark NJ & Anderson JS (1977) Initial reactions in biosynthesis of teichuronic acid of *Micrococcus lysodeikticus* cell walls. *J Biol Chem* **252**: 3460–3465.

Rohrer S & Berger-Bächi B (2003) FemABX peptidyl transferases: a link between branched-chain cell wall peptide formation and β-lactam resistance in gram-positive cocci. *Antimicrob Agents Chemother* **47**: 837–846.

Rose L, Kaufmann SH & Daugelat S (2004) Involvement of *Mycobacterium smegmatis* undecaprenyl phosphokinase in biofilm and smegma formation. *Microbes Infect* **6**: 965–971.

Salmond GP, Lutkenhaus JF & Donachie WD (1980) Identification of new genes in a cell envelope-cell division gene cluster of *Escherichia coli*: cell envelope gene *murG*. *J Bacteriol* **144**: 438–440.

Sandermann H Jr & Strominger JL (1972) Purification and properties of C<sub>55</sub>-isoprenoid alcohol phosphokinase from *Staphylococcus aureus*. *J Biol Chem* **247**: 5123–5131.

Scher M, Lennarz WJ & Sweeley CC (1968) The biosynthesis of mannosyl-1-phosphoryl-polyisoprenol in *Micrococcus lysodeikticus* and its role in mannan synthesis. *Proc Natl Acad Sci USA* **59**: 1313–1320.

Schleifer KH & Kandler O (1972) Peptidoglycan types of bacterial cell walls and their taxonomic implications. *Bacteriol Rev* **36**: 407–477.

Schneider T, Senn MM, Berger-Bächi B, Tossi A, Sahl HG & Wiedemann I (2004) *In vitro* assembly of a complete, pentaglycine interpeptide bridge containing cell wall precursor (lipid II-Gly5) of *Staphylococcus aureus*. *Mol Microbiol* **53**: 675–685.

Schouten JA, Bagga S, Lloyd AJ, de Pascale G, Dowson CG, Roper DI & Bugg TD (2006) Fluorescent reagents for *in vitro* studies of lipid-linked steps of bacterial peptidoglycan biosynthesis: derivatives of UDPMurNAc-pentapeptide containing D-cysteine at position 4 or 5. *Mol Biosyst* **2**: 484–491.

Schwartz B, Markwalder JA, Seitz SP, Wang Y & Stein RL (2002) A kinetic characterization of the glycosyltransferase activity of *Escherichia coli* PBP1b and development of a continuous fluorescence assay. *Biochemistry* **41**: 12552–12561.

Shimizu N, Koyama T & Ogura K (1998) Molecular cloning, expression, and purification of undecaprenyl diphosphate synthase. No sequence similarity between E- and Z-prenyl diphosphate synthases. *J Biol Chem* **273**: 19476–19481.

Siewert G & Strominger JL (1967) Bacitracin: an inhibitor of the dephosphorylation of lipid pyrophosphate, an Intermediate in

the biosynthesis of the peptidoglycan of bacterial cell walls. *Proc Natl Acad Sci USA* **57**: 767–773.

Siewert G & Strominger JL (1968) Biosynthesis of the peptidoglycan of bacterial cell walls. XI. Formation of the isoglutamine amide group in the cell walls of *Staphylococcus aureus*. *J Biol Chem* **243**: 783–790.

Singh AN, Hester LS & Raushel FM (1987) Examination of the mechanism of sucrose synthetase by positional isotope exchange. *J Biol Chem* **262**: 2554–2557.

Soldo B, Lazarevic V & Karamata D (2002) *tagO* is involved in the synthesis of all anionic cell-wall polymers in *Bacillus subtilis* 168. *Microbiology* **148**: 2079–2087.

Stachyra T, Dini C, Ferrari P, Bouhss A, van Heijenoort J, Mengin-Lecreulx D, Blanot D, Biton J & Le Beller D (2004) Fluorescence detection-based functional assay for high-throughput screening for MraY. *Antimicrob Agents Chemother* **48**: 897–902.

Stickgold RA & Neuhaus FC (1967) On the initial stage in peptidoglycan synthesis. Effect of 5-fluorouracil substitution on phospho-N-acetylmuramyl-pentapeptide translocase (uridine 5′-phosphate). *J Biol Chem* **242**: 1331–1337.

Stone KJ & Strominger JL (1971) Mechanism of action of bacitracin: complexation with metal ion and C<sub>55</sub>-isoprenyl pyrophosphate. *Proc Natl Acad Sci USA* **68**: 3223–3227.

Storm DR & Strominger JL (1973) Complex formation between bacitracin peptides and isoprenyl pyrophosphates. The specificity of lipid-peptide interactions. *J Biol Chem* **248**: 3940–3945.

Struve WG & Neuhaus FC (1965) Evidence for an initial acceptor of UDP-Nac-muramyl-pentapeptide in the synthesis of bacterial mucopeptide. *Biochem Biophys Res Commun* **18**: 6–12.

Struve WG, Sinha RK & Neuhaus FC (1966) On the initial stage in peptidoglycan synthesis. Phospho-N-acetylmuramyl-pentapeptide translocase (uridine monophosphate). *Biochemistry* **5**: 82–93.

Stuekey J & Carman GM (1997) Identification of a novel phosphatase sequence motif. *Protein Sci* **6**: 469–472.

Szymanski CM & Wren BW (2005) Protein glycosylation in bacterial mucosal pathogens. *Nat Rev Microbiol* **3**: 225–237.

Takahashi I & Ogura K (1982) Prenyltransferases of *Bacillus subtilis*: undecaprenyl pyrophosphate synthetase and geranylgeranyl pyrophosphate synthetase. *J Biochem (Tokyo)* **92**: 1527–1537.

Takahashi S & Koyama T (2006) Structure and function of cis-prenyl chain elongating enzymes. *Chem Rec* **6**: 194–205.

Takayama S, Chung SJ, Igarashi Y, Ichikawa Y, Sepp A, Lechler RI, Wu J, Hayashi T, Siuzdak G & Wong CH (1999) Selective inhibition of β-1,4- and α-1,3-galactosyltransferases: donor sugar-nucleotide based approach. *Bioorg Med Chem* **7**: 401–409.

Thanassi JA, Hartman-Neumann SL, Dougherty TJ, Dougherty BA & Pucci MJ (2002) Identification of 113 conserved essential genes using a high-throughput gene disruption system in *Streptococcus pneumoniae*. *Nucleic Acids Res* **30**: 3152–3162.
Thorne KJ & Kodicek E (1966) The structure of bactoprenol, a lipid formed by lactobacilli from mevalonic acid. *Biochem J* 99: 123–127.

Trent MS, Ribeiro AA, Lin S, Cotter RJ & Raetz CR (2001) An inner membrane enzyme in *Salmonella* and *Escherichia coli* that transfers 4-amino-4-deoxy-L-arabinose to lipid A: induction on polymyxin-resistant mutants and role of a novel lipid-linked donor. *J Biol Chem* 276: 43122–43131.

Troy FA, Frerman FE & Heath EC (1971) The biosynthesis of capsular polysaccharide in *Aerobacter aerogenes*. *J Biol Chem* 246: 118–133.

Troy FA, Vijay IK & Tesche N (1975) Role of undecaprenyl phosphate in synthesis of polymers containing sialic acid in *Escherichia coli*. *J Biol Chem* 250: 156–163.

Umbreit JN & Strominger JL (1972a) Complex lipid requirements for detergent-solubilized phosphoacetylmuramyl-pentapeptide translocase from *Micrococcus luteus*. *Proc Natl Acad Sci USA* 69: 1972–1974.

Umbreit JN & Strominger JL (1972b) Isolation of the lipid intermediate in peptidoglycan biosynthesis from *Escherichia coli*. *J Bacteriol* 112: 1306–1309.

Umbreit JN, Stone KJ & Strominger JL (1972) Isolation of polyisoprenyl alcohols from *Streptococcus faecalis*. *J Bacteriol* 112: 1302–1305.

van Dam V, Sijbrandi R, Kol M, Swiezewska E, de Kruijff B & Breukink E (2007) Transmembrane transport of peptidoglycan precursors across model and bacterial membranes. *Mol Microbiol* 64: 1105–1114.

van den Brink-van der Laan E, Boots JW, Spelbrink RE, Kool GM, Breukink E, Killian JA & de Kruijff B (2003) Membrane interaction of the glycosyltransferase MurG: a special role for cardiolipin. *J Bacteriol* 185: 3773–3779.

van Heijenoort J (2001a) Formation of the glycan chains in the synthesis of bacterial peptidoglycan. *Glycobiology* 11: 25R–36R.

van Heijenoort J (2001b) Recent advances in the formation of the bacterial peptidoglycan monomer unit. *Nat Prod Rep* 18: 503–519.

van Heijenoort Y, Gomez M, Derrien M, Ayala J & van Heijenoort J (1992) Membrane intermediates in the peptidoglycan metabolism of *Escherichia coli*: possible roles of PBP 1b and PBP 3. *J Bacteriol* 174: 3549–3557.

VanNieuwenhze MS, Mauldin SC, Zia-Ebrahimi M, Aikins JA & Blaszczak LC (2001) The total synthesis of lipid I. *J Am Chem Soc* 123: 6983–6988.

VanNieuwenhze MS, Mauldin SC, Zia-Ebrahimi M, Winger BE, Hornback WJ, Saha SL, Aikins JA & Blaszczak LC (2002) The first total synthesis of lipid II: the final monomeric intermediate

in bacterial cell wall biosynthesis. *J Am Chem Soc* 124: 3656–3660.

Vilim A, Woods MC & Carroll KK (1973) Polyprenols of *Listeria monocytogenes*. *Can J Biochem* 51: 939–941.

Vollmer W, Blanot D & de Pedro MA (2008) Peptidoglycan structure and architecture. *FEMS Microbiol Rev*, in press.

Ward JB & Perkins HR (1974) Peptidoglycan biosynthesis by preparations from *Bacillus licheniformis*: cross-linking of newly synthesized chains to preformed cell wall. *Biochem J* 139: 781–784.

Watkinson RJ, Hussey H & Baddiley J (1971) Shared lipid phosphate carrier in the biosynthesis of teichoic acid and peptidoglycan. *Nat New Biol* 229: 57–59.

Weppner WA & Neuhaus FC (1977) Fluorescent substrate for nascent peptidoglycan synthesis. Uridine diphosphate-N-acetylmuramyl-(N<sup>ε</sup>-5-dimethylaminonaphthalene-1-sulfonyl)pentapeptide. *J Biol Chem* 252: 2296–2303.

Weppner WA & Neuhaus FC (1978) Biosynthesis of peptidoglycan. Definition of the microenvironment of undecaprenyl diphosphate-N-acetylmuramyl-(5-dimethylaminonaphthalene-1-sulfonyl) pentapeptide by fluorescence spectroscopy. *J Biol Chem* 253: 472–478.

Weppner WA & Neuhaus FC (1979) Initial membrane reaction in peptidoglycan synthesis. Interaction of lipid with phospho-N-acetylmuramyl-pentapeptide translocase. *Biochim Biophys Acta* 552: 418–427.

Willoughby E, Highasi Y & Strominger JL (1972) Enzymatic dephosphorylation of C<sub>55</sub>-isoprenylphosphate. *J Biol Chem* 247: 5113–5115.

Wolucka BA, McNeil MR, de Hoffmann E, Chojnacki T & Brennan PJ (1994) Recognition of the lipid intermediate for arabinogalactan/arabinomannan biosynthesis and its relation to the mode of action of ethambutol on mycobacteria. *J Biol Chem* 269: 23328–23335.

Wright A, Dankert M, Fennessey P & Robbins PW (1967) Characterization of a polyisoprenoid compound functional in O-antigen biosynthesis. *Proc Natl Acad Sci USA* 57: 1798–1803.

Yamashita A, Norton E, Petersen PJ, Rasmussen BA, Singh G, Yang Y, Mansour TS & Ho DM (2003) Muraymycins, novel peptidoglycan biosynthesis inhibitors: synthesis and SAR of their analogues. *Bioorg Med Chem Lett* 13: 3345–3350.

Ye XY, Lo MC, Brunner L, Walker D, Kahne D & Walker S (2001) Better substrates for bacterial transglycosylases. *J Am Chem Soc* 123: 3155–3156.

Zawadzke LE, Wu P, Cook L, Fan L, Casperson M, Kishnani M, Calambur D, Hofstead SJ & Padmanabha R (2003) Targeting the MraY and MurG bacterial enzymes for antimicrobial therapeutic intervention. *Anal Biochem* 314: 243–252.
