
# Elucidation of the Final Reactions of DIMBOA-Glucoside Biosynthesis in Maize: Characterization of Bx6 and Bx7${}^{1}$[W][OA]

Rafal Jonczyk, Holger Schmidt, Anne Osterrieder${}^{2}$, Andreas Fiesselmann, Katrin Schullehner, Martin Haslbeck, Dieter Sicker, Diana Hofmann${}^{3}$, Nasser Yalpani, Carl Simmons, Monika Frey*, and Alfons Gierl

Technische Universität München, Lehrstuhl für Genetik, 85354 Freising, Germany (R.J., H.S., A.O., A.F., K.S., M.F., A.G.); Technische Universität München, Department Chemie, Lehrstuhl Biotechnologie, 85747 Garching, Germany (M.H.); Institut for Organic Chemistry (D.S.) and Institut für Analytische Chemie (D.H.), Universität Leipzig, 04103 Leipzig, Germany (D.S.); and Pioneer Hi-Bred International, Inc., Johnston, Iowa 50131 (N.Y., C.S.)

Benzoxazinoids were identified in the early 1960s as secondary metabolites of the grasses that function as natural pesticides and exhibit allelopathic properties. Benzoxazinoids are synthesized in seedlings and stored as glucosides (glcs); the main aglucone moieties are 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one (DIBOA) and 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one (DIMBOA). The genes of DIBOA-glc biosynthesis have previously been isolated and the enzymatic functions characterized. Here, the enzymes for conversion of DIBOA-glc to DIMBOA-glc are identified. DIBOA-glc is the substrate of the dioxygenase BENZOXAZINLESS6 (BX6) and the produced 2,4,7-trihydroxy-2H-1,4-benzoxazin-3-(4H)-one-glc is metabolized by the methyltransferase BX7 to yield DIMBOA-glc. Both enzymes exhibit moderate $K_{\mathrm{m}}$ values (below 0.4 mM) and $k_{\mathrm{cat}}$ values of 2.10 s${}^{-1}$ and 0.25 s${}^{-1}$, respectively. Although BX6 uses a glucosylated substrate, our localization studies indicate a cytoplasmic localization of the dioxygenase. $Bx6$ and $Bx7$ are highest expressed in seedling tissue, a feature shared with the other $Bx$ genes. At present, $Bx6$ and $Bx7$ have no close relatives among the members of their respective gene families. $Bx6$ and $Bx7$ map to the cluster of $Bx$ genes on the short arm of chromosome 4.

Plants produce a remarkably diverse array of over 200,000 low-mass natural products, known as secondary metabolites. This rich diversity results in part from an evolutionary process driven by selection for acquisition of improved chemical defense against microbial attack and herbivore predation. Related plant taxa generally make use of related chemical structures for defense. Benzoxazinoids are abundant in grasses, including the major agricultural crops maize (*Zea mays*), wheat (*Triticum aestivum*), and rye (*Secale cereale*). Outside the Gramineae these secondary metabolites are found dispersed in isolated dicotyledonous species.

(Niemeyer, 1988; Sicker et al., 2000). In rye shoots and wild barley (*Hordeum* spp.) species 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one (DIBOA) is the predominant aglucone; in maize and wheat its methoxy derivative 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one (DIMBOA) prevails.

The benzoxazinone DIBOA and its glucoside (glc) were originally reported more than 40 years ago (Hietala and Virtanen, 1960; Virtanen and Hietala, 1960). The discovery that natural benzoxazinoids function as pesticides, insecticides, and show allelopathic effects (Barnes and Putnam, 1987; Grombacher et al., 1989; Bravo and Lazo, 1993; Sicker et al., 2000) made benzoxazinone biosynthesis a subject not only for biochemical analysis (Tipton et al., 1973; Kumar et al., 1994), but included these metabolites in breeding strategies for maize elite lines in the late 1960s (Klun et al., 1970). In 1964, a maize mutant, named *benzoxazinless1* (*bx1*) was described (Hamilton, 1964). The $Bx1$ gene was isolated and the biosynthetic pathway to DIBOA elucidated in 1997 (Frey et al., 1997; Fig. 1). BX1, a homolog of the Trp synthase $\alpha$-subunit, catalyzes the formation of indole, the first specific step in the pathway (Frey et al., 1997; Melanson et al., 1997). The introduction of four oxygen atoms into the indole moiety that yields DIBOA is catalyzed by four cytochrome P450 monooxygenases termed BX2 to BX5. Orthologous genes for $ZmBx1$ to $ZmBx5$ from wheat and wild barley DIBOA biosynthesis

---

1. This work was supported by Bayerisches Staatsministerium für Umwelt, Gesundheit und Verbraucherschutz (to R.J. and H.S.), and Fonds der Chemischen Industrie (M.H.).
2. Present address: School of Sciences, Oxford Brookes University, Oxford OX3 0BP, UK.
3. Present address: Forschungszentrum Jülich, ZCH/BioSpec, Leo-Brandt-Str., 52428 Jülich, Germany.
* Corresponding author; e-mail monika.frey@wzw.tum.de.
The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (www.plantphysiol.org) is: Monika Frey (monika.frey@wzw.tum.de).
[W] The online version of this article contains Web-only data.
[OA] Open Access articles can be viewed online without a subscription.
www.plantphysiol.org/cgi/doi/10.1104/pp.107.111237

Plant Physiology, March 2008, Vol. 146, pp. 1053–1063, www.plantphysiol.org © 2008 American Society of Plant Biologists 1053 Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org Copyright © 2008 American Society of Plant Biologists. All rights reserved.

Jonczyk et al.

Indole-3-glycerol phosphate

$\downarrow$

BX1, BX2, BX3, BX4, BX5

$\downarrow$

![Chemical Structure](chemical_structure_1.png)

DIBOA

$\downarrow$

BX8/BX9

$\downarrow$

![Chemical Structure](chemical_structure_2.png)

DIBOA-glucoside

$\downarrow$

BX6

$\downarrow$

![Chemical Structure](chemical_structure_3.png)

TRIBOA-glucoside

$\downarrow$

BX7

$\downarrow$

![Chemical Structure](chemical_structure_4.png)

DIMBOA-glucoside

The conversion of DIBOA to DIMBOA requires hydroxylation and methylation at C-7 of DIBOA. A 2-oxoglutarate-dependent dioxygenase (2ODD) was identified by transposon tagging in maize (Frey et al., 2003). Hydroxylation at C-7 was defective in the transposon-induced mutants, and the gene was termed $Bx6$. Here, we show in vitro that BX6 accepts only DIBOA-glc as substrate producing the hydroxy derivative 2,4,7-trihydroxy-2H-1,4-benzoxazin-3-(4H)-one-glc (TRIBOA-glc). In addition, an O-methyltransferase (OMT) named BX7 was isolated and characterized. BX7 catalyzes the formation of DIMBOA-glc from TRIBOA-glc. Thus, DIMBOA biosynthesis in maize is now completely described on the gene and enzyme level.

## RESULTS

### The Substrate of BX6 Is DIBOA-glc

Recently, it has been shown by reverse genetic analysis that the 2ODD BX6 is responsible for the hydroxylation in position C-7 of the benzoxazinoids in maize (Fig. 1; Frey et al., 2003). We have now determined the biochemical function of BX6 by expression in *Escherichia coli* and analysis of the recombinant protein. Incubation of DIBOA-glc with BX6 protein under standard conditions for 2ODDs (see “Materials and Methods”) led to the formation of a new product. This product has the same retention time in HPLC analysis and the same absorption spectrum as enzymatically produced TRIBOA-glc from chemically synthesized TRIBOA glucosylated by the glucosyltransferase BX8. The product identity was further verified by mass spectrometry (MS; see “Materials and Methods”). The TRIBOA-glc appears predominantly as $(M-H + HOAc)$ adduction (mass-to-charge ratio = 418 D) beside $(M-H)^-$ ion (mass-to-charge ratio = 358 D). The adduct ions are formed exclusively with acetic acid, not with isopropanol (same nominal mass), which was demonstrated independently in a second experiment.

DIBOA was not hydroxylated by BX6 (Fig. 2). The result is consistent with the accumulation of DIBOA-glc in $Bx6$ mutant plants (Fig. 3). A pH optimum of 6 was determined for the DIBOA-glc hydroxylation reaction. The steady-state kinetic constants for the substrates DIBOA-glc and 2-oxoglutarate (Table I) were determined. The reaction is characterized by a high $k_{\text{cat}}$ for both substrates and a significant difference in $K_m$ values for the two substrates $K_m^{\text{DIBOA-glc}}$ (373 $\mu$M) is about 6 times higher than $K_m^{\text{2-oxoglutarate}}$ (70 $\mu$M).

### BX6 Is Localized in the Cytoplasm

In plants, glucosylation plays a key role in the detoxification of reactive secondary metabolites, with their glcs often accumulating in the vacuoles. Multi-drug resistance-associated protein (MRP)-type ATP-binding cassette transporters are reported to be involved in the vacuolar sequestration (Rea, 2007). In plants, where DIBOA-glc is the final product of

Figure 1. Enzymes and intermediates of DIMBOA-glc biosynthesis in maize. Conversion of indole-3-glycerol phosphate to indole by the enzyme BX1 represents the branch point from primary metabolism. The introduction of four oxygen atoms into the indole moiety that yields DIBOA is catalyzed by four cytochrome P450-dependent monooxygenases, BX2 to BX5. The resulting DIBOA is glucosylated by the UDP-glucosyltransferases BX8 and BX9. Hydroxylation in position C-7 of the glc is catalyzed by the 2ODD BX6. The final step of the pathway is the O-methylation of TRIBOA-glc by BX7 to produce DIMBOA-glc.

sis were reported subsequently (Nomura et al., 2002, 2003, 2005; Grün et al., 2005).

Benzoxazinoids are found in the plant mainly as glcs that are stored in the vacuole (Sicker et al., 2000). A specific glucosidase, located in the chloroplast (Esen, 1992), catalyzes the formation of the toxic aglucone when the cell is damaged and disintegrates. The isolation of pathway-specific glucosyltransferases (von Rad et al., 2001) completed the molecular characterization of the pathway to DIBOA-glc.

DIMBOA Biosynthesis in Maize

Figure 2. Analysis of BX6 enzyme activity using heterologously expressed protein and the substrates DIBOA and DIBOA-glc. Overlay of the HPLC traces. The inserts A, B, and C depict the spectra of the respective main peaks. Thick gray line: enzyme assay using DIBOA-glc as a substrate; the substrate is almost completely converted to TRIBOA-glc (spectrum A). Thick black line: enzyme assay with DIBOA as a substrate (spectrum C); no conversion is detected. Thin black line: incubation of DIBOA-glc under assay conditions without enzyme (spectrum B). Assay conditions were described in "Materials and Methods"; incubation time was 30 min. mAU, Milli absorption units.

benzoxazinoid biosynthesis, this secondary metabolite is stored in the vacuole. Therefore we tested the possibility that the conversion of DIBOA-glc to TRIBOA-glc would occur in this cellular compartment. From in silico analysis, no clear indication of the subcellular location of BX6 by prediction algorithms (PREDOTAR, iSPORT, and TargetP) was obtained. However, signal peptides and the requirements for protein targeting to the vacuole are poorly defined (Carter et al., 2004). These targeting signals can be located either at the amino or the carboxy terminus of the protein. We therefore sandwiched the GFP gene between the amino-terminal half of Bx6 and the complete Bx6 sequence (see “Materials and Methods”). Bx6 is highly expressed in young plants and transcript levels exceed the levels of the housekeeping gene GAP C (Fig. 6). Since the native promoter of Bx6 is not yet defined, the 35S promoter, conferring moderate to high transcript levels in monocot plants (Schledzewski and Mendel, 1994), was employed to drive the gene. The chimeric gene Bx6part-GFP-Bx6 was expressed transiently in maize protoplasts and transformed into maize plants. In transient

Figure 3. Benzoxazinoids in Bx6 mutant plants. Benzoxazinoids were isolated from mutant and wild-type plants (thick black line, mutant with insertion at position 1; thick gray line, mutant with insertion at position 4; thin black line, wild type; Frey et al., 2003) and analyzed as described in “Materials and Methods.” DIBOA-glc is the predominant benzoxazinoid in the mutant. The spectra of the respective main peaks are displayed in the inserts. A and B, DIBOA-glc. C, DIMBOA-glc. mAU, Milli absorption units.

Plant Physiol. Vol. 146, 2008
Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org
Copyright © 2008 American Society of Plant Biologists. All rights reserved.

Jonczyk et al.

Table I. Catalytic properties of BX6 and BX7

n.d., Not detectable.

| Enzyme | pH Optima | Substrate       | $K_{\mathrm{m}}$   | $V_{\max }$     | $k_{\mathrm{cat}}$ | $k_{\mathrm{cat}} / K_{\mathrm{m}}$ |
|--------|-----------|-----------------|--------------------|------------------|--------------------|-------------------------------------|
|        |           |                 | $mM$              | $\mu \mathrm{kat} g^{-1}$ | $s^{-1}$          | $mM^{-1} s^{-1}$                   |
| BX6    | 6         | DIBOA-glc       | $0.373 \pm 0.032$ | $59.0 \pm 1.5$   | $2.1 \pm 0.1$      | $5.6 \pm 1.6$                      |
|        |           | DIBOA           | n.d.              | n.d.             | n.d.               | n.d.                               |
| BX7    | 7         | TRIBOA-glc$^{a}$| $0.272 \pm 0.012$ | $6.79 \pm 0.11$  | $0.249 \pm 0.004$  | $0.91 \pm 0.34$                    |
|        |           | TRIBOA$^{b}$    | n.d.              | n.d.             | n.d.               | n.d.                               |
|        |           | Caffeic acid$^{c}$| n.d.              | n.d.             | n.d.               | n.d.                               |
|        |           | Ferulic acid$^{d}$| n.d.              | n.d.             | n.d.               | n.d.                               |
|        |           | Apigenin$^{e}$  | n.d.              | n.d.             | n.d.               | n.d.                               |
|        |           | Quercetin$^{f}$ | n.d.              | n.d.             | n.d.               | n.d.                               |

$a-f$ Mg$^{2+}$ (1 mM), EDTA (1 mM), 5% ethanol, or 6% DMSO had no effect on BX7 activity. The substrate is assayed in a concentration of: 0.2 mM (a), 1 mM and 4% ethanol (b and c), 0.3 mM and 6% DMSO (d), 0.2 mM and 5% ethanol (e and f). A minor activity was detected (just above detection level) but kinetic constants could not be analyzed. Higher substrate concentration could not be tested due to solubility (b–f).

and stably transformed cells, GFP was consistently detected in the cytoplasm and there is no indication for a vacuolar localization of BX6 (Fig. 4). Therefore, hydroxylation of DIBOA-glc most likely takes place in the cytosol.

### Isolation and Characterization of Bx7

TRIBOA-glc was used as substrate to purify the putative OMT that catalyzes the last step in benzoxazinoid biosynthesis of maize. Benzoxazinoid-glcs are not stable in raw protein extracts from maize plantlets under standard OMT conditions. Therefore, affinity chromatography on adenosine agarose was adopted as a first purification step to get rid of unfavorable enzyme activities. Protein fractions eluting during application of a gradient of S-adenosyl-Met (SAM) were tested for the conversion of TRIBOA-glc to DIMBOA-glc. Active fractions were pooled and applied to anion-exchange chromatography. Analysis of individual fractions on SDS-PAGE gels revealed a band with a molecular mass of about 40 kD that correlated with TRIBOA-OMT activity in the enzyme assay (Supplemental Fig. S1). This band was excised and digested following the protocol of Schäfer et al. (2001). The sequence of several fragments of the trypsin digest was determined by mass spectroscopy (Supplemental Fig. S2). The deduced peptide sequences were compared with a set of predicted maize OMT sequences. The genes were selected from the Pioneer Hi-Bred EST collection by the criterion of high expression in seedling tissue. One gene demonstrated significant similarity to the deduced peptides (Supplemental Fig. S2A), especially in a unique sequence stretch of 14 amino acids at the amino terminus. The corresponding peptide is not present in other OMT sequences; the other peptides are located in more conserved regions of the protein and are less indicative. cDNA was isolated for the candidate gene and expressed in E. coli as a His-tagged protein. Subsequently, trypsin-digested purified protein was subjected to matrix-assisted laser-desorption ionization (MALDI)-MS

analysis. The generated peptide pattern was congruent with the pattern of the protein isolated from maize (Supplemental Fig. S2B). The recombinant enzyme was tested in vitro for the methylation of different substrates. The only substance that served as a substrate was TRIBOA-glc, neither the aglucone TRIBOA nor well-known substrates of plant OMTs (Table I) were efficiently converted by the enzyme. In case of quercetin, a minor activity was detected (just above detection level). The reaction was independent of Mg$^{2+}$. The steady-state kinetic constants of BX7 for the substrate TRIBOA-glc are in a reasonable range (Table I); $V_{\max }$ values of 0.45 $\mu \mathrm{kat} g^{-1}$ (Christensen et al., 1998) to about 80 $\mu \mathrm{kat} g^{-1}$ (Gauthier et al., 1998) have been reported for plant OMTs. The $K_{\mathrm{m}}^{\text {TRIBOA-glc }}$ of the isolated BX7 is in the higher range, but similar values were determined

![Figure 4](https://i.imgur.com/yourimage.png)

Figure 4. Subcellular localization of BX6. The fusion protein BX6part-GFP-BX6 is located in the cytoplasm of the cell. No GFP signal is detected in the vacuole. A and C, Control 2 × 35S promoter-GFP. B and D, BX6part-GFP-BX6 fusion. A and B, Transient expression. C and B, Transgenic plants. The bar equals 10 $\mu$m.
for Vanilla planifolia and Rosa spp. OMTs (Scalliet et al., 2002; Li et al., 2006). The pH optimum for the reaction is at 7.0; pH optima of 7 to 8 are often displayed by OMTs (Gang et al., 2002; Lavid et al., 2002).

### Phylogenetic Relationship of Bx7 in the OMT Gene Family

Plant OMTs have been categorized into two classes, based primarily on protein sequences (Joshi and Chiang, 1998). According to the occurrence and spacing of conserved amino acid sequence motifs, BX7 belongs to class II OMTs. The most comprehensively studied representatives of this class are caffeic acid OMTs (COMTs), which are involved in the biosynthesis of lignin, the preferred substrates being caffeoyl aldehyde and 5-hydroxyconiferaldehyde (Osakabe et al., 1999; Parvathi et al., 2001). Other class II OMTs catalyze the methylation of flavonoids and phenolics. In the phylogenetic network, Bx7 is joined to a web that connects OMTs of berberine alkaloid, isoflavone, and phenylpropene biosynthesis (Fig. 5). BX7 has no catalytic activity toward phenylpropanoids and flavonoids (Table I). Especially, apigenin, the substrate of a flavonoid-7-OMT from barley, is no substrate, although this gene is the closest relative in phylogenetic analysis (Fig. 5). BX7 shares the origin in the net with this OMT from barley and the maize OMT ZRP4. It has been proposed that ZRP4 is involved in suberin biosynthesis (Held et al., 1993); however, no experimental evidence is available to our knowledge. The common root may simply reflect the fact that three enzymes of grasses are compared and does not indicate related functions.

A comprehensive survey of TUSC, the Pioneer reverse genetic resource (Benson et al., 1995), and public collections of maize mutants did not result in the identification of a Bx7 mutant. Hence, the genetic proof that Bx7 is not only capable to perform the last step in DIMBOA-glc biosynthesis but is the only enzyme in maize that catalyzes this step is not yet given.

### Bx6 and Bx7 Are Predominantly Expressed in the Maize Seedling

The benzoxazinoid content is highest in the young maize plant and correlating with this distribution, major amounts of Bx1 to Bx5 transcripts are present in seedling tissue (von Rad et al., 2001; Frey et al., 2003). The same expression pattern is displayed for the 2ODD Bx6 and the methyltransferase Bx7 (Fig. 6). Peak levels of transcripts are detected in 3- and 4-d-old plantlets. Highest transcriptional expression levels of Bx6 and Bx7 are displayed in the scutellar node. In this tissue, the amount of mRNA encoding the housekeeping gene GAP C is exceeded by each of the Bx gene transcripts. Mature organs like leaf and husk of 10-week-old plants, cob, and tassel show low to nondetectable levels of the Bx gene transcripts. While primary and adventitious roots of plants 3 weeks after imbibition display low transcript levels, the newly formed crown roots

---

DIMBOA Biosynthesis in Maize

of 10-week-old plants reach almost the Bx gene transcript levels present in seedling roots.

### All DIMBOA-glc Biosynthetic Genes Are Located on the Short Arm of Chromosome 4

It was previously shown that the Bx genes are clustered at the short arm of chromosome 4 (Frey et al., 1997; von Rad et al., 2001; Frey et al., 2003). The set of recombinant inbred lines based on the cross CM37×T232 (Burr and Burr, 1991) was used for mapping of the Bx7 gene employing a PCR polymorphism. Bx7 is located 9 cM distal to the marker adh2 at position 47 (Fig. 7); the complete set of DIMBOA-glc biosynthetic genes is found within 41 cM. Linkage of Bx7 to the 6-cM cluster of Bx1 to Bx5, Bx6, and Bx8 genes, however, is loose. Two other OMTs, Zrp4 and the maize COMT1 Brown midrip3 (Bm3), were located in the same chromosomal region.

### DISCUSSION

#### All Genes of Benzoxazinone Biosynthesis in Maize Are Isolated

The isolation of Bx6 and Bx7 completes the characterization of benzoxazinoid biosynthesis. The pathway displays the typical features of plant secondary metabolic biosynthesis: The branch point from the primary metabolism is created by gene duplication, in this case of the TSA (Trp synthase α) gene, and subsequent modification of the duplicated gene to Bx1 (Frey et al., 1997). The same sequence of events is reported for the branch point reaction of pyrrolizidine alkaloids and saponins (Ober and Hartmann, 1999; Qi et al., 2004). A set of enzymatic functions commonly found in secondary metabolic pathways, namely, cytochrome P450s (BX2–BX5), a 2ODD (BX6), and an OMT (BX7), are recruited for functionalization of the primary product. The enzymes employed in the pathway are quite specific: It has been shown that the P450 enzymes are substrate specific and produce only one oxygenation product (Glawischnig et al., 1999). Substrate specificity was also observed for the last enzymes in the pathway, the 2ODD BX6 and the OMT BX7. Both enzymes accept exclusively the glcs as substrates. BX6 and BX7 do not have a close relative among the characterized members of the respective gene family (Fig. 5; Frey et al., 2003).

#### The 2-Oxoglutarate-Dependent Dioxygenase BX6 and OMT BX7 Require a Glucosylated Substrate

2ODDs are well characterized in flavonoid biosynthesis (flavanon 3β-hydroxylase, flavonol synthase, anthocyanidin synthase); the biosynthesis of GAs (e.g. GA 7-and GA 20-oxidase), ethylene (1-aminocyclopropane-1-carboxylate oxidase), and abscisic acid (9-cis-epoxy-carotenoid dioxygenase); and in alkaloid biosynthesis. The pathway-specific 2ODDs can have overlapping
Jonczyk et al.

Figure 5. Phylogenetic net of plant OMTs. Enzymes with defined substrates were subjected to the analysis using the program T-Coffee for the alignment (Poirot et al., 2003), and the evolutionary network was computed with SplitsTree (Huson and Bryant, 2006). COMTs from different species form a distinct relation group; a second group comprises enzymes (SAGAMT, JAGAMT, IAMT, and BAMT) responsible for methylation of plant hormones (salicylic acid, jasmonic acid). The third cluster includes loosely related OMTs involved in methylation of intermediates of different secondary metabolite pathways. BX7 is linked to this group.

substrate and product spectra (e.g. flavonoid biosynthesis; Turnbull et al., 2004) and may catalyze successive reactions of a pathway (e.g. hydroxylation and epoxidation of hyoscyamine to scopolamine by H6H in *Anisodus tanguticus*; Liu et al., 2005). Other ODDs use only defined substrates such as the flavanol-6-hydroxylase of *Chrysosplenium americanum* (Anzellotti and Ibrahim, 2000, 2004) that requires a defined methylation pattern of the substrate. The 2ODD BX6 of benzoxazinone biosynthesis, originally defined by the analysis of insertion mutants (Frey et al., 2003), similarly exhibits substrate specificity: The only benzoxazinoid that is accepted by the enzyme is DIBOA-glc. The reaction is characterized by a $V_{\text{max}}$ of 59 $\mu$kat g$^{-1}$ that is in the higher range determined for 2ODDs (e.g. petunia [*Petunia hybrida*] flavanon 3$\beta$-hydroxylase, substrate naringenin, 31.8 $\mu$kat g$^{-1}$; Wellmann et al., 2004) and a relatively high $K_{\text{m}}$ of 373 $\mu$M for DIBOA-glc. Since benzoxazinoids in maize seedlings reach concentrations of 40 mM (data not shown), the natural milieu contains probably sufficient substrate concentrations for BX6.

OMTs have been originally categorized as promiscuous enzymes that methylate phenylpropanoid and alkaloid compounds (Frick and Kutchan, 1999). Progress in cloning of OMTs and the advent of first crystal structures (Zubieta et al., 2001, 2002; Gang et al., 2002)

revealed that OMTs may have subtle structure-function relationships and are substrate specific. Recently, the evolution of defined substrate requirements was demonstrated for compounds in the scent of *V. planifolia* (Li et al., 2006). BX7 may represent an OMT with a narrow substrate spectrum. TRIBOA-glc is the only known substrate that is metabolized, the aglucone is not accepted. Several common substrates of OMTs were assayed but no methylation was observed. Especially, apigenin, the substrate of the flavonoid-7-methyltransferase isolated from barley (Christensen et al., 1998), is not metabolized; flavonoid-7-methyltransferase is the closest known relative to BX7 with a defined substrate requirement.

In conclusion, BX7 and BX6 represent enzymes that perform precisely one catalytic step each in the benzoxazinoid biosynthetic pathway. Both enzymes have $K_{\text{m}}$ values at the upper level found for the enzyme class combined with a sound $V_{\text{max}}$.

Secondary metabolites are often stored in the vacuole. Glycosylation is discussed as an essential feature for transport across the tonoplast. In maize and petunia, cyanidine 3-glc is transported to the tonoplast by a carrier protein (BZ2 and AN9, respectively) and delivered to a multidrug resistance-like protein in the vacuolar membrane (Goodman et al., 2004). However, glycosylation is not necessarily the final step of the

Plant Physiol. Vol. 146, 2008

Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.

The toxic aglucone is produced by a specific glucosidase (Esen, 1992). The toxicity depends largely on the reactivity of both the N-OH function and the presence of a cyclic hemiacetal unit. The hemiacetal undergoes an oxocyclo tautomerization. It has been shown that the aldehyde group of the oxo form reacts with the ε-NH2 group of N-α-acetyl-Lys, a model substrate for Lys residues in proteins (Perez and Niemeyer, 1989). DIBOA and DIMBOA function as enzyme inhibitors of, for example, α-chymotrypsin (Cuevas et al., 1990), aphid cholinesterase (Cuevas and Niemeyer, 1993), and plasma membrane H⁺-ATPase (Friebe et al., 1997). DIBOA biosynthesis appears as the core pathway for benzoxazinoids. In some species (e.g. maize and wheat), DIBOA is further modified to yield DIMBOA. DIMBOA provides the plant with additional functions. It is a distinctively more reactive compound than DIBOA. As a donor, the 7-MeO group facilitates N-O bond heterolysis (Hashimoto and Shudo, 1996) and the dehydration of DIMBOA (Hofmann and Sicker, 1999). The first process leads to the formation of a reactive, multicentered cationic electrophile; the latter results in the generation of a reactive formyl donor toward -NH2, -OH, and -SH groups. Hence, biosynthesis of DIMBOA instead of DIBOA seems to be of evolutionary advantage for the plant by producing a more reactive chemical defense.

In plants, genes associated with common metabolic pathways are generally unlinked. Remarkably, all six genes of the core biosynthesis in maize, including a

**Figure 6. Expression of DIMBOA-glc biosynthesis genes.** Three- and 4-d-old etiolated seedlings of line B73 were dissected in root, scutellar node and mesocotyl (sc), and shoot (leaves and coleoptile); after 6 d, mesocotyl, leaves, and coleoptile were pooled (mshoot). Tissues of older plants were taken after 3 weeks (root) and 10 weeks. Steady-state mRNA levels were measured by real-time reverse transcription-PCR using gene-specific primer pairs. GAP C mRNA quantities were determined in parallel and used for normalization.

biosynthetic pathway. Examples for glucosylated intermediates are given in the glucosinolate biosynthesis of Arabidopsis (*Arabidopsis thaliana*; Grubb et al., 2004), in loganin biosynthesis of *Lonicera japonica* (Katano et al., 2001), and in aurone biosynthesis of *Antirrhinum majus* (Nakayama et al., 2000). These pathways include a sulfotransferase, a cytochrome P450 enzyme, and a polyphenol oxidase, respectively, that modifies the glucosylated substrate. To our knowledge there are no reports of plant 2ODDs and class II OMTs that are specific for a glucosylated substrate. At first glance, one would expect the biosynthetic pathway of benzoxazinoid biosynthesis to proceed through the final modification step to DIMBOA and to conclude with glucosylation. However, DIBOA is the first toxic intermediate of the pathway (see below). Reduction of its reactivity by glucosylation might be required to reduce autotoxicity and to provide a stable metabolite for further modifications.

Modification of glcs may be catalyzed by enzymes with vacuolar location, e.g. the aureusidin synthase AmAS1 of *A. majus* is located in the vacuole (Ono et al., 2006). However, cytoplasmic localization of BX6 was substantiated by the BX6-GFP fusion analysis. According to our findings, both enzymes, BX6 and BX7, display no characteristics of vacuolar proteins: No signal peptides are detectable and the pH optima are not shifted to the acidic range of vacuoles. Hence, it is inferred that DIBOA-glc is converted to DIMBOA-glc prior to translocation to the vacuole.

**Evolution of DIMBOA-glc Biosynthesis**

Upon cell damage, the benzoxazinone glc is released from the vacuole that is used as a storage compartment.

Plant Physiol. Vol. 146, 2008

Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
Jonczyk et al.

lyase, P450s, and a glucosyltransferase, and the ODD Bx6 are linked within 6 cM on the short arm of chromosome 4 (Fig. 7). The OMT Bx7 is more loosely associated but also located on the short arm of chromosome 4. In wheat, the genes of the core biosynthesis are found on two chromosomes and it was proposed that clustering was similar to maize in the original state of a putative a wheat progenitor (Nomura et al., 2003). One other example for genetic linkage of defense pathway-related genes is reported in oats (*Avena sativa*). Five genes of saponin biosynthesis, including a gene for a glycosylating enzyme, map within 3.6 cM in *Avena sativa* (Qi et al., 2004). It has been suggested that clustering has the potential to facilitate coordinate regulation of expression at the chromatin level (Qi et al., 2004). Additionally, we suggest that clustering of the genes is of selective advantage in a population once beneficial allelic combinations are established in coupling phase.

There is ample evidence that the core benzoxazinoid biosynthesis is of monophyletic origin in grasses. It has been shown for wheat that a (not-yet cloned) 2ODD is responsible for C-7 oxygenation (Frey et al., 2003), hence, the same enzyme class is used for modification of DIBOA in wheat and maize. However, it remains to be shown if the C-7 hydroxylation and methylation are carried out by orthologous functions in the Poaceae.

### MATERIALS AND METHODS

All chemicals used were pro analysis or HPLC grade. The substrates apigenin (5,7,4′-trihydroxyflavon), caffeic acid (3,4-dihydroxy-cinnamic acid), ferulic acid (trans-4-hydroxy-3-methoxycinnamic acid), and quercetin were purchased from Sigma-Aldrich. Plasmid Bluescript KS+ (Stratagene) was used as a cloning vector. Sequences were isolated from libraries described by Frey et al. (1997).

#### Plant Materials

Protein purification was performed with the hybrid maize (*Zea mays*) line LG22.44 (Limagrain). Whole seedlings were used for protein preparation. Expression data were collected on inbred line B73. Rye (*Secale cereale*) inbred Halo (Lochow-Petkus GmbH) and *Lamium galeobdolon* grown in Staudengarten, Freising, were used for isolation of DIBOA-glc.

#### Synthesis of TRIBOA

Two alternative syntheses for TRIBOA have been already reported (Kluge et al., 1995). In the former procedure improvements have been introduced consisting in the careful extractive removal of tiny amounts of lactam that hitherto accompanied the hydroxamic acid precursor 7-benzyloxy-4-hydroxy-2-methoxy-2/*H*/-1,4-benzoxazin-3(4/*H*/)-one as a by-product of the reductive cyclization of a suitable nitro precursor by transfer hydrogenation to this hydroxamic acid. To avoid any side components, the following synthetic sequence was reinvestigated: Methyl 2-methoxy-2-(5-benzyloxy-2-nitrophenoxy) acetate was used as starting material and subjected to a reductive cyclization by transfer hydrogenation with Pt-C/NaBH₄ in aqueous methanol to form 7-benzyloxy-4-hydroxy-2-methoxy-2*H*-1,4-benzoxazin-3(4*H*)-one. A procedure has been developed to extract the cyclic hydroxamic acid from the crude cyclization product with NaOH followed by acidification. The material thus obtained was hydrogenated at ambient pressure over Pd-C in methanol to yield 4,7-dihydroxy-2-methoxy-2*H*-1,4-benzoxazin-3(4*H*)-one. Eventually, this methyl acetal of TRIBOA was cleaved with boron trichloride in methylene chloride to yield crude TRIBOA. TRIBOA was purified by a column chromatography over Lichroprep RP-18 silica gel and isolated from the aqueous eluent fraction carefully by lyophilization. TRIBOA thus obtained was free of other heterocycles (according to thin-layer chromatography and to NMR analysis) and showed the correct mp 172°C to 173°C (off-white microcrystalline powder).

#### Isolation of Substrates and Standards

DIBOA-glc, DIMBOA-glc, and DIBOA were isolated as described by von Rad et al. (2001). TRIBOA-glc was generated by two alternative approaches. For the generation of the reference substance chemically synthesized TRIBOA was glucosylated in vitro using the heterologously expressed UDP-glucosyltransferase BX8 (von Rad et al., 2001). Routine large-scale production of TRIBOA-glc was carried out by incubation of DIBOA-glc with raw extracts of isopropylthio-β-galactoside-induced *Escherichia coli* BL21(DE3) expressing *Bx6* from the plasmid pET3a (Studier and Moffatt, 1986). Three hours after induction with 1 mM isopropylthio-β-galactoside, the cells were resuspended in 100 mM Tris-HCl (pH 7.5), 20 mM NaCl, 5 mM dithiothreitol (DTT), 1 mM phenylmethylsulfonyl fluoride, 20% glycerol, and 1 mg/mL lysosome (1 g of bacteria pellet per 3 mL of buffer). Five cycles of freeze (liquid N₂) and thaw (ice) were applied, RNase and DNase (10 μg/mL each) were added, and solution was cleared by centrifugation. Aliquots were stored at −70°C. For TRIBOA-glc synthesis, 100 μL of the protein solution were used for 1 mL assay volume. The reaction buffer contained 100 mM MES-NaOH (pH 6.0), 5 mM DTT, 10 mM ascorbate, 10 mM 2-oxoglutarate, 4 mM FeSO₄, and 3 mM DIBOA-glc. The reaction was incubated at 30°C with gentle agitation and after 45 min stopped by the addition of 1 volume of methanol. Precipitated protein was removed by centrifugation and the supernatant was applied to HPLC. TRIBOA-glc was eluted with 9% acetonitrile and 91% of 0.3% formic acid using Merck LiChroCART RP-18e.

#### HPLC Analysis

All samples were analyzed on a Beckman HPLC System Gold, with Programmable Solvent Module 126, Diode Array Detector Module 168, and Autosampler 508. Data were collected and processed by using 32 Karat software. Instrumental conditions for analysis were: Merck LiChroCART RP-18e (250 × 4 mm, 5 μm analytical, and 250 × 10 mm, 10 μm preparative) reversed-phase column at 25°C. Mobile phases were: 0.3% formic acid (A) and acetonitrile (B). Injection volume was 50 μL or 2 mL and the flow rate was 1 or 5 mL min⁻¹ in analytical and preparative analysis, respectively. The following gradient was used for separation for the analytical analysis: at 0 min, 7.5% B; 1 min, 7.5% B; 2 min, 10.5% B; 9 min, 10.5% B; 10 min, 13% B; 11 min, 13% B; 12 min, 22% B; 17 min, 22% B; and 18 min, 100% B. Under these conditions, the following retention times were obtained for each compound: DIBOA-glc, 12 min; TRIBOA-glc, 6.4 min; DIMBOA-glc, 16.1 min. The detection was carried out at 254 nm for DIBOA-glc and 266 nm for DIMBOA-glc and TRIBOA-glc.

#### MS

The HPLC-MS investigations were performed with a HP 1100 liquid chromatograph equipped with a diode array detector (280 nm) and a triple quadrupole mass spectrometer (API 2000, Applied Biosystems). A Nucleosil C₁₈ (Jasco) column was used, 3 mm × 250 mm × 5 μm. The solvent system consisted of pure water (solvent A) and methanol/isopropanol (95/5), containing 0.025% acetic acid (solvent B), respectively. The mobile phase was used as a three-step gradient: The first part was isocratic (8% B for 2 min), subsequently followed by a linear gradient (from 8%–50% B in 9 min) and followed by a third, again isocratic part (50% B). The flow was 150 μL/min; mass spectra were recorded in the negative mode.

#### Purification of BX7

Maize seedlings were grown on wet paper for 4 d in the dark at 28°C. All purification procedures were carried out at 4°C. Protein was extracted from 120 g seedling shoots with 4 volumes (w/v) extraction buffer (20 mM Tris-HCl, pH 7.5, 140 mM NaCl, 1 mM EDTA, and 10% glycerol) and 0.3 volumes (w/w) Polyclar (Serva). After filtration through miracloth and centrifugation for 1 h at 10,000g, the supernatant was sterile filtered (0.2 μm Sartorius AG). Aliquots of the protein extract (about 50 mg each) were applied to the affinity column. The affinity column was prepared according to the manufacturer’s instructions and as described by Attieh et al. (1995). Briefly, 20 mL of 20% ethanol was added to

Plant Physiol. Vol. 146, 2008

Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
5 mL of adenosine 5-monophosphate agarose (Sigma). After 1 h of swelling, the supernatant was decanted, and the slurry was resuspended first in water and subsequently in 50 mm Tris-HCl, pH 8.5. The agarose was incubated with 1,000 units of alkaline phosphatase type VII (Sigma-Aldrich) at 37°C overnight in a continuously rotating reaction vial. The material was washed once with 20% ethanol and twice with water, the slurry transferred to a column, and equilibrated in extraction buffer. After application of the protein extract, the column was washed with a flow of 0.2 mL/min with extraction buffer for 1 h. After washing with buffer B (extraction buffer with 0.2 M KCl, flow 0.2 mL/min), proteins were eluted with a gradient of SAM in extraction buffer (0–4 mM in 100 min, flow 0.2 mL/min). Active fractions of four runs of the affinity chromatography were pooled, concentrated with VivaSpin columns (molecular weight cutoff 10,000; Sartorius), and a total of about 50 μg protein applied to a MonoQ column (Amersham Pharmacia Biotech, flow rate 40 mL/h) equilibrated in buffer B. Elution was with 20 column volumes 0.2 to 2.0 M KCl in buffer B. Fractions were assayed for methyltransferase activity and subjected to PAGE analysis. Candidate bands were cut from the gel (see Supplemental Fig. S3) and digested following the protocol of Schäfer et al. (2001). Sample preparation for MALDI-MS was performed using ZipTips (Qiagen) following the manufacturer’s protocol. For MALDI-MS and MALDI MS/MS analysis, an Ultraflex I ToF/ToF mass spectrometer (Bruker Daltonik) was used. Data analysis was performed using the BioTools, RapiDeNovo (Bruker Daltonik), and Mascot (Matrix Science) software packages. For determination of de novo amino acid sequences, no modifications were allowed and the b and y ion series were weighted 5 times higher than other ion series. The amino acids Ile and Gln were excluded. The most sequence parts determined with highest significance were searched against the National Center for Biotechnology Information and OMT sequences selected from the Pioneer EST collection. The selected sequences were in silico digested with trypsin and directly compared to the observed MS spectra of the tryptic digests. The sequence with the highest peptide overlap was selected.

### Heterologous Protein Expression and Purification of His-Tag Proteins

For heterologous expression *Bx6* cDNA was fused to the C-terminal His-tag of pET3a-His and *Bx7* cDNA was joined to the N-terminal His-tag of pET28 via introduction of an *NdeI* restriction cut comprising the start codon. Induction and purification of the proteins under native conditions was as described by the manufacturer of the nickel-nitrilotriacetic acid agarose (Qiagen). Eluted proteins were dialyzed against 50 mM Tris-HCl (pH 7.5), 1 mM DTT, frozen in liquid N₂, and stored in aliquots in −70°C. Protein concentrations were determined using the Bio-Rad protein assay kit with bovine serum albumin as standard.

### Enzyme Assays

Heterologous BX6 expression in *E. coli* was as a C-terminal His-tagged protein and without His-tag. Purified His-tagged protein and untagged protein in crude extract were assayed to determine the influence of the tag on BX6 activity. Both enzymes had very similar activity, indicating that the His-tag does not interfere with catalysis. Purified His-tagged protein was used for determination of kinetic parameters; crude extract was used for determination of pH dependency.

For determination pH optimum of BX6 activity the assay (1.5 mg/mL) contained 100 mM buffer (acetate, 4.5–5.0; citrate, pH 5.0–6.4; MES-NaOH, 5.0–6.5; potassium-phosphate, 6.0–8.0; BisTris-HCl, 6.0–7.0; Tris-HCl, 6.5–9.5; AMP-HCl, 9.0–10.5; CAPS-NaOH, 10.0–11.0), 5 mM DTT, 10 mM ascorbate, 10 mM 2-oxoglutarate, 4 mM FeSO₄, 0.5 mM DIBOA-glc. At 1, 2, 3, 4, 6, 8, 10, and 15 min after initiation, 100-μL aliquots were removed from the reaction, immediately combined with 1 volume of methanol, and frozen in liquid N₂. Precipitated protein was removed by centrifugation and the supernatant was analyzed on HPLC. The amount of DIMBOA-glc formation was quantified using 32 Karat software.

BX6 initial rate data were obtained using the assay as described above, using 100 mM potassium phosphate buffer (pH 7.0) and 20 μg/mL His-tag purified protein. Except for DIBOA-glc, the assay components were mixed and incubated at 30°C for 10 min. The reaction was then initiated by the addition of DIBOA-glc. When 2-oxoglutarate was the varied substrate, 2-oxoglutarate instead of DIBOA-glc was used to initiate the assay; DIBOA-glc was fixed at 2.5 mM concentration. Initial rates were calculated from progress curves of TRIBOA-glc formation using the exact numerical method described by Cornish-Bowden (1975). Lineweaver-Burk plots were constructed to determine apparent *K*~m~ and *V*~max~ values. Calculations were performed using GraphPad Prism v.4.0 software.

BX7 initial rate data were obtained using an assay as described previously (Schröder et al., 2004). The standard assay contained 50 mM Tris-HCl (pH 7.0), 2 mM DTT, 0.2 mM SAM, and 20 μg/mL His-tagged protein. Except for TRIBOA-glc, the assay components were mixed and incubated at 30°C for 10 min. The assay was then initiated by the addition of TRIBOA-glc, and the reaction was monitored essentially as described for BX6.

The pH dependency of BX7 was monitored as described above, using 50 mM buffer (acetate, 4.5–5.0; citrate; MES-NaOH, 5.0–6.5; Tris-HCl, 6.5–9.5; AMP-HCl, 9.0–10.5; CAPS-NaOH, 10.0–11.0), 2 mM DTT, 50 μM SAM, 0.1 mM TRIBOA-glc, and 0.1 mg/mL His-tagged BX7 protein.

### Subcellular Localization

For the determination of subcellular localization of BX6, the complete *Bx6* coding sequence was translationally fused to the C terminus of GFP variant mGFP6 (Curtis and Grossniklaus, 2003) via a 10-mer Ala linker. Ligation of the respective restriction fragments yields the cassette *mGFP-(Ala)₁₀-Bx6*. The first 440 bp of *Bx6* were amplified by PCR, and a 10-mer Ala linker was introduced at the C terminus of the encoded polypeptide [*Bx6N-part-(Ala)₁₀*]. The sequenced PCR fragment was joined with *mGFP-(Ala)₁₀-Bx6* to yield the translational fusion *Bx6(N-part)-(Ala)₁₀-mGFP6-(Ala)₁₀-Bx6* (*Bx6part-GFP-Bx6*). The fusion construct was joined with the 2 × 35S-promoter and nos-terminator of pMDC201 (Curtis and Grossniklaus, 2003) in pUC19. The cassette *2x35S-promoter-mGFP6-nos-terminator* in pUC19 was used as a control construct in the expression analysis. For stable transformation, the respective promoter-gene-terminator cassettes were isolated as *HindIII-EcoRI* restriction fragments and ligated into the Ti-Plasmid pTF101Ubi, a derivative of pTF101.1 that drives the bar gene with the *ubiquitin* promoter. Transgenic lines were generated as described by Frame et al. (2002).

For transient expression of the GFP-reporter constructs, protoplasts were isolated from aseptically grown plants, 12 d after imbibition. Preparation and electroporation was as described by J. Sheen (http://genetics.mgh.harvard.edu/sheenweb). Thirty micrograms of the pUC19-based plasmid were used for each electroporation of 1 to 2 × 10⁵ protoplasts. Analysis was 24 h after electroporation with Zeiss Axiophot equipped with HQ-Filterset for enhanced GFP (AHF Analysentechnik AG). Photographs were taken with DCS 670 Digital Nikon F5 SLR camera and analyzed with the software Kodak DCS Photodesk. Protoplasts from transgenic plants were isolated and analyzed accordingly.

### Determination of Transcription Levels by Quantitative Reverse Transcription-PCR

RNA isolation, cDNA synthesis, and real-time reverse transcription-PCR was as described (von Rad et al., 2001). The gene-specific primer pair *Bx6F*: 5′-CCTGAGCGACATAATTTCTCC-3′ and *Bx6R*: 5′-TGGAGTAGAGCGCC-TGCTTCGCCCT-3′ was used for the amplification of 250-bp fragment specific for *Bx6*. A *Bx7*-specific fragment of 440 bp was amplified with the primer pair *Bx7F*: 5′-TCGACCGCTGCGGGGGAGC-3′ and *Bx7R*: 5′-ATGCTCTCGTA-GAAGCTGGCCCCT-3′. Two biological replicates were used for RNA isolation and the analysis of every RNA sample was done in replicate.

### Mapping

*Bx7* was mapped by PCR amplification of allele-specific fragments. The primer pair CM37F: 5′-CCAACCACGACTACCTCCGG-3′ and CM37R: 5′-GAG-GATGCTCGGGGTAGTAGC-3′ amplified a band of 204 bp on CM37 and no band on T232 genomic DNA. The primer pair T232F: 5′-CCAACCACGACTACCTC-CGA-3′ and T232R: 5′-GAGGATGCTCGGGGTAGTAGT-3′ amplified a band of 218 bp on T232 genomic DNA and no band with CM37. Annealing was at 65°C for 30 s and elongation was for 30 s. A final concentration of 3% dimethyl sulfoxide was included in the reaction buffer.

Novel materials described in this publication may be available for noncommercial research purposes upon acceptance and signing of a material transfer agreement. In some cases such materials may contain or be derived from materials obtained from a third party. In such cases, distribution of material will be subject to the requisite permission from any third-party owners, licensors, or controllers of all or parts of the material. Obtaining any permissions will be the sole responsibility of the requestor. Plant germplasm and transgenic material
Jonczyk et al.

will not be made available except at the discretion of the owner and then only in accordance with all applicable governmental regulations.

Sequence data from this article can be found in the GenBank/EMBL data libraries under the following accession numbers: ARATHOMT, NP_200227; BAMT, AAF98284; BX7, EU192149; CATROCOMT, AY028439; CLABRCOMT, O23760; CHRAEOMT, Q42653; COFCACOMT, AAN03727; COPJACOOMT, Q8H9A8; COPJASMT, Q39522; COPJA3OMT, Q9LEL6; 2COPJAOMT, Q9LEL5; GRAMINVUNMT, AAC18643; HVFlavonoid-7OMT, CAA54616; IAMT, NP_200336; JAGMT, NP_200441; MEDSAOMT, O24529; MESCRIMT, P45986; OCIBACVMT, Q93WU3; OCIBAEOMT, Q93WU2; ORYSAOMT, Q6ZD89; POPKICOMT, Q43047; PRUDUCOMT, Q43609; SAGAMT, NP_194372; ZINELCOMT, ZEU19911; ZMAYSCOMT, Q06509; ZMAYSZRP4, P47917.

### Supplemental Data

The following materials are available in the online version of this article.

- **Supplemental Figure S1.** Purification of BX7.
- **Supplemental Figure S2.** Comparison of BX7 sequence with identified peptide sequences of the purified enzyme.

### ACKNOWLEDGMENTS

We thank Peter Dobos for assisting with plant transformation and Regina Hüttl for excellent technical assistance. D.S. wants to thank Martin Krug and Enzo Mai for their contributions in improving the synthesis of TRIBOA.

Received October 18, 2007; accepted January 8, 2008; published January 11, 2008.

### LITERATURE CITED

- Anzellotti D, Ibrahim RK (2000) Novel flavonol 2-oxoglutarate dependent dioxygenase: affinity purification, characterization, and kinetic properties. Arch Biochem Biophys 382: 161–172
- Anzellotti D, Ibrahim RK (2004) Molecular characterization and functional expression of flavonol 6-hydroxylase. BMC Plant Biol 13: 20
- Attieh JM, Hanson AD, Saini HS (1995) Purification and characterization of a novel methyltransferase responsible for biosynthesis of halo-methanes and methanethiol in Brassica oleracea. J Biol Chem 270: 9250–9257
- Barnes JP, Putnam AR (1987) Role of benzoxazinones in allelopathy by rye (Secale cereale L.). J Chem Ecol 13: 889–990
- Benson RJ, Johal GS, Crane VC, Tossberg JT, Schnable PS, Meeley RB, Briggs SP (1995) Cloning and characterization of the maize *An1* gene. Plant Cell 7: 75–84
- Bravo HR, Lazo W (1993) Antimicrobial activity of cereal hydroxamic acids and related compounds. Phytochemistry 33: 569–571
- Burr B, Burr FA (1991) Recombinant inbreds for molecular mapping in maize: theoretical and practical considerations. Trends Genet 7: 55–60
- Carter C, Pan S, Zouhar J, Avila EL, Girke T, Raikhel NV (2004) The vegetative vacuole proteome of *Arabidopsis thaliana* reveals predicted and unexpected proteins. Plant Cell 16: 3285–3303
- Christensen AB, Gregersen PL, Olsen CE, Collinge DB (1998) A flavonoid 7-O-methyltransferase is expressed in barley leaves in response to pathogen attack. Plant Mol Biol 36: 219–227
- Cornish-Bowden A (1975) The use of direct linear plot for determining initial velocities. Biochem J 149: 305–312
- Cuevas L, Niemeyer HM (1993) Effect of hydroxamic acids from cereals on aphid cholinesterase. Phytochemistry 34: 983–995
- Cuevas L, Niemeyer HM, Perez FJ (1990) Reaction of DIMBOA, a resistance factor from cereals, with α-chymotrypsin. Phytochemistry 29: 1429–1432
- Curtis MD, Grossniklaus U (2003) A Gateway cloning vector set for high-throughput functional analysis of genes in planta. Plant Physiol 133: 462–469
- Esen A (1992) Purification and partial characterization of maize (*Zea mays*) β-glucosidase. Plant Physiol 98: 174–182
- Frame BR, Shou H, Chikwamba RK, Zhang Z, Xiang C, Fonger TM, Pegg

SE, Li B, Nettleton DS, Pei D, et al (2002) Agrobacterium tumefaciens-mediated transformation of maize embryos using a standard binary vector system. Plant Physiol 129: 13–22
Frey M, Chomet P, Glawischnig E, Stettner C, Grün S, Winklmair A, Eisenreich W, Bacher A, Meeley RB, Briggs SP, et al (1997) Analysis of a chemical plant defense mechanism in grasses. Science 277: 696–699
Frey M, Huber K, Park WJ, Sicker D, Lindberg P, Meeley RB, Simmons CR, Yalpani N, Gierl A (2003) A 2-oxoglutarate-dependent dioxygenase is integrated in DIMBOA-biosynthesis. Phytochemistry 62: 371–376
Frick S, Kutchan TM (1999) Molecular cloning and functional expression of O-methyltransferases common to isoquinoline alkaloid and phenylpropanoid biosynthesis. Plant J 17: 329–339
Friebe A, Roth U, Kück P, Schnabl H, Schulz M (1997) Effects of 2,4-dihydroxy-1,4-benzoxazin-3-ones on the activity of plasma membrane H⁺-ATPase. Phytochemistry 44: 979–983
Gang DR, Lavidb N, Zubietac C, Chena F, Beuerle T, Lewinohn E, Noel JP, Pichersky E (2002) Characterization of phenylpropene O-methyltransferases from sweet basil: facile change of substrate specificity and convergent evolution within a plant O-methyltransferase family. Plant Cell 14: 505–519
Gauthier A, Gulick PJ, Ibrahim RK (1998) Characterization of two cDNA clones which encode O-methyltransferases for the methylation of both flavonoid and phenylpropanoid compounds. Arch Biochem Biophys 351: 243–249
Glawischnig E, Grun S, Frey M, Gierl A (1999) Cytochrome P450 monooxygenases of DIBOA biosynthesis: specificity and conservation among grasses. Phytochemistry 50: 925–930
Goodman CD, Casati P, Walbot V (2004) A multidrug resistance-associated protein involved in anthocyanin transport in *Zea mays*. Plant Cell 16: 1812–1826
Grombacher AW, Russell WA, Guthrie WD (1989) Resistance to first-generation European corn borer (Lepidoptera: Pyralidae) and DIMBOA concentration in midwhorl leaves of the BS9 maize synthetic. J Kans Entomol Soc 62: 103–107
Grubb CD, Zipp BJ, Ludwig-Müller J, Masuno MN, Molinski TE, Abel S (2004) Arabidopsis glucosyltransferase UGT74B1 functions in glucosinolate biosynthesis and auxin homeostasis. Plant J 40: 893–908
Grün S, Frey M, Gierl A (2005) Evolution of the indole alkaloid biosynthesis in the genus Hordeum: distribution of gramine and DIBOA and isolation of the benzoxazinoid biosynthesis genes from *Hordeum lechleri*. Phytochemistry 66: 1264–1272
Hamilton RH (1964) A corn mutant deficient in 2,4-dihydroxy-7-methoxy-1,4-benzoxazin-3-one with an altered tolerance of atrazine. Weeds 12: 27–30
Hashimoto Y, Shudo K (1996) Chemistry of biologically active benzoxazinoids. Phytochemistry 43: 551–559
Held BM, Wang H, John I, Wurtele ES, Colbert JT (1993) An mRNA putatively coding for an O-methyltransferase accumulates preferentially in maize roots and is located predominantly in the region of the endodermis. Plant Physiol 102: 1001–1008
Hietala PK, Virtanen AI (1960) Precursors of benzoxazolinone in rye plants. II. Precursor I, the glucoside. Acta Chem Scand A 14: 502–504
Hofmann A, Sicker D (1999) A formylating agent by dehydration of the natural product DIMBOA. J Nat Prod 62: 1151–1153
Huson DH, Bryant D (2006) Application of phylogenetic networks in evolutionary studies. Mol Biol Evol 23: 254–267
Joshi CP, Chiang VL (1998) Conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferases. Plant Mol Biol 37: 663–674
Katano N, Yamamoto H, Iio R, Inoue K (2001) 7-Deoxyloganin 7-hydroxylase in *Lonicera japonica* cell cultures. Phytochemistry 58: 53–58
Kluge M, Hartenstein H, Hantschmann A, Sicker D (1995) First syntheses of natural products with the 2,7-dihydroxy-2H-1,4-benzoxazin-3(4H)-one skeleton. J Heterocycl Chem 32: 395–402
Klun JA, Guthrie WD, Hallauer AR, Russell WA (1970) Genetic nature of the concentration of 2,4-dihydroxy-7-methoxy-(2H)-1,4-benzoxazin-3(4H)-one and resistance to the European corn borer in a diallel set of eleven maize inbreds. Crop Sci 10: 87–90
Kumar P, Moreland DE, Chilton WS (1994) 2H-1,4-benzoxazin-3(4H)-one, an intermediate in the biosynthesis of cyclic hydroxamic acids in maize. Phytochemistry 36: 893–898
Lavid N, Wang J, Shalit M, Guterman I, Bar E, Beuerle T, Menda N, Shafir S, Zamir D, Adam Z, et al (2002) O-methyltransferases involved in the biosynthesis of volatile phenolic derivatives in rose petals. Plant Physiol 129: 1899–1907

Plant Physiol. Vol. 146, 2008

Downloaded from www.plantphysiol.org on September 12, 2015 - Published by www.plant.org  
Copyright © 2008 American Society of Plant Biologists. All rights reserved.
Li HM, Rotter D, Hartman TG, Pak FE, Havkin-Frenkel D, Belanger FC (2006) Evolution of novel O-methyltransferases from the Vanilla planifolia caffeic acid O-methyltransferase. Plant Mol Biol 61: 537–552

Liu T, Zhu P, Cheng KD, Meng C, He HX (2005) Molecular cloning, expression and characterization of hyoscyamine 6beta-hydroxylase from hairy roots of Anisodus tanguticus. Planta Med 71: 249–253

Melanson D, Chilton MD, Masters-Moore D, Chilton WS (1997) A deletion in an indole synthase gene is responsible for the DIMBOA-deficient phenotype of bxbx maize. Proc Natl Acad Sci USA 94: 13345–13350

Nakayama T, Yonekura-Sakakibara K, Sato T, Kikuchi S, Fukui Y, Fukuchi-Mizutani M, Ueda T, Nakao M, Tanaka Y, Kusumi T, et al (2000) Aureusidin synthase: a polyphenol oxidase homolog responsible for flower coloration. Science 290: 1163–1166

Niemeyer HH (1988) Hydroxamic acids (4-hydroxy-1,4-benzoxazin-3-ones), defense chemicals in the gramineae. Phytochemistry 27: 3349–3358

Nomura T, Ishihara A, Imaishi H, Endo TR, Ohkawa H, Iwamura H (2002) Molecular characterization and chromosomal localization of cytochrome P450 genes involved in the biosynthesis of cyclic hydroxamic acids in hexaploid wheat. Mol Genet Genomics 267: 210–217

Nomura T, Ishihara A, Imaishi H, Ohkawa H, Endo TR, Iwamura H (2003) Rearrangement of the genes for the biosynthesis of benzoxazinones in the evolution of Triticeae species. Planta 217: 776–782

Nomura T, Ishihara A, Yanagita RC, Endo TR, Iwamura H (2005) Three genomes differentially contribute to the biosynthesis of benzoxazinones in hexaploid wheat. Proc Natl Acad Sci USA 102: 16490–16495

Ober D, Hartmann T (1999) Homospermidine synthase, the first pathway-specific enzyme of pyrrolizidine alkaloid biosynthesis, evolved from deoxyhypusine synthase. Proc Natl Acad Sci USA 96: 14777–14782

Ono E, Hatayama M, Isono Y, Sato T, Watanabe R, Yonekura-Sakakibara K, Fukuchi-Mizutani M, Tanaka Y, Kusumi T, Nishino T, et al (2006) Localization of a flavonoid biosynthetic polyphenol oxidase in vacuoles. Plant J 45: 133–143

Osakabe K, Tsao CC, Li L, Popko JL, Umezawa T, Carraway DT, Smeltzer RH, Joshi CP, Chiang VL (1999) Coniferyl aldehyde 5-hydroxylation and methylation direct syringyl lignin biosynthesis in angiosperms. Proc Natl Acad Sci USA 96: 8955–8960

Parvathi K, Chen F, Guo D, Blount JW, Dixon RA (2001) Substrate preferences of O-methyltransferases in alfalfa suggest new pathways for 3-O-methylation of monolignols. Plant J 25: 193–202

Perez FJ, Niemeyer HM (1989) Reaction of DIMBOA with amines. Phytochemistry 28: 1831–1834

Poirot O, O’Toole E, Notredame C (2003) Tcoffee@igs: a web server for computing, evaluating and combining multiple sequence alignments. Nucleic Acids Res 31: 3503–3506

Qi X, Bakht S, Leggett M, Maxwell C, Melton R, Osbourn A (2004) A gene

cluster for secondary metabolism in oat: implications for the evolution of metabolic diversity in plants. Proc Natl Acad Sci USA 101: 8233–8238

Rea PA (2007) Plant ATP-binding cassette transporters. Annu Rev Plant Biol 58: 347–375

Scalliet G, Journot N, Jullien F, Baudino S, Magnard JL, Channelière S, Vergne P, Dumas C, Bendahmane M, Cock JM, et al (2002) Biosynthesis of the major scent components 3,5-dimethoxytoluene and 1,3,5-trimethoxybenzene by novel rose O-methyltransferases. FEBS Lett 523: 113–118

Schäfer H, Nau K, Sickmann A, Erdmann R, Meyer HE (2001) Identification of peroxisomal membrane proteins of *Saccharomyces cerevisiae* by mass spectrometry. Electrophoresis 22: 2955–2968

Schledzewski K, Mendel RR (1994) Quantitative transient gene expression: comparison of the promoters for maize polyubiquitin1, rice actin1, maize-derived Emu and CaMV 35S in cells of barley, maize and tobacco. Transgenic Res 3: 249–255

Schröder G, Wehinger E, Lukacin R, Wellmann F, Seefelder W, Schwab W, Schröder J (2004) Flavonoid methylation: a novel 4′-O-methyltransferase from *Catharanthus roseus*, and evidence that partially methylated flavanones are substrates of four different flavonoid dioxygenases. Phytochemistry 65: 1085–1094

Sicker D, Frey M, Schulz M, Gierl A (2000) Role of natural benzoxazinones in the survival strategy of plants. In KW Jeon, ed, International Review of Cytology, A Survey of Cell Biology, Vol 198. Academic Press, San Diego, pp 319–346

Studier FW, Moffatt BA (1986) Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J Mol Biol 189: 113–130

Tipton CL, Wang MC, Tsao FHC, Tu CCL, Husted RR (1973) Biosynthesis of 1,4-benzoxazin-3-ones in Zea mays. Phytochemistry 12: 347–352

Turnbull JJ, Nakajima J, Welford RW, Yamazaki M, Saito K, Schofield CJ (2004) Mechanistic studies on three 2-oxoglutarate-dependent oxygenases of flavonoid biosynthesis: anthocyanidin synthase, flavonol synthase, and flavanone 3beta-hydroxylase. J Biol Chem 279: 1206–1216

Virtanen AI, Hietala PK (1960) Precursors of benzoxazolinone in rye plants. II. Precursor II, the aglucone. Acta Chem Scand A 14: 499–502

von Rad U, Hüttl R, Lottspeich F, Gierl A, Frey M (2001) Two glucosyltransferases are involved in detoxification of benzoxazinoids in maize. Plant J 28: 633–642

Wellmann F, Matern U, Lukacin R (2004) Significance of C-terminal sequence elements for *Petunia* flavanone 3beta-hydroxylase activity. FEBS Lett 561: 149–154

Zubieta C, He XZ, Dixon RA, Noel JP (2001) Structures of two natural product methyltransferases reveal the basis for substrate specificity in plant O-methyltransferases. Nat Struct Biol 8: 271–279

Zubieta C, Kota P, Ferrer JL, Dixon RA, Noel JP (2002) Structural basis for the modulation of lignin monomer methylation by caffeic acid/5-hydroxyferulic acid 3/5-O-methyltransferase. Plant Cell 14: 1265–1277

A
a b c d e f g h
55 —
36 —

B
b c d e f g h

A
MGHQAQHGTDDTEELLAahr*QLWCHALGYVK*SMALK*CALDLRIPDTIDR*CGGSA
TLGELLAASEISASNHDYLR*R*VMR*TLTAMR*IFAASHDPAK*ADDAAAISSYOLTP
ASR*LLVSSSSSVDDAAGASK*ENTTTPSILPNIAHLVR*PNTISLLFSMGEWMK*DE
SAASVSLYETVHR*QGMWACVEDDAANR*ASFYESMDADTR*LVMQAVVRR*CPHVFD
GIK*SLVDVGGGR*GTA AA AVVA AFPHIQR*CTVMDLPHVVAEAPAGTAGLSFHGGDM
FEHIPSADALMLK*WILHDWDEDK*CIK*IMMER*CK*EAIGGKEAGGKVIIIDTVLG
SR*ADDDDDDK*TCR*ETYVLDLHILSFVNGAER*EEHEWR*R*IFLAAGFR*DYK*I
THTR*GIPSIIEVFP

B


The image contains a graph with the following axes and labels:

- **X-axis**: m/z (mass-to-charge ratio)
- **Y-axis**: Intens. [a.u.] (Intensity in arbitrary units)

The graph shows multiple peaks at various m/z values, with some labeled as follows:
- 775.414
- 859.455
- 931.535
- 1050.613
- 1185.708
- 1269.630
- 1356.628
- 1473.695
- 1535.857
- 1649.824
- 1764.849
- 1802.923
- 1892.939
- 2028.939
- 2127.970
- 2256.035
- 2602.258
- 2720.313
- 2807.299
- 2932.588

The graph also includes a zoomed-in section showing peaks around the m/z range of 1944 to 1954. The intensity scale for this section is from 0 to 5000.
SUPPLEMENTAL FIGURE LEGENDS

Supplemental Figure S1: Purification of BX7. A Coomassie-stained Laemmli gel. Lane a: Molecular weight marker (116, 97, 84, 66, 55, 36, 31, 20, 14 kDa), lane b: selected fraction from affinity column, lane c through h: individual fractions from anion-exchange chromatography (MonoQ). The asterisk marks the band representing BX7. B HPLC analysis (section) of enzyme assays using the respective fractions displayed in panel A. The arrow points to the position of DIMBOA-glc.

Supplemental Figure S2: Comparison of BX7 sequence with identified peptide sequences of the purified enzyme. A Sequence homology: Peptide fragments of BX7 that show up in the MS analysis are highlighted. Blue color indicates a full match, grey color specifies similarity between BX7 and a deduced peptide sequence. The asterisk marks trypsin cutting sides. B Overlay of molecular mass profiles derived from purified protein and heterologous expressed BX7.
