
Molecular Cloning and Characterization of CYP80G2, a Cytochrome P450 That Catalyzes an Intramolecular C–C Phenol Coupling of (S)-Reticuline in Magnoflorine Biosynthesis, from Cultured *Coptis japonica* Cells*

Received for publication, June 20, 2007, and in revised form, January 15, 2008 Published, JBC Papers in Press, January 29, 2008, DOI 10.1074/jbc.M705082200

Nobuhiro Ikezawa†, Kinuko Iwasa§, and Fumihiko Sato‡§

From the †Division of Integrated Life Science, Graduate School of Biostudies, Kyoto University, Kitashirakawa, Sakyo, Kyoto 606-8502 and the §Kobe Pharmaceutical University, Kobe, Hyogo 658-8558, Japan

Cytochrome P450s (P450) play a key role in oxidative reactions in plant secondary metabolism. Some of them, which catalyze unique reactions other than the standard hydroxylation, increase the structural diversity of plant secondary metabolites. In isoquinoline alkaloid biosyntheses, several unique P450 reactions have been reported, such as methylenedioxy bridge formation, intramolecular C–C phenol-coupling and intermolecular C–O phenol-coupling reactions. We report here the isolation and characterization of a C–C phenol-coupling P450 cDNA (CYP80G2) from an expressed sequence tag library of cultured *Coptis japonica* cells. Structural analysis showed that CYP80G2 had high amino acid sequence similarity to *Berberis stolonifera* CYP80A1, an intermolecular C–O phenol-coupling P450 involved in berbamunine biosynthesis. Heterologous expression in yeast indicated that CYP80G2 had intramolecular C–C phenol-coupling activity to produce (S)-corytuberine (aporphine-type) from (S)-reticuline (benzylisoquinoline type). Despite this intriguing reaction, recombinant CYP80G2 showed typical P450 properties: its C–C phenol-coupling reaction required NADPH and oxygen and was inhibited by a typical P450 inhibitor. Based on a detailed substrate-specificity analysis, this unique reaction mechanism and substrate recognition were discussed. CYP80G2 may be involved in magnoflorine biosynthesis in *C. japonica*, based on the fact that recombinant *C. japonica* S-adenosyl-L-methionine:coclaurine N-methyltransferase could convert (S)-corytuberine to magnoflorine.

Isoquinoline alkaloids are a large group of alkaloids and include many pharmacologically useful compounds; e.g. the analgesic morphinan alkaloid morphine, the anti-tussive alkaloid codeine, and the anti-microbial alkaloids berberine and sanguinarine. Due to the importance of these pharmaceutically useful alkaloids, their biosynthetic pathways have been well investigated, and several of them have been completely clarified at the enzyme level (1–3). It is now known that many isoquinoline alkaloids share a common biosynthetic pathway from L-tyrosine to the key intermediate (S)-reticuline. (S)-Reticuline is a central precursor of various types of isoquinoline alkaloids such as morphinans, aporphines, pavines, protoberberines, protopines, and benzophenanthridines (1, 4). Although the molecular origin of this chemical diversity has not yet been clarified, recent studies have shown that many of their oxidative steps are catalyzed by cytochrome P450s (P450)² (Fig. 1) (1–3, 5–11). Members of the P450 family are found in a very large number of species, especially in the plant kingdom (246 and 356 species in *Arabidopsis thaliana* and *Oryza sativa* in contrast to 57 and 84 species in human and *Drosophila melanogaster*) (12, 13), and many of them have been shown to be involved in plant secondary metabolism (14, 15).

In the biosyntheses of isoquinoline alkaloids, P450-mediated hydroxylation, methylenedioxy bridge formation, and phenol-coupling reactions have been reported. Although members of the CYP80B subfamily catalyze hydroxylation from (S)-N-methylcoclaurine to (S)-3′-hydroxy-N-methylcoclaurine in (S)-reticuline biosynthesis (5), other P450 reactions, including methylenedioxy bridge formation and phenol-coupling reactions, are involved in the biosynthesis of rather specific isoquinoline alkaloids (6–11). In the biosynthesis of berberine (protoberberine type) and macarpine (benzophenanthridine type), three methylenedioxy bridge-forming reactions, *i.e.* canadine synthase, cheilanthifoline synthase, and stylopine synthase reactions, have been reported (3). Methylenedioxy bridge formation is the cyclization of an ortho-methoxyphenol moiety, and is commonly found in many secondary metabolites, including lignans. Our recent research identified canadine synthase cDNA from *Coptis japonica* (CYP719A1) (6) and stylopine synthase cDNAs from *Eschscholzia californica* (CYP719A2 and CYP719A3) (8). Currently, P450 species that catalyze methylenedioxy bridge-forming reactions are rather rare: only the

* This research was supported in part by a grant-in-aid from the Ministry of Education, Culture, Sports, Science and Technology of Japan (to F. S.) and by a fellowship from the Japan Society for the Promotion of Science (to N.I.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The on-line version of this article (available at http://www.jbc.org) contains supplemental Figs. S1 and S2.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AB288053.
* To whom correspondence should be addressed. Tel.: 81-75-753-6381; Fax: 81-75-753-6398; E-mail: fsato@lif.kyoto-u.ac.jp.

² The abbreviations used are: P450, cytochrome P450; EST, expressed sequence tag; RACE, rapid amplification of cDNA ends; HPLC, high performance liquid chromatography; LC-MS, liquid chromatography-mass spectroscopy; CNMT, S-adenosyl-L-methionine:coclaurine N-methyltransferase; CjCNMT, CNMT of *C. japonica*; BBE, berberine bridge enzyme; MS/MS, tandem mass spectrometry.

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

---

**L-Tyrosine**


     H3CO
      |
HO —— C —— NH
      |       \
      H       OH
          (S)-Coclaurine


**CNMT**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OH
          (S)-N-Methylcoclaurine


**CYP80B**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OH
          (S)-3′-Hydroxy-
          N-methylcoclaurine


**CYP80G2**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OCH3
          (S)-Reticuline


**Salutaridine synthase**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       O
          Salutaridine


**Morphine**


     HO
      |
O —— C —— NCH3
      |       \
      H       HO
          Morphine


**CYP80A1**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OH
          (R)-N-Methylcoclaurine


**Berbamunine**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OH —— O —— CH2 —— OH
                          |
                          H3CO
                              |
                              NCH3
                                  |
                                  H
                                      Berbamunine


**CYP80G2**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OCH3
          (S)-Reticuline


**BBE**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       OCH3
          (S)-Scoulerine


**Berberine**


     O
      |
O —— C —— N+
      |       \
      H       OCH3
          |
          OCH3
              Berberine


**CNMT?**


     H3CO
      |
HO —— C —— NCH3
      |       \
      H       H
          (S)-Corytuberine


**Magnoflorine**


     H3CO
      |
HO —— C —— N+ —— CH3
      |       \    |
      H       CH3  H3CO
          Magnoflorine


---

**FIGURE 1.** A schematic depiction of the biosynthetic pathways for a variety of structurally related isoquinoline alkaloids. (S)-Reticuline is considered to be a key branch-point intermediate that can be channeled to morphinan-type (morphine), or protoberberine- (berberine) and benzophenanthridine- (sanguinarine) type alkaloids by the activity of specific enzymes. Enzymes discussed further are noted in bold. CNMT, S-adenosyl-l-methionine:coclaurine N-methyltransferase; CYP80A1, berbamunine synthase from *B. stolonifera*; CYP80B, (S)-N-methylcoclaurine-3′-hydroxylase; BBE, berberine bridge enzyme. The dashed box indicates the biosynthetic pathway found in *C. japonica*. The dotted arrows indicate more than one conversion step.

CYP719A subfamily has been found in isoquinoline alkaloid biosynthesis and the CYP81Q subfamily has been found in sesamin biosynthesis (16).

Two types of P450-mediated phenol-coupling reactions have been reported in isoquinoline alkaloid biosynthesis. One is the intramolecular C–C phenol-coupling reaction, catalyzed by salutaridine synthase, from (R)-reticuline to produce salutaridine in morphine biosynthesis (9). The other is the intermolecular C–O phenol-coupling reaction, which is catalyzed by berbamunine synthase to make berbamunine (bis-benzylisoquinoline alkaloid) using (R)- and (S)-N-methylcoclaurine as substrates (10, 11). So far, only berbamunine synthase has been cloned from *Berberis stolonifera* and has been designated CYP80A1 (11).

Japanese goldthread *C. japonica* (Ranunculaceae) produces a large amount of berberine and moderate amounts of several other isoquinoline alkaloids (Fig. 2). Although our previous study showed that two P450 genes (*CYP80B2* and *CYP719A1*) were involved in berberine biosynthesis (6), there has been no other previous identification of P450 genes involved in isoquinoline alkaloid biosynthesis in *C. japonica*. Because the biosyntheses of magnoflorine and coptisine were suggested, based on their chemical structures, to require oxidative steps from their respective precursors, (S)-reticuline and (S)-scoulerine, we speculated that they include several P450 reactions. Based on this idea, to clone their biosynthetic genes, we decided to search for candidate P450 genes using 4032 expressed sequence tags (ESTs) prepared from cultured *C. japonica* cells. As a result, we found a novel P450 cDNA fragment, which was significantly similar to *B. stolonifera* CYP80A1 and isolated its full-length cDNA. This novel P450, designated CYP80G2 by the P450 nomenclature committee, was heterologously expressed in yeast to characterize its enzyme activity. Recombinant CYP80G2 showed intramolecular C–C phenol-coupling activity to convert (S)-reticuline to corytuberine.

This is the first report of the isolation of cDNA of a eukaryotic microsomal-bound P450, which catalyzes a C–C phenol-coupling reaction. A detailed substrate specificity analysis of CYP80G2 was conducted to obtain information about its reaction mechanism and substrate recognition. In addition, we discuss the role of the unique amino acid residue in the helix I region of CYP80G2 in its C–C phenol-coupling reaction. We also discuss the involvement of CYP80G2 in magnoflorine biosynthesis.

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

synthesis, because corytuberine has been proposed to be the precursor of magnoflorine (17, 18), an aporphine-type alkaloid produced by cultured *C. japonica* cells.

**EXPERIMENTAL PROCEDURES**

Plant Material—The original cultured cells were induced from rootlets of *C. japonica* Makino var. dissecta (Yatabe) Nakai. A cell line (156-1) that produces large amounts of alkaloids was established and subcultured as described previously (19). Ten-day-old cultured cells were harvested and used for the extraction of mRNA and alkaloids.

Metabolite Analysis of Cultured *C. japonica* Cells—Cells were collected from a 10-day-old cell culture and extracted with MeOH, and an aliquot of the MeOH extract was analyzed directly by high performance liquid chromatography (HPLC). Reversed-phase HPLC was performed with a Shimadzu LC-10A system: column, TSKgel ODS-80TM (4.6 × 250 mm; Tosoh); solvent system, acetonitrile/H₂O/acetic acid (30:69:1); flow rate, 0.8 ml/min; detection, absorbance measurement at 280 nm with an SPD6A photodiode array detector.

Chemicals—(S)-Reticuline was a gift from Dr. P.J. Facchini of the University of Calgary. (S)-N-Methylcoclaurine was a gift from Dr. Y. Sugimoto of Kobe University. (S)-Coclaurine was a gift from Dr. N. Nagakura of Kobe Pharmaceutical University. Magnoflorine was a gift from Dr. R. Nishida of Kyoto University. (R,S)-Reticuline, (R,S)-norreticuline, (R,S)-6-O-methylnorlaudanosoline, and (R,S)-norlaudanosine were gifts from Mitsui Chemicals, Inc., Japan. (R,S)-Laudanosine, (R,S)-lau-danosoline, and (R,S)-norlaudanosoline were purchased from Sigma-Aldrich, Inc. (R,S)-Norlaudanine and (R,S)-norps-eudocodamine were prepared as described previously (20). (R,S)-Orientaline, (R,S)-codamine, (R,S)-6-O-methyl Laudanosine, (R,S)-4′-O-methyl Laudanosine, and (R,S)-pseudocodamine were prepared as described previously (21). Ketoconazole was purchased from Wako Pure Chemical Industries, Ltd., Japan.

Construction and Sequencing of a cDNA Library of *C. japonica*—A cDNA library of cultured *C. japonica* cells was constructed as described previously (22). Sequencing of the cDNA library was performed for ~4032 clones, which included redundant clones.³ The obtained ESTs were annotated using a BLAST search (blastx, available at www.ncbi.nlm.nih.gov/BLAST/).

5′-Rapid Amplification of cDNA Ends—5′-Rapid amplification of cDNA ends (RACE) was performed using a Gene Racer kit (Invitrogen) following the manufacturer’s instructions. A total RNA sample (3 μg) from 7-day-old cultured cells was used. A gene-specific primer, 5′-GSP (5′-AAGCCATGAC-CGTGGGTTGAGTACC-3′), was designed. The sequence of the universal primer for 5′-RACE was given in the user manual for the kit. The resultant PCR products at ~350 bp were sub-cloned into pT7Blue T-vector (Novagen), and their nucleotide sequences were determined completely.

Alignment Analysis—The predicted protein sequences were aligned using ClustalW (23, 24) and Boxshade.

Construction of Yeast Expression Vectors—The coexpression vector pGYR for P450 and yeast NADPH-P450 reductase was provided by Dr. Y. Yabusaki (Sumitomo Chemical Co., Ltd.). This vector contained glyceraldehyde-3-phosphate dehydrogenase promoter and terminator (25). The cloning site of pGYR was further modified to contain an SpeI site to construct pGYR-SpeI.

Full-length CYP80G2 cDNA was amplified by PCR using single strand cDNAs synthesized from 1.3 μg of total RNA of cultured *C. japonica* cells with oligo(dT) primer and SuperScript III RNase H-reverse transcriptase (Invitrogen). The following primers were designed to introduce an SpeI site (ACTAGT, underlined): forward primer (5′-ACTAGTTTCAGAACCAAGGATAGAGATTCAAATGG-3′) and reverse primer (5′-ACTAGTAAAACGTGAAATTTCTTATTGCCGCAAC-3′). PCR products were first subcloned into pT7Blue T-vector, and their nucleotide sequences were confirmed and then digested with SpeI to produce CYP80G2 coding fragments. The coding fragments were ligated into the SpeI site of pGYR-SpeI to generate yeast expression vector, pGS-CYP80G2.

Heterologous Expression of CYP80G2 in Yeast—The expression plasmid for CYP80G2 (pGS-CYP80G2) was introduced into yeast strain AH22 (26) by the LiCl method (27). These recombinant yeast cells were cultivated in concentrated SD medium (5.4% yeast nitrogen base without amino acids, 8% glucose, and 160 mg/l histidine) at 30 °C, 220 rpm (28). Yeast microsomal fractions were prepared as described previously (6) and suspended in a buffer (50 mM HEPES/NaOH (pH 7.6)) for the enzyme assay. Protein concentration was determined according to Bradford (29) with bovine serum albumin as a standard.

Large Scale Preparation of Reaction Product of CYP80G2 and LC NMR Analysis—To determine the structure of the CYP80G2 reaction product, we converted (R,S)-reticuline to its corresponding product using CYP80G2-expressing yeast cells *in vivo*. CYP80G2-expressing yeast cells, grown to the logarithmic phase in concentrated SD medium at 30 °C, 220 rpm, were harvested and resuspended in 50 mM HEPES/NaOH buffer (pH 7.6) containing (R,S)-reticuline at 100 μM, and then incubated for over 30 h, which resulted in the moderate production of CYP80G2 product. The use of 50 mM HEPES/NaOH buffer (pH 7.6) was essential for this conversion, because the low pH environment, which was due to incubation in concentrated SD medium, interfered with the uptake of (R,S)-reticuline by yeast and resulted in no conversion. Because the reaction product was released into the incubation buffer, it was collected from the buffer using Sep-Pak® Plus C18 cartridges (Waters).

The CYP80G2 reaction product was analyzed by LC NMR (¹H NMR) as described previously (30), and its ¹H NMR spectrum was compared with that of authentic corytuberine (described below). For LC NMR analysis of the CYP80G2 reaction product, a TSKgel ODS-80TM column (4.6 × 250 mm, Tosoh) was used.

Preparation of Authentic Corytuberine—Authentic corytuberine was prepared by the acid-catalyzed ether cleavage of (+)-isocorydine (purchased from Sigma-Aldrich, Inc.) as follows. A solution of (+)-isocorydine in 47% HBr was refluxed for 10 min and evaporated *in vacuo* to give a crystalline mixture and then separated by preparative HPLC with the following system: column, COSMOSIL 5C₁₈-AR-II (20 × 250 mm, Nacalai Tesque, Inc.); solvent system, 0.1 M NH₄OAc (0.05% trifluoroacetic acid)/acetonitrile (0.05% trifluoroacetic acid).

³ Y. Kokabu, N. Kato, E. Dubouzet, J. Dubouzet, and F. Sato, unpublished data.

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

quantitatively by reversed-phase HPLC with a Shimadzu LC-10A system: column, TSKgel ODS-80TM (4.6 × 250mm, Tosoh); solvent system, 0.1 M NH₄OAc (0.05% trifluoroacetic acid)/acetonitrile (0.05% trifluoroacetic acid) (70:30); flow rate, 0.8 ml/min; detection, absorbance measurement at 312 nm with an SPD6A photodiode array detector.

Product formation and substrate specificity were analyzed by LC-MS (LCMS-2010, Shimadzu) with the same conditions as in HPLC analysis except for the solvent system (acetonitrile/H₂O/ acetic acid (10~25:89~74:1)) and the flow rate (0.5 ml/min). When conversion rates were determined in the substrate specificity assay, the same solvent system and flow rate as in the LC-MS analysis were used in HPLC analysis.

Stereoselectivity Assay of CYP80G2—The stereoselectivity of the CYP80G2 reaction was analyzed using normal-phase HPLC with the following conditions: column, CHIRALCEL OD-H (4.6 × 250 mm, Daicel Chemical Industries, Ltd., Japan); solvent system, hexane/2-propanol/diethylamine (80:20:0.1); flow rate, 0.8 ml/min; detection, absorbance measurement at 280 nm with an SPD6A photodiode array detector. HPLC samples were prepared by extracting the reaction mixture with ethyl acetate after an enzyme reaction for 30 min, and the ethyl acetate layer was directly subjected to normal-phase HPLC analysis.

Determination of Kinetic Parameters—To determine the kinetic parameters of CYP80G2, the amount of corytuberine produced was estimated by the calibration curve of corytuberine (picomoles versus peak area) at 312 nm, which was drawn using the calibration curve of (R,S)-reticuline (picomoles versus peak area) at 283 nm, the absorption ratio of reticuline and corytuberine at 283 nm (1.00:0.93), and the absorption ratio of corytuberine between 283 and 312 nm (1.00:1.10). The data were fitted to the Michaelis-Menten equation by using a nonlinear least-square iterative method using KaleidaGraph (Synergy Software, Reading, PA). Three sets of kinetic parameters were obtained from three independent experiments and then simply averaged to yield the final estimates. The final estimates are shown with the standard errors for the three sets.

**RESULTS**

Isolation of Cytochrome P450 cDNAs—Cultured *C. japonica* cells produce several kinds of isoquinoline alkaloids (Fig. 2). Because we speculated that magnoflorine and coptisine biosyntheses would include some of the oxidative reactions catalyzed by P450s, we searched for candidates of their biosynthetic P450 genes using 4032 redundant ESTs prepared from cultured *C. japonica* cells. A BLAST search (blastx) (www.ncbi.nlm.nih.

CYP80G2, an Intramolecular C-C Phenol-coupling Enzyme

| CYP80B1 | 1: .GTSTVALIAVIISSTILYLFGGSGHKNLPPGPKPWPIVGNLLQLGEKP....HAQFAEL :55 |
| --- | --- |
| CYP80B2 | 1: MEVLSTIAIVSFSFLLFFIRDSRPNLPPGPRPSPIVGNLLQLGDKP....HAEFAKL :56 |
| CYP80A1 | 1: .MDYIVGFVSTSLVALLYFLFKPKHTNLPPSPPAWPIVGHLPDLLSKNSPFLDYMSNIT :59 |
| CYP80G2 | 1: .MDLQIALFSLIPVILVFILLKPKYKNLPPGPHPWPLIGNLPIILFTNTEVPLHITLANM :59 |

we expected that this novel P450 species, which was temporarily designated CYP80A1-like, may also be responsible for isoquinoline alkaloid biosynthesis. Thus, its full-length cDNA was isolated.

Nucleotide Sequences and Predicted Amino Acid Sequences—Among the four EST clones of CYP80A1-like, the longest had 1517 nucleotides and still lacked the 5'-region of full-length P450cDNA. To isolate the full-length of CYP80A1-like, 5'-RACE was conducted. Full-length CYP80A1-like cDNA, which was re-amplified from a cDNA library, contained 1700 nucleotides with an open reading frame of 486 amino acids (DDBJ/GenBank™/EMBL accession number AB288053) (Fig. 3). Although a blastx search showed that the amino acid sequence of CYP80A1-like was most similar to that of *B. stolonifera* CYP80A1 (53.1% identity), CYP80A1-like was classified into the CYP80G subfamily and designated CYP80G2 by the P450 nomenclature committee (c/o Dr. D.R. Nelson, University of Tennessee, Memphis, TN). The nomenclature committee informed us that CYP80G2 had quite high amino acid sequence similarity (85% identity) with CYP80G1, which has been isolated from ESTs of *Aquilegia formosa × Aquilegia pubescens* (Ranunculaceae) (GenBank™ accession numbers for its ESTs are DR912881.1, DR923264, and DT766891, and its amino acid sequence is available from Dr. D.R. Nelson). Although this high degree of homology indicated that CYP80G2 is an ortholog of CYP80G1, the function of CYP80G1 had not yet been clarified, and therefore we decided to characterize CYP80G2 (CYP80A1-like).

Structural analysis showed that CYP80G2 had conserved eukaryotic P450 regions: a helix K region, an aromatic region, and a heme-binding region at the C-terminal end (Fig. 3). In addition, its N-terminal region contained hydrophobic domains corresponding to the membrane anchor sequences of microsomal P450 species, suggesting that CYP80G2 is localized in the endoplasmic reticulum. Notably, CYP80G2 had a unique amino acid substitution in a consensus amino acid sequence ((A/G)GX(D/E)T(T/S)) of the helix I region, which should be involved in interaction with the substrate and iron-bound oxy-

FIGURE 3. Amino acid sequence alignment of P450s in the CYP80 family. Boxes indicate conserved regions of eukaryotic P450, which include a helix-K region, aromatic region, and heme-binding region. The asterisk indicates the position of Pro-290 of CYP80G2 and CYP80A1, which replaces the conserved alanine/glycine. CYP80B1, (S)-N-methylcoclaurine-3′-hydroxylase from California poppy, *E. californica* (GenBank™ accession number AF014800); CYP80B2, (S)-N-methylcoclaurine-3′-hydroxylase from *C. japonica* (GenBank™ accession number AB025030); and CYP80A1, berbamunine synthase from *B. stolonifera* (GenBank™ accession number U09610).

gov/BLAST/) showed that these ESTs included P450 clones corresponding to CYP80B2, CYP719A1, and uncharacterized P450s (3, 5, and 17 clones, respectively). Because this EST library had multiple clones of berberine biosynthetic genes (CYP80B2 and CYP719A1), we expected that it would be useful for isolating a novel P450 gene involved in magnoflorine and coptisine biosyntheses. Further analysis of the uncharacterized P450s revealed that 17 clones were classified into 7 species, one of which, consisting of four clones, showed high homology to *B. stolonifera* CYP80A1 or *E. californica* CYP80B1 (the E values ranged from 1e-32 to 3e-50). Because both CYP80A1 and CYP80B1 were involved in isoquinoline alkaloid biosynthesis,

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

**FIGURE 4.** LC-MS analysis of the reaction product generated by microsomal fractions of vector control (A) and CYP80G2-expressing (B) yeasts with (S)-reticuline. The reaction mixtures were incubated at 30 °C for 30 min. The reaction was terminated by the addition of trichloroacetic acid at a final concentration of 2%. After protein precipitation, aliquots were analyzed directly by LC-MS. Total ion chromatographs (TIC) as well as selected ion monitoring (328 m/z for reaction product and 330 m/z for reticuline) are shown.

gen (32); i.e. CYP80G2 had a proline instead of alanine/glycine (underlined) at the position corresponding to Gly-248 in P450cam (33). The same substitution was also found in CYP80A1 (Fig. 3).

**Heterologous Expression of CYP80G2 in Yeast and Its Activity—** To determine the function of CYP80G2, yeast expression plasmid for CYP80G2 was constructed and introduced into yeast AH22. Because CYP80G2 had a putative endoplasmic reticulum-localizing signal, microsomal fractions were prepared from recombinant yeast cells, and its enzymatic activity was determined using LC-MS analysis.

Because CYP80G2 showed high homology to CYP80A1 and CYP80B1, whose natural substrates are N-methylcoclaurine derivatives, we expected that it may use N-methylcoclaurine derivatives as a substrate (Fig. 1). Based on this idea, we examined the reactivity of CYP80G2 toward (S)-coclaurine, (S)-N-methylcoclaurine, and (S)-reticuline.

LC-MS analysis showed that microsomal fractions of CYP80G2-expressing yeast could react with all three of the above substrates, but most strongly with (S)-reticuline. CYP80G2 converted (S)-reticuline (m/z 330) to a large amount of major product (m/z 328) (Fig. 4) and a minor fraction of by-product (m/z 328) (data not shown). The by-product, which showed a longer retention time than the major product, had much less peak intensities (<8%) than the major products in several LC-MS analyses. Further analysis showed that the by-product should be dehydroteticuline, the oxidative product of reticuline (data not shown).

Because the major product had a shorter retention time than (S)-reticuline (Fig. 4), and its absorption maxima, as measured

**FIGURE 5.** Comparison of the ${}^{1}$H NMR spectra of CYP80G2 reaction product (A) and authentic corytuberine (B). Stopped-flow LC NMR analysis was performed to identify CYP80G2 reaction product with (S)-reticuline. Authentic corytuberine was synthesized, and its structure was confirmed based on its ${}^{1}$H NMR and nuclear Overhauser effect spectroscopy spectra. Both of these ${}^{1}$H NMR spectra showed three aromatic protons at δ 6.87 (1H, d, $J = 8$ Hz, H-9), 6.75 (1H, s, H-3), and 6.74 (1H, d, $J = 8$ Hz, H-8) (peak 1), two methoxy groups at δ 3.78 (3H, s, $C_{2}-OCH_{3}$) and 3.77 (3H, s, $C_{10}-OCH_{3}$) (peak 2), and one N-methyl group at δ 3.02 (3H, s) (peak 3). *, the signal belongs to CH${}_{3}$OH.

by photodiode array analysis of HPLC, was similar to that of authentic magnoflorine (data not shown), we speculated that it would be an aporphine-type alkaloid. It might be corytuberine or isoboldine, which are produced by ortho-ortho or ortho-para C–C phenol coupling of reticuline, respectively.

To identify the major product of CYP80G2, LC NMR analysis was performed (Fig. 5). First, a large-scale preparation of CYP80G2 product from reticuline was conducted using intact CYP80G2-expressing yeast cells. In this preparation, >10 μg of the product was obtained from reticuline in yeast cells. Next, authentic corytuberine was prepared from (+)-isocorydine by acid-catalyzed ether cleavage and separation using preparative HPLC, after the structure was confirmed by LC NMR analysis (${}^{1}$H NMR and nuclear Overhauser effect spectroscopy spectra) (supplemental Fig. S1). Finally, we measured the ${}^{1}$H NMR spectrum of CYP80G2 product by stopped-flow LC NMR analysis, compared it with that of authentic corytuberine, and found that they had identical spectra. They showed identical signals derived from three aromatic protons at δ 6.87 (1H, d, $J = 8$ Hz, H-9), 6.75 (1H, s, H-3), and 6.74 (1H, d, $J = 8$ Hz, H-8), two

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

TABLE 1  
NADPH or oxygen dependency of C–C phenol-coupling activity of CYP80G2  

The reaction was carried out under standard assay conditions with (R,S)-reticuline as the substrate, after the removal of NADPH or oxygen with glucose/glucose oxidase.

| Addition                                      | Relative activity (%) |
|-----------------------------------------------|-----------------------|
| None                                          | 100                   |
| –NADPH                                        | 0                     |
| 40 mM glucose + 5 units of glucose oxidase + 10 units of catalase | 10                    |
| 40 mM glucose + boiled glucose oxidase + 10 units of catalase | 104                   |

methoxy groups at δ 3.78 (3H, s, C₂-OCH₃) and 3.77 (3H, s, C₁₀-OCH₃), and one N-methyl group at δ 3.02 (3H, s) (Fig. 5). Further LC-MS/MS analysis confirmed the identity of the CYP80G2 reaction product and authentic corytuberine; i.e., they had the same production patterns (*m/z* 297, 282, 267, 265, 237, 222, 219, 207, and 191). Thus, we concluded that the CYP80G2 product was corytuberine. Corytuberine is an aporphine-type isoquinoline alkaloid, which has a C–C bond between C8 and C2′ of reticuline. This result indicated that CYP80G2 was an intramolecular C–C phenol-coupling enzyme.

Because CYP80G2 catalyzed the C–C phenol-coupling reaction from reticuline to corytuberine, the proposed precursor of magnoflorine (17, 18), this result suggested that CYP80G2 is involved in magnoflorine biosynthesis. Because the enzymological properties of CYP80G2 were not clear, further analysis was performed using recombinant CYP80G2.

**Enzymatic Characterization of CYP80G2—** Because CYP80G2 catalyzed a unique C–C phenol-coupling reaction, i.e., oxidation of the substrate without the concomitant incorporation of oxygen, we examined the P450 nature of CYP80G2. A reduced CO-difference spectrum of CYP80G2 showed a characteristic peak at 448 nm, like a typical P450 (data not shown), and the content of CYP80G2 was calculated to be 25.2 pmol/mg of protein of microsomal fraction. CYP80G2 activity also depended on NADPH and oxygen: the absence of NADPH or removal of O₂ by the glucose/glucose oxidase/catalase system (7) clearly reduced the CYP80G2 activity for (R,S)-reticuline (Table 1). A synthetic fungicide, ketoconazole, which is a typical P450 inhibitor that interacts with the prosthetic heme group (34), also inhibited C–C phenol-coupling activity, and its half-inhibition concentration (*IC₅₀*) was estimated to be 0.7 μM.

Further, we examined the stereoselectivity of the CYP80G2 reaction using (R,S)-reticuline as a substrate (Fig. 6). HPLC analysis with a chiral column showed that CYP80G2 used (S)-reticuline, but not (R)-reticuline, indicating that CYP80G2 was (S)-form specific.

**Substrate Specificity of CYP80G2—** Because CYP80G2 could use substrates other than reticuline as mentioned above, we carefully examined its substrate specificity. The intramolecular C–C phenol-coupling activity of CYP80G2 toward 16 benzylisoquinoline alkaloids was examined (Table 2).

First, we conducted LC-MS analysis to monitor product formation. The enzyme reactions were carried out with a 10-fold excess of the standard enzyme preparation to efficiently detect

FIGURE 6. Analysis of the stereoselectivity of CYP80G2 reaction toward (R,S)-reticuline. Unconsumed reticuline was monitored using normal-phase HPLC with a chiral column after vector control (A) and CYP80G2 (B) reactions. They are shown with authentic (S)-reticuline (C) to determine which isomer was consumed by CYP80G2. The enzyme reaction was carried out at 30 °C for 30 min with a 10-fold excess of the standard enzyme preparation described under “Experimental Procedures.” The reaction mixture was then extracted with ethyl acetate, and an aliquot of the ethyl acetate extract was analyzed directly by normal-phase HPLC at 280 nm. The peak areas of remaining (R)-reticuline were the same (~100%) between samples after vector control (A) and CYP80G2 (B) reactions, which were carried out under the same assay conditions.

C–C phenol-coupling activity. As a result, CYP80G2 converted (R,S)-norreticuline, (R,S)-orientaline, (S)-N-methylcoclaurine, and (S)-coclaurine to their corresponding products with a reduction of 2-*m/z*. (R,S)-Codamine, however, showed two products with a reduction of 14-*m/z* and 16-*m/z*. The major product with a reduction of 14-*m/z* was identified as orientaline by direct comparison with authentic (R,S)-orientaline, whereas the minor product with a reduction of 16-*m/z* was identical to the (R,S)-orientaline-derived C–C phenol-coupling product. This result indicated that CYP80G2 catalyzed the 4′-O-demethylation of codamine to produce orientaline, and subsequent C–C phenol coupling of orientaline. CYP80G2 did not react with any of the other 11 compounds ((R,S)-6-O-methylau-

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

TABLE 2  
Substrate specificity for CYP80G2  
n.d.= not detected.

| Substrate                          | $R_{1}$   | $R_{2}$   | $R_{3}$   | $R_{4}$   | $R_{5}$   | Relative activity |
|-------------------------------------|-----------|-----------|-----------|-----------|-----------|-------------------|
| $(R,S)$-Reticuline                 | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{CH}_{3}$ | $100$         |
| $(R,S)$-Norreticuline              | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{H}$     | $19^{a}$      |
| $(R,S)$-Orientaline                | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{CH}_{3}$ | $2.7^{b}$     |
| $(R,S)$-Codamine                   | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{CH}_{3}$ | $12^{c}$      |
| $(S)$-$N$-Methylcoclaurine         | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{H}$ | $\mathrm{OH}$ | $\mathrm{CH}_{3}$ | $2^{b}$       |
| $(S)$-Coclaurine                   | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{H}$ | $\mathrm{OH}$ | $\mathrm{H}$     | $<0.1$        |
| $(R,S)$-$6-O$-Methyl Laudanosoline | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-$6-O$-Methylnorlaudanosoline | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{H}$     | n.d.          |
| $(R,S)$-Laudanine                  | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-Norlaudanine               | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{H}$     | n.d.          |
| $(R,S)$-$4^{\prime}-O$-Methyl Laudanosoline | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OCH}_{3}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-Pseudocodamine             | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-Norpseudocodamine          | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OH}$ | $\mathrm{H}$     | n.d.          |
| $(R,S)$-Laudanosine                | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-Norlaudanosine             | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{OCH}_{3}$ | $\mathrm{H}$     | n.d.          |
| $(R,S)$-Laudanosoline              | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{CH}_{3}$ | n.d.          |
| $(R,S)$-Norlaudanosoline           | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{OH}$ | $\mathrm{H}$     | n.d.          |

$a$ $(R,S)$-Norreticuline-derived products consisted of one major product and one minor product. While the minor product showed the same $m/z$ (314) and a shorter retention time than the major product (data not shown), its amount was only $\sim 2-3\%$ of that of the major product based on a comparison of their ionized peak intensities with LC-MS. The conversion rate was calculated only for the major product.

$b$ These products might have different structures than aporphines based on their UV spectra. In particular, they might be proaporphines, which are made by ortho-para oxidative coupling of $(R,S)$-orientaline or $(S)$-$N$-methylcoclaurine.

$c$ $(R,S)$-Codamine-derived products consisted of one major product and one minor product. The major product was $(R,S)$-orientaline, $4^{\prime}$-$O$-demethylation product of $(R,S)$-codamine, and the minor one was proposed to be the same compound as $(R,S)$-orientaline-derived phenol coupling product. The conversion rate was calculated for the sum of major and minor products.

danosoline, $(R,S)$-$6-O$-methylnorlaudanosoline, $(R,S)$-laudanine, $(R,S)$-norlaudanine, $(R,S)$-$4^{\prime}-O$-methyl Laudanosoline, $(R,S)$-pseudocodamine, $(R,S)$-norpseudocodamine, $(R,S)$-laudanosine, $(R,S)$-norlaudanosine, $(R,S)$-laudanosoline, and $(R,S)$-norlaudanosoline).

Next, we determined the conversion rates of $(R,S)$-norreticuline, $(R,S)$-orientaline, $(S)$-$N$-methylcoclaurine, and $(R,S)$-codamine under the standard enzyme assay condition using HPLC, although $(S)$-coclaurine-derived product was not detectable under this condition. The product peak areas

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

at their absorption maxima were compared with that of  
(R,S)-reticuline-derived product (corytuberine) as 100% to  
calculate the conversion rates. Photodiode array analysis  
showed that the products derived from (R,S)-reticuline,  
(R,S)-norreticuline, (R,S)-orientaline, (S)-N-methylcoclau-  
rine, and (R,S)-codamine had absorption maxima at 266 and  
302 nm, 266 and 302 nm, 281 nm, 282 nm, and 281 nm (for  
both the 4′-O-demethylated and its C–C phenol-coupling  
products), respectively, in an acetonitrile/H₂O/acetic acid  
solvent system. Because (R,S)-norreticuline-derived product  
has the same absorption maximum as corytuberine, it was  
suggested to have an aporphine-type structure. Its conver-  
sion rate was calculated to be 19% from their peak areas at  
302 nm. On the other hand, because (R,S)-orientaline- and  
(S)-N-methylcoclaurine-derived products showed unique  
absorption maxima (281 or 282 nm), they were suggested to  
have structures different from an aporphine type, such as a  
proaporphine type, made by the ortho-para oxidative cou-  
pling of (R,S)-orientaline and (S)-N-methylcoclaurine. Their  
conversion rates were calculated from their maximal peak  
areas at 281 or 282 nm and the peak area of corytuberine at  
266 nm. The calculated values were 2.7%, 2%, and 12% for  
(R,S)-orientaline-, (S)-N-methylcoclaurine-, and the sum of  
(R,S)-codamine-derived products, respectively.

Substrate Affinity of CYP80G2—Because CYP80G2 showed  
high substrate specificity for reticuline, it was suggested to be  
the natural substrate of CYP80G2 in *C. japonica*. The substrate  
affinity of CYP80G2 was then determined using HPLC with  
(R,S)-reticuline as a substrate. CYP80G2 followed Michaelis-  
Menten-type reaction kinetics when the substrate concentra-  
tion was varied. The kinetic parameters, $K_m$ and $V_{max}$, of  
CYP80G2 were estimated to be 34.1 ± 1.9 μM and 338 ± 10  
pmol of product/min/pmol of P450, respectively.

Involvement of CYP80G2 in Magnoflorine Biosynthesis—  
Although (S)-corytuberine, produced from (S)-reticuline by  
CYP80G2, has been proposed to be a precursor of magnoflo-  
rine (17, 18), no experimental evidence has been provided,  
either *in vivo* or *in vitro*. Thus, we examined whether (S)-  
corytuberine could be converted to magnoflorine using het-  
erologously expressed S-adenosyl-L-methionine:coclaurine  
N-methyltransferase (CNMT) of *C. japonica* (22). LC-MS  
analysis showed that (S)-corytuberine (*m/z* 328) was con-  
verted to magnoflorine (*m/z* 342) by the crude enzyme prep-  
aration of recombinant *C. japonica* CNMT (supplemental  
Fig. S2). On the other hand, presumably N-methylated (S)-  
reticuline, converted from (S)-reticuline by CjCNMT, was  
not converted by CYP80G2 to magnoflorine (data not  
shown). These results suggested that magnoflorine is biosyn-  
thesized by CYP80G2 and CjCNMT via (S)-corytuberine as  
an intermediate *in vivo*. CjCNMT, however, catalyzes the  
crucial N-methylation step from (S)-coclaurine to (S)-N-  
methylcoclaurine in (S)-reticuline biosynthesis (Fig. 1). In  
addition, it can convert various types of isoquinoline alka-  
loids such as benzylisoquinolines, aporphines, and protober-  
berines (Ref. 22 and unpublished data). Because tetrahydro-  
protoberberine cis-N-methyltransferase recently isolated  
from opium poppy showed rather high substrate specificity  
(35), it is possible that some other enzyme more specific than  

CjCNMT might be involved in magnoflorine biosynthesis. Fur-  
ther careful investigations will be needed to conclude that CjC-  
NMT is involved in magnoflorine biosynthesis *in vivo*.

DISCUSSION

We isolated a novel cytochrome P450 cDNA (CYP80G2)  
from an EST library of cultured *C. japonica* cells. Because the  
biological function of *Aquilegia* CYP80G1, an expected  
ortholog of CYP80G2, had not yet been characterized, we  
attempted to clarify the function of *C. japonica* CYP80G2. A  
structural analysis showed that CYP80G2 had significant  
amino acid sequence similarity to *B. stolonifera* CYP80A1  
(53.1% identity), as well as the same unique amino acid substi-  
tution in the helix I region as CYP80A1 (Fig. 3). Heterologous  
expression in yeast showed that CYP80G2 catalyzed the  
intramolecular C–C phenol-coupling reaction from (S)-reticu-  
line to (S)-corytuberine (Figs. 4, 5).

C–C phenol-coupling reactions, catalyzed by plant P450s,  
are also found in morphine and colchicine biosyntheses, *i.e.* in  
the conversion of (R)-reticuline to salutaridine (salutaridine  
synthase reaction) and in the conversion of autumnaline to  
isoandrocymbine (9, 36, 37). Morphine and colchicine are an  
analgesic drug used in the treatment of cancer and an estab-  
lished inhibitor of microtubule assembly, respectively (38).  
Despite the great importance of C–C phenol coupling in the  
production of such pharmaceutically useful alkaloids, our iden-  
tification of CYP80G2 as corytuberine synthase is the first suc-  
cessful cloning of a C–C phenol-coupling enzyme involved in  
alkaloid biosynthesis.

Several examples of C–C phenol-coupling P450s have  
been reported other than those from eukaryotic sources.  
Some bacterial P450 genes, which are responsible for C–C  
phenol-coupling reactions, have been described; *i.e.* Amyco-  
latopsis mediterranei OxyC involved in the biosynthesis of  
vancomycin-related glycopeptide (balhimycin) (39), a Strep-  
tomycetes coelicolor flaviolin oxidase (40), a Streptomyces gri-  
seus tetrahydroxynaphthalene oxidase (41), and Streptomy-  
ces longisporoflavus aryl-aryl coupling enzyme involved in  
staurosporine biosynthesis (42). Because all of these exam-  
ples are for bacterial P450, CYP80G2 is the first eukaryotic  
C–C phenol-coupling P450 for which the gene was cloned  
and characterized.

The reaction mechanism of CYP80G2 is best explained by a  
biradical process, which was proposed in a report on berbam-  
une synthase (CYP80A1) (10); *i.e.* its phenol-coupling reac-  
tion most likely takes place by two one-electron transfers with  
subsequent radical coupling (Fig. 7A). CYP80G2 would act as a  
catalyst for dehydrogenation reactions, such as peroxidases.  
The active species of the P450 reaction cycle were actually pro-  
posed to be reminiscent of those of peroxidases (compound I  
(oxoferrylporphyrin π-cation radical, FeᴵV(=O)por⁺•) and  
compound II (FeᴵV-OH)). Thus, CYP80G2 sequentially  
abstracts hydrogen atoms, probably with compound I and com-  
pound II, to give a substrate biradical.

Our detailed characterization of the substrate specificity  
of and the reaction catalyzed by CYP80G2 provided infor-  
mation about the reaction sequence. Thus, the 4′-O-dem-  
ethylation of codamine by CYP80G2 suggests that CYP80G2

CYP80G2, an Intramolecular C–C Phenol-coupling Enzyme

A

![Chemical Structure](https://i.imgur.com/1234567.png)

(S)-Reticuline

B

![Chemical Structure](https://i.imgur.com/7654321.png)

(R,S)-Codamine

FIGURE 7. Postulated reaction mechanism of CYP80G2. A, CYP80G2-mediated phenol-coupling reaction may proceed by two one-electron transfers with subsequent radical coupling. Based on the result of B, we speculated that the first hydrogen atom abstraction occurs at the 3′-hydroxyl group of the C-ring, followed by the second oxidation at the 7-hydroxyl group of the A-ring. B, postulated 4′-O-demethylation mechanism of codamine by CYP80G2. Demethylation reactions catalyzed by P450 are often seen in mammalian P450-mediated drug metabolism (48). The 4′-O-demethylation is postulated to proceed after hydroxylation of the 4′-methoxy group, followed by the release of formaldehyde.

first reacts with the C-ring, not the A-ring, of the substrate (Fig. 7B). This result is consistent with the reactions of CYP80A1 and CYP80B, both of which have high amino acid sequence homology to CYP80G2 and react at the C-rings of their substrates (5, 10, 11). The reaction product of codamine by CYP80G2 indicated that hydrogen abstraction would occur first at the 4′-methoxy group of the C-ring with oxoferrylporphyrin π-cation radical (compound I), followed by hydroxy radical rebound from the heme to the same position of the substrate, and the release of formaldehyde, to form demethylated product (Fig. 7B). Similarly, when CYP80G2 reacts with (S)-reticuline, the first hydrogen atom abstraction might occur with compound I from the 3′-hydroxy group of the C-ring to generate a phenoxy radical, followed by subsequent oxidation by the ferryl oxidant (compound II) at the 7-hydroxy group of the A-ring to generate a second radical. The reaction would be achieved by the coupling of biradical, i.e. bond formation between C8 and C2′ of (S)-reticuline, by CYP80G2, which likely has a conformation to arrange the A- and C-rings of the substrate in close proximity. However, we cannot exclude the possibility that the reaction starts with hydrogen abstraction from the 7-hydroxy group of the A-ring. Further investigation will be needed to clarify the exact reaction order.

Our analysis of substrate specificity also provided information about the substrate recognition/reaction of CYP80G2. CYP80G2 could use (R,S)-orientaline, (R,S)-codamine, (S)-N-methylcoclaurine, and (S)-coclaurine, which are variants of reticuline at the C-ring, indicating that it could accept functional groups with a variety of structures at the C-ring. On the

other hand, CYP80G2 did not use (R,S)-laudanine or (R,S)-4′-O-methyl Laudanosoline, whose functional groups are different from those of reticuline only at the A-ring. These results suggested that the A-ring might be more important than the C-ring for the substrate recognition and/or reaction mechanism of CYP80G2.

Several reports on the CYP80 family, including this study, have shown that they catalyze quite different reactions (Fig. 1); i.e., CYP80A1, CYP80B1, and CYP80G2 catalyze intermolecular C–O phenol coupling, hydroxylation, and intramolecular C–C phenol-coupling reactions, respectively (5, 10, 11). Because they have high sequence similarity as a whole, their reaction diversity may originate from rather slight differences in their primary structures. In fact, CYP80A1 and CYP80G2 have characteristic amino acid residues in the central region of helix I, whereas CYP80B1 follows a consensus amino acid sequence ((A/G)GX(D/E)T(T/S)) in this region (32). They had unique proline residues at the position of A/G (underlined), and CYP80A1 also had an asparagine residue at the position of T/S (double underlined). The central region of helix I is known to be involved in interaction with the substrate and iron-bound oxygen, and several P450s, which have an unusual amino acid in this region, catalyze unusual substrates and/or reactions (32). In isoquinoline alkaloid biosynthesis, members of the CYP719A family (methylenedioxy bridge-forming P450s) also have unique leucine and serine residues at the position of A/G (underlined) and conserved threonine (boldface), respectively (6, 8). Based on these facts, we speculated that the unique proline residue in CYP80G2 should play an important role in its C–C phenol-coupling reaction. However, our mutagenesis experiments showed that the change of this unique proline residue (Pro-290) to each of the conserved amino acids (Ala/ Gly) in the helix I region did not destroy its ability to convert (S)-reticuline to (S)-corytuberine (data not shown). This unexpected result indicated that the Pro-290 residue in helix I of CYP80G2 is not essential for its C–C phenol-coupling reaction, although it might play a role in other enzymatic properties, such as substrate affinity.

Our cultured *C. japonica* cells produce large amounts of berberine and moderate amounts of magnoflorine (Fig. 2). (S)-Reticuline, a common precursor, is located at a branch-point in their biosyntheses; i.e., it is oxidized to (S)-corytuberine by CYP80G2 in magnoflorine biosynthesis, or to (S)-scoulerine by berberine bridge enzyme (BBE) in berberine biosynthesis. Thus, the oxidation of (S)-reticuline might play a key role in regulating the metabolic flow of isoquinoline alkaloids in *C. japonica*. The $K_m$ value of CYP80G2 toward (R,S)-reticuline was estimated to be 34.1 μM and was probably ~17 μM toward (S)-reticuline, because CYP80G2 could only use (S)-reticuline, the amount of which was equal to (R)-isomer in (R,S)-reticuline (Fig. 6). On the other hand, the $K_m$ value of BBE from *E. californica* toward (S)-reticuline was reported to be 3 μM (43). Although the enzymological property of *C. japonica* BBE has not yet been characterized, if the BBEs of *E. californica* and *C. japonica* have similar properties, the $K_m$ difference between CYP80G2 and BBE toward (S)-reticuline would explain the ratio of berberine and magnoflorine in *C.*

japonica. However, detailed studies on their relative expression levels and localization will also be needed to understand the regulation of their biosyntheses in *C. japonica.*

Although the biosynthesis of major isoquinoline alkaloids, such as berberine, has been studied extensively (5, 6, 22, 43–46), the biosynthesis of minor ones has not been a subject of research. In this study, we used an EST library of cultured *C. japonica* cells and isolated a novel P450 gene, which is involved in magnoflorine biosynthesis. This success, as well as a previous gene isolation of S-adenosyl-L-methionine:columbamine O-methyltransferase in palmatine biosynthesis (47), suggests that EST libraries are useful for isolating a new biosynthetic gene of a little characterized biosynthetic pathway. Further screening of EST libraries may help us to identify novel biosynthetic genes of diverse isoquinoline alkaloids.

Acknowledgments—We thank Drs. P.J. Facchini, N. Nagakura, and R. Nishida, as well as Mitsui Chemicals, Inc., for generous gifts of the alkaloids. We thank Dr. Y. Sugimoto for the generous gift of the alkaloid and for discussing substrate specificity. We thank Dr. Y. Yabusaki for providing the expression vector, pGYR. We also thank Drs. R. Shinkyo and M. Mizutani of Kyoto University for technical assistance in constructing pGYR-SpeI and for the reduced CO-difference spectra measurement, respectively.

---

**REFERENCES**

1. Croteau, R., Kutchan, T. M., and Lewis, N. (2000) *Biochemistry and Molecular Biology of Plants* (Buchanan, B., Gruissem, W., and Jones, R., eds) pp. 1250–1318, American Society of Plant Physiologists, Rockville, MD

2. Zenk, M. H. (1994) *Pure & Appl. Chem.* **66**, 2023–2028

3. Kutchan, T. M. (1998) in *The Alkaloids—Chemistry and Biology*, vol. 50 (Cordell, G. A., ed) pp. 257–316, Academic Press, San Diego, CA

4. Preininge r, V. (1986) in *The Alkaloids—Chemistry and Pharmacology*, vol. 29 (Brossi, A., ed) pp. 1–98, Academic Press, San Diego, CA

5. Pauli, H. H., and Kutchan, T. M. (1998) *Plant J.* **13**, 793–801

6. Ikezawa, N., Tanaka, M., Nagayoshi, M., Shinkyo, R., Sakaki, T., Inouye, K., and Sato, F. (2003) *J. Biol. Chem.* **278**, 38557–38565

7. Bauer, W., and Zenk, M. H. (1991) *Phytochemistry* **30**, 2953–2961

8. Ikezawa, N., Iwasa, K., and Sato, F. (2007) *FEBS J.* **274**, 1019–1035

9. Gerardy, R., and Zenk, M. H. (1993) *Phytochemistry* **32**, 79–86

10. Stadler, R., and Zenk, M. H. (1993) *J. Biol. Chem.* **268**, 823–831

11. Kraus, P. F. X., and Kutchan, T. M. (1995) *Proc. Natl. Acad. Sci. U. S. A.* **92**, 2071–2075

12. Nelson, D. R., Schuler, M. A., Paquette, S. M., Werck-Reichhart, D., and Bak, S. (2004) *Plant Physiol.* **135**, 756–772

13. Nelson, D. R., Zeldin, D. C., Hoffman, S. M. G., Maltais, L. J., Wain, H. M., and Nebert, D. W. (2004) *Pharmacogenetics* **14**, 1–18

14. Chapple, C. (1998) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **49**, 311–343

15. Werck-Reichhart, D., Bak, S., and Paquette, S. (2002) in *The Arabidopsis Book* (Somerville, C. R., and Meyerowitz, E. M., eds) American Society of Plant Biologists, Rockville, MD

16. Ono, E., Nakai, M., Fukui, Y., Tomimori, N., Fukuchi-Mizutani, M., Saito, M., Satake, H., Tanaka, T., Katsuta, M., Umezawa, T., and Tanaka, Y. (2006) *Proc. Natl. Acad. Sci. U. S. A.* **103**, 10116–10121

17. Bhakuni, D. S., Jain, S., and Singh, R. S. (1980) *Tetrahedron* **36**, 2525–2528

18. Milanowski, D. J., Winter, R. E. K., Elvin-Lewis, M. P. F., and Lewis, W. H. (2002) *J. Nat. Prod.* **65**, 814–819

19. Sato, F., and Yamada, Y. (1984) *Phytochemistry* **23**, 281–285

20. Iwas a, K., Cui, W., Sugiura, M., Takeuchi, A., Moriyasu, M., and Takeda, K. (2005) *J. Nat. Prod.* **68**, 992–1000

21. Cui, W., Iwas a, K., Tokuda, H., Kashihara, A., Mitani, Y., Hasegawa, T.,

Nishiyama, Y., Moriyasu, M., Nishino, H., Hanaoka, M., Mukai, C., and Takeda, K. (2006) Phytochemistry 67, 70–79

22. Choi, K. B., Morishige, T., Shitan, N., Yazaki, K., and Sato, F. (2002) J. Biol. Chem. 277, 830–835

23. Thompson, J. D., Higgins, D. G., and Gibson, T. J. (1994) Nucleic Acids Res. 22, 4673–4680

24. Higgins, D. G., Thompson, J. D., and Gibson, T. J. (1996) Methods Enzymol. 266, 383–402

25. Sakaki, T., Akiyoshi-Shibata, M., Yabusaki, Y., and Ohkawa, H. (1992) J. Biol. Chem. 267, 16497–16502

26. Oeda, K., Sakaki, T., and Ohkawa, H. (1985) DNA 4, 203–210

27. Ito, H., Fukuda, Y., Murata, K., and Kimura, A. (1983) J. Bacteriol. 153, 163–168

28. Sakaki, T., Shibata, M., Yabusaki, Y., Murakami, H., and Ohkawa, H. (1990) DNA Cell Biol. 9, 603–614

29. Bradford, M. M. (1976) Anal. Biochem. 72, 248–254

30. Iwasa, K., Kuribayashi, A., Sugiura, M., Nishiyama, Y., Ichimaru, M., Moriyasu, M., and Lee, D.-U. (2004) Pharmazie 59, 480–483

31. Omura, T., and Sato, R. (1964) J. Biol. Chem. 239, 2379–2385

32. Durst, F., and Nelson, D. R. (1995) Drug Metabol. Drug Interact. 12, 189–206

33. Poulos, T. L., Finzel, B. C., Gunsalus, I. C., Wagner, G. C., and Kraut, J. (1985) J. Biol. Chem. 260, 16122–16130

34. Muto, N., Inouye, K., Inada, A., Nakanishi, T., and Tan, L. (1987) Biochem. Biophys. Res. Commun. 146, 487–494

35. Liscombe, D. K., and Facchini, P. J. (2007) J. Biol. Chem. 282, 14741–14751

36. Nasreen, A., Rueffer, M., and Zenk, M. H. (1996) Tetrahedron Lett. 37,

8161–8164

37. Herbert, R. B. (2001) Nat. Prod. Rep. 18, 50–65

38. Graening, T., and Schmalz, H.-G. (2004) Angew. Chem. Int. Ed. 43, 3230–3256

39. Bischoff, D., Pelzer, S., Bister, B., Nicholson, G. J., Stockert, S., Schirle, M., Wohlleben, W., Jung, G., and Süssmuth, R. D. (2001) Angew. Chem. Int. Ed. 40, 4688–4691

40. Zhao, B., Guengerich, F. P., Bellamine, A., Lamb, D. C., Izumikawa, M., Lei, L., Podust, L. M., Sundaramoorthy, M., Kalaitzis, J. A., Reddy, L. M., Kelly, S. L., Moore, B. S., Stec, D., Voehler, M., Falck, J. R., Shimada, T., and Waterman, M. R. (2005) J. Biol. Chem. 280, 11599–11607

41. Funa, N., Funabashi, M., Ohnishi, Y., and Horinouchi, S. (2005) J. Bacteriol. 187, 8149–8155

42. Howard-Jones, A. R., and Walsh, C. T. (2006) J. Am. Chem. Soc. 128, 12289–12298

43. Dittrich, H., and Kutchan, T. M. (1991) Proc. Natl. Acad. Sci. U. S. A. 88, 9969–9973

44. Minami, H., Dubouzet, E., Iwasa, K., and Sato, F. (2007) J. Biol. Chem. 282, 6274–6282

45. Morishige, T., Tsujita, T., Yamada, Y., and Sato, F. (2000) J. Biol. Chem. 275, 23398–23405

46. Takeshita, N., Fujiwara, H., Mimura, H., Fitchen, J. H., Yamada, Y., and Sato, F. (1995) Plant Cell Physiol. 36, 29–36

47. Morishige, T., Dubouzet, E., Choi, K. B., Yazaki, K., and Sato, F. (2002) Eur. J. Biochem. 269, 5659–5667

48. Miller, G. P., and Guengerich, F. P. (2001) Biochemistry 40, 7262–7272

Enzyme Catalysis and Regulation:
Molecular Cloning and Characterization of
CYP80G2, a Cytochrome P450 That C
atalyzes an Intramolecular C–C Phenol
Coupling of ( S)-Reticuline in Magnoflorine
Biosynthesis, from Cultured *Coptis*
japonica Cells

Nobuhiro Ikezawa, Kinuko Iwasa and
Fumihiko Sato
J. Biol. Chem. 2008, 283:8810-8821.
doi: 10.1074/jbc.M705082200 originally published online January 29, 2008

Access the most updated version of this article at doi: [10.1074/jbc.M705082200](http://dx.doi.org/10.1074/jbc.M705082200)

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

Supplemental material:
[http://www.jbc.org/content/suppl/2008/01/29/M705082200.DC1.html](http://www.jbc.org/content/suppl/2008/01/29/M705082200.DC1.html)

This article cites 44 references, 18 of which can be accessed free at
[http://www.jbc.org/content/283/14/8810.full.html#ref-list-1](http://www.jbc.org/content/283/14/8810.full.html#ref-list-1)
