
REVIEW ARTICLE

# Cytoplasmic steps of peptidoglycan biosynthesis

Hélène Barreteau${}^{1}$, Andreja Kovač${}^{2}$, Audrey Boniface${}^{1}$, Matej Sova${}^{2}$, Stanislav Gobec${}^{2}$ & Didier Blanot${}^{1}$

${}^{1}$Laboratoire des Enveloppes Bactériennes et Antibiotiques, Institut de Biochimie et Biophysique Moléculaire et Cellulaire, Univ Paris-Sud, Orsay, France;  
and ${}^{2}$Fakulteta za farmacijo, Univerza v Ljubljani, Ljubljana, Slovenia

---

**Correspondence:** Didier Blanot, Laboratoire des Enveloppes Bactériennes et Antibiotiques, IBBMC, UMR 8619 CNRS, Bâtiment 430, Univ Paris-Sud, 91405 Orsay, France. Tel.: +33 1 69 15 81 65; fax: +33 1 69 85 37 15; e-mail: didier.blanot@u-psud.fr

Received 31 July 2007; revised 17 November 2007; accepted 19 December 2007. First published online February 2008.

DOI: 10.1111/j.1574-6976.2008.00104.x

Editor: Jacques Coyette

**Keywords**  
bacterial cell wall; peptidoglycan; UDP-$N$-acetylglucosamine; UDP-$N$-acetylmuramic acid; $D$-alanine; $D$-glutamic acid.

---

## Abstract

The biosynthesis of bacterial cell wall peptidoglycan is a complex process that involves enzyme reactions that take place in the cytoplasm (synthesis of the nucleotide precursors) and on the inner side (synthesis of lipid-linked intermediates) and outer side (polymerization reactions) of the cytoplasmic membrane. This review deals with the cytoplasmic steps of peptidoglycan biosynthesis, which can be divided into four sets of reactions that lead to the syntheses of (1) UDP-$N$-acetylglucosamine from fructose 6-phosphate, (2) UDP-$N$-acetylmuramic acid from UDP-$N$-acetylglucosamine, (3) UDP-$N$-acetylmuramyl-pentapeptide from UDP-$N$-acetylmuramic acid and (4) $D$-glutamic acid and dipeptide $D$-alanyl-$D$-alanine. Recent data concerning the different enzymes involved are presented. Moreover, special attention is given to (1) the chemical and enzymatic synthesis of the nucleotide precursor substrates that are not commercially available and (2) the search for specific inhibitors that could act as antibacterial compounds.

---

## Introduction

Peptidoglycan (or murein) is a major component of the cell wall of almost all eubacteria. It is a complex heteropolymer that is composed of long glycan chains that are cross-linked by short peptides (Rogers *et al.*, 1980). The glycan chains are made up of alternating $N$-acetylglucosamine (GlcNAc) and $N$-acetylmuramic acid (MurNAc) residues linked by $\beta 1 \rightarrow 4$ bonds. The $D$-lactoyl group of each MurNAc residue is substituted by a peptide stem with a composition most often seen as $L$-Ala-$\gamma$-$D$-Glu-$meso$-$A_2pm$ (or $L$-Lys)-$D$-Ala-$D$-Ala ($A_2pm$, 2,6-diaminopimelic acid) in nascent peptidoglycan, the last $D$-Ala residue being removed in the mature macromolecule. Cross-linking of the glycan chains generally occurs between the carboxyl group of $D$-Ala at position 4 and the amino group of the diaminoacid at position 3, either directly or through a short peptide bridge. Minor variations in the glycan chain, the peptide stem or the peptide bridge are present in the bacterial world and are detailed in the accompanying review by Vollmer *et al.* (2008).

The biosynthesis of peptidoglycan is a complex process that involves c. 20 reactions that take place in the cytoplasm (synthesis of the nucleotide precursors) and on the inner side (synthesis of lipid-linked intermediates) and outer side (polymerization reactions) of the cytoplasmic membrane. The present review deals with the cytoplasmic steps of peptidoglycan synthesis; the subsequent steps are reviewed by Bouhss *et al.* (2008) and Sauvage *et al.* (2008) in this issue.

The cytoplasmic steps (Fig. 1) can be divided into four sets of reactions: (1) formation of UDP-GlcNAc from fructose-6-phosphate, (2) formation of UDP-MurNAc from UDP-GlcNAc, (3) assembly of the peptide stem leading to UDP-MurNAc-pentapeptide and (4) ‘side’ or ‘annex’ pathways of synthesis of $D$-glutamic acid and the dipeptide $D$-alanyl-$D$-alanine. The discovery of the enzyme activities involved in these processes was the subject of the pioneering works by J. T. Park, J. L. Strominger, E. Ito, E. J. J. Lugtenberg, F. C. Neuhaus and others during the 1960s and 1970s. Their results have been summarized in previous reviews to which we refer the reader (Rogers *et al.*, 1980; Neuhaus & Hammes, 1981; Ward, 1984; Walsh, 1989; Bugg & Walsh, 1992; van Heijenoort, 2001; El Zoeiby *et al.*, 2003a; Katz & Caufield, 2003; Kotnik *et al.*, 2007a). In the present review, we will focus on recent data concerning the specificities, the kinetic and catalytic mechanisms and the three-dimensional structures of these enzymes. Moreover, because the nucleotide precursors that are substrates (or products)

Cytoplasmic steps of peptidoglycan biosynthesis

Fructose-6-P
Glucose-6-P
Glucosamine-1-P
N-Acetylglucosamine-1-P
UDP-GlcNAc
UDP-GlcNAc-enolpyruvate
Peptidoglycan

Fig. 1. Cytoplasmic steps of peptidoglycan biosynthesis. DA, diamino acid (generally meso-A₂pm or L-Lys).

of several of these activities are not commercially available, we will give special attention to their chemical or enzymatic synthesis. Finally, as these enzymes are targets for antibacterial compounds, we present an overview of the existing inhibitors and of the current search for new specific inhibitors.

Biosynthesis of UDP-N-acetylg glucosamine

In bacteria, UDP-GlcNAc biosynthesis from fructose-6-phosphate requires four successive enzyme activities: glucosamine-6-phosphate synthase (GlmS), phosphoglucosamine mutase (GlmM), glucosamine-1-phosphate acetyltransferase
and $N$-acetylglucosamine-1-phosphate uridyltransferase (the latter two activities are carried by the GlmU bifunctional enzyme) (Fig. 1). UDP-GlcNAc is also present in eukaryotes because GlcNAc is an important building block for major biomolecules such as chitin and glycoproteins. However, as the eukaryotic pathway of UDP-GlcNAc biosynthesis is different from the prokaryotic pathway, the latter can be considered to be a target for specific antibacterial compounds.

### GlmS

GlmS is an amidotransferase that takes part in the first committed step of hexosamine metabolism. It has been purified from *Escherichia coli* (Badet *et al.*, 1987) and *Thermus thermophilus* (Badet-Denisot *et al.*, 1997), and characterized. It is an essential and dimeric enzyme that catalyses the conversion of D-fructose-6-phosphate into D-glucosamine-6-phosphate, using L-glutamine as the nitrogen source. It follows an ordered bi-bi mechanism (Badet *et al.*, 1988).

The GlmS monomer is composed of two structurally and functionally distinct domains (Fig. 2). The N-terminal 30-kDa glutaminase domain promotes glutamine hydrolysis into glutamate and ammonia, whereas the C-terminal 40-kDa isomerase domain binds the nitrogen acceptor and uses the ammonia that is produced for the conversion of fructose-6-phosphate into glucosamine-6-phosphate. The glutamine hydrolysis reaction uses the N-terminal cysteine thiol, which forms a $\gamma$-glutamyl thioester intermediate. The ketose/aldose isomerase activity proceeds by abstraction of the C-1 $pro-R$ hydrogen of a putative fructosimine-6-phosphate intermediate, to form a transient cis-enolamine that, upon reprotonation at the re face of the C-2 $sp^2$ carbon, yields glucosamine-6-phosphate (Fig. 3) [see references in Badet-Denisot *et al.* (1993)].

The individual crystal structures of these two domains as complexes with their respective reaction products have been solved (Obmolova *et al.*, 1994; Isupov *et al.*, 1996; Teplyakov *et al.*, 1998). As shown by its conserved N-terminal catalytic cysteine, GlmS belongs to the N-terminal nucleophile (Ntn) family and shares common structural and catalytic mechanisms with other Ntn family members. Two other invariant residues appear to have key roles in substrate recognition by anchoring the $\alpha-\mathrm{COO}^{-}$ and $\alpha-\mathrm{NH}_{3}^{+}$ functions of glutamine. However, no sequence similarity has been detected between other isomerases and GlmS. Moreover, with respect to other isomerases, the catalytic residues of GlmS belong to different polypeptide chains, whereas they are usually located on only one subunit in other isomerases. The intact *Escherichia coli* GlmS crystal structure was solved (Teplyakov *et al.*, 2001), thus allowing an understanding as to how the ammonia produced at the glutamine site is delivered to the sugar phosphate site: an intramolecular nitrogen channelling between the two active sites that are separated by 18 Å is involved. Both sites contribute to the formation of this channel due to several residues that are strictly conserved in the GlmS sequence (Fig. 2). Moreover, the dimerization of GlmS appears to be crucial for the closure of the sugar-binding site and the formation of the channel. Recently, a study combining molecular dynamics simulations and site-directed mutagenesis allowed to define amino acid residues involved in the channelling process (Floquet *et al.*, 2007a).

The GlmS activity is inhibited by glutamine analogues that may be either naturally occurring or synthetic. Some of these possess an electrophilic functionality at the $\gamma$-position that can establish a covalent bond with the N-terminal cysteine residue located in the glutamine-binding site. Among the naturally occurring compounds, there is antapsin, an epoxyamino acid that is liberated after hydrolysis of the dipeptide antibiotic bacilysin (tetaine) (Kenig *et al.*, 1976) and that inactivates the enzyme; glutamine protects against inactivation (Chmara *et al.*, 1984). Other natural inhibitors are known, like azaserine and albizziiin (Winterburn & Phelps, 1971). On the basis of these naturally occurring compounds, synthetic inhibitors have been studied, like 6-diazo-5-oxo-L-norleucine (DON) (Badet *et al.*, 1987) and $N^{3}$-fumaroyl L-2,3-diaminopropionic acid derivatives (Chmara *et al.*, 1985; Badet *et al.*, 1988). Other glutamine analogues with low micromolar $K_{\mathrm{i}}$ values have

![Structure Image]

**Fig. 2.** Three-dimensional structure of the *Escherichia coli* GlmS monomer (PDB entry: 1jxa). The glutaminase and isomerase domains are shown in green and blue, respectively. Substrate fructose-6-phosphate is shown in magenta. The ammonia channel is indicated by red dots.
Cytoplasmic steps of peptidoglycan biosynthesis

Fig. 3. Mechanism of the ketose/aldose isomerization reaction catalysed by GlmS. From the reaction of the open form of fructose-6-phosphate and ammonia (originating from L-glutamine), a fructosimine intermediate is formed. Stereospecific abstraction of the pro-R proton of the C-1 of the intermediate gives a transient cis-enolamine that, upon reprotonation at the re face of the C-2 sp² carbon, yields the open form of glucosamine-6-phosphate. According to Golinelli-Pimpaneau et al. (1989).

been synthesized, such as the γ-dimethylsulfonium derivative (Walker et al., 2000). Some carbohydrate compounds, like 2-amino-2-deoxy-D-glucitol-6-phosphate, have been shown to inhibit the enzyme competitively with respect to fructose-6-phosphate (Badet-Denisot et al., 1995); recently, other carbohydrate-based inhibitors have been designed as analogues of the reaction intermediates (Bearne & Blouin, 2000). N-iodoacetylg glucosamine-6-phosphate is an active site-directed irreversible inactivator of GlmS from Escherichia coli, and it interacts with both the sugar- and the glucosamine-binding sites (Bearne, 1996). Very recently, new inhibitors of bacterial GlmS have been discovered through virtual screening; interestingly, these compounds were predicted to interact with the protein region that forms a pocket at the interface between the two enzyme monomers, which opens the way to new molecules that alter the dimerization process (Floquet et al., 2007b).

### GlmM

GlmM is the second enzyme involved in the biosynthesis of UDP-GlcNAc, and it catalyses the interconversion of the glucosamine-6-phosphate and glucosamine-1-phosphate isomers. GlmM was first characterized and purified to near homogeneity in Escherichia coli (Mengin-Lecreulx & van Heijenoort, 1996). Its amino acid sequence contains the characteristic signature of hexosephosphate mutases, including the serine residue (Ser102) for which phosphorylation is required for enzyme activity. Accordingly, GlmM is active only in its phosphorylated form, although the dephosphorylated form also exists in vivo and both forms of the enzyme can be separated by HPLC (Jolly et al., 1999). The GlmM enzymes from Staphylococcus aureus and Pseudomonas aeruginosa have also been purified (Jolly et al., 1997; Tavares et al., 2000).

The reaction catalysed by GlmM follows a ping-pong bi–bi mechanism, where GlcN-1,6-diphosphate appears as an intermediate in the catalytic process, acting as both the first product and the second substrate (Fig. 4). The GlmM enzyme also catalyses the interconversion of the 1-phosphate and 6-phosphate isomers of glucose, although with reduced rate constants (Jolly et al., 1999). The inactive, dephosphorylated form of GlmM undergoes an autophosphorylation reaction when it is incubated with ATP in the presence of divalent cations. The site of phosphorylation has been shown to be the aforementioned Ser102 residue (Jolly et al., 2000).

No crystal structure of the GlmM enzyme is available to date. However, the crystal structure of another member of the hexosephosphate mutase family, rabbit muscle phosphoglucomutase, has been reported (Dai et al., 1992). With this phosphoglucomutase, the estimated volume of the active site cleft involving the catalytic serine residue has been described as being large enough to accommodate an ATP molecule.

### GlmU

First identified in Bacillus subtilis, the GlmU protein was initially thought to be a GlcNAc-1-phosphate uridylyltransferase
that catalyses the formation of UDP-GlcNAc from GlcNAc-1-phosphate and UTP (Hove-Jensen, 1992; Mengin-Lecreulx & van Heijenoort, 1993). However, in *Escherichia coli*, GlmU was shown to be a bifunctional enzyme, catalysing both acetyltransfer and uridyltransfer during the transformation of GlcN-1-phosphate to UDP-GlcNAc (Mengin-Lecreulx & van Heijenoort, 1994). Moreover, the order of the chemical reaction is not random but imposed by the enzyme: GlmU first catalyses acetyltransfer from AcCoA to GlcN-1-P with the release of GlcNAc-1-P, then uridyltransfer from UTP to GlcNAc-1-P in the presence of Mg²⁺, yielding inorganic pyrophosphate and UDP-GlcNAc (Mengin-Lecreulx & van Heijenoort, 1994; Gehring et al., 1996). However, it should be mentioned that, under certain conditions, GlmU is capable of catalysing the two reactions in the reverse order (i.e. uridyltransfer precedes acetyltransfer), but with greatly reduced kinetic parameters (Pompeo et al., 2001).

The construction of truncated forms of GlmU has shown that this bifunctional enzyme is organized in two domains that operate without substrate channelling and that are individually active (Gehring et al., 1996; Brown et al., 1999; Pompeo et al., 2001). However, both these activities are essential for cell viability (Pompeo et al., 2001). The sizes of the two domains appear to be roughly equivalent, each one representing about half of the 49-kDa protein (Fig. 5).

The crystal structure of a truncated form of GlmU has shown that the two distinct domains are connected by a long α-helical arm (Brown et al., 1999). The C-terminal domain catalyses the first reaction, which is responsible for the CoA-dependent acetylation of GlcN-1-phosphate, and it shows sequence similarities with a number of acetyltransferases (Mengin-Lecreulx & van Heijenoort, 1994; Raetz & Roderick, 1995; Gehring et al., 1996). It is characterized by an imperfect, tandem hexapeptide repeat sequence motif [LIV]-[GAED]-X₂-[STAV]-X (Vaara, 1992), which folds into a left-handed β-helix (LβH) (Raetz & Roderick, 1995) (Fig. 5). As with most enzymes that contain an LβH structure, the GlmU acetyltransferase domain has a trimeric organization that is absolutely required for the activity. Moreover, the catalytic site is formed by complementary regions of contact between the three adjacent monomers, as confirmed by the crystal structures of the truncated and the entire GlmU (Olsen & Roderick, 2001; Pompeo et al., 2001). In the C-terminal domain, there are also four cysteine residues, which are apparently located near the active site. These do not appear to be directly involved in the catalytic process, but nevertheless two of them, Cys307 and Cys324, have important roles in the acetyltransferase activity (Pompeo et al., 1998). Recently, the crystal structure of the *Escherichia coli* GlmU acetyltransferase active site was determined in complexes with AcCoA and with CoA/GlcN-1-P (Olsen et al., 2007).

Fig. 5. Three-dimensional structure of the *Escherichia coli* GlmU monomer (PDB entry: 1hv9). The N- and C-terminal residues are marked with N and C, respectively. The acetyltransferase and uridyltransferase domains are shown in yellow and green, respectively. The α-helical arm connecting the two domains is shown in blue. Products CoA and UDP-GlcNAc are shown in magenta. The Co²⁺ ion cocrystallized with GlmU is depicted as a black sphere.

The second reaction that is catalysed by GlmU involves uridyltransfer from UDP to GlcNAc-1-phosphate and occurs at the N-terminal domain. This domain shares sequence homology with a variety of nucleotidyltransferases over residues Met1-Ala120, which are also known as nucleotide diphosphate sugar pyrophosphorylases, and has strict conservation of the L-X₂-G-X-G-T-X-M-(X)₄-P-K motif (Mio et al., 1998). Its crystal structure shows that the uridine-binding site is a large open pocket, bounded by two lobes. The first lobe comprises residues that interact with the nucleotide (Asn3-Val111 and His216-Asn227), and the second lobe comprises those residues that interact with the sugar moiety (Glu112-Val215) (Brown et al., 1999; Olsen & Roderick, 2001) (Fig. 5). Contrary to the acetyltransferase domain, trimerization is not essential for expression of the uridyltransferase activity of GlmU; however, some of the interactions between the two domains appear to participate in the folding and stability of the N-terminal domain (Pompeo et al., 2001).

The crystal structure of the GlmU protein from *Streptococcus pneumoniae* has been solved both in its apo form and

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved  
in its complex with UDP-GlcNAc and Mg²⁺ (Kostrewa et al., 2001; Sulzenbacher et al., 2001). Pneumococcal GlmU contains the common LβH motif with high-sequence conservation, suggesting that similar LβH motifs exist in all the GlmU structures and have important roles in the acetyltransferase activity.

GlmU acetyltransferase activity is inactivated in the presence of thiol-specific reagents, such as iodoacetamide and N-substituted maleimides (Mengin-Lecreulx & van Heijenoort, 1994; Pompeo et al., 1998). Recently, some thiol-specific reagents, including N-ethylmaleimide and its derivatives, have been reported to inhibit the growth of bacterial pathogens and are thereby speculated to inhibit GlmU (Zentz et al., 2002; Burton et al., 2006).

### Comparison of the prokaryotic and eukaryotic systems

In eukaryotes, the pathway for UDP-GlcNAc biosynthesis appears to be significantly different: acetyltransfer occurs on GlcN-6-P and not on GlcN-1-P and, most importantly, acetyltransferase and uridyltransferase activities are carried by two distinct monofunctional enzymes (Fig. 6).

In mammalian cells, the GlmS equivalent, known as glutamine:fructose-6-P amidotransferase (GFAT), is an insulin-regulated enzyme that controls the flux of glucose into the hexosamine pathway (Traxinger & Marshall, 1991). GFAT is about 280 kDa in size, and it is composed of four subunits; it belongs to the Ntn-amidotransferase family and possesses a Cys1 residue, as does the Escherichia coli GlmS (Huynh et al., 2000). However, unlike its bacterial equivalent, GFAT is subject to allosteric regulation by UDP-GlcNAc (Traxinger & Marshall, 1991) and can also be

regulated by its product GlcN-6-P (Broschat et al., 2002). In mammals, the pyrophosphorylase that condenses UTP and GlcNAc-1-P has also been identified; purified to near homogeneity from pig liver extracts, it appears to be a homodimer that is composed of two 64-kDa subunits. It requires a divalent cation (Mn²⁺) for activity and has an unusual specificity: at high concentrations, it uses UDP-GalNAc as a substrate as well as UDP-GlcNAc in the reverse direction and GalNAc-1-P as well as GlcNAc-1-P in the forward direction (Szumilo et al., 1996). Two isoforms of this enzyme, AGX1 and AGX2, have been identified in humans (Peneff et al., 2001b).

In yeast such as Saccharomyces cerevisiae and Candida albicans, the hexosamine metabolism has also been well studied (Milewski et al., 2006) and the four different enzymes involved in this pathway have been shown to be essential for cell viability. The first reaction, which sees the formation of GlcN-6-P from fructose-6-P, is catalysed by GFA1 (Watzele & Tanner, 1989). Like its mammalian homologue, this enzyme is a homotetramer of 80 kDa subunits that has been crystallized (Raczynska et al., 2007). Then, GlcN-6-P is N-acetylated by the GNA1 acetyltransferase to yield GlcNAc-6-P (Mio et al., 1999). The three-dimensional structure of GNA1 has been shown to be a dimer of two identical subunits (Peneff et al., 2001a). GlcNAc-6-P is further isomerized into GlcNAc-1-P by the GlcNAc-phosphate mutase AGM1 (Hofmann et al., 1994). The crystal structures of this 60-kDa protein from Candida albicans have been reported, both for the apoform and for complexes with substrates and products (Nishitani et al., 2006). Finally, UDP-GlcNAc is produced from GlcNAc-1-P by a monofunctional UDP-GlcNAc pyrophosphorylase, known as UAP1, whose sequence is well conserved in the human homologue (Mio et al., 1998; Peneff et al., 2001b). The crystal structure of the Candida albicans enzyme has been solved recently. Unlike prokaryotic GlmU, which has a metal ion that acts as a cofactor, no metal ion has been seen in this candidal UAP1; instead, the terminal amino function of a conserved lysine residue occupies the virtual metal ion-binding site (Maruyama et al., 2007).

#### Biosynthesis of UDP-N-acetylmuramic acid

The first committed stage towards the creation of the peptidoglycan polymer involves the formation of UDP-MurNAc from UDP-GlcNAc by two enzymes: MurA and MurB (Fig. 1). MurA (formerly known as MurZ) catalyses the first step of this transformation by transferring the enolpyruvate moiety of phosphoenolpyruvate (PEP) to the 3′-hydroxyl of UDP-GlcNAc with the release of inorganic phosphate (Pi). The resulting product, UDP-GlcNAc-enolpyruvate, undergoes a reduction catalysed by MurB using

Fructose-6-P  
Glucosamine-6-P  
Glucosamine-1-P  
N-Acetylglucosamine-1-P  
UDP-N-acetylglycosamine  

Glucose-6-P  
N-Acetylglucosamine-6-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose-1-P  

Glucose
one equivalent of NADPH and a solvent-derived proton. This two-electron reduction creates the lactyl ether of UDP-MurNAc.

**MurA**

The MurA reaction constitutes a rare biochemical process. The only other known enolpyruvyl transfer from PEP to an OH group with the concomitant release of Pi occurs during the shikimic acid pathway, in the reaction catalysed by 5-enolpyruvylshikimate-3-phosphate synthase AroA (Walsh et al., 1996; Byczynski et al., 2003). Interestingly, although MurA and AroA share only c. 25% sequence identity, they exhibit the same protein architecture. The MurA enzymes from Enterobacter cloacae and Escherichia coli have been overproduced and purified (Marquardt et al., 1992; Wanke et al., 1992), thereafter serving for extensive mechanistic and structural studies. The MurA reaction pathway (Fig. 7) follows an addition–elimination mechanism, with the formation of a noncovalently bound phospholactyl-UDP-GlcNAc tetrahedral intermediate (Marquardt et al., 1993). Two other intermediates have been characterized: a phospholactyl-enzyme adduct and an O-phosphothioketal intermediate, both of which are linked to the Cys115 residue (Wanke & Amrhein, 1993; Brown et al., 1994; Ramilo et al., 1994). However, it was demonstrated that they are not essential for catalysis (Kim et al., 1996).

The X-ray structures of several forms of the protein have been solved, both unliganded and in a complex (Schönbrunn et al., 1996; Skarzynski et al., 1996, 1998; Eschenburg & Schönbrunn, 2000; Schönbrunn et al., 2000a,b) (Fig. 8). Unliganded MurA appears as a two-domain protein with an unusual fold (inside-out α/β barrel) that is built up from the sixfold repetition of the same βααββ motif (Fig. 8a). Near the hinge region, there is an isoaspartyl residue that is the product of a posttranslational modification of the Asn67–Gly68 dipeptide moiety. Upon UDP-GlcNAc binding, the reaction follows an induced-fit mechanism in which the two-domain structure undergoes large conformational changes that lead to a closed form. This has been confirmed by fluorescence spectroscopy and small-angle X-ray scattering (Schönbrunn et al., 1998). The Pro112–Pro121 loop containing Cys115 is flexible; it is solvent-exposed in the open conformation, but forms a lid around the interdomain section in the closed conformation (Fig. 8b).

The role of the amino acid residues involved in the reaction pathway (Fig. 7) has been recently questioned. Cys115 was initially suggested to act as an acid–base catalyst in the addition–elimination reaction (Skarzynski et al., 1998; Krekel et al., 2000). However, it seems that it is essential for product release only (Eschenburg et al., 2005a). Similarly, Asp305, which was considered to be the

![Figure 7](#fig7)

Fig. 7. Mechanism of the addition–elimination reaction catalysed by MurA. X, Y, and Z stand for side-chains involved in the reaction. A proton is added to C-3 of PEP whereas the 3′-OH group of UDP-GlcNAc gets deprotonated. After formation of a tetrahedral intermediate (central formula), the elimination of phosphate results in the formation of UDP-GlcNAc-enolpyruvate. The initial proposal that Cys115 could function as Y and Z and Asp305 as X was recently questioned. In fact, Asp305 would be responsible for the final proton abstraction leading to the elimination of phosphate (Z). According to Marquardt et al. (1993) and Eschenburg et al. (2003, 2005a).

base abstracting a proton from the 3′-hydroxyl group of UDP-GlcNAc (Skarzynski et al., 1996; Samland et al., 2001), would be responsible for the final proton abstraction from the C-3 atom of the tetrahedral intermediate (Eschenburg et al., 2003). The Lys22 residue, which is strictly conserved in the MurA enzymes, was shown to be involved in the binding of PEP and to participate in the conformational change that

© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved

Cytoplasmic steps of peptidoglycan biosynthesis

Fig. 8. Three-dimensional structures of MurA.
(a) Open, ligand-free form of Enterobacter cloacae MurA (PDB entry: 1naw). (b) Closed form of Escherichia coli MurA complexed with UDP-GlcNAc and fosfomycin (PDB entry: 1uae). The two domains are shown in blue and green. The Pro12–Pro121 loop is shown in red. UDP-GlcNAc is shown as sticks in magenta. Cys115 (a) and covalently linked Cys115-fosfomycin (b) are depicted as ball-and-stick models in red.

leads to the formation of the catalytically competent enzyme complex (Samland *et al.*, 1999, 2001); it has been presumed recently to also accomplish the two-proton transfer required for the addition of the substrates (Eschenburg *et al.*, 2003).

The stereochemical course of the reaction has been revised as well. Initial studies using PEP analogues had led to the conclusion that the proton addition step at C-3 of PEP proceeded at the 2-*si* face and that the stereochemistry of the pair of addition and elimination steps was *anti/syn* (Kim *et al.*, 1995; Lees & Walsh, 1995; Skarzynski *et al.*, 1998). However, recent examination of the crystal structure of the D305A mutant complexed with the tetrahedral intermediate favours an addition at the *re* face of PEP and an *anti/syn* stereochemistry (Eschenburg *et al.*, 2003).

MurA is the target of the naturally occurring broad-spectrum antibiotic fosfomycin, which forms a covalent adduct with the reactive Cys115 residue (Marquardt *et al.*, 1994). In the MurA enzymes of species that are naturally resistant to fosfomycin, such as *Mycobacterium tuberculosis* (De Smet *et al.*, 1999) and *Chlamydia trachomatis* (McCoy *et al.*, 2003), the corresponding Cys residue is changed into Asp. *Escherichia coli* MurA that contains the C115D mutation is enzymatically active and resistant to inactivation by fosfomycin (Kim *et al.*, 1996).

Gram-negative bacteria have one copy of the *murA* gene (Brown *et al.*, 1995), while Gram-positive bacteria have two (*murA1* and *murA2*), which have probably arisen from gene duplication (Du *et al.*, 2000). The MurA1 and MurA2 enzymes from *Streptococcus pneumoniae* have been purified: their catalytic parameters are similar, and they are both inhibited by fosfomycin (Du *et al.*, 2000).

It has been reported recently that UDP-MurNAc tightly binds to and inhibits *Escherichia coli* MurA; a possible role of the nucleotide in the regulation of peptidoglycan biosynthesis has been inferred (Mizyed *et al.*, 2005).


MurB

MurB from *Escherichia coli* has been overproduced and purified. It is a 38-kDa protein that contains a stoichiometric amount of bound FAD (Benson *et al.*, 1993; Tayeh *et al.*, 1995). It follows a ping-pong bi-bi mechanism, with weak and strong substrate inhibition by NADPH and UDP-GlcNAc-enolpyruvate, respectively. It is activated by cations, such as K<sup>+</sup>, NH<sub>4</sub><sup>+</sup> and Rb<sup>+</sup> (Dhalla *et al.*, 1995).

The reaction catalysed by MurB involves two half-reactions in which FAD serves as the redox intermediate (Fig. 9). The first half-reaction is the reduction of FAD to FADH<sub>2</sub> by NADPH. This starts with the binding of NADPH to MurB and the transfer of the 4-*pro*-S hydrogen of NADPH to N-5 of the enzyme-bound flavin. The release of NADP<sup>+</sup> is followed by the binding of UDP-GlcNAc-enolpyruvate. The second half-reaction is the reduction of the vinylic enol ether by FADH<sub>2</sub>. Hydride transfer from the reduced flavin to C-3 of the enolpyruvyl moiety of the nucleotide substrate generates a carbanionic intermediate that is then protonated at C-2 by a solvent-equilibrated proton (Benson *et al.*, 1993, 1997b). The stereochemistry of the reduction has been studied through the use of UDP-GlcNAc-(E)-enolbutyrate as a mechanistic probe (Lees *et al.*, 1996).

The crystal structures of unliganded MurB and of its dead-end complex with UDP-GlcNAc-enolpyruvate have been solved (Benson *et al.*, 1995, 1996, 1997a) (Fig. 10). The protein is composed of three domains: domains 1 and 2 mediate FAD binding, and domain 3 mediates the binding of its substrates. The binding of UDP-GlcNAc-enolpyruvate induces a substantial movement of domain 3. With the perdeuterated, <sup>13</sup>C/<sup>15</sup>N-labelled MurB studied by NMR spectroscopy (Farmer *et al.*, 1996; Constantine *et al.*, 1997), it was deduced that NADP<sup>+</sup> binds in the same pocket as UDP-GlcNAc-enolpyruvate, inducing structural changes,

Fig. 9. Reaction mechanism of the MurB enzyme from *Escherichia coli*. (a) The first half-reaction consists in the transfer of the 4-pro-S hydride from NADPH to N-5 of Enz-FAD. (b) The second half-reaction consists in the transfer of this hydride from Enz-FADH₂ to C-3 of the enol/pyruvyl moiety of UDP-GlcNAc-enolpyruvate, followed by quenching at C-2 by a solvent-exchangeable proton, to yield the d-configuration of the lactyl ether product. B₁H and B₂H stand for side-chains (Glu325 and/or Arg 159) stabilizing by hydrogen bonding the carbanion intermediate as an enol. B₃H stands for is a general acid catalyst (Ser229) serving as the proton donor to quench the carbanion/enol intermediate and deliver the proton at C-2. From Benson et al. (1993, 1995, 1997a, b).

and that NADPH transfers a hydride to the si face of the FAD isoalloxazine ring.

Comparison of the X-ray structures of *Escherichia coli* and *Staphylococcus aureus* MurB enzymes has revealed notable distinctions with respect to their structural elements. *Escherichia coli* MurB, classified as type I, contains a Tyr loop and a split βαββ fold (Fig. 10a), whereas *Staphylococcus aureus* MurB, classified as type II, lacks these secondary elements (Fig. 10b). This has consequences on the mode of substrate binding (Benson et al., 2001). Recently, a type II MurB enzyme (*Thermus caldophilus*) was crystallized in the presence of UDP-GlcNAc-enolpyruvate. X-ray data and sequence alignments allowed the definition of two subtypes, type II-a (e.g. *Staphylococcus aureus* and *Bacillus subtilis*) and type II-b (e.g. *Thermus caldophilus* and *Chlamydia pneumoniae*), that contain a serine or a cysteine residue, respectively, as a proton donor to quench the carbanionic intermediate (Kim MK et al., 2007). Six other conserved amino acids were also shown by site-directed mutagenesis to be essential for MurB activity (Nishida et al., 2006).

Cytoplasmic steps of peptidoglycan biosynthesis

# Inhibitors of MurA and MurB

As already mentioned, the most well-known inhibitor of MurA is fosfomycin 1 (Fig. 11), an epoxide compound that reacts with the Cys115 residue. A number of novel inhibitors of MurA have been discovered recently through various high-throughput screening efforts in the pharmaceutical industry (Dai et al., 2002; DeVito et al., 2002; Labaudiniere et al., 2005). Three noncovalent inhibitors show submicromolar IC₅₀ values (IC₅₀ = 0.2–0.9 μM); however, they also show nonspecific inhibition of DNA, RNA and protein biosynthesis (Baum et al., 2001). In a whole-cell peptidoglycan synthesis assay, two new inhibitors were identified: a derivative of diarylmethane and a substituted imidazole

(Barbosa et al., 2002). Sesquiterpene lactones [e.g. 2: IC₅₀ = 10.3 μM (Pseudomonas aeruginosa) and 16.7 μM (Escherichia coli)] were shown to alkylate the thiol group of Cys115 of MurA from Pseudomonas aeruginosa and Escherichia coli and thereby to act as irreversible inhibitors (Bachelier et al., 2006). A dodecapeptide inhibitor of MurA from Pseudomonas aeruginosa (IC₅₀ = 200 μM) was selected by phage display and appeared to be competitive with respect to UDP-GlcNAc (Molina-López et al., 2006).

The fluorescent dye 8-anilino-1-naphthalene sulfonate 3 binds to the solvent-exposed region of MurA from Enterobacter cloacae, and its X-ray cocrystal structure provides the basis for an alternative approach in the design of new inhibitors (Schönbrunn et al., 2000a). Thus,

**Fig. 10.** Three-dimensional structures of MurB.
(a) Structure of *Escherichia coli* MurB (type I) (PDB entry: 1mbt).
(b) Structure of *Staphylococcus aureus* MurB (type II) (PDB entry: 1hsk). The three domains are shown in yellow, blue, and green. FAD is shown in magenta. Structural elements that are absent in type II MurB are shown in red.

**Fig. 11.** Formulae of inhibitors of MurA and MurB.

FEMS Microbiol Rev **32** (2008) 168–207
© 2008 Federation of European Microbiological Societies
Published by Blackwell Publishing Ltd. All rights reserved
5-sulfonoxy-anthranilic acid derivatives that were obtained by high-throughput screening bind to the same region of MurA and obstruct its transition from the open to the closed forms (Eschenburg *et al.*, 2005b).

Tri-substituted thiazolidinones [e.g. 4: IC₅₀ (*Escherichia coli*) = 7.7 μM] were the first small-molecule inhibitors of MurB; they were designed to mimic the diphosphate moiety of UDP-GlcNAc-enolpyruvate and prepared by a parallel synthesis approach (Andres *et al.*, 2000). As their bioisosteric replacement, a series of imidazolinone analogues were synthesized and found to possess potent MurB inhibitory activity (the best IC₅₀ values for *Escherichia coli* ranged from 12 to 40 μM) as well as promising antibacterial activity against *Staphylococcus aureus* (MIC values: 2–4 μg mL⁻¹) (Bronson *et al.*, 2003). Two inhibitors of *Staphylococcus aureus* MurB, with *K*ₐ values in the submicromolar range (0.19 and 0.14 μM), were discovered by high-throughput screening (Sarver *et al.*, 2002). 4-Alkyl and 4,4′-dialkyl 1,2-bis(4-chlorophenyl)pyrazolidine-3,5-dione derivatives were found to inhibit MurA (the best IC₅₀ values for *Escherichia coli* ranged from 9.8 to 50 μM) and MurB (the best IC₅₀ values for *Escherichia coli* and *Staphylococcus aureus* ranged from 5.1 to 50 μM) (Kutterer *et al.*, 2005). In addition, four structurally related 3,5-dioxopyrazolidines inhibited MurB with IC₅₀ values between 4 and 35 μM, and also MurA and MurC to weaker extents. The crystal structure of a complex with compound 5 indicated that the 3,5-dioxopyrazolidine core occupies the same region of the MurB active site as the *N*-acetyl group of the substrate (Yang *et al.*, 2006). Antibacterial activity against Gram-positive bacteria was seen for many inhibitors from the pyrazolidine-3,5-dione series; however, when tested in the presence of bovine serum albumin (BSA), the activity was lost, indicating high protein-binding properties of these compounds (Kutterer *et al.*, 2005; Yang *et al.*, 2006). A similar decline in antibacterial activity due to the presence of BSA was also seen for thiazolyl urea and carbamate derivatives [e.g. 6: IC₅₀ (*Staphylococcus aureus*) = 19 μg mL⁻¹], which were good inhibitors of MurA and MurB as well as of the growth of some Gram-positive bacteria (Francisco *et al.*, 2004).

### Biosynthesis of the UDP-MurNAc-peptides

#### Generality of the Mur ligases

The stepwise assembly of the peptide stem of peptidoglycan is ensured by a series of four essential enzymes, known as the Mur ligases (MurC, D, E and F). These provide for additions of L-alanine (MurC), D-glutamic acid (MurD), a diamino acid, generally *meso*-diaminopimelic acid or L-lysine (MurE) and dipeptide D-Ala-D-Ala (MurF) onto the D-lactoyl group of UDP-MurNAc (Fig. 1). A fifth enzyme, Mpl, adds the tripeptide L-Ala-γ-D-Glu-*meso*-A₂pm directly onto UDP-

MurNAc during peptidoglycan recycling. The Mur ligases catalyse the formation of an amide or a peptide bond with simultaneous formation of ADP and Pᵢ from ATP. A divalent cation, Mg²⁺ or Mn²⁺, is essential for the reaction. Their mechanism of action has been studied through biochemical experiments, amino acid sequence examination, site-directed mutagenesis, assays of transition-state analogue inhibitors and X-ray structure determination. These studies have shown that the Mur ligases share three characteristics:

(1) They have the same reaction mechanism, which consists first in the activation of the carboxyl group of the UDP-precursor by ATP, generating an acyl phosphate intermediate and ADP; the acyl phosphate then undergoes the nucleophilic attack of the amino group of the condensing amino acid (or dipeptide), leading to the formation of a high-energy tetrahedral intermediate, which eventually breaks down into amide or peptide and Pᵢ (Fig. 12) [see references in Bouhss *et al.* (2002)].

(2) They have a series of six invariant residues in addition to an ATP-binding consensus sequence. This finding led to the definition of the Mur ligases as a new family of enzymes (Bouhss *et al.*, 1997, 1999b; Eveland *et al.*, 1997). Three other enzymes that are not related to peptidoglycan biosynthesis also belong to this family: the folylpoly-γ-L-glutamate synthetase FolC (Sheng *et al.*, 2000), the C-terminal region of cyanophycin synthetase CphA (Ziegler *et al.*, 1998) and the poly-γ-glutamate synthetase CapB from Bacilli (Candela & Fouet, 2006).

(3) They have the same three-dimensional structures in three domains, as seen by crystallographic studies (Fig. 13).

![Reaction Mechanism Diagram](attachment://reaction_mechanism.png)

**Fig. 12.** Reaction mechanism of the Mur ligases. First, the reaction of the carboxyl group of the UDP-precursor (R-COO⁻) with ATP generates an acyl phosphate intermediate and ADP. Then, the acyl phosphate undergoes the nucleophilic attack of the amino group of the amino acid or dipeptide (R′-NH₂) to form a tetrahedral transition state, which eventually breaks down into amide or peptide (R-CO-NH-R′) and Pᵢ.
Cytoplasmic steps of peptidoglycan biosynthesis

Fig. 13. Three-dimensional structures of the Mur ligases. (a) Open, ligand-free form of *Haemophilus influenzae* MurC (PDB entry: 1gqq). (b) Closed form of *Escherichia coli* MurD complexed with UDP-MurNAc-L-Ala, ADP and Mg²⁺ (PDB entry: 2uag). (c) Closed form of *Escherichia coli* MurE complexed with UDP-MurNAc-L-Ala-γ-D-Glu-meso-A₂pm (PDB entry: 1e8c). (d) Closed form of *Streptococcus pneumoniae* MurF complexed with a sulfonamide inhibitor (compound 14 in Fig. 15) (PDB entry: 2am1). The N-terminal, central and C-terminal domains are shown in green, blue and yellow, respectively. Ligands are shown in magenta. Mg²⁺ ions in (b) are depicted as black spheres. The black arrows in (a) show the direction of movement of the N- and C-terminal domains upon going from the open to the closed form.

The N-terminal domain is involved in the binding of the UDP-precursor, the central domain in the binding of ATP, and the C-terminal domain in the binding of the amino acid or dipeptide. Whereas the topologies of the central and C-terminal domains are similar among the Mur ligases, that of the N-terminal domain shows differences, with MurC and MurD on the one hand, and MurE and MurF on the other hand. These differences are related to the lengths of the UDP-precursor substrates. These enzymes exist in ‘closed’ and ‘open’ conformations. The closure of the conformation is thought to be provoked by ligand binding. The description and the comparison of the three-dimensional structures of the Mur ligases has been reviewed excellently by Smith (2006).

## MurC

The MurC ligase adds the first amino acid of the peptide stem. In most bacterial species, this amino acid is L-alanine; in rare cases, glycine or L-serine is added instead (Schleifer & Kandler, 1972). MurC from *Escherichia coli* (Liger et al., 1995; Falk et al., 1996; Gubler et al., 1996), *Mycobacterium tuberculosis* and *Mycobacterium leprae* (Mahapatra et al., 2000), *Pseudomonas aeruginosa* (El Zoeiby et al., 2000) and *Chlamydia trachomatis* (Hesse et al., 2003) have been purified and characterized. The enzymatic properties of MurC from *Escherichia coli* have been studied extensively. Its preferred substrate is L-Ala, although Gly and L-Ser, as well as several compounds that are structurally related to L-Ala, can be added with lower efficiencies (Liger et al., 1991, 1995; Emanuele et al., 1996). The stereospecificity is strict: D-Ala is not a substrate (Liger et al., 1995). A sequential, ordered kinetic mechanism has been demonstrated, with ATP binding first, UDP-MurNAc second and L-Ala third (Emanuele et al., 1997). The reaction is reversible and the exchange reaction is phosphate- and ADP-dependent (Liger et al., 1996). The existence of the acyl phosphate intermediate has been inferred from several studies, namely isotope exchange (Falk et al., 1996), rapid kinetics (Emanuele et al., 1997), radioactive labelling (Liger et al., 1996) and chemical trapping (Bouhss et al., 2002). Although MurC activity is dependent on the presence of a reducing agent, site-directed mutagenesis of the two cysteine residues present in the active site of MurC has shown that these two are not essential (Nosal et al., 1998). In solution, the *Escherichia coli* enzyme is present as an equilibrium between monomeric and dimeric forms; however, this has no influence on its specific activity (Jin et al., 1996).

The specificities of the MurC enzymes from species other than *Escherichia coli* have also been studied. Interestingly, MurC from *Mycobacterium tuberculosis* and *Mycobacterium leprae* have the same in vitro specificity patterns towards L-Ala and Gly; however, the amino acid found in the first position of the peptide stem is different (L-Ala for the former and Gly for the latter). This appears to be due to growth conditions (Mahapatra et al., 2000). Another
interesting case is that of *Chlamydia trachomatis*, which contains a bifunctional protein with a MurC domain and a Ddl domain (Chopra *et al.*, 1998). The MurC domain has been purified alone (Hesse *et al.*, 2003): it adds L-Ala, L-Ser and Gly with similar efficiencies, thereby preventing the deduction of the nature of the first amino acid of the putative chlamydial peptidoglycan (Moulder, 1993). The entire MurC-Ddl fusion protein has been purified recently (McCoy & Maurelli, 2005) (see ‘Formation of D-Ala-D-Ala’).

The crystal structures of the MurC enzymes from *Haemophilus influenzae* (Mol *et al.*, 2003) (Fig. 13a), *Thermotoga maritima* (Spraggon *et al.*, 2004) and *Escherichia coli* (Deva *et al.*, 2006) have been solved. Unexpectedly, in the apo structures of these last two enzymes, the N-terminal and central domains were seen to be in their closed conformation, suggesting that in its unliganded form MurC may exist in different conformations in solution (Smith, 2006).

### MurD

The second amino-acid residue of the peptide stem is in most species D-glutamic acid (Schleifer & Kandler, 1972). The few variations encountered (D-isoglutamine, threo-3-hydroxyglutamic acid) are due to modifications at a latter stage in the biosynthesis, and thus D-Glu can be said to be the amino acid substrate of MurD in all species (Vollmer *et al.*, 2008). The enzymes from *Escherichia coli* (Pratviel-Sosa *et al.*, 1991; Auger *et al.*, 1998), *Staphylococcus aureus*, *Haemophilus influenzae* and *Enterococcus faecalis* (Walsh *et al.*, 1999) have been purified and characterized. Here again, the specificity of the enzyme from *Escherichia coli* has been studied in detail (Pratviel-Sosa *et al.*, 1994). An almost exclusive preference for D-Glu was seen. Only a few closely related derivatives (homocysteic acid, 3- or 4-methyl-D-Glu, cyclopentane or cyclohexane analogues of D-Glu) were fairly good substrates. L-Glu is not a substrate (Pratviel-Sosa *et al.*, 1994); however, an N-sulfonyl derivative of L-Glu was shown to be a competitive inhibitor towards D-Glu (Kotnik *et al.*, 2007b). Among the four Mur ligases, MurD differs by its low specificity towards the UDP-precursor: whereas the UMP moiety is essential for the three other ligases, 1-phospho-MurNAc-L-Ala is a substrate for MurD (Michaud *et al.*, 1987). The reaction is reversible; however, in contrast to MurC, the exchange reaction is not ADP-dependent (Vagnay *et al.*, 1996). Chemical trapping experiments have established the existence of the acyl phosphate intermediate (Bouhss *et al.*, 2002), and the tight binding of phosphinate analogues of the high-energy tetrahedral intermediate has strongly suggested its occurrence (Tanner *et al.*, 1996; Gegnas *et al.*, 1998). MurD is capable of producing adenosine 5′-tetraphosphate, which originates from a reaction of the acyl phosphate with ATP (Bouhss *et al.*, 1999a); among the Mur ligase family, only FolC appears to share this property (Dementin, 2001; Sun *et al.*, 2001). MurD from other species have been characterized. Differences between Gram-negative and Gram-positive bacteria regarding substrate inhibition by UDP-MurNAc-L-Ala and effects of monovalent ions have been seen. These differences have been interpreted in terms of regulation of peptidoglycan synthesis (Walsh *et al.*, 1999).

The first crystal structure of a Mur ligase that was solved was that of *Escherichia coli* MurD complexed with UDP-MurNAc-L-Ala (Bertrand *et al.*, 1997). Other structures (complexes with products, metal ions or inhibitors, ‘open’ forms) were then reported (Bertrand *et al.*, 1999, 2001; Kotnik *et al.*, 2007b), with strong structural similarities between MurD and FolC being seen (Sheng *et al.*, 2000; Bertrand *et al.*, 2001). The MurD molecule contains two magnesium ions (Fig. 13b): the ‘classical’ one (Mg1) involved in ATP binding, and a second one (Mg2) involved in acyl phosphate formation. Mg2 is coordinated with two water molecules that are hydrogen bonded to a carbamoylated lysine residue (Bertrand *et al.*, 1999). The importance of this carbamoyl group, which is also present in MurE and MurF, has been investigated by chemical rescue experiments (Dementin *et al.*, 2001). Recently, a targeted molecular dynamics study has increased our understanding of the substrate binding and domain closure processes (Perdih *et al.*, 2007).

### MurE

The third amino acid of the peptide stem is generally either meso-A₂pm (most Gram-negative bacteria and Bacilli) or L-lysine (most Gram-positive bacteria), although in certain species, other amino acids are encountered (L-ornithine, LL-A₂pm, meso-lanthionine, L-diaminobutyric acid, L-homoserine, for example) (Schleifer & Kandler, 1972). In most cases, the MurE enzyme is highly specific for the relevant amino acid, incorporation of a ‘wrong’ amino acid (e.g. L-Lys in *Escherichia coli*) leading to cell lysis (Mengin-Lecreulx *et al.*, 1999). *Bacillus sphaericus* possesses two MurE enzymes: one that adds L-Lys that is active during vegetative growth, and one that adds meso-A₂pm that is active during spore cortex formation (Anwar & Vlaovic, 1986). As for the second amino acid, certain variants of the third amino acid (aminated meso-A₂pm, acetylated diaminobutyric acid) necessitate a subsequent enzymatic activity (Vollmer *et al.*, 2008).

The L-Lys-adding enzyme from *Bacillus sphaericus* and the meso-A₂pm-adding enzyme from *Escherichia coli* are strongly activated by phosphate, a product of the reaction (Anwar & Vlaovic, 1986; Michaud *et al.*, 1990). Previous studies on MurE from *Escherichia coli* have shown that a few analogues of meso-A₂pm (LL-A₂pm, lanthionine and cystathionine) can be accepted as substrates either in vitro or in genetically engineered cells (Mengin-Lecreulx *et al.*, 1988,
Cytoplasmic steps of peptidoglycan biosynthesis

1994; Michaud *et al.*, 1990; Richaud *et al.*, 1993; Auger *et al.*, 1996). The specificities of the MurE enzymes from a Gram-negative (*Escherichia coli*) and a Gram-positive (*Staphylococcus aureus*) species towards the amino acid substrate have been compared recently (Boniface, 2007). While the *Escherichia coli* enzyme has a very weak L-Lys-adding activity, the *Staphylococcus aureus* MurE is totally unable to add meso-A₂pm. Furthermore, *Escherichia coli* MurE does not accept L-Orn as a substrate, contrary to the staphylococcal enzyme, which has a weak L-Orn-adding activity.

In at least two species, the MurE enzyme appears to be devoid of strict specificity. MurE from *Bifidobacterium globosum* can incorporate two amino acids indifferently, L-Lys and L-Orn, which are both retrieved in peptidoglycan (Hammes *et al.*, 1977). MurE from *Thermotoga maritima*, a Gram-negative species for which peptidoglycan contains similar proportions of both enantiomers of lysine, but no meso-A₂pm (Huber *et al.*, 1986), can add L-Lys, D-Lys and meso-A₂pm *in vitro* with comparable efficiencies (Boniface *et al.*, 2006). In the UDP-MurNAc-tripeptide products, the D-Glu-L-Lys bond has the conventional γ → α arrangement; however, D-Lys is acylated on its ε-amino group, leading to the synthesis of a new nucleotide, UDP-MurNAc-L-Ala-D-Glu(γ → ε)D-Lys. The absence of meso-A₂pm in *Thermotoga maritima* peptidoglycan is explained by its very low intracellular pool (Boniface *et al.*, 2006).

The X-ray structure of *Escherichia coli* MurE in complex with its product, UDP-MurNAc-L-Ala-γ-D-Glu-meso-A₂pm, has been solved (Gordon *et al.*, 2001) (Fig. 13c). A binding pocket for the distal (nonreacting) site of meso-A₂pm is seen (Fig. 14a). Sequence alignments have revealed consensus sequences in the binding pockets of meso-A₂pm and L-Lys: DNPR and D(D,N)P(N,A), respectively (Dementin, 2001; Gordon *et al.*, 2001) (Fig. 14b). The main difference between these consensus sequences is the arginine residue, which is H-bonded with the carboxyl group of the distal site of meso-A₂pm (Gordon *et al.*, 2001) (Fig. 14a). In the *Thermotoga maritima* enzyme, the consensus sequence (DDPR) is undoubtedly a meso-A₂pm-adding one (Fig. 14b), thereby explaining the ‘upside-down’ binding of D-lysine. Interestingly, MurE from *Escherichia coli* produces UDP-MurNAc-L-Ala-D-Glu(γ → ε)D-Lys *in vitro*, although at a low rate (Mengin-Lecreulx *et al.*, 1994; Boniface *et al.*, 2006). The determination of the three-dimensional structure of an L-Lys-binding pocket would be of great interest; however, despite attempts with MurE from *Streptococcus pneumoniae* (Blewett *et al.*, 2004) and *Staphylococcus aureus* (Boniface, 2007), no data are currently available.

dipeptide. D-Ala-D-Ser and D-Ala-D-Lac are found in vancomycin-resistant strains (Healy *et al.*, 2000a). The enzyme from *Escherichia coli* has been purified and studied (Duncan *et al.*, 1990; Anderson *et al.*, 1996), and as for MurC, it follows a sequential, ordered kinetic mechanism. The two forms of UDP-MurNAc-tripeptide (meso-A₂pm and L-Lys) are equally effective as substrates, and a strong inhibition by excess of UDP-MurNAc-tripeptide has been seen; this effect was suppressed by the addition of 0.5 M NaCl (Anderson *et al.*, 1996). The specificity profile for the dipeptide substrate has been the subject of many studies; however, because most of them used *in vivo* systems [see references in van Heijenoort (2001)], the results obtained are indirect. When the pure enzyme has been available, it was firmly established that it has a high degree of specificity for the C-terminal amino acid (Duncan *et al.*, 1990; Bugg *et al.*, 1991). This is complementary to the specificity of D-Ala:D-Ala ligase (Ddl), which resides mainly on the N-terminal amino acid, and this constitutes a ‘double sieving’ mechanism that ensures the synthesis of UDP-MurNAc-pentapeptide ending mainly in D-Ala-D-Ala (Neuhaus & Struve, 1965; Duncan *et al.*, 1990). An interesting observation is the ability of MurF to incorporate the dipeptide 3-fluoro-D-Ala-3-fluoro-D-Ala that is synthesized by Ddl *in vivo*, thereby explaining the autoantagonistic effect of high concentrations of the antibiotic 3-fluoro-D-Ala (Kollonitsch *et al.*, 1973; Duncan *et al.*, 1990). The MurF enzyme from *Thermotoga maritima* has been isolated; not unexpectedly, it adds D-Ala-D-Ala to the L-Lys-containing UDP-MurNAc-tripeptide, but not to the D-Lys-containing nucleotide. However, the fact that the latter is a good substrate for MraY explains the incorporation of D-Lys into *Thermotoga maritima* peptidoglycan (Boniface *et al.*, 2006).

The crystal structure of the MurF apoenzyme from *Escherichia coli* has been determined; it is an open structure that is expected to undergo domain closure upon substrate binding (Yan *et al.*, 2000). Recently, three structures of the enzyme from *Streptococcus pneumoniae* were cocrystallized with sulfonamide inhibitors (Fig. 13d) and showed interdomain closure (Longenecker *et al.*, 2005; Stamper *et al.*, 2006).

### Mpl

Murein peptide ligase (Mpl) is a nonessential enzyme that is found in some Gram-negative species. The *mpl* gene was identified by a search of databases for proteins with significant homology with MurC. Mpl participates in the recycling of peptidoglycan by adding the tripeptide L-Ala-γ-D-Glu-meso-A₂pm onto UDP-MurNAc (Mengin-Lecreulx *et al.*, 1996). The enzyme from *Escherichia coli* has been purified recently and its *in vitro* substrate specificity has been studied thoroughly (Hervé *et al.*, 2007).

Fig. 14. Binding pocket for the distal site of meso-A₂pm in MurE enzymes. (a) Interactions of the A₂pm moiety of UDP-MurNAc-tripeptide with Escherichia coli MurE (PDB entry: 1e8c); Only side chains involved in hydrogen bonds (dashed lines) are shown. The green lines represent loops connecting structural elements (β19-α14, residues 414 to 416; β21-β22, residues 464 to 468). (b) Consensus sequences for the recognition of meso-A₂pm and L-lysine in various bacteria. The amino-acid numbering is that of Escherichia coli for meso-A₂pm bacteria and Staphylococcus aureus for L-lysine bacteria. The two conserved tetrapeptide motifs are shown in bold type. According to Dementin (2001), Gordon et al. (2001), and Boniface et al. (2006).

meso-A₂pm-containing tri-, tetra- and pentapeptides were accepted as substrates with high, similar catalytic efficiencies. Their L-Lys-containing counterparts were accepted with a lower (c. 500-fold), albeit still significant, efficiency. Weak additions of L-Ala (a MurC-type activity) and L-Ala-D-Glu were seen.

Although not essential, the Mpl enzyme may be interesting as a potential target for antibacterial compounds. Indeed, its broad specificity raises the possibility of incorporating toxic peptides into peptidoglycan. In this regard, Escherichia coli cells were grown in the presence of large concentrations of the synthetic tripeptide L-Ala-γ-D-Glu-L-Lys; however, no effects on cell growth or morphology were seen (Hervé et al., 2007). This disappointing result was explained by the poor uptake of the lysine-containing peptides by Escherichia coli (Le Roux et al., 1991). Nevertheless, further work is in progress to identify more permeant Mpl peptide substrates that are endowed with antibacterial activity.

### Synthesis of UDP-MurNAc and the UDP-MurNAc-peptides

The UDP-MurNAc and the UDP-MurNAc-peptides were initially isolated from Staphylococcus aureus by Park (1952). Over the next four decades, these compounds were mainly prepared from bacterial extracts [see references in Flouret et al. (1981)]. These procedures were tedious and time-consuming; moreover, for some nucleotides (e.g. UDP-MurNAc-L-Ala), the yields were low (Michaud et al., 1987). Two advances contributed to major improvements in this field: (1) the development of methods of synthesis of UDP-MurNAc and (2) the availability of pure Mur ligases in large quantities.

The first chemical synthesis of UDP-MurNAc was published 40 years ago, although the description was incomplete and the authors could not separate the α and β anomers of the product (Heymann et al., 1968). Blanot and coworkers described a synthesis starting from commercially available benzyl N-acetyl-4,6-O-benzylidene muramic acid and separated the two anomers of the product by HPLC (Blanot et al., 1994). This synthesis was improved and scaled up by Dini et al. (2000). Recently, the synthesis was further modified and optimized using different protecting and coupling strategies (Babič & Pečar, 2007; Kurosu et al., 2007). The key steps of all these synthetic schemes are the introduction of the phosphate group at the anomeric centre of a suitably protected MurNAc derivative, and the coupling of MurNAc-1-phosphate with uridine 5′-phosphomorpholidate.

The in vitro enzymatic synthesis of UDP-MurNAc from UDP-GlcNAc using MurA and MurB was described for the first time by Benson et al. (1993). The sensitivity of MurB to substrate and product inhibition and the intrinsic NADPH oxidase activity of MurB complicate this procedure. Improvements have been described, though, such as operating

Cytoplasmic steps of peptidoglycan biosynthesis

under an argon atmosphere (Reddy *et al.*, 1999) or using an *in situ* NADPH regeneration system (Liu *et al.*, 2001). An enzymatic synthesis of UDP-[¹⁴C]MurNAc has been published (Bouhss *et al.*, 2002), while using MurA alone, UDP-GlcNAc-enolpyruvate, the substrate of MurB, can be prepared (Benson *et al.*, 1993b).

The UDP-MurNAc-peptides are synthesized from UDP-MurNAc by the use of the Mur ligases that act either individually (Jin *et al.*, 1996; Bertrand *et al.*, 1997; Auger *et al.*, 1998; Raymond *et al.*, 2003) or at the same time (Reddy *et al.*, 1999; Bouhss *et al.*, 2004; Kurosu *et al.*, 2007). The enzymes from *Escherichia coli* are used generally, but recently those from *Staphylococcus aureus* (Girardin *et al.*, 2003), *Pseudomonas aeruginosa* (Paradis-Bleau *et al.*, 2006) and *Thermotoga maritima* (Boniface *et al.*, 2006; Babič *et al.*, 2007) have been used. The nonabsolute substrate specificity of the MurE enzymes has been exploited for the synthesis of nucleotides containing unusual amino acids at position 3, such as LL-A₂pm (Mengin-Lecreulx *et al.*, 1988), meso-lanthionine and L- allo-cystathionine (Mengin-Lecreulx *et al.*, 1994), L-ornithine, L-aminopimelate and amidated meso-A₂pm (Girardin *et al.*, 2003). Similarly, Schouten *et al.* (2006) used MurF to prepare UDP-MurNAc-pentapeptide analogues containing D-Cys at position 4 or 5; these derivatives were labelled with pyrene maleimide, thereby yielding fluorescent nucleotide precursors. A total chemical synthesis of UDP-MurNAc-pentapeptide (Hitchcock *et al.*, 1998; Narayan & VanNieuwenhze, 2007), as well as a chemoenzymatic synthesis of its depsipeptide (D-Ala-D-Lac-containing) analogue, have been reported (Liu *et al.*, 2001). Nucleotides modified on the uracil (Bertrand *et al.*, 1997) and MurNAc (Ueda *et al.*, 2004) moieties have also been prepared, while radioactive forms of the UDP-MurNAc-peptides have been synthesized from radiolabelled amino acids (Mengin-Lecreulx *et al.*, 1988; Michaud *et al.*, 1990; Pratviel-Sosa *et al.*, 1994; Bouhss *et al.*, 2004; Boniface *et al.*, 2006).

The analysis and purification of the UDP-MurNAc-peptides have been aided considerably by the introduction of reverse-phase HPLC (Flouret *et al.*, 1981). As well as for synthesis, this technique has found applications in the measurement of the enzymatic activity of the Mur ligases (Liger *et al.*, 1991; Auger *et al.*, 1995) and in the analysis of the intracellular pools of these nucleotide precursors (Mengin-Lecreulx *et al.*, 1982).

For the setting up of *in vitro* inhibitory assays, studies have used the *in situ* production of UDP-MurNAc-peptides from UDP-GlcNAc by MurA, MurB and the Mur ligases (Wong *et al.*, 1998; El Zoeiby *et al.*, 2001). Similarly, a MurF inhibitory assay in which the meso-A₂pm-containing or L-Lys-containing UDP-MurNAc-tripeptide is produced by Mpl *in situ* has been described (Baum *et al.*, 2006). In both cases, HPLC was used to analyse the nucleotides formed.

---

Inhibitors of the Mur ligases

Several simple compounds that are structurally related to L-Ala have been reported to be moderate inhibitors of MurC from *Escherichia coli* (Liger *et al.*, 1991, 1995). A series of phosphinate transition-state analogues of MurC were prepared and compound 7 (Fig. 15) was identified as its most potent inhibitor, with an IC₅₀ value of 49 nM. Biochemical characterization revealed that it has a mixed-type inhibition with respect to all three substrates. Any structural modification of this inhibitor significantly reduced the inhibitory activity (Marmor *et al.*, 2001; Reck *et al.*, 2001). Benzylidene rhodanines inhibited MurC with micromolar IC₅₀ values (12–27 μM): in a whole-cell assay, they were active against methicillin-resistant *Staphylococcus aureus* (MRSA), but not against *Escherichia coli* (Sim *et al.*, 2002). 2-Phenyl-5,6-dihydro-2H-thieno[3,2-c]pyrazol-3-ol derivatives with the general formula 8 showed good inhibitory activities against *Staphylococcus aureus* MurB (the best IC₅₀ values ranged from 3.6 to 24 μg mL⁻¹), MurC (IC₅₀ values between 7 and 25 μg mL⁻¹) and MurD (IC₅₀ values between 8.3 and 25 μg mL⁻¹) enzymes and promising antimicrobial activities against Gram-positive bacteria, including resistant strains. Also here, the MIC values increased to above 128 μg mL⁻¹ when these compounds were tested in the presence of BSA (Li *et al.*, 2003). A benzofuran acyl-sulfonamide derivative was discovered by AstraZeneca and shown to act competitively with ATP and UDP-MurNAc (IC₅₀ = 2.3 μM), but unfortunately it also has high-affinity binding to BSA (Ehmann *et al.*, 2004). More than 180 pulvinones [e.g. 9: IC₅₀ (MurC) = 8 μg mL⁻¹] were synthesized and evaluated as inhibitors of MurA-D. They consistently inhibited MurC (IC₅₀ values in the 1–10 μg mL⁻¹ range), and also MurA and MurB to lesser extents, while demonstrating antibacterial activities against Gram-positive bacteria, including resistant strains (Antane *et al.*, 2006). Two peptide inhibitors of MurC from *Pseudomonas aeruginosa*, with IC₅₀ values of 1.5 and 0.9 mM, were selected by phage display (El Zoeiby *et al.*, 2003b).

The effects of various analogues of D-Glu on MurD from *Escherichia coli* have been studied, and have yielded some moderate inhibitors (Pratviel-Sosa *et al.*, 1994). Many phosphonic acids and phosphinates (e.g. 10: IC₅₀ = 20 nM) have been developed so far as substrate analogues and tetrahedral transition-state analogue inhibitors of MurD, respectively (Auger *et al.*, 1995; Tanner *et al.*, 1996; Gegnas *et al.*, 1998; Snyder *et al.*, 1999; Victor *et al.*, 1999; Gobec *et al.*, 2001; Štrancar *et al.*, 2006), and quantitative structure–activity relationship (QSAR) studies have been performed for some of these (Kotnik *et al.*, 2004). From a library of N-acyl-D-Glu derivatives, several inhibitors have been identified; those containing an indole moiety appeared to be of special interest (Victor *et al.*, 1999). Recently, the

H. Barreteau et al.

![Chemical Structures](chemical_structures.png)

Fig. 15. Formulae of inhibitors of the Mur ligases.

high-resolution crystal structures of MurD in complexes with $N$-sulfonyl-$D$-Glu ($IC_{50} = 280 \, \mu M$) and $N$-sulfonyl-$L$-Glu ($IC_{50} = 710 \, \mu M$) (11) have been solved (Fig. 16). The binding modes of these inhibitors have also been characterized kinetically (Kotnik *et al.*, 2007b). Using a *de novo* structure-based molecular design, a series of macrocyclic inhibitors 12 were developed, which showed $IC_{50}$ values between 0.7 and 5.1 $\mu M$ (Horton *et al.*, 2003). In addition, peptide inhibitors of MurD from *Pseudomonas aeruginosa* (best $IC_{50}$ value: 4 $\mu M$) have been obtained by the screening of phage display libraries using competitive biopanning approaches (Paradis-Bleau *et al.*, 2006).

Analogues of A$_2$pm (90% inhibition at 5 mM compound) and some $N$-acyl-dipeptide derivatives ($IC_{50} = 0.6–10 \, \text{mM}$) have shown moderate inhibitory activities on MurE from *Escherichia coli* (Abo-Ghalia *et al.*, 1985, 1988; Michaud *et al.*, 1990; van Assche *et al.*, 1991; Le Roux *et al.*, 1992; Auger *et al.*, 1996). Phosphinates designed as transition-state analogues of MurE inhibited the *Escherichia coli* enzyme in the micromolar range; the best of these (13) had an $IC_{50}$ value of 1.1 $\mu M$, whereas the derivative devoid of the

![Binding Mode](binding_mode.png)

Fig. 16. Binding mode of an $N$-sulfonyl-$D$-Glu derivative in the active site of *Escherichia coli* MurD (PDB entry: 1jff). The inhibitor (compound 11, D isomer, in Fig. 15) is shown in magenta. Hydrogen bonds are indicated by dashed lines. The green lines represent sequence elements not directly involved in the interaction.

Cytoplasmic steps of peptidoglycan biosynthesis

UMP part was a poor inhibitor (IC₅₀ = 700 μM) (Zeng *et al.*, 1998). In addition, some phosphinates and β-sulfonamido-peptides designed as transition-state analogue inhibitors of MurD can inhibit MurE from *Staphylococcus aureus*, presumably by acting as substrate analogues (Humljan *et al.*, 2006; Štrancar *et al.*, 2007).

The first inhibitors of MurF were pseudo-tripeptide and pseudo-tetrapeptide aminoalkylphosphinic acids of the general structure X-Lys-Ψ(PO₂H)-Gly-Ala that had been synthesized as transition-state analogues. They acted as reversible competitive inhibitors, with \( K_i \) values in the range of 200–700 μM (Miller *et al.*, 1998). With the affinity selection screening, two small-molecule MurF leads (e.g. 14: IC₅₀ = 1 μM) were discovered by Abbott Laboratories and cocrystallized with the enzyme (Gu *et al.*, 2004; Longenecker *et al.*, 2005; Comess *et al.*, 2006) (Fig. 13d). Both these were bound in the substrate-binding region and induced domain closure. After the structure-based lead optimization, a series of potent inhibitors were obtained, culminating in compound 15 (IC₅₀ = 22 nM), although none of these exhibited significant antibacterial activities even in the presence of bacterial cell permeabilizers (Gu *et al.*, 2004; Stamper *et al.*, 2006). Recently, three QSAR models were constructed to further facilitate the search for new inhibitors with extensive physicochemical properties (Kong *et al.*, 2007). Using an Mpl-based assay, a thiazolylaminopyrimidine series of MurF inhibitors with IC₅₀ values as low as 2.5 μM have also been identified (Baum *et al.*, 2006).

### Side pathways

Besides the hexoses and nucleotides described in the previous sections, the cytoplasmic steps of peptidoglycan biosynthesis involve a certain number of substrates (Fig. 1). Most participate in many metabolic pathways (acetyl-CoA, UTP, PEP, NADPH, ATP and L-amino acids), but some are more specific (*meso*-A₂pm, D-Ala, D-Ala-D-Ala and D-Glu). Because *meso*-A₂pm is the last intermediate in the L-Lys biosynthesis pathway, which has been the subject of several reviews (Patte, 1983; Cox, 1996; Born & Blanchard, 1999; Cox *et al.*, 2000; Hutton *et al.*, 2007), it will not be considered here. Therefore, only the formation of D-Ala-D-Ala (and its precursor D-Ala) and D-Glu will be described.

### Formation of D-Ala-D-Ala

Essentially found in peptidoglycan, D-Ala is also present in lipoteichoic acids of Gram-positive organisms (Volkman *et al.*, 2001). It is produced from L-Ala through the action of a pyridoxal 5′-phosphate (PLP)-dependent enzyme, alanine racemase. Some organisms contain only one alanine racemase, while others have two, encoded by the *alr* and *dadX* genes. The latter group includes *Escherichia coli* (Wild

et al., 1985), *Salmonella typhimurium* (Wasserman *et al.*, 1984; Esaki & Walsh, 1986) and *Pseudomonas aeruginosa* (Strych *et al.*, 2000). Expression of the *alr* gene is constitutive and provides the D-Ala that is necessary to maintain cell growth, while the *dadX*-encoded racemase is inducible and required only when L-Ala is used as a carbon and energy source. While Alr from *Enterococcus faecalis* (Badet & Walsh, 1985), *Salmonella typhimurium* (Esaki & Walsh, 1986), *Thermus thermophilus* (Seow *et al.*, 2000), *Shigella* sp. (Yokoigawa *et al.*, 2001) and *Helicobacter pylori* (Saito *et al.*, 2007) have been described as monomers, those from *Escherichia coli*, *Pseudomonas aeruginosa* (Strych & Benedik, 2002), *Bifidobacterium bifidum* (Yamashita *et al.*, 2003), *Corynebacterium glutamicum* (Oikawa *et al.*, 2006), *Acidiphilium organovorum* (Seow *et al.*, 1998), *Mycobacterium* sp. (Strych *et al.*, 2001) and *Bacillus stearothermophilus* (Inagaki *et al.*, 1986) are dimers that are formed between identical polypeptide chains of c. 40 kDa. Structural studies on Alr from *Bacillus stearothermophilus* (Shaw *et al.*, 1997; Morollo *et al.*, 1999) and *Mycobacterium tuberculosis* (LeMagueres *et al.*, 2005) have established that each monomer consists of two different domains: an N-terminal domain made up of an α/β-barrel, and a C-terminal one primarily made up of β-strands. Each subunit contains one PLP molecule (Fig. 17). Kinetic analyses (Sawada *et al.*, 1994) and X-ray crystallographic studies (Shaw *et al.*, 1997; Watanabe *et al.*, 2002) have revealed that the Alr reaction proceeds via a two-base mechanism (Fig. 18). First, the PLP bound to the active-site residue Lys39 (*Bacillus stearothermophilus* numbering) reacts with L-Ala to form an external Schiff base through transaldimination. The reaction continues with the

![Three-dimensional structure of the dimer of alanine racemase from Mycobacterium tuberculosis (PDB entry: 1xfc). Monomers are shown in blue and green. PLP is shown in magenta. The red spheres represent the side-chain of Lys42 (equivalent to Lys39 of Alr from Bacillus stearothermophilus).](image.png)

Fig. 17. Three-dimensional structure of the dimer of alanine racemase from *Mycobacterium tuberculosis* (PDB entry: 1xfc). Monomers are shown in blue and green. PLP is shown in magenta. The red spheres represent the side-chain of Lys42 (equivalent to Lys39 of Alr from *Bacillus stearothermophilus*).

Fig. 18. Two-base mechanism of the reaction catalysed by alanine racemase. Lys39 and Tyr265′ (Bacillus stearothermophilus numbering) act as the catalytic bases abstracting the α-hydrogen from D-Ala and L-Ala, respectively. The reaction proceeds through a planar carbanionic intermediate (top, right and bottom, left). Owing to the participation of the carboxylate of Ala, Lys39 and Tyr265′ remain unionized throughout the catalytic process. According to Watanabe et al. (2002).

abstraction of the Cα proton of the external aldimine (L-Ala- PLP adduct) by Tyr265 from the second monomer. In the next step, the planar carbanion intermediate is reprotonated by Lys39 from the opposite side to yield the D-enantiomer. Finally, the internal aldimine between Lys39 and PLP is formed again, displacing D-Ala from its covalent linkage to the cofactor. Conversion of D- into L-enantiomer proceeds in the opposite manner. The importance of the catalytic bases Lys39 and Tyr265 has been revealed by several studies using mutant enzymes (Watanabe et al., 1999a,b). Replacing Tyr265 by Ala in Alr from Bacillus stearothermophilus resulted in a novel aldolase activity (Seebeck & Hilvert, 2003). Another interaction of great importance for the racemization process is the direct hydrogen bond between the unproto- nated pyridine nitrogen of PLP and Arg219 (Shaw et al., 1997; Morollo et al., 1999). In other PLP-dependent en- zymes, but not with Alr, the pyridine nitrogen of the cofactor is in the protonated form and acts as an effective electron sink, stabilizing the carbanion intermediate by forming a

quinonoid species. The unusual interaction between the positively charged guanidino group of Arg219 and the pyridine nitrogen of PLP in Alr was explored by analysis of the R219K, R219A and R219E mutants (Sun & Toney, 1999). It was thus demonstrated that for efficient catalysis, a positively charged residue is required in this position. Molecular dynamics simulations have revealed that the enhancement of the carbon acidity of the α-amino acid by PLP via unprotonated pyridine is mainly due to solvation effects, in contrast to the intrinsic electron-withdrawing stabilization by the pyridinium ion to form a quinonoid intermediate (Major & Gao, 2006).

Cycloserine is a naturally occurring suicide substrate of many PLP-dependent enzymes; in this respect, D-cycloserine (Fig. 19) inactivates Alr in a time-dependent manner (Wang & Walsh, 1978). After the initial formation of an external aldimine between cycloserine and PLP, a proton is trans- ferred from the C-2 of the substrate to the C-4′ of the cofactor, resulting in the formation of a ketimine species.

Cytoplasmic steps of peptidoglycan biosynthesis

This ketimine proceeds through a second prototropic shift, forming a stable isoxazole (Fig. 19). Together with kinetic studies, the crystal structures of L- and D-cycloserine-inactivated Alr (Fenn et al., 2003) and the Y265F mutant enzyme (Fenn et al., 2005) from *Bacillus stearothermophilus* have emphasized the importance of the catalytic bases Tyr265′ and Lys39 for the inactivation process.

A series of N(2)-substituted derivatives of compound 16 (Kim et al., 2003b) (Fig. 20) and five- and six-membered heterocycles (Kim et al., 2003a) were prepared and evaluated for inhibitory activities against Alr from various bacterial species, as well as in growth inhibition assays. Some of the heterocycles had moderate inhibitory activities against Alr from *Escherichia coli*, *Staphylococcus aureus*, *Pseudomonas aeruginosa* and *Mycobacterium tuberculosis*. Besides D-cycloserine, several other compounds are irreversible time-dependent mechanism-based inactivators (suicide substrates) of Alr, like O-carbamyl-D-serine (Wang & Walsh, 1978), β-chloro- and β-fluoro-D-alanine (Wang & Walsh, 1978; Badet et al., 1984), β,β-difluoro- (Wang & Walsh, 1981) and β,β,β-trifluoroalanine 17 (Faraci & Walsh, 1989) and 3-halovinylglycines 18 (Thornberry et al., 1987, 1991). Other alanine analogues that are inhibitors of Alr have been reviewed by Neuhaus & Hammes (1981). To improve in vivo antimicrobial activities, several di- and tripeptides containing β-chloro-L-Ala (Cheung et al., 1983, 1986; Boisvert et al., 1986; Le Roux et al., 1991) and halovinylglycine (Patchett et al., 1988) were synthesized as transport systems for the intracellular delivery of potentially bactericidal amino acids. L-Norvalyl-L-chlorovinylglycine showed good activity against Gram-positive organisms, including methicillin-resistant *Staphylococcus* species (Patchett et al., 1988). Furthermore, (β-chloro-, (β-β-dichloro- and (β-β-β-trichloro-α-aminoethy) phosphonic

**Fig. 19. Mechanism of inactivation of alanine racemase by D-cycloserine. Lys39 and Tyr265′ (*Bacillus stearothermophilus* numbering) catalyse two proton transfers leading successively to the ketimine intermediate and the final isoxazole derivative. PMP, pyridoxamine phosphate. According to Fenn et al. (2003).**

**Fig. 20. Formulae of inhibitors of alanine racemase and d-Ala-d-Ala ligase.**

acids (Vo-Quang *et al.*, 1986a) and the phosphonic analogue of vinylglycine, DL-(1-amino-2-propenyl)phosphonic acid (Vo-Quang *et al.*, 1986b), have been evaluated against Alr and Ddl from *Pseudomonas aeruginosa* and *Enterococcus faecalis*. The monochloro, dichloro and vinylglycine derivatives of Ala-P exhibited strong inhibition of Alr from both species tested, whereas only the Ddl from *Enterococcus faecalis* was inhibited by these compounds.

The phosphonic analogue of alanine, L-1-aminoethyl-phosphonic acid 19 (L-AlaP) (Atherton *et al.*, 1979b), is a time-dependent irreversible inactivator of Gram-positive (*Bacillus*, *Staphylococcus* and *Streptococcus*), but not Gram-negative (*Escherichia coli*, *Salmonella* and *Pseudomonas*) racemases (Badet & Walsh, 1985; Badet *et al.*, 1986). It was used as the phosphonodipeptide alaphosphin (L-Ala-L-AlaP), which can be transported by bacterial cell wall permeases and then hydrolysed to L-Ala and L-AlaP (Allen *et al.*, 1978; Atherton *et al.*, 1979a). The formation of an AlaP-PLP Schiff base linkage was shown by solid-state ${}^{15}$N-NMR of the [${}^{15}$N]AlaP-racemase complex (Copié *et al.*, 1988) and the cocrystal structure of L-AlaP with Alr (Stamper *et al.*, 1998). The latter showed an inappropriate orientation of the external aldimine for efficient Cα proton abstraction and revealed interactions of the phosphonate group with putative catalytic residues, thereby rendering them unavailable for catalysis. Several phosphonodipeptides containing L-amino-L-methylethanephosphonic (Zboinska *et al.*, 1990), L-amino-nomethylphosphonic (Atherton *et al.*, 1982) and L-amino-cyclopropanephosphonic acid 20 (Erion & Walsh, 1987) were generally less potent inhibitors of the Alr and Ddl enzymes than phosphonodipeptides based on AlaP.

The replacement of the AlaP phosphonate with a boronic acid group led to the alanine analogue (1-aminoethyl)boronic acid 21, which is a slow-binding, time-dependent inhibitor of Alr (*Bacillus stearothermophilus*) and Ddl (*Salmonella typhimurium*). The inhibition is most probably due to the formation of a tetrahedral boronate anion that acts as a transition-state analogue (Duncan *et al.*, 1989). On the other hand, compounds mimicking the PLP-Ala complex were inactive (Leung *et al.*, 1985).

A structure-based strategy for the identification of novel *Bacillus stearothermophilus* Alr inhibitors with a dynamic receptor-based pharmacophore model was developed using the LigBuilder programme (Mustata & Briggs, 2002); however, these compounds have not yet been evaluated for their inhibitory activities.

The condensation of two molecules of D-Ala that were formed previously by alanine racemase is catalysed by a specific ATP-dependent D-Ala:D-Ala ligase (Ddl). Early kinetic and specificity studies of the reaction were carried out with the ligase purified from *Enterococcus faecalis* (Neuhaus, 1962a): they provided evidence for two D-Ala-binding sites that have different specificity patterns and

Michaelis constants (Neuhaus, 1962b). The binding site for the N-terminal D-Ala, designated as the donor site, is highly specific for D-Ala whereas the C-terminal D-Ala-binding site, the acceptor site, is less specific and accepts a variety of D-amino acids. Ddl enzymes from *Oceanobacillus iheyensis*, *Synechocystis sp.* (Sato *et al.*, 2005) and *Thermotoga maritima* (Sato *et al.*, 2006) were also isolated and characterized for their substrate specificities. Recently, the entire MurC-Ddl fusion protein from *Chlamydia trachomatis* (see ‘Biosynthesis of the UDP-MurNAc-peptides’) was purified; the presence of the MurC domain appeared to be required for Ddl activity (McCoy & Maurelli, 2005). The existence of the two distinct genes *ddlA* and *ddlB* in *Escherichia coli* (Zawadzke *et al.*, 1991; Al-Bar *et al.*, 1992) and *Salmonella typhimurium* (Daub *et al.*, 1988) was demonstrated. Both genes were cloned and overexpressed, and their products have been purified. Despite a difference in size, both enzymes show very similar kinetic characteristics and substrate specificities.

The kinetic mechanism, which has been well studied with the enzyme from *Salmonella typhimurium*, has been shown to be an ordered ter–ter reaction with ATP as the first substrate to bind, and ADP the last product off. The reaction is reversible; however, as the exchange reaction is not strictly ADP-dependent, this can argue in favour of some randomness in the kinetic mechanism (Mullins *et al.*, 1990).

The reaction starts with the attack on the first D-Ala by the γ-phosphate of ATP, to give an acylphosphate. This is followed by an attack by the amino group of the second D-Ala to yield a tetrahedral intermediate, which collapses into D-Ala-D-Ala and P${}_{i}$ (Mullins *et al.*, 1990; Healy *et al.*, 2000b). In this respect, the mechanism is similar to that of the Mur ligases (see ‘Biosynthesis of the UDP-MurNAc-peptides’). Ddl is strongly inhibited by its reaction product D-Ala-D-Ala (Mullins *et al.*, 1990).

Despite the similarity of the Ddl and Mur ligase reactions, Ddl does not belong to the Mur ligase family. Sequence alignments and crystal structures have revealed that it belongs to the ATP-grasp family, which is composed of highly diverse enzymes that catalyse the ATP-dependent ligation of a carboxyl group to an amino or imino nitrogen, a hydroxyl oxygen or a thiol sulphur (Galperin & Koonin, 1997; Kobayashi & Go, 1997) (Fig. 21).

The most important inhibitor of Ddl is undoubtedly a structural analogue of D-Ala, D-cycloserine (formula in Fig. 19) (Strominger *et al.*, 1960; Neuhaus & Lynch, 1964). A series of phosphinates [e.g. 22: IC${}_{50}$ (*Streptococcus faecalis*) = 35 μM] (Fig. 20), phosphonates and phosphonamidates have been developed as transition-state analogue inhibitors or as analogues of D-alanyl phosphate (Lacoste *et al.*, 1979; Parsons *et al.*, 1988; Chakravarty *et al.*, 1989; Lacoste *et al.*, 1991; Ellsworth *et al.*, 1996). It was shown that transition-state mimetics can be phosphorylated by Ddl and

Cytoplasmic steps of peptidoglycan biosynthesis

![image](https://i.imgur.com/your_image_url.png)

Fig. 21. Three-dimensional structure of the *Escherichia coli* DdlB complexed with ADP and the phosphorylated derivative of phosphinate 22 in Fig. 20 (PDB entry: 2dln). The three domains characteristic of the ATP-grasp fold are shown in green, yellow and blue. Ligands are shown in magenta. Two Mg²⁺ ions cocrystallized with DdlB are depicted as black spheres.

inhibit the reaction by their tight binding to the enzyme after this phosphorylation (Duncan & Walsh, 1988; McDermott et al., 1990). Although their antibacterial activities are low, they enabled the crystallographic determination of complexes of *Escherichia coli* DdlB with ADP/phosphorylated phosphinate (Fan et al., 1994) (Fig. 21) and the *Escherichia coli* Y216F DdlB mutant with ADP/phosphorylated phosphinate (Fan et al., 1997). Recently, two patent applications have been filed describing structure-based drug discovery methods for identifying inhibitors that target the ATP-binding site as well as new heterocyclic inhibitors (Moe et al., 2003; Navia et al., 2003). Using a *de novo* structure-based molecular design, cyclopropane derivative 23 [Kᵢ (*Escherichia coli*) = 12.5 μM] was developed as an inhibitor of DdlB (Besong et al., 2005). An allosteric inhibitor of Ddl from *Staphylococcus aureus* (24: Kᵢ = 4 μM) was discovered by high-throughput screening, and it was cocrystallized with the enzyme (Liu et al., 2006). Recently, diazenedicarboxamides [e.g. 25: IC₅₀ (*Escherichia coli*) = 15 μM] were described as inhibitors of DdlB from *Escherichia coli* (Kovač et al., 2007). In addition, the crystal structure of the apo form of Ddl from *Thermus caldophilus* has been solved, providing an insight into the substrate-induced conformational changes, which could be important for inhibitor design (Lee et al., 2006a).

In vancomycin-resistant strains, the D-Ala-D-Ala termini are replaced by either D-Ala-D-Lac or by D-Ala-D-Ser. The affinities of these alternate termini for the antibiotic vancomycin are three and one orders of magnitude, respectively, lower than that of the conventional D-Ala-D-Ala termini. Depsipeptide D-Ala-D-Lac and dipeptide D-Ala-D-Ser are synthesized by enzymes known as VanA-G according to the

vancomycin-resistance phenotype considered. Together with Ddl, the Van enzymes constitute a superfamily of D-Ala:D-X ligases (X = D-Ala, D-Lac or D-Ser) that share a common reaction mechanism and the ATP-grasp three-dimensional structure. Differences in the specificity of the acceptor site have been explained through their pH profiles and active-site residues [see Healy et al. (2000a) for a review]. The crystal structures of D-Ala:D-Lac ligases from *Leuconostoc mesenteroides* and *Enterococcus faecium* are also available (Kuzin et al., 2000; Roper et al., 2000).

# Formation of D-glutamic acid

In addition to its occurrence in peptidoglycan (Vollmer et al., 2008), D-glutamic acid is a key component of capsular poly-γ-glutamate of some bacterial genera (Ashiuchi & Misono, 2002; Candela & Fouet, 2006). Two distinct enzymes have been identified for the formation of D-glutamate: glutamate racemase (MurI) and D-amino acid aminotransferase (D-AAT).

Glutamate racemase catalyses the interconversion of D- and L-enantiomers of glutamate (Doublet et al., 1993). The murI genes (also called racE) from various bacteria have been cloned. Contrary to the situation in most organisms (including *Escherichia coli*) that contain only one glutamate racemase gene [see references in Dodd et al. (2007)], the *Bacillus subtilis* and *Bacillus anthracis* genomes have two genes that have been named racE1 and racE2 for the former species, and racE (also called glr) and yrpC for the latter (Ashiuchi et al., 1999; Kimura et al., 2004; Shatalin & Neyfakh, 2005).

The MurI racemases from c. 10 species, including *Bacillus pumilus*, *Pediococcus pentosaceus*, *Lactobacillus fermenti* and *Escherichia coli*, have been purified [see references in Dodd et al. (2007)], as well as the RacE1 and RacE2 enzymes from *Bacillus anthracis* (Dodd et al., 2007; May et al., 2007) and the RacE and YrpC enzymes from *Bacillus subtilis* (Ashiuchi et al., 1998, 1999). Thorough biochemical studies have been performed for some of them. The *Escherichia coli* MurI racemase requires UDP-MurNAc-L-Ala for its activation, and it has been shown that this nucleotide precursor exerts its effects by increasing both the substrate-binding affinity and the turnover rate of the enzyme (Doublet et al., 1994; Ho et al., 1995). Therefore, the formation of D-glutamic acid is regulated by UDP-MurNAc-L-Ala and thus adjusted to the requirement of peptidoglycan synthesis. Moreover, an excessive racemization of the large intracellular pool of L-glutamic acid is avoided (Doublet et al., 1993, 1994). This activating mechanism appears to be unique for *Escherichia coli* because glutamate racemases from other bacteria are not activated by UDP-MurNAc-L-Ala (Yoshimura & Esaki, 2003; Lundqvist et al., 2007). It was hypothesized that a 21-amino-acid extension at the N-terminus of the *Escherichia coli* enzyme,
which is the major difference between *Escherichia coli* and other bacteria, was responsible for the nucleotide-mediated activation (Doublet *et al.*, 1994; Ho *et al.*, 1995); however, the N-terminal-truncated *Escherichia coli* enzyme retained the activation, although to lesser extents (Ho *et al.*, 1995; Doublet *et al.*, 1996). The glutamate racemase from *Pediococcus pentosaceus* is inhibited by haemin due to the formation of a stoichiometric complex (Choi *et al.*, 1994). Recently, it was shown that glutamate racemases from *Escherichia coli*, *Bacillus subtilis* (YrpC, but not RacE) and *Mycobacterium tuberculosis* inhibit DNA gyrase activity; in the case of *Escherichia coli*, the presence of the nucleotide precursor was required (Ashiuchi *et al.*, 2002, 2003; Sengupta *et al.*, 2006).

Glutamate racemase belongs to a small family of amino acid racemases and epimerases (Tanner, 2002; Yoshimura & Esaki, 2003). While most amino acid racemases use PLP for abstraction of the α-proton, glutamate racemase operates without any cofactor or metal ion (Gallo *et al.*, 1993; Glavas & Tanner, 2001). Instead, a ‘two-base’ mechanism that involves two cysteine residues has been suggested (Fig. 22). A thiolate from one of the Cys residues promotes the deprotonation of the substrate at the α position, to yield a carbanionic intermediate; then reprotonation by the other Cys residue occurs in the opposite stereochemical sense (Gallo *et al.*, 1993; Glavas & Tanner, 1999, 2001). The catalytic mechanism of glutamate racemases from *Aquifex pyrophilus*, *Lactobacillus fermenti* and *Bacillus subtilis* has been investigated closely (Hwang *et al.*, 1999; Glavas & Tanner, 2001; Ruzheinikov *et al.*, 2005). Two possibilities, which differ in the protonation states of the catalytic bases prior to the initial deprotonation, have been proposed (Glavas & Tanner, 2001; Ruzheinikov *et al.*, 2005). Furthermore, stereochemistry, substrate ligation and active-site protonation states have been investigated by molecular dynamics simulations (Möbitz & Bruice, 2004; Puig *et al.*, 2005). Recently, the results from computational simulations on a quantum mechanical/molecular mechanical potential energy surface (Puig *et al.*, 2006) indicated two possible roles for Murl as a catalyst and supported the mechanistic proposal by Rios *et al.* (2000) for the PLP-independent amino acid racemases.

Several crystal structures of bacterial glutamate racemases have been reported (Hwang *et al.*, 1999; Ruzheinikov *et al.*, 2005; Kim *et al.*, 2007; Lundqvist *et al.*, 2007; May *et al.*, 2007) (Fig. 23). Along with analytical methods such as size-exclusion chromatography or equilibrium ultracentrifugation, structure examination has led to a classification of glutamate racemases according to their quaternary structure (Lundqvist *et al.*, 2007; May *et al.*, 2007): monomer (*Escherichia coli*, *Lactobacillus fermenti*, *Bacillus subtilis* YrpC) (Fig. 23a), monomer–dimer equilibrium (*Aquifex pyrophilus*, *Bacillus anthracis* RacE1 and *Bacillus subtilis* RacE) or dimer (*Helicobacter pylori*, *Enterococcus faecalis*, *Enterococcus faecium*, *Staphylococcus aureus* and *Bacillus anthracis* RacE2). Among dimers, a tail-to-tail orientation (i.e. with the active sites opposed and fully exposed to the solvent) is the general case; however, the *Helicobacter pylori* enzyme appears as a head-to-head dimer (i.e. with the active sites in close proximity in a face-to-face orientation) (Fig. 23b).

Owing to the essential role of glutamate racemase for the viability of bacteria, this enzyme is an attractive target for the development of antibacterial agents. Several inhibitors of glutamate racemase have been reported to date (Fig. 24). Ashiuchi *et al.* (1993) showed that L-serine O-sulphate 26 behaves as a suicide substrate of the enzyme from *Pediococcus pentosaceus*; a mechanism by which an α-aminoacrylate intermediate reacts with a nucleophilic group of the enzyme was proposed. Tanner and coworkers synthesized aziridino glutamate 27, which irreversibly inactivates *Lactobacillus fermenti* glutamate racemase through the alkylation of a Cys residue in the active site (Tanner & Miao, 1994), and D-N-hydroxyglutamate 28, which was shown to be a competitive inhibitor (*K*~i~ = 56 μM) (Glavas & Tanner, 1997). The research group at Eli Lilly discovered the first potent competitive inhibitors that showed antibacterial activity: 4-substituted D-Glu analogues with various aryl and hetero-aryl substituents (e.g. 29), with IC~50~ values as low as 30 nM and a good correlation between inhibitory and antibacterial activities against *Streptococcus pneumoniae* (de Dios *et al.*, 2002). Pyrazolopyrimidinedione derivative 30 was discovered from a high-throughput screen by

![Figure 22](https://i.imgur.com/1234567.png)

**Fig. 22.** Reaction mechanism of the Murl enzyme. The enzyme employs two active-site cysteine residues as acid/base catalysts. A thiolate from one of the cysteines abstracts the α-proton, forming a carbanionic intermediate (central formula). The other cysteine thiol delivers a proton to the opposite face of the intermediate. According to Gallo *et al.* (1993).

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved
Cytoplasmic steps of peptidoglycan biosynthesis

Fig. 23. Three-dimensional structures of two glutamate racemases. (a) *Escherichia coli* MurI (PDB entry: 2jfn) The protein appears as a monomer. Substrate D-Glu and activator UDP-MurNAc-L-Ala are shown in magenta. (b) *Helicobacter pylori* MurI (PDB entry: 2jfx). The protein appears as a head-to-head dimer. The two monomers are shown in green and blue. Substrate D-Glu is shown in magenta. Glu151 and Lys117' side-chains are shown in red; the arrow indicates a putative intermonomer salt bridge contact between them.

![Figure 24](https://i.imgur.com/yourimage.png)

Fig. 24. Formulae of inhibitors of MurI.

AstraZeneca; it behaved as an uncompetitive inhibitor of *Helicobacter pylori* MurI ($\mathrm{IC}_{50}=1.4 \mu \mathrm{M}$) and prevented the growth of *Helicobacter pylori* cells ($\mathrm{MIC}=4 \mu \mathrm{g} \mathrm{mL}^{-1}$) (Lundqvist et al., 2007). Kim et al. (2000) isolated peptide ligands from a random phage display library that inhibit *Escherichia coli* glutamate racemase activity. Among these, the peptide sequence His-Pro-Trp-His-Lys-Lys-His-Pro-Asp-Arg-Lys-Thr has the highest affinity for the enzyme ($\mathrm{IC}_{50}=160 \mu \mathrm{M}$); however, no bactericidal or antibacterial effects were seen.

D-Amino acid aminotransferase (D-AAT) yields D-glutamate and pyruvate from D-alanine and α-ketoglutarate (Fig. 25) (Fotheringham et al., 1998), and D-AAT activity has been

FEMS Microbiol Rev 32 (2008) 168–207

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved

D-alanine + α-ketoglutarate $\xrightarrow{\text{D-AAT}}$ D-glutamate + Pyruvate

![Formulae of D-AAT inhibitors](https://i.imgur.com/1234567.png)

Fig. 25. Equation of the D-AAT reaction (top) and formulae of D-AAT inhibitors (bottom).

reported for various Gram-positive bacteria: *Bacillus* (Tanizawa et al., 1989; Taylor & Fotheringham, 1997; Fotheringham et al., 1998; Peisach et al., 1998; Berger et al., 2003), *Staphylococcus* (Pucci et al., 1995) and *Listeria* (Thompson et al., 1998). Recently, the D-AATs from *Geobacillus sp.* were cloned and characterized for their genetic, catalytic and structural aspects (Lee et al., 2006b). The three-dimensional structures of various forms of D-AAT from *Bacillus sp.* YM-1 have been solved (Sugio et al., 1995, 1998; Peisach et al., 1998; van Ophem et al., 1999) and the pathway of the enzymatic reaction has been described in structural terms. Tyr30, Arg98 and His100 are the major residues relating to the stereospecificity of D-AAT, whereby it specifically carries out a transamination reaction using PLP as a cofactor and a catalytic lysine as a general base/acid. The accepted pathway for transamination, which has been characterized for a number of PLP-dependent enzymes, proceeds through a ping-pong kinetic mechanism.

A number of compounds have been reported to be inhibitors of D-AAT activity (Neuhaus & Hammes, 1981; Axelsson et al., 1994). β-Chloro-D-alanine 31 and D-serine-O-sulphate 32 (Fig. 25) are suicide substrates; they are converted into a putative aminoacrylate intermediate, which either reacts with PLP to form a stable adduct or is protonated and hydrolysed to yield pyruvate (Axelsson et al., 1994; Adams et al., 2005).

Both the transaminase and the glutamate racemase routes have been reported for some bacterial species: *Staphylococcus haemolyticus* (Pucci et al., 1995), *Bacillus sphaericus* (Fotheringham et al., 1998) and *Bacillus subtilis* (Wipat et al., 1996). In other organisms, such as *Escherichia coli*, *Lactobacillus sp.* and *Pediococcus sp.*, only the glutamate racemase activity exists (Nakajima et al., 1986; Doublet et al., 1993; Gallo & Knowles, 1993).

### Genetic organization

The organization of the genes involved in the biosynthetic process reviewed here is well established in *Escherichia coli*, where it has been facilitated by the isolation of thermosensitive mutants. Most of the genes are clustered in two regions, known as *mra* and *mrb* (murein regions A and B, respectively) (Miyakawa et al., 1972). The *mra* (or *dcw*, division cell wall) region is located at 2 min on the chromosome and contains genes involved either in peptidoglycan synthesis or in cell division in the order: *mraZ-mraW-ftsL-ftsI-murE-murF-mraY-murD-ftsW-murG-murC-ddlB-ftsQ-ftsZ-envA* (Mengin-Lecreulx et al., 1989). The first nine genes are under the control of the $P_{mra}$ promoter (Mengin-Lecreulx et al., 1998). The *mrb* region is located at 90 min and contains the *murI* and *murB* genes. The *glmM* and *murA* genes are in the 69 min region, and the *glmU* and *glmS* genes in the 83 min region. The *ddlA*, *dadX*, *alr* and *mpl* genes are located at 8.5, 27, 92 and 96 min, respectively (Berlyn et al., 1996). The *mra* region has been located in other bacteria such as *Bacillus subtilis*, *Staphylococcus aureus*, *Enterococcus faecalis* and *Pseudomonas aeruginosa* (Daniel & Errington, 1993; Pucci et al., 1997; El Zoeiby et al., 2000; Azzolina et al., 2001); in the last of these, its organization is exactly the same as in *Escherichia coli*.

Peptidoglycan integrity being necessary for the bacterial cell's survival, genes involved in its biosynthesis must be essential provided no isogenes are present. Data about the essential character of the genes coding for the cytoplasmic enzymes can be found either in former literature [see references in van Heijenoort (2001)], or inferred from the recent systematic construction of single-gene knock-out mutants of *Escherichia coli* (Baba et al., 2006). In this organism, the *glmS*, *glmM*, *glmU*, *murA*, *murB*, *murC*, *murD*, *murE*, *murF* and *murI* genes are essential. Isogenes (*ddlA/ddlB*, *alr/dadX*) can be individually inactivated, showing that the remaining isozyme is sufficient to sustain growth (Wild et al., 1985; Zawadzke et al., 1991). As other genes involved in the recycling pathway, the *mpl* gene is dispensable for *Escherichia coli* growth (Mengin-Lecreulx et al., 1996).

A systematic gene inactivation was also performed in *Bacillus subtilis* (Kobayashi et al., 2003). The *glmS*, *ybbT* (*glmM*), *gcaD* (*glmU*), *murA*, *murB*, *murC*, *murD*, *murE*, *alr*, *ddl* and *racE* genes could not be inactivated. As far as *alr* and *ddl* are concerned, this is explained because *Bacillus subtilis*, contrary to *Escherichia coli*, contains only one alanine racemase and one Ddl. The *racE* gene was claimed to be essential as no knockout mutants could be found on solid medium, suggesting that *yrpC* cannot take over. However, the disruption of the *racE* gene has been described recently (Ashiuchi et al., 2007); the RacE-less mutants obtained can grow with a reduced rate in liquid medium even in the absence of exogenous D-glutamate. The authors concluded that the RacE activity, although important for ensuring maximum growth rate, is dispensable, and that the YrpC racemase probably operates as an anaplerotic enzyme for RacE (Ashiuchi et al., 2007).

Although deletion of both *murA1* and *murA2* genes in *Streptococcus pneumoniae* is lethal, each can be inactivated
individually without effect on cell growth (Du *et al.*, 2000). In *Bacillus anthracis*, while the *racE1* knock-out leads to moderate growth defect that can be alleviated fully by D-glutamate, the *racE2* knock-out severely inhibits bacterial growth, which is only partially restored by exogenous D-glutamate (Shatalin & Neyfakh, 2005).

It is important to mention that some mutations in genes involved in the cytoplasmic steps in MRSA affect their susceptibility to methicillin; this is the case for *glmS* (Komatsuzawa *et al.*, 2004), *glmM* (Jolly *et al.*, 1997; Glanzmann *et al.*, 1999) and *murE* (Ludovice *et al.*, 1998). In *Pseudomonas aeruginosa*, a transcriptional repressor of the *glmS* expression, known as *glmR*, has been described. The inactivation of the *glmR* gene dramatically sensitizes the microorganism to a large variety of antibiotics, and in particular the aminoglycosides (Ramos-Aires *et al.*, 2004).

## Concluding remarks

Since the discovery of Park's nucleotide in 1952, the cytoplasmic steps of peptidoglycan biosynthesis have been defined completely. In particular, considerable progress was made during the 1990s, where the overproduction and purification of the enzymes became possible, along with the production of the noncommercially available substrates by chemical and/or enzymatic synthesis. Studies using enzyme kinetics, isotope-based experiments, NMR, sequence alignments, site-directed mutagenesis, etc., have allowed the reaction mechanisms to be established and their substrate specificities to be defined. More recently, the availability of X-ray and NMR structures has improved our knowledge of these mechanisms; furthermore, these investigations have revealed important conformational changes upon ligand binding. Generally, they were performed with a representative enzyme, and often that of *Escherichia coli*. In the future, it will be important to compare enzymes from different species with respect to ligand binding and reaction mechanism. This is obvious when one of the substrates differs (e.g. MurE from Gram-negative vs. Gram-positive species, D-Ala:D-X ligases); however, even when the substrates are invariant, significant differences may prevail (e.g. MurB types I and II, alanine racemases). Such comparative enzymology of bacterial species needs to be performed.

As peptidoglycan is an essential component of the cell wall of eubacteria, these enzymes represent targets for antibacterial agents. However, they have been under-exploited so far, perhaps due to the very few natural antibiotics that are known to inhibit them (bacilysin, fosfomycin, D-cycloserine). The proliferation of resistant bacterial strains renders the search for new antibacterial compounds urgent, and in this respect, these enzymes involved in the cytoplasmic steps need to be considered seriously. Although good inhibitors have been described

recently, most are devoid of antibacterial activity due to their inability to cross the cytoplasmic membrane, and thereby to reach their target. The challenge for the next few years will be to design compounds that are endowed with penetration properties while retaining their affinities for their respective targets.

# Acknowledgements

This work was supported by the European Commission through the EUR-INTAFAR project (LSHM-CT-2004-512138), the Centre National de la Recherche Scientifique (UMR 8619 and PICS 3729), the Délégation Générale pour l'Armement (Contrats Jeune Chercheur 036000104 and 056000030 to A.B.), the Ministry of Education, Science and Sports of the Republic of Slovenia, the Franco-Slovene Proteus programme and the Institut Charles Nodier (Ljubljana). The authors thank Samo Turk for help in preparing protein figures, and Chris Berrie for critically reading the manuscript. They apologize to those researchers whose work could not be quoted because of limited space.

# Statement

Protein figures presented in this review were prepared with PyMol [DeLano WL, The PyMol Molecular Graphics System (2002), DeLano Scientific, Palo Alto, CA, USA; http://www.pymol.org].

Although its essential character had been demonstrated by Mengin-Lecreulx & van Heijenoort (1996), *glmM* (referred to in the 'Genetic organization' section above) was annotated as nonessential in the article of Baba *et al.* (2006). As a matter of fact, it has been proved that the *glmM* strain of Baba *et al.* contains an intact copy of the *glmM* gene (D. Mengin-Lecreulx, pers. commun.).

# References

Abo-Ghalia M, Michaud C, Blanot D & van Heijenoort J (1985) Specificity of the uridine-diphosphate-N-acetylmuramyl-L-alanyl-D-glutamate: meso-2,6-diaminopimelate synthetase from *Escherichia coli*. Eur J Biochem 153: 81–87.

Abo-Ghalia M, Flegel M, Blanot D & van Heijenoort J (1988) Synthesis of inhibitors of the meso-diaminopimelate-adding enzyme from *Escherichia coli*. Int J Peptide Protein Res 32: 208–222.

Adams B, Lowpatch K, Thorndycroft F, Whyte SM & Young DW (2005) Stereochemistry of reactions of the inhibitor/substrates L- and D-chloroalanine with β-mercaptoethanol catalysed by L-aspartate aminotransferase and D-amino acid aminotransferase respectively. Org Biomol Chem 3: 3357–3364.

Al-Bar OAM, O'Connor CD, Giles IG & Akhtar M (1992) D-alanine:D-alanine ligase of *Escherichia coli*. Expression,
purification and inhibitory studies on the cloned enzyme. *Biochem J* 282: 747–752.

Allen JG, Atherton FR, Hall MJ, Hassall CH, Holmes SW, Lambert RW, Nisbet LJ & Ringrose PS (1978) Phosphonopeptides, a new class of synthetic antibacterial agents. *Nature* 272: 56–58.

Anderson MS, Eveland SS, Onishi HR & Pompliano DL (1996) Kinetic mechanism of the *Escherichia coli* UDPMurNAc-tripeptide D-alanyl-D-alanine-adding enzyme: use of a glutathione S-transferase fusion. *Biochemistry* 35: 16264–16269.

Andres CJ, Bronson JJ, D'Andrea SV *et al.* (2000) 4-Thiazolidinones: novel inhibitors of the bacterial enzyme MurB. *Bioorg Med Chem Lett* 10: 715–717.

Antane S, Caufield CE, Hu W *et al.* (2006) Pulvinones as bacterial cell wall biosynthesis inhibitors. *Bioorg Med Chem Lett* 16: 176–180.

Anwar RA & Vlaovic M (1986) UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine synthetase from *Bacillus sphaericus*: activation by potassium phosphate. *Biochem Cell Biol* 64: 297–303.

Ashiuchi M & Misono H (2002) Biochemistry and molecular genetics of poly-γ-glutamate synthesis. *Appl Microbiol Biotechnol* 59: 9–14.

Ashiuchi M, Yoshimura T, Esaki N, Ueno H & Soda K (1993) Inactivation of glutamate racemase of *Pediococcus pentosaceus* by L-serine-O-sulfate. *Biosci Biotechnol Biochem* 57: 1978–1979.

Ashiuchi M, Tani K, Soda K & Misono H (1998) Properties of glutamate racemase from *Bacillus subtilis* IFO 3336 producing poly-γ-glutamate. *J Biochem (Tokyo)* 123: 1156–1163.

Ashiuchi M, Soda K & Misono H (1999) Characterization of *yrpC* gene product of *Bacillus subtilis* IFO 3336 as glutamate racemase isozyme. *Biosci Biotechnol Biochem* 63: 792–798.

Ashiuchi M, Kuwana E, Yamamoto T, Komatsu K, Soda K & Misono H (2002) Glutamate racemase is an endogenous DNA gyrase inhibitor. *J Biol Chem* 277: 39070–39073.

Ashiuchi M, Kuwana E, Komatsu K, Soda K & Misono H (2003) Differences in effects on DNA gyrase activity between two glutamate racemases of *Bacillus subtilis*, the poly-γ-glutamate synthesis-linking Glr enzyme and the YrpC (MurI) isozyme. *FEMS Microbiol Lett* 223: 221–225.

Ashiuchi M, Nishikawa Y, Matsunaga K, Yamamoto M, Shimanouchi K & Misono H (2007) Genetic design of conditional D-glutamate auxotrophy for *Bacillus subtilis*: use of a vector-borne poly-γ-glutamate synthetic system. *Biochem Biophys Res Commun* 362: 646–650.

Atherton FR, Hall MJ, Hassall CH, Lambert RW & Ringrose PS (1979a) Phosphonopeptides as antibacterial agents: rationale, chemistry, and structure-activity relationships. *Antimicrob Agents Chemother* 15: 677–683.

Atherton FR, Hall MJ, Hassall CH, Lambert RW, Lloyd WJ & Ringrose PS (1979b) Phosphonopeptides as antibacterial agents: mechanism of action of alaphosphin. *Antimicrob Agents Chemother* 15: 696–705.


Atherton FR, Hall MJ, Hassall CH, Lambert RW, Lloyd WJ, Ringrose PS & Westmacott D (1982) Antibacterial activity and mechanism of action of phosphonopeptides based on aminomethylphosphonic acid. *Antimicrob Agents Chemother* 22: 571–578.

Auger G, van Heijenoort J, Blanot D & Deprun C (1995) Synthesis of N-acetylmuramic acid derivatives as potential inhibitors of the D-glutamic acid-adding enzyme. *J Prakt Chem* 337: 351–357.

Auger G, van Heijenoort J, Vederas JC & Blanot D (1996) Effect of analogues of diaminopimelic acid on the meso-diaminopimelate-adding enzyme from *Escherichia coli*. *FEBS Lett* 391: 171–174.

Auger G, Martin L, Bertrand J, Ferrari P, Fanchon E, Vaganay S, Pétillot Y, van Heijenoort J, Blanot D & Dideberg O (1998) Large-scale preparation, purification, and crystallization of UDP-N-acetylmuramoyl-L-alanine: D-glutamate ligase from *Escherichia coli*. *Protein Expr Purif* 13: 23–29.

Axelsson BS, Floss HG, Lee SG, Saeed A, Spencer PA & Young DW (1994) Stereochemistry of conversion of the suicide substrates β-chloro-D-alanine and D- and L-serine O-sulfates into pyruvate by D-amino acid aminotransferase and by L-aspartate aminotransferase. *J Chem Soc Perkin Trans 1* 2137–2142.

Azzolina BA, Yuan X, Anderson MS & El-Sherbeini M (2001) The cell wall and cell division gene cluster in the *mra* operon of *Pseudomonas aeruginosa*: cloning, production, and purification of active enzymes. *Protein Expr Purif* 21: 393–400.

Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner BL & Mori H (2006) Construction of *Escherichia coli* K-12 in-frame, single-gene knockout mutants: the Keio collection. *Mol Syst Biol* 2: 1–11.

Babič A & Pečar S (2007) An improved total synthesis of UDP-N-acetyl-muramic acid. *Tetrahedron Lett* 48: 4403–4405.

Babič A, Patin D, Boniface A, Hervé M, Mengin-Lecreulx D, Pečar S, Gobec S & Blanot D (2007) Chemoenzymatic synthesis of the nucleotide substrates of the Mur ligases. *5th Joint Meeting on Medicinal Chemistry*, June 17–21, Portorož, Slovenia, (Kikelj D, ed.), pp. 1–4. Medimond Srl, Bologna, Italy.

Bachelier A, Mayer R & Klein CD (2006) Sesquiterpene lactones are potent and irreversible inhibitors of the antibacterial target enzyme MurA. *Bioorg Med Chem Lett* 16: 5605–5609.

Badet B & Walsh C (1985) Purification of an alanine racemase from *Streptococcus faecalis* and analysis of its inactivation by (1-aminoethyl)phosphonic acid enantiomers. *Biochemistry* 24: 1333–1341.

Badet B, Roise D & Walsh CT (1984) Inactivation of the *dadB Salmonella typhimurium* alanine racemase by D and L isomers of β-substituted alanines: kinetics, stoichiometry, active site peptide sequencing, and reaction mechanism. *Biochemistry* 23: 5188–5194.

Badet B, Inagaki K, Soda K & Walsh CT (1986) Time-dependent inhibition of *Bacillus stearothermophilus* alanine racemase by (1-aminoethyl)phosphonate isomers by isomerization to noncovalent slowly dissociating enzyme-(1-aminoethyl) phosphonate complexes. *Biochemistry* 25: 3275–3282.

Cytoplasmic steps of peptidoglycan biosynthesis

Badet B, Vermoote P, Haumont PY, Lederer F & Le Goffic F (1987) Glucosamine synthetase from *Escherichia coli*: purification, properties, and glutamine-utilizing site location. *Biochemistry* 26: 1940–1948.

Badet B, Vermoote P & Le Goffic F (1988) Glucosamine synthetase from *Escherichia coli*: kinetic mechanism and inhibition by *N*²-fumaroyl-ʟ-2,3-diaminopropionic derivatives. *Biochemistry* 27: 2282–2287.

Badet-Denisot MA, René L & Badet B (1993) Mechanistic investigations on glucosamine-6-phosphate synthase. *Bull Soc Chim Fr* 130: 249–255.

Badet-Denisot MA, Leriche C, Massière F & Badet B (1995) Nitrogen transfer in *E. coli* glucosamine-6-phosphate synthase investigations using substrate and bisubstrate analogs. *Bioorg Med Chem Lett* 5: 815–820.

Badet-Denisot MA, Fernandez-Herrero LA, Berenguer J, Ooi T & Badet B (1997) Characterization of ʟ-glutamine:ᴅ-fructose-6-phosphate amidotransferase from an extreme thermophile *Thermus thermophilus* HB8. *Arch Biochem Biophys* 337: 129–136.

Barbosa MDFS, Yang G, Fang J, Kurilla MG & Pompliano DL (2002) Development of a whole-cell assay for peptidoglycan biosynthesis inhibitors. *Antimicrob Agents Chemother* 46: 943–946.

Baum EZ, Montenegro DA, Licata L, Turchi I, Webb GC, Foleno BD & Bush K (2001) Identification and characterization of new inhibitors of the *Escherichia coli* MurA enzyme. *Antimicrob Agents Chemother* 45: 3182–3188.

Baum EZ, Crespo-Carbone SM, Abbanat D, Foleno B, Maden A, Goldschmidt R & Bush K (2006) Utility of muropeptide ligase for identification of inhibitors of the cell wall biosynthesis enzyme MurF. *Antimicrob Agents Chemother* 50: 230–236.

Bearne SL (1996) Active site-directed inactivation of *Escherichia coli* glucosamine-6-phosphate synthase. Determination of the fructose 6-phosphate binding constant using a carbohydrate-based inactivator. *J Biol Chem* 271: 3052–3057.

Bearne SL & Blouin C (2000) Inhibition of *Escherichia coli* glucosamine-6-phosphate synthase by reactive intermediate analogues. The role of the 2-amino function in catalysis. *J Biol Chem* 275: 135–140.

Benson TE, Marquardt JL, Marquardt AC, Etzkorn FA & Walsh CT (1993) Overexpression, purification, and mechanistic study of UDP-*N*-acetylenolpyruvylglucosamine reductase. *Biochemistry* 32: 2024–2030.

Benson TE, Filman DJ, Walsh CT & Hogle JM (1995) An enzyme-substrate complex involved in bacterial cell wall biosynthesis. *Nat Struct Biol* 2: 644–653.

Benson TE, Walsh CT & Hogle JM (1996) The structure of the substrate-free form of MurB, an essential enzyme for the synthesis of bacterial cell walls. *Structure* 4: 47–54.

Benson TE, Walsh CT & Hogle JM (1997a) X-ray crystal structures of the S229A mutant and wild-type MurB in the presence of the substrate enolpyruvyl-UDP-*N*-acetylglucosamine at 1.8-Å resolution. *Biochemistry* 36: 806–811.


Benson TE, Walsh CT & Massey V (1997b) Kinetic characterization of wild-type and S229A mutant of MurB: evidence for the role of Ser 229 as a general catalyst. *Biochemistry* 36: 796–805.

Benson TE, Harris MS, Choi GH, Cialdella JI, Herberg JT, Martin JP Jr & Baldwin ET (2001) A structural variation for MurB: X-ray crystal structure of *Staphylococcus aureus* UDP-*N*-acetylenolpyruvylglucosamine reductase (MurB). *Biochemistry* 40: 2340–2350.

Berger BJ, English S, Chan G & Knodel MH (2003) Methionine regeneration and aminotransferases in *Bacillus subtilis*, *Bacillus cereus*, and *Bacillus anthracis*. *J Bacteriol* 185: 2418–2431.

Berlyn KB, Low KB & Rudd KE (1996) Linkage map of *Escherichia coli* K-12, edition 9. *Escherichia coli and Salmonella*: Cellular and Molecular Biology, Vol. 2 (Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, Magasanik B, Reznikoff WS, Riley M, Schaechter M & Umbarger HE, eds), pp. 1715–1902. ASM Press, Washington, DC.

Bertrand JA, Auger G, Fanchon E, Martin L, Blanot D, van Heijenoort J & Dideberg O (1997) Crystal structure of UDP-*N*-acetylmuramoyl-ʟ-alanine: ᴅ-glutamate ligase from *Escherichia coli*. *EMBO J* 16: 3416–3425.

Bertrand JA, Auger G, Martin L, Fanchon E, Blanot D, Le Beller D, van Heijenoort J & Dideberg O (1999) Determination of the MurD mechanism through crystallographic analysis of enzymes complexes. *J Mol Biol* 289: 579–590.

Bertrand JA, Fanchon E, Martin L, Chantalat L, Auger G, Blanot D, van Heijenoort J & Dideberg O (2001) ‘Open’ structures of MurD: domain movements and structural similarities with folylpolyglutamate synthetase. *J Mol Biol* 301: 1257–1266.

Besong GE, Bostock JM, Stubbings W, Chopra I, Roper DI, Lloyd AJ, Fishwick CW & Johnson AP (2005) A de novo designed inhibitor of ᴅ-Ala-ᴅ-Ala ligase from *E. coli*. *Angew Chem Int Ed Engl* 44: 6403–6406.

Blanot D, Auger G, Liger D & van Heijenoort J (1994) Synthesis of α and β anomers of UDP-*N*-acetylmuramic acid. *Carbohydr Res* 252: 107–115.

Blewett AM, Lloyd AJ, Echalier A, Fülöp V, Dowson CG, Bugg TDH & Roper DI (2004) Expression, purification, crystallization and preliminary characterization of uridine 5′-diphospho-*N*-acetylmuramoyl ʟ-alanyl-ᴅ-glutamate: lysine ligase (MurE) from *Streptococcus pneumoniae* 110K/70. *Acta Cryst D60*: 359–361.

Boisvert W, Cheung KS, Lerner SA & Johnston M (1986) Mechanisms of action of chloroalanyl antibacterial peptides. Identification of the intracellular enzymes inactivated on treatment of *Escherichia coli* JSR-O with the dipeptide βCl-ʟAla-βCl-ʟAla. *J Biol Chem* 261: 7871–7878.

Boniface A (2007). *Etude des relations structure-activité au sein de la famille des Mur synthétases, enzymes de la voie de biosynthèse du peptidoglycane*. Ph.D. Thesis, Université Paris-Sud, Orsay, France.

Boniface A, Bouhss A, Mengin-Lecreulx D & Blanot D (2006) The MurE synthetase from *Thermotoga maritima* is endowed with
an unusual D-lysine adding activity. J Biol Chem 281: 15680–15686.

Born TL & Blanchard JS (1999) Structure/function studies on enzymes in the diaminopimelate pathway of bacterial cell wall biosynthesis. Curr Opin Chem Biol 3: 603–613.

Bouhss A, Mengin-Lecreulx D, Blanot D, van Heijenoort J & Parquet C (1997) Invariant amino acids in the Mur peptide synthetases of bacterial peptidoglycan synthesis and their modification by site-directed mutagenesis in the UDP-MurNAc: L-alanine ligase from Escherichia coli. Biochemistry 36: 11556–11563.

Bouhss A, Dementin S, van Heijenoort J, Parquet C & Blanot D (1999a) Formation of adenosine 5′-tetraphosphate from the acyl phosphate intermediate: a difference between the MurC and MurD synthetases of Escherichia coli. FEBS Lett 453: 15–19.

Bouhss A, Dementin S, Parquet C, Mengin-Lecreulx D, Bertrand JA, Le Beller D, Dideberg O, van Heijenoort J & Blanot D (1999b) Role of the ortholog and paralog amino acid invariants in the active site of the UDP-MurNAc-L-alanine: D-glutamate ligase (MurD). Biochemistry 38: 12240–12247.

Bouhss A, Dementin S, van Heijenoort J, Parquet C & Blanot D (2002) MurC and MurD synthetases of peptidoglycan biosynthesis: borohydride trapping of acyl-phosphate intermediates. Methods Enzymol 354: 189–196.

Bouhss A, Crouvoisier M, Blanot D & Mengin-Lecreulx D (2004) Purification and characterization of the bacterial MraY translocase catalyzing the first membrane step of peptidoglycan biosynthesis. J Biol Chem 279: 29974–29980.

Bouhss A, Trunkfield AE, Bugg TDH & Mengin-Lecreulx D (2008) The biosynthesis of peptidoglycan lipid-linked intermediates. FEMS Microbiol Rev, doi: 10.1111/j.1574-6976.2007.00089.x

Bronson JJ, DenBleyker KL, Falk PJ, Mate RA, Ho HT, Pucci MJ & Snyder LB (2003) Discovery of the first antibacterial small molecule inhibitors of MurB. Bioorg Med Chem Lett 13: 873–875.

Broschat KO, Gorka C, Page JD, Martin-Berger CL, Davies MS, Huang H, Gulve EA, Salsgiver WJ & Kasten TP (2002) Kinetic characterization of human glutamine-fructose-6-phosphate amidotransferase I. Potent feedback inhibition by glucosamine 6-phosphate. J Biol Chem 277: 14764–14770.

Brown ED, Marquardt JL, Lee JP, Walsh CT & Anderson KS (1994) Detection and characterization of a phospholactoyl-enzyme adduct in the reaction catalyzed by UDP-N-acetylglycosamine enolpyruvyl transferase, MurZ. Biochemistry 33: 10638–10645.

Brown ED, Vivas EI, Walsh CT & Kolter R (1995) MurA (MurZ), the enzyme that catalyzes the first committed step in peptidoglycan biosynthesis, is essential in Escherichia coli. J Bacteriol 177: 4194–4197.

Brown K, Pompeo F, Dixon S, Mengin-Lecreulx D, Cambillau C & Bourne Y (1999) Crystal structure of the bifunctional N-acetylglucosamine 1-phosphate uridyltransferase from Escherichia coli: a paradigm for the related pyrophosphorylase superfamily. EMBO J 18: 4096–4107.

Bugg TDH & Walsh CT (1992) Intracellular steps of bacterial cell wall peptidoglycan biosynthesis: enzymology, antibiotics, and antibiotic resistance. Nat Prod Rep 9: 199–215.

Bugg TDH, Wright GD, Dutka-Malen S, Arthur M, Courvalin P & Walsh CT (1991) Molecular basis of vancomycin resistance in Enterococcus faecium BM4147: biosynthesis of depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry 30: 10408–10415.

Burton E, Gawande PV, Yakandawala N, LoVetri K, Zhanel GG, Romeo T, Friesen AD & Madhyastha S (2006) Antibiofilm activity of GlmU enzyme inhibitors against catheter-associated uropathogens. Antimicrob Agents Chemother 50: 1835–1840.

Byczynski B, Mizyed S & Berti PJ (2003) Nonenzymatic breakdown of the tetrahedral (α-carboxyketal phosphate) intermediates of MurA and AroA, two carboxyvinyl transferases. Protonation of different functional groups controls the rate and fate of breakdown. J Am Chem Soc 125: 12541–12550.

Candela T & Fouet A (2006) Poly-gamma-glutamate in bacteria. Mol Microbiol 60: 1091–1098.

Chakravarty PK, Greenlee WJ, Parsons WH, Patchett AA, Combs P, Roth A, Busch RD & Mellin TN (1989) (3-Amino-2-oxoalkyl)phosphonic acids and their analogues as novel inhibitors of D-alanine : D-alanine ligase. J Med Chem 32: 1886–1890.

Cheung KS, Wasserman SA, Dudek E, Lerner SA & Johnston M (1983) Choralanyl and propargylglycyl dipeptides. Suicide substrate containing antibacterials. J Med Chem 26: 1733–1741.

Cheung KS, Boisvert W, Lerner SA & Johnston M (1986) Chloroalanyl antibiotic peptides: antagonism of their antimicrobial effects by L-alanine and L-alanyl peptides in Gram-negative bacteria. J Med Chem 29: 2060–2068.

Chmara H, Zähner H & Borowski E (1984) Anticapsin, an active-site directed irreversible inhibitor of glucosamine 6-phosphate synthetase from Escherichia coli. J Antibiotics 37: 1038–1043.

Chmara H, Andruszkiewicz R & Borowski E (1985) Inactivation of glucosamine-6-phosphate synthetase from Salmonella typhimurium LT2 by fumaroyl diamino-propanoic acid derivatives, a novel group of glutamine analogs. Biochim Biophys Acta 870: 357–366.

Choi SY, Esaki N, Ashiuchi M, Yoshimura T & Soda K (1994) Bacterial glutamate racemase has high sequence similarity with myoglobins and forms an equimolar inactive complex with hemin. Proc Natl Acad Sci USA 91: 10144–10147.

Chopra I, Storey C, Falla JT & Pearce JH (1998) Antibiotics, peptidoglycan synthesis and genomics: the chlamydial anomaly revisited. Microbiology 144: 2673–2678.

Comess KM, Schurdak ME, Voorbach MJ et al. (2006) An ultraefficient affinity-based high-throughput screening process: application to bacterial cell wall biosynthesis enzyme MurF. J Biomol Screen 11: 743–754.

Constantine KL, Mueller L, Goldfarb V, Wittekind M, Metzler WJ, Yanchunas J Jr, Robertson JG, Malley MF, Friedrichs MS & Farmer II BT (1997) Characterization of NADP⁺ binding to

© 2008 Federation of European Microbiological Societies Published by Blackwell Publishing Ltd. All rights reserved
perdeuterated MurB: backbone atom NMR assignments and chemical-shift changes. *J Mol Biol* 267: 1223–1246.

Copié V, Faraci SW, Walsh CT & Griffin RG (1988) Inhibition of alanine racemase by alanine phosphonate: detection of an imine linkage to pyridoxal 5′-phosphate in the enzyme-inhibitor complex by solid-state <sup>15</sup>N nuclear magnetic resonance. *Biochemistry* 27: 4966–4970.

Cox RJ (1996) The DAP pathway of lysine as a target for antimicrobial agents. *Nat Prod Rep* 13: 29–43.

Cox RJ, Sutherland A & Vederas JC (2000) Bacterial diaminopimelate metabolism as a target for antibiotic design. *Bioorg Med Chem* 8: 843–871.

Dai HJ, Parker CN & Bao JJ (2002) Characterization and inhibition study of MurA enzyme by capillary electrophoresis. *J Chromatogr B Analyt Technol Biomed Life Sci* 766: 123–132.

Dai JB, Liu Y, Ray WJ Jr & Konno M (1992) The crystal structure of muscle phosphoglucomutase refined at 2.7 angstrom resolution. *J Biol Chem* 267: 6322–6337.

Daniel RA & Errington J (1993) DNA sequence of the murE-murD region of *Bacillus subtilis* 168. *J Gen Microbiol* 139: 361–370.

Daub E, Zawadzke LE, Botstein D & Walsh CT (1988) Isolation, cloning, and sequencing of the *Salmonella typhimurium ddlA* gene with purification and characterization of its product, D-alanine:D-alanine ligase (ADP forming). *Biochemistry* 27: 3701–3708.

de Dios A, Prieto L, Martin JA *et al.* (2002) 4-Substituted D-glutamic acid analogues: the first potent inhibitors of glutamate racemase (MurI) enzyme with antibacterial activity. *J Med Chem* 45: 4559–4570.

De Smet KA, Kempsell KE, Gallagher A, Duncan K & Young DB (1999) Alteration of a single amino acid residue reverses fosfomycin resistance of recombinant MurA from *Mycobacterium tuberculosis*. *Microbiology* 145: 3177–3184.

Dementin S (2001) *Etude du mécanisme réactionnel des Mur synthétases, enzymes impliquées dans la biosynthèse du peptidoglycane bactérien*. Ph.D. Thesis, Université Paris-Sud, Orsay, France.

Dementin S, Bouhss A, Auger G, Parquet C, Mengin-Lecreulx D, Dideberg O, van Heijenoort J & Blanot D (2001) Evidence of a functional requirement for a carbamoylated lysine residue in MurD, MurE and MurF synthetases as established by chemical rescue experiments. *Eur J Biochem* 268: 5800–5807.

Deva T, Baker EN, Squire CJ & Smith CA (2006) Structure of *Escherichia coli* UDP-N-acetylmuramoyl:L-alanine ligase (MurC). *Acta Cryst* D62: 1466–1474.

DeVito JA, Mills JA, Liu VG *et al.* (2002) An array of target-specific screening strains for antibacterial discovery. *Nat Biotechnol* 20: 478–483.

Dhalla AM, Yanchunas J Jr, Ho HT, Falk PJ, Villafranca JJ & Robertson JG (1995) Steady-state kinetic mechanism of *Escherichia coli* UDP-N-acetylenolpyruvylglucosamine reductase. *Biochemistry* 34: 5390–5402.


Dini C, Drochon N, Ferrari P & Aszodi J (2000) Multi gram synthesis of UDP-N-acetylmuramic acid. *Bioorg Med Chem Lett* 10: 143–145.

Dodd D, Reese JG, Louer CR, Ballard JD, Spies MA & Blanke SR (2007) Functional comparison of the two *Bacillus anthracis* glutamate racemases. *J Bacteriol* 189: 5265–5275.

Doublet P, van Heijenoort J, Bohin J-P & Mengin-Lecreulx D (1993) The *murI* gene of *Escherichia coli* is an essential gene that encodes a glutamate racemase activity. *J Bacteriol* 175: 2970–2979.

Doublet P, van Heijenoort J & Mengin-Lecreulx D (1994) The glutamate racemase activity from *Escherichia coli* is regulated by peptidoglycan precursor UDP-N-acetylmuramoyl-L-alanine. *Biochemistry* 33: 5285–5290.

Doublet P, van Heijenoort J & Mengin-Lecreulx D (1996) Regulation of the glutamate racemase of *Escherichia coli* investigated by site-directed mutagenesis. *Microb Drug Resist* 2: 43–49.

Du W, Brown JR, Sylvester DR, Huang J, Chalker AF, So SY, Holmes DJ, Payne DJ & Wallis NG (2000) Two active forms of UDP-N-acetylglucosamine enolpyruvyl transferase in Gram-positive bacteria. *J Bacteriol* 182: 4146–4152.

Duncan K & Walsh CT (1988) ATP-dependent inactivation and slow binding inhibition of *Salmonella typhimurium* D-alanine:D-alanine ligase (ADP) by (aminoalkyl)phosphonate and aminophosphonate analogues of D-alanine. *Biochemistry* 27: 3709–3714.

Duncan K, Faraci WS, Matteson DS & Walsh CT (1989) (1-Aminoethyl)boronic acid: a novel inhibitor for *Bacillus stearothermophilus* alanine racemase and *Salmonella typhimurium* D-alanine:D-alanine ligase (ADP-forming). *Biochemistry* 28: 3541–3549.

Duncan K, van Heijenoort J & Walsh CT (1990) Purification and characterization of the D-alanyl-D-alanine-adding enzyme from *Escherichia coli*. *Biochemistry* 29: 2379–2386.

Ehmann DE, Demeritt JE, Hull KG & Fisher SL (2004) Biochemical characterization of an inhibitor of *Escherichia coli* UDP-N-acetylmuramyl-L-alanine ligase. *Biochim Biophys Acta* 1698: 167–174.

Ellsworth BA, Tom NJ & Bartlett PA (1996) Synthesis and evaluation of inhibitors of bacterial D-alanine:D-alanine ligases. *Chem Biol* 3: 37–44.

El Zoeiby A, Sanschagrin F, Lamoureux J, Darveau A & Levesque RC (2000) Cloning, over-expression and purification of *Pseudomonas aeruginosa murC* encoding uridine diphosphate N-acetylmuramate:L-alanine ligase. *FEMS Microbiol Lett* 183: 281–288.

El Zoeiby A, Sanschagrin F, Havugimana PC, Garnier A & Levesque RC (2001) *In vitro* reconstruction of the biosynthetic pathway of peptidoglycan cytoplasmic precursor in *Pseudomonas aeruginosa*. *FEMS Microbiol Lett* 201: 229–235.

El Zoeiby A, Sanschagrin F & Levesque RC (2003a) Structure and function of the Mur enzymes: development of novel inhibitors. *Mol Microbiol* 47: 1–12.

El Zoeiby A, Sanschagrin F, Darveau A, Brisson J-R & Levesque RC (2003b) Identification of novel inhibitors of *Pseudomonas aeruginosa* MurC enzyme derived from phage-displayed peptide libraries. *J Antimicrob Chemother* 51: 531–543.

Emanuele JJ Jr, Jin H, Jacobson BL, Chang CY, Einspahr HM & Villafranca JJ (1996) Kinetic and crystallographic studies of *Escherichia coli* UDP-N-acetylmuramate: L-alanine ligase. *Prot Sci* 5: 2566–2574.

Emanuele JJ Jr, Jin H, Yanchunas J & Villafranca JJ (1997) Evaluation of the kinetic mechanism of *Escherichia coli* uridine diphosphate-N-acetylmuramate: L-alanine ligase. *Biochemistry* 36: 7264–7271.

Erion MD & Walsh CT (1987) 1-Aminocyclopropanephosphonate: time-dependent inactivation of 1-aminocyclopropanecarboxylate deaminase and *Bacillus stearothermophilus* alanine racemase by slow dissociation behavior. *Biochemistry* 26: 3417–3425.

Esaki N & Walsh CT (1986) Biosynthetic alanine racemase of *Salmonella typhimurium*: purification and characterization of the enzyme encoded by the *alr* gene. *Biochemistry* 25: 3261–3267.

Eschenburg S & Schönbrunn E (2000) Comparative X-ray analysis of the un-liganded fosfomycin-target murA. *Proteins* 40: 290–298.

Eschenburg S, Kabsch W, Healy ML & Schönbrunn E (2003) A new view of the mechanisms of UDP-N-acetylglucosamine enolpyruvyl transferase (MurA) and 5-enolpyruvylshikimate-3-phosphate synthase (AroA) derived from X-ray structures of their tetrahedral reaction intermediate states. *J Biol Chem* 278: 49215–49222.

Eschenburg S, Priestman M & Schönbrunn E (2005a) Evidence that the fosfomycin target Cys<sup>115</sup> in UDP-N-acetylglucosamine enolpyruvyl transferase (MurA) is essential for product release. *J Biol Chem* 280: 3757–3763.

Eschenburg S, Priestman MA, Abdul-Latif FA, Delachaume C & Fassy FE (2005b) A novel inhibitor that suspends the induced fit mechanism of UDP-N-acetylglucosamine enolpyruvyl transferase (MurA). *J Biol Chem* 280: 14070–14075.

Eveland SS, Pompliano DL & Anderson MS (1997) Conditionally lethal *Escherichia coli* murein mutants contain point defects that map to regions conserved among murein and folyly poly-γ-glutamate ligases: identification of a ligase superfamily. *Biochemistry* 36: 6223–6229.

Falk PJ, Ervin KM, Volk KS & Ho H-T (1996) Biochemical evidence for the formation of a covalent acyl-phosphate linkage between UDP-N-acetylmuramate and ATP in the *Escherichia coli* UDP-N-acetylmuramate: L-alanine ligase-catalyzed reaction. *Biochemistry* 35: 1417–1422.

Fan C, Moews PC, Walsh CT & Knox JR (1994) Vancomycin resistance: structure of D-alanine : D-alanine ligase at 2.3 Å resolution. *Science* 266: 439–443.

Fan C, Park IS, Walsh CT & Knox JR (1997) D-alanine : D-alanine ligase: phosphonate and phosphinate intermediates with wild type and the Y216F mutant. *Biochemistry* 36: 2531–2538.


Faraci WS & Walsh CT (1989) Mechanism of inactivation of alanine racemase by β, β, β-trifluoroalanine. *Biochemistry* 28: 431–437.

Farmer II BT, Constantine KL, Goldfarb V, Friedrichs MS, Wittekind M, Yanchunas J Jr, Robertson JG & Mueller L (1996) Localizing the NADP<sup>+</sup> binding site on the MurB enzyme by NMR. *Nat Struct Biol* 3: 995–997.

Fenn TD, Stamper GF, Morollo AA & Ringe D (2003) A side reaction of alanine racemase: transamination of cycloserine. *Biochemistry* 42: 5775–5783.

Fenn TD, Holyoak T, Stamper GF & Ringe D (2005) Effect of a Y265F mutant on the transamination-based cycloserine inactivation of alanine racemase. *Biochemistry* 44: 5317–5327.

Floquet N, Mouilleron S, Daher R, Maigret B, Badet B & Badet-Denisot MA (2007a) Ammonia channeling in bacterial glucosamine-6-phosphate synthase (Glms): molecular dynamics simulations and kinetic studies of protein mutants. *FEBS Lett* 581: 2981–2987.

Floquet N, Richez C, Durand P, Maigret B, Badet B & Badet-Denisot MA (2007b) Discovering new inhibitors of bacterial glucosamine-6P synthase (GlmS) by docking simulations. *Bioorg Med Chem Lett* 17: 1966–1970.

Flouret B, Mengin-Lecreulx D & van Heijenoort J (1981) Reverse-phase high-pressure liquid chromatography of uridine diphosphate N-acetylmuramyl peptide precursors of bacterial cell wall peptidoglycan. *Anal Biochem* 114: 59–63.

Fotheringham IG, Bledig SA & Taylor PP (1998) Characterization of the genes encoding D-amino acid transaminase and glutamate racemase, two D-glutamate biosynthetic enzymes of *Bacillus sphaericus* ATCC 10208. *J Bacteriol* 180: 4319–4323.

Francisco GD, Li Z, Albright JD *et al.* (2004) Phenyl thiazoly urea and carbamate derivatives as new inhibitors of bacterial cell-wall biosynthesis. *Bioorg Med Chem Lett* 14: 235–238.

Gallo KA & Knowles JR (1993) Purification, cloning, and cofactor independence of glutamate racemase from *Lactobacillus*. *Biochemistry* 32: 3981–3990.

Gallo KA, Tanner ME & Knowles JR (1993) Mechanism of the reaction catalyzed by glutamate racemase. *Biochemistry* 32: 3991–3997.

Galperin MY & Koonin EV (1997) A diverse superfamily of enzymes with ATP-dependent carboxylate-amine/thiol ligase activity. *Protein Sci* 6: 2639–2643.

Gegnas LD, Waddell ST, Chabin RM, Reddy S & Wong KK (1998) Inhibitors of the bacterial cell wall biosynthesis enzyme MurD. *Bioorg Med Chem Lett* 8: 1643–1648.

Gehring AM, Lees WJ, Mindiola DJ, Walsh CT & Brown ED (1996) Acetyltransfer precedes uridylyltransfer in the formation of UDP-N-acetylglucosamine in separable active sites of the bifunctional GlmU protein of *Escherichia coli*. *Biochemistry* 35: 579–585.

Girardin SE, Travassos LH, Hervé M, Blanot D, Boneca IG, Philpott DJ, Sansonetti PJ & Mengin-Lecreulx D (2003) Peptidoglycan molecular requirements allowing detection by Nod1 and Nod2. *J Biol Chem* 278: 41702–41708.

Glanzmann P, Gustafson J, Komatsuzawa H, Ohta K & Berger-Bächi B (1999) *glmM* operon and methicillin-resistant *glmM* suppressor mutants in *Staphylococcus aureus*. *Antimicrob Agents Chemother* 43: 240–245.

Glavas S & Tanner ME (1997) The inhibition of glutamate racemase by D-N-hydroxyglutamate. *Bioorg Med Chem Lett* 7: 2265–2270.

Glavas S & Tanner ME (1999) Catalytic acid/base residues of glutamate racemase. *Biochemistry* 38: 4106–4113.

Glavas S & Tanner ME (2001) Active site residues of glutamate racemase. *Biochemistry* 40: 6199–6204.

Gobec S, Urleb U, Auger G & Blanot D (2001) Synthesis and biochemical evaluation of some novel N-acyl phosphono- and phosphinoalanine derivatives as potential inhibitors of the D-glutamic acid-adding enzyme. *Pharmazie* 56: 295–297.

Golinelli-Pimpaneau B, Le Goffic F & Badet B (1989) Glucosamine-6-phosphate synthase from *Escherichia coli*: mechanism of the reaction at the fructose 6-phosphate binding site. *J Am Chem Soc* 111: 3029–3034.

Gordon E, Flouret B, Chantalat L, van Heijenoort J, Mengin-Lecreulx D & Dideberg O (2001) Crystal structure of the UDP-N-acetylmuramoyl-L-alanyl-D-glutamate: meso-diaminopimelate ligase from *Escherichia coli*. *J Biol Chem* 276: 10999–11006.

Gu YG, Florjancic AS, Clark RF *et al.* (2004) Structure-activity relationships of novel potent MurF inhibitors. *Bioorg Med Chem Lett* 14: 267–270.

Gubler M, Appoldt Y & Keck W (1996) Overexpression, purification, and characterization of UDP-N-acetylmuramyl-L-alanine ligase from *Escherichia coli*. *J Bacteriol* 178: 906–910.

Hammes WP, Neukam R & Kandler O (1977) On the specificity of the uridine diphospho-N-acetylmuramyl-alanyl-D-glutamic acid: diamino acid ligase of *Bifidobacterium globosum*. *Arch Microbiol* 115: 95–102.

Healy VL, Lessard IA, Roper DI, Knox JR & Walsh CT (2000a) Vancomycin resistance in enterococci: reprogramming of the D-Ala-D-Ala ligases in bacterial peptidoglycan biosynthesis. *Chem Biol* 7: R109–R119.

Healy VL, Mullins LS, Li X, Hall SE, Raushel FM & Walsh CT (2000b) D-Ala-D-X ligases: evaluation of D-alanyl phosphate intermediate by MIX, PIX and rapid quench studies. *Chem Biol* 7: 505–514.

Hervé M, Boniface A, Gobec S, Blanot D & Mengin-Lecreulx D (2007) Biochemical characterization and physiological properties of *Escherichia coli* UDP-N-acetylmuramate: L-alanyl-γ-D-glutamyl-meso-diaminopimelate ligase (Mpl). *J Bacteriol* 189: 3987–3995.

Hesse L, Bostock J, Dementin S, Blanot D, Mengin-Lecreulx D & Chopra I (2003) Functional and biochemical analysis of *Chlamydia trachomatis* MurC, an enzyme displaying UDP-N-acetylmuramate: amino acid ligase activity. *J Bacteriol* 185: 6507–6512.

Heymann H, Turdiu R, Lee BK & Barkulis SS (1968) A synthesis of uridinediphospho-N-acetylmuramic acid and its use as an acceptor of L-[¹⁴C]alanine. *Biochemistry* 7: 1393–1399.


Hitchcock SA, Eid CN, Aikins JA, Zia-Ebrahimi M & Blaszczak LC (1998) The first total synthesis of bacterial cell wall precursor UDP-N-acetylmuramyl-pentapeptide (Park nucleotide). *J Am Chem Soc* 120: 1916–1917.

Ho HT, Falk PJ, Ervin KM, Krishnan BS, Discotto LF, Dougherty TJ & Pucci MJ (1995) UDP-N-acetylmuramyl-L-alanine functions as an activator in the regulation of the *Escherichia coli* glutamate racemase activity. *Biochemistry* 34: 2464–2470.

Hofmann M, Boles E & Zimmermann FK (1994) Characterization of the essential yeast gene encoding N-acetylglucosamine-phosphate mutase. *Eur J Biochem* 221: 741–747.

Horton JR, Bostock JM, Chopra I, Hesse L, Phillips SE, Adams DJ, Johnson AP & Fishwick CW (2003) Macrocyclic inhibitors of the bacterial cell wall biosynthesis enzyme MurD. *Bioorg Med Chem Lett* 13: 1557–1560.

Hove-Jensen B (1992) Identification of *tms-26* as an allele of the *gcaD* gene, which encodes N-acetylglucosamine 1-phosphate uridyltransferase in *Bacillus subtilis*. *J Bacteriol* 174: 6852–6856.

Huber R, Langworthy TA, König H, Thomm M, Woese CR, Sleytr UB & Stetter KO (1986) *Thermotoga maritima* sp. nov. represents a new genus of unique extremely thermophilic eubacteria growing up to 90 °C. *Arch Microbiol* 144: 324–333.

Humljan J, Kotnik M, Boniface A, Šolmajer T, Urleb U, Blanot D & Gobec S (2006) A new approach towards peptidosulfonamides: synthesis of potential inhibitors of bacterial peptidoglycan biosynthesis enzymes MurD and MurE. *Tetrahedron* 62: 10980–10988.

Hutton CA, Perugini MA & Gerrard JA (2007) Inhibition of lysine biosynthesis: an evolving antibiotic strategy. *Mol BioSyst* 3: 458–465.

Huynh QK, Gulve EA & Dian T (2000) Purification and characterization of glutamine: fructose 6-phosphate amidotransferase from rat liver. *Arch Biochem Biophys* 379: 307–313.

Hwang KY, Cho C-S, Kim SS, Sung H-C, Yu YG & Cho Y (1999) Structure and mechanism of glutamate racemase from *Aquifex pyrophilus*. *Nat Struct Biol* 6: 422–426.

Inagaki K, Tanizawa K, Badet B, Walsh CT, Tanaka H & Soda K (1986) Thermostable alanine racemase from *Bacillus stearothermophilus*: molecular cloning of the gene, enzyme purification, and characterization. *Biochemistry* 25: 3268–3274.

Isupov MN, Obmolova G, Butterworth S, Badet-Denisot MA, Badet B, Polikarpov I, Littlechild JA & Tepljakov A (1996) Substrate binding is required for assembly of the active conformation of the catalytic site in Ntn amidotransferases: evidence from the 1.8 Å crystal structure of the glutaminase domain of glucosamine 6-phosphate synthase. *Structure* 4: 801–810.

Jin H, Emanuele JJ Jr, Fairman R, Robertson JG, Hail ME, Ho H-T, Falk PJ & Villafranca JJ (1996) Structural studies of *Escherichia coli* UDP-N-acetylmuramate: L-alanine ligase. *Biochemistry* 35: 1423–1431.

Jolly L, Wu S, van Heijenoort J, de Lencastre H, Mengin-Lecreulx D & Tomasz A (1997) The *femR315* gene from *Staphylococcus aureus*, the interruption of which results in reduced methicillin resistance, encodes a phosphoglucosamine mutase. *J Bacteriol 179*: 5321–5325.

Jolly L, Ferrari P, Blanot D, van Heijenoort J, Fassy F & Mengin-Lecreulx D (1999) Reaction mechanism of phosphoglucosamine mutase from *Escherichia coli*. *Eur J Biochem 262*: 202–210.

Jolly L, Pompeo F, van Heijenoort J, Fassy F & Mengin-Lecreulx D (2000) Autophosphorylation of phosphoglucosamine mutase from *Escherichia coli*. *J Bacteriol 182*: 1280–1285.

Katz AH & Caufield CE (2003) Structure-based approaches to cell wall biosynthesis inhibitors. *Curr Pharm Design 9*: 857–866.

Kenig M, Vandamme E & Abraham EP (1976) The mode of action of bacilysin and anticapsin and biochemical properties of bacilysin-resistant mutants. *J Gen Microbiol 94*: 46–54.

Kim DH, Lees WJ & Walsh CT (1995) Stereochemical analysis of the tetrahedral adduct formed at the active site of UDP-GlcNAc enolpyruvyl transferase from the pseudosubstrates, (*E*)- and (*Z*)-fluorophosphoenolpyruvate, in *D₂O*. *J Am Chem Soc 117*: 6380–6381.

Kim DH, Lees WJ, Kempsell KE, Lane WS, Duncan K & Walsh CT (1996) Characterization of a Cys115 to Asp substitution in the *Escherichia coli* cell wall biosynthetic enzyme UDP-GlcNAc enolpyruvyl transferase (MurA) that confers resistance to inactivation by the antibiotic fosfomycin. *Biochemistry 35*: 4923–4928.

Kim K-H, Bong Y-J, Park JK, Shin K-J, Hwang KY & Kim EE (2007) Structural basis for glutamate racemase inhibition. *J Mol Biol 372*: 434–443.

Kim MG, Strych U, Krause KL, Benedik MJ & Kohn H (2003a) Evaluation of amino-substituted heterocyclic derivatives as alanine racemase inhibitors. *Med Chem Res 12*: 130–138.

Kim MG, Strych U, Krause K, Benedik M & Kohn H (2003b) N(2)-substituted D,L-cycloserine derivatives: synthesis and evaluation as alanine racemase inhibitors. *J Antibiot (Tokyo)* 56: 160–168.

Kim MK, Cho MK, Song HE *et al.* (2007) Crystal structure of UDP-N-acetylenolpyruvylglucosamine reductase (MurB) from *Thermus caldophilus*. *Proteins 66*: 751–754.

Kim W-C, Rhee H-I, Park B-K, Suk K-H & Cha S-H (2000) Isolation of peptide ligands that inhibit glutamate racemase activity from a random phage display library. *J Biomol Screen 5*: 435–440.

Kimura K, Tran LS & Itoh Y (2004) Roles and regulation of the glutamate racemase isogenes, *racE* and *yrpC*, in *Bacillus subtilis*. *Microbiology 150*: 2911–2920.

Kobayashi K, Ehrlich SD, Albertini A *et al.* (2003) Essential *Bacillus subtilis* genes. *Proc Natl Acad Sci USA 100*: 4678–4683.

Kobayashi N & Go N (1997) ATP binding proteins with different folds share a common ATP-binding structural motif. *Nat Struct Biol 4*: 6–7.

Kollonitsch J, Barash L, Kahan FM & Kropp H (1973) New antibacterial agent via photofluorination of a bacterial cell wall constituent. *Nature 243*: 346–347.


Komatsuzawa H, Fujiwara T, Nishi H, Yamada S, Ohara M, McCallum N, Berger-Bächi B & Sugai M (2004) The gate controlling cell wall synthesis in *Staphylococcus aureus*. *Mol Microbiol 53*: 1221–1231.

Kong D-X, Zhu W-L, Wu D-L, Shen X & Jiang H-L (2007) Comparison of 3D-QSAR methods using a novel class of MurF inhibitors. *J Theor Comput Chem 6*: 63–80.

Kostrewa D, D’Arcy A, Takacs B & Kamber M (2001) Crystal structures of *Streptococcus pneumoniae* N-acetylglucosamine-1-phosphate uridyltransferase, GlmU, in apo form at 2.33 Å resolution and in complex with UDP-N-acetylglucosamine and Mg²⁺ at 1.96 Å resolution. *J Mol Biol 305*: 279–289.

Kotnik M, Oblak M, Humljan J, Gobec S, Urleb U & Šolmajer T (2004) Quantitative structure-activity relationships of *Streptococcus pneumoniae* MurD transition-state inhibitors. *QSAR Comb Sci 23*: 399–405.

Kotnik M, Štefanič Anderluh P & Preželj A (2007a) Development of novel inhibitors targeting intracellular steps of peptidoglycan biosynthesis. *Curr Pharm Des 13*: 2283–2309.

Kotnik M, Humljan J, Contreras-Martel C *et al.* (2007b) Structural and functional characterization of enantiomeric glutamic acid derivatives as potential transition state analogue inhibitors of MurD ligase. *J Mol Biol 370*: 107–115.

Kovač A, Majce V, Lenaršič R, Bombek S, Bostock JM, Chopra I, Polanc S & Gobec S (2007) Diazenedicarboxamides as inhibitors of D-alanine-D-alanine ligase (Ddl). *Bioorg Med Chem Lett 17*: 2047–2054.

Krekel F, Samland AK, Macheroux P, Amrhein N & Evans JN (2000) Determination of the p*Kₐ* value of C115 in MurA (UDP-N-acetylglucosamine enolpyruvyltransferase) from *Enterobacter cloacae*. *Biochemistry 39*: 12671–12677.

Kurosu M, Mahapatra S, Narayanasamy P & Crick DC (2007) Chemoenzymatic synthesis of Park’s nucleotide: toward the development of high-throughput screening of MraY inhibitors. *Tetrahedron Lett 48*: 799–803.

Kutterer KM, Davis JM, Singh G, Yang Y, Hu W, Severin A, Rasmussen BA, Krishnamurthy G, Failli A & Katz AH (2005) 4-Alkyl and 4,4′-dialkyl 1,2-bis(4-chlorophenyl)pyrazolidine-3,5-dione derivatives as new inhibitors of bacterial cell wall biosynthesis. *Bioorg Med Chem Lett 15*: 2527–2531.

Kuzin AP, Sun T, Jorczak-Baillass J, Healy VL, Walsh CT & Knox JR (2000) Enzymes of vancomycin resistance: the structure of D-alanine-D-lactate ligase of naturally resistant *Leuconostoc mesenteroides*. *Structure 8*: 463–470.

Labaudiniere RF, Xiang Y, Jalluri RK & Arvanites AC (2005) Antibiotic cycloalkyltetrahydroquinolone derivatives. US Patent, application number WO2005025556 A2.

Lacoste A-M, Poulsen M, Cassaigne A & Neuzil E (1979) Inhibition of D-alanyl-D-alanine ligase in different bacterial species by amino phosphonic acids. *Curr Microbiol 2*: 113–117.

Lacoste A-M, Chollet-Gravey A-M, Vo Quang L, Vo Quang Y & Le Goffic F (1991) Time-dependent inhibition of *Streptococcus faecalis* D-alanine : D-alanine ligase by α-aminophosphonamidic acids. *Eur J Med Chem 26*: 255–260.

Lee JH, Na Y, Song HE *et al.* (2006a) Crystal structure of the apo form of D-alanine:D-alanine ligase (Ddl) from *Thermus caldophilus*: a basis for the substrate-induced conformational changes. *Proteins* 64: 1078–1082.

Lee SG, Hong SP, Song JJ, Kim SJ, Kwak MS & Sung MH (2006b) Functional and structural characterization of thermostable D-amino acid aminotransferases from *Geobacillus spp*. *Appl Environ Microbiol* 72: 1588–1594.

Lees WJ & Walsh CT (1995) Analysis of the enol ether transfer catalyzed by UDP-GlcNAc enolpyruvyl transferase using (*E*)- and (*Z*)-isomers of phosphoenolbutyrate: stereochemical, partitioning, and isotope effect studies. *J Am Chem Soc* 117: 7329–7337.

Lees WJ, Benson TE, Hogle JM & Walsh CT (1996) (*E*)-Enolbutyryl-UDP-N-acetylglucosamine as a mechanistic probe of UDP-N-acetylenolpyruvylglucosamine reductase (MurB). *Biochemistry* 35: 1342–1351.

LeMagueres P, Im H, Ebalunode J, Strych U, Benedik MJ, Briggs JM, Kohn H & Krause KL (2005) The 1.9 Å crystal structure of alanine racemase from *Mycobacterium tuberculosis* contains a conserved entryway into the active site. *Biochemistry* 44: 1471–1481.

Le Roux P, Blanot D, Mengin-Lecreulx D & van Heijenoort J (1991) Peptides containing 2-aminopimelic acid. Synthesis and study of *in vitro* effects on bacterial cells. *Int J Peptide Protein Res* 37: 103–111.

Le Roux P, Auger G, van Heijenoort J & Blanot D (1992) Synthesis of new peptide inhibitors of the meso-diaminopimelate-adding enzyme. *Eur J Med Chem* 27: 899–907.

Leung DK, Andrews PR, Craik DJ, Iskander MN & Winkler DA (1985) Design, synthesis and testing of transition state analogues of alanine racemase as antibacterials. *Australian J Chem* 38: 297–206.

Li Z, Francisco GD, Hu W *et al.* (2003) 2-Phenyl-5,6-dihydro-2H-thieno[3,2-*c*]pyrazol-3-ol derivatives as new inhibitors of bacterial cell wall biosynthesis. *Bioorg Med Chem Lett* 13: 2591–2594.

Liger D, Blanot D & van Heijenoort J (1991) Effect of various alanine analogues on the L-alanine-adding enzyme from *Escherichia coli*. *FEMS Microbiol Lett* 80: 111–116.

Liger D, Masson A, Blanot D, van Heijenoort J & Parquet C (1995) Over-production, purification and properties of the uridine-diphosphate-N-acetylmuramate:L-alanine ligase from *Escherichia coli*. *Eur J Biochem* 230: 80–87.

Liger D, Masson A, Blanot D, van Heijenoort J & Parquet C (1996) Study of the overproduced uridine-diphosphate-N-acetylmuramate:L-alanine ligase from *Escherichia coli*. *Microb Drug Resist* 2: 25–27.

Liu H, Sadamoto R, Sears PS & Wong C-H (2001) An efficient chemoenzymatic strategy for the synthesis of wild-type and vancomycin-resistant bacterial cell-wall precursors: UDP-N-acetylmuramyl-peptides. *J Am Chem Soc* 123: 9916–9917.

Liu S, Chang JS, Herberg JT, Horng M-M, Tomich PK, Lin AH & Marotti KR (2006) Allosteric inhibition of *Staphylococcus*

aureus D-alanine:D-alanine ligase revealed by crystallographic studies. *Proc Natl Acad Sci USA* 103: 15178–15183.

Longenecker K, Stamper GF, Hajduk PJ *et al.* (2005) Structure of MurF from *Streptococcus pneumoniae* co-crystallized with a small molecule inhibitor exhibits interdomain closure. *Protein Sci* 14: 3039–3047.

Ludovice AM, Wu SW & de Lencastre H (1998) Molecular cloning and DNA sequencing of the *Staphylococcus aureus* UDP-N-acetylmuramyl tripeptide synthetase (*murE*) gene, essential for the optimal expression of methicillin resistance. *Microb Drug Resist* 4: 85–90.

Lundqvist T, Fisher SL, Kern G, Folmer RH, Xue Y, Newton DT, Keating TA, Alm RA & de Jonge BL (2007) Exploitation of structural and regulatory diversity in glutamate racemases. *Nature* 447: 817–822.

Mahapatra S, Crick DC & Brennan PJ (2000) Comparison of the UDP-N-acetylmuramate:L-alanine ligase enzymes from *Mycobacterium tuberculosis* and *Mycobacterium leprae*. *J Bacteriol* 182: 6827–6830.

Major DT & Gao J (2006) A combined quantum mechanical and molecular mechanical study of the reaction mechanism and α-amino acidity in alanine racemase. *J Am Chem Soc* 128: 16345–16357.

Marmor S, Petersen CP, Reck F, Yang W, Gao N & Fisher SL (2001) Biochemical characterization of a phosphinate inhibitor of *Escherichia coli* MurC. *Biochemistry* 40: 12207–12214.

Marquardt JL, Siegele DA, Kolter R & Walsh CT (1992) Cloning and sequencing of *Escherichia coli murZ* and purification of its product, a UDP-N-acetylglucosamine enolpyruvyl transferase. *J Bacteriol* 174: 5748–5752.

Marquardt JL, Brown ED, Walsh CT & Anderson KS (1993) Isolation and structural elucidation of a tetrahedral intermediate in the UDP-N-acetylglucosamine enolpyruvoyl transferase enzymatic pathway. *J Am Chem Soc* 115: 10398–10399.

Marquardt JL, Brown ED, Lane WS, Haley TM, Ichikawa Y, Wong CH & Walsh CT (1994) Kinetics, stoichiometry, and identification of the reactive thiolate in the inactivation of UDP-GlcNAc enolpyruvoyl transferase by the antibiotic fosfomycin. *Biochemistry* 33: 10646–10651.

Maruyama D, Nishitani Y, Nonaka T, Kita A, Fukami TA, Mio T, Yamada-Okabe H, Yamada-Okabe T & Miki K (2007) Crystal structure of uridine-diphospho-N-acetylglucosamine pyrophosphorylase from *Candida albicans* and catalytic reaction mechanism. *J Biol Chem* 282: 17221–17230.

May M, Mehboob S, Mulhearn DC, Wang Z, Yu H, Thatcher GR, Santarsiero BD, Johnson ME & Mesecar AD (2007) Structural and functional analysis of two glutamate racemase isozymes from *Bacillus anthracis* and implications for inhibitor design. *J Mol Biol* 371: 1219–1237.

McCoy AJ & Maurelli AT (2005) Characterization of *Chlamydia* MurC-Ddl, a fusion protein exhibiting D-alanyl-D-alanine ligase activity involved in peptidoglycan synthesis and D-cycloserine sensitivity. *Mol Microbiol* 57: 41–52.
McCoy AJ, Sandlin RC & Maurelli AT (2003) *In vitro* and *in vivo* functional activity of *Chlamydia* MurA, a UDP-N-acetylglucosamine enolpyruvyl transferase involved in peptidoglycan synthesis and fosfomycin resistance. *J Bacteriol* 185: 1218–1228.

McDermott AE, Creuzet F, Griffin RG, Zawadzke LE, Ye Q-Z & Walsh CT (1990) Rotational resonance determination of the structure of an enzyme-inhibitor complex: phosphorylation of an (aminoalkyl)phosphinate inhibitor of D-alanyl-D-alanine ligase by ATP. *Biochemistry* 29: 5767–5775.

Mengin-Lecreulx D & van Heijenoort J (1993) Identification of the *glmU* gene encoding N-acetylglucosamine-1-phosphate uridyltransferase in *Escherichia coli*. *J Bacteriol* 175: 6150–6157.

Mengin-Lecreulx D & van Heijenoort J (1994) Copurification of glucosamine-1-phosphate acetyltransferase and N-acetylg glucosamine-1-phosphate uridyltransferase activities of *Escherichia coli*: characterization of the *glmU* gene product as a bifunctional enzyme catalyzing two subsequent steps in the pathway for UDP-N-acetylglucosamine synthesis. *J Bacteriol* 176: 5788–5795.

Mengin-Lecreulx D & van Heijenoort J (1996) Characterization of the essential gene *glmM* encoding phosphoglucosamine mutase in *Escherichia coli*. *J Biol Chem* 271: 32–39.

Mengin-Lecreulx D, Flouret B & van Heijenoort J (1982) Cytoplasmic steps of peptidoglycan synthesis in *Escherichia coli*. *J Bacteriol* 151: 1109–1117.

Mengin-Lecreulx D, Michaud C, Richaud C, Blanot D & van Heijenoort J (1988) Incorporation of LL-diaminopimelic acid into peptidoglycan of *Escherichia coli* mutants lacking diaminopimelate epimerase encoded by *dapF*. *J Bacteriol* 170: 2031–2039.

Mengin-Lecreulx D, Parquet C, Desviat LR, Plá J, Flouret B, Ayala JA & van Heijenoort J (1989) Organization of the *murE-murG* region of *Escherichia coli*: identification of the *murD* gene encoding the D-glutamic-acid-adding enzyme. *J Bacteriol* 171: 6126–6134.

Mengin-Lecreulx D, Blanot D & van Heijenoort J (1994) Replacement of diaminopimelic acid by cystathionine or lanthionine in the peptidoglycan of *Escherichia coli*. *J Bacteriol* 176: 4321–4327.

Mengin-Lecreulx D, van Heijenoort J & Park JT (1996) Identification of the *mpl* gene encoding UDP-N-acetylmuramate: L-alanyl-γ-D-glutamyl-meso-diaminopimelate ligase in *Escherichia coli* and its role in recycling of cell wall peptidoglycan. *J Bacteriol* 178: 5347–5352.

Mengin-Lecreulx D, Ayala J, Bouhss A, van Heijenoort J, Parquet C & Hara H (1998) Contribution of the P<sub>*mra*</sub> promoter to expression of genes in the *Escherichia coli mra* cluster of cell envelope biosynthesis and cell division genes. *J Bacteriol* 180: 4406–4412.

Mengin-Lecreulx D, Falla T, Blanot D, van Heijenoort J, Adams DJ & Chopra I (1999) Expression of the *Staphylococcus aureus* UDP-N-acetylmuramoyl-L-alanyl-D-glutamate: L-lysine ligase in *Escherichia coli* and effects on peptidoglycan biosynthesis and cell growth. *J Bacteriol* 181: 5909–5914.

Michaud C, Blanot D, Flouret B & van Heijenoort J (1987) Partial purification and specificity studies of the D-glutamate-adding and D-alanyl-D-alanine-adding enzymes from *Escherichia coli* K12. *Eur J Biochem* 166: 631–637.

Michaud C, Mengin-Lecreulx D, van Heijenoort J & Blanot D (1990) Over-production, purification and properties of the uridine-diphosphate-N-acetylmuramoyl-L-alanine-D-glutamate: meso-2,6-diaminopimelate ligase from *Escherichia coli*. *Eur J Biochem* 194: 853–861.

Milewski S, Gabriel I & Olchowy J (2006) Enzymes of UDP-GlcNAc biosynthesis in yeast. *Yeast* 23: 1–14.

Miller J, Hammond SM, AnderluZZi D & Bugg TDH (1998) Aminoalkylphosphinate inhibitors of D-Ala-D-Ala adding enzyme. *J Chem Soc, Perkin Trans 1* 131–142.

Mio T, Yabe T, Arisawa M & Yamada-Okabe H (1998) The eukaryotic UDP-N-acetylglucosamine pyrophosphorylases: gene cloning, protein expression, and catalytic mechanism. *J Biol Chem* 273: 14392–14397.

Mio T, Yamada-Okabe T, Arisawa M & Yamada-Okabe H (1999) Saccharomyces cerevisiae GNA1, an essential gene encoding a novel acetyltransferase involved in UDP-N-acetylglucosamine synthesis. *J Biol Chem* 274: 424–429.

Miyakawa T, Matsuzawa H, Matsuhashi M & Sugino Y (1972) Cell wall peptidoglycan mutants of *Escherichia coli* K-12: existence of two clusters of genes, *mra* and *mrb*, for cell wall peptidoglycan biosynthesis. *J Bacteriol* 112: 950–958.

Mizyed S, Oddone A, Byczynski B, Hughes DW & Berti PJ (2005) UDP-N-acetylmuramic acid (UDP-MurNAc) is a potent inhibitor of MurA (enolpyruvyl-UDP-GlcNAc synthase). *Biochemistry* 44: 4011–4017.

Möbitz H & Bruice TC (2004) Multiple substrate binding states and chiral recognition in cofactor-independent glutamate racemase: a molecular dynamics study. *Biochemistry* 43: 9685–9694.

Moe ST, Ala PJ & Perola E (2003) Heterocyclic compounds and uses thereof as D-alanyl-D-alanine ligase inhibitors. US Patent, application number PCT/US02/20567, WO 03/001887 A2.

Mol CD, Brooun A, Dougan DR, Hilgers MT, Tari LW, Wijnands RA, Knuth MW, McRee DE & Swanson RW (2003) Crystal structures of active fully assembled substrate- and product-bound complexes of UDP-N-acetylmuramic acid: L-alanine ligase (MurC) from *Haemophilus influenzae*. *J Bacteriol* 185: 4152–4162.

Molina-López J, Sanschagrin F & Levesque RC (2006) A peptide inhibitor of MurA UDP-N-acetylglucosamine enolpyruvyl transferase: the first committed step in peptidoglycan biosynthesis. *Peptides* 27: 3115–3121.

Morollo AA, Petsko GA & Ringe D (1999) Structure of a Michaelis complex analogue: propionate binds in the substrate carboxylate site of alanine racemase. *Biochemistry* 38: 3293–3301.

Moulder JW (1993) Why is *Chlamydia* sensitive to penicillin in the absence of peptidoglycan? *Infect Agents Dis* 2: 87–99.

Mullins LS, Zawadzke LE, Walsh CT & Raushel FM (1990) Kinetic evidence for the formation of D-alanyl phosphate in

© 2008 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved

FEMS Microbiol Rev **32** (2008) 168–207
the mechanism of D-alanyl-D-alanine ligase. J Biol Chem 265: 8993–8998.

Mustata GI & Briggs JM (2002) A structure-based design approach for the identification of novel inhibitors: application to an alanine racemase. J Comput Aided Mol Des 16: 935–953.

Nakajima N, Tanizawa K, Tanaka H & Soda K (1986) Cloning and expression in Escherichia coli of the glutamate racemase gene from Pediococcus pentosaceus. Agric Biol Chem 50: 2823–2830.

Narayan RS & VanNieuwenhze MS (2007) Synthesis of substrates and biochemical probes for study of the peptidoglycan biosynthetic pathway. Eur J Org Chem 1399–1414.

Navia MA, Ala PJ, Griffith JP, Ali JA, Faerman CH, Moe ST, Magee AS, Connelly PR & Perola E (2003) Structure-based drug design methods for identifying D-Ala-D-Ala ligase inhibitors as antibacterial drugs. US Patent, application number PCT/US02/20465, WO 03/002063 A2.

Neuhaus FC (1962a) The enzymatic synthesis of D-alanyl-D-alanine. I. Purification and properties of D-alanyl-D-alanine synthetase. J Biol Chem 237: 778–786.

Neuhaus FC (1962b) The enzymatic synthesis of D-alanyl-D-alanine. II. Kinetic studies on D-alanyl-D-alanine synthetase. J Biol Chem 237: 3128–3135.

Neuhaus FC & Lynch JL (1964) The enzymatic synthesis of D-alanyl-D-alanine. III. On the inhibition of D-alanyl-D-alanine synthetase by the antibiotic D-cycloserine. Biochemistry 3: 471–480.

Neuhaus FC & Struve WG (1965) Enzymatic synthesis of analogs of the cell-wall precursor. I. Kinetics and specificity of uridine diphospho-N-acetylmuramyl-L-alanyl-D-glutamyl-L-lysine: D-alanyl-D-alanine ligase (adenosine diphosphate) from Streptococcus faecalis R. Biochemistry 4: 120–131.

Neuhaus FC & Hammes WP (1981) Inhibition of cell wall biosynthesis by analogues of alanine. Pharmac Ther 14: 265–319.

Nishida S, Kurokawa K, Matsuo M, Sakamoto K, Ueno K, Kita K & Sekimizu K (2006) Identification and characterization of amino acid residues essential for the active site of UDP-N-acetylenolpyruvylglucosamine reductase (MurB) from Staphylococcus aureus. J Biol Chem 281: 1714–1724.

Nishitani Y, Maruyama D, Nonaka T, Kita A, Fukami TA, Mio T, Yamada-Okabe H, Yamada-Okabe T & Miki K (2006) Crystal structures of N-acetylglucosamine-phosphate mutase, a member of the α-D-phosphohexomutase superfamily, and its substrate and product complexes. J Biol Chem 281: 19740–19747.

Nosal F, Masson A, Legrand R, Blanot D, Schoot B, van Heijenoort J & Parquet C (1998) Site-directed mutagenesis and chemical modification of the two cysteine residues of the UDP-N-acetylmuramoyl: L-alanine ligase of Escherichia coli. FEBS Lett 426: 309–313.

Obmolova G, Badet-Denisot MA, Badet B & Teplyakov A (1994) Crystallization and preliminary X-ray analysis of the two domains of glucosamine-6-phosphate synthase from Escherichia coli. J Mol Biol 242: 703–705.


Oikawa T, Tauch A, Schaffer S & Fujioka T (2006) Expression of alr gene from Corynebacterium glutamicum ATCC 13032 in Escherichia coli and molecular characterization of the recombinant alanine racemase. J Biotechnol 125: 503–512.

Olsen LR & Roderick SL (2001) Structure of the Escherichia coli GlmU pyrophosphorylase and acetyltransferase active sites. Biochemistry 40: 1913–1921.

Olsen LR, Vetting MW & Roderick SL (2007) Structure of the E. coli bifunctional GlmU acetyltransferase active site with substrates and products. Protein Sci 16: 1230–1235.

Paradis-Bleau C, Beaumont M, Boudreault L, Lloyd A, Sanschagrin F, Bugg TD & Levesque RC (2006) Selection of peptide inhibitors against the Pseudomonas aeruginosa MurD cell wall enzyme. Peptides 27: 1693–1700.

Park JT (1952) Uridine 5′-pyrophosphate derivatives. I. Isolation from Staphylococcus aureus. J Biol Chem 194: 877–884.

Parsons WH, Patchett AA, Bull HG *et al.* (1988) Phosphinic acid inhibitors of D-alanyl-D-alanine ligase. J Med Chem 31: 1772–1778.

Patchett AA, Taub D, Weissberger B, Valiant ME, Gadebusch H, Thornberry NA & Bull HG (1988) Antibacterial activities of fluorovinyl- and chlorovinylglycine and several derived dipeptides. Antimicrob Agents Chemother 32: 319–323.

Patte J-C (1983) Diaminopimelate and lysine. *Amino Acids: Biosynthesis and Genetic Regulation* (Herrmann KL & Somerville RL, eds), pp. 213–228. Addison-Wesley Publishing Co., Reading, MA.

Peisach D, Chipman DM, Van Ophem PW, Manning JM & Ringe D (1998) Crystallographic study of steps along the reaction pathway of D-amino acid aminotransferase. Biochemistry 37: 4958–4967.

Peneff C, Mengin-Lecreulx D & Bourne Y (2001a) The crystal structures of Apo and complexed Saccharomyces cerevisiae GNA1 shed light on the catalytic mechanism of an amino-sugar N-acetyltransferase. J Biol Chem 276: 16328–16334.

Peneff C, Ferrari P, Charrier V, Taburet Y, Monnier C, Zamboni V, Winter J, Harnois M, Fassy F & Bourne Y (2001b) Crystal structures of two human pyrophosphorylase isoforms in complexes with UDPGlc(Gal)NAc: role of the alternatively spliced insert in the enzyme oligomeric assembly and active site architecture. EMBO J 20: 6191–6202.

Perdih A, Kotnik M, Hodošček M & Šolmajer T (2007) Targeted molecular dynamics simulation studies of binding and conformational changes in E. coli MurD. Proteins 68: 243–254.

Pompeo F, van Heijenoort J & Mengin-Lecreulx D (1998) Probing the role of cysteine residues in glucosamine-1-phosphate acetyltransferase activity of the bifunctional GlmU protein from Escherichia coli: site-directed mutagenesis and characterization of the mutant enzymes. J Bacteriol 180: 4799–4803.

Pompeo F, Bourne Y, van Heijenoort J, Fassy F & Mengin-Lecreulx D (2001) Dissection of the bifunctional Escherichia coli N-acetylglucosamine-1-phosphate uridyltransferase enzyme into autonomously functional domains and evidence that trimerization is absolutely required for glucosamine-1-

phosphate acetyltransferase activity and cell growth. *J Biol Chem* 276: 3833–3839.

Pratviel-Sosa F, Mengin-Lecreulx D & van Heijenoort J (1991) Over-production, purification and properties of the uridine diphosphate N-acetylmuramoyl-L-alanine: D-glutamate ligase from *Escherichia coli*. *Eur J Biochem* 202: 1169–1176.

Pratviel-Sosa F, Acher F, Trigalo F, Blanot D, Azerad R & van Heijenoort J (1994) Effect of various analogues of D-glutamic acid on the D-glutamate-adding enzyme from *Escherichia coli*. *FEMS Microbiol Lett* 115: 223–228.

Pucci MJ, Thanassi JA, Ho H-T, Falk PJ & Dougherty TJ (1995) *Staphylococcus haemolyticus* contains two D-glutamic acid biosynthetic activities, a glutamate racemase and a D-amino acid transaminase. *J Bacteriol* 177: 336–342.

Pucci MJ, Thanassi JA, Discotto IF, Kessler RE & Dougherty TJ (1997) Identification and characterization of cell wall-cell division gene clusters in pathogenic Gram-positive cocci. *J Bacteriol* 179: 5632–5635.

Puig E, Garcia-Viloa M, González-Lafont A, López I, Daura X & Lluch JM (2005) A molecular dynamics simulation of the binding modes of D-glutamate and D-glutamine to glutamate racemase. *J Chem Theory Comput* 1: 737–749.

Puig E, Garcia-Viloa M, González-Lafont A & Lluch JM (2006) On the ionization state of the substrate in the active site of glutamate racemase. A QM/MM study about the importance of being zwitterionic. *J Phys Chem A* 110: 717–725.

Raczynska J, Olchowy J, Konariiev PV, Svergun DI, Milewski S & Rypniewski W (2007) The crystal and solution studies of glucosamine-6-phosphate synthase from *Candida albicans*. *J Mol Biol* 372: 672–688.

Raetz CR & Roderick SL (1995) A left-handed parallel beta helix in the structure of UDP-N-acetylglucosamine acyltransferase. *Science* 270: 997–1000.

Ramilo C, Appleyard RJ, Wanke C, Krekel F, Amrhein N & Evans IN (1994) Detection of the covalent intermediate of UDP-N-acetylglucosamine enolpyruvyl transferase by solution-state and time-resolved solid-state NMR spectroscopy. *Biochemistry* 33: 15071–15079.

Ramos-Aires J, Plésiat P, Kocjancic-Curty L & Köhler T (2004) Selection of an antibiotic-hypersusceptible mutant of *Pseudomonas aeruginosa*: identification of the GlmR transcriptional regulator. *Antimicrob Agents Chemother* 48: 843–851.

Raymond JB, Price NP & Pavelka MS Jr (2003) A method for the enzymatic synthesis and HPLC purification of the peptidoglycan precursor UDP-N-acetylmuramic acid. *FEMS Microbiol Lett* 229: 83–89.

Reck F, Marmor S, Fisher S & Wuonola MA (2001) Inhibitors of the bacterial cell wall biosynthesis enzyme MurC. *Bioorg Med Chem Lett* 11: 1451–1454.

Reddy S, Waddell ST, Kuo DW, Wong KK & Pompliano DL (1999) Preparative enzymatic synthesis and characterization of the cytoplasmic intermediates of murein biosynthesis. *J Am Chem Soc* 121: 1175–1178.


Richaud C, Mengin-Lecreulx D, Pochet S, Johnson EJ, Cohen GN & Marlière P (1993) Directed evolution of biosynthetic pathways: recruitment of cysteine thiethers for constructing the cell wall of *Escherichia coli*. *J Biol Chem* 268: 26827–26835.

Rios A, Amyes TL & Richard JP (2000) Formation and stability of organic zwitterions in aqueous solution: enolates of the amino acid glycine and its derivatives. *J Am Chem Soc* 122: 9373–9385.

Rogers HJ, Perkins HR & Ward JB (1980) *Microbial Cell Walls and Membranes*. Chapman and Hall, London, UK.

Roper DI, Huyton T, Vagin A & Dodson G (2000) The molecular basis of vancomycin resistance in clinically relevant *Enterococci*: crystal structure of D-alanyl-D-lactate ligase (VanA). *Proc Natl Acad Sci USA* 97: 8921–8925.

Ruzheinikov SN, Taal MA, Sedelnikova SE, Baker PJ & Rice DW (2005) Substrate-induced conformational changes in *Bacillus subtilis* glutamate racemase and their implications for drug discovery. *Structure* 13: 1707–1713.

Saito M, Nishimura K, Hasegawa Y, Shinohara T, Wakabayashi S, Kurihara T, Ishizuka M & Nagata Y (2007) Alanine racemase from *Helicobacter pylori* NCTC 11637: purification, characterization and gene cloning. *Life Sci* 80: 788–794.

Samland AK, Amrhein N & Macheroux P (1999) Lysine 22 in UDP-N-acetylglucosamine enolpyruvyl transferase from *Enterobacter cloacae* is crucial for enzymatic activity and the formation of covalent adducts with the substrate phosphoenolpyruvate and the antibiotic fosfomycin. *Biochemistry* 38: 13162–13169.

Samland AK, Etezady-Esfarjani T, Amrhein N & Macheroux P (2001) Asparagine 23 and aspartate 305 are essential residues in the active site of UDP-N-acetylglucosamine enolpyruvyl transferase from *Enterobacter cloacae*. *Biochemistry* 40: 1550–1559.

Sarver DW, Rogers JM & Epps DM (2002) Determination of ligand-MurB interactions by isothermal denaturation: application as a secondary assay to complement high throughput screening. *J Biomol Screen* 7: 21–28.

Sato M, Kirimura K & Kino K (2005) D-Amino acid dipeptide production utilizing D-alanine-D-alanine ligases with novel substrate specificity. *J Biosci Bioeng* 99: 623–628.

Sato M, Kirimura K & Kino K (2006) Substrate specificity of thermostable D-alanine-D-alanine ligase from *Thermotoga maritima* ATCC 43589. *Biosci Biotechnol Biochem* 70: 2790–2792.

Sauvage E, Charlier P, Terrak M & Ayala JA (2008) The structure of Penicillin-Binding Proteins and their role in peptidoglycan biosynthesis. *FEMS Microbiol Rev*, doi: 10.1111/j.1574-6976.2008.00105.x

Sawada S, Tanaka Y, Hayashi S, Ryu M, Hasegawa T, Yamamoto Y, Esaki N, Soda K & Takahashi S (1994) Kinetics of thermostable alanine racemase of *Bacillus stearothermophilus*. *Biosci Biotechnol Biochem* 58: 807–811.

Schleifer KH & Kandler O (1972) Peptidoglycan types of bacterial cell walls and their taxonomic implications. *Bacteriol Rev* 36: 407–477.

Schönbrunn E, Sack S, Eschenburg S, Perrakis A, Krekel F, Amrhein N & Mandelkow E (1996) Crystal structure of UDP-N-acetylglucosamine enolpyruvyltransferase, the target of the antibiotic fosfomycin. *Structure* 4: 1065–1075.

Schönbrunn E, Svergun DI, Amrhein N & Koch MH (1998) Studies on the conformational changes in the bacterial cell wall biosynthetic enzyme UDP-N-acetylglucosamine enolpyruvyltransferase (MurA). *Eur J Biochem* 253: 406–412.

Schönbrunn E, Eschenburg S, Luger K, Kabsch W & Amrhein N (2000a) Structural basis for the interaction of the fluorescence probe 8-anilino-1-naphthalene sulfonate (ANS) with the antibiotic target MurA. *Proc Natl Acad Sci USA* 97: 6345–6349.

Schönbrunn E, Eschenburg S, Krekel F, Luger K & Amrhein N (2000b) Role of the loop containing residue 115 in the induced-fit mechanism of the bacterial cell wall biosynthetic enzyme MurA. *Biochemistry* 39: 2164–2173.

Schouten JA, Bagga S, Lloyd AJ, de Pascale G, Dowson CG, Roper DI & Bugg TD (2006) Fluorescent reagents for *in vitro* studies of lipid-linked steps of bacterial peptidoglycan biosynthesis: derivatives of UDPMurNAc-pentapeptide containing D-cysteine at position 4 or 5. *Mol Biosyst* 2: 484–491.

Seebeck FP & Hilvert D (2003) Conversion of a PLP-dependent racemase into an aldolase by a single active site mutation. *J Am Chem Soc* 125: 10158–10159.

Sengupta S, Shah M & Nagaraja V (2006) Glutamate racemase from *Mycobacterium tuberculosis* inhibits DNA gyrase by affecting its DNA-binding. *Nucleic Acids Res* 34: 5567–5576.

Seow TK, Inagaki K, Tamura T, Soda K & Tanaka H (1998) Alanine racemase from an acidophile, *Acidiphilium organovorum*: purification and characterisation. *Biosci Biotechnol Biochem* 62: 242–247.

Seow TK, Inagaki K, Nakamura T, Maeda R, Tamura T & Tanaka H (2000) Purification and some characteristics of a monomeric alanine racemase from an extreme thermophile, *Thermus thermophilus*. *J Biosci Bioeng* 90: 344–346.

Shatalin KY & Neyfakh AA (2005) Efficient gene inactivation in *Bacillus anthracis*. *FEMS Microbiol Lett* 245: 315–319.

Shaw JP, Petsko GA & Ringe D (1997) Determination of the structure of alanine racemase from *Bacillus stearothermophilus* at 1.9-Å resolution. *Biochemistry* 36: 1329–1342.

Sheng Y, Sun X, Shen Y, Bognar AL, Baker EN & Smith CA (2000) Structural and functional similarities in the ADP-forming amide bond ligase superfamily: implications for a substrate-induced conformational change in folylpolyglutamate synthetase. *J Mol Biol* 302: 427–440.

Sim MM, Ng SB, Buss AD, Crasta SC, Goh KL & Lee SK (2002) Benzylidene rhodanines as novel inhibitors of UDP-N-acetylmuramate/L-alanine ligase. *Bioorg Med Chem Lett* 12: 697–699.

Skarzynski T, Mistry A, Wonacott A, Hutchinson SE, Kelly VA & Duncan K (1996) Structure of UDP-N-acetylglucosamine enolpyruvyl transferase, an enzyme essential for the synthesis of bacterial peptidoglycan, complexed with substrate UDP-N-acetylglucosamine and the drug fosfomycin. *Structure* 4: 1465–1474.

Skarzynski T, Kim DH, Lees WJ, Walsh CT & Duncan K (1998) Stereochemical course of enzymatic enolpyruvyl transfer and catalytic conformation of the active site revealed by the crystal structure of the fluorinated analogue of the reaction tetrahedral intermediate bound to the active site of the C115A mutant of MurA. *Biochemistry* 37: 2572–2577.

Smith CA (2006) Structure, function and dynamics in the mur family of bacterial cell wall ligases. *J Mol Biol* 362: 640–655.

Snyder NJ, Tebbe MJ, Victor F *et al.* (1999) Synthesis and evaluation of *Streptococcus pneumoniae* MurD transition state inhibitors. *Abstracts of the 39th Interscience Conference on Antimicrobial Agents and Chemotherapy*, September 26–29, San Francisco, USA, p. 330.

Spraggon G, Schwarzenbacher R, Kreusch A *et al.* (2004) Crystal structure of an Udp-N-acetylmuramate-alanine ligase MurC (TM0231) from *Thermotoga maritima* at 2.3 Å resolution. *Proteins* 55: 1078–1081.

Stamper GF, Morollo AA & Ringe D (1998) Reaction of alanine racemase with 1-aminoethylphosphonic acid forms a stable external aldimine. *Biochemistry* 37: 10438–10445.

Stamper GF, Longenecker K, Fry EH *et al.* (2006) Structure-based optimization of MurF inhibitors. *Chem Biol Drug Res* 67: 58–65.

Strancar K, Blanot D & Gobec S (2006) Design, synthesis and structure-activity relationships of new phosphinate inhibitors of MurD. *Bioorg Med Chem Lett* 16: 343–348.

Strancar K, Boniface A, Blanot D & Gobec S (2007) Phosphinate inhibitors of UDP-N-acetylmuramoyl-L-alanyl-D-glutamate: L-lysine ligase (MurE). *Arch Pharm Chem Life Sci* 340: 127–134.

Strominger JL, Ito E & Threnn RH (1960) Competitive inhibition of enzymatic reactions by oxamycin. *J Am Chem Soc* 82: 998–999.

Strych U & Benedik MJ (2002) Mutant analysis shows that alanine racemases from *Pseudomonas aeruginosa* and *Escherichia coli* are dimeric. *J Bacteriol* 184: 4321–4325.

Strych U, Huang HC, Krause KL & Benedik MJ (2000) Characterization of the alanine racemases from *Pseudomonas aeruginosa* PAO1. *Curr Microbiol* 41: 290–294.

Strych U, Penland RL, Jimenez M, Krause KL & Benedik MJ (2001) Characterization of the alanine racemases from two mycobacteria. *FEMS Microbiol Lett* 196: 93–98.

Sugio S, Petsko GA, Manning JM, Soda K & Ringe D (1995) Crystal structure of a D-amino acid aminotransferase: how the protein controls stereoselectivity. *Biochemistry* 34: 9661–9669.

Sugio S, Kashima A, Kishimoto K, Peisach D, Petsko GA, Ringe D, Yoshimura T & Esaki N (1998) Crystal structures of L201A mutant of D-amino acid aminotransferase at 2.0 Å resolution: implication of the structural role of Leu201 in transamination. *Protein Eng* 11: 613–619.

Sulzenbacher G, Gal L, Peneff C, Fassy F & Bourne Y (2001) Crystal structure of *Streptococcus pneumoniae* N-acetylglucosamine-1-phosphate uridyltransferase bound to acetyl-coenzyme A reveals a novel active site architecture. *J Biol Chem* 276: 11844–11851.

Sun S & Toney MD (1999) Evidence for a two-base mechanism involving tyrosine-265 from arginine-219 mutants of alanine racemase. *Biochemistry* 38: 4058–4065.

Sun X, Cross JA, Bognar AL, Baker EN & Smith CA (2001) Folate-binding triggers the activation of folylpolyglutamate synthetase. *J Mol Biol* 310: 1067–1078.

Szumilo T, Zeng Y, Pastuszak I, Drake R, Szumilo H & Elbein AD (1996) Purification to homogeneity and properties of UDP-GlcNAc (GalNAc) pyrophosphorylase. *J Biol Chem* 271: 13147–13154.

Tanizawa K, Masu Y, Asano S, Tanaka H & Soda K (1989) Thermostable D-amino acid aminotransferase from a thermophilic *Bacillus* species. Purification, characterization, and active site sequence determination. *J Biol Chem* 264: 2445–2449.

Tanner ME (2002) Understanding nature’s strategies for enzyme-catalyzed racemization and epimerization. *Acc Chem Res* 35: 237–246.

Tanner ME & Miao S (1994) The synthesis and stability of aziridino-glutamate, an irreversible inhibitor of glutamate racemase. *Tetrahedron Lett* 35: 4073–4076.

Tanner ME, Vaganay S, van Heijenoort J & Blanot D (1996) Phosphinate inhibitors of the D-glutamic acid-adding enzyme of peptidoglycan biosynthesis. *J Org Chem* 61: 1756–1760.

Tavares IM, Jolly L, Pompeo F, Leitão JH, Fialho AM, Sá-Correia I & Mengin-Lecreulx D (2000) Identification of the *Pseudomonas aeruginosa glmM* gene, encoding phosphoglucosamine mutase. *J Bacteriol* 182: 4453–4457.

Tayeh MA, Dotson GD, Clemens JC & Woodard RW (1995) Overproduction and one-step purification of *Escherichia coli* UDP-N-acetylglucosamine enolpyruvyl reductase. *Protein Expr Purif* 6: 757–762.

Taylor PP & Fortheringham IG (1997) Nucleotide sequence of the *Bacillus licheniformis* ATCC 10716 *dat* gene and comparison of the predicted amino acid sequence with those of other bacterial species. *Biochim Biophys Acta* 1350: 38–40.

Teplyakov A, Obmolova G, Badet-Denisot MA, Badet B & Polikarpov I (1998) Involvement of the C terminus in intramolecular nitrogen channeling in glucosamine 6-phosphate synthase: evidence from a 1.6 Å crystal structure of the isomerase domain. *Structure* 6: 1047–1055.

Teplyakov A, Obmolova G, Badet B & Badet-Denisot MA (2001) Channeling of ammonia in glucosamine-6-phosphate synthase. *J Mol Biol* 313: 1093–1102.

Thompson RJ, Bouwer HG, Portnoy DA & Frankel FR (1998) Pathogenicity and immunogenicity of a *Listeria monocytogenes* strain that requires D-alanine for growth. *Infect Immun* 66: 3552–3561.

Thornberry NA, Bull HG, Taub D, Greenlee WJ, Patchett AA & Cordes EH (1987) 3-Halovinylglycines. Efficient irreversible inhibitors of *E. coli* alanine racemase. *J Am Chem Soc* 109: 7543–7544.

Thornberry NA, Bull HG, Taub D, Wilson KE, Gimenez-Gallego G, Rosegay A, Soderman DD & Patchett AA (1991)

Mechanism-based inactivation of alanine racemase by 3-halovinylglycines. *J Biol Chem* 266: 21657–21665.

Traxinger RR & Marshall S (1991) Coordinated regulation of glutamine: fructose-6-phosphate amidotransferase activity by insulin, glucose, and glutamine. Role of hexosamine biosynthesis in enzyme regulation. *J Biol Chem* 266: 10148–10154.

Ueda T, Feng F, Sadamoto R, Niikura K, Monde K & Nishimura S-I (2004) Synthesis of 4-fluorinated UDP-MurNAc pentapeptide as an inhibitor of bacterial growth. *Org Lett* 6: 1753–1756.

Vaara M (1992) Eight bacterial proteins, including UDP-N-acetylglycosamine acyltransferase (LpxA) and three other transferases of *Escherichia coli*, consist of a six-residue periodicity theme. *FEMS Microbiol Lett* 76: 249–254.

Vaganay S, Tanner ME, van Heijenoort J & Blanot D (1996) Study of the reaction mechanism of the D-glutamic acid-adding enzyme from *Escherichia coli*. *Microb Drug Resist* 2: 51–54.

van Assche I, Soroka M, Haemers A, Hooper M, Blanot D & van Heijenoort J (1991) Synthesis and antibacterial evaluation of phosphonic acid analogues of diaminopimelic acid. *Eur J Med Chem* 26: 505–515.

van Heijenoort J (2001) Recent advances in the formation of the bacterial peptidoglycan monomer subunit. *Nat Prod Rep* 18: 503–519.

van Ophem PW, Peisach D, Erickson SD, Soda K, Ringe D & Manning JM (1999) Effects of the E177K mutation in D-amino acid transaminase. Studies on an essential coenzyme anchoring group that contributes to stereochemical fidelity. *Biochemistry* 38: 1323–1331.

Victor F, Tebbe MJ, Birch GM, Smith MC, Letourneau DL & Wu CE (1999) D-Glutamic acid analogs as *Streptococcus pneumoniae* UDP-N-acetyl muramoyl-alanine:D-glutamate ligase (MurD) inhibitors. *Abstracts of the 39th Interscience Conference on Antimicrobial Agents and Chemotherapy*, September 26–29, San Francisco, USA, p. 330.

Volkman BF, Zhang Q, Debabov DV, Rivera E, Kresheck GC & Neuhaus FC (2001) Biosynthesis of D-alanyl-lipoteichoic acid: the tertiary structure of apo-D-alanyl carrier protein. *Biochemistry* 40: 7964–7972.

Vollmer W, Blanot D & de Pedro MA (2008) Peptidoglycan structure and architecture. *FEMS Microbiol Rev*, doi: 10.1111/j.1574-6976.2007.00094.x

Vo-Quang Y, Carniato D, Vo-Quang L, Lacoste AM, Neuzil E & Le Goffic F (1986a) (β-Chloro-α-aminoethyl)phosphonic acids as inhibitors of alanine racemase and D-alanine : D-alanine ligase. *J Med Chem* 29: 148–151.

Vo-Quang Y, Carniato D, Vo-Quang L, Lacoste AM, Neuzil E & Le Goffic F (1986b) (1-Amino-2-propenyl) phosphonic acid, an inhibitor of alanine racemase and D-alanine : D-alanine ligase. *J Med Chem* 29: 579–581.

Walker B, Brown MF, Lynas JF, Martin SL, McDowell A, Badet B & Hill AJ (2000) Inhibition of *Escherichia coli* glucosamine synthetase by novel electrophilic analogues of glutamine –

comparison with 6-diazo-5-oxo-norleucine. *Bioorg Med Chem Lett* **10**: 2795–2798.

Walsh CT (1989) Enzymes of the D-alanine branch of bacterial cell wall peptidoglycan assembly. *J Biol Chem* **264**: 2393–2396.

Walsh CT, Benson TE, Kim DH & Lees WJ (1996) The versatility of phosphoenolpyruvate and its vinyl ether products in biosynthesis. *Chem Biol* **3**: 83–91.

Walsh AW, Falk PJ, Thanassi J, Discotto L, Pucci MJ & Ho H-T (1999) Comparison of the D-glutamate-adding enzymes from selected Gram-positive and Gram-negative bacteria. *J Bacteriol* **181**: 5395–5401.

Wang E & Walsh C (1978) Suicide substrates for the alanine racemase of Escherichia coli B. *Biochemistry* **17**: 1313–1321.

Wang EA & Walsh C (1981) Characteristics of β,β-difluoroalanine and β,β-trifluoroalanine as suicide substrates of Escherichia coli B alanine racemase. *Biochemistry* **20**: 7539–7546.

Wanke C & Amrhein N (1993) Evidence that the reaction of the UDP-N-acetylglucosamine 1-carboxyvinyltransferase proceeds through the O-phosphothioketal of pyruvic acid bound to Cys115 of the enzyme. *Eur J Biochem* **218**: 861–870.

Wanke C, Falchetto R & Amrhein N (1992) The UDP-N-acetylglucosamine 1-carboxyvinyl-transferase of Enterobacter cloacae. Molecular cloning, sequencing of the gene and overexpression of the enzyme. *FEBS Lett* **301**: 271–276.

Ward JB (1984) Biosynthesis of peptidoglycan: points of attack of wall inhibitors. *Pharmac Ther* **25**: 327–369.

Wasserman SA, Daub E, Grisafi P, Botstein D & Walsh CT (1984) Catabolic alanine racemase from Salmonella typhimurium: DNA sequence, enzyme purification, and characterization. *Biochemistry* **23**: 5182–5187.

Watanabe A, Yoshimura T, Mikami B & Esaki N (1999a) Tyrosine 265 of alanine racemase serves as a base abstracting α-hydrogen from L-alanine: the counterpart residue to lysine 39 specific to D-alanine. *J Biochem (Tokyo)* **126**: 781–786.

Watanabe A, Kurokawa Y, Yoshimura T, Kurihara T, Soda K & Esaki N (1999b) Role of lysine 39 of alanine racemase from Bacillus stearothermophilus that binds pyridoxal 5′-phosphate. Chemical rescue studies of Lys³⁹ → Ala mutant. *J Biol Chem* **274**: 4189–4194.

Watanabe A, Yoshimura T, Mikami B, Hayashi H, Kagamiyama H & Esaki N (2002) Reaction mechanism of alanine racemase from Bacillus stearothermophilus: X-ray crystallographic studies of the enzyme bound with N-(5′-phosphopyridoxyl)alanine. *J Biol Chem* **277**: 19166–19172.

Watzele G & Tanner W (1989) Cloning of the glutamine: fructose-6-phosphate amidotransferase gene from yeast. Pheromonal regulation of its transcription. *J Biol Chem* **264**: 8753–8758.

Wild J, Hennig J, Łobocka M, Walczak W & Kłopotowski T (1985) Identification of the dadX gene coding for the predominant isozyme of alanine racemase in Escherichia coli K12. *Mol Gen Genet* **198**: 315–322.

Winterburn PJ & Phelps CF (1971) The binding of substrates and modifiers to glucosamine synthetase. *Biochem J* **121**: 721–730.

Wipat A, Carter N, Brignell SC, Guy BJ, Piper K, Sanders J, Emmerson PT & Harwood CR (1996) The dnaB-pheA (256 degrees–240 degrees) region of the Bacillus subtilis chromosome containing genes responsible for stress responses, the utilization of plant cell walls and primary metabolism. *Microbiology* **142**: 3067–3078.

Wong KK, Kuo DW, Chabin RM, Fournier C, Gegnas LD, Waddell ST, Marsilio F, Leiting B & Pompliano DL (1998) Engineering a cell-free murein biosynthetic pathway: combinatorial enzymology in drug discovery. *J Am Chem Soc* **120**: 13527–13528.

Yamashita T, Ashiuchi M, Ohnishi K, Kato S, Nagata S & Misono H (2003) Molecular characterization of alanine racemase from Bifidobacterium bifidum. *J Mol Catal B Enzymatic* **23**: 213–222.

Yan Y, Munshi S, Leiting B, Anderson MS, Chrzas J & Chen Z (2000) Crystal structure of Escherichia coli UDPMurNAc-tripeptide D-alanyl-D-alanine-adding enzyme (MurF) at 2.3 Å resolution. *J Mol Biol* **304**: 435–445.

Yang Y, Severin A, Chopra R *et al.* (2006) 3,5-Dioxopyrazolidines, novel inhibitors of UDP-N-acetylenolpyruvylglucosamine reductase (MurB) with activity against Gram-positive bacteria. *Antimicrob Agents Chemother* **50**: 556–564.

Yokoigawa K, Hirasawa R, Ueno H, Okubo Y, Umesako S & Soda K (2001) Gene cloning and characterization of alanine racemases from Shigella dysenteriae, Shigella boydii, Shigella flexneri and Shigella sonnei. *Biochem Biophys Res Commun* **288**: 676–684.

Yoshimura T & Esaki N (2003) Amino acid racemases: functions and mechanisms. *J Biosci Bioeng* **96**: 103–109.

Zawadzke LE, Bugg TDH & Walsh CT (1991) Existence of two D-alanine : D-alanine ligases in Escherichia coli: cloning and sequencing of the ddlA gene and purification and characterization of the DdlA and DdlB enzymes. *Biochemistry* **30**: 1673–1682.

Zboińska E, Sztajer H, Lejczak B & Kafarski P (1990) Antibacterial activity of phosphono dipeptides based on 1-amino-1-methylethanephosphonic acid. *FEMS Microbiol Lett* **58**: 23–28.

Zeng B, Wong KK, Pompliano DL, Reddy S & Tanner ME (1998) A phosphinate inhibitor of the meso-diaminopimelic acid-adding enzyme (MurE) of peptidoglycan biosynthesis. *J Org Chem* **63**: 10081–10086.

Zentz F, Valla A, Le Guillou R, Labia R, Mathot AG & Sirot D (2002) Synthesis and antimicrobial activities of N-substituted imides. *Farmaco* **57**: 421–426.

Ziegler K, Diener A, Herpin C, Richter R, Deutzmann R & Lockau W (1998) Molecular characterization of cyanophycin synthetase, the enzyme catalyzing the biosynthesis of the cyanobacterial reserve material multi-L-arginyl-poly-L-aspartate (cyanophycin). *Eur J Biochem* **254**: 154–159.
