
RESEARCH LETTER

# Identification of genes coding for hydrolytic dehalogenation in the metagenome derived from a denitrifying 4-chlorobenzoate degrading consortium

Jong-Chan Chae¹, Bongkeun Song² & Gerben J. Zylstra¹

¹Biotechnology Center for Agriculture and the Environment, School of Environmental and Biological Sciences, Rutgers University, New Brunswick, NJ, USA; and ²Department of Biology and Marine Biology, University of North Carolina, Wilmington, NC, USA

---

**Correspondence:** Jong-Chan Chae,  
Biotechnology Center for Agriculture and the Environment, School of Environmental and Biological Sciences, Rutgers University, 59 Dudley Road, New Brunswick, NJ 08901-8520, USA. Tel.: +1 732 932 8165, ext.: 323; fax: +1 732 932 0312; e-mail: chae@aesop.rutgers.edu  

Received 12 November 2007; accepted 21 January 2008.  
First published online March 2008.  

DOI: 10.1111/j.1574-6968.2008.01106.x  

Editor: Clive Edwards  

Keywords  
denitrifying consortium; metagenome; hydrolytic dechlorination; 4-chlorobenzoate.

---

## Abstract

A metagenomic approach was taken to investigate the genetic basis for the ability of an anaerobic consortium to grow on either 4-chlorobenzoate or 4-bromo-benzoate under denitrifying conditions. Degenerate PCR primers were designed for the family of 4-chlorobenzoyl-CoA dehalogenase genes. The primers were utilized to screen a metagenome library and two overlapping clones were identified which yield a PCR product. The complete sequence of one metagenome clone was determined and genes encoding 4-chlorobenzoyl-CoA ligase (FcbA) and 4-chlorobenzoyl-CoA dehalogenase (FcbB) were identified. Analysis of the ORFs present in the nucleotide sequence suggests that the metagenome clone originated from an uncultured denitrifying microorganism belonging to the Betaproteobacteria. Interestingly, unlike similar gene clusters reported in aerobes, a gene encoding 4-hydroxybenzoyl-CoA thioesterase was not present in the gene cluster. This suggests that 4-hydroxybenzoyl-CoA is further degraded via the anaerobic reduction pathway in the corresponding microorganism instead of through thioester hydrolysis to yield 4-hydroxybenzoate.

---

## Introduction

Recent microbiology studies are coping with the difficulties in understanding the biosphere because <1% of microorganisms in nature are recovered by culturing methods (Amann *et al.*, 1995; Rondon *et al.*, 2000). Metagenomic studies, therefore, have been adapted to investigate the genetic information in environments containing unculturable microorganisms, exploring environmental microbial diversity, and recruiting target genes from an environmental sample directly (Seow *et al.*, 1997; Henne *et al.*, 2000; Rondon *et al.*, 2000; Entcheva *et al.*, 2001; Voget *et al.*, 2003).

The metagenomic approach is dependent on activity or sequence-based analyses to screen the large gene pool derived from environmental sources (Lorenz *et al.*, 2002; Voget *et al.*, 2003; Daniel, 2004). The former approach is used to identify functionally active clones from libraries. Although activity screening is more attractive to obtain evolutionary distant or novel genes, it is afflicted with the difficulties of heterologous gene expression requiring specific circumstance for enzyme activity such as oxygen-sensitive or multiple components enzymes (Henne *et al.*, 2000; Rondon *et al.*, 2000). In sequence-based screening approaches the targets are identified by PCR or hybridization methods using DNA sequences originating from conserved regions of already known genes or protein families (Seow *et al.*, 1997; Lorenz *et al.*, 2002). While this technique is extremely efficient in detecting target genes in a metagenomic library, the approach is limited to identifying genes matching the probe or primers utilized.

Halogenated aromatics are a common class of pollutants that occur in the environment due to their widespread use by industry (Häggblom, 1992). Halogenated aromatics are often recalcitrant to degradation due to the great stability of the aromatic ring structure and the halogen substituent. Previously Song *et al.* (2002) established a consortium capable of degrading 4-chlorobenzoate (4CBA) or 4-bromobenzoate (4BBA) under denitrifying conditions.
by selective enrichment method. Analysis of the microbial community suggested that at least six genera exist in the microbial consortium. However, none of the pure cultures derived from this consortium could degrade 4CBA under denitrifying conditions. Because repeated attempts to isolate pure cultures degrading 4CBA or 4BBA were not successful, a metagenomic study was employed to elucidate the dehalogenation pathway and genes in this consortium.

# Materials and methods

## Construction of a metagenomic library and sequencing

The DNA used for the preparation of a metagenomic library was directly isolated from the 4CBA degrading denitrifying consortium which was established with sediments from Arthur Kill estuary in New Jersey, USA, and enriched in the presence of 4CBA (Song *et al.*, 2002). The purified metagenomic DNA was physically sheared into c. 40-kb fragments. The sheared DNA was prepared for packaging with the pWEB::TNC™ vector following the supplier’s protocol (Epicentre). The resulting cosmid library was transfected into *Escherichia coli* EP100 (Epicentre). A QPix2 Colony Picker (Genetix Co., UK) was used to pick and transfer colonies to 96-well microplates containing Luria–Bertani (LB) broth supplemented with ampicillin (100 μg mL⁻¹).

The cosmid clone selected for sequencing was mechanically sheared into 2–3-kb fragments. The sheared DNA was end-repaired and ligated with the pCR®-Blunt vector (Invitrogen) and transformed into *E. coli* TOP10 (Invitrogen). The plasmids from 192 clones were isolated with the Concert™ 96 plasmid purification system (Invitrogen) and used as the template for sequencing. Nucleotide sequences of the small-fragment library were determined with an ABI PRISM® 3100 Genetic Analyzer (Applied Biosystems) using primers based on the pCR®-Blunt vector sequences (5′-CCAGTGTTGATG GATATCTGC-3′, BR; 5′-TCGGATCCACTAGTAACGGC-3′, BL). The gap sequences between contigs were obtained by a primer-walking strategy. The sequence data were assembled and analyzed with the LASERGENE software (DNASTAR, Inc.), and then compared with existing sequences in the GenBank database by performing a BLAST search (Altschul *et al.*, 1997).

## Degenerate primers for 4-chlorobenzoyl-CoA dehalogenase (FcbB) gene

To design primers for the *fcbB* gene, the corresponding nucleotide sequences were imported from the GenBank database: *Alcaligenes* sp. strain AL3007 (AF537222), *Comamonas* sp. strain DJ-12 (AF051771), *Pseudomonas* sp. strain CBS-3 (B42560), *Arthrobacter* sp. strain FHP1 (AB041030), *Arthrobacter* sp. strain SU (AF030397), *Arthrobacter* sp.

strain TM1 (AF043490), and *Arthrobacter globiformis* KZT1 (AF304300). The GenBank accession numbers are given in parentheses. After the alignment using the LASERGENE software (DNASTAR, Inc.), the most conserved region in the nucleotide sequences were selected to design degenerate primers that produce a 158-bp amplicon (5′-SCGMM GDGCRATGGAG-3′, forward; 5′-CYTGSAGRTGBGTSGG-3′, reverse). PCR products were obtained with a 10-μL reaction. The reactions were initiated at 94°C (2 min); followed by 40 cycles of denaturing at 94°C (1 min), annealing at 50°C (1 min), and extension at 72°C (20 s); and ended with incubation at 72°C for 10 min.

## Cloning and expression

To verify the catalytic correspondence of 4-chlorobenzoyl-CoA ligase (FcbA) and FcbB the structural genes *fcbBA* of the metagenome were amplified by PCR and cloned into the pCR®T7/CT-TOPO vector. The primers were 5′-ATGTAC GAGGCAATCGC-3′ (forward) and 5′-TCAGGTCTTCGG CAACG-3′, (reverse). To make a constitutively expressing clone, the XbaI–HindIII fragment harboring the *fcbBA* genes with the RBS of the vector was transferred into the pBluescript II SK (+) vector (Stratagene) and then transformed into the *E. coli* TOP10 strain. The expression of this clone is controlled by the *lacZ* promoter.

## HPLC analysis

The *E. coli* transformants were cultivated in LB medium until mid-log phase, washed twice with 50 mM potassium phosphate buffer (pH 7.0), and resuspended in the same buffer for a resting cell assay. The assay was performed with 0.3 mM 4-halobenzoates. Supernatant samples collected at various time points were filtered and analyzed by HPLC (Agilent 1100 series, Germany). Compounds were separated on a C-18 column (Spherisorb, 4.6 mm in diameter; 250 mm in length; Phenomenex) at a flow rate of 1 mL min⁻¹ with mobile phase of methanol–water–acetic acid (55:45:2), and monitored by absorbance at wavelength 254 nm.

# Results and discussion

## Design of degenerate primers and detection of dehalogenase gene

PCR was employed using two kinds of degenerate primers corresponding to the genes encoding reductive dehalogenases (Rhee *et al.*, 2003) and hydrolytic dehalogenases (FcbB). 4-chlorobenzoyl-CoA (4CBA-CoA) dehalogenase was chosen as one of candidates responsible for dehalogenation in the consortium because hydrolytic dehalogenation is known as a hydrolysis step in which the chloride substituent is replaced by a hydroxyl group derived from water as shown
Metagenomic dehalogenase gene

(a)


Cl
COA
FcbA
4CBA



Cl
CO-SCoA
H₂O HCl
FcbB
4CBA-CoA



OH
CO-SCoA
FcbC
4HBA-CoA



OH
COOH
4HBA


(b)


S 1
(Kb)
1.6
1.0
0.5
0.1


Fig. 1. Hydrolytic dehalogenation pathway of 4-chlorobenzoate reported from aerobic microorganisms (a) and PCR amplification of 4-chlorobenzoyl-CoA dehalogenase gene (*fcbB*) with a metagenome isolated from the 4CBA degrading denitrifying consortium (b). 4CBA, 4-chlorobenzoate; 4CBA-CoA, 4-chlorobenzoyl-CoA; 4HBA-CoA, 4-hydroxybenzoyl-CoA; 4HBA, 4-hydroxybenzoate; FcbA, 4-chlorobenzoyl-CoA ligase; FcbB, 4-chlorobenzoyl-CoA dehalogenase; FcbC, 4-hydroxybenzoyl-CoA thioesterase; lane S, 1 kb plus DNA ladder; lane 1, PCR product amplified from the metagenome.

in Fig. 1a (Chang *et al.*, 1992). The genes and enzymes for hydrolytic dechlorination of 4CBA-CoA have been characterized only from aerobic organisms (Chang *et al.*, 1992; Schmitz *et al.*, 1992; Chae *et al.*, 2000; Zhou *et al.*, 2004). Although genes for hydrolytic dehalogenation of 4CBA have not been reported from anaerobic microorganisms, it has been presented by Coschigano *et al.* (1994) that the genes from aerobic organisms can be expressed in denitrifying strains and allow them to grow on 4CBA under anaerobic denitrifying conditions. This observation indicates that dehalogenation can occur by both reductive and hydrolytic mechanisms under anaerobic conditions.

The degenerate primer set for *fcbB* gene described in 'Materials and Methods' was successfully tested for the ability to amplify the *fcbB* genes from the Gram-negative strain *Comamonas* sp. strain DJ-12 (Chae *et al.*, 2000) and the Gram-positive strain *Arthrobacter* sp. strain J2 (unpublished) which were capable of degrading 4CBA and harbor *fcbB* genes. PCR with isolated metagenomic DNA using the *fcbB* primer set yielded a 158-bp product with a nucleotide sequence related to known *fcbB* genes (Fig. 1b), while primers (Rhee *et al.*, 2003) targeting genes encoding reductive dehalogenases did not yield any PCR product (data not shown). Although this result suggests that the consortium degrades 4CBA or 4BBA via hydrolytic dehalogenation, we could not exclude by this means that reductive dehalogenation is also actively occurring in the consortium.

### Screening of a metagenomic library

The 1440 clone metagenome library was condensed by pooling the rows and columns of the microtiter plates. Subsequently, each pool represented 180 (rows) or 120 (columns) cosmid clones. The recombinant plasmids purified from 20 condensed pools were subjected to PCR with the *fcbB* degenerate primers. A detected PCR product identified the row and column of the clone in the pooled 15 plates. A second PCR reaction was then performed to identify the exact plate by running the 15 clones separately. Following this strategy, two cosmid clones, pMLC1 and pMLC2, were identified. A DNA restriction analysis ensured that two clones were not exact duplicates but were in fact overlapping clones. The pMLC1 clone was subjected to a detailed molecular and functional analysis.

#### Sequence analysis of pMLC1

The complete nucleotide sequence of pMLC1 was determined including the *fcb* gene cluster for hydrolytic dechlorination of 4CBA. The insert size is 33,768 bp with a G+C content of 70.23%. Sequence analysis identified 25 ORFs preceded by putative ribosome-binding sequences, summarized in Table 1 and Fig. 2a. The putative genes identified were mainly genes encoding proteins involved in cellular processes. Most (20 proteins) of the 25 identified putative proteins were highly similar to proteins derived from *Betaproteobacteria*. Five proteins showed homology with proteins from *Alphaproteobacteria*, *Gammaproteobacteria*, or other microbial species. This observation suggests that the DNA fragment cloned in pMLC1 possibly originated from unculturable microorganisms belonging to the *Betaproteobacteria*.

The hydrolytic dechlorination of 4CBA to 4-hydroxybenzoate (4HBA) is carried out by a sequential reaction of FcbA, FcbB, and 4-hydroxybenzoyl-CoA (4HBA-CoA) thioesterase (FcbC) in aerobic microorganisms (Chang *et al.*, 1992). While an ORF encoding a thioesterase was not identified on pMLC1, the two ORFs responsible for hydrolytic dehalogenation of 4CBA are found adjacent to each other and in the order *fcbB–fcbA*. In addition, an ORF encoding a putative periplasmic solute binding protein exists just downstream of *fcbA* as an operonic structure (Fig. 2b). It is interesting that the *fcbC* gene is not found in the clone even though it clusters with the *fcbA* and *fcbB* genes in all known aerobic 4CBA-degrading microorganisms. To date, the *fcbAB* genes have been reported to arrange along with the *fcbC* gene either as B-A-C (in Gram-negative) or A-B-C (in Gram-positive) although the transporter genes are located between *fcbA* and *fcbC* in *Comamonas* sp. DJ-12 and *Alcaligenes* sp. AL3007 (Schmitz *et al.*, 1992; Chae *et al.*, 2000; Zhou *et al.*, 2004; Chae &

Table 1. Identified genes in pMLC1 and similar gene products

| ORF    | Length (bp) | Putative function                                                                 | Microorganism                          | Identity (%) | GenBank accession number | E-value   |
|--------|-------------|----------------------------------------------------------------------------------|-----------------------------------------|--------------|--------------------------|------------|
| ORF 1  | 1023        | Biotin synthase                                                                  | Polaromonas sp. JS666                  | 76           | ABE43519                 | 7E-143     |
| ORF 2  | 1110        | Unknown                                                                          |                                         |               |                          |            |
| ORF 3  | 723         | Hypothetical protein                                                            | Ralstonia eutropha JMP134              | 59           | AAZ62847                 | 6E-81      |
| ORF 4  | 402         | Endoribonuclease                                                                | Nitrosospira multiformis              | 65           | ABB75555                 | 1E-42      |
| ORF 5  | 1209        | ABC-transporter, substrate-binding protein                                      | Azoarcus evansii                      | 63           | AAN39370                 | 3E-137     |
| ORF 6  | 1533        | 4-Chlorobenzoyl-CoA ligase                                                      | Alcaligenes sp. AL3007                | 53           | AAN10109                 | 5E-142     |
| ORF 7  | 801         | 4-Chlorobenzoyl-CoA dehalogenase                                                | Pseudomonas sp. DJ-12                 | 77           | AAC05576                 | 4E-107     |
| ORF 8  | 675         | Transcriptional regulator (TetR family)                                         | Sphingomonas sp. SKA58                | 29           | EAT07899                 | 6E-18      |
| ORF 9  | 1035        | Aceto-in dehydrogenase alpha-subunit                                            | Oceanicola batsensis                  | 53           | EAQ02393                 | 6E-89      |
| ORF 10 | 1002        | Pyruvate dehydrogenase beta-subunit                                             | Oceanicola batsensis                  | 61           | EAQ02392                 | 6E-107     |
| ORF 11 | 486         | Asparaginase                                                                    | Anaeromyxobacter dehalogenans         | 53           | ABC81912                 | 5E-39      |
| ORF 12 | 1002        | Phospholipase                                                                   | Burkholderia thailandensis            | 58           | ABC39098                 | 2E-85      |
| ORF 13 | 633         | istB-like ATP-binding protein                                                  | Novosphingobium aromaticivorans       | 57           | ABD25901                 | 1E-20      |
| ORF 14 | 831         | Molybdopterin dehydrogenase                                                    | Ralstonia eutropha JMP134              | 59           | AAZ59805                 | 1E-76      |
| ORF 15 | 2481        | Carbon monoxide dehydrogenase                                                  | Polaromonas sp. JS666                 | 63           | ABE45303                 | 0.0        |
| ORF 16 | 462         | Ferredoxin                                                                      | Roseiflexus sp. RS-1                  | 66           | EAT27643                 | 2E-54      |
| ORF 17 | 567         | Hypothetical protein                                                            | Synechococcus sp. WH5701              | 58           | EAQ73927                 | 9E-60      |
| ORF 18 | 2820        | ABC-transporter                                                                 | Azoarcus evansii                      | 50           | CAI10053                 | 0.0        |
| ORF 19 | 540         | Hypothetical protein                                                            | Burkholderia pseudomallei             | 59           | ZP_01210400              | 6E-42      |
| ORF 20 | 1596        | Glycosyl transferase                                                            | Burkholderia vietnamiensis G4          | 59           | EAM32768                 | 1E-131     |
| ORF 21 | 699         | Oxidoreductase/Short chain dehydrogenase                                        | Burkholderia pseudomallei             | 64           | ABA53407                 | 6E-77      |
| ORF 22 | 618         | Histidinol phosphatase                                                          | Burkholderia pseudomallei             | 57           | ZP_00887783              | 4E-54      |
| ORF 23 | 1314        | Glycosyl transferase                                                            | Burkholderia vietnamiensis G4          | 50           | EAM32757                 | 4E-95      |
| ORF 24 | 1176        | ADP-heptose:LPS heptocyltransferase                                             | Burkholderia mallei                   | 51           | ZP_01024695              | 3E-70      |
| ORF 25 | 936         | Glycosyl transferase                                                            | Ralstonia eutropha JMP134              | 65           | AAZ63734                 | 4E-114     |

(a)

0(bp)     4000     8000     12000    16000
1 → 2     3 → 4 → 5 ← 6 ← 7     8 → 9 → 10 → 11 → 12
16000    20000    24000    28000    32000
13 ← 14 ← 15 ← 16 → 17 → 18 → 19 → 20 → 21 → 22 → 23 → 24 → 25


(b)

Consortium
B → A → T
266     510     402

Pseudomonas sp. CBS3
B → A → C
269     528     141

Comamonas sp. DJ-12
B → A → T1 → T2 → T3 → C
269     505     326     181     439     142

Alcaligenes sp. AL3007
B → A → T1 → T2 → C
276     501     371     258     142

Arthrobacter sp. FHP1
A1 → B1 → C1 → A2 → B2 → C2 → T
520     276     151     2.7kb   520     276     123     444

Arthrobacter sp. SU
A1 → B1 → C1 → A2 → B2 → C2 → T
522     276     151     276     123     444

Arthrobacter sp. TM1
A1 → B1 → C1 → A2 → B2 → C2 → T
522     276     151     3.3kb   522     276     123     444

Arthrobacter globiformis
A → B → C
518     276     151


Fig. 2. Genetic map of pMLC1 (a) and arrangement of the fcb gene clusters in 4CBA-degrading bacteria (b). A, B, and C indicate the genes encoding 4CBA-CoA ligase, 4CBA-CoA dehalogenase, and 4HBA-CoA thioesterase, respectively. T represents the gene encoding a transporter. ORFs determined in this study are described as the numbers on (a). The numbers on (b) indicate the number of amino acid residues.

Metagenomic dehalogenase gene

(a)

![Diagram](https://i.imgur.com/1234567.png)

(b)

![Diagram](https://i.imgur.com/8901234.png)

Fig. 3. Phylogenetic relationship of FcbA (a) and FcbB (b) from pMLC1 with homologues. Sequences were obtained from GenBank under the following accession numbers.: *Alcaligenes* sp. strain AL 3007, AF537222 (A-AL3007 and B-AL3007); *Comamonas* sp. strain DJ-12, AF051771 (A-DJ12 and B-DJ12); *Pseudomonas* sp. strain CBS-3, A42560 (A-CBS3) and B42560 (B-CBS3); *Arthrobacter* sp. strain FHP1, AB041030 (A1-FHP1, A2-FHP1, B1-FHP1, and B2-FHP1); *Arthrobacter* sp. strain SU, M93187 (A1-SU and B1-SU) and AF030397 (B2-SU); *Arthrobacter* sp. strain TM1, AF030397 (A1-TM1, A2-TM1, B1-TM1, and B2-TM1); *Arthrobacter globiformis* KZT1, AF304300 (A-glob and B-glob). The sequences encoding long-chain fatty acid CoA ligase from *Burkholderia xenovorans* LB400 (GenBank accession no. YP_558944) and enoyl-CoA hydratase from *Pseudomonas* sp. strain Y2 (GenBank accession no. CAE45103) were used in phylogenetic analysis of FcbA and FcbB as outgroups, respectively.

This implies that the *fcbC* gene existing with *fcbA* and *B* as an operon in aerobic 4CBA-degrading bacteria might have been recruited to accelerate the sequential reaction from 4HBA-CoA to 4HBA under aerobic conditions during metabolic evolution. The lack of an *fcbC* gene is in accordance with what is known about anaerobic degradation of aromatic compounds. Anaerobically, 4HBA is metabolized through 4HBA-CoA (Brackmann & Fuchs, 1993; Gibson *et al.*, 1994). Therefore, it makes no sense to possess an *fcbC* gene encoding a 4HBA-CoA thioesterase as part of a 4CBA anaerobic degradation pathway.

The deduced amino acid sequence of the *fcbA* gene shows the highest identity (53%) with one from *Alcaligenes* sp. strain AL 3007 (Gulick *et al.*, 2004) and falls into a cluster as a subfamily along with those of the Gram-negative strains *Comamonas* sp. strain DJ-12, *Pseudomonas* sp. strain CBS-3,

and *Alcaligenes* sp. strain AL 3007, as shown in Fig. 3a. G163, G166, P168, and K169 necessary to catalyze the adenylation of 4CBA (Chang *et al.*, 1997) are analogous in a putative ATP/AMP binding motif. The alignment with homologues as shown in Fig. 3 shows that the amino acid residues consisting of an aryl binding pocket are conserved in F184, H207, V209, F249, A280, I303, G305, M310, and N311 out of 10 residues suggested by the crystal structure of FcbA from *Alcaligenes* sp. strain AL 3007 (Gulick *et al.*, 2004).

The deduced amino acid sequence of *fcbB* gene exhibits 74–77% identity to those of the corresponding enzymes from the Gram-negative strains *Comamonas* sp. strain DJ-12, *Pseudomonas* sp. strain CBS-3, and *Alcaligenes* sp. strain AL 3007, clustering together under the enoyl-CoA hydratase superfamily as shown in Fig. 3b. The FcbB sequence contains the enoyl-CoA hydratase/isomerase signature, 104-LAAVNGVAAGGGGLGLTLASDM-124, which is typical in the enoyl-CoA hydratase superfamily. In hydrolytic dehalogenation of 4CBA-CoA to 4HBA-CoA, E232, D145, and H90 play roles in providing an electrostatic force, releasing the chloride ion with hydrolysis to form 4HBA-CoA (Wu *et al.*, 2006). These residues are also conserved in consortium FcbB.

As indicated by the gene organization in Table 1 and Fig. 2a ORF5 is located just downstream of *fcbA* and is possibly transcribed along with *fcbAB*. The potential 43-kDa protein encoded by this ORF is 78–76% similar with solute binding proteins which are ABC transporter subunits in *Azoarcus avansii* (GenBank accession nos. AAL02070, AAL02078, AAN39370, and CAI09138). These related genes are clustered with operons encoding proteins responsible for benzoate (Gescher *et al.*, 2002) or 2-aminobenzoate metabolism (Schühe *et al.*, 2001).

Fig. 4. Resting cell incubation of *Escherichia coli* harboring the *fcbBA* genes of pMLC1. Closed triangle, 4-chlorobenzoate; closed rectangle, 4-bromobenzoate; closed diamond, 4-iodobenzoate; closed circle, 4-fluorobenzoate. *E. coli* carrying the vector without any insert as a negative control did not exhibit any dehalogenation activity on the substrates.

Expression of FcbBA and transformation of 4CBA

The *fcbBA* genes were subcloned into the pBluescript II SK (+) vector for constitutive expression. 4-Chloro, 4-bromo, and 4-iodobenzoate in a resting cell assay with this recombinant *E. coli* strain were totally depleted with a proportional accumulation of 4HBA, whereas 4-fluorobenzoate was not dehalogenated (Fig. 4). The 4HBA-CoA produced from 4CBA by the catalysis of FcbA and FcbB was converted into the final product 4HBA because the thioester hydrolysis of 4HBA-CoA into 4HBA could occur by the action of thioesterases in *E. coli* as reported by Schmitz *et al.* (1992). This result along with our bioinformatic analysis confirms that the gene products are responsible for the hydrolytic dechlorination of 4CBA.

### Conclusion

Dehalogenation of 4CBA was investigated in a denitrifying consortium able to degrade 4CBA or 4BBA using a metagenomic approach. The genes encoding the hydrolytic dehalogenation of 4CBA were identified in a metagenomic library. Sequence analysis of the ORFs and the phylogenetic relationship of FcbA and FcbB suggest that the origin of the cloned DNA fragment is genomic DNA from an uncultured denitrifying microorganism belonging to the Gram-negative Betaproteobacteria. The nucleotide sequences determined in this study have been deposited under GenBank accession no. DQ826744.

### Acknowledgements

This work was supported by NSF CHE-0221978.

### References

Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W & Lipman DJ (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. *Nucleic Acids Res* **25**: 3389–3402.

Amann RI, Ludwig W & Schleifer KH (1995) Phylogenetic identification and *in situ* detection of individual microbial cells without cultivation. *Microbiol Rev* **59**: 143–169.

Brackmann R & Fuchs G (1993) Enzymes of anaerobic metabolism of phenolic compounds. 4-Hydroxybenzoyl-CoA reductase (dehydroxylating) from a denitrifying *Pseudomonas* species. *Eur J Biochem* **213**: 563–571.

Chae J-C & Zylstra GJ (2006) 4-Chlorobenzoate uptake in *Comamonas* sp. strain DJ-12 is mediated by a tripartite ATP-independent periplasmic transporter. *J Bacteriol* **188**: 8407–8412.

Chae J-C, Kim Y, Kim YC, Zylstra GJ & Kim C-K (2000) Genetic structure and functional implication of the *fcb* gene cluster for hydrolytic dechlorination of 4-chlorobenzoate from *Pseudomonas* sp. DJ-12. *Gene* **258**: 109–116.

Chang KH, Liang PH, Beck W, Scholten JD & Dunaway-Mariano D (1992) Isolation and characterization of the three polypeptide components of 4-chlorobenzoate dehalogenase from *Pseudomonas* sp. strain CBS-3. *Biochemistry* **31**: 5605–5610.

Chang KH, Xiang H & Dunaway-Mariano D (1997) Acyl-adenylate motif of the acyl-adenylate/thioester-forming enzyme superfamily: a site-directed mutagenesis study with the *Pseudomonas* sp. strain CBS3 4-chlorobenzoate:coenzyme A ligase. *Biochemistry* **36**: 15650–15659.

Coschigano PW, Häggblom MM & Young LY (1994) Metabolism of both 4-chlorobenzoate and toluene under denitrifying conditions by a constructed bacterial strain. *Appl Environ Microbiol* **60**: 989–995.

Daniel R (2004) The soil metagenome – a rich resource for the discovery of novel natural products. *Curr Opin Biotechnol* **15**: 199–204.

Entcheva P, Liebl W, Johann A, Hartsch T & Streit WR (2001) Direct cloning from enrichment cultures, a reliable strategy for isolation of complete operons and genes from microbial consortia. *Appl Environ Microbiol* **67**: 89–99.

Gescher J, Zaar A, Mohamed M, Schägger H & Fuchs G (2002) Genes coding for a new pathway of aerobic benzoate metabolism in *Azoarcus evansii*. *J Bacteriol* **184**: 6301–6315.

Gibson J, Dispensa M, Fogg GC, Evans DT & Harwood CS (1994) 4-Hydroxybenzoate-coenzyme A ligase from *Rhodopseudomonas palustris*: purification, gene sequence, and role in anaerobic degradation. *J Bacteriol* **176**: 634–641.

Gulick AM, Lu X & Dunaway-Mariano D (2004) Crystal structure of 4-chlorobenzoate: CoA ligase/synthetase in the unliganded and aryl substrate-bound states. *Biochemistry* **43**: 8670–8679.

Häggblom MM (1992) Microbial breakdown of halogenated aromatic pesticides and related compounds. *FEMS Microbiol Rev* **9**: 29–71.

Henne A, Schmitz RA, Bömeke M, Gottschalk G & Daniel R (2000) Screening of environmental DNA libraries for the presence of genes conferring lipolytic activity on *Escherichia coli*. *Appl Environ Microbiol* **66**: 3113–3116.

Lorenz P, Liebeton K, Niehaus F & Eck J (2002) Screening for novel enzymes for biocatalytic processes: accessing the metagenome as a resource of novel functional sequence space. *Curr Opin Biotechnol* **13**: 572–577.

Rhee S-K, Fennell DE, Häggblom MM & Kerkhof LJ (2003) Detection by PCR of reductive dehalogenase motifs in a sulfidogenic 2-bromophenol-degrading consortium enriched from estuarine sediment. *FEMS Microbiol Ecol* **43**: 317–324.

Rondon MR, August PR, Bettermann AD *et al.* (2000) Cloning the soil metagenome: a strategy for accessing the genetic and functional diversity of uncultured microorganisms. *Appl Environ Microbiol* **66**: 2541–2547.

Metagenomic dehalogenase gene

Schmitz A, Gartemann K-H, Fiedler J, Grund E & Eichenlaub R (1992) Cloning and sequence analysis of genes for dehalogenation of 4-chlorobenzoate from *Arthrobacter* sp. strain SU. *Appl Environ Microbiol* **58**: 4068–4071.

Schühele K, Jahn M, Ghisla S & Fuchs G (2001) Two similar gene clusters coding for enzymes of a new type of aerobic 2-aminobenzoate (anthranilate) metabolism in the bacterium *Azoarcus evansii*. *J Bacteriol* **183**: 5268–5278.

Seow K-T, Meurer G, Gerlitz M, Wendt-Pienkowski E, Hutchinson CR & Davies J (1997) A study of iterative type II polyketide synthases, using bacterial genes cloned from soil DNA: a means to access and use genes from uncultured microorganisms. *J Bacteriol* **179**: 7360–7368.

Song B, Kerkhof LJ & Häggblom MM (2002) Characterization of bacterial consortia capable of degrading 4-chlorobenzoate and

4-bromobenzoate under denitrifying conditions. *FEMS Microbiol Lett* **213**: 183–188.

Voget S, Leggewie C, Uesbeck A, Raasch C, Jaeger K-E & Streit WR (2003) Prospecting for novel biocatalysts in a soil metagenome. *Appl Environ Microbiol* **69**: 6235–6242.

Wu J, Xu D, Lu X, Wang C, Guo H & Dunaway-Mariano D (2006) Contributions of long-range electrostatic interactions to 4-chlorobenzoyl-CoA dehalogenase catalysis: a combined theoretical and experimental study. *Biochemistry* **45**: 102–112.

Zhou L, Marks TS, Poh RPC, Smith RJ, Chowdhry BZ & Smith ARW (2004) The purification and characterization of 4-chlorobenzoate:CoA ligase and 4-chlorobenzoyl CoA dehalogenase from *Arthrobacter* sp. strain TM-1. *Biodegradation* **15**: 97–109.
