
Metabolic Diversification—Independent Assembly of Operon-Like Gene Clusters in Different Plants
Ben Field and Anne E. Osbourn
Science 320, 543 (2008);
DOI:10.1126/science.1154990

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by clicking here.

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines here.

The following resources related to this article are available online at www.sciencemag.org (this information is current as of August 12, 2014):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
http://www.sciencemag.org/content/320/5875/543.full.html

Supporting Online Material can be found at:
http://www.sciencemag.org/content/suppl/2008/03/20/1154990.DC1.html

This article cites 25 articles, 13 of which can be accessed free:
http://www.sciencemag.org/content/320/5875/543.full.html#ref-list-1

This article has been cited by 26 article(s) on the ISI Web of Science

This article has been cited by 39 articles hosted by HighWire Press; see:
http://www.sciencemag.org/content/320/5875/543.full.html#related-urls

This article appears in the following subject collections:

Botany
http://www.sciencemag.org/cgi/collection/botany

Genetics
http://www.sciencemag.org/cgi/collection/genetics

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2008 by the American Association for the Advancement of Science; all rights reserved. The title Science is a registered trademark of AAAS.

3. J. M. McClellan, E. Susser, M.-C. King, *Br. J. Psychiatry* 190, 194 (2007).
4. J. A. Lee, J. R. Lupski, *Neuron* 52, 103 (2006).
5. J. Sebat *et al.*, *Science* 316, 445 (2007).
6. D. J. MacIntyre, D. H. R. Blackwood, D. J. Porteous, B. S. Pickard, W. J. Muir, *Mol. Psychiatry* 8, 275 (2003).
7. J. E. Chubb, H. J. Bradshaw, D. C. Soares, D. J. Porteous, J. K. Millar, *Mol. Psychiatry* 13, 36 (2008).
8. D. Kamnasaran, W. J. Muir, M. A. Ferguson-Smith, D. W. Cox, *J. Med. Genet.* 40, 325 (2003).
9. H. Liu *et al.*, *Proc. Natl. Acad. Sci. U.S.A.* 99, 16859 (2002).
10. G. Kirov *et al.*, *Hum. Mol. Genet.* 17, 458 (2008).
11. J. I. Friedman *et al.*, *Mol. Psychiatry* 13, 261 (2008).
12. Materials and methods are available as supporting material on Science Online.
13. A. J. lafrate *et al.*, *Nat. Genet.* 36, 949 (2004).
14. J. L. Rapoport, G. Inoff-Germain, *Curr. Psychiatry Rep.* 2, 410 (2000).
15. A. Sporn *et al.*, *Mol. Psychiatry* 9, 225 (2004).
16. J. R. Idol, A. M. Addington, R. T. Long, J. L. Rapoport, E. D. Green, *J. Autism Dev. Disord.* 38, 668 (2008).
17. J. L. Seal *et al.*, *J. Med. Genet.* 43, 887 (2006).
18. A. J. Sharp *et al.*, *Am. J. Hum. Genet.* 77, 78 (2005).
19. J. M. Friedman *et al.*, *Am. J. Hum. Genet.* 79, 500 (2006).
20. The Autism Genome Project Consortium, *Nat. Genet.* 39, 319 (2007).
21. R. A. Kumar *et al.*, *Hum. Mol. Genet.* 17, 628 (2007).
22. L. A. Weiss *et al.*, *N. Engl. J. Med.* 358, 667 (2008).
23. J. R. Lupski, P. Stankiewicz, *PLoS Genet.* 1, e49 (2005).
24. H. Mi, N. Guo, A. Kejariwal, P. D. Thomas, *Nucleic Acids Res.* 35, D247 (2007).

25. Ingenuity Systems, www.ingenuity.com.
26. National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD, www.ncbi.nlm.nih.gov.
27. C. Rio, H. I. Rieff, P. Qi, T. S. Khurana, G. Corfas, *Neuron* 19, 39 (1997).
28. R. S. Woo *et al.*, *Neuron* 54, 599 (2007).
29. B. Li, R. S. Woo, L. Mei, R. Malinow, *Neuron* 54, 583 (2007).
30. P. M. Beart, R. D. O’Shea, *Br. J. Pharmacol.* 150, 5 (2007).
31. R. E. Smith, V. Haroutunian, K. L. Davis, J. H. Meador-Woodruff, *Am. J. Psychiatry* 158, 1393 (2001).
32. K. M. Giehl, *Prog. Exp. Tumor Res.* 39, 1 (2007).
33. J. D. Buxbaum *et al.*, *Mol. Psychiatry* 13, 162 (2008).
34. R. A. Garcia, K. Vasudevan, A. Buonanno, *Proc. Natl. Acad. Sci. U.S.A.* 97, 3596 (2000).
35. R. M. Esper, M. S. Pankonin, J. A. Loeb, *Brain Res. Rev.* 51, 161 (2006).
36. A. Makoff, C. Pilling, K. Harrington, P. Emson, *Brain Res. Mol. Brain Res.* 40, 165 (1996).
37. M. Nedergaard, T. Takano, A. J. Hansen, *Nat. Rev. Neurosci.* 3, 748 (2002).
38. J. Q. Wang, E. E. Fibuch, L. Mao, *J. Neurochem.* 100, 1 (2007).
39. S. Sammut, D. J. Park, A. R. West, *J. Neurochem.* 103, 1145 (2007).
40. J. A. Bernard-Trifilo *et al.*, *J. Neurochem.* 93, 834 (2005).
41. J. B. Manent, A. Represa, *Neuroscientist* 13, 268 (2007).
42. J. I. Heng, G. Moonen, L. Nguyen, *Eur. J. Neurosci.* 26, 537 (2007).
43. C. A. Ghiani *et al.*, *Neurochem. Res.* 32, 363 (2007).
44. N. Norton *et al.*, *Am. J. Med. Genet. B Neuropsychiatr. Genet.* 141, 96 (2006).

45. A. J. Law, J. E. Kleinman, D. R. Weinberger, C. S. Weickert, *Hum. Mol. Genet.* 16, 129 (2007).
46. I. Benzel *et al.*, *Behav. Brain Funct.* 3, 31 (2007).
47. Single-letter abbreviations for the amino acid residues are as follows: A, Ala; C, Cys; D, Asp; E, Glu; F, Phe; G, Gly; H, His; I, Ile; K, Lys; L, Leu; M, Met; N, Asn; P, Pro; Q, Gln; R, Arg; S, Ser; T, Thr; V, Val; W, Trp; and Y, Tyr.
48. We thank M. Wigler, E. Susser, and J. Watson for helpful discussion. This work was supported by The Forrest C. and Frances H. Lattner Foundation; a NARSAD Young Investigator Award; The NARSAD Constance and Steven Lieber Distinguished Investigator Award; The Simons Foundation; a gift of Ted and Vada Stanley; The Stanley Medical Research Foundation; the Howard Hughes Medical Institute; NIH grants HD043569, MH061464, MH061528, MH061355, NS052108, RR025014, and RR000046; NIH intramural research programs of the National Institute on Aging and the National Institute of Mental Health; and the Mental Health Division, Washington State Department of Social and Health Services.

**Supporting Online Material**

www.sciencemag.org/cgi/content/full/1155174/DC1

Materials and Methods  
Figs. S1 to S3  
Tables S1 to S4  
References  

2 January 2008; accepted 14 March 2008  
Published online 27 March 2008;  
10.1126/science.1155174  

Include this information when citing this paper.


# Metabolic Diversification—Independent Assembly of Operon-Like Gene Clusters in Different Plants

Ben Field and Anne E. Osbourn*

Operons are clusters of unrelated genes with related functions that are a feature of prokaryotic genomes. Here, we report on an operon-like gene cluster in the plant *Arabidopsis thaliana* that is required for triterpene synthesis (the thalianol pathway). The clustered genes are coexpressed, as in bacterial operons. However, despite the resemblance to a bacterial operon, this gene cluster has been assembled from plant genes by gene duplication, neofunctionalization, and genome reorganization, rather than by horizontal gene transfer from bacteria. Furthermore, recent assembly of operon-like gene clusters for triterpene synthesis has occurred independently in divergent plant lineages (*Arabidopsis* and oat). Thus, selection pressure may act during the formation of certain plant metabolic pathways to drive gene clustering.

Triterpenes protect plants against pests and diseases and are also important drugs and anticancer agents (1–4). Like sterols, these compounds are synthesized from the isoprenoid pathway by cyclization of 2,3-oxidosqualene (1, 3). The *Arabidopsis* genome contains 13 predicted oxidosqualene cyclase (OSC) genes (3, 5). Of these, one encodes cycloartenol synthase (CAS), which is required for sterol biosynthesis, and another encodes lanosterol synthase (LAS), which is conserved across the eudicots and whose function in plants is unknown (Fig. 1A). The 11 remaining OSCs fall into two major clades (I and II) (Fig. 1A). These OSCs produce various triterpenes

Department of Metabolic Biology, John Innes Centre, Colney Lane, Norwich NR4 7UH, UK.

*To whom correspondence should be addressed. E-mail: anne.osbourn@bbsrc.ac.uk


a gene cluster that includes genes required for acylation, glucosylation, and other steps in the pathway (2, 7). The avenacin biosynthesis genes are tightly regulated and expressed only in the root epidermis, the site of accumulation of the pathway end product (6, 8). The avenacin gene cluster lies within a region of the oat genome lacking synteny with rice and other cereals (7).

We examined the genomic regions around each of the 13 *Arabidopsis* OSC genes in the *Arabidopsis* genome to establish whether genes for triterpene synthesis might be clustered (9). Four OSC genes are flanked by genes predicted to encode other classes of enzymes implicated in secondary metabolism. These four OSCs all belong to clade II, which appears to have undergone accelerated evolution compared with other *Arabidopsis* OSCs (Fig. 1A). We focused on a region containing four contiguous genes predicted to encode an OSC (*At5g48010*), two CYP450s (*At5g48000* and *At5g47990*), and a BAHD family acyltransferase (ACT) (*At5g47980*) (Fig. 1B). The expression of all four genes is highly correlated (Fig. 1C) and occurs primarily in the root epidermis (fig. S1), which suggests that the genes are functionally related (10).

The OSC gene within this region, *At5g48010*, converts 2,3-oxidosqualene to the triterpene thalianol when expressed in yeast (11). However, thalianol has not been reported in plants. We detected low levels of thalianol in roots but not leaves of wild-type *Arabidopsis* (Fig. 2, C and D), consistent with the expression of *At5g48010*. Thalianol was not detectable in root extracts of a null insertion mutant of *At5g48010* (*thas1-1*) (Fig. 2E), which indicated that the *At5g48010* gene product [hereafter, named thalianol synthase (THAS)] is required for synthesis of thalianol in *Arabidopsis*

roots. Overexpression of THAS in *Arabidopsis*  
resulted in thalianol accumulation in leaves (Fig.  
2F) and in dwarfing (Fig. 3A). Mutations in the  
gibberellin, brassinosteroid, and primary steroid  
pathways can result in dwarfing (12–14). However,  
the dwarf phenotype of the THAS-overexpressing  
lines was not rescued by application of gibberellin  

or brassinosteroid, and the sterol content of these  
plants was not significantly altered (*P* > 0.05,  
Student’s *t* test, *n* = 3), which suggests that thalianol  
may be detrimental to plant growth.  

Gas chromatography–mass spectrometry  
(GC-MS) analysis of wild-type root extracts re-  
vealed additional peaks that were absent in THAS  
mutants [(Fig. 4A); wild type and *thas1-1*]. Because these peaks were dependent on THAS, this  
suggested that thalianol (peak 1) is converted to  
unknown downstream products in wild-type  
*Arabidopsis* plants (peaks 2a, 2b, 2c, 3a, and 3b).  
Therefore, the coexpressed genes adjacent to  
At5g48010 (THAS) were examined to determine  

**Fig. 1.** (A) Neighbor-  
joining tree of *Arabidopsis*  
and oat OSC enzymes (per-  
cent bootstrap support in-  
dicated). *Arabidopsis* OSCs  
are indicated with *Arabi-  
dopsis* Genome Initiative  
gene codes. Those genes  
residing in candidate meta-  
bolic gene clusters are  
starred. Oat OSC National  
Center for Biotechnology  
Information accession num-  
bers: SAD1 (CAC84558),  
AsCS1 (CAC84559). (B)  
Map of the triterpene gene  
cluster on *Arabidopsis* chro-  
mosome 5. T-DNA inser-  
tion mutants are indicated.  
(C) Microarray expression  
profiles of the genes in  
Fig. 1B and the two im-  
mediately flanking genes  
At5g47970 and At5g48020  
(neither of which are  
implicated in secondary  
metabolism). Absolute ex-  
pression values are shown  
with identical scales (± SE,  
*n* ≥ 3). Data were retrieved  
from Genevestigator (26).  
The genes in Fig. 1B were  
highly coexpressed across  
392 microarray experi-  
ments [average correlation  
coefficient (*r*) = 0.86]. The  
flanking genes were not  
coexpressed with genes in  
the cluster region or with  
one another (*r* < 0.15).  
Coexpression analysis was  
performed at the Bio-  
Array Resource (27) with  
data from NASCArrays  
(http://arabidopsis.info/).

![Figure 1](attachment://fig1.png)

Tissue  
At5g47970  
At5g47980  
At5g47990  
At5g48000  
At5g48010  
At5g48020  
Callus  
Cell suspension  
Seeding  
cotyledons  
hypocotyl  
radicle  
Inflorescence  
flower  
carpel  
ovary  
stigma  
petal  
sepal  
stamen  
pollen  
pedicel  
silique  
seed  
stem  
node  
shoot apex  
cauline leaf  
Rosette  
juvenile leaf  
adult leaf  
petiole  
senescent leaf  
hypocotyl  
xylem  
cork  
Roots  
lateral root  
root tip  
elongation zone  
root hair zone  
endodermis  
endodermis+cortex  
epidermis+trichoblasts  
lateral root cap  
stele


whether they function in thalianol modification. We analyzed transferred DNA (T-DNA) insertion mutants in At5g48000, which is immediately adjacent to THAS. At5g48000 is predicted to encode a CYP450 (CYP708A2) belonging to the functionally uncharacterized CYP708 family, a CYP450 family specific to the Brassicaceae (15). Root extracts of mutants affected at At5g48000

(thahl-1 and thahl-2) showed increased thalianol levels compared with the wild type (Fig. 4A and fig. S3, A and B), which suggests that the CYP450 encoded by At5g48000 is required for conversion of thalianol to a downstream product. Furthermore, we observed that peaks 2a to 3b were absent in root extracts of thahl-1 (Fig. 4A) and, so, may correspond to downstream pathway intermediates.

The second CYP450 in this region (At5g47990) (Fig. 1B) belongs to the CYP705 family, another functionally uncharacterized Brassicaceae-specific CYP450 family (16). The CYP705 and CYP708 families belong to the CYP71 and CYP85 clans respectively, which demonstrates that At5g47990 is not a tandem duplicate of At5g48000. Null insertion mutants and RNA interference (RNAi) knockdown lines for At5g47990 had enhanced levels of peaks 2a and 2b relative to wild-type plants (Fig. 4A and fig. S3, C and D). Peaks 2a, 2b, and 2c, with similar ion fragmentation patterns, were identified as thalian-diol [(3S,13S,14R)- malabarica-8,17,21-trien-3,?-diol] (fig. S4) and are likely to represent different thalian-diol isomers. Peaks 2a to 2c were not present in root extracts of thasl-1, which confirmed that production of thalian-diol depends on THAS (Fig. 4A). These peaks were also absent from root extracts of thahl-1 (Fig. 4A), which implicated At5g48000 in thalian-diol biosynthesis. Overexpression of THAH with THAS in Arabidopsis leaves resulted in conversion of thalianol to thalian-diol (fig. S6A), and overexpression of THAH in thahl-1 restored the thalianol pathway (fig. S6B). On the basis of these data, we concluded that At5g48000 encodes thalianol hydroxylase (hereafter, referred to as THAH). Plants that overaccumulate thalian-diol are dwarfed (fig. S6B), which suggests that thalian- diol, like thalianol, is detrimental to growth when produced in the above-ground parts of the plant.

The increased levels of thalian-diol in thadl-1 suggested that At5g47990 was required for conversion of thalian-diol to a further downstream product. We observed that peaks 3a and 3b, which are present in wild-type plants, are absent in thasl-1, thahl-1, and thadl-1 (Fig. 4A). These two peaks were identified as isomers of desaturated thalian-diol [(3S,13S,14R)-malabarica- 8,15,17,21-tetraen-3,?-diol] (fig. S5). Conversion of thalian-diol to desaturated thalian-diol involves introduction of a double bond at carbon 15 (Fig. 4). CYP450 enzymes can catalyze desaturation reactions of this kind [see ref. (17)]. On the basis of these data, we concluded that At5g47990 encodes thalian-diol desaturase (hereafter, referred to as THAD).

These data show that THAS, THAH, and THAD are contiguous coexpressed genes encoding biosynthetic enzymes required for three consecutive steps in the synthesis and modification of thalianol (Fig. 4B). The fourth gene in the cluster (At5g47980) is predicted to encode a BAHD acyltransferase (ACT). As for THAS, THAH, and THAD, this enzyme also belongs to a Brassicaceae-specific enzyme subgroup. Because At5g47980 has an expression pattern very similar to that of THAS, THAH, and THAD and is implicated in secondary metabolism, it is likely to be required for modification of desaturated thalian-diol. However, we have not detected acylated desaturated thalian-diol in Arabidopsis root extracts. This may be because this compound is further modified or sequestered or is present at very low levels.


**Figure Caption:**

Fig. 2. Extracts from yeast and Arabidopsis were analyzed for triterpene content by GC-MS: TIC, total ion chromatograms; EIC 229, extracted ion chromatograms at a mass-charge ratio (m/z) of 229. Data are representative of at least two separate experiments, each with triplicate samples. (A) Yeast empty-vector control; (B) yeast expressing the At5g48010 cDNA; (C and D) from wild-type Arabidopsis; (E) from an At5g48010 (thas1-1) knockout line, (F) from an Arabidopsis line overexpressing THAS. The ion fragmentation pattern of thalianol was as reported (fig. S2) (11). Thalianol epoxide, a product of bis-oxidosqualene cyclization, was detected in yeast only. The chromatograms are scaled to the highest peak. Unlabeled peaks are sterols.

The avenacin gene cluster in oat (*Avena* spp.) confers broad-spectrum resistance to fungal pathogens (2). We tested whether the *Arabidopsis thaliana* gene cluster was also defense-related. We challenged the roots of mutant and wild-type plants with strains of fungal and bacterial plant pathogens (*Alternaria brassicicola*, *Botrytis cinerea*, and *Pseudomonas syringae* pv tomato DC3000) but saw no discernible differences in disease progression (fig. S7). However, examination of data from a recent survey of genome-wide polymorphisms in *Arabidopsis* (18) revealed that the thalianol pathway genes represent one of the most conserved regions of the genome. This is the hallmark of a recent selective sweep and implies that this gene cluster confers an important (and as yet unidentified) selective advantage.

Genes for most metabolic pathways are not clustered in plants. However, clustering facilitates the inheritance of beneficial combinations of genes (7, 19, 20); furthermore, disruption of metabolic gene clusters can lead to accumulation of deleterious intermediates (21). The observations that ectopic overaccumulation of thalianol (Fig. 2A) or thalian-diol (fig. S6B) lead to severe dwarfing in *Arabidopsis* are consistent with a need for tight coordinate regulation of the pathway. It is noteworthy that lines that accumulate elevated levels of either of these compounds have significantly longer roots than the wild type (Fig. 3B), which suggests distinct and organ-specific effects of thalianol and thalian-diol on plant growth. We

**Fig. 3.** (A) Plants overexpressing thalianol synthase (THAS) are dwarfed. (B) Roots from 7-day-old plants that accumulate thalianol (*thah1-1*) or thalian-diol (*thad1-1*) are significantly longer than those of the wild type or *thas1-1* (which lacks the entire pathway). Plants that overexpress THAS and, thus, have elevated levels of thalianol also have significantly longer roots than the control. Error bars are ± SE, *n* = 68 to 90 for three replicate experiments.

**Fig. 4.** (A) Detection of thalianol (1) and other pathway intermediates in root extracts from wild-type and T-DNA insertion lines. TIC, total ion chromatograms; EIC 227 and 229, extracted ion chromatograms at *m/z* of 227 and 229, respectively. (B) Scheme of the thalianol pathway showing the structures of 2,3-oxidosqualene, thalianol (1), thalian-diol (2a, 2b, and 2c), and desaturated thalian-diol (3a and 3b) (see figs. S2, S4, and S5, for respective ion fragmentation patterns). The hydroxyl group introduced to thalianol by THAH to give thalian-diol is drawn in red. GC-MS ionization data indicate that this hydroxyl group is located at one of the four available carbon positions in rings B or C. Peaks 2a to 2c are isomers of thalian-diol and are likely to differ in the position of the hydroxyl group. Because of the low levels of these compounds in *Arabidopsis* root extracts, we were unable to determine the precise position of the hydroxyl group in these isomers by nuclear magnetic resonance. The chromatograms are scaled to the highest peak. The peaks between 26 and 28 min (TIC/EIC) are plant sterols. The data are representative of at least two separate experiments, each with triplicate samples.

Fig. 5. Model of thalianol cluster evolution. The OSC tree is as in Fig. 1. Colored circles next to each OSC indicate the presence of adjacent genes encoding specific classes of other biosynthetic enzymes (see key). The colored diamonds indicate points at which common ancestors of genes for these other specific classes of enzyme are hypothesized to have become located in the vicinity of an ancestral OSC gene. The reconstruction minimizes the number of rearrangement and gene loss events required to reach the present-day chromosomal arrangement. The product of At5g48010 (the OSC gene that lies within the functional gene cluster reported in this paper) is indicated in bold. The existence of other triterpene gene clusters is inferred by association of other clade II OSCs with genes for other enzymes implicated in secondary metabolism.

- OSC
- CYP708/CYP702 (CYP85 clan)
- CYP705 (CYP71 clan)
- BAHD ACT
- CYP51H (CYP51 clan)
- Non-BAHD ACT

At4g15340  
At4g15370  
At5g48010 (THAS)  
At5g36150  
At1g78500  
At5g42600  
At1g78950  
At1g78955  
At1g78970  
At1g66960  
At1g78960  
Oat β-amyrin synthase (SAD1)  
At3g45130 Lanosterol synthase  
At2g07050 Cycloartenol synthase  
Oat cycloartenol synthase (AsCS1)

note that the four coexpressed genes within the thalianol gene cluster have marked histone H3 lysine 27 trimethylation, whereas the immediate flanking genes do not (22), which suggests that clustering may also facilitate coordinate regulation of the gene cluster at the chromatin level.

Although oats and *Arabidopsis* both contain gene clusters for triterpene synthesis (the avenacin and thalianol clusters, respectively), these two gene clusters are unlikely to share a common origin. This is supported by the fact that the oat *Sad1* and *Sad2* genes do not have orthologs in *Arabidopsis* and are monocot-specific (6–8). Furthermore, there is no evidence for horizontal transfer of either gene cluster from microbes or elsewhere. Phylogenetic analysis suggests that an ancestral gene cluster formed in *Arabidopsis* around the progenitor of the lineage-specific OSC clade II (Fig. 5). Sequential rearrangements, duplications, and gene loss presumably led to formation of the present-day thalianol cluster. Cluster formation may have been accompanied by the rapid expansion and functional diversification of the lineage-specific OSC clade II, along with the lineage-specific CYP702/708, CYP705, and ACT gene families. In addition, whereas THAS makes thalianol, the other OSCs in clade II produce different triterpene products when expressed in yeast (3, 5). Some of these OSCs may be components of other functional triterpene gene clusters (Fig. 1A), as suggested by genome-wide coexpression of CYP450s (23).

An obvious assumption may be that gene clusters of the kind that we have observed were inherited from early evolutionary progenitor species. However, our data clearly indicate that the thalianol and avenacin gene clusters are the products of separate and recent evolutionary events.

This finding suggests that eukaryotic genomes are capable of remarkable plasticity and can assemble operon-like gene clusters de novo, which raises intriguing questions about the molecular mechanisms and evolutionary pressures that have acted to cause these gene clustering arrangements to assemble and become fixed. Comparative genomics will now enable us to trace the origins of such gene clusters and so to gain insights into the mechanisms that drive their formation. A further intriguing question is concerned with why genes for some metabolic pathways are clustered, whereas others are not. Our identification of two triterpene gene clusters [for thalianol in *Arabidopsis* (this paper) and for avenacins in oat (2, 6–8, 10)] implies that triterpene pathways may be predisposed to gene clustering. There are two other examples of gene clusters for plant defense compounds (for rice diterpenes and maize benzoxazinoids) (19, 24, 25). As we learn more about why genes for some metabolic pathways are clustered and others are not, we may need to redefine our understanding of plant metabolism.

### References and Notes

1. K. Hostettmann, A. Marston, *Saponins* (Cambridge Univ. Press, Cambridge, 1995).
2. K. Papadopoulou, R. E. Melton, M. Leggett, M. J. Daniels, A. E. Osbourn, *Proc. Natl. Acad. Sci. U.S.A.* **96**, 12923 (1999).
3. D. R. Phillips, J. M. Rasbery, B. Bartel, S. P. Matsuda, *Curr. Opin. Plant Biol.* **9**, 305 (2006).
4. K. T. Liby, M. M. Yore, M. B. Sporn, *Nat. Rev. Cancer* **7**, 357 (2007).
5. I. Abe, *Nat. Prod. Rep.* **24**, 1311 (2007).
6. K. Haralampidis et al., *Proc. Natl. Acad. Sci. U.S.A.* **98**, 13431 (2001).
7. X. Qi et al., *Proc. Natl. Acad. Sci. U.S.A.* **101**, 8233 (2004).
8. X. Qi et al., *Proc. Natl. Acad. Sci. U.S.A.* **103**, 18848 (2006).
9. Materials and methods are available as supporting material on Science Online.
10. M. B. Eisen, P. T. Spellman, P. O. Brown, D. Botstein, *Proc. Natl. Acad. Sci. U.S.A.* **95**, 14863 (1998).
11. G. C. Fazio, R. Xu, S. P. Matsuda, *J. Am. Chem. Soc.* **126**, 5678 (2004).
12. M. Koornneef, J. H. Vanderveen, *Theor. Appl. Genet.* **58**, 257 (1980).
13. T. Yokota, *Trends Plant Sci.* **2**, 137 (1997).
14. S. D. Clouse, *Plant Cell* **14**, 1995 (2002).
15. B. Hamberger, J. Bohlmann, *Biochem. Soc. Trans.* **34**, 1209 (2006).
16. D. R. Nelson, M. A. Schuler, S. M. Paquette, D. Werck-Reichhart, S. Bak, *Plant Physiol.* **135**, 756 (2004).
17. A. E. Rettie, A. W. Rettenmeier, W. N. Howald, T. A. Baillie, *Science* **235**, 890 (1987).
18. J. O. Borevitz et al., *Proc. Natl. Acad. Sci. U.S.A.* **104**, 12057 (2007).
19. A. Gierl, M. Frey, *Planta* **213**, 493 (2001).
20. S. Wong, K. H. Wolfe, *Nat. Genet.* **37**, 777 (2005).
21. P. Mylona et al., *Plant Cell* **20**, 201 (2008).
22. X. Zhang et al., *PLoS Biol.* **5**, e129 (2007).
23. J. Ehlting, N. J. Provart, D. Werck-Reichhart, *Biochem. Soc. Trans.* **34**, 1192 (2006).
24. R. Jonczyk et al., *Plant Physiol.* **146**, 1053 (2008).
25. K. Shimura et al., *J. Biol. Chem.* **282**, 34013 (2007).
26. P. Zimmermann, M. Hirsch-Hoffmann, L. Hennig, W. Gruissem, *Plant Physiol.* **136**, 2621 (2004).
27. K. Toufighi, S. M. Brady, R. Austin, E. Ly, N. J. Provart, *Plant J.* **43**, 153 (2005).

We thank D. Baulcombe, L. Dolan, R. Field, S. Kopriva, K. Papadopoulou, P. Shaw, A. Smith, D. Studholme, and T. Wang for comments; John Innes Center (JIC) Metabolite Services for metabolite analysis; L. Peña-Rodríguez and M. Rejzek for advice on chemistry; R. Melton and members of the Sainsbury laboratory for assistance with the pathogenicity work; and the Biotechnology and Biological Sciences Research Council (U.K.) for funding.

### Supporting Online Material

www.sciencemag.org/cgi/content/full/1154990/DC1  
Materials and Methods  
Figs. S1 to S8  
Table S1  
References  

8 January 2008; accepted 7 March 2008  
Published online 20 March 2008  
10.1126/science.1154990  
Include this information when citing this paper.
