
Structural and Functional Relationships of the XPF/MUS81 Family of Proteins

Alberto Ciccia,$^{1,3}$ Neil McDonald,$^{1,2}$ and Stephen C. West$^{1}$

$^{1}$London Research Institute, Cancer Research UK, Clare Hall Laboratories, Hertfordshire EN6 3LD, United Kingdom; email: stephen.west@cancer.org.uk  
$^{2}$School of Crystallography, Birkbeck College, London WC1E 7HX, United Kingdom  
$^{3}$Present address: Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115  

### Key Words
cross-link repair, Fanconi anemia, meiosis, nucleotide excision repair, recombination

### Abstract
Proteins belonging to the XPF/MUS81 family play important roles in the repair of DNA lesions caused by UV-light or DNA cross-linking agents. Most eukaryotes have four family members that assemble into two distinct heterodimeric complexes, XPF-ERCC1 and MUS81-EME1. Each complex contains one catalytic and one noncatalytic subunit and exhibits endonuclease activity with a variety of 3′-flap or fork DNA structures. The catalytic subunits share a characteristic core containing an excision repair cross complementation group 4 (ERCC4) nuclease domain and a tandem helix-hairpin-helix (HhH)$_{2}$ domain. Diverged domains are present in the noncatalytic subunits and may be required for substrate targeting. Vertebrates possess two additional family members, FANCM and Fanconi anemia-associated protein 24 kDa (FAAP24), which possess inactive nuclease domains. Instead, FANCM contains a functional Superfamily 2 (SF2) helicase domain that is required for DNA translocation. Determining how these enzymes recognize specific DNA substrates and promote key repair reactions is an important challenge for the future.

Contents
INTRODUCTION 260
STRUCTURAL DOMAINS AND EVOLUTIONARY RELATIONSHIPS WITHIN THE XPF/MUS81 FAMILY 261
The ERCC4 Domain 263
The (HhH)₂ Domain 264
SF2 Helicase Domains 267
XPF-ERCC1 (RAD1-RAD10) 267
Biochemical Activities 267
Role in Nucleotide Excision Repair 268
Role in Homologous Recombinational Repair 270
Relationship to Aging 271
MUS81 COMPLEXES 272
Biochemical Activities 272
Meiotic Recombination 273
Repair of Blocked or Collapsed Replication Forks 274
Tumorigenesis 276
FANCM-FAAP24 277
Biochemical Activities 277
Fanconi Anemia 278
Interstrand Cross-Link Repair 279
CONCLUSIONS 281

INTRODUCTION

The faithful conservation of the genome is essential for cell survival. In order to maintain genomic integrity, our cellular DNA needs to be protected from spontaneous decay and damage that is induced by environmental agents. Spontaneous DNA damage, commonly observed as base or sugar fragmentation, can be induced by reactive oxygen species (ROS) derived from normal cellular metabolic processes. Environmental DNA damage includes that produced by physical sources, such as ultraviolet (UV) light or ionizing radiation (IR), or by chemicals, such as alkylating or cross-linking agents. The coding potential of DNA can also be altered by base deamination events or by the misincorporation of nucleotides during DNA replication.

The accumulation of DNA lesions is known to interfere with basic cellular processes such as DNA replication, transcription, and cell division. In particular, DNA damage can impair the progression of DNA and RNA polymerases. Moreover, radiation-induced lesions and DNA cross-links can lead to the formation of aberrant chromosome structures that result in the missegregation of chromosomes during mitosis.

To counteract the deleterious effects of DNA damage and maintain genome integrity, all organisms have evolved elaborate mechanisms for DNA repair. These are often highly specific for particular types of lesions. Defined repair pathways promote the recognition and subsequent removal of DNA lesions in reactions that generally involve the nucleolytic cleavage of DNA strands. A wide variety of nucleases exist that can digest DNA from its termini or from nicks and breaks (exonucleases), or can catalyze nick incision within regions of single-stranded (ss) or double-stranded (ds) DNA (endonucleases). Endonucleases can cleave DNA in a sequence- or structure-specific manner, and the structure-specific nucleases have been classified according to their preferred DNA substrate. Common structures processed by the structure-specific nucleases include four-way DNA junctions and 5′- or 3′-flap and fork structures.

Four-way DNA junctions, also known as Holliday junctions (HJs), occur naturally in cells as intermediates of either genetic recombination or recombinational DNA repair (1). A specific class of nucleases, the HJ resolvases, recognizes and cleaves these structures to release nicked duplex products. HJ resolvases have been identified in bacteriophage, bacteria, and archaea. The first to be identified were bacteriophage T4 endonuclease VII and T7 endonuclease I, followed by cellular resolvases such as *Escherichia coli* RuvC and RusA, and archaeal Hjc and Hje (2). In eukaryotes, mitochondrial HJ resolvases, such
as Saccharomyces cerevisiae Cce1 and Schizosac-
charomyces pombe Ydc2, have been identified,
but at present, the resolvases that act upon
HJs within the nucleus are unknown. HJ res-
olution generally occurs by the introduction
of two symmetrically related nicks across the
junction.

Structures that contain 5′-flaps are gener-
ated during repair of DNA base lesions or
during normal DNA replication, and nucle-
ases that exhibit specific cleavage activity on
these DNA structures belong to the FEN-1
(Flap Endonuclease-1) family of endonucle-
ases (3). In humans, the prototypic member of
this family is FEN-1 itself (originally known
as DNase IV), which plays an important role
in fragment processing during lagging strand
DNA synthesis. The yeast homolog of FEN-1
is Rad27. Other members of this eukaryotic
nuclease family include human XPG (and its
yeast homolog Rad2), which is involved in nu-
cleotide excision repair (NER) (4).

Nucleases that act upon 3′-flap structures
are known as the 3′-flap endonucleases and are
involved in DNA repair pathways that are nec-
essary for the removal of UV-light-induced
DNA lesions and cross-links between DNA
strands. Many 3′-flap endonucleases belong
to the XPF/MUS81 family, which is the sub-
ject of this review. Individuals with defects in
proteins belonging to this family are prone
to cancer and/or premature aging. However,
not all members of the XPF/MUS81 family
exhibit nuclease activity, leading us to detail
the structural relationships and multiple func-
tions of all currently known members of the
protein family.

**STRUCTURAL DOMAINS**
**AND EVOLUTIONARY**
**RELATIONSHIPS WITHIN**
**THE XPF/MUS81 FAMILY**

Members of the XPF/MUS81 protein fam-
ily are present throughout the eukaryal and
archaeal kingdoms but are not found in
prokaryotes (Figure 1) (5). They exist ei-
ther as heterodimers (eukaryotes) or as homo-
dimers (archaea) in their functionally compe-
tent states (6–9).

Most eukaryotes have at least four family
members that assemble into two distinct het-
erodimeric endonucleases, consisting of a cat-
alytic and a noncatalytic subunit (Figure 2).
In human cells, there are three endonucle-
ases known as XPF-ERCC1, MUS81-EME1,
and MUS81-EME2 (the catalytic subunit
is defined first) (6, 10–14). Orthologs of
XPF-ERCC1 include Rad1-Rad10 in S. cere-
visiae (15, 16) and Rad16-Swi10 in S. pombe
(17). MUS81-EME1 orthologs bear the same
name in other organisms except in S. cerevisiae
where these orthologs are known as Mus81-
Mms4 (7, 18). Two additional XPF-MUS81
family members are found exclusively in ver-
tebrates and form a heterodimer known as
FANCM-FAAP24 (14). To date, no endonu-
clease activity has been found to be associated
with this protein. MUS81 and ERCC1 sub-
units have not been identified in the chicken
genome, suggesting that a different quater-
nary arrangement may exist in this species.

In eukaryotes, the XPF/MUS81 fam-
ily can be functionally and structurally di-
vided into catalytic and noncatalytic fam-
ily members (Figure 2). Catalytic forms all
share a characteristic core consisting of an
excision repair cross complementation group
4 (ERCC4) endonuclease domain and a tan-
dem helix-hairpin-helix (HhH)₂ domain (19)
(Figure 3a). This ERCC4-(HhH)₂ core is
the smallest functional unit required to both
target and cleave DNA junction structures
(8, 20). The noncatalytic subunits retain the
same modular structure, but their ERCC4
domains have extensively diverged from the
catalytic counterparts and lack intrinsic en-
donuclease activity. Additional domains and
motifs embellish the architecture of individ-
ual family members and most likely contribute
to specialized molecular functions and distinct
DNA substrate specificities.

Archaeal genomes encode for either a
short XPF/MUS81 homolog containing the
ERCC4-(HhH)₂ core and a C-terminal
PCNA-interacting peptide (5, 21) or a long

www.annualreviews.org • The XPF/MUS81 Family of Proteins
261

---

**FEN-1:** Flap
Endonuclease-1

**NER:** nucleotide
excision repair

**ERCC:** excision
repair cross
complementation

**FAAP24:** Fanconi
anemia-associated
protein 24 kDa

**(HhH)₂:** tandem
helix-hairpin-helix
Bacteria

Archaea
- Crenarchaea
  - ApeXPF
    - ERCC4 248 Stalled fork repair, NER (?)
  - SsoXPF
    - ERCC4 233 Stalled fork repair, NER (?)

Euryarchaea
- PfuHef
  - DEAH ERCC4 764 Stalled fork repair, NER (?)

Plants
- ScRad1
  - DEAH ERCC4 1100 NER, ICL repair
- ScRad10
  - ERCC4 210 NER, ICL repair
- SpRad16
  - DEAH ERCC4 892 NER, ICL repair
- SpSwi10
  - ERCC4 252 NER, ICL repair

Fungi
- ScMus81
  - ERCC4 632 Stalled fork repair, meiotic recombination
- ScMms4
  - ERCC4 691 Stalled fork repair, meiotic recombination
- SpMus81
  - ERCC4 572 Stalled fork repair, meiotic recombination
- SpEme1
  - ERCC4 738 Stalled fork repair, meiotic recombination

Invertebrates
- HsXPF
  - DEAH ERCC4 905 NER, ICL repair
- HsERCC1
  - ERCC4 297 NER, ICL repair
- GgXPF
  - DEAH ERCC4 903 NER, ICL repair

Animals
- Vertebrates
  - HsMUS81
    - ERCC4 551 Stalled fork repair, ICL repair
  - HsEME1
    - ERCC4 583 Stalled fork repair, ICL repair
  - HsEME2
    - ERCC4 379 ?
  - GgEME1
    - ERCC4 548 Stalled fork repair, ICL repair
  - GgEME2
    - ERCC4 259 ?
  - HsFANCM
    - DEAH ERCC4 2048 ICL repair
  - HsFAAP24
    - ERCC4 215 ICL repair
  - GgFANCM
    - DEAH ERCC4 1773 ICL repair
  - GgFAAP24
    - ERCC4 215 ICL repair

Figure 1

Evolutionary tree of the XPF and MUS81 family. The domain organization and some functions of the archaeal and eukaryotic members of the XPF and MUS81 family are indicated. Orthologs are indicated with similar colors. Superfamily 2 DEAH helicase domains (blue), excision repair cross complementation group 4 (ERCC4) nuclease domains (red), and HhH motifs (black ovals) are indicated. Inactive DEAH and ERCC4 domains and pseudo-HhH motifs are labeled. Abbreviations: ICL, interstrand cross-link; NER, nucleotide excision repair.

ANRV 345-BI77-12 ARI 6 May 2008 14:29

Nucleotide excision repair Recombination/replication fork restart/cross-link repair

| H. sapiens | S. cerevisiae | S. pombe |
| --- | --- | --- |
| XPF ERCC1 | MUS81 EME1 MUS81 EME2 FANCM FAAP24 |  |
| Rad1 Rad10 | Mus81 Mms4 |  |
| Rad16 Swi10 | Mus81 Eme1 |  |

**Figure 2**

Heterodimeric associations of the XPF/MUS81 family members. The eukaryotic XPF/MUS81 family members are indicated. Catalytic and noncatalytic subunits are shown on the left and right of each heterodimer, respectively. The primary functions of each complex are indicated.

XPF/MUS81 that contains a functional Superfamily 2 (SF2) helicase domain (22, 23) (Figure 1). Archaeal XPF/MUS81 homologs are closer in sequence and structure to XPF-ERCC1 but are functionally akin to MUS81-EME1 in their preference for 3′-flaps, forks, and other branched structures (24). Despite their homodimeric quaternary structure, archaeal XPF/MUS81 endonucleases require only a single active site for nuclease activity, consistent with their eukaryal counterparts (25). A single active site is sufficient to explain the cleavage of a single phosphodiester bond from one strand of a duplex in a branched DNA substrate (Figure 3c).

**The ERCC4 Domain**

When first identified, XPF was known as ERCC4. The nuclease domain within XPF has since been known as the ERCC4 domain. The region is unique to the XPF/MUS81 family and has two elements; a subdomain distantly related to the nuclease domain of type II restriction endonucleases and a helix-β-strand-helix dimerization motif (26). The ERCC4 domains of the catalytic subunits XPF and MUS81 have a GDXnERKXxD motif located within a central β-sheet (Figures 3a and 4). This motif is required for metal-dependent endonuclease activity but not for DNA junction binding (8, 27). There is a good correspondence between the active site GDXnERKXxD motif and the PDXn(E/D)XK motif of type II restriction endonucleases (26). The acidic residues within the motif coordinate a divalent metal ion that is required for catalysis (8) (Figure 3b). However, the precise role of the essential conserved lysine and arginine in the motif is not

SF2: Superfamily 2

www.annualreviews.org • The XPF/MUS81 Family of Proteins 263

The ERCC4 nuclease domain. (a) Schematic of the ERCC4 endonuclease domain fold, highlighting the positions of the essential catalytic and metal-binding residues. Pink β-strands highlight regions present in type II restriction enzymes. The dimerization motif of the ERCC4 domain is indicated. (b) Close-up of the conserved active site residues within the *Aeropyrum pernix* XPF. A single divalent metal ion is shown in purple. The figure was made using PyMol (http://pymol.sourceforge.net). (c) Reaction catalyzed by XPF/MUS81 family endonucleases. A single phosphodiester bond in one strand of the duplex is cleaved thereby generating 5′ phosphate and 3′ OH termini. Cleavage occurs close to the transition from dsDNA to ssDNA. The X and Y arms may be double or single stranded, depending on the XPF/MUS81 complex (see Figure 5). Abbreviations: β1-β6, β-strands; DG and ERKD, amino acids; Mg²⁺, bound divalent metal.

yet clear (8, 27). The lysine side chain is lacking from the ERCC4 domain in human FANCM, consistent with observations indicating that FANCM-FAAP24 has no endonuclease activity (14, 28). The equivalent regions of the noncatalytic subunits (ERCC1, EME1, and FAAP24) have diverged (Figure 4). This may reflect other noncatalytic roles that have been proposed for these domains, such as binding ssDNA or an ability to target the endonuclease to defined DNA structures or sites of DNA damage (20, 29).

### The (HhH)₂ Domain

XPF/MUS81 endonucleases recognize ss/ds DNA junctions with a precise polarity (Figures 3c and 5). The major determinants for binding branched DNA junctions map to the (HhH)₂ domain (8, 9, 23). This globular domain consists of two copies of a sequence-independent HhH DNA-binding motif, which are bridged by a fifth connecting helix (19, 30, 31). (HhH)₂ domains are found in many nonspecific DNA-binding proteins,

MUS81 320 PANPGEVLVLDHIVERKRLDDLCSSIIDGRFRERQFRLKRCGLERRVYLVEEHGSVHNLSLPESLTL
EME1 348 -TDITAKTAGKALSILVIVDQ--EKCFSLELLFFDFLPCTSAQ--NPPRRGKQGANKQTKKQQQRQ
EME2 114 LLEPEEFLQGVATLTQISGP--THWVPWISEPETTARPHLAVIGLDAYLWSRQHVSRTQ-QPESP
XPF 706 ----ILTPEMCVERKSISDLIGSLNNGRLYSQCISMSRYK--RPVLLIEFDPSKPFSLTSRGA
ERCC1 132 ----LGQSTCALFLSLRYHN--LHPDYIHGRLQSLGKNFA--LRVLLVQVDVKDPQ----
FANCM 1852 -GCDYIVSNRMVVERRSQSE-----------------------------MLNSVNKNKFIEQIQHLQSME
FAAP24 47 -TPDFYLSNRCCIILYVTEAD----------------------------LVAGNGYRKRLVRVRNSNNLK

ERKXXXD

MUS81 385 LQAVTNTQVIDGFFVKRTADIKESAAYLALLTRGLQRLYQGHTLRSRPWGTPGNPESGAMTSPNP
EME1 408 PEASIGSMVSRVDAEEALVDLQLHTEAQAQIVQSWKEELADFTCAFTKAVAEAPFKKLRDDET----
EME2 206 KVAGAEVAVSWPEVEEAALVLLQLWANLDVLLVASWQELSERHVCAVTKALAQYPCLKYRESQ----
XPF 764 LFQEISSNDISSKLTLLTLHFPRLLILWCPSPHATAELFEELKQSKPQPDAAATALAITADS----
ERCC1 181 ALKEALKMCILLADCTLI--------LAWS--PEEAGRYLETYKAYEQKP----------ADL---
FANCM 1892 ERICVIVEKDREKTGDTSRMFFRRTKSYDSLLTTLIGAGIRILFSSEQETADLLKELSOLVE----
FAAP24 87 G--IVVVEKTR----MSEQYFPALQKFT-----VLDLGMVLLPVASQMEASCLVIQLVQEQ----

MUS81 450 LCSLLTFSDD---FNAGAIKNKAQS-VREVFARQLMQVRGVSGEKAALVDRYSTPASLLAAYDAC
EME1 469 TFSFCLESD---WAGGVKVDLAGRGLALVWRRQIQQLNRVSLEMASAVVNAYPSPQLLVQAYQC
EME2 267 AFSFCTAGR---WAAEPVARDGAGLQAAWRRQIRQFSRVSPAVALIDVTAFPSPRLLQQALEAC
XPF 825 -ETLPESEK---YNPGBP---------QDFLLKMPGVNAKNCRLMHHVKNIAELAALSQDE
ERCC1 223 -LMEKLEQD---FVSRV---------TECLTTVKSVNKTDSQTLLTTFGSLEQLIAASRED
FANCM 1953 ----QR---KNVGITHVPTVVNSNKSEALQFYLSIPNITSYITALNMCHQFSSVKRMANSSEQ
FAAP24 137 ----TKEPSKNPLLGGKKRALLLSEPSSLIRTVQQIPGVGKVKAPELLLQKFPSIQOLSNASIGE

MUS81 511 ATPKEQETLLSTIKCGR-------LQRNLGBALSRTLSQLYCSYGPLT---------
EME1 531 FSDKERQNLLADIQVRRGEGVTSTRRTIGBELSRRITYLQMTTLQPHLSDLSDAD---------
EME2 329 STERERMGLLADLPVPPSEGGR--PRRVGPELSRRICLFITTANPDLLLLDLGS---------
XPF 873 -----TSIL---------GNAAANKQIYDFIHTSFAEVVSKGKGKK---------
ERCC1 271 -----LACP---------GLGPQKARRLFDVDLHEPFLKVP---------
FANCM 2008 ----ISMYA--------QVTHQKAEEIYYTIHYVDIQMLPNDLNQDRKLKSDI
FAAP24 195 ----LEQ---------VVGQAVAQQI------HAFFTQPR---------

Figure 4

Sequence alignment of the ERCC4 nuclease domain and the HhH motifs of the *Homo sapiens* XPF and MUS81 family. Identical and similar residues are indicated in red and light blue boxes, respectively. Residues required for nuclease activity are shown in green. Hydrophobic amino acids critical for the formation of HhH motifs are indicated in dark blue. Sequence alignments were carried out using ClustalW.

including the prokaryotic HJ-binding protein RuvA and the NER repair nuclease UvrC (32, 33). (HhH)₂ domains use their hairpins to bridge across the minor groove face of duplex DNA by engaging a back-bone phosphate from each strand separated by three base pairs (9, 32). As well as binding DNA, the (HhH)₂ domains associate as dimers and contribute to the stability and oligomeric state of XPF/MUS81 endonucleases. As dimers, they can potentially present two surfaces for binding duplex DNA (9, 25). Crystallographic and DNA footprinting stud-

ies have shown that, in archaea, both *Aeropyrum pernix* XPF and the *Pyrococcus furiosus* XPF homolog, Hef, utilize their (HhH)₂ dimer dimers to engage both the upstream and downstream duplex regions of their preferred branched substrates (9, 22, 25). Structural and functional data for the XPF-ERCC1 (HhH)₂ domain heterodimer indicate that the complex binds DNA only through the ERCC1 hairpins and not those of XPF, implying a substrate-targeting function for ERCC1 (34). This could explain why XPF-ERCC1 preferentially cleaves splayed-arm and bubble

**P. furiosus Hef**

3′-flap , Replication fork

Replication fork with lagging strand gap Mobile replication fork

---

**S. solfataricus XPF**

3′-flap , Replication fork , Nicked HJ D-loop Splayed arm HJ

---

**S. cerevisiae Rad1/Rad10 and H. sapiens XPF/ERCC1**

Splayed arm Bubble Stem loop 3′-flap

---

**Yeast Mus81/Eme1 and Mus81/Mms4**

Nicked HJ 3′-flap Replication fork D-loop HJ

---

**H. sapiens MUS81/EME1 and MUS81/EME2**

3′-flap Replication fork Splayed arm HJ

---

Conversion of one DNA species Relative levels of cleavage of  
into another by junction migration one DNA compared to another  

**Figure 5**

Substrate specificity of members of the XPF/MUS81 family. Schematic representation of the substrates that are cleaved in vitro by various XPF/MUS81 family proteins. A red arrow indicates the approximate sites of cleavage within each DNA structure. Preferential cleavage of one particular substrate over another is indicated by > and >>. Substrates cleaved with equal efficiency are separated by commas. Black circles indicate 5′ termini. Green arrows indicate conversion of one DNA species into another by junction migration. Abbreviations: HJ, Holliday junction; H. sapiens, Homo sapiens; P. furiosus, Pyrococcus furiosus; S. cerevisiae, Saccharomyces cerevisiae; S. solfataricus, Sulfolobus solfataricus.

substrates bearing only a single stretch of duplex DNA (35).

**SF2 Helicase Domains**

Both the Hef and FANCM proteins contain a functional SF2 helicase domain (23, 36) (Figure 1). Biochemical data for Hef demonstrates how this helicase domain cooperates with and stimulates endonuclease activity with fork structures (37). The Hef helicase domain consists of three structural domains: two RecA-like folds, bearing the characteristic helicase motifs crucial for ATP hydrolysis and DNA translocation, separated by a unique α-helical domain (23, 38). This insert domain contributes to branched DNA recognition and is reminiscent of the “thumb” domain from DNA polymerases (23). XPF and its orthologs have a degenerate SF2 helicase-like domain, which lacks key residues required for ATP binding and hydrolysis (39, 40). For example, acidic residues within the DEAD/DExH motif III of the first RecA domain typically coordinate Mg²⁺ but are absent in XPF (40). Therefore within the XPF/MUS81 family, there are examples of functional and nonfunctional SF2 helicase domains analogous to the examples described for functional/nonfunctional ERCC4 and (HhH)₂ domains. Although it would appear at first glance that the nonfunctional counterparts have diverged through sequence drift and in so doing have lost a catalytic or binding function, it is also likely that this has driven the acquisition of new and unique functions for individual family members. However, the precise role(s) of the inactive DEAH motif of XPF and its orthologs has yet to be determined.

The biochemical activities of the DEAH, ERCC4, and (HhH)₂ domains of the eukaryotic heterodimers of the XPF/MUS81 family are described in the next sections. In addition, we examine the physiological role of these heterodimers in the various DNA repair pathways in which they are thought to operate.

---

**XPF-ERCC1 (RAD1-RAD10)**

The most well-characterized proteins of the XPF/MUS81 family are S. *cerevisiae* Rad1-Rad10 and its human homolog XPF-ERCC1. Both proteins play key roles in the excision of DNA lesions in NER and are also required for certain types of homologous recombination events and for the repair of DNA cross-links.

**Biochemical Activities**

Early biochemical studies of Rad1-Rad10 purified from S. *cerevisiae* showed that Rad1-Rad10 possessed ssDNA endonuclease activity, as indicated by their ability to degrade covalently closed M13 bacteriophage ssDNA (41, 42). Endonuclease activity required the formation of Rad1-Rad10 complex, which is dependent on the C-terminal regions of Rad1 (amino acids 809–997) and Rad10 (amino acids 90–210) (42, 43). Subsequent experiments demonstrated that Rad1-Rad10 preferentially cleaves splayed-arm structures by introducing nicks in the dsDNA region on the DNA strand with the 3′ tail (Figure 5) (44). Efficient cleavage was also observed for DNA bubble structures, which are nicked by Rad1-Rad10 on the 5′ side of the bubble (45). Similarly, Rad1-Rad10 cleaves 3′-flap structures, although approximately twofold less efficiently than splayed-arm structures (Figure 5) (46).

Like its yeast counterpart, vertebrate XPF-ERCC1 functions as an endonuclease that preferentially cleaves splayed-arm, bubble, and stem-loop structures. Cleavage occurs on dsDNA regions close to the transition between dsDNA and ssDNA (Figure 5) (6, 35, 47). As mentioned above, the critical residues for endonuclease activity reside in the ERCC4 domain of XPF. These residues are absent in the ERCC4 domain of ERCC1 (Figure 4) (27, 48). Recent data indicate that ERCC1 could play a critical role in targeting the XPF-ERCC1 complex to DNA (20, 34). The XPF-ERCC1 complex has been proposed to have two distinct DNA-binding interfaces: the central domain of ERCC1 and
the HhH motifs. In particular, the residues Gly276 and Gly278, which form a classical glycine-hydrophobic residue-glycine (GhG) hairpin in the second HhH motif of ERCC1, are thought to be involved in DNA binding (Figure 4) (amino acids 276–278 of ERCC1). In contrast, the HhH motifs of XPF, which do not contain classical GhG hairpins, fail to interact with DNA (Figure 4) (amino acids 878–880 of XPF). Taken together, these observations support a model in which DNA binding and nuclease activities are located in different subunits of the XPF-ERCC1 complex (34).

**Role in Nucleotide Excision Repair**

In *S. cerevisiae*, Rad1 and Rad10 are required for the excision of DNA lesions generated by UV radiation (49). Similarly, XPF and ERCC1 are necessary for the efficient repair of UV lesions in mammals (50, 51). In humans, *XP* mutations have been identified in patients with an inheritable condition known as xeroderma pigmentosum (XP) (6). XP is a heterogeneous genetic disorder caused by mutations in one of eight genes (*XPA, XPB, XPC, XPD, XPE, XPF, XPG,* and *XPV*) (52). XP patients display exquisite sensitivity to UV radiation and exhibit more than 1000-fold increased incidence of sun-induced skin cancer (53).

All XP proteins, except for XPV, are involved in the NER pathway. NER is required for the repair of DNA lesions caused by UV radiation, DNA cross-linking agents, or other chemicals that cause distortions of the DNA double helix. According to the transcriptional status of the genomic locus that is repaired, NER can be subdivided into two pathways: global-genome NER (GG-NER) and transcription-coupled repair (TCR). GG-NER is the basic NER pathway that repairs DNA lesions in the entire genome, whereas TCR is an NER pathway that is specifically activated when RNA polymerase stalls upon encountering DNA lesions.

The GG-NER pathway has been reconstituted in vitro with purified yeast or human proteins, and the mechanism of repair is now well understood (52). In humans, the repair reaction is initiated by the XPC-hHR23B complex, which scans DNA for disruptions to normal base pairing (see Figure 6, step $a$). XPA, RPA, TFIIH, and XPG are then recruited to the site of damage (step $b$). The TFIIH subunits XPB and XPD then unwind the duplex DNA on both sides of the lesion and create a bubble structure that is stabilized by the single-strand binding protein RPA (step $b$). Following bubble formation, the DNA lesion is removed by a dual incision reaction in which nicks are introduced at the $5^{\prime}$ and $3^{\prime}$ side of the lesion. $5^{\prime}$ and $3^{\prime}$ cleavage involves the nuclease activities of XPF-ERCC1 and XPG, respectively (step $c$) (4, 6). Thus, XPF-ERCC1 is acting as a specific $3^{\prime}$-flap endonuclease. The double-incision reaction mediated by the two nucleases allows the release of a fragment of ssDNA $\sim 30 \mathrm{nt}$ long that contains the lesion. Subsequent steps involve DNA synthesis and ligation to restore genomic integrity (steps $d$ and $e$).

In addition to their role in GG-NER, XPF-ERCC1 and Rad1-Rad10 are also involved in TCR. As previously mentioned, TCR is activated by stalled RNA polymerase. It is thought that, following a transcription block, the RNA polymerase is polyubiquitylated and subsequently degraded, which allows the repair of the DNA lesion (54). In mammals, ubiquitylation of RNA polymerase involves CSA and CSB, two proteins that are defective in patients with Cockayne syndrome (55). Cockayne syndrome is a genetic disorder characterized by sensitivity to UV radiation, growth defects, and severe neurological impairment (56). Following the removal of RNA polymerase (Figure 6, steps $f$ and $g$), the recruitment of XPA, RPA, TFIIH, XPG, and XPF-ERCC1 promotes the repair of the DNA lesion through a mechanism similar to GG-NER.
Figure 6

Schematic representation of the mechanism of mammalian nucleotide excision repair. Following the recruitment of XPA, XPG, TFIIH, and RPA, a DNA bubble is generated by the helicase activity of the TFIIH subunits XPB and XPD. The DNA strand containing the DNA lesion is then cleaved on the 5′ side by XPF-ERCC1 and on the 3′ side by XPG (step c), resulting in the excision of a short fragment of ssDNA containing the lesion. For other details see text. Abbreviations: CSA, Cockayne syndrome group A protein; CSB, Cockayne syndrome group B protein; DDB1, DNA damage binding protein 1; ERCC1, excision repair cross complementing group (ERCC) 1 protein; hHR23B, human homolog of yeast Rad23B; RNA Pol II, RNA polymerase II; RPA, Replication Protein A; TFIIH, transcription factor IIH complex; Ub, ubiquitin; XPA, XPC, XPE, XPF, XPG; xeroderma pigmentosum complementation groups A, C, E, F, and G proteins.

Role in Homologous Recombinational Repair

Rad1-Rad10 and XPF-ERCC1 are also involved in certain types of homologous recombination repair (HRR) (57, 58). HRR provides an accurate mechanism for the repair of ss-DNA gaps and double-strand breaks (DSBs), which can be produced by DNA-damaging agents, such as IR and UV radiation, or can occur during DNA replication as a consequence of replication fork blockage or collapse (59, 60). When DSBs are generated in a region of DNA containing repeated sequences, such as those that occur at rDNA loci, a mechanism of HRR known as single-strand annealing (SSA) can repair the break (Figure 7). Exonuclease-mediated resection at the break site is thought to produce 3′-ss tails that can anneal in reactions catalyzed by the RAD52 protein (see Figure 7, steps $a$ and $b$) (61, 62). The resulting annealed structure is likely to contain 3′-flap structures, and their processing and removal are catalyzed by Rad1-Rad10 and XPF-ERCC1 (steps $c$ and $d$) (57, 63–65). Genetic studies also indicate that the integration of linear DNA molecules into homologous genomic loci may involve SSA, as *S. cerevisiae rad1* or *rad10* mutants and *ERCC1⁻/⁻*-defective mammalian cells show defects in these integration events.

In addition to promoting DSB repair in somatic cells, homologous recombination is employed to generate genetic diversity during meiosis. Both XPF and ERCC1 are highly expressed in mouse testes, and ERCC1 is required for normal spermatogenesis and oogenesis, but no direct evidence of a role for XPF or ERCC1 in meiotic recombination has been obtained (66, 67). However, in *Drosophila melanogaster*, it is clear that *MEI-9*, which encodes the ortholog of XPF, is required for both NER and meiotic recombination (68). Indeed, *mei-9* mutants exhibit a 90% to 95% decrease in the number of crossover products and exhibit chromosomal nondisjunction. Given that crossover products are primarily generated by the resolution of HJ intermediates,

Figure 7

Single-strand annealing (SSA) pathway of recombination. DNA double-strand breaks (DSBs) generated within repetitive sequences can be repaired by the SSA pathway. Following the resection of the DSB ends, the RAD52 protein promotes the annealing of the complementary single-stranded sequences (steps $a$ and $b$). The endonuclease activity of XPF-ERCC1 may then trim off nonhomologous DNA tails (step $c$).
these observations may indicate that MEI-9 is involved in resolving HJs (68). It has been reported that MEI-9 has two different partners, MUS312 and ERCC1 (69). MUS312 appears to be unique to *Drosophila* and is required for the formation of crossover products, whereas defects in ERCC1 have only a small effect (69, 70). At present, however, there are no biochemical data to support the concept that MEI-9-MUS312 complex is able to resolve HJs.

### Relationship to Aging

Mice disrupted for *XPF* or *ERCC1*, but not other components of the NER pathway, exhibit severe defects typical of accelerated aging, including a greatly reduced lifespan (three weeks), cachexia, liver and kidney abnormalities, reduction of the number of hematopoietic progenitors, neuronal degeneration, and skin atrophy (71, 72). Similarly, a novel human progeroid syndrome, known as XFE, is caused by a mutation within *XPF* (73). In contrast to most individuals who carry hypomorphic *XPF* mutations that do not completely eliminate *XPF* function, the single XFE patient, so far classified, had a more severe *XPF* mutation and displayed symptoms of accelerated aging, such as weight loss, ataxia, cerebral atrophy, visual and hearing loss, anemia, and liver dysfunction (73, 74). Recently, the first patient with an *ERCC1* mutation was identified (75). The phenotype was similar to that seen with the *ERCC1⁻/⁻* mouse, as the patient exhibited severe developmental defects, including cerebral, ocular, and skeletal abnormalities, which are typical of the cerebro-oculo-facial-skeletal syndrome. The individual died just one year after birth. Taken together, these observations indicate the critical role played by *XPF-ERCC1* in human development and aging.

A clue to the role of *XPF-ERCC1* in aging was recently obtained when it was shown that *ERCC1⁻/⁻* mice, similar to old mice, displayed a decrease in the levels of insulin-like growth factor 1 (IGF-1) (73). The insulin/IGF-1 pathway is one of the major regulators of aging, and it is currently believed that inactivation of this pathway promotes longevity. In response to DNA damage, the insulin/IGF-1 pathway is inactivated to inhibit cell growth and proliferation and to minimize the formation of DNA lesions. However, despite this protective response, it is thought that the persistent accumulation of DNA damage could induce accelerated aging (73).

The exquisite sensitivity to cross-linking agents seen with *XPF⁻/⁻* and *ERCC1⁻/⁻* mice indicates that interstrand cross-links (ICLs) may be the primary DNA lesions responsible for the accelerated aging phenotype. However, other factors, such as the accumulation of ROS, generated by cellular metabolism, have also been linked to aging (76). ROS can fragment the DNA backbone or DNA bases and induce the formation of single- and double-strand breaks. Genetic analyses in yeast have suggested a role for Rad1-Rad10 in the removal of 3′-flap structures that are generated during the repair of oxidative DNA lesions, and a similar role has been proposed for *XPF-ERCC1* (77, 78).

In addition to ROS damage, telomere shortening has also been associated with aging (79). Studies in mammalian cells have implicated *XPF-ERCC1* in telomere maintenance (80). In particular, *ERCC1⁻/⁻* cells were shown to have telomere deletions and to accumulate double-minute chromosomes containing telomeric DNA. These observations imply that *XPF-ERCC1* might be involved in the maintenance of telomere length by removing telomeric recombination intermediates (T-loops) that could lead to telomere deletions (80). In agreement with such a proposal, *XPF-ERCC1* was found in a complex with TRF2, a protein that is involved in telomeric end protection (80, 81). Moreover, the inhibition or overexpression of TRF2 was shown to induce extensive telomere loss by mechanisms dependent on *XPF-ERCC1* (80, 82).

HRR: homologous recombination repair  
ICL: interstrand cross-link  
MUS81 COMPLEXES

Mus81 was first identified on the basis of its interaction with the S. *cerevisiae* recombination/repair protein Rad54 and the S. *pombe* replication checkpoint kinase Cds1 (10, 83). A human ortholog of MUS81 was subsequently found using homology searches (11). Analysis of the MUS81 sequence indicates that, in contrast to other members of the XPF-MUS81 family, the two HhH motifs are separated, with one found at the N terminus and the other at the C terminus (Figure 1). Given that the HhH motifs cooperate to promote DNA binding, it is likely that these motifs are in close proximity in the three-dimensional structure of the protein. The C-terminal region of MUS81 also contains a pseudo-HhH motif, which has acquired a small insertion in the hairpin compared to classical HhH motifs (see Figure 4 and compare amino acids 521–530 of MUS81 with amino acids 272–275 of ERCC1) (31). It is not known whether this pseudo-HhH is functional. The partners of MUS81, EME1, EME2, and their yeast counterparts also have one classical and one pseudo-HhH motif containing a hairpin insertion in their C-terminal regions (see Figure 4 and compare amino acids 541–556 of EME1 and amino acids 339–352 of EME2 with amino acids 272–275 of ERCC1).

that *P. furiosus* Hef and *S. solfataricus* XPF have substrate specificities more similar to the MUS81 complexes than to XPF-ERCC1 (22, 24). Both *P. furiosus* Hef and *S. solfataricus* XPF cleave 3′-flap and replication fork structures in preference to splayed-arm DNA. Indeed, the splayed-arm substrate is cleaved by *S. solfataricus* XPF approximately 10-fold less efficiently than the 3′-flap substrate (Figure 5) (22, 24). *S. solfataricus* XPF also cleaves D-loop substrates and nicked HJs quite efficiently (24).

Recent reports have shown that recombination intermediates, such as D-loop structures and nicked HJs, are cleaved by yeast Mus81-Eme1 (*S. pombe*) and Mus81-Mms4 (*S. cerevisiae*) and that the efficiencies of cleavage approach those observed with 3′-flap and replication fork structures (Figure 5) (85, 86). In contrast, intact HJs served as poor substrates for recombinant Mus81-Eme1, Mus81-Mms4, or human MUS81-EME1 and MUS81-EME2 (Figure 5) (7, 12, 14, 87, 88). Cleavage occurs asymmetrically and by a mechanism distinct from that exhibited by the RuvC-like HJ resolvases. However, it was recently observed that His-tagged Mus81-Eme1 complexes, which appear to purify in a hetero-tetrameric form, exhibit enhanced cleavage reactions with intact HJs compared to previous protein preparations (89).

Human MUS81-EME1 and MUS81-EME2, as well as their yeast and archaeal orthologs, appear to cut 3′-flap and replication fork substrates by similar mechanisms. With *P. furiosus* Hef and *A. pernix* XPF, the HhH motifs are thought to form a bridge between the duplex arms of a fork structure, allowing the ERCC4 nuclease domain to bind and cleave near the center of the fork (25). Consistent with this proposal, *S. cerevisiae* Mus81-Mms4 recognizes the 5′ terminus of the DNA strand that lies downstream of the branch point in a 3′-flap structure and introduces nicks into the duplex region adjacent to the 3′-flap (84). Interestingly, 3′-flap structures in which the 5′ termini are positioned more than 5 nucleotides away from the branch point

Biochemical Activities

As mentioned above, HhH motifs are important for the DNA-binding activities of XPF/MUS81 family members (25). Differences in these HhH motifs could therefore determine the substrate specificity of each protein complex. Indeed, in contrast to Rad1-Rad10 and XPF-ERCC1, MUS81-EME1 and MUS81-EME2 (and their yeast orthologs) cleave 3′-flap and replication fork substrates significantly more efficiently than splayed-arm structures (Figure 5) (12, 14, 84).

In vitro studies with the archaeal members of the XPF/MUS81 family have shown

cannot be cleaved (84). *S. cerevisiae* Mus81-Mms4 and *S. pombe* Mus81-Eme1 introduce nicks at sites located 3 to 7 nucleotides away from the junction point (88). Similar results have also been observed with *P. furiosus* Hef, human MUS81-EME1, and MUS81-EME2 (22) (A. Ciccia and S.C. West, unpublished data). Taken together, these experiments indicate that MUS81 in complex with its partners has an evolutionarily conserved structure-dependent mechanism of cleavage.

### Meiotic Recombination

Meiotic recombination is initiated by the introduction of a Spo11-induced double-stranded break that is subsequently resected to give rise to 3′-extended ssDNA tails (90). Subsequent events are similar to those that occur during mitotic recombination and include the formation of a RAD51 nucleoprotein filament, which initiates a search for homologous sequences (step b). Homologous alignment and strand invasion then lead to the formation of recombination intermediates containing HJs (steps c and d). It is thought that eukaryotic cells may possess several ways to resolve recombination intermediates, one of which is mediated by Mus81-Eme1 (Figure 8).

*S. pombe mus81* or *eme1* mutants exhibit very pronounced meiotic defects consistent with problems with the processing of recombination intermediates. In agreement with this finding, the meiotic phenotype can be partially rescued by ectopic expression of the RusA HJ resolvase (7). These studies led to the suggestion that *S. pombe* Mus81-Eme1 is an HJ resolvase (7). However, more recent studies, showing that Mus81-Eme1 prefers to cleave D-loops and nicked HJs, indicate that D-loops and nicked HJs might more truly reflect the DNA species that are cleaved during meiotic recombination (Figure 8) (85, 86). It has been suggested that Mus81-Eme1 could nick both the D-loop structure formed by strand invasion and the unligated HJ

generated after the capture of the second end (86, 91). By this mechanism, Mus81-Eme1 would generate crossover products consistent with the observed phenotype indicating, especially in *S. pombe*, that *mus81* mutants have defects in generating crossover products, whereas noncrossovers occur with normal frequency (86, 92). Less severe defects in crossover formation are observed in *S. cerevisiae mus81* mutants, suggesting that Mus81-Eme1 might be particularly adapted to the resolution of recombination intermediates in *S. pombe*. In this regard, it is interesting to note that recent studies of this organism suggest that meiotic recombination proceeds through a single HJ intermediate and that such structures have been shown to accumulate in *S. pombe mus81* cells (93).

In *S. cerevisiae*, the mild crossover defects of *mus81* and *mms4* cells indicate the presence of alternative Mus81-independent pathways for the generation of crossovers (94, 95). Genetic data suggest that meiotic crossover products can be generated in *S. cerevisiae* either by the Mus81 pathway or by a mechanism that involves the Msh4-Msh5 proteins, as indicated by the observation that *mms4 msh5* and *mus81 msh5* mutants exhibit more pronounced meiotic defects than any of the single mutants (94–96).

Given that *MUS81⁻/⁻* mice do not exhibit any obvious meiotic defects, it is unlikely that MUS81-EME1 is involved in meiotic recombination in vertebrates (97, 98). Whether MUS81-EME1 plays a role in mitotic recombination remains to be elucidated as one report has shown that RNA interference-mediated depletion of MUS81 in human somatic cells caused a twofold reduction in the frequency of mitotic recombination (99), whereas another showed that mouse *EME1⁻/⁻* ES cells do not display any recombination defects (100). One possibility is that MUS81-EME1, a classical RuvC-like resolvase activity, and junction-dissolution BLM-topoIII complex provide alternative means to resolve HJ intermediates (101, 102).

CPT: camptothecin

**Repair of Blocked or Collapsed Replication Forks**

The Mus81-Eme1 complex may be particularly important in somatic cells for the repair of stalled replication forks, as indicated by observations showing that yeast *mus81* mutants are sensitive to DNA-damaging agents that impair DNA replication progression, such as UV radiation, methylmethane sulfonate, hydroxyurea (HU), or camptothecin (CPT) (10, 83, 84, 87). In contrast, *mus81* mutants do not show defects in the repair of DSBs generated after IR treatment (10, 83).

![Diagram](attachment://diagram.png)

274 Ciccia • McDonald • West
The importance of restoring fork progression is indicated by observations showing that several mechanisms for fork repair exist. UV damage is known to induce the formation of ssDNA gaps opposite UV lesions (Figure 9, steps $a$ and $b$) (103, 104). These can be repaired either by specialized translesion synthesis polymerases, such as polη or polι (step $k$) or by recombinational repair (step $c$) (105). Additionally, recombinational repair mechanisms can be employed to repair the ssDNA gaps. For example, double HJs that form by recombination could be removed by the HJ dissolution activity of the BLM-topoIII (Figure 9, steps $g$ and $h$) or resolved by an HJ resolvase (steps $c-f$). Alternatively, fork repair and restoration could occur by the synthesis-dependent strand annealing (SDSA) pathways of recombination (steps $i-j$). In yeast, Mus81 is thought to be involved in the Srs2-dependent SDSA pathway for the repair of UV-induced ssDNA gaps, as suggested by the epistatic relationship between $srs2$ and $mus81$ after UV radiation (106). Mus81-Eme1 (or Mus81-Mms4) could cleave 3′-flap structures that are generated when the displaced invading strand reanneals to the original DNA filament (step $i$) (107). According to this model, Mus81 and Eme1 (Mms4) are part of a pathway that functions as an alternative to the RecQ helicases Sgs1 or Rqh1, and consistent with this proposal, $mus81$ or $eme1$ ($mms4$) mutations are synthetically lethal in combination with $rqb1$

or $sgsl$ mutations in $S$. *pombe* or $S$. *cerevisiae*, respectively (87, 108).

Mus81 and the RecQ helicases could also be involved in alternative repair pathways, leading to the repair of replication forks blocked at replication fork barriers (RFBs). RFBs are sites of replication fork blockage, either accidental or programmed (109). In $S$. *pombe*, Rqh1 is required for the stabilization of the replication fork blocked at the RFB RTS1, as indicated by the formation of DSBs in the absence of Rqh1 (110). RecQ helicases could maintain fork integrity by resetting replication forks that have regressed at DNA replication blocks, therefore allowing DNA replication to restart when the block has been removed (111–113). In the absence of RecQ helicases, Mus81-Eme1 could be required to cleave blocked replication forks, as suggested in $S$. *pombe* (110). As a consequence of Mus81-Eme1 cleavage, DNA replication could restart after the DSBs are repaired by HRR. A similar role has been proposed for MUS81-EME1 in the cleavage of replication forks blocked at ICLs in mammalian cells (114).

In addition to playing a role in the repair of ssDNA gaps and blocked replication forks, Mus81 is important for the restart of collapsed replication forks, as indicated by the exquisite sensitivity of yeast $mus81$ mutants to camptothecin (84, 87, 115). CPT is a topoisomerase inhibitor that induces the formation of single-strand breaks

---

### Figure 8

Recombination pathways for the repair of a double-strand break. In meiosis, DNA breaks, introduced by Spo11-induced cleavage, are resected to produce 3′-ssDNA tails. RAD51 forms nucleoprotein filaments on the ssDNA (in reactions facilitated by RPA and RAD52) and promotes the invasion of one single 3′-ssDNA end into the homologous DNA duplex to form a D-loop (steps $a$ and $b$). The D-loop may then be stabilized by the RAD54 protein and polymerase-mediated DNA repair synthesis (step $b$). Extension of the D-loop allows capture of the second end of the resected DSB in reactions that may involve the annealing activity of RAD52 (step $c$). Some models for recombination invoke the formation of double Holliday junctions (HJs) (step $d$) that can be removed by the dissolution activity of RecQ helicases in combination with topoisomerase activity (steps $e$ and $f$) or be resolved (*brown arrows*) by a RuvC-like HJ resolvase (steps $g$ and $h$). These mechanisms of resolution can produce noncrossover or crossover recombinants, as indicated. Other models for recombination invoke Mus81-Eme1 as an HJ resolvase (*green arrows*). Alternatively, prior to HJ formation, the invading D-loop intermediate may be acted upon by Mus81-Eme1 to produce crossover recombinants (steps $j-l$).
that are subsequently converted to DSBs during replication. The collapsed replication fork could then be repaired by HRR. However, this role for Mus81 might be restricted to yeast as mammalian $MUS81^{-/-}$ cells exhibit little sensitivity to CPT (97).

**Tumorigenesis**

$MUS81^{-/-}$ mouse embryo fibroblasts and $MUS81^{-/-}$ human colon cancer HCT116 cell lines exhibit proliferation defects, owing to the activation of the checkpoint kinases CHK1 and CHK2 and subsequent delay of

![Diagram](attachment:diagram.png)

SCE Non-SCE
the cell cycle (97, 116). Consistent with the hypothesis that the checkpoint might be activated by spontaneous generation of DNA damage, $MUS81^{-/-}$ mouse embryo fibroblasts and $EME1^{-/-}$ ES cells accumulate chromosomal aberrations, including breaks, fusions, and triradials (97, 100). Chromosome abnormalities have also been observed in both $MUS81^{+/-}$ and $MUS81^{-/-}$ activated T cells or $MUS81^{+/-}$ and $MUS81^{-/-}$ HCT116 cell lines (98, 116). Therefore inactivation of a single copy of $MUS81$ appears to be sufficient to cause genomic instability (98).

$MUS81^{+/-}$ and $MUS81^{-/-}$ mice have been shown to develop tumors, in particular non-Hodgkin's lymphomas, during the first year of life (98). Contrarily, other studies have demonstrated that $MUS81^{+/-}$ and $MUS81^{-/-}$ mice are healthy and viable during the first 15 months of life (97), and the basis of this discrepancy is not clear.

The requirement of $MUS81$ for tumor avoidance is likely to reflect its role in normal DNA replication. During synthesis (S) phase, $MUS81$ accumulates in the nucleus and the nucleolus, where the repetitive rDNA is located (117), but replication blockage leads to its recruitment to sites of DNA damage (117). The sensitivity of $MUS81^{-/-}$ and $EME1^{-/-}$ cells to mitomycin C (MMC) and cisplatin, but not HU, UV, CPT, or IR, suggests that the

---

### FANCM-FAAP24

Recently, two new members of the XPF-MUS81 family, FANCM and FAAP24, were identified. FANCM, was initially designated HEF because of its similarity to the *P. furiosus* Hef protein (Figure 1). FANCM and *P. furiosus* Hef share the highest homology in their DEAH and ERCC4 domains. The DEAH helicase domain is typical of DNA/RNA helicases of SF2 and is characterized by the presence of seven helicase motifs (I-Ia-II-III-IV-V-VI) (25). Motifs I and II contain the conserved Walker A and B sequences typical of ATPases.

#### Biochemical Activities

The N-terminal DEAH domain of *P. furiosus* Hef has been shown to unwind replication fork structures and HJs in vitro (37). Also, the helicase domain of *P. furiosus* Hef appears to stimulate the nuclease activity of Hef’s ERCC4 domain on mobile replication fork substrates (37). It has been shown that the *P. furiosus* Hef helicase domain can regress forks with lagging strand gaps, thereby forming “chicken foot” or HJ structures, consistent

---

### Figure 9

Proposed mechanisms for replication fork restart after the fork encounters a lesion in the leading strand. DNA base lesions can block the extension of leading-strand DNA synthesis, resulting in the formation of extensive ssDNA regions at the replication fork, owing to the uncoupling of lagging from leading-strand synthesis (step $a$). Repriming of leading-strand synthesis downstream of the lesion leaves ssDNA gaps opposite the damage (step $b$). The ssDNA gaps can be repaired by homologous recombination repair (HRR), through RAD51-mediated strand invasion of the 3′ end of the ssDNA gap into the homologous region of the undamaged sister chromatid (step $c$). Strand invasion and DNA synthesis may be followed by double HJ formation or synthesis-dependent strand annealing (SDSA) (steps $d$ and $i$). Resolution of the HJs by an HJ resolvase (*red arrows*) can result in the formation of crossover products between sister chromatids, which are referred to as sister chromatid exchanges (SCEs) (steps $e$ and $f$). Alternatively, junctions can be dissolved by the BLM/TopoIII complex, which does not generate SCEs (steps $g$ and $h$). In SDSA, strand invasion can be followed by reannealing of the invading strand to the original ssDNA-gapped region, in reactions dependent on Srs2 (step $i$). If the invading strand has undergone DNA synthesis extending over the length of the ssDNA gap, 3′-flap structures are generated after the reannealing of the invading strand and can be cleaved by MUS81-EME1 (steps $i$ and $j$). In addition to HRR, translesion DNA polymerases, such as polη and polτ, can promote the repair of ssDNA gaps left opposite the DNA lesion after the repriming of the leading-strand synthesis (step $k$).with a role in the restoration of blocked replication forks (Figure 5) (37).

FANCM orthologs have not been found in fungi or invertebrates (Figure 1). However, proteins containing the DEAH domain of FANCM, but not the ERCC4 domain, such as *D. melanogaster* CG7922 and *S. cerevisiae* Mph1 proteins, can be identified in insects and in yeast (28, 36). The Mph1 protein has been shown to exhibit $3' \rightarrow 5'$ DNA helicase activity in vitro (118). To date, however, the presence of a classical DNA helicase activity has not been reported for human FANCM, although the protein appears to translocate along DNA (28). As expected, the DEAH domain of FANCM binds ssDNA and exhibits ATPase activity (36).

As noted above, the ERCC4 nuclease domain of FANCM appears inactive. Indeed, in human FANCM, the ERCC4 catalytic motif **ERKX<sub>3</sub>D** has diverged to **ERRX<sub>3</sub>E** (Figure 2) (28). Various amino acid substitutions are also present in the ERCC4 domain of FANCM in other vertebrate species. The C-terminal region of FANCM interacts specifically with FAAP24, the most recently identified member of the XPF/MUS81 family. To date, however, FANCM-FAAP24 is unique among XPF/MUS81 family members in that no nuclease activity has been detected. However, it has not as yet been possible to purify a full-length FAMCM-FAAP24 complex in sufficient amounts for biochemical study, and all experiments have been carried out with C-terminally truncated human FANCM (containing the ERCC4 domain and HhH motifs) in complex with FAAP24 (14).

The FANCM-FAAP24 complex has been shown to bind DNA in vitro, and like XPF-ERCC1, FANCM-FAAP24 binds preferentially to branched structures containing ssDNA regions, such as splayed-arm and $3'$-flap DNA structures (14). Biochemical analyses have shown that the ssDNA-binding affinity of FANCM-FAAP24 is similar to that shown by ERCC1 for ssDNA (14, 20), and it has been suggested that FAAP24 might be important for targeting FANCM to DNA structures containing ssDNA regions (14). Consistent with this proposal, FAAP24, like ERCC1, contains a classical GhG hairpin, which is known to be critical in ERCC1 for DNA interactions, whereas no classical GhG hairpins are present in FANCM (Figure 2) (compare amino acids 168–170 of FAAP24 with amino acids 1982–1984 and 2014–2016 of FANCM) (14, 34).

### Fanconi Anemia

FANCM was recently identified as a novel gene that is mutated in Fanconi anemia (FA) complementation group M (28). FA is a rare autosomal recessive and X-linked disorder that affects less than 1 in 100,000 people (119). The clinical features of FA include early childhood skeletal abnormalities (thumb, arm, spine, and hip abnormalities); cardiac, gastrointestinal, and renal dysfunctions; and abnormal skin pigmentation (also known as café au lait spots) (120). At an early age, individuals with FA develop pancytopenia owing to apoptosis of the hematopoietic progenitor cells. FA is also associated with an 800-fold increased risk of developing myelodysplasia and acute myeloid leukemia by age 14 (121). Moreover, the risk of developing solid tumors, such as head, neck, gynecological, and gastrointestinal squamous cell carcinomas, is highly increased. The average survival of FA patients is 16 years, with bone marrow failure and cancer being the most common causes of death. At the cellular level, the diagnostic test for FA is a hypersensitivity to cross-linking agents, such as MMC, diepoxybutane, and cisplatin (122). After treatment with cross-linking agents, FA cells show chromosome breaks and the formation of multiple radial structures (120). These defects are attributed to the inability of FA cells to properly repair DSBs generated following treatment with cross-linking agents (123).

FA is characterized by genetic heterogeneity and classified into 13 different complementation groups (FA-A, B, C, D1, D2, E, F, G, I, J, L, M, and N) (123–126).The most common genes mutated in FA patients are $FANCA (60\%)$, $FANCC (15\%)$, and $FANCG (10\%)$ (124). Mutations in $FANCD1$ or $FANCD2$ are the next most common ($5\%$ each), whereas defects in the other FA genes are rare (124). To date, only two patients have been identified with $FANCM$ mutations (28). The FA-M patients possess a nonsense mutation in exon 13 and a deletion of exon 15 of the $FANCM$ gene. So far, no individuals have been identified with mutations in $FAAP24$.

$FANCM$ and $FAAP24$ are part of the FA core complex (14, 28, 36). Eight additional FA proteins ($FANCA$, $FANCB$, $FANCC$, $FANCE$, $FANCF$, $FANCG$, $FANCL$, and $FAAP100$) are part of the FA core complex (124). Loss of FA proteins, such as $FANCA$, $FANCG$, or $FANCM$, results in the instability of the FA core complex (36, 127). The FA core complex is thought to form by the sequential assembly of subcomplexes (128). It is possible that $FANCM$ is an adaptor around which the $FANCA$-$FANCG$ and $FANCB$-$FANCL$-$FAAP100$ complexes may assemble (127–129). Indeed, the absence of $FANCM$ results in decreased protein levels of $FANCA$, $FANCG$, and $FANCL$ and no complex formation between these proteins (28, 128). Assembly of this complex is important for the stability of both $FANCM$ and $FAAP24$, as indicated by the reduced $FANCM$ and $FAAP24$ protein levels in cell lines defective for $FANCB$ (14).

The FA core complex is required for the monoubiquitylation of the $FANCD2$-$FANCI$ complex after DNA damage (125, 130). Loss of any of the proteins of the FA core complex results in defects in monoubiquitylation (14, 28, 129–133). The FA core component $FANCL$ is thought to be the E3 ligase responsible for $FANCD2$ monoubiquitylation, although direct evidence of in vitro monoubiquitylation of $FANCD2$ by $FANCL$ is currently lacking (131). $FANCD2$ monoubiquitylation is believed to be the signal for the activation of the FA pathway (130). This modification is induced by a variety of DNA-damaging agents, such as UV radiation, IR, HU, and cross-linking agents, and it also occurs during normal S phase of the cell cycle (130, 133, 134). Following monoubiquitylation, the $FANCD2$-$FANCI$ complex is thought to relocalize to chromatin by a mechanism that is dependent upon $FANCM$ (36, 125, 130, 135).

There are interesting links between FA, recombinational repair, and inheritable breast cancers. The monoubiquitylated $FANCD2$-$FANCI$ complex colocalizes to nuclear foci with $BRCA1$ and $BRCA2$ (mutations that lead to breast cancer), $RAD51$ recombinase, and the $MRE11$-$NBS1$-$RAD50$ complex (130, 136–138). Moreover, biallelic mutations in $BRCA2$ have been identified in FA patients of the complementation group D1 (134). Also, it was recently shown that mutations in the genes coding for the $BRCA1$-interacting protein $BRIP1$ and the $BRCA2$-interacting protein $PALB2$ are found in FA patients from complementation groups J and N, respectively (139–144). However, in contrast to the proteins of the FA core complex, $FANCD1$ ($BRCA2$), $FANCJ$ ($BRIP1$), and $FANCN$ ($PALB2$) appear to play a late role in the FA pathway, as indicated by observations showing that their absence does not affect $FANCD2$ monoubiquitylation.

### Interstrand Cross-Link Repair

Despite being activated by any condition of DNA stress, the FA pathway is required primarily for ICL repair, as indicated by the exquisite sensitivity of FA cells to cross-linking agents, but not to UV radiation, HU, and IR (120). Like other FA proteins, $FANCM$ and $FAAP24$ are important for ICL repair, as indicated by the hypersensitivity to DNA cross-linking agents and chromosomal instability of cell lines deficient for $FANCM$ and $FAAP24$ (14, 28, 36). The observation that chicken cell lines with a C-terminal truncation of $FANCM$ are only mildly sensitive to cross-linking agents might indicate that the nuclease domain of $FANCM$ is partly dispensable for ICL repair in these cells (36).
ANRV345-BI77-12 ARI 6 May 2008 14:29

Access provided by University of California - San Francisco UCSF on 12/01/14. For personal use only.

![Diagram](attachment:diagram.png)

Recombinational repair

280 Ciccia • McDonald • West
In mammalian cells, ICL repair is activated in S phase when replication forks stall (145, 146). It has been proposed that FANCM-FAAP24 could translocate along DNA and target the FA core complex to blocked replication forks (see Figure 10, step $a$) (14, 28). On the basis of its biochemical properties, the FANCM-FAAP24 complex could bind replication fork intermediates containing ssDNA regions, which are likely to be generated by replication fork blockage (14, 104, 147). The FANCM-FAAP24 complex could then act as a signal transducer and induce the monoubiquitylation of FANCD2-FANCI by the FA core complex in a manner dependent on the checkpoint kinase ATR (step $b$). It is also known that FANCD2 and FANCI can be phosphorylated in an ATR-dependent manner (see Figure 10, step $c$) (125, 148) and that FANCM is hyperphosphorylated in response to DNA damage in reactions which may involve the ATR kinase (step $d$) (28).

To begin the repair of replication forks blocked at ICLs, DSBs may be generated by cleavage of the replication fork upstream of the lesion (see Figure 10, step $f$) (145, 146). The endonuclease responsible for the formation of ICL-induced DSBs was initially thought to be XPF-ERCC1, owing to the ability of XPF-ERCC1 to cleave branched DNA structures containing ICLs (149). However, it now appears that ICL-induced DSBs are generated in the absence of XPF or ERCC1 (145, 150). More likely, MUS81-EME1 activity is required for DSB formation after S phase arrest at ICLs (step $f$) (114).

ICL repair requires the introduction of a second incision known as ICL unhooking, and this cleavage reaction is catalyzed by XPF-ERCC1 (see Figure 10, step $g$) (149–151). Consistent with this, cell lines defective for $XPF$ or $ERCC1$ are hypersensitive to cross-linking agents and defective in the repair of ICL-induced DSBs (146, 150, 152). Moreover, studies in mammalian cells have shown that XPF colocalizes with FANCA and FANCD2 after treatment with cross-linking agents (153, 154). Thus, both XPF-ERCC1 and MUS81-EME1 complexes are required to bring about the incision reactions necessary for cross-link repair.

### CONCLUSIONS

The XPF/MUS81 family of proteins possesses a domain structure that permits them to play essential roles in a variety of DNA repair processes, including NER, homologous recombination, replication fork restart, and interstrand cross-link repair. The vertebrate proteins function as heterodimers with one active and one inactive subunit. The active

---

**Figure 10**

Speculative model for the involvement of Fanconi anemia (FA) proteins in interstrand cross-link (ICL) repair. Cross-links can block the progression of replication forks and generate ssDNA regions, which could be recognized by the FANCM and Fanconi anemia-associated protein 24 kDa (FAAP24) complex (step $a$). The recognition of ICL-blocked replication forks by FANCM-FAAP24 may be important for the activation of the replication checkpoint response regulated by ATR (step $b$). Following ATR activation, the FANCD2-FANCI complex is monoubiquitylated and phosphorylated (steps $c$ and $d$) and relocates to chromatin (step $e$). Replication forks blocked at ICLs are cleaved by MUS81 in complex with either EME1 or EME2 (step $f$). A second incision can then be introduced by XPF-ERCC1 in order to unhook the ICL (step $g$). Sites of cleavage are indicated by red arrows. The ssDNA gap generated by the unhooking reaction can be repaired by the translesion polymerases REV1 and pol $\zeta$ (step $h$). Following removal of the residual cross-link adduct by nucleotide excision repair or other mechanisms, the DSB is repaired by homologous recombination repair, leading to restoration of the replication fork (steps $i$ and $j$). Abbreviations: ATR, ATM and Rad3-related protein kinase; BLAP75, Bloom's syndrome-associated protein 75 kDa; BLM, Bloom's syndrome protein; BRCA1 and BRAC2, the breast cancer tumor suppressors 1 and 2; CORE, Fanconi anemia core complex; FAAP100, Fanconi anemia-associated protein 100 kDa; A, B, D2, E, F, G, J, I, L, M, and N, Fanconi anemia proteins; M/R/N, MRE11-RAD50-NBS1 complex; P, phosphorylation; Ub, ubiquitylation.
subunit generally contains an active ERCC4 nuclease domain and an (HhH)₂ domain. The noncatalytic subunits contain the same modular structure, but their ERCC4 domains are divergent. These are likely to help determine specific interactions with defined DNA structures. Some members of the family also contain SF2 helicase domains, or degenerate derivatives thereof, although as yet none of the proteins have been shown to exhibit DNA helicase activity. The precise role(s) of these domains has yet to be established.

Defects in XPF/MUS81 family proteins are associated with a variety of human diseases, such as xeroderma pigmentosum (e.g., XPF-ERCC1), premature aging (e.g., XPF-ERCC1), tumorigenesis (e.g., MUS81-EME1), or Fanconi anemia (e.g., FANCM), reflecting their critically important roles in DNA repair and maintenance of the genome. As such, these enzymes represent an important class of repair proteins that exhibit diverse cellular functions while maintaining a common domain structure.

---

### SUMMARY POINTS

1. Proteins belonging to the XPF/MUS81 family are found in eukaryal and archaeal kingdoms but not in prokaryotes.
2. They exist as homodimers (archaea) or heterodimers (eukaryotes). Only one subunit needs to be catalytically active in order to exhibit nuclease activity.
3. All members of the family contain a characteristic core consisting of an ERCC4 nuclease domain and a tandem helix-hairpin-helix (HhH)₂ domain. This is the smallest functional unit required to both target and cleave DNA junction structures. The noncatalytic subunits retain the same modular structure but have diverged such that they lack endonuclease activity.
4. Some members of the XPF/MUS81 family contain a functional (FANCM) or degenerate (XPF) SF2 helicase domain.
5. All XPF/MUS81 family members play critical roles in DNA repair, in particular NER, HRR, and ICL repair.
6. All known vertebrate XPF/MUS81 family members (XPF-ERCC1, MUS81-EME1, and FANCM-FAAP24) are important for the recognition or repair of interstrand DNA cross-links.
7. Unlike other proteins of the family, the FANCM-FAAP24 complex does not appear to exhibit nuclease activity.
8. In humans, mutations in the XPF/MUS81 family of proteins lead to diseases such as xeroderma pigmentosum, Fanconi anemia, premature aging, or tumorigenesis.

---

### FUTURE ISSUES

1. Do other XPF/MUS81 family members remain to be identified?
2. Define the functional role and specificity of mammalian MUS81-EME2 in relation to MUS81-EME1.

3. How are XPF/MUS81 family complexes targeted to the specific DNA substrates, and what is the structural basis for DNA junction recognition?

4. Are the XPF/MUS81 proteins incorporated into larger macromolecular machines that promote DNA repair, or do they simply cooperate with other components of the repair machinery?

**DISCLOSURE STATEMENT**

The authors are not aware of any biases that might be perceived as affecting the objectivity of this review.

**ACKNOWLEDGMENTS**

We thank Cancer Research UK, the EU DNA Repair Network, the Breast Cancer Campaign, and the Louis-Jeantet Foundation for their support.

**LITERATURE CITED**

1. Liu Y, West SC. 2004. *Nat. Rev. Mol. Cell Biol.* 5:937–44
2. Lilley DMJ, White MF. 2001. *Nat. Rev. Mol. Cell Biol.* 2:433–43
3. Lieber MR. 1997. *BioEssays* 19:233–40
4. O'Donovan A, Davies AA, Moggs JG, West SC, Wood RD. 1994. *Nature* 371:432–35
5. White MF. 2003. *Biochem. Soc. Trans.* 31:690–93
6. Sijbers AM, Delaat WL, Ariza RR, Biggerstaff M, Wei YF, et al. 1996. *Cell* 86:811–22
7. Boddy MN, Gaillard PHL, McDonald WH, Shanahan P, Yates JR, Russell P. 2001. *Cell* 107:537–48
8. Nishino T, Komori K, Ishino Y, Morikawa K. 2003. *Structure* 11:445–57
9. Newman M, Murray-Rust J, Lally J, Rudolf J, Fadden A, et al. 2005. *EMBO J.* 24:895–905
10. Interthal H, Heyer WD. 2000. *Mol. Gen. Genet.* 263:812–27
11. Chen XB, Melchionna R, Denis CM, Gaillard PHL, Blasina A, et al. 2001. *Mol. Cell* 8:1117–27
12. Ciccia A, Constantinou A, West SC. 2003. *J. Biol. Chem.* 278:25172–78
13. Ögrünc M, Sancar A. 2003. *J. Biol. Chem.* 278:21715–20
14. Ciccia A, Ling C, Coulthard R, Yan Z, Xue Y, et al. 2007. *Mol. Cell* 25:331–43
15. Bailly V, Sommers CH, Sung P, Prakash L, Prakash S. 1992. *Proc. Natl. Acad. Sci. USA* 89:8273–77
16. Bardwell L, Cooper AJ, Friedberg EC. 1992. *Mol. Cell. Biol.* 12:3041–49
17. Carr AM, Schmidt H, Kirchhoff S, Muriel WJ, Sheldrick KS, et al. 1994. *Mol. Cell. Biol.* 14:2029–40
18. Kaliraman V, Mullen JR, Fricke WM, Bastin-Shanower SA, Brill SJ. 2001. *Genes Dev.* 15:2730–40
19. Shao X, Grishin NV. 2000. *Nucleic Acids Res.* 28:2643–50
20. Tsodikov OV, Enzlin JH, Schärer OD, Ellenberger T. 2005. *Proc. Natl. Acad. Sci. USA* 102:11236–41
21. Roberts JA, Bell SD, White MF. 2003. *Mol. Microbiol.* 48:361–71

www.annualreviews.org • The XPF/MUS81 Family of Proteins 283

22. Komori K, Fujikane R, Shinagawa H, Ishino Y. 2002. *Genes Genet. Syst.* 77:227–41
23. Nishino T, Komori K, Tsuchiya D, Ishino Y, Morikawa K. 2005. *Structure* 13:143–53
24. Roberts JA, White MF. 2005. *J. Biol. Chem.* 280:5924–28
25. Nishino T, Komori K, Ishino Y, Morikawa K. 2005. *Structure* 13:1183–92
26. Pingoud A, Fuxreiter M, Pinfoud V, Wende W. 2005. *Cell. Mol. Life Sci.* 62:685–707
27. Enzlin JH, Scharer OD. 2002. *EMBO J.* 21:2045–53
28. Meetei AR, Medhurst AL, Ling C, Xue YT, Singh TR, et al. 2005. *Nat. Genet.* 37:958–63
29. Li L, Elledge SJ, Peterson CA, Bales ES, Legerski RJ. 1994. *Proc. Natl. Acad. Sci. USA* 91:5012–16
30. Thayer MM, Ahern H, Xing D, Cunningham RP, Tainer JA. 1995. *EMBO J.* 14:4108–20
31. Doherty AJ, Serpell LC, Ponting CP. 1996. *Nucleic Acids Res.* 24:2488–97
32. Rafferty JB, Ingleson SM, Hargreaves D, Artymiuk PJ, Sharples GJ, et al. 1998. *J. Mol. Biol.* 278:105–16
33. Singh S, Folkers GE, Bonvin AM, Boelens R, Wechselberger R, et al. 2002. *EMBO J.* 21:6257–66
34. Tripsianes K, Folkers G, Ab E, Das D, Odijk H, et al. 2005. *Structure* 13:1849–58
35. De Laat WL, Appeldoorn E, Jaspers NGJ, Hoeijmakers JHJ. 1998. *J. Biol. Chem.* 273:7835–42
36. Mosedale G, Niedzwiedz W, Alpi A, Perrina F, Pereira-Leal JB, et al. 2005. *Nat. Struct. Mol. Biol.* 12:763–71
37. Komori K, Hidaka M, Horiuchi T, Fujikane R, Shinagawa H, Ishino Y. 2004. *J. Biol. Chem.* 279:53175–85
38. Caruthers JM, McKay DB. 2002. *Curr. Opin. Struct. Biol.* 12:123–33
39. Aravind L, Makarova KS, Koonin EV. 2000. *Nucleic Acids Res.* 28:3417–32
40. Sgouros J, Gaillard PHL, Wood RD. 1999. *Trends Biochem. Sci.* 24:95–97
41. Sung P, Reynolds P, Prakash L, Prakash S. 1993. *J. Biol. Chem.* 268:26391–99
42. Tomkinson AE, Bardwell AJ, Bardwell L, Tappe NJ, Friedberg EC. 1993. *Nature* 362:860–62
43. Bardwell AJ, Bardwell L, Johnson DK, Friedberg EC. 1993. *Mol. Microbiol.* 8:1177–88
44. Bardwell AJ, Bardwell L, Tomkinson AE, Friedberg EC. 1994. *Science* 265:2082–85
45. Davies AA, Friedberg EC, Tomkinson AE, Wood RD, West SC. 1995. *J. Biol. Chem.* 270:24638–41
46. Rodriguez K, Wang ZG, Friedberg EC, Tomkinson AE. 1996. *J. Biol. Chem.* 271:20551–58
47. Park CH, Bessho T, Matsunaga T, Sancar A. 1995. *J. Biol. Chem.* 270:22657–60
48. Aravind L, Walker DR, Koonin EV. 1999. *Nucleic Acids Res.* 27:1223–42
49. Reynolds RJ, Love JD, Friedberg EC. 1981. *J. Bacteriol.* 147:705–8
50. Westerveld A, Hoeijmakers JHJ, van Duin M, de Wit J, Odijk H, et al. 1984. *Nature* 310:425–29
51. Kondo S, Mamada A, Miyamoto C, Keong CH, Satoh Y, Fujiwara Y. 1989. *Photo-dermatology* 6:89–95
52. Hoeijmakers JH. 2001. *Nature* 411:366–74
53. Cleaver JE. 2005. *Nat. Rev. Cancer* 7:564–73
54. Svejstrup JQ. 2002. *Nat. Rev. Mol. Cell Biol.* 3:21–29
55. Bregman DB, Halaban R, Van Gool AJ, Henning KA, Friedberg EC, Warren SL. 1996. *Proc. Natl. Acad. Sci. USA* 93:11586–90
56. Lehmann AR. 2003. *Biochimie* 85:1101–11
57. Sargent RG, Brenneman MA, Wilson JH. 1997. *Mol. Cell. Biol.* 17:267–77

58. Schiestl RH, Prakash S. 1988. *Mol. Cell. Biol.* 8:3619–26  
59. Paques F, Haber JE. 1999. *Microbiol. Mol. Biol. Rev.* 63:349–404  
60. McGlynn P, Lloyd RG. 2002. *Nat. Rev. Mol. Cell Biol.* 3:859–70  
61. Mortensen UH, Bendixen C, Sunjevaric I, Rothstein R. 1996. *Proc. Natl. Acad. Sci. USA* 93:10729–34  
62. Van Dyck E, Stasiak AZ, Stasiak A, West SC. 2001. *EMBO Rep.* 2:905–9  
63. Adair GR, Rolig RL, Moore-Faver D, Zabelshansky M, Wilson JH, Nairn RS. 2000. *EMBO J.* 19:5552–61  
64. Fishman-Lobell J, Haber JE. 1992. *Science* 258:480–84  
65. Ivanov EL, Haber JE. 1995. *Mol. Cell. Biol.* 15:2245–51  
66. Hsia KT, Millar MR, King S, Selfridge J, Redhead NJ, et al. 2003. *Development* 130:369–78  
67. Shannon M, Lamerdin JE, Richardson L, McCutchen-Maloney SL, Hwang MH, et al. 1999. *Genomics* 62:427–35  
68. Yildiz O, Kearney H, Kramer BC, Sekelsky JJ. 2004. *Genetics* 167:263–73  
69. Yildiz O, Majumder S, Kramer B, Sekelsky JJ. 2002. *Mol. Cell* 10:1503–9  
70. Radford SJ, Goley E, Baxter K, McMahan S, Sekelsky J. 2005. *Genetics* 170:1737–45  
71. Prasher JM, Lalai AS, Heijmans-Antonissen C, Ploemacher RE, Hoeijmakers JH, et al. 2005. *EMBO J.* 24:861–71  
72. Weeda G, Donker I, de Wit J, Morreau H, Janssens R, et al. 1997. *Curr. Biol.* 7:427–39  
73. Niedernhofer LJ, Garinis GA, Raams A, Lalai AS, Robinson AR, et al. 2006. *Nature* 444:1038–43  
74. Matsumura Y, Nishigori C, Yagi T, Imamura S, Takebe H. 1998. *Hum. Mol. Genet.* 7:969–74  
75. Jaspers NG, Raams A, Silengo MC, Wijgers N, Niedernhofer LJ, et al. 2007. *Am. J. Hum. Genet.* 80:457–66  
76. Lombard DB, Chua KF, Mostoslavsky R, Franco S, Gostissa M, Alt FW. 2005. *Cell* 120:497–512  
77. Guillet M, Boiteux S. 2002. *EMBO J.* 21:2833–41  
78. Guzder SN, Torres-Ramos C, Johnson RE, Haracska L, Prakash L, Prakash S. 2004. *Genes Dev.* 18:2283–91  
79. Stewart SA, Weinberg RA. 2006. *Annu. Rev. Cell Dev. Biol.* 22:531–57  
80. Zhu X-D, Niedernhofer L, Kuster B, Mann M, Hoeijmakers JHJ, de Lange T. 2003. *Mol. Cell* 12:1489–98  
81. van Steensel B, Smogorzewska A, de Lange T. 1998. *Cell* 92:401–13  
82. Munoz P, Blanco R, Flores JM, Blasco MA. 2005. *Nat. Genet.* 37:1063–71  
83. Boddy MN, Lopez-Girona A, Shanahan P, Interthal H, Heyer WD, Russell P. 2000. *Mol. Cell. Biol.* 20:8758–66  
84. Bastin-Shanower SA, Fricke WM, Mullen JR, Brill SJ. 2003. *Mol. Cell. Biol.* 23:3487–96  
85. Gaillard P-HL, Noguchi E, Shanahan P, Russell P. 2003. *Mol. Cell* 12:747–59  
86. Osman F, Dixon J, Doe CL, Whitby MC. 2003. *Mol. Cell* 12:761–74  
87. Doe CL, Ahn JS, Dixon J, Whitby MC. 2002. *J. Biol. Chem.* 277:32753–59  
88. Whitby MC, Osman F, Dixon J. 2003. *J. Biol. Chem.* 278:6928–35  
89. Gaskell LJ, Osman F, Gilbert RJ, Whitby MC. 2007. *EMBO J.* 26:1891–901  
90. Neale MJ, Keeney S. 2006. *Nature* 442:153–58  
91. Whitby MC. 2005. *Biochem. Soc. Trans.* 33:1451–55  
92. Smith GR, Boddy MN, Shanahan P, Russell P. 2003. *Genetics* 165:2289–93  
93. Cromie GA, Hyppa RW, Taylor AF, Zakharyevich K, Hunter N, Smith GR. 2006. *Cell* 127:1167–78

94. de los Santos T, Loidl J, Larkin B, Hollingsworth NM. 2001. *Genetics* 159:1511–25

95. de los Santos T, Hunter N, Lee C, Larkin B, Loidl J, Hollingsworth NM. 2003. *Genetics* 164:81–94

96. Hollingsworth NM, Brill SJ. 2004. *Genes Dev.* 18:117–25

97. Dendouga N, Gao H, Moechars D, Janicot M, Vialard J, McGowan CH. 2005. *Mol. Cell. Biol.* 25:7569–79

98. McPherson JP, Lemmers B, Chahwan R, Pamidi A, Migon E, et al. 2004. *Science* 304:1822–26

99. Blais V, Gao H, Elwell CA, Boddy MN, Gaillard PHL, et al. 2004. *Mol. Biol. Cell* 15:552–62

100. Abraham J, Lemmers B, Hande MP, Moynahan ME, Chahwan C, et al. 2003. *EMBO J.* 22:6137–47

101. Constantinou A, Chen X-B, McGowan CH, West SC. 2002. *EMBO J.* 21:5577–85

102. Wu L, Hickson ID. 2003. *Nature* 426:870–74

103. Lehmann AR. 1972. *J. Mol. Biol.* 66:319–37

104. Lopes M, Foiani M, Sogo JM. 2006. *Mol. Cell* 21:15–27

105. Lehmann AR. 2005. *FEBS Lett.* 579:873–76

106. Doe CL, Whitby MC. 2004. *Nucleic Acids Res.* 32:1480–91

107. Fabre F, Chan A, Heyer WD, Gangloff S. 2002. *Proc. Natl. Acad. Sci. USA* 99:16887–92

108. Mullen JR, Kaliraman V, Ibrahim SS, Brill SJ. 2001. *Genetics* 157:103–18

109. Lambert S, Carr AM. 2005. *Biochimie* 87:591–602

110. Ahn JS, Osman F, Whitby MC. 2005. *EMBO J.* 24:2011–23

111. Doe CL, Dixon J, Osman F, Whitby MC. 2000. *EMBO J.* 19:2751–62

112. Karow JK, Constantinou A, Li J-L, West SC, Hickson ID. 2000. *Proc. Natl. Acad. Sci. USA* 97:6504–8

113. Ralf C, Hickson ID, Wu L. 2006. *J. Biol. Chem.* 281:22839–46

114. Hanada K, Budzowska M, Modesti M, Maas A, Wyman C, et al. 2006. *EMBO J.* 25:4921–32

115. Kai M, Boddy MN, Russell P, Wang TS-F. 2005. *Genes Dev.* 19:919–32

116. Hiyama T, Katsura M, Yoshihara T, Ishida M, Kinomura A, et al. 2006. *Nucleic Acids Res.* 34:880–92

117. Gao H, Chen XB, McGowan CH. 2003. *Mol. Biol. Cell* 14:4826–34

118. Prakash R, Krejci L, Van Komen S, Schurer KA, Kramer W, Sung P. 2005. *J. Biol. Chem.* 280:7854–60

119. Fei PW, Yin JH, Wang WD. 2005. *Cell Cycle* 4:80–86

120. D’Andrea AD, Grompe M. 2003. *Nat. Rev. Cancer* 3:23–34

121. Alter BP. 2003. *Cancer* 97:425–40

122. Ishida R, Buchwald M. 1982. *Cancer Res.* 42:4000–6

123. Niedernhofer LJ, Lalai AS, Hoeijmakers JHJ. 2005. *Cell* 123:1191–98

124. Kennedy RD, D’Andrea AD. 2005. *Genes Dev.* 19:2925–40

125. Smogorzewska A, Matsuoka S, Vinciguerra P, McDonald ER, Hurov KE, et al. 2007. *Cell* 129:289–301

126. Patel KJ. 2007. *Nat. Genet.* 39:142–43

127. Garcia-Higuera I, Kuang Y, Denham J, D’Andrea AD. 2000. *Blood* 96:3224–30

128. Medhurst AL, Laghmani el H, Steltenpool J, Ferrer M, Fontaine C, et al. 2006. *Blood* 108:2072–80

129. Ling C, Ishiai M, Ali AM, Medhurst AL, Neveling K, et al. 2007. *EMBO J.* 24:2104–14

130. Garcia-Higuera I, Taniguchi T, Ganesan S, Meyn MS, Timmers C, et al. 2001. *Mol. Cell* 7:249–62

131. Meetei AR, de Winter JP, Medhurst AL, Wallisch M, Waisfisz Q, et al. 2003. *Nat. Genet.* 35:165–70

132. Meetei AR, Levitus M, Xue YT, Medhurst AL, Zwaan M, et al. 2004. *Nat. Genet.* 36:1219–24

133. Taniguchi T, D'Andrea AD. 2002. *Int. J. Hematol.* 75:123–28

134. Howlett NG, Taniguchi T, Olson S, Cox B, Waisfisz Q, et al. 2002. *Science* 297:606–9

135. Montes de Oca R, Andreassen PR, Margossian SP, Gregory RC, Taniguchi T, et al. 2005. *Blood* 105:1003–9

136. Hussain S, Wilson JB, Medhurst AL, Hejna J, Witt E, et al. 2004. *Hum. Mol. Genet.* 13:1241–48

137. Pichierri P, Averbeck D, Rosselli F. 2002. *Hum. Mol. Genet.* 11:2531–46

138. Wang XZ, Andreassen PR, D'Andrea AD. 2004. *Mol. Cell. Biol.* 24:5850–62

139. Levitus M, Waisfisz Q, Godthelp BC, de Vries Y, Hussain S, et al. 2005. *Nat. Genet.* 37:934–35

140. Levran O, Attwooll C, Henry RT, Milton KL, Neveling K, et al. 2005. *Nat. Genet.* 37:931–33

141. Litman R, Peng M, Jin Z, Zhang F, Zhang J, et al. 2005. *Cancer Cell* 8:255–65

142. Rahman N, Seal S, Thompson D, Kelly P, Renwick A, et al. 2007. *Nat. Genet.* 39:165–67

143. Reid S, Schindler D, Hanenberg H, Barker K, Hanks S, et al. 2007. *Nat. Genet.* 39:162–64

144. Xia B, Dorsman JC, Ameziane N, de Vries Y, Rooimans MA, et al. 2007. *Nat. Genet.* 39:159–61

145. De Silva IU, McHugh PJ, Clingen PH, Hartley JA. 2000. *Mol. Cell. Biol.* 20:7980–90

146. Rothfuss A, Grompe M. 2004. *Mol. Cell. Biol.* 24:123–34

147. Niedernhofer LJ. 2007. *Mol. Cell* 25:487–90

148. Pichierri P, Rosselli F. 2004. *EMBO J.* 23:1178–87

149. Kuraoka I, Kobertz WR, Ariza RR, Biggerstaff M, Essigmann JM, Wood RD. 2000. *J. Biol. Chem.* 275:26632–36

150. Niedernhofer LJ, Odijk H, Budzowska M, van Drunen E, Maas A, et al. 2004. *Mol. Cell. Biol.* 24:5776–87

151. Bessho T, Mu D, Sancar A. 1997. *Mol. Cell. Biol.* 17:6822–30

152. Collins AR. 1993. *Mutat. Res.* 293:99–118

153. Macé G, Bogliolo M, Guervilly JH, Dugas du Villard JA, Rosselli F. 2005. *Biochimie* 87:647–58

154. Sridharan D, Brown M, Lambert WC, McMahon LW, Lambert MW. 2003. *J. Cell. Sci.* 116:823–35

Contents

**Prefatory Chapters**

- Discovery of G Protein Signaling  
  Zvi Selinger . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Structural and Functional Relationships of the XPF/MUS81 Family of Proteins  
Alberto Ciccia, Neil McDonald, and Stephen C. West  . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Maturation of Iron-Sulfur Proteins in Eukaryotes: Mechanisms,  
Connected Processes, and Diseases  
Roland Lill and Ulrich Mühlenhoff ………………………………………… 669  

CFTR Function and Prospects for Therapy  
John R. Riordan ……………………………………………………………… 701  

Aging and Survival: The Genetics of Life Span Extension  
by Dietary Restriction  
William Mair and Andrew Dillin ………………………………………… 727  

Cellular Defenses against Superoxide and Hydrogen Peroxide  
James A. Imlay ……………………………………………………………… 755  

Toward a Control Theory Analysis of Aging  
Michael P. Murphy and Linda Partridge ………………………………… 777  

Indexes  

Cumulative Index of Contributing Authors, Volumes 73–77 …………… 799  
Cumulative Index of Chapter Titles, Volumes 73–77 …………………… 803  

Errata  

An online log of corrections to *Annual Review of Biochemistry* articles may be found  
at http://biochem.annualreviews.org/errata.shtml

ANNUAL REVIEWS
It's about time. Your time. It's time well spent.

New From Annual Reviews:
Annual Review of Statistics and Its Application
Volume 1 • Online January 2014 • http://statistics.annualreviews.org

Editor: Stephen E. Fienberg, Carnegie Mellon University
Associate Editors: Nancy Reid, University of Toronto
Stephen M. Stigler, University of Chicago

The Annual Review of Statistics and Its Application aims to inform statisticians and quantitative methodologists, as well as all scientists and users of statistics about major methodological advances and the computational tools that allow for their implementation. It will include developments in the field of statistics, including theoretical statistical underpinnings of new methodology, as well as developments in specific application domains such as biostatistics and bioinformatics, economics, machine learning, psychology, sociology, and aspects of the physical sciences.

Complimentary online access to the first volume will be available until January 2015.

TABLE OF CONTENTS:

- What Is Statistics? Stephen E. Fienberg
- A Systematic Statistical Approach to Evaluating Evidence from Observational Studies, David Madigan, Paul E. Stang, Jesse A. Berlin, Martijn Schuemie, J. Marc Overhage, Marc A. Suchard, Bill Dumouchel, Abraham G. Hartzema, Patrick B. Ryan
- The Role of Statistics in the Discovery of a Higgs Boson, David A. van Dyk
- Brain Imaging Analysis, F. DuBois Bowman
- Statistics and Climate, Peter Guttorp
- Climate Simulators and Climate Projections, Jonathan Rougier, Michael Goldstein
- Probabilistic Forecasting, Tilmann Gneiting, Matthias Katzfuss
- Bayesian Computational Tools, Christian P. Robert
- Bayesian Computation Via Markov Chain Monte Carlo, Radu V. Craiu, Jeffrey S. Rosenthal
- Build, Compute, Critique, Repeat: Data Analysis with Latent Variable Models, David M. Blei
- Structured Regularizers for High-Dimensional Problems: Statistical and Computational Issues, Martin J. Wainwright

- High-Dimensional Statistics with a View Toward Applications in Biology, Peter Bühlmann, Markus Kalisch, Lukas Meier
- Next-Generation Statistical Genetics: Modeling, Penalization, and Optimization in High-Dimensional Data, Kenneth Lange, Jeanette C. Papp, Janet S. Sinsheimer, Eric M. Sobel
- Breaking Bad: Two Decades of Life-Course Data Analysis in Criminology, Developmental Psychology, and Beyond, Elena A. Erosheva, Ross L. Matsueda, Donatello Telesca
- Event History Analysis, Niels Keiding
- Statistical Evaluation of Forensic DNA Profile Evidence, Christopher D. Steele, David J. Balding
- Using League Table Rankings in Public Policy Formation: Statistical Issues, Harvey Goldstein
- Statistical Ecology, Ruth King
- Estimating the Number of Species in Microbial Diversity Studies, John Bunge, Amy Willis, Fiona Walsh
- Dynamic Treatment Regimes, Bibhas Chakraborty, Susan A. Murphy
- Statistics and Related Topics in Single-Molecule Biophysics, Hong Qian, S.C. Kou
- Statistics and Quantitative Risk Management for Banking and Insurance, Paul Embrechts, Marius Hofert

Access this and all other Annual Reviews journals via your institution at www.annualreviews.org.
