
Appl Microbiol Biotechnol (2008) 80:637–645  
DOI 10.1007/s00253-008-1591-2  

---

**APPLIED GENETICS AND MOLECULAR BIOTECHNOLOGY**

**Genetic organization of the putative salbostatin biosynthetic gene cluster including the 2-epi-5-epi-valiolone synthase gene in *Streptomyces albus* ATCC 21838**

Woo Sik Choi · Xiumei Wu · Yong-Hoon Choeng · Taifo Mahmud · Byeong Chul Jeong · Sang Hee Lee · Yong Keun Chang · Chang-Joon Kim · Soon-Kwang Hong

Received: 24 March 2008 / Revised: 18 June 2008 / Accepted: 20 June 2008 / Published online: 22 July 2008  
© Springer-Verlag 2008

---

**Abstract** The cyclization of sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone, catalyzed by the 2-epi-5-epi-valiolone synthases, is the first committed step in the biosynthesis of C<sub>7</sub>N-aminocyclitol-containing natural products, such as validamycin and acarbose. These natural products contain in their structures a valienamine unit, which is important for their biological activity. The same core unit is also found in salbostatin, a related pseudodisaccharide that has strong trehalase inhibitory activity. In silico analysis of the putative biosynthetic gene cluster of salbostatin from *Streptomyces albus* ATCC 21838 revealed 20 open reading frames, including an *acbC* homolog gene (*salQ*), which is believed to be involved in the biosynthesis of salbostatin. The *salQ* gene was overexpressed in *Escherichia coli* and the catalytic function of the recombinant protein was confirmed to be a 2-epi-5-epi-valiolone synthase. In addition, SalF, SalL, SalM, SalN, SalO, and SalR were found to be homologous to AcbR, AcbM, AcbL, AcbN, AcbO, and AcbP from the acarbose pathway, respectively, which suggests that the biosynthesis of C<sub>7</sub>N-aminocyclitol moiety of salbostatin may be very similar to that of acarbose.

**Keywords** Salbostatin · 2-epi-5-epi-valiolone synthase · *Streptomyces albus* · C<sub>7</sub>N-aminocyclitol

---

**Electronic supplementary material** The online version of this article (doi:10.1007/s00253-008-1591-2) contains supplementary material, which is available to authorized users.

---

W. S. Choi · Y.-H. Choeng · B. C. Jeong · S. H. Lee · S.-K. Hong  
Department of Biological Science and Institute  
of Bioscience and Biotechnology, Myongji University,  
Yongin 449-728, South Korea  

X. Wu · T. Mahmud  
Genetics Program, College of Agricultural Sciences,  
Oregon State University,  
Corvallis, OR 97331-3507, USA  

T. Mahmud  
Department of Pharmaceutical Sciences, College of Pharmacy,  
Oregon State University,  
Corvallis, OR 97331-3507, USA  

Y. K. Chang  
Department of Chemical and Biomolecular Engineering,  
Korea Advanced Institute of Science and Technology,  
373-1, Kusong-dong, Yusung-gu,  
Taejon 305-701, South Korea  

C.-J. Kim  
Department of Chemical and Biological Engineering,  
Gyeongsang National University,  
900 Gajwadong,  
Jinju, Gyeongnam 660-701, South Korea  

S.-K. Hong (✉)  
Department of Biological Science, Myongji University,  
San 38-2, Nam-dong,  
Yongin, Gyeonggi-do 449-728, South Korea  
e-mail: skhong@mju.ac.kr  

---

**Introduction**

The C<sub>7</sub>N-aminocyclitol family of natural products, e.g., validamycin and acarbose, represents important microbial secondary metabolites that are used in agricultural and biomedical applications (Fig. 1). Validamycin A, a pseudo-trisaccharide, is an antifungal antibiotic produced by *Streptomyces hygroscopicus* var. *limoneus* (Iwasa et al. 1971), which has a strong trehalase inhibitory activity and has been used to control sheath blight disease of rice caused by *Rhizoctonia solani* (Shibata et al. 1981). The α-glucosidase inhibitor acarbose (a pseudotetrasaccharide) is a valuable member of the C<sub>7</sub>N-aminocyclitol family of natural products because it is currently used for the treatment of type II insulin-independent diabetes (Mahmud 2003). Both compounds share a common

Fig. 1 Chemical structures of salbostatin and related C<sub>7</sub>N-aminocyclitol compounds

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

unsaturated aminocyclitol core moiety, valienamine (Fig. 1), which is known to be responsible for their biological activities toward various glycosidic hydrolases (Dong et al. 2001). Thus, the valienamine moiety has been used as a scaffold in many synthetic projects to develop valuable drugs.

The biosynthetic pathway to the valienamine moiety had been studied in the validamycin and acarbose producers, and sedoheptulose 7-phosphate was deduced to be a precursor in both cases (Yu et al. 2005). Based on biochemical studies, the putative biosynthetic gene cluster for the acarbose was cloned from the producer *Actinoplanes* sp. 50/110 (Stratmann et al. 1999; Stockmann and Piepersberg 1992). The *acbC* gene encoding a protein with significant similarity to 3-dehydroquinate synthases (AroB proteins) was later shown to be a C<sub>7</sub>-cyclitol synthase using sedoheptulose 7-phosphate as its substrate (Stratmann et al. 1999). The cyclization product, 2-epi-5-epi-valiolone, was found to be the first precursor of the valienamine moiety of acarbose (Mahmud et al. 1999). The biosynthetic gene cluster for validamycin was also cloned using *acbC* as probe, and ValA (or VldA in Singh et al. 2006) was identified as another 2-epi-5-epi-valiolone synthase (C<sub>7</sub>-cyclitol synthase; Yu et al. 2005).

Salbostatin is a basic, nonreducing pseudodisaccharide that consists of valienamine moiety linked to 2-amino-1,5-dihydro-2-deoxy-D-glucitol (Fig. 1). It was isolated from the fermentation broth of the polyether antibiotic salinomycin producer, *Streptomyces albus* ATCC 21838 (Vertesy et al. 1994). Although it has a similar chemical structure as other pseudodisaccharides, salbostatin does not inhibit the enzymes' saccharase and maltase nor does it show antibiotic activity. However, it can effectively inhibit trehalase activity from porcine kidney because of the structural similarity to the naturally occurring trehalose. Recently, we cloned the putative biosynthetic gene cluster for salbostatin from a cosmid library of genomic DNA of S. *albus* ATCC 21838. Among 32 open reading frames (ORFs) assigned within the 36-kb DNA fragment, 20 ORFs (named as *salA* to *salT*) were predicted to be involved in salbostatin biosynthesis. Interestingly, five genes (*salB/C/D/E/F*) in the cluster were deduced to be responsible for trehalose biosynthesis, and their ability dictating trehalose biosynthesis was proved in *Streptomyces lividans* (Choeng et al. 2007). In this paper, the arrangement of the putative salbostatin biosynthetic gene cluster and functional identification of SalQ as the 2-epi-5-epi-valiolone synthase in the cluster is described.

### Materials and methods

#### Bacterial strains and plasmids

The *S. albus* ATCC 21838 (KCTC 9015) was obtained from Korean-Type Culture Collection, Korea (Vertesy et al. 1994). *Escherichia coli* strains and various cloning vectors used are listed in Table 1. *E. coli* was maintained and cultivated at 37°C in Luria–Bertani (LB) medium (Sambrook and Russell 2001). *Streptomyces* strains were maintained on R2YE agar medium and cultivated in R2YE liquid broth at 28°C (Kieser et al. 2000).

#### Enzymes and chemicals

Restriction endonucleases and other DNA-modifying enzymes were purchased from Takara Shuzo Inc. (Tokyo, Japan). Chemicals were purchased from Sigma Chemical Co. (St. Louis, MO, USA). Primers for polymerase chain reaction (PCR)-based cloning were purchased from DyneBio Inc. (Seoul, Korea).

#### DNA manipulations and transformations

DNA preparation and manipulation was performed using the methods of Sambrook and Russell (2001) and Kieser et al.

Table 1 Bacterial strains and plasmids used in this study

| Strains or plasmids | Properties or product                                                                                   | Source      |
|---------------------|-------------------------------------------------------------------------------------------------------|-------------|
| $E. coli$ strains   |                                                                                                       |             |
| DH5$\alpha$         | F$^{-}$, $\phi$80dlacZ$\Delta$M15, $\Delta$(lacZYA-argF)U169, deoR, recA1, endA1, hsdR17(rk$^{-}$, mk$^{+}$), phoA, supE44, $\lambda^{-}$, thi-1, gyrA96, relA1 | Stratagene |
| BL21(DE3)S          | F$^{-}$, ompT, hsdS$_{\mathrm{B}}$(rB$^{-}$mB$^{-}$), gal, dcm(DE3), [pLysS, Cam$^{\prime}$]                                              | Stratagene |
| SKH 37–20           | $E. coli$ DH5$\alpha$ carrying a Supercos 37-20 plasmid                                                | This study  |
| SQ1                | $E. coli$ DH5$\alpha$ carrying a pSQ1 plasmid                                                          | This study  |
| SQ2                | $E. coli$ DH5$\alpha$ carrying a pSQ2 plasmid                                                          | This study  |
| Plasmids            |                                                                                                       |             |
| pCR2.1 TOPO         | TA cloning vector with topoisomerase I, LacZ                                                          | Invitrogen  |
| pET-30a             | T7 promoter, expression of His-tagged fusion protein                                                   | Novagen     |
| pSQ1               | pCR2.1 TOPO vector carrying a 1.3-kb PCR product of salQ from Supercos 37-20                             | This study  |
| pSQ2               | pET-30a carrying a salQ gene, a 1.3-kb BamHI–HindIII fragment from pSQ1                                 | This study  |

(2000). DNA samples were digested with restriction endonucleases and ligated with T4 DNA ligase according to the supplier’s recommendations. Plasmid DNAs and their digested products were analyzed by horizontal agarose gel electrophoresis with the TAE buffer system. Competent cells of $E. coli$ for transformation were routinely prepared according to the frozen storage protocol, and transformation was performed as described by Hanahan (1983). Construction and screening of genomic library of $S$. albus ATCC 21838 were carried out as described in our previous paper (Choeng et al. 2007).

### Heterologous expression and purification of recombinant His$_6$-tagged SalQ in $E. coli$

The salQ gene was amplified by PCR using Supercos 37-20 as template and primers SQF (5′-GGATCCATGACCGGTACGAGCCTGA-3′) containing a BamHI site and SQR (5′-AAGCTTCTAAAAGGTTCCTGCCGCCGC-3′) containing a HindIII site. The PCR reaction was performed as previously described (Choi et al. 2006). The PCR product was cloned into the pCR 2.1 TOPO TA cloning vector (Invitrogen), resulting in pSQ1. The BamHI–HindIII fragment carrying the salQ gene from pSQ1 was then ligated into pET-30a expression vector (Novagen), resulting in pSQ2. pSQ2 was transferred into $E. coli$ BL21(DE3)pLysS (Stratagene) by electroporation. The transformant was cultured in 200 ml of LBBS medium (LB containing 1 M betaine and 2.5 M sorbitol), supplemented with kanamycin (50 μg/ml) and chloramphenicol (25 μg/ml), at 37°C and 200 rpm, to an OD$_{600}$ of 0.5. Isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) was then added (0.4 mM), and the culture was allowed to grow for an additional 24 h at 28°C. Cells were harvested by centrifugation (5,000×g, 10 min) and disrupted by sonication in disruption buffer [25 mM K$_2$HPO$_4$–KH$_2$PO$_4$ (pH 7.5) containing 300 mM NaCl, 20 mM imidazole, 0.2 mM CoCl$_2$, and AEBSF]. The cell debris was removed by centrifugation and the protein solution was mixed with Ni$^{2+}$-NTA agarose and let stand for 1 h. The Ni$^{2+}$-NTA agarose was washed three times with the same buffer and the His$_6$-tagged protein was eluted with buffer containing imidazole (250 mM). The eluted protein was dialyzed in dialysis buffer [K$_2$HPO$_4$–KH$_2$PO$_4$ pH 7.5 (25 mM), CoCl$_2$ (0.2 mM)]. Protein concentration was measured using the Bradford protein microassay kit (Bio-Rad) with bovine serum albumin as the standard protein (Bradford 1976).

#### Enzyme assay of salQ

Enzyme assays (100 μl volume) were performed at 30°C for 2 h in 25-mM phosphate buffer (pH 7.5) containing protein (50 μl, 0.5 mg protein/ml), sedoheptulose 7-phosphate (5 mM), NAD$^{+}$ (1 mM), CoCl$_2$ (0.05 mM), and KF (2 mM). After incubation, samples were analyzed by thin-layer chromatography (eluent: butanol–ethanol–water, 9:7:4), and both the substrate and the SalQ reaction product were visualized using a mixture of Ce(SO$_4$)$_2$–phosphomolybdic acid–sulfuric acid reagent (Yu et al. 2005). The reaction mixture was lyophilized and the products were extracted with methanol (50 μl). After drying, the extract was mixed with four drops of SIGMA-SIL-A reagent and dried with argon gas. The products were then extracted with n-hexane and injected to gas chromatography–mass spectrometry (GC–MS; Hewlett Packard 5890 series II gas chromatograph) for analysis.

### Results

#### In silico analysis of the putative salbostatin biosynthetic gene cluster

We previously reported the cloning of putative salbostatin biosynthetic gene cluster through a PCR screening of a genomic library of $S$. albus ATCC 21838 using primers which were designed based on the acarbose 2-epi-5-epi-

valiolone synthase gene (acbC). In silico analysis of the DNA sequence revealed 32 ORFs within the 36-kb DNA fragment, and 20 (salA to salT) of them, including the AcbC homolog gene (salQ), were predicted to be involved in salbostatin biosynthesis (Table 2). Although functional analysis of five genes (salBCDEF) related to trehalose biosynthesis was previously reported (Choeng et al. 2007), other genes within the putative salbostatin cluster have not been described. Thus, the genetic organization and the putative functions of the entire gene cluster are summarized in Fig. 2 and Table 2.

From the sequence data, the salC-salF, salG-salH, and SalI-salR subclusters were predicted to transcribe as three distinct operons in the same direction, and the salB-salA to transcribe as one operon in the opposite direction. Interestingly, many of these genes, e.g., salL, salM, salN, salO, salP, and salQ, were found to closely resemble genes from

the acarbose cluster (Fig. 2). The deduced product of salL has a conserved domain related to sugar kinases and shows 60% identity with the 2-epi-5-epi-valiolone 7-kinase (AcbM) that is functionally identified to phosphorylate 2-epi-5-epi-valiolone to form 2-epi-5-epi-valiolone 7-phosphate in acarbose biosynthesis (Zhang et al. 2002).

SalM has an alcohol dehydrogenase GroES-like domain (pfam 08240) that is a typical catalytic domain of alcohol dehydrogenases. It has 68% identity with the cyclitol dehydrogenase (AcbL) that has been speculated to reduce the C-1 keto group of 5-epi-valiolone 7-phosphate to form 5-epi-valiolol 7-phosphate in acarbose biosynthesis (Wehmeier and Piepersberg 2004). SalN shows significant homology to many alcohol dehydrogenases and quinone dehydrogenases, including a 63% identity with the cyclitol oxidoreductase (AcbN), which dehydrates 5-epi-valiolol 7-phosphate to form 1-epi-valienol 7-phosphate in acarbose biosynthesis.

Table 2 Deduced functions of the open reading frames in the sal gene cluster

| Protein | Amino acid number | Proposed function | Origin of homologs (accession number in database) | Identity/similarity (%) | Accession number |
|---------|-------------------|--------------------|--------------------------------------------------|------------------------|-------------------|
| SalA    | 382               | N-acetylglucosamine-6-phosphate deacetylase | Streptomyces avermitilis MA-4680 (NP_825118.1) | 50/63                 | EU822205          |
| SalB    | 307               | Sugar kinase-N-acetyl amino sugar kinase | Streptomyces coelicolor A3(2) (NP_628457.1) | 46/64                 | EF394361          |
| SalC    | 491               | Trehalose-6-phosphate synthase-glycosyltransferase (ValL) | Streptomyces hygroscopicus subsp. Jinggangensis (ABA 41504.1) | 41/56                 | EF394362          |
| SalD    | 385               | Major facilitator superfamily transporter | Rhodobacter sphaeroides 2.4.1 (YP_353908.1) | 32/49                 | EF394363          |
| SalE    | 294               | Trehalose-6-phosphate synthase-phosphatase | Thermobifida fusca YX (YP_288285.1) | 33/43                 | EF394364          |
| SalF    | 390               | 1-epi-valienol-1, 7-bisphosphate-1-adenylyltransferase (AcbR) | Actinoplanes sp. SE 50/110 (CAJ 81025.1) | 69/79                 | EF394365          |
| SalG    | 213               | Response DNA binding regulatory protein | Streptomyces avermitilis MA-4680 (NP_827215.1) | 33/50                 | EU822206          |
| SalH    | 423               | Unknown with glycosyltransferase motif-dehydrogenase | Lactobacillus salivarius subsp. salivarius UCC 118 (YP_535283.1) | 31/53                 | EU822207          |
| SalI    | 443               | Glycosyltransferase-N-acetylglicosaminyl transferase | Syntrophus aciditrophicus SB (YP_462866.1) | 35/52                 | EU822208          |
| SalJ    | 253               | Oxidoreductase-dehydrogenase | Xanthobacter autotrophicus Py 2 (YP_001417656.1) | 32/50                 | EU822209          |
| SalK    | 394               | Major facilitator superfamily transporter | Stigmatella aurantiaca DW 4/3-1 (ZP_01464615.1) | 34/50                 | EU822210          |
| SalL    | 314               | 2-epi-5-epi-valiolone 7-kinase (AcbM) | Actinoplanes sp. SE 50/110 (CAD 29482.2) | 60/69                 | EU822214          |
| SalM    | 366               | Cyclitol dehydrogenase (AcbL) | Actinoplanes sp. SE 50/110 (CAD 29483.2) | 68/78                 | EU822215          |
| SalN    | 254               | Cyclitol oxidoreductase (AcbN) | Actinoplanes sp. SE 50/110 (CAD 29484.1) | 63/72                 | EU822216          |
| SalO    | 268               | 2-epi-5-epi-valiolone-7-phosphate 2-epimerase (AcbO) | Actinoplanes sp. SE 50/110 (CAD 29485.2) | 61/68                 | EU822217          |
| SalP    | 225               | Phosphohexomutase, phosphatase | Rhizobium etli CFN 42 (YP_469076.1) | 51/64                 | EU822211          |
| SalQ    | 411               | 2-epi-5-epi-valiolone synthase (AcbC) | Actinoplanes sp. SE 50/110 (CAA 77208.2) | 72/83                 | EU141958          |
| SalR    | 181               | NTP pyrophosphohydrolase (AcbP) | Actinoplanes sp. SE 50/110 (CAJ 81026.1) | 67/76                 | EU822212          |
| SalS    | 319               | Quinone oxidoreductase | Streptomyces coelicolor A3(2) (NP_628012.1) | 59/73                 | EU822213          |
| SalT    | 203               | TetR-family regulatory protein | Streptomyces coelicolor A3(2) (NP_630856.1) | 44/62                 | EU822218          |

Fig. 2 Genetic organization of the putative salbostatin biosynthetic gene cluster. The genetic organizations of the putative salbostatin and acarbose biosynthetic gene clusters were compared. Arrowheads indicate the ORFs and their directions of transcription. The function of each ORF was deduced from the database and matched with those of acarbose biosynthesis. The thick black arrows between two clusters indicate the ORFs that may have identical function in salbostatin and acarbose biosynthesis, respectively.

![Diagram](https://i.imgur.com/yourimage.png)

- MFS transporter
- Regulatory
- Glucosyl unit modification
- Cyclitol biosynthesis
- Condensation of cyclitol with a glycosyl unit
- Trehalose biosynthesis
- Unknown

The TIM alpha-beta barrel structure (pfam 01261) that is found in xylose isomerase of bacterial myo-inositol catabolism proteins is also found in SalO. SalO has 61% identity with the 2-epi-5-epi-valiolone 7-phosphate 2-epimerase (AcbO), a new type of epimerase that converts 2-epi-5-epi-valiolone 7-phosphate into its epimer, 5-epi-valiolone 7-phosphate, in the acarbose biosynthetic pathway (Zhang et al. 2003). SalP has a phosphatase domain (PRK11587), showing 51% identity and 64% similarity with a putative 2-deoxyglucose-6-phosphate phosphatase protein (YP_469076.1) from *Rhizobium etli* CFN42.

The SalQ (411 aa, GenBank accession no., EU141958) protein showed 52% and 72% identity with ValA and AcbC proteins which cyclize sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone in validamycin and acarbose biosynthesis, respectively. The SalQ protein also showed significant similarity (36% identity) to the AroB protein from *Emericella nidulans* which is known to be responsible for the cyclization of 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAHP) to dehydroquinate (Carpenter et al. 1998). Comparisons of amino acids sequence of SalQ with those of other 2-epi-5-epi-valiolone synthases, such as AcbC, ValA, and CetA from the cetoniacytone producer, *Actinomyces* sp. strain Lu9419 (Schlörke et al. 2002), and Be-Orf9 from the BE-40644 producer, *Actinoplanes* sp. strain A40644 (Torigoe et al. 1996), revealed that a total of 13 amino acid residues (120R, 134D, 167K, 177N, 209E, 212K, 265P, 274R, 303D, 338H, 343P, 356H, and 364P) which had been suggested to be important for the catalytic function in DAHQ synthetase (Carpenter et al. 1998) and 2-epi-5-epi-valiolone synthase (Wu et al. 2007) was well conserved.

Comparative analysis of the putative genes within the salbostatin cluster with those of the acarbose and validamycin pathways revealed that *salF* may also be involved in the biosynthesis of the cyclitol unit. The deduced amino acid sequence of SalF shows 69% identity with the 1-epi-valienol 1,7-bisphosphate-1-adenylyltransferase (AcbR) that catalyzes the formation of NDP-1-epi-valienol 7-phosphate from 1,7-diphospho-1-epi-valienol in the acarbose biosynthetic pathway (Wehmeier and Piepersberg 2004). Additionally, it also shares 68% identity with glucose-1-phosphate adenyl transferase (ValB) in the validamycin biosynthetic gene cluster from *S. hygroscopicus*.

Finally, four genes (*salA*, *salB*, *salJ*, *salS*) were predicted to be involved in the formation of 2-amino-1,5-anhydro-2-deoxy-D-glucitol moiety and either *salC* (encodes a trehalose-6-phosphate synthase), *salH* (encodes an unknown protein with a glycosyltransferase motif), or *sall* (encodes a N-acetylglucosaminyltransferase) may be involved in the coupling reaction between the two moieties of salbostatin.

Expression, purification, and characterization of SalQ protein

Based on the biosynthetic pathways of acarbose and validamycin, cyclization of sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone is a common step for valienamine moiety, and SalQ was expected to be responsible for this cyclization reaction in salbostatin biosynthesis (Fig. 3a). Thus, *salQ* was subcloned in the expression vector pET-30a and the product, pSQ2, was subsequently transferred into *E. coli* BL21(DE3). Expression of *salQ* under the control of the T7 promoter was induced by IPTG, which gave rise to a 50.55-kDa soluble polyhistidine-tagged protein. The SalQ protein was purified to almost homogeneity (>90%) by Ni²⁺-agarose affinity column chromatography (Fig. 3b).
To determine the function of purified protein, the reaction was carried out using sedoheptulose 7-phosphate as a substrate under reaction conditions used previously for the AcbC protein of acarbose biosynthesis (Stratmann et al. 1999). The SalQ reaction product had the same $R_{\mathrm{f}}$ value of 0.5 as that of authentic 2-epi-5-epi-valiolone on thin-layer chromatography (Fig. 3c). Moreover, incubation with cell extracts of *E. coli* harboring the empty pET-30a vector gave no product at $R_{\mathrm{f}}$ 0.5, and the SalQ reaction product showed increased intensity as reaction time elapsed (data not shown). The product was extracted from the lyophilized reaction mixture by methanol, converted to its trimethylsilylated derivative, and analyzed by GC–MS (Fig. 4). An isotopically labeled authentic sample, 2-epi-5-epi-[6,6-$^{2}$H$_{2}$]valiolone (Mahmud et al. 1999), chemically synthesized from D-mannose was used for comparison (Fig. 4a). The enzyme product was detected as a tetra-trimethylsilyl derivative of 2-epi-5-epi-valiolone [$m/z$ 480 ($M^{+}$)] with major fragment ions at $m/z$ 335, 276, 217, 147, 103, and 73 (Fig. 4b). This fragmentation pattern is consistent with that of the authentic sample, which showed $m/z$ 482 ($M^{+}$), 335, 278, 217, 147, 103, and 73 (Fig. 4). This result clearly indicates that SalQ is a 2-epi-5-epi-valiolone synthase originated from *S. albus* ATCC 21838. Enzyme assay of purified SalQ with different sugar phosphate substrates, such as amino DAHP, DAHP, glucose 6-phosphate, fructose 6-phosphate, mannose 6-phosphate, and ribose 5-phosphate (50 mM), did not show any converted product, which suggests that SalQ has high specificity for sedoheptulose 7-phosphate as substrate (data not shown).

### Discussion

Similar to many C$_{7}$N-aminocyclitol-containing natural products, salbostatin has a valienamine moiety in its structure (Fig. 1). Therefore, as in the case of acarbose and validamycin A biosynthesis (Wehmeier and Piepersberg 2004; Bai et al. 2006), cyclization of sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone was thought to be an indispensable step for valienamine formation in salbostatin. Among the 32 ORFs assigned in the cloned 36-kb DNA, SalQ showed significant similarities to the 2-epi-5-epi-valiolone synthases AcbC (72% identity) and ValA (52% identity). The SalQ protein selectively catalyzed the cyclization of sedoheptulose 7-phosphate but not 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAHP) or other substrates, similar to the specificity of AcbC and ValA (data not shown). In addition, *salQ/L/O/M/N/F* are highly similar to the acarbose *acbC/M/O/L/N/R*, which led us to believe that the cloned DNA harbors the salbostatin biosynthetic gene cluster (Fig. 2).

![Figure 3](#fig3)

**Fig. 3** Heterologous expression and catalytic activity of SalQ. **a** SalQ catalyzes cyclization of sedoheptulose 7-phosphate to 2-epi-5-epi-valiolone. **b** SDS-PAGE of SalQ produced in *E. coli*. 1, cell-free extract of *E. coli* BL21 (DE3)pLysS carrying *salQ*; 2, purified SalQ; 3, protein marker. **c** TLC analysis of SalQ product. *a*, cell-free extract of *E. coli* containing empty vector; *b*, incubation of sedoheptulose 7-phosphate with boiled cell-free extract of *E. coli* BL21 (DE3)pLys carrying *salQ*; *c*, incubation of sedoheptulose 7-phosphate with cell-free extract containing SalQ; *d*, 2-epi-5-epi-valiolone standard

![Chemical Reaction](#chemical-reaction)

**sedoheptulose 7-phosphate** $\xrightarrow{\text{SalQ}}$ **2-epi-5-epi-valiolone**

![SDS-PAGE and TLC Analysis](#sdspage-tlc)

**b** ![SDS-PAGE](#sdspage)  
**c** ![TLC](#tlc)  
Fig. 4 GC–MS analysis of SalQ product.  
a Chromatogram and mass spectrum of tetra-trimethylsilyl (TMS) derivative of 2-epi-5-epi-[6,6-²H₂]valiolone standard.  
b Chromatogram and mass spectrum of tetra-TMS derivative of SalQ product.

Other major peaks: v (M⁺ 448) is an unknown impurity or a TMS derivative of 2-epi-5-epi-[6,6-²H₂]valiolone degradation product, w (M⁺ 390) is a tri-trimethylsilyl derivative of dehydrated 2-epi-5-epi-valiolone, x (M⁺ 480) is proposed to be a tetra-TMS derivative of 2-epi-5-epi-valiolone isomer, y (M⁺ 449) is an unknown compound, z (M⁺ 552) is a penta-TMS derivative of 2-epi-5-epi-valiolone

However, due to the lack of applicable transformation system for *S. albus* ATCC 21838, our effort to prove their functions by gene disruption or complementation was not successful. Thus, we placed the 28-kb DNA fragment containing the region from *salA* to *salT* on pOJ446 (Kieser et al. 2000) and introduced the resulting plasmid (pOJ446-SalAT) into *S. lividans* TK24. An ESI mass analysis of the culture broth of the transformant yielded several major molecular ions including *m/z* 322.3 (M⁺) for salbostatin (*C₁₃H₂₃NO₈*, calcd 321.3275). Although authentic salbostatin was not available, MS fragmentation of *m/z* 322.3 (M⁺) gave many fragmented ions of 164.3, 285.8, 303.9, and 322.5 (Supplementary data S1). This fragmentation pattern is consistent with that of the authentic salbostatin, which showed *m/z* (M⁺) of 322 (100%), 304 (11%), 286 (9%), and 164 (33%; Vertesy et al. 1994). Although our attempts to isolate salbostatin through a chromatographic procedure failed,

these data strongly indicate that salbostatin can be produced by the introduction of pOJ446SalAT in *S. lividans* TK24.

Based on the proposed biosynthetic pathway to acarbose by Piepersberg and coworkers (Zhang et al. 2002), 2-epi-5-epi-valiolone is first converted to its activated form, 2-epi-5-epi-valiolone 7-phosphate, by the action of the 2-epi-5-epi-valiolone 7-kinase (AcbM), followed by epimerization at the C-2 position by AcbO to give 5-epi-valiolone 7-phosphate. The latter compound is proposed to be successively converted to 5-epi-valiolol 7-phosphate, 1-epi-valienol 7-phosphate, 1,7-diphospho-1-epi-valienol, and then NDP-1-epi-valienol 7-phosphate by AcbL (polyol dehydrogenase), AcbN (short-chain dehydrogenase), AcbU/J, and AcbR (NDP-polyol synthase), respectively. Condensation of NDP-1-epi-valienol 7-phosphate with dTDP-4-amino-4,6-dideoxy-D-glucose may finally result in dTDP-acarviose 7-phosphate, which is then converted to acarbose by a condensation of

Fig. 5 A proposed biosynthetic pathway for salbostatin in *S. albus* ATCC 21838. Proteins that are highly similar to those of the acarbose pathway are depicted in **bold** and proteins with deduced function are presented with a *question mark*

![Diagram](attachment:diagram.png)

maltose (Zhang et al. 2002). While enzymes for the first three steps leading to the formation of 5-epi-valiolone 7-phosphate have been characterized, further downstream, the pathway remains speculative. By an analogy to acarbose biosynthesis, the 2-epi-5-epi-valiolone catalyzed by SalQ may have the same fate of being phosphorylated by SalL and then further converted to cyclitol moiety (probably NDP-1-epi-valienol) of salbostatin by SalO, SalM, SalN, SalP, and SalF. Based on this assumption, a putative biosynthetic pathway for salbostatin is postulated in Fig. 5.

In contrast to the acarbose and validamycin A clusters, no gene for an aminotransferase that leads to amino sugar formation was found in the cloned DNA and neighboring chromosomal region. Although it is possible that the aminotransferase gene is located distantly from the sequenced region, in silico analysis of other genes in the cluster suggests that the nitrogen may be originated from N-acetylglucosamine. We propose that N-acetylglucosamine is phosphorylated by SalB and reduced to N-acetyl-1-deoxyglucosamine-6-phosphate by SalS (galactitol 1-phosphate dehydrogenase) or SalJ (oxidoreductase–dehydrogenase). Deacetylation of the product by SalA would give 2-amino-1,5-anhydro-2-deoxyglucitol, setting the stage for a coupling reaction with NDP-(1-epi-)valienol. The coupling reaction may be catalyzed by either SalI (N-acetylglucosaminyl transferase) or SalC (trehalose 6-phosphate synthase), to form salbostatin 6′-phosphate. Alternatively, the unknown protein with glycosyltransferase motif (SalH) may be responsible for this unusual glycosylation. Finally, dephosphorylation of salbostatin 6′-phosphate by the phosphatase SalE would give the final product salbostatin.

Alternatively, SalC (similar to OtsA) is proposed to catalyze the synthesis of trehalose 6-phosphate and SalE (similar to OtsB) to catalyze the dephosphorylation of the product to give trehalose similar to the OtsA–OtsB pathway in *E. coli* (De Smet et al. 2000). In fact, introduction of *salB–salF* induced the production of trehalose, abundant sporulation, and production of actinorhodin in *S. lividans* TK24, suggesting that higher concentration of trehalose are affecting the morphological and physiological differentiation in *S. lividans* (Choeng et al. 2007). Thus, the neighboring presence of the trehalose biosynthetic gene cluster and the salbostatin (trehalase inhibitor) biosynthetic gene cluster implies that precise control of trehalose metabolism is necessary for survival or physiological control in *S. albus*. In addition, putative regulatory genes such as *salG* and *salT* may play some important role in balancing the trehalose metabolism.

Acknowledgments The authors thank Patricia M. Flatt for critical reading of the manuscript. This research was supported by the Driving Force Project for the Next Generation, Gyeonggi Provincial Government, Republic of Korea and general research funds from the College of Pharmacy Oregon State University.

References

Bai L, Li L, Xu H, Minagawa K, Yu Y, Zhang Y, Zhou X, Floss HG, Mahmud T, Deng Z (2006) Functional analysis of the validamycin biosynthetic gene cluster and engineered production of validoxylamine A. Chem Biol 13:387–397

Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 72:248–254

Carpenter EP, Hawkins AR, Frost JW, Brown KA (1998) Structure of dehydroquinate synthase reveals an active site capable of multistep catalysis. Nature 394:299–302

Choeng YH, Yang JY, Delcroix G, Kim YJ, Chang YK, Hong SK (2007) Expression and characterization of trehalose biosynthetic modules in the adjacent locus of salbostatin biosynthetic gene cluster. J Microbiol Biotechnol 17:1675–1681

Choi SS, Kim JH, Kim JH, Kang DK, Kang SS, Hong SK (2006) Functional analysis of *sprD* gene encoding *Streptomyces griseus* protease D (SGPD) in *Streptomyces griseus*. J Microbiol Biotechnol 16:312–317

De Smet KA, Weston A, Brown IN, Young DB, Robertson BD (2000) Three pathways for trehalose biosynthesis in mycobacteria. Microbiology 146:199–208

Dong H, Mahmud T, Tornus I, Lee S, Floss HG (2001) Biosynthesis of the validamycins: identification of intermediates in the biosynthesis of validamycin A by *Streptomyces hygroscopicus* var. *limoneus*. J Am Chem Soc 123:2733–2742

Hanahan D (1983) Studies on transformation of *Escherichia coli* with plasmids. J Mol Biol 166:557–580

Iwasa T, Kameda Y, Asai M, Horii S, Mizuno K (1971) Studies on validamycins. IV. Isolation and characterization of validamycin A and B. J Antibiot (Tokyo) 24:119–123

Kieser T, Bibb MJ, Chater KF, Butter MJ, Hopwood DA (2000) Practical *Streptomyces* genetics. A laboratory manual. John Innes Foundation, Norwich

Mahmud T (2003) The C<sub>7</sub>N aminocyclitol family of natural products. Nat Prod Rep 20:137–166

Mahmud T, Tornus I, Egelkrout E, Wolf E, Uy C, Floss HG, Lee S (1999) Biosynthetic studies on the α-glucosidase inhibitor acarbose in *Actinoplanes* sp.: 2-epi-5-epi-valiolone is the direct precursor of the valienamine moiety. J Am Chem Soc 121:6973–6983

Sambrook J, Russell DW (2001) Molecular cloning, 3rd edn. Cold Spring Harbor Laboratory Press, New York

Schlörke O, Krastel P, Müller I, Usón I, Dettner K, Zeeck A (2002) Structure and biosynthesis of cetoniacytone A, a cytotoxic aminocarba sugar produced by an endosymbiotic *Actinomyces*. J Antibiot (Tokyo) 55:635–642

Shibata M, Ueda M, Mori K (1981) Stimulation of the extension of validamycin-inhibited hyphae by the hyphal extension factor present in *Rhizoctonia solani*. J Antibiot (Tokyo) 34:447–451

Singh D, Seo MJ, Kwon HJ, Rajkarnikar A, Kim KR, Kim SO, Suh JW (2006) Genetic localization and heterologous expression of validamycin biosynthetic gene cluster isolated from *Streptomyces hygroscopicus* var. *limoneus* KCCM 11405 (IFO 12704). Gene 376:13–23

Stockmann M, Piepersberg W (1992) Gene probes for the detection of 6-deoxyhexose metabolism in secondary metabolite-producing streptomycetes. FEMS Microbiol Lett 69:185–189

Stratmann A, Mahmud T, Lee S, Distler J, Floss HG, Piepersberg W (1999) The AcbC protein from *Actinoplanes* species is a C<sub>7</sub>-cyclitol synthase related to 3-dehydroquinate synthases and is involved in the biosynthesis of the α-glucosidase inhibitor acarbose. J Biol Chem 274:10889–10896

Torigoe K, Wakasugi N, Sakaizumi N, Ikejima T, Suzuki H, Kojiri K, Suda H (1996) BE-40644, a new human thioredoxin system inhibitor isolated from *Actinoplanes* sp. A40644. J Antibiot (Tokyo) 49:314–317

Vertesy L, Fehlhaber HW, Schulz A (1994) The trehalose inhibitor salbostatin, a novel metabolite from *Streptomyces albus* ATCC21838. Angew Chem Int Ed 33:1844–1846

Wehmeier UF, Piepersberg W (2004) Biotechnology and molecular biology of the α-glucosidase inhibitor acarbose. Appl Microbiol Biotechnol 63:613–625

Wu X, Flatt PM, Schlörke O, Zeeck A, Dairi T, Mahmud T (2007) A comparative analysis of the sugar phosphate cyclase superfamily involved in primary and secondary metabolism. ChemBioChem 8:239–248

Yu Y, Bai L, Minagawa K, Jian X, Li L, Li J, Chen S, Cao E, Mahmud T, Floss HG, Zhou X, Deng Z (2005) Gene cluster responsible for validamycin biosynthesis in *Streptomyces hygroscopicus* subsp. *jinggangensis* 5008. Appl Environ Microbiol 71:5066–5076

Zhang CS, Stratmann A, Block O, Bruckner R, Podeschwa M, Altenbach HJ, Wehmeier UF, Piepersberg W (2002) Biosynthesis of the C(7)-cyclitol moiety of acarbose in *Actinoplanes* species SE50/110. 7-O-phosphorylation of the initial cyclitol precursor leads to proposal of a new biosynthetic pathway. J Biol Chem 277:22853–22862

Zhang CS, Podeschwa M, Altenbach HL, Piepersberg W, Wehmeier UF (2003) The acarbose-biosynthetic enzyme AcbO from *Actinoplanes* sp. SE 50/110 is a 2-epi-5-epi-valiolone-7-phosphate 2-epimerase. FEBS Lett 540:47–52
