
Molecular Microbiology (2008) 70(5), 1054–1061

doi:10.1111/j.1365-2958.2008.06482.x  
First published online 22 October 2008

# MicroReview

## The diverse CheC-type phosphatases: chemotaxis and beyond

### Travis J. Muff*† and George W. Ordal

Department of Biochemistry, Colleges of Medicine and Liberal Arts and Sciences, University of Illinois, Urbana, IL 61801, USA.

---

### Summary

A new class of protein phosphatases has emerged in the study of bacterial/archaeal chemotaxis, the CheC-type phosphatases. These proteins are distinct and unrelated to the well-known CheY-P phosphatase CheZ, though they have convergently evolved to dephosphorylate the same target. The family contains a common consensus sequence D/S-X₃-E-X₂-N-X₂₂-P that defines the phosphatase active site, of which there are often two per protein. Three distinct subgroups make up the family: CheC, FliY and CheX. Further, the CheC subgroup can be divided into three classes. *Bacillus subtilis* CheC typifies the first class and might function as a regulator of CheD. Class II CheCs likely function as phosphatases in systems other than chemotaxis. Class III CheCs are found in the archaeal class Halobacteria and might function as class I CheCs. FliY is the main phosphatase in the *B. subtilis* chemotaxis system. CheX is quite divergent from the rest of the family, forms a dimer and some may function outside chemotaxis. A model for the evolution of the family is discussed.

---

### Introduction

Signal transduction is largely accomplished in bacteria, and a few instances in plants and fungi, using the two-component signalling systems (West and Stock, 2001). Such systems consist of a histidine kinase protein, which senses an environmental cue, and a response regulator, which becomes phosphorylated and generates the response. These kinases differ from traditional serine/threonine kinases in that the histidine kinase autophosphorylates at a conserved histidine residue. The response regulator phosphorylates itself at an aspartate residue, using the phosphoryl-histidine as a substrate. These response regulator proteins have intrinsic dephosphorylation activity whose the rate reflects the signalling requirements of the particular system. Additionally, some of these systems contain response regulator phosphatases for further regulation.

The well-known bacterial chemotaxis system is a highly specialized two-component system that utilizes phosphatases (Stock *et al.*, 2002; Szurmant and Ordal, 2004). In this system, the CheA histidine kinase lacks a traditional sensor domain. Instead, chemoreceptor proteins (methylated chemotaxis proteins or MCPs) sense the chemo-effectors and control CheA autophosphorylation activity. The response regulator CheY is the target of CheA. Chemotaxis is achieved by controlling the rotation of the bacterial flagella through modulation of CheY-P (phosphorylated CheY) levels. In *Bacillus subtilis*, binding of attractant to the MCPs causes activation of CheA and an increase in CheY-P levels. Association of CheY-P and FliM (a component of the flagella) causes a switch in the flagellar rotation from clockwise (CW), effecting a reorienting tumble, to counter clockwise (CCW) and a unidirectional smooth swim. By prolonging the smooth swims under favourable conditions and reorienting itself under unfavourable conditions, the cell performs a biased random walk towards a better environment.

In *Escherichia coli* and other proteobacteria, the CheZ phosphatase helps control CheY phosphorylation levels and colocalizes to the poles of the cell with CheA and the MCPs. This location in the cell is thought to equilibrate concentrations of CheY-P across the cell and to prevent the formation of a gradient (Vaknin and Berg, 2004). However, CheZ is not found outside the proteobacteria (Szurmant and Ordal, 2004). In other bacteria (and the archaea) the CheY-P phosphatase role is handled by members of the CheC family (Muff and Ordal, 2007a).

Two members of this family, CheC and FliY, were initially identified in the major *che/fla* operon of *B. subtilis* (Bischoff and Ordal, 1992; Rosario *et al.*, 1995). FliY is a multidomain protein, the C-terminus of which is homologous to the FliN flagellar component of *E. coli* (Toker and Macnab, 1997). Although it contains additional domains,

---

Accepted 29 September, 2008. *For correspondence. E-mail tjmuff@scripps.edu or tjmuff@gmail.com; Tel. (858) 784-7909; Fax (858) 784-7966.* †Present address: Department of Molecular and Experimental Medicine, The Scripps Research Institute, La Jolla, CA 92037, USA.

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd

CheX
![Diagram](https://i.imgur.com/1234567.png)

CheC I
![Diagram](https://i.imgur.com/8901234.png)

CheC II
![Diagram](https://i.imgur.com/5678901.png)

CheC III
![Diagram](https://i.imgur.com/2345678.png)

FliY
![Diagram](https://i.imgur.com/9012345.png)

FliM
![Diagram](https://i.imgur.com/6789012.png)

Fig. 1. Primary architecture of the CheC family members. White boxes represent protein domains (Y-B is a CheY-P binding motif of FliY and FliM). Coloured boxes represent functional motifs of the CheC domain (green – active site one, blue – active site two, red – CheD binding helix). Letters represent approximate locations of conserved residues in the CheC domain. Black areas represent linker regions.

FliY fulfils the role of FliN in *B. subtilis* (Bischoff and Ordal, 1992). *E. coli* does not have a homologue of CheC, which is classified as an adaptational chemotaxis protein. Both of these proteins are now known to be phosphatases of CheY, though with very different activities (Szurmant et al., 2003; 2004). CheX, another chemotaxis protein sharing sequence similarity with CheC and FliY, was later shown to share this CheY-P phosphatase activity (Park et al., 2004; Muff et al., 2007).

Whereas CheZ is limited to the Proteobacteria, the CheC family is much more widespread and found across the Firmicutes and Thermotogales (Szurmant and Ordal, 2004). Though FliY is mainly limited to these phyla, CheC and CheX also exist in Proteobacteria, and CheX homologues are present in the Spirochetes. Furthermore, some of these proteins do not seem to be limited to chemotaxis systems. It therefore appears that the CheC family comprises a diverse family of response regulator phosphatases involved in two-component signal transduction systems.

Sequence identity in the family is generally limited to the phosphatase active sites. The consensus sequence D/S-X₃-E-X₂-N-X₂₂-P defines these phosphatase active sites (Fig. 1) (Park et al., 2004). CheC and FliY each have two active sites, while CheX has only one. Though a structure of CheY-P bound CheC has not to date been determined, mutational analysis of *cheC* has revealed the functions of the individual active site residues (Park et al., 2004; Muff and Ordal, 2007b). The conserved glutamate residue is critical for the association with CheY-P, whereas the asparagine residue is absolutely necessary for the phosphatase activity. The aspartate/serine residue likely helps to stabilize the CheC/CheY-P interaction. The proline residue does not appear to be needed for the phosphatase reaction and it is likely a solely structural element. The critical glutamate and asparagine residues likely mimic the conserved aspartate and glutamine residues in CheZ (Zhao et al., 2002). Here, the amide residue is thought to position a water molecule for attack on the phospho-aspartate of CheY. The carboxyl residue forms a salt bridge with CheY residues (notably a lysine residue conserved throughout response regulator receiver domains). Thus, though the CheC-type phosphatases and CheZ evolved separately, they likely have a convergently evolved active site mechanism (Muff and Ordal, 2007b).

The low level of sequence similarity makes it difficult to identify divergent family members in sequence database searches (Muff and Ordal, 2007a). In fact, protein domain identification programs (such as Pfam or SMART) often only identify CheC family members by the presence of the conserved active site residues. These programs often identify only one active site (as in CheX) or fail to identify the CheC-domain altogether. With molecular structures now available, it becomes apparent the CheC family shares a common structural fold and evolutionary ancestor (Park et al., 2004; 2006). The family varies around a common α/β/α sandwich fold (Fig. 3). The structure itself is a tandem repeat with the N-terminus (Fig. 3, blue) and C-terminus (Fig. 3, gold) internally similar. The phosphatase active site residues Asp/Ser, Glu and Asn are found on the two α-helices (α1 and α1′) on one side of the central six-stranded β-sheet (Fig. 3, residues represented by stars). The Pro residue is on an adjacent β-strand (β1 and β1′) capping the sides of the central sheet. Opposite the central β-sheet from the active sites are two smaller α-helices (α2 and α2′) or, in the case of CheX, two β-strands (Fig. 3). As described later, this variable structure

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd, *Molecular Microbiology*, **70**, 1054–1061

CheX

CheC  
class III  

FliM  

FliY  

CheC  
class I  

CheC  
class II  

Fig. 2. Phylogenetic tree of the CheC family. The tree was constructed based on an alignment of the CheC domains of each protein. This demonstrates the basis for the subfamily identifications, especially the class I and II CheCs.

tural element has major functional implications. Two additional α-helices (α3 and α3′) cap the turn ends of the central β-sheet.

CheC

CheC is the namesake of this protein family and the most commonly known member. The basic family structure defined above is that of CheC. However, phylogenetic analysis reveals that this is not a mono-cladistic group, but that there are three distinct subgroups (here defined as classes I, II and III, Figs 1 and 2). While only the class I CheCs have been well studied, we can infer functional differences from sequence data and genomic context.

Class I

The class I cheC genes are generally found in Firmicute and Archaeal (Fig. 2) chemotaxis operons with cheD, and the two genes are usually translationally coupled (Szurmant and Ordal, 2004; Szurmant et al., 2004). In addition to the familial phosphatase activity, class I CheCs interact with the CheD protein (Rosario and Ordal, 1996). This protein is responsible for deamidation of certain glutamine residues to glutamates on the chemotaxis receptor proteins (Kristich and Ordal, 2002). CheD has also been implicated in MCP demethylation (Chao et al., 2006) and is critical for proper excitation of CheA by the MCPs (Rosario et al., 1995; Kirby et al., 2001).

The α2′-helix of CheC acts as a mimic of the natural enzymatic target of CheD, the α-helical receptors (Chao

et al., 2006), and serves as the binding site for CheD (Fig. 4). Additionally, CheD contains a conserved phenylalanine (F93 in *Thermotoga maritima*, F102 in *B. subtilis*) that inserts in a hydrophobic pocket on CheC adjacent to the α2′-helix. CheC/CheD heterodimerization creates a more potent CheY-P phosphatase than the rather weak CheC alone (a fivefold increase in activity) (Szurmant et al., 2004; Muff and Ordal, 2007b). Additionally, the CheC/CheY-P association was shown to increase the affinity of CheC for CheD (Chao et al., 2006; Muff and Ordal, 2007b). In addition to the conserved active site residues, an Asp and Ala are conserved on the N-terminus of the α2′ CheD-binding helix in all class I CheCs (Fig. 1). Both residues have been shown to be critical for the CheD association (Muff and Ordal, 2007b) and make up a sequence motif distinguishing class I and II CheCs.

The class I CheCs are involved in adaptation of the chemotaxis system (Rosario et al., 1995). Adaptation (in chemotaxis) refers to the ability of the system to respond to an effector and then return to pre-stimulus activity despite static concentrations of the attractant or repellent. This allows the system to respond to changing concentrations of an effector (as it moves through a gradient) and not just absolute concentrations. A *cheC* mutant is unable to fully complete chemotactic adaptation. Two models have been proposed to explain how CheC effects adaptation. In the first model, CheC acts primarily as a phosphatase (Szurmant et al., 2004). In the pre-stimulus system, CheC helps to keep the correct CheY/CheY-P balance by joining FliY in counteracting CheA activity. CheD is assumed to be sequestered at the MCPs at this point. When CheA activity increases in response to stimulus, CheD is released from the MCPs and is free to bind CheC. The more powerful CheC/CheD phosphatase complex then acts to return CheY-P to its pre-stimulus levels.

Fig. 3. CheC and CheX monomers. The conserved Glu, Asn and Pro residues are represented by stars. Note that at the ‘top’ of CheX (versus CheC) the α3′-helix is missing and also the substitution of the βx and βx′ strands for the α2 and α2′-helices. Reprinted from Park et al. (2004) with permission.

Fig. 4. CheC/CheD heterodimer and CheX homodimer. This demonstrates the dimeric form of both CheC and CheX. The α2′-helix of CheC and the βx′ strand of CheX, important for dimerization, are highlighted in purple. Reprinted from Chao et al. (2006) with permission.

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd, *Molecular Microbiology*, **70**, 1054–1061

In the second model, CheC is viewed as a regulator of CheD (Chao *et al.*, 2006; Muff and Ordal, 2007b). CheD association with the MCPs is thought to heighten the activation of CheA. When the receptors bind an attractant, they activate CheA and the pool of CheY-P increases. CheC then senses the increased pool of CheY-P and its affinity for CheD is increased. This enhanced attraction creates competition for CheD with the MCPs, thus deactivating the receptors and lowering CheA activity. CheY-P levels then return to their pre-stimulus levels, thus completing adaptation. This model is currently favoured, as a CheC mutant unable to dephosphorylate CheY-P is partially functional (Muff and Ordal, 2007b).

### Class II

The class II CheCs are, at first glance, simply CheCs found in the Proteobacteria phylum (Szurmant and Ordal, 2004). However, upon inspection, this class diverges from class I CheCs dramatically in sequence conservation (Fig. 2). The most noticeable difference is that the class II CheC proteins do not have the well-conserved aspartate residue in the CheD-binding helix of class I CheCs (Fig. 1). Although the structure of class II CheCs remains unsolved, it would appear by sequence comparison that the putative α2′-helix is much shorter than its class I counterpart. Further, none of the class II *cheC* genes are found in operons with *cheD*. If found in these organisms, the CheD protein lacks the conserved phenylalanine necessary for CheC binding. Thus, the models proposed for class I CheC function do not fit the class II CheCs. These proteins are likely to be dedicated phosphatases.

The class II *cheC* genes are not found in chemotaxis operons. These *cheC* genes, such as those in *Shewanella oneidensis* (SO2546) or *Vibrio cholerae* (VC1083), are often in operons containing more archetypical two-component signalling components. Others are found in non-signalling operons or as orphans. While most studies on the CheC family to date have focused on their involvement in the chemotaxis system, it would seem that the class II CheCs are involved in other, non-chemotactic two-component systems. However, the systems in which these CheCs function have not been studied; the inputs and outputs are unknown.

The *Myxococcus xanthus* CheC homologue DifG seems to be a class I CheC by sequence and phylogeny (Fig. 2). However, the Dif system, and *M. xanthus* overall, does not contain a CheD homologue, nor does DifG contain the conserved aspartate residue critical for CheD binding. This suggests DifG functions as a class II CheC. Yeast two-hybrid assays have confirmed an interaction between DifG and CheY homologue DifD (Lancero *et al.*, 2005; Yang and Li, 2005), supporting the notion that DifG (and likely all class II CheCs) are dedicated response regulator phosphatases. Strangely, a *difG* mutant has a phenotype similar to that of the *difD* (encoding the CheY paralogue) mutant (Black and Yang, 2004). These results point to a DifG function beyond phosphatase activity, as it is unexpected for mutants of a response regulator and its potential phosphatase to generate the same phenotype. Although its molecular function has yet to be determined, DifG might prove to be an example of how a class II CheC functions.

Another feature of the class II CheCs is that the second active site often diverges from the consensus sequence. The conserved serine and glutamate residues of the second active site are often substituted by a glutamate and aspartate respectively. These deviations might mean that the second active sites of the class II CheCs are not active as phosphatases or that they must contact their phosphorylated response regulator in a different way than indicated in the model for *B. subtilis* CheC (Muff and Ordal, 2007b).

A small set of class II CheCs from the gammaproteobacteria has an unusual architecture. Proteins such as PsyR_0886 from *Pseudomonas syringae* are response regulators with an N-terminal CheY-like receiver domain and a C-terminal CheC domain. Annotations or domain identifications will often not identify the CheC domain of these proteins at all. Genes for these CheYCs are found in operons without genes for histidine kinases or other response regulators. They might function within a two-component system as a phosphatase under control of the receiver domain, as a phosphatase sink (dephosphorylating itself), or have a completely novel function. A CheYX fusion (CAC0585) is found in *Clostridium acetobutylicum* and might have a similar function. This CheYX protein can be phosphorylated by *B. subtilis* CheA and dephosphorylates faster than CheY, suggesting a phosphate sink role (T.J. Muff, thesis 2007).

### Class III

A third class of CheC is present in the archaeal class Halobacteria (Fig. 1). In a phylogenetic tree, these proteins form a clade close to CheX, but it would appear that they are in fact CheCs and not CheXs. This is due primarily to the fact that they contain the α3′-helix that is missing in CheX proteins. The reason for this clade rooting close to the CheX clade (Fig. 2) is due to the class III CheCs not having the conserved residues (D, E, N, P) in the first active site. Furthermore, these CheCs appear to have a full α2′-helix (by sequence analysis) and many have the conserved aspartate residue critical for CheD-binding (in contrast to CheX and class II CheCs). So, it appears that the class III CheCs structurally resemble class I CheCs, despite the absence of the first active site.

From this sequence analysis insight, class III CheCs are assumed to function as class I CheCs.
supported by observations from *B. subtilis* and *Halobacterium salinarium* (Rudolph and Oesterhelt, 1996; Saulmon *et al.*, 2004). In both cases, a strain deleted for *cheC* (termed *cheJ* in *H. salinarium*) had a decrease in flagellar rotational switching (longer rotational events). The molecular explanation for this decrease is not yet understood, but it demonstrates a common link between the CheCs of the two systems. Furthermore, the *cheJ* deletion had a similar rotational phenotype as a *cheY* mutant, which would be the opposite of what would be expected if CheJ functioned solely as a phosphatase like CheX (see below). These results suggest the class III CheCs are in fact members of the CheC subfamily and not CheXs, as their lone active site would suggest.

Additionally, a mutational study of conserved active site residues in *B. subtilis* CheC indicated that the second active site is more important for function than the first active site both *in vitro* and *in vivo* (Muff and Ordal, 2007b). In fact, mutations in the first active site had little effect on CheC function. Thus, members of class III could have arisen by loss of the unnecessary first active site through mutational drift.

### FliY

The flagellar protein FliY was the first member of the CheC family to be identified as a CheY-P phosphatase (Szurmant *et al.*, 2003). This was uncovered through the overall similarity between FliY and FliM. Both proteins have an N-terminal CheY-P binding domain, a C-terminal FliN-like (SpoA domain in the Pfam database) domain, and a central CheC-like domain (Fig. 1), though FliM does not contain the CheC consensus sequence (and is not a CheY-P phosphatase). However, when the CheY-P binding domain was removed from each, opposite phenotypes were found (CW flagellar rotation for FliMΔ6-15, CCW for FliYΔ6-15). This effect was due to the CheY-P phosphatase activity of FliY (Szurmant *et al.*, 2003).

Because of its FliN-like domain, FliY is assumed to be found at the flagellum (Szurmant *et al.*, 2003). The FliN protein is a structural component of the flagellar C ring and is involved in the flagellar switching mechanism (Paul and Blair, 2006). Homologues of FliN are found in all flagellated bacteria and FliY fulfils the role of FliN in the Firmicutes and Thermotogales. However, the reason for locating a phosphatase at the flagellum is unknown.

In *B. subtilis*, a *fliY* mutant lacking the CheY binding domain was almost completely smooth swimming, correlating with an excess of CheY-P (Szurmant *et al.*, 2003). Thus, FliY is the main phosphatase of the *B. subtilis* chemotaxis system, as is CheZ in *E. coli*. However, the phosphatase CheZ is found at the poles of the cell with the CheA/MCP complex (Cantwell *et al.*, 2003). This phosphatase localization has been shown to flatten

CheY-P distribution across the cell (Vaknin and Berg, 2004). The reason for localizing a phosphatase at the target of CheY-P (FliY at the flagella), rather than at the generation point (CheZ at the CheA-MCP complex), remains a mystery. The flagellar location might be unimportant and simply be an artefact of evolution from a FliM/FliY progenitor (see below), or it might serve to equalize the signal seen by each flagellum (Rao *et al.*, 2005), but neither hypothesis has been tested experimentally. Development of a more detailed model of FliY function will require further study.

FliY residues are highly conserved, with 41% identity among distant orthologues. This is likely due to its function: FliY is an essential part of the flagellum with multiple protein-protein interactions (CheY, FliM, FliG, etc.). This would cause the protein to resist genetic drift at these sites of interaction, resulting in the low variation from species to species.

### CheX

The *cheX* gene was originally identified in the chemotaxis operons of the Spirochaetes *Treponema pallidum* and *Borrelia burgdorferi* (Ge and Charon, 1997; Greene and Stamm, 1999; Lux *et al.*, 2002). This was a serendipitous oddity, as most *cheX* genes are found as orphans. To date, these are the only species in which *cheX* mutants have been created. The Spirochaete *cheX* mutant had a phenotype opposite that of a *cheY* mutant, consistent with its role as a phosphatase.

To see CheX’s effect in a better studied system, *cheX* from *Bacillus halodurans* was heterologously expressed in *B. subtilis*, which lacks this gene (Muff *et al.*, 2007). CheX was unable to fully complement a *B. subtilis cheC* mutant, demonstrating that the two proteins have different functions in chemotaxis despite both being CheY-P phosphatases. Furthermore, CheX caused wild-type *B. subtilis* cells to tumble more frequently (lower CheY-P). This finding was consistent with *in vitro* studies that showed CheX was the most powerful of the three CheC-type phosphatases (Muff *et al.*, 2007).

CheX is dramatically different from CheC and FliY in sequence and in structure (Park *et al.*, 2004). Most obvious is the lack of the first active site (Fig. 1). The CheXs also have even less sequence conservation than the rest of the family; the conserved residues are largely limited to the consensus active site residues. A well-conserved glycine is in the same position (five residues C-terminal to the active site proline) as the CheD-associated aspartate on class I and III CheCs and was previously identified as a possible CheX determinant (Park *et al.*, 2004). However, this might be misleading as some class II and III CheCs also have a glycine at this position. A feature not readily noticed from sequence analysis, but obvious from the structure, is the

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd, *Molecular Microbiology*, **70**, 1054–1061
substitution of the α2 and α2′-helices with β-strands (βx and βx′, Fig. 3). CheX forms a dimer with the βx′ strand contacting the central β-sheet of the other monomer and these two strands become contiguous with the central β-sheet (Fig. 4). Thus, despite CheX having only one active site per monomer, it contains two active sites per functional unit. CheX also differs in the lack of a α3′-helix found at the C-terminal end of CheC (Figs 1 and 3). This last feature makes CheXs readily discernable from CheCs, especially the class III CheCs.

CheX appears to be a dedicated phosphatase, unlike class I CheCs. Other than CheY and itself, the only protein with which CheX has been shown to interact is CheA (Sim et al., 2005). This is an interesting observation, as a truncated CheA (CheA short) is known to be the factor that localizes CheZ to the cell poles in *E. coli* (Cantwell et al., 2003). A short CheA has also been observed in *B. subtilis*, though this has not been confirmed as a true CheA-short (T.J. Muff, thesis 2007). It might be that CheX is localized to the cell poles and fulfils a role similar to that of CheZ in preventing the formation of a CheY-P gradient across the cell, allowing each flagellum to see a similar level of CheY-P (Muff et al., 2007).

CheXs cluster in one clade of the family phylogenetic tree (Fig. 2). No subfamily is apparent from analysis of the sequences; the homologues simply seem to be separated by speciation. This suggests that the family members have one common function as a CheY-P phosphatase. However, due to the fact that CheXs are usually orphans, this probable function cannot be taken for granted. It is likely that many CheXs, especially in the proteobacteria or species with multiple homologues (*Clostridium tetani*), target a response regulator other than CheY.

## Discussion

Although FliM lacks the phosphatase activity and active site residues (Toker and Macnab, 1997; Szurmant et al., 2003; Park et al., 2006), FliY and FliM are very similar. In fact, the two proteins have substantial identity over their entire length (20% in *B. subtilis*), including the FliN-like C-terminal domains. Thus, despite their differing functions in chemotaxis, they likely have a common ancestor. Although the structure of a FliY protein has not been determined, that of FliM is known (Park et al., 2006) and we can assume that the structure of FliY is similar. FliM, and likely FliY, contains CheC-like α2-helices, as opposed to the βx-strands in CheX. Along with the fact that CheXs lack the first active site (Fig. 1), it is reasonable to assume that CheX was the last of the family to evolve.

The CheC family likely originated from the flagellum (Fig. 5). First, a FliM-FliY progenitor probably arose in the flagellar switch complex. Then, through gene duplication, FliY evolved as a separate switch component and gained

**Fig. 5.** Evolutionary model of the CheC family. The family likely arose from a component of the flagella which under went gene duplication (1). One of these genes led to *fliM*, while the other encoded a product that gained CheY phosphatase activity. A duplication of just the region encoding the CheC-like domain of FliY became the progenitor of *cheX* and *cheC* (2). The genes encoding these two proteins then diverged with CheX’s gain of dimerization and loss of the α3′-helix (3). The *cheC*’s separated with the gain of the CheC/CheD interaction (4) and class III’s loss of the first active site (5).

the CheY-P phosphatase ability. A further duplication of the CheC-like domain of FliY produced the soluble progenitor of CheC and CheX. This protein then diverged into the CheC and CheX ancestors through gain of homo-dimerization and loss of the α3′-helix from CheX. A deletion of the CheC domain from FliY likely gave rise to the FliN protein found in Proteobacteria and others. The discovery of another protein with a common ancestor to the CheC family would be needed to confirm this evolutionary model and ‘root’ the phylogenetic tree (Fig. 2).

The CheC/CheD interaction is the distinction between the class I (and III) and class II CheCs. It is likely that the common ancestor of the two classes was a dedicated CheY phosphatase, but the acquisition of the CheD interaction caused the split (Fig. 5). The involvement of CheD in chemotaxis was probably already established before the CheC class diverged. The interaction likely developed through evolution of CheC’s α2′-helix (a holdover from FliY/FliM internal symmetry) into a binding site for CheD, with appropriate compensating changes in CheD such as the conserved phenylalanine mentioned previously. This new interaction then imparted a chemotactic advantage, likely enhanced adaptation due to the regulation of CheD by CheC. The phosphatase ability, now not strictly necessary for its function, then degraded through evolutionary drift, with the first active site lost altogether in the class III CheCs (Fig. 5).

It is likely that the CheC family arose before CheZ, as the latter is limited to the proteobacteria. However, as discussed here, the CheC family functions in the chemotaxis system in roles quite different from CheZ, despite sharing phosphatase ability. In addition to chemotaxis, the

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd, *Molecular Microbiology*, **70**, 1054–1061
CheC family has also likely been adapted for use in varied other two-component systems. Due to the low sequence conservation, it might be that undetected homologues exist in two-component systems not yet investigated. It is tempting to think that this family might be even more diverse than we now realize.

## Acknowledgements

This work was supported by National Institutes of Health Grant RO1GM54365 (to G.W.O.).

## References

Bischoff, D.S., and Ordal, G.W. (1992) Identification and characterization of FliY, a novel component of the *Bacillus subtilis* flagellar switch complex. *Mol Microbiol* 6: 2715–2723.

Black, W.P., and Yang, Z. (2004) *Myxococcus xanthus* chemotaxis homologs DifD and DifG negatively regulate fibril polysaccharide production. *J Bacteriol* 186: 1001–1008.

Cantwell, B.J., Draheim, R.R., Weart, R.B., Nguyen, C., Stewart, R.C., and Manson, M.D. (2003) CheZ phosphatase localizes to chemoreceptor patches via CheA-short. *J Bacteriol* 185: 2354–2361.

Chao, X., Muff, T.J., Park, S.Y., Zhang, S., Pollard, A.M., Ordal, G.W., *et al.* (2006) A receptor-modifying deamidase in complex with a signaling phosphatase reveals reciprocal regulation. *Cell* 124: 561–571.

Ge, Y., and Charon, N.W. (1997) Molecular characterization of a flagellar/chemotaxis operon in the spirochete *Borrelia burgdorferi*. *FEMS Microbiol Lett* 153: 425–431.

Greene, S.R., and Stamm, L.V. (1999) Molecular characterization of a chemotaxis operon in the oral spirochete, *Treponema denticola*. *Gene* 232: 59–68.

Kirby, J.R., Kristich, C.J., Saulmon, M.M., Zimmer, M.A., Garrity, L.F., Zhulin, I.B., and Ordal, G.W. (2001) CheC is related to the family of flagellar switch proteins and acts independently from CheD to control chemotaxis in *Bacillus subtilis*. *Mol Microbiol* 42: 573–585.

Kristich, C.J., and Ordal, G.W. (2002) *Bacillus subtilis* CheD is a chemoreceptor modification enzyme required for chemotaxis. *J Biol Chem* 277: 25356–25362.

Lancero, H.L., Castaneda, S., Caberoy, N.B., Ma, X., Garza, A.G., and Shi, W. (2005) Analysing protein–protein interactions of the *Myxococcus xanthus* Dif signalling pathway using the yeast two-hybrid system. *Microbiology* 151: 1535–1541.

Lux, R., Sim, J.H., Tsai, J.P., and Shi, W. (2002) Construction and characterization of a cheA mutant of *Treponema denticola*. *J Bacteriol* 184: 3130–3134.

Muff, T.J., and Ordal, G.W. (2007a) Assays for CheC, FliY, and CheX as representatives of response regulator phosphatases. *Methods Enzymol* 423: 336–348.

Muff, T.J., and Ordal, G.W. (2007b) The CheC phosphatase regulates chemotactic adaptation through CheD. *J Biol Chem* 282: 34120–34128.

Muff, T.J., Foster, R.M., Liu, P.J., and Ordal, G.W. (2007) CheX in the three phosphatase system of bacterial chemotaxis. *J Bacteriol* 189: 7007–7013.

Park, S.Y., Chao, X., Gonzalez-Bonet, G., Beel, B.D., Bilwes,A.M., and Crane, B.R. (2004) Structure and function of an unusual family of protein phosphatases: the bacterial chemotaxis proteins CheC and CheX. *Mol Cell* 16: 563–574.

Park, S.Y., Lowder, B., Bilwes, A.M., Blair, D.F., and Crane, B.R. (2006) Structure of FliM provides insight into assembly of the switch complex in the bacterial flagella motor. *Proc Natl Acad Sci USA* 103: 11886–11891.

Paul, K., and Blair, D.F. (2006) Organization of FliN subunits in the flagellar motor of *Escherichia coli*. *J Bacteriol* 188: 2502–2511.

Rao, C.V., Kirby, J.R., and Arkin, A.P. (2005) Phosphatase localization in bacterial chemotaxis: divergent mechanisms, convergent principles. *Phys Biol* 2: 148–158.

Rosario, M.M., and Ordal, G.W. (1996) CheC and CheD interact to regulate methylation of *Bacillus subtilis* methyl-accepting chemotaxis proteins. *Mol Microbiol* 21: 511–518.

Rosario, M.M., Kirby, J.R., Bochar, D.A., and Ordal, G.W. (1995) Chemotactic methylation and behavior in *Bacillus subtilis*: role of two unique proteins, CheC and CheD. *Biochemistry* 34: 3823–3831.

Rudolph, J., and Oesterhelt, D. (1996) Deletion analysis of the che operon in the archaeon *Halobacterium salinarium*. *J Mol Biol* 258: 548–554.

Saulmon, M.M., Karatan, E., and Ordal, G.W. (2004) Effect of loss of CheC and other adaptational proteins on chemotactic behaviour in *Bacillus subtilis*. *Microbiology* 150: 581–589.

Sim, J.H., Shi, W., and Lux, R. (2005) Protein–protein interactions in the chemotaxis signalling pathway of *Treponema denticola*. *Microbiology* 151: 1801–1807.

Stock, J.B., Levit, M.N., and Wolanin, P.M. (2002) Information processing in bacterial chemotaxis. *Sci STKE* 2002: PE25.

Szurmant, H., and Ordal, G.W. (2004) Diversity in chemotaxis mechanisms among the bacteria and archaea. *Microbiol Mol Biol Rev* 68: 301–319.

Szurmant, H., Bunn, M.W., Cannistraro, V.J., and Ordal, G.W. (2003) *Bacillus subtilis* hydrolyzes CheY-P at the location of its action, the flagellar switch. *J Biol Chem* 278: 48611–48616.

Szurmant, H., Muff, T.J., and Ordal, G.W. (2004) *Bacillus subtilis* CheC and FliY are members of a novel class of CheY-P-hydrolyzing proteins in the chemotactic signal transduction cascade. *J Biol Chem* 279: 21787–21792.

Toker, A.S., and Macnab, R.M. (1997) Distinct regions of bacterial flagellar switch protein FliM interact with FliG, FliN and CheY. *J Mol Biol* 273: 623–634.

Vaknin, A., and Berg, H.C. (2004) Single-cell FRET imaging of phosphatase activity in the *Escherichia coli* chemotaxis system. *Proc Natl Acad Sci USA* 101: 17072–17077.

West, A.H., and Stock, A.M. (2001) Histidine kinases and response regulator proteins in two-component signaling systems. *Trends Biochem Sci* 26: 369–376.

Yang, Z., and Li, Z. (2005) Demonstration of interactions among *Myxococcus xanthus* Dif chemotaxis-like proteins by the yeast two-hybrid system. *Arch Microbiol* 183: 243–252.

Zhao, R., Collins, E.J., Bourret, R.B., and Silversmith, R.E. (2002) Structure and catalytic mechanism of the *E. coli* chemotaxis phosphatase CheZ. *Nat Struct Biol* 9: 570–575.

© 2008 The Authors  
Journal compilation © 2008 Blackwell Publishing Ltd, *Molecular Microbiology*, **70**, 1054–1061