
Biosynthesis of Phosphonic and Phosphinic Acid Natural Products

William W. Metcalf${}^{1,2}$  
and Wilfred A. van der Donk${}^{2,3,4}$  

${}^{1}$Department of Microbiology, ${}^{2}$Institute for Genomic Biology, ${}^{3}$Department of Chemistry  
and ${}^{4}$Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign,  
Urbana, Illinois 61801; email: metcalf@uiuc.edu, vddonk@uiuc.edu  

Annu. Rev. Biochem. 2009. 78:65–94  
The *Annual Review of Biochemistry* is online at biochem.annualreviews.org  
This article’s doi: 10.1146/annurev.biochem.78.091707.100215  
Copyright © 2009 by Annual Reviews. All rights reserved  
0066-4154/09/0707-0065$20.00  

**Key Words**  
antibiotic, bialaphos, fosfomycin, fosmidomycin, FR-900098, phosphinothricin, phosphonates  

**Abstract**  
Natural products containing carbon-phosphorus bonds (phosphonic and phosphinic acids) have found widespread use in medicine and agriculture. Recent years have seen a renewed interest in the biochemistry and biology of these compounds with the cloning of the biosynthetic gene clusters for several family members. This review discusses the commonalities and differences in the molecular logic that lie behind the biosynthesis of these compounds. The current knowledge regarding the metabolic pathways and enzymes involved in the production of a number of natural products, including the approved antibiotic fosfomycin, the widely used herbicide phosphinothricin (PT), and the clinical candidate for treatment of malaria FR-900098, is presented. Many of the enzymes involved in the biosynthesis of these compounds catalyze chemically and biologically unprecedented transformations, and a wealth of new biochemistry has been revealed through their study. These investigations have also suggested new strategies for natural product discovery.

Contents

INTRODUCTION ..... 66
C-P CONTAINING
MACROMOLECULES:
PHOSPHONOLIPIDS,
GLYCANS, AND
PROTEINS ..... 68
Discovery, Distribution, and
Diversity of Phosphonolipids ..... 69
Other C-P-Containing
Macromolecules ..... 69
The Biological Role of
C-P-Containing Macromolecules ..... 69
Biosynthesis of C-P
Macromolecules ..... 70
THE 2-AMINOETHYLPHOSPHO-
NATE BIOSYNTHETIC
PATHWAY ..... 70
Phosphoenolpyruvate Mutase ..... 70
Phophonopyruvate
Decarboxylase ..... 71
AEP Transaminase ..... 71
FOSFOMYCIN ..... 71
Biosynthetic Pathway ..... 72
Phosphonoacetaldehyde
Reductase ..... 73
2-Hydroxyethylphosphonate
Methyltransferase ..... 73
2-Hydroxypropylphosphonate
Epoxidase ..... 75
PHOSPHINOTHRICIN-
CONTAINING
PEPTIDES ..... 76
Biosynthetic Pathway ..... 76
2-Hydroxyethylphosphonate
Dioxygenase ..... 76
Synthesis of
Carboxyphosphoenolpyruvate ..... 79
Carboxyphosphoenolpyruvate
Mutase ..... 79
Nonribosomal Peptide
Synthetases ..... 80
P-Methyltransferase ..... 80
PT Acetyltransferase ..... 82
DISCOVERY AND BIOLOGICAL
ACTIVITY OF FR-900098 ..... 82
The FR-900098 Gene Cluster
and Biosynthetic Pathway ..... 82
Immunity ..... 83
DEHYDROPHOS ..... 84
Biosynthetic Gene Cluster
Sequence ..... 84
TRIPEPTIDE K-26 ..... 84
CONCLUSIONS AND
OUTLOOK ..... 85
Metabolic Diversity of PEP
Mutase-Initiated Pathways ..... 85
Genome Mining of PEP Mutase
Biosynthetic Gene Clusters ..... 85

INTRODUCTION

Phosphonic and phosphinic acids represent an underexploited group of bioactive compounds with great promise for the treatment of human disease. These molecules, generically termed C-P compounds, are similar to phosphate esters and anhydrides, but contain carbon-phosphorus bonds (C-P in phosphonates and C-P-C in phosphinates) in place of one or more oxygen-phosphorus bonds. They are nearly isosteric to the labile phosphate esters but are quite stable and can withstand harsh chemical treatments, such as boiling in a strong acid or base (1). These unique structural and chemical features lend useful biological properties to many phosphonic and phosphinic acids, a feature that has not gone unnoticed. The past few decades have seen extensive use of C-P compounds in medicine and agriculture, and thousands of papers describe their use (2). Prominent man-made examples include glyphosate, one of the most widely applied herbicides in the world, and alendronate, a commonly prescribed treatment

C-P compounds: natural products containing one or more carbon-phosphorus bonds
for osteoporosis. Naturally produced phosphonates were first identified in 1959, opening a new chapter in the biology of phosphorus (3).

Although humans have only recently begun to appreciate the biological potential of C-P compounds, nature has long recognized their value as exemplified by the biosynthesis of a wide variety of phosphonic and phosphinic acids by living organisms. These include phosphorylated macromolecules such as lipids, exopolysaccharides, and glycoproteins as well as a fascinating array of small bioactive molecules, which are the primary focus of this review. In some organisms, these compounds are the dominant phosphorus-containing molecules in the cell. For example, eggs of the freshwater snail *Helisoma* contain 95% of their phosphorus in the form of 2-aminoethylphosphonate-modified phosphoglycans (4), whereas the sea anemone *Tealia* possesses up to 50% of its phosphorus in a variety of phospholipids, phosphoglycans, and phosphoglycoproteins (5). Other organisms, such as the protist *Tetrahymena*, have lower overall levels of phosphonate but still synthesize as much as 30% of their membrane lipids in the form of phospholipids (6). The prevalence of C-P compounds in nature is perhaps best exemplified by the recent discovery that as much as 20% to 30% of the available phosphorus in the world’s oceans consists of phosphonic acids (7). Given that phosphorus is often a limiting nutrient, this fact indicates that C-P compounds play an important, perhaps critical, role in the global environment.

A particularly important group of phosphonate natural products includes more than a dozen small molecules with potent bioactive properties (Figure 1). These include antibiotics, such as fosfomycin, dehydrophos, and plumbemycin; herbicides, such as phos-phinothricin tripeptide (PTT) and phospho-nothrixin; antimalarial compounds, such as FR-900098 and fosmidomycin; and the antihypertensive peptides K-4 and K-26. Members of

the actinobacteria produce the majority of these compounds, but many other microbes including *Bacillus* sp., *Pseudomonas* sp., and some filamentous fungi also synthesize C-P compounds (8).

The bioactivity of both man-made and naturally produced C-P compounds stems from their structural similarity to analogous phosphate esters and carboxylic acids, competing with these analogs for binding to enzyme active sites (Figure 1). Because the affinity of C-P compounds is often high, many phosphonates and phosphinates act as potent competitive inhibitors. Moreover, some C-P compounds such as fosfomycin have functional groups that react with the enzyme, causing irreversible inhibition (9). Given the ubiquitous role of phosphate esters and carboxylic acids in biology, the number of potential targets for phosphate and phosphinate inhibitors is very large. Furthermore, numerous regulatory events are controlled by protein phosphorylation (phosphate esters) and proteolysis (carboxylate chemistry), and these processes are also subject to interference by C-P inhibitors.

An area of particular interest, which has revealed a wealth of novel biochemistry, is the molecular and biochemical characterization of phosphonate and phosphinate metabolism. In the 50 years since the discovery of C-P natural products, our knowledge of their biosynthesis has greatly expanded. Although completely characterized biosynthetic pathways are rare, we are beginning to understand the commonalities and differences involved in the synthesis of several prominent phosphonates and phosphinates. As might be expected from their unique structures, the biosynthetic pathways include a number of unusual and interesting reactions. In this review, we discuss the biosynthesis of several well-characterized bioactive C-P compounds, with particular emphasis on these unusual reactions. We also cover the utility of these studies in natural product discovery and highlight remaining unanswered questions and future directions of the field.

PTT: phosphinothricin tripeptide

www.annualreviews.org • Biosynthesis of C-P Natural Products 67
ANRV378-BI78-04 ARI 29 April 2009 10:3

![Chemical Structures](chemical_structures.png)

**Figure 1**

Structures of naturally occurring small-molecule phosphonates and phosphinates. The P-C bonds are red. The inset illustrates as a representative sampling the structural resemblance between the phosphinate/phosphonate natural products (blue) and the substrates of the enzymes they target shown directly below them (red).

---

### C-P CONTAINING MACROMOLECULES: PHOSPHONOLIPIDS, GLYCANS, AND PROTEINS

Naturally occurring C-P compounds can be roughly divided into small molecules with

68 Metcalf • van der Donk

Discovery, Distribution, and Diversity of Phosphonolipids

In 1959, Horiguchi & Kandatsu (3) reported the first observation of a naturally produced C-P compound, 2-aminoethylphosphonate (AEP) (Figure 1), which was isolated from acid hydrolyzed extracts of protozoa found in the rumen of sheep. AEP was subsequently found in a variety of organisms and shown as the head-group of membrane lipids analogous to phosphatidylethanolamine. These early reports led to a wave of studies demonstrating the presence of phosphonolipids in a wide diversity of organisms (reviewed in References 1, 10, and 11). Phosphonolipids are especially abundant in ciliated protozoa, coelenterates, gastropods, and bivalves. They have also been observed in plants, bacteria, and several vertebrates, including humans; however, in the latter cases, these are almost certainly assimilated from dietary sources, rather than synthesized de novo. Nevertheless, this observation indicates that phosphonates play a role in the metabolism of these organisms as well. The phosphonolipids contain a variety of fatty acid side chains and different phosphonate headgroups, including choline analogs with one, two, or three N-linked methyl groups on AEP. Phosphonolipids with either 2-hydroxyethylphosphonate (HEP) (Figure 1) or 1-hydroxy-2-aminoethylphosphonate have also been identified. 2-Amino-3-phosphonopropionic acid, known as phosphonoalanine (P-Ala or PALA), was detected in acid-hydrolyzed cell extracts of *Tetrahymena*, but it is unclear whether it is derived from a lipid or some other macromolecule (see below).

Other C-P-Containing Macromolecules

Phosphonates have also been observed as a component of exopolysaccharides, which have been generically referred to as phosphoglycans (reviewed in References 1 and 10). Unfortunately, the detailed structure of these complex molecules has rarely been determined, leaving open many questions regarding the nature and role of phosphate-modified carbohydrates. Most phosphoglycans contain AEP, although N-methylated AEP derivatives and HEP have also been observed. Phosphoglycans have been identified in protozoa, invertebrates, snails, and bacteria. Phosphoglycans from sea anemone and *Tetrahymena* do not react with reagents that target primary amines, suggesting that the linkage to the sugar chain was via the amine of AEP; however, it is also possible that the nitrogen of AEP is acetylated. In contrast, the capsular polysaccharide of *Bacteroides fragilis* is modified with AEP via an ester linkage of the phosphonate moiety to the sugar (12).

Both AEP- and HEP-containing phosphoproteins have been characterized from a number of lower eukaryotes (10). In cases where the nature of the phosphonate substituent has been established, these are invariably glycoproteins carrying a phosphoglycan. Thus, although the conclusion is not firmly established, direct phosphonate attachment to the peptide backbone seems not to occur. Finally, phosphonate lipopolysaccharides, with the phosphonate on the lipid, or the glycan, or both, have been identified in some organisms.

The Biological Role of C-P-Containing Macromolecules

The function of phosphonate-containing macromolecules has not been well established in any organism. On the basis of their abundance and distribution, they may play an important role in the organisms that produce them. It has been speculated that the nonhydrolyzable C-P bond enhances the stability of lipids, especially with respect to phospholipases (1, 10). Alternatively, the phosphonate moiety could alter the fluidity of membranes containing phosphonolipids or the structural properties of phosphoglycans. Others have suggested that the molecules may play a role in cell-cell signaling as specific receptor molecules or as phosphorus storage molecules in P-limited environments. To our knowledge, only one experimental study addresses these questions directly, albeit with inconclusive results.

results. Purified AEP-containing polysaccharide from *B. fragilis* is known to promote abscess formation; however, deletion mutants lacking the ability to produce this phosphonoglycan are viable and retain virulence in a rodent model system (13). Thus, the function of these macromolecules remains enigmatic.

**Biosynthesis of C-P Macromolecules**

The biosynthesis of phosphonolipids and other C-P-containing macromolecules has been extensively reviewed elsewhere (10). With the possible exception of K-26 discussed below, all phosphate/phosphonite biosynthetic pathways involve the same initial steps, which were first solved by analysis of AEP biosynthesis in *Tetrahymena*. Therefore, the biosynthesis of this common component of C-P-containing macromolecules and small molecules is discussed first.

**THE 2-AMINOETHYLPHOSPHONATE BIOSYNTHETIC PATHWAY**

The biosynthesis of AEP from phosphoenolpyruvate (PEP) is the shortest known pathway for the construction of a natural phosphonate, requiring just three enzymes: PEP mutase, phosphonopyruvate decarboxylase, and AEP transaminase (Figure 2a). Although orthologs of all three proteins from various sources have been characterized in some detail, at present no studies have reported the characterization of a complete set of enzymes from one organism. Given the absence of AEP biosynthetic enzymes in humans and the noted importance of AEP for various pathogens, these enzymes could be attractive targets for inhibition.

**Phosphoenolpyruvate Mutase**

In vivo labeling studies of the source of the phosphorus atom in AEP in phosphonolipids in *Tetrahymena pyriformis* suggested PEP as a potential precursor, but numerous early attempts to experimentally confirm the putative conversion of PEP to phosphonopyruvate (PnPy) failed (14). In 1988, the groups of Knowles (15), Dunaway-Mariano (16), and Seto (17) first succeeded in isolating the enzyme responsible for this transformation (Figure 2a).

![Figure 2](#)

(a) Biosynthetic pathway for 2-aminoethylphosphonate (AEP). (b) The originally proposed mechanism for phosphoenolpyruvate (PEP) mutase, involving nucleophilic catalysis. In panels b and c, oxygen atoms are colored to illustrate the stereochemistry of the transformations. (c) Most recently proposed mechanism of catalysis for PEP mutase, involving a dissociative process with a metaphosphate intermediate.

thermodynamics of the reaction were shown to favor PnPy by more than 500-fold (16), providing an explanation for the failure of earlier investigations. The equilibrium was not anticipated to lie toward PnPy given the general notion of PEP as a high-energy metabolite, but the inherently stronger P-O bond, compared to a P-C bond, accounts for much of the lower thermodynamic energy of PEP (16). For the enzyme from *T. pyriformis*, the reaction occurs with retention of stereochemistry at phosphorus (18, 19), which suggested the intermediacy of a phosphoenzyme intermediate. This intermediate would be formed with inversion of configuration in the reaction of PEP with an enzyme residue (Figure 2b). Subsequent attack of the pyruvate enolate onto the phosphoenzyme intermediate, again with inversion of configuration, would account for the observed net retention. Indeed, when the first crystal structure of a PEP mutase in complex with the inhibitor oxalate was solved, Asp58 in the active site appeared to be in an orientation in which it could act as the phosphate carrier (20). However, attempts to trap the phosphoenzyme and/or enolate intermediates have not been successful (21), and in a later structure in complex with the product analog sulfopyruvate, Asp58 was located almost 6 Å from the sulfur atom of the inhibitor (22). Furthermore, although the Asp58Asn mutant displayed greatly reduced turnover, it was active and therefore inconsistent with a role of Asp58 as nucleophile. Other candidate nucleophiles in the active site have also been investigated by site-directed mutagenesis, and collectively, they have not identified such a residue (22, 23). Dunaway-Mariano with Herzberg and their coworkers (22) have therefore proposed that PEP mutase catalyzes its reaction through a dissociative process with metaphosphate as an intermediate (Figure 2c), which would be the first example of such a mechanism. Interactions with active site residues are proposed to hold the metaphosphate in place as rotation occurs around the C1-C2 bond of the pyruvate enolate to allow attack of C3 onto the metaphosphate. This proposal would account for the observed

retention of stereochemistry at phosphorus as well as the stereochemistry at C1 of PnPy (24).

### Phosphonopyruvate Decarboxylase

A corollary to the highly unfavorable equilibrium of the PEP mutase reaction is that the biosynthesis of phosphonates via this enzyme requires a highly exergonic ensuing step, which in the case of AEP consists of decarboxylation of PnPy to phosphonoacetaldehyde (PnAA) by phosphonopyruvate decarboxylase (14, 25, 26). The protein from *T. pyriformis* is membrane bound and has been difficult to characterize, but the enzyme from *B. fragilis* has proven to be more amenable to in vitro analysis (27). The orthologous enzyme involved in phosphinothricin (PT) biosynthesis in *Streptomyces hygroscopicus* has also been purified from the native host and characterized biochemically (28). Both proteins are members of the α-ketodecarboxylase family, utilizing thiamin pyrophosphate and a divalent metal ion for catalysis.

### AEP Transaminase

At present, aminotransferases involved in the anabolism of AEP have not been characterized biochemically, but several enzymes catalyzing the same reaction in organisms that use AEP as a source of nutrients have been studied (29–31). These proteins from *Salmonella enterica* (31) and *Pseudomonas aeruginosa* (32) utilize pyridoxal 5′-diphosphate (PLP) as cofactor and pyruvate as ammonium acceptor. The mechanism of the transformation and structure of the enzyme from *Salmonella typhimurium* (30, 31) is similar to that of the α-family of PLP-dependent enzymes (33).

### FOSFOMYCIN

(1R,2S)-Epoxypropylphosphonic acid known as fosfomycin (previously called phosphonomycin) was originally isolated by Merck researchers from *Streptomyces fradiae*,

HPP: 2-hydroxy-
propylphosphonate
MeCbl: methylcobalamin

Streptomyces wedmorensis, and Streptomyces viridochromogenes (34) and was later shown to also be produced by Pseudomonas syringae (35) and Pseudomonas viridiflava (36). In the United States, fosfomycin tromethamine [tris(hydroxymethyl)aminomethane] is used under the name Monurol® and is an FDA-approved drug for the treatment of uncomplicated acute cystitis (urinary tract infections) (37–40). In other countries, e.g., Japan, Germany, Spain, and France, the compound is also used for gastrointestinal infections. The compound has broad-spectrum activity and is effective against both gram-negative and gram-positive bacterial infections in mammals (41). It has activity against methicillin- and vancomycin-resistant Staphylococcus aureus (42, 43) and enterococci (44), and its successful clinical use beyond urinary tract and gastrointestinal infections was recently reviewed (45). Fosfomycin inactivates UDP-N-acetyl-glucosamine-3-O-enolpyruvyltransferase (MurA), an essential enzyme that catalyzes the first committed step in cell wall biosynthesis (46–48) by covalent alkylation of an active site cysteine (49, 50).

**Biosynthetic Pathway**

Pioneering studies from the Seto laboratory (51) on the biosynthesis of fosfomycin, using genetic techniques complemented with in vivo experiments using isotopically labeled precursors, provided evidence for the biosynthetic pathway shown in Figure 3 (8). The first two enzymatic steps are the same as in the biosynthesis of AEP, but the one-carbon homologation of PnAA is unprecedented in biology. On the basis of labeling studies, the methyl group in 2-hydroxypropyl phosphonic acid (HPP) was suggested to be derived from methylcobalamin (MeCbl) (8, 51–53). Supporting the involvement of MeCbl, a mutant S. wedmorensis strain that lacked the ability to synthesize B₁₂ could produce only fosfomycin after addition of MeCbl (52). Initially, a direct conversion of PnAA to HPP via nucleophilic addition of a methyl anion of MeCbl to

![Diagram](attachment://image.png)

**Figure 3**

Originally proposed biosynthetic pathway for fosfomycin biosynthesis (*dashed arrow*) and more recent revised pathway involving FomC and 2-HEP. The proposed origins of the methyl group in HPP are shown in red; all biosynthetic enzymes are shown in blue. Abbreviations: AEP, 2-aminoethylphosphonate; FomC, PnAA reductase; Fom1, PEP mutase; Fom2, PnPy decarboxylase; Fom3, HEP C-methyltransferase; HPP, 2-hydroxypropyl phosphonic acid; HppE, HPP epoxidase; MeCbl, methylcobalamin; PEP, phosphoenolpyruvate; PnAA, phosphonoacetaldehyde; SAM, S-adenosylmethionine.

Metcalf • van der Donk
the aldehyde was proposed (Figure 3, dashed arrow) (8, 53, 54). This proposed role of MeCbl would be fundamentally different from known MeCbl-dependent methyltransferases in which methylation takes place via nucleophilic attack by the substrate at the methyl group of MeCbl (i.e., involves a methyl cation equivalent) (55). More recently, analysis of the gene cluster from S. fradiae suggested an alternative pathway in which PnAA is first reduced to HEP by the enzyme FomC (see below) (56). *fomC* was essential for heterologous production of fosfomycin in *Streptomyces lividans*, and a *fomC* disruption mutant was successfully complemented by the addition of HEP to the media, whereas the compound did not complement fosfomycin production in a *fom3* disrupted clone (57). These findings suggested that HEP is a productive intermediate as the product of FomC and the substrate of Fom3. The revised pathway shown in Figure 3 is consistent with all previously reported studies and also explains why neither PnAA nor AEP could be converted to fosfomycin by *S. lividans* expressing *fom3* and *fom4*, whereas HPP was converted (51).

### Phosphonoacetaldehyde Reductase

The function of FomC from *S. wedmorensis* was recently confirmed in vitro with heterologously expressed enzyme (58). The protein belongs to the group III iron-dependent alcohol dehydrogenase family and uses NADPH as a cofactor. A sequence alignment of FomC with structurally characterized family members revealed several conserved residues, including three ligands to the required Fe²⁺ (59). Typically, a His serves as a fourth ligand; however, a Gln was found in FomC. The orthologous PnAA reductases from the biosynthetic pathways of PT and dehydrophos (see below) were also characterized. The former, PhpC, utilizes NADH and Zn²⁺ for catalysis, whereas the latter, DhpG, utilizes NADPH and Fe²⁺ (58). Sequence comparison and a homology model generated for FomC suggested that phosphonate recognition in these proteins may be achieved

by two conserved residues, Ser157 and Tyr261, that form hydrogen bonds to the oxygens of the phosphonate group in HEP.

### 2-Hydroxyethylphosphonate Methyltransferase

The revised biosynthetic pathway shown in Figure 3 incorporated HEP as an additional intermediate but did not resolve the enigma of how the methyl group, present in the final product, is introduced. The question has changed from how to add a methyl group to an aldehyde to how to insert a methyl group into a C-H bond α to a primary alcohol (Figure 3), a likewise unprecedented transformation in chemistry and biology. Sequence analysis of Fom3, the protein that putatively carries out this reaction, showed that it has two conserved domains (57). The N-terminal domain was identified as a B₁₂-like binding domain, whereas the C-terminal domain shows homology to the radical-SAM protein family (60), containing three conserved Cys residues that serve as ligands to a [4Fe-4S] cluster.

With this information, a chemically feasible mechanism was proposed, as depicted in Figure 4a (61), in which SAM and a reduced [4Fe-4S] cluster form a 5′-deoxyadenosyl radical, as in other radical-SAM proteins (62–64). On the basis of feeding studies with stereospecifically labeled HEP (65), the adenosyl radical would then abstract the *pro-R* hydrogen from the C2 position of HEP. Such hydrogen atom abstractions α to a hydroxyl group by a 5′-deoxyadenosyl radical are well preceded in adenosylcobalamin-dependent enzymes (55). The resulting organic free radical can then react with the methyl group of MeCbl, yielding HPP and cob(II)alamin. This mechanism finds support in model studies demonstrating that organic radicals react with MeCbl to form methylated products (66, 67).

The proposed Fom3 mechanism is consistent with in vivo labeling studies by Hamerschmidt and coworkers (53, 54, 65). When (2,2-²H₂)-HEP was fed to *S. fradiae*, 34% of the fosfomycin produced contained a deuterium

www.annualreviews.org • Biosynthesis of C-P Natural Products 73
(a) Proposed mechanism of methyl transfer to 2-hydroxyethylphosphonate (HEP) catalyzed by Fom3. (b) Proposed mechanisms for conversion of 2-hydroxypropyl phosphonic acid (2-HPP) to fosfomycin by HPP epoxidase (FomE or Fom4). The fate of the hydroxyl oxygen of HPP is shown in blue, and the steps requiring electron input are shown in red. Abbreviation: MeCbl, methylcobalamin.

label (54). Feeding (S)-(2-²H)-HEP or (R)-(2-²H)-HEP resulted in 32% retention of the label in fosfomycin from the (S)-isomer, whereas no label was retained from (R)-(²H)-HEP (65). Moreover, when HEP ¹⁸O-labeled on the hydroxyl oxygen was fed, 50% of the fosfomycin isolated was ¹⁸O-labeled (53). These studies revealed for the first time the unusual nature of the epoxidation step discussed in the next section. In the pathway shown in Figure 3, the label would never be in an exchangeable position, explaining the high level of retention of the ¹⁸O-label with the observed label dilution probably attributed to endogenously biosynthesized HEP. Determining whether this proposed mechanism to achieve methylation of nonactivated carbon centers through stoichiometric use of both SAM and MeCbl is correct will require in vitro reconstitution of enzyme activity.

### 2-Hydroxypropylphosphonate Epoxidase

The last enzyme involved in the biosynthesis of fosfomycin catalyzes another highly unusual reaction, the conversion of (S)-2-HPP to fosfomycin. Initially termed Fom4, the protein has been renamed HPP epoxidase (HppE). In vivo labeling studies established that the epoxide oxygen is not derived from molecular oxygen (68). Conversely, HPP with an ¹⁸O-labeled hydroxyl group was converted by *S. fradiae* into ¹⁸O-labeled fosfomycin (69). Hence, these studies pointed at a unique mechanism of epoxide formation that involves dehydrogenation of a secondary alcohol. Cloning of the gene for the epoxidase in the Seto group (51) allowed Liu and his coworkers (70) to heterologously express the protein from *S. wedmorensis* and demonstrate the dehydrogenative epoxidation activity in vitro. The transformation required Fe(II), O₂, an electron carrier (either a general reductase or catalytic amounts of FMN), and NADH as reductant, but not α-ketoglutarate. At present, the physiological reductase has not been identified.

Extensive mechanistic investigations using spectroscopic techniques, substrate analogs, and mutants show that the enzyme is a nonheme mononuclear iron-dependent oxidase (71–74) that carries out epoxide formation by one of the two mechanisms shown in Figure 4b (74). The exact timing of entry of the two electrons required for the overall reaction distinguishes the two models in which either a Fe(III)-superoxide (I) or a Fe(IV)-oxo species (II) abstracts a hydrogen atom from C1 of the substrate. In both mechanisms, the substrate radical at C1 is believed to undergo a very interesting intramolecular rebound-like reaction. The overall stereochemistry of the epoxidation occurs with net inversion at C1 (54).

This mechanism is supported by a series of crystal structures of the enzyme from *S. wedmorensis* (75). The protein is a member of the cupin superfamily with a characteristic jelly roll β-barrel domain, with the 2 His/1 Glu facial triad iron ligand set, and a structurally unique smaller domain that consists of five short α-helices. Two substrate binding modes were observed in these structures, monodentate coordination of HPP through one of the phosphate oxygens and a bidentate interaction involving both the phosphonate and hydroxyl groups. The latter binding mode induces a conformational change that effectively closes off the active site and may serve to protect reactive intermediates. Support for bidentate HPP binding during catalysis was provided via electron paramagnetic resonance studies with ¹⁷O-labeled substrate and using NO as surrogate for oxygen (74). It has been noted (75) that this bidentate binding of HPP has similarities with the binding geometry of α-ketoglutarate (α-KG) in the cupin family, which is believed to activate the ferrous iron toward reaction with molecular oxygen. In these enzymes, the α-KG cosubstrate provides two electrons in addition to two electrons typically provided by the substrate, allowing the net four-electron reduction of molecular oxygen. In HppE, two exogenous electrons must be supplied for the overall conversion.

HMP: hydroxymethylphosphonate

**PHOSPHINOTHRICIN-CONTAINING PEPTIDES**

PT, a nonproteinogenic amino acid found in a number of peptide antibiotics, is the only known phosphinic acid natural product (Figure 1). In the early 1970s, independent groups in Germany and Japan discovered the compound as a component of a tripeptide antibiotic (PT-Ala-Ala) produced by *S. viridochromogenes* [designated phosphinothricin tripeptide, PTT] (76)] or *S. hygroscopicus* [designated bialaphos (77)]. PT was later found as a component of phosalacine, a PT-Ala-Leu tripeptide produced by *Kitasatospora phosalicina* (78), and of trialaphos (PT-Ala-Ala-Ala), a tetrapeptide produced by *S. hygroscopicus* KSB-1285 (79). PT is a structural analog of glutamate and a potent inhibitor of glutamine synthetase. As a free amino acid, PT has relatively poor antibiotic activity, probably owing to inefficient transport. Many organisms, however, readily take up the peptide versions that are hydrolyzed by cytoplasmic peptidases, releasing the active component, a strategy dubbed a “Trojan Horse” mechanism. Although PTT has excellent antibacterial activity in vitro when minimal media are used, it is a relatively poor therapeutic antibiotic because its activity can easily be counteracted by the presence of glutamine in host tissues. However, because glutamine synthetase plays an essential role in pH homeostasis in plants, PT is an outstanding herbicide, and both the tripeptide and synthetic versions of the monomer are widely used in agriculture (80).

of Thompson (81) and Wohlleben (82) using *S. viridochromogenes* bolstered the understanding of the biosynthetic pathway, especially with regard to synthesis of the tripeptide. In combination, these studies led to a proposed pathway that is substantially similar to that shown in Figure 5, but involving several fewer enzymes and intermediates (83).

The complete PTT gene cluster from *S. viridochromogenes* was recently sequenced and contains 24 genes in a contiguous 33-kb segment of the chromosome (82, 84). The complete *S. hygroscopicus* gene cluster has also recently been sequenced and is nearly identical (W.W. Metcalf & J.A.V. Blodgett, unpublished data). Although the majority of the genes had previously been identified, three new genes designated *phpC*, *phpD*, and *phpE*, whose functions are discussed below, were identified. No other unique genes are required for synthesis of PTT as shown by the ability of the cloned gene cluster to confer antibiotic production on heterologous hosts (84). The availability of these genes allowed genetic and biochemical experiments that led to the revised pathway presented in Figure 5, which includes several unprecedented biochemical transformations (83). The revised pathway involves synthesis of HEP via PEP mutase, PnPy decarboxylase, and PnAA reductase, using genes and enzymes that are homologous to those discussed in the preceding sections. HEP is then converted to phosphinopyruvate (PPA) by a series of unprecedented reactions, some of which are analogous to those of the Embden-Meyerhof-Parnas pathway for glycolysis (Figure 5).

**Biosynthetic Pathway**

Interest in the unique C-P-C bond motif and the biotechnological applications of PT has spurred a large number of studies on its biosynthesis (reviewed in References 8, 80, and 81). The PTT biosynthetic pathway of *S. hygroscopicus* was largely solved by the Seto group (8) using an elegant combination of in vivo labeling, in vitro biochemistry, genetics, and gene cloning. Subsequent studies in the laboratories

**2-Hydroxyethylphosphonate Dioxygenase**

The first unique enzyme in the PT biosynthetic pathway converts HEP to hydroxymethylphosphonate (HMP), an unprecedented apparent excision of a methylene unit (Figure 5). The product of the *phpD* gene from *S. viridochromogenes* was heterologously expressed, purified, and shown to carry out this reaction without any requirements for external reductants or

PEP → PnPy → PnAA → HEP → HMP  
PF → PF-CMP → 3-PG → CPEP → CPnP → PPA → Acetyl-CoA  
AcDM-PTT → AcPPT → PTT  

Figure 5

The most recent proposed biosynthetic pathway for the biosynthesis of PTT. Three highly unusual transformations are shown in blue, and the steps with similarities with glycolytic transformations are in red. For comparison, the corresponding reactions in glycolysis are shown in the box. Abbreviations: AcDMPT, N-acetyl demethylphosphinothricin; AcDM-PTT, N-acetyl demethylphosphinothricin tripeptide; CPEP, carboxyphosphoenolpyruvate; HEP, 2-hydroxyethylphosphonate; HMP, hydroxymethylphosphonate; PEP, phosphoenolpyruvate; PF-CMP, phosphonoformyl-CMP; PnAA, phosphonoacetaldehyde; PnP, phosphonopyruvate; PT, phosphinothricin.

cofactors. Thus, all four electrons required for reduction of O₂ are provided by HEP. Experiments with isotopically labeled substrate showed that the excised carbon is converted to formate and that the hydrogens on Cl of HEP are retained in HMP (85). Experiments using ¹⁸O₂ demonstrated that both HMP and formate contained label, but surprisingly, the label in HMP was substoichiometric, suggesting an exchange process occurred during catalysis. Indeed, experiments performed in H₂¹⁸O also resulted in incorporation of ¹⁸O into HMP. Because both atoms of molecular oxygen end up in the products, the enzyme

has been named 2-hydroxyethylphosphonate dioxygenase (HEPD) (85).

The overall structure of HEPD (85) consists of tandem repeats of a bi-domain architecture with each of the repeats consisting of an all α-helical domain linked to a β-barrel fold characteristic of the cupin superfamily (86). In spite of a lack of appreciable similarity in the primary sequence, each of the two repeats is structurally homologous to the HppE monomer discussed above (75). The β-barrel domain contains the metal ligands of the 2 His-1 Glu facial triad with the metal site occupied by a Cd²⁺ ion in the X-ray structure. The disposition of these ligands within the β-barrel in HEPD is unlike that in HppE as Glu176 is situated on a different β-strand than the equivalent Glu142 of HppE (75), and the spacing between the first two metal ligands in HEPD (HX₄₆E) is unique

as these residues are closely spaced (HX₁₋₄E/D) in other facial triad enzymes (75). As described for HPP in HppE, bidentate binding was observed for HEP, but ligand binding did not induce the large conformational change seen in HppE, possibly because of the crystallization conditions that required high concentrations of Cd²⁺.

Although oxidative scission of carbon-carbon bonds is well documented, the proteins involved typically act on substrates that contain aromatic (87), alkene (88), or 1,2-dihydroxy functionalities (89). Such activating groups are not present in HEP. A working model has been proposed for the HEPD reaction that is based on the isotope labeling studies (85). Upon substrate binding to Fe(II), reaction with O₂ would result in a Fe(III)-[O₂⁻] intermediate (I, Figure 6). This species can abstract a hydrogen

![Figure 6](#fig6)

Proposed mechanism for the conversion of 2-hydroxyethylphosphonate (HEP) to hydroxymethylphosphonate (HMP) by 2-hydroxyethylphosphonate dioxygenase (HEPD). Half-filled circles indicate oxygen atoms derived from O₂ that have exchanged with solvent.
atom from C2 of HEP to generate intermediate II, similar to the mechanism proposed for isopenicillin N synthase (IPNS) and myo-inositol oxygenase (89, 90) and one of the two mechanisms presented in Figure \(4b\) for HppE. Two different pathways have been suggested to complete catalysis and account for the labeling studies. The substrate radical may attack the hydroperoxide, generating a feryl intermediate \((\mathrm{Fe}^{\mathrm{IV}}=\mathrm{O})\) and a hemiacetal (Figure 6). The latter can undergo a retro-Claisen-type C-C bond scission with the incipient negative charge on C1 attacking the electrophilic feryl, either in a concerted step as shown or stepwise with a carbanion intermediate stabilized by metal coordination. This type of C-C bond cleavage between sp³-hybridized carbons is unprecedented, but an Fe(IV)-oxo intermediate would account for the observed exchange with solvent. A second mechanism that explains the observed data involves conversion of intermediate II to the hydroperoxide III, which after a Criegee-type rearrangement would provide the formate ester of HMP. Hydrolysis of this ester would normally occur by attack at the carbonyl carbon, but such a mechanism would not account for the incorporation of oxygen derived from solvent into HMP. The solvent exchange can be explained if hydrolysis took place via attack at C1 as shown in Figure 6. Further studies will be required to distinguish between these or alternative mechanisms and to provide insights into the factors that result in epoxide formation by HppE and C-C bond cleavage by HEPD.

for this conclusion: Mutants lacking \(phpE\) accumulate HMP, and \(phpf\) mutants accumulate aminomethylphosphonate. The transamination of free aldehydes to the corresponding amine has been commonly observed in blocked mutants (83).

The synthesis of carboxyphosphoenolpyruvate (CPEP) was originally suggested by Hidaka et al. (91) to be achieved through a transesterification reaction of PEP using PF; however, current evidence suggests that this is unlikely. Mutant studies have implicated the \(phpH\) and \(phpG\) gene products in the synthesis of CPEP. These proteins are highly homologous to enolase and phosphoglycerate mutase, glycolytic enzymes that are unlikely to catalyze transesterification reactions (83, 92). An alternative model has been proposed based on the sequence similarity in which PhpG catalyzes a Pi/PF exchange reaction similar to that catalyzed by 1,3-diphosphoglycerate-independent phosphoglycerate mutase (93) (Figure 5). Subsequently, 2-phosphonoformyl-glycerate would be dehydrated by PhpH to produce CPEP, in a reaction analogous to that catalyzed by the homologous enolase. It is important to note that for this model to hold, PhpG would need to be initially activated by priming with a reactive form of PF. It has been proposed that this reactive form of PF is phosphonoformyl-CMP (PF-CMP) (Figure 5), which is a phosphonate analog of CTP. Production of PF-CMP from PF and CTP catalyzed by the PhpF protein has been demonstrated in vitro, lending credence to this model (83), which utilizes well-established biochemistry with a little twist.

### Synthesis of Carboxyphosphoenolpyruvate

Conversion of HMP to phosphonoformate (PF) likely occurs via sequential oxidation of HMP to phosphonoformaldehyde and then PF (Figure 5) (83). These reactions are believed to be catalyzed by the products of the \(phpE\) and \(phpf\) genes, respectively, which are members of the NAD(P)-dependent alcohol and aldehyde dehydrogenase families. Although neither enzyme has been characterized in vitro, genetic experiments provided in vivo support

### Carboxyphosphoenolpyruvate Mutase

At about the same time that PEP mutase activity was first demonstrated (see above), Seto and coworkers (94, 95) discovered the interesting related enzyme CPEP mutase. The enzyme from *S. hygroscopicus* was purified to homogeneity from the producer strain and shown to catalyze the conversion of CPEP isolated from the fermentation broth of a mutant strain to phosphinopyruvate (Figure 5). The gene encodingthe protein was subsequently cloned, and the enzyme was expressed heterologously (96). In this study, CPEP was synthesized chemically and demonstrated to be relatively stable. Like the reaction catalyzed by PEP mutase, the conversion of CPEP to phosphinopyruvate first involves a P-O to P-C bond rearrangement reaction that is thought to be energetically unfavorable. For pathways involving PEP mutase, the unfavorable equilibrium is offset by an ensuing energetically favorable or irreversible reaction, such as decarboxylation as discussed above for the biosynthesis of AEP. With CPEP mutase, the product of the rearrangement step, carboxyphosphonopyruvate (CPnPy) (Figure 5), undergoes a decarboxylation without the need of another enzyme, and the decarboxylation reaction cleaves a P-C rather than a C-C bond. To test whether the enzyme assists in this process, Knowles and coworkers (97) synthesized CPnPy and showed it to be a stable molecule, ruling out spontaneous decarboxylation. CPEP mutase did catalyze the decarboxylation of CPnPy to phosphinopyruvate, establishing chemical competence of the compound as an intermediate. However, the catalytic rate constants were much reduced compared to the overall reaction of CPEP to phosphinopyruvate, demonstrating that the intermediate is not kinetically competent to be a free intermediate in the overall process. It was suggested that CPnPy is an intermediate in the enzyme-catalyzed reaction but that it is not released into solution and that its on rate is slow (97).

### Nonribosomal Peptide Synthetases

The PTT gene cluster contains three genes encoding stand-alone nonribosomal peptide synthetase modules. PhsA contains an adenylation domain (A domain) and a peptidyl carrier protein domain (also termed the thiolation or T domain) (98). PhsA was demonstrated to adenylate both N-acetylphosphinothricin (AcPT) and N-acetyl-demethylphosphinothricin (AcDMPT), with the latter displaying somewhat higher activity in pyrophosphate exchange assays (99); nonacylated analogs were not substrates. A mutant strain blocked in *phsA* accumulated AcDMPT, suggesting this amino acid is loaded in the initiation step (100). PhsB and PhsC are larger polypeptides that also contain condensation domains, with PhsB possessing a second T domain with a domain arrangement of T-C-A-T (101). Biochemical studies with all three enzymes isolated from *S. viridochromogenes* showed that PhsB and PhsC can adenylate alanine as well as other small amino acids and transfer the activated amino acid to the thiol of the phosphopantetheine in the peptide carrier domain (99). Although this finding was expected, it is very interesting that these two proteins that both activate Ala have relatively little sequence identity (38%), that the sequences of their A domains predict they would activate Ser (PhsB) and Pro (PhsC), and that they cannot substitute for each other in deletion mutants (101). The mechanism of product release is still unclear. Neither PhsB nor PhsC contain a typical C-terminal thioesterase domain. Two other thioesterase genes, *theA* and *theB*, have been identified in the PTT gene cluster, but disruption of neither gene abolished PTT production (102). Interestingly, a thioesterase motif (GXSXG) was found near the N terminus of PhsA. Mutation of the Ser in this motif to Ala resulted in a mutant that was unable to restore PTT production in a *phsA* mutant strain. How this N-terminal thioesterase domain might release the final product is not known.

### P-Methyltransferase

P-methyltransferase (Pmet or PhpK) is believed to catalyze the transfer of a methyl group to the phosphorus atom of the tripeptide N-acetyl-demethylphosphinothricin tripeptide (AcDM-PTT) (Figure 5). Similar to the origin of the methyl group in fosfomycin discussed above, MeCbl has been suggested as the methyl donor (51, 103). Production of PTT in *S. hygroscopicus* is enhanced by the addition of cobalt salts to the growth medium (8, 104), and a blocked mutant grown in the absence of Co²⁺ accumulated demethylphosphinothricin
Figure 7

Proposed mechanisms for P-methyltransferase. (a) Heterolytic pathway transferring a methyl group from methylcobalamin (MeCbl) as a methyl cation equivalent. (b) Homolytic pathway transferring the methyl group as a radical. Abbreviations: AdoMet, S-adenosylmethionine; Co, cobalt; Co(I), cobalt in the +1 oxidation state; N-AcPT, N-acetylphosphinothricin; N-AcPTT, N-acetyl phosphinothricin tripeptide.

(DM-PT) as well as the demethylphosphinothricin tripeptide (DM-PTT) (105, 106). Subsequent in vivo and whole-cell lysate labeling studies using AcDM-PT or AcDM-PTT as substrate revealed a high level of incorporation of radiolabel from ${}^{14}$CH${}_{3}$-MeCbl into the methyl group of AcPT and AcPTT, whereas the free amino acid DM-PT was not methylated (8, 103, 105). The gene sequence of the P-methyltransferase was subsequently deduced from complementation studies (107). Seto and coworkers (8, 103, 105, 106) concluded that the P-methyltransferase is an unusual MeCbl-dependent enzyme.

Alkylphosphinic acids such as AcDM-PTT can exist in the tetracoordinated or the tautomeric tricoordinated form (108) (Figure 7a). The equilibrium constant for these compounds usually strongly favors the tetracoordinated tautomer (109, 110). Nevertheless, phosphorous acids and phosphonites are known to carry out both electrophilic and nucleophilic reactions (110, 111). The most straightforward mechanism for the methyl transfer to

resulting cob(I)alamin would reset the enzyme  
for another turnover.

**PT Acetyltransferase**

Antibiotic-producing organisms typically require a mechanism of self-resistance. For PTT, this is conferred by the enzyme phosphinothricin acetyltransferase, which N-acetylates either PT or demethylphosphinothricin using acetyl coenzyme A (CoA). The enzyme is encoded by the *bar* gene in *S. hygroscopicus* (112) and the *pat* gene in *S. viridochromogenes* (113), with 85% sequence identity. Interestingly, this resistance mechanism is also required for biosynthesis of PTT. Demethylphosphinothricin is acetylated prior to peptide bond formation and P-methylation.

The potent herbicidal activity of both PTT and PT has led to the development of the *pat* gene as a selectable marker for genetic engineering of plants, and it has been widely used in this capacity. Furthermore, the availability of crops carrying this resistance allele has led to a recent boom in the paired use of PT with recombinant plants in agriculture.

**DISCOVERY AND BIOLOGICAL ACTIVITY OF FR-900098**

FR-900098 is one of a group of N-hydroxypropylphosphonic acids that also includes fosmidomycin, FR-33289, and FR-32863 (Figure 1), which are produced by strains of *Streptomyces rubellomurinus* and *Streptomyces lavendulae* (114–116). These compounds were discovered by researchers at Fujisawa Pharmaceutical Co. (115, 116) in the late 1970s and block the nonmevalonate pathway for isoprene biosynthesis. They inhibit the first committed step catalyzed by deoxyxylulose phosphate reductoisomerase (DXR) (117). Both FR-900098 and fosmidomycin have very low toxicity in animals and humans and excellent antibacterial activity against most gram-negative bacteria (118–120), but neither compound has achieved widespread clinical use. Renewed interest in these antibiotics came

with the genome sequence of *Plasmodium falciparum* (the causative agent of malaria), which revealed the unexpected presence of the target pathway in this eukaryote. Fosmidomycin and FR-900098 are effective antimalarial agents in animal models and have shown promise in early human trials, including against drug-resistant strains (121–123).

**The FR-900098 Gene Cluster and Biosynthetic Pathway**

The FR-900098 biosynthetic gene cluster was identified by screening a large insert fosmid library from *S. rubellomurinus* with degenerate PCR primers designed to amplify PEP mutase-encoding (*ppm*) genes (124). Further screening of *ppm*-positive clones led to the identification of a clone that conferred production of FR-900098 to the heterologous host *S. lividans* and, therefore, contains all of the unique genes needed for synthesis of the antibiotic. Sequence analysis of the clone revealed the presence of genes encoding a putative PEP mutase and a number of genes related to those encoding enzymes of the tricarboxylic acid (TCA) cycle. The predicted functions of these genes, in combination with in vitro biochemistry and mass spectrometric identification of intermediates, provide strong support for the proposed biosynthetic pathway shown in Figure 8.

Unlike other phosphonate biosynthetic pathways, PnPy decarboxylase does not provide the thermodynamic driving force needed to pull the unfavorable PEP mutase reaction. Instead, a homolog of homocitrate synthase, FrbC, catalyzes the exergonic condensation of acetyl-CoA and PnPy to form 2-phosphonomethylmalate. This activity has been verified in vivo using synthetic PnPy as substrate and also in a coupled reaction with the *S. rubellomurinus* PEP mutase (FrbD) using PEP as the starting material (124). Subsequent steps similar to those found in the TCA cycle are predicted to convert this intermediate into 2-oxo-4-phosphonobutyrate, an analog of 2-oxo-glutarate, although this has yet to be demonstrated. An unknown

ANRV 378-BI78-04 ARI 29 April 2009 10:3

![Chemical Reaction Diagram](chemical-reaction-diagram.png)

**Figure 8**

Proposed biosynthetic pathway of the FR-900098 biosynthetic gene cluster on the basis of gene homologies and in vitro biochemistry. Biosynthetic enzymes are in blue, cofactors in red. Abbreviations: Ac-CoA, acetyl coenzyme A; PEP, phosphoenolpyruvate; PPi, pyrophosphate.

host-encoded transaminase is thought to convert this intermediate to 2-amino-4-phosphonobutyrate. FrbH is a two-domain protein containing discrete nucleotidyl-transferase and PLP-dependent decarboxylase/aminotransferase domains that was predicted to convert 2-amino-4-phosphonobutyrate into 3-aminopropylphosphonate. In vitro decarboxylation activity of FrbH requires CTP and produces 5′-CMP-3-aminopropylphosphonate (H. Zhao, personal communication). The biological rationale for this nucleotide modification is unclear; however, the CMP-modified product is competent as a substrate

for in vitro acetylation with FrbF and N-hydroxylation by FrbG, whereas free 3-aminopropylphosphonate is not. Removal of the CMP is catalyzed by FrbI in vitro, although this gene is not required in vivo, suggesting that other cellular phosphodiesterases can fulfill this function as well.

**Immunity**

Interestingly, the FR-900098 gene cluster also includes a gene (*dxrB*) that encodes a homolog of DXR, the target of the antibiotic. The *dxrB* gene was proposed to encode an

www.annualreviews.org • Biosynthesis of C-P Natural Products 83

FR-900098-insensitive allele of DXR and thus to be involved in self-resistance to the antibiotic (124). This idea has yet to be tested experimentally, but if true, the structure of this protein may lend insight into the nature of the interaction of the antibiotic and the enzyme, allowing design of more effective derivatives.

### DEHYDROPHOS

Scientists at Eli Lilly (125) first isolated a phosphonate from *Streptomyces luridus* and named the compound A53868. It has broad-spectrum antibiotic activity, acting on both gram-positive and gram-negative bacteria (125). A structure was reported, but recent synthesis of this structure showed that it could not be correct because it displayed different ${}^{1}$H and ${}^{31}$P NMR spectra than the compound isolated from the producing strain (126). Subsequent spectroscopic studies on the compound purified from bacteria, grown on various isotopically labeled media, resulted in the structure shown in Figure 1; this proposed structure was verified by synthesis (126). Interestingly, the revised structure contains a phosphonate analog of dehydroalanine, which is found in many natural products such as the lantibiotics (127), microcystins (128), and thiostrepton (129). On the basis of the revised structure, the compound was renamed dehydrophos. The reassignment of the structure raises interesting questions with respect to its mode of action. As mentioned for PTT, the peptidic phosphonate antibiotics promote uptake by the target organism(s). Upon translocation, these peptides are hydrolyzed to release the active phosphonate-containing amino acids (130–134). Hydrolysis of the C-terminal peptide linkage of dehydrophos would result in an enamine that is expected to hydrolyze to methyl acetylphosphonate, which is a structural analog of pyruvate and could inhibit pyruvate-utilizing enzymes. By contrast, the reactive vinyl phosphonate moiety in dehydrophos may inhibit a cellular process directly, analogous to the activity of the phosphonotripeptide K-26 (see below) that inhibits the angiotensin-converting enzyme without the requirement of hydrolysis

(135). Future studies should shed light onto these questions.

#### Biosynthetic Gene Cluster Sequence

The dehydrophos biosynthetic gene cluster has been cloned and sequenced using a strategy similar to that described above for PTT and FR-900098 (A.C. Eliot & W.W. Metcalf, unpublished data). The minimal gene cluster, as defined by the smallest fragment capable of conferring dehydrophos production upon heterologous hosts, consists of 20 genes, including ones whose products are highly homologous to PEP mutase, PnPy decarboxylase, and PnAA reductase. Thus, like PTT and fosfomycin, it appears that this antibiotic is synthesized through HEP. Interestingly, the gene cluster does not contain canonical NRPS proteins nor any other proteins with homology to enzymes known to be involved in peptide synthesis. This suggests that a novel mechanism for peptide antibiotic synthesis is used.

### TRIPEPTIDE K-26

The tripeptide K-26 was isolated from *Aspergillus hypotensionis* in a screen for inhibitors of angiotensin-converting enzyme. The compound is a potent inhibitor and shows promise in the treatment of high blood pressure (135). K-26 contains a phosphonate analog of tyrosine as its C-terminal residue that is shared in several analogs produced by *Actinomadura* strains (136, 137) (Figure 1). At present, the biosynthesis of (*R*)-1-amino-2-(4-hydroxyphenyl)ethylphosphonic acid (AHEP) is not well understood, and its gene cluster has not yet been located. Feeding studies with isotopically labeled tyrosines provided a labeling pattern suggesting that, unlike the other phosphonates discussed in this article, K-26 may not be biosynthesized using a PEP mutase reaction to install the C-P bond. Instead, AHEP appears to be derived from Tyr (138). Both the nitrogen of Tyr as well as the hydrogens on the aromatic ring and β-carbon were incorporated into
the AHEP moiety of K-26 isolated from the producing strain. Furthermore, synthetic racemic 1,2-<sup>13</sup>C<sub>2</sub>-AHEP was incorporated with high efficiency into K-26, suggesting it is a discrete precursor. By contrast, isotopically labeled tyramine, the decarboxylation product of tyrosine, was not incorporated (139). Future studies may reveal the molecular logic of P-C bond formation in this interesting compound.

### CONCLUSIONS AND OUTLOOK

Recent years have seen a renewed interest in the biosynthetic pathways of phosphonate and phosphinate natural products. The number of fully sequenced gene clusters has greatly increased, and the biosynthetic pathways have been established for a handful of family members. Although the collective knowledge of this class of compounds is still limited, a number of interesting trends are becoming apparent.

#### Metabolic Diversity of PEP Mutase-Initiated Pathways

Examination of the phosphonate and phosphinate biosynthetic pathways described above reveals a number of common themes (Figure 9). First, owing to the unfavorable thermodynamics of C-P bond formation, a favorable reaction is required in subsequent steps. This driving force can be supplied by decarboxylation, as in the CPEP mutase and PnPy decarboxylase reactions, or by hydrolysis of acetyl-CoA as in the FrbD-catalyzed step of the FR-900098 pathway. This energetic requirement places constraints on the evolution of potential C-P-producing pathways and leaves unanswered questions regarding the synthesis of some known C-P natural products. For example, it has been suggested that phosphonoalanine is made by transamination of PnPy (140); however, the freely reversible nature of this reaction suggests that net synthesis of C-P bonds via this pathway would be problematic unless transamination is followed by another exergonic reaction. Whether additional driving reactions exist is an open question that is relevant to the

potential chemical structures of as yet unknown C-P compounds.

A second common theme is the apparent borrowing of reactions from central metabolic pathways. These include the glycolysis-like reactions of the PTT pathway (described above) and two distinct, but similar, borrowings from the TCA cycle that occur in the PTT and FR-900098 pathways (boxed insets, Figure 9). Accordingly, these pathways include steps for both phosphonates and phosphinates that are analogous to those catalyzed by citrate synthase, aconitase, and isocitrate dehydrogenase. Given the known molecular mimicry of C-P compounds, this is perhaps not surprising. The similarity of these molecules to the analogous carboxylates allows them to be substrates for the TCA cycle enzymes. In some cases, this has led to the evolution of dedicated enzymes for catalyzing these reactions (FrbA, FrbB, FrbD, FrbE, Pmi, and Pms), but in other cases, it is likely that the C-P molecules are sufficiently similar that normal cellular enzymes catalyze the reaction (141–143). A similar argument involving phosphate ester mimicry can be made for the glycolytic pathway discussed above. The ubiquitous pathways for carboxylate and phosphate ester synthesis in biology therefore provide a rich source for the evolution of pathways for the synthesis of novel C-P natural products. In this regard, it is tempting to speculate that nature produces a large number of phosphonate inhibitors simply by evolution from the target pathways. It seems very likely that additional examples will emerge with continued study of these interesting biosynthetic pathways.

#### Genome Mining of PEP Mutase Biosynthetic Gene Clusters

Finally, the PEP mutase reaction has potential utility for discovery of C-P biosynthetic gene clusters. As noted above, the homology of known PEP mutases allows design of degenerate PCR primers that can be used to identify *ppm*-containing clones from genomic libraries. Coupled with the fact that most antibiotic producers cluster the biosynthetic

www.annualreviews.org • Biosynthesis of C-P Natural Products 85
ANRV378-BI78-04 ARI 29 April 2009 10:3

![Diagram](#)

Figure 9

Overview of currently known biosynthetic pathways of phosphonates and phosphinates starting from phosphoenolpyruvate (PEP). The steps resembling tricarboxylic acid (TCA) cycle reactions for phosphonates (*red box*) and phosphinates (*blue box*) are compared with the reactions in the TCA cycle (*green box*). The corresponding homologous enzymes are shown in blue. Common intermediates in the phosphonate biosynthetic pathways are shown in red. Abbreviations: Ac-CoA, acetyl coenzyme A; AEP, aminoethylphosphonate; HEP, 2-hydroxyethylphosphonate; HMP, hydroxymethylphosphonate; PepM, phosphoenolpyruvate mutase; Pmi, phosphinomethylmalate isomerase; Pms, phosphinomethylmalate synthase; PnAA, phosphonoacetaldehyde; P-Ala, phosphonoalanine; Ppd, phosphonopyruvate decarboxylase; PnPy, phosphonopyruvate; PTT, phosphothricin tripeptide.

genes, this PCR assay has allowed the cloning of the genes needed to make PTT, fosfomycin, and FR-900098 (56, 84, 124). Moreover, we have recently cloned and sequenced the genes needed for production of dehydrophos, fosmidomycin, SF2312, rhizoctcin, and plumbemycin via this technique (unpublished data of W.W. Metcalf, A.C. Eliot, S.A. Borisova, R.D. Woodyer, W.A. van der Donk, T.W. Johannes, & H. Zhao, University of Illinois). Only the K-26 genes were not identified by this means; however, given the recent data suggesting that

PEP mutase is not involved in this pathway (138), this finding is not surprising. Of particular interest for the discovery of new antibiotics is the observation that ~5% of randomly isolated actinomycetes possess the *ppm* gene, as defined by the PCR assay (B. Circello & W.W. Metcalf, unpublished). This observation suggests that a wealth of new C-P natural products and metabolic pathways awaits discovery. Thus, the field of phosphonate and phosphinate biology continues to expand, and ample research opportunities remain for interested scientists.

---

### SUMMARY POINTS

1. Phosphonates and phosphinates function by mimicking phosphate esters or anhydrides or carboxylate groups in enzyme substrates. As such, a large number of enzymes can be potential targets of this class of compounds.
2. The number of unprecedented reactions involved in the biosynthesis of fosfomycin, phosphinothricin, and FR-900098 is an indication of the wealth of novel biochemistry used in the biosynthesis of this class of compounds.
3. Phosphoenolpyruvate (PEP) mutase catalyzes the C-P bond-forming step in all naturally occurring phosphonates for which the gene clusters are currently known. Hence, degenerate primers for PEP mutase can be used for the discovery of new phosphonate-encoding gene clusters and new natural products.
4. Given the current commercial use of phosphonates and phosphinates in medicine and agriculture, discovery of new naturally occurring compounds beyond the 20 or so currently known structures may provide an important untapped source of new products for human use.

---

### FUTURE ISSUES

1. On the basis of the large number of metabolic processes that can be targeted with phosphonates, numerous phosphonate- and phosphinate-containing natural products likely remain to be discovered. Future studies will confirm or refute this hypothesis.
2. The fundamentally new radical-SAM strategy for methylation of unactivated carbon centers must be confirmed or disproven by in vitro studies.
3. The molecular logic that dictates epoxidation by HppE and C-C bond cleavage in HEPD is not understood and requires further investigation. Fine-tuning the activities of these enzymes may result in useful catalysts for synthetic purposes.
4. What is the cellular target of dehydrophos?
5. What alternative strategies, which do not rely on the PEP mutase-like reaction, are used in nature to install a C-P bond?

www.annualreviews.org • Biosynthesis of C-P Natural Products 87

# DISCLOSURE STATEMENT

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

# ACKNOWLEDGMENTS

This work was supported by the National Institutes of Health (GM PO1 GM077596). We thank the coworkers in our laboratories as well as collaborators in the groups of Huimin Zhao, Neil Kelleher, and Satish Nair (University of Illinois), who have contributed to the work described in this review.

# LITERATURE CITED

1. Horiguchi M. 1984. Occurrence, identification and properties of phosphonic and phosphinic acids. In *Biochemistry of Natural C-P Compounds*, ed. T Hori, M Horiguchi, A Hayashi, pp. 24–52. Shiga, Jpn: Jpn. Assoc. Res. Biochem. C-P Compd.
2. Kafarski P, Lejczak B. 2001. Aminophosphonic acids of potential medical importance. *Curr. Med. Chem. Anti-Cancer Agents* 1:301–12
3. Horiguchi M, Kandatsu M. 1959. Isolation of 2-aminoethane phosphonic acid from rumen protozoa. *Nature* 184(Suppl. 12): 901–2
4. Miceli MV, Henderson TO, Myers TC. 1980. 2-Aminoethylphosphonic acid metabolism during embryonic development of the planorbid snail helisoma. *Science* 209:1245–47
5. Quin LD. 1965. The presence of compounds with a carbon-phosphorus bond in some marine invertebrates. *Biochemistry* 4:324–30
6. Kennedy KE, Thompson GA Jr. 1970. Phosphonolipids: localization in surface membranes of Tetrahymena. *Science* 168:989–91
7. Clark LL, Ingall ED, Benner R. 1999. Marine organic phosphorus cycling: novel insights from nuclear magnetic resonance. *Am. J. Sci.* 299:724–37
8. Seto H, Kuzuyama T. 1999. Bioactive natural products with carbon-phosphorus bonds and their biosynthesis. *Nat. Prod. Rep.* 16:589–96
9. Wanke C, Amrhein N. 1993. Evidence that the reaction of the UDP-N-acetylglucosamine 1-carboxyvinyltransferase proceeds through the O-phosphothioketal of pyruvic acid bound to Cys115 of the enzyme. *Eur. J. Biochem.* 218:861–70
10. Hilderbrand RL, ed. 1983. *The Role of Phosphonates in Living Systems*. Boca Raton: CRC Press. 207 pp.
11. Moschidis MC. 1985. Phosphonolipids. *Prog. Lipid Res.* 23:223–46
12. Baumann H, Tzianabos AO, Brisson JR, Kasper DL, Jennings HJ. 1992. Structural elucidation of two capsular polysaccharides from one strain of *Bacteroides fragilis* using high-resolution NMR spectroscopy. *Biochemistry* 31:4081–89
13. Coyne MJ, Kalka-Moll W, Tzianabos AO, Kasper DL, Comstock LE. 2000. *Bacteroides fragilis* NCTC 9343 produces at least three distinct capsular polysaccharides: cloning, characterization, and reassignment of polysaccharide B and C biosynthesis loci. *Infect. Immun.* 68:6176–81
14. Warren WA. 1968. Biosynthesis of phosphonic acids in *Tetrahymena*. *Biochim. Biophys. Acta* 156:340–46
15. Seidel HM, Freeman S, Seto H, Knowles JR. 1988. Phosphonate biosynthesis: isolation of the enzyme responsible for the formation of a carbon-phosphorus bond. *Nature* 335:457–58
16. Bowman E, McQueeney M, Barry RJ, Dunaway-Mariano D. 1988. Catalysis and thermodynamics of the phosphoenolpyruvate/phosphopyruvate rearrangement. Entry into the phosphonate class of naturally occurring organophosphorus compounds. *J. Am. Chem. Soc.* 110:5575–76
17. Hidaka T, Mori M, Imai S, Hara O, Nagaoka K, Seto H. 1989. Studies on the biosynthesis of bialaphos (SF-1293). 9. Biochemical mechanism of C-P bond formation in bialaphos: discovery of phosphoenolpyruvate phosphomutase which catalyzes the formation of phosphonopyruvate from phosphoenolpyruvate. *J. Antibiot.* 42:491–94

18. Freeman S, Seidel HM, Schwalbe CH, Knowles JR. 1989. Phosphonate biosynthesis: the stereochemical course of phosphoenolpyruvate phosphomutase. *J. Am. Chem. Soc.* 111:9233–34

19. McQueney MS, Lee SL, Swartz WH, Ammon HL, Mariano PS, Dunaway-Mariano D. 1991. Evidence for an intramolecular, stepwise reaction pathway for PEP phosphomutase catalyzed phosphorus-carbon bond formation. *J. Org. Chem.* 56:7121–30

20. Huang K, Li Z, Jia Y, Dunaway-Mariano D, Herzberg O. 1999. Helix swapping between two alpha/beta barrels: crystal structure of phosphoenolpyruvate mutase with bound Mg²⁺-oxalate. *Structure* 7:539–48

21. Kim J, Dunaway-Mariano D. 1996. Phosphoenolpyruvate mutase catalysis of phosphoryl transfer in phosphoenolpyruvate: kinetics and mechanism of phosphorus-carbon bond formation. *Biochemistry* 35:4628–35

22. Liu S, Lu Z, Jia Y, Dunaway-Mariano D, Herzberg O. 2002. Dissociative phosphoryl transfer in PEP mutase catalysis: structure of the enzyme/sulfopyruvate complex and kinetic properties of mutants. *Biochemistry* 41:10270–76

23. Jia Y, Lu Z, Huang K, Herzberg O, Dunaway-Mariano D. 1999. Insight into the mechanism of phosphoenolpyruvate mutase catalysis derived from site-directed mutagenesis studies of active site residues. *Biochemistry* 38:14165–73

24. Hammerschmidt F, Kählig H. 1992. Incorporation of d-[1-²H₁]glucose into 2-aminoethylphosphonic acid in *Tetrahymena thermophila* and into fosfomycin in *Streptomyces fradiae*. The stereochemical course of a phosphoenolpyruvate mutase-catalyzed reaction. *Liebigs Ann. Chem.* 1201–3

25. Horiguchi M. 1972. Biosynthesis of 2-aminoethylphosphonic acid in cell-free preparations from *Tetrahymena*. *Biochim. Biophys. Acta* 261:102–13

26. Barry RJ, Bowman E, McQueney M, Dunaway-Mariano D. 1988. Elucidation of the 2-aminoethylphosphonate biosynthetic pathway in *Tetrahymena pyriformis*. *Biochem. Biophys. Res. Commun.* 153:177–82

27. Zhang G, Dai J, Lu Z, Dunaway-Mariano D. 2003. The phosphonopyruvate decarboxylase from *Bacteroides fragilis*. *J. Biol. Chem.* 278:41302–8

28. Nakashita H, Watanabe K, Hara O, Hidaka T, Seto H. 1997. Studies on the biosynthesis of bialaphos. Biochemical mechanism of C-P bond formation: discovery of phosphonopyruvate decarboxylase which catalyzes the formation of phosphonoacetaldehyde from phosphonopyruvate. *J. Antibiot.* 50:212–19

29. Lacoste AM, Dumora C, Balas L, Hammerschmidt F, Vercauteren J. 1993. Stereochemistry of the reaction catalyzed by 2-aminoethylphosphonate aminotransferase. A ¹H-NMR study. *Eur. J. Biochem.* 215:841–44

30. Chen CC, Zhang H, Kim AD, Howard A, Sheldrick GM, et al. 2002. Degradation pathway of the phosphonate ciliate: crystal structure of 2-aminoethylphosphonate transaminase. *Biochemistry* 41:13162–69

31. Kim AD, Baker AS, Dunaway-Mariano D, Metcalf WW, Wanner BL, Martin BM. 2002. The 2-aminoethylphosphonate-specific transaminase of the 2-aminoethylphosphonate degradation pathway. *J. Bacteriol.* 184:4134–40

32. Dumora C, Lacoste AM, Cassaigne A. 1983. Purification and properties of 2-aminoethylphosphonate:pyruvate aminotransferase from *Pseudomonas aeruginosa*. *Eur. J. Biochem.* 133:119–25

33. Alexander FW, Sandmeier E, Mehta PK, Christen P. 1994. Evolutionary relationships among pyridoxal-5′-phosphate-dependent enzymes. Regio-specific alpha, beta and gamma families. *Eur. J. Biochem.* 219:953–60

34. Hendlin D, Stapley EO, Jackson M, Wallick H, Miller AK, et al. 1969. Phosphonomycin, a new antibiotic produced by strains of streptomycetes. *Science* 166:122–23

35. Shoji J, Kato T, Hino H, Hattori T, Hirooka K, et al. 1986. Production of fosfomycin (phosphonomycin) by *Pseudomonas syringae*. *J. Antibiot.* 39:1011–12

36. Katayama N, Tsubotani S, Nozaki Y, Harada S, Ono H. 1990. Fosfazecin and fosfocytocin, new nucleotide antibiotics produced by bacteria. *J. Antibiot.* 43:238–46

37. Stein GE. 1998. Single-dose treatment of acute cystitis with fosfomycin tromethamine. *Ann. Pharmacother.* 32:215–19

38. Bailey RR. 1993. Management of lower urinary tract infections. *Drugs* 45(Suppl. 3):139–44

39. Reeves DS. 1992. Treatment of bacteriuria in pregnancy with single dose fosfomycin trometamol: a review. *Infection* 20(Suppl. 4):S313–16

40. Bailey RR. 1990. Brief overview of single-dose therapy for uncomplicated urinary tract infections. *Chemotherapy* 36(Suppl. 1):27–30

41. Reeves DS. 1994. Fosfomycin trometamol. *J. Antimicrob. Chemother.* 34:853–58

42. Nakazawa H, Kikuchi Y, Honda T, Isago T, Nozaki M. 2003. Enhancement of antimicrobial effects of various antibiotics against methicillin-resistant *Staphylococcus aureus* (MRSA) by combination with fosfomycin. *J. Infect. Chemother.* 9:304–9

43. Cassone M, Campanile F, Pantosti A, Venditti M, Stefani S. 2004. Identification of a variant “Rome clone” of methicillin-resistant *Staphylococcus aureus* with decreased susceptibility to vancomycin, responsible for an outbreak in an intensive care unit. *Microb. Drug Resist.* 10:43–49

44. Allerberger F, Klare I. 1999. In-vitro activity of fosfomycin against vancomycin-resistant enterococci. *J. Antimicrob. Chemother.* 43:211–17

45. Falagas ME, Giannopoulou KP, Kokolakis GN, Rafailidis PI. 2008. Fosfomycin: use beyond urinary tract and gastrointestinal infections. *Clin. Infect. Dis.* 46:1069–77

46. Skarzynski T, Mistry A, Wonacott A, Hutchinson SE, Kelly VA, Duncan K. 1996. Structure of UDP-N-acetylglucosamine enolpyruvyl transferase, an enzyme essential for the synthesis of bacterial peptidoglycan, complexed with substrate UDP-N-acetylglucosamine and the drug fosfomycin. *Structure* 4:1465–74

47. Schonbrunn E, Svergun DI, Amrhein N, Koch MH. 1998. Studies on the conformational changes in the bacterial cell wall biosynthetic enzyme UDP-N-acetylglucosamine enolpyruvyltransferase (MurA). *Eur. J. Biochem.* 253:406–12

48. De Smet KA, Kempsell KE, Gallagher A, Duncan K, Young DB. 1999. Alteration of a single amino acid residue reverses fosfomycin resistance of recombinant MurA from *Mycobacterium tuberculosis*. *Microbiology* 145:3177–84

49. Marquardt JL, Brown ED, Lane WS, Haley TM, Ichikawa Y, et al. 1994. Kinetics, stoichiometry, and identification of the reactive thiolate in the inactivation of UDP-GlcNAc enolpyruvyl transferase by the antibiotic fosfomycin. *Biochemistry* 33:10646–51

50. Kim DH, Lees WJ, Kempsell KE, Lane WS, Duncan K, Walsh CT. 1996. Characterization of a Cys115 to Asp substitution in the *Escherichia coli* cell wall biosynthetic enzyme UDP-GlcNAc enolpyruvyl transferase (MurA) that confers resistance to inactivation by the antibiotic fosfomycin. *Biochemistry* 35:4923–28

51. Hidaka T, Goda M, Kuzuyama T, Takei N, Kidaka M, Seto H. 1995. Cloning and nucleotide sequence of fosfomycin biosynthetic genes of *Streptomyces wedmorensis*. *Mol. Gen. Genet.* 249:274–80

52. Kuzuyama T, Hidaka T, Kamigiri K, Imai S, Seto H. 1992. Studies on the biosynthesis of fosfomycin. 4. The biosynthetic origin of the methyl group of fosfomycin. *J. Antibiot.* 45:1812–14

53. Hammerschmidt F. 1994. Incorporation of L-[Methyl-<sup>2</sup>H<sub>3</sub>]methionine and 2-[Hydroxy-<sup>18</sup>O]hydroxyethylphosphonic acid into fosfomycin in *Streptomyces fradiae*—an unusual methyl transfer. *Angew. Chem. Int. Ed. Engl.* 33:341–42

54. Hammerschmidt F, Kählig H. 1991. Biosynthesis of natural products with a P-C bond. 7. Synthesis of [1,1-<sup>2</sup>H<sub>2</sub>]-, [2,2-<sup>2</sup>H<sub>2</sub>]-, (R)-[1-<sup>2</sup>H<sub>1</sub>](2-hydroxyethyl)phosphonic acid and (R,S)-[1-<sup>2</sup>H<sub>1</sub>](1,2-dihydroxyethyl)phosphonic acid and incorporation studies into fosfomycin in *Streptomyces fradiae*. *J. Org. Chem.* 56:2364–70

55. Banerjee R, ed. 1999. *The Chemistry and Biochemistry of B<sub>12</sub>*. New York: Wiley

56. Woodyer RD, Shao Z, Metcalf WM, Thomas PM, Kelleher NL, et al. 2006. Heterologous production of fosfomycin and identification of the minimal biosynthetic cluster. *Chem. Biol.* 13:1171–82

57. Woodyer RD, Li G, Zhao H, van der Donk WA. 2007. New insight into the mechanism of methyl transfer during the biosynthesis of fosfomycin. *Chem. Commun.* (4):359–61

58. Shao Z, Blodgett JA, Circello BT, Eliot AC, Woodyer R, et al. 2008. Biosynthesis of 2-hydroxyethylphosphonate, an unexpected intermediate common to multiple phosphonate biosynthetic pathways. *J. Biol. Chem.* 283:23161–68

59. Montella C, Bellsolell L, Pérez-Luque R, Badía J, Baldoma L, et al. 2005. Crystal structure of an iron-dependent group III dehydrogenase that interconverts L-lactaldehyde and L-1,2-propanediol in *Escherichia coli*. *J. Bacteriol.* 187:4957–66

60. Sofia HJ, Chen G, Hetzler BG, Reyes-Spindola JF, Miller NE. 2001. Radical SAM, a novel protein superfamiliy linking unresolved steps in familiar biosynthetic pathways with radical mechanisms: functional characterization using new analysis and information visualization methods. *Nucleic Acids Res.* 29:1097–106

61. van der Donk WA. 2006. Rings, radicals, and regeneration: the early years of a bioorganic laboratory. *J. Org. Chem.* 71:9561–71

62. Layer G, Heinz DW, Jahn D, Schubert WD. 2004. Structure and function of radical SAM enzymes. *Curr. Opin. Chem. Biol.* 8:468–76

63. Frey PA. 2001. Radical mechanisms of enzymatic catalysis. *Annu. Rev. Biochem.* 70:121–48

64. Frey PA, Magnusson OT. 2003. S-adenosylmethionine: A wolf in sheep’s clothing, or a rich man’s adenosylcobalamin? *Chem. Rev.* 103:2129–48

65. Hammerschmidt F. 1992. Biosynthesis of natural products with a P-C bond 9. Synthesis and incorporation of (S)-2-hydroxy-[2-2H-1]ethylphosphonic and (R)-2-hydroxy-[2-H-2]ethylphosphonic acid into fosfomycin by *Streptomyces fradiae*. *Liebigs Ann. Chem.* 553–57

66. Mosimann H, Kräutler B. 2000. Methylcorrinoids methylate radicals—Their second biological mode of action? *Angew. Chem. Int. Ed. Engl.* 39:393–95

67. Glasenapp-Breiling M, Montforts FP. 2000. Unusual alkylation reactions in the biosynthesis of natural products and elucidation of their reaction mechanisms. *Angew. Chem. Int. Ed. Engl.* 39:721–23

68. Hammerschmidt F, Bovermann G, Bayer K. 1990. Das Oxiran-Sauerstoff-Atom des Fosfomycins entstammt nicht dem Luft-Sauerstoff. *Liebigs Ann. Chem.* 1055–61

69. Hammerschmidt F. 1991. Biosynthesis of natural products with a P-C bond. Part 8: On the origin of the oxirane oxygen atom of fosfomycin in *Streptomyces fradiae*. *J. Chem. Soc. Perkin Trans.* 1:1993–96

70. Liu P, Murakami K, Seki T, He X, Yeung SM, et al. 2001. Protein purification and function assignment of the epoxidase catalyzing the formation of fosfomycin. *J. Am. Chem. Soc.* 123:4619–20

71. Zhao Z, Liu P, Murakami K, Kuzuyama T, Seto H, Liu HW. 2002. Mechanistic studies of HPP epoxidase: configuration of the substrate governs its enzymatic fate. *Angew. Chem. Int. Ed. Engl.* 41:4529–32

72. Liu P, Liu A, Yan F, Wolfe MD, Lipscomb JD, Liu HW. 2003. Biochemical and spectroscopic studies on (S)-2-hydroxypropylphosphonic acid epoxidase: a novel mononuclear nonheme iron enzyme. *Biochemistry* 42:11577–86

73. Yan F, Li T, Lipscomb JD, Liu A, Liu HW. 2005. Site-directed mutagenesis and spectroscopic studies of the iron-binding site of (S)-2-hydroxypropylphosphonic acid epoxidase. *Arch. Biochem. Biophys.* 442:82–91

74. Yan F, Moon SJ, Liu P, Zhao Z, Lipscomb JD, et al. 2007. Determination of the substrate binding mode to the active site iron of (S)-2-hydroxypropylphosphonic acid epoxidase using 17O-enriched substrates and substrate analogues. *Biochemistry* 46:12628–38

75. Higgins LJ, Yan F, Liu P, Liu HW, Drennan CL. 2005. Structural insight into antibiotic fosfomycin biosynthesis by a mononuclear iron enzyme. *Nature* 437:838–44

76. Bayer H, Gugel KH, Hägele K, Hagenmaier H, Jessipow S, et al. 1972. Stofwechselprodukte von Mikroorganismen 98. Phosphinothricin und phosphinothricyl-alanyl-alanin. *Helv. Chim. Acta* 55:224–39

77. Ogawa H, Tsuruoka T, Inouye S, Niida T. 1973. Studies on a new antibiotic SF-1293. *Sci. Rep. Meiji Seika Kaisha* 13:42–48

78. Omura S, Murata M, Hanaki H, Hinotozawa K, Oiwa R, Tanaka H. 1984. Phosalacine, a new herbicidal antibiotic containing phosphinothricin. Fermentation, isolation, biological activity and mechanism of action. *J. Antibiot.* 37:829–35

79. Kato H, Nagayama K, Abe H, Kobayashi R, Ishihara E. 1991. Isolation, structure and biological activity of trialaphos. *Agric. Biol. Chem.* 55:1133–34

80. Thompson CJ, Seto H. 1995. Bialaphos. *Biotechnology* 28:197–222

81. Thompson CJ, Seto H. 1995. Bialaphos. In *Genetics and Biochemistry of Antibiotic Production*, ed. LC Vining, C Stuttard, pp. 197–222. Newton, MA: Butterworth-Heinemann

82. Schwartz D, Berger S, Heinzelmann E, Muschko K, Welzel K, Wohlleben W. 2004. Biosynthetic gene cluster of the herbicide phosphinothricin tripeptide from *Streptomyces viridochromogenes* Tü494. *Appl. Environ. Microbiol.* 70:7093–102

83. Blodgett JA, Thomas PM, Li G, Velasquez JE, van der Donk WA, et al. 2007. Unusual transformations in the biosynthesis of the antibiotic phosphothricin tripeptide. *Nat. Chem. Biol.* 3:480–85

84. Blodgett JA, Zhang JK, Metcalf WW. 2005. Molecular cloning, sequence analysis, and heterologous expression of the phosphothricin tripeptide biosynthetic gene cluster from *Streptomyces viridochromogenes* DSM 40736. *Antimicrob. Agents Chemother.* 49:230–40

85. Cicchillo RM, Zhang H, Blodgett JAV, Whiteck JT, Li G, et al. 2009. An unusual carbon-carbon bond cleavage reaction in the biosynthesis of the antibiotic phosphothricin. *Nature* In press

86. Dunwell JM, Purvis A, Khuri S. 2004. Cupins: the most functionally diverse protein superfamily? *Phytochemistry* 65:7–17

87. Costas M, Mehn MP, Jensen MP, Que L Jr. 2004. Dioxygen activation at mononuclear nonheme iron active sites: enzymes, models, and intermediates. *Chem. Rev.* 104:939–86

88. Grogan G. 2005. Emergent mechanistic diversity of enzyme-catalysed beta-diketone cleavage. *Biochem. J.* 388:721–30

89. Xing G, Diao Y, Hoffart LM, Barr EW, Prabhu KS, et al. 2006. Evidence for C-H cleavage by an iron-superoxide complex in the glycol cleavage reaction catalyzed by myo-inositol oxygenase. *Proc. Natl. Acad. Sci. USA* 103:6130–35

90. Burzlaff NI, Rutledge PJ, Clifton IJ, Hensgens CM, Pickford M, et al. 1999. The reaction cycle of isopenicillin N synthase observed by X-ray diffraction. *Nature* 401:721–24

91. Hidaka T, Hidaka M, Uozumi T, Seto H. 1992. Nucleotide sequence of a carboxyphosphoenolpyruvate phosphomutase gene isolated from a bialaphos-producing organism, *Streptomyces hygroscopicus*, and its expression in *Streptomyces lividans*. *Mol. Gen. Genet.* 233:476–78

92. Babbitt PC, Hasson MS, Wedekind JE, Palmer DR, Barrett WC, et al. 1996. The enolase superfamily: a general strategy for enzyme-catalyzed abstraction of the alpha-protons of carboxylic acids. *Biochemistry* 35:16489–501

93. Potters MB, Solow BT, Bischoff KM, Graham DE, Lower BH, et al. 2003. Phosphoprotein with phosphoglycerate mutase activity from the archaeon *Sulfolobus solfataricus*. *J. Bacteriol.* 185:2112–21

94. Hidaka T, Seto H, Imai S. 1989. Biosynthetic mechanism of C-P bond formation. Isolation of carboxyphosphoenolpyruvate and its conversion to phosphinopyruvate. *J. Am. Chem. Soc.* 111:8012–13

95. Hidaka T, Imai S, Hara O, Anzai H, Murakami T, et al. 1990. Carboxyphosphoenolpyruvate phosphomutase, a novel enzyme catalyzing C-P bond formation. *J. Bacteriol.* 172:3066–72

96. Freeman S, Pollack SJ, Knowles JR. 1992. Synthesis of the unusual metabolite carboxyphosphonoenolpyruvate. Cloning and expression of carboxyphosphoenolpyruvate mutase. *J. Am. Chem. Soc.* 114:377–78

97. Pollack SJ, Freeman S, Pompliano DL, Knowles JR. 1992. Cloning, overexpression and mechanistic studies of carboxyphosphoenolpyruvate mutase from *Streptomyces hygroscopicus*. *Eur. J. Biochem.* 209:735–43

98. Schwartz D, Alijah R, Nussbaumer B, Pelzer S, Wohlleben W. 1996. The peptide synthetase gene *phsA* from *Streptomyces viridochromogenes* is not juxtaposed with other genes involved in nonribosomal biosynthesis of peptides. *Appl. Environ. Microbiol.* 62:570–77

99. Grammel N, Schwartz D, Wohlleben W, Keller U. 1998. Phosphothricin-tripeptide synthetases from *Streptomyces viridochromogenes*. *Biochemistry* 37:1596–603

100. Alijah R, Dorendorf J, Talay S, Puhler A, Wohlleben W. 1991. Genetic analysis of the phosphothricin tripeptide biosynthetic pathway of *Streptomyces viridochromogenes* Tü494. *Appl. Microbiol. Biotechnol.* 34:749–55

101. Schwartz D, Grammel N, Heinzelmann E, Keller U, Wohlleben W. 2005. Phosphothricin tripeptide synthetases in *Streptomyces viridochromogenes* Tü494. *Antimicrob. Agents Chemother.* 49:4598–607

102. Eys S, Schwartz D, Wohlleben W, Schinko E. 2008. Three thioesterases are involved in the biosynthesis of phosphothricin tripeptide in *Streptomyces viridochromogenes* Tü494. *Antimicrob. Agents Chemother.* 52:1686–96

103. Kamigiri K, Hidaka T, Imai S, Murakami T, Seto H. 1992. Studies on the biosynthesis of bialaphos (SF-1293) 12. C-P bond formation mechanism of bialaphos: discovery of a P-methylation enzyme. *J. Antibiot.* 45:781–87

104. Takebe H, Imai S, Ogawa H, Satoh A, Tanaka H. 1989. Studies on the production of bialaphos. (I). Breeding of bialaphos producing strains from a biochemical engineering viewpoint. *J. Ferment. Bioeng.* 67:226–32

105. Imai S, Seto H, Sasaki T, Tsuruoka T, Ogawa H, et al. 1985. Studies on the biosynthesis of bialaphos (SF-1293). 6. Production of N-acetyl-demethylphosphinothricin and N-acetylbialaphos by blocked mutants of *Streptomyces hygroscopicus* SF-1293 and their roles in the biosynthesis of bialaphos. *J. Antibiot.* 38:687–90

106. Seto H, Sasaki T, Imai S, Tsuruoka T, Ogawa H, et al. 1983. Studies on the biosynthesis of bialaphos (SF-1293). 2. Isolation of the first natural products with a C-P-H bond and their involvement in the C-P-C bond formation. *J. Antibiot.* 36:96–98

107. Hidaka T, Hidaka M, Kuzuyama T, Seto H. 1995. Sequence of a P-methyltransferase-encoding gene isolated from a bialaphos-producing *Streptomyces hygroscopicus*. *Gene* 158:149–50

108. Hartley FR. 1990. Introduction. In *The Chemistry of Organophosphorus Compounds*. Vol. 1, ed. FR Hartley, pp. 1–7. New York: Wiley

109. Guthrie P. 1979. Tautomerization equilibria for phosphorous acid and its ethyl esters, free energies of formation of phosphorous and phosphonic acids and their ethyl esters, and pKa values for ionization of the P-H bond in phosphonic acid and phosphonic esters. *Can. J. Chem.* 57:236–39

110. Dahl O. 1996. The preparation and properties of tervalent phosphorus acid derivatives. See Ref. 144, pp. 1–45

111. Edmunson RS. 1996. The synthesis of phosphonic and phosphinic acids and their derivatives. See Ref. 144, pp. 48–144

112. Thompson CJ, Movva NR, Tizard R, Crameri R, Davies JE, et al. 1987. Characterization of the herbicide-resistance gene bar from *Streptomyces hygroscopicus*. *EMBO J.* 6:2519–24

113. Wohlleben W, Arnold W, Broer I, Hillemann D, Strauch E, Pühler A. 1988. Nucleotide sequence of the phosphinothricin N-acetyltransferase gene from *Streptomyces viridochromogenes* Tu494 and its expression in *Nicotiana tabacum*. *Gene* 70:25–37

114. Iguchi E, Okuhara M, Kohsaka M, Aoki H, Imanaka H. 1980. Studies on new phosphonic acid antibiotics. II. Taxonomic studies on producing organisms of the phosphonic acid and related compounds. *J. Antibiot.* 33:19–23

115. Okuhara M, Kuroda Y, Goto T, Okamoto M, Terano H, et al. 1980. Studies on new phosphonic acid antibiotics. I. FR-900098, isolation and characterization. *J. Antibiot.* 33:13–17

116. Okuhara M, Kuroda Y, Goto T, Okamoto M, Terano H, et al. 1980. Studies on new phosphonic acid antibiotics. III. Isolation and characterization of FR-31564, FR-32863 and FR-33289. *J. Antibiot.* 33:24–28

117. Shigi Y. 1989. Inhibition of bacterial isoprenoid synthesis by fosmidomycin, a phosphonic acid-containing antibiotic. *J. Antimicrob. Chemother.* 24:131–45

118. Kuemmerle HP, Murakawa T, De Santis F. 1987. Pharmacokinetic evaluation of fosmidomycin, a new phosphonic acid antibiotic. *Chemioterapia* 6:113–19

119. Tsuchiya T, Ishibashi K, Terakawa M, Nishiyama M, Itoh N, Noguchi H. 1982. Pharmacokinetics and metabolism of fosmidomycin, a new phosphonic acid, in rats and dogs. *Eur. J. Drug Metab. Pharmacokin.* 7:59–64

120. Murakawa T, Sakamoto H, Fukada S, Konishi T, Nishida M. 1982. Pharmacokinetics of fosmidomycin, a new phosphonic acid antibiotic. *Antimicrob. Agents Chemother.* 21:224–30

121. Missinou MA, Borrmann S, Schindler A, Issifou S, Adegnika AA, et al. 2002. Fosmidomycin for malaria. *Lancet* 360:1941–42

122. Wiesner J, Borrmann S, Jomaa H. 2003. Fosmidomycin for the treatment of malaria. *Parasitol. Res.* 90(Suppl. 2):S71–76

123. Jomaa H, Wiesner J, Sanderbrand S, Altincicek B, Weidemeyer C, et al. 1999. Inhibitors of the nonmevalonate pathway of isoprenoid biosynthesis as antimalarial drugs. *Science* 285:1573–76

124. Eliot AC, Griffin BM, Thomas PM, Johannes TW, Kelleher NL, et al. 2008. Cloning, expression, and biochemical characterization of *Streptomyces rubellomurinus* genes required for biosynthesis of antimalarial compound FR-900098. *Chem. Biol.* 15:765–70

125. Hunt AH, Elzey TK. 1988. Revised structure of A53868A. *J. Antibiot.* 41:802

126. Whitteck JT, Ni W, Griffin BM, Eliot AC, Thomas PM, et al. 2007. Reassignment of the structure of the antibiotic A53868 reveals an unusual amino dehydrophosphonic acid. *Angew. Chem. Int. Ed. Engl.* 46:9089–92

127. Chatterjee C, Paul M, Xie L, van der Donk WA. 2005. Biosynthesis and mode of action of lantibiotics. *Chem. Rev.* 105:633–84

128. Rinehart KL, Harada K, Namikoshi M, Chen C, Harvis C, et al. 1988. Nodularin, microcystin, and the configuration of Adda. *J. Am. Chem. Soc.* 110:8557–58

129. Anderson B, Hodgkin DC, Viswamitra MA. 1970. The structure of thiostrepton. *Nature* 225:233–35

130. Leason M, Cunliffe D, Parkin D, Lea PJ, Miflin BJ. 1982. Inhibition of pea leaf glutamine synthetase by methionine sulfoximine, phosphinothricin, and other glutamate analogs. *Phytochemistry* 21:855–57

131. Tachibana K, Watanabe T, Sekizawa Y, Takematsu T. 1986. Action mechanism of bialaphos. I. Inhibition of glutamine synthetase and quantitative changes of free amino acids in shoots of bialaphos-treated Japanese barnyard millet. *Nippon Noyaku Gakkaishi; J. Pestic. Sci.* 11:27–31

132. Atherton FR, Hall MJ, Hassall CH, Lambert RW, Lloyd WJ, Ringrose PS. 1979. Phosphonopeptides as antibacterial agents: mechanism of action of alaphosphin. *Antimicrob. Agents Chemother.* 15:696–705

133. Allen JG, Atherton FR, Hall MJ, Hassall CH, Holmes SW, et al. 1978. Phosphonopeptides, a new class of synthetic antibacterial agents. *Nature* 272:56–58

134. Park BK, Hirota A, Sakai H. 1977. Studies on new antimetabolite produced by microorganism. Part III. Structure of plumbemycin A and B, antagonists of L-threonine from *Streptomyces plumbeus*. *Agric. Biol. Chem.* 41:573–79

135. Yamato M, Koguchi T, Okachi R, Yamada K, Nakayama K, et al. 1986. K-26, a novel inhibitor of angiotensin I converting enzyme produced by an actinomycete K-26. *J. Antibiot.* 39:44–52

136. Koguchi T, Yamada K, Yamato M, Okachi R, Nakayama K, Kase H. 1986. K-4, a novel inhibitor of angiotensin I converting enzyme produced by *Actinomadura spiculosa*. *J. Antibiot.* 39:364–71

137. Kido Y, Hamakado T, Anno M, Miyagawa E, Motoki Y, et al. 1984. Isolation and characterization of I5B2, a new phosphorus containing inhibitor of angiotensin I converting enzyme produced by *Actinomadura* sp. *J. Antibiot.* 37:965–69

138. Ntai I, Manier ML, Hachey DL, Bachmann BO. 2005. Biosynthetic origins of C-P bond containing tripeptide K-26. *Org. Lett.* 7:2763–65

139. Ntai I, Phelan VV, Bachmann BO. 2006. Phosphonopeptide K-26 biosynthetic intermediates in *Astrosporangium hypotensionis*. *Chem. Commun.* 21:4518–20

140. Roberts E, Simonsen DG, Horiguchi M, Kittredge JS. 1968. Transamination of aminoalkylphosphonic acids with alpha ketoglutarate. *Science* 159:886–88

141. Shimotohno K, Seto H, Otake N, Imai S, Satoh A. 1986. Studies on the biosynthesis of bialaphos (SE-1293). 7. The absolute configuration of 2-phosphinomethylmalic acid, a biosynthetic intermediate of bialaphos. *J. Antibiot.* 39:1356–59

142. Seto H, Imai S, Sasaki T, Shimotohno K, Tsuruoka T, et al. 1984. Studies on the biosynthesis of bialaphos (SF-1293). 5. Production of 2-phosphinomethylmalic acid, an analogue of citric acid by *Streptomyces hygroscopicus* SF-1293 and its involvement in the biosynthesis of bialaphos. *J. Antibiot.* 37:1509–11

143. Hidaka T, Shimotohno KW, Morishita T, Seto H. 1999. Studies on the biosynthesis of bialaphos (SF-1293). 18. 2-phosphinomethylmalic acid synthase: a descendant of (R)-citrate synthase? *J. Antibiot.* 52:925–31

144. Hartley FR, ed. 1996. *The Chemistry of Organophosphorus Compounds*. Vol. 4. New York: Wiley

Contents

Preface . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Recognition and Processing of Ubiquitin-Protein Conjugates  
by the Proteasome  
Daniel Finley ……………………………………………………………… 477  

Degradation of Activated Protein Kinases by Ubiquitination  
Zhimin Lu and Tony Hunter …………………………………………… 435  

The Role of Ubiquitin in NF-κB Regulatory Pathways  
Brian Skaug, Xiaomo Jiang, and Zhijian J. Chen ……………………… 769  

Biological and Chemical Approaches to Diseases of Proteostasis  
Deficiency  
Evan T. Powers, Richard I. Morimoto, Andrew Dillin, Jeffery W. Kelly,  
and William E. Balch …………………………………………………… 959  

Gene Expression  

RNA Polymerase Active Center: The Molecular Engine  
of Transcription  
Evgeny Nudler ………………………………………………………… 335  

Genome-Wide Views of Chromatin Structure  
Oliver J. Rando and Howard Y. Chang ………………………………… 245  

The Biology of Chromatin Remodeling Complexes  
Cedric R. Clapier and Bradley R. Cairns ……………………………… 273  

The Structural and Functional Diversity of Metabolite-Binding  
Riboswitches  
Adam Roth and Ronald R. Breaker ……………………………………… 305  

Lipid and Membrane Biogenesis  

Genetic and Biochemical Analysis of Non-Vesicular Lipid Traffic  
Dennis R. Voelker ……………………………………………………… 827  

Cholesterol 24-Hydroxylase: An Enzyme of Cholesterol Turnover  
in the Brain  
David W. Russell, Rebekkah W. Halford, Denise M.O. Ramirez, Rahul Shah,  
and Tiina Kotti ………………………………………………………… 1017  

Lipid-Dependent Membrane Protein Topogenesis  
William Dowhan and Mikhail Bogdanov ………………………………… 515  

Single-Molecule Studies of the Neuronal SNARE Fusion Machinery  
Axel T. Brunger, Keith Weninger, Mark Bowen, and Steven Chu ………… 903  

Mechanisms of Endocytosis  
Gary J. Doherty and Harvey T. McMahon ……………………………… 857  

Recent Advances in Biochemistry

Motors, Switches, and Contacts in the Replisome  
Samir M. Hamdan and Charles C. Richardson …………………… 205  

Large-Scale Structural Biology of the Human Proteome  
Aled Edwards ……………………………………………………………… 541  

Collagen Structure and Stability  
Matthew D. Shoulders and Ronald T. Raines …………………… 929  

The Structural and Biochemical Foundations of Thiamin Biosynthesis  
Christopher T. Jurgenson, Tadhg P. Begley, and Steven E. Ealick … 569  

Proton-Coupled Electron Transfer in Biology: Results from  
Synergistic Studies in Natural and Model Systems  
Steven Y. Reece and Daniel G. Nocera ……………………………… 673  

Mechanism of Mo-Dependent Nitrogenase  
Lance C. Seefeldt, Brian M. Hoffman, and Dennis R. Dean ………… 701  

Inorganic Polyphosphate: Essential for Growth and Survival  
Narayana N. Rao, María R. Gómez-García, and Arthur Kornberg …… 605  

Essentials for ATP Synthesis by F₁F₀ ATP Synthases  
Christoph von Ballmoos, Alexander Wiedenmann, and Peter Dimroth … 649  

The Chemical Biology of Protein Phosphorylation  
Mary Katherine Tarrant and Philip A. Cole …………………………… 797  

Sphingosine 1-Phosphate Receptor Signaling  
Hugh Rosen, Pedro J. Gonzalez-Cabrera, M. Germana Sanna, and Steven Brown … 743  

The Advent of Near-Atomic Resolution in Single-Particle Electron  
Microscopy  
Yifan Cheng and Thomas Walz ………………………………………… 723  

Super-Resolution Fluorescence Microscopy  
Bo Huang, Mark Bates, and Xiaowei Zhuang ………………………… 993  

Indexes  

Cumulative Index of Contributing Authors, Volumes 74–78 ………… 1041  
Cumulative Index of Chapter Titles, Volumes 74–78 ………………… 1045  

Errata  

An online log of corrections to *Annual Review of Biochemistry* articles may be found at  
http://biochem.annualreviews.org/errata.shtml
