
DOI: 10.1002/cbic.200900323

# Of Two Make One: The Biosynthesis of Phenazines

Matthias Mentel,${}^{[b]}$ Ekta G. Ahuja,${}^{[a]}$ Dmitri V. Mavrodi,${}^{[c]}$ Rolf Breinbauer,${}^{[d]}$ Linda S. Thomashow,${}^{[c,\,e]}$ and Wulf Blankenfeldt*${}^{[a]}$

Dedicated to Roger S. Goody on the occasion of his 65th birthday.

## Introduction

Physicians of the 19th century were familiar with the conspicuous occurrence of “blue pus”, which they sometimes observed in patients with severe purulent wounds. Even older are reports of and folk remedies against “blue milk”, a coloration that sometimes developed in fresh milk after some days. Key insight into these phenomena was provided in 1859—exactly 150 years ago—when Mathurin-Joseph Fordos, at a session of the Société d’Émulation pour les Sciences Pharmaceutiques, reported the isolation of the blue pigment “pyocyanine” (from the Greek words for “pus” and “blue”; pyocyanine is nowadays more commonly spelled as pyocyanin) from purulent wound dressings by chloroform extraction.[1] Pyocyanin (5-N-methyl-1-hydroxophenazinium betaine) was the first example of a phenazine natural product, a compound class that has grown to well over 100 members since this first report. Due to the improved understanding of their importance to the phenazine-generating and also to commensal species, the phenazines have been reviewed several times in recent years.[2–5] Here, we provide a historical perspective of the more than 100 years of research that led us to our current picture of the interesting biosynthesis of phenazine natural products.

The details of Fordos’ pyocyanin isolation method, chloroform extraction followed by acidification and partition into an aqueous phase, were published one year later and are still in use today,[6] but it took until 1882 for the French pharmacist Carle Gessard to show that the blue coloration in pus was due to the presence of a microorganism that he then termed *Bacillus pyocyaneus*.[7] *B. pyocyaneus* is nowadays known as *Pseudomonas aeruginosa*, and the Latin term still reflects this strain’s capacity to secrete colored compounds in the modern name: “aerugo” is the Latin word for verdigris, the blue-green coating that develops on copper after long exposure to air. *P. aeruginosa* is an important human opportunistic pathogen responsible for a large number of nosocomial infections, and it is also the main cause of low life expectancy in patients with cystic fibrosis due to chronic infections of the lungs.[8] The production of pyocyanin is used both for identification in the clinic and as a reporter signal in *P. aeruginosa* research until today. The occurrence of blue milk, on the other hand, is probably due to an environmental strain of *P. fluorescens*, and it is not yet clear if this coloration also is a consequence of phenazine production.

Gessard’s discovery of *P. aeruginosa* was resonated in many publications from the medical field, but it required more than

50 years before the correct chemical structure of pyocyanin was established. The chemical composition was first studied by Ledderhose,[9] who derived a formula that was later corrected by McCombie and Scarborough[10] and by Wrede and Strack.[11] Wrede and Strack were also the first to discover a phenazine moiety in a breakdown product of pyocyanin,[12] but their studies were hampered by the fact that they could only obtain a defined molecular weight when working in glacial acetic acid, under which circumstances they obtained a pyocyanin dimer.[13] This dimer was questioned by the results of electrochemical studies by Elema[14] and by Friedheim and Michaelis,[15] before Hillemann finally derived the correct structure in 1938.[16] In retrospect, it seems possible that the conditions employed by Wrede and Strack induced a 1:1 charge-transfer complex of reduced and oxidized pyocyanin, similar to the phenazine derivative chlororaphin, which is also produced by *P. aeruginosa* (Figure 1). Jensen and Holten later measured the dipole moment of pyocyanin and found that its zwitterionic mesomer is present in considerable amounts.[17]

In the course of these studies, it became clear that pyocyanin is a redox-active compound that changes its color depending on pH and oxidation state. This also explained the “chameleon phenomenon”, which describes a temporary color change of *P. aeruginosa* cultures on solid media after exposure to air by disturbance with a platinum needle.[18] Since the first isolation by Fordos, more than 100 phenazine derivatives modified at all positions of the ring system and colored in all shades of

[a] Dr. E. G. Ahuja, Dr. W. Blankenfeldt  
Department of Physical Biochemistry  
Max-Planck-Institute of Molecular Physiology  
Otto-Hahn-Strasse 11, 44227 Dortmund (Germany)  
Fax: (+49) 231-1332399  
E-mail: wulf.blankenfeldt@mpi-dortmund.mpg.de  

[b] Dr. M. Mentel  
Institute of Organic Chemistry, University of Leipzig  
Johannisallee 29, 04103 Leipzig (Germany)  

[c] Dr. D. V. Mavrodi, Dr. L. S. Thomashow  
Department of Plant Pathology, Washington State University  
Pullman, WA 99164-6430 (USA)  

[d] Prof. Dr. R. Breinbauer  
Institute of Organic Chemistry, Graz University of Technology  
Stremayrgasse 16, 8010 Graz (Austria)  

[e] Dr. L. S. Thomashow  
USDA, Agricultural Research Service  
Root Disease and Biological Control Unit  
Pullman, WA 99164-6430 (USA)

CHEMBIOCHEM

W. Blankenfeldt et al.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Figure 1. A collection of naturally occurring phenazine derivatives. The core phenazines PCA and PDC (box, second row) are precursors of other bacterial phenazines. Streptomycetes generate more complex derivatives (third row). Methanophenazine (fourth row) is the only archaeal phenazine known to date and could arise through a different biosynthetic route than that of the bacterial derivatives.

the rainbow have been obtained from microbial sources, and thousands of derivatives have been synthesized.[2,19] Strains of *Pseudomonas* and *Streptomyces* are the most prolific phenazine producers, and the latter tend to generate more complex molecules. A collection of phenazine derivatives depicting different types of substitution and their provenience is shown in Figure 1.

Even if some strains produce copious amounts of phenazines, reaching gram quantities per liter of culture in some cases, the physiological importance of these compounds has still not been fully understood.[3] While some of the biological effects of phenazine derivatives might be associated with their capacity to intercalate DNA, it seems clear that most of their actions are linked to their redox activity. For example, phenazines can reduce molecular oxygen to toxic reactive oxygen species, which explains their broad-spectrum antibiotic activity and is one of the reasons why phenazines act as important virulence factors in infectious disease.[20] In the case of *P. aeruginosa* lung infection, for example, it has been shown that pyocyanin-deficient strains provoked a less pronounced inflammatory response and were also attenuated in their virulence.[21] The fact that phenazine-producing strains excrete these compounds into the environment was for a long time discussed as a trait that provides a competitive advantage against other microorganisms. In fact, culture extracts from *P. aeruginosa* were employed under the name “pyocyanase” to treat bacterial infections before β-lactam antibiotics were discovered.[22] While pyocyanase was later abandoned due to not being reliable and showing side effects,[23] phenazine-producing bacterial strains are still investigated for the biological control of soil-borne pathogens.[24] However, the idea that phenazines function primarily as antibiotics might need revision since recent studies have revealed that these metabolites also play an important role in the physiology of phenazine-producing bacteria. For example, Hernandez et al. suggested that excreted phenazines contribute to bacterial iron acquisition by reducing Fe³⁺-containing minerals to more soluble Fe²⁺ forms, which can then be taken up by other siderophores.[25] Some phenazines could also be important within the cell of the phenazine producer itself. It has recently been demonstrated that pyocyanin acts as a signaling molecule in *P. aeruginosa* by directly activating the iron-containing transcription factor SoxR, thereby controlling numerous genes involved in transmembrane transport and oxidative stress response.[26] Under anaerobic growth conditions, for example, in the deeper layers of biofilm, pyocyanin could also be necessary to reoxidize NADH to sustain glycolysis,[27] and it is therefore not surprising that less viable bacteria were recovered in a mouse model of lung infection with pyocyanin-deficient *P. aeruginosa* strains.[21] The function of phenazines as intracellular electron transporters seems to

2296 www.chembiochem.org © 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim ChemBioChem 2009, 10, 2295–2304

be also reflected in the discovery of the membrane-anchored derivative methanophenazine in Methanosarcina mazei Gö1 (Figure 1), where it plays an important role in membrane-bound electron transport processes. [28]

New insight into the physiological roles of phenazines is currently paralleled by progress in understanding details of their biosynthesis at the molecular level, which will be introduced in the following sections.

### Phenazine Biosynthesis

#### Precursors of the phenazine moiety

Before the enzymes required for phenazine biosynthesis were discovered in the 1990s, most efforts toward an understanding of phenazine biosynthesis were directed toward the identification of phenazine precursor molecules. This work has been summarized in an excellent review by Turner and Messenger, [19] and only the most important milestones will be detailed here.

Research in the second half of the 19th century was occupied with investigating the influence of culture media composition on the microbial production of pyocyanin, which was the only phenazine derivative known at that time. These studies were compounded by the fact that work was pursued with different clinical isolates and with rich solid or liquid media of poorly defined composition, but they clearly demonstrated that culture conditions and provenience of the bacterial isolate are of key influence on the amount of pyocyanin formed. They also indicated that *P. aeruginosa*, in addition to pyocyanin, produces a variety of other colored compounds, most importantly a green fluorescent dye [29] now known as pyoverdine, a siderophore not related to pyocyanin. A detailed overview of these findings was provided by Edwin O. Jordan in 1899, who was also the first to employ defined synthetic media to study pyocyanin production, [30] following the example of Gessard, who had done similar work to study pyoverdine fluorescence several years earlier. [29] Jordan found that pyocyanin production required neither phosphate nor sulfate and occurred in nonproteinid as well as in proteid media. His results were later revised by more systematic studies showing that pyocyanin production is stimulated by a mixture of glycine, leucine and d/l-alanine together with low concentrations of K₂HPO₄, MgSO₄ and FeSO₄, [31,32] and even simpler media with 2-ketoglucolate as the sole carbon source were developed by Ingledew and Campbell. [33] King et al. demonstrated that, in contrast to other pseudomonads, [34] phenazine formation in *P. aeruginosa* is inhibited by high concentrations of phosphate [35] and applied this information in the development of diagnostic King’s A (peptone, MgCl₂ and K₂SO₄) and King’s B (peptone, K₂HPO₄ and MgSO₄) media, which detect the production of pyocyanin and siderophores, respectively, and are still used to identify *P. aeruginosa*. The availability of suitable minimal media also made experiments with radioactively labeled nutrients possible. Blackwood and Neish used this to show that ¹⁴C from glycerol and dihydroxyacetone was accumulated in pyocyanin and that alanine, leucine and isoleucine were preferred amino acid substrates for phenazine biosynthesis. [36] Finally, Korth demonstrated that optimal phenazine production in various strains of *Pseudomonas* required trace amounts of iron and that nonaerated cultures produced much higher levels of pigment. [37] While the requirement for iron had already been noted several years earlier, [32] the observation of increased phenazine production in nonaerated cultures anticipated more recent work highlighting the physiological importance of these pigments as “respiratory compounds” by Price-Whelan et al. [27] and by Dietrich et al. [38]

However, all early studies concentrated on rather common nutrients, and none of them provided direct insight into the immediate precursors of the phenazine moiety. This situation only improved after new methodology developed in the late 1940s enabled the selection of growth-arrested mutants of *E. coli* with penicillin, which led to the discovery of shikimic acid as a precursor of many microbial aromatic compounds [39] and also enabled the production of radioactively labeled shikimic acid for feeding experiments. In 1962, inspired by two previous studies suggesting anthranilate as a precursor, [40,41] Millican used ¹⁴C-labeled shikimic acid to show that this compound was efficiently incorporated into pyocyanin, whereas radioactive anthranilate was not; this indicated that pyocyanin biosynthesis branches off from the shikimate pathway at a stage before anthranilate is formed. [42] This was also confirmed for various other phenazine derivatives, [43–46] indicating that all of these molecules originate through the same or at least related pathways and providing a molecular basis for the growing number of phenazine derivatives found in Nature.

A general problem of the first studies with ¹⁴C-labeled shikimate arose from the fact that large amounts of the precursor were degraded before being incorporated into phenazines. As a consequence, only a minor fraction of radioactivity was recovered, so that the attractive hypothesis that the symmetrical phenazine moiety originates from two shikimate molecules could not ultimately be confirmed. This situation was alleviated in 1969 when Ingledew and Campbell employed a *P. aeruginosa* mutant unable to degrade quinic acid and showed the virtually quantitative incorporation of shikimate into pyocyanin. [47]

Five chemical steps are required to convert shikimate into anthranilate, and consequently, five intermediates (shikimate, shikimate-5-phosphate, 3-enolpyruvyl-shikimate-5-phosphate, chorismate and 2-amino-2-deoxyisochorismate) are possible branch points in phenazine biosynthesis. By extending the concept of degradation-inhibited mutants further, Calhoun et al. [48] and Longley et al. [49] in 1972 simultaneously identified chorismic acid as the common precursor of pyocyanin and other phenazine pigments. However, their evidence was only indirect since chorismic acid is not efficiently transported across the bacterial cell wall such that radioactive chorismic acid could not be used for these experiments.

### Early understanding of phenazine ring assembly

The first insight into the molecular details of phenazine ring assembly was gained by Podojil and Gerber, who employed position-specific [1,6-¹⁴C]-labeled shikimic acid and assessed the ra-

dioactivity of iodinin degradation products.[50] Iodinin is a phenazine derivative from *Brevibacterium iodinum* and owes its name to the fact that it forms insoluble purple crystals reminiscent of iodine.[51] Podojil and Gerber were attracted to study this derivative because of its symmetrical structure (Figure 1) and because of the possibility that the hydroxo groups at C1 and C6 could have been derived from shikimate itself, indicating that iodinin and pyocyanin biosynthesis follow principally different routes.[45] While their work provided independent evidence for the incorporation of two shikimate-derived molecules into phenazines, they concluded, in discordance with the current view, a diagonal but asymmetric pairing scheme of the two precursors, which probably was a consequence of the imperfect separation of radioactive products in their method of degrading phenazine derivatives to CO₂ and pyrazine (Scheme 1 A). Their degradation method was nevertheless employed in a number of follow-up studies that focused on different phenazine derivatives from various strains and employed shikimic acid labeled at different positions. As a consequence of the imperfect methodology used to recover the radioactive label, three different pairing schemes for the two shikimate moieties existed until 1973 (Scheme 1 B).[50,52,53] when Hollstein and McCamey showed that a diagonal symmetrical pairing of the precursors agreed best with data from experiments performed with both [1,6-¹⁴C]- and [6-¹⁴C]-labeled shikimic acid.[54] Ultimate proof of this scheme was difficult due to the symmetrical nature of the phenazine moiety, which made it impossible to distinguish between label incorporation with the groups C2, C3, C7 and C8; C1, C4 and C6 and C9; and C4a, C5a, C9a and C10a. This problem was resolved in 1976, when Herbert et al.

measured deuterium exchange rates of iodinin produced from [2-²H]-labeled shikimic acid, showing that the label was incorporated at two equivalent positions, providing new evidence for a diagonal symmetrical pairing of two shikimate-derived precursors.[55] Further support was provided by Hollstein et al., who employed NMR spectroscopy to show ¹³C incorporation at C5a and C10a in iodinin from feeding experiments with [6-¹³C]-labeled shikimic acid.[56]

### Phenazine biosynthesis genes

Genes involved in the biosynthesis of pyocyanin in *P. aeruginosa* were first reported in 1990 by Essar et al., who recognized the presence of an additional anthranilate synthase in the genome of *P. aeruginosa*, the deletion of which caused dramatically reduced levels of pyocyanin.[57] In contrast to the preceding work by others published nearly 30 years earlier, this led the authors to conclude that anthranilate was a precursor of pyocyanin, and they consequently named the corresponding anthranilate synthase genes *phnA* and *phnB*. While these data led to some confusion in follow-up studies of phenazine biosynthesis, the conflicting evidence of anthranilate having an important role in phenazine biosynthesis of *P. aeruginosa* can now be resolved since it has recently been recognized that the biosynthesis of pyocyanin is under the tight control of a specific small signaling molecule termed *Pseudomonas* quinolone signal (PQS).[58] PQS is anthranilate-derived and produced under the participation of the *phnAB* genes,[59] which might account for the lower pyocyanin production in Essar's study.

---

**Scheme 1. Early understanding of phenazine ring assembly.**

A) Controlled chemical degradation of iodinin for the study of label incorporation from shikimic acid developed by Podojil and Gerber.[50]

B) Different pairing schemes of the two shikimate-derived precursors in phenazines from work with [1,6-¹⁴C]-labeled shikimic acid (*).[50,52,54] The correct orientation for diagonal symmetric pairing is indicated by a box.

True progress was made when Pierson et al. first isolated and later sequenced part of the phenazine biosynthesis gene cluster from *P. aureofaciens* 30-84. Amongst the genes discovered was an anthranilate-synthase-related continuous open reading frame then termed *phzB* (now known as *phzE*), which immediately questioned previous findings about the importance of the discontinuous *phnAB* genes in phenazine biosynthesis. Even if a promoter region of the cluster was not identified and most of the initial functional assignments for the discovered open reading frames had to be refined in the meantime, this study nevertheless opened the door for still ongoing work towards the elucidation of the molecular mechanisms in phenazine biosynthesis recounted in the next paragraph. Mavrodí et al. completed the sequencing of the gene cluster in *P. aureofaciens* and also in *P. fluorescens* 2-79 in 1998, showing that it forms a defined operon containing the seven genes *phzABCDEF*. *P. aeruginosa* was found to contain two copies of the cluster, which could be one reason why it is one of the most active phenazine producers.

The *phz* operon has in the meantime been confirmed in all phenazine-producing Gram-negative and -positive bacteria, and a recent study provided evidence that the gene cluster has been spread by horizontal gene transfer. This is a strong indication that phenazine biosynthesis follows the same principles in all bacterial phenazine producers, and that the large diversity of phenazine derivatives found in Nature is the consequence of different modifications applied to a limited number of precursor molecules. These “core” phenazine molecules are probably phenazine-1-carboxylic acid (PCA) and phenazine-1,6-dicarboxylic acid (PDC); first, because PCA is the only phenazine derivative produced by *P. fluorescens* 2-79, the strain with the simplest *phz* operon, and second, because PDC is an attractive precursor for the symmetrical phenazines known. In fact, several earlier studies showed high incorporation of PCA or PDC and their methyl esters into strain-specific phenazines (summarized by Turner and Messenger). However, the Phz enzymes of *P. fluorescens* 2-79 were found not to convert PDC into PCA, which indicates that PDC is not a direct precursor of PCA. The mechanistic foundations of this unexpected observation will be discussed below.

A compilation of phenazine biosynthesis clusters from various bacteria is shown in Figure 2. From this collection, it can be concluded that five genes are required for core phenazine biosynthesis, namely *phzB*, *phzD*, *phzE*, *phzF*, and *phzG*. Interestingly, all phenazine-producing pseudomonads possess an immediately neighboring and approximately 80% identical copy of the *phzB* gene termed *phzA*. In addition, they and some other bacteria carry the *phzC* gene, which encodes a type-II 3-deoxy-D-arabinooheptulosonate 7-phosphate (DAHP) synthase. DAHP synthases catalyze the first step of the shikimate pathway.

---

![Figure 2](#)

**Figure 2.** Compilation of bacterial phenazine biosynthesis clusters. Colors indicate probable function (red: core phenazine biosynthesis, green: phenazine modification, yellow: regulation, cyan: transport and resistance). Note that the DAHP synthase *PhzC* is not conserved in all *phz* operons and that the *phzA/phzB* gene duplication is restricted to pseudomonads.

---

ChemBioChem 2009, 10, 2295–2304  
© 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
www.chembiochem.org 2299

CHEMBIOCHEM

way, a condensation of phosphoenol pyruvate and erythrose-4-phosphate, upstream of phenazine biosynthesis (Scheme 2). Whereas bacterial isoenzymes normally belong to the type-I group and are feedback-inhibited by aromatic amino acids, PhzC is rather plant-like and lacks a loop region required for allosteric control.[69] Since phenazine biosynthesis is under a cell-density-dependent control mechanism known as quorum sensing (QS) and is, therefore, normally only initiated at later stages of cell culture, PhzC seems to ensure sufficient metabolite flow into phenazine biosynthesis when other DAHP synthases are inhibited. Surprisingly, the *phz* operon of *Streptomyces cinnamonensis* was found to not contain a *phzF* gene, which, as will be outlined below, is absolutely required for phenazine biosynthesis. Here, the *phzF* gene is located elsewhere in the genome (K. Seeger and L. Heide, personal communication).

In most cases, the *phz* operon contains phenazine-modifying enzymes. Only a minority of these enzymes has been studied in detail, but it is often possible to identify their enzymatic activity from sequence analysis so that the phenazine derivatives produced by newly sequenced strains can be inferred even if they have not yet been isolated. Other open reading frames in extended gene clusters include phenazine resistance factors, regulatory proteins and exporter systems, which all seem required in the phenazine biosynthetic circuitry of the respective strain (Figure 2). For example, whereas it is believed that pseudomonads possess superoxide dismutases with elevated activity to protect themselves against the toxic action of phenazines,[70] some plant-colonizing bacteria like *Pantoea agglomerans* and *Pectobacterium atrosepticum* possess a resistance protein specific for the self-produced phenazine derivative.[64] Notably, no eubacterial-like *phz* gene cluster can be identified in the genome of *Methanosarcina mazei* Gö1, the only known archaeal phenazine producer.[28,71] This opens the possibility that phenazine biosynthesis follows a completely different pathway in this organism.

### Molecular mechanisms of phenazine ring assembly

Our current understanding of core phenazine biosynthesis in bacteria is summarized in Scheme 2. This scheme was in part derived by McDonald et al., who published the first systematic study of the enzymes PhzA–G in 2001.[68] Working with cell-free extracts of *E. coli* expressing all or parts of the *phz* operon from *P. fluorescens* 2-79, these authors demonstrated the complete conversion of 2-amino-2-desoxyisochorismate (ADIC) but not anthranilate into PCA. While this second finding reiterated preceding studies, the quantitative incorporation of ADIC suggested strongly that PhzE is a defective anthranilate synthase that cannot convert and, therefore, releases the transient intermediate ADIC, which is also generated but not released before being converted to anthranilate in related enzymes. ADIC as a precursor of the phenazine moiety had already been postulated by Römer and Herbert in 1982[72] and it should be mentioned that free ADIC has recently been shown to be an inter-

![Diagram](attachment:diagram.png)

**Scheme 2. Current understanding of bacterial core phenazine biosynthesis.** E4P: erythrose-4-phosphate, PEP: phosphoenol pyruvate, DAHP: 3-deoxy-d-arabinohexulose-7-phosphate, ADIC: 2-amino-2-desoxyisochorismic acid, DHHA: trans-2,3-dihydro-3-hydroxyanthranilic acid, PCA: phenazine-1-carboxylic acid, PDC: phenazine-1,6-dicarboxylic acid.

mediate in the biosynthesis of
the benzoxazinolate moiety from
the enediyne antitumor antibiotic C-1027 from Streptomyces glo-
bisporus.[73] In phenazine biosyn-
thesis, PhzE would then also be
responsible for integrating the
nitrogen atoms of phenazines
from glutamine, which had al-
ready been identified as the ni-
trogen source by Herbert and
colleagues in 1982 and was later
reconfirmed by Galbraith
et al.[72,74,75] The production of
ADIC by PhzE could, however,
not be confirmed directly since
incubation of the cell-free ex-
tracts led to degradation of cho-
rismate, probably by other meta-
bolic enzymes.

    ADIC is next hydrolyzed to
trans-2,3-dihydro-3-hydroxyan-
thranilic acid (DHHA) and pyru-
vate by PhzD, an enzyme related
to the isochorismatase domain
of EntB in the biosynthesis of
the siderophore enterobactin;
in fact, PhzD was shown to also
utilize isochorismate as well as
4-amino-4-desoxychorismate and
chorismate as substrates.[68] The
crystal structure revealed that
PhzD, unlike related enzymes
that possess an active site cys-
teine and that had been studied
by that time, employs an aspart-
ic acid residue (D38) to proton-
ate the vinyl ether in the course
of the hydrolysis reaction (Fig-
ure 3 A).[76] A similar active site
aspartate was later also found in
the crystal structure of EntB.[77]
DHHA as a potential precursor
of phenazines was first men-
tioned by Herbert et al. in
1979.[78]

McDonald et al. further dem-
onstrated that PhzF and proba-
bly also PhzG were absolutely re-
quired for phenazine biosynthe-
sis. The absence of *phzA* or of both *phzA* and *phzB* reduced
the yield of PCA by four- and eightfold, respectively; this indi-
cates that these proteins, which showed no sequence relation-
ship to any other known enzymes at that time but were later
predicted to belong to the Δ⁵-3-ketosteroid isomerase/nuclear
transport factor 2 family,[79] are only required for the efficiency
of the pathway but not for the ensuing chemistry itself. The

Figure 3. Crystal structures of phenazine biosynthesis enzymes. Boxes within ribbon plots on the left side indicate
the position of active centers. The models of enzyme/substrate complexes shown on the right are based on avail-
able crystallographic data. Note that the orientation has been adjusted with respect to the ribbon plots to pro-
vide an unobstructed view. A) PhzD from *P. aeruginosa* (PDB ID: 1NF8 and 1NF9). The reaction is initiated by the
protonation of the ethylene group of ADIC by D38.[76] B) PhzF from *P. fluorescens* 2-79 (PDB ID: 1U1W). E45 trans-
fers a proton from C3 to C1 of DHHA. The partial surface shows a semisecluded cavity thought to enhance the
condensation of 1.[80] C) PhzA/B from *B. cepacia* R18194 (PDB ID: 3DZL). The enzyme catalyzes the twofold con-
densation of two ketone molecules 1, probably in the indicated order (orange numbers). Thin gray lines indicate
alternative positions of the catalytic residues R41, S77 and E140 observed in other crystal complexes. D) PhzG
from *P. fluorescens* 2-79.[83] The figure was prepared with PyMOL.[86]

observation that PhzG is similar to flavin mononucleotide
(FMN)-dependent pyridoxamine oxidases[61] led the authors
speculating that PhzG oxidizes DHHA to the respective 3-keto
compound, and that two of these molecules would condense
to form a tricyclic phenazine precursor. It was speculated that
enolization to the corresponding 3-hydroxyanthranilic acid,
which is not incorporated into PCA, was prevented by PhzF,

ChemBioChem 2009, 10, 2295–2304
© 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim
www.chembiochem.org 2301

which was assumed to hold the ketone in a conformation favoring dimerization. This attractive hypothesis, however, was falsified when we and Parsons et al. independently demonstrated that DHHA is not the substrate of PhzG but of PhzF.[80,81] This enzyme acts as an isomerase that converts DHHA to 6-amino-5-oxocyclohex-2-ene-1-carboxylic acid (1). Insight into the reaction mechanism was provided by the crystal complex of PhzF with the substrate/product analogue 3-hydroxyanthranilate. The structure suggested that PhzF utilizes E45 to abstract the proton at C3, and that reprotonation appears at C1 after delocalization of the negative charge through the double-bond system of DHHA. Since NMR spectroscopy revealed that the newly transferred proton at C1 was not lost when the experiment was performed in D₂O,[80] E45 seems to only shift the proton from C3 to C1 (Figure 3B); this suggests that PhzF might be one of the very few enzymes that catalyzes a pericyclic reaction.

Intriguingly, turnover of DHHA by PhzF does not stop at the stage of isomerization to the ketone, but the ketone reacts further by self-condensation with a second molecule, forming a tricyclic phenazine precursor. The structure of PhzF, which is a homodimer with two independent active sites facing each other, initially suggested that this condensation reaction takes place at the monomer–monomer interface; that is, PhzF is a bifunctional enzyme that catalyzes both isomerization and condensation.[80] This was, however, recently disproved when we showed that the formation of the tricycle is catalyzed by the PhzA/B proteins;[82] this implies that the condensation reaction does not absolutely require enzymatic catalysis, and at the same time explains the observation by McDonald et al. that the deletion of the phzA/B genes only decreased but did not abolish PCA biosynthesis.[68] The fact that pseudomonads carry two copies of the gene might therefore explain why these strains are amongst the most proficient phenazine producers. However, when the PhzA protein of *P. fluorescens* 2-79 was tested, it was found not to be active, which seems to be the consequence of two active site substitutions (H73L and S77L). The seemingly contradictory observation that the deletion of phzA still led to decreased PCA production[68] could indicate that PhzA and PhzB, which are homodimers when expressed separately, form heterodimers with only one functional active site in vivo. This would pay tribute to the fact that these proteins catalyze a very unusual reaction in which two identical substrate molecules are consumed per product molecule generated. In an active homodimer, the enzyme possesses four substrate binding sites, of which two are expected to have higher affinity but are located in the two independent active centers of the dimer. As a consequence, under conditions at which the substrate concentration is low, the active sites would compete for substrate to complete the reaction cycle.

In the active form, PhzB catalyzes two condensation reactions by first placing the substrate molecules in an opposite orientation and then achieving twofold condensation by acid/base catalysis, involving protonation of the tetrahedral intermediates by E140 first and probably by H73 together with S77 second (Figure 3C). When the product of the PhzA/B protein from *Burkolderia cepacia* R18194 was analyzed by HPLC-coupled NMR spectroscopy, it was found not to possess the anticipated symmetrical structure 2 shown in Scheme 2 but was rearranged to contain four conjugated double bonds instead (2a). While it is currently unclear if the required rearrangement reactions are spontaneous or catalyzed by PhzA/B, they probably stabilize the condensation product against back-hydrolysis to the ketone and also prime the product for the next reaction, which is a rapid oxidative decarboxylation to intermediate 3; this decarboxylation was also observed when PhzA/B was not included in the reaction mixture.[80] This step consumes oxygen but seems otherwise uncatalyzed, explaining why PCA and not PDC is the end product of core phenazine biosynthesis in pseudomonads and also why PDC was not converted to PCA by cell-free extracts of *E. coli* over-expressing the phz operon.[68]

Intermediate 3 needs to undergo two additional two-electron oxidations to become fully aromatized, but the phz operon contains only one bona fide oxidase, the FMN-dependent PhzG. The crystal structures of the enzymes from *P. fluorescens* 2-79 and *P. aeruginosa* have been determined (Figure 3D), and while the enzyme has not yet been characterized in detail, Parsons et al. hypothesized that it is involved in a step after the condensation of ketone 1.[83] From the available data, it is tempting to speculate that PhzG oxidizes 3 to 5,10-dihydrophenazine-1-carboxylic acid, which is the reduced electron-shuttle form of PCA and could, therefore, present a valid end product of the core pathway. Since phenazine modification is coupled to core phenazine biosynthesis in the cell, it is conceivable that phenazine-modifying enzymes possess higher activity towards reduced phenazines, which was confirmed in our own laboratory (N. Gohain and W.B., unpublished results) and also in a recent report in which the biosynthesis of prenylated phenazines from *Streptomyces annulatus* was studied.[66]

Interestingly, the terminal oxidation reactions in core phenazine biosynthesis do not require enzymatic catalysis, and reaction mixtures containing only PhzF and PhzA/B from *B. cepacia* acting on DHHA produced, in addition to PCA, also trace amounts of PDC and unsubstituted phenazine (Scheme 2).[82] This suggests that two principle roads lead to the aromatization of tricyclic phenazine precursors: one involving direct oxidative decarboxylation and a second probably requiring PhzG/flavin-dependent oxidation without the concomitant loss of carboxylate in vivo.

## Open Questions and Future Directions

One hundred and fifty years after Fordos’ isolation of pyocyanin, we have now reached a principle understanding of most of the chemical steps in the biosynthesis of phenazines from chorismate in bacteria. Nevertheless, a number of important details still need to be clarified. A fundamental question derives from the observation that no phz operon can be identified in the archaeon *Methanosarcina mazei* Gö1, suggesting that phenazines arise through a different biosynthetic route in this organism.[71] It will be interesting to see if phenazines are generated from the same precursors or if the biosynthesis pathway is completely different in Archaea.

With respect to bacterial core phenazine biosynthesis, the first and foremost remaining problem seems to be the question about PCA versus PDC production, since PDC suggests itself as the logical precursor of symmetrical phenazine derivatives, whereas recent studies suggest that core phenazine biosynthesis involves a rapid nonenzymatic oxidative decarboxylation.[80–82] Because this step involves molecular oxygen, it is possible that efficient PDC production is only possible in microorganisms that live in an anaerobic environment. However, PDC as a precursor to strain-specific phenazine derivatives would also require an additional enzyme-catalyzed oxidation of the condensation product 2a within the core phenazine biosynthesis pathway. Since this enzyme would most likely be encoded within the conserved phz operon, and the operon was found to contain only one oxidase, PhzG, one would have to assume that this enzyme is promiscuous and can oxidize both 2a and 3. As an extension to this hypothesis, it is tempting to speculate about the formation of a multienzyme complex of the Phz proteins, in which the exposure of intermediates to oxygen is avoided. Although such a complex has been postulated previously,[68] experimental evidence for its existence is still lacking.

A more indirect approach towards understanding the relationship between PCA and PDC is to assay the increasing number of known phenazine-modifying enzymes for their preference of utilizing either of the two or their reduced forms as substrates. With the better understanding of core phenazine biosynthesis, phenazine-modifying enzymes as a source of interesting phenazine derivatives will undoubtedly receive more attention in the future, as recent detailed studies of the pyocyanin-generating enzymes PhzM[84] and PhzS[85] from *P. aeruginosa* or of the phenazine-specific isoprenyl transferase[66] from *S. annulatus* already indicate.

Other interesting areas of phenazine biochemistry that extend beyond biosynthesis are molecular mechanisms involved in its regulation, in phenazine resistance and in the interaction of phenazines with target factors. With respect to regulation, it is generally believed that phenazine biosynthesis is under transcriptional control through cell-density-dependent QS, such that phenazines are only produced after a threshold cell density has been reached. It is, however, possible that PhzE, which catalyzes the committed step of phenazine biosynthesis, adds an additional layer of control through allosteric regulation, similar to some of the related anthranilate synthases.

Regarding resistance toward the toxicity of self-synthesized phenazines, it is interesting to note that at least two different mechanisms seem to exist: while *P. aeruginosa* was found to exhibit increased superoxide dismutase and catalase activities under conditions favoring pyocyanin production,[70] phenazine-producing strains of the *Pectobacterium* and *Pantoea* genus encode a resistance factor ehpR within their phz operon.[64] EhpR belongs to the glyoxalase I/bleomycin resistance protein family of resistance factors (S. Yu and W.B., unpublished results) and probably escorts newly synthesized phenazines to a specific transporter also encoded within the phz operon of the respective species.


Even if most of the physiological effects of phenazines can be attributed to their redox activity as outlined above, new data make it increasingly apparent that their action is not restricted to small molecular electron acceptors like oxygen or ferric iron, but that they also target biological macromolecules directly. In this regard, the recent example of direct pyocyanin-activated SoxR-dependent transcription in *P. aeruginosa* is only the first example for such an interaction,[26] and it is expected that more such cases both with pathogen and host factors will be identified in the future. Given the importance of phenazine derivatives for both the physiology and pathogenicity of phenazine-producing bacteria, a better understanding of these relationships will clarify if phenazine biosynthesis can be exploited both as a target for or as a source of novel therapeutics.


# Acknowledgements

Work of the authors summarized in this review was supported by the Max Planck Society, the Bioband of the Technical University of Dortmund (to R.B. and W.B.) and by the Deutsche Forschungsgemeinschaft (to W.B., BL587).


**Keywords:** antibiotics · biosynthesis · condensation · shikimate · virulence factors


1. M. J. Fordos, *Recl. Trav. Soc. d’Émul. Sci. Pharm.* **1859**, 3, 30.
2. J. B. Laursen, J. Nielsen, *Chem. Rev.* **2004**, 104, 1663.
3. A. Price-Whelan, L. E. Dietrich, D. K. Newman, *Nat. Chem. Biol.* **2006**, 2, 71.
4. D. V. Mavrodi, W. Blankenfeldt, L. S. Thomashow, *Annu. Rev. Phytopathol.* **2006**, 44, 417.
5. D. V. Mavrodi, L. S. Thomashow, W. Blankenfeldt in *Pseudomonas: Model Organism, Pathogen, Cell Factory* (Ed.: B. Rehm), Wiley-VCH, Weinheim, 2008, pp. 331.
6. M. J. Fordos, C. R. Hebd. Seances Acad. Sci. **1860**, 51, 215.
7. M. C. Gessard, C. R. Hebd. Seances Acad. Sci. **1882**, 165, 536.
8. J. Emerson, M. Rosenfeld, S. McNamara, B. Ramsey, R. L. Gibson, *Pediatr. Pulmonol.* **2002**, 34, 91.
9. G. Ledderhose, *Langenbecks Arch. Klin. Chir. Ver. Dtsch. Z. Chir.* **1888**, 28, 201.
10. H. McCombie, H. A. Scarborough, *J. Chem. Soc.* **1923**, 123, 3279.
11. F. Wrede, E. Strack, *Hoppe-Seyler’s Z. Physiol. Chem.* **1924**, 140, 1.
12. F. Wrede, E. Strack, *Hoppe-Seyler’s Z. Physiol. Chem.* **1924**, 142, 103.
13. F. Wrede, E. Strack, *Hoppe-Seyler’s Z. Physiol. Chem.* **1929**, 181, 58.
14. B. Elema, *Recl. Trav. Chim. Pays-Bas* **1931**, 50, 807.
15. E. Friedheim, L. Michaelis, *J. Biol. Chem.* **1931**, 91, 355.
16. H. Hillemann, *Ber. Dtsch. Chem. Ges. B* **1938**, 71, 46.
17. K. A. Jensen, C. H. Holten, *Acta Chem. Scand.* **1949**, 3, 1446.
18. P. Ernst, *Z. Hyg. Infektionskr.* **1887**, 2, 369.
19. J. M. Turner, A. J. Messenger, *Adv. Microb. Ecol.* **1986**, 27, 211.
20. G. W. Lau, D. J. Hassett, H. Ran, F. Kong, *Trends Mol. Med.* **2004**, 10, 599.
21. G. W. Lau, H. Ran, F. Kong, D. J. Hassett, D. Mavrodi, *Infect. Immun.* **2004**, 72, 4275.
22. R. Emmerich, O. Loew, *Zeit. Hyg.* **1899**, 31, 1.
23. G. N. Rolinson, *Zentralbl. Bakteriol. Mikrobiol. Hyg. A* **1988**, 267, 307.
24. D. Haas, G. Defago, *Nat. Rev. Microbiol.* **2005**, 3, 307.
25. M. E. Hernandez, A. Kappler, D. K. Newman, *Appl. Environ. Microbiol.* **2004**, 70, 921.
26. L. E. Dietrich, A. Price-Whelan, A. Petersen, M. Whiteley, D. K. Newman, *Mol. Microbiol.* **2006**, 61, 1308.
27. A. Price-Whelan, L. E. Dietrich, D. K. Newman, *J. Bacteriol.* **2007**, 189, 6372.
28. H. J. Abken, M. Tietze, J. Brodersen, S. Baumer, U. Beifuss, U. Deppenmeier, *J. Bacteriol.* **1998**, 180, 2027.

[29] M. C. Gessard, Ann. Inst. Pasteur 1892, 6, 801.
[30] E. O. Jordan, J. Exp. Med. 1899, 4, 627.
[31] M. O. Burton, B. A. Eagles, J. J. R. Campbell, Can. J. Res. Sect. C-Bot. Sci. 1947, 25, 121.
[32] M. O. Burton, J. J. R. Campbell, B. A. Eagles, Can. J. Res. Sect. C-Bot. Sci. 1948, 26, 15.
[33] W. M. Ingledew, J. J. Campbell, Can. J. Microbiol. 1969, 15, 595.
[34] A. J. M. Messenger, J. M. Turner, J. Gen. Microbiol. 1983, 129, 1013.
[35] E. O. King, M. K. Ward, D. E. Raney, J. Lab. Clin. Med. 1954, 44, 301.
[36] A. C. Blackwood, A. C. Neish, Can. J. Microbiol. 1957, 3, 165.
[37] H. Korth, Arch. Mikrobiol. 1971, 77, 59.
[38] L. E. Dietrich, T. K. Teal, A. Price-Whelan, D. K. Newman, Science 2008, 321, 1203.
[39] B. D. Davis, J. Biol. Chem. 1951, 191, 315.
[40] M. Kurachi, Bull. Inst. Chem. Res. Kyoto Univ. 1959, 37, 85.
[41] R. E. Carter, J. H. Richards, J. Am. Chem. Soc. 1961, 83, 495.
[42] R. C. Millican, Biochim. Biophys. Acta 1962, 57, 407.
[43] M. E. Levitch, E. R. Stadtman, Arch. Biochem. Biophys. 1964, 106, 194.
[44] M. E. Levitch, P. Rietz, Biochemistry 1966, 5, 689.
[45] M. Podojil, N. N. Gerber, Biochemistry 1967, 6, 2701.
[46] P. C. Chang, A. C. Blackwood, Can. J. Biochem. 1968, 46, 925.
[47] W. M. Ingledew, J. J. Campbell, Can. J. Microbiol. 1969, 15, 535.
[48] D. H. Calhoun, M. Carson, R. A. Jensen, J. Gen. Microbiol. 1972, 72, 581.
[49] R. P. Longley, J. E. Halliwell, J. J. Campbell, W. M. Ingledew, Can. J. Microbiol. 1972, 18, 1357.
[50] M. Podojil, N. N. Gerber, Biochemistry 1970, 9, 46164
[51] G. R. Clemo, H. McIlwain, J. Chem. Soc. 1938, 479.
[52] R. B. Herbert, F. G. Holliman, P. N. Ibberson, J. Chem. Soc. Chem. Commun. 1972, 355.
[53] U. Hollstein, L. G. Marshall, J. Org. Chem. 1972, 37, 3510.
[54] U. Hollstein, D. A. McCamey, J. Org. Chem. 1973, 38, 3415.
[55] R. B. Herbert, F. G. Holliman, J. B. Sheridan, Tetrahedron Lett. 1976, 17, 639.
[56] U. Hollstein, D. L. Mock, R. R. Sibbitt, U. Roisch, F. Lingens, Tetrahedron Lett. 1978, 19, 2987.
[57] D. W. Essar, L. Eberly, A. Hadero, I. P. Crawford, J. Bacteriol. 1990, 172, 884.
[58] J. F. Dubern, S. P. Diggle, Mol. BioSyst. 2008, 4, 882.
[59] J. M. Farrow, E. C. Pesci, J. Bacteriol. 2007, 189, 3425.
[60] L. S. Pierson, III, L. S. Thomashow, Mol. Plant Microbe Interact. 1992, 5, 330.
[61] L. S. Pierson, III, T. Gaffney, S. Lam, F. Gong, FEMS Microbiol. Lett. 1995, 134, 299.
[62] D. V. Mavrodi, V. N. Ksenzenko, R. F. Bonsall, R. J. Cook, A. M. Boronin, L. S. Thomashow, J. Bacteriol. 1998, 180, 2541.
[63] D. V. Mavrodi, R. F. Bonsall, S. M. Delaney, M. J. Soule, G. Phillips, L. S. Thomashow, J. Bacteriol. 2001, 183, 6454.
[64] S. R. Giddens, Y. Feng, H. K. Mahanty, Mol. Microbiol. 2002, 45, 769.

[65] Y. Haagen, K. Gluck, K. Fay, B. Kammerer, B. Gust, L. Heide, ChemBioChem 2006, 7, 2016.
[66] O. Saleh, B. Gust, B. Boll, H. P. Fiedler, L. Heide, J. Biol. Chem. 2009, 284, 14439.
[67] D. A. Fitzpatrick, J. Mol. Evol. 2009, 68, 171.
[68] M. McDonald, D. V. Mavrodi, L. S. Thomashow, H. G. Floss, J. Am. Chem. Soc. 2001, 123, 9459.
[69] C. J. Webby, H. M. Baker, J. S. Lott, E. N. Baker, E. J. Parker, J. Mol. Biol. 2005, 354, 927.
[70] D. J. Hassett, L. Charniga, K. Bean, D. E. Ohman, M. S. Cohen, Infect. Immun. 1992, 60, 328.
[71] U. Deppenmeier, A. Johann, T. Hartsch, R. Merkl, R. A. Schmitz, R. Martinez-Arias, A. Henne, A. Wiezer, S. Baumer, C. Jacobi, H. Bruggemann, T. Lienard, A. Christmann, M. Bomeke, S. Steckel, A. Bhattacharyya, A. Lykidis, R. Overbeek, H. P. Klenc, R. P. Gunsalus, H. J. Fritz, G. Gottschalk, J. Mol. Microbiol. Biotechnol. 2002, 4, 453.
[72] A. Roemer, R. B. Herbert, Z. Naturforsch. C: Biosci. 1982, 37C, 1070.
[73] S. G. Van Lanen, S. Lin, B. Shen, Proc. Natl. Acad. Sci. USA 2008, 105, 494.
[74] R. B. Herbert, J. Mann, A. Roemer, Z. Naturforsch. C: Biosci. 1982, 37C, 159.
[75] M. D. Galbraith, S. R. Giddens, H. K. Mahanty, B. Clark, Can. J. Microbiol. 2004, 50, 877.
[76] J. F. Parsons, K. Calabrese, E. Eisenstein, J. E. Ladner, Biochemistry 2003, 42, 5684.
[77] E. J. Drake, D. A. Nicolai, A. M. Gulick, Chem. Biol. 2006, 13, 409.
[78] R. B. Herbert, F. G. Holliman, P. N. Ibberson, J. B. Sheridan, J. Chem. Soc. Perkin Trans. 1 1979, 2411.
[79] E. G. Ahuja, D. V. Mavrodi, L. S. Thomashow, W. Blankenfeldt, Acta Crystallogr. Sect. D: Biol. Crystallogr. 2004, 60, 1129.
[80] W. Blankenfeldt, A. P. Kuzin, T. Skarina, Y. Korniyenko, L. Tong, P. Bayer, P. Janning, L. S. Thomashow, D. V. Mavrodi, Proc. Natl. Acad. Sci. USA 2004, 101, 16431.
[81] J. F. Parsons, F. Song, L. Parsons, K. Calabrese, E. Eisenstein, J. E. Ladner, Biochemistry 2004, 43, 12427.
[82] E. G. Ahuja, P. Janning, M. Mentel, A. Graebsch, R. Breinbauer, W. Hiller, B. Costisella, L. S. Thomashow, D. V. Mavrodi, W. Blankenfeldt, J. Am. Chem. Soc. 2008, 130, 17053.
[83] J. F. Parsons, K. Calabrese, E. Eisenstein, J. E. Ladner, Acta Crystallogr. Sect. D: Biol. Crystallogr. 2004, 60, 2110.
[84] J. F. Parsons, B. T. Greenhagen, K. Shi, K. Calabrese, H. Robinson, J. E. Ladner, Biochemistry 2007, 46, 1821.
[85] B. T. Greenhagen, K. Shi, H. Robinson, S. Gamage, A. K. Bera, J. E. Ladner, J. F. Parsons, Biochemistry 2008, 47, 5281.
[86] W. L. DeLano in The PyMOL User’s Manual, DeLano Scientific, San Carlos, CA, 2002.

Received: May 25, 2009
Published online on August 5, 2009

2304 www.chembiochem.org © 2009 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim ChemBioChem 2009, 10, 2295–2304
