
NATURE|Vol 460|13 August 2009|doi:10.1038/nature08302

**Molybdenum cofactors, enzymes and pathways**

Günter Schwarz¹, Ralf R. Mendel² & Markus W. Ribbe³

The trace element molybdenum is essential for nearly all organisms and forms the catalytic centre of a large variety of enzymes such as nitrogenase, nitrate reductases, sulphite oxidase and xanthine oxidoreductases. Nature has developed two scaffolds holding molybdenum in place, the iron-molybdenum cofactor and pterin-based molybdenum cofactors. Despite the different structures and functions of molybdenum-dependent enzymes, there are important similarities, which we highlight here. The biosynthetic pathways leading to both types of cofactor have common mechanistic aspects relating to scaffold formation, metal activation and cofactor insertion into apoenzymes, and have served as an evolutionary ‘toolbox’ to mediate additional cellular functions in eukaryotic metabolism.

Molybdenum is bioavailable as molybdate (MoO₄²⁻). Once molybdate enters the cell, it is subsequently incorporated by complex biosynthetic machineries into metal cofactors¹². All enzymes that depend on molybdenum catalyse redox reactions by taking advantage of the versatile redox chemistry of the metal, which is controlled by the cofactor itself and the enzyme environment³. Within the enzyme, molybdenum shuttles between three oxidation states (+4, +5 and +6), thereby catalysing two-electron reduction–oxidation (redox) reactions. In most cases, regeneration of the active site involves single-electron steps, resulting in a paramagnetic molybdenum intermediate. Molybdenum enzymes are found in nearly all organisms, with Saccharomyces as a prominent eukaryotic exception⁴. Many anaerobic archaea and some bacteria are molybdenum independent but require tungsten for their growth. Tungstate, which is 100-fold less abundant than molybdate, is enriched in deep-sea hydrothermal vents, reflecting conditions on the primitive Earth. Many of the known tungsten-dependent hyperthermophilic bacteria and archaea are found in such vents⁵.

In nature, two very different systems have developed to control the redox state and catalytic power of molybdenum, which functions as an efficient catalyst in oxygen-transfer reactions. In either case, at least three sulphur and two oxygen atoms form ligands to molybdenum (Fig. 1). One type of molybdenum cofactor is the iron–sulphur-cluster-based iron–molybdenum cofactor (FeMo-co) that is unique to the molybdenum nitrogenase⁶, with one [4Fe–3S] and one [Mo–3Fe–3S] partial cubane bridged by three sulphides and one μ₆ central atom, X (which may be carbon, oxygen or nitrogen)⁷. The molybdenum of FeMo-co is further coordinated by homocitrate (Fig. 1a). The core structure of the other type of molybdenum cofactor (Moco) is a pterin-based cofactor (molybdopterin or metal-binding pterin (MPT)), with a C6-substituted pyrano ring, a terminal phosphate and a unique dithiolate group binding molybdenum⁸. The metal can be attached to one or two pterin moieties with additional terminal oxygen and sulphur ligands (Fig. 1c, d). Both cofactors are oxygen sensitive and very unstable outside their respective apoenzymes.

In this Review, we give a short overview of the different families of molybdenum-containing enzymes, focusing on the biosynthetic machineries that lead to the ‘biological’ activation of the metal in different molybdenum cofactors.

### Molybdenum-dependent enzymes

On the basis of cofactor composition and catalytic function, molybdenum-dependent enzymes can be grouped into two categories: bacterial nitrogenases containing an FeMo-co in the active site, and pterin-based molybdenum enzymes. The second category is divided into three families, exemplified by sulphite oxidase, xanthine oxidase and dimethyl sulphoxide reductase (DMSOR), which each have a distinct active-site structure³ (Fig. 1). Tungsten-dependent formate dehydrogenase is classified as part of the DMSOR family, whereas aldehyde:ferredoxin oxidoreductases form a separate family of tungsten-cofactor (W-co) containing enzymes found only in archaea. The biochemistry of W-co enzymes has been summarized very recently⁵.

### Molybdenum nitrogenase

Nitrogenases provide the biochemical machinery for nucleotide-dependent reduction of dinitrogen (N₂) to ammonia (NH₃)⁶. The overall reaction catalysed by nitrogenases is usually depicted as N₂ + 8H⁺ + 16MgATP + 8e⁻ → 2NH₃ + H₂ + 16MgADP + 16Pi, where Pi denotes an inorganic phosphate. This reaction not only represents a major entry point of reduced nitrogen into the global nitrogen cycle, but also embodies the complex chemistry of breaking the triple bond of N₂ under ambient conditions (see page 814). Three homologous nitrogenase systems have been identified so far⁹. The best-characterized molybdenum nitrogenase is a binary enzyme system comprising two redox-active metalloproteins (Fig. 2a). One, designated Fe-protein, is an α₂ homodimer with one [4Fe–4S] cluster bridged between the subunits and one MgATP-binding site located in each subunit; the other, termed MoFe-protein, is an α₂β₂ heterotetramer containing two unique metal centres: the P-cluster, an [8Fe–7S] cluster that is ligated between each αβ-subunit dimer; and the FeMo-co, a [Mo–7Fe–9S–X–homocitrate] cluster (where X may be carbon, oxygen or nitrogen) that is buried within each α-subunit⁷,¹⁰ (Fig. 1a). Catalysis by molybdenum nitrogenase probably involves the repeated association and dissociation of Fe-protein and MoFe-protein

---

¹Institute of Biochemistry, Department of Chemistry & Centre for Molecular Medicine, University of Cologne, 47 Zülpicher Street, 50674 Cologne, Germany.  
²Institute of Plant Biology, Braunschweig University of Technology, 1 Humboldt Street, 38106 Braunschweig, Germany.  
³Department of Molecular Biology and Biochemistry, School of Biological Sciences, University of California, Irvine, California 92697, USA.

INSIGHT REVIEW

NATURE | Vol 460 | 13 August 2009

![](image.png)

Figure 1 | Molybdenum-containing cofactors. Chemical and three-dimensional structures of FeMo-co (a), Mo-bis-MGD (b) and the Moco from enzymes of the sulphite oxidase (c) and xanthine oxidase (d) families. The three-dimensional structures were taken from the crystal structures

and ATP-dependent transfer of electrons from the [4Fe–4S] cluster of Fe-protein, through the P-cluster, to the FeMo-co of MoFe-protein, where substrate reduction eventually takes place.

### Molybdenum enzymes with a pterin cofactor

More than 50 different pterin-containing molybdenum enzymes are known and classified on the basis of the coordination chemistry of molybdenum in their active site. All eukaryotic molybdenum enzymes belong exclusively to either the sulphite oxidase or the xanthine oxidase family with a simple MPT-type cofactor (Fig. 1c, d). They differ in the nature of the third Mo–S ligand, which is either provided by an enzyme-derived cysteine (sulphite oxidase) or as a terminal sulphido ligand (xanthine oxidase). By contrast, members of the DMSOR family coordinate molybdenum by two pterin moieties, each carrying in addition guanosine monophosphate, which together form a Mo-bis-MPT guanine dinucleotide cofactor (Mo-bis-MGD; Fig. 1b). Detailed mechanistic aspects of all these enzyme families have been discussed elsewhere.

Members of the DMSOR family are very diverse in reaction, function and structure. Whereas DMSOR of *Rhodobacter sphaeroides* is monomeric and lacks other redox centres, DMSOR from *Escherichia coli* is a membrane-bound enzyme composed of three subunits (DmsA, DmsB and DmsC) with high similarity to *E. coli* dissimilatory nitrate reductase (Fig. 2b). Both represent complex members of this family and additionally harbour a [3Fe–4S] cluster in the catalytic Mo-bis-MGD-containing subunit (NarG), four [4Fe–4S] clusters in the electron-transfer subunit (NarH) and a pair of type- $b$ cytochromes in the membrane-spanning subunit (NarI), together forming a dimer of trimers. By contrast with the tricyclic pyranopterin structure of Moco in all known molybdenum and tungsten enzymes, NarG–NarH–NarI shows the presence of an open bicyclic structure in one of the pterins, suggesting a reversible cyclization during catalysis. Most members of the DMSOR family function under anaerobic conditions whereby their

of nitrogenase, NarG–NarH–NarI, *Pichia pastoris* nitrate reductase and bovine xanthine oxidase. Structures are shown in ball-and-stick presentation (Mo, cyan; C, grey; O, red; S, yellow; Fe, brown; N, blue) and rendered with PYMOL (http://www.pymol.org/).

respective cofactors serve as terminal electron acceptors in respiratory metabolism.

Animal sulphite oxidase and eukaryotic nitrate reductase are dimeric enzymes forming another family of molybdenum enzymes with a high degree of structural conservation. In addition to the molybdenum domain, they both harbour a cytochrome-$b_{5}$-type haem domain that either receives electrons from the molybdenum centre (sulphite oxidase; Fig. 2c) or donates electrons to the molybdenum site (nitrate reductase). However, animal sulphite oxidase shuttles electrons derived from sulphite oxidation towards cytochrome $c$, whereas nitrate reductase receives electrons needed for nitrate reduction from NADPH. The catalytic cycle of sulphite oxidase involves sulphite oxidation coupled to molybdenum reduction, followed by two individual electron-transfer steps through the cytochrome $b_{5}$ domain to cytochrome $c$, a process associated with large spatial movements of the sulphite oxidase haem domain. Sulphite oxidases and homologous enzymes are also found in plants and bacteria, where they either form a homodimer of two molybdenum subunits lacking the haem domain (plants) or assemble into a heterodimer of a single molybdenum- and cytochrome $c$-containing subunit (bacteria). Sulphite dehydrogenase from *Starkeya novella* is localized in the periplasm, whereas plant sulphite oxidase functions in peroxisomes, generating H₂O₂ on transfer of electrons to dioxygen. Plant, fungal and algal nitrate reductases are localized in the cytosol and provide the second major entry point for nitrogen into the living world. By comparison with sulphite oxidase, nitrate reductase contains an additional carboxy-terminal FAD domain where either NADH (plant and algae) or NADPH (fungi) provides reducing equivalents for nitrate reduction.

All members of the xanthine oxidase family (also termed molybdenum hydroxylases) catalyse hydroxylations of carbon centres (in aldehydes and aromatic heterocycles) using oxygen derived from water. They are molybdo-flavoenzymes forming homodimers in eukaryotes with three distinct domains in each subunit (Fig. 2d). Xanthine oxidase is the key enzyme in purine degradation, catalysing oxidation of hypoxanthine via
xanthine to uric acid. Electrons derived from substrate hydroxylation are transferred through [2Fe–2S] clusters and FAD to either molecular oxygen, yielding superoxide anions (xanthine oxidase) ${ }^{12}$ or NAD ${ }^{+}$ (xanthine dehydrogenase) ${ }^{23}$. In bacteria, such as *Rhodobacter capsulatus*, xanthine dehydrogenase assembles from two different subunits, harbouring the Moco on one subunit and the iron-containing and FAD-containing domains on the other subunit and showing a remarkable structural and functional similarity to the bovine enzyme ${ }^{24,25}$. Eukaryotic aldehyde oxidases, which are derived from ancient gene duplications of xanthine oxidase, convert a wide range of aromatic and non-aromatic aldehydes and function in detoxification (animals) ${ }^{26}$ and hormone synthesis (abscisic acid, plants) ${ }^{27}$. Bacterial members of the xanthine oxidase family contain a mono-MPT cytosine dinucleotide cofactor and include aldehyde oxidoreductases ${ }^{28}$ and carbon monoxide dehydrogenase ${ }^{29}$, the latter being characterized by a special dinuclear Cu–S–Mo centre. Apart from substrate-dependent production of reactive oxygen species, xanthine oxidase and aldehyde oxidases show NADH oxidase activity with simultaneous production of superoxide ${ }^{30}$, which has a number of proposed physiological functions in the metabolism of reactive oxygen species during stress response ${ }^{23}$.

Recently a new mitochondria-associated molybdenum enzyme was found in mammals ${ }^{31}$ that promotes reduction of $N$-hydroxylated amines in concert with cytochrome $b_{5}$ and cytochrome $b_{5}$ reductase, a reaction that may be associated with cellular detoxification. It remains unclear what molybdenum-enzyme family this enzyme belongs to, but its occurrence is widespread, homologues having been found among plants and bacteria.

### Biosynthesis of FeMo-co

Assembly of nitrogenase FeMo-co is a considerable chemical feat because of its complexity and intricacy. Recent progress in the chemical synthesis of FeMo-co analogues has provided significant insights into this process ${ }^{32}$. Elucidation of the biosynthesis of FeMo-co, on the other hand, is further complicated by the large ensemble of participating gene products ${ }^{1,33}$. The exact functions of these gene products and the precise sequence of events in FeMo-co assembly have remained unclear until recently, when the characterization of a number of assembly-related intermediates afforded a better understanding of this biosynthetic ‘black box’ (Fig. 3).

### Formation of the Fe–S core of FeMo-co

Assembly of FeMo-co is probably initiated by NifU and NifS, which mobilize iron and sulphur for the assembly of small Fe–S fragments (see page 831). NifS is a pyridoxal phosphate-dependent cysteine desulphrase and is responsible for the formation of a protein-bound cysteine persulphide that is subsequently donated to NifU for the sequential formation of [2Fe–2S] and [4Fe–4S] clusters ${ }^{34,35}$ (Fig. 3). These small Fe–S clusters are then transferred to NifB and further processed into a large Fe–S core that possibly contains all the iron and sulphur necessary for the generation of a mature cofactor ${ }^{36}$. The exact function of NifB in this process is unclear. Nevertheless, NifB is an indispensable constituent of FeMo-co biosynthesis, as deletion of *nifB* results in the generation of a cofactor-deficient MoFe-protein ${ }^{37}$. Sequence analysis indicates that NifB contains a CXXXCXXC (where X is any amino acid) signature motif at the amino terminus, which is typical for a family of radical S-adenosyl-L-methionine (SAM)-dependent enzymes ${ }^{1,38}$. In addition, there is an abundance of potential ligands in the NifB sequence that are available to coordinate the entire complement of iron atoms of FeMo-co ${ }^{1}$. Thus, formation of the Fe–S core on NifB may represent a new synthetic route to bridged metal clusters that relies on radical chemistry at the SAM domain of NifB. For example, NifB could link two [4Fe–4S] subcubanes by inserting a sulphur atom along with the central atom, X, thereby building a fully complemented Fe–S core that could be rearranged later into the core structure of FeMo-co (Fig. 3).

### Insertion of molybdenum into the Fe–S core on NifEN

The function of NifEN (NifE–NifN) as a scaffold protein for FeMo-co maturation was initially proposed on the basis of a significant degree of sequence homology between NifEN and the MoFe-protein, which has led to the hypothesis that NifEN contains a ‘P-cluster site’ that houses a P-cluster homologue and an ‘FeMo-co site’ that hosts the conversion of FeMo-co precursor to a mature cofactor ${ }^{1,33}$. Whereas the P-cluster homologue in NifEN was identified earlier as a [4Fe–4S] cluster ${ }^{33}$, a molybdenum-free precursor of FeMo-co was captured on NifEN only recently ${ }^{39}$. Iron K-edge X-ray absorption spectroscopy reveals that this precursor closely resembles the Fe–S core of the mature FeMo-co despite slightly elongated interatomic distances ${ }^{40}$ (Fig. 3). This finding implies that, instead of being assembled by the previously postulated mechanism that involves the coupling of [4Fe–3S] and [Mo–3Fe–3S] subclusters, the FeMo-co is assembled by having the complete Fe–S core structure in place before the insertion of molybdenum.

The precursor on NifEN can be converted, *in vitro*, to a fully complemented FeMo-co on incubation with Fe-protein, MgATP, molybdate and homocitrate ${ }^{41}$. Iron and molybdenum K-edge X-ray absorption spectroscopy reveals that the FeMo-co on NifEN is nearly identical in structure to the native cofactor in MoFe-protein, except for an asymmetric coordination of molybdenum that is probably due to the presence of a different ligand environment at the molybdenum end of the cofactor in NifEN ${ }^{41}$. Homocitrate is supplied by NifV (that is, homocitrate

**Figure 2 | Three-dimensional structures of representative members of molybdenum-containing enzymes.** a, Molybdenum nitrogenase from *Azotobacter vinelandii* ${ }^{17}$; b, *E. coli* membrane-bound dissimilatory nitrate reductase A (NarG–NarH–NarI) ${ }^{15}$; c, chicken sulphite oxidase ${ }^{16}$; d, bovine xanthine oxidase ${ }^{24}$. All enzymes shown operate as dimers of which one half is shown as a ribbon diagram with bound cofactors as spheres. For clarity, in the other half of the dimer, only the cofactors are shown, highlighting their spatial arrangement to maintain intramolecular electron transfer. The domain or subunit that binds the molybdenum-containing cofactor is rendered in orange. The FeMo-co-containing nitrogenase MoFe-protein assembles from two different subunits into a heterotetramer ($\alpha_{2}\beta_{2}$). Subunit compositions and domain structures are depicted below (DD, dimerization domain; FAD, FAD domain; Fe, [2Fe–2S] cluster; H, haem domain; Moco, Moco-binding domain). A scale bar 100 residues long is provided beneath c to indicate protein size. Structures were rendered with PYMOL and coloured as follows: Mo, cyan; C, green; O, red; N, blue; S, yellow; Fe, brown.
synthase) in vivo 42, but molybdenum mobilization within the cell that occurs before the intervention of Fe-protein remains a topic of debate 43,44. Nevertheless, the fact that the cluster is completely converted before its exit from NifEN points to Fe-protein having a significant role in FeMo-co maturation.

Fe-protein re-isolated after incubation with molybdate, homocitrate and MgATP is 'loaded' with molybdenum and homocitrate that can be subsequently inserted into the precursor on NifEN 45. The molybdenum K-edge X-ray absorption spectrum of the loaded Fe-protein is consistent with a decreased number of Mo=O bonds (two or three instead of the four found in molybdate) as well as a decrease in the effective oxidation state of molybdenum due to either a change in the formal oxidation state of molybdenum or a change in molybdenum ligation. Interestingly, the electron paramagnetic resonance spectrum of loaded Fe-protein assumes a line shape intermediate between those of the MgADP- and MgATP-bound states of the Fe-protein 45. This observation is consistent with that from the initial crystallographic analysis of an ADP-bound form of Fe-protein, in which molybdenum is attached at a position that corresponds to the γ-phosphate of ATP 46. Such an ADP/molybdenum-binding mode (Fig. 4a) may reflect the initial attachment of molybdenum to Fe-protein, particularly when the structural analogy between phosphate and molybdate is considered. Remarkably, similar nucleotide-assisted processes are proposed for the molybdenum insertion in pterin-based cofactors (see below; Fig. 4b).

### Insertion of FeMo-co into apo-MoFe-protein

The completion of FeMo-co assembly on NifEN signals the delivery of FeMo-co to its destined location in MoFe-protein. The absolute requirement of intermediary FeMo-co carrier(s) between NifEN and MoFe-protein was precluded by the observations of unaffected nitrogen-fixing activity of the host after deletions of proposed carrier-encoding gene(s) 47 and direct FeMo-co transfer between NifEN and MoFe-protein on protein–protein interactions 41. Sequence comparison between NifEN and MoFe-protein reveals that certain residues that either provide a covalent ligand or tightly pack FeMo-co within the polypeptide matrix of MoFe-protein are not duplicated in the corresponding NifEN sequence. It is possible, therefore, that the respective

**Figure 3 | Biosynthesis of FeMo-co.**  
**a,** Sequence of events during FeMo-co assembly. The biosynthetic flow of FeMo-co is NifU–NifS → NifB → NifEN → MoFe-protein. The combined action of NifU–NifS generates small Fe–S fragments on NifU (stages 1 and 2), which are used as building blocks for the formation of a large Fe–S core on NifB (stage 3). This Fe–S core is further processed into a molybdenum-free precursor (stage 4), which can be converted to a mature FeMo-co on NifEN on Fe-protein-mediated insertion of molybdenum and homocitrate (stage 5). After the completion of FeMo-co assembly on NifEN, FeMo-co is delivered to its destined location in MoFe-protein (stage 6). The permanent metal cluster sites in NifEN and MoFe-protein are brought into close proximity, allowing the subsequent diffusion of FeMo-co from its biosynthetic site in NifEN (low-affinity site) to its binding site in MoFe-protein (high-affinity site). On its delivery to MoFe-protein, FeMo-co interacts with a number of MoFe-protein residues en route to its target location within the protein. Identification of these residues 48–50 was assisted by the crystallographic analysis of a P-cluster-intact yet FeMo-co-deficient form of MoFe-protein, which contains a positively charged funnel in the α-subunit that is of sufficient size to accommodate the insertion of the negatively charged FeMo-co 37.

**Biosynthesis of pterin-based molybdenum cofactors**

Although widespread in all kingdoms, Moco is synthesized by a conserved biosynthetic pathway divided into four steps according to the biosynthetic intermediates: cyclic pyranopterin monophosphate (cPMP), MPT and MPT–AMP. The biosynthetic pathway has been summarized in detail 2 with particular focus on plants 51, bacteria 52 and humans 53, and is believed to be very similar to W-co synthesis 52. In prokaryotes a final modification by a nucleotide can occur, whereas in MPT-type enzymes Moco maturation either involves a terminal sulphuration (xanthine oxidase family) or cysteine ligation to the apoenzyme (sulphite oxidase family).

#### Synthesis of the metal-binding pterin

Biosynthesis starts with the conversion of GTP into cPMP (previously identified as precursor Z; ref. 54) catalysed by two proteins: a radical SAM enzyme (for example MoaA in bacteria) harbouring two oxygen-sensitive [4Fe–4S] clusters, and an accessory hexameric protein involved in pyrophosphate release (for example MoaC in bacteria) 55. MoaA harbours an N-terminal Fe–S cluster involved in radical SAM generation and a MoaA-specific C-terminal Fe–S cluster crucial for substrate binding 55. Although the reaction mechanism of cPMP synthesis is not yet fully understood, it is well established that each carbon of the ribose and purine is incorporated into cPMP 55,56. Furthermore, the structure of cPMP as a fully reduced pyranopterin with a terminal cyclic phosphate and geminal diol (Fig. 5) supports its physicochemical properties 57. With respect to the observed geminal diol, it remains to be

centres of the scaffold proteins are coloured pink; the transient cluster intermediates are coloured yellow. HC, homocitrate. **b,** Structures of intermediates during FeMo-co assembly. Shown are the cluster types that have been identified (on NifU, NifEN and MoFe-protein) or proposed (for NifB). Hypothetically, NifB could bridge two [4Fe–4S] clusters by inserting a sulphur atom along with the central atom, X, thereby generating an Fe–S scaffold that could be rearranged into a precursor closely resembling the core structure of the mature FeMo-co. In the case of the NifEN-associated precursor, only the 8Fe model is shown. The potential presence of X in the intermediates of FeMo-co biosynthesis is indicated by a question mark.

NATURE|Vol 460|13 August 2009

**REVIEW INSIGHT**

provides an evolutionary origin for ubiquitin-like protein conjugation. AMP-activated MoaD becomes sulphurated by sulphide transfer, which is catalysed by a cysteine desulphurase<sup>66</sup> and a rhodanese<sup>67</sup>; the latter is fused in eukaryotes, as the C-terminal domain, to an MoeB-homologous domain<sup>68</sup>.

### Metal insertion and nucleotide attachment

On completion of MPT synthesis, the metal is transferred by a multistep reaction. Whereas *E. coli* encodes two separate proteins involved in this step, eukaryotes catalyse metal transfer by homologous two-domain proteins, such as Cnx1 (plants) and gephyrin (human) (Fig. 5), pointing to a functional cooperation between their domains. The physiological functions of their domains were discovered by determining the crystal structure of the N-terminal G domain of Cnx1 in complex with substrate and product<sup>69</sup>. The latter was found to be MPT–AMP, a common intermediate in bacterial and eukaryotic Moco synthesis<sup>70</sup> synthesized by G domains and homologous proteins (MogA in bacteria)<sup>71</sup>. Subsequently, a transfer of MPT–AMP to the E domain in Cnx1 was demonstrated<sup>72</sup>. In the presence of divalent cations and molybdate, bound MPT–AMP is hydrolysed and molybdenum is transferred to the MPT dithiolate, resulting in Moco release. This Moco most probably carries two oxo ligands and one OH group depicted (Figs 4a and 5) in a deprotonated form<sup>72</sup>, as supported by preliminary spectroscopic data derived from a storage-protein-bound Moco (see below; G.S., unpublished observation). There is no experimental evidence for a reduction of molybdenum at this state.

W-co biosynthesis is believed to be conserved up to MPT formation<sup>5</sup>, with differences in metal transfer. The tungsten-dependent archaeon *Pyrococcus furiosus* and related thermophiles lack *mogA*; instead, they all express genes encoding an MoaB-like protein, which also catalyses MPT adenylylation, confirming MPT–AMP as an essential and general prerequisite before metal insertion<sup>73</sup>. Furthermore, *P. furiosus* expresses two different MoeA-like proteins, suggesting metal-selective activities<sup>5</sup>.

Finally, enzymes of the DMSOR family need to be further modified by the attachment of a nucleotide molecule (Fig. 5), a reaction dependent on the preceding metal insertion<sup>72</sup>. In *E. coli*, MobA catalyses the conversion of MPT and GTP to Mo–bis-MGD<sup>74</sup>. Interaction studies with proteins catalysing metal insertion and Mo–bis-MGD formation identified a transient Moco-synthesizing machinery comprising MogA, MoeA, MobA and molybdenum-enzyme-specific chaperones<sup>75</sup>.

### Cofactor maturation, storage and transfer

Molybdenum hydroxylases such as aldehyde oxidase and xanthine oxidase require a final step of maturation to gain enzymatic activity, namely the addition of a terminal sulphido ligand to the molybdenum centre, which is catalysed by a Moco sulphurase (that is, Aba3 in plants or HMCS (also known as MOCOS) in humans), a two-domain protein<sup>76</sup> acting as a homodimer (Fig. 6). In a pyridoxal phosphate-dependent manner, the N-terminal NifS-like domain abstracts sulphur from L-cysteine and forms a persulphide intermediate on a conserved cysteine residue<sup>77</sup>. Subsequently this sulphur is transferred via a second cysteine persulphide intermediate to bound Moco. Both of these steps are catalysed by the C-terminal Moco-binding domain of Aba3 (ref. 78), which selectively stabilizes sulphurated Moco. The same mechanism operates in HMCS (R.R.M., unpublished observations). Among prokaryotes, no homologues to eukaryotic Moco sulphurases have been found. However, for xanthine dehydrogenase from *R. capsulatus*, its enzyme-specific chaperone XdhC was found to fulfil Moco sulphuration<sup>79</sup>. By contrast with enzymes of the xanthine oxidase family, sulphite oxidase and nitrate reductase incorporate Moco without further modification. The proposed tri-oxo coordination of molybdenum in mature Moco<sup>72</sup> (Figs 4b and 5) suggests a simple mechanism of cysteine ligation to the molybdenum accompanied by loss of one of the oxygens as water.

As Moco is highly unstable once liberated from proteins, it was assumed that Moco does not occur in a ‘free state’; rather, Moco might be bound to a carrier protein that protects and stores it until further use. Whereas some

© 2009 Macmillan Publishers Limited. All rights reserved

bacteria have molybdate-binding proteins such as Mop<sup>80</sup>, the alga *Chlamydomonas reinhardtii* produces a homotetrameric protein<sup>81</sup> that holds four Moco molecules in a surface-exposed binding pocket<sup>82</sup>. In higher plants, gene families with 8–12 homologous Moco-binding proteins have been discovered recently (R.R.M., unpublished observations). It is still unclear whether these proteins represent a buffer in which to store Moco or whether they are part of the default pathway for Moco allocation and insertion into molybdenum enzymes, a mechanism poorly understood in eukaryotes. Because Moco is deeply buried within the holoenzymes, it needs to be incorporated before the completion of folding and oligomerization of enzyme subunits/domains; for this, many bacterial molybdenum enzymes require the presence of chaperones, such as NarJ for *E. coli* nitrate reductase, TorD for trimethylamine N-oxide reductase and DmsD for DMSOR, which bind and protect the apoenzymes, assist in cofactor insertion and control transmembrane targeting<sup>83</sup>.

### Molybdenum homeostasis and disorders

#### Cellular uptake
Bacterial molybdenum uptake requires specific systems to scavenge molybdate in the presence of competing anions. This involves a high-affinity ATP-binding cassette (ABC) transporter: molybdate is captured by one component, a periplasmic molybdate-binding protein (ModA), and transferred to another, the transmembrane channel (ModB). The crystal structure of an ABC transporter from *Archaeoglobus fulgidus*<sup>84</sup> suggests a conserved two-state mechanism by which ATP hydrolysis and the release of ADP plus P<sub>i</sub> at the cytoplasmic protein (ModC) controls conformation of the transmembrane protein, ModB. For tungstate, two ABC-type transporters, TupA-TupB–TupC and WtpA–WtpB–WtpC, have been identified<sup>73</sup>, the latter being highly selective for tungstate over molybdate owing to a unique octahedral substrate coordination<sup>85</sup>.

Algae and multicellular plants are the only eukaryotes for which the molybdate-uptake mechanisms have been recently determined. Two proteins belonging to the large sulphate-carrier family have been shown to transport molybdate with high affinity<sup>86–88</sup>. Unexpectedly, none of them was found to reside in the plasma membrane. Contradictory reports localized them to the endomembrane system<sup>86</sup> or the mitochondrial envelope<sup>88</sup>. It is likely that additional transporters, not only in autotrophs but also in animals, will be discovered soon.

### Molybdenum-iron and -copper crosstalk
Molybdenum metabolism is strictly dependent on iron metabolism at different levels. FeMo-co biosynthesis and nitrogenase maturation are based on the synthesis of complex Fe–S clusters, and enzymes participating in the first step of Moco biosynthesis contain two [4Fe–4S] clusters<sup>55</sup>. Furthermore, all molybdenum hydroxylases and several members

Figure 5 | Biosynthesis of the pyranopterin-based molybdenum cofactors. Shown is a generalized scheme of the pathway based on data derived from studies in *E. coli*, plants and humans. All known/characterized intermediates of the pathway are presented sequentially in the four steps in which Moco is synthesized. A fifth step present only in prokaryotes results in the formation of Mo-bis-MGD. Proposed (first step)<sup>55</sup> or partially characterized (second step) intermediates<sup>62</sup> are indicated in parentheses (circled P, phosphate group). The intermediate of the second step has been suggested on the basis of mechanistic studies of MPT synthase yielding an intermediate with a linear phosphate<sup>62</sup>. Proteins catalysing the individual steps are depicted in different colours, and a similar shade is applied to domains/proteins involved in nucleotide transfer. Homologous proteins from *E. coli* ('Mo' nomenclature), humans ('MOCS' nomenclature except gephyrin) and plants ('Cnx' nomenclature) are shown for comparison. The separate pathway regenerating MPT synthase is grouped within the grey box. For simplicity, only the MoeB protein is shown. We note that eukaryotes express fusion proteins that contain a MoeB domain and a C-terminal rhodanese-like domain involved in sulphur transfer to the small subunit of MPT synthase.

NATURE|Vol 460|13 August 2009

**Figure 6 | Domain structure and function of Moco sulphurase Aba3 from *Arabidopsis thaliana*.** The Aba3 protein and other homologues act as dimers catalysing the sulphuration of Moco, a reaction needed for the activation of molybdenum hydroxylases. Each Aba3 monomer can be divided into two domains, an N-terminal domain showing sequence homology to cysteine desulphurases (NifS-like enzymes) and a C-terminal domain that binds Moco. Recently, it was shown that a persulphide sulphur generated by the NifS-like domain is transferred to the C-terminal domain for the conversion of bound desulpho-Moco into sulphurated Moco, which is subsequently required to activate the target enzymes of Aba3 (that is, xanthine dehydrogenase and aldehyde oxidase). PLP, pyridoxal phosphate.

of the DMSOR family use Fe–S clusters for intramolecular electron transfer. Finally, enzymes of the sulphite oxidase family contain haem cofactors.

Recently, another link between the metabolic pathways of molybdenum and iron was discovered. In plants (and most probably also in animals), enzymes catalysing cPMP synthesis, such as Cnx2 and Cnx3, were localized within the mitochondrial matrix (R.R.M., unpublished observations), which necessitates the export of cPMP from mitochondria into the cytosol. Here, the mitochondrial ABC-type transporter Atm3 (also known as Stal) from *A. thaliana* seems to fulfil a dual function: it not only exports Fe–S-cluster precursors to the cytosol⁸⁹, but it is somehow also involved in cPMP translocation. Atm3-deficient plants showed defects in Fe–S-dependent cytosolic enzymes and accumulated large amounts of cPMP in mitochondria; consequently, activities of all molybdenum enzymes were strongly reduced (R.R.M., unpublished observations).

Only a few cases and conditions of limited molybdate availability have been reported so far⁵,⁹⁰. Among these, the shortage of molybdenum in Australian farmland triggered excessive fertilization, resulting in molybdenum overload of the soil that caused pathological symptoms of molybdenosis in animals; this, in particular in ruminants, triggered secondary copper deficiency⁹¹. Later, these molybdenum-induced conditions of copper deficiency revealed the pathology of two copper-homeostasis disorders: Menkes disease (copper deficiency) and Wilson’s disease (copper overload)⁹² (see page 823). Consequently, potent copper chelators such as tetrathiomolybdates were used to treat Wilson’s disease and a number of other disorders that are linked to copper homeostasis, such as neurodegeneration, cancer and inflammation⁹³.

Another antagonism between molybdenum and copper has been found recently. The crystal structure of Cnx1G, which catalyses MPT adenylylation, revealed the presence of a covalently bound copper ion (most probably Cu¹⁺) at the MPT dithiolate in both the substrate- and product-bound states⁶⁹. The function of copper during Moco biosynthesis is still unknown. It may participate in the sulphur-transfer reaction enabled by MPT synthase, act as a protecting group for MPT and/or function within molybdenum insertion⁷². In vitro studies suggested a competition between copper and molybdenum during Moco formation⁶⁹, ultimately raising the question of whether Moco biosynthesis might be affected under conditions of copper overload or deficiency⁹².

---

### Molybdenum cofactor deficiency

Human Moco deficiency (MoCD) results in the complete loss of sulphite oxidase, xanthine oxidase and aldehyde oxidase activity. Patients diagnosed with MoCD are classified into three groups according to the affected steps within the biosynthetic pathway (Fig. 7a). They are characterized by progressive neurological damage, leading to early childhood death in most cases⁹⁴. Symptoms are mainly caused by the deficiency of sulphite oxidase protecting the organs (in particular the brain) from elevated concentrations of toxic sulphite.

The function of sulphite oxidase is important, as it represents the last step in the oxidative degradation of sulphur-containing amino acids and lipids. Mainly in liver, sequential events in cysteine catabolism through cysteine sulphinate (cytosol) and β-sulphinyl pyruvate (mitochondria) lead to the formation of sulphite (Fig. 7b). Sulphite oxidase, localized in the intermembrane space, oxidizes sulphite to sulphate. Under conditions of Moco or sulphite oxidase deficiency, sulphite accumulates in plasma and serum, crosses the blood–brain barrier and rapidly triggers neuronal death⁹⁴. Impaired ATP synthesis has been suggested as one possible mechanism of sulphite toxicity⁹⁵. Sulphite accumulation also triggers the reduction of cystine (Fig. 7b), the main carrier for cysteine in serum and plasma, which consequently causes the formation of S-sulphocysteine⁹⁴ (Fig. 7b), a potential agonist of glutamate receptors⁹⁶. The latter may explain the observed seizures, convulsions, contractions and twitching associated with MoCD, causing damage of cortical neurons as documented by abnormal magnetic resonance imaging of the brain and loss of white matter⁹⁴.

To study MoCD, an animal model with *Mocs1* knockout has been constructed⁹⁷. Homozygous mice displayed a severe phenotype that reflects all biochemical characteristics of human Moco-deficient patients. They failed to thrive, and died within the first 12 days of life. The lethal

**Figure 7 | Human Moco deficiency.**  
**a,** Classification of Moco-deficient patients⁵³ according to the three distinguishable steps in human Moco biosynthesis. Genes encoding proteins catalysing the individual steps are shown. Type-A patients cannot form cPMP, whereas type-B patients accumulate cPMP, which is excreted in the urine. So far, only one type-C patient has been described, with a deletion of gephyrin due to an early stop codon in the gephyrin gene⁹⁹,¹⁰⁰, a protein also needed for the formation of inhibitory synapses. **b,** Degradation pathway of methionine and cysteine. Catabolic intermediates are depicted and metabolites that accumulate in Moco-deficient patients owing to increased sulphite concentrations are S-sulphocysteine, thiosulphate and taurine. Simultaneously, cystine concentrations are very low. S-Sulphocysteine is thought to make a major contribution to neurodegeneration because of its structural similarity to glutamate. We note that sulphite is formed from spontaneous decomposition of β-sulphinyl pyruvate. Sulphite crosses the mitochondrial inner membrane and is oxidized in the intermembrane space by molybdenum-dependent sulphite oxidase.

phenotype could be effectively neutralized by repeated injection of cPMP, which was purified from *E. coli*⁹⁸. Mice treated with cPMP developed normally, gained weight and reached adulthood and fertility like their wild-type littermates. Notably, withdrawal of cPMP from *Mocs1*-knockout mice caused death within 10–14 days. These promising results were verified by a recent clinical trial (G.S., unpublished observations).

### Conclusion

The versatile redox chemistry of molybdenum is mirrored by the plethora and complexity of enzymes using molybdenum and, to some extent, tungsten. Nature has developed two very different systems to control the redox state and catalytic power of molybdenum. Although the contribution of each cofactor scaffold to the overall chemistry of a given enzyme requires further investigation, it is likely that the various scaffolds represent pathways generating sulphur-containing chelators that can trap, activate and control the transition elements as catalysts.

Future research in the field of molybdenum enzymes is likely to focus on the mechanistic details of cofactor biosynthesis, cofactor allocation and the functions of cofactors in particular enzymes. Further understanding of the differences between molybdenum and tungsten biochemistry will help to explain the unique presence of molybdenum in eukaryotes and to answer fundamental questions regarding functional specificity, metal selectivity, regulation, allocation, compartmentalization and assembly of this fascinating family of enzymes and cofactors.

As seen for both FeMo-co and Moco biosynthesis, different assembly machineries can use similar mechanisms, such as the radical SAM-based chemistry and the nucleotide-assisted molybdenum activation, which are employed for the synthesis of both cofactors. The role of molybdenum in other homeostatic circuits, such as copper and iron metabolism, awaits further investigation, which could address questions regarding the pathophysiology of related metabolic disorders. Finally, the biosynthetic machineries of these cofactors or the enzymes themselves (for example nitrogenase) are ancient natural inventions. Some of the proteins are precursors of those with specialized functions in eukaryotes, such as ubiquitin-like protein conjugation (MPT synthesis)⁶⁴ and G-protein-based signalling³; others, such as gephyrin, which is crucial for synaptogenesis⁹⁹, have gathered additional functions.

---

1. Dos Santos, P.C., Dean, D.R., Hu, Y. & Ribbe, M.W. Formation and insertion of the nitrogenase iron-molybdenum cofactor. *Chem. Rev.* **104**, 1159–1173 (2004).
2. Schwarz, G. Molybdenum cofactor biosynthesis and deficiency. *Cell. Mol. Life Sci.* **62**, 2792–2810 (2005).
3. Hille, R. Molybdenum and tungsten in biology. *Trends Biochem. Sci.* **27**, 360–367 (2002).
4. Zhang, Y. & Gladyshev, V.N. Molybdoproteomes and evolution of molybdenum utilization. *J. Mol. Biol.* **379**, 881–899 (2008).
5. Bevers, L.E., Hagedorn, P.-L. & Hagen, W.R. The bioinorganic chemistry of tungsten. *Coord. Chem. Rev.* **253**, 269–290 (2009).
6. Burgess, B.K. & Lowe, D.J. Mechanism of molybdenum nitrogenase. *Chem. Rev.* **96**, 2983–3012 (1996).
7. Einsle, O. *et al.* Nitrogenase MoFe-protein at 1.16 Å resolution: a central ligand in the FeMo-cofactor. *Science* **297**, 1696–1700 (2002).
8. Rajagopalan, K.V. & Johnson, J.L. The pterin molybdenum cofactors. *J. Biol. Chem.* **267**, 10199–10202 (1992).
9. Eady, R.R. Structure-function relationships of alternative nitrogenases. *Chem. Rev.* **96**, 3013–3030 (1996).
10. Schindelin, H. *et al.* Structure of ADP·AIF₄⁻-stabilized nitrogenase complex and its implications for signal transduction. *Nature* **387**, 370–376 (1997).
11. Rothery, R.A., Workun, G.J. & Weiner, J.H. The prokaryotic complex iron-sulfur molybdoenzyme family. *Biochim. Biophys. Acta* **1778**, 1897–1929 (2008). This paper is a systematic and comprehensive overview of prokaryotic molybdenum-containing enzymes of the DMSOR family.
12. Hille, R. Molybdenum-containing hydroxylases. *Arch. Biochem. Biophys.* **433**, 107–116 (2005). This paper is a comprehensive overview of structure and reaction mechanisms of molybdenum-containing hydroxylases of the xanthine oxidase family.
13. Feng, C., Tollin, G. & Enemark, J.H. Sulfite oxidizing enzymes. *Biochim. Biophys. Acta* **1774**, 527–539 (2007).
14. Moura, J.J., Brondino, C.D., Trincao, J. & Romao, M.J. Mo and W bis-MGD enzymes: nitrate reductases and formate dehydrogenases. *J. Biol. Inorg. Chem.* **9**, 791–799 (2004).
15. Bertero, M.G. *et al.* Insights into the respiratory electron transfer pathway from the structure of nitrate reductase A. *Nature Struct. Biol.* **10**, 681–687 (2003).
16. Kisker, C. *et al.* Molecular basis of sulfite oxidase deficiency from the structure of sulfite oxidase. *Cell* **91**, 973–983 (1997).
17. Fischer, K. *et al.* Crystal structure of the yeast nitrate reductase molybdenum domain provides insight into eukaryotic nitrate assimilation. *Plant Cell* **17**, 1167–1179 (2005).
18. Eilers, T. *et al.* Identification and biochemical characterization of *Arabidopsis thaliana* sulfite oxidase. A new player in plant sulfur metabolism. *J. Biol. Chem.* **276**, 46989–46994 (2001).
19. Schrader, N. *et al.* The crystal structure of plant sulfite oxidase provides insights into sulfite oxidation in plants and animals. *Structure* **11**, 1251–1263 (2003).
20. Kappler, U. & Bailey, S. Molecular basis of intramolecular electron transfer in sulfite-oxidizing enzymes is revealed by high resolution structure of a heterodimeric complex of the catalytic molybdopterin subunit and a c-type cytochrome subunit. *J. Biol. Chem.* **280**, 24999–25007 (2005).
21. Hansch, R. *et al.* Plant sulfite oxidase as novel producer of H₂O₂: combination of enzyme catalysis with a subsequent non-enzymatic reaction step. *J. Biol. Chem.* **281**, 6884–6888 (2006).
22. Campbell, W.H. Nitrate reductase structure, function and regulation: bridging the gap between biochemistry and physiology. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **50**, 277–303 (1999).
23. Mendel, R.R. & Bittner, F. Cell biology of molybdenum. *Biochim. Biophys. Acta* **1763**, 621–635 (2006).
24. Enroth, C. *et al.* Crystal structures of bovine milk xanthine dehydrogenase and xanthine oxidase: structure-based mechanism of conversion. *Proc. Natl Acad. Sci. USA* **97**, 10723–10728 (2000).
25. Truglio, J.J. *et al.* Crystal structures of the active and alloxanthine-inhibited forms of xanthine dehydrogenase from *Rhodobacter capsulatus*. *Structure* **10**, 115–125 (2002).
26. Garattini, E. *et al.* Mammalian molybdo-flavoenzymes, an expanding family of proteins: structure, genetics, regulation, function and pathophysiology. *Biochem. J.* **372**, 15–32 (2003).
27. Seo, M. *et al.* The *Arabidopsis* aldehyde oxidase 3 (AAO3) gene product catalyzes the final step in abscisic acid biosynthesis in leaves. *Proc. Natl Acad. Sci. USA* **97**, 12908–12913 (2000).
28. Neumann, M. *et al.* A periplasmic aldehyde oxidoreductase represents the first molybdopterin cytosine dinucleotide cofactor containing molybdo-flavoenzyme from *Escherichia coli*. *FEBS J.* **276**, 2762–2774 (2009).
29. Dobbek, H. *et al.* Catalysis at a dinuclear [CuSMo(=O)OH] cluster in a CO dehydrogenase resolved at 1.1-Å resolution. *Proc. Natl Acad. Sci. USA* **99**, 15971–15976 (2002).
30. Harrison, R. Structure and function of xanthine oxidoreductase: where are we now? *Free Radic. Biol. Med.* **33**, 774–797 (2002).
31. Havemeyer, A. *et al.* Identification of the missing component in the mitochondrial benzamidoxime prodrug-converting system as a novel molybdenum enzyme. *J. Biol. Chem.* **281**, 34796–34802 (2006).
32. Groysman, S. & Holm, R. Biomimetic chemistry of iron, nickel, molybdenum, and tungsten in sulfur-ligated protein sites. *Biochemistry* **48**, 2310–2320 (2009). This paper is a comprehensive review of the chemical synthesis of analogues of pterin cofactors of molybdenum enzymes and the FeMo-co and P-cluster of nitrogenase MoFe-protein.
33. Hu, Y. *et al.* Assembly of nitrogenase MoFe protein. *Biochemistry* **47**, 3973–3981 (2008). This paper reports a recent development in the biosynthesis of the FeMo-co and P-cluster of nitrogenase MoFe-protein.
34. Smith, A.D. *et al.* NifS-mediated assembly of [4Fe-4S] clusters in the N- and C-terminal domains of the NifU scaffold protein. *Biochemistry* **44**, 12955–12969 (2005).
35. Johnson, D.C., Dean, D.R., Smith, A.D. & Johnson, M.K. Structure, function, and formation of biological iron-sulfur clusters. *Annu. Rev. Biochem.* **74**, 247–281 (2005).
36. Allen, R.M., Chatterjee, R., Ludden, P.W. & Shah, V.K. Incorporation of iron and sulfur from NifB cofactor into the iron-molybdenum cofactor of dinitrogenase. *J. Biol. Chem.* **270**, 26890–26896 (1995).
37. Schmid, B. *et al.* Structure of a cofactor-deficient nitrogenase MoFe protein. *Science* **296**, 352–356 (2002).
38. Curatti, L., Ludden, P.W. & Rubio, L.M. NifB-dependent in vitro synthesis of the iron-molybdenum cofactor of nitrogenase. *Proc. Natl Acad. Sci. USA* **103**, 5297–5301 (2006).
39. Hu, Y., Fay, A.W. & Ribbe, M.W. Identification of a nitrogenase FeMo cofactor precursor on NifEN complex. *Proc. Natl Acad. Sci. USA* **102**, 3236–3241 (2005).
40. Corbett, M.C. *et al.* Structural insights into a protein-bound iron-molybdenum cofactor precursor. *Proc. Natl Acad. Sci. USA* **103**, 1238–1243 (2006). This paper is the first X-ray absorption spectroscopy/extended X-ray absorption fine structure (XAS/EXAFS)-based structural analysis of the FeMo-co precursor on NifEN.
41. Hu, Y. *et al.* FeMo cofactor maturation on NifEN. *Proc. Natl Acad. Sci. USA* **103**, 17119–17124 (2006).
42. Zheng, L., White, R.H. & Dean, D.R. Purification of the *Azotobacter vinelandii nifV*-encoded homocitrate synthase. *J. Bacteriol.* **179**, 5963–5966 (1997).
43. Imperial, J., Ugalde, R.A., Shah, V.K. & Brill, W.J. Role of the *nifQ* gene product in the incorporation of molybdenum into nitrogenase in *Klebsiella pneumoniae*. *J. Bacteriol.* **158**, 187–194 (1984).
44. Hernandez, J.A. *et al.* Metal trafficking for nitrogen fixation: NifQ donates molybdenum to NifEN/NifH for the biosynthesis of the nitrogenase FeMo-cofactor. *Proc. Natl Acad. Sci. USA* **105**, 11679–11684 (2008).
45. Hu, Y. *et al.* Nitrogenase Fe protein: a molybdate/homocitrate insertase. *Proc. Natl Acad. Sci. USA* **103**, 17125–17130 (2006).
46. Georgiadis, M.M. *et al.* Crystallographic structure of the nitrogenase iron protein from *Azotobacter vinelandii*. *Science* **257**, 1653–1659 (1992).
47. Rubio, L.M. *et al.* Cloning and mutational analysis of the γ gene from *Azotobacter vinelandii* defines a new family of proteins capable of metallocluster binding and protein stabilization. *J. Biol. Chem.* **277**, 14299–14305 (2002).
48. Hu, Y. *et al.* Molecular insights into nitrogenase FeMoco insertion: TRP-444 of MoFe protein α-subunit locks FeMoco in its binding site. *J. Biol. Chem.* **281**, 30534–30541 (2006).

NATURE|Vol 460|13 August 2009

49. Hu, Y., Fay, A. W. & Ribbe, M. W. Molecular insights into nitrogenase FeMo cofactor insertion: the role of His 362 of the MoFe protein α subunit in FeMo cofactor incorporation. *J. Biol. Inorg. Chem.* **12**, 449-460 (2007).

50. Fay, A. W., Hu, Y., Schmid, B. & Ribbe, M. W. Molecular insights into nitrogenase FeMoco insertion — the role of His 274 and His 451 of MoFe protein α subunit. *J. Inorg. Biochem.* **101**, 1630-1641 (2007).

51. Schwarz, G. & Mendel, R. R. Molybdenum cofactor biosynthesis and molybdenum enzymes. *Annu. Rev. Plant Biol.* **57**, 623-647 (2006).

52. Schwarz, G., Hagedoorn, P. L. & Fischer, K. in *Molecular Microbiology of Heavy Metals* (eds Nies, D. H. & Silver, S.) 421-451 (Springer, 2007).

53. Reiss, J. & Johnson, J. L. Mutations in the molybdenum cofactor biosynthetic genes MOCS1, MOCS2, and GEPH. *Hum. Mutat.* **21**, 569-576 (2003).

54. Wuebbens, M. M. & Rajagopalan, K. V. Structural characterization of a molybdopterin precursor. *J. Biol. Chem.* **268**, 13493-13498 (1993).

55. Hanzelmann, P. & Schindelin, H. Binding of 5′-GTP to the C-terminal FeS cluster of the radical S-adenosylmethionine enzyme MoaA provides insights into its mechanism. *Proc. Natl Acad. Sci. USA* **103**, 6829-6834 (2006). This paper describes the structural basis of radical SAM-based conversion of GTP into cPMP.

56. Wuebbens, M. M. & Rajagopalan, K. V. Investigation of the early steps of molybdopterin biosynthesis in *Escherichia coli* through the use of in vivo labeling studies. *J. Biol. Chem.* **270**, 1082-1087 (1995).

57. Santamaria-Araujo, J. A. *et al.* The tetrahydropyranopterin structure of the sulfur-free and metal-free molybdenum cofactor precursor. *J. Biol. Chem.* **279**, 15994-15999 (2004).

58. Daniels, J. N., Wuebbens, M. M., Rajagopalan, K. V. & Schindelin, H. Crystal structure of a molybdopterin synthase-precursor Z complex: insight into its sulfur transfer mechanism and its role in molybdenum cofactor deficiency. *Biochemistry* **47**, 615-626 (2008).

59. Hanzelmann, P., Schwarz, G. & Mendel, R. R. Functionality of alternative splice forms of the first enzymes involved in human molybdenum cofactor biosynthesis. *J. Biol. Chem.* **277**, 18303-18312 (2002).

60. Gutzke, G., Fischer, B., Mendel, R. R. & Schwarz, G. Thiocarboxylation of molybdopterin synthase provides evidence for the mechanism of dithiolene formation in metal-binding pterins. *J. Biol. Chem.* **276**, 36268-36274 (2001).

61. Rudolph, M. J., Wuebbens, M. M., Rajagopalan, K. V. & Schindelin, H. Crystal structure of molybdopterin synthase and its evolutionary relationship to ubiquitin activation. *Nature Struct. Biol.* **8**, 42-46 (2001).

62. Wuebbens, M. M. & Rajagopalan, K. V. Mechanistic and mutational studies of *Escherichia coli* molybdopterin synthase clarify the final step of molybdopterin biosynthesis. *J. Biol. Chem.* **278**, 14523-14532 (2003).

63. Stallmeyer, B. *et al.* Human molybdopterin synthase gene: identification of a bicistronic transcript with overlapping reading frames. *Am. J. Hum. Genet.* **64**, 698-705 (1999).

64. Lake, M. W., Wuebbens, M. M., Rajagopalan, K. V. & Schindelin, H. Mechanism of ubiquitin activation revealed by the structure of a bacterial MoeB-MoaD complex. *Nature* **414**, 325-329 (2001).

65. Lee, I. & Schindelin, H. Structural insights into E1-catalyzed ubiquitin activation and transfer to conjugating enzymes. *Cell* **134**, 268-278 (2008).

66. Leimkühler, S. & Rajagopalan, K. V. A sulfurtransferase is required in the transfer of cysteine sulfur in the in vitro synthesis of molybdopterin from precursor Z in *Escherichia coli*. *J. Biol. Chem.* **276**, 22024-22031 (2001).

67. Forlani, F. *et al.* The cysteine-desulfurase IscS promotes the production of the rhodanese RhdA in the persulfurated form. *FEBS Lett.* **579**, 6786-6790 (2005).

68. Matthies, A., Rajagopalan, K. V., Mendel, R. R. & Leimkuhler, S. Evidence for the physiological role of a rhodanese-like protein for the biosynthesis of the molybdenum cofactor in humans. *Proc. Natl Acad. Sci. USA* **101**, 5946-5951 (2004).

69. Kuper, J. *et al.* Structure of the molybdopterin-bound Cnx1G domain links molybdenum and copper metabolism. *Nature* **430**, 803-806 (2004). This paper reports the first protein-complex structures of Moco intermediates and the identification of a novel reaction intermediate essential for metal transfer.

70. Bevers, L. E. *et al.* Function of MoaB proteins in the biosynthesis of the molybdenum and tungsten cofactors. *Biochemistry* **47**, 949-956 (2008).

71. Llamas, A., Mendel, R. R. & Schwarz, G. Synthesis of adenylated molybdopterin: an essential step for molybdenum insertion. *J. Biol. Chem.* **279**, 55241-55246 (2004).

72. Llamas, A. *et al.* The mechanism of nucleotide-assisted molybdenum insertion into molybdopterin. A novel route toward metal cofactor assembly. *J. Biol. Chem.* **281**, 18343-18350 (2006).

73. Bevers, L. E., Hagedoorn, P. L., Krijger, G. C. & Hagen, W. R. Tungsten transport protein A (WtpA) in *Pyrococcus furiosus*: the first member of a new class of tungstate and molybdate transporters. *J. Bacteriol.* **188**, 6498-6505 (2006).

74. Stevenson, C. E. M. *et al.* Crystal structure of the molybdenum cofactor biosynthesis protein MobA from *Escherichia coli* at near-atomic resolution. *Structure* **8**, 1115-1125 (2000).

75. Vergnes, A. *et al.* Involvement of the molybdenum cofactor biosynthetic machinery in the maturation of the *Escherichia coli* nitrate reductase A. *J. Biol. Chem.* **279**, 41398-41403 (2004).

76. Bittner, F., Oreb, M. & Mendel, R. R. ABA3 is a molybdenum cofactor sulfurase

required for activation of aldehyde oxidase and xanthine dehydrogenase in *Arabidopsis thaliana*. *J. Biol. Chem.* **276**, 40381-40384 (2001).

77. Heidenreich, T., Wollers, S., Mendel, R. R. & Bittner, F. Characterization of the NifS-like domain of ABA3 from *Arabidopsis thaliana* provides insight into the mechanism of molybdenum cofactor sulfuration. *J. Biol. Chem.* **280**, 4213-4218 (2005).

78. Wollers, S. *et al.* Binding of sulfurated molybdenum cofactor to the C-terminal domain of ABA3 from *Arabidopsis thaliana* provides insight into the mechanism of molybdenum cofactor sulfuration. *J. Biol. Chem.* **283**, 9642-9650 (2008). This paper describes the mechanism and platform principle of Moco sulfuration in eukaryotes.

79. Neumann, M. *et al.* Rhodobacter capsulatus XdhC is involved in molybdenum cofactor binding and insertion into xanthine dehydrogenase. *J. Biol. Chem.* **281**, 15701-15708 (2006).

80. Pau, R. N. & Lawson, D. M. Transport, homeostasis, regulation, and binding of molybdate and tungstate to proteins. *Met. Ions Biol. Syst.* **39**, 31-74 (2002).

81. Ataya, F. S. *et al.* Mcp1 encodes the molybdenum cofactor carrier protein in *Chlamydomonas reinhardtii* and participates in protection, binding, and storage functions of the cofactor. *J. Biol. Chem.* **278**, 10885-10890 (2003).

82. Fischer, K. *et al.* Function and structure of the molybdenum cofactor carrier protein from *Chlamydomonas reinhardtii*. *J. Biol. Chem.* **281**, 30186-30194 (2006).

83. Sargent, F. Constructing the wonders of the bacterial world: biosynthesis of complex enzymes. *Microbiology* **153**, 633-651 (2007).

84. Hollenstein, K., Frei, D. C. & Locher, K. P. Structure of an ABC transporter in complex with its binding protein. *Nature* **446**, 213-216 (2007). This paper provides the first structural insight into the family of molybdate and tungstate ABC transporters

85. Hollenstein, K. *et al.* Distorted octahedral coordination of tungstate in a subfamily of specific binding proteins. *J. Biol. Inorg. Chem.* **14**, 663-672 (2009).

86. Tomatsu, H. *et al.* An *Arabidopsis thaliana* high-affinity molybdate transporter required for efficient uptake of molybdate from soil. *Proc. Natl Acad. Sci. USA* **104**, 18807-18812 (2007).

87. Tejada-Jimenez, M. *et al.* A high-affinity molybdate transporter in eukaryotes. *Proc. Natl Acad. Sci. USA* **104**, 20126-20130 (2007).

88. Baxter, I. *et al.* Variation in molybdenum content across broadly distributed populations of *Arabidopsis thaliana* is controlled by a mitochondrial molybdenum transporter (MOT1). *PLoS Genet.* **4**, e1000004 (2008).

89. Chen, S. *et al.* Functional characterization of AtATM1, AtATM2, and AtATM3, a subfamily of *Arabidopsis* half-molecule ATP-binding cassette transporters implicated in iron homeostasis. *J. Biol. Chem.* **282**, 21561-21571 (2007).

90. Suttle, N. F. The interactions between copper, molybdenum, and sulphur in ruminant nutrition. *Annu. Rev. Nutr.* **11**, 121-140 (1991).

91. Mason, J. Thiomolybdates: mediators of molybdenum toxicity and enzyme inhibitors. *Toxicology* **42**, 99-109 (1986).

92. Mercer, J. F. The molecular basis of copper-transport diseases. *Trends Mol. Med.* **7**, 64-69 (2001).

93. Brewer, G. J. Anticopper therapy against cancer and diseases of inflammation and fibrosis. *Drug Discov. Today* **10**, 1103-1109 (2005).

94. Johnson, J. L. & Duran, M. in *The Metabolic and Molecular Bases of Inherited Disease* (eds Scriver, C. R., Beaudet, A. L., Sly, W. S. & Valle, D.) 3163-3177 (McGraw-Hill, 2001).

95. Zhang, X., Vincent, A. S., Halliwell, B. & Wong, K. P. A mechanism of sulfite neurotoxicity: direct inhibition of glutamate dehydrogenase. *J. Biol. Chem.* **279**, 43035-43045 (2004).

96. Tan, W. H. *et al.* Isolated sulfite oxidase deficiency: a case report with a novel mutation and review of the literature. *Pediatrics* **116**, 757-766 (2005).

97. Lee, H.-J. *et al.* Molybdenum cofactor-deficient mice resemble the phenotype of human patients. *Hum. Mol. Genet.* **11**, 3309-3317 (2002).

98. Schwarz, G. *et al.* Rescue of lethal molybdenum cofactor deficiency by a biosynthetic precursor from *Escherichia coli*. *Hum. Mol. Genet.* **13**, 1249-1255 (2004). This paper describes the successful therapeutic use of a Moco intermediate (cPMP) to cure Moco deficiency.

99. Stallmeyer, B. *et al.* The neurotransmitter receptor-anchoring protein gephyrin reconstitutes molybdenum cofactor biosynthesis in bacteria, plants, and mammalian cells. *Proc. Natl Acad. Sci. USA* **96**, 1333-1338 (1999).

100. Reiss, J. *et al.* A mutation in the gene for the neurotransmitter receptor-clustering protein gephyrin causes a novel form of molybdenum cofactor deficiency. *Am. J. Hum. Genet.* **68**, 208-213 (2001).

Acknowledgements Support from the Deutsche Forschungsgemeinschaft (G.S., R.R.M.), the Bundesministerium für Bildung und Forschung (G.S.), the Fonds der Chemischen Industrie (G.S.), the European Union (R.R.M.) and the US National Institutes of Health (grant GM-67626; M.W.R.) is gratefully acknowledged, as is the contribution of all co-workers, especially graduate students and post-docs, during the past ten years.

Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Correspondence should be addressed to G.S. (gschwarz@uni-koeln.de).
