
Journal of  
proteome  
research  

## Organization of the Sec61 Translocon, Studied by High Resolution Native Electrophoresis

Kurt Dejgaard,$^{*,†,\#}$ Jean-Francois Theberge,$^{†,\#}$ Hannah Heath-Engel,$^{†}$ Eric Chevet,$^{§}$ Michel L. Tremblay,$^{†,\dagger}$ and David Y. Thomas$^{†}$

**Department of Biochemistry, and McGill Cancer Centre, McIntyre Medical Building, McGill University, 3655 Promenade Sir William Osler, Montreal H3B 1Y6, Canada, and GREF INSERM U889, IFR66, Université Victor Segalen Bordeaux 2, 146 rue Léo Saignat, 33076 Bordeaux, France**

Received October 7, 2009

Cotranslational translocation of polypeptides into the ER is controlled by the dynamic interaction of ribosome and translocon components. Analysis of the steps involved in this process by high resolution techniques such as gel electrophoresis is precluded by the high molecular masses of these complexes. We show, here, that modifications to standard native electrophoresis protocols can overcome these problems and lead to an increase in mass range and resolution. Using the modified technique, we show that ER ribosome anchored membrane protein (RAMP) complexes resolve into 3 stable and semistable complexes which range in size between 4 and 8 MDa and are sensitive to relevant concentrations of divalent metals. We demonstrate the molecular composition of the complexes and identify a number of modular components that differentiate them. The components that are common to all three RAMP complexes include the OST translocon subcomplex, Glucosidase I and microtubule tethering protein CLIMP63. The two larger complexes further include the kinesin motor binding protein p180 and Sec61, and the largest complex includes the TRAP translocon component and apoptotic regulator BAP31. On the lumenal side, the BiP cochaperone ERdj3 resides with the three RAMP complexes. Our observations may hint at how subcompartmentalization is achieved in the ER membrane continuum.

### Keywords: High Resolution Native Electrophoresis • Protein Complexes • Organelle Complex Proteomics

---

### Introduction

Macromolecular complexes and their cellular functions have been studied through a variety of approaches, such as conventional purification of protein complexes, or by *in vitro* assays and by reconstitution of their functions. More recent approaches have made use of genomic information to map interactomes using data from yeast two-hybrid and affinity-tagging approaches: Yeast two-hybrid studies with both *Saccharomyces cerevisiae* and *Caenorhabditis elegans* on a genome-wide scale have been performed to create comprehensive protein interaction maps.$^{1–3}$ In an equally comprehensive approach, yeast genes have been fused with TAP or FLAG affinity purification tags and complexes isolated and the components identified by mass spectrometry.$^{4–6}$ These approaches all require genetic intervention, insofar as genes of interest must be expressed as tagged fusion proteins; the development of methods to determine protein interactions and

* Corresponding author. E-mail: kurt.dejgaard@mcgill.ca. Fax: +514 398 7384.
+ Department of Biochemistry, McGill University.
# Both authors contributed equally to this work.
† McGill Cancer Centre, McGill University.
§ Université Victor Segalen Bordeaux 2.

10.1021/pr900900x © 2010 American Chemical Society

Journal of Proteome Research **2010**, 9, 1763–1771 1763  
Published on Web 01/29/2010

research articles

Table 1. Wide Molecular Weight Span 'WiSE Native' Recipe for 2–7% Gradient Minigels $^{a}$

| acrylamide 30%, 29:1 | 10× gel buffer | glycerol 50% | rhinohide | TDOC 10% | water | APS 10% | TEMED | total |
|----------------------|-----------------|--------------|-----------|---------|-------|--------|-------|-------|
| 7%                   | 1.2 mL          | 0.5 mL       | 1 mL      | -       | 0.1 mL | 2.2 mL | 12.5 μL | 1.7 μL | 5 mL |
| 2%                   | 0.33 mL         | 0.5 mL       | -         | 1 mL    | 0.1 mL | 3.1 mL | 30 μL  | 2.3 μL | 5 mL |

Cathode buffer:  
1× Gel buffer (50 mM Bis-Tris Acetate pH 7.0), 0.2% TDOC  

Anode buffer:  
1× Gel buffer  

Extraction buffer A:  
1× Gel buffer, 1% TDOC, 5 mM Iodoacetamide (Bromophenol Blue and 10% glycerol added prior to sample loading).

$^{a}$ Gels and running buffers are open to modification with additional salts and changes to pH.

Rhinohide polyacrylamide gel strengthener was purchased from Molecular Probes/Invitrogen. Sodium taurodeoxycholic acid (TDOC) was from Sigma. Bis-Tris was from Fisher Biotech, and the acrylamide/bisacrylamide 29:1 was from BioRad. BS³ cross-linking reagent was from Pierce.

Monoclonal antibodies against pyruvate dehydrogenase E1α were from Molecular Probes; against proteasomal 20S proteins α1, 2, 3, 5, 6 ε from Affinity. Polyclonal antibodies against TCP1ε, Calnexin, PDI, and Erp57 were from Stressgen; against Grp170 and Erp72 were gifts from Michael Greene (Washington University, St. Louis, MO); and against p97 were a gift from Martin Latterich (Université de Montréal); against the ribosomal L4 protein, Grp94, Sec61α, and TRAPα and ribophorin I were gifts from Christopher Nicchitta; against p62/CLIMP63 were a gift from Dorothy Mundy (Texas State University); against p23/p24δ were a gift from Jean Gruenberg (Zurich); against the p180 ribosome receptor were a gift from Kiyoko Ogawa-Goto²⁴ and against glucosidase I were a gift from Ernst Bause and Christof Volker.²⁵ Horseradish peroxidase (HRP)-tagged species-specific anti-immunoglobulin antibodies were from Santa Cruz Biotechnology. Antibody binding was visualized with Lumi-Light Western Blotting Substrate 1 and 2, from Roche.

Sample Preparation for Native Electrophoresis. Semiconfluent HeLa and NIH-3T3 cells were washed twice with PBS, harvested from a 10 cm tissue culture dish, and centrifuged into a pellet at 3000 rpm in Eppendorf tubes. The cell pellets were extracted in a minimum of 3 pellet volumes of buffer A (1% TDOC, 50 mM Bis-Tris, pH 7.5, and 5 mM iodoacetamide) for 10 min on ice. CPM were extracted directly in a minimum of 3 vol of buffer A. Following a 10 min centrifugation at 100,000 g at 4°C, glycerol was added to 10% final concentration, followed by 1 μL of 1% Bromophenol Blue and loaded onto a native gel. For radiolabeling, cells were propagated in 24-well tissue culture plates, and at semiconfluency, cells were exposed to 50 μCi ³⁵S-Methionine overnight in methionine-free labeling medium (Gibco) supplemented with dialyzed 10% FBS.

Chemical Cross-Linking. Bis[sulfosuccinimidyl] suberate (BS³) was dissolved in DMSO to a stock concentration of 50 mM. For BS³ cross-linking, CPM were washed and extracted with 1% TDOC in potassium acetate buffer (115 mM potassium acetate; 25 mM Hepes, pH 7). After centrifugation at 100,000 g, BS³ was added to a final concentration of 1.5 mM and incubated at 4°C for 30 min. Excess cross-linker was quenched by glycine (final concentration of 20 mM) and samples were then either mixed with Laemmli buffer for SDS gel electrophoresis²⁶ or loaded onto native electrophoresis gels as described above.

Native Gel Preparation. The 1.5 mm thick, 2–7% or 4–7% gradient native mini-gels were cast in a BioRad minigel system

Materials and Methods

Reagents and Antibodies. The HeLa cell line was provided by Imad Gallouzi and the 3T3 cell line was provided by Nahum Sonenberg both of the Dept. of Biochemistry, McGill University. Cells were propagated under standard conditions in Dulbecco's Modified Eagle Medium (DMEM) supplemented with 10% Fetal Bovine Serum (FBS), glutamine, and antibiotics. EDTA-washed canine pancreatic microsomes (CPM) were prepared as described previously.²³ (CPM should be stored at –80°C and used within 6 months of preparation.)

Organization of the Sec61 Translocon

Jagow,^{8,27} but with 0.2% TDOC substituting for the Coomassie Blue G 250 in the cathode buffer (Supplemental Figure 2).

Second-Dimension SDS-PAGE and Western Blotting. Native gels were stained for 0.5 h with Coomassie Blue R and destained according to standard procedures. (Note: Rhinohide gels will destain in the high methanol content of a conventional Coomassie destainer, but Rhinohide gels cannot be colloidal Coomassie stained as Rhinohide itself binds Coomassie Blue.) Coomassie stained native gels were then washed in water. Strips of stained gels were excised and equilibrated in Laemmli sample buffer for 20–30 min and loaded onto regular SDS acrylamide gels (either 6% or 10% acrylamide or 8–12% acrylamide gradient gels). At the end of the second-dimension SDS electrophoresis (under conditions as by Laemmli),^{26} the gel was either silver stained,^{28} or, for radiolabeled samples, dried and exposed to Kodak Xomat film or transferred to nitrocellulose membranes^{29} and the subsequent Western blots visualized with ECL.

Mass Spectrometry. Complex III was excised from a native gel, reduced (10 mM DTT, 10 min) and alkylated (55 mM Iodoacetamide, 30 min) and subjected to digestion by trypsin (12 ng/μL), overnight, according to standard procedures.^{30}

The resulting peptide digest was analyzed by reverse phase separation, followed by MSMS on a Bruker HCT Ultra ion trap mass spectrometer. Data files were formatted to mgf files with Bruker ‘Compass Data Analysis’ software (standard settings) and searched on the mammalian NCBI database (version NCBInr 20080907 and later), using Mascot v. 2.2 (Matrix Sciences) with active autovalidation. The summarized data presented in Table 2 were selected, based on a cutoff at a minimum of 3 unique peptides with a *p*-value of *p* < 0.05 (Mowse score of 50 or better).

Molecular Weight Estimates. To estimate the molecular weights of proteins separated by “WiSE Native” electrophoresis, the following markers were mapped by Western blotting using a mouse NIH 3T3 cell extract (Figure 1b): 7MDa (pyruvate dehydrogenase),^{31} 4MDa (ribosomal protein L4), 950 kDa (CCT complex),^{32} 700 kDa (20S proteasome),^{33} 600 kDa (p97),^{34} 400 and 100 kDa (Grp94, Supplemental Figure 4).

Results

Electrophoretic Principles and Conditions. Most native charge shift gel electrophoretic techniques use zwitterionic running buffers. At near-isoelectric pH, zwitterionic running buffers lead to stacking electrophoresis or isotachophoresis (Supplemental Figures 1 and 2), and require relatively steep acrylamide gradients for discontinuous protein separation. The isotachophoresis also invites aggregation during electrophoresis as proteins are concentrated into the restricted space of a stacking zone.

As visualized with two-dimensional (2D) TDOC/SDS native gels in Supplemental Figure 2, protein aggregation in charge shift TDOC electrophoresis can be substantially reduced if zwitterions are eliminated from native electrophoresis, and proteins therefore not concentrated into a stacking zone (changing the electrophoretic principle from discontinuous isotachophoresis to continuous electrophoresis). Combined with a reduction in initial acrylamide gradient concentration to 2%, facilitated by the gel stabilizing component ‘Rhinohide’,^{35} charge shift electrophoresis using 0.2% TDOC as the charge shift agent (Coomassie Blue binds to and is incompatible with Rhinohide, in the absence of organic solvents), the resulting electrophoretic protocol can separate proteins and

Journal of Proteome Research • Vol. 9, No. 4, 2010 1765

research articles

Table 2. Gene Products Identified by at Least 3 Peptides via Mass Spectrometry of 'WISE' Native Gel Ribosomal Membrane Complex III

| function               | protein                | gi          | seq. cov.    | I blot$^a$ | II blot$^a$ | III blot$^a$ |
|------------------------|------------------------|-------------|--------------|------------|-------------|---------------|
| Ribosome               | 40 most abundant       |             | 40% (ave)    | +++        | +++         | +++           |
| OST                    | Ribophorin I          | 73984484    |              | (+++)      | (+++)       | (+++)         |
|                        | Ribophorin II         | 73991908    |              |            |             |               |
|                        | STT3                   | 73954525    | 36% (ave)    | +++        | +++         | +++           |
|                        | STT3B                  | 30578410    |              |            |             |               |
|                        | OST 48                 | 50979164    |              |            |             |               |
|                        | Glucosidase I          | 73980965    | 20%          | +++        | +++         | +++           |
|                        | BAP 31                 | 57112869    | 26%          | -          | +/-         | +++           |
| Signal peptidases      | sig.pep.compl. 2       | 50979162    | 25% (ave)    |            |             |               |
|                        | SEC 11-like 3          | 50979148    |              |            |             |               |
| Translocon             | Sec61 alpha           | 7019415     | 15%          | -          | +           | +++           |
| TRAP                   | TRAP alpha            | 551638      | 24% (ave)    | -          | -/+         | +++           |
|                        | TRAP delta            | 57112873    |              |            |             |               |
| Kinectins              | p180                  | 50978924    | 36%          |            | +++         | +++           |
|                        | Kinectin              | 34098395    | 7%           |            |             |               |
| Microtubule tethering  | CLIMP 63              | 73969959    | 51%          | +++        | +++         | +++           |
| Chaperones             | Erdj 3                | 172046690   | 25%          | +++        | +++         | +++           |
| ER exit sites          | p24s                  |             |              | -          | -           | -             |
|                        | surf 4                | 73967690    | 20% (ave)    |            |             |               |
|                        | transm. traffic.pl    | 73963665    |              |            |             |               |
| Unknown membrane proteins | KIAA 0090           | 149694325   | 11%          |            |             |               |
|                        | KIAA 0152             | 194043083   | 34%          |            |             |               |
| Others                 | CG3408                | 73966367    | 31%          |            |             |               |

$^a$ Blot data for Ribophorin I (in brackets), not included in Figures 2 and 3.

protein complexes ranging from a low kilodalton (kDa) to a high megadalton (MDa) range in a native gel (Figure 1, Supplemental Figure 3d).

Continuous electrophoresis did not require high acrylamide concentrations to separate lower $M_w$ proteins and complexes nor did it require acrylamide gradients to separate complexes spanning a wide range of molecular weights (Supplemental Figure 2e) and it separates proteins and protein complexes at a resolution comparable to the well-known denaturing SDS electrophoresis (Figure 1a).

Ribosome/Translocon Complex Resolution. We used the modified native electrophoresis protocol to analyze canine pancreas microsomes (CPM) enriched for rough ER (EDTA washed and pelleted during preparation to remove polysomes). A number of experimental conditions were varied, including concentrations of calcium or magnesium ions to extracts and gels, similar to those found in the ER (we applied concentrations between 0.1 and 1 mM (Figure 3)). ER membrane-attached ribosomes resolved into three major populations under those conditions: Two lower molecular weight populations that are present in roughly similar amounts, separated in size by approximately 1.5 MDa (Figures 2a and 3a, complexes I and II. Neither of these 2 complexes are affected by presence of calcium or magnesium ions, nor by EDTA added to both sample extracts and gels (for EDTA: results not shown). A third complex (Figure 3a, complex III), approximately 2 MDa larger than complex II, was found exclusively in native gels supplemented with either calcium or magnesium ions.

We identified protein components of the native gel excised complex III by mass spectrometry (Table 2). Several of the identified proteins were validated, profiling members of all major functional categories of transmembrane translocon subcomplexes to complexes I–III by Western blots of the native

gel protein complexes, separated by SDS in a second dimension (Figures 2 and 3, summarized in Figure 4).

The Role of Divalent Cations in the Stability of Ribosome-Translocon Complex(es). Western blots of native gels run without divalent metal ions showed that Sec61α does not comigrate substantially with the ribosome complexes I and II in the absence of divalent metal ions. Instead, the majority of Sec61α was found to trail toward a lower native molecular weight, suggesting that they originate from a disintegrating larger complex. The lowest observed apparent molecular weight for Sec61 under these conditions was <100 kDa, consistent with the size of the heterotrimeric Sec61 complex (Figure 2c). Conversely, in the presence of calcium or magnesium, close to half of the Sec61p population shifted to migrate in a high molecular weight complex that coincide with complex III (Figure 3c).

This suggests that complex III represents an assembled ribosome/translocon complex, as defined by components that are necessary for functional polypeptide translocation. This notion was further substantiated by Western blotting against TRAPα. The transmembrane TRAP complex is closely associated with the Sec61 complex,²² and our data indicate that its association with complex III in response to divalent metal ions shares similarities with Sec61α. An organizational pattern similar to those of Sec61 and TRAP was also observed for BAP 31, a protein that is linked to calcium release from the ER during cellular apoptosis and which was demonstrated, recently, to interact physically with Sec61β, TRAM and the retrotranslocon component Derlin1³⁶ (Figures 2c, 3c, and 4).

Glucosidase I and OST Subunits Are Components of the Minor Ribosome Anchor of the ER. Several suggestions to the nature of the ER membrane ribosome anchor have been forwarded, over the years. Sec61p, itself, has been demonstrated to also function as a ribosome receptor *in vitro*,¹⁵,¹⁶,³⁷ while work by others has suggested that the Ribophorin/OST complex

Organization of the Sec61 Translocon

**Figure 2.** WiSE Native 2D gel and Western blots of CPM ribosome:RAMP complexes. (a) Composite image of native TDOC gel, projected on top of the subsequent second-dimension silver stained gel image of canine pancreatic microsomes, run under conditions as in Figure 1. The positions of ribosome associated membrane complexes I and II are marked. Indicated native molecular weights are based on the same internal markers as used in Figure 1 (Grp94 and p97). Internal denatured molecular weight markers used are UGGT (170 kDa) and PDI (60 kDa). Gallery of Western blots prepared from similar 2D gels consist of (b) Ribosomal protein L4, (c) Sec61α, TRAPα and Bap31, (d) STT3 and Glc I, (e) p180/‘ribosome receptor’ and CLIMP63, and (f) ERdj3. (For b–f, only relevant SDS $M_{w}$ region of the Western blot images is shown.)

is the ribosome receptor^{38,39} and others have suggested it to be p180.^{40,41}

We reasoned that we should be able to determine an *in vitro* “minimal” ribosome anchor among membrane spanning proteins associated with the lowest $M_{w}$ complex with ribosomes (Complex I). As observed in Figures 2d–f and 3d–f, we identify several membrane proteins and a single lumenal ER protein that comigrate with complexes I, II and III. The transmembrane

**Figure 3.** Ca/Mg WiSE Native 2D gel and Western blots of CPM ribosome:RAMP complexes. (a) Composite image of native TDOC gel, projected on top of the subsequent second-dimension silver stained gel image of canine pancreatic microsomes similar to that in Figure 2, but with gel and running buffer supplemented with 0.5 mM magnesium acetate. Positions of ribosome associated membrane complexes I, II and III are marked. Indicated native and denatured molecular weights are based on the same internal markers as used in Figure 2. Gallery of Western blots prepared from similar 2D gels consist of (b) Ribosomal protein L4, (c) Sec61α, TRAPα and Bap31, (d) STT3 and Glc I, (e) p180/ ‘ribosome receptor’ and CLIMP63, (f) ERdj3, and (g) p24δ. (For b–g, only relevant SDS $M_{w}$ region of the Western blot images is shown. See Table 2.)

proteins include CLIMP63, member(s) of the OST complex (STT3 and Ribophorin I), as well as glucosidase I (Glc I), an enzyme that functions immediately downstream of the OST complex at early stages of nascent chain translocation.^{42} But whereas there is a quantitative excess of the OST complex, with a substantial amount not associated with microsome attached

research articles

I II III

![Diagram](#)

- p180 ribosome receptor
- Ribosome anchor (OSTs, GlcI, CLIMP63)
- Ribosome
- Core translocon (Sec61, TRAP, Bap31)

Figure 4. Overview of the composition and size of the three ER membrane-associated ribosome complexes as indicated by the Western blots from Figures 2 and 3: Complex III decomposes to complex II in the absence of divalent metal ions. No further decomposition of complex II to complex I is observed. Similar signal intensities from ribosomal proteins of both the large and small subunits, observed by mass spectrometry, suggest that the ribosome is assembled and intact. Approximate sizes of the complexes: Complex I migrates to nearly the same position in native TDOC gels as intact ribosomes. Proposed size, 4–5 MDa. Complex III migrates to nearly the same position in the native TDOC gels as the pyruvate dehydrogenase complex (derived from mitochondria that partially copurify in microsome preparations). Proposed size, 7–8 MDa.

ribosomes, the same is not the case for Glc I (and CLIMP63) which we find exclusively in complex with ribosomes.

The Microtubule Interactor Membrane Proteins, CLIMP63 and p180, Interact with Membrane-Associated Ribosomes. The p180/‘ribosome receptor’ is an abundant ER membrane protein that interacts directly with ribosomes *in vitro* and is functionally linked to protein translocation into the ER,^41^ although it is not widely considered to play a key role in the translocation process itself, since the protein has no known ortholog in yeast. The protein may influence its compartmental distribution, since it is a genetic homologue of the transmembrane kinesin motor interactor, *kinectin*, and it interacts, *in vitro*, with a kinesin motor protein. In native gels, we observed it migrating almost exclusively with ribosomal complexes II and III (Figures 2e and 3e) and we suggest that p180 association with ribosome complex structures is a major component that differentiates ER ribosome complexes I and II. Evaluation, based both on Western blots and visual inspection of silver-stained 2D gels, suggests that the metal ion-stabilized translocon (complex III) contains p180, and no lower $M_w$ complexes can be resolved between complexes II and III which contain the core translocon components, but no p180.

Mass spectrometry of complex III also identified CLIMP63 as a major protein component. CLIMP63 is a microtubule tethering membrane protein, and like p180, it has no known ortholog in yeast. It has previously been hypothesized to play a role in the RER subcompartmental organization of translocons^42^ and was subsequently demonstrated to affect the rate of lateral diffusion of translocons in CLIMP63 knockdown cells.^44^ Western blots (Figures 2e and 3e) confirmed that CLIMP63 comigrates with all resolved ribosome complexes in the native gels, similar to the OST complex and Glc I. This observation suggests that the biophysical effect of CLIMP63, observed by Kreibich and co-workers, originates from a direct association between the translocon and CLIMP63. Similar to

Dejgaard et al.

Glc I, no substantial population of CLIMP63 was observed outside the 3 RAMP complexes, suggesting it as a further component of a putative ‘minimal’ ribosome anchor of the ER (see also Figure 4).

Chaperones and Cofactors: Translocon Components Are Putative Receptors for Erdj3. Besides Erdj3, mass spectrometry of complex III (Table 2) did not detect substantial amounts of chaperones that may associate with the ribosome complex(es). 2D Western blots prepared with and without divalent metal ions during native electrophoresis were probed for 8 prominent ER chaperones. The blots demonstrated that, for the chaperones assayed (ERp72, ERp57, PDI, GRp94, BiP, GRp170, Calnexin, and Erdj3), only trace amounts could be observed comigrating with some or all of the complexes, or not with any at all, with the notable exception of Erdj3, one of several DnaJ-domain proteins in the ER that functions as regulator of the ER lumenal chaperone BiP (For Erdj3, see Figures 2f and 3f. For the remainder, results not shown).

All detectable Erdj3 comigrated with complex I and II, extending to complex III in the presence of calcium/magnesium ions. A small but significant amount of Erp72 was found associated with the RAMP complexes in response to chemical cross-linkers, suggesting a low affinity interaction may exist (results not shown). None of the other chaperones mentioned displayed any association with any of the RAMP complexes in response to cross-linking.

p24 Proteins Are Not Bona Fide Translocon Components. Earlier work^10^ demonstrates that the p24 family (24 kDa) proteins, which are in oligomeric complexes that shuttle between ER exit sites and the cis-Golgi network,^45,46^ cosediment with sedimented membrane-bound ribosomes, suggesting they might be RAMPS.

We found by mass spectrometry of complex III that the p24 protein family members, along with a few other cis-Golgi network proteins, are present and are readily detected (Table 2). Western blots with antibodies directed against one of the p24 family members (p24δ, according to the nomenclature suggested in ref 45) show that the protein family is in complexes with a native molecular weight ranging from 1 to 4 MDa, in line with previous observations.^45^ Minor quantities of p24δ were observed at even higher native molecular weights with the membrane-bound ribosomal complexes (see Figure 3g), but without discrete localization to any of these complexes.

Our data, presented here, are in contrast to earlier observations by Skach and co-workers,^10^ and suggest that whereas p24 proteins appear at native molecular size(s) that approach those of the membrane-bound ribosome complexes, they are not bona fide RAMP components.

### Discussion

Native electrophoresis, with a few simple modifications that improve resolution and widen the resolved molecular weight span and reduce protein aggregation, is a powerful tool to separate and analyze high molecular weight protein complexes. In combination with electrophoretic gel-shift, it may additionally be definitive and help elucidate structural organization. In the presented work, we controlled gel shift of the ribosome-associated translocon complex by manipulation of bivalent cations.

We have demonstrated the resolution of at least 3 different membrane-attached ribosome structures (complexes I–III) including a ribosome/translocon structure that expands considerably the hitherto confirmed structurally organized complex.

Organization of the Sec61 Translocon

plex.²² We conclude that complex III is the most complete assembled structure, under the conditions specified. Lack of divalent cations destabilize complex III which, shedding core translocon components, gives rise to complex II. We have not been able to destabilize ribosome complex II further to complex I, nor have we observed any experimental conditions that result in detectable amounts of ribosome-free p180/‘ribosome receptor’. Also, we have not observed any stable complex structures larger than complex I that do not contain the p180/‘ribosome receptor’. Together, these observations suggest the existence of two functionally distinct ER membrane attached populations of ribosomes, one that may cotranslationally translocate proteins across the ER membrane through an associated translocon, and one that may not. Stably attached ER membrane ribosomes involved in translation of nontranslocated protein products has been proposed and demonstrated (refs 47 and 48; see ref 49 for review). It is important to note, however, that analyses by mass spectrometry have also not found any components that are unique to neither complexes I or II (our unpublished observations) and thus may identify them as functionally distinct from what we perceive as, and suggest is, the fully assembled translocation competent complex III.

Unfortunately, our native electrophoresis system has not proven able to address questions regarding the translocation competence of the complexes, since immature translation products (from truncated mRNA of lactoglobulin, lacking a stop codon) were released from the ribosome complexes during sample extraction or later in the electrophoretic process (results not shown). Our observation that the BiP cochaperone Erdj3 localizes to the RAMP complexes is puzzling, in light of this. Shen and Hendershot⁵⁰ have demonstrated that Erdj3 precedes BiP interaction with nascent polypeptide chains in the ER. But if nascent polypeptides do not stay associated with the ribosome:translocon complexes, our data suggest that Erdj3 targets one or more RAMP components directly. In chemically cross-linked extracts, we observed an electrophoretic shift of a subpopulation of Erdj3 toward a region of the native gels occupied in part by several chaperones, including BiP (results not shown). This may suggests that Erdj3 does not actually reside at the translocon in a dynamic system, but rather may relocate to it, when not functionally active elsewhere, to perhaps await the next cycle of nascent chain decoration.

Three components identified by mass spectrometry (Kinetin, p180/ribosome receptor and CLIMP63) have ultrastructural organization of the translocon at the subcompartmental level in higher eukaryotes as a common theme. Two of them (CLIMP63 and the p180/ribosome receptor) have been demonstrated, here, to associate with all (CLIMP63) or a subset (p180) of the 3 ribosomal membrane complexes. Not surprisingly, such components will not necessarily be apparent in *in vitro* reconstitution assays, through which practically all hitherto known translocon associated cofactors have been identified, an assay type in which subcompartmental sorting/localizationcanplaynorole.Yet *in vivo*, the ER is subcompartmentalized and divided into structurally distinct entities in higher eukaryotes: the central rough ER, the smooth ER and a large number of distal discrete transitional ER exit sites. For CLIMP63, our observations complement resent observations by Kreibich and co-workers,⁴⁴ that expression knock-down of this protein leads to increased diffusion in the ER membrane of a GFP-tagged translocon component in intact cells (Note: The same does not apply for yeast, which also does not have gene orthologs for neither p180 nor CLIMP63.)

Of other novel components presented as translocon associated, Glc I has been demonstrated to operate cotranslationally on nascent chain substrates at a stage no later than N-linked glycosylation by the OST complex.⁴² Indeed, it is with the ribosome bound subpopulation of the OST complex that we observe Glc I to colocalize, hinting at an additional role for Glc I participating in a ribosome membrane anchor. The association of Bap31 at the organizational level of complex III remains enigmatic. Recent studies by Shore and co-workers³⁶ have shown that Bap31 interacts with the translocon core components Sec61β and TRAM in yeast two-hybrid studies. Our results confirm that Bap31 exhibits an electrophoretic behavior similar to translocon core components, in that it dissociates from ribosome complex III in a Ca²⁺/Mg²⁺ dependent manner, albeit at what appears visually to be at rates that are not identical to, for example, TRAPα.

Further validation by Western blots with specific antibodies will be needed before conclusions can be drawn about other putative translocon-associated protein components that we have identified by mass spectrometry. The components include one that has previously been hypothesized as translocon components of the OST subcomplex, based on its Blue Native electrophoretic mobility: KIAA0090/CG2943-PA¹⁰ (see Table 2). The same applies for other putative components of the translocon complex that were detected by mass spectrometry of complex III, but did not meet the cutoff threshold that was chosen for the data presented here (min 3 unique peptides with significant Mowse scores, see Materials and Methods). They include the GPI anchoring transamidases PIG-S, PIG-U and PIG-T,⁵¹,⁵² TRAM, the SRP receptor, Derlin1 and Keratinocyte associated protein 2, another putative OST component.¹⁰

While we have highlighted here, in terms of size and complexity, an extreme case, we propose that the demonstrated improvements to native size electrophoresis should be applicable to a wide variety of protein interaction studies as a both versatile and simple supplement to existing techniques. Current approaches to studying protein interactions include the yeast two-hybrid system and TAP/FLAG tagged affinity purification protocols. Both methods have been very informative in the development of interactomes and require genetic intervention. The modified native electrophoresis protocol described here extends on work by others, using TDOC as an electrophoretic charge shift agent for membrane proteins,⁸ and combines both high resolution and, with the modification including ‘Rhinohide’, allows separation of complexes of >10 MDa. It can be applied to native whole cell and tissue lysates, which has hitherto posed significant problems for native electrophoresis⁷ when resolving other than respiratory chain complexes of the mitochondria. It complements affinity tag methods applied to interactomic studies by offering native size estimates to proteins, whether they are monomeric, homooligomeric or hetero-oligomeric of origin, or, as in the study presented here, a combination of multiple complex states.

**Acknowledgment.** K.D. was financed through part of this work by a Richard H. Tomlinson postdoctoral fellowship.

**Supporting Information Available:** Figure legends to supplemental Figures 1–5. Supplemental Figure 1, illustration of the separating principles of native electrophoresis; Supplemental Figure 2, discontinuous isoelectrophoresis and continuous electrophoresis of total cell extract; Supplemental Figure 3, acrylamide size exclusion and competitive effects of nonionic

Journal of Proteome Research • Vol. 9, No. 4, 2010 1769

detergents on charge shift electrophoresis; Supplemental Figure 4, prediction of native size according to ‘monomer diagonal’; Supplemental Figure 5, extraction of representative membrane proteins by TDOC. This material is available free of charge via the Internet at http://pubs.acs.org.

## References

(1) Uetz, P.; Giot, L.; Cagney, G.; Mansfield, T. A.; Judson, R. S.; Knight, J. R.; Lockshon, D.; Narayan, V.; Srinivasan, M.; Pochart, P.; Qureshi-Emili, A.; Li, Y.; Godwin, B.; Conover, D.; Kalbfleisch, T.; Vijayadamodar, G.; Yang, M.; Johnston, M.; Fields, S.; Rothberg, J. M. A comprehensive analysis of protein-protein interactions in *Saccharomyces cerevisiae*. *Nature* **2000**, *403*, 623–627.

(2) Ito, T.; Tashiro, K.; Muta, S.; Ozawa, R.; Chiba, T.; Nishizawa, M.; Yamamoto, K.; Kuhara, S.; Sakaki, Y. Toward a protein-protein interaction map of the budding yeast: A comprehensive system to examine two-hybrid interactions in all possible combinations between the yeast proteins. *Proc. Natl. Acad. Sci. U.S.A.* **2000**, *97*, 1443–1447.

(3) Li, S.; Armstrong, C. M.; Bertin, N.; Ge, H.; Milstein, S.; Boxem, M.; Vidalain, P.-O.; Han, J. D. J.; Chesneau, A.; Hao, T.; Goldberg, D. S.; Li, N.; Martinez, M.; Rual, J.-F.; Lamesch, P.; Xu, L.; Tewari, M.; Wong, S. L.; Zhang, L. V.; Berriz, G. F.; Jacotot, L.; Vaglio, P.; Reboul, J.; Hirozane-Kishikawa, T.; Li, Q.; Gabel, H. W.; Elewa, A.; Baumgartner, B.; Rose, D. J.; Yu, H.; Bosak, S.; Sequerra, R.; Fraser, A.; Mango, S. E.; Saxton, W. M.; Strome, S.; van den Heuvel, S.; Piano, F.; Vandenhaute, J.; Sardet, C.; Gerstein, M.; Doucette-Stamm, L.; Gunsalus, K. C.; Wade Harper, J.; Cusick, M. E.; Roth, F. P.; Hill, D. E.; Vidal, M. A. Map of the Interactome network of the metazoan *C.elegans*. *Science* **2004**, *303*, 540–543.

(4) Gavin, A. C.; Bosche, M.; Krause, R.; Grandi, P.; Marzioch, M.; Bauer, A.; Schultz, J.; Rick, J. M.; Michon, A. M.; Cruciat, C. M.; Remor, M.; Hofert, C.; Schelder, M.; Brajenovic, M.; Ruffner, H.; Merino, A.; Klein, K.; Hudak, M.; Dickson, D.; Rudi, T.; Gnau, V.; Bauch, A.; Bastuck, S.; Huhse, B.; Leutwein, C.; Heurtier, M. A.; Copley, R. R.; Edelmann, A.; Querfurth, E.; Rybin, V.; Drewes, G.; Raida, M.; Bouwmeester, T.; Bork, P.; Seraphin, B.; Kuster, B.; Neubauer, G.; Superti-Furga, G. Functional organization of the yeast proteome by systematic analysis of protein complexes. *Nature* **2002**, *415*, 141–147.

(5) Ho, Y.; Gruhler, A.; Heilbut, A.; Bader, G. D.; Moore, L.; Adams, S. L.; Millar, A.; Taylor, P.; Bennett, K.; Boutilier, K.; Yang, L.; Wolting, C.; Donaldson, I.; Schandorf, S.; Shewnarane, J.; Vo, M.; Taggart, J.; Goudreault, M.; Muskat, B.; Alfarano, C.; Dewar, D.; Lin, Z.; Michalickova, K.; Willems, A. R.; Sassi, H.; Nielsen, P. A.; Rasmussen, K. J.; Andersen, J. R.; Johansen, L. E.; Hansen, L. H.; Jespersen, H.; Podtelejnikov, A.; Nielsen, E.; Crawford, J.; Poulsen, V.; Sørensen, B. D.; Matthiesen, J.; Hendrickson, R. C.; Gleeson, F.; Pawson, T.; Moran, M. F.; Durocher, D.; Mann, M.; Hogue, C. W.; Figeys, D.; Tyers, M. Systematic identification of protein complexes in *Saccharomyces cerevisiae* by mass spectrometry. *Nature* **2002**, *415*, 180–183.

(6) Krogan, N. J.; Cagney, G.; Yu, H.; Zhong, G.; Guo, X.; Ignatchenko, A.; Li, J.; Pu, S.; Datta, N.; Tikuisis, A. P.; Punna, T.; Peregrín-Alvarez, J. M.; Shales, M.; Zhang, X.; Davey, M.; Robinson, M. D.; Paccanaro, A.; Bray, J. E.; Sheung, A.; Beattie, B.; Richards, D. P.; Canadien, V.; Lalev, A.; Mena, F.; Wong, P.; Starostine, A.; Canete, M. M.; Vlasblom, J.; Wu, S.; Orsi, C.; Collins, S. R.; Chandran, S.; Haw, R.; Rilstone, J. J.; Gandi, K.; Thompson, N. J.; Musso, G.; St Onge, P.; Ghanny, S.; Lam, M. H.; Butland, G.; Altaf-Ul, A. M.; Kanaya, S.; Shilatifard, A.; O’Shea, E.; Weissman, J. S.; Ingles, C. J.; Hughes, T. R.; Parkinson, J.; Gerstein, M.; Wodak, S. J.; Emili, A.; Greenblatt, J. F. Global landscape of protein complexes in the yeast *Saccharomyces cerevisiae*. *Nature* **2006**, *440*, 637–643.

(7) Camacho-Carvajal, M. M.; Wollscheid, B.; Aebersold, R.; Steimle, V.; Schamel, W. W. A Two-dimensional blue native/SDS gel electrophoresis of multi-protein complexes from whole cellular lysates - a proteomics approach. *Mol. Cell. Proteomics* **2004**, *3*, 176–182.

(8) Schägger, H.; von Jagow, G. Blue native electrophoresis for isolation of membrane protein complexes in enzymatically active form. *Anal. Biochem.* **1991**, *199*, 223–231.

(9) Wang, L.; Dobberstein, B. Oligomeric complexes involved in translocation of proteins across the membrane of the endoplasmic reticulum. *FEBS Lett.* **1999**, *457*, 316–322.

(10) Shibatani, T.; David, L. L.; McCormack, A. L.; Frueh, K.; Skach, W. R. Proteomic analysis of mammalian oligosaccharyltransferase reveals multiple subcomplexes that contain Sec61, TRAP, and two potential new subunits. *Biochemistry* **2005**, *44*, 5982–5992.

(11) Johnson, A. E.; van Waes, M. A. THE TRANSLOCON: A dynamic gateway at the ER membrane. *Annu. Rev. Cell Dev. Biol.* **1999**, *15*, 799–842.

(12) Nicchitta, C. V.; Blobel, G. Assembly of translocation-competent proteoliposomes from detergent-solubilized rough microsomes. *Cell* **1990**, *60*, 259–269.

(13) Nicchitta, C. V.; Migliaccio, G.; Blobel, G. Biochemical fractionation and assembly of the membrane components that mediate nascent chain targeting and translocation. *Cell* **1991**, *65*, 587–598.

(14) Görlich, D.; Hartmann, E.; Prehn, S.; Rapoport, T. A. A protein of the endoplasmic reticulum involved early in polypeptide translocation. *Nature* **1992**, *357*, 47–52.

(15) Görlich, D.; Prehn, S.; Hartmann, E.; Kalies, K. U.; Rapoport, T. A. A mammalian homolog of SEC61p and SECYp is associated with ribosomes and nascent polypeptides during translocation. *Cell* **1992**, *71*, 489–503.

(16) Görlich, D.; Rapoport, T. A. Protein translocation into proteoliposomes reconstituted from purified components of the endoplasmic reticulum membrane. *Cell* **1993**, *75*, 615–630.

(17) Krieg, U. C.; Johnson, A. E.; Walter, P. Protein translocation across the endoplasmic reticulum membrane: identification by photo-cross-linking of a 39 kD integral membrane glycoprotein as part of a putative translocation tunnel. *J. Cell Biol.* **1989**, *109*, 2033–2043.

(18) Wiedmann, M.; Görlich, D.; Hartmann, E.; Kurzchalia, T. V.; Rapoport, T. A. Photo-crosslinking demonstrates proximity of a 34 kDa membrane protein to different portions of preprolactin during translocation through the endoplasmic reticulum. *FEBS Lett.* **1989**, *257*, 263–268.

(19) High, S.; Görlich, D.; Wiedmann, M.; Rapoport, T. A.; Dobberstein, B. The identification of proteins in the proximity of signal-anchor sequences during their targeting to and insertion into the membrane of the ER. *J. Cell Biol.*, **1991**, *113*, 35–44.

(20) Thrift, R. N.; Andrews, D. W.; Walter, P.; Johnson, A. E. A nascent membrane protein is located adjacent to ER membrane proteins throughout its integration and translation. *J. Cell Biol.* **1991**, *112*, 809–821.

(21) Snapp, E. L.; Reinhart, G. A.; Bogert, B. A.; Lippincott-Schwartz, J.; Hegde, R. S. The organization of engaged and quiescent translocons in the endoplasmic reticulum of mammalian cells. *J. Cell Biol.* **2004**, *164*, 997–1007.

(22) Menetret, J. F.; Hegde, R. S.; Heinrich, S. U.; Chandramouli, P.; Ludtke, S. J.; Rapoport, T. A.; Akey, C. W. Architecture of the ribosome-channel complex derived from native membranes. *J. Mol. Biol.* **2005**, *348*, 445–457.

(23) Chevet, E.; Wong, H. N.; Gerber, D.; Cochet, C.; Fazel, A.; Cameron, P. H.; Gushue, J. N.; Thomas, D. Y.; Bergeron, J. J. Phosphorylation by CK2 and MAPK enhances calnexin association with ribosomes. *EMBO J.* **1999**, *18*, 3655–3666.

(24) Ogawa-Goto, K.; Irie, S.; Omori, A.; Miura, Y.; Katano, H.; Hasegawa, H.; Kurata, T.; Sata, T.; Arora, Y. An endoplasmic reticulum protein, p180, is highly expressed in human cytomegalovirus-permissive cells and interacts with the tegument protein encoded by UL48. *J. Virol.* **2002**, *76*, 2350–2362.

(25) Hardt, B.; Kalz-Füller, B.; Aparicio, R.; Volker, C.; Bause, E. (Arg)3 within the N-terminal domain of glucosidase I contains ER targeting information but is not required absolutely for ER localization. *Glycobiology* **2003**, *13*, 159–168.

(26) Laemmli, U. K. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* **1970**, *227*, 680–685.

(27) Schägger, H.; Cramer, W. A.; von Jagow, G. Analysis of molecular masses and oligomeric states of protein complexes by blue native electrophoresis and isolation of membrane protein complexes by two dimensional native electrophoresis. *Anal. Biochem.* **1994**, *217*, 220–230.

(28) O’Connell, K. L.; Stults, J. T. Identification of mouse liver proteins on two-dimensional electrophoresis gels by matrix-assisted laser desorption/ionization mass spectrometry of in situ enzymatic digests. *Electrophoresis* **1997**, *18*, 349–359.

(29) Towbin, H.; Staehelin, T.; Gordon, J. Electrophoretic transfer of proteins from polyacrylamide gels to nitrocellulose sheets: procedure and some applications. *Proc. Natl. Acad. Sci. U.S.A.* **1979**, *76*, 4350–4354.

(30) Shevchenko, A.; Wilm, M.; Vorm, O.; Mann, M. Mass spectrometric sequencing of proteins from silver-stained polyacrylamide gels. *Anal. Chem.* **1996**, *68*, 850–858.

(31) Henderson, N. S.; Nijtmans, L. G. J.; Lindsay, J. G.; Lamantea, E.; Zeviani, M.; Holt, I. J. Separation of intact pyruvate dehydrogenase complex using native agarose gel electrophoresis. *Electrophoresis* **2000**, *21*, 2925–2931.

Organization of the Sec61 Translocon

(32) Gutsche, I.; Essen, L. O.; Baumeister, W. Group II chaperonins: new TRiC(k)s and turns of a protein folding machine. *J. Mol. Biol.* **1999**, *293*, 295–312.

(33) Coux, O.; Tanaka, K.; Goldberg, A. L. Structure and functions of the 20S and 26S proteasomes. *Annu. Rev. Biochem.* **1996**, *65*, 801–847.

(34) Peters, J. M.; Harris, J. R.; Lustig, A.; Müller, S.; Engel, A.; Volker, S.; Franke, W. W. Ubiquitous soluble Mg(2+)-ATPase complex. A structural study. *J. Mol. Biol.* **1992**, *223*, 557–571.

(35) Schulenberg, B.; Arnold, B.; Patton, W. F. An improved mechanically durable electrophoresis gel matrix that is fully compatible with fluorescence-based protein detection technologies. *Proteomics* **2003**, *3*, 1196–1205.

(36) Wang, B.; Heath-Engel, H.; Zang, D.; Ngyen, N.; Thomas, D. Y.; Hanrahan, J. W.; Shore, G. C. BAP31 interacts with Sec61 translocons and promotes retrotranslocation of CFTRΔF508 via the Derlin-1 complex. *Cell* **2008**, *133*, 1080–1092.

(37) Kalies, K. U.; Görlich, D.; Rapoport, T. A. Binding of ribosomes to the rough endoplasmic reticulum mediated by the Sec61p-complex. *J. Cell Biol.* **1994**, *126*, 925–934.

(38) Marcantonio, E. E.; Amar-Costesec, A.; Kreibich, G. Segregation of the polypeptide translocation apparatus to regions of the endoplasmic reticulum containing ribophorins and ribosomes. II. Rat liver microsomal subfractions contain equimolar amounts of ribophorins and ribosomes. *J. Cell Biol.* **1984**, *99*, 2254–2259.

(39) Yu, Y. H.; Sabatini, D. D.; Kreibich, G. Antiribophorin antibodies inhibit the targeting to the ER membrane of ribosomes containing nascent secretory polypeptides. *J. Cell Biol.* **1990**, *111*, 1335–1342.

(40) Savitz, A. J.; Meyer, D. I. Identification of a ribosome receptor in the rough endoplasmic reticulum. *Nature* **1990**, *346*, 540–544.

(41) Savitz, A. J.; Meyer, D. I. 180-kD ribosome receptor is essential for both ribosome binding and protein translocation. *J. Cell Biol.* **1993**, *120*, 853–863.

(42) Deprez, P.; Gautschi, M.; Helenius, A. More Than One Glycan Is Needed for ER Glucosidase II to Allow Entry of Glycoproteins into the Calnexin/Calreticulin Cycle. *Mol. Cell* **2005**, *19*, 183–195.

(43) Nikonov, A. V.; Kreibich, G. Organization of translocon complexes in ER membranes. *Biochem. Soc. Trans.* **2003**, *31*, 1253–1256.

(44) Nikonov, A. V.; Hauri, H.-P.; Lauring, B.; Kreibich, G. Climp-63-mediated binding of microtubules to the ER affect the lateral mobility of translocon complexes. *J. Cell Sci.* **2007**, *120*, 2248–2258.

(45) Dominguez, M.; Dejgaard, K.; Füllekrug, J.; Dahan, S.; Fazel, A.; Paccaud, J. P.; Thomas, D. Y.; Bergeron, J. J.; Nilsson, T. gp25L/emp24/p24 protein family members of the cis-Golgi network bind both COP I and II coatomer. *J. Cell Biol.* **1998**, *140*, 751–765.

(46) Füllekrug, J.; Suganuma, T.; Tang, B. L.; Hong, W.; Storrie, B.; Nilsson, T. Localization and recycling of gp27 (hp24gamma3): complex formation with other p24 family members. *Mol. Biol. Cell* **1999**, *10*, 1939–1955.

(47) Potter, M. D.; Nicchitta, C. V. Endoplasmic reticulum-bound ribosomes reside in stable association with the translocon following termination of protein synthesis. *J. Biol. Chem.* **2002**, *277*, 23314–23320.

(48) Stephens, S. B.; Dodd, R. D.; Brewer, J. W.; Lager, P. J.; Keene, J. D.; Nicchitta, C. V. Stable ribosome binding to the endoplasmic reticulum enables compartment-specific regulation of mRNA translation. *Mol. Biol. Cell* **2005**, *16*, 5819–5831.

(49) Nicchitta, C. V.; Lerner, R. S.; Stephens, S. B.; Dodd, R. D.; Pyhtila, B. Pathways for compartmentalizing protein synthesis in eukaryotic cells: the template-partitioning model. *Biochem Cell Biol.* **2005**, *83*, 687–695.

(50) Shen, Y.; Hendershot, L. M. ERdj3, a stress-inducible endoplasmic reticulum DnaJ homologue, serves as a cofactor for BiP’s interactions with unfolded substrates. *Mol. Biol. Cell* **2005**, *16*, 40–50.

(51) Ohishi, K.; Inoue, N.; Kinoshita, T. PIG-S and PIG-T, essential for GPI anchor attachment to proteins, form a complex with GAAl and GPI8. *EMBO J.* **2001**, *20*, 4088–4098.

(52) Hong, Y.; Ohishi, K.; Kang, J. Y.; Tanaka, S.; Inoue, N.; Nishimura, J.; Maeda, Y.; Kinoshita, T. Human PIG-U and yeast Cdc91p are the fifth subunit of GPI transamidase that attaches GPI-anchors to proteins. *Mol. Biol. Cell* **2003**, *14*, 1780–1789.
