
REVIEW

www.rsc.org/molecularbiosystems | Molecular BioSystems

**When ribosomes go bad: diseases of ribosome biogenesis**

Emily F. Freed,$^{a}$ Franziska Bleichert,$^{a}$ Laura M. Dutca$^{b}$ and Susan J. Baserga*$^{abc}$

Received 21st September 2009, Accepted 22nd October 2009  
First published as an Advance Article on the web 11th January 2010  
DOI: 10.1039/b919670f

Ribosomes are vital for cell growth and survival. Until recently, it was believed that mutations in ribosomes or ribosome biogenesis factors would be lethal, due to the essential nature of these complexes. However, in the last few decades, a number of diseases of ribosome biogenesis have been discovered. It remains a challenge in the field to elucidate the molecular mechanisms underlying them.

---

### Introduction

Ribosomes are the cellular factories responsible for making proteins. Ribosome biogenesis, the process of making ribosomes, is a complex and energy-intensive process that involves several hundred factors. These factors include multiple enzymes, transcription regulators, chaperones, nuclear export proteins, and more, all of which ensure that the ribosomal RNA (rRNA) is properly folded and assembled with the ribosomal proteins (r-proteins). Much of the progress in elucidating the factors involved in making ribosomes has been carried out in the single-celled model organism, *Saccharomyces cerevisiae*. Because of its genetic tractability, the yeast is ideal for high-throughput studies, including yeast two-hybrid, microarrays, and affinity purification followed by mass spectrometry; these techniques have enabled interaction maps of ribosome biogenesis factors to be created,${}^{1-4}$ furthering our understanding of this complex process. Furthermore, due to the high conservation of both the RNA and protein moieties of ribosomes and of the ribosome biogenesis machinery from yeast to humans, in general what we have learned in yeast also applies to humans (for recent reviews see ref. 5, 6). Ribosome biogenesis is so important for cell growth that a growing yeast cell synthesizes approximately 2000 ribosomes every minute, requiring 60% of total cellular transcription. ${}^{7}$ In mammalian cells, this number is even higher; for example, a HeLa cell makes 7500 ribosomal subunits per minute. ${}^{8}$

In eukaryotic cells, ribosome biogenesis begins with the transcription of the ribosomal DNA (rDNA) into a 47S (35S in yeast) polycistronic precursor by RNA polymerase I (Pol I) in the nucleolus (Fig. 1). Hundreds of r-proteins, non-ribosomal proteins involved in processing, and small nucleolar RNAs (snoRNAs) associate co-transcriptionally with the pre-rRNA forming the 90S pre-ribosome. ${}^{9,10}$ In the same transcription unit but in the opposite direction, the 5S rRNA is transcribed by RNA polymerase III (Pol III) and may associate with the 90S pre-ribosome. ${}^{6}$ The 47S pre-rRNA undergoes multiple cleavage ${}^{11-13}$ and modification ${}^{14-16}$ events, and after association with r-proteins, gives rise to the pre-40S small subunit (SSU) and the pre-60S large subunit (LSU). These subunits are then exported to the cytoplasm where final maturation takes place. The mature ribosome consists of the SSU, which contains the 18S rRNA associated with about 30 r-proteins, and the LSU, which contains the 28S (25S in yeast), 5.8S, and 5S rRNAs and about 45 r-proteins, depending on the species.

Nearly all ribosome biogenesis factors, as well as the r-proteins themselves, are essential for cell survival. It is

---

**Emily F. Freed received her BS from Stanford University. She is now a PhD candidate at Yale studying how ribosome dysfunction leads to disease.**

**Franziska Bleichert, a PhD student at Yale University, is studying small ribonucleoproteins involved in ribosome biogenesis. Her work focuses on understanding the architecture and structure of these ribonucleoproteins and how this structure dictates the functions of these complexes.**

This journal is © The Royal Society of Chemistry 2010

Mol. BioSyst., 2010, 6, 481–493 | 481

ribosome biogenesis
proteins

small nucleolar
ribonucleoproteins

ribosomal proteins

Fig. 1 Model of pre-rRNA processing in human cells. The primary pre-rRNA is transcribed and associates co-transcriptionally with processing factors. Processing can occur by either of two pathways, beginning with cleavage at site 1 or site 2, respectively. Final maturation takes place in the cytoplasm.

therefore somewhat surprising that some defects in ribosome biogenesis lead to detectable human disease, rather than embryonic lethality. Nonetheless, over the last few decades, a growing variety of diseases linked to ribosomes and ribosome biogenesis have been reported (Tables 1–3). Furthermore, all of these diseases show some degree of tissue-proclivity, which is unexpected given that ribosomes are present in all tissues in the body.

### Genetic diseases of ribosome biogenesis proteins

The small subunit (SSU) processome is a large ribonucleo-
protein (RNP) that is required for the maturation of the 18S rRNA of the SSU of the ribosome. It was first discovered in yeast through a combination of directed and high throughput

techniques, and later was also described in humans. This complex contains the U3 snoRNA and over 40 proteins, many of which first assemble into subcomplexes. In addition to the SSU processome proteins, many other proteins are required for the maturation of the large and small ribosomal subunits. The majority of these processing proteins are not present in the mature ribosome. Almost all of these proteins are essential in yeast and several have also been shown to be essential in mammals, suggesting conservation from yeast to humans. Although complete loss of these proteins is expected to be lethal, mutations in ribosome biogenesis factors can act either as the causative agents of disease or, in rare cases, as modifying agents, by increasing the severity of a disease caused by a mutation in a non-pre-ribosome associated protein (Table 1).

Table 1 Genetic diseases of ribosome biogenesis proteins

| Gene       | human     | yeast    | Impaired molecular function                                                                 | Disease                          | Clinical manifestations                                                                                              | Inheritance and incidence (if known)                                                                                     | Online Mendelian Inheritance in Man (OMIM) # | References |
|------------|-----------|----------|-----------------------------------------------------------------------------------------|----------------------------------|----------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|------------------------------------------|------------|
| TCOF1      | —         | —        | rDNA transcription and methylation of 18S rRNA                                              | Treacher Collins syndrome (TCS)   | craniofacial abnormalities                                                                                           | autosomal dominant; incidence: 1 in 10,000 to 50,000 live births                                                        | #154500                             | 21,24,154  |
| UTP14c     | UTP14     | —        | maturation of the 18S rRNA of the 40S ribosomal subunit                                      | male infertility                  | severe oligospermia or azoospermia                                                                                    | unknown                                                                                                       | N/A                                  | 9,26       |
| CIRH1A     | UTP4      | —        | maturation of the 18S rRNA of the 40S ribosomal subunit                                      | Native American Indian childhood cirrhosis (NAIC)           | neonatal jaundice progressing to biliary cirrhosis; lethal by adolescence without liver transplant                         | autosomal recessive; < 100 cases reported                                                          | #604901                             | 9,28,31    |
| EMG1       | EMG1      | —        | maturation of the 40S ribosomal subunit                                                     | Bowen-Conradi syndrome (BCS)      | growth retardation, psychomotor delay, skeletal abnormalities; lethal by early childhood                               | autosomal recessive; incidence: 1 in 355 live births in the Hutterite population                                        | #211180                             | 33         |
| RBM28      | NOP4      | —        | maturation of the 60S ribosomal subunit                                                    | alopecia, neurological defects, and endocrinopathy syndrome (ANE syndrome) | growth retardation, loss of motor ability, mental retardation, skeletal and skin abnormalities, hair loss, central adrenal insufficiency | autosomal recessive                                                                                 | #612079                             | 37         |
| SBDS       | SDO1      | —        | maturation and export of the 60S ribosomal subunit                                          | Shwachman-Diamond syndrome (SDS) | growth retardation, exocrine pancreas insufficiency, hematologic defects, skeletal abnormalities, cancer predisposition | autosomal recessive; incidence: 1 in 76,000 live births                                                                | #260400                             | 39,44,155  |
| WDR36      | UTP21     | —        | maturation of the 18S rRNA of the 40S ribosomal subunit                                      | candidate gene for primary open angle glaucoma (POAG)       | degeneration of optical nerve leading to blindness                                                                    | multifactorial                                                                                                 | #137760                             | 9,48       |
| HCA66      | UTP6      | —        | maturation of the 18S rRNA of the 40S ribosomal subunit                                      | modifier of neurofibromatosis type 1 (NF1)                 | mental retardation, craniofacial and connective tissue abnormalities, congenital heart defects, tumor predilection          | autosomal dominant                                                                                 | #162200                             | 9,53       |

Mutations in small ribosomal subunit biogenesis proteins that cause disease

Treacle—Treacher Collins syndrome. Treacher Collins syndrome (TCS) is an autosomal dominant craniofacial disorder which varies widely in disease severity, but usually includes abnormalities of the ears, eyes, and facial bones, particularly the lower jaw and cheek. Five mutations in the gene encoding Treacle (TCOF1) were first reported to cause

Laura M. Dutca studied chemistry at West University of Timisoara, Romania. In 2006 she graduated with a PhD in Chemistry from Iowa State University, where she developed an interest in the study of large macromolecular assemblies. As a postdoctoral researcher at Yale University she is developing biochemical methods to investigate structural changes in pre-ribosomal RNA and formation of macro-molecular complexes during ribosome biogenesis in yeast.

Susan J. Baserga, MD, PhD, received her degrees from Yale University and is a Professor of Molecular Biophysics & Biochemistry, Genetics and Therapeutic Radiology at Yale.

Table 2 Genetic diseases of small nucleolar ribonucleoproteins

| Gene       | Mutated protein/RNA                                                                 | Impaired molecular function                                                                                     | Disease                              | Clinical manifestations                                                                                          | Inheritance and incidence (if known)                                      | OMIM# References |
|------------|-----------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|-----------------|
| human      | yeast                                                                            |                                                                                                               |                                       |                                                                                                                 |                                                                              |                 |
| RMRP       | NME1                                                                             | noncoding RNA component of RNase MRP                                                                           | maturation of 5.8S rRNAof60S ribosomal subunit                                                                 | Anauxetic dysplasia (AD)                                                   | extreme short stature, hypodontia, mild mental retardation, severely reduced number of chondrocytes | autosomal recessive; < 200, 000 cases in the US                             | #607095         | 63, 156, 157 |
| RMRP       | NME1                                                                             | as above                                                                                                     | maturation of 5.8S rRNAof60S ribosomal subunit; degradation of cell-cycle regulated RNAs; mitochondrial DNA replication | Cartilage-hair dysplasia (also called metaphyseal dysplasia, hair chondrodysplasia McKusick type; CHH) | disproportional short stature, metaphyseal dysplasia, hair hypoplasia, anemia, immunodeficiency, cancer predisposition | autosomal recessive; < 200, 000 cases in the US                             | #250250         | 59, 62      |
| RMRP       | NME1                                                                             | as above                                                                                                     | as above                                                                                         | Metaphyseal dysplasia without hypotrichosis (MDWH)                          | short stature, metaphyseal skeletal changes                                                                 | autosomal recessive; < 200, 000 cases in the US                             | #250460         | 59, 157, 158 |
| DKC1       | CBF5                                                                             | pseudouridine synthase of box H/ACA snoRNP RNPs (snoRNPs, scaRNPs, and telomerase)                                    | telomerase deficiency, disease aggravated by box H/ACA snoRNP pseudouridylation defects | Dyskeratosis (also called Zinsser-Engman-Cole syndrome)                    | mucocutaneous abnormalities, bone marrow failure, cancer predisposition                                                              | X-linked recessive; prevalence: 1 in 1,000,000                               | #305000         | 70, 72, 78  |
| DKC1       | CBF5                                                                             | as above                                                                                                     | as above                                                                                         | Hoyeraal-Hreidarsson syndrome (some cases are severe variants of Dyskeratosis congenita) | severe growth retardation, bone marrow failure, immunodeficiency, microcephaly, cerebellar hypoplasia | X-linked recessive; prevalence: 1 in 1,000,000                               | #300240         | 70, 71      |
| NOP10      | NOP10                                                                            | protein component of box H/ACA RNPs (snoRNPs, scaRNPs, and telomerase)                                             | telomerase deficiency, disease aggravated by box H/ACA snoRNP pseudouridylation defects | Dyskeratosis (also called Zinsser-Engman-Cole syndrome)                   | mucocutaneous abnormalities, bone marrow failure, cancer predisposition                                                              | autosomal recessive; #224230 prevalence: 1 in 1,000,000                       | #224230         | 70, 159     |
| NHP2       | NHP2                                                                             | protein component of box H/ACA RNPs (snoRNPs, scaRNPs, and telomerase)                                             | telomerase deficiency, disease aggravated by box H/ACA snoRNP pseudouridylation defects | Dyskeratosis (also called Zinsser-Engman-Cole syndrome)                   | mucocutaneous abnormalities, bone marrow failure, cancer predisposition                                                              | autosomal recessive; #224230 prevalence: 1 in 1,000,000                       | #224230         | 70, 160     |
| HBII-85    | —                                                                                | deleted box C/D snoRNA cluster                                                                                 | specific function unknown                                                                         | Prader-Willi syndrome (PWS)                                                | neonatal hypotonia, short stature, hyperphagia, obesity, mental retardation, hypogonadism                                         | autosomal, paternal deletion or maternal disomy of Chr. incidence: 1 in 10,000 to 15,000 in live births | #176270         | 90          |

TCS over 10 years ago²¹ and since that time, more than 200 mutations have been reported in the TCOF1 database (www.genoma.ib.usp.br/TCOF1_database). The mutations include primarily nonsense mutations, insertions, deletions, and alternative splicing, all of which lead to truncation of the protein. There is no correlation between genotype and disease severity, which may be explained by the observation that genetic background greatly alters the phenotype in mouse models.²² Treacle is a putative nucleolar phosphoprotein that plays a role in rDNA transcription²³ and methylation of the 18SrRNA.²⁴

All TCS patients are heterozygous for a TCOF1 mutation and the disease appears to be due to haploinsufficiency rather than dominant negative effects.²³ Extensive work in the mouse has shown that Tcof1 is required for neural crest cell formation and proliferation. In Tcof1⁺/⁻ mice, an increase in caspase 3-mediated apoptosis is observed specifically in the neuroepithelium, resulting in a smaller starting population of cells then proliferates more slowly than cells expressing wild-type Tcof1, further reducing the number of neural crest

Table 3 Genetic diseases of ribosomal proteins

| Gene     | yeast    | Impaired function                          | Disease                     | Clinical manifestations                                                                 | Inheritance and incidence (if known)                                      | OMIM#   | References |
|----------|----------|-------------------------------------------|-----------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------|---------|------------|
| RPS 7    | RPS7A    | 5'-end processing of 18S rRNA             | Diamond Blackfan anemia (DBA)| anemia, bone marrow failure, craniofacial abnormalities, cardiac defects, limb and urogenital abnormalities, cancer predisposition | sporadic, autosomal dominant (40-45%); incidence: 1 in 100,000 to 200,000 live births | #105650 | 104,107-112 |
|          | RPS7B    |                                           |                             |                                                                                       |                                                                         |         |            |
| RPS 17   | RPS17A   | 5'-end processing of the 18S rRNA         | as above                    | as above                                                                              | as above                                                                | as above| as above   |
|          | RPS17B   |                                           |                             |                                                                                       |                                                                         |         |            |
| RPS 19   | RPS19A   | 5'-end processing of the 18S rRNA         | as above                    | as above                                                                              | as above                                                                | as above| as above   |
|          | RPS19B   |                                           |                             |                                                                                       |                                                                         |         |            |
| RPS 24   | RPS24A   | 5'-end processing of 18S rRNA             | as above                    | as above                                                                              | as above                                                                | as above| as above   |
|          | RPS24B   |                                           |                             |                                                                                       |                                                                         |         |            |
| RPL 5    | RPL 5    | processing of pre-rRNA of the LSU        | as above                    | as above                                                                              | as above                                                                | as above| as above   |
| RPL 11   | RPL11A   | processing of pre-rRNA of the LSU         | as above                    | as above                                                                              | as above                                                                | as above| as above   |
|          | RPL11B   |                                           |                             |                                                                                       |                                                                         |         |            |
| RPL35A   | RPL33A   | processing of pre-rRNA of the LSU         | as above                    | as above                                                                              | as above                                                                | as above| as above   |
|          | RPL33A   |                                           |                             |                                                                                       |                                                                         |         |            |
| RPS 14   | RPS14A   | pre-18S rRNA processing                   | 5q syndrome                 | severe macrocytic anemia, myelodysplastic syndrome, cancer predisposition | unknown                                                               | #153550 | 131        |
|          | RPS14B   |                                           |                             |                                                                                       |                                                                         |         |            |

cells and leading to the hypoplasia observed in TCS. Both the apoptosis and the slow growth rate are likely due to a lack of mature ribosomes; indeed fewer ribosomes are observed specifically in the neuroepithelium of $T c o f l^{+/}$ mice. The cause of the neuroepithelium-specific defect remains to be elucidated since TC OF 1 is expressed in a wide variety of fetal and adult tissues and ribosomes are required in all cells in all tissue types.

UTP 14-male infertility. In yeast, Utp14 is an SSU processome protein and is essential for 18S rRNA maturation; however, its specific function is unknown. In humans, there are three genes encoding UTP14: UTP14a, UTP14b, and UTP14c. UTP14a is the X-linked ancestral copy of the gene and is ubiquitously expressed across tissue types. UTP14b is a degraded retroposon located within ACSL3 and is not expressed in human tissue. UTP14c is an active retroposon that inserted into the 3' UTR of GT8 (a putative glycosyl transferase-containing gene) and is only expressed in the testis and ovary.

Mutations in Utp14 were first identified in a naturally occurring infertile male mouse, the jsd mouse. A subsequent screen for UTP14 mutations in infertile men found a heterozygous mutation that resulted in truncation of the UTP14c protein by 28 amino acids (Y738X) in three patients. Two of the patients presented with nonobstructive azoospermia and one presented with severe oligospermia. A testicular biopsy was available for one of the patients with azoospermia and showed that germ cells were arrested at the early stage of pachytene spermatocytes. Hormone levels were normal. These findings suggest that the failure in spermatogenesis is due to haploinsufficiency of UTP14c protein is being produced to support the large demand for ribosomes during sperm development. Alternatively, the production of a truncated UTP14c protein could have a dominant negative effect if its incorporation into the SSU processome prevents proper formation of the complex.

Cirhin-North American Indian childhood cirrhosis (NAIC). North American Indian childhood cirrhosis is an autosomal recessive disorder found in the Ojibway-Cree population in northwestern Quebec. The disease presents as neonatal jaundice and then progresses to biliary cirrhosis. The only treatment is liver transplantation, which is required by early adolescence. Chagnon et al. reported that a missense mutation in the C-terminus of Cirhin causes NAIC, with all patients carrying two copies of the R565W mutation. Cirhin/UTP4 is a member of the t-Utp subcomplex of the SSU processome, a subcomplex that is required for optimal transcription of the rDNA in both yeast and human cells. Although Cirhin is known to be required for ribosome biogenesis, little is known about the molecular mechanism(s) that leads to this disease.

EMG1-Bowen-Conradi syndrome (BCS). Bowen-Conradi syndrome (BCS) is a lethal, autosomal recessive syndrome described primarily in the Hutterite population. Patients with BCS exhibit a variety of symptoms including pre-and post-natal growth retardation, psychomotor delay, microcephaly and multiple joint abnormalities. Patients rarely survive beyond the first year. Recently, a missense mutation in EMG1 was reported to be the cause of BCS. EMG1 is a putative methyltransferase that is required for biogenesis of the 40S subunit of the ribosome.

Modeling of the EMG1 protein suggests that the D86G substitution found in BCS patients may interfere with the formation of a salt bridge, leading to aggregation and subsequent degradation of EMG1. This hypothesis is further supported by two results: (1) the D86G mutation increases dimerization of EMG1 in the yeast two-hybrid system; and (2)
a dramatic reduction in EMG1 protein levels is observed in BCS fibroblasts, while \(EMG1\) mRNA expression remains unchanged between patient and control samples. \({ }^{33}\) Thus, partial loss of EMG1 protein expression and a subsequent reduction in ribosome biogenesis provide a plausible molecular mechanism for Bowen-Conradi syndrome.

Mutations in large ribosomal subunit biogenesis proteins that cause disease

- RBM28—alopecia, neurological defects, and endocrinopathy syndrome (ANE syndrome). Alopecia, neurological defects, and endocrinopathy syndrome (ANE syndrome) is an autosomal recessive disease that is clinically heterogeneous. ANE syndrome patients display multiple signs including a varied amount of hair loss, mental retardation, progressive loss of motor ability beginning in the second decade of life, hypogonadism, central adrenal insufficiency, short stature, microcephaly, and several other skeletal and skin abnormalities. \({ }^{37}\) A missense mutation in RBM28, a protein required for biogenesis of the 60S subunit of the ribosome, \({ }^{38}\) has been reported to cause ANE syndrome. \({ }^{37}\) Similar to EMG1, the L351P mutation in RBM28 leads to a nearly complete loss of protein expression in patient cells, although the mRNA level remains constant. This loss of RBM28 protein results in a decrease in the number of ribosomes in patient fibroblasts, \({ }^{37}\) likely due to a defect in 60S ribosomal subunit biogenesis.

- SBDS—Shwachman-Bodian-Diamond syndrome (SDS). Similar to ANE syndrome, Shwachman-Bodian-Diamond syndrome (SDS) is an autosomal recessive disorder that is pleiotropic. \({ }^{39}\) Symptoms include pancreatic insufficiency likely caused by replacement of pancreatic tissue with adipose tissue, hematologic defects including neutropenia and occasionally also anemia and/or thrombocytopenia, skeletal abnormalities, short stature, enlarged liver with elevated liver enzymes, endocrine abnormalities, defects in T- and B-cell function, and predisposition to leukemia. \({ }^{40}\) Up to 90% of cases of SDS are reported to be caused by mutations in SBDS, a protein that has an unknown specific function, but is involved in maturation and export of the ribosomal 60S subunit. \({ }^{41,42}\) SBDS may also play a role in gene expression at both the transcriptional and translational levels. \({ }^{43,44}\)

The vast majority of mutations found in SDS patients are due to conversions between SBDS and a pseudogene copy of SBDS that share 97% sequence identity. The two most common mutations result in a truncated SBDS protein, due to the introduction of an in-frame stop codon (K62X), or a donor splice-site mutation with a subsequent frameshift (84Cfs3), respectively. While the 84Cfs3 mutation is found in both heterozygous and homozygous states, the K62X mutation is always heterozygous suggesting that it is a null mutation and is lethal when homozygous. \({ }^{39}\) Studies on cells from SDS patients show that SBDS mutations lead to a decrease in newly synthesized rRNA with no corresponding change in polysome profile, \({ }^{41}\) shortening of telomeres, \({ }^{45}\) and an increase in Fas-mediated apoptosis. \({ }^{46}\) It is unclear whether these are primary or secondary effects of reduced SBDS levels. Nonetheless, defects in ribosome biogenesis should be considered as an etiologic agent for SDS.

Mutations in ribosome biogenesis proteins that modify disease

- WDR36—modifier/cause of primary open angle glaucoma (POAG). Primary open angle glaucoma (POAG), a complex disorder that arises as the result of multiple genetic and environmental factors, is the leading cause of blindness worldwide. POAG usually presents in individuals over age 35 and includes degeneration of the optic nerve and often increased intraocular pressure. \({ }^{47}\) Unlike the diseases discussed above, no single gene has been identified in the etiology of POAG; however, there are three main candidates. \({ }^{47}\) One of these candidates is \(WDR36 / UTP21\), which encodes a protein of unknown specific function, but is a member of the UtpB subcomplex of the SSU processome. \({ }^{3}\) Genotyping of 130 families with POAG revealed that 9 families had \(WDR36\) variants, all resulting in single amino acid substitutions in the protein, that segregate with the disease. These variants were divided into “disease causing” and “disease susceptibility” mutations based on their frequency of occurrence in POAG versus control populations. \({ }^{48}\) A subsequent study found that POAG patients with \(WDR36\) variants had a more severe form of the disease than patients without \(WDR36\) variants, suggesting that \(WDR36\) is a modifier gene of POAG. \({ }^{49}\) Most variants are heterozygous, suggesting that haploinsufficiency of \(WDR36\) could either cause or modify the manifestation of POAG. However, since numerous other studies find the same percentage of \(WDR36\) variants in both POAG and control populations, \({ }^{50,51}\) the role of \(WDR36\) in glaucoma remains unclear.

- HCA66—modifier of neurofibromatosis type 1 (NF1). Neurofibromatosis type 1 (NF1) is an autosomal dominant tumor predilection syndrome caused by deletion of the \(NF1\) gene. In approximately 5% of cases, a heterozygous microdeletion of genes surrounding \(NF1\) also occurs. In such cases, the disease is more severe and includes early onset and increased number of neurofibromas, mental retardation, congenital heart defects, dysmorphic features of the head and face, and connective tissue dysplasia. \({ }^{52}\) There are four candidate genes in the microdeletion region including \(HCA66 / UTP6,{ }^{53}\) which encodes a protein that is also a member of the UtpB subcomplex of the SSU processome. \({ }^{3}\) While the specific function of UTP6 in ribosome biogenesis is unknown, UTP6 has been reported to play a role in promoting apoptosis via an interaction with APAF-1. \({ }^{54}\) The additional clinical features seen in the microdeletion syndrome may be due to haploinsufficiency of UTP6, with the increase in tumors resulting from a decrease in UTP6/APAF-1 mediated apoptosis and all other signs resulting from a decrease in ribosome biogenesis.

Genetic diseases of small nuclear ribonucleoproteins

Small nuclear ribonucleoproteins (snoRNPs) are complexes composed of both RNA and proteins that localize to the nucleolus. Three major types that are involved in ribosome biogenesis can be distinguished by their different RNA and protein components: RNase MRP, box H/ACA snoRNPs, and box C/D snoRNPs. \({ }^{55}\) These snoRNPs perform enzymatic functions, including endonucleolytic cleavage and chemical modifications of the pre-rRNA, and also function as chaperonesto facilitate folding of the pre-rRNA. Mutations in genes encoding either the non-coding RNA components or the protein components of these RNPs can result in disease. To date, diseases caused by mutations in all three classes of snoRNPs have been identified (Table 2).

### RNase MRP snoRNA—skeletal dysplasias

RNase MRP (mitochondrial RNA processing) is an RNP composed of the non-coding MRP RNA and at least 10 different proteins. Studies in yeast and mammalian cells show that RNase MRP is essential and has multiple functions within cells: (1) it is involved in ribosome biogenesis, being an endonuclease that cleaves the pre-rRNA and separates the precursors of the SSU from the LSU pre-rRNA and is specifically required for the maturation of the 5.8S rRNA; \({ }^{13}\) (2) it is involved in cell cycle regulation by cleaving the cyclin B2 mRNA; \({ }^{56}\) and (3) it is involved in mitochondrial DNA replication. \({ }^{57,58}\)

A variety of mutations in the gene for the non-coding RNA component of RNase MRP have been linked to genetically inherited skeletal dysplasias. Depending on the severity of the skeletal dysplasia and the accompanying extraskeletal manifestations, they are clinically classified into: metaphyseal dysplasia without hypotrichosis (MDWH) with mild dysplasia, cartilage-hair dysplasia (CHH) with moderate dysplasia, and anauxetic dysplasia (AD) with severe dysplasia. CHH is also associated with a variety of extraskeletal abnormalities, including hypotrichosis, hypoplastic anemia, immunodeficiency, and an increased predisposition to cancer. \({ }^{59}\)

Mutations associated with these diseases all affect the MRP gene and cluster in the promoter region, where they can result in a decreased expression of the encoded RNA, or in evolutionarily highly conserved regions of the MRP RNA. \({ }^{60-62}\) Recent phenotype-genotype correlation studies indicate that mutations that impair the rRNA processing function of RNase MRP more severely affect the skeletal system and are less frequently associated with extraskeletal manifestations (as in AD). Conversely, mutations that affect both rRNA processing and mRNA processing are observed in patients with less severe skeletal abnormalities but with extraskeletal symptoms (as in CHH). \({ }^{63,64}\) Thus, different mutations in RNase MRP appear to separate the different functions of RNase MRP genetically, resulting in a varied spectrum of clinical manifestations. While so far disease-causing mutations have been found only in the RNA component of RNase MRP, it remains to be seen whether future studies and genetic analysis of more patients with these clinical signs will also reveal mutations in genes encoding the protein components of RNase MRP that impair function.

The molecular mechanisms by which these pathogenic mutations affect the different functions of RNase MRP remain poorly understood. Several scenarios, some of which are supported by the literature, can be imagined: (1) mutations in structurally important regions of the MRP RNA can result in RNA structural changes that may interfere with RNP assembly; \({ }^{65}\) (2) reduced transcription of the MRP RNA due to promoter mutations \({ }^{60,61}\) and instability of the MRP RNA transcript \({ }^{66}\) may reduce steady state levels of both the RNA

and the RNase MRP snoRNP below a threshold required for normal cellular function; and (3) altered MRP RNA and RNP may interfere with substrate recognition or catalytic activity itself. A combined effort of genetic (including mouse models), biochemical, and structural studies will be necessary to better understand the mutant RNase MRP complex and the underlying molecular mechanisms resulting in these diseases.

#### Box H/ACA snoRNPs—Dyskeratosis congenita (DC)

Dyskeratosis congenita (DC) is a genetic disease characterized by genetic as well as symptomatic heterogeneity. Clinically, patients can present with mucocutaneous abnormalities, a predisposition to a variety of cancers, and bone marrow failure, the latter being the leading cause of mortality. \({ }^{67-70}\) In severe cases, DC can also be associated with immunodeficiency, growth retardation and neurological symptoms (Hoyeraal-Hreidarsson syndrome). \({ }^{71}\) Genetic inheritance occurs as either X-linked recessive or autosomal dominant or recessive. Hence, DC can be caused by mutations in a variety of different genes: the X-linked form of DC is caused by mutations in the dyskerin gene, encoding an essential component of box H/ACA RNPs (including snoRNPs, scaRNPs [small Cajal body-specific RNPs], and telomerase); the autosomal recessive form can be caused either by mutations in \(N H P 2, N O P 10\) (both encoded proteins are also essential components of box H/ACA RNPs), or TERT (gene encoding the reverse transcriptase of the telomerase RNP); and the autosomal dominant form is associated with mutations in the telomerase RNA or TERT. \({ }^{68,70}\)

The original discovery of mutations in the dyskerin gene suggested that impaired ribosome biogenesis is the etiological molecular defect, since dyskerin is an essential protein component of box H/ACA snoRNPs, which catalyze pseudo-uridylation of rRNAs and are required for pre-rRNA processing. \({ }^{72}\) This conclusion is supported by results from studies of mouse models of the X-linked recessive form of DC, which target the dyskerin gene and reproduce some of the clinical phenotypes of the disease including the ribosome biogenesis defect, although to varying degrees. \({ }^{73-76}\) Furthermore, it has been shown that ribosomes in DC patient cells are functionally distinct from normal ribosomes, since IRES-dependent translation of some mRNAs is reduced. \({ }^{77}\) However, the subsequent discovery of mutations in telomerase RNA or the gene for telomerase reverse transcriptase (TERT) and the fact that telomerase is a box H/ACA RNP containing all protein components that are found in box H/ACA snoRNPs suggests instead that DC is primarily caused by impaired function of telomerase. \({ }^{70,78}\) This hypothesis is confirmed by the observation of shortened telomeres, reduced telomerase levels and activity, but normal rRNA processing in DC patient cells. \({ }^{79,80}\)

So what is the underlying molecular etiology of DC? Given the genetic and clinical heterogeneity of DC, it is most likely that, at least in some cases, the disease is caused by a combined effect of impaired telomerase, box H/ACA snoRNP and box H/ACA scaRNP functions, especially in the case of the X-linked form. Interestingly, the X-linked form of DC caused by mutations in the dyskerin gene typically presents with moresevere symptoms than the autosomal dominant form of DC that is caused by mutations in telomerase RNA or the TERT gene directly. The more severe disease caused by dyskerin mutations could be explained by a combined effect of dysfunction in both telomere maintenance and ribosome biogenesis. Future studies will be necessary to uncouple both functions of H/ACA RNP components better genetically. Until then, a contribution of ribosome biogenesis defects to the molecular pathogenesis of DC, especially the X-linked form, should not be neglected.

### Box C/D snoRNPs—Prader-Willi syndrome (PWS)

Prader-Willi syndrome (PWS) is a complex disease characterized by neonatal hypotonia, short stature, hyperphagia and obesity, hypogonadism and characteristic facial features. Genetic mapping linked PWS to chromosomal region 15q11-q13. Gene expression in this region is regulated by imprinting, and PWS results from loss of expression of genes normally only expressed from the paternal chromosome. Most cases of PWS result from parental deletions or maternal disomy of chromosome 15q11-q13, and in some cases imprinting defects are responsible for silencing of the genes. Interestingly, while the PWS locus contains few protein encoding genes, it encodes numerous non-coding RNAs of the box C/D type. Fine mapping studies have shown that a loss of expression of the HBII-85 box C/D snoRNA cluster is responsible for most of the symptoms observed in PWS patients. Consistent with this, mouse models that include deletion of these snoRNAs of the PWS locus recapitulate some of the phenotypes observed in PWS patients. However, a contribution of other box C/D snoRNA genes to the complex clinical manifestations observed in PWS patients cannot be completely ruled out at this point.
- Box C/D snoRNAs in humans associate with the conserved proteins, 15.5K, NOP56, NOP58, and fibrillarin to form box C/D type RNPs. Box C/D snoRNPs typically catalyze 2′-O-ribose methylation of ribosomal RNA, participate in pre-rRNA folding, and some are also essential for pre-rRNA processing. The specificity of these events is dictated by base pairing of the guide sequences of box C/D snoRNAs with the substrate RNAs. Besides their function in ribosome biogenesis, box C/D type RNPs have also been shown to perform 2′-O-ribose methylations of snRNAs (box C/D scaRNPs) and at least one mRNA; the latter has been shown to regulate mRNA splicing and mRNA editing. Very recently, box C/D type RNAs have also been suggested to be precursors for microRNA-like small RNAs. Thus, box C/D type RNAs and RNPs function in a variety of different aspects of RNA processing.
- Box C/D snoRNAs encoded by the PWS locus most likely assemble into box C/D RNPs since they can be immunoprecipitated with antibodies against fibrillarin. However, the target RNAs of the snoRNAs encoded in the PWS locus are largely unknown, with the exception of the HBII-52 snoRNAs and a few yet unvalidated snoRNAs predicted to target ribosomal RNAs. Unfortunately, the targets of the HBII-85 snoRNA cluster remain completely obscure, and hence it is unknown how loss of expression of these snoRNAs

results in PWS. Identification of their target RNAs as well as defining the function of these RNAs will be crucial for understanding the molecular mechanism underlying this disease.

### Genetic diseases of ribosomal proteins

In mammals there are about 80 ribosomal proteins (r-proteins) that are structural components of the ribosome, with some of them also participating in the control of the translation process. Many r-proteins have roles in the processing of pre-rRNA during ribosome biogenesis and are important for the assembly of the ribosomal subunits. Some have additionally been shown to have extraribosomal functions, but in general, their known functions are in ribosome biogenesis or in its surveillance. Surprisingly, it was discovered that haploinsufficiency of some r-proteins leads to diseases of the bone marrow such as Diamond Blackfan anemia and the 5q syndrome (Table 3).

#### Diamond Blackfan anemia (DBA)

Diamond-Blackfan anemia (DBA) is an inherited bone marrow failure syndrome of children characterized by proapoptotic hematopoiesis, bone marrow failure, birth defects and a predisposition to cancer. About 40% of the patients also present with craniofacial, cardiac, limb, and urogenital abnormalities. Recently, a number of reviews have discussed different aspects of the disease, its diagnosis, treatment, and molecular pathogenesis. Unexpectedly, all the genes currently shown to be involved in DBA are r-protein genes and account for about 50% of DBA cases. Haploinsufficiency of these r-proteins is thus likely to be the basis for DBA. Different types of mutations have been found in genes corresponding to both small and large subunit r-proteins: *RPS19* (25%), *RPL5* (6.6%), *RPL11* (4.8%), *RPL35A* (3%), *RPS24* (2%), *RPS17* (1%), and *RPS7* (<1%). Possible mutations were also reported for *RPS15*, *RPS27A*, and *RPL36*. Mutations in the most commonly affected gene, *RPS19*, include nonsense and missense mutations, small insertions and deletions, splice site defects and large deletions, and rearrangements.

Correlations between some of the symptoms observed in DBA and the mutated r-protein are emerging. Physical malformations are more often associated with *RPL5* and *RPL11* mutations (~70%) compared to *RPS19* (~46%). Furthermore, mutations in *RPL5* seem to give rise to a more severe manifestation and mutations in only this gene are associated with cleft lip and/or cleft palate abnormalities in DBA.

*RPS7*, *RPS17*, *RPS19*, and *RPS24* are involved in the maturation of the 18S rRNA and are required for the assembly of the 40S subunit. Interestingly, Rps7 was also shown to be a component of the SSU processome in yeast. Similarly, the large ribosomal subunit proteins *RPL5*, *RPL11* and *RPL35A* participate in the processing of the rRNAs that compose the large subunit and are essential for the assembly of the mature large subunit. Defects in pre-rRNA processing were observed in primary fibroblasts and bone marrow progenitor cells from patients with DBA for all of ther-proteins mentioned, ${}^{111,112,119-121}$ confirming that defects in ribosome biogenesis contribute to DBA.

The r-proteins involved in DBA also have other functions that could contribute to the development of the disease, and many of these functions could contribute to the predisposition to cancer observed in DBA patients. For example, RPS 19 interacts with the PIM-1 oncoprotein (a serine-threonine kinase which phosphorylates proteins like CDC25C and c-MYC) in vivo and PIM-1 phosphorylates RPS 19 in vitro. ${}^{122}$ RPL11 can modulate the activity of both the tumor suppressor p53 ${}^{123}$ and c-MYC, an oncoprotein and transcription factor. ${}^{124}$ Similarly, RPS7, RPL5 and RPL23 can also modify p53 activity. ${}^{123,125}$

All the mutations found so far in DBA patients have been in genes encoding r-proteins, including components of both small and large ribosomal subunits, making it very likely that DBA is a ribosome biogenesis disease. All of the r-proteins identified are required for both pre-rRNA processing and ribosome assembly. Interestingly, not all of them affect the same step in ribosome biogenesis, and they can affect the formation of the small or large subunit. To complicate the issue even further, often times when the level of expression of one r-protein gene is reduced other r-proteins are expressed at lower levels also. ${}^{116}$ To explain the pathogenesis of DBA, the emerging consensus is that ribosome biogenesis defects lead to a reduced number of ribosomes available to the cell and consequently to reduced translation, growth in cell size and cell division. During the proliferation and differentiation of red blood cell precursors there is a high demand for ribosomes, since red blood cells proliferate very rapidly. It has been proposed that because these processes require very high translation rates, they present an elevated sensitivity to the effects of diminished ribosome production. ${}^{106}$ Another hypothesis involves the connection between ribosome biogenesis and the tumor suppressor p53: the “ribosomal stress hypothesis”. ${}^{104,105}$ In particular, aberrant ribosome biogenesis as a result of a decrease in an r-protein would activate p53-mediated cell cycle arrest and apoptosis. Studies in zebrafish and mouse models support this hypothesis. In zebrafish, p53 activation is involved in the mechanism by which *rps19* haploinsufficiency gives rise to a reduction in erythrocytes and physical abnormalities, ${}^{126}$ and loss of *Rpl11* affects embryonic development. ${}^{127}$ A new mouse model with a mutation in *Rps19* presented with some of the DBA signs, including decreased red blood cells and reticulocyte levels and increased apoptosis. ${}^{128}$ Somewhat puzzling is that no defects in the bone marrow were observed in these mice. More details on the links between ribosome biogenesis and the p53 pathway are presented in the section “Cancer and ribosome biogenesis”.

### 5q− syndrome

The 5q− syndrome is a myelodysplastic syndrome, preponderant in adult females, characterized by a defect in erythroid differentiation and associated with progression to acute myeloid leukemia (AML). As the name implies, the 5q− syndrome is characterized by a deletion in the long arm of chromosome 5. ${}^{129}$ The critical region for the 5q− syndrome contains 40 genes, ${}^{130}$ including *RPS14*. Recently, a study that used an RNA interference-based functional screen showed that *RPS14* is the haploinsufficient tumor suppressor gene associated with the syndrome. ${}^{131}$ Rps14 is a component of both the small subunit of the ribosome and of the SSU processome in yeast, ${}^{117}$ and has been shown to be required for processing of the 18S rRNA and ribosome assembly. ${}^{131,132}$ To further confirm the importance of ribosome biogenesis for 5q− syndrome, about 55 ribosomal-and translation-related factors were found to be differentially expressed in cells of patients with 5q−, including some r-proteins. ${}^{133}$

Notably, there are some striking similarities between the 5q− syndrome and DBA (Table 4). Even though the age of onset is quite different, as the 5q− syndrome appears in adulthood and DBA in early childhood, they both present with macrocytic anemia, and both result in a predilection for AML. For both, the underlying molecular defect is a defective or absent r-protein. It is thus plausible that a similar type of molecular mechanism acts in both diseases.

## Cancer and ribosome biogenesis

The links between ribosome biogenesis and cancer are complex and not yet well understood. Ribosome biogenesis and nuclear structure are altered significantly to meet the needs of cancer cells, ${}^{134,135}$ which are usually characterized by high protein synthesis and rRNA transcription rates. The nucleolar number, size and morphology are convenient diagnostic markers for cancer. ${}^{134,135}$ Dysregulation of ribosome biogenesis has been associated with alterations in cell cycle, cell proliferation and cell growth and often contributes to increased susceptibility to cancer. ${}^{134,136}$ It is not surprising, therefore, that ribosome biogenesis and ribosome function have become targets for anticancer therapies. ${}^{137}$

R-proteins can regulate the c-MYC oncogene and the p53 tumor suppressor, which in turn both function as ribosome synthesis regulators. ${}^{138,139}$ One of the mechanisms by which the activity of the transcription factor p53 can be regulated by r-proteins is through binding to MDM2, the main E3-ligase of p53 which ubiquitinates p53 and promotes its degradation. ${}^{140}$ Three large subunit r-proteins, RPL5, RPL11 and RPL23, ${}^{123,141,142}$ and a small subunit r-protein, RPS7, ${}^{123,125,143}$ bind MDM2 and activate p53 upon ribosomal stress. Recently, it was also proposed that RPL5 and RPL11 act together in this function. ${}^{144}$ A number of other r-proteins were also proposed to regulate the activity of p53, and although their mechanism of action is not known, it was proposed that they do not necessarily act through MDM2. ${}^{145}$ RPL26 binds to the 5′ and/or 3′UTR of human p53 mRNA and selectively promotes p53 translation, ${}^{146}$ and haploinsufficiency of RPS6 activates the p53 tumor suppressor. ${}^{147,148}$ Recently, it was proposed that reduction in the expression of small subunit r-proteins, in particular RPS6 and RPS7, and subsequent disruption of 40S ribosome biogenesis will activate p53 by an RPL11-translation mechanism. ${}^{147}$ Specifically, in the aforementioned conditions, the translation of mRNAs with a polypyrimidine tract at the transcriptional start (5′-TOP mRNAs) will be upregulated, regardless of the decrease in global translation. Since all
Table 4 Comparison of Diamond Blackfan anemia and 5q syndrome characteristics

| Characteristic                | Diamond Blackfan anemia                                      | 5q syndrome                                   |
|------------------------------|-----------------------------------------------------------|---------------------------------------------|
| Age and gender predominance   | Younger than a year, both sexes equally affected            | Older adults, female predominance             |
| Macrocytic anemia             | Present                                                    | Present                                      |
| Cancer predisposition         | Increased (AML, solid tumors)                              | Can progress to AML                          |
| Congenital abnormalities      | Frequently present                                         | Not present                                  |
| Treatment                    | Transfusions, corticosteroids, spontaneous remission frequent | Transfusions, lenalidomide                   |

r-proteins have a 5′-TOP, RPL11 would be overexpressed, inhibit MDM2, and activate p53.¹⁴⁷

The c-MYC oncogene is a transcription factor that activates genes involved in processes including cell division, apoptosis and angiogenesis. Furthermore, c-MYC has an important role in ribosome biogenesis, since it regulates transcription by RNA polymerases.¹³⁸,¹⁴⁹ Haploinsufficiency of some r-proteins has been shown to suppress c-MYC oncogenic activity,¹²⁴,¹⁵⁰ which is unexpected given that haploinsufficiency of r-proteins often leads to an increase in tumors. Interestingly, RPL11 is one of the r-proteins that inhibits c-MYC activity and does so through a negative feedback mechanism.¹²⁴,¹⁵¹

Several r-proteins are overexpressed in a variety of tumors,¹³⁴,¹³⁵,¹³⁷ but it is not yet clear if these alterations are a cause or a result of tumorigenesis. In addition, it has also been shown that reduced ribosome biogenesis or loss of function of r-protein genes can be correlated with malignant transformation in zebrafish,¹⁵² while overexpression of r-protein genes can prevent tumor growth.

Taken together, the most perplexing feature of all of the ribosome biogenesis diseases is the tissue proclivity of the manifestations. Ribosomes are present in all cells in all tissues in the body and are required for cell survival, cellular growth and division; therefore, all tissues should be equally affected by defects in ribosome biogenesis. However, this is not the case. For some diseases, such as male infertility, the tissue proclivity is easy to explain since UTP14c expression is under the control of a testis-specific promoter.²⁶ However, in most cases, the cause of tissue proclivity remains somewhat of a mystery. It has been proposed that highly proliferating tissues, such as the bone marrow, would have a high need for ribosomes, and would be more affected than the other organ systems.¹⁰⁶ Indeed, this is observed in DBA and the 5q syndrome. Nonetheless, this hypothesis cannot entirely account for the tissue proclivity since in a developing embryo, all tissues are dividing rapidly and therefore they should all be affected if adequate numbers of ribosomes cannot be made. Furthermore, the bone marrow is not the only self-renewing organ system in the adult human.

So how does tissue specificity arise? Although little published evidence exists, we can speculate on several possibilities: the disease-associated proteins may have tissue-specific interaction partners, resulting in impaired function in only particular tissue types. Or, rather than tissue-specific interacting proteins, their gene expression may be regulated by tissue-specific microRNAs. Furthermore, the encoded mRNAs may have tissue-specific alternatively spliced forms, an appealing possibility given that almost 95% of human genes are alternatively spliced¹⁵³ and that tissue-specific transcripts have been observed for some of the mRNAs that encode ribosome biogenesis factors (e.g. see ref. 21, 39). Another possibility is that these proteins have extra-ribosomal functions, as is observed with some of the r-proteins implicated in Diamond Blackfan anemia,¹²²,¹²³,¹⁵¹ and it is these other functions that confer tissue proclivity. Elucidation of the molecular mechanisms of ribosome biogenesis diseases, and in particular the reasons for tissue proclivity, remain some of the main challenges for the future.

### Conclusions and perspective

Ribosomes are essential for life. If functional, mature ribosomes are not synthesized, a decrease in protein translation occurs, resulting in reduced cell growth and cell death. Therefore, it is expected that mutations in ribosomal proteins or ribosome biogenesis factors would result in embryonic lethality; surprisingly this is not always the case. Over the last few decades, multiple rare, genetic diseases have been attributed to defects in ribosome function or ribosome biogenesis. With the current advances in whole genome sequencing, more diseases of ribosome biogenesis are likely to be discovered.

While diseases of ribosome biogenesis span a wide range of symptoms, impaired molecular processes, and modes of inheritance, some common patterns between diseases can be observed (see Tables 1–3). First, nearly all of the described diseases are the result of haploinsufficiency (e.g. Treacher Collins syndrome and 5q syndrome) or partial loss of protein expression (e.g. Bowen-Conradi syndrome), supporting the idea that homozygous null mutations in ribosome biogenesis factors are embryonic lethal. Second, although no one symptom is present in all diseases, many diseases share clinical features such as short stature, mental retardation, joint abnormalities, anemia and predisposition to cancer. Third, mutations in many ribosomal and ribosome biogenesis proteins seem to activate the p53 pathway, providing an underlying mechanism for cell death and perhaps partially explaining the connection between ribosome biogenesis and cancer, as discussed earlier.

### Acknowledgements

This work was supported by NIH grant R01GM 52581 (S.J.B.), Yale Liver Center Grant NIDDK P30-34989 (S.J.B.), the Anna Fuller Fund Fellowship in Molecular Oncology (L.M.D.) and NIH5T32HD 07149-32. We would like to thank E. Champion and A. Szekely for their help and contributions.

References

1. A. C. Gavin, *et al.*, Nature, 2002, 415, 141.
2. T. Ito, K. Tashiro, S. Muta, R. Ozawa, T. Chiba, M. Nishizawa, K. Yamamoto, S. Kuhara and Y. Sakaki, Proc. Natl. Acad. Sci. U. S. A., 2000, 97, 1143.
3. N. J. Krogan, *et al.*, Mol. Cell, 2004, 13, 225.
4. W. T. Peng, *et al.*, Cell, 2003, 113, 919.
5. S. Granneman and S. J. Baserga, Exp. Cell Res., 2004, 296, 43.
6. A. K. Henras, J. Soudet, M. Gérus, S. Lebaron, M. Caizergues-Ferrer, A. Mougin and Y. Henry, Cell. Mol. Life Sci., 2008, 65, 2334.
7. J. R. Warner, J. Vilardell and J. H. Sohn, Cold Spring Harbor Symp. Quant. Biol., 2001, 66, 567.
8. J. D. Lewis and D. Tollervey, Science, 2000, 288, 1385.
9. F. Dragon, J. E. Gallagher, P. A. Compagnone-Post, B. M. Mitchell, K. A. Porwancher, K. A. Wehner, S. Wormsley, R. E. Settlage, J. Shabanowitz, Y. Osheim, A. L. Beyer, D. F. Hunt and S. J. Baserga, Nature, 2002, 417, 967.
10. P. Grandi, V. Rybin, J. Bassler, E. Petfalski, D. Strauss, M. Marzioch, T. Schäfer, B. Kuster, H. Tschochner, D. Tollervey, A. C. Gavin and E. Hurt, Mol. Cell, 2002, 10, 105.
11. F. Bleichert, S. Granneman, Y. N. Osheim, A. L. Beyer and S. J. Baserga, Proc. Natl. Acad. Sci. U. S. A., 2006, 103, 9464.
12. S. A. Elela, H. Igel and M. Ares, Jr., Cell, 1996, 85, 115.
13. M. E. Schmitt and D. A. Clayton, Mol. Cell Biol., 1993, 13, 7935.
14. P. Ganot, M. L. Bortolin and T. Kiss, Cell, 1997, 89, 799.
15. Z. Kiss-László, Y. Henry, J. P. Bachellerie, M. Caizergues-Ferrer and T. Kiss, Cell, 1996, 85, 1077.
16. M. Nicoloso, L. H. Qu, B. Michot and J. P. Bachellerie, J. Mol. Biol., 1996, 260, 178.
17. A. J. Turner, A. A. Knox, J. L. Prieto, B. McStay and N. J. Watkins, Mol. Cell. Biol., 2009, 29, 3007.
18. J. Pérez-Fernández, A. Román, J. De Las Rivas, X. R. Bustelo and M. Dosil, Mol. Cell. Biol., 2007, 27, 5414.
19. A. Richter, G. A. Mitchell and A. Rasquin, Med. Sci. (Paris), 2007, 23, 1002.
20. S. Zhang, M. Shi, C. C. Hui and J. M. Rommens, Mol. Cell. Biol., 2006, 26, 6656.
21. The Treacher Collins Syndrome Collaborative Group, Nat. Genet., 1996, 12, 130.
22. J. Dixon and M. J. Dixon, Dev. Dyn., 2004, 229, 907.
23. B. C. Valdez, D. Henning, R. B. So, J. Dixon and M. J. Dixon, Proc. Natl. Acad. Sci. U. S. A., 2004, 101, 10709.
24. B. Gonzales, D. Henning, R. B. So, J. Dixon, M. J. Dixon and B. C. Valdez, Hum. Mol. Genet., 2005, 14, 2035.
25. J. Dixon, N. C. Jones, L. L. Sandell, S. M. Jayasinghe, J. Crane, J. P. Rey, M. J. Dixon and P. A. Trainor, Proc. Natl. Acad. Sci. U. S. A., 2006, 103, 13403.
26. J. Rohozinski, D. J. Lamb and C. E. Bishop, Biol. Reprod., 2006, 74, 644.
27. J. Rohozinski and C. E. Bishop, Proc. Natl. Acad. Sci. U. S. A., 2004, 101, 11695.
28. P. Chagnon, J. Michaud, G. Mitchell, J. Mercier, J. F. Marion, E. Drouin, A. Rasquin-Weber, T. J. Hudson and A. Richter, Am. J. Hum. Genet., 2002, 71, 1443.
29. C. Bétard, A. Rasquin-Weber, C. Brewer, E. Drouin, S. Clark, A. Verner, C. Darmond-Zwaig, J. Fortin, J. Mercier, P. Chagnon, T. M. Fujiwara, K. Morgan, A. Richter, T. J. Hudson and G. A. Mitchell, Am. J. Hum. Genet., 2000, 67, 222.
30. J. E. Gallagher, D. A. Dunbar, S. Granneman, B. M. Mitchell, Y. Osheim, A. L. Beyer and S. J. Baserga, Genes Dev., 2004, 18, 2506.
31. J. L. Prieto and B. McStay, Genes Dev., 2007, 21, 2041.
32. P. Bowen and G. J. Conradi, Birth Defects Orig. Artic. Ser., 1976, 12, 101.
33. J. Armistead, *et al.*, Am. J. Hum. Genet., 2009, 84, 728.
34. D. Eschrich, M. Buchhaupt, P. Kötter and K. D. Entian, Curr. Genet., 2002, 40, 326.
35. N. Leulliot, M. T. Bohnsack, M. Graille, D. Tollervey and H. Van Tilburg, Nucleic Acids Res., 2008, 36, 629.
36. P. C. Liu and D. J. Thiele, Mol. Biol. Cell, 2001, 12, 3644.
37. J. Nousbeck, *et al.*, Am. J. Hum. Genet., 2008, 82, 1114.
38. C. Sun and J. L. Woolford, Jr., EMBO J., 1994, 13, 3127.
39. G. R. Boocock, J. A. Morrison, M. Popovic, N. Richards, L. Ellis, P. R. Durie and J. M. Rommens, Nat. Genet., 2003, 33, 97.
40. A. Shimamura, Semin. Hematol., 2006, 43, 178.
41. K. A. Ganapathi, K. M. Austin, C. S. Lee, A. Dias, M. M. Malsch, R. Reed and A. Shimamura, Blood, 2007, 110, 1458.
42. T. F. Menne, B. Goyenechea, N. Sánchez-Puig, C. C. Wong, L. M. Tonkin, P. J. Ancliff, R. L. Brost, M. Costanzo, C. Boone and A. J. Warren, Nat. Genet., 2007, 39, 486.
43. C. Hesling, C. C. Oliveira, B. A. Castilho and N. I. Zanchin, Exp. Cell Res., 2007, 313, 4180.
44. P. Rujkijyanont, S. L. Adams, J. Beyene and Y. Dror, Br. J. Haematol., 2009, 145(6), 806–15.
45. R. T. Calado, S. A. Graf, K. L. Wilkerson, S. Kajigaya, P. J. Ancliff, Y. Dror, S. J. Chanock, P. M. Lansdorp and N. S. Young, Blood, 2007, 110, 1141.
46. P. Rujkijyanont, K. Watanabe, C. Ambekar, H. Wang, A. Schimmer, J. Beyene and Y. Dror, Haematologica, 2008, 93, 363.
47. R. R. Allingham, Y. Liu and D. J. Rhee, Exp. Eye Res., 2009, 88, 837.
48. S. Monemi, G. Spaeth, A. DaSilva, S. Popinchalk, E. Ilitchev, J. Liebmann, R. Ritch, E. Héon, R. P. Crick, A. Child and M. Sarfarazi, Hum. Mol. Genet., 2005, 14, 725.
49. M. A. Hauser, R. R. Allingham, K. Linkroum, J. Wang, K. LaRocque-Abramson, D. Figueiredo, C. Santiago-Turla, E. A. del Bono, J. L. Haines, M. A. Pericak-Vance and J. L. Wiggs, Invest. Ophthalmol. Visual Sci., 2006, 47, 2542.
50. J. H. Fingert, W. L. Alward, Y. H. Kwon, S. P. Shankar, J. L. Andorf, D. A. Mackey, V. C. Sheffield and E. M. Stone, Arch. Ophthalmol., 2007, 125, 434.
51. A. W. Hewitt, D. P. Dimasi, D. A. Mackey and J. E. Craig, Am. J. Ophthalmol., 2006, 142, 324.
52. K. A. Mensink, R. P. Ketterling, H. C. Flynn, R. A. Knudson, N. M. Lindor, B. A. Heese, R. J. Spinner and D. Babovic-Vuksanovic, J. Med. Genet., 2006, 43, e08.
53. B. Bartelt-Kirbach, M. Wuepping, M. Dodrimont-Lattke and D. Kaufmann, Neurogenetics, 2009, 10, 79.
54. V. Piddubnyak, P. Rigou, L. Michel, J. C. Rain, O. Geneste, P. Wolkenstein, D. Vidaud, J. A. Hickman, A. Mauviel and J. L. Poyet, Cell Death Differ., 2007, 14, 1222.
55. A. G. Matera, R. M. Terns and M. P. Terns, Nat. Rev. Mol. Cell Biol., 2007, 8, 209.
56. T. Gill, T. Cai, J. Aulds, S. Wierzbicki and M. E. Schmitt, Mol. Cell. Biol., 2004, 24, 945.
57. D. D. Chang and D. A. Clayton, Science, 1987, 235, 1178.
58. D. D. Chang and D. A. Clayton, EMBO J., 1987, 6, 409.
59. A. N. Martin and Y. Li, Cell Res., 2007, 17, 219.
60. L. Bonafé, E. T. Dermitzakis, S. Unger, C. R. Greenberg, B. A. Campos-Xavier, A. Zankl, C. Ucla, S. E. Antonarakis, A. Superti-Furga and A. Reymond, PLoS Genet., 2005, 1, e47.
61. P. Hermanns, A. A. Bertuch, T. K. Bertin, B. Dawson, M. E. Schmitt, C. Shaw, B. Zabel and B. Lee, Hum. Mol. Genet., 2005, 14, 3723.
62. M. Ridanpää, H. van Eenennaam, K. Pelin, R. Chadwick, C. Johnson, B. Yuan, W. van Venrooij, G. Pruijn, R. Salmela, S. Rockas, O. Mäkitie, I. Kaitila and A. de la Chapelle, Cell, 2001, 104, 195.
63. C. T. Thiel, D. Horn, B. Zabel, A. B. Ekici, K. Salinas, E. Gebhart, F. Rüschendorf, H. Sticht, J. Spranger, D. Müller, C. Zweier, M. E. Schmitt, A. Reis and A. Rauch, Am. J. Hum. Genet., 2005, 77, 795.
64. C. T. Thiel, G. Mortier, I. Kaitila, A. Reis and A. Rauch, Am. J. Hum. Genet., 2007, 81, 519.
65. T. J. Welting, S. Mattijssen, F. M. Peters, N. L. van Doorn, L. Dekkers, W. J. van Venrooij, H. A. Heus, L. Bonafé and G. J. Pruijn, Biochim. Biophys. Acta, Mol. Cell Res., 2008, 1783, 455.
66. E. Nakashima, J. R. Tran, T. J. Welting, G. J. Pruijn, Y. Hirose, G. Nishimura, H. Ohashi, S. H. Schurman, J. Cheng, F. Candotti, R. Nagaraja, S. Ikegawa and D. Schlessinger, Am. J. Med. Genet., Part A, 2007, 143a, 2675.
67. M. Kirwan and I. Dokal, Clin. Genet., 2008, 73, 103.

68 S. A. Savage and B. P. Alter, *Hematol./Oncol. Clin. North Am.*, 2009, **23**, 215.

69 A. J. Walne and I. Dokal, *Mech. Ageing Dev.*, 2008, **129**, 48.

70 A. J. Walne and I. Dokal, *Br. J. Haematol.*, 2009, **145**, 164.

71 R. Yaghmai, A. Kimyai-Asadi, K. Rostamiani, N. S. Heiss, A. Poustka, W. Eyaid, J. Bodurtha, H. C. Nousari, A. Hamosh and A. Metzenberg, *J. Pediatr.*, 2000, **136**, 390.

72 N. S. Heiss, S. W. Knight, T. J. Vulliamy, S. M. Klauck, S. Wiemann, P. J. Mason, A. Poustka and I. Dokal, *Nat. Genet.*, 1998, **19**, 32.

73 B. W. Gu, M. Bessler and P. J. Mason, *Proc. Natl. Acad. Sci. U. S. A.*, 2008, **105**, 10173.

74 J. He, S. Navarrete, M. Jasinski, T. Vulliamy, I. Dokal, M. Bessler and P. J. Mason, *Oncogene*, 2002, **21**, 7740.

75 Y. Mochizuki, J. He, S. Kulkarni, M. Bessler and P. J. Mason, *Proc. Natl. Acad. Sci. U. S. A.*, 2004, **101**, 10756.

76 D. Ruggero, S. Grisendi, F. Piazza, E. Rego, F. Mari, P. H. Rao, C. Cordon-Cardo and P. P. Pandolfi, *Science*, 2003, **299**, 259.

77 A. Yoon, G. Peng, Y. Brandenburger, Y. Brandenburg, O. Zollo, W. Xu, E. Rego and D. Ruggero, *Science*, 2006, **312**, 902.

78 J. R. Mitchell, E. Wood and K. Collins, *Nature*, 1999, **402**, 551.

79 L. Montanaro, A. Chillà, D. Trerè, A. Pession, M. Govoni, P. L. Tazzari and M. Derenzini, *J. Invest. Dermatol.*, 2002, **118**, 193.

80 J. M. Wong and K. Collins, *Genes Dev.*, 2006, **20**, 2848.

81 D. C. Bittel and M. G. Butler, *Expert Rev. Mol. Med.*, 2005, **7**, 1, DOI: 10.1017/S1462399405009531.

82 J. Peters, *Nat. Genet.*, 2008, **40**, 688.

83 J. Cavaille, K. Buiting, M. Kiefmann, M. Lalande, C. I. Brannan, B. Horsthemke, J. P. Bachellerie, J. Brosius and A. Hüttenhofer, *Proc. Natl. Acad. Sci. U. S. A.*, 2000, **97**, 14311.

84 T. de los Santos, J. Schweizer, C. A. Rees and U. Francke, *Am. J. Hum. Genet.*, 2000, **67**, 1067.

85 J. A. de Smith, *et al.*, *Hum. Mol. Genet.*, 2009.

86 D. Kanber, J. Giltay, D. Wieczorek, C. Zogel, R. Hochstenbach, A. Caliebe, A. Kuechler, B. Horsthemke and K. Buiting, *Eur. J. Hum. Genet.*, 2009, **17**, 582.

87 M. Meguro, K. Mitsuya, N. Nomura, M. Kohda, A. Kashiwagi, R. Nishigaki, H. Yoshioka, M. Nakao, M. Oishi and M. Oshimura, *Hum. Mol. Genet.*, 2001, **10**, 383.

88 M. Runte, A. Hüttenhofer, S. Gross, M. Kiefmann, B. Horsthemke and K. Buiting, *Hum. Mol. Genet.*, 2001, **10**, 2687.

89 M. Runte, R. Varon, D. Horn, B. Horsthemke and K. Buiting, *Hum. Genet.*, 2005, **116**, 228.

90 T. Sahoo, D. del Gaudio, J. R. German, M. Shinawi, S. U. Peters, R. E. Person, A. Garnica, S. W. Cheung and A. L. Beaudet, *Nat. Genet.*, 2008, **40**, 719.

91 F. Ding, H. H. Li, S. Zhang, N. M. Solomon, S. A. Camper, P. Cohen and U. Francke, *PLoS One*, 2008, **3**, e1709.

92 B. V. Skryabin, L. V. Gubar, B. Seeger, J. Pfeiffer, S. Handel, T. Robeck, E. Karpova, T. S. Rozhdestvensky and J. Brosius, *PLoS Genet.*, 2007, **3**, e235.

93 T. F. Tsai, Y. H. Jiang, J. Bressler, D. Armstrong and A. L. Beaudet, *Hum. Mol. Genet.*, 1999, **8**, 1357.

94 S. L. Reichow, T. Hamma, A. R. Ferré-D'Amaré and G. Varani, *Nucleic Acids Res.*, 2007, **35**, 1452.

95 C. M. Doe, D. Relkovic, A. S. Garfield, J. W. Dalley, D. E. Theobald, T. Humby, L. S. Wilkinson and A. R. Isles, *Hum. Mol. Genet.*, 2009, **18**, 2140.

96 S. Kishore and S. Stamm, *Cold Spring Harbor Symp. Quant. Biol.*, 2006, **71**, 329.

97 S. Kishore and S. Stamm, *Science*, 2006, **311**, 230.

98 P. Vitali, E. Basyuk, E. Le Meur, E. Bertrand, F. Muscatelli, J. Cavaille and A. Hüttenhofer, *J. Cell Biol.*, 2005, **169**, 745.

99 C. Ender, A. Krek, M. R. Friedländer, M. Beitzinger, L. Weinmann, W. Chen, S. Pfeffer, N. Rajewsky and G. Meister, *Mol. Cell*, 2008, **32**, 519.

100 R. J. Taft, E. A. Glazov, T. Lassmann, Y. Hayashizaki, P. Carninci and J. S. Mattick, *RNA*, 2009, **15**, 1233.

101 P. Sridhar, H. H. Gan and T. Schlick, *J. Biomed. Sci.*, 2008, **15**, 697.

102 S. Caldarola, M. C. De Stefano, F. Amaldi and F. Loreni, *FEBS J.*, 2009, **276**, 3199.

103 J. R. Warner and K. B. McIntosh, *Mol. Cell*, 2009, **34**, 3.

104 J. M. Lipton and S. R. Ellis, *Hematol./Oncol. Clin. North Am.*, 2009, **23**, 261.

105 I. Dianzani and F. Loreni, *Haematologica*, 2008, **93**, 1601.

106 S. R. Ellis and J. M. Lipton, *Curr. Top. Dev. Biol.*, 2008, **82**, 217.

107 A. Vlachos, *et al.*, *Br. J. Haematol.*, 2008, **142**, 859.

108 R. Cmejla, J. Cmejlova, H. Handrkova, J. Petrak and D. Pospisilova, *Hum. Mutat.*, 2007, **28**, 1178.

109 N. Draptchinskaia, P. Gustavsson, B. Andersson, M. Pettersson, T. N. Willig, I. Dianzani, S. Ball, G. Tchernia, J. Klar, H. Matsson, D. Tentler, N. Mohandas, B. Carlsson and N. Dahl, *Nat. Genet.*, 1999, **21**, 169.

110 J. E. Farrar, *et al.*, *Blood*, 2008, **112**, 1582.

111 H. T. Gazda, *et al.*, *Am. J. Hum. Genet.*, 2006, **79**, 1110.

112 H. T. Gazda, *et al.*, *Am. J. Hum. Genet.*, 2008, **83**, 769.

113 I. Boria, P. Quarello, F. Avondo, E. Garelli, A. Carando, M. F. Campagnoli, I. Dianzani and U. Ramenghi, *Hum. Mutat.*, 2008, **29**, E263.

114 V. Choemel, S. Fribourg, A. H. Aguissa-Touré, N. Pinaud, P. Legrand, H. T. Gazda and P. E. Gleizes, *Hum. Mol. Genet.*, 2008, **17**, 1253.

115 I. Léger-Silvestre, J. M. Caffrey, R. Dawaliby, D. A. Alvarez-Arias, N. Gas, S. J. Bertolone, P. E. Gleizes and S. R. Ellis, *J. Biol. Chem.*, 2005, **280**, 38177.

116 S. Robledo, R. A. Idol, D. L. Crimmins, J. H. Ladenson, P. J. Mason and M. Bessler, *RNA*, 2008, **14**, 1918.

117 K. A. Bernstein, J. E. Gallagher, B. M. Mitchell, S. Granneman and S. J. Baserga, *Eukaryotic Cell*, 2004, **3**, 1619.

118 P. Martín-Marcos, A. G. Hinnebusch and M. Tamame, *Mol. Cell. Biol.*, 2007, **27**, 5968.

119 V. Choemel, D. Bacqueville, J. Rouquette, J. Noaillac-Depeyre, S. Fribourg, A. Crétien, T. Leblanc, G. Tchernia, L. Da Costa and P. E. Gleizes, *Blood*, 2007, **109**, 1275.

120 J. Flygare, A. Aspesi, J. C. Bailey, K. Miyake, J. M. Caffrey, S. Karlsson and S. R. Ellis, *Blood*, 2007, **109**, 980.

121 R. A. Idol, S. Robledo, H. Y. Du, D. L. Crimmins, D. B. Wilson, J. H. Ladenson, M. Bessler and P. J. Mason, *Blood Cells, Mol. Dis.*, 2007, **39**, 35.

122 A. Chiocchetti, L. Gibello, A. Carando, A. Aspesi, P. Secco, E. Garelli, F. Loreni, M. Angelini, A. Biava, N. Dahl, U. Dianzani, U. Ramenghi, C. Santoro and I. Dianzani, *Haematologica.*, 2005, **90**, 1453.

123 M. S. Dai, S. X. Zeng, Y. Jin, X. X. Sun, L. David and H. Lu, *Mol. Cell. Biol.*, 2004, **24**, 7654.

124 M. S. Dai, H. Arnold, X. X. Sun, R. Sears and H. Lu, *EMBO J.*, 2007, **26**, 3332.

125 Y. Zhu, M. V. Poyurovsky, Y. Li, L. Biderman, J. Stahl, X. Jacq and C. Prives, *Mol. Cell*, 2009, **35**, 316.

126 T. Uechi, Y. Nakajima, A. Chakraborty, H. Torihara, S. Higa and N. Kenmochi, *Hum. Mol. Genet.*, 2008, **17**, 3204.

127 A. Chakraborty, T. Uechi, S. Higa, H. Torihara and N. Kenmochi, *PLoS One*, 2009, **4**, e4152.

128 K. A. McGowan, J. Z. Li, C. Y. Park, V. Beaudry, H. K. Tabor, A. J. Sabnis, W. Zhang, H. Fuchs, M. H. de Angelis, R. M. Myers, L. D. Attardi and G. S. Barsh, *Nat. Genet.*, 2008, **40**, 963.

129 B. L. Ebert, *Leukemia*, 2009, **23(7)**, 1252–6.

130 J. Boulton, C. Fidler, A. J. Strickson, F. Watkins, S. Gama, L. Kearney, S. Tosi, A. Kasprzyk, J. F. Cheng, R. J. Jaju and J. S. Wainscoat, *Blood*, 2002, **99**, 4638.

131 B. L. Ebert, J. Pretz, J. Bosco, C. Y. Chang, P. Tamayo, N. Galili, A. Raza, D. E. Root, E. Attar, S. R. Ellis and T. R. Golub, *Nature*, 2008, **451**, 335.

132 S. Ferreira-Cerca, G. Pöll, H. Kühn, A. Neueder, S. Jakob, H. Tschochner and P. Milkereit, *Mol. Cell*, 2007, **28**, 446.

133 A. Pellagatti, E. Hellström-Lindberg, A. Giagounidis, J. Perry, L. Malcovati, M. G. Della Porta, M. Jädersten, S. Killick, C. Fidler, M. Cazzola, J. S. Wainscoat and J. Boulton, *Br. J. Haematol.*, 2008, **142**, 57.

134 D. Ruggero and P. P. Pandolfi, *Nat. Rev. Cancer*, 2003, **3**, 179.

135 L. B. Maggi, Jr. and J. D. Weber, *Cancer Invest.*, 2005, **23**, 599.

136 L. Montanaro, D. Trerè and M. Derenzini, *Am. J. Pathol.*, 2008, **173**, 301.

137 B. Bilanges and D. Stokoe, *Oncogene*, 2007, **26**, 5973.

138 T. Oskarsson and A. Trumpp, *Nat. Cell Biol.*, 2005, **7**, 215.

139 R. J. White, *Nat. Rev. Mol. Cell Biol.*, 2005, **6**, 69.

140 J. P. Kruse and W. Gu, *Cell*, 2009, **137**, 609.

141 M. S. Dai and H. Lu, *J. Biol. Chem.*, 2004, **279**, 44475.

142 Y. Zhang, G. W. Wolf, K. Bhat, A. Jin, T. Allio, W. A. Burkhart and Y. Xiong, *Mol. Cell. Biol.*, 2003, **23**, 8902.

143 D. Chen, Z. Zhang, M. Li, W. Wang, Y. Li, E. R. Rayburn, D. L. Hill, H. Wang and R. Zhang, *Oncogene*, 2007, **26**, 5029.

144 H. F. Horn and K. H. Vousden, *Oncogene*, 2008, **27**, 5774.

145 M. E. Castro, J. F. Leal, M. E. Lleonart, Y. S. Ramon Cajal and A. Carnero, *Carcinogenesis*, 2008, **29**, 1343.

146 M. Takagi, M. J. Absalon, K. G. McLure and M. B. Kastan, *Cell*, 2005, **123**, 49.

147 S. Fumagalli, A. Di Cara, A. Neb-Gulati, F. Natt, S. Schwemberger, J. Hall, G. F. Babcock, R. Bernardi, P. P. Pandolfi and G. Thomas, *Nat. Cell Biol.*, 2009, **11**, 501.

148 S. Volarevic, M. J. Stewart, B. Ledermann, F. Zilberman, L. Terracciano, E. Montini, M. Grompe, S. C. Kozma and G. Thomas, *Science*, 2000, **288**, 2045.

149 S. Adhikary and M. Eilers, *Nat. Rev. Mol. Cell Biol.*, 2005, **6**, 635.

150 M. Barna, A. Pusic, O. Zollo, M. Costa, N. Kondrashov, E. Rego, P. H. Rao and D. Ruggero, *Nature*, 2008, **456**, 971.

151 M. S. Dai, R. Sears and H. Lu, *Cell Cycle*, 2007, **6**, 2735.

152 A. Amsterdam, K. C. Sadler, K. Lai, S. Farrington, R. T. Bronson, J. A. Lees and N. Hopkins, *PLoS Biol.*, 2004, **2**, el39.

153 E. T. Wang, R. Sandberg, S. Luo, I. Khrebtukova, L. Zhang, C. Mayr, S. F. Kingsmore, G. P. Schroth and C. B. Burge, *Nature*, 2008, **456**, 470.

154 M. R. Passos-Bueno, C. C. Ornelas and R. D. Fanganiello, *Am. J. Med. Genet.*, Part A, 2009, **149a**, 1853.

155 S. Goobie, M. Popovic, J. Morrison, L. Ellis, H. Ginzberg, G. R. Boocock, N. Ehtesham, C. Bétard, C. G. Brewer, N. M. Roslin, T. J. Hudson, K. Morgan, T. M. Fujiwara, P. R. Durie and J. M. Rommens, *Am. J. Hum. Genet.*, 2001, **68**, 1048.

156 D. Horn, E. Ruprecht, J. Kunze and J. Spranger, *J. Med. Genet.*, 2001, **38**, 262.

157 A. Verloes, G. E. Pierard, M. Le Merrer and P. Maroteaux, *J. Med. Genet.*, 1990, **27**, 693.

158 L. Bonafé, K. Schmitt, G. Eich, A. Giedion and A. Superti-Furga, *Clin. Genet.*, 2002, **61**, 146.

159 A. J. Walne, T. Vulliamy, A. Marrone, R. Beswick, M. Kirwan, Y. Masunari, F. H. Al-Qurashi, M. Aljurf and I. Dokal, *Hum. Mol. Genet.*, 2007, **16**, 1619.

160 T. Vulliamy, R. Beswick, M. Kirwan, A. Marrone, M. Digweed, A. Walne and I. Dokal, *Proc. Natl. Acad. Sci. U. S. A.*, 2008, **105**, 8073.
