
ISSN 0006-2979, Biochemistry (Moscow), 2010, Vol. 75, No. 1, pp. 19-24. © Pleiades Publishing, Ltd., 2010.
Published in Russian in Biokhimiya, 2010, Vol. 75, No. 1, pp. 27-33.
Originally published in Biochemistry (Moscow) On-Line Papers in Press, as Manuscript BM09-096, December 13, 2009.

---

# Structure of the O-Antigen and Characterization
of the O-Antigen Gene Cluster of Escherichia coli O108
Containing 5,7-Diacetamido-3,5,7,9-tetradeoxy-L-glycero-
D-galacto-non-2-ulosonic (8-Epilégionaminic) Acid

A. V. Perepelov${}^{1*}$, Bin Liu${}^{2,3}$, S. N. Senchenkova${}^{1}$, A. S. Shashkov${}^{1}$,
S. D. Shevelev${}^{1}$, Lu Feng${}^{2,3}$, Lei Wang${}^{2,3}$, and Y. A. Knirel${}^{1}$

${}^{1}$Zelinsky Institute of Organic Chemistry, Russian Academy of Sciences, Leninskii pr. 47,
119991 Moscow, Russia; fax: (499) 137-6148; E-mail: perepel@ioc.ac.ru
${}^{2}$TEDA School of Biological Sciences and Biotechnology, Nankai University,
23 HongDa Street, TEDA, Tianjin 300457, P. R. China
${}^{3}$Tianjin Key Laboratory for Microbial Functional Genomics, TEDA College,
Nankai University, 23 HongDa Street, TEDA, Tianjin 300457, P. R. China

Received March 20, 2009  
Revision received June 3, 2009

Abstract—On mild acid degradation of the lipopolysaccharide of *Escherichia coli* O108, the O-polysaccharide was isolated and studied by sugar analysis and one- and two-dimensional ${}^{1}\mathrm{H}-$ and ${}^{13}\mathrm{C}-\mathrm{NMR}$ spectroscopy. The polysaccharide was found to contain an unusual higher sugar, 5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-D-galacto-non-2-ulosonic acid (di-$N$-acetyl-8-epilégionaminic acid, 8eLeg5Ac7Ac). The following structure of the tetrasaccharide repeating unit of the polysaccharide was established: $\rightarrow 4)-\alpha-8 \mathrm{eLeg} 5 \mathrm{Ac} 7 \mathrm{Ac}-(2 \rightarrow 6)-\alpha-\mathrm{D}-\mathrm{Galp}-(1 \rightarrow 3)-\alpha-\mathrm{L}-\mathrm{FucpNAc}-(1 \rightarrow 3)-\alpha-\mathrm{D}-\mathrm{GlcpNAc}-(1 \rightarrow$. Functions of the $E$. coli O108 antigen biosynthetic genes, including seven putative genes for synthesis of 8eLeg5Ac7Ac, were assigned by sequencing the O-antigen gene cluster along with comparison with gene databases and known biosynthetic pathways for related nonulosonic acids.

DOI: 10.1134/S0006297910010037

Key words: *Escherichia coli*, lipopolysaccharide, O-antigen, biosynthesis, O-polysaccharide structure, O-antigen gene cluster, nonulosonic acid

---

*Escherichia coli* is a clonal species and consists of both commensal and pathogenic strains, which are normally classified by a combination of their O-, H-, and K- antigens. The O-antigen, also called O-polysaccharide (OPS), is a part of the lipopolysaccharide present on the outer membrane of Gram-negative bacteria. It largely contributes to the antigenic specificity of the cell surface, serves as a receptor for bacteriophages, and plays an important role in the host immune response. O-Antigen diversity is important for bacteria as it offers selective advantage for survival of various clones in their specific niche. The O-antigen is also an important virulence factor, and its loss makes many pathogens serum sensitive or otherwise seriously impaired in virulence. Currently, 190 O-serogroups have been detected in *E*. coli and *Shigella* combined [1]. The OPS structures have been elucidated in only a little more than half of the serogroups [2].

Now we present a structure of *E*. coli O108 OPS, which contains a rarely occurring in nature higher sugar, 5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-D-galacto-non-2-ulosonic acid (di-$N$-acetyl-8-epilégionaminic acid, 8eLeg5Ac7Ac). Hitherto, a monosaccharide of this class, namely 5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-L-manno-non-2-ulosonic acid (di-$N$-acetylpseudaminic acid, Pse5Ac7Ac), has been found within *E*. coli O-antigens only once in *E*. coli O136 [2]. In addition, we characterized functionally the O-antigen gene cluster of *E*. coli O108, including putative genes of the di-$N$-acetyl-8-epilégionaminic acid biosynthetic pathway.

Abbreviations: COSY, correlation spectroscopy; HSQC, heteronuclear single-quantum coherence; OPS, O-polysaccharide (O-antigen); ROESY, rotating-frame nuclear Overhauser effect spectroscopy; TOCSY, total correlation spectroscopy.

* To whom correspondence should be addressed.

MATERIALS AND METHODS

Escherichia coli O108 type strain G1198 was obtained from the Institute of Medical and Veterinary Science (IMVS), Adelaide, Australia. Sequencing of the chromosome region between *galF* and *gnd*, analysis of genes, and search of databases for possible gene functions were performed as described [3].

Bacteria were grown to late log phase in 8 liters of Luria-Bertani liquid medium using a 10-liter fermentor (Biostat C-10; B. Braun Biotech International, Germany) under constant aeration at 37°C and pH 7.0. Bacterial cells were washed and dried as described [4].

The lipopolysaccharide (700 mg) was isolated from dried cells (10 g) by the phenol-water method [5] and purified by precipitation of nucleic acids and proteins with aqueous 50% trichloroacetic acid (to pH 2.0) followed by dialysis. Mild acid degradation of the lipopolysaccharide (270 mg) was performed with 0.1 M sodium acetate, pH 4.3, at 100°C until precipitation of lipid (1.5 h). The precipitate was removed by centrifugation (13,000g, 20 min) and the supernatant fractionated by gel chromatography on a column (56 × 2.6 cm) of Sephadex G-50 (S) in 0.05 M pyridinium acetate buffer, pH 4.5, with monitoring using a Knauer differential refractometer (Germany). The yield of the OPS was ~35% of the lipopolysaccharide weight.

The OPS was hydrolyzed with 2 M CF₃CO₂H (120°C, 2 h), monosaccharides were reduced with 0.25 M NaBH₄ in aqueous 1 M ammonia (20°C, 1 h), acetylated with a 1:1 (v/v) mixture of pyridine and acetic anhydride (120°C, 30 min), and analyzed by GLC. The absolute configurations of the monosaccharides were determined by GLC of the acetylated (*S*)-(+)-2-octyl glycosides according to the published method [6]. GLC was performed using a Hewlett-Packard 5890 chromatograph

(USA) equipped with an Ultra-1 column using a temperature gradient of 150 to 290°C at 5°C/min.

Samples were deuterium-exchanged by freeze-drying from 99.90% D₂O and then examined as solutions in 99.96% D₂O at 30°C. NMR spectra were recorded on a Bruker DRX-500 spectrometer (Germany) using internal TSP (δₕ 0) and acetone (δₐ 31.45) as references. 2D NMR spectra were obtained using standard Bruker software. Mixing times of 200 and 100 msec were used in TOCSY and ROESY experiments, respectively.

RESULTS AND DISCUSSION

Structure elucidation of the O-polysaccharide. The lipopolysaccharide was isolated from *E. coli* O108 by the phenol-water procedure. Mild acid degradation of the lipopolysaccharide with 2% acetic acid to cleave the lipid moiety resulted in significant destruction of the polysaccharide chain, and the O-polysaccharide was obtained only in a low yield. Degradation at pH 4.2 afforded a high molecular mass O-polysaccharide in a good yield. Sugar analysis using GLC of the alditol acetates after full acid hydrolysis of the O-polysaccharide revealed galactose, *N*-acetylglucosamine (GlcNAc), and *N*-acetylgalactosamine (GalNAc). GLC analysis of the acetylated glycosides with (+)-2-octanol indicated that Gal and GlcN have the D-configuration and FucN has the L-configuration.

The ¹³C-NMR spectrum of the O-polysaccharide (Fig. 1) contained signals for four anomeric carbons at δ 94.7, 98.9, 101.0 (a low intensity signal of a quaternary carbon; data from a DEPT experiment), and 102.5, four nitrogen-bearing carbons at δ 49.6–54.8, one C-CH₂-C at δ 37.7, two CH₃-C group at δ 17.0 and 20.7, two OCH₂-C groups at 62.0 and 63.0, 13 oxygen-bearing sugar ring car-

![Figure 1](attachment://image.png)

Fig. 1. ¹³C-NMR spectrum of the O-polysaccharide from *E. coli* O108. Arabic numerals refer to carbons in sugar residues denoted as follows: G, Gal; F, FucN; GN, GlcN; L, 8eLeg.

O-ANTIGEN AND O-ANTIGEN GENE CLUSTER OF E.coli O108

'H-and '³C-NMR data of the O-polysaccharide from E.coli O108 (δ, ppm)

| Sugar residue | NMR | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 |
|---------------|-----|---|---|---|---|---|---|---|---|---|
| →4)-α-8eLegp-(2→ | 'H | 174.2 | 101.0 | 1.55, 2.44 | 3.95 | 3.79 | 4.06 | 4.00 | 4.11 | 1.19 |
|               | '³C |   |   | 37.7 | 72.4 | 51.6 | 74.2 | 53.9 | 69.4 | 20.7 |
| →6)-α-D-Galp-(1→ | 'H | 5.02 | 3.73 | 3.84 | 4.05 | 4.14 | 3.38, 3.64 |   |   |   |
|               | '³C | 102.5 | 69.8 | 70.6 | 70.3 | 70.2 | 63.0 |   |   |   |
| →3)-α-L-FucpN-(1→ | 'H | 4.94 | 4.26 | 3.88 | 3.85 | 4.39 | 1.20 |   |   |   |
|               | '³C | 98.9 | 49.6 | 77.9 | 72.5 | 68.1 | 17.0 |   |   |   |
| →3)-α-D-GlcpN-(1→ | 'H | 4.86 | 4.05 | 3.64 | 3.54 | 3.54 | 3.79, 3.82 |   |   |   |
|               | '³C | 94.7 | 54.6 | 77.1 | 69.4 | 74.0 | 62.0 |   |   |   |

Note: Additional chemical shifts for N-acetyl groups are δₕ 1.93–2.04, δc 23.1–23.7(CH₃) and 175.1–176.0(CO).

bons in the region δ 68.1–77.9, four CH₃CON groups at δ  
23.1–23.7, and CO groups at δ 175.1–176.0. Accordingly,  
the 'H-NMR spectrum showed signals for three anomer-  
ic carbons at δ 4.86–5.02, one C-CH₂-C group at δ 1.55  
and 2.44, two CH₃-C groups at δ 1.19 and 1.20, and four  
CH₃CON groups at δ 1.93–2.04.  
These data suggested that, in addition to the mono-  
saccharides mentioned above, the polysaccharide con-  
tains a higher amino sugar, most likely a 5, 7-diamino-  
3, 5, 7, 9-tetradexoy-non-2-ulosonic acid [7]. Monosac-  
charides of this class are acid-labile and, therefore, the  
higher sugar was not detected in sugar analysis. All amino  
groups of the amino sugars in the polysaccharide are  
N-acetylated.  
The 'H-and '³C-NMR spectra of the O-polysaccha-  
ride were assigned using 2D 'H, 'HCOSY, TOCSY,  
ROESY, and 'H, '³CHSQC experiments (table). Tracing  
connectivities in the 2D spectra revealed spin systems for  
Gal, FucN, GlcN, and a 5, 7-diamino-3, 5, 7, 9-tetradexoy-  
non-2-ulosonic acid. Characteristic chemical shifts and 'J  
coupling constants of 'J₁,₂ of 3–4 Hz showed that Gal,  
FucN, and GlcN are in the form of α-pyranosides.  
The nonulosonic acid was identified as the L-glycero-  
α-D-galacto isomer (α-8-epilegionaminic acid, 8eLeg) as  
follows:  
—the '³C-NMR spectrum contained characteristic  
signals for C-1–C-9 of a 5, 7-diamino-3, 5, 7, 9-  
tetradexoy-non-2-ulosonic acid including those for C-2  
(keto group), C-3, and C-9 (deoxy units), nitrogen-bear-  
ing C-5 and C-7, oxygen-bearing C-4, C-6, and C-8;  
—relatively large 'J₃ax,₄, 'J₄,₅, and 'J₅,₆ coupling con-  
stants ~10 Hz demonstrated the axial orientation of H-4,  
H-5, and H-6; a small 'J₆,₇ value (~2 Hz) and J₇,₈ of  
~6 Hz showed the three configuration of the C-6–C-7  
and C-7–C-8 fragments; therefore, the nonulosonic acid  
has the L-glycero-D-galacto configuration (compare pub-  
lished data [7] of various isomers of 5, 7-diamino-3, 5, 7, 9-  
tetradexoy-non-2-ulosonic acids);

—a relatively large difference (~0.9 ppm) between  
the H-3ax and H-3eq chemical shifts is typical of 3-  
deoxy-aldulosonic acids with the axial position of the car-  
boxyl group, i.e. the α-configuration of 8eLeg. The iden-  
tity of 8eLeg was confirmed by isolation of 8eLeg5Ac7Ac  
in the free state by three successive Smith degradations of  
the polysaccharide and comparison of its 'H-and '³C-  
NMR spectra with those of the authentic synthetic sam-  
ple [8].  
The 2D ROESY experiment revealed the following  
strong cross-peaks between anomeric protons and pro-  
tons at the linkage carbons: Gal H-1, FucN H-3, FucN  
H-1, GlcN H-3, and GlcN H-1, 8eLeg H-4 at δ  
5.02/3.88, 4.94/3.64, and 4.86/3.95, respectively. These  
data demonstrated a Gal-(1→3)-FucN-(1→3)-GlcN-  
(1→4)-8eLeg fragment. Relatively low-field positions of  
the signal for C-3 of FucN, C-3 of GlcN, and C-4 of  
8eLeg, as compared with their positions in the correspon-  
ding non-substituted monosaccharide [7, 9], confirmed  
the sugar glycosylation pattern in this fragment. Only a  
small down-field shift (<1 ppm) of the signal for C-6 of  
Gal caused by glycosylation with 8eLeg is typical of keto-  
sidic linkage.  
Based on the data obtained, it was concluded that the  
OPS of E.coli O108 has a tetrasaccharide repeating unit  
(O-unit) having the following structure:  
→4)-α-8eLegp5Ac7Ac-(2→6)-α-D-Galp-(1→3)-α-L-  
FucpNAc-(1→3)-α-D-GlcpNAc-(1→.  

Di-N-acetyl derivative of 8-epilegionaminic acid  
present in the polysaccharide occurs rarely in nature.  
After the first discovery of 8eLeg in the O-polysaccharide  
of Pseudomonas aeruginosa O12 (formerly O13) [10], this  
nonulosonic acid has been found in several other bacteri-  
al polysaccharides, including O-antigens of Yersinia ruck-  
eri O1, Salmonella arizonae O61, Morganella morganii RF  
1676, Shewanella petrefaciens A6 [7], and Providencia

BIOCHEMISTRY (Moscow) Vol. 75 No. 1 2010

alcalifaciens O20 [9]. A wrong configuration originally assigned to 8eLeg [10] was revised later using chemical synthesis of various isomers of 5,7-diamino-3,5,7,9-tetradeoxy-non-2-ulosonic acids [7].

Characterization of the O-antigen gene cluster of *E. coli* O108. Sequencing revealed the O-antigen gene cluster of *E. coli* O108 consisting of 21,225 bp and located between *galF* and *gnd*. Eighteen open reading frames

a

| C-5-Epimerase/ | C-2-Epimerase | Acetyltransferase | Nucleotidase | Cytidylyl transferase | Glycosyl transferase |
| --- | --- | --- | --- | --- | --- |
| 4,6-dehydratase | Aminotransferase | Synthetase |  | Flippase | O-Antigen polymerase |
|  |  |  |  | Glycosyl transferase | 4,6-Dehydratase/C-5-epimerase |
|  |  |  |  |  | 4-Reductase |
|  |  |  |  |  | C-2-Epimerase |
|  |  |  |  |  | Glycosyl transferase |


galF orf1 orf2 orf3 orf4 | orf6 orf7 | orf9 orf10 wzx wzy orf13 fnlA fnlB fnlC wbuB | gnd
                      orf5       orf8


CMP-8eLeg5Ac7Ac biosynthesis genes

UDP-L-FucNAc biosynthesis genes

b


CH₂OH
    O
HO
    CH₃
        HO
            AcNH
                OUDP


Orf1


O
CH₃
    HO
        AcNH
            OUDP


H₂O

L-Glu

α-ketoglutarate


H₂N
    O
CH₃
    HO
        AcNH
            OUDP


Orf5


AcN
    H
CH₃
    HO
        AcNH
            OUDP


Ac-CoA

CoA


AcN
    H
CH₃
    HO
        AcNH
            OUDP


Orf6


AcN
    H
CH₃
    HO
        OH
            PEP


H₂O

UDP

Pᵢ


OH
    CH₃
        H
            AcN
                OH
                    CO₂H
                        AcN
                            H
                                HO
                                    AcN
                                        H
                                            HO


Orf8


CH₃
    H
        AcN
            OH
                O
                    OCM
                        CO₂H
                            AcN
                                H
                                    HO
                                        AcN
                                            H
                                                HO


CTP

PPᵢ

Fig. 2. Organization of the O-antigen gene cluster of *E. coli* O108 (a) and proposed biosynthetic pathway of 8eLeg5Ac7Ac in *E. coli* O108 (b). PEP, phosphoenolpyruvate.

(orfs) were identified in the cluster, all having the same transcriptional direction from galF to gnd (Fig. 2a). Most orfs were assigned functions based on their similarity to those from available databases and taking into account the E. coli O108 antigen structure.

Genes for the synthesis of the nucleotide precursors of common sugars (GlcNAc and Gal in case of E. coli O108) are located outside the O-antigen gene cluster [11], and only genes for the synthesis of CMP-8eLeg5Ac7Ac and UDP-L-FucNAc were expected inside.

8eLeg5Ac7Ac is highly similar to isomeric nonulosonic acids found in bacterial carbohydrates, Pse5Ac7Ac and Leg5Ac7Ac, having the L-glycero-L-manno and D-glycero-D-galacto configuration, respectively. For synthesis of CMP-Pse5Ac7Ac, bacteria Campylobacter jejuni and Helicobacter pylori employ six enzymes (PseB, PseC, PseF, PseG, PseH, and PseI) [12–14]. PseB exhibits both C-6-dehydratase and C-5-epimerase activity that are responsible for the conversion of UDP-D-GlcNAc into both UDP-2-acetamido-2,6-dideoxy-β-L-arabino-hex-4-ulose and its α-D-xylo isomer. PseC is an aminotransferase, which utilizes only the first keto sugar as substrate for synthesis of UDP-2-acetamido-4-amino-2,4,6-trideoxy-β-L-altropyranose. PseH is an acetyltransferase, which is responsible for the 4-N-acetylation of the PseC product to yield UDP-2,4-diacetamido-2,4,6-trideoxy-β-L-altropyranose. PseG is a nucleotidase, which removes the UDP group from the PseH product to afford 2,4-diacetamido-2,4,6-trideoxy-β-L-altropyranose. PseI catalyzes the condensation of the PseG product with phosphoenolpyruvate giving rise to Pse5Ac7Ac. PseF is a cytidylyl transferase, which converts Pse5Ac7Ac into CMP-Pse5Ac7Ac.

A biosynthetic pathway of Leg5A7Ac, which has been studied in the bacterium Legionella pneumophila [15], is similar to that of Pse5Ac7Ac, but there are two major differences:

- the first enzyme of the pathway, PglF, converts UDP-D-GlcNAc to UDP-2-acetamido-2,6-dideoxy-α-D-xylo-hex-4-ulose and thus has only C-6-dehydratase activity with no C-5-epimerase activity;
- the hydrolyzing enzyme that removes UDP from UDP-2,4-diacetamido-2,4,6-trideoxy-α-D-glucopyranose (NeuC homolog) possesses both C-2-epimerase and nucleotidase activity giving rise to 2,4,6-trideoxy-D-mannopyranose.

The biosynthesis pathway of 8eLeg5Ac7Ac, which was isolated for the first time from the OPS of E. coli O108 and unambiguously identified only recently [8], remains unknown. The similarity of its structure to those of Leg5Ac7Ac and Pse5Ac7Ac suggest that the biosynthesis pathway of 8eLeg5Ac7Ac in E. coli O108 is similar too (Fig. 2b). This was confirmed by molecular analysis of the genes in the E. coli O108 O-antigen gene cluster that are putatively involved with the 8eLeg5Ac7Ac synthesis.

Orf1 belongs to a NAD-dependent epimerase/dehydratase family (PF01370, $E_{\text{value}} = 4.4 \cdot e^{-214}$). It was proposed to have the same function as PseB and to catalyze conversion of UDP-D-GlcNAc into UDP-2-acetamido-2,6-dideoxy-β-L-arabino-hex-4-ulose. Orf2 shares 58% similarity to PglE of C. jejuni, which has been identified as aminotransferase [13]. Therefore, it was suggested that Orf2 is an aminotransferase too, whose function is similar to that of PseC. The enzymes have the same substrate but the Orf2 product is UDP-2-acetamido-4-amino-2,4,6-trideoxy-β-L-idose rather than UDP-2-acetamido-4-amino-2,4,6-trideoxy-β-L-altropyranose produced by PseC in the Pse5Ac7Ac pathway. Orf5 shares 68% similarity to acetyltransferase of Vibrio vulnificus, and its putative function is to add an acetyl group to the Orf2 product to give UDP-2,4-diacetamido-2,4,6-trideoxy-β-L-idose.

Orf3 shares 77% identity to C-2-epimerase of V. vulnificus. Therefore, Orf3 was proposed to catalyze the conversion UDP-2,4-diacetamido-2,4,6-trideoxy-β-L-idose to its C-2-epimer, UDP-2,4-diacetamido-2,4,6-trideoxy-β-L-gulose. Orf6 shares 66% similarity to nucleotidyltransferase of Nitrobacter winogradskyi, and we proposed Orf6 is a nucleotidase responsible for the removal of UDP from the Orf3 product. Orf4 is a homolog of PseI and was proposed to catalyze the condensation of 2,4-diacetamido-2,4,6-trideoxy-β-L-gulose with phosphoenolpyruvate to afford 8eLeg5Ac7Ac. Orf8 shares 67% identity with cytidylyl transferase of Vibrio fischeri and, therefore, catalyzes the synthesis of CMP-8eLeg5Ac7Ac from 8eLeg5Ac7Ac.

Orf14, Orf15, and Orf16 share 92, 79, and 94% identity to FnlA, FnlB, and FnlC of E. coli O15, respectively. These three have been identified as enzymes for the biosynthesis of UDP-L-FucNAc [16]. Therefore, Orf14–Orf16 in E. coli O108 are responsible for the synthesis of UDP-L-FucNAc.

Wzx and Wzy are highly hydrophobic membrane proteins involved with O-antigen processing. They usually share little sequence identities with their homologs. In the E. coli O108 O-antigen gene cluster, Orf11 and Orf12 are the only two proteins with predicted transmembrane segments. Orf11 was predicted to have 11 well-proportioned transmembrane segments and shares 47% similarity to Wzx of Streptococcus pneumoniae. Orf12 was found to have nine predicted transmembrane segments and a large periplasmic loop of 92 amino acid residues, which is a typical topological character of Wzy proteins [17]. Orf12 shares 46% similarity to Wzy of Salmonella enterica group E1. Therefore, Orf11 and Orf12 are O-unit flippase and O-antigen polymerase responsible for the translocation through the periplasmic membrane and polymerization of the O-unit, respectively.

GlcNAc is the first sugar of the O-unit, which is transferred to a lipid carrier by GlcNAc-1-phosphate transferase WecA. As the wecA gene is located outside the O-antigen gene cluster [18], only three glycosyl trans-

ferase genes for the assembly of the tetrasaccharide O-
unit of *E. coli* O108 were expected in the O-antigen gene
cluster. Orf13 shares 42% similarity to glycosyl transferase
of *S. enterica* serovar Paratyphi A. Orf17 shares 87% iden-
tity to WbuB of *E. coli* O15, which is a L-FucNAc trans-
ferase involved in the O-antigen synthesis [19, 20]. Orf9
belongs to glycosyl transferases group 1 family (PF00534,
$E_{\text {value }} = 2.2 \cdot e^{-44}$). Therefore, *orf9*, *orf13*, and *orf17* are
putative genes for 8eLeg5Ac7Ac, D-Gal, and L-FucNAc
glycosyl transferases involved with the O-unit assembly in
*E. coli* O108.

Orf18 shares 81% identity to WbuC of *E. coli* O15. It
has a reduced size and was suggested to be a non-func-
tional gene remnant [19]. No function could be assigned
to *orf7* and *orf10* by searching currently available database.
Therefore, all genes necessary for the O-antigen
biosynthesis were found in the *E. coli* O108 O-antigen
gene cluster. Molecular analysis of the corresponding
genes suggests the hitherto unknown 8eLeg5Ac7Ac
biosynthetic pathway, which is basically similar to that of
Pse5Ac7Ac and Leg5Ac7Ac. The following differences
are observed between the pathways of the three non-
ulosonic acids:
- C-6 dehydration is accompanied by C-5 epimer-
ization for synthesis of Pse5Ac7Ac and 8eLeg5Ac7Ac but
not Leg5Ac7Ac;
- removal of UDP is accompanied by C-2 epimer-
ization in the pathways of Leg5Ac7Ac and 8eLeg5Ac7Ac
but not Pse5Ac7Ac;
- one enzyme (hydrolyzing C-2-epimerase) or two
enzymes (C-2-epimerase and nucleotidase) are involved
at this step of Leg5Ac7Ac and 8eLeg5Ac7Ac synthesis,
respectively.

The proposed 8eLeg5Ac7Ac biosynthesis pathway
requires biochemical confirmation.

This work was supported by the Russian Foundation
for Basic Research (grants No. 08-04-01205 and 08-04-
92225), the Council on Grants at the President of the
Russian Federation for Support of Young Russian
Scientists (grant MK-157.2007.4), the National Natural
Science Foundation of China (NSFC) Key Program
(grant Nos. 30530010 and 20536040) and General
Program (grant No. 30670038), the National 973 pro-
gram of China (grant No. 2009CB522603), and the
Chinese National Science Fund for Distinguished Young
Scholars (grant No. 30788001).

---

**REFERENCES**

1. Liu, B., Knirel, Y. A., Feng, L., Perepelov, A. V., Senchenkova, S. N., Wang, Q., Reeves, P., and Wang, L. (2008) *FEMS Microbiol. Rev.*, **32**, 627-653.
2. Staaf, M., Weintraub, A., and Widmalm, G. (1999) *Eur. J. Biochem.*, **263**, 656-661.
3. Feng, L., Senchenkova, S. N., Yang, J., Shashkov, A. S., Tao, J., Guo, H., Cheng, J., Ren, Y., Knirel, Y. A., Reeves, P., and Wang, L. (2004) *J. Bacteriol.*, **186**, 4510-4519.
4. Robbins, P. W., and Uchida, T. (1962) *Biochemistry*, **1**, 323-335.
5. Westphal, O., and Jann, K. (1965) *Meth. Carbohydr. Chem.*, **5**, 83-91.
6. Leontein, K., and Lönngren, J. (1993) *Meth. Carbohydr. Chem.*, **9**, 87-89.
7. Knirel, Y. A., Shashkov, A. S., Tsvetkov, Y. E., Jansson, P.-E., and Zähringer, U. (2003) *Adv. Carbohydr. Chem. Biochem.*, **58**, 371-417.
8. Knirel, Y. A., Senchenkova, S. N., Shashkov, A. S., Perepelov, A. V., Shevelev, S. D., Liu, B., Feng, L., and Wang, L. (2009) *Adv. Sci. Lett.*, **2**, 384-387.
9. Lipkind, G. M., Shashkov, A. S., Knirel, Y. A., Vinogradov, E. V., and Kochetkov, N. K. (1988) *Carbohydr. Res.*, **175**, 59-75.
10. Knirel, Y. A., Vinogradov, E. V., Shashkov, A. S., Dmitriev, B. A., Kochetkov, N. K., Stanislavsky, E. S., and Mashilova, G. M. (1987) *Eur. J. Biochem.*, **163**, 627-637.
11. Samuel, G., and Reeves, P. (2003) *Carbohydr. Res.*, **338**, 2503-2519.
12. Schoenhofen, I. C., McNally, D. J., Brisson, J.-R., and Logan, S. M. (2006) *Glycobiology*, **16**, 8C-14C.
13. Schoenhofen, I. C., McNally, D. J., Vinogradov, E., Whitfield, D., Young, N. M., Dick, S., Wakarchuk, W. W., Brisson, J. R., and Logan, S. M. (2006) *J. Biol. Chem.*, **281**, 723-732.
14. Chou, W. K., Dick, S., Wakarchuk, W. W., and Tanner, M. E. (2005) *J. Biol. Chem.*, **280**, 35922-35928.
15. Glaze, P.A., Watson, D. C., Young, N. M., and Tanner, M. E. (2008) *Biochemistry*, **47**, 3272-3282.
16. Kneidinger, B., O'Riordan, K., Li, J., Brisson, J. R., Lee, J. C., and Lam, J. S. (2003) *J. Biol. Chem.*, **278**, 3615-3627.
17. Daniels, C., Vindurampulle, C., and Morona, R. (1998) *Mol. Microbiol.*, **28**, 1211-1222.
18. Reeves, P., and Wang, L. (2002) *Curr. Top. Microbiol. Immunol.*, **264**, 109-135.
19. D'Souza, J. M., Wang, L., and Reeves, P. (2002) *Gene*, **297**, 123-127.
20. Beutin, L., Tao, J., Feng, L., Krause, G., Zimmermann, S., Gleier, K., Xia, Q., and Wang, L. (2005) *J. Clin. Microbiol.*, **43**, 703-710.
