
Biochimica et Biophysica Acta 1799 (2010) 671–680

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbagrm

Review

Chromatin remodeling regulation by small molecules and metabolites

Giosalba Burgio $^{1}$, Maria C. Onorati $^{1}$, Davide F.V. Corona $^{\ast}$

Dipartimento di Biologia Cellulare e dello Sviluppo, Università degli Studi di Palermo, 90100 Palermo, Italy

Istituto Telethon Dulbecco, Università degli Studi di Palermo, 90100 Palermo, Italy

---

**ARTICLE INFO**

Article history:

- Received 25 February 2010
- Accepted 8 May 2010
- Available online 20 May 2010

Keywords:
- Chromatin remodeling
- ATP
- AcCoA
- SAM
- NAD
- Poly-ADP-ribosylation
- Phosphatidylinositide
- Inositol polyphosphates

---

**ABSTRACT**

The eukaryotic genome is a highly organized nucleoprotein structure comprising of DNA, histones, non-histone proteins, and RNAs, referred to as chromatin. The chromatin exists as a dynamic entity, shuttling between the open and closed forms at specific nuclear regions and loci based on the requirement of the cell. This dynamicity is essential for the various DNA-templated phenomena like transcription, replication, and repair and is achieved through the activity of ATP-dependent chromatin remodeling complexes and covalent modifiers of chromatin. A growing body of data indicates that chromatin enzymatic activities are finely and specifically regulated by a variety of small molecules derived from the intermediary metabolism. This review tries to summarize the work conducted in many laboratories and on different model organisms showing how ATP-dependent chromatin remodeling complexes are regulated by small molecules and metabolites such as adenosine triphosphate (ATP), acetyl coenzyme A (AcCoA), S-adenosyl methionine (SAM), nicotinamide adenine dinucleotide (NAD), and inositol polyphosphates (IPs).

© 2010 Elsevier B.V. All rights reserved.

---

1. Introduction

Eukaryotic DNA is organized into chromatin, a dynamic combination of DNA and proteins that makes up chromosomes. The basic repeating element of eukaryotic chromatin is the nucleosome, which consists of two molecules each of the histone proteins H2A, H2B, H3, and H4 wrapped around 147 bp of DNA [1]. In order to achieve the high level of control required to coordinate nuclear processes such as transcription, DNA replication, and repair, eukaryotic cells have developed a variety of mechanisms to locally and specifically modulate chromatin structure and function. This can involve covalent modification of histones, the incorporation of histone variants, and the non-covalent remodeling of nucleosomes by ATP-dependent nucleosome remodeling enzymes.

ATP-dependent nucleosome remodeling factors are classified into subfamilies depending on the presence of other domains within the ATPase-containing subunit. The four main subfamilies characterized so far are the SWI/SNF, ISWI, CHD, and INO80 subfamilies (Fig. 1a). Remodeling enzymes have been shown to slide nucleosomes along DNA, to exchange/remove histones, and to disrupt histone–DNA contacts (Fig. 1b). At present, it is not clear whether all of these represent distinct reactions or merely alternative outcomes of a common mechanism. Nevertheless, what is shared among all ATP-dependent chromatin remodeling enzymes is that their activity results in altered DNA accessibility.

The variety of functions associated with nucleosome remodeling factors indicates that these nuclear enzymatic activities are highly regulated. Chromatin remodeling enzymes usually work in the context of multisubunit complexes. Indeed, it has been shown in many model organisms that chromatin remodeling factors can be regulated by their subunits [2,3]. However, recent works highlighted important roles also for small active molecules coming from the cell intermediary metabolism such as adenosine triphosphate (ATP), acetyl coenzyme A (AcCoA), S-adenosyl methionine (SAM), nicotinamide adenine dinucleotide (NAD), and inositol polyphosphates (IPs) in the regulation of chromatin remodeling activity (Fig. 2). These metabolites can themselves directly regulate the activity of nucleosome remodeling factors or can be used by other enzymes to covalently modify remodelers or other proteins regulating them. This review aims at illustrating the multitude of ways by which ATP-dependent chromatin remodeling complexes can be regulated by small molecules and metabolites such as ATP, AcCoA, SAM, NAD, and IPs.

2. AcCoA and acetylation

Acetylation is a regulatory post-translational modification catalyzed by enzymes (acetyltransferases) that use the AcCoA as a substrate to

---

* Corresponding author. Tel.: +39 091 238 97345; fax: +39 091 238 60721.
* E-mail address: dcorona@unipa.it (D.F.V. Corona).
$^{1}$ Equal contribution.

1874-9399/$ – see front matter © 2010 Elsevier B.V. All rights reserved.

doi:10.1016/j.bbagrm.2010.05.007

G. Burgio et al. / Biochimica et Biophysica Acta 1799 (2010) 671–680

**a**

SWI/SNF family  
HSA  
Bromo  

DExx  
HELICc  

ISWI family  

DExx  
HELICc  

SANT SLIDE  

CHD family  
Chromo  

DExx  
HELICc  

INO80 family  
HSA  

DExx  
Long spacer  
HELICc  

**b**

Nucleosome Sliding  

Nucleosome Exchange  

Nucleosome Eviction  

Altered Nucleosome Structure  

Fig. 1. A summary of the different classes of nucleosome remodeling ATPases and reactions they catalyze. (a) Each catalytic subunit is characterized by an ATPase domain (DExx-spacer-HELICc) flanked by unique domains characteristic of each family. Members of the SWI/SNF subfamily contain a bromodomain (red) that binds acetylated histones. ISWI contains a SANT-SLIDE (blue) module recognizing unmodified histones. The CHD subfamily contains a chromodomain (yellow) recognizing methylated histones. The INO80 subfamily has a characteristic long spacer (gray) within the ATPase domain and like the SWI/SNF subfamily also contains an HSA (helicase-SANT) domain (green). (b) Chromatin remodeling factors can use the energy of ATP hydrolysis to catalyze a variety of reactions on the nucleosome substrate without covalently modifying the DNA or its associated histones. Remodelers can slide nucleosomes to make certain chromatin domains differently accessible. The same task can also be achieved by evicting histones, replacing them with variants, or by exposing a DNA loop on the surface of the nucleosome.

transfer an acetyl group to lysine residues of other proteins [4]. This covalent modification is counterbalanced by the activity of other enzymes (deacetylases) that can remove the acetyl group by different mechanisms [5]. Acetyltransferases can directly covalently modify the catalytic ATPase subunit of chromatin remodeling complexes (Fig. 3a). Studies conducted in mouse fibroblasts have shown that BRM, one of the two ATPase subunits of the SWI/SNF complex, has two motifs containing lysine residues that can be modified by PCAF acetyltransferase, in vitro at the C-terminal domain [6]. It has been shown that the acetylation of BRM has a role in destabilizing its interaction with the retinoblastoma onco-suppressor protein pRb, resulting in deregulation of cell proliferation [2]. Moreover, treatment of BRM non-expressing cancer cell lines with inhibitors of histone deacetylases (HDACi) restores BRM expression. However, HDACi treatment can also induce BRM acetylation that in turn blocks its function [7].

Despite the high homology existing between BRM and BRG1 (the second ATPase subunit of the SWI/SNF complex [8]), there is no direct evidence for BRG1 acetylation at day. One interesting possibility is that BRG1 could be indirectly regulated by BRM acetylation. Acetylated BRM could have a dominant negative effect on BRG1 activity, by competing for

the SWI/SNF complex subunits [6]. Alternatively, since BRG1 is part of the nuclear receptor corepressor (N-CoR) complex that could associate with HDAC activities [9], an intriguing speculation is that HDACs in N-CoR complex may maintain BRG1 in a deacetylated state thus regulating the catalytic activity of the SWI/SNF complex [3].

In vivo and in vitro studies have also shown that in Drosophila the nucleosome remodeling ATPase ISWI is acetylated at lysine K753 by the GCN5 acetyltransferase. Interestingly, acetylated ISWI associates with NURF301 but not with ACF1 [10], two regulatory subunits of distinct ISWI complexes [11]. Although the exact biological function of ISWI acetylation has still to be elucidated, one intriguing observation is that acetylated ISWI localizes on condensed metaphase chromosomes in early embryos [10].

The ATPase subunits of chromatin remodelers are usually embedded in multiprotein complexes [2]. Factors associated with the catalytic remodeling ATPase subunit could be themselves post-translationally modified, to modulate nucleosome remodeling reactions (Fig. 3a). Indeed, RSC4 is a component of the yeast RSC chromatin remodeling complex, highly related to SWI/SNF [12]. RSC4 is acetylated at lysine K25 by GCN5 and this modification prevents its binding to histone H3

G. Burgio et al. / Biochimica et Biophysica Acta 1799 (2010) 671–680

![image](https://i.imgur.com/1234567.png)

AcCoA

![image](https://i.imgur.com/2345678.png)

ATP

![image](https://i.imgur.com/3456789.png)

NAD

![image](https://i.imgur.com/4567890.png)

SAM

Fig. 2. Small molecules coming from the intermediary metabolism possess active chemical groups (red squared boxes) that can be transferred to the amino acid residues of chromatin remodeling ATPases or their regulators. Acetyltransferase can use AcCoA as donor of acetyl group, kinases utilize ATP to transfer the orthophosphate group, NAD can be used by many nuclear enzymes to produce poly-ADP-ribose, phospholipases can split phosphatidylinositides into different forms of inositol polyphosphates (IPs), methyltransferases can transfer the thio-methyl group present in the SAM. Acetyl coenzyme A (AcCoA), adenosine triphosphate (ATP), nicotinamide adenine dinucleotide (NAD), inositol polyphosphates (IPs), and S-adenosyl methionine (SAM).

acetylated in K14 (H3K14Ac) [13]. In another example, the SNF2h ATPase, a mammalian homolog of ISWI, is associated with TIP5 in nucleolar remodeling complex (NoRC), a chromatin remodeling complex regulating ribosomal gene expression [14]. MOF-dependent acetylation of TIP5 on lysine K633 regulates the formation of silent heterochromatin at rDNA loci [15].

Histones are targets of a large number of covalent modifications, defining a complex network of epigenetic information known as “histone code” [16,17]. Histone acetyltransferases (HATs) can covalently attach the acetyl group of AcCoA to lysine residues present in the histone N- and C-terminal tails [18]. Acetylation of histone tails is often necessary for the recruitment of ATP-dependent chromatin remodelers or for the modulation of their activity (Fig. 3b and c). Data coming from yeast show that acetylated histones mediate the retention of SWI/SNF chromatin remodeling complex at the HO promoter [19]. Moreover, the retention of the SWI/SNF complex at the PHO5 promoter is compromised in cells mutated for gcn5, showing the existence of an interdependence between nucleosome remodeling and HAT activities [20]. Furthermore, GCN5-dependent histone acetylation of the human IFN-β promoter provides a high-affinity surface for SWI/SNF [21]. Subsequent studies have eventually shown that GCN5 specifically acetylates histone H4 on K8 (H4K8Ac), which mediates the recruitment of the SWI/SNF complex [22]. PCAF, another HAT, has been shown to acetylate histone H3 at K14 (H3K14Ac) of mouse MMTV promoter, providing an anchor site for BRG1 and BRM, to catalyze the histone H2A/H2B dimer removal through the BAF complex [23].

Fig. 3. Chromatin remodeling regulation by AcCoA. Schematic representation of nucleosome remodeling reactions regulation by direct acetylation of the remodeling ATPases, their complex subunits and histone tails.
The interaction of the yeast RSC complex with nucleosomes acetylated on histone H3 in K9 (H3K9Ac) induces conformational changes of RSC itself, resulting in the stabilization of a closed conformation of the enzyme, a prerequisite for the remodeling mechanism [24]. Acetylated histones can also bind Bdf1, a TFIID-interacting protein, that in turn can recruit the yeast SNF2-family SWR-C complex in order to modulate heterochromatin boundary elements near telomeres through the exchange of the histone Htz1 variant [25]. Another component of the RSC complex, RSC4, through its bromodomain can be recruited by histone H3 acetylations in K14 (H3K14Ac), thus regulating genes involved in the nicotinic acid synthesis and cell wall integrity (Fig. 3a) [12].

Site-specific acetylation of histone H4 on K16 (H4K16Ac) is an important post-translational modification involved in transcriptional activation and euchromatin maintenance [26,27]. In *Drosophila melanogaster*, H4K16Ac is involved in the increased male X linked genes transcription, occurring during dosage compensation [28,29]. This histone modification has important effect on chromatin remodeling regulation. *Drosophila* ISWI ATPase activity is counteracted by H4K16Ac *in vitro* and *in vivo* [30]. Moreover, the incorporation of H4K16Ac in nucleosomal particles impairs the ISWI-containing complex ACF to mobilize mononucleosomes *in vitro* [31]. On the other hand, H4K16Ac is necessary for the targeted chromatin recruitment of NURF301, the largest subunit of NURF, another ISWI-containing complex [11,32]. Furthermore, H4K16Ac is also required for chromatin binding of the NoRC complex subunit TIP5, to promote the deposition of histone heterochromatin epigenetic marks and to induce transcriptional silencing of rRNA genes (Fig. 3a) [33].

### 3. ATP and phosphorylation

Protein phosphorylation is a key event during cell cycle transition from interphase to mitosis [34]. In particular, it has been estimated that more than 50 proteins become phosphorylated during mitosis [35]. In response to different cellular stimuli, chromatin remodeling factors can also be directly or indirectly regulated by phosphorylation (Fig. 4). Indeed, the nucleosome remodeling factor BRG1 has been identified in a phosphorylated form in Xenopus embryos [36]. Studies in mammals have shown that phosphorylated hBRM and BRG1 have low affinity for the nuclear structure in early M phase and are excluded from mitotic chromosomes, suggesting a mechanism in which the exclusion of BRM and BRG1 is functional to the transcriptional arrest occurring during mitosis [37]. In addition to BRG1 and BRM, hSWI/SNF is phosphorylated during mitosis at the level of the hSWI3 subunit by the ERK1 kinase. Phosphorylated hSWI3 also promotes the switch of hSWI/SNF complex to an inactive form that is compatible with the repressed state of chromatin during mitosis [38].

Similarly, the *Drosophila* Mi-2 chromatin remodeler is a phosphoprotein *in vivo*. The dCK2 kinase binds and phosphorylates Mi-2 at its N-terminal domain, and dephosphorylation is a step necessary to promote Mi-2-dependent nucleosome binding and spacing activity [39]. The Williams Syndrome Transcription Factor (WSTF) protein, a subunit of the WSTF Including Nucleosome Assembly Complex (WINAC) chromatin remodeling complex, is also phosphorylated by MAPK kinases on Ser158. In this case, the phosphorylation of WSTF is essential to maintain its association with other complex components and to promote WINAC remodeling activity [40].

As it has been shown for histone acetylation events, histone phosphorylation also plays an essential role in regulating ATP-dependent chromatin remodeling reactions. For example, immunodepletion of INCEP-aurora B kinase complex reduces the association of XCAP-F, the *Xenopus* ortholog of ISWI, with mitotic chromosomes suggesting a functional dependence of XCAP-F binding to phosphorylated H3 on S10 (H3S10P) [41]. On the other hand, H3S10P stimulates the recruitment of the BAF complex at the MMTV promoter to induce its activation [42]. However, H3S10P can also block the RSC complex by inhibiting RSC4 binding to H3K14Ac [13].

Other cellular cues can elicit chromatin remodeling regulation by phosphorylation. Indeed, during exposure to DNA damaging agents, the les-4 subunit of the INO80 chromatin remodeling complex is target of phosphorylation catalyzed by the Mec1/Tel1 kinase (ATM/ATR in mammals), a covalent modification important for the DNA damage checkpoint response [43]. Complementary studies have shown that the INO80 chromatin remodeling complex is recruited by the Double Strand Break (DSB)-induced HO endonuclease by phosphorylated histone H2A

![Diagram](attachment:diagram.png)

**Fig. 4.** Chromatin remodeling regulation by ATP. Schematic representation of nucleosome remodeling reactions regulation by direct phosphorylation of the remodeling ATPases, their complex subunits and histone tails.
at S129 (H2AS129P). In cells lacking H2AS129P, the recruitment of INO80 is compromised, suggesting that ATP-dependent chromatin remodelers are also able to read DNA damage-repair histone marks [44,45].

# 4. SAM and methylation

Works conducted in several model organisms have identified many nuclear protein methyltransferases, but evidence for direct post-translational methylation of chromatin remodeling ATPases and their associated subunits is still not present in literature. However, lysine and arginine residues of histones can be good targets for methylation by histone methyltransferases [46–48]. In fact, methylated histones could offer a specific interacting surface for nucleosome remodeling enzymes and their regulators (Fig. 5).

The methylation of histone H3 at K4 (H3K4me) is normally associated with actively transcribed genes [49,50]. The chromatin remodeler CHD1 is the first identified chromodomain-containing protein recognizing methylated histones. Interestingly, the human but not the yeast CHD1 directly and specifically recognizes H3K4me [51]. Interestingly, CHD1 was identified as a component of SAGA and SLIK, two complexes associated with HAT activity. Indeed, CHD1 chromodomain mediated binding to H3K4me is a prerequisite for histone hyperacetylation associated with transcriptional activity [52,53].

The trithorax group protein Kismet-L (KIS-L) is a member of the CHD subfamily of chromatin remodeling factors that plays a global role in RNA polymerase II transcription. Despite the significant overlap between the distributions of KIS-L and Ash1 (the histone methyltransferase responsible for H3K4me) on fly polytene chromosomes, KIS-L did not bind methylated histone tails *in vitro*, and loss of Ash1 function did not alter the association of KIS-L with chromatin. By contrast, loss of *kis* function led to a dramatic reduction in the levels of Ash1 associated with chromatin and was accompanied by an increased methylation of histone H3 at K27 (H3K27me), a histone mark required for transcriptional repression by Polycomb group proteins. These data suggest that KIS-L could counteract Polycomb repression by recruiting the Ash1 histone methyltransferases to chromatin [54].

In addition to the CHD subfamily of chromatin remodeler, factors containing the plant homeodomain (PHD) finger domain, such as ING2 and BPTF, directly associate with dimethylated and trimethylated histone H3 on K4 (H3K4me2 and H3K4me3) [55]. In particular, the ING2 PHD domain binds with high-affinity H3K4me2 or H3K4me3, promoting a more stable binding of the mSin3A/HDAC1 complex on proliferation gene promoters [56]. On the other hand, studies conducted in *S. cerevisiae* have shown that the trimethylation of histone H3 at K36 (H3K36me3) is another recognition site for other PHD finger containing factors (ING2 family members) highlighting a general function for this domain to transduce lysine methylation signals [57].

PHD fingers are also present in BPTF, the largest subunit of the human ISWI-containing complex hNURF [58]. BPTF preferentially associates with H3K4me3 chromatin and loss of H3K4me3 causes partial release of BPTF from chromatin with consequent defective recruitment of the associated ATPase SNF2L at HOX gene promoters [59]. Interestingly, loss of BPTF in Xenopus embryos compromises spatial control of HOX gene expression, suggesting that NURF-mediated ATP-dependent chromatin remodeling is directly coupled to H3K4me3 to maintain HOX gene expression patterns during development [59].

Furthermore, *Drosophila* NURF301 presents different isoforms associated in distinct NURF chromatin remodeling complexes. Full-length NURF301 contains a C-terminal bromodomain that binds H4K16Ac and a juxtaposed PHD finger binding H3K4me3. The NURF301 isoform lacking the C-terminal bromodomain and PHD finger can still assemble into a multisubunit complex. However, NURF remodeling complex deficient in H3K4me3 and H4K16Ac recognition modules causes germ-line-specific defects [32].

In addition to H3K4me3, the yeast Isw1 ATPase also recognizes H3K4me2 on chromatin [60]. These histone methylations catalyzed by

![Figure 5](attachment:Figure_5.png)

**Fig. 5.** Chromatin remodeling regulation by SAM. Schematic representation of nucleosome remodeling reactions regulation by histone and DNA methylations.
the Set1p HMT mediates Isw1 binding on chromatin and the subsequent chromatin remodeling necessary for the correct distribution of RNA Polymerase II over the coding region of genes involved in the methionine biosynthetic pathway [60].

The Ash1 histone methyltransferase methylates histone H3 at K4 and K9 (H3K4me, H3Kme9) as well as histone H4 at K20 (H4K20me). Histone methylation catalyzed by Ash1 represents a specific signal for the establishment of epigenetic active transcription patterns [61]. Indeed, transcriptional activation by Ash1 coincides with methylation of these three lysine residues at the promoter of Ash1 target genes [61]. The methylation pattern established by Ash1 recruits the BRM chromatin remodeling complex and at the same time inhibits the interaction of transcriptional repressor, leading to the *Ultrabithorax* gene transcriptional activation in flies [61]. Recent findings have also shown the importance of histone methylation in myogenesis control. During myogenesis Carm1/PRMT4, a protein arginine methyltransferases, responsible for histone H3 dimethylation at R17 (H3R17me2), facilitates SWI/SNF chromatin remodeling of late myogenesis genes [62].

In addition to histones, chromatin can also be methylated in its DNA component by DNA methyltransferase enzymes (DNMTs). In vertebrates DNA methylation is a post-replicative modification occurring exclusively at the C5 position of cytosine residues in the context of CpG dinucleotides [63]. Proper DNA methylation is necessary for normal development and generally correlates with gene repression, X-chromosome inactivation, imprinting, and carcinogenesis [64]. Recent studies have shown that DNA methylation is a chromatin mark with many cross-talks with other epigenetic pathways [65]. For example, the human chromatin remodeler SNF2H co-immunoprecipitates with the DNMT3B enzyme, probably indicating a direct functional connection between DNMTs and the ISWI family of chromatin remodeling enzymes [66]. The interaction between DNA methylation and ATP-dependent chromatin remodelers is also mediated by methyl-CpG binding domain proteins (MBDs) [67]. The hBRM and BRG1 localization at methylated genes is mediated by MeCP2, an MBD protein [68–70] and the Mi-2/NuRD chromatin remodeling complex is part of multiprotein complexes containing both MBD2 and MBD3 [71,72]. These data strongly indicate that methylated CpG dinucleotides can be recruiting elements for chromatin remodelers through MBD proteins.

## 5. Inositol polyphosphates

Inositol polyphosphates (IPs) have been shown to modulate several ATP-dependent chromatin remodeling complexes (Fig. 6a). Indications for a direct role of IPs in regulating chromatin remodeling reactions come from a variety of in vivo data. In yeast mutations in genes encoding for polyphosphate kinases responsible for IP₄, IP₅, and IP₆ biogenesis affect the inositol-1-phosphate synthase (*INO1*) gene transcription [73]. Moreover, mutations in the *ARG82/IPK2* gene, encoding for a nuclear inositol polyphosphate kinase, leads to impaired remodeling of *PHO5* promoter associated to inefficient recruitment of SWI/SNF and INO80 complexes on the promoter, suggesting a role of IPs in transcriptional control [74,75].

In vitro data have shown that NURF-, ISW2-, and INO80-stimulated nucleosome mobilization is inhibited by inositol hexakisphosphate (IP₆). On the contrary, inositol tetrakisphosphate (IP₄) and inositol pentakisphosphate (IP₅) stimulate nucleosome mobilization catalyzed by SWI/SNF complex [73]. Interestingly, the phosphatidylinositol 4,5-bisphosphate (PIP₂) has been shown to control the in vitro localization of the BAF complex on chromatin but has no effects on SNF2L containing chromatin remodeling complexes [76]. In addition it has been shown that mammalian BAF complex is able to bind PIP2 micelles and PIP2-containing mixed lipid vesicles. This association of BAF with PIP2 mediates an interaction with actin filaments, which in turn contact at least two distinct domains of BRG1 C-terminal portion [77].

At day the existence of specific nuclear receptors for IPs is still unclear. However, there are some data indicating that the PHD finger

containing protein ING2, a candidate tumor suppressor protein, could be a nuclear IP receptor. Indeed, the PHD fingers of ING2 bind both in vitro and in vivo IP₅ and this interaction influences ING2 ability to regulate p53 activation and p53-dependent apoptotic pathways [78].

## 6. NAD and PARylation

Poly-ADP-ribosylation (PARylation) is a post-translational modification of proteins catalyzed by the poly-ADP-ribose polymerases (PARPs), a family of abundant and ubiquitous nuclear enzymes that transfer and polymerize ADP-ribose units from NAD+ on a variety of nuclear proteins to form a branched polymer known as poly-ADP-ribose (PAR) [79]. PAR metabolism is involved in a wide range of biological processes, such as maintenance of genome stability, transcriptional regulation, energy metabolism, DNA repair, and programmed cell death [80]. In vitro, the main target of PARylation seems to be the PARP enzyme itself, even if in vivo H1 and H2B histones are the main substrate for this post-translational modification [79,81,82]. It has been also shown that non-covalent interactions between histones and PAR polymers could affect chromatin dynamics by blocking chromatin accessibility of remodeling factors [83]. Indeed, a number of chromatin proteins possess PAR-binding domains [84,85].

Recent data strongly indicate that PARP may play a direct role in regulating chromatin remodeling (Fig. 6b). Indeed, the *Drosophila* ATPase ISWI is directly regulated by covalent PARylation. PARylated ISWI has reduced nucleosome binding and ATPase activity [86,87]. On the other hand, amplified in liver cancer 1 (Alc1), a member of the SNF2 ATPase superfamily encoded by an oncogene implicated in the pathogenesis of hepatocellular carcinoma, is strongly activated by PARylated PARP and by free PAR [88].

## 7. Conclusions

ATP-dependent chromatin remodeling enzymes play a critical role in making chromatin dynamic and have been implicated in the control of RNA transcription, chromosome organization, DNA replication, and damage response. However, little is known about how remodelers activity is regulated and targeted to specific biological processes and how their function is integrated in the complex network of covalent modifications of chromatin. Recent work highlighted that ATP-dependent chromatin remodeling complexes are regulated by small molecules and metabolites such as ATP, AcCoA, SAM, NAD, and IPs. While remodelers and their regulators are essential modulators of many nuclear reactions, they may lead to human diseases when their function is altered.

The identification and characterization of chromatin remodeling regulators will provide important support for the development of novel therapeutic approaches to cure or ameliorate a variety of human disorders. The recent spurt in the understanding of the role of epigenetics in cellular physiology and its link to disease and therapeutics is leading to the discovery of several small molecule modulators of chromatin enzymatic activities. The possibility of acting on epigenetic pathways is obviously of the highest interest for gene therapy applications and conceptually alternative to gene replacement. In fact, thanks to its dynamic nature, the Epigenome offers the chance to modulate gene activity and underlying gene networks by tackling specific enzymatic activities via drugs and small molecules opening novel therapeutic potentially revolutionary avenues for the cure of human genetic diseases.

## Acknowledgment

We would like to thank Giovanni Spinelli, Fabrizio Gianguzzo, Aldo Di Leonardo, Vincenzo Cavalieri, and Anna Sala for their critical reading and feedbacks on this manuscript. We would also like to apologize with all our colleagues whose work was not properly cited due to space restriction. G.B. is supported by an FIRC fellowship. M.C.
G. Burgio et al. / Biochimica et Biophysica Acta 1799 (2010) 671–680

**Fig. 6. Chromatin remodeling regulation by inositol polyphosphates and NAD. Schematic representation of nucleosome remodeling reactions regulation by (a) IPs and (b) poly-ADP-ribose.**

### References

[1] E.I. Campos, D. Reinberg, Histones: annotating chromatin, Annu. Rev. Genet. 43 (2009) 559–599.

[2] C.R. Clapier, B.R. Cairns, The biology of chromatin remodeling complexes, Annu. Rev. Biochem. 78 (2009) 273–304.

[3] C. Hogan, P. Varga-Weisz, The regulation of ATP-dependent nucleosome remodeling factors, Mutat. Res. 618 (2007) 41–51.

[4] K.G. Tanner, R.C. Trievel, M.H. Kuo, R.M. Howard, S.L. Berger, C.D. Allis, R. Marmorstein, J.M. Denu, Catalytic mechanism and function of invariant glutamic acid 173 from the histone acetyltransferase GCN5 transcriptional coactivator, J. Biol. Chem. 274 (1999) 18157–18160.

[5] M.H. Kuo, C.D. Allis, Roles of histone acetyltransferases and deacetylases in gene regulation, Bioessays 20 (1998) 615–626.

[6] B. Bourachot, M. Yaniv, C. Muchardt, Growth inhibition by the mammalian SWI-SNF subunit Brm is regulated by acetylation, EMBO J. 22 (2003) 6505–6515.

[7] S. Glaros, G.M. Cirrincione, C. Muchardt, C.G. Kleer, C.W. Michael, D. Reisman, The reversible epigenetic silencing of BRM: implications for clinical targeted therapy, Oncogene 26 (2007) 7058–7066.

[8] P.A. Khavari, C.L. Peterson, J.W. Tamkun, D.B. Mendel, G.R. Crabtree, BRG1 contains a conserved domain of the SWI2/SNF2 family necessary for normal mitotic growth and transcription, Nature 366 (1993) 170–174.

[9] C. Underhill, M.S. Qutob, S.P. Yee, J. Torchia, A novel nuclear receptor corepressor complex, N-CoR, contains components of the mammalian SWI/SNF complex and the corepressor KAP-1, J. Biol. Chem. 275 (2000) 40463–40470.

[10] R. Ferreira, A. Eberharter, T. Bonaldi, M. Chioda, A. Imhof, P.B. Becker, Site-specific acetylation of ISWI by GCN5, BMC Mol. Biol. 8 (2007) 73.

[11] D.F. Corona, J.W. Tamkun, Multiple roles for ISWI in transcription, chromosome organization and DNA replication, Biochim. Biophys. Acta 1677 (2004) 113–119.

[12] M. Kasten, H. Szerlong, H. Erdjument-Bromage, P. Tempst, M. Werner, B.R. Cairns, Tandem bromodomains in the chromatin remodeler RSC recognize acetylated histone H3 Lys14, EMBO J. 23 (2004) 1348–1359.

[13] A.P. VanDemark, M.M. Kasten, E. Ferris, A. Heroux, C.P. Hill, B.R. Cairns, Autoregulation of the rsc4 tandem bromodomain by gcn5 acetylation, Mol. Cell 27 (2007) 817–828.

[14] R. Strohner, A. Nemeth, P. Jansa, U. Hofmann-Rohrer, R. Santoro, G. Langst, I. Grummt, NoRC—a novel member of mammalian ISWI-containing chromatin remodeling machines, EMBO J. 20 (2001) 4892–4900.

[15] Y. Zhou, K.M. Schmitz, C. Mayer, X. Yuan, A. Akhtar, I. Grummt, Reversible acetylation of the chromatin remodeling complex NoRC is required for noncoding RNA-dependent silencing, Nat. Cell Biol. 11 (2009) 1010–1016.
[16] A. Imhof, P.B. Becker, Modifications of the histone N-terminal domains. Evidence for an “epigenetic code”? Mol. Biotechnol. 17 (2001) 1–13.
[17] T. Jenuwein, C.D. Allis, Translating the histone code, Science (New York, N.Y.) 293 (2001) 1074–1080.
[18] B.M. Turner, Histone acetylation and an epigenetic code, Bioessays 22 (2000) 836–845.
[19] A.H. Hassan, K.E. Neely, J.L. Workman, Histone acetyltransferase complexes stabilize swi/snf binding to promoter nucleosomes, Cell 104 (2001) 817–827.
[20] P. Syntichaki, I. Topalidou, G. Thireos, The Gcn5 bromodomain co-ordinates nucleosome remodelling, Nature 404 (2000) 414–417.
[21] T. Agalioti, S. Lomvardas, B. Parekh, J. Yie, T. Maniatis, D. Thanos, Ordered recruitment of chromatin modifying and general transcription factors to the IFN-beta promoter, Cell 103 (2000) 667–678.
[22] T. Agalioti, G. Chen, D. Thanos, Deciphering the transcriptional histone acetylation code for a human gene, Cell 111 (2002) 381–392.
[23] G.P. Vicent, R. Zaurin, C. Ballare, A.S. Nacht, M. Beato, Erk signaling and chromatin remodeling in MMTV promoter activation by progestins, Nucl. Recept. Signal. 7 (2009) e008.
[24] G. Skiniotis, D. Moazed, T. Walz, Acetylated histone tail peptides induce structural rearrangements in the RSC chromatin remodeling complex, J. Biol. Chem. 282 (2007) 20804–20808.
[25] N.J. Krogan, M.C. Keogh, N. Datta, C. Sawa, O.W. Ryan, H. Ding, R.A. Haw, J. Pootoolal, A. Tong, V. Canadien, D.P. Richards, X. Wu, A. Emili, T.R. Hughes, S. Buratowski, J.F. Greenblatt, A Snf2 family ATPase complex required for recruitment of the histone H2A variant Htz1, Mol. Cell 12 (2003) 1565–1576.
[26] A. Hilfiker, D. Hilfiker-Kleiner, A. Pannuti, J.C. Lucchesi, Mof, a putative acetyl transferase gene related to the Tip60 and MOZ human genes and to the SAS genes of yeast, is required for dosage compensation in Drosophila, EMBO J. 16 (1997) 2054–2060.
[27] N. Suka, K. Luo, M. Grunstein, Sir2p and Sas2p opposingly regulate acetylation of yeast histone H4 lysine16 and spreading of heterochromatin, Nat. Genet. 32 (2002) 378–383.
[28] Y. Park, M.I. Kuroda, Epigenetic aspects of X-chromosome dosage compensation, Science (New York, N.Y.) 293 (2001) 1083–1085.
[29] E.R. Smith, C.D. Allis, J.C. Lucchesi, Linking global histone acetylation to the transcription enhancement of X-chromosomal genes in Drosophila males, J. Biol. Chem. 276 (2001) 31483–31486.
[30] D.F. Corona, C.R. Clapier, P.B. Becker, J.W. Tamkun, Modulation of ISWI function by site-specific histone acetylation, EMBO Rep. 3 (2002) 242–247.
[31] M. Shogren-Knaak, H. Ishii, J.M. Sun, M.J. Pazin, J.R. Davie, C.L. Peterson, Histone H4–K16 acetylation controls chromatin structure and protein interactions, Science (New York, N.Y.) 311 (2006) 844–847.
[32] S.Y. Kwon, H. Xiao, C. Wu, P. Badenhorst, Alternative splicing of NURF301 generates distinct NURF chromatin remodeling complexes with altered modified histone binding specificities, PLoS Genet. 5 (2009) e1000574.
[33] Y. Zhou, I. Grummt, The PHD finger/bromodomain of NoRC interacts with acetylated histone H4K16 and is sufficient for rDNA silencing, Curr. Biol. 15 (2005) 1434–1438.
[34] R.W. King, P.K. Jackson, M.W. Kirschner, Mitosis in transition, Cell 79 (1994) 563–571.
[35] F.M. Davis, T.Y. Tsao, S.K. Fowler, P.N. Rao, Monoclonal antibodies to mitotic cells, Proc. Natl Acad. Sci. U. S. A. 80 (1983) 2926–2930.
[36] P.T. Stukenberg, K.D. Lustig, T.J. McGarry, R.W. King, J. Kuang, M.W. Kirschner, Systematic identification of mitotic phosphoproteins, Curr. Biol. 7 (1997) 338–348.
[37] C. Muchardt, J.C. Reyes, B. Bourachot, E. Leguoy, M. Yaniv, The hbrm and BRG-1 proteins, components of the human SNF/SWI complex, are phosphorylated and excluded from the condensed chromosomes during mitosis, EMBO J. 15 (1996) 3394–3402.
[38] S. Sif, P.T. Stukenberg, M.W. Kirschner, R.E. Kingston, Mitotic inactivation of a human SWI/SNF chromatin remodeling complex, Genes Dev. 12 (1998) 2842–2851.
[39] K. Bouazoune, A. Brehm, dMi-2 chromatin binding and remodeling activities are regulated by dCK2 phosphorylation, J. Biol. Chem. 280 (2005) 41912–41920.
[40] H. Oya, A. Yokoyama, I. Yamaoka, R. Fujiki, M. Yonezawa, M.Y. Youn, I. Takada, S. Kato, H. Kitagawa, Phosphorylation of Williams syndrome transcription factor by MAPK induces a switching between two distinct chromatin remodeling complexes, J. Biol. Chem. 284 (2009) 32472–32482.
[41] D.E. MacCallum, A. Losada, R. Kobayashi, T. Hirano, ISWI remodeling complexes in Xenopus egg extracts: identification as major chromosomal components that are regulated by INCENP-aurora B, Mol. Biol. Cell 13 (2002) 25–39.
[42] G.P. Vicent, C. Ballare, A.S. Nacht, J. Clausell, A. Subtil-Rodriguez, I. Quiles, A. Jordan, M. Beato, Induction of progesterone target genes requires activation of Erk and Msk kinases and phosphorylation of histone H3, Mol. Cell 24 (2006) 367–381.
[43] A.J. Morrison, J.A. Kim, M.D. Person, J. Highland, J. Xiao, T.S. Wehr, S. Hensley, Y. Bao, J. Shen, S.R. Collins, J.S. Weissman, J. Delrow, N.J. Krogan, J.E. Haber, X. Shen, Mec1/Tel1 phosphorylation of the INO80 chromatin remodeling complex influences DNA damage checkpoint responses, Cell 130 (2007) 499–511.
[44] A.J. Morrison, J. Highland, N.J. Krogan, A. Arbel-Eden, J.F. Greenblatt, J.E. Haber, X. Shen, INO80 and gamma-H2AX interaction links ATP-dependent chromatin remodeling to DNA damage repair, Cell 119 (2004) 767–775.
[45] H. van Attikum, O. Fritsch, B. Hohn, S.M. Gasser, Recruitment of the INO80 complex by H2A phosphorylation links ATP-dependent chromatin remodeling with DNA double-strand break repair, Cell 119 (2004) 777–788.

[46] S.C. Dillon, X. Zhang, R.C. Trievel, X. Cheng, The SET-domain protein superfamily: protein lysine methyltransferases, Genome Biol. 6 (2005) 227.
[47] C. Martin, Y. Zhang, The diverse functions of histone lysine methylation, Nat. Rev. 6 (2005) 838–849.
[48] S. Pal, S. Sif, Interplay between chromatin remodelers and protein arginine methyltransferases, J. Cell. Physiol. 213 (2007) 306–315.
[49] T. Kouzarides, Chromatin modifications and their function, Cell 128 (2007) 693–705.
[50] A. Shilatifard, Chromatin modifications by methylation and ubiquitination: implications in the regulation of gene expression, Annu. Rev. Biochem. 75 (2006) 243–269.
[51] R.J. Sims III, C.F. Chen, H. Santos-Rosa, T. Kouzarides, S.S. Patel, D. Reinberg, Human but not yeast CHD1 binds directly and selectively to histone H3 methylated at lysine 4 via its tandem chromodomains, J. Biol. Chem. 280 (2005) 41789–41792.
[52] M.G. Pray-Grant, J.A. Daniel, D. Schieltz, J.R. Yates III, P.A. Grant, Chd1 chromodomain links histone H3 methylation with SAGA- and SLIK-dependent acetylation, Nature 433 (2005) 434–438.
[53] J.F. Flanagan, L.Z. Mi, M. Chruszcz, M. Cymborowski, K.L. Clines, Y. Kim, W. Minor, F. Rastinejad, S. Khorasanizadeh, Double chromodomains cooperate to recognize the methylated histone H3 tail, Nature 438 (2005) 1181–1185.
[54] S. Srinivasan, K.M. Dorighi, J.W. Tamkun, Drosophila Kismet regulates histone H3 lysine 27 methylation and early elongation by RNA polymerase II, PLoS Genet. 4 (2008) e1000217.
[55] A. Shilatifard, Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation, Curr. Opin. Cell Biol. 20 (2008) 341–348.
[56] X. Shi, T. Hong, K.L. Walter, M. Ewalt, E. Michishita, T. Hung, D. Carney, P. Pena, F. Lan, M.R. Kaadige, N. Lacoste, C. Cayrou, F. Davrazou, A. Saha, B.R. Cairns, D.E. Ayer, T.G. Kutateladze, Y. Shi, J. Cote, K.F. Chua, O. Gozani, ING2 PHD domain links histone H3 lysine 4 methylation to active gene repression, Nature 442 (2006) 96–99.
[57] X. Shi, I. Kachirskaia, K.L. Walter, J.H. Kuo, A. Lake, F. Davrazou, S.M. Chan, D.G. Martin, I.M. Fingerman, S.D. Briggs, L. Howe, P.J. Utz, T.G. Kutateladze, A.A. Lugovskoy, M.T. Bedford, O. Gozani, Proteome-wide analysis in Saccharomyces cerevisiae identifies several PHD fingers as novel direct and selective binding modules of histone H3 methylated at either lysine 4 or lysine 36, J. Biol. Chem. 282 (2007) 2450–2455.
[58] M. Bienz, The PHD finger, a nuclear protein-interaction domain, Trends Biochem. Sci. 31 (2006) 35–40.
[59] J. Wysocka, T. Swigut, H. Xiao, T.A. Milne, S.Y. Kwon, J. Landry, M. Kauer, A.J. Tackett, B.T. Chait, P. Badenhorst, C. Wu, C.D. Allis, A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling, Nature 442 (2006) 86–90.
[60] H. Santos-Rosa, R. Schneider, B.E. Bernstein, N. Karabetsou, A. Morillon, C. Weise, S.L. Schreiber, J. Mellor, T. Kouzarides, Methylation of histone H3 K4 mediates association of the Isw1p ATPase with chromatin, Mol. Cell 12 (2003) 1325–1332.
[61] C. Beisel, A. Imhof, J. Greene, E. Kremmer, F. Sauer, Histone methylation by the Drosophila epigenetic transcriptional regulator Ash1, Nature 419 (2002) 857–862.
[62] C.S. Dacwag, M.T. Bedford, S. Sif, A.N. Imbalzano, Distinct protein arginine methyltransferases promote ATP-dependent chromatin remodeling function at different stages of skeletal muscle differentiation, Mol. Cell. Biol. 29 (2009) 1909–1921.
[63] Y. Gruenbaum, R. Stein, H. Cedar, A. Razin, Methylation of CpG sequences in eukaryotic DNA, FEBS Lett. 124 (1981) 67–71.
[64] A. Bird, DNA methylation patterns and epigenetic memory, Genes Dev. 16 (2002) 6–21.
[65] A. Rottach, H. Leonhardt, F. Spada, DNA methylation-mediated epigenetic control, J. Cell. Biochem. 108 (2009) 43–51.
[66] T.M. Geiman, U.T. Sankpal, A.K. Robertson, Y. Zhao, Y. Zhao, K.D. Robertson, DNMT3B interacts with hSNF2H chromatin remodeling enzyme, HDACs 1 and 2, and components of the histone methylation system, Biochem. Biophys. Res. Commun. 318 (2004) 544–555.
[67] O. Bogdanovic, G.J. Veenstra, DNA methylation and methyl-CpG binding proteins: developmental requirements and function, Chromosoma 118 (2009) 549–565.
[68] K.N. Harikrishnan, M.Z. Chow, E.K. Baker, S. Pal, S. Bassal, D. Brasacchio, L. Wang, J. M. Craig, P.L. Jones, S. Sif, A. El-Osta, Brahma links the SWI/SNF chromatin-remodeling complex with MeCP2-dependent transcriptional silencing, Nat. Genet. 37 (2005) 254–264.
[69] K. Hu, X. Nan, A. Bird, W. Wang, Testing for association between MeCP2 and the brahma-associated SWI/SNF chromatin-remodeling complex, Nat. Genet. 38 (2006) 962–964 author reply 964–967.
[70] K.Y. Song, C.S. Kim, C.K. Hwang, H.S. Choi, P.Y. Law, L.N. Wei, H.H. Loh, uAUG-mediated translational initiations are responsible for human mu opioid receptor gene expression, J. Cell. Mol. Med. (2009).
[71] X. Le Guezennec, M. Vermeulen, A.B. Brinkman, W.A. Hoeijmakers, A. Cohen, E. Lasonder, H.G. Stunnenberg, MBD2/NuRD and MBD3/NuRD, two distinct complexes with different biochemical and functional properties, Mol. Cell. Biol. 26 (2006) 843–851.
[72] Y. Zhang, H.H. Ng, H. Erdjument-Bromage, P. Tempst, A. Bird, D. Reinberg, Analysis of the NuRD subunits reveals a histone deacetylase core complex and a connection with DNA methylation, Genes Dev. 13 (1999) 1924–1935.
[73] X. Shen, H. Xiao, R. Ranallo, W.H. Wu, C. Wu, Modulation of ATP-dependent chromatin-remodeling complexes by inositol polyphosphates, Science (New York, N.Y.) 299 (2003) 112–114.
[74] D.J. Steger, E.S. Haswell, A.L. Miller, S.R. Wente, E.K. O'Shea, Regulation of chromatin remodeling by inositol polyphosphates, Science (New York, N.Y.) 299 (2003) 114–116.
[75] A.R. Odom, A. Stahlberg, S.R. Wente, J.D. York, A role for nuclear inositol 1, 4, 5-trisphosphate kinase in transcriptional control, Science (New York, N.Y.) 287 (2000) 2026–2029.

[76] K. Zhao, W. Wang, O.J. Rando, Y. Xue, K. Swiderek, A. Kuo, G.R. Crabtree, Rapid and phosphoinositol-dependent binding of the SWI/SNF-like BAF complex to chromatin after T lymphocyte receptor signaling, Cell 95 (1998) 625–636.

[77] O.J. Rando, K. Zhao, P. Janmey, G.R. Crabtree, Phosphatidylinositol-dependent actin filament binding by the SWI/SNF-like BAF chromatin remodeling complex, Proc. Natl Acad. Sci. U. S. A. 99 (2002) 2824–2829.

[78] O. Gozani, P. Karuman, D.R. Jones, D. Ivanov, J. Cha, A.A. Lugovskoy, C.L. Baird, H. Zhu, S.J. Field, S.L. Lessnick, J. Villasenor, B. Mehrrotra, J. Chen, V.R. Rao, J.S. Brugge, C.G. Ferguson, B. Payrastre, D.G. Myszka, L.C. Cantley, G. Wagner, N. Divecha, G.D. Prestwich, J. Yuan, The PHD finger of the chromatin-associated protein ING2 functions as a nuclear phosphoinositide receptor, Cell 114 (2003) 99–111.

[79] D. D'Amours, S. Desnoyers, I. D'Silva, G.G. Poirier, Poly(ADP-ribosyl)ation reactions in the regulation of nuclear functions, Biochem. J. 342 (Pt 2) (1999) 249–268.

[80] V. Schreiber, F. Dantzer, J.C. Ame, G. de Murcia, Poly(ADP-ribose): novel functions for an old molecule, Nat. Rev. 7 (2006) 517–528.

[81] P. Adamietz, A. Rudolph, ADP-ribosylation of nuclear proteins in vivo. Identification of histone H2B as a major acceptor for mono- and poly(ADP-ribose) in

dimethyl sulfate-treated hepatoma AH 7974 cells, J. Biol. Chem. 259 (1984) 6841–6846.

[82] G. Krupitza, P. Cerutti, Poly(ADP-ribosylation) of histones in intact human keratinocytes, Biochemistry 28 (1989) 4054–4060.

[83] J. Ausio, D.W. Abbott, X. Wang, S.C. Moore, Histone variants and histone modifications: a structural perspective, Biochem. Cell Biol. 79 (2001) 693–708.

[84] D. Quenet, R. El Ramy, V. Schreiber, F. Dantzer, The role of poly(ADP-ribosyl)ation in epigenetic events, Int. J. Biochem. Cell Biol. 41 (2009) 60–65.

[85] A. Reale, G.D. Matteis, G. Galleazzi, M. Zampieri, P. Caiafa, Modulation of DNMT1 activity by ADP-ribose polymers, Oncogene 24 (2005) 13–19.

[86] A. Sala, D.F. Corona, Poly-ADP-ribose (PAR) as an epigenetic flag, Epigenetics 4 (2009) 287–290.

[87] A. Sala, G. La Rocca, G. Burgio, E. Kotova, D. Di Gesu, M. Collesano, A.M. Ingrassia, A. V. Tulin, D.F. Corona, The nucleosome-remodeling ATPase ISWI is regulated by poly-ADP-ribosylation, PLoS Biol. 6 (2008) e252.

[88] A.J. Gottschalk, G. Timinszky, S.E. Kong, J. Jin, Y. Cai, S.K. Swanson, M.P. Washburn, L. Florens, A.G. Ladurner, J.W. Conaway, R.C. Conaway, Poly(ADP-ribosyl)ation directs recruitment and activation of an ATP-dependent chromatin remodeler, Proc. Natl Acad. Sci. U. S. A. 106 (2009) 13770–13774.
