
RESEARCH LETTER

# Quantitative proteomics of Chlorobaculum tepidum: insights into the sulfur metabolism of a phototrophic green sulfur bacterium

Lasse G. Falkenby¹, Monika Szymanska¹, Carina Holkenbrink², Kirsten S. Habicht³, Jens S. Andersen¹, Mette Miller¹ & Niels-Ulrik Frigaard²

¹Department of Biochemistry and Molecular Biology, University of Southern Denmark, Odense, Denmark; ²Department of Biology, University of Copenhagen, Helsingør, Denmark; and ³Institute of Biology, University of Southern Denmark, Odense, Denmark

---

**Correspondence:** Niels-Ulrik Frigaard, Section for Marine Biology, Department of Biology, University of Copenhagen, Strandpromenaden 5, DK-3000 Helsingør, Denmark. Tel.: +45 35 32 19 57; fax: +45 35 32 19 51; e-mail: nuf@bio.ku.dk

Received 21 May 2011; revised 19 July 2011; accepted 25 July 2011. Final version published online 25 August 2011.

DOI: 10.1111/j.1574-6968.2011.02370.x

Editor: Christiane Dahl

**Keywords**
quantitative proteomics; photosynthetic bacteria; sulfur metabolism; dissimilatory sulfite reductase; SOX enzyme system; sulfite oxidation.

---

## Abstract

Chlorobaculum (*Cba.*) *tepidum* is a green sulfur bacterium that oxidizes sulfide, elemental sulfur, and thiosulfate for photosynthetic growth. To gain insight into the sulfur metabolism, the proteome of *Cba.* *tepidum* cells sampled under different growth conditions has been quantified using a rapid gel-free, filter-aided sample preparation (FASP) protocol with an in-solution isotopic labeling strategy. Among the 2245 proteins predicted from the *Cba.* *tepidum* genome, approximately 970 proteins were detected in unlabeled samples, whereas approximately 630–640 proteins were detected in labeled samples comparing two different growth conditions. Wild-type cells growing on thiosulfate had an increased abundance of periplasmic cytochrome *c*-555 and proteins of the periplasmic thiosulfate-oxidizing SOX enzyme system when compared with cells growing on sulfide. A *dsrM* mutant of *Cba.* *tepidum*, which lacks the dissimilatory sulfite reductase DsrM protein and therefore is unable to oxidize sulfur globules to sulfite, was also investigated. When compared with wild type, the *dsrM* cells exhibited an increased abundance of DSR enzymes involved in the initial steps of sulfur globule oxidation (DsrABCL) and a decreased abundance of enzymes putatively involved in sulfite oxidation (Sat-AprAB-QmoABC). The results show that *Cba.* *tepidum* regulates the cellular levels of enzymes involved in sulfur metabolism and other electron-transferring processes in response to the availability of reduced sulfur compounds.

---

## Introduction

Quantitative mass spectrometry-based proteomics has become widely used for examining differences in global expression level of proteins in various cellular states (Bantscheff *et al.*, 2007; Elliott *et al.*, 2009; Walther & Mann, 2010). In this method, proteins from samples obtained from different experimental conditions can be distinguished by incorporation of unique, stable isotopes with disparate masses in one of the samples. In this way, various samples can be combined and analyzed in a single LC-MS/MS analysis allowing estimation of the relative intensities of the peptides of interest from the labeled and unlabelled samples. Metabolic (Ong *et al.*, 2002) and chemical (Boersema *et al.*, 2009) labeling are two common procedures used for introducing heavy isotopes into cellular proteins. A pre-requisite for metabolic labeling of proteins is that the cells efficiently take up a labeled substrate in culture and incorporate it into proteins. However, this approach does not always result in a sufficient degree of labeling. Alternatively, as used in the present work, isotopic labeling can be performed by chemical labeling of peptides resulting from post-digestion of the cellular protein fractions.

The green sulfur bacterium (GSB) *Chlorobaculum* (*Cba.*) *tepidum* is a strictly anaerobic, photosynthetic

FEMS Microbiol Lett **323** (2011) 142–150

© 2011 Federation of European Microbiological Societies  
Published by Blackwell Publishing Ltd. All rights reserved

Quantitative proteomics of *Chlorobaculum tepidum*

bacterium that lives in anaerobic aquatic environments, where reduced sulfur compounds, predominantly sulfide and light occur at the same time (Wahlund *et al.*, 1991; Overmann, 2008). *Chlorobaculum tepidum* oxidizes sulfide, elemental sulfur, and thiosulfate for use as electron donor in its photosynthesis. The 2.15-Mbp genome of *Cba.* *tepidum* has been sequenced and revealed about 2245 protein-encoding genes (Eisen *et al.*, 2002). Currently, 15 genome sequences of GSB have been determined (Gregersen *et al.*, 2011). This information has allowed a detailed analysis of the sulfur metabolism of GSB, but many processes are still poorly described (Frigaard & Bryant, 2004, 2008; Frigaard & Dahl, 2009; Sakurai *et al.*, 2010). Table 1 lists 57 enzymes putatively involved in the oxidative sulfur metabolism of *Cba.* *tepidum*, some of which have been functionally investigated.

Figure 1 shows a simplified scheme of the pathways and enzymes of the oxidative sulfur metabolism of *Cba.* *tepidum*. Sulfide is oxidized by sulfide:quinone oxidoreductases (SQR; Chan *et al.*, 2009); additional unknown enzyme activity contributes to sulfide oxidation (Holkenbrink *et al.*, 2011). Thiosulfate is oxidized exclusively by the sulfur oxidation (SOX) enzyme system in the periplasm (Ogawa *et al.*, 2008, 2010; Azai *et al.*, 2009). Both of these processes give rise to a putative oligosulfide pool, which presumably is in equilibrium with an extracellular pool of sulfur globules that sometimes is referred to as ‘elemental sulfur’ (*S⁰*). Oxidation of the oligosulfide pool is dependent on the dissimilatory sulfite reductase (DSR) enzyme system (Holkenbrink *et al.*, 2011), which consists of at least 15 subunits (Table 1). The DSR system presumably forms intracellular sulfite that is oxidized by an enzyme system consisting of Sat, Apr, and Qmo proteins (Rodriguez *et al.*, 2011). The electron acceptors, cytochrome *c*, and menaquinone (Fig. 1) are ultimately oxidized by the photosynthetic reaction center.

In cultures of *Cba.* *tepidum* that contain both sulfide and thiosulfate, sulfide is oxidized preferentially while sulfur globules are formed (Chan *et al.*, 2008; Azai *et al.*, 2009; Holkenbrink *et al.*, 2011). Following sulfide depletion, thiosulfate and sulfur globules are oxidized to sulfate. The molecular mechanism of this phenomenon is poorly understood. Sulfide possibly inhibits thiosulfate oxidation either by substrate competition between sulfide and thiosulfate (the SOX system oxidizes sulfide *in vitro*; Ogawa *et al.*, 2010) or by saturation of the electron acceptor pool. Regulation of sulfur metabolism genes in GSB is poorly described, but it is known that SoxA is induced by thiosulfate in *Chlorobaculum thiosulfatiphilum* (Verté *et al.* 2002). In the purple sulfur bacterium, *Allochromatium vinosum*, *sox* and *dsr* genes are expressed at a low constitutive level in the absence of reduced sulfur

substrates and are induced by thiosulfate and sulfide, respectively (Grimm *et al.*, 2010, 2011).

# Materials and methods

## Cell cultures and chemical measurements

*Chlorobaculum tepidum* TLS was grown under incandescent illumination in CL medium (Frigaard *et al.*, 2004). For experiments comparing early and late exponential growth phase, wild-type cells were grown at 45 °C in 1-L flasks under a light intensity of 200 μmol photons m⁻² s⁻¹. For experiments comparing wild type and the *dsrM* mutant (Holkenbrink *et al.*, 2011), cells were grown at 42 °C in 15-mL tubes under a light intensity of 50 μmol photons m⁻² s⁻¹ and harvested in the late exponential growth phase. Cells were harvested by centrifugation and stored at –20 °C prior to analysis.

Bacteriochlorophyll *c* was determined by extracting the cell pellet with acetone : methanol (7 : 2 by vol) (Frigaard *et al.*, 1997). Sulfide was measured using the colorimetric methylene blue method (Cline 1969). Sulfate and thiosulfate were measured by ion chromatography (Dionex, Hvidovre, Denmark) using a carbonate buffer as eluent. Samples for analysis of elemental sulfur were dissolved in methanol and analyzed as S₈ by HPLC using a Sykam pump (S1100), UV–VIS detector (Sykam S3200), Zorbax ODS-column (125 × 4 mm, 5 μm; Knauer, Berlin, Germany) and methanol as the eluent at a flow rate of 1 mL min⁻¹. Elemental sulfur was detected at 265 nm.

## Protein preparation for proteome analysis

Cell pellets were thawed and extracted in acetone : methanol (7 : 2 by vol) to remove pigments. The colorless cell pellets were solubilized in an SDS-containing buffer (Laemmli, 1970) supplemented with a complete protease inhibitor cocktail (Roche, Hvidovre, Denmark) at 95 °C for 3–5 min and cleared by centrifugation. Prior to digestion, proteins were reduced with dithiothreitol (1 mM) and alkylated with iodoacetamide (5 mM).

The filter-aided sample preparation (FASP) protocol was used essentially as described (Wisniewski *et al.*, 2009). Protein extract (20 μL) was mixed with solution UA (200 μL; 8 M urea in H₂O, pH 8.5). This solution was loaded onto a 10-kDa cut-off filter spin filter and centrifuged (14 000 g, 40 min). The retentate was washed three times with solution UA and the flow-through discarded. Then a solution of iodoacetamide (100 μL; 0.05 M in-solution UA) was added to the filter and incubated for 5 min. The filters were then centrifuged (14 000 g, 30 min) and washed twice with a urea solution (100 μL; 8 M in H₂O, pH 8.0). After each wash, the filter units were centrifuged (14 000 g; 40 min).

Table 1. Gene products in Chlorobaculum tepidum known or suggested to be involved in oxidative sulfur metabolism*

| Locus | Name | Detected in unlabeled samples* | Detected in labeled samples* | References |
|-------|------|--------------------------------|-------------------------------|------------|
| CT0073 | Membrane-bound cytochrome c-555, protein CycB | Yes | Yes |  |
| CT0075 | Soluble cytochrome c-555, protein CycA (periplasmic) | Yes | Yes | Azai et al. (2009) |
| CT0117 | Sulfide:quinone oxidoreductase SqrD | Yes | Yes | Chan et al. (2009), Holkenbrink et al. (2011) |
| CT0494 | Polysulfide-reductase-like enzyme, subunit PsrC | Yes | No |  |
| CT0495 | Polysulfide-reductase-like enzyme, subunit PsrB | Yes | No |  |
| CT0496 | Polysulfide-reductase-like enzyme, subunit PsrA | Yes | No |  |
| CT0851 | Dissimilatory sulfite reductase protein DsrC, copy 1† | Yes | Yes | Holkenbrink et al. (2011) |
| CT0852 | Dissimilatory sulfite reductase protein DsrA, copy 1† | Yes | Yes | Holkenbrink et al. (2011) |
| CT0853 | Dissimilatory sulfite reductase protein DsrB, copy 1† | Yes | Yes | Holkenbrink et al. (2011) |
| CT0854 | Dissimilatory sulfite reductase protein DsrL, copy 1† | Yes | Yes | Holkenbrink et al. (2011) |
| CT0855 | Dissimilatory sulfite reductase protein DsrE | Yes | Yes |  |
| CT0856 | Dissimilatory sulfite reductase protein DsrF | Yes | Yes |  |
| CT0857 | Dissimilatory sulfite reductase protein DsrH | Yes | Yes |  |
| CT0862 | Sulfate adenyltransferase, protein Sat | Yes | Yes |  |
| CT0864 | Adenylylsulfate reductase, subunit AprB | Yes | Yes |  |
| CT0865 | Adenylylsulfate reductase, subunit AprA | Yes | Yes |  |
| CT0866 | Quinone-modifying oxidoreductase, subunit QmoA | Yes | Yes |  |
| CT0867 | Quinone-modifying oxidoreductase, subunit QmoB | Yes | Yes | Rodriguez et al. (2011) |
| CT0868 | Quinone-modifying oxidoreductase, subunit QmoC | Yes | Yes | Rodriguez et al. (2011) |
| CT0876 | Sulfide:quinone oxidoreductase SqrE§ | Yes | Yes | Chan et al. (2009), Gregersen et al. (2011) |
| CT1015 | Sulfur-oxidizing protein SoxJ | Yes | Yes | Ogawa et al. (2010), Gregersen et al. (2011) |
| CT1016 | Sulfur-oxidizing protein SoxX | Yes | Yes | Ogawa et al. (2008, 2010) |
| CT1017 | Sulfur-oxidizing protein SoxY | Yes | Yes | Ogawa et al. (2008, 2010) |
| CT1018 | Sulfur-oxidizing protein SoxZ | Yes | Yes | Ogawa et al. (2008, 2010) |
| CT1019 | Sulfur-oxidizing protein SoxA | Yes | Yes | Ogawa et al. (2008, 2010) |
| CT1020 | Sulfur-oxidizing protein SoxK | Yes | Yes | Ogawa et al. (2008, 2010) |
| CT1021 | Sulfur-oxidizing protein SoxB | Yes | Yes | Azai et al. (2009), Ogawa et al. (2008, 2010) |
| CT1023 | Sulfur-oxidizing protein SoxW | Yes | Yes |  |
| CT1025 | Sulfide dehydrogenase (SoxF/SqrC homolog) | Yes | Yes | Gregersen et al. (2011) |
| CT1087 | Sulfide:quinone oxidoreductase SqrF | No | No | Chan et al. (2009), Holkenbrink et al. (2011) |
| CT1245 | Heterodisulfide-reductase-like enzyme, subunit D | Yes | Yes |  |
| CT1246 | Heterodisulfide-reductase-like enzyme, subunit A | Yes | Yes |  |
| CT1247 | Heterodisulfide-reductase-like enzyme component | Yes | Yes |  |
| CT1248 | Heterodisulfide-reductase-like enzyme component | Yes | No |  |
| CT1249 | Heterodisulfide-reductase-like enzyme, subunit B | Yes | Yes |  |
| CT1250 | Heterodisulfide-reductase-like enzyme, subunit G | Yes | Yes |  |
| CT1772 | RuBisCO-like protein (RLP) | Yes | Yes | Hanson and Tabita (2001) |
| CT1891 | Sulfhydrogenase-like enzyme, subunit B | No | No |  |
| CT1892 | Sulfhydrogenase-like enzyme, subunit G | No | No |  |
| CT1893 | Sulfhydrogenase-like enzyme, subunit D | No | No |  |
| CT1894 | Sulfhydrogenase-like enzyme, subunit A | Yes | Yes |  |

Quantitative proteomics of Chlorobaculum tepidum

Table 1. Continued

| Locus    | Name                                                                 | Detected in unlabeled samples† | Detected in labeled samples† | References                |
|----------|----------------------------------------------------------------------|-------------------------------|-----------------------------|---------------------------|
| CT2080   | Flavocytochrome c, subunit FccA                                      | No                            | No                          |                           |
| CT2081   | Flavocytochrome c, subunit FccB                                      | No                            | No                          |                           |
| CT2238   | Dissimilatory sulfite reductase protein DsrW                         | Yes                           | No                          | Holkenbrink et al. (2011) |
| CT2239   | Dissimilatory sulfite reductase protein DsrV                         | No                            | No                          | Holkenbrink et al. (2011) |
| CT2240   | Dissimilatory sulfite reductase protein DsrP                         | No                            | No                          |                           |
| CT2241   | Dissimilatory sulfite reductase protein DsrO                         | Yes                           | Yes                         |                           |
| CT2242   | Dissimilatory sulfite reductase protein DsrJ                         | Yes                           | No                          |                           |
| CT2243   | Dissimilatory sulfite reductase protein DsrK                         | Yes                           | Yes                         |                           |
| CT2244   | Dissimilatory sulfite reductase protein DsrM                         | Yes                           | No                          | Holkenbrink et al. (2011) |
| CT2245   | Dissimilatory sulfite reductase protein DsrT                         | No                            | No                          | Holkenbrink et al. (2011) |
| CT2246   | Dissimilatory sulfite reductase protein DsrU                         | Yes                           | Yes                         | Holkenbrink et al. (2011) |
| CT2247   | Dissimilatory sulfite reductase protein DsrL, copy 2‡               | Yes                           | Yes                         | Holkenbrink et al. (2011) |
| CT2248   | Dissimilatory sulfite reductase protein DsrB, copy 2‡,§             | Yes                           | Yes                         | Holkenbrink et al. (2011) |
| CT2249   | Dissimilatory sulfite reductase protein DsrA, copy 2‡               | Yes                           | Yes                         | Holkenbrink et al. (2011) |
| CT2250   | Dissimilatory sulfite reductase protein DsrC, copy 2‡               | Yes                           | Yes                         | Holkenbrink et al. (2011) |
| CT2251   | Dissimilatory sulfite reductase protein DsrN                         | No                            | No                          |                           |

*Based on Eisen et al. (2002), Frigaard & Bryant (2008), and references in the table.

†This work.

‡The DsrA, DsrB, and DsrC proteins are each encoded by two nearly identical gene copies resulting in identical proteins, whereas two *dsrL* genes encode DsrL proteins with 99.2% amino acid sequence identity.

§SQR activity has not been demonstrated for this protein (Chan et al., 2009).

¶A frameshift mutation in gene CT2248 was reported by Eisen et al. (2002). However, our *Cba.* *tepidum* strain has a restored reading frame (Holkenbrink et al., 2011).


![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 1. Simplified overview of the predicted pathways and enzymes of the oxidative sulfur metabolism in *Chlorobaculum tepidum* (Frigaard & Bryant, 2008; Holkenbrink et al., 2011). Cyt c, cytochrome c; MK, menaquinone.


### Chemical labeling of peptides

Dimethyl labeling was performed essentially as described by Boersema et al. (2009). Briefly, the isolated proteins on the filter device were subjected to a Lys-C digestion. The resulting peptides were reconstituted in 100 mM TEAB buffer (Sigma, St. Louis, MO). Samples for ‘light’ labeling were mixed with formaldehyde (4% in H₂O; Sigma). Samples for ‘heavy’ labeling were mixed with formaldehyde-D₂ (4% in H₂O; Sigma). Both samples were then mixed with freshly prepared sodium cyanoborohydride (0.6 M). After incubation for 1 h at room temperature, the reaction was quenched with ammonia solution (1% v/v) and TFA. The acidified samples were desalted on StageTips made from C18 disks excised from Empore High Performance Extraction Disks (3M, St. Paul, MN) in a pipette tip (Rappsilber et al. 2007).


### Peptide purification and mass spectrometry

Peptide mixtures were separated by nanoLC using an Agilent 1200 nanoflow system connected to either an LTQ Orbitrap XL or LTQ FT Ultra mass spectrometer (both from Thermo Electron, Bremen, Germany) equipped with a nanoelectrospray ion source (Proxeon Biosystem, Odense, Denmark). Chromatographic separation of the peptides took place in an in-house packed 20 cm fused silica emitter (75-µm i.d.) with reverse-phase ReproSil-Pur C₁₈-AQ (3 µm) resin (Maisch GmbH, Ammerbuch-Entringen, Germany). Peptides were injected onto the column (flow rate 500 nL min⁻¹) and eluted with a flow of 250 nL min⁻¹ from 5% to 40% acetonitrile in 0.5% acetic acid over 2 h. A ‘top 6’ acquisition method was set up on the mass spectrometer, utilizing the high mass accuracy of the Orbitrap for intact peptides and the speed and sensitivity

of the LTQ (iontrap) for fragment spectra. The initial scan event was the intact peptide mass spectrum in the Orbitrap with range \( m/z \) 300–1800 and resolution \( R = 60\,000 \) at \( m/z \) 400. Six CID fragmentation spectra in the iontrap (AGC target 5000, maximum injection time 150 ms) of the six most intense ions from the Orbitrap scan were recorded. Dynamic exclusion (2.5 min) and charge state screening requiring charge 2+ or more were enabled.

The obtained tandem MS spectra were matched against theoretical spectra from a protein sequence database derived from the *Cba. tepidum* genome (GenBank acc. no. NC_002932) using Mascot (Matrix Science Ltd; www.matrixscience.com). Results were validated using the MSQUANT software package (Mortensen et al., 2010) requiring minimum two unique peptides per protein and minimum six amino acids per unique peptide. In silico analyses showed that a maximum 2159 of the 2245 proteins (96%) encoded by the *Cba. tepidum* genome are theoretically detectable using this approach. Nearly all theoretically undetectable proteins were small hypothetical proteins (<100 amino acid residues). All proteins listed in Table 1 were theoretically detectable. MSQUANT was used to make supervised quantitation of the identified proteins based on averaged peptide ratios. The relative standard deviation of averaged peptide ratios was 5–20% for most proteins; protein quantitations with higher than 30% relative standard deviation were discarded.

## Results and discussion

### Proteome analyses of unlabeled samples

About 970 proteins were routinely detected in unlabeled samples of *Cba. tepidum* cells prepared using FASP. This corresponds to about 43% of the 2245 proteins predicted by the genome sequence (Eisen et al., 2002). Table S1 (Supporting Information) compiles all the proteins detected in the present study. When the same cellular material was analyzed after separation into 10 fractions on 1-D SDS-PAGE, about 1230 proteins were detected (results not shown). Thus, the FASP method revealed almost 80% of the proteins detected with the more labor- and time-consuming gel-based method. In comparison, 1162 proteins were found in *Cba. tepidum* after sample preparation using capillary iso-electric focusing prior to MS analysis (Zhou et al., 2007).

Figure 2 shows the 970 detected proteins segregated according to functional category. The highest percentage of detection was obtained among proteins involved in translation and metabolism of carbohydrates, amino acids, and nucleotides (73–76%). The lowest percentage of detection was obtained among the poorly characterized proteins and hypothetical proteins (23%), probably

Carbohydrate transport and metabolism (76%)
Translation (74%)
Amino acid transport and metabolism (73%)
Nucleotide transport and metabolism (73%)
Posttranslational modification, chaperones (65%)
Energy production and conversion (58%)
Coenzyme transport and metabolism (55%)
Cell wall and membrane biogenesis (50%)
Intracellular trafficking and secretion (46%)
Lipid transport and metabolism (45%)
Cell cycle (45%)
Signal transduction (39%)
Transcription (37%)
Defense mechanisms (36%)
Inorganic ion transport and metabolism (36%)
Replication, recombination and repair (35%)
Poorly characterized (23%)

■ Predicted
□ Detected

Number of proteins

Fig. 2. Comparison of the number of proteins predicted by the genome sequence of *Chlorobaculum tepidum* (Eisen et al., 2002) and the number of proteins experimentally detected in the unlabeled FASP samples. The percentage of detected to predicted is shown by each category (COG functional categories). The number of predicted proteins in the category 'Poorly characterized' (which includes hypothetical proteins) is 943.

reflecting that some of the hypothetical proteins are not produced by the cell. A low percentage of protein detection was also observed in categories of DNA replication and transport and metabolism of inorganic ions (35–36%). Forty-four (77%) of the 57 proteins putatively involved in oxidative sulfur metabolism were detected (Table 1). The most active SQR (SqrD; Chan et al., 2009) and all SOX proteins (SoxJXYZAKBW) were detected, but the less active SQR (SqrF; Chan et al., 2009) and flavocytochrome c (FccAB) were not detected.

Technical difficulties with analyzing hydrophobic proteins could potentially introduce a bias against such proteins in the MS analysis (Bantscheff et al., 2007). Figure 3 shows the distribution of hydrophobicity calculated as the GRAVY score among the 2245 proteins predicted by the genome sequence and the proteins detected experimentally. The figure shows that significant bias against hydrophobic proteins in *Cba. tepidum* was only observed for proteins with GRAVY scores above 0.3. About 14% of all 2245 predicted proteins have GRAVY scores above 0.3. Membrane-bound transporters are highly represented in this group (Table S1).

### Quantitative proteome analyses of labeled samples

Stable isotope labeling of cellular proteins by adding labeled amino acids directly to cell cultures (SILAC) has

Quantitative proteomics of *Chlorobaculum tepidum*

successfully been used for quantification of proteins in numerous quantitative proteome studies (Ong *et al.*, 2002). However, experiments showed that addition of deuterated lysine to cultures of *Cba.* *tepidum* gave only about 10% labeling of the protein fraction (results not shown). Therefore, the post-cultivation chemical labeling approach described by Boersema *et al.* (2009) was used to analyze the *Cba.* *tepidum* proteome. The labeled peptides had a mass increase of 28 Da ('light labeling') or 32 Da ('heavy labeling'), per primary amine, when labeled with either formaldehyde or deuterated formaldehyde, respectively. Representative mass spectra of unlabeled and labeled preparations of the same peptide are shown in Fig. S1.

### Proteome comparison of early and late exponential growth phase

A sample was collected 4 h after inoculation, where the cells were in the early exponential growth phase and consuming sulfide. Sulfide was depleted at 10 h, after which the cells started consuming thiosulfate. A sample was then collected in the late exponential phase (43 h after inoculation) where the cells had consumed almost all thiosulfate (Fig. 4).

In total, 629 proteins of *Cba.* *tepidum* were identified and quantified in the MS analysis of the mixed early and late phase samples (Table S1; Fig. S2). The variation in protein abundance showed only a few extremes; only 7% of the proteins had abundance ratios larger than 2 or smaller than 0.5. Proteins with highly increased abundance in the late exponential phase (greater than a factor of 2) included cytochrome *c* (*CycA*), a photosynthetic reaction center component (*PscC*), and certain proteins involved in biosynthesis of bacteriochlorophylls (*BchE*, *BchT*, *BchP*, *HemA*). The latter can possibly be explained by the cells increasing their bacteriochlorophyll-to-protein ratio in the late exponential phase due to self-shading. Proteins with highly decreased abundance in the late exponential phase (less than a factor of 0.5) included certain ribosomal proteins and other proteins related to translation (*CT0011*, *CT0285*, *CT0240*, *CT1252*) consistent with stalling of growth.

Among the 57 proteins proposed to be involved in the oxidative sulfur metabolism of *Cba.* *tepidum* (Table 1), 35–37 proteins were identified and quantified. Figure 5a shows the relative abundance of these proteins grouped according to the position of their genes in the genome. It is evident in that the abundance of the sulfur metabolism enzymes is regulated. All SOX proteins (*SoxJXYZAKBW*) are more abundant in the late growth phase consistent with their function in thiosulfate oxidation. In fact, the similar increase in abundance of these eight Sox proteins

Fig. 3. Comparison of the hydrophobicity of predicted and detected proteins in *Chlorobaculum tepidum*. (a) Distribution of hydrophobicity among predicted and detected proteins in the unlabeled FASP samples. (b) Fraction of detected proteins to predicted proteins in the unlabeled (gray circles) and labeled (white circles) FASP samples. The hydrophobicity was calculated as grand average of hydropathy (GRAVY) scores (Kyte & Doolittle, 1982). The GRAVY score bin size was 0.1.

Fig. 4. Transformation of sulfur compounds in a 1-L culture of wild-type *Chlorobaculum tepidum*. Triangles and straight line, sulfide; circles and dotted line, elemental sulfur; squares and dashed line, thiosulfate; diamonds and dashed-dotted line, sulfate.

is consistent with the suggestion that the sox gene cluster (soxJXYZAKBW) is transcribed as a single transcript (Gregersen et al., 2011). There is also a drastic increase in the abundance of periplasmic cytochrome c (CycA) in the late growth phase, consistent with the dependence of the SOX system on this protein: The SOX system uses periplasmic cytochrome c as electron acceptor, whereas SQR (which is more active in the early growth phase) uses membrane-bound menaquinone as electron acceptor. Interestingly, there is little variation in the abundance of SqrD, which is in agreement with the observed constitutive expression of the sqrD gene (Chan et al., 2009). The abundance of the core enzyme of the DSR system, DsrAB, is not exceptionally different between early and late growth phase. However, the DsrEFH proteins are less abundant in the late growth phase, which could be due to the change in the sulfur substrates available to the cells. The DsrEFH proteins form a complex in Alc. vinosum that appears to be involved in cytoplasmic sulfur transfer in many sulfur-oxidizing bacteria (Dahl et al.,

There is a slight increase in the APR and Qmo proteins in the late growth phase consistent with the suggestion that these proteins are responsible for sulfite oxidation and thus sulfate production in Cba. tepidum (Rodriguez et al., 2011).

### Proteome comparison of wild type and dsrM mutant

The dsrM mutant strain lacks a functional DSR system and oxidizes sulfide and thiosulfate, but not sulfur globules (Holkenbrink et al., 2011). This mutant was constructed by transposon mutagenesis of the dsrM gene such that polar effects on adjacent genes are minimal (wild-type phenotype was shown to be restored by complementation of dsrM in trans; Holkenbrink et al., 2011). The cells were sampled in the late exponential growth phase, where thiosulfate and sulfur globules are available for oxidation. Figure 5b shows the relative expression of sulfur metabolism enzymes grouped according to the position of their genes.

![Figure 5](https://i.imgur.com/placeholder.png)

Fig. 5. Abundance ratios of proteins involved in oxidative sulfur metabolism in Chlorobaculum tepidum cultures determined on labeled FASP samples. The proteins are sorted on the abscissa according to the order of the genes in the genome. Boxes indicate proteins that putatively are encoded in operons and whose abundance ratios are markedly deviating from 1. (a) Cells harvested in late exponential growth phase (43 h; thiosulfate utilizing) relative to cells harvested in early exponential growth phase (4 h; sulfide utilizing). (b) dsrM mutant cells relative to wild-type cells (both cultures were harvested in the late exponential growth phase). DsrA1/CT0852 and DsrA2/CT2249 have identical amino acid sequence and cannot be distinguished; the same is true for DsrB1/DsrB2 and DsrC1/DsrC2.

in the genome. Seventeen per cent of the detected proteins showed large variation between wild type and *dsrM* mutant (>2 or <0.5) (Fig. S2). About one-third of these proteins are annotated as hypothetical proteins, which is a clear indication that the physiology of oxidative sulfur metabolism in *Cba.* *tepidum* is far from understood.

The core enzyme of the DSR enzyme, DsrAB, is significantly more abundant in the *dsrM* mutant. This may be explained by the fact that the substrate of the DsrAB enzyme presumably is present in high concentration (some form of reduced sulfur derived from the sulfur globules). However, DsrAB cannot transfer electrons to the putative DsrTMKJOP complex, and therefore oxidation of sulfur globules to sulfite cannot proceed (Fig. 1). The complete absence of sulfite probably explains why the Sat-Apr-Qmo enzyme system is less abundant in the *dsrM* mutant. Thus, the abundance of the individual components of the DSR system appears to be regulated according to the abundance of substrate. Finally, the absence of DsrM may explain why DsrK and DsrO are so little abundant in the *dsrM* mutant. DsrMKJOP constitute a tight complex in *Alc.* *vinosum* (Grein *et al.*, 2010) – therefore, elimination of one subunit (here DsrM) may cause disintegration of the entire complex and subsequent degradation of the subunits.

## Conclusion

Most of the enzymes putatively involved in oxidative sulfur metabolism in *Cba.* *tepidum* (Table 1) were detected, and quantitative information on their relative abundance in cells grown under different conditions was obtained (Fig. 5). Although differential protein abundance does not always correlate directly with changes in cellular activity of a particular pathway, changes in some enzymes were detected that correlated with the physiologic activity (e.g. increased Sox and CycA abundance when thiosulfate was oxidized and decreased Sat-Apr-Qmo when sulfite was not produced by the DSR system).

The proteome studies presented here shown that the FASP protocol (Wisniewski *et al.*, 2009) is a powerful alternative to gel-based separation techniques. We have also shown that in-solution labeling (Boersema *et al.*, 2009) can be combined with the FASP approach to compare proteomes from two different cell samples. These approaches may be useful in general for high-throughput analyses of cell material from laboratory and natural cultures (e.g. Zhou *et al.*, 2007; Habicht *et al.*, 2011).

## Acknowledgements

The authors thank the Danish Natural Science Research Council for support.

## References

Azai C, Tsukatani Y, Harada J & Oh-oka H (2009) Sulfur oxidation in mutants of the photosynthetic green sulfur bacterium *Chlorobium tepidum* devoid of cytochrome c-554 and SoxB. *Photosynth Res* 100: 57–65.

Bantscheff M, Schirle M, Sweetman G, Rick J & Kuster B (2007) Quantitative mass spectrometry in proteomics: a critical review. *Anal Bioanal Chem* 389: 1017–1031.

Boersema PJ, Raijmakers R, Lemeer S, Mohammed S & Heck AJ (2009) Multiplex peptide stable isotope dimethyl labeling for quantitative proteomics. *Nat Protoc* 4: 484–494.

Chan LK, Weber TS, Morgan-Kiss RM & Hanson TE (2008) A genomic region required for phototrophic thiosulfate oxidation in the green sulfur bacterium *Chlorobium tepidum* (syn. *Chlorobaculum tepidum*). *Microbiology* 154: 818–829.

Chan LK, Morgan-Kiss RM & Hanson TE (2009) Functional analysis of three sulfide:quinone oxidoreductase homologs in *Chlorobaculum tepidum*. *J Bacteriol* 191: 1026–1034.

Cline JD (1969) Spectrophotometric determination of hydrogen sulfide in natural waters. *Limnol Oceanogr* 14: 454–458.

Dahl C, Engels S, Pott-Sperling AS, Schulte A, Sander J, Lübbe Y, Deuster O & Brune DC (2005) Novel genes of the *dsr* gene cluster and evidence for close interaction of Dsr proteins during sulfur oxidation in the phototrophic sulfur bacterium *Allochromatium vinosum*. *J Bacteriol* 187: 1392–1404.

Eisen JA, Nelson KE, Paulsen IT *et al.* (2002) The complete genome sequence of *Chlorobium tepidum* TLS, a photo-synthetic, anaerobic, green-sulfur bacterium. *P Natl Acad Sci USA* 99: 9509–9514.

Elliott MH, Smith DS, Parker CE & Borchers C (2009) Current trends in quantitative proteomics. *J Mass Spectrom* 44: 1637–1660.

Frigaard N-U & Bryant DA (2004) Seeing green bacteria in a new light: genomics-enabled studies of the photosynthetic apparatus in green sulfur bacteria and filamentous anoxygenic phototrophic bacteria. *Arch Microbiol* 182: 265–276.

Frigaard N-U & Bryant DA (2008) Genomics insights into the sulfur metabolism of phototrophic green sulfur bacteria. *Advances in Photosynthesis and Respiration*, Vol. 27 (Hell R, Dahl C, Knaff DB & Leustek T, eds), pp. 337–355. Springer, Dordrecht.

Frigaard N-U & Dahl C (2009) Sulfur metabolism in phototrophic sulfur bacteria. *Advances in Microbial Physiology*, vol. 54 (Poole RK, ed.), pp. 103–200. Academic Press, London.

Frigaard N-U, Takaichi S, Hirota M, Shimada K & Matsuura K (1997) Quinones in chlorosomes of green sulfur bacteria and their role in the redox-dependent fluorescence studied in chlorosome like bacteriochlorophyll c aggregates. *Arch Microbiol* 167: 343–349.

Frigaard N-U, Sakuragi Y & Bryant DA (2004) Gene inactivation in the cyanobacterium *Synechococcus* sp. PCC 7002 and the
green sulfur bacterium Chlorobium tepidum using in vitro-  
made DNA constructs and natural transformation. Methods  
in Molecular Biology, vol. 274 (Carpentier R, ed.), pp. 325–  
340. Humana Press, Totowa, NJ.  
Gregersen LH, Bryant DA & Frigaard N-U (2011) Mechanisms  
and evolution of oxidative sulfur metabolism in green sulfur  
bacteria. Front Microbiol 2: 116.  
Grein F, Pereira IA & Dahl C (2010) Biochemical  
characterization of individual components of the  
Allochromatium vinosum DsrMKJOP transmembrane  
complex aids understanding of complex function in vivo.  
J Bacteriol 192: 6369–6377.  
Grimm F, Dobler N & Dahl C (2010) Regulation of dsr genes  
encoding proteins responsible for the oxidation of stored  
sulfur in Allochromatium vinosum. Microbiology 156: 764–  
773.  
Grimm F, Franz B & Dahl C (2011) Regulation of  
dissimilatory sulfur oxidation in the purple sulfur bacterium  
Allochromatium vinosum. Front Microbiol 2: 51.  
Habicht KS, Miller M, Cox RP, Frigaard N-U, Tonolla M,  
Peduzzi S, Falkenby LG & Andersen JS (2011) Comparative  
proteomics and activity of a green sulfur bacterium through  
the water column of Lake Cadagno, Switzerland. Environ  
Microbiol 13: 203–215.  
Hanson TE & Tabita FR (2001) A ribulose-1,5-bisphosphate  
carboxylase/oxygenase (RubisCO)-like protein from  
Chlorobium tepidum that is involved with sulfur metabolism  
and the response to oxidative stress. PNAS 98: 4397–4402.  
Holkenbrink C, Ocón Barbas S, Mellerup A, Otaki H &  
Frigaard N-U (2011) Sulfur globule oxidation in green  
sulfur bacteria is dependent on the dissimilatory sulfite  
reductase system. Microbiology 157: 1229–1239.  
Kyte J & Doolittle RF (1982) A simple method for displaying the  
hydrophobic character of a protein. J Mol Biol 157: 105–132.  
Laemmli UK (1970) Cleavage of structural proteins during the  
assembly of the head of bacteriophage T4. Nature 227: 680–  
685  
Mortensen P, Gouw JW, Olsen JV et al. (2010) MSQuant,  
an open source platform for mass spectrometry-based  
quantitative proteomics. J Proteome Res 9: 393–403.  
Ogawa T, Furusawa T, Nomura R, Seo D, Hosoya-Matsuda N,  
Sakurai H & Inoue K (2008) SoxAX binding protein, a  
novel component of the thiosulfate oxidizing multienzyme  
system in the green sulfur bacterium Chlorobium tepidum.  
J Bacteriol 190: 6097–6110.  
Ogawa T, Furusawa T, Shiga M, Seo D, Sakurai H & Inoue K  
(2010) Biochemical studies of a soxF-encoded monomeric  
flavoprotein purified from the green sulfur bacterium  
Chlorobaculum tepidum that stimulates in vitro thiosulfate  
oxidation. Biosci Biotechnol Biochem 74: 771–780.  
Ong SE, Blagoev B, Kratchmarova I, Kristensen DB, Steen H,  
Pandey A & Mann M (2002) Stable isotope labeling by  

amino acids in cell culture, SILAC, as a simple and accurate  
approach to expression proteomics. Mol Cell Proteomics 1:  
376–386.  
Overmann J (2008) Ecology of phototrophic sulfur bacteria.  
Sulfur Metabolism in Phototrophic Organisms (Hell R, Dahl  
C, Knaff DB & Leustek T, eds), pp. 375–396. Springer,  
Dordrecht.  
Rappsilber J, Mann M & Ishihama Y (2007) Protocol for  
micro-purification, enrichment, pre-fractionation and  
storage of peptides for proteomics using StageTips. Nat  
Protoc 2: 1896–1906.  
Rodriguez J, Hiras J & Hanson TE (2011) Sulfite oxidation in  
Chlorobaculum tepidum. Front Microbiol 2: 112.  
Sakurai J, Ogawa T, Shiga M & Inoue K (2010) Inorganic  
sulfur oxidizing system in green sulfur bacteria. Photosynth  
Res 104: 163–176.  
Verté F, Kostanjevecki V, De Smet L, Meyer TE, Cusanovich  
MA & Van Beeumen JJ (2002) Identification of a thiosulfate  
utilization gene cluster from the green phototrophic  
bacterium Chlorobium limicola. Biochemistry 41: 2932–2945.  
Wahlund TM, Woese CR, Castenholz RW & Madigan MT  
(1991) A thermophilic green sulfur bacterium from New  
Zealand hot springs, Chlorobium tepidum sp. nov. Arch  
Microbiol 156: 81–90.  
Walther TC & Mann M (2010) Mass spectrometry-based  
proteomics in cell biology. J Cell Biol 190: 491–500.  
Wisniewski JR, Zougman A, Nagaraj N & Mann M (2009)  
Universal sample preparation method for proteome analysis.  
Nat Methods 6: 359–362.  
Zhou F, Hanson TE & Johnston MV (2007) Intact protein  
profiling of Chlorobium tepidum by capillary isoelectric  
focusing, reversed-phase liquid chromatography, and mass  
spectrometry. Anal Chem 79: 7145–7153.  

## Supporting information

Additional Supporting Information may be found in the  
online version of this article:  

Fig. S1. Mass spectra of a peptide (TVDYAGFVDL-  
VEEHQVNSWL) derived from the CT0857/DsrH protein.  
Fig. S2. Protein abundance ratios in Cba. tepidum cul-  
tures determined on labeled FASP samples.  
Table S1. Predicted and detected proteins in Cba. tepi-  
dum.  

Please note: Wiley-Blackwell is not responsible for the  
content or functionality of any supporting materials sup-  
plied by the authors. Any queries (other than missing  
material) should be directed to the corresponding author  
for the article.  