
environmental microbiology

Environmental Microbiology (2012) 14(4), 1024–1034

doi:10.1111/j.1462-2920.2011.02682.x

# Effect of oxygen on the anaerobic methanotroph ‘Candidatus Methylomirabilis oxyfera’: kinetic and transcriptional analysis

Francisca A. Luesken,$^{1\dagger}$ Ming L. Wu,$^{1\dagger}$ Huub J. M. Op den Camp,$^1$ Jan T. Keltjens,$^1$ Henk Stunnenberg,$^2$ Kees-Jan Francoijs,$^2$ Marc Strous$^{3,4}$ and Mike S. M. Jetten$^{1*}$

$^1$Department of Microbiology, Radboud University Nijmegen, Institute for Water and Wetland Research, Heyendaalseweg 135, NL-6525 AJ Nijmegen, The Netherlands.

$^2$Department of Molecular Biology, Faculty of Sciences, Nijmegen, Centre of Molecular Life Sciences, Radboud University Nijmegen, Geert Grooteplein 28, NL-6525 GA, Nijmegen, The Netherlands.

$^3$MPI for Marine Microbiology, Celsiusstrasse 1, D-28359, Bremen, Germany.

$^4$Centre for Biotechnology, University of Bielefeld, Germany.

## Summary

‘Candidatus Methylomirabilis oxyfera’ is a denitrifying methanotroph that performs nitrite-dependent anaerobic methane oxidation through a newly discovered intra-aerobic pathway. In this study, we investigated the response of a *M. oxyfera* enrichment culture to oxygen. Addition of either 2% or 8% oxygen resulted in an instant decrease of methane and nitrite conversion rates. Oxygen exposure also led to a deviation in the nitrite to methane oxidation stoichiometry. Oxygen-uptake and inhibition studies with cell-free extracts displayed a change from cytochrome c to quinol as electron donor after exposure to oxygen. The change in global gene expression was monitored by deep sequencing of cDNA using Illumina technology. After 24 h of oxygen exposure, transcription levels of 1109 (out of 2303) genes changed significantly when compared with the anoxic period. Most of the genes encoding enzymes of the methane oxidation pathway were constitutively expressed. Genes from the denitrification pathway, with exception of one of the putative nitric oxide reductases, norZ2, were severely downregulated. The majority of known genes involved in the vital cellular functions, such as nucleic acid and protein biosynthesis and cell division processes, were downregulated. The alkyl hydroperoxide reductase, *ahpC*, and genes involved in the synthesis/repair of the iron-sulfur clusters were among the few upregulated genes. Further, transcription of the *pmoCAB* genes of aerobic methanotrophs present in the non-*M. oxyfera* community were triggered by the presence of oxygen. Our results show that oxygen-exposed cells of *M. oxyfera* were under oxidative stress and that in spite of its oxygenic capacity, exposure to microoxic conditions has an overall detrimental effect.

## Introduction

Methane munching microbes play an important role in the global carbon cycle and mitigate the greenhouse effect by oxidizing methane before it reaches the atmosphere (Hanson and Hanson, 1996). This process can be performed either aerobically by methanotrophic bacteria (methanotrophs), members of the phyla *Proteobacteria* and *Verrucomicrobia*, or anaerobically by sulfate-dependent methanotrophic archaea (Knittel and Boetius, 2009; Op den Camp *et al.*, 2009). Recently, a new group of bacterial methanotrophs affiliated to the ‘NC10’ phylum was discovered (Raghoebarsing *et al.*, 2006). The representative bacterium, tentatively named ‘Candidatus Methylomirabilis oxyfera’, performs nitrite-dependent anaerobic methane oxidation, with the following stoichiometry (Raghoebarsing *et al.*, 2006; Ettwig *et al.*, 2010):

$$
3 \text{ CH}_4 + 8 \text{ NO}_2^- + 8 \text{ H}^+ \rightarrow 3 \text{ CO}_2 + 4 \text{ N}_2 + 10 \text{ H}_2\text{O}
$$

Interestingly, this process is performed via a classical aerobic methane oxidation pathway in the total absence of externally supplied O₂ (Ettwig *et al.*, 2010). This apparent paradox is explained by the unique ability of *M. oxyfera* to produce intracellular O₂ through an alternative denitrification pathway that does not involve nitrous oxide (N₂O) as an intermediate. In this pathway, the nitric oxide (NO) generated from the reduction of nitrite (NO₂⁻) is suggested to be disproportionated into dinitrogen gas (N₂) and O₂ (Fig. 1) (Ettwig *et al.*, 2010; Wu *et al.*, 2011a). The

Received 16 June, 2011; revised 15 November, 2011; accepted 25 November, 2011. *For correspondence. E-mail m.jetten@science.ru.nl; Tel. (+31) 24 3562969; Fax (+31) 24 3652830. †These authors contributed equally to this study.

© 2012 Society for Applied Microbiology and Blackwell Publishing Ltd

'Candidatus Methylomirabilis oxyfera' exposed to oxygen 1025

variation seems mainly caused by differences in sensitivity of enzymes and regulators involved in the reactions of the denitrification pathway. In some denitrifiers, higher growth rates were observed when O₂ and nitrate were simultaneously used as electron acceptors (Robertson and Kuenen, 1984; Bonin and Gilewicz, 1991).

In addition to the possible effects on the central energy metabolism, a direct consequence of increased O₂ utilization as terminal electron acceptor is the concomitant increase in the formation of highly reactive O₂ species (ROS), which are derived from the partial reduction of O₂. The increased intracellular ROS can damage essential cellular components when not scavenged (Fridovich, 1978; Imlay *et al.*, 1988). To counteract this effect, many organisms have developed O₂-detoxification systems: superoxide anions (O₂⁻) are converted to hydrogen peroxide (H₂O₂) by superoxide dismutase (SOD), and H₂O₂ is eliminated by the action of catalases and peroxidases (Farr and Kogoma, 1991).

The response of *M. oxyfera* to O₂ has not been investigated so far. Here, cells from a *M. oxyfera* enrichment culture were exposed to two different O₂ concentrations (2% and 8%). Kinetics of substrate utilization under anoxic conditions and upon O₂ exposure was measured. For the cells treated with 8% O₂, the differential O₂ reduction potential by the cell-free extracts was measured and the total mRNA profiles were examined to evaluate the changes in global gene expression.

Fig. 1. Postulated model for central catabolism and energy conservation in *M. oxyfera*. Open triangles, direction of proton flow. $bc_1$, cytochrome $bc_1$ complex; mdh, methanol dehydrogenase; ndh, NAD(P)H dehydrogenase complex; nir, nitrate reductase; nod, nitric oxide dismutase; pmmo, particulate methane monooxygenase; Q, co-enzyme Q. Adapted from Wu and colleagues (2011a).

majority of the generated O₂ is used for activation and oxidation of methane in the strictly O₂-dependent reaction catalysed by particulate methane monooxygenase (pMMO). It has been proposed that the remainder of the O₂ is used by other processes, like 'normal' respiration by terminal O₂ reductases (TOR). The presence of four sets genes encoding TORs in the genome of *M. oxyfera* and the previously reported functional production of a bo-type ubiquinol (UbqO) TOR suggests that this organism is capable of nitrite and O₂ co-respiration (Wu *et al.*, 2011b).

Thus far, *M. oxyfera* and *M. oxyfera*-like bacteria have been found in various freshwater habitats, in close proximity to the oxic/anoxic interface (Raghoebarsing *et al.*, 2006; Ettwig *et al.*, 2008; Hu *et al.*, 2011; Luesken *et al.*, 2011a,b). Such habitats are subjected to fluctuations in O₂ concentrations and in transient exposure to O₂. Thus, being able to profit or simply tolerate external O₂ might be essential for the survival and competitive fitness of *M. oxyfera* cells in natural O₂-limited ecological habitats.

The effects of O₂ exposure on organisms can be manifold. In *M. oxyfera* it could enhance the rates of O₂-dependent reactions, such as the ones catalysed by pMMO and TORs. During electron transport, more energy is conserved by O₂ respiration due to the involvement of TORs, when compared with denitrification; hence, an O₂ surplus could be beneficial to *M. oxyfera*. On the other hand, the denitrification pathway itself, which is often O₂-sensitive, can be inhibited. Studies have shown that O₂ can suppress the synthesis of denitrifying enzymes and reversibly inhibit their activity. The extent of this effect is species-dependent and the O₂-tolerance varies greatly among microbes. *Thiomicrospira denitrificans* requires absolute anoxia for denitrification, while *Paracoccus pantotropha* can still denitrify at 90% air saturation (Hoor, 1975; Robertson and Kuenen, 1984; Zumft, 1997). This

Results

Effects of O₂ on nitrite-dependent anaerobic methane oxidation rates

Biomass from a *M. oxyfera* enrichment culture was exposed to O₂ in two different experiments (2% and 8% O₂). In the anoxic period of both experiments, as well as in the control experiment, the methane and nitrite conversion rates occurred close to the expected 3 CH₄: 8 NO₂⁻ stoichiometry (Raghoebarsing *et al.*, 2006) (Fig. 2 and Table 1). After addition of O₂, the methane and nitrite conversions decreased instantaneously and the stoichiometry deviated substantially (1 CH₄: 2 NO₂⁻) from the theoretical values (Fig. 2 and Table 1).

In the second experiment (8% O₂), the produced dinitrogen gas was analysed for the masses ¹⁵-¹⁵N₂, ¹⁵-¹⁴N₂ and ¹⁴-¹⁴N₂ because unlabeled residual nitrite was present in the *M. oxyfera* biomass before labelled nitrite was added. The production rate of ¹⁵-¹⁵N₂ was 1.3 nmol min⁻¹ mg protein⁻¹ and in total 1.8 nmol N₂ min⁻¹ mg protein⁻¹ was formed. The initial concentration of O₂ (8%) was depleted within 42 h. In the following anoxic period (69.5–77.5 h), the anaerobic methane oxidation coupled to denitrification did not fully recover. The measured anoxic conversion of

© 2012 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **14**, 1024–1034

1026 F. A. Luesken et al.

![Figure 2](#fig-2)

Fig. 2. Methane (closed squares) and nitrite (open circles) consumption by the whole enrichment culture performing nitrite-dependent anaerobic methane oxidation.
A. In the anoxic period (white background) methane and nitrite were converted according to stoichiometry (Eq. 1). After addition of O₂ (gray background) the conversions rates of methane and nitrite decreased by 25% and 57% respectively.
B. In the second activity experiment conversions of methane and nitrite in the oxic period (gray background) compared with the anoxic period (white background) decreased by 72% and 81% respectively.

methane during this period was 0.7 nmol min⁻¹ mg protein⁻¹, which was lower than the initial methane oxidation rate measured. In the anoxic control (0.14 g protein), kept in parallel for 79 h, the methane and nitrite conversions were close to the rates in the anoxic period from the second experiment (8% O₂).

**Effects in the O₂-uptake by cell-free extracts**

O₂ consumption by cell-extracts of M. oxyfera culture under anoxic (t = 0) and after 24 h of exposure to 8% of O₂ was assessed using specific reductants: TMPD/cyt c or UbqO (Table 2). CN⁻ and SHAM were used as specific inhibitors for haem-copper TORs (Garcia-Horsman et al., 1994) and diiron alternative O₂ reductase (AOX) (Schonbaum et al., 1971) respectively. The total O₂-uptake capacity by M. oxyfera cell-free extracts was not significantly affected by O₂ exposure. Under anoxic conditions, a higher activity for TMPD/cyt c-dependent O₂-uptake was observed. UbqO-dependent O₂-uptake only accounted for about 20% of the total reduced O₂.

When assayed with inhibitors, the UbqO-dependent O₂-uptake was insensitive to both CN⁻ and SHAM. In contrast, the TMPD/cyt c-dependent respiration was inhibited up to 50% by CN⁻. Thus, under anoxic conditions, 50% of the total consumed O₂ can be assigned to the activity of TOR(s). After exposure to O₂, a shift to UbqO-dependent O₂-uptake was observed. The TMPD/cyt c O₂-uptake was twofold lower in comparison to the samples from the anoxic condition and the UbqO-dependent O₂-uptake increased up to threefold. Now, the TMPD/cyt c O₂-uptake was not significantly affected by CN⁻, whereas the UbqO-dependent O₂-uptake inhibited by CN⁻ and SHAM respectively. This results show that exposure to O₂ stimulated the UbqO-dependent O₂-uptake. Still, only 33% of the total O₂ consumption can be assigned to TOR enzymes.

**Changes in transcript levels upon O₂ exposure**

We monitored the change in the gene expression of M. oxyfera by comparing the transcript profiles of cells

Table 1. Methane, nitrite and O₂ conversion rates under anoxic conditions and after exposure to either 2% or 8% of pure O₂.

| Experiment | Anoxic          | Oxic           |
|------------|-----------------|----------------|
|            | nmol min⁻¹ mg protein⁻¹ | nmol min⁻¹ mg protein⁻¹ |
| 2%         | CH₄             | 0.7            | CH₄             | 0.5            |
|            | NO₂⁻            | 2.3            | NO₂⁻            | 1.0            |
| 8%         | CH₄             | 1.8            | CH₄             | 0.5            |
|            | NO₂⁻            | 5.3            | NO₂⁻            | 1.0            |
| Control    | CH₄             | 2.1            | O₂              | 0.9            |
|            | NO₂⁻            | 5.6            |                 | 4.0            |

The control experiment received 30 ml of helium after 28.5 h, thus remained anoxic with stable conversion rates (the control was measured until t = 79 h).

© 2012 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, 14, 1024–1034

Table 2. O₂ reduction activity and inhibition of M. oxyfera cell-free extracts from the anoxic condition and after 24 h exposure to 8% O₂.

| Substrate          | Anoxic       | Oxic        |
|--------------------|--------------|-------------|
| TMPD + cyt c      | 341 ± 22     | 148 ± 9     |
| + 5 mM CN⁻        | 170 ± 19     | 125 ± 2     |
| Q₁H₂              | 66 ± 9       | 169 ± 14    |
| + 5 mM CN⁻        | 86 ± 6       | 109 ± 7     |
| + 5 mM SHAM       | 67 ± 4       | 125 ± 2     |

a. Results are the means with standard deviations from at least three independent assays. Cell-free extract protein concentrations in the (0.5 ml) polarographic chamber during the assay ranged between 0.8 and 2.7 mg ml⁻¹.

under anoxic conditions (t = 0) and after 24 h of exposure to 8% of O₂ (Tables 3–5 and S1). From 2303 transcripts, 148 were upregulated, 961 downregulated, and 856 transcripts did not change significantly. Some transcripts were only detected under anoxic (296) or O₂ exposed conditions (42).

The analysis of the cellular functions of proteins encoded by genes showing differential transcription can elucidate the mechanisms by which O₂ exposure affects M. oxyfera. From this point of view, three major groups could be distinguished, which include genes encoding enzymes involved in the (i) central energy metabolism; (ii) direct response to the presence of O₂, such as TORs and ROS defence; (iii) protein and nucleic acid synthesis process, and cell division processes.

### Genes involved in the central energy metabolism

The transcription profile of genes encoding enzymes involved in methane oxidation were not significantly affected by exposure to O₂ (Table 3). Out of 18 transcripts, including a variety of enzymes that catalyse the oxidation of methane to carbon dioxide, only 5 were downregulated. These comprised genes encoding enzymes that catalyse partial reactions succeeding the formation of formaldehyde, such as formaldehyde activating enzyme (fae) and formate dehydrogenase (fdh).

The denitrification pathway, on the other hand, was more significantly affected by the presence of O₂. From the 13 genes, 9 were downregulated (Table 4). The expression of gene clusters encoding for nitrate reductase (NAR) and nitrite reductase (NIR) were notably suppressed, with several genes showing a 10-fold lower expression. Interestingly, two gene transcripts that were not suppressed by O₂ were the ones that encode for the putative nitric oxide reductase (NOR) enzyme: norZ3 was transcribed at the same level and norZ2 was upregulated up to 7.5-fold.

Table 4. Transcriptome analysis of genes encoding enzymes involved in the denitrification pathway in M. oxyfera under anoxic conditions and after 24 h exposure to 8% O₂.

| Enzyme               | Gene          | ORF identifier   | # unique reads detected | Average coverage | Ratio (oxic vs. anoxic) |
|----------------------|---------------|------------------|-------------------------|------------------|--------------------------|
| Nitrate reductase    | narG          | DAMO_0778        | 1452                    | 4.0              | 0.2                      |
|                      | narH          | DAMO_0776        | 2732                    | 17.9             | 0.1                      |
|                      | narL          | DAMO_0774        | 259                     | 3.8              | 0.2                      |
|                      | narJ          | DAMO_0775        | 353                     | 3.9              | 0.1                      |
| Periplasmic nitrate reductase | napA          | DAMO_2411        | 250                     | 1.1              | 0.5                      |
|                      | napB          | DAMO_2410        | 8                       | 0.2              | 0.4                      |
|                      | napS          | DAMO_2415        | 2134                    | 13.0             | 0.1                      |
| Nitrite reductase    | nirJ          | DAMO_2413        | 116                     | 1.0              | 0.6                      |
|                      | nirF          | DAMO_2412        | 88                      | 0.7              | 0.5                      |
|                      | nirD/nirG/nirH/nirL | DAMO_2409        | 48                      | 0.4              | 0.1                      |
|                      | norZ1         | DAMO_1889        | 201                     | 0.9              | 0.3                      |
| Nitric oxide reductase | norZ2         | DAMO_2434        | 2229                    | 9.2              | 69.4                     |
|                      | norZ3         | DAMO_2437        | 9380                    | 38.7             | 50.3                     |

---

Table 5. Transcriptome analysis of genes encoding enzymes involved in the response to oxidative stress and TORs in M. oxyfera under anoxic conditions and after 24 h exposure to 8% O₂.

| Enzyme               | Gene          | ORF identifier   | # unique reads detected | Average coverage | Ratio (oxic vs. anoxic) |
|----------------------|---------------|------------------|-------------------------|------------------|--------------------------|
| Cyt c-dependent TOR  | cox1          | DAMO_1165        | 103                     | 0.6              | 0.5                      |
|                      | cox2          | DAMO_1166        | 93                      | 1.3              | 0.7                      |
|                      | cox3          | DAMO_1164        | 43                      | 0.8              | 0.1                      |
| Cyt c-dependent TOR  | cox4          | DAMO_1162        | 13                      | 0.5              | 0.2                      |
|                      | cbaA          | DAMO_0801        | 267                     | 1.9              | 0.5                      |
|                      | cbaB          | DAMO_0802        | 166                     | 3.0              | 0.8                      |
| UbqO-dependent TOR   | cyoB          | DAMO_1118        | 73                      | 0.5              | 1.6                      |
|                      | cyoA          | DAMO_1119        | 19                      | 0.3              | 4.7                      |
| Diriron cyanide-insensitive AOX | aoX          | DAMO_2910        | 41                      | 0.6              | 16                       |
| Dyp-type peroxidase  | dyp           | DAMO_0362        | 35                      | 0.4              | 0.2                      |
| Di-haem cytochrome c peroxidase | ccp           | DAMO_1661        | 41                      | 0.6              | 0.1                      |
| Alkyl hydroperoxidase | ahpC          | DAMO_1666        | 93                      | 0.6              | 0.1                      |
| Superoxide dismutase | sodB          | DAMO_0395        | 215                     | 3.6              | 0.4                      |
| Rubrerythrin         | rbr           | DAMO_1284        | 129                     | 2.6              | 0.6                      |

Light grey = downregulated genes; dark grey = upregulated genes; white = constitutively expressed.

Genes involved in direct response to the presence of O₂: ROS defence system and TORs

A direct effect of the presence of O₂ is the production of ROS, which can cause damage at various cellular levels if it exceeds the capacity of the cell defence system. Analysis of the genome of *M. oxyfera* showed that it contained a repertoire of genes potentially encoding enzymes that are implicated in the defence against ROS: (i) a Fe-SOD (*sodB*) (ii) three haem iron peroxidases (2 ccp; 1 dyp) (iii) a non-haem iron rubrerythrins (*rbr*) and an alkyl hydroperoxidase reductase (*ahpC*). A catalase-encoding gene was not present in the genome. Differential expressions of these genes are shown in Table 5. The *sodB* gene encoding SOD, which catalyses dismutation of O₂⁻ to hydrogen peroxide, was constitutively expressed. Interestingly, two of the peroxidases and *rbr* were down-regulated. *AhpC*, a hydrogen peroxide scavenger and member of the thioredoxin fold superfamily (Schröder and Ponting, 1998), was the only gene of O₂ defence that was upregulated (fourfold) upon O₂ exposure.

The genome of *M. oxyfera* contains four gene clusters encodings TORs; these include two haem-copper cytochrome c-dependent TORs, one haem-copper UbqO-dependent TOR and a diiron cyanide-insensitive AOX (Wu et al., 2011b). It was expected that an O₂ surplus would induce the expression of TORs as more substrate was available. However, the expressions of these genes did not change significantly (Table 5).

---

Genes involved in the protein synthesis, nucleic acids synthesis and cell division

Differential analysis of the genes involved in the protein synthesis and repair suggests that *M. oxyfera* was under oxidative stress after O₂ exposure (Table S1). Overall, genes involved in cell division were repressed in the presence of external O₂; these include for instance genes belonging to the *fts* operon and the rod-shape maintaining bacterial actin homologue *mreB* (Cabeen and Jacobs-Wagner, 2005). The majority of the genes involved in DNA replication were either constitutively expressed or down-regulated. Exceptionally, *dnaK*, which is also implicated in the hyperosmotic shock (Bianchi and Baneyx, 1999), was upregulated.

Proteins containing iron–sulfur (Fe/S) clusters are very prone to O₂⁻ attack. Exposure of *M. oxyfera* to O₂ resulted in an increased expression of *nifU/nifS* and SUF operon (*sufABCDSE*) genes, which encode for proteins involved in Fe/S biosynthesis and/or repair (Ayala-Castro et al., 2008). It is noteworthy that the gene encoding for the flavin-containing glycolate oxidase and chaperone-like peptidyl-prolyl cis-trans isomerase (PPIase) were repressed after O₂ exposure.

---

Table 6. Analysis of *M. oxyfera* and non-*M. oxyfera* pmo and amo gene cluster transcripts under anoxic conditions (2282 reads) and after 24 h exposure to 8% O₂ (2110 reads).

| Organism          | Anoxic   | Oxic    |
|-------------------|----------|---------|
|                   | # reads  | %       | # reads | %     |
| *M. oxyfera*      | 2275     | 99.7    | 1911    | 90.6  |
| Methylobacter     | 0        | 0       | 4       | 0.2   |
| Methylovulum      | 0        | 0       | 11      | 0.5   |
| Methylomicrobium  | 0        | 0       | 9       | 0.4   |
| Methylomonas      | 4        | 0.2     | 108     | 5.1   |
| Methylococcus     | 1        | 0       | 24      | 1.1   |
| Methylosinus      | 0        | 0       | 15      | 0.7   |
| Methylocystis     | 2        | 0.1     | 28      | 1.3   |
| Nitrosomonas      | 0        | 0       | 2       | 0.1   |

---

Analysis of non-*M. oxyfera* community

Due to the complexity of the enrichment culture used, O₂ can affect not only *M. oxyfera* but also the non-*M. oxyfera* community. Thus, the community composition of the culture was monitored using FISH and complemented by the analysis of the total *pmo* gene transcripts under anoxic and O₂ exposed conditions. When monitored by FISH, no significant differences were observed between the anoxic and oxic periods, using general probes targeting organisms like α-, β- and γ-Proteobacteria, and specific probes targeting *M. oxyfera* (Fig. S1). This was corroborated by 70% percentage of Illumina reads that matched to *M. oxyfera* in both periods. Examination of the total *pmo* gene transcripts showed that O₂ triggered the onset of transcription of *pmo* genes from various aerobic methanotrophs and ammonium monooxygenase (*amo*) genes from Nitrosomonas (Table 6). However, the majority of *pmo* transcripts matched to *M. oxyfera* in both periods.

---

Discussion

Three properties of *M. oxyfera* make it a very interesting system to study the effects of O₂: (i) the ability to perform anaerobic methane oxidation coupled to denitrification, (ii) the ability to produce O₂ intra-aerobically, and (iii) the possession of an apparently suitable ROS defence system. In this study, the effects of O₂ additions (2% and 8%) to incubations of *M. oxyfera* were investigated. Physiology studies of *M. oxyfera* are often cumbersome mostly caused by the unstable nature of the enrichment culture and the limited amount of biomass available (Ettwig et al., 2009). Therefore, a complementary array of methods was used to study the physiological changes in *M. oxyfera* cells.

The anoxic *M. oxyfera* incubations showed methane oxidation coupled to nitrite reduction close to the expected 3:8 stoichiometry as reported previously (Raghoebarsing

1030 F. A. Luesken et al.

et al., 2006) (Table 1). Addition of either 2% or 8% O₂ resulted in an instant decrease in the nitrite conversion rates leaving only 43% and 19% of the original nitrite converting activity respectively (Fig. 2 and Table 1). This decrease is strongly corroborated by the decrease in expression levels of nitrate and nitrite reductase genes. Substrate limitation can be ruled out because both methane and nitrite were present in sufficient amounts throughout the experiment. Therefore, the observed decrease in activity is most likely a direct effect of the O₂ added to the incubations. Upon O₂ depletion, it appeared that *M. oxyfera* does not resume activity to its original capacity. The above mentioned findings are highly interesting, taken into account that *M. oxyfera* is an oxygenic bacterium. One possible explanation is that being a slow-growing and slow-metabolizing organism, both the ROS defence system and the electron transport chain in *M. oxyfera* have a low capacity, and thus, a surplus of O₂ does not result in a complete transfer of electrons to O₂ and efficient elimination of ROS.

After O₂ addition, both the methane and nitrite conversion rates decreased but also the stoichiometry shifted towards methane oxidation independent of denitrification. This effect could be due to an increased susceptibility of the denitrification enzymes to oxidative conditions, a preference for O₂ as electron acceptor, or a combination of these effects. In line with the observed shift in the stoichiometry and the susceptibility of enzymes involved in denitrification, the majority of genes involved in the denitrification pathway were downregulated upon exposure to 8% O₂ (Table 4). Interestingly, the transcription of the *norZ2* gene encoding for the NOR was highly induced by O₂. The *norZ2* gene is one of the candidate enzymes for NO disproportionation, the reaction responsible for the oxygenic capacity of *M. oxyfera* (Ettwig et al., 2010). NOR enzymes can have O₂ reductase activity. For instance, the cb-type NOR from *P. denitrificans*, reduces O₂ to water as a side reaction (Fujiwara and Fukumori, 1996). Thus, it is conceivable that under oxidative conditions, the upregulation of *norZ2* serves for O₂ respiration or detoxification. Nevertheless, further investigation is necessary to test these hypotheses. Due to the community complexity of the enrichment culture, it is also possible that the surplus of methane, uncoupled to nitrite reduction, was consumed by other members of the community, such as aerobic methanotrophs. Although no significant variation in the non-*M. oxyfera* community was observed using FISH, a more sensitive analysis of *pmo* gene transcripts showed that O₂ triggered the onset of *pmo* transcription of a variety of aerobic methanotrophs (Table 6). Thus, it is likely that a small part of the methane oxidation observed in the oxic period is due to the activity of aerobic methanotrophs. In addition, at both O₂ concentrations tested, O₂ reduction occurred at rates higher than could be assigned

to methane oxidation alone. The source of the excess of O₂ consumed might be heterotrophic, given that internal substrates can be generated by decaying biomass or by using reserve material.

O₂-uptake and inhibition studies using specific inhibitors showed that the TOR-dependent activity decreased from 50% in the anoxic period to 33% when exposed to 8% O₂ (Table 2). O₂ exposure also resulted in a shift in the electron donor specificity: under anoxic conditions, the major part of O₂ consumption can be assigned to a cytochrome c-dependent TORs, whereas after O₂ exposure the UbqO TOR activity was dominant. Despite of this variation, no significant variation in the level of transcripts was found for the four TORs. This shift could be due to: (i) regulation of these enzymes at the translation level, (ii) O₂-consumption by aerobic methanotrophs, and alternatively (iii) it is possible that at least part of the increased UbqO-respiration derives from NOR activity, which would be consistent with the marked upregulation of *norZ2*.

Genome analysis suggests that *M. oxyfera* is suitably equipped with a ROS defence system. Among genes implicated in the ROS systems, only *ahpC* was induced in O₂-treated cells. Two of the haem iron peroxidases and *rbr* were downregulated. The exact physiological role of rubrerythrins in defence against oxidative stress has not been entirely elucidated. However, a peroxidase activity has been suggested (Sztukowska et al., 2002). The constitutive expression of the other *ccp* (DAMO_1666) and *sodB* may serve to provide *M. oxyfera* with a standby defence against the internally produced O₂, whereas *ahpC* functions as a major peroxide scavenger under oxidative conditions. In the strict anaerobe *Clostridium perfringens*, transcription of both *sodB* and *rbr* did not change in O₂-treated cells (Geissmann et al., 1999). On the other hand, increased levels of SOD have been reported for anaerobically grown cultures of *Streptococcus faecalis* and *Escherichia coli* B exposed to O₂ (Gregory and Fridovich, 1973). These observations suggest the response to O₂ among microorganism in not uniformly similar.

Two additional observations support the notion that *M. oxyfera* was under oxidative stress upon O₂ exposure. First, the majority of the differentially expressed genes were downregulated (961 down, 148 up). Second, most of the genes involved in various vital cellular functions were among the downregulated ones. For example, genes involved in DNA replication, protein folding and stabilization, and cell division, including *mreB* and the one from *fts* operon, were downregulated. The lower amounts of the chaperone-like PPlase transcripts is a factor that could prevent the folding/stabilization of de novo synthesized proteins (Ideno et al., 2001) as previously reported for O₂-treated cells of the strict anaerobe *Desulfovibrio vulgaris* (Fournier et al., 2006). Flavin-containing proteins,
such as glycolate oxidase, may generate hydrogen peroxide in response to O₂. Lower expression of glycolate oxidase by *M. oxyfera* might serve the means of preventing excessive hydrogen peroxide formation. In *D. vulgaris* lower levels of glycolate oxidase were detected in the O₂-treated cells (Fournier et al., 2006). The genes encoded by the *nifU/nifS* and SUF operon have a role of synthesis and repair of Fe/S clusters (Dos Santos et al., 2004); induction of these genes point to Fe/S clusters damage in the O₂-treated *M. oxyfera* cells.

Despite the long duration of our experiment, the question remains whether in a continuous culture with alternating oxic/anoxic conditions as described by Strous and colleagues (1997) *M. oxyfera* would adapt to or even benefit from microoxic conditions. In conclusion, the applied O₂ conditions in this study have an overall damaging effect in *M. oxyfera*. The inhibition of the central energy metabolism and the deviation in the stoichiometry when cells are exposed to O₂ suggests that O₂ production and consumption by *M. oxyfera* is a tightly controlled process.

### Experimental procedures

#### M. oxyfera enrichment culture

*Methylomirabilis oxyfera* was enriched in an anoxic sequencing batch reactor (15 l) at 30°C as described before (Ettwig et al., 2009). During prolonged enrichment (> 6 months), O₂ levels were always below the detection level as monitored by a Clark-type O₂ electrode. *M. oxyfera* made up about 70–80% of the total bacterial population, as shown by FISH and metagenome analysis.

#### Experimental set-up

As the O₂ tolerance of *M. oxyfera* was unknown, two experiments with different O₂ concentrations were performed, one with initially 2% pure O₂ in the headspace and one with initially 8% pure O₂. Experiments were carried out at 30°C in bottles with a total volume of 2.3 l or 1.1 l. Biomass from the *M. oxyfera* enrichment was used to inoculate the bottles, which were equipped with a gas and liquid sampling ports. To buffer the liquid 3-(N-morpholino) propanesulfonic acid (MOPS) was added to a final concentration of 30 mM resulting in a pH of 7.2. The headspace of the bottles including MOPS, were flushed with helium for 30 min before the experiment. In both experiments biomass from the *M. oxyfera* enrichment culture containing 0–10 mg l⁻¹ nitrite was pumped directly into the anoxic bottles using Masterflex easy-load II pumps with 16 mm Norprene tubing (Cole Parmer, USA), to prevent exposure to air. Overpressure was obtained by adding either 150 or 100 ml of helium to the first and second experiments respectively. There was an incubation period of approximately 12 h to reach equilibrium for the added gases, before the measurements started. Bottles were stirred at 150 r.p.m. during the course of the experiment.

In the first experiment, a 2.3 l bottle was inoculated with 1 l biomass. Labelled nitrite (stock solution: 1 M ¹⁵NO₂⁻) was directly added to the bottle, with a final concentration of 3 mM. Subsequently labelled methane (50 ml ¹³CH₄) was added to the headspace of the bottle. At *t* = 27 h, 20 ml of pure O₂ was supplied to the headspace and at *t* = 49 h another 10 ml of O₂ was added to the bottle.

In the second experiment, a 1.1 l bottle was inoculated with 700 ml bottle. At the start of the experiment, before addition of ¹⁵NO₂⁻, the residual nitrite concentration was 0.2 mM. The stock solution of labelled nitrite (1 M ¹⁵NO₂⁻) was first diluted in medium [according to (Ettwig et al., 2009) without NO₂⁻] to 60 mM and subsequently added to the bottle with a final concentration of 2.5 mM nitrite. The total volume of the sample was 750 ml. Labelled methane (40 ml ¹³CH₄) was injected in the headspace, followed by argon (0.3%), which was used as internal control for leakage and remained constant throughout the experiment. After an anoxic period of 27.5 h, 30 ml of pure O₂ was added to the culture, resulting in a headspace concentration of 8%. This experiment was prolonged until the initial O₂ concentration (8%) was depleted. Subsequently, methane was measured in this second anoxic period from *t* = 69.5–77.5 h.

A control experiment was run in parallel, where 30 ml of helium instead of pure O₂ was added. Samples (70 ml) for the O₂-uptake assays were taken after 24 h of O₂ exposure and RNA was extracted for deep sequencing of cDNA using Illumina technology (see below).

Gas samples were taken for CH₄ analysis using gas chromatography (100 μl per injection); N₂ and O₂ were analysed using a gas chromatograph coupled to a mass spectrometer (30 μl per injection) as described below.

### Transcriptomics

Total RNA was extracted from samples (6 ml) from the anoxic bottle culture and after 24 h exposure to 8% O₂ using the RiboPure – Bacteria kit (Ambion, AM1925, The Netherlands), according to the manufacturer’s instructions. After isolation, an additional DNase treatment was performed. Quality was assessed by 1.2% agarose gel electrophoresis and concentrations were measured with NanoDrop ND-1000 spectrophotometer (Isogen Life science). First-stranded cDNA was synthesized with random primers using the RevertAid H Minus First Strand cDNA Synthesis Kit, and the second strand was synthesized using DNA polymerase (Fermentas Life Sciences) according to the manufacturer’s instructions. Purification of dsDNA was performed using the phenol chloroform method. Single-end Illumina sequencing was performed as described previously (Ettwig et al., 2010). The obtained Illumina reads (read length: 75 nt; 27 890 382 for oxic conditions and 30 582 570 reads under anoxic conditions; Table S1) were mapped with CLC bio genomics software to the genome assembly of *M. oxyfera* (NCBI Accession No. GSE18535) excluding the rRNA genes. Reads were counted if they had a minimum identity of 90% and minimum length coverage of 90% to the *M. oxyfera* genes. Excluding the rRNA genes, 158 159 reads were mapped to *M. oxyfera* genes for the oxic period and 250 081 for the anoxic period. This resulted in a average coverage, defined as N × L/G, were N is the number of reads, L the average read length and

© 2012 Society for Applied Microbiology and Blackwell Publishing Ltd, *Environmental Microbiology*, **14**, 1024–1034
1032 F. A. Luesken et al.

G length of the genome, of 4.5- and 7.2-fold for the anoxic and oxic conditions respectively. The percentages of total reads mapped to *M. oxyfera* including ribosomal RNA in anoxic and oxic conditions were 73.8% and 69.3% respectively. The relative expression of the *M. oxyfera* genes was calculated as described previously (Ettwig et al., 2010). Only differential expressions with a twofold lower or higher than average coverage were considered significant. In order to analyse reads that had a match with methane monooxygenase genes, 7076 *pmo* genes were downloaded from NCBI (14 June 2011). Illumina reads were mapped with a minimum DNA identity of 66% and minimum length coverage of 80%. In this way, 2282 and 2110 reads were obtained for the anoxic and oxic period respectively. To assess the diversity in those reads, a Blastx (and Blastn) analysis versus the non-redundant NCBI database was performed using an *E*-value of 0.0001.

### Preparation of cell-free extracts

Approximately 1.6 g of biomass (wet weight) from the anoxic period and biomass exposed to 24 h of 8% O₂ was collected and washed three times in 20 mM Tris-HCl, pH 8.0. Cells were resuspended in 10 ml buffer, containing 20 mM Tris-HCl, pH 8.0, 50 mM sodium pyrophosphate, 0.5 mM phenylmethanesulfonyl fluoride (PMSF), and a few grains of solid DNase. Cells were broken by sonication, cell debris was removed by centrifugation (6000 × *g*, 15 min, 4°C) and the supernatant was collected as cell-free extract.

### O₂ reduction measurements

O₂ consumption by the *M. oxyfera* cell-free extracts was measured polarographically at 30°C using a microsensor Clarke-type electrode (Unisense, Denmark). Anoxic stock solutions of the substrates and inhibitors were prepared in 100% helium gas before injection to the reaction mixtures. Bovine heart cytochrome c (10 μM) and N,N,N′,N′-tetramethyl-p-phenylenediamine (TMPD; 2 mM) O₂ reductase activities were assayed in the presence of 2 mM sodium ascorbate. Ubiquinone-1 (Q₁; 60 μM) reduced with 10 mM dithiothreitol (DTT) was used to assay UbqO O₂ reductase activity. In the inhibition assay, the samples were pre-incubated with cyanide (CN⁻; 5 mM) or salicylhydroxamic acid (SHAM; 5 mM) for 1 h. The donor-specific O₂ consumption rates were corrected for the auto-oxidation and endogenous respiration rates as described by Wu and colleagues (2011b).

### Fluorescence in situ hybridization (FISH)

Biomass samples (2 ml) from the anoxic and oxic period were fixed and hybridized according to Luesken and colleagues (2011b) using 20–50% formamide, depending on the specificity of the oligonucleotide probes. The following probes were used: S-*-DBACT-0193-a-A-18 (DBACT193) and S-*-DBACT-1027-a-A-18 (DBACT1027), specific for dominant bacteria affiliated with the ‘NC10’ phylum (Raghoebarsing et al., 2006); S-D-Bact-0338-a-A-18 (EUB338), specific for most *Bacteria* (Amann et al., 1990); L-C-gProt-1027-a-A-17 (GAM42a), specific for γ-Proteobacteria (Manz et al., 1992); L-C-bProt-1027-a-A-17 (BET42a), specific for β-Proteobacteria (Manz et al., 1992), S-Sc-aProt-0968-a-A-18 (ALF 968), specific for α-Proteobacteria, except Rickettsiales (Neef, 1997) and S-P-Planc-0046-a-A-18 (PLA 46), specific for Planctomycetes (Neef et al., 1998). The slides were examined using a Zeiss Axioplan II epifluorescence microscope with digital video camera and image analysis software (Axiovision, Zeiss, Germany).

### Analytical methods

For routine nitrite analysis, Merckoquant test strips (0–80 mg l⁻¹ nitrite; Merck, Germany) were used. Nitrite and nitrate samples (1 ml) from the batch experiments were measured colorimetrically (Kartal et al., 2006). Methane concentrations were determined by gas chromatography (Ettwig et al., 2008). To analyse N₂ (labelled and non-labelled) and O₂, gas chromatography was used (6890 series; Agilent, USA) with a Porapak Q column at 80°C (4 min) using helium as carrier gas (flow rate, 24 ml min⁻¹). The gas chromatograph was coupled to a mass spectrometer (Agilent 5975C inert MSD; Agilent, USA). Molecular masses of 32 Da (O₂) and 28–30 Da (N₂) were quantified and analysed using Enhanced MSD Chem Station software (version E.02.00.493; Agilent). The total protein content in the incubations was determined by the bicinchoninic acid assay (BCA, Pierce, USA) according to manufacturer’s protocol. Bovine serum albumin (BSA; Thermo Scientific, USA) was used as a standard. The protein content of the cell-free extracts was determined as described previously (Bradford, 1976) using a Bio-Rad protein assay kit (Bio-Rad, Veenendaal) with BSA as standard.

### Acknowledgements

We would like to thank Katharina F. Ettwig, Harry Harhangi and Laura van Niftrik for discussions. F.L. is supported by the Foundation for Applied Research (STW, project 07736), M.L.W. by a Horizon Grant (050–71-58), MSMJ by the European Research Council (ERC, 232937) and M.S. by ERC (242635).

### References

Amann, R.I., Binder, B.J., Olson, R.J., Chisholm, S.W., Devereux, R., and Stahl, D.A. (1990) Combination of 16S rRNA-targeted oligonucleotide probes with flow cytometry for analyzing mixed microbial populations. *Appl Environ Microbiol* **56**: 1919–1925.

Ayala-Castro, C., Saini, A., and Outten, F.W. (2008) Fe-S cluster assembly pathways in bacteria. *Microbiol Mol Biol Rev* **72**: 110–125.

Bianchi, A.A., and Baneyx, F. (1999) Hyperosmotic shock induces the σ32 and σE stress regulons of *Escherichia coli*. *Mol Microbiol* **34**: 1029–1038.

Bonin, P., and Gilewicz, M. (1991) A direct demonstration of ‘co-respiration’ of oxygen and nitrogen oxides by *Pseudomonas nautica*: some spectral and kinetic properties of the respiratory components. *FEMS Microbiol Lett* **80**: 183–188.

Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* 72: 248–254.

Cabeen, M.T., and Jacobs-Wagner, C. (2005) Bacterial cell shape. *Nat Rev Microbiol* 3: 601–610.

Dos Santos, P.C., Smith, A.D., Frazzon, J., Cash, V.L., Johnson, M.K., and Dean, D.R. (2004) Iron-sulfur cluster assembly: NifU-directed activation of the nitrogenase Fe protein. *J Biol Chem* 279: 19705–19711.

Ettwig, K.F., Shima, S., van de Pas-Schoonen, K.T., Kahnt, J., Medema, M.H., Op den Camp, H.J., *et al.* (2008) Denitrifying bacteria anaerobically oxidize methane in the absence of Archaea. *Environ Microbiol* 10: 3164–3173.

Ettwig, K.F., van Alen, T.A., van de Pas-Schoonen, K.T., Jetten, M.S.M., and Strous, M. (2009) Enrichment and molecular detection of denitrifying methanotrophic bacteria of the NC10 phylum. *Appl Environ Microbiol* 75: 3656–3662.

Ettwig, K.F., Butler, M.K., Le Paslier, D., Pelletier, E., Mangenot, S., Kuypers, M.M.M., *et al.* (2010) Nitrite-driven anaerobic methane oxidation by oxygenic bacteria. *Nature* 464: 543–548.

Farr, S.B., and Kogoma, T. (1991) Oxidative stress responses in *Escherichia coli* and *Salmonella typhimurium*. *Microbiol Rev* 55: 561–585.

Fournier, M., Aubert, C., Dermoun, Z., Durand, M.C., Moinier, D., and Dolla, A. (2006) Response of the anaerobe *Desulfovibrio vulgaris* Hildenborough to oxidative conditions: proteome and transcript analysis. *Biochimie* 88: 85–94.

Fridovich, I. (1978) The biology of oxygen radicals. *Science* 201: 875–880.

Fujiwara, T., and Fukumori, Y. (1996) Cytochrome *cb*-type nitric oxide reductase with cytochrome *c* oxidase activity from *Paracoccus denitrificans* ATCC 35512. *J Bacteriol* 178: 1866–1871.

Garcia-Horsman, J.A., Barquera, B., Rumbley, J., Ma, J., and Gennis, R.B. (1994) The superfamiliy of heme-copper respiratory oxidases. *J Bacteriol* 176: 5587–5600.

Geissmann, T.A., Teuber, M., and Meile, L. (1999) Transcriptional analysis of the rubrerythrin and superoxide dismutase genes of *Clostridium perfringens*. *J Bacteriol* 181: 7136–7139.

Gregory, E.M., and Fridovich, I. (1973) Induction of superoxide dismutase by molecular oxygen. *J Bacteriol* 114: 543–548.

Hanson, R.S., and Hanson, T.E. (1996) Methanotrophic bacteria. *Microbiol Rev* 60: 439–471.

Hoor, A.T.-T. (1975) A new type of thiosulphate oxidizing, nitrate reducing microorganism: *Thiomicrospira denitrificans* sp. Nov. *Neth J Sea Res* 9: 344–346. IN349, 347–350.

Hu, S., Zeng, R.J., Keller, J., Lant, P.A., and Yuan, Z. (2011) Effect of nitrate and nitrite on the selection of microorganisms in the denitrifying anaerobic methane oxidation process. *Environ Microbiol Rep* 3: 315–319.

Ideno, A., Yoshida, T., Lida, T., Furutani, M., and Maruyama, T. (2001) FK506-binding protein of the hyperthermophilic archaeum, *Thermococcus* sp. KS-1, a cold-shock-inducible peptidyl-prolyl cis-trans isomerase with activities

'trap and refold denatured proteins. *Biochem J* 357: 465–471.

Imlay, J.A., Chin, S.M., and Linn, S. (1988) Toxic DNA damage by hydrogen peroxide through the Fenton reaction in vivo and in vitro. *Science* 240: 640–642.

Kartal, B., Koleva, M., Arsov, R., van der Star, W., Jetten, M.S.M., and Strous, M. (2006) Adaptation of a freshwater anammox population to high salinity wastewater. *J Biotechnol* 126: 546–553.

Knittel, K., and Boetius, A. (2009) Anaerobic oxidation of methane: progress with an unknown process. *Annu Rev Microbiol* 63: 311–334.

Luesken, F.A., Zhu, B., van Alen, T.A., Butler, M.K., Rodriguez Diaz, M., Song, B., *et al.* (2011a) *pmoA* primers for detection of anaerobic methanotrophs. *Appl Environ Microbiol* 77: 3877–3880.

Luesken, F.A., van Alen, T.A., van der Biezen, E., Frijters, C., Toonen, G., Kampman, C., *et al.* (2011b) Diversity and enrichment of nitrite-dependent anaerobic methane oxidizing bacteria from wastewater sludge. *Appl Microbiol Biotechnol* 92: 845–854.

Manz, W., Amann, R., Ludwig, W., Wagner, M., and Schleifer, K.H. (1992) Phylogenetic oligodeoxynucleotide probes for the major subclasses of Proteobacteria – problems and solutions. *Syst Appl Microbiol* 15: 593–600.

Neef, A. (1997) Anwendung Der in Situ Einzelzell-Identifizierung Von Bakterien Zur Populationsanalyse in Komplexen Mikrobiellen Biozönosen. München: Technische Universität München.

Neef, A., Amann, R., Schlesner, H., and Schleifer, K.H. (1998) Monitoring a widespread bacterial group: in situ detection of planctomycetes with 16S rRNA-targeted probes. *Microbiology* 144: 3257–3266.

Op den Camp, H.J.M., Islam, T., Stott, M.B., Harhangi, H.R., Hynes, A., Schouten, S., *et al.* (2009) Environmental, genomic and taxonomic perspectives on methanotrophic Verrucomicrobia. *Environ Microbiol Rep* 1: 293–306.

Raghoebarsing, A.A., Pol, A., van de Pas-Schoonen, K.T., Smolders, A.J.P., Ettwig, K.F., Rijpstra, W.I.C., *et al.* (2006) A microbial consortium couples anaerobic methane oxidation to denitrification. *Nature* 440: 918–921.

Robertson, L.A., and Kuenen, J.G. (1984) Aerobic denitrification: a controversy revived. *Arch Microbiol* 139: 351–354.

Schonbaum, G.R., Bonner, W.D.R., Storey, B.T., and Bahr, J.T. (1971) Specific inhibition of the cyanide-insensitive respiratory pathway in plant mitochondria by hydroxamic acids. *Plant Physiol* 47: 124–128.

Schröder, E., and Ponting, C.P. (1998) Evidence that peroxiredoxins are novel members of the thioredoxin fold superfamily. *Protein Sci* 7: 2465–2468.

Strous, M., van Gerven, E., Kuenen, J.G., and Jetten, M.S.M. (1997) Effects of aerobic and microaerobic conditions on anaerobic ammonium-oxidizing (Anammox) sludge. *Appl Environ Microbiol* 63: 2446–2448.

Sztukowska, M., Bugno, M., Potempa, J., Travis, J., and Kurtz, D.M., Jr (2002) Role of rubrerythrin in the oxidative stress response of *Porphyromonas gingivalis*. *Mol Microbiol* 44: 479–488.

Wu, M.L., Ettwig, K.F., Jetten, M.S.M., Strous, M., Keltjens, J.T., and van Niftrik, L. (2011a) A new intra-aerobic metabolism in the nitrite-dependent anaerobic methane-oxidizing

1034 F. A. Luesken et al.

bacterium ‘Candidatus Methylomirabilis oxyfera’. *Biochem Soc Trans* **39**: 243–248.

Wu, M.L., de Vries, S., van Alen, T.A., Butler, M.K., Op den Camp, H.J.M., Keltjens, J.T., *et al.* (2011b) Physiological role of the respiratory quinol oxidase in the anaerobic nitrite-reducing methanotroph ‘Candidatus Methylomirabilis oxyfera’. *Microbiology* **157**: 890–898.

Zumft, W.G. (1997) Cell biology and molecular basis of denitrification. *Microbiol Mol Biol Rev* **61**: 533–616.

Supporting information

Additional Supporting Information may be found in the online version of this article:

Fig. S1. Example of the community composition and its abundant members visualized using Fluorescent *in situ* Hybridization (FISH) in the anoxic and oxic period. The general bacterial probe EUB338 (Cy5, dark blue) was combined with specific probe DBACT1027 for ‘NC10’ affiliated bacteria (Cy3, red).

A. In the anoxic period *M. oxyfera* dominates the population with 70–80%.

B. On *t* = 31.5 h (5 h after addition of 8% O₂) no differences could be observed compared with the anoxic period.

C. After 99 h (72 h after the addition of O₂) the population was still dominated by *M. oxyfera*. Scale bars = 20 μm.

Table S1. (XLS file): Illumina reads (75 nt) of *M. oxyfera* incubations under anoxic and oxic conditions mapped on the *M. oxyfera* genome (NCBI Accession No. GSE18535) using CLC bio genomics software.

Please note: Wiley-Blackwell are not responsible for the content or functionality of any supporting materials supplied by the authors. Any queries (other than missing material) should be directed to the corresponding author for the article.
