
Autophagy: Pathways for Self-Eating in Plant Cells

Yimo Liu$^{1}$ and Diane C. Basham$^{1,2}$

$^{1}$Department of Genetics, Development, and Cell Biology and Interdepartmental Genetics Program, Iowa State University, Ames, Iowa 50011

$^{2}$Plant Sciences Institute, Iowa State University, Ames, Iowa 50011; email: bassham@iastate.edu

---

Annu. Rev. Plant Biol. 2012. 63:215–37

First published online as a Review in Advance on January 10, 2012

The Annual Review of Plant Biology is online at plant.annualreviews.org

This article’s doi: 10.1146/annurev-arplant-042811-105441

Copyright © 2012 by Annual Reviews. All rights reserved

1543-5008/12/0602-0215$20.00

---

### Keywords

abiotic stress, programmed cell death, pathogen, TOR, senescence

---

### Abstract

Plants have developed sophisticated mechanisms to survive when in unfavorable environments. Autophagy is a macromolecule degradation pathway that recycles damaged or unwanted cell materials upon encountering stress conditions or during specific developmental processes. Over the past decade, our molecular and physiological understanding of plant autophagy has greatly increased. Most of the essential machinery required for autophagy seems to be conserved from yeast to plants. Plant autophagy has been shown to function in various stress responses, pathogen defense, and senescence. Some of its potential upstream regulators have also been identified. Here, we describe recent advances in our understanding of autophagy in plants, discuss areas of controversy, and highlight potential future directions in autophagy research.

Contents

INTRODUCTION 216
MACHINERY AND MECHANISMS OF AUTOPHAGY IN PLANTS 217
- Ubiquitin-Like Conjugation Systems 217
- ATG9 Cycling System 220
- PtdIns3K Complex 220
- Vti12 SNARE 221
FUNCTIONS OF AUTOPHAGY DURING ABIOTIC STRESS 221
FUNCTIONS OF AUTOPHAGY DURING DEVELOPMENT 222
FUNCTIONS OF AUTOPHAGY DURING PROGRAMMED CELL DEATH 223
- Functions of Autophagy During Developmental Programmed Cell Death 224
- Functions of Autophagy During Pathogen Infection 224
REGULATION OF AUTOPHAGY IN PLANTS 226
EVIDENCE FOR SELECTIVE AUTOPHAGY 228

INTRODUCTION

Autophagy (meaning “self-eating”) is a macromolecule degradation process in which cells recycle cytoplasmic contents when under stress conditions or during developmental transitions. The basic autophagy process is conserved among eukaryotes from yeast to animals and plants (9, 82, 135). Several types of autophagy have been described in many species, including microautophagy (84), macroautophagy (135), chaperone-mediated autophagy (92), and organelle-specific autophagy (103). In plants, microautophagy and macroautophagy have been shown to occur (11). Microautophagy involves the formation of a small intravacuolar vesicle called an autophagic body by invagination of the tonoplast, thus engulfing cytoplasmic components, whereas in macroautophagy, cytoplasmic autophagosomes enclose components to be degraded (11). Here, we focus on the macroautophagy pathway in plants, hereafter referred to as autophagy.

The principal characteristic of autophagy is the formation of double-membrane structures called autophagosomes (Figure 1). Upon induction of autophagy, an autophagosome forms around the material that is destined for degradation, and the autophagosome delivers this cargo to the vacuole. The outer membrane of the autophagosome fuses with the vacuole membrane, after which vacuolar hydrolases degrade both the cargo and the inner membrane in the vacuole.

Although plant autophagy was discovered several decades ago (80, 81, 120), we have only recently begun to understand its molecular mechanism. Most genes functioning in the autophagy pathway were first identified via mutagenesis studies in yeast. More than 30 autophagy-related genes have been identified in yeast; these genes can be divided into several functional groups (136): the Atg1-Atg13 kinase complex, Atg9 and associated proteins, a phosphatidylinositol 3-kinase (PtdIns3K) complex, and two ubiquitin-like conjugation systems. These studies in yeast have greatly facilitated the identification of homologous genes in plants that are required for autophagy and have provided direction for investigating their molecular functions.

In animals, autophagy is implicated in health and disease processes such as cancer, neurodegeneration, aging, and longevity (136); in plants, it is associated with a variety of stresses, pathogen infection, and senescence (9, 10, 38). Under normal conditions, basal autophagy functions as a housekeeping process to clear damaged or unwanted cytoplasmic contents, whereas under certain stresses, autophagy is upregulated (47, 108, 133). Autophagy-defective plants usually senesce earlier and are more susceptible to stress conditions compared with wild-type (WT) plants (Table 1). Several markers have been developed to study autophagy in plants, most commonly green

Autophagosome: a double-membrane structure formed upon autophagy induction; it engulfs portions of cytoplasm and delivers them to the vacuole for degradation

PP63CH09-Bas sham ARI 27 March 2012 9:10

![Diagram](#)

Figure 1

Pathway for autophagy in plant cells. Upon induction of autophagy, a double-membrane structure called an autophagosome forms around a portion of the cytoplasm (cargo). The autophagosome then transports the cargo to the vacuole. During the fusion process, the outer autophagosome membrane fuses with the vacuole membrane, and the remaining single-membrane structure (termed an autophagic body) is delivered inside the vacuole. The autophagic bodies are then broken down by vacuolar hydrolases, and the products are exported from the vacuole to the cytoplasm for reuse.

fluorescent protein (GFP)-ATG8s (19, 117, 142) and monodansylcadaverine (MDC) staining (19). These markers allow the rapid and straightforward detection of autophagy occurrence in plant cells by the specific labeling of autophagosomes. In this review, we summarize recent advances in our understanding of plant autophagy, including the essential machinery, regulation, and physiological roles, and briefly discuss the emerging evidence for selective autophagy.

**MACHINERY AND MECHANISMS OF AUTOPHAGY IN PLANTS**

**Ubiquitin-Like Conjugation Systems**

The yeast autophagy pathway requires two ubiquitin-like conjugation systems, which in turn involve two ubiquitin-like proteins, Atg8 and Atg12 (for reviews on mechanistic aspects, see 90, 135). After Atg8 is synthesized, its C-terminus is first cleaved by a cysteine protease, Atg4, to expose a glycine residue. The exposed

Table 1 Common phenotypes of Arabidopsis autophagy mutants and their corresponding affected genes

| Phenotype(s)                                                                 | Gene(s) involved                                                                 | Reference(s)                                                                 |
|-----------------------------------------------------------------------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
| Accelerated senescence, hypersensitivity to starvation conditions, slower growth | ATG2, ATG4s, ATG5, ATG6, ATG7, ATG8s, ATG9, ATG10, ATG12s, ATG18a, VTI12 | 16, 27, 28, 34, 36, 94, 97, 113, 117, 132, 142, 143                          |
| Stunted growth, increased anthocyanin production, decreased silique production, abnormal pollen germination | ATG6                                                                              | 28, 36, 98                                                                    |
| Hypersensitivity to drought and salt stresses                                  | ATG18a                                                                            | 74                                                                             |
| Hypersensitivity to oxidative stress                                           | ATG18a                                                                            | 107, 134                                                                       |
| Altered resistance to pathogen infection                                       | ATG2, ATG5, ATG6, ATG7, ATG9, ATG10, ATG18a                                    | 42, 62, 68, 94, 125, 143                                                      |

www.annualreviews.org • Autophagy in Plants 217

glycine is bound by an E1-like enzyme, Atg7, and the Atg8 is then transferred to an E2-like enzyme, Atg3. Finally, the Atg8 is conjugated to the membrane lipid phosphatidylethanolamine (PE). This Atg8-PE conjugation is reversible, as the protease Atg4 also cleaves Atg8 from PE; the released Atg8 can thus be recycled. Another ubiquitin-like protein, Atg12, is also activated by Atg7; it is then transferred to an E2-like enzyme, Atg10, and finally conjugated to Atg5. The Atg12-Atg5 conjugate further interacts with a coiled-coil protein, Atg16, to form a tetrameric Atg12-Atg5·Atg16 complex (in which the hyphen indicates a covalent bond and the dot indicates a noncovalent interaction) via Atg16 oligomerization. This complex is also essential for autophagy. Atg8-PE conjugates and Atg12-Atg5·Atg16 complexes reside on the preautophagosomal structure (PAS), and upon induction of autophagy, both of them localize to the expanding phagophore. Atg8-PE conjugates show an equal localization on both the inner and outer autophagosome membrane, whereas the Atg12-Atg5·Atg16 complexes associate mainly with the outer membrane (135).

In *Arabidopsis*, all of the counterparts of the two yeast conjugation systems are also well conserved (Figure 2). In contrast to yeast, which has a single *ATG8*, *ATG4*, and *ATG12* gene, *Arabidopsis* contains nine members of the *AtATG8* family (*AtATG8a–AtATG8i*), two members of the *AtATG4* family (*AtATG4a*, *AtATG4b*), and two members of the *AtATG12* family (*AtATG12a*, *AtATG12b*) (27, 34).

Both *AtATG4s* are ubiquitously expressed in plants, and their expression levels are elevated after nitrogen starvation (142). Phenotypic analysis of an *Atatg4a4b* double mutant showed that it displays the phenotypes of early senescence and reduced silique production that are typical of autophagy defects (142). There are contradictory reports on whether the growth of primary roots and lateral roots in the *Atatg4a4b* double mutant are arrested under nitrogen-limited conditions (16, 142). As in yeast, AtATG4s function as proteases to process the C-terminus of AtATG8s, and autophagosomes cannot form in the *Atatg4a4b* double

mutant (142). These findings indicate that the *AtATG4s* are essential for plant autophagy and that they function similarly to *ATG4* in yeast.

Under normal growth conditions, all of the nine *AtATG8s* are expressed throughout the plant, although different members show distinct expression patterns (108, 142), implying that each member may have a distinct function during development or under various stress conditions. Several independent studies have demonstrated the existence of the AtATG8 ubiquitin-like conjugation system in *Arabidopsis* (16, 27, 29, 117, 142). The C-terminus of the AtATG8s is cleaved by AtATG4; bound to an E1-like enzyme, AtATG7; transferred to an E2-like enzyme, AtATG3; and finally conjugated to PE (Figure 2). ATG8 orthologs have also been identified in *Chlamydomonas* (96), rice (17, 112), and maize (17).

In mammals, ATG8s are divided into two subfamilies according to their protein sequence similarity: the LC3 subfamily (four members) and the GABARAP/GATE-16 subfamily (four members). The LC3 subfamily is involved in elongation of the phagophore membrane, whereas the GABARAP/GATE-16 subfamily is involved in autophagosome maturation (129). It has been suggested that in *Arabidopsis*, AtATG8s bind to microtubules, indicating the possible involvement of the cytoskeleton in plant autophagy (57). In one study, overexpression of a GFP-AtATG8f-HA fusion protein in *Arabidopsis* plants enhanced growth and altered stress responses, and cytokinin-mediated regulation of root architecture and root-shoot communication were affected (109); this suggests that *AtATG8s* may have a physiological role in responding to hormones and abiotic stresses. Recently, several studies showed that in yeast and animals, ATG8/LC3 may be crucial during selective autophagy in recognition of a specific protein motif, the ATG8-interacting motif (AIM) (89). Atg19 and p62 use the same WXXL motif to interact with ATG8/LC3 family members (88). NBR1 (neighbor of *BRCA1* gene), Atg32, and Nix also use a similar motif to bind to the ATG8/LC3 family (89). So far, in plants, two proteins have been found to use an AIM to

Figure 2

Two ubiquitin-like conjugation systems in *Arabidopsis*. (Right) The AtATG8 conjugation system. The C-terminus of the AtATG8 proteins is cleaved by AtATG4; bound to an E1-like enzyme, AtATG7; transferred to an E2-like enzyme, AtATG3; and finally conjugated to phosphatidylethanolamine (PE). The orange Xs represent C-terminal sequences of AtATG8s that are cleaved by AtATG4s (142). There are eight members in the AtATG8 family; AtATG8h and AtATG8i do not possess additional amino acids after glycine and are not thought to be subject to cleavage by AtATG4. (Left) The AtATG12 conjugation system. AtATG12 proteins first bind to an E1-like enzyme, AtATG7; are then transferred to an E2-like enzyme, AtATG10; and are finally conjugated to AtATG5. The AtATG12-AtATG5 conjugate may also function as an E3-like enzyme during the AtATG8-PE conjugation. Whether AtATG12-AtATG5 interacts with AtATG16 to form a tetrameric ATG12-ATG5·ATG16 complex (as it does in yeast) and whether the ATG12-ATG5·ATG16 complex is also able to function as an E3-like enzyme are still unknown (*dashed lines*).

interact with ATG8s: AtNBR1 and AtTSPO (tryptophan-rich sensory protein) (116, 122). AtNBR1 is a selective autophagic substrate that binds to AtATG8s via an AIM motif (116). The stress-induced AtTSPO is degraded by the autophagy pathway, and the AIM-like motif is required for this process (122).

In *Arabidopsis*, there are two members of the AtATG12 gene family: AtATG12a and AtATG12b (16, 27, 34). These two genes share high amino acid similarities (95%) and have functional overlap; however, *AtATG12b* is more important during basal autophagy, whereas *AtATG12a* is more important during induced autophagy (16). All of the components of the AtATG12 conjugation system have been characterized (16, 27, 97, 117). *Atatg5*, *Atatg7*, *Atatg10*, and *Atatg12a12b* mutants are all autophagy-defective and display the typical phenotypes of early senescence and hypersensitivity to nutrient-limited conditions (16, 27, 97, 115, 117). Just as in yeast, the
AtATG12-AtATG5 conjugate functions in the formation of the AtATG8-PE conjugate (Figure 2) (16, 29, 33). In rice, two *ATG10* genes have been identified, *OsATG10a* and *OsATG10b* (107), which play an important role in autophagosome formation and in survival during oxidative stress (107). Maize ZmATG12 has also been identified and found to interact with ZmATG7 (17). These findings suggest that the ATG12 conjugation system is conserved in plants and is essential for the plant autophagy pathway.

### ATG9 Cycling System

The source of the lipid membrane during the formation of autophagosomes is a major puzzle. In yeast, the phagophore is thought to be generated at a single perivacuolar site, the PAS (114). An integral membrane protein, Atg9, has been proposed to deliver lipid to the forming autophagosomes (39, 131). Atg9 localizes to the PAS and several non-PAS punctate structures; it is postulated that Atg9 cycles between the PAS and non-PAS structures. In yeast, the Atg9 non-PAS puncta apparently consist of tubulovesicular clusters adjacent to mitochondria, and the autophagosome membrane forms de novo (79). In contrast, mammalian cells may possess multiple PASs, and mAtg9 localizes to a juxtanuclear region corresponding to the trans-Golgi network and late endosomes (145). It has been proposed that in animals, the autophagosome membrane derives from several sources, including the endoplasmic reticulum (ER) (37, 139), mitochondria (32), and plasma membrane (100). In plants, however, the membrane origin is not known.

In yeast, the movement of Atg9 from non-PAS puncta to the PAS requires the function of Atg11, Atg23, and Atg27 (131, 135). The movement of Atg9 from the PAS to non-PAS puncta involves the Atg1-Atg13 kinase complex, Atg2, Atg18, and PtdIns3K complex I (131). Defects in any of these components lead to the accumulation of Atg9 at the PAS. Atg2 and Atg18 are peripheral membrane proteins, both of which can interact with Atg9 (131, 135). Atg18 can bind to phosphatidylinositol 3-phosphate [PtdIns(3)P] and phosphatidylinositol 3,5-bisphosphate [PtdIns(3,5)P₂], and binding to PtdIns(3)P protects Atg8-PE from unregulated cleavage by Atg4 and is required for autophagy (85).

In *Arabidopsis*, although homologs of *ATG1, ATG2, ATG9, ATG13,* and *ATG18* have been found (6), of these only the *ATG2, ATG9,* and *ATG18* homologs have been characterized in detail. *Arabidopsis* has a single *AtATG9* gene and a single *AtATG2* gene, which are expressed ubiquitously throughout the plant (34, 125, 143). Both *Atatg9* and *Atatg2* knockout mutants display typical autophagy-defective phenotypes during senescence and stress conditions (34, 125, 143). There are eight members in the *AtATG18* gene family (*AtATG18a–AtATG18h*) (34, 132); each member has a different expression pattern, and only one (*AtATG18a*) shows an increased transcript level in starvation conditions and during senescence (132). *AtATG18a* expression is also upregulated and is required for autophagy during oxidative, salt, and osmotic stresses (74, 133, 134). RNA interference (RNAi)-*AtATG18a* plants are autophagy-defective and display a typical autophagy phenotype, and are more sensitive to various stress conditions (Figure 3) (74, 132–134). The identification of these counterparts to yeast genes in *Arabidopsis* may suggest conserved roles. However, their subcellular localizations and specific functions during autophagosome biogenesis remain to be determined.

### PtdIns3K Complex

In yeast, a PtdIns3K complex is required for autophagy and localizes to the PAS (135). It includes a class III PtdIns3K, Vps34; a serine/threonine kinase, Vps15; Vps30/Atg6; and Atg14. Vps15 is required for the membrane association of Vps34. Atg14 is thought to connect Vps34 and Vps30/Atg6 (131, 135), whereas the function of Vps30/Atg6 is not clear. The PtdIns3K complex has been postulated to recruit PtdIns(3)P binding proteins, including Atg18, to the PAS (135).
although this phenotype may be unrelated to its role in autophagy (28, 36, 98). Plants with disrupted AtATG6 have increased anthocyanin production, short roots, early leaf senescence, dwarfism, fewer flowers, and low fertility (36, 94, 98). Also, AtATG6 antisense plants fail to limit the pathogen-associated cell death response (94). In tobacco (*Nicotiana benthamiana*), NbATG6 is required for pathogen-induced autophagy and regulation of programmed cell death (PCD) (73). There is one VPS15 homolog in the *Arabidopsis* genome (6), but its function has not been studied in detail; ATG14 appears to be missing. Moreover, how these genes are coordinated to regulate autophagy in plants is still unclear.

**Vti12 SNARE**

After the double-membrane autophagosome forms, it is delivered to the vacuole for further degradation. This process requires the fusion of the autophagosome membrane with the vacuole membrane. Several components of the SNARE (soluble N-ethylmaleimide-sensitive-factor attachment protein receptor) machinery are required for the fusion process and autophagosome membrane expansion in yeast (86, 135). Several homologs of the VTI1 vesicle SNARE are present in *Arabidopsis* (105), and one of these homologs, VTI12, is thought to function during autophagy (113). An *vti12* mutant is more sensitive to starvation conditions and shows an accelerated senescence phenotype, suggesting a role in plant autophagy (113). The partner target SNAREs of VTI12 that may also function in autophagy are not known.

**FUNCTIONS OF AUTOPHAGY DURING ABIOTIC STRESS**

The first and most common abiotic stress shown to induce autophagy was nutrient deprivation (27, 34, 97, 117, 132). Autophagy-defective plants display accelerated starvation-induced chlorosis, most likely because autophagy is required for nutrient remobilization during the starvation response. In recent

www.annualreviews.org • Autophagy in Plants 221

to complete their life cycles but have an early-
senescence phenotype even under nutrient-rich
conditions (27, 34, 94, 97, 98, 117, 132, 142),
which suggests that autophagy has some func-
tion under these conditions. Because autophagy
recycles cytoplasmic materials, it is under-
standable that it functions during senescence
and germination, the two large-scale nutrient
remobilization processes in the plant life cycle.
About 80% of leaf nitrogen is contained in
chloroplasts (78); during senescence, plants re-
cycle nutrients from senescing leaf chloroplasts
to the newly forming organs such as developing
seeds (31). Small vesicles containing chloroplast
stromal components [RuBisCO-containing
bodies (RCBs); for more discussion, see Ev-
idence for Selective Autophagy, below] have
been shown to be degraded by the autophagy
pathway (48, 49, 123, 124). In *Arabidopsis*,
chloroplasts normally decrease in both size and
number during senescence, which does not oc-
cur in an autophagy mutant (124). This raises an
interesting question regarding the relationship
between autophagy and chloroplast breakdown
during leaf senescence. If autophagy is involved
in chloroplast degradation, delayed senescence
might be predicted if autophagic degradation
is impaired. However, autophagy-defective
mutants unexpectedly show early-senescence
phenotypes, with accelerated loss of chloro-
phyll and chloroplast proteins. This indicates
that an *ATG* gene-independent mechanism
exists that is at least partially responsible for
chloroplast recycling (11, 71)—for example,
the action of chloroplast-localized proteases.
One possibility is that autophagy is normally
activated early in the senescence pathway to be-
gin the protein degradation process, leaving the
photosynthetic machinery intact to continue
photosynthesis. Additional catabolic pathways,
including chloroplast-localized pathways for
chlorophyll degradation, would be activated
later in the senescence process (43). It is
therefore hypothesized that when autophagy is
blocked, the autophagy-independent pathways
are activated prematurely, leading to early
breakdown of chloroplast components and
premature senescence.

**FUNCTIONS OF AUTOPHAGY**
**DURING DEVELOPMENT**

Although autophagy has been investigated
most extensively during stress conditions,
plants maintain a basal level of housekeeping
autophagy even under favorable growth condi-
tions (47, 108, 138). This basal autophagy may
function to eliminate damaged proteins and
organelles, which are continually generated
under normal growth conditions (11). Almost
all of the autophagy-defective mutants are able

RuBisCO-containing
body (RCB): a small
vesicle containing only
stromal components
that pinches off from
the chloroplast during
senescence

Figure 4

Autophagy is induced under salt and osmotic stresses. A fluorescence
microscope was used to visualize autophagy induction in 7-day-old green
fluorescent protein (GFP)-AtATG8e transgenic *Arabidopsis* roots. After
treatment with 0.16-M NaCl or 0.35-M mannitol, numerous GFP-AtATG8e-
labeled autophagosomes appeared, whereas few were present in control
conditions. Arrows indicate GFP-labeled autophagosomes. Figure adapted
from Reference 74.

years, autophagy has been shown to be a rather
general response to a variety of abiotic stresses
(9). When oxidative stress is introduced using
H₂O₂ or methyl viologen, autophagy is quickly
induced. The autophagy-defective RNAi-
AtATG18a plants are more sensitive to methyl
viologen treatment and accumulate higher
levels of oxidized proteins (133, 134); similar
phenomena have also been observed in a rice
Osatg10 mutant (107). These results suggest
a role for autophagy in degrading oxidized
proteins in plant cells. Autophagy is also
required for plant tolerance to drought and salt
stresses (Figures 3 and 4) (74), implying that
autophagy plays a role in removing damaged
proteins or organelles during these stresses.
AtTSPO, which is an abscisic acid-induced
protein, has been shown to be degraded by the
autophagy pathway, indicating the involvement
of autophagy in responses to abscisic acid in
plants (122).
To ensure seed viability, plants synthesize large amounts of seed storage proteins and deposit them into protein storage vacuoles (PSVs) during seed development. Upon seed germination, these proteins are degraded to support the growth of newly forming organs (46). In wheat, during seed development, the prolamin storage proteins are synthesized in the ER and form ER protein bodies; the direct transport of prolamins from ER to PSVs involves a pathway that, under electron microscopy, resembles autophagy (70). In *Vigna mungo*, breakdown of starch granules during seed germination is also associated with the autophagy pathway (119). In *Arabidopsis*, several *ATG* genes are upregulated during seed maturation and desiccation (5), but no obvious defect in seed formation or germination has been observed in the *ATG* mutants under normal growth conditions. During salt stress, the germination of RNAi-*AtATG18a* seeds lags behind that of WT seeds (74), implying that autophagy may be involved in salt tolerance during seed germination. In maize aleurone cells, the prolamin storage protein zein was recently found to be delivered from ER to PSVs in atypical prevacuolar compartments. These zein-containing compartments have multilayered membranes and engulfed cytoplasmic materials, thus morphologically resembling autophagosomes (104). However, they are neither surrounded by a double membrane nor decorated by the Atg8 protein, suggesting that there is an atypical autophagy pathway to deliver storage proteins from ER to PSVs that is independent of *ATG8* (104).

It has been previously suggested that autophagy plays a role in vacuole biogenesis (11, 80). However, *ATG* mutants do not have vacuole defects in either nutrient-limited or nutrient-rich conditions. Research in tobacco (BY-2) miniprotoplasts (protoplasts lacking the large central vacuole) demonstrated that the autophagy pathway involved in vacuole biogenesis is mechanistically distinct from the stress-induced and basal autophagy observed throughout the plant growth cycle (137). In miniprotoplasts, after researchers used

cysteine protease inhibitors to inhibit vacuolar protein degradation, cytoplasmic contents were detected in the newly generated vacuole, indicating the occurrence of autophagy during this process. However, inhibitors typically used to block stress-induced and basal autophagy did not affect the autophagy in miniprotoplast vacuole formation, suggesting that this is an atypical autophagy with a mechanism different than that of stress-induced and basal autophagy (137). In animals, alternative autophagy pathways that use only part of the canonical autophagy machinery to form functional autophagosomes have been described, including Atg5/Atg7-independent autophagy (87) and Beclin-1-independent autophagy (106). However, whether such noncanonical autophagy pathways exist in plants is still unknown.

**FUNCTIONS OF AUTOPHAGY DURING PROGRAMMED CELL DEATH**

In animals, PCD can be morphologically divided into three types: apoptosis, autophagic cell death, and necrosis (59). However, there is not an absolute distinction between the different forms of cell death, as several examples display mixed features (59). To further complicate analysis, in many cases that are defined as autophagic cell death, the studies show only that PCD occurred with concomitant activation of autophagy, rather than that the PCD process was carried out by autophagy (59). In plant cells, the typical animal apoptosis process does not seem to occur, because the presence of a cell wall prevents the dead cell from being engulfed by adjacent cells. Plant PCD seems to share some features with both apoptosis and autophagy in animals (for a more comprehensive review on plant PCD, see 63, 121). For example, the cell takes up organelles into the vacuole, the organelles are degraded, the vacuolar size increases, and eventually the vacuole lyses; these are all characteristics of autophagic cell death. However, the cell also undergoes chromatin condensation, nuclear fragmentation, and DNA laddering, which are characteristics
of apoptotic cell death. In mammalian PCD,  
a group of cysteine proteases called caspases  
are important regulators of apoptosis (21).  
True caspase homologs have not been identified in plants (101), although several groups  
have found caspase-like activities. In barley,  
VEIDase was found to have a caspase-like activity; it is localized to autophagosomes, linking  
the caspase activity to autophagic PCD (12). In  
*Arabidopsis*, type I metacaspases were also  
shown to control cell death (18). Collectively,  
known features of PCD in plants indicate that  
classification is not as clear as it is in animals.  
Plant PCD seems to have conserved functions  
but also unique characteristics. Research on  
plant PCD has focused mainly on two categories: PCD during normal development and  
PCD during the hypersensitive response (HR)  
triggered by pathogen infection. Evidence  
implies that autophagy plays critical roles in  
both processes.

**Functions of Autophagy During**  
**Developmental Programmed**  
**Cell Death**

There are many well-known examples of PCD  
during various developmental stages (121), including xylem and phloem formation, senescence, shoot elimination, leaf shape formation,  
and pollen germination and tube growth. Morphological studies have shown that most of  
these processes involve the gradual disappearance  
of organelles and eventually the collapse  
of the tonoplast and plasma membrane, and it  
has been suggested that autophagy is responsible for the cell death (121). In wheat, accelerated plant development caused by long-day  
growth conditions can trigger developmentally  
generated sugar starvation, which in turn initiates autophagic cell death of florets and leads  
to decreased fertile floret numbers (30). The involvement of autophagy was also demonstrated  
by the analysis of PCD during xylem fiber maturation in *Populus*. Not only does this type of  
PCD morphologically resemble autophagic cell  
death, but several autophagy-related genes are  
upregulated (20).

Recently, one study (61) showed that autophagy also functions in xylem tracheary element (TE) differentiation in *Arabidopsis*. In  
this study, both protoxylem and metaxylem  
cell numbers decreased significantly in an  
*Arabidopsis atg5* mutant compared with WT  
plants, suggesting that ATG5-dependent autophagy is involved in xylem development. LysoTracker Green staining showed that autophagy is induced during xylem differentiation. In addition, the study showed that a small GTP-binding protein, RabG3b, which was previously identified as a salicylic acid (SA)-responsive protein, is a positive regulator of autophagy during TE differentiation. These results demonstrate that RabG3b functions as a component of autophagy and regulates TE differentiation by activating the PCD process.

**Functions of Autophagy During**  
**Pathogen Infection**

It has been suggested that autophagy can serve  
both a “prosurvival” and a “prodeath” role upon  
pathogen infection, depending on the type of  
pathogen, the type of immune factor involved,  
and the age of the plant (38, 42, 67, 68, 140,  
144). Phytopathogens can be divided into two  
types based on their lifestyles: necrotrophic  
and biotrophic. Necrotrophic pathogens release toxins and lytic enzymes to kill the host  
cell, whereas biotrophic pathogens depend on  
their host cell to survive (67). In *Arabidopsis*, infection with the necrotrophic fungal pathogen *Botrytis cinerea* induces autophagy in both the infected and the surrounding areas (62). Compared with WT plants, autophagy-defective mutants are hypersensitive to *Botrytis cinerea* and *Alternaria brassicicola*: They have more dead cells, the degradation of certain proteins accelerates (62), and they develop spreading necrosis (68). Interestingly, the transcription factor WRKY33, which is important for plant resistance to necrotrophic pathogens, interacts with the autophagy protein ATG18a in the nucleus (62). These results clearly show that autophagy functions in a prosurvival role during necrotrophic pathogen infection.
The role of autophagy upon biotrophic pathogen infection is more complicated. When tobacco (*Nicotiana benthamiana*) plants are infected with tobacco mosaic virus, autophagy is induced in both the infected and the uninfected area (73). *BECLIN1/ATG6/VPS30*-silenced plants are capable of initiating HR-PCD; however, the PCD is unrestricted and spreads to healthy uninfected tissue and distal leaves (73). Similar results were later obtained in *Arabidopsis ATG6* RNAi plants and *atg5* knock-out mutants (94, 143). After infection with an avirulent bacterial pathogen, *Pseudomonas syringae* pv. tomato (*Pst*) DC3000 (*avrRPM1*), the HR-PCD escapes from the infected area and spreads to adjacent healthy tissues (94, 143). These results suggest that autophagy functions to restrict the HR-PCD and prevent runaway cell death, thus serving a prosurvival role. However, another study has reported that cell death is suppressed and resistance is increased in *atg7* and *atg9* mutants infected with the avirulent bacterial strain *Pst* DC3000 (*AvrRps4*) (42). Similarly, the *atg2* mutant displays enhanced powdery mildew resistance (125). Additional *Arabidopsis* autophagy-defective plants were recently tested for their immunity response to different types of pathogens (68); compared with WT plants, the *atg*-defective plants all showed enhanced resistance to the virulent bacterial strain *Pst* DC3000 or to the avirulent *Hyaloperonospora arabidopsidis*, possibly owing to an elevated SA level in the *atg* mutant (discussed below) (42, 68). These results indicate that autophagy serves a prodeath role upon biotrophic pathogen infection. It has been suggested that the contradictory results during biotrophic pathogen infection are caused by the age differences in plants used in the experiments, which is related to SA signaling (see below). The spreading of the PCD was observed only in older leaves of older plants (7–8 weeks old), whereas no difference was observed in younger leaves or younger plants (4–5 weeks old) (38, 62, 143, 144).

The difference in autophagic response to necrotrophic and biotrophic pathogens may be due to the engagement of different plant hormones. The defense response to necrotrophic pathogens depends on jasmonic acid (JA) and ethylene signals, whereas the defense response to biotrophic pathogens involves the SA signal (52, 58, 111). Because the *atg* mutants display altered immune response to both types of pathogens, several research groups have analyzed hormone levels and the expression of hormone-regulated genes during pathogen infection in both WT and autophagy-defective plants (62, 68, 143). Compared with uninfected WT plants, uninfected *atg5*, *atg7*, and *atg18a* mutants had significantly higher basal JA levels and higher transcription levels of a JA-regulated defense gene, *PDF1.2* (62, 143). Upon infection with the necrotrophic fungus *Botrytis*, the *PDF1.2* transcription level was greatly reduced in the *atg* mutants compared with WT plants, despite the elevated basal level. This result suggests that autophagy negatively regulates JA-mediated *PDF1.2* gene expression in healthy plants but positively regulates its level in infected plants (62).

SA was also elevated in uninfected *atg* mutants (62, 68, 143). When *atg* mutants were crossed with several SA-signaling defective mutants, both the early-senescence phenotype and the unrestricted PCD during *Pst avrRpm1* infection were rescued (143). Crossing the *atg* mutants to JA-related mutants did not produce the same result, suggesting a JA-independent mechanism (143). Upon treatment with the SA agonist benzo-(1,2,3)-thiadiazole-7-carbothioic acid, autophagy was induced in WT plants but not in the *npr1* mutant (a downstream component of SA signaling), indicating that SA-induced autophagy is dependent on NPR1 (143). Collectively, these results demonstrate that the premature senescence and the infection-induced spreading of HR-PCD are the result of an increased basal SA level in the *atg* mutants, and that in WT plants autophagy functions to eliminate SA via a negative-feedback loop, which in turn suppresses the senescence and unrestricted PCD (141, 143).

The discovery of the involvement of SA signaling may help to explain three phenomena.
First, the increased resistance to biotrophic pathogen infection in *atg* mutants may result from the constitutively slightly enhanced SA level and therefore increased expression of SA-induced defense genes (67). Second, as mentioned above, the spreading of HR-PCD in *atg* mutants has been observed only in the older leaves of older plants, and the increased resistance has been observed in younger leaves and younger plants (42, 94, 143). It is possible that in the younger plants, the *atg* mutants have only a slightly increased SA level, which is responsible for the increased resistance during pathogen infection. However, the older *atg* mutants generate more SA, which leads to the phenotypes of early senescence and hypersusceptibility to infection (38, 144). Autophagy can serve both a prodeath and a prosurvival role during biotrophic pathogen infection, and this determination is age dependent. Third, the role of autophagy during starvation stress is well understood: It helps to recycle nutrients, leading to increased sensitivity of *atg* mutants to starvation stress. However, the role of autophagy in nutrient-rich conditions is still under investigation. It has long been known that *atg* mutants have an early-senescence phenotype, but the underlying mechanism is unclear. Now it has been shown that senescence is modulated by the SA level, and autophagy functions to eliminate excessive SA, thus preventing premature senescence (140, 141).

and other eukaryotes have only one TOR gene (95). In both yeast and animals, TOR forms two different functional complexes, TORC1 and TORC2, each of which contains distinct TOR binding partners (75). TORC2 controls spatial cell growth by regulating polarization of the actin cytoskeleton in a rapamycin-insensitive manner, whereas TORC1 is sensitive to rapamycin and controls temporal cell growth by promoting translation, transcription, ribosome biogenesis, and nutrient transport and negatively regulating autophagy (25, 75, 135). In yeast, TORC1 regulates the Atg1-Atg13-Atg17 complex depending on nutritional status. Under normal conditions, TORC1 is activated and causes the phosphorylation of Atg13; this hyperphosphorylated form of Atg13 has a low affinity for Atg1 and Atg17. Upon starvation or rapamycin treatment, TORC1 is inactivated and causes the dephosphorylation of Atg13; this hypophosphorylated form of Atg13 has a higher affinity for Atg1 and Atg17. The Atg1 kinase activity is activated after its interaction with Atg13 and Atg17, leading to the induction of autophagy (54, 135). Surprisingly, even though the key components of this regulatory pathway are conserved, in mammals and *Drosophila* the regulation and relationships between TORC1, Atg1, and Atg13 are divergent (40). Intriguingly, despite TOR being a key regulator for nutrient stress–induced autophagy, studies in HeLa cells and primary fibroblasts suggest that autophagy induced upon pathogen infection may be TOR independent (126).

The TOR homolog in *Arabidopsis* has been identified (83), but the investigation of TOR function is impeded for two reasons: First, disruption of the *AtTOR* gene is embryonic lethal (83), making the study of its postembryonic function difficult; and second, the *Arabidopsis* FKBP12 protein does not bind to rapamycin, making it insensitive to rapamycin, which means that this inhibitor cannot be used to study the TOR pathway in this species (77). Several strategies have been developed to overcome these problems. Transgenic *Arabidopsis* plants expressing yeast ScFKBP12 showed rapamycin sensitivity and displayed

## REGULATION OF AUTOPHAGY IN PLANTS

A milestone in our understanding of the regulation of autophagy was the identification of the TOR (target of rapamycin) kinase (13, 60). TOR is a PtdIns3K-related kinase that functions as a serine/threonine protein kinase and as a nutrient sensor that integrates multiple upstream signals (45). The antibiotic rapamycin inhibits TOR kinase activity by forming a ternary complex with the FRB (FKBP12 and rapamycin binding) domain of TOR and the FKBP12 protein (15). In yeast, two TOR genes have been identified; plants, mammals,

rapamycin-dependent arrest of root growth (110). *Arabidopsis* transfer-DNA insertion lines with increased *AtTOR* expression and RNAi lines with decreased *AtTOR* expression showed that root and shoot growth are correlated with TOR expression level, indicating a role in growth regulation (23).

Recently, a mutant screen for suppressors of the root-hair cell-wall-formation mutant *lrx1* led to identification of the *rol5* mutant (66). ROL5 is structurally and functionally similar to the yeast Ncs6 protein, which is a component of the yeast TOR pathway. The authors suggested that the suppression of *lrx1* by *rol5* is based on alteration of TOR signaling, because treating *lrx1* plants expressing yeast FKBP12 with rapamycin also relieves the *lrx1* phenotype. This finding reveals that the TOR pathway is involved in cell wall formation in *Arabidopsis* (66).

Two studies provided direct evidence that TOR is a regulator of autophagy in photosynthetic species (72, 96). The green alga *Chlamydomonas reinhardtii* is sensitive to rapamycin, and autophagy was induced upon rapamycin treatment (96). In *Arabidopsis*, RNAi-*AtTOR* plants showed constitutive activation of autophagy (72). These results indicate that TOR function is conserved and that this factor serves as a negative regulator of autophagy in plants (Figure 5).

As mentioned above, TOR works in two different complexes. Although the TORC2 subunits remain to be identified in plants, some of the TORC1 binding partners have been identified (25). These include Raptor (35), which recruits substrates and presents them to TOR for phosphorylation, and LST8 (128), which stabilizes the TOR complex. In animal cells, rapamycin inhibits TOR activity partially by uncoupling the TOR/RAPTOR interaction (93). Two RAPTOR homologs have been identified in *Arabidopsis*: *AtRAPTOR1A* and *AtRAPTOR1B* (4, 22). Both of them are expressed in growing tissues throughout the plant, although *AtRAPTOR1B* is the most highly expressed isoform (4, 22). Disruption of *AtRAPTOR1A* has no obvious phenotype,

**Figure 5**

Potential TOR signaling pathways in plants. The TORC1 complex, including TOR, RAPTOR, and LST8, senses and integrates multiple upstream signals such as nutrient starvation, insulin-like growth factors, or other stresses (*dashed arrows*). TOR may serve as a negative regulator of autophagy. Some TORC1 substrates have been identified, including AML1, EBP1, Tap46 (a regulatory subunit of PP2A), and S6K. These substrates may function to control translation, cell growth, and possibly autophagy.

whereas disruption of *AtRAPTOR1B* has been reported to have two contradictory phenotypes. One study showed that loss of *AtRAPTOR1B* causes seed abortion and a complete arrest of embryo development at a preglobular stage (22). Another showed that an *Atraptor1B* mutant is viable but has defects in both root and shoot growth, resulting in delayed development (4). An *Atraptor1A Atraptor1B* double mutant exhibited normal embryonic development but was unable to maintain postembryonic meristem-driven growth (4). This phenotype discrepancy can potentially be explained by a variation in growth conditions, because the addition of 1% sucrose to the growth medium partially rescued this phenotype (4). Another TOR binding partner,
LST8, has been identified in *Chlamydomonas reinhardtii* (24). CrTOR and CrLST8 exist in a large complex in which CrLST8 interacts with the CrTOR kinase domain; together, they colocalize to ER membranes. *CrLST8* is able to complement the yeast *lst8* mutant, implying a conserved function for this protein (24).

As a kinase, TOR signals through a combination of direct phosphorylation of downstream targets and repression of phosphatase activity (99). Recent research has shown that TOR controls embryogenesis and postembryonic development through its kinase domain in *Arabidopsis* (102). To date, several potential TOR substrates have been identified in plants. As mentioned above, the Atg1-Atg13-Atg17 complex is a TORC1 substrate, and in yeast and animals it is regulated by TORC1 depending on the nutritional status. In *Arabidopsis*, three putative ATG1 homologs and two putative ATG13 homologs have been identified (25); however, their functions and interactions have not been experimentally studied. In yeast, autophagy is regulated through PP2A (protein phosphatase type 2A), and a regulatory subunit of PP2A, Tap42/α4, has been shown to be a downstream substrate of the Tor protein. Tap42 is phosphorylated and tightly associates with PP2A under nutrient-rich conditions; upon starvation or rapamycin treatment, Tap42 is dephosphorylated and dissociates from PP2A (135). A plant homolog of Tap42/α4, Tap46, has been identified in both *Arabidopsis* and tobacco (*Nicotiana tabacum*) (2). Tap46 interacts with PP2A and is phosphorylated by TOR, suggesting that Tap46 is a direct substrate of TOR. Loss of Tap46 function results in growth arrest and activation of autophagy. These findings suggest that Tap46 functions as a component of the TOR signaling pathway (2).

Mei2 is a meiosis signaling molecule that has been suggested to be a potential TOR substrate in yeast (127). In *Arabidopsis*, the Mei2 homolog AML1 (*Arabidopsis Mei2-like1*) interacts with RAPTOR1B in a yeast two-hybrid assay, which implicates AML1 as a candidate substrate of TOR (3). EBP1 (ErbB-3 epidermal growth factor receptor binding protein) is a nucleolar and cytoplasmic regulator of ribosome assembly and translation; it has been shown to regulate cell growth and proliferation in plants (44). In *Arabidopsis*, the expression of the *EBP1* gene is correlated with TOR expression level, which indicates that EBP1 might be a possible substrate that functions downstream of TOR (23). S6K (ribosomal p70 S6 kinase) is a translational regulator that interacts with RAPTOR in *Arabidopsis*, which also makes it a potential TOR substrate (77). However, the relationship of these potential TOR substrates to autophagy induction has not been tested.

In animals, insulin is a key upstream regulator of the TOR pathway (7), and a maize insulin-related peptide, ZmIGF (*Zea mays* insulin-like growth factor), promotes cell growth via ZmTOR kinase activity (26). Additionally, maize has been shown to be responsive to rapamycin treatment (1) and the TOR-S6K pathway seems to be conserved (26), making maize a possibly superior model for studying the TOR-autophagy pathway in plants.

In yeast and animals, other signaling pathways that control autophagy have been identified. For example, the mammalian AMPK (AMP-activated protein kinase) and its yeast homolog SNF1 (sucrose nonfermenting 1) have been shown to regulate autophagy induction (135). In plants, SNF1/AMPK-related kinases (SNRKs) have been identified as SNF1/AMPK homologs. The *Arabidopsis* SNRK AKIN10 has been shown to induce several autophagy genes (8), suggesting a conserved role for AMPK in the regulation of autophagy in plants. However, further analysis is needed to confirm this interpretation.

---

**EVIDENCE FOR SELECTIVE AUTOPHAGY**

In general, autophagy is considered to be a nonselective degradation process. However, in yeast and mammals, several studies have shown that certain organelles or protein aggregates can be selectively targeted by autophagy (for a more comprehensive review on selective autophagy, see 51, 103, 135). Inyeast, the biosynthetic cytoplasm-to-vacuole targeting pathway is a unique type of selective autophagy. The precursors of the vacuolar hydrolases Ape1 (aminopeptidase 1) and Ams1 (alpha-mannosidase 1) are delivered into the vacuole via double-membrane vesicles. The cargo receptor Atg19 recognizes and interacts with both the cargo and Atg8, and together they are transported to the PAS (135). Atg19 recognizes Atg8 through the AIM (see above) (88); this process also requires Atg11 as an adaptor. Various organelle-specific autophagy types have also been described in yeast (103)—for example, the selective degradation of mitochondria (mitophagy), peroxisomes (pexophagy), and ribosomes (ribophagy). In animals, two autophagic adaptors, p62/SQSTM1 (sequestosome 1) and NBR1, have been studied extensively and seem to play an important role during selective autophagy (51). Both p62 and NBR1 harbor a ubiquitin-associated domain that can bind ubiquitinated proteins and an AIM that can interact with Atg8/LC3. It has been proposed that both p62 and NBR1 are selective autophagy substrates and work as cargo receptors: First, protein aggregates are ubiquitinated, and then p62 and NBR1 are recruited to the ubiquitinated substrates; this is followed by interaction with Atg8, leading to the formation of autophagosomes around the cargo (51).

Plant NBR1 homologs were recently identified. In *Arabidopsis*, a single *AtNBR1* (At4g24690) gene has been identified; it binds ubiquitinated proteins via a C-terminal ubiquitin-associated domain and interacts with AtATG8 through the AIM motif (116). In tobacco (*Nicotiana tabacum*), Joka2 has been identified as a structural and possibly a functional homolog of p62 and NBR1 proteins. Joka2 interacts with NtATG8f, and its expression level increases during nutrient deprivation conditions (146). These findings suggest that the selective autophagy machinery is conserved to some extent from yeast to animals and plants.

In plants, mitophagy and pexophagy have not been studied, although some research indicates that autophagy is capable of selectively removing certain proteins or structures. In

BY-2 cells, a fusion protein between cytochrome b5 (Cyt b5) and red fluorescent protein (RFP) is transported to the vacuole for processing during nitrogen starvation (118). Interestingly, the vacuolar degradation rate of Cyt b5–RFP appears to be faster than that of other proteins, and the percentage of colocalization between Cyt b5–RFP and yellow fluorescent protein (YFP)–Atg8 is higher than with mitochondria, indicating that the Cyt b5–RFP proteins are engulfed by autophagosomes at a higher frequency. This study suggests that there is some autophagy selectivity toward the Cyt b5–RFP proteins (118).

In *Arabidopsis*, RNS2 is a conserved ribonuclease of the RNase T2 gene family, and is essential for normal ribosomal RNA (rRNA) decay. *Arabidopsis* plants lacking RNS2 activity have longer-lived rRNA, accumulate RNA in the vacuole, and have constitutive autophagy (41). It has been proposed that RNS2 may participate in a possible “ribophagy”-like mechanism in plants, functioning in ribosome turnover under normal growth conditions. The absence of RNS2 disrupts cell homeostasis, resulting in constitutive autophagy to restore the housekeeping role of the ribophagy-like process (41, 76).

As mentioned above, plants recycle nutrients from senescing leaf chloroplasts to newly forming organs at least partially through the autophagy pathway. During leaf senescence, small vesicles containing only stromal components (RCBs) pinch off from the chloroplasts and are delivered to the vacuole. GFP-AtATG8 colocalizes with stroma-targeted DsRed in spherical bodies in the vacuole, and in an *atg5* mutant this vacuolar RCB accumulation is compromised. These data suggest that after the RCBs form, they are enclosed within autophagosomes and delivered to the vacuole for degradation (14, 49, 50, 124). The discovery of RCBs demonstrates that autophagy selectively degrades stromal fractions of chloroplasts during leaf senescence. This exciting finding indicates that autophagy can thereby degrade certain components of an organelle and not others.Despite the emerging evidence for organelle-specific autophagy in plants, its underlying mechanism is still unclear. As mentioned above, cytoplasmic ubiquitinated proteins may be recognized by p62 and NBR1 and thus selectively incorporated into autophagosomes. However, how or-ganelles or parts of organelles are recognized for degradation is unknown. In yeast, the mitophagy-specific protein Atg32 serves as a tag for mitochondrial degradation (56, 91). Atg32 localizes on the mitochondrial outer membrane, binds to the selective autophagy adaptor Atg11, and along with mitochondria is further recruited to the vacuole for degradation. However, Atg32 and another mitophagy-specific protein, Atg33 (55), do not seem to have corresponding plant homologs (103). This raises the question of whether plants have other types of organelle-specific autophagy, such as mitophagy or pexophagy. If they do, do plants use a different set of genes or mechanisms? Future studies are anticipated to address these questions.

### SUMMARY POINTS

1. The autophagy core machinery is conserved from yeast to animals and plants, although some of the identified yeast autophagy (ATG) genes are present as gene families in plants. Several autophagy-defective plants with impaired ATG genes have been characterized; most of them are able to complete their life cycles but display early senescence and hypersensitivity to starvation conditions.
2. Plant autophagy functions during various abiotic stresses, including nitrogen and carbon starvation as well as oxidative, salt, and osmotic stresses.
3. Plants have a basal level of housekeeping autophagy even under favorable growth conditions, which may function during vacuole biogenesis, help in the elimination of damaged proteins and organelles, and remobilize nutrients during leaf senescence and seed germination.
4. Plant autophagy is involved in PCD during pathogen infection. However, its exact role is unclear. The SA signal may mediate this process.
5. Just as in yeast and animals, evidence suggests that the TOR kinase serves as a negative regulator of autophagy in plants. Some components of the TOR complexes and TOR substrates have also been identified in plants.

### FUTURE ISSUES

1. Although many yeast ATG gene homologs have been identified in plants, a few genes are missing from plant genomes, and some genes have expanded to gene families. It will be interesting to determine whether novel autophagy genes exist in plants and whether the genes within families have different roles under different circumstances.
2. The role of autophagy upon pathogen infection is still unclear. Several studies show that autophagy can function in both a prosurvival and a prodeath role. It seems that both pathogen type and plant age can affect the plant response, and the hormone SA is implicated. The role of autophagy and the signals that determine autophagy function during the immune response still need to be resolved.
3. Although TOR probably serves as an autophagy regulator in plants, its upstream signals and downstream substrates that control the autophagy pathway still need to be investigated. Also, TOR-independent regulatory pathways are yet to be unveiled.

4. Do plants also have organelle-specific autophagy? If so, do they use the same mechanisms as in yeast?

5. Despite the important role of plant autophagy, autophagy-defective plants are able to complete their life cycles. Whether a compensatory pathway exists that ensures survival when the autophagy pathway is impaired remains an open question.

**DISCLOSURE STATEMENT**

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

**ACKNOWLEDGMENTS**

This work was supported by grants IOB-0515998 and MBC-1051818 from the National Science Foundation to D.C.B.

**LITERATURE CITED**

1. Agredano-Moreno LT, Reyes de la Cruz H, Martinez-Castilla LP, Sanchez de Jimenez E. 2007. Distinctive expression and functional regulation of the maize (*Zea mays* L.) TOR kinase ortholog. *Mol. BioSyst.* 3:794–802

2. Ahn CS, Han JA, Lee HS, Lee S, Pai HS. 2011. The PP2A regulatory subunit Tap46, a component of the TOR signaling pathway, modulates growth and metabolism in plants. *Plant Cell* 23:185–209

3. Anderson GH, Hanson MR. 2005. The Arabidopsis Mei2 homologue AML1 binds AtRaptor1B, the plant homologue of a major regulator of eukaryotic cell growth. *BMC Plant Biol.* 5:2

4. Anderson GH, Veit B, Hanson MR. 2005. The Arabidopsis *AtRaptor* genes are essential for postembryonic plant growth. *BMC Biol.* 3:12

5. Angelovici R, Fait A, Zhu X, Szymanski J, Feldmesser E, et al. 2009. Deciphering transcriptional and metabolic networks associated with lysine metabolism during Arabidopsis seed development. *Plant Physiol.* 151:2058–72

6. Avin-Wittenberg T, Honig A, Galili G. 2012. Variations on a theme: plant autophagy in comparison to yeast and mammals. *Protoplasma*. In press

7. Avruch J, Hara K, Lin Y, Liu M, Long X, et al. 2006. Insulin and amino-acid regulation of mTOR signaling and kinase activity through the Rheb GTPase. *Oncogene* 25:6361–72

8. Baena-Gonzalez E, Rolland F, Thevelein JM, Sheen J. 2007. A central integrator of transcription networks in plant stress and energy signalling. *Nature* 448:938–42

9. Bassham DC. 2007. Plant autophagy—more than a starvation response. *Curr. Opin. Plant Biol.* 10:587–93

10. Bassham DC. 2009. Function and regulation of macroautophagy in plants. *Biochim. Biophys. Acta* 1793:1397–403

11. Bassham DC, Laporte M, Marty F, Moriyasu Y, Ohsumi Y, et al. 2006. Autophagy in development and stress responses of plants. *Autophagy* 2:2–11

12. Boren M, Hoglund AS, Bozhkov P, Jansson C. 2006. Developmental regulation of a VEIDase caspase-like proteolytic activity in barley caryopsis. *J. Exp. Bot.* 57:3747–53

13. Brown EJ, Albers MW, Shin TB, Ichikawa K, Keith CT, et al. 1994. A mammalian protein targeted by G1-arresting rapamycin-receptor complex. *Nature* 369:756–58

14. Chiba A, Ishida H, Nishizawa NK, Makino A, Mae T. 2003. Exclusion of ribulose-1,5-bisphosphate carboxylase/oxygenase from chloroplasts by specific bodies in naturally senescing leaves of wheat. *Plant Cell Physiol.* 44:914–21

15. Choi J, Chen J, Schreiber SL, Clardy J. 1996. Structure of the FKBP12-rapamycin complex interacting with the binding domain of human FRAP. *Science* 273:239–42

16. Chung T, Phillips AR, Vierstra RD. 2010. ATG8 lipidation and ATG8-mediated autophagy in Arabidopsis require ATG12 expressed from the differentially controlled ATG12A AND ATG12B loci. *Plant J.* 62:483–93

17. Chung T, Suttangkakul A, Vierstra RD. 2009. The ATG autophagic conjugation system in maize: ATG transcripts and abundance of the ATG8-lipid adduct are regulated by development and nutrient availability. *Plant Physiol.* 149:220–34

18. Coll NS, Vercammen D, Smidler A, Clover C, Van Breusegem F, et al. 2010. *Arabidopsis* type I metacaspases control cell death. *Science* 330:1393–97

19. Contento AL, Xiong Y, Bassham DC. 2005. Visualization of autophagy in Arabidopsis using the fluorescent dye monodansylcadaverine and a GFP-AtATG8e fusion protein. *Plant J.* 42:598–608

20. Courtois-Moreau CL, Pesquet E, Sjodin A, Muniz L, Bollhoner B, et al. 2009. A unique program for cell death in xylem fibers of *Populus* stem. *Plant J.* 58:260–74

21. Denault JB, Salvesen GS. 2002. Caspases: keys in the ignition of cell death. *Chem. Rev.* 102:4489–500

22. Deprost D, Truong HN, Robaglia C, Meyer C. 2005. An *Arabidopsis* homolog of RAPTOR/KOG1 is essential for early embryo development. *Biochem. Biophys. Res. Commun.* 326:844–50

23. Deprost D, Yao L, Sormani R, Moreau M, Leterreux G, et al. 2007. The *Arabidopsis* TOR kinase links plant growth, yield, stress resistance and mRNA translation. *EMBO Rep.* 8:864–70

24. Diaz-Troya S, Florencio FJ, Crespo JL. 2008. Target of rapamycin and LST8 proteins associate with membranes from the endoplasmic reticulum in the unicellular green alga *Chlamydomonas reinhardtii*. *Eukaryot. Cell* 7:212–22

25. Diaz-Troya S, Perez-Perez ME, Florencio FJ, Crespo JL. 2008. The role of TOR in autophagy regulation from yeast to plants and mammals. *Autophagy* 4:851–65

26. Dinkova TD, De La Cruz HR, García-Flores C, Aguilar R, Jiménez-García LF, De Jiménez ES. 2007. Dissecting the TOR-S6K signal transduction pathway in maize seedlings: relevance on cell growth regulation. *Physiol. Plant.* 130:1–10

27. Along with Reference 34, provides the first description of the phenotypes of autophagy mutants in *Arabidopsis* and demonstrates two key functions of autophagy in plants.

27. Doelling JH, Walker JM, Friedman EM, Thompson AR, Vierstra RD. 2002. The APG8/12-activating enzyme APG7 is required for proper nutrient recycling and senescence in *Arabidopsis thaliana*. *J. Biol. Chem.* 277:33105–14

28. Fujiki Y, Yoshimoto K, Ohsumi Y. 2007. An *Arabidopsis* homolog of yeast ATG6/VPS30 is essential for pollen germination. *Plant Physiol.* 143:1132–39

29. Fujioka Y, Noda NN, Fujii K, Yoshimoto K, Ohsumi Y, Inagaki F. 2008. *In vitro* reconstitution of plant Atg8 and Atg12 conjugation systems essential for autophagy. *J. Biol. Chem.* 283:1921–28

30. Ghiglione HO, Gonzalez FG, Serrago R, Maldonado SB, Chilcott C, et al. 2008. Autophagy regulated by day length determines the number of fertile florets in wheat. *Plant J.* 55:1010–24

31. Guiboileau A, Sormani R, Meyer C, Masclaux-Daubresse C. 2010. Senescence and death of plant organs: nutrient recycling and developmental regulation. *C. R. Biol.* 333:382–91

32. Hailey DW, Rambold AS, Satpute-Krishnan P, Mitra K, Sougrat R, et al. 2010. Mitochondria supply membranes for autophagosome biogenesis during starvation. *Cell* 141:656–67

33. Hanada T, Noda NN, Satomi Y, Ichimura Y, Fujioka Y, et al. 2007. The Atg12-Atg5 conjugate has a novel E3-like activity for protein lipidation in autophagy. *J. Biol. Chem.* 282:37298–302

34. Hanaoka H, Noda T, Shirano Y, Kato T, Hayashi H, et al. 2002. Leaf senescence and starvation-induced chlorosis are accelerated by the disruption of an *Arabidopsis* autophagy gene. *Plant Physiol.* 129:1181–93

35. Hara K, Maruki Y, Long X, Yoshino K, Oshiro N, et al. 2002. Raptor, a binding partner of target of rapamycin (TOR), mediates TOR action. *Cell* 110:177–89

36. Harrison-Lowe NJ, Olsen LJ. 2008. Autophagy protein 6 (ATG6) is required for pollen germination in *Arabidopsis thaliana*. *Autophagy* 4:339–48

37. Hayashi-Nishino M, Fujita N, Noda T, Yamaguchi A, Yoshimori T, Yamamoto A. 2009. A subdomain of the endoplasmic reticulum forms a cradle for autophagosome formation. *Nat. Cell Biol.* 11:1433–37

38. Hayward AP, Dinesh-Kumar SP. 2010. What can plant autophagy do for an innate immune response? *Annu. Rev. Phytopathol.* 49:557–76

39. He C, Klionsky DJ. 2007. Atg9 trafficking in autophagy-related pathways. *Autophagy* 3:271–74

40. He C, Klionsky DJ. 2009. Regulation mechanisms and signaling pathways of autophagy. *Annu. Rev. Genet.* 43:67–93

41. Hillwig MS, Contento AL, Meyer A, Ebony D, Bassham DC, Macintosh GC. 2011. RNS2, a conserved member of the RNase T2 family, is necessary for ribosomal RNA decay in plants. *Proc. Natl. Acad. Sci. USA* 108:1093–98

42. Hofius D, Schultz-Larsen T, Joensen J, Tsitsigiannis DI, Petersen NH, et al. 2009. Autophagic components contribute to hypersensitive cell death in *Arabidopsis*. *Cell* 137:773–83

43. Hortensteiner S, Krautler B. 2011. Chlorophyll breakdown in higher plants. *Biochim. Biophys. Acta* 1807:977–88

44. Horvath BM, Magyar Z, Zhang Y, Hamburger AW, Bako L, et al. 2006. EBP1 regulates organ size through cell growth and proliferation in plants. *EMBO J.* 25:4909–20

45. Hunter T. 1995. When is a lipid kinase not a lipid kinase? When it is a protein kinase. *Cell* 83:1–4

46. Ibl V, Stoger E. 2012. The formation, function and fate of protein storage compartments in seeds. *Protoplasma*. In press

47. Inoue Y, Suzuki T, Hattori M, Yoshimoto K, Ohsumi Y, Moriyasu Y. 2006. *AtATG* genes, homologs of yeast autophagy genes, are involved in constitutive autophagy in *Arabidopsis* root tip cells. *Plant Cell Physiol.* 47:1641–52

48. Ishida H, Yoshimoto K. 2008. Chloroplasts are partially mobilized to the vacuole by autophagy. *Autophagy* 4:961–62

49. Ishida H, Yoshimoto K, Izumi M, Reisen D, Yano Y, et al. 2008. Mobilization of Rubisco and stroma-localized fluorescent proteins of chloroplasts to the vacuole by an ATG gene-dependent autophagic process. *Plant Physiol.* 148:142–55

50. Izumi M, Wada S, Makino A, Ishida H. 2010. The autophagic degradation of chloroplasts via Rubisco-containing bodies is specifically linked to leaf carbon status but not nitrogen status in *Arabidopsis*. *Plant Physiol.* 154:1196–209

51. Johansen T, Lamark T. 2011. Selective autophagy mediated by autophagic adapter proteins. *Autophagy* 7:279–96

52. Jones JD, Dangl JL. 2006. The plant immune system. *Nature* 444:323–29

53. Joo JH, Yoo HJ, Hwang I, Lee JS, Nam KH, Bae YS. 2005. Auxin-induced reactive oxygen species production requires the activation of phosphatidylinositol 3-kinase. *FEBS Lett.* 579:1243–48

54. Kamada Y, Funakoshi T, Shintani T, Nagano K, Ohsumi M, Ohsumi Y. 2000. Tor-mediated induction of autophagy via an Apg1 protein kinase complex. *J. Cell Biol.* 150:1507–13

55. Kanki T, Wang K, Baba M, Bartholomew CR, Lynch-Day MA, et al. 2009. A genomic screen for yeast mutants defective in selective mitochondria autophagy. *Mol. Biol. Cell* 20:4730–38

56. Kanki T, Wang K, Cao Y, Baba M, Klionsky DJ. 2009. Atg32 is a mitochondrial protein that confers selectivity during mitophagy. *Dev. Cell* 17:98–109

57. Ketelaar T, Voss C, Dimmock SA, Thumm M, Hussey PJ. 2004. *Arabidopsis* homologues of the autophagy protein Atg8 are a novel family of microtubule binding proteins. *FEBS Lett.* 567:302–6

58. Koornneef A, Pieterse CM. 2008. Cross talk in defense signaling. *Plant Physiol.* 146:839–44

59. Kroemer G, Galluzzi L, Vandenabeele P, Abrams J, Alnemri ES, et al. 2009. Classification of cell death: recommendations of the Nomenclature Committee on Cell Death 2009. *Cell Death Differ.* 16:3–11

60. Kunz J, Henriquez R, Schneider U, Deuter-Reinhard M, Movva NR, Hall MN. 1993. Target of rapamycin in yeast, TOR2, is an essential phosphatidylinositol kinase homolog required for G1 progression. *Cell* 73:585–96

61. Kwon SI, Cho HJ, Jung JH, Yoshimoto K, Shirasu K, Park OK. 2010. The Rab GTPase RabG3b functions in autophagy and contributes to tracheary element differentiation in *Arabidopsis*. *Plant J.* 64:151–64

62. Lai Z, Wang F, Zheng Z, Fan B, Chen Z. 2011. A critical role of autophagy in plant resistance to necrotrophic fungal pathogens. *Plant J.* 66:953–68

63. Lam E. 2004. Controlled cell death, plant survival and development. *Nat. Rev. Mol. Cell Biol.* 5:305–15

64. Lee Y, Bak G, Choi Y, Chuang WI, Cho HT. 2008. Roles of phosphatidylinositol 3-kinase in root hair growth. *Plant Physiol.* 147:624–35

65. Lee Y, Kim ES, Choi Y, Hwang I, Staiger CJ, Chung YY. 2008. The Arabidopsis phosphatidylinositol 3-kinase is important for pollen development. *Plant Physiol.* 147:1886–97

66. Leiber RM, John F, Verhertbruggen Y, Diet A, Knox JP, Ringli C. 2010. The TOR pathway modulates the structure of cell walls in *Arabidopsis*. *Plant Cell* 22:1898–908

67. Lenz HD, Haller E, Melzer E, Gust AA, Nuernberger T. 2011. Autophagy controls plant basal immunity in a pathogenic lifestyle-dependent manner. *Autophagy* 7:773–74

68. Lenz HD, Haller E, Melzer E, Kober K, Wurster K, et al. 2011. Autophagy differentially controls plant basal immunity to biotrophic and necrotrophic pathogens. *Plant J.* 66:818–30

69. Leshem Y, Seri L, Levine A. 2007. Induction of phosphatidylinositol 3-kinase-mediated endocytosis by salt stress leads to intracellular production of reactive oxygen species and salt tolerance. *Plant J.* 51:185–97

70. Levanon YH, Rubin R, Altschuler Y, Galili G. 1992. Evidence for a novel route of wheat storage proteins to vacuoles. *J. Cell Biol.* 119:1117–28

71. Levine B, Klionsky DJ. 2004. Development by self-digestion: molecular mechanisms and biological functions of autophagy. *Dev. Cell* 6:463–77

72. Liu Y, Bassham DC. 2010. TOR is a negative regulator of autophagy in *Arabidopsis thaliana*. *PLoS ONE* 5:e11883

73. Provides the first report of functional analysis of autophagy during pathogen responses in plants.

73. Liu Y, Schiff M, Czimmek K, Talloczy Z, Levine B, Dinesh-Kumar SP. 2005. Autophagy regulates programmed cell death during the plant innate immune response. *Cell* 121:567–77

74. Liu Y, Xiong Y, Bassham DC. 2009. Autophagy is required for tolerance of drought and salt stress in plants. *Autophagy* 5:954–63

75. Loewith R, Jacinto E, Wullschleger S, Lorberg A, Crespo JL, et al. 2002. Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control. *Mol. Cell* 10:457–68

76. Macintosh GC, Bassham DC. 2011. The connection between ribophagy, autophagy and ribosomal RNA decay. *Autophagy* 7:662–63

77. Mahfouz MM, Kim S, Delauney AJ, Verma DP. 2006. *Arabidopsis* TARGET OF RAPAMYCIN interacts with RAPTOR, which regulates the activity of S6 kinase in response to osmotic stress signals. *Plant Cell* 18:477–90

78. Makino A, Osmond B. 1991. Effects of nitrogen nutrition on nitrogen partitioning between chloroplasts and mitochondria in pea and wheat. *Plant Physiol.* 96:355–62

79. Mari M, Griffith J, Rieter E, Krishnappa L, Klionsky DJ, Reggiori F. 2010. An Atg9-containing compartment that functions in the early steps of autophagosome biogenesis. *J. Cell Biol.* 190:1005–22

80. Marty F. 1978. Cytochemical studies on GERL, provacuoles, and vacuoles in root meristematic cells of *Euphorbia*. *Proc. Natl. Acad. Sci. USA* 75:852–56

81. Matile PH, Winkenbach F. 1971. Function of lysosomes and lysosomal enzymes in the senescing corolla of the morning glory. *J. Exp. Bot.* 22:759–71

82. Mehrpour M, Esclatine A, Beau I, Codogno P. 2010. Overview of macroautophagy regulation in mammalian cells. *Cell Res.* 20:748–62

83. Menand B, Desnos T, Nussaume L, Berger F, Bouchez D, et al. 2002. Expression and disruption of the *Arabidopsis* TOR (target of rapamycin) gene. *Proc. Natl. Acad. Sci. USA* 99:6422–27

84. Mijaljica D, Prescott M, Devenish RJ. 2011. Microautophagy in mammalian cells: revisiting a 40-year-old conundrum. *Autophagy* 7:673–82

85. Nair U, Cao Y, Xie Z, Klionsky DJ. 2010. Roles of the lipid-binding motifs of Atg18 and Atg21 in the cytoplasm to vacuole targeting pathway and autophagy. *J. Biol. Chem.* 285:11476–88

86. Nair U, Jotwani A, Geng J, Gammoh N, Richerson D, et al. 2011. SNARE proteins are required for macroautophagy. *Cell* 146:290–302

87. Nishida Y, Arakawa S, Fujitani K, Yamaguchi H, Mizuta T, et al. 2009. Discovery of Atg5/Atg7-independent alternative macroautophagy. *Nature* 461:654–58

88. Noda NN, Kumeta H, Nakatogawa H, Satoo K, Adachi W, et al. 2008. Structural basis of target recognition by Atg8/LC3 during selective autophagy. *Genes Cells* 13:1211–18

89. Noda NN, Ohsumi Y, Inagaki F. 2010. Atg8-family interacting motif crucial for selective autophagy. *FEBS Lett.* 584:1379–85

90. Ohsumi Y. 2001. Molecular dissection of autophagy: two ubiquitin-like systems. *Nat. Rev. Mol. Cell Biol.* 2:211–16

91. Okamoto K, Kondo-Okamoto N, Ohsumi Y. 2009. Mitochondria-anchored receptor Atg32 mediates degradation of mitochondria via selective autophagy. *Dev. Cell* 17:87–97

92. Orenstein SJ, Cuervo AM. 2010. Chaperone-mediated autophagy: molecular mechanisms and physiological relevance. *Semin. Cell Dev. Biol.* 21:719–26

93. Oshiro N, Yoshino K, Hidayat S, Tokunaga C, Hara K, et al. 2004. Dissociation of raptor from mTOR is a mechanism of rapamycin-induced inhibition of mTOR function. *Genes Cells* 9:359–66

94. Patel S, Dinesh-Kumar SP. 2008. Arabidopsis ATG6 is required to limit the pathogen-associated cell death response. *Autophagy* 4:20–27

95. Pattingre S, Espert L, Biard-Piechaczyk M, Codogno P. 2008. Regulation of macroautophagy by mTOR and Beclin 1 complexes. *Biochimie* 90:313–23

96. Perez-Perez ME, Florencio FJ, Crespo JL. 2010. Inhibition of target of rapamycin signaling and stress activate autophagy in *Chlamydomonas reinhardtii*. *Plant Physiol.* 152:1874–88

97. Phillips AR, Suttangkakul A, Vierstra RD. 2008. The ATG12-conjugating enzyme ATG10 is essential for autophagic vesicle formation in *Arabidopsis thaliana*. *Genetics* 178:1339–53

98. Qin G, Ma Z, Zhang L, Xing S, Hou X, et al. 2007. *Arabidopsis* AtBECLIN 1/AtAtg6/AtVps30 is essential for pollen germination and plant development. *Cell Res.* 17:249–63

99. Raught B, Gingras AC, Sonenberg N. 2001. The target of rapamycin (TOR) proteins. *Proc. Natl. Acad. Sci. USA* 98:7037–44

100. Ravikumar B, Moreau K, Jahreiss L, Puri C, Rubinsztein DC. 2010. Plasma membrane contributes to the formation of pre-autophagosomal structures. *Nat. Cell Biol.* 12:747–57

101. Reape TJ, McCabe PF. 2010. Apoptotic-like regulation of programmed cell death in plants. *Apoptosis* 15:249–56

102. Ren M, Qiu S, Venglat P, Xiang D, Feng L, et al. 2011. Target of rapamycin regulates development and ribosomal RNA expression through kinase domain in *Arabidopsis*. *Plant Physiol.* 155:1367–82

103. Reumann S, Voitsekhovskaja O, Lillo C. 2010. From signal transduction to autophagy of plant cell organelles: lessons from yeast and mammals and plant-specific features. *Protoplasma* 247:233–56

104. Reyes FC, Chung T, Holding D, Jung R, Vierstra R, Otegui MS. 2011. Delivery of prolamins to the protein storage vacuole in maize aleurone cells. *Plant Cell* 23:769–84

105. Sanmartin M, Ordóñez A, Sohn EJ, Robert S, Sanchez-Serrano JJ, et al. 2007. Divergent functions of VTI12 and VTI11 in trafficking to storage and lytic vacuoles in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 104:3645–50

106. Scarlatti F, Maffei R, Beau I, Codogno P, Ghidoni R. 2008. Role of non-canonical Beclin 1-independent autophagy in cell death induced by resveratrol in human breast cancer cells. *Cell Death Differ.* 15:1318–29

107. Shin JH, Yoshimoto K, Ohsumi Y, Jeon JS, An G. 2009. OsATG10b, an autophagosome component, is needed for cell survival against oxidative stresses in rice. *Mol. Cells* 27:67–74

108. Slavikova S, Shy G, Yao Y, Glozman R, Levanony H, et al. 2005. The autophagy-associated Atg8 gene family operates both under favourable growth conditions and under starvation stresses in *Arabidopsis* plants. *J. Exp. Bot.* 56:2839–49

109. Slavikova S, Ufaz S, Avin-Wittenberg T, Levanony H, Galili G. 2008. An autophagy-associated Atg8 protein is involved in the responses of *Arabidopsis* seedlings to hormonal controls and abiotic stresses. *J. Exp. Bot.* 59:4029–43

110. Sormani R, Yao L, Menand B, Ennar N, Lecampion C, et al. 2007. *Saccharomyces cerevisiae* FKBP12 binds *Arabidopsis thaliana* TOR and its expression in plants leads to rapamycin susceptibility. *BMC Plant Biol.* 7:26

111. Spoel SH, Dong X. 2008. Making sense of hormone crosstalk during plant immune responses. *Cell Host Microbe* 3:348–51

112. Su W, Ma H, Liu C, Wu J, Yang J. 2006. Identification and characterization of two rice autophagy associated genes, OsAtg8 and OsAtg4. *Mol. Biol. Rep.* 33:273–78

113. Surpin M, Zheng H, Morita MT, Saito C, Avila E, et al. 2003. The VTI family of SNARE proteins is necessary for plant viability and mediates different protein transport pathways. *Plant Cell* 15:2885–99

114. Suzuki K, Kirisako T, Kamada Y, Mizushima N, Noda T, Ohsumi Y. 2001. The pre-autophagosomal structure organized by concerted functions of APG genes is essential for autophagosome formation. *EMBO J.* 20:5971–81

115. Suzuki NN, Yoshimoto K, Fujioka Y, Ohsumi Y, Inagaki F. 2005. The crystal structure of plant ATG12 and its biological implication in autophagy. *Autophagy* 1:119–26

116. Svenning S, Lamark T, Krause K, Johansen T. 2011. Plant NBR1 is a selective autophagy substrate and a functional hybrid of the mammalian autophagic adapters NBR1 and p62/SQSTM1. *Autophagy* 7:993–1010

117. Thompson AR, Doelling JH, Suttangkakul A, Vierstra RD. 2005. Autophagic nutrient recycling in Arabidopsis directed by the ATG8 and ATG12 conjugation pathways. *Plant Physiol.* 138:2097–110

118. Toyooka K, Moriyasu Y, Goto Y, Takeuchi M, Fukuda H, Matsuoka K. 2006. Protein aggregates are transported to vacuoles by a macroautophagic mechanism in nutrient-starved plant cells. *Autophagy* 2:96–106

119. Toyooka K, Okamoto T, Minamikawa T. 2001. Cotyledon cells of *Vigna mungo* seedlings use at least two distinct autophagic machineries for degradation of starch granules and cellular components. *J. Cell Biol.* 154:973–82

120. Van der Wilden W, Herman EM, Chrispeels MJ. 1980. Protein bodies of mung bean cotyledons as autophagic organelles. *Proc. Natl. Acad. Sci. USA* 77:428–32

121. van Doorn WG, Woltering EJ. 2005. Many ways to exit? Cell death categories in plants. *Trends Plant Sci.* 10:117–22

122. Vanhee C, Guillot S, Masquelier D, Degand H, Deleu M, et al. 2011. A TSPO-related protein localizes to the early secretory pathway in *Arabidopsis*, but is targeted to mitochondria when expressed in yeast. *J. Exp. Bot.* 62:497–508

123. Wada S, Ishida H. 2009. Chloroplasts autophagy during senescence of individually darkened leaves. *Plant Signal Behav.* 4:565–67

124. Wada S, Ishida H, Izumi M, Yoshimoto K, Ohsumi Y, et al. 2009. Autophagy plays a role in chloroplast degradation during senescence in individually darkened leaves. *Plant Physiol.* 149:885–93

125. Wang Y, Nishimura MT, Zhao T, Tang D. 2011. ATG2, an autophagy-related protein, negatively affects powdery mildew resistance and mildew-induced cell death in *Arabidopsis*. *Plant J.* 68:74–87

126. Wang Y, Weiss LM, Orlofsky A. 2009. Host cell autophagy is induced by *Toxoplasma gondii* and contributes to parasite growth. *J. Biol. Chem.* 284:1694–701

127. Watanabe Y, Yamamoto M. 1994. S.pombe mei2+ encodes an RNA-binding protein essential for premeiotic DNA synthesis and meiosis I, which cooperates with a novel RNA species meiRNA. *Cell* 78:487–98

128. Wedaman KP, Reinke A, Anderson S, Yates J III, McCaffery JM, Powers T. 2003. Tor kinases are in distinct membrane-associated protein complexes in *Saccharomyces cerevisiae*. *Mol. Biol. Cell* 14:1204–20

129. Weidberg H, Shvets E, Shpilka T, Shimron F, Shinder V, Elazar Z. 2010. LC3 and GATE-16/GABARAP subfamilies are both essential yet act differently in autophagosome biogenesis. *EMBO J.* 29:1792–802

130. Welters P, Takegawa K, Emr SD, Chrispeels MJ. 1994. AtVPS34, a phosphatidylinositol 3-kinase of *Arabidopsis thaliana*, is an essential protein with homology to a calcium-dependent lipid binding domain. *Proc. Natl. Acad. Sci. USA* 91:11398–402

131. Xie Z, Klionsky DJ. 2007. Autophagosome formation: core machinery and adaptations. *Nat. Cell Biol.* 9:1102–9

132. Xiong Y, Contento AL, Bassham DC. 2005. AtATG18a is required for the formation of autophagosomes during nutrient stress and senescence in *Arabidopsis thaliana*. *Plant J.* 42:535–46

133. Xiong Y, Contento AL, Bassham DC. 2007. Disruption of autophagy results in constitutive oxidative stress in *Arabidopsis*. *Autophagy* 3:257–58

134. Xiong Y, Contento AL, Nguyen PQ, Bassham DC. 2007. Degradation of oxidized proteins by autophagy during oxidative stress in *Arabidopsis*. *Plant Physiol.* 143:291–99

135. Yang Z, Klionsky DJ. 2009. An overview of the molecular mechanism of autophagy. *Curr. Top. Microbiol. Immunol.* 335:1–32

136. Yang Z, Klionsky DJ. 2010. Eaten alive: a history of macroautophagy. *Nat. Cell Biol.* 12:814–22

137. Yano K, Hattori M, Moriyasu Y. 2007. A novel type of autophagy occurs together with vacuole genesis in miniprotoplasts prepared from tobacco culture cells. *Autophagy* 3:215–21

138. Yano K, Suzuki T, Moriyasu Y. 2007. Constitutive autophagy in plant root cells. *Autophagy* 3:360–62

139. Yla-Anttila P, Vihinen H, Jokitalo E, Eskelinen EL. 2009. 3D tomography reveals connections between the phagophore and endoplasmic reticulum. *Autophagy* 5:1180–85

140. Yoshimoto K. 2010. Physiological roles of autophagy in plants: Does plant autophagy have a pro-death function? *Plant Signal Behav.* 5:494–96

141. Yoshimoto K. 2010. Plant autophagy puts the brakes on cell death by controlling salicylic acid signaling. *Autophagy* 6:192–93

142. Yoshimoto K, Hanaoka H, Sato S, Kato T, Tabata S, et al. 2004. Processing of ATG8s, ubiquitin-like proteins, and their deconjugation by ATG4s are essential for plant autophagy. *Plant Cell* 16:2967–83

143. Yoshimoto K, Jikumaru Y, Kamiya Y, Kusano M, Consonni C, et al. 2009. Autophagy negatively regulates cell death by controlling NPR1-dependent salicylic acid signaling during senescence and the innate immune response in *Arabidopsis*. *Plant Cell* 21:2914–27

144. Yoshimoto K, Takano Y, Sakai Y. 2010. Autophagy in plants and phytopathogens. *FEBS Lett.* 584:1350–58

145. Young AR, Chan EY, Hu XW, Kochl R, Crawshaw SG, et al. 2006. Starvation and ULK1-dependent cycling of mammalian Atg9 between the TGN and endosomes. *J. Cell Sci.* 119:3888–900

146. Zientara-Rytter K, Lukomska J, Moniuszko G, Gwozdecki R, Surowiecki P, et al. 2011. Identification and functional analysis of Joka2, a tobacco member of the family of selective autophagy cargo receptors. *Autophagy* 7:1145–58

142. Provides the first report of the use of ATG8 as an autophagosome marker in plants and demonstrates that *Arabidopsis* and yeast process ATG8s similarly.

146. Reports on the analysis of Joka2, a potential selective autophagy receptor in plants; Reference 116 reports similarly on another potential receptor.

Contents

There Ought to Be an Equation for That  
*Joseph A. Berry* ………………………………………… 1  

Photorespiration and the Evolution of C₄ Photosynthesis  
*Rowan F. Sage, Tammy L. Sage, and Ferit Kocacinar* …………………… 19  

The Evolution of Flavin-Binding Photoreceptors: An Ancient Chromophore Serving Trendy Blue-Light Sensors  
*Aba Losi and Wolfgang Gärtner* ………………………………………… 49  

The Shikimate Pathway and Aromatic Amino Acid Biosynthesis in Plants  
*Hiroshi Maeda and Natalia Dudareva* …………………………………… 73  

Regulation of Seed Germination and Seedling Growth by Chemical Signals from Burning Vegetation  
*David C. Nelson, Gavin R. Flematti, Emilio L. Ghisalberti, Kingsley W. Dixon,*  
*and Steven M. Smith* …………………………………………………… 107  

Iron Uptake, Translocation, and Regulation in Higher Plants  
*Takanori Kobayashi and Naoko K. Nishizawa* ………………………… 131  

Plant Nitrogen Assimilation and Use Efficiency  
*Guohua Xu, Xiaorong Fan, and Anthony J. Miller* …………………… 153  

Vacuolar Transporters in Their Physiological Context  
*Enrico Martinoia, Stefan Meyer, Alexis De Angeli, and Réka Nagy* … 183  

Autophagy: Pathways for Self-Eating in Plant Cells  
*Yimo Liu and Diane C. Bassham* ………………………………………… 215  

Plasmodesmata Paradigm Shift: Regulation from Without Versus Within  
*Tessa M. Burch-Smith and Patricia C. Zambryski* …………………… 239  

Small Molecules Present Large Opportunities in Plant Biology  
*Glenn R. Hicks and Natasha V. Raikhel* ……………………………… 261  

Genome-Enabled Insights into Legume Biology  
*Nevin D. Young and Arvind K. Bharti* ………………………………… 283

Synthetic Chromosome Platforms in Plants  
Robert T. Gaeta, Rick E. Masonbrink, Lakshminarasimhan Krishnaswamy, Changzeng Zhao, and James A. Birchler  . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .