
Identification of a Novel Dihydrodaidzein Racemase Essential for Biosynthesis of Equol from Daidzein in Lactococcus sp. Strain 20-92

Yoshikazu Shimada,$^{a}$ Masayuki Takahashi,$^{a}$ Norihiro Miyazawa,$^{b}$ Yasuhiro Abiru,$^{c}$ Shigeto Uchiyama,$^{c}$ and Haretsugu Hishigaki$^{a}$

Institute of Biomedical Innovation, Otsuka Pharmaceutical Co., Ltd., Tokushima, Japan$^{a}$; Department of Drug Metabolism, Drug Safety Research Center, Tokushima Research Institute, Otsuka Pharmaceutical Co., Ltd., Tokushima, Japan$^{b}$; and Saga Nutraceuticals Research Institute, Otsuka Pharmaceutical Co., Ltd., Saga, Japan$^{c}$

Equol is metabolized from daidzein, a soy isoflavone, by the gut microflora. In this study, we identified a novel dihydrodaidzein racemase (L-DDRC) that is involved in equol biosynthesis in a lactic acid bacterium, *Lactococcus* sp. strain 20-92, and confirmed that histidine-tagged recombinant L-DDRC (L-DDRC-His) was able to convert both the (*R*)- and (*S*)-enantiomers of dihydrodaidzein to the racemate. Moreover, we showed that recombinant L-DDRC-His was essential for *in vitro* equol production from daidzein by a recombinant enzyme mixture and that efficient *in vitro* equol production from daidzein was possible using at least four enzymes, including L-DDRC. We also proposed a model of the metabolic pathway from daidzein to equol in *Lactococcus* strain 20-92.

The intake of isoflavones through soy foods (e.g., miso, tofu, and natto) has many reported health benefits. Because isoflavones are structurally similar to the female hormone estrogen, the associated health benefits are thought to be due to their abilities to bind to estrogen receptors ([9], [10], [13]). Among the isoflavonoids, equol, a metabolite of daidzein produced by members of the gut microflora ([4]), is thought to be the primary soy isoflavone derivative that is responsible for the prevention of several sex hormone-dependent diseases because of its potent estrogenic activity ([13]). However, individual differences exist in the ability of the enteric microflora to produce equol, and equol cannot be produced in more than half of the individuals who consume soy isoflavones ([2]). Therefore, it has been proposed that the development of probiotics using safe bacteria capable of producing equol from daidzein could allow the production of equol in the enteric environments of all individuals.

*Lactococcus* sp. strain 20-92, isolated from healthy human feces, is an equol-producing lactic acid bacterium that produces equol from daidzein and is classified as *Lactococcus garvieae*, which is found in several traditional Italian cheeses and in healthy human intestines ([6], [16]). The application of *Lactococcus* strain 20-92 to foods is currently being investigated.

Equol is thought to be produced sequentially from daidzein via dihydrodaidzein and tetrahydrodaidzein by intestinal bacteria ([7], [17]). To elucidate the metabolic pathway from daidzein to equol in *Lactococcus* strain 20-92, we recently identified three enzymes (daidzein reductase [L-DZNR], dihydrodaidzein reductase [L-DHDR], and tetrahydrodaidzein reductase [L-THDR]) that catalyze the successive conversion steps of the metabolic pathway and showed that the genes for these enzymes form a gene cluster in the bacterium’s genome ([14], [15]). Dihydrodaidzein, the first metabolite in the production of equol from daidzein, has two enantiomers, as does equol, because of the presence of an asymmetric carbon atom at the C-3 position. We recently showed that although the dihydrodaidzein that was produced by recombinant L-DZNR was largely (*S*)-dihydrodaidzein, a racemic mixture of dihydrodaidzein was detected when daidzein was converted using a cell extract of *Lactococcus* strain 20-92, suggesting the existence of a dihydrodaidzein racemase in strain 20-92 ([15]). Similarly, it has been reported that (*R*)- and (*S*)-dihydrodaidzein are rapidly converted into the racemate in a cell extract of *Eggerthella* sp. strain Julong 732 ([7]). Therefore, we speculated that the dihydrodaidzein racemase may play an important role in equol biosynthesis in *Lactococcus* strain 20-92, and we further analyzed the region around the enzyme gene cluster to identify the gene for the dihydrodaidzein racemase.

Here, we report the identification of a dihydrodaidzein racemase (L-DDRC) involved in the biosynthesis of equol from daidzein; the efficient production of equol from daidzein is possible using a mixture of the four recombinant proteins (L-DZNR, L-DHDR, L-THDR, and L-DDRC). In addition, based on our current understanding, a model of the metabolic pathway from daidzein to equol in *Lactococcus* 20-92 is proposed.

**MATERIALS AND METHODS**

**Chemicals.** Daidzein, dihydrodaidzein (racemate), and equol were purchased from Fujicco Co. (Kobe, Japan), Toronto Research Chemicals Inc. (North York, Ontario, Canada), and LC Laboratories (Woburn, MA), respectively. *cis*-Tetrahydrodaidzein and *trans*-tetrahydrodaidzein were synthesized from daidzein in-house, as previously described ([14]). The optical resolution of the dihydrodaidzein racemate was performed by Daicel Chemical Industries (Osaka, Japan), and we obtained two enantiomers, (*R*)-dihydrodaidzein and (*S*)-dihydrodaidzein.

**Bacteria and culture conditions.** *Escherichia coli* strain JM109 (TaKaRa Bio Inc., Otsu, Japan) was used as the host for the standard cloning experiments and was grown in Luria-Bertani (LB) broth containing ampicillin (50 μg/ml) at 37°C. *E. coli* strain BL21(DE3) (Novagen, Madison, WI) was used as the host for the recombinant protein expression and was grown in LB broth containing ampicillin (50 μg/ml) at 30°C.

**Dihydrodaidzein racemase assay.** Assays for the dihydrodaidzein racemase activity were performed by chiral high-performance liquid chromatography (HPLC) monitoring of the racemization of each dihydrodaidzein enantiomer.

Received 14 February 2012 Accepted 30 April 2012  
Published ahead of print 11 May 2012  
Address correspondence to Yoshikazu Shimada, yshimada@research.otsuka.co.jp.  
Supplemental material for this article may be found at <http://aem.asm.org/>.  
Copyright © 2012, American Society for Microbiology. All Rights Reserved.  
doi:10.1128/AEM.00410-12

Dihydrodaidzein Racemase from *Lactococcus* Strain 20-92

![Diagram](#)

Previously reported Genomic region (Shimada et al. 2011)
IVN3-HindIII
IVN4-ApaI
IVN5-HindIII
IVC3-Sall
IVC4-SaI
IVC5-ApaI
IVC6-SmaI
IVC7-BamHI
2 kb

FIG 1 Physical map of the genomic region of the predicted gene cluster that is responsible for equol biosynthesis. The coding regions and orientations of the L-DZNR, L-DHDR, and L-THDR genes and open reading frames are indicated by the closed and open arrows, respectively. The bold line and the two-headed arrows below denote the genomic DNA region containing the L-DZNR, L-DHDR, and L-THDR genes that was previously reported (GenBank/EMBL/DDBJ accession no. AB593374) (14) and the DNA fragments that were amplified by inverse PCR in this study, respectively.

drodaidzein enantiomer. An enzyme source was added to the ice-cooled substrate solution, and the mixture was incubated at 37°C for 1 h. The final reaction mixture (1 ml) contained a 40 μM concentration of one of the dihydrodaidzein enantiomers, 2 mM dithiothreitol (DTT), and 5 mM sodium hydrosulfite in 0.1 M potassium phosphate buffer (KPB), pH 7.0. Following the incubation, the reaction solution was extracted with 3 ml of ethyl acetate. The organic solvent was vacuum dried, and the dried extract was dissolved in 0.5 ml of 55% (vol/vol) methanol-water. A 50-μl aliquot of the dissolved extract was then analyzed using an HPLC system (Shimadzu, Kyoto, Japan) equipped with an optically active SUMICHIRAL OA-7000 column (5-μm, 250-mm, and 4.6-mm internal diameter; Sumika Chemical Analysis Service, Osaka, Japan). The elution was performed isocratically with a 55% (vol/vol) methanol-water mobile phase for 30 min at 40°C at a flow rate of 1.0 ml/min, and the absorbance of the eluate was monitored at 280 nm.

Quantitation of isoflavones by HPLC. Quantification of the isoflavone metabolites (dihydrodaidzein, *cis*-tetrahydrodaidzein, *trans*-tetrahydrodaidzein, and equol) and daidzein was performed by HPLC according to a previously described method (14, 15).

Sequencing of the genomic DNA. For isolation of the upstream and downstream genomic regions adjacent to the gene cluster (L-DZNR, L-DHDR, and L-THDR genes), we performed step-by-step inverse PCR (11, 14) using previously constructed genomic DNA libraries (14) as the templates according to a previously described method (14). The nucleotide sequence of the entire genomic region that includes the putative equol-producing gene cluster was determined by aligning the eight DNA fragments which were obtained by step-by-step inverse PCR and the previously determined genomic DNA sequence (14) (Fig. 1) using SEQUENCHER software (Gene Codes Corp., Ann Arbor, MI).

Gene analysis. The nucleotide and amino acid sequence similarities were determined using the BLAST program (1), available on the website of the National Center for Biotechnology Information ([http://www.ncbi.nlm.nih.gov](http://www.ncbi.nlm.nih.gov)). The amino acid sequences of L-DDRC and methylmalonyl coenzyme A (methylmalonyl-CoA) epimerases from *Chlorobium* spp. were aligned using the CLUSTALW program (5), available on the website of the GenomeNet ([http://www.genome.jp](http://www.genome.jp)).

Construction of recombinant protein expression plasmid. For the construction of the expression plasmid, pET-US6-His, encoding the C-terminal histidine-tagged ORF-US6, the entire coding region of orf-US6 was amplified by PCR using primers exp.US6 F (5′-TATACATATGCAAGGCACAGCTCAACC-3′) and exp.US6 R (5′-GCTCGAATTCCACTTGCGTCCCAGTCGCAG-3′). The underlined sequences are the NdeI and EcoRI restriction sites, respectively. The pET-US6-His expression plasmid was obtained by digesting the PCR products with NdeI and EcoRI and then ligating the fragments into pET-21a(+) (Novagen) between the NdeI and EcoRI sites.

Purification of recombinant proteins. The recombinant US6-His was expressed by introducing the pET-US6-His plasmid into competent *E. coli* BL21(DE3) cells. Cultures of the transformants and *E. coli* BL21(DE3) cells containing pET-21a(+) were grown at 30°C until the optical density at 600 nm (OD~600~) was 0.8. The expression of the recombinant US6-His protein was then induced with 1 mM isopropyl-beta-D-thiogalactopyranoside (IPTG). The cells were then cultured for another 3 h at 30°C and harvested by centrifugation at 5,000 × g for 20 min. The collected cells were resuspended with BugBuster protein extraction solution (Novagen) containing 15,000 U/ml lysozyme (Sigma-Aldrich, St. Louis, MO), 25 U/ml Benzonase (Novagen), 0.5 mM DTT, and 0.5 mM phenylmethylsulfonyl fluoride (PMSF) and rotated slowly at room temperature for 30 min. The resulting lysate was centrifuged at 10,000 × g for 20 min, and the supernatant was applied to His GraviTrap columns (GE Healthcare, Uppsala, Sweden) equilibrated with buffer (20 mM Tris-HCl, 500 mM NaCl, 0.5 mM DTT, and 0.5 mM PMSF) containing 20 mM imidazole (Wako Pure Chemicals), pH 7.4. The column was washed twice with 20 ml of the buffer containing 40 mM imidazole, and the bound protein was eluted with the buffer containing 500 mM imidazole. The eluate was collected and assayed by SDS-PAGE using a 5 to 20% (wt/vol) gradient polyacrylamide gel (SuperSep; Wako Pure Chemicals). The protein on the SDS-PAGE gel was stained and visualized with QuickBlue staining solution (BioDynamics Laboratory Inc., Tokyo, Japan). The other three His-tagged recombinant proteins, L-DZNR-His, L-DHDR-His, and L-THDR-His, were expressed and purified by the same methods as described previously (14, 15).

Production of equol by recombinant enzyme mixtures. For the production of equol from daidzein *in vitro*, the purified recombinant enzymes (L-DZNR-His, L-DHDR-His, L-THDR-His, and US6-His; 2 μg each) were added to a reaction mixture (0.1 M sodium phosphate, pH 7.0, containing 2 mM DTT, 10 mM sodium hydrosulfite, and 40 μM daidzein) with a final volume of 1 ml. After incubation at 37°C for 1 h, the reaction solutions were extracted with 3 ml of ethyl acetate, vacuum dried, and dissolved in solvents. The production of equol and other metabolic intermediates was analyzed by HPLC, as described above.

Nucleotide sequence accession number. The sequence of the L-DDRC gene (orf-US6) has been deposited in the GenBank/EMBL/DDBL/DDBJ database under the accession number AB694972.

## RESULTS

Sequencing and gene analysis of the genomic region surrounding the equol-producing gene cluster. To identify the expected dihydrodaidzein racemase, we performed step-by-step inverse PCR analysis to amplify the upstream and downstream regions of the equol-producing gene cluster (14). Three (IVN3-HindIII, 4.4 kb; IVN4-ApaI, 2.1 kb; and IVN5-HindIII, 3.9 kb) and five (IVC3-SalI, 1.2 kb; IVC4-SaI, 1.2 kb; IVC5-ApaI, 4.1 kb; IVC6-SmaI, 6.5 kb; and IVC7-BamHI, 4.5 kb) DNA fragments were obtained for the upstream and downstream regions, respectively. The nucleo-

Shimada et al.

L-DDRC
C. phaeobacteroides_MMCE
C. parvum_MMCE

MIKAQLNR|ALRAADADKAVEDLNKLLGVT--FYGPYDDEHMGLRVALPKSG--GIEMAPMHDDAIGAYKALQTVGEGISGIAMRVDDFDAAKEHFAS 96
-MIKKIDHIAIAVENLERAIEFKSVLGCASETIIIEEVASENVRVALIPLGESRIELLOPLGDDNAIARFLAKN--GEGMHHIALETD DISHETERLES 107
-MIDRIDHIAIAVQDLNSANDTFVAVGGERSSVTIHEVPAEKVRVAFIPV6QTKIELLEPLGDDGAIAKFLSKN--GEGMHHIALATDDL NRETBERAAG 107

L-DDRC
C. phaeobacteroides_MMCE
C. parvum_MMCE

LGLTPVVEFNHGKFREMFPPQPATHNLEIAVNEFPDANGAAIQVALDMGVDWTDVCDWDAK 158
SGLKP_LGSQKEGAGGKQIIFLHPKOTNRVILLEITQEPH 145
LGLTPLGEPSEGANDKRIVFLHPRQTSHVLVEYVEPKSPHQPSQQS 153

FIG 2 Sequence alignment of the deduced amino acid sequences of orf-US6 (L-DDRC) from Lactococcus strain 20-92 and methylmalonyl-CoA epimerases from Chlorobium phaeobacteroides (GenBank/EMBL/DDBJ accession no. ABL65204) and Chlorobium parvum (accession no. ACF11246). The amino acids are numbered on the right side. Identical amino acid residues are indicated in black.

DISCUSSION

In this study, we have succeeded in the identification of the gene encoding dihydrodaidzein racemase (L-DDRC) in the equol-producing gene cluster of Lactococcus strain 20-92 (15) by the cloning and expression of this gene (Fig. 1).

The deduced amino acid sequence of L-DDRC was weakly similar to the amino acid sequences of bacterial methylmalonyl-CoA epimerases (EC 5.1.99.1). Figure 2 shows the alignment of the deduced amino acid sequences for orf-US6 and the methylmalonyl-CoA epimerases derived from Chlorobium phaeobacteroides (GenBank/EMBL/DDBJ accession no. ABL65204; 28.5% identity) and Chlorobium parvum (accession no. ACF11246; 30.2% identity). orf-US6 consists of an open reading frame of 477 nucleotides, and the deduced amino acid sequence consists of 158 amino acids (see Fig. S1 in the supplemental material).

Expression and purification of the recombinant US6-His. To investigate the activity of the US6-His protein, we prepared lysate from E. coli BL21(DE) cells harboring pET-US6-His and purified the recombinant US6-His protein. Our SDS-PAGE analysis of the purified recombinant protein showed one major band with a molecular mass of approximately 20 kDa, which is consistent with the expected size (Fig. 3).

Dihydrodaidzein racemase activity of the recombinant US6-His. To confirm the dihydrodaidzein racemase activity of the recombinant US6-His, we performed enzyme assays using the purified recombinant US6-His protein. As shown in Fig. 4, when dihydrodaidzein enantiomers were individually coincubated with the recombinant US6-His, the resulting dihydrodaidzein was detected as the racemate in each case, showing that this recombinant enzyme is able to convert each dihydrodaidzein enantiomer into the other to yield the racemate. Accordingly, we concluded that orf-US6 of Lactococcus strain 20-92 encodes a dihydrodaidzein racemase (L-DDRC).

Production of equol by the recombinant enzyme mixture. We attempted to synthesize equol from daidzein in vitro using an enzyme mixture consisting of recombinant L-DZNR-His, L-DHDR-His, L-THDR-His, and US6-His (L-DDRC-His) and observed efficient equol production (35.7 μM; 89.4% yield) from daidzein (40 μM) (Fig. 5a). In contrast, without the addition of the recombinant US6-His protein, the amount of produced equol was small (6.1 μM; 15.3%). Significant amounts of dihydrodaidzein (7.8 μM; 19.6%) and cis-tetrahydrodaidzein (18.5 μM; 46.3%) were also detected (Fig. 5b).

FIG 3 SDS-PAGE analysis of the purified recombinant US6-His (L-DDRC-His): 1 μg of the purified L-DDRC-His protein was separated. Left, molecular mass (MM) standards; right, purified L-DDRC-His. The arrow indicates a band corresponding to the purified L-DDRC-His protein.

three-enzyme mixture. These results clearly showed that L-DDRC plays an essential role in equol biosynthesis in *Lactococcus* strain 20-92 and that equol can be efficiently produced from daidzein in a cell-free system using the following four enzymes: L-DZNR, L-DHDR, L-THDR, and L-DDRC.

It is reported that (S)-equol, but not (R)-equol, was exclusively produced from tetrahydrodaidzein in enantiomeric mixtures of dihydrodaidzein in *in vitro* fermentation experiments using *Eggerthella* sp. strain Julong 732 (17). Furthermore, the hypothesis that the stereoselective production of (S)-equol might be the result of the enantioselective preference of the Julong 732 strain for the starting compound, (S)-dihydrodaidzein, and that the stereochemistry at the C-3 position was subsequently unchanged during the course of the equol biosynthesis from dihydrodaidzein was proposed (8, 17).

Therefore, the results described above for the *in vitro* production of equol by the recombinant enzyme mixture contradict the hypothesis that the stereochemical C-3 configurations of the intermediates are retained throughout the course of equol biosynthesis from (S)-dihydrodaidzein (17), as (S)-dihydrodaidzein is the major product from daidzein by L-DZNR (15). Kim et al. (7) showed that the DHDR-catalyzed product from dihydrodaidzein is (3R,4S)-tetrahydrodaidzein, which was then converted to (3S)-equol accompanying a stereochemical inversion at C-3; however, this pathway seems unlikely. In fact, the same authors also indisputably reported later that only (3S,4R)-tetrahydrodaidzein is produced from dihydrodaidzein and is then converted into (3S)-equol (8).

When we converted daidzein into dihydrodaidzein using recombinant L-DZNR-His, the stereochemical configuration of dihydrodaidzein was determined by the elution order in the chiral HPLC analysis, using the data from Wang et al. (18) as a reference. Although further study to assign the exact stereochemical configuration of the dihydrodaidzein produced by L-DZNR-His is necessary, the configuration of the dihydrodaidzein produced by L-DZNR-His may not be (S)-dihydrodaidzein but, rather, (R)-dihydrodaidzein. In fact, Park et al. (12) recently reported that the dihydrodaidzein produced from daidzein by an anaerobic human intestinal microorganism, MRG-1, was (R)-dihydrodaidzein and not (S)-dihydrodaidzein and that the elution order of each enantiomer during chiral HPLC was opposite to that of the report of Wang et al. (18). Therefore, in the present study (Fig. 4), the dihydrodaidzein that eluted earlier during chiral HPLC was designated (S)-dihydrodaidzein, and the later-eluting molecule was designated (R)-dihydrodaidzein. Thus, the assignment is different from that in our previous report (15).

If the DZNR-produced dihydrodaidzein is indeed the R form, the biosynthetic pathway from daidzein to (S)-equol could be explained as shown in Fig. 6, based on the following results. We found that the reduction reaction of dihydrodaidzein to tetrahydrodaidzein catalyzed by L-DHDR-His is a diastereoselective reduction: individual dihydrodaidzein enantiomers, (R)-dihydrodaidzein and (S)-dihydrodaidzein, are converted by L-DHDR-His into *cis*-tetrahydrodaidzein and *trans*-tetrahydrodaidzein, respectively (see Fig. S2 in the supplemental material). Furthermore, L-DHDR-His demonstrated a preference for the conversion of (S)-dihydrodaidzein into *trans*-tetrahydrodaidzein rather than the conversion of (R)-dihydrodaidzein into *cis*-tetrahydrodaidzein (see Fig. S3 in the supplemental material). These results support the idea that *trans*-tetrahydrodaidzein, not *cis*-tetrahydro-

Shimada et al.

![Image Description](image1.png)

FIG 5 Elution profiles of the reaction products using the recombinant enzyme mixture and daidzein in the presence (a) or absence (b) of recombinant L-DDRC-His (US6-His). The elution profiles of the reference standards daidzein (DZN), dihydrodaidzein (DHD), *cis*-tetrahydrodaidzein (*c*-THD), *trans*-tetrahydrodaidzein (*t*-THD), and equol (EQL) are shown in the “STD” panel.

daidzein, was derived from (*S*)-dihydrodaidzein and was converted into the final product, (*S*)-equol. Moreover, we have reported that both L-DHDR and L-THDR possess the reverse activity, catalyzing the conversion of tetrahydrodaidzein into dihydrodaidzein (15). Therefore, the *cis*-tetrahydrodaidzein produced by L-DHDR is likely to be converted back into (*R*)-dihydrodaidzein and then converted by L-DDRC into (*S*)-dihydrodaidzein, an L-DHDR-available substrate for (*S*)-equol production. The facts that although the amount of equol produced from daidzein was small when using the mixture of three recombinant enzymes (no L-DDRC-His), significant amounts of dihydrodaidzein and *cis*-tetrahydrodaidzein were detected as intermediates (Fig. 5b) and that *cis*-tetrahydrodaidzein did not appear in the reaction products using the mixture of four recombinant enzymes, which exhibited high equol production, further support the explanations given above.

The reports describing the biosynthetic pathway from daidzein to (*S*)-equol and the stereochemistry of the intermediates, especially with regard to the C-3 position, are complicated and controversial. However, it is reasonable to assume that daidzein is first converted into (*3R*)-dihydrodaidzein by L-DZNR and then into (*3S*)-dihydrodaidzein by L-DDRC. The final product, (*3S*)-equol, is generated from (*3S*)-*trans*-tetrahydrodaidzein by L-THDR following the conversion of (*3S*)-dihydrodaidzein into (*3S*)-*trans*-tetrahydrodaidzein. This biosynthetic pathway is feasible in that the configuration at C-3 is retained through the serial reductive reactions and that this pathway can provide an explanation as to why there is another enzyme, L-DDRC, in the equol biosynthetic

![Image Description](image2.png)

FIG 6 Model of the equol biosynthetic pathway starting from daidzein in *Lactococcus* strain 20-92. The enzyme activities of L-DZNR, L-DHDR, L-THDR, and L-DDRC are indicated by a green, blue, orange, and red arrow(s), respectively. First, daidzein is converted into (*R*)-dihydrodaidzein by L-DZNR. The (*R*)-dihydrodaidzein produced is then rapidly racemized by L-DDRC, and the produced (*S*)-dihydrodaidzein is then preferentially converted into *trans*-tetrahydrodaidzein by L-DHDR. Subsequently, *trans*-tetrahydrodaidzein is converted into (*S*)-equol by L-THDR. Abbreviations are the same as for Fig. 5.

pathway from daidzein to equol. In the equol biosynthetic pathway, L-DDRC may play the role equivalent to that of methylmalonyl-CoA epimerase in β-oxidation of odd-numbered-chain fatty acids (3).

Thus, the main avenue of the biosynthetic pathway from daidzein to (S)-equol seems to be elucidated; however, uncertainties about several details remain. In fact, in addition to the complexity of the pathway, including the stereochemistry, the functions of most of the genes in the equol-producing gene cluster have not yet been characterized. The functions of these gene products and the mechanisms of equol biosynthesis are expected to be clarified in greater detail in the near future.

### ACKNOWLEDGMENTS

We thank the staff of the Institute of Biomedical Innovation, the Saga Nutraceuticals Research Institute, and the Microbiological Institute of Otsuka Pharmaceutical Co., Ltd., for technical assistance.

### REFERENCES

1. Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. Basic local alignment search tool. J. Mol. Biol. 215:403–410.
2. Arai Y, et al. 2000. Comparison of isoflavones among dietary intake, plasma concentration and urinary excretion for estimation of phytoestrogen intake. J. Epidemiol. 10:127–135.
3. Bobik TA, Rasche ME. 2004. Purification and partial characterization of the *Pyrococcus horikoshii* methylmalonyl-CoA epimerase. Appl. Microbiol. Biotechnol. 63:682–685.
4. Bowey E, Adlercreutz H, Rowland I. 2003. Metabolism of isoflavones and lignans by the gut microflora: a study in germ-free and human flora associated rats. Food Chem. Toxicol. 41:631–636.
5. Chenna R, et al. 2003. Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res. 31:3497–3500.
6. Fortina MG, et al. 2003. Genetic characterization of some lactic acid bacteria occurring in an artisanal protected denomination origin (PDO) Italian cheese, *Toma piemontese*. Food Microbiol. 20:397–404.
7. Kim M, et al. 2009. Stereospecific biotransformation of dihydrodaidzein into (3S)-equol by the human intestinal bacterium *Eggerthella* strain Julong 732. Appl. Environ. Microbiol. 75:3062–3068.
8. Kim M, Marsh EN, Kim SU, Han J. 2010. Conversion of (3S,4R)-tetrahydrodaidzein to (3S)-equol by THD reductase: proposed mechanism involving a radical intermediate. Biochemistry 49:5582–5587.
9. Kostelac D, Rechkemmer G, Briviba K. 2003. Phytoestrogens modulate binding response of estrogen receptors alpha and beta to the estrogen response element. J. Agric. Food Chem. 51:7632–7635.
10. Morito K, et al. 2001. Interaction of phytoestrogens with estrogen receptors alpha and beta. Biol. Pharm. Bull. 24:351–356.
11. Ochman H, Gerber AS, Hartl DL. 1988. Genetic applications of an inverse polymerase chain reaction. Genetics 120:621–623.
12. Park HY, Kim M, Han J. 2011. Stereospecific microbial production of isoflavones from isoflavones and isoflavone glucosides. Appl. Microbiol. Biotechnol. 91:1173–1181.
13. Setchell KD, Brown NM, Olsen EL. 2002. The clinical importance of the metabolite equol—a clue to the effectiveness of soy and its isoflavone. J. Nutr. 132:3577–3584.
14. Shimada Y, et al. 2011. Identification of two novel reductases involved in equol biosynthesis for *Lactococcus* strain 20-92. J. Mol. Microbiol. Biotechnol. 21:160–172.
15. Shimada Y, et al. 2010. Cloning and expression of a novel NADP(H)-dependent daidzein reductase, an enzyme involved in the metabolism of daidzein, from equol-producing *Lactococcus garvieae* 20-92. Appl. Environ. Microbiol. 76:5892–5901.
16. Uchiyama S, Ueno T, Suzuki T. 2007. Identification of a newly isolated equol-producing lactic acid bacterium from the human feces. J. Intestinal Microbiol. 21:217–220.
17. Wang XL, Hur HG, Lee JH, Kim KT, Kim SI. 2005. Enantioselective synthesis of S-equol from dihydrodaidzein by a newly isolated anaerobic human intestinal bacterium. Appl. Environ. Microbiol. 71:214–219.
18. Wang XL, Shin KH, Hur HG, Kim SI. 2005. Enhanced biosynthesis of dihydrodaidzein and dihydrogenistein by a newly isolated bovine rumen anaerobic bacterium. J. Biotechnol. 115:261–269.
