
CHAPTER 6

Bacteriophage Protein–Protein Interactions

Roman Häuser,${}^{*,†}$ Sonja Blasche,${}^{†}$ Terje Dokland,${}^{‡}$ Elisabeth Haggård-Ljungquist,${}^{§}$ Albrecht von Brunn,${}^{\|}$ Margarita Salas,${}^{\dagger\dagger}$ Sherwood Casjens,${}^{\#}$ Ian Molineux,${}^{**}$ and Peter Uetz${}^{\dagger\dagger,1}$

---

Contents  
I. Introduction                                                                                       221  
II. Methods Used to Detect Protein–Protein Interactions                                              222  
A. Genetic detection of interactions                                                              222  
B. Electron microscopy                                                                           222  
C. Yeast two-hybrid screens                                                                      224  
D. Protein complex purification and mass spectrometry                                               224  
E. Biochemistry of protein complexes                                                              225  
F. X-ray crystallography and nuclear magnetic resonance                                            225  
III. Protein–Protein Interactions of Bacteriophages                                                  226  
A. Coliphage T7                                                                                  226  
B. Coliphage $\lambda$                                                                           232  

* Institute of Toxicology and Genetics, Karlsruhe Institute of Technology, Karlsruhe, Germany  
† Deutsches Krebsforschungszentrum, Heidelberg, Germany  
‡ Department of Microbiology, University of Alabama at Birmingham, Birmingham, Alabama, USA  
§ Department of Genetics, Microbiology and Toxicology, Stockholm University, SE-106 91 Stockholm, Sweden  
$\|$ Max-von-Pettenkofer-Institut, Lehrstuhl Virologie, Ludwig-Maximilians-Universität, München, Germany  
$\dagger\dagger$ Centro de Biología Molecular “Severo Ochoa” (CSIC-UAM), Cantoblanco, Madrid, Spain  
# Division of Microbiology and Immunology, Pathology Department, University of Utah School of Medicine, Salt Lake City, Utah  
** Molecular Genetics and Microbiology, Institute for Cell and Molecular Biology, University of Texas–Austin, Austin, Texas, USA  
$\dagger\dagger$ Center for the Study of Biological Complexity, Virginia Commonwealth University, Richmond, Virginia, USA  
$^{1}$ Corresponding author, E-mail: peter@uetz.us

Advances in Virus Research, Volume 83  
ISSN 0065-3527, DOI: 10.1016/B978-0-12-394438-2.00006-2  
© 2012 Elsevier Inc. All rights reserved.

C. Salmonella phage P22                                      246
D. Enterobacteriophage P2 and its satellite phage, P4         253
E. Pneumococcus phage Dp-1 and Cp-1                          263
F. Bacillus subtilis phage φ29-like viruses                  266
IV. Conclusions                                              275
A. High-throughput data vs small-scale interactions          275
B. Phage of the same host has                                276
   different interactomes                                    276
C. Evolution of interactions [λ, P22]                        276
Acknowledgments                                             277
References                                                  277

---

Abstract  
Bacteriophages T7, λ, P22, and P2/P4 (from *Escherichia coli*), as well as φ29 (from *Bacillus subtilis*), are among the best-studied bacterial viruses. This chapter summarizes published protein interaction data of intraviral protein interactions, as well as known phage-host protein interactions of these phages retrieved from the literature. We also review the published results of comprehensive protein interaction analyses of *Pneumococcus* phages Dp-1 and Cp-1, as well as coliphages λ and T7. For example, the ≈55 proteins encoded by the T7 genome are connected by ≈43 interactions with another ≈15 between the phage and its host. The chapter compiles published interactions for the well-studied phages λ (33 intra-phage/22 phage-host), P22 (38/9), P2/P4 (14/3), and φ29 (20/2). We discuss whether different interaction patterns reflect different phage lifestyles or whether they may be artifacts of sampling. Phages that infect the same host can interact with different host target proteins, as exemplified by *E. coli* phage λ and T7. Despite decades of intensive investigation, only a fraction of these phage interactomes are known. Technical limitations and a lack of depth in many studies explain the gaps in our knowledge. Strategies to complete current interactome maps are described. Although limited space precludes detailed overviews of phage molecular biology, this compilation will allow future studies to put interaction data into the context of phage biology.

---

ABBREVIATIONS:  
EM     electron microscopy  
LC     liquid chromatography  
MS     mass spectrometry  
NMR    nuclear magnetic resonance  
ORF    open reading frame  
PPI    protein–protein interaction  
Y2H    yeast two-hybrid

# I. INTRODUCTION

Bacteriophages have been extremely important model systems in molecular biology. They were critical to many breakthrough discoveries, such as the nature of the genetic material (Hershey, 1952), the structure of genes and genomes and the first genome sequence (Sanger *et al.*, 1977), recombination (Kaiser and Jacob, 1957), transcriptional regulation (e.g., Roberts, 1969), and many others. In addition, phages have become important technical tools in molecular genetics and nanobiology, but these aspects are not reviewed here.

A great deal of work has been invested in learning the mechanistic details of the life cycles of a number of phages, including those described in this chapter. Although phages are tiny biological systems, surprisingly, not even the biology of main “model system phages” is completely understood. Despite a plethora of papers, few comprehensive summaries exist of the molecular interaction events that occur in both phage gene regulation and virion assembly. As in all organisms, protein–protein interactions (PPIs) play a fundamental role in phage biology. PPIs are important in all virus-related processes, including host infection, transcriptional regulation, DNA replication, virion assembly, and lysis. Current protein interaction databases (Lehne and Schlitt, 2009) have consistently neglected phage interactions. For instance, Intact, one of the most comprehensive databases for protein–protein interactions, listed only 22 interactions for Caudovirales, the only phage group represented at the time of this writing (April 2011). We have thus attempted to compile such information for several phages that infect Gram-negative hosts, including λ, T7, P22, and P2/P4, as well as others that infect Gram-positive bacteria, specifically φ29, Dp-1, and Cp-1 (Table I). Coliphage

## TABLE I Summary of phage interactomes described in this chapter${}^{a}$

| Phage   | Proteins | Phage–phage PPIs          | Phage–host PPIs |
|---------|----------|---------------------------|------------------|
| T7      | 55       | ~43 (including 7 HTS${}^{b}$) | 15               |
| λ       | 73       | 33 (+ 81 HTS)             | 26               |
| P22     | 64       | 38                        | 9                |
| P2/P4   | 42/13    | 14                        | 3                |
| φ29     | 30       | 20                        | 3                |
| Dp-1    | 72       | 156 (HTS)                 | 38 (HTS)         |
| Cp-1    | 28       | 17 (HTS)                  | 11 (HTS)         |

${}^{a}$ Note that all interactions from this chapter are available as a downloadable supplement from [http://www.uetz.us](http://www.uetz.us).

${}^{b}$ High-throughput data (see text for details).
T4 is one of the best understood of all the obligate lytic phages. T4 has been an outstanding model for phage biology, particularly for DNA replication, control of transcription, and virion assembly. Unfortunately, this chapter cannot discuss the extensive T4 PPIs here for reasons of space. Many of the small DNA phages, such as φX174 or M13, have been very well studied, but because of their tiny proteomes they do not have many protein–protein interactions. It would be highly desirable to establish their PPIs so that more comprehensive analyses of phage systems biology would be possible. Nevertheless, it is hoped that this chapter proves useful to the phage community and encourages similar compilations for other phages. The interactions collected for this chapter are available in Intact ([http://www.ebi.ac.uk/intact/](http://www.ebi.ac.uk/intact/)) for easy access.

## II. METHODS USED TO DETECT PROTEIN–PROTEIN INTERACTIONS

### A. Genetic detection of interactions

Methods used to study protein–protein interactions in phages have evolved over the decades. A few important methods are illustrated in Figure 1. In the beginnings of phage molecular biology, detection of genetic interactions dominated. For instance, certain mutants in the λ J gene abolished virion binding to host *Escherichia coli* cells. Similarly, mutants of the *E. coli* gene *lamB* made the cells resistant to λ infections. From such experiments it was concluded that J protein bound to LamB and that LamB was the λ receptor. This hypothesis has been confirmed by isolating compensatory mutations in both genes that regenerate the interaction (*Werts et al.*, 1994).

### B. Electron microscopy

Another important method for the investigation of physical relationships between phage components is electron microscopy, especially in combination with genetic analysis. Mutants often have an aberrant morphology. If certain structures are missing in the mutant virion, it can be concluded that the mutated gene encodes this structure or is at least involved in its assembly. For instance, many laboratory strains of λ harbor a frameshift mutation in the *stf* gene, and these mutants lack side tail fibers (*Hendrix and Duda*, 1992). Many such studies not only showed which genes encode which proteins, but also indicated how the proteins interact (e.g., when particular structures could be “removed” from the virion by defined gene knockouts).
A Yeast two-hybrid
B Split-YFP

C LUMIER
D affinity purification

FIGURE 1 Selected methods for the study of protein–protein interactions. (A) The yeast two-hybrid (Y2H) system is based on the levels of two fusion proteins (typically in yeast but any cell type can be used). One of the proteins contains a DNA-binding domain (DBD), which can bind to the promoter of a reporter gene (here: HIS3), and a second protein X, the bait. The second fusion protein consists of a transcription-activation domain (AD) and a second protein, Y. If proteins X and Y interact, a transcription factor is formed and the reporter gene is activated. In this case, that means that the cell can grow on histidine-free medium. A yeast colony growing on such medium thus indicates an interaction of the two inserted proteins. (B) Protein complementation assay (PCA), e.g., split-YFP. As in the Y2H assay, two interacting proteins bring together two protein fragments that are inactive when separate but active when in close proximity. Here, fragments of yellow-fluorescent protein (YFP) reassociate and fluoresce when reassembled. Other fluorescent proteins, such as the green fluorescent protein (GFP), have been used in a similar way. (C) LUMIER (LUMInescence-based mammalian intERactome). Two fusion proteins are purified by means of an epitope tag (here: FLAG tag), usually on an antibody-coated matrix. Interactions between X and Y can be detected using Luciferase, whose gene is fused to Y and which emits light when luciferin is added. (D) Affinity purification. Protein complexes can be purified from cellular lysates using an affinity epitope, as in (C) with a FLAG tag. Components of the complex can then be identified using mass spectrometry, using the unique mass of peptides when the protein is digested by trypsin.

More recently, many bacteriophage capsids and phage protein complexes have been analyzed by cryoelectron microscopy and three-dimensional reconstructions. This technique gives resolutions approaching
that of X-ray crystallography, especially for proteins present in symmetric arrays in the virion, and has proven ideal for analyzing protein–protein interactions in large complexes, especially in combination with high-resolution structural data from other techniques, such as X-ray crystallography.

### C. Yeast two-hybrid screens

More recently, the yeast two-hybrid system (Y2H) has become an important method to detect pairwise protein interactions. Interaction screens of phage proteins can be carried out easily by cloning all open reading frames in appropriate vectors and then testing these in a systematic pairwise fashion. This has been done for several phages, including coliphages T7 ([Bartel et al., 1996](https://doi.org/10.1038/384485a0)) and λ ([Rajagopala et al., 2011](https://doi.org/10.1038/nature09977)), as well as *Streptococcus pneumoniae* phages Cp-1 and Dp-1 ([Häuser et al., 2011](https://doi.org/10.1038/srep00001); [Sabri et al., 2010](https://doi.org/10.1038/nature09007)). The whole process can also be automated.

One disadvantage of the Y2H system is that a typical pairwise screen, also called an “array screen” because of the systematic arrangement of clones in microtiter plates, may only detect 20–30% of all interactions. That is, the rate of false negatives is on the order of 70–80%. This can, in theory, be overcome by using multiple vectors that differ in the structure of the fusion proteins, for instance in the production of N- or C-terminal fusion proteins ([Stellberger et al., 2010](https://doi.org/10.1038/nmeth.1488)). A combination of multiple vectors can achieve interaction detection rates of up to 80% (i.e., false negative rates as low as 20%) ([Chen et al., 2010](https://doi.org/10.1038/nmeth.1488)).

Y2H also suffers from false positives that can only be understood with additional experimentation. Nonetheless, a multivector Y2H strategy can help in the recognition of false positives. For instance, if multiple vector combinations are used, interactions found in all or most cases are more likely to be “true positives,” whereas interactions detected in a lower fraction of cases are less reliable and may be false positives.

### D. Protein complex purification and mass spectrometry

Whole phage virions can be purified, typically by CsCl density gradient centrifugation. Proteins from purified phage can then be denatured and separated on polyacrylamide gels or subjected directly to trypsin digestion and subsequent liquid chromatography (LC) and mass spectrometry (MS). The mass of peptides determined by MS can usually identify the corresponding proteins unambiguously, and thus composition of the phage particles. Once all proteins in a virion are identified, at least some can be assigned easily to known structures, such as the icosahedral capsid or tail sheath. Typically, only a fraction of all phage-encoded proteins are represented in the mature virion (e.g., less than 20 of the ≈70 phage λ
proteins). Especially for little-studied phages it can be extremely helpful
to know which proteins are in the virion. For example, in an analysis
of *Streptococcus* phage Dp-1, only 8 of the 72 predicted proteins were
identified by LC/MS and were thus confirmed structural proteins
(see later).

### E. Biochemistry of protein complexes

Biochemical approaches often involve the purification of protein complexes
and thus yield protein interaction information. This approach can be made
technically easier if individual proteins are affinity tagged. For instance,
phage-encoded proteins can often be fused to affinity column-binding
proteins such as glutathione-S-transferase or oligohistidine without affecting
their function. Such tagged proteins and proteins bound to them can
then be purified using affinity reagents. The untagged proteins in the
complex can then be identified by mass spectrometry or immunoblotting
(if antibodies against specific proteins are available). A range of experimental
techniques is available for the characterization of phage protein
complexes and their activities; a comprehensive overview cannot be given
here. For instance, chemical cross-linking of interacting proteins is used
frequently to spatially locate physically associated proteins, and variable
spacer lengths can determine the steric proximity of the binding partners.
Other tests include activity assays (e.g., proteases or other modifying
enzymes that may be required for phage assembly), replication, or other
processes. Some examples of such activities are described here.

### F. X-ray crystallography and nuclear magnetic resonance

Only atomic structures of all phage proteins and their interactions will
give a mechanistic understanding of phage biology, and progress in this
area has been accelerating. Individual proteins and protein complexes, if
not the whole phage particle, can be crystallized and the structure of such
proteins solved by X-ray analysis. Alternatively, protein structures can be
solved by nuclear magnetic resonance (NMR) if high enough concentrations
can be achieved. Unfortunately, size restrictions on NMR structural
determinations limit the analysis of large protein complexes, but monomeric
structures can be extremely useful in devising testable models for
the interactions proteins may undergo. However, many phage proteins
are refractory to structural analysis (especially those involved in virion
assembly, as they often assemble only at high concentrations), and even in
classical model systems such as phage λ less than a third of all phage-
encoded proteins have been crystallized—most of these as pure proteins
rather than as protein complexes (*Rajagopala et al.*, 2011; *Yura et al.*, 2006).

III. PROTEIN–PROTEIN INTERACTIONS OF  
BACTERIOPHAGES

A. Coliphage T7

Bacteriophage T7 was defined in 1945 as a member of the seven Type ("T") phages that grow lytically on *E. coli* B (Demerec and Fano, 1945), although it is probably identical to phage δ, used earlier by Delbrück; a close relative of T7 was likely studied by d’Herelle in the 1920s (d’Herelle, 1926). Relatives of T7 exhibit high synteny across their entire genomes, with the closest relatives differing primarily only by the number and location of homing endonucleases or other selfish elements not known to have any role in phage development. These phages therefore will exhibit life cycles and PPIs that parallel T7 itself, modified only by the resources available in their host organisms.

1. Growth physiology

T7 grows on rough strains of *E. coli* (i.e., those without full-length O-antigen polysaccharide on their surface) and some other enteric bacteria, but close relatives have been described that infect smooth and even encapsulated strains (Molineux, 2006a). In such phages, tail fibers present on T7 are replaced with tailspikes with enzymatic activity that degrades the O- or K-antigens on the cell surface. T7 has a short latent period of 17 min at 37°C, which has resulted in most physiological studies being conducted at 30°C where infected cells lyse after 30 min. However, adaptation of T7 to high fitness with an excess of an *E. coli* K-12 strain growing under optimal conditions at 37°C in rich media results in a phage with a latent period of only ≈11 min. This adapted phage can undergo an effective expansion of its population by more than 10<sup>13</sup> in 1 hr of growth (Heineman and Bull, 2007). There is no obvious subtlety in the T7 developmental pathway: the early region of its genome is transcribed efficiently by the host RNA polymerase, leading to synthesis of a phage-specific RNA polymerase and the concurrent inhibition of the host enzyme. Thus, T7 rapidly takes control of the transcriptional capacity of the infected cell. As *E. coli* mRNAs are also intrinsically unstable, T7 therefore also gains control of the translational capacity of the cell. Further, because T7 uses nucleotides derived from the host chromosome for its own replication, more than 80% of the DNA in progeny is derived from the bacterial chromosome and only a small burst is observed after infection of DNA-free minicells. T7 development is not known to be dependent on host chaperones, although it does code for two proteins that have been suggested to have chaperone activity in capsid and tail assembly. Aside from host components used for phage adsorption, plus general biosynthetic processes and the translational apparatus, T7 development

is remarkably independent of host proteins: only *E. coli* RNA polymerase, thioredoxin—the cofactor for T7 DNA polymerase—and (deoxy)cytidine monophosphate kinase are known to be essential.

2. Early steps in infection

T7 is a member of *Podoviridae*, possessing only a short stubby tail that needs to be extended functionally in order to span the cell envelope, which thus allows its genome to be translocated into the cell. Tail extension is affected through internal head proteins that are ejected into the cell at the initiation of infection prior to or concomitant with transport of the leading end of the genome ([Chang et al., 2010](https://doi.org/10.1038/nature09456); [Kemp et al., 2005](https://doi.org/10.1038/nature03586)). A schematic of the T7 virion is shown in Figure 2. In T7, the core consists of several copies each of three proteins (gp14, gp15, and gp16), which must interact, albeit perhaps loosely, with themselves and each other, in the phage capsid. However, during their ejection into the cell they must at least partially dissociate and also partially unfold in order to pass through the narrow channel through the portal and tail ([Molineux, 2006b](https://doi.org/10.1016/j.femsre.2006.03.005)). The N-terminal domain of gp16 refolds spontaneously in the periplasm of the infected cell to form an enzyme with lytic transglycosylase activity ([Moak and Molineux, 2000](https://doi.org/10.1073/pnas.200080697), [2004](https://doi.org/10.1073/pnas.0402845101)). This muralytic activity is only conditionally essential for T7 infection ([Moak and Molineux, 2000](https://doi.org/10.1073/pnas.200080697)).

The C-terminal residues of gp16, likely together with a more central portion of gp15, also refold in the periplasm and then insert spontaneously into the cytoplasmic membrane, completing the channel across the cell envelope for DNA translocation into the cell. Likely the last of the core proteins to exit the phage head, gp14 is found exclusively within the outer membrane of the infected cell. Using energy derived from the membrane potential, these proteins then ratchet the leading 1 kb of the T7 genome into the cell ([Chang et al., 2010](https://doi.org/10.1038/nature09456); [Kemp et al., 2005](https://doi.org/10.1038/nature03586)). In both T7 and T3, genome internalization by this process stops, and the remainder of the DNA is normally pulled into the cytoplasm as a consequence of host and then phage RNA polymerase-catalyzed transcription ([Garcia and Molineux, 1995](https://doi.org/10.1073/pnas.92.11.5014), [1996](https://doi.org/10.1073/pnas.93.11.5549); [Kemp et al., 2004](https://doi.org/10.1073/pnas.0402845101)). Several strong promoters lie on this leading 1-kb segment of the T7 genome.

3. Early genes

Transcription from T7 early promoters not only internalizes the remainder of the genome but also leads to expression of early T7 genes, some of which interact and inhibit or modulate the activity of host proteins ([Molineux, 2006a](https://doi.org/10.1016/j.femsre.2006.03.005)). Of the early proteins, only T7 RNA polymerase is essential for phage growth. The first T7 protein to be synthesized is gp0.3, an anti-type I restriction protein ([Studier, 1975](https://doi.org/10.1073/pnas.72.10.4068)). The structure of gp0.3 reveals that it resembles double-stranded DNA, and the protein binds to the specificity subunit of the heteropentameric EcoKI enzyme,

228 Roman Häuser et al.

Early genes | DNA Replication | Virus structure and assembly
--- | --- | ---
T7 genome | 0.3 | 2 | 4 | 6 | 8 | 10 | 12 | 14 | 16 | 18 | 19.5
| 1 | 3 | 5 | 7 | 9 | 11 | 13 | 15 | 17 | 19

PifA  
FxsA  
Dgt (dGTPase)  

nucleotide kinase  
RNA pol.  
Lysozyme  
DNA ligase  
S/T Kinase  
endonuclease  
H-NS thioredoxin  
recBCD  

EcoKI  
EF-G  
RpsF  
RNase III  
RNase E  
RNAP (RpoC)  

E. coli proteins  

scaffolding  

Bartel et al. (1997)  
other literature  

FIGURE 2 Phage T7: genome, virion, and interactome. ORFeome and protein interaction map of bacteriophage T7. Cloned open reading frames (ORFs) and random fragments were used to generate a two-dimensional interaction map. ORFeomes can also be used by structural genomics projects to derive three-dimensional structures. A combination of interaction maps and crystal structures often allows reconstruction of protein complexes as in viral particles or their subunits. (Top) T7 genome with each ORF represented as a box. ORFs indicated by integral numbers were identified originally as essential by genetic screens. Other genes have decimal numbers; of these, only 2.5 and 7.3 are essential. Proteins found to interact with other proteins are indicated by colored boxes; colors indicate a simplified assignment to functional classes as shown. Interactions between proteins are shown as lines, with self-interactions shown as dimers. Hatched lines indicate expected interactions that have not been found by two-hybrid analysis. Gray proteins correspond to host proteins (*E. coli*). Blue proteins are involved in virus assembly and structure, and, where known, their location in the virus particle is shown on the right. Proteins with thick borders have been crystallized and their structure determined. Genetic map modified after **Dunn and Studier (1983)**. Figure modified after **Uetz et al. (2004)**. Interactions based primarily on data in **Table II** (see references therein).

preventing both it and the cognate modification enzyme from recognizing its target sequence (**Atanasiu et al., 2001, 2002; Kennaway et al., 2009; Murray, 2000; Stephanou et al., 2009**). T7 gp0.3 is active against all type I enzymes and thus T7 grows normally on B, C, and K-12 strains of *E. coli* regardless of its previous host. However, the protein affords no protection against type II and little, if any, against type III enzymes. Gp0.3 proteins of different T7-like phages fall into two distinct groups with no sequence

similarity; both have anti-restriction activity but the group typified by T3 gp0.3 also has SAMase activity (Studier and Movva, 1976). The second major early protein whose function is known is gp0.7, a serine-threonine kinase (Rahmsdorf *et al.*, 1974) that is also responsible for the shut off of host transcription. The two activities are separable (Simon and Studier, 1973). Gp0.7 phosphorylates several host proteins, including RNA polymerase (Severinova and Severinov, 2006; Zillig *et al.*, 1975), the translational apparatus including EF-G and the small ribosomal subunit protein S6 (Robertson and Nicholson, 1990, 1992; Robertson *et al.*, 1994), and both RNase III (Marchand *et al.*, 2001) and RNase E (Mayer and Schweiger, 1983). T7 shuts down translation of host mRNAs but it remains unclear whether EF-G or S6 is involved directly. An unknown T7 late gene also modulates the specificity of RNase E (Savalia *et al.*, 2010). Although it is likely that others do so, only 1 other of the 10 early T7 proteins is currently known to interact with host proteins. Gp1.2 binds to *E. coli* dGTPase, converting it into a rGTP-binding protein (Huber *et al.*, 1988; Nakai and Richardson, 1990); gp1.2 also binds to the membrane protein FxsA and the F plasmid protein PifA (Cheng *et al.*, 2004; Schmitt and Molineux, 1991; Wang *et al.*, 1999).

## 4. T7 RNA polymerase

The hallmark of T7 is of course its phage-encoded RNA polymerase, which, together with its regulator T7 lysozyme, is responsible for most T7 gene expression. It is also essential for synthesizing RNA primers at the primary origin of replication (Fujiyama *et al.*, 1981) and for initiation of DNA packaging (Zhang and Studier, 1997, 2004). In addition to several complexes containing a promoter and product RNAs (Cheetham and Steitz, 1999; Cheetham *et al.*, 1999; Kennedy *et al.*, 2007; Tahirov *et al.*, 2002; Yin and Steitz, 2002, 2004), the crystal structure of the RNA polymerase–lysozyme complex has been determined (Jeruzalmi and Steitz, 1998). Many mutants in both RNA polymerase and lysozyme have been characterized that affect their functional interaction, but it is not obvious from the crystal structures how the mutations affect the regulation of transcription. Lysozyme is thought to target the initial transcribing complex in a promoter strength-dependent manner and have no effect on transcription elongation (Huang *et al.*, 1999; Stano and Patel, 2004; Zhang and Studier, 1997, 2004). Lysozyme is, however, important in pausing or terminating transcription at the class II terminator CJ, where it probably interacts directly with the T7 terminase protein gp19 and thus directs the packaging machinery to create the DNA terminus (Lyakhov *et al.*, 1997, 1998; Zhang and Studier, 2004).

After T7 RNA polymerase has been synthesized, there is no further requirement for the host polymerase and it actually becomes inhibitory to phage development. Inactivation is caused by the T7 gp2 protein, which

binds to the β′ subunit and inhibits transcription by interfering with strand separation of the open complex at most promoters (Camara *et al.*, 2010; Nechaev and Severinov, 1999). In vivo, the only T7 promoter that must be inhibited by gp2 is A3 (Savalia *et al.*, 2010). Gp2 was found to interact with T7 endonuclease gp3 by a yeast two-hybrid screen (Bartel *et al.*, 1996). An interaction between gp2 and gp3 (albeit not necessarily direct) was suggested earlier (Mooney *et al.*, 1980), but the idea has not yet been critically evaluated experimentally.

5. DNA replication

T7 has been one of the primary model systems for understanding the biochemistry of DNA replication. Central to T7 replication is T7 DNA polymerase, which comprises a heterodimer of gp5 and the host thioredoxin. Binding of gp5 to thioredoxin is strong, with the latter serving as the processivity factor for DNA replication, increasing the length of DNA synthesized per binding event 100-fold. The interface between the two proteins also provides the docking site for binding of the other two proteins, SSB (gp2.5) and the primase–helicase gp4, which are central to T7 replication in vivo. The process of T7 replication in vitro has been reviewed recently (Hamdan and Richardson, 2009).

Two other proteins in the DNA replication cluster are known to interact with host proteins. The gene 5.5 product binds to the abundant nucleoid-associated protein H-NS (Ali *et al.*, 2011) and in vitro relieves its inhibition of both *E. coli* and T7 RNA polymerase-catalyzed transcription (Studier, 1981). In vivo, T7 5.5 mutants direct reduced rates of DNA replication and exhibit a small plaque phenotype (Lin, 1992; Liu and Richardson, 1993; Studier, 1981). However, the phenotype persists in *hns*-defective strains, although an *hns* mutant that renders gene 5.5 essential has been isolated (Liu and Richardson, 1993). Gp5.5 also interacts with the λ proteins of RexAB system; it is not known whether this is a direct protein–protein interaction or whether gp5.5 interacts with a product of RexAB activity. A gene 5.5 missense, but not a null mutant, makes T7 sensitive to Rex-mediated exclusion. The product of gene 5.9 inhibits the *E. coli* RecBCD nuclease in a comparable manner to λ Gam protein; this interaction is also nonessential as mutants lacking gene 5.9 grow normally in *rec(BCD)*⁺ hosts (Lin, 1992).

The assembly of T7 virions involves several symmetry mismatches and many protein–protein interactions, some of which may be helped by other T7 proteins that potentially have a chaperone-like activity (Kemp *et al.*, 2005). Note that, GenBank annotations notwithstanding, gp13 is not a structural component of the virion (Kemp *et al.*, 2005). Assembly is thought to initiate with oligomerization of the portal protein gp8 to form a dodecameric ring, although both 12- and 13-mers are observed following overexpression of gene 8 (Cerritelli and Studier, 1996b; Kemp

et al., 2005; Valpuesta et al., 1992, 2000). The pathway of prohead formation may then involve assembly of the internal core (gp14, gp15, and gp16) on the inner surface of the portal, perhaps concomitant with the recruitment of capsid protein gp10 hexamers bound to scaffolding protein gp9. Subsequent recruitment of gp10 pentamers and hexamers results in shell formation around a scaffold of gp9 (Cerritelli et al., 2003a). The 12-fold symmetrical portal protein faces a symmetry mismatch with the 5-fold vertex of the icosahedral capsid shell. The internal core of T7 contains 4 copies of gp16, 8 copies of gp15, and either 10 or 12 copies of gp14 (Agirrezabala et al., 2007; Cerritelli et al., 2003b; Kemp et al., 2005). These proteins are ejected into the infected cell in order to transport the phage genome into the cytoplasm, and the protein–protein interactions in the mature virion are likely quite different than those found in the infected cell. Gp14 lies at the base of the internal core and is therefore juxtaposed to the 12-fold symmetrical portal protein. Complicating our understanding of the T7 virion is the identification of ≈18 copies of the small gp6.7 protein, which is ejected into the outer membrane of the infected cell at the initiation of infection (Kemp et al., 2005). It is not known where gp6.7 is located; it likely forms part of the internal core or lies within the portal channel.

## 6. Tail

The outer face of the portal is the site of assembly of the sixfold symmetrical stubby tail. The tail is composed of three proteins—gp7.3, 11 and 12—to which six trimers of the tail fiber protein gp17 are attached. Each trimer consists of three domains: an N-terminal third that binds to the tail; this domain is conserved in phages related to T7 that have a different host range. This domain is followed by a triple-stranded coiled-coil of ≈120 residues and then a C-terminal half consisting of globular domains that interact with the cell surface lipopolysaccharide (Steven et al., 1988). The body of the tail consists of 12 copies of gp11, 6 copies of gp12, and ≈30 copies of the small protein gp7.3 (Kemp et al., 2005). How these proteins are arranged is unknown. It has been suggested that gp7.3 functions as a tail assembly protein that remains with the mature virion; the protein is ejected into the outer membrane of the infected cell (Chang et al., 2010; Kemp et al., 2005). Host range mutants of T7 suggest that all three proteins interact directly with the lipopolysaccharide of the host cell.

## 7. Interactome screening

T7 was the focus of the first genome-wide screen for phage protein–protein interactions using the yeast two-hybrid assay (Bartel et al., 1996). As proof of principle, T7 was a good choice: the extensive genetic, molecular biology, and biochemical studies that had been performed on T7 provided a solid basis for evaluating the accuracy and completeness of
the approach. Twenty-five interactions were identified in the screen, 15 of which were intramolecular. Many of these are to be expected as even monomeric polypeptides fold into a three-dimensional structure. Virion assembly clearly involves extensive protein–protein interactions, but as discussed by the authors, many of those that were known or plausibly expected from known structures were missed in the screen ([Bartel *et al.*, 1996](https://doi.org/10.1073/pnas.93.24.13604)). Conversely, screen-predicted interactions involving gp1.7 that were confirmed subsequently by its purification as an oligomer ([Tran *et al.*, 2010](https://doi.org/10.1073/pnas.1008276107)) and the predicted interactions between the spanin proteins gp18.5 and gp18.7 predated the discovery of a heterodimer even in phage λ ([Casjens *et al.*, 1989](https://doi.org/10.1016/0022-2836(89)90352-8)). Six intermolecular interactions were predicted that have not yet found confirmatory support, and our current understanding of T7 suggests that some are implausible. Improved screening procedures should help resolve these outstanding issues and, if more intermolecular interactions can be found that are consistent with, in particular, the pathways of virion assembly, the utility of those screening procedures in predicting the protein interactome of lesser-studied phages will be enhanced greatly. All published T7 interactions are summarized in Table II.

### B. Coliphage λ

Phage λ may be the best-studied temperate phage and has been under investigation since the early 1950s. There are probably thousands of papers that have been published on phage λ and its closer relatives N15, HK97, and P22. For instance, there are more than 400 citations in a review of lambdoid phages that do not even focus on the genetic switch of λ, the paradigm for gene regulation ([Hendrix and Casjens, 2006](https://doi.org/10.1016/j.mib.2006.02.012)). The switch has been described in detail by many other papers and in its own book ([Court *et al.*, 2007](https://doi.org/10.1016/S0076-6879(07)79001-9); [Ptashne, 2004](https://doi.org/10.1016/B978-0-12-564041-5.X5000-1)). This section summarizes current knowledge of the interactions among phage λ proteins. Finally, a summary of interactions with its host, *E. coli*, is given. For a more detailed description of phage λ biology and regulation, the authors refer the reader to other reviews ([Calendar, 2006](https://doi.org/10.1016/S0076-6879(06)79001-5); [Court *et al.*, 2007](https://doi.org/10.1016/S0076-6879(07)79001-9); [Hendrix and Casjens, 2006](https://doi.org/10.1016/j.mib.2006.02.012); [Hendrix *et al.*, 1983](https://doi.org/10.1016/0022-2836(83)90021-5); [Oppenheim *et al.*, 2005](https://doi.org/10.1016/j.jmb.2005.03.067); [Ptashne, 2004](https://doi.org/10.1016/B978-0-12-564041-5.X5000-1)).

#### 1. The λ virion

The phage λ genome and virion are shown in Figure 3. Note that the virion contains tail fibers that are not visible in most λ micrographs. The variant of λ used in most laboratories around the world, λPaPa, has a frameshift mutation in the structural *stf* gene and lacks the long “side tail fibers” of λ, Ur-λ, that comes directly from the original *E. coli* K-12 lysogen. It is still not entirely clear how many different proteins are in the λ particle but 12 are known with confidence and 2 more (gpM and gpL) are possibly in the particle in very low numbers. A current model of
TABLE II Protein-protein interactions of phage T7

| Protein | Function | Interacts with | References |
|---------|----------|----------------|------------|
| Class I |          |                |            |
| gp0.3   | B-DNA mimic | Host HsdRMS | Atanasiu et al., 2001, 2002; Kennaway et al., 2009; Murray, 2000; Stephanou et al., 2009 |
| gp0.3   |          | gp0.3 (dimer) | Atanasiu et al., 2001 |
| gp0.3   |          | gp4.5         | Bartel et al., 1996 |
| gp0.7   | Protein kinase | Host RNAP (activity change) | Rahmsdorf et al., 1974; Severinova and Severinov, 2006; Zillig et al., 1975 |
| gp0.7   |          | RNase III (activity change) | Mayer and Schweiger, 1983 |
| gp0.7   |          | RNase E (activity change) | Marchand et al., 2001 |
| gp0.7   |          | Translation components | Robertson et al., 1994; Robertson and Nicholson, 1990, 1992 |
| gp0.7   |          | gp4.5         | Bartel et al., 1996 |
| gp1     | T7 RNA polymerase | gp3.5 | Jeruzalmi and Steitz, 1998; Zhang and Studier, 2004 |
| gp1     |          | gp19 (genetic) | Zhang and Studier, 1995, 2004 |
| gp1.2   | E.coli dGTPase inhibitor; F-exclusion | Host Dgt (dGTPase) | Huber et al., 1988; Nakai and Richardson, 1990; Saito and Richardson, 1981 |
| gp1.2   |          | F plasmid PifA | Cheng et al., 2004 |
| gp1.2   |          | Host FxsA | Schmitt and Molineux, 1991; Wang et al., 1999; Cheng et al., 2004; Wang et al., 1999 |
| Class II |          |                |            |
| gp1.7   | Nucleotide kinase | gp1.7 (oligomer) | Tran et al., 2010; Bartel et al., 1996 |
| gp2     | E.coli RNAP inhibitor | Host RNAP | Camara et al., 2010; Nechaev and Severinov, 1999 |
| gp2     |          | gp3           | Bartel et al., 1996 |

TABLE II (continued)

| Protein | Function | Interacts with | References |
|---------|----------|----------------|------------|
| gp2.5   | SSB      | gp5 (with thioredoxin) | Hamdan et al., 2005; He et al., 2003 |
| gp2.5   |          | gp4            | Ghosh et al., 2008, 2010; Marintcheva et al., 2009 |
|         |          |                | Ghosh et al., 2008; Hamdan et al., 2005; He and Richardson, 2004 |
| gp2.5   |          | gp2.5 (dimer)  | Rezende et al., 2002; Bartel et al., 1996 |
| gp3     | Endonuclease I, HJ resolvase | gp3 (dimer) | Hadden et al., 2007; Parkinson and Lilley, 1997 |
| gp3.5   | Amidase (lysozyme) | gp1 | Jeruzalmi and Steitz, 1998; Zhang and Studier, 2004 |
| gp4     | gp4A primase-helicase; gp4B helicase | gp5 (with thioredoxin) | Hamdan et al., 2005; He and Richardson, 2004 |
|         |          | gp2.5          | Hamdan et al., 2005; He and Richardson, 2004 |
|         |          | gp4 (hexamer)  | Singleton et al., 2000; Bartel et al., 1996 |
| gp4.5   |          | gp0.7          | Bartel et al., 1996 |
| gp5     | DNA polymerase | gp4 (with thioredoxin) | Ghosh et al., 2008; Hamdan et al., 2005; Bartel et al., 1996 |
| gp5     |          | gp2.5 (with thioredoxin) | Ghosh et al., 2008, 2010; He et al., 2003; Marintcheva et al., 2009; Hamdan et al., 2005 |
| gp5     |          | gp4.7          | Bartel et al., 1996 |
| gp5     |          | gp6.5          | Bartel et al., 1996 |
| gp5     |          | Host thioredoxin | Ghosh et al., 2008 |
| gp5.3   |          | gp5.3          | Bartel et al., 1996 |
| gp5.5   |          | Host H-NS      | Lin, 1992; Liu and Richardson, 1993 |
| gp5.9   | Inhibits RecBCD nuclease | Host RecBCD | Lin, 1992 |

Class III

| gp6.7 | Head completion; ejected virion protein | Other head proteins likely. gp13? | Kemp et al., 2005 |
| --- | --- | --- | --- |
| gp7.3 | Tail chaperone; ejected structural virion protein | gp7.3, gp11, gp12 | Kemp et al., 2005 |
| gp7.3 |  | Host LPS | Kemp et al., 2005 |
| gp8 | Head-tail connector (portal) protein | gp19 | Fujisawa et al., 1991; Morita et al., 1995 |
| gp8 |  | gp9/10/gp6.7/7.3/11/12/ 14 (structure) | Agirrezabala et al., 2005a, 2007; Cerritelli and Studier, 1996a; Liu et al., 2010 |
| gp8 |  | gp8 (dodecamer) (structure) | Agirrezabala et al., 2005b; Cerritelli and Studier, 1996b |
| gp9 | Scaffolding protein | gp8, gp10 (structure) | Agirrezabala et al., 2007; Cerritelli and Studier, 1996a,b Bartel et al., 1996 |
| gp9 |  | gp14/15/16 (structure) | — |
| gp10 | Major capsid protein (-1 frame-shift -> minor gp10B) | gp8, gp9, gp14 (structure) | Agirrezabala et al., 2005a,b, 2007; Cerritelli and Studier, 1996a; Cerritelli and Studier, 1996b |
| gp10 |  | gp10 (pentamer/hexamer) (structure) | Cerritelli and Studier, 1996a; Bartel et al., 1996 |
| gp10 |  | Host FxsA; F plasmid PifA | Cheng et al., 2004; Schmitt and Molineux, 1991; Wang et al., 1999 |
| gp11 | Tail protein | gp7.3, gp8, gp11, gp12 (structure) | Kemp et al., 2005; Agirrezabala et al., 2007; Liu et al., 2010 |
| gp11 |  | Host LPS | Kemp et al., 2005 |
| gp11 |  | gp2.5 | Bartel et al., 1996 |
| gp12 | Tail protein | gp7.3, gp8, gp11, gp12 (structure) | Kemp et al., 2005; Liu et al., 2010 |
| gp12 |  | Host LPS | Kemp et al., 2005; Qimron et al., 2006 |

(continued)

TABLE II (continued)

| Protein | Function               | Interacts with                                      | References                                                                 |
|---------|------------------------|----------------------------------------------------|---------------------------------------------------------------------------|
| gp13    | Virion chaperone       | gp6.7, gp8, gp12, gp14,                            | Kemp et al., 2005; Stone and Miller, 1985                                   |
|         |                        | gp17, gp19 (genetics)                              |                                                                           |
| gp14    | Ejected core protein   | gp8, gp14, gp15, gp16                              | Agirrezabala et al., 2007; Liu et al., 2010                                |
|         |                        | (structure)                                        |                                                                           |
| gp15    | Ejected core protein   | gp14, gp15, gp16 (structure)                       | Agirrezabala et al., 2007; Chang et al., 2010; Liu et al., 2010; Bartel et al., 1996 |
| gp16    | Ejected core protein   | gp14, gp15, gp16 (structure)                       | Agirrezabala et al., 2007; Chang et al., 2010; Liu et al., 2010            |
| gp16    | Peptidoglycan          |                                                    | Moak and Molineux, 2000, 2004                                               |
| gp17    | Tail fiber protein     | gp7.3/11/12 (structure)                            | Agirrezabala et al., 2007; Chang et al., 2010; Liu et al., 2010            |
| gp17    |                        | gp17 (trimer)                                      | Steven et al., 1988; Bartel et al., 1996                                    |
| gp17    |                        | Host LPS                                           | Steven et al., 1988; Qimron et al., 2006                                    |
| gp18    | Small terminase        | gp18 (octamer)                                     | White and Richardson, 1987; Hamada et al., 1986                             |
| gp18.5- | gp18.7                | Rz-Rz1 homologues; spanin                           | gp18.5–gp18.7                                                             |
|         | Casjensen et al., 1989 Summer |                                                |                                                                           |
|         | et al., 2007; Bartel et al., 1996 |                                          |                                                                           |
| gp19    | Large terminase        | gp8                                                | Fujisawa et al., 1991; Hamada et al., 1986; Morita et al., 1994, 1995      |
| gp19    |                        | gp19 (hexamer in the                               | Fujisawa et al., 1991                                                      |
|         |                        | presence of prohead)                               |                                                                           |
| gp19    |                        | gp4.7                                              | Bartel et al., 1996                                                        |

Bacteriophage Protein–Protein Interactions 237

A

| Head | Tail | Tail tip | Tail fibers | b2 region |
| --- | --- | --- | --- | --- |
| cos | terminase W | Nu3 | J | central tail fiber | lom | Ea47 |
| Nu1 | A | B | C | D | E | F | G | H | I | K | L | M | N | O | P | Q | R | S | T | U | V | W | X | Y | Z |
| 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | 24 | 25 |

b2 region | Site-spec. recombin | general recombin | PPEL region | DNA replication | ninregion | PPL regionlysis | ERE region
Ea47 | att | xis | int | Ea8.5 | Ea2.9 | exo | bet | kil | clI | ral | sieB | rexB | rexA | cro | cl | P_RE | P_RM | P_L | P_R | ren | ori | O | P | B | G | Q | S | R | Rz1* | bo | cos
sib | P_I | gan | E4 | N | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | * | *
TABLE III Published protein-protein interactions of phage λ

| λ1   | λ2    | Notes                                                                 | References                                                                                          |
|------|-------|-----------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
| gpA  | gpNu1 | gpA (N-terminal) – gpNu1 (C-terminal)                                 | Catalano, 2000; de Beer *et al.*, 2002; Hang *et al.*, 1999                                           |
| gpA  | gpB   | gpA (C-terminal) – gpB (= portal)                                     | Catalano, 2000; Duffy and Feiss, 2002                                                              |
| gpW  | gpB   | gpW (NMR, head-tail joining)                                          | Casjens, 1974; Casjens *et al.*, 1972                                                               |
| gpW  | gpFII | gpW required for gpFII binding                                        | Casjens, 1974; Maxwell *et al.*, 2001                                                               |
| gpB  | gpB   | 12-mer (22 amino acids removed from gpB N-terminal)                   | Kochan, Carrascosa and Murialdo, 1984; Tsui and Hendrix, 1980                                       |
| gpC  | gpE   | Covalent PPI (in virion?)                                             | Hendrix and Casjens, 1974; Hendrix and Duda, 1998                                                   |
| gpC' | gpB   |                                                                       | Murialdo, 1979                                                                                     |
| gpC  | gpNu3 | gpC may degrade gpNu3 (before DNA packaging)                          | Hendrix and Casjens, 1975; Hohn *et al.*, 1975; Medina *et al.*, 2010; Murialdo and Becker, 1978     |
| gpD  | gpD   | gpD forms trimers                                                     | Mikawa *et al.*, 1996; Sternberg and Hoess, 1995; Yang *et al.*, 2000                               |
| gpE  | gpE   | Major capsid protein                                                  | Casjens and Hendrix, 1974; Dokland and Murialdo, 1993; Lander *et al.*, 2008                       |
| gpU  | gpU   | "Probably a hexamer"                                                 | Katsura and Tsugita, 1977                                                                           |
| gpD  | gpE   |                                                                       | Casjens and Hendrix, 1974; Dokland and Murialdo, 1993; Lander *et al.*, 2008                       |
| gpU  | gpV   | gpU is head-proximal tail protein                                     | Pell *et al.*, 2009                                                                                |
| gpV  | gpV   |                                                                       | Buchwald *et al.*, 1970a,b; Casjens and Hendrix, 1974; Katsura, 1983                                |
| gpO  | gpP   |                                                                       | Wickner and Zahn, 1986; Zahn and Blattner, 1985, 1987; Zahn and Landy, 1996)                        |

gpO   gpO   gpO-gpO interactions when bound to ori DNA           Alfano and McMacken, 1989
gpB   gpA   Genetic evidence                                   Frackman et al., 1985; Sippy and Feiss, 1992
gpFI  gpA   Genetic evidence                                   Catalano and Tomka, 1995
gpFI  gpE   Genetic evidence                                   Murialdo and Tzamtzis, 1997
gpH   gpG/  gpG/gpGT holds gpH in an extended fashion          Hendrix and Casjens, 2006
      gpGT
gpV   gpGT  T domain binds soluble gpV                         Hendrix and Casjens, 2006
gpH   gpV   gpV probably assembles around gpH, displacing      Xu et al., 2004
       gpG/gpGT
CI    CI    Forms octamer that links OR to OL                  Bell and Lewis, 2001; Dodd et al., 2001
Exo   Bet                                                          Radding et al., 1971
SieB  Esc   Esc is encoded in frame in SieB and inhibits SieB  Ranade and Poteete, 1993a,b
CII   CII   Homotetramers                                       Ho and Rosenberg, 1982
Xis   Int                                                          Sam et al., 2002
Xis   Xis   Xis-Xis binding mediates cooperative DNA binding   Sam et al., 2004
Int   Int   Dimer                                                Warren et al., 2003
Stf   Stf   Likely homotrimer by homology with T4 fibers       Savva et al., 2008
gpS   gpS   Large ring in inner membrane                       Grundling et al., 2000
gpS   gpS'  gpS' inhibits gpS ring formation                   Hendrix and Casjens, 1975
gpB   gpE   Copurify in procapsid                              C. Catalano, personal communication
gpNu3 pNu3 Monomer-dimer equilibrium                           Halder et al., 2007
CIII  CIII  Dimer                                               Anderson et al., 1979
Cro   Cro   Dimer; X-ray str                                    Summer et al., 2007
gpRz  gpRz1 Heteromultimer supposed to span the periplasm      Ray and Murialdo, 1975
gpNu3 gpB  gpNu3 required for gpB incorporation into procapsid

FIGURE 4 Assembly of the phage λ head. Head assembly has been subdivided into five steps, although most steps are not very well understood in mechanistic terms. Note that the tail is assembled independently. GpC protease, scaffolding protein gpNu3, and portal protein gpB form an ill-defined initiator structure. Protein gpE joins this complex in a step requiring the chaperonins GroES and GroEL. Proteins gpNul, gpA, and gpFI are required for DNA packaging. GpD joins and stabilizes the capsid as a structural protein; gpFII and gpW are connecting the head to the tail that joins once the head is completed. Modified after Georgopoulos et al. (1983) and Lander et al. (2008).

subsequently get trimmed to make slightly shortened products named X1 and X2 (Hendrix and Casjens, 1974). The authors note, however, that attempts to identify X1 and X2 were unsuccessful and thus X1 and X2 may be artifacts (Medina et al., 2010). GpNu3 acts as putative scaffolding protein that aids coat shell assembly and is removed from the capsid by proteolysis before DNA is packaged. Structures of the procapsid and mature λ capsid have been solved by cryo-EM, showing the T=7 arrangement of capsid protein gpE clustered into hexamers and pentamers (Dokland and Murialdo; Lander et al., 2008). Trimers of the decoration protein gpD bind to the capsid at trivalent interaction points between the capsomers and stabilize the capsid (Sternberg and Weisberg, 1977). Comparison of the procapsid with the mature capsid reveals the considerable structural transitions that occur upon capsid maturation and DNA packaging, but this process is not well understood mechanistically.

3. Assembly of virion tail (Fig. 5)

Tail assembly begins with the antireceptor protein gpJ. Proteins gpI, gpK, gpL, and gpM assemble around gpJ in that order, although no mechanistic or structural details are known about this process. In parallel events, gpH, gpG, and gpG-T (the frameshifted form of gpG including the

Bacteriophage Protein–Protein Interactions 241

![Diagram](attachment:diagram.png)

FIGURE 5 Assembly of the phage λ tail. The λ tail is made of at least six proteins (gpU, V, J, H, tfa, stf) with another seven required for assembly (gpl, M, L, K, G/T, Z). Assembly starts with protein gpJ, which then, in a poorly characterized fashion, requires proteins gpl, gpL, gpK, and gpG/T to add the tape measure protein H. GpM joins and then gpG and gpG/T leave the complex so that the main tail protein gpV can assemble on the gpJ/H scaffold. Finally, gpU is added to the head-proximal end of the tail. GpZ is required to connect the tail to the preassembled head. Protein gpH is cleaved by the action of gpU and gpZ (Tsui and Hendrix, 1983). It remains unclear if proteins gpM and gpL are part of the final particle (Hendrix and Casjens, 2006). From http://www.pitt.edu/~duda/lambdatail.html.

$T$ reading frame) form a complex that binds the tail tip and the major tail protein (gpV). The tape measure protein gpH determines the length of the phage tail (Katsura, 1990). Once the correct length is achieved, gpV assembly stops and the proximal tail protein gpU is added. Protein gpZ somehow facilitates head–tail joining without being assembled into the final virion.

## 4. Lysogeny and induction

After phage DNA ejection, circularization, and ligation by the host DNA ligase, the earliest genes to be expressed include $N$, $O$, and $P$, as well as $cII$ and $cIII$, which play a key role in the lysis/lysogeny decision (Echols, 1986). Both CII and CIII can interact with and be cleaved by the bacterial FtsH (HflB) protease directly, which in turn is complexed with the host HflK and HflC proteins at the cytoplasmic membrane. HflD, a cytoplasmic membrane protein, also binds CII, making it both more susceptible to FtsH and less able to bind DNA (Kihara et al., 2001; Parua et al., 2010). As a substrate for FtsH, CIII acts as a competitive inhibitor of CII degradation and, as the half-life of CII protein increases, leads to an increase in CI repressor synthesis (Datta et al., 2005a; Herman et al., 1997; Kobiler et al., 2007). Except when $cIII$ is overexpressed, lysogeny is only established when the activity of these proteases (FtsH, Lon, and perhaps others)

is low. Once established, lysogeny is a stable state until lysogens are exposed to ultraviolet light or other DNA-damaging agents that lead to the activation of RecA, which in turn accelerates autocleavage of the $\lambda$ repressor CI. Loss of CI leads to prophage induction and the lytic cycle. The mechanism by which phage $\lambda$ decides whether to grow lytically or become a lysogen is reviewed elsewhere (Court *et al.*, 2007; Friedman and Court, 2001; Gottesman and Weisberg, 2004; Ptashne, 2006). However, Table IV contains the host–phage protein–protein interactions critical for gene expression and thus the genetic switch of phage $\lambda$.

### 5. Anti-termination

*Escherichia coli* RNA polymerase transcribes $\lambda$ DNA starting from the immediate early promoters $P_{L}$ and $P_{R}$, which leads to expression of the $N$ and $cro$ genes, among others. As soon as the gpN-binding sites (*nut* sites) are transcribed, gpN modifies the transcription complex by recognizing a stem-loop structure in the mRNA and by binding to host Nus proteins [note that gpN interacts with *nut* RNA (not DNA)]. This modified complex is capable of overriding the termination signal that normally stops immediate early transcription. Because gpN is continually being degraded by the Lon protease, ongoing anti-termination requires continual transcription of the $N$ gene (Fig. 6). In addition to the anti-terminator for early lytic genes (gpN), a second anti-terminator, gpQ, is required for late gene expression. GpQ anti-termination differs from that of gpN in that gpQ binds to the “Q binding element” (QBE, also known as the *qut* site) of the $P_{R^{'}}$ promoter (i.e., unlike the gpN protein, gpQ binds DNA). When RNA polymerase (RNAP) initiates transcription at the $P_{R^{'}}$ promoter, it almost immediately pauses at a pause-inducing sequence. Pausing allows gpQ to contact $\sigma^{70}$ in the RNAP, which causes a conformational change in the enzyme that enables gpQ to bind the $\beta$ subunit (RpoB). The gpQ-containing RNAP is then able to override the termination signal at $t_{R^{'}}$ (Deighan *et al.*, 2008; Nickels *et al.*, 2002).

### 6. Replication

After $\lambda$ gene $O$ and $P$ protein synthesis, replication of the phage genome is initiated (Fig. 6). Replication initiation starts with dimeric gpO bound to the four 18-bp direct repeats of the $\lambda$ origin, located within the sequence of the $O$ gene. Each repeat binds a gpO dimer, leading to a total of eight monomers bound directly to the DNA, in addition to non-DNA-bound gpO that associates with the complex solely by PPIs (Dodson *et al.*, 1986). This large nucleoprotein complex is called the O-some. GpP extracts the host DnaB helicase from its native complex with DnaC (Zylicz *et al.*, 1989). The DnaB helicase is inactive as long as it is tightly associated with gpP. The gpP–DnaB pair is then added to the O-some through interactions between gpP and gpO. Three host heat shock chaperone proteins—DnaK,

TABLE IV Published phage $\lambda$-host interactions${}^{a}$

| $\lambda$ | Host     | Description                                                                 | References                                                                 |
|-----------|----------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------|
| **Transcription** |       |                                                                           |                                                                           |
| CI        | RecA     | RecA degrades CI${}^{b}$                                                   | Kobiler *et al.*, 2004                                                    |
| CI        | RpoA     |                                                                           | Kędzierska *et al.*, 2007                                                 |
| CI        | RpoD     |                                                                           | Li *et al.*, 1994; Nickels *et al.*, 2002                                  |
| CII       | ClpYQ    | ClpYQ degrades CII $in vitro$                                              | Kobiler *et al.*, 2004                                                    |
| CII       | ClpAP    | ClpAP degrades CII $in vitro$                                              | Kobiler *et al.*, 2004                                                    |
| CII       | HflD     | HflD makes CII more vulnerable to FtsH ($hfl$, high-frequency lysogenization) | Kihara *et al.*, 2001                                                     |
| CII       | HflB${}^{c}$ | HflB (= FtsH) protease degrades cII                                     | Kobiler *et al.*, 2002; Shotland *et al.*, 2000                            |
| CII       | RpoA     |                                                                           | Kędzierska *et al.*, 2004; Marr *et al.*, 2004                             |
| CII       | RpoD     |                                                                           | Kędzierska *et al.*, 2004                                                 |
| CIII      | HflB     | CIII inhibits HflB; HflB degrades CIII                                      | Herman *et al.*, 1997; Kornitzer *et al.*, 1991                            |
| gpN       | NusA     | Transcriptional regulation                                                  | Friedman and Court, 2001                                                  |
| gpN       | Lon      | Lon degrades gpN                                                           | Kobiler *et al.*, 2004; Gottesman *et al.*, 1981                           |
| gpQ       | $\sigma^{70}$ | gpQ makes RNAP insensitive to $cis$ terminal                          | Marr *et al.*, 2001; Nickels *et al.*, 2002; Roberts and Roberts, 1996; Roberts *et al.*, 1998 |
| **Head**  |          |                                                                           |                                                                           |
| gpB       | GroE     | genetic interaction; $E$. coli GroE                                        | Ang *et al.*, 2000; Georgopoulos *et al.*, 1973; Murialdo, 1979           |
| gpE       | GroE     | genetic interaction                                                        | Georgopoulos *et al.*, 1973                                               |
| **Tail**  |          |                                                                           |                                                                           |
| gpJ       | LamB     | LamB is the $E$. coli receptor                                             | Buchwald and Siminovitch, 1969; Clement *et al.*, 1983; Mount *et al.*, 1968; Wang *et al.*, 2000a; Werts *et al.*, 1994 |
| Stf       | OmpC     | OmpC is a secondary $E$. coli receptor                                      | Hendrix and Duda, 1992                                                   |

(continued)

TABLE IV (continued)

| $\lambda$ | Host    | Description                                                                 | References                                                                 |
|-----------|---------|-----------------------------------------------------------------------------|--------------------------------------------------------------------------|
| Recombination |        |                                                                             |                                                                            |
| Xis       | Lon     | Xis is degraded by E. coli Lon protease                                    | Leffers and Gottesman, 1998                                              |
| Xis       | FtsH    | Xis is degraded by E. coli FtsH protease                                   | Leffers and Gottesman, 1998                                              |
| Xis       | Fis     | both required for excision                                                 | Ball and Johnson, 1991; Cho *et al.*, 2002; Esposito and Gerard, 2003      |
| Int       | IHF     | Both catalyze recombination at attP/attB                                   | Campbell *et al.*, 2002; Crisona *et al.*, 1999                           |
| Gam       | RecB    | Gam inhibits RecBCD                                                        | Marsic *et al.*, 1993                                                    |
| NinB      | SSB     | NinB also binds ssDNA                                                     | Maxwell *et al.*, 2005                                                   |
| Replication |        |                                                                             |                                                                            |
| gpO       | ClpXP   | ClpXP degrades gpO                                                         | Kobiler *et al.*, 2004                                                   |
| gpP       | DnaA    |                                                                             | Datta *et al.*, 2005b,c                                                  |
| gpP       | DnaB    |                                                                             | Mallory *et al.*, 1990                                                   |

$a$ CbpA (*Wegrzyn et al.*, 1996) perhaps interacts with DnaA and SeqA (*Potrykus et al.*, 2002; *Wegrzyn and Wegrzyn*, 2001, 2002).

$b$ After DNA damage, RecA bound to single-stranded DNA stimulates autocleavage of the CI repressor (*Kobiler et al.*, 2004).

$c$ HflB = FtsH.

Bacteriophage Protein–Protein Interactions 245

A

B
Infection and
DNA Injection
phage lambda
gpJ
outer membrane
LamB
inner membrane
PtsM
lambda DNA

C DNA ligation
host DNA ligase
COS site

D Gene Regulation: N anti-termination

E DNA Replication

FIGURE 6 Interactions of phage λ with its host, E. coli. (A) Overview of λ activities in E. coli: (a) free phage, (b) binding and entry, (c) DNA circularization and supercoiling, (d) transcription, (e), replication, and (f) virion assembly. Details are shown in B–E. (B) Infection and DNA ejection, (C) DNA ligation, (D) gene regulation. An asterisk means weakly bound to the RNA polymerase. (E) DNA replication. Phage proteins are labeled in red, host proteins in black. Modified after Das (1992) and Mason and Greenblatt (1991).

DnaJ, and GrpE—are subsequently recruited (DnaK through direct interaction with gpP and gpO) and activate the DnaB helicase by liberating it from its inhibitory interaction with gpP. Finally gpP and the chaperones are released, and replication starts with the help of the host DnaG primase, DNA polymerase III, gyrase, and single-stranded DNA-binding (SSB) proteins (Dodson et al., 1986; LeBowitz and McMacken, 1986). The RNA polymerase β subunit was shown to interact directly with gpO (Szambowska et al., 2010). Even though not yet fully understood, how E. coli RNA polymerase activates λ ori seems to be far more complex than the original idea that it simply changed DNA topology.

Studies on λ ori-dependent plasmids have revealed interactions with even more host proteins, including the DnaJ homolog CbpA (Wegrzyn et al., 1996), and through transcriptional activation of origin activity, perhaps also with DnaA and SeqA (Potrykus et al., 2002; Wegrzyn and

Wegrzyn, 2001, 2002). As λ ori-dependent plasmid maintenance and copy number are very sensitive to changes in host physiology, interactions of the O-some with additional proteins may yet be revealed.

7. The λ interactome

A λ Y2H interactome analysis has been completed (Rajagopala et al., 2011). All λ open reading frames (ORFs) were cloned and tested in all possible pairs for interactions, using three different Y2H vectors. These screens identified 97 interactions among 68 tested λ proteins, of which 16 were known previously. Notably, because only 33 interactions had been reported previously, this screen found more than 50% of all previously published interactions in a single experiment! At least 18 of the newly observed Y2H interactions seem to be plausible, based on the functions of the interacting proteins, indicating that even in λ many new discoveries can be made. Interestingly, relatively few interactions involving morphogenetic proteins were revealed, and possible reasons for the still significant fraction of apparent false negatives are discussed later.

C. Salmonella phage P22

P22 infects *Salmonella enterica* serovar Typhimurium and was originally isolated by Zinder and Lederberg (1952). It is a temperate member of Podoviridae. Its virion proteins are not particularly closely related to other tailed phage groups, but its early genes and overall lifestyle are very closely related to phage λ. For this reason, it is often placed in the “lambdoid” phage group. Phage P22 is also a very important tool for those who use *Salmonella* genetics, as it is particularly easy to handle in the laboratory and is a very good generalized transducing phage (i.e., it packages a significant amount of host DNA by accident and can be used to move host alleles from one *Salmonella* strain to another).

1. P22 life cycle

The early operons of P22 are organizationally similar to those of λ and carry parallel prophage repressor, Cro, recombination, DNA replication, and transcription anti-termination functions (Susskind and Botstein, 1978). The P22 repressor has a different DNA operator target specificity from that of λ repressor and encodes nonhomologous types of recombination and replication proteins, but its late operon anti-termination protein is 97% identical to that of λ and has the same target specificity. Unlike λ, in addition to its prophage repressor (C2) and lysogeny control proteins (C1, C3, Cro), P22 carries an “immunity I” region that encodes an anti-repressor protein, Ant, that binds to the prophage repressor and blocks its ability to bind its operator. Two additional repressors, Mnt and Arc, regulate Ant synthesis (Susskind and Botstein, 1975; Vershon et al., 1987a,b).

Also, instead of a protein that recruits the host DnaB protein to the replication origin, P22 encodes its own DnaB homologue (Backhaus and Petri, 1984; Wickner, 1984). The repressed P22 prophage encodes several “lysogenic conversion” genes, several of which encode a set of three Gtr (glucose transfer) proteins that modify the host O-antigen polysaccharide by glucosylation (Broadbent et al., 2010; Makela, 1973; Vander Byl and Kropinski, 2000). Finally, P22 has been studied as a prototypical headful DNA packaging phage, which is different from the packaging strategies used by the other phages discussed here; unlike these other phages that package DNA between two specific nucleotide sequences in the DNA substrate, P22 packages DNA until the head is full and the nucleotide sequence at the termination point does not matter (Casjens and Hayden, 1988; Jackson et al., 1978; Tye et al., 1974).

2. P22 virion assembly

The P22 virion is assembled by proteins encoded by 14 genes in its late operon (Fig. 7) (Botstein et al., 1973; Eppler et al., 1991; King et al., 1973; Poteete and King, 1977; Youderian and Susskind, 1980). Some P22-like phages, for example, phages ε34 and L, have a 15th virion assembly gene, *dec*, at the promoter proximal end of the cluster, whose encoded protein “decorates” the outside of the head and stabilizes it (Gilcrease et al., 2005; Tang et al., 2006; Villafane et al., 2008). The P22 late operon encodes lysis proteins very similar to those of λ; however, the virion assembly genes are not closely related to λ or to any other tailed phage type. P22 virion assembly is best known for the fact that its head-assembling scaffolding protein was the first to be discovered and studied (Cortines et al., 2011; King and Casjens, 1974; Prevelige et al., 1988; Weigele et al., 2005). The P22 procapsid assembles as a complex of 415 coat protein molecules, 12 portal ring subunits, and about 250 scaffolding protein molecules. At about the time when DNA is packaged, all 250 scaffolding proteins are released intact from the procapsid and are reused with newly synthesized coat protein in subsequent rounds of procapsid assembly. Thus, the P22 scaffolding protein acts catalytically in head assembly and individual molecules can participate in at least five rounds of procapsid assembly. A particularly high-resolution asymmetric cryo-EM reconstruction of the virion has been determined (Lander et al., 2006; Tang et al., 2011), and because assembly of the four proteins of its short tail has also been quite well studied [X-ray structures are known for three tail proteins and the portal protein (Olia et al., 2006, 2007; Steinbacher et al., 1997)], the P22 virion is perhaps the best understood of all tailed phage virions. The details of P22 virion assembly have been reviewed in light of its evolution and genome mosaicism (Casjens and Thuman-Commike, 2011), and the authors refer the reader to this review for further details. The published PPIs of phage P22 are summarized in Table V. Interactions between P22

248 Roman Häuser et al.

A

| Head | Tail | Ejection | Tailspike | O-antigen |
| --- | --- | --- | --- | --- |
| terminate |  |  |  | modification |
| pac |  |  |  |  |
| 3 | 2 | 1 | 8 | 5 | 4 | 10 | 26 | 7' | 20 | 16 | *sieA* | *mnt* | *arc* | *ant* | 9 | *gtrC* | B | A |
|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | P<sub>mnt</sub> | P<sub>sieB</sub> | 16 | 17 | 18 | 19 | 20 | 21 | P<sup>gtr</sup> |

Late operon

Site-specific recombination

Homologous recombination

DNA replication nin region lysis

att xis eac

int

eaA eaD

abc2 abc1 kil c3 ral *24* *cro*

sieB c2 c1 ori

Rz1

13 15 rha

21 22 23 24 25 26 27 28 29 P<sub>sieB</sub> 30 P<sub>L</sub> P<sub>RM</sub> P<sub>RE</sub> 33 34 35 36 P<sub>aQ</sub> P<sub>R</sub> 37 38 39 40 41 kbp

Early left operon Early right operon Late operon

* expressed in lysogen  
♦ regulatory  
' proteolytically modified  

B

5 (coat, 405)

DNA (1)

1 (portal, 12)

9 (tailspike, 18)

10 (6)

26 (needle, 3)

C

~60 nm

FIGURE 7 Genome and virion of phage P22. (A) Genome of phage P22 with a scale in kbp below. Green open reading frames (ORFs; rectangles) are transcribed left to right; red ORFs are transcribed right to left. Known RNA polymerase promoters and transcripts are shown as black flags and arrows, respectively. (B) The asymmetric (not icosahedrally averaged) P22 virion three-dimensional cryo-EM reconstruction from Tang et al. (2011). Numbers in parentheses indicate molecules/virion, and bold numbers indicate proteins for which X-ray structures are known. (C). Negatively stained electron micrograph of P22 virions.

and its host are much less well studied than those of λ and are mostly understood by homology arguments in the case of proteins that are similar in the two phages. Known P22-host interactions are listed in Table VI.

3. Comparison of P22 and λ interactomes

As noted earlier, comparison of P22 and λ is a particularly informative way to understand the kinds of relationships among the proteins of different tailed phages. The "lambdoid" phages all encode similar functions and have similar gene organization, but these "similar functions" can be of several different types. Because these phages undergo a

TABLE V P22 intraphage protein interactions

| P221 | P222 | Description                                                                                   | References                                                                                     |
|------|------|-----------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
| gp3  | gp3  | Small terminase subunit; homononomer in solution                                                | Nemecek et al., 2007, 2008                                                                      |
| gp3  | gp2  | Purify as complex; bind each other in vitro                                                    | Nemecek et al., 2007; Poteete and Botstein, 1979                                               |
| gp2  | gp1  | gp2 is large terminase subunit; assembles in packaging motor to                                 | Casjens, 2011                                                                                  |
|      |      | gp1 by analogy with other phages (gp2 by itself is monomer in                                   |                                                                                                |
|      |      | solution); evolutionary argument for P22gp2-gp1 interaction                                     |                                                                                                |
| gp1  | gp1  | Portal protein; homododecamer; EM of 12-mer; EM of virion; X-ray structure                       | Bazinet et al., 1988; Lander et al., 2006; Olia et al., 2011                                    |
| gp1  | gp5  | Purify as complex (procapsids); three-dimensional (3D) reconstructions of virion and procapsid   | Botstein et al., 1973; Chen et al., 2011; Lander et al., 2006                                   |
| gp1  | gp8  | gp8 required for assembly of gp1 into procapsid; interaction seen in procapsid 3D reconstruction | Chen et al., 1989; Greene and King, 1996; Weigele et al., 2005                                |
| gp8  | gp8  | Scaffolding protein; forms homodimers and tetramers in solution                                  | Parker et al., 1997                                                                            |
| gp8  | gp5  | Purify as complex (procapsid); bind in vitro                                                   | Fuller and King, 1982; Greene and King, 1994; Parker et al., 1998                              |
| gp5  | gp5  | Coat (major capsid) protein; forms T=7l icosahedral shell of virion                             | Botstein et al., 1973; Casjens, 1979; Lander et al., 2006                                      |
| gp5  | gp1  | Purify as complex (procapsids); interaction seen in 3D reconstruction                            | Chen et al., 2011; Lander et al., 2006                                                         |
| gp4  | gp4  | Tail protein; monomer in solution (naturally unfolded?); gp4s barely touching each other in dodecamer ring in virion (i.e., contacts not large); EM, X-ray structure | Olia et al., 2011; Tang et al., 2011                                                           |
| gp4  | gp5  | Contact in high-resolution 3D reconstruction of virion                                         | Olia et al., 2006, 2011; Tang et al., 2011                                                     |
| gp4  | gp1  | Order of assembly, EM, 3D reconstruction; X-ray cocrystal structure                             | Olia et al., 2006, 2011; Tang et al., 2011                                                     |

(continued)

TABLE V (continued)

| P221 | P222 | Description                                                                                   | References                                                                 |
|------|------|----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
| gp4  | gp10 | 12gp4s contact 6gp10s; order of assembly, EM, 3D reconstruction                                | Lander et al., 2006; Olia et al., 2007                                      |
| gp4  | gp9  | gp4 ring contacts 6gp9 trimers; 3D reconstruction                                             | Lander et al., 2006                                                        |
| gp10 | gp10 | Tail protein; probably hexamer in solution; 3D reconstruction                                 | Lander et al., 2006; Olia et al., 2007                                      |
| gp10 | gp9  | 6gp4s contact 6gp9 trimers; 3D reconstruction                                                 | Lander et al., 2006                                                        |
| gp10 | gp26 | 6gp10s contact one gp26 trimer                                                                | Lander et al., 2006                                                        |
| gp26 | gp26 | Tail needle protein; trimer in solution - crystal structure; one trimer in virion             | Olia et al., 2007                                                          |
| gp16 | gp8  | Mutants of gp8 do not incorporate gp16 into virion                                           | Chen et al., 1989; Greene and King, 1996; Weigle et al., 2005               |
| C1   | C1   | C1 is homologue of λCII; homotetramer                                                        | Ho et al., 1992                                                            |
| Mnt  | Mnt  | Repressor of antirepressor gene; tetramer                                                    | Nooren et al., 1999                                                        |
| Arc  | Arc  | Repressor of antirepressor gene; dimer in solution, tetramer on DNA                           | Bowie and Sauer, 1989; Breg et al., 1990; Brown, Bowie and Sauer, 1990     |
| Ant  | C2   | Antirepressor; binds to and inhibits C2 repressor                                            | Susskind and Botstein, 1975                                                 |
| gp9  | gp9  | Tailspike protein; trimer in solution, 6 trimers in virion; crystal structure; binds to polysaccharide on cell surface | Lander et al., 2006; Steinbacher et al., 1994, 1997                        |
| Int  | Xis  | Form heteromultimer                                                                           | Cho et al., 2002                                                           |
| Xis  | Xis  | Excisionase; homomultimers                                                                    | Mattis et al., 2008                                                        |
| Erf  | Erf  | Recombination protein; homomultimeric ring of 10–14 subunits                                  | Poteete et al., 1983                                                       |
| C3   | C3   | C3 is homologue of λCIII; λCIII is dimer by disulfide bond; P22 interaction by homology with λCIII | Halder et al., 2007                                                        |
| SieB | Esc  | Similar to λ but not studied in detail                                                       | Ranade and Poteete, 1993a                                                  |
| C2   | C2   | Prophage repressor homologue of λCI; homodimer                                               | De Anda et al., 1983                                                       |
| Cro  | Cro  | Dimer; solution studies and crystal structure                                                 | Darling et al., 2000; Newlove et al., 2004                                  |

gp18 gp12 Origin binding protein gp18 recruits DnaB-like gp12 to replication origin by analogy with λ (note that gp12 is not a homologue of λP protein; gp18 is homologous to λO protein only in N-terminal domain) Backhaus and Petri, 1984

Rz Rz1 Rz and Rz1 are supposed to bind to one another and form a bridge that spans the periplasm in phage λ; P22 gp15 and Rz1 should do the same by homology. Summer et al., 2007

gp13 gp13 Holin forms rings of large diameter in λ; by homology same should be true of P22 holin Savva et al., 2008

gp13 gp13' Antiholin-holin interactions delay the formation of holin rings in λ; by homology same should be true in P22 Rennell and Poteete, 1985

Dec Dec Trimer; Note that P22 has no Dec, but its very close relative phage L does and L Dec binds to P22 capsids. Tang et al., 2006

Dec gp5 Dec occupies a subset of the quasi-equivalent local threefold positions on the shell Tang et al., 2006

TABLE VI P22-host protein interactions

| P22       | Host                | Description                                                                 | References                     |
|-----------|---------------------|-----------------------------------------------------------------------------|--------------------------------|
| gp5       | GroEL               | Solution studies                                                           | de Beus *et al.*, 2000         |
| gp24      | NusA? etc?          | Homologue/analogue of λ gpN; likely binds same host factors as λ gpN           | Franklin, 1985; Hendrix and Casjens, 2006 |
| C3        | FtsH(HflB)          | C3 is homologue of λ CIII; inhibition of FstH by homology with λ CIII          | Semerjian *et al.*, 1989       |
| C1        | FtsH(HflB)          | C1 is homologue of λ CII; CII is degraded by FtsH                             | Backhaus and Petri, 1984       |
| C2        | RecA                | P22 repressor autocleavage stimulated by RecA binding like λ                   | Prell and Harvey, 1983; Tokuno and Gough, 1976 |
| Xis, gp18, gp24, C1 | Host protein degradation machinery | These may be degraded analogously to their λ homologues, but direct information is not available | Backhaus and Petri, 1984       |
| gp23      | RNA polymerase?     | By homology to λ; gp23 is nearly identical to λ gpQ                           | Pedulla *et al.*, 2003         |
| gp12      | Host replication machinery | Possible interaction with host DnaG primase                              | Wickner, 1984                  |
| Abc2      | RecC                | Solution studies                                                           | Murphy, 2000                   |

significant amount of horizontal transfer of genetic information (especially if they are “closely” related), their functions can be homologous or even nearly identical (Casjens, 2005; Hendrix, 2002). For example, the two “gpQ” late operon control anti-termination proteins of the two phages are nearly identical and can replace one another in function. A second kind of relationship is represented by the prophage repressors. These proteins from the two phages have weak but recognizable homology and similar folds, and probably interact with host proteins the same way, but have different operator-binding specificity. Thus, one repressor cannot functionally replace the other without additional changes in the
DNA-binding sites of the protein. A third kind of relationship is that of the virion coat, terminase, and portal proteins, λ gpE, gpA, and gpB and P22 gp5, gp2, and gp1, respectively. These proteins have the same functions in the two phages—coat protein and DNA packaging nuclease/ motor ATPase and procapsid DNA entry channel, respectively (summarized in Hendrix and Casjens, 2006). These cognate proteins are not recognizably similar in overall amino acid sequence between the two phages, although the two ATPase proteins have similar ATP-binding motifs. However, the two proteins almost certainly have the same basic fold. Thus, gpE and gp5 and gpA and gp2, as well as gpB and gp1, are all very likely ancient homologous pairs that have diverged beyond the point of having recognizable sequence similarity. Finally, similar functions are performed by nonhomologues in the two phages. Examples of this relationship are the (i) nonhomologous Exo (λ) and Erf (P22) proteins that catalyze homologous recombination in completely different ways (Little, 1967; Poteete and Fenton, 1983); (ii) the endolysin proteins gpR and gp19, where the λ protein is a transglycosylase and the P22 protein is a true lysozyme that both cleave the same peptidoglycan bond (Bienkowska-Szewczyk et al., 1981; Rennell and Poteete, 1989); (iii) the gpW and gp4 proteins, which bind between the portal ring of the head and the tail and yet have no recognizable homology or structural similarity; and (iv) the λ receptor binding “fiber” (gpJ and STF) and P22 tailspike (gp9), which have no homology, recognize different structures, and act in different ways [the P22 gp9 enzyme cleaves its polysaccharide receptor (Iwashita and Kanegasaki, 1976)]. Thus, there are potentially a number of different ways in which parallel λ and P22 functions might interact with the hosts of these two phages, and it will be interesting to understand these in detail in both cases. But are there very similar phage proteins that have different host interactions? Do very different phage proteins that have the same function have equally different host interactions? Although no such situations have been identified to date, it remains possible. A comparison of PPIs in λ and P22 is given in Tables VII and VIII.

### D. Enterobacteriophage P2 and its satellite phage, P4

Bacteriophage P2 was originally isolated from the Lisbonne & Carrère strain of *E. coli* by Bertani in 1951 and is a member of the *Myoviridae* family of viruses. It has an icosahedral head and a contractile tail and is a 33.6-kb double-stranded DNA genome (Bertani, 1951; Bertani and Six, 1988; Nilsson and Haggård-Ljungquist, 2006). P2-like prophages are common in the environment (Breitbart et al., 2002) and are present in about 30% of strains in the *E. coli* reference collection (Nilsson et al., 2004).

Bacteriophage P2 has attracted much interest as a “helper” for bacteriophage P4, an 11.6-kb replicon that can exist either as a plasmid or integrated
TABLE VII Comparison of $\lambda$ and P22 interactions

| $\lambda 1$ | $\lambda 2$ | P22 1 | P22 2 | Description |
|-------------|-------------|-------|-------|-------------|
| gpA         | gpB         | gp2   | gp1   | Likely similar contacts—by evolutionary argument $^{a}$ |
| gpA         | gpNu1       | gp2   | gp3   | Likely similar contacts—by evolutionary argument $^{a}$ |
| gpNu1       | gpNu1       | gp3   | gp3   | gpNu1 N-terminal fragment is dimer; gp3 is nonomer; not known if they have the same fold (i.e., are ancient homologues) |
| gpW         | gpB         | gp4   | gp1   | gpW and gp4 have no known homology, but are analogous in that they are the first proteins to bind the underside of the the portal ring; gp4 has no structural homology to $\lambda$ gpFII or gpW (Cardarelli et al., 2010; Olia et al., 2011) |
| gpW         | gpFII       |       |       | No clear gpFII homologue in P22 (gp4 and gp10 of P22 have no sequence similarity) |
| gpB         | gpB         | gp1   | gp1   | Both form dodeamer portal rings $^{a}$ |
| gpC         | gpE         |       |       | No gpC analogue in P22 |
| gpC'        | gpB         |       |       | No gpC analogue in P22 |
| gpC         | gpNu3       |       |       | No gpC analogue in P22 |
| gpNu3       | gpNu3       | gp8   | gp8   | Monomer/dimer/tetramer for gp8; monomer/dimer for gpNu3 |
| gpB         | gpE         | gp1   | gp5   | Seen in reconstructions $^{a}$ |
| gpNu3       | gpB         | gp8   | gp1   | Scaffolds required for portal assembly into procapsid |
|             |             | gp8   | gp5   | Scaffold-coat interaction not studied for $\lambda$ |
| gpD         | gpD         | Dec   | Dec   | Both trimers; bind slightly different sites. $^{a}$ Note that P22 has no Dec, but its very close relative phage L does and L Dec binds to P22 capsids |
| gpE         | gpE         | gp5   | gp5   | Both make icosahedral T=7 head shells $^{a}$ |
| gpU         | gpU         |       |       | No analogue in P22 |
| gpD         | gpE         | Dec   | gp5   | Dec is more discriminatory than D, i.e., unlike D, Dec does not occupy all of the quasi-equivalent local threefold positions on the shell |
| gp4         | gp5         |       |       | Unknown in $\lambda$ |
| gp4         | gp10        |       |       | No gp10 analogue in $\lambda$ (but see gpFII above) |
| gp4         | gp9         |       |       | No gp9 analogue in $\lambda$ (but see Stf below?) |
| gp10        | gp9         |       |       | No gp9 analogue in $\lambda$ (but see Stf below?) |
| gp10        | gp10        |       |       | No gp10 analogue in $\lambda$ (but see gpFII above) |
| gp10        | gp26        |       |       | No gp10 or gp26 (?) analogue in $\lambda$ |

| Stf | Stf | gp26 | gp26 | No gp 26 analogue in λ (maybe gpJ?) |
| --- | --- | --- | --- | --- |
| gpU | gpV | gp9 | gp9 | Both trimers (Stf by homology to T4 fibers); analogues, not homologues |
| gpV | gpV |  |  | No analogues in P22 |
| gpO | gpP | gp18 | gp12 | gpO and gp18 are homologous but gpP and gp12 are not; nonetheless, a good argument can be made that gp18 and gp12 must interact similarly to gpO and gpP to get replication started |
| gpW | gpW | gp4 | gp4 | Analogues—first proteins to bind below the portal ring (see gpW-gpB interaction above) |
| gpFI | gpA |  |  | No gpFI analogue in P22 |
| gpFI | gpE |  |  | No gpFI analogue in P22 |
| gpH | gpG/gpGT |  |  | No analogues in P22 |
| gpV | gpGT |  |  | No analogues in P22 |
| gpH | gpV |  |  | No analogues in P22 |
| CI | CI | Arc | Arc | No analogue in λ |
| Exo | Bet | Ant | C2 | No Ant in λ |
| SieB | Esc | Mnt | Mnt | No analogue in λ |
| CII | CII | C2 | C2 | Homologues (Sevilla-Sierra et al., 1994) |
| Xis | Int | Erf | Erf | λ Exo and P22 Erf proteins are not homologues and function in basically different ways, but they are both essential for homologous recombination |
| Xis? | Xis? | SieB | Esc | Good homologues in P22 but they are unstudied |
| CIII | CIII | C1 | C1 | Homotetramers in both cases |
| Cro | Cro | Xis | Int | Xis proteins are quite different |
| gpRz | gpRz1 | Xis? | Xis | Monomer in λ, but binds cooperatively so could interact on DNA (Abbani et al., 2007); multimer in P22 |
|  |  | C3 | C3 | C3 is homologue of λ CIII; λ CIII is dimer (Halder et al., 2007); P22 interaction by homology with λ CIII |
|  |  | Cro | Cro | Both homodimers |
|  |  | gp15 | gp15 | Predicted for P22 by homology |

$a$ Not recognizably similar in amino acid sequence but *very* likely ancient homologues.

TABLE VIII Comparison of phage $\lambda$-host and P22-host interactions

| $\lambda$ | Host     | P22 | Host    | Notes                                                                 |
|-----------|----------|-----|---------|----------------------------------------------------------------------|
| CII       | HflB     | C1  | HflB    | Should be true for P22 by homology                                   |
| CIII      | HflB     | C3  | HflB    | Should be true for P22 by homology                                   |
| gpJ       | LamB     |     |         | P22 tails recognize polysaccharide so no analogue in P22             |
| Stf       | OmpC     |     |         | P22 tails recognize polysaccharide so no analogue in P22             |
| Xis       | Lon      | Xis | Lon?    | Might be true for P22 by homology                                    |
| Xis       | FtsH     | Xis | FtsH?   | Might be true for P22 by homology                                    |
| gpN       | RNAP     | gp24| RNAP?   | Likely true for P22 by similarity of function                        |
| Int       | IHF      |     |         | P22 does not interact with IHF (Cho et al., 1999)                   |
| Xis       | Fis      |     |         | Unknown for P22                                                     |
| gpQ       | $\sigma^{70}$ | gp23 | $\sigma^{70}$ | Likely true for P22 by similarity of function              |
| P         | DnaB     | gp12| DnaG?   | No analogue of 12 in $\lambda$                                      |
| gpE       | GroE     | gp5 | GroE    | Not known for P22                                                   |
| gpB       | GroE     |     |         | Not known for P22                                                   |
| gpN       | Lon      | gp24| Lon     | Not known for P22                                                   |
| CI        | RecA     | C2  | RecA    | Might be true for P22 by homology and induction by ultraviolet light |
| CII       | ClpYQ    | C1  | ClpYQ   | Might be true for P22 by homology                                    |
| CII       | ClpAP    | C1  | ClpAP   | Might be true for P22 by homology                                    |
| gpO       | ClpXP    | gp18| ClpXP   | Might be true for P22 by homology                                    |
| gpN       | NusA     | gp24| NusA    | Likely true by similarity of function                               |
| CII       | HflD     | C1  | HflD    | Might be true for P22 by homology                                   |
| Gam       | RecB     |     |         | No Gam in P22                                                       |
|           |          | Abc2| RecC    | No Abc2 in $\lambda$                                                |

into the host genome as a prophage (Briani et al., 2001; Deho and Ghisotti, 2006; Lindqvist et al., 1993). P4 lacks genes encoding major structural proteins, but can utilize the structural gene products encoded by P2 for assembly of its own capsid (Christie and Calendar, 1990; Lindqvist et al., 1993; Six, 1975). An overview of the P2 structural biology and a summary of its interactions are shown in Figure 8 and Table IX.

### 1. Gene control circuits
The decision whether to grow lytically or form a lysogen after P2 infection is dependent on the levels of two repressors: the immunity repressor gpC, which blocks lytic growth, and Cox, which blocks the expression of the

FIGURE 8 (A) Schematic diagram of the 33.6-kb P2 genome. Open reading frames (ORFs) are indicated by arrows that reflect their direction and size and are color coded as follow: red/orange, capsid-associated proteins (gpQ, O, N, L); blue, terminase proteins (gpP, M); yellow, lysis proteins (gpY, K, lysA, lysB); green, tail-related proteins (gpR, S, FI, FII, T, U, D); blue, base plate and tail fiber-related proteins (gpV, W, J, H, G); pink, transcriptional control proteins (Ogr, C, Cox); brown, integrase (Int); and purple, replication-related proteins (gpA,B). Nonessential genes and ORFs of unknown function are white. Promoters are indicated by arrows above ORFs. (B) Diagram of protein–protein interactions listed in Table IX. Each P2 and P4 protein is shown as a circle, colored as given earlier. Host proteins are shown as white boxes. (C) Schematic diagram of the P2 virion, color coded as given earlier. Copy numbers of proteins, when known, are indicated in parentheses. (D) Electron micrograph of a P2 virion, stained negatively with uranyl acetate.

genes required for lysogenization. GpC acts as a symmetric dimer, and its three-dimensional structure has been determined (Eriksson et al., 2000; Massad et al., 2010). Cox is multifunctional and acts as a repressor, as well as an excisionase, thereby coupling the transcriptional switch that controls lytic growth versus formation of lysogeny with the integration/excision of the phage genome in or out of the host chromosome. Cox is a tetramer in solution that self-associates to octamers in the absence of DNA (Eriksson and Haggard-Ljungquist, 2000). The transcriptional activator Ogr is required to turn on late gene transcription. Ogr is a Zn finger-containing protein that interacts with the α subunit of the *E. coli* RNA polymerase, recruiting it to the late promoters (Ayers et al., 1994; Fujiki et al., 1976; King et al., 1992; Sunshine and Sauer, 1975; Wood et al., 1997).

Some P2 prophage genes are not blocked by the action of gpC and instead are expressed from the prophage providing the host cell with new properties. Among these “lysogenic conversion” genes, P2 has three genes, *old*, *fun*, and *tin*, that block superinfection of unrelated phages,

TABLE IX Protein-protein interaction of phages P2 and P4

| Protein | Function | Interactions | Notes | References |
|---------|----------|--------------|-------|------------|
| P2-P2 proteins |  |  |  |  |
| gpC | Immunity repressor | Dimer | Y2H<br>3D structure | Eriksson et al., 2000; Massad et al., 2010 |
| Cox | Transcriptional repressor, excisionase | Tetramer that can self-associate to octamers | In vitro; cross-linking and gel filtration | Eriksson and Haggard-Ljungquist, 2000 |
| Int | Integrase | Dimer | In vitro; cross-linking<br>Y2H | Frumerie et al., 2005 |
| gpF<sub>I</sub> | Tail sheath | Polymer, six molecules per sheath striation | Electron microscopy | Lengyel et al., 1974 |
| gpF<sub>II</sub> | Tail tube | Polymer, six molecules per tube striation | Electron microscopy | Lengyel et al., 1974 |
| gpM | Small terminase subunit | Polymer, 8-to 10-fold ring | Biochemistry | Dokland, unpublished data |
| gpM | Small terminase subunit | gpP | Biochemistry | Bowden and Modrich, 1985 |
| gpN | Major capsid protein | T=7 icosahedron (415 copies) | Cryo-EM and 3D reconstruction | Dokland et al., 1992 |
| gpO | Internal scaffolding protein | Dimer | Biochemistry<br>Biochemistry, chromatography | Chang et al., 2008, 2009<br>Chang et al., 2008, 2009 |
| gpP | Large terminase subunit | gpP | Biochemistry | Bowden and Modrich, 1985 |
| gpQ | Connector, portal protein | Dodecameric ring | EM, crystallography | Doan and Dokland, 2007;<br>Rishovd et al., 1994 |

gpT    Tape measure          gpFII (gpFI?)         Inside tail tube
gpV    Tailspike            Tetramer             Sedimentation velocity and molecular mass determination
                                                                 Haggard-Ljungquist et al., 1995; Kageyama et al., 2009
gpV    Tailspike            gpW, base plate component. Belongs to the T4 gene 25-like superfamily
                                                                 Copurification
                                                                 Haggard-Ljungquist et al., 1995

P2–other phage proteins
Tin    Block superinfection of T-even phages        T4SSB protein (gp32)     Genetic evidence
                                                                 Mosig et al., 1997
gpC    Immunity repressor                            P4 anti-repressor ε      Genetic evidence.
                                                                 Y2H
                                                                 Geisselsoder et al., 1981; Liu et al., 1998; Eriksson et al., 2000
gpN    Major capsid protein                          P4 Sid; size determinant, external scaffold
                                                                 Cryo-EM and 3D reconstruction
                                                                 Marvik et al., 1995; Wang et al., 2000; Dokland et al., 2002
gpN    Major capsid protein                          P4 Psu; polarity suppressor; decoration protein
                                                                 Cryo-EM and 3D reconstruction
                                                                 Dokland et al., 1993

P2–host proteins
gpB    Helicase loader. Required for lagging strand synthesis           E. coli helicase DnaB       Genetic evidence.
                                                                 Electron microscopy. In vitro binding.
                                                                 Sunshine et al., 1975; Funnell and Inman, 1983; Odegrip et al., 2000
Ogr    Transcriptional activator of late operons                         E. coli RNA polymerase α C-terminal domain   Genetic evidence
                                                                 Sunshine and Sauer, 1975; Fujiki et al., 1976; King et al., 1992; Ayers et al., 1994; Wood et al., 1997

P4–host proteins
Psu    Polarity suppressor; anti-terminator                                Rho                        Genetics; chemical cross-linking
                                                                 Sauer et al., 1981; Pani et al., 2009
Delta Transcriptional activator, contains two Ogr domains                E. coli RNA polymerase      Genetics; homology to Ogr
                                                                 Julien et al., 1997
but only gpTin is known to have a PPI. P2 gpTin excludes phage T4 by binding to and poisoning T4 gp32 (SSB), thereby inhibiting DNA replication ([Mosig et al., 1997](https://doi.org/10.1016/S0022-2836(97)80155-8)).

Satellite phage P4 has to get access to the morphogenetic genes of its helper P2 to grow lytically. This is accomplished by P4-encoded functions during coinfection with P2. The P4 antirepressor Epsilon derepresses the P2 prophage by interacting with gpC, leading to the expression of P2 lytic genes ([Eriksson et al., 2000](https://doi.org/10.1016/S0022-2836(00)92846-7); [Geisselsoder et al., 1981](https://doi.org/10.1016/0022-2836(81)90283-8); [Liu et al., 1998](https://doi.org/10.1016/S0022-2836(98)80005-9)). P4 also has the capacity to trans-activate P2 late genes in the absence of Ogr using the transcriptional activator Delta. Delta contains two Ogr-like domains and recognizes the same DNA sequence as Ogr. Like Ogr, Delta interacts with the α subunit of *E. coli* RNA polymerase ([Christie et al., 2003](https://doi.org/10.1016/j.jmb.2003.08.047); [Julien et al., 1997](https://doi.org/10.1016/S0022-2836(97)80155-8); [Souza et al., 1977](https://doi.org/10.1016/0022-2836(77)90283-8)).

## 2. P2 DNA replication

After infection, the linear DNA molecule is circularized by ligation of the cohesive (*cos*) ends by the host DNA ligase, and if P2 enters the lytic pathway, DNA replication is initiated by the cis-acting gpA that creates a single-stranded cut at the origin. GpA remains linked covalently to the 5′ end of the cleaved strand ([Liu and Haggard-Ljungquist, 1994](https://doi.org/10.1016/0022-2836(94)90056-7)). Replication occurs via a modified rolling-circle mechanism that generates a double-stranded monomeric circle that is the packaging substrate ([Pruss et al., 1975](https://doi.org/10.1016/0022-2836(75)90337-8)). The cleavage/joining reactions are promoted by two tyrosine residues interspaced by three amino acids in gpA ([Odegrip and Haggard-Ljungquist, 2001](https://doi.org/10.1016/S0022-2836(01)00564-8)). The host Rep helicase is required for all DNA replication ([Calendar et al., 1970](https://doi.org/10.1016/0022-2836(70)90283-8)), while the phage-encoded gpB is a helicase loader that interacts with the *E. coli* DnaB helicase required for lagging strand synthesis ([Funnel and Inman, 1983](https://doi.org/10.1016/0022-2836(83)90283-8); [Odegrip et al., 2000](https://doi.org/10.1016/S0022-2836(00)80155-8)).

## 3. Head structure and assembly

The main structural proteins involved in P2 capsid assembly are gpN, capsid protein; gpO, internal scaffolding protein; and gpQ, connector or portal protein ([Chang et al., 2008](https://doi.org/10.1016/j.jmb.2008.03.047); [Lengyel et al., 1973](https://doi.org/10.1016/0022-2836(73)90283-8); [Linderoth et al., 1991](https://doi.org/10.1016/0022-2836(91)90283-8)). P2 capsids are assembled as empty precursor procapsids consisting of 415 copies of the gpN capsid protein arranged with T=7 icosahedral symmetry and clustered in hexamers and pentamers that make trivalent interactions ([Dokland et al., 1992](https://doi.org/10.1016/0022-2836(92)90283-8)) (Fig. 9). GpQ forms a dodecameric ring that is incorporated into the capsid at one fivefold vertex and through which the DNA is subsequently packaged ([Doan and Dokland, 2007](https://doi.org/10.1016/j.jmb.2007.03.047); [Rishovd et al., 1994](https://doi.org/10.1016/0022-2836(94)90283-8)).

GpO plays a dual role in the assembly process as both a protease and a scaffolding protein ([Chang et al., 2009](https://doi.org/10.1016/j.jmb.2009.03.047); [Dokland, 2011](https://doi.org/10.1016/j.jmb.2011.03.047)). The full-length, 284 residue gpO protein is associated with procapsids at early times ([Marvik et al., 1994](https://doi.org/10.1016/0022-2836(94)90283-8)), but is cleaved autolytically between residues

Bacteriophage Protein–Protein Interactions 261

FIGURE 9 P2 assembly. Schematic diagram of the P2/P4 capsid assembly pathway. (A) Capsid protein gpN, scaffolding protein gpO, and connector (portal) protein gpQ are assembled into the T=7 P2 procapsid. (B) GpN, gpO, and gpQ are then processed to their mature forms, N*, O*, and Q*. DNA is packaged into the procapsid by the P2 terminase complex (gpM and gpP), accompanied by expansion into the mature, angular capsid (B). (C) In the presence of the P4-encoded external scaffolding protein Sid, P2 structural proteins are assembled into a small, T=4 P4 procapsid. Loss of Sid (D), protein processing, and DNA packaging lead to the mature, expanded P4 capsid (E). The decoration protein Psu is added to the mature capsid (F).

141 and 142, leaving an N-terminal fragment, gpO* (formerly known as h7), that remains inside the mature capsid (Chang et al., 2008; Lengyel et al., 1973). Concomitantly, gpN and gpQ are cleaved—presumably by gpO—to their mature forms, gpN* and gpQ*, by removal of 31 and 26 residues, respectively, from the N terminus (Rishovd and Lindqvist, 1992; Rishovd et al., 1994). The protease activity of gpO resides in its N-terminal domain, which binds tightly to gpN and constitutes a serine protease similar to that of the herpesviruses (Chang et al., 2009; Cheng et al., 2004; Dokland, 2011). The C-terminal 90 amino acids comprise a scaffolding domain, which is necessary and sufficient for promoting capsid assembly (Chang et al., 2008, 2009; Wang et al., 2006). It too presumably binds to gpN during assembly, but any interaction is transient (Chang et al., 2009). Both N- and C-terminal domains form dimers in solution (Chang et al., 2009).

Circular P2 genomic DNA substrates are packaged into empty procapsids by the terminase complex, which consists of the small and large terminase proteins (gpM and gpP, respectively). Terminase recognizes and cleaves P2 DNA at the cos sites (Bowden and Modrich, 1985; Ziermann and Calendar, 1990). DNA packaging is associated with major structural transitions that lead to a larger, more angular and thin-shelled,
mature capsid (Chang et al., 2008; Dokland et al., 1992). A head completion protein, gpL, is added (Chang et al., 2008), and tails, which are assembled via a separate pathway, are attached to the unique gpQ-containing portal vertex (Lengyel et al., 1974).

### 4. Tail structure

The 135-nm-long tail is composed of an inner tube and a contractile sheath (Lengyel et al., 1974). The head-proximal end of the free tail has two disc-like structures of different thickness and diameter. In the complete phage, only the thinner disc is seen outside the head. The distal end of the tail contains a baseplate with a spike and six fibers. Sixteen essential tail genes have been identified located in three transcription units. GpFI and gpFII constitute the tail sheath and tail tube, respectively, and there are about 200 copies of each protein per tail (Temple et al., 1991). RpR and gpS are involved in tail completion, as amber mutations in the respective gene produce giant tails and giant tail sheaths (Linderoth et al., 1994). GpV is the tail spike, which forms a trimer in solution (Haggard-Ljungquist et al., 1995; Kageyama et al., 2009). GpW copurifies with gpV and is thus thought to be part of the baseplate, while gpJ localizes to the edge of the baseplate or proximal part of the tail fiber by immunoelectron microscopy (Haggard-Ljungquist et al., 1995). GpH is believed to be the distal part of the tail fiber because it contains regions similar to tail fiber proteins of phages Mu, P1, λ, K3, and T2 (Haggard-Ljungquist et al., 1992). A region of gene $H$ is highly similar to the variable part of the $S$ gene ($Sv$) of phage Mu and the variable part of gene 19 ($19v$) of phage P1, in both cases in the (+) orientation of the invertible segments. P2 does not have an invertible segment, but it contains a sequence identical to the left repeat of the Mu invertible G segment located within the $H$ gene and upstream of the region with a high similarity to Mu $Sv$ and P1 $19v$, indicating that the P2 $H$ gene has been frozen in the (+) orientation, probably by a deletion covering the right inverted repeat and the invertase. This region is therefore inferred to constitute the receptor-binding tip of the tail fiber. By its similarities to gpU and gpU′ of phages Mu and P1, and gp38 of phages T4, Tula, and Tulb, gpG is probably required for fiber assembly. GpT is likely the tape measure, as it is the only protein long enough to span the length of the tail shaft (Christie et al., 2002). Function of the products of genes $X$, $I$, $E$, $E'$ (programmed -1 frameshift extension of $E$), $U$, and $D$ remains unknown.

### 5. P4-induced redirection of P2 capsid assembly

Satellite phage P4 cannot form phage particles by itself, but in the presence of phage P2, P4 genomes are packaged into phage particles using structural proteins supplied by P2 (Six, 1975). However, the P4 capsid is smaller than P2 and contains only 235 copies of gpN, arranged on a T=4
icosahedral lattice (Dokland *et al.*, 1992) (Fig. 9). This size control is affected by the P4 gene product Sid (size determination)(Barrett *et al.*, 1976), which forms an dodecahedral cage-like scaffold around the outside of P4 procapsids (Marvik *et al.*, 1995). Sid self-associates into trimers and dimers that interact with gpN only in two hexamer subunits near the twofold symmetry axes (Wang *et al.*, 2000; Dokland *et al.*, 2002). Gene N mutants, called *sir* (*sid* responsiveness), render the capsid protein unable to form small capsids even in the presence of Sid (Six *et al.*, 1991). Conversely, the so-called super-*sid* or *nms* (*N* mutation sensitive) mutations in *sid* act as second-site suppressors of the *sir* mutations and enable the formation of small capsids (Kim *et al.*, 2001). Although well-formed small procapsids can be assembled from Sid and gpN alone (Dokland *et al.*, 2002), gpO is required for the formation of viable P4 phage (Six, 1975). Presumably this is because its protease activity is essential, and gpO may also facilitate incorporation of the portal. Indeed, *sid* complements the P2 Oam279 mutant, which produces a 237 residue N-terminal fragment of gpO that includes the protease domain but lacks scaffolding activity (Agarwal *et al.*, 1990). The P4 genome contains P2 *cos* sites and is packaged into small capsids using the P2 terminase (Ziermann and Calendar, 1990). The P4-encoded protein Psu is a suppressor of Rho-dependent transcription termination (Pani *et al.*, 2009; Sauer *et al.*, 1981); it also functions as a decoration protein that binds to gpN hexamers and stabilizes the P4 capsid (Dokland *et al.*, 1993; Isaksen *et al.*, 1993).

### E. *Pneumococcus* phage Dp-1 and Cp-1

The lytic bacteriophages Dp-1 and Cp-1 infect the Gram-positive bacterium *Streptococcus pneumoniae*, which colonizes the mucosal surface of the upper respiratory tract and causes serious invasive infections such as pneumonia, meningitis, and sepsis (van der Poll and Opal, 2009). Dp-1 was the first isolated *Streptococcus* phage and is an unassigned member of the Siphoviridae (Fig. 10) (McDonnell, Lain and Tomasz, 1975). It was reported that the Dp-1 virion surprisingly contains lipids that might be arranged as a bilayer surrounding the capsid (Lopez *et al.*, 1977), a feature not reported for any other phage belonging to the Caudovirales. The lytic enzyme Pal of Dp-1 has been studied intensively because it kills virulent streptococci efficiently and has been used as an enzybiotic to cure pneumoccal infections in animals (Loeffler *et al.*, 2001; Rodriguez-Cerrato *et al.*, 2007). The podovirus Cp-1 lytic enzyme Cpl1 was also demonstrated to be an efficient enzybiotic (Loeffler *et al.*, 2003). The Cp-1 genome is a linear, 20-kbp dsDNA predicted to encode 28 proteins and a packaging RNA (pRNA). Like φ29-like viruses, Cp-1 uses protein priming for the initiation of DNA replication (Martin *et al.*, 1996). The biology of Dp-1 and Cp-1 is otherwise only poorly understood.

264 Roman Häuser et al.

### A Dp-1 genome and binary PPIs

| Cluster | Queuosine biosynthesis | DNA replication and recombination | DNA packaging | Unknown | DNA pack. | Capsid | Tail | Lysis | DNA replication |
|---------|------------------------|------------------------------------|---------------|----------|-----------|-------|------|-------|------------------|
| Putative transcripts |  |  |  |  |  |  |  |  |  |
| Genome map |  |  |  |  |  |  |  |  |  |
| PPIs |  |  |  |  |  |  |  |  |  |

---

### B Gene products

| 23 | Terminase-like |
| 37 | Terminase, large subunit |
| 38 | Portal protein |
| 40 | Minor structural protein |
| 41 | Minor capsid protein |
| 42 | Hypothetical protein |
| 43 | Major capsid protein |
| 45 | Hypothetical protein |
| 49 | Listeria Bacteroides repeat domain protein |
| 52 | Tail length tape measure protein |
| 53 | Putative tail protein |
| 54 | Receptor binding protein |
| 55 | Minor structural protein |
|  | Hypothetical proteins |

---

### C Detected PPIs

---

### D PPIs not found

---

### E Unexpected PPIs

---

### F

---

**FIGURE 10** The Dp-1 genome and interactome. (A) Dp-1 genome map. Open reading frames (ORFs) corresponding to their transcription direction are shown as arrows (genome map). Predicted transcripts and functional gene clusters are indicated. All protein–protein interactions (PPIs) identified by systematic Y2H screens among Dp-1 proteins are shown as lines that connect the corresponding ORF symbols. Proteins that bind themselves (homomers) are highlighted by gray ORF symbols. Other ORF colors: refer to Figure C-E. (B–E) A Dp-1 virion model based on identified binary PPIs, mass spectrometry analysis (MS) of mature Dp-1 virions, and homology predictions. (B) Dp-1 virion proteins, including structural proteins identified by MS analysis (in red), or proteins with a homology-based annotation that indicates a virion-related function (in black). (C) Core model of the Dp-1 virion: proteins highlighted in red were identified by MS analysis as structural components. Proteins highlighted in blue represent hypothetical gene products that interact with structural components, thus presumably playing a transient role in virion assembly as they were not identified by MS analysis of Dp-1 virions. Red edges that connect the proteins were shown to interact in Y2H tests. (D) Y2H screens do not identify all expected PPIs. Red lines represent PPIs that were expected to occur among the corresponding proteins known from homologues but were not identified in the systematic Y2H screens. (E) Systematic Y2H screens reveal unexpected PPIs.
The 56.5-kbp genome of Dp-1 is predicted to encode 72 proteins (Fig. 10A) (Sabri *et al.*, 2011). Approximately half of the gene products could be functionally annotated by homology predictions. However, in this and hundreds of other completely sequenced phage genomes, more than half of phage gene products could not be annotated. Proteome-wide Y2H screens, coupled with mass spectrometry analysis (MS) of mature Dp-1 virions, identified 156 unique PPIs among 57 Dp-1 proteins [a detailed list can be found in Sabri *et al.* (2011)], and MS revealed that the mature virion is composed of eight structural proteins. A Dp-1 virion model (Figs. 10B and 10C) physically links many proteins with unknown function to the virion. Because they bind structural proteins but are absent from the mature virion, these Y2H PPIs either indicate novel structural components (when found by MS of the virion particles) or virion assembly facilitators. For instance, homology comparisons of gp40, gp41, and gp53 predict that these proteins are structural (Fig. 10B). However, MS analysis did not detect them as components of the virion. Even though functional predictions by homology comparisons are extremely helpful in proteome annotation, a certain fraction of proteins may be wrongly annotated in the absence of other experimental data (i.e., whether a protein is annotated as a structural component or an assembly facilitator can have an important impact on future experiments).

Such a comprehensive analysis also reveals unexpected interactions that normally would not be tested in hypothesis-driven experiments (Fig. 10E). For example, Dp-1 proteins believed to be involved in DNA replication and recombination, transcriptional regulation, and other pathways exhibit interactions with structural components. This indicates that DNA packaging and DNA replication/repair might be coupled (as in phage T4), for example, by resolving intermediate DNA structures in parallel with packaging. Although the identified PPIs were helpful in obtaining the first hints about the function of the proteins, not all expected PPIs between the structural proteins were detected (Fig. 10D). Nevertheless, this study demonstrated that homology-based genome annotation complemented by proteomic approaches is a useful strategy to gain novel insights into the biology of poorly understood bacteriophages.

Proteins are included that were shown to bind with structural proteins and putative morphogenetic factors, e.g., DNA metabolism (DNA polymerase subunits, RecB, DNA ligase, DnaG), queuosine biosynthesis (Que proteins), lysis (Holins), and transcription (sigma factor). Interactions are symbolized by edges that connect the corresponding proteins (in the case of sigma factor gp69, interaction partners are highlighted by an asterisk). (F) Electron micrograph of a mature Dp-1 virion. C–F from Sabri *et al.* (2010), with permission.The same combination of genome annotation and proteomics was repeated for the 28 putative Cp-1 proteins (Häuser *et al.*, 2011). They detected a total of 12 structural proteins by MS analysis. More than half the proteins showed at least one protein–protein interaction with 17 binary interactions in total. Notably, nearly all were detected between proteins encoded in the structural gene cluster, reflecting a highly modular organization of its interactome. In contrast, Dp-1, with its larger proteome/interactome, shows more cross-communication between proteins belonging to different pathways, although a functional enrichment of interactions between proteins belonging to related cellular processes was still detected (Sabri *et al.*, 2011). It is likely that larger phage proteomes/interaction networks routinely exhibit a higher frequency of protein cross-communication at the protein interaction level due to regulation between various pathways (i.e., the larger the proteome, the more promiscuous the interaction network).

### F. Bacillus subtilis phage φ29-like viruses

φ29-like phages belong to the Picovirinae subfamily of the Podoviridae. Members include φ29, φ15, PZA, BS32, B103, M2Y (M2), Nf, and GA-1. All are lytic; most infect *B.* subtilis, but some grow in *B.* pumilus, *B.* amyloliquefaciens, and *B.* licheniformis (Meijer *et al.*, 2001a). The characteristics of their virions are a prolate capsid (e.g., φ29 is 41.5 by 31.5 nm) with attached head fibers, a connector, a short noncontractile tail (e.g., φ29 with 32.5 nm in length), and the presence of tail appendages (Anderson *et al.*, 1966) (Fig. 11). GA-1 and M2Y lack the gene for head fibers (Yoshikawa *et al.*, 1986), which may stabilize the capsid (Tao *et al.*, 1998). The approximate 20-kbp genome of φ29-like phages is linear and encodes about 25 proteins. The 5′ ends of the genome are attached covalently to a terminal protein (TP). φ29-like viruses are among the smallest dsDNA phages isolated so far and thus represent minimal model systems (Meijer *et al.*, 2001a).

#### 1. Protein–protein interactions

A literature and database search on φ29-like viruses retrieved 30 PPIs (Tables X and XI) with evidence mainly from directed experiments and structure determination studies. Twenty-three are attributed to phage φ29 itself, five to GA-1, and two to Nf, emphasizing the role of φ29 as a model for the whole group. The biology of other φ29-like phages is poorly understood, and only a few proteins have been characterized experimentally. In some cases, φ29 PPIs are conserved among other φ29-like homologues, supporting a close relationship of these phages. However, there are also differences that reveal interesting insights into their evolution. Known φ29 protein–protein interactions include proteins involved in

Bacteriophage Protein–Protein Interactions 267

**A φ29 genome**

| Cluster | DNA replication and early proteins | Virion structure and morphogenesis | Lysis and DNA packaging | Early proteins |
| --- | --- | --- | --- | --- |
| Genome map | 0.2 56 1 2 | 3 4 5 5.5 6 7 | 8 8.5 | 9 10 11 | 12 13 14 15 16 16.5 16.7 16.8 17 | 19.3 kb |
| Transcripts | early | A2c/b A3 | late | early | c2 |

**B Transcription**

- Late genes activator
- RNP α
- dsDNA bind. protein.
- Replication organizer membrane Protein MreB
- UDG UDG inhibitor

**C DNA replication**

- TP
- Membrane protein DNA polymerase

**D Virion**

- Major capsid protein
- Scaffolding protein
- Encapsulation ATPase
- Connector
- Appendage

FIGURE 11 Genome map and literature curated φ29 interactome. (A) Genome and simplified transcriptional map of φ29 (after Meijer et al., 2001a). Major promoters are indicated by their names, and major transcripts are shown as dashed (early) and solid (late) arrows. Other promoters are not indicated for clarity. Open reading frame (ORF) arrows indicate transcription direction. The color of ORF symbols corresponds to the color used for protein symbols in B, C, and D. PPIs known for φ29, organized by their cellular function (transcription, DNA replication, or virion). *B. subtilis* proteins are shown as rectangles, phage proteins as circles. Equal protein symbols contacting each other indicate homomers. Heteromeric PPIs are highlighted by protein symbols that are connected by lines. (D) Schematic cross view of the mature φ29 virion (after Xiang et al., 2008). UDG, uracil–DNA glycosylase; RNAP, RNA polymerase; TP, terminal protein.

transcriptional regulation, DNA replication, and virion structure. Only three PPIs between φ29 and *B. subtilis* proteins have been identified so far.

## 2. Protein-primed DNA replication

Phage φ29 DNA polymerase (DNAP, gp2) interacts with terminal protein (TP, gp3) to initiate genome replication. Interactions have also been shown between the corresponding GA-1 and Nf orthologs (Table XI). The structure of the φ29 TP–DNAP complex has been determined and suggests that the priming domain of TP occupies the dsDNA-binding tunnel, allowing the DNAP to bind negatively charged amino acid residues of TP (Kamtekar et al., 2006).

Phage φ29 replication depends on the presence of a single dAMP residue linked covalently via its 5′-phosphoryl group to the hydroxyl group of Ser²³² of TP. Formation of the phosphoester bond is catalyzed

TABLE X Protein-protein interactions of phage φ29 including phage-host interactions$^a$

| φ291 | Function 1                          | φ292 | Function 2                              | References                                                                 |
|------|-------------------------------------|------|-----------------------------------------|--------------------------------------------------------------------------|
| gp2  | DNA polymerase                      | gp3  | DNA terminal protein TP                 | Kamtekar et al., 2006; Blanco et al., 1987; Dufour et al., 2000; González-Huici et al., 2000 |
| gp3  | DNA terminal protein TP             | gp1  | Membrane-associated early protein       | Bravo et al., 2000                                                        |
| gp3  | DNA terminal protein TP             | gp3  | DNA terminal protein TP                 | Serna-Rico et al., 2003                                                   |
| gp3  | DNA terminal protein                | gp16 | Packaging ATPase                       | Guo et al., 1987                                                          |
| gp4  | Late gene activator                 | gp4  | Late gene activator                     | Grimes and Anderson, 1997; Badia et al., 2006; Mencia et al., 1996b         |
| gp4  | Late gene activator                 | gp6  | Histone-like, nonspecific DNA-binding protein | Calles et al., 2002; Camacho and Salas, 2001                                |
| gp6  | Histone-like, nonspecific dsDNA-binding protein | gp6 | Histone-like, nonspecific dsDNA-binding protein | Abril et al., 1999; Pastrana et al., 1985                                   |
| gp7  | Scaffolding protein                 | gp7  | Scaffolding protein                     | Morais et al., 2003                                                       |
| gp7  | Scaffolding protein                 | gp8  | Major head protein                      | Lee and Guo, 1995                                                        |
| gp7  | Scaffolding protein                 | gp10 | Upper collar/connector protein          | Guo et al., 1991; Lee and Guo, 1995                                       |
| gp10 | Upper collar/connector protein      | gp10 | Upper collar/connector protein           | Guasch et al., 2002; Simpson et al., 2001                                 |
| gp12 | Preneck appendage protein (anti-receptor) | gp12 | Preneck appendage protein (anti-receptor) | Xiang et al., 2009                                                        |
| gp13 | Tail-associated, peptidoglycan-degrading enzyme | gp9 | Tail knob protein                      | García et al., 1983                                                      |

gp13                      Tail-associated, peptidoglycan- degrading enzyme                     gp13                      Tail-associated, peptidoglycan-degrading enzyme                     Xiang et al., 2008
gp16.7                    Replication organizer; DNA-binding membrane protein p16.7                gp16.7                    Replication organizer; DNA-binding membrane protein p16.7             Asensio et al., 2005; Albert et al., 2005; Crucitti et al., 1998; Serna-Rico et al., 2003; Meijer et al., 2001b
gp16.7                    Replication organizer; DNA-binding membrane protein p16.7                gp3                       DNA terminal protein TP                                          Serna-Rico et al., 2003
gp17                      Early protein involved in early DNA replication                         gp6                       Histone-like, nonspecific dsDNA-binding protein                     Crucitti et al. 2003
gp17                      Early protein involved in early DNA replication                         gp17                      Early protein involved in early DNA replication                     Crucitti et al., 2003
gp56 = gp0.8 Host uracil-DNA glycosylase inhibitor                                         gp56 = gp0.8 Host uracil-DNA glycosylase inhibitor                          Serrano-Heras et al., 2007)
ϕ29–Bacillus subtilis interactions                                                                                                                  
gp4                       Late gene activator (3)                                            RpoA                      RNA polymerase subunit α                                        Mencía et al., 1996a; Monsalve et al., 1996b
gp16.7                    Replication organizer, DNA-binding membrane protein gp16.7 (2)         MreB                      Actin-like cytoskeleton protein                                 Muñoz-Espín et al., 2009
gp56 = gp0.8 UDG inhibitor (1)                                                          UDG                       Uracil-DNA glycosylase                                         Serrano-Heras et al., 2006

$^a$ Data from BOND, IntACT, and PDB (ϕ29, Taxon id 186846). All proteins are termed “gp” (gene product), although they are also termed “p” (protein), e.g., gp4 = p4. In the phage-host interaction list, the first protein is from ϕ29, the second from *B. subtilis*. (1) Host RNAP binds via α subunit to p4 (i) bound at the late promoter PA3, thus activating late transcription and (ii) bound to the promoter PA2c, thus blocking transcription through promoter clearance inhibition. (2) PPI plays a role in the organization of ϕ29 DNA replication as peripheral helix-like structures. (3) Inhibition of host UDG; avoids removal of uracil from ϕ29 DNA via UDG-base excision repair.

TABLE XI Protein-protein interactions of φ29-related phages$^a$

| Phage | Protein 1 | Function | Protein 2 | Function | References |
|-------|------------|-----------|------------|-----------|-------------|
| GA-1  | gp6        | Histone-like, nonspecific dsDNA binding protein | gp6        | Histone-like, nonspecific dsDNA-binding protein | Freire et al., 1996 |
| Nf    | gp6        | dsDNA binding protein | gp6        | dsDNA-binding protein | Freire et al., 1996 |
| GA-1  | gp2        | DNA polymerase | gp3        | DNA terminal protein | Illana et al., 1996; Longás et al., 2006 |
| Nf    | gp2        | DNA polymerase | gp3        | DNA terminal protein | Longás et al., 2006; González-Huici et al., 2000 |
| GA-1  | gp5        | Single-stranded DNA-binding protein SSB (1) | gp5        | Single-stranded DNA-binding protein SSB | Gascón et al., 2000 |
| GA-1  | gp12       | Neck appendage protein (2) | gp12       | Neck appendage protein | Schulz et al., 2010 |
| GA-1  | gp4$_G$    | Transcriptional regulator (3) | gp4$_G$    | Transcriptional regulator | Horcajadas et al., 1999 |

$^a$ Data from BOND, IntACT, and PDB (searched for φ29, Taxon id 186846). (1) φ29 and Nf SSB were shown to be stable monomers in solution, whereas GA-1 SSB forms homohexamers. (2) Homotrimer; autocatalytic cleavage similar to φ29 gp12. PDB: 3GUD. (3) Homodimer; negative regulator of early transcription; was shown not to interact with RNAP α subunit as the homologue gp4 of φ29.
by the DNAP itself (Blanco and Salas, 1984; Hermoso *et al.*, 1985); thus formation of a stable DNAP–TP heterodimeric protein complex is required (Blanco *et al.*, 1987). The TP–dAMP is elongated by the same DNAP (Méndez *et al.*, 1992). After the synthesis of six to nine nucleotides, DNAP dissociates from TP and starts continuous, linear synthesis (Méndez *et al.*, 1997). This mechanism of protein-dependent DNA replication initiation was shown for Nf and GA-1 (Illana *et al.*, 1996; Longás *et al.*, 2008), *S.* pneumoniae phage Cp-1, *E.* coli phage PRD1, adenoviruses, and some plasmids and bacterial genomes (e.g., Streptomyces), although there may be differences in the various priming mechanisms (de Jong and van der Vliet, 1999; Martin *et al.*, 1996; Salas, 1991).

The TP–DNAP interaction plays a central role in φ29 DNA replication initiation. However, other PPIs with TP are known, and thus protein-primed DNA replication may be more complicated than is currently known (Figs. 11C and Fig. 12). Chemical cross-linking shows that TP forms homodimers (Serna-Rico *et al.*, 2003). Analysis of TP mutants suggests that the parental TP (TP bound to the parental genome) recruits the free TP–DNAP complex via a TP–TP interaction (Illana *et al.*, 1999), but an additional interaction between parental TP and DNAP might be important for the specificity of replication initiation (González-Huici *et al.*, 2000).

TP interacts with the replication organizer gp16.7 (Serna-Rico *et al.*, 2003) (Fig. 12). Gp16.7, which has the potential to bind to both ssDNA and dsDNA, is an integral membrane protein consisting of an N-terminal membrane anchor and a C-terminal cytoplasmic domain. It localizes φ29 DNA replication to the membrane (Meijer *et al.*, 2000, 2001b; Serna-Rico *et al.*, 2002, 2003). The C-terminal domain of gp16.7 dimerizes in solution but forms larger filaments in the presence of ssDNA (Asensio *et al.*, 2005; Serna-Rico *et al.*, 2003). Replication, which starts at both DNA ends, first produces a so-called type I replication intermediate consisting of dsDNA with ssDNA regions caused by displacement of the non-template DNA parental strand. Later, when the two replication forks have merged, the parental DNA strands become separated into two type II replication intermediates, still containing ssDNA regions in the parental template. A crystal structure of the gp16.7 DNA-binding C-terminal domain complexed with dsDNA reveals three gp16.7 dimers forming a deep, positively charged longitudinal cavity that interacts nonspecifically with the DNA phosphate backbone (Albert *et al.*, 2005). Rp16.7 localizes in peripheral helix-like structures in a bacterial MreB cytoskeleton-dependent manner (Muñoz-Espín *et al.*, 2010). In fact, protein gp16.7 interacts directly with MreB; this interaction is required for efficient φ29 DNA replication, allowing simultaneous replication of multiple templates at various peripheral sites.

Phage φ29 gp1 is another early protein that binds TP (Bravo *et al.*, 2000). Rp1, like gp16.7, is membrane associated and self-associates into long filamentous structures (Bravo and Salas, 1997, 1998). Its membrane
272 Roman Häuser et al.

![Diagram](attachment:diagram.png)

Legend

- p1: Membrane associated early protein
- p2: DNA polymerase
- p3: Terminal protein
- p5: Single-stranded DNA-binding protein
- p6: Histone-like, nonspecific dsDNA binding protein
- p16.7: Replication organizer; ssDNA-binding membrane protein
- p17: Early protein; enhances early DNA replication

FIGURE 12 Interactions in φ29 genome replication and membrane localization. Proteins that contact each other have been shown to interact. For clarity, only one genome end is shown. Parental TP is highlighted by a black symbol, whereas primer TP is in white. The upper part of the figure includes PPIs that preferentially play a role in replication initiation. The lower part of the figure indicates elongated DNA with the associated ssDNA regions. Because p16.7 and p5 (SSB) both bind ssDNA, they could either redirect the parental ssDNA strand to the membrane via TP-p16.7 and p16.7–ssDNA interactions or stabilize it in the cytoplasm via SSB alone (Meijer et al., 2001b). After Bravo et al. (2000) and Serna-Rico et al. (2003).

association and polymerization are mediated through a C-terminal hydrophobic sequence, whereas binding to TP involves an N-terminal region. In contrast to gp16.7, gp1 does not bind DNA (Fig. 12) and is thought to be another component that directs φ29 DNA replication to the membrane (Bravo et al., 2000). Any interplay between gp16.7 and gp1 in this process is unknown.

The φ29 histone-like, nonspecific dsDNA-binding protein gp6 forms a nucleoprotein complex with φ29 DNA that covers most of the genome,

implicating it in genome organization (Serrano *et al.*, 1994). Rp6 either unwinds the genome termini or interacts with TP directly, thus making a TP complex competent to bind the genome ends (Blanco *et al.*, 1986; Serrano *et al.*, 1994). Rp6 forms long oligomeric protein filaments thought to function as a protein backbone for DNA binding (Abril *et al.*, 1999). GA-1 and Nf gp6 orthologues also self-associate (Freire *et al.*, 1996). Rp6 also binds gp17, an early protein that stimulates the first viral replication round *in vivo* when other replication proteins are present at low levels (Crucitti *et al.*, 1998, 2003). In contrast to gp6, gp17 does not bind DNA but its interaction with gp6 enhances gp6 binding to the replication origins, thereby stimulating early rounds of replication.

A homomeric interaction was demonstrated for the single-stranded DNA-binding protein of GA-1. GA-1 SSB was surprisingly found as a homohexamer in solution, whereas SSBs of Nf and φ29 (p5) appear to be stable monomers (Gascón *et al.*, 2000). The homohexamer binds to ssDNA with higher affinity than the monomer and exhibits a higher stimulatory effect on DNA replication.

Bacteriophages have evolved different mechanisms to protect their genomes from degradation by host nucleases in the cell. One protective strategy includes the incorporation of noncanonical nucleotides into their DNA. For instance, *B.* subtilis phage PBS2 uses uracil instead of thymine (Takahashi and Marmur, 1963). PBS2 achieves this unusual goal by increasing the dUTP pool, relative to dTTP, after infection. In addition, PBS2 encodes Ugi, a protein that specifically inhibits the host uracil–DNA glycosylase (UDG)(Cone *et al.*, 1980), which is involved in the first step of removing incorporated uracil residues via the base excision repair pathway. Normally, this takes place to avoid mutational effects in the host genome but it would impair the incorporation of uracil into the PBS2 genome. Cocrystallization of Ugi and UDG revealed that Ugi mimics uracil-containing duplex DNA, thereby blocking efficient DNA binding of UDG (Putnam *et al.*, 1999). φ29 blocks the host UDG by an interaction with the early protein gp56 (gp0.8) that competes with the DNA-binding ability of UDG (Serrano-Heras *et al.*, 2006, 2007, 2008). Although the φ29 genome does not contain uracil (Serrano-Heras *et al.*, 2006), φ29 DNAP is able to incorporate dUMP with a catalytic activity only twofold lower than for that of dTMP. Because φ29 DNAP does not discriminate stringently between dTMP and dUMP, UDG inhibition via gp56 minimizes uracil incorporation.

## 3. Interactions during transcriptional regulation

Transcriptional regulation of the φ29 genome has been studied extensively (Meijer *et al.*, 2001a; Rojo *et al.*, 1998). The transition from early to late gene expression is controlled by gp4 (Fig. 11). Gp4 is a homodimer in solution (Mencía *et al.*, 1996b); it forms a tetramer when bound to the intergenic region between gene 6 and gene 7, which contains the A2b,
A2c, and A3 promoters (Fig. 11A). One binding site is upstream of A3, the late promoter that lacks a -35 box (Barthelemy and Salas, 1989). Once gp4 is bound, it recruits, via a PPI, the host RNAP, leading both to its stabilization on the promoter and transcriptional activation (Nuez et al., 1992). Although gp4 activates late gene expression, it represses early gene expression from the A2b and A2c promoters (Monsalve et al., 1997; Rojo and Salas, 1991). Interestingly, repression of A2c and activation of A3 involve the same residues of gp4 with the C-terminal domain of the RNAP α subunit (Mencía et al., 1996a; Monsalve et al., 1996a,b). Rp6 also interacts with gp4 in regulating the transcription of promoters A2b, A2c, and A3. Gp6 binds gp4 cooperatively to form a repression complex at promoters A2b and A2c while stimulating A3 (Calle et al., 2002; Elías-Arnanz and Salas, 1999).

### 4. Virion structure and morphogenesis

Cryo-EM analyses have revealed many details about the φ29 virion structure (Morais et al., 2003, 2005; Tao et al., 1998; Xiang et al., 2008, 2009), and many structures of virion proteins and factors are known (Table X). Assembly starts from a connector core particle consisting of 12 subunits of gp10 on which the scaffolding protein gp7 binds, followed by recruitment of the major capsid protein gp8 (Guo et al., 1991; Lee and Guo, 1995). The scaffolding protein is needed to create a lattice of concentric shells (Morais et al., 2003) and helps to organize the single capsomers built up by gp8 except for the basal connector capsomer. φ29 proheads were shown to consist of 235 copies of gp8 arranged as hexameric and pentameric capsomeres, indicating that direct interactions between gp8 monomers occur (Morais et al., 2005). The capsid fiber protein gp8.5, which is present in φ29 proheads, is presumably connected via gp8 to the prohead and is suggested to be organized as homotrimers via interactions of a C-terminal coiled-coil region (Morais et al., 2005). In addition to gp8, gp8.5, and gp10, φ29 proheads contain a homopentameric packaging RNA (pRNA) that binds to the connector, forming a basket-like complex (Simpson et al., 2000). pRNA is needed to translocate DNA into the prohead. Furthermore, the DNA translocation machinery needs the ATPase gp16 to drive the packaging reaction (see also chapter 9 of this volume). Rp16 was shown to bind to the connector protein (Ibarra et al., 2001). While other phages use specific DNA sequences to recognize and recruit the genome to the procapsids, φ29 DNA packaging initiation is dependent on the TP gp3 present at the genome ends, indicating that an interaction between gp16 and DNA-bound gp3 recruits the DNA to proheads and initiates packaging (Guo et al., 1987). Release of the scaffolding protein is associated with DNA packaging and pRNA and gp16 are released when there is no more DNA to package.

An interesting aspect of φ29 is that the virion contains several enzymes that are associated with structural proteins. For instance, gp12 is ahomotrimer that participates in host cell recognition and entry. It also shows autocatalytic cleavage, although it remains unclear what the function of that reaction is. Similarly, gp13 is bound as a dimer at the distal end of the φ29 tail knob to gp9; gp13 cleaves both the polysaccharide backbone and peptide cross-links of the cell wall during infection.

# IV. CONCLUSIONS

## A. High-throughput data vs small-scale interactions

It took several decades to collect the interactions of phage λ, even though it has only a few dozen genes. These interactions have been studied in painstaking individual experiments. Since the mid-1990s, new strategies have been developed to identify and analyze PPIs on a larger scale, particularly two-hybrid-related methods and techniques involving protein complex purification and mass spectrometry. Despite the pioneering efforts of (Bartel *et al.*, 1997) to map the interactome of phage T7 in a “single” experiment, their results remain somewhat contentious. Why is that? Studies show that each two-hybrid system produces system-specific data, and each system may only be able to identify as few as 20–30% of all interactions ([Chen et al., 2010](https://doi.org/10.1038/nmeth.1458)). It has been shown that the application of up to 10 different Y2H systems can identify up to 80% of all interactions, but there are still some that go undetected ([Chen et al., 2010](https://doi.org/10.1038/nmeth.1458)). This limited sensitivity can be explained by differences between the individual Y2H systems, expression levels, the way fusion proteins are constructed (e.g., N-terminal fusions vs C-terminal fusions), or linkers between fusion proteins. Some interactions may be completely undetectable by Y2H assays as these experiments are typically done in yeast, which may not have the necessary chaperones for bacterial or phage proteins to fold properly or the proteins may localize differently than in their normal organism.

Similar limitations apply to strategies that involve protein complex purification and mass spectrometry. Because many purifications are sensitive to small changes in protocols, even when a complex is purified within the same laboratory different preparations often result in different protein compositions in subsequent MS analyses ([Gavin et al., 2006](https://doi.org/10.1038/nature04676)). If two different laboratories purify the same protein complexes using different methods, the results may differ even more dramatically ([Hu et al., 2009](https://doi.org/10.1038/msb.2009.1)).

Even if large-scale screens identify the majority of interactions, they will always also identify false positives. For instance, a screen of λPPIs yielded 97 interactions among λ proteins ([Rajagopala et al., 2011](https://doi.org/10.1038/msb.2011.20)). While the authors detected about half of the 33 published interactions listed in
Table III, they also found many new interactions. Many do not seem plausible and are likely false positives. However, there is no way of knowing which ones are in fact false. We can only make educated guesses and rank all interactions by their plausibility, selecting only the best candidates for more detailed studies.

### B. Phage of the same host has different interactomes

Where it is known, all bacteria are infected by numerous unrelated phages. While unrelated phages cannot have identical intraphage interactions, their proteins may interact with the same targets in their host cells. While interaction databases do not have enough information about phage–host interactions to draw general conclusions, several lines of evidence indicate that this is not generally true. For instance, the well-studied phages T7 and λ interact with different host proteins. They seem to share only one common interaction target, namely the RecBCD recombinase complex (see Tables II and IV). λ certainly has about twice as many host–phage interactions than T7, but this is certainly due to λ being temperate. T4 probably does interact somewhat more extensively with the host than T7, and one could argue that this is due to genome size. Similarly, in *Streptococcus* phage Dp-1 the authors found 38 interactions among the 72 phage proteins and its host, while the 28 (unrelated) proteins of phage Cp1 showed only 11 such interactions (*Häuser et al.*, 2011). Together, Dp-1 and Cp-1 target a total of 36 host proteins, of which only 2 are shared. More host–phage interactomes are needed to draw more general conclusions though.

### C. Evolution of interactions [λ, P22]

We expect that related phage have similar interaction patterns, both among phage proteins and between phage and host proteins. This is certainly true for very closely related phages. However, phages evolve much faster than any other living system and thus may lose or gain interactions faster too. Unfortunately, there is not enough interaction data to make detailed comparisons, even among the well-studied lambdoid phages. However, many individual cases of proteins are not obviously homologous by sequence but which are believed to share a common ancestor, for example, the capsid proteins of λ and HK97. These proteins also have similar structures and make similar contacts in the capsid (*Hendrix*, 2005). Evolutionary relationships are also supported by the extensive similarities in parts of their genomes, that is, genome mosaicism (*Casjens*, 2011; *Casjens and Thuman-Commike*, 2011; *Hendrix and Casjens*, 2006). It remains to be seen how many interactions are conserved and how successful phages are in inventing new interactions (or discarding them).

More systematic studies of multiple phages in parallel will be required to obtain comparable data sets in which differences in interaction patterns are not based on the use of noncomparable methods.

# ACKNOWLEDGMENTS

Work on this chapter was partially funded by NIH Grant R01GM79710 to P.U., the European Union (Grant HEALTH-F3-2009-223101)(P.U., R.H., and S.B.), NIH Grant RO1 AI074825 (S.R.C), and the Spanish Ministry of Science and Innovation (Grant BFU2008-00215) to M.S.

# REFERENCES

Abbani, M. A., Papagiannis, C. V., Sam, M. D., Cascio, D., Johnson, R. C., and Clubb, R. T. (2007). Structure of the cooperative Xis-DNA complex reveals a micronucleoprotein filament that regulates phage lambda intasome assembly. *Proc. Natl. Acad. Sci. USA* 104:2109–2114.

Abril, A. M., Marco, S., Carrascosa, J. L., Salas, M., and Hermoso, J. M. (1999). Oligomeric structures of the phage phi29 histone-like protein p6. *J. Mol. Biol.* 292:581–588.

Agarwal, M., Arthur, M., Arbeit, R. D., and Goldstein, R. (1990). Regulation of icosahedral virion capsid size by the *in vivo* activity of a cloned gene product. *Proc. Natl. Acad. Sci. USA* 87:2428–2432.

Agirrezabala, X., Martin-Benito, J., Caston, J. R., Miranda, R., Valpuesta, J. M., and Carrascosa, J. L. (2005a). Maturation of phage T7 involves structural modification of both shell and inner core components. *EMBO J.* 24:3820–3829.

Agirrezabala, X., Martin-Benito, J., Valle, M., Gonzalez, J. M., Valencia, A., Valpuesta, J. M., and Carrascosa, J. L. (2005b). Structure of the connector of bacteriophage T7 at 8Å resolution: Structural homologies of a basic component of a DNA translocating machinery. *J. Mol. Biol.* 347:895–902.

Agirrezabala, X., Velazquez-Muriel, J. A., Gomez-Puertas, P., Scheres, S. H., Carazo, J. M., and Carrascosa, J. L. (2007). Quasi-atomic model of bacteriophage t7 procapsid shell: insights into the structure and evolution of a basic fold. *Structure* 15:461–472.

Albert, A., Munoz-Espin, D., Jimenez, M., Asensio, J. L., Hermoso, J. A., Salas, M., and Meijer, W. J. (2005). Structural basis for membrane anchorage of viral phi29 DNA during replication. *J. Biol. Chem.* 280:42486–42488.

Alfano, C., and McKacken, R. (1989). Ordered assembly of nucleoprotein structures at the bacteriophage lambda replication origin during the initiation of DNA replication. *J. Biol. Chem.* 264:10699–10708.

Ali, S.S., Beckett, E., Bae, S.J., and Navarre, W.W. (2011). The 5.5 protein of phage T7 inhibits H-NS through interactions with the central oligomerization domain. *J. Bacteriol* 193:4881–4892.

Anderson, D. L., Hickman, D. D., and Reilly, B. E. (1966). Structure of *Bacillus subtilis* bacteriophage phi 29 and the length of phi 29 deoxyribonucleic acid. *J. Bacteriol.* 91:2081–2089.

Anderson, W. F., Takeda, Y., Echols, H., and Matthews, B. W. (1979). The structure of a repressor: Crystallographic data for the Cro regulatory protein of bacteriophage lambda. *J. Mol. Biol.* 130:507–510.

Ang, D., Keppel, F., Klein, G., Richardson, A., and Georgopoulos, C. (2000). Genetic analysis of bacteriophage-encoded cochaperons. *Annu. Rev. Genet.* 34:439–456.

Asensio, J. L., Albert, A., Muñoz-Espín, D., González, C., Hermoso, J., Villar, L., Jiménez-Barbero, J., Salas, M., and Meijer, W. J. (2005). Structure of the functional domain of phi29 replication organizer: Insights into oligomerization and DNA binding. *J. Biol. Chem.* 280:20730–20739.

Atanasiu, C., Byron, O., McMiken, H., Sturrock, S. S., and Dryden, D. T. (2001). Characterisation of the structure of ocr, the gene 0.3 protein of bacteriophage T7. *Nucleic Acids Res.* 29:3059–3068.

Atanasiu, C., Su, T. J., Sturrock, S. S., and Dryden, D. T. (2002). Interaction of the ocr gene 0.3 protein of bacteriophage T7 with EcoKI restriction/modification enzyme. *Nucleic Acids Res* 30:3936–3944.

Ayers, D. J., Sunshine, M. G., Six, E. W., and Christie, G. E. (1994). Mutations affecting two adjacent amino acid residues in the alpha subunit of RNA polymerase block transcriptional activation by the bacteriophage P2 Ogr protein. *J. Bacteriol.* 176:7430–7438.

Backhaus, H., and Petri, J. B. (1984). Sequence analysis of a region from the early right operon in phage P22 including the replication genes 18 and 12. *Gene* 32:289–303.

Badia, D., Camacho, A., Pérez-Lago, L., Escandón, C., Salas, M., and Coll, M. (2006). The structure of phage phi29 transcription regulator p4-DNA complex reveals an N-hook motif for DNA. *Mol. Cell.* 22:73–81.

Ball, C. A., and Johnson, R. C. (1991). Efficient excision of phage lambda from the Escherichia coli chromosome requires the Fis protein. *J. Bacteriol.* 173:4027–4031.

Barrett, K. J., Marsh, M. L., and Calendar, R. (1976). Interactions between a satellite bacteriophage and its helper. *J. Mol. Biol.* 106:683–707.

Bartel, P. L., Roecklein, J. A., SenGupta, D., and Fields, S. (1996). A protein linkage map of Escherichia coli bacteriophage T7. *Nat. Genet.* 12:72–77.

Barthelemy, I., and Salas, M. (1989). Characterization of a new prokaryotic transcriptional activator and its DNA recognition site. *J. Mol. Biol.* 208:225–232.

Bazinet, C., Benbasat, J., King, J., Carazo, J. M., and Carrascosa, J. L. (1988). Purification and organization of the gene 1 portal protein required for phage P22 DNA packaging. *Biochemistry* 27:1849–1856.

Bell, C. E., and Lewis, M. (2001). Crystal structure of the lambda repressor C-terminal domain octamer. *J. Mol. Biol.* 314:1127–1136.

Bertani, G. (1951). Studies on lysogeny. I. The mode of phage liberation by lysogenic Escherichia coli. *J. Bacteriol.* 62:293–300.

Bertani, L. E., and Six, E. (1988). The P2-like phages and their parasite, P4. (R. Calendar, ed.), pp. 73-143. Plenum Press, New York.

Bienkowska-Szewczyk, K., Lipinska, B., and Taylor, A. (1981). The R gene product of bacteriophage lambda is the murein transglycosylase. *Mol. Gen. Genet.* 184:111–114.

Blanco, L., Gutierrez, J., Lazaro, J. M., Bernad, A., and Salas, M. (1986). Replication of phage phi 29 DNA in vitro: Role of the viral protein p6 in initiation and elongation. *Nucleic Acids Res.* 14:4923–4937.

Blanco, L., Prieto, I., Gutierrez, J., Bernad, A., Lazaro, J. M., Hermoso, J. M., and Salas, M. (1987). Effect of NH4+ ions on phi 29 DNA-protein p3 replication: Formation of a complex between the terminal protein and the DNA polymerase. *J. Virol.* 61:3983–3991.

Blanco, L., and Salas, M. (1984). Characterization and purification of a phage phi 29-encoded DNA polymerase required for the initiation of replication. *Proc. Natl. Acad. Sci. USA* 81:5325–5329.

Botstein, D., Waddell, C. H., and King, J. (1973). Mechanism of head assembly and DNA encapsulation in Salmonella phage P22. I. Genes, proteins, structures and DNA maturation. *J. Mol. Biol* 80:669–695.

Bowden, D. W., and Modrich, P. (1985). In vitro maturation of circular bacteriophage P2 DNA: Purification of ter components and characterization of the reaction. *J. Biol. Chem.* 260:6999–7007.

Bowie, J. U., and Sauer, R. T. (1989). Equilibrium dissociation and unfolding of the Arc repressor dimer. *Biochemistry* 28:7139–7143.

Bravo, A., Illana, B., and Salas, M. (2000). Compartmentalization of phage phi29 DNA replication: Interaction between the primer terminal protein and the membrane-associated protein p1. *EMBO J.* 19:5575–5584.

Bravo, A., and Salas, M. (1997). Initiation of bacteriophage phi29 DNA replication in vivo: assembly of a membrane-associated multiprotein complex. *J. Mol. Biol.* 269:102–112.

Bravo, A., and Salas, M. (1998). Polymerization of bacteriophage phi 29 replication protein p1 into protfilament sheets. *EMBO J.* 17:6096–6105.

Breg, J. N., van Opheusden, J. H., Burgering, M. J., Boelens, R., and Kaptein, R. (1990). Structure of Arc repressor in solution: Evidence for a family of beta-sheet DNA-binding proteins. *Nature* 346:586–589.

Breitbart, M., Salamon, P., Andresen, B., Mahaffy, J. M., Segall, A. M., Mead, D., Azam, F., and Rohwer, F. (2002). Genomic analysis of uncultured marine viral communities. *Proc. Natl Acad. Sci. USA* 99:14250–14255.

Briani, F., Deho, G., Forti, F., and Ghisotti, D. (2001). The plasmid status of satellite bacteriophage P4. *Plasmid* 45:1–17.

Broadbent, S. E., Davies, M. R., and van der Woude, M. W. (2010). Phase variation controls expression of Salmonella lipopolysaccharide modification genes by a DNA methylation-dependent mechanism. *Mol. Microbiol.* 77:337–353.

Brown, B. M., Bowie, J. U., and Sauer, R. T. (1990). Arc repressor is tetrameric when bound to operator DNA. *Biochemistry* 29:11189–11195.

Buchwald, M., Murialdo, H., and Siminovitch, L. (1970a). The morphogenesis of bacteriophage lambda. II. Identification of the principal structural proteins. *Virology* 42:390–400.

Buchwald, M., and Siminovitch, L. (1969). Production of serum-blocking material by mutants of the left arm of the lambda chromosome. *Virology* 38:1–7.

Buchwald, M., Steed-Glaister, P., and Siminovitch, L. (1970b). The morphogenesis of bacteriophage lambda. I. Purification and characterization of lambda heads and lambda tails. *Virology* 42:375–389.

Calendar, R. (2006). “The Bacteriophages” 2nd Ed. Oxford University Press, Oxford.

Calendar, R., Lindqvist, B., Sironi, G., and Clark, A. (1970). Characterization of REP-mutants and their interaction with P2 phage. *Virology* 40:72–83.

Calle, B., Salas, M., and Rojo, F. (2002). The phi29 transcriptional regulator contacts the nucleoid protein p6 to organize a repression complex. *EMBO J.* 21:6185–6194.

Camacho, A., and Salas, M. (2001). Mechanism for the switch of phi29 DNA early to late transcription by regulatory protein p4 and histone-like protein p6. *EMBO J.* 20:6060–6070.

Camara, B., Liu, M., Reynolds, J., Shadrin, A., Liu, B., Kwok, K., Simpson, P., Weinzierl, R., Severinov, K., Cota, E., Matthews, S., and Wigneshweraraj, S. R. (2010). T7 phage protein Gp2 inhibits the Escherichia coli RNA polymerase by antagonizing stable DNA strand separation near the transcription start site. *Proc. Natl. Acad. Sci. USA* 107:2247–2252.

Campbell, A., del-Campillo-Campbell, A., and Ginsberg, M. L. (2002). Specificity in DNA recognition by phage integrases. *Gene* 300:13–18.

Cardarelli, L., Lam, R., Tuite, A., Baker, L. A., Sadowski, P. D., Radford, D. R., Rubinstein, J. L., Battaile, K. P., Chirgadze, N., Maxwell, K. L., and Davidson, A. R. (2010). The crystal structure of bacteriophage HK97 gp6: Defining a large family of head-tail connector proteins. *J. Mol. Biol.* 395:754–768.

Casjens, S. (1974). Bacteriophage lambda FII gene protein: Role in head assembly. *J. Mol. Biol.* 90:1–20.

Casjens, S. (1979). Molecular organization of the bacteriophage P22 coat protein shell. *J. Mol. Biol.* 131:1–14.

Casjens, S. (2011). Evolution of mosaically related tailed bacteriophage genomes seen through the lens of phage P22 virion assembly. *Virology*, in press.

Casjens, S., Eppler, K., Parr, R., and Poteete, A. R. (1989). Nucleotide sequence of the bacteriophage P22 gene 19 to 3 region: Identification of a new gene required for lysis. *Virology* **171**:588–598.

Casjens, S., and Hayden, M. (1988). Analysis *in vivo* of the bacteriophage P22 headful nuclease. *J. Mol. Biol.* **199**:467–474.

Casjens, S., Horn, T., and Kaiser, A. D. (1972). Head assembly steps controlled by genes F and W in bacteriophage lambda. *J. Mol. Biol.* **64**:551–563.

Casjens, S.R. (2005). Comparative genomics and evolution of the tailed-bacteriophages. *Curr. Opin. Microbiol.* **8**:451–458.

Casjens, S. R., and Hendrix, R. W. (1974). Locations and amounts of major structural proteins in bacteriophage lambda. *J. Mol. Biol.* **88**:535–545.

Casjens, S. R., and Thuman-Commike, P. A. (2011). Evolution of mosaically related tailed bacteriophage genomes seen through the lens of phage P22 virion assembly. *Virology* **411**:393–415.

Catalano, C.E. (2000). The terminase enzyme from bacteriophage lambda: A DNA-packaging machine. *Cell. Mol. Life Sci.* **57**:128–148.

Catalano, C. E., and Tomka, M. A. (1995). Role of gpFI protein in DNA packaging by bacteriophage lambda. *Biochemistry* **34**:10036–10042.

Cerritelli, M. E., Conway, J. F., Cheng, N., Trus, B. L., and Steven, A. C. (2003a). Molecular mechanisms in bacteriophage T7 procapsid assembly, maturation, and DNA containment. *Adv. Protein Chem.* **64**:301–323.

Cerritelli, M. E., and Studier, F. W. (1996a). Assembly of T7 capsids from independently expressed and purified head protein and scaffolding protein. *J. Mol. Biol.* **258**:286–298.

Cerritelli, M. E., and Studier, F. W. (1996b). Purification and characterization of T7 head-tail connectors expressed from the cloned gene. *J. Mol. Biol.* **258**:299–307.

Cerritelli, M. E., Trus, B. L., Smith, C. S., Cheng, N., Conway, J. F., and Steven, A. C. (2003b). A second symmetry mismatch at the portal vertex of bacteriophage T7: 8-fold symmetry in the procapsid core. *J. Mol. Biol.* **327**:1–6.

Chang, C. Y., Kemp, P., and Molineux, I. J. (2010). Gp15 and gp16 cooperate in translocating bacteriophage T7 DNA into the infected cell. *Virology* **398**:176–186.

Chang, J. R., Poliakov, A., Prevelige, P. E., Mobley, J. A., and Dokland, T. (2008). Incorporation of scaffolding protein gpO in bacteriophages P2 and P4. *Virology* **370**:352–361.

Chang, J. R., Spilman, M. S., Rodenburg, C. M., and Dokland, T. (2009). Functional domains of the bacteriophage P2 scaffolding protein: Identification of residues involved in assembly and protease activity. *Virology* **384**:144–150.

Cheetham, G. M., Jeruzalmi, D., and Steitz, T. A. (1999). Structural basis for initiation of transcription from an RNA polymerase-promoter complex. *Nature* **399**:80–83.

Cheetham, G. M., and Steitz, T. A. (1999). Structure of a transcribing T7 RNA polymerase initiation complex. *Science* **286**:2305–2309.

Chen, D. H., Baker, M. L., Hryc, C. F., DiMaio, F., Jakana, J., Wu, W., Dougherty, M., Haase-Pettingell, C., Schmid, M. F., Jiang, W., Baker, D., King, J. A., *et al.* (2011). Structural basis for scaffolding-mediated assembly and maturation of a dsDNA virus. *Proc. Natl. Acad. Sci. USA* **108**:1355–1360.

Chen, K. C., Vannais, D. B., Jones, C., Patterson, D., and Davidson, J. N. (1989). Mapping of the gene encoding the multifunctional protein carrying out the first three steps of pyrimidine biosynthesis to human chromosome 2. *Hum. Genet.* **82**:40–44.

Chen, Y. C., Rajagopala, S. V., Stellberger, T., and Uetz, P. (2010). Exhaustive benchmarking of the yeast two-hybrid system. *Nature Methods* **7**:667–668.

Cheng, H., Shen, N., Pei, J., and Grishin, N. V. (2004). Double-stranded DNA bacteriophage prohead protease is homologous to herpesvirus protease. *Protein Sci.* **13**:2260–2269.

Cheng, X., Wang, W., and Molineux, I. J. (2004). F exclusion of bacteriophage T7 occurs at the cell membrane. *Virology* **326**:340–352.

Cho, E. H., Gumpert, R. I., and Gardner, J. F. (2002). Interactions between integrase and excisionase in the phage lambda excisive nucleoprotein complex. *J. Bacteriol.* 184:5200–5203.

Cho, E. H., Nam, C. E., Alcaraz, R., Jr., and Gardner, J. F. (1999). Site-specific recombination of bacteriophage P22 does not require integration host factor. *J. Bacteriol.* 181:4245–4249.

Christie, G., Anders, D., McAlister, V., Goodwin, T., Julien, B., and Calendar, R. (2003). Identification of upstream sequences essential for activation of a bacteriophage P2 late promoter. *J. Bacteriol* 185:4609–4614.

Christie, G., Temple, L., Bartlett, B., and Goodwin, T. (2002). Programmed translational frameshift in the bacteriophage P2 FETUD tail gene operon. *J. Bacteriol* 184:6522–6531.

Christie, G. E., and Calendar, R. (1990). Interactions between satellite bacteriophage P4 and its helpers. *Annu. Rev. Genet.* 24:465–490.

Clement, J. M., Lepouce, E., Marchal, C., and Hofnung, M. (1983). Genetic study of a membrane protein: DNA sequence alterations due to 17 lamB point mutations affecting adsorption of phage lambda. *EMBO J.* 2:77–80.

Cone, R., Bonura, T., and Friedberg, E. C. (1980). Inhibitor of uracil-DNA glycosylase induced by bacteriophage PBS2: Purification and preliminary characterization. *J. Biol. Chem.* 255:10354–10358.

Cortines, J. R., Weigele, P. R., Gilcrease, E. B., Casjens, S. R., and Teschke, C. M. (2011). Decoding bacteriophage P22 assembly: Identification of two charged residues in scaffolding protein responsible for coat protein interaction. *Virology* 421:1–11.

Court, D. L., Oppenheim, A. B., and Adhya, S. L. (2007). A new look at bacteriophage lambda genetic networks. *J. Bacteriol.* 189:298–304.

Crisona, N. J., Weinberg, R. L., Peter, B. J., Sumners, D. W., and Cozzarelli, N. R. (1999). The topological mechanism of phage lambda integrase. *J. Mol. Biol.* 289:747–775.

Crucitti, P., Abril, A. M., and Salas, M. (2003). Bacteriophage phi 29 early protein p17. Self-association and hetero-association with the viral histone-like protein p6. *J. Biol. Chem* 278:4906–4911.

Crucitti, P., Lazaro, J. M., Benes, V., and Salas, M. (1998). Bacteriophage phi29 early protein p17 is conditionally required for the first rounds of viral DNA replication. *Gene* 223:135–142.

d’Herelle, F. (1926). The Bacteriophage and Its Behavior. Williams & Wilkins, Baltimore, MD: 490–497.

Darling, P.J., Holt, J. M., and Ackers, G. K. (2000). Coupled energetics of lambda cro repressor self-assembly and site-specific DNA operator binding I: Analysis of cro dimerization from nanomolar to micromolar concentrations. *Biochemistry* 39:11500–11507.

Das, A. (1992). How the phage lambda N gene product suppresses transcription termination: communication of RNA polymerase with regulatory proteins mediated by signals in nascent RNA. *J. Bacteriol.* 174:6711–6716.

Datta, A. B., Roy, S., and Parrack, P. (2005a). Role of C-terminal residues in oligomerization and stability of lambda CII: Implications for lysis-lysogeny decision of the phage. *J. Mol. Biol.* 345:315–324.

Datta, I., Banik-Maiti, S., Adhikari, L., Sau, S., Das, N., and Mandal, N. C. (2005b). The mutation that makes Escherichia coli resistant to lambda P gene-mediated host lethality is located within the DNA initiator Gene dnaA of the bacterium. *J. Biochem. Mol. Biol.* 38:89–96.

Datta, I., Sau, S., Sil, A. K., and Mandal, N. C. (2005c). The bacteriophage lambda DNA replication protein P inhibits the oriC DNA- and ATP-binding functions of the DNA replication initiator protein DnaA of Escherichia coli. *J. Biochem. Mol. Biol.* 38:97–103.

De Anda, J., Poteete, A. R., and Sauer, R. T. (1983). P22 c2 repressor. Domain structure and function. *J. Biol. Chem* 258:10536–10542.

de Beer, T., Fang, J., Ortega, M., Yang, Q., Maes, L., Duffy, C., Berton, N., Sippy, J., Overduin, M., Feiss, M., and Catalano, C. E. (2002). Insights into specific DNA recognition during the assembly of a viral genome packaging machine. *Mol. Cell*. **9**:981–991.

de Beus, M. D., Doyle, S. M., and Teschke, C. M. (2000). GroEL binds a late folding intermediate of phage P22 coat protein. *Cell Stress Chaperones* **5**:163–172.

de Jong, R. N., and van der Vliet, P. C. (1999). Mechanism of DNA replication in eukaryotic cells: Cellular host factors stimulating adenovirus DNA replication. *Gene* **236**:1–12.

Deho, G., and Ghisotti, D. (2006). The satellite phage P4. (R. Calendar, ed.), pp. 391-408. Oxford University Press, New York.

Deighan, P., Diez, C. M., Leibman, M., Hochschild, A., and Nickels, B. E. (2008). The bacteriophage lambda Q antiterminator protein contacts the beta-flap domain of RNA polymerase. *Proc. Natl. Acad. Sci. USA* **105**:15305–15310.

Demerec, M., and Fano, U. (1945). Bacteriophage-resistant mutants in *Escherichia coli*. *Genetics* **30**:119–136.

Doan, D. N. P., and Dokland, T. (2007). The gpQ portal protein of bacteriophage P2 forms dodecameric connectors in crystals. *J. Struct. Biol.* **157**:432–436.

Dodd, I. B., Perkins, A. J., Tsemitsidis, D., and Egan, J. B. (2001). Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny. *Genes Dev.* **15**:3013–3022.

Dodson, M., Echols, H., Wickner, S., Alfano, C., Mensa-Wilmot, K., Gomes, B., LeBowitz, J., Roberts, J. D., and McMacken, R. (1986). Specialized nucleoprotein structures at the origin of replication of bacteriophage lambda: Localized unwinding of duplex DNA by a six-protein reaction. *Proc. Natl. Acad. Sci. USA* **83**:7638–7642.

Dokland, T. (2011). gpO peptidase (Enterobacteria phage P2). *In “Handbook of proteolytic enzymes”* (N. D. Rawlings and G. Salvesen, eds.). Elsevier, Oxford.

Dokland, T., Isaksen, M. L., Fuller, S. D., and Lindqvist, B. H. (1993). Capsid localization of the bacteriophage P4 Psu protein. *Virology* **194**:682–687.

Dokland, T., Lindqvist, B. H., and Fuller, S. D. (1992). Image reconstruction from cryoelectron micrographs reveals the morphopoietic mechanism in the P2-P4 bacteriophage system. *EMBO J. Eur. Mol. Biol. Organ. J* **11**:839–846.

Dokland, T., and Murialdo, H. (1993). Structural transitions during maturation of bacteriophage lambda capsids. *J. Mol. Biol.* **233**:682–694.

Dokland, T., Wang, S., and Lindqvist, B. H. (2002). The structure of P4 procapsids produced by coexpression of capsid and external scaffolding proteins. *Virology* **298**:224–231.

Duffy, C., and Feiss, M. (2002). The large subunit of bacteriophage lambda’s terminase plays a role in DNA translocation and packaging termination. *J. Mol. Biol.* **316**:547–561.

Dufour, E., Méndez, J., Lázaro, J. M., de Vega, M., Blanco, L., and Salas, M. (2000). An aspartic acid residue in TPR-1, a specific region of protein-priming DNA polymerases, is required for the functional interaction with primer terminal protein. *J. Mol. Biol.* **304**:289–300.

Dunn, J. J., and Studier, F. W. (1983). Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. *J. Mol. Biol.* **166**:477–535.

Echols, H. (1986). Multiple DNA-protein interactions governing high-precision DNA transactions. *Science* **233**:1050–1056.

Elías-Arnanz, M., and Salas, M. (1999). Functional interactions between a phage histone-like protein and a transcriptional factor in regulation of phi29 early-late transcriptional switch. *Genes Dev.* **13**:2502–2513.

Eppler, K., Wyckoff, E., Goates, J., Parr, R., and Casjens, S. (1991). Nucleotide sequence of the bacteriophage P22 genes required for DNA packaging. *Virology* **183**:519–538.

Eriksson, J. M., and Haggard-Ljungquist, E. (2000). The multifunctional bacteriophage P2 cox protein requires oligomerization for biological activity. *J. Bacteriol.* **182**:6714–6723.

Eriksson, S. K., Liu, T., and Haggard-Ljungquist, E. (2000). Interacting interfaces of the P4 antirepressor E and the P2 immunity repressor C. *Mol. Microbiol.* **36**:1148–1155.

Esposito, D., and Gerard, G. F. (2003). The *Escherichia coli* Fis protein stimulates bacteriophage lambda integrative recombination *in vitro*. *J. Bacteriol.* **185**:3076–3080.

Frackman, S., Siegele, D. A., and Feiss, M. (1985). The terminase of bacteriophage lambda: Functional domains for cosB binding and multimer assembly. *J. Mol. Biol.* **183**:225–238.

Franklin, N. C. (1985). Conservation of genome form but not sequence in the transcription antitermination determinants of bacteriophages lambda, phi 21 and P22. *J. Mol. Biol.* **181**:75–84.

Freire, R., Serrano, M., Salas, M., and Hermoso, J. M. (1996). Activation of replication origins in phi29-related phages requires the recognition of initiation proteins to specific nucleoprotein complexes. *J. Biol. Chem.* **271**:31000–31007.

Friedman, D. I., and Court, D. L. (2001). Bacteriophage lambda: Alive and well and still doing its thing. *Curr. Opin. Microbiol.* **4**:201–207.

Frumerie, C. M., Eriksson, J., Dugast, M., and Haggard-Ljungquist, E. (2005). Dimerization of bacteriophage P2 integrase is not required for binding to its DNA target but for its biological activity. *Gene* **344**:221–231.

Fujiki, H., Palm, P., Zillig, W., Calendar, R., and Sunshine, M. (1976). Identification of a mutation within the structural gene for the α subunit of DNA-dependent RNA polymerase of *E. coli*. *Mol. Gen. Genet* **145**:19–22.

Fujisawa, H., Shibata, H., and Kato, H. (1991). Analysis of interactions among factors involved in the bacteriophage T3 DNA packaging reaction in a defined *in vitro* system. *Virology* **185**:788–794.

Fujiyama, A., Kohara, Y., and Okazaki, T. (1981). Initiation sites for discontinuous DNA synthesis of bacteriophage T7. *Proc. Natl. Acad. Sci. USA* **78**:903–907.

Fuller, M. T., and King, J. (1982). Assembly *in vitro* of bacteriophage P22 procapsids from purified coat and scaffolding subunits. *J. Mol. Biol.* **156**:633–665.

Funnell, B. E., and Inman, R. B. (1983). Bacteriophage P2 DNA replication: Characterization of the requirement of the gene B protein *in vivo*. *J. Mol. Biol.* **167**:311–334.

García, J. A., Carrascosa, J. L., and Salas, M. (1983). Assembly of the tail protein of the *Bacillus subtilis* phage phi 29. *Virology* **125**:18–30.

Garcia, L. R., and Molineux, I. J. (1995). Rate of translocation of bacteriophage T7 DNA across the membranes of *Escherichia coli*. *J. Bacteriol.* **177**:4066–4076.

Garcia, L. R., and Molineux, I. J. (1996). Transcription-independent DNA translocation of bacteriophage T7 DNA into *Escherichia coli*. *J. Bacteriol.* **178**:6921–6929.

Gascón, I., Gutiérrez, C., and Salas, M. (2000). Structural and functional comparative study of the complexes formed by viral o29, Nf and GA-1 SSB proteins with DNA. *J. Mol. Biol.* **296**:989–999.

Gavin, A. C., Aloy, P., Grandi, P., Krause, R., Boesche, M., Marzioch, M., Rau, C., Jensen, L. J., Bastuck, S., Dumpelfeld, B., Edelmann, A., Heurtier, M. A., *et al.* (2006). Proteome survey reveals modularity of the yeast cell machinery. *Nature* **440**:631–636.

Geisselsoder, J., Youdarian, P., Deho, G., Chidambaram, M., Goldstein, R., and Ljungquist, E. (1981). Mutants of satellite virus P4 that cannot derepress their bacteriophage P2 helper. *J. Mol. Biol.* **148**:1–19.

Georgopoulos, C., Tilly, K., and Casjens, S. (1983). Lambdoid phage head assembly. In “Lambda II” (R. Hendrix, J. Roberts, F. W. Stahl, and R. Weisberg, eds.), pp. 279–304. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.

Georgopoulos, C. P., Hendrix, R. W., Casjens, S. R., and Kaiser, A. D. (1973). Host participation in bacteriophage lambda head assembly. *J. Mol. Biol.* **76**:45–60.

Ghosh, S., Hamdan, S. M., Cook, T. E., and Richardson, C. C. (2008). Interactions of *Escherichia coli* thioredoxin, the processivity factor, with bacteriophage T7 DNA polymerase and helicase. *J. Biol. Chem.* **283**:32077–32084.

Ghosh, S., Hamdan, S. M., and Richardson, C. C. (2010). Two modes of interaction of the single-stranded DNA-binding protein of bacteriophage T7 with the DNA polymerase-thioredoxin complex. *J. Biol. Chem.* **285**:18103–18112.

Gilcrease, E. B., Winn-Stapley, D. A., Hewitt, F. C., Joss, L., and Casjens, S. R. (2005). Nucleotide sequence of the head assembly gene cluster of bacteriophage L and decoration protein characterization. *J. Bacteriol.* 187:2050–2057.

González-Huici, V., Lázaro, J. M., Salas, M., and Hermoso, J. M. (2000). Specific recognition of parental terminal protein by DNA polymerase for initiation of protein-primed DNA replication. *J. Biol. Chem.* 275:14678–14683.

Gottesman, M. E., and Weisberg, R. A. (2004). Little lambda, who made thee? *Microbiol. Mol. Biol. Rev.* 68:796–813.

Gottesman, S., Gottesman, M., Shaw, J. E., and Pearson, M. L. (1981). Protein degradation in *E. coli*: The lon mutation and bacteriophage lambda N and cII protein stability. *Cell* 24:225–233.

Greene, B., and King, J. (1994). Binding of scaffolding subunits within the P22 procapsid lattice. *Virology* 205:188–197.

Greene, B., and King, J. (1996). Scaffolding mutants identifying domains required for P22 procapsid assembly and maturation. *Virology* 225:82–96.

Grimes, S., and Anderson, D. (1997). The bacteriophage phi29 packaging proteins supercoil the DNA ends. *J. Mol. Biol.* 266:901–914.

Grundling, A., Smith, D. L., Blasi, U., and Young, R. (2000). Dimerization between the holin and holin inhibitor of phage lambda. *J. Bacteriol.* 182:6075–6081.

Guasch, A., Pous, J., Ibarra, B., Gomis-Ruth, F. X., Valpuesta, J. M., Sousa, N., Carrascosa, J. L., and Coll, M. (2002). Detailed architecture of a DNA translocating machine: The high-resolution structure of the bacteriophage phi29 connector particle. *J. Mol. Biol.* 315:663–676.

Guo, P., Peterson, C., and Anderson, D. (1987). Initiation events in *in vitro* packaging of bacteriophage phi 29 DNA-gp3. *J. Mol. Biol.* 197:219–228.

Guo, P. X., Erickson, S., Xu, W., Olson, N., Baker, T. S., and Anderson, D. (1991). Regulation of the phage phi 29 prohead shape and size by the portal vertex. *Virology* 183:366–373.

Hadden, J. M., Declais, A. C., Carr, S. B., Lilley, D. M., and Phillips, S. E. (2007). The structural basis of Holliday junction resolution by T7 endonuclease I. *Nature* 449:621–624.

Haggard-Ljungquist, E., Halling, C., and Calendar, R. (1992). DNA sequences of the tail fiber genes of bacteriophage P2: evidence for horizontal transfer of tail fiber genes among unrelated bacteriophages. *J. Bacteriol.* 174:1462–1477.

Haggard-Ljungquist, E., Jacobsen, E., Rishovd, S., Six, E. W., Nilssen, O., Sunshine, M. G., Lindqvist, B. H., Kim, K. J., Barreiro, V., Koonin, E. V., *et al.* (1995). Bacteriophage P2: Genes involved in baseplate assembly. *Virology* 213:109–121.

Halder, S., Datta, A. B., and Parrack, P. (2007). Probing the antiprotease activity of lambda-CIII, an inhibitor of the *Escherichia coli* metalloprotease HflB (FtsH). *J. Bacteriol.* 189:8130–8138.

Hamada, K., Fujisawa, H., and Minagawa, T. (1986). Overproduction and purification of the products of bacteriophage T3 genes 18 and 19, two genes involved in DNA packaging. *Virology* 151:110–118.

Hamdan, S. M., Marintcheva, B., Cook, T., Lee, S. J., Tabor, S., and Richardson, C. C. (2005). A unique loop in T7 DNA polymerase mediates the binding of helicase-primase, DNA binding protein, and processivity factor. *Proc. Natl. Acad. Sci. USA* 102:5096–5101.

Hamdan, S. M., and Richardson, C. C. (2009). Motors, switches, and contacts in the replisome. *Annu. Rev. Biochem.* 78:205–243.

Hang, Q., Woods, L., Feiss, M., and Catalano, C. E. (1999). Cloning, expression, and biochemical characterization of hexahistidine-tagged terminase proteins. *J. Biol. Chem.* 274:15305–15314.

Häuser, R., Sabri, M., Moineau, S., and Uetz, P. (2011). The proteome and interactome of Streptococcus pneumoniae phage Cp-1. *J. Bacteriol.* 193:3135–3138.

He, Z. G., Rezende, L. F., Willcox, S., Griffith, J. D., and Richardson, C. C. (2003). The carboxyl-terminal domain of bacteriophage T7 single-stranded DNA-binding protein modulates DNA binding and interaction with T7 DNA polymerase. *J. Biol. Chem.* 278:29538–29545.

He, Z. G., and Richardson, C. C. (2004). Effect of single-stranded DNA-binding proteins on the helicase and primase activities of the bacteriophage T7 gene 4 protein. *J. Biol. Chem.* 279:22190–22197.

Heineman, R. H., and Bull, J. J. (2007). Testing optimality with experimental evolution: Lysis time in a bacteriophage. *Evolution* 61:1695–1709.

Hendrix, R., and Casjens, S. (2006). Bacteriophage lambda and its genetic neighborhood. Chapter 27. In “The Bacteriophages” (R. Calendar, ed.), pp. 409–445. Oxford University Press, Oxford.

Hendrix, R., Roberts, J., Stahl, F. W., and Weisberg, R. (1983). Lambda II. p. 694. CSHL Press, Cold Spring Harbor, NY.

Hendrix, R. W. (2002). Bacteriophages: Evolution of the majority. *Theor. Popul. Biol.* 61:471–480.

Hendrix, R. W. (2005). Bacteriophage HK97: Assembly of the capsid and evolutionary connections. *Adv. Virus Res.* 64:1–14.

Hendrix, R. W., and Casjens, S. R. (1974). Protein fusion: A novel reaction in bacteriophage lambda head assembly. *Proc. Natl. Acad. Sci. USA* 71:1451–1455.

Hendrix, R. W., and Casjens, S. R. (1975). Assembly of bacteriophage lambda heads: Protein processing and its genetic control in petit lambda assembly. *J. Mol. Biol.* 91:187–199.

Hendrix, R. W., and Duda, R. L. (1992). Bacteriophage lambda PaPa: Not the mother of all lambda phages. *Science* 258:1145–1148.

Hendrix, R. W., and Duda, R. L. (1998). Bacteriophage HK97 head assembly: A protein ballet. *Adv. Virus Res.* 50:235–288.

Herman, C., Thevenet, D., D’Ari, R., and Bouloc, P. (1997). The HflB protease of *Escherichia coli* degrades its inhibitor lambda cIII. *J. Bacteriol* 179:358–363.

Hermoso, J. M., Méndez, E., Soriano, F., and Salas, M. (1985). Location of the serine residue involved in the linkage between the terminal protein and the DNA of phage phi 29. *Nucleic Acids Res.* 13:7715–7728.

Hershey, A. D. (1952). Inheritance in bacteriophage. *Ann. N.Y. Acad. Sci* 54:960–962.

Ho, Y., and Rosenberg, M. (1982). Characterization of the phage lambda regulatory protein cII. *Ann. Microbiol.* 133:215–218.

Ho, Y. S., Pfarr, D., Strickler, J., and Rosenberg, M. (1992). Characterization of the transcription activator protein C1 of bacteriophage P22. *J. Biol. Chem.* 267:14388–14397.

Hohn, T., Flick, H., and Hohn, B. (1975). Petit lambda, a family of particles from coliphage lambda infected cells. *J. Mol. Biol.* 98:107–120.

Horcajadas, J. A., Monsalve, M., Rojo, F., and Salas, M. (1999). The switch from early to late transcription in phage GA-1: Characterization of the regulatory protein p4G. *J. Mol. Biol.* 290:917–928.

Hu, P., Janga, S. C., Babu, M., Diaz-Mejia, J. J., Butland, G., Yang, W., Pogoutse, O., Guo, X., Phanse, S., Wong, P., Chandran, S., Christopoulos, C., *et al.* (2009). Global functional atlas of *Escherichia coli* encompassing previously uncharacterized proteins. *PLoS Biol.* 7:e96.

Huang, J., Villemain, J., Padilla, R., and Sousa, R. (1999). Mechanisms by which T7 lysozyme specifically regulates T7 RNA polymerase during different phases of transcription. *J. Mol. Biol.* 293:457–475.

Huber, H. E., Beauchamp, B. B., and Richardson, C. C. (1988). *Escherichia coli* dGTP triphosphohydrolase is inhibited by gene 1.2 protein of bacteriophage T7. *J. Biol. Chem* 263:13549–13556.

Ibarra, B., Valpuesta, J. M., and Carrascosa, J. L. (2001). Purification and functional characterization of p16, the ATPase of the bacteriophage phi29 packaging machinery. *Nucleic Acids Res.* 29:4264–4273.

Illana, B., Blanco, L., and Salas, M. (1996). Functional characterization of the genes coding for the terminal protein and DNA polymerase from bacteriophage GA-1: Evidence for a sliding-back mechanism during protein-primed GA-1 DNA replication. *J. Mol. Biol.* 264:453–464.

Illana, B., Lázaro, J. M., Gutierrez, C., Meijer, W. J., Blanco, L., and Salas, M. (1999). Phage phi29 terminal protein residues Asn80 and Tyr82 are recognition elements of the replication origins. *J. Biol. Chem.* 274:15073–15079.

Isaksen, M. L., Dokland, T., and Lindqvist, B. H. (1993). Characterization of the capsid associating activity of bacteriophage P4’s Psu protein. *Virology* 194:647–681.

Iwashita, S., and Kanegasaki, S. (1976). Enzymic and molecular properties of base-plate parts of bacteriophage P22. *Eur. J. Biochem.* 65:87–94.

Jackson, E. N., Jackson, D. A., and Deans, R. J. (1978). EcoRI analysis of bacteriophage P22 DNA packaging. *J. Mol. Biol.* 118:365–388.

Jeruzalmi, D., and Steitz, T. A. (1998). Structure of T7 RNA polymerase complexed to the transcriptional inhibitor T7 lysozyme. *EMBO J.* 17:4101–4113.

Julien, B., Lefevre, P., and Calendar, R. (1997). The two P2 Ogr-like domains of the delta protein from bacteriophage P4 are required for activity. *Virology* 230:292–299.

Kageyama, Y., Murayama, M., Onodera, T., Yamada, S., Fukada, H., Kudou, M., Tsumoto, K., Toyama, Y., Kado, S., Kubota, K., and Takeda, S. (2009). Observation of the membrane binding activity and domain structure of gpV, which comprises the tail spike of bacteriophage P2. *Biochemistry* 48:10129–10135.

Kaiser, A. D., and Jacob, F. (1957). Recombination between related bacteriophages and the genetic control of immunity and prophage localization. *Virology* 4:509–517.

Kamtekar, S., Berman, A. J., Wang, J., Lázaro, J. M., de Vega, M., Blanco, L., Salas, M., and Steitz, T. A. (2006). The phi29 DNA polymerase:protein-primer structure suggests a model for the initiation to elongation transition. *EMBO J.* 25:1335–1343.

Katsura, I. (1983). Tail assembly and injection. In “‘Lambda II’” (R. Hendrix, J. Roberts, F. W. Stahl, and R. Weisberg, eds.), pp. 331–346. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.

Katsura, I. (1990). Mechanism of length determination in bacteriophage lambda tails. *Adv. Biophys.* 26:1–18.

Katsura, I., and Tsugita, A. (1977). Purification and characterization of the major protein and the terminator protein of the bacteriophage lambda tail. *Virology* 76:129–145.

Kędzierska, B., Lee, D. J., Węgrzyn, G., Busby, S. J., and Thomas, M. S. (2004). Role of the RNA polymerase alpha subunits in CI-dependent activation of the bacteriophage lambda pE promoter: identification of important residues and positioning of the alpha C-terminal domains. *Nucleic Acids Res.* 32:834–841.

Kędzierska, B., Szambowska, A., Herman-Antosiewicz, A., Lee, D. J., Busby, S. J., Węgrzyn, G., and Thomas, M. S. (2007). The C-terminal domain of the Escherichia coli RNA polymerase alpha subunit plays a role in the CI-dependent activation of the bacteriophage lambda pM promoter. *Nucleic Acids Res.* 35:2311–2320.

Kemp, P., Garcia, L. R., and Molineux, I. J. (2005). Changes in bacteriophage T7 virion structure at the initiation of infection. *Virology* 340:307–317.

Kemp, P., Gupta, M., and Molineux, I. J. (2004). Bacteriophage T7 DNA ejection into cells is initiated by an enzyme-like mechanism. *Mol. Microbiol.* 53:1251–1265.

Kennaway, C. K., Obarska-Kosinska, A., White, J. H., Tuszynska, I., Cooper, L. P., Bujnicki, J. M., Trinick, J., and Dryden, D. T. (2009). The structure of M.EcoKI Type I DNA methyltransferase with a DNA mimic antirestriction protein. *Nucleic Acids Res* 37:762–770.

Kennedy, W. P., Momand, J. R., and Yin, Y. W. (2007). Mechanism for de novo RNA synthesis and initiating nucleotide specificity by t7 RNA polymerase. *J. Mol. Biol.* 370:256–268.

Kihara, A., Akiyama, Y., and Ito, K. (2001). Revisiting the lysogenization control of bacteriophage lambda: Identification and characterization of a new host component. *HflD. J. Biol. Chem.* 276:13695–13700.

Kim, K.-J., Sunshine, M. G., Lindqvist, B. H., and Six, E. W. (2001). Capsid size determination in the P2–P4 bacteriophage system: suppression of sir mutations in P2’s capsid gene N by supersid mutations in P4’s external scaffold gene sid. *Virology* 283:49–58.

King, J., and Casjens, S. (1974). Catalytic head assembling protein in virus morphogenesis. *Nature* 251:112–119.

King, J., Lenk, E. V., and Botstein, D. (1973). Mechanism of head assembly and DNA encapsulation in Salmonella phage P22. II. Morphogenetic pathway. *J. Mol. Biol.* 80:697–731.

King, R. A., Anders, D. L., and Christie, G. E. (1992). Site-directed mutagenesis of an amino acid residue in the bacteriophage P2 ogr protein implicated in interaction with Escherichia coli RNA polymerase. *Mol. Microbiol.* 6:3313–3320.

Kobiler, O., Koby, S., Teff, D., Court, D., and Oppenheim, A. B. (2002). The phage lambda CII transcriptional activator carries a C-terminal domain signaling for rapid proteolysis. *Proc. Natl. Acad. Sci. USA* 99:14964–14969.

Kobiler, O., Oppenheim, A. B., and Herman, C. (2004). Recruitment of host ATP-dependent proteases by bacteriophage lambda. *J. Struct. Biol.* 146:72–78.

Kobiler, O., Rokney, A., and Oppenheim, A. B. (2007). Phage lambda CIII: A protease inhibitor regulating the lysis-lysogeny decision. *PLoS One* 2:e363.

Kochan, J., Carrascosa, J. L., and Murialdo, H. (1984). Bacteriophage lambda preconnectors. *Purification and structure. J. Mol. Biol.* 174:433–447.

Kornitzer, D., Altuvia, S., and Oppenheim, A. B. (1991). The activity of the CIII regulator of lambdoid bacteriophages resides within a 24-amino acid protein domain. *Proc. Natl. Acad. Sci. USA* 88:5217–5221.

Lander, G. C., Evilevitch, A., Jeembaeva, M., Potter, C. S., Carragher, B., and Johnson, J. E. (2008). Bacteriophage lambda stabilization by auxiliary protein gpD: Timing, location, and mechanism of attachment determined by cryo-EM. *Structure* 16:1399–1406.

Lander, G. C., Tang, L., Casjens, S. R., Gilcrease, E. B., Prevelige, P., Poliakov, A., Potter, C. S., Carragher, B., and Johnson, J. E. (2006). The structure of an infectious P22 virion shows the signal for headful DNA packaging. *Science* 312:1791–1795.

LeBowitz, J. H., and McMacken, R. (1986). The Escherichia coli dnaB replication protein is a DNA helicase. *J. Biol. Chem.* 261:4738–4748.

Lee, C. S., and Guo, P. (1995). Sequential interactions of structural proteins in phage phi 29 procapsid assembly. *J. Virol.* 69:5024–5032.

Leffers, G. G., Jr., and Gottesman, S. (1998). Lambda Xis degradation in vivo by Lon and FtsH. *J. Bacteriol.* 180:1573–1577.

Lehne, B., and Schlitt, T. (2009). Protein-protein interaction databases: Keeping up with growing interactomes. *Hum. Genom.* 3:291–297.

Lengyel, J. A., Goldstein, R. N., Marsh, M., and Calendar, R. (1974). Structure of the bacteriophage P2 tail. *Virology* 62:161–174.

Lengyel, J. A., Goldstein, R. N., Marsh, M., Sunshine, M. G., and Calendar, R. (1973). Bacteriophage P2 head morphogenesis: cleavage of the major capsid protein. *Virology* 53:1–23.

Li, M., Moyle, H., and Susskind, M. M. (1994). Target of the transcriptional activation function of phage lambda cI protein. *Science* 263:75–77.

Lin, L. (1992). “Study of Bacteriophage T7 Gene 5.9 and Gene 5.5” SUNY, Stonybrook, NY.

Linderoth, N. A., Julien, B., Flick, K. E., Calendar, R., and Christie, G. (1994). Molecular cloning and characterization of bacteriophage P2 genes R and S involved in tail completion. *Virology* 200:347–359.

Linderoth, N. A., Ziermann, R., Haggård-Ljungquist, E., Christie, G. E., and Calendar, R. (1991). Nucleotide sequence of the DNA packaging and capsid synthesis genes of bacteriophage P2. *Nucl. Acids Res.* 19:7207–7214.

Lindqvist, B. H., Deho, G., and Calendar, R. (1993). Mechanisms of genome propagation and helper exploitation by satellite phage P4. *Microbiol. Rev.* 57:683–702.

Little, J. W. (1967). An exonuclease induced by bacteriophage lambda. II. Nature of the enzymatic reaction. *J. Biol. Chem* 242:679–686.

Liu, Q., and Richardson, C. C. (1993). Gene 5.5 protein of bacteriophage T7 inhibits the nucleoid protein H-NS of *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* 90:1761–1765.

Liu, T., Renberg, S. K., and Haggard-Ljungquist, E. (1998). The E protein of satellite phage P4 acts as an anti-repressor by binding to the C protein of helper phage P2. *Mol. Microbiol.* 30:1041–1050.

Liu, X., Zhang, Q., Murata, K., Baker, M. L., Sullivan, M. B., Fu, C., Dougherty, M. T., Schmid, M. F., Osburne, M. S., Chisholm, S. W., and Chiu, W. (2010). Structural changes in a marine podovirus associated with release of its genome into Prochlorococcus. *Nat. Struct. Mol. Biol.* 17:830–836.

Liu, Y., and Haggard-Ljungquist, E. (1994). Studies of bacteriophage P2 DNA replication: localization of the cleavage site of the A protein. *Nucleic Acids Res.* 22:5204–5210.

Loeffler, J. M., Djurkovic, S., and Fischetti, V. A. (2003). Phage lytic enzyme Cpl-1 as a novel antimicrobial for pneumococcal bacteremia. *Infect. Immun.* 71:6199–6204.

Loeffler, J. M., Nelson, D., and Fischetti, V. A. (2001). Rapid killing of *Streptococcus pneumoniae* with a bacteriophage cell wall hydrolase. *Science* 294:2170–2172.

Longás, E., de Vega, M., Lázaro, J. M., and Salas, M. (2006). Functional characterization of highly processive protein-primed DNA polymerases from phages Nf and GA-1, endowed with a potent strand displacement capacity. *Nucleic Acids Res.* 34:6051–6063.

Longás, E., Villar, L., Lázaro, J. M., de Vega, M., and Salas, M. (2008). Phage phi29 and Nf terminal protein-priming domain specifies the internal template nucleotide to initiate DNA replication. *Proc. Natl. Acad. Sci. USA* 105:18290–18295.

Lopez, R., Ronda, C., Tomas, A., and Portoles, A. (1977). Properties of “diplophage”: A lipid-containing bacteriophage. *J. Virol.* 24:201–210.

Lyakhov, D. L., He, B., Zhang, X., Studier, F. W., Dunn, J. J., and McAllister, W. T. (1997). Mutant bacteriophage T7 RNA polymerases with altered termination properties. *J. Mol. Biol.* 269:28–40.

Lyakhov, D. L., He, B., Zhang, X., Studier, F. W., Dunn, J. J., and McAllister, W. T. (1998). Pausing and termination by bacteriophage T7 RNA polymerase. *J. Mol. Biol.* 280:201–213.

Makela, P. H. (1973). Glucosylation of lipopolysaccharide in Salmonella: Mutants negative for O antigen factor 1221. *J. Bacteriol.* 116:847–856.

Mallory, J. B., Alfano, C., and McMacken, R. (1990). Host virus interactions in the initiation of bacteriophage lambda DNA replication: Recruitment of *Escherichia coli* DnaB helicase by lambda P replication protein. *J. Biol. Chem.* 265:13297–13307.

Marchand, I., Nicholson, A. W., and Dreyfus, M. (2001). Bacteriophage T7 protein kinase phosphorylates RNase E and stabilizes mRNAs synthesized by T7 RNA polymerase. *Mol. Microbiol.* 42:767–776.

Marintcheva, B., Qimron, U., Yu, Y., Tabor, S., and Richardson, C. C. (2009). Mutations in the gene 5 DNA polymerase of bacteriophage T7 suppress the dominant lethal phenotype of gene 2.5 ssDNA binding protein lacking the C-terminal phenylalanine. *Mol. Microbiol* 72:869–880.

Marr, M. T., Datwyler, S. A., Meares, C. F., and Roberts, J. W. (2001). Restructuring of an RNA polymerase holoenzyme elongation complex by lambdoid phage Q proteins. *Proc. Natl. Acad. Sci. USA* 98:8972–8978.

Marr, M. T., Roberts, J. W., Brown, S. E., Klee, M., and Gussin, G. N. (2004). Interactions among CII protein, RNA polymerase and the lambda PRE promoter: Contacts between

RNA polymerase and the -35 region of PRE are identical in the presence and absence of CII protein. *Nucleic Acids Res.* **32**:1083–1090.

Marsic, N., Roje, S., Stojiljkovic, I., Salaj-Smic, E., and Trgovcevic, Z. (1993). In vivo studies on the interaction of RecBCD enzyme and lambda Gam protein. *J. Bacteriol.* **175**:4738–4743.

Martin, A. C., Blanco, L., Garcia, P., Salas, M., and Mendez, J. (1996). In vitro protein-primed initiation of pneumococcal phage Cp-1 DNA replication occurs at the third 3′ nucleotide of the linear template: A stepwise sliding-back mechanism. *J. Mol. Biol.* **260**:369–377.

Marvik, O. J., Dokland, T., Nøkling, R. H., Jacobsen, E., Larsen, T., and Lindqvist, B. H. (1995). The capsid size-determining protein Sid forms an external scaffold on phage P4 procapsids. *J. Mol. Biol.* **251**:59–75.

Marvik, O. J., Jacobsen, E., Dokland, T., and Lindqvist, B. H. (1994). Bacteriophage P2 and P4 morphogenesis: assembly precedes proteolytic processing of the capsid proteins. *Virology* **205**:51–65.

Mason, S. W., and Greenblatt, J. (1991). Assembly of transcription elongation complexes containing the N protein of phage lambda and the *Escherichia coli* elongation factors NusA, NusB, NusG, and S10. *Genes Dev.* **5**:1504–1512.

Massad, T., Skaar, K., Nilsson, H., Damberg, P., Henriksson-Peltola, P., Haggard-Ljungquist, E., Hogbom, M., and Stenmark, P. (2010). Crystal structure of the P2 C-repressor: A binder of non-palindromic direct DNA repeats. *Nucleic Acids Res.* **38**:7778–7790.

Mattis, A. N., Gumpert, R. I., and Gardner, J. F. (2008). Purification and characterization of bacteriophage P22 Xis protein. *J. Bacteriol.* **190**:5781–5796.

Maxwell, K. L., Reed, P., Zhang, R. G., Beasley, S., Walmsley, A. R., Curtis, F. A., Joachimiak, A., Edwards, A. M., and Sharples, G. J. (2005). Functional similarities between phage lambda Orf and *Escherichia coli* RecFOR in initiation of genetic exchange. *Proc. Natl. Acad. Sci. USA* **102**:11260–11265.

Maxwell, K. L., Yee, A. A., Booth, V., Arrowsmith, C. H., Gold, M., and Davidson, A. R. (2001). The solution structure of bacteriophage lambda protein W, a small morphogenic protein possessing a novel fold. *J. Mol. Biol.* **308**:9–14.

Mayer, J. E., and Schweiger, M. (1983). RNase III is positively regulated by T7 protein kinase. *J. Biol. Chem.* **258**:5340–5343.

McDonnell, M., Lain, R., and Tomaszc, A. (1975). “Diplophage”: A bacteriophage of Diplococcus pneumoniae *Virology* **63**:577–582.

Medina, E., Wieczorek, D., Medina, E. M., Yang, Q., Feiss, M., and Catalano, C. E. (2010). Assembly and maturation of the bacteriophage lambda procapsid: gpC is the viral protease. *J. Mol. Biol.* **401**:813–830.

Meijer, W. J., Horcajadas, J. A., and Salas, M. (2001a). Phi29 family of phages. *Microbiol. Mol. Biol. Rev.* **65**:261–287.

Meijer, W. J., Lewis, P. J., Errington, J., and Salas, M. (2000). Dynamic relocalization of phage phi 29 DNA during replication and the role of the viral protein p16.7. *EMBO J* **19**:4182–4190.

Meijer, W. J., Serna-Rico, A., and Salas, M. (2001b). Characterization of the bacteriophage phi29-encoded protein p16.7: A membrane protein involved in phage DNA replication. *Mol. Microbiol.* **39**:731–746.

Mencía, M., Monsalve, M., Rojo, F., and Salas, M. (1996a). Transcription activation by phage phi29 protein p4 is mediated by interaction with the alpha subunit of *Bacillus subtilis* RNA polymerase. *Proc. Natl. Acad. Sci. USA* **93**:6616–6620.

Mencía, M., Monsalve, M., Salas, M., and Rojo, F. (1996b). Transcriptional activator of phage phi 29 late promoter: mapping of residues involved in interaction with RNA polymerase and in DNA bending. *Mol. Microbiol.* **20**:273–282.

Méndez, J., Blanco, L., Esteban, J. A., Bernad, A., and Salas, M. (1992). Initiation of phi 29 DNA replication occurs at the second 3′ nucleotide of the linear template: A sliding-back mechanism for protein-primed DNA replication. *Proc. Natl. Acad. Sci. USA* **89**:9579–9583.

Méndez, J., Blanco, L., and Salas, M. (1997). Protein-primed DNA replication: A transition between two modes of priming by a unique DNA polymerase. *EMBO J.* **16**:2519–2527.

Mikawa, Y. G., Maruyama, I. N., and Brenner, S. (1996). Surface display of proteins on bacteriophage lambda heads. *J. Mol. Biol.* **262**:21–30.

Moak, M., and Molineux, I. J. (2000). Role of the Gp16 lytic transglycosylase motif in bacteriophage T7 virions at the initiation of infection. *Mol. Microbiol.* **37**:345–355.

Moak, M., and Molineux, I. J. (2004). Peptidoglycan hydrolytic activities associated with bacteriophage virions. *Mol. Microbiol.* **51**:1169–1183.

Molineux, I. J. (2006a). The T7 group. Chapter 20 In “The Bacteriophages” (R. Calendar, ed.), pp. 277–301. Oxford University Press, Oxford.

Molineux, I. J. (2006b). Fifty-three years since Hershey and Chase; much ado about pressure but which pressure is it? *Virology* **344**:221–229.

Monsalve, M., Calles, B., Mencía, M., Salas, M., and Rojo, F. (1997). Transcription activation or repression by phage psi 29 protein p4 depends on the strength of the RNA polymerase-promoter interactions. *Mol. Cell.* **1**:99–107.

Monsalve, M., Mencía, M., Rojo, F., and Salas, M. (1996a). Activation and repression of transcription at two different phage phi29 promoters are mediated by interaction of the same residues of regulatory protein p4 with RNA polymerase. *EMBO J.* **15**:383–391.

Monsalve, M., Mencía, M., Salas, M., and Rojo, F. (1996b). Protein p4 represses phage phi 29 A2c promoter by interacting with the alpha subunit of *Bacillus subtilis* RNA polymerase. *Proc. Natl. Acad. Sci. USA* **93**:8913–8918.

Mooney, P. Q., North, R., and Molineux, I. J. (1980). The role of bacteriophage T7 gene 2 protein in DNA replication. *Nucleic Acids Res.* **8**:3043–3053.

Morais, M. C., Choi, K. H., Koti, J. S., Chipman, P. R., Anderson, D. L., and Rossmann, M. G. (2005). Conservation of the capsid structure in tailed dsDNA bacteriophages: The pseudoatomic structure of phi29. *Mol. Cell.* **18**:149–159.

Morais, M. C., Kanamaru, S., Badasso, M. O., Koti, J. S., Owen, B. A., McMurray, C. T., Anderson, D. L., and Rossmann, M. G. (2003). Bacteriophage phi29 scaffolding protein gp7 before and after prohead assembly. *Nat. Struct. Biol.* **10**:572–576.

Morita, M., Tasaka, M., and Fujisawa, H. (1994). Analysis of functional domains of the packaging proteins of bacteriophage T3 by site-directed mutagenesis. *J. Mol. Biol.* **235**:248–259.

Morita, M., Tasaka, M., and Fujisawa, H. (1995). Structural and functional domains of the large subunit of the bacteriophage T3 DNA packaging enzyme: Importance of the C-terminal region in prohead binding. *J. Mol. Biol.* **245**:635–644.

Mosig, G., Yu, S., Myung, H., Haggard-Ljungquist, E., Davenport, L., Carlson, K., and Calendar, R. (1997). A novel mechanism of virus-virus interactions: Bacteriophage P2 Tin protein inhibits phage T4 DNA synthesis by poisoning the T4 single-stranded DNA binding protein, gp32. *Virology* **230**:72–81.

Mount, D. W., Harris, A. W., Fuerst, C. R., and Siminovitch, L. (1968). Mutations in bacteriophage lambda affecting particle morphogenesis. *Virology* **35**:134–149.

Muñoz-Espín, D., Daniel, R., Kawai, Y., Carballido-Lopez, R., Castilla-Llorente, V., Errington, J., Meijer, W. J., and Salas, M. (2009). The actin-like MreB cytoskeleton organizes viral DNA replication in bacteria. *Proc. Natl. Acad. Sci.*, USA.

Muñoz-Espín, D., Holguera, I., Ballesteros-Plaza, D., Carballido-Lopez, R., and Salas, M. (2010). Viral terminal protein directs early organization of phage DNA replication at the bacterial nucleoid. *Proc. Natl. Acad. Sci. USA* **107**:16548–16553.

Murialdo, H. (1979). Early intermediates in bacteriophage lambda prohead assembly. *Virology* **96**:341–367.

Murialdo, H., and Becker, A. (1978). Head morphogenesis of complex double-stranded deoxyribonucleic acid bacteriophages. *Microbiol. Rev.* **42**:529–576.

Murialdo, H., and Tzamtzis, D. (1997). Mutations of the coat protein gene of bacteriophage lambda that overcome the necessity for the Fl gene; the EFi domain. *Mol. Microbiol.* 24:341–353.

Murphy, K. C. (2000). Bacteriophage P22 Abc2 protein binds to RecC increases the 5′ strand nicking activity of RecBCD and together with lambda bet, promotes Chi-independent recombination. *J. Mol. Biol.* 296:385–401.

Murray, N. E. (2000). Type I restriction systems: Sophisticated molecular machines (a legacy of Bertani and Weigle). *MMBR* 64:412–434.

Nakai, H., and Richardson, C. C. (1990). The gene 1.2 protein of bacteriophage T7 interacts with the *Escherichia coli* dGTP triphosphohydrolase to form a GTP-binding protein. *J. Biol. Chem* 265:4411–4419.

Nechaev, S., and Severinov, K. (1999). Inhibition of *Escherichia coli* RNA polymerase by bacteriophage T7 gene 2 protein. *J. Mol. Biol.* 289:815–826.

Nemecek, D., Gilcrease, E. B., Kang, S., Prevelige, P. E., Jr., Casjens, S., and Thomas, G. J., Jr. (2007). Subunit conformations and assembly states of a DNA-translocating motor: The terminase of bacteriophage P22. *J. Mol. Biol.* 374:817–836.

Nemecek, D., Lander, G. C., Johnson, J. E., Casjens, S. R., and Thomas, G. J., Jr. (2008). Assembly architecture and DNA binding of the bacteriophage P22 terminase small subunit. *J. Mol. Biol.* 383:494–501.

Newlove, T., Konieczka, J. H., and Cordes, M. H. (2004). Secondary structure switching in Cro protein evolution. *Structure (Camb.)* 12:569–581.

Nickels, B. E., Roberts, C. W., Sun, H., Roberts, J. W., and Hochschild, A. (2002). The sigma (70) subunit of RNA polymerase is contacted by the (lambda)Q antiterminator during early elongation. *Mol. Cell.* 10:611–622.

Nilsson, A. S., and Haggård-Ljungquist, E. (2006). The P2-like bacteriophages. (R. Calendar, ed.), pp. 365-390. Oxford University Press, New York.

Nilsson, A. S., Karlsson, J. L., and Haggård-Ljungquist, E. (2004). Site-specific recombination links the evolution of P2-like coliphages and pathogenic enterobacteria. *Mol. Biol. Evol.* 21:1–13.

Nooren, I. M., Kaptein, R., Sauer, R. T., and Boelens, R. (1999). The tetramerization domain of the Mnt repressor consists of two right-handed coiled coils. *Nat. Struct. Biol.* 6:755–759.

Nuez, B., Rojo, F., and Salas, M. (1992). Phage phi 29 regulatory protein p4 stabilizes the binding of the RNA polymerase to the late promoter in a process involving direct protein-protein contacts. *Proc. Natl. Acad. Sci. USA* 89:11401–11405.

Odegrip, R., and Haggard-Ljungquist, E. (2001). The two active-site tyrosine residues of the a protein play non-equivalent roles during initiation of rolling circle replication of bacteriophage p2. *J Mol Biol* 308:147–163.

Odegrip, R., Schoen, S., Haggard-Ljungquist, E., Park, K., and Chattoraj, D. K. (2000). The interaction of bacteriophage P2 B protein with *Escherichia coli* DnaB helicase. *J. Virol.* 74:4057–4063.

Olia, A., Prevelige, P., Jr., Johnson, J., and Cingolani, G. (2011). Three-dimensional structure of a viral genome-delivery portal vertex. *Nat. Struct. Mol. Biol* in press.

Olia, A. S., Al-Bassam, J., Winn-Stapley, D. A., Joss, L., Casjens, S. R., and Cingolani, G. (2006). Binding-induced stabilization and assembly of the phage P22 tail accessory factor gp4. *J. Mol. Biol.* 363:558–576.

Olia, A. S., Bhardwaj, A., Joss, L., Casjens, S., and Cingolani, G. (2007). Role of gene 10 protein in the hierarchical assembly of the bacteriophage P22 portal vertex structure. *Biochemistry* 46:8776–8784.

Oppenheim, A. B., Kobiler, O., Stavans, J., Court, D. L., and Adhya, S. (2005). Switches in bacteriophage lambda development. *Annu. Rev. Genet.* 39:409–429.

Pani, B., Ranjan, A., and Sen, R. (2009). Interaction surface of bacteriophage P4 protein Psu required for complex formation with the transcription terminator Rho. *J. Mol. Biol.* 389:647–660.

Parker, M. H., Casjens, S., and Prevelige, P. E., Jr. (1998). Functional domains of bacteriophage P22 scaffolding protein. *J. Mol. Biol.* 281:69–79.

Parker, M. H., Stafford, W. F., 3rd, and Prevelige, P. E., Jr. (1997). Bacteriophage P22 scaffolding protein forms oligomers in solution. *J. Mol. Biol.* 268:655–665.

Parkinson, M. J., and Lilley, D. M. (1997). The junction-resolving enzyme T7 endonuclease I: Quaternary structure and interaction with DNA. *J. Mol. Biol.* 270:169–178.

Parua, P. K., Mondal, A., and Parrack, P. (2010). HflD, an *Escherichia coli* protein involved in the lambda lysis-lysogeny switch, impairs transcription activation by lambdaCII. *Arch. Biochem. Biophys.* 493:175–183.

Pastrana, R., Lazaro, J. M., Blanco, L., Garcia, J. A., Mendez, E., and Salas, M. (1985). Overproduction and purification of protein P6 of *Bacillus subtilis* phage phi 29: Role in the initiation of DNA replication. *Nucleic Acids Res.* 13:3083–3100.

Pedulla, M. L., Ford, M. E., Karthikeyan, T., Houtz, J. M., Hendrix, R. W., Hatfull, G. F., Poteete, A. R., Gilcrease, E. B., Winn-Stapley, D. A., and Casjens, S. R. (2003). Corrected sequence of the bacteriophage p22 genome. *J. Bacteriol.* 185:1475–1477.

Pell, L. G., Liu, A., Edmonds, L., Donaldson, L. W., Howell, P. L., and Davidson, A. R. (2009). The X-ray crystal structure of the phage lambda tail terminator protein reveals the biologically relevant hexameric ring structure and demonstrates a conserved mechanism of tail termination among diverse long-tailed phages. *J. Mol. Biol.* 389:938–951.

Poteete, A. R., and Botstein, D. (1979). Purification and properties of proteins essential to DNA encapsulation by phage P22. *Virology* 95:565–573.

Poteete, A. R., and Fenton, A. C. (1983). DNA-binding properties of the Erf protein of bacteriophage P22. *J. Mol. Biol.* 163:257–275.

Poteete, A. R., and King, J. (1977). Functions of two new genes in Salmonella phage P22 assembly. *Virology* 76:725–739.

Poteete, A. R., Sauer, R. T., and Hendrix, R. W. (1983). Domain structure and quaternary organization of the bacteriophage P22 Erf protein. *J. Mol. Biol.* 171:401–418.

Potrykus, K., Baranska, S., Wegrzyn, A., and Wegrzyn, G. (2002). Composition of the lambda plasmid heritable replication complex. *Biochem. J.* 364:857–862.

Prell, H. H., and Harvey, A. M. (1983). P22 antirepressor protein prevents *in vivo* recA-dependent proteolysis of P22 repressor. *Mol. Gen. Genet.* 190:427–431.

Prevelige, P. E., Jr., Thomas, D., and King, J. (1988). Scaffolding protein regulates the polymerization of P22 coat subunits into icosahedral shells *in vitro*. *J. Mol. Biol.* 202:743–757.

Pruss, G. J., Wang, J. C., and Calendar, R. (1975). In vitro packaging of covalently-closed circular monomers of bacteriophage DNA. *J. Mol. Biol.* 98:465–478.

Ptashne, M. (2004). A Genetic Switch: Phage Lambda Revisited. Third Ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

Ptashne, M. (2006). A Genetic Switch. Phage Lambda Revisited. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

Putnam, C. D., Shroyer, M. J., Lundquist, A. J., Mol, C. D., Arvai, A. S., Mosbaugh, D. W., and Tainer, J. A. (1999). Protein mimicry of DNA from crystal structures of the uracil-DNA glycosylase inhibitor protein and its complex with *Escherichia coli* uracil-DNA glycosylase. *J. Mol. Biol.* 287:331–346.

Qimron, U., Marintcheva, B., Tabor, S., and Richardson, C. C. (2006). Genomewide screens for *Escherichia coli* genes affecting growth of T7 bacteriophage. *Proc. Natl. Acad. Sci. USA* 103:19039–19044.

Radding, C. M., Rosenzweig, J., Richards, J., and Cassuto, E. (1971). Appendix: Separation and characterization of exonuclease, β protein and a complex of both. *J. Biol. Chem.* 146:2510–2512.

Rahmsdorf, H. J., Pai, S. H., Ponta, H., Herrlich, P., Roskoski, R., Jr., Schweiger, M., and Studier, F. W. (1974). Protein kinase induction in *Escherichia coli* by bacteriophage T7. *Proc. Natl. Acad. Sci. USA* 71:586–589.

Rajagopala, S. V., Casjens, S., and Uetz, P. (2011). The protein interaction map of bacteriophage lambda. *BMC Microbiol.* 11:213.

Ranade, K., and Poteete, A. R. (1993a). Superinfection exclusion (sieB) genes of bacteriophages P22 and lambda. *J. Bacteriol.* 175:4712–4718.

Ranade, K., and Poteete, A. R. (1993b). A switch in translation mediated by an antisense RNA. *Genes Dev.* 7:1498–1507.

Ray, P., and Murialdo, H. (1975). The role of gene Nu3 in bacteriophage lambda head morphogenesis. *Virology* 64:247–263.

Rennell, D., and Poteete, A. R. (1985). Phage P22 lysis genes: Nucleotide sequences and functional relationships with T4 and lambda genes. *Virology* 143:280–289.

Rennell, D., and Poteete, A. R. (1989). Genetic analysis of bacteriophage P22 lysozyme structure. *Genetics* 123:431–440.

Rezende, L. F., Hollis, T., Ellenberger, T., and Richardson, C. C. (2002). Essential amino acid residues in the single-stranded DNA-binding protein of bacteriophage T7: Identification of the dimer interface. *J. Biol. Chem.* 277:50643–50653.

Rishovd, S., and Lindqvist, B. H. (1992). Bacteriophage P2 and P4 morphogenesis: protein processing and capsid size determination. *Virology* 187:548–554.

Rishovd, S., Marvik, O. J., Jacobsen, E., and Lindqvist, B. H. (1994). Bacteriophage P2 and P4 morphogenesis: Identification and characterization of the portal protein. *Virology* 200:744–751.

Roberts, C. W., and Roberts, J. W. (1996). Base-specific recognition of the nontemplate strand of promoter DNA by *E. coli* RNA polymerase. *Cell* 86:495–501.

Roberts, J. W. (1969). Termination factor for RNA synthesis. *Nature* 224:1168–1174.

Roberts, J. W., Yarnell, W., Bartlett, E., Guo, J., Marr, M., Ko, D. C., Sun, H., and Roberts, C. W. (1998). Antitermination by bacteriophage lambda Q protein. *Cold Spring Harb. Symp. Quant. Biol.* 63:319–325.

Robertson, E. S., Aggison, L. A., and Nicholson, A. W. (1994). Phosphorylation of elongation factor G and ribosomal protein S6 in bacteriophage T7-infected *Escherichia coli*. *Mol. Microbiol.* 11:1045–1057.

Robertson, E. S., and Nicholson, A. W. (1990). Protein kinase of bacteriophage T7 induces the phosphorylation of only a small number of proteins in the infected cell. *Virology* 175:525–534.

Robertson, E. S., and Nicholson, A. W. (1992). Phosphorylation of *Escherichia coli* translation initiation factors by the bacteriophage T7 protein kinase. *Biochemistry* 31:4822–4827.

Rodriguez-Cerrato, V., Garcia, P., Del Prado, G., Garcia, E., Gracia, M., Huelves, L., Ponte, C., Lopez, R., and Soriano, F. (2007). In vitro interactions of LytA, the major pneumococcal autolysin, with two bacteriophage lytic enzymes (Cpl-1 and Pal), cefotaxime and moxifloxacin against antibiotic-susceptible and -resistant *Streptococcus pneumoniae* strains. *J. Antimicrob. Chemother.* 60:1159–1162.

Rojo, F., Mencia, M., Monsalve, M., and Salas, M. (1998). Transcription activation and repression by interaction of a regulator with the alpha subunit of RNA polymerase: The model of phage phi 29 protein p4. *Prog. Nucleic Acid Res. Mol. Biol.* 60:29–46.

Rojo, F., and Salas, M. (1991). A DNA curvature can substitute phage phi 29 regulatory protein p4 when acting as a transcriptional repressor. *EMBO J.* 10:3429–3438.

Sabri, M., Häuser, R., Ouellette, M., Liu, J., Dehbi, M., Moeck, G., Garcia, E., Titz, B., Uetz, P., and Moineau, S. (2011). Genome annotation and intraviral interactome for the Streptococcus pneumoniae virulent phage Dp-1. *J. Bacteriol.* 193:551–562.

Saito, H., and Richardson, C. C. (1981). Processing of mRNA by ribonuclease III regulates expression of gene 1.2 of bacteriophage T7. *Cell* 27:533–542.

Salas, M. (1991). Protein-priming of DNA replication. *Annu. Rev. Biochem.* 60:39–71.

Sam, M. D., Papagiannis, C. V., Connolly, K. M., Corselli, L., Iwahara, J., Lee, J., Phillips, M., Wojciak, J. M., Johnson, R. C., and Clubb, R. T. (2002). Regulation of directionality in bacteriophage lambda site-specific recombination: structure of the Xis protein. *J. Mol. Biol.* 324:791–805.

Sam, M. D., Cascio, D., Johnson, R. C., and Clubb, R. T. (2004). Crystal structure of the excisionase-DNA complex from bacteriophage lambda. *J. Mol. Biol.* 338:229–240.

Sanger, F., Air, G. M., Barrell, B. G., Brown, N. L., Coulson, A. R., Fiddes, C. A., Hutchison, C. A., Slocombe, P. M., and Smith, M. (1977). Nucleotide sequence of bacteriophage phi X174 DNA. *Nature* 265:687–695.

Sauer, B., Ow, D., Ling, L., and Calendar, R. (1981). Mutants of satellite bacteriophage P4 that are defective in the suppression of transcriptional polarity. *J. Mol. Biol.* 145:29–46.

Savalia, D., Robins, W., Nechaev, S., Molineux, I., and Severinov, K. (2010). The role of the T7 Gp2 inhibitor of host RNA polymerase in phage development. *J. Mol. Biol.* 402:118–126.

Savva, C. G., Dewey, J. S., Deaton, J., White, R. L., Struck, D. K., Holzenburg, A., and Young, R. (2008). The holin of bacteriophage lambda forms rings with large diameter. *Mol. Microbiol.* 69:784–793.

Schmitt, C. K., and Molineux, I. J. (1991). Expression of gene 1.2 and gene 10 of bacteriophage T7 is lethal to F plasmid-containing *Escherichia coli*. *J. Bacteriol.* 173:1536–1543.

Schulz, E. C., Dickmanns, A., Urlaub, H., Schmitt, A., Muhlenhoff, M., Stummeyer, K., Schwarzer, D., Gerardy-Schahn, R., and Ficner, R. (2010). Crystal structure of an intramolecular chaperone mediating triple-beta-helix folding. *Nat. Struct. Mol. Biol.* 17:210–215.

Semerjian, A. V., Malloy, D. C., and Poteete, A. R. (1989). Genetic structure of the bacteriophage P22 PL operon. *J. Mol. Biol.* 207:1–13.

Serna-Rico, A., Munoz-Espin, D., Villar, L., Salas, M., and Meijer, W. J. (2003). The integral membrane protein p16.7 organizes *in vivo* phi29 DNA replication through interaction with both the terminal protein and ssDNA. *EMBO J* 22:2297–2306.

Serna-Rico, A., Salas, M., and Meijer, W. J. (2002). The *Bacillus subtilis* phage phi 29 protein p16.7, involved in phi 29 DNA replication, is a membrane-localized single-stranded DNA-binding protein. *J. Biol. Chem* 277:6733–6742.

Serrano, M., Gutierrez, C., Freire, R., Bravo, A., Salas, M., and Hermoso, J. M. (1994). Phage phi 29 protein p6: A viral histone-like protein. *Biochimie* 76:981–991.

Serrano-Heras, G., Bravo, A., and Salas, M. (2008). Phage phi29 protein p56 prevents viral DNA replication impairment caused by uracil excision activity of uracil-DNA glycosylase. *Proc. Natl. Acad. Sci. USA* 105:19044–19049.

Serrano-Heras, G., Ruiz-Maso, J. A., del Solar, G., Espinosa, M., Bravo, A., and Salas, M. (2007). Protein p56 from the *Bacillus subtilis* phage phi29 inhibits DNA-binding ability of uracil-DNA glycosylase. *Nucleic Acids Res.* 35:5393–5401.

Serrano-Heras, G., Salas, M., and Bravo, A. (2006). A uracil-DNA glycosylase inhibitor encoded by a non-uracil containing viral DNA. *J. Biol. Chem.* 281:7068–7074.

Severinova, E., and Severinov, K. (2006). Localization of the *Escherichia coli* RNA polymerase beta' subunit residue phosphorylated by bacteriophage T7 kinase Gp0.7. *J. Bacteriol* 188:3470–3476.

Sevilla-Sierra, P., Otting, G., and Wuthrich, K. (1994). Determination of the nuclear magnetic resonance structure of the DNA-binding domain of the P22 c2 repressor (1 to 76) in solution and comparison with the DNA-binding domain of the 434 repressor. *J. Mol. Biol.* 235:1003–1020.

Shotland, Y., Shifrin, A., Ziv, T., Teff, D., Koby, S., Kobiler, O., and Oppenheim, A. B. (2000). Proteolysis of bacteriophage lambda CII by Escherichia coli FtsH (HflB). *J. Bacteriol.* 182:3111–3116.

Simon, M. N., and Studier, F. W. (1973). Physical mapping of the early region of bacteriophage T7 DNA. *J. Mol. Biol.* 79:249–265.

Simpson, A. A., Leiman, P. G., Tao, Y., He, Y., Badasso, M. O., Jardine, P. J., Anderson, D. L., and Rossmann, M. G. (2001). Structure determination of the head-tail connector of bacteriophage phi29. *Acta Crystallogr. D Biol. Crystallogr.* 57:1260–1269.

Simpson, A. A., Tao, Y., Leiman, P. G., Badasso, M. O., He, Y., Jardine, P. J., Olson, N. H., Morais, M. C., Grimes, S., Anderson, D. L., Baker, T. S., and Rossmann, M. G. (2000). Structure of the bacteriophage phi29 DNA packaging motor. *Nature* 408:745–750.

Singleton, M. R., Sawaya, M. R., Ellenberger, T., and Wigley, D. B. (2000). Crystal structure of T7 gene 4 ring helicase indicates a mechanism for sequential hydrolysis of nucleotides. *Cell* 101:589–600.

Sippy, J., and Feiss, M. (1992). Analysis of a mutation affecting the specificity domain for prohead binding of the bacteriophage lambda terminase. *J. Bacteriol.* 174:850–856.

Six, E. W. (1975). The helper dependence of satellite bacteriophage P4: which gene functions of bacteriophage P2 are needed by P4?. *Virology* 67:249–263.

Six, E. W., Sunshine, M. G., Williams, J., Haggård-Ljungquist, E., and Lindqvist, B. H. (1991). Morphopoietic switch mutations of bacteriophage P2. *Virology* 182:34–46.

Souza, L., Calendar, R., Six, E., and Lindqvist, B. (1977). A transactivation mutant of satellite phage P4. *Virology* 81:81–90.

Stano, N. M., and Patel, S. S. (2004). T7 lysozyme represses T7 RNA polymerase transcription by destabilizing the open complex during initiation. *J. Biol. Chem.* 279:16136–16143.

Steinbacher, S., Miller, S., Baxa, U., Budisa, N., Weintraub, A., Seckler, R., and Huber, R. (1997). Phage P22 tailspike protein: Crystal structure of the head-binding domain at 2.3 Å, fully refined structure of the endorhamnosidase at 1.56 Å resolution, and the molecular basis of O-antigen recognition and cleavage. *J. Mol. Biol.* 267:865–880.

Steinbacher, S., Seckler, R., Miller, S., Steipe, B., Huber, R., and Reinemer, P. (1994). Crystal structure of P22 tailspike protein: Interdigitated subunits in a thermostable trimer. *Science* 265:383–386.

Stellberger, T., Häuser, R., Baiker, A., Pothineni, V. R., Haas, J., and Uetz, P. (2010). Improving the yeast two-hybrid system with permuted fusions proteins: The Varicella Zoster Virus interactome. *Proteome Sci.* 8:8.

Stephanou, A. S., Roberts, G. A., Cooper, L. P., Clarke, D. J., Thomson, A. R., MacKay, C. L., Nutley, M., Cooper, A., and Dryden, D. T. (2009). Dissection of the DNA mimicry of the bacteriophage T7 Ocr protein using chemical modification. *J. Mol. Biol.* 391:565–576.

Sternberg, N., and Hoess, R. H. (1995). Display of peptides and proteins on the surface of bacteriophage lambda. *Proc. Natl. Acad. Sci. USA* 92:1609–1613.

Sternberg, N., and Weisberg, R. (1977). Packaging of coliphage lambda DNA. II. The role of the gene D protein. *J. Mol. Biol.* 117:733–759.

Steven, A. C., Trus, B. L., Maizel, J. V., Unser, M., Parry, D. A., Wall, J. S., Hainfeld, J. F., and Studier, F. W. (1988). Molecular substructure of a viral receptor-recognition protein. The gp17 tail-fiber of bacteriophage T7. *J. Mol. Biol.* 200:351–365.

Stone, J. C., and Miller, R. C., Jr. (1985). Spontaneous temperature-sensitive mutations in bacteriophage T7. *J. Virol.* 54:886–888.

Studier, F. W. (1975). Gene 0.3 of bacteriophage T7 acts to overcome the DNA restriction system of the host. *J. Mol. Biol.* 94:283–295.

Studier, F. W., and Movva, N. R. (1976). SAMase gene of bacteriophage T3 is responsible for overcoming host restriction. *J. Virol.* 19:136–145.

Studier, F. W. (1981). Identification and mapping of five new genes in bacteriophage T7. *J. Mol. Biol.* 153:493–502.

Summer, E. J., Berry, J., Tran, T. A., Niu, L., Struck, D. K., and Young, R. (2007). Rz/Rz1 lysis gene equivalents in phages of Gram-negative hosts. *J. Mol. Biol.* 373:1098–1112.

Sunshine, M., Usher, D., and Calendar, R. (1975). Interaction of P2 bacteriophage with the dnaB gene of Escherichia coli. *J. Virol.* 16:284–289.

Sunshine, M. G., and Sauer, B. (1975). A bacterial mutation blocking P2 phage late gene expression. *Proc. Natl. Acad. Sci. USA* 72:2770–2774.

Susskind, M. M., and Botstein, D. (1975). Mechanism of action of Salmonella phage P22 antirepressor. *J. Mol. Biol.* 98:413–424.

Susskind, M. M., and Botstein, D. (1978). Molecular genetics of bacteriophage P22. *Microbiol. Rev.* 42:385–413.

Szambowska, A., Pierechod, M., Węgrzyn, G., and Glinkowska, M. (2010). Coupling of transcription and replication machineries in λ DNA replication initiation: Evidence for direct interaction of Escherichia coli RNA polymerase and the λ O protein. *Nucleic Acids Res.* 39:168–177.

Tahirov, T. H., Temiakov, D., Anikin, M., Patlan, V., McAllister, W. T., Vassylyev, D. G., and Yokoyama, S. (2002). Structure of a T7 RNA polymerase elongation complex at 2.9 Å resolution. *Nature* 420:43–50.

Takahashi, I., and Marmur, J. (1963). Replacement of thymidylic acid by deoxyuridylic acid in the deoxyribonucleic acid of a transducing phage for Bacillus subtilis. *Nature* 197:794–795.

Tang, J., Lander, G., Olia, A., Li, R., Casjens, S., Prevelige, P., Cingolani, G., Baker, T., and Johnson, J. (2011). Peering down the barrel of a bacteriophage portal: The genome packaging and release valve in P22. *Structure* 19:496–502.

Tang, L., Gilcrease, E. B., Casjens, S. R., and Johnson, J. E. (2006). Highly discriminatory binding of capsid-cementing proteins in bacteriophage L. *Structure* 14:837–845.

Tao, Y., Olson, N. H., Xu, W., Anderson, D. L., Rossmann, M. G., and Baker, T. S. (1998). Assembly of a tailed bacterial virus and its genome release studied in three dimensions. *Cell* 95:431–437.

Temple, L., Forsburg, S., Calendar, R., and Christie, G. (1991). Nucleotide sequence of the genes encoding the major tail sheath and tail tube proteins of bacteriophage P2. *Virology* 181:353–358.

Tokuno, S. I., and Gough, M. (1976). UV sensitivity of a nonrepressor regulatory protein of bacteriophage P22. *J. Virol.* 18:65–70.

Tran, N. Q., Lee, S. J., Richardson, C. C., and Tabor, S. (2010). A novel nucleotide kinase encoded by gene 1.7 of bacteriophage T7. *Mol. Microbiol* 77:492–504.

Tsui, L., and Hendrix, R. W. (1980). Head-tail connector of bacteriophage lambda. *J. Mol. Biol.* 142:419–438.

Tsui, L. C., and Hendrix, R. W. (1983). Proteolytic processing of phage lambda tail protein gpH: Timing of the cleavage. *Virology* 125:257–264.

Tye, B. K., Huberman, J. A., and Botstein, D. (1974). Non-random circular permutation of phage P22 DNA. *J. Mol. Biol.* 85:501–528.

Uetz, P., Rajagopala, S. V., Dong, Y. A., and Haas, J. (2004). From ORFeomes to protein interaction maps in viruses. *Genome Res.* 14:2029–2033.

Valpuesta, J. M., Fujisawa, H., Marco, S., Carazo, J. M., and Carrascosa, J. L. (1992). Three-dimensional structure of T3 connector purified from overexpressing bacteria. *J. Mol. Biol.* 224:103–112.

Valpuesta, J. M., Sousa, N., Barthelemy, I., Fernandez, J. J., Fujisawa, H., Ibarra, B., and Carrascosa, J. L. (2000). Structural analysis of the bacteriophage T3 head-to-tail connector. *Journal of structural biology* 131:146–155.

van der Poll, T., and Opal, S. M. (2009). Pathogenesis, treatment, and prevention of pneumococcal pneumonia. *Lancet* 374:1543–1556.

Vander Byl, C., and Kropinski, A. M. (2000). Sequence of the genome of Salmonella bacteriophage P22. *J. Bacteriol.* 182:6472–6481.

Vershon, A. K., Liao, S. M., McClure, W. R., and Sauer, R. T. (1987a). Bacteriophage P22 Mnt repressor: DNA binding and effects on transcription *in vitro*. *J. Mol. Biol.* 195:311–322.

Vershon, A. K., Liao, S. M., McClure, W. R., and Sauer, R. T. (1987b). Interaction of the bacteriophage P22 Arc repressor with operator DNA. *J. Mol. Biol.* 195:323–331.

Villafane, R., Zayas, M., Gilcrease, E. B., Kropinski, A. M., and Casjens, S. R. (2008). Genomic analysis of bacteriophage epsilon 34 of *Salmonella enterica* serovar Anatum (15+). *BMC Microbiol.* 8:227.

Wang, J., Hofnung, M., and Charbit, A. (2000a). The C-terminal portion of the tail fiber protein of bacteriophage lambda is responsible for binding to LamB, its receptor at the surface of *Escherichia coli* K-12. *J. Bacteriol.* 182:508–512.

Wang, S., Chang, J. R., and Dokland, T. (2006). Assembly of bacteriophage P2 and P4 procapsids with internal scaffolding protein. *Virology* 348:133–140.

Wang, S., Palasingam, P., Nøkling, R. H., Lindqvist, B. H., and Dokland, T. (2000). *In vitro* assembly of bacteriophage P4 procapsids from purified capsid and scaffolding proteins. *Virology* 275:133–144.

Wang, W. F., Cheng, X., and Molineux, I. J. (1999). Isolation and identification of fxsA, an *Escherichia coli* gene that can suppress F exclusion of bacteriophage T7. *J. Mol. Biol.* 292:485–499.

Warren, D., Sam, M. D., Manley, K., Sarkar, D., Lee, S. Y., Abbani, M., Wojciak, J. M., Clubb, R. T., and Landy, A. (2003). Identification of the lambda integrase surface that interacts with Xis reveals a residue that is also critical for Int dimer formation. *Proc. Natl. Acad. Sci. USA* 100:8176–8181.

Węgrzyn, A., Taylor, K., and Węgrzyn, G. (1996). The cbpA chaperone gene function compensates for dnaJ in lambda plasmid replication during amino acid starvation of *Escherichia coli*. *J. Bacteriol.* 178:5847–5849.

Węgrzyn, A., and Węgrzyn, G. (2001). Inheritance of the replication complex: A unique or common phenomenon in the control of DNA replication? *Arch. Microbiol.* 175:86–93.

Węgrzyn, G., and Węgrzyn, A. (2002). Stress responses and replication of plasmids in bacterial cells. *Microb. Cell Fact.* 1:2.

Weigele, P. R., Sampson, L., Winn-Stapley, D., and Casjens, S. R. (2005). Molecular genetics of bacteriophage P22 scaffolding protein’s functional domains. *J. Mol. Biol.* 348:831–844.

Werts, C., Michel, V., Hofnung, M., and Charbit, A. (1994). Adsorption of bacteriophage lambda on the LamB protein of *Escherichia coli* K-12: point mutations in gene J of lambda responsible for extended host range. *J. Bacteriol.* 176:941–947.

White, J. H., and Richardson, C. C. (1987). Gene 18 protein of bacteriophage T7: Overproduction, purification, and characterization. *J. Biol. Chem.* 262:8845–8850.

Wickner, S. (1984). DNA-dependent ATPase activity associated with phage P22 gene 12 protein. *J. Biol. Chem.* 259:14038–14043.

Wickner, S. H., and Zahn, K. (1986). Characterization of the DNA binding domain of bacteriophage lambda O protein. *J. Biol. Chem.* 261:7537–7543.

Wood, L. F., Tszine, N. Y., and Christie, G. E. (1997). Activation of P2 late transcription by P2 Ogr protein requires a discrete contact site on the C terminus of the alpha subunit of *Escherichia coli* RNA polymerase. *J. Mol. Biol.* 274:1–7.

Xiang, Y., Leiman, P. G., Li, L., Grimes, S., Anderson, D. L., and Rossmann, M. G. (2009). Crystallographic insights into the autocatalytic assembly mechanism of a bacteriophage tail spike. *Mol. Cell.* 34:375–386.

Xiang, Y., Morais, M. C., Cohen, D. N., Bowman, V. D., Anderson, D. L., and Rossmann, M. G. (2008). Crystal and cryoEM structural studies of a cell wall degrading enzyme in the bacteriophage phi29 tail. *Proc. Natl. Acad. Sci. USA* 105:9552–9557.

Xu, J., Hendrix, R. W., and Duda, R. L. (2004). Conserved translational frameshift in dsDNA bacteriophage tail assembly genes. *Mol. Cell*. **16**:11–21.

Yang, F., Forrer, P., Dauter, Z., Conway, J. F., Cheng, N., Cerritelli, M. E., Steven, A. C., Pluckthun, A., and Wlodawer, A. (2000). Novel fold and capsid-binding properties of the lambda-phage display platform protein gpD. *Nat. Struct. Biol.* **7**:230–237.

Yin, Y. W., and Steitz, T. A. (2002). Structural basis for the transition from initiation to elongation transcription in T7 RNA polymerase. *Science* **298**:1387–1395.

Yin, Y. W., and Steitz, T. A. (2004). The structural mechanism of translocation and helicase activity in T7 RNA polymerase. *Cell* **116**:393–404.

Yoshikawa, H., Elder, J. H., and Ito, J. (1986). Comparative studies on the small *Bacillus* bacteriophages. *J. Gen. Appl. Microbiol.* **155**:392–401.

Youderian, P., and Susskind, M. M. (1980). Identification of the products of bacteriophage P22 genes, including a new late gene. *Virology* **107**:258–269.

Yura, K., Yamaguchi, A., and Go, M. (2006). Coverage of whole proteome by structural genomics observed through protein homology modeling database. *J. Struct. Funct. Genom.* **7**:65–76.

Zahn, K., and Blattner, F. R. (1985). Binding and bending of the lambda replication origin by the phage O protein. *EMBO J.* **4**:3605–3616.

Zahn, K., and Blattner, F. R. (1987). Direct evidence for DNA bending at the lambda replication origin. *Science* **236**:416–422.

Zahn, K., and Landy, A. (1996). Modulation of lambda integrase synthesis by rare arginine tRNA. *Mol. Microbiol.* **21**:69–76.

Zhang, X., and Studier, F. W. (1995). Isolation of transcriptionally active mutants of T7 RNA polymerase that do not support phage growth. *J. Mol. Biol.* **250**:156–168.

Zhang, X., and Studier, F. W. (1997). Mechanism of inhibition of bacteriophage T7 RNA polymerase by T7 lysozyme. *J. Mol. Biol.* **269**:10–27.

Zhang, X., and Studier, F. W. (2004). Multiple roles of T7 RNA polymerase and T7 lysozyme during bacteriophage T7 infection. *J. Mol. Biol.* **340**:707–730.

Ziermann, R., and Calendar, R. (1990). Characterization of the cos sites of bacteriophages P2 and P4. *Gene* **96**:9–15.

Zillig, W., Fujiki, H., Blum, W., Janekovic, D., Schweiger, M., Rahmsdorf, H., Ponta, H., and Hirsch-Kauffmann, M. (1975). *In vivo* and *in vitro* phosphorylation of DNA-dependent RNA polymerase of *Escherichia coli* by bacteriophage-T7-induced protein kinase. *Proc. Natl. Acad. Sci. USA* **72**:2506–2510.

Zinder, N. D., and Lederberg, J. (1952). Genetic exchange in *Salmonella*. *J. Bacteriol.* **64**:679–699.

Żylicz, M., Ang, D., Liberek, K., and Georgopoulos, C. (1989). Initiation of lambda DNA replication with purified host- and bacteriophage-encoded proteins: The role of the dnaK, dnaJ and grpE heat shock proteins. *EMBO J.* **8**:1601–1608.
