
FEBS Letters 586 (2012) 2563–2569

journal homepage: www.FEBSLetters.org

# Functional analysis of an indole-diterpene gene cluster for lolitrem B biosynthesis in the grass endosymbiont *Epichloë festucae*

Sanjay Saikia ${}^{a,1}$, Daigo Takemoto ${}^{a,b,2}$, Brian A. Tapper ${}^{c}$, Geoff A. Lane ${}^{c}$, Karl Fraser ${}^{c}$, Barry Scott ${}^{a,b,*}$

${}^{a}$ Institute of Molecular Biosciences, Massey University, Private Bag 11222, Palmerston North, New Zealand  
${}^{b}$ Bioprotection Research Centre, Massey University, Private Bag 11222, Palmerston North, New Zealand  
${}^{c}$ AgResearch, Grasslands Research Centre, Tennent Drive, Private Bag 11008, Palmerston North, New Zealand

## ARTICLE INFO

**Article history:**  
Received 28 May 2012  
Accepted 20 June 2012  
Available online 29 June 2012  

Edited by Ulf-Ingo Flügge  

**Keywords:**  
Indole-diterpene  
Lolitrem B  
Metabolic grid  
*ltm* gene  
Endophyte  
Symbiosis  
*Epichloë festucae*

## ABSTRACT

*Epichloë festucae* Fl1 in association with *Lolium perenne* synthesizes a diverse range of indole-diterpene bioprotective metabolites, including lolitrem B, a potent tremorgen. The *ltm* genes responsible for the synthesis of these metabolites are organized in three clusters at a single sub-telomeric locus in the genome of *E. festucae*. Here we resolve the genetic basis for the remarkable indole-diterpene diversity observed *in planta* by analyzing products that accumulate in associations containing *ltm* deletion mutants of *E. festucae* and in cells of *Penicillium paxilli* containing copies of these genes under the control of a *P. paxilli* biosynthetic gene promoter. We propose a biosynthetic scheme to account for this metabolic diversity.

© 2012 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

## 1. Introduction

*Epichloë festucae* is a symbiotic fungus (Clavicipitaceae, Ascomycota) that systemically colonizes the intercellular spaces of vegetative and reproductive tissues of temperate grasses of the subfamily Poöideae [1–3]. Previous studies have established that *E. festucae* and related endophytes confer bioprotective benefits to their host plants [4–6], the best known being protection from insect and mammalian herbivory [2,3].

Indole-diterpenes are one important class of bioprotective metabolites synthesized by *E. festucae* in association with *Festuca* and *Lolium* spp. [7]. The most abundant product in *L. perenne* associations is the lolitremane, lolitrem B [8], a potent tremorgenic mycotoxin responsible for the mammalian syndrome, known as ‘ryegrass staggers’ [5]. The structural similarity between lolitrem B and paxilline, a paspalane that is an abundant metabolite of *Penicillium paxilli* (Fig. 1A), suggests that the more complex indole-diterpenes found in epichloae endophytes are derived from either paxilline or proximate precursors such as paspaline [9]. Molecular cloning and sequence analysis of the lolitrem B (*ltm*) biosynthesis genes from *E. festucae* revealed a complex locus comprised of ten genes, organized in three clusters, interspersed with transposon relics [7,10] (Fig. 1B).

The first cluster comprises three genes, *ltmG*, *ltmM* and *ltmK*, two of which have been shown to be functional orthologues of *paxG* and *paxM*, genes that encode enzymes required for early steps in paxilline biosynthesis [10]. Recently, a fourth putative gene, annotated as *ltmS*, has been identified in this cluster [11]. The second *ltm* gene cluster comprises five genes, *ltmP*, *ltmQ*, *ltmF*, *ltmC* and *ltmB*, four of which appear to be homologues of *paxP* (P450 monooxygenase), *paxQ* (P450 monooxygenase), *paxC* (prenyl transferase) and *paxB* (unknown function) [12–14] (Fig. 1B). The third *ltm* gene cluster comprises two genes, *ltmE* and *ltmJ*, which are predicted to encode a multifunctional prenyl transferase and a cytochrome P450 monooxygenase, respectively (Fig. 1B), and appear to be required for the biosynthesis of lolitrem B, and lolitremanes in general [15].

Although *E. festucae ltmP* and *ltmQ* appear to be P450 monooxygenase homologues of *P. paxilli paxP* and *paxQ*, respectively, there are differences with respect to the core structures of the major indole-diterpenes that accumulate between the two fungi. In *P. paxilli*, PaxP is required to convert paspaline to 13-desoxypaxilline in a series of oxidation steps while PaxQ is required to convert

---

* Corresponding author at: Institute of Molecular Biosciences, Massey University, Palmerston North, New Zealand. Fax: +64 6 3502267.  
E-mail address: d.b.scott@massey.ac.nz (B. Scott).  
${}^{1}$ Current address: Michael Smith Laboratories, University of British Columbia, Vancouver, Canada V6T1Z4.  
${}^{2}$ Current address: Plant Pathology Laboratory, Graduate School of Bioagricultural Sciences, Nagoya University, Chikusa, Nagoya 464-8601, Japan.

0014-5793/$36.00 © 2012 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.  
http://dx.doi.org/10.1016/j.febslet.2012.06.035

2564 S. Saikia et al./FEBS Letters 586 (2012) 2563–2569

A

![image](https://i.imgur.com/1234567.png)

paxilline

B

pax cluster

![image](https://i.imgur.com/8901234.png)

ltm cluster

![image](https://i.imgur.com/5678901.png)

Fig. 1. Indole-diterpene structures and biosynthetic gene clusters. (A) Structure of paxilline and lolitrem B. Ring names and positions of carbon atoms described in the text are labeled. (B) A physical map of *P. paxilli* pax cluster (top) and the *E. festucae* ltm cluster (bottom). Homologous genes that are found in both clusters are shown by green arrows and the genes that are unique to each cluster are shown by blue or pink arrows.

13-desoxypaxilline to paxilline in a single step [13]. Consequently, *P. paxilli* accumulates 11,12-ene (see Fig. 1A for numbering system) and 10-β-hydroxyl paspalanes (e.g., paxilline). However, *E. festucae* accumulates 11,12-epoxides and 10-α-hydroxyl derivatives (e.g., lolitrem E and lolitrem B). Hence, there are likely to be functional differences between these homologues accounting for the differences in stereochemistry and epoxidation.

The objective of this study was to analyze the function of *ltmE*, *ltmF*, *ltmJ* and *ltmK*, which are unique to the *ltm* cluster as well as *ltmP* and *ltmQ*, homologues of the functionally characterized genes, *paxP* and *paxQ*, from *P. paxilli*, by chemical analyses of the paspalanes and lolitremanes that accumulate in *E. festucae* mutant symbiota and in *P. paxilli* containing *E. festucae* genes and fed putative substrates.

## 2. Materials and methods

### 2.1. Fungal strains, endophyte inoculation and growth conditions

Cultures of *Epichloë festucae* strain Fl1 (PN2278) (Supplementary data, Table S1) were grown on 2.4% potato dextrose agar under conditions described previously [16,17]. Inoculation of endophyte-free seedlings of perennial ryegrass (*Lolium perenne* cv. Samson) with *E. festucae* was performed by the method of Latch and Christensen [18]. Plants were grown and tested for the presence of endophyte as described previously [19].

### 2.2. Molecular biology methods

The molecular biology methods used are as previously described [19,20]. Sequences of PCR primers used in this study are listed in Supplementary data, Table S2. The methods for the preparation of deletion, complementation and expression constructs used in this study are described in Supplementary data.

### 2.3. *E. festucae* and *P. paxilli* transformations

Protoplasts of *E. festucae* were prepared as described previously [10,20]. Protoplasts were transformed with 5 μg linear PCR product amplified from pPN178, pPN180, pPN182, pPN184 or pPN186 with primer set plI99–2 and plI99–3 or co-transformed with 5 μg of circular plI99 and 10 μg of pPN179, pPN181, pPN183, pPN185 or pPN187 as described in Supplementary data using the methods described previously [21,22] Deletions were confirmed by Southern analysis (Supplementary data). Protoplasts of *P. paxilli* pax cluster deletion mutant strain LMP1, LMQ226 and LM662 were prepared and transformed with 5 μg of circular plasmids plI99, pSS56, or pSS58 as described previously [23].

#### 2.4. Feeding of substrate metabolites to fungal cultures and indole-diterpene analysis

Indole-diterpene substrate metabolites paspaline and 13-desoxypaxilline were fed to *P. paxilli* cultures as described previously [13]. *P. paxilli* mycelium was analyzed for indole-diterpenes by normal-phase TLC and reverse-phase HPLC as described previously [23]. Indole-diterpenes were extracted from *P. paxilli* mycelia and perennial ryegrass tissue and analyzed by HPLC and MS as described in Supplementary data.

## 3. Results and discussion

### 3.1. Analysis of *E. festucae* ltm deletion strains

To investigate the role of *E. festucae* Fl1 *ltmQ*, *ltmF*, *ltmK*, *ltmE*, and *ltmJ* genes in lolitrem B biosynthesis and in symbiosis, disruption constructs for each *ltm* gene, were prepared and transformed into *E. festucae* and knockouts identified by PCR and Southern screening (Supplementary data, Fig. S1). The *ltmQ-70* (*ΔltmQ*), *ltmF-11* (*ΔltmF*), *ltmK-4* (*ΔltmK*), *ltmE-94* (*ΔltmE*), and *ltmJ-2* (*ΔltmJ*) mutants had similar culture and in planta growth and morphology phenotypes to Fl1 wild-type. However, the indole-diterpene profile of the ryegrass seedlings infected with these mutants was distinct from those infected with the wild-type (Fig. 2). APCI⁺ LC-MS analysis showed that paspaline (*m/z* 404 + 422) and 13-desoxypaxilline (*m/z* 402 + 420) were present in all plant extracts (Fig. 2, Table 1 Group 2 ions) but none had detectable lolitrem B (*m/z* 686) (Fig. 2, Table 1 Group 1 ions).
suggesting that all the $ltm$ genes tested have a role in lolitrem B biosynthesis and not in the biosynthesis of paspaline and 13-desoxypaxilline.

### 3.2. Analysis of role of $ltmQ$ and $ltmP$ genes

Extracts of plant samples with the $\Delta ltmQ$ mutant had wild-type levels of paspaline and high levels of 13-desoxypaxilline (Fig. 2, Group 2 ions). No other paspalane with $m/z > 422$ was identified. Introduction of a wild-type copy of $ltmQ$ (pPN181; Supplementary data, Fig. S1) into the $\Delta ltmQ$ mutant restored lolitrem B, lolitriol and lolitrem E biosynthesis (Supplementary data, Fig. S2) confirming a role for $ltmQ$ in lolitrem B biosynthesis and 11,12-epoxide formation.

Given the paspalane 13-desoxypaxilline accumulated in the $ltmQ$ mutant treatment, the LC–MS data was critically examined for a hypothetical lolitremane of otherwise equivalent structure. A compound of this nominal mass (Calcd. $m/z$ 586) with a retention time ($R_t$) of 17.0 min was detected, but only in the $ltmQ$ mutant. High resolution ESI$^+$ mass spectrometry, Calcd. $m/z$ 586.3517 identified a compound (13-desoxylolilline; Fig. 5) that was consistent with a lolitremane formula of $C_{37}H_{48}O_5N$ (Calcd. $m/z$ 586.35270).

To determine the function of $ltmP$ and $ltmQ$, an approach previously used for analyzing early steps in lolitrem biosynthesis was used to dissect the biosynthesis of the more complex indole-diterpenes such as 11,12-epoxy-paspalanes and lolitrem B [7,13]. However, the inability of $ltmP$ or $ltmQ$ to complement $P$. paxilli $\Delta paxP$ and $\Delta paxQ$ mutants suggests that $ltmP$ and $ltmQ$ are not functional orthologues of the $P$. paxilli $pax$ genes. The alternative approach of feeding putative precursors to a $P$. paxilli $pax$ gene cluster deletion strain LM662 containing $ltmP$, $ltmQ$ or both $ltmP$ and $ltmQ$ was tested (Table 2). Extracts of paspaline-fed LM662/$ltmP$ strains contained $\beta$-PC-M6 (Supplementary data, Fig. S3), and two unknown compounds with $m/z$ 438 (IDT438a and IDT438b with $R_t$ 14.7 and 16.5 min, respectively, Fig. 3A) identified as paspalane indole-diterpenes based on their MS$^2$ fragmentation spectra (Supplementary data, Fig. S4). All three compounds had ions characteristic of paspalanes including fragments arising from the cyclopentaindolic moiety ($m/z$ 130, $m/z$ 182) and from neutral losses of $CH_3\cdot$radical and $CH_4$ from the pendant gem-dimethyl group. These compounds were also observed as products when paspaline was fed to the LM662/$ltmP/ltmQ$ strain, together with terpendole E and terpendole F (Supplementary data, Fig. S5), and two further unknown indole-diterpenes (IDT422 with $m/z$ 422, $R_t$ 23.48 min; IDT452 with $m/z$ 452, $R_t$ 11.87 min) (Fig. 3B). The latter two compounds also showed characteristic product ions of paspalane indole-diterpenes, as above, in the MS$^2$ spectra (Supplementary data, Fig. S6).

![Figure 2](https://i.imgur.com/yourimage.png)

Fig. 2. LC–MS analysis of ryegrass samples infected with $ltm$ gene knock-out strains. Perennial ryegrass was infected with $E$. festucae Fl1 wild-type or its derivatives containing deletions of $ltmQ$, $ltmF$, $ltmK$, $ltmE$, or $ltmJ$. Samples of infected ryegrass were analyzed for indole-diterpenes by LC–MS. Data for each sample is presented as three groups of extracted-ion chromatograms representing selected lolitremanes, simple paspalanes or prenylated paspalanes appropriately scaled for observed relative ion intensity. The selection of compounds and corresponding ions chosen for each group chromatogram are listed in Table 1.
Table 1  
Indole-diterpene APCI⁺ ions arranged in groups for *ltm* gene knock-out results.

| Label | Compound                     | Retention time $R_t$ (min) | Nominal ion masses (Da) | Identification$^a$ |
|-------|------------------------------|---------------------------|------------------------|---------------------|
| Group 1 ions (lolitremanes)         |                            |                        |                      |                     |
| a      | Lolitriol$^a$               | 6.0                       | 620                    | $R_t$, MS3          |
| b      | Lolitrem J                  | 9.1                       | 662                    | Tentative           |
| c      | Lolitrem B$^a$              | 23.1                      | 686                    | $R_t$, MS3          |
| d      | Lolitrem E$^a$              | 24.9                      | 688                    | $R_t$, MS3          |
| Group 2 ions (simple paspalanes)     |                            |                        |                      |                     |
| e      | 13-Desoxypaxilline$^a$      | 15.2                      | 402+420                | $R_t$, MS3          |
| f      | Paspaline$^a$               | 28.0                      | 404+422                | $R_t$, MS3          |
| Group 3 ions (prenylated paspalanes) |                            |                        |                      |                     |
| g      | IDT 534                      | 14.9                      | 534                    | New                 |
| h      | Terpendole M$^a$            | 16.0                      | 536                    | $R_t$, MS3          |
| i      | IDT 518                      | 19.7                      | 518                    | Terpendole K$^d$   |
| j      | Terpendole C$^a$            | 20.9                      | 520                    | $R_t$               |
| k      | IDT 538                      | 13.9                      | 538                    | New                 |
| l      | IDT 522                      | 22.1                      | 522                    | Terpendole J$^d$   |
| m      | IDT 590$^c$                 | 25.9                      | 534, 536               | Source fragments    |

$^a$ Identification based upon cross reference to standards with same $R_t$ and/or mass spectra to MS3 in LTQ ion trap, otherwise from reports of compounds known or likely to occur.  
$^b$ $R_t \pm 0.1$ min (since sets of samples were analyzed at different time points).  
$^c$ Compound seen only in the $\Delta ltmJ$ treatment and considered to be a new indole ring prenylated indole-diterpene derivative of mass 589 with APCI⁺ MS1 spectrum including base peak M+1 at $m/z$ 590 and fragments at 572, 554, 536, and 534.  
$^d$ Compound previously reported in literature.

When 13-desoxypaxilline was fed to the LM662/ltmQ strain paxilline (Supplementary data, Fig. S7) was observed as a product at a low level in the extract, together with an earlier-eluting product of $m/z$ 436 (IDT436, Fig. 4A, Supplementary data, Fig. S8) that was also found in the LM662 parent strain. A similar pattern of products was observed in an extract of 13-desoxypaxilline-fed LM662/ltmP/ltmQ strain (Fig. 4B, Supplementary data, Fig. S8), suggesting that the unknown compound is not a product of either LtmQ or LtmP.

The absence of any identifiable 11,12-epoxypaspalanes or 11,12-epoxylolitremanes in the *ltmQ* mutants *in planta* suggests that *ltmQ* is essential for 11,12-epoxide formation. However, 13-desoxypaxilline, a metabolite accumulating in the $\Delta ltmQ$ treatment, when applied as a substrate in the *P. paxilli* LM622/ltmP/ltmQ culture did not yield a recognizable 11,12-epoxide product or more immediate precursor indicating that 13-desoxypaxilline does not appear to be the intermediate for epoxide generation. Given that the *P. paxilli* LM622/ltmP/ltmQ culture fed paspaline as substrate yielded terpendole E and terpendole F, would suggest that LtmQ and LtmP can sequentially oxidize paspaline to terpendole E and terpendole F. Alternatively, LtmP converts paspaline to intermediate #1 which is oxidized by LtmQ to terpendole F. These compounds are proposed as intermediates in lolitrem B biosynthesis (Fig. 5). Interestingly, LtmP and PaxP appear to catalyse just α-face oxidations and LtmQ and PaxQ β-face oxidations [13].

Despite no direct evidence for the pathway leading to 11,12-epoxides, terpendole B is proposed as the first 11,12-epoxide based

Table 2  
Indole-diterpene product ions detected by ESI⁺ in LM662/ltmP and LM662/ltmQ strains.

| Nominal ion $m/z$ | Retention time $R_t$ (min) | Compound        | Identification$^a$ |
|-------------------|---------------------------|-----------------|--------------------|
| 422               | 19.5–20.7                 | β-PC-M6$^a$     | $R_t$, MS2         |
| 422               | 23.5                      | IDT422          | Novel              |
| 436               | 15.8–16.9                 | IDT436          | Novel              |
| 436               | 16.4–17.6                 | Paxilline$^a$   | $R_t$, MS3         |
| 438               | 14.7–16.0                 | IDT438a         | Novel              |
| 438               | 16.5–17.5                 | IDT438b         | Novel              |
| 438               | 18.3                      | Terpendole E$^a$ | $R_t$, MS3         |
| 452               | 11.9                      | Terpendole G$^b$ | Tentative          |
| 454               | 7.1                       | Terpendole F$^a$ | $R_t$, MS3         |

$^a$ Identification based upon cross reference to standards with same $R_t$ and/or mass spectra to MS2 or MS3 in LTQ ion trap.  
$^b$ Tentative identification from MS2 and MS3 data and co-occurrence with terpendole F and terpendole E.

While the ΔltmK samples lacked lolitrem B and terpendole C they did contain lolitrem E and two other paspalanes, IDT522 (m/z 522, attributed to terpendole J) and IDT538 (m/z 538, a possible hydroxylation product of IDT522) (Fig. 2, Group 1 and Group 3 ions). A minor product of m/z 688 (IDT688) eluted at a very similar retention time to lolitrem B (Supplementary data, Fig. S2). This product, if naturally occurring, would be subsumed within the larger m/z 688 lolitrem B isotopologue peak, under the LC–MS conditions used. IDT688 is attributed to a stereoisomer of lolitrem E, possibly similar to the known isomer difference between lolitrem B and lolitrem F [27]. The introduction of a wild-type copy of ltmK (Supplementary data, Fig. S1) into the ltmF deletion mutant restored lolitrem B (m/z 686 ion) biosynthesis (Supplementary data, Fig. S2). Lolitrem B was clearly absent in the ΔltmK sample (Supplementary data, Fig. S9 panel C) but present in the complement (Supplementary data, Fig. S9 panel E) and distinguishable from IDT688 (Supplementary data, Fig. S9 panel D peak c). These results indicate that ltmK is required for the oxidative acetal ring formation.

### 3.4. Analysis of ltmE and ltmJ genes

Plant samples with ΔltmE or ΔltmJ mutants showed no detectable lolitremanes but did contain both simple (Fig. 2, Group 2 ions) and O-prenylated paspalanes (Fig. 2, Group 3 ions) similar to those present in wild-type. In addition an unknown of m/z 590 (IDT590) was observed and is assumed to be a prenylated derivative of the paspalanes that accumulate in these mutants due to an inability to synthesize lolitremanes. The introduction of wild-type copies of ltmE and ltmJ (Supplementary data, Fig. S1) restored Lolitrem B biosynthesis (Supplementary data, Fig. S2) to these mutants.

These results are consistent with LtmE being required for C20- and C21-prenylations of the indole ring of paspalanes and acting together with LtmJ, a cytochrome P450 monooxygenase, to yield lolitremanes by multiple oxidations and ring closures in a way special to this class of compounds found so far only in some epichloae endophytes. The stereoisomer pairs of lolitriol and lolitrem N or lolitrem B and lolitrem F may be attributed to variations in the way in which ring closure can occur under the action of LtmJ.

Although the order of action of LtmK must follow that of LtmF and that of LtmJ must follow LtmE, the order of action of the other gene products cannot be specified, in either qualitative or quantitative terms. Not all of the broad range of paspalanes and lolitremanes that have been catalogued [8] were detected in this study but some maybe intermediates in the proposed metabolic grid (Fig. 5). Further studies will be required to define all major pathways and minor shunts associated with this very complex metabolic grid.

Although terpendole E was detected only where paspaline was a substrate in a genetically modified fungal culture, the related lolitremane, loline A, is known to occur in *N. lolii*-infected *Lolium perenne* [26]. The pathway proposed here also suggests an intermediate #3 as a potential new paspalane, but this compound has yet to be identified.

### 3.3. Analysis of ltmF and ltmK genes

In addition to paspaline and 13-desoxypaxilline, the ΔltmF plant samples contained lolitriol (m/z 620) and a small amount of lolitrem J (m/z 662) (Fig. 2, Group 1 and Group 2 ions). This chemical profile was similar to that observed for wild-type samples except there was no production of lolitrem B or lolitrem E or the O-prenylated paspalanes, terpendoles C and J. The introduction of a wild-type copy of ltmF (Supplementary data, Fig. S1) into the ltmF deletion mutant restored lolitrem B biosynthesis in ryegrass plants inoculated with PN2436 (Supplementary data, Fig. S2). Taken together, these results attribute the role of 27-hydroxyl-O-prenylation to LtmF.

S. Saikia et al./FEBS Letters 586 (2012) 2563–2569

![Diagram](attachment:diagram.png)

Fig. 5. Lolitrem B biosynthesis scheme. A scheme for lolitrem B biosynthesis in *E. festucae* Fl1 that includes pathway options for both observed and hypothetical intermediates and oxidation of paspaline by either the sequential or discontinuous action of LtmP and LtmQ.

remarkable versatility in their regio- and stereo-specificities to generate a diverse range of metabolites that are products of a metabolic grid rather than a linear pathway. This catalytic versatility provides great opportunities for metabolic engineering of type and yield of chemical product.

### Acknowledgments

This work was supported by a Grant from the Foundation for Research, Science and Technology (Contract C10X0815) and Massey University. We thank Emily Parker for access to her laboratory

and discussions on some experiments, Ruth Wrenn (Massey University) and Wayne Simpson (AgResearch) for technical assistance, and Sarah Finch (AgResearch) for provision of indole-diterpene standards. We also thank Carolyn Young (Samuel Roberts Noble Foundation) for comments on the manuscript.

### Appendix A. Supplementary data

Supplementary data associated with this article can be found, in the online version, at [http://dx.doi.org/10.1016/j.febslet.2012.06.035](http://dx.doi.org/10.1016/j.febslet.2012.06.035).

References

[1] Philipson, M.N. and Christey, M.C. (1986) The relationship of host and endophyte during flowering, seed formation, and germination of *Lolium perenne*. N.Z. J. Bot. 24, 125–134.

[2] Scott, B. (2001) *Epichloë* endophytes: symbionts of grasses. Curr. Opin. Microbiol. 4, 393–398.

[3] Schardl, C.L., Leuchtmann, A. and Spiering, M.J. (2004) Symbioses of grasses with seedborne fungal endophytes. Annu. Rev. Plant Biol. 55, 315–340.

[4] Clay, K. (1990) Fungal endophytes of grasses. Ann. Rev. Ecol. Syst. 21, 275–297.

[5] Easton, H.S. (1999) Endophyte in New Zealand ryegrass pastures, an overview in: Ryegrass endophyte: an essential New Zealand symbiosis. Grasslands Research and Practice Series No. 7 (Woodfield, D.R. and Matthew, C., Eds.), pp. 1–9, New Zealand Grasslands Association, Palmerston North.

[6] Schardl, C.L., Scott, B., Florea, S. and Zhang, D. (2009) *Epichloë* endophytes: clavicipitaceous symbionts of grasses in: The Mycota. Volume V – Plant Relationships (Deising, H.B., Ed.), pp. 275–306, Springer-Verlag, Berlin.

[7] Young, C.A., Felitti, S., Shields, K., Spangenberg, G., Johnson, R.D., Bryan, G.T., Saikia, S. and Scott, B. (2006) A complex gene cluster for indole-diterpene biosynthesis in the grass endophyte *Neotyphodium lolii*. Fungal Genet. Biol. 43, 679–693.

[8] Sings, H. and Singh, S.B. (2003) Tremorgenic and non-tremorgenic 2,3-fused indole diterpenoids in: The alkaloids: chemistry and biology (Cordell, G.A., Ed.), pp. 52–163, Elsevier, USA.

[9] Parker, E.J. and Scott, D.B. (2004) Indole-diterpene biosynthesis in: Handbook of Industrial Mycology (An, Z., Ed.), pp. 405–426, Marcel Dekker, New York.

[10] Young, C.A., Bryant, M.K., Christensen, M.J., Tapper, B.A., Bryan, G.T. and Scott, B. (2005) Molecular cloning and genetic analysis of a symbiosis-expressed gene cluster for lolitrem biosynthesis from a mutualistic endophyte of perennial ryegrass. Mol. Genet. Genomics 274, 13–29.

[11] Schardl, C.L., Young, C.A., Faulkner, J.R., Florea, S. and Pan, J. Chemotypic diversity of epichloae, fungal symbionts of grasses. Fungal Ecology 5, 331–344.

[12] McMillan, L.K. et al. (2003) Molecular analysis of two cytochrome P450 monooxygenase genes required for paxilline biosynthesis in *Penicillium paxilli*, and effects of paxilline intermediates on mammalian maxi-K ion channels. Mol. Genet. Genomics 270, 9–23.

[13] Saikia, S., Parker, E.J., Koulman, A. and Scott, B. (2007) Defining paxilline biosynthesis in *Penicillium paxilli*: functional characterization of two cytochrome P450 monooxygenases. J. Biol. Chem. 282, 16829–16837.

[14] Young, C., McMillan, L., Telfer, E. and Scott, B. (2001) Molecular cloning and genetic analysis of an indole-diterpene gene cluster from *Penicillium paxilli*. Mol. Microbiol. 39, 754–764.

[15] Young, C.A., Tapper, B.A., May, K., Moon, C.D., Schardl, C.L. and Scott, B. (2009) Indole-diterpene biosynthetic capability of *Epichloë* endophytes as predicted by *ltm* gene analysis. Appl. Environ. Microbiol. 75, 2200–2211.

[16] Moon, C.D., Tapper, B.A. and Scott, B. (1999) Identification of *Epichloë* endophytes *in planta* by a microsatellite-based PCR fingerprinting assay with automated analysis. Appl. Environ. Microbiol. 65, 1268–1279.

[17] Moon, C.D., Scott, B., Schardl, C.L. and Christensen, M.J. (2000) The evolutionary origins of *Epichloë* endophytes from annual ryegrasses. Mycologia 92, 1103–1118.

[18] Latch, G.C.M. and Christensen, M.J. (1985) Artificial infection of grasses with endophytes. Ann. Appl. Biol. 107, 17–24.

[19] Tanaka, A., Tapper, B.A., Popay, A., Parker, E.J. and Scott, B. (2005) A symbiosis expressed non-ribosomal peptide synthetase from a mutualistic fungal endophyte of perennial ryegrass confers protection to the symbiotum from insect herbivory. Mol. Microbiol. 57, 1036–1050.

[20] Young, C., Itoh, Y., Johnson, R., Garthwaite, I., Miles, C.O., Monday-Finch, S.C. and Scott, B. (1998) Paxilline-negative mutants of *Penicillium paxilli* generated by heterologous and homologous plasmid integration. Curr. Genet. 33, 368–377.

[21] Itoh, Y., Johnson, R. and Scott, B. (1994) Integrative transformation of the mycotoxin-producing fungus, *Penicillium paxilli*. Curr. Genet. 25, 508–513.

[22] Tanaka, A., Christensen, M.J., Takemoto, D., Park, P. and Scott, B. (2006) Reactive oxygen species play a role in regulating a fungus-perennial ryegrass mutualistic association. Plant Cell 18, 1052–1066.

[23] Saikia, S., Parker, E.J., Koulman, A. and Scott, B. (2006) Four gene products are required for the fungal synthesis of the indole diterpene paspaline. FEBS Lett. 580, 1625–1630.

[24] Huang, X.-H., Nishida, H., Tomoda, H., Tabata, N., Shiomi, K., Yang, D.-J., Takayanagi, H. and Omura, S. (1995) Terpendoles, novel ACAT inhibitors produced by *Albophoma yamanashiensis*. II. Structure elucidation of terpendoles A, B, C and D. J. Antibiot. 48, 5–11.

[25] Tomoda, H., Tabata, N., Yang, D., Takayanagi, H. and Omura, S. (1995) Terpendoles, novel ACAT inhibitors produced by *Albophoma yamanashiensis*. III. Production, isolation and structure elucidation of new components. J. Antibiot. 48, 793–804.

[26] Monday-Finch, S.C., Wilkins, A.L. and Miles, C.O. (1998) Isolation of lolicine A, lolicine B, lolitriol, and lolitrem N from *Lolium perenne* infected with *Neotyphodium lolii* and evidence for the natural occurrence of 31-epilolitrem N and 31-epilolitrem F. J. Agric. Food Chem. 46, 590–598.

[27] Monday-Finch, S.C., Wilkins, A.L., Miles, C.O., Ede, R.M. and Thomson, R.A. (1996) Structure elucidation of Lolitrem F, a naturally occurring stereoisomer of the tremorgenic mycotoxin Lolitrem B, isolated from *Lolium perenne* infected with *Acremonium lolii*. J. Agric. Food Chem. 44, 2782–2788.

[28] Monday-Finch, S.C., Wilkins, A.L., Miles, C.O., Tomoda, H. and Omura, S. (1997) Isolation and structure elucidation of lolilline, a possible biosynthetic precursor of the lolitrem family of tremorgenic mycotoxins. J. Agric. Food Chem. 45, 199–204.
