
TIMI-936; No. of Pages 11

**ARTICLE IN PRESS**

---

**Review**

Small RNAs and their role in biofilm formation

Jacob R. Chambers and Karin Sauer

Department of Biological Sciences, Binghamton University, Binghamton, NY 13902, USA

The formation of biofilms is initiated by bacteria transitioning from the planktonic to the surface-associated mode of growth. Several regulatory systems have been described to govern the initiation and subsequent formation of biofilms. Recent evidence suggests that regulatory networks governing the decision of bacteria whether to attach and form biofilms or remain as planktonic cells are further subject to regulation by small non-coding RNAs (sRNAs). This is accomplished by sRNAs fine-tuning regulatory networks to enable concentration-specific responses by sequestering, antagonizing, or activating regulatory proteins in response to environmental cues, or by directly affecting the synthesis of proteins promoting or disfavoring the formation of biofilms. This review gives an overview of the contribution of sRNAs in regulating the switch from the planktonic to the sessile bacterial lifestyle by highlighting how sRNAs converge with known regulatory systems required for biofilm formation.

The converging worlds of small RNAs and biofilms

Since the first description in 1981 of a 108 nt RNA involved in blocking replication of the ColE1 plasmid, an increasing number of sRNAs, primarily in the length range of 25–500 nt, have been discovered to play regulatory roles in both prokaryotes and eukaryotes. sRNAs regulate gene expression by binding to mRNA or proteins, resulting in the modulation of translation primarily through altering target stability, affecting protein–DNA binding, or inducing conformational changes within the mRNA via direct anti-sense base-pairing between the sRNA and mRNA [1]. Targets of sRNA regulation tend to be regulatory genes themselves, thus enabling additional levels of control in regulatory networks. In eukaryotes, sRNA regulation has been implicated in processes such as differentiation in late development and programmed cell death. Although sRNA regulation in bacteria was initially described as being predominantly involved in coordinating stress responses, recent evidence suggests that, similarly to eukaryotic cells, bacteria also use sRNAs as key elements in the control of developmental processes as well as multi-cellular behavior [2]. The formation of biofilms is one such process. Biofilms are surface-associated multicellular communities encased in a self-produced extracellular matrix composed of proteins, polysaccharides, and DNA [3,4]. Biofilms likely represent the prevalent microbial mode of existence in nature, with estimates suggesting that more than 90% of bacteria exist within biofilms. Existence in a biofilm affords bacteria many advantages over a planktonic existence, including improved adaptation to nutrient deprivation and increased resistance to predation and antimicrobial agents, characteristics that render biofilms extremely difficult to control in medical, industrial, and agricultural settings [5,6]. Biofilm-associated microorganisms have been shown to colonize a wide variety of man-made and medical devices, and have been implicated in over 80% of chronic inflammatory and infectious diseases of soft tissues and chronic infections of humans with underlying predispositions [5]. The formation of biofilms is initiated with surface attachment by planktonic (single-cell) bacteria that, once attached, grow into a complex community characterized by the presence of differentiated, mushroom- or pillar-like structures or microcolonies interspersed with fluid-filled channels [7]. The developmental progression leading to mature biofilms not only coincides with observable phenotypic changes but also requires cell density-dependent (quorum sensing, QS) and -independent regulated gene expression, many of which are governed by multiple regulatory networks. Following the initial discovery by Romeo and colleagues of the RNA-binding protein CsrA and two sRNAs (CsrB, CsrC) playing a role in *Escherichia coli* biofilm formation [8,9], multiple sRNAs have been identified that modulate the expression or activity of transcriptional regulators and components of regulatory networks important for attachment and biofilm formation (Table 1).

The function of sRNAs in the regulation of biofilm formation occurs via two general mechanisms, (i) sRNAs acting by base-pairing with other RNAs and (ii) protein binding. Protein-binding sRNAs antagonize and sequester their cognate regulatory proteins by mimicking the protein binding sequences found in several mRNAs. Base-pairing sRNAs are categorized as *cis* or *trans* based on their location within the bacterial genome relative to their mRNA targets. sRNAs transcribed from the DNA strand directly opposite to their mRNA targets are designated *cis*-encoded sRNAs and, in general, share extensive complementarity to their targets. By contrast, *trans*-encoded sRNAs are located elsewhere on the genome, function in *trans* as diffusible molecules, and share only limited (10–25 bp) complementarity in their base-pairing interactions [10]. *Trans*-encoded sRNAs often rely on the RNA chaperone Hfq to form limited base-pairing interactions with target mRNAs. The importance of Hfq in *trans*-encoded sRNA-mediated regulation pathways likely accounts for the pleiotropic phenotypes observed in *hfq* mutant strains, including reduced virulence and biofilm formation [11–13].

Corresponding author: Sauer, K. (ksauer@binghamton.edu).

Keywords: small RNAs; biofilm; quorum sensing.

Table 1. Summary of known sRNAs, their target regulatory circuits, and role in attachment and biofilm formation

| Targets       | sRNA                | Organism                          | Regulation of target mRNA          | Phenotype related to sRNA levels                                                                                   | Refs     |
|---------------|---------------------|-----------------------------------|------------------------------------|------------------------------------------------------------------------------------------------------------------|----------|
| Hfq           |                     | E. coli                           |                                     | Deletion reduced attachment                                                                                       | [13]     |
|               |                     | S. Typhimurium                    |                                     | Deletion reduced rdar phenotype (includes reduced attachment and decreased EPS-component expression)              | [32]     |
| CsgD          | OmrA, OmrB          | E. coli                           | Repression                         | Overexpression decreased curli and cellulose production                                                            | [20]     |
|               | GcvB                | E. coli                           | Repression                         | Basal expression decreased curli production                                                                        | [22]     |
|               | RprA                | E. coli                           | Repression                         | Overexpression decreased CsgD synthesis                                                                            | [26]     |
|               | McsS                | E. coli                           | Repression                         | Overexpression decreased curli formation                                                                           | [28]     |
|               | ArcZ                | S. Typhimurium, E. coli           | Activation                         | Overexpression decreased motility and increased biofilm formation. Deletion reduced rdar phenotype                 | [32,33]  |
|               | SdsR                | S. Typhimurium                    | Activation                         | Deletion reduced rdar phenotype                                                                                    | [32]     |
| PgaA          | McsS                | E. coli                           | Activation                         | Overexpression increased curli-independent attachment (potentially via CsrA sequestration)                        | [22]     |
| CsrA          | CsrB, CsrC          | E. coli, Y. pseudotuberculosis    | Repression by sequestration        | Deletion reduced biofilm formation                                                                                 | [8,57,61]|
| YdaM          | RprA                | E. coli                           | Repression                         | Direct binding reduced YdaM expression (likely decreasing c-di-GMP levels)                                        | [26]     |
| Crc           | CrcZ                | Pseudomonas spp.                 | Repression by sequestration        | Phenotype related to biofilms has not been reported                                                                | [73]     |
| RsmA          | RsmY, RsmZ          | P. aeruginosa                     | Repression by sequestration        | Biofilm development                                                                                                 | [101]    |
| AphA          | Qrr1-4              | V. cholerae                       | Activation                         | Overexpression increased AphA (likely resulting in increased biofilm formation)                                   | [89]     |
|               | Qrr1-5              | V. harveyi                        | Activation                         | Overexpression increased AphA (likely resulting in decreased biofilm formation)                                  | [89]     |
| HapR (LuxR homolog) | Qrr1-4 | V. cholerae                       | Repression                         | Overexpression decreased HapR (likely resulting in increased biofilm formation)                                   | [89]     |
| LuxR         | Qrr1-5              | V. harveyi                        | Repression                         | Overexpression decreased LuxR (likely resulting in decreased biofilm formation)                                  | [89]     |
| PqsR         | PhrS                | P. aeruginosa                     | Activation                         | Phenotype related to biofilms has not been reported                                                                | [87]     |
| RpoS         | ArcZ                | S. Typhimurium, E. coli           | Activation                         | Overexpression increased RpoS reporter activity                                                                    | [32,41]  |
|               | RprA                | E. coli                           | Activation                         | Overexpression activates RpoS synthesis                                                                            | [25]     |
|               | OxyS                | E. coli                           | Repression                         | Overexpression decreased RpoS reporter activity                                                                    | [41]     |
|               | DsrA                | E. coli                           | Activation                         | Overexpression increased RpoS reporter activity                                                                    | [41]     |

(Table 1). Base-pairing between sRNAs and the target mRNAs leads to changes in mRNA translation and stability by altering the accessibility to ribosome binding sites (RBS) or enhancing ribonuclease (RNase)-mediated degradation, thereby influencing target gene expression [14]. Considering the growing interest in biofilms and appreciation for the need to prevent and control biofilms in the medical setting and beyond, this review will highlight the role of sRNAs in biofilm developmental processes by focusing on well-characterized regulatory systems governing the transition from the planktonic to the surface-associated mode of growth. To that end, the roles of sRNAs that affect surface attachment and motility, QS, stress response, and modulation of adhesiveness will be addressed with a particular emphasis on their contribution to the underlying regulatory mechanisms in these processes.

Should I stay or should I go: CsgD as a key regulator in the switch between planktonic and sessile modes of growth

Adhesins such as pili and flagella have been demonstrated in many bacterial species to contribute to initial contact with a surface, with attachment triggering alterations in gene expression that allow bacteria to develop a more permanent association with the surface via surface-associated motility (e.g., twitching) and exopolysaccharide biosynthesis [15].

Such a transition to the surface-associated lifestyle requires reprogramming of gene expression profiles. In E. coli and Salmonella strains this shift relies on control cascades that inhibit flagellar expression and activate the synthesis of adhesive curli fimbriae [16–18]. The transcriptional regulator CsgD, a key player in the complex regulatory circuit that decides whether E. coli or Salmonella strains form biofilms, has been shown to be required for attachment and subsequent biofilm formation by activating the production and export of curli fimbriae while repressing the expression of several flagellar biosynthesis genes [16–18]. CsgD also transcriptionally activates adrA (previously known as yaiC) encoding a diguanylate cyclase which synthesizes bis-(3',5')-cyclic-diguanosine monophosphate (c-di-GMP), the second messenger that allosterically stimulates the production of cellulose, an extracellular polymeric substance (EPS). However, different E. coli isolates synthesize various EPS including cellulose, lipopolysaccharides, K antigen, colanic acid, and the cell-bound poly-β-1,6-N-acetyl-D-glucosamine (PGA) [19]. Thus, CsgD acts as a switch between the planktonic and sessile lifestyle by inversely coordinating the expression of genes involved in flagellar/motility control and adhesiveness. Given that CsgD lies at the heart of the complex switch from a motile to a sessile lifestyle, it comes as no surprise that its expression is regulated by a multitude of cellular cues (Figure 1). No less than a dozen

TIMI-936; No. of Pages 11

ARTICLE IN PRESS

Review

Trends in Microbiology xxx xxxx, Vol. xxx, No. x

Stationary phase  
Glucose availability (via cAMP, CRP)  
Envelope stress (via RcsCD, RcsB)  
Osmolarity (via EnvZ)  
Amino acid metabolism (via GcvA/GcvR)

Key:  
- Repression  
- Activation  
- CsgD-independent activation  
- c-di-GMP  
- sRNA  
- Cyclase  

Planktonic, motility  
Biofilm  

Figure 1. Regulatory network controlling CsgD and effect on biofilm formation and motility. *csgD* expression is controlled at the transcriptional level by signal integration by the sigma factor RpoS, the TCS OmpR and indirectly by the cyclase YdaM via the modulation of c-di-GMP levels. In addition, *csgD* expression is controlled post-transcriptionally via several antisense sRNAs whose expression is affected by a wide range of environmental stimuli. Cell-surface stress activates RprA (via RcsB), GcvB is activated depending on the glycine status of the cell (via GcvA), osmotic imbalance results in the activation of OmrA/B via OmpR, and quality of carbon source/growth rate results in the activation of McaS gene expression [transcription factor(s) unknown]. Activation of CsgD represses genes associated with the planktonic lifestyle (motility; e.g., *flhDC*) and activates genes associated with the surface-attached mode of growth, including curli fimbriae and polysaccharide gene expression (e.g., for PGA and cellulose). In *E. coli*, RpoS is regulated by several sRNAs in response to environmental stimuli, including low temperature (DsrA; transcription factor(s) unknown) and oxygen tension (ArcA–ArcB represses ArcZ under anaerobic conditions), which result in activation of RpoS. RpoS is repressed upon sensing oxidative stress (OxyR activates OxyS). Adapted with permission from Boehm and Vogel [35].

different transcription factors, as well the signaling molecule c-di-GMP produced by diguanylate cyclases such as YdaM, are known to activate the *csgD* promoter. In addition to transcriptional regulation, expression of *csgD* is regulated at the mRNA level by no less than five Hfq-dependent sRNAs (McaS, RprA, OmrA/OmrB, and GcvB) in response to environmental cues (Figure 1, Table 1). All five trans-encoded sRNAs act as repressors by base-pairing with the 5′ untranslated region (5′UTR, ~150 nt in length) of the *E. coli* *csgD* mRNA, probably by occluding the RBS and interfering with translational initiation. However, each belongs to a different regulon and is expressed under different growth conditions. For example, expression of OmrA/B is regulated in response to high osmolarity via the two-component regulatory system (TCS) EnvZ–OmpR. Overexpression of these two redundant sRNAs results in curli deficiency, likely due to downregulation of *csgD*, and conditions favoring the planktonic mode of growth [20]. The sRNA GcvB is a global post-transcriptional regulator of amino acid transport and synthesis genes that also represses *csgD* in response to amino acid availability and is in turn regulated by GcvA and GcvR, the two primary transcription factors involved in regulation of the glycine cleavage system [21,22].

The sRNAs not only serve as input modules for diverse environmental cues through their own transcriptional regulators but also crosslink individual branches of the CsgD network (Figure 1). Expression of the *rprA* sRNA is stimulated by the Rcs proteins, a multi-component phosphorelay system that responds to cell-envelope stress. RprA is a post-transcriptional activator of RpoS, the general stress-response sigma factor, with RpoS in turn being a major regulator/activator of *csgD* expression and curli biosynthesis upon entry into stationary phase [23–25]. RprA has also been reported to inhibit the synthesis of the diguanylate cyclase YdaM by binding downstream of the translation initiation region of the *ydaM* mRNA, thus preventing/reducing *csgD* transcription via YdaM and c-di-GMP [26]. The findings indicate that RprA not only interferes with *csgD* expression directly by hampering its translation but also indirectly by modulating optimal *csgD* transcription. Moreover, the Rcs system activates the expression of a further biofilm matrix component, the exopolysaccharide colanic acid, and the Rcs-controlled *csgD* repressor RprA may serve the additional function of preventing undesired expression of colonic acid with curli/EPS [27] or, alternatively, may function in balancing the expression of both matrix components, cellulose and colanic acid, in response to environmental cues. Similarly to RprA, McaS (a 95 nt sRNA, whose expression is induced by non-preferred carbon sources and entry into stationary phase [28]) not only represses the synthesis of CsgD directly but also activates the expression of genes that further promote the planktonic mode of growth (thus disfavoring biofilm formation) by directly inducing genes involved in flagellar biosynthesis [22,28,29].

In *Salmonella enterica* serovar Typhimurium, CsgD is equally required as a switch between the planktonic and biofilm mode of growth in response to environmental cues and is responsible for the rdar (red, dry, and rough) morphotype, a biofilm behavior characterized by the presence of curli fimbriae and the production of cellulose [17,30,31]. However, the close sibling of *E. coli*, *Salmonella* not only

lacks the McaS sRNA but also shows sequence deviation in  
the 5'UTR of *csgD*, suggesting that a different set of sRNAs  
repress *csgD* translation. The findings furthermore suggest that although CsgD in *Salmonella* and *E. coli* are  
homologous systems, CsgD is subject to substantially different regulation in these bacterial species. Recent findings  
by Monteira *et al.* [32] implicated a different set of Hfq-  
dependent sRNAs, ArcZ and SdsR, in CsgD-mediated bio-  
film formation in *Salmonella*. SdsR was found to be required for maintaining steady-state levels of *csgD*, whereas  
ArcZ was shown to repress *fliC*, a gene encoding a core  
component of the flagellar machinery [32,33]. Although  
neither sRNA has been shown to interact directly with  
*csgD* transcripts, both either regulate RpoS [33] or are  
directly regulated by RpoS [34].

Considering the large number of regulators acting upon  
CsgD, it is not surprising that *csgD* mRNA is considered a  
hub for signal integration via multiple sRNAs and a key  
player in the complex regulatory circuit that decides  
whether *E. coli* or *Salmonella* spp. form biofilms [35].  
One is left to wonder, however, why CsgD is such a hotspot  
for regulation. Considering that bacteria must cope with  
fluctuations in nutrient availability and stress conditions,  
sRNAs may provide feedback regulation to titrate *csgD*  
synthesis under physiological conditions (stationary phase  
and biofilms) that call for motility or in response to environmental cues that are unfavorable for biofilm formation.  
This is further supported by sRNAs not only modulating  
*csgD* synthesis but also directly influencing the decision to  
make flagella with ArcZ, OmrA, and OmrB negatively, and  
McaS positively, regulating motility [36].

### Stress management

CsgD is a module within the general stress response, for  
which the general stress-response sigma factor RpoS acts  
as the master regulator [23,24]. The accumulation of RpoS  
is regulated at multiple levels, including post-transcriptionally by sRNAs (Table 1). In *E. coli*, the alternative  
sigma factor RpoS responds to multiple stresses and is  
strongly upregulated during entry into stationary phase.  
RpoS activates a large number of genes that allow bacteria  
to adapt to changing environmental conditions, and *rpoS*  
expression correlates with downregulation of motility and  
activation of YdaM expression (Figure 1), all of which are  
essential to activate transcription of *csgD* [24]. Given its  
connection to CsgD, and that biofilm formation has been  
generally linked to slow growth and stressful growth conditions, it is not surprising that inactivation of *rpoS* results  
in reduced *E. coli* biofilm formation and increased expression of genes involved in flagella synthesis [37,38]. The  
accumulation of RpoS in *E. coli* and *Salmonella* spp. is  
regulated at multiple levels, including the regulation of its  
translation by four Hfq-dependent sRNAs, OxyS, ArcZ,  
DsrA, and RprA (Figure 1) [39]. OxyS is the only repressor  
of *rpoS* translation whereas ArcZ, DsrA, and RprA activate  
translation. In the absence of activation factors, translation  
of *rpoS* mRNA is hindered by a stem-loop structure that  
sequesters the Shine-Dalgarno (SD) RBS. Base-pairing of  
ArcZ, DsrA, and RprA with one strand of the stem-loop  
releases the occluded SD site for effective ribosome binding  
[40,41]. Similarly to that of CsgD, post-transcriptional  

regulation of RpoS is linked to environmental cues and  
physiological conditions. OxyS is induced upon sensing  
oxidative stress; ArcZ is under the control of the aerobic/  
anaerobic-sensing TCS ArcA/B and is only expressed under  
aerobic conditions, whereas DsrA is induced at low temperatures [41]. In addition to regulating RpoS, DsrA, itself  
regulated by the *E. coli* autoinducer 2 (AI2)-based QS system, also functions as a regulator of both capsular polysac-  
charide biosynthesis and multidrug efflux pumps [42–44].  
Nishino *et al.* [44] identified DsrA in an *E. coli* mutant screen  
to identify regulatory elements involved in the expression of  
other multidrug resistance systems. Inactivation of *dsrA* correlated with decreased drug susceptibility to oxacillin,  
whereas overexpression of *dsrA* conferred resistance to  
oxacillin, erythromycin, rhodamine 6G, and novobiocin in  
*E. coli* lacking a functional *acrB* (AcrB is a component of the  
nodulation-cell division efflux pump). Although no detailed  
mechanism was elucidated, DsrA expression coincided with  
increased polysaccharide synthesis and *rpoS* expression,  
and increased expression of *mdtE*, a component of the  
MdtEF multidrug efflux pump in *E. coli* [44].

The global regulatory activity of RpoS in relation to  
biofilm growth is also present in various other bacteria. In  
*Pseudomonas aeruginosa* PAO1, where the prevalence of a  
slow-growing population within these communities has  
been repeatedly demonstrated under different conditions,  
*rpoS* expression is increased in biofilms compared to stationary phase [45,46], with inactivation resulting in altered biofilm architecture compared to the wild type [47].  
However, RpoS plays slightly different roles in biofilm  
development in *E. coli* and *P. aeruginosa*, as evidenced  
by the recent findings of RpoS being a positive regulator of  
*psl* gene expression in *P. aeruginosa* PAO1 [48]. Psl not  
only contributes to attachment but is also considered to be  
the primary structural polysaccharide essential for biofilm  
maturation [49]. In the plant pathogen *Serratia* sp. ATCC  
39006, the Hfq-binding sRNA RprA directly increases *rpoS*  
translation, leading to decreased antibiotic production  
[50]. Although the importance of biofilm formation remains  
to be studied in this organism, RpoS-mediated biofilm  
growth is important in the closely related rhizospheric  
bacterium *Serratia plymuthica* IC1270, where inactivation  
results in reduction of biofilm formation [51]. Direct regulation of *rpoS* translation upon sensing stress conditions  
plays a large role in biofilm formation. However, although  
RpoS is a key player in biofilm formation, the above-cited  
work highlights that the regulatory function of RpoS is  
interconnected with multiple regulatory systems governing  
the transition from the planktonic to the biofilm mode  
of growth.

### Suck it up: sequestration as a transition switch towards  
surface-associated growth

Early attempts to understand the underlying mechanisms  
of RpoS as a central activator of general stress responses  
and stationary-phase gene expression [39] led to the recognition that CsrA, like an ‘evil twin’ of RpoS, counters the  
activity of RpoS by repressing stationary-phase gene expression and activating genes needed for growth [52].  
Originally identified as regulating glycogen biosynthesis  
in *E. coli*, the RNA-binding protein CsrA regulates primary  

and secondary metabolic pathways, motility, virulence circuitry of pathogens, quorum sensing, and genes involved in the stress response [9,52,53]. Considering its RpoS- opposing activity, it is not surprising that CsrA represses biofilm formation [8] by activating the translation of genes involved in the planktonic lifestyle while repressing the synthesis of genes associated with the sessile lifestyle. This is accomplished by CsrA dimers binding to GGA motifs in the 5'UTR of target mRNAs, thereby altering their translation and/or turnover. For instance, the polysaccharide PGA is required for attachment, cell–cell adherence, and stabilization of the biofilm structure. The pgaABCD operon is required for PGA synthesis (PgaC and PgaD) and secretion (PgaA and PgaB) [54]. CsrA inhibits PGA synthesis and biofilm formation by cooperatively binding to six sites in the pgaABCD mRNA leader, competing with the ribosome for binding and repressing translation of pgaA [55]. Recent evidence suggests that pgaABCD translation is further regulated by McsA, potentially by acting as an antagonist of CsrA [28], because high levels of McsA resulted in increased PGA-dependent biofilm formation, whereas a strain lacking McsA exhibited reduced biofilm formation. In the case of flhDC mRNA (encoding a DNA-binding protein that initiates a regulatory cascade for the expression of genes required for motility and chemotaxis), CsrA binding stabilizes the respective mRNA, and overexpression of csrA results in increased motility [56].

CsrA activity is counteracted by two sRNAs, CsrB and CsrC. Both sRNAs contain multiple CsrA binding sites, which mimic the GGA binding sites of CsrA target mRNAs. CsrB is a 366 nt sRNA containing 18 CsrA binding sites, whereas CsrC is 245 nt in length and contains only nine

CsrA binding sites, with these sites capable of sequestering CsrA and thereby inhibiting the regulatory activity of the protein [57]. Thus, when CsrB and CsrC levels increase, the sRNAs effectively sequester the CsrA protein away from target mRNA leaders (Figure 2a). The ability of these two sRNAs to negatively regulate CsrA activity impacts upon the ability of E. coli to form and maintain biofilms because both depletion of CsrA and overexpression of CsrB/C lead to cellular autoaggregation and enhanced biofilm formation [57]. Transcription of the csrB and csrC genes is induced by the BarA–UvrB two-component regulators when cells encounter nutrient poor growth conditions, oxidative stress, weak acids (formate and acetate), or perturbations in the levels of Krebs cycle intermediates [58]. The CsrB and CsrC RNAs also are regulated at the level of stability through the CsrD protein which is not an RNase but instead recruits RNase E to degrade the sRNAs [59] (Figure 2b). Interestingly, CsrD protein contains GGDEF and EAL domains, and although both domains are required for CsrD activity, the regulation of CsrB/C decay does not involve cyclic di-GMP metabolism [59]. The Csr system contains multiple layers of regulation by making use of non-coding RNAs that sequester multiple copies of CsrA to provide precise control of CsrA levels and activity and, thus, of the transition to the surface-associated lifestyle. Negative feedback loops of the Csr system exist, with CsrA repressing csrD expression in E. coli as well as Salmonella [59,60].

### Csr homologous systems

CsrA homologs are highly conserved among many pathogenic bacteria, including *P. aeruginosa*, *Salmonella enterica*, *Helicobacter pylori*, *Erwinia* spp., *Legionella pneumophila*,

![Figure 2](#)

**Figure 2.** CsrA and CsrA homolog RsmA and effect of sequestration by sRNA. (a) Gene expression is controlled by CsrA binding to leader segments of target mRNAs (e.g., pgaABCD involved in PGA biosynthesis and export), affecting their translation and stability. CsrA activity is repressed via sequestration of CsrA by sRNAs CsrB/C, thereby inhibiting CsrA regulatory activity. (b) CsrA activity is enhanced by CsrD as well as by the sRNA McsA, a component of the CsgD network which represses CsgD. Repression by CsrA on pga mRNA translation is negated by NhaR, an activator of PGA biosynthesis. (c) Schematic overview of the signaling cascade that converges on the sRNAs RsmY and RsmZ, which act by sequestering the translational repressor RsmA. RsmA reciprocally regulates factors involved in the planktonic/sessile switch, as indicated by activation of genes involved in motility or by repressing genes required for biofilm formation (Pel, Psl polysaccharides). Dashed lines indicate that the connection has not been fully demonstrated or is not understood at the molecular level. Adapted with permission from Mikkelsen et al. [101].

and *Vibrio cholerae*, and post-transcriptionally modulate diverse transcripts to control virulence mechanisms and group behaviors. In *Yersinia pseudotuberculosis*, sequestration of CsrA by CsrB/C results in a loss of motility, likely through decreased CsrA activation of flagella biosynthesis genes [61]. Unlike in *E. coli*, however, CsrB and CsrC expression decreases rather than increases following activation of the BarA/UvrY homologs. Moreover, CsrB and CsrC are differentially expressed depending on environmental conditions and nutrient availability, with CsrC expression increasing during growth in rich complex medium and CsrB expression increasing during late stationary phase [61]. *S.* Typhimurium also possesses a CsrA homolog, and deletion of both CsrB and CsrC results in increased motility but decreased biofilm formation due to increased CsrA activity [62].

Among the Csr homologous systems, however, few have received more attention than the *P. aeruginosa* LadS/RetS/Gac/Rsm signal-transduction network [63,64] (Table 1). This intricate signaling system has been implicated as a switch between planktonic and biofilm modes of growth, as well as between acute and chronic infections, by reciprocally regulating gene expression associated with type III and VI secretion and exopolysaccharide production via the CsrA homolog RsmA (regulator of stationary-phase metabolites) [65] (Figure 2c). Similarly to CsrA, RsmA controls gene expression by binding to leader segments of target mRNAs, affecting their translation and stability. For instance, Irie et al. [48] demonstrated binding of RsmA to the 5′UTR of *psl* mRNA, thus preventing ribosome access and protein translation. In *P. aeruginosa*, RsmA function is antagonized by the sRNAs RsmZ and RsmY whose expression is directly controlled by GacA–GacS, the TCS homolog of the *E. coli* BarA/UvrY system. GacA–GacS function is in turn inversely controlled by the TCS hybrids RetS and LadS [64,66,67]. RetS negatively controls RsmYZ gene expression and inactivation of *retS* results in hyperattachment with elevated Psl exopolysaccharide gene expression and suppression of the type III secretion system (TTSS), and the phenotype was abolished by a secondary mutation in *gacS* [64]. LadS controls sRNA levels positively, and *ladS* inactivation results in decreased attachment, reduced Psl production, and elevated TTSS expression, suggesting that LadS may function to counteract RetS. Although the mode of LadS activity remains uncharacterized, RetS reduces sRNA expression by interfering with GacS autophosphorylation through the formation of RetS–GacS heterodimers [68].

Recent evidence suggests that the Rsm signal-transduction network does not simply function as a switch to enable the transition from the planktonic to the sessile mode of growth. This is based on the findings that although increased expression of RsmYZ results in enhanced initial attachment to abiotic surfaces, subsequent surface-attached growth and biofilm development are hampered by high levels of these sRNAs, in particular RsmZ [69,70]. Moreover, the histidine phosphotransfer protein B (HptB) and SagS contribute to sRNA modulation. SagS is a sensor-regulator hybrid that participates in a phospho-transfer event with HptB [71,72] and, similarly to HptB, is involved in the regulation of attachment and biofilm

formation [70]. SagS represses sRNA levels predominantly under planktonic growth conditions, and inactivation of *sagS* results in a temporary enhancement of attachment but a defect in the later stages of biofilm formation [70]. Considering that the Δ*sagS* phenotype furthermore superseded those of both *gacA* and *rsmYZ* mutants, these findings suggested that SagS represents a novel level of Gac/Rsm regulation of attachment [70]. The findings further indicated a requirement for tight modulation of RsmYZ levels as the bacterial population progresses through the different phases of biofilm growth [69] (Figure 2c).

Although not sharing sequence similarity or binding specificities with CsrA, the mRNA-binding protein Crc (catabolite repression control) of Pseudomonads is likewise a global regulator of carbon metabolism and other processes and its activity is governed by binding to antagonistic sRNAs [73]. In *Pseudomonas putida*, Crc is controlled by the functionally redundant sRNAs CrcZ and CreY [73]. Catabolite-repression conditions correlated with low CrcZY levels, and inactivation of both sRNAs led to constitutive catabolite repression that compromised growth on some carbon sources. Although the authors did not link their findings to biofilm formation, it is likely that at least CrcZ (CrcY is present in *Pseudomonas fluorescens*, *P. putida*, and *Pseudomonas syringae*, but absent from *P. aeruginosa* [73]) is linked to biofilm formation because a *crc* mutant has been characterized by impaired type IV pili production, decreased synthesis of exopolysaccharide, and reduced biofilm formation [74,75].

Regulation of stickiness in bacterial lifestyle switching Another key player governing the transition to the surface-associated mode of growth by inversely controlling motility and curli/EPS expression is the signaling molecule c-di-GMP. c-di-GMP, predicted to be present in 85% of all bacteria, controls the switch between biofilm formation and motility depending on its intracellular concentration, with high levels favoring the sessile lifestyle and thus the stickiness or adhesiveness of the bacterial community [76–78]. Likewise, the transition of biofilm bacteria to the planktonic growth state, a process called dispersion, has been linked to modulation of c-di-GMP levels [79]. c-di-GMP is synthesized from two GTP molecules by diguanylate cyclase (DGC) enzymes containing GGDEF domains consisting of approximately 170 amino acids, and is degraded by phosphodiesterase (PDE) enzymes containing EAL or HD-GYP domains that are approximately 250 amino acids in length. As a key player in the decision between the motile planktonic and sedentary biofilm-associated bacterial ‘lifestyles’, c-di-GMP binds to an unprecedented range of effector components and controls diverse outputs, including transcription and the activities of enzymes and larger cellular structures [78].

Recent evidence suggests that the modulation of c-di-GMP levels is subject to regulation by sRNAs, both directly and indirectly. In *E. coli*, indirect regulation of c-di-GMP levels has been linked to the sRNA-controlled biofilm regulator CsgD. For instance, CsgD activates the DGC AdrA. Expression of *adrA* has been indirectly correlated (via increased levels of c-di-GMP and CsgD) with inhibition of flagellum production and rotation, and increased biofilm

formation [16]. *adrA* expression has also been linked to increased cellulose synthesis [80]. Moreover, CsgD is itself subject to regulation by c-di-GMP via the DGC YdaM which is required for the expression of the biofilm-associated curli fimbriae [80] (Figures 1, 2b). Both CsgD and YdaM are negatively controlled by the sRNA RprA [26]. Thus, RprA creates a negative feedforward loop, directly resulting in the downregulation of CsgD- and YdaM-regulated genes and ensuring that c-di-GMP levels to remain too low to activate *csgD* expression.

Similarly to CsgD, the Csr system has been linked to c-di-GMP modulation. Jonas *et al.* [60] demonstrated that inactivation of *csrA* resulted in increased expression of two genes encoding cyclases, *ycdT* and *ydeH*, and led to modestly increased levels of c-di-GMP. Overexpression of *ycdT* and *ydeH*, however, increased the intracellular c-di-GMP levels 20-fold [60]. It is of interest to note that *ycdT* and *ydeH* are among a handful of target mRNAs which have been shown to be directly regulated by CsrA at the post-transcriptional level [81]. Conversely, activation of CsrA in *Salmonella* spp. led to repression of DGC activity but stimulation of PDE activity, thereby causing c-di-GMP levels to decrease. Decreased c-di-GMP levels coincided with increased expression of motility-associated genes, whereas increased c-di-GMP levels favored the switch to a sessile lifestyle [60, 82]. Moreover, CsrA was found to regulate the expression of five additional GGDEF/EAL proteins [60]. An additional level of c-di-GMP-related regulation exists via CsrD, which is essential for the RNase E-mediated decay of the CsrA antagonistic sRNAs CsrB/C in *E. coli* [59] (Figure 2b). Interestingly, although CsrD, a member of the GGDEF/EAL domain family, neither produces nor degrades c-di-GMP, CsrD activity requires the presence of both the GGDEF and EAL domains [59]. Together, these data demonstrate a global role for CsrA in the regulation of c-di-GMP metabolism by regulating the expression of GGDEF/EAL proteins at the post-transcriptional level. Moreover, by tightly modulating c-di-GMP levels, sRNAs are capable of altering global gene expression responsible for transitioning between the motile and sessile lifestyles.

Fine-tuning bacterial communication

QS enables bacteria to communicate using extracellular signaling molecules termed autoinducers (AIs), and is achieved by the synthesis, secretion, and detection of these molecules that accumulate in proportion to increasing cell density [83]. This process ensures that bacteria behave as individuals at low cell density but exhibit group behaviors at high cell density due to the coordination of gene expression on a population-wide scale [83]. The QS circuits identified in Gram-negative bacteria resemble the canonical QS circuit of the symbiotic bacterium *Vibrio fischeri* and contain, at a minimum, homologs of two *V. fischeri* regulatory proteins named LuxI and LuxR. LuxI homologs are responsible for the biosynthesis of AIs (acylated homoserine lactones, AHLs) whereas LuxR-type proteins are transcriptional regulators that bind to cognate AIs. Given the close proximity and density of cells within biofilms, it is not surprising that QS plays an important role in biofilm formation. This was first demonstrated by Davies *et al.*

[84] by showing that QS in *P. aeruginosa* is crucial for proper biofilm formation. Specifically, a *P. aeruginosa ΔlasI* mutant formed only flat, undifferentiated biofilms (monolayers), unlike wild type biofilms which are characterized by a structured and differentiated architecture. Exogenous addition of the AI synthesized by LasI restored biofilm formation to wild type levels, indicating that the defect was a result of AI absence. *P. aeruginosa* possesses two hierarchically organized QS circuits (Las and Rhl) that are activated by two types of AHLs that control the expression of more than 300 genes. Another type of bacterial signal molecule is the *Pseudomonas* quinolone signal (PQS), which positively regulates a subset of QS-dependent genes and biofilm formation [85]. PQS is regulated by the transcription factor PqsR [86], which in turn is post-transcriptionally activated by the *cis*-acting sRNA PhrS that base-pairs to a short open reading frame located directly upstream of the translationally coupled *pqsR* gene [87]. PhrS binding resolves an inhibitory secondary structure, resulting in the unmasking of the RBS.

sRNAs have also been demonstrated to play a role in QS and biofilm formation by *Vibrio harveyi*. This QS circuit possesses features reminiscent of both Gram-negative and Gram-positive QS systems. Similarly to other Gram-negative bacteria, *V. harveyi* produces and responds to AIs of the AHL class, while also utilizing a TCS circuit containing a phosphorelay cascade comparable to the QS system present in Gram-positive bacteria for QS signal transduction. In the absence of AI (low cell density), LuxO, a membrane-bound response regulator hybrid with histidine kinase activity, is phosphorylated and indirectly activates the translation of the low cell density master regulator AphA. AphA controls ~300 low cell density target genes [88] and inhibits translation of the high cell-density master regulator LuxR. By contrast, when LuxO is dephosphorylated at high cell densities, AphA translation stops and LuxR translation occurs, with LuxR controlling ~700 high cell-density target genes (Figure 3) [89]. Reciprocal control of LuxR and AphA by LuxO is achieved through five Hfq-dependent sRNAs termed Qrr1–5 (quorum regulatory RNA) whose expression is activated by phosphorylated LuxO. Once expressed, Qrr sRNAs activate translation of AphA while simultaneously repressing translation of LuxR (Figure 3) [89]. Shao and Bassler [89] demonstrated that Qrr1 is less effective than Qrr2–5 in activating *aphA* because Qrr1 lacks one of two required pairing regions. Qrr1, however, was found to be as effective as the other Qrr sRNAs at controlling targets like *luxR*. Moreover, the Qrr sRNAs are not redundant but act additively, resulting in the production of a LuxR protein concentration gradient that differentially affects the expression of target genes, including those involved in virulence, pilus and flagellar synthesis, and biofilm formation [89, 90]. Additionally, Qrr sRNAs negatively regulate their own expression by base-pairing with and inhibiting the expression of *luxO*, which is responsible for the activation of the *qrr* genes, further emphasizing that the sRNAs are integrated into regulatory QS circuits. Thus, in addition to fine-tuning the QS output, Qrr sRNAs participate in regulatory or feedback loops not only by regulating the translation of LuxR and HapR but also their own expression.

TIMI-936; No. of Pages 11

ARTICLE IN PRESS

Review

Trends in Microbiology xxx xxxx, Vol. xxx, No. x

![Diagram](attachment:diagram.png)

Key:
- Low cell density
- High cell density
- Cell density-independent

Figure 3. Model for Qrr sRNA regulation of QS in *Vibrio* spp. At low cell density (green arrows), phospho-LuxO activates expression of the *qrr* genes encoding the Qrr sRNAs. The Qrr sRNAs promote translation of the low cell-density master regulator AphA and inhibit translation of the high cell-density master regulator LuxR/HapR. At high cell density (red arrows), Qrr sRNA production ceases because dephosphorylated LuxO is inactive. AphA translation stops and LuxR/HapR translation occurs. LuxO production is repressed by the Qrr sRNAs in a negative feedback loop. AphA and LuxR repress each other at the transcriptional level. LuxO activity is furthermore regulated via CsrA and sRNAs CsrB/C (blue arrows). Adapted with permission from Shao and Bassler [89].

The QS circuit of the closely related pathogenic bacterium *V. cholerae* resembles that of *V. harveyi*, but *V. cholerae* only has Qrr1–4, which are largely redundant [91]. Moreover, in contrast to other bacterial pathogens that induce virulence factor production and/or biofilm formation at high cell density in the presence of AIs, *V. cholerae* represses these behaviors at high cell density (Figure 3). Hammer and Bassler [92] demonstrated that inactivation of the LuxR homolog *hapR* 'locked' *V. cholerae* in a state mimicking low density, while enhancing attachment. Enhanced attachment correlated with increased expression of genes involved in the synthesis of the VPS exopolysaccharide required for *V. cholerae* biofilm development in freshwater environments [92, 93]. By contrast, high cell-density conditions achieved by inactivation of *luxO* impaired attachment. An additional layer of sRNA-mediated regulation of HapR and QS-dependent gene expression exists in *V. cholerae* via the CsrA protein and BarA-UvrY homologs VarS-VarA [94]. VarS-VarA controls transcription of CsrBCD, three redundant sRNAs that, similarly to the sRNAs CsrBC of *E. coli*, bind to and sequester the global regulatory protein CsrA. When active, CsrA acts through LuxO to increase expression of Qrr sRNAs, and activation of CsrB is required for full HapR activity [94, 95]. Thus, expression of Qrrs, in addition to being activated by phosphorylated LuxO and repressed by Qrrs, is subject to regulation by the Csr system. Consistent with the regulatory network described above, Jang *et al.* [96] determined that inactivation of *varS* resulted in decreased expression of *csrBC* and *hapR*, and increased attachment by *V. cholerae* 2740-80. Likewise, biofilm formation is repressed upon agr QS activation in *Staphylococcus epidermidis* and *S. aureus* [97, 98]. To our knowledge, the only sRNA described so far to play a role in staphylococcal biofilm formation is the multifunctional regulatory RNA named RNAIII, an effector molecule of the agr QS system which primarily acts as a repressor of translation [99].

### Concluding remarks

This review was aimed at giving an overview of the role of sRNAs involved in biofilm development, a research field still in its infancy. The overview is by no means complete and may occasionally present conflicting information. Nevertheless, it is apparent from the above-cited studies that sRNAs are important players in regulatory networks controlling the transition to the surface-attached lifestyle. Not only are sRNAs regulatory components that enable fine-tuning with respect to induction or repression of gene expression in a dose-dependent manner, as in the case of Qrr sRNAs or the Csr system, they also enable feedback or feedforward loops and mediate crosstalk between global regulatory networks. An example of the latter is the role of the *V. cholerae* Qrr1 in repressing QS as well as in activating translation of *vca0939* encoding a GGDEF domain protein involved in the synthesis of cyclic di-GMP [100]. Moreover, several studies only inferred the role of sRNAs in biofilm formation based on their effect on components of major regulatory networks previously described to either enhance or impair attachment and biofilm formation. Studies lacking such associations were not included, with one exception, because the study in

question provided a possible link between sRNAs and drug tolerance, a hallmark of biofilms involving enhanced resistance to antimicrobial compounds [5]. Although no detailed mechanism was elucidated, and a role of DsrA in biofilm drug tolerance was not determined, the findings by Nishino *et al.* [44] underscore the need to determine sRNA function in relation to biofilm developmental aspects, in particular when considering that biofilm formation requires regulatory cascades that control the temporal and spatial expression of genes. A combination of genetic and molecular techniques used in conjunction with microscopy will be required to elucidate these complex pathways fully, including the identification and characterization of novel sRNAs. Obtaining a complete picture of sRNA regulation during biofilm development will require moving beyond studies focusing on attachment into the full lifecycle of biofilm growth, maturation, and eventual dispersion. Given their abundance in a wide variety of cellular processes including initial biofilm formation, it is very likely that additional sRNAs involved in biofilm growth await discovery.

### Acknowledgment

This work was supported by grants from the National Institutes of Health (R01A107525701, R01AI080710).

### References

1. Gottesman, S. and Storz, G. (2011) Bacterial small RNA regulators: versatile roles and rapidly evolving variations. *Cold Spring Harb. Perspect. Biol.* 3, a003798
2. Beisel, C.L. and Storz, G. (2010) Base pairing small RNAs and their roles in global regulatory networks. *FEMS Microbiol. Rev.* 34, 866–882
3. Costerton, J.W. *et al.* (1995) Microbial biofilms. *Annu. Rev. Microbiol.* 49, 711–745
4. Flemming, H.C. and Wingender, J. (2010) The biofilm matrix. *Nat. Rev. Microbiol.* 8, 623–633
5. Costerton, J.W. *et al.* (1999) Bacterial biofilms: a common cause of persistent infections. *Science* 284, 1318–1322
6. Mah, T.F. and O'Toole, G.A. (2001) Mechanisms of biofilm resistance to antimicrobial agents. *Trends Microbiol.* 9, 34–39
7. Stoodley, P. *et al.* (2002) Biofilms as complex differentiated communities. *Annu. Rev. Microbiol.* 56, 187–209
8. Jackson, D.W. *et al.* (2002) Biofilm formation and dispersal under the influence of the global regulator CsrA of *Escherichia coli*. *J. Bacteriol.* 184, 290–301
9. Romeo, T. *et al.* (1993) Identification and molecular characterization of *csrA*, a pleiotropic gene from *Escherichia coli* that affects glycogen biosynthesis, gluconeogenesis, cell size, and surface properties. *J. Bacteriol.* 175, 4744–4755
10. Thomason, M.K. and Storz, G. (2010) Bacterial antisense RNAs: how many are there and what are they doing? *Annu. Rev. Genet.* 44, 167–188
11. Tsui, H-C.T. *et al.* (1994) Characterization of broadly pleiotropic phenotypes caused by an *hfq* insertion mutation in *Escherichia coli* K-12. *Mol. Microbiol.* 13, 35–49
12. Chao, Y. and Vogel, J. (2010) The role of Hfq in bacterial pathogens. *Curr. Opin. Microbiol.* 13, 24–33
13. Kulesus, R.R. *et al.* (2008) Impact of the RNA chaperone Hfq on the fitness and virulence potential of uropathogenic *Escherichia coli*. *Infect. Immun.* 76, 3019–3026
14. Aiba, H. (2007) Mechanism of RNA silencing by Hfq-binding small RNAs. *Curr. Opin. Microbiol.* 10, 134–139
15. Kuchma, S.L. and O'Toole, G.A. (2000) Surface-induced and biofilm-induced changes in gene expression. *Curr. Opin. Biotechnol.* 11, 429–433
16. Ogasawara, H. *et al.* (2011) Role of the biofilm master regulator CsgD in cross-regulation between biofilm formation and flagellar synthesis. *J. Bacteriol.* 193, 2587–2597

17 Romling, U. *et al.* (1998) Curli fibers are highly conserved between *Salmonella typhimurium* and *Escherichia coli* with respect to operon structure and regulation. *J. Bacteriol.* 180, 722–731
18 Barnhart, M.M. and Chapman, M.R. (2006) Curli biogenesis and function. *Annu. Rev. Microbiol.* 60, 131–147
19 Beloin, C. *et al.* (2008) *Escherichia coli* biofilms. *Curr. Top. Microbiol. Immunol.* 322, 249–289
20 Holmqvist, E. *et al.* (2010) Two antisense RNAs target the transcriptional regulator CsgD to inhibit curli synthesis. *EMBO J.* 29, 1840–1850
21 Urbanowski, M.L. *et al.* (2000) The *gcvB* gene encodes a small untranslated RNA involved in expression of the dipeptide and oligopeptide transport systems in *Escherichia coli*. *Mol. Microbiol.* 37, 856–868
22 Jørgensen, M.G. *et al.* (2012) Small regulatory RNAs control the multi-cellular adhesive lifestyle of *Escherichia coli*. *Mol. Microbiol.* 84, 36–50
23 Hengge-Aronis, R. (2002) Signal transduction and regulatory mechanisms involved in control of the sigma(S) (RpoS) subunit of RNA polymerase. *Microbiol. Mol. Biol. Rev.* 66, 373–395
24 Pesavento, C. *et al.* (2008) Inverse regulatory coordination of motility and curli-mediated adhesion in *Escherichia coli*. *Genes Dev.* 22, 2434–2446
25 Majdalani, N. *et al.* (2001) Regulation of RpoS by a novel small RNA: the characterization of RprA. *Mol. Microbiol.* 39, 1382–1394
26 Mika, F. *et al.* (2012) Targeting of *csgD* by the small regulatory RNA RprA links stationary phase, biofilm formation and cell envelope stress in *Escherichia coli*. *Mol. Microbiol.* 84, 51–65
27 Majdalani, N. *et al.* (2005) Role of RcsF in signaling to the Rcs phosphorelay pathway in *Escherichia coli*. *J. Bacteriol.* 187, 6770–6778
28 Thomason, M.K. *et al.* (2012) A small RNA that regulates motility and biofilm formation in response to changes in nutrient availability in *Escherichia coli*. *Mol. Microbiol.* 84, 17–35
29 Pratt, L.A. and Kolter, R. (1998) Genetic analysis of *Escherichia coli* biofilm formation: roles of flagella, motility, chemotaxis and type I pili. *Mol. Microbiol.* 30, 285–293
30 Romling, U. (2005) Characterization of the rdar morphotype, a multicellular behaviour in Enterobacteriaceae. *Cell. Mol. Life Sci.* 62, 1234–1246
31 Gerstel, U. and Romling, U. (2003) The *csgD* promoter, a control unit for biofilm formation in *Salmonella typhimurium*. *Res. Microbiol.* 154, 659–667
32 Monteiro, C. *et al.* (2012) Hfq and Hfq-dependent small RNAs are major contributors to multicellular development in *Salmonella enterica* serovar Typhimurium. *RNA Biol.* 9, 489–502
33 Papenfort, K. *et al.* (2009) Specific and pleiotropic patterns of mRNA regulation by ArcZ, a conserved, Hfq-dependent small RNA. *Mol. Microbiol.* 74, 139–158
34 Fröhlich, K.S. *et al.* (2012) A conserved RpoS-dependent small RNA controls the synthesis of major porin OmpD. *Nucleic Acids Res.* 40, 3623–3640
35 Boehm, A. and Vogel, J. (2012) The *csgD* mRNA as a hub for signal integration via multiple small RNAs. *Mol. Microbiol.* 84, 1–5
36 De Lay, N. and Gottesman, S. (2012) A complex network of small non-coding RNAs regulate motility in *Escherichia coli*. *Mol. Microbiol.* 86, 524–538
37 Ito, A. *et al.* (2008) Significance of *rpoS* during maturation of *Escherichia coli* biofilms. *Biotechnol. Bioeng.* 99, 1462–1471
38 Adams, J.L. and McLean, R.J. (1999) Impact of *rpoS* deletion on *Escherichia coli* biofilms. *Appl. Environ. Microbiol.* 65, 4285–4287
39 Battesti, A. *et al.* (2011) The RpoS-mediated general stress response in *Escherichia coli*. *Annu. Rev. Microbiol.* 65, 189–213
40 Wang, W. *et al.* (2011) Cooperation of *Escherichia coli* Hfq hexamers in DsrA binding. *Genes Dev.* 25, 2106–2117
41 Mandin, P. and Gottesman, S. (2010) Integrating anaerobic/aerobic sensing and the general stress response through the ArcZ small RNA. *EMBO J.* 29, 3094–3107
42 Sledjeski, D. and Gottesman, S. (1995) A small RNA acts as an antisilencer of the H-NS-silenced *rcsA* gene of *Escherichia coli*. *Proc. Natl. Acad. Sci. U.S.A.* 92, 2003–2007

Review

43 Li, J. *et al.* (2007) Quorum sensing in *Escherichia coli* is signaled by AI-2/LsrR: effects on small RNA and biofilm architecture. *J. Bacteriol.* 189, 6011–6020

44 Nishino, K. *et al.* (2011) Effect of overexpression of small non-coding DsrA RNA on multidrug efflux in *Escherichia coli*. *J. Antimicrob. Chemother.* 66, 291–296

45 Waite, R. *et al.* (2006) Clustering of *Pseudomonas aeruginosa* transcriptomes from planktonic cultures, developing and mature biofilms reveals distinct expression profiles. *BMC Genomics* 7, 162

46 Xu, K.D. *et al.* (2001) Gene expression and protein levels of the stationary phase sigma factor, RpoS, in continuously-fed *Pseudomonas aeruginosa* biofilms. *FEMS Microbiol. Lett.* 199, 67–71

47 Heydorn, A. *et al.* (2002) Statistical analysis of *Pseudomonas aeruginosa* biofilm development: impact of mutations in genes involved in twitching motility, cell-to-cell signaling, and stationary-phase sigma factor expression. *Appl. Environ. Microbiol.* 68, 2008–2017

48 Irie, Y. *et al.* (2010) *Pseudomonas aeruginosa* biofilm matrix polysaccharide Psl is regulated transcriptionally by RpoS and post-transcriptionally by RsmA. *Mol. Microbiol.* 78, 158–172

49 Colvin, K.M. *et al.* (2011) The Pel polysaccharide can serve a structural and protective role in the biofilm matrix of *Pseudomonas aeruginosa*. *PLoS Pathog.* 7, e1001264

50 Wilf, N.M. and Salmond, G.P.C. (2012) The stationary phase sigma factor, RpoS, regulates the production of a carbapenem antibiotic, a bioactive prodigiosin and virulence in the enterobacterial pathogen *Serratia sp.* ATCC 39006. *Microbiology* 158, 648–658

51 Zaitseva, I.V. *et al.* (2010) Involvement of the global regulators GrrS, RpoS, and SplIR in formation of biofilms in *Serratia plymuthica*. *Genetika* 46, 616–621

52 Romeo, T. *et al.* (2012) Post-transcriptional regulation on a global scale: form and function of Csr/Rsm systems. *Environ. Microbiol.* http://dx.doi.org/10.1111/j.1462-2920.2012.02794.x

53 Babitzke, P. and Romeo, T. (2007) CsrB sRNA family: sequestration of RNA-binding regulatory proteins. *Curr. Opin. Microbiol.* 10, 156–163

54 Itoh, Y. *et al.* (2008) Roles of *pgaABCD* genes in synthesis, modification, and export of the *Escherichia coli* biofilm adhesin poly-β-1,6-N-acetyl-D-glucosamine. *J. Bacteriol.* 190, 3670–3680

55 Wang, X. *et al.* (2005) CsrA post-transcriptionally represses *pgaABCD*, responsible for synthesis of a biofilm polysaccharide adhesin of *Escherichia coli*. *Mol. Microbiol.* 56, 1648–1663

56 Wei, B.L. *et al.* (2001) Positive regulation of motility and *flhDC* expression by the RNA-binding protein CsrA of *Escherichia coli*. *Mol. Microbiol.* 40, 245–256

57 Weilbacher, T. *et al.* (2003) A novel sRNA component of the carbon storage regulatory system of *Escherichia coli*. *Mol. Microbiol.* 48, 657–670

58 Suzuki, K. *et al.* (2002) Regulatory circuitry of the CsrA/CsrB and BarA/UvrY systems of *Escherichia coli*. *J. Bacteriol.* 184, 5130–5140

59 Suzuki, K. *et al.* (2006) Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. *Genes Dev.* 20, 2605–2617

60 Jonas, K. *et al.* (2008) The RNA binding protein CsrA controls cyclic di-GMP metabolism by directly regulating the expression of GGDEF proteins. *Mol. Microbiol.* 70, 236–257

61 Heroven, A.K. *et al.* (2008) A Csr-type regulatory system, including small non-coding RNAs, regulates the global virulence regulator RovA of *Yersinia pseudotuberculosis* through RovM. *Mol. Microbiol.* 68, 1179–1195

62 Teplitski, M. *et al.* (2006) Contribution of the SirA regulon to biofilm formation in *Salmonella enterica* serovar Typhimurium. *Microbiology* 152, 3411–3424

63 Lapouge, K. *et al.* (2008) Gac/Rsm signal transduction pathway of gamma-proteobacteria: from RNA recognition to regulation of social behaviour. *Mol. Microbiol.* 67, 241–253

64 Goodman, A.L. *et al.* (2004) A signaling network reciprocally regulates genes associated with acute infection and chronic persistence in *Pseudomonas aeruginosa*. *Dev. Cell* 7, 745–754

65 Brencic, A. and Lory, S. (2009) Determination of the regulon and identification of novel mRNA targets of *Pseudomonas aeruginosa* RsmA. *Mol. Microbiol.* 72, 612–632

66 Ventre, I. *et al.* (2006) Multiple sensors control reciprocal expression of *Pseudomonas aeruginosa* regulatory RNA and virulence genes. *Proc. Natl. Acad. Sci. U.S.A.* 103, 171–176

67 Sonnleitner, E. and Haas, D. (2011) Small RNAs as regulators of primary and secondary metabolism in *Pseudomonas* species. *Appl. Microbiol. Biotechnol.* 91, 63–79

68 Goodman, A.L. *et al.* (2009) Direct interaction between sensor kinase proteins mediates acute and chronic disease phenotypes in a bacterial pathogen. *Genes Dev.* 23, 249–259

69 Petrova, O.E. and Sauer, K. (2010) The novel two-component regulatory system BfiSR regulates biofilm development by controlling the small RNA *rsmZ* through CafA. *J. Bacteriol.* 192, 5275–5288

70 Petrova, O.E. and Sauer, K. (2011) SagS contributes to the motile-sessile switch and acts in concert with BfiSR to enable *Pseudomonas aeruginosa* biofilm formation. *J. Bacteriol.* 193, 6614–6628

71 Hsu, J.L. *et al.* (2008) Characterization of the histidine-containing phosphotransfer protein B-mediated multistep phosphorelay system in *Pseudomonas aeruginosa* PAO1. *J. Biol. Chem.* 283, 9933–9944

72 Lin, C-T. *et al.* (2006) Identification of an HptB-mediated multi-step phosphorelay in *Pseudomonas aeruginosa* PAO1. *Res. Microbiol.* 157, 169–175

73 Moreno, R. *et al.* (2012) Two small RNAs, CrcY and CrcZ, act in concert to sequester the Crc global regulator in *Pseudomonas putida*, modulating catabolite repression. *Mol. Microbiol.* 83, 24–40

74 Linares, J.F. *et al.* (2010) The global regulator Crc modulates metabolism, susceptibility to antibiotics and virulence in *Pseudomonas aeruginosa*. *Environ. Microbiol.* 12, 3196–3212

75 O'Toole, G.A. *et al.* (2000) The global carbon metabolism regulator Crc is a component of a signal transduction pathway required for biofilm development by *Pseudomonas aeruginosa*. *J. Bacteriol.* 182, 425–431

76 Römling, U. and Amikam, D. (2006) Cyclic di-GMP as a second messenger. *Curr. Opin. Microbiol.* 9, 218–228

77 Galperin, M.Y. (2004) Bacterial signal transduction network in a genomic perspective. *Environ. Microbiol.* 6, 552–567

78 Hengge, R. (2009) Principles of c-di-GMP signalling in bacteria. *Nat. Rev. Microbiol.* 7, 263–273

79 McDougald, D. *et al.* (2012) Should we stay or should we go: mechanisms and ecological consequences for biofilm dispersal. *Nat. Rev. Microbiol.* 10, 39–50

80 Weber, H. *et al.* (2006) Cyclic-di-GMP-mediated signalling within the sigma network of *Escherichia coli*. *Mol. Microbiol.* 62, 1014–1034

81 Jonas, K. and Melefors, Ö. (2009) The *Escherichia coli* CsrB and CsrC small RNAs are strongly induced during growth in nutrient-poor medium. *FEMS Microbiol. Lett.* 297, 80–86

82 Jonas, K. *et al.* (2010) Complex regulatory network encompassing the Csr, c-di-GMP and motility systems of *Salmonella typhimurium*. *Environ. Microbiol.* 12, 524–540

83 Miller, M.B. and Bassler, B.L. (2001) Quorum sensing in bacteria. *Annu. Rev. Microbiol.* 55, 165–199

84 Davies, D.G. *et al.* (1998) The involvement of cell-to-cell signals in the development of a bacterial biofilm. *Science* 280, 295–298

85 Diggle, S.P. *et al.* (2003) The *Pseudomonas aeruginosa* quinolone signal molecule overcomes the cell density-dependency of the quorum sensing hierarchy, regulates *rhl*-dependent genes at the onset of stationary phase and can be produced in the absence of LasR. *Mol. Microbiol.* 50, 29–43

86 Diggle, S.P. *et al.* (2006) 4-Quinolone signalling in *Pseudomonas aeruginosa*: old molecules, new perspectives. *Int. J. Med. Microbiol.* 296, 83–91

87 Sonnleitner, E. *et al.* (2011) The small RNA PhrS stimulates synthesis of the *Pseudomonas aeruginosa* quinolone signal. *Mol. Microbiol.* 80, 868–885

88 Rutherford, S.T. *et al.* (2011) AphA and LuxR/HapR reciprocally control quorum sensing in vibrios. *Genes Dev.* 25, 397–408

89 Shao, Y. and Bassler, B.L. (2012) Quorum-sensing non-coding small RNAs use unique pairing regions to differentially control mRNA targets. *Mol. Microbiol.* 83, 599–611

90 Tu, K.C. and Bassler, B.L. (2007) Multiple small RNAs act additively to integrate sensory information and control quorum sensing in *Vibrio harveyi*. *Genes Dev.* 21, 221–233

91 Lenz, D.H. and Bassler, B.L. (2007) The small nucleoid protein Fis is involved in *Vibrio cholerae* quorum sensing. *Mol. Microbiol.* 63, 859–871

TIMI-936; No. of Pages 11

ARTICLE IN PRESS

Review

Trends in Microbiology xxx xxxx, Vol. xxx, No. x

92 Hammer, B.K. and Bassler, B.L. (2003) Quorum sensing controls biofilm formation in *Vibrio cholerae*. Mol. Microbiol. 50, 101–114

93 Kierek, K. and Watnick, P.I. (2003) Environmental determinants of *Vibrio cholerae* biofilm development. Appl. Environ. Microbiol. 69, 5079–5088

94 Lenz, D.H. *et al.* (2005) CsrA and three redundant small RNAs regulate quorum sensing in *Vibrio cholerae*. Mol. Microbiol. 58, 1186–1202

95 Tsou, A.M. *et al.* (2011) The VarS/VarA two-component system modulates the activity of the *Vibrio cholerae* quorum-sensing transcriptional regulator HapR. Microbiology 157, 1620–1628

96 Jang, J. *et al.* (2010) Regulation of hemagglutinin/protease expression by the VarS/VarA–CsrA/B/C/D system in *Vibrio cholerae*. Microb. Pathog. 48, 245–250

97 Coelho, L.R. *et al.* (2008) agr RNAIII divergently regulates glucose-induced biofilm formation in clinical isolates of *Staphylococcus aureus*. Microbiology 154, 3480–3490

98 Xu, L. *et al.* (2006) Role of the luxS quorum-sensing system in biofilm formation and virulence of *Staphylococcus epidermidis*. Infect. Immun. 74, 488–496

99 Felden, B. *et al.* (2011) The *Staphylococcus aureus* RNOME and its commitment to virulence. PLoS Pathog. 7, e1002006

100 Hammer, B.K. and Bassler, B.L. (2007) Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic *Vibrio cholerae*. Proc. Natl. Acad. Sci. U.S.A. 104, 11145–11149

101 Mikkelsen, H. *et al.* (2011) Key two-component regulatory systems that control biofilm formation in *Pseudomonas aeruginosa*. Environ. Microbiol. 13, 1666–1681
