
Molecular Architecture and Assembly of the Eukaryotic Proteasome

Robert J. Tomko, Jr. and Mark Hochstrasser

Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06520; email: robert.tomko@yale.edu, mark.hochstrasser@yale.edu

Annu. Rev. Biochem. 2013. 82:415–45

First published online as a Review in Advance on March 13, 2013

The Annual Review of Biochemistry is online at biochem.annualreviews.org

This article's doi: 10.1146/annurev-biochem-060410-150257

Copyright © 2013 by Annual Reviews. All rights reserved

### Keywords
proteasome, ubiquitin, assembly, ATPase, deubiquitylation, proteolysis

### Abstract
The eukaryotic ubiquitin-proteasome system is responsible for most aspects of regulatory and quality-control protein degradation in cells. Its substrates, which are usually modified by polymers of ubiquitin, are ultimately degraded by the 26S proteasome. This 2.6-MDa protein complex is separated into a barrel-shaped proteolytic 20S core particle (CP) of 28 subunits capped on one or both ends by a 19S regulatory particle (RP) comprising at least 19 subunits. The RP coordinates substrate recognition, removal of substrate polyubiquitin chains, and substrate unfolding and translocation into the CP for degradation. Although many atomic structures of the CP have been determined, the RP has resisted high-resolution analysis. Recently, however, a combination of cryo-electron microscopy, biochemical analysis, and crystal structure determination of several RP subunits has yielded a near-atomic-resolution view of much of the complex. Major new insights into chaperone-assisted proteasome assembly have also recently emerged. Here we review these novel findings.

(Figure 1a). Additional Ubs can also be ligated to the initial Ub via one of its seven lysines (or its N terminus), forming polyubiquitin (polyUb) chains. A chain of four or more Ubs is generally necessary and sufficient to create a targeting signal for delivery to the proteasome (3). The proteasome is responsible for recognizing the substrate, removing its polyUb tag, unfolding the substrate, and cleaving it into short peptides.

In this review, we focus on the structure and assembly of the proteasome, particularly the regulatory particle (RP), as numerous breakthroughs have occurred in these areas over the past several years. For recent, more general reviews of the UPS, we refer the reader to References 1 and 2 and references therein.

## THE 26S PROTEASOME: ANATOMY OF A PROTEIN-DESTROYING MACHINE

The 26S proteasome is the largest and most complex member of an ancient superfamily of ATP-dependent chambered proteases found in all domains of life (4, 5). These proteases are characterized by an ATPase associated with various cellular activities (AAA)+ family ATPase ring responsible for unfolding substrates and threading them through a narrow central pore into an interior proteolytic chamber, which is usually formed by an associated multimeric protein complex (Figure 1b). Sequestration of the proteolytic active sites within such a chamber prevents the unregulated destruction of folded cellular proteins. The ATPase ring converts the chemical energy of ATP hydrolysis into mechanical force for substrate unfolding, allowing a polypeptide end or interior loop to be translocated into the central chamber (Figure 1b). In the eukaryotic 26S proteasome (Figure 2a), the proteolytic chamber is referred to as the 20S core particle (CP), whereas the AAA+ ATPase ring is contained within a separable complex called the RP. In addition to the ATPase ring, the RP contains many additional specialized subunits

### INTRODUCTION

The ubiquitin-proteasome system (UPS) is responsible for the majority of regulatory and quality-control protein degradation in eukaryotic cells. Nearly every cellular process is affected by the UPS (1, 2). The UPS utilizes ATP hydrolysis at several steps to mediate the selective destruction of its substrates. Degradation of a protein by the UPS is typically mediated by the energy-dependent covalent attachment of the small protein ubiquitin (Ub) to one or more lysines within the target protein via the concerted action of three enzymes: E1, E2, and E3

### Ubiquitin (Ub): a 76-residue protein that is covalently conjugated to substrates, often in the form of polymers (polyUb)

### 26S proteasome: an ATP-dependent intracellular protease composed of a 20S proteolytic CP and 19S RP

BI82CH15-Hochstrasser     ARI     2 May 2013     11:36

**Figure 1**

Schematics of the ubiquitin (Ub)-proteasome system and an AAA+ chambered protease. $(a)$ Through the sequential actions of E1, E2, and E3 enzymes, a protein to be degraded is modified with a polyUb chain, which serves as a targeting signal for the proteasome. $(b)$ A cutaway view of an AAA+ chambered protease that displays the path of substrates through the ATPase ring and into the proteolytic chamber. The width of the passage into the protease chamber is delineated by yellow lines, and the proteolytic active sites are shown as purple dots. In this protease, the catalytic chamber is bracketed by two antechambers. Panel $b$ adapted with permission from Nature Publishing Group (142), copyright © 2012.

including ones that function as receptors for the polyUb tag and others that cleave the tag from substrates prior to their degradation.

The 26S proteasome is highly conserved throughout the Eukarya, where it is found in both the nucleus and cytoplasm. Simpler forms of the proteasome have been identified in archaea as well as in actinobacteria, although Ub has been found only in eukaryotes (6, 7). Several other protein complexes can bind the ends of the CP cylinder, but their precise contributions to the regulation of the CP remain less clear (Table 1). Because of space constraints, only RP-CP complexes, i.e., the 26S proteasome, are discussed in this review.

### The 20S Core Particle

The twofold-symmetric CP comprises 28 related polypeptides encoded by 14 separate genes. Atomic structures of the CP from yeast and mammals have been solved (8–12), and their overall structures and subunit arrangements are virtually identical, despite millions of years of evolutionary divergence between these species. The CP consists of four axially stacked heteroheptameric rings (Figure 2b). The inner rings contain seven distinct $\beta$-subunits ($\beta 1-\beta 7$), whereas the outer rings consist of seven different $\alpha$-subunits ($\alpha 1-\alpha 7$). The $\beta 1-$, $\beta 2-$, and $\beta 5$-subunits contain the proteolytic active sites, and each site cleaves preferentially after particular amino acid residues (13). In mammals, four additional $\beta$-subunits have been discovered: $\beta 1i$, $\beta 2i$, $\beta 5i$, and $\beta 5t$, where “$i$” and “$t$” stand for immuno- and thymo-, respectively (14, 15). These subunits are highly expressed in certain immune system tissues or are induced by particular stimuli, such as interferon-$\gamma$ exposure, and they replace the canonical active site-bearing $\beta$-subunits within the CP, altering CP proteolytic specificity. The $\beta 1i-$, $\beta 2i-$, $\beta 5i$-substituted CP, called the immunoproteasome, generates substrate cleavage patterns that enhance loading of peptides onto the class I major histocompatibility complex for immune presentation to killer T cells (16). Thymoproteasomes, which contain $\beta 1i$, $\beta 2i$, and $\beta 5t$, appear to increase the repertoire of self-peptides for positive selection during T cell development in the thymus (15).

In the crystal structures of the isolated CP, the entrances into the internal proteolytic chamber are usually occluded by the

www.annualreviews.org • Proteasome Structure and Assembly 417

BI82CH15-Hochstrasser     ARI     2 May 2013     11:36

a

| 19S RP |
| --- |
| RP lid |

| 20S CP |
| --- |
| CP |

Assembly chaperones:

- **Lid:** Hsp90? Others?
- **Base:** Rpn14, Nas6, Nas2, Hsm3
- **CP:** Ump1, Pba1/2, Pba3/4

b

| α-ring |
| --- |
| β-ring |
| β-ring |
| α-ring |

c

d

---

**Figure 2**

Structure of the 26S proteasome. (a) The cryoelectron microscopy density of the 26S proteasome is shown. The 19S regulatory particle (RP) lid subcomplex is displayed in yellow, the RP base subcomplex in blue, and the 20S core particle (CP) in gray. Known and putative assembly chaperones for each subcomplex are displayed to the right. Figure adapted with permission from Nature Publishing Group (46), copyright © 2011. (b) Space-filling model of the 20S CP atomic structure from yeast [Protein Data Bank (PDB) ID: 1RYP]. (c) A view into the axial pore of the 20S CP from *Thermoplasma acidophilum*. In this model (PDB ID: 3IPM), the archaeal ATPase HbYX motif (*red ribbons*) is inserted into the pockets formed at the interfaces of two adjacent α-subunits, which helps open the CP gate. (d) The HbYX motif binds a critical lysine residue (*yellow*) in the α-pocket.

418 Tomko • Hochstrasser

| Subparticle          | Standardized name | Yeast gene name(s)         | Human standardized name | Function                                      |
|----------------------|------------------|---------------------------|-----------------------|----------------------------------------------|
| α-ring               | α1               | SCL 1                     | PSMA 6                |                                              |
|                      | α2               | PRE 8                     | PSMA 2                |                                              |
|                      | α3               | PRE 9                     | PSMA 4                |                                              |
|                      | α4               | PRE 6                     | PSMA 7                |                                              |
|                      | α5               | DOA 5; PUP 2              | PSMA 5                |                                              |
|                      | α6               | PRE 5                     | PSMA 1                |                                              |
|                      | α7               | PRE 10                    | PSMA 3                |                                              |
| β-ring               | β1               | PRE 3                     | PSMB 6                | Postacidic protease activity               |
|                      | β2               | PUP 1                     | PSMB 7                | Trypsin-like protease activity             |
|                      | β3               | PUP 3                     | PSMB 3                |                                              |
|                      | β4               | PRE 1                     | PSMB 2                |                                              |
|                      | β5               | PRE 2                     | PSMB 5                | Chymotrypsin-like protease activity        |
|                      | β6               | PRE 7                     | PSMB 1                |                                              |
|                      | β7               | PRE 4                     | PSMB 4                |                                              |
| 20S-associated proteins | Pba1            | PBA 1; POC 1              | PAC 1                 | α-ring chaperone                           |
|                      | Pba2             | PBA 2; POC 2; ADD 66       | PAC 2                 | α-ring chaperone                           |
|                      | Pba3             | PBA 3; POC 3; DMP 2; IRC 25 | PAC 3                 | α-ring chaperone                           |
|                      | Pba4             | PBA 4; POC 4; DMP 1        | PAC 4                 | α-ring chaperone                           |
|                      | Ump1             | UMP 1; RNS 2              | POMP; hUMP 1          | β-ring chaperone                           |
|                      | Blm10            | BLM 10                    | PA200                 | CP regulator                               |

N-terminal tails of specific α-subunits (8, 9). peptides suggest that their insertion into the α-
ring pockets induces conformational changes in
the α-ring that displace (at least in part) the α-
ring N-terminal tails from the pore of the CP,
allowing substrate entry (25).

The 19S Regulatory Particle

The RP is responsible for the binding, deubiq-
uitylation, unfolding, and translocation of sub-
strates into the CP as well as the opening of
the CP α-ring gate. As such, the RP functions
as a highly regulated gatekeeper for the major-
ity of proteasome substrates. It contains at least
19 subunits, each usually present in a sin-
gle copy. The RP can split under certain
in vitro conditions into two subcomplexes,
termed the lid and base (Figure 2a). The
base consists of nine subunits: six RP triphos-
phatases, Rpt1-6, and three RP non-ATPases,
Rpn1, 2, and 13. Rpt1-6 are paralogous AAA+

www.annualreviews.org · Proteasome Structure and Assembly 419

ATPases that form a heterohexameric ring (26). The ATPase ring directly contacts the surface of the CP α-ring and exerts upon substrates the ATP-dependent unfolding force that is required for their translocation into the CP for degradation. Rpn1 and Rpn2 are the two largest subunits of the proteasome. The central portions of these subunits are composed of 11 α-helical proteasome/cyclosome (PC) repeats (27), which are thought to form scaffolds onto which substrates and other factors dock (Figure 3). Rpn13, as well as the Rpn10 protein, directly binds Ub, and both proteins function as polyUb-substrate receptors (28–32). In addition to Rpn10 and Rpn13, several extrinsic Ub receptors have been identified, including the UBA-UBL proteins Rad23, Dsk2, and Ddi1 (33–36). The Ub-associated (UBA) domains of these proteins bind the polyUb chains on substrates and shuttle them to the proteasome through interactions of their UBL-like (UBL) domains with the Rpn1 subunit (37, 38).

The lid consists of nine different Rpn subunits, Rpn3, 5–9, 11, 12, and Rpn15 (Sem1). The Rpn11 subunit is a deubiquitylating

**Proteasome/cyclosome (PC) repeats:** α-helical repeats found in proteasome subunits Rpn1 and Rpn2 and in the APC/cyclosome that fold into a toroid or superhelix

**Ubiquitin-associated (UBA) domain:** a conserved domain found in many Ub-binding proteins that directly recognizes ubiquitin

![Image Description](image.png)

**Rpn13**

**Rpn2**

**Rpn10**

**Rpn1**

**Rpt1–6**

**Figure 3**

Organization of the regulatory particle, including atomic and pseudoatomic models of base subunits. A bird’s-eye view of the 26S proteasome electron microscopy (EM) density is shown. Where available, the atomic or pseudoatomic models of Rpt1–6 [modeled using Protein Data Bank (PDB) ID: 3H4M, 3H43], Rpn2 (PDB ID: 4ADY), Rpn10 von Willebrand domain (PDB ID: 2X5N), and Rpn13 pleckstrin-like receptor for ubiquitin domain (PDB ID: 2R2Y) are shown and colored according to their respective EM densities. Figure adapted with permission from *Proceedings of the National Academy of Science of the U.S.A.* (47), copyright © 2012.
enzyme (DUB) (39, 40). The lid is structurally related to the COP9 signalosome (CSN) and the eIF3 translation initiation complexes (41, 42). Although the compositions of the CSN and eIF3 vary somewhat across the Eukarya (43, 44), the canonical form of each of these complexes contains six subunits with proteasome/CSN/initiation complex (PCI) domains and two subunits with Mpr1/Pad1/N-terminal (MPN) domains. The composition of the lid appears invariant in eukaryotes, consisting of the canonical six PCI proteins (Rpn3, 5, 6, 7, 9, and 12) and two MPN subunits (Rpn8 and Rpn11) as well as Sem1, a small acidic protein with neither PCI nor MPN domains.

### ARCHITECTURE OF THE 19S REGULATORY PARTICLE

An unprecedented view of RP structure and subunit architecture has recently emerged. Although the overall shape of the RP had been observed long ago by electron microscopy (EM) (45), the positions of its subunits and their high-resolution structures were unknown. Over the past three years, a series of high-resolution cryo-EM-based reconstructions, along with biochemical experiments and crystal-structure determinations for several isolated subunits, has yielded a much clearer picture of RP architecture (Figure 3).

Surprisingly, the RP lid and base terminology (41) turns out to be somewhat misleading. The base includes subunits positioned farther from the CP than any lid subunit (46–48). Rather than the lid capping the base, it instead straddles the long edge of the RP, contacting both the base and the CP α-ring (Figure 2a). Nevertheless, to avoid confusion, we continue to use the lid and base terms here. We describe the new proteasome structural data in the following sections.

### The Regulatory Particle Base

As noted earlier, the RP base includes a heterohexamer of AAA+ ATPases and three non-ATPase subunits. The new structural data

---

have revealed important details about the three-dimensional structures and interactions of many of these subunits. The data also show unexpected features regarding how the base interacts with the CP and RP lid.

**The Rpt ATPases.** As expected from structures of other AAA+ chambered proteases (5), the Rpt ATPase ring directly abuts the ends of the CP cylinder. However, the new EM models unexpectedly indicate that the pore of the ATPase ring does not align perfectly with the pore of the CP, as it is axially offset by as much as 30 Å and tilted on its vertical axis by approximately 5–10° (49, 50). The functional significance of this remains unknown. The six ATPases adopt a single arrangement in normal cells: Rpt1-Rpt2-Rpt6-Rpt3-Rpt4-Rpt5 (26). Although this arrangement was originally predicted on the basis of sequence divergence among the proteasomal ATPases (51) and later by molecular modeling approaches (52), several alternative arrangements had been proposed on the basis of yeast two-hybrid (53), chemical cross-linking (54), and other (55) studies. Further, whether the ATPases normally adopt multiple arrangements had been unclear. The Rpt1-2-6-3-4-5 arrangement was unambiguously established only recently using engineered disulfide cross-linking (26); this analysis was guided by the structure of an archaeal homohexameric ATPase ring, proteasome-activating nucleotidase (PAN) (56, 57). The success of this approach implies a close evolutionary conservation of the core ATPase ring architecture.

The Rpts, PAN, and the actinobacterial ortholog ATPase forming ring-shaped complexes (ARC) share a similar domain organization (Figure 4a) (56, 57). Each contains an N-terminal region that includes a coiled-coil domain (CC), an oligonucleotide/oligosaccharide-binding (OB) domain, the AAA+ ATPase domain, and a C-terminal α-helical domain that sits on the outer surface of the ATPase ring. In AAA+ ATPases, the protein sequence N-terminal to the ATPase domain is typically specialized for its particular

---

**Ubiquitin-like (UBL) domain:** a protein domain that adopts a β-grasp fold similar to ubiquitin; this domain often mediates interaction with Ub-binding proteins

**Deubiquitylating enzyme (DUB):** a specialized protease that cleaves Ub from substrates such as other proteins and Ub precursors

**Proteasome/COP9 signalosome/initiation complex (PCI) domain:** a conserved domain with an N-terminal α-helical region followed by a winged-helix fold that is found in six RP lid subunits

**Mpr1/Pad1/N-terminal (MPN) domain:** a conserved fold found in subunits of the RP lid and COP9 signalosome; the MPN+/JAMM variant has metalloprotease activity

www.annualreviews.org • Proteasome Structure and Assembly 421
a

| PAN | 1 | Coiled coil | P | OB fold | AAA+ ATPase core | C-terminal domain |
| --- | --- | --- | --- | --- | --- | --- |

b

c

Figure 4

Proteasomal AAA+ ATPase structure and hexameric ring organization. (a) The domain architecture of the Methanocaldococcus jannaschii proteasome-activating nucleotide (PAN) is shown. The eukaryotic ATPases Rpt1–6 share a similar domain organization as PAN. (b) A pseudoatomic model of the proteasomal ATPase ring (modeled as in Figure 3) is shown. Domains are colored as in panel $a$. (c) Arrangement of the eukaryotic ATPases in the heterohexameric ring. The inferred proline cis/trans isomerism of each Rpt subunit is listed. Subunits forming pairs in the trimer of dimers model are similarly colored. Abbreviations: OB, oligonucleotide/oligosaccharide-binding; P, proline.

function, and the combination of N-terminal CC and OB domains appears unique to the proteasomal ATPases. In PAN and ARC, there is a critical and highly conserved proline residue at the junction between the N-terminal CC region and the OB domain (Figure 4a) (56, 57). Proline is unusual in that the peptide bond it forms with the preceding residue can adopt either a cis or trans configuration. In the crystal structures of PAN N domains, this peptide bond alternates between cis and trans conformations in adjacent subunits, allowing their N-terminal helices to form pairwise CCs. The ATPase subunits in the PAN crystal structure thus form a trimer of dimers (Figure 4b) (56, 57). This cis-trans pairing of ATPases appears to be conserved in the eukaryotic ATPase ring: Rpt2, Rpt3, and Rpt5 contain highly conserved prolines that align with the critical PAN proline (26, 56, 57) and take up alternating places within the ATPase ring

(Figure 4c). In agreement, many isolated base subcomplexes, which are thought to be assembly intermediates, contain pairs of Rpt subunits, and in each case they are a predicted cis-trans pair.

**Rpn1 and Rpn2.** Until recently, the structures and positions of Rpn1 and Rpn2 within the RP remained unclear and have been subject to extensive debate. Most investigators expected the PC repeats of these proteins to take on toroidal or horseshoe-like superfolds (58, 59); some further proposed that these subunits were nestled inside the pore of the ATPase ring (or perhaps stacked on top) (58, 60). However, the strong structural similarity of the eukaryotic ATPase ring compared with PAN argued against this latter possibility because the central pore of PAN is extremely narrow and unable to accommodate a large folded domain even at its widest point (26, 52, 56, 57).
He et al. (27) recently determined the crystal structure of yeast Rpn2. It resembles a tobacco pipe (Figure 5a), with a central region of 11 PC repeats that make up the pipe's barrel that is bracketed by an extended N-terminal α-helical domain comprising the stem and a small C-terminal globular domain that packs against the barrel. Although the PC repeats of Rpn2 form a toroid as predicted, their detailed arrangement was quite unexpected: They adopt a tightly packed shape in which the pore formed by the PC repeats is plugged by two α-helices extending into the toroid (Figure 5b). This packing explains the highly hydrophobic character of these two central helices, as they are largely sequestered from solvent. Thus, the toroidal PC repeat domain of Rpn2 is very densely packed and is likely quite rigid. The molecular model of Rpn2 is readily docked into the RPEM density owing to its large size and distinctive shape (Figure 3). Rpn2 is one of the most distally positioned subunits along the proteasomal long axis. Both its N-terminal stem-like extension and part of the toroidal domain interact with the N termini of Rpt3 and Rpt6, whereas the center of the toroid and the C-terminal domain (CTD) are positioned over the pore of the ATPase ring (48).

Comparison of purified Rpn2 and Rpn1 by EM indicates that these two proteins adopt very similar folds, consistent with their ~20% sequence identity (27, 58). In the 26S proteasome cryo-EM structures, Rpn1 also harbors a toroidal domain, presumably encoded by its PC repeats. Notably, all known docking sites of the extrinsic Ub receptors lie within this toroid (37, 38). Thus, the Rpn1 (and perhaps Rpn2) toroids likely serve as loading platforms for incoming substrates or other proteins. In contrast to Rpn2, Rpn1 makes extensive contacts with the outside face of the ATPase ring in the 26S holoenzyme EM structures (Figure 3) (46–48), and its density is somewhat variable, suggesting it may undergo movement. The close association of Rpn1 with the ATPase ring may help to control substrate docking by coupling conformational changes in Rpn1 with the binding and/or hydrolysis of ATP by the ATPases.

---

**Rpn10, Rpn13, and the extrinsic shuttle factors.** Two general pathways exist for the delivery of ubiquitylated substrates to the proteasome: Ub recognition by the intrinsic receptors Rpn10 and Rpn13 or shuttling to the proteasome via the extrinsic receptors, which include Rad23, Dsk2, and Ddi1 (Figure 5c). Whereas the cellular protein levels of the extrinsic and intrinsic receptors are very similar, at least in yeast (61), the extrinsic receptors are generally substoichiometric on purified proteasomes. In contrast, the occupancy of the intrinsic receptors Rpn10 and Rpn13 is typically greater than 50% (62). As the extrinsic receptors may bind the proteasome transiently, this low apparent occupancy could potentially reflect their steady-state binding levels in vivo. Rpn13 interacts with a short peptide segment in the CTD of Rpn2, positioning it far from the CP (Figure 3) (27, 63). Rpn13 adopts a pleckstrin homology domain-like fold (Figure 3) named the pleckstrin-like receptor for Ub (PRU) (28). Rpn13 binds Ub chains with high affinity (~90 nMK<sub>D</sub> for Lys48-linked diUb) via loops extending from the β-strands comprising the PRU domain (Figure 5d). Like Rpn13, Rpn10 directly binds Ub, but in this case via its α-helical Ub-interacting motif (UIM) (Figure 5d). Rpn10 contains one to three UIMs at its C terminus depending on the species, but even Rpn10 orthologs with a single UIM prefer polyUb chains over monoUb (64, 65). Solution-structure studies indicate that the UIM region of Rpn10 is highly flexible (64), and this domain appears as a diffuse, poorly resolved density in EM structures of the 26S proteasome (62). In contrast, the N-terminal region of Rpn10 consists of a tightly folded von Willebrand factor A (vWA) domain that is readily resolved in these structures (Figure 3) (62, 66). Despite findings that loss of Rpn10 or disruption of its vWA domain destabilizes lid-base association within the RP (41, 53, 67), the vWA domain makes extensive contact only with subunits in the lid (46–48), suggesting it functions indirectly to promote or maintain RP stability (see Lid-base association and Rpn10 incorporation, below).

---

**Pleckstrin-like receptor for ubiquitin (PRU) domain:** domain found in the proteasome Ub receptor Rpn13; in many species, Rpn13 has a domain that binds the UCH37 DUB

**Ubiquitin-interacting motif (UIM):** a hydrophobic, α-helical motif that binds Ub and was originally identified in the proteasome Ub receptor Rpn10

www.annualreviews.org • Proteasome Structure and Assembly 423
BI82CH15-Hochstrasser     ARI     2 May 2013     11:36

a
![Image Description](image_a.png)

b
![Image Description](image_b.png)

c
| Rpn10 | 1 | vWA | UIM | 268 |
| --- | --- | --- | --- | --- |
| Rpn13 | 1 | PRU | 156 |
| Rad23 | 1 | UBL | UBA | UBA | 398 |
| Dsk2 | 1 | UBL | UBA | 373 |
| Ddi1 | 1 | UBL | UBA | 428 |

d
![Image Description](image_d.png)

**Figure 5**

The architecture of the 19S base scaffold subunit Rpn2 and ubiquitin (Ub) receptor subunits. (a) Ribbon structure of Rpn2 from *Saccharomyces cerevisiae* [Protein Data Bank (PDB) ID: 4ADY]. The N-terminal helical domain is colored blue, the toroidal domain green, and the C-terminal domain red. (b) An axial view of the Rpn2 toroid illustrating the central α-helices. Proteasome/cyclosome repeats are colored blue (N-terminal) through orange (C-terminal) with the central helices in red. (c) Domain organization of the intrinsic Ub receptors Rpn10 and Rpn13 and the extrinsic receptors Rad23, Dsk2, and Ddi1. Domains and amino acid numbering are according to the *S. cerevisiae* gene products. (d) Distinct modes of interaction with Ub are utilized by each Ub receptor. Amino acids of each Ub-binding domain (*gray*) that contact Ub (*green*) are highlighted in red (PDB ID: 2Z59, 2KDE, and 1WR1 for PRU, UIM, and UBA, respectively).

Abbreviations: PRU, Pleckstrin-like receptor for ubiquitin domain; UBA, ubiquitin-associated domain; UBL, ubiquitin-like domain; UIM, ubiquitin-interacting motif; vWA, von Willebrand factor A domain.

424 Tomko • Hochstrasser

BI82CH15-Hochstrasser     ARI     2 May 2013     11:36

# The Regulatory Particle Lid

EM structures are now available for the RP lid as an isolated complex (46) and in the context of the 26S holoenzyme (46–48). The lid appears to act, in part, as a clamp to keep the RP and CP together (68). The lid looks similar to a horseshoe, with the arc formed by the PCI domains of (from end to end) Rpn9, 5, 6, 7, 3, and 12 (Figure 6). The N-terminal helical extensions of each PCI subunit extend radially away from the horseshoe, much like fingers on a hand. Within the 26S proteasome, the Rpn5 and Rpn6 N termini extend toward the CP and make extensive contact with the outer flanks of both the RP base and the CP (46, 68), suggesting the potential for coupling between the enzymatic activities of the lid and the rest of the proteasome. In the center of the horseshoe are the two MPN subunits, Rpn8 and Rpn11, which sit side by side to form a heterodimer positioned over the ATPase ring in the 26S holoenzyme (46, 69). Each MPN subunit makes extensive contact with several PCI subunits. The MPN subunit N termini appear to be flexible or disordered in the EM density of the isolated lid but become more ordered in the 26S proteasome (46), suggesting that their positioning is constrained upon their incorporation into the RP. Similarly, large movements in Rpn5 and Rpn3 likely occur as the lid is integrated into the RP (or 26S); these movements may be important for accommodation of base subunits upon RP assembly (46). Although the exact position of Sem1 in the lid has not been determined, it has been found in a complex with Rpn3 and Rpn7 (67), indicating it directly binds one or both of these subunits.

## STRUCTURAL INSIGHTS INTO SUBSTRATE SELECTION AND DEGRADATION

### Substrate Recognition and Positioning for Degradation

With a nearly complete three-dimensional model of the RP at subnanometer resolution, it now becomes apparent that the placement

---

**Figure 6**

Molecular architecture of the proteasome lid. The electron microscopy structure of the purified lid complex and the densities contributed by each subunit are shown. Figure adapted with permission from Nature Publishing Group (46), copyright © 2012.

of subunits and proteasome-associated factors within the RP is likely organized in a way that facilitates movement of the substrate from the outer edges of the RP toward the CP for degradation. The subunits responsible for each successive step in substrate manipulation (i.e., recognition, deubiquitylation, unfolding, and translocation into the CP) are positioned sequentially closer to the CP pore. At the periphery of the RP are the Ub receptors Rpn10 and Rpn13. Both are located above the ATPase ring (below the ring is defined as the CP-binding surface) with an unobstructed potential substrate path toward the ATPase pore. Rpn13 is positioned above Rpt1 and Rpt2, whereas affinity labeling of the Rpn10 UIM domain indicates this region projects internally toward the Rpt4/Rpt5 CC (62). Given the placement of the Ub receptors, one might hypothesize that the CCs of specific pairs of Rpt subunits are responsible for binding substrates that dock at the proteasome via particular Ub receptors.

In contrast to Rpn10 and Rpn13, the positions and orientations of the shuttle factors upon binding the proteasome are unknown. Interpretation of how they dock onto the proteasome is complicated by their interaction with multiple subunits within the RP and the variability of Rpn1 density within EM images (37, 64, 65, 70). Although Rpn1 is considered the major binding site for Rad23, Dsk2, and Ddi1, the UBL domains of the human orthologs of Rad23, Dsk2, and Ddi1 can bind the UIM and PRU domains of hsRpn10 and

---

**Shuttle factor:** a polyUb-binding protein that also binds the proteasome RP and functions to deliver polyUb-modified substrates to the proteasome

www.annualreviews.org • Proteasome Structure and Assembly
425

hsRpn13 (29, 65). The physiological significance of such shuttle factor UBL binding to Rpn10 and Rpn13 and whether these receptors cooperate with Rpn1 to process substrates bound to the shuttle factors remain to be explored. Deletions of the shuttle factor genes display synthetic genetic defects when combined with *RPN10* or *RPN13* deletions (29), which argues against a model in which the shuttle factors function exclusively by handing off substrates to the intrinsic receptors.

### Substrate Deubiquitylation

After a substrate binds the proteasome, its polyUb tag must be removed either prior to or potentially during substrate degradation. The proteasome harbors several DUBs: two less tightly associated DUBs [Ubp6 (Usp14 in humans) and Uch37 (not in all species)] and the Rpn11 subunit of the RP lid (Table 2) (39, 40). Additionally, several Ub ligases, most notably Hul5, are found associated with proteasomes, strongly suggesting that the polyUb chains of substrates can undergo remodeling on the proteasome, perhaps as a substrate triage or quality-control step (71). Whereas the nonintegral proteasomal DUBs trim polyUb chains from the distal end, Rpn11 instead removes whole chains at their point of attachment to the substrate. This observation and the severe growth phenotypes associated with attenuation of Rpn11 catalytic activity strongly suggest it is the primary DUB responsible for removal of polyUb chains immediately prior to or concomitant with the initiation of protein unfolding and degradation by the proteasome.

The catalytic metalloprotease domain of Rpn11 is related to the structurally determined MPN+/JAMM (JAB1/MPN/Mov34 metalloenzyme) domain of AMSH (associated molecule with SH3 domain of STAM), a functionally distinct DUB. Modeling of the atomic structure of the AMSH MPN+/JAMM domain into the EM density of Rpn11 in the 26S proteasome suggests that the Rpn11 active site is positioned 10–20 Å over the pore of the ATPase ring (46), and near-atomic-resolution

EM recently confirmed this positioning (69). This proximity to the ATPase pore suggests a model in which substrates are unfolded until the polyUb chain-attachment site is pulled near the pore of the ATPase ring, bringing it into position for deubiquitylation by Rpn11. Conversely, positioning of the polyUb chain-attachment site and/or deubiquitylation may trigger the initiation of unfolding and threading through the ATPase ring. The first model is supported by the finding that unfolding generally requires an unstructured or structurally destabilized region for initiation (72). Furthermore, in the case of substrates with polyUb chains attached at multiple sites, Rpn11 may need to act multiple times on the substrate; in this case, the threading action of the ATPases on the substrate could pull successive polyUb attachment sites into the Rpn11 active site during unfolding and translocation.

### Substrate Unfolding by the AAA+ ATPase Ring

Many inferences about the function and mechanism of the proteasomal ATPase ring have been drawn from the simpler homomeric ATPases from archaea and bacteria. Like the eukaryotic ATPases, PAN forms a hexameric ring that binds the archaeal CP in an ATP-dependent manner (73). Using PAN as a model, investigators deduced recently that the ATPase ring can bind a maximum of four nucleotides (ATP or ADP) at once, with the remaining two sites being empty owing to negative allostery (74). Based on this finding and information about the relative rates of ATP binding and hydrolysis, it appears that nucleotides tend to bind to the ATPase ring and be hydrolyzed in pairs by subunits *para* to one another in the six-membered ring.

Although PAN has been a powerful model for elucidating some of the basic features of the proteasomal ATPases, emerging structural, biochemical, and genetic data suggest that the eukaryotic ATPase ring diverges from PAN in several ways. Aside from the obvious diversification of eukaryotic Rpt sequences
compared with PAN, a recent EM analysis of  
the 26S holoenzyme suggests that the ATPases  
actually take up a corkscrew-like arrangement  
similar to that seen in some ATP-dependent  
helicases, in which the pore residues form  
a downward-spiraling staircase, whereas the  
N-terminal OB-fold ring and the small CTDs  
of the ATPases remain largely planar (46, 69).  
This observation raises the possibility that in-  
stead of, or in addition to, the mechanical forces  
from the ATPases being exerted through small,  
local movements of pore loops (as has been  
proposed for PAN) (75), substrate unfolding  
may be driven by movements of the entire large  
domains if these domains were able to proceed  
sequentially through the positions observed in  
the staircase arrangement (46). Biochemical  
and phenotypic assays comparing mutations  
at paralogous positions in each eukaryotic  
ATPase subunit have unveiled Rpt-specific de-  
fects in substrate unfolding, CP gate opening,  
and resistance to proteasome stresses (76, 77).  
We do not yet know whether these apparently  
specialized roles of the ATPases reflect a mech-  
anism allowing processing of substrates with  
more diverse sequences and structures than  
those processed by the archaeal and bacterial  
ATPases, or if instead this functional diversifi-  
cation helps to couple unfolding and proteolysis  
to the activity of the proteasomal lid, which is  
not present in these simpler proteasomes.

### ASSEMBLY OF THE 26S  
PROTEASOME  

The proteasomes of archaea and actinobacte-  
ria are structurally much simpler than those of  
eukaryotes (78). In most species, a single type  
of α-subunit, β-subunit, and ATPase subunit  
is predicted. Thus, assembly of these composi-  
tionally simple proteasomes requires only that  
the correct number of subunits be placed in each  
ring and that the rings stack properly. In con-  
trast, eukaryotic proteasomes contain seven dis-  
tinct copies of each α- and each β-subunit, and  
there are six different ATPases. Each subunit  
shows high similarity to its paralogs, but it must  
typically occupy a single, defined site within the  

final structure. Thus, controlling the relative  
position of each paralogous subunit to form the  
appropriate subunit arrangement is necessary.  
Similarly, the task of finding the appropriate  
register between each ring and its neighboring  
rings is more complicated than in the simpler  
prokaryotic proteasomes, as the rotational sym-  
metry within each ring is broken.  

In many cases, a specific order of subunit in-  
corporations into assembly intermediates may  
be important to prevent competition for similar  
binding sites among subunits or structural oc-  
clusion of binding sites during assembly. These  
alternative or incomplete arrangements could  
stall proteasome biogenesis or lead to dead-end  
assembly products. This may be especially true  
in the eukaryotic RP, which contains 13 non-  
ATPase subunits not present in simpler protea-  
somes. Finally, the high level of coordination  
of multiple enzymes within the proteasome re-  
quires that the subunits harboring enzymatic  
activity be restrained until the proteasome  
is fully assembled to prevent the decoupling  
of deubiquitylation, substrate unfolding, and  
proteolysis.  

Eukaryotic cells have evolved several strate-  
gies to meet these demands. These generally  
fall into three categories: intrinsic regulatory  
elements such as certain CP β-subunit ap-  
pendages, extrinsic assembly chaperones that  
control the assembly of specific proteasomal  
subcomplexes, and hierarchical assembly mech-  
anisms that act to regulate the order in which  
specific subunits or subcomplexes of the pro-  
teasome associate (78, 79). We present an  
overview of 20S CP (Figure 7) and 19S RP  
(Figures 8 and 9) assembly and highlight ex-  
amples of how these strategies are utilized in  
proteasome biogenesis.  

### 20S Core Particle Assembly  

Assembly of the eukaryotic CP initiates with the  
formation of the heptameric α-ring (Figure 7).  
The exact order of α-subunit incorporation is  
unknown, and the possibility of assembling α-  
rings with aberrant subunit arrangements (e.g.,  
homohexameric rings) is well known (80–82).  
| Subparticle | Standardized name | Human name | Other mammalian names | Activity/domains | Function |
|--------------|-------------------|-------------|-----------------------|------------------|----------|
| Base         | Rpt1              | PSMC2       | S7; p48               | AAA+ATPase       | Unfoldase |
|              | Rpt2              | PSMCI       | S4; p56               | AAA+ATPase       | Unfoldase |
|              | Rpt3              | PSMC4       | S6; S6b; p47           | AAA+ATPase       | Unfoldase |
|              | Rpt4              | PSMC6       | S10b; p42              | AAA+ATPase       | Unfoldase |
|              | Rpt5              | PSMC3       | S6'; s6a; p50          | AAA+ATPase       | Unfoldase |
|              | Rpt6              | PSMC5       | S8; p45                | Ub receptor      | S1; p97; PSMD1 |
|              | Rpn1              | RPNI; HRD2; NASI | S2; p112             | Rpn13 docking    | S2; p112; PSMD2 |
|              | Rpn2              | RPNI0; MCB1; SUNI | S1; p97              | Ub receptor      | S5a; p54; PSMD4 |
|              | Rpn3              | RPNI3       | AD RM1                | Ub receptor; Uch37 receptor; docking | AD RM1 |
| Lid          | Rpn5              | PSMD3       | S3; p58               | PCI domain       | Scaffold? |
|              | Rpn6              | PSMD12      | p55                   | PCI domain       | Scaffold? |
|              | Rpn7              | PSMD11      | S9; p44.5             | PCI domain       | Scaffold? |
|              | Rpn8              | PSMD6       | S10a; p44             | PCI domain       | Scaffold? |
|              | Rpn9              | PSMD7       | S12; p40; MOV34        | MPN domain       | S12; p40; MOV34; PSMD7 |
|              | Rpnll             | PSMD4       | S11; p40.5            | PCI domain       | Scaffold? |
|              | Rpn12             | PSMD8       | S14; p31              | Largely unstructured | Scaffold? |
|              | Seml              | PSMD1       | S15; p27L; DSS1        | MPN+domain; DUB activity | Substrate deubiquitylation |
|              |                   | PSMD14; Padl; |                      |                  | S13; PSMD14; Padl; Pohl |
|              |                   | Dssl; PSMD9 |                      |                  | S14; p31; PSMD8 |
|              |                   | S15; p27L; Rpn15; |                      |                  | Dssl; PSMD9 |
|              |                   |             |                      |                  | (Continued) |

| RP-associated | Dsk2 | PLIC-2 | UBL domain; UBA domain | Extrinsic Ub receptor |
| --- | --- | --- | --- | --- |
| proteins | Rad23 | hHR23b | UBL domain; UBA domain | Extrinsic Ub receptor |
|  | Ddi1 | DDII; VSMI | UBL domain; UBA domain | Extrinsic Ub receptor |
|  | Ubp6 | USP14 | UBL domain; DUB activity | pUb chain editing |
|  | Uch37 | No ortholog | UCH domain; DUB activity | pUb chain editing |
|  | Hul5 | HUL5 | HECT family E3 ligase activity | E3 ligase; substrate triage? |
|  | Ecm29 | KIAA10/E3a | HEAT repeats | Proteasome stabilizer? |
|  | Spg5 | ECM29 | No obvious ortholog | Unknown |
|  | Rac1 | PSMD10 | p28; gankyrin | Ankryin repeats |
|  | Rac2 | NAS6 | PSMD9 | p27; BRIDGE | PDZ domain |
|  | Rac3 | NAS2 | S5b | HEAT repeats |
|  | Rac4 | HSM3 | PAAF1 | WD-40 domain |
|  |  | RPN14 |  | RP assembly chaperone |
|  |  |  | RP assembly chaperone |
|  |  |  | RP assembly chaperone |

Figure 7

The 20S core particle assembly pathway.

β-subunits, several of which are synthesized as precursors with N-terminal propeptides, then add to the α-ring in an ordered fashion, creating a half-proteasome (83–86). Upon dimerization of two half-proteasomes to form the preholoproteasome (PHP), the active site-bearing β-subunits undergo autocatalytic cleavage of their propeptides, generating the mature catalytic sites and completing CP assembly (87).

At least three dedicated chaperone factors facilitate CP assembly. Two heterodimeric chaperone complexes participate in the assembly of the α-ring: proteasome biogenesis-associated 1/2 (Pba1/2) and Pba3/4 (PAC1/2 and PAC3/4 in mammals) (22, 88–92). Pba1/2 is a heterodimer that can bind isolated α-subunits (89) and associates with proteasomal precursors but not the mature CP in vivo, suggesting that it is released upon maturation of the catalytic active sites (22). Pba1 and Pba2 both contain HbYX motifs and bind to the α-ring similarly to proteasomal activators (22, 93). However, Pba1/2 does not enhance proteolytic activity (89). The interaction of Pba1/2 with the outer face of the α-ring suggests it may prevent premature docking of activators to the assembling CP. In mammalian cells, knockdown of PAC1/2 causes accumulation of large particles containing α-subunits but no β-subunits (89). This observation led to the proposal that Pba1/2 also functions to prevent the formation of aberrant α-ring dimers that might stall proper proteasome biogenesis.

Pba3 and Pba4 also form a heterodimer, which binds to α5 in vitro (88, 95). A crystal structure of Pba3/4 with α5 revealed that instead of binding to the outer α-ring surface, Pba3/4 instead binds to the inner surface recognized by the β-subunits (88). Modeling suggested that Pba3/4 would clash sterically with the incoming β4-subunit, indicating that Pba3/4 is likely released prior to or concomitant with β4 incorporation (Figure 7).

Whereas *PBA3/4* deletions enhanced the defects caused by a point mutation in α5, they did not exacerbate defects associated with loss of the α3-subunit (95). This suggested that Pba3/4 function was no longer required in cells lacking α3. In α3Δ yeast, a second copy of α4 fills the slot normally occupied by α3 (96). Deletion of *PBA4* resulted in a large fraction of CPs bearing the alternative α4-α4

BI82CH15-Hochstrasser ARI 2 May 2013 11:36

a

+Rpn2,  
Rpn13  

Base precursor  

b  

c  

d  

Rpt1-C  

Hsm3  

Nas6  

Rpt3-C  

Rpn14  

Figure 8  

Assembly of the regulatory particle (RP) base is mediated by RP assembly chaperones. (a) The RP base assembly pathway in yeast. (b,c) Interaction between (b) *Saccharomyces cerevisiae* Nas6 and the C domain of Rpt3 [Protein Data Bank (PDB) ID: 2DZN] and (c) *S. cerevisiae* Hsm3 and the Rpt1 C domain (PDB ID: 3VLF). C domains are similarly oriented. (d) Atomic structure of *S. cerevisiae* Rpn14 (PDB: 3ACP).

www.annualreviews.org • Proteasome Structure and Assembly 431

BI82CH15-Hochstrasser     ARI    2 May 2013    11:36

a

Rpn5, 8, 9, 11

Module 1

Rpn3, 7; 15/Sem1

LP3

LP2

Lid

b

ATPase ring

Rpn2

Rpn11

Rpn10

Rpn12

Figure 9

Assembly and attachment of the regulatory particle (RP) lid. $(a)$ The RP lid assembly pathway inferred from studies in yeast. Mpr1/Pad1/N-terminal subunits are shown in purple, proteasome/CSN/initiation complex domain subunits are shown in peach, and Sem1 is shown in light blue. $(b)$ A model for the stabilization of lid-base interaction by Rpn10 and the Rpn12 C-terminal tail via interaction with Rpn2 during RP assembly. In this model, Rpn10 positions Rpn11 to interact with Rpn2 (*curved arrow*), whereas the C-terminal tail of Rpn12 directly contacts the Rpn2 N-terminal extension (*straight arrow*). The N and C termini of Rpn2 are marked.

configuration. Thus, Pba3/4 functions, at least in part, to ensure that $\alpha 3$ is inserted into its appropriate place in the $\alpha$-ring. Intriguingly, cells with increased amounts of $\alpha 4$-$\alpha 4$ proteasomes display a growth advantage under certain stressors, suggesting a potential physiological role for these alternative proteasomes.

Upon completion of the $\alpha$-ring, $\beta$-subunits assemble onto the inner surface of the $\alpha$-ring. Initiation of $\beta$-ring assembly is accompanied by the binding of Ump1 to the assembly intermediate, at least in mammalian cells (97). Ump1 may help prevent the premature dimerization of precursors containing incomplete sets of $\beta$-subunits (91, 98, 99). Completion of the $\beta$-ring overcomes this Ump1-dependent checkpoint (91, 99) and is followed closely by dimerization of half-proteasomes and degradation of Ump1 upon maturation of the CP catalytic sites (98). Systematic knockdown of $\beta$-subunits in mammalian cells suggested that $\beta$-subunit addition begins with $\beta 2$ and proceeds in a defined order, although the timing of $\beta 1$ addition was ambiguous (97). Ump1 was already associated with the $\alpha$-ring in $\beta 3$ knockdown cells in which the only

$\beta$-subunit present on the $\alpha$-ring is $\beta 2$, suggesting Ump1 binds with or before $\beta 2$. The ordered association of $\beta$-subunits is driven, at least in part, by the propeptides and peptide tails of certain $\beta$-subunits. Both the $\beta 2$ propeptide and C-terminal tail (which wraps around $\beta 3$) are required for efficient incorporation of the $\beta 3$-subunit in mammals (and probably also yeast) (97, 100). Similarly, the $\beta 5$ propeptide helps drive incorporation of $\beta 6$, and the C-terminal tail of $\beta 7$ is required for efficient $\beta 7$ incorporation. The $\beta 7$ tail in conjunction with the $\beta 5$ propeptide also promotes the dimerization of fully assembled half-proteasomes to form the PHP.

**Assembly of the 19S Regulatory Particle**

Like the CP, the RP also requires the assistance of dedicated chaperones for its efficient and accurate assembly in vivo. The lid and base subcomplexes of the RP appear to assemble independently and then associate with one another (and Rpn10) to complete the RP (67, 101). The
recent RP EM structures show a dense mesh  
of protein-protein interactions between lid and  
base subunits. Therefore, forming these sub-  
complexes prior to their association to limit as-  
sembly of RPs with aberrant compositions may  
be advantageous. This mode of assembly may  
also help restrict RP DUB and ATPase activi-  
ties until assembly is completed, thereby ensur-  
ing the coupling of substrate deubiquitylation,  
unfolding, and degradation.

**Base assembly.** Although a single arrange-  
ment of the proteasomal ATPases is observed  
in vivo (26), the Rpt subunits do not seem to  
encode within their primary structure all of  
the information required for their appropriate  
arrangement. For example, heterologously ex-  
pressed Rpt4 or coexpressed Rpt1 and Rpt2  
subunits have a propensity to form nonnative  
structures capable of hydrolyzing ATP (102).  
To help guide the proper formation of the  
base, eukaryotes utilize at least four dedicated  
RP assembly chaperones (RACs): Rpn14, Nas6,  
Nas2, and Hsm3 (PAAF1, gankyrin/p28, p27,  
and S5b in mammals) (Table 2) (55, 103–107).  
Each RAC binds a distinct subset of base sub-  
units (Figure 8a) but is not present in the 26S  
holoenzyme, at least in yeast, consistent with  
roles as assembly chaperones. Deletion of each  
RAC either alone or in combination impairs  
base formation but does not affect assembly of  
the CP or lid. Thus, the known RACs act specif-  
ically at the base assembly stage, although a role  
in lid-base and/or initial RP-CP joining cannot  
be excluded.

Our current understanding of base assembly  
derives primarily from studies of relatively  
stable base subcomplexes isolated from yeast  
mutants or mammalian cells subjected to small  
interfering (si)RNA-mediated knockdowns.  
Similar to the trimer of dimers proposed  
for the archaeal PAN complex, two of the  
earliest steps in base assembly seem to be the  
formation of Rpt cis-trans dimers—Rpt3-Rpt6,  
Rpt5-Rpt4, and Rpt2-Rpt1 (Figure 4c)—and  
their association with their specific RACs  
(and for Rpt1–2, association with Rpn1) (78).  
Whether ATPase dimerization precedes Rpt  

binding to RACs or non-ATPase subunits is  
unclear, although the RACs can bind their  
cognate ATPase (or a fragment thereof) in  
the absence of the paired ATPase subunit  
(106, 108). Although ATP binding may not be  
required for formation of Rpt cis-trans pairs, it  
does appear to be important for association of  
these pairs with other ATPase pairs (109).

Following formation of the Rpn14-Rpt6-  
Rpt3-Nas6, Rpt4-Rpt5-Nas2, and Hsm3-  
Rpt1-Rpt2-Rpn1 complexes (referred to as the  
Rpn14/Nas6, Nas2, and Hsm3 modules, re-  
spectively), they associate with one another,  
along with Rpn2 and Rpn13, to form the base  
(Figure 8a). In yeast, the Rpn14/Nas6 and  
Nas2 modules appear to associate first, and  
upon incorporation of the Hsm3 module (or  
before), Nas2 is released from the complex (26).  
In mammalian cells, a complex between the  
Rpn14/Nas6 and Hsm3 modules has been ob-  
served that specifically lacks Rpt4 and Rpt5 (55),  
possibly indicating that the order in which these  
modules come together to form the base may  
vary among species. Alternatively, the modules  
may assemble in multiple orders. Rpn14, Nas6,  
and Hsm3 each can be found bound to the fully  
assembled RP but not to the 26S proteasome,  
suggesting that they are normally released prior  
to or during RP-CP binding.

**Structure and function of the regulatory  
particle assembly chaperones.** Each RAC  
contains distinct protein-protein interaction  
domains: Rpn14 contains seven WD40 re-  
peats, Nas6 seven ankyrin repeats, Nas2 a  
PDZ domain, and Hsm3 11 HEAT repeats  
(108, 110–113). These domains mediate  
interaction of each RAC with its cognate AT-  
Pase: Rpn14-Rpt6, Nas6-Rpt3, Nas2-Rpt5,  
and Hsm3-Rpt1. Studies have determined  
the crystal structures of Nas6 and Hsm3  
bound to fragments of their cognate ATPases  
(Figure 8b,c) (111–113); the repeat domain  
in each recognizes the respective Rpt CTD.  
Rpn14 forms a β-propeller as expected  
(Figure 8d) (110). Although Rpn14 has not  
been crystallized with its binding partner,  
yeast two-hybrid and protein pulldown assays  indicate it interacts with the Rpt6 CTD. In contrast, recognition of Rpt5 by Nas2 depends on the Rpt5 HbYX motif; deletion of these three C-terminal residues is sufficient to abolish Nas2 binding (108).

Despite this structural information, the molecular mechanisms by which the RACs control base assembly are still poorly understood. In some models, docking of Nas6 and Hsm3 onto the ATPase ring would clash sterically with the 20S CP (106), perhaps explaining why these RACs are absent in the full 26S proteasome. Rpn14, Nas6, and Hsm3 appear positioned either to mediate association of their cognate ATPases with their dimerization partners or perhaps to control the arrangement of the ATPase-heterodimer pairs in the ATPase ring. Hsm3 uses distinct elements to make contacts with both Rpt1 and Rpt2, and the integrity of both of these elements is required for efficient base assembly in vivo (112). Thus, Hsm3 appears to function, at least in part, by promoting the specific pairing of Rpt1 and Rpt2. In contrast, Rpt4 and Rpt5 appear to be quite stable together in the absence of their chaperone, Nas2 (105). The specific binding of Nas2 to the C-terminal tail of Rpt5 instead suggests a role in preventing the premature docking of this subunit, either alone or in complex with other base subunits, onto the CP (108).

### Is the core particle a template for regulatory particle base assembly?

In addition to the RACs, the CP may also function as a base assembly chaperone by providing a template or scaffold for RP base formation. This hypothesis was first proposed because base subparticles accumulated when subunits or assembly chaperones of the yeast CP were mutated (95). In human cells, a nascent RP subcomplex containing Rpn2, Rpn10, Rpn11, and Rpn13 (and the proteasome-associated protein Txl1), but apparently lacking the remaining RP subunits, coimmunoprecipitates with the mature CP under some conditions (114). However, none of these subunits directly contacts the CP in the 26S holoenzyme, suggesting either that large-scale rearrangement of the RP-CP interface

may occur during assembly or that the observed complex may be a dissociation product of the RP.

In yeast, deletion of the most C-terminal residue of specific ATPases greatly disrupts RP formation without significantly affecting their association with their cognate RACs (107, 108). As these mutations disrupt interaction of the RP with the CP, the CP was hypothesized to serve as a template for base formation via insertion of the C-terminal tails of the assembling ATPases into specific surface pockets of the α-ring. Further, as purified CPs could displace the RACs from their cognate ATPases, proper docking of ATPases onto the α-ring was proposed to be the trigger for release of bound assembly chaperones. In contrast, others have assembled functional RPs equally efficiently in the absence of the CP from RP intermediates purified from bovine erythrocytes, arguing against a stringent requirement for the CP in base assembly (115).

Although the data are clear that disruption of the CP or RP-CP interactions correlates with an accumulation of base assembly intermediates, whether this reflects a templating function for the CP in base assembly remains uncertain. Base intermediates might accumulate for other reasons. In yeast, proteasome gene expression is driven by the transcription factor Rpn4 (Nrf1 in humans), which is itself a substrate of the proteasome (116, 117). Thus, low proteasome activity will increase Rpn4 levels, which will in turn increase the cellular levels of proteasomes (and their assembly intermediates). Whereas nearly all proteasome subunit genes contain Rpn4 binding sites in their promoters, the RAC genes do not. Thus, the chaperoning capacity of the RACs may become limiting when proteasome synthesis rates are very high, e.g., owing to defective RP-CP association, leading to the accumulation of base assembly intermediates even though assembly per se is not compromised. Accordingly, deletion of *RPN4* in the context of specific ATPase C-terminal deletions reduced the levels of RP assembly intermediates without affecting the apparent levels of 26S proteasomes (107), although the levels of these intermediates were still higher than in wild-type cells. Lower
effective RAC levels may also be engendered by  
tight binding to accumulating base or RP intermediates that do not allow their ready release.  
This could occur either if the RP-CP interface  
is compromised or if defects in CP formation  
lead to an excess of free RP. Hence, the CP-  
templating model for RP assembly, though attractive, requires additional experiments to address these ambiguities.

**Lid assembly.** In contrast to the base and CP,  
much less is known about lid assembly; moreover, no lid-specific assembly chaperones have  
been identified to date. Although the general  
molecular chaperone Hsp90 has been implicated in RP assembly in yeast (118), whether  
it contributes directly or indirectly to RP biogenesis is unclear. Coexpression of the nine lid  
subunits with Hsp90 in *Escherichia coli* is sufficient to yield a particle containing at least eight  
of the nine subunits (Sem1 was not detected) (46). The recombinant lid incorporates Zn²⁺  
into the Rpn11 active site and, when incorporated into the 26S proteasome, displays DUB  
activity comparable with proteasomes containing yeast-derived lid, strongly suggesting that,  
other than potentially Hsp90, no eukaryote-  
specific chaperones are required.

The high-resolution EM structures of the  
lid and the mapping of the subunits within it are  
sufficient to allow the unambiguous identification of the winged-helix domains and adjacent  
α-helices in each PCI subunit (27, 46, 47). Each  
PCI domain associates laterally with the PCI  
domain of its neighbor(s) to form a horseshoe-  
shaped lattice (Figure 6). Although this lateral  
association could in principle initiate with  
any PCI subunit and finish in any order, only  
specific subcomplexes of the lid are observed  
in vivo (67, 119–121), pointing to a potential  
hierarchical assembly mechanism in which subsequent subunit incorporations are triggered by the binding of the preceding subunits.  
In vitro binding studies, although still limited,  
support this idea (see below in this section) (65).  
Assembly of specific subcomplexes may cause  
conformational changes that favor binding of  
subsequent subunits or may provide additional  

binding sites that enhance the affinity for other  
subunits. The MPN domain subunits Rpn8 and  
Rpn11 appear to contact several PCI subunits  
and may help to stabilize specific interactions  
between PCI subunits or dictate the order in  
which the PCI subunits bind one another.

As with base assembly, our understanding  
of lid assembly derives primarily from the cataloging of the contents of lid subparticles that  
accumulate in yeast strains harboring mutations  
in specific lid subunits (67, 119–121). Lid assembly appears to begin via formation of two  
nonoverlapping and complementary subcomplexes: Rpn5/6/8/9/11 [herein called module 1  
(122)] and Rpn3/7/Sem1 [called lid particle 3  
(LP3)] (Figure 9a). The sequence(s) of subunit additions that form these two particles are  
unknown, although a module 1-related complex lacking Rpn6 has also been observed in an  
*rpn6* mutant (121). In *rpn12* mutants, a lid subcomplex (LP2) containing all subunits of module 1 and LP3 (but lacking Rpn12) accumulates (67, 119), and from this result, module 1 and  
LP3 were inferred to join one another to form  
LP2 (Figure 9a). Notably, mutation of any lid  
subunit (besides Rpn12) leads to the accumulation of free Rpn12, suggesting that Rpn12 addition to the lid precursor is the final step in  
lid assembly (67). Consistent with this, purified  
Rpn12 and LP2 associate with one another in  
vitro to form a particle indistinguishable from  
the lid. Notably, purified LP2 added to Rpn12-deficient yeast extracts fails to incorporate into  
26S proteasomes even after extended incubation. In contrast, LP2 addition to LP2-deficient  
yeast extracts that have ample free Rpn12 causes  
rapid incorporation of both LP2 and Rpn12 into 26S proteasomes. These results support  
the idea of a hierarchical or ordered assembly mechanism in which LP2 does not associate  
with the base and continue RP assembly unless  
Rpn12 first joins to complete lid formation.

**Lid-base association and Rpn10 incorporation.** The finding that LP2 cannot incorporate  
into the RP until Rpn12 has joined suggests  
an important role for this subunit in governing lid-base association. A highly conserved

C-terminal tail of Rpn12 is essential for efficient lid-base association in the absence of Rpn10, indicating that these two proteins have overlapping roles in securing the lid onto the base (67). Within the 26S proteasome, Rpn10 and Rpn12 are positioned on opposite sides of the Rpn2 structure (Figure 9b), and whereas Rpn12 makes extensive contact with Rpn2, Rpn10 does not (46, 47). Instead, Rpn10 interacts with the MPN domain of Rpn11, which in turn binds Rpn2. Thus, Rpn2 may act like a saddle horn onto which the lid attaches in a manner aided by Rpn10-dependent stabilization of Rpn2-Rpn11 binding (Figure 9b). Of course, there are many points of interaction between the lid and base subcomplexes, but these two interactions may be the first formed and may stabilize the lid until it can seat firmly onto the base. Further experiments are necessary to clarify the contributions of distinct subdomains of the lid and base subunits to this late stage of RP assembly.

**Assembly of Alternative Proteasome Isoforms**

As a detailed assembly map has developed over the past several years for the canonical 26S proteasome, an emerging question is how the assembly of the canonical proteasomes versus the noncanonical immuno-, thymo-, and in the case of yeast (and perhaps humans), α4-α4 proteasomes is governed in cells. In cells expressing alternative proteasome subunits, such as the cortical thymus, all isoforms of the variant subunits (i.e., β1, β1i, β2, β2i, β5, β5i, β5t) are readily detectable by immunoblot analysis of whole-cell extracts, but only certain configurations appear in mature CPs (15). For example, β5t-containing proteasomes seem to include only the β1i- and β2i-subunits and not β1 and β2. Although these subunits are expressed at higher levels than their canonical counterparts, this may not be sufficient to account for the specificity of proteasome formation observed. In support of this, immunoproteasome subunits help recruit other immunoproteasome subunits into assembling CPs (123–126). This is to some extent mediated by the different

propeptides of these subunits (126), but other factors may also contribute. Isoform-specific assembly chaperones represent one intriguing but as yet untested possibility for such factors. Besides the RP, the α-ring surface of CPs can bind at least three alternative regulators: PA200/Blm10, PA28, and REGγ (only Blm10 is found in yeast) (reviewed in Reference 127). A common function of these regulators is to open the gate in the α-subunit ring; each regulator also forms a structure that contains an opening that could potentially provide a channel for passage of substrates through it to the CP (20, 21, 24, 128–130). In addition, the homohexameric AAA+-family ATPase Cdc48 (p97 in mammals) from archaea was recently reported to bind and activate the archaeal CP (131, 132). As Cdc48 is highly conserved in eukaryotes, it may function as a proteasomal activator in eukaryotes as well. In principle, assembling different regulators onto the ends of distinct CPs could generate a multitude of proteasomal species, although the range of such potential regulator-CP complexes in vivo and the control of their assembly remain unknown.

**Quality Control of Proteasome Assembly**

The evolutionary conservation of proteasome assembly chaperones across eukaryotes (and in some cases extending to archaea) suggests a considerable investment in controlling proteasome assembly. Invariably, however, assembly errors will occur; what becomes of these misassembled proteasomes? Recent work suggests that in addition to stringent cellular control over the assembly process, post-assembly quality controls may exist as well. The highly conserved, proteasome-associated protein Ecm29 is enriched on proteasomes in mutant yeast bearing certain proteasomal defects (94, 108, 133). Disruption of efficient proteasome assembly by deletion of the CP assembly chaperone gene UMP1 (133), mutation of the critical conserved lysine residues in the CP α-ring pockets (94), or deletion of the C-terminal residue of Rpt5 necessary for efficient docking into the
CP surface (108) resulted in enhanced Ecm29 association with RP-CP-related complexes. Deletion of *UMP1* resulted in the apparent absence of the β3-subunit in Ecm29-containing RP-CP complexes and failure to properly process the β5 propeptide from its precursor form (133). Each of the aforementioned mutations would be expected to affect the RP-CP interface (22, 134). From EM reconstructions, Ecm29 appears to contact both the RP and CP (135), and it is not known to bind either in the absence of the other (94, 108, 133–135). Thus, Ecm29 appears to specifically recognize proteasomes with an aberrant RP-CP interface, at least in yeast.

Exactly why Ecm29 is associated with these defective proteasomes is still unclear. Ecm29 was reported to inhibit the catalytic activity of RP-CP complexes with which it was associated based on reduced cleavage of a fluorogenic peptide substrate compared with *ecm29Δ* proteasomes (108). However, proteasomes to which Ecm29 is bound may be misassembled rather than simply less active. Similarly, addition of recombinant β3-subunit to Ecm29-bound proteasomes (which lacked β3) resulted in its incorporation into the CP and release of Ecm29 (133). However, whether Ecm29 simply sequesters defective proteasomes or facilitates their repair remains unclear.

In mammalian cells, Ecm29 interacts with several proteins of the microtubule and molecular motor machinery and decorates the surfaces of late endosomes presumably destined for the lysosome (136, 137). Normal proteasomes are slowly turned over by lysosomes in rat liver (138) and may associate with the vacuole in yeast (139). Thus, Ecm29 may promote the recruitment and degradation of malformed proteasomes via the lysosome/vacuole. Ecm29 is not required for gross localization of proteasomes to the vacuole in yeast (139), but whether a defective subpopulation of proteasomes is degraded, repaired, or instead sequestered from general circulation in some way remains to be determined.

---

**CONCLUDING REMARKS**

The ever-growing list of human diseases in which protein homeostasis is disrupted is a testament to the importance of the UPS for normal cellular function and its potential as a therapeutic target. Notably, both inhibition and augmentation of protein degradation may have clinical value. Proteasome inhibition has been an accepted antineoplastic strategy since the approval of bortezomib for treatment of multiple myeloma in 2003. In contrast, recent studies have indicated that enhancement of proteasome activity, typically by blocking proteasome-associated DUBs, may in principle be beneficial in protein misfolding disorders, such as many forms of neurodegeneration, and may block replication of some viruses (140, 141). Similarly, unique types of proteasomes in lymphoid tissues provide novel targets for immune modulation via proteasome inhibition (10). Thus, modulation of proteasome biogenesis may also provide opportunities to regulate proteolysis in cells for therapeutic gain.

Although the recent advances in our understanding of proteasome structure will undoubtedly serve as a template for improved understanding of its function and assembly, detailed structural and biochemical information on the proteasome during its various catalytic steps is necessary to more fully understand its mechanisms of action. Similarly, although knowledge of the architecture of subunits within the 26S holoenzyme provides a convenient end point for assessment of proteasome assembly, more detailed characterization of assembly intermediates is also needed. We remain far from understanding the basis for the impressive efficiency and specificity of proteasome assembly in vivo. Given the myriad factors involved in proteasomal assembly and proteolysis and the discovery of several uniquely configured proteasomes, the analysis of proteasome mechanism and assembly should continue to yield fundamental insights of both basic and clinical importance.
SUMMARY POINTS

1. Recent structural and biochemical analyses of the eukaryotic proteasome RP and its prokaryotic antecedents have provided a detailed view of RP subunit architecture.
2. The Rpt ATPase subunits, which are responsible for substrate unfolding for degradation, form a uniquely ordered heterohexameric ring that directly abuts the entrance to the CP.
3. The polyUb-substrate-binding sites of the RP are generally located most distally from the CP, whereas the subunits responsible for removing the polyUb tag and unfolding the substrate are poised over the entrances to the RP ATPase ring and CP ring, respectively.
4. Despite the availability of atomic structures of several RP assembly chaperones bound to domains of their cognate Rpt subunits, the mechanistic basis of chaperone function in RP biogenesis remains poorly understood.
5. No dedicated chaperones have been found for RP lid assembly, and a hierarchical mechanism of subunit addition may largely account for high-fidelity lid assembly and lid-base joining.
6. Compositional plasticity in the 20S CP provides an additional layer of complexity to proteasomal assembly, and CP assembly chaperones may have key roles in mediating biogenesis of these alternative forms.
7. Post-assembly proteasome quality controls may also be important in eukaryotes, as suggested by recent work on the proteasome-associated Ecm29 protein in yeast.

FUTURE ISSUES

1. How are the different enzymatic activities of the proteasome coordinated?
2. Why has the proteasome evolved six distinct ATPases whereas all other ATP-dependent proteases have homomeric rings? How does this heterogeneity contribute to substrate binding, unfolding, and translocation into the CP?
3. How do the multiple intrinsic and extrinsic Ub receptors contribute to substrate recognition and processing?
4. What are the relative contributions of the CP and the RACs to RP base assembly?
5. How is association of the CP with its multiple regulator complexes governed in cells?
6. How is post-assembly proteasome quality control orchestrated, and what are the pathways and mediators controlling it?
7. Where in the cell do proteasomal subparticles assemble, and how are the localization dynamics of assembly and disassembly regulated?

DISCLOSURE STATEMENT

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

ACKNOWLEDGMENTS

We thank Yanjie Li and Nicole R. Wilson for comments on the manuscript. Work on proteasomes in our lab is funded by National Institutes of Health grant GM83050 to M.H. R.J.T. Jr. is supported in part by an American Cancer Society New England Division—Massachusetts Biotech Council Cancer Research Challenge—Millennium: The Takeda Oncology Company Postdoctoral Fellowship.

LITERATURE CITED

1. Finley D. 2009. Recognition and processing of ubiquitin-protein conjugates by the proteasome. *Annu. Rev. Biochem.* 78:477–513
2. Ravid T, Hochstrasser M. 2008. Diversity of degradation signals in the ubiquitin-proteasome system. *Nat. Rev. Mol. Cell Biol.* 9:679–90
3. Thrower JS, Hoffman L, Rechsteiner M, Pickart CM. 2000. Recognition of the polyubiquitin proteolytic signal. *EMBO J.* 19:94–102
4. Pickart CM, Cohen RE. 2004. Proteasomes and their kin: proteases in the machine age. *Nat. Rev. Mol. Cell Biol.* 5:177–87
5. Sauer RT, Baker TA. 2011. AAA+ proteases: ATP-fueled machines of protein destruction. *Annu. Rev. Biochem.* 80:587–612
6. Dahlmann B, Kopp F, Kuehn L, Niedel B, Pfeifer G, et al. 1989. The multicatalytic proteinase (prosome) is ubiquitous from eukaryotes to archaebacteria. *FEBS Lett.* 251:125–31
7. Gille C, Goede A, Schloetelburg C, Preissner R, Kloetzel PM, et al. 2003. A comprehensive view on proteosomal sequences: implications for the evolution of the proteasome. *J. Mol. Biol.* 326:1437–48
8. Groll M, Ditzel L, Lowe J, Stock D, Bochtler M, et al. 1997. Structure of 20S proteasome from yeast at 2.4 Å resolution. *Nature* 386:463–71
9. Groll M, Bajorek M, Kohler A, Moroder L, Rubin DM, et al. 2000. A gated channel into the proteasome core particle. *Nat. Struct. Biol.* 7:1062–67
10. Huber EM, Basler M, Schwab R, Heinemeyer W, Kirk CJ, et al. 2012. Immuno- and constitutive proteasome crystal structures reveal differences in substrate and inhibitor specificity. *Cell* 148:727–38
11. Unno M, Mizushima T, Morimoto Y, Tomisugi Y, Tanaka K, et al. 2002. The structure of the mammalian 20S proteasome at 2.75 Å resolution. *Structure* 10:609–18
12. Unno M, Mizushima T, Morimoto Y, Tomisugi Y, Tanaka K, et al. 2002. Structure determination of the constitutive 20S proteasome from bovine liver at 2.75 Å resolution. *J. Biochem.* 131:171–73
13. Marques AJ, Palanimurugan R, Matias AC, Ramos PC, Dohmen RJ. 2009. Catalytic mechanism and assembly of the proteasome. *Chem. Rev.* 109:1509–36
14. Kloetzel PM. 2004. Generation of major histocompatibility complex class I antigens: functional interplay between proteasomes and TPPII. *Nat. Immunol.* 5:661–69
15. Murata S, Sasaki K, Kishimoto T, Niwa S, Hayashi H, et al. 2007. Regulation of CD8⁺ T cell development by thymus-specific proteasomes. *Science* 316:1349–53
16. Gaczynska M, Rock KL, Goldberg AL. 1993. γ-interferon and expression of MHC genes regulate peptide hydrolysis by proteasomes. *Nature* 365:264–67
17. Coux O, Tanaka K, Goldberg AL. 1996. Structure and functions of the 20S and 26S proteasomes. *Annu. Rev. Biochem.* 65:801–47
18. Osmulski PA, Hochstrasser M, Gaczynska M. 2009. A tetrahedral transition state at the active sites of the 20S proteasome is coupled to opening of the α-ring channel. *Structure* 17:1137–47
19. Smith DM, Chang SC, Park S, Finley D, Cheng Y, Goldberg AL. 2007. Docking of the proteasomal ATPases’ carboxyl termini in the 20S proteasome’s α ring opens the gate for substrate entry. *Mol. Cell* 27:731–44
20. Dange T, Smith D, Noy T, Rommel PC, Jurzitza L, et al. 2011. Blm10 protein promotes proteasomal substrate turnover by an active gating mechanism. *J. Biol. Chem.* 286:42830–39
21. Sadre-Bazzaz K, Whitby FG, Robinson H, Formosa T, Hill CP. 2010. Structure of a Blm10 complex reveals common mechanisms for proteasome binding and gate opening. *Mol. Cell* 37:728–35

22. Kusmierczyk AR, Kunjappu MJ, Kim RY, Hochstrasser M. 2011. A conserved 20S proteasome assembly factor requires a C-terminal HbYX motif for proteasomal precursor binding. *Nat. Struct. Mol. Biol.* 18:622–29

23. Yu Y, Smith DM, Kim HM, Rodriguez V, Goldberg AL, Cheng Y. 2010. Interactions of PAN’s C-termini with archaeal 20S proteasome and implications for the eukaryotic proteasome-ATPase interactions. *EMBO J.* 29:692–702

24. Forster A, Masters EI, Whitby FG, Robinson H, Hill CP. 2005. The 1.9 Å structure of a proteasome-11S activator complex and implications for proteasome-PAN/PA700 interactions. *Mol. Cell* 18:589–99

25. Rabl J, Smith DM, Yu Y, Chang SC, Goldberg AL, Cheng Y. 2008. Mechanism of gate opening in the 20S proteasome by the proteasomal ATPases. *Mol. Cell* 30:360–68

26. Tomko RJ Jr, Funakoshi M, Schneider K, Wang J, Hochstrasser M. 2010. Heterohexameric ring arrangement of the eukaryotic proteasomal ATPases: implications for proteasome structure and assembly. *Mol. Cell* 38:393–403

27. He J, Kulkarni K, da Fonseca PC, Krutauz D, Glickman MH, et al. 2012. The structure of the 26S proteasome subunit Rpn2 reveals its PC repeat domain as a closed toroid of two concentric α-helical rings. *Structure* 20:513–21

28. Schreiner P, Chen X, Husnjak K, Randles L, Zhang N, et al. 2008. Ubiquitin docking at the proteasome through a novel pleckstrin-homology domain interaction. *Nature* 453:548–52

29. Husnjak K, Elsasser S, Zhang N, Chen X, Randles L, et al. 2008. Proteasome subunit Rpn13 is a novel ubiquitin receptor. *Nature* 453:481–88

30. van Nocker S, Sadis S, Rubin DM, Glickman M, Fu H, et al. 1996. The multiubiquitin-chain-binding protein Mcbl is a component of the 26S proteasome in *Saccharomyces cerevisiae* and plays a nonessential, substrate-specific role in protein turnover. *Mol. Cell. Biol.* 16:6020–28

31. Deveraux Q, Ustrell V, Pickart C, Rechsteiner M. 1994. A 26S protease subunit that binds ubiquitin conjugates. *J. Biol. Chem.* 269:7059–61

32. van Nocker S, Deveraux Q, Rechsteiner M, Vierstra RD. 1996. *Arabidopsis MBP1* gene encodes a conserved ubiquitin recognition component of the 26S proteasome. *Proc. Natl. Acad. Sci. USA* 93:856–60

33. Bertolaet BL, Clarke DJ, Wolff M, Watson MH, Henze M, et al. 2001. UBA domains of DNA damage-inducible proteins interact with ubiquitin. *Nat. Struct. Biol.* 8:417–22

34. Chen L, Madura K. 2002. Rad23 promotes the targeting of proteolytic substrates to the proteasome. *Mol. Cell. Biol.* 22:4902–13

35. Schauber C, Chen L, Tongaonkar P, Vega I, Lambertson D, et al. 1998. Rad23 links DNA repair to the ubiquitin/proteasome pathway. *Nature* 391:715–18

36. Funakoshi M, Sasaki T, Nishimoto T, Kobayashi H. 2002. Budding yeast Dsk2p is a polyubiquitin-binding protein that can interact with the proteasome. *Proc. Natl. Acad. Sci. USA* 99:745–50

37. Elsasser S, Gali RR, Schwickart M, Larsen CN, Leggett DS, et al. 2002. Proteasome subunit Rpn1 binds ubiquitin-like protein domains. *Nat. Cell Biol.* 4:725–30

38. Gomez TA, Kolawa N, Gee M, Sweredoski MJ, Deshaies RJ. 2011. Identification of a functional docking site in the Rpn1 LRR domain for the UBA-UBL domain protein Ddi1. *BMC Biol.* 9:33

39. Yao T, Cohen RE. 2002. A cryptic protease couples deubiquitination and degradation by the proteasome. *Nature* 419:403–7

40. Verma R, Aravind L, Oania R, McDonald WH, Yates JR 3rd, et al. 2002. Role of Rpn11 metalloprotease in deubiquitination and degradation by the 26S proteasome. *Science* 298:611–15

41. Glickman MH, Rubin DM, Coux O, Wefes I, Pfeifer G, et al. 1998. A subcomplex of the proteasome regulatory particle required for ubiquitin-conjugate degradation and related to the COP9-signalosome and eIF3. *Cell* 94:615–23

42. Hofmann K, Bucher P. 1998. The PCI domain: a common theme in three multiprotein complexes. *Trends Biochem. Sci.* 23:204–5

43. Yu Z, Kleifeld O, Lande-Atir A, Bsoul M, Kleiman M, et al. 2011. Dual function of Rpn5 in two PCI complexes, the 26S proteasome and COP9 signalosome. *Mol. Biol. Cell* 22:911–20

44. Pick E, Hofmann K, Glickman MH. 2009. PCI complexes: beyond the proteasome, CSN, and eIF3 Troika. *Mol. Cell* 35:260–64

45. Walz J, Erdmann A, Kania M, Typke D, Koster AJ, Baumeister W. 1998. 26S proteasome structure revealed by three-dimensional electron microscopy. *J. Struct. Biol.* 121:19–29

46. Lander GC, Estrin E, Matyskiela ME, Bashore C, Nogales E, Martin A. 2011. Complete subunit architecture of the proteasome regulatory particle. *Nature* 482:186–91

47. Lasker K, Forster F, Bohn S, Walzthoeni T, Villa E, et al. 2012. Molecular architecture of the 26S proteasome holocomplex determined by an integrative approach. *Proc. Natl. Acad. Sci. USA* 109:1380–87

48. da Fonseca PC, He J, Morris EP. 2012. Molecular model of the human 26S proteasome. *Mol. Cell* 46:54–66

49. Bohn S, Beck F, Sakata E, Walzthoeni T, Beck M, et al. 2010. Structure of the 26S proteasome from *Schizosaccharomyces pombe* at subnanometer resolution. *Proc. Natl. Acad. Sci. USA* 107:20992–97

50. Nickell S, Beck F, Scheres SH, Korinek A, Forster F, et al. 2009. Insights into the molecular architecture of the 26S proteasome. *Proc. Natl. Acad. Sci. USA* 106:11943–47

51. Wollenberg K, Swaffield JC. 2001. Evolution of proteasomal ATPases. *Mol. Biol. Evol.* 18:962–74

52. Forster F, Lasker K, Beck F, Nickell S, Sali A, Baumeister W. 2009. An atomic model AAA-ATPase/20S core particle sub-complex of the 26S proteasome. *Biochem. Biophys. Res. Commun.* 388:228–33

53. Fu H, Reis N, Lee Y, Glickman MH, Vierstra RD. 2001. Subunit interaction maps for the regulatory particle of the 26S proteasome and the COP9 signalosome. *EMBO J.* 20:7096–107

54. Hartmann-Petersen R, Tanaka K, Hendil KB. 2001. Quaternary structure of the ATPase complex of human 26S proteasomes determined by chemical cross-linking. *Arch. Biochem. Biophys.* 386:89–94

55. Kaneko T, Hamazaki J, Iemura S, Sasaki K, Furuyama K, et al. 2009. Assembly pathway of the mammalian proteasome base subcomplex is mediated by multiple specific chaperones. *Cell* 137:914–25

56. Zhang F, Hu M, Tian G, Zhang P, Finley D, et al. 2009. Structural insights into the regulatory particle of the proteasome from *Methanocaldococcus jannaschii*. *Mol. Cell* 34:473–84

57. Djuranovic S, Hartmann MD, Habeck M, Ursinus A, Zwickl P, et al. 2009. Structure and activity of the N-terminal substrate recognition domains in proteasomal ATPases. *Mol. Cell* 34:580–90

58. Kajava AV. 2002. What curves α-solenoids? Evidence for an α-helical toroid structure of Rpn1 and Rpn2 proteins of the 26S proteasome. *J. Biol. Chem.* 277:49791–98

59. Effantin G, Rosenzweig R, Glickman MH, Steven AC. 2009. Electron microscopic evidence in support of α-solenoid models of proteasomal subunits Rpn1 and Rpn2. *J. Mol. Biol.* 386:1204–11

60. Rosenzweig R, Osmulski PA, Gaczynska M, Glickman MH. 2008. The central unit within the 19S regulatory particle of the proteasome. *Nat. Struct. Mol. Biol.* 15:573–80

61. Ghaemmaghami S, Huh WK, Bower K, Howson RW, Belle A, et al. 2003. Global analysis of protein expression in yeast. *Nature* 425:737–41

62. Sakata E, Bohn S, Mihalache O, Kiss P, Beck F, et al. 2012. Localization of the proteasomal ubiquitin receptors Rpn10 and Rpn13 by electron cryomicroscopy. *Proc. Natl. Acad. Sci. USA* 109:1479–84

63. Chen X, Lee BH, Finley D, Walters KJ. 2010. Structure of proteasome ubiquitin receptor hRpn13 and its activation by the scaffolding protein hRpn2. *Mol. Cell* 38:404–15

64. Zhang D, Chen T, Ziv I, Rosenzweig R, Matiuhin Y, et al. 2009. Together, Rpn10 and Dsk2 can serve as a polyubiquitin chain-length sensor. *Mol. Cell* 36:1018–33

65. Zhang N, Wang Q, Ehlinger A, Randles L, Lary JW, et al. 2009. Structure of the S5a:K48-linked diubiquitin complex and its interactions with Rpn13. *Mol. Cell* 35:280–90

66. Riedinger C, Boehringer J, Trempe JF, Lowe ED, Brown NR, et al. 2010. Structure of Rpn10 and its interactions with polyubiquitin chains and the proteasome subunit Rpn12. *J. Biol. Chem.* 285:33992–4003

67. Tomko RJ Jr, Hochstrasser M. 2011. Incorporation of the Rpn12 subunit couples completion of proteasome regulatory particle lid assembly to lid-base joining. *Mol. Cell* 44:907–17

68. Pathare GR, Nagy I, Bohn S, Unverdorben P, Hubert A, et al. 2012. The proteasomal subunit Rpn6 is a molecular clamp holding the core and regulatory subcomplexes together. *Proc. Natl. Acad. Sci. USA* 109:149–54

69. Beck F, Unverdorben P, Bohn S, Schweitzer A, Pfeifer G, et al. 2012. Near-atomic resolution structural model of the yeast 26S proteasome. *Proc. Natl. Acad. Sci. USA* 109:14870–75

70. Saeki Y, Sone T, Toh-e A, Yokosawa H. 2002. Identification of ubiquitin-like protein-binding subunits of the 26S proteasome. *Biochem. Biophys. Res. Commun.* 296:813–19

71. Crosas B, Hanna J, Kirkpatrick DS, Zhang DP, Tone Y, et al. 2006. Ubiquitin chains are remodeled at the proteasome by opposing ubiquitin ligase and deubiquitinating activities. *Cell* 127:1401–13

72. Prakash S, Tian L, Ratliff KS, Lehotzky RE, Matouschek A. 2004. An unstructured initiation site is required for efficient proteasome-mediated degradation. *Nat. Struct. Mol. Biol.* 11:830–37

73. Smith DM, Kafri G, Cheng Y, Ng D, Walz T, Goldberg AL. 2005. ATP binding to PAN or the 26S ATPases causes association with the 20S proteasome, gate opening, and translocation of unfolded proteins. *Mol. Cell* 20:687–98

74. Smith DM, Fraga H, Reis C, Kafri G, Goldberg AL. 2011. ATP binds to proteasomal ATPases in pairs with distinct functional effects, implying an ordered reaction cycle. *Cell* 144:526–38

75. Zhang F, Wu Z, Zhang P, Tian G, Finley D, Shi Y. 2009. Mechanism of substrate unfolding and translocation by the regulatory particle of the proteasome from *Methanocaldococcus jannaschii*. *Mol. Cell* 34:485–96

76. Rubin DM, Glickman MH, Larsen CN, Dhruvakumar S, Finley D. 1998. Active site mutants in the six regulatory particle ATPases reveal multiple roles for ATP in the proteasome. *EMBO J.* 17:4909–19

77. Erales J, Hoyt MA, Troll F, Coffino P. 2012. Functional asymmetries of proteasome translocase pore. *J. Biol. Chem.* 287:18535–43

78. Tomko RJ Jr, Hochstrasser M. 2011. Order of the proteasomal ATPases and eukaryotic proteasome assembly. *Cell Biochem. Biophys.* 60:13–20

79. Kusmierczyk AR, Hochstrasser M. 2008. Some assembly required: dedicated chaperones in eukaryotic proteasome biogenesis. *Biol. Chem.* 389:1143–51

80. Yao Y, Toth CR, Huang L, Wong ML, Dias P, et al. 1999. α5 subunit in *Trypanosoma brucei* proteasome can self-assemble to form a cylinder of four stacked heptamer rings. *Biochem. J.* 344:349–58

81. Gerards WL, Enzlin J, Haner M, Hendriks IL, Aebi U, et al. 1997. The human α-type proteasomal subunit HsC8 forms a double ringlike structure, but does not assemble into proteasome-like particles with the β-type subunits HsDelta or HsBPROS26. *J. Biol. Chem.* 272:10080–86

82. Gerards WL, de Jong WW, Bloemendal H, Boelens W. 1998. The human proteasomal subunit HsC8 induces ring formation of other α-type subunits. *J. Mol. Biol.* 275:113–21

83. Frentzel S, Pesold-Hurt B, Seelig A, Kloetzel PM. 1994. 20S proteasomes are assembled via distinct precursor complexes. Processing of LMP2 and LMP7 proproteins takes place in 13–16S preproteasome complexes. *J. Mol. Biol.* 236:975–81

84. Yang Y, Fruh K, Ahn K, Peterson PA. 1995. In vivo assembly of the proteasomal complexes, implications for antigen processing. *J. Biol. Chem.* 270:27687–94

85. Nandi D, Woodward E, Ginsburg DB, Monaco JJ. 1997. Intermediates in the formation of mouse 20S proteasomes: implications for the assembly of precursor β subunits. *EMBO J.* 16:5363–75

86. Schmidtke G, Schmidt M, Kloetzel PM. 1997. Maturation of mammalian 20S proteasome: purification and characterization of 13S and 16S proteasome precursor complexes. *J. Mol. Biol.* 268:95–106

87. Chen P, Hochstrasser M. 1996. Autocatalytic subunit processing couples active site formation in the 20S proteasome to completion of assembly. *Cell* 86:961–72

88. Yashiroda H, Mizushima T, Okamoto K, Kameyama T, Hayashi H, et al. 2008. Crystal structure of a chaperone complex that contributes to the assembly of yeast 20S proteasomes. *Nat. Struct. Mol. Biol.* 15:228–36

89. Hirano Y, Hendil KB, Yashiroda H, Iemura S, Nagane R, et al. 2005. A heterodimeric complex that promotes the assembly of mammalian 20S proteasomes. *Nature* 437:1381–85

90. Le Tallec B, Barrault MB, Courbeyrette R, Guerois R, Marsolier-Kergoat MC, Peyroche A. 2007. 20S proteasome assembly is orchestrated by two distinct pairs of chaperones in yeast and in mammals. *Mol. Cell* 27:660–74

91. Li X, Kusmierczyk AR, Wong P, Emili A, Hochstrasser M. 2007. β-Subunit appendages promote 20S proteasome assembly by overcoming an Ump1-dependent checkpoint. *EMBO J.* 26:2339–49

92. Hirano Y, Hayashi H, Iemura S, Hendil KB, Niwa S, et al. 2006. Cooperation of multiple chaperones required for the assembly of mammalian 20S proteasomes. *Mol. Cell* 24:977–84

93. Stadtmueller BM, Kish-Trier E, Ferrell K, Petersen CN, Robinson H, et al. 2012. Structure of a proteasome Pba1-Pba2 complex: implications for proteasome assembly, activation, and biological function. *J. Biol. Chem.* 287:37371–82

94. Park S, Kim W, Tian G, Gygi SP, Finley D. 2011. Structural defects in the regulatory particle-core particle interface of the proteasome induce a novel proteasome stress response. *J. Biol. Chem.* 286:36652–66

95. Kusmierczyk AR, Kunjappu MJ, Funakoshi M, Hochstrasser M. 2008. A multimeric assembly factor controls the formation of alternative 20S proteasomes. *Nat. Struct. Mol. Biol.* 15:237–44

96. Velichutina I, Connerly PL, Arendt CS, Li X, Hochstrasser M. 2004. Plasticity in eucaryotic 20S proteasome ring assembly revealed by a subunit deletion in yeast. *EMBO J.* 23:500–10

97. Hirano Y, Kaneko T, Okamoto K, Bai M, Yashiroda H, et al. 2008. Dissecting β-ring assembly pathway of the mammalian 20S proteasome. *EMBO J.* 27:2204–13

98. Ramos PC, Hockendorff J, Johnson ES, Varshavsky A, Dohmen RJ. 1998. Ump1p is required for proper maturation of the 20S proteasome and becomes its substrate upon completion of the assembly. *Cell* 92:489–99

99. Marques AJ, Glanemann C, Ramos PC, Dohmen RJ. 2007. The C-terminal extension of the β7 subunit and activator complexes stabilize nascent 20S proteasomes and promote their maturation. *J. Biol. Chem.* 282:34869–76

100. Ramos PC, Marques AJ, London MK, Dohmen RJ. 2004. Role of C-terminal extensions of subunits β2 and β7 in assembly and activity of eukaryotic proteasomes. *J. Biol. Chem.* 279:14323–30

101. Isono E, Nishihara K, Saeki Y, Yashiroda H, Kamata N, et al. 2007. The assembly pathway of the 19S regulatory particle of the yeast 26S proteasome. *Mol. Biol. Cell* 18:569–80

102. Takeuchi J, Tamura T. 2004. Recombinant ATPases of the yeast 26S proteasome activate protein degradation by the 20S proteasome. *FEBS Lett.* 565:39–42

103. Saeki Y, Toh EA, Kudo T, Kawamura H, Tanaka K. 2009. Multiple proteasome-interacting proteins assist the assembly of the yeast 19S regulatory particle. *Cell* 137:900–13

104. Le Tallec B, Barrault MB, Guerois R, Carre T, Peyroche A. 2009. Hsm3/S5b participates in the assembly pathway of the 19S regulatory particle of the proteasome. *Mol. Cell* 33:389–99

105. Funakoshi M, Tomko RJ Jr, Kobayashi H, Hochstrasser M. 2009. Multiple assembly chaperones govern biogenesis of the proteasome regulatory particle base. *Cell* 137:887–99

106. Roelofs J, Park S, Haas W, Tian G, McAllister FE, et al. 2009. Chaperone-mediated pathway of proteasome regulatory particle assembly. *Nature* 459:861–65

107. Park S, Roelofs J, Kim W, Robert J, Schmidt M, et al. 2009. Hexameric assembly of the proteasomal ATPases is templated through their C termini. *Nature* 459:866–70

108. Lee SY, De la Mota-Peynado A, Roelofs J. 2011. Loss of Rpt5 protein interactions with the core particle and Nas2 protein causes the formation of faulty proteasomes that are inhibited by Ecm29 protein. *J. Biol. Chem.* 286:36641–51

109. Lee SH, Moon JH, Yoon SK, Yoon JB. 2012. Stable incorporation of ATPase subunits into 19S regulatory particle of human proteasome requires nucleotide binding and C-terminal tails. *J. Biol. Chem.* 287:9269–79

110. Kim S, Saeki Y, Fukunaga K, Suzuki A, Takagi K, et al. 2010. Crystal structure of yeast Rpn14, a chaperone of the 19S regulatory particle of the proteasome. *J. Biol. Chem.* 285:15159–66

111. Nakamura Y, Umehara T, Tanaka A, Horikoshi M, Padmanabhan B, Yokoyama S. 2007. Structural basis for the recognition between the regulatory particles Nas6 and Rpt3 of the yeast 26S proteasome. *Biochem. Biophys. Res. Commun.* 359:503–9

112. Barrault MB, Richet N, Godard C, Murciano B, Le Tallec B, et al. 2012. Dual functions of the Hsm3 protein in chaperoning and scaffolding regulatory particle subunits during the proteasome assembly. *Proc. Natl. Acad. Sci. USA* 109:E1001–10

113. Takagi K, Kim S, Yukii H, Ueno M, Morishita R, et al. 2012. Structural basis for specific recognition of Rpt1p, an ATPase subunit of 26S proteasome, by proteasome-dedicated chaperone Hsm3p. *J. Biol. Chem.* 287:12172–82

114. Hendil KB, Kriegenburg F, Tanaka K, Murata S, Lauridsen AM, et al. 2009. The 20S proteasome as an assembly platform for the 19S regulatory complex. *J. Mol. Biol.* 394:320–28

115. Thompson D, Hakala K, DeMartino GN. 2009. Subcomplexes of PA700, the 19S regulator of the 26S proteasome, reveal relative roles of AAA subunits in 26S proteasome assembly and activation and ATPase activity. *J. Biol. Chem.* 284:24891–903

116. Xie Y, Varshavsky A. 2001. RPN4 is a ligand, substrate, and transcriptional regulator of the 26S proteasome: a negative feedback circuit. *Proc. Natl. Acad. Sci. USA* 98:3056–61

117. Radhakrishnan SK, Lee CS, Young P, Beskow A, Chan JY, Deshaies RJ. 2010. Transcription factor Nrf1 mediates the proteasome recovery pathway after proteasome inhibition in mammalian cells. *Mol. Cell* 38:17–28

118. Imai J, Maruya M, Yashiroda H, Yahara I, Tanaka K. 2003. The molecular chaperone Hsp90 plays a role in the assembly and maintenance of the 26S proteasome. *EMBO J.* 22:3557–67

119. Fukunaga K, Kudo T, Toh-e A, Tanaka K, Saeki Y. 2010. Dissection of the assembly pathway of the proteasome lid in *Saccharomyces cerevisiae*. *Biochem. Biophys. Res. Commun.* 396:1048–53

120. Isono E, Saeki Y, Yokosawa H, Toh-e A. 2004. Rpn7 is required for the structural integrity of the 26S proteasome of *Saccharomyces cerevisiae*. *J. Biol. Chem.* 279:27168–76

121. Isono E, Saito N, Kamata N, Saeki Y, Toh EA. 2005. Functional analysis of Rpn6p, a lid component of the 26S proteasome, using temperature-sensitive rpn6 mutants of the yeast *Saccharomyces cerevisiae*. *J. Biol. Chem.* 280:6537–47

122. Sharon M, Taverner T, Ambroggio XI, Deshaies RJ, Robinson CV. 2006. Structural organization of the 19S proteasome lid: insights from MS of intact complexes. *PLoS Biol.* 4:e267

123. Groettrup M, Standera S, Stohwasser R, Kloetzel PM. 1997. The subunits MECL-1 and LMP2 are mutually required for incorporation into the 20S proteasome. *Proc. Natl. Acad. Sci. USA* 94:8970–75

124. DeM, Jayarapu K, Elenich L, Monaco JJ, Colbert RA, Griffin TA. 2003. β2 subunit propeptides influence cooperative proteasome assembly. *J. Biol. Chem.* 278:6153–59

125. Griffin TA, Nandi D, Cruz M, Fehling HJ, Kaer LV, et al. 1998. Immunoproteasome assembly: cooperative incorporation of interferon γ (IFN-γ)-inducible subunits. *J. Exp. Med.* 187:97–104

126. Kingsbury DJ, Griffin TA, Colbert RA. 2000. Novel propeptide function in 20S proteasome assembly influences β subunit composition. *J. Biol. Chem.* 275:24156–62

127. Stadtmueller BM, Hill CP. 2011. Proteasome activators. *Mol. Cell* 41:8–19

128. Iwanczyk J, Sadre-Bazzaz K, Ferrell K, Kondrashkina E, Formosa T, et al. 2006. Structure of the Blm10-20S proteasome complex by cryo-electron microscopy. Insights into the mechanism of activation of mature yeast proteasomes. *J. Mol. Biol.* 363:648–59

129. Stadtmueller BM, Ferrell K, Whitby FG, Heroux A, Robinson H, et al. 2010. Structural models for interactions between the 20S proteasome and its PAN/19S activators. *J. Biol. Chem.* 285:13–17

130. Whitby FG, Masters EI, Kramer L, Knowlton JR, Yao Y, et al. 2000. Structural basis for the activation of 20S proteasomes by 11S regulators. *Nature* 408:115–20

131. Barthelme D, Sauer RT. 2012. Identification of the Cdc48·20S proteasome as an ancient AAA+ proteolytic machine. *Science* 337:843–46

132. Forouzan D, Ammelburg M, Hobel CF, Stroh LJ, Sessler N, et al. 2012. The archaeal proteasome is regulated by a network of AAA ATPases. *J. Biol. Chem.* 287:39254–62

133. Lehmann A, Niewienda A, Jechow K, Janek K, Enenkel C. 2010. Ecm29 fulfils quality control functions in proteasome assembly. *Mol. Cell* 38:879–88

134. Kleijnen MF, Roelofs J, Park S, Hathaway NA, Glickman M, et al. 2007. Stability of the proteasome can be regulated allosterically through engagement of its proteolytic active sites. *Nat. Struct. Mol. Biol.* 14:1180–88

135. Leggett DS, Hanna J, Borodovsky A, Crosas B, Schmidt M, et al. 2002. Multiple associated proteins regulate proteasome structure and function. *Mol. Cell* 10:495–507

136. Gorbea C, Pratt G, Ustrell V, Bell R, Sahasrabudhe S, et al. 2010. A protein interaction network for Ecm29 links the 26S proteasome to molecular motors and endosomal components. *J. Biol. Chem.* 285:31616–33

137. Gorbea C, Goellner GM, Teter K, Holmes RK, Rechsteiner M. 2004. Characterization of mammalian Ecm29, a 26S proteasome-associated protein that localizes to the nucleus and membrane vesicles. *J. Biol. Chem.* 279:54849–61

138. Cuervo AM, Palmer A, Rivett AJ, Knecht E. 1995. Degradation of proteasomes by lysosomes in rat liver. *Eur. J. Biochem.* 227:792–800

139. Kleijnen MF, Kirkpatrick DS, Gygi SP. 2007. The ubiquitin-proteasome system regulates membrane fusion of yeast vacuoles. *EMBO J.* 26:275–87

140. Nag DK, Finley D. 2012. A small-molecule inhibitor of deubiquitinating enzyme USP14 inhibits Dengue virus replication. *Virus Res.* 165:103–6

141. Lee BH, Lee MJ, Park S, Oh DC, Elsasser S, et al. 2010. Enhancement of proteasome activity by a small-molecule inhibitor of USP14. *Nature* 467:179–84

142. Tian G, Finley D. 2012. Cell biology: destruction deconstructed. *Nature* 482:170–71

Contents

**Prefatory**

Christian Raetz: Scientist and Friend Extraordinaire  
*William Dowhan, Hiroshi Nikaido, JoAnne Stubbe, John W. Kozarich,*  
*William T. Wickner, David W. Russell, Teresa A. Garrett, Kathryn Brozek,*  
*and Paul Modrich*  
1  

**Recent Advances in Biochemistry**

Mechanisms for Initiating Cellular DNA Replication  
*Alessandro Costa, Iris V. Hood, and James M. Berger*  
25  

The Chromatin Response to DNA Breaks: Leaving a Mark on Genome Integrity  
*Godelieve Smeenk and Haico van Attikum*  
55  

Readout of Epigenetic Modifications  
*Dinshaw J. Patel and Zhanxin Wang*  
81  

Flap Endonuclease 1  
*Lata Balakrishnan and Robert A. Bambara*  
119  

New Mechanistic and Functional Insights into DNA Topoisomerases  
*Stefanie Hartman Chen, Nei-Li Chan, and Tao-shih Hsieh*  
139  

Arrest Peptides: Cis-Acting Modulators of Translation  
*Koreaki Ito and Shinobu Chiba*  
171  

Structural Basis of the Translational Elongation Cycle  
*Rebecca M. Voorhees and V. Ramakrishnan*  
203  

CRISPR-Mediated Adaptive Immune Systems in Bacteria and Archaea  
*Rotem Sorek, C. Martin Lawrence, and Blake Wiedenheft*  
237  

Correlating Structure and Energetics in Protein-Ligand Interactions: Paradigms and Paradoxes  
*Stephen F. Martin and John H. Clements*  
267  

Extracellular Chaperones and Proteostasis  
*Amy R. Wyatt, Justin J. Yerbury, Heath Ecroyd, and Mark R. Wilson*  
295

Molecular Chaperone Functions in Protein Folding and Proteostasis  
Yujin E. Kim, Mark S. Hipp, Andreas Bracher, Manajit Hayer-Hartl, and F. Ulrich Hartl  . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Systemic Amyloidoses  
*Luis M. Blancas-Mejía and Marina Ramirez-Alvarado* …………………… 745  

Nanobodies: Natural Single-Domain Antibodies  
*Serge Muyldermans* …………………………………………………… 775  

Indexes  

Cumulative Index of Contributing Authors, Volumes 78–82 ………………… 799  
Cumulative Index of Article Titles, Volumes 78–82 ………………………… 803  

Errata  

An online log of corrections to *Annual Review of Biochemistry* articles may be found at  
http://biochem.annualreviews.org/errata.shtml
