
Article

![](https://example.com/image.png)

# A Balanced Ratio of Proteins from Gene G and Frameshift-Extended Gene GT Is Required for Phage Lambda Tail Assembly

## Jun Xu, Roger W. Hendrix and Robert L. Duda

### Department of Biological Sciences, University of Pittsburgh, Pittsburgh, PA 15260, USA

**Correspondence to Robert L. Duda:** [duda@pitt.edu](mailto:duda@pitt.edu)  
[http://dx.doi.org/10.1016/j.jmb.2013.07.002](http://dx.doi.org/10.1016/j.jmb.2013.07.002)  
**Edited by J. Johnson**

## Abstract

In bacteriophage λ, the overlapping open reading frames G and T are expressed by a programmed translational frameshift similar to that of the gag-pol genes of many retroviruses to produce the proteins gpG and gpGT. An analogous frameshift is widely conserved among other dsDNA tailed phages in their corresponding “G” and “GT” tail genes even in the absence of detectable sequence homology. The longer protein gpGT is known to be essential for tail assembly, but the requirement for the shorter gpG remained unclear because mutations in gene G affect both proteins. A plasmid system that can direct the efficient synthesis of tails was created and used to show that gpG and gpGT are both essential for correct tail assembly. Phage complementation assays under conditions where levels of plasmid-expressed gpG or gpGT could be altered independently revealed that the correct molar ratio of these two related proteins, normally determined by the efficiency of the frameshift, is also crucial for efficient assembly of functional tails. Finally, the physical connection between the G and T domains of gpGT, a consequence of the frameshift mechanism of protein expression, appears to be important for efficient tail assembly.

© 2013 Elsevier Ltd. All rights reserved.

---

## Introduction

Genes Z, U, V, G, T, H, M, L, K, I, and J are all required for λ tail assembly [7]. Early studies showed that gpV was the major tail shaft protein [8] and that the tail fiber (and host range determinant) was composed of gpJ [9–11]. Many of the protein products of these essential genes were identified in SDS polyacrylamide gels [12]. The order of interaction of the protein gene products and the overall assembly pathway (Fig. 1b) were determined in large part from the results of *in vitro* complementation experiments between extracts of mutant-infected cells [1] and additional experiments by Katsura and many others as reviewed by Katsura [2]. From these studies, it was clear that a slow sedimenting (15S) precursor produced by genes J, I, L, and K could be converted by the actions of genes H, G, and M to a faster sedimenting (25S) initiator of tail shaft polymerization, but the exact composition and nature of the 15S and 25S complexes remained unclear and no images of these complexes were ever obtained by electron microscopy. The exact roles of many of the proteins have remained unclear to this day, although it was

0022-2836/$ - see front matter © 2013 Elsevier Ltd. All rights reserved.  
[J. Mol. Biol. (2013) 425, 3476–3487](https://doi.org/10.1016/j.jmb.2013.07.002)

gpG/gpGT Ratio Critical for λ Tail Assembly

(a)

![Diagram](https://i.imgur.com/1234567.png)

(b)

![Diagram](https://i.imgur.com/8901234.png)

Fig. 1. (a) λ tail proteins gpG and gpGT are related by a programmed –1 translational frameshift. Near the end of gene G, a “slippery sequence” G GGA AAG causes about 3.5% of the ribosomes to switch reading frame to the T reading frame. The shifted ribosomes continue to translate the Torf to make a larger fusion protein gpGT. gpGT contains almost the entire amino acid sequence of gpG. Top: Schematic of the G/T region of the lambda genome; the arrows represent the two translated proteins. (b) λ tail assembly pathway. At least 11 proteins are required in the λ tail assembly pathway. The assembly pathway can be divided roughly into three stages: formation of the initiator, tail polymerization, and termination. The intermediates in the assembly pathway are defined by their mobility on sucrose gradients and by their *in vitro* complementation activity [1–4]. The sequence of addition of gpG, gpGT, and gpH was not originally defined because phages with amber mutations in these genes fail to complement each other.

later shown that gpH functions as a tape measure protein that determines the overall length of tails [13,14]. The function of gene “T” in the production of gpGT by frameshifting was not discovered until much later [5]; however, we have updated the pathway in Fig. 1b to show the likely entry point of gpGT into the tail assembly scheme.

The protein gpG is expressed to a high level during phage infections, similar to the level seen for the major tail protein gpV [5,12]. This is surprising because neither gpG nor gpGT is present in the mature tail [5], especially since phages typically express structural proteins in approximately the same ratio as they are found in the mature structure, with the notable exceptions of scaffolding or chaperone proteins. Earlier studies using an amber mutant in the T region showed that gpGT is required for formation of the initiator and no visible tail structure was seen in the absence of gpGT [1,3,5]. (Note that because of the overlapping relationship of the two “genes” [5], amber mutants in the G and T orfs fail to complement each other genetically. As a result, both types of mutants were initially considered to map to the same gene (G). The T amber mutant used here is called Gam901 in the earlier literature.) Because any mutation in the G region will affect not only gpG but also the essential gpGT, it is not known whether gpG is required for tail assembly. One extreme possibility is that gpG is not essential, but it is expressed only to provide a basis for producing the appropriate amount of gpGT through an inefficient frameshift. This seems unlikely, since the expression level of most phage structural genes can be determined at the level of translational initiation efficiency, and the variation among genes can be as much as 800-fold [12,15,16]. Furthermore, the expression of gpG itself is likely controlled at the level of translational initiation, which means gpGT's expression level is also indirectly controlled at translation initiation. An alternative view is that gpG is essential. Since the frameshift is highly conserved among different phages, even though often no primary sequence similarity can be detected, this would argue that the outcome from the frameshift expression mechanism—producing two proteins sharing the same N-terminus in a fixed ratio—is important.

A plasmid-based tail assembly system was established in which the 11-kb tail gene region was cloned under the control of a T7 promoter. The resulting construct can produce active tails. Through a series of complementation experiments in which we can decouple the relative amounts of the two proteins from the efficiency of the frameshift, we

show that gpG is indeed essential and that the appropriate ratio of gpG to gpGT is important for successful tail assembly.

---

### Results

#### Biologically active λ tails made from a plasmid

λ tail assembly has been studied extensively using phage infection or induction of lysogens carrying amber or deletion mutations in the tail region of the prophage [2]. Constructing targeted point mutations is difficult in that context, so we cloned ~11 kb of DNA containing the essential λ tail genes (including Z, U, V, G, T, H, M, L, K, I, and J, but not tail fiber genes *ta* and *stf*) into a T7 promoter-based vector, creating pETail4. Figure 2a shows an SDS-PAGE gel of the radiolabeled proteins expressed from pETail4. Note the presence of gpH*, which is the processed form of gpH, an indication of completed tail assembly [17]. The proteins expressed from pETail4 assemble into biologically active tails as determined by their ability to join free heads to form infective phages in complementation assays [19]. The expression system is highly efficient, as judged by the high yield from *in vitro* complementation with a head donor extract. Extracts from cells carrying pETail4 gave more than 40-fold higher complementation activity (data not shown) than extracts from an induced lysogen defective in head assembly. The tails made using pETail4 were purified using a modification of a protocol developed for phage HK97 prohead purification (see Materials and Methods). Figure 2b shows the proteins from the purified tails separated by SDS-PAGE and visualized by Coomassie blue stain. Only five proteins were seen on the gel, namely, gpU, gpM, gpV, gpH* (cleaved product of gpH), and gpJ. The products of the remaining tail genes I, K, L, and Z have essential roles in the early steps of tail assembly but were not found in tails here. An electron micrograph of negatively stained purified tails is shown in Fig. 2c, demonstrating that their morphology is normal and uniform. This plasmid-based tail production system (using a modified host, see Materials and Methods) became the basis for all further studies, since it makes it easy to build plasmids containing different subsets of the tail genes and to make mutations at will.

#### G is required for tail assembly

Neither gpG nor gpGT is incorporated into mature λ tails or phage [5]. An amber mutation in the T region blocks tail assembly, indicating that gpGT is essential [1]. However, the special relationship between gpG and gpGT makes it less straightforward to determine whether gpG is required because any mutation in the G region will also affect gpGT. To ask whether gpG is essential, we created a mutation (gpGT-mut) that fuses the G and T orfs and therefore allows gpGT to be expressed without a frameshift (Fig. 3a). If the resulting plasmid construct can produce active tails, it would argue that gpG is not required because gpGT alone can support tail assembly. On the other hand, if the resulting

---

**Fig. 2.** Plasmid pETail4 produces λ tails. (a) The pETail4 plasmid map. (b) Proteins expressed from plasmid pETail4 were radiolabeled with ${}^{35}$S-methionine and separated by 12.5% SDS-PAGE and autoradiographed. The identification of bands is based on previous studies [12]. The identity of gpJ, gpH, gpH*, gpGT, gpV, gpG, gpU, and gpM was previously established. The identity of gpL, gpl, gpK, and gpZ is not firmly established. gpH* is produced by assembly-dependent cleavage of gpH, and its presence is an indication of assembly [17]. (c) Tails produced from plasmid pETail4 were purified using a protocol similar to one developed for purifying HK97 procapsids [18]. Proteins from the purified tails were separated by 12.5% SDS-PAGE and visualized using Coomassie brilliant blue R250 stain. Only five proteins were seen on the gel, gpJ, gpH*, gpU, and gpM. (d) An electron micrograph of purified tails. The tails are uniform in length and have the same number of disks as the tails on a normal phage, indicating correct assembly.

(a) Wild type G/GT


---GTT TCT GC_G GGA AAG TGT TCG ACG GTG AGC TGA
     V   S    A   G   K   C   S   T   V   S   .
shifts into "-1" frame
GGG AAA GTG TTC GAC GGT GAG CTG
G   K   V   F   D   G   E   L ---


(b) GTmut


---GTT TCT GC_G GGA AAG GTG TTC GAC GGT GAG CTG
     V   S    A   G   K   V   F   D   G   E   L ---
GTmut shifts -1
extra G shifts translation to "-2" frame
gpG* --- GGG AAA GG_T GTT CGA CGG TGA GCT G
       G   K   G   V   R   R   .


(c)

(d) autoradiogram          immuno-blot


pETail4
pETail5



gpJ -
gpH -
gpH*- 
gpGT-
gpV-
gpG-
gpU-
G>GT GT>G


Fig. 3. Construction and effects of the GT-mut fusion mutation that produces gpGT without a frameshift. (a) An extra G was inserted right after the slippery sequence at an XmnI site (see Fig. 1). The extra G allows gpGT to be expressed without a frameshift [see (b)]. The slippery sequence is underlined and the extra G inserted is boxed. Before the mutation, about 3.5% of the ribosomes change orf at the slippery sequence to synthesize gpGT, and the amino acid sequence at the junction point of gpGT is VSAG → KVFD (VSAG is in the G reading frame and KVFD is in the T reading frame). After the insertion of a single nucleotide G right after the slippery sequence, the ribosomes can read directly into the T reading frame without a frameshift as shown in (b). The gpGT synthesized this way has exactly the same amino acid sequence as the ones synthesized by a frameshift mechanism. (b) A –1 frameshift can create a new protein gpG* in the GTmut fusion mutant. As in (a), a fraction of ribosomes change orf at the slippery sequence to synthesize gpG* in which the five C-terminal amino acids of gpG (CSTVS) are replaced with a different sequence (GVRR). (c) Comparison of proteins made by pETail4 and pETail5. An autoradiograph of an SDS-PAGE gel shows that in pETail5, gpGT is overexpressed and the gpG band disappears as expected. (d) Proteins made by plasmids with the wild-type and GTmut frameshift sites. Left: Radiolabeled proteins produced by pT7-5-GT, which expresses wild-type gpG and gpGT, and pT7-5-GT-mut, which contains the GT fusion mutation. Cells containing the appropriate plasmid were radiolabeled as described in Materials and Methods. The proteins were separated on a 12.5% SDS polyacrylamide gel and visualized by autoradiography. Right: Western blot of extracts expressing pT7-5-GT and pT7-5-GT-mut. The proteins were expressed, separated by SDS-PAGE, transferred to polyvinyl difluoride membrane, and probed with anti-gpG antibody.

construct does not produce active tails, it would be consistent with the view that gpG is required for the assembly but does not prove it because expression of gpGT without frameshift will result in a 30-fold increase in the amount of gpGT produced. The non-complementation result might be due to an abnormally large amount of gpGT blocking tail assembly.

The GT fusion mutation was introduced into pETail4 to create plasmid pETail5. Radiolabeling of the proteins produced from pETail5 shows the overproduction of gpGT (Fig. 3c). Notably absent on the pETail5 gel was the gpH* band that was clearly observed for pETail4 (the cleavage product of gpH generated during tail assembly [20]), which indicates that tail assembly has not reached completion. A complementation assay carried out by mixing free heads with an extract prepared from cells expressing pETail5 failed to produce tail activity above background level, which indicates that the GT fusion mutation blocks tail assembly (data not shown).

3480

gpG/gpGT Ratio Critical for λ Tail Assembly

Table 1. Plasmids and phages used in the spot complementation experiments

| Plasmid           | Vector               | Proteins expressed       | Description                                                                 |
|-------------------|----------------------|--------------------------|-----------------------------------------------------------------------------|
| pT7-5-GT          | pT7-5                | gpG, gpGT                | Wild-type G/GT region                                                       |
| pT7-5-GT-mut      | pT7-5                | gpGT, (gpG)              | GT fusion mutation (Fig. 3)                                                 |
| pET21a-T          | pET21a (Novagen)     | gpT                      | T7 tag in pET21a fused to the T orf                                          |
| pBAD33-GT         | pBAD33 [21]          | gpG, gpGT                | Wild-type G/GT region                                                       |
| pBAD33-GT-mut     | pBAD33               | gpGT, (gpG)              | GT fusion mutation (Fig. 3)                                                 |
| pBAD33-T          | pBAD33               | gpT                      | T7-tagged gpT in pET21a-T transferred to pBAD33                              |

| Phage             | Short name   | Mutations    | Phenotypes                                                                 |
|-------------------|--------------|--------------|-----------------------------------------------------------------------------|
| λGam447clts1      | λGam         | G, cl        | No gpG, no gpGT, cl temperature-sensitive mutation                          |
| λTam901clts1      | λTam         | "T", cl      | no gpGT, cl temperature-sensitive mutation                                  |
| λcl857            | λ-wt         | cl/857       | Wild type, cl temperature-sensitive mutation                               |

<sup>a</sup> Tam901 appears earlier as Gam901 [1]; we renamed it here for ease of discussion because it maps to gene T [5].

---

gpGT alone is not enough to complement λ with an amber mutation in the G region

We next used complementation experiments to more specifically test whether gpG is essential. In this experiment, we tested whether cells that express gpGT only from a plasmid can support the growth of λ phage with an amber mutation in the G region, which produces neither gpG nor gpGT. If the cells expressing gpGT alone do not complement phages expressing neither gpG nor gpGT but do complement phages expressing gpG but not gpGT, this would argue that gpG is indeed required for tail assembly.

In each plate in the complementation experiments, increasing concentrations of different amber mutant

Plasmid | pT7-5-GT | pT7-5-GT-mut | pET21a-T | pT7-5
--------|----------|--------------|----------|------
proteins expressed | gpG, gpGT | gpGT | gpT | control

(a)
![Image](image1)

(b)
![Image](image2)

(c)
![Image](image3)

PHAGE
λGam
(no gpG, no gpGT)
λTam
(no gpGT)
λCl857
(wild type)

1 2 3 4

Fig. 4. Protein gpG is required for tail assembly. Cells containing plasmids pT7-5-GT-mut (1), pT7-5-GT (2), pET21a-T (3), and empty vector control plasmid pT7-5 (4) were spotted with three types of phages (see Table 1), λGam (amber mutation in the G region), λTam (amber mutation in the T region), and λ-wt (our wild-type λ). The first row indicates the plasmid being tested, the next row indicates the proteins produced from the plasmid, and the last column shows the type of phage being spotted and the defect in each phage. Each plate was spotted from left to right with 0.004 ml of 10-fold serial dilutions of phages, where the leftmost spot has about 10⁶ phages. 2a shows that plasmid pT7-5-GT-mut, which contains the GT fusion mutation and produces gpGT only, clearly does not complement λGam, which produces neither gpG nor gpGT, but 2b shows that it does complement λTam, which produces gpG but not gpGT. This suggests that gpG is required for phage growth. Plasmid pT7-5-GT contains the wild-type G/GT region and produces both gpG and gpGT. Plasmid pT7-5-GT supports the growth of λGam very well (1a) and λTam to a modest level (1a). These results show that both gpG and gpGT have to be present to support phage growth. The fact the pET21a-T, which expresses the T frame fused to a T7 tag in the vector, does not complement any of the amber phages (3a and 3b) indicates that the G part of gpGT is also important for assembly.

which produces neither gpG nor gpGT (Fig. 4a, 2). However, λGam can be complemented by pT7-5-GT, which expresses both gpG and gpGT (Fig. 4a, 1). An important control shows that plasmid pT7-5-GT-mut is able to complement the λTam phage, which produces gpG but not gpGT (Fig. 4a, 2). Taken together, the results show that complementation only works when both gpG and gpGT are present, provided either from plasmids or from the phages themselves, which argues that gpG is required for productive tail assembly. We also found that a plasmid that expresses the T orf alone (pET21a-T) cannot complement the λTam phage that makes gpG, but no gpGT (Fig. 4b, 3), showing that the T domain of gpGT cannot efficiently replace the missing gpGT, even in the presence of gpG. We also note that plasmid pT7-5-GT, which expresses both gpG and gpGT only very weakly, complements the λTam phage, which produces only gpG (Fig. 4b, 1). The 100-fold reduced complementation in this situation suggests that the ratio of G to GT may be important, perhaps disrupted in this case due to excess gpG produced by the λTam phage that cannot be overcome by supplying both gpG and gpGT (in their normal ratio) in trans.

We recognize that it is not strictly true that plasmid pT7-5-GT-mut, which produces gpGT as fusion protein, makes no gpG. As a result of translational frameshifting at the normal site, we expect that a small amount of a new protein will be made in which the five C-terminal amino acids of gpG are replaced with a different sequence (Fig. 3b). The SDS gel and Western blot in Fig. 3d confirm this expectation and we refer to the C-terminal altered gpG as gpG*. The effect of the GT-mut fusion mutation is therefore twofold. First, by making gpG* rather than gpGT as the product of the frameshift, it changes the gpG (or gpG*)-to-gpGT ratio by roughly 900-fold (from 30:1 gpG/gpGT to 1:30 gpG*/gpGT). Second, the mutation leads to the five-amino-acid swap at the C-terminus of gpG mentioned above. The fact that the GT-mut mutant does not make functional tails may be because gpG* is not functional, but we believe a more likely explanation is that the gpG/gpGT ratio is inappropriate in the mutant. We show in the following section that the ratio must be approximately correct to allow efficient assembly. We have not seen any indication of biological activity for gpG*, either stimulating or inhibiting tail production, but would not expect to because in all experiments, it is expressed in much lower amounts than gpGT. We do not believe that the production of gpG* in the mutant significantly affects any of our conclusions about tail assembly.

The frameshift provides a mechanism to produce two proteins that share the same N-terminus. The

ratio of the two proteins normally is determined by the efficiency of the frameshifting. We again used a spot complementation experiment to investigate whether the ratio between gpG and gpGT is important for tail assembly. In this experiment, we provided gpG and gpGT from an arabinose-controlled plasmid expression system. The arabinose promoter has the property that the expression level can be modulated in a linear fashion by varying the amount of arabinose inducer added [21]. The ratio of gpGT to gpG inside the cell can be represented by the formula below where the subscripts *plasmid* and *phage* indicate the source of the protein.

\[
\text{Ratio}_{\text{gpGT/gpG}} = \frac{\text{gpGT}_{\text{plasmid}} + \text{gpGT}_{\text{phage}}}{\text{gpG}_{\text{plasmid}} + \text{gpG}_{\text{phage}}}
\]

Assuming the proteins produced from the input phages stay in similar levels ($\text{gpGT}_{\text{phage}}$ and $\text{gpG}_{\text{phage}}$ stay the same) across different concentrations of inducer used, a higher amount of inducer will result in a higher concentration of the protein produced from the plasmid (higher level of $\text{gpGT}_{\text{plasmid}}$ and $\text{gpG}_{\text{plasmid}}$) and hence change the ratio of gpG versus gpGT.

The plasmids used for the arabinose-modulated expression experiments are described in Table 1 and the experimental results are shown in Fig. 5. Figure 5a shows the complementation results for λGam (amber in the G region that produces neither gpG nor gpGT) on plasmids pBAD33-GT (produces wild-type gpG and gpGT), pBAD33-GT-mut (produces gpGT only), and pBAD33-T (produces the T part of gpGT). As the concentration of arabinose increases, pBAD33-GT begins to complement and complementation signal increases with increasing inducer (Fig. 5a, 1a–f), while plasmid pBAD33-GT-mut, which provides gpGT only, does not complement under any of the tested concentrations of arabinose (Fig. 5a, 2a–f). This result is consistent with the previous result that gpG is required for tail assembly and supplying gpGT alone in trans does not complement a phage that produces neither gpG nor gpGT. pBAD33-T does not complement, showing as expected that the T region alone cannot substitute for gpG and gpGT.

Figure 5b shows the complementation results with λTam (amber mutation in the T region that produces gpG but not gpGT) on the same set of plasmids as in Fig. 5a. As the concentration of arabinose increases, pBAD33-GT begins to complement λTam but not to the extent seen for λGam arguably because the excess gpG produced by the λTam phage cannot be overcome by supplying both gpG and gpGT (in their normal ratio) in trans. For pBAD33-GT-mut, which provides gpGT only, as the concentration of arabinose increases, λTam begins to complement (Fig. 5b, 2a–d), and then as the arabinose concentration further increases, the complementation level decreases (Fig. 5b, 2e and f). Both of these results

3482

**gpG/gpGT Ratio Critical for λ Tail Assembly**

| Plasmid | pBAD33-GT | pBAD33-GT-mut | pBAD33-T | PHAGE |
|---------|-----------|---------------|----------|-------|
| proteins made | gpG, gpGT | gpGT | gpT | (gpG? gpGT?) |

(a)

| [Arabinose] | 1 | 2 | 3 |
|--------------|---|---|---|
| a            |   |   |   |
| b            |   |   |   |
| c            |   |   |   |
| d            |   |   |   |
| e            |   |   |   |
| f            |   |   |   |

(b)

| [Arabinose] | 1 | 2 | 3 |
|--------------|---|---|---|
| a            |   |   |   |
| b            |   |   |   |
| c            |   |   |   |
| d            |   |   |   |
| e            |   |   |   |
| f            |   |   |   |

(c)

| [Arabinose] | 1 | 2 | 3 |
|--------------|---|---|---|
| a            |   |   |   |
| b            |   |   |   |
| c            |   |   |   |
| d            |   |   |   |
| e            |   |   |   |
| f            |   |   |   |

Fig. 5. A balanced ratio of gpG and gpGT is important for tail assembly. An arabinose promoter-based system was used to study the effect of the ratio between gpG and gpGT. The genes were cloned into the pBAD33 vector [21], which has the arabinose promoter. Three plasmids, pBAD33-GT, pBAD33-GT-mut, and pBAD33-T, were tested. pBAD33-GT contains the wild-type G/GT region and expresses both gpG and gpGT. pBAD33-GT-mut contains the GT fusion mutation that expresses gpGT only. pBAD33-T contains the T7-tag fused Tframe that allows the T frame to be expressed independent of gpG. Each column spanning through all the three panels represents the same plasmid being tested. The amount of protein expressed from the plasmid is modulated by changing the concentration of arabinose in the plate. Six different arabinose concentrations were tested (0%, 2 × 10⁻⁶%, 2 × 10⁻⁵%, 2 × 10⁻⁴%, 2 × 10⁻³%, and 2 × 10⁻²%). Three types of phage were tested, λTam, λGam, and λ-wt (see Table 1). (a) Phage λGam, which produces neither gpG nor gpGT, was tested on three different plasmids. In agreement with previous results, λGam can only be complemented with plasmid pBAD33-GT, which provides both gpG and gpGT. Neither gpGT alone nor gpT complements. The higher the concentration of arabinose, the better pBAD33-GT complements λGam. (b) Phage λTam, which produces gpG but not gpGT, was tested on three different plasmids. Similar to the result in (a), increasing the concentration of arabinose allows pBAD33-GT to complement better but to a lower level than with λGam in (a). pBAD33-GT-mut complements best at an arabinose concentration of 2 × 10⁻⁴% but decreases when the concentration is raised. (c) Phage λ-wt was tested as a control. Note that at the maximum arabinose concentration (2f), pBAD33-GT-mut interferes with the ability of λ-wt to grow. A similar but weak inhibitory effect on λ-wt growth was seen for “gpT”, but only at the highest level of induction (3c).

show that either too much gpGT or too little gpGT supplied from the plasmid does not support efficient assembly and that only at an intermediate concentration of gpGT, corresponding to an intermediate gpGT/gpG ratio, does the complementation work well. This result supports the idea that the ratio of gpG to gpGT is important for assembly.

When the T domain of gpGT (“gpT”) is expressed alone from the plasmid and supplied to the phage that makes gpG alone (Fig. 5b, 3a–3f), a weak complementation signal can be seen, but it is exquisitely dependent on the level of expression from the plasmid. This shows that gpT can have some limited functional activity and, because this activity was not evident when gpT was supplied to the phage that makes neither gpG nor gpGT (Fig. 5a, 3a–3f), we conclude that both gpG and gpT must be present in order for the activity of gpT to be realized. In parallel to what we saw when gpGT was supplied to the phage producing only gpG (Fig. 5b, 2a–f), the gpT-dependent tail assembly activity decreases at the highest level of gpT expression, arguing that gpT is most effective at a particular gpT/gpG ratio. Importantly, the amount of complementation activity

at the optimal gpT/gpG ratio (Fig. 5b, 3e) is only about 1% of the complementation activity supported by the optimal gpGT/gpG ratio (Fig. 5b, 2d). This result argues that the physical connection between the G and T domains of gpGT is important for achieving efficient assembly of functional tails.

Figure 5c shows the complementation results from plating our wild-type phage λc/857 on the same set of plasmids. Expression of the normal ratio of gpG and gpGT from plasmid pBAD33-GT at any level has no effect on the growth of wild-type phage (Fig. 5c, 1a–f), but at the highest concentration of arabinose induction of pBAD33-GT-mut, plasmid expression clearly inhibits the growth of wild-type phage (Fig. 5c, 2f). This again is consistent with the explanation that the excess gpGT provided from the plasmid disturbs the ratio of gpG and gpGT. The effect of disturbance of pBAD33-GT-mut on wild-type phage is not as severe as that on λTam, even though in theory the ratio is disturbed more in wild-type phage under higher concentration of arabinose because wild-type phage produces gpGT too [Eq. (1)]. A possible explanation for this observation is given in the Discussion. We also noted a weak inhibitory effect of the highest level of expression of “gpT” on the growth of wild-type phage. This is shown by the slightly poorer growth of λc/857 on the cells producing maximal “gpT” (Fig. 5c, 3f). This supports the conclusion that “gpT” can have biological activity, even when it is not properly attached to gpG, if it is expressed at sufficiently high levels.

## Discussion

### G is essential for tail assembly

The conservation of the frameshift strategy at the region corresponding to the λ G/GT region among a broad range of tailed phages [6] argues that the frameshift plays a central role in tail assembly and makes it very interesting to know what function gpG and gpGT provide in the tail assembly pathway. The protein gpG is expressed at about the same high level as major tail protein gpV, but it is not incorporated into mature tails [5,12]. It is known from earlier experiments that gpGT is essential, as amber mutations in the T region block tail assembly [1]. However, evidence supporting an essential role for gpG was lacking because any mutation in the G region will also affect gpGT. To address this question, we created a plasmid-based tail assembly system that can produce active tails and found that introducing a mutation that allows gpGT to be expressed without frameshifting abolishes tail assembly. This result is consistent with the view that gpG is essential. However, the mutation also resulted in a 30-fold increase of gpGT relative to the other tail proteins. That is to say, the relative ratios of the tail proteins have been altered due to the mutation. Consequently, the inability of the gpGT-overexpressing plasmid pETail5 to produce active tails could be solely the result of a non-functional ratio of tail proteins. To further investigate the role of gpG in tail production, we used a plasmid-by-phage complementation system in which plasmids providing gpG and/or gpGT *in trans* were tested for their ability to complement amber mutations in the G or T regions of an infecting phage. From the different combinations of complementation tests (Fig. 4), we can draw the following conclusions: (1) gpG and gpGT produced from the plasmid are functional, as they can complement amber mutations in the phage; (2) complementation only works when both gpG and gpGT are present, either provided from a phage or from a plasmid. These results argue strongly that gpG is essential for tail assembly. They also confirm the essential nature of gpGT.

#### The gpG/gpGT ratio is important

The effect of the λ tail gene frameshift is twofold. On the one hand, it creates two proteins, gpG and gpGT, with gpGT containing almost the entire sequence of gpG. On the other hand, the molar ratio of the two proteins is determined by the efficiency of frameshifting. The arabinose-controlled plasmid by phage complementation system provides us a tool to regulate the relative amounts of gpG and gpGT independently of the frameshift by regulating the expression of gpG and gpGT from plasmids. When we modulated the expression of gpGT by inducing under a wide range of arabinose concentrations, we found that there is a gpGT expression level at which complementation works best, and this argues that there is a gpGT/gpG ratio that is optimal for tail assembly.

A key assumption in the experiments with the arabinose-mediated expression system is that a higher concentration of arabinose will result in higher expression from the plasmid in each individual cell. It has been shown for the plasmids we used here that a higher concentration of arabinose results in a higher level of protein expression in cell cultures and that the level of protein expression can be modulated linearly over a range of arabinose concentrations [21]. This result can be explained simply if the level of the expressed protein in each individual cell is modulated by the level of the arabinose inducer. However, it is also consistent with a model in which individual cells are either fully repressed or fully induced, with the graded response to inducer reflecting the fraction of the population in the fully induced state. It has been argued that the latter scenario is likely to be the case [22]. However, our results appear to be incompatible with the latter scenario and consistent with the view that protein expression is modulated in a graded manner in each

individual cell. In the complementation experiment in which gpGT is supplied to the phage that cannot make gpGT, the level of complementation at first increases with increasing amounts of arabinose inducer but then decreases sharply at the highest levels of arabinose. This decrease at high arabinose is simply explained if gpGT is increasing in each cell to a point where the gpGT/gpG ratio is too high to allow tail assembly. It is inconsistent with the model in which the fraction of fully induced cells in the population is increasing with increasing arabinose, unless we assume that high concentrations of arabinose itself are strongly inhibitory to tail assembly. We can rule out the latter possibility because in experiments in which the normal gpGT/gpG ratio is maintained (e.g., Fig. 5c, 1a–f), maximal levels of plaque formation are seen at all arabinose concentrations, including the highest.

We have also investigated the significance of the gpGT/gpG ratio using a variation of the complementation conditions used in Fig. 4: the basal levels of protein expression levels of gpG and/or gpGT from T7 promoter-based plasmids [uninduced in host BL21(DE3)Δtail] were elevated by removing the pLysS plasmid that was present in the experiment shown in Fig. 4 [23]. These experiments recapitulated the part of the arabinose results showing that complementation efficiency declines substantially when too much gpGT is supplied (data not shown). Taken together, the complementation results provide strong evidence that efficient assembly of tails requires not only the presence of both gpG and gpGT but also their presence in the appropriate molar ratio.

gpG and gpGT act as chaperones for gpH and recruit the major tail protein for assembly

In a separate publication [24], we show that both gpG and the “G” part of gpGT bind to almost the entire length of gpH and thus appear to act as chaperones for the tail tape measure protein. We also show that gpH only functions efficiently when complexed with gpG and gpGT and we believe that gpG and gpGT keep the tape measure protein soluble and competent for assembly. In addition, we show that the “T” part of gpGT binds to the lambda major tail protein, gpV, and also induces gpV to polymerize into tubular forms that resemble tails. These biochemical results place constraints on how we can understand the roles of gpG and gpGT in tail assembly. We suggest that tape measure protein gpH, as a coiled coil of a few molecules, is initially coated with a mixture of gpG and gpGT and then the gpG and gpGT are replaced by the major tail subunit gpV, which forms a tube around the coiled coil of gpH. We believe that this transition most likely includes a crucial role for the activity of the “T” domain of gpGT in binding gpV and in converting it to a polymerization-competent form.

Possibly, it is successful polymerization of gpV around gpH that displaces the gpG and gpGT chaperones. Testing this or any other model for how gpV becomes part of the tail and how gpG and gpGT depart will require, we believe, microscopic or other structural evidence about the nature of the complex of the gpG and gpGT chaperones with the tape measure protein. In particular, it will be important to determine how molecules of gpG and gpGT are distributed in the complex, as that is likely to illuminate the question of why the gpG/gpGT ratio is important for correct function. None of this information is currently available.

The physical linkage between the G and T domains of gpGT is important for tail assembly

A consequence of the frameshift mechanism of expression is that the T protein domain is always found joined to the G domain as a part of gpGT and never as a free-standing protein. Given the strong conservation of the frameshift mechanism among tailed phages, this suggests that the physical connection between the G and T domains of gpGT may be important for the function of the T domain. Our complementation experiments provide support for this view. When the T domain is artificially expressed as a separate protein (gpT), it has only a very weak ability to complement a phage that cannot make gpGT. As with the complementation of the same phage by a plasmid producing gpGT, the complementation by gpT shows a maximum at a particular gpT/gpG ratio. However, the maximum level of complementation provided by gpT is reduced at least 100-fold relative to the parallel complementation by gpGT. Since the only difference between these two situations is ostensibly the presence or absence of the physical connection between the G and T domains of gpGT, this argues that physical connection is an essential part of the function of the T domain and therefore an essential part of tail assembly.

Materials and Methods

Plasmid construction

pETail4: 16 kB BamHI-EcoRI fragment of λ DNA containing the entire 11 tail genes was cloned into pBluescript(sk-) (Stratagene) BamHI-EcoRI sites to create pBS-λ-tail I. The large fragment of pBS-λ-tail I BswI-HindIII digestion was filled in and circularized to create pBS-λ-tail 3. pBS-λ-tail 3 was cut with BamHI, filled in, and then cut with NsiI and ligated to 2.6 kb BsaAI-NsiI fragment of pBS-λ-tail I to create pBStail 5. The Sacl-Xhol fragment of pBStail 5 was inserted between the Sacl-Xhol sites of pET21+ to create pETail I. BstBI-Xhol digestion of PCR fragment produced by using primers J1 (gaggagtttc-gaaag), J2 (ggctcgagacgaacctctgtaac), and pETail I as

template was inserted back into the BstBI-Xhol sites of pETail I. The resulting construct removes gene *lom*: pETail 4 contains all of genes Z, U, V, G, T, H, M, L, K, I, and J, including 236 bp before the start of gene Z and 30 bp after the end of gene J.

- pETail5: PstI-MscI fragment of pBRUM-1 (Uma Chandran, PhD thesis, 1990, University of Pittsburgh) containing part of G–T was cloned into PstI and EcoRV sites of pBluescript(sk-) to create pBS-GT.
- The PCR product using GT1 (gccctgcagcgcatggagcatctc) and GT2 (gaaggcctttcccgcagaaaacagg) as primer and pBRUM-1 as template was cut with PstI-Stul and XmnI-partial digest vector. The resulting construct (pBS-GT-mut) added one more G right after the slippery sequence, which allowed gpGT to be synthesized without a frameshift. pETail5 was created by replacing the Apal-Nsil fragment of pETail4 with the Apal-Nsil fragment from pBS-GT-mut. The resulting plasmid contains all the tail genes and GT-fusion mutation.
- pT7-5-GT: MscI fragment of pBRUM-1 (Uma Chandran, PhD thesis, 1990, University of Pittsburgh) was cloned into pT7-5 Smal site to create pT7-5-VT. The 1-kb Xbal-HincII fragment containing G/GT of pT7-5-VT was inserted into Xbal-Smal sites of pT7-5. The resulting plasmid will express gpG and gpGT.
- pT7-5-GT-mut: The Apal-Nsil fragment of pBS-GT-mut was inserted into pT7-5-VT Apal-Nsil site to create pT7-5-VT-mut. The 1-kb Xbal-HincII fragment of pT7-5-VT-mut was inserted into Xbal-Smal sites of pT7-5 to create pT7-5-GT-mut.
- pBAD33-GT: Sacl-HindIII fragment of pT7-5-GT was inserted into Sacl-HindIII sites of pBAD33.
- pBAD33-GT-mut: Sacl-HindIII fragment of pT7-5-GT-mut was inserted into Sacl-HindIII sites of pBAD33.
- pET21a-T: pET21a was digested with EcoRI, blunted with mung bean nuclease, and then cut with Hind III. The resulting vector was ligated to the pBS-GT XmnI-HindIII fragment to create pET21a-T. The resulting plasmid contains only the T part of GT, fused to the T7 tag from the vector.
- pBAD33-T: pET21a was digested with EcoRI, blunted with mung bean nuclease, and then cut with Hind III. The resulting vector was ligated to the pBS-GT XmnI-HindIII fragment to create pET21a-T. The resulting plasmid contains only the T part of GT and fused to the T7 tag from the vector. The Xbal-HindIII fragment of pET21a-T was inserted into Xbal-HindIII sites of pBAD33 to create pBAD33-T.

and the transformed cells were plated on an LB/CAM plate and incubated at 42 °C. Because of the *ts* replication origin, cells with chloramphenicol resistance must have the plasmid integrated into the chromosome by homologous recombination. The colonies that grew were plated onto a plate with 5% sucrose at 30 °C to select for cells that lost the *SacB* gene. The loss of the plasmid is due to a second recombination event that either leaves the original chromosome intact or a deletion. The cells with the deletion were identified by colony PCR and designated BL21(DE3Δtail).

### Protein expression

T7 promoter-controlled plasmids were expressed in BL21(DE3) or BL21(DE3)pLysS cells [23,26] as described previously [18], except BL21(DE3Δtail) was used. Overnight cells grown in LB/Amp (50 μg/ml Amp) [BL21(DE3)] or LB/Amp/Cam (50 μg/ml Amp, 25 μg/ml Cam) [BL21(DE3)pLysS] were inoculated with 1:2000 dilution. The cells were grown at 37 °C with vigorous shaking to a density of approximately 4 × 10^8^ cells/ml and induced with IPTG at a final concentration of 0.4 mM for pT7-5-based vectors and 1 mM for pET21-based vectors (Novagen). The cells were grown for another 3.5 h, chilled, and harvested by centrifugation. Radiolabeling was performed as previously described [27]. Briefly, BL21(DE3) containing appropriate plasmid was grown at 37 °C in RG-glucose medium until the culture was at ~4 × 10^8^ cells/ml. IPTG was added at a final concentration of 0.4 mM to induce for 15 min after which rifampicin was added to a final concentration of 0.2 mg/ml, to shut down expression of host genes for 15 min. ^35^S-methionine was added to 10 μCi/ml and the culture was incubated for another 15 min. The cells were then harvested by centrifugation for further analysis.

### Tail purification

Plasmid pETail4 or pT7-5-Tail4 containing the complete set of tail genes was expressed in BL21(DE3) pLysS cells as described above. Induced cells were harvested and lysed in lysis buffer (50 mM Tris–HCl, pH 8.0, and 5 mM ethylenediaminetetraacetic acid). The lysate was clarified by centrifugation, and tails were precipitated by adding potassium glutamate to 0.7 M and polyethylene glycol 8000 to 5.0% (w/v) followed by sedimentation to collect the precipitate. The pellet was resuspended in TKG buffer (20 mM Tris–HCl, pH 7.5, and 100 mM potassium glutamate), loaded onto a 10–30% glycerol gradient and centrifuged in the SW28 rotor (Beckman) at 27 krpm for 5 h. The tail band was extracted with a 20-G needle and the tails were pelleted by ultracentrifugation in a Ti45 rotor (Beckman) at 40,000 rpm for 4.5 h. The tail pellet was resuspended in a small volume, typically 1 ml (from 1 L of culture), of TKG buffer.

### SDS-PAGE

SDS gel electrophoresis was performed as described in Ref. [28] in a 1-mm mini-gel apparatus. Protein samples were mixed with proper amount of 4× SDS sample buffer and boiled for 3 min. The samples were then loaded onto a

### Modification of the expression host

The host used to express T7 promoter-based plasmids, BL21(DE3), has a λ derivative prophage, DE3, bearing the T7 RNA polymerase gene integrated into the chromosome. To avoid interference from the λ tail genes in the DE3 prophage, either by *trans* expression or by recombination with the plasmid, we knocked out the tail genes using an integrating suicide vector gene replacement strategy [25]. DNA flanking both sides of the region to be deleted was cloned into the PKO5 vector [25]. PKO5 has a temperature-sensitive replication origin from pSC101, a chloramphenicol resistance gene (CAM), and a *SacB* gene that is a counterselective marker in the presence of sucrose. The plasmid was transformed into BL21(DE3)

gel and run at 150 V constant voltage until the blue dye ran out of the gel.

## Spot EOP complementation tests

Spot tests compare the relative EOP (efficiency of plating) of amber mutant phage (lacking gpGT, or both gpG and gpGT) grown on hosts synthesizing various amounts and ratios of gpG and gpGT (from a plasmid). Spot tests are variants of phage T4 techniques [29,30] and have also been used with HK97 [31]. Overnight culture of cells carrying the appropriate plasmid (150 μl) was plated in 2.5 ml of soft agar overlays onto tryptone agar plates with appropriate antibiotics [and for some experiments, arabinose was added to different levels to induce varying levels of gene expression: 2 × 10⁻⁶%, 2 × 10⁻⁵%, 2 × 10⁻⁴%, 2 × 10⁻³%, or 2 × 10⁻²% (w/v) arabinose was added]. The soft agar on the plates was allowed to set for a few minutes before proceeding. Phage stocks (wild type or mutant) were serially diluted to obtain a set of stocks with 10⁸, 10⁷, 10⁶, 10⁵, 10⁴, and 10³ plaque-forming units per milliliter and a 4-μl aliquot of each dilution was spotted onto the lawn. The plate was incubated at 37 °C overnight and complementation was scored by recording the number of clear spots formed by growth of phage within the spot or by recording an image of the plate using a scanner or camera. The number of spots appears to be a true indicator of complementation, as distinguished from marker rescue due to recombination between the phage and plasmid genes. This is internally controlled for in the panels of Fig. 5 where in each vertical set of data the opportunity for recombination is always the same (both plasmid and phage are present), but spots (indicating complementation) appear only at particular levels of the inducer, arabinose.

---

## Acknowledgements

This work was supported by National Institutes of Health grant R01 GM47795 to R.W.H. We thank Jon Beckwith for providing us the pBAD vector series.

Received 24 April 2013;  
Received in revised form 21 June 2013;  
Accepted 2 July 2013  
Available online 12 July 2013  

### Keywords:
assembly chaperones;  
protein complex assembly;  
protein–protein interactions;  
length regulation;  
bacteriophage assembly  

Present address: J. Xu, Mason Business Center, The Procter & Gamble Company, 8700S Mason-Montgomery Road, Mason, OH 45040, USA.

### Abbreviation used:
orf, open reading frame.

## References

[1] Katsura I, Kuhl PW. Morphogenesis of the tail of bacteriophage lambda. III. Morphogenetic pathway. J Mol Biol 1975;91:257–73.  
[2] Katsura I. Tail assembly and injection. In: Hendrix R, Roberts J, Stahl F, Weisberg RA, editors. Lambda II. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory; 1983. p. 331–46.  
[3] Katsura I. Morphogenesis of bacteriophage lambda tail. Polymorphism in the assembly of the major tail protein. J Mol Biol 1976;107:307–26.  
[4] Katsura I, Kuhl PW. Morphogenesis of the tail of bacteriophage lambda. II. In vitro formation and properties of phage particles with extra long tails. Virology 1975;63:238–51.  
[5] Levin ME, Hendrix RW, Casjens SR. A programmed translational frameshift is required for the synthesis of a bacteriophage lambda tail assembly protein. J Mol Biol 1993;234:124–39.  
[6] Xu J, Hendrix RW, Duda RL. Conserved translational frameshift in dsDNA bacteriophage tail assembly genes. Mol Cell 2004;16:11–21.  
[7] Parkinson JS. Genetics of the left arm of the chromosome of bacteriophage lambda. Genetics 1968;59:311–25.  
[8] Buchwald M, Steed-Glaister P, Siminovitch L. The morphogenesis of bacteriophage lambda. I. Purification and characterization of lambda heads and lambda tails. Virology 1970;42:375–89.  
[9] Buchwald M, Siminovitch L. Production of serum-blocking material by mutants of the left arm of the lambda chromosome. Virology 1969;38:1–7.  
[10] Dove WF. Action of the lambda chromosome. I. Control of functions late in bacteriophage development. J Mol Biol 1966;19:187–201.  
[11] Mount DW, Harris AW, Fuerst CR, Siminovitch L. Mutations in bacteriophage lambda affecting particle morphogenesis. Virology 1968;35:134–49.  
[12] Murialdo H, Siminovitch L. The morphogenesis of bacteriophage lambda. IV. Identification of gene products and control of the expression of the morphogenetic information. Virology 1972;48:785–823.  
[13] Katsura I. Determination of bacteriophage lambda tail length by a protein ruler. Nature 1987;327:73–5.  
[14] Katsura I, Hendrix RW. Length determination in bacteriophage lambda tails. Cell 1984;39:691–8.  
[15] Ray PN, Pearson ML. Evidence for post-transcriptional control of the morphogenetic genes of bacteriophage lambda. J Mol Biol 1974;85:163–75.  
[16] Sampson LL, Hendrix RW, Huang WM, Casjens SR. Translation initiation controls the relative rates of expression of the bacteriophage lambda late genes. Proc Natl Acad Sci USA 1988;85:5439–43.  
[17] Hendrix RW, Casjens SR. Protein cleavage in bacteriophage lambda tail assembly. Virology 1974;61:156–9.  
[18] Duda RL, Hempel J, Michel H, Shabanowitz J, Hunt D, Hendrix RW. Structural transitions during bacteriophage HK97 head assembly. J Mol Biol 1995;247:618–35.  
[19] Weigle J. Assembly of phage lambda in vitro. Proc Natl Acad Sci USA 1966;55:1462–6.  
[20] Tsui LC, Hendrix RW. Proteolytic processing of phage lambda tail protein gph: timing of the cleavage. Virology 1983;125:257–64.  
[21] Guzman LM, Belin D, Carson MJ, Beckwith J. Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter. J Bacteriol 1995;177:4121–30.

[22] Siegele DA, Hu JC. Gene expression from plasmids containing the araBAD promoter at subsaturating inducer concentrations represents mixed populations. Proc Natl Acad Sci USA 1997;94:8168–72.

[23] Studier FW, Rosenberg AH, Dunn JJ, Dubendorff JW. Use of T7 RNA polymerase to direct expression of cloned genes. Methods Enzymol 1990;185:60–89.

[24] Xu J, Hendrix RW, Duda RL. Chaperone-protein interactions that mediate assembly of the bacteriophage lambda tail to the correct length. J Mol Biol 2013 (accepted, under revision).

[25] Link AJ, Phillips D, Church GM. Methods for generating precise deletions and insertions in the genome of wild-type *Escherichia coli*: application to open reading frame characterization. J Bacteriol 1997;179:6228–37.

[26] Studier FW. Use of bacteriophage T7 lysozyme to improve an inducible T7 expression system. J Mol Biol 1991;219:37–44.

[27] Duda RL, Martinic K, Hendrix RW. Genetic basis of bacteriophage HK97 prohead assembly. J Mol Biol 1995;247:636–47.

[28] Laemmli UK. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 1970;227:680–5.

[29] Benzer S. Fine structure of a genetic region in bacteriophage. Proc Natl Acad Sci USA 1955;41:344–54.

[30] Doermann AH, Boehner L. The identification of complex genotypes in bacteriophage T4. I. Methods. Genetics 1970;66:417–28.

[31] Ross PD, Cheng N, Conway JF, Firek BA, Hendrix RW, Duda RL, Steven AC. Crosslinking renders bacteriophage HK97 capsid maturation irreversible and effects an essential stabilization. EMBO J 2005;24:1352–63.
