
Acta Crystallographica Section D
Biological
Crystallography
ISSN 0907-4449

Trapped intermediates in crystals of the
FMN-dependent oxidase PhzG provide insight into
the final steps of phenazine biosynthesis

Ningna Xu,$^{a,b}$ Ekta Gayanji Ahuja,$^{b,c}$ Petra Janning,$^{d}$
Dmitri Valeryevich Mavrodi,$^{e}$ Linda S. Thomashow$^{f}$ and Wulf Blankenfeldt$^{a,b,g,*}$

$^{a}$Lehrstuhl für Biochemie, Universität Bayreuth,
Universitätsstrasse 30, 95447 Bayreuth,
Germany, $^{b}$Physikalische Biochemie,
Max-Planck-Institut für Molekulare Physiologie,
Otto-Hahn-Strasse 11, 44227 Dortmund,
Germany, $^{c}$Mithros Chemicals Pvt. Ltd.,
IKP Knowledgepark, Turkapally,
Hyderabad 500 078, India, $^{d}$Chemische Biologie, Max-Planck-Institut für Molekulare Physiologie, Otto-Hahn-Strasse 11,
44227 Dortmund, Germany, $^{e}$Department of Plant Pathology, Washington State University,
Pullman, WA 99164-6430, USA, $^{f}$Root Disease and Biological Control Research Unit,
Agricultural Research Service, USDA-ARS,
Pullman, WA 99164-4234, USA, and
$^{g}$Bayreuther Zentrum für Molekulare Biowissenschaften (BZMB), Universität Bayreuth, Germany

Correspondence e-mail:
wulf.blankenfeldt@uni-bayreuth.de

Phenazines are redox-active secondary metabolites that many bacteria produce and secrete into the environment. They are broad-specificity antibiotics, but also act as virulence and survival factors in infectious diseases. Phenazines are derived from chorismic acid, but important details of their biosynthesis are still unclear. For example, three two-electron oxidations seem to be necessary in the final steps of the pathway, while only one oxidase, the FMN-dependent PhzG, is conserved in the phenazine-biosynthesis *phz* operon. Here, crystal structures of PhzG from *Pseudomonas fluorescens* 2-79 and from *Burkholderia lata* 383 in complex with excess FMN and with the phenazine-biosynthesis intermediates hexahydrophenazine-1,6-dicarboxylate and tetrahydrophenazine-1-carboxylate generated *in situ* are reported. Corroborated with biochemical data, these complexes demonstrate that PhzG is the terminal enzyme in phenazine biosynthesis and that its relaxed substrate specificity lets it participate in the generation of both phenazine-1,6-dicarboxylic acid (PDC) and phenazine-1-carboxylic acid (PCA). This suggests that competition between flavin-dependent oxidations through PhzG and spontaneous oxidative decarboxylations determines the ratio of PDC, PCA and unsubstituted phenazine as the products of phenazine biosynthesis. Further, the results indicate that PhzG synthesizes phenazines in their reduced form. These reduced molecules, and not the fully aromatized derivatives, are the likely end products *in vivo*, explaining why only one oxidase is required in the phenazine-biosynthesis pathway.

### 1. Introduction

The phenazines are a class of currently more than 100 secondary metabolites that both Gram-positive and Gram-negative bacteria, mostly from the fluorescent *Pseudomonas*, *Burkholderia* and *Streptomyces* genera, produce and excrete into the environment (Turner & Messenger, 1986; Laursen & Nielsen, 2004; Mavrodi *et al.*, 2006, 2013). They possess a number of biological activities that are in general linked to their redox activity (Price-Whelan *et al.*, 2006). For example, it has been known for a long time that phenazines can reduce O₂ to toxic reactive oxygen species, which explains their action as broad-specificity antibiotics and their function as virulence factors in infectious diseases (Lau, Hassett *et al.*, 2004). More recent data indicate that phenazines may also have important physiological functions within phenazine-producing bacteria themselves. For example, it has been observed that the phenazine derivative pyocyanin leads to decreased levels of NADH in *P. aeruginosa* grown under anoxic conditions. This implies that under oxygen depletion pyocyanin reoxidizes NADH for glycolysis and the TCA cycle, and thus may be

© 2013 International Union of Crystallography
Printed in Singapore – all rights reserved

Acta Cryst. (2013). D69, 1403–1413
doi:10.1107/S0907444913008354 1403

required for survival (Price-Whelan *et al.*, 2007). The importance of phenazines as survival factors has also been shown in a mouse model of lung infections with *P. aeruginosa*, in which the number of bacteria that could be recovered decreased by several orders of magnitude when the genetic material required for phenazine biosynthesis was deleted (Lau, Ran *et al.*, 2004). *P. aeruginosa* is one of the leading agents of nosocomial infections and also chronically infects the lungs in nearly all cases of cystic fibrosis; it is now the main cause of the shortened lifespan of these patients (reviewed in Driscoll *et al.*, 2007). Phenazine biosynthesis should therefore be an attractive drug target, and this has recently been corroborated by an effort to repurpose the approved drug raloxifene (Ho Sui *et al.*, 2012).

Phenazines are intensely coloured molecules that give rise to readily discernible phenotypes. This also led to their discovery over 150 years ago, when pyocyanin was first isolated from purulent wound dressings that had turned blue as a consequence of *P. aeruginosa* infection (Fordoš, 1859). The genes required for phenazine biosynthesis were discovered in the 1990s (Pierson *et al.*, 1995; Mavrodi *et al.*, 1998) and have since been found in all phenazine-producing bacteria. Here phenazines are biosynthesized from chorismic acid, and five distinct enzymes, which are usually encoded within the conserved *phz* operon, are required to generate the ‘core’ phenazines phenazine-1,6-dicarboxylic acid (PDC) and/or phenazine-1-carboxylic acid (PCA) as precursors of strain-specific derivatives. The activities of most of these enzymes have been clarified and mechanistically investigated by biochemical and structural studies in recent years (Fig. 1; reviewed in Mentel *et al.*, 2009). Phenazine biosynthesis begins with the conversion of chorismic acid to 2-amino-2-desoxyisochorismic acid (ADIC) by PhzE, a bifunctional enzyme related to anthranilate synthase (McDonald *et al.*, 2001; Li *et al.*, 2011). Hydrolysis by the isochorismatase PhzD generates trans-2,3-dihydro-3-hydroxyanthranilic acid (DHHA; Parsons *et al.*, 2003), which PhzF isomerizes to the ketone 6-amino-5-oxo-cyclohex-2-ene-1-carboxylic acid (AOCHC; Blankenfeldt *et al.*, 2004; Parsons, Song *et al.*, 2004). Subsequently, two AOCHC molecules condense into a tricyclic intermediate, which was detected as (1R,5aS,6R)-1,2,5,5a,6,7-hexahydrophenazine-1,6-dicar-

Chorismic acid

![Diagram](https://i.imgur.com/yourimage.png)

Figure 1  
Current understanding of phenazine biosynthesis from chorismic acid. Grey arrows indicate uncatalysed steps. The exact role of PhzG is unclear.

boxylic acid 1a (Ahuja *et al.*, 2008). At high AOCHC concentrations condensation to hexahydrophenazine-1,6-dicarboxylic acid (HHPDC) does not require catalysis, but PhzA/B strongly accelerates HHPDC formation *in vitro*, suggesting that the reaction is catalysed by PhzA/B in the cell (Ahuja *et al.*, 2008).

The exact chemistry of the steps after condensation of AOCHC is uncertain because of the instability of HHPDC and the following intermediates. It is also not clear how the same set of enzymes leads to PDC production in one strain and to PCA in another. The available data indicate that the underlying differences must be rooted in the steps past synthesis of HHPDC and in the last remaining conserved enzyme of the *phz* operon, PhzG. We have previously shown that HHPDC undergoes an uncatalyzed oxidative decarboxylation to a not fully structurally characterized isomer of tetrahydrophenazine-1-carboxylic acid (THPCA; shown as isomer 2a in Fig. 1) followed by further spontaneous oxidations that ultimately lead to the formation of PCA, albeit with a low yield (Blankenfeldt *et al.*, 2004; Ahuja *et al.*, 2008). This suggests that PhzG, the remaining conserved enzyme encoded in the *phz* operon, catalyses at least one of these final reactions.

PhzG is a flavin-dependent enzyme related to the well characterized pyridoxine-5'-phosphate oxidase PdxH. Its crystal structure has been determined and a role in the oxidation of tricyclic phenazine-biosynthesis intermediates has been postulated (Parsons, Calabrese *et al.*, 2004), but unequivocal functional annotation is missing. Here, we provide structural and biochemical evidence that PhzG catalyses one or two of the terminal oxidations of the phenazine-biosynthesis pathway. Structures of PhzG from the two phenazine producers *P. fluorescens* 2-79 and *Burkholderia lata* 383 in complex with trapped unstable intermediates indicate that the enzyme possesses relaxed substrate selectivity and oxidizes HHPDC and THPCA isomers without concomitant decarboxylation, thus participating in the production of both PDC and PCA. Furthermore, comparison with the related pyridoxine-5'-phosphate oxidase PdxH also suggests that HHPDC isomer 1a (Fig. 1) is only a rearranged less reactive form of the true product of PhzA/B. Finally, our data provide a rationale for the finding that the *phz* operon contains only one oxidase while biosynthesis of the fully aromatic phenazine moiety would require three two-electron oxidations.

### 2. Materials and methods

#### 2.1. Protein purification

*P. fluorescens* 2-79 PhzG was cloned into pET15b (Merck Millipore) as described by Parsons, Calabrese *et al.* (2004). *phzG* from *B. lata* 383 (UniProt entry Q396C5) was amplified from genomic DNA using primers 5'-AAC AAA CAT ATG AAC ACC AGT CGA TTC-3' and 5'-ACA GAC AGG ATC CTT ATC CGC AAG-3' for ligation into the same vector after digestion with *NdeI* and *BamHI*. The plasmids were transformed into *Escherichia coli* Rosetta2 pLysS cells (Merck Millipore) and the cells were grown at 310 K in Terrific Broth supplemented with 100 mg l⁻¹ ampicillin and with 34 mg l⁻¹ chloramphenicol. At an OD₆₀₀ of 0.7, the temperature was reduced to 293 K and overnight gene expression was induced with 1 mM IPTG. The cells were harvested by centrifugation, lysed and the supernatant was applied onto a HiTrap Chelating column (GE Healthcare Life Sciences) loaded with 100 mM NiSO₄ and subsequently equilibrated with buffer A (300 mM NaCl, 50 mM Na₂HPO₄ pH 8). The column was washed with 2% buffer B (buffer A supplemented with 1 M imidazole) and then eluted with a gradient to 50% buffer B. Fractions that contained the desired protein were pooled, concentrated and dialysed against 150 mM Tris-HCl pH 8 supplemented with one unit of thrombin per 20 mg protein to remove the His₆ tag before application onto a Superdex 75 size-exclusion column in the same buffer. Pure proteins were concentrated to 10 mg ml⁻¹ or higher, flash-frozen in small aliquots using liquid nitrogen and stored at 193 K until further use, since no differences in crystallization behaviour or activity could be noted between fresh and frozen protein samples.

Calabrese *et al.* (2004) but with PEG 4000 replaced by PEG 3350 as the precipitant. The optimized reservoir was composed of 21–24%(w/v) PEG 3350, 0.2*M* ammonium sulfate, 0.1*M* bis-tris pH 6.2–6.6. Additionally, the protein was incubated with 1 mM (for complexes with intermediates) or 10 mM FMN (for the complex with a second FMN bound to the active site) prior to setup of crystallization experiments and crystals were grown at room temperature from hanging drops composed of 1 μl protein solution mixed with 1 μl mother liquor. The reservoir volume was 500 μM in all cases. Cryoprotection was achieved by washing crystals briefly in mother liquor in which the concentration of PEG 3350 was increased to 30%(w/v) (for the complex with a second FMN) or in mother liquor supplemented with 5%(v/v) glycerol (for the complexes with intermediates).

*B. cepacia* PhzG was diluted to ~10 mg ml⁻¹ in 20 mM Tris-HCl pH 8.0, 1 mM EDTA, 1 mM DTT, 0.1 mM FMN and crystallization conditions were identified using the Classics, PACT and JCSG+ Suites from Qiagen. Crystals for diffraction data collection were obtained from hanging drops consisting of 1 μl protein solution plus 1 μl reservoir solution after 1–2 d at room temperature with a 500 μl reservoir consisting of 16–20%(w/v) PEG 3350, 0.1*M* trisodium citrate pH 5.2–5.6. The crystals were cryoprotected by washing them in 25%(w/v) PEG 3350 supplemented with 5%(v/v) PEG 400.

Complexes with the intermediates HHPDC and THPCA were generated by soaking. 100 mM DHHA (kindly donated by DSM Anti Infectives, Delft, The Netherlands) in 200 mM Tris-HCl pH 7.5 was reacted with 15 μM PhzF and 30 μM PhzA/B from *B. lata* 383 for 30 min at room temperature and then used as is (*i.e.* without removing PhzF and PhzA/B) to prepare a soaking solution from 50%(w/v) PEG and 1*M* salt and buffer stock solutions supplemented with the reaction mixture to the desired concentration. Crystals were then soaked for between 3 h and 5 d before flash-cooling in liquid nitrogen with an appropriate cryoprotectant.

Diffraction data were collected at 100 K on beamlines ID29 at the European Synchrotron Radiation Facility (Grenoble, France) and X10SA at the Swiss Light Source (Paul Scherrer Institute, Villigen, Switzerland). All data were indexed and integrated with XDS (Kabsch, 2010) and scaled with AIMLESS from the CCP4 package (Winn *et al.*, 2011). The structure of *B. cepacia* PhzG was solved by molecular replacement with MOLREP (Vagin & Teplyakov, 2010) using the structure of PhzG from *P. fluorescens* 2-79 (PDB entry 1ty9; Parsons, Calabrese *et al.*, 2004) as a search model. All structures were refined by alternating rounds of manual adjustment in Coot (Emsley *et al.*, 2010) and maximum-likelihood refinement in PHENIX (Adams *et al.*, 2010) using anisotropic *B*-factor refinement for all atoms since this led to significantly better free *R* factors in all cases (a decrease of 0.5% compared with other strategies). Data-collection statistics are provided in Table 1 and Supplementary Table S1$^1$;

---

$^1$ Supplementary material has been deposited in the IUCr electronic archive (Reference: CB5027). Services for accessing this material are described at the back of the journal.

### 2.2. Crystallization, data collection and refinement

Crystals of *P. fluorescens* PhzG were obtained under conditions similar to those previously described by Parsons,

research papers

Table 1  
Data-collection statistics for *P. fluorescens* 2-79 PhzG complexes.  
Values in parentheses are for the highest resolution shell. All data sets were collected from single crystals.

| Beamline† | Complex with excess FMN | Complex with 1a | Complex with 2a (1d) | Complex with 2a (5d) |
|-----------|--------------------------|------------------|----------------------|----------------------|
| Wavelength (Å) | ID 29, ESRF | X10SA, SLS | X10SA, SLS | X10SA, SLS |
|             | 0.9791 | 0.98004 | 0.98004 | 0.98004 |
| Resolution range (Å) | 19–1.33 (1.35–1.33) | 45–1.42 (1.44–1.42) | 44–1.56 (1.59–1.56) | 44–1.45 (1.47–1.45) |
| Space group | $P2_12_12_1$ | $P2_12_12_1$ | $P2_12_12_1$ | $P2_12_12_1$ |
| Unit-cell parameters (Å) | $a = 57.74, b = 63.69,$<br>$c = 133.50$ | $a = 57.59, b = 63.42,$<br>$c = 133.50$ | $a = 57.43, b = 63.29,$<br>$c = 133.00$ | $a = 57.67, b = 63.58,$<br>$c = 133.20$ |
| Mosaicity‡ (°) | 0.181 | 0.158 | 0.121 | 0.097 |
| Total No. of measured reflections | 446194 (20533) | 390047 (18825) | 296394 (14282) | 371363 (17912) |
| Unique reflections | 113420 (5568) | 89891 (4296) | 69696 (3410) | 87433 (4240) |
| Multiplicity | 3.9 (3.7) | 4.3 (4.4) | 4.3 (4.2) | 4.2 (4.2) |
| Mean $I/\sigma(I)$ | 10.2 (2.2) | 20.3 (2.0) | 15.4 (2.0) | 15.9 (2.0) |
| Completeness (%) | 99.8 (100) | 96.9 (94.5) | 99.9 (100) | 99.9 (99.9) |
| $R_{\text{meas}}§$ (%) | 8.6 (76.7) | 5.0 (96.9) | 6.4 (84.9) | 5.5 (87.4) |
| $R_{\text{p.i.m.}}∥$ (%) | 4.3 (39.0) | 2.4 (45.6) | 3.1 (40.7) | 2.7 (42.1) |

† ESRF: European Synchrotron Radiation Facility, Grenoble, France. SLS: Swiss Light Source, Paul Scherrer Institute, Villigen, Switzerland. ‡ Mosaicity values reported by XDS (Kabsch, 2010). § $R_{\text{meas}} = \sum_{hkl} \{ N(hkl)/[N(hkl)-1] \}^{1/2} \sum_i |I_i(hkl) - \langle I(hkl) \rangle| / \sum_{hkl} \sum_i I_i(hkl)$, where $N(hkl)$ is the number of observations of the reflection with index $hkl$ and $I_i(hkl)$ is the intensity of its $i$th observation. ∥ $R_{\text{p.i.m.}} = \sum_{hkl} \{ 1/[N(hkl)-1] \}^{1/2} \sum_i |I_i(hkl) - \langle I(hkl) \rangle| / \sum_{hkl} \sum_i I_i(hkl)$ (Weiss, 2001).

Table 2  
Refinement statistics for *P. fluorescens* 2-79 PhzG complexes.  
Values in parentheses are for the highest resolution shell.

| Complex with | Complex with | Complex with | Complex with |
|--------------|--------------|--------------|--------------|
| excess FMN   | 1a           | 2a (1d)      | 2a (5d)      |
| Resolution range (Å) | 19–1.33 (1.35–1.33) | 44–1.42 (1.44–1.42) | 44–1.56 (1.58–1.56) | 44–1.45 (1.47–1.45) |
| $R_{\text{cryst}}$ (%) | 12.1 (23.0) | 11.6 (20.9) | 12.3 (20.9) | 11.8 (21.0) |
| $R_{\text{free}}$ (%) | 15.4 (26.0) | 15.3 (26.1) | 16.0 (26.2) | 15.0 (28.7) |
| No. of non-H atoms |  |  |  |  |
| Protein | 3778 | 3740 | 3677 | 3670 |
| Ion | 20 | 20 | 10 | 5 |
| Ligand | 102 | 108 | 85 | 85 |
| Water | 613 | 476 | 439 | 502 |
| R.m.s. deviations |  |  |  |  |
| Bonds (Å) | 0.012 | 0.017 | 0.006 | 0.010 |
| Angles (°) | 1.360 | 1.667 | 1.088 | 1.322 |
| Average $B$ factors (Å²) |  |  |  |  |
| Protein | 15 | 20 | 23 | 16 |
| Ion | 48 | 59 | 66 | 66 |
| Ligand | 19 | 16 | 17 | 13 |
| Water | 31 | 36 | 40 | 33 |
| Ramachandran plot (%) |  |  |  |  |
| Favoured regions | 98.9 | 98.8 | 98.8 | 99.3 |
| Outliers | 0 | 0 | 0 | 0 |
| MolProbity score† | 1.24 | 1.42 | 1.27 | 1.21 |
| PDB code | 4hms | 4hmt | 4hmw |  |

† As reported by MolProbity (http://molprobity.biochem.duke.edu/; Chen *et al.*, 2010).

refinement statistics and PDB codes are provided in Table 2 and Supplementary Table S2.

### 2.3. HPLC-coupled mass spectrometry

HPLC-coupled mass spectrometry was performed as described previously (Ahuja *et al.*, 2008). Briefly, turnover of reaction mixtures consisting of 1 mM DHHA and 0.03–5 μM *P. fluorescens* 2-79 PhzG in 50 mM Tris–HCl pH 7.5 was initiated with 1 μM recombinant PhzF from the same strain. An autosampler kept at 283 K was used to remove 1 μl samples at 11 min intervals, which were subsequently analysed using an Agilent 1100 HPLC system coupled to an LTQ mass spectrometer (Thermo Fisher Scientific) operated in positive-ion detection mode with atmospheric pressure chemical ionization (APCI). A Waters Atlantis dC18 column (3 μm, 4.6 × 100 mm) was developed at 1 ml min⁻¹ using 0.1% formic acid in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B) with the following gradient: 0–2.2 min, 100% $A$; 2.2–2.25 min, 100–70% $A$; 2.25–7.9 min, 70–30% $A$; 7.9–10 min, 100% $A$. 10 μM caffeine served as an internal inert standard. Mass spectra were quantitatively analysed using the Xcalibur software package (Thermo Fisher Scientific) by integrating the mass signal of each investigated molecular species and normalizing it to that of caffeine.

3. Results and discussion

3.1. The substrates of PhzG are tricyclic molecules

The structures of PhzG from *P. fluorescens* 2-79 and *P. aeruginosa* PAO1 have previously been determined (PDB entries 1ty9 and 1t9m; Parsons, Calabrese *et al.*, 2004). The authors speculated that PhzG is involved in the steps after tricycle formation in phenazine biosynthesis (Parsons, Calabrese *et al.*, 2004), but experimental data supporting this hypothesis are lacking. We obtained the first indication that PhzG may indeed accept tricyclic substrates when we incubated *P. fluorescens* PhzG with a large excess of FMN (10 mM) to enhance crystallization. This led to additional electron density in the vicinity of the FMN cofactor of PhzG which can clearly be identified as a second FMN molecule in one of the active centres (Fig. 2a and Supplementary Fig. S1a). The quality of the electron density indicates that this binding must be weak, and a more detailed analysis reveals that it is dominated by a parallel displaced π-stacking interaction between the two isoalloxazine groups. Parallel displacement of the two isoalloxazines reflects theoretical calculations that identified such a relative orientation as an optimal position for π-stacking of two benzene molecules (Sinnokrot *et al.*, 2002).

While the isoalloxazine moiety is somewhat similar to phenazines and catalyses similar redox chemistry, it is not an optimal surrogate for the true substrates of PhzG because it possesses an aromatic system that extends over three rings while the intermediates in phenazine biosynthesis (HHPDC or THPCA; Fig. 1) are expected to be at most partially aromatic. In order to gain additional insight, we therefore attempted to trap such intermediates in crystals of PhzG from *P. fluorescens* 2-79 by reacting DHHA at a high concentration (100 mM) with PhzF and PhzA/B from *B. lata* 383 for 30 min. PhzF and PhzA/B convert DHHA to the HHPDC isomer 1a (Blankenfeldt *et al.*, 2004; Parsons, Song *et al.*, 2004; Ahuja *et al.*, 2008), which further decomposes to THPCA by oxidative decarboxylation. Surprisingly, when the crystals were soaked for 3 h we observed HHPDC and not THPCA in both active centres of *P. fluorescens* PhzG (Fig. 2b and Supplementary Fig. S1b), despite the fact that the fluorescent pseudomonads have been reported to produce PCA exclusively, which at first suggested that THPCA would be the substrate of the PhzG enzymes from these strains.

![Figure 2](#fig2)

(a)  
(b)  
(c)  
(d)

Figure 2  
$|F_{o}-F_{c}|$ difference electron densities of different ligands before incorporation into the structural model, displayed at $3\sigma$. (a) and (b) are shown as cross-eyed stereo plots. The times provided in (b), (c) and (d) refer to the soaking time before flash-cooling crystals in liquid nitrogen. Note that the electron density of the carboxylate groups at C6 and C1 deteriorates over time, indicating the binding of a mixture of different intermediates and water molecules. $|2F_{o}-F_{c}|$ densities are provided in Supplementary Fig. S1. All molecular presentations were prepared with PyMOL (Schrödinger).

research papers

The electron density was interpreted as HHPDC isomer 1a (see §3.5). Its binding involves hydrogen bonds to the side and main chain of Ser18. This anchors the flexible N-terminus of PhzG (Fig. 3), yet the first 15 amino acids still could not be traced in the electron density. These disordered residues do not seem to be important for catalytic activity, however, since PhzG from *Enterobacter agglomerans* Eh1087 (UniProt entry Q8GPH1) is 12 residues shorter at the N-terminus (Supplementary Fig. S2). Further polar interactions between the protein and HHPDC include hydrogen bonds to the side chains of His90* (where * indicates the other monomer), Ser147* and Arg139*. Arg139*, which forms a salt bridge with Glu17 in the apo structure, now adopts a different rotamer, allowing contacts with both carboxylate groups of HHPDC. The C-terminus at Pro222 is also involved in substrate binding through water-mediated contacts. With respect to the second FMN molecule in the complex with excess FMN, the position of the tricycle is shifted to the side such that two of its rings stack onto the nitrogen-containing rings of the FMN cofactor (Fig. 3 and Supplementary Fig. S3). This indicates that the binding is now dominated by the specific interactions with the protein side chains mentioned above and not by parallel displaced π-stacking as in the case of excess FMN. The isoalloxazine moiety of the cofactor is bent very slightly (by 3° and 5° in the two active centres), which is not significantly different from the other structures described here and in previous reports (Parsons, Calabrese *et al.*, 2004).

Both carboxylate groups of HHPDC point away from the same face of the tricycle, which proves the stereochemical configuration of the intermediates after DHHA postulated in our previous work (Blankenfeldt *et al.*, 2004; Ahuja *et al.*, 2008). It is also interesting to note that the angles at which they project away from the tricycle are not identical (Fig. 2b and Supplementary Fig. S1b). This allows the two possible orientations of the asymmetric HHPDC isomer 1a in the active centre to be distinguished such that the outer ring that contains four $sp^2$-hybridized C atoms interacts with Ser18 (Fig. 3). Cl of 1a is perfectly positioned for oxidation through hydride abstraction by N5 of the cofactor (at a distance of 3.4 Å). However, closer analysis of the chemical requirements of this reaction suggests that HHPDC isomer 1a is probably not well suited for oxidation by PhzG and that the direct substrate is more likely to be its isomer 1 (Fig. 4; see §3.5).

### 3.2. PhzG possesses relaxed substrate selectivity

The complex with HHPDC isomer 1a suggests that flavin-catalysed oxidation by PhzG provides phenazine-biosynthesis intermediates with a pathway to aromaticity that avoids concomitant decarboxylation. Dissociation of the first product tetrahydro-PDC (THPDC), oxidative regeneration of the cofactor and rebinding of THPDC after rotation, followed by a second round of FMN-dependent oxidation, then explains how PDC is biosynthesized (Fig. 4). However, because we have previously observed that HHPDC undergoes a rapid uncatalysed oxidative decarboxylation to an isomer of THPCA (Ahuja *et al.*, 2008) and because *P. fluorescens* strain 2-79 is known to produce PCA and not PDC, the observation of the nondecarboxylated HHPDC in the active centre of *P. fluorescens* 2-79 was at first surprising to us. We therefore investigated whether this finding is a consequence of the artificial *in vitro* conditions introduced by batch-converting a large amount of DHHA. Under these circumstances, the concentration of dissolved

Figure 3  
Overall structure of the PhzG dimer. N and C indicate the positions of the N- and C-termini, respectively. The first 15 and 17 residues of the two chains are disordered. The stereo plot provides details of the binding of HHPDC isomer 1a. Thin black lines show the position of Arg139* in the apo structure (PDB entry 1ty9; Parsons, Calabrese *et al.*, 2004) and the black arrow indicates the likely path of hydride transfer during catalysis.

oxygen in the soaking solution is expected to be ~0.25 mM. The high concentration of HHPDC induced by turnover of 100 mM DHHA will therefore rapidly deplete oxygen via the spontaneous oxidative decarboxylation of HHPDC to small amounts of THPCA, and consequently HHPDC will accumulate in the soaking experiment. In the cell, in contrast, phenazine biosynthesis consists of a number of coupled steps and substrate concentrations are not expected to reach such high values, so that an accumulation of intermediates is unlikely.

In order to circumvent oxygen depletion and to simultaneously ensure high intermediate concentrations, we extended the crystal soaking time from 3 h to 1 d and 5 d, reasoning that this would lead to gradual oxygen replenishment, generation of THPCA isomers and subsequent replacement of HHPDC in the active centre. In the resulting diffraction data sets, the electron density of the carboxylate group at C6 indeed increasingly vanishes and is simultaneously gradually replaced by two water molecules (Figs. 2c and 2d and Supplementary Figs. S1c and S1d). This indicates that the PhzG molecules in

![image](https://i.imgur.com/your_image_url.png)

Figure 4  
Details of the final steps of phenazine biosynthesis. Uncatalysed reactions leading to less reactive intermediates are shown in light grey. The second dehydration step in the PhzA/B-catalysed condensation of two AOCHC molecules generates the central HHPDC isomer **1**, which can then immediately be oxidized by PhzG or undergo oxidative decarboxylation, leading to PDC, PCA or phenazine via the indicated routes. DHPDC and DHPCA (highlighted in green) are the final products of the pathway and act as precursors for strain-specific phenazine derivatives.

the crystal are populated with a mixture of HHPDC and THPCA that increasingly shifts towards the latter. The remaining carboxylate at C1 occupies the same position as in the complex with 1a, thus showing that PhzG can also directly participate in the biosynthesis of PCA by oxidizing THPCA. In contrast to the complex with HHPDC, clear electron density was only observed in one of the active centres of the dimer contained in the asymmetric unit, which was fitted as THPCA isomer 2a [(1S,10aS)-1-(prop-1-en-2-yl)-1,2,10,10a-tetrahydrophenazine], which is likely to be the most stable isomer of this intermediate. The electron density of the carboxylate group at C1 is already weak after 1 d and decays further with increased soaking time, suggesting that oxidative decarboxylation progresses and the crystal contains PDC, PCA and unsubstituted phenazine derivatives (Figs. 2b–2d and Supplementary Figs. S1b–S1d). Similar observations were also made when crystals of PhzG from *B. lata* 383 (45% sequence identity to PhzG from *P. fluorescens* 2-79) were treated analogously (Supplementary Fig. S4). Although the phenazine derivatives produced by this species have not been identified and it is hence not known whether it uses PCA or PDC as a precursor for its strain-specific phenazines, this shows that the principles of substrate selection are conserved in PhzGs from other phenazine-generating strains. This is also confirmed by sequence alignment with the PhzG proteins from the two confirmed PDC producers *E. agglomerans* Eh1087 (EhpE; UniProt entry Q8GPH1_ENTAG; Giddens *et al.*, 2002) and *S. antibioticus* Tü 2706 (EsmA2; UniProt entry H6ACX8_STRAT; Rui *et al.*, 2012), which contain all of the

residues involved in binding HHPDC or THPCA (Supplementary Fig. S2).

Together, the complex structures presented here suggest that PhzG is a somewhat promiscuous enzyme that can oxidize different intermediates of the phenazine-biosynthesis pathway, thereby contributing to the generation of both PDC and PCA, as indicated in Fig. 4. The difference between PCA-producing and PDC-producing strains would then result from the relative activities of PhzG versus PhzA/B and from the availability of oxygen. If the AOCHC-condensing enzyme PhzA/B is highly active and sufficient oxygen is available, the strain produces PCA owing to the fact that FMN-dependent oxidation by PhzG cannot compete with uncatalyzed oxidative decarboxylation of HHPDC to THPCA. If PhzA/B is less active with respect to PhzG, PhzG has sufficient time to oxidize HHPDC and PDC is the main end product of phenazine biosynthesis in this strain. This scenario is also supported by a recent finding of Rui and coworkers, who showed that PhzA/B and PhzG from *S. antibioticus* Tü 2706 (EsmA1 and EsmA2, respectively) lead to larger amounts of PDC than the *P. fluorescens* enzymes when coupled to PhzC-F of *P. fluorescens* (Rui *et al.*, 2012). However, the situation is further complicated by the fact that the FMN cofactor of PhzG needs to be reoxidized for the next round of catalysis. This is likely to involve molecular oxygen, as in the related PdxH enzymes (Kazarinoff & McCormick, 1975), resulting in further competition for the available oxygen, and it is not possible to predict whether reoxidation of the enzyme or the uncatalysed oxidative decarboxylations will be faster.

![Figure 5](#fig:5)

**Figure 5**  
HPLC-MS quantification of different intermediates of phenazine biosynthesis in response to changing concentrations of PhzG over a course of ~6 h. Note that the graphs are not on an absolute scale.

3.3. PhzG increases PDC and PCA production

For experimental verification that PhzG indeed oxidizes both HHPDC and THPCA and thereby increases the production of both PDC and PCA, we used HPLC-MS analysis to semi-quantitatively follow phenazine-biosynthesis intermediates and end products in mixtures in which 1 μM PhzF was used to turnover 1 mM DHHA. We have previously shown that this is sufficient to induce all of the intermediates of the phenazine-biosynthesis pathway, albeit with low efficiency (Blankenfeldt *et al.*, 2004), which may be desirable here to better approximate the expected low cellular concentrations of these compounds. When increasing amounts of PhzG were included in these reaction mixtures, we observed the enhanced production of PDC and PCA at the cost of HHPDC and THPCA (Fig. 5), which demonstrates that HHPDC and THPCA are indeed substrates of PhzG. The strongest increase resulted for PDC, which was nearly undetectable when PhzF was used alone and continued to increase until the PhzF:PhzG ratio was 1:1, whereas the production of PCA had already started to decline again. As hypothesized in the previous section, this indicates that the presence of PhzG alters the balance between FMN-catalysed oxidations and spontaneous oxidative decarboxylations in phenazine biosynthesis. Consequently, the amount of unsubstituted phenazine, which is a 'shunt' byproduct of phenazine biosynthesis and probably arises through two consecutive oxidative decarboxylations of HHPDC, as shown in Fig. 4 (Ahuja *et al.*, 2008), decreases to become almost undetectable with increasing concentrations of PhzG (Fig. 5, bottom right panel).

3.4. The products of PhzG are reduced phenazines

Flavins cannot accept more than two electrons, and the immediate products of PhzG arising from the oxidation of HHPDC or THPCA therefore cannot be the fully aromatic phenazines because reaching aromaticity requires six-electron and four-electron oxidations, respectively. Indeed, careful inspection of the HPLC-MS data led to the identification of another species that co-migrated with Na₂S₂O₄-reduced PCA and that possessed a molecular mass corresponding to 5,10-dihydro-PCA (DHPCA). With increasing concentrations of PhzG, the amount of this species also increased, which indicates that it is a product of PhzG. However, its decay towards the end of the experiment (Fig. 5, top right panel) indicates that this DHPCA is only an unstable intermediate, whereas PDC, PCA and phenazine all showed accumulation, as would be expected for stable end products (Fig. 5, bottom row). Together, these experiments suggest that PhzG produces PDC and PCA in their reduced 5,10-dihydro forms and further that these reduced phenazines are the true final products of core phenazine biosynthesis, since they represent the physiologically relevant reduced electron-shuttle state of phenazines. This is also supported by work with phenazine-modifying enzymes which are coupled to PDC and PCA biosynthesis and thus use the end products of the conserved Phz enzymes as their substrates. For example, Saleh and coworkers observed that only DHPCA was accepted by the phenazine-modifying

enzymes in *S. anulatus* (Saleh *et al.*, 2009). Work in our own group showed that the FAD-dependent monooxygenase PhzS, which catalyses one step in the biosynthesis of the phenazine derivative pyocyanin in *P. aeruginosa*, also requires reduced PCA for *in vitro* turnover (unpublished work). The final oxidation to the fully aromatic phenazines observed in the assays described here will then result from another uncatalysed oxidation by air (Fig. 4).

3.5. The trapped ligands are less reactive isomers of the substrates of PhzG

The positions of HHPDC and THPCA in our crystal structures indicate that oxidation by PhzG proceeds through hydride abstraction from C1 by N5 of FMN (Fig. 3). This is also corroborated by superimposition with a PLP complex of the well studied pyridoxine-5′-phosphate oxidase PdxH, in which the C4A C atom that PdxH oxidizes lines up perfectly with C1 of HHPDC or THPCA (PDB entry 1g79; Safo *et al.*, 2001; Fig. 6a). The reduced FMN probably balances the negative charge that emerges at N1 through a proton shuttle that involves its ribityl moiety and the conserved Lys95 (Fig. 6c).

Because hydride abstraction by PhzG generates a positive charge in its substrates, oxidation must be accompanied by deprotonation, and the only conserved amino acid that could act as the required base in PhzG is His203, which lies in the vicinity of C4 of the ligands observed in the crystal structures. The corresponding His199 in *E. coli* PdxH has been tested by mutagenesis and it was concluded that it does not act as a base in catalysis, since no changes in *k*~cat~ resulted from replacement with alanine. *K*~m~, on the other hand, increased over 200-fold, which was taken as an indication that the histidine is only involved in substrate binding in PdxH (Di Salvo *et al.*, 2002). In PhzG, however, the situation may be different because the atom of the substrate of PdxH that superimposes with C4 of HHPDC/THPCA is a relatively acidic phenolic O atom (O3; Fig. 6a). Therefore, the histidine may indeed only polarize the O3 hydroxo group or maintain its negative charge in PdxH, whereas it acts as a genuine Brønsted base that abstracts a proton from C4 in PhzG (Figs. 6b and 6c).

In order to be deprotonated by His203, the acidity of the H atoms at C4 needs to be increased. This could be achieved if C4 was *sp³* hybridized, carried two H atoms and had neighbouring double bonds that conjugate the evolving free electron pair to the positive charge that emerges during hydride abstraction at C1. Thus, the outer rings of the tricyclic PhzG substrates are expected to have double bonds between C2 and C3 and between C7 and C8, rather than between C3 and C4 and between C8 and C9 as in **1a** and **2a** (Figs. 1, 4, 6b and 6c). The substrates of PhzG are therefore likely to be HHPDC isomer **1** and THPCA isomers **2b** and **2c**. Their oxidation leads to an aromatic ring, either directly as in the reactions from **1** to **2** and from **2c** to DHPCA, or after imine tautomerization as in those from **2b** to DHPCA *via* **3b** and from **2** to 5,10-dihydro-PDC *via* **3** (Fig. 4), and this aromatization will provide further driving force to the reaction.

research papers

![Figure 6](#fig6)

**Figure 6**
Catalytic mechanism of PhzG. (a) Stereo plot of the pyridoxine-5′-phosphate oxidase PdxH in complex with pyridoxine-5′-phosphate (PNP) based on PDB entry 1g79 (Safo *et al.*, 2001). Superimposed HHPDC is shown in thin black lines. PdxH abstracts a hydride from C4A of PNP, and His199 is positioned to abstract a proton from O3 or to keep it negatively charged. (b) Two perpendicular views of a model of PhzG in complex with HHPDC isomer 1. The left panel is a stereo plot. Hydride abstraction at C1 is accompanied by deprotonation at C4 through His203. (c) Proposed chemical mechanism of PhzG-catalysed substrate oxidation.

We have previously investigated the product of the preceding enzyme PhzA/B using one- and two-dimensional HPLC-coupled NMR spectroscopy, and these spectra unequivocally determined the structure as HHPDC isomer 1a (Ahuja *et al.*, 2008). In order to understand the contradiction between these data and the mechanistic preference of HHPDC isomer 1 as a substrate for PhzG, it has to be realised that the NMR data of 1a were measured over several hours and under slightly acidic conditions (Ahuja *et al.*, 2008). Under these circumstances, it is likely that spontaneous rearrangement of 1 to 1a, which is expected to be more stable because of the four conjugated double bonds, proceeds to completion. Since the crystal complex of PhzG with HHPDC was generated over a similar time frame, it is likely that the ligand observed in the crystal structure also is HHPDC isomer 1a. Because of its higher stability and because it is less easily oxidized by PhzG than isomer 1, its turnover in the crystal will be slow, which may have contributed to the successful trapping of intermediates in the experiments described here. A similar isomeric equilibrium has previously been observed for AOCHC, where we detected a species with the same molecular mass but with lower reactivity (Fig. 4; Ahuja *et al.*, 2008).

Taken together, these data lead to the core phenazine-biosynthesis pathway shown in Figs. 1 and 4. At variance with our previous work (Ahuja *et al.*, 2008), we suggest that the immediate product of the condensing enzyme PhzA/B is HHPDC isomer 1, which arises by double-bond conjugation in the second dehydration step of PhzA/B from the intermediate shown in parentheses in Fig. 4. Because 1 is unstable, it can then follow different pathways involving conjugation of its double bonds to give 1a, spontaneous oxidative decarboxylation and/or PhzG-catalysed oxidations, which lead to PDC, PCA and phenazine, respectively.

### 4. Conclusions

Our data show that PhzG catalyses oxidations in the terminal steps of core phenazine biosynthesis. Complexes with different trapped intermediates suggest that the enzyme is not absolutely specific; therefore, it can accept tricyclic intermediates

immediately after they have been generated or after they have already gained partial aromaticity through oxidative decarboxylation. In this regard, the phenazine-biosynthesis pathway is unusual because the end product is not only determined by the enzyme specificity but also by the relative levels of activity with respect to both the participating enzymes and the velocity of uncatalysed reactions involving isomerizations and oxidative decarboxylations. It is possible that the flow through these different routes is further influenced by protein–protein interactions, as has been suggested by McDonald *et al.* (2001), but experimental evidence for this is lacking. The end products of the pathway are reduced phenazines and not the fully aromatic molecules purported in the literature. Together, this explains why the *phz* operon contains only one oxidase while three two-electron oxidations are formally required to generate the aromatic phenazine moiety from chorismic acid.

We thank Roger S. Goody for supporting this work. Holger Schmitte is acknowledged for technical assistance. DHHA was a kind gift from DSM Anti Infectives (Delft, The Netherlands). We are grateful to the European Synchrotron Radiation Facility (Grenoble, France) and to the Swiss Light Source (Paul Scherrer Institute, Villigen, Switzerland) for providing access to their facilities, and to the X-ray communities at the Max Planck Institutes in Dortmund and Heidelberg for help with data collection. This project was supported in part by the Deutsche Forschungsgemeinschaft (grant BL 587/1-1/2 to WB).

### References

Adams, P. D. *et al.* (2010). *Acta Cryst.* D66, 213–221.

Ahuja, E. G., Janning, P., Mentel, M., Graebsch, A., Breinbauer, R., Hiller, W., Costisella, B., Thomashow, L. S., Mavrodi, D. V. & Blankenfeldt, W. (2008). *J. Am. Chem. Soc.* 130, 17053–17061.

Blankenfeldt, W., Kuzin, A. P., Skarina, T., Korniyenko, Y., Tong, L., Bayer, P., Janning, P., Thomashow, L. S. & Mavrodi, D. V. (2004). *Proc. Natl Acad. Sci. USA,* 101, 16431–16436.

Chen, V. B., Arendall, W. B., Headd, J. J., Keedy, D. A., Immormino, R. M., Kapral, G. J., Murray, L. W., Richardson, J. S. & Richardson, D. C. (2010). *Acta Cryst.* D66, 12–21.

Salvo, M. L. di, Ko, T.-P., Musayev, F. N., Raboni, S., Schirch, V. & Safo, M. K. (2002). *J. Mol. Biol.* 315, 385–397.

Driscoll, J. A., Brody, S. L. & Kollef, M. H. (2007). *Drugs,* 67, 351–368.

Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. (2010). *Acta Cryst.* D66, 486–501.

Fordos, M. J. (1859). *Rec. Trav. Soc. Emul. Sci. Pharm.* 3, 30.

Giddens, S. R., Feng, Y. & Mahanty, H. K. (2002). *Mol. Microbiol.* 45, 769–783.

Ho Sui, S. J., Lo, R., Fernandes, A. R., Caulfield, M. D. G., Lerman, J. A., Xie, L., Bourne, P. E., Baillie, D. L. & Brinkman, F. S. L. (2012). *Int. J. Antimicrob. Agents,* 40, 246–251.

Kabsch, W. (2010). *Acta Cryst.* D66, 125–132.

Kazarinoff, M. N. & McCormick, D. B. (1975). *J. Biol. Chem.* 250, 3436–3442.

Lau, G. W., Hassett, D. J., Ran, H. & Kong, F. (2004). *Trends Mol. Med.* 10, 599–606.

Lau, G. W., Ran, H., Kong, F., Hassett, D. J. & Mavrodi, D. (2004). *Infect. Immun.* 72, 4275–4278.

Laursen, J. B. & Nielsen, J. (2004). *Chem. Rev.* 104, 1663–1686.

Li, Q.-A., Mavrodi, D. V., Thomashow, L. S., Roessle, M. & Blankenfeldt, W. (2011). *J. Biol. Chem.* 286, 18213–18221.

Mavrodi, D. V., Blankenfeldt, W. & Thomashow, L. S. (2006). *Annu. Rev. Phytopathol.* 44, 417–445.

Mavrodi, D. V., Ksenzenko, V. N., Bonsall, R. F., Cook, R. J., Boronin, A. M. & Thomashow, L. S. (1998). *J. Bacteriol.* 180, 2541–2548.

Mavrodi, D. V., Parejko, J. A., Mavrodi, O. V., Kwak, Y.-S., Weller, D. M., Blankenfeldt, W. & Thomashow, L. S. (2013). *Environ. Microbiol.* 15, 675–686.

McDonald, M., Mavrodi, D. V., Thomashow, L. S. & Floss, H. G. (2001). *J. Am. Chem. Soc.* 123, 9459–9460.

Mentel, M., Ahuja, E. G., Mavrodi, D. V., Breinbauer, R., Thomashow, L. S. & Blankenfeldt, W. (2009). *Chembiochem,* 10, 2295–2304.

Parsons, J. F., Calabrese, K., Eisenstein, E. & Ladner, J. E. (2003). *Biochemistry,* 42, 5684–5693.

Parsons, J. F., Calabrese, K., Eisenstein, E. & Ladner, J. E. (2004). *Acta Cryst.* D60, 2110–2113.

Parsons, J. F., Song, F., Parsons, L., Calabrese, K., Eisenstein, E. & Ladner, J. E. (2004). *Biochemistry,* 43, 12427–12435.

Pierson, L. S., Gaffney, T., Lam, S. & Gong, F. (1995). *FEMS Microbiol. Lett.* 134, 299–307.

Price-Whelan, A., Dietrich, L. E. & Newman, D. K. (2006). *Nature Chem. Biol.* 2, 71–78.

Price-Whelan, A., Dietrich, L. E. & Newman, D. K. (2007). *J. Bacteriol.* 189, 6372–6381.

Rui, Z., Ye, M., Wang, S., Fujikawa, K., Akerele, B., Aung, M., Floss, H. G., Zhang, W. & Yu, T. W. (2012). *Chem. Biol.* 19, 1116–1125.

Safo, M. K., Musayev, F. N., di Salvo, M. L. & Schirch, V. (2001). *J. Mol. Biol.* 310, 817–826.

Saleh, O., Gust, B., Boll, B., Fiedler, H. P. & Heide, L. (2009). *J. Biol. Chem.* 284, 14439–14447.

Sinnokrot, M. O., Valeev, E. F. & Sherrill, C. D. (2002). *J. Am. Chem. Soc.* 124, 10887–10893.

Turner, J. M. & Messenger, A. J. (1986). *Adv. Microb. Ecol.* 27, 211–275.

Vagin, A. & Teplyakov, A. (2010). *Acta Cryst.* D66, 22–25.

Weiss, M. S. (2001). *J. Appl. Cryst.* 34, 130–135.

Winn, M. D. *et al.* (2011). *Acta Cryst.* D67, 235–242.
