
Molecular Microbiology (2014) 91(5), 887–899

doi:10.1111/mmi.12501

First published online 14 January 2014

# Identification of a broad family of lipid A late acyltransferases with non-canonical substrate specificity

**Erica J. Rubin,**¹ **John P. O’Brien,**² **Petko L. Ivanov,**³ **Jennifer S. Brodbelt**² and **M. Stephen Trent**³*

¹ Institute for Cellular and Molecular Biology,  
² Department of Chemistry and ³Department of Molecular Biosciences, University of Texas at Austin, Austin, TX, USA.

## Summary

Most Gram-negative organisms produce lipopolysaccharide (LPS), a complex macromolecule anchored to the bacterial membrane by the lipid A moiety. Lipid A is synthesized via the Raetz pathway, a conserved nine-step enzymatic process first characterized in *Escherichia coli*. The Epsilonproteobacterium *Helicobacter pylori* uses the Raetz pathway to synthesize lipid A; however, only eight of nine enzymes in the pathway have been identified in this organism. Here, we identify the missing acyltransferase, Jhp0255, which transfers a secondary acyl chain to the 3′-linked primary acyl chain of lipid A, an activity similar to that of *E. coli* LpxM. This enzyme, reannotated as LpxJ due to limited sequence similarity with LpxM, catalyses addition of a C12:0 or C14:0 acyl chain to the 3′-linked primary acyl chain of lipid A, complementing an *E. coli* LpxM mutant. Enzymatic assays demonstrate that LpxJ and homologues in *Campylobacter jejuni* and *Wolinella succinogenes* can act before the 2′ secondary acyltransferase, LpxL, as well as the 3-deoxy-D-manno-octulosonic acid (Kdo) transferase, KdtA. Ultimately, LpxJ is one member of a large class of acyltransferases found in a diverse range of organisms that lack an *E. coli* LpxM homologue, suggesting that LpxJ participates in lipid A biosynthesis in place of an LpxM homologue.

## Introduction

The outer membrane of most Gram-negative bacteria is an asymmetric bilayer, with the inner leaflet composed of phospholipids and the outer leaflet consisting of lipopolysaccharide (LPS) (Funahara and Nikaido, 1980). LPS is anchored to the outer bacterial membrane by the lipid A domain. Lipid A, also known as endotoxin, is a bioactive molecule that triggers the host innate immune response when it binds the Toll-like receptor 4/myeloid differentiation factor 2 (TLR4/MD2) complex (Raetz and Whitfield, 2002). High LPS concentrations experienced during infection can activate signalling cascades that lead to endotoxic shock, which can prove fatal for the host (Poltorak *et al.*, 1998; Hoshino *et al.*, 1999). In this way, lipid A is a major virulence factor for most Gram-negative organisms.

Interestingly, some organisms produce lipid A that does not elicit a strong immune response. For example, *Helicobacter pylori* lipid A is up to 1000-times less immunostimulatory than the typical lipid A of *Escherichia coli* (Muotiala *et al.*, 1992; Ogawa *et al.*, 1997; 2003). The discrepancy in toxicity between *E. coli* and *H. pylori* lipid A can be attributed to important structural differences in the two biomolecules (Fig. 1). *E. coli* generates a β-(1′,6)-linked disaccharide of glucosamine that is bis-phosphorylated, hexa-acylated, and attached to two Kdo sugars (Fig. 1A) (Raetz and Whitfield, 2002). Conversely, the final *H. pylori* lipid A structure displays alterations in acyl chain length and number, phosphate group and Kdo sugar number, as well as addition of a polar functional group, phosphoethanolamine (pEtN) (Fig. 1B, right) (Tran *et al.*, 2004; Stead *et al.*, 2005; 2008; 2010; Cullen *et al.*, 2011). Importantly, such structural changes have been shown to alter the TLR4 response and therefore the innate immune reaction in the host (Raetz *et al.*, 2007; Needham *et al.*, 2013). Indeed, *H. pylori* lipid A is a poor stimulator of the TLR4/MD2 complex, which allows it to evade the host immune response and establish a long-term infection in the gastric epithelium (Ogawa *et al.*, 2003; Lepper *et al.*, 2005; Moran, 2007; Cullen *et al.*, 2011).

As in other Gram-negative organisms, *H. pylori* lipid A is synthesized via the Raetz pathway, which builds the lipid A starting from an activated nucleotide sugar donor (Raetz, 1990). This process occurs in the bacterial cytoplasm and produces a structure very similar to that of *E. coli* lipid A (Fig. 1B, left). This hexa-acylated species is then heavily modified by a 5-enzyme cascade during transport across the inner and outer membranes, yielding the final surface-presented structure (Fig. 1B, right) (Tran *et al.*, 2004; Stead *et al.*, 2005; 2008; Cullen *et al.*, 2011; Rubin and Trent, 2013).

A E. coli

B H. pylori

Ec Lipid IV A

Ec Kdo₂-lipid A

LpxM
G₁₄-ACP

LpxL
C₁₂-ACP

KdtA
(2X) CMP-Kdo

5 modification steps

H/p Lipid A
outer membrane

H/p Kdo₂-Lipid A
Raetz pathway product

LpxLHp
(jhp0265)

Fig. 1. Final steps of the Raetz pathway in *E. coli* and the missing late acyltransferase of *H. pylori*.  
A. In *E. coli*, once lipid IVₐ is synthesized, KdtA then transfers 2-Kdo sugars to the hydroxyl group at the 6′ position. Only then are secondary acyl chains transferred to the 2′- and 3′-primary acyl chains by LpxL and LpxM, respectively, generating first penta-acylated lauroyl-Kdo₂-lipid IVₐ and finally Kdo₂-lipid A. Carbon numbers are shown for reference in the left-most structure.  
B. *H. pylori* produces a hexa-acylated lipid A product via the Raetz pathway (left), which is then modified by a series of enzymes to create the structure presented in the outer membrane (right). Previously unknown was the identity of the acyltransferase that transfers the secondary acyl chain to the 3′-primary acyl chain during Raetz pathway biosynthesis (red question mark). We have identified this enzyme as LpxJₕₚ, encoded by gene jhp0255 in *H. pylori* strain J99.
In $E$. coli lipid A biosynthesis, the Kdo sugars must be present in order for 2′- and 3′-secondary acylation of lipid IV$_A$, catalysed by late acyltransferases LpxL and LpxM respectively (Fig. 1A) (Belunis *et al.*, 1995). Further, the substrate preferences of the late acyltransferases establish their enzymatic order; LpxL acts before LpxM (Clementz *et al.*, 1996; 1997). Here, we characterize a lipid A acyltransferase in $H$. pylori that is functionally similar to but lacks complete or significant sequence homology to $E$. coli LpxM (Table S1). This enzyme, from hereon referred to as LpxJ, transfers the secondary acyl chain to the 3′-linked primary acyl chain of $H$. pylori lipid A, similarly to LpxM in $E$. coli. However, LpxJ can perform acyltransfer regardless of the presence of Kdo sugars and can do so before the $H$. pylori LpxL homologue, Jhp0265 (Stead *et al.*, 2008). The enzymatic flexibility of LpxJ allows for the unprecedented option of multiple routes at the end of the lipid A biosynthesis pathway, a possibility not observed in $E$. coli (Fig. 6). Ultimately, the altered order of 3′-secondary acylation could reflect the importance of this chain in downstream processes, such as transport and presentation of $H$. pylori lipid A. We examine LpxJ homologues in two closely related organisms and demonstrate that these enzymes are also capable of non-ordered 3′-secondary acylation activity. Together, the enzymes studied here represent a previously uncharacterized family of lipid A late acyltransferases with homologues in a wide range of bacterial species. While functionally related to $E$. coli LpxM, these proteins are different at the primary sequence level, resulting in unique enzymatic characteristics and a less stringently ordered lipid A biosynthesis pathway.

### Results

#### H. pylori encodes an enzyme functionally comparable to E. coli LpxM

Previously, we characterized $H$. pylori LpxL$_{Hp}$ (jhp0265), a homologue of $E$. coli LpxL that transfers a C18:0 secondary acyl chain to the 2′-linked acyl chain of $H$. pylori lipid IV$_A$ (Stead *et al.*, 2008). Because $H$. pylori has been shown to produce a hexa-acylated lipid A precursor (Fig. 1B, left) that is further processed by the modification pathway, it follows that an unidentified acyltransferase functionally similar to $E$. coli LpxM is required (Moran *et al.*, 1997; Stead *et al.*, 2005; Tran *et al.*, 2006). Stead *et al.* observed a putative LpxM-like activity by demonstrating that purified $H$. pylori membranes are capable of adding a secondary acyl chain to the 3′-linked primary chain of penta-acylated lipid A substrate *in vitro* (Stead *et al.*, 2008). However, a PSI-BLAST search using LpxM as the query sequence revealed only LpxL$_{Hp}$, suggesting that $H$. pylori encodes a functional equivalent of $E$. coli LpxM that is not identifiable based on sequence homology.

We identified jhp0255 as a strong candidate to encode the ‘missing’ late acyltransferase by consulting the Pfam database, an online tool that groups proteins by predicted functional domains (Punta *et al.*, 2011). Specifically, we looked within the acyltransferase clan (CL0228), which contains proteins with a putative acyl acceptor pocket domain, suggesting a functional role in acyl transfer reactions. This clan has four families: a lipid A acyltransferase family, of which LpxM and LpxL$_{Hp}$ are members, the diacylglycerol acyltransferase family, glycerolipid acyltransferase family, and the domain of unknown function family (DUF374). The DUF374 protein for $H$. pylori is Jhp0255, which is previously uncharacterized and located close on the chromosome to other known LPS biosynthesis genes including $lpxL_{Hp}$ (Alm *et al.*, 1999). Because both Jhp0255 and $E$. coli LpxM are grouped into separate families within the same Pfam clan, they likely share a distant common ancestor that diverged through evolution. At the primary sequence level, however, the $H$. pylori enzyme differs significantly from LpxM (Table S1) and the acyl acceptor domain appears to be the only common sequence thread between the two proteins. Accordingly, we have reannotated Jhp0255 as LpxJ, as it was discovered in $H$. pylori strain J99.

#### LpxJ$_{Hp}$ complements a penta-acylated E. coli lpxM mutant, restoring a hexa-acylated phenotype

To investigate the role of LpxJ$_{Hp}$ in $H$. pylori lipid A biosynthesis, we attempted to generate a mutant strain by replacing its coding sequence in the genome with a kanamycin antibiotic resistance cassette. However, multiple efforts to obtain this mutant were unsuccessful in several $H$. pylori strains, suggesting that the gene product is essential for viability. Indeed, insertional deletion of $lpxJ_{Hp}$ was only successful after providing the bacterium with a second copy of the gene by chromosomal complementation at the $rdxA$ locus (Fig. S1). Inactivation of the $H$. pylori $rdxA$ locus leads to metronidazole resistance, which can be used to select for colonies containing the complemented gene of interest at this locus.

In the absence of a viable deletion mutant, we expressed LpxJ$_{Hp}$ in the $E$. coli $lpxM$ mutant strain BN2 (Table S3), which produces a penta-acylated lipid A, to determine if LpxJ$_{Hp}$ could generate hexa-acylated lipid A species *in vivo*. Indeed, chromatographic separation of radiolabelled lipid A isolated from BN2 expressing LpxJ$_{Hp}$ revealed a migration pattern consistent with the presence of hexa-acylated lipid A (Fig. 2, lane 3). Hexa-acylated lipid A isolated and analysed from strain BN1 served as a positive control (Table S3) (Fig. 2, lane 1). Strain BN2 containing empty vector produced only penta-acylated lipid A (Fig. 2, lane 2), suggesting that LpxJ$_{Hp}$ catalyses transfer of the secondary acyl chain to the 3′-linked primary acyl chain of

© 2013 John Wiley & Sons Ltd, *Molecular Microbiology*, **91**, 887–899
teria may encode homologues of LpxJ instead of LpxM. In two closely related Epsilonproteobacteria, we identified Campylobacter jejuni protein Cjj81176_0482 (LpxJ<sub>Cj</sub>) and Wolinella succinogenes protein Ws1775 (LpxJ<sub>Ws</sub>) as homologues of LpxJ<sub>Hp</sub> and members of the DUF374 family. The *C*. *jejuni* and *W*. *succinogenes* proteins were expressed individually in strain BN2 and radiolabelled lipid A was isolated from each strain. Both proteins complemented the penta-acylated mutant, leading to formation of a hexa-acylated lipid A species (Fig. 2, lanes 4, 5). TLC results for *C*. *jejuni* and *W*. *succinogenes* proteins were confirmed with MALDI-TOF mass spectrometry to show that LpxJ<sub>Cj</sub> transfers a C16:0 acyl chain and LpxJ<sub>Ws</sub> transfers a C12:0 chain to the penta-acylated lipid A framework (Fig. 3C and D).

### DUF374 proteins are found in a diverse range of organisms

Further investigation into Pfam family DUF374 revealed that over 500 species encode at least one protein that can be classified into this predicted acyltransferase family. LpxJ<sub>Hp</sub> has putative homologues in a surprisingly large number of organisms ranging from other bacterial classes including Alpha-, Delta- and Zetaproteobacteria, to other phyla including Fusobacteria, Chlamydiae, Acidobacteria and Spirochaetes (Table 1). Relevant BLAST scores demonstrating homology between LpxJ<sub>Hp</sub> and a number of other DUF374 proteins are provided in Table S2. Of note, the organisms that encode a homologue of LpxJ<sub>Hp</sub> possess a homologue to *E*. *coli* LpxL, but not LpxM (Table 1). Although enzymatic activity has not been confirmed for all members of DUF374, our results strongly suggest that this previously unstudied protein family contains acyltransferases that participate in lipid A biosynthesis in place of LpxM.

#### Fig. 2. LpxJ<sub>Hp</sub> and homologues in two other Epsilonproteobacteria complement a hexa-acylated lipid A phenotype in a penta-acylated *E*. *coli* LpxM mutant. Radiolabelled lipid A from various *E*. *coli* strains was isolated and separated by TLC. Expression of LpxJ<sub>Hp</sub> (lane 3), LpxJ<sub>Cj</sub> (lane 4) and LpxJ<sub>Ws</sub> (lane 5) in strain BN2, which normally produces penta-acylated lipid A (lane 2), complements this penta-acylated strain to a hexa-acylated phenotype (as shown for positive control strain BN1, lane 1). Asterisks (*) in lanes 3 and 4 correspond to lipid A species formed by alternative acylation order (see cartoon at left). The presence of residual penta-acylated product in the LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> strains (dark spots migrating at the same level as BN2 vector control, lane 2) suggests these enzymes may not be as efficient as LpxJ<sub>Hp</sub> in this system.

lipid A. These findings were confirmed by MALDI-TOF mass spectrometry; LpxJ<sub>Hp</sub> transfers a C12:0 or C14:0 chain (Fig. 3B), to generate major molecular ions at *m/z* 1768.3 and 1796.3. The lipid A species that results from transfer of a C14:0 chain by LpxJ<sub>Hp</sub> is almost identical in molecular mass to the lipid A of hexa-acylated control strain BN1 (*m/z* 1796.0) (Fig. S2). Taken together, these results strongly suggest that LpxJ<sub>Hp</sub> transfers a C12:0 or C14:0 secondary acyl chain to the 3′-linked primary acyl chain of *H*. *pylori* lipid A.

### Homologues of LpxJ<sub>Hp</sub> are found in other Epsilonproteobacteria

Due to the unexpected sequence difference between LpxJ<sub>Hp</sub> and *E*. *coli* LpxM, we hypothesized that other bac-

© 2013 John Wiley & Sons Ltd, *Molecular Microbiology*, **91**, 887–899

Table 1. LpxJ homologues in Pfam DUF374.

| Organism                          | Strain         | Class              | E. coli LpxL homologue     | E. coli LpxM homologue | LpxJ homologue       |
|-----------------------------------|----------------|--------------------|----------------------------|-----------------------|----------------------|
| Helicobacter pylori               | J99            | Epsilonproteobacteria | Yes/Jhp0265             | No/–                 | Yes/LpxJ<sub>Hp</sub> |
| Helicobacter hepaticus           | ATCC 51449     | Epsilonproteobacteria | Yes/HH0982             | No/–                 | Yes/HH1275          |
| Campylobacter jejuni              | 81-176         | Epsilonproteobacteria | Yes/Cjj81176_1151      | No/–                 | Yes/LpxJ<sub>Cj</sub> |
| Campylobacter fetus subsp. fetus | 82-40          | Epsilonproteobacteria | Yes/CFF8240_1400       | No/–                 | Yes/CFF8240_1376     |
| Wolinella succinogenes           | DSM1740        | Epsilonproteobacteria | Yes/Ws1908             | No/–                 | Yes/LpxJ<sub>Ws</sub> |
| Bartonella bacilliformis         | KC583          | Alphaproteobacteria   | Yes/BARBAKC583_0617    | No/–                 | Yes/BARBAKC583_1205  |
| Brucella melitensis              | ATCC 23457     | Alphaproteobacteria   | Yes/BMEA_A0889         | No/–                 | Yes/BMEA_A0457      |
| Rickettsia rickettsii            | Hlp #2         | Alphaproteobacteria   | Yes/RPK06045          | No/–                 | Yes/RPK00655        |
| Desulfovibrio vulgaris           | Miyazaki F     | Deltaproteobacteria   | Yes/DvMF_0365          | No/–                 | Yes/DvMF_0366       |
| Fusobacterium nucleatum          | CC53           | Fusobacteriales       | Yes/H848_01687         | No/–                 | Yes/H848_04757      |
| Escherichia coli                 | MG1655         | Gammaproteobacteria   | Yes/LpxL               | Yes/LpxM             | No/–                |
| Salmonella enterica serovar Typhimurium | LT2 | Gammaproteobacteria   | Yes/LpxL               | Yes/LpxM             | No/–                |

suggests that its activity can precede LpxL<sub>Hp</sub> activity, such that secondary acylation at the 3′-primary chain precedes secondary acylation at the 2′-primary chain. This order is reverse of the strict LpxL-then-LpxM order found in *E. coli* and many other Gram-negative organisms (Clementz *et al.*, 1996; 1997).

LpxJ<sub>Hp</sub> does not, however, transfer an acyl chain to hexa-acylated substrate Kdo<sub>2</sub>-lipid A, which already has secondary acyl chains attached to the 2′- and 3′-primary chains (Fig. 4A). With this substrate, any further acylation by LpxJ<sub>Hp</sub> would have to occur at the hydroxyl groups of the primary acyl chains attached to the proximal glucosamine of the lipid A molecule. These results demonstrate that LpxJ<sub>Hp</sub> only transfers an acyl chain to the distal glucosamine (2′- or 3′-primary chain) and not the proximal (2- or 3-primary chain) side of the lipid A molecule. To confirm this result, we expressed LpxJ<sub>Hp</sub> in a tetra-acylated *E. coli* strain (MKV15b, Table S3) to allow the enzyme access to all possible secondary acylation positions, then subjected isolated lipid A to ultraviolet photodissociation (UVPD) mass spectrometry. UVPD of isolated lipid A allowed identification of a fragment ion (*m/z* 738.4, cleavage 12) that represents the unmodified proximal glucosamine of lipid A, which does not change between vector control and the strain expressing LpxJ<sub>Hp</sub> (Fig. S3A and B). This confirms that the acylation pattern of the proximal sugar does not change upon expression of the *Helicobacter* acyltransferase, and therefore acylation by this enzyme must occur on the distal glucosamine.

With regard to acyl chain donor, LpxJ<sub>Hp</sub> is analogous to *E. coli* LpxM in that it can only utilize acyl-ACPs rather than acyl coenzyme A (acyl-CoA), preferring C12:0-ACP and C14:0-ACP (Fig. 4B) (Clementz *et al.*, 1997). Importantly, isolated *H. pylori* lipid A has either a C12:0 or a C14:0 secondary acyl chain at the 3′-primary acyl chain (Cullen *et al.*, 2011), suggesting that our *in vitro* system

closely mimics an *in vivo* environment for this acyltransferase. These results combined with the ability of this enzyme to promote formation of hexa-acylated lipid A in a penta-acylated mutant allow us to definitively conclude that it adds a secondary acyl chain to the 3′-linked primary acyl chain of the lipid A molecule.

Campylobacter and Wolinella LpxJ only use tetra-acylated lipid A as substrate

Enzymatic activities of LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> were assayed with the same *in vitro* system and radiolabelled lipid substrates used for LpxJ<sub>Hp</sub>. Both enzymes were unable to acylate a penta-acylated lipid A substrate and could only act on a tetra-acylated lipid A substrate, independent of Kdosylation state (Fig. 5A). These results were initially confounding, given that both enzymes were capable of rescuing a penta-acylated mutant to a hexa-acylated phenotype *in vivo*. Taken together, this evidence indicates that LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> can only function before their respective 2′-secondary acyltransferase LpxL enzymes, requiring exclusively tetra-acylated substrate for activity (Fig. 6). Similarly to LpxJ<sub>Hp</sub>, neither LpxJ<sub>Cj</sub> nor LpxJ<sub>Ws</sub> utilized hexa-acylated Kdo<sub>2</sub>-lipid A (Fig. 5A). These results were also confirmed by UVPD mass spectrometry of lipid A isolated from strains individually expressing LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub>. The fragment ion representing the unmodified proximal side of the lipid A molecule was observed in lipid A from both strains (*m/z* 738.4, cleavage 12), which demonstrates that acyl chain addition by each enzyme occurs at the distal glucosamine (Fig. S3A, C and D).

In agreement with LpxJ<sub>Hp</sub>, both LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> use acyl-ACP as a donor molecule, but not acyl-CoA, where LpxJ<sub>Cj</sub> can use C16:0 and C18:0 acyl-ACPs and LpxJ<sub>Ws</sub> prefers C12:0 acyl-ACP (Fig. 5B and C). Although the Campylobacter enzyme can transfer C18:0, this is not

A BN2 + Empty Vector

B BN2 + LpxJ<sub>Hp</sub>

C BN2 + LpxJ<sub>Cj</sub>

D BN2 + LpxJ<sub>Ws</sub>

[M-H]<sup>-</sup>
1586.0

[M-H]<sup>-</sup>
1796.3

[M-H]<sup>-</sup>
1824.3

[M-H]<sup>-</sup>
1585.9

Exact mass: 1587.0

Exact mass: 1797.2

Exact mass: 1825.2

Exact mass: 1587.0

[M-H]<sup>-</sup>
1614.0

[M-H]<sup>-</sup>
1768.0

* 

*

% Intensity

% Intensity

Mass/Charge

Mass/Charge

Fig. 3. MALDI-TOF mass spectrometry of *E. coli* LpxM mutant, BN2 expressing acyltransferase genes from three Epsilonproteobacteria.

A. Strain BN2 produces a major molecular ion peak at *m/z* 1586.0, corresponding to penta-acylated lipid A.

B. Expression of LpxJ<sub>Hp</sub> in the BN2 background produces two main peaks at *m/z* 1768.3 and *m/z* 1796.3. The former peak corresponds to addition of a C12:0 group and the latter corresponds to addition of a C14:0 group to the 3′-linked acyl chain of lipid A. The asterisk (*), which indicates the peak at *m/z* 1614.0, represents penta-acylated lipid IV<sub>A</sub> with the fifth chain on the 3′-primary acyl chain, which can form before 2′ secondary acylation occurs. This species is the same as the (*)-designated species in Fig. 2, lane 3.

C. Expression of LpxJ<sub>Cj</sub> in the BN2 background produces a major peak at *m/z* 1824.3, corresponding to C16:0 addition to the lipid A. The peak at *m/z* 1586.0 represents residual penta-acylated lipid A. The asterisk (*), which indicates the peak at *m/z* 1642.1, represents penta-acylated lipid IV<sub>A</sub> with the fifth chain on the 3′-primary acyl chain, which can form before 2′ secondary acylation occurs. This species is the same as the (*)-designated species in Fig. 2, lane 4.

D. Expression of LpxJ<sub>Ws</sub> in the BN2 background produces hexa-acylated lipid A with a peak at 1768.0, corresponding to C12:0 addition to the lipid A.

likely to represent what occurs *in vivo*, as *C. jejuni* does not typically incorporate C18:0 into its lipid A structure. This activity may reflect relaxed substrate specificity or result from the available acyl-ACP substrate pool within the bacterial cell (Cullen *et al.*, 2013).

LpxJ activity can precede LpxL activity

In vitro assay results for the three LpxJ enzymes demonstrate their ability, or necessity, to perform acyltransfer to the 3′-primary chain before their respective LpxL enzymes transfer to the 2′-primary chain. This altered acylation order is unprecedented and in stark contrast to the strict LpxL-then-LpxM order established in *E. coli*. Evidence for reversed secondary acylation order can also be found in the radiolabelled lipid A isolated from strain BN2 expressing LpxJ<sub>Hp</sub> and LpxJ<sub>Cj</sub>. The asterisks (*) in Fig. 2, lanes 3 and 4, mark penta-acylated lipid A species that likely have the fifth chain attached as a secondary acyl chain to the 3′- as opposed to the 2′-primary chain. For example, LpxJ<sub>Cj</sub> transfers a C16:0 chain to the 3′-primary acyl chain, which generates a more hydrophobic product (*, lane 4) than penta-acylated BN2 lipid A, which has a secondary C12:0 chain at the 2′-primary chain (lower spot). The same is true for LpxJ<sub>Hp</sub>, which transfers a C14:0 chain yielding a product (*, lane 3) that is slightly more hydrophobic than penta-acylated BN2 lipid A. Importantly, LpxJ<sub>Hp</sub> is more efficient in this *in vivo* system than the other two enzymes, probably due to its more flexible substrate specificity. Both the alternative penta-acylated species produced by LpxJ<sub>Hp</sub> and LpxJ<sub>Cj</sub> are visible in their MALDI-TOF spectra, annotated with an asterisk (*) in Fig. 3B and C respectively. Because LpxJ<sub>Ws</sub> transfers a C12:0 chain, its unique 3′-penta-acylated structure cannot be discerned from the *E. coli* structure (Fig. 2, lane 5; Fig. 3D).


![Figure 4](https://i.imgur.com/yourimage.png)

Fig. 4. Enzymatic characteristics of LpxJ<sub>Hp</sub>.

A. Membranes from strain BN2 containing empty vector or pWSK29jhp0255 (*lpxJ<sub>Hp</sub>*) were assayed with various [4′-<sup>32</sup>P]-labelled lipid substrates and 5 μM C12:0 acyl-ACP. Membranes from the empty vector strain showed no activity from endogenous *E. coli* proteins, while membranes expressing LpxJ<sub>Hp</sub> acylated all substrates except Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid A. Acylated reaction products are indicated with red asterisks (*).

B. BN2 membranes expressing LpxJ<sub>Hp</sub> were tested with Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid IV<sub>A</sub> and 5 μM acyl-ACPs/acyl-CoAs as labelled. LpxJ<sub>Hp</sub> uses C12:0 and C14:0 acyl-ACPs as donor molecules. Acylated reaction products are indicated with red asterisks (*).

© 2013 John Wiley & Sons Ltd, *Molecular Microbiology*, **91**, 887–899

Fig. 5. Enzymatic characteristics of LpxJ<sub>cj</sub> and LpxJ<sub>ws</sub>.

A. Membranes from strain BN2 containing pWSK29cjj81176_0482 (LpxJ<sub>cj</sub>) or pWSK29ws1775 (LpxJ<sub>ws</sub>) were assayed with various [4′-<sup>32</sup>P]-labelled lipid substrates and 5 μM C12:0 acyl-ACP. LpxJ<sub>cj</sub> and LpxJ<sub>ws</sub> acylate [4′-<sup>32</sup>P] lipid IV<sub>A</sub> and Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid IV<sub>A</sub> substrates. Acylated reaction products are indicated with red asterisks (*). Empty vector control is the same as shown in Fig. 4A.

B and C. LpxJ<sub>cj</sub> membranes (B) and LpxJ<sub>ws</sub> membranes (C) were assayed with Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid IV<sub>A</sub> and 5 μM acyl-ACPs/acyl-CoAs as labelled. LpxJ<sub>cj</sub> uses C16:0 and C18:0 acyl-ACPs as donor molecules, and LpxJ<sub>ws</sub> uses C12:0 acyl-ACP. Acylated reaction products are indicated with red asterisks (*).

### Discussion

Most Gram-negative organisms produce lipid A, a complex bioactive molecule that anchors lipopolysaccharide to the outer bacterial membrane and is required for bacterial viability (Raetz, 1990). Although the lipid A biosynthesis pathway is generally well-conserved across Gram-negative bacteria, there remain exceptions. Here, we identify and characterize a previously unknown lipid A acyltransferase in *H. pylori* and two homologues in closely related Epsilonproteobacteria. These enzymes are distant by sequence from *E. coli* LpxM but perform a similar function, transferring a secondary acyl chain to the existing 3′ hydroxyacyl chain of lipid A (Figs 1 and 6).

Although LpxJ and *E. coli* LpxM both transfer an acyl chain to the same position during lipid A biosynthesis in their respective organisms, they are unexpectedly different at the primary sequence level (Table S1) and possess distinct enzymatic characteristics. The only feature that LpxJ shares with LpxM is the presence of a putative acyl-acceptor binding pocket requisite for acyltransferase activity. Even within this binding pocket, however, there are divergences between LpxJ and LpxM. Specifically, LpxM contains the conserved H(X)<sub>4</sub>D/E active-site motif, which was first identified in the glycerol-3-phosphate acyltransferase protein and is found in glycerophospholipid acyltransferases across bacterial, plant and animal kingdoms (Heath and Rock, 1998; Turnbull et al., 2001;

Tamada et al., 2004; Six et al., 2012). Within this active-site motif the histidine and aspartic acid, or glutamic acid, residues act as an invariant catalytic dyad that activates the acceptor hydroxyl on the 3′-primary acyl chain of lipid A for nucleophilic attack of the acyl-ACP thioester bond (Six et al., 2012). LpxJ and homologues in the DUF374 family lack the proximal co-ordinating aspartic or glutamic acid residue, which suggests that the active site in DUF374-containing proteins has a different structural organization or acts through a different co-ordinating residue (Heath and Rock, 1998). Ultimately, this altered active-site organization could change enzymatic substrate specificity, as the H(X)<sub>4</sub>D/E motif found in LpxL and LpxM is known to contact and co-ordinate the substrate during catalysis.

Escherichia coli LpxM strongly prefers penta-acylated lipid A substrates and only shows activity on a tetra-acylated substrate when artificially overexpressed (Clementz et al., 1997). This preference determines its activity as the last enzyme of the *E. coli* Raetz pathway, generating the LpxL-then-LpxM acylation order. Conversely, a novel feature of LpxJ<sub>Hp</sub> is its ability to act on both penta- and tetra-acylated substrates (Fig. 4A). This suggests that it is capable of adding a secondary chain to the 3′-primary acyl chain before LpxL<sub>Hp</sub> adds a secondary chain to the 2′-primary acyl chain. Therefore, this enzyme does not require presence of a secondary acyl chain at the 2′-primary chain for enzymatic activity. Interestingly,

A H. pylori

B C. jejuni

Fig. 6. Multiple routes are possible at the end of the Raetz pathway in *H. pylori* and *C. jejuni*.

A. Due to the relaxed substrate specificity of LpxJ<sub>Hp</sub>, *H. pylori* lipid A biosynthesis could follow three routes to the hexa-acylated end-product.

B. LpxJ<sub>Cj</sub> is specific for a tetra-acylated substrate regardless of Kdo sugars. Therefore, it could follow two routes to the hexa-acylated lipid A biosynthesis end-product. The same is true in *W. succinogenes* lipid A biosynthesis. This model does not consider the substrate specificities of KdtA and LpxL in these organisms, which are not fully characterized and will likely also determine the precise pathway taken.

LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> add only to a tetra-acylated lipid A, implying that these proteins strictly act in the reverse order of the canonical *E. coli* pathway, specifically requiring the absence of a secondary acyl chain on the 2′-primary chain for activity (Figs 5A and 6). In the case of LpxJ<sub>Hp</sub>, the observed enzymatic flexibility could be the result of altered substrate specificity relative to the *C. jejuni*, *W. succinogenes* and *E. coli* proteins, each of which utilizes a very specific (albeit different) lipid substrate. Further, LpxJ<sub>Hp</sub>, LpxJ<sub>Cj</sub> and LpxJ<sub>Ws</sub> are able to perform acyltransfer regardless of the presence of Kdo sugars (Figs 4A and 5A); this introduces another deviation from the *E. coli* pathway, which requires Kdosylated lipid A before LpxL and LpxM can add the secondary acyl chains (Clementz *et al.*, 1996; 1997). With this, the enzymatic characteristics of LpxJ introduce the possibility of several routes in the final steps of the Raetz pathway, specifically, altered acylation and Kdosylation order (Fig. 6). Some uncertainties remain as to how the observed *in vitro* enzymatic characteristics of these acyltransferases compare to precise activities *in vivo*. For example, LpxL<sub>Hp</sub> has not yet been shown to use LpxJ<sub>Hp</sub>-acylated Kdo<sub>2</sub>-lipid IV<sub>A</sub> (that is, Kdo<sub>2</sub>-lipid IV<sub>A</sub> with a secondary acyl chain on the 3′-primary acyl chain) as substrate, only Kdo<sub>2</sub>-lipid IV<sub>A</sub>. This is in part due to the biochemical difficulty associated with generating the 3′-penta-acylated *in vitro* substrate. Without this result, it is difficult to definitively conclude that LpxL<sub>Hp</sub> participates in reversed acylation order during *in vivo* lipid A biosynthesis. Interestingly, the *C. jejuni* LpxJ protein can work in concert with *E. coli* LpxL to produce hexa-acylated lipid A, despite the fact that both enzymes prefer a tetra-acylated substrate (Fig. 2, lane 4). This result suggests a possible trade-off in substrate specificity *in vivo*, allowing both LpxJ<sub>Cj</sub> and LpxL to sometimes use a less preferred penta-acylated substrate in order to complete lipid A biosynthesis in the *in vivo E. coli* system. However, this described activity for LpxJ<sub>Cj</sub> would be best understood in the organism of origin, *C. jejuni*.

The evolutionary pressure selecting for various orders of secondary acylation is unclear, but may be determined
© 2013 John Wiley & Sons Ltd, Molecular Microbiology, 91, 887–899
by the specificity of lipid transport protein, MsbA. This flippase translocates lipid A across the inner membrane from the cytoplasmic to periplasmic lipid leaflet after completion of the Raetz pathway; translocation by MsbA is required for ultimate presentation of lipid A molecules on the bacterial surface (Raetz *et al.*, 2007). We hypothesize that the essential nature of LpxJ could be tied to MsbA, which might only recognize hexa-acylated lipid A or penta-acylated lipid A with a secondary acyl chain present at the 3′-primary acyl chain. If this were true, MsbA would not be able to flip the lipid A to the periplasmic leaflet in an LpxJ mutant, thus halting transport of the molecule to the bacterial cell surface leading to cell death. This theory is corroborated by the fact that mutants in LpxL<sub>Hp</sub> are viable (Stead *et al.*, 2008), suggesting that the precise location of the secondary acyl chain at the 3′- and not the 2′-primary chain is critical for recognition of the lipid substrate by MsbA in *H. pylori*. The reversal in late acylation order could be a response to this potential narrow substrate range of MsbA as an effort to ensure that the 3′-primary chain gets acylated first since survival depends on it. In *C. jejuni* and *W. succinogenes*, there may be stronger pressure to maintain the reversed late acylation order, as their respective LpxJ enzymes can act only before LpxL, while LpxJ activity in *H. pylori* can precede or follow LpxL. It is currently unclear why these organisms function only in the reverse order while the *H. pylori* enzyme is more flexible.

The three enzymes characterized in this article are all found in Epsilonproteobacterial species. However, the DUF374 protein class by which they are functionally classified represents an expansive array of other organisms including classes of bacteria such as Alpha-, Delta- and Zetaproteobacteria; and even other phyla such as Fusobacteria, Chlamydiae, Acidobacteria and Spirochaetes. The presence of DUF374 proteins in LpxM-deficient bacteria combined with the biochemical data presented here strongly suggest that LpxJ represents a hitherto uncharacterized group of lipid A secondary acyltransferases.

The three DUF374 enzymes studied in this work possess novel characteristics relative to *E. coli* LpxM in terms of substrate specificity and order of acylation, suggesting that differences at the primary sequence level can explain differences in function. Additional work is required to determine a molecular explanation for the divergence in biochemical activity based on differences in residues within the predicted acyltransfer active site and is ongoing in our laboratory. Study of these enzymes is particularly important because modification and ultimate surface presentation of *H. pylori* lipid A is dependent on formation of a Raetz pathway end-product that can be recognized by the MsbA transport machinery. The unusual order of *H. pylori* lipid A biosynthesis demonstrates that one enzyme and its biochemical characteristics can alter the

### Experimental procedures

#### Chemicals and other materials
[γ-<sup>32</sup>P]-ATP and <sup>32</sup>P<sub>i</sub> were obtained from Perkin Elmer. Precoated glass-backed thin-layer chromatography plates were purchased from EMD chemicals. Luria–Bertani (LB) agar and LB broth were from Fisher. Defibrinated sheep blood for blood agar plates was from Remel. Triton X-100 was from Pierce. All other chemicals were reagent-grade and purchased from Fisher.

#### Bacterial strains and growth conditions
The bacterial strains and plasmids used in this study are summarized in Table S3. Primary plate cultures of *H. pylori* were grown from glycerol stock on blood agar medium containing vancomycin (10 μg ml<sup>−1</sup>) at 37°C for 36 to 60 h in a microaerobic atmosphere (5% O<sub>2</sub>, 10% CO<sub>2</sub>, 85% N<sub>2</sub>). *E. coli* was grown at 37°C in LB broth. When required for selection of plasmids, cells were grown in the presence of ampicillin (100 μg ml<sup>−1</sup>).

#### Recombinant DNA techniques
Plasmids were isolated using a QIAprep Spin Miniprep Kit (Qiagen). Primers were obtained from Integrated DNA Technologies. PCR was performed using reagents from Agilent and PCR clean-up with the QIAquick PCR Purification Kit (Qiagen). DNA fragments were extracted from agarose gels using the QIAquick Gel Extraction Kit (Qiagen). Restriction endonucleases, T4 DNA ligase and Antarctic phosphatase were purchased from New England Biolabs. All enzymes were used according to the manufacturers’ instructions.

#### Construction of genetic mutant and chromosomal complementation of *H. pylori* J99 *jhp0255* (*lpxJ<sub>Hp</sub>*)
The *jhp0255* coding sequence, including 1369 bp upstream and 1000 bp downstream was amplified by PCR (primers 1 and 2, Table S4) from *H. pylori* J99 genomic DNA using Pfu Turbo (Stratagene) according to the manufacturer’s instructions. The resultant DNA fragment was digested with NheI and XhoI, gel purified, and cloned behind the T7lac promoter in vector pET21a. Next, a kanamycin resistance cassette was obtained by PCR (primers 5 and 6) from the *E. coli–H. pylori* shuttle vector, pHel3, and inserted into pET21a containing *jhp0255*. This was accomplished by encoding BamHI and NotI restriction sites on the pET21a:*jhp0255* vector via inverse PCR (primers 3 and 4), then digesting the kanamycin fragment and pET21a:*jhp0255* vector with BamHI and NotI. This allowed excision of the *jhp0255* coding sequence and subsequent replacement/ligation with the kanamycin cassette. The resulting plasmid (*jhp0255KO+/−*) containing an interrupted *jhp0255*-coding gene, was transformed into *H. pylori* J99 by natural transformation (Haas *et al.*, 1993).Due to the essential nature of the gene, knockout attempts were unsuccessful (see Results).

Chromosomal complementation was achieved by cloning *jhp0255* (*lpxJ*<sub>*H*<sub>*p*</sub></sub> in Fig. S1) into vector pET0634comp. This plasmid allows for insertion of a chosen coding sequence into the *rdxA* locus on the genome, inactivating the *rdxA* gene. RdxA is a nitroreductase that converts metronidazole from an inactive pro-drug to its active form, so inactivation of the gene renders the bacterium resistant to metronidazole (Smeets *et al.*, 2000; Stead *et al.*, 2008). The *jhp0255* coding sequence was amplified by PCR (primers 13 and 14) from *H. pylori* J99 genomic DNA using *Pfu* Turbo (Stratagene) according to the manufacturer’s instructions. The resultant DNA fragment was digested with BamHI and EcoRI, gel purified, and ligated to BamHI/EcoRI, alkaline phosphatase-treated pET0634comp vector. The completed plasmid (pET0634comp*jhp0255*) was transformed into wild-type *H. pylori* J99 by natural transformation and resistant colonies were selected on blood agar plates containing 10 μg ml<sup>−1</sup> metronidazole. Resistant colonies were restreaked on fresh metronidazole plates and successful insertion of the complementation cassette at the *rdxA* locus was confirmed by PCR of genomic DNA (primers 15 and 16). With a second copy of *jhp0255* now present at the *rdxA* locus, we proceeded to prove that we could then knock out *jhp0255* at its normal locus. To achieve this, we transformed plasmid *jhp0255KO+/* into the *rdxA::jhp0255* strain by natural transformation. Resistant colonies were obtained when bacteria were plated on blood agar containing 30 μg ml<sup>−1</sup> kanamycin. Colonies were restreaked on fresh kanamycin plates and successful insertion of the kanamycin cassette at the *jhp0255* locus was confirmed by PCR of genomic DNA (primers 17 and 18).

### Cloning of acyl transferases into low-copy plasmid pWSK29 with RBS

The acyltransferases of *H. pylori* J99 (*jhp0255*, primers 7 and 8), *C. jejuni* 81-176 (*cjj81176_0482*, primers 9 and 10) and *W. succinogenes* (*ws1775*, primers 11 and 12) were cloned into vector pET21a (Novagen) behind the T7/lac promoter. The PCR product from *jhp0255*, amplification was digested with Ndel and BamHI, and *cjj81176_0482* and *ws1775* were digested with Ndel and SalI. Digested PCR products were ligated individually overnight into pET21a at 16°C using T4 DNA ligase (New England BioLabs) to generate pET21a*jhp0255*, pET21a*cjj81176_0482*, pET21aws1775. Next, each plasmid was cut with Xbal and Xhol, excising the respective gene coding regions along with a ribosomal binding site. The resultant fragments were ligated individually into pWSK29 (Wang and Kushner, 1991) to give pWSK29*jhp0255*, pWSK29*cjj81176_0482* and pWSK29*ws1775* for expression of the acyltransferases in *E. coli*. The three vectors were transformed into XL-1 Blue (Stratagene) for propagation, then BN2 (Needham *et al.*, 2013) and MKV15b (Vorachek-Warren *et al.*, 2002) for radiolabelling and mass spectrometry experiments respectively.

### Preparation of cell-free extracts, double-spun cytosol and washed membranes

Two hundred millilitres of *E. coli* cultures were grown to an A<sub>600</sub> of ~1.0 at 37°C and harvested by centrifugation at 10 000 g for 10 min at 4°C. Cell-free extracts, membrane-free cytosol and washed membranes from all samples were prepared at 4°C as previously described and stored in aliquots at −20°C (Trent *et al.*, 2001). Protein concentration was determined by the bicinchoninic acid assay method using bovine serum albumin as the standard (Smith *et al.*, 1985).

#### Preparation of radiolabelled substrates

The substrate [4′-<sup>32</sup>P] lipid IV<sub>A</sub> was prepared from 125 μCi of [γ-<sup>32</sup>P]-ATP and the tetra-acyl-disaccharide 1-phosphate lipid acceptor, using the overexpressed 4′ kinase in the membranes of *E. coli* strain BLR(DE3)/pLysS/pJK2 as previously described (Trent *et al.*, 2001). Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid IV<sub>A</sub> was generated by adding purified *E. coli* Kdo transferase (KdtA) immediately following the 4′ kinase, as previously described (Belunis and Raetz, 1992). Following the Kdo transfer reaction, membranes from *E. coli* strain BLR(DE3)HtrB and *E. coli* strain BLR(DE3)/MsbB as well as C12:0-acyl carrier protein (ACP) were added to generate Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid A, as previously described (Trent *et al.*, 2001). Kdo<sub>2</sub>-lauroyl-[4′-<sup>32</sup>P] lipid IV<sub>A</sub> was generated as Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid A, with the omission of BLR(DE3)/MsbB membranes and addition of C12:0-ACP at a 1:1 molar ratio.

#### In vitro assay of *LpxJ*<sub>*H*<sub>*p*</sub></sub>, *LpxJ*<sub>*Cj*</sub> and *LpxJ*<sub>*Ws*</sub> activities

All enzymes were assayed under optimized conditions in a 10 μl reaction mixture containing 50 mM HEPES, pH 7.5, 0.2% Triton X-100, 2.5 μM lipid A substrate ([4′-<sup>32</sup>P] lipid IV<sub>A</sub>, Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid IV<sub>A</sub>, Kdo<sub>2</sub>-lauroyl-[4′-<sup>32</sup>P] lipid IV<sub>A</sub>, or Kdo<sub>2</sub>-[4′-<sup>32</sup>P] lipid A at ~5000 cpm nmol<sup>−1</sup>) with either 5 μM ACPSH, C12:0-ACP/CoA, C14:0-ACP/CoA, C16:0-ACP/CoA or C18:0-ACP/CoA. Washed BN2 membranes expressing each late acyltransferase were used in the following concentrations as enzyme source: LpxJ<sub>*H*<sub>*p*</sub></sub> 0.01 mg ml<sup>−1</sup>, LpxJ<sub>*Cj*</sub> 0.01 mg ml<sup>−1</sup>, LpxJ<sub>*Ws*</sub> 0.005 mg ml<sup>−1</sup>. Reactions were incubated at 30°C for 30 min, terminated by spotting 4.5 μl onto Silica Gel thin-layer chromatography (TLC) plates, and dried under cool air for 20 min. Reaction products were separated using a chloroform, pyridine, 88% formic acid, and water solvent system (30:70:16:10, vol/vol). TLC plates were exposed to a PhosphorImager screen and analysed using a Bio-Rad Molecular Imager in conjunction with Quantity One software.

#### Isolation and analysis of lipid A species from <sup>32</sup>P-labelled cells

BN2 cells containing plasmids pWSK29*jhp0255*, pWSK29*cjj81176_0482* and pWSK29*ws1775* were grown in LB medium at 37°C with 2.5 μCi ml<sup>−1</sup> <sup>32</sup>P. Bacteria were harvested at OD ~0.8 and washed with 5 ml phosphate-buffered saline. <sup>32</sup>P-labelled lipid A was isolated as described previously and spotted onto a Silica Gel TLC plate (~10 000 cpm per lane) (Tran *et al.*, 2004). Lipids were separated using a chloroform, pyridine, 88% formic acid, and water solvent system (50:50:16:5, vol/vol). TLC plates were exposed to a PhosphorImager screen and analysed using a Bio-Rad Molecular Imager in conjunction with Quantity One software.

© 2013 John Wiley & Sons Ltd, *Molecular Microbiology*, **91**, 887–899
898 E. J. Rubin et al.

### Large-scale isolation of lipid A for mass spectrometry analysis

Plasmids pWSK29jhp0255, pWSK29cjj81176_0482 and pWSK29ws1775 were transformed into tetra-acylated strain MKV15b. Two hundred and fifty millilitres of each resultant strain was then grown in LB at 37°C, harvested by centrifugation at 6000 g for 15 min, washed once with PBS, and final pellets were resuspended in 20 ml PBS. Lipid A was extracted and purified as previously described and stored at –20°C (Tran *et al.*, 2004).

### Mass spectrometry of lipid A species

Matrix assisted laser desorption-ionization time-of-flight (MALDI TOF) mass spectrometry was performed as previously described (Stead *et al.*, 2008). Ultraviolet photodissociation (UVPD) mass spectrometry was performed in the negative mode on a Thermo Scientific Orbitrap Elite mass spectrometer (Bremen, Germany) equipped with a 193 nm ArF excimer laser from Coherent (ExiStar XS, Santa Clara, CA) using a recently described set-up (Shaw *et al.*, 2013). Lipid A samples were prepared for UVPD-MS by diluting samples to approximately 1 μM with 50:50 methanol : chloroform (Sigma Aldrich, St. Louis, MO). Lipid A solutions were directly infused via electrospray ionization using a potential of 4 kV and a sheath gas flow rate of 10 arbitrary units. All UVPD mass spectra were acquired in the Orbitrap mass analyser using a resolution of 15 000 and a total of ten 6 mJ pulses per scan.

### Acknowledgements

This work was supported by the National Institutes of Health (NIH) (Grant GM103655 to J.S.B. and AI064184 to M.S.T.), the Army Research Office (Grant W911NF-12-1-0390 to M.S.T.) and the Welch Foundation (Grant F1155 to J.S.B.).

### References

Alm, R.A., Ling, L.S., Moir, D.T., King, B.L., Brown, E.D., Doig, P.C., *et al.* (1999) Genomic-sequence comparison of two unrelated isolates of the human gastric pathogen *Helicobacter pylori*. Nature **397**: 176–180.

Belunis, C.J., and Raetz, C.R. (1992) Biosynthesis of endotoxins. Purification and catalytic properties of 3-deoxy-D-manno-octulosonic acid transferase from *Escherichia coli*. J Biol Chem **267**: 9988–9997.

Belunis, C.J., Clementz, T., Carty, S.M., and Raetz, C.R. (1995) Inhibition of lipopolysaccharide biosynthesis and cell growth following inactivation of the *kdtA* gene in *Escherichia coli*. J Biol Chem **270**: 27646–27652.

Clementz, T., Bednarski, J.J., and Raetz, C.R. (1996) Function of the *htrB* high temperature requirement gene of *Escherichia coli* in the acylation of lipid A: HtrB catalyzed incorporation of laurate. J Biol Chem **271**: 12095–12102.

Clementz, T., Zhou, Z., and Raetz, C.R. (1997) Function of the *Escherichia coli msbB* gene, a multicopy suppressor of htrB knockouts, in the acylation of lipid A. Acylation by MsbB follows laurate incorporation by HtrB. J Biol Chem **272**: 10353–10360.

Cullen, T.W., Giles, D.K., Wolf, L.N., Ecobichon, C., Boneca, I.G., and Trent, M.S. (2011) *Helicobacter pylori* versus the host: remodeling of the bacterial outer membrane is required for survival in the gastric mucosa. PLoS Pathog **7**: e1002454.

Cullen, T.W., O’Brien, J.P., Hendrixson, D.R., Giles, D.K., Hobb, R.I., Thompson, S.A., *et al.* (2013) EptC of *Campylobacter jejuni* mediates phenotypes involved in host interactions and virulence. Infect Immun **81**: 430–440.

Funahara, Y., and Nikaido, H. (1980) Asymmetric localization of lipopolysaccharides on the outer membrane of *Salmonella typhimurium*. J Bacteriol **141**: 1463–1465.

Haas, R., Meyer, T.F., and van Putten, J.P. (1993) Aflagellated mutants of *Helicobacter pylori* generated by genetic transformation of naturally competent strains using transposon shuttle mutagenesis. Mol Microbiol **8**: 753–760.

Heath, R.J., and Rock, C.O. (1998) A conserved histidine is essential for glycerolipid acyltransferase catalysis. J Bacteriol **180**: 1425–1430.

Hoshino, K., Takeuchi, O., Kawai, T., Sanjo, H., Ogawa, T., Takeda, Y., *et al.* (1999) Cutting edge: toll-like receptor 4 (TLR4)-deficient mice are hyporesponsive to lipopolysaccharide: evidence for TLR4 as the Lps gene product. J Immunol **162**: 3749–3752.

Lepper, P.M., Triantafilou, M., Schumann, C., Schneider, E.M., and Triantafilou, K. (2005) Lipopolysaccharides from *Helicobacter pylori* can act as antagonists for Toll-like receptor 4. Cell Microbiol **7**: 519–528.

Moran, A.P. (2007) Lipopolysaccharide in bacterial chronic infection: insights from *Helicobacter pylori* lipopolysaccharide and lipid A. Int J Med Microbiol **297**: 307–319.

Moran, A.P., Lindner, B., and Walsh, E.J. (1997) Structural characterization of the lipid A component of *Helicobacter pylori* rough- and smooth-form lipopolysaccharides. J Bacteriol **179**: 6453–6463.

Muotiala, A., Helander, I.M., Pyhälä, L., Kosunen, T.U., and Moran, A.P. (1992) Low biological activity of *Helicobacter pylori* lipopolysaccharide. Infect Immun **60**: 1714–1716.

Needham, B.D., Carroll, S.M., Giles, D.K., Georgiou, G., Whiteley, M., and Trent, M.S. (2013) Modulating the innate immune response by combinatorial engineering of endotoxin. Proc Natl Acad Sci USA **110**: 1464–1469.

Ogawa, T., Suda, Y., Kashihara, W., Hayashi, T., Shimoyama, T., Kusumoto, S., and Tamura, T. (1997) Immunobiological activities of chemically defined lipid A from *Helicobacter pylori* LPS in comparison with *Porphyromonas gingivalis* lipid A and *Escherichia coli*-type synthetic lipid A (compound 506). Vaccine **15**: 1598–1605.

Ogawa, T., Asai, Y., Sakai, Y., Oikawa, M., Fukase, K., Suda, Y., *et al.* (2003) Endotoxic and immunobiological activities of a chemically synthesized lipid A of *Helicobacter pylori* strain 206-1. FEMS Immunol Med Microbiol **36**: 1–7.

Poltorak, A., He, X., Smirnova, I., Liu, M.Y., Van Huffel, C., Du, X., *et al.* (1998) Defective LPS signaling in C3H/HeJ and C57BL/10ScCr mice: mutations in Tlr4 gene. Science **282**: 2085–2088.

Punta, M., Coggill, P.C., Eberhardt, R.Y., Mistry, J., Tate, J., Boursnell, C., *et al.* (2011) The Pfam protein families database. Nucleic Acids Res **40**: D290–D301.

Raetz, C.R. (1990) Biochemistry of endotoxins. Annu Rev Biochem **59**: 129–170.

Raetz, C.R.H., and Whitfield, C. (2002) Lipopolysaccharide endotoxins. *Annu Rev Biochem* **71**: 635–700.

Raetz, C.R.H., Reynolds, C.M., Trent, M.S., and Bishop, R.E. (2007) Lipid A modification systems in gram-negative bacteria. *Annu Rev Biochem* **76**: 295–329.

Rubin, E.J., and Trent, M.S. (2013) Colonize, evade, flourish: how glyco-conjugates promote virulence of *Helicobacter pylori*. *Gut Microbes* **4**: 1–14.

Shaw, J.B., Li, W., Holden, D.D., Zhang, Y., Griep-Raming, J., Fellers, R.T., *et al.* (2013) Complete protein characterization using top-down mass spectrometry and ultraviolet photodissociation. *J Am Chem Soc* **135**: 12646–12651.

Six, D.A., Lambert, B., Raetz, C.R.H., and Doerrler, W.T. (2012) Density gradient enrichment of *Escherichia coli* lpxL mutants. *Biochim Biophys Acta* **1821**: 989–993.

Smeets, L.C., Bijlsma, J.J., Boomkens, S.Y., Vandenbroucke-Grauls, C.M., and Kusters, J.G. (2000) *comH*, a novel gene essential for natural transformation of *Helicobacter pylori*. *J Bacteriol* **182**: 3948–3954.

Smith, P.K., Krohn, R.I., Hermanson, G.T., Mallia, A.K., Gartner, F.H., Provenzano, M.D., *et al.* (1985) Measurement of protein using bicinchoninic acid. *Anal Biochem* **150**: 76–85.

Stead, C., Tran, A., Ferguson, D., McGrath, S., Cotter, R., and Trent, S. (2005) A novel 3-deoxy-D-manno-octulosonic acid (Kdo) hydrolase that removes the outer Kdo sugar of *Helicobacter pylori* lipopolysaccharide. *J Bacteriol* **187**: 3374–3383.

Stead, C.M., Beasley, A., Cotter, R.J., and Trent, M.S. (2008) Deciphering the unusual acylation pattern of *Helicobacter pylori* lipid A. *J Bacteriol* **190**: 7012–7021.

Stead, C.M., Zhao, J., Raetz, C.R.H., and Trent, M.S. (2010) Removal of the outer Kdo from *Helicobacter pylori* lipopolysaccharide and its impact on the bacterial surface. *Mol Microbiol* **78**: 837–852.

Tamada, T., Feese, M.D., Ferri, S.R., Kato, Y., Yajima, R., Toguri, T., and Kuroki, R. (2004) Substrate recognition and selectivity of plant glycerol-3-phosphate acyltransferases (GPATs) from *Cucurbita moschata* and *Spinacea oleracea*. *Acta Crystallogr D Biol Crystallogr* **60**: 13–21.

Tran, A.X., Karbarz, M.J., Wang, X., Raetz, C.R.H., McGrath, S.C., Cotter, R.J., and Trent, M.S. (2004) Periplasmic cleavage and modification of the 1-phosphate group of *Helicobacter pylori* lipid A. *J Biol Chem* **279**: 55780–55791.

Tran, A.X., Whittimore, J.D., Wyrick, P.B., McGrath, S.C., Cotter, R.J., and Trent, M.S. (2006) The lipid A 1-phosphatase of *Helicobacter pylori* is required for resistance to the antimicrobial peptide polymyxin. *J Bacteriol* **188**: 4531–4541.

Trent, M.S., Pabich, W., Raetz, C.R., and Miller, S.I. (2001) A PhoP/PhoQ-induced Lipase (PagL) that catalyzes 3-O-deacylation of lipid A precursors in membranes of *Salmonella typhimurium*. *J Biol Chem* **276**: 9083–9092.

Turnbull, A.P., Rafferty, J.B., Sedelnikova, S.E., Slabas, A.R., Schierer, T.P., Kroon, J.T., *et al.* (2001) Analysis of the structure, substrate specificity, and mechanism of squash glycerol-3-phosphate (1)-acyltransferase. *Struct* **9**: 347–353. Lond Engl 1993.

Vorachek-Warren, M.K., Ramirez, S., Cotter, R.J., and Raetz, C.R.H. (2002) A triple mutant of *Escherichia coli* lacking secondary acyl chains on lipid A. *J Biol Chem* **277**: 14194–14205.

Wang, R.F., and Kushner, S.R. (1991) Construction of versatile low-copy-number vectors for cloning, sequencing and gene expression in *Escherichia coli*. *Gene* **100**: 195–199.

**Supporting information**

Additional supporting information may be found in the online version of this article at the publisher’s web-site.
