
Journal of Bacteriology

Identification in Haloferax volcanii of Phosphomevalonate Decarboxylase and Isopentenyl Phosphate Kinase as Catalysts of the Terminal Enzyme Reactions in an Archaeal Alternate Mevalonate Pathway

John C. VanNice, D. Andrew Skaff, Andrew Keightley, James K. Addo, Gerald J. Wyckoff and Henry M. Miziorko

J. Bacteriol. 2014, 196(5):1055. DOI: 10.1128/JB.01230-13.

Published Ahead of Print 27 December 2013.

---

Updated information and services can be found at:

http://jb.asm.org/content/196/5/1055

These include:

This article cites 16 articles, 8 of which can be accessed free at:

http://jb.asm.org/content/196/5/1055#ref-list-1

**REFERENCES**

**CONTENT ALERTS**

Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more »](http://jb.asm.org/content/196/5/1055)

---

Information about commercial reprint orders: http://journals.asm.org/site/misc/reprints.xhtml

To subscribe to to another ASM Journal go to: http://journals.asm.org/site/subscriptions/

---

Journals.ASM.org

Identification in *Haloferax volcanii* of Phosphomevalonate Decarboxylase and Isopentenyl Phosphate Kinase as Catalysts of the Terminal Enzyme Reactions in an Archaeal Alternate Mevalonate Pathway

John C. VanNice, D. Andrew Skaff, Andrew Keightley, James K. Addo, Gerald J. Wyckoff, Henry M. Miziorko

Division of Molecular Biology and Biochemistry, School of Biological Sciences, University of Missouri—Kansas City, Kansas City, Missouri, USA

Mevalonate (MVA) metabolism provides the isoprenoids used in archaeal lipid biosynthesis. In synthesis of isopentenyl diphosphate, the classical MVA pathway involves decarboxylation of mevalonate diphosphate, while an alternate pathway has been proposed to involve decarboxylation of mevalonate monophosphate. To identify the enzymes responsible for metabolism of mevalonate 5-phosphate to isopentenyl diphosphate in *Haloferax volcanii*, two open reading frames (HVO_2762 and HVO_1412) were selected for expression and characterization. Characterization of these proteins indicated that one enzyme is an isopentenyl phosphate kinase that forms isopentenyl diphosphate (in a reaction analogous to that of *Methanococcus jannaschii* MJ0044). The second enzyme exhibits a decarboxylase activity that has never been directly attributed to this protein or any homologous protein. It catalyzes the synthesis of isopentenyl phosphate from mevalonate monophosphate, a reaction that has been proposed but never demonstrated by direct experimental proof, which is provided in this account. This enzyme, phosphomevalonate decarboxylase (PMD), exhibits strong inhibition by 6-fluoromevalonate monophosphate but negligible inhibition by 6-fluoromevalonate diphosphate (a potent inhibitor of the classical mevalonate pathway), reinforcing its selectivity for monophosphorylated ligands. Inhibition by the fluorinated analog also suggests that the PMD utilizes a reaction mechanism similar to that demonstrated for the classical MVA pathway decarboxylase. These observations represent the first experimental demonstration in *H. volcanii* of both the phosphomevalonate decarboxylase and isopentenyl phosphate kinase reactions that are required for an alternate mevalonate pathway in an archaeon. These results also represent, to our knowledge, the first identification and characterization of any phosphomevalonate decarboxylase.

A rchaea represent an evolutionarily distinct domain of organisms that includes many extremophiles (1). Among these are the halophiles, including *Haloferax volcanii*, which can function at high salt levels. Proteins from such halophiles are physiologically active at 2 to 3.5 M Cl⁻ concentrations and thus differ considerably from any homologs that exist and function at lower ionic strength in bacteria and animals. The basis for such differences invites detailed study as isolated forms of the halophile proteins become available. In particular, the early steps in biosynthesis of isoprenoids are relevant to the production of cell membranes of the halophiles, as well as other archaea. Their cell membrane lipids contain, as a major component, polyisoprenoids in ether linkages to glycerol (2). Radiolabeling experiments suggest that the mevalonate (MVA) pathway provides the metabolites used in biosynthesis of archaeal membrane lipid (3). While a functional demonstration of the last two reactions of the classical mevalonate pathway has been shown in *Sulfolobus solfataricus* (4), other investigations have yet to identify enzymes catalyzing such reactions in most other archaea. This has prompted hypotheses that archaeal isoprenoids are synthesized, not by the classical MVA pathway (4) (Fig. 1), but by an alternate series of enzymatic reactions (phosphorylation and decarboxylation) that also result in formation of the isopentenyl diphosphate.

Grochowski et al., working with *Methanocaldococcus jannaschii* (5), isolated an isopentenyl phosphate kinase (IPK). This prompted their hypothesis concerning the operation of an alternate pathway (Fig. 1) in which decarboxylation (of mevalonate 5-phosphate) preceded phosphorylation (of isopentenyl phosphate).

Subsequently, Chen and Poulter (6) described IPK enzymes from *Thermoplasma acidophilum* and *Methanothermobacter thermautotrophicus*, adding support for the alternate-pathway hypothesis. In *H. volcanii*, genome annotations are confusing in that they suggest that enzymes of both the classical MVA pathway (mevalonate diphosphate decarboxylase [MDD]) and an alternate MVA pathway (IPK) are encoded. This apparent contradiction drew our attention to the *Haloferax* pathway.

Using newly available tools (7) for expression of native proteins in these archaea, the open reading frame proposed to encode the MVA pathway enzyme HMG-coenzyme A (CoA) synthase was introduced into plasmids, and the protein was expressed in *Haloferax*. Recovery of active enzyme allowed confirmation of the proposed function (8) and suggested that this approach could be extended to investigate the functions of proteins that had ambiguous annotations. Recovery of active enzymes has now led to the identification of *H. volcanii* IPK (*HvIPK*) and the monophosphate-specific phosphomevalonate decarboxylase (PMD) and, thus, the first experimental demonstration of both of these remaining reactions of the alternate MVA pathway in an archaeon.

Received 16 October 2013 Accepted 18 December 2013  
Published ahead of print 27 December 2013  
Address correspondence to Henry M. Miziorko, miziorkoh@umkc.edu.  
Copyright © 2014, American Society for Microbiology. All Rights Reserved.  
doi:10.1128/JB.01230-13

VanNice et al.

![Diagram](attachment:diagram.png)

FIG 1 Classical and alternate mevalonate pathways for isopentenyl-diphosphate synthesis. In the alternate pathway, PMD corresponds to HVO_1412 and IPK corresponds to HVO_2762.

# MATERIALS AND METHODS

## Materials
Commercially available chemicals and biochemicals were purchased from Fisher or Sigma-Aldrich-Fluka. (R,S)-Mevalonate 5-phosphate was synthesized as described by Wang and Miziorko (9). (R,S)-Mevalonate diphosphate was synthesized as described by Reardon and Abeles (10). (R)-Mevalonate 5-P and (R)-fluoromevalonate monophosphate and diphosphate were synthesized enzymatically, as described by Voynova et al. (11). Isopentenyl phosphate was synthesized analogously to the method of Chen and Poulter (6). The purity of the compounds synthesized for this study was >85%. *H. volcanii* strain H1209 and plasmid pTA963 were generously provided by Thorsten Allers and Julie Maupin-Furlow.

## Identification and cloning of HvPMD and HvIPK
Protein sequences encoding bacterial and eukaryotic MDDs were obtained from NCBI and aligned against the *H. volcanii* genome using BLASTP. An open reading frame (HVO_1412) that successfully aligned with both was PCR amplified from the *H. volcanii* H1209 genomic DNA using *Pfu* DNA polymerase with the addition of 5′ EcoRI and 3′ BamHI restriction sites. An open reading frame annotated as an IPK (HVO_2762) was also PCR amplified from *H. volcanii* H1209 genomic DNA using *Pfu* DNA polymerase with the addition of 5′ EcoRI and 3′ BamHI restriction sites.

The PCR products and pTA963 were restriction digested using EcoRI and BamHI, ligated using T4 DNA ligase, transformed into *H. volcanii* H1209 cells, and stored in 20% glycerol at −80°C as previously described (8). The pTA963 vector contains a tryptophan-inducible promoter and an N-terminal His affinity tag directly upstream of the EcoRI and BamHI subcloning site. The frozen cells were then inoculated into 5 ml of Hv-YPC (18% salt water, yeast extract, peptone, Casamino Acids) (12), grown for 2 days at 45°C at 200 rpm, and then added to 50 ml of Hv-YPC and grown at 45°C at 200 rpm. For protein overexpression, 50 ml of cell culture was transferred to two 2.5-liter baffled flasks containing 1 liter of Hv-YPC medium and grown for 16 h at 42°C at 200 rpm in the presence of 1.3 mM tryptophan and then for 1 h with 3 mM tryptophan.

## Enzyme assay and methods for characterization of salt and pH dependence
Enzymatic activity in this study was monitored by measuring the appearance of ADP (as the disappearance of NADH) using a pyruvate kinase (PK)-lactate dehydrogenase (LDH)-coupled assay with a PerkinElmer λ35 spectrophotometer with a Peltier temperature controller or an Agilent Systems Cary 60 UV-visible light (Vis) spectrophotometer with a water bath for temperature control. The dependence of the salt concentration on *HvPMD* activity was assayed in 50 mM HEPES-KOH (pH 7.5), 10 mM MgCl₂, with 40 U of both pyruvate kinase and lactate dehydrogenase from rabbit muscle in increasing concentrations of KCl from 0 to 4 M in the presence of 2 mM (R,S)-mevalonate 5-P and 2 mM ATP at 30°C. A similar approach was used to evaluate the dependence of KCl on IPK activity. The dependence of pH on *HvPMD* enzyme activity was determined using the same assay by increasing the pH in 0.5-unit increments from 6.5 to 8.5.

### HvIPK isolation
For *H. volcanii* H1209 cells transformed with pTA963-*HvIPK*, cells were pelleted and then resuspended in 25 mM Tris-Cl (pH 8.0), 1 M KCl, 1 mM dithiothreitol (DTT), 10 mM imidazole. The cell suspension was homogenized using a microfluidizer (which affords a continuous high-shear method of cell disruption), and the lysate was centrifuged at 105,000 × g. *HvIPK* His-tagged protein was isolated from the supernatant by gravity Ni column chromatography. The loaded resin was washed with 25 mM Tris-Cl (pH 8.0), 1 M KCl, 1 mM DTT, 30 mM imidazole, and the target protein was eluted with the same lysis buffer containing 100 mM imidazole. *HvIPK* was then concentrated, and the buffer was exchanged for 20 mM Tris, pH 7.5, with 1 M KCl and 5 mM DTT using an Amicon concentrator with a 10-kDa cutoff. The protein can be stored without loss of activity at −80°C in this buffer at concentrations of at least 5 mg/ml. The yield was typically 2 to 3 mg of purified protein per liter of medium. Due to a low intrinsic $A_{280}$ value, IPK protein was estimated based on a Bio-Rad assay using bovine serum albumin as a standard.

### HvIPK kinetic characterization
The $V_{\text{max}}$ and $K_m$ values were determined using velocity measurements made at 340 nm with an Agilent Systems Cary 60 UV-Vis spectrophotometer. Data were obtained by incubation of 5 μg of enzyme in 0.1 M HEPES, pH 7.5, 4 M KCl, 10 mM MgCl₂, 0.5 mM DTT, 0.4 mM phosphoenolpyruvate (PEP), 20 U PK, 40 U LDH, 0.2 mM NADH. All reactions were initiated by addition of isopentenyl phosphate. For determination of the $K_m$ value for ATP, the concentration of ATP was varied between 0.05 and 1.0 mM.

concentration of isopentenyl phosphate was held constant at 0.5 mM, and the ATP concentration was varied from 0.04 to 2.5 mM. The \( K_m \) value for isopentenyl phosphate was determined by holding the ATP concentration constant at 2.5 mM, and the concentration of isopentenyl phosphate was varied from 8 to 500 μM. The data were analyzed using SigmaPlot 10 with the Enzyme Kinetics Module 1.3.

**HvPMD isolation.** For *H. volcanii* H1209 cells transformed with pTA963-HvPMD, cells were pelleted at 6,000 × g at 4°C, and the supernatant was removed and then resuspended in 100 mM Tris-Cl (pH 7.5), 2 M KCl, 20 mM imidazole, 1 mM DTT, 1 mM phenylmethylsulfonyl fluoride (PMSF). The cell suspension was then homogenized using a microfluidizer, and the cell lysate was centrifuged at 11,000 × g at 4°C for 30 min. HvPMD His-tagged protein was isolated from the cell lysate by gravity Ni column chromatography. After loading the supernatant, the column was washed with 40 mM imidazole in 100 mM Tris-Cl (pH 7.5), 2 M KCl, 1 mM DTT. Elution of the PMD enzyme was performed by a stepwise increase of imidazole in the buffer to 200 mM. The imidizole was removed by passing the eluate over a Sephadex G-25 column. Fractions exhibiting enzymatic activity were pooled and concentrated using a 20-ml, 10-kDa-cutoff Amicon concentrator at 5,000 × g for 15 min at 4°C. The yield was 2 mg of purified protein per liter of culture.

**HvPMD kinetic characterization.** The \( V_{\text{max}} \) and \( K_m \) for HvPMD were determined for (R,S)-mevalonate 5-P and (R)-mevalonate 5-P by incubating HvPMD in 50 mM HEPES-KOH (pH 7.5), 10 mM MgCl₂, 400 μM PEP, 200 μM NADH, 2 mM ATP, and 4 U of both pyruvate kinase and lactate dehydrogenase at 30°C. The reaction was started by the addition of increasing amounts of (R,S)-mevalonate 5-P or (R)-mevalonate 5-P from 12.5 to 1,600 μM; reaction progress was monitored at \( A_{340} \). The \( V_{\text{max}} \) and \( K_m \) for ATP were determined by incubating HvPMD in 50 mM HEPES-KOH (pH 7.5), 10 mM MgCl₂, 400 μM PEP, 200 μM NADH, and 4 U of both pyruvate kinase and lactate dehydrogenase with increasing amounts of ATP from 20 to 4,000 μM at 30°C. The reaction was started with the addition of 2 mM (R,S)-mevalonate 5-P. The data were analyzed using SigmaPlot 10 with the Enzyme Kinetics Module 1.3 or Graph Pad 5.0.

**Molecular mass estimation.** The molecular masses of HvIPK and HvPMD were determined by matrix-assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry (MS) using a Perseptive Biosystems-Voyager DE Pro operating in a positive, linear mode. HvPMD and HvIPK were desalted (C₁₈ Zip Tip; Millipore) and eluted onto the MALDI plate with 50% acetonitrile containing 10 mg/ml sinapinic acid matrix (3,5 dimethoxy-4-hydroxycinnaminic acid) for these analyses.

**Enzymatic and mass determination of the HvPMD reaction product.** Twenty micrograms of HvPMD was incubated in a 1-ml cuvette containing 50 mM HEPES-KOH (pH 7.5), 10 mM MgCl₂, 200 nmol of NADH, 400 nmol of PEP, 2 μmol of ATP, and 4 U of pyruvate kinase and lactate dehydrogenase at 30°C. The reaction was started with the addition of 50, 100, and 150 nmol of (R,S)-mevalonate 5-P, and the disappearance of NADH was monitored at \( A_{340} \). The amount of (R,S)-mevalonate 5-P converted to product was determined by the reaction endpoint. To test for the presence of mevalonate 5-diphosphate, after the initial reaction had gone to completion, MDD from *Staphylococcus epidermidis* (SeMDD) was added to the reaction mixture. The reaction was allowed to go to completion, and the amount of product formed was determined by the reaction endpoint. To test whether the reaction had instead produced isopentenyl-phosphate, HvIPK was added to the reaction mixture, and the disappearance of NADH was again monitored at \( A_{340} \). The reaction was allowed to go to completion, and the amount of product converted was determined by the endpoint. Reaction stoichiometries were calculated on the basis of the nanomoles of substrate converted to nanomoles of product based on these assay endpoints. To identify the product of HvPMD by mass spectrometry, 4 ml of 50 mM Tris-Cl (pH 7.5), 10 mM MgCl₂, 625 μM (R,S)-mevalonate 5-P, and 2 mM ATP were incubated at 40°C for 30 min in the presence or absence of 25 μg of HvPMD. Twelve milliliters of ice-cold ethanol was then added to precipitate nucleotides and protein. The samples were centrifuged at 16,000 × g for 15 min at 4°C. The supernatant was collected, and ethanol was removed by rotary evaporation. The samples were mixed with 20 mM triethylammonium (TEA)-bicarbonate, pH 8.5, buffer and passed over a Sephadex A-25 (2 ml resin in 5-mm diameter) column. The column was washed with 5 column volumes of 20 mM TEA-HCO₃ (pH 8.5), and compounds were eluted using a gradient of 100 to 500 mM TEA-HCO₃. The TEA-HCO₃ buffer was then removed by rotary evaporation. The reaction substrate and product were then detected and verified by electrospray ionization (ESI)-MS (nanospray) using a Thermo Finnigan LTQ linear ion trap MS system. Samples were diluted in 80% water-20% acetonitrile to approximately 100 pmol per μl for direct infusion at 1 to 3 μl per minute after equilibration by syringe pump. The mass analyzer was operated in negative ion mode, acquiring centered (centroid) or profile data. At least 20 s of scans (multiple spectra) were averaged to generate the data used in the figures. The monoisotopic masses of the negatively charged (-1) ions discussed are mevalonate 5-phosphate (M-H)⁻, 227.0320, and isopentenyl 5-phosphate (M-H)⁻, 165.0316, and the phosphate species are H₂PO₄⁻, 96.967, and PO₃⁻, 78.959. All masses were resolved within 0.2 Da of the monoisotopic mass (approximately 500 ppm) in the ion trap, averaging 227.2 (mevalonate 5-phosphate), 165.2 (isopentenyl 5-phosphate), 79.1 (PO₃⁻), and 97.1 (H₂PO₄⁻). However, these ions are assigned and discussed at unit resolution for simplicity in the figures and text. For tandem MS (MS-MS) acquisition (see Fig. 4E only), the isolation width was set at 2 (m/z), and the collision energy was set at 35.

**Inhibition of HvPMD by fluoromevalonate derivatives.** (R)-Fluoromevalonate 5-P or (R)-fluoromevalonate 5-PP was serially diluted 2-fold from 200 μM to 0.39 μM in 50 mM HEPES-KOH (pH 7.5), 10 mM MgCl₂ in a 96-well plate. Fifty millimolar HEPES-KOH (pH 7.5), 10 mM MgCl₂, 400 μM PEP, 200 μM NADH, 2 mM ATP, 10 μg/ml of HvPMD, and 4 U of both pyruvate kinase and lactate dehydrogenase were added as a final concentration to each well of the 96-well plate. The reaction was started with the addition of (R,S)-mevalonate monophosphate at a final concentration of 150 μM. Enzymatic activity was monitored at \( A_{340} \) for 5 min at 30°C in a Molecular Devices Spectramax 250 96-well plate reader. The data were analyzed with Graph Pad 5.0.

---

**RESULTS**

The prospects for success in any attempt to identify the enzymes that support the kinase and decarboxylase reactions of the isopentenyl diphosphate biosynthesis pathway in *Haloferax* seemed to be uncertain (13). Previous work on the HMG-CoA synthase provided no indication of adjacent genes that might be involved in other enzymes of the pathway. Also, numerous metabolite kinases are expected in *Haloferax*, and discriminating between their putative functions would require an unacceptably large investment of time and effort. Without prior knowledge of the details of how the putative decarboxylase functions, attempts to focus on coding sequences that suggest typical decarboxylase features, e.g., cofactor binding sites or similar clues implying a function, did not seem attractive. Previous articles published on the alternate pathway's IPK (5, 6) facilitated identification of a *Haloferax* coding sequence for a low-homology (~30% identity at the protein level) 248-residue kinase (HVO_2762). An apparent contradiction developed when a BLAST search against S. *epidermidis* MDD identified a low-homology (~30% identity at the protein level) *Haloferax* 324-amino-acid protein as the classical pathway’s MDD (HVO_1412). This protein contains several residues that are critical to the decarboxylation reaction catalyzed by MDD. In an attempt to resolve the apparent contradiction of protein components of both pathways in a single organism, work on expression of these putative enzymes was undertaken.

Expression in *H. volcanii* of an N-terminally His-tagged form of the putative IPK facilitated the immobilized metal affinity chromatography.

VanNice et al.

| MW | HvPMD | HvIPK |
| --- | --- | --- |

FIG 2 SDS-PAGE of purified *H. volcanii* isopentenyl-phosphate kinase and phosphomevalonate decarboxylase. Left lane, molecular weight markers; center lane, phosphomevalonate decarboxylase (4 μg); right lane, isopentenyl-phosphate kinase (4 μg).

matography (IMAC) isolation of a high-purity preparation of the protein (Fig. 2). The MALDI estimate of the subunit mass (26,417 ± 3 Da) is in reasonable agreement with the mass calculated for the His-tagged protein (26,402 Da). The SDS-PAGE results showed a position for the IPK band between the carbonic anhydrase (30-kDa) and ovalbumin (45-kDa) markers. The anomalous position of IPK is not uncommon for low-pI halophilic enzymes (*Haloferax* IPK calculated pI = 5.06). Chemical synthesis of isopentenyl phosphate (6) allowed tests of IPK activity and more detailed kinetic characterization. As indicated in Table 1, under our assay conditions, the enzyme exhibited an IPK-specific activity (*V*~m~) of 37 ± 2 U/mg, corresponding to a *k*~cat~ of 16.3 ± 0.9/s. This specific activity represents an ~3.5-fold increase in the rate at 4 M KCl versus no added KCl. For isopentenyl phosphate, the *K*~m~ was 77 ± 9 μM, an estimate that falls within the wide range of values (Table 1) reported for IPK from *M. jannaschii* (5), *M. thermautotrophicus*, and *T. acidophilum* (6). The *K*~m~ for ATP was 290 μM, a value >10-fold larger than that reported for other IPK enzymes from *M. thermautotrophicus* or *T. acidophilum* but similar to the report for *M. jannaschii*. The catalytic rate constant (*k*~cat~) of 16.3/s was comparable to several other values reported for these enzymes. There is considerable discrepancy between the kinetic constants reported for *M. jannaschii* IPK by two different laboratories (5, 14). Differences in the assay buffer concentration, pH, and temperature (or other unspecified assay methodology) may partially account for these discrepancies. The kinetic characterization results presented here for *H. volcanii* IPK place it firmly into the alternate mevalonate pathway. Moreover, as indicated below, this enzyme has proven useful in developing an assay for the decarboxylase.

The His-tagged form of the predicted decarboxylase (HVO_1412) was expressed and purified by the strategy already described for IPK. The supernatant of the *Haloferax* lysate was prepared, and the target protein was IMAC purified. The MALDI estimate of its subunit mass (37,328 ± 21 Da) was in accord with the mass calculated for the His-tagged protein (37,304 Da).

In an attempt to determine whether this decarboxylase catalyzed the enzymatic decarboxylation reaction expected for MDD versus PMD, an ATP-coupled decarboxylation assay was attempted. This assay choice was based on the need, for any decarboxylation of mevalonate C-1, to activate the C-3 hydroxyl to an improved leaving group (by ATP-dependent phosphorylation) that would transiently produce the C-3 carbocation that drives cleavage between C-1 and C-2 to form isopentenyl phosphate and CO₂. In an initial evaluation of substrate utilization, at 1 mM concentrations of mevalonate, mevalonate 5-phosphate, and mevalonate 5-diphosphate in the assay, a robust rate of ADP production in the presence of mevalonate 5-phosphate but little (~2%) with mevalonate and even less with mevalonate 5-diphosphate was observed. Upon testing the reaction product as a substrate for MDD or IPK (Table 2), the reaction product was completely metabolized by IPK, in contrast to MDD, which showed only minimal activity with the PMD reaction product.

These observations suggested the identity of the PMD reaction product was isopentenyl phosphate. The stoichiometry of the PMD reaction product converted to isopentenyl phosphate by IPK (Table 2), as determined by the coupled assay, approached unity, providing additional support for assignment of the decarboxylase as PMD.

The enzyme’s preference for a monophosphorylated substrate (as indicated above) prompted an investigation of inhibitor preferences. Enzyme inhibition experiments were performed to compare the inhibitory efficacies of fluoromevalonate 5-phosphate and fluoromevalonate 5-diphosphate, a compound that inhibits bacterial and animal MDD at submicromolar concentrations. In experiments with the *Haloferax* decarboxylase, 6-fluoromevalonate 5-phosphate inhibited with a 50% inhibitory concentration (*IC*~50~) of 16 μM (Fig. 3). In contrast, no inhibition of the enzyme by 6-fluoromevalonate 5-diphosphate was observed even at millimolar levels of the compound, further indicating enzyme selectivity for monophosphorylated compounds.

The identity of the product of the decarboxylase reaction became evident upon comparison of ESI-MS analyses of chemically synthesized phosphomevalonate [(M-H)^−^, *m/z* = 227] (Fig. 4A) and isopentenyl phosphate [(M-H)^−^, *m/z* = 165] (Fig. 4B) with the reaction product, isopentenyl phosphate (*m/z* = 165), predicted to be formed in the enzymatic reaction catalyzed by PMD.

TABLE 1 Kinetic constants for isopentenyl phosphate kinases

| Source | *k*~cat~ (s^−1^) | *K*~m~ (μ M) |
| --- | --- | --- |
|  |  | IP | ATP |
| *H. volcanii* | 16.3 ± 0.9^d^ | 77 ± 9 | 290 ± 30 |
| *M. jannaschii*^a^ | 330 | 256 | NA^e^ |
| *M. jannaschii*^b^ | 1.46 ± 0.03 | 4.3 ± 0.6 | 200 ± 30 |
| *M. thermautotrophicus*^c^ | 27.5 ± 0.3 | 12.7 ± 0.6 | 13.4 ± 0.8 |
| *T. acidophilum*^c^ | 8.0 ± 0.2 | 4.4 ± 0.5 | 6.0 ± 0.5 |

^a^ Data for *M. jannaschii* previously reported by Grochowski et al. (5).

^b^ Data for *M. jannaschii* previously reported by Dellas and Noel (14).

^c^ Data for *M. thermautotrophicus* and *T. acidophilum* previously reported by Chen and Poulter (6).

^d^ *V*~max~ = 37 ± 2 U/mg.

^e^ NA, not applicable.

Phosphomevalonate Decarboxylase and Isopentenyl-P Kinase

TABLE 2 Enzymatic Identification of the PMD reaction product$^a$

| Parameter                                                                 | Value for sample: |
|---------------------------------------------------------------------------|-------------------|
| Selectivity of PMD product metabolism by IPK or MDD                       |                   |
| $(R)$-MVA 5-P present in reaction mixture (nmol)                          | A                 | B                 | C                 | D                 | E                 |
| Product formation:                                                        |                   |                   |                   |                   |                   |
| PMD + IPK$^b$ added to A (nmol)                                           | $51.1 \pm 2.5$    | $55.2 \pm 0.1$    |                   |                   |                   |
| Product formation:                                                        |                   |                   |                   |                   |                   |
| PMD + MDD$^c$ added to B (nmol)                                           |                   |                   | $52.2 \pm 1.8$    |                   |                   |
| Reaction stoichiometry                                                    |                   |                   |                   |                   |                   |
| $(R)$-MVA 5-P added (nmol)                                                |                   |                   | $25$              | $50$              | $75$              |
| $Hv$PMD reaction product formed (nmol)                                    |                   |                   | $24.6 \pm 2.5$    | $51.1 \pm 3.6$    | $74.9 \pm 0.3$    |
| PMD reaction product utilized by $Hv$IPK (nmol)                           |                   |                   | $24.7 \pm 1.1$    | $52.2 \pm 1.8$    | $76.5 \pm 2.0$    |
| Reaction stoichiometry                                                    |                   |                   | $1.01 \pm 0.04$   | $1.02 \pm 0.01$   | $1.02 \pm 0.03$   |

$a$ Conversion of mevalonate 5-phosphate by PMD and subsequent ATP-dependent conversion of the PMD reaction product (or any unreacted mevalonate 5-phosphate) by secondary enzymes (IPK or MDD) was measured using the pyruvate kinase/lactate dehydrogenase-coupled assay described in Materials and Methods.

$b$ $H$. volcanii IPK (5 $\mu$g) was incubated with the PMD reaction product.

$c$ $S$. epidermidis MDD (20 $\mu$g) was incubated with the PMD reaction product.

In Fig. 4C, substrate $(R,S)$-mevalonate 5-phosphate ($m/z = 227$) was reacted with PMD and excess ATP to form product ($m/z = 165$). In Fig. 4D (negative control), no PMD was included in the mixture, so there was no reaction of substrate ($m/z = 227$) to form product ($m/z = 165$). We observed that there were phosphate ions ($PO_3^{-}$, $m/z = 79$, and $H_2PO_4^{-}$, $m/z = 97$) present in the MS spectra for the assays, as well as in the chemically synthesized compounds. However, isolation of the product ($m/z$ 165, with an isolation width of $2\ m/z$) for MS-MS generated phosphate ions (Fig. 4E), clearly demonstrating that it is a phosphorylated molecule. The phosphate ions were also observed in MS-MS spectra of the chemically synthesized phosphomevalonate and isopentenyl phosphate (data not shown), which provided standard spectra for comparison of the MS-MS data from the reaction product.

Thus, both the reaction stoichiometry based on the ATP-coupled assay and the ESI-MS analyses argued for the identity of the protein as the PMD enzyme proposed to function in the alternate archaeal mevalonate pathway. Taken together, the results indicate that both steps in the alternate pathway function in $H$. volcanii.

Enzyme characterization studies indicate that Haloferax PMD exhibits optimal catalytic activity at a pH of $\sim$7.5 but, surprisingly, exhibits activity that is not strongly dependent on KCl over a 0 to 4 M range. Kinetic characterization (Table 3) indicated a $V_{\text{max}}$ of 5.6 U/mg (where 1 U = 1 $\mu$mol/min conversion of substrate to product) and a $k_{\text{cat}}$ of 3.5 s$^{-1}$. These values are comparable to those of bacterial and human MDD enzymes. The $K_m$ determinations for $(R,S)$-mevalonate 5-phosphate, $(R)$-mevalonate 5-phosphate, and ATP were 159 $\mu$M, 75 $\mu$M, and 289 $\mu$M, respectively.

DISCUSSION

There are several available rationales for how and why the alternative MVA pathway has developed in some archaea. Among these is the possibility of the existence of both classical and alternate MVA pathways in organisms ancestral to archaea (as well as bacteria and eukaryotes). Differential specialization of pathways in various groups of these organisms may have led to the specific pathway that now exists in each group. An alternative hypothesis is that the different MVA pathways have long existed in different groups of organisms and that the current functional pathway in archaea may represent a horizontal or lateral transfer between bacteria and archaea of the genes encoding the functional MVA pathway enzyme(s). While future evolutionary biology work will be required to test and refine these hypotheses, the recent demonstration (4) of the classical pathway in $S$. solfataricus provides an example of the second possibility. At present, the advantages to archaea between the function of classical and alternate MVA pathways remain unclear.

Nishimura and colleagues (4) have recently provided evidence that the classical mevalonate pathway is operative in $S$. solfataricus. The report by Grochowski et al. (5) that the $M$. jannaschii genome

FIG 3 IC$_{50}$ determination for inhibition of $Hv$PMD by 6-fluoromevalonate 5-phosphate (A) and 6-fluoromevalonate 5-diphosphate (B). Differential sensitivity of phosphomevalonate decarboxylase to inhibition by 6-fluoromevalonate 5-phosphate and 6-fluoromevalonate 5-diphosphate is apparent. The curves indicate an IC$_{50}$ of 16 $\mu$M for the monophosphate-containing inhibitor, while the diphosphate-containing compound exhibits no inhibition.

VanNice et al.

![Image Description](#)

FIG 4 ESI-MS analysis of metabolites of the phosphomevalonate decarboxylase reaction. All spectra were collected in the negative ion mode. (A) MS spectrum of chemically synthesized (R,S)-mevalonate 5-phosphate (\( m/z = 227 \)). (B) MS spectrum of chemically synthesized isopentenyl 5-phosphate (\( m/z = 165 \)). (C) MS spectrum of a PMD reaction mixture indicating residual unreacted (S)-mevalonate 5-phosphate (\( m/z = 227 \)) and showing product isopentenyl 5-phosphate (\( m/z = 165 \)). (D) MS spectrum of negative control in which no enzyme was included in the reaction mixture. Unreacted (R,S)-mevalonate 5-phosphate substrate is apparent (\( m/z = 227 \)), but no formation of a product (\( m/z = 165 \)) peak is observed. (E) MS-MS spectrum of an \( m/z \) of 165, yielding phosphate ions (\( m/z = 79 \) and \( m/z = 97 \)).

encoded an IPK was confirmed by the work of Chen and Poulter (6) on recombinant IPK enzymes encoded in thermophilic archaea and expressed in *Escherichia coli*. Halophilic enzymes are often difficult to express in *E. coli*, so the tools introduced by Allers and colleagues (12) for expression of those enzymes in *Haloferax* have enabled a broader investigation of the steps of isoprenoid biosynthesis in halophiles. Thus, it has become increasingly interesting to determine whether any of the archaea in which IPK is

Journal of Bacteriology

Phosphomevalonate Decarboxylase and Isopentenyl-P Kinase

TABLE 3 Kinetic constants for *H. volcanii* PMD, *S. epidermidis* MDD, and *Homo sapiens* MDD$^a$

| Enzyme                | $K_m (\mu M)$ | $(R,S)$-Mevalonate 5-P | $(R,S)$-Mevalonate 5-PP | ATP     | $V_{max} (\mu mol/min/mg)$ | $k_{cat} (s^{-1})$ | $k_{cat}/K_m (s^{-1}M^{-1})$ |
|-----------------------|----------------|------------------------|-------------------------|---------|-------------------------------|---------------------|--------------------------------|
| *H. volcanii* PMD     |                | $159 \pm 15^b$        | NA                      | $289 \pm 15$ | $5.6 \pm 0.1$             | $3.5 \pm 0.1$      | $(2.2 \pm 0.2) \times 10^4$   |
| *S. epidermidis* MDD$^c$ |                | NA                     | $9.1 \pm 0.9$           | $27 \pm 3$ | $9.8 \pm 0.3$             | $5.9 \pm 0.2$      | $(6.5 \pm 0.7) \times 10^5$   |
| *H. sapiens* MDD$^d$ |                | NA                     | $28.9 \pm 3.3$          | $690 \pm 70$ | $6.1 \pm 0.5$             | $4.4 \pm 0.4$      | $(1.6 \pm 0.2) \times 10^5$   |

$a$ Values are means and standard errors.

$b$ The $K_m$ for $(R)$-mevalonate 5-phosphate is $75 \pm 8 \, \mu M$.

$c$ Data for *S. epidermidis* MDD previously reported by Barta et al. (16).

$d$ Data for *H. sapiens* MDD previously reported by Voynova et al. (11).

$e$ NA, not applicable.

VanNice et al.

HvPMD 1 MKA--T--------AKAHPICGLVKYHGMRDTERRMFYHDSISVCT--APSHQT TVEFRPD
EfMDD 1 MLS-GK--------ARAHTNIALIKYWGKANEYILPMNSSLSITL--DAFYTETTVIFDAH
SeMDD 1 MVKSGK--------ARAHTNIALIKYWGKADETYITPMNNSLSVTL--DRFYTETKVTFDPD
HsMDD 1 MAS-EKPLAAVTCTAPVNIAVIKYWGKRDEELVLPLINSSLSVTLHQDQLKTTTTAWISKD

HvPMD 51 ADEDVYVIGGEVEGRGAERIQA VVDRVR ELAGFDH-----------------RVRL ESENSF
EfMDD 52 YSEDVFIILDGILQNEKQTKKVKEFLNLVRQQADC TW-----------------FAKVESQN EV
SeMDD 53 FTEDCLILNGNEVNAKEEKIQNYMNIVRDLAGNRL-----------------HARIESENYV
HsMDD 60 FTEDRIWLNGREEDV-GQPRLQAC LREIRCLARKRRNSRDGDPLPSSLSCKVHVASVNNF

HvPMD 97 FS NIGFGSSASGEAAAAMALAEEAADDMTRPEVSTIARRGSASAARAVTGAE SHLYSGMN
EfMDD 98 PTAAGLASSASGLAALAGACNV ALGLNLSAKDLSRLARRGSGSACRSIFGGFAQWNKGHS
SeMDD 99 PTAAGLASSASAYAALAAAACNE ALSLN LSDTDLSRLARRGSGSASRSIFGGFAEWEKGHD
HsMDD 119 PTAAGLASSAAGYACLAYTLARVYGV---ESDLSEVARRGSGSACRSLYGGEVEWMGEQ

HvPMD 157 --DTDCR SERIET-DLEDDL RIVAAHVPA YKET-------EQAHAEAADS HMFAQARM AH-IHA
EfMDD 158 --DETSFAENIPANNWENEIAMLFILINDGEKDVS SRDGMKRTVETSSFYQGWLDN-VEK
SeMDD 159 --DLTSYA HGIN SNGWEKDLSMIFVVINNQS KKVSSRS GMSLTRDTSRFYQYWLDH-VDE
HsMDD 176 ADGKDSIARQVAPESHWP ELRVLILV VS AEKKLTGSTVGMRA SVETS PLLRFR AESVVP A

HvPMD 209 QIDDMR DALYDGDF DAAFEILA EHDSISLAATMTGPAGMVYWQPTIAVFNAVRKL RN-E
EfMDD 215 DISQVHEAIKT KDFPRLGEII EANGLRMHGTTLGAVPPFTYWSE GSLQAMALVRQARA--
SeMDD 216 DLNEAKEAVKNQDFQRLGEVIEANGLRMHATNLGAQPPFTYLVQESYDMA TVEQCRK--
HsMDD 236 RNAEMARC IEREDFBSF AQTMKDSNQFHATCLDTFPBISYLNA ISWRIHHLVHRFNHH

HvPMD 268 EDVPVYFSTD TGASVYINTTEEHVBRVEEAVADCG-----------------
EfMDD 273 KGIPCYFTMDAGPNVKVLVEK KNLEALKTFISEHFS-----------------
SeMDD 274 NLPCYFTMDAGPNVKVLVEK KNKQAVMEQFLKVFD-----------------
HsMDD 296 GDTKVA YTFDAGPNAVIFTLDDTVAEFVAAWHGFPPGSNGDTFLKGLQVRPAPLSAELQ

HvPMD 303 ----------VETD-VWGVGGPA-EVLDES----------E----ALF
EfMDD 309 ----------KEQL-VP AFA GPGIELFET----------KG---MDK
SeMDD 310 ----------ESKI IASDISSGVEI------------------TK
HsMDD 356 AALAMEPTPGGVKYI-IVTQVGBGPQILDDPCA HLLGPDGLPKPAA

FIG 6 Multiple-sequence alignment of phosphomevalonate decarboxylase and mevalonate diphosphate decarboxylases. The sequence of phosphomevalonate decarboxylase from *H. volcanii* (HvPMD) was aligned with mevalonate diphosphate decarboxylases from *Enterococcus faecalis* (EfMDD), *S. epidermidis* (SeMDD), and *Homo sapiens* (HsMDD). Conserved residues are shaded in black, and similar residues are shaded in gray.

ACKNOWLEDGMENTS

Chris Willig (UMKC) investigated and documented the requirements for effective use of animal pyruvate kinase/lactate dehydrogenase-coupling enzymes in the high-salt assays of *Haloferax* enzymes. Rana Zalmai (UMKC) participated in producing the plasmid for expression of HVO_2762 in *Haloferax*, as well as in the initial demonstration that a soluble, active target protein could be produced.

REFERENCES

1. Woese CR, Kandler O, Wheelis ML. 1990. Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya. Proc. Natl. Acad. Sci. U. S. A. 87:4576–4579. [http://dx.doi.org/10.1073/pnas.87.12.4576](http://dx.doi.org/10.1073/pnas.87.12.4576)
2. Koga Y, Mori H. 2005. Recent advances in structural research on ether lipids from archaea including comparative and physiological aspects. Biochem. Biotechnol. Biochem. 69:2019–2034. [http://dx.doi.org/10.1271/bbb.69.2019](http://dx.doi.org/10.1271/bbb.69.2019)
3. De Rosa M, Gambacorta A, Gliozzi A. 1986. Structure, biosynthesis, and physicochemical properties of archaebacterial lipids. Microbiol. Rev. 50: 70–80.
4. Nishimura H, Azami Y, Miyagawa M, Hashimoto C, Yoshimura T, Hemmi H. 2013. Biochemical evidence supporting the presence of the classical mevalonate pathway in the thermoacidophilic archaeon *Sulfolobus solfataricus*. J. Biochem. 153:415–420. [http://dx.doi.org/10.1093/jb/mvt006](http://dx.doi.org/10.1093/jb/mvt006)
5. Grochowski LL, Xu H, White RH. 2006. Methanocaldococcus jannaschii uses a modified mevalonate pathway for biosynthesis of isopentenyl

diphosphate. J. Bacteriol. 188:3192–3198. [http://dx.doi.org/10.1128/JB.188.9.3192-3198.2006](http://dx.doi.org/10.1128/JB.188.9.3192-3198.2006)

6. Chen M, Poulter CD. 2010. Characterization of thermophilic archaeal isopentenyl phosphate kinases. Biochemistry 49:207–217. [http://dx.doi.org/10.1021/bi9017957](http://dx.doi.org/10.1021/bi9017957)

7. Allers T. 2010. Overexpression and purification of halophilic proteins in *Haloferax volcanii*. Bioeng. Bugs 1:288–290. [http://dx.doi.org/10.4161/bbug.1.4.11794](http://dx.doi.org/10.4161/bbug.1.4.11794)

8. VanNice JC, Skaff DA, Wyckoff GJ, Miziorko HM. 2013. Expression in *Haloferax volcanii* of 3-hydroxy-3-methylglutaryl coenzyme A synthase facilitates isolation and characterization of the active form of a key enzyme required for polyisoprenoid cell membrane biosynthesis in halophilic archaea. J. Bacteriol. 195:3854–3862. [http://dx.doi.org/10.1128/JB.00485-13](http://dx.doi.org/10.1128/JB.00485-13)

9. Wang CZ, Miziorko HM. 2003. Methodology for synthesis and isolation of 5-phosphomevalonic acid. Anal. Biochem. 321:272–275. [http://dx.doi.org/10.1016/S0003-2697(03)00435-4](http://dx.doi.org/10.1016/S0003-2697(03)00435-4)

10. Reardon JE, Abeles RH. 1987. Inhibition of cholesterol biosynthesis by fluorinated mevalonate analogues. Biochemistry 26:4717–4722. [http://dx.doi.org/10.1021/bi00389a018](http://dx.doi.org/10.1021/bi00389a018)

11. Voynova NE, Fu Z, Battalie KP, Herdendorf TJ, Kim JJ, Miziorko HM. 2008. Human mevalonate diphosphate decarboxylase: characterization, investigation of the mevalonate diphosphate binding site, and crystal structure. Arch. Biochem. Biophys. 480:58–67. [http://dx.doi.org/10.1016/j.abb.2008.08.024](http://dx.doi.org/10.1016/j.abb.2008.08.024)

12. Allers T, Barak S, Liddell S, Wardell K, Mevarech M. 2010. Improved strains and plasmid vectors for conditional overexpression of His-tagged proteins in *Haloferax volcanii*. Appl. Environ. Microbiol. 76:1759–1769. [http://dx.doi.org/10.1128/AEM.02670-09](http://dx.doi.org/10.1128/AEM.02670-09)

13. Smit A, Mushegian A. 2000. Biosynthesis of isoprenoids via mevalonate in Archaea: the lost pathway. Genome Res. 10:1468–1484. [http://dx.doi.org/10.1101/gr.145600](http://dx.doi.org/10.1101/gr.145600).

14. Dellas N, Noel JP. 2010. Mutation of archaeal isopentenyl phosphate kinase highlights mechanism and guides phosphorylation of additional isoprenoid monophosphates. ACS Chem. Biol. 5:589–601. [http://dx.doi.org/10.1021/cb1000313](http://dx.doi.org/10.1021/cb1000313).

15. Matsumi R, Atomi H, Driessen AJ, van der Oost J. 2011. Isoprenoid

Phosphomevalonate Decarboxylase and Isopentenyl-P Kinase

biosynthesis in Archaea: biochemical and evolutionary implications. Res. Microbiol. 162:39–52. [http://dx.doi.org/10.1016/j.resmic.2010.10.003](http://dx.doi.org/10.1016/j.resmic.2010.10.003).

16. Barta ML, Skaff DA, McWhorter WJ, Herdendorf TJ, Miziorko HM, Geisbrecht BV. 2011. Crystal structures of Staphylococcus epidermidis mevalonate diphosphate decarboxylase bound to inhibitory analogs reveal new insight into substrate binding and catalysis. J. Biol. Chem. 286: 23900–23910. [http://dx.doi.org/10.1074/jbc.M111.242016](http://dx.doi.org/10.1074/jbc.M111.242016).
