
Cellular Microbiology (2014)  doi:10.1111/cmi.12284

# Co-ordination between BrlA regulation and secretion of the oxidoreductase FmqD directs selective accumulation of fumiquinazoline C to conidial tissues in *Aspergillus fumigatus*

**Fang Yun Lim**,¹ **Brian Ames**,² **Christopher T. Walsh**² and **Nancy P. Keller**¹*

¹ Department of Medical Microbiology and Immunology, University of Wisconsin-Madison, Madison, WI, USA.  
² Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA, USA.

## Summary

Aerial spores, crucial for propagation and dispersal of the Kingdom Fungi, are commonly the initial inoculum of pathogenic fungi. Natural products (secondary metabolites) have been correlated with fungal spore development and enhanced virulence in the human pathogen *Aspergillus fumigatus* but mechanisms for metabolite deposition in the spore are unknown. Metabolomic profiling of *A. fumigatus* deletion mutants of fumiquinazoline (Fq) cluster genes reveal that the first two products of the Fq cluster, FqF and FqA, are produced to comparable levels in all fungal tissues but the final enzymatically derived product, FqC, predominantly accumulates in the fungal spore. Loss of the sporulation-specific transcription factor, BrlA, yields a strain unable to produce FqA or FqC. Fluorescence microscopy showed FmqD, the oxidoreductase required to generate FqC, was secreted via the Golgi apparatus to the cell wall in an actin-dependent manner. In contrast, all other members of the Fq pathway including the putative transporter, FmqE – which had no effect on Fq biosynthesis – were internal to the hyphae. The co-ordination of BrlA-mediated tissue specificity with FmqD secretion to the cell wall presents a previously undescribed mechanism to direct localization of specific secondary metabolites to spores of the differentiating fungus.

---

## Introduction

Fungi produce a wide range of bioactive natural products whose assembly typically requires multiple biosynthetic steps. With a few exceptions, localization of these steps in the fungal thallus – whether tissue specificity or cellular localization – is largely unknown. A few studies have shown that certain secondary metabolites localize to specific tissues, such as endocrocin and fumigaclavines to asexual spores (conidia) of the human opportunistic pathogen *Aspergillus fumigatus* (Coyle *et al.*, 2007; Berthier *et al.*, 2013). The mechanism(s) resulting in placement of metabolites into spores is not known although the study on fumigaclavine synthesis suggests a need for BrlA, the *Aspergillus* transcription factor required for asexual sporulation (Coyle *et al.*, 2007). Existing studies addressing subcellular localization of individual steps have implicated the peroxisome as one synthesis site of certain steps of several secondary metabolites including aflatoxin, penicillin, paxilline and AK-toxin (Maggio-Hall *et al.*, 2005; Saikia and Scott, 2009; Imazaki *et al.*, 2010; Bartoszewska *et al.*, 2011). Aflatoxin synthesis initiates in the peroxisome (Maggio-Hall *et al.*, 2005) but the last steps are housed in large, fused vesicle/vacuole compartments termed aflatoxisomes (Chanda *et al.*, 2009; Roze *et al.*, 2013). The terminal step(s) in fungal metabolite synthesis is assumed to involve some type of export, as most secondary metabolites are secreted molecules. However, despite this similar end destination, mechanisms of secretion are largely unknown.

Many fungal secondary metabolite clusters contain putative transporters, which seem like excellent candidates for secretion. Following this line of reasoning, some cluster transporters have been deleted and fungal strains examined for impact on product formation. The results are inconsistent: some transporter deletions show no phenotype [e.g. aflatoxin (Chang *et al.*, 2004)], some do not impact production but afford self-protection towards the endogenous metabolite [e.g. sirodesmin and gliotoxin (Gardiner *et al.*, 2005)], whereas others are required for product formation [e.g. zearalenone (Lee *et al.*, 2011)]. Thus the presence of a cluster transporter does not

Received 6 March, 2013; revised 11 February, 2014; accepted 12 February, 2014. *For correspondence. E-mail npkeller@wisc.edu; Tel. (+1) 608 262 9795; Fax (+1) 608 262 8418.

© 2014 John Wiley & Sons Ltd

necessarily present the mechanism of small molecule secretion. Exocytosis is also theorized to be involved in release of secondary metabolites, as is hypothesized for aflatoxin secretion from aflatoxosomes (Roze *et al.*, 2013). Taken together, these studies suggest there are additional, unknown mechanisms involved in the secretion of fungal secondary metabolites.

One type of secondary metabolite, well characterized in terms of biochemistry, is the fumiquinazolines (Fqs), which comprise a related, sequentially generated family of cytotoxic peptidyl alkaloids that are signature metabolites of *A. fumigatus* (Takahashi *et al.*, 1995; Belofsky *et al.*, 2000). Originally isolated from a marine *A. fumigatus* isolate, Fqs were found in over 40 wild-type (WT) isolates (Takahashi *et al.*, 1995; Frisvad *et al.*, 2009). A recent study also reported the isolation of fumiquinazoline C (FqC) from the conidial extracts of *A. fumigatus* (Gauthier *et al.*, 2012). The Fq framework is built by non-ribosomal peptide synthetase (NRPS) enzymatic machinery with anthranilate as a key non-proteinogenic amino acid building block (Ames and Walsh, 2010). Bioinformatic analyses followed by heterologous expression and purification in *Escherichia coli* of *A. fumigatus* proteins predicted to be involved in Fq biosynthesis led to the identification of a four-enzymatic reaction process that builds increasingly complex Fq scaffolds, starting with the trimodular NRPS FmqA (AFUA_6G12080) and ending with the FAD-dependent oxidoreductase FmqD (AFUA_6G12070) [(Ames and Walsh, 2010; Ames *et al.*, 2010; 2011) and depicted in Fig. 1A and B].

Despite the detailed knowledge of the biosynthetic pathway, neither the encoding gene cluster nor tissue/subcellular location of the Fq metabolites/enzymes has been characterized *in vivo*. Here we provide *in vivo* validation to the Fq biosynthetic framework and elucidate the subcellular localization of the enzymes responsible for generating each step of the Fq pathway. The cluster contains four biosynthetic enzyme-encoding genes termed *fmqA–D* and one transporter gene termed *fmqE* (AFUA_6G12040). Whereas the first two intermediates in the Fq pathway, FqF and FqA, are found at comparable levels in all fungal tissues, the final enzymatic product FqC predominates in conidial tissues. FqC accumulation requires localization of FmqD, the oxidoreductase responsible for FqC synthesis, to spore cell walls. Deletion of the FmqD N-terminal signal peptide eliminates localization of the oxidoreductase to the cell wall and yields a similar chemical profile as a Δ*fmqD* strain. BrlA is also required for FqC and FqA syntheses as a Δ*brlA* strain only produces FqF. This work presents a prototype for the networking of both spore-specific (BrlA) and secretion mechanisms (FmqD cell wall localization) in deposition and selective accumulation of specific natural products to spores of the differentiating fungus.


Results

The fumiquinazoline cluster consists of four biosynthetic genes *fmqA–fmqD*

To validate the activities observed for FmqA, FmqB (AFUA_6G12060), FmqC (AFUA_6G12050) and FmqD in the heterologous *E. coli* system (Ames and Walsh, 2010; Ames *et al.*, 2010; 2011), single deletants in each of these genes were examined for Fq synthesis (Fig. 1C). Mutants were created in the Af293 background and some also assessed in the CEA17 ΔKU80 background as noted below. Extracts from the *fmqA*, *fmqB*, *fmqC* and *fmqD* deletion mutants corroborate the findings obtained with purified enzymes published previously (Ames and Walsh, 2010; Ames *et al.*, 2010; 2011).

The trimodular NRPS, FmqA is required for production of all Fq metabolites in *A. fumigatus*. Inactivation of FmqA (Δ*fmqA*) renders a strain deficient for all fumiquinazolines (Fig. 1C). Deletion of the FAD-dependent monooxygenase, FmqB, responsible for oxidation of FqF, which then is acted upon by the monomodular NRPS, FmqC, to form FqA, resulted in loss of both FqA and FqC (Fig. 1C). This mutant also showed a 19-fold accumulation in FqF production over WT further supporting FqF being the biological precursor to FqA (Fig. 1C). Deletion of the monomodular NRPS, FmqC, also abolished FqA and FqC production (Fig. 1C). However, unlike the Δ*fmqB* strain, the Δ*fmqC* strain did not accumulate FqF [or any detectable amounts of products from FmqB activity such as oxidized forms of FqF, e.g. FqF-diol (Ames *et al.*, 2010)] but rather produced FqF levels similar to that of WT.

Interestingly, deletion of *fmqD* encoding the last enzymatic step of the pathway resulted in a significant decrease but not complete loss of FqC production when the strain is grown on solid media (Fig. 1C). The Δ*fmqD* mutant accumulates FqA (81-fold increase over WT) and FqF (15-fold increase over WT) but decreased FqC (10-fold decrease compared to WT). In contrast to the small amount of FqC produced in this strain grown on solid agar culture, FqC production is not observed when the Δ*fmqD* mutant is grown under liquid shake conditions (data not shown). CEA17 ΔKU80 mutants gave similar profiles as the Af293 mutants (data not shown). We also thought it is possible that *fmqE* (Fig. 1A), encoding a putative MFS transporter, could be involved in Fq metabolism and transport. However, loss of this gene had no impact on Fq production under conditions tested (data not shown).

Fumiquinazoline C selectively accumulates in conidial tissues

Comparison of individual Fq metabolite ratios from cultures grown on solid defined medium shows that both WT

© 2014 John Wiley & Sons Ltd, *Cellular Microbiology*

Fumiquinazoline biosynthesis in Aspergillus fumigatus 3

A)

Aspergillus fumigatus Af293 ——[2 kb]——[Ala]——[Module 3]——[Module 2]——[Module 1]
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |        |
                                                                 |          |         |
4 F. Y. Lim, B. Ames, C. T. Walsh and N. P. Keller

A) A. fumigatus (254 nm trace)

EIC trace overlay
Af293
FqC
FqA
CEA17
FqC
FqA
FqF
FqF

B) Af293 (254 nm trace)

EIC trace overlay
media
FqC
FqA
mycelia
FqF
FqC
FqA
FqF
media
mycelia

Fig. 2. LC-MS comparison of Fq metabolite profiles from solid growth and liquid shake conditions.
A. Fq profile from total crude extracts of AF293 and CEA17ΔKU80 on solid growth culture.
B. Fq profile from fungal tissue (mycelia) and growth filtrate (media) of AF293 grown in liquid shake. Left panel: UV trace overlay (from 254 nm detection). Right panel: Overlay of extracted ion chromatogram (EIC) for the Fq metabolites FqA (R<sub>T</sub> = 7.1 min), FqC (R<sub>T</sub> = 7.4 min), and FqF (R<sub>T</sub> = 7.9 min). The Fqs were identified with the Agilent MassHunter software using the ‘Find Compound by Formula’ option.

tissues of the fungus comprised of (i) conidiophores and conidia (spores), (ii) vegetative hyphae and some dislodged conidia and (iii) invasive hyphae and secreted metabolites (Fig. 3A). The FqC:[FqF+FqA] ratio was highest in the conidial fraction (5:1) and lowest in the invasive hyphae fraction (1:2) (Fig. 3B).

BrlA, but not AbaA and WetA, is necessary for fumiquinazoline C production

The results above show selective accumulation of FqC in asexual reproduction structures. To further examine the link between Fq production and spore development, Fq profiling was conducted in characterized mutants aberrant in conidiophore development. BrlA is a conidiation-specific transcription factor involved in the early stage of asexual development and necessary for conidiophore formation in the Aspergilli (Mah and Yu, 2006). Previous published results showed that FqC production is abolished in the ΔbrlA mutant (Gauthier et al., 2012). When we analysed the production of other Fq metabolites in this mutant, we observed that this mutant is still capable of producing FqF but not other Fq moieties (Fig. 4A). Next we asked if AbaA and WetA, two BrlA-regulated transcription factors involved in the middle and late stages of conidiophore development and maturation, were also required for Fq production. Fq production resembles that of WT in the ΔabaA mutant and is increased in the ΔwetA mutant (Fig. 4B). We also examined Fq production in a ΔflbE mutant, FlbE being an upstream regulator of BrlA. The ΔflbE mutant is delayed in conidiophore production and is decreased in brlA expression. As expected, Fq production is decreased but not eliminated in this mutant (Fig. 4B). In all cases, enhanced accumulation of FqC is still maintained in the ΔabaA, ΔwetA, and ΔflbE mutants.

These results demonstrate that brlA expression is both necessary and sufficient for Fq biosynthesis, which remains largely normal in mutants with reproductive structure abnormalities as in ΔabaA and ΔwetA strains. To further assess the regulatory impact of BrlA on Fq production, expression analysis of the Fq cluster genes was performed comparing both WT and the ΔbrlA mutant

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

Fumiquinazoline biosynthesis in Aspergillus fumigatus 5

A)

B)
DAD (254 nm) | EIC trace overlay
--- | ---
Conidia | Conidia
Top Agar | Top Agar
Bottom Agar | Bottom Agar

Fig. 3. FqC production is associated with fungal sporulation.
A. Schematic of fractionation. Fungal cultures are separated into three fractions: Conidial (conidiophores and conidia), top agar (primarily vegetative hyphae, some conidiophores and dislodged conidia) and bottom agar (invasive hyphae and secreted metabolites).
B. Fq profiles of the three fractions sampled from AF293 solid growth culture.

(Fig. 4C). We observed that *fmqA*, *fmqB*, *fmqC*, *fmqD*, and *fmqE* transcripts are undetectable in the ΔbrlA mutant.

The N-terminal signal peptide in FmqD is important for cell wall localization and fumiquinazoline C biosynthesis

Thus, we expressed the same two alleles using the constitutive promoter, *gpdA(p)*. These two overexpression variants are termed OEFmqD::GFP and OEFmqD[Δ1–21]::GFP respectively. Northern analysis confirmed higher expression levels of the overexpression OEFmqD::GFP variants compared to that of the native FmqD::GFP variants (Fig. S1). In addition, we overexpressed the nuclear protein, histone H2A, fused to GFP (OEH2A::GFP) to control for the possibility of subcellular mislocalization due to artefactual overexpression of a protein. As shown in Fig. 5B, nuclear localization of histone H2A is still maintained in all stages of fungal development (conidia, germlings, hyphae) examined in this study, demonstrating that subcellular localization is still maintained in the overexpression strains.

Examination of the overexpression FmqD::GFP strains showed that FmqD is localized to the cell wall of the conidia as seen by the ring-like fluorescence (Fig. 5C). A time-course fluorescence imaging of the germination process showed that the cell wall localization decreases during isotropic growth (swelling of spores, Fig. S2) and is regained in mature hyphae (Fig. 5C). Deletion of the N-terminal signal peptide FmqD[Δ1–21] results in loss of cell wall accumulation and localization of FmqD[Δ1–21] to the cytoplasm (Fig. 5C). In the hyphae, FmqD is localized to both the cell wall and septa while FmqD[Δ1–21] again accumulates in the cytoplasm (Fig. 5C). Cell wall

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

A)

AF293 WT

AF293 ΔbrlA

B)

1.20E+06

8.00E+05

6.00E+05

4.00E+05

2.00E+05

0E+00

WT

ΔbrlA

ΔabaA

ΔwetA

ΔflbE

C)

Asexual Development

WT

ΔbrlA

-

+

AFUA_6G12090 (HET-domain protein)

AFUA_6G12080 (fmqA)

AFUA_6G12070 (fmqD)

AFUA_6G12060 (fmqB)

AFUA_6G12050 (fmqC)

AFUA_6G12040 (fmqE)

AFUA_6G12030 (L-rhamnosidase, putative)

AFUA_6G12020 (DNA-binding protein, putative)

brlA

actin

hpi

18

24

18

24

Fig. 4. BrlA regulates production of FqC.

A. Fq profiles comparing wild-type AF293 and ΔbrlA mutant on solid growth culture shows that FqA and FqC production is BrlA-dependent. Left panel: UV trace overlay (from 254 nm detection). Right panel: Overlay of extracted ion chromatogram (EIC) for the Fq metabolites FqA (R<sub>T</sub> = 7.1 min), FqC (R<sub>T</sub> = 7.4 min), and FqF (R<sub>T</sub> = 7.9 min). The Fqs were identified with the Agilent MassHunter software using the ‘Find Compound by Formula’ option.

B. Fq production in various conidiophore developmental mutants of A. fumigatus. y-axis represents peak area of each Fq metabolites.

C. Semi-quantitative PCR of fmq cluster and border genes comparing wild-type AF293 and ΔbrlA 18 and 24 h after induction of asexual development (conidiophore formation).

Fumiquinazoline biosynthesis in Aspergillus fumigatus 7

Fig. 5. Characterization of N-terminal signal peptide involvement in FqC biosynthesis.
A. Fq profiles of FmqD::GFP and FmqD[Δ1–21]::GFP on solid growth cultures. Left panel: Diode array detector (DAD) trace from 254 nm detection. Right panel: Overlay of extracted ion chromatogram (EIC) for the Fq metabolites FqA (R<sub>T</sub> = 7.1 min), FqC (R<sub>T</sub> = 7.4 min), and FqF (R<sub>T</sub> = 7.9 min). The Fqs were identified with the Agilent MassHunter software using the ‘Find Compound by Formula’ option.
B. Overexpression of histone H2A still maintained nuclear localization in various developmental tissues examined in this study. From left to right: conidia, germlings, and hyphae. Top row: GFP. Bottom row: Merged GFP and differential interference contrast (DIC).
C. FmqD is cell wall-associated. Top panel: Conidia. Middle Panel: Hyphae. Bottom panel: Protoplasts. Protoplasts from the OEFmqD::GFP strain lose cell wall fluorescence while those from the OEFmqD[Δ1–21]::GFP strain retained cytoplasmic fluorescence. Top row: GFP. Bottom row: differential interference contrast (DIC).

localization of FmqD was confirmed via a protoplasting experiment where GFP fluorescence in the cell wall was largely lost from protoplasts of the OEFmqD::GFP mutant but is retained in the cytoplasm of protoplasts from the

FmqD[Δ1–21]::GFP mutant (Fig. 5C). Taken together, the above results demonstrate that the N-terminal signal peptide is important in both directing FmqD cell wall localization and FqC production in the fungus.

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

Localization of FmqD to the cell wall is dependent on ER-Golgi transport and actin polymerization

Three-dimensional modelling of FmqD predicts three surface-exposed N-glycosylation sites (Asn95, 257, and 284) (Ames *et al.*, 2011). These types of modifications are often associated with protein folding and trafficking through the ER-Golgi apparatus in *A. fumigatus* and other organisms (Chen *et al.*, 1994; Yanez *et al.*, 1998; Ames *et al.*, 2011; Jin, 2012). To address possible trafficking through the Golgi apparatus, the OEFmqD::GFP mutant was treated with Brefeldin A, a known inhibitor of ER-Golgi transport, and localization of FmqD observed at 2 and 4 h post treatment (Khalaj *et al.*, 2001). At 2 h post treatment, cell wall localization of FmqD is highly reduced in the hyphae of the treated samples but rather localized to a reticulate network (ER) dispersed throughout the hyphae (Fig. 6A). At 4 h post treatment, cell wall localization of FmqD is completely abolished and FmqD is mainly localized to punctate spots between the reticulate networks with low amounts still localized to the periphery of the reticulate networks (Fig. 6A). At both time points, the solvent carrier controls still maintained cell wall and septum localization (Fig. 6A). These observations demonstrate that the ER-Golgi transport is necessary to localize FmqD to the fungal cell wall. To further assess if FmqD localization to the cell wall is actin-dependent, we treated OEFmqD::GFP with cytochalasin A, a fungal compound known to inhibit actin polymerization (Khalaj *et al.*, 2001). As shown in Fig. 6B, cell wall localization is abolished with cytochalasin A treatment. Rather fluorescence appeared to be slightly diffused throughout the cytoplasm and also as punctate spots throughout the fungal hyphae (Fig. 6B).

FmqD as part of the *A. fumigatus* secretome

As many cell wall-associated proteins from *A. fumigatus* are secreted proteins that are transiently localized to the cell wall prior to release into the extracellular environment (Bernard *et al.*, 2002; Latge *et al.*, 2005), we asked if FmqD is also part of the *A. fumigatus* secretome. To assess this, we performed an extraction from the cell wall of *A. fumigatus* WT, OEH2A::GFP, and OEFmqD::GFP strains grown under liquid shake conditions at both 48 and 72 h post inoculation to isolate non-covalently bound proteins that are in transit through the cell wall. During the preparation of the cell wall, visualization of the cell wall fragments (hyphal ghosts) using fluorescence microscopy showed abundance of GFP fluorescence in the hyphal ghosts and residual intact hyphae of the OEFmqD::GFP strain while the OEH2A::GFP strain showed lack of GFP fluorescence in the hyphal ghosts along with minute amounts of nuclear GFP fluorescence in residual intact hyphae (data not shown). As depicted in Fig. 7A, minute amounts of the nuclear-associated H2A::GFP fusion protein can be observed in the non-covalent cell wall extracts of the OEH2A::GFP at both 48 and 72 h (lanes 2 and 6 respectively). An 82 kDa band corresponding to the size of the cell wall-associated FmqD::GFP fusion protein is observed in the OEFmqD::GFP strain at both 48 and 72 h (lanes 3 and 7 respectively) and appears at a much higher intensity compared to the H2A::GFP bands, thus supporting the above microscopy visualization.

To further strengthen our observations, proteins were also isolated from the growth medium of the above cultures. As shown in Fig. 7B, Western blot analysis of culture filtrates at 48 h showed presence of the FmqD::GFP fusion protein in the OEFmqD::GFP strain (lanes 3 and 6) and absence of the nuclear-associated H2A::GFP fusion protein in the OEH2A::GFP strain (lanes 2 and 5). At 72 h post inoculation, the accumulation of a 27 kDa band corresponding to the size of free GFP is also observed, most probably due to the cleavage/degradation of the fusion protein (lane 6).

Subcellular localization of the remaining Fmq proteins

To complete our understanding of the spatial distribution of other Fmq enzymes and Fq biosynthetic steps in the fungal body, we also examined subcellular localization of other members of the *fmq* gene cluster. In contrast to FmqD, none of the other enzymes are localized to the cell wall. Figure 8 shows that FmqA is associated with vesicles that varied in size but almost even spatial distribution between one another within the hyphae, FmqB and FmqC appeared evenly distributed in the cytoplasm and the putative transporter, FmqE is also localized to network-like vesicles within the hyphae but in a pattern distinct from FmqA (Fig. 8). Attempting to resolve the nature of the FmqA-associated vesicles, the OEFmqA::GFP mutant was stained with both FM4–64 and Hoechst 33342. FM4–64 is used to visualize endocytic processes, vacuoles and mitochondria in *Aspergillus* (Penalva, 2005) and Hoechst 33342 labels the fungal nuclei. FmqA did not colocalize with either stain (Fig. S3) and thus, leaving the nature of the vesicles unresolved.

Discussion

The fumiquinazolines comprise a family of cytotoxic peptidyl alkaloids that are signature metabolites of the opportunistic human pathogen *A. fumigatus*. This family of metabolites has received considerable interest due to their complex biochemistry and potential antitumour properties (Takahashi *et al.*, 1995; Liu *et al.*, 2005) but until now, nothing is known of their biosynthesis or localization *in vivo*. We show here that deletion of the Fq cluster genes *fmqA–D* yield Fq intermediates as predicted by

Fumiquinazoline biosynthesis in Aspergillus fumigatus 9

A)
| Carrier control | +Brefeldin A | Carrier control | +Brefeldin A |
| --- | --- | --- | --- |
| ![Image](image1) | ![Image](image2) | ![Image](image3) | ![Image](image4) |
| ![Image](image5) | ![Image](image6) | ![Image](image7) | ![Image](image8) |

2 HPT
4 HPT

B)
| Carrier control | +Cytochalasin A |
| --- | --- |
| ![Image](image9) | ![Image](image10) |
| ![Image](image11) | ![Image](image12) |

2 HPT

Fig. 6. Cell wall localization of FmqD is actin-dependent and requires ER-Golgi transport.
A. OEFmqD::GFP mutant incubated in the presence of Brefeldin A (BFA) shows lost of cell wall accumulation and presence of reticulate network (ER proliferation) at 2 h post treatment (HPT). Fluorescence is localized to punctated vesicles throughout the hyphae at 4 h post treatment.
B. Treatment with cytochalasin A (actin-depolymerizing agent) abolished cell wall localization of FmqD::GFP. The methanol and DMSO carrier control is depicted to the left of each time point presented. Top row: GFP. Bottom row: Merged GFP and differential interference contrast (DIC).

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

10 F. Y. Lim, B. Ames, C. T. Walsh and N. P. Keller

A)
kDa
260-
160-
110-
80-
60-
50-
40-
30-
20-
1   2   3   4
48 hr

B)
kDa
260-
160-
110-
80-
60-
50-
40-
30-
1   2   3   4   5   6
48 hr    72 hr

Fig. 7. FmqD is a non-covalent cell wall-associated protein.
A. Western blot analysis of non-covalent cell wall extracts from WT (lanes 1 and 5), OEH2A::GFP (lanes 2 and 6), and OEFmqD::GFP (lanes 3 and 7) isolated from 48 and 72 h grown cultures using primary monoclonal GFP antibodies. Lanes 4 and 8 depict 10 ng of purified C-terminal epitope-tagged GFP.
B. Western blot analysis of culture filtrate extracts from WT (lanes 1 and 4), OEH2A::GFP (lanes 2 and 5), and OEFmqD::GFP (lanes 3 and 6) isolated from 48 and 72 h grown cultures using primary monoclonal GFP antibodies. Fusion protein sizes on blot: H2A::GFP (45 kDa), FmqD::GFP (82 kDa).

biochemical studies (Ames and Walsh, 2010; Ames et al., 2010; 2011). Deletion of *fmqA* abolished all Fq production while deletions of *fmqB* and *fmqC* yielded only the first Fq precursor, FqF. Finally, *fmqD* mutants produced primarily FqF and FqA with trace amounts of FqC present only in crude extracts of solid grown cultures. This trace amount of FqC observed in solid grown cultures of Δ*fmqD* may be attributable to another promiscuous oxidoreductase with sufficient similarity to FmqD to weakly engage FqA. We find that FqC is concentrated in the conidial tissues compared to other Fq moieties and that this localization is dependent on the co-ordinate action of BrlA and secretion of FmqD to the cell wall. On the other hand, deletion of the putative transporter FmqE had no impact on Fq production.

The data supporting differential tissue distribution of the Fq moieties included enrichment of FqC in conidial fractions of solid grown culture, the lack of FqC enrichment in liquid grown culture (where few spores are generated) and the absence of this metabolite in the Δ*brlA* mutant. Unlike many secondary metabolite clusters, the Fq cluster lacks a specific transcription factor. The regulation of such clusters is largely unknown but when identified, has been found to be due to the global regulation.

Hyphae

WT | H2A | FmqA | FmqB | FmqC | FmqE
--- | --- | --- | --- | --- | ---
FITC |  |  |  |  |
Merged |  |  |  |  |

Fig. 8. Subcellular localization of the remaining Fmq proteins in the fungal hyphae. From left to right: wild-type, OEH2A::GFP, OEFmqA::GFP, OEFmqB::GFP, OEFmqC::GFP, and OEFmqE::GFP. Both FmqA::GFP and FmqE::GFP localize to vesicles within the hyphae while FmqB::GFP and FmqC::GFP localizes to the cytoplasm. H2A::GFP control strain still maintain nuclear localization. Top row: GFP. Bottom row: Merged GFP and differential interference contrast (DIC).

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

Fumiquinazoline biosynthesis in Aspergillus fumigatus 11

**lators such as LaeA and/or trans-acting transcription factors as characterized for several gene clusters in *Aspergillus* spp. (Brakhage et al., 2009; Lim et al., 2012a; Yin et al., 2012; Forseth et al., 2013). We found that a previously published microarray analysis showed that loss of BrlA resulted in downregulation of *fmqC* and *fmqD*, here found to be required for FqA and FqC synthesis, respectively (Twumasi-Boateng et al., 2009), suggestive of BrlA transcriptional control of at least part of this gene cluster.**

Our work supports BrlA as a major regulator of Fq synthesis and *fmq* gene expression. The role of BrlA in Fq production appears twofold as it is not only required for expression of the *fmq* cluster genes but also provides the structure (conidiophore and associated conidia) necessary for FmqD secretion and FqC deposition. Delay of *brlA* expression and/or conidiophore development in the ΔflbE (Kwon et al., 2010) can reduce but not eliminate Fq biosynthesis. On the other hand, once BrlA is activated, downstream mutations leading to deformed conidiophore development have no negative impact on Fq production. The wild-type levels of Fq products in the Δ*abaA* mutant suggests it is not the spore *per se* that is required for FqC accumulation as this mutant forms long chains of terminal cells on the conidiophore head unable to terminate in spores (Tao and Yu, 2011). We note that, to date, all BrlA-regulated *A. fumigatus* spore metabolites including FqC, endocrocin (Berthier et al., 2013) and trypacidin (Gauthier et al., 2012) are also regulated by LaeA which itself regulates BrlA (Perrin et al., 2007).

Aside from BrlA control, another factor important in FqC production and localization is the accumulation of FmqD to the cell wall. Figure 5 shows that FmqD localizes to both conidial and hyphal cell walls; however, in this study, the FmqD::GFP allele was driven by the *gpdA* promoter and thus resulting in high expression in the hyphal cell wall and septum. Figures 2 and 3 support a view that FmqD primarily accumulates in the conidium cell wall. Whereas many studies have addressed the importance of subcellular localization and compartmentalization of biosynthetic enzymes and pathway intermediates in secondary metabolism in fungi (Chanda et al., 2009; Saikia and Scott, 2009; Imazaki et al., 2010; Meijer et al., 2010; Bartoszewska et al., 2011), no study to date has demonstrated the secretion of a secondary metabolite biosynthetic enzyme. However, secretion of biotechnologically valuable fungal proteins (glucoamylases, cellulases and lipases) has been well studied in several genera including *Aspergillus* (Fleissner and Dersch, 2010; Saloheimo and Pakula, 2012). Many of these known secreted proteins are shown to first localize to the cell wall via the secretory pathway and eventually diffuse to the environment. *In silico* analysis of FmqD predicts that it bears attributes indicative of secretion through the classical ER-Golgi-mediated secretory pathway including an N-terminal signal peptide and putative surface-exposed N-glycosylation sites that have been shown to be important in the proper folding and localization of other *A. fumigatus* cell wall proteins (Zhang et al., 2008; 2009; Kotz et al., 2010; Li et al., 2011; Jin, 2012). Our study supports trafficking of FmqD through this pathway as deletion of the signal peptide abolished FmqD cell wall localization and greatly reduced FqC production [despite knowing that the signal peptide is not required for enzyme activity *in vitro* (Ames et al., 2011)]. Treatment of OEFmqD::GFP with a known inhibitor of ER-Golgi transport (Brefeldin A) and an actin-depolymerizing agent (cytochalasin A) disrupted the localization of FmqD, further demonstrating secretion of FmqD via the classical secretory pathway in an actin-dependent manner. Taken together, this suggests that proper transport and localization of FmqD and not just enzyme activity is important for FqC synthesis and accumulation.

The presence of FqF in the Δ*brlA* mutant suggests either a basal expression of *fmqA* independent of BrlA and undetectable by semi-quantitative PCR or participation of another NRPS in FqF synthesis. Towards this end, recent studies have shown considerable cross-talk between NRPS pathways in *A. fumigatus* (O’Hanlon et al., 2012; Yin et al., 2013a). Regardless of the route of production, the presence of a basal level of FqF could allow for a constant supply of the starting materials (FqF and oxidized form of FqF) in the fungal hyphae, which presumably would be readily available for conversion to FqA and FqC on the onset of conidiation. We also note that FqF is a branchpoint intermediate between the later stage fumiquinazoles and the tryptoquivaline/tryptoquialanine pathways (Gao et al., 2011; Afiyatullov et al., 2012) and speculate that storage of this precursor metabolite may be advantageous in quick conversion to final products dependent on environmental signals to the fungus.

A major conclusion from our work is that secondary metabolite localization in the fungal thallus is dependent on several mechanisms: in this case BrlA regulation of gene expression and conidiophore formation followed by cell wall localization of FmqD, the terminal enzyme of the Fq biosynthetic pathway. It is known that spore metabolites can contribute to the virulence of pathogenic fungi such as *A. fumigatus*; for instance, the spore metabolites endocrocin and trypacidin/fumiquinazoles exhibit neutrophil inhibiting and cytotoxic properties respectively (Gauthier et al., 2012). Furthermore, the localization of FmqD protein itself to the cell wall of the conidium may have a significant implication on host–pathogen interactions by either masking or aiding in immune detection of the fungus. As FmqD is non-covalently linked to the cell

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

Table 1. Strains used in this study.

| Fungal strain | Genotype | Source |
|---------------|----------|--------|
| AF293         | WT       | Xue et al. (2004) |
| AF293.1       | pyrG1    | Xue et al. (2004) |
| CEA17 KU80 pyrG+ | ΔakuB pyrG+ | da Silva Ferreira et al. (2006) |
| CEA17 KU80 pyrG– | ΔakuB pyrG– | da Silva Ferreira et al. (2006) |
| TFYL10.1      | ΔfmqA::A. parasiticus pyrG; pyrG1 | This study |
| TFYL11.1      | ΔfmqD::A. parasiticus pyrG; pyrG1 | This study |
| TFYL12.1      | ΔfmqB::A. parasiticus pyrG; pyrG1 | This study |
| TFYL13.1      | ΔfmqC::A. parasiticus pyrG; pyrG1 | This study |
| TFYL24.1      | ΔakuB; ΔfmqE::A. parasiticus pyrG | This study |
| TFYL25.1      | ΔakuB; ΔfmqB::A. parasiticus pyrG | This study |
| TFYL26.1      | ΔakuB; ΔfmqD::A. parasiticus pyrG | This study |
| TFYL27.1      | ΔakuB; ΔfmqA::A. parasiticus pyrG | This study |
| TFYL28.1      | ΔakuB; fmqD::eGFP::A. fumigatus pyrG | This study |
| TFYL29.1      | ΔakuB; fmqD[Δ1–21]::eGFP::A. fumigatus pyrG | This study |
| TFYL34.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqD::eGFP::A. fumigatus pyrG | This study |
| TFYL33.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqD[Δ1–21]::eGFP::A. fumigatus pyrG | This study |
| TFYL48.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqA::eGFP::A. fumigatus pyrG | This study |
| TFYL49.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqB::eGFP::A. fumigatus pyrG | This study |
| TFYL50.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqC::eGFP::A. fumigatus pyrG | This study |
| TFYL51.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::fmqE::eGFP::A. fumigatus pyrG | This study |
| TFYL52.1      | ΔakuB::A. fumigatus pyrG; gpdA(p)::H2A::eGFP::A. fumigatus pyrG | This study |
| TKSS6.07      | pyrG1 flbE::A. nidulans pyrG | Kwon et al. (2010) |
| TSGa17        | pyrG1 ΔabaA::A. nidulans pyrG | Tao and Yu (2011) |
| TSGw4         | pyrG1 ΔwetA::A. nidulans pyrG | Tao and Yu (2011) |
| ΔAfbrlA7      | pyrG1 ΔbrlA::A. fumigatus pyrG | Mah and Yu (2006) |

wall, it is highly possible that the protein along with its secondary metabolite substrate, FqC, is being released as the spores swell both interstitially within the lung and intracellularly within resident phagocytes upon engulfment of spores. Despite being beyond the scope of this manuscript, the implications of both FmqD localization to the conidia cell wall and all Fqs on the pathogenicity of A. fumigatus will be of interest for future studies in our lab. Our findings from this work also provide a platform to explain results from previous studies where deletions of putative cluster transporters did not appear to have major impact on secondary metabolite production (Chang et al., 2004; Gardiner et al., 2005; Schrettl et al., 2010). Similarly, we found that deletion of the putative transporter FmqE did not affect FqC production under the conditions examined. Rather it was the localization of FmqD that impacted FqC production and presumably release to the environment.

### Experimental procedures

#### Fungal strains and growth conditions

All A. fumigatus strains used in this study are listed in Table 1. Two sequenced WT isolates were used to generate the fmq mutants. Strains were maintained as glycerol stocks and activated on solid glucose minimal medium (GMM) at 37°C (Shimizu and Keller, 2001). Growth media was supplemented with 5 mM of uridine and uracil for pyrG auxotrophs. For Fq profiling on solid medium, A. fumigatus strains were point-inoculated onto solid GMM at 1 × 10⁴ conidia total (in 5 μl) and cultured at 29°C or 37°C for 4 or 6 days respectively unless otherwise noted. For Fq profiling in liquid shake conditions, A. fumigatus strains were inoculated into 50 ml of liquid GMM at 1 × 10⁶ conidia ml⁻¹ and cultured at 37°C and 250 r.p.m. for 3 days unless otherwise noted.

#### Fumiquinazoline mutant construction

All primers used in this study are listed in Table S1 and gene deletion mutants in this study were made by targeted integration of the deletion constructs through transformation (Szewczyk et al., 2006; Lim et al., 2012b). The deletion constructs were made using a double-joint fusion PCR (DJ-PCR) approach (Szewczyk et al., 2006; Lim et al., 2012b). To construct the deletion cassettes, two 1.0–1.5 kb fragments flanking the deletion region of each gene were amplified from AF293 genomic DNA template. The selectable auxotrophic marker, Aspergillus parasiticus pyrG, was amplified from plasmid pJW24 (Calvo et al., 2004). The deletion constructs are verified with restriction enzyme digestion. Genomic DNA isolation, protoplast generation, and transformation procedures were carried out as previously described (Szewczyk et al., 2006; Lim et al., 2012b). The transformants were subjected to multiplex PCR using internal primers to the gene of interest and control internal primers to A. fumigatus gpdA (Table S1). To construct the FmqD::GFP fusion strains, FmqD was fused at its C-terminus to GFP::A. fumigatus pyrG with and without the N-terminal 1–21 amino acid residues. GFP::A. fumigatus pyrG was amplified from plasmid pFNO3 (Yang et al., 2004). The OEFmqD::GFP mutants were constructed as above with the exception of the native fmqD promoter replaced with the constitutive promoter, gpdA(p), amplified from plasmid pJMP9.1 (Lim et al., 2012a). The above transformants were subjected to an initial PCR screen for the

presence/absence of the signal peptide and FmqD::GFP fusion construct. The constructs for all OEFmqA::GFP, OEFmqB::GFP, OEFmqC::GFP, OEFmqE::GFP are generated using an *in vivo* yeast recombination cloning system (Yin *et al.*, 2013b). Briefly, the vector, pYH-wA-pyrG (Yin *et al.*, 2013b), was subjected to HindIII/SacI digestion. Individual segments consisting of (i) ~1 kb 5′flanking region from wild-type genomic DNA, (ii) *gpdA(p)* amplified from pJMP9.1, (iii) *fmq* open reading frame from wild-type genomic DNA, (iv) *GFP::A. fumigatus pyrG* amplified from pFNO3, and (v) ~1 kb 3′flanking region from wild-type genomic DNA were amplified by high fidelity PCR. The above segments along with the digested vector were subjected to *in vivo* yeast recombination as previously described (Yin *et al.*, 2013b) to create plasmids pFYL7 (OEFmqA::GFP), pFYL5 (OEFmqB::GFP), pFYL6 (OEFmqC::GFP), and pFYL8 (OEFmqE::GFP). Each of the above plasmid was linearized and transformed into strain CEA17 ΔakuB pyrG- to yield strains TFYL48.1, TFYL49.1, TFYL50.1, and TFYL51.1 respectively. For all of the transformants generated above, correct integration of the transformation construct at the locus of interest was verified via Southern analysis using two restriction digest profiles according to standard procedures (Sambrook and Russell, 2001). One representative Southern blot for all mutants can be found in Fig. S4.

**Fq extraction and profiling**

**Sample preparation for Fq profiling from solid and liquid shake conditions.** Agar cores (approximately 1.5 cm in diameter) were prepared in triplicate for each strain cultured as described above (fungal strains and growth conditions). Three cores from each plate were extracted with 2 ml of methanol by briefly vortexing then incubated at room temperature for 20 min submerged in solvent. Subsequently, 0.5 ml of the sample extract was filtered using a 0.45 μm PTFE Mini-UniPrep filter vial (Agilent) and 10 μl of the filtrate injected for LC-MS analysis.

For liquid shake extracts, the mycelia (cultured as described in fungal strains and growth conditions) were removed by filtration, washed with distilled water, and lyophilized. Mycelia were extracted with 5 ml of methanol by vortexing and incubating as described in the previous paragraph. Total filtered culture media was lyophilized and also extracted with 5 ml methanol. Half millilitre of samples from mycelia or media extracts were filtered using a filter vial and 10 μl of the filtrate injected for LC-MS analysis.

**Extraction of different fungal tissues for Fq production.** Plates were prepared by overlaying 10 ml GMM top agar (0.75% agar) onto a solidified base layer of GMM agar (1.5% agar). Plates were point inoculated with 1 × 10⁶ conidia and grown at 37°C for 4 days followed by 25°C for 3 days. Conidia were harvested by flooding the plate with 0.1% Tween 20 (2 × 6 ml) followed by gentle agitation with a plastic spreader. The conidial fraction was inspected under the microscope to be largely free of hyphae and conidiophores. The conidial fraction was collected by centrifugation, washed once with 6 ml 0.1% Tween 20, and extracted with 5 ml 1:1 mixture of ethyl acetate:methanol (EtOAc:MeOH). The original plate was washed (3 × 10 ml volumes of 0.1% Tween 20) and the top agar layer (containing dislodged conidiophores, vegetative mycelia, and media) was removed with a spatula and

extracted with 15 ml EtOAc:MeOH (1:1). The bottom agar layer (which contained some substrate mycelia and the bulk of the growth media) was washed with 10 ml water then removed, finely chopped, and extracted with 25 ml EtOAc:MeOH (1:1). Two millilitres from each fraction was dried *in vacuo*, the residue reconstituted in 0.5 ml of 50% acetonitrile (MeCN), filtered as described in the previous section, and 10 μl samples of each fraction injected for LC-MS analysis.

**LC-MS analysis and Fq profiling.** LC-MS data were acquired on an Agilent Technologies 6520 Accurate-Mass QTOF instrument supported by the Taplin Funds for Discovery Program. Samples were injected onto a Gemini-NX C18 column (50 × 2.0 mm, 5 micron, 100 Å, Phenomenex) equilibrated in solvent system A (water + 0.1% formic acid) plus 20% B (acetonitrile + 0.1% formic acid) and the following LC method was run: 20% B for 2 min, ramp to 60% B over 10 min, ramp to 100% B over 0.5 min, and a hold at 100% B for 1 min. The column was equilibrated in 20% B for 4 min prior to beginning the next injection sequence. MS data were collected using electrospray ionization in positive ion detection mode (capillary voltage of 3500 V, fragmentor set to 125 V, skimmer to 70 V, 12 l min⁻¹ drying gas at 295°C, 30 psig nebulizer) with real-time internal reference mass correction (API-TOF Reference Mass Solution, Agilent). UV data were collected using diode array detection. Data were analysed using Agilent MassHunter Qualitative Analysis software. TIC, total ion current chromatogram; DAD, diode-array detector (UV₂₅₄nm trace); and EIC, extracted ion chromatogram for all WT, Δ*fmqA*, Δ*fmqB*, Δ*fmqC*, and Δ*fmqD* are showed in supplemental Figs S5–S9 respectively. HPLC data on the various conidiation mutants were acquired using a Perkin Elmer Flexar Instrument. Samples were injected onto a ZORBAX Eclipse XDB-C18 column (150 × 4.6 mm, 5 micron, 100 Å, Agilent) equilibrated in solvent system A (water + 0.1% formic acid) plus 20% B (acetonitrile + 0.1% formic acid) and the following HPLC method was run: 20% B for 2 min, ramp to 60% B over 10 min, ramp to 100% B over 0.5 min, and a hold at 100% B for 5 min. The column was equilibrated in 20% B for 4 min prior to beginning the next injection sequence.

**Fluorescence microscopy**

A coverslip was placed in a small Petri dish and subjected to a 15 min exposure to a germicidal UV-C lamp in a biosafety cabinet. *A. fumigatus* strains were inoculated at 2 × 10² conidia ml⁻¹ in liquid GMM into the above prepared Petri dish and incubated at 30°C overnight. For spore germination assay, strains were cultured as above but at 37°C. Media was decanted from the Petri dish and the coverslip along with adhered fungal hyphae was then mounted onto a pre-cleaned microscope slide in mounting solution (50 mM sodium phosphate buffer pH 8.0, 50% glycerol, 0.1% propyl gallate), and sealed with nail polish. For staining with FM4–64 and Hoechst 33342, the coverslips were removed from the Petri dishes and excess media removed. Two hundred microlitres of staining solution was pipetted onto the coverslips and incubated for 10–15 min at room temperature. Coverslips were washed twice in PBS and mounted onto a pre-cleaned microscope slide in mounting solution. Images were taken with a Zeiss AxioImager A10 equipped with a Zeiss EC Plan-NEOFLUAR 40×/1.3 Oil DIC/∞/0.17 and Zeiss EC Plan-NEOFLUAR 100×/1.3 Oil DIC/∞/0.17 objective and a series 120 X-Cite® light source (EXFO). GFP, Hoescht, and FM4–64 were

imaged using a Zeiss filter set 10, Zeiss filter set 49, and Zeiss filter set 00 respectively. Microscope settings were kept identical for all images. For imaging of spores, *A. fumigatus* strains were cultured at 37°C for 4 days, harvested in 0.1% Tween 20, and subjected to the same imaging procedure as described above. For imaging of protoplasts, *A. fumigatus* strains were protoplasted as described previously (Szewczyk et al., 2006; Lim et al., 2012b) and imaged as described above. Briefly, overnight grown fungal culture is subjected to treatment using VinoTaste® Pro (Novozymes) and lysing enzymes from Trichoderma harzianum (#L1412; Sigma Aldrich) to remove the cell wall. For brefeldin A and cytochalasin A treatment, liquid GMM containing wild-type, OEFmqD::GFP and OEFmqD[Δ1–21]::GFP spores were inoculated into coverslip filled Petri dishes and incubated overnight at 30°C. Coverslips were removed and treated with 100 μg ml⁻¹ Brefeldin A (BFA) and 80 μg ml⁻¹ cytochalasin A (CA) along with a solvent carrier control (methanol and DMSO respectively). Coverslips were removed at 2 and 4 h post BFA treatment and 2 h post CA treatment, mounted onto pre-cleaned microscope slide in mounting solution, and imaged as described above.

### Culture conditions and protein preparation for FmqD secretion

#### Growth and sample preparation from mycelium and culture filtrates.
To determine if FmqD is secreted into the growth medium, *A. fumigatus* WT, OEH2A::GFP, and OEFmqD::GFP strains were inoculated into 50 ml of liquid GMM at 1 × 10⁶ conidia ml⁻¹ and cultured at 25°C and 250 r.p.m. Both mycelia and culture filtrate were collected at 48 and 72 h post inoculation and separated via filtration through Miracloth. Mycelia were rinsed twice with distilled water, flash frozen in liquid nitrogen, and stored at –80°C until ready to be used. The culture filtrates were immediately centrifuged at 4°C and 4000 r.p.m. for 10 min to remove any remaining mycelia and filtered through 0.8 μM membrane filters (Corning), lyophilized, and stored at –80°C until ready to be used.

#### Preparation of non-covalently bound proteins from cell wall.
To prepare non-covalent cell wall extract (NC-CWE), approximately equal amounts (wet weight) of mycelia were ground in liquid nitrogen to a fine powder using pestle and mortar. Ground mycelia were immediately reconstituted in buffer B [100 mM Tris/ HCl (pH 7.5), 300 mM NaCl, 10% glycerol, 0.1% NP-40, 1 mM EDTA, 1 mM PMSF, 1 tablet in 50 ml of cOmplete Protease Inhibitor Cocktail (Roche)] and centrifuged at 5500 r.p.m. for 10 min. Supernatant containing soluble cytoplasmic proteins were carefully removed. The cell wall pellet were reconstituted in 200 mM Tris/HCl (pH 7.8) buffer containing 20 mM EDTA and 1 mM PMSF and centrifuged at 200–300 r.p.m. for 2 min. Upon centrifugation, the upper gradient containing hyphal ghosts was carefully separated from the lower gradient containing larger fragments of intact hyphae. This step was repeated multiple times to remove most of the intact hyphae and the sample inspected under the microscope to show mainly hyphal ghosts. The cell wall fractions were combined, centrifuged at 5500 r.p.m. for 10 min, and the cell wall pellets were washed six to eight times in 200 mM Tris/HCl (pH 7.8) buffer containing 20 mM EDTA and 1 mM PMSF followed by three washes in 1 M NaCl. Non-covalently bound proteins were extracted from the cell wall by boiling the cell wall pellets twice in 50 mM Tris/HCl (pH 7.4) buffer containing 50 mM EDTA, 2% SDS, and 40 mM β-mercaptoethanol for 10 min (Bernard et al., 2002). The boiled samples were centrifuged and the supernatant containing NC-CWE were combined and stored at –80°C until ready to be used.

#### Protein analysis.
The lyophilized culture filtrates were solubilized in Buffer B and precipitated using 20% trichloroacetic acid (TCA). Briefly, one volume of 100% TCA was added to four volumes of the solubilized culture filtrates, incubated on ice for an hour, and centrifuged to pellet the precipitated proteins. The pellets were washed twice in pre-chilled acetone (w/v) and reconstituted in 2× LDS buffer [0.5 M Tris/HCl (pH 8.5), 20% glycerol, 4% LDS, 1 mM EDTA, 0.44 mM Coomassie, 5% β-mercaptoethanol] and stored at –20°C until ready for use. The solubilized NC-CWE materials were precipitated with 4 volumes of ethanol, reconstituted in 2× LDS buffer, and stored at –20°C until ready for use. For gel electrophoresis, the samples were heated at 95°C for 5 min prior to loading and resolved using a 10% Bis Tris gel in MOPS running buffer. Equal loading of proteins were visualized using Coomassie blue staining.

#### Western blot analysis.
The samples along with a 10 ng of purified C-terminal epitope-tagged GFP (Stalder et al., 2011) were subjected to gel electrophoresis as described above and transferred onto a PVDF membrane (Millipore) at 15 V for an hour on a Trans-Blot® SD Semi-Dry transfer cell (Bio-Rad) using standard manufacturer’s protocol. The membranes were then visualized using Ponceau red staining for successful transfer and blocked for 2 h at room temperature in TBS-T containing 5% non-fat milk. The membranes were then incubated in 1:2000 (v/v) primary monoclonal GFP antibody (NeoClone) overnight at 4°C in TBS-T containing 0.1% non-fat milk and washed four times for 15 min each in TBS-T. The membranes were then incubated in 1:5000 (v/v) secondary HRP-conjugated polyclonal goat anti-mouse antibody (BioLegend) in TBS-T containing 0.1% non-fat milk for 50 min at room temperature and washed in TBS-T as described above. Samples were then incubated in Clarity™ Western ECL substrate (Bio-Rad) for 5 min following manufacturer’s protocol and subjected to 2 min film exposure.

### Acknowledgements
This research was funded by GM49338 to C.W. by Award F32GM090475 from the National Institute of General Medical Sciences to B.A. and by NIH 1 R01 AI065728-01 and NIH Grant GM084077 to N.K. The authors would like to thank Dr Jae-Hyuk Yu at University of Wisconsin-Madison for kindly providing the conidiophore development mutants in *A. fumigatus*. The authors would also like to thank Drs Richard R. Burgess and Nancy E. Thompson at the University of Wisconsin-Madison for kindly providing the purified epitope-tagged GFP for Western analysis.

### References
Afiyatullov, S., Zhuravleva, O.I., Antonov, A.S., Kalinovsky, A.I., Pivkin, M.V., Menchinskaya, E.S., and Aminin, D.L. (2012) New metabolites from the marine-derived fungus *Aspergillus fumigatus*. Nat Prod Commun 7: 497–500.

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

Ames, B.D., and Walsh, C.T. (2010) Anthranilate-activating modules from fungal nonribosomal peptide assembly lines. *Biochemistry* 49: 3351–3365.

Ames, B.D., Liu, X., and Walsh, C.T. (2010) Enzymatic processing of fumiquinazoline F: a tandem oxidative-acylation strategy for the generation of multicyclic scaffolds in fungal indole alkaloid biosynthesis. *Biochemistry* 49: 8564–8576.

Ames, B.D., Haynes, S.W., Gao, X., Evans, B.S., Kelleher, N.L., Tang, Y., and Walsh, C.T. (2011) Complexity generation in fungal peptidyl alkaloid biosynthesis: oxidation of fumiquinazoline A to the heptacyclic hemiaminal fumiquinazoline C by the flavoenzyme Af12070 from *Aspergillus fumigatus*. *Biochemistry* 50: 8756–8769.

Bartoszewska, M., Opalinski, L., Veenhuis, M., and van der Klei, I.J. (2011) The significance of peroxisomes in secondary metabolite biosynthesis in filamentous fungi. *Biotechnol Lett* 33: 1921–1931.

Belofsky, G.N., Anguera, M., Jensen, P.R., Fenical, W., and Kock, M. (2000) Oxepinamides A–C and fumiquinazoles H–I: bioactive metabolites from a marine isolate of a fungus of the genus *Acremonium*. *Chemistry (Easton)* 6: 1355–1360.

Bernard, M., Mouyna, I., Dubreucq, G., Debeaupuis, J., Fontaine, T., Vorgias, C., *et al.* (2002) Characterization of a cell-wall acid phosphatase (PhoAp) in *Aspergillus fumigatus*. *Microbiology* 148: 2819–2829.

Berthier, E., Lim, F.Y., Deng, Q., Guo, C.-J., Kontoyiannis, D.P., Wang, C.C.C., *et al.* (2013) Low-volume toolbox for the discovery of immunosuppressive fungal secondary metabolites. *PLoS Pathog* 9: e1003289.

Brakhage, A.A., Thon, M., Sprote, P., Scharf, D.H., Al-Abdallah, Q., Wolke, S.M., and Hortschansky, P. (2009) Aspects on evolution of fungal beta-lactam biosynthesis gene clusters and recruitment of trans-acting factors. * Phytochemistry* 70: 1801–1811.

Calvo, A.M., Bok, J., Brooks, W., and Keller, N.P. (2004) veA is required for toxin and sclerotial production in *Aspergillus parasiticus*. *Appl Environ Microbiol* 70: 4733–4739.

Chanda, A., Roze, L.V., Kang, S., Artymovich, K.A., Hicks, G.R., Raikhel, N.V., *et al.* (2009) A key role for vesicles in fungal secondary metabolism. *Proc Natl Acad Sci USA* 106: 19533–19538.

Chang, P.K., Yu, J., and Yu, J.H. (2004) *aflT*, a MFS transporter-encoding gene located in the aflatoxin gene cluster, does not have a significant role in aflatoxin secretion. *Fungal Genet Biol* 41: 911–920.

Chen, H.M., Ford, C., and Reilly, P.J. (1994) Substitution of asparagine residues in *Aspergillus awamori* glucoamylase by site-directed mutagenesis to eliminate N-glycosylation and inactivation by deamidation. *Biochemical Journal* 301: 275–281.

Coyle, C.M., Kenaley, S.C., Rittenour, W.R., and Panaccione, D.G. (2007) Association of ergot alkaloids with conidiation in *Aspergillus fumigatus*. *Mycologia* 99: 804–811.

Fleissner, A., and Dersch, P. (2010) Expression and export: recombinant protein production systems for *Aspergillus*. *Appl Microbiol Biotechnol* 87: 1255–1270.

Forseth, R.R., Amaike, S., Schwenk, D., Affeldt, K.J., Hoffmeister, D., Schroeder, F.C., and Keller, N.P. (2013)

Homologous NRPS-like gene clusters mediate redundant small-molecule biosynthesis in *Aspergillus flavus*. *Angew Chem Int Ed Engl* 52: 1590–1594.

Frisvad, J.C., Rank, C., Nielsen, K.F., and Larsen, T.O. (2009) Metabolomics of *Aspergillus fumigatus*. *Med Mycol* 47 (Suppl. 1): 53–71.

Gao, X., Chooi, Y.H., Ames, B.D., Wang, P., Walsh, C.T., and Tang, Y. (2011) Fungal indole alkaloid biosynthesis: genetic and biochemical investigation of the tryptoquialanine pathway in *Penicillium aethiopicum*. *J Am Chem Soc* 133: 2729–2741.

Gardiner, D.M., Jarvis, R.S., and Howlett, B.J. (2005) The ABC transporter gene in the sirodesmin biosynthetic gene cluster of *Leptosphaeria maculans* is not essential for sirodesmin production but facilitates self-protection. *Fungal Genet Biol* 42: 257–263.

Gauthier, T., Wang, X., Sifuentes Dos Santos, J., Fysikopoulos, A., Tadrist, S., Canlet, C., *et al.* (2012) Trypacidin, a spore-borne toxin from *Aspergillus fumigatus*, is cytotoxic to lung cells. *PLoS ONE* 7: e29906.

Imazaki, A., Tanaka, A., Harimoto, Y., Yamamoto, M., Akimitsu, K., Park, P., and Tsuge, T. (2010) Contribution of peroxisomes to secondary metabolism and pathogenicity in the fungal plant pathogen *Alternaria alternata*. *Eukaryot Cell* 9: 682–694.

Jin, C. (2012) Protein glycosylation in *Aspergillus fumigatus* is essential for cell wall synthesis and serves as a promising model of multicellular eukaryotic development. *Int J Microbiol* 2012: 654251.

Khalaj, V., Brookman, J.L., and Robson, G.D. (2001) A study of the protein secretory pathway of *Aspergillus niger* using a glucoamylase-GFP fusion protein. *Fungal Genet Biol* 32: 55–65.

Kotz, A., Wagener, J., Engel, J., Routier, F.H., Echtenacher, B., Jacobsen, I., *et al.* (2010) Approaching the secrets of N-glycosylation in *Aspergillus fumigatus*: characterization of the AfOch1 protein. *PLoS ONE* 5: e15729.

Kwon, N.J., Shin, K.S., and Yu, J.H. (2010) Characterization of the developmental regulator FlbE in *Aspergillus fumigatus* and *Aspergillus nidulans*. *Fungal Genet Biol* 47: 981–993.

Latge, J.P., Mouyna, I., Tekaia, F., Beauvais, A., Debeaupuis, J.P., and Nierman, W. (2005) Specific molecular features in the organization and biosynthesis of the cell wall of *Aspergillus fumigatus*. *Med Mycol* 43 (Suppl. 1): S15–S22.

Lee, S., Son, H., Lee, J., Lee, Y.R., and Lee, Y.W. (2011) A putative ABC transporter gene, ZRA1, is required for zearalenone production in *Gibberella zeae*. *Curr Genet* 57: 343–351.

Li, K., Ouyang, H., Lu, Y., Liang, J., Wilson, I.B., and Jin, C. (2011) Repression of N-glycosylation triggers the unfolded protein response (UPR) and overexpression of cell wall protein and chitin in *Aspergillus fumigatus*. *Microbiology* 157: 1968–1979.

Lim, F.Y., Hou, Y., Chen, Y., Oh, J.H., Lee, I., Bugni, T.S., and Keller, N.P. (2012a) Genome-based cluster deletion reveals an endocrocin biosynthetic pathway in *Aspergillus fumigatus*. *Appl Environ Microbiol* 78: 4117–4125.

Lim, F.Y., Sanchez, J.F., Wang, C.C., and Keller, N.P. (2012b) Toward awakening cryptic secondary metabolite gene

© 2014 John Wiley & Sons Ltd, Cellular Microbiology

clusters in filamentous fungi. *Methods Enzymol* 517: 303–324.

Liu, J.F., Ye, P., Zhang, B., Bi, G., Sargent, K., Yu, L., *et al.* (2005) Three-component one-pot total syntheses of glyantrypine, fumiquinazoline F, and fiscalin B promoted by microwave irradiation. *J Org Chem* 70: 6339–6345.

Maggio-Hall, L.A., Wilson, R.A., and Keller, N.P. (2005) Fundamental contribution of beta-oxidation to polyketide mycotoxin production *in planta*. *Mol Plant Microbe Interact* 18: 783–793.

Mah, J.H., and Yu, J.H. (2006) Upstream and downstream regulation of asexual development in *Aspergillus fumigatus*. *Eukaryot Cell* 5: 1585–1595.

Meijer, W.H., Gidijala, L., Fekken, S., Kiel, J.A., Van den Berg, M.A., Lascaris, R., *et al.* (2010) Peroxisomes are required for efficient penicillin biosynthesis in *Penicillium chrysogenum*. *Appl Environ Microbiol* 76: 5702–5709.

O'Hanlon, K.A., Gallagher, L., Schrettl, M., Jochl, C., Kavanagh, K., Larsen, T.O., and Doyle, S. (2012) Nonribosomal peptide synthetase genes *pesL* and *pes1* are essential for Fumigaclavine C production in *Aspergillus fumigatus*. *Appl Environ Microbiol* 78: 3166–3176.

Penalva, M.A. (2005) Tracing the endocytic pathway of *Aspergillus nidulans* with FM4–64. *Fungal Genet Biol* 42: 963–975.

Perrin, R., Fedorova, N., Bok, J., Cramer, R., Wortman, J., Kim, H., *et al.* (2007) Transcriptional regulation of chemical diversity in *Aspergillus fumigatus* by LaeA. *PLoS Pathog* 3: 508–517.

Roze, L.V., Hong, S.Y., and Linz, J.E. (2013) Aflatoxin biosynthesis: current frontiers. *Annu Rev Food Sci Technol* 4: 293–311.

Saikia, S., and Scott, B. (2009) Functional analysis and subcellular localization of two geranylgeranyl diphosphate synthases from *Penicillium paxilli*. *Mol Genet Genomics* 282: 257–271.

Saloheimo, M., and Pakula, T.M. (2012) The cargo and the transport system: secreted proteins and protein secretion in *Trichoderma reesei* (*Hypocrea jecorina*). *Microbiology* 158: 46–57.

Sambrook, J., and Russell, D.W. (2001) *Molecular Cloning: A Laboratory Manual*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Schrettl, M., Carberry, S., Kavanagh, K., Haas, H., Jones, G.W., O'Brien, J., *et al.* (2010) Self-protection against gliotoxin-A component of the gliotoxin biosynthetic cluster, GliT, completely protects *Aspergillus fumigatus* against exogenous gliotoxin. *PLoS Pathog* 6: e1000952.

Shimizu, K., and Keller, N.P. (2001) Genetic involvement of a cAMP-dependent protein kinase in a G protein signaling pathway regulating morphological and chemical transitions in *Aspergillus nidulans*. *Genetics* 157: 591–600.

da Silva Ferreira, M.E., Kress, M.R., Savoldi, M., Goldman, M.H., Hartl, A., Heinekamp, T., *et al.* (2006) The *akuB(KU80)* mutant deficient for nonhomologous end joining is a powerful tool for analyzing pathogenicity in *Aspergillus fumigatus*. *Eukaryot Cell* 5: 207–211.

Stalder, E.S., Nagy, L.H., Batalla, P., Arthur, T.M.,

Thompson, N.E., and Burgess, R.R. (2011) The epitope for the polyol-responsive monoclonal antibody 8RB13 is in the flap-domain of the beta-subunit of bacterial RNA polymerase and can be used as an epitope tag for immunoaffinity chromatography. *Protein Expr Purif* 77: 26–33.

Szewczyk, E., Nayak, T., Oakley, C.E., Edgerton, H., Xiong, Y., Taheri-Talesh, N., *et al.* (2006) Fusion PCR and gene targeting in *Aspergillus nidulans*. *Nat Protoc* 1: 3111–3120.

Takahashi, C., Matsushita, T., Doi, M., Minoura, K., Shingu, T., Kumeda, Y., and Numata, A. (1995) Fumiquinazoles A–G, novel metabolites of a fungus separated from a *Pseudolabrus marine* fish. *J Chem Soc [Perkin 1]* (18): 2345–2353. doi: 10.1039/p19950002345

Tao, L., and Yu, J.H. (2011) AbaA and WetA govern distinct stages of *Aspergillus fumigatus* development. *Microbiology* 157: 313–326.

Twumasi-Boateng, K., Yu, Y., Chen, D., Gravelat, F.N., Nierman, W.C., and Sheppard, D.C. (2009) Transcriptional profiling identifies a role for BrlA in the response to nitrogen depletion and for StuA in the regulation of secondary metabolite clusters in *Aspergillus fumigatus*. *Eukaryot Cell* 8: 104–115.

Xue, T., Nguyen, C.K., Romans, A., Kontoyiannis, D.P., and May, G.S. (2004) Isogenic auxotrophic mutant strains in the *Aspergillus fumigatus* genome reference strain AF293. *Arch Microbiol* 182: 346–353.

Yanez, E., Carmona, T.A., Tiemblo, M., Jimenez, A., and Fernandez-Lobato, M. (1998) Expression of the *Schwan-niomyces occidentalis* SWA2 amylase in *Saccharomyces cerevisiae*: role of N-glycosylation on activity, stability and secretion. *Biochem J* 329: 65–71.

Yang, L., Ukil, L., Osmani, A., Nahm, F., Davies, J., de Souza, C.P., *et al.* (2004) Rapid production of gene replacement constructs and generation of a green fluorescent protein-tagged centromeric marker in *Aspergillus nidulans*. *Eukaryot Cell* 3: 1359–1362.

Yin, W.B., Amaike, S., Wohlbach, D.J., Gasch, A.P., Chiang, Y.M., Wang, C.C., *et al.* (2012) An *Aspergillus nidulans* bZIP response pathway hardwired for defensive secondary metabolism operates through *aflR*. *Mol Microbiol* 83: 1024–1034.

Yin, W.B., Baccile, J.A., Bok, J.W., Chen, Y., Keller, N.P., and Schroeder, F.C. (2013a) A nonribosomal peptide synthetase-derived iron(III) complex from the pathogenic fungus *Aspergillus fumigatus*. *J Am Chem Soc* 135: 2064–2067.

Yin, W.B., Chooi, Y.H., Smith, A.R., Cacho, R.A., Hu, Y., White, T.C., and Tang, Y. (2013b) Discovery of cryptic polyketide metabolites from dermatophytes using heterologous expression in *Aspergillus nidulans*. *ACS Synth Biol* 2: 629–634.

Zhang, L., Zhou, H., Ouyang, H., Li, Y., and Jin, C. (2008) Afcwh41 is required for cell wall synthesis, conidiation, and polarity in *Aspergillus fumigatus*. *FEMS Microbiol Lett* 289: 155–165.

Zhang, L., Feng, D., Fang, W., Ouyang, H., Luo, Y., Du, T., and Jin, C. (2009) Comparative proteomic analysis of an *Aspergillus fumigatus* mutant deficient in glucosidase I (AfCwh41). *Microbiology* 155: 2157–2167.

Supporting information

Additional Supporting Information may be found in the online version of this article at the publisher’s web-site:

Fig. S1. Northern analysis of various FmqD::GFP strains. FmqD::GFP mRNA levels are much higher in both the OEFmqD::GFP and OEFmqD[Δ1–21]::GFP strains compared to that of FmqD::GFP and FmqD[Δ1–21]::GFP where the fusion proteins are placed under the governance of the native *fmqD* promoter. Top row represents *fmqD* transcript while bottom row represents a gel image of the ribosomal RNAs.

Fig. S2. Time-course fluorescence microscopy of FmqD localization during spore germination. First three panels show FmqD localization in spores at 2 h post inoculation (HPI) of WT, OEFmqD[Δ1–21]::GFP, and OEFmqD::GFP respectively. Panels 3–6 and 7–9 depict FmqD localization in spores at 6 and 10 HPIs respectively. Top row represents GFP images of spores while bottom row represents the merged GFP and differential interference contrast (DIC) images of spores.

Fig. S3. OEFmqA::GFP does not colocalize with endocytic membrane and nuclear stains, FM4–64 Hoescht respectively. Top row: OEFmqA::GFP stained with FM4–64. Bottom row: OEFmqA::GFP stained with Hoescht. From left to right: Merged imaged of GFP and FM4–64/Hoechst 33342, GFP alone, FM4–64/Hoechst 33342 alone, and differential interference contrast (DIC) alone.

Fig. S4. Southern analyses of Fq mutants.

Fig. S5. LC-MS analysis of the methanol extract of an agar core prepared from Af293 WT. TIC, total ion current chromatogram; DAD, diode-array detector (UV<sub>254nm</sub> trace); EIC, extracted ion chromatogram.

Fig. S6. LC-MS analysis of the methanol extract of an agar core prepared from Δ*fmqA*. TIC, total ion current chromatogram; DAD, diode-array detector (UV<sub>254nm</sub> trace); EIC, extracted ion chromatogram.

Fig. S7. LC-MS analysis of the methanol extract of an agar core prepared from Δ*fmqB*. TIC, total ion current chromatogram; DAD, diode-array detector (UV<sub>254nm</sub> trace); EIC, extracted ion chromatogram.

Fig. S8. LC-MS analysis of the methanol extract of an agar core prepared from Δ*fmqC*. TIC, total ion current chromatogram; DAD, diode-array detector (UV<sub>254nm</sub> trace); EIC, extracted ion chromatogram.

Fig. S9. LC-MS analysis of the methanol extract of an agar core prepared from Δ*fmqD*. TIC, total ion current chromatogram; DAD, diode-array detector (UV<sub>254nm</sub> trace); EIC, extracted ion chromatogram.

Table S1. Primers used in this study.
