
Biomolecules 2014, 4, 217-234; doi:10.3390/biom4010217

OPEN ACCESS

**biomolecules**

ISSN 2218-273X  
www.mdpi.com/journal/biomolecules/

Review

# Structure and Function of the Bi-Directional Bacterial Flagellar Motor

Yusuke V. Morimoto ${}^{1,2}$ and Tohru Minamino ${}^{2,*}$

1. Quantitative Biology Center, RIKEN, 6-2-3 Furuedai, Suita, Osaka 565-0874, Japan; E-Mail: ymorimoto@fbs.osaka-u.ac.jp  
2. Graduate School of Frontier Biosciences, Osaka University 1-3 Yamadaoka, Suita, Osaka 565-0871, Japan  

* Author to whom correspondence should be addressed; E-Mail: tohru@fbs.osaka-u.ac.jp; Tel: +81-6-6879-4625; Fax: +81-6-6879-4652.

Received: 30 December 2013; in revised form: 24 January 2014 / Accepted: 4 February 2014 / Published: 18 February 2014

---

Abstract: The bacterial flagellum is a locomotive organelle that propels the bacterial cell body in liquid environments. The flagellum is a supramolecular complex composed of about 30 different proteins and consists of at least three parts: a rotary motor, a universal joint, and a helical filament. The flagellar motor of *Escherichia coli* and *Salmonella enterica* is powered by an inward-directed electrochemical potential difference of protons across the cytoplasmic membrane. The flagellar motor consists of a rotor made of FliF, FliG, FliM and FliN and a dozen stators consisting of MotA and MotB. FliG, FliM and FliN also act as a molecular switch, enabling the motor to spin in both counterclockwise and clockwise directions. Each stator is anchored to the peptidoglycan layer through the C-terminal periplasmic domain of MotB and acts as a proton channel to couple the proton flow through the channel with torque generation. Highly conserved charged residues at the rotor-stator interface are required not only for torque generation but also for stator assembly around the rotor. In this review, we will summarize our current understanding of the structure and function of the proton-driven bacterial flagellar motor.

Keywords: Bacterial flagellum; rotary motor; motility; stator; rotor; torque generation; mechanosensor

Biomolecules 2014, 4

1. Introduction

Many bacteria swim in liquid media by rotating bacterial flagella. The bacterial flagellum is a supramolecular complex made of about 30 different proteins with copy numbers ranging from a few to a few tens of thousands (Figure 1). The flagellum consists of at least three parts: the basal body, the hook and the filament. The basal body is embedded within the cell membranes and acts as a rotary motor. The hook and filament extend outwards in the cell exterior. The filament works as a helical screw to produce the thrust. The hook exists between the basal body and filament and functions as a universal joint to smoothly transmit torque produced by the motor to the helical filament [1–3].

The flagellar motor of *Escherichia coli* and *Salmonella enterica* is fueled by the electrochemical potential of protons (H⁺) across the cytoplasmic membrane, i.e., the proton motive force [1,3–5]. The flagellar motor can operate in either a counter-clockwise (CCW, viewed from filament to motor), or clockwise (CW) direction. When most of the motors rotate in the CCW direction, the filaments form a bundle and propel the cell smoothly. When one or more motors spin in the CW direction, the bundle is disrupted and hence the cell tumbles and changes the swimming direction [6,7]. *E. coli* and *Salmonella* can move towards more favorable conditions and escape from undesirable ones for their survival by sensing temporal variations of environmental stimuli, such as chemical attractants and repellents, temperature and pH via methyl-accepting chemotaxis proteins (MCP). MCPs are transmembrane proteins with a large cytoplasmic domain involved in interactions with a histidine kinase CheA, and an adaptor protein CheW. The MCPs control CheA autophosphorylation. Phosphorylated CheA transfers its phosphate group to a response regulator CheY. Phosphorylated CheY (CheY-P) binds to the cytoplasmic face of the flagellar motor, allowing the motor to spin in the CW direction [8].

The flagellar motor consists of a rotor and a dozen stators (Figure 2). The rotor consists of four flagellar proteins, FliF, FliG, FliM and FliN. FliF forms the MS ring within the cytoplasmic membrane [9,10]. FliG, FliM and FliN form the C ring on the cytoplasmic face of the MS ring [11,12]. FliG, FliM and FliN are also responsible for switching the direction of motor rotation, enabling the motor to rotate in either CCW or CW [13]. Each stator consists of two integral membrane proteins, MotA and MotB [14–16] and is postulated to be anchored to the peptidoglycan (PG) layer through a well-conserved peptidoglycan binding (PGB) motif within the C-terminal periplasmic domain of MotB (MotBc) [17]. MotA and MotB form a proton channel complex to couple proton flow through the channel with torque generation [18–21]. Torque is generated by interactions between MotA and FliG [22]. This article covers our current discoveries of the proton-driven flagellar motor of *E. coli* and *Salmonella* with the emphasis being on the structure, function and dynamic properties of the motor.

Biomolecules 2014, 4

Figure 1. Schematic diagram of the bacterial flagellum. The flagellum is a locomotive organelle for bacterial propulsion. The flagellum consists of the basal body, which acts as a reversible rotary motor, the hook, which functions as a universal joint and the filament, which works as a helical screw. OM, outer membrane; PG, peptidoglycan layer; CM, cytoplasmic membrane.

![Diagram of Bacterial Flagellum](image1.png)

Figure 2. Schematic diagram of the proton-driven bacterial flagellar motor. The flagellar motor consists of a reversible rotor made of FliF, FliG, FliM and FliN and a dozen stators, each of which consists of MotA and MotB. FliF forms the MS ring within the cytoplasmic membrane. FliG, FliM and FliN form the C ring on the cytoplasmic face of the MS ring. OM, outer membrane; PG, peptidoglycan layer; CM, cytoplasmic membrane.

![Diagram of Proton-Driven Bacterial Flagellar Motor](image2.png)

Biomolecules 2014, 4

2. Rotor

The rotor consists of two ring structures, the MS and C rings (Figure 2). The MS ring is formed by 26 copies of FliF and acts as an assembly platform [10]. Using the MS ring as a template, FliG, FliM and FliN assemble into the C ring onto the cytoplasmic face of the MS ring [12]. FliG directly associates with the cytoplasmic face of the MS ring with a 1 FliG : 1 FliF stoichiometry [23,24], suggesting that the FliG ring has the 26-fold symmetry. FliG consists of three domains, FliG<sub>N</sub>, FliG<sub>M</sub> and FliG<sub>C</sub> [25]. FliG<sub>N</sub> directly binds to the MS ring [26,27]. FliG<sub>C</sub> is directly involved in the interaction with a stator protein MotA [22]. This suggests that FliG<sub>C</sub> is located at the upper part of the C ring. FliG<sub>M</sub> provides a binding site for FliM [28–31]. FliM and FliN form a stable complex with a 1 FliM: 4 FliN stoichiometry and occupy most part of the C ring [32]. The C ring has rotational symmetry that varies from 32-fold to 36-fold [33], and the diameter actually shows a range of variability (45 ~ 49.0 nm) [34]. Crystal structures of FliG [25,35–37], FliM [38], FliN [32,39] and the FliG-FliM complex [29–31] have been solved, and possible models for their organization have been proposed. Please see a recent review article focusing on the current models of the rotor ring structure [40].

Certain mutations in FliG, FliM and FliN affect the switching between CW and CCW rotation, indicating that these three proteins are responsible for switching the direction of motor rotation [13]. CheY-P binds to FliM and FliN, inducing cooperative conformational changes of the FliG ring to allow the motor to spin in the CW direction [41,42]. The switching between the CW and CCW states is highly cooperative [43]. The cooperative switching mechanism can be explained by a conformational spread model, in which a switching event is mediated by conformational changes in a ring of subunits that spread from subunit to subunit via nearest-neighbor interactions [44,45]. When motor speed is increased up to 100–150 Hz, both the CW-to-CCW and CCW-to-CW switching rates increase linearly and then decrease with speed to an equivalent degree, without changing the motor bias [46]. This suggests that the motor switch can sense the load on the stator-rotor interaction as well as the cytoplasmic level of CheY-P [46]. A mathematical model to explain this load-switching relationship of the motor has been proposed [47]. Recently, it has been reported that the nucleotide second messenger cyclic di-GMP (c-di-GMP), which promotes transition of single motile cells to surface-attached multicellular biofilm, regulates flagella-driven motility. A c-di-GMP binding protein YcgR binds to MotA, FliG and FliM in the presence of c-di-GMP, resulting not only in a significant reduction in motor speed but also in the CCW bias in the motor rotation [48–50].

Mutations located in and around helix<sub>MC</sub>, which connects FliG<sub>M</sub> and FliG<sub>C</sub>, generate a diversity of phenotype, including motors that are strongly CW biased, infrequent switchers, rapid switchers, and transiently or permanently paused. This suggests that helix<sub>MC</sub> is involved in switching of the direction of flagellar motor rotation [51]. The Salmonella FliG(ΔPAA) mutant motor, which has three-amino-acid deletion at positions 169 to 171, are extremely CW biased [52]. This mutant motor remains in the CW rotation even in the absence of CheY-P, indicating that the motor is locked in the CW state [37]. This suggests that the PAA deletion induces a conformational change of FliG at the rotor-stator interface in a way similar to the binding of CheY-P to FliM and FliN. Limited proteolysis has shown that the PAA deletion results in a conformational change in a hinge between FliG<sub>M</sub> and helix<sub>MC</sub>. This result is in agreement with the crystal structure of *Thermotoga maritima* FliG<sub>MC</sub>(ΔPEV) showing that the orientation of helix<sub>MC</sub> relative to FliG<sub>M</sub> is distinct from that seen in the wild-type *T. maritima*

FliG<sub>MC</sub> structure [36, 37]. However, it remains unclear how such a change in the orientation of helix<sub>MC</sub> relative to FliG<sub>M</sub> affects the switching process.

The flagellar motor containing a single stator unit rotates with 26 steps per revolution, which is in agreement with the stoichiometry of the FliG ring [53, 54]. The elementary process of torque generation by stator-rotor interactions is symmetric in CCW and CW rotation [54], although the torque-speed curves are distinct between them [55]. To achieve the symmetric elementary process of torque generation in both CCW and CW rotation, FliG<sub>C</sub> may need to go through a 180° rotation to reorient its charged residues in the two opposite directions. However, it remains unknown how it occurs.

FliM-GFP (or YPet) and GFP-FliN show turnovers between the motor and their cytoplasmic pools [56, 57], indicating that the C ring is a highly dynamic structure rather than a static one. The turnover of FliM occurs only in the presence of CheY-P, suggesting that this turnover is directly involved in the switching process. The number of FliM subunits in the C ring increases in response to a reduction in the concentration of CheY-P, suggesting that the flagellar motor adapts to changes in the steady-state level of CheY-P by adjusting the number of FliM molecules to which CheY-P binds [58, 59].

## 3. Stator

### 3.1. Proton Channel of the MotA/B Complex

The energy for flagellar motor rotation is supplied by the electrochemical potential of specific ions across the cytoplasmic membrane [60–63]. Based on the coupling ions, the flagellar motors are divided into at least two types (Figure 3): one is the H<sup>+</sup>-driven type found in *E. coli* and *Salmonella* [18, 21, 52, 65, 66] and the other is the Na<sup>+</sup>-driven type in marine *Vibrio* and extremely alkalophilic *Bacillus* [67–69]. Recently, it has been reported that extremely alkalophilic *Bacillus alcalophilus* Vedder 1934 utilizes both Na<sup>+</sup> and K<sup>+</sup> as the coupling ions for flagellar rotation [64]. The stator of the H<sup>+</sup>-driven motor consists of four copies of MotA and two copies of MotB [70] and acts as a proton channel to couple proton flow through the channel with torque generation (Figure 4) [18]. Terahara et al. [71] have reported that the MotA/B complex of alkaliphilic *Bacillus clausii* can conduct both H<sup>+</sup> and Na<sup>+</sup> at different pH ranges. The Na<sup>+</sup>-coupled stator complex of *Vibrio alginolyticus* consists of four proteins, PomA, PomB, MotX and MotY [72]. PomA and PomB form a Na<sup>+</sup>-conducting pathway within the cytoplasmic membrane in a way similar to the MotA/B proton channel [73]. MotX and MotY form the T ring located beneath the P ring for efficient assembly and stable anchoring of the PomA/B complex to its binding site on the motor [74]. Interestingly, both H<sup>+</sup>-coupled and Na<sup>+</sup>-coupled stators are incorporated into a single flagellar motor in *Bacillus subtilis* and *Shewanella oneidensis* MR-1 [75, 76]. Please see a review article summarizing the current understanding of the Na<sup>+</sup>-driven flagellar motor in *Vibrio alginolyticus* [72].

MotA consists of four transmembrane spans (TMs), two short periplasmic loops and two extensive cytoplasmic regions. MotB consists of an N-terminal cytoplasmic region, one TM and MotB<sub>c</sub> containing a well-conserved PGB motif. Asp-33 of *Salmonella* MotB, which corresponds to Asp-32 of *E. coli* MotB, is a highly conserved aspartic residue [66, 77]. This aspartic acid residue is located at the proximal end of MotB-TM and is critical for proton translocation through the channel [52, 65, 66]. MotB exists as a homo-dimer in the stator and these aspartic acid residues are positioned on the surface

of the MotB-TM dimer facing MotA-TMs [78]. These observations suggest that the MotA/B complex has two proton-conducting pathways (Figure 4). MotA-TM3 and MotA-TM4 are close to MotB-TM while MotA-TM1 and MotA-TM2 are in the outer positions of the MotA/B complex [79]. This suggests that MotB-TM forms a proton channel along with the TM-3 and TM-4 helices of MotA (Figure 4). The V35F mutation, which lies in MotA-TM2, has been isolated as an extragenic suppressor of the motB(D33E) mutant [80]. It is impossible for the side chain of Val-35 in MotA-TM2 to directly access the proton-conducting pathway formed at the interface between MotA and MotB because it is likely to be located within the hydrophobic core of the four transmembrane α-helix bundle of MotA. This suggests that TM2 is presumably close to TM3 and TM4 to maintain the proton pathway. This is supported by site-directed disulfide cross-linking experiments [81].

Figure 3. Schematic diagrams of the bacterial H⁺-driven and Na⁺-driven flagellar motors. *E. coli* and *Salmonella* have the H⁺-driven flagellar motor (left) whereas *Vibrio alginolyticus* has the Na⁺-driven polar flagellar motor (right). The stator of the flagellar motor consists of MotA and MotB in the H⁺-driven motor and of PomA and PomB in the Na⁺-driven motor. MotX and MotY are required for efficient assembly of the PomA/B complex around the motor in *Vibrio*.

![Diagram](attachment:diagram.png)

Figure 4. Arrangement of transmembrane segments of the MotA/B complex, which consists of four copies of MotA and two copies of MotB. The view is from the periplasmic side of the membrane. The complex has two proton conducting pathway shown by orange ellipsoids [79–81].

![Complex Diagram](attachment:complex_diagram.png)

Pro-173 of MotA, which is highly conserved among MotA orthologues, is located at the proximal end of MotA-TM3 (Figure 4) [82,83]. Site-directed disulfide cross-linking experiments have shown that Pro-173 of *E. coli* MotA is in relative close proximity to Asp-32 of *E. coli* MotB [81]. Co-overproduction of MotA with the MotB-TetA chimera results in proton leakage, arresting cell growth [18]. The *E. coli* MotB(D32N) mutation suppresses the proton leakage through the MotA/B-TetA proton channel. Interestingly, introduction of the MotA(P173D) mutation into the MotA/B(D32N)-TetA complex causes massive proton flow again, largely impairing the cell growth. This indicates that this Asp-173 residue of MotA allows protons to go through the channel in the absence of Asp-32 of MotB, probably by presenting a carboxyl residue as an alternative proton-binding site [83]. These observations suggest that Pro-173 of MotA forms a part of the proton-conducting pathway along with Asp-32 of MotB. The *Salmonella* MotA(P173A) mutant motor produces torque at the wild-type level when the motor operates under a very high load condition [84]. However, the MotA(P173A) mutation causes ca. 90% reduction in the maximum rotation rate, suggesting that the P173A mutation significantly reduces the rate of the mechanochemical reaction cycle of the motor [84]. Simulation of the torque-speed curve of the MotA(P173A) mutant motor by a simple kinetic model indicates that the P173A mutation slows the rates of conformational changes of the MotA/B complex by 25 times [84]. Because a prolyl residue induces bending in TM to facilitate ion transport [85], Pro-173 of MotA is proposed to facilitate the conformational changes of the MotA/B complex that support the rapid mechanocmemical cycle when the motor rotates at high speed [81,84].

Over-expression of the MotA/B complex from a plasmid does not affect cell growth at all. In contrast, in-frame deletion of a plug segment formed by residues 53 to 66 in MotB causes massive proton flow through a proton channel, thereby arresting cell growth [21,86]. This suggests that the plug segment interferes with proton channel formation, thereby suppressing undesirable proton flow through the channel when the MotA/B complex is not installed into the motor. Interestingly, however, the cell growth is not significantly impaired by in-frame deletion of residues 51 to 100 in *Salmonella* MotB, which contains the plug segment although MotB(Δ51–100) is partially functional [87,88]. This indicates that proton channel formation of the MotA/B(Δ51–100) complex does not occur prior to its assembly around the motor although the plug is missing. The L119P and L119E substitutions allow unassembled MotA/B(Δ51–100) complex to conduct protons to a significant degree, impairing the cell growth [21,88]. This suggests that not only the plug segment but also some other region within MotB<sub>C</sub> regulate proper proton channel formation.

### 3.2. Role of MotB<sub>C</sub> in Stator Assembly around a Rotor

MotB<sub>C</sub> forms a homo-dimer and its dimerization is required for MotB function [88,89]. MotB<sub>C</sub> is required not only for proper anchoring of the stator to the PG layer [90] but also for proper alignment of the stator relative to the rotor [91]. The crystal structure of the core domain of MotB<sub>C</sub>, which consists of residues 99 to 276 in MotB, shows considerable structural similarities with the PGB domain of Pal (Figure 5a) [88]. Interestingly, a chimeric MotB-Pal protein, in which the PGB domain of *E. coli* MotB is replaced by that of *E. coli* Pal, exerts MotB function to a considerable degree, suggesting that MotB<sub>C</sub> binds to the PG layer through its PGB motif in a way similar to Pal [92]. Site-directed disulfide crosslinking experiments have shown that MotB<sub>C</sub> is in relatively close proximity to the P ring of the basal body [93].

Biomolecules 2014, 4

Figure 5. The crystal structure of MotB<sub>C</sub> from *Salmonella*. (a) Ca ribbon representation of *Salmonella* MotB<sub>C</sub> dimer in the crystal (PDB ID: 2ZVY). The peptidoglycan binding (PGB) motif is shown by orange. (b) The MotB<sub>C</sub> structure in the vicinity of Leu119. Leu119 in helix α1 shows hydrophobic interactions with Leu149 in strand β2 and Val183 and Ile187 after helix α3.

Upon induction of the MotA/B complex from a plasmid in a *motAB* null mutant, a stepwise increment in speed is observed at high load, estimating that there are as many as 11 stators around the rotor [96]. Interestingly, abrupt, stepwise drops and restorations of the rotation speed of the motor are also observed even in steadily rotating motors, indicating that the stators are replaced frequently [14, 15, 53]. Consistently, GFP-fused MotB shows rapid exchanges between the motor and the cytoplasmic membrane pool, indicating that an interaction between MotB<sub>C</sub> and the PG layer is highly dynamic [97]. These observations suggest that the MotA/B complexes are no permanently fixed in place around the motor during motor rotation.

Figure 6. Torque-speed curves of the wild-type (black line) and MotB(Δ72–100) mutant motors (red line). Torque-speed curves of the wild-type motor with different number of stator units are indicated (*N*<sub>1</sub>–*N*<sub>11</sub>).

The torque-speed curve of the flagellar motor consists of two regimes: a high-load, low-speed regime and a low-load, high-speed regime (Figure 6). In the high-load, low-speed regime, the speed of the motor is proportional to the number of stators, whereas in the low-load, high-speed regime, one stator unit can spin the motor at the maximum speed [94, 95]. Recently, it has been reported that a few stators spin the motor at low loads and that 6–11 stators drive the same motor at high loads. This suggests that the flagellar motor controls the number of functional stators in the motor in response to

changes in external load [98, 99]. In contrast to the torque-speed curve of the wild-type motor with a gradual decrease in the high-load regime and a rapid drop in the low-load regime, in-frame deletion of 72 to 100 in MotB causes an unusual torque-speed curve of the motor with a rapid drop in the high-load regime and a slow decrease in the low-load regime (Figure 6) [100]. At a low speed near stall, the MotB(Δ72–100) mutant motor produces torque at the wild-type level, indicating that the deletion does not affect an actual torque-generation step by stator-rotor interactions coupled with proton translocation through the proton channel. In contrast, as motor speed increases, torque produced by the MotB(Δ72–100) mutant motor drops off at much higher load than that of the wild-type motor due to a rapid decrease in the number of stators driving the motor at much higher load than that the wild-type motor responds to dissociate its stator from the motor [100]. As torque generation by the motor applies an equal and opposite force on the PG layer through MotB<sub>C</sub>, MotB<sub>C</sub> is proposed to act as a load-sensitive structural switch to regulate the assembly and disassembly cycle of the stators in response to the load changes and that an appropriate length of a linker connecting the PGB domain to MotB-TM may stabilize this structural switch during the torque generation cycle [100].

### 3.3. Load-Sensitive Coupling between Proton Translocation and Torque Generation

Both protonation and deprotonation of MotB-Asp33 induce conformational changes of the cytoplasmic loop between TM2 and TM3 of MotA (MotA<sub>c</sub>), which is directly involved in the interaction with FliG. This suggests that proton translocation through the proton channel is coupled with cyclic conformational changes of the MotA/B complex for torque generation [101]. The MotB(D33E) mutation in MotB causes a slow motile phenotype [52]. This mutation results in about 50% reduction in near-stall torque. Interestingly, the suppressor mutations, which exist in MotA-TM2, MotA-TM3, MotB-TM and MotB<sub>C</sub>, restore stall torque to the wild-type level [80]. The D33E mutation does not decrease the number of stators around the rotor, indicating that the decrease in stall torque is not due to the reduction in the number of stators driving the motor [102]. Since the rotation rate of the flagellar motor is not limited by the rate of the mechanochemical cycle of the motor at low speed near stall [103], it is suggested that the D33E mutation misaligns MotA<sub>c</sub> relative to FliG at the rotor-stator interface, causing the 50% reduction in the energy coupling efficiency of the motor and that the suppressor mutations readjusts the alignment of MotA<sub>c</sub> relative to FliG, thereby restoring the coupling efficiency to the wild-type level [102].

The MotB(D33E) mutation causes a 90% reduction in the maximum rotation speed of the flagellar motor at a very low load, indicating that this mutation results in a considerable reduction in the maximum rate of the mechanochemical cycle of the motor. The suppressor mutations do not support the maximal rotation rate at low load [80]. Because the proton conductivity of the unplugged proton channels of the MotA/B(D33E) and its suppressor MotA(V35F)/B(D33E) complexes are measured to be ca. 50% and ca. 60% of that through unplugged wild-type channel, the decreased proton conductivity through the channel itself is not the main cause of the marked reduction in the maximal rotation speed at low load [102]. The rate of proton flow through the MotA/B proton channel is limited by the rate of conformational change of the MotA/B complex, suggesting that the 90% reduction in the maximal rotation speed of these mutant motors is due to a considerable decrease in rates of the conformational changes of the MotA/B complex coupled with proton translocation through the channel [102].

The stator has a high duty ratio and hence remains attached to the rotor for most of the mechanochemical reaction cycle [94, 95]. The rotation speed of the wild-type motor is very stable over a wide range of rotation rates [95]. However, the rotation speeds of the MotA/B(D33E) and MotA(V35F)/B(D33E) motors show large fluctuations at low load although they are stable at high load. Interestingly, frequent pauses are also observed for these two mutant motors at low load [102]. These observations suggest that the wild-type MotA/B complex can sense even a small load to continue the mechanochemical reaction cycle for torque generation by efficiently coupling proton translocation with its conformational changes. Because the stators are dynamic mechanosensors and change their structure to control the number of stators driving the motor in response to changes in external load [98, 99], it is proposed that the MotA/B complex is a load-sensitive proton channel that efficiently couples proton translocation with torque generation in response to changes in the load and that MotB-Asp33 is important for this coordinated proton translocation [102].

## 4. Role of Stator-Rotor Interactions in Efficient Assembly of the Stators into the Motor

MotA-Arg90 and MotA-Glu98, which are located in MotA<sub>C</sub>, are highly conserved residues among MotA orthologues and are critical for motor function [104]. The extragenic FliG(D289A) and FliG(D289K) suppressor mutations partially rescue the motility of the *motA(R90E)* mutant whereas the extragenic FliG(R281V) and FliG(R281W) suppressor mutations restore the motility of the *motA(E98K)* mutant, indicating genetic interactions between MotA-Arg90 and FliG-Asp289 and between MotA-Glu98 and FliG-Arg281 (Figure 7) [22]. This suggests that these interactions at the rotor-stator interface are directly involved in flagellar rotation. The MotA(R90E), MotA(E98K), FliG(D289K) and FliG(R281V) mutations considerably affect the assembly of GFP-labeled MotA/B complexes around the rotor, suggesting that these interactions are also critical for stator assembly around the motor [105, 106]. The FliG(D289K) mutation significantly recovers the localization of GFP-MotB to the motor in the *motA(R90E)* mutant whereas the FliG(R281V) mutation does not recover the GFP-MotB localization in the *motA(E98K)* mutant. These suggest that the MotA-Arg90–FliG-Asp289 interaction is critical for the proper positioning of the stators around the rotor, whereas the MotA-Glu98–FliG-Arg281 interaction is more important for torque generation [106].

Salmonella MotB missing residues 51 to 100 retains the ability to form the functional stator with MotA to a considerable degree [87]. The distance between the surface of the hydrophobic core layer of the cytoplasmic membrane and that of the PG layer is about 100 Å. The crystal structure of a Salmonella MotB<sub>C</sub> fragment corresponding to residues 99 to 276 of MotB is only about 50 Å tall [88] and hence it cannot reach the PG layer if connected directly to MotB-TM by the deletion of residues 51–100. Therefore, a large conformational change must be required for the PGB sites on the top surface of MotB<sub>C</sub> to reach the PG layer. Leu119 in helix α1 shows hydrophobic interactions with Leu149 in strand β2 and Val183 and Ile187 after helix α3 (Figure 5b) [88]. The L119P and L119E mutations, which not only cause massive proton flow through the proton channel of the MotA/B(Δ51–100) complex but also increase the assembly efficiency of the MotA/B(Δ51–100) complex into the motor, would destabilize these hydrophobic interactions [21, 88]. This raises a plausible hypothesis that these hydrophobic interactions act as a structural switch to regulate the assembly and disassembly of the stators [88, 100]. Over-expression of MotA inhibits wild-type motility due to a reduction in the number

of the functional stator around the rotor [105]. Consistently, MotA-mCherry co-localizes with GFP-FliG even in the absence of MotB [105]. These observations indicate that MotA alone can be installed into the motor. Because conserved charged residues at the rotor–stator interface are directly involved in stator assembly around the motor [105, 106], interactions between MotA and FliG may trigger a large conformational change in MotB<sub>C</sub> that open the proton channel and allow the stator to be anchored to the PG layer.

Figure 7. Schematic diagram of interactions at the rotor–stator interface. FliG is postulated to occupy the upper part of the C ring. The interactions between MotA-Arg90 and FliG-Asp289 and between MotA-Glu98 and FliG-Arg281 are directly involved not only in torque generation but also in stator assembly around the rotor.

![Diagram](attachment:diagram.png)

5. Conclusions

Both the rotor and stator of the flagellar motor are highly dynamic structures rather than static ones. [56–58, 97–99] Two rotor proteins, FliM and FliN, show highly dynamic turnovers between the motor and their cytoplasmic pools. The FliM turnover plays an important role in switching the direction of flagellar motor rotation [56, 58, 59]. However, little is known about the switching mechanism of the rotor ring complex. The stators are machanosensors to regulate the number of the stators driving the motor in response to changes in external load [98–100]. Each stator is also a load-sensitive proton channel that efficiently couples proton translocation with torque generation in response to the load changes [102]. However, it remains unknown how the stators sense the amount of external load to control the number of functional stators around the rotor as well as the proton flow through the channel. Interactions at the rotor–stator interface are important, not only for torque generation [22], but also for efficient stator assembly around the rotor [105, 106]. However, it remains unknown how the rotor–stator interactions allow the stator to open the proton channel and to be anchored to the PG layer. We need to investigate the switching mechanism of the flagellar motor in

much more detail by biophysical and structural techniques combined with genetic and biochemical approaches. Super-resolution fluorescence microscopic techniques would be required for understanding of the mechanosensing mechanism of the stator. Furthermore, not only in vitro reconstitution experiments but also high-resolution structural analysis by X-ray crystallography and electron cryomicroscopy would be essential to advance our mechanistic understanding of the energy coupling mechanism of the flagellar motor.

# Acknowledgments

We acknowledge Keiichi Namba and Masahiro Ueda for continuous support and encouragement. Y.V.M. has been supported by the Special Postdoctoral Researchers program of RIKEN. Our research is supported in part by the Japanese Ministry of Education, Culture, Sports, Science and Technology.

# Conflict of Interest

The authors declare no conflict of interest.

# References

1. Berg, H.C. The rotary motor of bacterial flagella. *Ann. Rev. Biochem.* 2003, 72, 19–54.
2. Macnab, R.M. How bacteria assemble flagella. *Annu. Rev. Microbiol.* 2003, 57, 77–100.
3. Minamino, T.; Imada, K.; Namba, K. Molecular motors of the bacterial flagella. *Curr. Opin. Struct. Biol.* 2008, 18, 693–701.
4. Blair, D.F. Flagellar movement driven by proton translocation. *FEBS Lett.* 2003, 545, 86–95.
5. Sowa, Y.; Berry, R.M. Bacterial flagellar motor. *Q. Rev. Biophys.* 2008, 41, 103–132.
6. Macnab, R.M.; Ornston, M.K. Normal-to-curly flagellar transitions and their role in bacterial tumbling. Stabilization of an alternative quaternary structure by mechanical force. *J. Mol. Biol.* 1977, 112, 1–30.
7. Turner, L.; Ryu, W.S.; Berg, H.C. Real-time imaging of fluorescent flagellar filaments. *J. Bacteriol.* 2000, 182, 2793–2801.
8. Sourjik, V.; Armitage, J.P. Spatial organization in bacterial chemotaxis. *EMBO J.* 2010, 29, 2724–2733.
9. Ueno, T.; Oosawa, K.; Aizawa, S. M ring, S ring and proximal rod of the flagellar basal body of *Salmonella typhimurium* are composed of subunits of a single protein, FliF. *J. Mol. Biol.* 1992, 227, 672–677.
10. Suzuki, H.; Yonekura, K.; Namba, K. Structure of the rotor of the bacterial flagellar motor revealed by electron cryomicroscopy and single-particle image analysis. *J. Mol. Biol.* 2004, 337, 105–113.
11. Khan, I.H.; Reese, T.S.; Khan, S. The cytoplasmic component of the bacterial flagellar motor. *Proc. Natl. Acad. Sci. USA* 1992, 89, 5956–5960.
12. Francis, N.R.; Sosinsky, G.E.; Thomas, D.; DeRosier, D.J. Isolation, characterization, and structure of bacterial flagellar motors containing the switch complex. *J. Mol. Biol.* 1994, 235, 1261–1270.

13. Yamaguchi, S.; Aizawa, S.; Kihara, M.; Isomura, M.; Jones, C.J.; Macnab, R.M. Genetic evidence for a switching and energytransducing complex in the flagellar motor of *Salmonella typhimurium*. *J. Bacteriol.* 1986, **168**, 1172–1179.

14. Block, S.M.; Berg, H.C. Successive incorporation of force generating units in the bacterial rotary motor. *Nature* 1984, **309**, 470–472.

15. Blair, D.F.; Berg, H.C. Restoration of torque in defective flagellar motors. *Science* 1988, **242**, 1678–1681.

16. Wilson, M.L.; Macnab, R.M. Co-overproduction and localization of the *Escherichia coli* motility proteins MotA and MotB. *J. Bacteriol.* 1990, **172**, 3932–3939.

17. De Mot, R.; Vanderleyden, J. The C-terminal sequence conservation between OmpA-related outer membrane proteins and MotB suggests a common function in both gram-positive and gram-negative bacteria, possibly in the interaction of these domains with peptidoglycan. *Mol. Microbiol.* 1994, **12**, 333–334.

18. Blair, D.F.; Berg, H.C. The MotA protein of *E. coli* is a proton-conducting component of the flagellar motor. *Cell* 1990, **60**, 439–449.

19. Stolz, B.; Berg, H.C. Evidence for interactions between MotA and MotB, torque-generating elements of the flagellar motor of *Escherichia coli*. *J. Bacteriol.* 1991, **173**, 7033–7037.

20. Tang, H.; Braun, T.F.; Blair, D.F. Motility protein complexes in the bacterial flagellar motor. *J. Mol. Biol.* 1996, **261**, 209–221.

21. Morimoto, Y.V.; Che, Y-S.; Minamino, T.; Namba, K. Proton-conductivity assay of plugged and unplugged MotA/B proton channel by cytoplasmic pHluorin expressed in *Salmonella*. *FEBS lett.* 2010, **584**, 1268–1272.

22. Zhou, J.; Lloyd, S.A.; Blair, D.F. Electrostatic interactions between rotor and stator in the bacterial flagellar motor. *Proc. Natl. Acad. Sci. USA* 1998, **95**, 6436–6441.

23. Francis, N.R.; Irikura, V.M.; Yamaguchi, S.; DeRosier, D.J.; Macnab, R.M. Localization of the *Salmonella typhimurium* flagellar switch protein FliG to the cytoplasmic M-ring face of the basal body. *Proc. Natl. Acad. Sci. USA* 1992, **89**, 6304–6308.

24. Oosawa, K.; Ueno, T.; Aizawa, S. Overproduction of the bacterial flagellar switch proteins and their interactions with the MS ring complex *in vitro*. *J. Bacteriol.* 1994, **176**, 3683–3691.

25. Lee, L.K.; Ginsburg, M.A.; Crovace, C.; Donohoe, M.; Stock, D. Structure of the torque ring of the flagellar motor and the molecular basis for rotational switching. *Nature* 2010, **466**, 996–1000.

26. Kihara, M.; Miller, G.U.; Macnab, R.M. Deletion analysis of the flagellar switch protein FliG of *Salmonella*. *J. Bacteriol.* 2000, **182**, 3022–3028.

27. Levenson, R.; Zhou, H.; Dahlquist, F.W. Structural insights into the interaction between the bacterial flagellar motor proteins FliF and FliG. *Biochemistry* 2012, **51**, 5052–5060.

28. Brown, P.N.; Terrazas, M.; Paul, K.; Blair, D.F. Mutational analysis of the flagellar protein FliG: sites of interaction with FliM and implications for organization of the switch complex. *J. Bacteriol.* 2007, **189**, 305–312.

29. Paul, K.; Gonzalez-Bonet, G.; Bilwes, A.M.; Crane, B.R.; Blair, D. Architecture of the flagellar rotor. *EMBO J.* 2011, **30**, 2962–2971.

30. Vartanian, A.S.; Paz, A.; Fortgang, E.A.; Abramson, J.; Dahlquist, F.W. Structure of flagellar motor proteins in complex allows for insights into motor structure and switching. *J. Biol. Chem.* 2012, 287, 35779–35783.

31. Lam, K.H.; Lam, W.W.; Wong, J.Y.; Chan, L.C.; Kotaka, M.; Ling, T.K.; Jin, D.Y. Ottemann, K.M.; Au, S.W. Structural basis of FliG-FliM interaction in *Helicobacter pylori*. *Mol Microbiol.* 2013, 88, 798–812.

32. Brown, P.N.; Mathews, M.A.; Joss, L.A.; Hill, C.P.; Blair, D.F. Crystal structure of the flagellar rotor protein FliN from *Thermotoga maritima*. *J. Bacteriol.* 2005, 187, 2890–2902.

33. Thomas, D.R.; Morgan, D.G.; DeRosier, D.J. Rotational symmetry of the C ring and a mechanism for the flagellar rotary motor. *Proc. Natl. Acad. Sci. USA* 1999, 96, 10134–10139.

34. Thomas, D.R.; Francis, N.R.; Xu, C.; DeRosier, D.J. The three-dimensional structure of the flagellar rotor from a clockwise-locked mutant of *Salmonella enterica* serovar Typhimurium. *J. Bacteriol.* 2006, 188, 7039–7048.

35. Lloyd, S.A.; Whitby, F.G.; Blair, D.F.; Hill, C.P. Structure of the C-terminal domain of FliG, a component of the rotor in the bacterial flagellar motor. *Nature* 1999, 400, 472–475.

36. Brown, P.N.; Hill, C.P.; Blair, D.F. Crystal structure of the middle and C-terminal domains of the flagellar rotor protein FliG. *EMBO J.* 2002, 21, 3225–3234.

37. Minamino, T.; Imada, K.; Kinoshita, M.; Nakamura, S.; Morimoto, Y.V.; Namba, K. Structural insight into the rotational switching mechanism of the bacterial flagellar motor. *PLoS Biol.* 2011, 9, e1000616.

38. Park, S.Y.; Lowder, B.; Bilwes, A.M.; Blair, D.F.; Crane, B.R. Structure of FliM provides insight into assembly of the switch complex in the bacterial flagella motor. *Proc. Natl. Acad. Sci. USA* 2006, 103, 11886–11891.

39. Sircar, R.; Greenswag, A.R.; Bilwes, A.M.; Gonzalez-Bonet, G.; Crane, B.R. Structure and activity of the flagellar rotor protein FliY: a member of the CheC phosphatase family. *J. Biol. Chem.* 2013, 288, 13493–13502.

40. Stock, D.; Namba, K.; Lee, L.K. Nanorotors and self-assembling macromolecular machines: the torque ring of the bacterial flagellar motor. *Curr. Opin. Biotechnol.* 2012, 23, 545–554

41. Dyer, C.M.; Vartanian, A.S.; Zhou, H.; Dahlquist, F.W. A molecular mechanism of bacterial flagellar motor switching. *J. Mol. Biol.* 2009, 388, 71–84.

42. Sarkar, M.K.; Paul, K.; Blair, D. Chemotaxis signaling protein CheY binds to the rotor protein FliN to control the direction of flagellar rotation in *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* 2010, 107, 9370–9375.

43. Cluzel, P.; Surette, M.; Leibler, S. An ultrasensitive bacterial motor revealed by monitoring signaling proteins in single cells. *Science* 2000, 287, 1652–1655.

44. Duke, T.A.; Le Novere, N.; Bray, D. Conformational spread in a ring of proteins: a stochastic approach to allostery. *J. Mol. Biol.* 2001, 308, 541–553.

45. Bai, F.; Branch, R.W.; Nicolau, D.V. Jr.; Pilizota, T.; Steel, B.C.; Maini, P.K.; Berry, R.M. Conformational spread as a mechanism for cooperativity in the bacterial flagellar switch. *Science* 2010, 327, 685–689.

46. Yuan, J.; Fahrner, K.A.; Berg, H.C. Switching of the bacterial flagellar motor near zero load. *J. Mol. Biol.* 2009, 390, 394–400.

47. Bai, F.; Minamino, T.; Wu, Z.; Namba, K.; Xing, J. Coupling between switching regulation and torque generation in bacterial flagellar motor. *Phys. Rev. Lett.* 2012, **108**, 178105.

48. Boehm, A.; Kaiser, M.; Li, H.; Spangler, C.; Kasper, C.A.; Ackermann, M.; Kaever, V.; Sourjik, V.; Roth, V.; Jenal, U. Second messenger-mediated adjustment of bacterial swimming velocity. *Cell* 2010, **141**, 1–10.

49. Paul, K.; Nieto, V.; Carlquist, W.C.; Blair, D.F.; Harshey, R.M. (2010) The c-di-GMP binding protein YcgR controls flagellar motor direction and speed to affect chemotaxis by a "backstop brake" mechanism. *Mol. Cell.* 2010, **38**, 128–139.

50. Fang, X.; Gomelsky, M. A post-translational, c-di-GMP-dependent mechanism regulating flagellar motility. *Mol. Microbiol.* 2010, **76**, 1295–1305.

51. Van Way, S.M.; Millas, S.G.; Lee, A.H.; Manson, M.D. Rusty, jammed, and well-oiled hinges: Mutations affecting the interdomain region of FliG, a rotor element of the *Escherichia coli* flagellar motor. *J. Bacteriol.* 2004, **186**, 3173–3181.

52. Togashi, F.; Yamaguchi, S.; Kihara, M.; Aizawa, S.-I.; Macnab, R.M. An extreme clockwise switch bias mutation in *fliG* of *Salmonella typhimurium* and its suppression by slow-motile mutations in *motA* and *motB*. *J. Bacteriol.* 1997, **179**, 2994–3003.

53. Sowa, Y.; Rowe, A.D.; Leake, M.C.; Yakushi, T.; Homma, M.; Ishijima, A.; Berry, R.M. Direct observation of steps in rotation of the bacterial flagellar motor. *Nature* 2005, **437**, 916–919.

54. Nakamura, S.; Kami-ike, N.; Yokota, J.P.; Minamino, T.; Namba, K. Evidence for symmetry in the elementary process of bidirectional torque generation by the bacterial flagellar motor. *Proc. Natl. Acad. Sci. USA* 2010, **107**, 17616–17620.

55. Yuan, J.; Fahrner, K.A.; Turner, L.; Berg, H.C. Asymmetry in the clockwise and counterclockwise rotation of the bacterial flagellar motor. *Proc. Natl. Acad. Sci. USA* 2010, **107**, 12846–12849.

56. Delalez, N.J.; Wadhams, G.H.; Rosser, G.; Xue, Q.; Brown, M.T.; Dobbie, I.M.; Berry, R.M.; Leake, M.C.; Armitage, J.P. Signal-dependent turnover of the bacterial flagellar switch protein FliM. *Proc. Natl. Acad. Sci. USA* 2010, **107**, 11347–11351.

57. Fukuoka, H.; Inoue, Y.; Terasawa, S.; Takahashi, H.; Ishijima, A. Exchange of rotor components in functioning bacterial flagellar motor. *Biochem. Biophys. Res. Commun.* 2010, **394**, 130–135.

58. Yuan, J.; Branch, R.W.; Hosu, B.G.; Berg, H.C. Adaptation at the output of the chemotaxis signalling pathway. *Nature* 2012 **484**, 233–236.

59. Yuan, J.; Berg, H.C. Ultrasensitivity of an adaptive bacterial motor. *J. Mol. Biol.* 2013, **425**, 1760–1764.

60. Larsen, S.H.; Adler, J.; Gargus, J.J.; Hogg, R.W. Chemomechanical coupling without ATP: the source of energy for motility and chemotaxis in bacteria. *Proc. Natl. Acad. Sci. USA* 1974, **71**, 1239–1243.

61. Manson, M.D.; Tedesco, P.; Berg, H.C.; Harold, F.M.; Van der Drift, C. A protonmotive force drives bacterial flagella. *Proc. Natl. Acad. Sci. USA* 1977, **4**, 3060–3064.

62. Matsuura, S.; Shioi, J.; Imae, Y. Motility in *Bacillus subtilis* driven by an artificial protonmotive force. *FEBS Lett.* 1977, **82**, 187–190

63. Ravid, S.; Eisenbach, M. Minimal requirements for rotation of bacterial flagella. *J. Bacteriol.* 1984, **158**, 1208–1210.

64. Terahara, N.; Sano, M.; Ito, M. A *Bacillus* flagellar motor that can use both Na<sup>+</sup> and K<sup>+</sup> as a coupling ion is converted by a single mutation to use only Na<sup>+</sup>. *PLoS One* 2012, **7**, e46248.

65. Sharp, L.L.; Zhou, J.; Blair, D.F. Tryptophan scanning mutagenesis of MotB, an integral membrane protein essential for flagellar rotation in *Escherichia coli*. *Biochemistry* **1995**, *34*, 9166–9171.

66. Zhou, J.; Sharp, L.L.; Tang, H.L.; Lloyd, S.A.; Billings, S.; Braun, T.F.; Blair, D.F. Function of protonatable residues in the flagellar motor of *Escherichia coli*: A critical role for Asp 32 of MotB. *J. Bacteriol.* **1998**, *180*, 2729–2735.

67. Hirota, N.; Kitada, M.; Imae, Y. Flagellar motors of alkalophilic *Bacillus* are powered by an electrochemical potential gradient of Na<sup>+</sup>. *FEBS Lett.* **1981**, *132*, 278–280.

68. Hirota, N.; Imae, Y. Na<sup>+</sup>-driven flagellar motors of an alkalophilic *Bacillus* strain Yn-1. *J. Biol. Chem.* **1983**, *258*, 10577–10581.

69. Chernyak, B.V.; Dibrov, P.A.; Glagolev, A.N.; Sherman, M.Y.; Skulachev, V.P. A novel type of energetics in a marine alkali-tolerant bacterium: Δμ-Na-driven motility and sodium cycle. *FEBS Lett.* **1983**, *164*, 38–42

70. Kojima, S.; Blair, D.F. Solubilization and purification of the MotA/MotB complex of *Escherichia coli*. *Biochemistry* **2004**, *43*, 26–34.

71. Terahara, N.; Krulwich, T.A.; Ito, M. Mutations alter the sodium versus proton use of a *Bacillus clausii* flagellar motor and confer dual ion use on *Bacillus subtilis* motors. *Proc. Natl. Acad. Sci. USA* **2008**, *105*, 14359–14364.

72. Li, N.; Kojima, S.; Homma, M. Sodium-driven motor of the polar flagellum in marine bacteria *Vibrio*. *Genes Cells.* **2011** *16*, 985–999.

73. Sato, K.; Homma, M. Functional reconstitution of the Na<sup>+</sup>-driven polar flagellar motor component of *Vibrio alginolyticus*. *J. Biol. Chem.* **2000**, *275*, 5718–5722.

74. Terashima, H.; Fukuoka, H.; Yakushi, T.; Kojima, S.; Homma, M. The *Vibrio* motor proteins, MotX and MotY, are associated with the basal body of Na<sup>+</sup>-driven flagella and required for stator formation. *Mol. Microbiol.* **2006**, *62*, 1170–1180.

75. Ito, M.; Hicks, D.B.; Henkin, T.M.; Guffanti, A.A.; Powers, B.D.; Zvi, L.; Uematsu, K.; Krulwich, T.A. MotPS is the stator-force generator for motility of alkaliphilic *Bacillus* and its homologue is a second functional Mot in *Bacillus subtilis*. *Mol. Microbiol.* **2004**, *53*, 1035–1049

76. Paulick, A.; Koerdt, A.; Lassak, J.; Huntley, S.; Wilms, I.; Narberhaus, F.; Thormann, K.M. Two different stator systems drive a single polar flagellum in *Shewanella oneidensis* MR-1. *Mol. Microbiol.* **2009**, *71*, 836–850.

77. Hosking, E.R.; Manson, M.D. Clusters of charged residues at the C terminus of MotA and N terminus of MotB are important for function of the *Escherichia coli* flagellar motor. *J. Bacteriol.* **2008**, *190*, 5517–5521.

78. Braun, T.; Blair, D.F. Targeted disulfide cross-linking of the MotB protein of *Escherichia coli*: evidence for two H<sup>+</sup> channels in the stator complex. *Biochemistry* **2001**, *40*, 13051–13059.

79. Braun, T.F.; Al-Mawasawi, L.Q.; Kojima, S.; Blair, D.F. Arrangement of core membrane segments in the MotA/MotB protein-channel complex of *Escherichia coli*. *Biochemistry* **2004**, *43*, 35–45.

80. Che, Y.S.; Nakamura, S.; Kojima, S.; Kami-ike, N.; Namba, K.; Minamino, T. Suppressor analysis of the MotB(D33E) mutation to probe the bacterial flagellar motor dynamics coupled with proton translocation. *J. Bacteriol.* **2008**, *190*, 6660–6667.

81. Kim, E.A.; Price-Carter, M.; Carlquist, W.C.; Blair, D.F. Membrane segment organization in the stator complex of the flagellar motor: implications for proton flow and proton-induced conformational change. *Biochemistry* 2008, 47, 11332–11339.
82. Blair, D.F.; Berg, H.C. Mutations in the MotA protein of Escherichia coli reveal domains critical for proton conduction. *J. Mol. Biol.* 1991, 221, 1433–1442.
83. Braun, T.; Poulson, S.; Gully, J.B.; Empey, J.C.; Van Way, S.; Putnam, A.; Blair, D.F. Function of proline residues of MotA in torque generation by the flagellar motor of Escherichia coli. *J. Bacteriol.* 1999, 181, 3542–3551.
84. Nakamura, S.; Morimoto, Y.V.; Kami-ike, N.; Minamino, T.; Namba, K. Role of a conserved prolyl residue (Pro-173) of MotA in the mechanochemical reaction cycle of the proton-driven flagellar motor of Salmonella. *J. Mol. Biol.* 2009, 393, 300–307.
85. Brandl, C.J.; Deber, C.M. Hypothesis about the function of membrane-buried proline residues in transport proteins. *Proc. Natl. Acad. Sci. USA* 1986, 83, 917–921.
86. Hosking, E.R.; Vogt, C.; Bakker, E.P.; Manson, M.D. The Escherichia coli MotAB proton channel unplugged. *J. Mol. Biol.* 2006, 364, 921–937.
87. Muramoto, K.; Macnab, R.M. Deletion analysis of MotA and MotB, components of the force-generating unit in the flagellar motor of Salmonella. *Mol. Microbiol.* 1998, 29, 1191–202.
88. Kojima, S.; Imada, K.; Sakuma, M.; Sudo, Y.; Kojima, C.; Minamino, T.; Homma, M.; Namba, K. Stator assembly and activation mechanism of the flagellar motor by the periplasmic region of MotB. *Mol. Microbiol.* 2009, 73, 710–718.
89. Kojima, S.; Furukawa, Y.; Matsunami, H.; Minamino, T.; Namba, K. Characterization of the periplasmic domain of MotB and implications for its role in the stator assembly of the bacterial flagellar motor. *J. Bacteriol.* 2008, 190, 2259–2266.
90. Blair, D.F.; Kim, D.Y.; Berg, H.C. Mutant MotB proteins in Escherichia coli. *J. Bacteriol.* 1991, 173, 4049–4055.
91. Garza, A.G.; Biran, R.; Wohlschlegel, J.; Manson, M.D. Mutations in *motB* suppressible by changes in stator or rotor components of the bacterial flagellar motor. *J. Mol. Biol.* 1996, 25, 270–285.
92. Hizukuri, Y.; Morton, J.F.; Yakushi, T.; Kojima, S.; Homma, M. The peptidoglycan-binding (PGB) domain of the Escherichia coli pal protein can also function as the PGB domain in *E. coli* flagellar motor protein MotB. *J. Biochem.* 2009, 146, 219–229.
93. Hizukuri, Y.; Kojima, S.; Homma, M. Disulfide cross-linking between the stator and the bearing components in the bacterial flagellar motor. *J. Biochem.* 2010, 148, 309–318.
94. Ryu, W.S.; Berry, R.M.; Berg, H.C. Torque-generating units of the flagellar motor of Escherichia coli have a high duty ratio. *Nature* 2000, 403, 444–447.
95. Yuan, J.; Berg, H.C. Resurrection of the flagellar rotary motor near zero load. *Proc. Natl Acad. Sci. USA* 2008, 105, 1182–1185.
96. Reid, S.W.; Leake, M.C.; Chandler, J.H.; Lo, C.J.; Armitage, J.P.; Berry, R.M. The maximum number of torque-generating units in the flagellar motor of Escherichia coli is at least 11. *Proc. Natl. Acad. Sci. USA* 2006, 103, 8066–8071.
97. Leake, M.C.; Chandler, J.H.; Wadhams, G.H.; Bai, F.; Berry, R.M.; Armitage, J.P. Stoichiometry and turnover in single, functioning membrane protein complexes. *Nature* 2006, 443, 355–358.

98. Lele, P. P.; Hosu, B. G.; Berg, H. C. Dynamics of mechanosensing in the bacterial flagellar motor. *Proc. Natl. Acad. Sci. USA* **2013**, *110*, 11839–11844.

99. Tipping, M.J.; Delalez, N.J.; Lim, R.; Berry, R.M.; Armitage, J.P. Load-dependent assembly of the bacterial flagellar motor. *MBio* **2013**, *4*, e00551-13.

100. Castillo, D.J.; Nakamura, S.; Morimoto, Y.V.; Che, Y.-S.; Kamiike, N.; Kudo, S.; Minamino, T.; Namba, K. The C-terminal periplasmic domain of MotB is responsible for load-dependent control of the number of stators of the bacterial flagellar motor. *Biophysics* **2013**, *9*, 173–181

101. Kojima, S.; Blair, D.F. Conformational change in the stator of the bacterial flagellar motor. *Biochemistry* **2001**, *40*, 13041–13050.

102. Che, Y.S.; Nakamura, S.; Morimoto, Y.V.; Kami-ike, N.; Namba, K.; Minamino, T. Load-sensitive coupling of proton translocation and torque generation in the bacterial flagellar motor. *Mol. Microbiol.* **2014**, *91*, 175–184.

103. Chen, X.; Berg, H.C. Solvent-isotope and pH effects on flagellar rotation in *Escherichia coli*. *Biophys. J.* **2000**, *78*, 2280–2284.

104. Zhou, J.; Blair, D.F. Residues of the cytoplasmic domain of MotA essential for torque generation in the bacterial flagellar motor. *J. Mol. Biol.* **1997**, *273*, 428–439.

105. Morimoto, Y.V.; Nakamura, S.; Kami-ike, N.; Namba, K.; Minamino, T. Charged residues in the cytoplasmic loop of MotA are required for stator assembly into the bacterial flagellar motor. *Mol. Microbiol.* **2010**, *78*, 1117–1129.

106. Morimoto, Y.V.; Nakamura, S.; Hiraoka, K.D.; Namba, K.; Minamino, T. Distinct roles of highly conserved charged residues at the MotA-FliG interface in bacterial flagellar motor rotation. *J. Bacteriol.* **2013**, *195*, 474–481.

Copyright of Biomolecules (2218-273X) is the property of MDPI Publishing and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
