
ARTICLE

Received 24 Feb 2014 | Accepted 24 Jul 2014 | Published 9 Sep 2014 | DOI: 10.1038/ncomms5805

# Insights into mitochondrial fatty acid synthesis from the structure of heterotetrameric 3-ketoacyl-ACP reductase/3R-hydroxyacyl-CoA dehydrogenase

Rajaram Venkatesan¹, Shiv K. Sah-Teli¹, Luqman O. Awoniyi¹, Guangyu Jiang¹, Piotr Prus¹, Alexander J. Kastaniotis¹, J. Kalervo Hiltunen¹², Rik K. Wierenga¹ & Zhijun Chen²

Mitochondrial fatty acid synthesis (mtFAS) is essential for respiratory growth in yeast and mammalian embryonic survival. The human 3-ketoacyl-acyl carrier protein (ACP) reductase (KAR) of mtFAS is a heterotetrameric α₂β₂-assembly composed of 17β-hydroxysteroid dehydrogenase type-8 (HSD17B8, α-subunit) and carbonyl reductase type-4 (CBR4, β-subunit). Here we provide a structural explanation for the stability of the heterotetramer from the crystal structure with NAD⁺ and NADP⁺ bound to the HSD17B8 and CBR4 subunits, respectively, and show that the catalytic activity of the NADPH- and ACP-dependent CBR4 subunit is crucial for a functional HsKAR. Therefore, mtFAS is NADPH- and ACP dependent, employing the 3R-hydroxyacyl-ACP intermediate. HSD17B8 assists in the formation of the competent HsKAR assembly. The intrinsic NAD⁺- and CoA-dependent activity of the HSD17B8 subunit on the 3R-hydroxyacyl-CoA intermediates may indicate a role for this subunit in routing 3R-hydroxyacyl-CoA esters, potentially arising from the metabolism of unsaturated fatty acids, into the mitochondrial β-oxidation pathway.

---

¹Faculty of Biochemistry and Molecular Medicine and Biocenter Oulu, University of Oulu, PO Box 5400, Oulu FI-90014, Finland. ²State Key Laboratory of Supramolecular Structure and Materials, Jilin University, Changchun 130012, China. Correspondence and requests for materials should be addressed to R.V. (email: rajaram.venkatesan@oulu.fi) or to Z.C. (email: zchen@jlu.edu.cn).

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.

M itochondria play a crucial role in cellular activities of all eukaryotes, including energy conversion¹, lipid metabolism², apoptosis³ and innate immune responses⁴. A new pathway, mitochondrial fatty acid synthesis (mtFAS), has recently gained attention since its physiological significance was recognized⁵. mtFAS is similar to the bacterial FAS-II system, where each of the four catalytic steps are catalysed by distinct enzymes (Fig. 1), while the cytosolic FAS in eukaryotes is carried out by the FAS-I multifunctional enzyme complex. mtFAS provides the precursor of mitochondrially synthesized α-lipoic acid, a key cofactor for oxidative decarboxylation of α-keto acids and glycine within eukaryotic cells⁵. There is evidence for other physiologically relevant products of mtFAS in yeast⁶. Deficiency of mtFAS leads to respiratory chain defects and mitochondrial dysfunction⁶⁻⁸.

3-Ketoacyl-acyl carrier protein (ACP) reductase (KAR) catalyses the second step of the mtFAS pathway (Fig. 1). Although all the other mtFAS enzymes identified thus far are encoded by single genes, human KAR (HsKAR) is a heterotetrameric α₂β₂ enzyme formed by two subunits, 17β-hydroxysteroid dehydrogenase type 8 (HSD17B8 or KE6, α-subunit) and carbonyl reductase type 4 (CBR4 or SDR45C1, β-subunit)⁹. HsHSD17B8 is expressed in abundance in the prostate, placenta and kidney, and has been shown to catalyse the reversible oxidation/reduction of steroid molecules *in vitro*¹⁰. The physiological roles of HSD17B8 and CBR4 have remained elusive, and information is only available from studies of the respective homotetramers. The two subunits of HsKAR belong to the short-chain dehydrogenase/reductase (SDR) superfamily¹¹ and share 35% sequence identity. However, the orthologues of HsKAR (fabG or OarI) studied thus far are homotetrameric. A crystal structure of the HsHSD17B8 homotetramer is available in the protein data bank (PDB id: 2PD6). HsCBR4, the other component of HsKAR, is much less studied and there is no structural information. A previous report shows that CBR4 has quinone reductase activity *in vitro*¹². HsCBR4 displays the highest expression in neuronal and muscle tissues (http://cgap.nci.nih.gov/). It is important to note that when these two proteins are co-expressed in Escherichia coli,

they exclusively form α₂β₂ tetramers rather than forming homotetramers⁹.

The heterotetrameric structure triggers the questions why two different subunits are required and what are the roles of the HSD17B8 and the CBR4 subunits in the assembly, substrate recognition and catalysis. The crystal structure of HsKAR shows that the HSD17B8 and CBR4 subunits complement each other, making a much more stable quaternary structure than the respective homotetramers. Through crystallographic binding studies, structure-guided site-directed mutagenesis, enzyme assays, surface plasmon resonance (SPR) binding studies and *in vivo* complementation experiments in yeast, we demonstrate that the CBR4 subunit is the functional unit of HsKAR in mtFAS responsible for NADPH and acyl-ACP binding and catalysis. Thus, this work has identified the biological enzyme function of CBR4. The HSD17B8 subunit works as a scaffold protein, facilitating a competent and functional HsKAR assembly. In addition, we speculate that this subunit may act as an auxiliary mitochondrial β-oxidation enzyme oxidizing the 3R-hydroxyacyl-CoA metabolites arising from the metabolism of unsaturated fatty acids.

### Results

#### Crystal structure of the heterotetrameric HsKAR.
Our previous research indicated that HsKAR is composed of two components, HsHSD17B8 and HsCBR4. The crystals of HsKAR, cocrystallized with NAD⁺ (referred to as HsKAR-NAD⁺), belong to space group P2₁ and diffract up to a resolution of 2.85 Å (Table 1). The crystals contain four tetramers in the asymmetric unit and exhibit

| Table 1 | Data collection and refinement statistics of HsKAR. |
| --- | --- |
| **Data collection** | **HsKAR-NAD⁺** | **HsKAR-NAD(P)⁺** |
| Space group | P2₁ | P2₁ |
| Cell dimensions |  |  |
| $a, b, c (\text{Å})$ | 87.15, 237.11, 87.35 | 87.50 238.41 87.69 |
| $\alpha, \beta, \gamma (^{\circ})$ | 90.00, 94.52, 90.00 | 90.00, 94.18, 90.00 |
| Resolution (Å) | 49.78–2.85 (3.0–2.85)† | 49.25–2.34 (2.48–2.34) |
| $R_{\text{sym}} (\%)$ | 16.7 (68.6) | 13.5 (91.1) |
| $<I>/<\sigma I>$ | 6.6 (2.1) | 5.5 (1.0) |
| Completeness (%) | 91.9 (95.1) | 94.9 (91.1) |
| Redundancy | 2.9 (3.1) | 6.6 (1.8) |
| **Refinement** |  |  |
| Resolution (Å) | 2.85 | 2.34 |
| No. reflections | 75310 | 148285 |
| $R_{\text{work}} / R_{\text{free}} (\%)$ | 18.2/23.6 | 20.3/23.6 |
| No. atoms |  |  |
| Protein | 26275 | 27792 |
| Ligand | 352 | 758 |
| Water | 218 | 806 |
| **B-factors (Å²)** |  |  |
| Protein | 56.8 | 53.1 |
| Ligand | 62.9 | 58.7 |
| Water | 33.9 | 43.7 |
| **Root mean square deviations** |  |  |
| Bond lengths (Å) | 0.002 | 0.002 |
| Bond angles ($^{\circ}$) | 0.5 | 0.7 |
| *Statistics for each data set collected from a single crystal.* |  |  |
| † Values in parentheses are for the highest resolution shell. |  |  |

Figure 1 | Scheme of the human mtFAS pathway showing the mechanism of fatty acyl chain elongation by two carbon units per cycle. The four reactions correspond to KAS (3-ketoacyl-thioester synthase), KAR (3-ketoacyl-thioester reductase), HTD (3-hydroxyacyl-thioester dehydratase) and ETR (2-enoyl-thioester reductase). In our present studies, we show that the KAR activity of the mtFAS pathway is NADPH-dependent as also observed for ETR.

pseudomerohedral twinning, with the twin law l, -k, h and twin fraction of ~0.3. The HsKAR crystal complexed with both NAD⁺ and NADP⁺ (referred to as HsKAR-NAD(P)⁺) was isomorphous to the HsKAR-NAD⁺ crystals and diffracted up to a resolution of 2.34 Å (Table 1). The pseudomerohedral twinning was almost non-existent in this crystal (0.1).

The tertiary structures of the HSD17B8 and CBR4 subunits are well conserved¹³⁻²⁰ (Fig. 2a). The general structure of a CBR4 or HSD17B8 monomer features a central parallel β-sheet composed of seven β-strands (β1–β7) sandwiched between helices α1, α2 and α6 on one side and helices α3, α4 and α5 on the other side. Helices α′ and α″ are above the strands β6 and β7 (Fig. 2a).

In the HsKAR-NAD⁺ structure, the electron density is well defined for the two HSD17B8 subunits (chains A and D) with breaks mainly in the loop region connecting α2 and β3 (residues 56–67), which is slightly longer (Supplementary Fig. 1) compared with the previously studied KARs and HsCBR4. The electron densities for the CBR4 subunits (chains B and C) are less well defined. Despite slightly better definition of chain B, density for residues 184–194, corresponding to two helices (α′ and α″), above the proposed cofactor binding site, is missing. In chain C, residues 39 to 51 comprising helix α2 and part of strand β3 are poorly defined. In chains B and C, many side-chain densities of the surface residues are poor. In the HsKAR-NAD(P)⁺ structure, the overall electron density is of high quality, also for the CBR4 subunits. In two CBR4 subunits (chains F and J), continuous electron density is present for the entire chain. Owing to its superior diffraction quality and the presence of both the cofactors, the HsKAR-NAD(P)⁺ structure has been used for the further analysis and comparisons.

In each tetramer, there are two subunits each of HSD17B8 (for example, chains A and D) and CBR4 (chains B and C) (Fig. 2b). Therefore, the HsKAR heterotetramer is essentially a dimer of HSD17B8 and CBR4 dimers (Fig. 2b) with approximate 222 (D2) symmetry, resembling the previously studied homotetrameric KARs. Although the overall structures of the two molecules of HSD17B8 and CBR4 in a tetramer are very similar, there are subtle differences between these chains such that each chain can be uniquely identified in each of the four tetramers of the asymmetric unit. Two subunits of HSD17B8 (or CBR4) interact with each other through helices α4 and α5, forming a four-helix bundle (Fig. 2b). At the interaction surface between HSD17B8 and CBR4, the corresponding β7 β-strands from each subunit associate in an anti-parallel manner interacting through water-mediated H-bonds, forming an extended continuous β-sheet (Fig. 2b). Two helices (α6) from both subunits are also packed against each other. The structure of homotetrameric HsHSD17B8 (PDB id: 2PD6) is known. However, when HsHSD17B8 and HsCBR4 are co-expressed, they always form heterotetramers. Analysis of the interfaces by the web-based PDBePISA²¹ (Protein interfaces, surfaces and assemblies’ service at the European Bioinformatics Institute: http://www.ebi.ac.uk/pdbe/prot_int/pistart.html) shows that the homodimeric interfaces (for example, the AD and BC interfaces) in HsHSD17B8 and HsKAR-NAD(P)⁺ structures are comparable (~1,600 Å²). However, each heterodimeric interface in HsKAR (for example, the AB and DC interface) is ~200 Å² larger than the corresponding interface in the homotetrameric HsHSD17B8 (Fig. 2b and Supplementary Table 1). Moreover, the diagonally opposite protomers in the HsKAR tetramer (AC and DB interface) also have an interface of ~200 Å² each, whereas there is no interaction between the corresponding protomers in the HsHSD17B8 homotetramer (Supplementary Fig. 2). Together, these additional interfaces amount to ~800 Å², leading to a stabilization of the heterotetramer by ~8 kcal mol⁻¹ according to the PISA calculations. In addition, a hypothetical

homotetramer of HsCBR4 shows many steric clashes between the carboxy terminus of the two diagonally opposite protomers (that is, AC and DB interface). Therefore, the heterotetrameric assembly of HsHSD17B8 and HsCBR4 as seen in our crystal structure of HsKAR is much more favourable than the corresponding homotetramers.

Cofactor recognition and the catalytic triad in HsKAR. The crystals of HsKAR could only be obtained when cocrystallized with 2 mM NAD⁺. In the crystal structure, NAD⁺ is bound only to the HSD17B8 subunits and the CBR4 subunits are in the apo form. Several attempts to crystallize HsKAR in the apo form as well as in the presence of other ligands such as coenzyme A (CoA), NADP⁺, NADH and NADPH were not successful. Therefore, we soaked the crystals of HsKAR-NAD⁺ with a mixture of 5 mM each of NAD⁺ and NADP⁺ in the crystallization solution. The resulting structure showed the selective binding of NAD⁺ to the HSD17B8 subunits and of NADP⁺ to the CBR4 subunits. The 2′-phosphate moiety was unambiguously visible in the initial omit maps (Fig. 2c), demonstrating the cofactor preference of the two subunits. The conformation of NAD⁺ and NADP⁺ in the respective binding sites are similar to that of NADP⁺ in Brassica napus (B. napus) and E. coli fabG¹³,¹⁵. Both NAD⁺ and NADP⁺ are stabilized by many conserved hydrogen bonding interactions in their respective binding sites (Fig. 2c and Supplementary Table 2). The differences are mainly due to the presence of the 2′-phosphate moiety of NADP⁺. In HSD17B8, the side chain of αAsp42 (residues of HSD17B8 and CBR4 subunits are represented with the prefix α and β, respectively) has two key hydrogen bonding interactions with the two oxygen atoms (O2B, 2.5 Å and O3B, 2.7 Å) of the ribose moiety of the adenosine part of NAD⁺ (Fig. 2c and Supplementary Table 2). This is a conserved interaction in many NAD⁺ binding enzymes²². The αAsp42 is replaced by βAla33 in the NADP(H)-specific HsCBR4 providing enough space for the additional 2′-phosphate moiety of NADP⁺. This phosphate group is stabilized in its position by hydrogen bonding interactions from βArg12 (3.5 Å) and βArg34 (2.7 Å) (Fig. 2c). Therefore, αAsp42, βArg12 and βArg34 are crucial for the cofactor recognition and differentiation. In an HsKAR tetramer, NAD⁺ is bound equally well in both the HSD17B8 subunits, whereas the nicotinamide part of NADP⁺ is disordered in one of the two CBR4 subunits (chain C, for example).

Structural and sequence comparisons suggest that in HsHSD17B8 αSer156, αTyr169 and αLys173, and in HsCBR4 βSer135, βTyr148 and βLys152 are the catalytic triad residues involved in catalysis (Fig. 3a)¹³,²³. The side chains of these residues can be clearly identified and are positioned near the nicotinamide moiety (Fig. 2c). αTyr169 and αLys173 of HSD17B8 and the corresponding residues (βTyr148 and βLys152) of CBR4 have hydrogen bonding interactions with the ribose oxygen (O2D, 3.2 Å and 2.7 Å) of the nicotinamide part of NAD(P)⁺. The catalytic serine (αSer156 and βSer135) is proposed to provide an oxyanion hole for the thioester oxygen of the substrate²⁴. Interestingly, this serine in many of the subunits shows electron density extended beyond its side chain. The shape of this density is varied. For example, in chain D, the αSer156 is modelled as O-acetylserine (Oas), whereas in chain M an acetate ion and a water molecule can be modelled unambiguously (Supplementary Fig. 3). In chains H and P, the density is ambiguous although an acetate ion is modelled there. All these chains represent HSD17B8 subunits. In other chains, a water molecule is sufficient to account for the observed extra density. Only in chain O (CBR4 subunit), there is still some spurious positive density, which could not be modelled. The cause of the different behaviour of this serine in

ARTICLE

NATURE COMMUNICATIONS | DOI: 10.1038/ncomms5805

a

Chain A  
(HSD17B8)  

Chain B  
(CBR4)  

Chain D  
(HSD17B8)  

Chain C  
(CBR4)  

Chain A  
(HSD17B8)  

Chain B  
(CBR4)  

Chain D  
(HSD17B8)  

Chain C  
(CBR4)  

C

αAsp42  
βGly9  
αGly18  
βArg12  
αSer21  
NAD(P)+  
αLys173  
βLys152  
αTyr169  
βTyr148  
αSer156  
βSer135  

αAsp42  
βGly9  
αGly18  
βArg12  
αSer21  
NAD(P)+  
αLys173  
βLys152  
αTyr169  
βTyr148  
αSer156  
βSer135  

4

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.
different subunits of HsKAR is unclear. Subtle crystal packing effects could be one of the reasons. The modelled acetate ion is near the nicotinamide ring of NAD⁺ and to some extent identifies the fatty-acyl chain binding pocket. The sodium acetate buffer and the ammonium acetate used in the crystallization condition can be the source of these acetate ions. Acetylated serine is also observed in a few other crystal structures, which are either acetyltransferases or esterases where ligands such as acetyl-CoA or acetylcholine are used²⁵,²⁶. There is one protein structure of acetylxylan esterase (PDB id: 3FYU, Krastanova *et al.*, unpublished) crystallized in the presence of 0.2M sodium acetate and xylan containing acetylated catalytic²⁷ serines. Similar to that in HsKAR, the serine is acetylated only in some subunits.

Enzymological studies of HsKAR and its active site variants. A number of residues near the catalytic sites were targeted for mutagenesis in both HSD17B8 and CBR4 (Fig. 2c, and Supplementary Tables 3 and 4). Large-scale purification was attempted for all of these mutants. However, only purified samples of αD42A-HsKAR, αY169A-HsKAR, αK173A-HsKAR, βR34A-HsKAR and βK152A-HsKAR could be prepared following a protocol similar to that of the HsKAR purification. The reductase enzyme activities of HsKAR and mutated derivatives thereof were tested *in vitro* with acetoacetyl-CoA and 9,10-phenanthrene quinone (9,10-PQ) in the presence of NAD(P)H. The calculated specific activities and relative activities of these variants with respect to HsKAR are listed in Table 2. Overall, the specific activity was higher for HsKAR in the presence of NADH

---

**Figure 3 | Sequence alignment of KARs and ACPs.**

**(a)** Part of the sequence alignment of HsHSD17B8 and HsCBR4 of HsKAR with KAR orthologues. The secondary structure elements as calculated by DSSP and residue numbers are displayed above for HsHSD17B8 and below for HsCBR4. The conserved catalytic triad residues, Ser-Tyr-Lys are highlighted in orange. The arginine residues close to Arg129 and Arg172 in *E. coli* fabG essential for ACP binding are highlighted in yellow. The extended loop in *S. cerevisiae* Oar1 near Arg129 of *E. coli* fabG is shown in magenta. Other positively charged residues near this region are highlighted in cyan. The full sequence alignment can be found in the Supplementary Fig. 1.

**(b)** Part of the sequence alignment of ACPs from the same organisms as in Fig. 3a. The secondary structure elements above and below the sequences correspond to ACP from *E. coli* and *M. tuberculosis*, respectively. It can be seen that the helix α2, considered to be essential for the interaction with KAR, is quite well conserved in these organisms, in particular with respect to the two negatively charged residues (Asp76 and Glu82, *E. coli* ACP numbering). The full sequence alignment is provided in the Supplementary Fig. 5.

---

**Figure 2 | Structure of HsKAR.**

**(a)** Cartoon stereo diagram of the monomeric structure of HsKAR subunits HSD17B8 (α) (brown) and CBR4 (β) (blue). The secondary structure elements are labelled. The N- and C-termini of the polypeptides are marked.

**(b)** The tetrameric organization of HsKAR in stereo. The two HSD17B8 subunits are represented in brown (chain A) and pink (chain D) cartoon. The two CBR4 subunits are shown in blue (chain B) and yellow (chain C) cartoon. NAD⁺ bound in the two HSD17B8 subunits is represented in cyan ball and stick model (in chain A and D). The two NADP⁺ molecules as bound in the two CBR4 subunits are highlighted in magenta (in chain B and C) ball and stick model. The secondary structure elements are labelled in chain A and selected secondary structure elements are labelled in chain B.

**(c)** Ball and stick model of NAD⁺ (cyan) and NADP⁺ (magenta) as bound to the HSD17B8 and CBR4 subunits, respectively. The HSD17B8 and CBR4 subunits are superimposed to highlight the similar mode of cofactor binding. The residues chosen for mutagenesis on HSD17B8 (brown) and CBR4 (blue) near the cofactor binding site are shown as a ball and stick model. The important interactions are highlighted by black dashes and the distances are shown in angstrom. Superimposed is also the (2F~o~-F~c~) electron density map of the NADP⁺ contoured at 1σ.
ARTICLE

Table 2 | Specific activity of HsKAR and its mutated variants.

|  | Acetoacetyl-CoA | 9,10-Phenanthrene quinone |
| --- | --- | --- |
|  | Specific activity (μmol mg⁻¹ min⁻¹) | Relative activity (%) | Specific activity (μmol mg⁻¹ min⁻¹) | Relative activity (%) |
| NADH-dependent activity |  |  |  |  |
| HsKAR | 20.5 ± 4.7 | 100 | 20.8 ± 5.8 | 100 |
| αD42A-HsKAR | 7.3 ± 2.5 | 36 | 4.6 ± 2.2 | 22 |
| αY169A-HsKAR | 0.65 ± 0.18 | 3 | 0.85 ± 0.5 | 4 |
| αK173A-HsKAR | 0.06 ± 0.05 | 0.3 | 1.2 ± 0.6 | 6 |
| βR34A-HsKAR | 21 ± 2.3 | 102 | 20.7 ± 3.4 | 100 |
| βK152A-HsKAR | 22 ± 6 | 109 | 25.5 ± 3.7 | 123 |
| βR168E-HsKAR | 21.7 ± 2.2 | 106 | 7.9 ± 3.0 | 38 |
| βK169E-HsKAR | 19.3 ± 2.2 | 95 | 15.7 ± 1.3 | 76 |
| βQ126E/K169E-HsKAR | 31.8 ± 2.8 | 156 | 17.4 ± 1.1 | 84 |
| βQ126E/R168E/K169E-HsKAR | 14.8 ± 1 | 72 | 8.4 ± 3.5 | 40 |
| NADPH-dependent activity |  |  |  |  |
| HsKAR | 0.7 ± 0.25 | 3.5 (100)* | 2.6 ± 1.8 | 12 (100)* |
| αD42A-HsKAR | 4 ± 1.2 | 20 (566) | 5.4 ± 1.5 | 26 (211) |
| αY169A-HsKAR | 0.4 ± 0.1 | 2 (58) | 3.1 ± 2.1 | 15 (120) |
| αK173A-HsKAR | 0.5 ± 2 | 2.4 (70) | 4.5 ± 2.4 | 22 (178) |
| βR34A-HsKAR | 0.09 ± 0.03 | 0.44 (13) | 0.44 ± 0.3 | 2.1 (17) |
| βK152A-HsKAR | Not detectable | Not detectable | 0.46 ± 0.25 | 2.2 (18) |
| βR168E-HsKAR | 1.3 ± 0.5 | 6.3 (181) | 1.6 ± 0.3 | 8 (61) |
| βK169E-HsKAR | 1.1 ± 0.22 | 5.5 (158) | 1.3 ± 0.4 | 6 (49) |
| βQ126E/K169E-HsKAR | 0.88 ± 0.13 | 4.3 (124) | 1.7 ± 0.4 | 8 (65) |
| βQ126E/R168E/K169E-HsKAR | 0.1 ± 0.01 | 0.5 (14) | Not detectable | Not detectable |

*Values in parentheses correspond to the relative activity with respect to the HsKARNADPH-dependent activity.

NATURE COMMUNICATIONS | DOI: 10.1038/ncomms5805

ARTICLE

![Image](#)

Figure 4 | Binding studies with HsKAR and αD42A-HsKAR. Relative SPR responses for the binding of (a) NADPH, (b) NADH, (c) NADP⁺ and (d) NAD⁺ to HsKAR are shown in dots (●) and to αD42A-HsKAR are shown in squares (■). Each experiment was repeated at least three times and the error bars indicate the s.d. The concentration on the x axis is plotted on a logarithmic scale to visualize the two site binding observed for NADPH and NADH. It is evident that the high-affinity binding site is not present for NADH with αD42A-HsKAR. The curves fitted by nonlinear regression using two-site or one-site specific binding module of the Graphpad Prism software are shown in black. The figure was prepared using the Graphpad Prism software. The binding affinities are listed in Supplementary Table 5.

NAD(H) and two NADP(H) binding sites in HsKAR. Assuming that all of the immobilized protein is active, the theoretically calculated binding level in response units (RUs) for the binding of one ligand molecule to the HsKAR tetramer is ~60 RU. This is in agreement with the experimentally observed RU for the stronger binding affinity (53 ± 5 RU for NADPH and 53 ± 9 RU for NADH). Therefore, the number of binding sites for either NADH or NADPH with micromolar affinity is one per HsKAR tetramer. It is possible that NADH (or NADPH) binds to one of the two molecules of HSD17B8 (or CBR4) with micromolar affinity and that this binding reduces the affinity of the second binding site to millimolar levels, which would be a case of negative cooperativity. However, in our crystal structure we do not observe significant structural variation between the two binding sites. The main difference is that the nicotinamide part of NADP⁺ is disordered in one of the CBR4 subunits of each tetramer. Negative cooperativity of cofactor binding has been observed in *E. coli*¹⁴ and *Plasmodium falciparum* (*P. falciparum*)²⁸ fabGs in particular in the presence of ACP. Therefore, the mechanism of cooperativity in HsKAR could differ from that observed in *E. coli*¹⁴ and *P. falciparum* FabG²⁸. At this point, we do not know how ACP affects the cofactor binding for HsKAR. Moreover, positive cooperativity has been observed for *Mycobacterium tuberculosis* (*M. tuberculosis*)²⁹ and *Staphylococcus aureus* fabGs²⁰ even in the absence of ACP. We also determined the binding affinities for the αD42A-HsKAR mutant. For this variant, the NAD⁺ binding affinity decreased by ~15-fold with a *K*~D~ value of 70.8 mM. The high-affinity binding site of NADH was also abolished in this mutant exhibiting single-site binding with a *K*~D~ value of 15 mM. However, the association with NADPH or NADP⁺ was not affected (Supplementary Table 5), demonstrating the importance of αAsp42 to differentiate between NAD(H) and NADP(H) binding¹⁴,²⁰.

In vivo functional analysis. To evaluate the physiological significance of the two subunits of HsKAR, we set up a yeast complementation assay taking advantage of the dependence of yeast cells on a functional mitochondrial KAR for respiratory competence. Wild-type yeast cells can grow very well on either glucose or glycerol as the sole carbon source, while cells deficient of the *OAR1*, the gene encoding the native yeast mitochondrial KAR, survive only on fermentable media. To rescue the growth of *oar1Δ* cells on glycerol-based plates, the *OAR1* (or *fabG* or *HsKAR*) gene has to be transformed into the cells to complement the deficiency. Therefore, it is possible to use yeast as an in vivo system to evaluate the effects of the HsKAR mutant variants on respiratory growth. Intriguingly, these studies show that mutations at the NAD(H) binding and catalytic sites of the HSD17B8 of HsKAR did not affect the growth of the cells on synthetic complete (SC)-glycerol plates. However, NADP(H)-binding site and catalytic site mutant variants of CBR4 (residues βGly9, βSer135, βTyr148 and βLys152; Supplementary Table 4) are unable to support respiratory growth (Fig. 5a). This shows that CBR4 is the catalytic subunit carrying the KAR activity that is physiologically relevant in vivo for the complementation of the yeast mtFAS defect.

In vivo ACP recognition by HsKAR. For mtFAS, the substrates are fatty acyl conjugates of ACP, similar to that for bacterial FAS-II. ACP is highly conserved in amino acid sequence, especially, the helix α2, which is at the ACP-enzyme interaction site³⁰. The identity of amino acid sequences of the helix α2 between *E. coli*, *Saccharomyces cerevisiae* (*S. cerevisiae*) and human is 70% (Fig. 3b and Supplementary Fig. 5). In *E. coli* fabG, two arginine residues, Arg129 and Arg172, were found to play an important role in ACP recognition and binding³⁰. In enzyme

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.

ARTICLE

NATURE COMMUNICATIONS | DOI: 10.1038/ncomms5805

a
SCD SCG
wild type
oar1Δ
oar1Δ+oar1
oar1Δ+HsKAR
oar1Δ+αG18S-HsKAR
oar1Δ+αS21A-HsKAR
oar1Δ+αD42A-HsKAR
oar1Δ+αS156A-HsKAR
oar1Δ+αY169A-HsKAR
oar1Δ+αK173A-HsKAR
oar1Δ+βG9S-HsKAR
oar1Δ+βR12A-HsKAR
oar1Δ+βR34A-HsKAR
oar1Δ+βS135A-HsKAR
oar1Δ+βY148A-HsKAR
oar1Δ+βK152A-HsKAR

b
SCD SCG
oar1Δ+fabG
oar1Δ+R129E-fabG
oar1Δ+R172E-fabG
oar1Δ+R129E/R172E-fabG
oar1Δ+αR148E-HsKAR
oar1Δ+αR189E-HsKAR
oar1Δ+αR148E-HsKAR
oar1Δ+αR189A-HsKAR
oar1Δ+αR148E/R189E-HsKAR
oar1Δ+αR148A/R189A-HsKAR
oar1Δ+βQ126E-HsKAR
oar1Δ+βR168E-HsKAR
oar1Δ+βK169E-HsKAR
oar1Δ+βR168E/K169E-HsKAR
oar1Δ+βQ126E/R168E-HsKAR
oar1Δ+βQ126E/K169E-HsKAR
oar1Δ+βQ126E/R168E/K169E-HsKAR
oar1Δ+βQ124-126E/R168E/K169E-HsKAR

Figure 5 | Spotting assays for wild-type and OAR1-deficient yeast in SC media with glucose (D) or glycerol (G) as the sole carbon source.

(a) Transformation with plasmids harbouring OAR1 or HsKAR genes can rescue the growth of yeast in glycerol medium. These assays show that the mutations in the active site of HsHSD17B8 (α) subunit do not affect the complementation of HsKAR (top panels), whereas the mutations in the HsCBR4 (β) subunit destroy the yeast complementation properties of HsKAR (bottom panels). This reveals the importance of the NADPH-dependent CBR4 subunit in 'in vivo' KAR activity. (b) Similar spotting assays for variants of E. coli fabG (top panels), HSD17B8 variants (middle panels) and CBR4 variants (bottom panels) of the residues proposed to be involved in ACP recognition. These assays reveal the physiological significance of Arg172 of the E. coli fabG in the recognition of ACP (top panels). Growth of HSD17B8 variants are not affected, indicating they are not involved in ACP recognition (middle panels). The key residue for the ACP recognition is βLys169 of CBR4, as all the variants containing this mutation cannot complement the oar1 deletion in yeast (bottom panels).

activity assays, E. coli fabG variants fabG-R129E and fabG-R172E showed 8.5% and 3.5% of ACP-dependent activity, respectively, while CoA-dependent activities were largely unaffected³⁰. These arginines are well conserved in most of the fabG-type enzymes. However, in M. tuberculosis, yeast and HsKAR, they are not strictly maintained. In HsHSD17B8, there are two arginine residues Arg148 and Arg189 that are not exactly in the same positions as Arg129 and Arg172 of E. coli fabG, but in the neighbouring sites (Fig. 3a). In CBR4, βGln126 and βLys169 are located in the position corresponding to Arg129 and Arg172 of E. coli fabG, respectively. In CBR4, there is also an arginine (βArg168) in a position adjacent to that of Arg172 of E. coli fabG (Fig. 3a). These residue changes could affect ACP binding in subtle ways. In both HSD17B8 and CBR4, these residues are on the surface pointing into bulk solvent and are far away from the catalytic site of the same subunit. A superposition of the HsKAR structure and the ternary complex of M. tuberculosis fabG4 with bound NAD⁺ and hexanoyl-CoA³¹ showed the probable fatty- acyl binding pocket in both HSD17B8 and CBR4. Figure 6 illustrates the relative positions of the proposed ACP-recognition residues of HSD17B8 (chain A) and CBR4 (chain C) with respect to the neighbouring active site of CBR4 (chain B). This arrangement suggests two possibilities for ACP binding: the ACP interacts with HsKAR either (i) through the residues on the surface of CBR4 (chain C) and extend the fatty acyl chain to the active site of neighbouring CBR4 subunit (chain B) in a similar manner as in Fig. 6 or (ii) through the surface residues of HSD17B8 (chain A) and extend the fatty acyl chain through another groove to the CBR4 active site (chain B). To experimentally test these predictions, we created a number of single and multiple mutated variants of HsKAR (Supplementary Tables 3 and 4).

The arginine residues Arg129 and Arg172 in E. coli fabG have been demonstrated to be involved in ACP binding only by in vitro tests³⁰. Therefore, the variants fabG-R129E and fabG-R172E of E. coli fabG were re-investigated using our yeast in vivo system (Fig. 5b). These experiments confirmed that Arg172 is very likely to play a critical role in the fabG-ACP recognition in vivo as the fabG-R172E variant showed very poor growth on SC glycerol media. In contrast, the fabG-R129E variant did not severely affect the growth of the yeast cells. The corresponding HsKAR single and multiple mutant variants were also evaluated using the same system. The mutations of the HSD17B8 subunit did not impact on the growth of the yeast cells (Fig. 5b). However, the CBR4 βK169E mutation affected growth, suggesting that βLys169 of CBR4 in HsKAR is critical for ACP recognition. Single, double and triple mutants carrying this variant (βK169E-, βQ126E/ βK169E- and βQ126E/R168E/K169E- HsKAR, respectively) and additional single mutant (βR168E-HsKAR) were expressed in E. coli, purified and their specific activities measured (Table 2). The single and double mutants show CoA-dependent specific activities comparable to the wild-type HsKAR. Only the triple mutant shows reduced (14% remaining) NADPH-dependent activity. These results suggest that the mutation of βLys169 to Glu does not affect the folding, stability and CoA-dependent activity of HsKAR, but disrupts the ability to recognize ACP in vivo. It can be noted here that βLys169 of HsKAR is in the same position as Arg172 of E. coli fabG (Fig. 3a). Therefore, our yeast complementation studies clearly demonstrate that CBR4 is the catalytic as well as the ACP-binding subunit of HsKAR for the mtFAS activity.

Discussion

The importance of the mtFAS pathway has recently been recognized, as it is related to lipoic acid synthesis³²,³³, diseases such as cardiomyopathy in mammals³⁴, mitochondrial respiratory dysfunction and accelerated aging³³. HsKAR, one of the enzymes of this essential mtFAS pathway, is a unique heterotetramer formed by HsHSD17B8 and HsCBR4 (ref. 9). In this study, we show that the reductase activity of the HsCBR4 (β-subunit) is NADPH-dependent and responsible for the KAR activity of mtFAS. These data, for the first time, give a clue on the physiological function of CBR4. HsHSD17B8, the α-subunit of HsKAR, is NAD(H)-dependent and is necessary for a soluble and

8
NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.

NATURE COMMUNICATIONS | DOI: 10.1038/ncomms5805

ARTICLE

Chain B (CBR4)  
Chain A (HSD17B8)  
$\alpha$Arg148  
NADP⁺  
Hexanoyl-CoA  
$\beta$Arg168  
$\beta$Lys169  
$\beta$Gln124–126  
Chain C (CBR4)  
Chain D (HSD17B8)

Chain B (CBR4)  
Chain A (HSD17B8)  
$\alpha$Arg148  
NADP⁺  
Hexanoyl-CoA  
$\beta$Arg168  
$\beta$Lys169  
$\beta$Gln124–126  
Chain C (CBR4)  
Chain D (HSD17B8)

Figure 6 | Stereo diagram of the HsKAR tetramer in surface representation highlighting the residues chosen for mutation to study the ACP recognition. The chosen residues are represented as sticks on chain C (yellow; CBR4) and chain A (brown; HSD17B8). NADP⁺ is shown bound in chain B (magenta ball-and-stick). The pink ball-and-stick model is hexanoyl-CoA from *M. tuberculosis* fabG4 (PDB id: 3V1U) superposed on HsKAR (CBR4; chain B) to show the predicted fatty-acyl and pantetheine binding groove of HsKAR extending from CBR4 chain B to CBR4 chain C. The *in vivo* data indicate that the ACP conjugated to the fatty acyl tail binds to the CBR4 subunit (chain C) near $\beta$Lys169.

competent complex. As discussed further below, we speculate on a possible role for this subunit in routing the 3R-hydroxyacyl-CoA intermediates emerging from the metabolism of unsaturated fatty acids to the mitochondrial β-oxidation pathway.

The HsKAR crystal structures show that HsHSD17B8 and HsCBR4 form a physically interacting and functional HsKAR α₂β₂-tetramer. These results are in agreement with our previous study demonstrating that co-expression of HsHSD17B8 and HsCBR4 in the *oar1Δ* yeast cells rescues their growth on a non-fermentable carbon source. Purification experiments demonstrate that HSD17B8 only forms heterotetramers with CBR4 on heterologous co-expression in *E. coli*. Our attempts to purify the homotetrameric HsHSD17B8 and HsCBR4 separately resulted in very low protein yield due to their poor solubility compared with the heterotetrameric HsKAR. Nevertheless, the deposited structure of HsHSD17B8 (PDB id: 2PD6) shows that this protein can form homotetramers in the absence of CBR4. The observed heterotetrameric arrangement of HsKAR is similar to the homotetrameric orthologues of HsKAR, for example, from *B. napus* and *E. coli*, and the two subunits interact in an optimal way, providing higher stability to the heterotetramer than to the respective homotetramers.

The enzyme kinetic data define the distinctive functionalities of the HSD17B8 and CBR4 subunits of HsKAR. Mutations in the HSD17B8 active site ($\alpha$Y169A and $\alpha$K173A-HsKAR) specifically reduce much the NADH-dependent activities, while mutations in the CBR4 catalytic site ($\beta$K152A-HsKAR) affect only the NADPH-dependent activities (Table 2). The SPR binding studies show that NADPH has the highest affinity for HsKAR. In addition, NADPH and NADH showed a weak-affinity second binding site with $K_{D}$ in the millimolar range, which may be due to negative cooperativity in HsKAR. SPR binding studies with the $\alpha$D42A-HsKAR variant show that the NAD(H) binding is affected. These data, together with our crystallographic binding studies, show the selective recognition of NAD(H) and NADP(H) by the HsHSD17B8 and HsCBR4 subunits of the HsKAR complex, respectively.

The *in vivo* role of the NADPH-dependent CBR4 subunit of HsKAR is immediately evident from our yeast complementation experiments with HsKAR. These studies clearly show that mutations at the CBR4 active site impair the growth in the presence of glycerol, while the growth of the HSD17B8 mutants is the same as for the wild-type cells. This implies that CBR4 is the catalytically active subunit with respect to the mtFAS KAR activity, and the cofactor for the mtFAS is NADPH (Fig. 1) as also observed in other FAS-II pathways. As the reactions of mtFAS use thioesters of ACP as substrates, we assessed the role of HsKAR subunits in ACP recognition through mutagenesis and yeast complementation studies. These studies show that only the CBR4 variants of HsKAR containing the mutation $\beta$K169E were unable to rescue the growth of the *oar1Δ* yeast cells on glycerol as the sole carbon source, suggesting that $\beta$Lys169 is involved in ACP recognition. These experiments provide credence that CBR4 is responsible for both the catalytic activity associated with mtFAS pathway as well as ACP-recognition (Fig. 7). The relatively low catalytic activity of the NADPH-dependent CBR4 might also be due to the fact that the real substrate is the fatty-acyl-ACP conjugate.

The current data show that the CBR4 subunit of HsKAR is essential for complementation of the *oar1Δ* respiratory deficient phenotype *in vivo* in yeast. In the cytosolic FAS-I complex, KAR is composed of a catalytic and a non-catalytic domain ($\psi$KAR), the latter providing only a structural role. HSD17B8 may serve an analogous purpose. This is also supported by the observations that the HsKAR α₂β₂-tetramer possesses much higher solubility than when CBR4 is expressed alone, and that a hypothetical CBR4 tetramer shows many steric clashes mainly between the C terminus of diagonally opposite protomers in the tetramer. However, apart from being essential for the formation of a competent α₂β₂-complex, it is intriguing to note that the HSD17B8 subunit is highly active in the reductive (this study) as well as dehydrogenating direction as shown by us in a previous study unlike the non-catalytic $\psi$KAR domain of the FAS-I complex. This indicates that the function of the HSD17B8 subunit in HsKAR is not limited to just a structural role. The mitochondrial NADPH and NAD⁺ concentrations are in the millimolar range and are much higher than the concentration of NADP⁺ and NADH under conditions

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.

ARTICLE

NATURE COMMUNICATIONS | DOI: 10.1038/ncomms5805

![Diagram](attachment://diagram.png)

Figure 7 | Schematic representation of the catalytic properties of the HsKAR subunits (CBR4 ($\beta$) and HSD17B8 ($\alpha$)) in the context of the mtFAS pathway *in vivo*. The CBR4 subunit is the functional subunit of the KAR complex catalysing the conversion of 3-ketoacyl-ACP to 3R-hydroxyacyl-ACP (dashed arrow). CBR4 is a poor acceptor of fatty acyl-CoA conjugates (dotted arrow). The high affinity of the CBR4 subunit for NADPH will favour the reduction of 3-ketoacyl-ACP to 3R-hydroxyacyl-ACP whenever the NADPH/NADP⁺ ratio is high. The HSD17B8 subunit may not accept fatty acyl-ACP conjugates (dashed arrow with cross). However, the HSD17B8 subunit does efficiently catalyse the NAD⁺-dependent conversion of 3R-hydroxyacyl-CoA into 3-ketoacyl-CoA, and therefore can introduce 3R-hydroxyacyl-CoA into the mitochondrial β-oxidation pool.

pertaining in the coupled respiring mammalian mitochondria favouring the NADPH-dependent fatty acid synthesis and the NAD⁺-dependent β-oxidation. Therefore, in mitochondria, the NADP(H)-dependent CBR4 subunit of HsKAR works in reductive direction for fatty acid synthesis and the NAD(H)-dependent HSD17B8 subunit can work towards the dehydrogenating direction. Curiously, the NAD⁺-dependent dehydrogenase reaction of HSD17B8 is specific for 3R-hydroxyacyl-CoA⁹. In peroxisomes, the 3R-hydroxyacyl-CoA intermediates are metabolized by a homologue of HSD17B8, the NAD⁺-dependent hydratase domain of the multifunctional enzyme, type-2 (ref. 39). In mitochondria, β-oxidation proceeds via 3S-hydroxyacyl-CoA intermediates. A probable source of 3R-hydroxyacyl-CoA intermediates in mitochondria is the β-oxidation of polyunsaturated fatty acids (PUFAs). The unsaturated intermediates of Z-PUFAs are converted to 2E-enoyl-CoA with the help of the auxiliary enzymes of mitochondrial β-oxidation, Δ³,⁵, Δ²,⁴-dienoyl-CoA isomerase, Δ³,²-enoyl-CoA isomerase and Δ²,⁴-dienoyl-CoA reductase⁴⁰. However, if Z-PUFAs escape this auxiliary pathway, then 2Z-enoyl-CoA metabolites are formed. In mitochondria, these are converted to 3R-hydroxyacyl-CoA esters by enoyl-CoA hydratase 1 (ref. 41). Under such conditions, HSD17B8 of the HsKAR complex can prevent the accumulation of these intermediates by efficient conversion of 3R-hydroxyacyl-CoA to 3-ketoacyl-CoA. Therefore, we speculate that this HSD17B8 catalytic activity may act as a rescue route reintroducing 3R-hydroxyacyl-CoA esters into the mitochondrial β-oxidation intermediate pool (Fig. 7), functioning as a fourth auxiliary enzyme of the mitochondrial β-oxidation pathway. We have initiated studies to test this function *in vivo* in suitable genetically modified mouse models.

### Methods

#### Media and strains.
Yeast (*S. cerevisiae*) cells were grown either on YPD (1% yeast extract, 2% peptone and 2% D-glucose), YPG (1% yeast extract, 2% peptone and 3% glycerol) or SC medium (Sigma-Aldrich, Helsinki, Finland), SC-glucose (2%), SC-glycerol (3%), or other SC medium lacking one or more nutrients. The yeast and bacterial strains used in this study are listed in Supplementary Table 6. The BY4741 *oar1Δ* strain was obtained from EUROSCARF (Frankfurt, Germany).

#### Molecular cloning and site-directed mutagenesis.
The oligonucleotides used are listed in Supplementary Table 3. PCR amplification were carried out using Pfu polymerase (Stratagene, La Jolla, CA, USA). The p631-HsKAR construct for the bacterial protein production and YEp-HsKAR construct for yeast complementation experiments expressing HsKAR components HsHSD17B8 and HsCBR4 (ref. 9) were used as templates for generating mutated variants by applying a QuickChange site-directed mutagenesis kit (Stratagene). Mutations were verified by sequencing and the plasmids encoding mutants were transformed into BL21 (RARE) *E. coli* cells for protein synthesis or yeast BY4741 *oar1Δ* cells for complementation assays.

#### Protein expression and purification.
p631-HsKAR and its mutant variants were expressed and purified by following a similar protocol⁹. Briefly, BL21 (RARE) cells containing plasmids were cultured in M9ZB medium at 37°C until OD₆₀₀ reach 0.6. 0.1 mM isopropyl-β-D-thiogalactoside was then added into the media and cells were incubated at 18°C for additional 18–20 h. The collected cells were resuspended in 20 mM Tris buffer with 500 mM sodium chloride, 5 mM imidazole, 10% glycerol and 0.1% Triton-X 100 at pH 7.9, sonicated, and HsKAR was purified by Ni-NTA affinity chromatography from the soluble fraction using a buffer containing 20 mM Tris, 500 mM sodium chloride pH 7.9 and 5–500 mM imidazole. The fractions containing HsKAR were then pooled and loaded for further purification onto a Superdex 200 size-exclusion chromatography column (GE Healthcare, New York, USA) previously equilibrated with 20 mM HEPES buffer pH 7.2 containing 120 mM potassium chloride, 1 mM EDTA and 1 mM sodium azide. The purified protein was concentrated and stored at –70°C until used further.

#### Crystallization and structure determination.
HsKAR was crystallized using the sitting drop vapour diffusion method at room temperature. One microlitre of HsKAR at a concentration of 8 mg ml⁻¹ with 2 mM NAD⁺ was mixed with 1 μl of the crystallization solution containing 15–18% PEG3350 and 0.4 M ammonium acetate in sodium acetate buffer at pH 5.0. HsKAR crystals appeared in 1–2 weeks. The crystals of HsKAR complexed with both NAD⁺ and NADP⁺ were obtained by soaking the crystals co-crystallized with NAD⁺ in a solution containing the crystallization condition with additional 5 mM NAD⁺ and 5 mM NADP⁺ overnight. Before data collection, the crystals were transferred into a crystallization solution containing also 20% ethylene glycol as the cryo-protectant and subsequently were frozen in liquid nitrogen.

X-ray diffraction data of the HsKAR-NAD⁺ complex crystals were collected at the European Synchrotron Radiation Facility, Grenoble, France, beamline ID-29 at a wavelength of 0.9763 Å using the ADSC Quantum Q315r detector. The data were processed with the iMOSFLM⁴² and XDS⁴³ packages. Twinning was detected and the twin fraction was calculated by Phenix.xtriage⁴⁴. There are four tetramers per asymmetric unit. The data collection statistics are given in Table 1. The structure was solved by using the programme PHASER⁴⁵ with the HsHSD17B8 homotetramer (PDB id: 2PD6) as the starting model. The structure was initially refined using REFMAC5 (ref. 46) and at later stages twin refinement of Phenix.refine⁴⁴ was carried out. The estimated twin fraction is 0.3. The detection of twinning and the usage of the twin refinement option of ‘Phenix.refine’ greatly improved the resulting electron density maps as well as the refinement statistics.

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications

© 2014 Macmillan Publishers Limited. All rights reserved.

The structure was refined to a final R-free and R-work of 23.6% and 18.2%, respectively. Residues (94.9%) are in the favoured region of the Ramachandran plot, while 4.9% and 0.2% of the residues are in the allowed and outlier regions, respectively. The refinement statistics are presented in Table 1. For HsKAR crystals complexed with both NAD<sup>+</sup> and NADP<sup>+</sup> (HsKAR-NAD(P)<sup>+</sup>), the X-ray diffraction data were collected at the Diamond Light Source, Oxford, UK, beamline I04 at a wavelength of 0.9795 Å using a PILATUS detector. The data were processed using the XDS package. The twin fraction for this crystal was much lower being < 0.1. The HsKAR-NAD<sup>+</sup> structure was used as the initial model to further build and refine the HsKAR-NAD(P)<sup>+</sup> structure by the COOT and Phenix packages. The structure was refined to a final R-free and R-work of 23.6% and 20.3%, respectively. The distribution of residues in the Ramachandran plot is 93.3%, 3.5% and 0.1% in the favoured, allowed and outlier regions, respectively. A stereo image of a part of the 2F<sub>o</sub>–F<sub>c</sub> electron density map of the HsKAR-NAD<sup>+</sup> and HsKAR-NAD(P)<sup>+</sup> structures is shown in Supplementary Fig. 6.

**Structure validation and analysis.** The final structures were validated using Molprobity<sup>47</sup>. The α- and β-subunits of HsKAR belong to the SDR superfamily. Of the four tetramers in the asymmetric unit, a single tetramer (chains A, B, C and D) was used for further analysis and comparison. The secondary structures assigned in Supplementary Fig. 1 follow previous nomenclature conventions<sup>16</sup>. The HsKAR subunits were compared with homotetrameric KAR structures of the following species: (i) *B. napus*<sup>13</sup> (rape) (PDB id: 1EDU), (ii) *E.coli*<sup>14,15</sup> (PDB id: 1IO1 and 1Q7B) and (iii) HsHSD17B8, the α-chain homotetramer (PDB id: 2PD6). The *M. tuberculosis* fabG4 (ref. 31) structure (PDB id: 3V1U) was used to study the ternary complex geometry. The structures were superimposed using the SSM option<sup>48</sup> in COOT<sup>49</sup>. The figures of the protein structures were drawn using PyMol (The PyMOL Molecular Graphics System, Version 1.5.0.3 Schrödinger, LLC). The web-based ClustalW2 programme was used to make the sequence alignment<sup>50</sup>. The sequence alignment figures were prepared using the program ESPRIPT<sup>51</sup>.

**Enzyme assays.** The catalytic activity of HsKAR in the synthetic direction (the 3-ketoacyl-CoA reductase activity) was determined spectrophotometrically using a JASCO V660 spectrophotometer by monitoring the disappearance of NAD(P)H at 340 nm. Acetoacetyl-CoA was used as the substrate to assess the reductase activity. The reaction mixture contained 60 μM acetoacetyl-CoA and 125 μM NAD(P)H in 50 mM potassium phosphate buffer pH 7.4 with 100 μg ml<sup>−1</sup> BSA in a final volume of 0.5 ml at 25 °C. The possible effect of Ca<sup>2+</sup> or Mg<sup>2+</sup> on the HsKAR catalytic activity was verified<sup>15</sup>. These assays were therefore done also with additional 10 mM CaCl<sub>2</sub> or MgCl<sub>2</sub>. Although the use of CaCl<sub>2</sub> in assays resulted in the precipitation of buffer, MgCl<sub>2</sub> did not have any effect on the NAD(P)H-dependent activity. HsCBR4 has previously been characterized to have *in vitro* quinone reductase activity<sup>12</sup>. Therefore, we also tested quinone reductase activity using 60 μM 9,10-PQ (Sigma-Aldrich, Steinheim, Germany) as the substrate in the reaction mixture (0.5 ml) containing also 125 μM NAD(P)H in 100 mM potassium phosphate buffer pH 7.5 with 50 μg ml<sup>−1</sup> BSA. The reaction was initiated by adding 60 ng of HsKAR and the disappearance of NAD(P)H at 340 nm was monitored. All the reported values for the enzymatic activity are the averages of at least three independent measurements and the reported errors are the s.d. of these measurements.

**Yeast spotting assay.** Yeast cells (BY4741 and BY4741oar1Δ mutant) were transformed with different combinations of plasmids by one-step transformation in stationary phase<sup>52</sup>. For each transformation, ~1 × 10<sup>8</sup> cells of stationary phase were collected from overnight cell culture by centrifugation and resuspended with 80 μl of yeast one-step transformation solution (100 mM lithium acetate, 33% PEG3350 and 0.28 mg ml<sup>−1</sup> single-stranded DNA) and then mixed with 100 ng of plasmid DNA. The mixture was incubated at 45 °C for 45 min before plating. After transformation, the cells were plated on SC-Ura, SC-Leu, SC-Ura-Leu media containing 2% glucose. The colonies from plates were inoculated into appropriate selective liquid media and the cells were grown in a shaker at 30 °C until the OD<sub>600</sub> reached 8–10. The cells were then collected and spotted with a serial dilution on SC plates containing either 2% glucose or 3% glycerol as the sole carbon source and grown at 30 °C for 5–7 days.

**SPR studies.** Biacore T200 and the carboxymethyl-dextran-coated gold surface (CM5) sensor chip were used for the SPR measurements. HsKAR at a concentration of 50 μg ml<sup>−1</sup> in 10 mM sodium acetate buffer pH 5.0 was immobilized covalently by amine coupling on the CM5 chip, previously activated with 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride and N-hydroxysuccinimide to ~9,500 RU. The remaining reactive sites were blocked with ethanolamine. A reference cell was prepared in the same manner, except that HsKAR was not injected. PBS buffer (1×, pH 7.4) was used for all the measurements as well as for preparing the ligand solutions. The ligands, NAD<sup>+</sup> or NADP<sup>+</sup>, were passed over the chip containing HsKAR at various concentrations ranging from 76 nM to 10 mM at a rate of 30 μl min<sup>−1</sup> for 2 min. For NADH and NADPH, concentrations ranging from 76 nm to 40 mM were used. This was followed by a wash with PBS buffer pH 7.4 for 10–20 min to remove the bound ligands and regenerate the chip. This resulted in square pulses with fast association

and dissociation rates (Supplementary Fig. 4), and therefore the kinetics of ligand binding could not be studied. The *K*<sub>D</sub> values were calculated using either the two-site or the steady-state binding module of the Biacore T200 Evaluation software. For HsKAR and αD42A-HsKAR, sensograms of only up to 2.5 mM for NADP<sup>+</sup> and from 4.8 μM to 2.5 mM for NAD<sup>+</sup> were used for the calculation of *K*<sub>D</sub>. Moreover, for αD42A-HsKAR sensogram from 40 mM NADPH and sensograms up to 2.4 μM NADH were not used for the determination of *K*<sub>D</sub>. The measurements were repeated at least three times and s.d. were calculated.

**References**

1. Walker, J. E. The ATP synthase: the understood, the uncertain and the unknown. *Biochem. Soc. Trans.* **41**, 1–16 (2013).
2. Bhaumik, P., Koski, M. K., Glumoff, T., Hiltunen, J. K. & Wierenga, R. K. Structural biology of the thioester-dependent degradation and synthesis of fatty acids. *Curr. Opin. Struct. Biol.* **15**, 621–628 (2005).
3. Jiang, X. & Wang, X. Cytochrome C-mediated apoptosis. *Annu. Rev. Biochem.* **73**, 87–106 (2004).
4. West, A. P., Shadel, G. S. & Ghosh, S. Mitochondria in innate immune responses. *Nat. Rev. Immunol.* **11**, 389–402 (2011).
5. Hiltunen, J. K., Chen, Z., Haapalainen, A. M., Wierenga, R. K. & Kastaniotis, A. J. Mitochondrial fatty acid synthesis - an adopted set of enzymes making a pathway of major importance for the cellular metabolism. *Prog. Lipid Res.* **49**, 27–45 (2010).
6. Kursu, V. A. S. *et al.* Defects in mitochondrial fatty acid synthesis result in failure of multiple aspects of mitochondrial biogenesis in *Saccharomyces cerevisiae*. *Mol. Microbiol.* **90**, 824–840 (2013).
7. Torkko, J. M. *et al.* *Candida tropicalis* Etr1p and *Saccharomyces cerevisiae* Ybr026p (Mrfl'p), 2-enoyl thioester reductases essential for mitochondrial respiratory competence. *Mol. Cell Biol.* **21**, 6243–6253 (2001).
8. Harington, A., Herbert, C. J., Tung, B., Getz, G. S. & Slonimski, P. P. Identification of a new nuclear gene (CEM1) encoding a protein homologous to a beta-keto-acyl synthase which is essential for mitochondrial respiration in *Saccharomyces cerevisiae*. *Mol. Microbiol.* **9**, 545–555 (1993).
9. Chen, Z. *et al.* 17beta-hydroxysteroid dehydrogenase type 8 and carbonyl reductase type 4 assemble as a ketoacyl reductase of human mitochondrial FAS. *FASEB J.* **23**, 3682–3691 (2009).
10. Ohno, S., Nishikawa, K., Honda, Y. & Nakajin, S. Expression in *E. coli* and tissue distribution of the human homologue of the mouse Ke6 gene, 17beta-hydroxysteroid dehydrogenase type 8. *Mol. Cell Biochem.* **309**, 209–215 (2008).
11. Kavanagh, K. L., Jörnvall, H., Persson, B. & Oppermann, U. Medium- and short-chain dehydrogenase/reductase gene and protein families. *Cell. Mol. Life Sci.* **65**, 3895–3906 (2008).
12. Endo, S. *et al.* Human carbonyl reductase 4 is a mitochondrial NADPH-dependent quinone reductase. *Biochem. Biophys. Res. Commun.* **377**, 1326–1330 (2008).
13. Fisher, M. *et al.* The X-ray structure of *Brassica napus* beta-keto acyl carrier protein reductase and its implications for substrate binding and catalysis. *Structure* **8**, 339–347 (2000).
14. Price, A. C., Zhang, Y. M., Rock, C. O. & White, S. W. Structure of beta-ketoacyl-[acyl carrier protein] reductase from *Escherichia coli*: negative cooperativity and its structural basis. *Biochemistry* **40**, 12772–12781 (2001).
15. Price, A. C., Zhang, Y. M., Rock, C. O. & White, S. W. Cofactor-induced conformational rearrangements establish a catalytically competent active site and a proton relay conduit in FabG. *Structure* **12**, 417–428 (2004).
16. Cohen-Gonsaud, M. *et al.* Crystal structure of MabA from *Mycobacterium tuberculosis*, a reductase involved in long-chain fatty acid biosynthesis. *J. Mol. Biol.* **320**, 249–261 (2002).
17. Wickramasinghe, S. R. *et al.* Kinetic, inhibition and structural studies on 3-oxoacyl-ACP reductase from *Plasmodium falciparum*, a key enzyme in fatty acid biosynthesis. *Biochem. J.* **393**, 447–457 (2006).
18. Zaccai, N. R. *et al.* Crystal structure of a 3-oxoacyl-(acylcarrier protein) reductase (BA3989) from *Bacillus anthracis* at 2.4-Å resolution. *Proteins* **70**, 562–567 (2008).
19. Subramanian, S. *et al.* Structure of 3-ketoacyl-(acyl-carrier-protein) reductase from *Rickettsia prowazekii* at 2.25 Å resolution. *Acta Crystallogr. F Struct. Biol. Cryst. Commun.* **67**, 1118–1122 (2011).
20. Dutta, D., Bhattacharyya, S. & Das, A. K. Crystal structure and fluorescence studies reveal the role of helical dimeric interface of staphylococcal FabG1 in positive cooperativity for NADPH. *Proteins* **80**, 1250–1257 (2012).
21. Krissinel, E. & Henrick, K. Inference of macromolecular assemblies from crystalline state. *J. Mol. Biol.* **372**, 774–797 (2007).
22. Wierenga, R. K., Terpstra, P. & Hol, W. G. J. Prediction of the occurrence of the ADP-binding βαβ-fold in proteins, using an amino acid sequence fingerprint. *J. Mol. Biol.* **187**, 101–107 (1986).
23. Pillai, S. *et al.* Functional characterization of beta-ketoacyl-ACP reductase (FabG) from *Plasmodium falciparum*. *Biochem. Biophys. Res. Commun.* **303**, 387–392 (2003).

NATURE COMMUNICATIONS | 5:4805 | DOI: 10.1038/ncomms5805 | www.nature.com/naturecommunications
© 2014 Macmillan Publishers Limited. All rights reserved.

ARTICLE

24. Patel, M. P. *et al.* Kinetic and chemical mechanisms of the fabG-encoded *Streptococcus pneumoniae* β-ketoacyl-ACP reductase. *Biochemistry* 44, 16753–16765 (2005).

25. Colletier, J. *et al.* Structural insights into substrate traffic and inhibition in acetylcholinesterase. *EMBO J.* 25, 2746–2756 (2006).

26. Lejon, S., Ellis, J. & Valegård, K. The last step in cephalosporin C formation revealed: crystal structures of deacetylcephalosporin C acetyltransferase from *Acremonium chrysogenum* in complexes with reaction intermediates. *J. Mol. Biol.* 377, 935–944 (2008).

27. Vincent, F. *et al.* Multifunctional xylooligosaccharide/cephalosporin C deacetylase revealed by the hexameric structure of the *Bacillus subtilis* enzyme at 1.9 Å resolution. *J. Mol. Biol.* 330, 593–606 (2003).

28. Karmodiya, K. & Surolia, N. Analyses of co-operative transitions in *Plasmodium falciparum* beta-ketoacyl acyl carrier protein reductase upon co-factor and acyl carrier protein binding. *FEBS J.* 273, 4093–4103 (2006).

29. Silva, R. G., Rosado, L. A., Santos, D. S. & Basso, L. A. *Mycobacterium tuberculosis* β-ketoacyl-ACP reductase: α-secondary kinetic isotope effects and kinetic and equilibrium mechanisms of substrate binding. *Arch. Biochem. Biophys.* 471, 1–10 (2008).

30. Zhang, Y. M., Wu, B., Zheng, J. & Rock, C. O. Key residues responsible for acyl carrier protein and beta-ketoacyl-acyl carrier protein reductase (FabG) interaction. *J. Biol. Chem.* 278, 52935–52943 (2003).

31. Dutta, D., Bhattacharyya, S., Roychowdhury, A., Biswas, R. & Das, A. K. Crystal structure of hexanoyl-CoA bound to beta-ketoacyl reductase FabG4 of *Mycobacterium tuberculosis*. *Biochem. J.* 450, 127–139 (2013).

32. Yi, X. & Maeda, N. Endogenous production of lipoic acid is essential for mouse development. *Mol. Cell Biol.* 25, 8387–8392 (2005).

33. Smith, S. *et al.* Compromised mitochondrial fatty acid synthesis in transgenic mice results in defective protein lipoylation and energy disequilibrium. *PLoS ONE* 7, e47196 (2012).

34. Chen, Z. *et al.* Myocardial overexpression of *Mecr*, a gene of mitochondrial FAS II leads to cardiac dysfunction in mouse. *PLoS ONE* 4, e5589 (2009).

35. Maier, T., Leibundgut, M. & Ban, N. The crystal structure of a mammalian fatty acid synthase. *Science* 321, 1315–1322 (2008).

36. Yang, H. *et al.* Nutrient-sensitive mitochondrial NAD⁺ levels dictate cell survival. *Cell* 130, 1095–1107 (2007).

37. Mailloux, R. J. & Harper, M. Glucose regulates enzymatic sources of mitochondrial NADPH in skeletal muscle cells; a novel role for glucose-6-phosphate dehydrogenase. *FASEB J.* 24, 2495–2506 (2010).

38. Jackson, J. B. Proton translocation by transhydrogenase. *FEBS Lett.* 555, 176–177 (2003).

39. Haataja, T. J., Koski, M. K., Hiltunen, J. K. & Glumoff, T. Peroxisomal multifunctional enzyme type 2 from the fruitfly: dehydrogenase and hydratase act as separate entities, as revealed by structure and kinetics. *Biochem. J.* 435, 771–781 (2011).

40. van Weeghel, M. *et al.* Functional redundancy of mitochondrial enoyl-CoA isomerases in the oxidation of unsaturated fatty acids. *FASEB J.* 26, 4316–4326 (2012).

41. Wakil, S. J. Studies on the fatty acid oxidizing system of animal tissues. IX. Stereospecificity of unsaturated acyl CoA hydrase. *Biochim. Biophys. Acta* 19, 497–504 (1956).

42. Battye, T. G., Kontogiannis, L., Johnson, O., Powell, H. R. & Leslie, A. G. iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM. *Acta Crystallogr. D Biol. Crystallogr.* 67, 271–281 (2011).

43. Kabsch, W. Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants. *J. Appl. Crystallogr.* 26, 795–800 (1993).

44. Adams, P. D. *et al.* PHENIX: a comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr. D Biol. Crystallogr.* 66, 213–221 (2010).

45. McCoy, A. J. *et al.* Phaser crystallographic software. *J. Appl. Crystallogr.* 40, 658–674 (2007).

46. Vagin, A. A. *et al.* REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use. *Acta Crystallogr. D Biol. Crystallogr.* 60, 2184–2195 (2004).

47. Chen, V. B. *et al.* MolProbity: all-atom structure validation for macromolecular crystallography. *Acta Crystallogr. D Biol. Crystallogr.* 66, 12–21 (2010).

48. Krissinel, E. & Henrick, K. Secondary-structure matching (SSM), a new tool for fast protein structure alignment in three dimensions. *Acta Crystallogr. D Biol. Crystallogr.* 60, 2256–2268 (2004).

49. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr. D Biol. Crystallogr.* 66, 486–501 (2010).

50. Larkin, M. A. *et al.* Clustal W and Clustal X version 2.0. *Bioinformatics* 23, 2947–2948 (2007).

51. Gouet, P., Courcelle, E., Stuart, D. I. & Metoz, F. ESPript: analysis of multiple sequence alignments in PostScript. *Bioinformatics* 15, 305–308 (1999).

52. Chen, D. C., Yang, B. C. & Kuo, T. T. One-step transformation of yeast in stationary phase. *Curr. Genet.* 21, 83–84 (1992).


### Acknowledgements

This work was supported by the Academy of Finland, Sigrid Jusélius Foundation, Center for International Mobility (CIMO) and the National Natural Science Foundation of China (NSF) (No. 21372097). The research leading to these results has received funding from the European Community’s Seventh Framework Programme (FP7/2007-2013) under BioStruct-X (grant agreement number 283570). We thank Professor Lloyd W. Ruddock for helpful discussion on SPR studies.


### Author contributions

R.V., A.J.K., J.K.H., R.W. and Z.C. designed the experiments, analysed the results and wrote the manuscript. R.V. purified the protein, crystallized and determined the crystal structures. Z.C. performed molecular cloning, protein purification and yeast genetic experiments. S.S. and R.V. purified the mutated variants, carried out enzymological and SPR-binding studies, and performed molecular cloning. L.A. purified the protein for crystallization studies. G.J. performed yeast genetic experiments and prepared Fig. 7. P.P. helped with the SPR measurements.


### Additional information

**Accession codes:** The coordinates and the structure factor files for the crystal structure of the HsKAR-NAD⁺ and HsKAR-NAD(P)⁺ structures are deposited in the Protein Data Bank with the accession codes 4CQL and 4CQM, respectively.

**Supplementary Information** accompanies this paper at http://www.nature.com/naturecommunications

**Competing financial interests:** The authors declare no competing financial interests.

**Reprints and permission** information is available online at http://npg.nature.com/reprintpermissions/

**How to cite this article:** Venkatesan, R. *et al.* Insights into mitochondrial fatty acid synthesis from the structure of heterotetrameric 3-ketoacyl-ACP reductase/3R-hydroxyacyl-CoA dehydrogenase. *Nat. Commun.* 5:4805 doi: 10.1038/ncomms5805 (2014).
