
REVIEWS

MICROBIAL BIOFILMS

Living in the matrix: assembly and control of Vibrio cholerae biofilms

Jennifer K. Teschler¹, David Zamorano-Sánchez¹, Andrew S. Utada², Christopher J. A. Warner³, Gerard C. L. Wong², Roger G. Lington³ and Fitnat H. Yildiz¹

Abstract | Nearly all bacteria form biofilms as a strategy for survival and persistence. Biofilms are associated with biotic and abiotic surfaces and are composed of aggregates of cells that are encased by a self-produced or acquired extracellular matrix. Vibrio cholerae has been studied as a model organism for understanding biofilm formation in environmental pathogens, as it spends much of its life cycle outside of the human host in the aquatic environment. Given the important role of biofilm formation in the V. cholerae life cycle, the molecular mechanisms underlying this process and the signals that trigger biofilm assembly or dispersal have been areas of intense investigation over the past 20 years. In this Review, we discuss V. cholerae surface attachment, various matrix components and the regulatory networks controlling biofilm formation.

Planktonic state  
Single drifting cells that inhabit  
the water column.

Chitin  
A β(1→4)-linked homopolymer  
of N-acetyl-d-glucosamine,  
found in the exoskeleton of  
zooplankton and other  
crustaceans. It is an abundant  
source of carbon, nitrogen and  
energy for many  
microorganisms.

Filippo Pacini first isolated and described the Gram-negative bacterium Vibrio cholerae in 1854, the same year that John Snow’s ‘ghost maps’ revealed that a tainted water supply was the source of a deadly cholera outbreak. Pathogenic strains of V. cholerae cause cholera, which is an acute diarrhoeal disease that can result in hypotonic shock and death within 12 hours of the first symptoms¹. Approximately 3–5 million individuals are infected with V. cholerae annually, and 100,000–120,000 cases are fatal each year¹.

V. cholerae forms biofilms during the aquatic and intestinal phases of its life cycle²⁻⁴. Briefly, both toxigenic and non-toxigenic V. cholerae strains live in the aquatic environment year-round, either in a planktonic state or in a biofilm. When toxigenic strains of V. cholerae enter the human host, typically through the ingestion of contaminated water or food, they colonize the small intestine. Inside the intestine, V. cholerae multiplies and produces cholera toxin, which causes severe illness in the host. V. cholerae is then shed in the stool, from where it can re-enter the aquatic environment or infect a new host¹. The role of biofilms in the environmental persistence, dissemination and transmission of V. cholerae has been well established (FIG. 1). This growth mode provides protection from a number of environmental stresses, including nutrient limitation, predation by unicellular eukaryotes (known as protozoa) and attack by viruses that target bacteria (known as bacteriophages)⁵,⁶. Although V. cholerae can form biofilms on many biotic and abiotic surfaces, several field studies have shown that V. cholerae preferentially forms biofilms on phytoplankton, zooplankton and oceanic chitin⁷,⁸. The exoskeletons of zooplankton contain chitin, which V. cholerae can utilize as a sole carbon source⁹,¹⁰. Growth on chitin also induces natural competence and enables cells to acquire new genetic material¹¹. As the physical carriers and primary sources of nutrients for V. cholerae, zooplankton serve as reservoirs and disease vectors of cholera¹².

Although V. cholerae is found year-round in the coastal and estuarine environments where cholera is endemic, outbreaks are seasonal and correlate with changes in environmental conditions¹². The major environmental factor affecting seasonal outbreaks is thought to be the occurrence of plankton blooms, which are influenced by water temperature, hours of sunlight, sea surface height, rainfall and water salinity¹³. Simple water filtration practices that remove particles larger than 20 µm were shown to significantly reduce cholera cases, suggesting that the removal of V. cholerae biofilms from the environment can reduce transmission¹⁴.

Between epidemics, metabolically quiescent V. cholerae cells have been observed both in the planktonic state and in biofilms, and seem to contribute to V. cholerae persistence³,⁴. These quiescent cells can lose their typical curved rod shape, becoming coccoid, and cannot be cultured under standard laboratory conditions. They can return to an active state in response to signals produced by either active cells present in the environment or passage through a host, although the mechanism

¹Department of Microbiology and Environmental Toxicology, University of California, Santa Cruz, California 95064, USA.  
²Bioengineering Department, Chemistry and Biochemistry Department, California NanoSystems Institute, University of California, Los Angeles, California 90095, USA.  
³Department of Chemistry and Biochemistry, University of California, Santa Cruz, California 95064, USA.  
Correspondence to F.H.Y.  
e-mail: fyildiz@ucsc.edu  
doi:10.1038/nrmicro3433

NATURE REVIEWS | MICROBIOLOGY

© 2015 Macmillan Publishers Limited. All rights reserved

REVIEW S

Aquatic environment

V. cholerae  
biofilm  

Phytoplankton  

Planktonic V. cholerae  

MSHA pilus  

Flagellum  

Detritus  

Zooplankton  

Metabolically active cell  

Metabolically quiescent cell  

Ingestion  

Human host  

Mucus  

Small intestine  

TCP  

Stool  

Shedding  

TCP bundling  

Figure 1 | Biofilms in the Vibrio cholerae life cycle. In the aquatic environment, Vibrio cholerae is found in its highly mobile planktonic form as well as in biofilms formed on zooplankton, phytoplankton, detritus and other surfaces, such as sediments. Following the initial stages of attachment to abiotic and biotic surfaces, which involves the type IV pili mannose-sensitive haemagglutinin (MSHA) pili, cells produce the extracellular matrix, which is essential to achieve mature biofilms with a three-dimensional structure. As it is unknown whether the flagellum or MSHA pili are lost during biofilm formation, cells are depicted both with and without the flagellum or pili in biofilms. V. cholerae can be ingested by humans from environmental sources, causing seasonal outbreaks. During intestinal colonization, both planktonic cells and biofilm aggregates have been observed, and V. cholerae produces toxin coregulated pili (TCP). Both planktonic cells and biofilm aggregates resulting from Vibrio polysaccharide (VPS) production and TCP bundling are found in patient stool, and these cells can re-infect a new host or return to the aquatic environment.

of host-mediated activation is unknown³,¹⁵,¹⁶. Biofilms containing metabolically quiescent V. cholerae may have important biological relevance, as the reduced metabolic needs and slowed growth of these quiescent cells may enable them to survive harsh environmental conditions until circumstances improve. Following activation, they can act as seed cells for V. cholerae growth in the water supply and contribute to an outbreak¹⁵.

V. cholerae biofilms contain both high doses of bacteria and hyperinfective cells, and they therefore have a key role in transmission³,¹⁷. The hyperinfective state refers to a decrease in the number of cells required to cause disease, relative to the average number. In other words, the bacterial dose required for infection is lower than normal, and the risk of disease transmission is higher.

Rugosity  
Pertaining to a bacterial colony: a corrugated colony phenotype associated with high levels of biofilm matrix production.

However, the role of V. cholerae biofilm formation inside the host is poorly understood. Both single cells and dense clumps of V. cholerae were observed in a rabbit ileal loop infection model, supporting an earlier finding that biofilms can form *in vivo* and can subsequently be excreted in the stool³,¹⁸. Biofilms are composed of aggregates of cells encased by a self-produced or acquired extracellular matrix and thus may have greater resistance to host defence mechanisms than free bacterial cells. Although the role of biofilms in host resistance has not been well explored, several studies suggest that a key component of the biofilm, Vibrio polysaccharide (VPS), is produced during host infection¹⁹,²⁰. Additionally, deletion of genes involved in the production of VPS and the extracellular matrix protein rugosity and biofilm
structure modulator A (RbmA) led to a defect in intestinal colonization in a mouse model²¹. Collectively, these findings imply that biofilms play a part during *V. cholerae* infection, but further studies are needed to elucidate the underlying mechanisms and functions of *in vivo* biofilms. Currently, much of what we know about *V. cholerae* biofilm structure, function and regulation is based on *in vitro* findings.

In this Review, we discuss recent advances in our understanding of initial *V. cholerae* surface attachment, and provide an overview of the matrix components and of biofilm dispersal. We also review the regulatory network that governs *V. cholerae* biofilm formation, including the transcriptional regulators of key genes involved in biofilm formation, as well as the roles of small nucleotides and small RNAs (sRNAs). Finally, we discuss the impact of aquatic and host environmental inputs on biofilm formation, and we highlight new discoveries in small-molecule therapeutics that have the potential to control and inhibit *V. cholerae* biofilms.

---

### Roaming
A motility mode of surface-skimming *Vibrio cholerae* cells in which cells move with meandering, gently-curved trajectories that range over large areas of the surface.

### Orbiting
A motility mode of surface-skimming *Vibrio cholerae* cells in which cells move with tightly curved circular trajectories that hover over small areas of the surface.

### Flagellum
A motility structure composed of a cytoplasmic basal body that functions as a motor, a rod that extends from the cytoplasm through the membrane, and a long filamentous polymer projecting from the cell.

### Flow cell
A piece of equipment that is used for the *in vitro* culture and examination of bacterial biofilms under hydrodynamic flow conditions.

### Radius of gyration (*R*<sub>gyr</sub>)
As used here: a statistical measure of the spatial extent of a bacterial track. It is defined as the root of the mean square distance between each point of a track and the centre of mass of that track. For a perfect circle of radius *r*, *R*<sub>gyr</sub> = *r*.

### Pili
Proteinaceous filaments that are found on the surface of many bacteria and are often involved in adhesion or motility.

---

## Surface attachment

*V. cholerae* biofilm formation is a multistep process: bacteria mechanically ‘scan’ the surface using roaming or orbiting movements, attach to the surface and subsequently form microcolonies, which lead to the generation of organized, three-dimensional structures²² (FIG. 2).

### Orbiting and roaming motility
Bacteria swimming in close proximity to surfaces experience hydrodynamic forces that both attract them towards the surface and cause them to move in circular trajectories²³. *V. cholerae* is equipped with a single polar flagellum driven by a Na⁺ motor²⁴. As the flagellum sweeps past the surface, it is subjected to viscous drag forces, which induce torque on the cell body; this surface-induced torque deflects the directional movement of the cells into curved clockwise paths²⁵.

Using high-speed tracking of *V. cholera* grown in flow cell chambers, two types of trajectory have been identified: orbiting involves tight, repetitive, near-circular orbits with high curvatures (radius of gyration (*R*<sub>gyr</sub>) < 8 µm), whereas roaming involves long directional persistence and small curvatures (*R*<sub>gyr</sub> > 8 µm)²². In both motility modes, cells move in an oblique direction that strongly deviates from the cell axis and have strong nutations along the trajectory. Moreover, the direction of motion seems to be exclusively clockwise for both motility modes²². These motility modes are ablated in strains lacking mannose-sensitive haemagglutinin (MSHA) pili (a member of the type IV pilus (TFP) family) or a flagellum, suggesting that both types of appendage are necessary for these characteristic behaviours²².

Theoretical modelling was used to elucidate the origins of these orbiting and roaming motility behaviours²². In free-swimming cells, flagellar rotation causes the cell body to counter-rotate. For surface-skimming *V. cholerae*, this body rotation associated with swimming causes MSHA appendages to have periodic mechanical contact with the surface, enabling surface-skimming cells to continually assay the surface mechanically via friction.

Orbiting enables *V. cholerae* to ‘loiter’ over surface regions that interact more strongly with MSHA pili, whereas roaming enables the cells to pass over surface regions that interact more weakly with these pili²² (FIG. 2A).

Orbiting *V. cholerae* cells exhibit intermittent pauses of different durations before eventually attaching to the surface. Both the frequency and duration of these pauses significantly decreased when cells were incubated with a non-metabolizable mannose derivative to saturate MSHA-pili binding²², which suggests that MSHA-pili–surface interactions are mechanochemical in nature. Moreover, strains lacking MSHA pili are defective in initial surface attachment²²,²⁶. Taken together, these observations suggest that MSHA-pili–surface binding is crucial to arrest cell motion near the surface and to transition to surface attachment and microcolony formation.

It is important to note that the initial surface attachment behaviour of *V. cholerae* is unlike the behaviour of *Pseudomonas aeruginosa*, which reversibly attaches to surfaces in a vertical orientation and moves along random trajectories via TFP-driven ‘walking motility’ after initial attachment²⁷,²⁸. *P. aeruginosa* cells can then transition to an irreversibly attached state in which the cell axis is oriented parallel to the surface; these cells move along the surface by TFP-driven ‘twitching’, guided by a network of secreted polysaccharides and extracellular DNA (eDNA)²⁹,³⁰, ultimately resulting in the formation of microcolonies. By contrast, as discussed above, *V. cholerae* uses the polar flagellum and MSHA pili synergistically to scan a surface mechanically before surface attachment. The sites of surface attachment strongly correlate with the positions of microcolonies, which indicates that TFP-driven motility has a minor role in determining the positions of *V. cholerae* microcolonies²².

After surface attachment, it is unknown whether the *V. cholerae* flagellum is functional, whether it is lost and degraded, or whether it acts as a structural component in the biofilm. However, mutations in a flagellar structural gene, flagellin A (*flaA*), resulted in increased exopolysaccharide production, which suggests that the lack of a flagellum serves as a signal for biofilm formation³¹,³². Surprisingly, mutations in the flagellar motor genes *motB* and *motY* rescue this phenotype³¹,³², indicating that the Na⁺-driven flagellar motor might act as a mechanosensor, enabling *V. cholerae* to recognize when it encounters a surface and to subsequently induce the appropriate attachment response³¹,³².

## Macrocolony formation and the matrix

Following the initial stages of bacterial attachment, cells produce the extracellular matrix, which is essential to achieve mature biofilms with a three-dimensional structure. Distinct morphological and phenotypic differences can be observed among cells producing different quantities of biofilm matrix components (BOX 1). Compositional analysis of an intact *V. cholerae* biofilm matrix by solid-state nuclear magnetic resonance (NMR) using <sup>15</sup>N profiling and by spectroscopic analysis of the extracellular-matrix carbon pools showed that the extracellular matrix is primarily composed
REVIEW S

A

Roaming

MSHA pilus

Flagellum

Orbiting

Attachment

B

a

VPS

b

RbmA

c

Bap1

d

RbmC

e

f

Dispersal

Figure 2 | Building a Vibrio cholerae biofilm. A | Surface motility and initial attachment. Surface-skimming Vibrio cholerae cells use flagella to move and mechanically ‘scan’ the surface via mannose-sensitive haemagglutinin (MSHA) pili. Weak interactions between surfaces and pili lead to roaming behaviour (tight, repetitive, near-circular orbits with high curvatures), whereas strong surface–pili interactions lead to orbiting behaviour (long directional persistence and small curvatures). Orbiting enables cells to loiter over these regions and eventually attach and initiate microcolony formation. Motility trajectories are depicted by black lines on the surface. B | Microcolony formation and matrix production. Soon after initial attachment, Vibrio polysaccharide (VPS) is excreted from cell surfaces (part Ba), and VPS extrusion is observed throughout biofilm formation. Next, the biofilm matrix protein rugosity and biofilm structure modulator A (RbmA) accumulates on the cell surface (part Bb). During cell division, the daughter cell remains attached to the founder cell (also known as the parental cell), confirming the role of RbmA in cell–cell adhesion, and the biofilm matrix protein Bap1 is excreted between the two cells and on the substrate near the two cells (part Bc). Bap1 gradually accumulates radially on nearby surfaces, although the concentration of Bap1 remains the highest near the founder cell. Subsequently, the biofilm matrix protein RbmC is excreted and found on discrete sites on the cell surface (part Bd). As the biofilm develops, VPS, RbmC and Bap1 form envelopes that can grow as cells divide (part Be). The mature biofilm is a composite of organized clusters composed of cells, VPS, RbmA, Bap1 and RbmC, in addition to other matrix components, such as outer-membrane vesicles (OMVs) and extracellular DNA (eDNA; part Bf and inset). Outer-membrane proteins (OMPs) associate with Bap1 in OMVs and bind to antimicrobial peptides, thereby increasing V. cholerae resistance. The last stage in biofilm development is dispersal, whereby exiting V. cholerae cells seek out and colonize new resources; however, the underlying mechanism remains to be determined.

of polysaccharides, phospholipids, proteins and small amounts of nucleic acids³³. In addition, the V. cholerae biofilm matrix seems to be rich in sugar, especially when compared with the protein-rich biofilm matrix

of Escherichia coli³³. Defining and quantifying the major building blocks of the V. cholerae biofilm not only furthers our understanding of how individual components interact to support the formation of a complete biofilm

© 2015 Macmillan Publishers Limited. All rights reserved

Box 1 | Analysis of Vibrio cholerae biofilm formation

More than 200 serogroups of Vibrio cholerae have been identified, of which only the V. cholerae O1 and V. cholerae O139 serotypes are capable of causing pandemic cholera outbreaks<sup>95</sup>. The V. cholerae O1 serotype is further classified into the biotypes V. cholerae O1 classic and V. cholerae O1 El Tor on the basis of their biochemical properties and phage resistance<sup>95</sup>. Most of our understanding of V. cholerae biofilms has developed from studying V. cholerae O1 El Tor and V. cholerae O139, which include the commonly studied A1552, C6706, N16961 and MO10 strains. V. cholerae can generate phenotypic variants — that is, smooth and rugose colonies — that vary in their ability to form biofilms. The ability of V. cholerae to switch between a smooth and wrinkled colony morphology was first noted by Balteanu in 1926 (REF. 96). The wrinkled variant was termed rugose in 1938 by White<sup>97</sup>, who observed “that the rugose growth habit in vibrios results from abnormally active secretion of mucinous material”. Owing to their high production of Vibrio polysaccharide (VPS) and formation of robust biofilms, rugose variants have been used extensively to characterize biofilm matrix components and regulation<sup>34</sup>. In fact, the formation of corrugated colonies (see the figure, part **a**), also termed colony biofilms, is dependent on the production of biofilm matrix materials.

Various experimental methods are used to study and characterize V. cholerae biofilms. Crystal violet staining or high-throughput imaging microscopy can be used to visualize and quantify biofilms formed in multiwell plates<sup>26,98</sup>. Biofilms that are formed by V. cholerae under static or flow conditions (typically using strains engineered to constitutively produce a fluorescent protein) can be visualized by laser scanning confocal microscopy (LSCM) to analyse biofilm structure (see the figure, part **b**, which shows the top-down view of the biofilm in the central square and the respective side views of the biofilm in the adjacent rectangles)<sup>26,34</sup>. Biofilm parameters such as biomass, surface colonization, thickness and heterogeneity can be quantified using the COMSTAT biofilm analysis programme<sup>99</sup>. Architectural details of the biofilm structure can be evaluated by electron microscopy and high-resolution microscopy.

Pellicle biofilms (see the figure, part **c**) have also been used to assess the role of biofilm components in biofilm integrity, strength of attachment and thickness<sup>34,38</sup>. Recently, interfacial rheology was used to study the mechanical properties of pellicles, and to analyse pellicle strength and the morphology of biofilms containing cells that lack matrix protein<sup>43</sup>. Images courtesy of N. Fong, University of California, Santa Cruz, USA.

---

Pellicles  
Biofilms formed at an air–liquid interface.

V. cholerae biofilm matrix cluster (VcBMC). A genetic module comprising the Vibrio polysaccharide 1 (*vps-1*), rugosity and biofilm structure modulator (*rbm*) and *vps-2* gene clusters, which encode many of the proteins that generate VPS and major biofilm proteins.

Serogroups  
Groups of bacterial strains based on the structure of the surface O antigen group.

surface properties<sup>41</sup>. The dimer interface forms a wide groove, which is capable of accommodating large filamentous substrates, such as VPS, and a tight groove, which is capable of binding the negatively charged carbohydrates found on cell surfaces. Saturation transfer difference (STD) experiments indicate that these two binding sites preferentially bind monosaccharides from VPS and lipopolysaccharide (LPS), implying that RbmA acts as a biofilm scaffolding protein<sup>42</sup>. RbmA accumulates on the cell surface after initial attachment and VPS production<sup>36</sup> (FIG. 2Bb). RbmA was also shown to contribute to early elasticity and corrugation in pellicle biofilms, further corroborating its role in the development of biofilm architecture and stability<sup>43</sup>.

Two other major biofilm matrix proteins — Bap1 and the Bap1 homologue, RbmC — have 47% sequence similarity, but have non-redundant roles in biofilm formation<sup>37</sup>. Bap1 contains four overlapping Vibrio–Colwellia–Bradyrhizobium–Shewanella repeat (VCBS) domains, which may be involved in cell adhesion, and four FG-GAP domains, which are thought to be important for recognition and binding of an as-yet-unidentified ligand<sup>36,37,44</sup>. During biofilm formation, Bap1 is secreted at the cell-surface interface and gradually accumulates radially on nearby surfaces, although the concentration of Bap1 remains the highest near the founder cell (also known as the parental cell). These findings support a role for Bap1 in surface adhesion and suggest that the founder cell and its earliest descendants are primarily responsible for the production of Bap1 (REF. 36) (FIG. 2B). In rugose pellicles, Bap1 was found to be uniquely required for maintaining pellicle strength over time, and scanning electron microscopy revealed that a Bap1-deficient mutant exhibited a distinctly different pellicle microstructure. Bap1 was also shown to contribute greatly to pellicle hydrophobicity, enabling the protein to spread and remain at an air–water interface<sup>43</sup>.

The RbmC protein of *V. cholerae* also has four VCBS domains, but contains only two FG-GAP domains. RbmC is larger than Bap1, and it has two carboxy-terminal β-prism domains (whereas Bap1 has only one) and two amino-terminal domains of unknown function<sup>40</sup>. The β-prism domain has lectin-binding and carbohydrate-binding activity in other bacterial proteins, but the significance of the β-prism binding properties in RbmC is still being explored<sup>45</sup>. As biofilms develop and more cell division occurs, RbmC is secreted at discrete sites on the cell surface, and RbmC and Bap1 form flexible envelopes that surround the cells and can grow as cells divide<sup>36</sup> (FIG. 2Bd). During biofilm formation on a solid-water interface, RbmA, RbmC and Bap1 were unable to accumulate on the surface of cells that did not produce VPS, and RbmC was shown to be critical for incorporating VPS throughout the biofilm. Thus, the mature biofilm is a composite of organized clusters that include cells, VPS, RbmA, Bap1 and RbmC<sup>36</sup> (FIG. 2Be,f).

A recent study demonstrated that the type II secretion system (T2SS) — a multiprotein system that exports proteins from the cell by translocating them from the periplasm through the outer membrane — is responsible for the secretion of RbmA, RbmC and Bap1 (REF. 46).

T2SS mutants were unable to secrete RbmA, RbmC and Bap1 into culture and exhibited diminished biofilm formation, although VPS excretion from the cell remained unaffected. Additionally, deletion of the T2SS in a rugose strain abolished colony corrugation and pellicle formation, further supporting the hypothesis that the T2SS has a crucial role in biofilm formation and morphology<sup>46</sup>.

*V. cholerae* biofilm proteins have also been associated with outer-membrane vesicles (OMVs), which act as secretory vehicles. A study analyzing the content of OMVs from *V. cholerae* grown under in vitro virulence-inducing conditions identified 90 proteins that associate with OMVs, including RbmA, RbmC and Bap1 (REF. 47). It is unknown whether the association of biofilm proteins with OMVs is a regulated cellular programme or merely a result of the random inclusion of proteins that pass through the periplasm. However, in the presence of antimicrobial peptides, Bap1 was shown to bind to the surface of OMVs via its association with outer-membrane protein T (OmpT). Evidence suggests that OMV-associated Bap1 then binds to antimicrobial peptides and attenuates their impact on *V. cholerae*, thus increasing bacterial resistance<sup>44</sup>. Future studies are required to determine the contribution of other OMV-associated and free matrix proteins to biofilm structure and function.

**Dispersal.** The last stage in biofilm development is dispersal. Although dispersal is an important step in the biofilm cycle, as it enables exiting cells to seek out and colonize new resources, little is known about this process in *V. cholerae*. Two extracellular deoxyribonucleases, DNase (also known as Dns) and Xds, have been implicated in biofilm development and dispersal through their regulation of eDNA, which plays a part in nutrient delivery and biofilm structure<sup>48</sup>. eDNA that is released by cell lysis or active secretion may be taken up by competent cells during growth on chitin, and it can then serve as a source of organic nutrients for the competent cells or can become incorporated into the genome. Alternatively, eDNA can remain in the biofilm matrix, where it seems to act as an important structural component<sup>48</sup>. Deletion of DNase and Xds promoted biofilm formation independently of VPS production, altered biofilm structure and impaired detachment from biofilms<sup>48</sup>. Evidence indicates that degradation of eDNA by these nucleases reduces biofilm formation and might facilitate dispersal. Impaired in vivo colonization was also observed in DNase-deficient and Xds-deficient cells, which suggests that dispersal is necessary for colonization of the host<sup>48</sup>.

Additionally, *rbmB*, a gene in the *rbm* cluster of the VcBMC, encodes a putative polysaccharide lyase that has been proposed to have a role in VPS degradation and cell detachment. Strains lacking RbmB exhibit enhanced biofilm formation compared with strains that encode the protein, although the enzymatic activity of RbmB has not been experimentally demonstrated<sup>37</sup>. The downregulation of biofilm components, discussed in more detail below, is likely to play a part in dispersal; however, the proteins involved in the degradation of biofilm components remain to be identified. Extracellular signals such

Box 2 | Environmental signals controlling Vibrio cholerae biofilm formation

Vibrio cholerae encounters a number of fluctuating environmental signals during its life cycle. Regulation of biofilm formation in response to these external signals is an important factor in survival and persistence. The identification of such signals and determination of the molecular mechanism of signal integration into the biofilm regulatory network are crucial to increase our understanding of the regulation of biofilm formation during intestinal and aquatic survival of V. cholerae.

The bacterial phosphoenolpyruvate phosphotransferase system (PTS) is a highly conserved system that controls the transport of certain sugars into the cell. Sugars transported by the PTS, such as mannose and glucose, enhance biofilm formation in V. cholerae, suggesting a role for the PTS in determining the environmental suitability for biofilm growth<sup>100</sup>. The PTS contains the general components enzyme I (EI) and the phosphocarrier histidine-containing protein (HPr), both of which function upstream of the carbohydrate-specific enzymes EIIA and EIIBC<sup>100</sup>. The phosphorylation state of PTS components reflects nutrient availability: PTS enzymes will become phosphorylated when no sugars are available for import, whereas components will quickly become dephosphorylated when sugar transport is active<sup>100</sup>. Four independent PTS pathways have been shown to function in the activation or repression of V. cholerae biofilm formation, thus providing another link between the nutritional status of the cell and biofilm formation<sup>101,102</sup>.

In addition to nutrient availability, a number of other environmental signals are thought to have a role in V. cholerae biofilm formation. Salinity and osmolarity fluctuations in the aquatic environment can affect biofilm formation and Vibrio polysaccharide (vps) gene expression<sup>103–105</sup>. Two transcriptional regulators, OscR and CosR, regulate the biofilm in response to osmolarity and ionic strength, respectively. At low salinities, transcription of *oscR* is increased, and OscR inhibits VPS production and upregulates motility<sup>104</sup>. As ionic strength increases, CosR activates biofilm formation and represses motility<sup>105</sup>. The mechanism by which OscR and CosR sense a shift in osmolarity and ionic strength remains to be determined.

Phosphate limitation in the aquatic environment has been implicated in the negative regulation of V. cholerae biofilm formation<sup>106,107</sup>. PhoBR — a regulatory system that responds to phosphate limitation — upregulates motility and downregulates biofilm formation, possibly by repressing *vpsR* (the major positive regulator of the vps genes) and regulating the expression levels of genes encoding diguanylate cyclases (DGCs) and phosphodiesterases (PDEs)<sup>108,109</sup>. PhoBR-mediated regulation of biofilm formation and the stress response was shown to be independent of the transcriptional repressor HapR and the alternative RNA polymerase sigma factor RpoS, and may play a part in dispersal from environmental biofilms or host intestines<sup>108,110</sup>. Small organic cations (polyamines), such as norspermidine and spermidine, have been shown to induce and repress biofilm formation, respectively, in response to environmental signals<sup>87</sup>.

Ca<sup>2+</sup> levels vary in the aquatic environment, and extracellular Ca<sup>2+</sup> has been shown to decrease *vps* gene transcription and lead to the dissolution of biofilms<sup>111,112</sup> by regulating the expression of the two-component regulatory system CarRS<sup>111</sup>.

Indole, which is produced by bacteria found in the human gut, is thought to act as an extracellular signalling molecule that activates *vps* genes via a signalling cascade<sup>113</sup>. The role of bile in V. cholerae biofilm formation is also not completely clear.

Bile has been shown to induce biofilm formation in a VpsR-dependent manner<sup>114</sup>; it was also demonstrated that bile acids increase intracellular cyclic di-GMP levels and biofilm formation<sup>86</sup>. However, a recent study found that exposure of V. cholerae biofilms to a component of bile, taurocholate, can lead to abiotic degradation of the biofilm matrix and might therefore lead to in vivo biofilm dispersal and inhibition of biofilm formation<sup>49</sup>.

---

as the bile salt taurocholate might also act as signals for biofilm dispersal<sup>49</sup> (BOX 2). The identification of the proteins that are crucial for dispersal is essential and would further our understanding of how and when V. cholerae disperses from a biofilm.

Cyclic di-GMP (c-di-GMP). A key signalling molecule that controls the motile-to-biofilm transition and biofilm formation by inhibiting motility and stimulating the synthesis of cell-surface adhesins and/or exopolysaccharides.

**Vibrio cholerae biofilm regulation**

V. cholerae biofilm formation is controlled by an integrated regulatory network of transcriptional activators (VpsR, VpsT and AphA), transcriptional repressors (HapR and H-NS), alternative RNA polymerase sigma factors (RpoN (also known as σ<sup>54</sup>), RpoS and RpoE), sRNAs and signalling molecules (FIG. 3). Biofilm matrix production is controlled by a highly connected regulatory network that integrates at least three different nucleotide second messengers and the quorum sensing response (FIG. 3b). Biofilm formation is an energetically costly process; commitment to the biofilm lifestyle has major biological consequences and must therefore be both tightly regulated and plastic, enabling biofilm bacteria to be responsive to the various environmental cues that they experience during their life cycle (BOX 2).

Positive regulation. VpsR, the master regulator of biofilm formation in V. cholerae, is a member of the two-component signal transduction system (TCS) response regulator family. It has an N-terminal response regulator receiver (REC) domain, an ATPases associated with wide variety of cellular activities (AAA+) domain and a C-terminal helix-turn-helix (HTH) DNA-binding domain<sup>50</sup>. VpsR is required for biofilm formation, as disruption of *vpsR* prevents expression of the Vps family and matrix proteins, and abolishes the formation of biofilms. VpsR binds to the *vps* promoter regions to directly control gene expression<sup>51</sup> (FIG. 3a). VpsR also upregulates extracellular protein secretion (*eps*) genes (which encode the Eps proteins that form part of the T2SS) and matrix protein genes, as well as *aphA*, which is a major virulence regulator; VpsR-mediated regulation of *aphA* indicates that VpsR may also have a role in pathogenesis<sup>50,52,53</sup>. VpsR contains a conserved aspartate residue, Asp59, which seems to be critical for protein function. Conversion of this aspartate to alanine renders VpsR inactive, whereas conversion to glutamic acid results in a constitutively active VpsR, supporting the premise that phosphorylation controls the DNA-binding activity of VpsR<sup>32</sup>. It has been shown that VpsR can bind the second messenger cyclic di-GMP (c-di-GMP), although c-di-GMP does not alter the DNA-binding ability of the protein<sup>54</sup>. The sensor histidine kinase that partners VpsR (that is, the other TCS component) or the kinases that have a role in activating VpsR and positively regulating *vps* gene expression and biofilm formation have not yet been identified. The expression of VpsR is positively regulated by VpsT and negatively regulated by HapR, although other factors are likely to be involved and further study is needed to fully characterize VpsR regulation<sup>52</sup>.

A second positive regulator of biofilm formation, VpsT, is also a response regulator. VpsT consists of an N-terminal REC domain and a C-terminal HTH domain. Unlike other REC domains, the canonical (α/β)<sub>5</sub> fold in VpsT is extended by an additional helix (α6) at the C terminus<sup>55</sup>. Disruption of *vpsT* reduces the expression of *vps* and matrix protein genes and reduces the biofilm-forming capacity of the bacteria. Similarly to VpsR, VpsT binds to the *vps* promoter region to directly control the expression of *vps* genes<sup>51,55</sup> (FIG. 3a). VpsT binding to c-di-GMP is required for DNA association and transcriptional regulation<sup>55</sup>, and a dimer of c-di-GMP binds to a VpsT dimer with an affinity of 3.2 μM. The VpsT c-di-GMP-binding motif is Trp-(Phe/Leu/Met)-(Thr/Ser)-Arg<sup>55</sup>. Mutations in the sequence encoding the putative phosphorylation

REVIEWS

Figure 3 | The regulatory network controlling Vibrio cholerae biofilms.  
a | The transcriptional activators Vibrio polysaccharide protein R (VpsR) and VpsT and the transcriptional repressors HapR and H-NS directly and indirectly regulate several genes that have key roles in biofilm formation. Positive regulators of biofilm are shown in blue, whereas negative regulators are shown in pink. The targets of these regulators include the vps clusters and the rugosity and biofilm structure modulator (rbm) cluster, which all contain genes encoding biofilm matrix proteins and proteins that are involved in VPS production. The vps and rbm clusters constitute a functional genetic module, known as the Vibrio cholerae biofilm matrix cluster (VcBMC). In addition, bap1 has been shown to be regulated by these core regulators. The recognition sequences for VpsR, VpsT, HapR and H-NS have been identified in the regulatory regions of the vps-1 and vps-2 clusters, and in the regulatory regions of rbmA, rbmC and bap1 (which encode extracellular matrix proteins). Binding of VpsT to promoter regions requires its interaction with cyclic di-GMP (c-di-GMP). As shown, the VpsR and VpsT targets extensively overlap, although some biofilm genes seem to be directly regulated by only one of these activators. Additionally, the negative regulators directly downregulate many of the vps and rbm genes, as well as the genes that encode the positive transcriptional regulators of those genes (shown in part a).  
b | An extensive regulatory network governs V. cholerae biofilm formation. VpsR, VpsT and AphA are the main activators of biofilm formation, and HapR and H-NS are the main repressors (dashed box). VpsR, VpsT, HapR and H-NS directly regulate genes involved in biofilm formation (see part a). These core regulators directly and indirectly regulate each other and are modulated by a complex regulatory network in response to a number of environmental and host signals. The quorum sensing pathway, which responds to cell density via bacterial signalling, has a key role in the regulation of HapR and, thus, the other major biofilm regulators. The signalling molecules autoinducer 2 (AI-2) and cholerae autoinducer 1 (CAI-1) regulate a phosphorelay event that culminates at the histidine phosphotransfer protein LuxU and the response regulator LuxO. At low cell density, CAI-1 and AI-2 production is low (dashed arrows), leading to LuxO phosphorylation by the quorum sensing signal transduction pathway. Together with the alternative RNA polymerase sigma factor RpoN, phosphorylated LuxO activates transcription of the four small RNAs (sRNAs) quorum-regulatory RNA 1 (Qrr1)–Qrr4, which work in conjunction with the sRNA chaperone Hfq to prevent the translation of hapR. Thus, biofilm genes are expressed at low cell densities. The VarS–VarA system responds to an unknown environmental cue and represses biofilm production by post-transcriptionally upregulating HapR. This process involves the regulatory sRNAs CsrB, CsrC and CsrD, which bind to and titrate the RNA-binding protein CsrA, thereby interfering with LuxO-mediated activation of Qrr sRNAs. This leads to decreased levels of the Qrr sRNAs and enhanced HapR production. By contrast, the small protein Fis is a direct positive regulator of the Qrr sRNAs, thereby promoting HapR repression. Furthermore, the histidine kinase VpsS donates phosphate groups to LuxU, thus promoting HapR repression. In an additional layer of control, small-nucleotide molecules, including cyclic AMP (cAMP), guanosine tetraphosphate and guanosine pentaphosphate (collectively referred to as (p)ppGpp) and cyclic di-GMP (c-di-GMP), regulate the induction and repression of major biofilm regulators, including HapR, VpsT and VpsR. The sigma factor RpoS promotes the expression of hapR. The regulatory role of RpoS in biofilm formation seems to be dependent on growth conditions. It has been proposed that RpoS may positively regulate biofilm formation during the stringent response and provide negative regulation under other conditions. Of note, RpoS is depicted with (p)ppGpp because the stringent response-mediated regulation of vpsT and vpsR has been shown to occur partly through RpoS and partly through (p)ppGpp (via an unknown intermediate, denoted X). A key signalling molecule controlling V. cholerae motility and biofilm matrix production is the second messenger c-di-GMP. High cellular levels of c-di-GMP promote enhanced transcription of genes involved in biofilm formation, possibly by promoting VpsT-mediated transcriptional activation of vps genes. Several diguanylate cyclases (DGCs), which cumulatively contribute to c-di-GMP levels, and phosphodiesterases (PDEs), which are known to degrade cellular c-di-GMP to pGpG or GMP, are shown. The second messenger cAMP is involved in various cellular responses and acts as a repressor of V. cholerae biofilm formation. cAMP is generated by adenylyl cyclase (CyaA), and in complex with cAMP receptor protein (CRP), this nucleotide has been shown to upregulate HapR production through the positive regulation of the CAI-1 synthase and the negative regulation of Fis.

site — which were intended to produce a constitutively inactive or active state — do not alter the efficiency of VpsT, indicating that its role in regulating gene expression is independent of its phosphorylation status<sup>55</sup>. vpsT is positively regulated by VpsR, AphA and RpoS, and negatively regulated by HapR and HN-S<sup>52,56–58</sup>.

**TetR regulators**  
A family of proteins involved in the transcriptional control of several cellular processes, including biofilm formation, pathogenesis, catabolism, antibiotic resistance, and differentiation. TetR family members harbour a helix–turn–helix motif that is highly similar to the DNA-binding motif of TetR (a regulator that controls the expression of tetracycline resistance (tet) genes).

The VpsR and VpsT regulons extensively overlap and, although both proteins positively regulate the transcription of vps and biofilm-related genes (FIG. 3a), the magnitude of their respective gene regulation varies. The fact that VpsR and VpsT can each modulate the expression of the other could, in part, be responsible for the overlap in their regulons<sup>52,53</sup>. A recent study identified the VpsR and VpsT recognition sequences in the regulatory region of the first gene in the vps-2 operon, vpsL<sup>51</sup>. Moreover, in silico analysis to determine promoter recognition sites revealed that both VpsR and VpsT can bind to the regulatory region of the first gene in the vps-1 cluster, vpsU, as well as to the regulatory regions of rbmA and vpsT. This study also showed that the promoter of another gene in the vps-1 cluster, vpsA, contains only the VpsT recognition sequence, whereas rbmC and bap1 promoters contain only the VpsR recognition sequence<sup>51</sup>. These findings support the premise that these two regulators act in concert by directly targeting all of the regulatory regions in the VcBMC<sup>53,55,57</sup> (FIG. 3a).

**Negative regulation.** HapR is the main negative regulator of biofilm formation in V. cholerae, as disruption of hapR enhances biofilm formation<sup>59–61</sup>. HapR directly binds to the regulatory regions of vpsL (the first gene in the vps-2 operon) (FIG. 3a) and vpsT<sup>62</sup>. HapR has homology to TetR regulators; the N terminus contains a HTH domain, and the C terminus contains a dimerization domain that is predicted to have a binding pocket for an unidentified amphipathic ligand containing anionic moieties<sup>63</sup>.

the four sRNAs quorum-regulatory RNA 1 (Qrr1)–Qrr4, which work in conjunction with the sRNA chaperone Hfq to prevent the translation of *hapR*. This ultimately results in the upregulation of biofilm formation<sup>65</sup> (FIG. 3b). By contrast, at high cell densities (when AI-2 and CAI-1 levels are high), LuxO is dephosphorylated via the activity of the receptors CqsS and LuxQ, and the Qrr sRNAs are repressed. This activates HapR expression and results in the downregulation of biofilm formation<sup>65</sup>.

Several additional regulators have been shown to be involved in quorum sensing-mediated regulation of HapR and to thereby affect biofilm formation (FIG. 3b). The two-component system VarS–VarA upregulates *hapR* expression post-transcriptionally through a pathway that involves the regulatory sRNAs CsrB, CsrC and CsrD<sup>66,67</sup>. These sRNAs bind to and titrate the RNA-binding protein CsrA, interfering with the LuxO-mediated activation of the Qrr sRNAs; this leads to decreased levels of Qrr sRNAs and enhanced HapR production<sup>67</sup>. By contrast, the small DNA-binding protein Fis is a direct positive regulator of the Qrr sRNAs<sup>68</sup>, thereby promoting HapR repression. VpsS, a hybrid histidine kinase, also increases biofilm formation through the quorum sensing pathway by donating phosphate groups to the phosphotransfer protein LuxU<sup>69</sup>. The global regulator cyclic AMP receptor protein (CRP) has been shown to upregulate HapR production through positive regulation of the CAI-1 synthase and through negative regulation of Fis, suggesting that CRP functions at two regulatory junctions in the quorum sensing pathway<sup>70</sup>. Finally, *hapR* is also regulated independently of the quorum sensing pathway: the transcriptional regulator Vibrio quorum modulator A (VqmA) can directly activate *hapR* expression, and RpoS also promotes *hapR* expression<sup>53,71</sup>.

H-NS is a histone-like protein that has an important role in modulating nucleoid topology and also functions as a transcriptional regulator. It has low sequence specificity and shows a preference for AT-rich regions with high curvature<sup>72</sup>. In *V. cholerae*, H-NS negatively controls the expression of biofilm and virulence genes<sup>72,73</sup>. A strain lacking *hns* has a considerably enhanced ability to form biofilms; it has been shown that H-NS acts as a direct negative regulator of *vpsL*, *vpsA* and *vpsT* both *in vitro* and *in vivo*, although little is known about the role of H-NS in controlling other biofilm genes<sup>72</sup> (FIG. 3a). A recent study revealed that when VpsT is bound to the *vpsL* regulatory region, it prevents H-NS-mediated silencing; however, in the same study it was shown that VpsT also regulates biofilm formation independently of H-NS<sup>51</sup>.

Small-nucleotide signalling. A key signalling molecule controlling *V. cholerae* motility and biofilm matrix production, and thus the planktonic-to-biofilm transition, is the nucleotide-based second messenger c-di-GMP<sup>74</sup> (FIG. 3b). This messenger is synthesized by diguanylate cyclases (DGCs), which contain a GGDEF domain, and it is degraded by phosphodiesterases (PDEs), which contain an EAL or HD-GYP domain<sup>75</sup>. The *V. cholerae* genome encodes 31 proteins with a GGDEF domain,

The timing of *hapR* repression and activation is controlled by quorum sensing, and modulates the formation of mature biofilm structures (*hapR* repression) and dispersal from the biofilm (*hapR* activation)<sup>60,64</sup>. Quorum sensing-deficient mutants form thicker biofilms and do not detach as easily from the biofilm structure compared with wild type<sup>64</sup>. HapR production is negatively controlled at low cell densities through the quorum sensing pathway (reviewed in REF. 65). Briefly, the membrane-bound sensor histidine kinases LuxQ and CqsS recognize the signalling molecules autoinducer 2 (AI-2) and cholerae autoinducer 1 (CAI-1), respectively, and initiate a phosphorelay event that culminates at the histidine phosphotransfer protein LuxU and the response regulator LuxO<sup>65</sup> (FIG. 3b). At low cell density (when the concentrations of AI-2 and CAI-1 are low), phosphorylated LuxO, together with RpoN, activates transcription of

Cyclic AMP (cAMP). A second-messenger signalling molecule that is involved in the regulation of several cell processes, including cell division, catabolite repression, motility and biofilm formation.

12 proteins with an EAL domain and nine proteins with a HD-GYP domain<sup>76</sup>. Although an additional ten genes encode proteins with both GGDEF and EAL domains, this does not necessarily imply that the protein exhibits both DGC and PDE activity, as it is common for one domain to be degenerate. In *V. cholerae*, c-di-GMP is sensed by receptor proteins, including PilZ domain-containing proteins and VpsT, or by c-di-GMP-responsive riboswitches<sup>55,77,78</sup>.

At present, little is known about the precise molecular mechanisms by which c-di-GMP affects motility and the planktonic-to-biofilm transition in *V. cholerae*. Systematic phenotypic characterization of isogenic *V. cholerae* mutants with in-frame deletions in the genes encoding predicted DGCs and EAL domain-containing PDEs revealed that four DGCs (CdgH, CdgK, CdgL and CdgD) inhibit motility and two PDEs (CdgJ and RocS) promote motility<sup>79</sup>, and regulation of the abundance or activity of these proteins is predicted to be critical for the motile-to-sessile transition. Increases in cellular c-di-GMP can repress the transcription of flagellar genes, or act post-transcriptionally to regulate swimming velocity and alter flagellar rotational switching, possibly by interacting with a yet-to-be-identified c-di-GMP receptor or with flagellar motor proteins<sup>75</sup>. Transcriptional profiling experiments revealed that high concentrations of c-di-GMP promote the transcription of *msh* (the operon encoding the MSHA pilus), the *vps* clusters and other biofilm genes, and repress the transcription of flagellar genes<sup>80</sup>. Flagellar regulatory protein A (FlrA) represses flagellar genes when it is in the c-di-GMP-bound state; however, the molecular details of c-di-GMP-mediated repression of motility are not completely understood<sup>55,81</sup>.

High cellular levels of c-di-GMP promote enhanced transcription of genes involved in biofilm formation<sup>80</sup>; for example, when bound to c-di-GMP, VpsT induces the expression of biofilm genes<sup>55</sup>. Using a series of mutant strains containing in-frame deletions of genes encoding proteins with GGDEF, EAL, or GGDEF and EAL domains, analysis revealed that strains lacking the DGCs CdgA, CdgH, CdgK, CdgL or CdgM show lower levels of *vpsL* expression and biofilm formation than their wild-type counterparts<sup>76,82</sup>. Furthermore, whereas c-di-GMP levels decreased to 54–86% of wild-type levels in each single DGC deletion strain, in the Δ5DGC strain (containing deletions in the genes encoding all five of these DGCs), c-di-GMP levels decreased to 17% of wild-type levels<sup>82</sup>. These results show that multiple DGCs are involved in maintaining cellular c-di-GMP levels and that they additively contribute to biofilm formation and *vps* gene expression, probably owing to increased c-di-GMP binding to VpsT to enable *vps* gene expression. Conversely, mutants lacking the tested PDEs (CdgJ, CdgC, RocS, MbaA and VieA) exhibited enhanced biofilm formation compared with wild type<sup>74,79,82,83</sup>.

Cellular c-di-GMP levels could be maintained by transcriptional or post-transcriptional regulation of the proteins involved in c-di-GMP signalling, and we are only just beginning to understand how the vast repertoire of *V. cholerae* DGCs and PDEs is regulated. VpsR, VpsT and HapR all seem to play a part in the regulation of these enzymes. Transcriptome studies indicate that VpsR and VpsT influence the expression of ten genes involved in the regulation of c-di-GMP levels, specifically upregulating key DGCs that enhance biofilm formation while repressing key PDEs that inhibit biofilm formation<sup>52,53</sup>. The promoter regions of genes that are predicted to be important for c-di-GMP signalling — including *cdgA*, *cdgC*, *cdgD* and VCA0165 — have predicted VpsR-binding domains, which indicates that they may be directly regulated by VpsR<sup>53,84</sup>. HapR was shown to influence the expression of 14DGCs and PDEs, and it was shown to bind directly to the promoter regions of *cdgA*, *cdgG*, VCA0080, VC2370, VC1851 and VC1086. HapR was shown to specifically upregulate PDEs that promote a decrease in biofilm formation and downregulate DGCs that have been shown to enhance biofilm formation<sup>52,62</sup>. In addition, some DGCs and PDEs seem to be regulated by the quorum sensing pathway independently of HapR, through LuxO and the Qrr sRNAs<sup>85</sup>. Environmental signals such as polyamines and bile components have also been shown to modulate the abundance and activity of enzymes involved in c-di-GMP signalling<sup>52,62,86,87</sup>.

The second messenger cAMP is involved with various cellular responses and acts as a repressor of *V. cholerae* biofilm formation<sup>88</sup> (FIG. 3b). When glucose is limited, cAMP is synthesized by adenyl cyclase (CyaA) and binds CRP to initiate the carbon catabolite repression response. The cAMP–CRP complex downregulates expression of *rbmA*, *rbmC*, *bap1*, *vpsR* and other *vps* genes<sup>89</sup>. A number of DGC and PDE genes controlling c-di-GMP levels are also regulated by cAMP–CRP; for example, *rocS*, *cdgA*, *cdgH* and *cdgI* were shown to be downregulated by cAMP–CRP<sup>89</sup>. Interestingly, all of the proteins encoded by these genes contain the GGDEF domain required for DGC activity, but RocS and CdgI also contain EAL domains and are thought to act as PDEs rather than DGCs. This further highlights the complexity of the regulatory network that governs c-di-GMP synthesis and degradation, and its influence on biofilm formation. As mentioned above, cAMP–CRP also upregulates HapR and biosynthesis of the autoinducer CAI-1 (REF. 70,89). Thus, cAMP–CRP links the nutritional status of the cell with the regulation of biofilm formation.

The *V. cholerae* stringent response is triggered by nutritional stress and results in the synthesis of the two small molecules guanosine tetraphosphate and guanosine pentaphosphate, collectively called (p)ppGpp, by the synthases RelA, SpoT and RelV<sup>58,90,91</sup>. Compared with wild-type bacteria, mutants deficient in the stringent response were shown to have reduced, although not completely abolished, biofilm formation<sup>58</sup>. All three (p)ppGpp synthases are necessary for *vpsR* transcription, but only RelA is necessary for *vpsT* transcription. Whereas the regulation of *vpsT* expression through the stringent response is strongly dependent on RpoS, the regulation of *vpsR* probably involves additional factors that remain to be identified<sup>58</sup> (FIG. 3b).

**Riboswitches**  
Regulatory RNA sensors each composed of a structured non-coding RNA that binds to specific small molecules and regulates gene expression.

**Stringent response**  
A bacterial stress response that is triggered by nutritional stress and results in the synthesis of guanosine tetraphosphate and guanosine pentaphosphate, which in turn control anabolic and catabolic processes and thereby regulate growth rate.

**(p)ppGpp**  
The collective term for the two alarmones guanosine tetraphosphate (ppGpp) and guanosine pentaphosphate (pppGpp), which are synthesized in response to nutrient limitation and other stress conditions to induce the stringent response and subsequent changes in cell physiology.

Box 3 | Small-molecule therapeutics that target Vibrio cholerae

As our understanding of the physiology of the Vibrio cholerae biofilm has evolved, new targets have emerged for the disruption of biofilm formation with small-molecule therapeutics. These compounds fall into three classes: quorum sensing inhibitors; disruptors of cyclic di-GMP (c-di-GMP) signalling; and compounds with unknown molecular targets (see the figure).

The V. cholerae quorum sensing mechanism is unlike that of many other pathogenic bacteria because both suppression of virulence factors and dispersal of biofilms are observed in the presence of high concentrations of quorum sensing signalling molecules<sup>65</sup>. Therefore, both virulence factors and biofilm formation and dispersal can theoretically be controlled using a single quorum sensing molecule mimic. A number of reports have identified compounds that are capable of targeting the response regulator LuxO and the transcriptional repressor HapR — two key regulators in the quorum sensing pathway — as well as compounds that can target steps in the production of the two known V. cholerae quorum sensing signalling molecules, namely, cholerae autoinducer 1 (CAI-1) and autoinducer 2 (AI-2)<sup>115–117</sup>. The recent discovery of Ea-CAI-1 (see the figure, part a), a biosynthetic precursor of CAI-1 that targets the CqsS receptor, has enabled the development of the more potent pyrrole analogue of this precursor. This analogue is capable of repressing transcription of the toxin-coregulated pilin (tcpA) gene, and also of activating the production of HapR to the same levels as CAI-1 when the analogue is applied at up to tenfold greater dilution than CAI-1 (REFS 118,119).

The c-di-GMP signalling system is also an attractive target for therapeutic intervention (see the figure, part b). In V. cholerae, a number of compounds have been identified that can target diguanylate cyclases (DGCs). In particular, two compounds (denoted DGC inhibitor 1 and DGC inhibitor 2 in the figure) were identified as DGC inhibitors from an initial screen of 66,000 synthetic compounds. These compounds were shown to inhibit biofilm formation under both static and flow cell culture conditions<sup>120</sup>. One compound (DGC inhibitor 1) was also shown to be effective against other biofilm-forming pathogens, including Pseudomonas aeruginosa and Staphylococcus aureus, and to inhibit the formation of biofilms on the surface of silicone catheters. A second set of seven structurally related polyaromatic inhibitors discovered from the same screening campaign were also shown to inhibit DGCs. Interestingly, only two of these compounds (denoted DGC inhibitor 3 and DGC inhibitor 4 in the figure) showed a direct relationship between a decrease in c-di-GMP concentration and a reduction in biofilm coverage, as measured by crystal violet staining<sup>121</sup>. The remaining five compounds reduced biofilm formation but not global c-di-GMP levels, which suggests that their mechanism of action involves the inhibition of specific DGCs that affect biofilm formation but not overall c-di-GMP concentration.

Whole-cell phenotypic imaging is well suited to biofilm screening because biofilm structures are of a suitable size for segmentation and quantification using standard imaging tools<sup>98,122</sup>. Coupling this technique with cellular-viability measurements permits the differentiation of bactericidal agents and compounds that selectively disrupt biofilm formation without affecting cell survival. Using this approach, two novel scaffolds have been reported: the natural product oxazine and a quinoline-based molecule (unnamed; see the figure, part c)<sup>123,124</sup>. In both cases, strategies have been developed for the synthesis of analogue libraries for these compounds to determine the structural features required for biofilm inhibition and to develop synthetic analogues with higher potencies than the original lead compounds<sup>123,125</sup>.

### a Inhibitors that target quorum sensing

![Inhibitors that target quorum sensing](https://i.imgur.com/placeholder.png)

### b Inhibitors that target c-di-GMP

![Inhibitors that target c-di-GMP](https://i.imgur.com/placeholder.png)

### c Inhibitors of unknown targets

![Inhibitors of unknown targets](https://i.imgur.com/placeholder.png)

Small RNA-mediated regulation of Vibrio cholerae biofilms. The importance of sRNAs in the regulation of cellular processes is becoming increasingly recognized (FIG. 3b). As well as the sRNAs that control HapR levels, two additional sRNAs have been shown to regulate biofilm formation in V. cholerae. VrrA, the expression of which is controlled by RpoE, negatively regulates the expression of the biofilm matrix protein RbmC by directly pairing with the 5′ end of the rbmC mRNA, thereby inhibiting the translation of rbmC and downregulating biofilm formation<sup>92,93</sup>. This is the first link between biofilm formation and RpoE, and the first example of an sRNA bypassing the master regulators of biofilm formation to directly regulate a biofilm matrix component<sup>92</sup>. Furthermore, the sRNA RyhB, which is negatively regulated by iron and Fur, is involved in biofilm formation. A ryhB mutant was shown to exhibit a defect in biofilm formation when grown in low-iron medium, but this defect was rescued by the addition of excess iron or succinate; however, the molecular basis by which RyhB controls biofilm formation remains to be determined<sup>94</sup>. Although the role of sRNAs in biofilm formation is still largely unexplored, these few known examples add another level of regulation to the elaborate network that controls biofilm development.

REVIEW S

# Conclusions and future directions

Over the past 20 years, *V. cholerae* biofilms have been extensively studied, and great strides have been made in understanding both the molecular mechanisms of biofilm formation, and the role of biofilms in environmental persistence of the pathogen and in transmission to the human host.

Mature biofilms depend on the production of extracellular matrix components — polysaccharides (VPS) and matrix proteins — to establish cell–cell interactions and attach biofilms to environmental and host surfaces. Key structural components of the biofilm matrix, including VPS, RbmA, Bap1 and RbmC, have been identified and characterized. The genes within the *vps* gene clusters, which are required for biofilm formation and VPS synthesis, have been analysed, and the structure of the repeating unit of VPS has been determined. Three matrix proteins — RbmA, RbmC and Bap1 — have been shown to influence biofilm stability and architecture through their distinct locations in the matrix and their interactions with each other.

The major biofilm regulators VpsR, VpsT, HapR and H-NS directly control the expression of structural and regulatory genes. The nucleotide second messengers c-di-GMP, cAMP and (p)ppGpp are instrumental in controlling the expression and activity of these regulators and, in turn, their regulatory targets. The role of sRNAs in quorum sensing-dependent and quorum sensing-independent regulation of biofilms is being explored and integrated into our understanding of the major regulatory pathways. What is now known about the molecular underpinnings of biofilm regulation will provide a platform for further study and discovery to gain a complete understanding of this important process.

Despite recent advances that have bolstered our knowledge of how, why and when *V. cholerae* biofilms are formed, much remains to be discovered about the functions of each biofilm matrix component and about biofilm regulation in response to signals experienced by *V. cholerae* during the intestinal and aquatic life stages. Biological factors — such as growth in mixed-species biofilms, predation by protozoa and infection by bacteriophages — can have an impact on biofilm formation, but our knowledge about the effects of such factors is limited and should be expanded. Similarly, more work is required to elucidate the mechanisms and regulation of biofilm dispersal as well as the role of biofilms *in vivo*.

Nevertheless, as our understanding of *V. cholerae* biofilm physiology has evolved, a number of new key players have emerged that are not only crucial for biofilm development, but also suitable candidates for targeting with small-molecule therapeutics. These developments, coupled with continued improvements in biofilm screening technologies, are now providing medicinal chemists with a toolbox of screening strategies for the discovery of small-molecule inhibitors of biofilm formation (BOX 3). Known biofilm-inhibitory compounds fall into three main classes: quorum sensing inhibitors; disruptors of c-di-GMP signalling; and compounds with unknown targets that were discovered through unbiased biofilm imaging methods. Thus far, several compounds have shown promise for biofilm inhibition and treatment. Further exploration of the use of small molecules to target and inhibit biofilm formation may lead to the discovery of new therapeutics and better equip us to prevent and treat deadly cholera outbreaks.

---

1. Charles, R. C. & Ryan, E. T. Cholera in the 21st century. *Curr. Opin. Infect. Dis.* **24**, 472–477 (2011).
2. Islam, M. S. *et al.* Biofilm acts as a microenvironment for plankton-associated *Vibrio cholerae* in the aquatic environment of Bangladesh. *Microbiol. Immunol.* **51**, 369–379 (2007).
3. Faruque, S. M. *et al.* Transmissibility of cholera: *in vivo*-formed biofilms and their relationship to infectivity and persistence in the environment. *Proc. Natl Acad. Sci. USA* **103**, 6350–6355 (2006). This study demonstrates that the ingestion of metabolically quiescent *V. cholerae* cells can lead to the formation of *in vivo* biofilms that are hyperinfective when excreted in patient stool.
4. Alam, M. *et al.* Viable but nonculturable *Vibrio cholerae* O1 in biofilms in the aquatic environment and their role in cholera transmission. *Proc. Natl Acad. Sci. USA* **104**, 17801–17806 (2007). This report reveals the crucial role of biofilms in maintaining environmental reservoirs between epidemics as well as the role of host passage in the activation of metabolically quiescent cells.
5. Matz, C. *et al.* Biofilm formation and phenotypic variation enhance predation-driven persistence of *Vibrio cholerae*. *Proc. Natl Acad. Sci. USA* **102**, 16819–16824 (2005).
6. Beyhan, S. & Yildiz, F. H. Smooth to rugose phase variation in *Vibrio cholerae* can be mediated by a single nucleotide change that targets c-di-GMP signalling pathway. *Mol. Microbiol.* **63**, 995–1007 (2007).
7. Tamplin, M. L., Gauzens, A. L., Huq, A., Sack, D. A. & Colwell, R. R. Attachment of *Vibrio cholerae* serogroup O1 to zooplankton and phytoplankton of Bangladesh waters. *Appl. Environ. Microbiol.* **56**, 1977–1980 (1990).
8. Rawlings, T. K., Ruiz, G. M. & Colwell, R. R. Association of *Vibrio cholerae* O1 El Tor and O139 Bengal with the Copepods *Acartia tonsa* and *Eurytemora affinis*. *Appl. Environ. Microbiol.* **73**, 7926–7933 (2007).
9. Meibom, K. L. *et al.* The *Vibrio cholerae* chitin utilization program. *Proc. Natl Acad. Sci. USA* **101**, 2524–2529 (2004).
10. Li, X. & Roseman, S. The chitinolytic cascade in Vibrios is regulated by chitin oligosaccharides and a two-component chitin catabolic sensor/kinase. *Proc. Natl Acad. Sci. USA* **101**, 627–631 (2004).
11. Meibom, K. L., Blokesch, M., Dolganov, N. A., Wu, C.-Y. & Schoolnik, G. K. Chitin induces natural competence in *Vibrio cholerae*. *Science* **310**, 1824–1827 (2005).
12. Lipp, E. K., Huq, A. & Colwell, R. R. Effects of global climate on infectious disease: the cholera model. *Clin. Microbiol. Rev.* **15**, 757–770 (2002).
13. Huq, A. *et al.* Critical factors influencing the occurrence of *Vibrio cholerae* in the environment of Bangladesh. *Appl. Environ. Microbiol.* **71**, 4645–4654 (2005).
14. Colwell, R. R. *et al.* Reduction of cholera in Bangladeshi villages by simple filtration. *Proc. Natl Acad. Sci. USA* **100**, 1051–1055 (2003). This work shows that simple filtration to remove particulates > 20 μm could reduce cholera outbreaks by 48%, suggesting that the removal of biofilms can significantly reduce *V. cholerae* infections.
15. Bari, S. M. N. *et al.* Quorum-sensing autoinducers resuscitate dormant *Vibrio cholerae* in environmental water samples. *Proc. Natl Acad. Sci. USA* **110**, 9926–9931 (2013).
16. Colwell, R. R. *et al.* Viable but non-culturable *Vibrio cholerae* O1 revert to a cultivable state in the human intestine. *World J. Microbiol. Biotechnol.* **12**, 28–31 (1996).
17. Tamayo, R., Patimalla, B. & Camilli, A. Growth in a biofilm induces a hyperinfectious phenotype in *Vibrio cholerae*. *Infect. Immun.* **78**, 3560–3569 (2010).

18. Nielsen, A. T. *et al.* A bistable switch and anatomical site control *Vibrio cholerae* virulence gene expression in the intestine. *PLoS Pathog.* **6**, e1001102 (2010).
19. Hang, L. *et al.* Use of *in vivo*-induced antigen technology (IVIAT) to identify genes uniquely expressed during human infection with *Vibrio cholerae*. *Proc. Natl Acad. Sci. USA* **100**, 8508–8513 (2003).
20. Lombardo, M.-J. *et al.* An *in vivo* expression technology screen for *Vibrio cholerae* genes expressed in human volunteers. *Proc. Natl Acad. Sci. USA* **104**, 18229–18234 (2007).
21. Fong, J. C. N., Syed, K. A., Klose, K. E. & Yildiz, F. H. Role of *Vibrio* polysaccharide (*vps*) genes in VPS production, biofilm formation and *Vibrio cholerae* pathogenesis. *Microbiology* **156**, 2757–2769 (2010).
22. Utada, A. S. *et al.* *Vibrio cholerae* use pili and flagella synergistically to effect motility switching and conditional surface attachment. *Nature Commun.* **5**, 4913 (2014). This paper reveals the unique near-surface motility behaviours and attachment steps of *V. cholerae*.
23. Berke, A., Turner, L., Berg, H. & Lauga, E. Hydrodynamic attraction of swimming microorganisms by surfaces. *Phys. Rev. Lett.* **101**, 038102 (2008).
24. Kojima, S., Yamamoto, K., Kawagishi, I. & Homma, M. The polar flagellar motor of *Vibrio cholerae* is driven by an Na⁺ motive force. *J. Bacteriol.* **181**, 1927–1930 (1999).
25. Lauga, E., DiLuzio, W. R., Whitesides, G. M. & Stone, H. A. Swimming in circles: motion of bacteria near solid boundaries. *Biophys. J.* **90**, 400–412 (2006).
26. Watnick, P. I. & Kolter, R. Steps in the development of a *Vibrio cholerae* El Tor biofilm. *Mol. Microbiol.* **34**, 586–595 (1999).
27. Gibiansky, M. L. *et al.* Bacteria use type IV pili to walk upright and detach from surfaces. *Science* **330**, 197 (2010).

28. O'Toole, G. A. & Kolter, R. Flagellar and twitching motility are necessary for *Pseudomonas aeruginosa* biofilm development. *Mol. Microbiol.* **30**, 295–304 (1998).

29. Zhao, K. *et al.* Psl trails guide exploration and microcolony formation in *Pseudomonas aeruginosa* biofilms. *Nature* **497**, 388–391 (2013).

30. Gloag, E. S. *et al.* Self-organization of bacterial biofilms is facilitated by extracellular DNA. *Proc. Natl Acad. Sci. USA* **110**, 11541–11546 (2013).

31. Watnick, P. I., Lauriano, C. M., Klose, K. E., Croal, L. & Kolter, R. The absence of a flagellum leads to altered colony morphology, biofilm development and virulence in *Vibrio cholerae* O139. *Mol. Microbiol.* **39**, 223–235 (2001).

32. Lauriano, C. M., Ghosh, C., Correa, N. E. & Klose, K. E. The sodium-driven flagellar motor controls exopolysaccharide expression in *Vibrio cholerae*. *J. Bacteriol.* **186**, 4864–4874 (2004).

33. Reichhardt, C., Fong, J. C. N., Yildiz, F. & Cegelski, L. Characterization of the *Vibrio cholerae* extracellular matrix: a top-down solid-state NMR approach. *Biochim. Biophys. Acta* **1848**, 378–383 (2015).

34. Yildiz, F. H. & Schoolnik, G. K. *Vibrio cholerae* O1 El Tor: identification of a gene cluster required for the rugose colony type, exopolysaccharide production, chlorine resistance, and biofilm formation. *Proc. Natl Acad. Sci. USA* **96**, 4028–4033 (1999).

35. Yildiz, F., Fong, J., Sadovskaya, I., Grard, T. & Vinogradov, E. Structural characterization of the extracellular polysaccharide from *Vibrio cholerae* O1 El-Tor. *PLoS ONE* **9**, e86751 (2014). The article reports, for the first time, the chemical structure of the polysaccharide portion of VPS and introduces a new method for the reliable isolation of VPS from extracellular material.

36. Berk, V. *et al.* Molecular architecture and assembly principles of *Vibrio cholerae* biofilms. *Science* **337**, 236–239 (2012). This study uses an *in vivo* labelling strategy to visualize and characterize the spatial and temporal organization of the major *V. cholerae* biofilm components — VPS, RbmA, Bap1 and RbmC — revealing unique roles for each within the biofilm.

37. Fong, J. C. N. & Yildiz, F. H. The *rbmBCDEF* gene cluster modulates development of rugose colony morphology and biofilm formation in *Vibrio cholerae*. *J. Bacteriol.* **189**, 2319–2330 (2007).

38. Fong, J. C. N., Karplus, K., Schoolnik, G. K. & Yildiz, F. H. Identification and characterization of RbmA, a novel protein required for the development of rugose colony morphology and biofilm structure in *Vibrio cholerae*. *J. Bacteriol.* **188**, 1049–1059 (2006).

39. Nesper, J. *et al.* Characterization of *Vibrio cholerae* O1 El tor *galU* and *galE* mutants: influence on lipopolysaccharide structure, colonization, and biofilm formation. *Infect. Immun.* **69**, 435–445 (2001).

40. Absalon, C., Van Dellen, K. & Watnick, P. I. A communal bacterial adhesin anchors biofilm and bystander cells to surfaces. *PLoS Pathog.* **7**, e1002210 (2011).

41. Giglio, K. M., Fong, J. C., Yildiz, F. H. & Sondermann, H. Structural basis for biofilm formation via the *Vibrio cholerae* matrix protein RbmA. *J. Bacteriol.* **195**, 3277–3286 (2013).

42. Maestre-Reyna, M., Wu, W.-J. & Wang, A. H.-J. Structural insights into RbmA, a biofilm scaffolding protein of *V. cholerae*. *PLoS ONE* **8**, e82458 (2013).

43. Hollenbeck, E. C. *et al.* Molecular determinants of mechanical properties of *V. cholerae* biofilms at the air-liquid interface. *Biophys. J.* **107**, 2245–2252 (2014).

44. Duperthuy, M. *et al.* Role of the *Vibrio cholerae* matrix protein Bap1 in cross-resistance to antimicrobial peptides. *PLoS Pathog.* **9**, e1003620 (2013).

45. Levan, S. R. & Olson, R. Determination of the carbohydrate-binding specificity of lectin-like domains in *Vibrio cholerae* cytolysin. *Biophys. J.* **102**, 461a–462a (2012).

46. Johnson, T. L., Fong, J. C. N., Rule, C., Yildiz, F. H. & Sandkvist, M. The type II secretion system delivers matrix proteins for biofilm formation by *Vibrio cholerae*. *J. Bacteriol.* **196**, 4245–4252 (2014).

47. Altindis, E., Fu, Y. & Mekalanos, J. J. Proteomic analysis of *Vibrio cholerae* outer membrane vesicles. *Proc. Natl Acad. Sci. USA* **111**, E1548–E1556 (2014). This research shows that all three major biofilm matrix proteins are associated with OMVs, which implies that OMVs may have an as-yet-uncharacterized role in biofilm formation.

48. Seper, A. *et al.* Extracellular nucleases and extracellular DNA play important roles in *Vibrio cholerae* biofilm formation. *Mol. Microbiol.* **82**, 1015–1037 (2011). This work identifies two extracellular nucleases, DNase and Xds, and characterizes their role in the development of biofilm architecture, nutrient acquisition and biofilm dispersal, revealing previously unknown roles for extracellular nucleases and DNA in biofilm formation.

49. Hay, A. J. & Zhu, J. Host intestinal signal-promoted biofilm dispersal induces *Vibrio cholerae* colonization. *Infect. Immun.* **83**, 317–323 (2015).

50. Yildiz, F. H., Dolganov, N. A. & Schoolnik, G. K. VpsR, a member of the response regulators of the two-component regulatory systems, is required for expression of *vps* biosynthesis genes and EPS<sup>Et</sup>-associated phenotypes in *Vibrio cholerae* O1 El Tor. *J. Bacteriol.* **183**, 1716–1726 (2001).

51. Zamorano-Sánchez, D., Fong, J. C. N., Kilic, S., Erill, I. & Yildiz, F. H. Identification and characterization of VpsR and VpsT binding sites in *Vibrio cholerae*. *J. Bacteriol.* **197**, 1221–1235 (2015).

52. Beyhan, S., Bilecen, K., Salama, S. R., Casper-Lindley, C. & Yildiz, F. H. Regulation of rugosity and biofilm formation in *Vibrio cholerae*: comparison of VpsT and VpsR regulons and epistasis analysis of *vpsT*, *vpsR*, and *hapR*. *J. Bacteriol.* **189**, 388–402 (2007).

53. Yildiz, F. H., Liu, X. S., Heydorn, A. & Schoolnik, G. K. Molecular analysis of rugosity in a *Vibrio cholerae* O1 El Tor phase variant. *Mol. Microbiol.* **53**, 497–515 (2004).

54. Srivastava, D., Harris, R. C. & Waters, C. M. Integration of cyclic di-GMP and quorum sensing in the control of *vpsT* and *aphA* in *Vibrio cholerae*. *J. Bacteriol.* **193**, 6331–6341 (2011).

55. Krasteva, P. V. *et al.* *Vibrio cholerae* VpsT regulates matrix production and motility by directly sensing cyclic di-GMP. *Science* **327**, 866–868 (2010). This article reports the crystal structure of VpsT and demonstrates that VpsT dimerizes and directly binds a c-di-GMP dimer to activate the transcriptional regulation of genes involved in biofilm regulation.

56. Yang, M., Frey, E. M., Liu, Z., Bishar, R. & Zhu, J. The virulence transcriptional activator AphA enhances biofilm formation by *Vibrio cholerae* by activating expression of the biofilm regulator VpsT. *Infect. Immun.* **78**, 697–703 (2010).

57. Casper-Lindley, C. & Yildiz, F. H. VpsT is a transcriptional regulator required for expression of *vps* biosynthesis genes and the development of rugose colonial morphology in *Vibrio cholerae* O1 El Tor. *J. Bacteriol.* **186**, 1574–1578 (2004).

58. He, H., Cooper, J. N., Mishra, A. & Raskin, D. M. Stringent response regulation of biofilm formation in *Vibrio cholerae*. *J. Bacteriol.* **194**, 2962–2972 (2012).

59. Zhu, J. & Mekalanos, J. J. Quorum sensing-dependent biofilms enhance colonization in *Vibrio cholerae*. *Dev. Cell* **5**, 647–656 (2003).

60. Hammer, B. K. & Bassler, B. L. Quorum sensing controls biofilm formation in *Vibrio cholerae*. *Mol. Microbiol.* **50**, 101–104 (2003).

61. Jobling, M. G. & Holmes, R. K. Characterization of *hapR*, a positive regulator of the *Vibrio cholerae* HA/protease gene *hap*, and its identification as a functional homologue of the *Vibrio harveyi luxR* gene. *Mol. Microbiol.* **26**, 1023–1034 (1997).

62. Waters, C. M., Lu, W., Rabinowitz, J. D. & Bassler, B. L. Quorum sensing controls biofilm formation in *Vibrio cholerae* through modulation of cyclic di-GMP levels and repression of *vpsT*. *J. Bacteriol.* **190**, 2527–2536 (2008).

63. De Silva, R. S. *et al.* Crystal structure of the *Vibrio cholerae* quorum-sensing regulatory protein HapR. *J. Bacteriol.* **189**, 5683–5691 (2007).

64. Liu, Z., Stirling, F. R. & Zhu, J. Temporal quorum-sensing induction regulates *Vibrio cholerae* biofilm architecture. *Infect. Immun.* **75**, 122–126 (2007).

65. Ng, W.-L. & Bassler, B. L. Bacterial quorum-sensing network architectures. *Annu. Rev. Genet.* **43**, 197–222 (2009).

66. Tsou, A. M., Liu, Z., Cai, T. & Zhu, J. The VarS/VarA two-component system modulates the activity of the *Vibrio cholerae* quorum-sensing transcriptional regulator HapR. *Microbiology* **157**, 1620–1628 (2011).

67. Lenz, D. H., Miller, M. B., Zhu, J., Kulkarni, R. V. & Bassler, B. L. CsrA and three redundant small RNAs regulate quorum sensing in *Vibrio cholerae*. *Mol. Microbiol.* **58**, 1186–1202 (2005).

68. Lenz, D. H. & Bassler, B. L. The small nucleoid protein Fis is involved in *Vibrio cholerae* quorum sensing. *Mol. Microbiol.* **63**, 859–871 (2007).

69. Shikuma, N. J. *et al.* Overexpression of VpsS, a hybrid sensor kinase, enhances biofilm formation in *Vibrio cholerae*. *J. Bacteriol.* **191**, 5147–5158 (2009).

70. Liang, W., Pascual-Montano, A., Silva, A. J. & Benitez, J. A. The cyclic AMP receptor protein modulates quorum sensing, motility and multiple genes that affect intestinal colonization in *Vibrio cholerae*. *Microbiology* **153**, 2964–2975 (2007).

71. Liu, Z., Hsiao, A., Joelsson, A. & Zhu, J. The transcriptional regulator VqmA increases expression of the quorum-sensing activator HapR in *Vibrio cholerae*. *J. Bacteriol.* **188**, 2446–2453 (2006).

72. Wang, H., Ayala, J. C., Silva, A. J. & Benitez, J. A. The histone-like nucleoid structuring protein (H-NS) is a repressor of *Vibrio cholerae* exopolysaccharide biosynthesis (*vps*) genes. *Appl. Environ. Microbiol.* **78**, 2482–2488 (2012).

73. Stonehouse, E. A., Hulbert, R. R., Nye, M. B., Skorupski, K. & Taylor, R. K. H-NS binding and repression of the *ctx* promoter in *Vibrio cholerae*. *J. Bacteriol.* **193**, 979–988 (2011).

74. Tischler, A. D. & Camilli, A. Cyclic diguanylate (c-di-GMP) regulates *Vibrio cholerae* biofilm formation. *Mol. Microbiol.* **53**, 857–869 (2004). This report identifies and characterizes the phosphodiesterase VieA, and is the first study to demonstrate the significant role c-di-GMP in regulating biofilm formation.

75. Römling, U., Galperin, M. Y. & Gomelsky, M. Cyclic di-GMP: the first 25 years of a universal bacterial second messenger. *Microbiol. Mol. Biol. Rev.* **77**, 1–52 (2013).

76. Beyhan, S., Odell, L. S. & Yildiz, F. H. Identification and characterization of cyclic diguanylate signaling systems controlling rugosity in *Vibrio cholerae*. *J. Bacteriol.* **190**, 7392–7405 (2008).

77. Pratt, J. T., Tamayo, R., Tischler, A. D. & Camilli, A. PilZ domain proteins bind cyclic diguanylate and regulate diverse processes in *Vibrio cholerae*. *J. Biol. Chem.* **282**, 12860–12870 (2007).

78. Sudarsan, N. *et al.* Riboswitches in eubacteria sense the second messenger cyclic di-GMP. *Science* **321**, 411–413 (2008).

79. Liu, X., Beyhan, S., Lim, B., Linington, R. G. & Yildiz, F. H. Identification and characterization of a phosphodiesterase that inversely regulates motility and biofilm formation in *Vibrio cholerae*. *J. Bacteriol.* **192**, 4541–4552 (2010).

80. Beyhan, S., Tischler, A. D., Camilli, A. & Yildiz, F. H. Transcriptome and phenotypic responses of *Vibrio cholerae* to increased cyclic di-GMP level. *J. Bacteriol.* **188**, 3600–3613 (2006).

81. Srivastava, D., Hsieh, M.-L., Khataokar, A., Neiditch, M. B. & Waters, C. M. Cyclic di-GMP inhibits *Vibrio cholerae* motility by repressing induction of transcription and inducing extracellular polysaccharide production. *Mol. Microbiol.* **90**, 1262–1276 (2013).

82. Shikuma, N. J., Fong, J. C. N. & Yildiz, F. H. Cellular levels and binding of c-di-GMP control subcellular localization and activity of the *Vibrio cholerae* transcriptional regulator VpsT. *PLoS Pathog.* **8**, e1002719 (2012).

83. Lim, B., Beyhan, S. & Yildiz, F. H. Regulation of *Vibrio* polysaccharide synthesis and virulence factor production by CdgC, a GGDEF-EAL domain protein, in *Vibrio cholerae*. *J. Bacteriol.* **189**, 717–729 (2007).

84. Beyhan, S. & Yildiz, F. H. in *The Second Messenger Cyclic Di-GMP* (eds Wolfe, A. J. & Visick, K. L.) 253–269 (American Society for Microbiology Press, 2010).

85. Hammer, B. K. & Bassler, B. L. Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic *Vibrio cholerae*. *Proc. Natl Acad. Sci. USA* **104**, 11145–11149 (2007).

86. Koestler, B. J. & Waters, C. M. Bile acids and bicarbonate inversely regulate intracellular cyclic di-GMP in *Vibrio cholerae*. *Infect. Immun.* **82**, 3002–3014 (2014).

87. Cockerell, S. R. *et al.* *Vibrio cholerae* NspS, a homologue of ABC-type periplasmic solute binding proteins, facilitates transduction of polyamine signals independent of their transport. *Microbiology* **160**, 832–843 (2014).

88. Liang, W., Silva, A. J. & Benitez, J. A. The cyclic AMP receptor protein modulates colonial morphology in *Vibrio cholerae*. *Appl. Environ. Microbiol.* **73**, 7482–7487 (2007).

REVIEWS

89. Fong, J. C. N. & Yildiz, F. H. Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in *Vibrio cholerae* biofilm formation. *J. Bacteriol.* **190**, 6646–6659 (2008).

90. Das, B., Pal, R. R., Bag, S. & Bhadra, R. K. Stringent response in *Vibrio cholerae*: genetic analysis of *spoT* gene function and identification of a novel (p)ppGpp synthetase gene. *Mol. Microbiol.* **72**, 380–398 (2009).

91. Raskin, D. M., Judson, N. & Mekalanos, J. J. Regulation of the stringent response is the essential function of the conserved bacterial G protein CgtA in *Vibrio cholerae*. *Proc. Natl Acad. Sci. USA* **104**, 4636–4641 (2007).

92. Song, T. *et al.* *Vibrio cholerae* utilizes direct sRNA regulation in expression of a biofilm matrix protein. *PLoS ONE* **9**, e101280 (2014). This study shows that a conserved sRNA, VrrA, binds to and inhibits the translation of *rbmC* mRNA, thus providing the first example of direct regulation of a biofilm matrix component by an sRNA.

93. Song, T. *et al.* A new *Vibrio cholerae* sRNA modulates colonization and affects release of outer membrane vesicles. *Mol. Microbiol.* **70**, 100–111 (2008).

94. Mey, A. R., Craig, S. A. & Payne, S. M. Characterization of *Vibrio cholerae* RyhB: the RyhB regulon and role of *ryhB* in biofilm formation. *Infect. Immun.* **73**, 5706–5719 (2005).

95. Kaper, J. B., Morris, J. G. & Levine, M. M. Cholera. *Clin. Microbiol. Rev.* **8**, 48–86 (1995).

96. Balteanu, I. The receptor structure of *Vibrio cholerae* (*V. comma*) with observations on variations in cholera and cholera-like organisms. *J. Pathol. Bacteriol.* **29**, 251–277 (1926).

97. White, P. B. The rugose variant of vibrios. *J. Pathol. Bacteriol.* **46**, 1–6 (1938).

98. Peach, K. C. *et al.* An image-based 384-well high-throughput screening method for the discovery of biofilm inhibitors in *Vibrio cholerae*. *Mol. Biosyst.* **7**, 1176–1184 (2011). This investigation uses high-throughput epifluorescence microscopy imaging to evaluate a 3080-member small-molecule library and identifies inhibitors of *V. cholerae* biofilm formation, leading to the discovery of potential inhibitors.

99. Heydorn, A. *et al.* Quantification of biofilm structures by the novel computer program COMSTAT. *Microbiology* **146**, 2395–2407 (2000).

100. Houot, L., Chang, S., Absalon, C. & Watnick, P. I. *Vibrio cholerae* phosphoenolpyruvate phosphotransferase system control of carbohydrate transport, biofilm formation, and colonization of the germfree mouse intestine. *Infect. Immun.* **78**, 1482–1494 (2010).

101. Houot, L., Chang, S., Pickering, B. S., Absalon, C. & Watnick, P. I. The phosphoenolpyruvate phosphotransferase system regulates *Vibrio cholerae* biofilm formation through multiple independent pathways. *J. Bacteriol.* **192**, 3055–3067 (2010).

102. Ymele-Leki, P., Houot, L. & Watnick, P. I. Mannitol and the mannitol-specific enzyme IIB subunit activate *Vibrio cholerae* biofilm formation. *Appl. Environ. Microbiol.* **79**, 4675–4683 (2013).

103. Kapfhammer, D., Karatan, E., Pflughoeft, K. J. & Watnick, P. I. Role for glycine betaine transport in *Vibrio cholerae* osmoadaptation and biofilm formation within microbial communities. *Appl. Environ. Microbiol.* **71**, 3840–3847 (2005).

104. Shikuma, N. J. & Yildiz, F. H. Identification and characterization of OscR, a transcriptional regulator involved in osmolarity adaptation in *Vibrio cholerae*. *J. Bacteriol.* **191**, 4082–4096 (2009).

105. Shikuma, N. J., Davis, K. R., Fong, J. N. C. & Yildiz, F. H. The transcriptional regulator, CosR, controls compatible solute biosynthesis and transport, motility and biofilm formation in *Vibrio cholerae*. *Environ. Microbiol.* **15**, 1387–1399 (2013).

106. Correll, D. L. Phosphorus: a rate limiting nutrient in surface waters. *Poult. Sci.* **78**, 674–682 (1999).

107. Jahid, I. K., Silva, A. J. & Benitez, J. A. Polyphosphate stores enhance the ability of *Vibrio cholerae* to overcome environmental stresses in a low-phosphate environment. *Appl. Environ. Microbiol.* **72**, 7043–7049 (2006).

108. Pratt, J. T., McDonough, E. & Camilli, A. PhoB regulates motility, biofilms, and cyclic di-GMP in *Vibrio cholerae*. *J. Bacteriol.* **191**, 6632–6642 (2009).

109. Von Kruger, W. M. A., Humphreys, S. & Ketley, J. M. A role for the PhoBR regulatory system homologue in the *Vibrio cholerae* phosphate-limitation response and intestinal colonization. *Microbiology* **145**, 2463–2475 (1999).

110. Sultan, S. Z., Silva, A. J. & Benitez, J. A. The PhoB regulatory system modulates biofilm formation and stress response in El Tor biotype *Vibrio cholerae*. *FEMS Microbiol. Lett.* **302**, 22–31 (2010).

111. Bilecen, K. & Yildiz, F. H. Identification of a calcium-controlled negative regulatory system affecting *Vibrio cholerae* biofilm formation. *Environ. Microbiol.* **11**, 2015–2029 (2009).

112. Kierek, K. & Watnick, P. I. The *Vibrio cholerae* O139 O-antigen polysaccharide is essential for Ca<sup>2+</sup>-dependent biofilm development in sea water. *Proc. Natl Acad. Sci. USA* **100**, 14357–14362 (2003).

113. Mueller, R. S., Beyhan, S., Saini, S. G., Yildiz, F. H. & Bartlett, D. H. Indole acts as an extracellular cue regulating gene expression in *Vibrio cholerae*. *J. Bacteriol.* **191**, 3504–3516 (2009).

114. Hung, D. T., Zhu, J., Sturtevant, D. & Mekalanos, J. J. Bile acids stimulate biofilm formation in *Vibrio cholerae*. *Mol. Microbiol.* **59**, 193–201 (2006).

115. Ng, W. -L., Perez, L., Cong, J., Semmelhack, M. F. & Bassler, B. L. Broad spectrum pro-quorum-sensing molecules as inhibitors of virulence in vibrios. *PLoS Pathog.* **8**, e1002767 (2012).

116. Brackman, G. *et al.* Structure-activity relationship of cinnamaldehyde analogs as inhibitors of AI-2 based quorum sensing and their effect on virulence of *Vibrio* spp. *PLoS ONE* **6**, e16084 (2011).

117. Gutierrez, J. A. *et al.* Transition state analogs of 5′-methylthioadenosine nucleosidase disrupt quorum sensing. *Nature Chem. Biol.* **5**, 251–257 (2009).

118. Wei, Y., Perez, L. J., Ng, W.-L., Semmelhack, M. F. & Bassler, B. L. Mechanism of *Vibrio cholerae* autoinducer-1 biosynthesis. *ACS Chem. Biol.* **6**, 356–365 (2011).

119. Perez, L. J., Karagounis, T. K., Hurley, A., Bassler, B. L. & Semmelhack, M. F. Highly potent, chemically stable quorum sensing agonists for *Vibrio cholerae*. *Chem. Sci.* **5**, 151–155 (2014).

120. Sambanthamoorthy, K. *et al.* Identification of a novel benzimidazole that inhibits bacterial biofilm formation in a broad-spectrum manner. *Antimicrob. Agents Chemother.* **55**, 4369–4378 (2011).

121. Sambanthamoorthy, K. *et al.* Identification of small molecules that antagonize diguanylate cyclase enzymes to inhibit biofilm formation. *Antimicrob. Agents Chemother.* **56**, 5202–5211 (2012).

122. Navarro, G. *et al.* Image-based 384-well high-throughput screening method for the discovery of skyllamycins A to C as biofilm inhibitors and inducers of biofilm detachment in *Pseudomonas aeruginosa*. *Antimicrob. Agents Chemother.* **58**, 1092–1099 (2014).

123. León, B. *et al.* Development of quinoline-based disruptors of biofilm formation against *Vibrio cholerae*. *Org. Lett.* **15**, 1234–1237 (2013).

124. Peach, K. C., Cheng, A. T., Oliver, A. G., Yildiz, F. H. & Linington, R. G. Discovery and biological characterization of the auromomycin chromophore as an inhibitor of biofilm formation in *Vibrio cholerae*. *ChemBioChem* **14**, 2209–2215 (2013).

125. Warner, C. J. A., Cheng, A. T., Yildiz, F. H. & Linington, R. G. Development of benzo[1,4]oxazines as biofilm inhibitors and dispersal agents against *Vibrio cholerae*. *Chem. Commun. (Camb.)* **51**, 1305–1308 (2014).

Acknowledgements  
The authors thank L. Lang and A. Gurcan for figure preparation. They thank N. Fong for the photographs of *V. cholerae* colony, biofilm and pellicle. The authors also thank members of our laboratories for critical reading of the manuscript. This work was supported by the US National Institutes of Health (NIH) grants AI102584, AI114261 and AI055987.

Competing interests statement  
The authors declare no competing interests.
