
Cell. Mol. Life Sci.
DOI 10.1007/s00018-015-2087-8

**Cellular and Molecular Life Sciences**

---

### REVIEW

#### Mitophagy programs: mechanisms and physiological implications of mitochondrial targeting by autophagy

Anne Hamacher-Brady${}^{1,4}$ · Nathan Ryan Brady${}^{2,3,4}$

Received: 20 May 2015 / Revised: 30 October 2015 / Accepted: 2 November 2015  
© The Author(s) 2015. This article is published with open access at Springerlink.com

---

#### Abstract
Mitochondria are an essential source of ATP for cellular function, but when damaged, mitochondria generate a plethora of stress signals, which lead to cellular dysfunction and eventually programmed cell death. Thus, a major component of maintaining cellular homeostasis is the recognition and removal of dysfunctional mitochondria through autophagy-mediated degradation, i.e., mitophagy. Mitophagy further constitutes a developmental program, and undergoes a high degree of crosstalk with apoptosis. Reduced mitochondrial quality control is linked to disease pathogenesis, suggesting the importance of process elucidation as a clinical target. Recent work has revealed multiple mitophagy programs that operate independently or undergo crosstalk, and require modulated autophagy receptor activities at outer membranes of mitochondria. Here, we review these mitophagy programs, focusing on pathway mechanisms which recognize and target mitochondria for sequestration by autophagosomes, as well as mechanisms controlling pathway activities. Furthermore, we provide an introduction to the currently available methods for detecting mitophagy.

#### Keywords
Bnip3 · FUNDC1 · LC3-interacting region (LIR) · Macroautophagy · Mitophagy · Nix · Parkin E3 ligase · Ubiquitin

#### Abbreviations
- Atg: Autophagy-related protein
- GABARAP: Gamma-aminobutyric acid receptor-associated protein
- HIF1: Hypoxia-inducible factor 1
- IMM: Inner mitochondrial membrane
- LC3: Microtubule-associated protein light chain 3
- LIR: LC3-interacting region
- mTOR: Mammalian target of rapamycin
- OMM: Outer mitochondrial membrane
- PE: Phosphatidylethanolamine
- ROS: Reactive oxygen species
- TCA: Tricarboxylic acid
- TFEB: Transcription factor EB

---

#### Introduction

Mitochondria are organelles surrounded by a double membrane, comprised of the outer mitochondrial membrane (OMM) and the inner mitochondrial membrane (IMM) (Fig. 1a). Mitochondria are abundant in most cell types, and occupy approximately 10–40% of cellular volume [1]. For example, the mitochondrial population accounts for approximately 30% of the cellular volume of HeLa cells, and for 22–37% of the cardiac cell volume [2]. Furthermore, depending on cell types, mitochondrial morphologies [3] and numbers vary considerably. For example, cardiac myocytes contain several thousand morphologically similar mitochondria [4], while neurons carry several

---

Published online: 26 November 2015

A Outer mitochondrial membrane (OMM)
Inner mitochondrial membrane (IMM)
Matrix
Intermembrane space (IMS)
Electron transport chain

B
Reactive Oxygen Species (ROS)
Physiological
- Differentiation
- Antioxidant response
- Metabolic adaptation
Pathophysiological
- Lipid peroxidation
- DNA damage
- Protein modification
- Protein inactivation
- Cell death

C
Complex I
NADH
TCA Cycle
Complex II
FADH₂
Complex III
Q
Cyto c
Complex IV
O₂
H₂O
Complex V
ADP + Pi
ATP

gradient across the IMM that is used to produce ATP [11] (Fig. 1c). Ultimately, electrons are transferred to molecular oxygen (O₂), reducing it to H₂O. The coupling of these processes is termed mitochondrial oxidative phosphorylation, aka cellular respiration. Due to leakage of electrons at complex I or complex III of the electron transport chain, O₂ can be incompletely reduced and generate the superoxide anion, the precursor to most ROS [12]. Low levels of ROS play physiological roles [6], while high and/or prolonged elevations of ROS can oxidize proteins, lipids, and nucleic acids, leading to cellular dysfunction and programmed cell death [13].

Damaged mitochondria can signal programmed cell death [8], inflammation and aging [14]. Enhanced levels of damaged mitochondria aggravate many diseases, and participate in disease pathogenesis [15–17]. To maintain homeostasis of the mitochondrial population, cells rely on autophagy, a quality control process by which components of the cytoplasm are sequestered and delivered to lysosomes for degradation [18]. Dysfunctional mitochondria can be recognized and targeted for degradation by a specific mode of autophagy, termed mitophagy. Experimentally this has been evidenced in cell culture and in vivo models, whereby genetic knockout of autophagy proteins results in increased mitochondrial mass, and increased numbers of dysfunctional mitochondria and ROS levels [19, 20]. While targeting of mitochondria to lysosomes was detected over 50 years ago [21], recent years have yielded rapid progress in elucidating mechanisms which underlie distinct mitophagy programs. In this review, we focus on the mechanisms which orchestrate the targeting of the autophagic machinery to the OMM, discuss physiological roles of mitophagy, and present an overview of methods used to evidence mitophagic activity.

Fig. 1 Mitochondrial function and dysfunction. a A mitochondrion is enclosed by two membranes, the outer mitochondrial membrane (OMM) and inner mitochondrial membrane (IMM). The mitochondrial compartment between the OMM and IMM is referred to as the intermembrane space (IMS). The respiratory chain (electron transport chain, ETC), which is composed of complexes I–V and localized in the IMM, drives ATP synthesis in the mitochondrial matrix. b ROS are implicated in both physiological and pathophysiological signaling. c Electrons from the tricarboxylic acid (TCA) cycle substrates are transferred through the respiratory chain complexes (along the yellow arrows), driving the extrusion of protons (+) from the matrix, thereby generating the proton motive force. Proton flow through the ATP synthase (complex V) drives ATP production. Oxygen (O₂) serves as the terminal electron acceptor at complex IV, forming H₂O. At complexes I and/or III electron leak can produce the reactive oxygen species (ROS) superoxide anion (O₂⁻)

hundred dynamic and morphologically heterogeneous mitochondria [5]. The cellular functions of mitochondria are multifaceted and encompass the production of the bioenergetic carrier adenosine triphosphate (ATP), participation in reactive oxygen species (ROS) (Fig. 1b) [6] and calcium [7] signaling, metabolite synthesis, programmed cell death [8], and tumorigenesis [9, 10].

Mitochondrial bioenergetic function involves the oxidation of acetyl-CoA in the tricarboxylic acid (TCA) cycle to generate NADH and FADH₂, which transfer electrons to the electron transport chain to produce an electrochemical

### Mitochondrial dynamics prime mitochondria for mitophagy

The machinery regulating mitochondrial morphology dynamics is highly integrated with mitophagy initiation. Mitochondrial fission, i.e., division, is mediated by the GTPase dynamin-related protein 1 (Drp1), whereas fusion involves three GTPases; mitofusins 1 and 2 (Mfn1 and Mfn2) mediate outer membrane fusion and optic atrophy 1 (OPA1) mediates inner membrane fusion (Fig. 2) [22]. Upon fission, mitochondria can be segregated into polarized and depolarized daughter mitochondria. While polarized daughter mitochondria can undergo fusion, consistent with the polarization requirement for fusion [23], depolarized mitochondria are targeted by mitophagy [24]. Consistent with this observation, inhibiting the fission machinery or enhancing mitochondrial fusion were both shown to decrease mitophagy [24], while enhancing fission

Mitophagy programs: mechanisms and physiological implications of mitochondrial...

![Diagram](attachment://diagram.png)

Fig. 2 Mitochondrial dynamics are linked to mitophagy. The GTPases Mitofusin 1 (Mfn1) and Mitofusin 2 (Mfn2) mediate OMM fusion, and the GTPase OPA1 (Optic atrophy 1) mediates IMM fusion. Mitochondrial fragmentation requires the translocation of the GTPase Drp1. Normally cytosolic, Drp1 is recruited to mitochondria via OMM-bound receptor proteins Fis1, Mff, MID49, and MID51.

promotes mitophagy [25]. Furthermore, during nutrient starvation-induced autophagy, protein kinase A (PKA)-mediated phosphorylation of Drp1 inactivates fission events, driving the mitochondrial network to an increased fusion state, as a mitophagy inhibition mechanism [26, 27].

Notably, a recent in vivo study has revealed a higher level of complexity. In a conditional Drp1 deletion model in mouse heart, Drp1 knockout increased mitophagy, and promoted dilated cardiomyopathy associated with increased levels of necrotic cell death [28]. Conversely, conditional Mfn1/2 knockout resulted in accumulated dysfunctional mitochondria, lack of mitophagy, and hypertrophy without inducing cell death. These findings not only confirm that mitochondrial dynamics are an essential component of mitophagy, but also expand our understanding past cell culture-based experiments, demonstrating that in vivo the loss of mitochondrial quality control via mitochondrial morphology dynamics is able to engage compensating mitophagy pathways.

Protein kinase A (PKA)-mediated phosphorylation of Drp1 at serine 656 inhibits its activity, resulting in hyperfused mitochondrial networks, while dephosphorylation by the phosphatase calcineurin activates Drp1. Active Drp1 constricts and fragments mitochondria. Drp1-driven mitochondrial fragmentation is a critical quality control event upstream of mitophagy

machinery coordinates the formation and expansion of the so-called phagophore, which encloses cellular proteins and organelles within a double-membraned organelle, the autophagosome. These membrane events are controlled by autophagy-related (Atg) proteins [30]. Autophagosomal membranes likely originate from the endoplasmic reticulum, and may include membrane contents originating from mitochondria, plasma membrane, Golgi, and endosomes [31]. Once formed, the autophagosome fuses with endolysosomes to mature into the autolysosome, whereupon lysosomal hydrolases degrade the autophagosomal inner membrane and its contents [32]. The fusion of autophagosomes with endolysosomes is a highly regulated process (Fig. 3a), and dependent on the GTPase Rab7, which regulates endolysosomal trafficking [33, 34]. Recent findings have shown that maturation is regulated by adaptor proteins which bind autophagosomes and endolysosomes to promote fusion and maturation [35, 36]. The HOPS (homotypic fusion and vacuole protein sorting) tethering complex is responsible for autolysosome formation [37]. HOPS complexes with PLEKHM1 (pleckstrin homology domain containing protein family member 1), which binds Rab7, and can interact with autophagosomes to promote fusion between endolysosomes and autophagosomes [36]. Subsequent membrane fusion events are carried out by SNARE (soluble N-ethylmaleimide-sensitive factor attachment protein receptor) proteins; The SNARE Syntaxin 17 binds with the HOPS complex [37], and its interaction with the endolysosomal SNARE VAMP8 mediates fusion of autophagosomes with endolysosomes [38].

### Autophagy induction, maturation, and degradation

Macroautophagy (in this review referred to as autophagy) is a catabolic process that in the presence of growth factors and amino acids are negatively regulated by anabolic mTOR signaling. As such, autophagy is repressed under nutrient-rich conditions, and activated under conditions of decreased ATP and nutrients [29]. Upon its induction, the autophagic

(microtubule-associated protein light chain 3) and GABARAP (gamma-aminobutyric acid receptor-associated protein) subfamilies. The LC3 family includes LC3A, LC3B, LC3B2, and LC3C, and the GABARAP family includes GABARAP, GABARAP-L1, GABARAP-L2/GATE-16, and GABARAP-L3. In the following text, mammalian Atg8 family proteins in general are referred to as LC3. All LC3 family members are integrated into autophagosomes via a C-terminal glycine, covalently conjugated to PE. LC3 conjugation involves two ubiquitin-like reactions (Fig. 3b). The Atg4 cysteine protease first cleaves pro-LC3 at the C-terminus to expose a glycine residue, forming LC3-I. Upon autophagy induction E1 (Atg7) and E2 (Atg3) system conjugates PE to LC3-I, forming LC3-II. Atg12 is conjugated to Atg5 by Atg7 (E1-like) and Atg10 (E2). Atg7 and Atg10 conjugate Atg5 and Atg12, which then forms a complex with Atg16. The Atg5/12/16 complex acts as an E3 ligase, promoting PE conjugation to LC3 [41].

Upstream, the participation of LC3 proteins in autophagy is regulated by phosphorylation. For instance, integration of LC3B into autophagosomal membranes is suppressed by phosphorylation at serine 12 by protein kinase A (PKA) [42], and STK3/STK4-mediated phosphorylation of LC3B at threonine 50 is needed for efficient autolysosome formation [43] (Figs. 3a, 4a).

While the functional diversity of LC3 proteins has not been entirely established, it has been proposed that LC3 subfamily proteins participate in autophagosomal membrane elongation, while GABARAP subfamily proteins mediate autophagosome maturation [44]. Importantly, LC3 and GABARAP proteins can both serve as binding partners for autophagy receptors, thereby underlying specific modes of autophagy, and permitting autophagy control of diverse cell signaling events, including the antioxidant response [45], pathogen response [46], as well as mitophagy [47].

### LC3 interacting region (LIR) motifs as a basis for specific autophagy

LC3 proteins contain a conserved hydrophobic region, comprised the so-called W and L pockets [48, 49], which docks via hydrophobic interactions with a motif, termed LC3-interacting region (LIR), contained within autophagy receptors [47, 48, 50, 51] (Fig. 4a, b). This LIR motif, also referred to as AIM (Atg8-family-interacting motif) or LC3 recognition sequence (LRS), comprises a core consensus sequence of an aromatic residue followed by a hydrophobic residue [W/F/Y]xx[L/I/V] [47]. This sequence is preceded by negatively charged residues, which are critical for the interaction with positively charged residues on LC3 proteins. Moreover, serine/threonine residues within this LIR preceding region were shown to be fundamental for the

### Post-translational modification of Atg8 family proteins during autophagosome formation

A key process in the formation of the phagophore is the conjugation of Atg8 (autophagy-related protein 8) family protein members to phosphatidylethanolamine (PE) [39, 40], an integral membrane component of the autophagosome. In mammals, Atg8 proteins can be divided into LC3

Mitophagy programs: mechanisms and physiological implications of mitochondrial...

Fig. 4 The LC3-interacting region (LIR) motif as a mechanistic basis for mitophagy.

**a** LC3B structure (PDB 1UGM) [179], analyzed using Chimera [180]. *Upper panel* ribbon structure diagram indicating α-helices and β-sheets. LC3 proteins contain two N-terminal α-helices and a ubiquitin-like core formed from β strands. Hydrophobic pockets between β2 and α2 form the W-site, and between β2 and α3 form the L-site. *Middle panel* surface representation indicates topology of W- and L-sites (adapted from [49]), phosphorylated residues at S12 and T50, and putative binding sites for cardiolipin (R10 and R11) and ceramide (I35 and F52). *Lower panel*, N-terminal sequence alignments of LC3 homologues. Tcoffee alignment [181] of human LC3 member N-terminal regions using Jalview [182]. Blue shading indicates percentage identity, positively charged R, K and H residues, aligning to LC3B R10 and R11, are indicated in red.

**b** The LIR motif, also known as Atg8 family-interacting motif (AIM), is a short linear peptide motif found in autophagy receptors, which binds LC3-II and

**c** Sequence alignment of reported mitophagy receptor core LIR motifs and neighboring upstream and downstream regions. Phosphorylation can positively or negatively regulate LIR activity.

**d** Two main groups of autophagy receptors target mitochondria. E3 ligase-mediated ubiquitylation of OMM proteins recruits ubiquitin-binding, LIR motif-containing receptors, which leads to the binding of receptors and engagement of sequestration by autophagosomes. Alternatively, a group of autophagy receptors contain a transmembrane domain and are constitutively targeted to the OMM. LIR activity of these mitophagy receptors is regulated by phosphorylation events

thereby underlies selective autophagy. The LIR has a [W/Y/F]xx[L/I/V] core motif, and receptor-ligand interaction occurs through formation of an intermolecular β-sheet via hydrophobic interactions between the LIR motif and the conserved W and L sites on Atg8 proteins.

regulation of autophagy receptor activity through phosphorylation [46, 52, 53]. So far, it has not been determined if LC3 phosphorylation similarly alters its capacity to bind with autophagy receptors.

During mitophagy, OMM-localized autophagy receptors (mitophagy receptors) attach autophagosomes to the OMM via their LIR motif. To date, eight mechanistically distinct mitophagy receptors have been characterized (Fig. 4c), and can be grouped according to the manner in which they target mitochondria (Fig. 4d). One group of mitophagy receptors contains a ubiquitin-binding domain which localizes them to Parkin-ubiquitylated mitochondria, including p62/SQSTM1, NBR1, and optineurin [54–57]. In addition, mitochondrial-localized Parkin binds AMBRA1 to localize it at the OMM [58, 59]. The second group of mitophagy receptors is made up of Bnip3 [52, 60, 61], its homologue Bnip3L/Nix [62–64], FUNDC1 [53], and Bcl2L13 [65]. These mitophagy receptors contain transmembrane domains and upon expression constitutively localize to the OMM [53, 66–68].

Below we discuss protein mitophagy receptor systems which have been mechanistically elucidated, namely the PINK1/Parkin system, and transmembrane-containing mitophagy receptors. We further discuss the lipids ceramide and cardiolipin, which when localized to the OMM can directly bind LC3 and engage mitophagy [69, 70].

Fig. 5 Mitophagy program pathways: triggers and post-translational regulation.

a Upon mitochondrial depolarization PINK1 is stabilized at the OMM, resulting in phosphorylation of ubiquitin and Mfn2 and consequent recruitment of the E3 ligase Parkin. In the cytosol, Parkin de-ubiquitylation by USP8 is required for mitochondrial translocation. At the mitochondria, Parkin ubiquitylates OMM proteins, resulting in recruitment of ubiquitin-binding autophagy receptors such as p62, OPTN, and NBR1 which then can attach to autophagosomes via their LIR motifs. In addition, PINK1 directly recruits autophagy receptors OPTN and NDP52 through generation of phospho-ubiquitin. AMBRA1 can increase Parkin-mediated mitophagy by direct binding with LC3 and local autophagy stimulation. Sequestration is coordinated by the Rab7 GTPase-activating protein TBC1D15, which localizes to mitochondria by binding Fis1 and binds forming autophagosomes via a LIR.

b Bnip3 is induced by hypoxia, and localizes to the OMM via one transmembrane domain. Bnip3 LIR activity requires serine 17 phosphorylation, and is enhanced by serine 24 phosphorylation. Responsible kinase(s) and phosphatase(s) have not yet been identified. Pro-survival Bcl-xL activity functions to enhance Bnip3 binding to LC3. Nix contains an identical SWxxL LIR motif and is expected to undergo similar regulation by phosphorylation.

c FUNDC1 contains three transmembrane domains which accomplish its OMM localization. Under normal conditions, Src kinase phosphorylates tyrosine 18 and thereby inactivates the FUNDC1 LIR. Hypoxia inactivates Src kinase, permitting FUNDC1 LIR-mediated binding to autophagosomes. As a second negative regulatory mechanism, CK2 phosphorylates FUNDC1 at serine 13, inhibiting its LIR activity, and PGAM5 phosphatase antagonizes this reaction. The pro-survival Bcl-xL can bind PGAM5, inhibiting its pro-mitophagy activity. Under hypoxia and in response to depolarization, serine 17 phosphorylation by ULK1 enhances LIR activity.

d Bcl2L13 induces Drp1-independent mitochondrial fragmentation and is a LIR-containing mitophagy receptor and mammalian homologue of yeast Atg32 mitophagy protein.

e, f Mitophagy receptor systems undergo crosstalk with mitochondrial apoptosis. e Pro-survival Bcl-2 signaling suppresses Parkin translocation to mitochondria, and pro-apoptotic BH3-only proteins can activate Parkin-mediated mitophagy. At longer timescales, Parkin ubiquitylates pro-survival Mcl-1, resulting in its degradation and consequent activation of mitochondrial apoptosis. f Bnip3 LIR and BH3 domains confer dual functionality during apoptosis. Under conditions of LIR inactivation, the BH3-only protein Bnip3 increases TNF-mediated caspase activation. Under conditions of engaged LIR activity, the increased degradation of mitochondria can suppress the mitochondrial participation in apoptosis activation. Green arrows indicate positive regulation; red arrows indicate negative regulation.

Mitophagy receptor systems

The PINK1/Parkin program targets mitophagy receptors to depolarized mitochondria via ubiquitylation of the OMM

To date, the best understood mitophagy system is controlled by the serine/threonine kinase PINK1 (PTEN-induced putative kinase 1) and the E3 ligase Parkin1 [71] (Fig. 5a). Herein, PINK1 serves as the sensor for the mitochondrial polarization state. In respiring, polarized mitochondria, PINK1 is imported into the mitochondrial intermembrane space and rapidly degraded through combined activities of the protease PARL (presenilin-associated rhomboid-like protein) and the proteasome [72, 73], thereby maintaining low basal PINK1 levels under
normal conditions. Mitochondrial depolarization inactivates its import and proteasomal degradation, leading to PINK1 accumulation on the OMM and resulting in recruitment of Parkin from the cytosol. Parkin translocation to mitochondria has been reported to involve two mechanisms. PINK1 at the OMM phosphorylates Mfn2 at serine 442 and threonine 111, and phosphorylated Mfn2 can act as a receptor to recruit Parkin [74]. In addition, PINK1 phosphorylates ubiquitin at serine 65 [75, 76], and the ubiquitin-like domain of Parkin at serine 65 [77], which drive Parkin recruitment to the OMM and activation of its E3 ligase activity [78]. Once activated and recruited, Parkin E3 ligase activity results in the ubiquitylation of numerous OMM proteins [57, 71], which leads to the recruitment of different LIR-containing autophagy receptors which bind ubiquitin-tagged OMM proteins, including p62/SQSTM1 [79], optineurin [55] and NBR1 [56, 80] (Fig. 4d). Mechanistically, p62 has a role in clustering mitochondria during mitophagy [55, 81, 82], and has been reported to be required [79], or dispensable in downstream mitochondrial degradation [55, 81]. Recently, optineurin was shown to act as the LIR-dependent autophagy receptor downstream of Parkin activation [55]. Furthermore, optineurin can localize TBK1 to p62, and phosphorylate p62 at serine 403 phosphorylation which positively regulates p62 ubiquitin binding [83], generating a feedforward mechanism enhancing p62 targeting of mitochondria. Further, mitochondrial Parkin binds the autophagy-promoting protein AMBRA1 [84]. In response to mitochondrial depolarization, AMBRA1 interacts with Parkin at the OMM and contributes to Parkin-mediated mitophagy via local stimulation of autophagosome formation [58, 59]. Moreover, AMBRA1 was recently shown to bind LC3 through a LIR motif during Parkin-mediated mitophagy, and forced targeting of AMBRA1 to the OMM resulted in efficient depletion of mitochondria, independent of Parkin [85], identifying AMBRA1 as a mitophagy receptor. In addition, in the absence of Parkin, mitochondrial-targeted AMBRA1 was reported to activate mitochondrial ubiquitylation, without recruiting p62. However, the E3 ligase identity, and whether ubiquitylation recruits other ubiquitin-binding autophagy receptors, remains to be determined. A recent study using TALEN and CRISPR/Cas9 genome editing for combinatorial knockout of five autophagy receptors further consolidates our understanding of the relative contributions of specific autophagy receptors to PINK1/Parkin-mediated mitophagy [86]. Here, p62 and NBR1 were found to be dispensable, and primary but redundant autophagy receptor functions were defined for OPTN and NDP52. Furthermore, PINK1-mediated generation of Ser65 phospho-ubiquitin [75, 76] was revealed as a primary mitophagy signal, autonomously capable of recruiting OPTN and NDP52, thereby proposing a new

model within which Parkin functions to amplify PINK1-initiated mitophagy signaling.

### Deubiquitylation and modulation of Parkin-mediated mitophagy

Ubiquitylation is a reversible process, and recent work has evidenced a central role for deubiquitylating (DUB) enzymes in the regulation of mitophagy (Fig. 5a). Parkin can ubiquitylate itself, resulting in its reduced recruitment to depolarized mitochondria. USP8 deubiquitylation of auto-ubiquitylated Parkin is required for its localization to depolarized mitochondria, and thereby for efficient activation of mitophagy [87]. In addition, acting as negative regulators, the ubiquitin-specific proteases USP30, which localizes at the OMM via a transmembrane domain [88], and USP15, which can fractionally localize to mitochondria, remove Parkin-ligated ubiquitin from OMM proteins [88, 89]. Importantly, both USP30 and USP15 knockdown increased the cellular capacity for mitophagy, suggesting the potential for pharmacological enhancement of mitophagy.

#### Constitutively OMM-localized mitophagy receptors are phospho-regulated

In contrast to the PINK1/Parkin mitophagy system, which involves Parkin translocation to mitochondria and subsequent ubiquitin-dependent recruitment of mitophagy receptors, a group of LIR-containing mitophagy receptors, upon expression, constitutively localize at the OMM via transmembrane domains (Fig. 4d). This group of mitophagy receptors is transcriptionally regulated, and engagement of mitophagy receptor activity is controlled through the phosphorylation status of their LIR.

#### Bnip3 and Bnip3L/Nix: OMM-localized mediators of mitophagy and apoptosis signaling

Bnip3 (BCL2/adenovirus E1B 19 kDa interacting protein 3) and its homologue Bnip3L/Nix are atypical members of the pro-apoptotic Bcl-2 subfamily of BH3-only proteins [90], sharing 56% of the amino acids sequence identity [91]. Both, upon expression, are inserted into the OMM via their C-terminal transmembrane domains, with the N-terminus oriented toward the cytoplasm [66, 92]. At the mitochondria, the BH3 domains of Bnip3 [92] and Nix [93] suppress the function of anti-apoptotic Bcl-2 proteins. In addition to its apoptotic functions, Bnip3 was reported to trigger autophagy [94] and mitophagy [60], and Nix was found to mediate mitochondrial clearance during blood cell development [62, 63]. It was later discovered that both Bnip3 and Nix contain identical N-terminal LIRs (WxxL) [52, 61, 64] (Fig. 4c),thereby providing mechanistic explanation for their mitophagy-inducing function (Fig. 5b).

In addition to this function in coupling OMMs with autophagosomal membranes via their LIR, Bnip3 and Nix undergo crosstalk with the autophagy regulation machinery. Hypoxia-induced autophagy was shown to be supported by Bnip3 and Nix binding to Bcl-2 and consequent disruption of Bcl-2 binding to Beclin 1 [95]. Furthermore, Bnip3 and Nix can bind to the mTOR-activating protein Rheb via their N-terminus, which was associated with reduced activation of mTOR and enhanced autophagy [96]. Under glutamine-mediated enhancement of mitochondrial respiratory activity, Rheb interacted with Nix to induce mitophagy, presumably to meet an increased mitochondrial quality control demand [97]. In this study, Rheb localization to mitochondria did not alter mTOR activity, and it remains to be determined whether Rheb further functions to regulate Nix interaction with LC3.

We have recently shown that the mitophagic activity of Bnip3 is controlled through the phosphorylation state of serine residues adjacent to the LIR [52]. Phosphorylation of serine residues 17 and 24 flanking the Bnip3 LIR specifically promotes binding to LC3B and GATE-16 (Fig. 4c). Interestingly, phosphorylation of Bnip3 at serine 17 is a prerequisite for LC3B and GATE-16 binding, whereas phosphorylation at serine 24 further increased the affinity for both LC3B and GATE-16. Similar to Bnip3, Nix contains a SWxxL LIR motif and activity of its LIR is serine phosphorylation regulated (own unpublished results). To date it remains undetermined which kinases and phosphatases are responsible for controlling the phosphorylation state of the Bnip3 and Nix LIRs.

### FUNDC1

The OMM-localized protein FUNDC1 contains three transmembrane domains and an N-terminal cytosolic LIR motif (YxxL) for binding to LC3 and GABARAP proteins [53] (Fig. 4c). Also FUNDC1 undergoes positive and negative regulation by phosphorylation, at residues within and preceding the LIR. Under normal conditions, FUNDC1 mitophagy receptor activity is kept in check through inhibitory phosphorylation by Src kinase at tyrosine 18 [53] and by casein kinase 2 (CK2) at serine 13 [98] (Fig. 5c). In response to hypoxia or mitochondrial uncoupling, PGAM5 dephosphorylates CK2-phosphorylated serine 13 of FUNDC1 to activate LC3 binding [98]. In addition, ULK1 phosphorylates serine 17 of the FUNDC1 LIR motif, resulting in increased LC3 binding [99]. Interestingly, Bcl-xL antagonizes PGAM5-mediated dephosphorylation of FUNDC1 and thereby prevents LC3 binding [100], suggesting that in the FUNDC1 system, anti-apoptotic signaling antagonizes the mitophagy response.

---

### Bcl2L13/Bcl-Rambo

Bcl2L13 (Bcl2-like 13 or Bcl-Rambo) is an atypical Bcl-2 family member, which contains four BH motifs, but does not bind pro-death or pro-survival Bcl-2 members, and signals apoptosis via its C-terminal transmembrane domain, which targets the OMM [68]. Recently, Bcl2L13 was identified as a mitophagy receptor based on similarity to the yeast mitophagy receptor Atg32, with a WxxL LIR motif [65] (Fig. 4c). Bcl2L13 expression is sufficient to induce mitochondrial fragmentation and target fragmented mitochondria to autophagosomes and endolysosomes (Fig. 5d). All BH domains are required for fragmentation, but in contrast to other modes of mitophagy, fragmentation occurred independent of Drp1. Further, the authors report that CCCP-induced mitochondrial clearance by Bcl2L13 was independent of Parkin, and Bcl2L13 does not induce mitochondrial ubiquitylation in response to CCCP. It is suggested that increased Bcl2L13 levels activate its LIR via phosphorylation: CCCP rapidly increased Bcl2L13 levels, and the mutation of phosphorylation target S272 [101] to alanine reduced total serine phosphorylation of Bcl2L13. Further, while the Bcl2L13S272A mutant fragmented mitochondria, mitochondria did not co-localize with autophagosomes. Mutating S272 to phospho-mimicking glutamic or aspartic acid residues, and identifying the kinase and phosphatase responsible, will be instrumental for further elucidating the role of Bcl2L13 LIR phosphorylation-regulation.

It is remarkable that all known transmembrane-domain OMM-localized mitophagy receptors contain conserved serine/threonines preceding the LIR (Fig. 4c), indicating that LIR phosphorylation mechanisms represent a crucial target for altering mitophagy receptor function. Also optineurin contains a serine preceding the LIR, which is required for LC3 binding and its participation in xenophagy [46], rendering it likely that its role in mitophagy is also positively regulated by phosphorylation. Likewise, AMBRA1 contains two tyrosine residues preceding the LIR, which would also be expected to be phospho-regulated (Fig. 4c). Of note, opposed to FUNDC1, we observed that the phosphorylated state of serine 13 of Bnip3 had no impact on LIR activity [52], suggesting that the specific residues preceding the core LIR motif contribute to its regulation.

---

### Lipid-mediated mitophagy

In addition to protein mitophagy receptors, the lipids ceramide and cardiolipin, when localized to the OMM can directly bind LC3 and engage mitophagy [69, 70]. Increased levels of C18-ceramide at the OMM, either through exogenous addition, or endogenously-generated byceramide synthase 1 (CerS1), specifically bind to LC3, to induce Drp1-dependent mitophagy [69]. Intriguingly, knockdown of LC3B permitted tumor growth under conditions of increased ceramide production, indicating ceramide-mediated mitophagy promotes cell death. Furthermore, the anti-cancer agent sodium selenite was shown to activate this program via upregulation of CerS1. This intriguing example for a pro-death mode of mitophagy is independent of mitochondrial apoptotic signaling and caspase signaling, and presents a model to elucidate the molecular basis of mitophagic cell death.

Alternatively, cardiolipin, a negatively charged phospholipid, can bind LC3B in response to sub-apoptotic mitochondrial dysfunction. Under normal conditions, most cardiolipin is localized to the IMM, and in response to mitochondrial stress phospholipid scramblase-3 (PLS3) redistributes cardiolipin to the OMM, where it then can bind to LC3B [70]. Interestingly, mutational analysis of LC3B indicates that ceramide interacts with I35 and F52 of LC3B, while cardiolipin undergoes electrostatic interactions with positively charged R11 and R10, which are found in LC3A and LC3B and conserved in the homologous region of LC3C (R16 and K17), but not in members of the GABARAP subfamily (Fig. 4a). Notably, I35 and F52 bind to the LIR of p62, and R10 and R11 bind aspartic acid residues preceding the p62 LIR [51]. However, it remains to be determined whether either of these lipids functionally impact protein autophagy receptor interactions with LC3B.

### Crosstalk between mitophagy receptor systems: a lack of information concerning the level of complexity

It is notable that among the different mitophagy receptor systems, our understanding of crosstalk between mitophagy modes is, so far, restricted to Nix and Parkin pathways. In cell lines, Nix has been shown to promote Parkin translocation to mitochondria, and Parkin-mediated mitophagy [102, 103], and Parkin ubiquitylation of Nix recruits the mitophagy receptor NBR1 [103]. In future work, we propose that mutationally inactivating and activating the LIR using non-phosphorylatable and phosphoryl-mimicking mutations, in combination with genome editing approaches, will serve as fundamental approaches for elucidating whether/which mitophagy receptors form competitive, additive or exclusive programs. Moreover, similar to Parkin, cardiolipin-mediated mitophagy is activated in response to mitochondrial poisons in neuronal cells [70]. Thus, it will be interesting to determine the contributions of lipids to receptor-mediated mitophagy.

---

### Autophagosomal sequestration of mitochondria

While autophagy receptors capture autophagosomes at mitochondria via their LIR, an outstanding fundamental question for all mitophagy programs is how autophagosomes are localized to dysfunctional mitochondria, and how the sequestration process is coordinated. Upon depolarization, in Parkin-overexpressing cells, autophagy initiating factors, including ULK1, DFCP1 and WIPI-1, localize to mitochondria independently of LC3 [104], mediated by PINK1-recruited autophagy receptors NDP52 and OPTN [86]. Together, these data suggest that autophagosomes are produced at damaged mitochondria and LC3 processing can be localized for the downstream engulfment of mitochondria into autophagosomes. One candidate for such local regulation is AMBRA1 which was shown to exert pro-autophagic activity at mitochondria [58, 59] and additionally contains a LIR [85] which may locally activate and direct autophagosome capture at depolarized mitochondria. In addition, the cellular energy sensor AMPK (5′ AMP-activated protein kinase), which activates autophagy [105] and mitophagy [20], is spatially localized at mitochondria [106], providing a possible sensing mechanism to signal local production of autophagosomes. We observed that expression of a Bnip3 LIR mutant resulted in increased autophagosome content in regions localized next to mitochondria, albeit without mediating sequestration [52], which is consistent with a mechanism of localized autophagosome generation. Recent work has yielded insight into the sequestration process. Rab7 and the Rab GTPase-activating proteins (Rab-GAPs) TBC1D15 and TBC1D17 coordinate autophagosome sequestration of Parkin-targeted mitochondria [107]. TBC1D15 and TBC1D17 bind the OMM-localized, Drp1-binding protein Fis1 and contain LIRs which coordinate autophagosome binding to mitochondria. It remains to be determined if Fis1-TBC1D15 interaction coordinates OMM-localized FUNDC1/Bnip3/Nix-mediated mitophagic sequestration.

Of note, sequestration and degradation of mitochondria requires increased autophagic activity. Thus, the cellular autophagic capacity may be an important limiting factor. Autophagic capacity is under the control of the transcription factor EB (TFEB), which promotes expression of autophagosomal and lysosomal genes [108]. mTOR inhibits TFEB by phosphorylation, thereby coupling autophagy induction and lysosomal activity [109]. Indeed, increased TFEB signaling was shown to enhance Bnip3-induced mitophagy [110] and TFEB together with additional MiT/TFE family members contributes to Parkin-mediated mitophagy [111], supporting that the cellular autophagic capacity is linked to mitophagic efficiency.
Transcriptional level regulation of mitophagy receptors

Mitophagy programs are modulated by different transcriptional responses (Fig. 6). Parkin expression is induced by mitochondrial and endoplasmic reticulum stress via the PERK-ATF4 signaling pathway, and dominantly repressed by c-Jun [112]. In addition, nuclear p53 upregulates Parkin expression [113], while cytosolic p53 suppresses Parkin targeting of mitochondria [114], and Parkin suppresses p53 expression [115], suggesting a multi-tiered homeostatic feedback. The mitophagy receptor Bnip3 is prominently regulated at the transcriptional level. Bnip3 expression is driven by hypoxia-inducible factor HIF1 [116–118], which is enhanced by Ras [119] as well as E2F-1 [120], and antagonized by NFκB [121]. Similar to Bnip3, in human tumor cells also Nix is upregulated during hypoxia, via HIF1 [118] and p53 [122]. In cardiac cells on the other hand, Nix is constitutively expressed, and enhanced expression is driven by Gq-signaling via the transcription factor SP1 [123]. In addition, activated FOXO3 induces Bnip3 and Nix expression in fasting skeletal muscle [124]. However, under hypoxia FOXO3a suppresses HIF1-mediated Nix expression via the transcriptional cofactor CITED2 [125], suggesting negative feedback regulation under hypoxic conditions.

Recently, microRNAs have emerged as negative regulators of the cellular mitophagy capacity. During erythrocyte maturation, the transcriptional repressor KAP1 (Krüppel-associated box (KRAB)-associated protein 1) [125] inhibits expression of miR-351 and hsa-miR-125a-5p, which normally function to suppress Nix and mitophagy [126]. In addition, miR-137 negatively controls expression of Nix and FUNDC1 [127]. Hypoxia-associated down-regulation of miR-137 thereby permits mitophagy receptor expression. In prostate cancer, the p53 inducible miR-145 can suppress Bnip3 expression, thereby counteracting HIF1 upregulation [128].

Physiological and pathophysiological implications of mitophagy

As the mechanisms of mitophagy unravel, so do the physiological roles for mitophagy. Several lines of evidence link familiar Parkinson’s disease to impaired mitophagy. Mitochondria are defective in Parkinson’s disease [129, 130]. Deletions and point mutations of Parkin and PINK1 genes (PARK2 and PARK6, respectively) were identified in early-onset Parkinson’s disease patients [71] and these disease-associated mutations were shown to mechanistically result in defective mitophagy [54, 73, 81, 131]. However, extrapolating findings from mechanistic

A Parkin regulation

B Nix/FUNDC1 regulation

C Bnip3 regulation

Fig. 6 Transcriptional regulation of mitophagy. a Under conditions of mitochondrial and endoplasmic reticulum (ER) stress, Parkin expression can be induced by the PERK/ATF4 pathway, and suppressed by the JNK/c-Jun pathway. Further, p53 and Parkin interactions form complex crosstalk. p53 can induce Parkin expression, and Parkin can inhibit p53 expression, forming a negative feedback circuit. In addition, cytosolic p53 can inhibit Parkin translocation to mitochondria. b Nix expression is induced by G(q) signaling via SP1, by hypoxia via HIF1 and p53, and FOXO3. Nix expression can be suppressed via FOXO3 activation of CITED2. Both Nix and FUNDC1 are negatively co-regulated by microRNAs. miR-351 and hsa-miR-125a-5p target Nix, and miR-137 co-targets Nix and FUNDC1. c Bnip3 expression is increased by Ras activity and induced during hypoxia by HIF1. E2F and FOXO3 activate, and NFkB suppresses Bnip3 expression. Black arrows indicate post-translational regulation; green arrows indicate transcriptional activation; red arrows indicate transcriptional repression

PINK1/Parkin studies to in vivo relevance has been a subject of much debate: Most studies that have elucidated the molecular events of Parkin-mediated mitophagy utilize
a combination of artificial Parkin overexpression and chemically-induced mitochondrial depolarization in cancer cell lines. Notably, it was reported that, opposed to in cancer cell lines, mitochondrial depolarization does not lead to robust mitochondrial Parkin translocation in neurons and this was linked to bioenergetics differences between oxidative phosphorylation-dependent neurons and glycolytic cancer cell lines [132]. Moreover, mitochondrial Parkin translocation was shown to occur as a slow and subcellularly restricted process in mature cortical neurons [133], and Parkin accumulation was observed on only a fraction of depolarized mitochondria in neuronal axons [134]. Importantly, it was recently shown that Parkin-mediated mitophagy is constitutively engaged in primary neurons, with PINK1 functioning as the limiting factor for basal mitophagy [89]. In this study, inhibition of USP30, which antagonizes Parkin-mediated mitophagy, increased basal mitophagy by up to fourfold, and in vivo knockdown of USP30 in drosophila was associated with survival. Interestingly, a dominant negative USP30 mutant restored mitophagy in response to PINK1 knockdown, suggesting a fundamental suppressive action. However, it is important to consider that these findings reflect basal neuronal mitophagy activities, and further bioenergetic and oxidative stress-activated studies will be required to determine the extent to which USP30 regulates neuronal mitophagy of dysfunctional mitochondria, and/or autophagy-independent modes of selective mitochondrial degradation [135, 136]. Of note, Parkin knockout mice do not readily exert neurodegeneration or associated motor phenotypes [71], and Parkin knockout in mice with respiratory chain deficiency caused by neuron-specific mitochondrial transcription factor (Tfam) knockout did not impact clearance of mitochondrial aggregates or progression of neurodegeneration [137]. A recent study reports a neuroprotective role for endogenous Parkin, in a model of Parkin knockout mice with a mitochondrial dysfunction background caused by accumulation of mtDNA mutations due to deficiency in DNA polymerase γ [138]. However, while Parkin knockout affected mitochondrial function, no effect on mtDNA mutation burden was detected. Thus, it remains to be determined whether Parkin-mediated mitophagy or alternative functions of Parkin are key to its neuroprotective role in this context.

As mitophagy receptors also Bnip3 and Nix fulfill physiological functions, in addition to their pro-apoptotic function as atypical BH3-only proteins. Bnip3-mediated mitophagy participates in mitochondrial homeostasis in liver of adult mice to avoid metabolic defects [139] and is engaged as a cytoprotective program during ischemia/reperfusion injury in cardiac myocytes [52, 60, 140]. The

Bnip3 homologue Nix is transcriptionally upregulated together with Bcl-xL during erythrocyte maturation [141], to drive the removal of mitochondria [62, 63] via its mitophagy receptor function [64]. Furthermore, mouse cytomegalovirus-induced proliferation of antigen-specific natural killer cells is associated with mitochondrial dysfunction, and Bnip3- and Nix-mediated mitophagy were recently shown to be essential in the survival of memory natural killer cells [142]. In heart cells, both PINK1/Parkin-and Bnip3-/Nix-mediated mitophagy were connected to maintaining homeostasis. Parkin-deficiency resulted in enlarged cardiomyocyte mitochondria and respiration defects in Drosophila heart tubes [74]. Similar cardiac defects were observed in Mfn2 knockout mice, with a proposed connection to impaired Mfn2-mediated recruitment of Parkin [74]. However, as Mfn2 is a crucial regulator of mitochondrial morphology as well as of mitochondrial-ER junctions, a causative role for altered Mfn2-mediated mitophagy in cardiomyopathy remains to be established. Nix knockout mice developed cardiomyopathy, which developed faster in Bnip3 and Nix double knockout mice [143]. These in vivo findings demonstrate cardioprotective roles for Parkin, Bnip3 and Nix mitophagy programs in the regulation of mitochondrial homeostasis, and further suggest limits to compensating roles. Of note, an important mechanistic distinction from depolarization-driven mitophagy is that Bnip3 can induce mitophagy of polarized mitochondria [52], supporting divergent functional roles between mitophagy programs: In addition to being engaged in order to respond to dysfunctional mitochondria, Bnip3 and Nix appear to be responsible for basal turnover [143], or engaged for developmental [62–64] and immune response programs [142].

Mitophagy and cancer

Conceptually, a role of mitophagy in cancer is an intriguing hypothesis: Subpopulations of dysfunctional mitochondria can transform cells and promote tumorigenesis [144], suggesting mitophagy could function as a tumor suppressor mechanism. Alternatively, in cancer cells, mitophagy scavenging of pro-apoptotic mitochondria could be cytoprotective. However, to date the roles for mitophagy programs in cancer remain unclear. For example, Parkin may function as a tumor suppressor, as Parkin mutations are common and result in cell cycle deregulation [145, 146]. Furthermore, the expression of Bnip3 and Nix is commonly deregulated in cancer. Both Bnip3 and Nix translation is increased in hypoxic and peri-necrotic tumor regions [118, 147]. High Bnip3 is reported to correlate with invasive tumor behavior in breast [147] and colorectal[148] cancers, and poor prognosis in non-small cell lung [149], prostate [128] and endometrial [150] cancers. In addition, low Bcl2L13 expression correlates with good outcome in childhood acute lymphoblastic leukemia [151]. On the other hand, Bnip3 expression can be silenced in leukaemias [152] and pancreatic [153], colorectal and gastric [154] cancers, and lost Bnip3 [153] or high AMBRA1 [155] expression in pancreatic cancer correlate with worsened prognosis. Thus, these in vivo cancer studies suggest contradicting roles for mitophagy receptors and signaling regulators in cancer, likely due to their simultaneous participation in mitophagy-independent signaling. Therefore, the targeting of specific autophagy receptors in in vivo experiments, for various cancer types and treatment conditions, are required for translational understanding.

Such an approach was recently performed in a mouse model study of metastatic breast cancer [156]. The authors report that deleting Bnip3 promoted tumor growth and malignancy, and resulted in the accumulation of dysfunctional, ROS-producing mitochondria. The resulting increased oxidative stress was shown to activate HIF1α-mediated glycolysis and angiogenesis. Consistently, in human breast cancer low Bnip3 correlated with high HIF1α and poor patient prognosis.

### Role of mitophagy in apoptosis signaling

While it is well established that autophagy and apoptosis signaling undergo pronounced regulatory crosstalk [157], our understanding of the relationship between mitophagy and apoptosis is limited. Recent findings indicate that Parkin undergoes extensive crosstalk with apoptosis pathways (Fig. 5e). Mitochondrial translocation of Parkin was shown to be blocked by pro-survival Bcl-2 proteins, and activated by BH3-only proteins under conditions of inhibited caspase activity [80]. At longer time scales, Parkin can enhance mitochondrial uncoupling-induced apoptosis through degradation of anti-apoptotic Mcl-1 [158]. Consistent with this action, knockdown of USP30, which antagonizes Parkin-mediated mitophagy by deubiquitylating OMM proteins [89], sensitizes cells to activation of the mitochondrial apoptosis pathway [159]. This finding suggests that USP30 would drive mitochondria toward a mitophagy versus apoptosis decision event, and highlights the requirement for further mechanistic understanding.

We further reported that pro-survival Bcl-x<sub>L</sub> positively regulated the binding of Bnip3 to LC3B, and enhanced mitochondrial sequestration [52] (Fig. 5f). Notably, this regulation differs from FUNDC1, which is negatively regulated by Bcl-x<sub>L</sub> [100], suggesting separate and mutually exclusive mitophagy pathways.

Functionally, ectopic expression of wild type or LIR-mutated Bnip3 [61], or knockdown of FUNDC1 during hypoxia [53] have no measured impact on apoptosis induction. However, pre-activation of Bnip3-mediated mitophagy through expression of constitutively active Bnip3 receptor prior to TNF (tumor necrosis factor) treatment, significantly reduced effector caspase activation [52]. These findings suggest that enhanced mitophagy activity, and/or delayed activation of mitochondrial outer membrane permeabilization, can reduce the mitochondrial capacity to amplify apoptosis. However, different positive and negative feedbacks between individual mitophagy programs and both pro-survival and pro-death apoptosis signaling, operate at different time scales, and undergo crosstalk [102]. Therefore, it remains to be systematically determined to what extent, in which tissues, and under which (patho)physiological conditions, mitophagy receptors are expressed and regulated. Importantly, the characterized phospho-mimicking mutations [52, 99] will be instrumental in the elucidation of mitophagy receptor function in cell culture and in in vivo models.

#### Non-canonical modes of mitochondrial processing by endolysosomes

In addition to the above described canonical macroautophagy-mediated modes of mitophagy, mitochondrial stress can engage autophagy-independent endolysosomal interactions with mitochondria (Fig. 7). While chemically induced mitochondrial depolarization activates PINK1-mediated mitophagy, under conditions of low mitochondrial bioenergetic stress and enhanced oxidative stress, and prior to depolarization, PINK1 and Parkin induce mitochondria-derived vesicles (MDVs) which target oxidized mitochondrial proteins to lysosomes, independently of autophagy [135, 160]. The MDV pathway is proposed to function as a homeostatic quality control mechanism for damage levels which do not require the sacrifice of the entire mitochondrion. In addition, in response to oxidative stress the p53-induced Mieap (Mitochondria-eating protein) induces the autophagy- and Parkin-independent interaction of large endolysosome-like organelles with mitochondria [161, 162]. Mieap activity requires the interaction with Bnip3 and Nix [163], and is proposed to operate in mitochondrial quality control.

We recently reported that expression of the canonical BH3-only proteins tBid, Bim<sub>EL</sub>, Bik and Bad, or treatment with apoptosis inducers TNF or staurosporine, triggers E3 ligase XIAP (X-linked inhibitor of apoptosis protein)-mediated mitochondrial outer membrane permeabilization and XIAP entry into mitochondria [164, 165].
XIAP action at mitochondria results in prominent ubiquitylation at the OMM and IMM, and, intriguingly, autophagy-independent movement of endolysosomal machinery into mitochondria [164]. Inside mitochondria, XIAP catalyzes the proteasome- and lysosome-mediated degradation of its endogenous inhibitor Smac [164], and consequently reduced effector caspase activation during intrinsic apoptosis [165]. Intramitochondrial recruitment of endolysosomes is activated independently of Parkin, Bnip3 or Nix LIR receptor activity. Interestingly, expression of Bnip3 or Nix LIR mutants, with lost mitophagy receptor function, likewise results in activation of this novel pathway [164].

Notably, these three non-canonical modes of mitochondrial processing do not require LC3-decorated autophagosomes, and instead appear to result from direct interorganellar interactions between mitochondria and endolysosomes [135, 161, 164]. One possible advantage over canonical mitophagy, is that direct interactions between endolysosomes and mitochondria can be engaged at faster time scales, as they engage pre-existing organelles.

## Experimental approaches to detecting mitophagy

In this section, we highlight assays to detect mitophagy induction and capacity (Fig. 8). Overall, when assessing mitophagy it is important to apply a variety of techniques and to monitor both mitophagy induction and efficiency of degradation.

## High-resolution imaging and biochemical detection and quantification of mitophagy

Mitophagy was first detected using electron microscopy [21], and remains a standard for nanometer analysis of organelle ultrastructure and mitochondrial sequestration. More recently, mitophagy can be directly observed by confocal or widefield fluorescence microscopy, based on sequestration of mitochondria by GFP-LC3-labeled autophagosomes [60] (Fig. 8a). Of note, mitophagy programs vary in the composition of the autophagy machineries. For instance, Nix poorly binds LC3B [64], while Bnip3 preferentially binds LC3B and poorly binds GATE16 [52]. Thus, when assessing novel mitophagy settings, responses of both LC3 and GABARAP subfamilies should be determined. Alternatively, GFP-Rab7 is a suitable tool to detect downstream mitochondrial entry into endolysosomes, independent of LC3 homologue specificity [52] and to capture non-canonical modes of endolysosomal processing of mitochondria [161, 164, 166]. Furthermore,

A Detection of mitophagy and  
mitophagy receptor engagement  

B Mitochondrial quality control sensors  

C Mitophagy pathway sensors  

D LC3B Bnip3 LIR interaction  
evidenced by co-immunoprecipitation  

Fig. 8 Methodologies to measure mitophagy. Biochemical and imaging-based assays can measure different aspects of mitophagy program activities. a Mitochondria-containing autophagosomes can be detected via imaging of fluorescent protein (FP)-LC3-labeled autophagosome colocalization with immunofluorescence or FP-labeled mitochondria. Western blot and immunofluorescence detection of IMM and matrix proteins is most specific for detecting mitophagic degradation events. Mito-autolysosomes can be detected using FP-Rab7. b MitoTimer is a tetramer which matures as a green-to-red fluorescent protein. MitoTimer can be used under inducible- and constitutive-expression to analyze mitochondrial quality control dynamics. c Mitochondrial entry into the autolysosome and subsequent degradation can be measured using FP sensors targeted to mitochondria, that are sensitive to low pH and resistant to degradation by lysosomal hydrolases. Mito-Keima fluoresces green at neutral pH in the cytosol, and red upon entry into acidic autolysosomes.  

Tandem is a mitochondria-targeted RFP-GFP fusion. GFP fluorescence is acid-quenched while RFP fluorescence remains stable also at low pH, permitting live cell analysis of mitochondrial presence in autolysosomes. Note, lysosomal hydrolases degrade GFP more efficiently than RFP, permitting detection of mitophagy in fixed cells. d Example co-immunoprecipitation demonstrating phospho-regulated Bnip3 mitophagy receptor engagement, i.e., Bnip3 LIR binding with LC3B. MCF7 cells stably expressing GFP-LC3B were transfected with RFP-Bnip3 WT, or LIR-inactive 2SA or LIR-activated 2SE mutants. At 48 h of expression, co-immunoprecipitations (IP) were performed with anti-GFP antibody-coupled magnetic beads. Whole cell lysates (input) and IP samples were analyzed by western blotting (WB). This research was originally published in Zhu et al. [52]. © The American Society for Biochemistry and Molecular Biology  

the GFP-LC3-G120A mutant which cannot be conjugated to PE and insert into autophagosomal membranes [39] is applicable to ensure that mitochondrial localization of GFP-LC3 corresponds to autophagosome sequestration [52].  

Importantly, when detecting mitochondria using fluorescent dyes, caution and proper controls are paramount. MitoTracker dyes accumulate inside mitochondria based on their membrane potential, and therefore will not efficiently label dysfunctional mitochondria. Moreover, different mitochondrial dyes may be retained differently within mitochondria, dependent on, or independent of, the mitochondrial polarization state [167]. Therefore, strict controls need to be employed to distinguish between changes in mitochondrial mass and the fraction of polarized mitochondria. Alternatively, mitochondria-targeted fluorescent proteins can be used to detect mitochondria [168]. However, also here it needs to be considered that protein import into mitochondria is lost following depolarization [169], possibly resulting in mislocalization of reporter proteins to the
cytosol, depending on the experimental time scales. Thus, mitochondria-targeted fluorescent protein sensors may not be suitable for all applications. The use of antibodies targeting endogenous mitochondrial proteins, for mitophagy detection by either immunofluorescence or Western blotting, can circumvent this obstacle. When choosing mitochondrial protein readouts, it is of note that Parkin can activate the proteasomal degradation of OMM proteins in response to CCCP in the absence of mitophagy [170]. Assessing degradation of multiple mitochondrial proteins revealed that cytochrome c and VDAC1 are more rapidly degraded, than Complex Va, Complex III Core 1 and cyclophilin D [171], suggesting the latter proteins as more suitable readouts when comparing autophagic capacity between cell types and conditions.

### Mitochondria-targeted maturation-based mitophagy sensors

As a complementary approach, the use of high-content sensors can provide spatio-temporal information concerning mitochondrial biogenesis and lysosomal processing. The MitoTimer sensor (Fig. 8b) was developed based on the tetrameric fluorescent protein drFP583, a derivative of DsRed2, which emits green fluorescence when synthesized, and then progressively shifts to red fluorescence upon oxidation [172]. Two approaches have been developed to extrapolate mitophagy dynamics from MitoTimer fluorescence maturation kinetics. First, using pulsed biosensor expression via a doxycycline-inducible vector, the temporal evolution of red-to-green fluorescence ratio can be used to evidence both mitochondrial turnover and biogenesis [173]. Alternatively, using a constitutive promoter, changes in MitoTimer red-to-green ratio were used to detect changes to mitophagy dynamics in vivo [174]. Of note, while oxidation underlies the green-to-red conversion, it is not fully understood to what extent mitochondrial dysfunction, i.e., ROS production, would influence maturation [172]. A similar tool is mt-Keima (Fig. 8c), which is a mitochondrial matrix-targeted ratiometric pH-sensitive fluorescent protein that fluoresces red at low pH and green at higher pH [175]. Mt-Keima directly reports entry of mitochondria into lysosomes [89], and was used to evidence endogenous activity of Parkin-mediated mitophagy [89].

### RFP–GFP tandem-based mitophagy sensors

An alternative biosensor approach targets a tandem RFP–GFP fusion protein to mitochondria (Fig. 8c), where under live cell conditions GFP fluorescence is quenched under acidic conditions of the late endosome/lysosome, while RFP fluorescence remains stable [176]. RFP–GFP targeted to the OMM has been applied to report Parkin-induced mitophagy, and was used for a drug screen, identifying iron chelators as Parkin-independent activators of mitophagy [177]. Similarly, an IMM-targeted RFP–GFP tandem sensor was used to evidence hepatitis B virus-induced activation of mitophagy [178].

For all above fluorescence-based mitophagy assays, an important consideration is that each cell has 100s–1000s of mitochondria, making manual scoring of high-resolution imaging difficult and subjective. Thus, rigorous quantification of single cells and cell populations is required. Moreover, it is necessary to analyze total cellular fluorescence through the Z-axis, as the mitophagic response is not uniform throughout all optical slices of a cell. For imaging, the use of high-resolution objectives (60×–100×) and optical sectioning with 300 nm step sizes is advisable. Subsequently, using image analysis software, e.g., Fiji/ImageJ (http://fiji.sc/Fiji), plane-by-plane co-localization can be applied to quantify the extent of mitophagy. Alternatively, images can be manually segmented, and the fraction of mitochondria that is co-localized with autophagosomes can be extrapolated as a fractional cell response. Importantly, a large sample of images must be analyzed to provide meaningful statistics. In addition, high sampling of cells (100s–1000s), as can be achieved using image cytometry [52], or flow cytometry [173], provides a quantitative measure of the number of cells in a population exhibiting a mitophagy response, as well as the degree of response per cell. Of note, cytometry experiments require attention to resolve fluorescence from noise. Modern systems can have up to an 18-bit dynamic range, e.g., a 0–262,144 lower and upper value range. As a rule of thumb, using a log-scale, specific signals should be placed in the four-decade (10⁴/10,000) to five-decade (10⁵/100,000) lower and upper end-linear value ranges to ensure accurate and meaningful measurements.

### Monitoring mitophagy receptor activity using LC3 immunoprecipitation

Finally, a critical tool in the analysis of LIR receptor activities is the immunoprecipitation of tagged-LC3, and analysis of co-precipitated protein by Western blot. Typically this is performed using either of the following two approaches. Cell lysates expressing a receptor of interest can be incubated with purified GST-tagged LC3 proteins [64]. Alternatively, a tagged LC3 (e.g., GFP-LC3), can be co-expressed with a receptor of interest, and immunoprecipitated from the cell lysate [52] (Fig. 8d). Using this latter approach, stable expression of tagged-LC3B/GATE16 in combination with transient transfection of the protein of interest is recommendable to provide best reproducibility. In order to establish LIR-dependency of an interaction with LC3, mutations can be introduced in the putative LIR, e.g., tryptophan (W)-to-alanine (A) and/orleucine (L)-to-A, yielding core LIR-inactive mutants. Serine/threonine(S/T)-to-glutamate (E) or aspartate (D) mutations can be introduced to mimic the phosphorylated state, while S/T-to-A mutation prevents phosphorylation [46, 52]. In addition, basic residues at the W and L hydrophobic pockets of LC3 proteins, e.g., LC3B lysine 49 [49], which interact with acidic/phosphorylated residues preceding the receptor LIR motif, can be mutated to further establish specificity of receptor binding. Of note, mitophagy can be a slow process, and cell line-dependently engaged 24–48 h following transfection.

### Concluding remarks

Mitochondrial dysfunction is implicated in nearly all disease, and recent discoveries suggest paths toward therapeutic manipulation of mitophagy. To that end, the continued elucidation of mechanisms underlying positive and negative mitophagy regulation, as well as roles for mitophagy during normal and stressed conditions will be instrumental in developing treatment strategies. Among the important next steps will be (1) the identification of the kinases and phosphatases that regulate LIR engagement of Bnip3 and Nix to determine in vivo mitophagy significance; (2) the elucidation of how morphology dynamics underlie mitochondrial triage, prior to mitophagy engagement; and (3) an understanding of crosstalk between apoptosis and mitophagy programs, i.e., how mitophagy behavior can regulate mitochondrial participation in programmed cell death. To these ends, RNAi and drug screening using high-content mitophagy sensors will be increasingly important tools in functional and mechanistic elucidation.

#### Acknowledgments
This work was supported by the German Cancer Research Center (DKFZ), through SBCancer within the Helmholtz Alliance on Systems Biology funded by the Initiative and Networking Fund of the Helmholtz Association (NRB); and the ImmunoQuant Grant #0316170B (NRB) and e:Bio Grant #0316191 ‘LysoSys’ (AHB) of the Federal Ministry of Education and Research, Germany.

#### Open Access
This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ([http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/)), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made.

### References

1. Posakony JW, England JM, Attardi G (1977) Mitochondrial growth and division during the cell cycle in HeLa cells. J Cell Biol 74:468–491
2. Barth E, Stammler G, Speiser B, Schaper J (1992) Ultrastructural quantitation of mitochondria and myofilaments in cardiac muscle from 10 different animal species including man. J Mol Cell Cardiol 24:669–681
3. Collins TJ, Berridge MJ, Lipp P, Bootman MD (2002) Mitochondria are morphologically and functionally heterogeneous within cells. EMBO J 21:1616–1627
4. Birkedal R, Shiels HA, Vendelin M (2006) Three-dimensional mitochondrial arrangement in ventricular myocytes: from chaos to order. Am J Physiol Cell Physiol 291:C1148–C1158
5. Cheng A, Wan R, Yang JL, Kamimura N, Son TG, Ouyang X, Luo Y, Okun E, Mattson MP (2011) Involvement of PGC-1alpha in the formation and maintenance of neuronal dendritic spines. Nat Commun 3:1250
6. Sena LA, Chandel NS (2012) Physiological roles of mitochondrial reactive oxygen species. Mol Cell 48:158–167
7. Rizzuto R, Bernardi P, Pozzan T (2000) Mitochondria as all-round players of the calcium game. J Physiol 529(Pt 1):37–47
8. Green DR, Kroemer G (2004) The pathophysiology of mitochondrial cell death. Science 305:626–629
9. Viale A, Pettazzoni P, Lyssiotis CA, Ying H, Sanchez N, Marchesini M, Carugo A, Green T, Seth S, Giuliani V, Kost-Alimova M, Muller F, Colla S, Nezi L, Genovese G, Deem AK, Kapoor A, Yao W, Brunetto E, Kang Y, Yuan M, Asara JM, Wang YA, Heffernan TP, Kimmelman AC, Wang H, Fleming JB, Cantley LC, DePinho RA, Draetta GF (2014) Oncogene ablation-resistant pancreatic cancer cells depend on mitochondrial function. Nature 514:628–632
10. Weinberg SE, Chandel NS (2015) Targeting mitochondria metabolism for cancer therapy. Nat Chem Biol 11:9–15
11. Mitchell P (1961) Coupling of phosphorylation to electron and hydrogen transfer by a chemi-osmotic type of mechanism. Nature 191:144–148
12. Murphy MP (2009) How mitochondria produce reactive oxygen species. Biochem J 417:1–13
13. Hamanaka RB, Chandel NS (2010) Mitochondrial reactive oxygen species regulate cellular signaling and dictate biological outcomes. Trends Biochem Sci 35:505–513
14. Green DR, Galluzzi L, Kroemer G (2011) Mitochondria and the autophagy-inflammation-cell death axis in organismal aging. Science 333:1109–1112
15. Chan DC (2006) Mitochondria: dynamic organelles in disease, aging, and development. Cell 125:1241–1252
16. Itoh K, Nakamura K, Iijima M, Sesaki H (2013) Mitochondrial dynamics in neurodegeneration. Trends Cell Biol 23:64–71
17. Wallace DC (1999) Mitochondrial diseases in man and mouse. Science 283:1482–1488
18. Youle RJ, Narendra DP (2011) Mechanisms of mitophagy. Nat Rev Mol Cell Biol 12:9–14
19. Tal MC, Sasai M, Lee HK, Yordy B, Shadel GS, Iwasaki A (2009) Absence of autophagy results in reactive oxygen species-dependent amplification of RLR signaling. Proc Natl Acad Sci USA 106:2770–2775
20. Egan DF, Shackelford DB, Mihaylova MM, Gelino S, Kohnz RA, Mair W, Vasquez DS, Joshi A, Gwinn DM, Taylor R, Asara JM, Fitzpatrick J, Dillin A, Viollet B, Kundu M, Hansen M, Shaw RJ (2011) Phosphorylation of ULK1 (hATG1) by AMP-activated protein kinase connects energy sensing to mitophagy. Science 331:456–461
21. Ashford TP, Porter KR (1962) Cytoplasmic components in hepatic cell lysosomes. J Cell Biol 12:198–202
22. Mishra P, Chan DC (2014) Mitochondrial dynamics and inheritance during cell division, development and disease. Nat Rev Mol Cell Biol 15:634–646
23. Mattenberger Y, James DI, Martinou JC (2003) Fusion of mitochondria in mammalian cells is dependent on the

Springer
mitochondrial inner membrane potential and independent of
microtubules or actin. FEBS Lett 538:53–59
24. Twig G, Elorza A, Molina AJ, Mohamed H, Wikstrom JD,
Walzer G, Stiles L, Haigh SE, Katz S, Las G, Alroy J, Wu M, Py
BF, Yuan J, Deeney JT, Corkey BE, Shirihai OS (2008) Fission
and selective fusion govern mitochondrial segregation and
elimination by autophagy. EMBO J 27:433–446
25. Arnoult D, Rismanchi N, Grodet A, Roberts RG, Seeburg DP,
Estaquier J, Sheng M, Blackstone C (2005) Bax/Bak-dependent
release of DDP/TIMM8a promotes Drp1-mediated mitochondrial
fission and mitoptosis during programmed cell death. Curr
Biol 15:2112–2118
26. Gomes LC, Di Benedetto G, Scorrano L (2011) During autophagy mitochondria elongate, are spared from degradation and
sustain cell viability. Nat Cell Biol 13:589–598
27. Rambold AS, Kostelecky B, Elia N, Lippincott-Schwartz J
(2011) Tubular network formation protects mitochondria from
autophagosomal degradation during nutrient starvation. Proc
Natl Acad Sci USA 108:10190–10195
28. Song M, Mihara K, Chen Y, Scorrano L, Dorn GW 2nd (2015)
Mitochondrial fission and fusion factors reciprocally orchestrate
mitophagic culling in mouse hearts and cultured fibroblasts. Cell
Metab 21:273–285
29. Rabinowitz JD, White E (2010) Autophagy and metabolism.
Science 330:1344–1348
30. Mizushima N, Yoshimori T, Ohsumi Y (2011) The role of Atg
proteins in autophagosome formation. Annu Rev Cell Dev Biol
27:107–132
31. Lamb CA, Yoshimori T, Tooze SA (2013) The autophagosome:
origins unknown, biogenesis complex. Nat Rev Mol Cell Biol
14:759–774
32. Mizushima N (2007) Autophagy: process and function. Genes
Dev 21:2861–2873
33. Jager S, Bucci C, Tanida I, Ueno T, Kominami E, Saftig P,
Eskelinen EL (2004) Role for Rab7 in maturation of late
autophagic vacuoles. J Cell Sci 117:4837–4848
34. Gutierrez MG, Munafo DB, Beron W, Colombo MI (2004) Rab7
is required for the normal progression of the autophagic pathway
in mammalian cells. J Cell Sci 117:2687–2697
35. Popovic D, Akutsu M, Novak I, Harper JW, Behrends C, Dikic I
(2012) Rab GTPase-activating proteins in autophagy: regulation
of endocytic and autophagy pathways by direct binding to
human ATG8 modifiers. Mol Cell Biol 32:1733–1744
36. McEwan DG, Popovic D, Gubas A, Terawaki S, Suzuki H,
Stadel D, Coxon FP, Miranda de Stegmann D, Bhogaraju S,
Maddi K, Kirchof A, Gatti E, Helfrich MH, Wakatsuki S,
Behrends C, Pierre P, Dikic I (2015) PLEKHM1 regulates
autophagosome-lysosome fusion through HOPS complex and
LC3/GABARAP proteins. Mol Cell 57:39–54
37. Jiang P, Nishimura T, Sakamaki Y, Itakura E, Hatta T, Natsume
T, Mizushima N (2014) The HOPS complex mediates
autophagosome-lysosome fusion through interaction with syntaxin 17. Mol Biol Cell 25:1327–1337
38. Itakura E, Kishi-Itakura C, Mizushima N (2012) The hairpin-
type tail-anchored SNARE syntaxin 17 targets to autophagosomes for fusion with endosomes/lysosomes. Cell
151:1256–1269
39. Kabeya Y, Mizushima N, Ueno T, Yamamoto A, Kirisako T,
Noda T, Kominami E, Ohsumi Y, Yoshimori T (2000) LC3, a
mammalian homologue of yeast Apg8p, is localized in
autophagosome membranes after processing. EMBO J
19:5720–5728
40. Tanida I, Ueno T, Kominami E (2004) LC3 conjugation system
in mammalian autophagy. Int J Biochem Cell Biol
36:2503–2518

41. Nakatogawa H (2013) Two ubiquitin-like conjugation systems
that mediate membrane formation during autophagy. Essays
Biochem 55:39–50
42. Cherra SJ 3rd, Kulich SM, Uechi G, Balasubramani M,
Mountzouris J, Day BW, Chu CT (2010) Regulation of the
autophagy protein LC3 by phosphorylation. J Cell Biol
190:533–539
43. Wilkinson DS, Jariwala JS, Anderson E, Mitra K, Meisenhelder
J, Chang JT, Ideker T, Hunter T, Nizet V, Dillin A, Hansen M
(2015) Phosphorylation of LC3 by the Hippo kinases STK3/
STK4 is essential for autophagy. Mol Cell 57:55–68
44. Weidberg H, Shvets E, Shpilka T, Shimron F, Shinder V, Elazar
Z (2010) LC3 and GATE-16/GABARAP subfamilies are both
essential yet act differently in autophagosome biogenesis.
EMBO J 29:1792–1802
45. Jain A, Lamark T, Sjottem E, Larsen KB, Awuh JA, Overvatn
A, McMahon M, Hayes JD, Johansen T (2010) p62/SQSTM1 is
a target gene for transcription factor NRF2 and creates a positive
feedback loop by inducing antioxidant response element-driven
gene transcription. J Biol Chem 285:22576–22591
46. Wild P, Farhan H, McEwan DG, Wagner S, Rogov VV, Brady
NR, Richter B, Korac J, Waidmann O, Choudhary C, Dotsch V,
Bumann D, Dikic I (2011) Phosphorylation of the autophagy
receptor optineurin restricts Salmonella growth. Science
333:228–233
47. Birgisdottir AB, Lamark T, Johansen T (2013) The LIR motif—
crucial for selective autophagy. J Cell Sci 126:3237–3247
48. Noda NN, Kumeta H, Nakatogawa H, Satoo K, Adachi W, Ishii
J, Fujioka Y, Ohsumi Y, Inagaki F (2008) Structural basis of
target recognition by Atg8/LC3 during selective autophagy.
Genes Cells 13:1211–1218
49. Noda NN, Ohsumi Y, Inagaki F (2010) Atg8-family interacting
motif crucial for selective autophagy. FEBS Lett 584:1379–1385
50. Pankiv S, Clausen TH, Lamark T, Brech A, Bruun JA, Outzen
H, Overvatn A, Bjorkoy G, Johansen T (2007) p62/SQSTM1
binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy. J Biol Chem
282:24131–24145
51. Ichimura Y, Kumanomidou T, Sou YS, Mizushima T, Ezaki J,
Ueno T, Kominami E, Yamane T, Tanaka K, Komatsu M (2008)
Structural basis for sorting mechanism of p62 in selective
autophagy. J Biol Chem 283:22847–22857
52. Zhu Y, Massen S, Terenzio M, Lang V, Chen-Lindner S, Eils R,
Novak I, Dikic I, Hamacher-Brady A, Brady NR (2013)
Modulation of serines 17 and 24 in the LC3-interacting region of
Bnip3 determines pro-survival mitophagy versus apoptosis. J Biol Chem 288:1099–1113
53. Liu L, Feng D, Chen G, Chen M, Zheng Q, Song P, Ma Q, Zhu
C, Wang R, Qi W, Huang L, Xue P, Li B, Wang X, Jin H, Wang
J, Yang F, Liu P, Zhu Y, Sui S, Chen Q (2012) Mitochondrial
outer-membrane protein FUNDC1 mediates hypoxia-induced
mitophagy in mammalian cells. Nat Cell Biol 14:177–185
54. Narendra D, Tanaka A, Suen DF, Youle RJ (2008) Parkin is
recruited selectively to impaired mitochondria and promotes
their autophagy. J Cell Biol 183:795–803
55. Wong YC, Holzbaur EL (2014) Optineurin is an autophagy
receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an ALS-linked mutation. Proc Natl
Acad Sci USA 111:E4439–E4448
56. Chan NC, Salazar AM, Pham AH, Sweredoski MJ, Kolawa NJ,
Graham RL, Hess S, Chan DC (2011) Broad activation of the
ubiquitin-proteasome system by Parkin is critical for mitophagy. Hum Mol Genet 20:1726–1737
57. Sarraf SA, Raman M, Guarani-Pereira V, Sowa ME, Huttlin EL,
Gygi SP, Harper JW (2013) Landscape of the PARKIN-

dependent ubiquitylome in response to mitochondrial depolar-
ization. Nature 496:372–376
58. Van Humbeeck C, Cornelissen T, Hofkens H, Mandemakers W,
Gevaert K, De Strooper B, Vandenberghe W (2011) Parkin
interacts with Ambra1 to induce mitophagy. J Neurosci
31:10249–10261
59. Strappazzon F, Vietri-Rudan M, Campello S, Nazio F, Floren-
zano F, Fimia GM, Piacentini M, Levine B, Cecconi F (2011)
Mitochondrial BCL-2 inhibits AMBRA1-induced autophagy.
EMBO J 30:1195–1208
60. Hamacher-Brady A, Brady NR, Logue SE, Sayen MR, Jinno M,
Kirshenbaum LA, Gottlieb RA, Gustafsson AB (2007) Response
to myocardial ischemia/reperfusion injury involves Bnip3 and
autophagy. Cell Death Differ 14:146–157
61. Hanna RA, Quinsay MN, Orog AM, Giang K, Rikka S,
Gustafsson AB (2012) Microtubule-associated protein 1 light
chain 3 (LC3) interacts with Bnip3 protein to selectively remove
endoplasmic reticulum and mitochondria via autophagy. J Biol
Chem 287:19094–19104
62. Schweers RL, Zhang J, Randall MS, Loyd MR, Li W, Dorsey
FC, Kundu M, Opferman JT, Cleveland JL, Miller JL, Ney PA
(2007) NIX is required for programmed mitochondrial clearance
during reticulocyte maturation. Proc Natl Acad Sci USA
104:19500–19505
63. Sandoval H, Thiagarajan P, Dasgupta SK, Schumacher A,
Prchal JT, Chen M, Wang J (2008) Essential role for Nix in
autophagic maturation of erythroid cells. Nature 454:232–235
64. Novak I, Kirkin V, McEwan DG, Zhang J, Wild P, Rozenknop A,
Rogov V, Lohr F, Popovic D, Occhipinti A, Reichert AS, Terzic J,
Dotsch V, Ney PA, Dikic I (2010) Nix is a selective autophagy
receptor for mitochondrial clearance. EMBO Rep 11:45–51
65. Murakawa T, Yamaguchi O, Hashimoto A, Hikoso S, Takeda T,
Oka T, Yasui H, Ueda H, Akazawa Y, Nakayama H, Taneike M,
Misaka T, Omiya S, Shah AM, Yamamoto A, Nishida K,
Ohsumi Y, Okamoto K, Sakata Y, Otsu K (2015) Bcl-2-like
protein 13 is a mammalian Atg32 homologue that mediates
mitophagy and mitochondrial fragmentation. Nat Commun
6:7527
66. Chen G, Ray R, Dubik D, Shi L, Cizeau J, Bleackley RC,
Saxena S, Gietz RD, Greenberg AH (1997) The E1B 19K/Bcl-2-
binding protein Nip3 is a dimeric mitochondrial protein that
activates apoptosis. J Exp Med 186:1975–1983
67. Chen G, Cizeau J, Vande Velde C, Park JH, Bozek G, Bolton J,
Shi L, Dubik D, Greenberg A (1999) Nix and Nip3 form a
subfamily of pro-apoptotic mitochondrial proteins. J Biol Chem
274:7–10
68. Kataoka T, Holler N, Micheau O, Martinon F, Tinel A, Hof-
mann K, Tschopp J (2001) Bcl-rambo, a novel Bcl-2 homologue
that induces apoptosis via its unique C-terminal extension. J Biol
Chem 276:19548–19554
69. Sentelle RD, Senkal CE, Jiang W, Ponnamasy S, Gencer S,
Selvam SP, Ramsesh VK, Peterson YK, Lemasters JJ, Szulc
ZM, Bielawski J, Ogretmen B (2012) Ceramide targets
autophagosomes to mitochondria and induces lethal mitophagy.
Nat Chem Biol 8:831–838
70. Chu CT, Ji J, Dagda RK, Jiang JF, Tyurina YY, Kapralov AA,
Tyurin VA, Yanamala N, Shrivastava IH, Mohammadyan D,
Qiang Wang KZ, Zhu J, Klein-Seetharaman J, Balasubramanian
K, Amoscato AA, Borisenko G, Huang Z, Gusdon AM, Cheikhi
A, Steer EK, Wang R, Baty C, Watkins S, Bahar I, Bayir H,
Kagan VE (2013) Cardiolipin externalization to the outer
mitochondrial membrane acts as an elimination signal for
mitophagy in neuronal cells. Nat Cell Biol 15:1197–1205
71. Pickrell AM, Youle RJ (2015) The roles of PINK1, parkin, and
mitochondrial fidelity in Parkinson's disease. Neuron
85:257–273

72. Jin SM, Lazarou M, Wang C, Kane LA, Narendra DP, Youle RJ
(2010) Mitochondrial membrane potential regulates PINK1
import and proteolytic destabilization by PARL. J Cell Biol
191:933–942
73. Matsuda N, Sato S, Shiba K, Okatsu K, Saisho K, Gautier CA,
Sou YS, Saiki S, Kawajiri S, Sato F, Kimura M, Komatsu M,
Hattori N, Tanaka K (2010) PINK1 stabilized by mitochondrial
depolarization recruits Parkin to damaged mitochondria and
activates latent Parkin for mitophagy. J Cell Biol 189:211–221
74. Chen Y, Dorn GW 2nd (2013) PINK1-phosphorylated mitofusin
2 is a Parkin receptor for culling damaged mitochondria. Science
340:471–475
75. Kane LA, Lazarou M, Fogel AI, Li Y, Yamano K, Sarraf SA,
Banerjee S, Youle RJ (2014) PINK1 phosphorylates ubiquitin to
activate Parkin E3 ubiquitin ligase activity. J Cell Biol
205:143–153
76. Koyano F, Okatsu K, Kosako H, Tamura Y, Go E, Kimura M,
Kimura Y, Tsuchiya H, Yoshihara H, Hirokawa T, Endo T, Fon
EA, Trempe JF, Saeki Y, Tanaka K, Matsuda N (2014) Ubiquitin is phosphorylated by PINK1 to activate parkin. Nature
510:162–166
77. Shiba-Fukushima K, Imai Y, Yoshida S, Ishihama Y, Kanao T,
Sato S, Hattori N (2012) PINK1-mediated phosphorylation of
the Parkin ubiquitin-like domain primes mitochondrial translo-
cation of Parkin and regulates mitophagy. Sci Rep 2:1002
78. Ordureau A, Sarraf SA, Duda DM, Heo JM, Jedrychowski MP,
Sviderskiy VO, Olszewski JL, Koerber JT, Xie T, Beausoleil
SA, Wells JA, Gygi SP, Schulman BA, Harper JW (2014)
Quantitative proteomics reveal a feedforward mechanism for
mitochondrial PARKIN translocation and ubiquitin chain syn-
thesis. Mol Cell 56:360–375
79. Geisler S, Holmstrom KM, Skujat D, Fiesel FC, Rothfuss OC,
Kahle PJ, Springer W (2010) PINK1/Parkin-mediated mito-
phagy is dependent on VDAC1 and p62/SQSTM1. Nat Cell Biol
12:119–131
80. Hollville E, Carroll RG, Cullen SP, Martin SJ (2014) Bcl-2 family
proteins participate in mitochondrial quality control by regulating
Parkin/PINK1-dependent mitophagy. Mol Cell 55:451–466
81. Narendra D, Kane LA, Hauser DN, Fearnley IM, Youle RJ
(2010) p62/SQSTM1 is required for Parkin-induced mitochon-
drial clustering but not mitophagy; VDAC1 is dispensable for
both. Autophagy 6:1090–1106
82. Okatsu K, Saisho K, Shimanuki M, Nakada K, Shitara H, Sou
YS, Kimura M, Sato S, Hattori N, Komatsu M, Tanaka K,
Matsuda N (2010) p62/SQSTM1 cooperates with Parkin for
perinuclear clustering of depolarized mitochondria. Genes Cells
15:887–900
83. Matsumoto G, Wada K, Okuno M, Kurosawa M, Nukina N
(2011) Serine 403 phosphorylation of p62/SQSTM1 regulates
selective autophagic clearance of ubiquitinated proteins. Mol
Cell 44:279–289
84. Fimia GM, Stoykova A, Romagnoli A, Giunta L, Di Bartolomeo
S, Nardacci R, Corazzari M, Fuoco C, Ucar A, Schwartz P,
Gruss P, Piacentini M, Chowdhury K, Cecconi F (2007) Ambra1
regulates autophagy and development of the nervous system.
Nature 447:1121–1125
85. Strappazzon F, Nazio F, Corrado M, Cianfanelli V, Romagnoli
A, Fimia GM, Campello S, Nardacci R, Piacentini M, Cam-
panella M, Cecconi F (2014) AMBRA1 is able to induce
mitophagy via LC3 binding, regardless of PARKIN and p62/
SQSTM1. Cell Death Differ 22:419–432
86. Lazarou M, Sliter DA, Kane LA, Sarraf SA, Wang C, Burman
JL, Sideris DP, Fogel AI, Youle RJ (2015) The ubiquitin kinase
PINK1 recruits autophagy receptors to induce mitophagy. Nat-
ure 524:309–314

87. Durcan TM, Tang MY, Perusse JR, Dashti EA, Aguileta MA, McLelland GL, Gros P, Shaler TA, Faubert D, Coulombe B, Fon EA (2014) USP 8 regulates mitophagy by removing K6-linked ubiquitin conjugates from parkin. EMBO J 33: 2473–2491

88. Nakamura N, Hirose S (2008) Regulation of mitochondrial morphology by USP 30, a deubiquitinating enzyme present in the mitochondrial outer membrane. Mol Biol Cell 19: 1903–1911

89. Bingol B, Tea JS, Phu L, Reichelt M, Bakalarski CE, Song Q, Foreman O, Kirkpatrick DS, Sheng M (2014) The mitochondrial deubiquitinase USP 30 opposes parkin-mediated mitophagy. Nature 510: 370–375

90. Chinnadurai G, Vijayalingam S, Gibson SB (2008) BNIP 3 subfamily BH 3-only proteins: mitochondrial stress sensors in normal and pathological functions. Oncogene 27(Suppl 1): S114–S127

91. Matsushima M, Fujiwara T, Takahashi E, Minaguchi T, Eguchi Y, Tsujimoto Y, Suzumori K, Nakamura Y (1998) Isolation, mapping, and functional analysis of a novel human cDNA (BNIP3L) encoding a protein homologous to human NIP 3. Genes Chromosomes Cancer 21: 230–235

92. Ray R, Chen G, Vande Velde C, Cizeau J, Park JH, Reed JC, Gietz RD, Greenberg AH (2000) BNIP 3 heterodimerizes with Bcl-2/Bcl-X(L) and induces cell death independent of a Bcl-2 homology 3 (BH 3) domain at both mitochondrial and nonmitochondrial sites. J Biol Chem 275: 1439–1448

93. Imazu T, Shimizu S, Tagami S, Matsushima M, Nakamura Y, Miki T, Okuyama A, Tsujimoto Y (1999) Bcl-2/E1B 19 kDa-interacting protein 3-like protein (Bnip3L) interacts with bcl-2/ Bcl-xL and induces apoptosis by altering mitochondrial membrane permeability. Oncogene 18: 4523–4529

94. Daido S, Kanzawa T, Yamamoto A, Takeuchi H, Kondo Y, Kondo S (2004) Pivotal role of the cell death factor BNIP 3 in ceramide-induced autophagic cell death in malignant glioma cells. Cancer Res 64: 4286–4293

95. Bellot G, Garcia-Medina R, Gounon P, Chiche J, Roux D, Pouyssegur J, Mazure NM (2009) Hypoxia-induced autophagy is mediated through hypoxia-inducible factor induction of BNIP 3 and BNIP3L via their BH 3 domains. Mol Cell Biol 29: 2570–2581

96. Li Y, Wang Y, Kim E, Beemiller P, Wang CY, Swanson J, You M, Guan KL (2007) Bnip3 mediates the hypoxia-induced inhibition on mammalian target of rapamycin by interacting with Rheb. J Biol Chem 282: 35803–35813

97. Melser S, Chatelain EH, Lavie J, Mahfouf W, Jose C, Obre E, Goorden S, Priault M, Elgersma Y, Rezvani HR, Rossignol R, Benard G (2013) Rheb regulates mitophagy induced by mitochondrial energetic status. Cell Metab 17: 719–730

98. Chen G, Han Z, Feng D, Chen Y, Chen L, Wu H, Huang L, Zhou C, Cai X, Fu C, Duan L, Wang X, Liu L, Liu X, Shen Y, Zhu Y, Chen Q (2014) A regulatory signaling loop comprising the PGAM 5 phosphatase and CK2 controls receptor-mediated mitophagy. Mol Cell 54: 362–377

99. Wu W, Tian W, Hu Z, Chen G, Huang L, Li W, Zhang X, Xue P, Zhou C, Liu L, Zhu Y, Li L, Zhang L, Sui S, Zhao B, Feng D (2014) ULK1 translocates to mitochondria and phosphorylates FUNDC1 to regulate mitophagy. EMBO Rep 15: 566–575

100. Wu H, Xue D, Chen G, Han Z, Huang L, Zhu C, Wang X, Jin H, Wang J, Zhu Y, Liu L, Chen Q (2014) The BCL2L1 and PGAM 5 axis defines hypoxia-induced receptor-mediated mitophagy. Autophagy 10: 1712–1725

101. Bian Y, Song C, Cheng K, Dong M, Wang F, Huang J, Sun D, Wang L, Ye M, Zou H (2014) An enzyme assisted RP-RPLC approach for in-depth analysis of human liver phosphoproteome. J Proteomics 96: 253–262

102. Ding WX, Ni HM, Li M, Liao Y, Chen X, Stolz DB, Dorn GW 2nd, Yin XM (2010) Nix is critical to two distinct phases of

mitophagy, reactive oxygen species-mediated autophagy induction and Parkin-ubiquitin-p 62-mediated mitochondrial priming. J Biol Chem 285: 27879–27890

103. Gao F, Chen D, Si J, Hu Q, Qin Z, Fang M, Wang G (2015) The mitochondrial protein BNIP3L is the substrate of PARK2 and mediates mitophagy in PINK1/PARK2 pathway. Hum Mol Genet 24: 2528–2538

104. Itakura E, Kishi-Itakura C, Koyama-Honda I, Mizushima N (2012) Structures containing Atg9A and the ULK1 complex independently target depolarized mitochondria at initial stages of Parkin-mediated mitophagy. J Cell Sci 125: 1488–1499

105. Kim J, Kundu M, Viollet B, Guan KL (2011) AMPK and mTOR regulate autophagy through direct phosphorylation of Ulk1. Nat Cell Biol 13: 132–141

106. Miyamoto T, Rho E, Sample V, Akano H, Magari M, Ueno T, Gorshkov K, Chen M, Tokumitsu H, Zhang J, Inoue T (2015) Compartmentalized AMPK signaling illuminated by genetically encoded molecular sensors and actuators. Cell Rep 11: 657–670

107. Yamano K, Fogel AI, Wang C, van der Bliek AM, Youle RJ (2014) Mitochondrial Rab GAPs govern autophagosome biogenesis during mitophagy. Elife 3: e01612

108. Settembre C, Di Malta C, Polito VA, Garcia Arencibia M, Vetrini F, Erdin S, Erdin SU, Huynh T, Medina D, Colella P, Sardiello M, Rubinsztein DC, Ballabio A (2011) TFEB links autophagy to lysosomal biogenesis. Science 332: 1429–1433

109. Settembre C, Zoncu R, Medina DL, Vetrini F, Erdin S, Huynh T, Ferron M, Karsenty G, Vellard MC, Facchinetti V, Sabatini DM, Ballabio A (2012) A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB. EMBO J 31: 1095–1108

110. Ma X, Godar RJ, Liu H, Diwan A (2012) Enhancing lysosome biogenesis attenuates BNIP 3-induced cardiomyocyte death. Autophagy 8: 297–309

111. Nezich CL, Wang C, Fogel AI, Youle RJ (2015) MiT/TFE transcription factors are activated during mitophagy downstream of Parkin and Atg5. J Cell Biol 210: 435–450

112. Bouman L, Schlierf A, Lutz AK, Shan J, Deinlein A, Kast J, Galehdar Z, Palmisano V, Patenge N, Berg D, Gasser T, Augustin R, Trumbach D, Irrcher I, Park DS, Wurst W, Kilberg MS, Tatzelt J, Winklhofer KF (2011) Parkin is transcriptionally regulated by ATF4: evidence for an interconnection between mitochondrial stress and ER stress. Cell Death Differ 18: 769–782

113. Zhang C, Lin M, Wu R, Wang X, Yang B, Levine AJ, Hu W, Feng Z (2011) Parkin, a p53 target gene, mediates the role of p53 in glucose metabolism and the Warburg effect. Proc Natl Acad Sci USA 108: 16259–16264

114. Hoshino A, Mita Y, Okawa Y, Ariyoshi M, Iwai-Kanai E, Ueyama T, Ikeda K, Ogata T, Matoba S (2013) Cytosolic p53 inhibits Parkin-mediated mitophagy and promotes mitochondrial dysfunction in the mouse heart. Nat Commun 4: 2308

115. da Costa CA, Sunyach C, Giaime E, West A, Corti O, Brice A, Safe S, Abou-Sleiman PM, Wood NW, Takahashi H, Goldberg MS, Shen J, Checler F (2009) Transcriptional repression of p53 by parkin and impairment by mutations associated with autosomal recessive juvenile Parkinson's disease. Nat Cell Biol 11: 1370–1375

116. Bruick RK (2000) Expression of the gene encoding the proapoptotic Nip3 protein is induced by hypoxia. Proc Natl Acad Sci USA 97: 9082–9087

117. Guo K, Searfoss G, Krolikowski D, Pagnoni M, Franks C, Clark K, Yu KT, Jaye M, Ivashchenko Y (2001) Hypoxia induces the expression of the pro-apoptotic gene BNIP3. Cell Death Differ 8: 367–376

118. Sowter HM, Ratcliffe PJ, Watson P, Greenberg AH, Harris AL (2001) HIF-1-dependent regulation of hypoxic induction of the

cell death factors BNIP3 and NIX in human tumors. Cancer Res 61:6669–6673

119. An HJ, Maeng O, Kang KH, Lee JO, Kim YS, Paik SG, Lee H (2006) Activation of Ras up-regulates pro-apoptotic BNIP3 in nitric oxide-induced cell death. J Biol Chem 281:33939–33948

120. Yurkova N, Shaw J, Blackie K, Weidman D, Jayas R, Flynn B, Kirshenbaum LA (2008) The cell cycle factor E2F-1 activates Bnip3 and the intrinsic death pathway in ventricular myocytes. Circ Res 102:472–479

121. Shaw J, Yurkova N, Zhang T, Gang H, Aguilar F, Weidman D, Scramstad C, Weisman H, Kirshenbaum LA (2008) Antagonism of E2F-1 regulated Bnip3 transcription by NF-kappaB is essential for basal cell survival. Proc Natl Acad Sci USA 105:20734–20739

122. Fei P, Wang W, Kim SH, Wang S, Burns TF, Sax JK, Buzzai M, Dicker DT, McKenna WG, Bernhard EJ, El-Deiry WS (2004) Bnip3L is induced by p53 under hypoxia, and its knockdown promotes tumor growth. Cancer Cell 6:597–609

123. Galvez AS, Brunskill EW, Marreez Y, Benner BJ, Regula KM, Kirshenbaum LA, Dorn GW 2nd (2006) Distinct pathways regulate proapoptotic Nix and BNip3 in cardiac stress. J Biol Chem 281:1442–1448

124. Mammucari C, Milan G, Romanello V, Masiero E, Rudolf R, Del Piccolo P, Burden SJ, Di Lisi R, Sandri C, Zhao J, Goldberg AL, Schiaffino S, Sandri M (2007) FoxO3 controls autophagy in skeletal muscle in vivo. Cell Metab 6:458–471

125. Bakker WJ, Harris IS, Mak TW (2007) FOXO3a is activated in response to hypoxic stress and inhibits HIF1-induced apoptosis via regulation of CITED2. Mol Cell 28:941–953

126. Barde I, Rauwel B, Marin-Florez RM, Corsinotti A, Laurenti E, Verp S, Offner S, Marquis J, Kapopoulou A, Vanicek J, Trono D (2013) A KRAB/KAP1-miRNA cascade regulates erythropoiesis through stage-specific control of mitophagy. Science 340:350–353

127. Li W, Zhang X, Zhuang H, Chen HG, Chen Y, Tian W, Wu W, Li Y, Wang S, Zhang L, Li L, Zhao B, Sui S, Hu Z, Feng D (2014) MicroRNA-137 is a novel hypoxia-responsive micro-RNA that inhibits mitophagy via regulation of two mitophagy receptors FUNDC1 and NIX. J Biol Chem 289:10691–10701

128. Chen X, Gong J, Zeng H, Chen N, Huang R, Huang Y, Nie L, Xu M, Xia J, Zhao F, Meng W, Zhou Q (2010) MicroRNA145 targets BNIP3 and suppresses prostate cancer progression. Cancer Res 70:2728–2738

129. Schapira AH, Cooper JM, Dexter D, Jenner P, Clark JB, Marsden CD (1989) Mitochondrial complex I deficiency in Parkinson's disease. Lancet 1:1269

130. Henchcliffe C, Beal MF (2008) Mitochondrial biology and oxidative stress in Parkinson disease pathogenesis. Nat Clin Pract Neurol 4:600–609

131. Geisler S, Holmstrom KM, Treis A, Skujat D, Weber SS, Fiesel FC, Kahle PJ, Springer W (2010) The PINK1/Parkin-mediated mitophagy is compromised by PD-associated mutations. Autophagy 6:871–878

132. Van Laar VS, Arnold B, Cassady SJ, Chu CT, Burton EA, Berman SB (2011) Bioenergetics of neurons inhibit the translocation response of Parkin following rapid mitochondrial depolarization. Hum Mol Genet 20:927–940

133. Cai Q, Zakaria HM, Simone A, Sheng ZH (2012) Spatial parkin translocation and degradation of damaged mitochondria via mitophagy in live cortical neurons. Curr Biol 22:545–552

134. Ashrafi G, Schlehe JS, LaVoie MJ, Schwarz TL (2014) Mitophagy of damaged mitochondria occurs locally in distal neuronal axons and requires PINK1 and Parkin. J Cell Biol 206:655–670

135. McLelland GL, Soubannier V, Chen CX, McBride HM, Fon EA (2014) Parkin and PINK1 function in a vesicular trafficking pathway regulating mitochondrial quality control. EMBOJ 33:282–295

136. Vincow ES, Merrihew G, Thomas RE, Shulman NJ, Beyer RP, MacCoss MJ, Pallanck LJ (2013) The PINK1-Parkin pathway promotes both mitophagy and selective respiratory chain turnover in vivo. Proc Natl Acad Sci USA 110:6400–6405

137. Sterky FH, Lee S, Wibom R, Olson L, Larsson NG (2011) Impaired mitochondrial transport and Parkin-independent degeneration of respiratory chain-deficient dopamine neurons in vivo. Proc Natl Acad Sci USA 108:12937–12942

138. Pickrell AM, Huang CH, Kennedy SR, Ordureau A, Sideris DP, Hoekstra JG, Harper JW, Youle RJ (2015) Endogenous Parkin preserves dopaminergic substantia nigral neurons following mitochondrial DNA mutagenic stress. Neuron 87:371–381

139. Glick D, Zhang W, Beaton M, Marsboom G, Gruber M, Simon MC, Hart J, Dorn GW 2nd, Brady MJ, Macleod KF (2012) BNip3 regulates mitochondrial function and lipid metabolism in the liver. Mol Cell Biol 32:2570–2584

140. Hamacher-Brady A, Brady NR, Gottlieb RA, Gustafsson AB (2006) Autophagy as a protective response to Bnip3-mediated apoptotic signaling in the heart. Autophagy 2:307–309

141. Aerbjorni W, Giattina M, Lee YT, Raffeld M, Miller JL (2003) The proapoptotic factor Nix is coexpressed with Bcl-xL during terminal erythroid differentiation. Blood 102:712–717

142. O'Sullivan TE, Johnson LR, Kang HH, Sun JC (2015) BNIP3-and BNIP3L-mediated mitophagy promotes the generation of natural killer cell memory. Immunity 43:331–342

143. Dorn GW 2nd (2010) Mitochondrial pruning by Nix and BNip3: an essential function for cardiac-expressed death factors. J Cardiovasc Transl Res 3:374–383

144. Ichim G, Lopez J, Ahmed SU, Muthalagu N, Giampazolias E, Delgado ME, Haller M, Riley JS, Mason SM, Athineos D, Parsons MJ, van de Kooij B, Bouchier-Hayes L, Chalmers AJ, Rooswinkel RW, Oberst A, Blyth K, Rehm M, Murphy DJ, Tait SW (2015) Limited mitochondrial permeabilization causes DNA damage and genomic instability in the absence of cell death. Mol Cell 57:860–872

145. Poulogiannis G, McIntyre RE, Dimitriadis M, Apps JR, Wilson CH, Ichimura K, Luo F, Cantley LC, Wyllie AH, Adams DJ, Arends MJ (2010) PARK2 deletions occur frequently in sporadic colorectal cancer and accelerate adenoma development in Apc mutant mice. Proc Natl Acad Sci USA 107:15145–15150

146. Gong Y, Zack TI, Morris LG, Lin K, Hukkelhoven E, Raheja R, Tan IL, Turcan S, Veeriah S, Meng S, Viale A, Schumacher SE, Palmedo P, Beroukhim R, Chan TA (2014) Pan-cancer genetic analysis identifies PARK2 as a master regulator of G1/S cyclins. Nat Genet 46:588–594

147. Sowter HM, Ferguson M, Pym C, Watson P, Fox SB, Han C, Harris AL (2003) Expression of the cell death genes BNip3 and NIX in ductal carcinoma in situ of the breast; correlation of BNip3 levels with necrosis and grade. J Pathol 201:573–580

148. Koukourakis MI, Giatromanolaki A, Polychronidis A, Simopoulos C, Gatter KC, Harris AL, Sivridis E (2006) Endogenous markers of hypoxia/anaerobic metabolism and anemia in primary colorectal cancer. Cancer Sci 97:582–588

149. Giatromanolaki A, Koukourakis MI, Sowter HM, Sivridis E, Gibson S, Gatter KC, Harris AL (2004) BNIP3 expression is linked with hypoxia-regulated protein expression and with poor prognosis in non-small cell lung cancer. Clin Cancer Res 10:5566–5571

150. Giatromanolaki A, Koukourakis MI, Gatter KC, Harris AL, Sivridis E (2008) BNIP3 expression in endometrial cancer relates to active hypoxia inducible factor 1alpha pathway and prognosis. J Clin Pathol 61:217–220

151. Holleman A, den Boer ML, de Menezes RX, Cheok MH, Cheng C, Kazemier KM, Janka-Schaub GE, Gobel U, Graubner UB, Evans WE, Pieters R (2006) The expression of 70 apoptosis genes in relation to lineage, genetic subtype, cellular drug

resistance, and outcome in childhood acute lymphoblastic leukemia. Blood 107:769–776

152. Murai M, Toyota M, Satoh A, Suzuki H, Akino K, Mita H, Sasaki Y, Ishida T, Shen L, Garcia-Manero G, Issa JP, Hinoda Y, Tokino T, Imai K (2005) Aberrant DNA methylation associated with silencing BNIP3 gene expression in haematopoietic tumours. Br J Cancer 92:1165–1172

153. Erkan M, Kleeff J, Esposito I, Giese T, Ketterer K, Buchler MW, Giese NA, Friess H (2005) Loss of BNIP3 expression is a late event in pancreatic cancer contributing to chemoresistance and worsened prognosis. Oncogene 24:4421–4432

154. Murai M, Toyota M, Suzuki H, Satoh A, Sasaki Y, Akino K, Ueno M, Takahashi F, Kusano M, Mita H, Yanagihara K, Endo T, Hinoda Y, Tokino T, Imai K (2005) Aberrant methylation and silencing of the BNIP3 gene in colorectal and gastric cancer. Clin Cancer Res 11:1021–1027

155. Ko YH, Cho YS, Won HS, Jeon EK, An HJ, Hong SU, Park JH, Lee MA (2013) Prognostic significance of autophagy-related protein expression in resected pancreatic ductal adenocarcinoma. Pancreas 42:829–835

156. Chourasia AH, Tracy K, Frankenberger C, Boland ML, Sharifi MN, Drake LE, Sachleben JR, Asara JM, Locasale JW, Karczmar GS, Macleod KF (2015) Mitophagy defects arising from BNip3 loss promote mammary tumor progression to metastasis. EMBO Rep 16:1145–1163

157. Maiuri MC, Zalckvar E, Kimchi A, Kroemer G (2007) Self-eating and self-killing: crosstalk between autophagy and apoptosis. Nat Rev Mol Cell Biol 8:741–752

158. Carroll RG, Hollville E, Martin SJ (2014) Parkin sensitizes toward apoptosis induced by mitochondrial depolarization through promoting degradation of Mcl-1. Cell Rep 9:1538–1553

159. Liang JR, Martinez A, Lane JD, Mayor U, Clague MJ, Urbe S (2015) USP30 deubiquitylates mitochondrial Parkin substrates and restricts apoptotic cell death. EMBO Rep 16:618–627

160. Soubannier V, McLelland GL, Zunino R, Braschi E, Rippstein P, Fon EA, McBride HM (2012) A vesicular transport pathway shuttles cargo from mitochondria to lysosomes. Curr Biol 22:135–141

161. Miyamoto Y, Kitamura N, Nakamura Y, Futamura M, Miyamoto T, Yoshida M, Ono M, Ichinose S, Arakawa H (2011) Possible existence of lysosome-like organelle within mitochondria and its role in mitochondrial quality control. PLoS One 6:e16054

162. Kitamura N, Nakamura Y, Miyamoto Y, Miyamoto T, Kabu K, Yoshida M, Futamura M, Ichinose S, Arakawa H (2011) Mieap, a p53-inducible protein, controls mitochondrial quality by repairing or eliminating unhealthy mitochondria. PLoS One 6:e16060

163. Nakamura Y, Kitamura N, Shinogi D, Yoshida M, Goda O, Murai R, Kamino H, Arakawa H (2012) BNIP3 and NIX mediate Mieap-induced accumulation of lysosomal proteins within mitochondria. PLoS One 7:e30767

164. Hamacher-Brady A, Choe SC, Krijnse-Locker J, Brady NR (2014) Intramitochondrial recruitment of endolysosomes mediates Smac degradation and constitutes a novel intrinsic apoptosis antagonizing function of XIAP E3 ligase. Cell Death Differ 21:1862–1876

165. Hamacher-Brady A, Brady NR (2015) Bax/Bak-dependent, Drp1-independent targeting of XIAP into inner-mitochondrial compartments counteracts Smac-dependent effector caspase activation. J Biol Chem 290(36):22005–22018

166. Sugiura A, McLelland GL, Fon EA, McBride HM (2014) A new pathway for mitochondrial quality control: mitochondrial-derived vesicles. EMBO J 33:2142–2156

167. Buckman JF, Hernandez H, Kress GJ, Votyakova TV, Pal S, Reynolds JJ (2001) MitoTracker labeling in primary neuronal and astrocytic cultures: influence of mitochondrial membrane potential and oxidants. J Neurosci Methods 104:165–176

168. Rizzuto R, Brini M, Pizzo P, Murgia M, Pozzan T (1995) Chimeric green fluorescent protein as a tool for visualizing subcellular organelles in living cells. Curr Biol 5:635–642

169. Wiedemann N, Frazier AE, Pfanner N (2004) The protein import machinery of mitochondria. J Biol Chem 279:14473–14476

170. Yoshii SR, Kishi C, Ishihara N, Mizushima N (2011) Parkin mediates proteasome-dependent protein degradation and rupture of the outer mitochondrial membrane. J Biol Chem 286:19630–19640

171. Baudot AD, Haller M, Mrschtkik M, Tait SW, Ryan KM (2014) Using enhanced-mitophagy to measure autophagic flux. Methods 75:105–111

172. Terskikh A, Fradkov A, Ermakova G, Zaraisky A, Tan P, Kajava AV, Zhao X, Lukyanov S, Matz M, Kim S, Weissman I, Siebert P (2000) “Fluorescent timer”: protein that changes color with time. Science 290:1585–1588

173. Hernandez G, Thornton C, Stotland A, Lui D, Sin J, Ramil J, Magee N, Andres A, Quarato G, Carreira RS, Sayen MR, Wolkowicz R, Gottlieb RA (2013) MitoTimer: a novel tool for monitoring mitochondrial turnover. Autophagy 9:1852–1861

174. Laker RC, Xu P, Ryall KA, Sujkowski A, Kenwood BM, Chain KH, Zhang M, Royal MA, Hoehn KL, Driscoll M, Adler PN, Wessells RJ, Saucerman JJ, Yan Z (2014) A novel MitoTimer reporter gene for mitochondrial content, structure, stress, and damage in vivo. J Biol Chem 289:12005–12015

175. Katayama H, Kogure T, Mizushima N, Yoshimori T, Miyawaki A (2011) A sensitive and quantitative technique for detecting autophagic events based on lysosomal delivery. Chem Biol 18:1042–1052

176. Kimura S, Noda T, Yoshimori T (2007) Dissection of the autophagosome maturation process by a novel reporter protein, tandem fluorescent-tagged LC3. Autophagy 3:452–460

177. Allen GF, Toth R, James J, Ganley IG (2013) Loss of iron triggers PINK1/Parkin-independent mitophagy. EMBO Rep 14:1127–1135

178. Kim SJ, Khan M, Quan J, Till A, Subramani S, Siddiqui A (2013) Hepatitis B virus disrupts mitochondrial dynamics: induces fission and mitophagy to attenuate apoptosis. PLoS Pathog 9:e1003722

179. Sugawara K, Suzuki NN, Fujioka Y, Mizushima N, Ohsumi Y, Inagaki F (2004) The crystal structure of microtubule-associated protein light chain 3, a mammalian homologue of Saccharomyces cerevisiae Atg8. Genes Cells 9:611–618

180. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE (2004) UCSF Chimera—a visualization system for exploratory research and analysis. J Comput Chem 25:1605–1612

181. Notredame C, Higgins DG, Heringa J (2000) T-Coffee: a novel method for fast and accurate multiple sequence alignment. J Mol Biol 302:205–217

182. Clamp M, Cuff J, Searle SM, Barton GJ (2004) The Jalview Java alignment editor. Bioinformatics 20:426–427
