
Microbiology and Molecular Biology Reviews

# Microbial Surface Colonization and Biofilm Development in Marine Environments

**Hongyue Dang**,<sup>a,b</sup> **Charles R. Lovell**<sup>c</sup>

State Key Laboratory of Marine Environmental Science, and College of Ocean and Earth Sciences, Xiamen University, Xiamen, China<sup>a</sup>; Joint Research Center for Carbon Sink by the Institute of Marine Microbes and Ecospheres, Xiamen University and the Qingdao Institute of BioEnergy and Bioprocess Technology, CAS, Qingdao, China<sup>b</sup>; Department of Biological Sciences, University of South Carolina, Columbia, South Carolina, USA<sup>c</sup>

---

## SUMMARY ..... 91
## INTRODUCTION ..... 92
- Physiological Advantages and Ecological Functions of Microbial Surface Association ..... 92
- Physiological Challenges and Deleterious Effects of Microbial Surface Association ..... 93
- Biogeochemical Contributions of Marine Surface-Associated Microbiota ..... 93
- Impacts of Surface-Associated Microbiota on Ocean Carbon Sequestration ..... 97
- Environmental Change-Induced Surface-Associated Microbiota Responses and Impacts ..... 98
## MICROBIAL SENSING AND SIGNALING IN SURFACE COLONIZATION AND BIOFILM DEVELOPMENT ..... 99
- Microbial Two-Component Signal Transduction Systems ..... 99
- Microbial Chemotaxis ..... 100
- Microbial Quorum Sensing ..... 101
- Posttranscriptional Regulation by Small RNAs ..... 102
- Centralized Regulation by Second Messengers ..... 102
- An Example of Microbial Interaction with Surfaces: Vibrio Chitin Utilization and Its Implications ..... 103
## MICROORGANISM-SURFACE INTERACTIONS IN SURFACE COLONIZATION ..... 105
- Surface Conditioning Film Formation and the “Masking Effect” ..... 105
- Key Microbial Surface Components for Colonization ..... 105
- The Holdfast, a Specialized Colonizing Apparatus in Primary Surface Colonizers ..... 106
## COOPERATION AND COMPETITION IN SHAPING THE COMPOSITION, DYNAMICS, AND FUNCTION OF MULTISPECIES MICROBIAL BIOFILMS ..... 106
- Coaggregation, a Common Mechanism for Microorganism Recruitment to Surfaces ..... 106
- Cheating: It Happens in the Microbial World, Too ..... 107
- Deadly Competition: Chemical Agents, Predation, and Specialized Weapons ..... 108
- Extracellular Membrane Vesicles as Mechanisms for Both Cooperation and Competition ..... 109
## KEY MICROBIAL SURFACE COLONIZERS IN MARINE ENVIRONMENTS ..... 110
- The Marine Roseobacter Clade ..... 110
- Marine Bacteroidetes ..... 110
- Marine Roseobacter Clade Bacteria and Bacteroidetes in Surface-Associated Community Dynamics ..... 111
## FUTURE PERSPECTIVES ON STUDIES OF MARINE SURFACE-ASSOCIATED MICROBIOTA ..... 112
## APPENDIX ..... 114
- Definitions ..... 114
## ACKNOWLEDGMENTS ..... 115
## REFERENCES ..... 115
## AUTHOR BIOS ..... 138

---

### SUMMARY

Biotic and abiotic surfaces in marine waters are rapidly colonized by microorganisms. Surface colonization and subsequent biofilm formation and development provide numerous advantages to these organisms and support critical ecological and biogeochemical functions in the changing marine environment. Microbial surface association also contributes to deleterious effects such as biofouling, biocorrosion, and the persistence and transmission of harmful or pathogenic microorganisms and their genetic determinants. The processes and mechanisms of colonization as well as key players among the surface-associated microbiota have been studied for several decades. Accumulating evidence indicates that specific cell-surface, cell-cell, and interpopulation interactions shape the composition, structure, spatiotemporal dynamics, and functions of surface-associated microbial communities. Several key microbial processes and mechanisms, including (i) surface, population, and community sensing and signaling, (ii) intraspecies and interspecies communication and interaction, and (iii) the regulatory balance between cooperation and competition, have been identified as critical for the microbial surface association lifestyle. In this review, recent progress in the study of marine microbial surface colonization and biofilm development is synthesized and discussed. Major gaps in our knowledge remain. We pose questions for targeted investigation of surface-specific community-level microbial features, answers to which would advance

---

Published 23 December 2015  
Citation Dang H, Lovell CR. 2016. Microbial surface colonization and biofilm development in marine environments. Microbiol Mol Biol Rev 80:91–138. doi:[10.1128/MMBR.00037-15](https://doi.org/10.1128/MMBR.00037-15).  
Address correspondence to Hongyue Dang, DangHY@xmu.edu.cn, or Charles R. Lovell, lovell@biol.sc.edu.  
Copyright © 2015, American Society for Microbiology. All Rights Reserved.

March 2016 Volume 80 Number 1 Microbiology and Molecular Biology Reviews mmbr.asm.org 91

Dang and Lovell

our understanding of surface-associated microbial community ecology and the biogeochemical functions of these communities at levels from molecular mechanistic details through systems biological integration.

# INTRODUCTION

Numerous kinds of surfaces with distinct physicochemical and biological properties exist in marine environments. These surfaces include living animal and algal surfaces, various kinds of particles and aggregates, inert or bioreactive mineral substrata, and submerged constructs and vessel surfaces. Diverse aquatic microorganisms are capable of colonizing surfaces of various kinds, leading to the formation of biofilms and to the development of specialized processes within these structures (1, 2). Surface association appears to be an ancient, universal, and fundamental survival mechanism that provides microorganisms with critical advantages, including greater access to nutritional resources, enhanced organism interactions, and greater environmental stability. These features are of particular importance in natural aquatic environments in which nutrients are often growth limiting and ambient conditions are highly dynamic and sometimes deleterious (1, 3, 4). Alterations (usually stimulation) of microbial activities by surfaces in soil environments were first reported more than a century ago (5, 6), and a similar surface-associated stimulation of microbial activities was subsequently found to be prevalent in marine environments as well (7). Key genetic and ecophysiological processes and mechanisms that are fundamental to the life of marine bacteria on surfaces have been revealed. Some up-to-date reviews on marine biofilm- or particle-associated microorganisms are available (e.g., see references 8–16). These reviews, albeit insightful, focus mostly on specific microbial groups, processes, functions, or colonizable substrata. Systematic reviews on the surface-associated microbiota and particularly the mechanisms that control the formation and development of surface-colonizing microbial communities in the marine environment are currently lacking. Because *Bacteria* are the most diverse and important (compositionally, dynamically, and functionally) microorganisms on marine surfaces and early colonizers may determine the structure, dynamics, and function of mature biofilm communities (17, 18), this review focuses on *Bacteria* and their processes and mechanisms related to early surface colonization, biofilm formation, and biofilm functions.

## Physiological Advantages and Ecological Functions of Microbial Surface Association

Surfaces submerged in marine water are rapidly colonized by microorganisms (13). As stated above, surface colonization and subsequent biofilm formation provide these organisms with important advantages. Perhaps the most critical of these advantages in the context of the marine environment is access to resources. Charged and hydrophobic materials tend to accumulate on submerged surfaces, and biogenic particles such as phytoplankton detritus, zooplankton fecal pellets, and marine snow are generally rich in organic matter, resulting in enhanced availability of inorganic macronutrients, organic carbon and energy sources, micronutrients, and electron donors or acceptors in otherwise strongly nutrient-limited milieus (1, 3, 19–22). Surfaces have been shown to be “hot spots” of microbially catalyzed, biogeochemically important activities, as described in greater detail below. Surface colonization and the production of the shielding biofilm matrix, antiprotozoal factors, and stress response products also promote protection from predators, viruses, antibiotics, and other chemical toxins and deleterious environmental pressures (1, 3, 13, 19, 21–28). The biofilm matrix and the development within it of specific microenvironments promote the maintenance of extracellular enzyme structural integrity and activities (23, 29) as well as improved opportunities for physiological homeostasis of the bacteria (1, 3, 23).

Interactions of microorganisms in close spatial juxtaposition within the biofilm matrix facilitate metabolic cooperation (1, 3, 19, 21, 22, 26, 30) and genetic exchanges due to both the physical structure of the biofilm and community-level communication among organisms (21, 22, 30, 31). Biofilms often feature open-channel and pore structures, enhancing solute and microbial transport and promoting frequent cell-cell contacts (29, 32). The establishment of high microbial densities and the sensing, signaling, and adaptive responses of these dense assemblages of surface-associated microbiota in turn promote within-population microbial diversification, between-population niche specialization, and higher-level microbial community organization (13, 21–23, 26, 30, 33–39). The enhanced and sometimes unique ecophysiological activities of surface- and biofilm-associated microbial communities lay the foundations for biogeochemical functions that can sharply differ from those of free-living (i.e., planktonic) microbial communities in marine environments. For example, biofilm-associated microbial communities may thrive in extreme or hostile environments where individual microorganisms would find the maintenance of activity and growth, even survival, challenging (40–42). Biofilm formation also contributes to the development of organic aggregates and high-molecular-weight (HMW) complexes that have dynamics distinct from those of their constituents. In addition, the stabilizing effects of life in biofilms facilitate the decomposition of sinking particles by surface-associated microbial communities, altering carbon sequestration efficiency and thus the climate modulation capacity of the ocean via the “biological pump” mechanism (i.e., the vertical transportation of photosynthetically produced organic carbon from the euphotic surface ocean to the dark deep ocean). Carbon remineralized during sedimentation reenters the carbon cycle and ocean-air exchange quickly, while carbon that is transported from the euphotic zone to the interior and sediments of the deep ocean may remineralize and circulate with a long residence time (43–46).

Considering the substantial physiological advantages of surface colonization and biofilm formation, it is not surprising that the surface lifestyle plays important roles in microbial adaptation to and biogeochemical functioning in marine environments (3). Specific regulatory networks that modulate the expression of numerous genes and metabolic pathways in surface-associated cells dictate the biogeochemical functions that result (3, 22, 47–49). Consequently, differences in physiological status and activities between surface-associated and free-living cells of the same taxon are often observed (e.g., see references 1, 6, and 47). Surface colonization may be particularly important for the expression and secretion of biopolymer-targeted extracellular enzymes and for the transition of microbial cells to the competent state, in which the uptake and incorporation of extracellular DNA (eDNA) are effective (31, 50, 51). Surface-associated microorganisms play important roles in numerous critical marine processes, including organic matter remineralization (23, 52–54), nutrient regeneration
and element cycling (23, 52, 53, 55), contaminant (such as heavy metal) concentration and transfer in food webs (23), induction of benthic invertebrate larval settlement (10, 13, 56), and xenobiotic compound biodegradation (21, 30, 32, 57). The unique genetic, physiological, and ecological processes, mechanisms, and functions associated with the surface-associated microbiota make their study a fascinating and productive area in microbiology, with important implications for basic marine science and for applied biotechnology and bioengineering.

### Physiological Challenges and Deleterious Effects of Microbial Surface Association

Surface-associated microorganisms also face challenges that free-living microorganisms may avoid (27, 58, 59). High cell densities on surfaces, and particularly in biofilms, promote intense competition for nutrients and other resources as well as progressive deterioration of conditions due to depletion of resources and accumulation of metabolic wastes. This may force some microorganisms into inactive states or even kill them (27, 59–62). Resource limitation and waste accumulation due to encapsulation in the biofilm matrix are of particular importance in deeper biofilm layers (35, 39). Thus, the spatial heterogeneity of biofilms and the activities of neighboring microorganisms can contribute to the formation of optimal, suboptimal, and adverse microniches for a given microorganism within the biofilm three-dimensional structure. Changes in the distributions of chemical species and other biota in a biofilm may also change the microenvironments and thus influence the activities of a surface-associated microorganism. Marine particles are prone to concentrate viruses (63), and viral attack is sometimes augmented in surface-associated microbial communities due to their high cell densities or enhanced virus production induced by the quorum sensing (QS) mechanism (64–66). Surfaces and biofilms also have the tendency to absorb heavy metals and toxic organic compounds (23, 32, 67). Surface-associated microorganisms generally switch off the expression of genes involved in motility and switch on the expression of genes involved in adhesion and biofilm development (36). Thus, biofilms may become inhibitory if the microorganisms cannot escape when conditions become deleterious (68). In spite of these challenges, surface association is a major mode of microbial life. In addition to the direct benefit provided by surface-associated growth-stimulating nutrient enrichment, the advantages provided by cell-cell interactions may be a driving force behind the common surface association lifestyle.

Microbial surface colonization and surface-associated metabolic activities also exert macroscale deleterious effects, including biofouling (13, 56), biocorrosion (18, 55, 69), and the persistence and transmission of harmful or pathogenic microorganisms and virulence determinants (23, 33, 70). Biocorrosion is a particularly important consequence of biofilm development in marine environments and has major impacts on marine engineering, causing extensive damage and economic losses worldwide (71, 72). In many coastal marine environments, there is a particularly severe carbon steel corrosion phenomenon called accelerated low water corrosion (ALWC) (73, 74). The rate of this intensified corrosion is usually 10 times that of common carbon steel corrosion in seawater (75). Despite its severity and ubiquity and numerous investigations of this phenomenon, the process and mechanism of ALWC remain poorly understood. Neither the causative organisms nor the environmental inducers of this process are completely resolved (74–76). Sulfur-oxidizing bacteria (SOB) and sulfate-reducing bacteria (SRB) were originally proposed to be the key microorganisms responsible for ALWC (76, 77). However, as ALWC occurs in the photic zone (just above the extremely low water level) where bulk anoxic conditions usually do not exist due to photosynthetic oxygen production in the daytime, SRB may not contribute to ALWC prior to the development of a thick corroding biofilm and anoxic microenvironments (18). Recently, it was found that the microaerophilic, neutrophilic, marine iron-oxidizing bacteria (FeOB) in the newly defined Zetaproteobacteria lineage (78–80) contribute to carbon steel corrosion in natural and simulated coastal environments (81, 82). Marine Zetaproteobacteria FeOB may also play an important role in the initiation of ALWC (18). A key environmental factor for the induction of ALWC appears to be elevated concentrations of seawater inorganic nitrogen compounds (especially nitrate) (75, 83, 84). Single-cell genomic analyses showed that nitrogen acquisition is particularly important for the marine iron-oxidizing Zetaproteobacteria, which usually harbor nitrate reductase genes (85). Some Zetaproteobacteria FeOB also harbor nitrogen fixation genes, and diazotrophy has been confirmed for some of these bacteria (85). The importance of nitrogen acquisition to the iron-oxidizing activity of marine Zetaproteobacteria FeOB stresses the importance of environmental nitrate (and other nitrogenous nutrients) for the induction of ALWC.

The importance of environmental nitrate for the development of ALWC is also highlighted by other lines of experimental evidence. Nitrate inhibits the sulfate reduction activity of SRB and thus is often employed to control oil reservoir souring caused by SRB sulfidogenesis (86–91), but at the same time, it usually accelerates the biocorrosion rate (especially via pitting corrosion as in ALWC) (92–94). Furthermore, many marine FeOB are facultative iron-oxidizing nitrate reducers (95–99), which may carry out iron corrosion using nitrate as a terminal electron acceptor under anoxic conditions (100, 101). It is likely that some FeOB that carry out nitrate reduction-coupled iron oxidation may play an important role in exacerbating ALWC before SRB take a part in further steel corrosion. It is reasonable to hypothesize that sequential steel surface colonization and ordered biogeochemical activities by different functional groups of microorganisms may thus play a critical role in the initiation and development of ALWC in marine environments. In-depth investigations of the surface physicochemical and nutritional microenvironment and the composition and dynamics of surface-associated microbiota on submerged steels may help to fully resolve the mechanism of ALWC and provide solutions limiting damage due to this process.

### Biogeochemical Contributions of Marine Surface-Associated Microbiota

Surfaces are clearly “hot spots” of microbial activities (53, 102–105). Compared to their free-living counterparts, surface-associated microorganisms usually possess distinct compositions (generally more diverse or more surface specific) (104, 106–110), morphologies (usually larger cells) (102, 106, 111, 112), abundances (often enriched) (53, 102, 106, 107, 109, 113), dynamics (often greater seasonal compositional variations and diel and even hourly activity changes) (102, 105, 114, 115), and functions (e.g., often higher per-cell exoenzymatic activities) (53, 102, 105–107, 110, 112, 116–118). It has been proposed that surface-associated microorganisms are mainly copiotrophic, whereas free-living bac-
Dang and Lovell

teria are mainly oligotrophic (119–121). However, some studies have shown similar or only slightly different particle (surface)-associated and free-living microbial communities (122–124). These studies were conducted in estuaries that are eutrophic and in which most suspended particles are old and composed mainly of inorganic matter having terrestrial or sediment origins. Particles in estuarine waters of this kind are generally less rich in organic matter than those in pelagic waters, except during algal bloom events (118). This would tend to minimize the advantage of particle colonization. Indeed, it has long been recognized that nutrient-enriched conditions inhibit bacterial irreversible adhesion to surfaces and biofilm formation (2, 125). Supplementation of seawater with a high concentration of glucose induces convergence of the biofilm and free-living microbial communities, which were originally quite different when no glucose or low concentrations of glucose were supplied (126). On the other hand, environmental microorganisms tend to form biofilms under oligotrophic or starvation conditions (2). In non- and less-eutrophic marine environments such as pristine estuaries and offshore waters, where the majority of the particles are biogenic and thus organically enriched (123), significantly different community structures of particle-associated and free-living microorganisms are commonly found (108, 114, 127–129). Particle-associated bacterial communities are frequently enriched in the marine Roseobacter clade (MRC) bacteria of the Alphaproteobacteria; the Alteromonadaceae and Vibrionaceae groups of the Gammaproteobacteria; as well as the Deltaproteobacteria, Bacteroidetes, and Planctomycetes (108, 127, 130–137). Many of these bacteria produce extracellular enzymes for biopolymer degradation, and some require suboxic or anoxic microniches within particles to support microaerophilic or anaerobic metabolism. The contrasting nutritional conditions between particles and seawater have strong impacts on microbial lifestyle differentiation, particularly in oligotrophic environments. It is reasonable to hypothesize that the significance of the difference between the surface-associated and free-living microbial communities may increase from the eutrophic and terrigenous particle-dominated riverine estuaries to the biogenic particle-dominated oligotrophic open oceans, caused by the differences in organic matter content and nutrient bioavailability between particles in these two distinct environments. This hypothesis is consistent with a recent ecophysiological study of the marine group II Euryarchaea and numerous studies on marine bacteria (138).

Due to steep gradients of key geochemical parameters (e.g., O₂, pH, sulfide, and redox potential) and to enrichment of nutrients (e.g., organic and inorganic substrates, electron donors, and electron acceptors) (6, 139–141), surfaces usually support diverse and elevated microbial metabolic and biogeochemical activities (102, 107, 111, 117). Surface-associated marine microbiota participate in a plethora of C cycling processes. Photosynthetic and chemolithoautotrophic CO₂ fixation (102, 142–148), aerobic anoxygenic phototrophic energy conservation (149–151), seawater methane production and oxidation (143, 146, 147, 152), degradation of biopolymers and other organic matter (102, 117, 145, 153), and heterotrophic respiration (102, 104, 113, 145) are all enhanced on surfaces. N cycling processes such as N₂ fixation (154, 155), nitrification (142, 143, 156–158), denitrification (158–161), dissimilatory nitrate reduction (158, 159), anaerobic ammonium oxidation (anammox) (158, 162, 163), and nitrogenous organic compound degradation (158, 164–166) are also activities associated with surfaces and can be particularly important at specific depths in the water column. For example, in the oxygen minimum zone (OMZ) of the Eastern Tropical North Pacific, particles contribute 100% of the activity reducing nitrate to nitrite and 53 to 85% of N₂ production by denitrification and anammox (167). P cycling processes such as eDNA secretion (168, 169) and particulate organic phosphorus degradation (153, 170, 171) are enhanced on marine particles, as are microbial S cycling activities such as sulfate reduction (133, 141, 147, 153, 172), sulfur oxidation (133, 146–148, 153), and organic S compound (e.g., the algal osmolyte dimethylsulfoniopropionate [DMSP]) transformation and degradation (153, 173–175). As noted above, surface- and particle-associated microorganisms also contribute substantially to marine iron cycling processes, such as iron oxidation and reduction (78, 147, 176–178), siderophore-mediated iron solubilization and uptake (153, 179–181), iron transport among different oceanic environments (182), and biocorrosion (18, 69, 72, 81). Silica regeneration from diatom detritus (183, 184), H₂ production, H₂ oxidation-related energy metabolism and dark primary production (185), as well as many other biogeochemical cycling processes are driven by surface-associated microbial activities. The biogeochemical cycling processes of almost all of the environmentally important elements are highly complex and dynamic in the ocean, involving both free-living and surface-associated microbiota as drivers (186, 187). Cycles of the various elements are intrinsically interconnected, and microbial processes foster this connectivity via the metabolic intersection of different pathways and via functional cooperation between different microbial groups (188). For example, the cooperation of archaeal anaerobic methane oxidation and bacterial sulfate reduction in natural microbial aggregates plays a critical role in coupled marine C and S cycling in anoxic methane-rich environments and thus in the control of ocean methane emissions (189–191). Particles and biofilms in marine environments provide favorable niches for the coupling of different metabolic pathways and biogeochemical cycles. Particularly in oxic marine waters, some of these microbial processes and activities happen only within the suboxic and anoxic microenvironments within particles or biofilms, which thus define surfaces and surface-associated microbial activities as unique niches and biogeochemical processes that differ from those in bulk seawater (172).

Although it is generally accepted that surface-associated microorganisms play important roles in nutrient regeneration, element cycling, biological productivity, and food web energetics, the contributions of the surface-associated microbiota to the total microbial metabolic activities in different marine ecosystems are poorly quantified, and controversies remain. For example, metabolic activities of the particle-associated microbiota vary widely. This has several causes, including particle quality and quantity (16). There are many different types of particles in the marine environment, such as suspended riverine or sediment particles, phytoplankton detritus, zooplankton and fish fecal pellets, aggregates, marine snow, macrogels, transparent exopolymer particles (TEPs), microgels, and colloidal microparticles (186). These particles differ in their origins, spatiotemporal distributions, and quality as microbial resources. The quantity of suspended riverine particles in estuaries is directly related to the seasonal pattern of riverine discharges, and these particles are usually rich in minerals and poor in organic matter. The quantity of resuspended sediment particles usually has both tidal and seasonal patterns in coastal waters, and these particles are generally poor in organic matter due to long-

term remineralization. Marine snow, fecal pellets, and phytodetrital aggregates from algal blooms constitute the major types of biogenic particles that contribute the most to the biological pump export and sequestration of carbon in the ocean (16). The abundance and spatiotemporal distribution of these biogenic particles, which are rich in organic matter, depend on epipelagic primary productivity and food webs (16). While biogenic particles constitute the major colonizable surface type in open oceans, in estuarine and coastal seas, the surface types are usually more complex, consisting of both biogenic and abiogenic particles, each with various age distributions. The distinct nutritional conditions of different surface types certainly contribute to the metabolic variability of the surface-associated microbiota, leading to quantitative differences in microbial activities. In addition, although individual surface-associated microorganisms are typically much larger than free-living microorganisms, the level of activity of surface-associated microorganisms is highly variable and sometimes, even on a per-cell basis, lower than that of free-living microorganisms (111, 192). Only a fraction of the surface-associated microorganisms may be active at a given time or under specific environmental conditions (129). For example, the attached microbial community may be more active at night, while the free-living community dominates activity during the day (114). This can be explained by higher rates of activity of particle-associated microaerophilic and/or anaerobic microorganisms that can be more active during the night when dissolved O₂ levels are lower due to the absence of photosynthesis and to night community respiration (140). This switch to more suboxic conditions may result in niche segregation and functional separation between the surface-associated and free-living microorganisms. In some environments, marine particles are sites of enhanced extracellular hydrolytic enzyme activity but low substrate incorporation activity, indicating that surface-associated bacteria may be actively dissolving particulate organic matter (POM) without significant increases in their growth rates (44, 118). The activities of surface-associated microorganisms also influence the composition, abundance, dynamics, and ecological functions of the surrounding free-living microorganisms via the release of nutritional resources and extracellular hydrolytic enzymes from colonized surfaces. This contributes to distributed networks of metabolite exchange and other forms of cooperation that involve both surface-associated and free-living microbial communities (105, 147, 186, 193, 194). It is clear that the variability of biogeochemical contributions of surface-associated microbiota is high and should be considered in an ecosystem context.

Surface-associated microbiota may also play distinct roles at different depths in the water column. Surface-colonizing microorganisms themselves participate actively in biopolymer hydrolysis, fixed carbon remineralization, and microbial secondary production near the ocean surface, where they also help retain N, P, and Fe within the upper mixed layer of marine waters (Fig. 1) (144, 186). Based on their behavior in the water column, marine particles can be crudely divided into two categories: sinking particles and nonsinking particles (195). While nonsinking particles show nearly constant concentrations throughout the dark ocean, concentrations of sinking particles such as biogenic aggregates and fecal pellets decrease exponentially with depth (46). On the one hand, organic matter degradation and decomposition of sinking particles decrease the biological pump carbon sequestration efficiency (43–46). On the other hand, microbial particle colonization may potentially increase the stability, the specific density,

and hence the settling velocity of the colonized sinking particles via the production and release of polymeric substances. This moderates, to a certain degree, the decomposition rate of the sinking particles and thus the biological pump efficiency (46). Marine snow plumes generated by surface-associated microbial activities contain high concentrations of nutrients and dissolved organic matter (DOM), also stimulating the activity and growth of freeliving microorganisms in the deep ocean (44, 196, 197). The surface-associated and free-living microorganisms above the thermocline are adapted to warm ambient temperatures, whereas in deep waters, free-living microorganisms show optimal activity at in situ temperatures, implying long residence at depth (198). The distinct temperature optima of the transient deepwater surface-associated and resident free-living microorganisms provide a reasonable explanation for findings of limited species exchange between particle-associated and free-living microbial communities throughout the water column (104, 128, 199). It seems likely that the sinking particle-associated microbial communities are composed mainly of microorganisms that originated from surface seawater, and they lose substantial amounts of activity in the cold deep water (200, 201). This is consistent with observations that the metabolic activity and growth rate of deepwater surface-associated microbial communities are usually lower than those of the surrounding free-living microbial communities, although the surface-associated microorganisms are almost exclusively responsible for the production and activity of the extracellular hydrolytic enzymes required for nutrient and labile organic matter production from POM. Marine snow is most abundant in surface waters and decomposes substantially (~90%) in the twilight zone of the water column (202, 203). The dynamics of the nutritional composition of sinking particles may influence the succession and function of the surface-colonizing microbiota thereon. The marine snow particles that reach deeper waters are generally older and more recalcitrant to microbial utilization because these particles are progressively processed as they sink, and their C/N ratios and refractory matter percentage increase with depth (202, 204, 205). Surface colonization on these recalcitrant particles may prove to be of little benefit to the newly colonizing microorganisms. This provides an alternate explanation to the limited species exchange found between particle-associated and free-living microbial communities in the ocean (104). The particle origin, degree of decomposition, nutritional status, and environmental temperature may determine the composition, succession, and activity of the sinking-particle-colonizing microbial community, which may select in favor of a surface-associated composition distinct from that of the free-living community (104). Thus, extracellular enzymes are likely produced by surface water particle-colonizing microorganisms soon after the colonization event. While the sinking of the particles into deep water will decrease the physiological activity of the particle-associated microorganisms (200), the hydrolytic activities of the secreted extracellular enzymes may be retained (to a large extent), and thus, the resident free-living microorganisms may have a better opportunity to utilize the majority of the nutrients and organic matter released by enzymatic hydrolysis of sinking POM (44, 196, 197). This “uncoupled” hydrolysis (44, 117, 206) predicts that the separation of extracellular enzyme activity from the physiological activity of the enzyme producers and the allochthonous particle-associated microbial activity from the autochthonous free-living microbial activity during the sinking of particles from the surface into the

Dang and Lovell

$\mathrm{CO}_{2}$

- Phytoplankton
  - Nutrients (N, P)
    - Degradation & remineralization
      - POC
        - Euphotic zone
          - Dark ocean
            - Sinking
              - POC
                - Biological pump
                  - Long-term carbon sequestration
- Grazing
  - Zooplankton
    - Predation
      - Fish
        - Egestion
          - Feeding
            - Protists
              - Predation
                - Microbial loop
                  - DOC
                    - Assimilation
                      - Microbial heterotrophs
                        - Viral lysis
                          - Chemolithoautotrophs
                            - Nutrients (N, P)
                              - Degradation & remineralization
                                - DOC
                                  - Assimilation
                                    - Microbial heterotrophs
                                      - Microbial carbon pump
                                        - Long-term carbon sequestration

FIG 1 Key processes of the marine carbon cycle. Microorganisms colonize the surfaces of most marine organisms, such as phytoplankton, zooplankton, protists, and fish, and marine particles, including phytoplankton detritus, zooplankton and fish fecal pellets, and marine snow. The surface-associated microbiota participate in mutualistic or antagonistic interactions with algae or zooplankton. They also play important roles (indicated by asterisks) in the degradation and remineralization of particulate organic matter and in the enhancement of primary production (via inorganic nutrient regeneration to fuel phytoplankton in the euphotic zone and chemolithoautotrophic microbial communities in the aphotic zone). The surface-associated microbiota also influence long-term carbon sequestration in the ocean via both the biological pump and the microbial carbon pump mechanisms. All the respiration terms are omitted so that the graph is not too cluttered. Abbreviations: POC, particulate organic carbon; DOC, dissolved organic carbon.
deeper waters may have significant implications for microbial functions and biogeochemical processes in the deep ocean.

The distinction between surface ocean and deep ocean surface-associated microbial activities is also related to the importance of the surface-associated microorganisms for free-living chemolithoautotrophic activity in the deep ocean. It has long been recognized that deep ocean waters maintain high levels of nitrate, presumably produced by decomposition and nitrification (207). Furthermore, it was recently suggested that chemolithoautotrophic productivity in the mesopelagic and bathypelagic zones of the ocean may represent a substantial contribution to the ocean’s total primary production (46, 146, 208, 209). In deep waters of the ocean, Archaea generally constitute a substantial fraction (10 to 40%) of the prokaryotic community (210–212). The deep ocean Archaea are mainly chemolithoautotrophic ammonia-oxidizing Thaumarchaeota (211, 213–215), which may also participate in the heterotrophic or mixotrophic uptake of organic substrates such as amino acids and carboxylic acids (211, 216–218). The autotrophic CO₂ fixation activity of the ammonia-oxidizing Thaumarchaeota has been verified in diverse marine environments (211, 219–221), and it has been estimated that this group of marine Archaea fixes ~400 Tg C year⁻¹ (208). Marine ammonia-oxidizing Thaumarchaeota are also capable of producing recalcitrant organic carbon (i.e., glycerol dialkyl glycerol tetraether membrane lipids) (222) and may facilitate long-term carbon sequestration in the deep ocean and sediments (208). The source of deep ocean ammonia is generally accepted to be decomposing sinking POM (142, 165, 214). Bacteria are the major producers of extracellular enzymatic activity in the deep ocean down to the bathypelagic layers (223), although in surface waters, marine group II Euryarchaea are also preferentially particle associated and contribute to the catabolism of HMW substrates (138). POM enzymatic hydrolysis by Bacteria transforms marine particles into nutrient islands, with NH₄⁺ concentrations usually being >2 orders of magnitude higher than those in the surrounding seawater (139, 224). Ammonia-oxidizing Thaumarchaeota likely do not produce extracellular enzymes in sufficient quantities for degradation of POM in the deep ocean (223) and may rely on extracellular enzymatic activities of Bacteria for obtaining ammonia for activity and growth (142, 165). Metabolic cooperation, in a broad sense, between surface-associated Bacteria and free-living Archaea may play a substantial role in deep ocean nitrification and dark CO₂ fixation (Fig. 1). Similarly, the release of NH₄⁺ from sinking particles into the water column by surface-associated bacterial extracellular enzymatic activities may fuel anammox-mediated nitrogen loss and chemolithoautotrophic CO₂ fixation in the OMZs of the world’s oceans (225). In order to advance our understanding of the biogeochemical contributions of surface-associated microbiota, more studies, particularly of chemolithoautotrophic activities, are needed.

### Impacts of Surface-Associated Microbiota on Ocean Carbon Sequestration

The settling of biogenic and physically coaggregated particles and especially the macroscopic aggregates designated marine snow contributes substantially to the flux of organic carbon from surface seawater to the deep oceans and sediments (226, 227). Although there seems to be little consistency across the world’s oceans regarding the mechanisms that control the spatiotemporal variability of particulate export to the deep ocean, preferential microbial remineralization of POM P and N (versus C) is commonly found (228). Surface-associated microbial activities may enhance ocean carbon sequestration by disproportionally utilizing and recycling C, N, and P. Organic matter of all descriptions is usually the major constituent of large marine particles, which also contain inorganic materials such as silt, clay, and calcite accumulated from the surrounding water (102, 111). TEPs, formed mainly by large, sticky, and acidic algal polysaccharides, usually serve as the glue that facilitates large-particle formation, leading to rapid particle sinking from surface waters to the deep ocean (229–231). Surface attachment stimulates bacteria to produce their own exopolysaccharides, also enhancing aggregation (232). Particulate organic carbon (POC) contained in marine particles forms the second largest organic carbon pool (up to 30 Pg C) in modern oceans (233), contributing to the drawdown of atmospheric CO₂ and modulation of climate variability on a global scale (43–46). Particle-associated microorganisms have major impacts on marine POC dynamics and carbon sequestration (Fig. 1).

Although some surface-colonizing bacteria may increase diatom aggregate formation, the particle sinking rate, and, thus, the potential efficiency of the marine biological pump (234, 235), it is commonly accepted that POC degradation by surface-colonizing microorganisms generally decreases the carbon sequestration efficiency of the ocean (44, 46, 113, 209, 236). The extracellular hydrolytic enzymes produced by surface-associated microbial assemblages usually have compositions and substrate ranges that differ from those of free-living microbial assemblages, likely due to differences in microbial composition (108, 237, 238). The elevated quantity and activity of surface-associated hydrolytic enzymes are likely influenced by the involvement of the QS regulatory mechanism in surface-associated microbial communities (239–241), and the higher enzyme specificity is likely due to the capability for sensing, recognition, and regulatory responses toward available substrates on the surface by specific surface-associated microorganisms (53, 242).

Some surface-associated microorganisms possess astonishing carbon cycling capabilities. For example, Alteromonas species are widespread in the ocean and are common surface- and particle-colonizing bacteria in both shallow and deep waters (243–246). Alteromonas species are metabolic generalists capable of rapid responses to an environmental disturbance (247–249). They are also large bacteria with large genomes and are copiotrophic, with high specific metabolic activities. They can degrade and utilize a broad spectrum of organic substrates, including deep-sea recalcitrant organic matter (245, 250–252). Alteromonas produces and secretes a variety of extracellular enzymes that contribute to the hydrolysis of biopolymers, including polysaccharides (253–257), proteins (258, 259), nucleic acids (260, 261), and lipids (262), the major components of marine POM. Alteromonas and related species respond rapidly to phytoplankton blooms and especially to elevated POC concentrations (109, 263–265). Algal exudates from a variety of phytoplankton can be utilized by Alteromonas, making them a major microbial group in algal phycospheres (266–268). Some Alteromonas strains also possess algicidal activity (269), reflecting their intimate association with marine algae. Alteromonas sp. strain AltSIO was recently shown to be capable of utilizing as diverse a catalog of dissolved organic carbon (DOC) substrates as the entire *in situ* microbial assemblage (270). In addition to their surprisingly high carbon cycling capability, bacteria in this group were also found to be important in marine iron cycling and trans-
port, an ecophysiological trait likely related to their surface association capabilities (182, 271). Metabolic versatility and surface association accord the Alteromonas group bacteria great carbon cycling potential and make these bacteria a member of the marine master recyclers (249, 272).

The release of labile DOC from POC degradation by particle-associated microorganisms may not only fuel the microbial loop (198, 273, 274) but also facilitate the priming effect that stimulates the degradation of recalcitrant organic matter (275, 276). Furthermore, deep-water microorganisms usually have large genomes and inventories of genes contributing to surface association (199). Higher levels of cell-specific extracellular enzymatic activity, microbial production, and cellular respiration were found to be linked to the utilization of deep ocean recalcitrant organic matter (277, 278), indicating an important role of particle-associated microorganisms in deep-sea carbon cycling (132, 279–281). On the other hand, POC degradation by surface-colonizing microorganisms may also release recalcitrant DOC (209), facilitating carbon sequestration by the microbial carbon pump mechanism (Fig. 1) (272, 282–285). DOC (up to 662 Pg C) forms the largest organic carbon pool in the modern ocean, and a substantial fraction (~97%) of DOC consists of refractory and ultrarefractory molecules that persist for thousands of years in the marine environment (286). The quantitative contributions of surface-associated microorganisms to the size and dynamics of the ocean’s POC and DOC reservoirs and the influence of spatiotemporally different environmental conditions on surface-associated microbial processes and activities are still poorly understood, especially under anthropogenic perturbation and global change scenarios (16, 284–286). In-depth studies of the marine surface-associated microbiota are fundamental for a mechanistic and predictive understanding of the marine carbon cycle.

## Environmental Change-Induced Surface-Associated Microbiota Responses and Impacts

Ocean warming and acidification induced by increasing anthropogenic CO₂ emission may lower the carbon sequestration efficiency mediated by the biological pump (287–289). Uncertainties remain because the complex, nonlinear behaviors of most ecological processes and the synergistic ecosystem responses to changing global environmental conditions (e.g., increasing temperature, ocean stratification, ocean acidification, ocean oxygen depletion, and ocean nutrient regime shift) are not well understood (290–292). For example, the remineralization depth of sinking POC becomes shallower in warmer waters, indicating that the vertical POC flux and, thus, the carbon sequestration efficiency of the biological pump will be attenuated with future increases in ocean temperature (45, 293). Ocean acidification can significantly change the ballast composition, reduce the settling velocity of sinking particles, and thus force the sinking particles to remain in the epipelagic and mesopelagic zones of the water column, with longer residence times and greater microbial decomposition (294). This may make the POC remineralization depth shallower as well. However, ocean acidification may enhance the production of TEPs (295), facilitating the formation of large sinking particles and thus increasing the POC remineralization depth. Other environmental factors can also affect the marine POC remineralization depth, such as seawater oxygen concentration, stratification, organic matter content and origin of the sinking particles, and particle-colonizing microbial community composition and activity (45). Many of these factors, and particularly their synergistic effects, are still not well studied or quantified.

As a result of the increasing impacts of anthropogenic activities and global warming, both coastal hypoxic zones and oceanic OMZs are expanding, and prevalent microbial biogeochemical pathways are correspondingly being altered (296–300). Marine particles provide suboxic and anoxic microhabitats (140, 141, 154, 281, 301), and the gradients of oxygen and other bioactive resources within marine particles and biofilms support microbial compositional and physiological heterogeneity and diversity (34, 147, 151, 153, 172). Oxygen-limited and oxygen-depleted conditions facilitate various microaerophilic and anaerobic chemolithoautotrophic carbon fixation and heterotrophic respiration and fermentation processes (147, 302, 303). Different respiration pathways have distinctly different energy conservation efficiencies (147, 304), and hypoxic and anoxic conditions usually exert a negative influence on respiratory efficiency and thus the carbon sequestration efficiency of the ocean (284, 285). Furthermore, the increasing impacts of anthropogenic activities and global warming have also caused estuarine and coastal waters (including polar ocean coastal areas) to become more eutrophic and conversely have caused open oceans, particularly the giant subtropical gyres of the Pacific and Atlantic Oceans, to become more oligotrophic (305–310). The enhanced nutrient status and elevated export from the surface waters of sinking particles in the polar ocean waters, and the ocean stratification effect on the increase of the ocean’s deepwater residence time, lead to the deep oceans becoming more stagnant and nutrient rich; conversely, the surface waters of the open oceans become more oligotrophic (311, 312). The shift of the oceanic nutrient regime, changes in oceanic circulation, as well as ocean acidification and deoxygenation may have significant negative impacts on ocean carbon cycling. This, on the other hand, may exacerbate global climate change, as the ocean is the largest active carbon sink on Earth, and its change will no doubt disturb the atmosphere-ocean CO₂ exchange balance and the carbon sequestration capacity of the ocean.

Substantial evidence establishes the importance of marine surface-associated microbial communities in global carbon cycling. Microbial populations are enriched on marine snow relative to surrounding seawater free-living microorganisms, especially in oligotrophic open oceans (106, 313), and even these oligotrophic waters can produce very high concentrations of marine snow and diatom mats (314, 315). This indicates that even the microorganisms in oligotrophic oceanic waters have an abundance of substrata, though perhaps only intermittently, to colonize. The increased formation of colloidal, gelatinous, detrital, and aggregate particles due to escalated terrigenous nutrient and organic matter inputs and to algal and jellyfish blooms in estuarine and coastal oceans (312, 316–319); the augmented tendency of microbial surface associations in response to increasingly oligotrophic conditions in the open oceans (1, 2, 6, 7, 106); the enhanced activity of surface-associated microbiota in coastal waters of the polar oceans due to seawater warming, permafrost melting, enhanced primary production, and particle transport from the tundra (320, 321); and the elevated activity of surface-associated microbiota in deep waters due to nutrient enrichment (132, 199, 278–281) lead us to hypothesize that surface-associated microbial communities may play even greater roles in ocean carbon cycling under global change scenarios. However, the mechanisms by which surface-associated microbial processes impact the ocean’s biogeochemical

processes and carbon sequestration capacity and the magnitude of
this impact, especially under changing environmental conditions,
are not well understood. A fundamental understanding of the
mechanisms and impacts of microbial surface colonization, bio-
film formation, and surface-associated activities in marine envi-
ronments will be required to address profoundly important ques-
tions related to global climate change.

The processes of microbial surface colonization and biofilm
community development are highly dynamic and complex. These
processes usually start with a surface-sensing step, and diverse
environmental cues are likely involved in the induction of the
initial surface attachment event. Cell surface components play im-
portant roles in subsequent irreversible surface adhesion and,
thus, true colonization. Microbial intra- and interspecies interac-
tions, including cooperation and competition, shape the routes of
community succession, biofilm development, and functional
maturation, resulting ultimately in complex microbial communi-
ties. In the following sections, we examine (i) how the microor-
ganisms sense and respond to environmental cues to initiate sur-
face colonization and biofilm development, (ii) how the
microorganisms interact with the substratum surface to carry out
the actual colonization steps, (iii) how the microorganisms coop-
erate and compete to drive the development of the surface-colo-
nizing and biofilm communities, (iv) how two key marine bacte-
rial groups succeed as key surface colonizers, and (v) future
directions in the study of surface colonization and biofilm devel-
opment.

# MICROBIAL SENSING AND SIGNALING IN SURFACE
COLONIZATION AND BIOFILM DEVELOPMENT

Environmental factors play important roles in determining mi-
crobial surface colonization events (47, 60, 322–324). In general,
the interaction of microbial cells with the substratum surface un-
der specific physicochemical and nutritional conditions at the sea-
water-surface interface likely contributes substantially to the ini-
tiation and success of microbial surface colonization in marine
environments. Substratum physicochemical properties such as
surface free energy, electrostatic charge, hydrophobicity, wettabil-
ity, roughness, microtopography, and vulnerability to wear (such
as corrosibility of a metal surface) and surface chemodynamic
properties such as surface conditioning, nutrient enrichment, and
charge accumulation or alternation may influence the ability of
microorganisms to adhere to a particular abiotic surface (47, 325).
For example, environmental pH and ionic strength may alter the
surface charge of both the microorganism and the substratum
surface when they are exposed to the aquatic environment, influ-
encing microbial surface adhesion in various ways (325–327).
Nutrient limitation caused by N, P, or Fe scarcity in marine envi-
ronments may induce certain microorganisms to adapt to a sur-
face-associated lifestyle or to disperse from a biofilm to find more
favorable surfaces (2, 60, 328–332). Biofilm formation in the ma-
rine environment may play an important role in microbial selec-
tion of the optimal habitat (333, 334). In marine vibrios, certain
components of the pathways of catabolite repression (modulating
cellular responses to high-energy-carbohydrate availability), the
stringent response (modulating the use of available resources in
response to low-nutrient stress such as amino acid, fatty acid, or
iron starvation), and nucleoside scavenging (modulating nucleo-
side uptake and catabolism in response to environmental nucleo-
side scarcity) exert regulatory effects on surface colonization

and/or biofilm formation (335–340). The adaptations required
for successful surface colonization certainly include the ability to
detect and respond to surface-related cues.

Microorganisms utilize a variety of sensing mechanisms to
adapt to and exploit changing (micro)environments (341). The
environmental cues may be physical (the surface as a diffusion
barrier or potential energy barrier), chemical (redox potential,
conditioning film composition, adsorbed nutrients, metaboliz-
able substrates, and electron donors and acceptors), or physico-
chemical (microviscosity and water activity) (47). Known envi-
ronmental cues that attract individual microorganisms to surfaces
are diverse, particularly including high inorganic and organic nu-
trient levels, the availability of electron donors and acceptors, and
hydrodynamic conditions (342). Sensing may be the necessary
first step for marine microorganisms to establish a surface-associ-
ated lifestyle, and thus, microbial surface sensoritomes play a crit-
ical role in the primary interactions between the microbial cell and
the surface that is colonized (343, 344).

## Microbial Two-Component Signal Transduction Systems

Two-component signal transduction systems (TCSs) are very
common in both Bacteria and Archaea (284, 345, 346). These sys-
tems enable microorganisms to constantly sense and respond to
environmental changes and stresses, such as those caused by the
availability of inorganic nutrients and metabolizable organic sub-
strates, temperature, pH, O₂, redox potential, light intensity, os-
molarity, and toxins, including reactive oxygen and nitrogen spe-
cies as well as other substances (347). Upon activation by an
environmental stimulus via a specific TCS sensor histidine kinase
(HK) component, the cognate TCS response regulator (RR) com-
ponent may induce the binding of a regulatory molecule to DNA,
RNA, or protein or cause an increase in enzymatic activity (348).
These responses lead to changes in cellular transcriptional, enzy-
matic, or mechanistic properties and alterations in microbial
physiology and/or behavior (346, 349). For example, the genome
of *Vibrio cholerae* O1, frequently isolated from estuarine and
coastal environments and a causative agent of Asiatic cholera, har-
bors 43 HK and 52 RR genes (350). Twelve of these RRs were
found to have a role in host colonization (350). In addition, V.
cholerae O1 employs the VpsS hybrid HK and the VpsR and VpsT
RRs to regulate the production of the exopolysaccharide VPS
(*Vibrio* polysaccharide) that enables the formation of biofilms and
consequent resistance to oxidative stress and chlorine biocidal ac-
tivity (9, 351–354). In *Vibrio fischeri*, the RscS and SyfF HKs were
found to play an important role in inducing symbiotic biofilm
formation and squid colonization via the SypE and SyfG RRs that
modulate the transcription of the symbiosis polysaccharide (*syp*)
locus (355). A *V.* *cholerae* VpsR homologue also modulates V.
*fischeri* polysaccharide production and biofilm formation via a
putative cellulose biosynthesis locus found only in this vibrio spe-
cies (9). TCSs are also employed by other vibrio species to regulate
extracellular polysaccharide production and biofilm formation,
although the exact mechanisms may be slightly or even substan-
tially different in different organisms (9). Systematic studies of the
TCSs in *Pseudomonas aeruginosa*, an opportunistic pathogen and
an environmental bacterium found frequently in coastal waters
and sometimes even in open oceans (356–358), have confirmed
that biofilm formation is a highly regulated process that proceeds
through a number of distinct stages (347). An array of TCSs play a
key role in the regulation of the production of extracellular ap-
pendages, such as flagella, type IV pili, and Cup fimbriae, that are often involved in *P. aeruginosa* initial surface attachment as well as the production of extracellular exopolysaccharides, such as Pel and Psl, that are required for subsequent *P. aeruginosa* biofilm formation (347).

The *Escherichia coli* osmosensing EnvZ/OmpR system and the cell envelope disturbance-sensing CpxA/CpxR system were among the first TCSs identified as important sensing, signaling, and regulatory mechanisms for bacterial surface colonization and biofilm formation (359). The nutritional and ionic enrichment that is generally observed on submerged surfaces creates a microhabitat that has higher osmolarity than that of the surrounding aquatic environment (6, 144, 347). The EnvZ/OmpR and CpxA/CpxR systems provide a mechanism to promote the microbial response to such an osmolarity gradient (i.e., a nutrient gradient) and to promote microbial surface attachment (360). Homologues of the EnvZ/OmpR and CpxA/CpxR TCSs have been identified in marine bacteria, including *V. cholerae* (361, 362). However, these TCSs have been found to play a minimal role in the regulation of *V. cholerae* surface colonization and biofilm formation (15, 362). Instead, the OscR and CosR osmolarity-responsive regulators were found to play a major role in regulating osmolarity- and salinity-induced *V. cholerae* biofilm physiology (362–364). Therefore, marine and nonmarine bacteria may employ different mechanisms to regulate osmolarity-responsive surface colonization and biofilm formation due to environmental and evolutionary differences.

Many TCSs in environmental microorganisms are related to sensing and adaptive responses to inorganic nutrients and metabolizable organic substrates, such as organic acids, sugars, and amino acids (365–367). For example, the extracellular sensors of the TCS PhoQ, DcuS, CitA, and AbfS, involved in monitoring environmental divalent ions such as Ca²⁺ and Mg²⁺, C₄-dicarboxylic acids, citrate, and oligosaccharides, respectively (366), may help the microorganisms to detect a favorable surface based on nutrient enrichment. The phosphate-responsive PhoR/PhoB system involved in high-affinity phosphate-specific transport regulation under P starvation conditions is a common TCS in bacteria (368). This TCS is also involved in surface colonization, biofilm formation, or microbial dispersion from biofilms in some marine bacteria (330, 331, 369). In marine *Pseudoalteromonas piscicida* isolates, the CdsS/CdsR TCS regulates the expression of genes involved in chitin degradation (370), which is facilitated by bacterial surface colonization. In *V. cholerae*, the expression of chitin-inducible genes, including those involved in chitin degradation and utilization, chemotaxis, surface colonization, and natural competence, is modulated by the orphan TCS sensor kinase ChiS (371, 372). Chitin is the most abundant nitrogenous polysaccharide in the ocean and can be degraded and utilized by many marine bacteria (373). N and P are inarguably the most important macronutrients in the ocean, controlling the ocean’s primary and secondary productivity, carbon sequestration, and many other biogeochemical functions and ecosystem services (297, 374–379). How surface-associated microorganisms, using putatively diverse regulatory pathways, respond to and influence the ocean’s changing N and P regimes warrants in-depth investigation.

TCSs exist in >95% of Bacteria and ~50% of Archaea. Microbial genome sequencing has identified a great number of TCSs, yet the functions and environmental stimuli of most of these systems have not been determined. The MiST2.2 Microbial Signal Transduction Database (last accessed 10 October 2014) predicted 421,394 gene sequences that encode TCS proteins (not including chemotactic proteins) from a total of 7,937 (complete and draft) bacterial and archaeal genomes (380). The P2CS (Prokaryotic 2-Component Systems) database (last accessed 10 October 2014) predicted 164,651 gene sequences that encode TCS proteins, including 74,029 HKs and 81,882 RRs (381). On average, a single microbial species usually possesses >50 TCSs (382), and some bacteria possess hundreds of TCSs operating in parallel for the adaptive response to diverse environmental conditions (365). For example, *Vibrio parahaemolyticus* O1:Kuk strain FDA_R31 alone may harbor 497 TCS proteins, based on predictions of the MiST2.2 database. Microorganisms that live in rapidly changing environments typically possess a large number of TCSs, and the number of TCSs that a microorganism can possess appears to correlate with its habitat’s environmental complexity and niche diversity (346). It is reasonable to hypothesize that the marine microorganisms that are capable of major changes in lifestyle (such as the transition from the motile to the sessile lifestyle and vice versa) may harbor large numbers of TCSs functional in both free-living and surface-associated activities and in the transitions between these two lifestyles.

### Microbial Chemotaxis

Chemotaxis systems coordinate the sensing, signaling, and responsive motility of a bacterium or archaeon in response to chemical attractants or repellents (383) and are among the most thoroughly studied TCSs (384–386). Many marine microorganisms (up to 80%) are motile, especially in highly productive circumstances, such as the organic particle- and nutrient-enriched conditions that occur during algal bloom crashes (387–389). Motility is a physiological and behavioral trait usually linked to the response to environmental gradients (341). It is reasonable to hypothesize that a large fraction of aquatic microorganisms are chemotactic, although they tend to attach if a suitable surface exists (390). The MiST2.2 database (last accessed 10 October 2014) predicted 90,807 chemotaxis protein-encoding gene sequences from a total of 7,937 (complete and draft) bacterial and archaeal genomes (380).

The microbial chemotactic apparatus is highly sensitive, sensing and responding to as little as a 3 nM change in the concentration of an environmental chemical stimulus (391). Chemotaxis is used by environmental bacteria not only for increased acquisition of organic substrates but also for enhanced uptake of inorganic nutrients (392). *Thalassospira* sp. was found to be chemotactic toward inorganic phosphate during starvation, a behavior consistent with its natural habitat of the ultraoligotrophic eastern Mediterranean Sea (393). A recent study showed that coral surface-associated bacteria exhibited significantly higher levels of chemotaxis than free-living bacteria in nearby non-coral-associated waters (394). Numerous processes, such as cell lysis, phytoplankton exudation, animal excretion, food vacuole egestion, and particle degradation and dissolution, provide point sources rich in organic substrates and inorganic nutrients in marine waters (186, 392, 395). Chemotaxis toward marine particles or their nutrient plumes may facilitate carbon and nutrient cycling and the microbial loop (393).

Chemotactic responses driven by environment sensing and directed motility have frequently been proposed to facilitate microbial surface attachment (3, 6, 8, 47, 338, 392, 396–398). Results to
date show that chemotaxis can be important (399–405), advantageous (406), or dispensable (407) for initial surface colonization, indicating possible species- or strain-specific differences in the role of chemotaxis in microbial surface interactions or surface- or environment-specific differences in microbial chemotactic responses. Energy taxis can also be important to surface colonization by certain bacteria (408–411). The microenvironment near a submerged surface is highly heterogeneous in that multiple gradients exist, including gradients of oxygen, pH, osmolarity, electron donors, electron acceptors, metabolizable substrates, redox potential, and chemical cues, including chemotactic attractants or repellents (34). Most studies showing a positive effect of taxis on bacterial surface colonization employed biotic surfaces, with only a few cases in which abiotic surfaces were also tested (402, 404). Thus, we hypothesize that there are threshold concentrations of taxis signals, which dictate the initiation of the microbial attachment response driven by the chemotactic or energy taxis mechanisms. A submerged abiotic, inert surface likely does not satisfy this requirement, as the taxis signals in the environment near such a surface may not achieve high enough concentrations to induce the tactic response, even considering the accumulation of the conditioning film on the surface.

### Microbial Quorum Sensing

Many *Bacteria* and *Archaea* employ QS as a specialized intraspecies and interspecies communication mechanism for population density-dependent sensing, signaling, and responsive adaptation (412–417). A typical QS pathway is characterized by the production, release, and detection of small signal molecules collectively called autoinducers, resulting in coordinated behavior once a sufficient signal concentration, reflecting a sufficient quorum size, is reached. QS is a common strategy to achieve a group benefit and coordinated behavior in the prokaryotic world and is particularly important for surface- and biofilm-living microorganisms that often reach high densities (417).

QS plays important roles in regulating initial microorganism-surface interactions, microbial surface attachment, initiation of biofilm formation, and biofilm development (33). *V. fischeri* harbors three distinct QS systems (i.e., AinS/AinR, LuxI/LuxR, and LuxS/LuxPQ); however, only the AinS/AinR system is involved in the modulation of the initial steps of surface colonization, and only the AinS/AinR and LuxS/LuxPQ systems are involved in the modulation of subsequent biofilm development (418, 419). *V. cholerae* harbors four distinct QS systems (i.e., CqsA/CqsS, LuxS/LuxPQ, CqsR, and VpsS), and they all participate in the modulation of surface colonization and biofilm formation (420). Functional redundancy of the four QS receptors is employed by *V. cholerae* to prevent premature induction of a QS response that may be caused by signal perturbations (420). The QS systems in *V. fischeri* assist in establishing a symbiotic relationship between the bacterium and its host, *Euprymna scolopes*, while the QS systems in *V. cholerae* contribute to making this bacterium a deadly pathogen to humans.

QS autoinducers were found to enhance cell adhesion to sulfur and pyrite surfaces by *Acidithiobacillus ferrooxidans* (421), a chemolithoautotrophic bacterium that carries out CO₂ fixation coupled to ferrous iron and sulfur oxidation (422, 423). Extracellular polymeric substances (EPSs) identified as lipopolysaccharides appear to be a prerequisite for *A. ferrooxidans* attachment to pyrite and sulfur (424), and their biosynthesis is likely controlled by the cellular QS mechanism (425). Interestingly, *A. ferrooxidans* harbors two QS systems (426). A Lux-like system is upregulated when *A. ferrooxidans* is grown in sulfur medium (427, 428), while an Act-based system is upregulated when *A. ferrooxidans* is grown in medium containing iron instead of sulfur (426). Thus, it has been suggested that the two QS systems respond to different environmental signals that may be related to the abilities of *A. ferrooxidans* to colonize and use different solid sulfur- and iron-containing minerals (426).

FeOB and SOB are important participants in the biogeochemical cycling of iron and sulfur, bioleaching of metal ores, and bio-corrosion of metals (18, 78, 99, 178, 429–431). Most of the FeOB and SOB are microaerophiles that prefer low-oxygen conditions for chemolithoautotrophic CO₂ fixation, and some of them are facultative anaerobes that can carry out iron and sulfur oxidation by using alternative terminal electron acceptors such as nitrate and nitrite instead of oxygen (99, 133, 178, 429, 430, 432, 433). QS-related genes have recently been identified in sulfur-oxidizing Gammaproteobacteria and *Epsilonproteobacteria* and in *in situ* biofilms of deep-sea hydrothermal vents (434, 435). QS may help FeOB and SOB colonize surfaces to obtain inorganic Fe and S substrates and establish optimal niches within biofilms (436, 437). In seawater, particles may be a rich source of reduced iron (147, 177, 181, 186, 301, 438, 439) and reduced sulfur (133, 140, 141, 147). QS-mediated microbial surface colonization and biofilm formation may play an important role in the biogeochemical cycling of Fe and S in marine environments.

QS autoinducers and/or their synthetic genes have been found in marine microbial mats, subtidal biofilms, deep-sea hydrothermal vent biofilms, and marine organic particles (239–241, 435, 440, 441), indicating that QS may be common in marine surface-associated microbial communities. QS autoinducers modulate the production and activity of extracellular hydrolytic enzymes (e.g., lipases, aminopeptidases, and phosphatases) in marine snow- or *Trichodesmium* colony-associated microbial communities (240, 241, 442). The liberation of dissolved nutrients and organic substrates may benefit surface-associated microorganisms, as well as *Trichodesmium* bacteria themselves for CO₂ and N₂ fixation, especially in oligotrophic environments. Diverse and novel autoinducer synthase genes have been identified in the Global Ocean Sampling metagenomic database, which covers 68 stations across three oceans (443). Many environmentally important microorganisms, including ammonia-oxidizing, nitrite-oxidizing, anammox, denitrifying, nitrogen-fixing, and sulfur-oxidizing bacteria as well as methanogenic *Archaea*, employ QS systems (416, 435, 444–448). The *Proteobacteria* are the predominant QS autoinducer producers in natural environments (449). More than 80% of MRC bacteria harbor QS regulatory systems (8, 173, 272, 450, 451). It has been hypothesized that QS contributes to the surface colonization success of MRC bacteria (8), likely because submerged surfaces in seawater may serve as a source of organic nutrients that attract MRC bacteria and support growth to levels sufficient to support density-dependent QS regulation (450). In addition to its role as an important regulatory mechanism for initial microbial surface colonization, QS may also participate in mediating the interactions of surface colonizers. Autoinducer-2 molecules, an important family of QS signal compounds, are synthesized by many bacteria and appear to facilitate interspecies communications (452). These QS signals may be employed by certain pioneer surface-colonizing bacteria to alter the composition-

Dang and Lovell

tion and structure of the primary colonizer community and influence the subsequent succession of other microbial species on surfaces (452). A recent study has shown that a MRC bacterium promotes initial colonization and biofilm formation by other marine bacterial species via extracellular factor secretion (453). However, whether this extracellular factor is a QS autoinducer and whether the MRC QS systems are involved in the sequential succession of surface- and biofilm-associated microbial communities in marine environments remain to be determined.

**Posttranscriptional Regulation by Small RNAs**

Small RNAs (sRNAs), a group of noncoding regulatory RNAs, usually with lengths of 25 to 500 nucleotides, are an important type of regulator that binds to mRNA or proteins to modulate translation in diverse microbial physiological processes (454). sRNAs interact with TCSs, the primary mechanism for effective sensing of environmental cues in microorganisms, to form extensive regulatory networks (455). Many TCSs, via the regulation of sRNAs, control target gene expression with enhanced signaling flexibility, dynamics, and timing; conversely, via sRNA regulation, certain TCS regulons can be recruited into other regulatory networks, such as the QS systems, forming a sRNA-mediated feedback loop to achieve fine-tuning of gene regulation and homeostatic control of the involved regulators (455). The intrinsic interconnection of the sRNA, TCS, and QS regulatory systems implicates the sRNAs in microbial surface interaction and biofilm formation.

Experiments have shown that members of the CsrB family of sRNAs, coordinating with the global regulator CsrA (or its homologue proteins) and certain related TCS and QS regulators, play central roles in modulating the switch between motile and sessile bacterial lifestyles, although the precise roles may vary in different bacterial species (454). In *V. cholerae*, the VarS/VarA TCS; CsrA; and the CsrB, CsrC, and CsrD sRNAs regulate the activity of the QS response regulator LuxO (456), which can also be activated (phosphorylated) by the QS autoinducer-free sensor kinase proteins CqsS and LuxQ via the phosphotransfer protein LuxU (457). At a low cell density, phosphorylated LuxO activates the expression of four *qrr* (quorum-regulatory sRNA) genes that encode the sRNAs Qrr1 to Qrr4, which redundantly promote the translation of AphA (the low-cell-density QS master regulator), inhibit the translation of HapR (the high-cell-density QS master regulator), and activate the translation of Vca0939, stimulating biofilm formation (49, 458–460). In *Vibrio harveyi*, phosphorylated LuxO activates the expression of five *qrr* genes encoding the sRNAs Qrr1 to Qrr5, which additively promote the translation of AphA and inhibit the translation of LuxR (homologue of HapR), likely resulting in reduced biofilm formation, however (49, 460). Although the Qrr sRNAs seemingly participate in similar regulatory pathways in *V. cholerae* and *V. harveyi* (460), the collective actions (redundant versus additive) of the involved Qrr sRNAs and their ultimate effects (stimulation versus inhibition) on biofilm formation are different (49, 457, 461–463). In addition, some *Vibrionaceae*, such as *V. fischeri* and *Photobacterium angustum*, have only Qrr1, likely adding more variation to sRNA regulatory outcomes (460).

Recently, the sRNA VqmR has been identified as another regulator of biofilm formation in *V. cholerae* (464). The transcription of the *vqmR* gene is activated by the VqmA DNA-binding transcription factor, and VqmR directly modulates at least eight mRNA targets, including the *vpsT* transcriptional regulator of biofilm production (464). The *vpsT* regulator is also targeted by the histone-like nucleoid structuring protein H-NS, providing another layer of regulation of biofilm production by *V. cholerae* (15, 465). Besides the exopolysaccharide VPS, the expression of three biofilm matrix proteins, RbmA, RbmC, and Bap1, is also required for biofilm formation and structure in *V. cholerae* (15, 402, 466). These proteins facilitate biofilm formation at particular steps: RbmA is capable of binding the exopolysaccharide VPS and strengthening early cell-cell adhesion, Bap1 facilitates biofilm adhesion and recruits planktonic cells to the surface, and Bap1 and RbmC encase cell clusters that are attached to the surface (467–469). The type II secretion system (T2SS) delivers these biofilm matrix proteins for biofilm formation in *V. cholerae* (470). The expression of these biofilm matrix proteins involves regulation by the cyclic AMP (cAMP)-cAMP receptor protein (CRP) complex and the transcriptional regulator VpsR (471, 472). However, the expression of RbmC can bypass the global master regulators, virtually through direct regulation by the sRNA VrrA, the expression of which is in turn modulated by the alternative RNA polymerase sigma factor σE (473). Biofilm formation, as well as motility and chemotaxis in *V. cholerae*, also involves the sRNA RyhB that is negatively regulated by iron and the ferric uptake regulator Fur (474). Multiple sRNAs, as well as other regulatory pathways, provide *V. cholerae* with both specific and adaptable control over biofilm formation, a mechanism that is important for a versatile and error-proof response to the diverse environmental cues that can induce surface-associated living. sRNAs in other marine bacteria have received little attention, and many new mechanisms and pathways may await discovery (344, 475).

**Centralized Regulation by Second Messengers**

Second messenger molecules are employed in many microbial environmental signaling pathways to relay external signals from membrane receptors to intracellular effectors (476). cAMP, the first second messenger described, participates in the cAMP-CRP regulatory network, which exerts global control over key cellular physiology processes, including the production of flagella, microbial motility, cell surface hydrophobicity, quorum sensing, type IV pilus expression, surface attachment, and biofilm formation (27, 338, 476–479). Cyclic di-GMP (c-di-GMP) is another key and ubiquitous second messenger molecule in prokaryotes, playing central roles in microbial signaling and adaptability (480). Despite the tremendous diversity of microbial components and processes and mechanisms that are involved in the switch from the planktonic to the sessile lifestyle, most bacteria examined to date employ c-di-GMP as the central regulator to control surface colonization and biofilm formation (480–482). For example, *Ruegeria mobilis*, a member of the MRC, employs the c-di-GMP regulatory pathway to modulate biofilm formation and antibiotic production (483). In *Agrobacterium tumefaciens*, c-di-GMP activates, in a surface contact-dependent manner, enhanced production of the unipolar polysaccharide adhesin, which is functionally equivalent to a holdfast, for integrated control of the switch from a motile to a sessile lifestyle (484, 485). The environmental bacterium *Shewanella oneidensis* MR-1, which harbors a number of respiratory pathways, including the anaerobic reduction of iron(III), manganese(IV), and uranium(VI), forms biofilms on mineral surfaces through a process controlled by c-di-GMP (486). The marine bacterium *Shewanella woodyi* also employs the c-di-GMP signaling
pathway for controlling biofilm physiology (487). The c-di-GMP signaling pathway may play a key role in A. ferrooxidans biofilm formation and bioleaching of minerals (488), whereas diverse c-di-GMP signaling pathways control the switch between predatory and nonpredatory lifestyles of Bdellovibrio bacteriovorus, which preys upon other Gram-negative bacteria on surfaces and within biofilms (489). The intracellular c-di-GMP concentration is regulated by GGDEF domain-containing diguanylate cyclases (DGCs) that catalyze c-di-GMP synthesis from two molecules of GTP and by c-di-GMP-specific EAL or HD-GYP domain-containing phosphodiesterases (PDEs) that catalyze c-di-GMP hydrolysis (480). Bacteria usually contain multiple DGCs and PDEs. For example, the V. cholerae genome contains >60 genes predicted to encode distinct c-di-GMP-modulating DGCs and PDEs for a flexible environmental response and high-fidelity signaling (490). The c-di-GMP regulatory pathways are involved in modulating the expression of type IV pili, the exopolysaccharide VPS, and T2SS-facilitated secretion of biofilm matrix proteins, playing important roles in surface colonization and biofilm formation by V. cholerae (470, 491). Surface-associated bacteria usually harbor more c-di-GMP regulators than free-living bacteria, presumably as an adaptive strategy (120). O₂, H₂O₂, NO, redox potential, light, sucrose, amino acids, polyamines (such as norspermidine and spermidine), Zn²⁺, bile acids, bicarbonate, indole, QS autoinducers, cis-2-dodecenoic acid and cis-11-methyl-dodecenoic acid (unsaturated fatty acids that serve as bacterial diffusible signal factors), and nutritional conditions that cause starvation (or depletion of a specific carbon source such as glucose or glycerol) have been identified as environmental cues that induce the bacterial response via altering the intracellular c-di-GMP concentration (480, 492–508). However, the vast majority of the environmental signals that modulate the activity of the DGCs and PDEs remain unidentified.

Multiple sensory transduction pathways, including mainly QS (as well as TCSs and chemotaxis), that sense a vast array of extracellular signals have been found to interact with the c-di-GMP intracellular regulatory networks to influence microbial biofilm formation (330, 480, 509–511). The second messenger- and sRNA-mediated signaling pathways are also interconnected. A recent study indicates that the V. cholerae Vca0939 protein is a diguanylate cyclase, and its translation is activated at low cell density by the Qrr sRNAs, leading to c-di-GMP accumulation and thus enhanced VPS-dependent biofilm formation (512, 513). The cAMP signaling pathway also interacts with the c-di-GMP pathway, playing a role in regulating V. cholerae biofilm formation (472). The c-di-GMP signaling mechanism represents a unifying principle governing the microbial switch from a planktonic to a sessile lifestyle (514, 515).

### An Example of Microbial Interaction with Surfaces: Vibrio Chitin Utilization and Its Implications

Chitin is the most abundant biopolymer in aquatic environments and a major component of marine snow (166, 193, 516). About 10¹¹ metric tons are produced annually as marine detritus (120, 164). Chitin is highly insoluble, but its degradation products represent an abundant source of carbon, nitrogen, and metabolic energy for microbial communities. Chitin utilization constitutes a key pathway in global carbon and nitrogen cycling (186, 193), and this polymer also provides a surface for vibrios to colonize, particularly under adverse environmental conditions (516–520).

The utilization of chitin by vibrios involves multiple levels of gene regulation that govern motility, chemotaxis, extracellular polysaccharide and biofilm matrix protein synthesis and secretion, type IV pilus production, chitin-binding protein secretion, chitin surface attachment, biofilm formation, extracellular chitinase secretion, chitoporin expression, and competence (166, 338, 470, 520–523). Association with insoluble materials may be the preferred lifestyle of vibrios, including deep-sea hydrothermal vent species (15, 403, 524), and regulatory systems involving TCSs, chemotaxis, QS, sRNAs, cAMP, c-di-GMP, alternative sigma factors, and the stringent response enable vibrios to optimize resource utilization and survival (Fig. 2) (49, 339, 472, 475, 511, 518, 525–529).

Vibrio chitin utilization and biofilm formation are also regulated by the phosphoenolpyruvate:sugar phosphotransferase system (PTS) (337, 530), which catalyzes the transport and phosphorylation of numerous monosaccharides, disaccharides, amino sugars, polyols, and other sugar derivatives and possesses diverse regulatory functions related to processes such as chemotaxis, detection of QS molecules, virulence, potassium transport, and metabolism of carbon, nitrogen, and phosphorus (531, 532). Vibrios are strongly chemotactic toward several PTS substrates, including N-acetylglucosamine, trehalose, glucose, sucrose, mannose, and mannitol (164, 166, 533). Mannitol is of particular interest, as it enhances V. cholerae biofilm formation by activating, via the PTS, the transcription of the VPS exopolysaccharide synthesis genes (333). Mannitol is a common compatible solute and osmoprotectant and a primary photosynthetic product and carbon reserve compound of brown algae (534). Mannitol and its induction of biofilm formation provide V. cholerae additional fitness advantages in the highly variable marine environment. This also implies that biofilm formation induced by chitin or its degradation products may proceed through the PTS regulation pathway in other chitin-utilizing marine bacteria.

In addition to chitinous detritus and live zooplankton and their carcasses and molts, vibrios are frequently enriched on other marine particles or surfaces, such as marine snow, fecal pellets of zooplankton, and detritus from the demise of phytoplankton and jellyfish blooms (70, 172, 518, 535–538). It has also been estimated that there are 5.25 trillion plastic particles weighing 268,940 tons afloat at sea (539, 540). Some of the “plastisphere” components, such as polypropylene, are preferentially colonized by vibrios (541).

Although vibrios are usually detectable and sometimes abundant in coastal and brackish waters, especially in eutrophic environments, they are usually subdominant and opportunistic bacteria (247, 535, 542). Marine vibrios in natural biofilms may contribute to inducing the settlement of invertebrate larvae (543). In addition, vibrios are common producers of auxins such as indole-3-acetic acid (544), likely playing a role in stimulating the activity (such as mucus exudation) of marine algae and the formation of vibrio-alga associations (545). Surface association not only increases vibrio survival and fitness but also increases opportunities for vibrio intraspecies and interspecies exchanges of genes, including those for the utilization of unusual substrates and for virulence (15, 546–549). Such exchanges are supported by surface-induced natural competence and likely involve plasmids, phages, transposons, integrons/gene cassettes, and perhaps other horizontal gene transfer mechanisms (550–552). QS-mediated biofilm formation can include large numbers of toxigenic V. chol-
Dang and Lovell

Unknown cue  
AI-2  
CAI-1  
Glucose  
Chitin  
GlcNAc di-/oligomers  
Spermidine  
Nitric oxide  

VarS  
LuxP  
CqsS  
PTS  
CBP  
NspS  

LuxQ  
ChiS  

VarA  
LuxS  
CqsA  
CyaA  
TfoR + Hfq  

LuxU  

CsrB, C, D sRNAs  
LuxO  
Fis  
CdgA  
TfoX  

CsrA  
Qrr1-4 + Hfq  
VCA0939  
c-di-GMP  

HapR  

AphA  

Positive effect  
Negative effect  

VpsR  
VpsT  

LCD genes  
Virulence  

Type IV pili  
Competence + Chitin utilization  

msh  
T2SS & rbm  

vps  

HCD genes  
VPS  
Matrix proteins  
Biofilm formation  

FIG 2 Interacting sensing, signaling, and regulatory pathways important for the Vibrio cholerae sessile lifestyle. Diverse environmental cues such as chitin disaccharide and oligosaccharides, bile acids (not shown), nitric oxide, norspermidine (not shown), spermidine, carbon source depletion, and population size signals (such as the autoinducers cholerae autoinducer 1 [CAI-1] and autoinducer 2 [AI-2]) are sensed and processed by V. cholerae, which employs signal transduction sensor kinases (such as ChiS, VarS, LuxQ, CqsS, CqsR, VpsS, and HnoK) and response regulators (such as LuxO, VarA, HnoB, HnoD, TfoX, VpsR, and VpsT); the quorum-sensing master transcriptional regulators AphA and HapR; small RNAs (such as CsrB, CsrC, CsrD, Qrr1 to -4, and TfoR); and the RNA chaperone Hfq, cAMP, and c-di-GMP for signal relay and response regulation. The type IV pili are involved in initial surface attachment. The activated production of VPS (Vibrio polysaccharide) (the major component of the V. cholerae biofilm matrix) and biofilm matrix proteins contributes to biofilm formation. It is evident that most regulatory pathways converge on c-di-GMP, which plays a central role governing the microbial switch from the planktonic to the sessile lifestyle. There are some other surface- and biofilm-related sensing, signaling, and regulatory pathways, such as the CqsR and VpsS QS pathways, the chemotactic pathway that senses extracellular chitin disaccharide and oligosaccharides and modulates bacterial tactic movement toward chitin surfaces for efficient colonization and chitin utilization, the stringent response regulatory pathway that maximizes the use of available resources in response to various low-nutrient stresses, the nucleoside scavenging-and-signaling pathway for regulating natural competence, and the pathways mediated by H-NS and alternative sigma factors, which are not shown in order to avoid cluttering. LCD, low cell population density; HCD, high cell population density; LuxS, autoinducer-2 synthase; LuxP, autoinducer-2 periplasmic binding protein; LuxQ, autoinducer-2 membrane-bound sensor histidine kinase; CqsS, CAI-1 membrane-bound sensor histidine kinase; LuxU, autoinducer phosphorelay protein; LuxO, LuxU cognate response regulator; CBP, chitin-binding protein; NspS, periplasmic spermidine-binding protein; HnoX, NO sensor protein; OM, outer membrane; IM, inner membrane; CsrA, global posttranscriptional regulatory protein that activates LuxO via an unidentified regulatory factor (denoted “?”); Qrr, quorum regulatory small RNA; Hfq, RNA-binding and chaperone protein; cAMP-CRP, cAMP-cAMP receptor protein complex; Fis, factor for inversion stimulation, a small nucleoid protein; PTS, phosphoenolpyruvate phosphotransferase system; CyaA, adenylate cyclase that synthesizes cellular c-di-GMP; HnoB and the EAL domain of MbaA, phosphodiesterases that degrade c-di-GMP; HnoD, protein containing a degenerate phosphodiesterase functioning as an HnoB allosteric inhibitor; VpsR and VpsT, transcriptional regulators that modulate VPS synthesis, with VpsR also being a regulator of V. cholerae biofilm matrix protein synthesis; T2SS, type II secretion system; com, msh, rbm, and vps, gene operons for chitin-induced natural competence, type IV pilus production, biofilm matrix protein production, and VPS production, respectively. This figure is drawn based on information reported previously (15, 49, 51, 338, 414, 420, 470–472, 475, 491, 500, 511, 512, 527).

erae bacteria, usually in the viable-but-nonculturable (VBNC) state (33, 553, 554). Such biofilm-associated V. cholerae bacteria are more virulent than their free-living counterparts (512, 553, 555, 556), and VBNC V. cholerae can be resuscitated by QS auto-inducers (557), which also promote horizontal gene transfer to V. cholerae in multispecies biofilms (558). QS-regulated chitin me- tabolism also enhances the resistance of V. cholerae biofilms against heterotrophic protist grazing (559). These properties of biofilm vibrios exacerbate the impacts of vibrios on human health and greatly increase difficulties in monitoring pathogenic vibrios in marine environments. Furthermore, the increasing persistence and dissemination of vibrios in aquatic environments and the in-
creasing incidence of human vibrio illnesses worldwide are linked to phytoplankton blooms, ocean warming, and the capacity of vibrios to colonize surfaces (15, 265, 560–564). Higher temperature also significantly increases population abundance and up-regulates virulence determinants such as motility, resistance to antimicrobial compounds, hemolysis, and cytotoxicity in coral pathogens (565, 566). Surface-living vibrios are thus important not only in global carbon and nitrogen cycling but also in human and marine animal health. How vibrio ecophysiology, biogeochemical function, and pathogenicity may respond to, react with, and evolve in response to the impact of global climate change, ocean acidification, and ocean deoxygenation warrants further investigation.

### MICROORGANISM-SURFACE INTERACTIONS IN SURFACE COLONIZATION

Surfaces once submerged in marine waters are rapidly colonized, and subsequent biofilm formation follows a sequence of chemical and biological events. These events may include the rapid formation of an initial “conditioning film,” colonization by pioneer microorganisms (usually bacteria), recruitment of secondary colonizers and growth of microcolonies, and development and maturation of biofilm architecture and the biofilm microbial community (17, 22, 397, 567–569). Chemical interactions of solutes with substratum surfaces, biological interactions of microbial cells with surfaces and other microbial cells, and specific gene regulation events at the individual, population, and community levels may play important roles in microbial surface colonization, modification of surface physicochemical properties, structured biofilm development, and establishment and maturation of functional communities.

#### Surface Conditioning Film Formation and the “Masking Effect”

Almost any kind of solid substratum, once submerged in seawater, is quickly (in seconds) and inevitably covered with a layer of adsorbed molecules that form a conditioning film prior to the attachment of microbial cells (47, 342, 570). Proteins and glycoproteins are usually the major constituents of conditioning films (571, 572), although lipids, polysaccharides, nucleic acids, aromatic amino acids, uronic acids, humic acids, and some other biomolecules may also be present (573). The conditioning film affects the surface nutritional conditions and physicochemical properties, usually causing a convergence of surfaces that initially vary strongly in hydrophobicity and roughness (6, 7, 572). The initial surface-colonizing microbial communities thus may be similar due to the masking effect of the conditioning film on the surface chemistry of different substrata (17, 574). However, the net effect of the conditioning film on microbial surface adhesion remains controversial. Different surface components, such as proteins, nucleic acids, and lipids, may facilitate the attachment of different bacteria (575). Thus, the formation of the conditioning film may either stimulate or inhibit adhesion by specific organisms (325). Furthermore, surfaces with different substratum physicochemical properties may select different primary surface-colonizing microbial communities in spite of the masking effect of conditioning film (20, 576). The composition of the primary colonizing microbial community is likely to be determined by the relative contributions of the masking effect of the conditioning film and the native surface physicochemistry of the substratum. For example,

reactive or energetic surfaces may change or modify the chemical properties of the common conditioning film and thus may select for surface-colonizing microbial communities that differ from those on inert surfaces (18, 396). It is reasonable to hypothesize that the constituents of either the conditioning film or the surface substratum that exert the strongest biological effect may be the most influential factors controlling the composition and structure of the primary surface-colonizing microbial community. In addition, seawater TEPs may rapidly adhere to a newly submerged surface to form scattered organic film patches, participating in surface conditioning (though only in small and localized areas) and bringing preexisting TEP-colonizing microorganisms to the growing surface-associated microbial community (577). This adds another dimension of complexity to microorganism-surface interactions and surface-associated microbial community composition and dynamics in marine environments.

#### Key Microbial Surface Components for Colonization

The initial microbial association with a surface in natural seawater starts with the transport of the microorganism to the surface, likely facilitated by diffusive or convective transport and active swimming (6). Passive and active motion may play a critical role in helping to overcome the diffusion barrier and the potential energy barrier produced by electrostatic repulsive forces (6, 578). Microorganisms are able to sense and respond to surface environmental signals and actively initiate surface adhesion by altering gene expression with consequent changes in cell surface chemistry, physiology, and behavior (56, 359). Different microorganisms may respond to distinct environmental signals, allowing each microbial species to efficiently colonize its preferred surface (micro)environment and to avoid direct competition (360). Different microorganisms may also employ different surface adhesion mechanisms, which are usually associated with different secretion systems (343). Different cell surface components may have distinct functional roles in microbial surface colonization. In particular, some components may mediate specific interactions, and others may mediate nonspecific interactions with surfaces. The “secretome” of a microorganism thus defines its colonization potential on various substrata (343). Microbial attachment to abiotic surfaces is generally thought to be mediated by nonspecific processes, while attachment to biotic surfaces is usually mediated by specific processes (62). Cell surface components involved in specific substratum attachment may recognize distinct surface physicochemical or biotic properties, which may also play a role in regulating the expression and effectiveness of these surface-adhering components (579).

Specific ligand-receptor interactions have been found to play an important role in bacterial attachment to biological surfaces and biofilm formation (48). For nonspecific interactions, certain microbial outer surface structures, such as flagella, fimbriae, pili, and curli, as well as proteinaceous, polysaccharide, and eDNA components (other than adhesion-specific ligands and receptors), collectively known as adhesins, may be essential in the attachment process, particularly during the transition from the initial reversible interaction mediated by surface physicochemical properties to irreversible adhesion (48, 343, 580–584). In addition to these seemingly common mechanisms, some other and potentially unique mechanisms may exist in marine bacteria to facilitate surface colonization and biofilm development. For example, in the marine bacterium *Pseudoalteromonas* sp. strain D41, four outer membrane proteins were identified as being involved in surface adhesion and biofilm formation (585).brane proteins, homologous to a TonB-dependent receptor (TBDR), the OmpA and OmpW porins, and a type IV pilus biogenesis protein, respectively, were identified to be important for the biofilm formation process on hydrophobic and hydrophilic surfaces (585). So far, the involvement of the TBDR in biofilm formation has been found only in marine bacteria (585).

### The Holdfast, a Specialized Colonizing Apparatus in Primary Surface Colonizers

Numerous marine bacteria, especially those in the MRC group (8, 272), the iron-oxidizing Betaproteobacteria and Zetaproteobacteria classes (586), the sulfur-oxidizing Beggiatoaceae and Leucotrichaceae families (587, 588), morphotype IV of the Blastocaulis-Planctomycetes group (14, 589), and budding and prosthecate/stalked bacteria such as the Hyphomonadaceae and Caulobacteraceae (590–592), produce a polar holdfast structure to facilitate surface colonization. Some other marine bacteria may also produce a holdfast, as indicated by their ability to form rosette-like aggregates, a characteristic associated with (though not shown to date to be directly connected to) holdfast production (593–595). The expression of the holdfast seems to be inducible, by direct contact with a surface or other bacteria or by specific microbial physiological status or environmental conditions (591–593, 596–598). In Caulobacter crescentus, a sequence of specific steps is involved in surface colonization, with initial reversible adhesion mediated by pili, followed by an arrest of flagellar rotation and subsequent induction of a holdfast for irreversible adhesion (599). C. crescentus produces its holdfast only at the appropriate time for surface attachment, and the flagellum serves as the mechanosensor for the induction of holdfast expression and adhesion (481, 600). Some other marine bacteria, such as Hyphomonas sp. strain VP-6, may use a similar mechanism for surface colonization (591), and surface mechanosensing mechanisms employing flagella were found in several other bacteria, including marine vibrios (15, 481, 601, 602).

MRC bacteria have been identified as the key pioneer colonizers on both abiotic and biotic surfaces in marine environments (8, 13, 20, 136, 173, 272, 450, 574, 603, 604). Recently, the iron-oxidizing Zetaproteobacteria were identified as a group of pioneer colonizers contributing to early-stage carbon steel biocorrosion in marine environments (18, 81). Other putative holdfast-producing bacteria are also frequently detected in marine surface-associated environments. For example, Planctomycetales colonize marine particles and algal or abiotic surfaces (12, 14, 605–607), and sulfur-oxidizing, iron-corroding Leucotrichaceae colonize algal and submerged carbon steel surfaces (18, 608). Stalked or prosthecate Caulobacter and Hyphomonas bacteria are primary colonizers of algal or submerged surfaces (246, 266, 607). It is reasonable to hypothesize that holdfast-mediated irreversible attachment may be a key step in surface colonization by most of these pioneer bacteria.

Flagellum-mediated motility and surface mechanosensing may play critical roles in holdfast-mediated surface attachment by flagellated bacteria (593). Ruegeria sp. strain TM1040 (previously Silicibacter sp. strain TM1040) mutants defective in wild-type swimming motility, due to a loss of flagella or to increased cell length, are also defective in attachment to dinoflagellates (609). Although most holdfast-producing (or rosette-forming) bacteria possess a polar monotrichous flagellum (590–592), not all of the holdfast-producing bacteria have a polar flagellum or flagella. Some bacteria use polar fimbriae for initial surface contact, followed by the use of the holdfast for subsequent irreversible attachment, which is likely induced by fimbria-surface interactions (596, 597). Genes encoding the Aggregatibacter actinomycetemcomitans (previously Actinobacillus actinomycetemcomitans) homologue Tad and Flp fimbrial proteins are present in select MRC bacteria (610), likely playing a role in surface colonization (595). Some other bacteria may induce holdfast expression and attachment in response to direct cell surface-substratum contact. It is likely that the flagellum is the major, but not the sole, mechanism of surface mechanosensing and holdfast induction for surface colonization in marine microorganisms.

### Cooperation and Competition in Shaping the Composition, Dynamics, and Function of Multispecies Biofilms

Marine surface-associated communities are composed of diverse microbial species (17, 108, 151, 153, 574, 611, 612), which usually form biofilms with specific structures and functions (13). Biofilms are composed mainly of a highly hydrated EPS matrix that encases both surface-associated microorganisms and their extracellular products (29, 613). In addition to structural components such as extracellular polysaccharides, proteins, nucleic acids, lipids, and other biopolymers, such as humic substances, which collectively determine key biofilm microenvironmental physicochemical properties, including matrix density, porosity, water content, hydrophilicity, charge, sorption capacity, mechanical stability, fluid dynamics, and mass transport (29, 59, 584), the biofilm matrix also contains microbial functional components such as extracellular enzymes, intraspecies and interspecies signaling molecules, toxins, and extracellular membrane vesicles (EMVs) that facilitate microbial interactions (29, 53, 61, 62). Besides playing important structural and functional roles in mature biofilms, microbial interactions also occur during early surface colonization and biofilm development stages, contributing to the diversity and succession of surface-associated microbial communities (62, 567).

#### Coaggregation, a Common Mechanism for Microorganism Recruitment to Surfaces

Besides initial surface attachment, microorganisms can also be recruited to the surface-associated community by secondary microorganism-microorganism and microorganism-surface matrix interactions (397). Coaggregation has been proposed as a central mechanism, likely mediated by specific cell surface adhesin-receptor interactions between participating microorganisms (614), for the formation and development of multispecies biofilm communities (567, 615, 616). Some surface-colonizing microorganisms performing this bridging function recruit different microorganisms to join in the development of the surface microbiota (616). It has long been recognized that coaggregation may be a key driver shaping biofilm community composition and function in diverse environments (616). Primary surface-colonizing bacteria commonly employ coaggregation as a mechanism to recruit secondary surface colonizers to establish sequential successional dynamics and the ordered spatial structure of the biofilm community. This has been most thoroughly studied in freshwater and wastewater systems and in human oral biofilms to date, although the implications for marine biofilms are clear. Freshwater and wastewater bacteria coaggregate (567, 617–619), and some freshwater bacteria that can serve as bridging mediators for recruitingdifferent microorganisms to the surface-colonizing microbial community have also been identified (567, 620–622). Coaggregation of both early and later surface colonizers with the bridging microorganisms contributes to the dynamic changes of species composition and diversity observed in surface-associated microbial communities (567, 616, 623). Chemotaxis is normally required for effective microbial coaggregation (624), and freshwater bacterial coaggregation is strongly influenced by the metabolic status of the microorganisms involved and environmental conditions such as nutrient availability, pH, and ionic strength of the aquatic system (625, 626). Environment-regulated and/or cell physiology-controlled expression of the microbial coaggregation adhesins or receptors may be the reason for the observed phenomena (61). Interspecies coaggregation mediated by the specificity of adhesin-receptor interactions may be driven by the evolution of the microbial partners (627), which may also lead to the establishment of broader cooperative traits (628, 629). For example, ammonia-oxidizing bacteria (AOB) and nitrite-oxidizing bacteria (NOB) form symbiotic partnerships via coaggregation to carry out nitrification in activated sludge flocs and biofilms (619, 630, 631). This cooperation is essentially facilitated by the spatial juxtaposition of these two groups of bacteria and by the effective transfer of nitrite from AOB as a product of energy metabolism to NOB as a substrate for energy metabolism. In similar ways, coaggregation likely facilitates other metabolic cooperation processes via the transfer of other metabolites or energy in surface-associated microbial communities (632–634).

Coaggregation draws different microorganisms into close spatial juxtaposition within multispecies biofilms, which not only may increase the diversity of species composition and metabolic pathways of the community but also may enhance the opportunity for and efficiency of cell-cell signaling, metabolite transfer/exchange, cross-species protection, genetic exchange, and contact-dependent gene expression (61, 567, 628). Although investigations of coaggregation partnerships and their consequences have scarcely been undertaken with marine bacteria (635, 636), it is reasonable to hypothesize that coaggregation-based cell-cell interactions may play as important a role in marine biofilm formation as in freshwater environments (616, 628). According to McCormick et al. (628), microbial coaggregation associated with marine particles may help to establish a parsimonious food chain, increasing the energetic potential of metabolites of these resource islands in generally oligotrophic surroundings.

### Cheating: It Happens in the Microbial World, Too

Surface-colonizing microorganisms also possess diverse mechanisms of competition. Competition sensing is a recently proposed bacterial strategy for the direct detection of and response to, via stress responses such as those caused by nutrient limitation, ecological competition, which is particularly intense in surface- and biofilm-associated communities (637). Some primary surface colonizers may inhibit colonization by other and/or later-arriving taxa (638), and this competition may be quite subtle. *P. aeruginosa* in multispecies microbial biofilms increases the production of organic iron chelators, siderophores, in response to increasing Fe scarcity, thus enjoying a competitive advantage over other microorganisms (639–641). Although this strategy may increase the ability of *P. aeruginosa* to sequester iron and thus outperform other microorganisms, it also produces the opportunity for siderophore nonproducers, “cheaters,” may be able to harvest more iron via *P. aeruginosa*-produced siderophores without paying any cost of siderophore production (639). Such cheating microorganisms may actively colonize or be recruited as secondary colonizers of submerged surfaces.

Iron is an essential trace element, existing mainly as solid, very-low-solubility, Fe³⁺-bearing mineral phases in oxic and pH-neutral environments (433, 642). Due to its very low concentration (643), dissolved iron constitutes a limiting micronutrient for primary productivity in large areas of the world’s oceans, particularly in high-nutrient (nitrate, phosphate, and silicate) but relatively low-phytoplankton-biomass areas (644, 645). Furthermore, marine bacteria contain more iron per unit biomass than phytoplankton, and thus, bacterial iron assimilation may constitute another constraint on iron availability to phytoplankton (646). Iron is also a key resource limiting microbial N₂ fixation, phosphate acquisition, and, thus, productivity in the ocean (181, 647–649). It has been proposed that iron may control productivity in half of the world’s oceans (650, 651) and may have accounted for one-quarter of the decrease in the atmospheric CO₂ concentration during the Earth’s historical glacial maxima (652). Besides being a limiting resource for photosynthesis and N₂ fixation, iron is also an essential functional component of key enzymes in respiration, DNA replication, fatty acid metabolism, and other vital cellular and physiological processes (653). In the global ocean, the iron cycle affects, directly and indirectly, the biogeochemical cycling of C, N, P, Si, and S and thus exerts a strong influence on the regulation of the Earth’s climate (654). Although the bulk concentration of particulate iron (~0.4 nM) is similar to the bulk concentration of free dissolved iron (0.03 to 1.0 nM) in the surface ocean (642), marine particle-associated iron is highly localized and concentrated. Thus, marine particles may provide a rich source of iron for particle-associated microbial processes (53, 177, 181, 227, 301, 642, 655). This feature is exploited by many marine microorganisms, especially in environments with strong iron limitation, for CO₂ fixation, N₂ fixation, and other key biogeochemical processes (155, 656). Under iron stress conditions, many marine particle- and surface-associated microorganisms produce and secrete siderophores to facilitate iron dissolution and uptake (657, 658), and this physiological process is usually coregulated with the microbial surface- and biofilm-associated physiology and modulated by the TCS, QS, sRNA, and c-di-GMP regulatory systems (179, 659–663). Interestingly, some surface- or biofilm-associated bacteria possess the genes for siderophore-specific transport systems such as the TonB-dependent outer membrane transporters or ABC-type siderophore transport systems (664, 665) while lacking the genes for siderophore production (666, 667). These siderophore-nonproducing cheaters may gain the benefit of taking up siderophore-bound iron if they can recruit as surface colonizers localized near siderophore-producing microorganisms (35, 668). Cheating as a special form of competition may be important in surface- and biofilm-associated marine microbial communities.

Quorum sensing provides a fundamental means of microbial cooperation (39, 441). In natural environments, this cooperative behavior benefits the biofilm population and community, for example, by synchronizing the production and secretion of extracellular enzymes for efficient degradation and utilization of biopolymeric substrates (669, 670). However, QS-based cooperation may suffer from and even be compromised by cheating. For example, in natural environments, 50% of *V. cholerae* strains may be QS
deficient, and interestingly, some of these strains may actively cheat by false signaling to lure the QS-capable strains to produce QS-dependent “public goods” (670). These cheating bacteria behave very much like “swindlers,” whereas the above-mentioned siderophore-nonproducing bacteria behave very much like “thieves.” However, both strategies of cheating seem to be effective for competition in surface-associated microbial communities. Although cheating may lead to a disruption or breakdown of cooperation (671), cheaters are very common in the microbial world, especially within high-density populations or communities such as those on surfaces and in biofilms (672). Lee et al. (673) proposed that the existence of cheaters in a cooperative microbial community may provide a general mechanism for the evolution of diversity that is involved in providing public goods, such as siderophores for iron scavenging, extracellular enzymes for metabolizable substrate acquisition, quorum sensing autoinducers for population or community adaptivity, extracellular matrix biopolymers for biofilm formation and structure, surfactants for motility on surfaces, and exotoxins for host invasion (668, 671, 672, 674–676). The functionality and sustainability of a biofilm microbial community may depend upon the balance between cooperative and competitive interactions (677), likely driven by the coevolution of cooperators and cheaters and maintained by the compositional and metabolic diversity in the microbial system (678).

### Deadly Competition: Chemical Agents, Predation, and Specialized Weapons

There are various competition strategies that are more directly antagonistic than cheating among microorganisms (62, 679), especially on surfaces or in biofilms, where high microbial densities and close spatial proximities are achieved. More than 50% of marine bacterial isolates were found to be antagonistic toward other bacteria, and this trait was more common in particle-associated bacteria than in free-living bacteria (680, 681). For example, many MRC bacteria produce antimicrobial substances such as tropodithietic acid (TDA), indigoidine, tryptanthrin, and peptide antibiotics (8, 173, 272, 450, 594, 682). TDA biosynthesis is modulated by the QS and c-di-GMP regulatory systems in MRC bacteria, and TDA also induces its own synthesis as well as bacterial surface attachment, indicating its roles in the bacterial motile-to-sessile lifestyle switch and interspecies competition (483, 683, 684). Indigoidine synthesis is also regulated by QS and provides a competitive advantage that contributes to the surface colonization success of its producers (450, 685). Surface-associated MRC bacteria are over 10 times as likely as their free-living counterparts to produce antibiotics in marine environments (680), and this difference may be related to the differences in bacterial gene repertoires (8, 173, 594). Antibiotic production may play a role in the success and prevalence of MRC bacteria as pioneer surface colonizers in the ocean (8, 17, 173, 272, 450).

Some other marine bacteria, such as *Bdellovibrio* and like organisms (BALOs), which are affiliated with the *Bacteriovoracaceae* and *Bdellovibrionaceae* in the *Deltaproteobacteria* lineage and *Micavibrio* in the *Alphaproteobacteria* lineage, are obligate predators that prey on other environmental microorganisms (686). BALOs have evolved host interaction predatory-specific genomic islands (687). As surface-associated microbiota have much higher densities than free-living communities, BALOs are more abundant on surfaces, exploiting a rich resource of prey microorganisms in marine environments (688, 689). Living in biofilms also provides protection against extreme or hostile environmental conditions for BALOs, enhancing their survival in nature (690). BALOs may employ chemotaxis to respond to chemoattractants and to track prey bacteria (691) and employ gliding motility to “scout” for prey on surfaces (692). *Bdellovibrio bacteriovorus* predation requires the type IV pili (693), which may play an important role in initial attachment to a prey bacterium in aquatic environments and possibly in movement for locating prey bacteria within the matrix of biofilms (694). BALOs are phylogenetically and environmentally diverse in the ocean (695–697), and they display niche separation, different predation strategies, and prey selectivity such that some BALOs are more specific for particular prey organisms, while others are more prey generic (696, 698–700). Some other bacteria of the Proteobacteria (including Alpha-, Beta-, Gamma-, and Deltaproteobacteria), Actinobacteria, Bacteroidetes, and Chloroflexi lineages are also predatory (701). Marine predatory bacteria may play a role in shaping the composition, abundance, and biogeochemical functions of the affected surface-associated microbiota (62, 699, 702). However, this hypothesis has not yet been systematically tested.

Some surface-associated bacteria use contact-dependent growth inhibition (CDI) systems that constitute cognate toxin-immunity protein pairs for interbacterial competition (703, 704). The CDI systems are mainly type V secretion systems, and the secreted toxins display RNase, DNase, or membrane pore-forming activities toward target cells of the same species, suggesting the involvement of these systems in competition between closely related bacterial strains (703, 705, 706). Thus, CDI systems may enforce cooperation among surface-associated bacteria by inhibiting the growth of cheaters that lack cognate immunity proteins (703). Genomes of many Alpha-, Beta-, and Gammaproteobacteria, including marine species, harbor the genes that encode CDI systems (707–709), which may be prevalent in marine surface-associated microbial communities.

Recently, it was found that more than a quarter of Gram-negative bacteria harbor a type VI secretion system (T6SS), which is involved in bacterial predation on neighboring bacterial cells via a contact-dependent mechanism as well (705, 710). Furthermore, some bacteria have evolved a “tit-for-tat” counterattack strategy, also using the T6SS mechanism (711, 712). Surface association may be favorable to T6SS effector delivery (713), in which the threonine phosphorylation signal transduction pathway (TPP) may play an important role in surface-dependent T6SS activation (714). Nearly one-third of the identified T6SS gene clusters harbor TPP-related components; thus, surface activation of the T6SS by signal transduction may be very common in bacteria (715).

It has been found that the expression and secretion of the antimicrobial T6SS, such as T6SS1 in *V. parahaemolyticus*, are upregulated upon surface sensing (716). The effect of T6SS-mediated intraspecific and interspecific competition may be maximized particularly in high-density populations or multispecies communities such as microbial biofilms and aggregates (717–719). Bacterial T6SS mutants have severe impacts on biofilm formation (718, 720). In *V. cholerae*, the T6SS is required for host intestinal colonization (350, 721). The expression of T6SSs is usually induced by specific environmental signals such as temperature, salinity, cell density, and surface sensing and regulated by mainly the TCSs and by the QS, sRNA, c-di-GMP, cAMP-CRP, and alternative sigma factor systems (350, 713, 716, 722–727), all of which are

involved in the modulation of microbial surface colonization and in the regulation of gene expression in biofilms (414, 728). The coordination of the TCS or the QS, sRNA, or c-di-GMP regulatory system with the expression of T6SSs may potentially aid in microbial surface colonization fitness via the displacement of competing bacteria at locations having growth-supportive conditions and via the promotion of horizontal gene transfer (718, 729). For example, in *V. cholerae*, the expression of the T6SS is regulated by quorum-regulatory Qrr sRNAs, which repress the T6SS genes at low cell density (727). This process may help *V. cholerae* to conserve resources and gain benefit in a multisubpopulation or multispecies biofilm, as assembly of the T6SS requires the synthesis and secretion of a cluster of protein components and would not be advantageous when there are few targets available (713, 730, 731). Although there is no direct evidence showing a QS regulatory effect on bacterial T6SS functioning in natural marine environments, QS signal compounds have been identified in marine surface-associated microbial communities (440, 441), indicating that it is possible that both mechanisms may be functional and coordinated in bacterial ecophysiology in marine surface-associated microbiota.

Alphaproteobacteria (especially MRC bacteria) have been demonstrated to be the key primary surface colonizers in marine environments (8, 13, 20, 173, 272, 450, 604). However, the T6SSs are found primarily in Gammaproteobacteria among the marine bacteria (732). Recently, T6SSs have also been identified in some bacterial strains, genomes, and marine metagenomes of Bacteroidetes (733–736), an important group of secondary surface colonizers in marine environments (13, 17, 108, 130, 272, 574). Therefore, it is reasonable to hypothesize that T6SS-mediated microbial competition may contribute to the successional change and spatial variation of the composition and structure of surface-associated marine microbial communities, in which certain T6SS-harboring bacteria may constitute important groups of secondary colonizers.

### Extracellular Membrane Vesicles as Mechanisms for Both Cooperation and Competition

Cooperation and competition as important ecological characteristics are not just for surface-associated microorganisms. Free-living marine microorganisms, including the ocean ecosystem-dominant cyanobacterial genera *Prochlorococcus* and *Synechococcus*, many heterotrophic bacteria, and seawater microbial communities in both coastal and oligotrophic open-ocean environments, have recently been found to produce and secrete EMVs that contain proteins, lipids, DNA, and RNA (737). The proposed benefits of this ecological phenomenon include (i) enhancing microbial nutrient sensing and uptake via EMV-contained transport receptors, substrate-binding proteins, and degradative enzymes that target environmental HMW molecules; (ii) stimulating neighboring helper microorganisms to grow, produce, and share beneficial products such as enzymes, vitamins, siderophores, and other materials that the EMV-secreting microorganism cannot produce; (iii) facilitating microbial communication and horizontal gene transfer via mediating cell-to-cell exchange of signal molecules and DNA at the ecosystem level; and (iv) preventing phage attack by using the secreted EMVs as decoys (737–741). For example, the EMVs of the psychrotolerant bacterium *Pseudoalteromonas antarctica* NF3 contain proteases, peptidases, glycosyl hydrolases, lytic transglycosylases, glycosyltransferases, TBDRs, and sulfate-binding proteins (738). The EMVs of another Antarctic bacterium, *Shewanella livingstonensis* NF22, also contain TonB-dependent siderophore receptors, phosphate-binding periplasmic protein precursors, and proteins involved in aromatic hydrocarbon degradation (739). Most of these proteins are likely involved in organic matter degradation and nutrient uptake, potentially important to bacterial survival in the nutrient-limited Antarctic environment. EMVs can also be employed to attack competing microorganisms (19, 29, 741, 742). For example, the EMVs of *P. antarctica* NF3 also contain enzymes that hydrolyze bacterial cell wall polymers (738). However, the benefits provided by EMVs may come at a very high cost for free-living microorganisms, especially for those living in oligotrophic marine environments (740). It has been estimated that ~10⁴ to 10⁵ tons of fixed carbon are released into the ocean daily via EMVs by *Prochlorococcus* alone (737), without taking into account other EMV-secreting microorganisms and other limiting resources such as N, P, and Fe contained in the EMVs. In addition, the benefits are not very reliable for free-living microorganisms because there is little or no control of the fates and effects of the EMVs once they are released into the surrounding aquatic environment. Physical, chemical, and biotic (from nontarget organisms) factors may all exert impacts on the fates and effects of the secreted EMVs in marine waters.

EMV secretion is common in all three domains of life (743), and EMVs are an important component of the biofilm matrix (15, 19, 29). The secretion of EMVs is regarded as a bacterial stress response and may be regulated by environmental cues (744, 745). EMVs are beneficial for planktonic microbial cells in forming aggregates and contributing to adherence to surfaces and thus facilitate microbial surface colonization and early-stage biofilm development (744, 746, 747). For example, the EMVs of *P. antarctica* NF3 contain several types of adhesins (738). Adhesins in EMVs may promote microbial coaggregation and assist in the recruitment of secondary surface colonizers (748, 749). EMVs may also contain signaling molecules that facilitate intraspecific and interspecific communication and cooperation at the microbial population and community levels (741, 742). *V. cholerae* EMVs contain the major biofilm matrix proteins RbmA, Bap1, and RbmC (750) as well as chitinases (750), indicating once again that chitin is an important resource for vibrios in marine environments and highlighting the importance of surface colonization for chitin utilization. In *V. fischeri*, biofilm formation is correlated with EMV production, which is induced by the sensor kinase RscS (751). How is the production of EMVs regulated in other marine bacteria? Is there any environmental cue or physiological control for the production of EMVs? The answers to these questions will advance our understanding of the processes and mechanisms of microbial EMV production and their contributions to biofilm formation and development.

In biofilms, the EMVs and their contents are confined, at least partially, to the EPS matrix, and they should be both more chemically stable and physically closer to the target microbes (either as cooperators or competitors of the EMV-secreting microorganism). The cost of producing and secreting EMVs in biofilms may be better justified than in planktonic environments. Therefore, a free-living cyanobacterium living in the open ocean may have a harsher life than a cyanobacterium living in a microbial mat (752, 753). Living in a crowded “city,” called biofilm, may be a preferred lifestyle for most aquatic microorganisms if conditions permit.

(22). Furthermore, no matter which mode of life is undertaken, either free-living or surface associated, the microorganisms in nature need to benefit their cooperators, oppose their competitors, and maintain a benign or at least neutral interaction with other not-so-friendly and not-so-hostile neighbors. All of these activities require the expenditure of metabolic energy and precious resources, which may not be easy to obtain in many marine environments. Due to the close positioning of microorganisms, the protective nature of the EPS matrix, and the development of sensing, signaling, and regulatory mechanisms and social behaviors among different microorganisms in biofilms (21, 35), the functional efficiency of biofilm microbial communities should be higher and more stable than those of planktonic microbial communities. In future research, investigations of the ecological functions of the surface-associated and biofilm microbiota that take into account the molecular mechanisms, community processes, material and genetic fluxes, ecosystem metabolism efficiency, and other systems biology perspectives may be highly productive.

### KEY MICROBIAL SURFACE COLONIZERS IN MARINE ENVIRONMENTS

As is highlighted throughout this review, there are two major lifestyles in marine microorganisms, free-living and surface associated. Most marine microorganisms appear to prefer one lifestyle or the other, although some may switch their preference under certain environmental conditions or during certain life stages. For example, some marine bacteria, such as those affiliated with the SAR11 and SAR86 lineages, are mainly free-living (754, 755), while the marine Rhodobacteraceae group of the Alphaproteobacteria (i.e., the MRC), the Alteromonadaceae and Vibrionaceae groups of the Gammaproteobacteria, and Bacteroidetes (mainly the Flavobacteria group) are frequently surface associated (13, 272). The separation of the free-living and surface-associated lifestyles is likely a result of long-term evolution, and these distinct capabilities are deeply rooted in microbial genetics (756–758). The ecophysiology of surface-associated marine Alteromonas and Vibrio is discussed above, so the foci of this section are the MRC and Bacteroidetes.

#### The Marine Roseobacter Clade

MRC bacteria are ubiquitous in the world’s oceans and abundant as both free-living and sessile organisms. When sessile, they are usually associated with the phycospheres of diatoms, dinoflagellates, and other algae and with zooplankton fecal pellets, marine particles, and submerged surfaces (8, 13, 173, 272, 450, 594, 756, 759–761). These bacteria are generally heterotrophs, able to metabolize a variety of labile and recalcitrant organic substrates, including monocyclic and polycyclic aromatic hydrocarbons as well as various algal osmolytes and other metabolites (173, 175, 272, 594, 762–767). They usually react to and grow quickly after small increases in levels of labile organic substrates, such as amino acids, simple sugars, and DMSP, especially during the initial phase of algal blooms (8, 13, 252, 265, 272, 768–772).

Some of the MRC bacteria produce auxins (such as indoleacetic acid), essential vitamins, and siderophores, entering into mutualistic relationships with algae (272, 595, 603, 610, 758, 762, 765, 773, 774). All MRC bacterial genomes harbor the genes that encode c-di-GMP signaling systems, and more than half of the MRC genomes harbor the genes that encode motility, chemotaxis, and diverse chemoreceptor proteins (8, 595, 609, 682, 762, 775), likely important in locating algae and establishing tight interactions and associations (8, 595, 776). Many MRC bacteria have holdfasts, type I and type IV secretion systems, QS regulatory systems, and versatile physiological capabilities for living in suboxic and anoxic (micro)environments (employing denitrification, for example), which are important for living on surfaces and in biofilms (8, 173, 272, 450, 603, 758, 762, 777).

In addition to the physiological and genetic traits that are directly related to surface living, many MRC bacteria have other properties that are advantageous to living on surfaces or in biofilms. Some MRC bacteria conduct aerobic anoxygenic photosynthesis. This process seems to be enhanced by surface association (150, 151, 173, 762, 778–780) and may enhance ATP production, active transport, motility, and cell growth yield on available organic matter (272, 779). Some MRC bacteria synthesize and store poly-β-hydroxyalkanoates when carbon and energy resources are available but inorganic nutrients are limiting (781, 782), which may enhance bacterial viability via poly-β-hydroxyalkanoate catabolism when other sources of organic substrates are scarce (594). Some MRC bacteria also carry out lithotrophic sulfur oxidation to conserve additional energy (783, 784), which may be explored to enhance anaplerotic CO₂ fixation, growth, and survival (761, 785). These traits provide MRC bacteria with additional competitive advantages for energy acquisition, energy conservation, and stabilization of bacterium-alga and bacterium-surface associations.

Many MRC bacteria can produce antibiotics that may enhance their competitiveness against other surface-colonizing bacteria, especially in the phycosphere, where antibiotic production is probably not limited by the availability of energy and organic substrates (8, 272, 450). Furthermore, plasmids, chromids, and other extrachromosomal mobile genetic elements, such as the phage-like gene transfer agents (GTAs), are common in MRC bacteria, contributing to gene transfer, metabolic versatility, and fitness, especially in microhabitats such as those on surfaces and in biofilms (762, 786–789). In some MRC bacteria, the genes encoding the biosynthesis of TDA, siderophores, and extracellular polysaccharides are located on plasmids or chromids, indicating the direct involvement of extrachromosomal genetic material in bacterial surface associations (603, 789). For example, Marinovum algicola DG898 harbors three plasmids and eight chromids, and one of the chromids harbors the 52-kb biofilm functional gene cluster that is essential for surface attachment and adaptation to the phycosphere (790). MRC bacteria are well adapted to surface living, and this trait has been suggested to originate from the co-evolution of MRC bacteria with marine algae (757). Furthermore, MRC bacteria may promote surface colonization and biofilm formation of other marine bacteria (453), likely playing a role in shaping the composition and succession of surface-associated microbial communities (17). All the available evidence indicates that MRC bacteria are important primary surface colonizers in marine environments.

#### Marine Bacteroidetes

Diverse genomic and ecophysiological evidence indicates that marine Bacteroidetes are highly adapted to surface living and POM utilization. Studies have shown that Bacteroidetes bacteria are common in marine environments (791), abundant in organic particle-rich coastal waters (108, 113, 130, 611, 791), responsive to algal and jellyfish blooms (272, 537, 756, 791–793), copiotrophic
(252), and prone to leading a surface-associated life (12, 17, 246, 272, 574, 607) supported by the extracellular degradation of complex biopolymers such as polysaccharides and proteins (194, 791, 794–797). These bacteria harbor a large number of genes for adhesive exopolysaccharides, adhesion proteins, proteases, peptidases, glycoside hydrolases, and lipases, and several genes for biopolymer degradation are coregulated with the genes for TonB-dependent transport systems (794, 795, 798–802). These properties indicate that the marine Bacteroidetes are specialists in surface colonization and play a key role in the degradation and utilization of HMW DOM and POM (272, 791). A recent metagenomic study showed that submerged insoluble polysaccharides such as cellulose are colonized by biofilm-forming marine bacterial communities, of which the Bacteroidetes are a major group that harbors an extensive repertoire of genes encoding glycoside hydrolases (803). Marine Bacteroidetes may also harbor TonB-dependent transport systems that can be used for the uptake of polysaccharides (804).

Marine Bacteroidetes commonly possess gliding motility, which is important for surface living, chitin utilization, and other ecophysiological activities (805), but these bacteria generally do not produce flagella. So how do they locate and colonize surfaces such as marine particles? Azam and Malfatti (186) proposed that nonswimming particle colonizers, such as Bacteroidetes, first attach to the highly abundant small gel particles in seawater. Aggregation of small gel particles with larger particles and agglomeration with other materials, such as phytoplankton or detritus, bring attached bacteria to the large marine particles and aggregates such as marine snow (186). Recently, Bar-Zeev et al. (577) verified that microgel TEPs indeed facilitated biofilm formation in test aquatic systems. The Bacteroidetes were also the dominant group of bacteria attached to TEPs in mesocosm experimental systems, especially under intermediate- and high-turbulence conditions (806).

Some other mechanisms may also contribute to the success of surface living by Bacteroidetes. Certain marine Bacteroidetes harbor rhodopsin pigments for light energy harvesting (807–809), with proteorhodopsins functioning as light-driven H⁺ pumps (810, 811), KR2-type rhodopsins functioning as light-driven Na⁺ pumps (812), and NM-R3-type rhodopsins functioning as light-driven Cl⁻ pumps (813). In all three cases, membrane polarity can be produced and used to drive active transport. Proteorhodopsins are likely the most prevalent rhodopsins in Bacteroidetes and other marine bacteria, and genes encoding their synthesis are extremely abundant and highly expressed. These pigments likely play an important role in energy metabolism in the surface oceans, especially under oligotrophic or other stressful conditions (779, 814–817). Proteorhodopsin-mediated energy conservation of Bacteroidetes may promote growth and survival, facilitate the degradation of complex or recalcitrant biopolymers, and enhance the uptake of organic substrates at low concentrations (818). The phototrophic potential of proteorhodopsin-containing Bacteroidetes was found to be correlated with the quality and dynamics of environmental DOM (819). Furthermore, proteorhodopsin-mediated light energy harvesting significantly enhances Bacteroidetes anaplerotic CO₂ fixation when suitable organic substrates are available (819). Bacteroidetes may also possess sensory-like rhodopsins, which function in phototaxis to direct a bacterium toward desirable light conditions (820, 821). For example, the genome of Polaribacter sp. strain MED152 harbors a suite of genes for light sensing and responses (822). It has been suggested that the proteorhodopsins and the light-sensing proteins may play a role in the dispersion of

Bacteroidetes from particles (822, 823). This mechanism likely involves inducing the secretion of EPS-degrading enzymes that disrupt biofilm and release bacteria from the biofilm matrix, preventing particle-associated bacteria from sinking with the colonized particles into the dark deep water, which usually lacks metabolizable organic substrates. This mechanism may help Bacteroidetes maintain themselves in the relatively productive and labile particle-rich sunlit surface water, contributing to the success of Bacteroidetes on surfaces.

Gliding is a unique movement used by bacteria to explore surfaces (824). The gliding capability of many marine Bacteroidetes undoubtedly contributes to the success and fitness of this group of bacteria for life on surfaces (272). Some marine Bacteroidetes are gliding predators that can prey on other surface-associated bacteria (825) as well as diatoms and cyanobacteria (826). The gliding motility of Bacteroidetes is powered by proton motive force (824), and proteorhodopsin-harboring predatory Bacteroidetes may gain an extra advantage in surface life via light energy-powered gliding motility and microbial predation, although this is speculative. Predatory Bacteroidetes may contribute to the control over surface-associated microbial composition and abundance, algal associations and interactions, and marine carbon and nutrient cycling in photic seawater. However, it is still not clear if predatory capability is common among marine Bacteroidetes. Further investigations are necessary to better understand the diverse ecophysiological processes and biogeochemical roles of this group of bacteria in particles and in biofilms in the ocean.

### Marine Roseobacter Clade Bacteria and Bacteroidetes in Surface-Associated Community Dynamics

Although both MRC bacteria and Bacteroidetes lead a motile-sessile (or floating-sessile) biphasic lifestyle and are frequently found to colonize algal surfaces (595), they may respond to algal blooms differently. At the start of an algal bloom, when the algal population is small and growing, the algae are healthy and active in synthesizing labile DOM such as simple sugars, sugar alcohols, organic acids, amino acids, and DMSP, some of which may be released into the environment as algal exudates (209, 272, 758, 827). In this phase, algal cells provide localized and concentrated labile organic substrate sources, so surface colonization is advantageous to certain marine microorganisms, such as many MRC bacteria (758), and allows rapid responses to the labile DOM substrates. The capability for quick responses to algal exudates makes the MRC bacteria a major group of pioneer surface colonizers (136, 272). With bloom development and the buildup of environmental stresses such as increasing scarcity of inorganic nutrients and accumulation of waste products, the physiological status of the blooming algae changes (828). Usually, unhealthy and senescent algal cells produce and secrete more protective or stress-related extracellular substances, such as polysaccharides (227, 272, 828–830). For example, the secretion of extracellular polysaccharides and the production of TEPs by most marine algae increase significantly under nutrient-limiting conditions that are commonly experienced during the declining bloom phase (831–833). In the demise phase of an algal bloom, dying and broken algal cells release polymeric cellular contents, and some algae may enter the autocatalytic programmed cell death phase of the growth cycle and release huge amounts of TEPs into seawater (834). Furthermore, cellular organic matter can also be released from algal cells by cell lysis due to viral infections, grazing, and sloppy feeding,
Dang and Lovell

especially during blooms, when very high algal population densities are reached (272, 835, 836). The organic algal exudates, secreted TEPs, and lysed cellular contents may facilitate the formation of marine snow and sometimes the massive formation of mucilages in seawater (830, 837, 838). In these late and decaying bloom phases, marine Bacteroidetes may gain dominance in the microbial communities and become the predominant surface colonizers on senescent algae, algal aggregates, and detrital particles (109, 272, 839, 840), attacking and utilizing various kinds of biopolymers (841). Various studies, including recent genomic and environmental genomic investigations, have suggested that certain Bacteroidetes bacteria may prefer to utilize complex, biopolymeric substrates over simple, monomeric organic compounds as primary carbon and energy sources (272, 791, 822), indicating their major roles in cycling algal polymeric DOM and POM.

MRC bacteria and Bacteroidetes likely employ distinct physiological and metabolic strategies during algal blooms (272). Thus, it is reasonable to hypothesize that MRC bacteria and Bacteroidetes may adopt different strategies for exploiting resources from algal blooms, via substrate segregation (utilizing simple labile DOM versus complex HMW DOM and POM) and temporal separation (colonizing at early bloom phases versus at late bloom phases). Indeed, studies have shown that MRC bacteria are key pioneer surface colonizers and that Bacteroidetes are likely secondary surface colonizers of submerged surfaces in coastal seawater (17, 604). Marine Bacteroidetes may be specialized as secondary surface-colonizing experts, armed with T6SSs, EMVs, extracellular N-acyl homoserine lactonases (for disrupting QS-facilitated primary surface colonizer communities), and gliding predation capability for preying on and replacing some of the primary colonizers on the surface (733, 735–737, 825, 842, 843). However, both the MRC and Bacteroidetes groups are highly diverse (272, 594). Different species or strains in each group may have somewhat different ecophysiologies (such as distinct substrate spectra and different responses during bloom progression), and certain bacteria from different groups may have similar ecophysiologies and overlapping niche preferences (8, 272, 768, 844–846). Differentiation of the roles of the MRC bacteria and Bacteroidetes in algal bloom-related processes and ecofunctions may not be straightforward in some situations. Furthermore, although the species and physiological status of algae are important factors influencing the composition, succession, and function of alga-colonizing microbial communities (847), the composition, abundance, and dynamics of metabolizable organic compounds (such as labile DOM and HMW DOM and POM) from algal exudates and phytodetritus may play an even more important role (841, 848, 849). It is necessary to monitor the flows of matter and energy during blooms to gain a better understanding of bacterium-alga interactions and microbial community successions.

Recently, it was shown that the abundance of surface-associated MRC bacteria is decreased while that of Bacteroidetes is increased in response to increases in environmental temperature or elevated partial CO₂ pressure (pCO₂)-induced ocean acidification (322, 324). Similarly, the abundance of planktonic Bacteroidetes also increases in response to increased temperature and/or CO₂ content (and thus decreased pH) in both mesocosms and natural seawater environments (850–852). These results further emphasize the niche segregation and ecophysiological distinction of these two key groups of marine bacteria and indicate their distinct

roles in marine carbon cycling and other critical biogeochemical processes, especially under the scenario of global change.

# FUTURE PERSPECTIVES ON STUDIES OF MARINE SURFACE-ASSOCIATED MICROBIOTA

Although a wealth of information has been obtained regarding surface-associated microorganisms, major gaps in our knowledge remain, especially regarding community structure, dynamics, functions, and the impacts of the changing marine environment. The compositions and structures of the surface-associated microbiota and the processes and mechanisms of microbial surface colonization and biofilm development are highly complex, particularly in dynamic natural marine environments. Many working hypotheses regarding the marine surface-associated microbiota have been proposed in previous studies and throughout this review. Here we summarize a number of scientific questions that should be productive to pursue regarding marine surface- and biofilm-associated microbial communities.

- What are the physicochemical and nutritional environmental cues that marine microorganisms sense and respond to for initiating surface colonization and biofilm formation? Is there any consistency among these cues in different marine environments, such as in estuaries, coastal seas, and open oceans?
- What are the molecular apparatus and mechanisms employed by marine microorganisms to sense distinct environmental cues for initialization of surface colonization and biofilm formation? Do microorganisms in different phylogenetic or functional groups use the same sensing systems and mechanisms for the same environmental cue, or do they vary at group-, species-, or even strain-specific levels?
- How is an extracellular signal from an environmental cue relayed inside the microbial cell, and how is the intracellular response for surface colonization and biofilm formation elicited and regulated in the cell? How may the environmental signal be propagated among different microorganisms to induce a communal behavior in marine surface colonization and biofilm formation?
- What are the major cell surface components that are involved in surface sensing and surface colonization in marine microorganisms? Are *C. crescentus* flagellum-based surface mechanosensing and holdfast-based surface colonization common mechanisms in marine primary surface colonizers? What distinct functional roles may primary and secondary surface colonizers play in surface-associated microbial communities? What role may coaggregation play in the composition and succession of marine surface-associated microbial communities? How do the different mechanisms of microbial cooperation and competition influence the composition, structure, spatiotemporal dynamics, functions, and stability of marine surface-associated microbial communities?
- How may the composition, structure, and functions of surface-associated microbial communities be related to the physicochemical and nutritional differences of distinct surface or particle types in the ocean? How may the interactions of the surface-associated and free-living microbial communities influence each other? How may the surface-associated

microbial communities and functions be influenced by viruses, protozoa, and surface-grazing zooplankton?

- How may surface association influence the genetic, physiological, and ecological fitness of marine microorganisms? Do surface-associated microorganisms evolve faster, driven by more frequent horizontal gene transfer events, than their free-living counterparts? How do plasmids, chromids, GTAs, and other extrachromosomal elements impact surface-associated fitness and its dispersal among marine microorganisms? How can these genetic elements be engineered and employed to control microbial surface colonization and biofilm development, for instance, to prevent biofouling and biocorrosion?

- For clearly identified, commonly occurring marine surface colonizers such as MRC bacteria, Bacteroidetes, Alteromonadaceae, and Vibrionaceae, is there any interaction (cooperative or competitive) among these bacterial groups on surfaces and in biofilms? How may these interactions be influenced by changing marine environments?

- What are the quantitative contributions of marine particle-associated microbiota to the biogeochemical cycling of life-essential and environmentally important elements, primary production (in both photic zone and dark deep waters), and carbon sequestration of the ocean? How may these contributions be altered in response to anthropogenic perturbations and global environmental change impacts? Are surface-associated microbial communities functionally resilient in the face of these impacts? What are the roles of surface-associated microbiota in (accelerating or decelerating) global environmental change? How can we build a mechanistic and prediction-based model?

As microbial surface colonization and biofilm formation and development involve multiple levels of cell-surface and cell-cell interactions, by both direct contact and signal molecule- and metabolite-mediated communication and coordination, laboratory studies using single microbial species and simple mixtures of species are still necessary. Such studies are particularly important for gaining an in-depth understanding of the microbial physiological, biochemical, and genetic characteristics and their environmental responses during key stages of surface colonization and biofilm formation. For single bacterial species, transcriptomic and proteomic studies have revealed useful information about differences in gene expression and protein functions between planktonic and sessile bacterial populations and about the carbon and energy metabolic processes characteristic of distinct biofilm growth states (585, 853). For surface-associated microbial community analyses, molecular approaches such as gene clone library screening, fluorescence *in situ* hybridization (FISH) analyses, and activity assays (especially at the single-cell level) can provide vital information about the composition, abundance, and spatiotemporal variation of the major surface-colonizing microorganisms and their *in situ* activities and functions (854, 855). Mechanistic studies will provide more information about the functions and regulation of the surface-associated microbiota from an ecosystem perspective.

Marine surface-associated microbial communities are intrinsically complex and dynamic, involving diverse microbial species, functional groups, metabolic pathways, sensing and signaling networks, cooperative and competitive mechanisms, genetic exchange and evolutionary potentials, as well as spatiotemporal variation and acclimatization. Furthermore, many different surfaces (including various kinds of particles and aggregates) exist in marine environments (186). The differences among the colonizable substrata add another level of complexity and diversity to the surface-associated microbial communities and their ecophysiology and biogeochemical functions (245). Thus, systems biology approaches are necessary in order to gain an understanding of the community composition, dynamics, and especially the function and its regulation of the marine surface-associated microbiota (856). These approaches will be particularly helpful in decoding the higher-level characteristics of surface-associated microbial communities, such as various cooperative and other sociomicrobial functions (35, 61). “Omics” methods and related bioinformatics analytical tools, which are generally capable of dealing with high-throughput, rapid, and complex analyses, have been proposed as vital approaches (857).

Recently, the *in situ* gene expression of a chemolithoautotrophic Epsilonproteobacteria-dominated biofilm from a deep-sea hydrothermal chimney was analyzed by using metatranscriptomics (858). Although typical surface-associated microbial communities in marine waters may be much more complex than the biofilm communities in extreme environments such as deep-sea hydrothermal vents, the rapid development of omics-related approaches presents a promising opportunity to make strides in understanding the marine surface-associated microbiota (859). Omics techniques have already been adopted and are being investigated for marine biofilm microbiota studies (860, 861). Although there seems to be a multitude of opportunities to improve these techniques, it is reasonable to predict that substantial advances are in the offing.

Recent omics investigations of marine particle-associated microbial communities revealed a wealth of information about certain common characteristics of microbial particle colonizers. Metagenomic investigations showed that MRC bacteria are abundant on particles in estuarine, coastal, and polar waters (151, 758, 862), consistent with previous 16S rRNA clone library-, 454 pyrosequencing-, genomics-, and FISH-based study results that found that these bacteria are key primary surface colonizers in coastal waters (8, 13, 20, 54, 173, 272, 450, 594). Metagenomic investigations have also indicated that particle-associated microbial communities generally harbor a more diverse and complex gene repertoire than free-living communities, such as higher genetic potential for transporters of particle biopolymer degradation products and adaptations to life under hypoxic and anoxic conditions. This includes enrichment in genes related to sulfate reduction, methanogenesis, and anammox (151, 161, 863). Recently, high rates of N₂ fixation and *nifH* gene expression by heterotrophic diazotrophs were detected in fully oxygenated marine waters, suggesting that particle association may be the key mechanism to provide the hypoxic or anoxic conditions necessary for this process in such environments (155, 864). Metatranscriptomics studies indicate that different microorganisms may employ distinct adaptive strategies for the use of either free-living or particle-associated habitats in the ocean (865). Particle-associated communities usually harbor more genes that mediate microbial surface colonization, cell-cell interactions, signaling, and transposase-based mobile genetic element activity, all of which are important for surface living and fitness on marine particles (151, 161). In line with these findings, particle-associated microbial

Dang and Lovell

communities transcribed more copies of genes encoding signaling and surface adhesion cellular components that are related to biofilm formation than free-living communities (866). Particle-associated microbial communities also transcribed more copies of genes encoding metabolic pathways that are related to reducing environments (153). For example, transcripts encoding enzymes for microbial denitrifying N₂O and N₂ production were enriched up to 28-fold in particle-associated samples found in the OMZ of the Eastern Tropical North Pacific (167). Although there have been no metaproteomic studies on size-fractionated particles to date, preliminary metaproteomic studies without size fractionation have inferred certain key properties of particle-associated microbial groups in marine environments. For example, these studies have shown that MRC bacteria are rich in membrane transporter expression for the uptake of labile organic substrates, especially during algal blooms (867, 868). Metaproteomic studies have also suggested that marine Bacteroidetes (mainly the Flavobacteria group) are specialists in attachment to and growth on algal surfaces or detrital particles (194, 841, 867). A recent proteomic study revealed the major functional proteins in V. cholerae vesicles (750). These approaches are also suitable for studying microbial surfomes (869) to decode key surface-associated processes such as signaling, adhesion, transport, and cell-cell and cell-environment interactions. Metaproteomics also have great potential for studies of marine extracellular enzymes, vesicles, and cell surface proteinaceous determinants of surface-associated microbiota. In addition to the identification of surface-induced gene expression and functional adaptivity, metatranscriptomic and metaproteomic approaches show the potential to identify fine-scale spatiotemporal dynamics and interspecies interactions (such as cooperation and competition) in complex microbial communities (870, 871), suitable for process and functional analyses of surface-associated microbiota.

Diverse sensing mechanisms and cell surface and extracellular components are involved in microbial surface interactions and surface living. Several key cellular components or systems can thus be defined specifically for surface- or biofilm-associated microbial communities, such as the metasensoritomes, metasecretomes, and metasurfomes (343). These systems are involved in key steps of microbial surface colonization and biofilm development on surfaces, such as initial cell-surface interactions of the pioneer colonizing species, interspecies cell-cell interactions between primary colonizers and secondary colonizers for recruiting new microorganisms and metabolic pathways, and microbial interpopulation interactions leading to spatial variation and temporal succession of the colonizing community. Furthermore, genome-scale metabolic network reconstructions have become a powerful tool for systematic understanding, prediction, and discovery of the genetic and biochemical potentials of an organism (872). This technique also serves as a platform for constraint-based analyses and modeling of microbial communities (883). The integration of community metabolic network reconstructions (constrained by the community signaling and regulatory networks) with ecological and biogeochemical modeling may provide a fundamental framework for mechanism- and prediction-based modeling of both the biological components and the biogeochemical functions of marine ecosystems (874, 875). Thus, omics approaches, along with cultivation and conventional molecular approaches, provide the means for targeted investigation of surface-specific community-level microbial features, which may lead to an enhanced understanding of surface-associated microbial community ecology and biogeochemical functions, at the levels of both molecular mechanistic details and systems biological comprehension.

APPENDIX

**Definitions**

remineralization depth The depth at which particulate organic carbon is consumed and respired by marine animals and microorganisms. The remineralization of marine organic particles follows an exponential decay pattern with water depth. The remineralization depth determines the degree to which the respired CO₂ that is returned to the water column can influence air-sea CO₂ partitioning and, thus, climate.

POC (particulate organic carbon) The organic carbon in particulate form that is large enough to be retained on a filter (typically with a filter with a pore size of 0.7, 0.45, or 0.22 μm).

DOC (dissolved organic carbon) The organic carbon remaining in the filtrate after the sample is filtered (typically with a filter with a pore size of 0.7, 0.45, or 0.22 μm).

marine snow Mostly biogenic particles with a diameter of >0.5 mm. These organic particles are usually formed in the euphotic zone of the ocean and sink at high rates to serve as the principal means by which organic carbon is transported to the deep ocean and sediments.

transparent exopolymer particles Small organic particles (less than a few hundred micrometers) that are visible under a light microscope only by staining with an acidic polysaccharide-specific dye, such as alcian blue, or are otherwise transparent and invisible by light microscopy. They are abundant in marine waters and formed by extracellular biopolymeric substances exuded by phytoplankton and bacteria.

biological pump The vertical transportation of photosynthetically produced organic carbon, mainly in the particulate form, from the euphotic surface ocean to the dark deep ocean. The biological pump provides one of the biological mechanisms contributing to the oceanic sink of atmospheric CO₂ via the settlement of biogenic organic particles out of the ocean surface waters.

microbial carbon pump The process of microbial transformation of labile dissolved organic carbon to recalcitrant dissolved organic carbon, contributing to ocean carbon sequestration in the dissolved organic phase.

microbial loop The pathway of carbon and energy flow, via heterotrophic Bacteria and Archaea, from dissolved organic matter to bacteriovorous protists and further to other animals at higher trophic levels in the aquatic food web.

epipelagic zone Also referred to as the “euphotic zone,” the water layer from the air-sea interface down to the depth at which the light intensity falls to 0.1% of that at the ocean’s surface. This illuminated portion of the water column, usually in the upper 200 m in the clearest ocean water where sufficient sunlight is available, sustains net photosynthesis.

mesopelagic zone Also referred to as the “twilight zone,” the water layer immediately below the euphotic zone and usually between depths of 200 m and 1,000 m where sunlight is measurable but insufficient to support net photosynthesis. The mesopelagic zone is usually characterized by intense microbial heterotrophic activities.

bathypelagic zone The water layer from 1,000 m down to ~4,000 m, where only chemoautotrophs (but not photoautotrophs) contribute to marine primary production due to the complete lack of sunlight.

chemolithoautotroph A microorganism that uses CO₂ as its source of carbon for biomass production and cell growth and derives its metabolic energy from the oxidation of reduced inorganic compounds.

mixotroph A microorganism that can obtain its metabolic energy or carbon from more than one conventional source. For example, many chemolithoautotrophic microorganisms can assimilate some organic compounds as supplements to CO₂ fixation.

anaplerotic CO₂ fixation Heterotrophic CO₂ fixation (i.e., heterotrophic
CO₂ assimilation) processes that usually employ carboxylation reactions to incorporate CO₂ into organic intermediate metabolites to replenish the tricarboxylic acid cycle. This carbon assimilation reaction is an important metabolic activity for carbon acquisition, biomass production, and growth in some heterotrophic microorganisms that do not harbor autotrophic CO₂ fixation pathways.

**oligotroph** A microorganism that tends to live in an oligotrophic environment, such as the subtropical gyres of the ocean, which offers very low levels of organic carbon. An oligotroph is usually characterized by its genetic and ecophysiological adaptation to low-nutrient conditions and may also display slow growth and low rates of metabolism.

**copiotroph** A microorganism that tends to live in an environment that is rich in nutrients (particularly organic carbon). A copiotroph is usually characterized by its genetic and ecophysiological adaptation to high-nutrient conditions, fast growth, and high rates of metabolism when concentrations of suitable substrates are sufficient.

**priming effect** The positive influence of a labile organic matter input on the increased utilization and decomposition of originally refractory organic matter in the environment. This effect may result from enhanced microbial activity stimulated by the supply of labile organic matter.

**biofouling** The impairment or degradation of underwater surfaces, equipment, and structures as a result of the gradual and undesirable accumulation, growth, or activity of living organisms such as bacteria (and their extracellular products), protozoa, algae, barnacles, and other fouling animals on surfaces. Biofouling usually results in corrosion, clogging, contamination, or a decrease in the efficiency of moving parts. Biofouling is a major concern in bioinvasion as well.

**biocorrosion** Also called microbiologically influenced corrosion or microbially induced corrosion, corrosion caused or promoted by bacteria and other microorganisms, due mainly to their activities on the surfaces and/or in biofilms of the corroding material.

**ALWC (accelerated low water corrosion)** A particularly aggressive form of localized biocorrosion that affects marine steel structures in seawater near the low water tide mark in virtually all the world’s oceans and climates. ALWC usually results in very high rates of metal wastage, up to or even greater than 1 mm year⁻¹, whereas the steel corrosion rate in seawater without ALWC is 0.05 to 0.15 mm year⁻¹.

**phycosphere** The region surrounding a phytoplankton cell that represents a high-nutrient environment. This microhabitat usually harbors a unique microbiome and stimulates specific phytoplankton-bacterium interactions such as parasitism, communalism, or mutualism.

**plastisphere** A unique (micro)environment surrounding human-made plastic debris that is colonized by various microorganisms in the ocean. Plastic debris provides durable surfaces and vehicles for attachment, survival, and long-distance transportation of marine microorganisms (including human pathogens).

**chromid** An extrachromosomal genetic element that carries some core genes and has similar nucleotide composition (such as G+C content) and codon usage as chromosomes but instead harbors the plasmid-type maintenance and replication systems. Chromids are sometimes called “megaplasmids.”

**gene transfer agent** A bacteriophage-like extrachromosomal genetic element produced by some bacteria that mediates horizontal gene transfer via genomic DNA transduction from the donor bacterium to a recipient bacterium.

**sensoritome** The complete set of the diverse microbial surface-sensing machinery and its regulated response products that are involved in a microorganism’s sensing, signaling, and responsive reactions to extracellular environmental cues and population size signals.

**metasensoritome** The whole set of sensoritomes of all the participating microorganisms in a community or a specific environment that are involved in the sensing, signaling, and responsive reactions of the microbial assemblage to extracellular environmental cues and interorganism communications.

secretome The complete set of secretion systems and their secreted/translocated products, such as those involved in the surface colonization, biofilm formation, and development processes of a surface-associated microorganism.

metasecretome The whole set of secretomes of all the participating microorganisms in a community or a specific environment, such as those involved in the establishment, development, and maturation of a surface-associated microbiota.

surfome The complete set of microbial surface-exposed proteinaceous moieties that play important roles in signaling, adhesion, and transport of a microorganism.

metasurfome The whole set of surfomes of all the participating microorganisms in a community or a specific environment, such as those involved in the signaling, adhesion, and transport processes of a surface-associated microbiota.

## ACKNOWLEDGMENTS

We thank Luke E. Ulrich for providing valuable information about the MiST2.2 Database and Rongfang Huang for article style formatting. We also thank the three anonymous reviewers for their valuable comments and suggestions for manuscript improvement.

This work was supported by China NSFC grant 91328209, MOST 973 program grant 2013CB955700, China NSFC grant 91428308, SOA grant GASI-03-01-02-05, and CNOOC grants CNOOC-KJ 125 FZDXM 00TJ 001-2014 and CNOOC-KJ 125 FZDXM 00ZJ 001-2014.

## REFERENCES

1. Costerton JW, Lewandowski Z, Caldwell DE, Korber DR, Lappin-Scott HM. 1995. Microbial biofilms. Annu Rev Microbiol 49:711–745. <http://dx.doi.org/10.1146/annurev.mi.49.100195.003431>.

2. Petrova OE, Sauer K. 2012. Sticky situations: key components that control bacterial surface attachment. J Bacteriol 194:2413–2425. <http://dx.doi.org/10.1128/JB.00003-12>.

3. Hall-Stoodley L, Costerton JW, Stoodley P. 2004. Bacterial biofilms: from the natural environment to infectious diseases. Nat Rev Microbiol 2:95–108. <http://dx.doi.org/10.1038/nrmicro821>.

4. Trevors JT. 2011. Hypothesized origin of microbial life in a prebiotic gel and the transition to a living biofilm and microbial mats. C R Biol 334: 269–272. <http://dx.doi.org/10.1016/j.crvi.2011.02.010>.

5. Söhngen NL. 1913. Einfluss von Kolloiden auf mikrobiologische Prozesse. Centralbl Bakteriol Parasitenkd Infektionskr 38:621–647.

6. van Loosdrecht MC, Lyklema J, Norde W, Zehnder AJ. 1990. Influence of interfaces on microbial activity. Microbiol Rev 54:75–87.

7. ZoBell CE. 1937. The influence of solid surface upon the physiological activities of bacteria in sea water. J Bacteriol 33:86.

8. Slightom RN, Buchan A. 2009. Surface colonization by marine roseobacters: integrating genotype and phenotype. Appl Environ Microbiol 75:6027–6037. <http://dx.doi.org/10.1128/AEM.01508-09>.

9. Yildiz FH, Visick KL. 2009. Vibrio biofilms: so much the same yet so different. Trends Microbiol 17:109–118. <http://dx.doi.org/10.1016/j.tim.2008.12.004>.

10. Hadfield MG. 2011. Biofilms and marine invertebrate larvae: what bacteria produce that larvae use to choose settlement sites. Annu Rev Mar Sci 3:453–470. <http://dx.doi.org/10.1146/annurev-marine-120709-142753>.

11. Wahl M, Goecke F, Labes A, Dobretsov S, Weinberger F. 2012. The second skin: ecological role of epibiotic biofilms on marine organisms. Front Microbiol 3:292. <http://dx.doi.org/10.3389/fmicb.2012.00292>.

12. Egan S, Harder T, Burke C, Steinberg P, Kjelleberg S, Thomas T. 2013. The seaweed holobiont: understanding seaweed-bacteria interactions. FEMS Microbiol Rev 37:462–476. <http://dx.doi.org/10.1111/1574-6976.12011>.

13. Salta M, Wharton JA, Blache Y, Stokes KR, Briand JF. 2013. Marine biofilms on artificial surfaces: structure and dynamics. Environ Microbiol 15:2879–2893. <http://dx.doi.org/10.1111/1462-2920.12186>.

14. Lage OM, Bondoso J. 2014. Planctomycetes and macroalgae, a striking association. Front Microbiol 5:267. <http://dx.doi.org/10.3389/fmicb.2014.00267>.

15. Teschler JK, Zamorano-Sánchez D, Utada AS, Warner CJ, Wong GC, Linington RG, Yildiz FH. 2015. Living in the matrix: assembly and
control of Vibrio cholerae biofilms. Nat Rev Microbiol 13:255–268. http://dx.doi.org/10.1038/nrmicro3433.

16. Turner JT. 2015. Zooplankton fecal pellets, marine snow, phytodetritus and the ocean’s biological pump. Prog Oceanogr 130:205–248. http://dx.doi.org/10.1016/j.pocean.2014.08.005.

17. Dang HY, Li TG, Chen MN, Huang GQ. 2008. Cross-ocean distribution of Rhodobacterales bacteria as primary surface colonizers in temperate coastal marine waters. Appl Environ Microbiol 74:52–60. http://dx.doi.org/10.1128/AEM.01400-07.

18. Dang HY, Chen RP, Wang L, Shao SD, Dai LQ, Ye Y, Guo LZ, Huang GQ, Klotz MG. 2011. Molecular characterization of putative biocorrod-ing microbiota with a novel niche detection of Epsilon- and Zetaproteobacteria in Pacific Ocean coastal seawaters. Environ Microbiol 13:3059–3074. http://dx.doi.org/10.1111/j.1462-2920.2011.02583.x.

19. Beveridge TJ, Makin SA, Kadurugamuwa JL, Li Z. 1997. Interactions between biofilms and the environment. FEMS Microbiol Rev 20:291–303. http://dx.doi.org/10.1111/j.1574-6976.1997.tb00315.x.

20. Dang HY, Lovell CR. 2000. Bacterial primary colonization and early succession on surfaces in marine waters as determined by amplified rRNA gene restriction analysis and sequence analysis of 16S rRNA genes. Appl Environ Microbiol 66:467–475. http://dx.doi.org/10.1128/AEM.66.2.467-475.2000.

21. Davey ME, O’Toole GA. 2000. Microbial biofilms: from ecology to molecular genetics. Microbiol Mol Biol Rev 64:847–867. http://dx.doi.org/10.1128/MMBR.64.4.847-867.2000.

22. Watnick P, Kolter R. 2000. Biofilm, city of microbes. J Bacteriol 182:2675–2679. http://dx.doi.org/10.1128/JB.182.10.2675-2679.2000.

23. Decho AW. 2000. Microbial biofilms in intertidal systems: an overview. Cont Shelf Res 20:1257–1273. http://dx.doi.org/10.1016/S0278-4343(00)00022-4.

24. Matz C, McDougald D, Moreno AM, Yung PY, Yildiz FH, Kjelleberg S. 2005. Biofilm formation and phenotypic variation enhance predation-driven persistence of Vibrio cholerae. Proc Natl Acad Sci U S A 102:16819–16824. http://dx.doi.org/10.1073/pnas.0505350102.

25. Harrison JJ, Ceri H, Turner RJ. 2007. Multimetal resistance and tolerance in microbial biofilms. Nat Rev Microbiol 5:928–938. http://dx.doi.org/10.1038/nrmicro1774.

26. Claessen D, Rozen DE, Kuipers OP, Søgaard-Andersen L, van Wezel GP. 2014. Bacterial solutions to multicellularity: a tale of biofilms, filaments and fruiting bodies. Nat Rev Microbiol 12:115–124. http://dx.doi.org/10.1038/nrmicro3178.

27. Serra DO, Hengge R. 2014. Stress responses go three dimensional—the spatial order of physiological differentiation in bacterial macrocolony biofilms. Environ Microbiol 16:1455–1471. http://dx.doi.org/10.1111/1462-2920.12483.

28. Tan D, Svenningsen SL, Middelboe M. 2015. Quorum sensing determines the choice of antiphage defense strategy in Vibrio anguillarum. mBio 6:e00627-15. http://dx.doi.org/10.1128/mBio.00627-15.

29. Flemming HC, Wingender J. 2010. The biofilm matrix. Nat Rev Microbiol 8:623–633. http://dx.doi.org/10.1038/nrmicro2415.

30. Burmølle M, Ren D, Bjarnsholt T, Sørensen SJ. 2014. Interactions in multispecies biofilms: do they actually matter? Trends Microbiol 22:84–91. http://dx.doi.org/10.1016/j.tim.2013.12.004.

31. Molin S, Tolker-Nielsen T. 2003. Gene transfer occurs with enhanced efficiency in biofilms and induces enhanced stabilisation of the biofilm structure. Curr Opin Biotechnol 14:255–261. http://dx.doi.org/10.1016/S0958-1669(03)00036-3.

32. Wuertz S, Okabe S, Hausner M. 2004. Microbial communities and their interactions in biofilm systems: an overview. Water Sci Technol 49(11–12):327–336.

33. Huq A, Whitehouse CA, Grim CJ, Alam M, Colwell RR. 2008. Biofilms in water, its role and impact in human disease transmission. Curr Opin Biotechnol 19:244–247. http://dx.doi.org/10.1016/j.copbio.2008.04.005.

34. Stewart PS, Franklin MJ. 2008. Physiological heterogeneity in biofilms. Nat Rev Microbiol 6:199–210. http://dx.doi.org/10.1038/nrmicro1838.

35. Nadell CD, Xavier JB, Foster KR. 2009. The sociobiology of biofilms. FEMS Microbiol Rev 33:206–224. http://dx.doi.org/10.1111/j.1574-6976.2008.00150.x.

36. Tuson HH, Weibel DB. 2013. Bacteria-surface interactions. Soft Matter 9:4368–4380. http://dx.doi.org/10.1039/c3sm27705d.

37. McElroy KE, Hui JG, Woo JK, Luk AW, Webb JS, Kjelleberg S, Rice SA, Thomas T. 2014. Strain-specific parallel evolution drives short-term diversification during Pseudomonas aeruginosa biofilm formation. Proc

Natl Acad Sci U S A 111:E1419–E1427. http://dx.doi.org/10.1073/pnas.1314340111.

38. Yawata Y, Cordero OX, Menolascina F, Hehemann JH, Polz MF, Stocker R. 2014. Competition-dispersal tradeoff ecologically differentiates recently speciated marine bacterioplankton populations. Proc Natl Acad Sci U S A 111:5622–5627. http://dx.doi.org/10.1073/pnas.1318943111.

39. Hense BA, Schuster M. 2015. Core principles of bacterial autoinducer systems. Microbiol Mol Biol Rev 79:153–169. http://dx.doi.org/10.1128/MMBR.00024-14.

40. Hall-Stoodley L, Stoodley P. 2009. Evolving concepts in biofilm infections. Cell Microbiol 11:1034–1043. http://dx.doi.org/10.1111/j.1462-5822.2009.01323.x.

41. Ortega-Morales BO, Chan-Bacab MJ, De la Rosa-García SC, Camacho-Chab JC. 2010. Valuable processes and products from marine intertidal microbial communities. Curr Opin Biotechnol 21:346–352. http://dx.doi.org/10.1016/j.copbio.2010.02.007.

42. Poli A, Anzelmo G, Nicolaus B. 2010. Bacterial exopolysaccharides from extreme marine habitats: production, characterization and biological activities. Mar Drugs 8:1779–1802. http://dx.doi.org/10.3390/md8061779.

43. McCave IN. 1975. Vertical flux of particles in the ocean. Deep Sea Res 22:491–502.

44. Smith DC, Simon M, Alldredge AL, Azam F. 1992. Intense hydrolytic enzyme activity on marine aggregates and implications for rapid particle dissolution. Nature 359:139–142. http://dx.doi.org/10.1038/359139a0.

45. Kwon EY, Primeau F, Sarmiento JL. 2009. The impact of remineralization depth on the air-sea carbon balance. Nat Geosci 2:630–635. http://dx.doi.org/10.1038/ngeo612.

46. Herndl GJ, Reinthalter T. 2013. Microbial control of the dark end of the biological pump. Nat Geosci 6:718–724. http://dx.doi.org/10.1038/ngeo1921.

47. Cooksey KE, Wigglesworth-Cooksey B. 1995. Adhesion of bacteria and diatoms to surfaces in the sea: a review. Aquat Microb Ecol 9:87–96. http://dx.doi.org/10.3354/ame009087.

48. Dalton HM, March PE. 1998. Molecular genetics of bacterial attachment and biofouling. Curr Opin Biotechnol 9:252–255. http://dx.doi.org/10.1016/S0958-1669(98)80055-4.

49. Chambers JR, Sauer K. 2013. Small RNAs and their role in biofilm formation. Trends Microbiol 21:39–49. http://dx.doi.org/10.1016/j.tim.2012.10.008.

50. Stewart FJ. 2013. Where the genes flow. Nat Geosci 6:688–690. http://dx.doi.org/10.1038/ngeo1939.

51. Sun Y, Bernardy EE, Hammer BK, Miyashiro T. 2013. Competence and natural transformation in vibrios. Mol Microbiol 89:583–595. http://dx.doi.org/10.1111/mmi.12307.

52. Paerl HW, Pinckney JL. 1996. A mini-review of microbial consortia: their roles in aquatic production and biogeochemical cycling. Microb Ecol 31:225–247.

53. Azam F, Long RA. 2001. Sea snow microcosms. Nature 414:495–498. http://dx.doi.org/10.1038/35107174.

54. Dang HY, Lovell CR. 2002. Seasonal dynamics of particle-associated and free-living marine Proteobacteria in a salt marsh tidal creek as determined using fluorescence in situ hybridization. Environ Microbiol 4:287–295. http://dx.doi.org/10.1046/j.1462-2920.2002.00295.x.

55. Newman DK, Banfield JF. 2002. Geomicrobiology: how molecular-scale interactions underpin biogeochemical systems. Science 296:1071–1077. http://dx.doi.org/10.1126/science.1010716.

56. Steinberg PD, De Nys R, Kjelleberg S. 2002. Chemical cues for surface colonization. J Chem Ecol 28:1935–1951. http://dx.doi.org/10.1023/A:1020789625989.

57. Edwards SJ, Kjellerup BV. 2013. Applications of biofilms in bioremediation and biotransformation of persistent organic pollutants, pharmaceuticals/personal care products, and heavy metals. Appl Microbiol Biotechnol 97:9909–9921. http://dx.doi.org/10.1007/s00253-013-5216-z.

58. Peterson BW, He Y, Ren Y, Zerdoum A, Libera MR, Sharma PK, van Winkelhoff AJ, Neut D, Stoodley P, van der Mei HC, Busscher HJ. 2015. Viscoelasticity of biofilms and their recalcitrance to mechanical and chemical challenges. FEMS Microbiol Rev 39:234–245. http://dx.doi.org/10.1093/femsre/fuu008.

59. Steinberg N, Kolodkin-Gal I. 2015. The matrix reloaded: how sensing the extracellular matrix synchronizes bacterial communities. J Bacteriol 197:2092–2103. http://dx.doi.org/10.1128/JB.02516-14.

60. Karatan E, Watnick P. 2009. Signals, regulatory networks, and materials that build and break bacterial biofilms. Microbiol Mol Biol Rev 73:310–347. http://dx.doi.org/10.1128/MMBR.00041-08.
61. Elias S, Banin E. 2012. Multi-species biofilms: living with friendly neighbors. FEMS Microbiol Rev 36:990–1004. http://dx.doi.org/10.1111/j.1574-6976.2012.00325.x.
62. Rendueles O, Ghigo JM. 2012. Multi-species biofilms: how to avoid unfriendly neighbors. FEMS Microbiol Rev 36:972–989. http://dx.doi.org/10.1111/j.1574-6976.2012.00328.x.
63. Weinbauer MG, Bettarel Y, Cattaneo R, Luef B, Maier C, Motegi C, Peduzzi P, Mari X. 2009. Viral ecology of organic and inorganic particles in aquatic systems: avenues for further research. Aquat Microb Ecol 57:321–341. http://dx.doi.org/10.3354/ame01363.
64. Riemann L, Grossart HP. 2008. Elevated lytic phage production as a consequence of particle colonization by a marine Flavobacterium (Cellulophaga sp.). Microb Ecol 56:505–512. http://dx.doi.org/10.1007/s00248-008-9369-8.
65. Ghosh D, Roy K, Williamson KE, Srinivasiah S, Wommack KE, Radosevich M. 2009. Acyl-homoserine lactones can induce virus production in lysogenic bacteria: an alternative paradigm for prophage induction. Appl Environ Microbiol 75:7142–7152. http://dx.doi.org/10.1128/AEM.00950-09.
66. Tan D, Dahl A, Middelboe M. 2015. Vibriophages differentially influence biofilm formation by Vibrio anguillarum strains. Appl Environ Microbiol 81:4489–4497. http://dx.doi.org/10.1128/AEM.00518-15.
67. Hao L, Li J, Kappler A, Obst M. 2013. Mapping of heavy metal ion sorption to cell-extracellular polymeric substance-mineral aggregates by using metal-selective fluorescent probes and confocal laser scanning microscopy. Appl Environ Microbiol 79:6524–6534. http://dx.doi.org/10.1128/AEM.02454-13.
68. Oppenheimer-Shaanan Y, Steinberg N, Kolodkin-Gal I. 2013. Small molecules are natural triggers for the disassembly of biofilms. Trends Microbiol 21:594–601. http://dx.doi.org/10.1016/j.tim.2013.08.005.
69. Beech IB, Sunner J. 2004. Biocorrosion: towards understanding interactions between biofilms and metals. Curr Opin Biotechnol 15:181–186. http://dx.doi.org/10.1016/j.copbio.2004.05.001.
70. Cottingham KL, Chiavelli DA, Taylor RK. 2003. Environmental microbe and human pathogen: the ecology and microbiology of Vibrio cholerae. Front Ecol Environ 1:80–86. http://dx.doi.org/10.1890/1540-9295(2003)001[0080:EMAHP]2.0.CO;2.
71. Shi XM, Xie N, Gong J. 2011. Recent progress in the research on microbially influenced corrosion: a bird’s eye view through the engineering lens. Recent Pat Corros Sci 1:118–131. http://dx.doi.org/10.2174/2210683911101020118.
72. Enning D, Garrelfs J. 2014. Corrosion of iron by sulfate-reducing bacteria: new views of an old problem. Appl Environ Microbiol 80:1226–1236. http://dx.doi.org/10.1128/AEM.02848-13.
73. International Navigation Association. 2005. Accelerated low water corrosion—report of Working Group 44 of the Maritime Navigation Commission. PIANC, Brussels, Belgium.
74. Cross T. 2006. Accelerated low water corrosion—a universal phenomenon. Corros Manage 69:18–19.
75. Melchers RE, Jeffrey RJ. 2013. Accelerated low water corrosion of steel piling in harbours. Corros Eng Sci Technol 48:496–505. http://dx.doi.org/10.1179/1743278213Y.0000000103.
76. Beech IB, Campbell SA. 2008. Accelerated low water corrosion of carbon steel in the presence of a biofilm harbouring sulphate-reducing and sulphur-oxidising bacteria recovered from a marine sediment. Electrochim Acta 54:14–21. http://dx.doi.org/10.1016/j.electacta.2008.05.084.
77. Sand W, Gehrke T. 2003. Microbiologically influenced corrosion of steel in aqueous environments. Rev Environ Sci Biotechnol 2:169–176. http://dx.doi.org/10.1023/B:RESB.00000040468.88570.4e.
78. Emerson D, Fleming EJ, McBeth JM. 2010. Iron-oxidizing bacteria: an environmental and genomic perspective. Annu Rev Microbiol 64:561–583. http://dx.doi.org/10.1146/annurev.micro.112408.134208.
79. Barco RA, Emerson D, Sylvan JB, Orcutt BN, Jacobson Meyers ME, Ramírez GA, Zhong JD, Edwards KJ. 2015. New insight into microbial iron oxidation as revealed by the proteomic profile of an obligate iron-oxidizing chemolithoautotroph. Appl Environ Microbiol 81:5927–5937. http://dx.doi.org/10.1128/AEM.01374-15.
80. Emerson JB, Thomas BC, Alvarez W, Banfield JF. 28 February 2015. Metagenomic analysis of a high CO₂ subsurface microbial community populated by chemolithoautotrophs and bacteria and archaea from candidate phyla. Environ Microbiol http://dx.doi.org/10.1111/1462-2920.12817.
81. McBeth JM, Little BJ, Ray RI, Farrar KM, Emerson D. 2011. Neutrophilic iron-oxidizing “Zetaproteobacteria” and mild steel corrosion in nearshore marine environments. Appl Environ Microbiol 77:1405–1412. http://dx.doi.org/10.1128/AEM.02095-10.
82. Lee JS, McBeth JM, Ray RI, Little BJ, Emerson D. 2013. Iron cycling at corroding carbon steel surfaces. Biofouling 29:1243–1252. http://dx.doi.org/10.1080/08927014.2013.836184.
83. Melchers RE. 2013. Influence of dissolved inorganic nitrogen on accelerated low water corrosion of marine steel piling. Corrosion 69:95–103. http://dx.doi.org/10.5006/0728.
84. Melchers RE. 2014. Long-term immersion corrosion of steels in seawaters with elevated nutrient concentration. Corros Sci 81:110–116. http://dx.doi.org/10.1016/j.corsci.2013.12.009.
85. Field EK, Sczyrba A, Lyman AE, Harris CC, Woyke T, Stepanauskas R, Emerson D. 2015. Genomic insights into the uncultivated marine Zetaproteobacteria at Loihi Seamount. ISME J 9:857–870. http://dx.doi.org/10.1038/ismej.2014.183.
86. Myhr S, Lillebø BL, Sunde E, Beeder J, Torsvik T. 2002. Inhibition of microbial H₂S production in an oil reservoir model column by nitrate injection. Appl Microbiol Biotechnol 58:400–408. http://dx.doi.org/10.1007/s00253-001-0881-8.
87. Hubert C, Voordouw G. 2007. Oil field souring control by nitrate-reducing Sulfurospirillum spp. that outcompete sulfate-reducing bacteria for organic electron donors. Appl Environ Microbiol 73:2644–2652. http://dx.doi.org/10.1128/AEM.02332-06.
88. Gittel A, Sørensen KB, Skovhus TL, Ingvarsen K, Schramm A. 2009. Prokaryotic community structure and sulfate reducer activity in water from high-temperature oil reservoirs with and without nitrate treatment. Appl Environ Microbiol 75:7086–7096. http://dx.doi.org/10.1128/AEM.01123-09.
89. Lysnes K, Bødtker G, Torsvik T, Bjørnestad EO, Sunde E. 2009. Microbial response to reinjection of produced water in an oil reservoir. Appl Microbiol Biotechnol 83:1143–1157. http://dx.doi.org/10.1007/s00253-009-2015-7.
90. Voordouw G. 2011. Production-related petroleum microbiology: progress and prospects. Curr Opin Biotechnol 22:401–405. http://dx.doi.org/10.1016/j.copbio.2010.12.005.
91. Carlson HK, Kuehl JV, Hazra AB, Justice NB, Stoeva MK, Sczesnak A, Mullan MR, Iavarone AT, Engelbrektson A, Price MN, Deutschbauer AM, Arkin AP, Coates JD. 2015. Mechanisms of direct inhibition of the respiratory sulfate-reduction pathway by (per)chlorate and nitrate. ISME J 9:1295–1305. http://dx.doi.org/10.1038/ismej.2014.216.
92. Nemati M, Jenneman GE, Voordouw G. 2001. Impact of nitrate-mediated microbial control of souring in oil reservoirs on the extent of corrosion. Biotechnol Prog 17:852–859. http://dx.doi.org/10.1021/bp010084v.
93. Hubert C, Nemati M, Jenneman G, Voordouw G. 2005. Corrosion risk associated with microbial souring control using nitrate or nitrite. Appl Microbiol Biotechnol 68:272–282. http://dx.doi.org/10.1007/s00253-005-1897-2.
94. Rempel CL, Evitts RW, Nemati M. 2006. Dynamics of corrosion rates associated with nitrite or nitrate mediated control of souring under biological conditions simulating an oil reservoir. J Ind Microbiol Biotechnol 33:878–886. http://dx.doi.org/10.1007/s10295-006-0142-z.
95. Schippers A, Jørgensen BB. 2002. Biogeochemistry of pyrite and iron sulfide oxidation in marine sediments. Geochim Cosmochim Acta 66:85–92. http://dx.doi.org/10.1016/S0016-7037(01)00745-1.
96. Edwards KJ, Rogers DR, Wirsén CO, McCollom TM. 2003. Isolation and characterization of novel psychrophilic, neutrophilic, Fe-oxidizing, chemolithoautotrophic α- and γ-Proteobacteria from the deep sea. Appl Environ Microbiol 69:2906–2913. http://dx.doi.org/10.1128/AEM.69.5.2906-2913.2003.
97. Singer E, Webb EA, Nelson WC, Heidelberg JF, Ivanova N, Pati A, Edwards KJ. 2011. Genomic potential of Marinobacter aquaeolei, a biogeochemical “opportunitroph.” Appl Environ Microbiol 77:2763–2771. http://dx.doi.org/10.1128/AEM.01866-10.
98. Bosch J, Lee KY, Jordan G, Kim KW, Meckenstock RU. 2012. Anaerobic, nitrate-dependent oxidation of pyrite nanoparticles by Thiobacillus denitrificans. Environ Sci Technol 46:2095–2101. http://dx.doi.org/10.1021/es2022329.
99. Ilbert M, Bonnefoy V. 2013. Insight into the evolution of the iron

oxidation pathways. Biochim Biophys Acta 1827:161–175. http://dx.doi.org/10.1016/j.bbabio.2012.10.001.

100. Chen YC, Lee CM, Yen SK, Chyou SD. 2007. The effect of denitrifying Fe-oxidizing bacteria TPH-7 on corrosion inhibition of sodium molybdate. Corros Sci 49:3917–3925. http://dx.doi.org/10.1016/j.corsci.2007.03.042.

101. Iino T, Ito K, Wakai S, Tsurumaru H, Ohkuma M, Harayama S. 2015. Iron corrosion induced by nonhydrogenotrophic nitrate-reducing *Prolixibacter* sp. strain MIC1-1. Appl Environ Microbiol 81:1839–1846. http://dx.doi.org/10.1128/AEM.03741-14.

102. Alldredge AL, Silver MW. 1988. Characteristics, dynamics and significance of marine snow. Prog Oceanogr 20:41–82. http://dx.doi.org/10.1016/0079-6611(88)90053-5.

103. Stocker R, Seymour JR, Samadani A, Hunt DE, Polz MF. 2008. Rapid chemotactic response enables marine bacteria to exploit ephemeral microscale nutrient patches. Proc Natl Acad Sci U S A 105:4209–4214. http://dx.doi.org/10.1073/pnas.0709765105.

104. Arístegui J, Gasol JM, Duarte VM, Herndl GJ. 2009. Microbial oceanography of the dark ocean’s pelagic realm. Limnol Oceanogr 54:1501–1529. http://dx.doi.org/10.4319/lo.2009.54.5.1501.

105. Grossart HP. 2010. Ecological consequences of bacterioplankton lifestyles: changes in concepts are needed. Environ Microbiol Rep 2:706–714. http://dx.doi.org/10.1111/j.1758-2229.2010.00179.x.

106. Caron DA, Davis PG, Madin LP, Sieburth JM. 1982. Heterotrophic bacteria and bacterivorous protozoa in oceanic macroaggregates. Science 218:795–797. http://dx.doi.org/10.1126/science.218.4574.795.

107. Azam F, Smith DC, Steward GF, Hagström Å. 1994. Bacteria-organic matter coupling and its significance for oceanic carbon cycling. Microb Ecol 28:167–179. http://dx.doi.org/10.1007/BF00166806.

108. DeLong EF, Franks DG, Alldredge AL. 1993. Phylogenetic diversity of aggregate-attached vs. free-living marine bacterial assemblages. Limnol Oceanogr 38:924–934. http://dx.doi.org/10.4319/lo.1993.38.5.0924.

109. Riemann L, Steward GF, Azam F. 2000. Dynamics of bacterial community composition and activity during a mesocosm diatom bloom. Appl Environ Microbiol 66:578–587. http://dx.doi.org/10.1128/AEM.66.2.578-587.2000.

110. Fandino LB, Riemann L, Steward GF, Long RA, Azam F. 2001. Variations in bacterial community structure during a dinoflagellate bloom analyzed by DGGE and 16S rDNA sequencing. Aquat Microb Ecol 23:119–130. http://dx.doi.org/10.3354/ame023119.

111. Simon M, Grossart HP, Schweitzer B, Ploug H. 2002. Microbial ecology of organic aggregates in aquatic ecosystems. Aquat Microb Ecol 28:175–211. http://dx.doi.org/10.3354/ame028175.

112. Lapoussière A, Michel C, Starr M, Gosselin M, Poulin M. 2011. Role of free-living and particle-attached bacteria in the recycling and export of organic material in the Hudson Bay system. J Mar Syst 88:434–445. http://dx.doi.org/10.1016/j.jmarsys.2010.12.003.

113. Ploug H, Grossart HP, Azam F, Jørgensen BB. 1999. Photosynthesis, respiration, and carbon turnover in sinking marine snow from surface waters of Southern California Bight: implications for the carbon cycle in the ocean. Mar Ecol Prog Ser 179:1–11. http://dx.doi.org/10.3354/meps179001.

114. Ghiglione JF, Mevel G, Pujo-Pay M, Moussseau L, Lebaron P, Goutx M. 2007. Diel and seasonal variations in abundance, activity, and community structure of particle-attached and free-living bacteria in NW Mediterranean Sea. Microb Ecol 54:217–231. http://dx.doi.org/10.1007/s00248-006-9189-7.

115. Mével G, Vernet M, Goutx M, Ghiglione JF. 2008. Seasonal to hour variation scales in abundance and production of total and particle-attached bacteria in the open NW Mediterranean Sea (0–1000 m). Bio- geosciences 5:1573–1586. http://dx.doi.org/10.5194/bg-5-1573-2008.

116. Kirchman D, Mitchell R. 1982. Contribution of particle-bound bacteria to total microheterotrophic activity in five ponds and two marshes. Appl Environ Microbiol 43:200–209.

117. Karner M, Herndl G. 1992. Extracellular enzymatic activity and secondary production in free-living and marine-snow-associated bacteria. Mar Biol 113:341–347.

118. Murrell MC, Hollibaugh JT, Silver MW, Wong PS. 1999. Bacterioplankton dynamics in northern San Francisco Bay: role of particle association and seasonal freshwater flow. Limnol Oceanogr 44:295–308. http://dx.doi.org/10.4319/lo.1999.44.2.0295.

119. Polz MF, Hunt DE, Preheim SP, Weinreich DM. 2006. Patterns and mechanisms of genetic and phenotypic differentiation in marine mi-

crobes. Philos Trans R Soc Lond B Biol Sci 361:2009–2021. http://dx.doi.org/10.1098/rstb.2006.1928.

120. Lauro FM, McDougald D, Thomas T, Williams TJ, Egan S, Rice S, DeMaere MZ, Ting L, Ertan H, Johnson J, Ferriera S, Lapidus A, Anderson I, Kyrpides N, Munk AC, Detter C, Han CS, Brown MV, Robb FT, Kjelleberg S, Cavicchioli R. 2009. The genomic basis of trophic strategy in marine bacteria. Proc Natl Acad Sci U S A 106:15527–15533. http://dx.doi.org/10.1073/pnas.0903507106.

121. Yooseph S, Nealson KH, Rusch DB, McCrow JP, Dupont CL, Kim M, Johnson J, Montgomery R, Ferriera S, Beeson K, Williamson SJ, Tovchigrechko A, Allen AE, Zeigler LA, Sutton G, Eisenstadt E, Rogers YH, Friedman R, Frazier M, Venter JC. 2010. Genomic and functional adaptation in surface ocean planktonic prokaryotes. Nature 468:60–66. http://dx.doi.org/10.1038/nature09530.

122. Noble PA, Biddle KD, Fletcher M. 1997. Natural microbial community compositions compared by a back-propagating neural network and cluster analysis of 5S rRNA. Appl Environ Microbiol 63:1762–1770.

123. Hollibaugh JT, Wong PS, Murrell MC. 2000. Similarity of particle-associated and free-living bacterial communities in northern San Francisco Bay, California. Aquat Microb Ecol 21:103–114. http://dx.doi.org/10.3354/ame021103.

124. Ortega-Retuerta E, Joux F, Jeffrey WH, Ghiglione JF. 2013. Spatial variability of particle-attached and free-living bacterial diversity in surface waters from the Mackenzie River to the Beaufort Sea (Canadian Arctic). Biogeosciences 10:2747–2759. http://dx.doi.org/10.5194/bg-10-2747-2013.

125. Marshall KC, Stout R, Mitchell R. 1971. Mechanism of the initial events in the sorption of marine bacteria to surfaces. J Gen Microbiol 68:337–348. http://dx.doi.org/10.1099/00221287-68-3-337.

126. Pepe-Ranney C, Hall E. 2015. The effect of carbon subsidies on marine planktonic niche partitioning and recruitment during biofilm assembly. Front Microbiol 6:703. http://dx.doi.org/10.3389/fmicb.2015.00703.

127. Acinas SG, Antón J, Rodríguez-Valera F. 1999. Diversity of free-living and attached bacteria in offshore Western Mediterranean waters as depicted by analysis of genes encoding 16S rRNA. Appl Environ Microbiol 65:514–522.

128. Moeseneder MM, Winter C, Herndl GJ. 2001. Horizontal and vertical complexity of attached and free-living bacteria of the eastern Mediterranean Sea, determined by 16S rDNA and 16S rRNA fingerprints. Limnol Oceanogr 46:95–107. http://dx.doi.org/10.4319/lo.2001.46.1.0095.

129. Ghiglione JF, Conan P, Pujo-Pay M. 2009. Diversity of total and active free-living vs. particle-attached bacteria in the euphotic zone of the NW Mediterranean Sea. FEMS Microbiol Lett 299:9–21. http://dx.doi.org/10.1111/j.1574-6968.2009.01694.x.

130. Crump BC, Armbrust EV, Baross JA. 1999. Phylogenetic analysis of particle-attached and free-living bacterial communities in the Columbia River, its estuary, and the adjacent coastal ocean. Appl Environ Microbiol 65:3192–3204.

131. Kellogg CTE, Deming JW. 2009. Comparison of free-living, suspended particle, and aggregate-associated bacterial and archaeal communities in the Laptev Sea. Aquat Microb Ecol 57:1–18. http://dx.doi.org/10.3354/ame01317.

132. Eloe EA, Shulse CN, Fadrosh DW, Williamson SJ, Allen EE, Bartlett DH. 2011. Compositional differences in particle-associated and freeliving microbial assemblages from an extreme deep-ocean environment. Environ Microbiol Rep 3:449–458. http://dx.doi.org/10.1111/j.1758-2229.2010.00223.x.

133. Fuchsman CA, Kirkpatrick JB, Brazelton WJ, Murray JW, Staley JT. 2011. Metabolic strategies of free-living and aggregate-associated bacterial communities inferred from biologic and chemical profiles in the Black Sea suboxic zone. FEMS Microbiol Ecol 78:586–603. http://dx.doi.org/10.1111/j.1574-6941.2011.01189.x.

134. Crespo BG, Pommier T, Fernández-Gómez B, Pedrós-Alió C. 2013. Taxonomic composition of the particle-attached and free-living bacterial assemblages in the Northwest Mediterranean Sea analyzed by pyrosequencing of the 16S rRNA. Microbiologyopen 2:541–552. http://dx.doi.org/10.1002/mbo3.92.

135. Smith MW, Zeigler Allen L, Allen AE, Herfort L, Simon HM. 2013. Contrasting genomic properties of free-living and particle-attached microbial assemblages within a coastal ecosystem. Front Microbiol 4:120. http://dx.doi.org/10.3389/fmicb.2013.00120.

136. LeCleir GR, DeBruyn JM, Maas EW, Boyd PW, Wilhelm SW. 2014. Temporal changes in particle-associated microbial communities after

interception by nonlethal sediment traps. FEMS Microbiol Ecol 87:153–
163. http://dx.doi.org/10.1111/1574-6941.12213.
137. Fontanez KM, Eppley JM, Samo TJ, Karl DM, DeLong EF. 2015. Microbial community structure and function on sinking particles in the North Pacific Subtropical Gyre. Front Microbiol 6:469. http://dx.doi.org/10.3389/fmicb.2015.00469.
138. Orsi WD, Smith JM, Wilcox HM, Swalwell JE, Carini P, Worden AZ, Santoro AE. 2015. Ecophysiology of uncultivated marine euryarchaea is linked to particulate organic matter. ISME J 9:1747–1763. http://dx.doi.org/10.1038/ismej.2014.260.
139. Shanks AL, Trent JD. 1979. Marine snow: microscale nutrient patches. Limnol Oceanogr 24:850–854. http://dx.doi.org/10.4319/lo.1979.24.5.0850.
140. Alldredge AL, Cohen Y. 1987. Can microscale chemical patches persist in the sea? Microelectrode study of marine snow, fecal pellets. Science 235:689–691.
141. Shanks AL, Reeder ML. 1993. Reducing microzones and sulfide production in marine snow. Mar Ecol Prog Ser 96:43–47. http://dx.doi.org/10.3354/meps096043.
142. Karl DM, Knauer GA, Martin JH, Ward BB. 1984. Bacterial chemolithotrophy in the ocean is associated with sinking particles. Nature 309:54–56. http://dx.doi.org/10.1038/309054a0.
143. Bianchi M, Marty D, Teyssié JL, Fowler SW. 1992. Strictly aerobic and anaerobic bacteria associated with sinking particulate matter and zooplankton fecal pellets. Mar Ecol Prog Ser 88:55–60. http://dx.doi.org/10.3354/meps088055.
144. Kaltenböck E, Herndl GJ. 1992. Ecology of amorphous aggregations (marine snow) in the Northern Adriatic Sea. IV. Dissolved nutrients and the autotrophic community associated with marine snow. Mar Ecol Prog Ser 87:147–159.
145. Orcutt BN, Sylvan JB, Knab NJ, Edwards KJ. 2011. Microbial ecology of the dark ocean above, at, and below the seafloor. Microbiol Mol Biol Rev 75:361–422. http://dx.doi.org/10.1128/MMBR.00039-10.
146. Swan BK, Martinez-Garcia M, Preston CM, Sczyrba A, Woyke T, Lamy D, Reinthaler T, Poulton NJ, Masland ED, Gomez ML, Sieracki ME, DeLong EF, Herndl GJ, Stepanauskas R. 2011. Potential for chemolithoautotrophy among ubiquitous bacteria lineages in the dark ocean. Science 333:1296–1300. http://dx.doi.org/10.1126/science.1203690.
147. Wright JJ, Konwar KM, Hallam SJ. 2012. Microbial ecology of expanding oxygen minimum zones. Nat Rev Microbiol 10:381–394. http://dx.doi.org/10.1038/nrmicro2778.
148. Sheik CS, Jain S, Dick GJ. 2014. Metabolic flexibility of enigmatic SAR324 revealed through metagenomics and metatranscriptomics. Environ Microbiol 16:304–317. http://dx.doi.org/10.1111/1462-2920.12165.
149. Waidner LA, Kirchman DL. 2007. Aerobic anoxygenic phototrophic bacteria attached to particles in turbid waters of the Delaware and Chesapeake estuaries. Appl Environ Microbiol 73:3936–3944. http://dx.doi.org/10.1128/AEM.00592-07.
150. Cottrell MT, Ras J, Kirchman DL. 2010. Bacteriochlorophyll and community structure of aerobic anoxygenic phototrophic bacteria in a particle-rich estuary. ISME J 4:945–954. http://dx.doi.org/10.1038/ismej.2010.13.
151. Simon HM, Smith MW, Herfort L. 2014. Metagenomic insights into particles and their associated microbiota in a coastal margin ecosystem. Front Microbiol 5:466. http://dx.doi.org/10.3389/fmicb.2014.00466.
152. Karl DM, Tilbrook BD. 1994. Production and transport of methane in oceanic particulate organic matter. Nature 368:732–734. http://dx.doi.org/10.1038/368732a0.
153. Satinsky BM, Crump BC, Smith CB, Sharma S, Zielinski BL, Doherty M, Meng J, Sun S, Medeiros PM, Paul JH, Coles VJ, Yager PL, Moran MA. 2014. Microspatial gene expression patterns in the Amazon River Plume. Proc Natl Acad Sci U S A 111:11085–11090. http://dx.doi.org/10.1073/pnas.1402782111.
154. Paerl HW, Prufert LE. 1987. Oxygen-poor microzones as potential sites of microbial N₂ fixation in nitrogen-depleted aerobic marine waters. Appl Environ Microbiol 53:1078–1087.
155. Rahav E, Bar-Zeev E, Ohayon S, Elifantz H, Belkin N, Herut B, Mulholland MR, Berman-Frank I. 2013. Dinitrogen fixation in aphotic oxygenated marine environments. Front Microbiol 4:227. http://dx.doi.org/10.3389/fmicb.2013.00227.
156. Phillips CJ, Smith Z, Embley TM, Prosser JI. 1999. Phylogenetic differences between particle-associated and planktonic ammonia-
oxidizing bacteria of the β subdivision of the class Proteobacteria in the northwestern Mediterranean Sea. Appl Environ Microbiol 65:779–786.
157. Füssel J, Lam P, Lavik G, Jensen MM, Holtappels M, Günter M, Kuypers MM. 2012. Nitrite oxidation in the Namibian oxygen minimum zone. ISME J 6:1200–1209. http://dx.doi.org/10.1038/ismej.2011.178.
158. Klawonn I, Bonaglia S, Brüchert V, Ploug H. 2015. Aerobic and anaerobic nitrogen transformation processes in N₂-fixing cyanobacterial aggregates. ISME J 9:1456–1466. http://dx.doi.org/10.1038/ismej.2014.232.
159. Michotey V, Bonin P. 1997. Evidence for anaerobic bacterial processes in the water column: denitrification and dissimilatory nitrate ammonification in the northwestern Mediterranean Sea. Mar Ecol Prog Ser 160:47–56. http://dx.doi.org/10.3354/meps160047.
160. Wolgast DM, Carlucci AF, Bauer JE. 1998. Nitrate respiration associated with detrital aggregates in aerobic bottom waters of the abyssal NE Pacific. Deep Sea Res II 45:881–892. http://dx.doi.org/10.1016/S0967-0645(98)00006-X.
161. Ganesh S, Parris DJ, Delong EF, Stewart FJ. 2014. Metagenomic analysis of size-fractionated picoplankton in a marine oxygen minimum zone. ISME J 8:187–211. http://dx.doi.org/10.1038/ismej.2013.144.
162. Kuypers MM, Lavik G, Woebken D, Schmid M, Fuchs BM, Amann R, Jørgensen BB, Jetten MS. 2005. Massive nitrogen loss from the Benguela upwelling system through anaerobic ammonium oxidation. Proc Natl Acad Sci U S A 102:6478–6483. http://dx.doi.org/10.1073/pnas.0502088102.
163. Woebken D, Fuchs BM, Kuypers MM, Amann R. 2007. Potential interactions of particle-associated anammox bacteria with bacterial and archaeal partners in the Namibian upwelling system. Appl Environ Microbiol 73:4648–4657. http://dx.doi.org/10.1128/AEM.02774-06.
164. Meibom KL, Li XB, Nielsen AT, Wu CY, Roseman S, Schoolnik GK. 2004. The Vibrio cholerae chitin utilization program. Proc Natl Acad Sci U S A 101:2524–2529. http://dx.doi.org/10.1073/pnas.0308707101.
165. Karl DM. 2007. Microbial oceanography: paradigms, processes and promise. Nat Rev Microbiol 5:759–769. http://dx.doi.org/10.1038/nrmicro1749.
166. Pruzzo C, Vezzulli L, Colwell RR. 2008. Global impact of Vibrio cholerae interactions with chitin. Environ Microbiol 10:1400–1410. http://dx.doi.org/10.1111/j.1462-2920.2007.01559.x.
167. Ganesh S, Bristow LA, Larsen M, Sarode N, Thamdrup B, Stewart FJ. 7 April 2015. Size-fraction partitioning of community gene transcription and nitrogen metabolism in a marine oxygen minimum zone. ISME J http://dx.doi.org/10.1038/ismej.2015.44.
168. Dell’Anno A, Corinaldesi C, Stavrakakis S, Lykousis V, Danovaro R. 2005. Pelagic-benthic coupling and diagenesis of nucleic acids in a deep-sea continental margin and an open-slope system of the Eastern Mediterranean. Appl Environ Microbiol 71:6070–6076. http://dx.doi.org/10.1128/AEM.71.10.6070-6076.2005.
169. Das T, Sehar S, Manefield M. 2013. The roles of extracellular DNA in the structural integrity of extracellular polymeric substance and bacterial biofilm development. Environ Microbiol Rep 5:778–786. http://dx.doi.org/10.1111/1758-2229.12085.
170. Zoppini A, Puddu A, Fazi S, Rosati M, Sist P. 2005. Extracellular enzyme activity and dynamics of bacterial community in mucilaginous aggregates of the northern Adriatic Sea. Sci Total Environ 353:270–286. http://dx.doi.org/10.1016/j.scitotenv.2005.09.019.
171. Malfatti F, Turk V, Tinta T, Mozetič P, Manganelli M, Samo TJ, Ugalde JA, Kovač N, Stefanelli M, Antonioli M, Fonda-Umani S, Del Negro P, Cataletto B, Hozić A, Ivošević DeNardis N, Žutić V, Svetličič V, Mišić Radić T, Radić T, Fuks D, Azam F. 2014. Microbial mechanisms coupling carbon and phosphorus cycles in phosphorus-limited northern Adriatic Sea. Sci Total Environ 470–471:1173–1183. http://dx.doi.org/10.1016/j.scitotenv.2013.10.040.
172. Vojvoda J, Lamy D, Sintes E, Garcia JAL, Turk V, Herndl GJ. 2014. Seasonal variation in marine-snow-associated and ambient-water prokaryotic communities in the northern Adriatic Sea. Aquat Microb Ecol 73:211–224. http://dx.doi.org/10.3354/ame01718.
173. Wagner-Döbler I, Biebl H. 2006. Environmental biology of the marine Roseobacter lineage. Annu Rev Microbiol 60:255–280. http://dx.doi.org/10.1146/annurev.micro.60.080805.142115.
174. Hatton AD, Shenoy DM, Hart MC, Mogg A, Green DH. 2012. Metabolism of DMSP, DMS and DMSO by the cultivable bacterial community associated with the DMSP-producing dinoflagellate Scrippsiella trochoidea.

Dang and Lovell

175. Moran MA, Reisch CR, Kiene RP, Whitman WB. 2012. Genomic insights into bacterial DMSP transformations. Annu Rev Mar Sci 4:523–542. http://dx.doi.org/10.1146/annurev-marine-120710-100827.
176. Weber KA, Achenbach LA, Coates JD. 2006. Microorganisms pumping iron: anaerobic microbial iron oxidation and reduction. Nat Rev Microbiol 4:752–764. http://dx.doi.org/10.1038/nrmicro1490.
177. Balzano S, Statham PJ, Pancost RD, Lloyd JR. 2009. Role of microbial populations in the release of reduced iron to the water column from marine aggregates. Aquat Microb Ecol 54:291–303. http://dx.doi.org/10.3354/ame01278.
178. Hedrich S, Schlömann M, Johnson DB. 2011. The iron-oxidizing proteobacteria. Microbiology 157:1551–1564. http://dx.doi.org/10.1099/mic.0.045344-0.
179. Sandy M, Butler A. 2009. Microbial iron acquisition: marine and terrestrial siderophores. Chem Rev 109:4580–4595. http://dx.doi.org/10.1021/cr9002787.
180. Vraspir JM, Butler A. 2009. Chemistry of marine ligands and siderophores. Annu Rev Mar Sci 1:43–63. http://dx.doi.org/10.1146/annurev.marine.010908.163712.
181. Boyd PW, Ellwood MJ. 2010. The biogeochemical cycle of iron in the ocean. Nat Geosci 3:675–682. http://dx.doi.org/10.1038/ngeo964.
182. Li M, Toner BM, Baker BJ, Breier JA, Sheik CS, Dick GJ. 2014. Microbial iron uptake as a mechanism for dispersing iron from deep-sea hydrothermal vents. Nat Commun 5:3192. http://dx.doi.org/10.1038/ncomms4192.
183. Biddle KD, Azam F. 1999. Accelerated dissolution of diatom silica by marine bacterial assemblages. Nature 397:508–512. http://dx.doi.org/10.1038/17351.
184. Biddle KD, Brzezinski MA, Long RA, Jones JL, Azam F. 2003. Diminished efficiency in the oceanic silica pump caused by bacteria-mediated silica dissolution. Limnol Oceanogr 48:1855–1868. http://dx.doi.org/10.4319/lo.2003.48.5.1855.
185. Anantharaman K, Breier JA, Sheik CS, Dick GJ. 2013. Evidence for hydrogen oxidation and metabolic plasticity in widespread deep-sea sulfur-oxidizing bacteria. Proc Natl Acad Sci U S A 110:330–335. http://dx.doi.org/10.1073/pnas.1215340110.
186. Azam F, Malfatti F. 2007. Microbial structuring of marine ecosystems. Nat Rev Microbiol 5:782–791. http://dx.doi.org/10.1038/nrmicro1747.
187. Falkowski PG, Fenchel T, DeLong EF. 2008. The microbial engines that drive Earth’s biogeochemical cycles. Science 320:1034–1039. http://dx.doi.org/10.1126/science.1153213.
188. Dang HY, Zhou HX, Yang JY, Ge HM, Jiao NZ, Luan XW, Zhang CL, Klotz MG. 2013. Thaumarchaeotal signature gene distribution in sediments of the northern South China Sea: an indicator of the metabolic intersection of the marine carbon, nitrogen, and phosphorus cycles? Appl Environ Microbiol 79:2137–2147. http://dx.doi.org/10.1128/AEM.03204-12.
189. Knittel K, Boetius A. 2009. Anaerobic oxidation of methane: progress with an unknown process. Annu Rev Microbiol 63:311–334. http://dx.doi.org/10.1146/annurev.micro.61.080706.093130.
190. Offre P, Spang A, Schleper C. 2013. Archaea in biogeochemical cycles. Annu Rev Microbiol 67:437–457. http://dx.doi.org/10.1146/annurev-micro-092412-155614.
191. Orell A, Fröls S, Albers SV. 2013. Archaeal biofilms: the great unexplored. Annu Rev Microbiol 67:337–354. http://dx.doi.org/10.1146/annurev-micro-092412-155616.
192. Alldredge AL, Cole JJ, Caron DA. 1986. Production of heterotrophic bacteria inhabiting macroscopic organic aggregates (marine snow) from surface waters. Limnol Oceanogr 31:68–78. http://dx.doi.org/10.4319/lo.1986.31.1.0068.
193. Arnosti C. 2011. Microbial extracellular enzymes and the marine carbon cycle. Annu Rev Mar Sci 3:401–425. http://dx.doi.org/10.1146/annurev-marine-120709-142731.
194. Williams TJ, Wilkins D, Long E, Evans F, Demaere MZ, Raftery MJ, Cavicchioli R. 2013. The role of planktonic Flavobacteria in processing algal organic matter in coastal East Antarctica revealed using metagenomics and metaproteomics. Environ Microbiol 15:1302–1317. http://dx.doi.org/10.1111/1462-2920.12017.
195. Baltar F, Arístegui J, Sintes E, Gasol JM, Reinthaler T, Herndl GJ. 2010. Significance of non-sinking particulate organic carbon and dark CO₂ fixation to heterotrophic carbon demand in the mesopelagic north-east Atlantic. Geophys Res Lett 37:L09602.
196. Cho BC, Azam F. 1988. Major role of bacteria in biogeochemical fluxes in the ocean’s interior. Nature 332:441–443. http://dx.doi.org/10.1038/332441a0.
197. Nagata T, Fukuda H, Fukuda R, Koike I. 2000. Bacterioplankton distribution and production in deep Pacific waters: large-scale geographic variations and possible coupling with sinking particle fluxes. Limnol Oceanogr 45:426–435. http://dx.doi.org/10.4319/lo.2000.45.2.0426.
198. Azam F, Fenchel T, Field JG, Gray JS, Meyer-Reil LA, Thingstad F. 1983. The ecological role of water-column microbes in the sea. Mar Ecol Prog Ser 10:257–263. http://dx.doi.org/10.3354/meps010257.
199. DeLong EF, Preston CM, Mincer T, Rich V, Hallam SJ, Frigaard NU, Martinez A, Sullivan MB, Edwards R, Brito BR, Chisholm SW, Karl DM. 2006. Community genomics among stratified microbial assemblages in the ocean’s interior. Science 311:496–503. http://dx.doi.org/10.1126/science.1120250.
200. Iversen MH, Ploug H. 2013. Temperature effects on carbon-specific respiration rate and sinking velocity of diatom aggregates—potential implications for deep ocean export processes. Biogeosciences 10:4073–4085. http://dx.doi.org/10.5194/bg-10-4073-2013.
201. Tamelander T. 2013. Community composition and extracellular enzyme activity of bacteria associated with suspended and sinking particles in contrasting arctic and sub-arctic marine environments. Aquat Microb Ecol 69:211–221. http://dx.doi.org/10.3354/ame01641.
202. Volkman JK, Tanoue E. 2002. Chemical and biological studies of particulate organic matter in the ocean. J Oceanogr 58:265–279. http://dx.doi.org/10.1023/A:1015809708632.
203. Giering SLC, Sanders R, Lampitt RS, Anderson TR, Tamburini C, Boutrif M, Zubkov MV, Marsay CM, Henson SA, Saw K, Cook K, Mayor DJ. 2014. Reconciliation of the carbon budget in the ocean’s twilight zone. Nature 507:480–483. http://dx.doi.org/10.1038/nature13123.
204. Riley GA. 1970. Particulate matter in seawater. Adv Mar Biol 8:1–118.
205. Martin JH, Knauer GA, Karl DM, Broenkow WW. 1987. Vertex: carbon cycling in the Northeast Pacific. Deep Sea Res A 34:267–285. http://dx.doi.org/10.1016/0198-0149(87)90086-0.
206. Unanue M, Azúa I, Arrieta JM, Labirua-Iturburu A, Egea L, Iriberrí J. 1998. Bacterial colonization and ectoenzymatic activity in phytoplankton-derived model particles: cleavage of peptides and uptake of amino acids. Microb Ecol 35:136–146. http://dx.doi.org/10.1007/s002489900068.
207. Lam P, Jensen MM, Lavik G, McGinnis DF, Müller B, Schubert CJ, Amann R, Thamdrup B, Kuypers MM. 2007. Linking crenarchaeal and bacterial nitrification to anammox in the Black Sea. Proc Natl Acad Sci U S A 104:7104–7109. http://dx.doi.org/10.1073/pnas.0611081104.
208. Hügler M, Sievert SM. 2011. Beyond the Calvin cycle: autotrophic carbon fixation in the ocean. Annu Rev Mar Sci 3:261–289. http://dx.doi.org/10.1146/annurev-marine-120709-142712.
209. Benner R, Amon RMW. 2015. The size-reactivity continuum of major bioelements in the ocean. Annu Rev Mar Sci 7:185–205. http://dx.doi.org/10.1146/annurev-marine-010213-135126.
210. Karner MB, DeLong EF, Karl DM. 2001. Archaeal dominance in the mesopelagic zone of the Pacific Ocean. Nature 409:507–510. http://dx.doi.org/10.1038/35054051.
211. Herndl GJ, Reinthaler T, Teira E, van Aken H, Veth C, Perinthaler A, Perinthaler J. 2005. Contribution of Archaea to total prokaryotic production in the deep Atlantic Ocean. Appl Environ Microbiol 71:2303–2309. http://dx.doi.org/10.1128/AEM.71.5.2303-2309.2005.
212. Kirchman DL, Elifantz H, Dittel AI, Malmstrom RR, Cottrell MT. 2007. Standing stocks and activity of Archaea and Bacteria in the western Arctic Ocean. Limnol Oceanogr 52:495–507. http://dx.doi.org/10.4319/lo.2007.52.2.0495.
213. Könneke M, Bernhard AE, de la Torre JR, Walker CB, Waterbury JB, Stahl DA. 2005. Isolation of an autotrophic ammonia-oxidizing marine archaeon. Nature 437:543–546. http://dx.doi.org/10.1038/nature03911.
214. Wuchter C, Abbas B, Coolen MJ, Herfort L, van Bleijswijk J, Timmers P, Strous M, Teira E, Herndl GJ, Middelburg JJ, Schouten S, Sinninghe Damsté JS. 2006. Archaeal nitrification in the ocean. Proc Natl Acad Sci U S A 103:12317–12322. http://dx.doi.org/10.1073/pnas.0600756103.
215. Sintes E, Bergauer K, De Corte D, Yokokawa T, Herndl GJ. 2013. Archaeal amoA gene diversity points to distinct biogeography of ammo-

Marine Biofilm Development

216. Varela MM, van Aken HM, Sintes E, Herndl GJ. 2008. Latitudinal trends of Crenarchaeota and Bacteria in the meso- and bathypelagic water masses of the Eastern North Atlantic. Environ Microbiol 10:110–124. http://dx.doi.org/10.1111/j.1462-2920.2007.01437.x.

217. Stahl DA, de la Torre JR. 2012. Physiology and diversity of ammonia-oxidizing archaea. Annu Rev Microbiol 66:83–101. http://dx.doi.org/10.1146/annurev-micro-092611-150128.

218. Qin W, Amin SA, Martens-Habbena W, Walker CB, Urakawa H, Devol AH, Ingalls AE, Moffett JW, Armbrust EV, Stahl DA. 2014. Marine ammonia-oxidizing archaeal isolates display obligate mixotrophy and wide ecotypic variation. Proc Natl Acad Sci U S A 111:12504–12509. http://dx.doi.org/10.1073/pnas.1324115111.

219. Ingalls AE, Shah SR, Hansman RL, Aluwihare LI, Santos GM, Druffel ER, Pearson A. 2006. Quantifying archaeal community autotrophy in the mesopelagic ocean using natural radiocarbon. Proc Natl Acad Sci U S A 103:6442–6447. http://dx.doi.org/10.1073/pnas.0510157103.

220. Yakimov MM, La Cono V, Smedile F, Deluca TH, Juárez S, Ciordia S, Fernández M, Albar JP, Ferrer M, Golyshin PN, Giuliano L. 2011. Contribution of crenarchaeal autotrophic ammonia oxidizers to the dark primary production in Tyrrhenian deep waters (central Mediterranean Sea). ISME J 5:945–961. http://dx.doi.org/10.1038/ismej.2010.197.

221. Könneke M, Schubert DM, Brown PC, Hügler M, Standfest S, Schwander T, Schada von Borzyskowski L, Erb TJ, Stahl DA, Berg IA. 2014. Ammonia-oxidizing archaea use the most energy-efficient aerobic pathway for CO₂ fixation. Proc Natl Acad Sci U S A 111:8239–8244. http://dx.doi.org/10.1073/pnas.1402028111.

222. Schouten S, Hopmans EC, Sinninghe Damsté JS. 2013. The organic geochemistry of glycerol dialkyl glycerol tetraether lipids: a review. Org Geochem 54:19–63. http://dx.doi.org/10.1016/j.orggeochem.2012.09.006.

223. Baltar F, Arístegui J, Gasol JM, Yokokawa T, Herndl GJ. 2013. Bacterial versus archaeal origin of extracellular enzymatic activity in the Northeast Atlantic deep waters. Microb Ecol 65:277–288. http://dx.doi.org/10.1007/s00248-012-0126-7.

224. Gotschalk CC, Alldredge AL. 1989. Enhanced primary production and nutrient regeneration within aggregated marine diatoms. Mar Biol 103:119–129. http://dx.doi.org/10.1007/BF00391070.

225. Kalvelage T, Lavik G, Lam P, Contreras S, Arteaga L, Löscher CR, Oschlies A, Paulmier A, Stramma L, Kuypers MMM. 2013. Nitrogen cycling driven by organic matter export in the South Pacific oxygen minimum zone. Nat Geosci 6:228–234. http://dx.doi.org/10.1038/ngeo1739.

226. Eppley RW, Peterson BJ. 1979. Particulate organic matter flux and planktonic new production in the deep ocean. Nature 282:677–680. http://dx.doi.org/10.1038/282677a0.

227. Engel A, Thoms S, Riebesell U, Rochelle-Newall E, Zondervan I. 2004. Polysaccharide aggregation as a potential sink of marine dissolved organic carbon. Nature 428:929–932. http://dx.doi.org/10.1038/nature02453.

228. Boyd PW, Trull TW. 2007. Understanding the export of biogenic particles in oceanic waters: is there consensus? Prog Oceanogr 72:276–312. http://dx.doi.org/10.1016/j.pocean.2006.10.007.

229. Alldredge AL, Passow U, Logan BE. 1993. The abundance and significance of a class of large, transparent organic particles in the ocean. Deep Sea Res I 40:1131–1140. http://dx.doi.org/10.1016/0967-0637(93)90129-Q.

230. Passow U. 2002. Transparent exopolymer particles (TEP) in aquatic environments. Prog Oceanogr 55:287–333. http://dx.doi.org/10.1016/S0079-6611(02)00138-6.

231. Van Oostende N, Moerdijk-Poortvliet TC, Boschker HT, Vyverman W, Sabbe K. 2013. Release of dissolved carbohydrates by *Emiliania huxleyi* and formation of transparent exopolymer particles depend on algal life cycle and bacterial activity. Environ Microbiol 15:1514–1531. http://dx.doi.org/10.1111/j.1462-2920.2012.02873.x.

232. Vandevivere P, Kirchman DL. 1993. Attachment stimulates exopolysaccharide synthesis by a bacterium. Appl Environ Microbiol 59:3280–3286.

233. Mopper K, Degens ET. 1979. Organic carbon in the ocean: nature and cycling, p 293–316. In Bolin B, Degens ET, Kempe S, Ketner P (ed), The global carbon cycle. SCOPE 13. John Wiley & Sons, New York, NY.

234. Grossart HP, Czub G, Simon M. 2006. Algae-bacteria interactions and their effects on aggregation and organic matter flux in the sea.

Environ Microbiol 8:1074–1084. http://dx.doi.org/10.1111/j.1462-2920.2006.00999.x.

235. Gärdes A, Iversen MH, Grossart HP, Passow U, Ullrich MS. 2011. Diatom-associated bacteria are required for aggregation of *Thalassiosira weissflogii*. ISME J 5:436–445. http://dx.doi.org/10.1038/ismej.2010.145.

236. Steinberg DK, Van Mooy BAS, Buesseler KO, Boyd PW, Kobari T, Karl DM. 2008. Bacterial vs. zooplankton control of sinking particle flux in the ocean’s twilight zone. Limnol Oceanogr 53:1327–1338. http://dx.doi.org/10.4319/lo.2008.53.4.1327.

237. Ziervogel K, Arnosti C. 2008. Polysaccharide hydrolysis in aggregates and free enzyme activity in aggregate-free seawater from the northeastern Gulf of Mexico. Environ Microbiol 10:289–299. http://dx.doi.org/10.1111/j.1462-2920.2007.01451.x.

238. Mohit V, Archambault P, Toupoint N, Lovejoy C. 2014. Phylogenetic differences in attached and free-living bacterial communities in a temperate coastal lagoon during summer, revealed via high-throughput 16S rRNA gene sequencing. Appl Environ Microbiol 80:2071–2083. http://dx.doi.org/10.1128/AEM.02916-13.

239. Gram L, Grossart H-P, Schlingloff A, Kiøboe T. 2002. Possible quorum sensing in marine snow bacteria: production of acylated homoserine lactones by *Roseobacter* strains isolated from marine snow. Appl Environ Microbiol 68:4111–4116. http://dx.doi.org/10.1128/AEM.68.8.4111-4116.2002.

240. Hmelø LR, Mincer TJ, Van Mooy BAS. 2011. Possible influence of bacterial quorum sensing on the hydrolysis of sinking particulate organic carbon in marine environments. Environ Microbiol Rep 3:682–688. http://dx.doi.org/10.1111/j.1758-2229.2011.00281.x.

241. Jatt AN, Tang K, Liu J, Zhang Z, Zhang XH. 2015. Quorum sensing in marine snow and its possible influence on production of extracellular hydrolytic enzymes in marine snow bacterium *Pantoea ananatis* B9. FEMS Microbiol Ecol 91:1–13. http://dx.doi.org/10.1093/femsec/fiu030.

242. Ziervogel K, Steen AD, Arnosti C. 2010. Changes in the spectrum and rates of extracellular enzyme activities in seawater following aggregate formation. Biogeosciences 7:1007–1015. http://dx.doi.org/10.5194/bg-7-1007-2010.

243. Dang HY, Lovell CR. 2002. Numerical dominance and phyloype diversity of marine *Rhodobacter* species during early colonization of submerged surfaces in coastal marine waters as determined by 16S ribosomal DNA sequence analysis and fluorescence *in situ* hybridization. Appl Environ Microbiol 68:496–504. http://dx.doi.org/10.1128/AEM.68.2.496-504.2002.

244. García-Martínez J, Acinas SG, Massana R, Rodríguez-Valera F. 2002. Prevalence and microdiversity of *Alteromonas macleodii*-like microorganisms in different oceanic regions. Environ Microbiol 4:42–50. http://dx.doi.org/10.1046/j.1462-2920.2002.00255.x.

245. Ivars-Martínez E, Martin-Cuadrado AB, D’Auria G, Mira A, Ferriera S, Johnson J, Friedman R, Rodriguez-Valera F. 2008. Comparative genomics of two ecotypes of the marine planktonic copiotroph *Alteromonas macleodii* suggests alternative lifestyles associated with different kinds of particulate organic matter. ISME J 2:1194–1212. http://dx.doi.org/10.1038/ismej.2008.74.

246. Hollants J, Leliaert F, De Clerck O, Willems A. 2013. What we can learn from sushi: a review on seaweed-bacterial associations. FEMS Microbiol Ecol 83:1–16. http://dx.doi.org/10.1111/j.1574-6941.2012.01446.x.

247. Eilers H, Pernthaler J, Glöckner FO, Amann R. 2000. Culturability and in situ abundance of pelagic bacteria from the North Sea. Appl Environ Microbiol 66:3044–3051. http://dx.doi.org/10.1128/AEM.66.7.3044-3051.2000.

248. Allers E, Gómez-Consarnau L, Pinhassi J, Gasol JM, Šimek K, Pernthaler J. 2007. Response of *Alteromonadaceae* and *Rhodobacteriaceae* to glucose and phosphorus manipulation in marine mesocosms. Environ Microbiol 9:2417–2429. http://dx.doi.org/10.1111/j.1462-2920.2007.01360.x.

249. Nelson CE, Wear EK. 2014. Microbial diversity and the lability of dissolved organic carbon. Proc Natl Acad Sci U S A 111:7166–7167. http://dx.doi.org/10.1073/pnas.1405751111.

250. Gutierrez T, Singleton DR, Berry D, Yang T, Aitken MD, Teske A. 2013. Hydrocarbon-degrading bacteria enriched by the Deepwater Horizon oil spill identified by cultivation and DNA-SIP. ISME J 7:2091–2104. http://dx.doi.org/10.1038/ismej.2013.98.

251. Hunt DE, Lin Y, Church MJ, Karl DM, Tringe SG, Izzo LK, Johnson ZI. 2013. Relationship between abundance and specific activity of bacteria.

rioplankton in open ocean surface waters. Appl Environ Microbiol 79: 177–184. http://dx.doi.org/10.1128/AEM.02155-12.

252. Mayali X, Weber PK, Mabery S, Pett-Ridge J. 2014. Phylogenetic patterns in the microbial response to resource availability: amino acid incorporation in San Francisco Bay. PLoS One 9:e95842. http://dx.doi.org/10.1371/journal.pone.0095842.

253. Wong TY, Preston LA, Schiller NL. 2000. Alginate lyase: review of major sources and enzyme characteristics, structure-function analysis, biological roles, and applications. Annu Rev Microbiol 54:289–340. http://dx.doi.org/10.1146/annurev.micro.54.1.289.

254. Orikoshi H, Nakayama S, Miyamoto K, Hanato C, Yasuda M, Inamori Y, Tsujibo H. 2005. Roles of four chitinases (ChiA, ChiB, ChiC, and ChiD) in the chitin degradation system of marine bacterium Alteromonas sp. strain O-7. Appl Environ Microbiol 71:1811–1815. http://dx.doi.org/10.1128/AEM.71.4.1811-1815.2005.

255. Michel G, Nyval-Colen P, Barbeyron T, Czjzek M, Helbert W. 2006. Bioconversion of red seaweed galactans: a focus on bacterial agarases and carrageenases. Appl Microbiol Biotechnol 71:23–33. http://dx.doi.org/10.1007/s00253-006-0377-7.

256. Flament D, Barbeyron T, Jam M, Potin P, Czjzek M, Kloareg B, Michel G. 2007. Alpha-agarases define a new family of glycoside hydrolases, distinct from beta-agarase families. Appl Environ Microbiol 73: 4691–4694. http://dx.doi.org/10.1128/AEM.00496-07.

257. Neumann AM, Balmonte JP, Berger M, Giebel HA, Arnosti C, Voget S, Simon M, Brinkhoff T, Wietz M. 2015. Different utilization of alginate and other algal polysaccharides by marine Alteromonas macleodii ecotypes. Environ Microbiol 17:3857–3868. http://dx.doi.org/10.1111/1462-2920.12862.

258. Miyamoto K, Nukui E, Hirose M, Nagai F, Sato T, Inamori Y, Tsujibo H. 2002. A metalloprotease (MprIII) involved in the chitolytic system of a marine bacterium, Alteromonas sp. strain O-7. Appl Environ Microbiol 68:5563–5570. http://dx.doi.org/10.1128/AEM.68.11.5563-5570.2002.

259. Zhou MY, Chen XL, Zhao HL, Dang HY, Luan XW, Zhang XY, He HL, Zhou BC, Zhang YZ. 2009. Diversity of both the cultivable protease-producing bacteria and their extracellular proteases in the sediments of the South China Sea. Microb Ecol 58:582–590. http://dx.doi.org/10.1007/s00248-009-9506-z.

260. Wei CF, Alianell GA, Bencen GH, Gray HB, Jr. 1983. Isolation and comparison of two molecular species of the BAL 31 nuclease from Alteromonas espejiana with distinct kinetic properties. J Biol Chem 258: 13506–13512.

261. Lennon JT. 2007. Diversity and metabolism of marine bacteria cultivated on dissolved DNA. Appl Environ Microbiol 73:2799–2805. http://dx.doi.org/10.1128/AEM.02674-06.

262. Duflos M, Goutx M, Van Wambeke F. 2009. Determination of lipid degradation by marine lipase-producing bacteria: critical evaluation of lipase activity assays. Lipids 44:1113–1124. http://dx.doi.org/10.1007/s11745-009-3358-7.

263. Cunliffe M, Whiteley AS, Newbold L, Oliver A, Schäfer H, Murrell JC. 2009. Comparison of bacterioneuston and bacterioplankton dynamics during a phytoplankton bloom in a fjord mesocosm. Appl Environ Microbiol 75:7173–7181. http://dx.doi.org/10.1128/AEM.01374-09.

264. Tada Y, Taniguchi A, Nagao I, Miki T, Uematsu M, Tsuda A, Hamasaki K. 2011. Differing growth responses of major phylogenetic groups of marine bacteria to natural phytoplankton blooms in the western North Pacific Ocean. Appl Environ Microbiol 77:4055–4065. http://dx.doi.org/10.1128/AEM.02952-10.

265. Tinta T, Vojvoda J, Mozetič P, Talaber I, Vodopivec M, Malfatti F, Turk V. 2015. Bacterial community shift is induced by dynamic environmental parameters in a changing coastal ecosystem (northern Adriatic, northeastern Mediterranean Sea)—a 2-year time-series study. Environ Microbiol 17:3581–3596. http://dx.doi.org/10.1111/1462-2920.12519.

266. Hold GL, Smith EA, Rappé MS, Maas EW, Moore ERB, Stroempl C, Stephen JR, Prosser JI, Birkbeck TH, Gallacher S. 2001. Characterisation of bacterial communities associated with toxic and non-toxic dinoflagellates: *Alexandrium* spp. and *Scrippsiella trochoidea*. FEMS Microbiol Ecol 37:161–173. http://dx.doi.org/10.1111/j.1574-6941.2001.tb00864.x.

267. Hasegawa Y, Martin JL, Giewat MW, Rooney-Varga JN. 2007. Microbial community diversity in the phycosphere of natural populations of

the toxic alga, *Alexandrium fundyense*. Environ Microbiol 9:3108–3121. http://dx.doi.org/10.1111/j.1462-2920.2007.01421.x.

268. Sarmento H, Gasol JM. 2012. Use of phytoplankton-derived dissolved organic carbon by different types of bacterioplankton. Environ Microbiol 14:2348–2360. http://dx.doi.org/10.1111/j.1462-2920.2012.02787.x.

269. Mayali X, Azam F. 2004. Algicidal bacteria in the sea and their impact on algal blooms. J Eukaryot Microbiol 51:139–144. http://dx.doi.org/10.1111/j.1550-7408.2004.tb00538.x.

270. Pedler BE, Aluwihare LI, Azam F. 2014. Single bacterial strain capable of significant contribution to carbon cycling in the surface ocean. Proc Natl Acad Sci U S A 111:7202–7207. http://dx.doi.org/10.1073/pnas.1401887111.

271. Baker BJ, Sheik CS, Taylor CA, Jain S, Bhasi A, Cavalcoli JD, Dick GJ. 2013. Community transcriptomic assembly reveals microbes that contribute to deep-sea carbon and nitrogen cycling. ISME J 7:1962–1973. http://dx.doi.org/10.1038/ismej.2013.85.

272. Buchan A, LeCleir GR, Gulvik CA, González JM. 2014. Master recyclers: features and functions of bacteria associated with phytoplankton blooms. Nat Rev Microbiol 12:686–698. http://dx.doi.org/10.1038/nrmicro3326.

273. Azam F. 1998. Microbial control of oceanic carbon flux: the plot thickens. Science 280:694–696. http://dx.doi.org/10.1126/science.280.5364.694.

274. Kiørboe T, Jackson GA. 2001. Marine snow, organic solute plumes, and optimal chemosensory behavior of bacteria. Limnol Oceanogr 46:1309–1318. http://dx.doi.org/10.4319/lo.2001.46.6.1309.

275. Guenet B, Danger M, Abbadie L, Lacroix G. 2010. Priming effect: bridging the gap between terrestrial and aquatic ecology. Ecology 91: 2850–2861. http://dx.doi.org/10.1890/09-1968.1.

276. Bianchi TS. 2011. The role of terrestrially derived organic carbon in the coastal ocean: a changing paradigm and the priming effect. Proc Natl Acad Sci U S A 108:19473–19481. http://dx.doi.org/10.1073/pnas.1017982108.

277. Turley CM, Stutt ED. 2000. Depth-related cell-specific bacterial leucine incorporation rates on particles and its biogeochemical significance in the Northwest Mediterranean. Limnol Oceanogr 45:419–425. http://dx.doi.org/10.4319/lo.2000.45.2.0419.

278. Baltar F, Arístegui J, Sintes E, van Aken HM, Gasol JM, Herndl GJ. 2009. Prokaryotic extracellular enzymatic activity in relation to biomass production and respiration in the meso- and bathypelagic waters of the (sub)tropical Atlantic. Environ Microbiol 11:1998–2014. http://dx.doi.org/10.1111/j.1462-2920.2009.01922.x.

279. Herndl GJ, Agogué H, Baltar F, Reinthaler T, Sintes E, Varela MM. 2008. Regulation of aquatic microbial processes: the ‘microbial loop’ of the sunlit surface waters and the dark ocean dissected. Aquat Microb Ecol 53:59–68. http://dx.doi.org/10.3354/ame01225.

280. Lauro FM, Bartlett DH. 2008. Prokaryotic lifestyles in deep sea habitats. Extremophiles 12:15–25. http://dx.doi.org/10.1007/s00792-006-0059-5.

281. Bochdansky AB, van Aken HM, Herndl GJ. 2010. Role of macroscopic particles in deep-sea oxygen consumption. Proc Natl Acad Sci U S A 107:8287–8291. http://dx.doi.org/10.1073/pnas.0913744107.

282. Hopkinson CS, Jr, Vallino JJ. 2005. Efficient export of carbon to the deep ocean through dissolved organic matter. Nature 433:142–145. http://dx.doi.org/10.1038/nature03191.

283. Jiao N, Herndl GJ, Hansell DA, Benner R, Kattner G, Wilhelm SW, Kirchman DL, Weinbauer MG, Luo T, Chen F, Azam F. 2010. Microbial production of recalcitrant dissolved organic matter: long-term carbon storage in the global ocean. Nat Rev Microbiol 8:593–599. http://dx.doi.org/10.1038/nrmicro2386.

284. Dang H, Jiao N. 2014. Perspectives on the microbial carbon pump with special reference to microbial respiration and ecosystem efficiency in large estuarine systems. Biogeosciences 11:3887–3898. http://dx.doi.org/10.5194/bg-11-3887-2014.

285. Jiao N, Robinson C, Azam F, Thomas H, Baltar F, Dang H, Hardman-Mountford NJ, Johnson M, Kirchman DL, Koch BP, Legendre L, Li C, Liu J, Luo T, Luo YW, Mitra A, Romanou A, Tang K, Wang X, Zhang C, Zhang R. 2014. Mechanisms of microbial carbon sequestration in the ocean—future research directions. Biogeosciences 11:5285–5306. http://dx.doi.org/10.5194/bg-11-5285-2014.

286. Hansell DA. 2013. Recalcitrant dissolved organic carbon fractions. Annu Rev Mar Sci 5:421–445. http://dx.doi.org/10.1146/annurev-marine-120710-100757.

287. Laws EA, Falkowski PG, Smith WO, Jr, Ducklow H, McCarthy JJ.

2000. Temperature effects on export production in the open ocean. Glob Biogeochem Cycles 14:1231–1246. http://dx.doi.org/10.1029/1999GB001229.

288. Hofmann M, Schellnhuber HJ. 2009. Oceanic acidification affects marine carbon pump and triggers extended marine oxygen holes. Proc Natl Acad Sci U S A 106:3017–3022. http://dx.doi.org/10.1073/pnas.0813384106.

289. Kim JM, Lee K, Shin K, Yang EJ, Engel A, Karl DM, Kim HC. 2011. Shifts in biogenic carbon flow from particulate to dissolved forms under high carbon dioxide and warm ocean conditions. Geophys Res Lett 38: L08612.

290. Bowler C, Karl DM, Colwell RR. 2009. Microbial oceanography in a sea of opportunity. Nature 459:180–184. http://dx.doi.org/10.1038/nature08056.

291. Riebesell U, Körtzinger A, Oschlies A. 2009. Sensitivities of marine carbon fluxes to ocean change. Proc Natl Acad Sci U S A 106:20602–20609. http://dx.doi.org/10.1073/pnas.0813291106.

292. Passow U, Carlson CA. 2012. The biological pump in a high CO₂ world. Mar Ecol Prog Ser 470:249–271. http://dx.doi.org/10.3354/meps09985.

293. Marsay CM, Sanders RJ, Henson SA, Pabortsava K, Achterberg EP, Lampitt RS. 2015. Attenuation of sinking particulate organic carbon flux through the mesopelagic ocean. Proc Natl Acad Sci U S A 112:1089–1094. http://dx.doi.org/10.1073/pnas.1415311112.

294. de Jesus Mendes PA, Thomsen L. 2012. Effects of ocean acidification on the ballast of surface aggregates sinking through the twilight zone. PLoS One 7:e50865. http://dx.doi.org/10.1371/journal.pone.0050865.

295. MacGilchrist GA, Shi T, Tyrrell T, Richier S, Moore CM, Dumousseaud C, Achterberg EP. 2014. Effect of enhanced pCO₂ levels on the production of dissolved organic carbon and transparent exopolymer particles in short-term bioassay experiments. Biogeosciences 11:3695–3706. http://dx.doi.org/10.5194/bg-11-3695-2014.

296. Diaz RJ, Rosenberg R. 2008. Spreading dead zones and consequences for marine ecosystems. Science 321:926–929. http://dx.doi.org/10.1126/science.1156401.

297. Breitburg DL, Hondorp DW, Davias LA, Diaz RJ. 2009. Hypoxia, nitrogen, and fisheries: integrating effects across local and global landscapes. Annu Rev Mar Sci 1:329–349. http://dx.doi.org/10.1146/annurev.marine.010908.163754.

298. Keeling RF, Körtzinger A, Gruber N. 2010. Ocean deoxygenation in a warming world. Annu Rev Mar Sci 2:199–229. http://dx.doi.org/10.1146/annurev.marine.010908.163855.

299. Gilly WF, Beman JM, Litvin SY, Robison BH. 2013. Oceanographic and biological effects of shoaling of the oxygen minimum zone. Annu Rev Mar Sci 5:393–420. http://dx.doi.org/10.1146/annurev-marine-120710-100849.

300. Hawley AK, Brewer HM, Norbeck AD, Paša-Tolić L, Hallam SJ. 2014. Metaproteomics reveals differential modes of metabolic coupling among ubiquitous oxygen minimum zone microbes. Proc Natl Acad Sci U S A 111:11395–11400. http://dx.doi.org/10.1073/pnas.1322132111.

301. Cowen JP, Silver MW. 1984. The association of iron and manganese with bacteria on marine macroparticulate material. Science 224:1340–1342. http://dx.doi.org/10.1126/science.224.4655.1340.

302. Ulloa O, Canfield DE, DeLong EF, Letelier RM, Stewart FJ. 2012. Microbial oceanography of anoxic oxygen minimum zones. Proc Natl Acad Sci U S A 109:15996–16003. http://dx.doi.org/10.1073/pnas.1205009109.

303. Zhang W, Wang Y, Bougouffa S, Tian R, Cao H, Li Y, Cai L, Wong YH, Zhang G, Zhou G, Zhang X, Bajic VB, Al-Suwailem A, Qian PY. 2015. Synchronized dynamics of bacterial niche-specific functions during biofilm development in a cold seep brine pool. Environ Microbiol 17:4089–4104. http://dx.doi.org/10.1111/1462-2920.12978.

304. Burgin AJ, Yang WH, Hamilton SK, Silver WL. 2011. Beyond carbon and nitrogen: how the microbial energy economy couples elemental cycles in diverse ecosystems. Front Ecol Environ 9:44–52. http://dx.doi.org/10.1890/090227.

305. Behrenfeld MJ, O’Malley RT, Siegel DA, McClain CR, Sarmiento JL, Feldman GC, Milligan AJ, Falkowski PG, Letelier RM, Boss ES. 2006. Climate-driven trends in contemporary ocean productivity. Nature 444: 752–755. http://dx.doi.org/10.1038/nature05317.

306. Polovina JJ, Howell EA, Abecassis M. 2008. Ocean’s least productive waters are expanding. Geophys Res Lett 35:L03618.

307. Conley DJ, Paerl HW, Howarth RW, Boesch DF, Seitzinger SP, Harens KE, Lancelot C, Likens GE. 2009. Controlling eutrophication:

nitrogen and phosphorus. Science 323:1014–1015. http://dx.doi.org/10.1126/science.1167755.

308. Boyce DG, Lewis MR, Worm B. 2010. Global phytoplankton decline over the past century. Nature 466:591–596. http://dx.doi.org/10.1038/nature09268.

309. Schofield O, Ducklow HW, Martinson DG, Meredith MP, Moline MA, Fraser WR. 2010. How do polar marine ecosystems respond to rapid climate change? Science 328:1520–1523. http://dx.doi.org/10.1126/science.1185779.

310. Grebmeier JM. 2012. Shifting patterns of life in the Pacific Arctic and sub-Arctic seas. Annu Rev Mar Sci 4:63–78. http://dx.doi.org/10.1146/annurev-marine-120710-100926.

311. Chavez FP, Messié M, Pennington JT. 2011. Marine primary production in relation to climate variability and change. Annu Rev Mar Sci 3:227–260. http://dx.doi.org/10.1146/annurev.marine.010908.163917.

312. Bijma J, Pörtner HO, Yesson C, Rogers AD. 2013. Climate change and the oceans—what does the future hold? Mar Pollut Bull 74:495–505. http://dx.doi.org/10.1016/j.marpolbul.2013.07.022.

313. Caron DA, Davis PG, Madin LP, Sieburth JM. 1986. Enrichment of microbial populations in macroaggregates (marine snow) from surface waters of the North Atlantic. J Mar Res 44:543–565. http://dx.doi.org/10.1357/002224086788403042.

314. Dore JE, Letelier RM, Church MJ, Lukas R, Karl D. 2008. Summer phytoplankton blooms in the oligotrophic North Pacific Subtropical Gyre: historical perspective and recent observations. Prog Oceanogr 76: 2–38. http://dx.doi.org/10.1016/j.pocean.2007.10.002.

315. Kemp AES, Villareal TA. 2013. High diatom production and export in stratified waters—a potential negative feedback to global warming. Prog Oceanogr 119:4–23. http://dx.doi.org/10.1016/j.pocean.2013.06.004.

316. Paerl HW, Huisman J. 2009. Climate change: a catalyst for global expansion of harmful cyanobacterial blooms. Environ Microbiol Rep 1:27–37. http://dx.doi.org/10.1111/j.1758-2229.2008.00004.x.

317. Richardson AJ, Bakun A, Hays GC, Gibbons MJ. 2009. The jellyfish joyride: causes, consequences and management responses to a more gelatinous future. Trends Ecol Evol 24:312–322. http://dx.doi.org/10.1016/j.tree.2009.01.010.

318. Anderson DM, Cembella AD, Hallegraeff GM. 2012. Progress in understanding harmful algal blooms: paradigm shifts and new technologies for research, monitoring, and management. Annu Rev Mar Sci 4:143–176. http://dx.doi.org/10.1146/annurev-marine-120308-081121.

319. Purcell JE. 2012. Jellyfish and ctenophore blooms coincide with human proliferations and environmental perturbations. Annu Rev Mar Sci 4:209–235. http://dx.doi.org/10.1146/annurev-marine-120709-142751.

320. Garneau MÈ, Vincent WF, Terrado R, Lovejoy C. 2009. Importance of particle-associated bacterial heterotrophy in a coastal Arctic ecosystem. J Mar Syst 75:185–197. http://dx.doi.org/10.1016/j.jmarsys.2008.09.002.

321. Kellogg CTE, Carpenter SD, Renfro AA, Sallon A, Michel C, Cochran JK, Deming JW. 2011. Evidence for microbial attenuation of particle flux in the Amundsen Gulf and Beaufort Sea: elevated hydrolytic enzyme activity on sinking aggregates. Polar Biol 34:2007–2023. http://dx.doi.org/10.1007/s00300-011-1015-0.

322. Witt V, Wild C, Anthony KR, Diaz-Pulido G, Uthicke S. 2011. Effects of ocean acidification on microbial community composition of, and oxygen fluxes through, biofilms from the Great Barrier Reef. Environ Microbiol 13:2976–2989. http://dx.doi.org/10.1111/j.1462-2920.2011.02571.x.

323. Witt V, Wild C, Uthicke S. 2012. Terrestrial runoff controls the bacterial community composition of biofilms along a water quality gradient in the Great Barrier Reef. Appl Environ Microbiol 78:7786–7791. http://dx.doi.org/10.1128/AEM.01623-12.

324. Witt V, Wild C, Uthicke S. 2012. Interactive climate change and runoff effects alter O₂ fluxes and bacterial community composition of coastal biofilms from the Great Barrier Reef. Aquat Microb Ecol 66:117–131. http://dx.doi.org/10.3354/ame01562.

325. Palmer J, Flint S, Brooks J. 2007. Bacterial cell attachment, the beginning of a biofilm. J Ind Microbiol Biotechnol 34:577–588. http://dx.doi.org/10.1007/s10295-007-0234-4.

326. Ponsonnet L, Boureau M, Jaffrezic N, Othmane A, Dorel C, Lejeune P. 2008. Local pH variation as an initial step in bacterial surface-sensing and biofilm formation. Mater Sci Eng C 28:896–900. http://dx.doi.org/10.1016/j.msec.2007.10.031.

327. Sheng X, Ting YP, Pehkonen SO. 2008. The influence of ionic strength,

Dang and Lovell

nutrients and pH on bacterial adhesion to metals. J Colloid Interface Sci 321:256–264. http://dx.doi.org/10.1016/j.jcis.2008.02.038.

328. Tang KW, Grossart HP. 2007. Iron effects on colonization behavior, motility, and enzymatic activity of marine bacteria. Can J Microbiol 53: 968–974. http://dx.doi.org/10.1139/W07-059.

329. Kim HS, Park SJ, Lee KH. 2009. Role of NtrC-regulated exopolysaccharides in the biofilm formation and pathogenic interaction of Vibrio vulnificus. Mol Microbiol 74:436–453. http://dx.doi.org/10.1111/j.1365 -2958.2009.06875.x.

330. Pratt JT, McDonough E, Camilli A. 2009. PhoB regulates motility, biofilms, and cyclic di-GMP in Vibrio cholerae. J Bacteriol 191:6632– 6642. http://dx.doi.org/10.1128/JB.00708-09.

331. Sultan SZ, Silva AJ, Benitez JA. 2010. The PhoB regulatory system modulates biofilm formation and stress response in El Tor biotype Vibrio cholerae. FEMS Microbiol Lett 302:22–31. http://dx.doi.org/10.1111/j .1574-6968.2009.01837.x.

332. Santos-Beneit F. 2015. The Pho regulon: a huge regulatory network in bacteria. Front Microbiol 6:402. http://dx.doi.org/10.3389/fmicb.2015 .00402.

333. Ymele-Leki P, Houot L, Watnick PI. 2013. Mannitol and the mannitol-specific enzyme IIB subunit activate Vibrio cholerae biofilm formation. Appl Environ Microbiol 79:4675–4683. http://dx.doi.org/10.1128/AEM .01184-13.

334. Nagar E, Schwarz R. 2015. To be or not to be planktonic? Self-inhibition of biofilm development. Environ Microbiol 17:1477–1486. http://dx.doi .org/10.1111/1462-2920.12583.

335. Haugo AJ, Watnick PI. 2002. Vibrio cholerae CytR is a repressor of biofilm development. Mol Microbiol 45:471–483. http://dx.doi.org/10 .1046/j.1365-2958.2002.03023.x.

336. Houot L, Watnick PI. 2008. A novel role for enzyme I of the Vibrio cholerae phosphoenolpyruvate phosphotransferase system in regulation of growth in a biofilm. J Bacteriol 190:311–320. http://dx.doi.org/10 .1128/JB.01410-07.

337. Houot L, Chang S, Pickering BS, Absalon C, Watnick PI. 2010. The phosphoenolpyruvate phosphotransferase system regulates Vibrio chol- erae biofilm formation through multiple independent pathways. J Bac- teriol 192:3055–3067. http://dx.doi.org/10.1128/JB.00213-10.

338. Blokesch M. 2012. Chitin colonization, chitin degradation and chitin-induced natural competence of Vibrio cholerae are subject to catabolite repression. Environ Microbiol 14:1898–1912. http://dx.doi.org/10.1111 /j.1462-2920.2011.02689.x.

339. He H, Cooper JN, Mishra A, Raskin DM. 2012. Stringent response regulation of biofilm formation in Vibrio cholerae. J Bacteriol 194:2962– 2972. http://dx.doi.org/10.1128/JB.00014-12.

340. Pickering BS, Lopilato JE, Smith DR, Watnick PI. 2014. The transcrip- tion factor Mlc promotes Vibrio cholerae biofilm formation through re- pression of phosphotransferase system components. J Bacteriol 196: 2423–2430. http://dx.doi.org/10.1128/JB.01639-14.

341. Stocker R. 2012. Marine microbes see a sea of gradients. Science 338: 628–633. http://dx.doi.org/10.1126/science.1208929.

342. Bos R, van der Mei HC, Busscher HJ. 1999. Physico-chemistry of initial microbial adhesive interactions—its mechanisms and methods for study. FEMS Microbiol Rev 23:179–230. http://dx.doi.org/10.1111/j .1574-6976.1999.tb00396.x.

343. Chagnot C, Zorgani MA, Astruc T, Desvaux M. 2013. Proteinaceous determinants of surface colonization in bacteria: bacterial adhesion and biofilm formation from a protein secretion perspective. Front Microbiol 4:303. http://dx.doi.org/10.3389/fmicb.2013.00303.

344. Wozniak DJ, Parsek MR. 2014. Surface-associated microbes continue to surprise us in their sophisticated strategies for assembling biofilm com- munities. F1000Prime Rep 6:26. http://dx.doi.org/10.12703/P6-26.

345. Stock AM, Robinson VL, Goudreau PN. 2000. Two-component signal transduction. Annu Rev Biochem 69:183–215. http://dx.doi.org/10.1146 /annurev.biochem.69.1.183.

346. Capra EJ, Laub MT. 2012. Evolution of two-component signal trans- duction systems. Annu Rev Microbiol 66:325–347. http://dx.doi.org/10 .1146/annurev-micro-092611-150039.

347. Mikkelsen H, Sivaneson M, Filloux A. 2011. Key two-component reg- ulatory systems that control biofilm formation in Pseudomonas aerugi- nosa. Environ Microbiol 13:1666–1681. http://dx.doi.org/10.1111/j .1462-2920.2011.02495.x.

348. Voegel TM, Doddapaneni H, Cheng DW, Lin H, Stenger DC, Kirk- patrick BC, Roper MC. 2013. Identification of a response regulator

involved in surface attachment, cell-cell aggregation, exopolysaccharide production and virulence in the plant pathogen Xylella fastidiosa. Mol Plant Pathol 14:256–264. http://dx.doi.org/10.1111/mpp.12004.

349. Gao R, Stock AM. 2009. Biological insights from structures of two- component proteins. Annu Rev Microbiol 63:133–154. http://dx.doi.org /10.1146/annurev.micro.091208.073214.

350. Cheng AT, Ottemann KM, Yildiz FH. 2015. Vibrio cholerae response regulator VxrB controls colonization and regulates the type VI secretion system. PLoS Pathog 11:e1004933. http://dx.doi.org/10.1371/journal .ppat.1004933.

351. Yildiz FH, Dolganov NA, Schoolnik GK. 2001. VpsR, a member of the response regulators of the two-component regulatory systems, is re- quired for expression of vps biosynthesis genes and EPS<sup>ETR</sup>-associated phenotypes in Vibrio cholerae O1 El Tor. J Bacteriol 183:1716–1726. http: //dx.doi.org/10.1128/JB.183.5.1716-1726.2001.

352. Casper-Lindley C, Yildiz FH. 2004. VpsT is a transcriptional regulator required for expression of vps biosynthesis genes and the development of rugose colonial morphology in Vibrio cholerae O1 El Tor. J Bacteriol 186:1574–1578. http://dx.doi.org/10.1128/JB.186.5.1574-1578.2004.

353. Beyhan S, Bilecen K, Salama SR, Casper-Lindley C, Yildiz FH. 2007. Regulation of rugosity and biofilm formation in Vibrio cholerae: compar- ison of VpsT and VpsR regulons and epistasis analysis of vpsT, vpsR, and hapR. J Bacteriol 189:388–402. http://dx.doi.org/10.1128/JB.00981-06.

354. Shikuma NJ, Fong JC, Odell LS, Perchuk BS, Laub MT, Yildiz FH. 2009. Overexpression of VpsS, a hybrid sensor kinase, enhances biofilm formation in Vibrio cholerae. J Bacteriol 191:5147–5158. http://dx.doi .org/10.1128/JB.00401-09.

355. Norsworthy AN, Visick KL. 2015. Signaling between two interacting sensor kinases promotes biofilms and colonization by a bacterial symbi- ont. Mol Microbiol 96:233–248. http://dx.doi.org/10.1111/mmi.12932.

356. Khan NH, Ishii Y, Kimata-Kino N, Esaki H, Nishino T, Nishimura M, Kogure K. 2007. Isolation of Pseudomonas aeruginosa from open ocean and comparison with freshwater, clinical, and animal isolates. Microb Ecol 53:173–186. http://dx.doi.org/10.1007/s00248-006-9059-3.

357. Nonaka L, Inubushi A, Shinomiya H, Murase M, Suzuki S. 2010. Differences of genetic diversity and antibiotics susceptibility of Pseu- domonas aeruginosa isolated from hospital, river and coastal seawater. Environ Microbiol Rep 2:465–472. http://dx.doi.org/10.1111/j.1758 -2229.2010.00178.x.

358. Chakraborty J, Das S. 2014. Characterization and cadmium-resistant gene expression of biofilm-forming marine bacterium Pseudomonas aeruginosa JP-11. Environ Sci Pollut Res Int 21:14188–14201. http://dx .doi.org/10.1007/s11356-014-3308-7.

359. Lejeune P. 2003. Contamination of abiotic surfaces: what a colonizing bacterium sees and how to blur it. Trends Microbiol 11:179–184. http: //dx.doi.org/10.1016/S0966-842X(03)00047-7.

360. Stanley NR, Lazazzera BA. 2004. Environmental signals and regulatory pathways that influence biofilm formation. Mol Microbiol 52:917–924. http://dx.doi.org/10.1111/j.1365-2958.2004.04036.x.

361. Tow LA, Coyne VE. 1999. Cloning and characterisation of a novel ompB operon from Vibrio cholerae 569B. Biochim Biophys Acta 1444:269–275. http://dx.doi.org/10.1016/S0167-4781(98)00277-2.

362. Shikuma NJ, Yildiz FH. 2009. Identification and characterization of OscR, a transcriptional regulator involved in osmolarity adaptation in Vibrio cholerae. J Bacteriol 191:4082–4096. http://dx.doi.org/10.1128/JB .01540-08.

363. Dunlap PV. 2009. OscR, a new osmolarity-responsive regulator in Vibrio cholerae. J Bacteriol 191:4053–4055. http://dx.doi.org/10.1128/JB .00501-09.

364. Shikuma NJ, Davis KR, Fong JN, Yildiz FH. 2013. The transcriptional regulator, CosR, controls compatible solute biosynthesis and transport, motility and biofilm formation in Vibrio cholerae. Environ Microbiol 15:1387–1399. http://dx.doi.org/10.1111/j.1462-2920.2012.02805.x.

365. Laub MT, Goulian M. 2007. Specificity in two-component signal trans- duction pathways. Annu Rev Genet 41:121–145. http://dx.doi.org/10 .1146/annurev.genet.41.042007.170548.

366. Cheung J, Hendrickson WA. 2010. Sensor domains of two-component regulatory systems. Curr Opin Microbiol 13:116–123. http://dx.doi.org /10.1016/j.mib.2010.01.016.

367. Galperin MY. 2010. Diversity of structure and function of response regulator output domains. Curr Opin Microbiol 13:150–159. http://dx .doi.org/10.1016/j.mib.2010.01.005.

368. Hsieh YJ, Wanner BL. 2010. Global regulation by the seven-component

P<sub>i</sub> signaling system. Curr Opin Microbiol 13:198–203. http://dx.doi.org/10.1016/j.mib.2010.01.014.

369. Lamarche MG, Wanner BL, Crépin S, Harel J. 2008. The phosphate regulon and bacterial virulence: a regulatory network connecting phosphate homeostasis and pathogenesis. FEMS Microbiol Rev 32:461–473. http://dx.doi.org/10.1111/j.1574-6976.2008.00101.x.

370. Miyamoto K, Okunishi M, Nukui E, Tsuchiya T, Kobayashi T, Imada C, Tsujibo H. 2007. The regulator CdsS/CdsR two-component system modulates expression of genes involved in chitin degradation of *Pseudoalteromonas piscicida* strain O-7. Arch Microbiol 188:619–628. http://dx.doi.org/10.1007/s00203-007-0283-0.

371. Antonova ES, Bernardy EE, Hammer BK. 2012. Natural competence in *Vibrio cholerae* is controlled by a nucleoside scavenging response that requires CytR-dependent anti-activation. Mol Microbiol 86:1215–1231. http://dx.doi.org/10.1111/mmi.12054.

372. Yamamoto S, Mitobe J, Ishikawa T, Wai SN, Ohnishi M, Watanabe H, Izumiya H. 2014. Regulation of natural competence by the orphan two-component system sensor kinase ChiS involves a non-canonical transmembrane regulator in *Vibrio cholerae*. Mol Microbiol 91:326–347. http://dx.doi.org/10.1111/mmi.12462.

373. Souza CP, Almeida BC, Colwell RR, Rivera IN. 2011. The importance of chitin in the marine environment. Mar Biotechnol 13:823–830. http://dx.doi.org/10.1007/s10126-011-9388-1.

374. Canfield DE, Glazer AN, Falkowski PG. 2010. The evolution and future of Earth’s nitrogen cycle. Science 330:192–196. http://dx.doi.org/10.1126/science.1186120.

375. Lam P, Kuypers MMM. 2011. Microbial nitrogen cycling processes in oxygen minimum zones. Annu Rev Mar Sci 3:317–345. http://dx.doi.org/10.1146/annurev-marine-120709-142814.

376. Zehr JP, Kudela RM. 2011. Nitrogen cycle of the open ocean: from genes to ecosystems. Annu Rev Mar Sci 3:197–225. http://dx.doi.org/10.1146/annurev-marine-120709-142819.

377. Deutsch C, Weber T. 2012. Nutrient ratios as a tracer and driver of ocean biogeochemistry. Annu Rev Mar Sci 4:113–141. http://dx.doi.org/10.1146/annurev-marine-120709-142821.

378. Voss M, Bange HW, Dippner JW, Middelburg JJ, Montoya JP, Ward B. 2013. The marine nitrogen cycle: recent discoveries, uncertainties and the potential relevance of climate change. Philos Trans R Soc Lond B Biol Sci 368:20130121. http://dx.doi.org/10.1098/rstb.2013.0121.

379. Karl DM. 2014. Microbially mediated transformations of phosphorus in the sea: new views of an old cycle. Annu Rev Mar Sci 6:279–337. http://dx.doi.org/10.1146/annurev-marine-010213-135046.

380. Ulrich LE, Zhulin IB. 2010. The MiST2 database: a comprehensive genomics resource on microbial signal transduction. Nucleic Acids Res 38:D401–D407. http://dx.doi.org/10.1093/nar/gkp940.

381. Barakat M, Ortet P, Whitworth DE. 2011. P2CS: a database of prokaryotic two-component systems. Nucleic Acids Res 39:D771–D776. http://dx.doi.org/10.1093/nar/gkq1023.

382. Krell T, Lacal J, Busch A, Silva-Jiménez H, Guazzaroni ME, Ramos JL. 2010. Bacterial sensor kinases: diversity in the recognition of environmental signals. Annu Rev Microbiol 64:539–559. http://dx.doi.org/10.1146/annurev.micro.112408.134054.

383. Szurmant H, Ordal GW. 2004. Diversity in chemotaxis mechanisms among the bacteria and archaea. Microbiol Mol Biol Rev 68:301–319. http://dx.doi.org/10.1128/MMBR.68.2.301-319.2004.

384. Nixon BT, Ronson CW, Ausubel FM. 1986. Two-component regulatory systems responsive to environmental stimuli share strongly conserved domains with the nitrogen assimilation regulatory genes *ntrB* and *ntrC*. Proc Natl Acad Sci U S A 83:7850–7854. http://dx.doi.org/10.1073/pnas.83.20.7850.

385. Wadhams GH, Armitage JP. 2004. Making sense of it all: bacterial chemotaxis. Nat Rev Mol Cell Biol 5:1024–1037. http://dx.doi.org/10.1038/nrm1524.

386. Hazelbauer GL, Lai WC. 2010. Bacterial chemoreceptors: providing enhanced features to two-component signaling. Curr Opin Microbiol 13:124–132. http://dx.doi.org/10.1016/j.mib.2009.12.014.

387. Mitchell JG, Pearson L, Bonazinga A, Dillon S, Khouri H, Paxinos R. 1995. Long lag times and high velocities in the motility of natural assemblies of marine bacteria. Appl Environ Microbiol 61:877–882.

388. Grossart HP, Riemann L, Azam F. 2001. Bacterial motility in the sea and its ecological implications. Aquat Microb Ecol 25:247–258. http://dx.doi.org/10.3354/ame025247.

389. Johansen JE, Pinhassi J, Blackburn N, Zweifel UL, Hagström Å. 2002.

Variability in motility characteristics among marine bacteria. Aquat Microb Ecol 28:229–237. http://dx.doi.org/10.3354/ame028229.

390. Fenchel T. 2001. Eppur si muove: many water column bacteria are motile. Aquat Microb Ecol 24:197–201. http://dx.doi.org/10.3354/ame024197.

391. Sourjik V, Wingreen NS. 2012. Responding to chemical gradients: bacterial chemotaxis. Curr Opin Cell Biol 24:262–268. http://dx.doi.org/10.1016/j.ceb.2011.11.008.

392. Blackburn N, Fenchel T, Mitchell J. 1998. Microscale nutrient patches in planktonic habitats shown by chemotactic bacteria. Science 282:2254–2256. http://dx.doi.org/10.1126/science.282.5397.2254.

393. Hütz A, Schubert K, Overmann J. 2011. *Thalassospira* sp. isolated from the oligotrophic eastern Mediterranean Sea exhibits chemotaxis toward inorganic phosphate during starvation. Appl Environ Microbiol 77:4412–4421. http://dx.doi.org/10.1128/AEM.00490-11.

394. Tout J, Jeffries TC, Petrou K, Tyson GW, Webster NS, Garren M, Stocker R, Ralph PJ, Seymour JR. 2015. Chemotaxis by natural populations of coral reef bacteria. ISME J 9:1764–1777. http://dx.doi.org/10.1038/ismej.2014.261.

395. Taylor JR, Stocker R. 2012. Trade-offs of chemotactic foraging in turbulent water. Science 338:675–679. http://dx.doi.org/10.1126/science.1219417.

396. Taylor GT, Zheng D, Lee M, Troy PJ, Gyananath G, Sharma SK. 1997. Influence of surface properties on accumulation of conditioning films and marine bacteria on substrata exposed to oligotrophic waters. Biofouling 11:31–57. http://dx.doi.org/10.1080/08927019709378319.

397. Grossart HP, Kiørboe T, Tang K, Ploug H. 2003. Bacterial colonization of particles: growth and interactions. Appl Environ Microbiol 69:3500–3509. http://dx.doi.org/10.1128/AEM.69.6.3500-3509.2003.

398. Stocker R, Seymour JR. 2012. Ecology and physics of bacterial chemotaxis in the ocean. Microbiol Mol Biol Rev 76:792–812. http://dx.doi.org/10.1128/MMBR.00029-12.

399. O’Toole R, Milton DL, Wolf-Watz H. 1996. Chemotactic motility is required for invasion of the host by the fish pathogen *Vibrio anguillarum*. Mol Microbiol 19:625–637. http://dx.doi.org/10.1046/j.1365-2958.1996.412927.x.

400. Bordas MA, Balebona MC, Rodriguez-Maroto JM, Borrego JJ, Moriñigo MA. 1998. Chemotaxis of pathogenic *Vibrio* strains towards mucus surfaces of gilt-head sea bream (*Sparus aurata* L.). Appl Environ Microbiol 64:1573–1575.

401. Larsen MH, Larsen JL, Olsen JE. 2001. Chemotaxis of *Vibrio anguillarum* to fish mucus: role of the origin of the fish mucus, the fish species and the serogroup of the pathogen. FEMS Microbiol Ecol 38:77–80. http://dx.doi.org/10.1111/j.1574-6941.2001.tb00884.x.

402. Moorthy S, Watnick PI. 2005. Identification of novel stage-specific genetic requirements through whole genome transcription profiling of *Vibrio cholerae* biofilm development. Mol Microbiol 57:1623–1635. http://dx.doi.org/10.1111/j.1365-2958.2005.04797.x.

403. Mueller RS, McDougald D, Cusumano D, Sodhi N, Kjelleberg S, Azam F, Bartlett DH. 2007. *Vibrio cholerae* strains possess multiple strategies for abiotic and biotic surface colonization. J Bacteriol 189:5348–5360. http://dx.doi.org/10.1128/JB.01867-06.

404. Sonnenschein EC, Syit DA, Grossart HP, Ullrich MS. 2012. Chemotaxis of *Marinobacter adhaerens* and its impact on attachment to the diatom *Thalassiosira weissflogii*. Appl Environ Microbiol 78:6900–6907. http://dx.doi.org/10.1128/AEM.01790-12.

405. Kong W, Huang L, Su Y, Qin Y, Ma Y, Xu X, Lin M, Zheng J, Yan Q. 2015. Investigation of possible molecular mechanisms underlying the regulation of adhesion in *Vibrio alginolyticus* with comparative transcriptome analysis. Antonie Van Leeuwenhoek 107:1197–1206. http://dx.doi.org/10.1007/s10482-015-0411-9.

406. Deloney-Marino CR, Visick KL. 2012. Role for *cheR* of *Vibrio fischeri* in the *Vibrio*-squid symbiosis. Can J Microbiol 58:29–38. http://dx.doi.org/10.1139/w11-107.

407. Butler SM, Camilli A. 2005. Going against the grain: chemotaxis and infection in *Vibrio cholerae*. Nat Rev Microbiol 3:611–620. http://dx.doi.org/10.1038/nrmicro1207.

408. Taylor BL, Zhulin IB, Johnson MS. 1999. Aerotaxis and other energy-sensing behavior in bacteria. Annu Rev Microbiol 53:103–128. http://dx.doi.org/10.1146/annurev.micro.53.1.103.

409. Alexandre G, Greer-Phillips S, Zhulin IB. 2004. Ecological role of energy taxis in microorganisms. FEMS Microbiol Rev 28:113–126. http://dx.doi.org/10.1016/j.femsre.2003.10.003.

Dang and Lovell

410. Boin MA, Häse CC. 2007. Characterization of Vibrio cholerae aerotaxis. FEMS Microbiol Lett 276:193–201. http://dx.doi.org/10.1111/j.1574-6968.2007.00931.x.

411. Hiremath G, Hyakutake A, Yamamoto K, Ebisawa T, Nakamura T, Nishiyama S, Homma M, Kawagishi I. 2015. Hypoxia-induced localization of chemotaxis-related signaling proteins in Vibrio cholerae. Mol Microbiol 95:780–790. http://dx.doi.org/10.1111/mmi.12887.

412. Miller MB, Bassler BL. 2001. Quorum sensing in bacteria. Annu Rev Microbiol 55:165–199. http://dx.doi.org/10.1146/annurev.micro.55.1.165.

413. Paggi RA, Martone CB, Fuqua C, De Castro RE. 2003. Detection of quorum sensing signals in the haloalkaliphilic archaeon Natronococcus occultus. FEMS Microbiol Lett 221:49–52. http://dx.doi.org/10.1016/S0378-1097(03)00174-5.

414. Ng WL, Bassler BL. 2009. Bacterial quorum-sensing network architectures. Annu Rev Genet 43:197–222. http://dx.doi.org/10.1146/annurev-genet-102108-134304.

415. Tommonaro G, Abbamondi GR, Iodice C, Tait K, De Rosa S. 2012. Diketopiperazines produced by the halophilic archaeon, Haloterrigena hispanica, activate AHL bioreporters. Microb Ecol 63:490–495. http://dx.doi.org/10.1007/s00248-011-9980-y.

416. Zhang G, Zhang F, Ding G, Li J, Guo X, Zhu J, Zhou L, Cai S, Liu X, Luo Y, Zhang G, Shi W, Dong X. 2012. Acyl homoserine lactone-based quorum sensing in a methanogenic archaeon. ISME J 6:1336–1344. http://dx.doi.org/10.1038/ismej.2011.203.

417. Schuster M, Sexton DJ, Diggle SP, Greenberg EP. 2013. Acyl-homoserine lactone quorum sensing: from evolution to application. Annu Rev Microbiol 67:43–63. http://dx.doi.org/10.1146/annurev-micro-092412-155635.

418. Verma SC, Miyashiro T. 2013. Quorum sensing in the squid-Vibrio symbiosis. Int J Mol Sci 14:16386–16401. http://dx.doi.org/10.3390/ijms140816386.

419. Miyashiro T, Oehlert D, Ray VA, Visick KL, Ruby EG. 2014. The putative oligosaccharide translocase SypK connects biofilm formation with quorum signaling in Vibrio fischeri. Microbiologyopen 3:836–848. http://dx.doi.org/10.1002/mbo3.199.

420. Jung SA, Chapman CA, Ng WL. 2015. Quadruple quorum-sensing inputs control Vibrio cholerae virulence and maintain system robustness. PLoS Pathog 11:e1004837. http://dx.doi.org/10.1371/journal.ppat.1004837.

421. González A, Bellenberg S, Mamani S, Ruiz L, Echeverría A, Soulère L, Doutheau A, Demergasso C, Sand W, Queneau Y, Vera M, Guiliani N. 2013. AHL signaling molecules with a large acyl chain enhance biofilm formation on sulfur and metal sulfides by the bioleaching bacterium Acidithiobacillus ferrooxidans. Appl Microbiol Biotechnol 97:3729–3737. http://dx.doi.org/10.1007/s00253-012-4229-3.

422. Ramírez P, Guiliani N, Valenzuela L, Beard S, Jerez CA. 2004. Differential protein expression during growth of Acidithiobacillus ferrooxidans on ferrous iron, sulfur compounds, or metal sulfides. Appl Environ Microbiol 70:4491–4498. http://dx.doi.org/10.1128/AEM.70.8.4491-4498.2004.

423. Kucera J, Bouchal P, Lochman J, Potesil D, Janiczek O, Zdrahal Z, Mandl M. 2013. Ferrous iron oxidation by sulfur-oxidizing Acidithiobacillus ferrooxidans and analysis of the process at the levels of transcription and protein synthesis. Antonie Van Leeuwenhoek 103:905–919. http://dx.doi.org/10.1007/s10482-012-9872-2.

424. Gehrke T, Telegdi J, Thierry D, Sand W. 1998. Importance of extracellular polymeric substances from Thiobacillus ferrooxidans for bioleaching. Appl Environ Microbiol 64:2743–2747.

425. Banderas A, Guiliani N. 2013. Bioinformatic prediction of gene functions regulated by quorum sensing in the bioleaching bacterium Acidithiobacillus ferrooxidans. Int J Mol Sci 14:16901–16916. http://dx.doi.org/10.3390/ijms140816901.

426. Rivas M, Seeger M, Jedlicki E, Holmes DS. 2007. Second acyl homoserine lactone production system in the extreme acidophile Acidithiobacillus ferrooxidans. Appl Environ Microbiol 73:3225–3231. http://dx.doi.org/10.1128/AEM.02948-06.

427. Farah C, Vera M, Morin D, Haras D, Jerez CA, Guiliani N. 2005. Evidence for a functional quorum-sensing type AI-1 system in the extremophilic bacterium Acidithiobacillus ferrooxidans. Appl Environ Microbiol 71:7033–7040. http://dx.doi.org/10.1128/AEM.71.11.7033-7040.2005.

428. Rivas M, Seeger M, Holmes DS, Jedlicki E. 2005. A Lux-like quorum

sensing system in the extreme acidophile Acidithiobacillus ferrooxidans. Biol Res 38:283–297.

429. Ghosh W, Dam B. 2009. Biochemistry and molecular biology of lithotrophic sulfur oxidation by taxonomically and ecologically diverse bacteria and archaea. FEMS Microbiol Rev 33:999–1043. http://dx.doi.org/10.1111/j.1574-6976.2009.00187.x.

430. Stewart FJ. 2011. Dissimilatory sulfur cycling in oxygen minimum zones: an emerging metagenomics perspective. Biochem Soc Trans 39:1859–1863. http://dx.doi.org/10.1042/BST20110708.

431. Emerson D. 2012. Biogeochemistry and microbiology of microaerobic Fe(II) oxidation. Biochem Soc Trans 40:1211–1216. http://dx.doi.org/10.1042/BST20120154.

432. Carlson HK, Clark IC, Melnyk RA, Coates JD. 2012. Toward a mechanistic understanding of anaerobic nitrate-dependent iron oxidation: balancing electron uptake and detoxification. Front Microbiol 3:57. http://dx.doi.org/10.3389/fmicb.2012.00057.

433. Melton ED, Swanner ED, Behrens S, Schmidt C, Kappler A. 2014. The interplay of microbially mediated and abiotic reactions in the biogeochemical Fe cycle. Nat Rev Microbiol 12:797–808. http://dx.doi.org/10.1038/nrmicro3347.

434. Nunoura T, Takaki Y, Kazama H, Kakuta J, Shimamura S, Makita H, Hirai M, Miyazaki M, Takai K. 2014. Physiological and genomic features of a novel sulfur-oxidizing gammaproteobacterium belonging to a previously uncultivated symbiotic lineage isolated from a hydrothermal vent. PLoS One 9:e104959. http://dx.doi.org/10.1371/journal.pone.0104959.

435. Pérez-Rodríguez I, Bolognini M, Ricci J, Bini E, Vetriani C. 2015. From deep-sea volcanoes to human pathogens: a conserved quorum-sensing signal in Epsilonproteobacteria. ISME J 9:1222–1234. http://dx.doi.org/10.1038/ismej.2014.214.

436. Vera M, Schippers A, Sand W. 2013. Progress in bioleaching: fundamentals and mechanisms of bacterial metal sulfide oxidation—part A. Appl Microbiol Biotechnol 97:7529–7541. http://dx.doi.org/10.1007/s00253-013-4954-2.

437. Alterman E. 2014. Invited commentary: lubricating the rusty wheel, new insights into iron oxidizing bacteria through comparative genomics. Front Microbiol 5:386. http://dx.doi.org/10.3389/fmicb.2014.00386.

438. Kappler A, Straub KL. 2005. Geomicrobiological cycling of iron. Rev Miner Geochem 59:85–108. http://dx.doi.org/10.2138/rmg.2005.59.5.

439. von der Heyden BP, Roychoudhury AN, Mtshali TN, Tyliszczak T, Myneni SC. 2012. Chemically and geographically distinct solid-phase iron pools in the Southern Ocean. Science 338:1199–1201. http://dx.doi.org/10.1126/science.1227504.

440. Huang YL, Ki JS, Lee OO, Qian PY. 2009. Evidence for the dynamics of acyl homoserine lactone and AHL-producing bacteria during subtidal biofilm formation. ISME J 3:296–304. http://dx.doi.org/10.1038/ismej.2008.105.

441. Decho AW, Norman RS, Visscher PT. 2010. Quorum sensing in natural environments: emerging views from microbial mats. Trends Microbiol 18:73–80. http://dx.doi.org/10.1016/j.tim.2009.12.008.

442. Van Mooy BA, Hmelo LR, Sofen LE, Campagna SR, May AL, Dyhrman ST, Heithoff A, Webb EA, Momper L, Mincer TJ. 2012. Quorum sensing control of phosphorus acquisition in Trichodesmium consortia. ISME J 6:422–429. http://dx.doi.org/10.1038/ismej.2011.115.

443. Doberva M, Sanchez-Ferandin S, Toulza E, Lebaron P, Lami R. 2015. Diversity of quorum sensing autoinducer synthases in the Global Ocean Sampling metagenomic database. Aquat Microb Ecol 74:107–119. http://dx.doi.org/10.3354/ame01734.

444. González JE, Marketon MM. 2003. Quorum sensing in nitrogen-fixing rhizobia. Microbiol Mol Biol Rev 67:574–592. http://dx.doi.org/10.1128/MMBR.67.4.574-592.2003.

445. Toyofuku M, Nomura N, Kuno E, Tashiro Y, Nakajima T, Uchiyama H. 2008. Influence of the Pseudomonas quinolone signal on denitrification in Pseudomonas aeruginosa. J Bacteriol 190:7947–7956. http://dx.doi.org/10.1128/JB.00968-08.

446. De Clippeleir H, Defoirdt T, Vanhaecke L, Vlaeminck SE, Carballa M, Verstraete W, Boon N. 2011. Long-chain acylhomoserine lactones increase the anoxic ammonium oxidation rate in an OLAND biofilm. Appl Microbiol Biotechnol 90:1511–1519. http://dx.doi.org/10.1007/s00253-011-3177-7.

447. Gao J, Ma A, Zhuang X, Zhuang G. 2014. An N-acyl homoserine lactone synthase in the ammonia-oxidizing bacterium Nitrosospira mul-
tiformis. Appl Environ Microbiol 80:951–958. http://dx.doi.org/10.1128/AEM.03361-13.

448. Mellbye BL, Bottomley PJ, Sayavedra-Soto LA. 2015. The nitrite-oxidizing bacterium Nitrobacter winogradskyi produces N-acyl-homoserine lactone autoinducers. Appl Environ Microbiol 81:5917–5926. http://dx.doi.org/10.1128/AEM.01103-15.

449. Boyer M, Wisniewski-Dyé F. 2009. Cell-cell signalling in bacteria: not simply a matter of quorum. FEMS Microbiol Ecol 70:1–19. http://dx.doi.org/10.1111/j.1574-6941.2009.00745.x.

450. Cude WN, Buchan A. 2013. Acyl-homoserine lactone-based quorum sensing in the Roseobacter clade: complex cell-to-cell communication controls multiple physiologies. Front Microbiol 4:336. http://dx.doi.org/10.3389/fmicb.2013.00336.

451. Zan J, Liu Y, Fuqua C, Hill RT. 2014. Acyl-homoserine lactone quorum sensing in the Roseobacter clade. Int J Mol Sci 15:654–669. http://dx.doi.org/10.3390/ijms15010654.

452. Shrout JD, Nerenberg R. 2012. Monitoring bacterial twitter: does quorum sensing determine the behavior of water and wastewater treatment biofilms? Environ Sci Technol 46:1995–2005. http://dx.doi.org/10.1021/es203933h.

453. Kviatkovski I, Minz D. 2015. A member of the Rhodobacteraceae promotes initial biofilm formation via the secretion of extracellular factor(s). Aquat Microb Ecol 75:155–167. http://dx.doi.org/10.3354/ame01754.

454. Gottesman S, Storz G. 2011. Bacterial small RNA regulators: versatile roles and rapidly evolving variations. Cold Spring Harb Perspect Biol 3:a003798. http://dx.doi.org/10.1101/cshperspect.a003798.

455. Göpel Y, Görke B. 2012. Rewiring two-component signal transduction with small RNAs. Curr Opin Microbiol 15:132–139. http://dx.doi.org/10.1016/j.mib.2011.12.001.

456. Lenz DH, Miller MB, Zhu J, Kulkarni RV, Bassler BL. 2005. CsrA and three redundant small RNAs regulate quorum sensing in Vibrio cholerae. Mol Microbiol 58:1186–1202. http://dx.doi.org/10.1111/j.1365-2958.2005.04902.x.

457. Lenz DH, Mok KC, Lilley BN, Kulkarni RV, Wingreen NS, Bassler BL. 2004. The small RNA chaperone Hfq and multiple small RNAs control quorum sensing in Vibrio harveyi and Vibrio cholerae. Cell 118:69–82. http://dx.doi.org/10.1016/j.cell.2004.06.009.

458. Hammer BK, Bassler BL. 2007. Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic Vibrio cholerae. Proc Natl Acad Sci U S A 104:11145–11149. http://dx.doi.org/10.1073/pnas.0703860104.

459. Rutherford ST, van Kessel JC, Shao Y, Bassler BL. 2011. AphA and LuxR/HapR reciprocally control quorum sensing in vibrios. Genes Dev 25:397–408. http://dx.doi.org/10.1101/gad.2015011.

460. Shao Y, Bassler BL. 2012. Quorum-sensing non-coding small RNAs use unique pairing regions to differentially control mRNA targets. Mol Microbiol 83:599–611. http://dx.doi.org/10.1111/j.1365-2958.2011.07959.x.

461. Tu KC, Bassler BL. 2007. Multiple small RNAs act additively to integrate sensory information and control quorum sensing in Vibrio harveyi. Genes Dev 21:221–233. http://dx.doi.org/10.1101/gad.1502407.

462. Svenningsen SL, Tu KC, Bassler BL. 2009. Gene dosage compensation calibrates four regulatory RNAs to control Vibrio cholerae quorum sensing. EMBO J 28:429–439. http://dx.doi.org/10.1038/emboj.2008.300.

463. Hunter GA, Keener JP. 2014. Mechanisms underlying the additive and redundant Qrr phenotypes in Vibrio harveyi and Vibrio cholerae. J Theor Biol 340:38–49. http://dx.doi.org/10.1016/j.jtbi.2013.08.034.

464. Papenfort K, Förstner KU, Cong JP, Sharma CM, Bassler BL. 2015. Differential RNA-seq of Vibrio cholerae identifies the VqmR small RNA as a regulator of biofilm formation. Proc Natl Acad Sci U S A 112:E766–E775. http://dx.doi.org/10.1073/pnas.1500203112.

465. Wang H, Ayala JC, Silva AJ, Benitez JA. 2012. The histone-like nucleoid structuring protein (H-NS) is a repressor of Vibrio cholerae exopolysaccharide biosynthesis (*vps*) genes. Appl Environ Microbiol 78:2482–2488. http://dx.doi.org/10.1128/AEM.07629-11.

466. Fong JC, Yildiz FH. 2007. The *rbmBCDEF* gene cluster modulates development of rugose colony morphology and biofilm formation in Vibrio cholerae. J Bacteriol 189:2319–2330. http://dx.doi.org/10.1128/JB.01569-06.

467. Absalon C, Van Dellen K, Watnick PI. 2011. A communal bacterial adhesin anchors biofilm and bystander cells to surfaces. PLoS Pathog 7:e1002210. http://dx.doi.org/10.1371/journal.ppat.1002210.

Marine Biofilm Development

468. Berk V, Fong JC, Dempsey GT, Develioglu ON, Zhuang X, Liphardt J, Yildiz FH, Chu S. 2012. Molecular architecture and assembly principles of Vibrio cholerae biofilms. Science 337:236–239. http://dx.doi.org/10.1126/science.1222981.

469. Giglio KM, Fong JC, Yildiz FH, Sondermann H. 2013. Structural basis for biofilm formation via the Vibrio cholerae matrix protein RbmA. J Bacteriol 195:3277–3286. http://dx.doi.org/10.1128/JB.00374-13.

470. Johnson TL, Fong JC, Rule C, Rogers A, Yildiz FH, Sandkvist M. 2014. The type II secretion system delivers matrix proteins for biofilm formation by Vibrio cholerae. J Bacteriol 196:4245–4252. http://dx.doi.org/10.1128/JB.01944-14.

471. Fong JC, Karplus K, Schoolnik GK, Yildiz FH. 2006. Identification and characterization of RbmA, a novel protein required for the development of rugose colony morphology and biofilm structure in Vibrio cholerae. J Bacteriol 188:1049–1059. http://dx.doi.org/10.1128/JB.188.3.1049-1059.2006.

472. Fong JC, Yildiz FH. 2008. Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in Vibrio cholerae biofilm formation. J Bacteriol 190:6646–6659. http://dx.doi.org/10.1128/JB.00466-08.

473. Song T, Sabharwal D, Gurung JM, Cheng AT, Sjöström AE, Yildiz FH, Uhlin BE, Wai SN. 2014. Vibrio cholerae utilizes direct sRNA regulation in expression of a biofilm matrix protein. PLoS One 9:e101280. http://dx.doi.org/10.1371/journal.pone.0101280.

474. Mey AR, Craig SA, Payne SM. 2005. Characterization of Vibrio cholerae RyhB: the RyhB regulon and role of *ryhB* in biofilm formation. Infect Immun 73:5706–5719. http://dx.doi.org/10.1128/IAI.73.9.5706-5719.2005.

475. Nguyen AN, Jacq A. 2014. Small RNAs in the Vibrionaceae: an ocean still to be explored. Wiley Interdiscip Rev RNA 5:381–392. http://dx.doi.org/10.1002/wrna.1218.

476. McDonough KA, Rodriguez A. 2011. The myriad roles of cyclic AMP in microbial pathogens: from signal to sword. Nat Rev Microbiol 10:27–38. http://dx.doi.org/10.1038/nrmicro2688.

477. Liang W, Pascual-Montano A, Silva AJ, Benitez JA. 2007. The cyclic AMP receptor protein modulates quorum sensing, motility and multiple genes that affect intestinal colonization in Vibrio cholerae. Microbiology 153:2964–2975. http://dx.doi.org/10.1099/mic.0.2007/006668-0.

478. Kalivoda EJ, Brothers KM, Stella NA, Schmitt MJ, Shanks RM. 2013. Bacterial cyclic AMP-phosphodiesterase activity coordinates biofilm formation. PLoS One 8:e71267. http://dx.doi.org/10.1371/journal.pone.0071267.

479. Ono K, Oka R, Toyofuku M, Sakaguchi A, Hamada M, Yoshida S, Nomura N. 2014. cAMP signaling affects irreversible attachment during biofilm formation by Pseudomonas aeruginosa PAO1. Microbes Environ 29:104–106. http://dx.doi.org/10.1264/jsme2.ME13151.

480. Römling U, Galperin MY, Gomelsky M. 2013. Cyclic di-GMP: the first 25 years of a universal bacterial second messenger. Microbiol Mol Biol Rev 77:1–52. http://dx.doi.org/10.1128/MMBR.00043-12.

481. Belas R. 2014. Biofilms, flagella, and mechanosensing of surfaces by bacteria. Trends Microbiol 22:517–527. http://dx.doi.org/10.1016/j.tim.2014.05.002.

482. Petrova OE, Cherny KE, Sauer K. 2014. The Pseudomonas aeruginosa diguanylate cyclase GcbA, a homolog of *P. fluorescens* GcbA, promotes initial attachment to surfaces, but not biofilm formation, via regulation of motility. J Bacteriol 196:2827–2841. http://dx.doi.org/10.1128/JB.01628-14.

483. D’Alvise PW, Magdenoska O, Melchiorsen J, Nielsen KF, Gram L. 2014. Biofilm formation and antibiotic production in Ruegeria mobilis are influenced by intracellular concentrations of cyclic dimeric guanosine monophosphate. Environ Microbiol 16:1252–1266. http://dx.doi.org/10.1111/1462-2920.12265.

484. Tomlinson AD, Fuqua C. 2009. Mechanisms and regulation of polar surface attachment in Agrobacterium tumefaciens. Curr Opin Microbiol 12:708–714. http://dx.doi.org/10.1016/j.mib.2009.09.014.

485. Xu J, Kim J, Koestler BJ, Choi JH, Waters CM, Fuqua C. 2013. Genetic analysis of Agrobacterium tumefaciens unipolar polysaccharide production reveals complex integrated control of the motile-to-sessile switch. Mol Microbiol 89:929–948. http://dx.doi.org/10.1111/mmi.12321.

486. Chao L, Rakshe S, Leff M, Spormann AM. 2013. PdeB, a cyclic Di-GMP-specific phosphodiesterase that regulates Shewanella oneidensis MR-1 motility and biofilm formation. J Bacteriol 195:3827–3833. http://dx.doi.org/10.1128/JB.00498-13.
Dang and Lovell

487. Liu N, Xu Y, Hossain S, Huang N, Coursolle D, Gralnick JA, Boon EM. 2012. Nitric oxide regulation of cyclic di-GMP synthesis and hydrolysis in *Shewanella woodyi*. Biochemistry 51:2087–2099. [http://dx.doi.org/10.1021/bi201753f](http://dx.doi.org/10.1021/bi201753f).

488. Ruiz LM, Castro M, Barriga A, Jerez CA, Guiliani N. 2012. The extremophile *Acidithiobacillus ferrooxidans* possesses a c-di-GMP signaling pathway that could play a significant role during bioleaching of minerals. Lett Appl Microbiol 54:133–139. [http://dx.doi.org/10.1111/j.1472-765X.2011.03180.x](http://dx.doi.org/10.1111/j.1472-765X.2011.03180.x).

489. Hobley L, Fung RK, Lambert C, Harris MA, Dabhi JM, King SS, Basford SM, Uchida K, Till R, Ahmad R, Aizawa S, Gomelsky M, Sockett RE. 2012. Discrete cyclic di-GMP-dependent control of bacterial predation versus axenic growth in *Bdellovibrio bacteriovorus*. PLoS Pathog 8:e1002493. [http://dx.doi.org/10.1371/journal.ppat.1002493](http://dx.doi.org/10.1371/journal.ppat.1002493).

490. Camilli A, Bassler BL. 2006. Bacterial small-molecule signaling pathways. Science 311:1113–1116. [http://dx.doi.org/10.1126/science.1121357](http://dx.doi.org/10.1126/science.1121357).

491. Beyhan S, Tischler AD, Camilli A, Yildiz FH. 2006. Transcriptome and phenotypic responses of *Vibrio cholerae* to increased cyclic di-GMP level. J Bacteriol 188:3600–3613. [http://dx.doi.org/10.1128/JB.188.10.3600-3613.2006](http://dx.doi.org/10.1128/JB.188.10.3600-3613.2006).

492. Tischler AD, Camilli A. 2004. Cyclic diguanylate (c-di-GMP) regulates *Vibrio cholerae* biofilm formation. Mol Microbiol 53:857–869. [http://dx.doi.org/10.1111/j.1365-2958.2004.04155.x](http://dx.doi.org/10.1111/j.1365-2958.2004.04155.x).

493. Karatan E, Duncan TR, Watnick PI. 2005. NspS, a predicted polyamine sensor, mediates activation of *Vibrio cholerae* biofilm formation by norspermidine. J Bacteriol 187:7434–7443. [http://dx.doi.org/10.1128/JB.187.21.7434-7443.2005](http://dx.doi.org/10.1128/JB.187.21.7434-7443.2005).

494. McGinnis MW, Parker ZM, Walter NE, Rutkovsky AC, Cartaya-Marin C, Karatan E. 2009. Spermidine regulates *Vibrio cholerae* biofilm formation via transport and signaling pathways. FEMS Microbiol Lett 299:166–174. [http://dx.doi.org/10.1111/j.1574-6968.2009.01744.x](http://dx.doi.org/10.1111/j.1574-6968.2009.01744.x).

495. Mueller RS, Beyhan S, Saini SG, Yildiz FH, Bartlett DH. 2009. Indole acts as an extracellular cue regulating gene expression in *Vibrio cholerae*. J Bacteriol 191:3504–3516. [http://dx.doi.org/10.1128/JB.01240-08](http://dx.doi.org/10.1128/JB.01240-08).

496. Bernier SP, Ha DG, Khan W, Merritt JH, O’Toole GA. 2011. Modulation of *Pseudomonas aeruginosa* surface-associated group behaviors by individual amino acids through c-di-GMP signaling. Res Microbiol 162:680–688. [http://dx.doi.org/10.1016/j.resmic.2011.04.014](http://dx.doi.org/10.1016/j.resmic.2011.04.014).

497. Schaller RA, Ali SK, Klose KE, Kurtz DM, Jr. 2012. A bacterial hemerythrin domain regulates the activity of a *Vibrio cholerae* diguanylate cyclase. Biochemistry 51:8563–8570. [http://dx.doi.org/10.1021/bi3011797](http://dx.doi.org/10.1021/bi3011797).

498. Bharati BK, Swetha RK, Chatterji D. 2013. Identification and characterization of starvation induced *msdgC-1* promoter involved in the c-di-GMP turnover. Gene 528:99–108. [http://dx.doi.org/10.1016/j.gene.2013.07.043](http://dx.doi.org/10.1016/j.gene.2013.07.043).

499. Martínková M, Kitanishi K, Shimizu T. 2013. Heme-based globin-coupled oxygen sensors: linking oxygen binding to functional regulation of diguanylate cyclase, histidine kinase, and methyl-accepting chemotaxis. J Biol Chem 288:27702–27711. [http://dx.doi.org/10.1074/jbc.R113.473249](http://dx.doi.org/10.1074/jbc.R113.473249).

500. Plate L, Marletta MA. 2013. Nitric oxide-sensing H-NOX proteins govern bacterial communal behavior. Trends Biochem Sci 38:566–575. [http://dx.doi.org/10.1016/j.tibs.2013.08.008](http://dx.doi.org/10.1016/j.tibs.2013.08.008).

501. Zähringer F, Lacanna E, Jenal U, Schirmer T, Boehm A. 2013. Structure and signaling mechanism of a zinc-sensory diguanylate cyclase. Structure 21:1149–1157. [http://dx.doi.org/10.1016/j.str.2013.04.026](http://dx.doi.org/10.1016/j.str.2013.04.026).

502. Basu Roy A, Sauer K. 2014. Diguanylate cyclase NicD-based signalling mechanism of nutrient-induced dispersion by *Pseudomonas aeruginosa*. Mol Microbiol 94:771–793. [http://dx.doi.org/10.1111/mmi.12802](http://dx.doi.org/10.1111/mmi.12802).

503. Bouffartigues E, Duchesne R, Bazire A, Simon M, Maillot O, Dufour A, Feuilloley M, Orange N, Chevalier S. 2014. Sucrose favors *Pseudomonas aeruginosa* pellicle production through the extracytoplasmic function sigma factor SigX. FEMS Microbiol Lett 356:193–200. [http://dx.doi.org/10.1111/1574-6968.12482](http://dx.doi.org/10.1111/1574-6968.12482).

504. Burns JL, Deer DD, Weinert EE. 2014. Oligomeric state affects oxygen dissociation and diguanylate cyclase activity of globin coupled sensors. Mol Biosyst 10:2823–2826. [http://dx.doi.org/10.1039/C4MB00366G](http://dx.doi.org/10.1039/C4MB00366G).

505. Cockerell SR, Rutkovsky AC, Zayner JP, Cooper RE, Porter LR, Pendergraft SS, Parker ZM, McGinnis MW, Karatan E. 2014. *Vibrio cholerae* NspS, a homologue of ABC-type periplasmic solute binding proteins, facilitates transduction of polyamine signals independent of their transport. Microbiology 160:832–843. [http://dx.doi.org/10.1099/mic.0.075903-0](http://dx.doi.org/10.1099/mic.0.075903-0).

506. Enomoto G, Nomura R, Shimada T, Ni-Ni-Win, Narikawa R, Ikeuchi M. 2014. Cyanobacteriochrome SesA is a diguanylate cyclase that induces cell aggregation in *Thermosynechococcus*. J Biol Chem 289:24801–24809. [http://dx.doi.org/10.1074/jbc.M114.583674](http://dx.doi.org/10.1074/jbc.M114.583674).

507. Koestler BJ, Waters CM. 2014. Bile acids and bicarbonate inversely regulate intracellular cyclic di-GMP in *Vibrio cholerae*. Infect Immun 82:3002–3014. [http://dx.doi.org/10.1128/IAI.01664-14](http://dx.doi.org/10.1128/IAI.01664-14).

508. Oliveira MC, Teixeira RD, Andrade MO, Pinheiro GM, Ramos CH, Farah CS. 2015. Cooperative substrate binding by a diguanylate cyclase. J Mol Biol 427:415–432. [http://dx.doi.org/10.1016/j.jmb.2014.11.012](http://dx.doi.org/10.1016/j.jmb.2014.11.012).

509. Monds RD, O’Toole GA. 2009. The developmental model of microbial biofilms: ten years of a paradigm up for review. Trends Microbiol 17:73–87. [http://dx.doi.org/10.1016/j.tim.2008.11.001](http://dx.doi.org/10.1016/j.tim.2008.11.001).

510. Srivastava D, Harris RC, Waters CM. 2011. Integration of cyclic di-GMP and quorum sensing in the control of *vpsT* and *aphA* in *Vibrio cholerae*. J Bacteriol 193:6331–6341. [http://dx.doi.org/10.1128/JB.05167-11](http://dx.doi.org/10.1128/JB.05167-11).

511. Srivastava D, Waters CM. 2012. A tangled web: regulatory connections between quorum sensing and cyclic di-GMP. J Bacteriol 194:4485–4493. [http://dx.doi.org/10.1128/JB.00379-12](http://dx.doi.org/10.1128/JB.00379-12).

512. Bardill JP, Hammer BK. 2012. Non-coding sRNAs regulate virulence in the bacterial pathogen *Vibrio cholerae*. RNA Biol 9:392–401. [http://dx.doi.org/10.4161/rna.19975](http://dx.doi.org/10.4161/rna.19975).

513. Zhao X, Koestler BJ, Waters CM, Hammer BK. 2013. Post-transcriptional activation of a diguanylate cyclase by quorum sensing small RNAs promotes biofilm formation in *Vibrio cholerae*. Mol Microbiol 89:989–1002. [http://dx.doi.org/10.1111/mmi.12325](http://dx.doi.org/10.1111/mmi.12325).

514. Boyd CD, O’Toole GA. 2012. Second messenger regulation of biofilm formation: breakthroughs in understanding c-di-GMP effector systems. Annu Rev Cell Dev Biol 28:439–462. [http://dx.doi.org/10.1146/annurev-cellbio-101011-155705](http://dx.doi.org/10.1146/annurev-cellbio-101011-155705).

515. Martínez LC, Vadyvaloo V. 2014. Mechanisms of post-transcriptional gene regulation in bacterial biofilms. Front Cell Infect Microbiol 4:38. [http://dx.doi.org/10.3389/fcimb.2014.00038](http://dx.doi.org/10.3389/fcimb.2014.00038).

516. Williams TC, Ayrapetyan M, Oliver JD. 2014. Implications of chitin attachment for the environmental persistence and clinical nature of the human pathogen *Vibrio vulnificus*. Appl Environ Microbiol 80:1580–1507. [http://dx.doi.org/10.1128/AEM.03811-13](http://dx.doi.org/10.1128/AEM.03811-13).

517. Nahar S, Sultana M, Naser MN, Nair GB, Watanabe H, Ohnishi M, Yamamoto S, Endtz H, Cravioto A, Sack RB, Hasan NA, Sadique A, Huq A, Colwell RR, Alam M. 2012. Role of shrimp chitin in the ecology of toxigenic *Vibrio cholerae* and cholera transmission. Front Microbiol 2:260. [http://dx.doi.org/10.3389/fmicb.2011.00260](http://dx.doi.org/10.3389/fmicb.2011.00260).

518. Lutz C, Erken M, Noorian P, Sun S, McDougald D. 2013. Environmental reservoirs and mechanisms of persistence of *Vibrio cholerae*. Front Microbiol 4:375. [http://dx.doi.org/10.3389/fmicb.2013.00375](http://dx.doi.org/10.3389/fmicb.2013.00375).

519. Vezzulli L, Pezzati E, Stauder M, Stagnaro L, Venier P, Pruzzo C. 2015. Aquatic ecology of the oyster pathogens *Vibrio splendidus* and *Vibrio aestuarianus*. Environ Microbiol 17:1065–1080. [http://dx.doi.org/10.1111/1462-2920.12484](http://dx.doi.org/10.1111/1462-2920.12484).

520. Williams TC, Ayrapetyan M, Oliver JD. 2015. Identifying the molecular and physical factors that influence *Vibrio vulnificus* chitin attachment. Appl Environ Microbiol 81:6158–6165. [http://dx.doi.org/10.1128/AEM.00753-15](http://dx.doi.org/10.1128/AEM.00753-15).

521. Cao X, Studer SV, Wassarman K, Zhang Y, Ruby EG, Miyashiro T. 2012. The novel sigma factor-like regulator RpoQ controls luminescence, chitinase activity, and motility in *Vibrio fischeri*. mBio 3:e00285-11. [http://dx.doi.org/10.1128/mBio.00285-11](http://dx.doi.org/10.1128/mBio.00285-11).

522. Frischkorn KR, Stojanovski A, Paranjpye R. 2013. *Vibrio parahaemolyticus* type IV pili mediate interactions with diatom-derived chitin and point to an unexplored mechanism of environmental persistence. Environ Microbiol 15:1416–1427. [http://dx.doi.org/10.1111/1462-2920.12093](http://dx.doi.org/10.1111/1462-2920.12093).

523. Johnson CN. 2013. Fitness factors in vibrios: a mini-review. Microb Ecol 65:826–851. [http://dx.doi.org/10.1007/s00248-012-0168-x](http://dx.doi.org/10.1007/s00248-012-0168-x).

524. Hasan NA, Grim CJ, Lipp EK, Rivera IN, Chun J, Haley BJ, Taviani E, Choi SY, Hoq M, Munk AC, Brettin TS, Bruce D, Challacombe JF, Dettter JC, Han CS, Eisen JA, Huq A, Colwell RR. 2015. Deep-sea hydrothermal vent bacteria related to human pathogenic *Vibrio* species. Proc Natl Acad Sci U S A 112:E2813–E2819. [http://dx.doi.org/10.1073/pnas.1503928112](http://dx.doi.org/10.1073/pnas.1503928112).

525. Klose KE, Mekalanos JJ. 1998. Distinct roles of an alternative sigma factor during both free-swimming and colonizing phases of the Vibrio cholerae pathogenic cycle. J Bacteriol 180:5256–5259.

526. Krasteva PV, Fong JC, Shikuma NJ, Beyhan S, Navarro MV, Yildiz FH, Sondermann H. 2010. Vibrio cholerae VpsT regulates matrix production and motility by directly sensing cyclic di-GMP. Science 327: 866–868. http://dx.doi.org/10.1126/science.1181185.

527. Lo Scrudato M, Blokesch M. 2012. The regulatory network of natural competence and transformation of Vibrio cholerae. PLoS Genet 8:e1002778. http://dx.doi.org/10.1371/journal.pgen.1002778.

528. Dalia AB, Lazinski DW, Camilli A. 2014. Identification of a membrane-bound transcriptional regulator that links chitin and natural competence in Vibrio cholerae. mBio 5:e01028-13. http://dx.doi.org/10.1128/mBio.01028-13.

529. Haines-Menges B, Whitaker WB, Boyd EF. 2014. Alternative sigma factor RpoE is important for Vibrio parahaemolyticus cell envelope stress response and intestinal colonization. Infect Immun 82:3667–3677. http://dx.doi.org/10.1128/IAI.01854-14.

530. Lazazzera BA. 2010. The phosphoenolpyruvate phosphotransferase system: as important for biofilm formation by Vibrio cholerae as it is for metabolism in Escherichia coli. J Bacteriol 192:4083–4085. http://dx.doi.org/10.1128/JB.00641-10.

531. Pereira CS, Santos AJ, Bejerano-Sagie M, Correia PB, Marques JC, Xavier KB. 2012. Phosphoenolpyruvate phosphotransferase system regulates detection and processing of the quorum sensing signal autoinducer-2. Mol Microbiol 84:93–104. http://dx.doi.org/10.1111/j.1365-2958.2012.08010.x.

532. Deutscher J, Aké FM, Derkaoui M, Zébré AC, Cao TN, Bouraoui H, Kentache T, Mokhtari A, Milohanic E, Joyet P. 2014. The bacterial phosphoenolpyruvate:carbohydrate phosphotransferase system: regulation by protein phosphorylation and phosphorylation-dependent protein-protein interactions. Microbiol Mol Biol Rev 78:231–256. http://dx.doi.org/10.1128/MMBR.00001-14.

533. Mandel MJ, Schaefer AL, Brennan CA, Heath-Heckman EA, Deloney-Marino CR, McFall-Ngai MJ, Ruby EG. 2012. Squid-derived chitin oligosaccharides are a chemotactic signal during colonization by Vibrio fischeri. Appl Environ Microbiol 78:4620–4626. http://dx.doi.org/10.1128/AEM.00377-12.

534. Iwamoto K, Shiraiwa Y. 2005. Salt-regulated mannitol metabolism in algae. Mar Biotechnol 7:407–415. http://dx.doi.org/10.1007/s10126-005-0029-4.

535. Thompson FL, Iida T, Swings J. 2004. Biodiversity of vibrios. Microbiol Mol Biol Rev 68:403–431. http://dx.doi.org/10.1128/MMBR.68.3.403-431.2004.

536. Worden AZ, Seidel M, Smriga S, Wick A, Malfatti F, Bartlett D, Azam F. 2006. Trophic regulation of Vibrio cholerae in coastal marine waters. Environ Microbiol 8:21–29. http://dx.doi.org/10.1111/j.1462-2920.2005.00863.x.

537. Tinta T, Kogovšek T, Malej A, Turk V. 2012. Jellyfish modulate bacterial dynamic and community structure. PLoS One 7:e39274. http://dx.doi.org/10.1371/journal.pone.0039274.

538. Takemura AF, Chien DM, Polz MF. 2014. Associations and dynamics of Vibrionaceae in the environment, from the genus to the population level. Front Microbiol 5:38. http://dx.doi.org/10.3389/fmicb.2014.00038.

539. Hammer J, Kraak MH, Parsons JR. 2012. Plastics in the marine environment: the dark side of a modern gift. Rev Environ Contam Toxicol 220:1–44. http://dx.doi.org/10.1007/978-1-4614-3414-6_1.

540. Erikson M, Lebreton LC, Carson HS, Thiel M, Moore CJ, Bororro JC, Galgani F, Ryan PG, Reisser J. 2014. Plastic pollution in the world’s oceans: more than 5 trillion plastic pieces weighing over 250,000 tons afloat at sea. PLoS One 9:e111913. http://dx.doi.org/10.1371/journal.pone.0111913.

541. Zettler ER, Mincer TJ, Amaral-Zettler LA. 2013. Life in the “plastisphere”: microbial communities on plastic marine debris. Environ Sci Technol 47:7137–7146. http://dx.doi.org/10.1021/es401288x.

542. Sauret C, Böttjer D, Talarmin A, Guigue C, Conan P, Pujo-Pay M, Ghiglione JF. 2015. Top-down control of diesel-degrading prokaryotic communities. Microb Ecol 70:445–458. http://dx.doi.org/10.1007/s00248-015-0596-5.

543. Huggett MJ, Crocetti GR, Kjelleberg S, Steinberg PD. 2008. Recruitment of the sea urchin Heliocidaris erythrogramma and the distribution and abundance of inducing bacteria in the field. Aquat Microb Ecol 53: 161–171. http://dx.doi.org/10.3354/ame01239.

544. Gutierrez CK, Matsui GY, Lincoln DE, Lovell CR. 2009. Production of the phytohormone indole-3-acetic acid by estuarine species of the genus Vibrio. Appl Environ Microbiol 75:2253–2258. http://dx.doi.org/10.1128/AEM.02072-08.

545. Mazur H, Homme E. 1993. Presence of auxin indole-3-acetic acid in the northern Adriatic Sea: phytohormones and mucilage. Mar Ecol Prog Ser 99:163–168. http://dx.doi.org/10.3354/meps099163.

546. Alam M, Sultana M, Nair GB, Siddique AK, Hasan NA, Sack RB, Sack DA, Ahmed KU, Sadique A, Watanabe H, Grim CJ, Huq A, Colwell RR. 2007. Viable but nonculturable Vibrio cholerae O1 in biofilms in the aquatic environment and their role in cholera transmission. Proc Natl Acad Sci U S A 104:17801–17806. http://dx.doi.org/10.1073/pnas.0705599104.

547. Miller MC, Keymer DP, Avelar A, Boehm AB, Schoolnik GK. 2007. Detection and transformation of genome segments that differ within a coastal population of Vibrio cholerae strains. Appl Environ Microbiol 73:3695–3704. http://dx.doi.org/10.1128/AEM.02735-06.

548. Gutierrez West CK, Kline SL, Lovell CR. 2013. High frequency of virulence factor genes *tdh*, *trh*, and *tlh* in Vibrio parahaemolyticus strains isolated from a pristine estuary. Appl Environ Microbiol 79:2247–2252. http://dx.doi.org/10.1128/AEM.03792-12.

549. Klein SL, Gutierrez West CK, Mejia DM, Lovell CR. 2014. Genes similar to the Vibrio parahaemolyticus virulence-related genes *tdh*, *tlh*, and *vscC2* occur in other Vibrionaceae species isolated from a pristine estuary. Appl Environ Microbiol 80:595–602. http://dx.doi.org/10.1128/AEM.02895-13.

550. Hazen TH, Pan L, Gu JD, Sobecky PA. 2010. The contribution of mobile genetic elements to the evolution and ecology of vibrios. FEMS Microbiol Ecol 74:485–499. http://dx.doi.org/10.1111/j.1574-6941.2010.00937.x.

551. Rapa RA, Labbate M. 2013. The function of integron-associated gene cassettes in Vibrio species: the tip of the iceberg. Front Microbiol 4:385. http://dx.doi.org/10.3389/fmicb.2013.00385.

552. Seitz P, Blokesch M. 2013. Cues and regulatory pathways involved in natural competence and transformation in pathogenic and environmental Gram-negative bacteria. FEMS Microbiol Rev 37:336–363. http://dx.doi.org/10.1111/j.1574-6976.2012.00353.x.

553. Zhu J, Mekalanos JJ. 2003. Quorum sensing-dependent biofilms enhance colonization in Vibrio cholerae. Dev Cell 5:647–656. http://dx.doi.org/10.1016/S1534-5807(03)00295-8.

554. Kamruzzaman M, Udden SM, Cameron DE, Calderwood SB, Nair GB, Mekalanos JJ, Faruque SM. 2010. Quorum-regulated biofilms enhance the development of conditionally viable, environmental Vibrio cholerae. Proc Natl Acad Sci U S A 107:1588–1593. http://dx.doi.org/10.1073/pnas.0913404107.

555. Faruque SM, Biswas K, Udden SM, Ahmad QS, Sack DA, Nair GB, Mekalanos JJ. 2006. Transmissibility of cholera: *in vivo*-formed biofilms and their relationship to infectivity and persistence in the environment. Proc Natl Acad Sci U S A 103:6350–6355. http://dx.doi.org/10.1073/pnas.0601277103.

556. Tamayo R, Patimalla B, Camilli A. 2010. Growth in a biofilm induces a hyperinfectious phenotype in Vibrio cholerae. Infect Immun 78:3560–3569. http://dx.doi.org/10.1128/IAI.00048-10.

557. Bari SM, Roky MK, Mohiuddin M, Kamruzzaman M, Mekalanos JJ, Faruque SM. 2013. Quorum-sensing autoinducers resuscitate dormant Vibrio cholerae in environmental water samples. Proc Natl Acad Sci U S A 110:9926–9931. http://dx.doi.org/10.1073/pnas.1307697110.

558. Antonova ES, Hammer BK. 2011. Quorum-sensing autoinducer molecules produced by members of a multispecies biofilm promote horizontal gene transfer to Vibrio cholerae. FEMS Microbiol Lett 322:68–76. http://dx.doi.org/10.1111/j.1574-6968.2011.02328.x.

559. Sun S, Tay QX, Kjelleberg S, Rice SA, McDougald D. 2015. Quorum sensing-regulated chitin metabolism provides grazing resistance to Vibrio cholerae biofilms. ISME J 9:1812–1820. http://dx.doi.org/10.1038/ismej.2014.265.

560. Colwell RR. 1996. Global climate and infectious disease: the cholera paradigm. Science 274:2025–2031. http://dx.doi.org/10.1126/science.274.5295.2025.

561. Lipp EK, Huq A, Colwell RR. 2002. Effects of global climate on infectious disease: the cholera model. Clin Microbiol Rev 15:757–770. http://dx.doi.org/10.1128/CMR.15.4.757-770.2002.

562. Hall-Stoodley L, Stoodley P. 2005. Biofilm formation and dispersal and

the transmission of human pathogens. Trends Microbiol 13:7–10. http://dx.doi.org/10.1016/j.tim.2004.11.004.

563. Vezzulli L, Brettar I, Pezzati E, Reid PC, Colwell RR, Höfle MG, Pruzzo C. 2012. Long-term effects of ocean warming on the prokaryotic community: evidence from the vibrios. ISME J 6:21–30. http://dx.doi.org/10.1038/ismej.2011.89.

564. Vezzulli L, Colwell RR, Pruzzo C. 2013. Ocean warming and spread of pathogenic vibrios in the aquatic environment. Microb Ecol 65:817–825. http://dx.doi.org/10.1007/s00248-012-0163-2.

565. Kimes NE, Grim CJ, Johnson WR, Hasan NA, Tall BD, Kothary MH, Kiss H, Munk AC, Tapia R, Green L, Detter C, Bruce DC, Brettin TS, Colwell RR, Morris PJ. 2012. Temperature regulation of virulence factors in the pathogen Vibrio coralliilyticus. ISME J 6:835–846. http://dx.doi.org/10.1038/ismej.2011.154.

566. Tout J, Siboni N, Messer LF, Garren M, Stocker R, Webster NS, Ralph PJ, Seymour JR. 2015. Increased seawater temperature increases the abundance and alters the structure of natural Vibrio populations associated with the coral Pocillopora damicornis. Front Microbiol 6:432. http://dx.doi.org/10.3389/fmicb.2015.00432.

567. Rickard AH, Gilbert P, High NJ, Kolenbrander PE, Handley PS. 2003. Bacterial coaggregation: an integral process in the development of multi-species biofilms. Trends Microbiol 11:94–100. http://dx.doi.org/10.1016/S0966-842X(02)00034-3.

568. Chung HC, Lee OO, Huang YL, Mok SY, Kolter R, Qian PY. 2010. Bacterial community succession and chemical profiles of subtidal biofilms in relation to larval settlement of the polychaete Hydroides elegans. ISME J 4:817–828. http://dx.doi.org/10.1038/ismej.2009.157.

569. Fischer M, Friedrichs G, Lachnit T. 2014. Fluorescence-based quasi-continuous and in situ monitoring of biofilm formation dynamics in natural marine environments. Appl Environ Microbiol 80:3721–3728. http://dx.doi.org/10.1128/AEM.00298-14.

570. Baty AM, Frølund B, Geesev GG, Langille S, Quintero EJ, Suci PA, Weiner RM. 1996. Adhesion of biofilms to inert surfaces: a molecular level approach directed at the marine environment. Biofouling 10:111–121. http://dx.doi.org/10.1080/08927019609386274.

571. Compère C, Bellon-Fontaine MN, Bertrand P, Costa D, Marcus P, Poleunis C, Pradier CM, Rondot B, Walls MG. 2001. Kinetics of conditioning layer formation on stainless steel immersed in seawater. Biofouling 17:129–145. http://dx.doi.org/10.1080/0892701010109378472.

572. Bakker DP, Busscher HJ, van Zanten J, de Vries J, Klijnstra JW, van der Mei HC. 2004. Multiple linear regression analysis of bacterial deposition to polyurethane coatings after conditioning film formation in the marine environment. Microbiology 150:1779–1784. http://dx.doi.org/10.1099/mic.0.26983-0.

573. Jain A, Bhosle NB. 2009. Biochemical composition of the marine conditioning film: implications for bacterial adhesion. Biofouling 25:13–19. http://dx.doi.org/10.1080/08927010802411969.

574. Jones PR, Cottrell MT, Kirchman DL, Dexter SC. 2007. Bacterial community structure of biofilms on artificial surfaces in an estuary. Microb Ecol 53:153–162. http://dx.doi.org/10.1007/s00248-006-9154-5.

575. Andrews JS, Rolfe SA, Huang WE, Scholes JD, Banwart SA. 2010. Biofilm formation in environmental bacteria is influenced by different macromolecules depending on genus and species. Environ Microbiol 12:2496–2507. http://dx.doi.org/10.1111/j.1462-2920.2010.02223.x.

576. Camps M, Barani A, Gregori G, Bouchez A, Le Berre B, Bressy C, Blache Y, Briand JF. 2014. Antifouling coatings influence both abundance and community structure of colonizing biofilms: a case study in the northwestern Mediterranean Sea. Appl Environ Microbiol 80:4821–4831. http://dx.doi.org/10.1128/AEM.00948-14.

577. Bar-Zeev E, Berman-Frank I, Girshevitz O, Berman T. 2012. Revised paradigm of aquatic biofilm formation facilitated by microgel transparent exopolymer particles. Proc Natl Acad Sci U S A 109:9119–9124. http://dx.doi.org/10.1073/pnas.1203708109.

578. Watnick PI, Kolter R. 1999. Steps in the development of a Vibrio cholerae El Tor biofilm. Mol Microbiol 34:586–595. http://dx.doi.org/10.1046/j.1365-2958.1999.01624.x.

579. O’Toole G, Kaplan HB, Kolter R. 2000. Biofilm formation as microbial development. Annu Rev Microbiol 54:49–79. http://dx.doi.org/10.1146/annurev.micro.54.1.49.

580. Danhorn T, Fuqua C. 2007. Biofilm formation by plant-associated bacteria. Annu Rev Microbiol 61:401–422. http://dx.doi.org/10.1146/annurev.micro.61.080706.093316.

581. Mandlik A, Swierczynski A, Das A, Ton-That H. 2008. Pili in Gram-positive bacteria: assembly, involvement in colonization and biofilm development. Trends Microbiol 16:33–40. http://dx.doi.org/10.1016/j.tim.2007.10.010.

582. Amano A. 2010. Bacterial adhesins to host components in periodontitis. Periodontol 2000 52:12–37. http://dx.doi.org/10.1111/j.1600-0757.2009.00307.x.

583. Utada AS, Bennett RR, Fong JC, Gibiansky ML, Yildiz FH, Golestanian R, Wong GC. 2014. Vibrio cholerae use pili and flagella synergistically to effect motility switching and conditional surface attachment. Nat Commun 5:4913. http://dx.doi.org/10.1038/ncomms5913.

584. Okshevsky M, Meyer RL. 2015. The role of extracellular DNA in the establishment, maintenance and perpetuation of bacterial biofilms. Crit Rev Microbiol 41:341–352. http://dx.doi.org/10.3109/1040841X.2013.841639.

585. Ritter A, Com E, Bazire A, Goncalves MS, Delage L, Le Pennec G, Pineau C, Dreanno C, Compère C, Dufour A. 2012. Proteomic studies highlight outer-membrane proteins related to biofilm development in the marine bacterium Pseudoalteromonas sp. D41. Proteomics 12:3180–3192. http://dx.doi.org/10.1002/pmic.201100644.

586. Chan CS, Fakra SC, Emerson D, Fleming EJ, Edwards KJ. 2011. Lithotrophic iron-oxidizing bacteria produce organic stalks to control mineral growth: implications for biosignature formation. ISME J 5:717–727. http://dx.doi.org/10.1038/ismej.2010.173.

587. Salman V, Amann R, Girnth AC, Polerecky L, Bailey JV, Høgslund S, Jessen G, Pantoja S, Schulz-Vogt HN. 2011. A single-cell sequencing approach to the classification of large, vacuolated sulfur bacteria. Syst Appl Microbiol 34:243–259. http://dx.doi.org/10.1016/j.syapm.2011.02.001.

588. Seder-Colomina M, Morin G, Benzerara K, Ona-Nguema G, Pernelle J, Esposito G, van Hullebusch ED. 2014. Sphaerotilus natans, a neutrophilic iron-related sheath-forming bacterium: perspectives for metal remediation strategies. Geomicrobiol J 31:64–75. http://dx.doi.org/10.1080/01490451.2013.806611.

589. Giovannoni SJ, Stingl U. 2005. Molecular diversity and ecology of microbial plankton. Nature 437:343–348. http://dx.doi.org/10.1038/nature04158.

590. Stahl DA, Key R, Flesher B, Smit J. 1992. The phylogeny of marine and freshwater caulobacters reflects their habitat. J Bacteriol 174:2193–2198.

591. Langille SE, Weiner RM. 1998. Spatial and temporal deposition of Hyphomonas strain VP-6 capsules involved in biofilm formation. Appl Environ Microbiol 64:2906–2913.

592. Weiner RM, Melick M, O’Neill K, Quintero E. 2000. Hyphomonas adhaerens sp. nov., Hyphomonas johnsonii sp. nov. and Hyphomonas rosenbergii sp. nov., marine budding and prosthecate bacteria. Int J Syst Evol Microbiol 50:459–469. http://dx.doi.org/10.1099/00207713-50-2-459.

593. Moore RL, Marshall KC. 1981. Attachment and rosette formation by hyphomicrobia. Appl Environ Microbiol 42:751–757.

594. Buchan A, González JM, Moran MA. 2005. Overview of the marine Roseobacter lineage. Appl Environ Microbiol 71:5665–5677. http://dx.doi.org/10.1128/AEM.71.10.5665-5677.2005.

595. Geng H, Belas R. 2010. Molecular mechanisms underlying roseobacter-phytoplankton symbioses. Curr Opin Biotechnol 21:332–338. http://dx.doi.org/10.1016/j.copbio.2010.03.013.

596. Larkin JM, Nelson R. 1987. Mechanism of attachment of swarm cells of Thiothrix nivea. J Bacteriol 169:5877–5879.

597. Williams TM, Unz RF, Doman JT. 1987. Ultrastructure of Thiothrix spp. and “type 021N” bacteria. Appl Environ Microbiol 53:1560–1570.

598. Heindl JE, Wang Y, Heckel BC, Mohari B, Feirer N, Fuqua C. 2014. Mechanisms and regulation of surface interactions and biofilm formation in Agrobacterium. Front Plant Sci 5:176. http://dx.doi.org/10.3389/fpls.2014.00176.

599. Li G, Brown PJ, Tang JX, Xu J, Quardokus EM, Fuqua C, Brun YV. 2012. Surface contact stimulates the just-in-time deployment of bacterial adhesins. Mol Microbiol 83:41–51. http://dx.doi.org/10.1111/j.1365-2958.2011.07909.x.

600. Kirkpatrick CL, Viollier PH. 2012. Reflections on a sticky situation: how surface contact pulls the trigger for bacterial adhesion. Mol Microbiol 83:7–9. http://dx.doi.org/10.1111/j.1365-2958.2011.07913.x.

601. Anderson JK, Smith TG, Hoover TR. 2010. Sense and sensibility: flagellum-mediated gene regulation. Trends Microbiol 18:30–37. http://dx.doi.org/10.1016/j.tim.2009.11.001.

602. Zhu S, Kojima S, Homma M. 2013. Structure, gene regulation and

environmental response of flagella in Vibrio. Front Microbiol 4:410. http://dx.doi.org/10.3389/fmicb.2013.00410.

603. Thole S, Kalhoefer D, Voget S, Berger M, Engelhardt T, Liesegang H, Wollherr A, Kjelleberg S, Daniel R, Simon M, Thomas T, Brinkhoff T. 2012. Phaeobacter gallaeciensis genomes from globally opposite locations reveal high similarity of adaptation to surface life. ISME J 6:2229–2244. http://dx.doi.org/10.1038/ismej.2012.62.

604. Elifantz H, Horn G, Ayon M, Cohen Y, Minz D. 2013. Rhodobacteraceae are the key members of the microbial community of the initial biofilm formed in Eastern Mediterranean coastal seawater. FEMS Microbiol Ecol 85:348–357. http://dx.doi.org/10.1111/1574-6941.12122.

605. Bižić-Ionescu M, Zeder M, Ionescu D, Orlić S, Fuchs BM, Grossart HP, Amann R. 2015. Comparison of bacterial communities on limnic versus coastal marine particles reveals profound differences in colonization. Environ Microbiol 17:3500–3514. http://dx.doi.org/10.1111/1462-2920.12466.

606. Gifford SM, Sharma S, Moran MA. 2014. Linking activity and function to ecosystem dynamics in a coastal bacterioplankton community. Front Microbiol 5:185. http://dx.doi.org/10.3389/fmicb.2014.00185.

607. Singh RP, Reddy CR. 2014. Seaweed-microbial interactions: key functions of seaweed-associated bacteria. FEMS Microbiol Ecol 88:213–230. http://dx.doi.org/10.1111/1574-6941.12297.

608. Bland JA, Brock TD. 1973. The marine bacterium Leucothrix mucor as an algal epiphyte. Mar Biol 23:283–292. http://dx.doi.org/10.1007/BF00389335.

609. Miller TR, Belas R. 2006. Motility is involved in Silicibacter sp. TM1040 interaction with dinoflagellates. Environ Microbiol 8:1648–1659.

610. Moran MA, Belas R, Schell MA, González JM, Sun F, Sun S, Binder BJ, Edmonds J, Ye W, Orcutt B, Howard EC, Meile C, Palefsky W, Goesmann A, Ren Q, Paulsen I, Ulrich LE, Thompson LS, Saunders E, Buchan A. 2007. Ecological genomics of marine roseobacters. Appl Environ Microbiol 73:4559–4569. http://dx.doi.org/10.1128/AEM.02580-06.

611. Rath J, Wu KY, Herndl GJ, DeLong EF. 1998. High phylogenetic diversity in a marine-snow-associated bacterial assemblage. Aquat Microb Ecol 14:261–269. http://dx.doi.org/10.3354/ame014261.

612. D'Ambrosio L, Ziervogel K, MacGregor B, Teske A, Arnosti C. 2014. Composition and enzymatic function of particle-associated and free-living bacteria: a coastal/offshore comparison. ISME J 8:2167–2179. http://dx.doi.org/10.1038/ismej.2014.67.

613. Branda SS, Vik S, Friedman L, Kolter R. 2005. Biofilms: the matrix revisited. Trends Microbiol 13:20–26. http://dx.doi.org/10.1016/j.tim.2004.11.006.

614. Straight PD, Kolter R. 2009. Interspecies chemical communication in bacterial development. Annu Rev Microbiol 63:99–118. http://dx.doi.org/10.1146/annurev.micro.091208.073248.

615. Stacy AR, Diggle SP, Whiteley M. 2012. Rules of engagement: defining bacterial communication. Curr Opin Microbiol 15:155–161. http://dx.doi.org/10.1016/j.mib.2011.11.007.

616. Katharios-Lanwermeyer S, Xi C, Jakubovics NS, Rickard AH. 2014. Mini-review. Microbial coaggregation: ubiquity and implications for biofilm development. Biofouling 30:1235–1251. http://dx.doi.org/10.1080/08927014.2014.976206.

617. Adav SS, Lee DJ, Lai JY. 2008. Intergeneric coaggregation of strains isolated from phenol-degrading aerobic granules. Appl Microbiol Biotechnol 79:657–661. http://dx.doi.org/10.1007/s00253-008-1456-8.

618. Min KR, Rickard AH. 2009. Coaggregation by the freshwater bacterium Sphingomonas natatoria alters dual-species biofilm formation. Appl Environ Microbiol 75:3987–3997. http://dx.doi.org/10.1128/AEM.02843-08.

619. Almstrand R, Daims H, Persson F, Sörensson F, Hermansson M. 2013. New methods for analysis of spatial distribution and coaggregation of microbial populations in complex biofilms. Appl Environ Microbiol 79:5978–5987. http://dx.doi.org/10.1128/AEM.01727-13.

620. Simões LC, Simões M, Vieira MJ. 2008. Intergeneric coaggregation among drinking water bacteria: evidence of a role for Acinetobacter calcoaceticus as a bridging bacterium. Appl Environ Microbiol 74:1259–1263. http://dx.doi.org/10.1128/AEM.01747-07.

621. Vornhagen J, Stevens M, McCormick DW, Dowd SE, Eisenberg JN, Boles BR, Rickard AH. 2013. Coaggregation occurs amongst bacteria within and between biofilms in domestic showerheads. Biofouling 29:53–68. http://dx.doi.org/10.1080/08927014.2012.744395.

622. Cheng Z, Meng X, Wang H, Chen M, Li M. 2014. Isolation and characterization of broad spectrum coaggregating bacteria from different water systems for potential use in bioaugmentation. PLoS One 9:e94220. http://dx.doi.org/10.1371/journal.pone.0094220.

623. Little AE, Robinson CJ, Peterson SB, Raffa KF, Handelsman J. 2008. Rules of engagement: interspecies interactions that regulate microbial communities. Annu Rev Microbiol 62:375–401. http://dx.doi.org/10.1146/annurev.micro.030608.101423.

624. Caldwell DE, Atuku E, Wilkie DC, Wivcharuk KP, Karthikeyan S, Korber DR, Schmid DF, Wolfardt GM. 1997. Germ theory vs. community theory in understanding and controlling the proliferation of biofilms. Adv Dent Res 11:4–13. http://dx.doi.org/10.1177/08959374970110011501.

625. Rickard AH, Leach SA, Buswell CM, High NJ, Handley PS. 2000. Coaggregation between aquatic bacteria is mediated by specific-growth-phase-dependent lectin-saccharide interactions. Appl Environ Microbiol 66:431–434. http://dx.doi.org/10.1128/AEM.66.1.431-434.2000.

626. Min KR, Zimmer MN, Rickard AH. 2010. Physicochemical parameters influencing coaggregation between the freshwater bacteria Sphingomonas natatoria 2.1 and Micrococcus luteus 2.13. Biofouling 26:931–940. http://dx.doi.org/10.1080/08927014.2010.531128.

627. Caldwell DE, Costerton JW. 1996. Are bacterial biofilms constrained to Darwin’s concept of evolution through natural selection? Microbiologia 12:347–358.

628. McCormick DW, Stevens MRE, Boles BR, Rickard AH. 2011. Does it take two to tango? Importance of coaggregation in multispecies biofilms. Culture 32:1–5.

629. Mitri S, Foster KR. 2013. The genotypic view of social interactions in microbial communities. Annu Rev Genet 47:247–273. http://dx.doi.org/10.1146/annurev-genet-111212-133307.

630. Gruber-Dorninger C, Pester M, Kitzinger K, Savio DF, Loy A, Rattei T, Wagner M, Daims H. 2015. Functionally relevant diversity of closely related Nitrospira in activated sludge. ISME J 9:643–655. http://dx.doi.org/10.1038/ismej.2014.156.

631. Lücke S, Schwarz J, Gruber-Dorninger C, Spieck E, Wagner M, Daims H. 2015. Nitrotoga-like bacteria are previously unrecognized key nitrite oxidizers in full-scale wastewater treatment plants. ISME J 9:708–720. http://dx.doi.org/10.1038/ismej.2014.158.

632. Rotaru AE, Shrestha PM, Liu F, Markovaite B, Chen S, Nevin K, Lovley D. 2014. Direct interspecies electron transfer between Geobacter metallireducens and Methanosarcina barkeri. Appl Environ Microbiol 80:4599–4605. http://dx.doi.org/10.1128/AEM.00895-14.

633. Venkataraman A, Rosenbaum MA, Werner JJ, Winans SC, Angenent LT. 2014. Metabolite transfer with the fermentation product 2,3-butanediol enhances virulence by Pseudomonas aeruginosa. ISME J 8:1210–1220. http://dx.doi.org/10.1038/ismej.2013.232.

634. Ren D, Madsen JS, Sørensen SJ, Burmølle M. 2015. High prevalence of biofilm synergy among bacterial soil isolates in cocultures indicates bacterial interspecific cooperation. ISME J 9:81–89. http://dx.doi.org/10.1038/ismej.2014.96.

635. Zhang W, Liu M, Dai X. 2013. Biological characteristics and probiotic effect of Leuconostoc lactis strain isolated from the intestine of black porgy fish. Braz J Microbiol 44:685–691. http://dx.doi.org/10.1590/S1517-83822013005000053.

636. Sánchez-Ortiz AC, Luna-González A, Campa-Córdova Á, Escamilla-Montes IR, Flores-Miranda MD, Mazón-Suástegui JM. 2015. Isolation and characterization of potential probiotic bacteria from pustulose ark (Anadara tuberculosa) suitable for shrimp farming. Lat Am J Aquat Res 43:123–136. http://dx.doi.org/10.3856/vol43-issue1-fulltext-11.

637. Cornforth DM, Foster KR. 2013. Competition sensing: the social side of bacterial stress responses. Nat Rev Microbiol 11:285–293. http://dx.doi.org/10.1038/nrmicro2977.

638. Simões LC, Simões M, Vieira MJ. 2011. The effects of metabolite molecules produced by drinking water-isolated bacteria on their single and multispecies biofilms. Biofouling 27:685–699. http://dx.doi.org/10.1080/08927014.2011.597502.

639. Harrison F, Paul J, Massey RC, Buckling A. 2008. Interspecific competition and siderophore-mediated cooperation in Pseudomonas aeruginosa. ISME J 2:49–55. http://dx.doi.org/10.1038/ismej.2007.96.

640. Purschke FG, Hiller E, Trick I, Rupp S. 2012. Flexible survival strategies of Pseudomonas aeruginosa in biofilms result in increased fitness compared with Candida albicans. Mol Cell Proteomics 11:1652–1669. http://dx.doi.org/10.1074/mcp.M112.017673.

641. Trejo-Hernández A, Andrade-Domínguez A, Hernández M, Encar-

Dang and Lovell

nación S. 2014. Interspecies competition triggers virulence and mutability in Candida albicans-Pseudomonas aeruginosa mixed biofilms. ISME J 8:1974–1988. http://dx.doi.org/10.1038/ismej.2014.53.

642. Street JH, Paytan A. 2005. Iron, phytoplankton growth, and the carbon cycle. Met Ions Biol Syst 43:153–193.

643. Johnson KS, Gordon RM, Coale KH. 1997. What controls dissolved iron concentrations in the world ocean? Mar Chem 57:137–161. http://dx.doi.org/10.1016/S0304-4203(97)00043-1.

644. Martin JH, Fitzwater SE. 1988. Iron deficiency limits phytoplankton growth in the north-east Pacific subarctic. Nature 331:341–343. http://dx.doi.org/10.1038/331341a0.

645. Boyd PW, Watson AJ, Law CS, Abraham ER, Trull T, Murdoch R, Bakker DC, Bowie AR, Buesseler KO, Chang H, Charette M, Croot P, Downing K, Frew R, Gall M, Hadfield M, Hall J, Harvey M, Jameson G, LaRoche J, Liddicoat M, Ling R, Maldonado MT, McKay RM, Nodder S, Pickmere S, Pridmore R, Rintoul S, Safi K, Sutton P, Strzepek R, Tanneberger K, Turner S, Waite A, Zeldis J. 2000. A mesoscale phytoplankton bloom in the polar Southern Ocean stimulated by iron fertilization. Nature 407:695–702. http://dx.doi.org/10.1038/35037500.

646. Tortell PD, Maldonado MT, Price N. 1996. The role of heterotrophic bacteria in iron-limited ocean ecosystems. Nature 383:330–332. http://dx.doi.org/10.1038/383330a0.

647. Moore CM, Mills MM, Arrigo KR, Berman-Frank I, Bopp L, Boyd PW, Galbraith ED, Geider RJ, Guieu C, Jaccard SL, Jickells TD, La Roche J, Lenton TM, Mahowald NM, Marañón E, Marinov I, Moore JK, Nakatsuka T, Oschlies A, Saito MA, Thingstad TF, Tsuda A, Ulloa O. 2013. Processes and patterns of oceanic nutrient limitation. Nat Geosci 6:701–710. http://dx.doi.org/10.1038/ngeo1765.

648. Rodriguez F, Lillington J, Johnson S, Timmel CR, Lea SM, Berks BC. 2014. Crystal structure of the Bacillus subtilis phosphodiesterase PhoD reveals an iron and calcium-containing active site. J Biol Chem 289: 30889–30899. http://dx.doi.org/10.1074/jbc.M114.604892.

649. Yong SC, Roversi P, Lillington J, Rodriguez F, Krehenbrink M, Zeldin OB, Garman EF, Lea SM, Berks BC. 2014. A complex iron-calcium cofactor catalyzing phototransfer chemistry. Science 345:1170–1173. http://dx.doi.org/10.1126/science.1254237.

650. Moore JK, Doney SC, Glover DM, Fung IY. 2002. Iron cycling and nutrient-limitation patterns in surface waters of the world ocean. Deep Sea Res II 49:463–507. http://dx.doi.org/10.1016/S0967-0637(01)00074-7.

651. Hopkinson BM, Morel FM. 2009. The role of siderophores in iron acquisition by photosynthetic marine microorganisms. Biometals 22: 659–669. http://dx.doi.org/10.1007/s10534-009-9235-2.

652. Sigman DM, Boyle EA. 2000. Glacial/interglacial variations in atmospheric carbon dioxide. Nature 407:859–869. http://dx.doi.org/10.1038/35038000.

653. Behrenfeld MJ, Milligan AJ. 2013. Photophysiological expressions of iron stress in phytoplankton. Annu Rev Mar Sci 5:217–246. http://dx.doi.org/10.1146/annurev-marine-121211-172356.

654. Boyd PW, Jickells T, Law CS, Blain S, Boyle EA, Buesseler KO, Coale KH, Cullen JJ, de Baar HJ, Follows M, Harvey M, Lancelot C, Levasseur M, Owens NP, Pollard R, Rivkin RB, Sarmiento J, Schoemann V, Smetacek V, Takeda S, Tsuda A, Turner S, Watson AJ. 2007. Mesoscale iron enrichment experiments 1993-2005: synthesis and future directions. Science 315:612–617. http://dx.doi.org/10.1126/science.1131669.

655. Ellwood MJ, Nodder SD, King AL, Hutchins DA, Wilhelm SW, Boyd PW. 2014. Pelagic iron cycling during the subtropical spring bloom, east of New Zealand. Mar Chem 160:18–33. http://dx.doi.org/10.1016/j.marchem.2014.01.004.

656. Riemann L, Farnelid H, Steward GF. 2010. Nitrogenase genes in non-cyanobacterial plankton: prevalence, diversity and regulation in marine waters. Aquat Microb Ecol 61:235–247. http://dx.doi.org/10.3354/ame01431.

657. Barbeau K, Rue EL, Bruland KW, Butler A. 2001. Photochemical cycling of iron in the surface ocean mediated by microbial iron(III)-binding ligands. Nature 413:409–413. http://dx.doi.org/10.1038/35096545.

658. Collins AJ, Fullmer MS, Gogarten JP, Nyholm SV. 2015. Comparative genomics of Roseobacter clade bacteria isolated from the accessory nidalmental gland of Euprymna scolopes. Front Microbiol 6:123. http://dx.doi.org/10.3389/fmicb.2015.00123.

659. Wang Q, Liu Q, Ma Y, Rui H, Zhang Y. 2007. LuxO controls extracellular protease, haemolytic activities and siderophore production in fish pathogen Vibrio alginolyticus. J Appl Microbiol 103:1525–1534. http://dx.doi.org/10.1111/j.1365-2672.2007.03380.x.

660. Wen Y, Kim IH, Son JS, Lee BH, Kim KS. 2012. Iron and quorum sensing coordinately regulate the expression of vulnibactin biosynthesis in Vibrio vulnificus. J Biol Chem 287:26727–26739. http://dx.doi.org/10.1074/jbc.M112.374165.

661. Pereira CS, Thompson JA, Xavier KB. 2013. AI-2-mediated signalling in bacteria. FEMS Microbiol Rev 37:156–181. http://dx.doi.org/10.1111/j.1574-6976.2012.00345.x.

662. Frangipani E, Visaggio D, Heeb S, Kaever V, Câmara M, Visca P, Imperi F. 2014. The Gac/Rsm and cyclic-di-GMP signalling networks coordinately regulate iron uptake in Pseudomonas aeruginosa. Environ Microbiol 16:676–688. http://dx.doi.org/10.1111/1462-2920.12164.

663. Strehmel J, Neidig A, Nusser M, Geffers R, Brenner-Weiss G, Overhage J. 2015. Sensor kinase PA4398 modulates swarming motility and biofilm formation in Pseudomonas aeruginosa PA14. Appl Environ Microbiol 81:1274–1285. http://dx.doi.org/10.1128/AEM.02832-14.

664. Noinaj N, Guillier M, Barnard TJ, Buchanan SK. 2010. TonB-dependent transporters: regulation, structure, and function. Annu Rev Microbiol 64:43–60. http://dx.doi.org/10.1146/annurev.micro.112408.134247.

665. Schalk IJ, Mislin GL, Brillet K. 2012. Structure, function and binding selectivity and stereoselectivity of siderophore-iron outer membrane transporters. Curr Top Membr 69:37–66. http://dx.doi.org/10.1016/B978-0-12-394390-3.00002-1.

666. Quatrini R, Jedlicki E, Holmes DS. 2005. Genomic insights into the iron uptake mechanisms of the biomining microorganism Acidithiobacillus ferrooxidans. J Ind Microbiol Biotechnol 32:606–614. http://dx.doi.org/10.1007/s10295-005-0233-2.

667. Beyersmann PG, Chertkov O, Petersen J, Fiebig A, Chen A, Pati A, Ivanova N, Lapidus A, Goodwin LA, Chain P, Detter JC, Rohde M, Gronow S, Kyrpides NC, Woyke T, Simon M, Göker M, Klenc HP, Brinkhoff T. 2013. Genome sequence of Phaeobacter caeruleus type strain (DSM 24564T), a surface-associated member of the marine Roseobacter clade. Stand Genomic Sci 8:403–419. http://dx.doi.org/10.4056/sigs.3927626.

668. Cordero OX, Ventouras LA, Delong EF, Polz MF. 2012. Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations. Proc Natl Acad Sci U S A 109:20059–20064. http://dx.doi.org/10.1073/pnas.1213344109.

669. Keller L, Surette MG. 2006. Communication in bacteria: an ecological and evolutionary perspective. Nat Rev Microbiol 4:249–258. http://dx.doi.org/10.1038/nrmicro1383.

670. Katzianer DS, Wang H, Carey RM, Zhu J. 2015. “Quorum non-sensing”: social cheating and deception in Vibrio cholerae. Appl Environ Microbiol 81:3856–3862. http://dx.doi.org/10.1128/AEM.00586-15.

671. Popat R, Crusz SA, Messina M, Williams P, West SA, Diggle SP. 2012. Quorum-sensing and cheating in bacterial biofilms. Proc Biol Sci 279: 4765–4771. http://dx.doi.org/10.1098/rspb.2012.1976.

672. Dunny GM, Brickman TJ, Dworkin M. 2008. Multicellular behavior in bacteria: communication, cooperation, competition and cheating. Bioessays 30:296–298. http://dx.doi.org/10.1002/bies.20740.

673. Lee W, van Baalen M, Jansen VA. 2012. An evolutionary mechanism for diversity in siderophore-producing bacteria. Ecol Lett 15:119–125. http://dx.doi.org/10.1111/j.1461-0248.2011.01717.x.

674. Penn AS, Conibear TC, Watson RA, Kraaijeveld AR, Webb JS. 2012. Can Simpson’s paradox explain co-operation in Pseudomonas aeruginosa biofilms? FEMS Immunol Med Microbiol 65:226–235. http://dx.doi.org/10.1111/j.1574-695X.2012.00970.x.

675. Drescher K, Nadell CD, Stone HA, Wingreen NS, Bassler BL. 2014. Solutions to the public goods dilemma in bacterial biofilms. Curr Biol 24:50–55. http://dx.doi.org/10.1016/j.cub.2013.10.030.

676. Ross-Gillespie A, Kümmerli R. 2014. Collective decision-making in microbes. Front Microbiol 5:54. http://dx.doi.org/10.3389/fmicb.2014 .00054.

677. Boyle KE, Heilmann S, van Ditmarsch D, Xavier JB. 2013. Exploiting social evolution in biofilms. Curr Opin Microbiol 16:207–212. http://dx.doi.org/10.1016/j.mib.2013.01.003.

678. Zhang QG, Buckling A, Ellis RJ, Godfray HC. 2009. Coevolution between cooperators and cheats in a microbial system. Evolution 63: 2248–2256. http://dx.doi.org/10.1111/j.1558-5646.2009.00708.x.

679. Hibbing ME, Fuqua C, Parsek MR, Peterson SB. 2010. Bacterial com-
petition: surviving and thriving in the microbial jungle. Nat Rev Micro-
biol 8:15–25. http://dx.doi.org/10.1038/nrmicro2259.
680. Long RA, Azam F. 2001. Antagonistic interactions among marine pe-
lagic bacteria. Appl Environ Microbiol 67:4975–4983. http://dx.doi.org
/10.1128/AEM.67.11.4975-4983.2001.
681. Grossart HP, Schlingloff A, Bernhard M, Simon M, Brinkhoff T. 2004. Antagonistic activity of bacteria isolated from organic aggregates of the German Wadden Sea. FEMS Microbiol Ecol 47:387–396. http://dx.doi .org/10.1016/S0168-6496(03)00305-2.
682. Christie-Oleza JA, Piña-Villalonga JM, Bosch R, Nogales B, Armen- gaud J. 2012. Comparative proteogenomics of twelve Roseobacter exo- proteomes reveals different adaptive strategies among these marine bac- teria. Mol Cell Proteomics 11:M111.013110. http://dx.doi.org/10.1074 /mcp.M111.013110.
683. Geng H, Belas R. 2010. Expression of tropodithietic acid biosynthesis is controlled by a novel autoinducer. J Bacteriol 192:4377–4387. http://dx .doi.org/10.1128/JB.00410-10.
684. Berger M, Neumann A, Schulz S, Simon M, Brinkhoff T. 2011. Tropodithietic acid production in *Phaeobacter gallaeciensis* is regulated by N-acyl homoserine lactone-mediated quorum sensing. J Bacteriol 193:6576–6585. http://dx.doi.org/10.1128/JB.05818-11.
685. Cude WN, Prevette CW, Hadden MK, May AL, Smith RT, Swain CL, Campagna SR, Buchan A. 2015. *Phaeobacter* sp. strain Y4I utilizes two separate cell-to-cell communication systems to regulate production of the antimicrobial indigoidine. Appl Environ Microbiol 81:1417–1425. http://dx.doi.org/10.1128/AEM.02551-14.
686. Sockett RE. 2009. Predatory lifestyle of *Bdellovibrio bacteriovorus*. Annu Rev Microbiol 63:523–539. http://dx.doi.org/10.1146/annurev.micro .091208.073346.
687. Crossman LC, Chen H, Cerdeño-Tárraga AM, Brooks K, Quail MA, Pineiro SA, Hobley L, Sockett RE, Bentley SD, Parkhill J, Williams HN, Stine OC. 2013. A small predatory core genome in the divergent marine *Bacteriovorax marinus* and the terrestrial *Bdellovibrio bacterio- vorus*. ISME J 7:148–160. http://dx.doi.org/10.1038/ismej.2012.90.
688. Rice TD, Williams HN, Turng BF. 1998. Susceptibility of bacteria in estuarine environments to autochthonous bdellovibrios. Microb Ecol 35:256–264. http://dx.doi.org/10.1007/s002489900081.
689. Chauhan A, Williams HN. 2008. Biostimulation of estuarine microbi- ota on substrate coated agar slides: a novel approach to study diversity of autochthonous *Bdellovibrio*- and like organisms. Microb Ecol 55:640– 650. http://dx.doi.org/10.1007/s00248-007-9307-1.
690. Williams HN, Turng BF, Kelley JI. 2009. Survival response of *Bacterio- vorax* in surface biofilm versus suspension when stressed by extremes in environmental conditions. Microb Ecol 58:474–484. http://dx.doi.org /10.1007/s00248-009-9499-7.
691. Lambert C, Hobley L, Chang CY, Fenton A, Capeness M, Sockett L. 2009. A predatory patchwork: membrane and surface structures of *Bdell- ovibrio bacteriovorus*. Adv Microb Physiol 54:313–361. http://dx.doi.org /10.1016/S0065-2911(08)00005-2.
692. Lambert C, Fenton AK, Hobley L, Sockett RE. 2011. Predatory *Bdell- ovibrio* bacteria use gliding motility to scout for prey on surfaces. J Bac- teriol 193:3139–3141. http://dx.doi.org/10.1128/JB.00224-11.
693. Evans KJ, Lambert C, Sockett RE. 2007. Predation by *Bdellovibrio bacteriovorus* HD100 requires type IV pili. J Bacteriol 189:4850–4859. http://dx.doi.org/10.1128/JB.01942-06.
694. Chanyi RM, Koval SF. 2014. Role of type IV pili in predation by *Bdell- ovibrio bacteriovorus*. PLoS One 9:e113404. http://dx.doi.org/10.1371 /journal.pone.0113404.
695. Pineiro SA, Stine OC, Chauhan A, Steyert SR, Smith R, Williams HN. 2007. Global survey of diversity among environmental saltwater *Bacte- riovoracaceae*. Environ Microbiol 9:2441–2450. http://dx.doi.org/10 .1111/j.1462-2920.2007.01362.x.
696. Pineiro S, Chauhan A, Berhane TK, Athar R, Zheng G, Wang C, Dickerson T, Liang X, Lymeropoulou DS, Chen H, Christman M, Louime C, Babiker W, Stine OC, Williams HN. 2013. Niche partition of *Bacteriovorax* operational taxonomic units along salinity and temporal gradients in the Chesapeake Bay reveals distinct estuarine strains. Microb Ecol 65:652–660. http://dx.doi.org/10.1007/s00248-013-0186-3.
697. Li N, Williams HN. 2015. 454 pyrosequencing reveals diversity of *Bdell- ovibrio* and like organisms in fresh and salt water. Antonie Van Leeuwen- hoek 107:305–311. http://dx.doi.org/10.1007/s10482-014-0327-9.
698. Rogosky AM, Moak PL, Emmert EA. 2006. Differential predation by Marine Biofilm Development
Bdellovibrio bacteriovorus 109J. Curr Microbiol 52:81–85. http://dx.doi .org/10.1007/s00284-005-0038-6.
699. Chen H, Athar R, Zheng G, Williams HN. 2011. Prey bacteria shape the community structure of their predators. ISME J 5:1314–1322. http://dx .doi.org/10.1038/ismej.2011.4.
700. Pasternak Z, Njagi M, Shani Y, Chanyi R, Rotem O, Lurie-Weinberger MN, Koval S, Pietrokovski S, Gophna U, Jurkevitch E. 2014. In and out: an analysis of epibiotic vs periplasmic bacterial predators. ISME J 8:625–635. http://dx.doi.org/10.1038/ismej.2013.164.
701. Jurkevitch E. 2007. Predatory prokaryotes: biology, ecology, and evolu- tion. Springer, New York, NY.
702. Richards GP, Fay JP, Dickens KA, Parent MA, Soroka DS, Boyd EF. 2012. Predatory bacteria as natural modulators of *Vibrio parahaemolyti- cus* and *Vibrio vulnificus* in seawater and oysters. Appl Environ Microbiol 78:7455–7466. http://dx.doi.org/10.1128/AEM.01594-12.
703. Ruhe ZC, Low DA, Hayes CS. 2013. Bacterial contact-dependent growth inhibition. Trends Microbiol 21:230–237. http://dx.doi.org/10 .1016/j.tim.2013.02.003.
704. Rendueles O, Beloin C, Latour-Lambert P, Ghigo JM. 2014. A new biofilm-associated colicin with increased efficiency against biofilm bac- teria. ISME J 8:1275–1288. http://dx.doi.org/10.1038/ismej.2013.238.
705. Hayes CS, Aoki SK, Low DA. 2010. Bacterial contact-dependent deliv- ery systems. Annu Rev Genet 44:71–90. http://dx.doi.org/10.1146 /annurev.genet.42.110807.091449.
706. Hayes CS, Koskineniemi S, Ruhe ZC, Poole SJ, Low DA. 2014. Mecha- nisms and biological roles of contact-dependent growth inhibition sys- tems. Cold Spring Harb Perspect Med 4:a010025. http://dx.doi.org/10 .1101/cshperspect.a010025.
707. Aoki SK, Diner EJ, de Roodenbeke CT, Burgess BR, Poole SJ, Braaten BA, Jones AM, Webb JS, Hayes CS, Cotter PA, Low DA. 2010. A widespread family of polymorphic contact-dependent toxin delivery systems in bacteria. Nature 468:439–442. http://dx.doi.org/10.1038 /nature09490.
708. Koskineniemi S, Lamoureux JG, Nikolakakis KC, t’Kint de Roodenbeke C, Kaplan MD, Low DA, Hayes CS. 2013. Rhs proteins from diverse bacteria mediate intercellular competition. Proc Natl Acad Sci U S A 110:7032–7037. http://dx.doi.org/10.1073/pnas.1300627110.
709. Schreier HJ, Schott EJ. 2014. Draft genome sequence of the oyster larval probiotic bacterium *Vibrio* sp. strain OY15. Genome Announc 2(5): e01006-14. http://dx.doi.org/10.1128/genomeA.01006-14.
710. Bingle LE, Bailey CM, Pallen MJ. 2008. Type VI secretion: a beginner’s guide. Curr Opin Microbiol 11:3–8. http://dx.doi.org/10.1016/j.mib .2008.01.006.
711. Basler M, Ho BT, Mekalanos JJ. 2013. Tit-for-tat: type VI secretion system counterattack during bacterial cell-cell interactions. Cell 152: 884–894. http://dx.doi.org/10.1016/j.cell.2013.01.042.
712. Miller JF. 2013. Gaming the competition in microbial cell-cell interac- tions. EMBO J 32:778–780. http://dx.doi.org/10.1038/emboj.2013.45.
713. Silverman JM, Brunet YR, Cascales E, Mougous JD. 2012. Structure and regulation of the type VI secretion system. Annu Rev Microbiol 66:453–472. http://dx.doi.org/10.1146/annurev-micro-121809-151619.
714. Silverman JM, Austin LS, Hsu F, Hicks KG, Hood RD, Mougous JD. 2011. Separate inputs modulate phosphorylation-dependent and -inde- pendent type VI secretion activation. Mol Microbiol 82:1277–1290. http: //dx.doi.org/10.1111/j.1365-2958.2011.07889.x.
715. Boyer F, Fichant G, Berthod J, Vandenbrouck Y, Attree I. 2009. Dissecting the bacterial type VI secretion system by a genome wide in silico analysis: what can be learned from available microbial genomic resources? BMC Genomics 10:104. http://dx.doi.org/10.1186/1471-2164 -10-104.
716. Solomon D, Gonzalez H, Updegraff BL, Orth K. 2013. *Vibrio parah- aemolyticus* type VI secretion system 1 is activated in marine conditions to target bacteria, and is differentially regulated from system 2. PLoS One 8:e61086. http://dx.doi.org/10.1371/journal.pone.0061086.
717. Schwarz S, West TE, Boyer F, Chiang WC, Carl MA, Hood RD, Rohmer L, Tolker-Nielsen T, Skerrett SJ, Mougous JD. 2010. *Burk- holderia* type VI secretion systems have distinct roles in eukaryotic and bacterial cell interactions. PLoS Pathog 6:e1001068. http://dx.doi.org/10 .1371/journal.ppat.1001068.
718. Russell AB, Peterson SB, Mougous JD. 2014. Type VI secretion system effectors: poisons with a purpose. Nat Rev Microbiol 12:137–148. http: //dx.doi.org/10.1038/nrmicro3185.
719. Unterweger D, Miyata ST, Bachmann V, Brooks TM, Mullins T, March 2016 Volume 80 Number 1 Microbiology and Molecular Biology Reviews mmbr.asm.org 133
Dang and Lovell

720. Kostiuk B, Provenzano D, Pukatzki S. 2014. The Vibrio cholerae type VI secretion system employs diverse effector modules for intraspecific competition. Nat Commun 5:3549. http://dx.doi.org/10.1038/ncomms4549.

721. Tian Y, Zhao Y, Wu X, Liu F, Hu B, Walcott RR. 2015. The type VI protein secretion system contributes to biofilm formation and seed-to-seedling transmission of Acidovorax citrulli on melon. Mol Plant Pathol 16:38–47. http://dx.doi.org/10.1111/mpp.12159.

722. Fu Y, Waldor MK, Mekalanos JJ. 2013. Tn-Seq analysis of Vibrio cholerae intestinal colonization reveals a role for T6SS-mediated antibacterial activity in the host. Cell Host Microbe 14:652–663. http://dx.doi.org/10.1016/j.chom.2013.11.001.

723. Ishikawa T, Rompikuntal PK, Lindmark B, Milton DL, Wai SN. 2009. Quorum sensing regulation of the two *hcp* alleles in Vibrio cholerae O1 strains. PLoS One 4:e6734. http://dx.doi.org/10.1371/journal.pone.0006734.

724. Zheng J, Shin OS, Cameron DE, Mekalanos JJ. 2010. Quorum sensing and a global regulator TsrA control expression of type VI secretion and virulence in Vibrio cholerae. Proc Natl Acad Sci U S A 107:21128–21133. http://dx.doi.org/10.1073/pnas.1014998107.

725. Dong TG, Mekalanos JJ. 2012. Characterization of the RpoN regulon reveals differential regulation of T6SS and new flagellar operons in Vibrio cholerae O37 strain V52. Nucleic Acids Res 40:7766–7775. http://dx.doi.org/10.1093/nar/gks567.

726. Miyata ST, Bachmann V, Pukatzki S. 2013. Type VI secretion system regulation as a consequence of evolutionary pressure. J Med Microbiol 62:663–676. http://dx.doi.org/10.1099/jmm.0.053983-0.

727. Salomon D, Klimko JA, Orth K. 2014. H-NS regulates the Vibrio parahaemolyticus type VI secretion system 1. Microbiology 160:1867–1873. http://dx.doi.org/10.1099/mic.0.080028-0.

728. Shao Y, Bassler BL. 2014. Quorum regulatory small RNAs repress type VI secretion in Vibrio cholerae. Mol Microbiol 92:921–930. http://dx.doi.org/10.1111/mmi.12599.

729. Fuqua C, Parsek MR, Greenberg EP. 2001. Regulation of gene expression by cell-to-cell communication: acyl-homoserine lactone quorum sensing. Annu Rev Genet 35:439–468. http://dx.doi.org/10.1146/annurev.genet.35.102401.090913.

730. Borgeaud S, Metzger LC, Scrignari T, Blokesch M. 2015. The type VI secretion system of Vibrio cholerae fosters horizontal gene transfer. Science 347:63–67. http://dx.doi.org/10.1126/science.1260064.

731. Kapitein N, Mogk A. 2013. Deadly syringes: type VI secretion system activities in pathogenicity and interbacterial competition. Curr Opin Microbiol 16:52–58. http://dx.doi.org/10.1016/j.mib.2012.11.009.

732. Ho BT, Dong TG, Mekalanos JJ. 2014. A view to a kill: the bacterial type VI secretion system. Cell Host Microbe 15:9–21. http://dx.doi.org/10.1016/j.chom.2013.11.008.

733. Persson OP, Pinhassi J, Riemann L, Marklund B, Rhen M, Normark S, González JM, Hagström Å. 2009. High abundance of virulence gene homologues in marine bacteria. Environ Microbiol 11:1348–1357. http://dx.doi.org/10.1111/j.1462-2920.2008.01861.x.

734. Russell AB, Wexler AG, Harding BN, Whitney JC, Bohn AJ, Goo YA, Tran BQ, Barry NA, Zheng H, Peterson SB, Chou S, Gonen T, Goodlett DR, Goodman AL, Mougous JD. 2014. A type VI secretion-related pathway in Bacteroidetes mediates interbacterial antagonism. Cell Host Microbe 16:227–236. http://dx.doi.org/10.1016/j.chom.2014.07.007.

735. Salomon D, Kinch LN, Trudgian DC, Guo X, Klimko JA, Grishin NV, Mirzaei H, Orth K. 2014. Marker for type VI secretion system effectors. Proc Natl Acad Sci U S A 111:9271–9276. http://dx.doi.org/10.1073/pnas.1406110111.

736. Egan F, Reen FJ, O’Gara F. 2015. Tle distribution and diversity in metagenomic datasets reveal niche specialization. Environ Microbiol Rep 7:194–203. http://dx.doi.org/10.1111/1758-2229.12222.

737. Wilson MM, Anderson DE, Bernstein HD. 2015. Analysis of the outer membrane proteome and secretome of Bacteroides fragilis reveals a multiplicity of secretion mechanisms. PLoS One 10:e0117732. http://dx.doi.org/10.1371/journal.pone.0117732.

738. Biller SJ, Schubotz F, Roggensack SE, Thompson AW, Summons RE, Chisholm SW. 2014. Bacterial vesicles in marine ecosystems. Science 343:183–186. http://dx.doi.org/10.1126/science.1243457.

739. Nevot M, Deroncelé V, Messner P, Guinea J, Mercadé E. 2006. Characterization of outer membrane vesicles released by the psychrotolerant bacterium Pseudoalteromonas antarctica NF3. Environ Microbiol 8:1523–1533. http://dx.doi.org/10.1111/j.1462-2920.2006.01043.x.

740. Frias A, Manresa A, de Oliveira E, López-Iglesias C, Mercade E. 2010. Membrane vesicles: a common feature in the extracellular matter of cold-adapted Antarctic bacteria. Microb Ecol 59:476–486. http://dx.doi.org/10.1007/s00248-009-9622-9.

741. Scanlan D. 2014. Bacterial vesicles in the ocean. Science 343:143–144. http://dx.doi.org/10.1126/science.1248566.

742. Soler N, Krupovic M, Marguet E, Forterre P. 2015. Membrane vesicles in natural environments: a major challenge in viral ecology. ISME J 9:793–796. http://dx.doi.org/10.1038/ismej.2014.184.

743. Berleman J, Auer M. 2013. The role of bacterial outer membrane vesicles for intra- and interspecies delivery. Environ Microbiol 15:347–354. http://dx.doi.org/10.1111/1462-2920.12048.

744. Deatherage BL, Cookson BT. 2012. Membrane vesicle release in bacteria, eukaryotes, and archaea: a conserved yet underappreciated aspect of microbial life. Infect Immun 80:1948–1957. http://dx.doi.org/10.1128/IAI.06014-11.

745. Kulp A, Kuehn MJ. 2010. Biological functions and biogenesis of secreted bacterial outer membrane vesicles. Annu Rev Microbiol 64:163–184. http://dx.doi.org/10.1146/annurev.micro.091208.073413.

746. Bager RJ, Persson G, Nesta B, Soriani M, Serino L, Jeppsson M, Nielsen TK, Bojesen AM. 2013. Outer membrane vesicles reflect environmental cues in Gallibacterium anatis. Vet Microbiol 167:565–572. http://dx.doi.org/10.1016/j.vetmic.2013.09.005.

747. Baumgarten T, Sperling S, Seifert J, von Bergen M, Steiniger F, Wick LY, Heipieper HJ. 2012. Membrane vesicle formation as a multiple-stress response mechanism enhances Pseudomonas putida DOT-T1E cell surface hydrophobicity and biofilm formation. Appl Environ Microbiol 78:6217–6224. http://dx.doi.org/10.1128/AEM.01525-12.

748. Bonnington KE, Kuehn MJ. 2014. Protein selection and export via outer membrane vesicles. Biochim Biophys Acta 1843:1612–1619. http://dx.doi.org/10.1016/j.bbamcr.2013.12.011.

749. Kamaguchi A, Nakayama K, Ichiyama S, Nakamura R, Watanabe T, Ohta M, Baba H, Ohyama T. 2003. Effect of Porphyromonas gingivalis vesicles on coaggregation of Staphylococcus aureus to oral microorganisms. Curr Microbiol 47:485–491.

750. Grenier D. 2013. Porphyromonas gingivalis outer membrane vesicles mediate coaggregation and piggybacking of Treponema denticola and Lachnoanaerobaculum saburreum. Int J Dent 2013:305476. http://dx.doi.org/10.1155/2013/305476.

751. Altindis E, Fu Y, Mekalanos JJ. 2014. Proteomic analysis of Vibrio cholerae outer membrane vesicles. Proc Natl Acad Sci U S A 111:E1548–E1556. http://dx.doi.org/10.1073/pnas.1403683111.

752. Shibata S, Visick KL. 2012. Sensor kinase RscS induces the production of antigenically distinct outer membrane vesicles that depend on the symbiosis polysaccharide locus in Vibrio fischeri. J Bacteriol 194:185–194. http://dx.doi.org/10.1128/JB.05926-11.

753. Paerl HW, Pinckney JL, Steppe TF. 2000. Cyanobacterial-bacterial mat consortia: examining the functional unit of microbial survival and growth in extreme environments. Environ Microbiol 2:11–26. http://dx.doi.org/10.1046/j.1462-2920.2000.00071.x.

754. Stal LJ, Severin I, Bolhuis H. 2010. The ecology of nitrogen fixation in cyanobacterial mats. Adv Exp Med Biol 675:31–45. http://dx.doi.org/10.1007/978-1-4419-1528-3_3.

755. Morris RM, Rappé MS, Connon SA, Vergin KL, Siebold WA, Carlson CA, Giovannoni SJ. 2002. SAR11 clade dominates ocean surface bacterioplankton communities. Nature 420:806–810. http://dx.doi.org/10.1038/nature01240.

756. Dupont CL, Rusch DB, Yooseph S, Lombardo MJ, Richter RA, Valas R, Novotny M, Yee-Greenbaum J, Selengut JD, Haft DH, Halpern AL, Lasken RS, Nealson K, Friedman R, Venter JC. 2012. Genomic insights to SAR86, an abundant and uncultivated marine bacterial lineage. ISME J 6:1186–1199. http://dx.doi.org/10.1038/ismej.2011.189.

757. Amin SA, Parker MS, Armbrust EV. 2012. Interactions between diatoms and bacteria. Microbiol Mol Biol Rev 76:667–684. http://dx.doi.org/10.1128/MMBR.00007-12.

758. Luo H, Csuros M, Hughes AL, Moran MA. 2013. Evolution of divergent life history strategies in marine alphaproteobacteria. mBio 4:e00373-13. http://dx.doi.org/10.1128/mBio.00373-13.

759. Luo H, Moran MA. 2014. Evolutionary ecology of the marine Roseobacter clade. Microbiol Mol Biol Rev 78:573–587. http://dx.doi.org/10.1128/MMBR.00020-14.

760. Moran MA, Buchan A, Gonzalez JM, Heidelberg JF, Whitman WB, Kiene RP, Henriksen JR, King GM, Belas R, Fuqua C, Brinkac L, Lewis

M, Johri S, Weaver B, Pai G, Eisen JA, Rahe E, Sheldon WM, Ye W, Miller TR, Carlton J, Rasko DA, Paulsen IT, Ren Q, Daugherty SC, Deboy RT, Dodson RJ, Durkin AS, Madupu R, Nelson WC, Sullivan SA, Rosovitz MJ, Haft DH, Selengut J, Ward N. 2004. Genome sequence of *Silicibacter pomeroyi* reveals adaptations to the marine environment. Nature 432:910–913. http://dx.doi.org/10.1038/nature03170.

760. Mayali X, Franks PJ, Azam F. 2008. Cultivation and ecosystem role of a marine *Roseobacter* clade-affiliated cluster bacterium. Appl Environ Microbiol 74:2595–2603. http://dx.doi.org/10.1128/AEM.02191-07.

761. Luo H, Löytynoja A, Moran MA. 2012. Genome content of uncultivated marine roseobacters in the surface ocean. Environ Microbiol 14:41–51. http://dx.doi.org/10.1111/j.1462-2920.2011.02528.x.

762. Newton RJ, Griffin LE, Bowles KM, Meile C, Gifford S, Givens CE, Howard EC, King E, Oakley CA, Reisch CR, Rinta-Kanto JM, Sharma S, Sun S, Varaljay V, Vila-Costa M, Westrich JR, Moran MA. 2010. Genome characteristics of a generalist marine bacterial lineage. ISME J 4:784–798. http://dx.doi.org/10.1038/ismej.2009.150.

763. Lidbury I, Murrell JC, Chen Y. 2014. Trimethylamine N-oxide metabolism by abundant marine heterotrophic bacteria. Proc Natl Acad Sci U S A 111:2710–2715. http://dx.doi.org/10.1073/pnas.1317834111.

764. Cui Y, Suzuki S, Omori Y, Wong SK, Ijichi M, Kaneko R, Kameyama S, Tanimoto H, Hamasaki K. 2015. Abundance and distribution of dimethylsulfoniopropionate degradation genes and the corresponding bacterial community structure at dimethyl sulfide hot spots in the tropical and subtropical Pacific Ocean. Appl Environ Microbiol 81:4184–4194. http://dx.doi.org/10.1128/AEM.03873-14.

765. Durham BP, Sharma S, Luo H, Smith CB, Amin SA, Bender SJ, Dearth SP, Van Mooy BA, Campagna SR, Kujawinski EB, Armbrust EV, Moran MA. 2015. Cryptic carbon and sulfur cycling between surface ocean plankton. Proc Natl Acad Sci U S A 112:453–457. http://dx.doi.org/10.1073/pnas.1413137112.

766. Lidbury I, Kimberley G, Scanlan DJ, Murrell JC, Chen Y. 9 June 2015. Comparative genomics and mutagenesis analyses of choline metabolism in the marine *Roseobacter* clade. Environ Microbiol http://dx.doi.org/10.1111/1462-2920.12943.

767. Mishamandani S, Gutierrez T, Berry D, Aitken MD. 16 July 2015. Response of the bacterial community associated with a cosmopolitan marine diatom to crude oil shows a preference for the biodegradation of aromatic hydrocarbons. Environ Microbiol http://dx.doi.org/10.1111/1462-2920.12988.

768. Kujawinski EB. 2011. The impact of microbial metabolism on marine dissolved organic matter. Annu Rev Mar Sci 3:567–599. http://dx.doi.org/10.1146/annurev-marine-120308-081003.

769. Mayali X, Weber PK, Brodie EL, Mabery S, Hoeprich PD, Pett-Ridge J. 2012. High-throughput isotopic analysis of RNA microarrays to quantify microbial resource use. ISME J 6:1210–1221. http://dx.doi.org/10.1038/ismej.2011.175.

770. Nelson CE, Carlson CA, Ewart CS, Halewood ER. 2014. Community differentiation and population enrichment of Sargasso Sea bacterioplankton in the euphotic zone of a mesoscale mode-water eddy. Environ Microbiol 16:871–887. http://dx.doi.org/10.1111/1462-2920.12241.

771. Aylward FO, Eppley JM, Smith JM, Chavez FP, Scholin CA, DeLong EF. 2015. Microbial community transcriptional networks are conserved in three domains at ocean basin scales. Proc Natl Acad Sci U S A 112:5443–5448. http://dx.doi.org/10.1073/pnas.1502883112.

772. Haas AF, Rohwer F. 2015. Metabolic cascades in marine microbial communities. Proc Natl Acad Sci U S A 112:5552–5553. http://dx.doi.org/10.1073/pnas.1504310112.

773. Wagner-Döbler I, Ballhausen B, Berger M, Brinkhoff T, Buchholz I, Bunk B, Cypionka H, Daniel R, Drepper T, Gerdts G, Hahnke S, Han C, Jahn D, Kalhoefer D, Kiss H, Klenk HP, Kyrpides N, Liebl W, Liesegang H, Meincke L, Pati A, Petersen J, Piekarski T, Pommerenke C, Pradella S, Pukall R, Rabus R, Stackebrandt E, Thole S, Thompson L, Tielen P, Tomasch J, von Jan M, Wanphrut N, Wichels A, Zech H, Simon M. 2010. The complete genome sequence of the algal symbiont *Dinoroseobacter shibae*: a hitchhiker’s guide to life in the sea. ISME J 4:61–77. http://dx.doi.org/10.1038/ismej.2009.94.

774. Amin SA, Hmelo LR, van Tol HM, Durham BP, Carlson LT, Heal KR, Morales RL, Berthiaume CT, Parker MS, Djunaedi B, Ingalls AE, Parsek MR, Moran MA, Armbrust EV. 2015. Interaction and signalling between a cosmopolitan phytoplankton and associated bacteria. Nature 522:98–101. http://dx.doi.org/10.1038/nature14488.

775. Seymour JR, Simó R, Ahmed T, Stocker R. 2010. Chemoattraction to dimethylsulfoniopropionate throughout the marine microbial food web. Science 329:342–345. http://dx.doi.org/10.1126/science.1188418.

776. Miller TR, Hnilicka K, Dziedzic A, Desplats P, Belas R. 2004. Chemotaxis of *Silicibacter* sp. strain TM1040 toward dinoflagellate products. Appl Environ Microbiol 70:4692–4701. http://dx.doi.org/10.1128/AEM.70.8.4692-4701.2004.

777. Schaefer AL, Greenberg EP, Oliver CM, Oda Y, Huang JJ, Bittan-Banin G, Peres CM, Schmidt S, Juhaszova K, Sufrin JR, Harwood CS. 2008. A new class of homoserine lactone quorum-sensing signals. Nature 454:595–599. http://dx.doi.org/10.1038/nature07088.

778. Oz A, Sabehi G, Koblízek M, Massana R, Béjà O. 2005. *Roseobacter*-like bacteria in Red and Mediterranean Sea aerobic anoxygenic photosynthetic populations. Appl Environ Microbiol 71:344–353. http://dx.doi.org/10.1128/AEM.71.1.344-353.2005.

779. Moran MA, Miller WL. 2007. Resourceful heterotrophs make the most of light in the coastal ocean. Nat Rev Microbiol 5:792–800. http://dx.doi.org/10.1038/nrmicro1746.

780. Yutin N, Suzuki MT, Teeling H, Weber M, Venter JC, Rusch DB, Béjà O. 2007. Assessing diversity and biogeography of aerobic anoxygenic phototrophic bacteria in surface waters of the Atlantic and Pacific Oceans using the Global Ocean Sampling expedition metagenomes. Environ Microbiol 9:1464–1475. http://dx.doi.org/10.1111/j.1462-2920.2007.01265.x.

781. Xiao N, Jiao N. 2011. Formation of polyhydroxyalkanoate in aerobic anoxygenic phototrophic bacteria and its relationship to carbon source and light availability. Appl Environ Microbiol 77:7445–7450. http://dx.doi.org/10.1128/AEM.05955-11.

782. Wang H, Tomasch J, Jarek M, Wagner-Döbler I. 2014. A dual-species co-cultivation system to study the interactions between *Roseobacters* and dinoflagellates. Front Microbiol 5:311. http://dx.doi.org/10.3389/fmicb.2014.00311.

783. Lenk S, Moraru C, Hahnke S, Arnds J, Richter M, Kube M, Reinhardt R, Brinkhoff T, Harder J, Amann R, Mußmann M. 2012. *Roseobacter* clade bacteria are abundant in coastal sediments and encode a novel combination of sulfur oxidation genes. ISME J 6:2178–2187. http://dx.doi.org/10.1038/ismej.2012.66.

784. Pjevac P, Korlević M, Berg JS, Bura-Nakić E, Ciglenečki I, Amann R, Orlić S. 2015. Community shift from phototrophic to chemotrophic sulfide oxidation following anoxic holomixis in a stratified seawater lake. Appl Environ Microbiol 81:298–308. http://dx.doi.org/10.1128/AEM.02435-14.

785. Muthusamy S, Baltar F, González JM, Pinhassi J. 2014. Dynamics of metabolic activities and gene expression in the *Roseobacter* clade bacterium *Phaeobacter* sp. strain MED193 during growth with thiosulfate. Appl Environ Microbiol 80:6933–6942. http://dx.doi.org/10.1128/AEM.02038-14.

786. Zhao Y, Wang K, Budinoff C, Buchan A, Lang A, Jiao N, Chen F. 2009. Gene transfer agent (GTA) genes reveal diverse and dynamic *Roseobacter* and *Rhodobacter* populations in the Chesapeake Bay. ISME J 3:364–373. http://dx.doi.org/10.1038/ismej.2008.115.

787. Lang AS, Zhaxybayeva O, Beatty JT. 2012. Gene transfer agents: phage-like elements of genetic exchange. Nat Rev Microbiol 10:472–482. http://dx.doi.org/10.1038/nrmicro2802.

788. McDaniel LD, Young EC, Ritchie KB, Paul JH. 2012. Environmental factors influencing gene transfer agent (GTA)-mediated transduction in the subtropical ocean. PLoS One 7:e43506. http://dx.doi.org/10.1371/journal.pone.0043506.

789. Petersen J, Frank O, Göker M, Pradella S. 2013. Extrachromosomal, extraordinary and essential—the plasmids of the *Roseobacter* clade. Appl Microbiol Biotechnol 97:2805–2815. http://dx.doi.org/10.1007/s00253-013-4746-8.

790. Frank O, Göker M, Pradella S, Petersen J. 2015. Ocean’s twelve: flagellar and biofilm chromids in the multipartite genome of *Marinovum algicola* DG898 exemplify functional compartmentalization. Environ Microbiol 17:4019–4034. http://dx.doi.org/10.1111/1462-2920.12947.

791. Kirchman DL. 2002. The ecology of *Cytophaga-Flavobacteria* in aquatic environments. FEMS Microbiol Ecol 39:91–100. http://dx.doi.org/10.1111/j.1574-6941.2002.tb00910.x.

792. Fandino LB, Riemann L, Steward GF, Azam F. 2005. Population dynamics of *Cytophaga-Flavobacteria* during marine phytoplankton blooms analyzed by real-time quantitative PCR. Aquat Microb Ecol 40:251–257. http://dx.doi.org/10.3354/ame040251.

793. Dinasquet J, Granhag L, Riemann L. 2012. Stimulated bacterioplank-

ton growth and selection for certain bacterial taxa in the vicinity of the ctenophore *Mnemiopsis leidyi*. Front Microbiol 3:302. http://dx.doi.org/10.3389/fmicb.2012.00302.

794. Gómez-Pereira PR, Schüler M, Fuchs BM, Bennke C, Teeling H, Waldmann J, Richter M, Barbe V, Bataille E, Glöckner FO, Amann R. 2012. Genomic content of uncultured *Bacteroidetes* from contrasting oceanic provinces in the North Atlantic Ocean. Environ Microbiol 14: 52–66. http://dx.doi.org/10.1111/j.1462-2920.2011.02555.x.

795. Fernández-Gómez B, Richter M, Schüler M, Pinhassi J, Acinas SG, González JM, Pedrós-Alió C. 2013. Ecology of marine *Bacteroidetes*: a comparative genomics approach. ISME J 7:1026–1037. http://dx.doi.org/10.1038/ismej.2012.169.

796. Kabisch A, Otto A, König S, Becher D, Albrecht D, Schüler M, Teeling H, Amann RI, Schweder T. 2014. Functional characterization of polysaccharide utilization loci in the marine *Bacteroidetes ‘Gramella forsetii’* KT0803. ISME J 8:1492–1502. http://dx.doi.org/10.1038/ismej.2014.4.

797. Wietz M, Wemheuer B, Simon H, Giebel HA, Seibt MA, Daniel R, Brinkhoff T, Simon M. 2015. Bacterial community dynamics during polysaccharide degradation at contrasting sites in the Southern and Atlantic Oceans. Environ Microbiol 17:3822–3831. http://dx.doi.org/10.1111/1462-2920.12842.

798. Cottrell MT, Yu L, Kirchman DL. 2005. Sequence and expression analyses of *Cytophaga*-like hydrolases in a Western Arctic metagenomic library and the Sargasso Sea. Appl Environ Microbiol 71:8506–8513. http://dx.doi.org/10.1128/AEM.71.12.8506-8513.2005.

799. Hehemann JH, Correc G, Thomas F, Bernard T, Barbeyron T, Jam M, Helbert W, Michel G, Czjzek M. 2012. Biochemical and structural characterization of the complex agarolytic enzyme system from the marine bacterium *Zobellia galactanivorans*. J Biol Chem 287:30571–30584. http://dx.doi.org/10.1074/jbc.M112.377184.

800. Thomas F, Barbeyron T, Tonon T, Génicot S, Czjzek M, Michel G. 2012. Characterization of the first alginolytic operons in a marine bacterium: from their emergence in marine *Flavobacteria* to their independent transfers to marine *Proteobacteria* and human gut *Bacteroides*. Environ Microbiol 14:2379–2394. http://dx.doi.org/10.1111/j.1462-2920.2012.02751.x.

801. Mann AJ, Hahnke RL, Huang S, Werner J, Xing P, Barbeyron T, Huettel B, Stüber K, Reinhardt R, Harder J, Glöckner FO, Amann RI, Teeling H. 2013. The genome of the alga-associated marine flavobacterium *Formosa agariphila* KMM 3901T reveals a broad potential for degradation of algal polysaccharides. Appl Environ Microbiol 79:6813–6822. http://dx.doi.org/10.1128/AEM.01937-13.

802. Groisillier A, Labouré L, Michel G, Tonon T. 2015. The mannitol utilization system of the marine bacterium *Zobellia galactanivorans*. Appl Environ Microbiol 81:1799–1812. http://dx.doi.org/10.1128/AEM.02808-14.

803. Edwards JL, Smith DL, Connolly J, McDonald JE, Cox MJ, Joint I, Edwards C, McCarthy AJ. 2010. Identification of carbohydrate metabolism genes in the metagenome of a marine biofilm community shown to be dominated by Gammaproteobacteria and *Bacteroidetes*. Genes 1:371–384. http://dx.doi.org/10.3390/genes1030371.

804. Klindworth A, Mann AJ, Huang S, Wichels A, Quast C, Waldmann J, Teeling H, Glöckner FO. 2014. Diversity and activity of marine bacterioplankton during a diatom bloom in the North Sea assessed by total RNA and pyrotag sequencing. Mar Genomics 18:185–192. http://dx.doi.org/10.1016/j.margen.2014.08.007.

805. McBride MJ, Zhu Y. 2013. Gliding motility and Por secretion system genes are widespread among members of the phylum *Bacteroidetes*. J Bacteriol 195:270–278. http://dx.doi.org/10.1128/JB.01962-12.

806. Pedrotti ML, Beauvais S, Kerros ME, Iversen K, Peters F. 2009. Bacterial colonization of transparent exopolymeric particles in mesocosms under different turbulence intensities and nutrient conditions. Aquat Microb Ecol 55:301–312. http://dx.doi.org/10.3354/ame01308.

807. de la Torre JR, Christianson LM, Beja O, Suzuki MT, Karl DM, Heidelberg J, DeLong EF. 2003. Proteorhodopsin genes are distributed among divergent marine bacterial taxa. Proc Natl Acad Sci U S A 100: 12830–12835. http://dx.doi.org/10.1073/pnas.2133554100.

808. Venter JC, Remington K, Heidelberg JF, Halpern AL, Rusch D, Eisen JA, Wu D, Paulsen I, Nelson KE, Nelson W, Fouts DE, Levy S, Knap AH, Lomas MW, Nealson K, White O, Peterson J, Hoffman J, Parsons R, Baden-Tillson H, Pfannkoch C, Rogers YH, Smith HO. 2004. Environmental genome shotgun sequencing of the Sargasso Sea. Science 304:66–74. http://dx.doi.org/10.1126/science.1093857.

809. Campbell BJ, Waidner LA, Cottrell MT, Kirchman DL. 2008. Abundant proteorhodopsin genes in the North Atlantic Ocean. Environ Microbiol 10:99–109. http://dx.doi.org/10.1111/j.1462-2920.2007.01436.x.

810. Gómez-Consarnau L, González JM, Coll-Lladó M, Gourdon P, Pascher T, Neutze R, Pedrós-Alió C, Pinhassi J. 2007. Light stimulates growth of proteorhodopsin-containing marine *Flavobacteria*. Nature 445:210–213. http://dx.doi.org/10.1038/nature05381.

811. Yoshizawa S, Kawanabe A, Ito H, Kandori H, Kogure K. 2012. Diversity and functional analysis of proteorhodopsin in marine *Flavobacteria*. Environ Microbiol 14:1240–1248. http://dx.doi.org/10.1111/j.1462-2920.2012.02702.x.

812. Inoue K, Ono H, Abe-Yoshizumi R, Yoshizawa S, Ito H, Kogure K, Kandori H. 2013. A light-driven sodium ion pump in marine bacteria. Nat Commun 4:1678. http://dx.doi.org/10.1038/ncomms2689.

813. Yoshizawa S, Kumagai Y, Kim H, Ogura Y, Hayashi T, Iwasaki W, DeLong EF, Kogure K. 2014. Functional characterization of flavobacteria rhodopsins reveals a unique class of light-driven chloride pump in bacteria. Proc Natl Acad Sci U S A 111:6732–6737. http://dx.doi.org/10.1073/pnas.1403051111.

814. Frias-Lopez J, Shi Y, Tyson GW, Coleman ML, Schuster SC, Chisholm SW, DeLong EF. 2008. Microbial community gene expression in ocean surface waters. Proc Natl Acad Sci U S A 105:3805–3810. http://dx.doi.org/10.1073/pnas.0708897105.

815. Lami R, Cottrell MT, Campbell BJ, Kirchman DL. 2009. Light-dependent growth and proteorhodopsin expression by *Flavobacteria* and SAR11 in experiments with Delaware coastal waters. Environ Microbiol 11:3201–3209. http://dx.doi.org/10.1111/j.1462-2920.2009.02028.x.

816. Poretsky RS, Hewson I, Sun S, Allen AE, Zehr JP, Moran MA. 2009. Comparative day/night metatranscriptomic analysis of microbial communities in the North Pacific subtropical gyre. Environ Microbiol 11: 1358–1375. http://dx.doi.org/10.1111/j.1462-2920.2008.01863.x.

817. Feng S, Powell SM, Wilson R, Bowman JP. 2013. Light-stimulated growth of proteorhodopsin-bearing sea-ice psychrophile *Psychroflexus torquis* is salinity dependent. ISME J 7:2206–2213. http://dx.doi.org/10.1038/ismej.2013.97.

818. González JM, Pinhassi J, Fernández-Gómez B, Coll-Lladó M, González-Velázquez M, Puigbò P, Jaenicke S, Gómez-Consarnau L, Fernández-Guerra A, Goesmann A, Pedrós-Alió C. 2011. Genomics of the proteorhodopsin-containing marine flavobacterium *Dokdonia sp.* strain MED134. Appl Environ Microbiol 77:8676–8686. http://dx.doi.org/10.1128/AEM.06152-11.

819. Palovaara J, Akram N, Baltar F, Bunse C, Forsberg J, Pedrós-Alió C, González JM, Pinhassi J. 2014. Stimulation of growth by proteorhodopsin phototrophy involves regulation of central metabolic pathways in marine planktonic bacteria. Proc Natl Acad Sci U S A 111:E3650–E3658. http://dx.doi.org/10.1073/pnas.1402617111.

820. Bryant DA, Frigaard NU. 2006. Prokaryotic photosynthesis and phototrophy illuminated. Trends Microbiol 14:488–496. http://dx.doi.org/10.1016/j.tim.2006.09.001.

821. Fuhrman JA, Schwalbach MS, Stingl U. 2008. Proteorhodopsins: an array of physiological roles? Nat Rev Microbiol 6:488–494. http://dx.doi.org/10.1038/nrmicro1893.

822. González JM, Fernández-Gómez B, Fernández-Guerra A, Gómez-Consarnau L, Sánchez O, Coll-Lladó M, Del Campo J, Escudero L, Rodríguez-Martínez R, Alonso-Sáez L, Latasa M, Paulsen I, Nedashkovskaya O, Lekunberri I, Pinhassi J, Pedrós-Alió C. 2008. Genome analysis of the proteorhodopsin-containing marine bacterium *Polaribacter sp.* MED152 (*Flavobacteria*). Proc Natl Acad Sci U S A 105:8724–8729. http://dx.doi.org/10.1073/pnas.0712027105.

823. Pedrós-Alió C. 2012. The rare bacterial biosphere. Annu Rev Mar Sci 4:449–466. http://dx.doi.org/10.1146/annurev-marine-120710-100948.

824. Nan B, McBride MJ, Chen J, Zusman DR, Oster G. 2014. Bacteria that glide with helical tracks. Curr Biol 24:R169–R173. http://dx.doi.org/10.1016/j.cub.2013.12.034.

825. Banning EC, Casciotti KL, Kujawinski EB. 2010. Novel strains isolated from a coastal aquifer suggest a predatory role for flavobacteria. FEMS Microbiol Ecol 73:254–270. http://dx.doi.org/10.1111/j.1574-6941.2010.00897.x.

826. Saw JH, Yuryev A, Kanbe M, Hou S, Young AG, Aizawa S, Alam M. 2012. Complete genome sequencing and analysis of *Saprospira grandis* str. Lewin, a predatory marine bacterium. Stand Genomic Sci 6:84–93. http://dx.doi.org/10.4056/sigs.2445005.

827. Thornton DCO. 2014. Dissolved organic matter (DOM) release by phy-

toplankton in the contemporary and future ocean. Eur J Phycol 49:20–
46. http://dx.doi.org/10.1080/09670262.2013.875596.
828. Myklestad SM. 2000. Dissolved organic carbon from phytoplankton, p
111–148. *In* Wangersky P (ed), The handbook of environmental chemistry, vol 5, part D. Marine chemistry. Springer-Verlag, Berlin, Germany.
829. Guillard RRL, Wangersky PJ. 1958. The production of extracellular carbohydrates by some marine flagellates. Limnol Oceanogr 3:449–454. http://dx.doi.org/10.4319/lo.1958.3.4.0449.
830. Passow U, Alldredge AL, Logan BE. 1994. The role of particulate carbohydrate exudates in the flocculation of diatom blooms. Deep Sea Res I 41:335–357. http://dx.doi.org/10.1016/0967-0637(94)90007-8.
831. Obernosterer I, Herndl GJ. 1995. Phytoplankton extracellular release and bacterial growth: dependence on the inorganic N:P ratio. Mar Ecol Prog Ser 116:247–257. http://dx.doi.org/10.3354/meps116247.
832. Berman-Frank I, Rosenberg G, Levitan O, Haramaty L, Mari X. 2007. Coupling between autocatalytic cell death and transparent exopolymeric particle production in the marine cyanobacterium *Trichodesmium*. Environ Microbiol 9:1415–1422. http://dx.doi.org/10.1111/j.1462-2920 .2007.01257.x.
833. Wetz MS, Wheeler PA. 2007. Release of dissolved organic matter by coastal diatoms. Limnol Oceanogr 52:798–807. http://dx.doi.org/10 .4319/lo.2007.52.2.0798.
834. Biddle KD. 2015. The molecular ecophysiology of programmed cell death in marine phytoplankton. Annu Rev Mar Sci 7:341–375. http://dx.doi .org/10.1146/annurev-marine-010213-135014.
835. Møller EF. 2007. Production of dissolved organic carbon by sloppy feeding in the copepods *Acartia tonsa*, *Centropages typicus*, and *Temora longicornis*. Limnol Oceanogr 52:79–84. http://dx.doi.org/10.4319/lo.2007 .52.1.0079.
836. Jover LF, Effler TC, Buchan A, Wilhelm SW, Weitz JS. 2014. The elemental composition of virus particles: implications for marine biogeochemical cycles. Nat Rev Microbiol 12:519–528. http://dx.doi.org/10 .1038/nrmicro3289.
837. Martín J, Miquel JC. 2010. High downward flux of mucilaginous aggregates in the Ligurian Sea during summer 2002: similarities with the mucilage phenomenon in the Adriatic Sea. Mar Ecol 31:393–406.
838. Giani M, Sist P, Berto D, Serrazanetti GP, Ventrella V, Urbani R. 2012. The organic matrix of pelagic mucilaginous aggregates in the Tyrrhenian Sea (Mediterranean Sea). Mar Chem 132–133:83–94.
839. El-Swais H, Dunn KA, Bielawski JP, Li WK, Walsh DA. 2015. Seasonal assemblages and short-lived blooms in coastal north-west Atlantic bacterioplankton. Environ Microbiol 17:3642–3661. http://dx.doi.org /10.1111/1462-2920.12629.
840. Hahnke RL, Bennke CM, Fuchs BM, Mann AJ, Rhiel E, Teeling H, Amann R, Harder J. 2015. Dilution cultivation of marine heterotrophic bacteria abundant after a spring phytoplankton bloom in the North Sea. Environ Microbiol 17:3515–3526. http://dx.doi.org/10.1111/1462-2920 .12479.
841. Teeling H, Fuchs BM, Becher D, Klockow C, Gardebrecht A, Bennke CM, Kassabgy M, Huang S, Mann AJ, Waldmann J, Weber M, Klindworth A, Otto A, Lange J, Bernhardt J, Reinsch C, Hecker M, Peplies J, Bockelmann FD, Callies U, Gerdts G, Wichels A, Wiltshire KH, Glöckner FO, Schweder T, Amann R. 2012. Substrate-controlled succession of marine bacterioplankton populations induced by a phytoplankton bloom. Science 336:608–611. http://dx.doi.org/10.1126/science.1218344.
842. Elhenawy W, Debelyy MO, Feldman MF. 2014. Preferential packing of acidic glycosidases and proteases into Bacteroides outer membrane vesicles. mBio 5:e00909-14. http://dx.doi.org/10.1128/mBio.00909-14.
843. Tang K, Su Y, Brackman G, Cui F, Zhang Y, Shi X, Coenye T, Zhang XH. 2015. MomL, a novel marine-derived N-acyl homoserine lactonase from *Muricauda olearia*. Appl Environ Microbiol 81:774–782. http://dx .doi.org/10.1128/AEM.02805-14.
844. Kirchman DL, Dittel AI, Malmstrom RR, Cottrell MT. 2005. Biogeography of major bacterial groups in the Delaware Estuary. Limnol Oceanogr 50:1697–1706. http://dx.doi.org/10.4319/lo.2005.50.5.1697.
845. Mayali X, Franks PJS, Burton RS. 2011. Temporal attachment dynamics by distinct bacterial taxa during a dinoflagellate bloom. Aquat Microb Ecol 63:111–122. http://dx.doi.org/10.3354/ame01483.
846. Hahnke S, Brock NL, Zell C, Simon M, Dickschat JS, Brinkhoff T. 2013. Physiological diversity of *Roseobacter* clade bacteria co-occurring during a phytoplankton bloom in the North Sea. Syst Appl Microbiol 36:39–48. http://dx.doi.org/10.1016/j.syapm.2012.09.004.
847. Grossart HP, Levold F, Allgaier M, Simon M, Brinkhoff T. 2005.

Marine Biofilm Development

Marine diatom species harbour distinct bacterial communities. Environ Microbiol 7:860–873. http://dx.doi.org/10.1111/j.1462-2920 .2005.00759.x.
848. Rink B, Seeberger S, Martens T, Duurselen CD, Simon M, Brinkhoff T. 2007. Effects of phytoplankton bloom in a coastal ecosystem on the composition of bacterial communities. Aquat Microb Ecol 48:47–60. http://dx.doi.org/10.3354/ame048047.
849. Sapp M, Schwaderer AS, Wiltshire KH, Hoppe HG, Gerdts G, Wichels A. 2007. Species-specific bacterial communities in the phycosphere of microalgae? Microb Ecol 53:683–699. http://dx.doi.org/10.1007/s00248 -006-9162-5.
850. Lindh MV, Riemann L, Baltar F, Romero-Oliva C, Salomon PS, Granéli E, Pinhassi J. 2013. Consequences of increased temperature and acidification on bacterioplankton community composition during a mesocosm spring bloom in the Baltic Sea. Environ Microbiol Rep 5:252–262. http://dx.doi.org/10.1111/1758-2229.12009.
851. von Scheibner M, Dörge P, Biermann A, Sommer U, Hoppe HG, Jürgens K. 2014. Impact of warming on phyto-bacterioplankton coupling and bacterial community composition in experimental mesocosms. Environ Microbiol 16:718–733. http://dx.doi.org/10.1111/1462 -2920.12195.
852. Chauhan A, Pathak A, Rodolfo-Metalpa R, Milazzo M, Green SJ, Hall-Spencer JM. 2015. Metagenomics reveals planktonic bacterial community shifts across a natural CO₂ gradient in the Mediterranean Sea. Genome Announc 3(1):e01543-14. http://dx.doi.org/10.1128/genomeA .01543-14.
853. Clark ME, He Z, Redding AM, Joachimiak MP, Keasling JD, Zhou JZ, Arkin AP, Mukhopadhyay A, Fields MW. 2012. Transcriptomic and proteomic analyses of *Desulfovibrio vulgaris* biofilms: carbon and energy flow contribute to the distinct biofilm growth state. BMC Genomics 13:138. http://dx.doi.org/10.1186/1471-2164-13-138.
854. Muyzer G, Marty F. 2014. Molecular methods in microbiologically influenced corrosion research, monitoring and control, p 7–22. *In* Skovhus TL, Caffrey SM, Hubert CRJ (ed), Applications of molecular microbiological methods. Caister Academic Press, Norfolk, United Kingdom.
855. Neu TR, Lawrence JR. 2015. Innovative techniques, sensors, and approaches for imaging biofilms at different scales. Trends Microbiol 23: 233–242. http://dx.doi.org/10.1016/j.tim.2014.12.010.
856. Fuhrman JA. 2009. Microbial community structure and its functional implications. Nature 459:193–199. http://dx.doi.org/10.1038/nature 08058.
857. VerBerkmoes NC, Denef VJ, Hettich RL, Banfield JF. 2009. Systems biology: functional analysis of natural microbial consortia using community proteomics. Nat Rev Microbiol 7:196–205. http://dx.doi.org/10 .1038/nrmicro2080.
858. Dahle H, Roalkvam I, Thorseth IH, Pedersen RB, Steen IH. 2013. The versatile *in situ* gene expression of an *Epsilonproteobacteria*-dominated biofilm from a hydrothermal chimney. Environ Microbiol Rep 5:282– 290. http://dx.doi.org/10.1111/1758-2229.12016.
859. DeLong EF. 2009. The microbial ocean from genomes to biomes. Nature 459:200–206. http://dx.doi.org/10.1038/nature08059.
860. Leary DH, Hervey WJ, IV, Li RW, Deschamps JR, Kusterbeck AW, Vora GJ. 2012. Method development for metaproteomic analyses of marine biofilms. Anal Chem 84:4006–4013. http://dx.doi.org/10.1021 /ac203315n.
861. Leary DH, Li RW, Hamdan LJ, Hervey WJ, IV, Lebedev N, Wang Z, Deschamps JR, Kusterbeck AW, Vora GJ. 2014. Integrated metagenomic and metaproteomic analyses of marine biofilm communities. Biofouling 30:1211–1223. http://dx.doi.org/10.1080/08927014 .2014.977267.
862. Zeigler Allen L, Allen EE, Badger JH, McCrow JP, Paulsen IT, Elbourne LD, Thiagarajan M, Rusch DB, Nealson KH, Williamson SJ, Venter JC, Allen AE. 2012. Influence of nutrients and currents on the genomic composition of microbes across an upwelling mosaic. ISME J 6:1403–1414. http://dx.doi.org/10.1038/ismej.2011.201.
863. Allen AE, Allen LZ, McCrow JP. 2013. Lineage specific gene family enrichment at the microscale in marine systems. Curr Opin Microbiol 16:605–617. http://dx.doi.org/10.1016/j.mib.2013.10.001.
864. Moisander PH, Serros T, Paerl RW, Beinart RA, Zehr JP. 2014. Gammaproteobacterial diazotrophs and *nifH* gene expression in surface waters of the South Pacific Ocean. ISME J 8:1962–1973. http://dx.doi.org /10.1038/ismej.2014.49.

Dang and Lovell

865. Gifford SM, Sharma S, Booth M, Moran MA. 2013. Expression patterns reveal niche diversification in a marine microbial assemblage. ISME J 7:281–298. http://dx.doi.org/10.1038/ismej.2012.96.

866. Rinta-Kanto JM, Sun S, Sharma S, Kiene RP, Moran MA. 2012. Bacterial community transcription patterns during a marine phytoplankton bloom. Environ Microbiol 14:228–239. http://dx.doi.org/10.1111/j.1462-2920.2011.02602.x.

867. Williams TJ, Long E, Evans F, Demaere MZ, Lauro FM, Raftery MJ, Ducklow H, Grzymski JJ, Murray AE, Cavicchioli R. 2012. A metaproteomic assessment of winter and summer bacterioplankton from Antarctic Peninsula coastal surface waters. ISME J 6:1883–1900. http://dx.doi.org/10.1038/ismej.2012.28.

868. Georges AA, El-Swais H, Craig SE, Li WK, Walsh DA. 2014. Metaproteomic analysis of a winter to spring succession in coastal northwest Atlantic Ocean microbial plankton. ISME J 8:1301–1313. http://dx.doi.org/10.1038/ismej.2013.234.

869. Olaya-Abril A, Jiménez-Munguía I, Gómez-Gascón L, Rodríguez-Ortega MJ. 2014. Surfomics: shaving live organisms for a fast proteomic identification of surface proteins. J Proteomics 97:164–176. http://dx.doi.org/10.1016/j.jprot.2013.03.035.

870. Wilmes P, Bond PL. 2009. Microbial community proteomics: elucidating the catalysts and metabolic mechanisms that drive the Earth’s biogeochemical cycles. Curr Opin Microbiol 12:310–317. http://dx.doi.org/10.1016/j.mib.2009.03.004.

871. Hettich RL, Sharma R, Chourey K, Giannone RJ. 2012. Microbial metaproteomics: identifying the repertoire of proteins that microorganisms use to compete and cooperate in complex environmental communities. Curr Opin Microbiol 15:373–380. http://dx.doi.org/10.1016/j.mib.2012.04.008.

872. Hamilton JJ, Reed JL. 2014. Software platforms to facilitate reconstructing genome-scale metabolic networks. Environ Microbiol 16:49–59. http://dx.doi.org/10.1111/1462-2920.12312.

873. Biggs MB, Medlock GL, Kolling GL, Papin JA. 2015. Metabolic network modeling of microbial communities. Wiley Interdiscip Rev Syst Biol Med 7:317–334. http://dx.doi.org/10.1002/wsbm.1308.

874. Morales SE, Holben WE. 2011. Linking bacterial identities and ecosystem processes: can ‘omic’ analyses be more than the sum of their parts? FEMS Microbiol Ecol 75:2–16. http://dx.doi.org/10.1111/j.1574-6941.2010.00938.x.

875. Gilbert JA, Henry C. 2015. Predicting ecosystem emergent properties at multiple scales. Environ Microbiol Rep 7:20–22. http://dx.doi.org/10.1111/1758-2229.12258.


Hongyue Dang received his Ph.D. in marine science from University of South Carolina in 2001. After postdoctoral studies at SUNY Upstate Medical University and University at Buffalo, The State University of New York, he moved in 2004 to the CAS Key Laboratory of Marine Geology and Environment, Institute of Oceanology, Chinese Academy of Sciences, working on marine geomicrobiology as an Associate Professor. In 2008, he moved to the State Key Laboratory of Heavy Oil Processing, Centre for Bioengineering and Biotechnology, China University of Petroleum (East China), working on marine microbial ecology, biogeochemistry, and biotechnology as a Professor. Since 2013, he has worked as a Chair Professor in the State Key Laboratory of Marine Environmental Science and College of Ocean and Earth Sciences, Xiamen University. His main research interests are marine biocorrosion and surface-associated microbial communities, processes and functions, and marine carbon and nitrogen cycling in response to diverse environmental gradients and anthropogenic perturbations.

Charles R. Lovell received his Ph.D. in Biological Sciences from Purdue University in 1984. After postdoctoral studies at the University of Georgia, he accepted appointment as an Assistant Professor in the Department of Biological Sciences at the University of South Carolina in 1987. He was promoted to Associate Professor in 1993 and to Full Professor in 2000. His main research interests are in population and community dynamics of microorganisms, interactions between bacteria and higher organisms, and the physiology and ecology of anaerobic bacteria.
