
Journal of Molecular and Cellular Cardiology 95 (2016) 50–56

Contents lists available at ScienceDirect

Journal of Molecular and Cellular Cardiology

journal homepage: www.elsevier.com/locate/yjmcc

Review article

Receptor-mediated mitophagy

Osamu Yamaguchi ${ }^{a, *}$, Tomokazu Murakawa ${ }^{a}$, Kazuhiko Nishida ${ }^{b}$, Kinya Otsu ${ }^{b}$

${ }^{a}$ Department of Cardiovascular Medicine, Graduate School of Medicine, Osaka University, Suita, Osaka 565-0871, Japan

${ }^{b}$ Cardiovascular Division, King's College London British Heart Foundation Centre of Excellence, London SE5 9NU, United Kingdom

ARTICLE INFO

Article history:

Received 30 October 2015

Received in revised form 21 March 2016

Accepted 21 March 2016

Available online 25 March 2016

Keywords:

Autophagy

Mitophagy

Mitochondrial fusion and fission

Receptor

ABSTRACT

Mitochondria are essential organelles that supply ATP through oxidative phosphorylation to maintain cellular homeostasis. Extrinsic or intrinsic agents can impair mitochondria, and these impaired mitochondria can generate reactive oxygen species (ROS) as byproducts, inducing cellular damage and cell death. The quality control of mitochondria is essential for the maintenance of normal cellular functions, particularly in cardiomyocytes, because they are terminally differentiated. Accumulation of damaged mitochondria is characteristic of various diseases, including heart failure, neurodegenerative disease, and aging-related diseases. Mitochondria are generally degraded through autophagy, an intracellular degradation system that is conserved from yeast to mammals. Autophagy is thought to be a nonselective degradation process in which cytoplasmic proteins and organelles are engulfed by isolation membrane to form autophagosomes in eukaryotic cells. However, recent studies have described the process of selective autophagy, which targets specific proteins or organelles such as mitochondria. Mitochondria-specific autophagy is called mitophagy. Dysregulation of mitophagy is implicated in the development of chronic diseases including neurodegenerative diseases, metabolic diseases, and heart failure. In this review, we discuss recent progress in research on mitophagy receptors.

© 2016 Elsevier Ltd. All rights reserved.

Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .been well studied and will hereafter be referred to as autophagy. The process of autophagy is regulated by autophagy-related (Atg) genes, which were first identified in yeast. The principal role of autophagy is to supply nutrients for survival [3]. In addition, constitutive autophagy in mammals is also important for regulating the quality of proteins and organelles in order to maintain the functions of cell and organs [4,5]. Recent studies have suggested that autophagy induced protective effects in various organs [6,7]. Thus, autophagy is thought to function as a cell- and tissue-protective mechanism.

Mitochondria not only generate reactive oxygen species (ROS) as a byproduct of oxidative phosphorylation but also release pro-apoptotic factors, such as cytochrome c and apoptosis-inducing factor, which lead to apoptosis. Thus, quality control of mitochondria by elimination is important to avoid cell death. Damaged mitochondria are selectively degraded by specific autophagic elimination, called mitophagy, which was first investigated in yeast [8–10]. During mitophagy, dysfunctional mitochondria are selectively recognized through the mitophagy receptor, which is expressed on the outer mitochondrial membrane, to be removed [8,11,12]. Recent study reveals that the role of mitophagy is not only the elimination of dysfunctional mitochondria but mitochondrial turnover for metabolic transitioning from carbohydrates to fatty acids in cardiomyocytes during the perinatal period [13]. This observation is similar to the role of Nix-mediated mitophagy in the terminal stages of normal erythrocyte development [14]. Thus, the roles of mitophagy seem to be expanding to include physiological functions other than removal of damaged mitochondria.

The mitochondrial proteins Uth1p, Aup1p, and Atg32, were first identified in yeast and are involved in various mitophagy-related processes [10,15–17]. Uth1p was the first molecule to be identified as mediating mitophagy in response to rapamycin treatment and nutrient starvation [10]. However, the specific molecules involved in mitophagy and in sensing damaged mitochondria for selective engulfment by autophagosomes in mammalian cells remained unclear. Parkin, an E3 ubiquitin ligase, has been identified as a mitophagy-related factor that promotes selective degradation of damaged mitochondria [18]. Subsequent studies have described various other molecules involved in mitophagy. Some molecules on the outer mitochondrial membrane function as receptors to interact with autophagy-related proteins. Mitochondrial DNA is also degraded during mitophagic processes by DNaseII in lysosomes. Incomplete digestion of mitochondrial DNA induces inflammation in the heart and causes heart failure [19], because mitochondrial DNA may resemble bacterial DNA, which contains unmethylated CpG motifs that can promote inflammatory responses [20–22]. These inflammatory responses are mediated by Toll-like receptor 9, which mounts an innate immune response [23]. Thus, complete execution of mitophagy is important for cardiac protection.

Morphological changes in mitochondria through fission and fusion are also important steps in mitophagy. The dynamin-like GTPases mitofusin 1/2 (Mfn1/2) [24] on the outer mitochondrial membrane and Optic atrophy 1 (OPA1) [25] on the inner mitochondrial membrane are key regulators that mediate fusion, whereas, dynamin-related protein 1 (Drp1) has a pivotal role in mitochondrial fission [26]. Mitochondrial fragmentation by fission is thought to be one of the important steps during mitophagy both in yeast [27] and mammalian cells [28]. Elongated mitochondria, formed by fusion upon nutrient starvation, are protected from mitophagic degradation [29]. Thus, the size of mitochondria is important for engulfment by isolation membrane during the mitophagic response. Drp1 ablation induces lethal dilated cardiomyopathy in mice [30, 31]. Ablation of Drp1 in adult mouse cardiomyocytes decrease mitochondrial fission, leading to decreased respiratory function [31], on the other hand upregulates the expression level of Parkin to induce mitophagy and cardiac dysfunction [32]. Mitophagy markers, such as mitochondrial p62 and LC3, increase after knockdown of Drp1 [30]. These indicate that mitochondrial fission is not essential step for mitophagy.

The elimination of damaged mitochondria has an important role in the maintenance of cellular homeostasis and tissue function, particularly

in heart, because cardiomyocytes are terminally differentiated. Cardiac-specific knockout of Atg5 induces accumulation of abnormal mitochondria and cardiac dysfunction, and Atg5-dependent autophagy in cardiomyocytes is also tissue-protecting mechanism from pressure overload or aging [33,34]. In this review, we discuss the roles of mitophagic receptors.

## 2. Atg32 as a mitophagy receptor in yeast

In yeast, mitochondrial degradation is mediated through transport to the vacuole in an autophagy dependent manner [15,35]. Atg32 was identified by two independent groups as a mitophagy receptor on the outer mitochondrial membrane in yeast by genome-wide screening [16,17]. Atg32 is a 59-kDa protein that is anchored to the mitochondria via one transmembrane domain; N terminus is exposed to the cytosol, whereas the C terminus is located within the intermembrane space. Atg32 is necessary for mitophagy in post-log phase cells under respiratory growth, but not for canonical nonselective autophagy. Therefore, Atg32 is a mitophagy-specific factor. The expression level of Atg32 protein increases during respiratory growth [16], whereas the antioxidant N-acetylcysteine suppresses Atg32 induction and mitophagy, indicating that oxidative stress is one of the mechanisms to induce Atg32 expression. In Atg32-dependent mitophagy, Atg17, an essential molecule for bulk autophagy, is not required. Atg32 interacts with Atg8, a homolog of LC3 in yeast, via its WXXI motif to induce mitophagy, and with Atg11 via its residues 100–120 [36], which is an essential scaffold protein for selective autophagy in yeast. The WXXI (L) motif is called the Atg8-family interacting motif in yeast [37]. The mitochondrial inner membrane AAA (ATPases associated with diverse cellular activities) protease Yme1 processes Atg32 at the C terminus, and this processing is necessary for interaction with Atg11 and for mitophagic activity [38]. The interaction with Atg11 is also mediated through the phosphorylation of Ser114 on Atg32 by Hog1 and Pbs2, although Atg32 is not a direct substrate of Hog1 [36]. A recent study revealed that casein kinase 2 has a pivotal role in Atg32 phosphorylation at Ser114 and Ser119 [39]. Atg32 mutants, which do not bind to Atg8 or Atg11, cannot induce mitochondrial degradation [16,17,40]. Furthermore, the C-terminal intermembrane space domain of Atg32 is not essential for selective-autophagy. The mutant cytosolic domain of Atg32 anchored to peroxisomes can promote peroxisome autophagy (pexophagy) [40]. Atg11 also interacts with Dnm1, a yeast homolog of Drp1, and the fission components are recruited to the mitochondria via this interaction [27]. Thus, there is a close relationship between mitochondrial fission and mitophagy. However, no homologs or functional homologs of Atg11 have been identified in mammalian cells. In mammalian cells, BCL2L13/Bcl-rambo plays an important role in mitophagy, and act as a functional homolog of Atg32 [41]. Full details of the function of BCL2L13 will be described later. The function of this homolog of Atg32 in the heart is not elucidated yet (Fig. 1).

## 3. Expected characteristics of mammalian mitophagy receptors

As Atg32 in yeast, mitophagy receptors in mammalian cells are expected to localize to the outer mitochondrial membrane and interact with LC3 via the LC3-interacting region (LIR) motif in its cytosolic domain for mitophagy. The LIR motif is a tetrapeptide sequence W/YXXL/I, through which LC3 interacts with selective autophagy receptors. Several types of mitophagy receptors or receptor-related factors have been identified in mammalian cells, including NIX/BNIP3L, FUNDC1, PINK1/Parkin, and BCL2L13 (Bcl-rambo) [14,18,41–43] (Fig. 2).

The E3 ubiquitin ligase Parkin and phosphatase and tensin homolog-induced putative protein kinase 1 (PINK1) play important roles in elimination of damaged mitochondria, which lose mitochondrial membrane potential [18,44–46]. The involvement of PINK1/Parkin during induction of mitophagy will be discussed in detail in another paper in this
O. Yamaguchi et al. / Journal of Molecular and Cellular Cardiology 95 (2016) 50–56

**Yeast**

![Diagram](https://i.imgur.com/yeast_diagram.png)

**Mammalian cells**

![Diagram](https://i.imgur.com/mammalian_cells_diagram.png)

Fig. 1. Summary scheme indicates the molecular mechanisms for mitophagy of Atg32 in yeast and BCL2L13 in mammalian cells as a functional homolog of Atg32. The mitophagic activities of Atg32 and BCL2L13 are regulated by phosphorylation at serine. The homolog of scaffold protein Atg11 has not been identified in mammalian cells.

review series. Defects in the mitophagic pathway also induce cardiac disorders. The expression level of PINK1 is down-regulated in end-stage heart failure patients. PINK1 knockout mice develop pathological cardiac hypertrophy and left ventricular dysfunction [47]. Although Parkin is the downstream of PINK1, Parkin deficiency has no obvious effect on baseline cardiac phenotypes [48]. It is possible that other E3 ubiquitin ligase to compensate for the deficiency of Parkin.

p62/Sequestome 1 is an important molecule to link aggregated proteins to LC3 [49,50]. p62 can directly interact with ubiquitin through ubiquitin-binding domain, and with LC3 thorough LC3-interacting region. Depolarized mitochondria are ubiquitylated by Parkin, and then recruited-p62 on mitochondrial outer membrane binds to LC3 to form autophagosome. In this mitophagic process, p62 cooperates with Parkin for the elimination of damaged mitochondria [51]. Whether p62 is an essential molecule for mitophagy is controversial [52]. p62 is absent in lower eukaryotes like yeast. Recently CUET (coupling of ubiquitin endoplasmic reticulum degradation domain targeting) proteins are identified in yeast as a new class of ubiquitin-Atg8 adaptors; Cue5 from yeast and its human homolog Tollip [53]. Overexpression of Tollip promotes the clearance of Huntingtin-derived polyQ proteins in human cell lines.

VDAC1 is a mitochondrial target of Parkin-mediated ubiquitination, and is necessary for PINK1/Parkin-directed autophagy of damaged mitochondria [54]. Parkin is recruited to CCCP-treated mitochondria and associates with VDAC. In the VDACs-deficient cells, the recruitment of Parkin to damaged mitochondria and mitophagy are impaired [55]. VDACs are necessary for efficient elimination of damaged mitochondria thorough recruitment of Parkin from the cytosol. However, p62 is recruited to mitochondria by Parkin in VDAC1-independent manner [52].

Recently, elegant work by Youle's group showed that NDP52 and optineurin, which were previously linked to xenophagy, are the primary receptors for PINK1/Parkin-mediated mitophagy [56]. PINK1 directly activates mitophagy through recruitment of NDP52 and optineurin. In this process, PINK1-generated phospho-ubiquitin may act as mitophagy signal, and ubiquitin binding by NDP52 is essential for mitophagy. Thus, ubiquitin binding might be one of the expected characteristics of mammalian mitophagy receptors.

Cardiolipin is a phospholipid on inner mitochondrial membrane. Some mitophagic stimuli can induce externalization of cardiolipin to the mitochondrial surface [57]. Inhibition of externalization of cardiolipin decreased the mitophagic activity. LC3 contains cardiolipin-binding site to interact with cardiolipin directly. The LC3 deletion mutant of cardiolipin-binding site abolished mitophagy, but could induce non-selective autophagy [57]. These results may indicate that the interaction with cardiolipin is also one of the mitophagic receptor-related characters.

PINK1/Parkin also induces the formation of mitochondria-derived vesicles (MDVs) [11,58,59], which bud off mitochondria and are transferred to lysosomes to regulate mitochondrial quality control through a pathway that is distinct from common mitophagic pathway described in this review. The MDV pathway is thought to present the process of microautophagy because MDVs are directly transferred to lysosomes. Oxidative stress from within mitochondria induces MDVs in a PINK1/Parkin-dependent manner [60]. Other molecules involved in the formation of MDVs from mitochondria have not yet been identified [11,58].

### 4. NIP3-like protein X (NIX/BNIP3L) and BNIP3

NIX/BNIP3L, which is localized to the outer mitochondrial membrane [61], is identified as an adenovirus E1B-19 K-interacting protein with pro-apoptotic activity [62]. NIX knockout mice develop anemia with reduced mature erythrocytes, which exhibit mitochondrial

O. Yamaguchi et al. / Journal of Molecular and Cellular Cardiology 95 (2016) 50–56

![Diagram](attachment:diagram.png)

Fig. 2. The schema indicates the molecular mechanisms for mitophagy mediated by molecules on mitochondria. NIX, BIP3, and FUNDC1 interact with LC3 via LIR motif to induce mitophagy. FUNDC1 and BNIP3-dependent mitophagy is induced by hypoxia. The externalization of cardiolipin from inner mitochondrial membrane to outer mitochondrial membrane induces mitophagy by binding to LC3 directly via cardiolipin-binding site. The interaction between LC3 and SMURF1 is not elucidated.

retention [14]. NIX is highly induced during the late phase of erythrocyte maturation and is involved in the elimination of mitochondria from reticulocytes; this process is essential for red blood cell maturation [63]. NIX plays a role as a mitophagy receptor through its N-terminal LIR motif, which interacts with LC3 and GABARAP [63,64]. Mutants in the LIR motif of NIX abolish its interaction with LC3/GABARAP and show decreased mitophagic activity during reticulocytes maturation.

BNIP3 also contains a LIR motif and functions in hypoxia-induced mitophagy [65–67]. BNIP3-mediated mitophagy in cardiomyocytes is independent of mitochondrial permeability transition pore and pro-cell death activity [14,43,65]. Phosphorylation of BNIP3 at Ser17 and Ser24 flanking the LIR motif promotes its interaction with LC3 and GATE-16 to induce mitophagy [68]. The molecular basis of phosphorylation of BNIP3 is still unknown. A small GTPase of the Ras superfamily, Rheb, is recruited to the mitochondrial outer membrane under high oxidative phosphorylation activity, and promote mitophagic activity through interaction with NIX/BNIP3L and LC3-II [69].

NIX and BNIP3 have dual roles in the regulation of mitophagy as well as cell death in the heart. The regulatory mechanism of these two roles is not fully understood, but phosphorylation of BNIP3 at serine residues is one of the candidates [68]. Both NIX and BNIP3 mediate the progression of cardiac diseases through cardiomyocyte death [70–72]. NIX localizes to mitochondria in cell lines and induces cytochrome c release, caspase-3 activation, and cell death. Cardiac-specific overexpression of NIX leads lethal dilated cardiomyopathy through cardiomyocyte apoptosis [71]. Induction of BNIP3 to ventricular myocytes provokes cell death during hypoxia [57]. Cardiac-specific overexpression of BNIP3 also induces cardiomyocyte apoptosis and progressive cardiac dysfunction. Apoptotic cell death is significantly decreased in BNIP3 knockout myocardium 2 days after ischemia-reperfusion injury [72]. In these studies, mitophagic activity in the heart is not examined, therefore it is not elucidated whether mitophagy contributes to the cardiomyocyte apoptosis and cardiac phenotypes. BNIP3 overexpression in adult cardiomyocytes promotes translocation of Drp1 to mitochondria, and BNIP3-mediated translocation of Parkin to mitochondria and ubiquitination is dependent on Drp1. Translocation of Drp1 and Parkin is required for BNIP3-induced autophagy [73].

### 5. FUNDC1

FUNDC1 also induces mitophagy in mammalian cells [42]. FUNDC1 is an integral mitochondrial outer-membrane protein, and play an important role as a receptor for hypoxia-induced mitophagy. The interaction between BNIP3 and FUNDC1 during hypoxia-induced mitophagy is not clarified yet, however, there might be cooperation between these pathways in regulating mitophagy induced by hypoxia. FUNDC1 interacts with LC3 via the LIR motif, and mutations in this motif inhibit the interaction between FUNDC1 and LC3, impairing the mitophagic activity. The mitophagic activity induced by FUNDC1 is inhibited through its phosphorylation by Src kinase or casein kinase-2 at Tyr18 in the LIR motif and Ser13, respectively [74]. Upon stimulation with hypoxia, Src kinase is inactivated and FUNDC1 is dephosphorylated, thereby inducing mitophagy. Inhibition of both kinases also activates mitophagy. Bcl-xL, which prevents mitochondrial outer membrane permeabilization and apoptosis, inhibits mitophagy through FUNDC1. Bcl-xL interacts with PGAM5, a mitochondrial phosphatase, as a direct inhibitor to prevent the dephosphorylation of FUNDC1 at Ser13 [75]. Recent study showed that aged PGAM5-deficient mice show Parkinson's disease-like phenotypes [76]. PGAM5 stabilizes the expression of PINK1 on mitochondria, which induce the recruitment of Parkin and

mitophagy. This report may indicate the close relationship between PGAM5/FUNDC1 and PINK1/Parkin pathway during mitophagy. ULK1, the yeast ATG1 homolog [77,78] and Ser/Thr kinase translocates to fragmented mitochondria upon mitophagic induction, interacts with FUNDC1 and phosphorylates FUNDC1 at Ser17. The phosphorylation of Ser17 enhances FUNDC1 binding to LC3, inducing mitophagy [79]. MicroRNA-137 (*miR-137*), which is mainly expressed in the brain and regulates neural stem cell determination [80], is hypoxia-responsive microRNAs that targets two mitophagic receptors, NIX and FUNDC1, and significantly inhibits hypoxia-induced mitophagy [81]. Thus, these studies suggest that the level of mitophagic receptor expression can regulate the mitophagic activity induced by hypoxic conditions, and that there exists an interaction or a cooperative mechanism between these two mitophagic pathways. However, FUNDC1 is downregulated during exposure to hypoxic stimuli [42]. The mechanism through which FUNDC1 expression is regulated has not yet been elucidated.

### 6. Smad-ubiquitin regulatory factor 1 (SMURF1)-mediated autophagy

Orvedahl et al. identified SMURF1 as a selective autophagy factor using genome-wide siRNA screening. SMURF1 is a HECT-domain ubiquitin ligase that targets several cytoplasmic proteins [82] and is recruited to damaged mitochondria, where it promotes mitophagy [83]. SMURF1-deficient cells show diffuse accumulation of fragmented mitochondria. Furthermore, SMURF1-deficient mice accumulate abnormal mitochondria, which are swollen or fragmented, and contained abnormal cristae structure, in the heart, brain, and liver. However, the function of SMURF1 as a mitophagic receptor on mitochondria is unknown. The interaction between SMURF1 and LC3 has not yet been elucidated.

### 7. BCL2L13/Bcl-rambo is a novel mitophagy receptor in mammalian cells

As previously described, Atg32 is essential for mitophagy through its interaction with Atg8 and Atg11. Atg32 has a single transmembrane domain anchored to the outer mitochondrial membrane, and contains a WXXI motif binding to Atg8. PINK1/Parkin mediates mitophagy as described elsewhere. Parkin is expressed in most adult tissues; however, some fetal tissues and cell lines, such as HeLa cells, show little or no endogenous expression of Parkin [44,84,85]. Furthermore, Parkin-deficient mice show mild phenotypes, which exhibit grossly normal brain morphology [86], indicating that there may be unknown mechanisms of mitophagy in mammalian cells other than the PINK1/Parkin axis.

Using the molecular profile of Atg32 as a search tool, including mitochondrial localization, WXXL/I motifs, acidic amino acid clusters, and single membrane-spanning topology, we identified BCL2L13 (also known as Bcl-rambo) as a novel mitophagy receptor by screening the UniProt database [41]. BCL2L13 is a protein having 434 amino acids with a C-terminal single transmembrane domain and four conserved BCL2 homology domains (BH1–4) [87]. BCL2L13 is ubiquitously expressed [87] and localized in the outer mitochondrial membrane, and the N-terminus of BCL2L13 is exposed to the cytosol. Previous reports have revealed the pro-apoptotic function of BCL2L13 through its membrane-anchored C-terminal domain, independent of the BH domains. However, BCL2L13 does not induce apoptosis in HEK293 cells, even when overexpressed. Analysis of the sequence of mouse BCL2L13 revealed two WXXL/I motifs at positions 147–150 and 273–276. A yeast two-hybrid assay and glutathione S-transferase affinity isolation assay have shown that BCL2L13 can bind to LC3. Mutants of BCL2L13 in two WXXL/I motif analysis have shown that the second WXXL/I motif at residues 273–276, but not the first WXXL/I motif at residues 147–150, is a functional LIR motif that interacts with LC3.

Surprisingly, overexpression of BCL2L13 induces mitochondrial fragmentation in HEK293 cells, whereas the knockdown of BCL2L13 induces mitochondrial elongation, indicating that endogenous BCL2L13 has an important role in mitochondrial fission. Binding to LC3 via the second LIR motif is not required for BCL2L13-induced fragmentation. However, all BH1–4 domains are involved in mitochondrial fission. Mitochondrial localization of BCL2L13 with the transmembrane domain is also essential for mitochondrial fission. BCL2L13 can induce mitochondrial fission, even in cells in which Drp1, which is a central molecule in mitochondrial fission, is knocked down. However, the molecular mechanisms of mitochondrial fragmentation without Drp1 have not been unraveled.

Western blot analysis to detect LC3 conversion from LC3-I to LC3-II, immunocytochemical analysis to determine co-localization of mitochondria and lysosomes, and ultrastructural analysis by electron microscopy have collectively shown that BCL2L13 induces mitophagy. BCL2L13-induced mitophagy is mediated through its interaction with LC3 via the second LIR motif. BCL2L13 can induce mitophagy in Atg32-deficient yeast, and the second LIR motif is essential for BCL2L13-induced mitophagy in yeast, suggesting that BCL2L13 is a functional homolog of Atg32. BCL2L13 is the first molecule that has been shown to be able to rescue mitophagy in the context of Atg32 deficiency. BCL2L13 fails to induce mitophagy in Atg7-deficient yeast, indicating that BCL2L13-induced mitophagy is mediated through common autophagy machinery. The physiological significance of BCL2L13 has been elucidated in BCL2L13-knockdown experiments. Carbonyl cyanide *m*-chlorophenyl hydrazone (CCCP) induces loss of mitochondrial membrane potential, leading to mitophagy. The protein expression level of BCL2L13 is increased after the addition of CCCP. Knockdown of endogenous BCL2L13 attenuates CCCP-induced mitochondrial fragmentation and mitophagy. This indicates an important role of BCL2L13 during CCCP-induced mitochondrial fission and mitochondrial elimination by mitophagy. Moreover, phosphorylation of Ser114 of Atg32, which is close to the Atg8-family interacting motif, is an important step mediating the interaction between Atg32 and Atg11 [36]. Similarly, BCL2L13 is also phosphorylated at Ser272, which is close to the second LIR motif (residues 273–276). The BCL2L13S272A mutant induces mitochondrial fragmentation, but also decreases the interaction with LC3 and the mitophagic activity of the cells, as assessed by western blotting of LC3. Thus, while phosphorylation of BCL2L13 may regulate its mitophagic activity, the kinase for BCL2L13 is unknown. Prosurvival Bcl-2 family proteins, such as Bcl-2, Bcl-xL and Mcl-1, antagonize Parkin/PINK1-dependent mitophagy in a Beclin-1-independent manner [88]. Translocation of Parkin to damaged mitochondria is decreased by prosurvival Bcl-2 family proteins, and increased by BH3-only proteins. BCL2L13 can induce mitophagy in Parkin-deficient cells [41], and the crosstalk between BCL2L13 and Parkin during mitophagic process is not known.

The molecular mechanisms of BCL2L13-induced mitochondrial fission and mitophagy are not entirely clear. We hypothesize that BCL2L13 on the outer mitochondrial membrane recruits LC3 to the surface of the mitochondria via LIR-mediated interactions. Although the specific components of the complex involved in mitophagy have not been identified, BCL2L13 may bind to an unidentified mammalian homolog of Atg11 as a scaffold protein, similar to the mechanism observed in yeast. In yeast, Atg11 can interact with Dnm1 [27] to induce mitochondrial fission to be manageable for mitophagy. No mammalian homolog of Atg11 has been identified to date [89], and whether BCL2L13 can induce mitophagy, even in Atg32 and Atg11 double-deficient yeast, is still unclear. In other words, it is not clear whether BCL2L13 has dual functions of Atg32 and Atg11 in mitophagy. Interestingly, the dual roles of BCL2L13 in mitochondrial fission and mitophagy, facilitated by binding with LC3, are expected to contribute to effective cooperation of the two processes. BCL2L13 contains a C-terminal intermembrane space domain, suggesting that BCL2L13 can interact with other mitochondrial proteins. The regulation of the receptor function of BCL2L13, allowing it to sense changes in the integrity and status of the mitochondria, such as the loss of mitochondrial membrane

potential, may be mediated through the interaction with such factors. It has been reported that BCL2L13 can bind to adenine nucleotide translocator (ANT) and suppress its ADP-ATP-dependent translocation activity to induce mitochondrial permeability transition [90]. This interaction may one of the regulatory mechanisms of BCL2L13, however, much more work is required to clarify the molecular mechanisms and functions of BCL2L13. For example, it is not clear how the BH domains of BCL2L13 regulate mitochondrial fission. BCL2L13 does not interact with BCL2 family proteins, and the conserved BH domains of BCL2L13 are not essential for the induction of cell death [87]. Identification of novel binding proteins for the BH domains of BCL2L13 may shed light on the molecular mechanism underlying mitochondrial fission. In Drp1-knockdown HEK293 cells overexpressing BCL2L13, elongated mitochondria persist, and the levels of fragmented mitochondria are reduced, suggesting that there may ba an interaction between these two pathways. Interactions between BCL2L13 and other know factors involved in mitochondrial fission or fusion must be examined. Moreover, the kinase(s) or phosphatase(s) regulating phosphorylation at Ser272 of BCL2L13 should be defined. These enzymes may sense mitochondrial membrane potential. Finally, it is important to examine whether BCL2L13 may have physiological roles in vivo. To date, the involvement of BCL2L13 in mitophagy in the heart at baseline and pathophysiological role to induce cardiac diseases are not known. Analysis of genetically modified animal models, such as BCL2L13-knockout mice, will provide fundamental information about the in vivo role of this protein in mitochondrial quality control under physiological and pathological conditions not only in the heart but in other tissues.

### 8. Conclusions and perspectives

Cardiomyocytes must have many mitochondria in order to generate sufficient amounts of ATP for contraction. Moreover, cardiomyocytes are terminally differentiated cells that cannot divide to dilute toxic factors produced by mitochondria, such as ROS. Therefore, mitochondrial quality is important, and so mitochondrial quality control mechanisms such as mitophagy are candidate for pathophysiological processes. Some mitophagic receptors have been identified in mammalian cells as described; however, the full details of the mechanisms underlying mitophagy have not been elucidated. Thus, further studies are needed to identify novel targets associated with mitophagy for the treatment of cardiovascular diseases.

### Disclosures

The authors declare no competing financial interests.

### Acknowledgments

We appreciate all members we have worked with for their cooperation. This work was supported by JSPS KAKENHI Grant Number 15K09140.

### References

[1] N. Mizushima, Y. Ohsumi, T. Yoshimori, Autophagosome formation in mammalian cells, Cell Struct. Funct. 27 (2002) 421–429.

[2] B. Levine, D.J. Klionsky, Development by self-digestion: molecular mechanisms and biological functions of autophagy, Dev. Cell 6 (2004) 463–477.

[3] A. Kuma, M. Hatano, M. Matsui, A. Yamamoto, H. Nakaya, T. Yoshimori, et al., The role of autophagy during the early neonatal starvation period, Nature 432 (2004) 1032–1036.

[4] T. Hara, K. Nakamura, M. Matsui, A. Yamamoto, Y. Nakahara, R. Suzuki-Migishima, et al., Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice, Nature 441 (2006) 885–889.

[5] M. Komatsu, S. Waguri, T. Chiba, S. Murata, J. Iwata, I. Tanida, et al., Loss of autophagy in the central nervous system causes neurodegeneration in mice, Nature 441 (2006) 880–884.

[6] R.A. Gottlieb, R.M. Mentzer, Autophagy during cardiac stress: joys and frustrations of autophagy, Annu. Rev. Physiol. 72 (2010) 45–59.

[7] B. Ravikumar, S. Sarkar, J.E. Davies, M. Futter, M. Garcia-Arencibia, Z.W. Green-Thompson, et al., Regulation of mammalian autophagy in physiology and pathophysiology, Physiol. Rev. 90 (2010) 1383–1435.

[8] R.J. Youle, D.P. Narendra, Mechanisms of mitophagy, Nat. Rev. Mol. Cell Biol. 12 (2011) 9–14.

[9] J.J. Lemasters, Selective mitochondrial autophagy, or mitophagy, as a targeted defense against oxidative stress, mitochondrial dysfunction, and aging, Rejuvenation Res. 8 (2005) 3–5.

[10] I. Kissova, M. Deffieu, S. Manon, N. Camougrand, Uth1p is involved in the autophagic degradation of mitochondria, The Journal of biological chemistry. 279 (2004) 39068–39074.

[11] J.J. Lemasters, Variants of mitochondrial autophagy: types 1 and 2 mitophagy and micromitophagy (type 3), Redox Biol. 2 (2014) 749–754.

[12] I. Kim, S. Rodriguez-Enriquez, J.J. Lemasters, Selective degradation of mitochondria by mitophagy, Arch. Biochem. Biophys. 462 (2007) 245–253.

[13] G. Gong, M. Song, G. Csordas, D.P. Kelly, S.J. Matkovich, G.W. Dorn II, Parkin-mediated mitophagy directs perinatal cardiac metabolic maturation in mice, Science 350 (2015), aad2459.

[14] H. Sandoval, P. Thiagarajan, S.K. Dasgupta, A. Schumacher, J.T. Prchal, M. Chen, et al., Essential role for Nix in autophagic maturation of erythroid cells, Nature 454 (2008) 232–235.

[15] R. Tal, G. Winter, N. Ecker, D.J. Klionsky, H. Abeliovich, Aup1p, a yeast mitochondrial protein phosphatase homolog, is required for efficient stationary phase mitophagy and cell survival, J. Biol. Chem. 282 (2007) 5617–5624.

[16] K. Okamoto, N. Kondo-Okamoto, Y. Ohsumi, Mitochondria-anchored receptor Atg32 mediates degradation of mitochondria via selective autophagy, Dev. Cell 17 (2009) 87–97.

[17] T. Kanki, K. Wang, Y. Cao, M. Baba, D.J. Klionsky, Atg32 is a mitochondrial protein that confers selectivity during mitophagy, Dev. Cell 17 (2009) 98–109.

[18] D. Narendra, A. Tanaka, D.F. Suen, R.J. Youle, Parkin is recruited selectively to impaired mitochondria and promotes their autophagy, J. Cell Biol. 183 (2008) 795–803.

[19] T. Oka, S. Hikoso, O. Yamaguchi, M. Taneike, T. Takeda, T. Tamai, et al., Mitochondrial DNA that escapes from autophagy causes inflammation and heart failure, Nature 485 (2012) 251–255.

[20] L.R. Cardon, C. Burge, D.A. Clayton, S. Karlin, Pervasive CpG suppression in animal mitochondrial genomes, Proc. Natl. Acad. Sci. U. S. A. 91 (1994) 3799–3803.

[21] M.W. Gray, G. Burger, B.F. Lang, Mitochondrial evolution, Science 283 (1999) 1476–1481.

[22] Y. Pollack, J. Kasir, R. Shemer, S. Metzger, M. Szyf, Methylation pattern of mouse mitochondrial DNA, Nucleic Acids Res. 12 (1984) 4811–4824.

[23] O. Takeuchi, S. Akira, Pattern recognition receptors and inflammation, Cell 140 (2010) 805–820.

[24] T. Koshiba, S.A. Detmer, J.T. Kaiser, H. Chen, J.M. McCaffery, D.C. Chan, Structural basis of mitochondrial tethering by mitofusin complexes, Science 305 (2004) 858–862.

[25] C. Delettre, G. Lenaers, J.M. Griffoin, N. Gigarel, C. Lorenzo, P. Belenguer, et al., Nuclear gene OPA1, encoding a mitochondrial dynamin-related protein, is mutated in dominant optic atrophy, Nat. Genet. 26 (2000) 207–210.

[26] E. Smirnova, L. Griparic, D.L. Shurland, A.M. van der Bliek, Dynamin-related protein Drp1 is required for mitochondrial division in mammalian cells, Mol. Biol. Cell 12 (2001) 2245–2256.

[27] K. Mao, K. Wang, X. Liu, D.J. Klionsky, The scaffold protein Atg11 recruits fission machinery to drive selective mitochondria degradation by autophagy, Dev. Cell 26 (2013) 9–18.

[28] G. Twig, A. Elorza, A.J. Molina, H. Mohamed, J.D. Wikstrom, G. Walzer, et al., Fission and selective fusion govern mitochondrial segregation and elimination by autophagy, EMBO J. 27 (2008) 433–446.

[29] A.S. Rambold, B. Kostelecky, N. Elia, J. Lippincott-Schwartz, Tubular network formation protects mitochondria from autophagosomal degradation during nutrient starvation, Proc. Natl. Acad. Sci. U. S. A. 108 (2011) 10190–10195.

[30] M. Song, K. Mihara, Y. Chen, L. Scorrano, G.W. Dorn 2nd., Mitochondrial fission and fusion factors reciprocally orchestrate mitophagic culling in mouse hearts and cultured fibroblasts, Cell Metab. 21 (2015) 273–285.

[31] Y. Kageyama, M. Hoshijiima, K. Seo, D. Bedja, P. Sysa-Shah, S.A. Andrabi, et al., Parkin-independent mitophagy requires Drp1 and maintains the integrity of mammalian heart and brain, EMBO J. 33 (2014) 2798–2813.

[32] M. Song, G. Gong, Y. Burelle, A.B. Gustafsson, R.N. Kitsis, S.J. Matkovich, et al., Interdependence of Parkin-mediated mitophagy and mitochondrial fission in adult mouse hearts, Circ. Res. 117 (2015) 346–351.

[33] A. Nakai, O. Yamaguchi, T. Takeda, Y. Higuchi, S. Hikoso, M. Taniike, et al., The role of autophagy in cardiomyocytes in the basal state and in response to hemodynamic stress, Nat. Med. 13 (2007) 619–624.

[34] M. Taneike, O. Yamaguchi, A. Nakai, S. Hikoso, T. Takeda, I. Mizote, et al., Inhibition of autophagy in the heart induces age-related cardiomyopathy, Autophagy 6 (2010) 600–606.

[35] T. Kanki, D.J. Klionsky, Mitophagy in yeast occurs through a selective mechanism, J. Biol. Chem. 283 (2008) 32386–32393.

[36] Y. Aoki, T. Kanki, Y. Hirota, Y. Kurihara, T. Saigusa, T. Uchiumi, et al., Phosphorylation of Serine 114 on Atg32 mediates mitophagy, Mol. Biol. Cell 22 (2011) 3206–3217.

[37] N.N. Noda, Y. Ohsumi, F. Inagaki, Atg8-family interacting motif crucial for selective autophagy, FEBS Lett. 584 (2010) 1379–1385.

[38] K. Wang, M. Jin, X. Liu, D.J. Klionsky, Proteolytic processing of Atg32 by the mitochondrial i-AAA protease Yme1 regulates mitophagy, Autophagy 9 (2013) 1828–1836.

[39] T. Kanki, Y. Kurihara, X. Jin, T. Goda, Y. Ono, M. Aihara, et al., Casein kinase 2 is essential for mitophagy, EMBO Rep. 14 (2013) 788–794.

[40] N. Kondo-Okamoto, N.N. Noda, S.W. Suzuki, H. Nakatogawa, I. Takahashi, M. Matsunami, et al., Autophagy-related protein 32 acts as autophagic degron and directly initiates mitophagy, J. Biol. Chem. 287 (2012) 10631–10638.

[41] T. Murakawa, O. Yamaguchi, A. Hashimoto, S. Hikoso, T. Takeda, T. Oka, et al., Bcl-2-like protein 13 is a mammalian Atg32 homologue that mediates mitophagy and mitochondrial fragmentation, Nat. Commun. 6 (2015) 7527.

[42] L. Liu, D. Feng, G. Chen, M. Chen, Q. Zheng, P. Song, et al., Mitochondrial outer-membrane protein FUNDC1 mediates hypoxia-induced mitophagy in mammalian cells, Nat. Cell Biol. 14 (2012) 177–185.

[43] R.L. Schweers, J. Zhang, M.S. Randall, M.R. Lloyd, W. Li, F.C. Dorsey, et al., NIX is required for programmed mitochondrial clearance during reticulocyte maturation, Proc. Natl. Acad. Sci. U. S. A. 104 (2007) 19500–19505.

[44] N. Matsuda, S. Sato, K. Shiba, K. Okatsu, K. Saisho, C.A. Gautier, et al., PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy, J. Cell Biol. 189 (2010) 211–221.

[45] Y. Chen, G.W. Dorn 2nd., PINK1-phosphorylated mitofusin 2 is a Parkin receptor for culling damaged mitochondria, Science 340 (2013) 471–475.

[46] A. Eiyama, K. Okamoto, PINK1/Parkin-mediated mitophagy in mammalian cells, Curr. Opin. Cell Biol. 33 (2015) 95–101.

[47] F. Billia, L. Hauck, F. Konecny, V. Rao, J. Shen, T.W. Mak, PTEN-inducible kinase 1 (PINK1)/Park6 is indispensable for normal heart function, Proc. Natl. Acad. Sci. U. S. A. 108 (2011) 9572–9577.

[48] D.A. Kubli, X. Zhang, Y. Lee, R.A. Hanna, M.N. Quinsay, C.K. Nguyen, et al., Parkin protein deficiency exacerbates cardiac injury and reduces survival following myocardial infarction, J. Cell Biol. 288 (2013) 915–926.

[49] G. Bjorkoy, T. Lamark, A. Brech, H. Outzen, M. Perander, A. Overvatn, et al., p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death, J. Cell Biol. 171 (2005) 603–614.

[50] M. Komatsu, S. Waguri, M. Koike, Y.S. Sou, T. Ueno, T. Hara, et al., Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice, Cell 131 (2007) 1149–1163.

[51] K. Okatsu, K. Saisho, M. Shimanuki, K. Nakada, H. Shitara, Y.S. Sou, et al., p62/SQSTM1 cooperates with Parkin for perinuclear clustering of depolarized mitochondria, Genes Cells 15 (2010) 887–900.

[52] D. Narendra, L.A. Kane, D.N. Hauser, I.M. Fearnley, R.J. Youle, p62/SQSTM1 is required for Parkin-induced mitochondrial clustering but not mitophagy; VDAC1 is dispensable for both, Autophagy 6 (2010) 1090–1106.

[53] K. Lu, I. Psakhye, S. Jentsch, Autophagic clearance of PolyQ proteins mediated by ubiquitin-Atg8 adaptors of the conserved CUET protein family, Cell 158 (2014) 549–563.

[54] S. Geisler, K.M. Holmstrom, D. Skujat, F.C. Fiesel, O.C. Rothfuss, P.J. Kahle, et al., PINK1/Parkin-mediated mitophagy is dependent on VDAC1 and p62/SQSTM1, Nat. Cell Biol. 12 (2010) 119–131.

[55] Y. Sun, A.A. Vashisht, J. Tchieu, J.A. Wohlschlegel, L. Dreier, Voltage-dependent anion channels (VDACs) recruit Parkin to defective mitochondria to promote mitochondrial autophagy, J. Biol. Chem. 287 (2012) 40652–40660.

[56] M. Lazarou, D.A. Sliter, L.A. Kane, S.A. Sarraf, C. Wang, J.L. Burman, et al., The ubiquitin kinase PINK1 recruits autophagy receptors to induce mitophagy, Nature 524 (2015) 309–314.

[57] C.T. Chu, J. Ji, R.K. Dagda, J.F. Jiang, Y.Y. Tyurina, A.A. Kapralov, et al., Cardiolipin externalization to the outer mitochondrial membrane acts as an elimination signal for mitophagy in neuronal cells, Nat. Cell Biol. 15 (2013) 1197–1205.

[58] A. Sugiura, G.L. McLelland, E.A. Fon, H.M. McBride, A new pathway for mitochondrial quality control: mitochondrial-derived vesicles, EMBO J. 33 (2014) 2142–2156.

[59] V. Soubannier, G.L. McLelland, R. Zunino, E. Braschi, P. Rippstein, E.A. Fon, et al., A vesicular transport pathway shuttles cargo from mitochondria to lysosomes, Curr. Biol. 22 (2012) 135–141.

[60] G.L. McLelland, V. Soubannier, C.X. Chen, H.M. McBride, E.A. Fon, Parkin and PINK1 function in a vesicular trafficking pathway regulating mitochondrial quality control, EMBO J. 33 (2014) 282–295.

[61] J. Zhang, P.A. Ney, Mechanisms and biology of B-cell leukemia/lymphoma 2/adeno-virus E1B interacting protein 3 and Nip-like protein X, Antioxid. Redox Signal. 14 (2011) 1959–1969.

[62] J.M. Boyd, S. Malstrom, T. Subramanian, L.K. Venkatesh, U. Schaeper, B. Elangovan, et al., Adenovirus E1B 19 kDa and Bcl-2 proteins interact with a common set of cellular proteins, Cell 79 (1994) 341–351.

[63] I. Novak, V. Kirkin, D.G. McEwan, J. Zhang, P. Wild, A. Rozenknop, et al., Nix is a selective autophagy receptor for mitochondrial clearance, EMBO Rep. 11 (2010) 45–51.

[64] W.X. Ding, H.M. Ni, M. Li, Y. Liao, X. Chen, D.B. Stolz, et al., Nix is critical to two distinct phases of mitophagy, reactive oxygen species-mediated autophagy induction and Parkin-ubiquitin-p62-mediated mitochondrial priming, J. Biol. Chem. 285 (2010) 27879–27890.

[65] M.N. Quinsay, R.L. Thomas, Y. Lee, A.B. Gustafsson, Bnip3-mediated mitochondrial autophagy is independent of the mitochondrial permeability transition pore, Autophagy 6 (2010) 855–862.

[66] K. Tracy, B.C. Dibling, B.T. Spike, J.R. Knabb, P. Schumacker, K.F. Macleod, BNIP3 is an RB/E2F target gene required for hypoxia-induced autophagy, Mol. Cell. Biol. 27 (2007) 6229–6242.

[67] J. Zhang, P.A. Ney, Role of BNIP3 and NIX in cell death, autophagy, and mitophagy, Cell Death Differ. 16 (2009) 939–946.

[68] Y. Zhu, S. Massen, M. Terenzio, V. Lang, S. Chen-Lindner, R. Eils, et al., Modulation of serines 17 and 24 in the LC3-interacting region of Bnip3 determines pro-survival mitophagy versus apoptosis, J. Biol. Chem. 288 (2013) 1099–1113.

[69] S. Melser, E.H. Chatelain, J. Lavie, W. Mahfouf, C. Jose, E. Obre, et al., Rheb regulates mitophagy induced by mitochondrial energetic status, Cell Metab. 17 (2013) 719–730.

[70] K.M. Regula, K. Ens, L.A. Kirshenbaum, Inducible expression of BNIP3 provokes mitochondrial defects and hypoxia-mediated cell death of ventricular myocytes, Circ. Res. 91 (2002) 226–231.

[71] M.G. Yussman, T. Toyokawa, A. Odley, R.A. Lynch, G. Wu, M.C. Colbert, et al., Mitochondrial death protein Nix is induced in cardiac hypertrophy and triggers apoptotic cardiomyopathy, Nat. Med. 8 (2002) 725–730.

[72] A. Diwan, M. Krenz, F.M. Syed, J. Wansapura, X. Ren, A.G. Koesters, et al., Inhibition of ischemic cardiomyocyte apoptosis through targeted ablation of Bnip3 restrains postinfarction remodeling in mice, J. Clin. Investig. 117 (2007) 2825–2833.

[73] Y. Lee, H.Y. Lee, R.A. Hanna, A.B. Gustafsson, Mitochondrial autophagy by Bnip3 involves Drp1-mediated mitochondrial fission and recruitment of Parkin in cardiac myocytes, Am. J. Physiol. Heart Circ. Physiol. 301 (2011) H1924–H1931.

[74] G. Chen, Z. Han, D. Feng, Y. Chen, L. Chen, H. Wu, et al., A regulatory signaling loop comprising the PGAM5 phosphatase and CK2 controls receptor-mediated mitophagy, Mol. Cell 54 (2014) 362–377.

[75] H. Wu, D. Xue, G. Chen, Z. Han, L. Huang, C. Zhu, et al., The BCL2L1 and PGAM5 axis defines hypoxia-induced receptor-mediated mitophagy, Autophagy 10 (2014) 1712–1725.

[76] W. Lu, S.S. Karuppagounder, D.A. Springer, M.D. Allen, L. Zheng, B. Chao, et al., Genetic deficiency of the mitochondrial protein PGAM5 causes a Parkinson's-like movement disorder, Nat. Commun. 5 (2014) 4930.

[77] E.Y. Chan, S. Kir, S.A. Tooze, siRNA screening of the kinome identifies ULK1 as a multidomain modulator of autophagy, J. Biol. Chem. 282 (2007) 25464–25474.

[78] N. Mizushima, The role of the Atg1/ULK1 complex in autophagy regulation, Curr. Opin. Cell Biol. 22 (2010) 132–139.

[79] W. Wu, W. Tian, Z. Hu, G. Chen, L. Huang, W. Li, et al., ULK1 translocates to mitochondria and phosphorylates FUNDC1 to regulate mitophagy, EMBO Rep. 15 (2014) 566–575.

[80] G. Sun, P. Ye, K. Murai, M.F. Lang, S. Li, H. Zhang, et al., miR-137 forms a regulatory loop with nuclear receptor TLX and LSD1 in neural stem cells, Nat. Commun. 2 (2011) 529.

[81] W. Li, X. Zhang, H. Zhuang, H.G. Chen, Y. Chen, W. Tian, et al., MicroRNA-137 is a novel hypoxia-responsive microRNA that inhibits mitophagy via regulation of two mitophagy receptors FUNDC1 and NIX, J. Biol. Chem. 289 (2014) 10691–10701.

[82] L. Xing, M. Zhang, D. Chen, Smurf control in bone cells, J. Cell. Biochem. 110 (2010) 554–563.

[83] A. Orvedahl, R. Sumpter Jr., G. Xiao, A. Ng, Z. Zou, Y. Tang, et al., Image-based genome-wide siRNA screen identifies selective autophagy factors, Nature 480 (2011) 113–117.

[84] D.P. Huynh, M. Dy, D. Nguyen, T.R. Kiehl, S.M. Pulst, Differential expression and tissue distribution of parkin isoforms during mouse development, Brain Res. Dev. Brain Res. 130 (2001) 173–181.

[85] A.C. Pawlyk, B.I. Giasson, D.M. Sampathu, F.A. Perez, K.L. Lim, V.L. Dawson, et al., Novel monoclonal antibodies demonstrate biochemical variation of brain parkin with age, J. Biol. Chem. 278 (2003) 48120–48128.

[86] M.S. Goldberg, S.M. Fleming, J.J. Palacino, C. Cepeda, H.A. Lam, A. Bhatnagar, et al., Parkin-deficient mice exhibit nigrostriatal deficits but not loss of dopaminergic neurons, J. Biol. Chem. 278 (2003) 43628–43635.

[87] T. Kataoka, N. Holler, O. Micheau, F. Martinon, A. Tinel, K. Hofmann, et al., A novel Bcl-2 homologue that induces apoptosis via its unique C-terminal extension, J. Biol. Chem. 276 (2001) 19548–19554.

[88] E. Hollville, R.G. Carroll, S.P. Cullen, S.J. Martin, Bcl-2 family proteins participate in mitochondrial quality control by regulating Parkin/PINK1-dependent mitophagy, Mol. Cell 55 (2014) 451–466.

[89] N. Mizushima, T. Yoshimori, Y. Ohsumi, The role of Atg proteins in autophagosome formation, Annu. Rev. Cell Dev. Biol. 27 (2011) 107–132.

[90] J.Y. Kim, K.J. So, S. Lee, J.H. Park, Bcl-rambo induces apoptosis via interaction with the adenine nucleotide translocator, FEBS Lett. 586 (2012) 3142–3149.
