
ARTICLES
https://doi.org/10.1038/s41594-018-0060-6

Structural basis for membrane anchoring and fusion regulation of the herpes simplex virus fusogen gB

Rebecca S. Cooper¹, Elka R. Georgieva²,³, Peter P. Borbat²,³, Jack H. Freed²,³ and Ekaterina E. Heldwein²,³*

Viral fusogens merge viral and cell membranes during cell penetration. Their ectodomains drive fusion by undergoing large-scale refolding, but little is known about the functionally important regions located within or near the membrane. Here we report the crystal structure of full-length glycoprotein B (gB), the fusogen from herpes simplex virus, complemented by electron spin resonance measurements. The membrane-proximal (MPR), transmembrane (TMD), and cytoplasmic (CTD) domains form a uniquely folded trimeric pedestal beneath the ectodomain, which balances dynamic flexibility with extensive, stabilizing membrane interactions. The postfusion conformation of the ectodomain suggests that the CTD likewise adopted the postfusion form. However, hyperfusogenic mutations, which destabilize the prefusion state of gB, target key interfaces and structural motifs that reinforce the observed CTD structure. Thus, a similar CTD structure must stabilize gB in its prefusion state. Our data suggest a model for how this dynamic, membrane-dependent ‘clamp’ controls the fusogenic refolding of gB.

Herpesviruses infect most of the world’s population for life by establishing latent infections from which they periodically reactivate. The three subfamilies of herpesviruses—α, β, and γ—have distinct replication strategies and pathogenesis. α-herpesviruses, which include herpes simplex viruses type 1 and 2 (HSV-1 and HSV-2), cause skin lesions, encephalitis¹, and keratitis². β-herpesviruses, including cytomegalovirus (CMV), are frequently asymptomatic but cause disseminated infections in immunocompromised patients, for example, transplant recipients³, and developmental abnormalities in neonates⁴. γ-herpesviruses, which include Epstein–Barr virus⁵ (EBV), are linked causally to several cancers.

All herpesviruses have similar virion structures and penetrate cells by using multiple viral and host proteins⁶ to catalyze a merger of their viral envelope with a host cell membrane. In HSV-1 and HSV-2, fusion requires four essential viral surface glycoproteins—gB, gD, gH, and gL—in addition to host receptors for gD and, potentially, other viral and host molecules⁶. Fusion by other herpesviruses also depends on the core entry glycoproteins gB, gH, and gL, paired with different tropism-determining partners⁷.

The conserved surface glycoprotein gB is a fusogen. Viral fusogens are type I membrane proteins anchored in the viral envelope that merge the viral and cell membranes during cell entry. They engage both membranes and are thought to provide energy to drive fusion by refolding from a high-energy prefusion to a low-energy postfusion conformation⁸. Structures of the soluble extracellular portions (or ectodomains) of many viral fusogens in both conformations revealed large-scale fusogenic rearrangements and pinpointed the location of the hydrophobic fusion peptides, or loops, that bind target cell membranes. Several crystal structures of gB ectodomains in the postfusion conformation⁹⁻¹² and a cryoelectron tomography (cryo-ET) reconstruction of an alternative conformation of HSV-1 gB on the surface of exosomes are available.

In addition to the ectodomain, gB contains three other regions: an external MPR, a single-pass TMD, and an intraviral, or cytoplasmic, domain (CTD). These domains represent ~20% of the polypeptide and are essential for fusion¹³⁻¹⁶. The CTD also restrains the fusogenic activity of gB. Point mutations, insertions, or truncations within this domain increase cell–cell fusion in the context of infected cells or uninfected cells transfected to express core entry glycoproteins, and are referred to as hyperfusogenic¹⁴,¹⁷⁻¹⁹. But, in the absence of any structural information, the mechanistic contributions of the MPR, TMD, and CTD to the gB-mediated fusion process remain unexplained.

Here we report the crystal structure of full-length gB from HSV-1, in which the MPR-TMD-CTD forms a uniquely folded trimeric pedestal underneath the ectodomain. The structure and complementary electron spin resonance measurements reveal that the dynamic nature of this pedestal is offset by extensive, stabilizing membrane interactions. While the CTD trimer is appended to the postfusion ectodomain, hyperfusogenic mutations target trimeric interfaces and structural motifs within the CTD or its membrane-interacting elements, all of which stabilize the observed structure. Thus, the hyperfusogenic phenotype of the corresponding mutants can only be rationalized if the prefusion conformation of the CTD has a similar structure. We propose that the ordered, membrane-bound CTD acts as a clamp that restrains the fusogenic activity of gB by stabilizing the ectodomain in its prefusion conformation.

Results

gB structure determination. Construct gBΔ71, which lacks the signal peptide and proteolytically sensitive N terminus (Fig. 1a), was crystallized in a mixture of detergent and short amphipathic polymers (amphipols)²⁰, yielding two crystal forms (Table 1 and Supplementary Fig. 1). The 3.6-Å-resolution P321 structure and the 4.1-Å-resolution H32 structure were determined by molecular replacement using the ectodomain structure⁹ as a search model. The MPR and most of the TMD were resolved in both crystal forms, likely owing to restriction of TMD movement by multiple interactions.

---

¹Department of Molecular Biology and Microbiology, Tufts University School of Medicine, Boston, MA, USA. ²Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY, USA. ³National Biomedical Center for Advanced Electron Spin Resonance Technology (ACERT), Cornell University, Ithaca, NY, USA. *e-mail: katya.heldwein@tufts.edu

416 | NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

NATURE STRUCTURAL & MOLECULAR BIOLOGY ARTICLES

a
72 Ecto
725 MPR
753 TMD
770 h1a
775 h1b
801 CTD
805–812–
809 h2
814 h3
837
864
876
885
904

b
16 nm
Ecto
MPR
TMD
CTD

c
Hinge
770
753
Ecto
N
MPR
TMD
h1b
802
817
830
h1a
h2
h3
C
CTD

d
Top
Side
Bottom
MPR
TMD
CTD
N
C
N
C
N
C

Fig. 1 | The structure of gBΔ71. **a**, Bar diagram of the crystallized portion of gBΔ71. Unresolved regions are shown as dashed lines. Helix h3, unresolved in the crystals and identified from ESR experiments, is denoted by the hashed box. **b**, The structure of gBΔ71 is shown in surface (left) and cartoon (right) representations. The ectodomain rests atop a pedestal formed by the MPR-TMD-CTD trimer. All three protomers are colored by domain in the surface view, whereas in the cartoon view only one protomer is colored by domain. The inferred location of the membrane is shown schematically using peach (hydrophobic core) and light blue (polar headgroups) boxes. **c**, Secondary structure elements of the MPR-TMD-CTD pedestal are shown in an isolated protomer. **d**, Three views of the trimeric MPR-TMD-CTD pedestal, in which one protomer is colored by domain.

tions with the pendant hydrophobic groups of the amphipol, the ‘Gulliver effect’ phenomenon. However, the TMD C terminus and CTD were resolved only in the P321 structure, presumably owing to favorable crystal contacts (Supplementary Fig. 1). The more complete P321 structure was thus used in all subsequent analyses.

The nearly full-length gBΔ71 is a ~23-nm-long trimer, in which the ectodomain spike rests upon a pedestal composed of the MPR, TMD, and CTD (Fig. 1b). While the ectodomain adopted the post-fusion conformation, the uniquely folded trimeric pedestal is new.

The MPR structure. The top of the pedestal is formed by three symmetry-related MPR helices positioned underneath the ectodomain and parallel to the membrane bilayer. The N terminus of the MPR, which connects the MPR helix to the ectodomain, is unresolved in the structure (Fig. 1). The MPR helix makes no obvious contacts with the rest of the protein and is probably buttressed by the mixed micelle cushion. The poor side chain densities and high B factors in this region are consistent with the absence of restraining contacts and suggest that the MPR helix is dynamic. Three residues within the MPR helix, V764, G766, and F770, are invariant among herpesviruses (Supplementary Fig. 2a) and may be functionally important.

HSV-1 gB harboring a point mutation of residue G766 or F770 does not complement gB-null virus despite adequate surface expression.

The TMD trimer has an inverted teepee shape. The TMD forms a nearly straight helix positioned perpendicular to the MPR helix (Fig. 1c). It includes a hydrophilic fragment previously assigned to the CTD that extends beyond the cytoplasmic face of the lipid bilayer (Fig. 1b). The three symmetry-related TMD helices cross at a ~46° angle to form a unique inverted teepee (Figs. 1 and 2a,b), in which each protomer buries ~770.5 Å² of its accessible surface area (ASA). At the N terminus, the TMD helices are splayed and do not interact. Although the TMD trimer does not resemble a classic coiled coil, its narrow C terminus (residues A791–V798) is stabilized by knob-in-hole packing (Fig. 2c). The close fit of the TMD helices is enabled by the small or absent side chains of G787, A790, A791, and A794 (Fig. 2a). These residues are completely conserved among α-herpesviruses, implying a similar TMD arrangement (Supplementary Fig. 3). More distantly related β- and γ-herpesviruses (Supplementary Fig. 2a) have bulky residues at these positions, so their TMDs likely cross at different points and angles. Interactions between the hydrophilic C termini of the TMD helices and the CTD further stabilize this

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

ARTICLES

NATURE STRUCTURAL & MOLECULAR BIOLOGY

Table 1 | Data collection and refinement statistics

| FL-gBΔ71 (5V2S)<sup>a</sup> | FL-gBΔ71 (6BM8) |
| --- | --- |
| **Data collection** |  |
| Space group | P321 | H32 |
| Cell dimensions |  |  |
| $a, b, c$ (Å) | 118.66, 118.66, 216.46 | 118.60, 118.60, 800.58 |
| $\alpha, \beta, \gamma$ (°) | 90, 90, 120 | 90, 90, 120 |
| Resolution (Å) | 108.23–3.60<br>(3.73–3.60)<sup>b</sup> | 101.87–3.99<br>(4.37–3.99) |
| $R_{\text{merge}}$ | 0.3297 (1.833) | 0.109 (1.221) |
| $R_{\text{meas}}$ | 0.3256 (1.998) | 0.131 (1.46) |
| $R_{\text{pim}}$ | 0.1196 (0.6654) | 0.072 (0.790) |
| $I / \sigma(I)$ | 6.5 (1.15) | 7.00 (1.04) |
| $CC_{1/2}$ | 0.98 (0.61) | 0.999 (0.506) |
| Completeness (%) | 99.70 (99.76) | 99.00 (99.10) |
| Redundancy | 8.4 (8.5) | 3.3 (3.3) |
| **Refinement** |  |
| Resolution (Å) | 108.23–3.60 | 101.87–4.10 |
| No. reflections<sup>c</sup> | 21,048 (1,673) | 17,450 (1,742) |
| $R_{\text{work}} / R_{\text{free}}$ | 0.2444 / 0.2732 | 0.2686 / 0.2855 |
| No. atoms | 5,716 | 5,352 |
| Protein | 5,646 | 5,257 |
| Ligand<sup>d</sup> | 70 | 95 |
| $B$ factors |  |  |
| Protein | 119.48 | 182.02 |
| Ligand | 119.10 | 181.30 |
| R.m.s. deviations |  |  |
| Bond length (Å) | 150.39 | 221.81 |
| Bond angle (°) | 0.003 | 0.005 |

<sup>a</sup>FL-gBΔ71 (P321), 1 crystal; FL-gBΔ71 (H32), 1 crystal.<br>
<sup>b</sup>Values in parentheses are for highest-resolution shell.<br>
<sup>c</sup>Values in parentheses are for test set.<br>
<sup>d</sup>Ligands are three glycans (five $N$-acetylglucosamine moieties) in the P321 structure and four glycans (six $N$-acetylglucosamine moieties and one β-mannose) in the H32 structure.

are the only invariant CTD core positions among all herpesviruses (Supplementary Fig. 2a), which suggests that the zigzag is an essential conserved structural element within the TMD-CTD module.

Below the proline zigzag, the remaining CTD core residues form a triangular base stabilized by multiple interactions (Figs. 2d and 3, and Supplementary Table 1). Each protomer buries ~2,051 Å² of ASA, with helix h2 embraced by helices h1a, h1b, and h2 of one neighbor and contacting helices h1a, h1b, and h2 of the other neighbor (Figs. 1d and 2d). The trimeric CTD interface is highly conserved among α-herpesviruses (Supplementary Figs. 3 and 5) and is likely functionally important.

### Charge distribution and orientation in the membrane.

The placement of MPR-TMD-CTD in the membrane was inferred from the surface distribution of hydrophobic, polar, and charged residues (Fig. 3a). Both the hydrophobic TMD and the hydrophobic MPR helix are embedded in the hydrocarbon core, with only the serine spine formed by the S762, S765, and S769 side chains protruding into the headgroups of the outer leaflet (Fig. 3a,b). This arrangement places the ectodomain fusion loops into the headgroups of the outer leaflet, consistent with binding of isolated gB ectodomain to membranes<sup>24</sup>. Such deep MPR placement within the membrane is expected in other homologs on the basis of conservation of hydrophobic and polar residues (Supplementary Fig. 2a) and suggests that the MPR helices may promote fusion by facilitating lipid mixing.

On the opposite side of the bilayer, the side chains of R796 and R800 in the TMD and K807, R858, K862, K864, and K865 in the CTD form a positively charged ‘belt’ (Fig. 3a). Most of these residues are highly conserved in α-herpesviruses, and even gB homologs from β- and γ-herpesviruses have at least one basic residue at the TMD C terminus and the h2 C terminus (Supplementary Fig. 2a). This ‘basic belt’ could help brace the CTD against the membrane.

In contrast, the CTD base is negatively charged and rich in acidic residues (D834, D836, E837, E842, and E845) (Fig. 3c). Among 13 α-herpesvirus gB homologs (Supplementary Fig. 3), D836 is invariant and the other four residues are conserved. At several of these positions, substitutions that eliminate negative charge reduce the electrostatic potential of the CTD base. Yet, the distinct phenotypes of these mutants imply that different sections of this ‘acidic face’ have specific functional roles. The D836A mutant of HSV-2 gB is hyperfusogenic in virus-free cell–cell fusion assays<sup>25</sup>, and the D836 residue, located on the periphery of the CTD core, could participate in fusion by accepting fusion-triggering input from the gH cytail<sup>22</sup>. Conversely, E842S and E845S mutations in the HSV-1 gB have no effect on fusion in the same assay<sup>22</sup>, and these sites might instead provide a docking site for a positively charged binding partner, for example, a tegument protein, during viral morphogenesis.

### A membrane-anchoring helix within the CTD C terminus.

The CTD C terminus, residues 866–904, is unresolved in the crystal structure. This region restrains the fusogenic activity of gB, as its truncation leads to a syncytial phenotype<sup>26</sup>. It contains a putative amphipathic helix, h3, that forms in the presence of membranes and enables stable attachment of isolated CTD to membranes<sup>19</sup>. To determine the structure of the CTD C terminus and its orientation within the membrane, we employed continuous-wave electron spin resonance spectroscopy (CW-ESR) on isolated CTD, residues 801–904, bound to synthetic membrane vesicles composed of 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine (POPC) and 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphate (POPA) in 1:1 molar ratio. Single-cysteine CTD mutants from H861C to K885C were expressed in *Escherichia coli*, purified, and spin labeled. Mutations preserved gB fusogenicity, as assessed by virus-free cell–cell fusion assay (Supplementary Fig. 6 and Supplementary Note 1), and likely did not globally perturb membrane interactions.

NATURE STRUCTURAL & MOLECULAR BIOLOGY

ARTICLES

Fig. 2 | Residues involved in formation of the trimeric MPR-TMD-CTD pedestal. **a**, TMD helices interact exclusively through hydrophobic residues (yellow and orange side chains), which mediate both hydrophobic contacts (side chain atoms) and van der Waals contacts (main chain atoms). **b**, Surface representation of **a** emphasizing contact between the helices, in three views. **c**, A subset of the TMD interface residues participate in a stabilizing knob-in-hole interaction, where A791, A794, F795, and V798 (orange) form a hole into which the A794 (pink) knob of the neighboring protomer fits. **d**, Side and top views of the CTD showing residues that form salt bridges and hydrogen bonds (pink) or hydrophobic interactions (yellow). Salt bridges and hydrogen bonds are indicated by dotted black lines.

All CW-ESR spectra in solution displayed narrow, sharp peaks characteristic of a highly mobile spin label attached to unstructured protein (Fig. 4a). In contrast, the peaks of CW-ESR spectra collected in the presence of acidic liposomes were broadened to varying degrees, indicating restricted spin-label mobility²⁷ (Fig. 4a) and suggesting that residues H861–K885 become less dynamic upon membrane binding. The depth of spin-label insertion (Fig. 4b) into the membrane was determined from CW-ESR microwave power saturation analysis of each position’s Ni(II)-diammine-2,2′-(ethane-1,2-diyldiimino) diacetic acid (NiEDDA) and O₂ accessibility (Supplementary Fig. 7a). The depth follows a periodic pattern for V876–R884, with a period of ~3.65 residues, similar to the ideal α-helix periodicity of 3.6, and we conclude that this span forms an amphipathic helix straddling the boundary of the headgroup and aqueous interface (Fig. 4c). The ESR-derived boundaries of h3 are consistent with secondary structure predictions and the proteolytic sensitivity of the isolated CTD²⁶, whereas the variable insertion depth along h3 suggests that it is kinked or otherwise distorted. The deepest residue, M879C, approaches the bilayer core location of the spin-labeled lipid 1-palmitoyl-2-stearoyl-(5-doxyl)-sn-glycero-3-phosphocholine (5PC), while the rest of the residues lie near the headgroup and aqueous interface. Structural features identified through the depth analysis were confirmed by the mobility pattern (Supplementary Fig. 7b), with the mobility of residues in h3 reflecting both their containment in this structural element and the extent to which membrane interactions confine them. Thus, binding of the unstructured CTD C terminus to the bilayer transforms it into a shallow membrane anchor composed of an amphipathic helix, one face of which interacts with the polar headgroups. Amphipathic helices are also found in the HIV Env CTD²⁸ and the fusion peptide of influenza hemagglutinin (HA)²⁹ and may be employed by fusogens to grip the membranes they remodel.

DEER analysis of free and membrane-bound CTD. Formation of helix h3 in the presence of membranes is accompanied by global organization of the CTD, as determined by double electron-electron resonance (DEER) spectroscopy on isolated CTD. The CTD is a trimer in solution¹⁹, which allowed us to obtain interprotomer distance distributions between symmetry-related spin labels introduced by single-cysteine mutations S803C, E816C, or E830C (Fig. 5a). Distance distributions for all mutants in solution were broad and centered at ~40 Å but narrowed in the presence of anionic liposomes (Fig. 5b), indicating that S803, E816, and E830 lie in highly dynamic regions that become more ordered upon membrane binding. This concurs with the increase in helicity and decrease in proteolytic susceptibility of the CTD in the presence of anionic liposomes²⁶.

Of the three CTD mutants, E816C displayed the closest agreement between the center of its membrane-bound DEER distance distribution and the E816(Cα)-E816′(Cα) crystal structure distance (where the prime symbol denotes a neighboring protomer), with measurements of ~32 Å (Fig. 5b) and 31.98 Å (Fig. 5a), respectively. The distance distribution for the membrane-bound S803C mutant was centered at ~24 Å (Fig. 5b), which is longer than the

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

ARTICLES NATURE STRUCTURAL & MOLECULAR BIOLOGY

a

Fusion  
loops  

MPR  
and  
TMD  

Basic  
belt  

b Serine spines  

c Acidic  
face  

Hydrocarbon tails  
Polar headgroups  

Hydrophobic  
Polar  

Basic  
Acidic  

Electrostatics  
-5 +5  
$k_{\mathrm{b}} * T / e_{\mathrm{c}}$

Fig. 3 | Orientation of the MPR-TMD-CTD segment in the membrane. **a–c**, Orientation inferred from residue distribution (left) and electrostatic surface potential (right) is shown in side (**a**), top (**b**), and bottom (**c**) views. Residues are shown as follows: hydrophobic (A, V, L, I, F, W, Y, G, P, M) in yellow, polar (S, T, N, Q, H) in cyan, acidic (D, E) in red, and basic (K, R) in marine. The inferred location of the membrane is shown schematically using peach (hydrophobic core) and light blue (polar headgroups) boxes. The hydrophobic MPR and TMD are buried in the nonpolar bilayer core except for the serine spine of the MPR, which protrudes into the polar headgroups of the outer leaflet. The basic belt in the TMD and CTD interacts with the polar headgroups of the inner leaflet. The inner face of the CTD is largely acidic.

S803(Cα)-S803′(Cα) distance of 18.11 Å in the crystal structure (Fig. 5a). Similar differences between interspin and backbone measurements have been reported previously ${ }^{30,31}$ and typically reflect contributions from the length and relative positions of spin-label tethers. The lower-than-expected signal intensity observed with both the membrane-bound S803C and E816C mutants (Fig. 5b) may indicate that spin label at these positions destabilizes the folded CTD structure.

In the third mutant, E830C, strong signal in the presence of membrane and the relatively broad distance distribution are consistent with the position of the residue in a flexible loop. Yet, while this distribution is centered at ~28 Å (Fig. 5b), the E830(Cα)-E830′(Cα) distance in the crystal structure is 45.64 Å (Fig. 5a). We hypothesize that, when the CTD binds the membrane bilayer, the E830-containing loop becomes more ordered and compact. Additionally, crystal packing of gBΔ71 or absence of the TMD in the ESR constructs could affect the conformation of this loop.

Fusion regulation by the CTD. The gBΔ71 crystal structure together with ESR measurements provides an excellent model for understanding fusion regulation by the CTD. Certain mutations in the CTD and the TMD C terminus result in enhanced or impaired cell–cell fusion. We classified all known fusion-altering mutations in HSV-1 or HSV-2 gB on the basis of their location in and potential effect on the structure (Fig. 6 and Supplementary Table 2), regardless of whether they were identified in clinical HSV isolates or engineered.

The largest group of hyperfusogenic mutations ${ }^{14,17,22,25,32-36}$, group 1, targets residues located at the trimeric CTD interfaces. These mutations are predicted to disrupt interprotomer bonds, including hydrogen bonds (Y810A ${ }^{25}$, Y849A ${ }^{14}$, S854F ${ }^{32}$, E857A ${ }^{25}$, and E857D ${ }^{33}$), van der Waals contacts (T813I ${ }^{17}$), and hydrophobic contacts (V853A ${ }^{25}$), owing to side chain substitutions. Group 2 contains the P805A ${ }^{25}$ and T859P ${ }^{37}$ mutations that could indirectly destabilize trimeric interfaces by disrupting important structural elements, the zigzag structure and helix h2, respectively. Group 3 contains mutations that target the basic belt, a membrane-binding element: R796A ${ }^{17}$ and R800A ${ }^{17}$ in the TMD and R858H ${ }^{34,38}$ and R858C ${ }^{17}$, K864S ${ }^{22}$, and K865S ${ }^{22}$ in the CTD. These mutations probably weaken membrane interactions of the CTD by reducing the positive charge in the vicinity of the membrane. Group 3 also contains truncations of helix h3 ${ }^{18,26,39}$. Eliminating this important membrane anchor reduces the kinetic energy barrier that gB must overcome during prefusion-to-postfusion refolding ${ }^{22,40}$ in HSV-1 ${ }^{18,26,39}$ and EBV ${ }^{40-42}$. A similar mechanism may account for enhanced fusion in HIV Env truncation mutants ${ }^{43}$. Finally, group 4 contains mutations for which the hyperfusogenic phenotype cannot yet be readily explained. These mutations include D836A ${ }^{25}$ and A855V ${ }^{35,44}$ in h2 as well as L817P ${ }^{36}$, L817H ${ }^{35}$, and insertions between E816 and L817 ${ }^{34}$ in the region immediately preceding the unresolved CTD loop. All of these residues are located at the periphery of the CTD core and could accept fusion-triggering input from the gH cytotail ${ }^{22}$. Nevertheless, most hyperfusogenic mutations map to either the trimeric or membrane interfaces and would be expected to disrupt either the CTD trimer or its membrane interactions.

Unlike hyperfusogenic mutants, CTD mutants that fuse poorly are uncommon and are typically expressed on the cell surface at very low levels ${ }^{25}$, likely due to protein misfolding. Structure analysis shows that most of the corresponding mutations shorten the side chains of hydrophobic residues along the trimeric CTD axis and likely eliminate interactions critical for basal CTD trimer stability (Fig. 6b). The only known CTD mutation that reduces the rate of viral entry despite adequate surface expression, A851V ${ }^{17}$, may impede fusion by increasing the number of hydrophobic contacts at the trimeric interface and further stabilizing the CTD trimer.

The clustering of hyperfusogenic mutations in the CTD indicates that a central function of this domain is stabilization of gB in its prefusion form. Given that these mutations map to key interfaces and structural motifs of the observed CTD structure, their phenotype can only be rationalized if this or a similar structure exists in the prefusion gB. We propose that the membrane-bound, trimeric CTD structure, observed in the crystals and inferred from the ESR data, is essential for restraining the fusogenic activity of gB.

Discussion

Many viral fusogens require regions lying within or near the membrane for activity. In HIV Env, the MPER houses the epitopes of several broadly neutralizing antibodies ${ }^{45}$, whereas the TMD influences its conformation, stability, and antigenicity ${ }^{46}$. In influenza HA, the

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

NATURE STRUCTURAL & MOLECULAR BIOLOGY

ARTICLES

Fig. 4 | Membrane interactions of the CTD C terminus. **a**, Representative CW-ESR spectrum pairs for positions that display small (R864C), moderate (V876C), and large (M879C) mobility differences in the absence (black) and presence (red) of liposomes. Spectral broadening in the liposome-containing traces reflects secondary structure formation and the strength of membrane interactions at each position. All spectra were normalized to the same number of spins. In the plots, some of the spectra in lipids were scaled up by a factor of 1.5–2 for better visibility. **b**, Dependence of the depth parameter ($\Phi$) on spin-label position for CTD residues 861–885 and two spin-labeled lipid controls, 5PC and 1-palmitoyl-2-oleoylsn-glycero-3-phospho(tempo). **c**, Periodic arrangement of residues 876–884 reflecting their formation of the amphipathic helix h3, with one membrane-embedded and one solvent-exposed face.

TMD is required to bring fusion to completion ${ }^{47}$. Additionally, the CTDs of herpesvirus gB ${ }^{17,41,48}$, HIV Env ${ }^{45}$, and paramyxovirus F ${ }^{49}$ can control the fusogenic activity of their ectodomains across the membrane. However, structural information on these juxta-membrane regions is limited. Structures of the isolated MPER and TMD of HIV Env have been determined by NMR ${ }^{46,50}$, but no structures for large, lipid-associated CTDs have been described. Moreover, the only available high-resolution structures of full-length fusogens are of class II fusogens from dengue ${ }^{51}$ and Zika ${ }^{52}$ viruses, which lack CTDs and have ‘hook-shaped’ TMDs that do not span the membrane.

Here we presented the most complete structure of a fusogen with a membrane-spanning TMD and a large CTD. The gB structure shows that the MPR-TMD-CTD forms a uniquely folded trimeric pedestal beneath the ectodomain that interacts extensively with the membrane. The structure defines TMD boundaries more precisely and identifies regions of potential functional importance throughout the pedestal, including the MPR serine spine and the basic belt and acidic face of the CTD. Membrane binding organizes the CTD core and is accompanied by formation of the C-terminal amphipathic helix, a stabilizing bilayer anchor.

Collectively, structure-based analysis of a large panel of fusion-altering CTD mutations ${ }^{17,25,34-36,38}$ indicates that CTD trimerization is essential for gB folding and surface expression. Proper regulation of fusogenic activity further requires membrane binding of this trimer. The postfusion conformation of the gBΔ71 ectodomain suggests that the CTD pedestal was also captured in its postfusion form. However, because hyperfusogenic mutations both disrupt elements that rein-

force the observed CTD trimer structure and destabilize prefusion gB, a similar structure must stabilize the prefusion gB. Therefore, we hypothesize that, unlike the ectodomain, the CTD adopts similar conformations in the prefusion and postfusion states. Nevertheless, it is possible that the gBΔ71 structure is a hybrid of a postfusion ectodomain and a prefusion CTD, made possible by the amphipol and crystal packing, whereas the postfusion CTD structure is quite different.

We propose that the CTD—including the core observed in the crystals and the C-terminal amphipathic helix—constitutes an inhibitory membrane-dependent clamp that stabilizes the gB ectodomain in its prefusion conformation. Lacking this clamp, isolated gB ectodomain adopts only the postfusion conformation ${ }^{9,53}$. Moreover, the fully folded CTD clamp requires intact membrane, which explains why even full-length gB adopts the postfusion conformation upon detergent solubilization (this work and ref. ${ }^{54}$). Solubilization disrupts the bilayer scaffold surrounding gB and destabilizes the CTD clamp, enabling the ectodomain to refold into the postfusion conformation. In agreement with this hypothesis, non-postfusion gB conformations have thus far only been observed in gB-containing exosomes ${ }^{55,56}$. Our results illustrate the need to study complete fusogens in a bilayer environment, especially those containing large membrane-anchoring segments.

The gB structure presented here may help illuminate its unusual activation mechanism. All viral fusogens face the challenge of deploying at the correct moment. Class I and class II fusogens rely upon priming systems that involve proteolytic cleavage of an inactive precursor or chaperone to prevent premature activation, whereas vesicular stomatitis virus (VSV) G and gp64 from class III circumvent the issue through

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

ARTICLES

NATURE STRUCTURAL & MOLECULAR BIOLOGY

a

S803  
E830  
E816  

L801 (N)  
K865 (C)  

b

Normalized DEER amplitude

0.3  
— 100 μM prot  
No lipid  
— 150 μM prot  
P/L 1:275  

0.2  

0.1  

0.0  

0   1   2  

Time (μs)

S803C  

E816C  

E830C  

24 Å  
44 Å  
32 Å  
45 Å  
28 Å  
38 Å  

Distance (Å)

Fig. 5 | Interprotomer distances in the isolated CTD. **a**, Positions of DEER labeling sites and their distances in the crystal structure of gBΔ71. For clarity, only residues 801–865 of the CTD are shown. **b**, Experimental time-domain DEER data (left) and reconstructed interprotomer distance distributions (right) for the CTD mutants S803C, E816C, and E830C. Protein in buffer alone is disordered, with broad, widely varying separation between equivalent residues on different protomers. Conversely, distance distributions for samples with a protein: liposome (P/L) ratio of 1:275 coalesce around shorter separation distances, indicating global organization of the CTD. The S803C and E816C mutants were tested once, while two biological replicates of E830C produced superimposable distance distributions.

reversible conformational rearrangements. Lacking both a priming ‘safety system’ and the ability to regain its prefusion state, gB is instead stabilized in its prefusion conformation by its CTD and activated by other glycoproteins including the gH–gL heterodimer.

Although the ectodomain of the gH–gL heterodimer is thought to activate gB by directly binding its ectodomain, the 14-residue gH cytotail is also essential for gB activation. Cell–cell fusion is reduced by truncations of the gH cytotail. Moreover, fusion levels are proportionate to the length of the remaining gH cytotail such that an eight-residue tail maintains efficient fusion, a five-residue tail has twofold reduced fusion, and further truncations yield progressively lower fusion levels until a one-residue tail achieves only

a

b

Trimeric interfaces  
Secondary structure  
Membrane interfaces  
Unknown  

Axial trimeric interfaces  
Secondary structure  
Unknown  

Fig. 6 | Locations of fusion-altering mutations within the MPR-TMD-CTD structure. Mutated residues are shown as spheres and colored according to the scheme below. **a**, Hyperfusogenic mutations are divided into four groups depending on their location and effect on the CTD. Truncations of the amphipathic h3 helix (hashed orange boxes) disrupt membrane interactions and are assigned to group 3. **b**, Surface expression of gB is eliminated by alanine substitutions at several positions, notably at hydrophobic residues lining the central CTD axis (yellow).

background fusion levels. Although the precise role of the gH cytotail remains unclear, we hypothesize that it releases the gB clamp by acting as a wedge that disrupts either CTD core or its membrane interactions. Because its effectiveness is vastly reduced by truncation, the gH cytotail must be able to reach a certain point on the gB CTD to do this. Its potential targets could thus be any CTD features that stabilize the gB clamp; for example, interprotomer contacts in the CTD core or h3-membrane interactions. An extended eight-residue peptide would nearly span the height of the gB CTD and could interact with the CTD at multiple sites. Hyperfusogenic HSV-1 gB constructs still require activation by gH–gL heterodimer and are sensitive to gH tail length. Yet, they achieve higher fusion levels with shorter gH cytotails, probably because their mutated clamps are on a ‘hair trigger’ and are more easily released.

Combining our structure analysis and the published data, we propose a ‘clamp-and-wedge’ model of HSV fusion (Fig. 7). The CTD clamp initially restrains the gB ectodomain in its high-energy prefusion form by an unknown mechanism. Binding of gD to one of its receptors results in activation of the gH–gL heterodimer, prompting the gH cytotail wedge to interact with the CTD clamp and destabilize it in some manner. Freed from the CTD’s restraining influence, the prefusion ectodomain then refolds irreversibly into its thermodynamically favored postfusion conformation while the CTD returns to its initial conformation. This mechanism for releasing the CTD clamp may explain why multiple glycoproteins are required to activate gB in both HSV-1 and other herpesvirus

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

NATURE STRUCTURAL & MOLECULAR BIOLOGY

ARTICLES

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 7 | The clamp-and-wedge model of HSV fusion. The CTD clamp restrains the gB ectodomain in its high-energy prefusion form. Binding of gD to one of its receptors, herpes virus entry mediator (HVEM) or nectin, (1) relieves it from an autoinhibited dimeric state (not shown) and results in activation of the gH-gL heterodimer (2). The ectodomain of activated gH-gL heterodimer interacts with the gB ectodomain, while the gH cytotail releases the clamp by wedging between the intertwined CTD protomers (3). The destabilized clamp loses its hold on the ectodomain, and gB proceeds to refold into its thermodynamically favored postfusion conformation, drawing the virion and cell membranes together (4). The clamp then returns to its initial conformation (5).

homologs. Although the CTD trimer must be destabilized during fusion, it does not need to come apart completely. Major ectodomain rearrangements coupled to restricted CTD motion could be accommodated by asymmetric refolding of the ectodomain^{8} and extensive rearrangement of the flexible MPR. However, if the observed CTD trimer represents solely the prefusion form, this domain could undergo permanent refolding or unfolding during fusion.

To influence the ectodomain conformation, the CTD must convey its own status through the intervening MPR and TMD. Extension of the TMD helices beyond the inner leaflet and into the CTD 'space' could enable them to serve as levers to efficiently transmit CTD movements to the TMD. Indeed, when the CTD core is ordered, the TMD helices are fully assembled into an inverted teepee. However, a disordered CTD core enforces no such constraints on the TMD. This contrast is an extreme example of how rearrangements in the regulatory CTD might propagate across the membrane. Nevertheless, the same pathway could transmit smaller, functionally relevant conformational changes within the CTD due to syncytial mutations or in response to a fusion-triggering signal.

How the CTD status is transmitted to the ectodomain through the MPR is less clear. The N terminus of the MPR is unresolved in both of our structures and, as the visible C terminus (the MPR helix) does not appear to contact the postfusion ectodomain extensively enough to influence either its stability or conformation, this domain likely looks very different in prefusion gB. Furthermore, if the prefusion and postfusion CTD conformations are indeed similar, major rearrangement of the MPR (and possibly the TMD) may be necessary to couple the CTD to the vastly different ectodomain conformations that are predicted.

Our model of full-length HSV-1 gB provides a first look at its fusion-restraining clamp as well as the most comprehensive picture of the membrane-interacting domains of any fusogen containing a membrane-spanning TMD and a large CTD. Given the many ways in which the structure of this clamp is dependent on the surrounding viral envelope, we hypothesize that the structure and function of the herpesvirus fusogen gB are uniquely controlled by the membrane. This new paradigm provides a starting point for exploring the persistent mysteries that surround gB, such as the nature of its interaction with the gH-gL heterodimer and the structure of its prefusion conformation.

**Methods**

Methods, including statements of data availability and any associated accession codes and references, are available at [https://doi.org/10.1038/s41594-018-0060-6](https://doi.org/10.1038/s41594-018-0060-6).

**References**

1. Kennedy, P. G. & Steiner, I. Recent issues in herpes simplex encephalitis. *J. Neurovirol.* **19**, 346–350 (2013).
2. Rowe, A. M. et al. Herpes keratitis. *Prog. Retin. Eye Res.* **32**, 88–101 (2013).
3. Santos, C. A. Cytomegalovirus and other β-herpesviruses. *Semin. Nephrol.* **36**, 351–361 (2016).
4. Britt, W. J. Congenital human cytomegalovirus infection and the enigma of maternal immunity. *J. Virol.* **91**, e02392 (2017).
5. Young, L. S., Yap, L. F. & Murray, P. G. Epstein-Barr virus: more than 50 years old and still providing surprises. *Nat. Rev. Cancer* **16**, 789–802 (2016).
6. Eisenberg, R. J. et al. Herpes virus fusion and entry: a story with many characters. *Viruses* **4**, 800–832 (2012).
7. Sathiyamoorthy, K., Chen, J., Longnecker, R. & Jardetzky, T. S. The COMPLEXity in herpesvirus entry. *Curr. Opin. Virol.* **24**, 97–104 (2017).
8. Harrison, S. C. Viral membrane fusion. *Virology* **479–480C**, 498–507 (2015).
9. Heldwein, E. E. et al. Crystal structure of glycoprotein B from herpes simplex virus 1. *Science* **313**, 217–220 (2006).
10. Backovic, M., Longnecker, R. & Jardetzky, T. S. Structure of a trimeric variant of the Epstein-Barr virus glycoprotein B. *Proc. Natl Acad. Sci. USA* **106**, 2880–2885 (2009).
11. Burke, H. G. & Heldwein, E. E. Crystal structure of the human cytomegalovirus glycoprotein B. *PLoS Pathog.* **11**, e1005227 (2015).
12. Chandramouli, S. et al. Structure of HCMV glycoprotein B in the postfusion conformation bound to a neutralizing human antibody. *Nat. Commun.* **6**, 8176 (2015).
13. Jones, N. A. & Geraghty, R. J. Fusion activity of lipid-anchored envelope glycoproteins of herpes simplex virus type 1. *Virology* **324**, 213–228 (2004).
14. Fan, Z. et al. Truncation of herpes simplex virus type 2 glycoprotein B increases its cell surface expression and activity in cell-cell fusion, but these properties are unrelated. *J. Virol.* **76**, 9271–9283 (2002).
15. Wanas, E., Efler, S., Ghosh, K. & Ghosh, H. P. Mutations in the conserved carboxy-terminal hydrophobic region of glycoprotein gB affect infectivity of herpes simplex virus. *J. Gen. Virol.* **80**, 3189–3198 (1999).
16. Lin, E. & Spear, P. G. Random linker-insertion mutagenesis to identify functional domains of herpes simplex virus type 1 glycoprotein B. *Proc. Natl Acad. Sci. USA* **104**, 13140–13145 (2007).
17. Gage, P. J., Levine, M. & Glorioso, J. C. Syncytium-inducing mutations localize to two discrete regions within the cytoplasmic domain of herpes simplex virus type 1 glycoprotein B. *J. Virol.* **67**, 2191–2201 (1993).
18. Baghian, A., Huang, L., Newman, S., Jayachandra, S. & Kousoulas, K. G. Truncation of the carboxy-terminal 28 amino acids of glycoprotein B specified by herpes simplex virus type 1 mutant amb1511-7 causes extensive cell fusion. *J. Virol.* **67**, 2396–2401 (1993).
19. Chowdary, T. K. & Heldwein, E. E. Syncytial phenotype of C-terminally truncated herpes simplex virus type 1 gB is associated with diminished membrane interactions. *J. Virol.* **84**, 4923–4935 (2010).
20. Zoonens, M. & Popot, J. L. Amphipols for each season. *J. Membr. Biol.* **247**, 759–796 (2014).

NATURE STRUCTURAL & MOLECULAR BIOLOGY | VOL 25 | MAY 2018 | 416-424 | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

ARTICLES

21. Zheng, Z., Maidji, E., Tugizov, S. & Pereira, L. Mutations in the carboxyl-terminal hydrophobic sequence of human cytomegalovirus glycoprotein B alter transport and protein chaperone binding. *J. Virol.* 70, 8029–8040 (1996).

22. Rogalin, H. B. & Heldwein, E. E. The interplay between the HSV-1 gB cytodomains and the gH cytotail during cell–cell fusion. *J. Virol.* 89, 12262–12272 (2015).

23. Langelaan, D. N., Wieczorek, M., Blouin, C. & Rainey, J. K. Improved helix and kink characterization in membrane proteins allows evaluation of kink sequence predictors. *J. Chem. Inf. Model.* 50, 2213–2220 (2010).

24. Hannah, B. P. et al. Herpes simplex virus glycoprotein B associates with target membranes via its fusion loops. *J. Virol.* 83, 6825–6836 (2009).

25. Ruel, N., Zago, A. & Spear, P. G. Alanine substitution of conserved residues in the cytoplasmic tail of herpes simplex virus gB can enhance or abolish cell fusion activity and viral entry. *Virology* 346, 229–237 (2006).

26. Silverman, J. L., Greene, N. G., King, D. S. & Heldwein, E. E. Membrane requirement for folding of the herpes simplex virus 1 gB cytodomain suggests a unique mechanism of fusion regulation. *J. Virol.* 86, 8171–8184 (2012).

27. Mchaourab, H. S., Lietzow, M. A., Hideg, K. & Hubbell, W. L. Motion of spin-labeled side chains in T4 lysozyme. Correlation with protein structure and dynamics. *Biochemistry* 35, 7692–7704 (1996).

28. Haffar, O. K., Dowbenko, D. J. & Berman, P. W. The cytoplasmic tail of HIV-1 gp160 contains regions that associate with cellular membranes. *Virology* 180, 439–441 (1991).

29. Lai, A. L., Park, H., White, J. M. & Tamm, L. K. Fusion peptide of influenza hemagglutinin requires a fixed angle boomerang structure for activity. *J. Biol. Chem.* 281, 5760–5770 (2006).

30. Stoll, S. et al. Double electron–electron resonance shows cytochrome P450cam undergoes a conformational change in solution upon binding substrate. *Proc. Natl Acad. Sci. USA* 109, 12888–12893 (2012).

31. Zou, P., Bortolus, M. & McHaourab, H. S. Conformational cycle of the ABC transporter MsbA in liposomes: detailed analysis using double electron–electron resonance spectroscopy. *J. Mol. Biol.* 393, 586–597 (2009).

32. Walev, I., Lingen, M., Lazzaro, M., Weise, K. & Falke, D. Cyclosporin A resistance of herpes simplex virus–induced “fusion from within” as a phenotypical marker of mutations in the Syn 3 locus of the glycoprotein B gene. *Virus Genes* 8, 83–86 (1994).

33. Muggeridge, M. I. Characterization of cell–cell fusion mediated by herpes simplex virus 2 glycoproteins gB, gD, gH and gL in transfected cells. *J. Gen. Virol.* 81, 2017–2027 (2000).

34. Cai, W. H., Gu, B. & Person, S. Role of glycoprotein B of herpes simplex virus type 1 in viral entry and cell fusion. *J. Virol.* 62, 2596–2604 (1988).

35. Engel, J. P., Boyer, E. P. & Goodman, J. L. Two novel single amino acid syncytial mutations in the carboxy terminus of glycoprotein B of herpes simplex virus type 1 confer a unique pathogenic phenotype. *Virology* 192, 112–120 (1993).

36. Diakidi-Kosta, A., Michailidou, G., Kontogounis, G., Sivropoulou, A. & Arsenakis, M. A single amino acid substitution in the cytoplasmic tail of the glycoprotein B of herpes simplex virus 1 affects both syncytium formation and binding to intracellular heparan sulfate. *Virus Res.* 93, 99–108 (2003).

37. Muggeridge, M. I., Grantham, M. L. & Johnson, F. B. Identification of syncytial mutations in a clinical isolate of herpes simplex virus 2. *Virology* 328, 244–253 (2004).

38. Bzik, D. J., Fox, B. A., DeLuca, N. A. & Person, S. Nucleotide sequence of a region of the herpes simplex virus type 1 gB glycoprotein gene: mutations affecting rate of virus entry and cell fusion. *Virology* 137, 185–190 (1984).

39. Foster, T. P., Melancon, J. M. & Kousoulas, K. G. An α-helical domain within the carboxyl terminus of herpes simplex virus type 1 (HSV-1) glycoprotein B (gB) is associated with cell fusion and resistance to heparin inhibition of cell fusion. *Virology* 287, 18–29 (2001).

40. Chen, J., Zhang, X., Jardetzky, T. S. & Longnecker, R. The Epstein–Barr virus (EBV) glycoprotein B cytoplasmic C-terminal tail domain regulates the energy requirement for EBV-induced membrane fusion. *J. Virol.* 88, 11686–11695 (2014).

41. Garcia, N. J., Chen, J. & Longnecker, R. Modulation of Epstein–Barr virus glycoprotein B (gB) fusion activity by the gB cytoplasmic tail domain. *MBio* 4, e00571–12 (2013).

42. Haan, K. M., Lee, S. K. & Longnecker, R. Different functional domains in the cytoplasmic tail of glycoprotein B are involved in Epstein–Barr virus–induced membrane fusion. *Virology* 290, 106–114 (2001).

43. Postler, T. S. & Desrosiers, R. C. The tale of the long tail: the cytoplasmic domain of HIV-1 gp41. *J. Virol.* 87, 2–15 (2013).

44. Haanes, E. J., Nelson, C. M., Soule, C. L. & Goodman, J. L. The UL45 gene product is required for herpes simplex virus type 1 glycoprotein B–induced fusion. *J. Virol.* 68, 5825–5834 (1994).

45. Chen, J. et al. HIV-1 ENVELOPE. Effect of the cytoplasmic domain on antigenic characteristics of HIV-1 envelope glycoprotein. *Science* 349, 191–195 (2015).

46. Dev, J. et al. Structural basis for membrane anchoring of HIV-1 envelope spike. *Science* 353, 172–175 (2016).


NATURE STRUCTURAL & MOLECULAR BIOLOGY


47. Kemble, G. W., Danieli, T. & White, J. M. Lipid-anchored influenza hemagglutinin promotes hemifusion, not complete fusion. *Cell* 76, 383–391 (1994).

48. Nixdorf, R., Klupp, B. G., Karger, A. & Mettenleiter, T. C. Effects of truncation of the carboxy terminus of pseudorabies virus glycoprotein B on infectivity. *J. Virol.* 74, 7137–7145 (2000).

49. Waning, D. L., Russell, C. J., Jardetzky, T. S. & Lamb, R. A. Activation of a paramyxovirus fusion protein is modulated by inside-out signaling from the cytoplasmic tail. *Proc. Natl Acad. Sci. USA* 101, 9217–9222 (2004).

50. Sun, Z. Y. et al. Disruption of helix-capping residues 671 and 674 reveals a role in HIV-1 entry for a specialized hinge segment of the membrane proximal external region of gp41. *J. Mol. Biol.* 426, 1095–1108 (2014).

51. Zhang, X. et al. Cryo-EM structure of the mature dengue virus at 3.5-Å resolution. *Nat. Struct. Mol. Biol.* 20, 105–110 (2013).

52. Sirohi, D. et al. The 3.8 Å resolution cryo-EM structure of Zika virus. *Science* 352, 467–470 (2016).

53. Vitu, E., Sharma, S., Stampfer, S. D. & Heldwein, E. E. Extensive mutagenesis of the HSV-1 gB ectodomain reveals remarkable stability of its postfusion form. *J. Mol. Biol.* 425, 2056–2071 (2013).

54. Patrone, M. et al. Enhanced expression of full-length human cytomegalovirus fusion protein in non-swelling baculovirus-infected cells with a minimal fed-batch strategy. *PLoS One* 9, e90753 (2014).

55. Zeev-Ben-Mordehai, T. et al. Two distinct trimeric conformations of natively membrane-anchored full-length herpes simplex virus 1 glycoprotein B. *Proc. Natl Acad. Sci. USA* 113, 4176–4181 (2016).

56. Fontana, J. et al. The fusion loops of the initial prefusion conformation of herpes simplex virus 1 fusion protein point toward the membrane. *MBio* 8, e01268–17 (2017).

57. Harman, A., Browne, H. & Minson, T. The transmembrane domain and cytoplasmic tail of herpes simplex virus type 1 glycoprotein H play a role in membrane fusion. *J. Virol.* 76, 10708–10716 (2002).

58. Browne, H. M., Bruun, B. C. & Minson, A. C. Characterization of herpes simplex virus type 1 recombinants with mutations in the cytoplasmic tail of glycoprotein H. *J. Gen. Virol.* 77, 2569–2573 (1996).

59. Morin, A. et al. Collaboration gets the most out of software. *eLife* 2, e01456 (2013).


Acknowledgements

We thank NE-CAT staff for help with X-ray data collection and H. Rogalin for help with the cell–cell fusion assay. We also thank P. G. Spear (Northwestern University), R. J. Eisenberg (University of Pennsylvania), and G. H. Cohen (University of Pennsylvania) for the gift of plasmids and J. M. Coffin (Tufts University) for the gift of CHO cells. This work was funded by NIH grant 1R21AI107171 (E.E.H.), the Burroughs Wellcome Fund (E.E.H.), and the NIH Ruth L. Kirschstein NRSA postdoctoral fellowship 1F32GM115060 (R.S.C.). The research of E.E.H. was supported in part by a Faculty Scholar grant from the Howard Hughes Medical Institute. ESR experiments were funded by NIH grants P41GM103521 (J.H.F., ACERT) and R01GM123779 (J.H.F. and E.R.G.). This work is based upon research conducted at the Northeastern Collaborative Access Team beamlines, which are funded by the National Institute of General Medical Sciences from the NIH (P41GM103403). The Pilatus 6M detector on the 24-ID-C beamline is funded by an NIH-ORIP HEI grant (S10RR029205). This research used resources of the Advanced Photon Source, a US Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract DE-AC02-06CH11357. All software was installed and maintained by SBGrid<sup>59</sup>.


Author contributions

R.S.C. designed experiments, cloned the constructs, produced recombinant proteins, crystallized gBdelta71, collected diffraction data, phased the data and determined the structure, built and refined the models, carried out cell-cell fusion assays, analyzed the data, and wrote the manuscript. E.R.G. designed ESR studies, carried out spin labeling and lipid reconstitution, collected, analyzed, and interpreted ESR data, and wrote the manuscript. P.P.B. collected, analyzed, and interpreted ESR data, and wrote the manuscript. J.H.F. wrote the manuscript. E.E.H. designed experiments, phased the data and determined the structure, built and refined the models, analyzed the data, and wrote the manuscript.


Competing interests

The authors declare no competing interests.


Additional information

Supplementary information is available for this paper at [https://doi.org/10.1038/s41594-018-0060-6](https://doi.org/10.1038/s41594-018-0060-6).

Reprints and permissions information is available at [www.nature.com/reprints](http://www.nature.com/reprints).

Correspondence and requests for materials should be addressed to E.E.H.

Publisher’s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

NATURE STRUCTURAL & MOLECULAR BIOLOGY

**Methods**

Construction of gB CTD mutants. Sequences encoding single-cysteine CTD mutants were cloned into pET24b. Many mutants were subcloned into pKH52 (wild-type CTD with a C-terminal His₆ tag) from plasmids encoding equivalent full-length gB (FL-gB) single-cysteine mutants with PstI and XhoI. Others were prepared with a QuikChange PCR strategy using the primers listed in Supplementary Table 3. Finally, some constructs were prepared using ‘splicing overlap extension’ PCR (SOE PCR) to modify the pKH52 construct.

Construction of full-length gB for functional assays. The sequences for FL-gB constructs with single cysteine mutations were cloned into the pCAGGS background. The K862C, K864C, K865C, K866C, V876C, T877C, D878C, M879C, V880C, M881C, R882C, K883C, R884C, and R885C mutants were generated in pPEP98 (wild-type FL-gB) by SOE PCR. Other FL-gB single-cysteine mutants, namely S803C, E830C, H861C, A863C, G867C, T868C, S869C, A870C, L871C, L872C, S873C, A874C, and K875C, were subcloned from the equivalent gB CTD single-cysteine mutant expression construct with SacI and XhoI. To facilitate this process, a modified pPEP98 construct called pRC30 was created in which an XhoI site was introduced after codon 904 while other SacI and XhoI sites were destroyed. This resulted in amino acids LE between residue 904 and the stop codon. All primers are listed in Supplementary Table 3.

Construction of gBΔ71. The crystallization construct gBΔ71 encodes residues 72–904 of FL-gB, an N-terminal honeybee melittin signal sequence, and a C-terminal His₆ tag in a pFastBac1 vector (pRC4). This construct lacks the signal peptide, residues 1–29, which is cleaved during protein maturation and is absent from the mature gB, and the proteolytically sensitive N terminus, residues 30–71, that causes heterogeneity in gB samples. To make pRC4, the sequence encoding the gB CTD with a C-terminal His₆ tag was excised from pKH11 using NheI and MfeI and subcloned into the sequence for full-length, untagged gB (pEV34), generating FL-gB (pRC2). Next, SOE PCR using flanking primers 5′-CGCGGAATTCAAAGGCCCTAC-3′ and 5′-CTCGAC GATGCAGTTTACC G-3′ along with mutagenesis primers 5′-ATCTATGCGA ACAAAAAACCGAAAAACCCAACG-3′ and 5′-GGTTTTTTGTTCG CATAGATGTAAGAAATGTACAC-3′ was used to remove codons 30–71.

Expression and purification of gBΔ71. Recombinant baculovirus encoding C-terminally His₆-tagged HSV-1 gBΔ71 was generated using Bac-to-Bac technology (Thermo Fisher). *Spodoptera frugiperda* (Sf9) cells (Thermo Fisher) were grown in Sf900-SFM medium (Thermo Fisher) to a density of 2 × 10⁶ cells/ml and infected with recombinant baculovirus by adding 14 ml of the viral stock from the third passage (P3) to 1.4 liters of cells. Sixty to 72 h after infection, cells were harvested by centrifugation at 4,000g and resuspended in 50 mM Tris pH 8.0, 150 mM NaCl, and 10% glycerol (solubilization buffer, SB) plus 0.1 mM PMSF. After an additional centrifugation, cell pellets were frozen at –80 °C. Prior to purification, cell pellets were defrosted at 4 °C and resuspended in SB with 1 mM PMSF and 1× Roche protease inhibitor cocktail. The suspension was lysed by passing it three times through an M-110S microfluidizer, according to the manufacturer’s instructions, and clarified by centrifugation at 4,000 g for 25 min. Crude membrane fraction was isolated from the clarified lysate by ultracentrifugation at 150,000g for 90 min and resuspended in 50 ml of SB plus 0.1 mM PMSF using a dounce homogenizer. Membrane proteins were extracted by adding 1.2% n-dodecyl-β-D-maltopyranoside (DDM, D310LA, Anatrace) and incubating with 1 ml of Ni Superflow resin (GE Healthcare) in the presence of 10 mM imidazole. After incubation for 16 h, the resin was washed with 15 column volumes (CV) of 20 mM Tris pH 8, 150 mM NaCl, 5% glycerol, 0.05% DDM, and 20 mM imidazole (WB1) and 25 CV of 20 mM Tris pH 8.0, 150 mM NaCl, 5% glycerol, 0.05% DDM, and 35 mM imidazole (WB2). gBΔ71 was eluted with 20 mM Tris pH 7.5, 150 mM NaCl, 5% glycerol, 0.05% DDM, and 300 mM imidazole (EB) and concentrated in an Ultra-15 (molecular weight cutoff of 100 kDa) concentrator (Millipore EMD). gBΔ71 was further purified by size-exclusion chromatography using a Superdex 200 column (GE Healthcare) equilibrated with 20 mM Tris-HCl pH 7.5, 150 mM NaCl, 5% glycerol, and 0.05% DDM (GF buffer).

Crystallization and structure determination. Initially, gBΔ71 crystallized in several maltosides, yet the MPR-TMD-CTD portion was largely unresolved in these crystals. To stabilize the membrane regions, we obtained crystals in the presence of amphipols. Protein in GF buffer was concentrated to 3.6 mg/ml using an Ultra-4 (molecular weight cutoff of 100 kDa) concentrator (Millipore EMD) and passed through a 0.1-µm Ultrafree centrifugal filter (Millipore EMD). The protein was supplemented with A8-35 (Anatrace), mixed gently, and incubated at room temperature for 30 min. Crystals were grown by vapor diffusion in hanging drops containing 1 µl of protein and 1 µl of reservoir solution. Crystals appeared after 3–4 weeks and grew for an additional week. Higher PEG-3350 concentrations (16–18%) favored the formation of crystals in the P321 space group, whereas 12–14% PEG-3350 yielded crystals in the H32 space group. Some drops in the middle of this range contained H32 crystals at their periphery and P321 crystals in their center, potentially owing to uneven distribution of precipitant, detergent, and/or amphipol. While we hypothesize that association with A8-35 is necessary

to stabilize the TMD and enable P321 packing of gB, this highly anionic amphipol is thought to hinder crystallization, thereby requiring increased precipitant. The P321 crystal was grown using gBΔ71 in 0.05% DDM and 0.01% A8-35 and reservoir solution containing 16% PEG-3350, 0.1 M HEPES pH 7.2, and 0.15 M sodium formate. The H32 crystal was grown using gBΔ71 in 0.075% UM and 0.0075% A8-35 and reservoir solution containing 11% PEG-4000, 0.1 M Tris-HCl pH 7.5, and 0.1 M NaCl.

Cryoprotectant solution containing 20% glycerol was prepared by mixing two parts 50% glycerol with three parts reservoir solution. Crystals were cryoprotected by brief incubation in cryoprotectant followed by plunging into liquid nitrogen. X-ray data were collected at 100 K on beamlines 24ID-E and 24ID-C at the Advanced Photon Source at the Argonne National Laboratories. Both the P321 and H32 datasets were processed using XDS as implemented in the Northeastern Collaborative Access Team (NE-CAT) software pipeline RAPD (https://rapid.nec.aps.anl.gov).

A 3.6-Å-resolution P321 dataset was used to determine the structure by molecular replacement in Phaser using the HSV-1 gB ectodomain (PDB 2GUM) as a search model. There is a single gB polypeptide in the asymmetric unit, and a gB trimer is generated by a threefold crystallographic symmetry operation. The initial electron density revealed the locations of the MPR, TMD, and CTD, in addition to the ectodomain. The ectodomain adopted the postfusion conformation, presumably during membrane disruption caused by detergent extraction. The overall fold of the ectodomain within the gBΔ71 structure is very similar to that of isolated gB ectodomain, with r.m.s. deviation of 1.00 over 576 residues. An initial MPR-TMD-CTD model was built manually into the experimental density using Coot and refined against data truncated to 3.6-Å resolution. Prior to refinement, 8% of reflections were set aside for cross-validation. Model refinement included gradient minimization refinement of xyz coordinates and individual thermal parameters with optimization of X-ray/stereochemistry and X-ray/ADP weights, as implemented in phenix.refine. Secondary structure restraints were used throughout refinement. Rotamer restraints were imposed until the last stage of refinement. Iterative rounds of model rebuilding were carried out in Coot. The final model encompassed residues 104–865 (unresolved 71–103, 477–491, 726–752, 771–775, 818–829, 866–904, and the C-terminal His₆ tag) and four N-acetylglucosamine moieties at residues N141, N398, and N674. Sequence 754–770 was assigned to a helical density within the MPR region in the vicinity of the TMD, but, owing to poor side chain density, its precise register is unclear. The unresolved loop following residue 817 was also paired with the nearest residue, 830, rather than this position in more distant symmetry mates. This path is suggested by the weak electron density, and of the three possible pairings, it is the only one that does not require the polypeptide chain to make a long detour. According to MolProbity as implemented in phenix.refine, 95.7% of residues lie in the most favored regions of the Ramachandran plot and 4.3% lie in the additionally allowed regions. Relevant crystallographic statistics are listed in Table 1.

A 4.0-Å-resolution H32 dataset was used to determine the structure by molecular replacement in Phaser using the HSV-1 gB ectodomain structure (PDB 2GUM) as a search model. As in P321 crystals, there is a single gB polypeptide in the asymmetric unit and a gB trimer is generated by a threefold crystallographic symmetry operation. While the MPR and most of the TMD were visible in the initial electron density, in addition to the ectodomain, the CTD was unresolved. The model of MPR-TMD was built manually into the experimental density using Coot and refined against 4.1-Å-resolution data. Prior to refinement, 10% of reflections were set aside for cross-validation. Model refinement included gradient minimization refinement of xyz coordinates and individual thermal parameters, all as implemented in phenix.refine. Secondary structure restraints were used throughout refinement. Rotamer restraints were imposed until the last stage of refinement. Iterative rounds of model rebuilding were carried out in Coot. The final model encompassed residues 103–794 (unresolved 71–101, 483–489, 731–752, 796–904, and the C-terminal His₆ tag) and seven N-acetylglucosamine moieties at residues N141, N398, N430, and N674. As in the P321 structure, the precise location of residues 753–770 cannot be ascertained. But, residues 726–730 and the hinge between the MPR and TMD, residues 771–774, were resolved in the H32 structure, in contrast to the P321 structure. According to MolProbity, 92.56% of residues lie in the most favored regions and 7.13% lie in the additionally allowed regions of the Ramachandran plot. Two residues, P130 and P774, are Ramachandran outliers. Relevant crystallographic statistics are listed in Table 1.

Structure analysis. Sequence alignments were generated using Clustal Omega (https://www.ebi.ac.uk/Tools/msa/clustalo/) and ESPRIPT (http://esprit.ibcp.fr/ESPript/ESPript/). Interfaces were analyzed using PISA (http://www.ebi.ac.uk/pdbe/pisa/) and CCP4 Contact. All structure figures were made in PyMOL (http://www.pymol.org/). Electrostatic surface potential was calculated using the PyMOL ABPS Tools v. 2.1 plugin (pH 7, monovalent ion concentration 0.15 M, and dielectric constant of 2.0 for protein and 78.0 for solvent). Helix packing was analyzed using Socket (http://coiledcoils.chm.bris.ac.uk/socket/server.html).

CTD expression and purification. Soluble CTD proteins were expressed in *E. coli* Rosetta pLysS cells (Novagen). Freshly transformed Rosetta cells were grown at 37 °C in LB medium supplemented with 50 μg/ml kanamycin

NATURE STRUCTURAL & MOLECULAR BIOLOGY | www.nature.com/nsmb

© 2018 Nature America Inc., part of Springer Nature. All rights reserved.

and 34 μg/ml chloramphenicol to an optical density at 600 nm (OD<sub>600</sub>) of approximately 0.6. Protein production was induced with 1 mM isopropyl-β-D-thiogalactopyranoside (IPTG). After 3 h, cells were harvested and lysed in 50 mM Tris-HCl pH 8.0, 250 mM NaCl, 3 mM β-mercaptoethanol (BME), and 0.2 mM PMSF (lysis buffer) using a fluidizer. The cell lysate was cleared by centrifugation at 13,000 g and nutated with 2 ml of Ni-Sepharose 6B fast-flow resin (GE Healthcare) for 2 h. The resin was washed twice with 7.5 CV each of 50 mM Tris-HCl pH 8.0, 250 mM NaCl, 3 mM BME, and 0.1 mM PMSF (rinse buffer) and rinse buffer supplemented with 6 mM imidazole (WB1). It was then loaded into a gravity column and washed with 17.5 CV of rinse buffer containing 20 mM imidazole (WB2). Protein was eluted with rinse buffer containing 300 mM imidazole (EB). The eluate was concentrated in an Ultra-15 concentrator with a 30-kDa molecular weight cutoff (Millipore). Imidazole removal and further purification of the eluate were done by size-exclusion chromatography using a Superdex 200 column (GE Healthcare) equilibrated with 20 mM Tris-HCl pH 8.0, 150 mM NaCl, 1 mM EDTA, and 200 mM Tris(2-carboxyethyl)phosphine (TCEP) (TNET). Peak fractions were combined, supplemented with 0.1 mM PMSF, and flash frozen in liquid nitrogen for storage at –80 °C. The D878C mutant precipitated and was excluded from subsequent analysis.

**Liposome preparation.** Single unilamellar vesicles (SUVs) were prepared by combining POPC and POPA lipids dissolved in chloroform at 1:1 molar ratio and drying the lipid mixture first under nitrogen gas stream and then in a vacuum chamber overnight. The lipid mixture was then dissolved in 25 mM Tris-HCl pH 7.2, 150 mM NaCl, and 1 mM EDTA (EDTA was absent from samples with NiEDDA) and allowed to fully hydrate for 1 h at 4 °C. SUVs were formed by sonication and used the same day.

**ESR sample preparation and spin labeling.** To reduce disulfide bonds that had formed during storage, the CTD cysteine mutants were incubated with 600 μM free TCEP for 2–3 h. Thereafter, the buffer with a high TCEP concentration was exchanged by passing the protein solution through a NAP-5 size-exclusion column (GE Healthcare) equilibrated with 20 mM Tris-HCl pH 8.0, 150 mM NaCl, 50 μM TCEP, and 0.5–1 mM EDTA. Reduced protein samples (100 μM) were spin-labeled with 3-(2-iodoacetamido)-PROXYL spin label (ISL) at a 1:10 protein-to-ISL molar ratio. Spin labeling proceeded for 7–8 h at 4 °C. Thereafter, the unreacted spin label was removed by a combination of filtration on a NAP-10 size-exclusion column (GE Healthcare) and extensive washing with buffer consisting of 25 mM Tris-HCl pH 7.2 (at room temperature), 150 mM NaCl, and 40 μM TCEP in microconcentrators at 4 °C and 4000 g. Whenever the ESR spectra indicated trace amounts of free spin label, samples were further purified by passing through Zeba spin desalting columns (Thermo Fisher). Finally, 200 μM spin-labeled protein stock solutions were prepared and used for final dilutions to measure ESR spectra.

NiEDDA was used throughout the accessibility-scanning experiments. Sample deoxygenation was performed on a vacuum line by repeatedly evacuating the capillary tube with the sample to soft vacuum and filling it with argon gas. Finally, the capillary tube, filled with argon to ~0.9 bars, was flame sealed.

To obtain the half-saturation parameter, $P_{1/2}$, the central line of the nitroxide CW-ESR spectrum (width of ~30 G for lipid samples and ~15 G for protein without lipid) was recorded as a function of the MW power (varied from 0.5 mW to 200 mW in 20 steps). The measured intensity $A$ of the central line was plotted as a function of the square root of the applied MW power; the data were fitted to the equation

\[
A = I \sqrt{P[1 + (2^{1/\varepsilon} - 1)P/P_{1/2}]^{-\varepsilon}}
\]

where $P$ is the applied MW power, $\varepsilon$ is a line homogeneity parameter, and $I$ is a constant. We tested $\varepsilon$ values between 1 and 1.5, and in general, for all samples, good fits were obtained using $\varepsilon = 1.5$ (which is typical of a highly homogeneous spectrum), resulting in a goodness of fit parameter $R^2 > 0.98$. Accessibilities to O<sub>2</sub> and NiEDDA were calculated using the expression

\[
\Pi = (\Delta P_{1/2}/\Delta H)/({P_{1/2}}^{\text{ref}}/{\Delta H}^{\text{ref}})
\]

where $\Delta P_{1/2}$ is the offset of $P_{1/2}$ in samples with O<sub>2</sub> or NiEDDA as compared to sample in argon for the same cysteine mutant; $\Delta H$ (or $\Delta H^{\text{ref}}$) is the line width and the superscript “ref” indicates the reference. We used the data for spin-labeled R884C mutant in deoxygenated solution as a reference, similarly to previous studies [67,68].

The depth parameter $\Phi$, which visualizes better the difference in $\Pi(\text{O}_2)$ and $\Pi(\text{NiEDDA})$, was calculated as follows.

\[
\Phi = \ln[\Pi(\text{O}_2)/\Pi(\text{NiEDDA})]
\]

Spin label mobility was estimated using the inverse width ($1/\Delta H$) of the central line of the nitroxide CW-ESR spectrum [27]. The $\Phi$ and $1/\Delta H$ data for residues in h3 were fitted to the periodic function

\[
y(n) = a + A \cos(2\pi n/N + b)
\]

where $y$ is either $\Phi$ or $1/\Delta H$, $n$ is residue number, $N$ is the periodicity, $b$ is offset, and $a$ is a constant. The obtained $N$ values for $\Phi$ and $1/\Delta H$ were 3.65 and 3.75, respectively.

All fittings of CW-ESR power saturation data and spin-label mobility were performed in OriginLab software (OriginLab Corporation). The measurements were performed twice on most of the singly labeled residues in h3 and the 5PC-labeled sample.

**CW-ESR measurements.** CW-ESR spectra were recorded on multiple spin-labeled protein variants in helix h2 and helix h3 in buffer solution. CW-ESR spectra were recorded for all spin-labeled cysteine mutations in the gB CTD at a protein monomer-to-lipid molar ratio (P/L) of 1:660. The final protein concentration for the samples in buffer solution and in liposomes was 53 μM. Control samples were prepared by adding 5PC or TEMPO-PC in 1:500 molar ratio to an equimolar mixture of POPC and POPA and measured.

In the case of lipid-bound protein samples, complete binding of the protein to liposomes was verified by recording additional CW-ESR spectra at P/L values of 1:730 (58 μM protein) and 1:980 (43 μM protein) for M879C mutant. No difference in the spectral lineshape was observed. Therefore, the highest protein concentration was selected for the experiments. All samples for CW-ESR measurements were placed into 50-μl precision microcapillary glass pipettes (Kimble Glass) after closing the bottom of the pipettes; sample lengths were 10 mm (i.e., a volume of 6.7 μl). All ESR measurements were performed at 25 °C with an ELEXIS E500 ESR spectrometer (Bruker) using an ER 4122-SHQE super-high-Q resonator and a VT-31 variable-temperature accessory. Full-width nitroxide ESR spectra were recorded under the following conditions: incident microwave (MW) power of 1.26 mW; field modulation amplitude of 1.1 G for samples in buffer and 1.6–2.1 G for liposome samples depending on the spectral line width for the different mutants. All spectra were normalized to the same number of spins.

MW power saturation experiments were performed on the central line of the nitroxide CW-ESR spectrum [67], to measure the accessibility of the spin labels to the commonly used fast-relaxing agents, viz. oxygen (O<sub>2</sub>) and NiEDDA. O<sub>2</sub> has high solubility in the hydrophobic lipid membrane core, whereas NiEDDA is water soluble. Measurements were performed on gB CTD, 5PC, or TEMPO-PC samples that were either equilibrated in the presence of O<sub>2</sub>, or deoxygenated and equilibrated in the presence of argon, or deoxygenated and equilibrated in the presence of argon and NiEDDA. NiEDDA was added to the POPC:POPA lipid suspension in buffer, and liposomes with a uniform distribution of this fast-relaxing agent were then formed by sonication. Two final concentrations of NiEDDA were tested, 4.5 mM and 7 mM. However, some CW-ESR spectral broadening was observed with the 7 mM concentration, particularly in the case of spin label attached to highly solvent-exposed residue positions. Therefore, 4.5 μM

**Interspin distance measurements by DEER spectroscopy.** Four-pulse DEER [69] measurements at 17.3 GHz and 60 K were performed using a home-built Ku-band pulse ESR spectrometer [70] under standard experimental conditions. The π/2 – π – π pulse widths were 16 ns, 32 ns, and 32 ns, respectively, and the π pump pulse was 32 ns. A frequency separation of 70 MHz between detection and pump pulses was used. Detection pulses were applied at the low-field edge, and the pump pulse was positioned near the central maximum of the nitroxide spin-label spectrum. Interspin distances were reconstructed from the time-domain DEER data using L-curve Tikhonov regularization [71] and refined by the maximum entropy method [72].

**Reporting Summary.** A summary of the experimental design is enclosed in the Nature Research Reporting Summary.

**Data availability.** Atomic coordinates and structure factors for the HSV-1 gB structures have been deposited to PDB under accessions 5V2S (P321) and 6BM8 (H32). Source data for Supplementary Fig. 6 are available with the paper online. Source data for Figs. 4 and 5 and for Supplementary Fig. 7 are available upon reasonable request.

**References**

60. Charvolin, D., Picard, M., Huang, L. S., Berry, E. A. & Popot, J. L. Solution behavior and crystallization of cytochrome *bc*<sub>1</sub> in the presence of amphipols. *J. Membr. Biol.* **247**, 981–996 (2014).

61. Kabsch, W. Xds. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 125–132 (2010).

62. McCoy, A. J. et al. Phaser crystallographic software. *J. Appl. Crystallogr.* **40**, 658–674 (2007).

63. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 486–501 (2010).

64. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 213–221 (2010).

NATURE STRUCTURAL & MOLECULAR BIOLOGY | ARTICLES

65. Gouet, P., Courcelle, E., Stuart, D. I. & Métoz, F. ESPrift: analysis of multiple sequence alignments in PostScript. *Bioinformatics* **15**, 305–308 (1999).

66. Collaborative Computational Project, Number 4. The CCP4 suite: programs for protein crystallography. *Acta Crystallogr. D Biol. Crystallogr.* **50**, 760–763 (1994).

67. Georgieva, E. R., Xiao, S., Borbat, P. P., Freed, J. H. & Eliezer, D. Tau binds to lipid membrane surfaces via short amphipathic helices located in its microtubule-binding repeats. *Biophys. J.* **107**, 1441–1452 (2014).

68. Zou, P. & McHaourab, H. S. Alternating access of the putative substrate-binding chamber in the ABC transporter MsbA. *J. Mol. Biol.* **393**, 574–585 (2009).

69. Pannier, M., Veit, S., Godt, A., Jeschke, G. & Spiess, H. W. Dead-time free measurement of dipole–dipole interactions between electron spins. *J. Magn. Reson.* **142**, 331–340 (2000).

70. Borbat, P. P., Crepeau, R. H. & Freed, J. H. Multifrequency two-dimensional Fourier transform ESR: an X/Ku-band spectrometer. *J. Magn. Reson.* **127**, 155–167 (1997).

71. Chiang, Y. W., Borbat, P. P. & Freed, J. H. The determination of pair distance distributions by pulsed ESR using Tikhonov regularization. *J. Magn. Reson.* **172**, 279–295 (2005).

72. Chiang, Y. W., Borbat, P. P. & Freed, J. H. Maximum entropy: a complement to Tikhonov regularization for determination of pair distance distributions by pulsed ESR. *J. Magn. Reson.* **177**, 184–196 (2005).

nature research

Life Sciences Reporting Summary

Nature Research wishes to improve the reproducibility of the work that we publish. This form is intended for publication with all accepted life science papers and provides structure for consistency and transparency in reporting. Every life science submission will use this form; some list items might not apply to an individual manuscript, but all fields must be completed for clarity.

For further information on the points included in this form, see [Reporting Life Sciences Research](https://www.nature.com/nature-research/editorial-policies/reporting-summary). For further information on Nature Research policies, including our [data availability policy](https://www.nature.com/nature-research/editorial-policies/data-availability), see [Authors & Referees](https://www.nature.com/nature-research/editorial-policies/authors-referees) and the [Editorial Policy Checklist](https://www.nature.com/nature-research/editorial-policies/editorial-policy-checklist).

### Experimental design

1. **Sample size**
   - Describe how sample size was determined.
     - n/a

2. **Data exclusions**
   - Describe any data exclusions.
     - No data were excluded.

3. **Replication**
   - Describe whether the experimental findings were reliably reproduced.
     - All experimental findings were reproducible.

4. **Randomization**
   - Describe how samples/organisms/participants were allocated into experimental groups.
     - n/a

5. **Blinding**
   - Describe whether the investigators were blinded to group allocation during data collection and/or analysis.
     - n/a

Note: all studies involving animals and/or human research participants must disclose whether blinding and randomization were used.

6. **Statistical parameters**

For all figures and tables that use statistical methods, confirm that the following items are present in relevant figure legends (or in the Methods section if additional space is needed).

- ☐ The exact sample size ($n$) for each experimental group/condition, given as a discrete number and unit of measurement (animals, litters, cultures, etc.)
- ☐ A description of how samples were collected, noting whether measurements were taken from distinct samples or whether the same sample was measured repeatedly
- ☐ A statement indicating how many times each experiment was replicated
- ☐ The statistical test(s) used and whether they are one- or two-sided (note: only common tests should be described solely by name; more complex techniques should be described in the Methods section)
- ☐ A description of any assumptions or corrections, such as an adjustment for multiple comparisons
- ☐ The test results (e.g., $P$ values) given as exact values whenever possible and with confidence intervals noted
- ☐ A clear description of statistics including central tendency (e.g., median, mean) and variation (e.g., standard deviation, interquartile range)
- ☐ Clearly defined error bars

See the web collection on [statistics for biologists](https://www.nature.com/nature-research/editorial-policies/statistics-for-biologists) for further resources and guidance.

### Software

Policy information about [availability of computer code](https://www.nature.com/nature-research/editorial-policies/software-availability)

7. **Software**
   - Describe the software used to analyze the data in this
     - OriginLab

study.

XDS  
Coot v 0.8.8  
Phenix v 1.11.1  
Phaser  
Clustal Omega  
ESPrpt 3.0  
PISA v 1.52  
CCP4 Contact  
PyMOL v 1.8.4.0  
PyMOL ABPS Tools v. 2.1  

For manuscripts utilizing custom algorithms or software that are central to the paper but not yet described in the published literature, software must be made available to editors and reviewers upon request. We strongly encourage code deposition in a community repository (e.g. GitHub). Nature Methods [guidance for providing algorithms and software for publication](https://www.nature.com/articles/nmeth.f.713) provides further information on this topic.

### Materials and reagents

**Policy information about availability of materials**

#### 8. Materials availability
Indicate whether there are restrictions on availability of unique materials or if these materials are only available for distribution by a for-profit company.

All unique materials used in this study are available from authors.

#### 9. Antibodies
Describe the antibodies used and how they were validated for use in the system under study (i.e. assay and species).

No antibodies were used.

#### 10. Eukaryotic cell lines
a. State the source of each eukaryotic cell line used.
Spodoptera frugiperda (Sf9) use for protein expression were purchased from ThermoFisher, and E. coli Rosetta pLysS cells were purchased from Novagen. CHO cells were a gift from J. M. Coffin (Tufts University School of Medicine).

b. Describe the method of cell line authentication used.
Cell lines were not authenticated.

c. Report whether the cell lines were tested for mycoplasma contamination.
Cell lines were not tested for mycoplasma contamination by us.

d. If any of the cell lines used are listed in the database of commonly misidentified cell lines maintained by [ICLAC](http://iclac.org/), provide a scientific rationale for their use.
No commonly misidentified cell lines were used.

### Animals and human research participants

**Policy information about studies involving animals; when reporting animal research, follow the ARRIVE guidelines**

#### 11. Description of research animals
Provide details on animals and/or animal-derived materials used in the study.

No animals were used.

**Policy information about studies involving human research participants**

#### 12. Description of human research participants
Describe the covariate-relevant population characteristics of the human research participants.

The study did not involve research participants.
