
# Autophagy and Its Interaction With Intracellular Bacterial Pathogens

**Mariana da Silva Siqueira**, **Renato de Moraes Ribeiro**, and **Leonardo H. Travassos**

Laboratory of Immunoreceptors and Signaling, Immunobiology Program, Institute of Biophysics Carlos Chagas Filho, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil

Cellular responses to stress can be defined by the overwhelming number of changes that cells go through upon contact with and stressful conditions such as infection and modifications in nutritional status. One of the main cellular responses to stress is autophagy. Much progress has been made in the understanding of the mechanisms involved in the induction of autophagy during infection by intracellular bacteria. This review aims to discuss recent findings on the role of autophagy as a cellular response to intracellular bacterial pathogens such as, *Streptococcus pyogenes*, *Mycobacterium tuberculosis*, *Shigella flexneri*, *Salmonella typhimurium*, *Listeria monocytogenes*, and *Legionella pneumophila*, how the autophagic machinery senses these bacteria directly or indirectly (through the detection of bacteria-induced nutritional stress), and how some of these bacterial pathogens manage to escape from autophagy.

**Keywords:** autophagy, infection, *Mycobacterium tuberculosis*, *Shigella flexneri*, *Listeria monocytogenes*, *Salmonella typhimurium*, *Streptococcus pyogenes*, *Legionella pneumophila*

## INTRODUCTION

Autophagy is a homeostatic and highly conserved survival mechanism in which portions of the cytoplasm such as long-lived proteins and damaged organelles are sequestered in double-membrane vesicles (called autophagosomes). Then, autophagosomes fuse with lysosomes, leading to the degradation of the sequestered content and recycling of functional blocks for anabolic processes, especially during nutrient shortages (1). Indeed, for many years, autophagy was mainly considered as a breakdown process to degrade macromolecules to generate energy during nutrient deprivation. To date, three types of autophagy have been described, chaperone-mediated autophagy, microautophagy, and macroautophagy (1). Here, we discuss the interaction of the best-characterized type of autophagy (macroautophagy, hereafter autophagy), with intracellular bacterial pathogens, a process designated xenophagy.

The first report demonstrating induction of autophagy by bacteria was published in 1984. In this study, Rikihisa described the presence of vesicles containing glycogen granules and rickettsiae in Guinea pig polymorphonuclear (PMNs) cells infected with the bacteria (2). Despite this initial study, it was only after the studies from Nakagawa et al. and Gutierrez et al. that autophagy was regarded as an important cell autonomous arm of the innate immune system against intracellular bacteria. In their seminal and independent studies, Nakagawa et al. and Gutierrez et al. demonstrated a crucial role for autophagy in the sequestration and degradation of group A Streptococcus (GAS) and *Mycobacterium bovis* BCG, respectively (3, 4). Since then, an amazing number of elegant studies have demonstrated a key role of autophagy in the control of infection by different bacterial pathogens and also how some of these most well-succeeded pathogens circumvent or even use autophagy to establish replicative niches inside different cell types (5–7).

Siqueira et al.

---

### The Autophagosome Formation Core Machinery

Possibly one of the most exciting areas in the field of autophagy, the mechanisms involved in the formation of autophagosomes, the hallmark of this process, have been the focus of many research groups. Morphologically, autophagy begins with the formation of a cup-shaped double-membrane structure that surrounds the cargo. Upon its complete closure, the phagophore is now called an autophagosome, a transient organelle that delivers its content for degradation in lysosomes (8). After extensive work from several groups, the proteins that participate in autophagosome biogenesis can be categorized into complexes that take place in different steps of the autophagosome formation (1). Below, we will summarize the different steps of the autophagic process and the major protein groups that take part in each step of the whole process and discuss critical findings linking these proteins with bacterial-induced autophagy. For extensive literature on autophagosome formation machinery, please refer to Suzuki et al. (9) and Yin et al. (10).

#### Signal Induction

##### The ULK Complex and Autophagy Induction

The uncoordinated-51-like kinase (ULK1) complex comprising ULK1, ATG13, FAK family kinase-interacting protein of 200 kDa (FIP200), and ATG101 is responsible for sensing changes in nutrient status within the cell. Its activation is instrumental in the initiation of autophagy. This complex works downstream mammalian target of rapamycin complex 1 (mTORC1) and under nutrient-rich conditions is phosphorylated by mTOR, which inhibits ULK1 recruitment to the phagophore assembly site (PAS).

Under nutrient starvation, however, mTORC1 is inactivated, and ULK1 is released, allowing FIP200 phosphorylation and translocation of the complex to PAS for the recruitment of ATG proteins, required for autophagosome formation (11). Interestingly, components of the ULK complex have also been shown to target bacterial vacuoles during infection with intracellular bacteria (12). This is the case of FIP200 during infection with *Salmonella typhimurium*. Experiments performed by Kageyama et al. suggest that this protein is recruited to the vicinity of vacuoles containing *S. typhimurium*. See below for more detailed information regarding autophagy induced by this pathogen.

#### Nucleation

##### Class III Phosphatidylinositol 3-Kinase (PtdIns3K) Complex and Trafficking of Atg9 for Autophagosome Nucleation

The class III PtdIns3K complex consisting of Beclin 1, ATG14L, phosphoinositide 3-kinase regulatory subunit 4 (PIK3R4) are recruited to PAS to initiate phagophore membrane nucleation through the activation of PtdIns3-kinase class III (PtdIns3KC3). As a result, PtdIns3P is generated at this site, and the PtdIns3P-binding protein WD-repeat domain phosphoinositide-interacting 1 (WIPI1) and 2 (WIPI2) are recruited to the PAS, allowing ATG proteins to be recruited later on (13). Mammalian Atg9 (mAtg9) is another protein required for the assembly of phagophore, although its role is still not completely understood. It has been demonstrated that mAtg9 is not necessary for LC3 recruitment to phagophore, but essential for its generation following infection with *Salmonella typhimurium* (12).

#### Expansion

##### Ubiquitin-Like Conjugation Systems and Autophagosome Expansion

Pivotal for the formation of autophagosomes are two ubiquitin-like conjugation systems: Atg8/LC3 and Atg12. The Atg8/LC3 system modifies the core autophagy protein microtubule-associated 1 light chain 3 (LC3). LC3 has a diffuse cytosolic distribution pattern and is cleaved at its C-terminus by the cysteine protease Atg4 to form LC3-I, which has a C-terminal glycine residue. Upon autophagy induction, LC3-I is sequentially modified by the E1-like enzyme Atg7 and the E2-like enzyme Atg3 to form LC3-II after the conjugation of LC3-I to phosphatidylethanolamine (PE). This lipidated form of LC3 is attached to both outer and inner phagophore membrane being eventually removed from the autophagosomal membrane by Atg4 before the fusion with late endosomes/lysosomes (1, 14). In the Atg12 conjugation system, Atg5 and Atg12 proteins form a complex through the covalent binding of Atg12 to the C-terminus of Atg5 in a reaction involving Atg7 and Atg10. Then, the scaffold protein Atg16L1 is conjugated to Atg5 via its N-terminus, forming the 800 kDa Atg12–Atg5–Atg16L1 complex. It has been proposed that the Atg16L1 complex works as an E3-like enzyme to target LC3-I to its membrane site of lipid conjugation (15). Data from the literature suggest that these two systems work coordinately as in Atg3-deficient cells, where no LC3-II is found, Atg12–Atg5 conjugation is dramatically reduced (16).

Alternative (non-canonical) forms of autophagy have been identified and reported to target invading bacteria (17–19). In this review, however, we will focus on xenophagy and its implication in intracellular bacterial infections.

#### Cargo Selection During Infection With Bacterial Pathogens

Invasion of host cytosol by bacteria imposes a significant challenge to homeostasis and triggers several cellular and immune responses such as proinflammatory cascades and cell-autonomous in an attempt to control of bacterial replication, such as xenophagy.

In addition to the steps discussed above, autophagy has an additional and essential step that is cargo selection. One of the central questions regarding xenophagy relates to its specificity and how autophagy machinery specifically recognizes bacteria. This is of major importance as xenophagy, which eventually aims to reduce not only bacterial load but also prevent cellular stress resulting, for instance, from bacteria-induced amino acid starvation (see later in this review). To explain the central mechanisms involved in the selection of intracellular bacteria by the autophagy machinery, we will focus on bacterial models that helped us shape the field.

#### Mycobacterium tuberculosis

*Mycobacterium tuberculosis* is the causative agent of tuberculosis (TB), possibly one of the oldest human pathogens and still

Frontiers in Immunology | www.frontiersin.org
May 2018 | Volume 9 | Article 935

among the top 10 causes of death worldwide (20). *M. tuberculosis* is a non-motile and facultative intracellular pathogen of macrophages. In this regard, the infection of alveolar macrophages is a crucial requisite toward the establishment of a successful replicative niche. Experiments using mice depleted for resident alveolar macrophages have shown that these animals become protected from *M. tuberculosis* (21). One of the main features of TB pathogenesis is the ability of *M. tuberculosis* to survive within alveolar macrophages through the interference with phagolysosome biogenesis (3, 22).

In the last decade, autophagy emerged as an essential protective strategy employed by the host to restrict the spread of *M. tuberculosis*. The first piece of evidence on the role of autophagy in the control of Mycobacterium was provided by the cornerstone study of Gutierrez et al. (3). The authors demonstrated that

upon the induction of autophagy by starvation or rapamycin *M. tuberculosis* variant bovis BCG colocalized to LC3+ compartments in RAW 264.7 macrophages. Moreover, BCG phagosomes were shown to be positive for markers of acidification such as cathepsin D and Lamp-1, suggesting that xenophagy induction was able to override the blockade in phagosome maturation by BCG, with a clear impact on bacterial killing (Figure 1). Interferon-γ (IFN-γ) is essential for resistance to infection, by interfering with the transcription of more than 2,000 genes (23). In a more physiological context, Gutierrez et al. demonstrated that IFN-γ, a potent activator of macrophages, was able to mimic the effects of rapamycin or starvation on the induction of autophagy, through the immunity-related p47 guanosine triphosphatases (IRG) Irgm1 (LRG-47) (3) (Figure 1). These results put autophagy on the center stage of the immune mechanisms involved in the

---

**FIGURE 1 |** Autophagy targets *Mycobacterium tuberculosis* (Mtb) by different mechanisms. Stimulation with IFN-γ or Toll-like receptors (TLRs) ligands leads to an increase in the localization of (Mtb) into autophagosomes (left). 6-kDa early secretory antigenic target (ESAT-6) secretion system 1 (ESX-1)-induced phagosomal damage induces the exposure of Mtb to cytosolic autophagy adaptors such as Optneurin, p62, NDP52 and NBR1 which bind to ubiquitin associated with Mtb as a consequence of the E3-ligases SMURF-1 and Parkin, culminating with targeting of the bacteria for autophagic degradation (center). Extracellular bacterial DNA from Mtb is detected by STING to activate TBK1 and lead to Mtb ubiquitination and recruitment of p62 and NDP52 (middle-left). IRGM-induced increase in ROS provokes autophagic targeting of Mtb (right).

protection against *M. tuberculosis* infection. After their initial discoveries, in a subsequent study, the same group demonstrated that both Irgm1 and its human ortholog IRGM are necessary for the induction of autophagy, generating large autolysosomes that contributed with *M. tuberculosis* intracellular growth restriction upon macrophage activation by IFN-γ (24). The mechanism behind IRGM restriction of *M. tuberculosis* seems to rely on its interaction with cardiolipin in mitochondria to generate ROS and mitochondrial fission, both necessary for *M. tuberculosis* killing (25). The role of IFN-γ in autophagy also involves the participation of interferon-induced guanylate-binding (GBP), which are also upregulated in the presence of the cytokine. It has been demonstrated that GBPs promote oxidative killing and the delivery of antimicrobial peptides to autophagolysosomes, contributing to *M. tuberculosis* intracellular replication control (26) (Figure 1). Altogether, these studies demonstrated an essential *in vitro* role for xenophagy and its induction by IFN-γ in the control of *M. tuberculosis* intracellular replication.

Although the link between deficiencies in ATG proteins and defective xenophagy has been widely reported upon infection with *M. tuberculosis*, ATG5 have also been described to play a critical autophagy-independent role in an *in vivo* TB mouse model. Kimmey et al. have demonstrated that the deletion of Atg14L, Atg12, Atg16L1, Atg7, and Atg3 in the myeloid compartment did not affect the outcome of *M. tuberculosis* infection, suggesting that the loss of autophagy is not implicated with the progression of the disease. In sharp contrast, the authors reported that the loss of ATG5 in PMN but not in alveolar macrophages led to exacerbated immunopathology, sensitizing mice to *M. tuberculosis*. Together, these findings suggest that ATG5 has unique autophagy-independent features that are not shared with other ATG proteins, pointing for a reinterpretation of the role of ATG5 in the control of *M. tuberculosis* infection *in vivo* (27).

One primary open question that remained to be answered was how eukaryotic cells sense *M. tuberculosis* infection to induce autophagy. Toll-like receptors (TLRs) detect a myriad of extracellular and endolysosome located microbial products. It has been reported that Poly (I:C), LPS, and ssRNA, ligands for TLR3, TLR4, and TLR7, respectively, induce autophagosome formation through MyD88-dependent pathways. Interestingly, activation of TLR7 by its ligand increased the ability of macrophages to kill BCG (28) (Figure 1). However, it was not clear how TLRs would be able to detect BCG to induce autophagy in the absence of exogenous stimulation with their cognate ligands. The first molecular evidence of the detection of *M. tuberculosis*-derived microbial-associated molecular pattern (MAMP) triggering autophagy demonstrated that stimulation of interferon genes (STING), an important adaptor of TANK-binding kinase (TBK1) in the interferon stimulatory DNA pathway, senses the presence of cytosolic DNA to trigger the ubiquitination of *M. tuberculosis* after phagosome damage. Upon sensing of extracellular DNA from *M. tuberculosis* by STING, *M. tuberculosis* is ubiquitinated, leading to the recruitment by the autophagic adaptors p62/SQSTM1 (hereafter p62), a multi-domain protein that functions as an autophagic adaptor. p62 possesses an LC3-interacting protein region (LIR) and a C-terminal ubiquitin-associated (UBA) domain that binds ubiquitinated substrates and an LIR.

Together with p62 and nuclear dot protein 52 kDa (NDP52) work to link ubiquitinated substrates to LC3 recruitment, ensuring the efficient delivery of *M. tuberculosis* to autophagosomes (29) (Figure 1). Although the sequestration of *Mycobacteria* by xenophagy has been demonstrated to be mostly dependent on ATG proteins, one report has been shown that sequestration of ubiquitinated mycobacteria can occur in ATG5-independent manner. The authors found that following 6-kDa early secretory antigenic target (ESAT-6) secretion system 1 (ESX-1)-mediated phagosome escape, ubiquitinated bacteria were resquestered by structures that resembled autophagosomes and localized to Lamp-1<sup>+</sup> compartments. Notably, ubiquitinated *M. marinum* were never decorated with LC3 and ATG5 deficiency and did not affect bacterial counts. It remains to be elucidated if the finding that *M. marinum* did not localize to LC3<sup>+</sup> compartments represents a potential specific mechanism of escape from autophagy (30).

Upon phagosome damage mediated by ESX-1, *M. tuberculosis* is ubiquitinated, in an essential step required for the recruitment of the autophagic adaptors p62 and NDP52 and LC3. Although it has not been determined, which bacterial or host proteins (or both) are ubiquitinated during xenophagy, much progress has been made in the identification of host proteins that mediate ubiquitination involved in xenophagy. Several ubiquitin-ligases have been described as participants of bacterial ubiquitination. Parkin has a well-established role in mitophagy where it promotes the ubiquitination of mitochondrial surface proteins prior to the recruitment of p62 in order to direct malfunctioning mitochondria for autophagic degradation. In 2013, Parkin was also reported to be crucial in the conjugation of K63-ubiquitin chains to *M. tuberculosis* inside macrophages. In line with this finding, *Park2<sup>-/-</sup>* displayed increased *M. tuberculosis* replication in an *in vivo* TB model (31) (Figure 1). Of note, Parkin has also been demonstrated to participate in ubiquitination of other mycobacterial species such as *M. leprae* (32). Similarly, SMAD-specific E3 ubiquitin-ligase protein 1 (Smurf1) has been demonstrated to mediate K48- but not K63-ubiquitination and the recruitment of the autophagy adaptor NBR1 during *M. tuberculosis* infection to control its replication in human macrophages and to associate with bacteria present in the lung of patients with pulmonary TB (33). In their study, Franco et al. reported that Smurf1 - but not Parkin-dependent ubiquitination is necessary for the recruitment of proteasome and NBR1 for the vicinity of *M. tuberculosis*. In contrast, K63 ubiquitination by Parkin but not Smurf1 is required for the recruitment of p62 to the bacterial surface (Figure 1). It remains to be elucidated why host cells employ different ubiquitin-ligases with apparent redundant roles for targeting *M. tuberculosis* for xenophagy. One possibility is that the apparent redundancy of Smurf-1 and Parkin is a countermeasure resulting from the ability of *M. tuberculosis* to escape from autophagy. Also, the different ubiquitin moieties added to *M. tuberculosis* surface could help in the recruitment of various adaptors. Indeed, Smurf1-mediated ubiquitination recruits the adaptor NBR1, which is not recruited by Parkin-mediated activity.

Several recent studies have reported that *M. tuberculosis* uses sophisticated mechanisms to escape xenophagy and replicate inside host cells. In addition to the induction of miR33 and

Siqueira et al.

miR33* expression to manipulate cellular metabolism and energy levels (34) and miRNA125a to inhibit UVRAG expression (35) (discussed later in this review), *M. tuberculosis* also induces the expression of other microRNAs (miRNAs) to circumvent xenophagy by interfering with different aspects of cellular physiology. This is the case of miR30A that has its expression increased during infection with *M. tuberculosis* to decrease Beclin 1 expression levels, leading to inhibition autophagosome elongation to promote intracellular survival of *M. tuberculosis* (36). Similarly, miR144* inhibits antimicrobial responses against *M. tuberculosis* in monocytes by targeting the expression of DNA damage-regulated autophagy modulator 2, allowing *M. tuberculosis* replication (37). In contrast, miR155 has been demonstrated to play a pro-autophagic role during *M. tuberculosis* infection. Wang et al. reported that miR155 targets Ras homolog enriched in brain (Rheb), a negative regulator of autophagy to accelerate the process of xenophagy. Inhibition of autophagy by *M. tuberculosis* seems to aim not only xenophagy but other essential steps of the immune response as well. It has been recently demonstrated that the bacterial factor PE_PGRS47 inhibits autophagy through an unknown mechanism to block MHC II antigen presentation and dampen adaptive immune responses against *M. tuberculosis* (38). Altogether, these studies provide compelling evidence that despite the crucial role of xenophagy as an antimycobacterial mechanism, *M. tuberculosis* has developed means to escape autophagy and replicate within macrophages.

## Streptococcus pyogenes

*Streptococcus pyogenes* is the causative agent of a variety of infections, ranging from such as pharyngitis and skin infections to life-threatening necrotizin fasciitis and bacteremia (39). In 2004, Nakagawa et al. provide one of the first definitive evidence of the role of autophagy as a cell-autonomous antimicrobial mechanism. In this study, HeLa cells were shown to specifically target cytosolic GAS. This process was dependent on the toxin streptolysin O (SLO), a cholesterol-dependent pore-forming cytotoxin (40). Nakagawa et al. demonstrated that the majority of the cytosolic population of GAS colocalized to LC3+ compartments, in contrast to SLO-deficient mutants in which no colocalization with LC3 was found (4) (Figure 2). As demonstrated for other intracellular bacteria, the adaptors p62, NDP52, and NBR1 are essential for recognition of ubiquitin decorated GAS and recruitment of LC3 before autophagic degradation (41, 42). Evasion of xenophagy by GAS has been reported, and GAS has been shown to evade ubiquitin recognition by the abovementioned autophagic adaptors. Barnett et al. have found that the globally disseminated serotype M1T1 (strain 5448) clone of GAS can avoid xenophagy to replicate in the cytosol. This is achieved by the expression of SpeB, a cysteine protease that degrades p62, NDP52, and NBR1. M1T1 ΔspeB mutants fail to evade recognition by these proteins and are efficiently degraded through xenophagy (41) (Figure 2). These findings reveal a new mechanism by which GAS evades elimination by xenophagy. Notably, data from the literature demonstrate that xenophagy efficiently eliminates other GAS serotypes such as M6, M49, and M89. GAS is a successful human bacterial pathogen that causes a vast array of diseases and the work of Barnett et al. uncovers autophagy evasion as a determinant feature for the dissemination of GAS. The mechanisms employed by autophagy to target intracellular GAS also include the engagement of the CD46 pathogen receptor (43). CD46 is a glycoprotein expressed by all nucleated human cells that physically binds several pathogens such as adenoviruses B and D, human herpesvirus 6, *Neisseria*, and GAS (44) (Figure 2). Although several innate immune receptors such as TLRs have been described to trigger xenophagy upon infection or ligand stimulation, how these receptors are connected to the selective targeting of intracellular bacteria to lysosomes is still unclear. The findings from Joubert et al. provide an important piece of data to this open question. One possibility that needs to be experimentally tested is that CD46 might be concomitantly activated together with TLRs to promote xenophagy. Another known host factor that has been reported to participate in GAS targeting for xenophagy is 8-nitroguanosine 3′,5′-cyclic monophosphate (8-nitro-cGMP), a downstream mediator of nitric oxide that has been shown to promote protein S-guanylation on bacterial surface, which are then K63 ubiquitinated prior to the recruitment of LC3 (45). Although these findings shed light into a new xenophagy targeting mechanism during infection with GAS, some open questions remain, such as (i) is this mechanism specific for GAS? and (ii) which autophagy adaptors and ubiquitin-ligases

---

Frontiers in Immunology | www.frontiersin.org
May 2018 | Volume 9 | Article 935

participate in this process. One interesting question regarding the induction of xenophagy by GAS is the role of endothelial cells in this process. Despite different reports showing that xenophagy plays an important role in the clearance of intracellular GAS, in endothelial cells, the results are contrasting. While, Cutting et al. demonstrated the ability of endothelial cells to upregulate xenophagy in order to control GAS infection, a recent study from Lu et al. reports that endothelial cells fail to target GAS for degradation due to an intrinsic defect in the ubiquitination of intracellular bacteria (46, 47). Even though much progress has been done in the understanding of the mechanisms of GAS-induced autophagy, further studies are required in order to clarify whether endothelial cells are in fact defective in xenophagy, if this defect is specific for infection with GAS or if GAS can halt xenophagy in these cells and not in epithelial cells.

**Shigella flexneri**

Shigella spp. are Gram-negative and highly invasive enteropathogens and a significant cause of disease, especially in children under the age of 5 years, causing approximately one million deaths worldwide (48). A few minutes after its invasion of epithelial cells and macrophages, *S. flexneri* is able to lyse the phagocytic vacuole and access the cytosolic compartment where it replicates (49). As countermeasures, host cells trigger autophagy to restrict *S. flexneri* intracellular growth and cell-to-cell spreading. The first evidence of an interaction between Shigella and autophagy was provided by a study dating from 2005. In this study, it was demonstrated that wild-type *S. flexneri* can escape from autophagic targeting by employing IcsB, one of the effectors of its type 3 secretion system (T3SS). Ogawa et al. observed that deletion mutants for

IcsB, which is secreted by cytosolic bacteria and localizes to the bacterial surface were more efficiently targeted by autophagosomes. These results suggest that *S. flexneri* is able to escape from xenophagy. According to this study, the escape mechanism employed by *S. flexneri* involves IcsA/VirG, a 52 kDa protein that requires the bacterial chaperone IpgA for its stability, activates the complex-related proteins (Arp) 2/3 complex through the recruitment and activation of N-WASP, to induce actin polymerization and bacterial motility within the cell (50–54). Mechanistically, the study of Ogawa et al. demonstrated that, in ΔicsB mutants, IcsA/VirG triggers autophagy by binding to ATG5. According to the authors, IcsB inhibits IcsA/VirG affinity for ATG5. Thus, in wild-type *S. flexneri*, IcsB reduces IcsA/VirG affinity for ATG5 to initiate xenophagy (52) (Figure 3). More recently, a study added more complexity to the role of IcsB as a factor contributing to *S. flexneri* escape from autophagy. Baxt and Goldberg reported that IcsB also contributes to *S. flexneri* escape from xenophagy by recruiting transducer of CDC42-dependent actin assembly 1 (Toca-1) to prevent the recruitment of the adaptor NDP52 and LC3 (55).

The ubiquitination of *S. flexneri* has been reported to be essential for the recruitment of the adaptors p62 and NDP52 (56). However, in contrast to *S. typhimurium* and *M. tuberculosis* for which several ubiquitin-ligases that ubiquitinate bacterial surface have been described, the mechanism used to host cells to tag *S. flexneri* with ubiquitin is not clear. LUBAC is an ubiquitin-ligase that mediates the formation of M1-linked ubiquitin chains that culminate with xenophagy and bacterial degradation of *S. typhimurium* (see below). In contrast, LUBAC was reported to play no major role in the trafficking of *S. flexneri* to autophagosomes

---

**FIGURE 3** | *Shigella flexneri* employs different mechanisms to escape from autophagy. During bacterial entry into host cells, Nod1 and Nod2 recruit ATG16L1 to initiate autophagosome formation in order to restrict *S. flexneri* replication (left). Vacuole damage leads to β-glycan exposure and recognition by Galectins 3 and 8 and recruitment of NDP52, followed by bacterial ubiquitination and binding to p62 and NDP52, culminating to *S. flexneri* targeting for autophagic degradation (center). IcsB plays a central role in disguising autophagic machinery. This protein competes with IcsA/VirG for binding to ATG5, preventing p62, and NDP52 binding, septin caging and autophagosome formation (right). Septin caging and further recruitment of autophagic adaptors are blocked by IcsB expression.
(57). According to this study, *S. flexneri* escapes from LUBAC-dependent ubiquitination by secreting the effector E3-ligase H1.4 to antagonize the activity of LUBAC (57). Despite the lack of substantial data on the how *S. flexneri* is ubiquitinated, different adaptors that bind ubiquitin are independently recruited to the bacterial surface. For instance, p62 and NDP52 have been demonstrated to be recruited to *S. flexneri* surface and to regulate xenophagy mediated by each other. In agreement with the notion of its anti-autophagic role, IcsB also contributed to *S. flexneri* escape from autophagy by hiding IcsA/VirG from ubiquitin coating (56) (Figure 3). The reason why *S. flexneri* recruits different adaptors *S. flexneri* is not clear. However, it is possible that p62 and NDP52 may recognize different ubiquitin linkages as a result of the activity of different ubiquitin-ligases. Another hypothesis is that their LIR domains could be able to recruit different LC3 homologs and different adaptors that could contribute to membrane recruitment from various sources for the formation of autophagosome around bacteria. These hypotheses still lack experimental confirmation.

Shortly after *S. flexneri* entry in epithelial cells, the phagocytic vacuole is ruptured, membrane remnants expose host sugars in the cytosol, and galectin 3 promotes ubiquitination and recruitment of p62 to support xenophagy (58). In contrast to other reports from the literature, the authors did not observe increased recruitment of p62 in ΔicsB.

Interestingly, members of the NF-κB pathway such as TRAF6 and NEMO and the peptidoglycan receptor Nod1 were reported to localize to these membrane remnants (58). Similarly, NLRP3, NLRC4, ASC, and Caspase-1 were also found associated with Shigella vacuolar membrane remnants. The physiological meaning of these findings is still to be defined. One possibility is that these membrane portions might be used for the activation of inflammatory cascades and that this process is likely to be regulated by autophagy. Another possibility is that by recruiting these proteins to its vicinity, *S. flexneri* modulates NF-κB activation and inflammation to favor its replication and spread.

Septins are conserved GTP-binding proteins that play critical roles in cell division, cytoskeletal dynamics, and membrane remodeling (59). These proteins have been shown to form cages around *S. flexneri* actively polymerizing actin. Interestingly, colocalization of septins, p62, and LC3 on *S. flexneri* bacterial surface has been demonstrated and depletion of septins markedly reduced xenophagy of *S. flexneri*, suggesting an intimate relationship between these two processes (60). More recently, the precise mechanisms involved in *S. flexneri*-cage assembly were revealed. Sirianni et al. have found that mitochondrial proteins associate with *S. flexneri*-septin cages and that mitochondria promote the formation of septin cage assembly around *S. flexneri* for antibacterial xenophagy (61). *S. flexneri* has been demonstrated to induce mitochondrial damage and in the study by Siriani et al., this aspect was linked to dampening of septin cages and escape (61, 62). Of note, IcsB contributes to masking *S. flexneri* from septin caging (60). These results demonstrate that IcsB dampens xenophagy by at least three different mechanisms: competing with IcsA/VirG for binding to ATG5, by avoiding septin caging and targeting to autophagosomes, and by recruiting Toca-1 to inhibit the recruitment of NDP52 and LC3.

In addition to direct interaction of its virulence factors and autophagy proteins, pattern-recognition receptors also seem to participate in the interplay between *S. flexneri* and autophagic pathways. It has been demonstrated that the infection of macrophages by *S. flexneri* induces a robust activation of Caspase-1 that leads to inflammasome activation and cell death by pyroptosis in an NLRC4-dependent but ASC-independent mechanism (63). Interestingly, both Caspase-1 and NLRC4 were shown to negatively regulate autophagosome formation in macrophages infected with *S. flexneri* as demonstrated by studies in which bone marrow-derived macrophages (BMDMs) from knockout mice for the genes encoding these proteins were shown to induce the formation of GFP-LC3 positive membranes around bacteria in contrast to wild-type BMDMs (63). In contrast to previous studies, IcsA/VirG was not implicated in autophagy induction (52, 63), which can be explained by the different cell types used in these studies. In contrast to negative regulation of autophagy by NLRC4, NLRC1 (Nod1), and NLRC2 (Nod2), the founding members of the NLR family have been linked to autophagy induction. Nod1 and Nod2 are sensors of intracellular peptidoglycan that upon engagement lead to the activation of NF-κB activation through the recruitment of the adaptor protein RIP2 (64). Both Nod1 and Nod2 have been shown to recruit ATG16L1 at early stages of infection by *S. flexneri* to initiate autophagosome formation. As a result, Nod1- and Nod2-deficient MEFs display decreased numbers of GFP-LC3 positive bacteria, and interestingly, these findings did not rely on recruitment of RIP2 or NF-κB activation. Notably, in this study, the most common Nod2 mutation associated with Crohn disease (CD) resulted in impaired recruitment of ATG16L1 to the bacterial entry site and much less xenophagy, underscoring the notion that dysregulation of bacterial autophagy is likely to play an important role in the pathogenesis of CD (49) (Figure 3). It remains to be clarified if and in which conditions Nod1/2-dependent pro-autophagic signals would prevail over NLRC4-dependent anti-autophagy ones and vice versa.

### Salmonella typhimurium

*Salmonella typhimurium* is a pathogenic Gram-negative bacterium found in the intestinal lumen and a major cause of gastroenteritis in humans and other mammals (65). This pathogen uses two T3SS, encoded by *Salmonella* pathogenicity island 1 and 2 (SPI2) to enter non-phagocytic cells and establish a replicative niche within vacuoles termed *Salmonella*-containing vacuole (SCV). In 2006, it was first reported that a fraction of the bacterial population within the SCV previously demonstrated to form pores in this compartment was able to reach the cytosol being immediately targeted by LC3 and ATG proteins. In this study, the authors showed that Atg5-deficient MEFs infected with *S. typhimurium* had decreased fusion of LC3+ bacteria colocalized with Lamp1, suggesting diminished bacterial degradation in lysosomes. Indeed, these cells harbored increased bacterial numbers, confirming the role of autophagy in the control of *S. typhimurium* infection (66) (Figure 4). Importantly, xenophagy has been reported to be essential in the control of *S. typhimurium* in other models such as *Caenorhabditis elegans* and *Dictyostelium discoideum*, suggesting that the role of xenophagy as an
Siqueira et al.

and restriction of cytosolic ubiquitinated S. typhimurium (69). Other ubiquitin-binding proteins have also been reported to participate in Salmonella-induced autophagy. Similarly to p62, Optineurin harbors LIR and UBA domains and was shown to be necessary for the control of S. typhimurium. Interestingly, this mechanism requires Optineurin to be phosphorylated by TBK1 on serine-177 in order to enhance ubiquitin- and LC3-binding affinity to promote bacterial clearance (70). One aspect of S. typhimurium recognition by autophagy machinery that remained elusive was which bacterial substrates are ubiquitinated prior to detection by the adaptors p62, NDP52, and Optineurin. A recent study from Fiskin et al. in which ubiquitination site profiling was performed during infection with S. typhimurium revealed that outer membrane proteins are targets for ubiquitination (71).

Several ubiquitin-ligases have been reported as necessary for ubiquitination of S. typhimurium. Leucine-rich repeat and sterile α-motif-containing 1 (LRSAM1) was shown to play an essential role in the autophagic degradation of S. typhimurium. This E3-ligase was found to localize to cytosolic Salmonella upon infection of epithelial cells to ensure proper ubiquitination and autophagic control of bacterial replication. In line with these findings, a cohort study reported that lymphoblasts from patients with Charcot-Marie-Tooth disease, which harbor a frameshift mutation that truncates the RING domain of LRSAM1, present limited antibacterial activity as compared to cells from control individuals (72, 73). Another E3-ligase demonstrated to be involved in autophagic targeting of S. typhimurium is RNF166. This gene was identified in a screening for human E3-ligases as necessary for the recruitment of p62, NDP52, and LC3 for the bacterial surface in order to limit S. typhimurium replication. A unique feature of RNF166 is that, rather than tagging bacteria, it drives K29- and K33-linked ubiquitination of p62 at K91 and K189. According to the authors, this step is essential for p62-dependent bacterial targeting for autophagosomes (74) (Figure 4). More recently, the role of LUBAC, another E3-ligase, has been described. LUBAC generates linear (M1-linked) polyubiquitin patches on the surface of S. typhimurium to recruit the adaptors Optineurin, NDP52 and p62 and direct bacteria for autophagic degradation. Indeed, MEFs from cpdm−/− mice, which harbor a spontaneous mutation in LUBAC or MEFs silenced for the protein, display an increased time-dependent replication of S. typhimurium in comparison to wild-type or control-silenced cells, respectively (57). In addition to the recruitment of autophagy adaptors, LUBAC was reported to be crucial in triggering pro-inflammatory roles during infection with S. typhimurium (Figure 4). LUBAC-dependent generation of M1-linked polyubiquitin chains on the surface of the bacteria also recruits NEMO to this site (57, 75). These findings are of particular interest as it suggests that bacterial surface can provide mechanical support for the assembly of signaling platforms such as NF-kB activation, a major transcription factor that controls the production of inflammatory mediators such as cytokines and chemokines. Given that LRSAM1 was found to be only partially responsible for S. typhimurium ubiquitination, which RNF166 ubiquitates p62 rather than bacteria and that LUBAC required an upstream E3-ligase, Polajnar et al. hypothesized that other ubiquitin ligases were involved in the ubiquitination of S. typhimurium and identified Ring-between-Ring E3 ligase

anti-S. typhimurium mechanism has been conserved throughout evolution (67).

Although S. typhimurium targeting by autophagy was demonstrated, the means by which autophagosome formation machinery recognizes cytosolic Salmonella is not completely clear and has been the subject of many studies. Cytosolic, but not SCV residing bacteria, have been shown to be decorated with ubiquitin early during infection (66, 68). Interestingly, Ub+ bacteria colocalize with LC3, suggesting that the autophagic machinery can detect ubiquitinated substrates. Indeed, p62 has been shown to play a crucial role in the recognition, targeting to lysosomes
Siqueira et al.

---

ARIH1 (also known as HHARI) as an important protein ubiquitin-ligase for targeting this pathogen to autophagosomes (76). Notably, this study demonstrated that depletion of LRSAM1 and ARIH1 led to an enhancement in LUBAC-dependent ubiquitination and NF-κB activation, culminating with increased bacterial replication, in contrast to previous findings, reporting that NF-κB activation led to bacterial growth restriction (57, 75, 76). Together, these data indicate that recruitment of different ubiquitin-ligases (with different ubiquitin linkage abilities) to the bacterial surface may endow cells with several layers of protection against the replication of cytosolic S. typhimurium.

In addition to bacterial ubiquitination, lipid second messengers have also been reported to be required for efficient targeting of S. typhimurium. Shahnazari et al. demonstrated that diacylglycerol (DAG) is produced during infection with S. typhimurium in a phospholipase D- and phosphatidic acid phosphatase-dependent manner. DAG localization in bacteria-containing phagosomes seemed to be a requisite for autophagy and may occur in parallel to independent p62 and NDP52 recruitment, once again suggesting several layers of proteins involved in bacterial targeting (77).

The detection of damage in the SCV has been demonstrated to be an important step in the targeting of S. typhimurium for autophagic degradation (66). Galectin-8 is a β-galactoside-binding lectin that has been reported to monitor endosomal and lysosomal integrity and detects bacterial invasion by binding host glycans exposed on damaged SCVs. Recently, it has been demonstrated that among galectins 1–4, 7–10, and 12–14, only Galectin-8 colocalized to S. typhimurium during infection of HeLa cells. Interestingly, NDP52 was recruited to cytosolic exposed S. typhimurium, directly binding to Galectin-8 to restrict bacterial replication. These and previous findings lead to a model in which, upon SCV damage, host sugar molecules such as β-galactoside, usually confined to the lumen of endosomes are exposed in the cytosol and sensed by Galectin-8 that in turn recruits NDP52 and LC3 to SCV to initiate lysosomal degradation of S. typhimurium (42, 78). Despite their role in mediating S. typhimurium-induced autophagic degradation, p62 and NDP52 show independent targeting activity. In a study in which HeLa cells were silenced for p62 or NDP52, there was no interference in the number of NDP52+ or p62+ bacteria, respectively (Figure 4). Interestingly, it was demonstrated that these adaptors recognize ubiquitin deposited in distinct microdomains at the bacterial surface that could result from the activity from different ubiquitin-ligases (79). Future studies must provide explanations if and why cells preferably decide toward the employment of one or the other ubiquitin-ligase and autophagy adaptors.

### Listeria monocytogenes

Listeria monocytogenes is a Gram-positive bacterial pathogen that causes listeriosis, a self-limiting disease in healthy individuals that become severe in immunocompromised or elderly individuals and pregnant women (80). One of the main features of L. monocytogenes is its ability to replicate within several cell types during infection, including macrophages, a cell type usually able to kill the majority of intracellular bacteria (81).

Before its replication in the cytosol, L. monocytogenes must escape from the phagosome. This is achieved through the expression of several virulence factors rapidly upon entry. Possibly, the main bacterial factor associated with phagosome escape, listeriolysin O (LLO), is a cholesterol-dependent, pore-forming cytolyisin that form pores in the phagosomal membrane immediately after bacteria uptake (82–85). In experiments with fluorescently labeled molecules of increasing sizes, it has been demonstrated that the pores grow in size until large enough to allow bacterial escape (86). In addition to LLO pore-forming activity, two C-type phospholipases, phosphatidylinositol-specific (PI-PLC, plcB), and a broad-range phosphatidylcholine (PC-PLC, plcA) also contribute to L. monocytogenes escape from phagosome, likely digesting its membrane (86).

In order to successfully replicate in the cytosol, L. monocytogenes needs to circumvent several layers of host defense. Autophagy has been reported to contribute to the control of infection, although several studies show that the bacteria are able to escape from autophagic degradation (87, 88). Infection of RAW 264.7 macrophages of wild-type L. monocytogenes showed that ~40% of the intracellular bacterial population was targeted by LC3 by 1 h postinfection (p.i) in an LLO-dependent manner (Figure 5). However, at 8 h p.i, only 10% of the bacterial population was LC3-positive. These results suggest that L. monocytogenes was able to escape from autophagic degradation. Indeed, after initial targeting by LC3, replication rates robustly increased, in line with the drop in bacterial colocalization with LC3 observed at later stages of infection (89).

ActA, a key virulence factor of L. monocytogenes involved in intracellular motility, has also been implicated in autophagy evasion. In vitro studies demonstrate contradictory results with ΔactA mutants in different genetic backgrounds and cell types. While EGDe ΔactA mutants infecting Hela cells show time-dependent increase in the colocalization with LC3, 10403S ΔactA mutants in the 10403S background infecting macrophages loses its staining for LC3 at later time points during infection (88, 90). It remains to be elucidated whether the differences observed for both genetic backgrounds are related or not to the different cell types used. Despite this controversy, it is important to note that ΔactA mutants in both backgrounds display comparable replication in vitro (88–90). Importantly, ActA-dependent escape of autophagy does not rely on its ability to mediate bacterial motility. Using a series of ActA truncated mutants, Yoshikawa et al. demonstrated that as long as the capacity of ActA to recruit actin-related proteins (Arp) 2/3 complex, vasodilator-stimulated phosphoprotein or actin, is retained, bacteria are able to disguise autophagic recognition (88) (Figure 5).

The ubiquitination of L. monocytogenes and the involvement of autophagy adaptors such as p62 and NDP52 have been reported in the targeting L. monocytogenes to autophagosomes. In HeLa cells, p62 and NDP52 were shown to be recruited independently during the infection with the ΔactA EGDe (56). Recently, these strains were compared in regards to LC3, p62, and Ub during infection of macrophages. Although ΔactA mutants in EGDe and 10403S genetic backgrounds were reported to be sharply different regarding colocalization with LC3, p62, and Ub recruitment and replication were identical for both strains suggesting that ΔactA can block xenophagy downstream of ubiquitination and LC3 recruitment (90). The ubiquitin-ligases Parkin and SMURF1 were

---

Frontiers in Immunology | www.frontiersin.org  
May 2018 | Volume 9 | Article 935  
Siqueira et al.

---

**Listeria monocytogenes**

Wild-type L. monocytogenes

$\Delta_{\text{actA}}$ or $\Delta_{\text{plcA/B}}$ L. monocytogenes

LLO+

phagosome

LLO+

No recruitment of  
autophagic adaptors

ActA+

LLO-dependent  
phagosome  
damage

ActA-dependent  
autophagy escape

plcA/B+ --------

WIP1

ATG16

DFCP-1

plcA/B-induced stalled  
autophagosome  
formation

---

FIGURE 5 | ActA and phospholipases (plc) A and B are major *Listeria monocytogenes* virulence factors involved in the escape from autophagy. Upon Listeriolysin O-dependent escape from phagosome wild-type *L. monocytogenes* escapes from autophagic targeting due to the expression of ActA, that blocks the recruitment of p62 and NDP52 to the bacterial surface. The expression of plcA/B stalls the formation of autophagosomal membranes. Galectin recognition of damaged vacuoles and autophagy induction are not shown here.

demonstrated to play a role in the ubiquitination of *L. monocytogenes*. Park2$^{-/-}$ mice infected with *L. monocytogenes* showed up to 20-fold higher bacterial load relative to wild-type animals (31). Smurf1$^{-/-}$ macrophages infected with $\Delta_{\text{actA}}$ *L. monocytogenes* do not show recruitment of K48-ubiquitin to the bacterial surface while K63-linked ubiquitination was not affected. In line with this finding, Smurf1$^{-/-}$ mice infected with *L. monocytogenes* were shown to harbor significantly higher bacterial burdens in comparison to wild-type (33). As ubiquitination of cytosolic bacteria has been known to be essential for autophagic degradation, it is assumed that the higher bacterial burden in Smurf1$^{-/-}$ mice is a consequence of dampened antibacterial autophagy. More recently, NEDD4 (neuronal precursor cell expressed, developmentally downregulated 4), another ubiquitin-ligase has been implicated autophagic degradation of *L. monocytogenes* (Figure 5). However, in contrast to Parkin and Smurf1, NEDD4 does not recruit ubiquitin to the bacterial surface but enhances the mediated K6- and K27-linkage ubiquitination of BECN1, leading to higher stability of BECN1 and increased autophagy (91).

*Listeria monocytogenes* has been reported to induce amino acid starvation and activation of the general control nonderepressible 2 (GCN2)-eIF2$\alpha$ pathway upstream mTOR. GCN2 is one of four “stress kinases” that block translation by phosphorylating eIF2$\alpha$. GCN2 is thought to bind uncharged tRNAs to “sense” amino acids availability (92). Upon detection of a decrease in the amino acid pool, mTOR activity is reduced leading to autophagy activation to normalize this condition. Unlike what is observed during the infection of epithelial cells with *S. flexneri*, in *L. monocytogenes*-infected cells, autophagy is kept repressed, suggesting that *L. monocytogenes* possesses other virulence weapons to block autophagy (93–95). In addition to $\Delta_{\text{actA}}$-mediated escape from autophagy, *L. monocytogenes* employs its two C-type

---

Frontiers in Immunology | www.frontiersin.org
May 2018 | Volume 9 | Article 935

phospholipases to disrupt the autophagosome elongation step in order to inhibit autophagy-dependent degradation. In an *in vitro* study, it was observed that *L. monocytogenes* deleted for plcA and plcB were more strongly targeted to autophagosomes than wild-type bacteria at later time points of infection. In parallel, wild-type bacteria induced the accumulation of granules positive for LC3, ATG16L1, and as well as WIPI-2, a phosphatidylinositol 3-phosphate-binding protein that is present on maturing phagophores, suggesting blockade of pre-autophagosome structures. Interestingly, the authors demonstrate that in plcA/plcB *L. monocytogenes* mutants, the accumulation of such structures was not observed (93, 95). These results, together with the previous findings of Mitchell et al. point toward the combined effects of ActA and *L. monocytogenes* phospholipases in the escape from autophagy (87) (Figure 4).

The detection of MAMPs has also been described as an autophagy trigger during the infection of *L. monocytogenes*. In 2008, a study using *Drosophila melanogaster* as a model for *L. monocytogenes* infection reported that peptidoglycan-recognition protein (PGRP-LE) mediated autophagy-dependent control of bacterial replication *in vitro* and *in vivo* (96). Interestingly, the intracellular peptidoglycan receptor Nod1 has also been linked to xenophagy of *L. monocytogenes in vitro*. MEFs from Nod1-deficient mice were demonstrated to be defective in targeting *L. monocytogenes* to autophagosomes, indicating an important role for peptidoglycan recognition in the induction of autophagy during infection with this bacterium in mammals as well (49).

### Legionella pneumophila

The Gram-negative bacterium *L. pneumophila* was first identified as the causative agent of an epidemic of pneumonia at an American Legion convention in Philadelphia, PA, USA in 1976 (97). This disease is characterized by the inhalation of aerosols containing high numbers of *L. pneumophila* (98). Although usually found in freshwater protozoa and amebae, *L. pneumophila* can accidentally replicate in alveolar macrophages in human lung, especially in immune-compromised patients (99, 100). In order to replicate within its eukaryotic host, *L. pneumophila* employs strategies that involve blocking the fusion of phagosomes with lysosomes after phagocytic ingestion of the bacteria and the generation of endoplasmic reticulum (ER)-like compartment that affords its replication (7, 101). The *L. pneumophila*-containing vacuoles (LCVs) present features that are shared by autophagosomes, including its close association with ER membrane (100, 102). This led to the speculation that the formation of biogenesis could involve the autophagy machinery (103, 104). Initial studies that focused on the characterization of the LCV reported that this compartment did not fuse with acidic vesicles since proteins that localize to endolysosomes, such as Lamp-1 and Rab7 were absent in LCV membrane and that the ability of *L. pneumophila* to evade phagosomal maturation was dependent on its viability (98, 105). Further studies using avirulent strains of *L. pneumophila* identified the intracellular multiplication (*icm*) and defect in organelle trafficking (*dot*) loci as the genetic loci determinants required for intracellular multiplication and evasion phagosome–lysosome fusion (106, 107). The emergence of autophagy as an antimicrobial effector led to the examination of the role

**FIGURE 6** | *Legionella pneumophila* disrupts autophagy to create a replicative niche. *L.* *pneumophila* secretes RavZ through its Dot/Icm apparatus in order to deconjugate LC3 from ER-derived vesicles and block autophagy in order to escape from autophagic degradation.

of autophagy have been described as well. Phylogenetic analyses suggested a high degree of similarity between one *L. pneumophila* and the eukaryotic sphingosine-1 phosphate lyase (SPL) (111). The *L. pneumophila* SPL homolog (LpSpl) has similar enzymatic activities to the eukaryotic SPL, which finely regulates intracellular levels of sphingosine-1-phosphate (S1P) (112), which have been shown to stimulate autophagy (111, 113). Infection of macrophages with wild-type *L. pneumophila* but not the LpSpl-deficient mutant leads to a depletion in S1P levels and inhibition of autophagy, indicating that *L. pneumophila* uses molecular mimicry to block autophagy and replicate within macrophages (111).

### The Role AMP-Dependent Protein Kinase (AMPK) Activation and Bacterial-Induced Amino Acid Starvation in Bacterial Xenophagy

Living organisms obtain energy from the catabolism of nutrients whose molecular blocks are then converted into ATP and NADPH. The fact that cells are continually synthesizing ATP keeps its level close to maximal, with only small variations (114). However, under nutrient stress, when ATP levels drop, adenylate kinase shifts to an ATP synthesis mode to restore its levels. In turn, AMP levels increase significantly and, physiologically, changes in AMP concentrations are much higher than those observed to ATP (115), which makes the AMP/ATP ratio the most reliable marker of the cellular energetic status (114). Under such conditions, AMPK detects tiny changes in AMP levels and represents the principal cellular metabolism regulator (114). One of the main direct consequences of AMPK engagement is the activation of ULK1, suppressing mTORC1 inhibitory activity to allow the formation of autophagosomes (116).

In addition to its crucial role as a metabolic sensor, AMPK has also been widely reported to be involved in the activation of autophagy by bacteria. In a bacterial peritonitis-induced sepsis model, the use of the AMPK activator aminoimidazole carboxamide ribonucleotide (AICAR) increased bacterial killing, suggesting the implication of AMPK in the enhancement of the activity of phagocytic cells. Indeed, the use of these activators led to increased chemotaxis, phagocytosis, and bacterial killing of neutrophils infected with *Escherichia coli* (117). Evidence from the literature demonstrates that activation of AMPK by treatment with AICAR can also increase targeting of *M. tuberculosis* to LC3-positive compartments. Of note, when key autophagic proteins such as ATG7 were silenced, this effect was not observed, suggesting AICAR promotes the targeting of *M. tuberculosis* to autophagosomes. Moreover, AICAR-induced xenophagy was shown to contribute to bacterial killing *in vitro*, in a mechanism involving mTOR inhibition and increased mitochondrial biogenesis and ATP generation, likely as a result of energy drop during *M. tuberculosis* infection. Since it has been previously demonstrated that peroxisome proliferator-activated receptor-gamma, coactivator 1α (PPARGC1A) is important for the regulation of mitochondrial gene expression and glucose metabolism, it was speculated that PPARGC1A was involved in AICAR-induced xenophagy of *M. tuberculosis*. When PPARGC1A expression was silenced in macrophages infected with *M. tuberculosis*, the robust increase in mitochondrial biogenesis, ATP generation, and decreased *M. tuberculosis* replication induced by AICAR treatment were not

observed (118, 119). The findings of this study support those from Gutierrez et al. (3) demonstrating that induction of autophagy through rapamycin enhances antimicrobial defenses against *M. tuberculosis*. Although AMPK activation was found to be involved in the efficient xenophagy-dependent control of *M. tuberculosis*, this pathogen developed sophisticated mechanisms to manipulate AMPK activity in order to favor its replication. In another recent evidence, miRNAs emerged as important “fine-tuners” of gene expression in response to pathophysiological stimuli. These RNAs bind to the 3′-untranslated region of specific mRNAs to reduce protein expression by blocking mRNA translation or inducing its degradation (120). Accumulating evidence shows that many miRNAs regulate the complex interplay between mycobacterial survival strategies and host innate immune and metabolic pathways (121). One of these miRNAs, miR33 has been shown to the regulation of fatty acid metabolism and insulin signaling (122). *M. tuberculosis* seems to use the expression of miRNAs to subvert autophagy to create a favorable replicative niche. *M. tuberculosis* infection of macrophages induces the expression of miR-33 and its passenger strand miR-33* to dampen mitochondrial fatty acid oxidation and lipophagy (autophagy of lipid droplets) to increase cellular lipid content, which is essential for the bacilli as a nutrient source during infection (34). According to this study, autophagy inhibition was achieved by inhibition of AMPK, which controls transcription factor EB and Forkhead box transcription factor class O (FOXO3), transcriptional regulators of autophagy and lysosomal biogenesis gene programs, respectively (123, 124). Altogether, these studies demonstrate that AMPK is activated during infection with intracellular bacteria.

The mechanisms by which intracellular bacteria initiate xenophagy are not completely elucidated, but compelling evidence from the literature suggests that these pathogens trigger energy imbalance and cellular nutritional stress that result in the activation of cellular responses culminating in the upregulation of autophagic activity (93). It has been reported that the infection of epithelial cells with *S. flexneri* infection induces a general and persistent loss of amino acids, leading to amino acid starvation-induced stress (94). In contrast, *S. typhimurium* induces only a rapid and transient depletion of amino acid pools. Of note, during *S. flexneri-* and *S. typhimurium*-induced amino acid depletion, robust relocalization of mTORC1 is observed. While in *S. flexneri*-infected cells, S6K1 and 4EBP1, two major targets of mTOR, are downregulated, and mTORC1 dispersed in the cytosol throughout infection, during the infection with *S. typhimurium*, this is observed only in early time-points, suggesting that this bacterial pathogen developed means to manipulate mTOR signaling to favor its intracellular survival (94) (Figure 7). Indeed, in a recent study, Ganesan et al. demonstrated that despite sustained low levels of ATP in macrophages infected by *S. typhimurium*, AMPK was only transiently activated at early time-points and then returned to basal levels (125). AMPK activation is known to be regulated by a cytosolic complex consisting of liver kinase B1 (LKB1) and Sirtuin-1 (Sirt1), where Sirt1 is necessary for deacetylation and activation of LKB1 (126). Interestingly, the study by Ganesan et al. reports that *S. typhimurium* induces the lysosomal degradation of AMPK, LKB1, and Sirt1 to reactivate mTORC1 activity in order to inhibit autophagosome formation

Siqueira et al.

---

**Shigella flexneri**                                      **Salmonella Typhimurium**                               **Listeria monocytogenes**

![Diagram](#)

FIGURE 7 | Bacteria-induced amino acid starvation stress triggers xenophagy. *Shigella flexneri* induces a persistent amino acid starvation that leads to GCN2 activation and inhibition of mTORC1 to allow the formation of bacteria targeted autophagosomes (left). In contrast, *Salmonella Typhimurium* (center) and *Listeria monocytogenes* (right) trigger only a transient amino acid starvation and inhibition of mTORC1, allowing its reactivation at later time-points to block the formation of autophagosomes.

and escape from xenophagy. Notably, this reactivation was shown to be dependent on SsrB, a regulator of pathogenicity island 2 (SPI2) encoded virulence factors (127), and SsaV a component of the SPI2 type III secretion apparatus (128). *S.* typhimurium mutants lacking SsrB and SsaV failed to induce lysosomal degradation of the AMPK/LKB1/Sirt1 circuit and are efficiently targeted to autophagosomes (125). Together, the studies from Tattoli et al. and Ganesan et al. demonstrate that nutritional cellular stress induced by bacterial infection triggers xenophagy to control bacterial replication (in the case of *S.* flexneri infection) and that *S.* typhimurium developed mechanisms to escape from autophagy by reactivating mTORC1 activity. Together, the studies from Tattoli et al. and Ganesan et al. demonstrate that nutritional cellular stress induced by bacterial infection triggers xenophagy to control bacterial replication (in the case of *S.* flexneri infection) and that *S.* typhimurium developed mechanisms to escape from autophagy by reactivating mTORC1 activity.

*Listeria monocytogenes* has also been reported to induce amino acid starvation-induced cellular stress and activation of the GCN2-eIF2α pathway upstream mTOR. Upon detection of a decrease in the amino acid pool, mTOR activity is reduced leading to autophagy activation in order to normalize this condition. Unlike what is observed during the infection of epithelial cells with *S.* flexneri, in *L.* monocytogenes cells, autophagy is kept repressed, suggesting that *L.* monocytogenes possesses other virulence weapons to block autophagy (93–95) (Figure 7). Finally, AMPK has also been implicated in the enhancement of xenophagy during the infection with *E.* coli. According to this study, *E.* coli infection leads to an increase in intracellular calcium levels, which activates Ca(2+)/calmodulin-dependent protein kinase kinase β (CaMKKβ) to promote AMPK activation. AMPK was undoubtedly implicated in CaMKKβ-mediated xenophagy when macrophages were silenced for AMPK and control of *E.* coli replication was dampened (129).

## CONCLUDING REMARKS

Xenophagy has been widely reported to target bacteria for autophagic degradation, with clear impact on intracellular bacterial handling. Even with major advances in our understanding of the mechanisms involved in cargo selection, many questions remain unanswered. For example, why so many different mechanisms to target bacterial pathogens exposed to the cytosol? Still, why different autophagic adaptors and ubiquitin-ligases with apparent redundant functions? Although no evidence in this direction has been reported, we cannot exclude that different types of autophagosomes exist. Thus, p62, NDP52, NBR1, and Optineurin would function as sorters for different autophagosomes. It is possible that the different ubiquitin-ligases work in this direction as well by adding different ubiquitin linkages to the bacterial surface. Regarding bacteria-induced nutritional stress and autophagy induction, it is still to be elucidated whether amino acid starvation is induced upon infection with bacterial pathogens other than *Shigella*, *Salmonella*, and *Listeria*. Also, why bacteria induce amino acid starvation that leads to autophagy to subsequently inhibit it?

In vitro studies, it is clear that only a fraction of the intracellular bacterial population is targeted to autophagosomes, with modest impact in bacterial replication control following autophagy ablation. This is in sharp contrast to in vivo studies, which demonstrate much more pronounced differences in
bacterial replication in the absence of autophagy. How would these differences be explained?

Future goals in the field must address these open questions to provide a full understanding of the role of autophagy in bacterial infections.

## AUTHOR CONTRIBUTIONS

LT, MS, and RM wrote the paper. LT also revised the final document.

## FUNDING

Research in Leonardo Travassos' Lab was funded by Fundação Carlos Chagas Filho de Amparo à Pesquisa do Estado do Rio de Janeiro (Faperj) and Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq). LT is research fellow from Faperj (Cientista do Nosso Estado) and CNPq (1D). MS was supported by a fellowship from Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES). RM was supported by a fellowship from CNPQ.

## REFERENCES

1. Hussey S, Travassos LH, Jones NL. Autophagy as an emerging dimension to adaptive and innate immunity. *Semin Immunol* (2009) 21:233–41. doi: 10.1016/j.smim.2009.05.004
2. Rikihisa Y. Glycogen autophagosomes in polymorphonuclear leukocytes induced by rickettsiae. *Anat Rec* (1984) 208:319–27. doi: 10.1002/ar.1092080302
3. Gutierrez MG, Master SS, Singh SB, Taylor GA, Colombo MI, Deretic V. Autophagy is a defense mechanism inhibiting BCG and *Mycobacterium tuberculosis* survival in infected macrophages. *Cell* (2004) 119:753–66. doi: 10.1016/j.cell.2004.11.038
4. Nakagawa I, Amano A, Mizushima N, Yamamoto A, Yamaguchi H, Kamimoto T, et al. Autophagy defends cells against invading group A *Streptococcus*. *Science* (2004) 306:1037–40. doi: 10.1126/science.1103966
5. Amano A, Nakagawa I, Yoshimori T. Autophagy in innate immunity against intracellular bacteria. *J Biochem* (2006) 140:161–6. doi: 10.1093/jb/mvj162
6. Huang J, Brumell JH. Bacteria-autophagy interplay: a battle for survival. *Nat Rev Microbiol* (2014) 12:101–14. doi: 10.1038/nrmicro3160
7. Kagan JC, Stein MP, Pypaert M, Roy CR. *Legionella* subvert the functions of Rab1 and Sec22b to create a replicative organelle. *J Exp Med* (2004) 199:1201–11. doi: 10.1084/jem.20031706
8. Yang Z, Klionsky DJ. Mammalian autophagy: core molecular machinery and signaling regulation. *Curr Opin Cell Biol* (2010) 22:124–31. doi: 10.1016/j.ceb.2009.11.014
9. Suzuki H, Osawa T, Fujioka Y, Noda NN. Structural biology of the core autophagy machinery. *Curr Opin Struct Biol* (2017) 43:10–7. doi: 10.1016/j.sbi.2016.09.010
10. Yin Z, Pascual C, Klionsky DJ. Autophagy: machinery and regulation. *Microb Cell* (2016) 3:457–65. doi: 10.15698/mic2016.12.546
11. Jung CH, Jun CB, Ro SH, Kim YM, Otto NM, Cao J, et al. ULK-Atg13-FIP200 complexes mediate mTOR signaling to the autophagy machinery. *Mol Biol Cell* (2009) 20:1992–2003. doi: 10.1091/mbc.E08-12-1249
12. Kageyama S, Omori H, Saitoh T, Sone T, Guan J-L, Akira S, et al. The LC3 recruitment mechanism is separate from Atg9L1-dependent membrane formation in the autophagic response against *Salmonella*. *Mol Biol Cell* (2011) 22:2290–300. doi: 10.1091/mbc.E10-11-0893
13. Nair U, Cao Y, Xie Z, Klionsky DJ. Roles of the lipid-binding motifs of Atg18 and Atg21 in the cytoplasm to vacuole targeting pathway and autophagy. *J Biol Chem* (2010) 285:11476–88. doi: 10.1074/jbc.M109.080374
14. Carneiro LAM, Travassos LH. The interplay between NLRs and autophagy in immunity and inflammation. *Front Immunol* (2013) 4:361. doi: 10.3389/fimmu.2013.00361
15. Fujita N, Itoh T, Omori H, Fukuda M, Noda T, Yoshimori T. The Atg16L complex specifies the site of LC3 lipidation for membrane biogenesis in autophagy. *Mol Biol Cell* (2008) 19:2092–100. doi: 10.1091/mbc.E07-12-1257
16. Sou YS, Waguri S, Iwata J, Ueno T, Fujimura T, Hara T, et al. The Atg8 conjugation system is indispensable for proper development of autophagic isolation membranes in mice. *Mol Biol Cell* (2008) 19:4762–75. doi: 10.1091/mbc.E08-03-0309
17. Heckmann BL, Boada-Romero E, Cunha LD, Magne J, Green DR. LC3-associated phagocytosis and inflammation. *J Mol Biol* (2017) 429:3561–76. doi: 10.1016/j.jmb.2017.08.012
18. Martinez J, Malireddi RK, Lu Q, Cunha LD, Pelletier S, Gingras S, et al. Molecular characterization of LC3-associated phagocytosis reveals distinct roles for Rubicon, NOX2 and autophagy proteins. *Nat Cell Biol* (2015) 17:893–906. doi: 10.1038/ncb3192
19. Sanjuan MA, Dillon CP, Tait SWG, Moshiach S, Dorsey F, Connell S, et al. Toll-like receptor signalling in macrophages links the autophagy pathway to phagocytosis. *Nature* (2007) 450:1253–7. doi: 10.1038/nature06421
20. Espert L, Beaumelle B, Vergne I. Autophagy in *Mycobacterium tuberculosis* and HIV infections. *Front Cell Infect Microbiol* (2015) 5:49. doi: 10.3389/fcimb.2015.00049
21. Lam A, Prabhu R, Gross CM, Riesenberger LA, Singh V, Aggarwal S. Role of apoptosis and autophagy in tuberculosis. *Am J Physiol Lung Cell Mol Physiol* (2017) 313:L218–L229. doi: 10.1152/ajplung.00162.2017
22. Vergne I, Chua J, Singh SB, Deretic V. Cell biology of *Mycobacterium tuberculosis* phagosome. *Annu Rev Cell Dev Biol* (2004) 20:367–94. doi: 10.1146/annurev.cellbio.20.010403.114015
23. Zhang SY, Boisson-Dupuis S, Chapgier A, Yang K, Bustamante J, Puel A, et al. Inborn errors of interferon (IFN)-mediated immunity in humans: insights into the respective roles of IFN-α/β, IFN-γ, and IFN-λ in host defense. *Immunol Rev* (2008) 226:29–40. doi: 10.1111/j.1600-065X.2008.00698.x
24. Singh SB, Davis AS, Taylor GA, Deretic V. Human IRGM induces autophagy to eliminate intracellular mycobacteria. *Science* (2006) 313:1438–41. doi: 10.1126/science.1129577
25. Singh SB, Ornatowski W, Vergne I, Naylor J, Delgado M, Roberts E, et al. Human IRGM regulates autophagy and cell-autonomous immunity functions through mitochondria. *Nat Cell Biol* (2010) 12:1154–65. doi: 10.1038/ncb2119
26. Kim B-H, Shenoy AR, Kumar P, Das R, Tiwari S, Macmicking JD. A family of IFN-γ-inducible 65-kD GTPases protects against bacterial infection. *Science* (2011) 332:717–21. doi: 10.1126/science.1201711
27. Kimmey JM, Huynh JP, Weiss LA, Park S, Kambal A, Debnath J, et al. Unique role for ATG5 in neutrophil-mediated immunopathology during *M. tuberculosis* infection. *Nature* (2015) 528:565–9. doi: 10.1038/nature16451
28. Delgado MA, Elmaoued RA, Davis AS, Kyei G, Deretic V. Toll-like receptors control autophagy. *EMBO J* (2008) 27:1110–21. doi: 10.1038/emboj.2008.31
29. Watson RO, Manzanillo PS, Cox JS. Extracellular *M. tuberculosis* DNA targets bacteria for autophagy by activating the host DNA-sensing pathway. *Cell* (2012) 150:803–15. doi: 10.1016/j.cell.2012.06.040
30. Collins CA, De Maziere A, Van Dijk S, Carlsson F, Klumperman J, Brown EJ. Atg5-independent sequestration of ubiquitinated mycobacteria. *PLoS Pathog* (2009) 5:e1000430. doi: 10.1371/journal.ppat.1000430
31. Manzanillo PS, Ayres JS, Watson RO, Collins AC, Souza G, Rae CS, et al. The ubiquitin ligase parkin mediates resistance to intracellular pathogens. *Nature* (2013) 501:512–6. doi: 10.1038/nature12566
32. De Leuseleuc L, Orlova M, Cobat A, Girard M, Huong NT, Ba NN, et al. PARK2 mediates interleukin 6 and monocyte chemoattractant protein 1 production by human macrophages. *PLoS Negl Trop Dis* (2013) 7:e2015. doi: 10.1371/journal.pntd.0002015
33. Franco LH, Nair VR, Scharn CR, Xavier RJ, Torrealba JR, Shiloh MU, et al. The ubiquitin ligase smurf1 functions in selective autophagy of *Mycobacterium tuberculosis* and anti-tuberculous host defense. *Cell Host Microbe* (2017) 21:59–72. doi: 10.1016/j.chom.2016.11.002
34. Ouimet M, Koster S, Sakowski E, Ramkhelawon B, Van Solingen C, Oldebeken S, et al. *Mycobacterium tuberculosis* induces the miR-33 locus to reprogram autophagy and host lipid metabolism. *Nat Immunol* (2016) 17:677–86. doi:10.1038/ni.3434

35. Kim JK, Yuk JM, Kim SY, Kim TS, Jin HS, Yang CS, et al. MicroRNA-125a inhibits autophagy activation and antimicrobial responses during mycobacterial infection. *J Immunol* (2015) 194:5355–65. doi:10.4049/jimmunol.1402557

36. Chen Z, Wang T, Liu Z, Zhang G, Wang J, Feng S, et al. Inhibition of autophagy by MiR-30A induced by mycobacteria tuberculosis as a possible mechanism of immune escape in human macrophages. *Jpn J Infect Dis* (2015) 68:420–4. doi:10.7883/yoken.JJID.2014.466

37. Kim JK, Lee HM, Park KS, Shin DM, Kim TS, Kim YS, et al. MIR144* inhibits antimicrobial responses against *Mycobacterium tuberculosis* in human monocytes and macrophages by targeting the autophagy protein DRAM2. *Autophagy* (2017) 13:423–41. doi:10.1080/15548627.2016.1241922

38. Saini NK, Baena A, Ng TW, Venkataswamy MM, Kennedy SC, Kunnath-Velayudhan S, et al. Suppression of autophagy and antigen presentation by *Mycobacterium tuberculosis* PE_PGRS47. *Nat Microbiol* (2016) 1:16133. doi:10.1038/nmicrobiol.2016.133

39. Carapetis JR, Steer AC, Mulholland EK, Weber M. The global burden of group A streptococcal diseases. *Lancet Infect Dis* (2005) 5:685–94. doi:10.1016/S1473-3099(05)70267-X

40. Sakurai A, Maruyama F, Funao J, Nozawa T, Aikawa C, Okahashi N, et al. Specific behavior of intracellular *Streptococcus pyogenes* that has undergone autophagic degradation is associated with bacterial streptolysin O and host small G proteins Rab5 and Rab7. *J Biol Chem* (2010) 285:22666–75. doi:10.1074/jbc.M109.100131

41. Barnett TC, Liebl D, Seymour LM, Gillen CM, Lim JY, Larock CN, et al. The globally disseminated M1T1 clone of group A *Streptococcus* evades autophagy for intracellular replication. *Cell Host Microbe* (2013) 14:675–82. doi:10.1016/j.chom.2013.11.003

42. Thurston TL, Ryzhakov G, Bloor S, Von Muhlinen N, Randow F. The TBK1 adaptor and autophagy receptor NDP52 restricts the proliferation of ubiquitin-coated bacteria. *Nat Immunol* (2009) 10:1215–21. doi:10.1038/ni.1800

43. Joubert PE, Meiffren G, Gregoire IP, Pontini G, Richetta C, Flacher M, et al. Autophagy induction by the pathogen receptor CD46. *Cell Host Microbe* (2009) 6:354–66. doi:10.1016/j.chom.2009.09.006

44. Cattaneo R. Four viruses, two bacteria, and one receptor: membrane cofactor protein (CD46) as pathogens' magnet. *J Virol* (2004) 78:4385–8. doi:10.1128/JVI.78.9.4385-4388.2004

45. Ito C, Saito Y, Nozawa T, Fujii S, Sawa T, Inoue H, et al. Endogenous nitrated nucleotide is a key mediator of autophagy and innate defense against bacteria. *Mol Cell* (2013) 52:794–804. doi:10.1016/j.molcel.2013.10.024

46. Cutting AS, Rosario Y, Mu R, Rodriguez A, Till A, Subramani S, et al. The role of autophagy during group B *Streptococcus* infection of blood-brain barrier endothelium. *J Biol Chem* (2014) 289:35711–23. doi:10.1074/jbc.M114.588657

47. Lu SL, Kawabata T, Cheng YL, Omori H, Hamasaki M, Kusaba T, et al. Endothelial cells are intrinsically defective in xenophagy of *Streptococcus pyogenes*. *PLoS Pathog* (2017) 13:e1006444. doi:10.1371/journal.ppat.1006444

48. Kotloff KL, Riddle MS, Platts-Mills JA, Pavlinac P, Zaidi AKM. Shigellosis. *Lancet* (2017) 391(10122):801–12. doi:10.1016/S0140-6736(17)33296-8

49. Travassos LH, Carneiro LAM, Ramjeet M, Hussey S, Kim Y-G, Magalhães JG, et al. Nod1 and Nod2 direct autophagy by recruiting ATG16L1 to the plasma membrane at the site of bacterial entry. *Nat Immunol* (2009) 11:55–62. doi:10.1038/ni.1823

50. Egile C, Loisel TP, Laurent V, Li R, Pantaloni D, Sansonetti PJ, et al. Activation of the CDC42 effector N-WASP by the *Shigella flexneri* IcsA protein promotes actin nucleation by Arp2/3 complex and bacterial actin-based motility. *J Cell Biol* (1999) 146:1319–32. doi:10.1083/jcb.146.6.1319

51. Ogawa M, Suzuki T, Tatsuno I, Abe H, Sasakawa C. IcsB, secreted via the type III secretion system, is chaperoned by IpgA and required at the post-invasion stage of *Shigella* pathogenicity. *Mol Microbiol* (2003) 48:913–31. doi:10.1046/j.1365-2958.2003.03489.x

52. Ogawa M, Yoshimori T, Suzuki T, Sagara H, Mizushima N, Sasakawa C. Escape of intracellular *Shigella* from autophagy. *Science* (2005) 307:727–31. doi:10.1126/science.1106036

53. Suzuki T, Miki H, Takenawa T, Sasakawa C. Neural Wiskott-Aldrich syndrome protein is implicated in the actin-based motility of *Shigella flexneri*. *EMBO J* (1998) 17:2767–76. doi:10.1093/emboj/17.10.2767

54. Suzuki T, Mimuro H, Miki H, Takenawa T, Sasaki T, Nakanishi H, et al. Rho family GTPase Cdc42 is essential for the actin-based motility of *Shigella* in mammalian cells. *J Exp Med* (2000) 191:1905–20. doi:10.1084/jem.191.11.1905

55. Baxt LA, Goldberg MB. Host and bacterial proteins that repress recruitment of LC3 to *Shigella* early during infection. *PLoS One* (2014) 9:e94653. doi:10.1371/journal.pone.0094653

56. Mostowy S, Sancho-Shimizu V, Hamon M, Simeone R, Brosch R, Johansen T, et al. p62 and NDP52 Proteins target intracytosolic *Shigella* and *Listeria* to different autophagy pathways. *J Biol Chem* (2011) 286:26987–95. doi:10.1074/jbc.M111.223610

57. Noad J, Von Der Malsburg A, Pathe C, Michel MA, Komander D, Randow F. LUBAC-synthesized linear ubiquitin chains restrict cytosol-invading bacteria by activating autophagy and NF-κB. *Nat Microbiol* (2017) 2. doi:10.1038/nmicrobiol.2017.63

58. Dupont N, Lacas-Gervais S, Bertout J, Paz I, Freche B, Van Nhieu GT, et al. *Shigella* phagocytic vacuolar membrane remnants participate in the cellular response to pathogen invasion and are regulated by autophagy. *Cell Host Microbe* (2009) 6:137–49. doi:10.1016/j.chom.2009.07.005

59. Srougi MC, Burridge K. Undressing a cellular corset: septins exposed. *Nat Cell Biol* (2009) 11:9–10. doi:10.1038/ncb0109-9

60. Mostowy S, Bonazzi M, Hamon MAA, Tham TN, Mallet A, Lelek M, et al. Entrapment of intracytosolic bacteria by septin cage-like structures. *Cell Host Microbe* (2010) 8:433–44. doi:10.1016/j.chom.2010.10.009

61. Sirianni A, Krokowski S, Lobato-Marquez D, Buranyi S, Pfanzelter J, Galea D, et al. Mitochondria mediate septin cage assembly to promote autophagy of *Shigella*. *EMBO Rep* (2016) 17:1029–43. doi:10.15252/embr.201541832

62. Carneiro LAM, Travassos LH, Soares F, Tattoli I, Magalhaes JG, Bozza MT, et al. *Shigella* induces mitochondrial dysfunction and cell death in nonmyleoid cells. *Cell Host Microbe* (2009) 5:123–36. doi:10.1016/j.chom.2008.12.011

63. Suzuki T, Franchi L, Toma C, Ashida H, Ogawa M, Yoshikawa Y, et al. Differential regulation of caspase-1 activation, pyroptosis, and autophagy via Ipaf and ASC in *Shigella*-infected macrophages. *PLoS Pathog* (2007) 3:e111. doi:10.1371/journal.ppat.0030111

64. Inohara N, Koseki T, Lin J, Del Peso L, Lucas PC, Chen FF, et al. An induced proximity model for NF-kappa B activation in the Nod1/RICK and RIP signaling pathways. *J Biol Chem* (2000) 275:27823–31. doi:10.1074/jbc.M003415200

65. Haselbeck AH, Panzner U, Im J, Baker S, Meyer CG, Marks F. Current perspectives on invasive nontyphoidal *Salmonella* disease. *Curr Opin Infect Dis* (2017) 30:498–503. doi:10.1097/QCO.000000000000398

66. Birmingham CL, Smith AC, Bakowski MA, Yoshimori T, Brumell JH. Autophagy controls *Salmonella* infection in response to damage to the *Salmonella*-containing vacuole. *J Biol Chem* (2006) 281:11374–83. doi:10.1074/jbc.M509157200

67. Jia K, Thomas C, Akbar M, Sun Q, Adams-Huet B, Gilpin C, et al. Autophagy genes protect against *Salmonella typhimurium* infection and mediate insulin signaling-regulated pathogen resistance. *Proc Natl Acad Sci USA* (2009) 106:14564–9. doi:10.1073/pnas.0813319106

68. Perrin AJ, Jiang X, Birmingham CL, So N, Brumell JH. Recognition of bacteria in the cytosol of mammalian cells by the ubiquitin system. *Curr Biol* (2004) 14:806–11. doi:10.1016/j.cub.2004.04.033

69. Zheng YT, Shahbazari S, Brech A, Lamark T, Johansen T, Brumell JH. The adaptor protein p62/SQSTM1 targets invading bacteria to the autophagy pathway. *J Immunol* (2009) 183:5909–16. doi:10.4049/jimmunol.0900441

70. Wild P, Farhan H, Mcewan DG, Wagner S, Rogov VV, Brady NR, et al. Phosphorylation of the autophagy receptor optineurin restricts *Salmonella* growth. *Science* (2011) 333:228–33. doi:10.1126/science.1205405

71. Fiskin E, Bionda T, Dikic I, Behrends C. Global analysis of host and bacterial ubiquitinome in response to *Salmonella typhimurium* infection. *Mol Cell* (2016) 62:967–81. doi:10.1016/j.molcel.2016.04.015

72. Guernsey DL, Jiang H, Bedard K, Evans SC, Ferguson M, Matsuoka M, et al. Mutation in the gene encoding ubiquitin ligase LRSAM1 in patients with Charcot-Marie-Tooth disease. *PLoS Genet* (2010) 6:e1001081. doi:10.1371/journal.pgen.1001081

73. Huett A, Heath RJ, Begun J, Sassi SO, Baxt LA, Vyas JM, et al. The LRR and RING domain protein LRSAM1 is an E3 ligase crucial for ubiquitin-dependent

Siqueira et al.

---

12. autophagy of intracellular *Salmonella typhimurium*. *Cell Host Microbe* (2012) 12:778–90. doi:10.1016/j.chom.2012.10.019

74. Heath RJ, Goel G, Baxt LA, Rush JS, Mohanan V, Paulus GLCLC, et al. RNF166 determines recruitment of adaptor proteins during antibacterial autophagy. *Cell Rep* (2016) 17:2183–94. doi:10.1016/j.celrep.2016.11.005

75. Van Wijk SJL, Fricke F, Herhaus L, Gupta J, Hotte K, Pampaloni F, et al. Linear ubiquitination of cytosolic *Salmonella typhimurium* activates NF-kappaB and restricts bacterial proliferation. *Nat Microbiol* (2017) 2:17066. doi:10.1038/nmicrobiol.2017.66

76. Polajnar M, Dietz MS, Heilemann M, Behrends C. Expanding the host cell ubiquitylation machinery targeting cytosolic *Salmonella*. *EMBO Rep* (2017) 18:1572–85. doi:10.15252/embr.201643851

77. Shahnazari S, Yen W-LL, Birmingham CL, Shiu J, Namolovan A, Zheng YT, et al. A diacylglycerol-dependent signaling pathway contributes to regulation of antibacterial autophagy. *Cell Host Microbe* (2010) 8:137–46. doi:10.1016/j.chom.2010.07.002

78. Thurston TLM, Wandel MP, Muhlinen N, Foeglein Á, Randow F. Galectin 8 targets damaged vesicles for autophagy to defend cells against bacterial invasion. *Nature* (2012) 482:414–8. doi:10.1038/nature10744

79. Cemma M, Kim PK, Brumell JH. The ubiquitin-binding adaptor proteins p62/SQSTM1 and NDP52 are recruited independently to bacteria-associated microdomains to target *Salmonella* to the autophagy pathway. *Autophagy* (2011) 7:341–5. doi:10.4161/auto.7.3.14046

80. Lam GY, Czuczman MA, Higgins DE, Brumell JH. Interactions of *Listeria monocytogenes* with the autophagy system of host cells. *Adv Immunol* (2012) 113:7–18. doi:10.1016/B978-0-12-394590-7.00008-7

81. Radoshevich L, Cossart P. *Listeria monocytogenes*: towards a complete picture of its physiology and pathogenesis. *Nat Rev Microbiol* (2018) 16:32–46. doi:10.1038/nrmicro.2017.126

82. Beauregard KE, Lee KD, Collier RJ, Swanson JA. pH-dependent perforation of macrophage phagosomes by listeriolysin O from *Listeria monocytogenes*. *J Exp Med* (1997) 186:1159–63. doi:10.1084/jem.186.7.1159

83. Cossart P, Vicente MF, Mengaud J, Baquero F, Perez-Diaz JC, Berche P. Listeriolysin O is essential for virulence of *Listeria monocytogenes*: direct evidence obtained by gene complementation. *Infect Immun* (1989) 57: 3629–36.

84. Henry R, Shaughnessy L, Loessner MJ, Alberti-Segui C, Higgins DE, Swanson JA. Cytolysin-dependent delay of vacuole maturation in macrophages infected with *Listeria monocytogenes*. *Cell Microbiol* (2006) 8:107–19. doi:10.1111/j.1462-5822.2005.00604.x

85. Portnoy DA, Jacks PS, Hinrichs DJ. Role of hemolysin for the intracellular growth of *Listeria monocytogenes*. *J Exp Med* (1988) 167:1459–71. doi:10.1084/jem.167.4.1459

86. Shaughnessy LM, Hoppe AD, Christensen KA, Swanson JA. Membrane perforations inhibit lysosome fusion by altering pH and calcium in *Listeria monocytogenes* vacuoles. *Cell Microbiol* (2006) 8:781–92. doi:10.1111/j.1462- 5822.2005.00665.x

87. Mitchell G, Ge L, Huang Q, Chen C, Kianian S, Roberts MF, et al. Avoidance of autophagy mediated by PlcA or ActA is required for *Listeria monocytogenes* growth in macrophages. *Infect Immun* (2015) 83:2175–84. doi:10.1128/IAI.00110-15

88. Yoshikawa Y, Ogawa M, Hain T, Yoshida M, Fukumatsu M, Kim M, et al. *Listeria monocytogenes* ActA-mediated escape from autophagic recognition. *Nat Cell Biol* (2009) 11:1233–40. doi:10.1038/ncb1967

89. Birmingham CL, Canadien V, Gouin E, Troy EB, Yoshimori T, Cossart P, et al. *Listeria monocytogenes* evades killing by autophagy during colonization of host cells. *Autophagy* (2007) 3:442–51. doi:10.4161/auto.4450

90. Cemma M, Lam GY, Stöckli M, Higgins DE, Brumell JH. Strain-specific interactions of *Listeria monocytogenes* with the autophagy system in host cells. *PLoS One* (2015) 10:e0125856. doi:10.1371/journal.pone.0125856

91. Pei G, Buijze H, Liu H, Moura-Alves P, Goosmann C, Brinkmann V, et al. The E3 ubiquitin ligase NEDD4 enhances killing of membrane-perturbing intracellular bacteria by promoting autophagy. *Autophagy* (2017) 13(12):2041–55. doi:10.1080/15548627.2017.1376160

92. VanDe Velde LA, Guo XJ, Barbaric I, Smith AM, Oguin TH III, Thomas PG, et al. Stress kinase GCN2 controls the proliferative fitness and trafficking of cytotoxic T cells independent of environmental amino acid sensing. *Cell Rep* (2016) 17:2247–58. doi:10.1016/j.celrep.2016.10.079

---

93. Tattoli I, Sorbara MT, Philpott DJ, Girardin SE. Stalling autophagy: a new function for *Listeria* phospholipases. *Microb Cell* (2014) 1:48–50. doi:10.15698/mic2014.01.124

94. Tattoli I, Sorbara MT, Vuckovic D, Ling A, Soares F, Carneiro LA, et al. Amino acid starvation induced by invasive bacterial pathogens triggers an innate host defense program. *Cell Host Microbe* (2012) 11:563–75. doi:10.1016/j.chom.2012.04.012

95. Tattoli I, Sorbara MT, Yang C, Tooze SA, Philpott DJ, Girardin SE. *Listeria* phospholipases subvert host autophagic defenses by stalling pre-autophagosomal structures. *EMBO J* (2013) 32:3066–78. doi:10.1038/emboj.2013.234

96. Yano T, Mita S, Ohmori H, Oshima Y, Fujimoto Y, Ueda R, et al. Autophagic control of *Listeria* through intracellular innate immune recognition in *Drosophila*. *Nat Immunol* (2008) 9:908–16. doi:10.1038/ni.1634

97. Fraser DW, Tsai TR, Orenstein W, Parkin WE, Beecham HJ, Sharrar RG, et al. Legionnaires' disease: description of an epidemic of pneumonia. *N Engl J Med* (1977) 297:1189–97. doi:10.1056/NEJM197712012972201

98. Horwitz MA. Formation of a novel phagosome by the Legionnaires' disease bacterium (*Legionella pneumophila*) in human monocytes. *J Exp Med* (1983) 158:1319–31. doi:10.1084/jem.158.4.1319

99. Jo E-K, Yuk J-M, Shin D-M, Sasakawa C. Roles of autophagy in elimination of intracellular bacterial pathogens. *Front Immunol* (2013) 4. doi:10.3389/fimmu.2013.00097

100. Sherwood RK, Roy CR. Autophagy evasion and endoplasmic reticulum subversion: the Yin and Yang of *Legionella* intracellular infection. *Annu Rev Microbiol* (2016) 70:413–33. doi:10.1146/annurev-micro-102215-095557

101. Kagan JC, Roy CR. *Legionella* phagosomes intercept vesicular traffic from endoplasmic reticulum exit sites. *Nat Cell Biol* (2002) 4:945–54. doi:10.1038/ncb883

102. Eskelinen E-L. Maturation of autophagic vacuoles in mammalian cells. *Autophagy* (2005) 1:1–10. doi:10.4161/auto.1.1.1270

103. Amer AO, Swanson MS. Autophagy is an immediate macrophage response to *Legionella pneumophila*. *Cell Microbiol* (2005) 7:765–78. doi:10.1111/j.1462-5822.2005.00509.x

104. Dubuisson JF, Swanson MS. Mouse infection by *Legionella*, a model to analyze autophagy. *Autophagy* (2006) 2:179–82. doi:10.4161/auto.2831

105. Joshi AD, Sturgill-Koszycki S, Swanson MS. Evidence that Dot-dependent and -independent factors isolate the *Legionella pneumophila* phagosome from the endocytic network in mouse macrophages. *Cell Microbiol* (2001) 3:99–114. doi:10.1046/j.1462-5822.2001.00093.x

106. Berger KH, Isberg RR. Two distinct defects in intracellular growth complemented by a single genetic locus in *Legionella pneumophila*. *Mol Microbiol* (1993) 7:7–19. doi:10.1111/j.1365-2958.1993.tb01092.x

107. Marra A, Blander SJ, Horwitz MA, Shuman HA. Identification of a *Legionella pneumophila* locus required for intracellular multiplication in human macrophages. *Proc Natl Acad Sci USA* (1992) 89:9607–11. doi:10.1073/pnas.89.20.9607

108. Choy A, Dancourt J, Mugo B, O’connor TJ, Isberg RR, Melia TJ, et al. The *Legionella* effector RavZ inhibits host autophagy through irreversible Atg8 deconjugation. *Science* (2012) 338:1072–6. doi:10.1126/science.1227026

109. Kubori T, Bui XT, Hubber A, Nagai H. *Legionella* RavZ plays a role in preventing ubiquitin recruitment to bacteria-containing vacuoles. *Front Cell Infect Microbiol* (2017) 7:384. doi:10.3389/fcimb.2017.00384

110. Horenkamp FA, Kauffman KJ, Kohler LJ, Sherwood RK, Krueger KP, Shteyn V, et al. The *Legionella* anti-autophagy effector RavZ targets the autophagosome via PI3P- and curvature-sensing motifs. *Dev Cell* (2015) 34:569–76. doi:10.1016/j.devcel.2015.08.010

111. Rolando M, Escoll P, Nora T, Botti J, Boitez V, Bedia C, et al. *Legionella pneumophila* S1P-lyase targets host sphingolipid metabolism and restrains autophagy. *Proc Natl Acad Sci USA* (2016) 113:1901–6. doi:10.1073/pnas.1522067113

112. Takabe K, Paugh SW, Milstien S, Spiegel S. “Inside-out” signaling of sphingosine-1-phosphate: therapeutic targets. *Pharmacol Rev* (2008) 60:181–95. doi:10.1124/pr.107.07113

113. Bedia C, Levade T, Codogno P. Regulation of autophagy by sphingolipids. *Anticancer Agents Med Chem* (2011) 11:844–53. doi:10.2174/187152011797655131

114. Hardie DG. AMP-activated protein kinase: an energy sensor that regulates all aspects of cell function. *Genes Dev* (2011) 25:1895–908. doi:10.1101/gad.17420111

---

Autophagy and Bacteria

Frontiers in Immunology | www.frontiersin.org 16 May 2018 | Volume 9 | Article 935
115. Ataullakhanov FI, Vitvitsky VM. What determines the intracellular ATP concentration. *Biosci Rep* (2002) 22:501–11. doi:10.1023/A:1022069718709

116. Kim J, Kundu M, Viollet B, Guan K-L. AMPK and mTOR regulate autophagy through direct phosphorylation of Ulk1. *Nat Cell Biol* (2011) 13:132–41. doi:10.1038/ncb2152

117. Park DW, Jiang S, Tadie JM, Stigler WS, Gao Y, Deshane J, et al. Activation of AMPK enhances neutrophil chemotaxis and bacterial killing. *Mol Med* (2013) 19:387–98. doi:10.2119/molmed.2013.00065

118. Leick L, Fentz J, Bienso RS, Knudsen JG, Jeppesen J, Kiens B, et al. PGC-1{alpha} is required for AICAR-induced expression of GLUT4 and mitochondrial proteins in mouse skeletal muscle. *Am J Physiol Endocrinol Metab* (2010) 299:E456–65. doi:10.1152/ajpendo.00648.2009

119. Yang CS, Kim JJ, Lee HM, Jin HS, Lee SH, Park JH, et al. The AMPK-PPARGC1A pathway is required for antimicrobial host defense through activation of autophagy. *Autophagy* (2014) 10:785–802. doi:10.4161/auto.28072

120. Rottiers V, Najafi-Shoushtari SH, Kristo F, Gurumurthy S, Zhong L, Li Y, et al. MicroRNAs in metabolism and metabolic diseases. *Cold Spring Harb Symp Quant Biol* (2011) 76:225–33. doi:10.1101/sqb.2011.76.011049

121. Kim JK, Kim TS, Basu J, Jo E-KK. MicroRNA in innate immunity and autophagy during mycobacterial infection. *Cell Microbiol* (2017) 19. doi:10.1111/cmi.12687

122. Davalos A, Goedeke L, Smibert P, Ramirez CM, Warrier NP, Andreo U, et al. miR-33a/b contribute to the regulation of fatty acid metabolism and insulin signaling. *Proc Natl Acad Sci USA* (2011) 108:9232–7. doi:10.1073/pnas.1102281108

123. He C, Klionsky DJ. Regulation mechanisms and signaling pathways of autophagy. *Annu Rev Genet* (2008) 43:67–93. doi:10.1146/annurev-genet-102808-114910

124. Settembre C, Di Malta C, Polito VA, Garcia Arencibia M, Vetrini F, Erdin S, et al. TFEB links autophagy to lysosomal biogenesis. *Science* (2011) 332:1429–33. doi:10.1126/science.1204592

125. Ganesan R, Hos NJ, Gutierrez S, Fischer J, Stepek JM, Daglidu E, et al. *Salmonella typhimurium* disrupts Sirt1/AMPK checkpoint control of mTOR

to impair autophagy. *PLoS Pathog* (2017) 13:e1006227. doi:10.1371/journal.ppat.1006227

126. Lan F, Cacicedo JM, Ruderman N, Ido Y. SIRT1 modulation of the acetylation status, cytosolic localization, and activity of LKB1. Possible role in AMP-activated protein kinase activation. *J Biol Chem* (2008) 283:27628–35. doi:10.1074/jbc.M805711200

127. Walthers D, Carroll RK, Navarre WW, Libby SJ, Fang FC, Kenney LJ. The response regulator SsrB activates expression of diverse *Salmonella* pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS. *Mol Microbiol* (2007) 65:477–93. doi:10.1111/j.1365-2958.2007.05800.x

128. Grant AJ, Morgan FJ, Mckinley TJ, Foster GL, Maskell DJ, Mastroeni P. Attenuated *Salmonella typhimurium* lacking the pathogenicity island-2 type 3 secretion system grow to high bacterial numbers inside phagocytes in mice. *PLoS Pathog* (2012) 8:e1003070. doi:10.1371/journal.ppat.1003070

129. Liu X, Wang N, Zhu Y, Yang Y, Chen X, Chen Q, et al. Extracellular calcium influx promotes antibacterial autophagy in *Escherichia coli* infected murine macrophages via CaMKKbeta dependent activation of ERK1/2, AMPK and FoxO1. *Biochem Biophys Res Commun* (2016) 469:639–45. doi:10.1016/j.bbrc.2015.12.052

**Conflict of Interest Statement:** The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Copyright © 2018 Siqueira, Ribeiro and Travassos. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.