
Cell

Structural Basis of Poxvirus Transcription: Transcribing and Capping Vaccinia Complexes

Graphical Abstract

![Image Description](image1)

Authors
Hauke S. Hillen, Julia Bartuli, Clemens Grimm, ..., Aladar A. Szalay, Utz Fischer, Patrick Cramer

Correspondence
utz.fischer@biozentrum.uni-wuerzburg.de (U.F.), pcramer@mpibpc.mpg.de (P.C.)

In Brief
Coupled transcript elongation and capping by the poxvirus RNA polymerase rely on large-scale rearrangements triggered by the nascent RNA.

Highlights
- Structures of Vaccinia RNA polymerase during transcription and cap formation
- Capping enzyme contains mobile modules that bind to the polymerase and transcript
- Comparison with complete vRNAP suggests large rearrangements during initiation
- The Vaccinia-specific Rap94 resembles TFIIB and is displaced by nucleic acids

Hillen et al., 2019, Cell 179, 1525–1536  
December 12, 2019 © 2019 Elsevier Inc.  
https://doi.org/10.1016/j.cell.2019.11.023

Article

# Structural Basis of Poxvirus Transcription: Transcribing and Capping Vaccinia Complexes

Hauke S. Hillen,$^{1,5}$ Julia Bartuli,$^{2,5}$ Clemens Grimm,$^{2,5}$ Christian Dienemann,$^{1}$ Kristina Bedenk,$^{2}$ Aladar A. Szalay,$^{2,3}$ Utz Fischer,$^{2,3,4,*}$ and Patrick Cramer$^{1,6,*}$

$^{1}$Department of Molecular Biology, Max Planck Institute for Biophysical Chemistry, Am Fassberg 11, 37077 Göttingen, Germany  
$^{2}$Department of Biochemistry and Cancer Therapy Research Center (CTRC), Theodor Boveri-Institute, University of Würzburg, 97074 Würzburg, Germany  
$^{3}$Genelux Corporation, 3030 Bunker Hill Street, San Diego, CA 92109, USA  
$^{4}$Helmholtz Institute for RNA-based Infection Research (HIRI), Helmholtz Centre for Infection Research (HZI), 97080 Würzburg, Germany  
$^{5}$These authors contributed equally  
$^{6}$Lead Contact  

*Correspondence: utz.fischer@biozentrum.uni-wuerzburg.de (U.F.), pcramer@mpibpc.mpg.de (P.C.)  
https://doi.org/10.1016/j.cell.2019.11.023

## SUMMARY

Poxviruses use virus-encoded multisubunit RNA polymerases (vRNAPs) and RNA-processing factors to generate m⁷G-capped mRNAs in the host cytoplasm. In the accompanying paper, we report structures of core and complete vRNAP complexes of the prototypic Vaccinia poxvirus (Grimm et al., 2019; in this issue of *Cell*). Here, we present the cryo-electron microscopy (cryo-EM) structures of Vaccinia vRNAP in the form of a transcribing elongation complex and in the form of a co-transcriptional capping complex that contains the viral capping enzyme (CE). The trifunctional CE forms two mobile modules that bind the polymerase surface around the RNA exit tunnel. RNA extends from the vRNAP active site through this tunnel and into the active site of the CE triphosphatase. Structural comparisons suggest that growing RNA triggers large-scale rearrangements on the surface of the transcription machinery during the transition from transcription initiation to RNA capping and elongation. Our structures unravel the basis for synthesis and co-transcriptional modification of poxvirus RNA.

## INTRODUCTION

Poxviruses belong to a group of DNA viruses with exceptionally large genomes that replicate in the host cytoplasm. Vaccinia, the non-pathogenic virus strain used as a smallpox vaccine and as a promising agent in oncolytic virotherapy (Mell et al., 2017), contains a ~190 kbp double-stranded DNA genome that is transcribed in the cytosol by an eight-subunit virus-encoded RNA polymerase (vRNAP) (Broyles, 2003). Although most of these subunits share sequence homology with subunits of cellular RNA polymerase II (Pol II), their degree of similarity differs from very strong to barely detectable (Ahn et al., 1990, 1992; Amegadzie et al., 1992, 1991; Broyles and Moss, 1986; Knutson and Broyles, 2008; Mirzakhanyan and Gershon, 2017; Patel and

Cell 179, 1525–1536, December 12, 2019 © 2019 Elsevier Inc. 1525

Viral gene expression typically follows defined temporal patterns, which are referred to as early, intermediate, and late transcription (Baldick and Moss, 1993). Early genes are activated shortly after infection and encode proteins necessary for the expression and replication of the viral genome. In poxviruses, specific transcription factors facilitate early gene transcription. Initiation is mediated by Rap94 and the very early transcription factor (VETF) (Ahn et al., 1994; Broyles et al., 1991, 1988; Cassetti and Moss, 1996). After initial transcription, capping occurs when the nascent RNA reaches a length of 27–31 nt (Hagler and Shuman, 1992a). The CE is required not only for capping but also for termination of early gene transcription and is therefore also referred to as vaccinia termination factor (VTF) (Luo et al., 1995). Termination is mediated by a signal sequence in the nascent RNA and requires, in addition to CE, the helicase nucleoside triphosphatase I (NPH-I) (Christen et al., 1998; Rohrmann et al., 1986; Shuman et al., 1987).

In the accompanying paper, we describe the purification and structural analysis of viral transcription complexes from human cells infected with a recombinant Vaccinia virus strain (Grimm et al., 2019). These studies revealed the structure of the eight-subunit core vRNAP enzyme and of the complete vRNAP complex with early viral transcription factors. The latter contains, in addition to the core vRNAP enzyme, the transcription factors Rap94, VETF, CE, and NPH-I, the structural protein E11, and host tRNA<sup>Gln</sup>. This complex is capable of early promoter-dependent transcription initiation, elongation, and termination. It thus represents the unit facilitating early gene transcription and might also be packaged into viral progenies. However, how the vRNAP machinery interacts with nucleic acids to achieve transcription and RNA modification remains unknown.

Here, we report the structures of actively transcribing vRNAP complexes. The structure of vRNAP bound to a DNA template and an RNA transcript reveals a similar mechanism of transcript elongation as shown for other multisubunit RNA polymerases and also demonstrates specific features of the vRNAP active site. The structure of transcribing vRNAP bound to CE illustrates the path of the RNA from the active site of the polymerase to one of the active sites of the CE and unravels structural rearrangements that occur during the transition from transcription initiation to elongation. Together, these results define the structural basis of poxviral transcription and co-transcriptional capping and provide a framework for future mechanistic analysis of the transcription cycle of viral multisubunit RNA polymerases.

### RESULTS

#### Preparation of vRNAP-Transcribing Complexes
We purified Vaccinia vRNAP complexes as described (Grimm et al., 2019) and formed transcribing complexes on a DNA-RNA scaffold consisting of a double-stranded DNA with a mismatch bubble (Figures S1A and 1C), a strategy previously used for the structural characterization of Pols I, II, and III (Hoffmann et al., 2015; Kettenberger et al., 2004; Neyer et al., 2016). The DNA fragment was derived from an early gene in the Vaccinia genome, encoding the largest subunit of vRNAP, Rpo147. The single-stranded template strand in the mismatched region was hybridized to an RNA, the 3′ end of which was complementary to the template strand, to mimic the nucleic acids in an actively transcribing complex.

To facilitate formation of a co-transcriptionally capping complex, we used T7 RNA polymerase to produce RNA via in vitro transcription in order to contain the 5′ triphosphate moiety also found in naturally synthesized transcripts. We designed a 31-nt RNA on the basis of previous results that demonstrated that co-transcriptional capping occurs at a nascent RNA length of 27–31 nt (Hagler and Shuman, 1992a) (see STAR Methods). To assemble vRNAP elongation complexes, we incubated vRNAP with a large excess of the pre-formed DNA-RNA scaffold after initial FLAG purification (Figure S1A). After further purification by sucrose gradient centrifugation, two populations with distinct sedimentation coefficients could be observed, similar to the previously observed vRNAP complexes lacking nucleic acids (Figure S1B) (Grimm et al., 2019).

#### Structure Determination of vRNAP Bound to Nucleic Acids
Fractions containing the larger-molecular-weight complex were subjected to single-particle cryo-EM analysis (Figures S1B and S2). 3D classification of the obtained dataset showed two distinct populations of particles (Figure S2). The first closely resembled the previously determined core vRNAP structure (Grimm et al., 2019) but showed additional density for nucleic acids in the active center cleft. The second class showed large additional densities on the enzyme surface where the nascent RNA was expected to emerge. Further sub-classification and 3D refinement yielded high-resolution reconstructions for the first and second classes at 2.9 and 3.1 Å, respectively (Figures S2 and S3).

Analysis of the obtained densities confirmed that the first complex represents an elongation complex (EC) consisting of the core vRNAP enzyme and nucleic acids in the active center cleft (Figure 1A). The density for the nucleic acid was of high quality around the DNA-RNA hybrid (Figure 1B) and was somewhat weaker for downstream DNA. Density for the single-stranded portion of the non-template DNA strand and for the upstream DNA was visible at low thresholds but did not allow for modeling. The second reconstruction contained a large additional density that could be fitted with the crystal structure of Vaccinia CE (Kyrieleis et al., 2014). Continuous RNA density stretched from the vRNAP active site to the active site of the CE TPase, showing that this reconstruction represents a co-transcriptional capping complex (CCC).

#### Structure of the vRNAP Elongation Complex
The structure of the vRNAP EC shows the active state of the enzyme. The overall structure of the eight-subunit polymerase is largely unchanged in comparison with the core vRNAP structure described in the accompanying paper (Figure 1A) (Grimm et al., 2019). However, the viral transcription factor Rap94, which was associated with both core and complete vRNAP structures, is absent from the EC structure. The active center cleft is occupied by a 9-bp DNA-RNA hybrid (Figure 1B). This is reminiscent of other multisubunit and single-subunit RNA polymerases, which all bind a hybrid of 8–9 bp (Cramer, 2002; Martinez-Rucobo and Cramer, 2013).

A

Downstream DNA

Rpo132

Rpo18

Rpo30

Rpo7

Template strand

Non-template strand

Active site

Rpo35

Rpo22

Rpo147

Front view

Top view

B

Template DNA

Metal A

Rpo132

+1 template base

Rpo147 bridge helix

M763

C

RNA

G728

R478

T754

Upstream

Downstream

Template strand

Non-template strand

Figure 1. Structure of the vRNAP Elongation Complex

(A) Cartoon representation of the vRNAP EC. Subunit coloring is as indicated and as in the accompanying manuscript (Grimm et al., 2019). Helices are shown as cylinders. Nucleic acids are shown in blue (template strand DNA), cyan (non-template strand DNA), and red (RNA). Metal ions are shown as spheres.

(B) Close-up view of the active center of vRNAP. Protein and nucleic acids are shown as sticks and colored as in (A). The cryo-EM density is shown as gray mesh. The vRNAP EC is in the post-translocated state, and the +1 template base is ready to base pair with an incoming nucleotide. Residues unique to vRNAP as discussed in the text are highlighted in green.

(C) Schematic depiction of the nucleic acid scaffold used in this study. Individual bases are shown as circles, and the bases are abbreviated as one-letter codes. Bases visible in the EC structure are shown as solid circles, and invisible bases are shown as hollow circles. The active-site metal A is shown as a pink sphere. vRNAP residues within a distance of 4 Å of the nucleic acids are indicated and colored according to their conservation in S. cerevisiae Pol II. Residues unique to vRNAP as discussed in the text are highlighted in green.

See also Figures S1, S2, and S3.

In the structure, vRNAP adopts the active, post-translocated state (Figure 1B). The binding site for the nucleoside triphosphate substrate is empty, and the +1 template base is positioned for base pairing along the bridge helix that spans the polymerase cleft. The duplex axes of downstream DNA and the hybrid enclose an angle of ~90°. Analysis of the protein-nucleic-acid interactions in the vRNAP EC confirms a high structural conservation to eukaryotic cellular RNA polymerases. The majority of residues involved in nucleic acid interactions are either identical or conserved in S. cerevisiae Pol II (Figure 1C).

There are also some notable differences in the active site of vRNAP as compared with cellular RNA polymerases. In particular, residue 754 in the bridge helix that binds to the template DNA strand in between bases at positions +1 and +2 is unique.

Whereas this residue is a threonine in vRNAP, the corresponding residue is strictly conserved as tyrosine in Pols I, II, and III (Y836 in S. cerevisiae Pol II) (Figure 1B) (Gnatt et al., 2001). In addition, the arginine residue R478 in Rpo132 is unique to vRNAP because this position is strictly conserved to glycine in cellular polymerases. The arginine side chain projects toward the terminal 3′ nucleotide of the RNA and to the binding site for the substrate nucleoside triphosphate and thus could be involved in early RNA synthesis (Figure 1B). The conformation of the trigger loop, a structural element involved in catalysis by multisubunit RNA polymerases (Martinez-Rucobo and Cramer, 2013), appears most similar to the “locked” conformation observed in the Pol II-TFIIS reactivated complex (Cheung and Cramer, 2011). Despite these differences, our results indicate that the

Figure 2. Nucleic Acids Replace the Rpo30 C-Terminal Tail

(A) Cartoon representation of vRNAP in the EC and the complete vRNAP structure (PDB: 6RFL) ([Grimm et al., 2019](https://doi.org/10.1016/j.cell.2019.10.045)). Subunit coloring is as in [Figure 1](#fig1). Helices are shown as cylinders. Proteins, except for Rpo30 in the complete vRNAP complex, are shown transparently. Nucleic acids are shown in blue (template strand DNA), cyan (non-template strand DNA), and red (RNA). The Rpo30 C-tail occupies the hybrid binding site in the complete vRNAP structure.

(B) Stick representation of the DNA-RNA hybrid in the vRNAP-EC active site with the Rpo30 C-tail from the complete vRNAP complex (PDB: 6RFL) ([Grimm et al., 2019](https://doi.org/10.1016/j.cell.2019.10.045)) overlaid transparently. Both structures were aligned with the large vRNAP subunit Rpo147.

fundamental mechanism of DNA-dependent RNA synthesis is conserved between cellular and viral multisubunit RNA polymerases.

### Release of the Rpo30 Tail from the Catalytic Center

The EC structure also suggests rearrangements that must occur during the transition from the complete vRNAP structure to the EC. The complete vRNAP structure uncovered a surprising feature of the vRNAP subunit Rpo30. This subunit has a phosphorylated C-terminal tail (C-tail) that binds the active center ([Grimm et al., 2019](https://doi.org/10.1016/j.cell.2019.10.045)). Comparison of the EC structure described here with the complete vRNAP complex demonstrates that the Rpo30 C-tail would clash with both DNA and RNA in the hybrid duplex ([Figure 2](#fig2)A). In particular, the phosphate moieties on residues S228, S232, and S237 overlap with the positions of backbone phosphate groups in the hybrid ([Figure 2](#fig2)B). Whereas the phosphorylated residue S228 occupies the phosphate-binding site of the most 3′ RNA nucleotide in the EC, the phosphorylated residues S232 and S237 bind to the phosphate positions occupied by nucleotides –3 and –7, respectively, in the template DNA strand. This suggests that the Rpo30 tail can inhibit vRNAP in a phosphorylation-dependent manner. We speculate that Rpo30 phosphorylation provides a mechanism to regulate viral gene expression during the cellular replication phase and/or during the transition from the packaged state to the actively transcribing state.

### Structure of vRNAP Co-transcriptional Capping Complex

The structure of the CCC represents the viral polymerase during co-transcriptional capping. The conformation of the polymerase is essentially identical to that observed in the EC structure. The viral CE is bound around the site where RNA exits the enzyme ([Figure 3](#fig3)A). Both subunits of CE, D1 and D12, engage in interactions with vRNAP, mainly to subunits Rpo147, Rpo132, Rpo18, and Rpo35 ([Figures 3](#fig3)A and [4](#fig4)A–4C). The DNA-RNA hybrid is observed in the active center cleft, but the structure also reveals the trajectory of the RNA beyond the hybrid ([Figure 3](#fig3)B). At the upstream edge of the hybrid, the conserved residue F208 in the lid loop of vRNAP subunit Rpo147 separates RNA from the DNA template strand. RNA density is continuous through the RNA exit tunnel of the enzyme and through a surface groove formed by CE until its 5′ end, of which four bases are seen in the active site of the TPase domain of D1 ([Figures 3](#fig3)B, 3C, and [Figure 4](#fig4)E). The RNA appears to be partially mobile and scrunched in a central region located between the end of the hybrid and the TPase active site, precluding atomic modeling in this region (see [STAR Methods](#star-methods)).

### Capping Enzyme Contains Two Mobile Modules

Superposition of polymerase-bound CE with the free CE crystal structure ([Kyrieleis et al., 2014](https://doi.org/10.1038/nsmb.2772)) shows that the individual CE domains are essentially identical. However, the relative orientation of the domains to each other differs markedly between the free

A
Vaccinia virus D1
Triphosphatase
GTase
OB Domain
Methyltransferase
1
225
395
530
844
TP/GT module
MT/D12 module
Vaccinia virus D12
1
287

TPase
D12
MT/D12
module
Template strand
Non-template
strand
vRNAP
Active site Metal
RNA
MTase
TPase
GTase
OB domain
vRNAP
Back view
Top view
B
Non-template
DNA
Downstream
Template DNA
RNA
C
RNA
Downstream
Template DNA
Metal A
Triphosphatase
active site

Figure 3. Structure of the vRNAP Co-transcriptional Capping Complex

(A) Structure of the vRNAP CCC. (Top) Schematic representation of the D1 and D12 subunits of VTF/CE. (Bottom) Cartoon and surface representation of the vRNAP CCC. vRNAP is shown as gray transparent surface, and CE is shown as cartoon and colored as indicated above. Helices are depicted as cylinders. Nucleic acids are shown in blue (template strand DNA), cyan (non-template strand DNA), and red (RNA). Metal ions are shown as spheres. Parts of the RNA that were not included in the final model are shown as transparent backbone.

(B) Cryo-EM density of the CCC. The unsharpened cryo-EM density is shown as surface and colored according to (A). The trajectory of the entire RNA can be unambiguously traced.

(C) Modeled nucleic acids in the CCC shown as stick representation. The parts of the RNA that are likely mobile and scrunched and that were not included in the final model are shown as transparent backbone. Active-site metals are shown as spheres.

See also Figures S1, Figure S2, and S3.

CE crystal structure and the CCC (Figure S4A). Therefore, the CE apparently consists of two modules that can move with respect to each other. Whereas one module contains the TPase and GTase domains of subunit D1 (“TP/GT module”), the other module consists of the MTase domain of D1 and subunit D12 (“MT/D12 module”) (Figure 3A). Relative movement of the two CE modules with respect to each other is enabled by a flexible “intermodule” linker (residues 529–560), as predicted (Kyrieleis et al., 2014). The conformation of CE observed in the CCC positions the MT/D12 module in close proximity to the polymerase. Furthermore, the region between residues 116 and 124 of D12 is located close to exiting RNA, potentially enabling further interactions with the transcript. As a result of the rearrangement of modules, the three active sites of CE are positioned in the vicinity of the exiting RNA (Figures 4D and S4B), most likely facilitating shuttling of the RNA substrate between active sites during subsequent reaction steps.

Interactions between vRNAP and Capping Enzyme

The CCC structure shows how the CE subunits D1 and D12 interact with vRNAP during co-transcriptional capping (Figures 4A–4C). The TPase domain stacks against the large subunit of vRNAP and against the stalk subunit Rpo18 (Figures 4A and 4B), which appears slightly tilted in comparison with the

Figure 4. Detailed View of vRNAP-CE Interactions and Active Sites

(A) Close-up view of the vRNAP-CE interactions around the TP/GT module in side view. Proteins and RNA are shown as cartoons and colored as in Figure 1 (vRNAP) and Figure 3 (CE). The core vRNAP is additionally shown as transparent surface. Subunits Rpo18 and Rpo19 are colored purple and light blue, respectively.

(B) Close-up view of the vRNAP-CE interaction around the TP/GT module from the opposite side as in (A). Depiction and coloring are as in (A). Parts of the RNA not included in the final model are shown as a dashed line.

(C) Close-up view of the vRNAP-CE interactions around the MT/D12 module. Depiction is as in (A). Rpo35 is colored in red, and Rpo132 is colored in sand. The Vaccinia-specific Rpo35 region that might interact with the interdomain linker is indicated. Rpo147, Rpo18, and the DNA and RNA were omitted for clarity.

(D) Sequential arrangement of CE active sites. Back view of the CCC is depicted as in Figure 3, and CEs are shown transparently. Nucleic acids are shown as sticks, and metal ions are shown as spheres. Parts of the RNA that were not included in the final model are shown as a dashed line. GTP and SAM are shown as sticks. GTP was modeled by the superimposition of the CE crystal structure (PDB: 4CKB) (Kyrieleis et al., 2014) with its TP/GT module. Active sites are labeled with numbers according to their order of action on the RNA substrate.

(E) Close-up view of the CE TPase active site. Residues lining the inside of the catalytic beta-barrel and the RNA are shown as sticks. The catalytic metal is shown as a sphere.

(F) Close-up view of the CE MTase active site. The SAM cofactor is shown as sticks, and cryo-EM density is shown as gray mesh. Residues within 4 Å of the SAM molecule are shown as sticks.

See also Figure S4.


core vRNAP structure. As observed previously in the complete vRNAP complex (Grimm et al., 2019), the C-tail of Rpo147 interacts with D1 by inserting its terminal residue F1286 into a pocket formed at the interface of the TPase and GTase domains (Figure 4A). In contrast to the extended conformation that is observed in the complete vRNAP, in the CCC the C-tail adopts a helical conformation. Further interactions are mediated by the Dock domain of vRNAP, which is sandwiched between the TPase domain and the OB-fold of D1 (Figure 4B). The latter two form a positively charged groove, along which the RNA is guided toward the TPase active site. In addition, Y409 in the OB-fold can form stacking interactions with bases of the nascent RNA. The MTase domain and subunit D12 are positioned on the opposite side of the groove, where they bind to the Wall domain in vRNAP subunit Rpo132 (Figure 4C). The MTase domain contacts region 164–171 of Rpo35, which is absent in the homologous Pol II subunit Rpb3 (Figure 4C). The MTase domain is connected to the OB-fold by a flexible linker, which was also mobile in the previously reported crystal structure of the CE (Kyrieleis et al., 2014).


The Triphosphatase Interacts with the 5′ End of RNA

The structure of the CCC also shows the interactions between nascent RNA and CE during the first step of cap formation. The RNA 5′ end is stably bound to the TPase domain of CE (Figures 3, 4D, and 4E). The TPase active site is located inside a beta-barrel structure, which features basic residues lining one side and acidic residues lining the opposite side (Figure 4E) (Kyrieleis et al., 2014). The structure confirms that the RNA enters the

Figure 5. Transitions from the Complete vRNAP Complex to the CCC

(Top) Structure of the complete vRNAP complex (PDB: 6RFL) (Grimm et al., 2019). Proteins are depicted as a schematic surface. vRNAP is colored in gray. Rap94, NPH-I, VETF, E11, and rRNA are colored in forest green, red, purple, yellow and orange, respectively. Proteins that are likely to dissociate or rearrange upon formation of the CCC are shown transparently. The Rpo147 C-tail is colored in teal and highlighted. Arrows indicate the transitions that must occur upon formation of the CCC.

(Bottom) Structure of the CCC colored as in Figure 3. The Rpo147 C-tail, which adopts a helical conformation in the CCC, is highlighted. See also Figure S5.

bound crystal structure (Kyrieleis et al., 2014). The density is in good agreement with the S-adenosyl-methionine (SAM) cofactor required for methylating the RNA substrate (Figure 4F). Because SAM, like GTP, was not added during purification and sample preparation, it most likely originated from the source cells and was stably bound during the purification procedure. Understanding the structural mechanisms underlying the second and third steps of capping would require trapping of the CCC in corresponding functional states.

### Capping Enzyme Rearrangements

In contrast to the complete vRNAP structure reported in our accompanying manuscript (Grimm et al., 2019), the CCC lacks the viral transcription factors observed in the complete vRNAP complex. Despite extensive classification efforts, particle populations containing these transcription factors could not be detected (Figure S2). In the complete vRNAP, the orientation of CE relative to the vRNAP core differs substantially, as does the relative position of the two CE modules (Figure 5 and Video S1). The GT/TP module is located on the same face of the polymerase near the Rpo18 stalk but is rotated by ~90° and swung away from vRNAP. The MT/D12 module is hinged upward, rotated, and placed away from the polymerase surface. The different arrangement of the two CE modules in the complete vRNAP structure is stabilized by the N-terminal domain (NTD) of the transcription factor Rap94. The NTD interacts with the C-terminal part of NPH-I to form the CE connector (CEC) that wedges in between the two modules. Thus, formation of the active CCC described here must involve at least partial displacement of Rap94, which allows for a rearrangement of CE and its docking to the vRNAP surface around the exiting RNA substrate (Video S1).

### Repositioning of Capping Enzyme Intermodule Linker

Comparison of the CCC structure with the complete vRNAP complex also shows a repositioning of the intermodule linker that connects the two CE modules (D1 residues 529–560).

Figure 6. Growing RNA Displaces the Rap94 B-Homology Region

(Top) Schematic depiction of Rap94 and S.cerevisiae TFIIB with domains and boundaries indicated.

(Bottom) Comparison of the active center cleft of the complete vRNAP complex (PDB: 6RFL) (Grimm et al., 2019) and the S.cerevisiae Pol II initially transcribing complex (PDB: 4BBS) (Sainsbury et al., 2013). Proteins and nucleic acids are shown as cartoon representations and are colored as indicated. vRNAP and Pol II elements are colored as in Grimm et al. (2019) and Sainsbury et al. (2013). The nucleic acid structure from the CCC was overlaid with the complete vRNAP complex by the alignment of the large subunits Rpo147 and is shown transparently. The circle indicates the region where clashes occur. The Rudder loop in the polymerase, which interacts with the B-linker and B-reader in Pol II, adopts a different conformation in vRNAP than in Pol II.

See also Figures S6 and S7.

The intermodule linker is ordered in the complete vRNAP complex, most likely by interaction with the NTD of Rap94 (Grimm et al., 2019). Residues 550–560 are located near the MTase active site, and Y555 occupies the site for the adenine base of the SAM cofactor (Figure S5A). Thus, the intermodule linker sterically interferes with binding of the SAM cofactor to the MTase in the complete vRNAP particle. In the CCC structure, however, the interdomain linker is partially displaced, potentially by interaction with a Vaccinia-specific region of Rpo35, and adopts a conformation that is now compatible with SAM binding to the MTase (Figure S5B). This position of the intermodule linker residues 545–560 corresponds to that previously observed in crystal structures (Kyrieleis et al., 2014; De la Peña et al., 2007), and the linker was previously shown to contribute to SAM binding (De la Peña et al., 2007). Therefore, the interdomain linker most likely contributes to an inactivation of CE in the complete vRNAP, and its displacement and repositioning in the CCC are required for converting the CE to a fully active conformation.

### The C-Tail of Rpo147 Is a Spring-like Tether for CE

As described above, the location and orientation of CE in the complete vRNAP complex differ from those observed in the CCC (Figure 5). In the former complex, the extensive CE-vRNAP interactions that we observe in the CCC structure are not observed. The only CE-vRNAP contact present in the complete vRNAP complex is an interaction with the Rpo147 C-tail (residues 1,259–1,286). This C-tail apparently undergoes a folding transition during the major CE rearrangements that occur during conversion of the complete vRNAP into the CCC. In particular, the C-tail adopts an extended conformation in the complete vRNAP structure (Grimm et al., 2019), whereas it adopts an alpha helical conformation in the CCC (Figure 5). This suggests that the C-tail of Rpo147 forms a flexible tether for CE such that it acts like a loaded spring that could help to pull the TP/GT module onto the polymerase surface during CCC formation (Video S1).

### Rap94 Displacement during the Initiation-Elongation Transition

Because of steric restraints, repositioning of CE is possible only after the initiation factor Rap94 is at least partially displaced from its location in the complete vRNAP complex. This raises the question as to when and how this can be triggered. As described in the accompanying paper, Rap94 contains a domain that structurally resembles the eukaryotic general transcription initiation factor TFIIB (Grimm et al., 2019). This suggests that this domain, like TFIIB, is displaced during the initiation-elongation transition. Indeed, structural comparisons between the CCC and the complete vRNAP complex indicate that the growing RNA transcript could displace Rap94 from the vRNAP surface, similar to displacement of TFIIB from Pol II upon RNA extension (Figures 6 and 7) (Kostrewa et al., 2009; Sainsbury et al., 2013).

When the RNA grows to a length of 7–8 nt, it clashes with the B-reader element of Rap94, which is smaller than the B-reader element of TFIIB (Figure S6). When the RNA grows to a length of about 12 nt, it also clashes with the B-ribbon domain of Rap94. In addition, the upstream DNA duplex in the CCC structure resides at the location occupied by the B-cyclin domain of Rap94 and also requires Rap94 displacement upon EC formation. These observations show that elongation of the RNA transcript beyond a critical length leads to clashes with the B-homology region of Rap94, which by analogy to the situation during Pol II transcription initiation are predicted to displace the Rap94 B-like domain from the vRNAP surface. This could then in turn allow repositioning of the CE to its location observed in the CCC around the RNA exit tunnel.

### Binding of Rap94 and Nucleic Acids Is Mutually Exclusive

The model for the initiation-elongation transition described above predicts that the active center of vRNAP can accommodate either the B-homology region of Rap94 or the DNA-RNA hybrid, but not both. Evidence for this comes from a further

Figure 7. Comparison of Complete vRNAP Complex and S.cerevisiae Initially Transcribing Complex

The vRNAP-Rap94 complex has a similar topology as the Pol II-TFIIB complex.

(Left) vRNAP-Rap94 complex in the complete vRNAP complex (PDB: 6RFL) (Grimm et al., 2019). All other proteins were omitted for clarity. vRNAP is colored in gray, and Rap94 is colored in green, both with shadings as in Figure 6. Domain 2 and the CTD are shown transparently. Proteins are depicted as cartoon representations with cylindrical helices.

(Right) Structure of the S. cerevisiae Pol II initially transcribing complex (PDB: 4BBS) (Sainsbury et al., 2013). Depiction is as on the left, and nucleic acids are colored as in Figure 1.

classification of our cryo-EM data for the EC (Figure S2). We could sort out a fraction of particles lacking nucleic acids, which led to a reconstruction at an overall resolution of 4.2 Å (Figures S2 and S3). This reconstruction showed density for Rap94, including the B-homology region, but lacked the DNA-RNA hybrid (Figure S7). This shows that the absence of Rap94 from our EC and CCC structures cannot be attributed to a lack of the factor from our sample. Instead, Rap94 is present in our sample and must be displaced from vRNAP when nucleic acids bind to induce a functional state of the enzyme.

DISCUSSION

Here, we provide detailed structural information on Vaccinia virus transcribing complexes in two different forms. The structure of the EC demonstrates that the nucleic acid arrangement in the active center is highly similar to that observed in cellular multisubunit RNA polymerases, indicating the same general mechanism of DNA-dependent RNA synthesis. The structure of the CCC provides the first high-resolution snapshot of co-transcriptional capping by a multisubunit RNA polymerase and reveals how the RNA substrate binds the TPase active site. Together with published functional information and the structures of vRNAP reported in our accompanying paper (Grimm et al., 2019), the results elucidate viral transcription mechanisms and suggest the nature of the rearrangements that occur during the transition from transcription initiation to elongation.

From the available data, the following model of Vaccinia virus transcription emerges. First, vRNAP engages with the promoter DNA duplex, and this is mediated by the initiation factors Rap94 and VETF in a way that remains to be understood structurally (Broyles and Li, 1993; Broyles and Moss, 1988; Broyles et al., 1991; Broyles, 2003; Hagler and Shuman, 1992b). The partial similarity of Rap94 with the Pol II initiation factor TFIIB indicates that aspects of promoter binding resemble this process in the Pol II system, where TFIIB positions the DNA above the active center cleft of the polymerase (Kostrewa et al., 2009; Plaschka et al., 2016; Sainsbury et al., 2013). The DNA is then opened, and the template strand is inserted into the active site, where it can interact with the Rap94 elements called B-reader and B-linker. During formation of the open promoter complex, the Rpo30 C-tail must liberate the active center, and this could lead to a repositioning of the B-reader.

RNA synthesis can now commence, and this leads to a displacement of Rap94 when the RNA reaches a critical length and interferes with the B-homology region of Rap94 that occupies the RNA exit tunnel. Displacement of the B-homology region of Rap94 can trigger further rearrangements of the NTD of Rap94 and associated viral factors, thereby liberating the polymerase surface that binds the CE. The CE can now dock near the RNA exit tunnel, and this involves a major rearrangement of its two mobile modules (Video S1). As a result, the three active sites of CE are aligned around the tunnel exit where the nascent RNA 5′ end emerges from the polymerase surface.

For cap formation, the RNA 5′ end must now engage with the three active sites of CE in a sequential manner. The observed conformation of CE bound to vRNAP suggests an intuitive path for sequential transfer of the RNA substrate, which remains closely associated with the transcription machinery and thus is most likely protected from degradation. The RNA 5′ end can easily swing from the first active site, the TPase, into the neighboring second active site, the GTase, which resides in the same CE module. The third active site, the MTase, faced away from the GTase active site in a previous structure of free CE (Kyrieleis et al., 2014). However, rearrangement of the CE modules in our CCC structure reorients the MTase active site toward the GTase and creates a positively charged surface that can facilitate RNA transfer. How RNA transfer is triggered remains to be explored.

CEs display a surprising variety between different taxa in terms of mechanism and genetic organization (Ghosh and Lima, 2010). However, the Vaccinia CCC might share some conceptual similarities to co-transcriptional capping by Pol II. In S. cerevisiae, the first two steps of capping are carried out by a complex of two enzymes, Cet1 and Ceg1 (Rodriguez et al., 1999; Shibagaki et al., 1992; Tsukamoto et al., 1997), which are structurally related to Vaccinia D1 (Gu et al., 2010; Kyrieleis et al., 2014). A cryo-EM reconstruction of the Pol II EC with bound Cet1-Ceg1 complex indicated that Cet1 binds to the polymerase in a location similar to that of the TP/GT module of D1 but did not reveal any details because of the low resolution (Martinez-Rucobo et al., 2015). Furthermore, there is a similarity in

how vRNAP and Pol II recruit CE to the polymerase surface. Whereas the C-tail of the largest vRNAP subunit tethers CE in the viral system, the phosphorylated CTD of the largest Pol II subunit is known to bind the CE in yeast (Cho et al., 1997). The human CE differs from that of Vaccinia and yeast, but it is likely that topological similarities will be observed in the future because capping also occurs already when the RNA emerges on the Pol II surface (Chiu et al., 2002; Coppola et al., 1983; Moteki and Price, 2002).

Comparison with the capping machineries of other viruses shows the unique strategy employed by poxviruses. Other DNA viruses that replicate in the nucleus and utilize cellular polymerase for transcription usually hijack the cellular capping machinery. In contrast, RNA viruses that replicate in the cytoplasm use a viral RNA-dependent RNA polymerase (RdRp) and either encode a capping machinery or acquire the 5′ cap from cellular mRNAs through a mechanism called cap snatching (Decroly et al., 2011; Ghosh and Lima, 2010). The RdRps of some RNA viruses, such as flaviviruses, contain a dual-function GTase-MTase, suggesting that capping occurs co-transcriptionally (Saeedi and Geiss, 2013). Rhabdoviruses, such as the vesicular stomatitis virus (VSV), encode a RdRp (L protein) that contains GTase and MTase domains. However, they use an unconventional capping mechanism that involves an enzyme-RNA adduct (Decroly et al., 2011). Although the MTase domain is related to conventional MTase domains, such as that of Vaccinia, the GTase domain has a distinct structure (Liang et al., 2015). A detailed comparison of these RdRps with our Vaccinia CCC is impossible because these polymerases are structurally unrelated and undergo different types of structural rearrangements (Liang et al., 2015).

The viral transcription cycle requires the additional transcription factors Rap94, VETF, and NPH-I (Broyles, 2003). Our structures of functional vRNAP complexes do not reveal these factors, consistent with the finding that Rap94 is displaced or strongly rearranged when transcribing complexes are formed (Grimm et al., 2019, in Cell). Although Rap94 and other transcription factors must be displaced from the vRNAP surface to form the CCC, they can remain flexibly associated with the polymerase in the infected cell. After 5′ cap synthesis, transcription elongation can proceed to the end of the gene, where termination is mediated by NPH-I and VTF (CE) (Christen et al., 1999; Hindman and Gollnick, 2016). In the future, structural insights into initiation and termination should reveal how the virus-specific factors Rap94, VETF, and NPH-I mediate these phases of the transcription cycle. Results reported here and in the accompanying paper (Grimm et al., 2019) will enable such studies and provide the molecular basis for a complete mechanistic dissection of viral RNA synthesis during Poxvirus gene expression in the cytosol.

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- KEY RESOURCES TABLE
- LEAD CONTACT AND MATERIALS AVAILABILITY
- EXPERIMENTAL MODEL AND SUBJECT DETAILS

---

### METHOD DETAILS

- Isolation of vRNAP complexes
- Preparation of vRNAP elongation complex
- Cryo-electron microscopy
- Structure determination and model building

### QUANTIFICATION AND STATISTICAL ANALYSIS

### DATA AND CODE AVAILABILITY

---

### SUPPLEMENTAL INFORMATION

Supplemental Information can be found online at [https://doi.org/10.1016/j.cell.2019.11.023](https://doi.org/10.1016/j.cell.2019.11.023).

---

### ACKNOWLEDGMENTS

We would like to thank members of the Cramer and Fischer laboratories for helpful advice and discussion. P.C. was supported by the Deutsche Forschungsgemeinschaft within research groups SFB860 and SPP1935 and under Germany’s Excellence Strategy (EXC 2067/1-390729940), the European Research Council Advanced Investigator Grant TRANSREGULON (grant agreement no. 693023), and the Volkswagen Foundation. U.F. and A.A.S. were supported by the Deutsche Forschungsgemeinschaft (Fi 573 7-2 and Fi 573 18-1).

---

### AUTHOR CONTRIBUTIONS

H.S.H. carried out all experiments and data analysis, unless stated otherwise. J.B. purified vRNAP and assembled vRNAP complexes. C.D. assisted in cryo-EM sample preparation and data collection. K.B. established vRNAP purification. A.A.S., U.F., and P.C. designed the project. H.S.H, C.G., U.F., and P.C. wrote the manuscript.

---

### DECLARATION OF INTERESTS

The authors declare no conflicts of interest. A.S. and U.F. are former salaried employees of Genelux Corporation. A.S. and U.F. are company shareholders of Genelux Corporation.

---

### RECEIVED: April 26, 2019  
REvised: August 20, 2019  
Accepted: November 14, 2019  
Published: December 12, 2019  

---

### REFERENCES

Adams, P.D., Afonine, P.V., Bunkóczi, G., Chen, V.B., Davis, I.W., Echols, N., Headd, J.J., Hung, L.-W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. 66, 213–221.

Ahn, B.Y., Gershon, P.D., Jones, E.V., and Moss, B. (1990). Identification of rpo30, a vaccinia virus RNA polymerase gene with structural similarity to a eukaryotic transcription elongation factor. Mol. Cell. Biol. 10, 5433–5441.

Ahn, B.Y., Gershon, P.D., and Moss, B. (1994). RNA polymerase-associated protein Rap94 confers promoter specificity for initiating transcription of vaccinia virus early stage genes. J. Biol. Chem. 269, 7552–7557.

Ahn, B.Y., Rosel, J., Cole, N.B., and Moss, B. (1992). Identification and expression of rpo19, a vaccinia virus gene encoding a 19-kilodalton RNA polymerase subunit. J. Virol. 66, 971–982.

Amegadzie, B.Y., Ahn, B.Y., and Moss, B. (1992). Characterization of a 7-kilodalton subunit of vaccinia virus DNA-dependent RNA polymerase with structural similarities to the smallest subunit of eukaryotic RNA polymerase II. J. Virol. 66, 3003–3010.

Amegadzie, B.Y., Holmes, M.H., Cole, N.B., Jones, E.V., Earl, P.L., and Moss, B. (1991). Identification, sequence, and expression of the gene encoding the

second-largest subunit of the vaccinia virus DNA-dependent RNA polymerase. Virology 180, 88–98.

Baldick, C.J., Jr., and Moss, B. (1993). Characterization and temporal regulation of mRNAs encoded by vaccinia virus intermediate-stage genes. J. Virol. 67, 3515–3527.

Bernecky, C., Herzog, F., Baumeister, W., Plitzko, J.M., and Cramer, P. (2016). Structure of transcribing mammalian RNA polymerase II. Nature 529, 551–554.

Bond, C.S., and Schüttelkopf, A.W. (2009). ALINE: a WYSIWYG protein-sequence alignment editor for publication-quality alignments. Acta Crystallogr. D Biol. Crystallogr. 65, 510–512.

Broyles, S.S. (2003). Vaccinia virus transcription. J. Gen. Virol. 84, 2293–2303.

Broyles, S.S., and Li, J. (1993). The small subunit of the vaccinia virus early transcription factor contacts the transcription promoter DNA. J. Virol. 67, 5677–5680.

Broyles, S.S., Li, J., and Moss, B. (1991). Promoter DNA contacts made by the vaccinia virus early transcription factor. J. Biol. Chem. 266, 15539–15544.

Broyles, S.S., and Moss, B. (1986). Homology between RNA polymerases of poxviruses, prokaryotes, and eukaryotes: nucleotide sequence and transcriptional analysis of vaccinia virus genes encoding 147-kDa and 22-kDa subunits. Proc. Natl. Acad. Sci. USA 83, 3141–3145.

Broyles, S.S., and Moss, B. (1988). DNA-dependent ATPase activity associated with vaccinia virus early transcription factor. J. Biol. Chem. 263, 10761–10765.

Broyles, S.S., Yuen, L., Shuman, S., and Moss, B. (1988). Purification of a factor required for transcription of vaccinia virus early genes. J. Biol. Chem. 263, 10754–10760.

Cassetti, M.A., and Moss, B. (1996). Interaction of the 82-kDa subunit of the vaccinia virus early transcription factor heterodimer with the promoter core sequence directs downstream DNA binding of the 70-kDa subunit. Proc. Natl. Acad. Sci. USA 93, 7540–7545.

Cheung, A.C.M., and Cramer, P. (2011). Structural basis of RNA polymerase II backtracking, arrest and reactivation. Nature 471, 249–253.

Chiu, Y.-L., Ho, C.K., Saha, N., Schwer, B., Shuman, S., and Rana, T.M. (2002). Tat Stimulates Cotranscriptional Capping of HIV mRNA. Mol. Cell 10, 585–597.

Cho, E.J., Takagi, T., Moore, C.R., and Buratowski, S. (1997). mRNA capping enzyme is recruited to the transcription complex by phosphorylation of the RNA polymerase II carboxy-terminal domain. Genes Dev. 11, 3319–3326.

Christen, L.A., Sanders, M., and Niles, E.G. (1999). Interaction of the vaccinia virus nucleoside triphosphate phosphohydrolase I with linear oligonucleotides. Biochemistry 38, 8072–8079.

Christen, L.M., Sanders, M., Wiler, C., and Niles, E.G. (1998). Vaccinia virus nucleoside triphosphate phosphohydrolase I is an essential viral early gene transcription termination factor. Virology 245, 360–371.

Chu, C., Das, K., Tyminski, J.R., Bauman, J.D., Guan, R., Qiu, W., Montelione, G.T., Arnold, E., and Shatkin, A.J. (2011). Structure of the guanylyltransferase domain of human mRNA capping enzyme. Proc. Natl. Acad. Sci. USA 108, 10104–10108.

Cong, P., and Shuman, S. (1992). Methyltransferase and subunit association domains of vaccinia virus mRNA capping enzyme. J. Biol. Chem. 267, 16424–16429.

Coppola, J.A., Field, A.S., and Luse, D.S. (1983). Promoter-proximal pausing by RNA polymerase II in vitro: transcripts shorter than 20 nucleotides are not capped. Proc. Natl. Acad. Sci. 80, 1251–1255.

Cramer, P. (2002). Common structural features of nucleic acid polymerases. BioEssays 24, 724–729.

De la Peña, M., Kyrieleis, O.J., and Cusack, S. (2007). Structural insights into the mechanism and evolution of the vaccinia virus mRNA cap N7 methyl-transferase. EMBO J. 26, 4913–4925.

Decroly, E., Ferron, F., Lescar, J., and Canard, B. (2011). Conventional and unconventional mechanisms for capping viral mRNA. Nat. Rev. Microbiol. 10, 51–65.

Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486–501.

Fabrega, C., Hausmann, S., Shen, V., Shuman, S., and Lima, C.D. (2004). Structure and mechanism of mRNA cap (guanine-N7) methyltransferase. Mol. Cell 13, 77–89.

Ghosh, A., and Lima, C.D. (2010). Enzymology of RNA cap synthesis. Wiley Interdiscip. Rev. RNA 1, 152–172.

Ghosh, A., Shuman, S., and Lima, C.D. (2011). Structural insights to how mammalian capping enzyme reads the CTD code. Mol. Cell 43, 299–310.

Gnatt, A.L., Cramer, P., Fu, J., Bushnell, D.A., and Kornberg, R.D. (2001). Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 Å resolution. Science 292, 1876–1882.

Grimm, C., Hillen, H.S., Bedenk, K., Bartuli, J., Neyer, S., Zhang, Q., Chen, N., Hüttenhofer, A., Erlacher, M., Dienemann, C., Schlosser, A., et al. (2019). Structural basis of Poxvirus transcription: Vaccinia RNA polymerase complexes. Cell, this issue, 1537–1550.

Gu, M., Rajashankar, K.R., and Lima, C.D. (2010). Structure of the Saccharomyces cerevisiae Cet1-Ceg1 mRNA capping apparatus. Structure 18, 216–227.

Hagler, J., and Shuman, S. (1992a). A freeze-frame view of eukaryotic transcription during elongation and capping of nascent mRNA. Science 255, 983–986.

Hagler, J., and Shuman, S. (1992b). Structural analysis of ternary complexes of vaccinia RNA polymerase. Proc. Natl. Acad. Sci. USA 89, 10099–10103.

Håkansson, K., Doherty, A.J., Shuman, S., and Wigley, D.B. (1997). X-ray crystallography reveals a large conformational change during guanyl transfer by mRNA capping enzymes. Cell 89, 545–553.

Hindman, R., and Gollnick, P. (2016). Nucleoside Triphosphate Phosphohydrolase I (NPH I) Functions as a 5′ to 3′ Translocase in Transcription Termination of Vaccinia Early Genes. J. Biol. Chem. 291, 14826–14838.

Hoffmann, N.A., Jakobi, A.J., Moreno-Morcillo, M., Glatt, S., Kosinski, J., Haagen, W.J.H., Sachse, C., and Müller, C.W. (2015). Molecular structures of unbound and transcribing RNA polymerase III. Nature 528, 231–236.

Kettenberger, H., Armache, K.-J., and Cramer, P. (2004). Complete RNA polymerase II elongation complex structure and its interactions with NTP and TFIIS. Mol. Cell 16, 955–965.

Knutson, B.A., and Broyles, S.S. (2008). Expansion of poxvirus RNA polymerase subunits sharing homology with corresponding subunits of RNA polymerase II. Virus Genes 36, 307–311.

Kostrewa, D., Zeller, M.E., Armache, K.-J., Seizl, M., Leike, K., Thomm, M., and Cramer, P. (2009). RNA polymerase II-TFIIB structure and mechanism of transcription initiation. Nature 462, 323–330.

Kyrieleis, O.J.P., Chang, J., de la Peña, M., Shuman, S., and Cusack, S. (2014). Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus. Structure 22, 452–465.

Liang, B., Li, Z., Jenni, S., Rahmeh, A.A., Morin, B.M., Grant, T., Grigorieff, N., Harrison, S.C., and Whelan, S.P.J. (2015). Structure of the L Protein of Vesicular Stomatitis Virus from Electron Cryomicroscopy. Cell 162, 314–327.

Lima, C.D., Wang, L.K., and Shuman, S. (1999). Structure and mechanism of yeast RNA triphosphatase: an essential component of the mRNA capping apparatus. Cell 99, 533–543.

Liu, Y., Schmidt, B., and Maskell, D.L. (2010). MSAProbs: multiple sequence alignment based on pair hidden Markov models and partition function posterior probabilities. Bioinformatics 26, 1958–1964.

Luo, Y., Mao, X., Deng, L., Cong, P., and Shuman, S. (1995). The D1 and D12 subunits are both essential for the transcription termination factor activity of vaccinia virus capping enzyme. J. Virol. 69, 3852–3856.

Mao, X., and Shuman, S. (1994). Intrinsic RNA (guanine-7) methyltransferase activity of the vaccinia virus capping enzyme D1 subunit is stimulated by the D12 subunit. Identification of amino acid residues in the D1 protein required

for subunit association and methyl group transfer. J. Biol. Chem. 269, 24472–24479.

Martin, S.A., and Moss, B. (1975). Modification of RNA by mRNA guanylyltransferase and mRNA (guanine-7-)methyltransferase from vaccinia virions. J. Biol. Chem. 250, 9330–9335.

Martinez-Rucobo, F.W., and Cramer, P. (2013). Structural basis of transcription elongation. Biochim. Biophys. Acta 1829, 9–19.

Martinez-Rucobo, F.W., Kohler, R., van de Waterbeemd, M., Heck, A.J.R., Heumann, M., Herzog, F., Stark, H., and Cramer, P. (2015). Molecular Basis of Transcription-Coupled Pre-mRNA Capping. Mol. Cell 58, 1079–1089.

Mell, L.K., Brumund, K.T., Daniels, G.A., Advani, S.J., Zakeri, K., Wright, M.E., Onyeama, S.-J., Weisman, R.A., Sanghvi, P.R., Martin, P.J., and Szalay, A.A. (2017). Phase I Trial of Intravenous Oncolytic Vaccinia Virus (GL-ONC1) with Cisplatin and Radiotherapy in Patients with Locoregionally Advanced Head and Neck Carcinoma. Clin. Cancer Res. 23, 5696–5702.

Mirzakhanyan, Y., and Gershon, P.D. (2017). Multisubunit DNA-Dependent RNA Polymerases from Vaccinia Virus and Other Nucleocytoplasmic Large-DNA Viruses: Impressions from the Age of Structure. Microbiol. Mol. Biol. Rev. 81, e00010–e00017.

Moteki, S., and Price, D. (2002). Functional coupling of capping and transcription of mRNA. Mol. Cell 10, 599–609.

Müller, M., Weigand, J.E., Weichenrieder, O., and Suess, B. (2006). Thermodynamic characterization of an engineered tetracycline-binding riboswitch. Nucleic Acids Res. 34, 2607–2617.

Neyer, S., Kunz, M., Geiss, C., Hantsche, M., Hodirnau, V.-V., Seybert, A., Engel, C., Scheffer, M.P., Cramer, P., and Frangakis, A.S. (2016). Structure of RNA polymerase I transcribing ribosomal DNA genes. Nature 540, 607–610.

Patel, D.D., and Pickup, D.J. (1989). The second-largest subunit of the poxvirus RNA polymerase is similar to the corresponding subunits of prokaryotic and eukaryotic RNA polymerases. J. Virol. 63, 1076–1086.

Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612.

Plaschka, C., Hantsche, M., Dienemann, C., Burzinski, C., Plitzko, J., and Cramer, P. (2016). Transcription initiation complex structures elucidate DNA opening. Nature 533, 353–358.

Punjani, A., Rubinstein, J.L., Fleet, D.J., and Brubaker, M.A. (2017). cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat. Methods 14, 290–296.

Rodriguez, C.R., Takagi, T., Cho, E.J., and Buratowski, S. (1999). A Saccharomyces cerevisiae RNA 5′-triphosphatase related to mRNA capping enzyme. Nucleic Acids Res. 27, 2181–2188.

Rohrmann, G., Yuen, L., and Moss, B. (1986). Transcription of vaccinia virus early genes by enzymes isolated from vaccinia virions terminates downstream of a regulatory sequence. Cell 46, 1029–1035.

Saeedi, B.J., and Geiss, B.J. (2013). Regulation of flavivirus RNA synthesis and capping. Wiley Interdiscip. Rev. RNA 4, 723–735.

Sainsbury, S., Niesser, J., and Cramer, P. (2013). Structure and function of the initially transcribing RNA polymerase II-TFIIB complex. Nature 493, 437–440.

Scheres, S.H.W. (2012). RELION: implementation of a Bayesian approach to cryo-EM structure determination. J. Struct. Biol. 180, 519–530.

Schrodinger, LLC (2015). The PyMOL Molecular Graphics System, Version 1.8.

Shibagaki, Y., Itoh, N., Yamada, H., Nagata, S., and Mizumoto, K. (1992). mRNA capping enzyme. Isolation and characterization of the gene encoding mRNA guanylyltransferase subunit from Saccharomyces cerevisiae. J. Biol. Chem. 267, 9521–9528.

Shuman, S., Broyles, S.S., and Moss, B. (1987). Purification and characterization of a transcription termination factor from vaccinia virions. J. Biol. Chem. 262, 12372–12380.

Shuman, S., and Morham, S.G. (1990). Domain structure of vaccinia virus mRNA capping enzyme. Activity of the Mr 95,000 subunit expressed in Escherichia coli. J. Biol. Chem. 265, 11967–11972.

Tegunov, D., and Cramer, P. (2019). Real-time cryo-electron microscopy data preprocessing with Warp. Nature Methods 16, 1146–1152.

Tsukamoto, T., Shibagaki, Y., Imajoh-Ohmi, S., Murakoshi, T., Suzuki, M., Nakamura, A., Gotoh, H., and Mizumoto, K. (1997). Isolation and characterization of the yeast mRNA capping enzyme beta subunit gene encoding RNA 5′-triphosphatase, which is essential for cell viability. Biochem. Biophys. Res. Commun. 239, 116–122.

Zimmermann, L., Stephens, A., Nam, S.-Z., Rau, D., Kübler, J., Lozajic, M., Gabler, F., Söding, J., Lupas, A.N., and Alva, V. (2018). A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core. J. Mol. Biol. 430, 2237–2243.

Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, 163.

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| Bacterial and Virus Strains |  |  |
| GLV-1h439 | Grimm et al., 2019 | N/A |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| HindIIIHF restriction enzyme | New England Biolabs | Cat.No.R3104S |
| T7 RNA Polymerase | Thermo Fisher Scientific | Cat.No.EP0112 |
| T4 Polynucleotide Kinase | New England Biolabs | Cat.No.M0201S |
| Critical Commercial Assays |  |  |
| QIAGEN MaxiPrep Kit | QIAGEN | Cat.No. 12163 |
| Slide-a-Lyzer Mini Dialysis Pins | Thermo Fisher Scientific | Cat.No. 69555 |
| Deposited Data |  |  |
| vRNAP EC model | This paper | PDB:6RID |
| vRNAP CCC model | This paper | PDB:6RIE |
| vRNAP EC cryo-EM reconstruction | This paper | EMDB: 4889 |
| vRNAP CCC cryo-EM reconstruction | This paper | EMDB: 4890 |
| vRNAP nucleic acid free cryo-EM reconstruction | This paper | EMDB: 4891 |
| Core vRNAP structure | Grimm et al., 2019 | PDB:6RIC |
| Complete vRNAP structure | Grimm et al., 2019 | PDB:6RFL |
| Mammalian Pol II EC | Bernecky et al., 2016 | PDB:5FLM |
| Vaccinia Capping Enzyme Crystal Structure | Kyrieleis et al., 2014 | PDB:4CKB |
| Vaccinia Capping Enzyme Methyltransferase Structure | De la Peña et al., 2007 | PDB:2VDW |
| S. cerevisiae Cet1 Crystal Structure | Lima et al., 1999 | PDB:1D8H |
| S. cerevisiae Pol II ITC | Sainsbury et al., 2013 | PDB:4BBS |
| Experimental Models: Cell Lines |  |  |
| HeLaS 3 | Sigma-Aldrich | Cat.No. 87110901 |
| Oligonucleotides |  |  |
| Template-strand DNA: | IDT DNA | N/A |
| GACTTATGATCGGATAAGAGTCCAGCCAATGACA |  |  |
| GATGCCCTCATAGCC |  |  |
| Non-template strand DNA: | IDT DNA | N/A |
| GGCTATGAGGCATCCCATGCGTTGAGGACTCTTA |  |  |
| TCCGATCATAAGTC |  |  |
| RNA: | This paper | N/A |
| GAGUUGUAAUAACAAGGGAAAUGUCAUUUGG |  |  |
| Recombinant DNA |  |  |
| pSP64-vRNA31nt-HDV | This paper | N/A |
| Software and Algorithms |  |  |
| Relion 3.0 | Scheres, 2012; Zivanov et al., 2018 | https://github.com/3dem/relion |
| Warp | Tegunov and Cramer, 2019 | http://www.warpem.com |
| Phenix 1.14 | Adams et al., 2010 | http://www.phenix-online.org |
| PyMol 1.8 | Schrodinger, LLC, 2015 | http://www.pymol.org |
| UCSF Chimera | Pettersen et al., 2004 | https://www.cgl.ucsf.edu/chimera/ |
| Coot | Emsley et al., 2010 | https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/ |
| CryoSPARC | Punjani et al., 2017 | https://www.cryosparc.com |

LEAD CONTACT AND MATERIALS AVAILABILITY

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact Patrick Cramer ([pcramer@mpibpc.mpg.de](http://pcramer@mpibpc.mpg.de)). All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.

EXPERIMENTAL MODEL AND SUBJECT DETAILS

Human HeLa S3 cells were cultured in a 37°C incubator equilibrated with 5% CO₂ and 95% humidified atmosphere. The cells were cultured in DMEM (GIBCO) supplemented with 10% FCS and 1% Penicillin/Streptomycin.

METHOD DETAILS

**Isolation of vRNAP complexes**

For purification of vRNAP from infected cells, we used the recombinant virus GLV-1 h439 containing a HA/FLAG-doubletag at the end of A24R gene, encoding vRNAP subunit Rpo132 (see also [Grimm et al., 2019](http://Grimm%20et%20al.,%202019)). HeLa S3 cells were grown in 15-cm plates up to 80%–90% of confluence and infected with GLV-1 h439 with a MOI of 1.2. Cells were pelleted 24 h later and resuspended in lysis buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, 1,5 mM MgCl₂, 0.5% [v/v] NP-40, 1 mM DTT, and complete EDTA-free protease inhibitor cocktail [Sigma-Aldrich]). For vRNAP purification, the extract was incubated for 3 h at 4°C with 200 μL anti-FLAG Agarose beads (Sigma). Beads were washed four times with buffer containing 50 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl₂, 0.1% [v/v] NP-40 and 1 mM DTT and equilibrated with elution buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl₂ and 1 mM DTT). The beads-bound proteins were eluted with 3 x FLAG peptide and analyzed by SDS-PAGE.

**Preparation of vRNAP elongation complex**

Synthetic DNA oligonucleotides (template strand: 5′-GACTTATGATCGGATAAGAGTCCAGCCAATGACAGATGCCCTCATAGCC-3′ non-template strand: 5′-GGCTATGAGGCATCCCATGCGTTGAGGACTTTATCCGATCATAAGTC-3′) were purchased from Integrated DNA Technologies. The RNA containing a 5′-triphosphate (5′-GAGUUGUAUUAAACAAGGGAAAUGUCAUUGGC-3′) was in vitro transcribed from a modified pSP64 plasmid (Promega) containing a self-cleaving Hepatitis Delta Ribozyme (HDV) fused to the 3′ end of the sequence of interest ([Müller et al., 2006](http://Müller%20et%20al.,%202006)). Following large-scale plasmid purification using a Maxi Prep kit (QIAGEN), the plasmid was linearized with Hind III (New England Biolabs) and the product was purified by phenol chloroform extraction. In vitro transcription was carried out over night at 37°C using T7 RNA polymerase (Thermo Fisher Scientific) in the supplied buffer in the presence of 100 μg linearized template DNA and 4mM of each NTP. The RNA was precipitated with isopropanol and purified by gel electrophoresis on a 10% denaturing polyacrylamide gel. RNA visualization by UV shadowing revealed two closely co-migrating bands corresponding to the expected product size after HDV-cleavage, and the major product was excised from the gel. The RNA was extracted in 0.3M Sodium acetate (pH = 5.2) and precipitated with isopropanol. Residual salt was removed using a PD-10 desalting column (GE Healthcare). The 3′-terminal 2′-3′ cyclic phosphate resulting from the HDV cleavage reaction was removed using T4 polynucleotide kinase at 37°C over night and the product RNA was further purified by phenol-chloroform extraction followed by isopropanol precipitation. The purified RNA was annealed to the template strand by mixing equimolar amounts of both in water and heating to 95°C, followed by stepwise cooling to 4°C (90 s / °C). vRNAP was purified as described above (see also [Grimm et al., 2019](http://Grimm%20et%20al.,%202019)). To form the vRNAP-nucleic acid complexes, 4 μM of template strand-RNA scaffold were added to the FLAG-eluate and the sample was incubated at room temperature for 20 min before the addition of 8.45 μM non-template strand DNA (corresponding to a scaffold:vRNAP molar ratio of approx. 60:1). The sample was then concentrated and further purified by sucrose gradient ultracentrifugation as described in the accompanying manuscript ([Grimm et al., 2019](http://Grimm%20et%20al.,%202019)). In brief, the native transcribing vRNAP complex was layered on top of a 10% - 30% sucrose gradient and centrifuged for 16 h and 35.000 rpm at 4°C in a Beckman 60Ti swing-out rotor. Gradient fractions were fractionated manually, separated by SDS-PAGE and proteins and nucleic acids were visualized by silver staining and ethidium bromide staining, respectively.

**Cryo-electron microscopy**

The fractions corresponding to the larger molecular weight species (15 + 16) were pooled and dialyzed twice against 500 mL of dialysis buffer (10 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl₂, 2 mM DTT) at 4°C using Slide-a-lyzer mini dialysis pins (20,000 MW cut-off, Thermo Fisher). The sample was diluted in an equal volume of dialysis buffer and 4 μl were applied to glow-discharged UltrAuFoil R 2/2 grids (Quantifoil) and incubated for 10 s in a Vitrobot (FEI) at 100% humidity and 4°C prior to plunge-freezing in liquid ethane. Cryo-EM data was acquired on a Titan Krios (FEI) operated at 300 kV and equipped with a Gatan energy filter and K2 direct electron detector operated in counting mode and using a slit width of 25 eV. Movie stacks consisting of 40 frames were collected with a total dose of 40.63 electrons per Å² at a nominal magnification of 130,000 x, corresponding to a pixel size of 1.05 Å / pixel.

# Structure determination and model building

Motion correction, dose-weighting and CTF estimation was done on-the-fly using Warp (Tegunov and Cramer, 2019) and automated unsupervised particle picking was performed using a custom-trained neural network in Warp. The resulting particles were subjected to unsupervised 2D classification in Relion (Scheres, 2012; Zivanov et al., 2018) followed by initial 3D refinement using a low-pass filtered *ab initio* model generated in cryoSPARC as reference (Punjani et al., 2017). All subsequent steps were performed in Relion. The aligned particles were subjected to 3D classification, which resulted in two well-defined classes which differed with regard of the presence of VTF/CE. Further 3D sub-classification of each of these classes yielded homogeneous particle populations of the CCC and the EC, respectively (Figure S2). Automated 3D refinement using a soft mask around the entire complex followed by per-particle CTF refinement and repeated 3D refinement yielded EM reconstructions at 2.9 Å for the EC and 3.1 Å for the CCC after post processing, respectively (Figure S3). Resolution estimates are according of the FSC = 0.143 gold standard criterion and the sharpening B-factors were automatically determined as implemented in the Relion post processing algorithm (Table S1). In addition to these two reconstructions, sub-classification of the EC particle population revealed a small subset of particles to which neither nucleic acid nor VTF/CE was bound. 3D refinement of this particle subset did not reach high resolution due of the low number of particles, but the resulting density allowed for docking of the known structures and revealed the nucleic acid-free core vRNAP with Rap94 bound (Figure S2 and Figure S7).

The structure of the EC was modeled by placing the previously determined core vRNAP structure (PDB: 6RFL) (Grimm et al., 2019) in the density, followed by rigid body fitting and real space adjustment in Coot (Emsley et al., 2010). The tip of the stalk subunit Rpo18 showed weak density and residues 77-163 were therefore omitted from the final model. The region encompassing residues 86-165 of Rpo147, which contains one of the Zn-binding motifs, showed scattered density and was thus modeled as in the core vRNAP structure. An initial model of the nucleic acids was obtained by superimposing the mammalian Pol II elongation complex structure (PDB: 5FLM) (Bernecky et al., 2016) and was subsequently rigid body fitted and real space adjusted in Coot. The *in vitro* transcription template used encoded a 31 nt RNA with a 10 nt complementary stretch to the template strand (see above). Based on several observations, however, we conclude that the RNA present in the elongation and capping complexes is likely only 30 nt long, lacking the 3′-most nucleotide: (1) Denaturing gel electrophoresis following *in vitro* transcription revealed two closely co-migrating bands, suggesting 1 bp-heterogeneity which could be a result of mis-cleavage by the Hepatitis Delta Virus ribozyme that was fused of the 3′ end of the RNA in the *in vitro* transcription template. However, the smaller product represented the vast majority and was thus selectively excised. (2) When incubated with synthetic RNAs representing either the 30 or 31 nt RNA, vRNAP displays backtracking activity on the 31 nt template, but not the 30nt template, suggesting that the 31 nt RNA is cleaved at the 3′ end by the enzyme (data not shown). (3) The fit of the cryo-EM density confirms the proposed 30 nt RNA from which the 3′-most nucleotide is absent. We therefore modeled the RNA as shown in Figure 1C. The quality of the RNA density rapidly declined after the point of strand separation of the RNA from the template strand, and we thus refrained from modeling further. The EC structure was real space refined using phenix.real_space_refine (Adams et al., 2010) and shows excellent stereochemistry (Table S1).

The CCC structure was modeled by first fitting the EC structure into the CCC cryo-EM reconstruction using UCSF Chimera (Pettersen et al., 2004). This revealed a large, unmodeled density around the back of vRNAP, which could be unambiguously fitted with the TP/GT module of the previously reported VTF/CE crystal structure (PDB: 4CKB) (Kyrieleis et al., 2014). The MT/D12 module had to be substantially rotated and translated to accommodate the remaining density. The structure was then rebuilt manually in real space in Coot. The quality of the cryo-EM density declined in the distal parts of the GTase domain and of D12, but was nevertheless sufficient for unambiguous placement of the crystal structure and refinement. In addition to the previously observed DNA-RNA hybrid in the active site, the cryo-EM density allowed modeling of three additional bases past the point of strand separation at the upstream edge of the transcription bubble. While the trajectory of the entire nascent transcript was clearly visible in the unsharpened cryo-EM density (Figure 3B), the quality of the B-factor sharpened (Relion) or denoised (Warp) map was not sufficient for atomic modeling in the region between RNA residues 5 and 18, indicating conformational flexibility. Despite extensive efforts, the density for this region could not be improved by focused classification and refinement procedures. The length of the unmodeled RNA region (14 nt) suggests that it may be scrunched, which could explain the lower quality density for this region due to mobility. RNA residues 1-4, which engage in interactions with the CETPase barrel, showed well-defined density and thus allowed for atomic modeling. Similar as for the RNA, density of the D12 loop 116-124 in vicinity of the nascent transcript was weak, suggesting some mobility of this loop. Based on the density and comparison to the Cet1 crystal structure (Lima et al., 1999), the 5′ end of the RNA was modeled as a diphosphate product complex, as described in the main text, by modeling a GDP covalently attached to the RNA. The CE interdomain linker showed weak density for the region 549-560, but comparison to previous crystal structures clearly indicated an identical location of this helical fragment (Kyrieleis et al., 2014; De la Peña et al., 2007) and sidechain Y555 was thus modeled as in these structures, although it lacked clear sidechain density in our EM reconstruction. Importantly, based on the location of the peptide backbone, it cannot occupy the SAM binding site as observed in the complete vRNAP complex (Grimm et al., 2019). The CCC structure was real space refined using phenix.real_space_refine (Adams et al., 2010) and shows excellent stereochemistry (Table S1).

Figures were created with PyMol (Schrodinger, LLC, 2015) and UCSF Chimera (Pettersen et al., 2004). Angular distribution plots were created using a tool distributed with Warp (Tegunov and Cramer, 2019). Local resolution maps were generated with RELION (Scheres, 2012; Zivanov et al., 2018).

QUANTIFICATION AND STATISTICAL ANALYSIS

No statistical methods were used to predetermine sample size. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

DATA AND CODE AVAILABILITY

Coordinate files for the vRNAP EC and CCC structures were deposited with the Protein Data Bank with accession codes PDB: 6RID and PDB: 6RIE, respectively. The cryo-EM density maps for the vRNAP EC and CCC as well as for the nucleic-acid free particle population were deposited with the Electron Microscopy Data Base (EMDB) with accession codes EMDB: 4889, EMDB: 4890 and EMDB: 4891, respectively.

Supplemental Figures

**A**

Purification strategy

- Infection of HeLa S3 with GLV-1h439
- Lysis of infected HeLa S3 cells
- Anti-FLAG IP Elution with FLAG peptide
- Incubation with DNA/RNA scaffold
- Sucrose density centrifugation (10-30%)
- Analysis of gradient fractions by PAGE

**B**

10%

kDa

|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 |

Rpo147  
Rpo132  
D1  
Rap94  
NPH-I  
Rpo35/D12  
Rpo30  
Rpo22  
Rpo19  
Rpo18  
E11  
Rpo7  
tRNA<sup>Gln</sup>  
DNA-RNA scaffold  

---

Figure S1. Purification of Transcribing vRNAP Complexes, Related to Figures 1 and 3

(A) Schematic representation of the purification strategy of vRNAP bound to a DNA/RNA scaffold.

(B) Representative 10%–30% sucrose density gradient of affinity-purified vRNAP complexes bound to a DNA/RNA scaffold. Proteins and nucleic acids of individual fractions were separated by SDS-PAGE and visualized by silver staining (top) and EtBr staining (bottom), respectively. The fractions 15 and 16 were pooled and used for cryo-EM analysis.

A

B

C

4,958
micrographs
Warp
particle picking
632,458
good particles
3D auto refine
Relion
CryoSPARC
ab-initio reference

Global 3D classification

Elongation complex
22.9%
144,873 particles

Capping complex
21.0%
133,038 particles

15.8 %
98,961 particles

18.7%
119,625 particles

18.3%
115,023 particles

3.3%
21,298 particles

3.7 Å

3D classification capping complex subclass

32.7%
43,538 particles

25.7%
34,168 particles

19.1%
25,465 particles

22.4%
29,867 particles

3.1 Å
Capping complex
3D auto refine with mask
Relion3 CTF correction
& Bayesian polishing

3D classification elongation complex subclass

76.4%
110,733 particles

11.8%
17,052 particles

3.2%
4,645 particles

8.6%
12,433 particles

2.9 Å
Elongation complex
3D auto refine with mask
Relion3 CTF correction
& Bayesian polishing

4.2 Å
core vRNAP
Rap94 bound
3D auto refine
Relion3 CTF correction

(legend on next page)

Figure S2. Structure Determination of vRNAP EC and CCC, Related to Figures 1 and 3

(A) Representative cryo-EM micrograph from the dataset.

(B) The most populated classes of unsupervised 2D classification in Relion.

(C) Workflow for structure determination of the EC and CCC. Unsharpened final densities are shown colored according to their subunit composition as in Figures 1 and 3.

Figure S3. Cryo-EM Structure Statistics and Information, Related to Figures 1 and 3

(A) Fourier shell correlation plots for the EC, CCC and core vRNAP structures.

(B) Comparison of cryo-EM densities of the EC, CCC and core vRNAP reconstructions determined here. Densities are shown transparently in blue (EC), red (CCC) or green (core vRNAP) with the model of the Rpo147 funnel helices and parts of Rpo30 shown as sticks.

(C) Angular distribution and local resolution of the CCC reconstruction.

(D) Angular distribution and local resolution of the EC reconstruction.

(E) Angular distribution and local resolution of the core vRNAP reconstruction.

Figure S4. Details of Capping Enzyme, Related to Figures 3 and 4

(A) Comparison of the CCC structure to the CE crystal structure (PDB: 4CKB) (Kyrieleis et al., 2014). Depiction slightly rotated from the top view shown in Figure 3. The crystal structure was aligned to the CCC structure with the TP/GT module and is shown transparently. In the CCC, the MT/D12 module adopts a different orientation relative to the TP/GT module than in the crystal structure.

(B) Back view of the CCC. Protein and nucleic acids are depicted as cartoon with cylindrical helices and colored as in Figure 3. Parts of the RNA not included in the final model are shown as transparent backbone. The CE active sites are indicated. The bound S-adenosylmethionine cofactor is shown as sticks in the MTase active site.

(C) Close-up view of the TPase active site. Coloring as in Figure 3. Residues lining the inside of the beta barrel and the RNA are shown as sticks. The active site metal is shown as sphere.

(D) Comparison to the S.cerevisiae Cet1 structure. The TPase active site in the CCC is superimposed with the Cet1 crystal structure (PDB: 1D8H) (Lima et al., 1999) and the homologous catalytic glutamate residues are shown as sticks. Cet1 is shown transparently. The sulfate ion proposed to mimic the leaving gamma-phosphate in the crystal structure is indicated.

Figure S5. Comparison of the CE Interdomain Linker in the Complete vRNAP Complex and the CCC, Related to Figure 5

(A) Structure of the CE interdomain linker (residues 529–560) in the complete vRNAP complex (PDB: 6RFL) ([Grimm et al., 2019](https://doi.org/10.1016/j.cell.2019.03.048)). Proteins are shown transparently in cartoon representation with coloring as in [Figure 3](#fig3). This linker is colored in teal and highlighted. In the complete vRNAP complex, the linker is fully ordered and shifted toward the MTase active site. Residue Y555 occupies the binding site of the SAM cofactor. The SAH cofactor bound in the CE crystal structure (PDB: 4CKB) ([Kyrieleis et al., 2014](https://doi.org/10.1074/jbc.M114.568454)) is modeled based on its location in the crystal structure and shown as transparent sticks to illustrate the overlap.

(B) Structure of the CE interdomain linker (residues 529–560) in the CCC. Depiction as in (A). The linker is only partially ordered in the CCC structure and in previous crystal structures ([Kyrieleis et al., 2014](https://doi.org/10.1074/jbc.M114.568454); [De la Peña et al., 2007](https://doi.org/10.1074/jbc.M705586200)), but the backbone density in the CCC reconstruction clearly indicates an identical trajectory as in these crystal structures. In these structures, the backbone and Y555 are positioned away from the SAM binding site to allow cofactor binding. The region 543-547 of the interdomain linker is clearly visible in the EM density and is positioned in immediate vicinity of the Vaccinia-specific part of Rpo35 (residues 147-185), and K546 of D1 may form ionic interactions with D153 or E152 in Rpo35.

Figure S6. Sequence Comparison of Rap94 and S. cerevisiae TFIIB, Related to Figure 6

Structure-based alignment of the Rap94 B-homology region and S. cerevisiae TFIIB. Residues coordinating the structural Zn ion in the B-ribbon are colored in pink. The region in the TFIIB B-reader conserved between species is indicated and is not conserved in Rap94. Invariant residues are colored in blue and conserved residues in light blue. The alignment was generated with MSAProbs (Liu et al., 2010) within the MPI Bioinformatics Toolkit (Zimmermann et al., 2018), visualized using Aline (Bond and Schüttelkopf, 2009) and manually edited by comparison to the S. cerevisiae Pol II ITC structure (PDB: 4BBS) (Sainsbury et al., 2013).

Figure S7. Rap94 Is Not Present in the EC or CCC, Related to Figures 5 and 6

(A) The unsharpened cryo-EM reconstruction of the vRNAP EC is shown as transparent blue surface with the EC model shown as cartoon and colored as in Figure 1. The binding sites of Rap94 domains in the core and complete vRNAP complexes (Grimm et al., 2019) are indicated. No density for Rap94 is observed.

(B) The unsharpened cryo-EM reconstruction of the particle population lacking nucleic acids in our dataset is shown as transparent gray surface with the vRNAP-Rap94 model from the complete vRNAP complex (PDB: 6RFL) (Grimm et al., 2019) shown as cartoon and colored as in Figure 1. Rap94 is colored in forest green. Clear density is visible for the Rap94 Domain 2, the B-homology domain and the CTD, with only the NTD lacking density.

(C) The active center cleft is occupied by nucleic acid in the EC. Close up view of the active center cleft in the EC depicted as in (A). Density corresponding to nucleic acids is shown as solid surface and colored as in Figure 3B.

(D) The Rpo30 C-tail occupies the active center cleft in the particle population lacking nucleic acids. Close up view of the active center cleft of the particle population lacking nucleic acids depicted as in (B). Density corresponding to the Rpo30 C-tail is shown as solid surface and colored in orange.
