
Review

# Conserved and Diverse Traits of Adhesion Devices from Siphoviridae Recognizing Proteinaceous or Saccharidic Receptors

**Adeline Goulet** ${}^{1,2,*}$, **Silvia Spinelli** ${}^{1,2}$, **Jennifer Mahony** ${}^{3,4}$, and **Christian Cambillau** ${}^{1,2,*}$

1. Architecture et Fonction des Macromolécules Biologiques, Aix-Marseille Université, Campus de Luminy, 13288 Marseille, France; silvia.spinelli@afmb.univ-mrs.fr  
2. Architecture et Fonction des Macromolécules Biologiques, Centre National de la Recherche Scientifique (CNRS), Campus de Luminy, 13288 Marseille, France  
3. School of Microbiology, University College Cork, Cork T12 YN60, Ireland; J.Mahony@ucc.ie  
4. APC Microbiome Ireland, University College Cork, Cork T12 YN60, Ireland  

* Correspondence: adeline.goulet@afmb.univ-mrs.fr (A.G.); ccambillau@gmail.com (C.C.)

Received: 15 April 2020; Accepted: 3 May 2020; Published: 6 May 2020

---

**Abstract:** Bacteriophages can play beneficial roles in phage therapy and destruction of food pathogens. Conversely, they play negative roles as they infect bacteria involved in fermentation, resulting in serious industrial losses. *Siphoviridae* phages possess a long non-contractile tail and use a mechanism of infection whose first step is host recognition and binding. They have evolved adhesion devices at their tails' distal end, tuned to recognize specific proteinaceous or saccharidic receptors on the host's surface that span a large spectrum of shapes. In this review, we aimed to identify common patterns beyond this apparent diversity. To this end, we analyzed siphophage tail tips or baseplates, evaluating their known structures, where available, and uncovering patterns with bioinformatics tools when they were not. It was thereby identified that a triad formed by three proteins in complex, i.e., the tape measure protein (TMP), the distal tail protein (Dit), and the tail-associated lysozyme (Tal), is conserved in all phages. This common scaffold may harbor various functional extensions internally while it also serves as a platform for plug-in ancillary or receptor-binding proteins (RBPs). Finally, a group of siphophage baseplates involved in saccharidic receptor recognition exhibits an activation mechanism reminiscent of that observed in *Myoviridae*.

**Keywords:** Bacteriophage; *Siphoviridae*; baseplate; receptor-binding protein; phage-host interactions

---

## 1. Introduction

Viruses that infect bacteria, commonly named bacteriophages or phages, play an utmost role in biogeochemistry, biomes, health (phage therapy), and industry. In this latter case, phages may cause fermentation disruptions and product inconsistencies that are among the primary causes of economic losses in the associated dairy fermentation industry. Conversely, phages can be added to foods to prevent *Salmonella* [1] or *Listeria* [2] contaminations.

The genome of phages consists of double- or single-stranded DNA or RNA. Phages exhibit a large spectrum of structural morphology: capsid-only, tailed, filamentous, etc. [3]. However, tailed phage (*Caudovirales*), phages with a tail attached to the capsid and a double-stranded DNA genome, dominate among those found in public databases, e.g., NCBI, with 58% of all phages possessing this morphology [3]. The *Caudovirales* order may be divided into five families, the three dominant families being the *Myoviridae*, phages with a contractile tail; the *Podoviridae*, phages with a short tail; and the *Siphoviridae*, phages with a long non-contractile tail (Figure 1). *Siphoviridae* are the most abundant tailed phages in the public databases (66%) followed by *Myoviridae* (20%) and

[Viruses 2020, 12, 512; doi:10.3390/v12050512](doi:10.3390/v12050512)  
[www.mdpi.com/journal/viruses](www.mdpi.com/journal/viruses)

Podoviridae (14%). In contrast, the structural depiction of phages collected in oceans worldwide reveals the dominance of tailless phages (79%), where tailed phages account only for 21% of the total. Among them, Myoviridae dominates (67%), followed by Podoviridae (28%) and Siphoviridae (5%) [3]. In the Mediterranean Sea, however, an NCBI-like distribution is observed with a dominant population of Caudovirales, in particular Siphoviridae. It should not be forgotten that the majority of the phages collected in meta-studies (oceans, seas, gut, soil) remain unclassified. Furthermore, the population of phages annotated in the public databases may not reflect their sheer diversity but the fact that they interfere, negatively or positively, with human activity. Phages that infect lactic acid bacteria (LAB phages) involved in fermentation processes are over-represented, a clear illustration of this bias [4]. Another form of bias in phage studies arose from the initial scientific interest on phages infecting Escherichia coli, including the model coliphages T4 [5,6], lambda [7], HK97 [8], and T5 [9], as many strains of this bacterium (e.g., K12) are harmless and very easy to cultivate.

Siphophages feature a long, non-contractile, and flexible tail connected to an icosahedral or prolate capsid with the latter containing its dsDNA genome (Figure 1). During the last decade, there has been an upsurge of interest in siphophages, especially those infecting Gram-positive bacteria, driven by the public safety and economic relevance of their hosts, which started to slowly dissect their infection mechanisms [10–15]. Despite this increased interest, considerable knowledge gaps remain regarding the molecular details of the mechanisms by which they bind, puncture, and hijack their hosts. Any successful phage infection starts with the recognition of the host cell surface by specialized adhesion devices. Siphophages use a multi-protein recognition device at the tail distal end, referred to as the tail spike or the baseplate, to bind either to proteinaceous receptors or to carbohydrates (and associated motifs) located at the surface of the host cell wall [16–18] (Figure 1). The composition and architecture of phage adhesion machinery can vary from one group of phages to another, tightly associated with different host recognition mechanisms. Here, we review the features of siphophage adhesion devices centered around their structural building blocks, the tape measure protein (TMP), the distal tail protein (Dit), and the tail-associated lysozyme (Tal), examining their common features and their differences. We pursue by inserting in this frame the different peripheral proteins such as the receptor-binding proteins (RBPs), the tail fibers, and other ancillary proteins.


![Diagram](attachment:diagram.png)


Figure 1. Examples of siphophages infecting Gram-positive and Gram-negative bacteria. Phages are classified between those binding to host-encoded proteins (green, left) or cell wall polysaccharides (green, right) and those with a ready to bind adhesion device (orange, left) and those with an activable adhesion device (orange right).

2. Modules Forming the Siphoviridae Tail Tip

2.1. The TMP-Dit-Tal Triad

The overall organization of the morphogenesis modules of most phages is syntenous and follows the lambda model with packaging, capsid, and tail morphogenesis functions encoded. Within the latter are the *tmp*, *dit*, and *tal* triad, and in most siphophages examined to date, the *tmp* gene is (immediately) followed by the *dit* and *tal* genes (Figure 2a). This sequence is roughly mirrored by the gene products topology in the virion tail tip (Figure 2b). The Dit hexamer interacts with the Tal trimer N-terminal domains. The TMP, which determines the virion's tail length [19], carries its main course in the lumen of the stacked Major Tail Protein (MTP) rings and Dit hexamers. Its C-terminus abuts the Tal central cavity and interacts strongly with it [15,19]. Altogether, TMP, Dit, and Tal participate in the first assembly module of the tail, the Initiator Complex (IC) [20,21]. This IC complex is then completed by the RBP and other baseplate proteins, if present, as well as with the MTPs and the tail terminator, the protein that blocks the tail extension, resulting in the complete tail that is ready to attach to the capsid connector [20,22].

![Figure 2](#)

Figure 2. The adhesion device of siphophages. (A) Schematic representation of genes coding for the adhesion device of representative phages: lambda [7], T5 [9], SPP1 [9], c2 [23], p2 [24], 1358 [25], TP901-1 [21,26], Tuc2009 [27], PSA [13], STP1 [28], 80α [15]. Tape measure proteins (TMPs) are color-coded grey; distal tail proteins (Dits) are color-coded red; tail-associated lysozymes (Tals) are color-coded dark green (Nt-structural domain), light green (Tal extension), light blue for Tal extension tip bearing a receptor-binding domain; receptor-binding proteins (RBPs) are color-coded light blue; ancillary proteins and tail fibers are color-coded dark blue. White genes are non-structural. The c2 phage Dit and Tal are shaded grey as their annotation is highly hypothetic; (B) Schematic topology of the TMP-Dit-Tal triad (same color code as in (A)); (C) Example of a TMP-Dit-Tal triad from phage 80α baseplate [15] (same color code as in (A) and (B). The TMP is dark grey); (D) 90° rotation with respect to C.

Dits of known structures are composed of two domains [15,24,29–31]. The N-terminal domains form a ring with two layers of β-strands (Figure 2c,d). This "ring domain" shows a split barrel-like fold similar to that found in phages' MTP [32] and Hcp, a Type-VI secretion system (T6SS) protein [33–35]. A long kinked extension (the “belt”) of four β-strands embraces the next Dit core in the hexameric ring. This ring delineates a 40Å-wide channel to allow the transit of the dsDNA genome during infection. Dits from phages SPP1, p2, TP901-1, T5, and 80α possess C-terminal domains located at the ring periphery, which do not contact each other [15,24,29–31]. The first four display a galectin fold supplemented in phage p2 by a long extension (the “arm”, residues 147–188) having a critical role in baseplate assembly via the formation of a three-digit hand that anchors the N-terminal domain of the RBP (ORF18, see below) [24]. In phage 80α, a shorter loop is also involved in RBP binding, and a second loop establishes contacts with the MTP. In contrast, the fold of phage T5 C-terminal domain is that of an OB domain [31].

Tals of known structures include those of siphophages p2 [24], A118 (PDB ID: 3GS9), MuSo2 (PDB ID: 3CDD), and 80α [15] (Figure 2c,d). Furthermore, their fold is similar to those of gp27 of T4 myophages [5] and of the type VI secretion system (T6SS) spike protein VgrG [34,36]. Tal of phage p2 is 398 residues-long protein and harbors four domains [24]. It is found in closed or open conformations depending upon the state of the baseplate (see Section 4). Tals from most phages, however, are longer than ~400 residues, and in some cases reaching up to 2000 residues or more [28]. HHpred [37] analysis of these Tals reveals that they possess an N-terminal domain resembling the Tal domains mentioned above, for which the structures have been determined, followed by an extension exhibiting large structural and functional diversity. In phages lambda [7], T5 [38], and SPP1 [39], this extension harbors or binds the receptor-binding domain (RBD) that attaches to the LamB, FhuA, or YueB receptors, respectively. In phages TP901-1 and Tuc2009 [40], as well as in staphylococcal phage 80α [15], this extension possesses a lysin domain (hence the original name of Tal) that cleaves the peptidoglycan upon infection. Structure prediction of phage 80α Tal C-terminus suggests that it may also possess a peptidoglycan cleavage activity [15]. In some streptococcal phages as DT1 [41] or STP1 [28], Tals carry one or several carbohydrate-binding domains, capable of binding the cell wall surface saccharidic receptors. However, many Tal extensions are not annotated functionally.

Little is known on the TMPs structure. Functionally, it has been shown by using partial deletion mutants that they control the tail length in a quasi-linear fashion [19,42]. Parts of the TMP, however, are not involved in tail length determination but are believed to interact with the tail terminator with their N-terminus and the Dit and Tal proteins with their C-terminus [19]. In some cases, they also incorporate an unfolded lysin domain that becomes active upon TMP release from the tail [43]. Structurally, they are predicted as helical, with transmembrane predicted helices (TMH) at their center [19]. In Gram- bacteria, it has been suggested that TMP interacts with proteins from the inner membrane (IM) in which its TMHs form a pore when it is ejected from the tail, thereby allowing an easier transfer of dsDNA into the cytoplasm [8]. The only experimentally determined TMP (partial) structure from phage 80α reveals that it assembles as a trimer [15], a TMP monomer per Tal monomer, and confirms that the TMP C-terminus interacts strongly with the Dit and Tal proteins, in agreement with the formation of a defined and strong initiation complex [20].

In addition to the above core architecture of Dit and Tal, a large diversity of decorating modules has been observed. Dits can harbor one to two carbohydrate-binding modules (CBMs) inserted in the loops of their peripheral galectin domains (Figure 2c). These functionalized Dits, termed "evolved Dits" [44], have been documented in *Lactobacillus* phages [44,45], lactococcal phages [46,47], streptococcal phages [28], and beyond [46]. Of note, these CBMs are often observed to share the BppA CBM fold [46]. Evolved Dits may be present in up to 30% of all phages from the lactococcal 936 group and 100% of the dairy streptococcal phages from the dominant *cos* and *pac* groups [28]. Furthermore, using fluorescence microscopy, they have been shown to bind to the same host as the bonafide RBP, although with a lower affinity [46]. Besides the large diversity of their C-terminal extensions, Tals may also be “evolved Tals” possessing CBMs, although less frequently than Dits. These evolved Tals harbor

two decorating CBMs in their gp27-like N-terminal structural domain, as exemplified in streptococcal phages of the pac group [28] and beyond [48]. Other CBM decorations have also been mentioned in MTPs from phage lambda [49], phage SPP1 [50], and lactococcal phages [47]. In the latter case, the functionality of the CBMs has been demonstrated [47].

### 2.2. RBPs and Ancillary Proteins

While the annotation of the TMP-Dit-Tal triad is relatively straightforward, annotation of the remainder of the baseplate components is more challenging. An important observation, however, is that all baseplate-encoding genes (with few exceptions) are located in the genome between the tmp and the holin/lysin genes. This includes the Dit-Tal diad, the RBP, and possibly baseplate ancillary proteins. A source of complexity is that in most phage genomes, additional genes, which may or may not be functionally annotated, are also located between these boundaries. Therefore, one has to evaluate which gene products are candidate baseplate components. This involves widely used experimental methods such as SDS-PAGE and mass spectrometry, mutational analysis [20], as well as topological and structural comparison with other phages using HHpred [37].

Atomic structures of the RBPs of a small number of siphophages are available, i.e., those from lactococcal phages p2 [51], bIL170 [52], TP901-1 [53], Tuc2009 [54], and 1358 [55,56], as well as from the Listeria phage PSA [13] and staphylococcal phages φi11 [57] and 80α [15], all assembled as homotrimers. The lactococcal and listerial phage RBPs share topological elements (Figure 3). All assemble distinct domains permitting to bind to the rest of the baseplate (N-terminus), the receptor-binding domain (RBD; C-terminus), and a linker between both. In phage p2, the RBP N-terminus "shoulder" domain possesses an immunoglobulin fold. It is followed by an interlaced trimeric β-helix (the "neck") that maintains the RBP (termed the "head") (Figure 3b) [51]. All the lactococcal and listerial RBDs share topological elements such as a core β-barrel of 7–9 β-strands, with extra loops and strands in Tuc2009 and 1358 RBPs (Figure 3a–e). Of note, these folds also share similarity with the Reovirus attachment protein s1 (PDB ID 1KKE) and the head domain of the Adenovirus fiber (PDB ID 1QHV), present as a double Greek-key motif in their topology but without any sequence similarity [58–60] as well as with the phage T4 gp12 protein [61]. Conversely, the larger RBPs from phages φ11 and 80α exhibit a more complex topology with only very localized similarity compared to the other RBPs (Figure 3f,g) [15,62].

The modules that bind the RBP to the rest of the baseplate belong to one of two different types. These domains (residues 1 to 140) are very similar in phages p2 and 1358, assembling two 4-stranded anti-parallel β-sheets and a long helix contributed by each of the three domains allowing them to associate compactly (Figure 3a,b) [51,55]. It has been proposed that the same type should be also present in phage bIL170, which is highly related to p2. Strikingly, this domain has been uncovered in the Type VI secretion system (T6SS) trimeric protein TssK that ensures the binding of the tail to the membrane complex (see Section 4) [63,64]. Conversely, in lactococcal phages TP901-1 [30], Tuc2009 [54], listerial phage PSA [13], and staphylococcal phage 80α [15], the insertion of the RBP in the baseplate involves an N-terminal three helix bundle and loops that dock in cavities provided by the rest of the baseplate (Figure 3c–g). In listerial phage PSA, the RBP results from the fusion of a RBP module and a BppU module (as those of phage TP901-1) that binds to the rest of the baseplate [13]. Finally, in many phages, the baseplate incorporates ancillary proteins in order to maintain the RBPs or to provide additional binding domains. More details on these proteins are provided in the sections below.

Figure 3. X-ray structures of RBPs from siphophages binding to host cell wall saccharidic receptors.

(A) Lactococcal phage 1358 [55]; (B) Lactococcal phage p2 [51]; (C) Lactococcal phage TP901-1 [53]; (D) Lactococcal phage Tuc2009 [54]; (E) Receptor-binding domain and stem of listerial phage PSA [13]; (F) Staphylococcal phage 80α [15]; (G) Staphylococcal phage φ11 [57]. The various domains are identified: shoulders and stem, binding to the rest of the baseplate, the neck joining the shoulders/stem to the head. The head bearing the receptor-binding crevice. The stem N-terminus, involved in baseplate integration of the RBP, has been boxed in red.

## 3. Architecture of Siphophage Adhesion Devices Targeting Proteinaceous Receptors

Phages have adapted to bind to cell surface proteins or cell wall polysaccharides on the host cell, or in some cases, both. Phages that bind proteins are widespread among Gram-negative bacteria; their outer membrane (OM) is rich in membrane-embedded proteins that phages use to hijack their host. These phages historically represent those studied in most detail such as those infecting *E. coli*: lambda, HK97, and T5. Conversely, only a few examples of phage infecting Gram-positive bacteria, including the *Bacillus subtilis* phage SPP1 and the c2-group *Lactococcus lactis* phage c2, bind to a host membrane protein [16].

### 3.1. Gram-Negative Host

#### 3.1.1. Phage lambda

The genome of phage lambda has been thoroughly analyzed, as reviewed by Casjens and Hendrix [7]. However, structural knowledge on its host adhesion device is still fragmented. The gpH gene product has been assigned the TMP (Figure 2a). Downstream of gpH, gpM HHpred analysis reports a strong hit with phage T5 Dit (Table 1), suggesting that its Dit is only composed of the ring domain, and is devoid of any extension, an OB-fold in T5 and galectin in other phages (Figure 2a). The three following ORFs, gpL, gpK, and gpl, have been reported to encode tail-associated components
(Figure 2a, Figure 4a). The gpL protein contains two domains, the C-terminal one coordinating an iron-sulfur 4Fe4S cluster [65], while the N-terminus did not return any HHpred hit. Although the function of this cluster is unknown, it may play a role in stabilizing the conformation of gpL within the tail similarly as iron ions stabilize the membrane-piercing proteins of the enterobacteria myophages P2 and phi92 [66]. In contrast, gpK is assigned a peptidase domain by HHpred (99% probability; residues 52–182 out of 199 in total). Strikingly, TMHMM analysis [67] of gpI reports the presence of two transmembrane helices (residues 68-91 and 96-118) in this 190 residues protein, and HHpred identifies a $\alpha-\beta-\alpha-\beta$ motif for its first 64 residues. The ensuing gpJ protein comprises 1132 residues. Three copies of gpJ form the central fiber at the tail tip that binds to the host protein receptor. The 149 C-terminal residues of gpJ interact with the *E. coli* receptor LamB, a membrane porin involved in maltose uptake [7]. HHpred analysis of gpJ reports hits with the N-terminus of phage Tal proteins (Table 1). However, this segment is not located at the N-terminus of gpJ but starts at residues 326-330. If we assume that it resembles the classical Tal N-terminus domain of other phages, this domain should cover gpJ residues ~185-520. Another hit reports fibronectin III repeats (Table 1) that may serve as linkers between the N-terminal structural domain and the C-terminal receptor-binding domain (Figure 4a). The two following ORFs, encoded by the side tail fiber gene (*stf*) and the tail fiber assembly gene (*tfa*), comprise the lambda long and thin side tail fibers. Of note, the version of lambda virtually used in all laboratories and referred to as the wide-type lambda, contains a frame mutation in the *stf* gene and is devoid of side tail fibers [68]. Although these side tail fibers are not essential, they can reversibly bind to another host membrane protein OmpC, thereby assisting the virions to encounter their host for infection [69]. HHpred analysis indicates that the C-terminal domains of Stf resemble the tip domains of the long tail fiber protein gp37 from the myophage T4 (95.2% probability, residues 578-774 out of 774 in total) exhibiting an $\alpha/\beta$ collar domain, an elongated six-stranded antiparallel $\beta$-strand needle domain, and the OmpC-binding domain made up of $\beta$-strands and long loops [70] (Figure 4a). Therefore, the long tail fibers of the myophage T4 and siphophage lambda, both bind the *E. coli* OmpC receptor and can be functionally substituted from one phage to the other [71], likely share a common molecular architecture. Regarding Tfa, which assists the proper assembly of the Stf trimer and is a component of mature virions [68], it likely adopts a fold similar to that of the myophage Mu Tfa protein as suggested by HHpred analysis (99.4% probability, residues 62-192 out of 194 in total), except for the first 61 residues (Figure 4a). Indeed, the N-terminal domain of the Mu Tfa protein associates with the Mu Stf receptor-binding domain that binds to lipopolysaccharides and is structurally different from the T4 and lambda tail fiber proteins [72].
Figure 4. Architecture of the phage lambda, T5, and SPP1 host adhesion devices targeting membrane proteins.

(A) **Right.** The lambda host adhesion device is formed by the Dit hexameric ring (gpM) and the long trimeric Tal / RBP (gpJ) central fiber. The positions of gpK, gpL, and gpl proteins in the lambda tail spike are not known. Ur-lambda also contains long tail fibers (LTFs) (Stf and Tfa). **Left.** Structural models, based on HHpred analyses, of gpM, gpJ middle domains (residues 326-449 and 570-811), Stf C-terminal domains (residues 578-774), and Tfa (residues 62-192). The Dit ring is a tilted view. The crystal structures used to produce these representations are PDB ID 4JMQ [31] for gpM, PDB IDs 3D37, and 5UTK for gpJ, PDB ID 2XGF for Stf [70], and PDB ID 5YVQ for Tfa [72].

(B) **Right.** The T5 host adhesion device is formed by the Dit hexameric ring (pb9), the trimeric Tal (pb3), and central fiber (pb4), and the RBP (pb5). LTFs (pb1) are anchored to the Dit ring via p132. **Left.** Ribbon representations of the pb9 (PDB ID 4JMQ) and pb1 (PDB ID 4UW8) [73] crystal structures. The Dit ring is a tilted view. Structural models, based on HHpred analyses, of pb3 (residues 528-709 and 735-849), pb4 (residues 6-319), and p132. The crystal structures used to produce these representations are PDB IDs 3D37 and 5UTK for pb3, PDB ID 5UTK for pb4, and PDB ID 3UH8 [30] for p132.

(C) **Right.** The SPP1 host adhesion device is formed by the Dit hexameric ring (gp19.1) and the long trimeric Tal / RBP (gp21). The positions of gp22, gp23, gp23.1, and gp24 proteins in the SPP1 tail spike are not known. **Left.** Ribbon representations of the gp19.1 (PDB ID 2X8K), gp22 (PDB ID 2XC8), and gp23.1 (PDB ID 2XF7) crystal structures [74,75]. Structural models, based on HHpred analyses, of the gp21 N-terminal domains (residues 3-404 and 422-518). The crystal structure used to produce these representations are PDB IDs 3GS9 and 5E53 [76].

| Query           | Target (PDB ID, Source)                          | Probability (%) | E-Value   | Identity (%) | Query Residue Range (total) | Target Residue Range (Total) |
|-----------------|------------------------------------------------|----------------|------------|--------------|----------------------------|-----------------------------|
| lambda gpM (Dit) | 6F2M, T5                                      | 94             | 0.42       | 11           | 4-77 (109)                 | 12-85 (217)                 |
|                 | 3D37, N.meningitidis MC58                      | 95.9           | 0.18       | 14           | 327-499 (1132)             | 143-320 (381)               |
|                 | 3CDD, Shewanella MuSo2                         | 95.8           | 0.18       | 10           | 326-499                    | 146-324                     |
| lambda gpJ (Tal) | 1WRU, phage Mu                                | 94.1           | 0.69       | 16           | 326-499                    | 140-318                     |
|                 | 3GS9, L.monocytogenes A118                     | 85.0           | 12         | 8            | 334-499                    | 149-307                     |
|                 | 5UTK, Receptor extracellular region (fibronectin III repeats) | 98.7          | 1.6e-6      | 11           | 570-811                    | 64-297                      |
|                 | 3D37, N.meningitidis MC58                      | 97.3           | 0.012      | 11           | 529-725 (949)              | 143-350 (381)               |
|                 | 3CDD, Shewanella MuSo2                         | 97.1           | 0.011      | 12           | 528-709                    | 146-339 (361)               |
| T5 pb3 (Tal)     | 1WRU, phage Mu                                | 96.8           | 0.026      | 15           | 529-716                    | 141-339 (379)               |
|                 | 3GS9, Listeria                                 | 91.3           | 4.3        | 15           | 533-708                    | 146-320 (342)               |
|                 | 5E53, adherence molecule (fibronectin III repeats) | 98.6          | 1.9e-5      | 13           | 690-949                    | 133-295                     |
|                 | 3GS9, L.monocytogenes A118                     | 99.9           | 4.9e-25    | 12           | 3-404 (1111)               | 1-141 (342)                 |
|                 | 3CDD, Shewanella MuSo2                         | 98.9           | 4.1e-7      | 11           | 1-382                      | 2-340                       |
| SPP1 gp21 (Tal) | 3D37, N.meningitidis MC58                      | 98.9           | 7.7e-7      | 10           | 1-381                      | 2-336                       |
|                 | 1WRU, phage Mu                                | 98.9           | 9.2e-7      | 11           | 2-381                      | 1-334                       |
|                 | 5E53, adherence molecule (fibronectin III repeats) | 96.4          | 0.011      | 14           | 422-518                    | 204-301                     |

(97.4% probability, residues 6-319 out of 685 in total) but does not return any hit for its C-terminal domain (Figure 4b). Lastly, the pb5 protein (585 residues), located at the extremity of the central tail fiber, is the monomeric receptor-binding protein that binds to the *E. coli* ferrichrome receptor FhuA. Pb5 exhibits an elongated shape in solution and does not resemble any protein of known structure [78]. Surprisingly, the *pb5* gene does not belong to the tail assembly operon in which the *pb4* gene is followed by the *p132* and *pb1* genes encoding for the Long Tail fibers (LTF, pb1) and their collar proteins (p132) (Figure 2a) [9]. The LTFs are formed by trimers of pb1, anchored to the hexameric Dit ring via p132, that recognize polymannose O-antigens at the host cell surface. The pb1 C-terminal receptor-binding domain (residues 970-1263 out of 1396 in total, the last 133 residues being the intra-molecular chaperone that is auto-proteolyzed after folding) is formed by intertwined β-sheets with a bullet-like shape, which is different from the predicted fold of the lambda LTF C-terminal domains (Figure 4b) [73]. HHpred analysis of p132 reveals that this protein possesses the TP901-1 BppU N-terminal domain fold (98.8% probability, residues 10-138 out of 140 in total) (Figure 4b). Multiple copies of BppU in the phage TP901-1 baseplate interact with the Dit ring through their N-terminal domain [30]. Therefore, the fold of proteins that anchor receptor-binding proteins to the Dit central platform appears to be conserved among siphophages. However, the site(s) of association between p132 and pb1 to attach LTFs to the T5 host recognition device remains to be determined.

### 3.2. Gram-Positive Host

#### 3.2.1. Phage SPP1

Phages that bind Gram-positive bacteria using proteins as anchors are a minority. Few proteins are available at the bacterial surface, as Gram-positive bacteria are surrounded by a thick cell wall including peptidoglycan and other surface polysaccharides [79]. *B. subtilis* phage SPP1 is the best-studied representative siphophage that binds to proteinaceous receptor. The SPP1 tail tip establishes irreversible contacts with the host receptor membrane protein YueB after an initial and reversible interaction with glycosylated teichoic acids [80]. The SPP1 TMP is represented by gp18 [81] (Figure 2a). The gp19.1 Dit protein connects the tail tube to the long tail fiber protein, gp21 (Tal). The hexameric ring of gp19.1, showing C-terminal domains protruding out of the central core, associates with the trimeric gp21 (Tal) N-terminal domain [29,82] (Figure 4c). This Dit-Tal complex, which closes the tail tube in the SPP1 virion, opens upon host recognition and binding leaving an open channel for DNA ejection [82,83]. This closed-to-open conformational switch of Tal proteins, also observed in the *L. lactis* phage p2 [24], maybe a signal that is common to Gram-positive-infecting siphophages to trigger genome ejection. HHpred analysis of gp21 indicates that its first 404 residues fold as the A118 Tal N-terminal domain and are followed by a fibronectin repeat (Table 1, Figure 4c). The atomic structure of the SPP1 Tal C-terminal domain (residues 519-1111) remains unknown, although the Phyre protein fold recognition server [84] indicates that it may adopt a α-helical fold similar to that found in the structure of Podoviridae and Myoviridae tail fiber and tail spike proteins. The cryo-electron microscopy (cryoEM) observations of SPP1 virions mixed with the ectodomains of the YueB receptor show that, after the tail tip interaction with YueB and its loss, the Tal C-terminal domain adopts heterogeneous conformations [39]. Although four proteins, gp22, gp23, gp23.1, and gp24, are encoded between the Tal and lysis proteins (Figure 2a), their role and structure are poorly documented. In particular, the presence of these proteins in the tail spike remains to be determined. The gp22 crystal structure revealed a protein fold similar to the N-terminal “shoulder” domain of the lactococcal p2 RBP [74], indicating it may be a tail spike component (Figure 4c). The crystal structure of the gp23.1 hexameric assembly was also determined (Figure 4c), but it could not provide any conclusive insights into its function [75].

3.2.2. Lactococcal Phages c2 and bIL67

Phages c2 and bIL67 belong to the c2 group of lactococcal phages that represent one of 10 groups that infect this bacterial species [85]. The c2 group contains the only lactococcal phages that are described to bind to a proteinaceous receptor, an orthologue of *B. subtilis* YueB, Pip in the case of c2, YjaE for bIL67. The annotation of the adhesion device of these two phages is scarce and confused [86]. A striking example is the lack of identification of the TMP. Several pieces of evidence suggest that protein l10 is the TMP: the *l10* gene is the longest of the whole genome; it is located in between the capsid/tail genes and the tail tip and holin genes; its product exhibits a predicted transmembrane helix; it facilitates DNA entry in the host by binding to the host membrane protein [23]. Two non-structural proteins are encoded by genes downstream of the putative TMP, the first being non-annotated (l11) and the second annotated as a terminase (l12) [86]. HHpred analysis of the following ORFs, l13-l16, encoded by genes located upstream of the holin (l17) does not provide conclusive evidence of their functions. ORF l13 is 100 amino-acids long, while ORF l14 is 638 residues long. To note, Millen and Romero showed that recombinant phages bIL67 in which the c2 ORFs l14-l15-l16 were exchanged with bIL67 ORFs 34-35-36 were able to bind Pip, the c2 receptor [23]. The sizes of l13 and l14, their position in the genome, and the fact that l14 is host-specific suggest that l14 would be the best candidate to bind to the Pip receptor and that l13-l14 may be the Dit-Tal diad (Figure 2a). Conversely, the following ORFs, l15 and l16, are annotated as carbohydrate-binding proteins that may be involved in the first, reversible, host recognition step.

4. Architecture of Siphophage Adhesion Devices Targeting Hosts' Surface Polysaccharides

These phages are readily distinguishable from those that bind to a proteinaceous receptor even based on observations of low-resolution negative staining EM (nsEM) images. Whereas phages that bind to a host protein have an elongated tail tip, phages binding to cell wall surface polysaccharides possess a bulky distal tail feature named the baseplate. This large structure encompasses a high number of RBPs (typically 18 to 54) as avidity is required to overcome the rather low RBP affinity for sugars, as compared to high protein-protein affinity observed for example in phage T5 [78]. The highlighted examples below illustrate the conserved and diverse parts of these adhesion devices.

4.1. Binding-Ready Baseplate

4.1.1. Lactococcal Phage TP901-1

The X-ray structure of a TP901-1 baseplate complex, comprising ORFs 46 (Dit), 48 (upper baseplate protein, BppU), and 49 (RBP), but without the Tal (ORF47), has been determined [30,87,88]. The TP901-1 baseplate (320 Å wide and 160 Å high, mass of 1.76 MDa) exhibits a 6-fold symmetry (Figure 5a). From the proximal to the distal end, it is formed by a Dit hexamer surrounded by 18 copies of BppU holding 18 trimers of the RBP (Figure 5a). The Dit protein possesses a structure similar to that of phages SPP1 [29] and p2 but without the arm extension of the latter [24]. The 18 BppU assemble as six asymmetric trimers connecting the Dit central core. Each monomer is composed of an N-terminal globular domain (BppU-Nt, 1-122) followed by a helical trimeric linker and a globular C-terminal domain (195-299) [30]. The C-terminal domains fold as β-sandwiches and assemble as a 3-fold symmetric triangular-shaped trimer. This structure binds to the stem domains of three RBP trimers (Figure 5a). The conservation of the residues involved in the BppU/RBP interactions suggests that common architectural themes are found among lactococcal P335 group phages [30]. In the structure, the RBPs point in the opposite direction to the capsid, favorable to interacting with the host without further conformational change (Figure 5a).

Viruses 2020, 12, 512

A
![Image A](image_a.png)
TP901-1

B
![Image B](image_b.png)
A118 / PSA

C
![Image C](image_c.png)
80α

Figure 5. Structures of binding-ready baseplates from siphophages binding to a host cell wall saccharidic receptor. (A) X-ray structure of the baseplate from phage TP901-1; the Tal trimer has been modeled by addition of phage p2 Tal to the X-ray structure. (B) Model structure of the RBP from phages PSA or A118. (C) -cryoEM structure of phage 80α baseplate [15]. The color-coding is identical to that in Figure 2: Dit: red; tal: green; RBP: pale blue; ancillary proteins dark blue; in (C) Fiber proteins are colored beige and pale green (80α).

### 4.1.2. Phage Tuc2009

EM studies have demonstrated the structural resemblance of the P335 phage Tuc2009 baseplate with that of TP901-1 [89,90]. Although the overall sequence identity between Tuc2009 and TP901-1 phage genomes is higher than 95%, the RBPs differ significantly: the stem and neck domains display high sequence identity, while the head domain displays no identity at all. Indeed, both phages target different *L. lactis* strains (UC509.9 and 3107, respectively), which produce different host cell wall polysaccharides in terms of composition and structure [17,79]. The major difference between these two baseplates is the presence of an additional protein termed BppA in the Tuc2009 baseplate, whose gene is located between *bppU* and *bppL* (Figure 2). It was shown that this protein increases the binding specificity and/or affinity of the Tuc2009 tripod to its host receptor [90]. The structure of the Tuc2009 baseplate “tripod” assembles the BppU trimeric C-terminal domains to which are attached three RBP trimers and three BppA monomers [54]. As expected, the RBP head differs strongly from that of phage TP901-1 despite sharing a core with similar fold (Figure 3c,d). BppA assembles a compact 150 residues domain (domain 1), a junction segment, and a 50 residue β-sandwich (domain 2) harboring the extension that links it to a BppU C-terminal extension. BppA domain 1 exhibits similarity with carbohydrate-binding modules (PDB IDs 1GUI and 1GU3). BppA domain 2, together with the BppU extension, is very similar to a titin domain (PDB ID 2NZI). The BppA CBM was later found to be ubiquitous in decorations of lactococcal and streptococcal phages evolved Dits or MTPs [28,47,91].

### 4.1.3. Listerial Phages A118 and PSA

The adhesion modules of listerial phages A118 and PSA have been thoroughly analyzed by informatics or experimentally [13,62]. HHpred analysis revealed that phage PSA exhibits a classical Dit, while its Tal protein possesses an extension of ~300 amino acids after the N-terminal structural domain, harboring a glycosidase domain at its extremity. By analogy with phage TP901-1, the ORF

downstream Tal is believed to form a hexamer of trimers composed of N-terminal BppU-like domains, followed by a kinked triple helix and terminated by a domain whose structure has been determined experimentally: three helical segments complex a cation and the trimeric RBP domain is a double β-barrel similar but not identical to the RBP head of phage TP901-1 (Figure 3e,c) [13]. It has been suggested that phage A118 topology [62] resembles that of Phage PSA. Its Dit is also classical, but its Tal does not possess any extension as it is 341 amino-acids long instead of 705 in PSA. The RBP following Tal is also a composite of two domains and its fold is likely very similar to that of phage PSA.

### 4.1.4. Staphylococcal Phage 80α

The recent structure of staphylococcal 80α has uncovered many interesting features [15]. Besides its classical Dit and the structural domain of Tal (followed by a small part of the extension), it reveals how the RBP stem attaches directly to the Dit ring. The structure of phage 80α RBP is almost identical to that of phage φ11 [57], despite displaying different kink angles in their stem (Figure 3f,g). Two different hexamers of lateral fiber trimers or dimers have been identified and are connected to the MTPs via rings of BppU N-terminal domains similar to those observed in TP901-1. Another exciting feature is the first structural identification of a TMP segment. A trimer of C-terminal residues 1135–1154 has been identified tightly packed inside the trimeric Tal cavity. This definitively confirms that the oligomerization of the TMP is directed by the trimeric Tal and not the hexameric Dit in the IC. Furthermore, it suggests that upon Tal opening, as observed in phages p2 [24] and SPP1 [82], the TMP release may trigger portal opening and DNA release [15].

### 4.2. Activatable Baseplates

#### 4.2.1. Lactococcus lactis Phage p2

The baseplate of phage p2 (Dit, Tal, RBP) (Figure 6), a member of the 936 group of lactococcal phages, was expressed in complex with llama nanobodies (VHHs) (Figure 6d) [24]. Its structure assembles a Dit hexamer attached to a Tal trimer, this diad being surrounded by six RBP trimers (Figure 6b–d). The Dit galectin domain exhibits a long insertion in the loop immediately after its first β-strand. This elongated stretch, termed the “arm”, abuts to a loop of triangular shape, named the “hand”. The hand is inserted between the N-terminal trimeric domains (the “shoulders”) of the RBP (see below). As the Dit is in contact with the Tal trimer, the 6 to 3 change in symmetry is possible since two of the four domains of the Tal have similar topology, and each interacts with a Dit monomer ring domain, as described by the Rossmann group for T4 gp27 interaction with the tail [5]. Whereas the Dit exhibits a large ~40 Å central channel, the Tal is closed. There are no contact points between the trimeric Tal and the RBPs and, therefore, the Dit hexamer is a central hub to which Tals and RBPs are attached (Figure 6c). Surprisingly, the expressed baseplate revealed an "inverted" conformation. One would expect the head domains of the RBPs to point “downwards”, i.e., in the direction of the host cell surface, opposite to the capsid direction, as observed in phage TP901-1. Instead, the RBPs were observed in a “heads-up” conformation, pointing toward the capsid, an orientation that is incompatible with optimal adhesion to the host. However, there was no doubt on the validity of the structure of the baseplates crystal structure as this conformation was found in the negative staining EM (nsEM) structure of the expressed baseplate as well as in the full virion nsEM structure (Figure 6a). In both cases, the X-ray structure fitted perfectly within the baseplate region of EM reconstructions [24] (Figure 6c). However, comparing the X-ray structure of the baseplate complex with that of the expressed baseplate by nsEM revealed that the VHHs binding to RBPs expelled the second ring of Dit, attached back-to-back to the first one (Figure 5c,d). Strikingly, this Dit-2 maintains the RBPs conformation in the "heads-up" conformation through a contact between the RBP heads and the Dit "hand" loop extension (Figure 5c). The resolution of the nsEM map, however, does not allow a description of the atomic details of this interaction. By the addition of Ca²⁺ or Sr²⁺ to the crystallization medium of the baseplate (in the absence of VHHs), a different baseplate conformation

was obtained in which the RBPs turn by 200° in a direction opposite to the capsid, a conformation named the “activated form” by contrast to the previous “rest form” (Figure 5e,f). The Tal trimer was also affected, resulting in its opening and the formation of a channel of ~32 Å diameter, large enough for dsDNA passage. Due to the new conformation, the RBPs and Tal establish contacts that lock the RBPs in their “heads down” conformation. This represents an activation mechanism related to that described in Myoviridae such as phage T4 [6]. This adhesion mechanism is indeed very different from that observed in phages that are "ready to bind" such as TP901-1, PSA, and many other phages. However, as we show below, this activation mechanism is widespread in siphophages, although its functional utility remains undocumented.

![Figure 6](https://i.imgur.com/yourimageurl.png)

Figure 6. Structure of the activatable baseplates from siphophage p2. (A) negative staining EM (nsEM) structure of the phage p2 virion [92]. (B) Zoom on the baseplate with two RBPs fitted in the nsEM map. (C) Slicing of the nsEM map exhibiting the baseplate (Dits, Tals, and RBPs) fitted inside. (D) nsEM structure of the expressed baseplate; note the extra volume compared to the X-ray structure obtained in the presence of VHHs [24]. (E) Representation of phage p2 with the non-activated baseplate. (F) Representation of phage p2 with the X-ray determined activated baseplate localized at the tail extremity. The red arrows show the open Tal channel and the red dots the position of the receptor binding crevices. (G,H) sliced views of the structures shown in (E)- and (F)-, respectively. A model of dsDNA helix shown in the central closed or open channel. (I,J) schematic representation of the phenomenon associated with baseplate activation.

4.2.2. Lactococcus lactis Phage 1358

The nsEM structure of the lactococcal phage 1358 virion [93] has been determined [25], as well as the X-ray structure of its RBP [55,56] (Figures 7 and 3a). The trimeric RBP assembles two domains: a “shoulders” domain very similar to that of phage p2, and a “head” domain with a core resembling that of phages p2 or TP901-1 but decorated by several extra strands and loops (Figure 3a). The RBP’s X-ray structure was fitted into the virion’s baseplate structure. Strikingly, an extension assigned to 1358’s Dit inserts within the cavity formed by the three shoulder domains, similarly as in phage p2 (Figure 7a,b). Opposite to the shoulder domains, the head domains are in contact, without the second Dit extension, but with a structure belonging to a long MTP extension (Figure 7b,c). Of note, a second minor conformation of the phage baseplate, in which the RBPs were rotated by ~180°, was observed to spontaneously occur in the virion. This suggests that the baseplate activation mechanism may be similar, in many respects, to that of phage p2 during phage 1358 adhesion.

![Figure 7](https://i.imgur.com/yourimageurl.png)

Figure 7. Structure of the activatable baseplates from siphophage 1358 [25]. (A) The nsEM structure of the non-activated baseplate viewed from the phage bottom. Tal and Dit are colored blue. The RBP is colored red. Note the "arm and hand" of Dits interacting with base of the RBPs (white arrow). (B) Lateral view of the baseplate showing the interactions of the Dit with the RBP N-terminal domain (shoulders) (1), the interaction of the MTP lateral extension with the RBP head domain (2) and lateral RBP contacts (3). (C) Sliced lateral view of the baseplate showing the non-assigned volume that can be attributed to a second Dit hexamer (white arrows). Tal and Dit are colored blue, RBP red, and MTPs green. (D) Lateral view of the nsEM maps of the baseplate in the non-activated state (green) and the activated state (red) resulting from a rotation of the RBPs attached to the Dits arms and hands.

4.2.3. Other Phages

To date, a handful of siphophage virions and RBPs structures are available. However, thanks to the predictive power of tools such as HHpred, analysis of the baseplate genes may make it possible to identify putative phages with a baseplate activation mechanism. Without presenting an exhaustive view of phages with activatable baseplates, we selected a small number of putative candidates in different bacterial species that we analyzed with HHpred. *Listeria* phage P35 gp15, gp16, and gp17 share the p2 Dit, Tal, and RBP fold with 100% probability over their full length. A similar pattern is observed with *Listeria* phage P40, but with a CBM insertion in its Dit, which is also observed in other lactococcal 936 group phages. RBPs from *Leuconostoc* phages CHA, PhiLN025, and Phil-A4 exhibit fold similarity with that of p2 RBP with ~99% probability. Phage LC3, which belongs to the P335 group of lactococcal phages (along with TP901-1 and Tuc2009), exhibits baseplate characteristics of phage p2 [94]. As mentioned above, the attachment mode of the RBPs shoulders domains to the Dits’ extensions, is shared between phages p2 and 1358. Surprisingly, TssK, a component of the type VI secretion system, possesses a shoulders domain similar to that of phage p2’s RBP [63]. The cryoEM structure of T6SS baseplate-like complex revealed that this similarity extends to the binding mode of TssK to its partner, TssG. Two triangular stretches of residues belonging to TssG serve to attach two TssK proteins to the baseplate hub [64].

5. Perspectives

Modular shuffling within phage adhesion modules has indeed been observed and documented previously [34]. It is, however, very gratifying to note that, with a rather limited number of known structures in the siphophage sphere, our analysis retrieved so many hits. This suggests that through evolution, siphophages have exchanged a small number of efficient modules and dispensed them in different architectures, resembling a LEGO-like game assembling virions with common structural bricks. Striking examples of shared modules are indeed the Dit and Tal N-termini, but also the BppU N-terminal structural domain and the BppA CBM. Conversely and importantly, it may be efficient to concentrate structural efforts on «unknown» parts of phage proteins, as they may allow a cascade of new assignments. In particular, the adhesion devices of "classical coliphages" such as lambda, T5 in addition to other model phages such as SPP1 or c2 represent viral dark matter, from a structural viewpoint, that deserves attention. Undoubtedly, with the advent of atomic resolution cryoEM, new exciting features will be revealed.

Conflicts of Interest: The authors declare no conflict of interest.

## References

1. Guenther, S.; Herzig, O.; Fieseler, L.; Klumpp, J.; Loessner, M.J. Biocontrol of *Salmonella Typhimurium* in RTE foods with the virulent bacteriophage FO1-E2. *Int. J. Food Microbiol.* **2012**, *154*, 66–72. [CrossRef] [PubMed]

2. Guenther, S.; Huwyler, D.; Richard, S.; Loessner, M.J. Virulent bacteriophage for efficient biocontrol of *Listeria monocytogenes* in ready-to-eat foods. *Appl. Environ. Microbiol.* **2009**, *75*, 93–100. [CrossRef] [PubMed]

3. Dion, M.B.; Oechslin, F.; Moineau, S. Phage diversity, genomics and phylogeny. *Nat. Rev. Microbiol.* **2020**, *18*, 125–138. [CrossRef] [PubMed]

4. Mahony, J.; Cambillau, C.; van Sinderen, D. Host recognition by lactic acid bacterial phages. *FEMS Microbiol. Rev.* **2017**, *41*, S16–S26. [CrossRef] [PubMed]

5. Kanamaru, S.; Leiman, P.G.; Kostyuchenko, V.A.; Chipman, P.R.; Mesyanzhinov, V.V.; Arisaka, F.; Rossmann, M.G. Structure of the cell-puncturing device of bacteriophage T4. *Nature* **2002**, *415*, 553–557. [CrossRef] [PubMed]

6. Taylor, N.M.; Prokhorov, N.S.; Guerrero-Ferreira, R.C.; Shneider, M.M.; Browning, C.; Goldie, K.N.; Stahlberg, H.; Leiman, P.G. Structure of the T4 baseplate and its function in triggering sheath contraction. *Nature* **2016**, *533*, 346–352. [CrossRef]

7. Casjens, S.R.; Hendrix, R.W. Bacteriophage lambda: Early pioneer and still relevant. *Virology* **2015**, *479–480*, 310–330. [CrossRef]

8. Cumby, N.; Reimer, K.; Mengin-Lecreulx, D.; Davidson, A.R.; Maxwell, K.L. The phage tail tape measure protein, an inner membrane protein and a periplasmic chaperone play connected roles in the genome injection process of *E. coli* phage HK97. *Mol. Microbiol.* **2015**, *96*, 437–447. [CrossRef]

9. Zivanovic, Y.; Confalonieri, F.; Ponchon, L.; Lurz, R.; Chami, M.; Flayhan, A.; Renouard, M.; Huet, A.; Decottignies, P.; Davidson, A.R.; et al. Insights into bacteriophage T5 structure from analysis of its morphogenesis genes and protein components. *J. Virol.* **2014**, *88*, 1162–1174. [CrossRef]

10. Mahony, J.; van Sinderen, D. Structural aspects of the interaction of dairy phages with their host bacteria. *Viruses* **2012**, *4*, 1410–1424. [CrossRef]

11. Spinelli, S.; Veesler, D.; Bebeacua, C.; Cambillau, C. Structures and host-adhesion mechanisms of lactococcal siphophages. *Front. Microbiol.* **2014**, *5*, 3. [CrossRef] [PubMed]

12. Dowah, A.S.A.; Clokie, M.R.J. Review of the nature, diversity and structure of bacteriophage receptor binding proteins that target Gram-positive bacteria. *Biophys. Rev.* **2018**, *10*, 535–542. [CrossRef] [PubMed]

13. Dunne, M.; Rupf, B.; Tala, M.; Qabrati, X.; Ernst, P.; Shen, Y.; Sumrall, E.; Heeb, L.; Pluckthun, A.; Loessner, M.J.; et al. Reprogramming Bacteriophage Host Range through Structure-Guided Design of Chimeric Receptor Binding Proteins. *Cell Rep.* **2019**, *29*, 1336–1350e4. [CrossRef] [PubMed]

14. Guerrero-Ferreira, R.C.; Hupfeld, M.; Nazarov, S.; Taylor, N.M.; Shneider, M.M.; Obbineni, J.M.; Loessner, M.J.; Ishikawa, T.; Klumpp, J.; Leiman, P.G. Structure and transformation of bacteriophage A511 baseplate and tail upon infection of *Listeria* cells. *EMBO J.* **2019**, *38*, 3. [CrossRef]

15. Kizziah, J.L.; Manning, K.A.; Dearborn, A.D.; Dokland, T. Structure of the host cell recognition and penetration machinery of a *Staphylococcus aureus* bacteriophage. *PLoS Pathog.* **2020**, 16. [CrossRef]

16. Bertozzi Silva, J.; Storms, Z.; Sauvageau, D. Host receptors for bacteriophage adsorption. *FEMS Microbiol. Lett.* **2016**, *363*, 4. [CrossRef]

17. Ainsworth, S.; Sadovskaya, I.; Vinogradov, E.; Courtin, P.; Guerardel, Y.; Mahony, J.; Grard, T.; Cambillau, C.; Chapot-Chartier, M.P.; van Sinderen, D. Differences in lactococcal cell wall polysaccharide structure are major determining factors in bacteriophage sensitivity. *MBio* **2014**, *5*, e00880-14. [CrossRef] [PubMed]

18. Sadovskaya, I.; Vinogradov, E.; Courtin, P.; Armalyte, J.; Meyrand, M.; Giaouris, E.; Palussiere, S.; Furlan, S.; Pechoux, C.; Ainsworth, S.; et al. Another Brick in the Wall: A Rhamnan Polysaccharide Trapped inside Peptidoglycan of *Lactococcus lactis*. *MBio* **2017**, *8*, 5. [CrossRef] [PubMed]

19. Mahony, J.; Alqarni, M.; Stockdale, S.; Spinelli, S.; Feyereisen, M.; Cambillau, C.; Sinderen, D.V. Functional and structural dissection of the tape measure protein of lactococcal phage TP901-1. *Sci. Rep.* **2016**, *6*, 36667. [CrossRef]

20. Vegge, C.S.; Brondsted, L.; Neve, H.; Mc Grath, S.; van Sinderen, D.; Vogensen, F.K. Structural characterization and assembly of the distal tail structure of the temperate lactococcal bacteriophage TP901-1. *J. Bacteriol.* **2005**, *187*, 4187–4197. [CrossRef]

21. Bebeacua, C.; Bron, P.; Lai, L.; Vegge, C.S.; Brondsted, L.; Spinelli, S.; Campanacci, V.; Veesler, D.; van Heel, M.; Cambillau, C. Structure and molecular assignment of lactococcal phage TP901-1 baseplate. *J. Biol. Chem.* **2010**, *285*, 39079–39086. [CrossRef] [PubMed]

22. Mahony, J.; Stockdale, S.R.; Collins, B.; Spinelli, S.; Douillard, F.P.; Cambillau, C.; van Sinderen, D. *Lactococcus lactis* phage TP901-1 as a model for Siphoviridae virion assembly. *Bacteriophage* **2016**, *6*, e1123795. [CrossRef]

23. Millen, A.M.; Romero, D.A. Genetic determinants of lactococcal C2viruses for host infection and their role in phage evolution. *J. Gen. Virol.* **2016**, *97*, 1998–2007. [CrossRef] [PubMed]

24. Sciara, G.; Bebeacua, C.; Bron, P.; Tremblay, D.; Ortiz-Lombardia, M.; Lichiere, J.; van Heel, M.; Campanacci, V.; Moineau, S.; Cambillau, C. Structure of lactococcal phage p2 baseplate and its mechanism of activation. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 6852–6857. [CrossRef] [PubMed]

25. Spinelli, S.; Bebeacua, C.; Orlov, I.; Tremblay, D.; Klaholz, B.P.; Moineau, S.; Cambillau, C. Cryo-electron microscopy structure of lactococcal siphophage 1358 virion. *J. Virol.* **2014**, *88*, 8900–8910. [CrossRef] [PubMed]

26. Brondsted, L.; Ostergaard, S.; Pedersen, M.; Hammer, K.; Vogensen, F.K. Analysis of the complete DNA sequence of the temperate bacteriophage TP901-1: Evolution, structure, and genome organization of lactococcal bacteriophages. *Virology* **2001**, *283*, 93–109. [CrossRef] [PubMed]

27. Seegers, J.F.; Mc Grath, S.; O’Connell-Motherway, M.; Arendt, E.K.; van de Guchte, M.; Creaven, M.; Fitzgerald, G.F.; van Sinderen, D. Molecular and transcriptional analysis of the temperate lactococcal bacteriophage Tuc2009. *Virology* **2004**, *329*, 40–52. [CrossRef]

28. Lavelle, K.; Goulet, A.; McDonnell, B.; Spinelli, S.; van Sinderen, D.; Mahony, J.; Cambillau, C. Revisiting the Host Adhesion Determinants of *Streptococcus thermophilus* Siphophages. *Microb. Boitech.* **2020**, in press.

29. Veesler, D.; Robin, G.; Lichiere, J.; Auzat, I.; Tavares, P.; Bron, P.; Campanacci, V.; Cambillau, C. Crystal Structure of Bacteriophage SPP1 Distal Tail Protein (gp19.1): A Baseplate Hub Paradigm In Gram-Positive Infecting Phages. *J. Biol. Chem.* **2010**, *285*, 36666–36673. [CrossRef]

30. Veesler, D.; Spinelli, S.; Mahony, J.; Lichiere, J.; Blangy, S.; Bricogne, G.; Legrand, P.; Ortiz-Lombardia, M.; Campanacci, V.; van Sinderen, D.; et al. Structure of the phage TP901-1 1.8 MDa baseplate suggests an alternative host adhesion mechanism. *Proc. Natl. Acad. Sci. USA* **2012**, *109*, 8954–8958. [CrossRef]

31. Flayhan, A.; Vellieux, F.M.; Lurz, R.; Maury, O.; Contreras-Martel, C.; Girard, E.; Boulanger, P.; Breyton, C. Crystal Structure of pb9, the Distal Tail Protein of Bacteriophage T5: A Conserved Structural Motif among All Siphophages. *J. Virol.* **2014**, *88*, 820–828. [CrossRef] [PubMed]

32. Pell, L.G.; Kanelis, V.; Donaldson, L.W.; Howell, P.L.; Davidson, A.R. The phage lambda major tail protein structure reveals a common evolution for long-tailed phages and the type VI bacterial secretion system. *Proc. Natl. Acad. Sci. USA* **2009**, *106*, 4160–4165. [CrossRef] [PubMed]

33. Jobichen, C.; Chakraborty, S.; Li, M.; Zheng, J.; Joseph, L.; Mok, Y.K.; Leung, K.Y.; Sivaraman, J. Structural basis for the secretion of EvpC: A key type VI secretion system protein from *Edwardsiella tarda*. *PLoS ONE* **2010**, *5*, e12910. [CrossRef] [PubMed]

34. Veesler, D.; Cambillau, C. A common evolutionary origin for tailed-bacteriophage functional modules and bacterial machineries. MMBR 2011, 75, 423–433, first page of table of contents. [CrossRef]

35. Mougous, J.D.; Cuff, M.E.; Raunser, S.; Shen, A.; Zhou, M.; Gifford, C.A.; Goodman, A.L.; Joachimiak, G.; Ordonez, C.L.; Lory, S.; et al. A virulence locus of *Pseudomonas aeruginosa* encodes a protein secretion apparatus. Science 2006, 312, 1526–1530. [CrossRef]

36. Leiman, P.G.; Basler, M.; Ramagopal, U.A.; Bonanno, J.B.; Sauder, J.M.; Pukatzki, S.; Burley, S.K.; Almo, S.C.; Mekalanos, J.J. Type VI secretion apparatus and phage tail-associated protein complexes share a common evolutionary origin. Proc. Natl. Acad. Sci. USA 2009, 106, 4154–4159. [CrossRef]

37. Zimmermann, L.; Stephens, A.; Nam, S.Z.; Rau, D.; Kubler, J.; Lozajic, M.; Gabler, F.; Soding, J.; Lupas, A.N.; Alva, V. A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core. J. Mol. Biol. 2018, 430, 2237–2243. [CrossRef]

38. Wang, J.; Hofnung, M.; Charbit, A. The C-terminal portion of the tail fiber protein of bacteriophage lambda is responsible for binding to LamB, its receptor at the surface of *Escherichia coli* K-12. J. Bacteriol. 2000, 182, 508–512. [CrossRef]

39. Vinga, I.; Baptista, C.; Auzat, I.; Petipas, I.; Lurz, R.; Tavares, P.; Santos, M.A.; Sao-Jose, C. Role of bacteriophage SPP1 tail spike protein gp21 on host cell receptor binding and trigger of phage DNA ejection. Mol. Microbiol. 2012, 83, 289–303. [CrossRef]

40. Stockdale, S.R.; Mahony, J.; Courtin, P.; Chapot-Chartier, M.P.; van Pijkeren, J.P.; Britton, R.A.; Neve, H.; Heller, K.J.; Aideh, B.; Vogensen, F.K.; et al. The lactococcal phages Tuc2009 and TP901-1 incorporate two alternate forms of their tail fiber into their virions for infection specialization. J. Biol. Chem. 2013, 288, 5581–5590. [CrossRef]

41. Duplessis, M.; Moineau, S. Identification of a genetic determinant responsible for host specificity in *Streptococcus thermophilus* bacteriophages. Mol. Microbiol. 2001, 41, 325–336. [CrossRef] [PubMed]

42. Pedersen, M.; Ostergaard, S.; Bresciani, J.; Vogensen, F.K. Mutational analysis of two structural genes of the temperate lactococcal bacteriophage TP901-1 involved in tail length determination and baseplate assembly. Virology 2000, 276, 315–328. [CrossRef] [PubMed]

43. Boulanger, P.; Jacquot, P.; Plancon, L.; Chami, M.; Engel, A.; Parquet, C.; Herbeuval, C.; Letellier, L. Phage T5 straight tail fiber is a multifunctional protein acting as a tape measure and carrying fusogenic and muralytic activities. J. Biol. Chem. 2008, 283, 13556–13564. [CrossRef] [PubMed]

44. Dieterle, M.E.; Fina Martin, J.; Duran, R.; Nemirovsky, S.I.; Sanchez Rivas, C.; Bowman, C.; Russell, D.; Hatfull, G.F.; Cambillau, C.; Piuri, M. Characterization of prophages containing “evolved” Dit/Tal modules in the genome of *Lactobacillus casei* BL23. Appl. Microbiol. Biotechnol. 2016, 100, 9201–9215. [CrossRef]

45. Dieterle, M.E.; Spinelli, S.; Sadovskaya, I.; Piuri, M.; Cambillau, C. Evolved distal tail carbohydrate binding modules of *Lactobacillus* phage J-1: A novel type of anti-receptor widespread among lactic acid bacteria phages. Mol. Microbiol. 2017, 104, 608–620. [CrossRef]

46. Hayes, S.; Vincentelli, R.; Mahony, J.; Nauta, A.; Ramond, L.; Lugli, G.A.; Ventura, M.; van Sinderen, D.; Cambillau, C. Functional carbohydrate binding modules identified in evolved dits from siphophages infecting various Gram-positive bacteria. Mol. Microbiol. 2018, 110, 777–795. [CrossRef]

47. Hayes, S.; Mahony, J.; Vincentelli, R.; Ramond, L.; Nauta, A.; van Sinderen, D.; Cambillau, C. Ubiquitous Carbohydrate Binding Modules Decorate 936 Lactococcal Siphophage Virions. Viruses 2019, 11, 631. [CrossRef]

48. Philippe, C.; Levesque, S.; Dion, M.; Tremblay, D.M.; Horvath, P.; Cambillau, C.; Franz, C.; Neve, H.; Fremaux, C.; Heller, K.J.; et al. Genomic characterization of a novel genus of phages infecting *Streptococcus thermophilus*. Appl. Environ. Microbiol. 2020, in press.

49. Pell, L.G.; Gasmi-Seabrook, G.M.; Morais, M.; Neudecker, P.; Kanelis, V.; Bona, D.; Donaldson, L.W.; Edwards, A.M.; Howell, P.L.; Davidson, A.R.; et al. The solution structure of the C-terminal Ig-like domain of the bacteriophage lambda tail tube protein. J. Mol. Biol. 2010, 403, 468–479. [CrossRef]

50. Auzat, I.; Droge, A.; Weise, F.; Lurz, R.; Tavares, P. Origin and function of the two major tail proteins of bacteriophage SPP1. Mol. Microbiol. 2008, 70, 557–569. [CrossRef]

51. Spinelli, S.; Desmyter, A.; Verrips, C.T.; de Haard, H.J.; Moineau, S.; Cambillau, C. Lactococcal bacteriophage p2 receptor-binding protein structure suggests a common ancestor gene with bacterial and mammalian viruses. Nat. Struct. Mol. Biol. 2006, 13, 85–89. [CrossRef]

52. Ricagno, S.; Campanacci, V.; Blangy, S.; Spinelli, S.; Tremblay, D.; Moineau, S.; Tegoni, M.; Cambillau, C. Crystal structure of the receptor-binding protein head domain from *Lactococcus lactis* phage bIL170. *J. Virol.* 2006, 80, 9331–9335. [CrossRef] [PubMed]

53. Spinelli, S.; Campanacci, V.; Blangy, S.; Moineau, S.; Tegoni, M.; Cambillau, C. Modular structure of the receptor binding proteins of *Lactococcus lactis* phages. The RBP structure of the temperate phage TP901-1. *J. Biol. Chem.* 2006, 281, 14256–14262. [CrossRef] [PubMed]

54. Legrand, P.; Collins, B.; Blangy, S.; Murphy, J.; Spinelli, S.; Gutierrez, C.; Richet, N.; Kellenberger, C.; Desmyter, A.; Mahony, J.; et al. The Atomic Structure of the Phage Tuc2009 Baseplate Tripod Suggests that Host Recognition Involves Two Different Carbohydrate Binding Modules. *MBio* 2016, 7, 1. [CrossRef] [PubMed]

55. Farenc, C.; Spinelli, S.; Vinogradov, E.; Tremblay, D.; Blangy, S.; Sadovskaya, I.; Moineau, S.; Cambillau, C. Molecular insights on the recognition of a *Lactococcus lactis* cell wall pellicle by the phage 1358 receptor binding protein. *J. Virol.* 2014, 88, 7005–7015. [CrossRef] [PubMed]

56. McCabe, O.; Spinelli, S.; Farenc, C.; Labbé, M.; Tremblay, D.; Blangy, S.; Oscarson, S.; Moineau, S.; Cambillau, C. The targeted recognition of *Lactococcus lactis* phages to their polysaccharide receptors. *Mol. Microbiol.* 2015, 96, 875–886. [CrossRef]

57. Koc, C.; Xia, G.; Kuhner, P.; Spinelli, S.; Roussel, A.; Cambillau, C.; Stehle, T. Structure of the host-recognition device of *Staphylococcus aureus* phage varphi11. *Sci Rep.* 2016, 6, 27581. [CrossRef]

58. Chappell, J.D.; Prota, A.E.; Dermody, T.S.; Stehle, T. Crystal structure of reovirus attachment protein signal reveals evolutionary relationship to adenovirus fiber. *EMBO J.* 2002, 21, 1–11. [CrossRef]

59. Burmeister, W.P.; Guilligay, D.; Cusack, S.; Wadell, G.; Arnberg, N. Crystal structure of species D adenovirus fiber knobs and their sialic acid binding sites. *J. Virol.* 2004, 78, 7727–7736. [CrossRef]

60. Van Raaij, M.J.; Mitraki, A.; Lavigne, G.; Cusack, S. A triple beta-spiral in the adenovirus fibre shaft reveals a new structural motif for a fibrous protein. *Nature* 1999, 401, 935–938. [CrossRef]

61. Van Raaij, M.J.; Schoehn, G.; Burda, M.R.; Miller, S. Crystal structure of a heat and protease-stable part of the bacteriophage T4 short tail fibre. *J. Mol. Biol.* 2001, 314, 1137–1146. [CrossRef]

62. Cambillau, C. Bacteriophage module reshuffling results in adaptive host range as exemplified by the baseplate model of listerial phage A118. *Virology* 2015, 484, 86–92. [CrossRef] [PubMed]

63. Nguyen, V.S.; Logger, L.; Spinelli, S.; Legrand, P.; Huyen Pham, T.T.; Nhung Trinh, T.T.; Cherrak, Y.; Zoued, A.; Desmyter, A.; Durand, E.; et al. Type VI secretion TssK baseplate protein exhibits structural similarity with phage receptor-binding proteins and evolved to bind the membrane complex. *Nat. Microbiol.* 2017, 2, 17103. [CrossRef]

64. Park, Y.J.; Lacourse, K.D.; Cambillau, C.; DiMaio, F.; Mougous, J.D.; Veesler, D. Structure of the type VI secretion system TssK-TssF-TssG baseplate subcomplex revealed by cryo-electron microscopy. *Nat. Commun.* 2018, 9, 5385. [CrossRef] [PubMed]

65. Tam, W.; Pell, L.G.; Bona, D.; Tsai, A.; Dai, X.X.; Edwards, A.M.; Hendrix, R.W.; Maxwell, K.L.; Davidson, A.R. Tail Tip Proteins Related to Bacteriophage lambda gpL Coordinate an Iron-Sulfur Cluster. *J. Mol. Biol.* 2013, 425, 2450–2462. [CrossRef]

66. Browning, C.; Shneider, M.M.; Bowman, V.D.; Schwarzer, D.; Leiman, P.G. Phage pierces the host cell membrane with the iron-loaded spike. *Structure* 2012, 20, 326–339. [CrossRef] [PubMed]

67. Krogh, A.; Larsson, B.; von Heijne, G.; Sonnhammer, E.L. Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. *J. Mol. Biol.* 2001, 305, 567–580. [CrossRef]

68. Hendrix, R.W.; Duda, R.L. Bacteriophage lambda PaPa: Not the mother of all lambda phages. *Science* 1992, 258, 1145–1148. [CrossRef]

69. Guan, J.; Ibarra, D.; Zeng, L. The role of side tail fibers during the infection cycle of phage lambda. *Virology* 2019, 527, 57–63. [CrossRef]

70. Bartual, S.G.; Otero, J.M.; Garcia-Doval, C.; Llamas-Saiz, A.L.; Kahn, R.; Fox, G.C.; van Raaij, M.J. Structure of the bacteriophage T4 long tail fiber receptor-binding tip. *Proc. Natl. Acad. Sci. USA* 2010, 107, 20287–20292. [CrossRef]

71. Montag, D.; Schwarz, H.; Henning, U. A component of the side tail fiber of *Escherichia coli* bacteriophage lambda can functionally replace the receptor-recognizing part of a long tail fiber protein of the unrelated bacteriophage T4. *J. Bacteriol.* 1989, 171, 4378–4384. [CrossRef] [PubMed]

72. North, O.I.; Sakai, K.; Yamashita, E.; Nakagawa, A.; Iwazaki, T.; Buttner, C.R.; Takeda, S.; Davidson, A.R. Phage tail fibre assembly proteins employ a modular structure to drive the correct folding of diverse fibres. *Nat. Microbiol.* **2019**, *4*, 1645–1653. [CrossRef](https://doi.org/10.1038/s41564-019-0456-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/31089320)

73. Garcia-Doval, C.; Caston, J.R.; Luque, D.; Granell, M.; Otero, J.M.; Llamas-Saiz, A.L.; Renouard, M.; Boulanger, P.; van Raaij, M.J. Structure of the Receptor-Binding Carboxy-Terminal Domain of the Bacteriophage T5 L-Shaped Tail Fibre with and without Its Intra-Molecular Chaperone. *Viruses* **2015**, *7*, 6424–6440. [CrossRef](https://doi.org/10.3390/v7112924) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26580636)

74. Veesler, D.; Blangy, S.; Spinelli, S.; Tavares, P.; Campanacci, V.; Cambillau, C. Crystal structure of *Bacillus subtilis* SPP1 phage gp22 shares fold similarity with a domain of Lactococcal phage p2 RBP. *Protein Sci.* **2010**, *19*, 1439–1443. [CrossRef](https://doi.org/10.1002/pro.396) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20373500)

75. Veesler, D.; Blangy, S.; Lichiere, J.; Ortiz-Lombarda, M.; Tavares, P.; Campanacci, V.; Cambillau, C. Crystal structure of *Bacillus subtilis* SPP1 phage gp23.1, a putative chaperone. *Protein Sci.* **2010**, *19*, 1812–1816. [CrossRef](https://doi.org/10.1002/pro.400)

76. Nikolaienko, R.M.; Hammel, M.; Dubreuil, V.; Zalmai, R.; Hall, D.R.; Mehzabeen, N.; Karuppan, S.J.; Harroch, S.; Stella, S.L.; Bouyain, S. Structural Basis for Interactions Between Contactin Family Members and Protein-tyrosine Phosphatase Receptor Type G in Neural Tissues. *J. Biol. Chem.* **2016**, *291*, 21335–21349. [CrossRef](https://doi.org/10.1074/jbc.M116.727499)

77. Davidson, A.R.; Cardarelli, L.; Pell, L.G.; Radford, D.R.; Maxwell, K.L. Long noncontractile tail machines of bacteriophages. *Adv. Exp. Biol.* **2012**, *726*, 115–142.

78. Breyton, C.; Flayhan, A.; Gabel, F.; Lethier, M.; Durand, G.; Boulanger, P.; Chami, M.; Ebel, C. Assessing the conformational changes of pb5, the receptor-binding protein of phage T5, upon binding to its *Escherichia coli* receptor FhuA. *J. Biol. Chem.* **2013**, *288*, 30763–30772. [CrossRef](https://doi.org/10.1074/jbc.M113.479709)

79. Chapot-Chartier, M.P.; Vinogradov, E.; Sadovskaya, I.; Andre, G.; Mistou, M.Y.; Trieu-Cuot, P.; Furlan, S.; Bidnenko, E.; Courtin, P.; Pechoux, C.; et al. Cell surface of *Lactococcus lactis* is covered by a protective polysaccharide pellicle. *J. Biol. Chem.* **2010**, *285*, 10464–10471. [CrossRef](https://doi.org/10.1074/jbc.M109.083507)

80. Baptista, C.; Santos, M.A.; Sao-Jose, C. Phage SPP1 reversible adsorption to *Bacillus subtilis* cell wall teichoic acids accelerates virus recognition of membrane receptor YueB. *J. Bacteriol.* **2008**, *190*, 4989–4996. [CrossRef](https://doi.org/10.1128/JB.00303-08)

81. Godinho, L.M.; El Sadek Fadel, M.; Monniot, C.; Jakutyte, L.; Auzat, I.; Labarde, A.; Djacem, K.; Oliveira, L.; Carballido-Lopez, R.; Ayora, S.; et al. The Revisited Genome of *Bacillus subtilis* Bacteriophage SPP1. *Viruses* **2018**, *10*, 705. [CrossRef](https://doi.org/10.3390/v10110705)

82. Goulet, A.; Lai-Kee-Him, J.; Veesler, D.; Auzat, I.; Robin, G.; Shepherd, D.A.; Ashcroft, A.E.; Richard, E.; Lichiere, J.; Tavares, P.; et al. The opening of the SPP1 bacteriophage tail, a prevalent mechanism in Gram-positive-infecting siphophages. *J. Biol. Chem.* **2011**, *286*, 25397–25405. [CrossRef](https://doi.org/10.1074/jbc.M111.246839)

83. Plisson, C.; White, H.E.; Auzat, I.; Zafarani, A.; Sao-Jose, C.; Lhuillier, S.; Tavares, P.; Orlova, E.V. Structure of bacteriophage SPP1 tail reveals trigger for DNA ejection. *EMBO J.* **2007**, *26*, 3720–3728. [CrossRef](https://doi.org/10.1038/sj.emboj.7601738) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17551507)

84. Kelley, L.A.; Sternberg, M.J. Protein structure prediction on the Web: A case study using the Phyre server. *Nature Protocols* **2009**, *4*, 363–371. [CrossRef](https://doi.org/10.1038/nprot.2009.2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19247286)

85. Deveau, H.; Labrie, S.J.; Chopin, M.C.; Moineau, S. Biodiversity and classification of lactococcal phages. *Appl Environ. Microbiol.* **2006**, *72*, 4338–4346. [CrossRef](https://doi.org/10.1128/AEM.00011-06) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16672475)

86. Lubbers, M.W.; Waterfield, N.R.; Beresford, T.P.; Le Page, R.W.; Jarvis, A.W. Sequencing and analysis of the prolate-headed Lactococcal bacteriophage c2 genome and identification of the structural genes. *Appl. Environ. Microbiol.* **1995**, *61*, 4348–4356. [CrossRef](https://doi.org/10.1128/AEM.61.11.4348-4356.1995) [PubMed](https://pubmed.ncbi.nlm.nih.gov/7488300)

87. Campanacci, V.; Veesler, D.; Lichiere, J.; Blangy, S.; Sciara, G.; Moineau, S.; van Sinderen, D.; Bron, P.; Cambillau, C. Solution and electron microscopy characterization of Lactococcal phage baseplates expressed in *Escherichia coli*. *J. Struct. Biol.* **2010**, *172*, 75–84. [CrossRef](https://doi.org/10.1016/j.jsb.2010.06.005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20561521)

88. Shepherd, D.A.; Veesler, D.; Lichiere, J.; Ashcroft, A.E.; Cambillau, C. Unraveling Lactococcal phage baseplate assembly by mass spectrometry. *Mol. Cell Proteomics* **2011**, *10*, M111-009787. [CrossRef](https://doi.org/10.1074/mcp.M111.009787)

89. Sciara, G.; Blangy, S.; Siponen, M.; Mc Grath, S.; van Sinderen, D.; Tegoni, M.; Cambillau, C.; Campanacci, V. A topological model of the baseplate of lactococcal phage Tuc2009. *J. Biol. Chem.* **2008**, *283*, 2716–2723. [CrossRef](https://doi.org/10.1074/jbc.M707881200)

90. Collins, B.; Bebeacua, C.; Mahony, J.; Blangy, S.; Douillard, F.P.; Veesler, D.; Cambillau, C.; van Sinderen, D. Structure and functional analysis of the host recognition device of Lactococcal phage tuc2009. *J. Virol.* **2013**, *87*, 8429–8440. [CrossRef](https://doi.org/10.1128/JVI.00278-13)

91. Hayes, S.; Duhoo, Y.; Neve, H.; Murphy, J.; Noben, J.P.; Franz, C.; Cambillau, C.; Mahony, J.; Nauta, A.; van Sinderen, D. Identification of Dual Receptor Binding Protein Systems in Lactococcal 936 Group Phages. *Viruses* **2018**, *10*, 668. [CrossRef](https://doi.org/10.3390/v10070668) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

92. Bebeacua, C.; Tremblay, D.; Farenc, C.; Chapot-Chartier, M.P.; Sadovskaya, I.; van Heel, M.; Veesler, D.; Moineau, S.; Cambillau, C. Structure, adsorption to host, and infection mechanism of virulent Lactococcal phage p2. *J. Virol.* **2013**, *87*, 12302–12312. [CrossRef](https://doi.org/10.1128/JVI.02547-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

93. Dupuis, M.E.; Moineau, S. Genome organization and characterization of the virulent Lactococcal phage 1358 and its similarities to Listeria phages. *Appl Environ. Microbiol.* **2010**, *76*, 1623–1632. [CrossRef](https://doi.org/10.1128/AEM.02366-09)

94. Mahony, J.; Oliveira, J.; Collins, B.; Hanemaaijer, L.; Lugli, G.A.; Neve, H.; Ventura, M.; Kouwen, T.R.; Cambillau, C.; van Sinderen, D. Genetic and functional characterisation of the Lactococcal P335 phage-host interactions. *BMC Genomics* **2017**, *18*, 146. [CrossRef](https://doi.org/10.1186/s12864-017-3527-2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)
