
PLOS PATHOGENS

---

RESEARCH ARTICLE

Cryo-EM structures reveal two distinct conformational states in a picornavirus cell entry intermediate

Pranav N. M. Shah${}^{1\mathrm{a}}$, David J. Filman${}^{1}$, Krishanthi S. Karunatilaka${}^{1}$, Emma L. Hesketh${}^{2}$, Elisabetta Groppelli${}^{3\mathrm{b}}$, Mike Strauss${}^{4}$, James M. Hogle${}^{1*}$

1 Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA, United States of America, 2 Astbury Centre for Structural Molecular Biology, School of Molecular & Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds, United Kingdom, 3 School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds, United Kingdom, 4 Department of Anatomy and Cell Biology, McGill University, Montreal, Canada

${}^{\mathrm{a}}$ Current address: The Division of Structural Biology, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford, United Kingdom  
${}^{\mathrm{b}}$ Current address: Institute of Infection and Immunity, St George’s, University of London, Cranmer Terrace, Tooting, London United Kingdom  

* james_hogle@hms.harvard.edu

---

OPEN ACCESS

**Citation:** Shah PNM, Filman DJ, Karunatilaka KS, Hesketh EL, Groppelli E, Strauss M, et al. (2020) Cryo-EM structures reveal two distinct conformational states in a picornavirus cell entry intermediate. PLoS Pathog 16(9): e1008920. [https://doi.org/10.1371/journal.ppat.1008920](https://doi.org/10.1371/journal.ppat.1008920)

**Editor:** Z. Hong Zhou, University of California at Los Angeles, UNITED STATES

**Received:** January 17, 2020

**Accepted:** August 21, 2020

**Published:** September 30, 2020

**Peer Review History:** PLOS recognizes the benefits of transparency in the peer review process; therefore, we enable the publication of all of the content of peer review and author responses alongside final, published articles. The editorial history of this article is available here: [https://doi.org/10.1371/journal.ppat.1008920](https://doi.org/10.1371/journal.ppat.1008920)

**Copyright:** © 2020 Shah et al. This is an open access article distributed under the terms of the [Creative Commons Attribution License](https://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

**Data Availability Statement:** Cryo-EM maps and masks for m135, r135, h135, r135+m25 and r135+m37 conditions are deposited in the EMDB with accession codes EMD- 20275, EMD-20276, EMD-

---

### Abstract

The virions of enteroviruses such as poliovirus undergo a global conformational change after binding to the cellular receptor, characterized by a 4% expansion, and by the opening of holes at the two and quasi-three-fold symmetry axes of the capsid. The resultant particle is called a 135S particle or A-particle and is thought to be on the pathway to a productive infection. Previously published studies have concluded that the membrane-interactive peptides, namely VP4 and the N-terminus of VP1, are irreversibly externalized in the 135S particle. However, using established protocols to produce the 135S particle, and single particle cryo-electron microscopy methods, we have identified at least two unique states that we call the early and late 135S particle. Surprisingly, only in the “late” 135S particles have detectable levels of the VP1 N-terminus been trapped outside the capsid. Moreover, we observe a distinct density inside the capsid that can be accounted for by VP4 that remains associated with the genome. Taken together our results conclusively demonstrate that the 135S particle is not a unique conformation, but rather a family of conformations that could exist simultaneously.

---

### Author summary

Nonenveloped viruses need to provide mechanisms that allow their genomes to be delivered across membranes. This process remains poorly understood. For enteroviruses such as poliovirus, genome delivery involves a program of conformational changes that include expansion of the particle and externalization of two normally internal peptides, VP4 and the VP1 N-terminus, which then insert into the cell membrane, triggering endocytosis and the creation of pores that facilitate the transfer of the viral RNA genome across the

---

PLOS Pathogens | [https://doi.org/10.1371/journal.ppat.1008920](https://doi.org/10.1371/journal.ppat.1008920) September 30, 2020 1/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

20469, EMD-20546 and EMD-20474 respectively. Atomic co-ordinates for all the maps except r135 +m 37 can be accessed from the PDB with the following codes 6P90 (m135), 6P9W (r135), 6PSZ (h135), 6Q0B (r135+m25).

Funding: J.M.H was funded by National Institutes of Health grant AI020566 ([https://www.nih.gov/](https://www.nih.gov/)). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

Competing interests: The authors have declared that no competing interests exist.

endosomal membrane. This manuscript describes five high-resolution cryo-EM structures of altered poliovirus particles that represent a number of intermediates along this pathway. The structures reveal several surprising findings, including the discovery of a new intermediate that is expanded, but has not yet externalized the membrane interactive peptides; the clear identification of a unique exit site for the VP1 N-terminus; the demonstration that the externalized VP1 N-terminus partitions between two different sites in a temperature-dependent fashion; direct visualization of an amphipathic helix at the N-terminus of VP1 in an ideal position for interaction with cellular membranes; and the observation that a significant portion of VP4 remains inside the particle and accounts for a density feature that had previously been ascribed to part of the viral RNA. These findings represent significant additions to our understanding of the cell entry process of an important class of human pathogens.

# Introduction

Poliovirus is a small (~30nm) non-enveloped, positive sense, single-stranded RNA virus. It belongs to the genus Enterovirus of the Picornaviridae, and is the causative agent of poliomyelitis [1]. The icosahedral capsid of the virus, which is made of four proteins, namely VP1, VP2, VP3, and VP4, encloses the viral genome [2]. The capsid is structurally similar to other members of the family such as Rhinovirus, Coxsackievirus A16, Enterovirus 71 and Enterovirus D68, the latter three having caused recent epidemics of hand-foot-and-mouth disease, and even flaccid paralysis in China and the United States [1,3,4]. Therefore, studies on poliovirus are not only crucial to support the global effort to eradicate poliovirus, but also to tackle important emerging pathogens.

Poliovirus entry into cells is initiated when the virus interacts with its receptor, CD155, also known as poliovirus receptor (PVR) [5], an immunoglobulin-like molecule that is expressed at the intercellular junctions of epithelial cells [6]. Receptor engagement is concomitant with the loss of the “pocket factor” [7], a stabilizing ligand [8] that binds in the hydrophobic core of VP1 [9]. Structurally, receptor binding results in an icosahedral expansion of the capsid by 4% [10], yielding a 135S, or A-particle [11–16]. These particles are metastable, infectious [17,18] and RNA-containing. Therefore, it is thought that the 135S particle represents a critical step along the pathway to infection [18]. Unlike mature native virions (160S particles), whose capsids form a closed surface, the expanded 135S particles have holes in the capsid at the icosahedral 2-fold and quasi-3-fold axes. These openings permit the externalization of the membrane-interactive N-terminal extension of capsid protein VP1 [11,14] in poliovirus and in the 135S particles of other enteroviruses [19–26] and presumably VP4, which is myristoylated [27]. In vitro studies with model membranes have shown that these polypeptide chains are responsible for anchoring the virus to the host cell membrane [11,28]. Earlier tomographic studies from our group demonstrated umbilical connections between the virus and receptor-decorated liposomal membranes [29] that were wide enough to accommodate single-stranded genomic RNA. This could be the manner in which the genome is protected from the degradative effects of RNases during RNA transfer [30]. However, the molecular and structural underpinnings of this process still remain elusive.

In the present study we report five high-resolution reconstructions of the 135S uncoating intermediate particles, with refined atomic models for four of them.

Taken together, our results describe a multi-step model for picornavirus capsid uncoating and subsequent steps in genome translocation.

PLOS PATHOGENS

Picornavirus entry-intermediate structure

# Results

## Three structures produced by incubation with mAb

Initially we set out to investigate the structure of the “breathing” poliovirus particle. Previous studies showed that native poliovirions, when incubated at 37°C (but not at 25°C), transiently expose membrane-interactive polypeptides VP4 and the N-terminus of VP1 [31]. These two exposed polypeptides could be trapped outside the capsid by specific antibodies, some of which neutralized infectivity [31]. The release of the antibodies by freeze-thaw restored infectivity, demonstrating that the process was reversible [31]. An early attempt to characterize the breathing particle, trapped using a Fab that recognized residues 39–55 of VP1 [32], produced a low-resolution structure [33]. This structure revealed an RNA-containing virus particle whose capsid was trapped in an icosahedrally symmetric expanded state that was purported to be different from previously reported expanded forms of poliovirus. Weak and poorly ordered density located above the icosahedral 2-fold hole was attributed to peptide-bound Fab molecules getting "stuck" in random orientations when the VP1 N-terminus was partially withdrawn into the particle interior [33].

In the current study we have leveraged improvements in EM and computer hardware, data collection software, and image processing pipelines to re-examine with higher detail the antibody-trapped states relevant to the uncoating process, starting with the poliovirus "breathing" particle. This particle was produced by incubating native poliovirus with a commercially available monoclonal antibody (kind gift from Quidel Corp.) that was raised against a peptide corresponding to VP1 residues 42–55 (PALTA VETGATNPL). Although the antibodies used in this and previous studies recognize the same region of the N-terminus of VP1, this antibody differs from the one previously used, which is no longer available. Thus, in contrast to the earlier antibody which showed significant binding when incubating virus with the Fab, we could only obtain significant decoration of particles using the intact monoclonal antibody (mAb) provided by Quidel. Initial screening of conditions to trap the N-terminus outside the virus using the mAb at 37°C yielded very few decorated particles. Therefore, we performed additional cryo-EM screenings of the virus-mAb complexes for durations of up to 120 mins. at temperatures of 38°C and 39°C. An optimal number of complexes was observed after a 90-minute incubation of the virus with the mAb at 39°C.

Under those conditions, three main structural classes were obtained (Fig 1A), each containing roughly one-third of the usable particle images. Class 1 was not labeled by antibodies and was indistinguishable from previously solved 160S native virus [2], having an RNA core; VP4 and the N-terminus of VP1 bound on the inner surface of the capsid; and a closed protein shell that lacked large holes (left column, Fig 1A). Classes 2 and 3 both consisted of capsids that were expanded by about 4%, with large holes at the 2-fold axes. The class 2 particles appeared to contain a full complement of RNA (middle column, Fig 1A), whereas class 3 appeared similar to previously-studied 80S empty capsids [10,34,35] (rightmost column, Fig 1A), which lack RNA, VP4, order in residues 1–68 of VP1, and which have either unobstructed or partially obstructed holes at the quasi-3-fold axes.

The 80S-like class. In the icosahedral reconstructions of the class 3 particles, the density for the Fab portion of the bound antibody (for simplicity we will refer to this here and in subsequent sections as the Fab) was a flat roundish feature whose connection with the capsid was not directly apparent (rightmost column, Fig 1A). This density enclosed only the lower half of the Fab and was only visible at a much lower contour level than the capsid protein, suggesting both low occupancy, and variability in the position and angle of the Fab with respect to the virus. The position of the Fab, over the 2-fold hole, was similar to one of two sites for Fab binding in the low-resolution structure of an 80S-Fab complex described previously [36].

PLOS PATHOGENS

---

A.

Native
135S-like
80S-like

15nm

100Å
120Å
130Å
145Å
160Å
175Å

B.

Fab
VP2

VP1 N-terminus

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920  September 30, 2020  4/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

Fig 1. Anti-VP1 antibody trapped expanded particle. (A) Top row: The 3D classes from the dataset, with undecorated 160S particles (left, blue border), the anti-VP1 mAb decorated 135S-like particle (middle, orange border) and the mAb decorated empty particle (right, yellow border). Second row: Radially colored isosurface representations of the refined particles are shown at a high contour, where the holes at the 2-fold are clearly observed. Third row, isosurface representations of the mAb-decorated 135S (center) and 80S particles (right) contoured at a lower level showing the diffuse density attributable to the bottom half of the Fab part of the mAb. Bar indicates the radial range that each color corresponds to. (B) Density for an asymmetric focus class calculated from the mAb decorated 80S particle is depicted in the context of the 2-fold related VP2 subunits (yellow) and a model of a Fab (pink). The tube-like density for the N-terminus of VP1 (arrow) is seen above one of the two 2-fold related VP2 helices.

https://doi.org/10.1371/journal.ppat.1008920.g001

Focused classification methods, recently introduced into cryoEM, often succeed in avoiding the averaging of symmetry-related areas that have significantly different density patterns [37–39]. To further assess the interaction of the Fab with the capsid, we performed an asymmetric focused classification on 542,460 asymmetric units of the virus, using a cylindrical mask that enclosed the 2-fold axis of the capsid as well as the Fab. While the capsid was resolved to 4.2 Å in the icosahedrally averaged maps, the focused class in (Fig 1B) resolved to 6.5 Å. This class had a distinct tube-like density extending from above the VP2 helix (residues 90–100) at the 2-fold axis to the underside of the Fab (Fig 1B). We interpret this density to be the N-terminus of VP1.

135S-like particles (m135). Class 2 (which we will call m135) was the most interesting of the 3 classes (Fig 2 and middle column of Fig 1A), as it shared several of the physical characteristics of the canonical 135S particles. The m135 reconstruction was resolved to 2.8 Å resolution in the capsid region, after masking and "sharpening" (see Methods) (S1 Table, S1 Fig), and a detailed atomic model was constructed and refined for ordered amino acids in capsid proteins VP1, VP2, and VP3. Like a canonical 135S particle, the m135 reconstruction includes a significant central density for the RNA (but whose apparent icosahedral symmetry is an artefact of the calculation), lacks VP4 on the inner surface of the capsid, and has an expanded capsid with holes at the 2-fold and quasi-3-fold axes (middle column Fig 1A). In this structure and in all of the remaining 135S structures described below, the pocket factor is missing from the hydrophobic pocket of VP1 [9]. Similar to 135S-like particles for other enteroviruses [19–26], this structure has an additional portion of the N-terminal extension of VP1 (residues 64–69) that is ordered, and extends outward along the quasi-3-fold axis (Fig 2B–2D).

The density for the Fab is diffuse and rod-shaped, stretching between two 2-fold-related copies of the quasi-3-fold hole (Fig 2A and middle column of Fig 1A). The rod-like density feature is only evident above the noise level for the lower half of the Fab, which suggests variability in the position and orientation of the Fab. The weakness of the Fab density is not unexpected, given that the VP1 N-terminus is flexible, and entirely disordered in most non-native poliovirus structures. The weakness of the ordered Fab density (in the icosahedrally averaged maps) is consistent with a model in which specific Fab binding (to residues 42–55 of the VP1 N-terminus) at some of the quasi-3-fold sites near the virus surface is precluded by steric conflicts with previously bound Fabs.

In the high-resolution m135 structure, we see two general categories of structural change relative to native virus. One category corresponds to movement of the beta barrels of the major capsid proteins, VP1-3, away from one another when the capsid expands. As a consequence, several polypeptide segments (which, in native 160S virus, stabilize the "closed" capsid arrangement) detach from their neighbors and become disordered. On the inner surface of the capsid, these segments include VP4 and portions of the N-terminal extensions of VP1 and VP2. On the outer surface of the capsid, these segments include all C-termini of VP1, VP2, and VP3, as well as loops extending from the beta-barrel cores of VP1 and VP2 (see below). The general pattern of beta barrel shifts and polypeptide chain disordering that occurs upon the expansion of poliovirus has previously been analyzed [10,14,34,35], and is strikingly similar

PLOS PATHOGENS

---

A.
Fab
Capsid

B.
quasi 3-fold

C.
quasi 3-fold
VP1
N-terminus
VP1
GH-loop
VP3
GH-loop

D.
quasi 3-fold
VP1
N-terminus
VP2
"bottom" loop

E.
native-VP2
m135-VP2
m135-VP1

F.
native-VP2
m135-VP2
m135-VP1

---

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 6/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

Fig 2. Anti-VP1 antibody trapped genome containing particle (m135) is similar to the canonical 135S particle. (A) Cage density depiction ($\sigma = 0.25$) of the mAb-decorated m135 structure in the vicinity of the 2-fold axes showing the rod shaped density for the lower half of the Fab portion of the mAb and the better ordered density for the capsid proteins of the virus (B) The canonical poliovirus protomer consisting of VP1 (blue), VP2 (yellow) and VP3 (red) is shown juxtaposed with 5-fold related copies of VP1(cyan) and VP2 (gold). The cyan, gold, and red copies together form the “5-3-3 triangle” having one 5-fold and two 3-fold symmetry axes at its corners. The quasi 3-fold lies at the center of this triangle. Panels (B) and (C) are viewed from outside the virus. (C) The N-terminus of VP1 (dark blue) is seen exiting the virus capsid through the quasi 3-fold hole. To accommodate this, the GH loop of VP3 (labeled) becomes rearranged and partially disordered. Panels (D-F) are viewed from inside the virus. (D) The exiting N-terminus (dark blue) contacts the tip of the VP2 bottom loop (labeled) (E) The bottom loop of VP2, in m135 (yellow), is rearranged relative to native VP2 (magenta) (F) In m135, the C-terminus of VP2 (dark yellow) becomes ordered on the capsid inner surface, rather than exiting at the 2-fold, as in native virus (magenta).

https://doi.org/10.1371/journal.ppat.1008920.g002

to the shifts seen in the structures of 135S particles of other enteroviruses [19–26]. A specific listing of ordered and disordered residues is tabulated (see S2 Table) for the m135 atomic model, and for the three other atomic models described below.

The second, particularly interesting, set of structural changes involves the polypeptide segments that contact the VP1 N-terminus as it exits through the quasi-3-fold hole. Notably, the GH loop of VP3 (residues 176–185) uncoils, changing from a compact structure that plugs the quasi-3-fold in virions into a more extended structure that leaves room for residues 62–68 of VP1 (residues 1–61 were not modellable in this density) to thread through the quasi-3-fold (Fig 2C and 2D). Additionally, the "bottom" loop of VP2 (residues 26–58), which encircles the bottom of the VP2 beta barrel, rearranges at its tip (residues 43–55) to move out of the way of the externalizing VP1 N-terminus (Fig 2D and 2E). Finally, a portion of the GH loop of VP1 (residues 216–236) rearranges to facilitate the externalization of the VP1 N-terminus; and the C-terminal extension of VP2 (residues 262–272) retracts from its position on the outside of the virion to bind to the inner surface of the capsid (Fig 2F). The movement of the VP1-3 beta barrels is strikingly consistent with all of the 135S structures reported here and in those previously published for poliovirus and other enteroviruses [14,18–20,23–26,40] (see ordered residue lists, S2 Table). In particular, the four key contact areas (for the VP1 N-terminus) (Fig 2C–2F) either change their conformations or become disordered as a result of the 160S-to-135S conversion.

Finally, we considered the possibility that our antibody-trapped “breathing” particles would regain infectivity if the antibody were released by freeze-thawing, as was reported in the original report of “breathing” of poliovirus [31]. To test this possibility, virus samples were incubated with VP1 antibodies either at 39°C (where breathing is expected) or 25°C (where breathing is not expected) for 1.5 h (the same as for the EM experiment) and immunoprecipitated with Protein A/G coated magnetic beads at room temperature. Unbound virus was washed away, and the bound virus was released from the beads by freeze-thawing the solution. Next, the released virus was serially diluted and used to infect naïve Vero cells. Greater cytopathic effect was observed in virus particles retrieved from incubating the virus with the antibody at higher temperature than the virus particles retrieved from the room temperature incubation (S2A and S2B Fig). This suggests that some of the antibody-bound particles must have retained a full or nearly full complement of proteins (presumably including VP4) and remained infectious, even when accounting for non-specific binding of native virus to the antibody.

## Comparison of m135 particles with “canonical” 135S particles

Historically, three methods were used to convert 160S virus to 135S particles, and the resulting particles were not distinguishable by any of the tests then available. One method involved the binding of virus by the poliovirus receptor at the cell surface at 37°C in a natural infection [11–13]; the second involved incubation of virus with the soluble ectodomain of the receptor

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 7/26

at 37°C [41,42]; and the third involved heating 160S particles at 50°C in the presence of divalent cations that stabilize the meta-stable 135S intermediate [18,43]. To assess whether or not the m135 particle was indeed 135S-like, we decided to solve cryo-EM structures for both the soluble-receptor-triggered and heat-triggered 135S particles. Consistent with our "standard recipes" for producing 135S particles [8,17], we incubated native virus for 3 min, either with solubilized PVR at 37°C, or alone at 50°C. Note that PVR falls off the virus once expansion has been triggered. Receptor-triggered and heat-triggered 135S structures (designated "r135" and "h135", respectively) were solved at 3.2 Å resolution (after density-based masking and sharpening) (S1 Table, S1B and S2C Fig).

As expected, the capsids of both r135 and h135 were expanded by 4%, with holes at the 2-fold axes, were full of RNA, and failed to show well-ordered native-like VP4 pentamers lying flat on the inner surface of the capsid (Fig 3A). The r135 and h135 structures were qualitatively similar. There are no instances where several residues in succession are ordered in one model and disordered or differently ordered in the other. Minor differences between the two models include the occasional shift, by one or two residues along the polypeptide chain, where an order-to-disorder transition occurred, and scattered (rather than systematic) differences in the sets of "buildable" residues in native-like or non-native-like conformations (which are listed in S2 Table).

Surprisingly, both r135 and h135 maps lacked a visible extension of the VP1 N-terminus through the quasi-3-fold hole, which was supposed to be the structural hallmark of these particles [11,14,40]. In both cases, residue 69 was the first ordered residue of VP1; residues 43–53 of the bottom loop of VP2 (which supports the VP1 N-terminus) were disordered; and the quasi-3-fold hole was partially blocked by the GH loop of VP3 (Fig 3B). The lower portion of the loop was similar to that seen in native 160S virus (Fig 3C), but the upper portion was different from either the 160S structure or previously described 135S structures in which the VP1 N-termini were exposed. In short, while the literature [20,22,24,26] (and our own experience [14]) had led us to expect a 3 min incubation to produce an expanded particle with the VP1 N-termini extended, the 3 min incubation in this study produced an expanded and fenestrated capsid in which most or all of the VP1 N-termini remained inside the capsid. Therefore, we propose that the category of poliovirus particles that was previously called “135S” most likely consists of distinct states, namely, an “early state”, wherein the capsid has undergone an expansion but most copies of the VP1 N-terminus are on the inside of the capsid and the quasi-3-fold hole still remains blocked, and a subsequent “late state”, characterized by the opening of the quasi 3-fold hole and the emergence of many copies of the VP1 N-terminus through them. In support of this possibility, a careful re-reading of the literature [17,31,40] showed that 135S particles with extended N-termini were typically detected only after extensive purification and/or prolonged incubations. Interestingly, an expanded particle in which the VP1 N-termini remained inside the capsid has also been observed as a minority subpopulation (~5% of the particles) in preparations of 135S particles of enterovirus D68 [19]. However, in this subpopulation of the D68 135S particles the entire VP1 N-terminal extension is well ordered suggesting that the “early” state of the 135S-like particles may itself be comprised of two steps. Thus, in a first step the particle undergoes expansion with the VP1 N-terminus remaining in a well-ordered native-like arrangement on the inner surface of the expanded capsid; this is followed by a second step in which the VP1 N-terminal extension dissociates from the inner surface, prior to being externalized.

---

### Receptor-induced 135S particles incubated with mAb at 25°C

To test our hypothesis, that VP1 N-termini (from expanded particles) become externalized after an extended time, we followed the method described in Tsang et al [44], first incubating

PLOS PATHOGENS

---

A.

![Image Description](image1)

B.

![Image Description](image2)

h135  
r135  

VP3 GH-loop

C.

![Image Description](image3)

VP3 GH-loop of native r135 m135

---

PLOS Pathogens | [https://doi.org/10.1371/journal.ppat.1008920](https://doi.org/10.1371/journal.ppat.1008920) September 30, 2020 9/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

Fig 3. Receptor-catalyzed, and heat-triggered 135S particles (r/h135). (A) Radially colored isosurface rendering of the “early” 135S particle at a high contour (σ = 5) (left panel) reveals an opening at the 2-fold axis. The panel on the right is shown at a lower contour level (σ = 2) with a clipping plane inserted to highlight the re-organised density inside the capsid. These two panels are based on the r135 map, because the two structures are virtually identical. Bar indicates the radial range that each color corresponds to. (B) Close up of the quasi-3-fold region. Density for the GH loop of VP3, overlaid on the model, is very similar in the r135 (blue) and h135 (purple) maps. In this conformation, the GH loop of VP3 plugs the quasi-3-fold hole. (C) Stereo views compare the GH loops of VP3 from the r135 (red), m135 (salmon), and native (orange) structures. The r135 conformation is more similar to the native conformation, as opposed to the m135 model where the GH loop becomes more extended and forms new beta-sheet interactions with a radially-oriented segment of the VP3 D strand.

https://doi.org/10.1371/journal.ppat.1008920.g003

the virus with receptor for 3 mins at 37°C, followed by a 1hr incubation with antibody at 25°C. This protocol was carried out twice, once with a smaller "screening" data set, and once with a larger one. Data collection parameters are summarized in S1 Table.

As predicted, incubation of the pre-formed "early 135S particle" with antibody for 1hr at 25°C did decorate the virus particles with Fab-shaped density features. (Fig 4A). Surprisingly, the Fab portions of the antibody molecules did not bind at quasi-3-fold positions, to produce a low-level rod-shaped density, such as that seen in the m135 map (Fig 2A and Fig 1A middle column). Instead, the Fab density in the icosahedrally-averaged map was poised above the 2-fold hole. Its position was similar to that of the much weaker hemispherical density in the decorated 80S particles (rightmost column, Fig 1A). However, in the new "r135+m25" reconstruction, density for the Fab was much better ordered, with the upper half of the Fab domain visible, and separated from the lower half by an appropriately placed gap in the density between subdomains (Fig 4B). Unlike the decorated 80S map, where the Fab may have been freely spinning around the viral 2-fold axis, here the Fab appears to have a preferred orientation, with each of its two hinges sitting on a (partly disordered) EF loop of VP2. (Note that the Fab envelope is a sufficiently low-resolution feature that imposing an icosahedral twofold operation on the Fab density envelope had only a minimal effect on the density shape). The density for the Fab being weaker than that of the capsid can be attributed, at least in part, to the fact that at most 30 copies of the Fab can bind to the virion over the 2-fold axis. Both the contour level needed to visualize the Fab and the quality of fit of this density to the Fab model indicated higher occupancy and more structural consistency in this complex than we saw in our other complexes with mAb.

In the r135+m25 icosahedrally averaged map, and in all of the focused classes that were calculated from the r135+m25 image stack, we looked under the bottom surface of the Fab, trying to find density for a peptide connection (perhaps similar to the density connections that we saw in the focus classes of antibody-decorated 80S particles (Fig 1B). Instead, the bottom surface of the Fab (above the 2-fold axis) consistently lacked an obvious connection; and the 2-fold hole was occupied by a density that we tentatively identify as the partially retracted C-terminus of VP2 (Fig 4B). In contrast, in focused classifications centered on the 2-fold axis, including the quasi-3-fold holes, we observed 61% of the asymmetric particles exhibiting a clear density for the N-terminus of VP1 emerging from the quasi 3-fold hole (Fig 4C), with the GH loop of VP3 in an extended conformation as described in the m135 model (Fig 2C).

Visualizing the exposed N-terminal helix of VP1. Interestingly, we observed a focused class for the screening data comprising of 21% of the 966,840 asymmetric units of the virus where the lower half of the Fab contacts both VP2 shoulders, but asymmetrically tilts towards one of the shoulders (Fig 4D). This causes the top half of the Fab to become less well ordered compared to the lower half of the molecule. In this class, the tilted Fab leans against a long, straight, upward-projecting density feature. The projecting density is unusual in its appearance with one wide dimension and a flat dimension (Fig 4D).

PLOS PATHOGENS

---

A.

100Å  
120Å  
130Å  
145Å  
160Å  
175Å  

B.

Fab  
Capsid  

C.

quasi 3-fold  
VP1  
N-terminus  
VP1  
GH-loop  
VP3  
GH-loop  

D.

Fab  
VP1 N-terminus  
VP2  

90°  

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 11/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

Fig 4. Receptor catalyzed 135S particle incubated with the anti-VP1 antibody at 25°C (r135+m25). (A) Radially colored isosurface rendering of the “late” 135S particle shown at high contour (σ = 5) (left) to highlight the hole at the 2-fold axis and at a lower contour (σ = 0.5) (right) to highlight binding of the Fab (pink) relative to the capsid. Bar indicates the radial range that each color corresponds to. (B) The interaction between the Fab density and the capsid is better resolved following a focused classification centered around the 2-fold symmetry axis of the capsid. (C) A focus map from (B) contoured at a σ = 3 compared with the m135 model, helps to visualize the N-terminus of VP1 (blue) exiting the capsid from the quasi-3-fold hole. The density in this region is poorly resolved in the icosahedral averaged reconstruction (D) A focused class obtained from the screening data contoured at σ = 3, using a cylindrical mask centered on the 2-fold axis. This map clearly shows a column of density that accommodates the first 20 amino acids of the N-terminus of VP1 (blue). The helix bound to the top surface of VP2 (yellow) is stabilized in a unique orientation by contacts with the Fab (pink). Clearly, the portions of VP2 that bind the base of the helix must be rearranged, in some unknown way that differs from the native conformation of VP2 that is shown (yellow).

https://doi.org/10.1371/journal.ppat.1008920.g004

The observed shape of the density closely matches the shape of an idealized alpha helix built with the sequence of the first 20 residues of VP1 (the amphipathic character of this helix is known to be evolutionarily conserved [11]). The helical density observed in our map (Fig 4D) appears on the top surface of the VP2 EF loop, in the same location that a difference density peak was previously observed between low-resolution structures of V8 protease-treated and untreated 135S particles [40] (S3B Fig). Similar column-like density features were seen in some of our other focus classes, but the density was never quite as convincing as it was in this example.

Taken together these data suggest that the Fab in r135+m25 particles is bound to a copy of the VP1 N-terminus that has exited at a nearby quasi-3-fold axis, and that its location above the 2-fold hole is determined by interactions of the framework region of the Fab with the extreme N-terminal residues of VP1 and the VP2 GH loops.

## Receptor-induced 135S particles incubated with mAb at 37°C

To account for the difference in the binding pattern of the Fab between the m135 and r135 +m25 reconstructions, we postulated that lowering the temperature had an effect on the pattern of Fab binding. To that end the experiment was performed in the same manner as the r135+m25 conditions, except that the temperature for incubating with antibody was maintained at 37°C. The Fab density that was observed (Fig 5A and 5B) appeared to be a superposition of the m135 and r135+m25 binding patterns. Thus, the density for the Fab is rod-like (as it was in the m135 reconstruction), but there is also significant density above the 2-fold axis which includes density corresponding to the upper half of the Fab (as was observed in the r135 +m25 reconstruction). The reduced occupancy of well-ordered Fab molecules at the 2-fold site (in r135+m37, relative to r135+m25) suggests that the higher temperature of incubation with antibody destabilizes the interaction of Fab with the VP2 GH loop. As was the case in the m135 structure, the r135+m37 reconstruction has readily interpretable density for the VP1 N- terminus exiting at the quasi-3-fold hole. Given that the r135+m37 map appears to be a lower- resolution (see S1 Table) superposition of two better-determined structures, and not relevant to biological infection, we elect not to report an atomic model for it.

## Possible pentamers of capsid protein VP4

The small myristoylated capsid protein VP4 [27], is believed to play an important role in the protection of viral RNA (as RNA crosses the gap between virus and membrane) and in the penetration of host cell membranes [45–47]. In 160S virions, prior to capsid expansion, pentamers of poliovirus VP4 lie flat on the inner surface of the capsid, cupping the VP3 beta tube in a ring of five myristates [2,9,27] (Fig 6, leftmost panel). The N-terminal residues (1–45) of VP4 intertwine in a pinwheel arrangement to form a 60Å diameter disk, while the C-terminal residues (46–70) extend away from the disk, towards the 3-fold axes, in a 5-pointed star

PLOS PATHOGENS

Picornavirus entry-intermediate structure

A.

100Å
120Å
130Å
145Å
160Å
175Å

B.
Fab
Capsid

C.
quasi 3-fold
VP1
N-terminus
VP3
GH-loop
VP1
GH-loop

Fig 5. Receptor catalyzed 135S particle incubated with the anti-VP1 antibody at 37°C (r135+m37). (A) Radially colored isosurface rendering of the “late” 135S particle that is produced after receptor-catalyzed expanded virus is incubated with anti-VP1 antibody at 37°C for 1h. Bar indicates the radial range that each color corresponds to (A). The panel on the left is contoured at a higher level (σ = 5) to show surface details such as holes at the 2-fold and density in the quasi-3-fold. At a lower contour (σ = 0.1) (right), smeared density due to Fab binding (beige) is apparent. (A and B) The Fab density in r135+m37 appears to be a super-position of the rod-shaped Fab density that extends over the quasi-3-fold (as seen in m135) and the 2-fold-centered density (as seen in the r135+m25 reconstruction). (C) a close-up view of the showing the VP1 N-terminus emerging at quasi-3-fold (compare Figs 2C and 4C).

https://doi.org/10.1371/journal.ppat.1008920.g005

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 13/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

A.

Native          r135           m135           Empty

Fig 6. Location of VP4 before and after capsid expansion. Profile (top panel) and face-on (lower panel) view of VP4 (green) in native ($\sigma = 2$), r135 ($\sigma = 1.2$), m135 ($\sigma = 0.7$) and empty states ($\sigma = 2$). The native orientation and location of VP4 has been preserved to highlight the changes in the density, as the capsid expands. Prior to RNA release, VP4 pentamers, somewhat rearranged, appear to remain at their original radius, on the outer surface of the RNA ball, as the rest of the capsid expands outward.

https://doi.org/10.1371/journal.ppat.1008920.g006

arrangement [10]. In contrast, in 80S empty capsids, most if not all of the RNA is gone and there is no density for VP4 [34,35] (Fig 6, rightmost panel). But what happens to VP4 in between those initial and final states?

Our five high-resolution structures of 135S-like particles offer some intriguing indications that many of the VP4 pentamers may, indeed, remain intact within the virus capsid, with their distances from the protein shell increasing as the rest of the capsid shifts outward. Like many other 135S-like expanded picornavirus structures, our maps show a distinct 10 Å thick layer on the outer surface of the unstructured RNA density (top row center of Fig 1A, and Fig 3A right). Although this layer has commonly been attributed to RNA [20,24], we believe that its shape and dimensions are more consistent with VP4.

In "early 135S particles" (i.e. expanded but with the VP1 N-terminus mostly inside), as seen in the "r135" and "h135" structures, the putative VP4 density is consistently cup-shaped in profile (Fig 6 second panel from left). Seen face-on, the density feature has a radial position and a diameter similar to that of a native VP4 pentamer. Like VP4 pentamers, the density feature is around 10 Å thick (which is much too thin to be duplex RNA). As in native virus, the center of the cup (near the 5-fold axis) contacts the VP3 beta tube; but further from the 5-fold axis, unlike native virus, the density is seen to curve away from the capsid (as if it were peeling off of the inner surface of the capsid). Notably, the shape of the density is pentagonal, with a large central hole, which differs from the star-shaped density of the native pentamer. This would require some concerted conformational change to occur as the pentamer detached.

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920   September 30, 2020   14/26

In the "late 135S particles" (expanded, with the VP1 N-terminus mostly externalized), as seen in the m135, r135+m25 and r135+m37 data sets, the cup-shaped density feature is slightly flatter, and is positioned slightly further away from the inner capsid surface (~14 Å) ([Fig 6](#fig6), third panels from left).

The proposed identification is supported by the findings of Curry et al. [17], which clearly showed low levels of VP4 (up to 25% of the levels observed in virions) in highly purified preparations of poliovirus 135S particles. That study disproved the widely held claim that VP4 (which stains and radiolabels poorly and is difficult to detect in SDS gels optimized for the larger capsid proteins VP1-3) was totally lacking in 135S particles. Moreover, because VP4 plays a key role in viral infection [47], at least some VP4 must remain present in 135S particles to explain the observation that these particles remain infectious [17,18]. Together with the biochemical evidence, the position and shape of the density feature suggests that the thin layer immediately adjacent to the capsid that has previously been attributed to RNA is more likely to be residual VP4 pentamers that are not externalized until the RNA genome is released.

## Discussion

Picornavirus cell-entry is still an area of active research. In the present study we have gained further insight into the dynamics of the expanded genome-containing infectious intermediate of poliovirus, which is known as the 135S particle. Importantly, we show that the 135S particle, which was thought to be a unique structure, is instead an ensemble of structures, that expansion is not completely coupled to externalization the N-terminal extension of VP1, as previously thought, and that a significant fraction of VP4 remains in partially occupied but well-ordered structures on the inside of the particle approximately 14 Å from the inner surface of the remainder of the capsid.

## Decoupling expansion and externalization of the VP1 N-terminus

One of the unexpected findings of the current study was the observation of 135S particles that were expanded but had not externalized the VP1 N-terminus. Thus in the r135 and h135 structures, where the 135S particles had been prepared by protocols that were believed to result in both expansion and externalization of the VP1 N-terminus, icosahedrally averaged maps showed that the VP3 GH loop was in a conformation that blocked the hole at the quasi-3-fold axes and there was no detectable density for the VP1 N-terminus. Expanded particles with internal (but ordered) VP1 N-termini have also been observed as a minority population in preparations of 135S particles of enterovirus D68 [19].

This structural observation appears to contrast with previous structural and biochemical studies that showed exposed VP1 N-termini in 135S-like particles that were prepared using protocols similar to the ones that we used to prepare h135 and r135 particles. Specifically, the exposed N-termini were accessible to proteases and antibodies in the biochemical studies, and at least partially ordered in the structural studies of 135S particles of poliovirus and other enteroviruses. Indeed, we observed weak but significant density for the externalized VP1 N-terminus in a recent intermediate resolution (5.5Å) structure of 135S particles that were prepared using the same heating protocol [14]. To address this apparent discrepancy, we tried applying our current cryoEM software protocols for classification and data processing to the older lower-resolution data set and produced a fenestrated reconstruction that lacked significant density for the VP1 N-terminus. We conclude that the population of particles included a mixture of structures, and that the methods used to select particles to be included in the earlier lower-resolution reconstruction resulted in enrichment in a minority population of particles with the VP1 N-terminus exposed.

PLOS PATHOGENS

Picornavirus entry-intermediate structure

# Where does the VP1 N-terminus exit the virion?

Previous structural and biochemical characterization of the 135S particle of poliovirus (and other related picornaviruses) using cryoEM, crystallography, and antibody studies, led to a model wherein, upon expansion, the N-terminus of VP1 initially exits via the 2-fold hole, and later gets locked into the quasi-3-fold channel through a “gear-shift” motion [14,22,33,36]. However, in none of our recent panel of structures did we observe evidence for such a motion. Instead, the density for the N-terminus of VP1 in 135S particles, whenever it was externalized and trapped by an antibody, was consistently observed to occupy the quasi-3-fold channel. Although we cannot rule out a model in which VP1 transiently exits via a hole at the 2-fold axis during the dynamic process of externalization, our recent maps suggest that any such externalized polypeptide is either quickly re-internalized, or quickly migrates to the quasi-3-fold axis. It is also worth noting that even after a long incubation with the monoclonal antibody (m135, r135+m25, and r135+m37), in all cases densities for the VP1 N-terminal extension and for the antibody were consistently weaker than capsid, which suggests that some VP1 N-termini may remain inside the capsid.

# A revised model for the virus-to-135S transition

Taken together our current results suggest a process with three major steps. First, upon binding to the receptor at 37°C or exposure to high temperature (50°C) the virus expands and a significant fraction of VP4 is externalized. This step is concomitant with the loss of “pocket factor” and of internal and external stabilizing interactions among neighboring protomers through their N- and C-termini and loops on the outer surface. In a second step, the residues in the VP1 N-terminus are then reorganized such that residues 69–64 run up through the quasi-3-fold. It is clear that this second step must happen during infection because biochemical studies show that the VP1 N-terminus is inserted into membranes in *in vitro* models [11,28] and during infection [48], and plays a key role in the late stages of viral entry [48]. We would argue, however, that the externalization of the VP1 N-terminus is transient and reversible on the time scales relevant to infection (minutes) and that exposure of a significant fraction of the VP1 N-termini on this time scale requires the presence of a trap. Trapping agents could include cellular membranes during infection [47], liposomes in *in vitro* model systems [11,28,29], or antibodies that bind the VP1 N-terminus once exposed [31,33,36]. In order to explain the observation of stable structures in which the VP1 N-terminus is clearly externalized, we postulate a third step in which the VP1 N-terminus slowly becomes locked on the outside of the capsid even in the absence of trapping agents, perhaps by forming stable interactions with the GH loops of VP3 and VP1, as seen in the m135 structure, in our previous intermediate-resolution reconstruction of the poliovirus 135S particle [14], and in the structures of the 135S particles of other enteroviruses [19,20,22–26]. What proportion of the N-termini are exposed in 135S particles, during natural infections, in the presence of cellular membranes, is now an open question.

# Whither the VP1 N-terminus after leaving the capsid?

Although it is clear that residues 69–64 of the VP1 N-terminus exit the capsid at the quasi-3-fold, the course of the externalized residues 1–61 that are distal to the exit point is less clear. In all of our structures, residues 61–56 are disordered. Based on where Fab density is seen, residues 55–42 (which constitute the antibody binding site) can be located either above a 2-fold axis (r135+m25), or above a quasi-3-fold axis (m135), or both (r135+m37). Finally, the results from an earlier low-resolution structure of the 135S particles [40] and the clear density amphipathic helix of VP1 in the r135+m25 structure reported here demonstrates that a significant

PLOS PATHOGENS                                                                 Picornavirus entry-intermediate structure

![Image Description](image.png)

Fig 7. Schematic of VP1 N-terminal stabilization in the r135+m25 state. Four distinct areas in the N-terminal extension of VP1 (dashed blue curve) can be stabilized by binding to the Fab (pink) and the viral capsid (grey). This overall binding pattern is suggested by (i) the observation of residues 70–62 in the m135 complex; (ii) the expectation that amino acids from the 55–42 range (the epitope for the anti-VP1 Ab) will bind to the underside of the anti-peptide mAb; (iii) the anchoring of the N-terminal helix (around residue 21) to the top surface of VP2; and (iv) the nonspecific stabilization of the amphipathic N-terminal helix along the side of the Fab in a focus map. Although the pathway of the N-terminal extension is unlikely to be unique, the diagram does help us to understand why the increased prevalence of Fab moieties at the 2-fold hole (in r135+m25) is evidence for the increased anchoring of the amphipathic helix (residues 1–21) on the top surface of VP2, presumably stabilized by lower temperature and longer incubation before the antibody is introduced.

https://doi.org/10.1371/journal.ppat.1008920.g007

fraction of the first 20 amino acids of VP1 can bind at the top of the EF loop of VP2. These results are summarized in Fig 7.

## Significance of the amphipathic helix of VP1

Earlier studies from our group [11,28] demonstrated the externalization of the VP1 N-terminus and described its role in the binding of 135S particles to membranes. Using flotation assays and V8-protease-based cleavage of the 135S particles, the membrane-interacting part of the peptide was localized to the first 30 amino acids in the sequence [11,28]. Further analysis predicted that the first 20 amino acids would adopt a helical conformation [11], with the helix having a hydrophobic and a hydrophilic side (a common membrane-interactive structural motif). Subsequently, single particle cryoEM analysis of V8-cleaved and uncleaved 135S particles [40] suggested that the externalized VP1 N-terminus would sometimes be anchored on top of the VP2 EF-loop. However, the density for the amphipathic helix had never been

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920   September 30, 2020   17/26

visualized directly. In the present study, we were fortunate to be able to visualize clear density for the entire helix, and to confirm that the helix is located in the same location that the V8-cleavage difference density from an earlier study [40] had predicted it to be (Fig 4D, S3B Fig). The fact that the helix is only clearly visible in focused reconstructions from the screening data set suggests that the interactions between the helix and the VP2 may be weak. However, we speculate that, even if the localization of the helix at the tip of the VP2 EF loop is transient at physiological temperature, by increasing the number of copies of the N-termini at a higher elevation on the virus particle, this interaction may drive the association with membrane, may facilitate engulfment of the virus particle at the cell surface and internalization into endosomes [49], and may serve as an asymmetric trigger for the translocation of the genome into the cytosol [29].

## Difference in antibody density under different conditions

Another puzzling aspect of the structures presented here is the observation that the Fab in the complexes produced by incubation with PVR at 37°C then with antibody at 25°C binds above the 2-fold hole, whereas the complexes produced during higher-temperature incubations showed Fab density above the quasi-3-fold axes. A likely explanation for our observations is that some of the N-termini of VP1 become externalized via the quasi-3-fold hole during the initial 3-min incubation with receptor at 37°C, and their N-terminal helices find the binding site on the VP2 EF loop, where they remain trapped once the temperature is lowered to 25°C. Upon subsequent incubation with antibody at room temperature, this configuration is further stabilized by nonspecific contacts between the antibody, the VP1 N-terminal helix, and the VP2 EF loops. In the maps (Fig 4A and 4B), this binding mode manifests as a well-ordered density that recognizably corresponds to both halves of the Fab portion of the antibody.

In contrast, when the particles are maintained at 37°C or 39°C during antibody binding, the higher temperature reduces the stability of the specific interactions of the VP1 N-terminal helix with the VP2 EF loop, the nonspecific interactions of the hinges of the Fab with the VP2 EF loop “shoulders”; and the nonspecific interactions of the VP1 helix with the sides of the Fab. Thus, the 2-fold binding of Fab was reduced at 37°C, and almost eliminated at 39°C. Instead, a greater proportion of the Fab moieties were bound closer to the quasi-3-fold hole, where Fab orientations were less well-constrained, and the Fab density was consequently more diffuse. Asymmetric focus maps were calculated for cylindrical volumes near the 2-fold axes to see if any well-ordered Fab binding sites had been smeared out by symmetry-based density averaging; but no such well-ordered sites were seen.

## Presence of VP4 in an expanded particle

VP4 is a myristoylated, pore-forming protein that is implicated in the translocation the RNA genome across membranes[27,45,47]. This protein was thought to be completely lost after conversion to the 135S state [11,12,16]. However, biochemical and infectivity assays indicate that at least 25% of VP4 remains associated with the virus particle [17], and that the 135S particle remains infectious [17,18] (Fig 6B and S2 Fig). While these data by themselves do not establish whether VP4 remains on the interior or if it is otherwise bound to the capsid surface, they do underscore the presence of VP4 in highly purified 135S particles. In several related picornaviruses, reports of expanded virus structures have noted the disappearance of VP4 from the inner surface of the capsid, and have attributed all of the unstructured density inside of the capsid to RNA, including a thin 10-Å-thick layer at the periphery of the icosahedrally-averaged "RNA ball" [20,24]. However, in our panel of expanded genome-containing structures, we can alternatively account for the density under the 5-fold peak using a native atomic model of the

VP4 pentamer (Fig 6). Moreover, an unmistakable gradual curving of the VP4 layer away from the capsid, peeling off of the inner surface, was seen in the r135 and h135 structures (Fig 6) that is remarkably similar in appearance to the density that has been ascribed to RNA in previous studies with other enteroviruses [20,24]. VP4 has the capacity to bind to RNA [50] and perhaps that is the reason it is being held in place at its original radius in native virus while the rest of the capsid expands radially. Nevertheless, it remains unclear how VP4, thus separated from the capsid, possibly still associated with RNA, and with pentamers intact or not, manages to exit the capsid and to embed itself in the membrane. This will be an important avenue to explore in understanding the crucial genome-translocation step in picornavirus infection.

# Materials and methods

## Virus and antibody

Poliovirus type 1 Mahoney strain was purified by infecting Hela S3 suspension cells at an MOI = 10 for 6 hrs at 37°C. Infected cells were spun down, washed with phosphate buffered saline (PBS: 137 mM NaCl, 2.7 mM KCl, 1.1 mM KH₂PO₄, 6.5 mM Na₂HPO₄, 0.7 mM CaCl₂, 0.5 mM MgCl₂) and frozen at -80°C. Virus was released from the cells by repeated freeze-thawing of the infected cell pellet in RSB buffer (10 mM Tris pH 7.3, 10 mM NaCl, 1.5 mM MgCl₂, 1% IGEPAL CA-630) and clarification of the cell debris through centrifugation. The virus was pelleted through ultracentrifugation and layered on a CsCl density gradient. Subsequently, the virus band was collected and dialysed against PBS before being flash frozen in liquid nitrogen and stored at -80°C.

The anti-VP1 monoclonal antibody (one of two monoclonal antibodies in a pan-Enterovirus ELISA detection kit) was obtained from Quidel (Athens, Ohio).

## Plaque assay

Vero cells were grown in 6-well plates until confluency at 37°C and 5% CO₂. The cells were then infected with serial dilutions of poliovirus. At 36 h post-infection cells were fixed with 10% formaldehyde and stained with 0.1% crystal violet.

## In vitro generation of poliovirus 135S and 135S-like particles

Anti-VP1 antibody triggered 135S-like particles were generated by mixing 18 μL of virus suspension (0.4 mg/ml) with 3 μL of anti-VP1 antibody (3.2 mg/ml) and incubated for 1.5 h at 39°C in a thermal cycler. (Over numerous trials, these conditions of temperature and duration maximized the proportion of mAb-decorated RNA-containing virus particles. Complexes with Fab fragments were also attempted but did not produce detectable levels of decoration.) To generate 135S particles, PV suspension was first buffer exchanged into a low salt buffer (20 mM Tris-Cl pH 7.5, 2 mM MgCl₂). The receptor-triggered conversion of PV was induced by mixing 18.6 μL of PV (0.5 mg/ml) with 0.4 μL of PVR/CD155 (6.1 mg/ml) on ice before incubating at 37°C for 3 min. The reaction was quenched on ice. Heat-triggered 135S particles were generated by heating native PV (in the low salt buffer) to 50°C for 3 min, before quenching the reaction on ice. The r135+m25/37 particles were produced by incubating PV with PVR, as described above, followed by the addition of the anti-VP1 mAb (150x molar excess over virus) and incubation at 25°C or 37°C for 1 h.

## Cryo-electron microscopy and data collection

Carbon-backed lacey grids (Ted Pella) were glow discharged in an easiGLO (PELCO, Fresno, California) glow discharge unit by applying a current of 15 mA for 30 s. To the freshly glow-

discharged grids, 3.0–3.5 μL of virus or virus-ligand mixture was applied. The samples were then allowed to adsorb on the grid for 30 s before blotting away the excess buffer using a blot force of 6 and blot time of 6 s in a FEI Vitrobot Mark IV (ThermoFisher). The grids were plunge frozen in a liquid ethane slush and stored in liquid nitrogen until imaging. Data collection parameters are listed in S1 Table.

# Image processing

Image processing of micrographs was performed within Relion 3.0 [51]. Individual movie frames were aligned and dose-weighted with the Relion implementation of motion correction by applying a negative B-factor of -150 [52]. CTF parameter estimation was performed using Gctf v1.06 [53] on dose-weighted micrographs. Particle-picking was performed in crYOLO [54] by training the neural network on an initial subset of ~300 manually picked particles from several micrographs sampling a range of defocus values. Subsequently, the particle picker was deployed on the complete dataset to identify and save particle co-ordinates which were then imported into Relion. The picked particles were extracted, binned by 4 and subjected to a single round of 2D classification and obviously junk classes were discarded prior to 3D classification. The particle stack was refined against a representative class obtained at the end of the 3D classification. After the first round of refinement, the dataset was subjected to CTF refinement [52] by estimating beam tilt, per-particle defocus and astigmatism followed by Bayesian polishing [52] with default parameters. Icosahedral symmetry was imposed in every round of refinement and the reference model was filtered to 60 Å to prevent model bias. The final resolution (0.143 gold standard FSC) of the structures (S1 Table) was obtained through the Postprocessing job type in Relion, where a density-based mask enclosing the capsid was applied to estimate the FSC between two independently processed half datasets. Local resolution estimates were also obtained by running the Relion’s implementation of local resolution algorithm.

# Asymmetric focused classes

After assignment of icosahedral orientations to the particle images, a cylindrical mask was prepared using SPIDER [55] with a diameter of 105 Å, centered on the viral 2-fold and enclosing two copies of the quasi 3-fold axis. The initial height of 475 Å was then limited to a spherical shell by applying minimum cutoff at 154 Å. Then, using relion_particle_symmetry_expand a new STAR file containing 60 icosahedrally related orientations for each particle in the stack was generated. This symmetry expanded stack was then subjected to a masked 3D classification routine in which orientational and translational searches were disabled, the T-value was set to 40, and 5 density classes were generated.

# Model building and refinement

A starting model for m135 was assembled from 1HXS and previously published high-resolution structures of the expanded poliovirus structures [34,56] were docked into the density using COOT [57]. Where none of the pre-existing models for the density fit well, the segments were constructed *de novo*.

Models were iteratively built and refined in REFMAC5 and/or phenix_refine and/or phenix_real_space_refine, until no further improvement was possible [58–60]. To avoid model bias, the Fourier transforms of the experimental maps (both amplitudes and phases) were used as refinement standards. The model included a central protomer (VP1, VP2, and VP3) and neighboring proteins related by icosahedral symmetry. Strong restraints were included for stereochemistry, symmetry, and temperature factors.

PLOS PATHOGENS

---

Figure preparation

Figures and illustrations were prepared using PyMol (Schrödinger, LLC), Chimera [61], ChimeraX [62] sourced through the SBGrid Consortium [63] and compiled in Adobe Illustrator (Adobe, USA)

Supporting information

S1 Fig. Resolution and quality of maps. (A) Representative, raw micrographs of the datasets used in the analysis, (B) Masked (continuous line) and unmasked (dotted line) Fourier shell correlation plots of m135 (red), r135 (green), h135 (cyan), r135+m25 (blue) and r135+m37 (purple). At a 0.143 cut-off level, the resolution estimates range from 2.8 to 3.6Å. (C) Local resolution estimates in all the maps were calculated using Relion and the distribution of resolution values within in the masked region is plotted. (D) In the best resolved dataset (m135), individual amino acid side chains are easily discriminated.

(TIF)

S2 Fig. Binding of anti-VP1 mAb to the VP1 N-terminus and infectivity of mAb released poliovirions. (A) The anti-VP1 antibody specifically recognizes and binds the N-terminus of VP1 only after the N-terminus has been externalized at 39°C. Poliovirus particles were complexed with the antibody at room temperature or at 39°C for 1.5 h and immunoprecipitated with magnetic Protein A coated beads. After thorough washing of the beads to remove unbound material, the samples were examined on SDS-PAGE gels. Lane 1, Ladder; lane 2, PV particles only; lane 3, antibody; lane 4, soluble fraction of antibody plus 160S PV after binding at RT; lane 5, pelleted fraction of antibody plus 160S PV after binding at RT; lane 6, soluble fraction of antibody plus 160S PV after binding at 39°C; lane 7, pelleted fraction of antibody plus 160S PV after binding at 39°C. Bottom panel, silver staining of the gel to enhance the relatively weak VP4 signal (B) Expanded poliovirus particles are infectious. Native (160S) poliovirus particles previously incubated with anti-VP1 antibody at room temperature or at 39°C for 1.5 h were immunoprecipitated with magnetic Protein A coated beads. After thorough washing of the beads to remove unbound material, the beads were freeze-thawed to release poliovirus particles. Released poliovirus was serially diluted and plated on naive Vero cells. After 36 h of incubation at 37°C, plaques were visualized by staining with crystal violet.

(TIF)

S3 Fig. Visualising poliovirus VP1 N-terminus. (A) Asymmetric focused classes calculated for the r135+m25 dataset with percentage population per class. Class 3 is depicted in [Fig 4B](https://doi.org/10.1371/journal.ppat.1008920#fig4b) and [4C](https://doi.org/10.1371/journal.ppat.1008920#fig4c). (B) Difference density, shown here in stereo, was calculated from previously published low-resolution reconstructions of poliovirus 135S particles, either untreated or treated with V8 protease, which cleaves at residue 31 of VP1. As a guide, we have superimposed an alpha carbon model of r135+m25. The blue alpha helix (putatively residues 1–21 of VP1) was fitted to a focused class of r135+m25 (as shown in [Fig 4D and 4E](https://doi.org/10.1371/journal.ppat.1008920#fig4de)). Observe that the putative VP1 helix model, bound to the top of VP2, is similar in position and orientation to the previously reported difference map feature.

(TIF)

S1 Table. Data collection parameters and refinement statistics. *The r135+m25 model, which includes placeholders for the Fab and VP4 densities, was calculated with phenix_refine, using its automatic choice of weighting schemes. Other final atomic models were refined with Refmac5, using stronger stereochemical restraints.

(TIF)

PLOS PATHOGENS

Picornavirus entry-intermediate structure

S2 Table. List of residues that differ from the native structure.* Differently ordered residues are those residues that significantly differ in position from the native model (1HXS) following a least squares superposition of individual capsid proteins.

(TIF)

# Acknowledgments

We would like to thank Drs. Lawrence Shapiro and Oliver Harrison (Columbia University) for providing us with PVR, Sara McCoy (Quidel) for providing the anti VP1 antibody, Dr. David Bhella (University of Glasgow) and Dr. Simon Jenni (Harvard University) for helping us run the asymmetric focused classification. We would also like to thank members of the SBGrid team for maintaining the computing environment. Dr. Takanori Nakane (MRC-LMB) is acknowledged for sharing the code to plot the resolution distribution in 3D volumes. We would like to acknowledge the Astbury Biostructure Laboratory (ABSL) for Titan Krios cryo-EM data collection which was funded by the University of Leeds and the Wellcome Trust (108466/Z/15/Z) as well as the Harvard Cryo-EM Center for Structural Biology

# Author Contributions

Conceptualization: David J. Filman, Mike Strauss, James M. Hogle.

Data curation: Pranav N. M. Shah, David J. Filman.

Formal analysis: Pranav N. M. Shah, David J. Filman, Krishanthi S. Karunatilaka, James M. Hogle.

Funding acquisition: James M. Hogle.

Investigation: Pranav N. M. Shah, David J. Filman, Krishanthi S. Karunatilaka.

Methodology: Pranav N. M. Shah, David J. Filman, Krishanthi S. Karunatilaka, Emma L. Hesketh, Elisabetta Groppelli.

Project administration: James M. Hogle.

Resources: Pranav N. M. Shah, James M. Hogle.

Software: Pranav N. M. Shah, David J. Filman.

Supervision: David J. Filman, Mike Strauss, James M. Hogle.

Validation: David J. Filman, James M. Hogle.

Visualization: Pranav N. M. Shah, David J. Filman.

Writing – original draft: Pranav N. M. Shah, David J. Filman, James M. Hogle.

Writing – review & editing: Pranav N. M. Shah, David J. Filman, Elisabetta Groppelli, Mike Strauss, James M. Hogle.

# References

1. Ryu W-S. Chapter 11—Picornavirus. In: Ryu W-S, editor. Molecular Virology of Human Pathogenic Viruses. Boston: Academic Press; 2017. pp. 153–164. [https://doi.org/10.1016/B978-0-12-800838-6.00011-4](https://doi.org/10.1016/B978-0-12-800838-6.00011-4)
2. Hogle JM, Chow M, Filman DJ. Three-dimensional structure of poliovirus at 2.9 Å resolution. Science. 1985; 229: 1358–1365. [https://doi.org/10.1126/science.2994218 PMID: 2994218](https://doi.org/10.1126/science.2994218 PMID: 2994218)

PLOS Pathogens | [https://doi.org/10.1371/journal.ppat.1008920](https://doi.org/10.1371/journal.ppat.1008920) September 30, 2020 22/26

PLOS PATHOGENS

---

3. Ayers T, Lopez A, Lee A, Kambhampati A, Nix WA, Henderson E, et al. Acute Flaccid Myelitis in the United States: 2015–2017. Pediatrics. 2019; 144: e20191619. https://doi.org/10.1542/peds.2019-1619 PMID: 31591135

4. Sejvar JJ, Lopez AS, Cortese MM, Leshem E, Pastula DM, Miller L, et al. Acute Flaccid Myelitis in the United States, August-December 2014: Results of Nationwide Surveillance. Clin Infect Dis. 2016; 63: 737–745. https://doi.org/10.1093/cid/ciw372 PMID: 27318332

5. Mendelsohn CL, Wimmer E, Racaniello VR. Cellular receptor for poliovirus: molecular cloning, nucleotide sequence, and expression of a new member of the immunoglobulin superfamily. Cell. 1989; 56: 855–865. https://doi.org/10.1016/0092-8674(89)90690-9 PMID: 2538245

6. Takai Y, Irie K, Shimizu K, Sakisaka T, Ikeda W. Nectins and nectin-like molecules: roles in cell adhesion, migration, and polarization. Cancer Sci. John Wiley & Sons, Ltd (10.1111); 2003; 94: 655–667. https://doi.org/10.1111/j.1349-7006.2003.tb01499.x PMID: 12901789

7. Strauss M, Filman DJ, Belnap DM, Cheng N, Noel RT, Hogle JM. Nectin-like interactions between poliovirus and its receptor trigger conformational changes associated with cell entry. Kirkegaard K, editor. Journal of Virology. American Society for Microbiology; 2015; 89: 4143–4157. https://doi.org/10.1128/JVI.03101-14 PMID: 25631086

8. Tsang SK, Danthi P, Chow M, Hogle JM. Stabilization of poliovirus by capsid-binding antiviral drugs is due to entropic effects. J Mol Biol. 2000; 296: 335–340. https://doi.org/10.1006/jmbi.1999.3483 PMID: 10669591

9. Filman DJ, Syed R, Chow M, Macadam AJ, Minor PD, Hogle JM. Structural factors that control conformational transitions and serotype specificity in type 3 poliovirus. The EMBO Journal. John Wiley & Sons, Ltd; 1989; 8: 1567–1579. https://doi.org/10.1002/j.1460-2075.1989.tb03541.x PMID: 2548847

10. Belnap DM, Filman DJ, Trus BL, Cheng N, Booy FP, Conway JF, et al. Molecular tectonic model of virus structural transitions: the putative cell entry states of poliovirus. Journal of Virology. American Society for Microbiology; 2000; 74: 1342–1354. https://doi.org/10.1128/jvi.74.3.1342-1354.2000 PMID: 10627545

11. Fricks CE, Hogle JM. Cell-induced conformational change in poliovirus: externalization of the amino terminus of VP1 is responsible for liposome binding. Journal of Virology. American Society for Microbiology; 1990; 64: 1934–1945. https://doi.org/10.1128/JVI.64.5.1934-1945.1990 PMID: 2157861

12. de Sena J, Mandel B. Studies on the in vitro uncoating of poliovirus II. Characteristics of the membrane-modified particle. Virology. Academic Press; 1977; 78: 554–566. https://doi.org/10.1016/0042-6822(77)90130-1

13. Fenwick ML, Cooper PD. Early interactions between poliovirus and ERK cells: Some observations on the nature and significance of the rejected particles. Virology. Academic Press; 1962; 18: 212–223. https://doi.org/10.1016/0042-6822(62)90007-7

14. Butan C, Filman DJ, Hogle JM. Cryo-Electron Microscopy Reconstruction Shows Poliovirus 135S Particles Poised for Membrane Interaction and RNA Release. Journal of Virology. American Society for Microbiology Journals; 2014; 88: 1758–1770. https://doi.org/10.1128/JVI.01949-13 PMID: 24257617

15. Joklik WK, Darnell JE Jr. The adsorption and early fate of purified poliovirus in HeLa cells. Virology. Academic Press; 1961; 13: 439–447. https://doi.org/10.1016/0042-6822(61)90275-6

16. Crowell RL, Philipson L. Specific alterations of coxsackievirus B3 eluted from HeLa cells. Journal of Virology. American Society for Microbiology (ASM); 1971; 8: 509–515. https://doi.org/10.1128/JVI.8.4.509-515.1971 PMID: 4331654

17. Curry S, Chow M, Hogle JM. The poliovirus 135S particle is infectious. Journal of Virology. American Society for Microbiology Journals; 1996; 70: 7125–7131. https://doi.org/10.1128/JVI.70.10.7125-7131.1996 PMID: 8794359

18. Huang Y, Hogle JM, Chow M. Is the 135S Poliovirus Particle an Intermediate during Cell Entry? Journal of Virology. American Society for Microbiology Journals; 2000; 74: 8757–8761. https://doi.org/10.1128/jvi.74.18.8757-8761.2000 PMID: 10954579

19. Liu Y, Sheng J, van Vliet ALW, Buda G, van Kuppeveld FJM, Rossmann MG. Molecular basis for the acid-initiated uncoating of human enterovirus D68. Proc Natl Acad Sci USA. 2018; 115: E12209–E12217. https://doi.org/10.1073/pnas.1803347115 PMID: 30530701

20. Zhu L, Sun Y, Fan J, Zhu B, Cao L, Gao Q, et al. Structures of Coxsackievirus A10 unveil the molecular mechanisms of receptor binding and viral uncoating. Nature Communications. Nature Publishing Group; 2018; 9: 4985. https://doi.org/10.1038/s41467-018-07531-0 PMID: 30478256

21. Pickl-Herk A, Luque D, Vives-Adrián L, Querol-Audí J, Garriga D, Trus BL, et al. Uncoating of common cold virus is preceded by RNA switching as determined by X-ray and cryo-EM analyses of the subviral A-particle. Proc Natl Acad Sci USA. National Academy of Sciences; 2013; 110: 20063–20068. https://doi.org/10.1073/pnas.1312128110 PMID: 24277846

---

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 23/26

PLOS PATHOGENS

Picornavirus entry-intermediate structure

22. Ren J, Wang X, Hu Z, Gao Q, Sun Y, Li X, et al. Picornavirus uncoating intermediate captured in atomic detail. Nature Communications. 2013; 4: 1929. https://doi.org/10.1038/ncomms2889 PMID: 23728514

23. Shingler KL, Yoder JL, Carnegie MS, Ashley RE, Makhov AM, Conway JF, et al. The enterovirus 71 A-particle forms a gateway to allow genome release: a cryoEM study of picornavirus uncoating. Rey FA, editor. PLoS Pathog. 2013; 9: e1003240. https://doi.org/10.1371/journal.ppat.1003240 PMID: 23555253

24. Organtini LJ, Makhov AM, Conway JF, Hafenstein S, Carson SD. Kinetic and structural analysis of coxsackievirus B3 receptor interactions and formation of the A-particle. Journal of Virology. 2014; 88: 5755–5765. https://doi.org/10.1128/JVI.00299-14 PMID: 24623425

25. Xu L, Zheng Q, Li S, He M, Wu Y, Li Y, et al. Atomic structures of Coxsackievirus A6 and its complex with a neutralizing antibody. Nature Communications. Nature Publishing Group; 2017; 8: 505. https://doi.org/10.1038/s41467-017-00477-9 PMID: 28894095

26. Lyu K, Ding J, Han J-F, Zhang Y, Wu X-Y, He Y-L, et al. Human enterovirus 71 uncoating captured at atomic resolution. Hutt-Fletcher L, editor. Journal of Virology. 5 ed. American Society for Microbiology Journals; 2014; 88: 3114–3126. https://doi.org/10.1128/JVI.03029-13 PMID: 24352461

27. Chow M, Newman JFE, Filman D, Hogle JM, Rowlands DJ, Brown F. Myristylation of picornavirus capsid protein VP4 and its structural significance. Nature. Nature Publishing Group; 1987; 327: 482–486. https://doi.org/10.1038/327482a0 PMID: 3035380

28. Tuthill TJ, Bubeck D, Rowlands DJ, Hogle JM. Characterization of Early Steps in the Poliovirus Infection Process: Receptor-Decorated Liposomes Induce Conversion of the Virus to Membrane-Anchored Entry-Intermediate Particles. Journal of Virology. American Society for Microbiology Journals; 2006; 80: 172–180. https://doi.org/10.1128/JVI.80.1.172-180.2006 PMID: 16352541

29. Strauss M, Levy HC, Bostina M, Filman DJ, Hogle JM. RNA transfer from poliovirus 135S particles across membranes is mediated by long umbilical connectors. Journal of Virology. American Society for Microbiology; 2013; 87: 3903–3914. https://doi.org/10.1128/JVI.03209-12 PMID: 23365424

30. Groppelli E, Levy HC, Sun E, Strauss M, Nicol C, Gold S, et al. Picornavirus RNA is protected from cleavage by ribonuclease during virion uncoating and transfer across cellular and model membranes. Sarnow P, editor. PLoS Pathog. Public Library of Science; 2017; 13: e1006197. https://doi.org/10.1371/journal.ppat.1006197 PMID: 28166307

31. Li Q, Yafal AG, Lee YM, Hogle J, Chow M. Poliovirus neutralization by antibodies to internal epitopes of VP4 and VP1 results from reversible exposure of these sequences at physiological temperature. Journal of Virology. American Society for Microbiology; 1994; 68: 3965–3970. https://doi.org/10.1128/JVI.68.6.3965-3970.1994 PMID: 7514682

32. Rovainen M, Piirainen L, Ryä T, Närvänen A, Hovi T. An Immunodominant N-Terminal Region of VP1 Protein of Poliovirion That Is Buried in Crystal Structure Can Be Exposed in Solution. Virology. Academic Press; 1993; 195: 762–765. https://doi.org/10.1006/viro.1993.1427 PMID: 8393243

33. Lin J, Lee LY, Rovainen M, Filman DJ, Hogle JM, Belnap DM. Structure of the Fab-labeled “breathing” state of native poliovirus. Journal of Virology. American Society for Microbiology; 2012; 86: 5959–5962. https://doi.org/10.1128/JVI.05990-11 PMID: 22398295

34. Strauss M, Schotte L, Karunatilaka KS, Filman DJ, Hogle JM. Cryo-electron Microscopy Structures of Expanded Poliovirus with VHHs Sample the Conformational Repertoire of the Expanded State. Dermody TS, editor. Journal of Virology. American Society for Microbiology Journals; 2017; 91: e01443–16. https://doi.org/10.1128/JVI.01443-16 PMID: 27852863

35. Levy HC, Bostina M, Filman DJ, Hogle JM. Catching a virus in the act of RNA release: a novel poliovirus uncoating intermediate characterized by cryo-electron microscopy. Journal of Virology. American Society for Microbiology; 2010; 84: 4426–4441. https://doi.org/10.1128/JVI.02393-09 PMID: 20181687

36. Lin J, Cheng N, Chow M, Filman DJ, Steven AC, Hogle JM, et al. An externalized polypeptide partitions between two distinct sites on genome-released poliovirus particles. Journal of Virology. American Society for Microbiology; 2011; 85: 9974–9983. https://doi.org/10.1128/JVI.05013-11 PMID: 21775460

37. McElwee M, Vijayakrishnan S, Rixon F, Bhella D. Structure of the herpes simplex virus portal-vertex. Sugden B, editor. Plos Biol. Public Library of Science; 2018; 16: e2006191. https://doi.org/10.1371/journal.pbio.2006191 PMID: 29924793

38. Conley MJ, McElwee M, Azmi L, Gabrielsen M, Byron O, Goodfellow IG, et al. Calicivirus VP2 forms a portal-like assembly following receptor engagement. Nature. Nature Publishing Group; 2019; 565: 377–381. https://doi.org/10.1038/s41586-018-0852-1 PMID: 30626974

39. Snowden JS, Hurdiss DL, Adeyemi OO, Ranson NA, Herod MR, Stonehouse NJ. Dynamics in the murine norovirus capsid revealed by high-resolution cryo-EM. Chiu W, editor. Plos Biol. 2020; 18: e3000649. https://doi.org/10.1371/journal.pbio.3000649 PMID: 32231352

40. Bubeck D, Filman DJ, Cheng N, Steven AC, Hogle JM, Belnap DM. The structure of the poliovirus 135S cell entry intermediate at 10-angstrom resolution reveals the location of an externalized polypeptide that

PLOS PATHOGENS

binds to membranes. Journal of Virology. American Society for Microbiology; 2005; 79: 7745–7755. https://doi.org/10.1128/JVI.79.12.7745-7755.2005 PMID: 15919927

41. Kaplan G, Freistadt MS, Racaniello VR. Neutralization of poliovirus by cell receptors expressed in insect cells. Journal of Virology. American Society for Microbiology (ASM); 1990; 64: 4697–4702. https://doi.org/10.1128/JVI.64.10.4697-4702.1990 PMID: 2168959

42. Gómez Yafal AG, Kaplan G, Racaniello VR, Hogle JM. Characterization of poliovirus conformational alteration mediated by soluble cell receptors. Virology. 1993; 197: 501–505. https://doi.org/10.1006/viro.1993.1621 PMID: 8212594

43. Wetz K, Kucinski T. Influence of different ionic and pH environments on structural alterations of poliovirus and their possible relation to virus uncoating. Journal of General Virology. Microbiology Society; 1991; 72: 2541–2544. https://doi.org/10.1099/0022-1317-72-10-2541 PMID: 1655960

44. Tsang SK, McDermott BM, Racaniello VR, Hogle JM. Kinetic analysis of the effect of poliovirus receptor on viral uncoating: the receptor as a catalyst. Journal of Virology. American Society for Microbiology Journals; 2001; 75: 4984–4989. https://doi.org/10.1128/JVI.75.11.4984-4989.2001 PMID: 11333877

45. Panjwani A, Strauss M, Gold S, Wenham H, Jackson T, Chou JJ, et al. Capsid protein VP4 of human rhinovirus induces membrane permeability by the formation of a size-selective multimeric pore. Rey FA, editor. PLoS Pathog. Public Library of Science; 2014; 10: e1004294. https://doi.org/10.1371/journal.ppat.1004294 PMID: 25102288

46. Tosteson MT, Chow M. Characterization of the ion channels formed by poliovirus in planar lipid membranes. Journal of Virology. American Society for Microbiology; 1997; 71: 507–511. https://doi.org/10.1128/JVI.71.1.507-511.1997 PMID: 8985378

47. Danthi P, Tosteson M, Li Q-H, Chow M. Genome Delivery and Ion Channel Properties Are Altered in VP4 Mutants of Poliovirus. Journal of Virology. American Society for Microbiology Journals; 2003; 77: 5266–5274. https://doi.org/10.1128/jvi.77.9.5266-5274.2003 PMID: 12692228

48. Kirkegaard K. Mutations in Vp1 of Poliovirus Specifically Affect Both Encapsulation and Release of Viral RNA. Journal of Virology. American Society for Microbiology (ASM); 1990; 64: 195–206. https://doi.org/10.1128/JVI.64.1.195-206.1990 PMID: 2152812

49. Brandenburg B, Lee LY, Lakadamyali M, Rust MJ, Zhuang X, Hogle JM. Imaging Poliovirus Entry in Live Cells. Sugden B, editor. Plos Biol. Public Library of Science; 2007; 5: e183. https://doi.org/10.1371/journal.pbio.0050183 PMID: 17622193

50. McGregor S, Mayor HD. Biophysical and biochemical studies on rhinovirus and poliovirus. II. Chemical and hydrodynamic analysis of the rhinovirion. Journal of Virology. American Society for Microbiology (ASM); 1971; 7: 41–46. https://doi.org/10.1128/JVI.7.1.41-46.1971 PMID: 5543433

51. Zivanov J, Nakane T, Forsberg BO, Kimanius D, Hagen WJ, Lindahl E, et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife Sciences. 2018; 7: 163. https://doi.org/10.7554/eLife.42166 PMID: 30412051

52. Zivanov J, Nakane T, Scheres SHW. A Bayesian approach to beam-induced motion correction in cryo-EM single-particle analysis. IUCrJ. 2019; 6: 5–17. https://doi.org/10.1107/S205225251801463X PMID: 30713699

53. Zhang K. Gctf: Real-time CTF determination and correction. Journal of Structural Biology. 2016; 193: 1–12. https://doi.org/10.1016/j.jsb.2015.11.003 PMID: 26592709

54. Wagner T, Merino F, Stabrin M, Moriya T, Antoni C, Apelbaum A, et al. SPHIRE-crYOLO is a fast and accurate fully automated particle picker for cryo-EM. Commun Biol. 2019; 2: 218. https://doi.org/10.1038/s42003-019-0437-z PMID: 31240256

55. Frank J, Radermacher M, Penczek P, Zhu J, Li Y, Ladjadj M, et al. SPIDER and WEB: processing and visualization of images in 3D electron microscopy and related fields. Journal of Structural Biology. 1996; 116: 190–199. https://doi.org/10.1006/jsbi.1996.0030 PMID: 8742743

56. Strauss M, Schotte L, Thys B, Filman DJ, Hogle JM. Five of Five VHHs Neutralizing Poliovirus Bind the Receptor-Binding Site. Dermody TS, editor. Journal of Virology. American Society for Microbiology; 2016; 90: 3496–3505. https://doi.org/10.1128/JVI.03017-15 PMID: 26764003

57. Emsley P, Cowtan K. Coot: model-building tools for molecular graphics. Acta Crystallogr Sect D Biol Crystallogr. International Union of Crystallography; 2004; 60: 2126–2132. https://doi.org/10.1107/S0907444904019158 PMID: 15572765

58. Murshudov GN, Vagin AA, Dodson EJ. Refinement of macromolecular structures by the maximum-likelihood method. Acta Crystallogr Sect D Biol Crystallogr. International Union of Crystallography; 1997; 53: 240–255. https://doi.org/10.1107/S0907444996012255 PMID: 15299926

59. Afonine PV, Poon BK, Read RJ, Sobolev OV, Terwilliger TC, Urzhumtsev A, et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr D Struct Biol. International Union of Crystallography; 2018; 74: 531–544. https://doi.org/10.1107/S2059798318006551 PMID: 29872004

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 25/26

PLOS PATHOGENS                                                                                   Picornavirus entry-intermediate structure

60. Adams PD, Afonine PV, Bunkóczi G, Chen VB, Davis IW, Echols N, et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr Sect D Biol Crystallogr. International Union of Crystallography; 2010; 66: 213–221. https://doi.org/10.1107/S0907444909052925 PMID: 20124702

61. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, et al. UCSF Chimera?A visualization system for exploratory research and analysis. J Comput Chem. 2004; 25: 1605–1612. https://doi.org/10.1002/jcc.20084 PMID: 15264254

62. Goddard TD, Huang CC, Meng EC, Pettersen EF, Couch GS, Morris JH, et al. UCSF ChimeraX: Meeting modern challenges in visualization and analysis. Protein Science. 1998; 27: 14–25. https://doi.org/10.1002/pro.3235 PMID: 28710774

63. Morin A, Eisenbraun B, Key J, Sanschagrin PC, Timony MA, Ottaviano M, et al. Collaboration gets the most out of software. eLife Sciences. eLife Sciences Publications Limited; 2013; 2: e01456. https://doi.org/10.7554/eLife.01456 PMID: 24040512

PLOS Pathogens | https://doi.org/10.1371/journal.ppat.1008920 September 30, 2020 26/26
