
REVIEW

The structural basis of herpesvirus entry

Sarah A. Connolly<sup>1</sup>, Theodore S. Jardetzky<sup>2</sup> and Richard Longnecker<sup>3</sup>

Abstract | Herpesviruses are ubiquitous, double-stranded DNA, enveloped viruses that establish lifelong infections and cause a range of diseases. Entry into host cells requires binding of the virus to specific receptors, followed by the coordinated action of multiple viral entry glycoproteins to trigger membrane fusion. Although the core fusion machinery is conserved for all herpesviruses, each species uses distinct receptors and receptor-binding glycoproteins. Structural studies of the prototypical herpesviruses herpes simplex virus 1 (HSV-1), HSV-2, human cytomegalovirus (HCMV) and Epstein–Barr virus (EBV) entry glycoproteins have defined the interaction sites for glycoprotein complexes and receptors, and have revealed conformational changes that occur on receptor binding. Recent crystallography and electron microscopy studies have refined our model of herpesvirus entry into cells, clarifying both the conserved features and the unique features. In this Review, we discuss recent insights into herpesvirus entry by analysing the structures of entry glycoproteins, including the diverse receptor-binding glycoproteins (HSV-1 glycoprotein D (gD), EBV glycoprotein 42 (gp42) and HCMV gH–gL–gO trimer and gH–gL–UL128–UL130–UL131A pentamer), as well gH–gL and the fusion protein gB, which are conserved in all herpesviruses.

Enveloped viruses  
Viruses with an outer layer consisting of a lipid bilayer, in which the viral glycoproteins responsible for mediating virus entry into cells are embedded.

Conformational changes  
Changes in protein structure made possible by the intrinsic flexibility of the protein that can be triggered by environmental factors, such as binding to a receptor or another glycoprotein.

<sup>1</sup>Department of Health Sciences, DePaul University, Chicago, IL, USA.  
<sup>2</sup>Department of Structural Biology, Stanford University School of Medicine, Stanford University, CA, USA.  
<sup>3</sup>Department of Microbiology-Immunology, Feinberg School of Medicine, Northwestern University, Chicago, IL, USA.  

e-mail: sarah.connolly@dePaul.edu; r-longnecker@northwestern.edu  

https://doi.org/10.1038/s41579-020-00448-w  

The *Herpesviridae* are a family of large, double-stranded DNA, enveloped viruses that cause a range of diseases. The nine human herpesviruses are herpes simplex virus 1 (HSV-1), HSV-2, varicella zoster virus (VZV), human cytomegalovirus (HCMV), human herpesvirus 6A (HHV6A), HHV6B, Epstein–Barr virus (EBV), HHV7 and Kaposi sarcoma herpesvirus (KSHV). Herpesvirus infections are ubiquitous and establish lifelong latency in infected hosts.

Despite the viruses infecting a variety of cell types, entry into host cells occurs through a conserved mechanism. This Review focuses on structural studies of entry for prototypical viruses from each subfamily: HSV-1 and HSV-2 from the Alphaherpesvirinae, HCMV from the Betaherpesvirinae and EBV from the Gammaherpesvirinae. Extensive structural studies of these four viruses have been performed, and the structures of multiple entry glycoproteins have been resolved for each virus, allowing general features of herpesviruses to be discerned as well as virus-specific features.

All four herpesviruses infect a large proportion of the human population. HSV-1 and HSV-2 most commonly cause mucocutaneous lesions in the oral or genital regions, but infection also can cause encephalitis or meningitis under rare circumstances. HCMV infection during childhood is usually asymptomatic; however, primary infection later in life can result in serious complications for fetal development in pregnant women, for transplant recipients and for immunocompromised people. EBV typically causes infectious mononucleosis, but infection also can result in malignancies, including Burkitt lymphoma and Hodgkin lymphoma. An understanding of the entry mechanisms for these viruses may provide a basis for the design of antiviral drug candidates and/or subunit vaccines.

This Review explains the mechanism of herpesvirus entry into cells, as outlined in multiple structural studies of the entry glycoproteins required by each representative virus. The structures of the distinct receptor-binding proteins from each virus will be compared, including HSV-1 glycoprotein D (gD), EBV glycoprotein 42 (gp42) and the HCMV gH–gL–gO trimer and gH–gL–UL128–UL130–UL131A pentamer. Structures of the conserved entry glycoprotein heterodimer gH–gL and the fusion protein gB, examined by electron microscopy (EM) and crystallography, are also described.

These structural studies are linked to our understanding of the conformational changes and glycoprotein interactions required for entry and provide insight into how the fusion machinery drives virus entry into cells. These studies have broad implications because they provide targets to develop efficacious vaccines to prevent herpesvirus infections and associated disease.

Entry mechanism  
Herpesvirus entry into cells requires the coordinated interaction of multiple glycoproteins on the surface of the virion. The initial attachment of a virus to a host

cell tethers the virus to the cell, but does not trigger entry. This attachment is mediated by multiple viral glycoproteins and a variety of binding receptors. This Review focuses on the required entry events that occur after this attachment, including interactions among the glycoproteins and entry receptors that trigger membrane fusion and virus entry.

For herpesviruses, entry receptor binding and membrane fusion functions are performed by multifunctional viral glycoproteins ${ }^{1-3}$. The heterodimer gH–gL and the viral fusion protein gB represent a core set of entry glycoproteins that are required for all herpesviruses. By contrast, different herpesvirus subfamilies use distinct viral glycoprotein combinations to bind to various entry receptors. Even within a single virus species, different receptor-binding glycoprotein complexes may be required to mediate entry into different cell types.

In the current model of entry, binding to an entry receptor triggers conformational changes in the viral glycoproteins that signal to gB, the fusion protein, to execute membrane fusion (FIG. 1). Interactions among some of the entry glycoproteins, especially interactions with gB, have been difficult to capture, potentially because they are transient and/or of low affinity.

For HSV-1, gD serves the receptor-binding function, as described later (TABLE 1). gD binding to the receptor prompts an interaction between gD and gH–gL. gH–gL serves as a regulator of fusion that transmits a signal to gB. On triggering, gB undergoes a conformational change that results in insertion into the host cell membrane, followed by refolding to bring the cell and viral membranes together. The refolding of multiple gB trimers creates a pore in the membrane, allowing the viral capsid to enter the cellular cytoplasm and to be transported to the nucleus.

EBV enters cells in a similar manner, except that gp42 serves as the receptor-binding protein for EBV entry into B cells. gp42 forms a stable complex with gH–gL, and binding of the gp42–gH–gL complex to the receptor signals gB to mediate fusion. By contrast, for EBV entry into epithelial cells, gH–gL binds to the receptor directly before signalling gB to trigger fusion.

For HCMV, receptor binding is mediated by two distinct complexes: a trimeric complex including gO and gH–gL or a pentameric complex including UL128, UL130, UL131A and gH–gL. As for HSV-1 and EBV, binding of the HCMV trimer or pentamer to the receptor transmits a signal to gB to trigger fusion.

### Cell tropism

Herpesviruses exhibit broad cell tropism, and the routes of entry can depend on the cell type and/or the viral determinants. During infection, HSV-1 and HSV-2 typically infect epithelial cells and neurons; however, the virus can infect a wide range of cells, including fibroblasts and lymphocytes. Entry into epithelial cells occurs by low pH-dependent fusion with the endosomal membrane, whereas entry into neurons occurs by fusion at the plasma membrane ${ }^{5}$. HCMV also infects a variety of cells, including epithelial cells, endothelial cells, fibroblasts and leukocytes ${ }^{6}$. Entry into epithelial and endothelial cells requires endocytosis and low pH ${ }^{7}$, whereas fusion with fibroblasts occurs at the plasma membrane ${ }^{8}$. EBV infects mainly B cells and epithelial cells but can infect other cells such as monocytes ${ }^{9}$. B cell entry occurs after endocytosis in a pH-independent manner, whereas epithelial cell entry occurs after fusion with the plasma membrane ${ }^{10}$. Despite adapting to infect a broad range of host cells, herpesviruses share a common entry mechanism with conserved core fusion machinery (gH–gL and gB) and divergent receptor-binding proteins.

#### Receptor-binding protein structures

Herpesviruses encode distinct receptor-binding proteins that influence cell tropism but are not conserved among virus subfamilies, including gD for HSV-1 and HSV-2, gp42 for EBV, and gO and UL128–UL130–UL131A for HCMV (TABLE 1). Crystal structures of gD and gp42 have been resolved alone and in complex with their receptors ${ }^{11-15}$. A structure of gp42 bound to gH–gL has also been determined ${ }^{16}$. A complex of gH–gL with UL128–UL130–UL131A has been crystallized ${ }^{17}$, and a complex of gH–gL with gO has been visualized at low resolution by EM ${ }^{18}$. Receptors bound to both HCMV gH–gL complexes also have been observed by EM ${ }^{19,20}$. Although many binding receptors have been identified for these viruses and structures such as that of EBV gp350 (REF. ${ }^{21}$) have been determined, this Review focuses on validated entry receptors that have been examined in structural studies.

##### HSV gD

gD serves as the required receptor-binding protein for most alphaherpesviruses, with the notable exception of VZV, which lacks gD ${ }^{22}$. gD binds to three classes of receptors, including nectins, herpesvirus entry mediator (HVEM) and a modified form of heparan sulfate. Nectin 1, a cell adhesion protein, functions as an entry receptor for most alphaherpesviruses and is expressed on a variety of tissues, including neurons, a crucial cell type for alphaherpesviruses ${ }^{23}$. Nectin 2 mediates entry of some HSV-1 and HSV-2 strains, and the related poliovirus receptor can serve as an entry receptor for other alphaherpesviruses ${ }^{24,25}$. HVEM, a member of the tumour necrosis factor receptor (TNFR) family, is expressed primarily on immune cells and functions as a receptor for HSV-1 and HSV-2 but not other alphaherpesviruses ${ }^{26}$. 3-O-sulfonated derivatives of heparan sulfate (3-OST HS) can also serve as entry receptors for HSV-1 (REF. ${ }^{27}$).

Crystal structures of gD from HSV-1 (REFS ${ }^{12,13}$), HSV-2 (REF. ${ }^{28}$) and pseudorabies virus (PRV) ${ }^{29}$ show that the gD core adopts an immunoglobulin fold flanked by N-terminal and C-terminal extensions (FIG. 2). Structures of gD bound to its receptor reveal that gD binds to the membrane-distal portion of both HVEM and nectin 1 (REFS ${ }^{11,13,29,30}$). HVEM binds to an N-terminal loop on HSV-1 gD that is stabilized only in the presence of HVEM ${ }^{13}$. In the absence of the receptor, the gD N terminus is disordered. This gD N terminus is not conserved in bovine herpesvirus and PRV, consistent with the inability of gD from these alphaherpesviruses to bind HVEM ${ }^{31}$. Nectin 1 binds to gD at a distinct but overlapping site, adjacent to the HVEM-binding residues ${ }^{11,13,30,32,33}$. 3-OST HS also binds to gD and, although it has not been crystallized, mutational analysis

suggests that the 3-OST HS-binding site on gD also overlaps with that of HVEM$^{27,34}$.

The structure of a dimeric form of HSV-1 gD, artificially stabilized by an engineered intermolecular

Before fusion                      Receptor binding                     Fusion triggering                    After fusion

a HSV-1 and HSV-2

![Image](image1.png)

b EBV in epithelial cells

![Image](image2.png)

c EBV in B cells

![Image](image3.png)

d HCMV pentamer

![Image](image4.png)

e HCMV trimer

![Image](image5.png)

Fig. 1 | Model of the herpesvirus entry mechanism. **a** | Herpes simplex viruses 1 and 2 (HSV-1 and HSV-2) fuse with a host cell at the plasma or endosomal membrane. The glycoprotein D (gD) dimer (pink), gH–gL heterodimer (dark and light blue) and gB trimer (green) are necessary and sufficient for entry (column 1). gD binds to one of several entry receptors, including nectin 1 (grey; column 2). Receptor binding displaces the C terminus of the gD ectodomain and transmits a signal to gH–gL (small arrow). gH–gL activates the fusion protein gB (small arrow) to insert hydrophobic fusion loops into the cell membrane. **b** | Epstein–Barr virus (EBV) fuses with the plasma membrane of an epithelial cell. gH–gL (blue) and gB (green) are sufficient for fusion (column 1). The binding of gH–gL to ephrin type A receptor 2 (EphA2; grey) triggers gB to insert itself into the host cell (column 2). **c** | EBV fusion with B cells occurs in the endosome. A complex of glycoprotein 42 (gp42; pink) and gH–gL (blue) binds to human leukocyte antigen (HLA) class II (grey). The binding triggers gB and may impact membrane orientation (column 2). **d** | Human cytomegalovirus (HCMV) entry into epithelial and endothelial cells occurs after endocytosis and requires a pentamer complex of gH–gL (blue) bound to UL128–UL130–UL131A (shades of pink) (column 1). Pentamer binding to neuropilin 2 (NRP2) triggers gB (column 2). **e** | HCMV entry into all cells requires a trimer complex comprising gO and gH–gL (column 1). In fibroblasts, the trimer binds to platelet-derived growth factor receptor-α (PDGFRα) and triggers gB at the plasma membrane (column 2). The reason that the trimer is required for entry into epithelial and endothelial cells is currently unclear. After inserting itself into the target cell membrane, gB folds back on itself (column 3). Fusion most likely requires more than one gB trimer to be triggered. The other entry glycoproteins are not included in the figure for clarity. As gB refolds into its postfusion conformation, the viral and cell membranes are fused, creating a fusion pore, through which the viral capsid can enter the cell (column 4).

REVIEWS

Table 1 | Herpesvirus receptor-binding glycoproteins and receptors

| Subfamily           | Virus                          | Receptor-binding proteins          | Receptors                                                                 |
|---------------------|--------------------------------|-----------------------------------|--------------------------------------------------------------------------|
| Alphaherpesvirinae | Herpes simplex viruses 1 and 2 | gD                                | Herpesvirus entry mediator, nectin 1 and 3-O-sulfated heparan sulfate |
|                     | Varicella zoster virus        | gH–gL                             | Integrins                                                                |
| Betaherpesvirinae  | Human cytomegalovirus          | gH–gL–UL128–UL130–UL131A (pentamer) | Neuropilin 2                                                             |
|                     |                                | gH–gL–gO (trimer)                 | Platelet-derived growth factor receptor-α                               |
|                     | Human herpesviruses 6A and 6B | gH–gL–gQ1–gQ2                    | CD46 (for human herpesvirus 6A) and CD134 (for human herpesvirus 6B)   |
|                     |                                | gH–gL–gO                         | Unknown                                                                  |
| Gammaherpesvirinae | Epstein–Barr virus             | gH–gL–gp42                       | HLA class II                                                             |
|                     |                                | gH–gL                            | EphA2                                                                    |
|                     | Kaposi sarcoma herpesvirus    | gH–gL                            | EphA2 and EphA4                                                          |

EphA, ephrin type A receptor; g, glycoprotein; gp, glycoprotein; HLA, human leukocyte antigen.

and epithelial cells ${ }^{40}$, suggesting that these peptides may occlude a binding site on gH–gL for the epithelial receptor (see later). The gp42 N terminus anchors the gp42 C-terminal domain to gH–gL; however, the interaction between the gp42 C-terminal domain and gH–gL is not extensive. gH–gL contacts three residues within a hydrophobic pocket in the gp42 C-terminal domain, located at the canonical C-type lectin domain ligand-binding site. Amino acid substitutions in this gp42 hydrophobic pocket inhibit fusion without preventing gH–gL or HLA binding ${ }^{41}$, suggesting a functional role for the hydrophobic pocket. The binding of gp42 to gH–gL positions the HLA-binding site near the middle of the gH–gL–gp42 complex, in contrast to the arrangement seen for HCMV gH–gL complexes (see later).

EM reconstructions of gp42 simultaneously bound to HLA and gH–gL reveal multiple conformations consisting of open and closed forms ${ }^{38}$. gp42 bridges HLA and gH–gL, creating an approximately parallel orientation of gH–gL and HLA in the closed conformation and a highly variable orientation in open conformations. The transmembrane anchors of gH (linked to the virus) and HLA (linked to the cell) would be located on the same side of this complex in the closed conformation, suggesting that binding to HLA may impact the arrangement of the juxtaposed membranes during fusion.

site ${ }^{11,12,30}$ (FIG. 2). Thus, a conformational change must occur on receptor binding to displace this C-terminal gD extension. Truncation of the C terminus enhances gD binding to both HVEM and nectin 1 (REFS ${ }^{35,36}$). This conformational change may serve as a downstream signal for fusion. A panel of gD mutants designed to lock the C-terminal extension to the gD core at different sites successfully generated a mutant that retains the ability to bind to the receptor but fails to trigger fusion, demonstrating that movement of the gD C-terminal extension contributes the fusion signal beyond simply permitting receptor binding ${ }^{37}$. The C terminus of the gD ectodomain may serve as an interaction site for gH–gL or its movement may expose a gH–gL interaction site on gD (see later).

**EBV gp42.** Although EBV entry into epithelial cells requires only gH–gL and gB, gp42 serves as a required receptor-binding protein for EBV entry into B cells. gp42 binds to human leukocyte antigen (HLA) class II, a member of the C-type lectin family. Crystal structures have been determined for gp42 alone, gp42 bound to HLA class II and gp42 complexed with gH–gL plus the anti-gH E1D1 nAb (REFS ${ }^{14-16}$) (FIG. 3). By EM, the gp42–gH–gL–HLA complex also has been visualized ${ }^{38}$.

gp42 is a type II membrane protein that must be cleaved at the N-terminal transmembrane domain to generate a functional soluble form ${ }^{39}$. The C-terminal domain of gp42 adopts a C-type lectin domain fold and binds to the HLA class II receptor ${ }^{12}$. When gp42 is complexed with gH–gL, the N terminus of gp42 extends across gH, interacting with three gH domains ${ }^{16}$ (FIG. 4). Peptides derived from the gp42 N-terminal region can bind to gH–gL and inhibit entry into both B cells

HCMV $g O$ and UL128–UL130–UL131A. HCMV entry into all cells, including fibroblast, epithelial and endothelial cells, requires gH–gL in a trimeric complex with gO ${ }^{20,42}$. Infection of epithelial and endothelial cells also requires gH–gL to form a pentameric complex with UL128, UL130 and UL131A ${ }^{7,43}$. Nearly all of the gH–gL in the virion is complexed with either gO or UL128–UL130–UL131A ${ }^{44}$, and the formation of these complexes is mutually exclusive. Residue Cys144 in gL forms a disulfide bond with UL128 in the pentamer and with gO in the trimer ${ }^{18}$.

The gH–gL complexes bind to different receptors, although all of these have not been identified yet. The trimer binds to platelet-derived growth factor receptor-α (PDGFRα) to mediate entry into fibroblasts, whereas the pentamer binds to neuropilin 2 (NRP2) to mediate entry into epithelial and endothelial cells ${ }^{19,20,45}$. OR14I1 was identified recently as an additional epithelial cell receptor that mediates HCMV pentamer-dependent attachment and infection ${ }^{46}$, and CD147 was also found to promote pentamer-dependent entry, although direct binding did not occur ${ }^{47}$. In addition, the trimer is thought to interact with another, as yet unidentified receptor on epithelial and endothelial cells ${ }^{48}$.

The crystal structure of the pentamer bound to an antibody shows that UL128–UL130–UL131A assembles onto the N terminus of gH–gL, docking at an N-terminal extension of gL that is conserved in betaherpesviruses but not present in other subfamilies ${ }^{17}$ (FIG. 4a). The limited interface between UL128–UL130–UL131A and gH–gL permits flexibility, which was observed experimentally ${ }^{17}$. Although the gH–gL–gO trimer has not been determined by crystallography, it has been visualized by EM. Similarly to UL128–UL130–UL131A, gO also docks at

the tip of gH–gL ${}^{18,20}$ (FIG. 4b). This N-terminal gH–gL docking site differs from the gp42-docking site in the middle of gH–gL, suggesting that distinct arrangements of receptor complexes are capable of activating gH–gL between the subfamilies.

Low-resolution structures of the trimer bound to PDGFRα ${}^{20}$ and the pentamer bound to NRP2 receptor ${}^{19}$ have been visualized by EM (FIG. 4c,d). The receptors appear to interact nearly exclusively with gO or UL128–UL130–UL131A. Crosslinking analysis confirms that NRP2 contacts residues in gL, UL128, UL130 and UL131A ${}^{19}$.

gH and gL

The gH–gL heterodimer is an essential component of the fusion machinery that is conserved among all herpesviruses. gH–gL is proposed to regulate fusion by interacting with gB. gH–gL serves a receptor-binding function for many herpesviruses; however, receptor binding by gH–gL does not appear to be required for most alphaherpesviruses. Instead, gH–gL is proposed to relay a receptor-binding signal from gD to gB in alpha-herpesviruses. For some herpesviruses, gH–gL alone acts directly as a receptor-binding complex, as is observed for EBV entry into epithelial cells and VZV entry (TABLE 1). In other instances, gH–gL forms a stable complex with other viral proteins to bind to entry receptors, as described earlier for EBV and HCMV.

EBV entry into epithelial cells is mediated by gH–gL binding directly to ephrin type A receptor 2 (EphA2) ${}^{49}$. Similarly, KSHV entry is mediated by gH–gL binding to EphA2 or EphA4 (REFS 50–52). Soluble gp42 is able to block EphA2-mediated fusion ${}^{49}$ and gp42 N-terminal peptides can block EBV entry into epithelial cells, suggesting that the gp42-binding site may overlap with the epithelial receptor-binding site on gH–gL or alternatively with a site that is crucial for gB activation in epithelial cell entry ${}^{16,40}$. The E1D1 antibody, which binds exclusively to gL at a distinct site at the distal end of gH–gL, partially inhibits epithelial cell entry but not B cell entry, suggesting that E1D1 also inhibits epithelial receptor binding or post-receptor-binding activation ${}^{16,53,54}$ (FIG. 3c). These findings indicate that multiple regions of gH–gL are important for epithelial cell entry.

VZV gH–gL is proposed to interact with integrins, as cell–cell fusion can be inhibited by adding an anti-αV integrin or knocking down αV integrin expression ${}^{55}$. Although not required for fusion, HSV gH–gL also binds to αvβ3, αvβ6 and αvβ8 integrins ${}^{56,57}$.

Crystal structures of gH–gL have been resolved for HSV-2 (REF. 58), VZV ${}^{59}$, PRV ${}^{60}$, EBV ${}^{61}$ and HCMV ${}^{17}$. The gH–gL ectodomain is composed of four domains that adopt a boot-shaped organization for HSV-2 and VZV, and a more rod-like shape for PRV and EBV (FIG. 5). The overall architecture of HCMV gH–gL is intermediate with respect to the two previously mentioned forms. gH contains a C-terminal transmembrane domain that anchors the complex to the membrane. gL lacks a transmembrane anchor. The gH–gL structure can be described in four progressive domains, although specific domain designations between the gH–gL structures can differ ${}^{62}$. Domain I is the most membrane-distal region, comprising gL and the N terminus of gH. Domain II includes β-sheets and α-helices in a parallel arrangement. The N-terminal domains of gH are the least conserved in sequence, consistent with this region interacting with other species-specific viral proteins (see earlier). The two membrane-proximal domains are more conserved, including the helices of domain III and the β-sandwich of domain IV.

Neutralizing monoclonal antibodies (nAbs) map to several regions of gH–gL (FIG. 5), consistent with a requirement for gH–gL to interact with multiple partners during virus entry, including receptors, receptor-binding proteins and gB. In HSV-1, selection of nAb resistance

Fig. 2 | Herpes simplex virus 1 glycoprotein D crystal structures. a | A monomer from a dimeric form of glycoprotein D (gD) that was stabilized by introduction of a disulfide bond at the C terminus of the ectodomain (Protein Data Bank (PDB) ID 2C36) ${}^{12}$. gD is oriented to show the receptor-binding face. The core immunoglobulin fold of gD (pale pink) is flanked by N-terminal (red) and C-terminal (blue) extensions. An α-helix (yellow) that supports the receptor-binding site is shown and the N terminus and the C terminus are marked. b | Herpesvirus entry mediator (HVEM) receptor (transparent surface rendering) bound to gD (PDB ID 1JMA) ${}^{13}$. The N terminus of gD (red) forms a loop that contains all of the contact residues for HVEM. c | Nectin 1 receptor (transparent surface rendering) bound to gD (PDB ID 3SKU) ${}^{11}$. The HVEM-binding site and the nectin 1-binding site overlap. Binding of either HVEM or nectin 1 would displace the gDC terminus.

REVIEW S

Fig. 3 | Epstein–Barr virus glycoprotein 42 structures. **a** | Crystal structure of glycoprotein 42 (gp42) alone (pink ribbons). The C-terminal C-type lectin domain (CTLD) is shown (Protein Data Bank (PDB) ID 3FD4) [15]. **b** | Crystal structure of gp42 (pink) bound to the human leukocyte antigen (HLA) class II receptor (green and orange surface rendering, partly shown) (PDB ID 1KG0) [14]. HLA binds to the gp42 CTLD and the gp42 N terminus is not resolved. HLA binding does not change the gp42 conformation drastically. **c** | Crystal structure of gp42–glycoprotein H (gH)–gL–E1D1 complex (PDB ID 5T1D) [16]. Although the CTLD of gp42 (pink ribbon with surface rendering) contacts gH–gL (blue and cyan ribbons, respectively), most contacts lie in the gp42 N-terminal extension that extends down the length of gH–gL. Peptides from this gp42 N terminus can inhibit entry into cells. E1D1 (grey surface rendering, partly shown) is a monoclonal antibody (mAb) that binds to gL and partially neutralizes entry into epithelial cells but not B cells. **d** | Electron microscopy reconstructions of gp42–gH–gL–HLA complexes (grey surface). The fits inside the densities are the crystal structures of gH–gL (blue and cyan), gp42 (pink) and HLA (green and orange). Two conformations were observed: closed and open. In the closed conformation, HLA and gH–gL are arranged in a more parallel orientation than in the open conformation. Part **d** is adapted from REF. [38], CC BY 4.0 (<https://creativecommons.org/licenses/by/4.0/>).

mutations indicates that the nAb LP11 binds to domain II of gH–gL, whereas the nAb 52S binds to domain III on the opposite face of the complex [58,63]. In HCMV, EM reconstructions show the nAbs 3G16 and 13H11 binding to distinct sites on gH–gL, neither of which interferes with the docking of gO or UL128–UL130–UL131A [20,64]. 3G16 binds at the membrane-proximal end of gH–gL, whereas 13H11 binds at the middle of gH–gL. In EBV, two nAbs (CL40 and CL59) inhibit entry into all cell types and also map to separate sites on gH–gL [65]. A crystal structure of CL40 bound to gH–gL shows this nAb at the middle of gH–gL, overlapping the site of the gp42 C-terminal domain, in a location analogous to that of 13H11. This CL40-binding site partially overlaps with the binding site for another nAb (AMMO1), which blocks entry into both B cells and epithelial cells [66]. By contrast, an EM reconstruction of CL59 bound to gH–gL places this nAb at domains III and IV, in a position analogous to 3G16 (REF. [65]).

The gH cytoplasmic tail domain is short, ranging from 8 residues in EBV to 19 residues in PRV, but it contributes substantially to entry. Deletion or mutation of gH cytoplasmic tail residues inhibits fusion for multiple species [67–74] and enhances fusion for VZV [72]. When the gH–gL ectodomain is expressed in cells as a soluble protein or as a glycosylphosphatidylinositol-anchored form, cell–cell fusion either fails [73,75,76] or occurs at only low levels [4]. Some mutations in the gH cytoplasmic tail inhibit fusion without detectable changes to gH–gL ectodomain conformation or expression [69,71,74], suggesting that an intracellular interaction, potentially between the cytoplasmic tails of gH and gB, may contribute to fusion regulation [71,77]. Some EBV gH cytoplasmic tail truncation mutants result in decreased gH–gL binding to gp42, suggesting that the gH cytoplasmic tail may also impact the gH ectodomain [74].

### The fusion protein gB

gB is the viral fusion protein, the glycoprotein responsible for insertion into the host cell membrane and refolding to drive fusion of the viral envelope and cell membrane. Viral fusion proteins are membrane-anchored glycoproteins that initially fold into a prefusion conformation [78]. On triggering by receptor binding and/or exposure to acidic pH, fusion proteins undergo a conformational change to insert hydrophobic residues into the host membrane and then fold back on themselves to bring the two membranes together.

Crystal structures of gB from all three herpesvirus subfamilies have been resolved, including from HSV-1 (REFS [77,79]), PRV [80,81], VZV [82], HCMV [83,84] and EBV [85] (FIG. 6a–c). These gB homologues adopt similar structures. Unexpectedly, gB resembles the fusion proteins from rhabdovirus [86,87], baculovirus [88] and thogotovirus [89], despite a lack of sequence homology between gB and the fusion proteins of these unrelated viruses. Together, these fusion proteins comprise the ‘class III’ fusion proteins [90].

gB is a trimeric single-pass transmembrane protein with five extracellular domains. nAbs map to multiple sites in the gB ectodomain [83,91–93], and mutations in each of the gB domains can prevent fusion [94,95], suggesting that

Fusion loops  
Short stretches of hydrophobic  
residues within a fusion protein  
that insert themselves into the  
host cell membrane during  
the fusion event.

multiple regions of gB participate in the conformational  
changes required for fusion and/or interactions with the  
proteins that trigger gB. The formation of a fusion pore  
likely requires multiple gB trimers. Lateral interactions  
among gB trimers have been observed by EM and may  
contribute to the expansion of the fusion pore $^{96}$.

Receptors that bind to gB and mediate entry have  
been reported, such as paired immunoglobulin-like  
type 2 receptor-α $^{97}$, myelin-associated glycoprotein $^{98}$  
and myosin 9 (REF. $^{99}$). Triggering fusion by binding to gB  
directly departs from the current model of virus entry,  
and future structural studies of these receptors may  
explain how they enhance entry.

**Postfusion gB structure.** Crystal structures of the rhabdo-  
virus vesicular stomatitis virus (VSV) fusion protein G  
have been resolved in both the prefusion conformation $^{100}$  
and the postfusion conformation $^{86}$. All of the gB  
crystal structures resolved thus far resemble the postfu-  
sion form of VSV G. Removal of the gB transmembrane  
and cytoplasmic tail regions appears to destabilize gB,  
preventing the crystallization of a prefusion form $^{77,79}$.  
gB folds into a postfusion form even when mutations  

that reduce fusion are introduced $^{101-103}$, when the trans-  
membrane region is replaced $^{102}$ or when the membrane is  
disrupted with detergent $^{77,79}$. Antibodies specific for the  
prefusion conformation that might facilitate stabiliza-  
tion of a prefusion form have not been identified. All the  
antibodies that have been characterized, including nAbs,  
appear to be able to bind to the postfusion form $^{91,93}$.

Domain I lies at the base of the molecule in the  
postfusion structure and contains two fusion loops with  
hydrophobic residues (FIG. $6a-c$). These fusion loops are  
thought to insert themselves into the host cell mem-  
brane during fusion $^{104}$, and mutations within the loops  
demonstrate that they are required for fusion func-  
tion $^{150,105-107}$. Domain II contains a pleckstrin homology  
domain. Extended linker regions that lead into and out  
of domains I and II may allow large-scale conforma-  
tional rearrangements during fusion. Domain III forms  
an extended trimer of helices that forms the core of  
the structure. Domain IV forms a crown at the top  
of the structure. Domain V extends down the length of  
the molecule, packing against the helices of domain III  
in an antiparallel orientation that is reminiscent of  
the hallmark ‘six-helix bundle’ structure present in the

Fig. 4 | Structures of human cytomegalovirus glycoprotein H–glycoprotein L complexes. **a** | Crystal structure of  
pentamer bound to monoclonal antibody (mAb) 8l21 (grey surface rendering). UL128 (green), UL130 (orange) and UL131A  
(dark blue) assemble on the distal tip of glycoprotein H (gH)–gL (light blue and pink, respectively), contacting an N-terminal  
extension of gL that is unique to betaherpesviruses (Protein Data Bank ID 5V0B)$^{17}$. **b** | Electron microscopy (EM) reconstruction  
of gH–gL–gO trimer bound to mAb 3G16 (Electron Microscopy Data Bank entry EMD-6431). The gH–gL (blue and pink) and  
mAb (grey) structures have been fit inside the density. gO (green) maps to the distal end of gH–gL, in a position analogous to  
UL128–UL130–UL131A. **c** | EM reconstruction of pentamer (coloured as in part $\mathbf{a}$) bound to mAb 3G16 (grey) and neuropilin 2  
(NRP2) receptor (red) (Electron Microscopy Data Bank entry EMD-8884). Crystal structures have been fit inside the EM  
density and NRP2 maps to the distal tip of the complex, contacting the UL128–UL130–UL131A–gL portion of the complex.  
**d** | EM image of trimer bound to the receptor platelet-derived growth factor receptor-α (PDGFRα). A comparison of gH–gL–gO  
with and without the receptor demonstrates that the receptor binds at the gO side of the complex, outlined in red.  
Part $\mathbf{b}$ is adapted from REF.$^{64}$, CC BY 4.0 ([https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/)). Part $\mathbf{c}$ is adapted with permission  
from REF.$^{19}$, Elsevier. Part $\mathbf{d}$ is adapted from REF.$^{20}$, Springer Nature Limited.

REVIEWs

postfusion form of class I fusion proteins<sup>108</sup>. The formation of a stable six-helix bundle structure during fusion is thought to contribute energy for fusion pore formation<sup>109</sup>. Mutations in domain V of gB designed to weaken its packing against domain III result in reduced fusion and a small plaque phenotype<sup>110,111</sup>, suggesting that the interaction of domains III and V may also contribute to the energetics of fusion.

As seen for class I fusion proteins<sup>112,113</sup>, the cytoplasmic tail domain of gB regulates fusion. The mechanism for this regulation is unclear; however, interprotomer interactions among the cytoplasmic tails or between the cytoplasmic tails and the membrane may stabilize the prefusion state of the gB ectodomain, preventing premature activation<sup>77</sup>. Mutations in the gB cytoplasmic tail can inhibit or enhance fusion, depending on the specific mutation<sup>114–119</sup>. The crystal structure of HSV-1 gB including its transmembrane domain and cytoplasmic tail shows that the cytoplasmic tail forms a trimer that interacts with the membrane<sup>77</sup> (FIG. 6a). The gB cytoplasmic tail may also interact with the gH cytoplasmic tail (see later)<sup>71</sup>. Among class III fusion proteins, fusion

regulation by the cytoplasmic tail may be unique to the herpesviruses because the VSV and baculovirus fusion protein cytoplasmic tail domains are substantially shorter than the gB cytoplasmic tail<sup>120</sup>.

**Prefusion gB models.** Computational homology models of the prefusion gB of HSV and EBV have been created on the basis of the prefusion VSV G crystal structure<sup>85,121</sup>. These models suggest that gB undergoes substantial refolding to transition to postfusion.

More recently, alternative conformations of full-length gB anchored in a membrane have been studied by cryo-electron tomography (cryo-ET) <sup>103,122–124</sup> (FIG. 6d–f). These cryo-ET reconstructions reveal compact trimers that are shorter than postfusion gB (9–13 nm versus 16–18 nm in height). These structures may represent prefusion or intermediate conformations of gB. When gB domains from the crystal structures were fit into the cryo-ET reconstructions, the resulting models orient the fusion loops towards the membrane<sup>103,123,124</sup>. The postfusion conformation of domain III cannot be fit into the cryo-ET reconstructions, suggesting that

---

**Fig. 5 | Crystal structures of the glycoprotein H–glycoprotein L complex.**

**a** Herpes simplex virus 2 (HSV-2) glycoprotein H (gH)–gL (Protein Data Bank (PDB) ID 3M1C)<sup>58</sup>. Four progressive gH domains (DI–DIV) and gL are shown. gH DI (red) is intimately associated with gL (blue). gL requires gH for proper folding and anchoring to the membrane. DII (green) includes parallel β-sheets and helices. DIII (yellow) is mostly helical, and DIV (orange) includes a β-sandwich. Although domain designations between the gH–gL structures can differ, specifically for distinction between DII and DIII, this figure uses the domain designations identified for Epstein–Barr virus (EBV) gH–gL. The C terminus would extend from DIV into the transmembrane domain. The overall complex adopts a boot-like shape. Substitutions at HSV-1 gH residues 168 or 329 (magenta spheres) prevent binding of the neutralizing monoclonal antibody (nAb) LP11. On the opposite face, substitutions at gH residue 536 or 537 (cyan spheres) prevent binding of the nAb 52S<sup>63</sup>.

**b** EBV gH–gL (PDB ID 3PHF)<sup>61</sup>. The domains are coloured as in part **a** and they adopt a more linear orientation. Co-crystallization revealed that nAb CL40 contacts gH residues 184, 239, 243, 284 and 286 (magenta spheres) and overlaps the binding site for the gp42 C-terminal domain. Electron microscopy reconstruction of bound nAb CL59 shows that this nAb binds to a distinct site within DIII and DIV (cyan ribbons; gH residues 406–415, 456–468, 494–503, 568–577, 623–626 and 645–656)<sup>65</sup>. The CL40 epitope partially overlaps with the nAb AMMO1-binding site (grey spheres; gH residues 73 and 76)<sup>67</sup>.

**c** Human cytomegalovirus (HCMV) gH–gL (PDB ID 5VOC)<sup>17</sup>. Domains are coloured as in part **a**. HCMV gL includes an N-terminal extension that is absent from the other structures. Hydrogen–deuterium exchange coupled with mass spectrometry maps nAb 13H11 binding to gH residues 238–247 (magenta ribbons) and nAb 3G16 binding to gH residues 677–684 and 705–708 (cyan ribbons)<sup>64</sup>.

www.nature.com/nrmicro

Fig. 6 | Glycoprotein B structures.  
a | Crystal structure of full-length postfusion herpes simplex virus 1 (HSV-1) glycoprotein B (gB) (Protein Data Bank (PDB) ID 5V2S). The trimeric ectodomain comprises five domains (DI–DV). DI (blue) contains hydrophobic fusion loops (magenta sticks) that insert themselves into the host cell membrane. DIII (yellow) includes an extended central trimeric coiled coil, against which DV (red) packs in an antiparallel orientation, as DV proceeds through DI, towards the transmembrane domain. The membrane-proximal region (dark green), transmembrane domain (dark purple) and cytoplasmic tail domain (pink) are shown.  
b | Crystal structure of postfusion Epstein–Barr (EBV) gB ectodomain (PDB ID 3FVC).  
c | Crystal structure of postfusion human cytomegalovirus (HCMV) gB ectodomain (PDB ID 5CXF). For parts b and c, the domains are coloured as in part a. For crystallization, residues in the fusion loops of EBV and HCMV gB were replaced with HSV-1 residues.  
d | Cryo-electron tomography (cryo-ET) reconstruction of the compact conformation of a mutant form of HSV-1 gB, 12 nm in height. Density fitting of the gB domains from the postfusion gB structure is shown. Domains are coloured as in part a. The fusion loops of DI (light blue) are oriented towards the membrane. For comparison, the ectodomain of postfusion HSV-1 gB is shown at a similar scale on the left, at 16 nm in height.  
e | Alternative cryo-ET reconstruction of a compact form of HSV-1 gB, 9 nm in height. A pseudoatomic model of prefusion gB, based on the prefusion structure of vesicular stomatitis virus protein G (VSV G) (PDB ID 5I2S), was fit into the 3D electron microscopy reconstruction. Domains are coloured as in part a. DV is excluded from the model.  
f | Cryo-ET reconstruction of HCMV gB (Electron Microscopy Data Bank entry EMD-9328), 13 nm in height. Domains from a single protomer are coloured as in part a. Domain fitting using the postfusion HCMV structure placed DI (blue) near the membrane, with the fusion loops (magenta) oriented towards the membrane. As the electron microscopy reconstruction cannot accommodate the DIII postfusion conformation, DIII fit was modelled on VSV G. DV is excluded from the model. Part d is adapted with permission from REF., AAAS. Part e is adapted from REF., CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/). Part f is adapted from REF., CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/).

transition from this alternative compact form of gB to the postfusion form requires substantial structural rearrangement.

### Glycoprotein interactions

#### Interaction between HSV-1 and HSV-2 gD and gH–gL.
Although complexes of gH–gL and the receptor-binding proteins of HCMV and EBV have been examined structurally, a complex between the alphaherpesvirus gD and gH–gL has not been visualized. Biochemical and genetic studies suggest that they interact, perhaps transiently and/or with low affinity. gD and gH–gL can be crosslinked in the virion, and they are reported to co-precipitate. An interaction between gD and gH–gL has been observed by bimolecular fluorescence complementation (BiFC), and stabilization of the gD–gH–gL complex by the BiFC tags inhibits fusion, suggesting that the interaction between gD and gH–gL may need to be transient. A species-specific functional interaction between gD and gH–gL was shown by co-expression of combinations of entry glycoproteins from the non-complementing species HSV-1 and saimiriine herpesvirus 1 (SaHV1). gH–gL chimaeras

NATURE REVIEWS | MICROBIOLOGY

REVIEWS

**Interactions between gB and gH–gL.** By cryo-ET, an interaction between gB and a linear structure, presumed to be gH–gL, was visualized on HCMV virions [124]. The linear structure was frequently associated with a compact conformation of gB, which may represent prefusion gB, but not with the postfusion gB present in the membrane. Exclusive association with the prefusion form of gB is consistent with gH–gL stabilizing gB before the receptor-binding step that triggers fusion. Whether this gB complex forms with gH–gL trimers or pentamers, and how these proteins interact to coordinate virus entry, remains to be determined. This low-resolution structure suggests that gH–gL and gB may interact at a region adjacent to the membrane. A close association of the transmembrane domains is consistent with a proposed interaction between the gH and gB cytoplasmic tail domains [71,77]. The gB cytoplasmic tail may function as a clamp, preventing premature triggering of gB. In an alternative model, gH is proposed to act as a wedge, potentially triggering gB on receptor binding by disrupting gB cytoplasmic tail interprotomer interactions and/or gB cytoplasmic tail interactions with the membrane [77]. The proposed stabilizing and wedge functions of gH–gL are not mutually exclusive. In additional support for a functional interaction between the gH and gB cytoplasmic tail domains, the replacement of EBV gB cytoplasmic tail residues with those of rhesus lymphocryptovirus (rhLCV) confers partial fusion function with rhLCV gH–gL [74].

The ectodomains of gB and gH–gL are also proposed to interact. An interaction between gB and gH–gL detected by BiFC was blocked by nAbs that bind to the gB and gH–gL ectodomains [58,134], including nAb LP11 (FIG. 5a). Furthermore, gB and gH–gL from HSV-1 or HCMV can mediate low levels of entry when expressed on opposing membranes, as can a soluble form of the HSV-2 gH–gL ectodomain [4,135].

## Conclusions and perspectives

The complexity of herpesvirus entry into cells results from the broad range of receptor usage and cell tropism, and the requirement for multiple glycoproteins to mediate membrane fusion. The extensive structural studies described here clarify our understanding of the stepwise mechanism of virus entry and lay the groundwork for the development of antiviral interventions. The only herpesviruses vaccines currently available include a live attenuated VZV vaccine [136] and a VZV gE subunit vaccine [137]. Current antiviral medications for herpesviruses, such as acyclovir, target virus replication rather than the viral entry glycoproteins. Understanding the structural details of herpesvirus entry may aid the design of better vaccines, inhibitors or nAbs that target receptor binding, interactions among the entry glycoproteins, and/or the refolding of gB. The requirement of gH–gL and gB for all herpesviruses means that discovery of an inhibitor of one species could serve as a model for analogous inhibitors for other species. The entry glycoproteins generate nAbs, and the structural determination of the binding sites for these antibodies also provides candidate sites for the design of inhibitors.

Advances in EM imaging have provided 3D reconstructions of prefusion or intermediate conformations of gB [103,122–124], as well gB interacting with gH–gL [124]. For all of the herpesviruses, atomic resolution of the prefusion gB conformation and the gH–gL–gB complex remains to be determined. For HSV-1, the interaction between gD and gH–gL has not been visualized by any structural approach. For EBV, the structures of gH–gL bound to EphA2 and the gp42–HLA–gH–gL complex at atomic resolution remain unknown. For HCMV, the structures of the gH–gL–gO trimer and the gH–gL complexes bound to PDGFRα or NRP2 receptors at atomic resolution have not been determined.

Although biochemical and genetic approaches have provided models for the functional interactions among entry glycoproteins, future crystallography and cryo-ET studies will continue to identify the interaction sites and conformational changes required for virus-induced membrane fusion. These studies may be assisted by the use of antibodies and/or mutations that stabilize the complexes and/or transient conformations. Additional receptors for these viruses may be identified and characterized by structural methods as well.

Published online: 21 October 2020

1. Vallbracht, M., Backovic, M., Klupp, B. G., Rey, F. A. & Mettenleiter, T. C. Common characteristics and unique features: a comparison of the fusion machinery of the alphaherpesviruses Pseudorabies virus and Herpes simplex virus. *Adv. Virus Res.* **104**, 225–281 (2019).
2. Mohl, B. S., Chen, J. & Longnecker, R. Gammaherpesvirus entry and fusion: a tale how two human pathogenic viruses enter their host cells. *Adv. Virus Res.* **104**, 313–343 (2019).
3. Nishimura, M. & Mori, Y. Entry of betaherpesviruses. *Adv. Virus Res.* **104**, 283–312 (2019).
4. Atanasiu, D., Saw, W. T., Cohen, G. H. & Eisenberg, R. J. Cascade of events governing cell-cell fusion induced by herpes simplex virus glycoproteins gD, gH/gL, and gB. *J. Virol.* **84**, 12292–12299 (2010).
5. Nicola, A. V. Herpesvirus entry into host cells mediated by endosomal low pH. *Traffic* **17**, 965–975 (2016).
6. Gerna, G., Baldanti, F. & Revello, M. G. Pathogenesis of human cytomegalovirus infection and cellular targets. *Hum. Immunol.* **65**, 381–386 (2004).
7. Ryckman, B. J., Jarvis, M. A., Drummond, D. D., Nelson, J. A. & Johnson, D. C. Human cytomegalovirus entry into epithelial and endothelial cells depends on genes UL128 to UL150 and occurs by endocytosis and low-pH fusion. *J. Virol.* **80**, 710–722 (2006).

8. Compton, T., Nepomuceno, R. R. & Nowlin, D. M. Human cytomegalovirus penetrates host cells by pH-independent fusion at the cell surface. *Virology* **191**, 387–395 (1992).
9. Hutt-Fletcher, L. M. Epstein-Barr virus entry. *J. Virol.* **81**, 7825–7832 (2007).
10. Miller, N. & Hutt-Fletcher, L. M. Epstein-Barr virus enters B cells and epithelial cells by different routes. *J. Virol.* **66**, 3409–3414 (1992).
11. Di Giovine, P. et al. Structure of herpes simplex virus glycoprotein D bound to the human receptor nectin-1. *PLoS Pathog.* **7**, e1002277 (2011).
12. Krummenacher, C. et al. Structure of unliganded HSV gD reveals a mechanism for receptor-mediated activation of virus entry. *EMBO J.* **24**, 4144–4153 (2005). Compares the crystal structure of unbound HSV-1 gD with that of the previously determined gD–HVEM complex, revealing a conformational change that occurs on receptor-binding.
13. Carfi, A. et al. Herpes simplex virus glycoprotein D bound to the human receptor HveA. *Mol. Cell* **8**, 169–179 (2001).
14. Mullen, M. M., Haan, K. M., Longnecker, R. & Jardetzky, T. S. Structure of the Epstein-Barr virus gp42 protein bound to the MHC class II receptor HLA-DR1. *Mol. Cell* **9**, 375–385 (2002).

15. Kirschner, A. N., Sorem, J., Longnecker, R. & Jardetzky, T. S. Structure of Epstein-Barr virus glycoprotein 42 suggests a mechanism for triggering receptor-activated virus entry. *Structure* **17**, 223–233 (2009).
16. Sathiyamoorthy, K. et al. Structural basis for Epstein-Barr virus host cell tropism mediated by gp42 and gH/gL entry glycoproteins. *Nat. Commun.* **7**, 13557 (2016).
17. Chandramouli, S. et al. Structural basis for potent antibody-mediated neutralization of human cytomegalovirus. *Sci. Immunol.* https://doi.org/10.1126/sciimmunol.aan1457 (2017). Reports the crystal structure of the HCMV pentamer, demonstrating that HCMV gH/gL folds similarly to EBV gH–gL and UL128–UL130–UL131A binds to an extension in gL.
18. Ciferri, C. et al. Structural and biochemical studies of HCMV gH/gL and Pentamer reveal mutually exclusive cell entry complexes. *Proc. Natl Acad. Sci. USA* **112**, 1767–1772 (2015). EM reconstructions of the HCMV trimer and pentamer complexes demonstrate that both

gO and UL128–UL130–UL131A dock at the tip of gH–gL.

19. Martinez-Martin, N. et al. An unbiased screen for human cytomegalovirus identifies neuropilin-2 as a central viral receptor. *Cell* **174**, 1158–1171 e1119 (2018). Identifies NRP2 as an HCMV receptor and shows that this receptor binds to the distal end of the pentamer by EM reconstruction.
20. Kabanova, A. et al. Platelet-derived growth factor-alpha receptor is the cellular receptor for human cytomegalovirus gH/gL/gO trimer. *Nat. Microbiol.* **1**, 16082 (2016). Identifies PDGFRα as an HCMV receptor and shows that this receptor binds to the distal end of the trimer by EM.
21. Szakonyi, G. et al. Structure of the Epstein-Barr virus major envelope glycoprotein. *Nat. Struct. Mol. Biol.* **13**, 996–1001 (2006).
22. Oliver, S. L., Yang, E. & Arvin, A. M. Varicella-zoster virus glycoproteins: entry, replication, and pathogenesis. *Curr. Clin. Microbiol. Rep.* **3**, 204–215 (2016).
23. Geraghty, R. J., Krummenacher, C., Cohen, G. H., Eisenberg, R. J. & Spear, P. G. Entry of alphaherpesviruses mediated by poliovirus receptor-related protein 1 and poliovirus receptor. *Science* **280**, 1618–1620 (1998).
24. Krummenacher, C. et al. Comparative usage of herpesvirus entry mediator A and nectin-1 by laboratory strains and clinical isolates of herpes simplex virus. *Virology* **322**, 286–299 (2004).
25. Warner, M. S. et al. A cell surface protein with herpesvirus entry activity (HveB) confers susceptibility to infection by mutants of herpes simplex virus type 1, herpes simplex virus type 2, and pseudorabies virus. *Virology* **246**, 179–189 (1998).
26. Montgomery, R. I., Warner, M. S., Lum, B. J. & Spear, P. G. Herpes simplex virus-1 entry into cells mediated by a novel member of the TNF/NGF receptor family. *Cell* **87**, 427–436 (1996).
27. Shukla, D. et al. A novel role for 3-O-sulfated heparan sulfate in herpes simplex virus 1 entry. *Cell* **99**, 13–22 (1999).
28. Lee, C. C. et al. Structural basis for the antibody neutralization of herpes simplex virus. *Acta Crystallogr. D Biol. Crystallogr.* **69**, 1935–1945 (2013).
29. Li, A. et al. Structural basis of nectin-1 recognition by pseudorabies virus glycoprotein D. *PLoS Pathog.* **13**, e1006314 (2017).
30. Lu, G. et al. Crystal structure of herpes simplex virus 2 gD bound to nectin-1 reveals a conserved mode of receptor recognition. *J. Virol.* **88**, 13678–13688 (2014).
31. Connolly, S. A. et al. Glycoprotein D homologs in herpes simplex virus type 1, pseudorabies virus, and bovine herpes virus type 1 bind directly to human HveC (nectin-1) with different affinities. *Virology* **280**, 7–18 (2001).
32. Krummenacher, C. et al. Herpes simplex virus glycoprotein D can bind to poliovirus receptor-related protein 1 or herpesvirus entry mediator, two structurally unrelated mediators of virus entry. *J. Virol.* **72**, 7064–7074 (1998).
33. Whitbeck, J. C. et al. The major neutralizing antigenic site on herpes simplex virus glycoprotein D overlaps a receptor-binding domain. *J. Virol.* **73**, 9879–9890 (1999).
34. Yoon, M., Zago, A., Shukla, D. & Spear, P. G. Mutations in the N termini of herpes simplex virus type 1 and 2 gDs alter functional interactions with the entry/fusion receptors HVEM, nectin-2, and 3-O-sulfated heparan sulfate but not with nectin-1. *J. Virol.* **77**, 9221–9231 (2003).
35. Willis, S. H. et al. Examination of the kinetics of herpes simplex virus glycoprotein D binding to the herpesvirus entry mediator, using surface plasmon resonance. *J. Virol.* **72**, 5937–5947 (1998).
36. Krummenacher, C. et al. The first immunoglobulin-like domain of HveC is sufficient to bind herpes simplex virus gD with full affinity, while the third domain is involved in oligomerization of HveC. *J. Virol.* **73**, 8127–8137 (1999).
37. Lazear, E. et al. Engineered disulfide bonds in herpes simplex virus type 1 gD separate receptor binding from fusion initiation and viral entry. *J. Virol.* **82**, 700–709 (2008).
38. Sathiyamoorthy, K. et al. Assembly and architecture of the EBV B cell entry triggering complex. *PLoS Pathog.* **10**, e1004309 (2014). EM reconstructions of the gp42–HLA–gH–gL complex reveal open and closed conformations.

39. Sorem, J., Jardetzky, T. S. & Longnecker, R. Cleavage and secretion of Epstein-Barr virus glycoprotein 42 promote membrane fusion with B lymphocytes. *J. Virol.* **83**, 6664–6672 (2009).
40. Kirschner, A. N., Lowrey, A. S., Longnecker, R. & Jardetzky, T. S. Binding-site interactions between Epstein-Barr virus fusion proteins gp42 and gH/gL reveal a peptide that inhibits both epithelial and B-cell membrane fusion. *J. Virol.* **81**, 9216–9229 (2007).
41. Silva, A. L., Omerovic, J., Jardetzky, T. S. & Longnecker, R. Mutational analyses of Epstein-Barr virus glycoprotein 42 reveal functional domains not involved in receptor binding but required for membrane fusion. *J. Virol.* **78**, 5946–5956 (2004).
42. Zhou, M., Lanchy, J. M. & Ryckman, B. J. Human cytomegalovirus gH/gL/gO promotes the fusion step of entry into all cell types, whereas gH/gL/UL128–131 broadens virus tropism through a distinct mechanism. *J. Virol.* **89**, 8999–9009 (2015).
43. Wang, D. & Shenk, T. Human cytomegalovirus UL131 open reading frame is required for epithelial cell tropism. *J. Virol.* **79**, 10330–10338 (2005).
44. Zhou, M., Yu, Q., Wechsler, A. & Ryckman, B. J. Comparative analysis of gO isoforms reveals that strains of human cytomegalovirus differ in the ratio of gH/gL/gO and gH/gL/UL128–131 in the virion envelope. *J. Virol.* **87**, 9680–9690 (2013).
45. Wu, Y. et al. Human cytomegalovirus glycoprotein complex gH/gL/gO uses PDGFR-alpha as a key for entry. *PLoS Pathog.* **13**, e1006281 (2017).
46. E, X. et al. OR14l1 is a receptor for the human cytomegalovirus pentameric complex and defines viral epithelial cell tropism. *Proc. Natl Acad. Sci. USA* **116**, 7043–7052 (2019).
47. Vanarsdall, A. L. et al. CD147 promotes entry of pentamer-expressing human cytomegalovirus into epithelial and endothelial cells. *mBio* [https://doi.org/10.1128/mBio.00781-18](https://doi.org/10.1128/mBio.00781-18) (2018).
48. Liu, J., Jardetzky, T. S., Chin, A. L., Johnson, D. C. & Vanarsdall, A. L. The human cytomegalovirus trimer and pentamer promote sequential steps in entry into epithelial and endothelial cells at cell surfaces and endosomes. *J. Virol.* [https://doi.org/10.1128/JVI.01336-18](https://doi.org/10.1128/JVI.01336-18) (2018).
49. Chen, J. et al. Ephrin receptor A2 is a functional entry receptor for Epstein-Barr virus. *Nat. Microbiol.* **3**, 172–180 (2018).
50. TerBush, A. A., Hafkamp, F., Lee, H. J. & Coscoy, L. A Kaposi’s sarcoma-associated herpesvirus infection mechanism is independent of integrins alpha3beta1, alphaVbeta3, and alphaVbeta5. *J. Virol.* [https://doi.org/10.1128/JVI.00803-18](https://doi.org/10.1128/JVI.00803-18) (2018).
51. Chen, J., Zhang, X., Schaller, S., Jardetzky, T. S. & Longnecker, R. Ephrin receptor A4 is a new Kaposi’s sarcoma-associated herpesvirus virus entry receptor. *mBio* [https://doi.org/10.1128/mBio.02892-18](https://doi.org/10.1128/mBio.02892-18) (2019).
52. Hahn, A. S. et al. The ephrin receptor tyrosine kinase A2 is a cellular receptor for Kaposi’s sarcoma-associated herpesvirus. *Nat. Med.* **18**, 961–966 (2012).
53. Chesnokova, L. S. & Hutt-Fletcher, L. M. Fusion of Epstein-Barr virus with epithelial cells can be triggered by alphabeta5 in addition to alphabeta6 and alphabeta8, and integrin binding triggers a conformational change in glycoproteins gH/gL. *J. Virol.* **85**, 13214–13223 (2011).
54. Li, Q., Turk, S. M. & Hutt-Fletcher, L. M. The Epstein-Barr virus (EBV) BZLF2 gene product associates with the gH and gL homologs of EBV and carries an epitope critical to infection of B cells but not of epithelial cells. *J. Virol.* **69**, 3987–3994 (1995).
55. Yang, E., Arvin, A. M. & Oliver, S. L. Role for the alphaV integrin subunit in varicella-zoster virus-mediated fusion and infection. *J. Virol.* **90**, 7567–7578 (2016).
56. Parry, C., Bell, S., Minson, T. & Browne, H. Herpes simplex virus type 1 glycoprotein H binds to alphavbeta3 integrins. *J. Gen. Virol.* **86**, 7–10 (2005).
57. Gianni, T., Salvioli, S., Chesnokova, L. S., Hutt-Fletcher, L. M. & Campadelli-Fiume, G. alphavbeta6- and alphavbeta8-integrins serve as interchangeable receptors for HSV gH/gL to promote endocytosis and activation of membrane fusion. *PLoS Pathog.* **9**, e1003806 (2013).
58. Chowdary, T. K. et al. Crystal structure of the conserved herpesvirus fusion regulator complex gH-gL. *Nat. Strut Mol. Biol.* **17**, 882–888 (2010). Reports the crystal structure of HSV-2 gH–gL, exhibiting extensive interactions between gL and the N-terminal domain of gH.

59. Xing, Y. et al. A site of varicella-zoster virus vulnerability identified by structural studies of neutralizing antibodies bound to the glycoprotein complex gHgL. *Proc. Natl Acad. Sci. USA* **112**, 6056–6061 (2015).
60. Backovic, M. et al. Structure of a core fragment of glycoprotein H from pseudorabies virus in complex with antibody. *Proc. Natl Acad. Sci. USA* **107**, 22635–22640 (2010).
61. Matsuura, H., Kirschner, A. N., Longnecker, R. & Jardetzky, T. S. Crystal structure of the Epstein-Barr virus (EBV) glycoprotein H/glycoprotein L (gH/gL) complex. *Proc. Natl Acad. Sci. USA* **107**, 22641–22646 (2010).
62. Connolly, S. A., Jackson, J. O., Jardetzky, T. S. & Longnecker, R. Fusing structure and function: a structural view of the herpesvirus entry machinery. *Nat. Rev. Microbiol.* **9**, 369–381 (2011).
63. Gompels, U. A. et al. Characterization and sequence analyses of antibody-selected antigenic variants of herpes simplex virus show a conformationally complex epitope on glycoprotein H. *J. Virol.* **65**, 2393–2401 (1991).
64. Ciferr, C. et al. Antigenic characterization of the HCMV gH/gL/gO and pentamer cell entry complexes reveals binding sites for potently neutralizing human antibodies. *PLoS Pathog.* **11**, e1005230 (2015).
65. Sathiyamoorthy, K. et al. Inhibition of EBV-mediated membrane fusion by anti-gHgL antibodies. *Proc. Natl Acad. Sci. USA* [https://doi.org/10.1073/pnas.1704661114](https://doi.org/10.1073/pnas.1704661114) (2017).
66. Snijder, J. et al. An antibody targeting the fusion machinery neutralizes dual-tropic infection and defines a site of vulnerability on Epstein-Barr virus. *Immunity* **48**, 799–811 e799 (2018).
67. Wilson, D. W., Davis-Poynter, N. & Minson, A. C. Mutations in the cytoplasmic tail of herpes simplex virus glycoprotein H suppress cell fusion by a syncytial strain. *J. Virol.* **68**, 6985–6993 (1994).
68. Browne, H. M., Bruun, B. C. & Minson, A. C. Characterization of herpes simplex virus type 1 recombinants with mutations in the cytoplasmic tail of glycoprotein H. *J. Gen. Virol.* **77**, 2569–2573 (1996).
69. Jackson, J. O., Lin, E., Spear, P. G. & Longnecker, R. Insertion mutations in herpes simplex virus 1 glycoprotein H reduce cell surface expression, slow the rate of cell fusion, or abrogate functions in cell fusion and viral entry. *J. Virol.* **84**, 2038–2046 (2010).
70. Silverman, J. L. & Heldwein, E. E. Mutations in the cytoplasmic tail of herpes simplex virus 1 gH reduce the fusogenicity of gB in transfected cells. *J. Virol.* **87**, 10139–10147 (2013).
71. Rogalin, H. B. & Heldwein, E. E. Interplay between the herpes simplex virus 1 gB cytodomain and the gH cytotail during cell-cell fusion. *J. Virol.* **89**, 12262–12272 (2015).
72. Yang, E., Arvin, A. M. & Oliver, S. L. The cytoplasmic domain of varicella-zoster virus glycoprotein H regulates syncytia formation and skin pathogenesis. *PLoS Pathog.* **10**, e1004173 (2014).
73. Vallbracht, M., Fuchs, W., Klupp, B. G. & Mettenleiter, T. C. Functional relevance of the transmembrane domain and cytoplasmic tail of the pseudorabies virus glycoprotein H for membrane fusion. *J. Virol.* **92**, e00376–18 (2018).
74. Chen, J., Jardetzky, T. S. & Longnecker, R. The cytoplasmic tail domain of Epstein-Barr virus gH regulates membrane fusion activity through altering gH binding to gp42 and epithelial cell attachment. *mBio* [https://doi.org/10.1128/mBio.01871-16](https://doi.org/10.1128/mBio.01871-16) (2016).
75. Rowe, C. L., Connolly, S. A., Chen, J., Jardetzky, T. S. & Longnecker, R. A soluble form of Epstein-Barr virus gH/gL inhibits EBV-induced membrane fusion and does not function in fusion. *Virology* **436**, 118–126 (2013).
76. Jones, N. A. & Geraghty, R. J. Fusion activity of lipid-anchored envelope glycoproteins of herpes simplex virus type 1. *Virology* **324**, 213–228 (2004).
77. Cooper, R. S., Georgieva, E. R., Borbat, P. P., Freed, J. H. & Heldwein, E. E. Structural basis for membrane anchoring and fusion regulation of the herpes simplex virus fusogen gB. *Nat. Struct. Mol. Biol.* **25**, 416–424 (2018). Reports the crystal structure of full-length HSV-1 gB, including the membrane-proximal, transmembrane, and cytoplasmic tail domains.
78. White, J. M., Delos, S. E., Brecher, M. & Schornberg, K. Structures and mechanisms of viral membrane fusion proteins: multiple variations on a common theme. *Crit. Rev. Biochem. Mol. Bio* **43**, 189–219 (2008).

NATURE REVIEWS | MICROBIOLOGY

REVIEWS

79. Heldwein, E. E. et al. Crystal structure of glycoprotein B from herpes simplex virus 1. *Science* **313**, 217–220 (2006).

80. Vallbracht, M. et al. Structure-function dissection of the Pseudorabies virus glycoprotein B fusion loops. *J. Virol.* **92**, e01203–01201 (2017).

81. Li, X. et al. Two classes of protective antibodies against Pseudorabies virus variant glycoprotein B: implications for vaccine design. *PLoS Pathog.* **13**, e1006777 (2017).

82. Oliver, S. L. et al. A glycoprotein B-neutralizing antibody structure at 2.8 Å uncovers a critical domain for herpesvirus fusion initiation. *Nat. Commun.* **11**, 1–15 (2020).

83. Chandramouli, S. et al. Structure of HCMV glycoprotein B in the postfusion conformation bound to a neutralizing human antibody. *Nat. Commun.* **6**, 8176 (2015). Reports the crystal structure of the postfusion conformation of HCMV gB bound to a nAb.

84. Burke, H. G. & Heldwein, E. E. Crystal structure of the human cytomegalovirus glycoprotein B. *PLoS Pathog.* **11**, e1005227 (2015). Reports the crystal structure of the postfusion conformation of HCMV gB.

85. Backovic, M., Longnecker, R. & Jardetzky, T. S. Structure of a trimeric variant of the Epstein-Barr virus glycoprotein B. *Proc. Natl Acad. Sci. USA* **106**, 2880–2885 (2009).

86. Roche, S., Bressanelli, S., Rey, F. A. & Gaudin, Y. Crystal structure of the low-pH form of the vesicular stomatitis virus glycoprotein G. *Science* **313**, 187–191 (2006).

87. Yang, F. et al. Structural analysis of rabies virus glycoprotein reveals pH-dependent conformational changes and interactions with a neutralizing antibody. *Cell Host Microbe* (2020).

88. Kadlec, J., Loureiro, S., Abrescia, N. G., Stuart, D. I. & Jones, I. M. The postfusion structure of baculovirus gp64 supports a unified view of viral fusion machines. *Nat. Struct. Mol. Biol.* **15**, 1024–1030 (2008).

89. Peng, R. et al. Structures of human-infecting Thogotovirus fusogens support a common ancestor with insect baculovirus. *Proc. Natl Acad. Sci. USA* **114**, E8905–E8912 (2017).

90. Backovic, M. & Jardetzky, T. S. Class III viral membrane fusion proteins. *Curr. Opin. Struc. Biol.* **19**, 189–196 (2009).

91. Bender, F. C. et al. Antigenic and mutational analyses of herpes simplex virus glycoprotein B reveal four functional regions. *J. Virol.* **81**, 3827–3841 (2007).

92. Bootz, A. et al. Protective capacity of neutralizing and non-neutralizing antibodies against glycoprotein B of cytomegalovirus. *PLoS Pathog.* **13**, e1006601 (2017).

93. Cairns, T. M. et al. Mechanism of neutralization of herpes simplex virus by antibodies directed at the fusion domain of glycoprotein B. *J. Virol.* **88**, 2677–2689 (2014).

94. Lin, E. & Spear, P. G. Random linker-insertion mutagenesis to identify functional domains of herpes simplex virus type 1 glycoprotein B. *Proc. Natl Acad. Sci. USA* **104**, 13140–13145 (2007).

95. Reimer, J. J., Backovic, M., Deshpande, C. G., Jardetzky, T. & Longnecker, R. Analysis of Epstein-Barr virus glycoprotein B functional domains via linker insertion mutagenesis. *J. Virol.* **83**, 734–747 (2009).

96. Maurer, U. E. et al. The structure of herpesvirus fusion glycoprotein B-bilayer complex reveals the protein-membrane and lateral protein-protein interaction. *Structure* **21**, 1396–1405 (2013).

97. Satoh, T. et al. PILRalpha is a herpes simplex virus-1 entry coreceptor that associates with glycoprotein B. *Cell* **132**, 935–944 (2008).

98. Suenaga, T. et al. Myelin-associated glycoprotein mediates membrane fusion and entry of neurotropic herpesviruses. *Proc. Natl Acad. Sci. USA* **107**, 866–871 (2010).

99. Arii, J. et al. Non-muscle myosin IIA is a functional entry receptor for herpes simplex virus-1. *Nature* **467**, 859–862 (2010).

100. Roche, S., Rey, F. A., Gaudin, Y. & Bressanelli, S. Structure of the prefusion form of the vesicular stomatitis virus glycoprotein G. *Science* **315**, 843–848 (2007).

101. Silverman, J. L., Sharma, S., Cairns, T. M. & Heldwein, E. E. Fusion-deficient insertion mutants of herpes simplex virus 1 glycoprotein B adopt the trimeric postfusion conformation. *J. Virol.* **84**, 2001–2012 (2010).

102. Vitu, E., Sharma, S., Stampfer, S. D. & Heldwein, E. E. Extensive mutagenesis of the HSV-1 gB ectodomain reveals remarkable stability of its postfusion form. *J. Mol. Biol.* **425**, 2056–2071 (2013).

103. Vollmer, B. et al. The pre-fusion structure of herpes simplex virus glycoprotein B. *Sci. Adv.* **6**, eabc1726 (2020). EM reconstructions of a membrane-anchored HSV-1 gB mutant designed to trap the prefusion form reveal a compact structure at an overall resolution of 9 Å.

104. Falanga, A. et al. Biophysical characterization and membrane interaction of the two fusion loops of glycoprotein B from herpes simplex type I virus. *PLoS ONE* **7**, e32186 (2012).

105. Oliver, S. L. et al. Mutagenesis of varicella-zoster virus glycoprotein B: putative fusion loop residues are essential for viral replication, and the furin cleavage motif contributes to pathogenesis in skin tissue in vivo. *J. Virol.* **83**, 7495–7506 (2009).

106. Backovic, M., Jardetzky, T. S. & Longnecker, R. Hydrophobic residues that form putative fusion loops of Epstein-Barr virus glycoprotein B are critical for fusion activity. *J. Virol.* **81**, 9596–9600 (2007).

107. Hannah, B. P. et al. Herpes simplex virus glycoprotein B associates with target membranes via its fusion loops. *J. Virol.* **83**, 6825–6836 (2009).

108. Harrison, S. C. Viral membrane fusion. *Virology* **479–480**, 498–507 (2015).

109. Melikyan, G. B. et al. Evidence that the transition of HIV-1 gp41 into a six-helix bundle, not the bundle configuration, induces membrane fusion. *J. Cell Biol.* **151**, 413–423 (2000).

110. Connolly, S. A. & Longnecker, R. Residues within the C-terminal arm of the herpes simplex virus 1 glycoprotein B ectodomain contribute to its refolding during the fusion step of virus entry. *J. Virol.* **86**, 6386–6393 (2012).

111. Fan, Q., Kopp, S. J., Connolly, S. A. & Longnecker, R. Structure-based mutations in the herpes simplex virus 1 glycoprotein b ectodomain arm impart a slow-entry phenotype. *mBio* [https://doi.org/10.1128/mBio.00614-17](https://doi.org/10.1128/mBio.00614-17) (2017).

112. Waning, D. L., Russell, C. J., Jardetzky, T. S. & Lamb, R. A. Activation of a paramyxovirus fusion protein is modulated by inside-out signaling from the cytoplasmic tail. *Proc. Natl Acad. Sci. USA* **101**, 9217–9222 (2004).

113. Wyss, S. et al. Regulation of human immunodeficiency virus type 1 envelope glycoprotein fusion by a membrane-interactive domain on the gp41 cytoplasmic tail. *J. Virol.* **79**, 12231–12241 (2005).

114. Garcia, N. J., Chen, J. & Longnecker, R. Modulation of Epstein-Barr virus glycoprotein B (gB) fusion activity by the gB cytoplasmic tail domain. *mBio* **4**, e00571–12 (2013).

115. Nixdorf, R., Klupp, B. G., Karger, A. & Mettenleiter, T. C. Effects of truncation of the carboxy terminus of pseudorabies virus glycoprotein B on infectivity. *J. Virol.* **74**, 7137–7145 (2000).

116. Fan, Z. et al. Truncation of herpes simplex virus type 2 glycoprotein B increases its cell surface expression and activity in cell-cell fusion, but these properties are unrelated. *J. Virol.* **76**, 9271–9283 (2002).

117. Muggeridge, M. I., Grantham, M. L. & Johnson, F. B. Identification of syncytial mutations in a clinical isolate of herpes simplex virus 2. *Virology* **328**, 244–253 (2004).

118. Silverman, J. L., Greene, N. G., King, D. S. & Heldwein, E. E. Membrane requirement for folding of the herpes simplex virus 1 gB cytodomain suggests a unique mechanism of fusion regulation. *J. Virol.* **86**, 8171–8184 (2012).

119. Oliver, S. L. et al. An immunoreceptor tyrosine-based inhibition motif in varicella-zoster virus glycoprotein B regulates cell fusion and skin pathogenesis. *Proc. Natl Acad. Sci. USA* **110**, 1911–1916 (2013).

120. Cooper, R. S. & Heldwein, E. E. Herpesvirus gB: a finely tuned fusion machine. *Viruses* **7**, 6552–6569 (2015).

121. Gallagher, J. R. et al. Functional fluorescent protein insertions in herpes simplex virus gB report on gB conformation before and after execution of membrane fusion. *PLoS Pathog.* **10**, e1004373 (2014).

122. Zeev-Ben-Mordehai, T. et al. Two distinct trimeric conformations of natively membrane-anchored full-length herpes simplex virus 1 glycoprotein B. *Proc. Natl Acad. Sci. USA* **113**, 4176–4181 (2016).

123. Fontana, J. et al. The fusion loops of the initial prefusion conformation of herpes simplex virus 1 fusion protein point toward the membrane. *mBio* [https://doi.org/10.1128/mBio.01268-17](https://doi.org/10.1128/mBio.01268-17) (2017).

124. Si, Z. et al. Different functional states of fusion protein gB revealed on human cytomegalovirus by cryo electron tomography with Volta phase plate. *PLoS Pathog.* **14**, e1007452 (2018).

125. Handler, C. G., Cohen, G. H. & Eisenberg, R. J. Cross-linking of glycoprotein oligomers during herpes simplex virus type 1 entry. *J. Virol.* **70**, 6076–6082 (1996).

126. Gianni, T., Amasio, M. & Campadelli-Fiume, G. Herpes simplex virus gD forms distinct complexes with fusion executors gB and gH/gL in part through the C-terminal profusion domain. *J. Biol. Chem.* **284**, 17370–17382 (2009).

127. Perez-Romero, P., Perez, A., Capul, A., Montgomery, R. & Fuller, A. O. Herpes simplex virus entry mediator associates in infected cells in a complex with viral proteins gD and at least gH. *J. Virol.* **79**, 4540–4544 (2005).

128. Avitabile, E., Forghieri, C. & Campadelli-Fiume, G. Complexes between herpes simplex virus glycoproteins gD, gB, and gH detected in cells by complementation of split enhanced green fluorescent protein. *J. Virol.* **81**, 11532–11537 (2007).

129. Atanasiu, D. et al. Bimolecular complementation reveals that glycoproteins gB and gH/gL of herpes simplex virus interact with each other during cell fusion. *Proc. Natl Acad. Sci. USA* **104**, 18718–18723 (2007).

130. Fan, Q., Longnecker, R. & Connolly, S. A. Substitution of herpes simplex virus 1 entry glycoproteins with those of saimiriine herpesvirus 1 reveals a gD-gH/gL functional interaction and a region within the gD profusion domain that is critical for fusion. *J. Virol.* **88**, 6470–6482 (2014).

131. Fan, Q., Longnecker, R. & Connolly, S. A. A functional interaction between herpes simplex virus 1 glycoprotein gH/gL domains I and II and gD is defined by using alphaherpesvirus gH and gL chimeras. *J. Virol.* **89**, 7159–7169 (2015).

132. Atanasiu, D. et al. Regulation of herpes simplex virus gB-induced cell-cell fusion by mutant forms of gH/gL in the absence of gD and cellular receptors. *mBio* [https://doi.org/10.1128/mBio.00046-13](https://doi.org/10.1128/mBio.00046-13) (2013).

133. Cairns, T. M. et al. Surface plasmon resonance reveals direct binding of herpes simplex virus glycoproteins gH/gL to gD and Locates a gH/gL binding site on gD. *J. Virol.* [https://doi.org/10.1128/JVI.00289-19](https://doi.org/10.1128/JVI.00289-19) (2019). Demonstrates direct interaction between purified forms of the HSV-2 gD and gH–gL ectodomains.

134. Atanasiu, D. et al. Bimolecular complementation defines functional regions of Herpes simplex virus gB that are involved with gH/gL as a necessary step leading to cell fusion. *J. Virol.* **84**, 3825–3834 (2010).

135. Vanarsdall, A. L., Ryckman, B. J., Chase, M. C. & Johnson, D. C. Human cytomegalovirus glycoproteins gB and gH/gL mediate epithelial cell-cell fusion when expressed in cis or in trans. *J. Virol.* **82**, 11837–11850 (2008).

136. Oxman, M. N. et al. A vaccine to prevent herpes zoster and postherpetic neuralgia in older adults. *N. Engl. J. Med.* **352**, 2271–2284 (2005).

137. Cunningham, A. L. et al. Efficacy of the herpes zoster subunit vaccine in adults 70 years of age or older. *N. Engl. J. Med.* **375**, 1019–1032 (2016).

**Acknowledgements**

This work was supported by grants AI-137267 and AI-148478 from the US National Institute of Allergy and Infectious Diseases of the National Institutes of Health. All crystal structures in this Review were rendered with MacPyMOL.

**Author contributions**

The authors contributed equally to all aspects of the article.

**Competing interests**

The authors declare no competing interests.

**Peer review information**

*Nature Reviews Microbiology* thanks A. Carfi, K. Grünewald, A. Nicola, L. Perez, B. Vollmer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

**Publisher’s note**

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

**RELATED LINKS**

EMDataResource: [http://www.emdataresource.org/](http://www.emdataresource.org/)  
RCSB Protein Data Bank: [https://www.rcsb.org/](https://www.rcsb.org/)

© Springer Nature Limited 2020
