
J. Biochem. 2021; 00(0): 1–9 doi:10.1093/jb/mvab043

**JB Review**

**D-Amino acid metabolism in bacteria**

Received December 25, 2020; accepted March 19, 2021; published online March 31, 2021

Tetsuya Miyamoto* and Hiroshi Homma

Graduate School of Pharmaceutical Sciences, Kitasato University,  
5-9-1 Shirokane, Minato-ku, Tokyo 108-8641, Japan  

*Tetsuya Miyamoto, Graduate School of Pharmaceutical Sciences,  
Kitasato University, 5-9-1 Shirokane, Minato-ku, Tokyo 108-8641,  
Japan. Tel.: +81-3-5791-6379, Fax: +81-3-5791-6381, email: miya-  
motot@pharm.kitasato-u.ac.jp*

Bacteria produce diverse D-amino acids, which are essential components of cell wall peptidoglycan. Incorporation of these D-amino acids into peptidoglycan contributes to bacterial adaptation to environmental changes and threats. D-Amino acids have been associated with bacterial growth, biofilm formation and dispersal and regulation of peptidoglycan metabolism. The diversity of D-amino acids in bacteria is primarily due to the activities of amino acid racemases that catalyse the interconversion of the D- and L-enantiomers of amino acids. Recent studies have revealed that bacteria possess multiple enzymes with amino acid racemase activities. Therefore, elucidating D-amino acid metabolism by these enzymes is critical to understand the biological significance and behaviour of D-amino acids in bacteria. In this review, we focus on the metabolic pathways of D-amino acids in six types of bacteria.

Keywords: amino acid racemase; bacteria; D-amino acid; Thermotoga maritima; Thermus thermophilus.

Abbreviations: Aba, aminobutyric acid; Cit, citrulline; Dpm, diaminopimelate; GlcNAc, N-acetylglycosamine; Hse, homoserine; MurNAc, N-acetylmuramic acid; Nle, norleucine; Nva, norvaline; Orn, ornithine; PLP, pyridoxal 5'-phosphate.

In bacteria, D-amino acids are necessary components of the peptidoglycan layer of the cell wall, and as such

---

### Graphical Abstract

#### D-Ala
![Diagram for D-Ala](https://i.imgur.com/placeholder.png)

#### D-Glu
![Diagram for D-Glu](https://i.imgur.com/placeholder.png)

#### D-Ser
![Diagram for D-Ser](https://i.imgur.com/placeholder.png)

#### Non-canonical D-amino acid
![Diagram for Non-canonical D-amino acid](https://i.imgur.com/placeholder.png)

---

AlaR: Ala racemase  
GluR: Glu racemase  
CBL: Cystathionine β-lyase  
Dat: D-Amino acid aminotransferase  
Dsd: D-Ser dehydratase  
BSR: Broad-spectrum racemase  
α-KG: α-Ketoglutaric acid

© The Author(s) 2021. Published by Oxford University Press on behalf of the Japanese Biochemical Society. All rights reserved.

T. Miyamoto and H. Homma

A Escherichia coli

| D-Ala | D-Glu | D-Ser | D-Hse |
| --- | --- | --- | --- |
| L-Ala<br>Alr<br>DadX<br>MetC<br>MalY<br>D-Ala<br>Peptidoglycan | L-Glu<br>Murl<br>D-Glu<br>Peptidoglycan | L-Ser<br>MetC<br>D-Ser<br>MetC<br>DsdA<br>Pyruvate | L-Hse<br>YgeA<br>D-Hse<br>Unknown |

B Bacillus subtilis

| D-Ala | D-Glu | D-Arg / D-Lys / D-Orn |
| --- | --- | --- |
| L-Ala<br>AlrA<br>AlrB<br>Dat<br>D-Ala ↔ Pyruvate<br>Peptidoglycan | L-Glu<br>RacE<br>YrpC<br>Dat<br>D-Glu ↔ α-KG<br>Peptidoglycan | L-Arg / L-Lys / L-Orn<br>RacX<br>D-Arg / D-Lys / D-Orn<br>Unknown |

Fig. 1. Summary of D-amino acid metabolism in *E. coli* and *B. subtilis*. (A) Metabolism of D-Ala, D-Glu, D-Ser and D-Hse in *E. coli*. D-Ala is synthesized by Ala racemases (Alr and DadX) and cystathionine β-lyases (MetC and MalY). D-Glu is synthesized by Glu racemase (Murl). D-Ser is synthesized by cystathionine β-lyases (MetC) and degraded to pyruvate by D-Ser dehydratase (DsdA) and presumably by MetC. D-Hse is likely to be synthesized by a broad-spectrum racemase (YgeA), but the role of D-Hse is unknown. (B) Metabolism of D-Ala, D-Glu and D-Arg/D-Lys/D-Orn in *B. subtilis*. D-Ala is synthesized by Ala racemases (AlrA and AlrB). D-Glu is synthesized by Glu racemases (RacE and YrpC). In addition, D-Ala and D-Glu are likely produced from the corresponding α-keto acids (pyruvate; α-KG, α-ketoglutaric acid) by a D-amino acid aminotransferase (Dat). D-Arg, D-Lys and D-Orn are apparently synthesized by a broad-spectrum racemase (RacX), but the roles of these D-amino acids are unknown.

contribute to the preservation of cell structure and resistance to various environmental threats (1–3). D-Alanine (D-Ala) and D-glutamate (D-Glu) are the most common D-amino acids present in peptidoglycan. In addition, atypical D-amino acids (called non-canonical D-amino acids) are contained in peptidoglycan stem peptides (4–6). Incorporation of these non-canonical D-amino acids affects the regulation of peptidoglycan metabolism and biofilm stability (4–14). A previous study by our group also suggested that editing of non-canonical D-amino acids in the peptidoglycan by penicillin-binding proteins affects bacterial growth and biofilm formation (15). Accordingly, the production of various D-amino acids in bacteria is essential for adaptation to diverse environmental threats, and elucidation of D-amino acid metabolism in bacteria will contribute to our understanding of bacterial strategies for environmental adaptation. Bacteria predominantly synthesize D-amino acids using amino acid racemases that catalyse the stereochemical interconversion of amino acids, e.g., Ala racemase (EC 5.1.1.1) (2, 3), Glu racemase (EC 5.1.1.3) (2, 3), aspartate (Asp) racemase (EC 5.1.1.13) (16, 17), serine (Ser) racemase (EC 5.1.1.18) (18, 19), proline (Pro) racemase (EC 5.1.1.4) (20, 21), lysine

(Lys) racemase (EC 5.1.1.5) (22, 23), arginine (Arg) racemase (EC 5.1.1.9) (24, 25) and histidine (His) racemase (EC 5.1.1.24) (26). Recently, a broad-spectrum amino acid racemase has been identified in some bacteria. Furthermore, other enzymes can also produce D-amino acids. Here, we mainly review the recent findings of D-amino acid metabolism by several bacterial species (Figs 1–3).

Escherichia coli

Peptidoglycan consists of a long glycan chain of alternating N-acetylglucosamine (GlcNAc) and N-acetylmuramic acid (MurNAc), cross-linked by short-stem peptides. In Gram-negative bacteria, including *E. coli*, the peptide chain typically contains L-Ala, D-Glu, meso-diaminopimelate (meso-Dpm) and D-Ala (GlcNAc–MurNAc–L-Ala–D-Glu–meso-Dpm–D-Ala–D-Ala) (1). D-Ser and D-Asp are also incorporated into the stem peptides in vancomycin-resistant bacteria and lactic bacteria (27, 28). These D-amino acid components of peptidoglycan are generally synthesized from the corresponding L-amino acids by amino acid racemases, such as Ala racemase, Glu racemase, Ser racemase and Asp racemase (2, 3). Additionally, meso-Dpm is produced by a Dpm epimerase (EC 5.1.1.7), which catalyses the interconversion between

A Vibrio cholerae

D-Ala
| L-Ala |
| --- |
| AlrV |
| D-Ala |
| ↓ |
| Peptidoglycan |

D-Glu
| L-Glu |
| --- |
| Murl |
| D-Glu |
| ↓ |
| Peptidoglycan |

D-Met / D-Leu / D-Arg etc.
| L-Met / L-Leu / L-Arg |
| --- |
| BsrV |
| D-Met / D-Leu / D-Arg |
| ↓ |
| Peptidoglycan |

B Pseudomonas putida

D-Ala
| L-Ala |
| --- |
| DadX |
| D-Ala |
| ↓ |
| Peptidoglycan |

D-Glu
| L-Glu |
| --- |
| Murl |
| D-Glu |
| ↓ |
| Peptidoglycan |

D-Lys
| L-Lys |
| --- |
| Alr |
| D-Lys |
| Dad |
| Pip2C |

D-Arg
| L-Arg |
| --- |
| Alr |
| D-Arg |
| ↓ |
| Unknown |

Fig. 2. Summary of D-amino acid metabolism in *V. cholerae* and *P. putida*. (A) Metabolism of D-Ala, D-Glu and non-canonical D-amino acids such as D-Met, D-Leu and D-Arg in *V. cholerae*. D-Ala is synthesized by Ala racemase (AlrV). D-Glu is synthesized by Glu racemase (Murl). Non-canonical D-amino acids are synthesized by a broad-spectrum racemase (BsrV) and are incorporated into the peptidoglycan. (B) Metabolism of D-Ala, D-Glu, D-Lys and D-Arg in *P. putida*. D-Ala is synthesized by Ala racemase (DadX). D-Glu is synthesized by Glu racemase (Murl). D-Lys and D-Arg are synthesized by a broad-spectrum racemase (Alr). D-Lys is converted to Δ¹-piperidine-2-carboxylate (Pip2C) by D-amino acid dehydrogenase (Dad), and then Pip2C is converted to L-pipecolate by Pip2C/Δ¹-pyrroline-2-carboxylate reductase.

LL-Dpm and meso-Dpm (29). Amino acid racemases are grouped into two categories, pyridoxal 5′-phosphate (PLP)-dependent and PLP-independent enzymes. *E. coli* possesses two PLP-dependent Ala racemases (Alr and DadX) and one PLP-independent Glu racemase (Murl). Alr plays a role in D-Ala synthesis for peptidoglycan, whereas DadX is involved in D-Ala catabolism (30, 31). D-Ser, D-Asp, D-leucine (D-Leu) and D-valine (D-Val) are present in *E. coli* cells cultured in minimal medium or medium supplemented with 19 proteinogenic L-amino acids and glycine (32). Additionally, these non-canonical D-amino acids are also detected in the supernatant of *E. coli* cultures (4). We identified an amino acid racemase (YgeA) with broad substrate specificity that is likely to be involved in the biosynthesis of these non-canonical D-amino acids in *E. coli* (33). YgeA has homodimeric structure and is PLP-independent racemase that shows the activity towards 15 amino acids, including hydrophobic amino acids such as methionine (Met), Leu and Val (Table I); the D-forms of those amino acids are detected in *E. coli* cells and their culture media (4, 32). Interestingly, YgeA preferentially catalyses the racemization of a non-proteinogenic amino acid, homoserine (Hse), although its racemase activity towards Hse [in terms of catalytic efficiency, $k_{cat}/K_m$] is much lower than those of bacterial Ala racemase and Glu racemase. The physiological significance of D-Hse remains to be determined.

Excess D-amino acids can impair bacterial growth, presumably by inhibiting protein synthesis following the accumulation of tRNAs mischarged with cognate D-amino acids (34). Accordingly, the activity of YgeA may be maintained at a lower level. However, based on its pH and temperature dependencies (33), it is likely to be active under physiological conditions within *E. coli*.

Furthermore, we also found that two cystathionine β-lyases (MetC and MalY; EC 4.4.1.13) from *E. coli* are multifunctional enzymes with racemase activity and Ser dehydratase activity, in addition to lyase activity (35). Cystathionine β-lyase is PLP-dependent and catalyses the conversion of L-cystathionine to L-homocysteine, pyruvate and ammonia in the penultimate step of the L-Met biosynthetic pathway (36, 37). This enzyme also catalyses the degradation of L-cysteine (L-Cys) to pyruvate and hydrogen sulphate. Intriguingly, MetC has racemase activity towards 14 amino acids, whereas MalY has activity towards 13 amino acids (Table I). These two enzymes preferentially catalyse the racemization of Ala. The catalytic efficiency of MetC towards L-Ala ($60.2 \text{min}^{-1} \text{mM}^{-1}$) is significantly higher than that of MalY ($0.2 \text{min}^{-1} \text{mM}^{-1}$). Additionally, MetC and MalY can

T. Miyamoto and H. Homma

A Thermus thermophilus

D-Ala
| L-Ala |
| --- |
| AlaR |
| D-Ala |
| Peptidoglycan |

D-Glu
| L-Glu |
| --- |
| GluR |
| D-Glu |
| ? |
| ? |
| Peptidoglycan |

B Thermotoga maritima

D-Ala
| L-Ala |
| --- |
| LysR |
| CBL |
| D-Ala |
| Peptidoglycan |

D-Glu
| L-Glu |
| --- |
| CBL |
| D-Glu |
| Peptidoglycan |

D-Lys
| L-Lys |
| --- |
| LysR |
| D-Lys |
| Peptidoglycan |

Fig. 3. Summary of D-amino acid metabolism in *T. thermophilus* and *T. maritima*. (A) Metabolism of D-Ala and D-Glu in *T. thermophilus*. D-Ala is synthesized by Ala racemase (AlaR; TTHA 0062). D-Glu is synthesized by Glu racemase (GluR; TTHA 1643) and an alternative pathway. (B) Metabolism of D-Ala, D-Glu and D-Lys in *T. maritima*. D-Ala may be synthesized by both Lys racemase (LysR; TM1597) and cystathionine β-lyase (CBL; TM1270). D-Glu may be synthesized by CBL. D-Lys is synthesized by LysR.

rescue the D-Ala auxotrophic phenotype of Ala racemase-deficient *E. coli* strain (38). The catalytic efficiency of MetC lyase activity towards L-Cys (92.1 min⁻¹ mM⁻¹) is slightly higher than those of the racemase activities towards L-Ala and D-Ala (53.7 min⁻¹ mM⁻¹). On the other hand, the catalytic efficiency of MalY lyase activity towards L-Cys (2.2 min⁻¹ mM⁻¹) is 10-fold higher than its racemase activity towards L-Ala. Therefore, MetC may function as an amino acid racemase, whereas MalY may primarily act as a lyase. MetC has higher racemase activities towards L-aminobutyric acid (L-Aba) (2.3 min⁻¹ mM⁻¹) and L-Ser (1.5 min⁻¹ mM⁻¹). High levels of D-Ser can exert toxicity in bacteria by inhibiting protein synthesis (34); by contrast, the biological activity of D-Aba is unclear. Additionally, MetC has racemase activity towards Asp, whereas MalY has no activity towards Asp; therefore, MetC may be involved in the production of D-Asp, which is present in *E. coli* cells

(32). Notably in this regard, MalY from *Lactobacillus sakei* also has racemase activity towards 15 amino acids in addition to lyase activity (Table I) (39).

On the other hand, MetC may be involved in Ser degradation, as this enzyme has dehydratase activity towards D- and L-Ser, whereas MalY has much lower dehydratase activity towards L-Ser. According to the pH and temperature dependencies of dehydratase activity (35), MetC could function as a dehydratase under physiological conditions in *E. coli*. The catalytic efficiency of MetC dehydratase activity towards L-Ser (55.7 min⁻¹ mM⁻¹) is comparable to that of racemase activity towards D- and L-Ala, whereas the catalytic efficiency of its activity towards D-Ser (4.0 min⁻¹ mM⁻¹) is lower. MetC racemase activity towards Ala and dehydratase activity towards L-Ser are comparable to its lyase activity towards L-Cys, whereas the catalytic efficiency of its racemase activity towards L-Ser is much lower than that of its dehydratase activity towards L-Ser. Taken together, these observations

d-Amino acid metabolism in bacteria

Table I. Substrate specificities of enzymes

| Species       | Enzyme   | Activity     | Substrate                                                                                          | Ref.      |
|---------------|----------|--------------|---------------------------------------------------------------------------------------------------|-----------|
| E. coli       | YgeA     | Racemase     | Hse, Met, Leu, Val, His, Nva, Asn, Aba, Ile, Ser, Nle, Ala, Gln, Dpm, Phe                              | (33)      |
|               | MetC     | Racemase     | Ala, Aba, Ser, Met, Glu, Hse, Asn, His, Phe, Asp, Lys, Gln, Tyr, Arg                                  | (35, 36)  |
|               |          | Dehydratase  | L-Ser, D-Ser                                                                                      |           |
|               |          | Lyase        | L-Cystathionine, L-Cys, L-Homolanthione, meso-Lanthionine, L-Djenkolic acid                           |           |
|               | MalY     | Racemase     | Ala, Ser, Aba, Hse, Met, Arg, Lys, Phe, Gln, Tyr, Asn, Leu, His                                      | (35, 37)  |
|               |          | Dehydratase  | L-Ser                                                                                             |           |
|               |          | Lyase        | L-Cystathionine, L-Cys, L-Lanthionine, meso-Lanthionine, Thiolsine                                   |           |
| L. sakei      | MalY     | Racemase     | Ala, Arg, Val, Met, Ser, His, Lys, Asn, Gln, Trp, Leu, Thr, Glu, allo-Thr, Tyr                         | (39)      |
|               |          | Lyase        | L-Cystathionine, L-Cys                                                                            |           |
| B. subtilis   | RacX     | Racemase     | Dpm, Arg, Orn, Lys, His, Ala, Tyr, Phe, Ser, Gln, Met, Asn, Hse, Nle, Aba, Nva                        | (33)      |
| V. cholerae   | BsrV     | Racemase     | Gln, Ala, Ser, Met, Leu, Cys, Asn, His, Lys, Arg, Val, Ile                                           | (46)      |
| P. taetrolens | ArgR     | Racemase     | Lys, Arg, Orn, Met, Ala, Ethionine, Nva, Homoarginine, Leu, Aba, Phe                                  | (25)      |
| P. putida     | Alr      | Racemase     | Lys, Arg, Met, Gln, Ala, Ser, Leu, His, Asn                                                       | (49)      |
| T. maritima   | TM1597   | Racemase     | Lys, Orn, Ala, Arg, Aba, Dpm, Ser, Hse, Asn, Phe, Met                                              | (58)      |
|               | TM1522   | Racemase     | Dpm, Lys, Orn, Gln, Met, Cit, Ala, Ser, Hse, Leu, Aba, Phe, Arg, Asn                                |           |

Note: Aba, aminobutyric acid; Cit, citrulline; Dpm, diaminopimelate; Hse, homoserine; Nle, norleucine; Nva, norvaline; Orn, ornithine.

imply that MetC is more active in converting D- and L-Ser to pyruvate than catalysing the interconversion between D-Ser and L-Ser, and it is likely to acts as an Ala racemase and Ser dehydratase as well as lyase. E. coli also has a PLP-dependent D-Ser dehydratase (DsdA; EC 4.3.1.18) that catalyses the conversion of D-Ser to pyruvate (40). D-Ser induces an SOS-like response and downregulates the type III secretion system, which is essential for the attachment of enterohaemorrhagic E. coli (EHEC) to host cells (41). Therefore, detoxification and metabolism of D-Ser are crucial for this bacterium.

### Bacillus subtilis

In the Gram-positive bacterium *Bacillus subtilis*, the peptide chain in peptidoglycan contains two D-amino acids, D-Ala and D-Glu, and its structure is similar to that of *E. coli*. *B. subtilis* has two Ala racemases (AlrA and AlrB) and two Glu racemases (RacE and YrpC). RacE is involved in D-Glu synthesis both for poly-γ-D-Glu and peptidoglycan, whereas YrpC contributes specifically to D-Glu synthesis for peptidoglycan (42, 43). In addition, *Bacillus* species including strain YM-1 and *B. sphaericus* have a D-amino acid aminotransferase (EC 2.6.1.21) that catalyses transamination between D-amino acid and α-keto acid (44, 45); this enzyme may be involved in D-Ala and D-Glu synthesis for peptidoglycan, but its physiological function has not been fully explored. Non-canonical D-amino acids are also present in the supernatant of *B. subtilis* cultures (4); these D-amino acids may be produced by RacX, an amino acid racemase with broad substrate specificity (33), in addition to a D-amino acid aminotransferase (44, 45). RacX is assumed to be a homodimeric, PLP-independent enzyme with two Cys residues (Cys82 and Cys191) that catalyse racemization. RacX catalyses the racemization of 16 amino acids (Table I) and is preferentially active towards basic amino acids, such as Arg, Lys and ornithine (Orn); however, the presence of D-Arg, D-Lys and D-Orn is still unclear in *B. subtilis*. Its catalytic efficiency is much lower than those of typical amino acid racemases, but given its pH and temperature profiles (33), it is likely to function in the range of physiological condition of *B. subtilis*.

### Vibrio cholerae

*Vibrio cholerae* is a Gram-negative, comma-shaped bacterium that infects various organisms. The structure of peptidoglycan is similar to that of typical Gram-negative bacterium, such as *E. coli*. *V. cholerae* has a periplasmic broad-spectrum racemase (BsrV) in addition to Ala and Glu racemases (46). In fact, various D-amino acids such as D-Met, D-Leu and D-Arg were detected in the culture medium of *V. cholerae* (4, 47). BsrV is a PLP-dependent homodimeric enzyme that can synthesize 12 D-amino acids (Table I), while Ala racemase has high substrate specificity towards Ala and Ser (46). These non-canonical D-amino acids are incorporated into the peptidoglycan (terminal D-Ala position at the stem peptides) and regulate the peptidoglycan metabolism to adapt to the changing environment (4, 5). D-Arg inhibits the growth of some bacteria and regulates microbial biodiversity (47).

### Pseudomonas putida

*Pseudomonas putida* is a Gram-negative, rod-shaped and soil bacterium. Its peptidoglycan includes small amounts of L-Arg and L-Lys in addition to common components in Gram-negative bacteria (48). Pseudomonads have been extensively studied as a good model of D-amino acid study. For instance, a PLP-dependent Arg racemase with broad substrate specificity has been isolated in *Pseudomonas taetrolens* in 1971 (24). Recent studies revealed that Arg racemase localized in the periplasm has racemase activity towards 11 amino acids with the highest racemase activity towards basic amino acids (Table I), whereas Ala racemase is specific for Ala (25). *P. putida* also has an Ala racemase (DadX) and a broad-spectrum amino acid racemase (Alr) found in the periplasm (48, 49) besides Glu racemase. DadX shows the highest

activity towards Ala and measurable activities with L-Ser and D-Cys. Alr can synthesize 9 D-amino acids and preferentially catalyses interconversions of Lys and Arg (Table I). Alr plays a possible role to convert L-Lys to D-Lys, and D-Lys is subsequently catabolized through L-pipecolate metabolic pathway (48). D-Arg catabolic pathway is, however, unclear in *P. putida*, because two Arg dehydrogenases (DauA and DauB) found in *Pseudomonas aeruginosa* (50) is not present in *P. putida*.

### Thermus thermophilus

*Thermus thermophilus* HB8 is an extremely thermophilic bacterium that was originally isolated from a hot spring in Shizuoka, Japan (51). It is a Gram-negative, rod-shaped, non-sporulating aerobic bacterium. Its optimum temperature for growth is between 65°C and 72°C, and it can grow at temperatures as high as 85°C. The fundamental structure of its peptidoglycan is GlcNAc-MurNAc-L-Ala-D-Glu-L-Orn-D-Ala-D-Ala; moreover, the peptidoglycan does not contain meso-Dpm, a canonical component of peptidoglycan in Gram-negative bacteria (52). Two D-amino acids in peptidoglycan, D-Ala and D-Glu, are synthesized from their corresponding L-amino acids by Ala racemase (TTHA 0062) (53) and Glu racemase (TTHA 1643) (54), respectively. TTHA 0062 has a monomeric structure and is apparently PLP-dependent. Based on the optimal pH and an optimal temperature of this enzyme, it may function in the growth range of *T. thermophilus* HB8 (53).

We demonstrated that TTHA 1643 from *T. thermophilus* HB8 has Glu racemase activity (54). TTHA 1643 is a PLP-independent enzyme with a tetrameric structure. Glu racemases are usually PLP-independent, and racemization is generally catalyzed by two Cys residues in the active site; TTHA 1643 has two corresponding Cys residues (Cys 96 and 197) in its active site. TTHA 1643 has a strict substrate specificity and exhibits racemase activity only towards Glu, and not towards other 20 amino acids [Asp, Ala, Ser, threonine (Thr), glutamine (Gln), asparagine (Asn), Met, Val, Leu, isoleucine (Ile), Lys, Arg, phenylalanine (Phe), tryptophan (Trp), tyrosine (Tyr), His, Orn, citrulline (Cit), Aba and Hse]. The catalytic efficiencies of TTHA 1643 towards D- and L-Glu are comparable. Based on its pH and temperature profiles (54), it can function adequately as a Glu racemase under the growth conditions of *T. thermophilus* HB8. In fact, growth of a TTHA 1643-deficient strain in minimal medium lacking D-Glu is clearly impaired relative to the wild-type, although TTHA 1643 is not essential for the growth of *T. thermophilus* HB8. Thus, *T. thermophilus* may have an alternative metabolic pathway for production of D-Glu. Further studies are required to elucidate alternative pathways of D-Glu biosynthesis in this bacterium.

### Thermotoga maritima

*Thermotoga maritima* is a hyperthermophilic bacterium that was isolated from geothermally heated sea floor (55). It is a strictly anaerobic, rod-shaped, Gram-negative bacterium that is surrounded by a characteristic sheath-like structure. Its growth optimum temperature is 80°C and it can grow at temperatures up to 90°C. Intriguingly, its peptidoglycan contains an unusual D-Lys in addition to typical D-Ala and D-Glu, but does not contain meso-Dpm (55). Typical structures of its peptidoglycan peptide stems are GlcNAc–MurNAc–L-Ala–D-Glu–L-Lys–D-Ala and GlcNAc–MurNAc–L-Ala–D-Glu–D-Lys (56). D-Lys is an exceptionally rare component of peptidoglycan and had been previously reported only in *Butyribacterium rettgeri* (57). It remains unclear whether D-Lys contributes to the cellular thermostability of this species.

We identified and characterized a Lys racemase (TM1597) from *T. maritima* that produces D-Lys from L-Lys (58). TM1597 is a PLP-dependent homodimeric enzyme that catalyzes the racemization of 11 amino acids, with the highest activity towards Lys (Table I); however, its catalytic efficiency is highest towards Orn rather than Lys. D-Orn is a component of peptidoglycan in *B. rettgeri*, but its physiological roles in bacteria remain poorly understood (57). TM1597 can also produce D-Ala from L-Ala, although its racemase activity towards L-Ala is much lower than towards L-Lys and L-Orn, and the racemase reaction can occur within the growth range of *T. maritima* based on the pH and temperature dependencies (58). Thus, TM1597 is likely involved not only in D-Lys biosynthesis, but also in D-Ala biosynthesis.

In addition, we identified and characterized a Dpm epimerase (TM1522) from *T. maritima* (58). It is involved in the penultimate step of L-Lys biosynthesis in *T. maritima*. Dpm epimerase belongs to a family of PLP-independent amino acid racemases; consistent with this, TM1522 does not require PLP as a cofactor. Dpm epimerase usually has a pair of Cys residues that catalyze epimerization via deprotonation or reprotonation, and TM1522 with a trimeric structure has two putative catalytic residues, Cys 64 and Cys186. Intriguingly, TM1522 catalyzes not only epimerization of Dpm but also racemization of 13 amino acids including Lys (Table I) but also the racemase activity is markedly lower than the epimerase activity. On the other hand, Dpm epimerase from *Chlamydia trachomatis* has Glu racemase activity and plays a physiological role in D-Glu production (59). The catalytic efficiency of TM1522 is much higher (>10,000-fold) towards Dpm than Lys. The pH and temperature profiles of TM1522 allow the enzyme to function as a Dpm epimerase under the growth conditions for *T. maritima* (58). Taken together, these observations imply that TM1522 is involved in interconversion between LL-Dpm and meso-Dpm rather than the racemization of amino acids.

The biosynthetic pathway for L-Lys in *T. maritima* comprises nine steps. The starting material, L-Asp, is converted to LL-Dpm via seven steps. LL-Dpm is then subjected to epimerization into meso-Dpm by TM1522, and then meso-Dpm is converted to L-Lys by Dpm decarboxylase (TM1517). Finally, D-Lys is produced from L-Lys by TM1597. Given that TM1270 has both Ala and Glu racemase activity,
TM1597 and cystathionine β-lyase (TM1270) may be involved in the production of D-Ala and D-Glu (60). The biosynthetic pathways of these D-amino acids should be elucidated in future studies.

# Concluding Remarks

Extensive studies have revealed that bacteria synthesize not only common D-amino acids that are components of peptidoglycan but also multiple D-amino acids with physiological functions in spore germination and regulation of bacterial communities (47, 61). These D-amino acids are mainly produced by amino acid racemases, but L-amino acid-metabolizing enzymes such as cystathionine β-lyase, which has racemase activity, are involved in the production of D-amino acids (35). Therefore, bacteria can possess unexpected metabolic pathways of D-amino acids, and further studies may lead to discovery of new enzymes with racemase activity and disclosure of novel metabolic pathways. In addition, increasing attention has been focused on the physiological actions of D-amino acids of bacterial origin, and recent studies have demonstrated that D-amino acids influence host–bacteria interaction. For example, D-Trp produced by probiotic bacteria affects gut microbial flora and ameliorates allergic airway inflammation (62), and D-Ser derived from gut microbiota protects and aids in recovery from acute kidney injury (63). Elucidation of the metabolic pathways and physiological actions of D-amino acids in bacteria and their hosts should provide a basis for understanding the D-amino acid biosystem and could contribute to maintenance of human health and the development of cures for illness.

# Acknowledgements

We are deeply grateful to our laboratory staff and students for their generous support. We thank Prof Tairo Oshima and Dr Toshiyuki Moriya, Institute of Environmental Microbiology, Kyowa Kako Co., for a fruitful research collaboration.

# Funding

Our works described in this review were supported by the Japan Society for the Promotion of Science (JSPS) KAKENHI grant no. 17K18082 (to T.M.) and a Kitasato University Research Grant for Young Researchers (to T.M.).

# Conflict of Interest

None declared.

# References

1. Vollmer, W., Blanot, D., and de Pedro, M.A. (2008) Peptidoglycan structure and architecture. FEMS Microbiol. Rev. 32, 149–167
2. Radkov, A.D. and Moe, L.A. (2014) Bacterial synthesis of D-amino acids. Appl. Microbiol. Biotechnol. 98, 5363–5374
3. Hernández, S.B. and Cava, F. (2016) Environmental roles of microbial amino acid racemases. Environ. Microbiol. 18, 1673–1685
4. Lam, H., Oh, D.C., Cava, F., Takacs, C.N., Clardy, J., de Pedro, M.A., and Waldor, M.K. (2009) D-Amino acids govern stationary phase cell wall remodeling in bacteria. Science 325, 1552–1555
5. Cava, F., de Pedro, M.A., Lam, H., Davis, B.M., and Waldor, M.K. (2011) Distinct pathways for modification of the bacterial cell wall by non-canonical D-amino acids. EMBO J. 30, 3442–3453
6. Leiman, S.A., May, J.M., Lebar, M.D., Kahne, D., Kolter, R., and Losick, R. (2013) D-Amino acids indirectly inhibit biofilm formation in *Bacillus subtilis* by interfering with protein synthesis. J. Bacteriol. 195, 5391–5395
7. Kolodkin-Gal, I., Romero, D., Cao, S., Clardy, J., Kolter, R., and Losick, R. (2010) D-Amino acids trigger biofilm disassembly. Science 328, 627–629
8. Hochbaum, A.I., Kolodkin-Gal, I., Foulston, L., Kolter, R., Aizenberg, J., and Losick, R. (2011) Inhibitory effects of D-amino acids on *Staphylococcus aureus* biofilm development. J. Bacteriol. 193, 5616–5622
9. Sanchez, Z., Tani, A., and Kimbara, K. (2013) Extensive reduction of cell viability and enhanced matrix production in *Pseudomonas aeruginosa* PAO1 flow biofilms treated with a D-amino acid mixture. Appl. Environ. Microbiol. 79, 1396–1399
10. Ramón-Peréz, M.L., Diaz-Cedillo, F., Ibarra, J.A., Torales-Carden˜a, A., Rodríguez-Martínez, S., Jan-Roblero, J., Cancino-Diaz, M.E., and Cancino-Diaz, J.C. (2014) D-Amino acids inhibit biofilm formation in *Staphylococcus epidermidis* strains from ocular infections. J. Med. Microbiol. 63, 1369–1376
11. Rosen, E., Tsesis, I., Elbahary, S., Storzi, N., and Kolodkin-Gal, I. (2016) Eradication of *Enterococcus faecalis* biofilms on human dentin. Front. Microbiol. 7, 2055
12. Zilm, P.S., Butnejski, V., Rossi-Fedele, G., Kidd, S.P., Edwards, S., and Vasilev, K. (2017) D-Amino acids reduce *Enterococcus faecalis* biofilms in vitro and in the presence of antimicrobials used for root canal treatment. PLoS One 12, e0170670
13. Warraich, A.A., Mohammed, A.R., Perrie, Y., Hussain, M., Gibson, H., and Rahman, A. (2020) Evaluation of anti-biofilm activity of acidic amino acids and synergy with ciprofloxacin on *Staphylococcus aureus* biofilms. Sci. Rep. 10, 9021
14. Li, E., Wu, J., Wang, P., and Zhang, D. (2016) D-Phenylalanine inhibits biofilm development of a marine microbe, *Pseudoalteromonas* sp. SC2014. FEMS Microbiol. Lett. 363, fnw198
15. Miyamoto, T., Katane, M., Saitoh, Y., Sekine, M., and Homma, H. (2020) Involvement of penicillin-binding proteins in the metabolism of a bacterial peptidoglycan containing a non-canonical D-amino acid. Amino Acids 52, 487–497
16. Yamashita, T., Ashiuchi, M., Ohnishi, K., Kato, S., Nagata, S., and Misono, H. (2004) Molecular identification of monomeric aspartate racemase from *Bifidobacterium bifidum*. Eur. J. Biochem. 271, 4798–4803
17. Yamauchi, T., Choi, S.Y., Okada, H., Yohda, M., Kumagai, H., Esaki, N., and Soda, K. (1992) Properties of aspartate racemase, a pyridoxal 5′-phosphate-independent amino acid racemase. J. Biol. Chem. 267, 18361–18364
18. Arias, C.A., Weisner, J., Blackburn, J.M., and Reynolds, P.E. (2000) Serine and alanine racemase activities of VanT: a protein necessary for vancomycin resistance in *Enterococcus gallinarum* BM4174. Microbiology 146, 1727–1734
T. Miyamoto and H. Homma

19. Kubota, T., Shimamura, S., Kobayashi, T., Nunoura, T., and Deguchi, S. (2016) Distribution of eukaryotic serine racemases in the bacterial domain and characterization of a representative protein in *Roseobacter litoralis* Och 149. *Microbiology* **162**, 53–61

20. Stadtman, T.C. and Elliott, P. (1957) Studies on the enzymic reduction of amino acids. II. Purification and properties of D-proline reductase and a proline racemase from *Clostridium sticklandii*. *J. Biol. Chem.* **228**, 983–997

21. Goytia, M., Chamond, N., Cosson, A., Coatnoan, N., Hermant, D., Berneman, A., and Minoprio, P. (2007) Molecular and structural discrimination of proline racemase and hydroxyproline-2-epimerase from nosocomial and bacterial pathogens. *PLoS One* **2**, e885

22. Kuan, Y.C., Kao, C.H., Chen, C.H., Chen, C.C., Hu, H.Y., and Hsu, W.H. (2011) Biochemical characterization of a novel lysine racemase from *Proteus mirabilis* BCRC 10725. *Process Biochem.* **46**, 1914–1920

23. Kato, S., Hemmi, H., and Yoshimura, T. (2012) Lysine racemase from a lactic acid bacterium, *Oenococcus oeni*: structural basis of substrate specificity. *J. Biochem.* **152**, 505–508

24. Yorifuji, T. and Ogata, K. (1971) Arginine racemase of *Pseudomonas graveolens*. I. Purification, crystallization, and properties. *J. Biol. Chem.* **246**, 5085–5092

25. Matsui, D., Oikawa, T., Arakawa, N., Osumi, S., Lausberg, F., Stäbler, N., Freudl, R., and Eggeling, L. (2009) A periplasmic, pyridoxal-5′-phosphate-dependent amino acid racemase in *Pseudomonas taetrolens*. *Appl. Microbiol. Biotechnol.* **83**, 1045–1054

26. Adachi, M., Shimizu, R., Kato, S., and Oikawa, T. (2019) The first identification and characterization of a histidine-specific amino acid racemase, histidine racemase from a lactic acid bacterium, *Leuconostoc mesenteroides* subsp. sake NBRC 102480. *Amino Acids* **51**, 331–343

27. Grohs, P., Gutmann, L., Legrand, R., Schoot, B., and Mainardi, J.L. (2000) Vancomycin resistance is associated with serine-containing peptidoglycan in *Enterococcus gallinarum*. *J. Bacteriol.* **182**, 6228–6232

28. Bellais, S., Arthur, M., Dubost, L., Hugonnet, J.E., Gutmann, L., van Heijenoort, J., Legrand, R., Brouard, J.P., Rice, L., and Mainardi, J.L. (2006) Asl<sub>fm</sub>, the D-aspartate ligase responsible for the addition of D-aspartic acid onto the peptidoglycan precursor of *Enterococcus faecium*. *J. Biol. Chem.* **281**, 11586–11594

29. Wiseman, J.S. and Nichols, J.S. (1984) Purification and properties of diaminopimelic acid epimerase from *Escherichia coli*. *J. Biol. Chem.* **259**, 8907–8914

30. Walsh, C.T. (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. *J. Biol. Chem.* **264**, 2393–2396

31. He, W., Li, C., and Lu, C.D. (2011) Regulation and characterization of the *dadRAX* locus for D-amino acid catabolism in *Pseudomonas aeruginosa* PAO1. *J. Bacteriol.* **193**, 2107–2115

32. Miyamoto, T., Sekine, M., Ogawa, T., Hidaka, M., Homma, H., and Masaki, H. (2010) Detection of D-amino acids in purified proteins synthesized in *Escherichia coli*. *Amino Acids* **38**, 1377–1385

33. Miyamoto, T., Katane, M., Saitoh, Y., Sekine, M., and Homma, H. (2017) Identification and characterization of novel broad-spectrum amino acid racemases from *Escherichia coli* and *Bacillus subtilis*. *Amino Acids* **49**, 1885–1894

34. Soutourina, J., Plateau, P., and Blanquet, S. (2000) Metabolism of D-aminoacyl-tRNAs in *Escherichia coli*

and *Saccharomyces cerevisiae* cells. *J. Biol. Chem.* **275**, 32535–32542

35. Miyamoto, T., Katane, M., Saitoh, Y., Sekine, M., and Homma, H. (2018) Cystathionine β-lyase is involved in D-amino acid metabolism. *Biochem. J.* **475**, 1397–1410

36. Dwivedi, C.M., Ragin, R.C., and Uren, J.R. (1982) Cloning, purification, and characterization of β-cystathionase from *Escherichia coli*. *Biochemistry* **21**, 3064–3069

37. Zdych, E., Peist, R., Reidl, J., and Boos, W. (1995) MalY of *Escherichia coli* is an enzyme with the activity of a βC-S lyase (cystathionase). *J. Bacteriol.* **177**, 5035–5039

38. Kang, L., Shaw, A.C., Xu, D., Xia, W., Zhang, J., Deng, J., Wöldike, H.F., Liu, Y., and Su, J. (2011) Upregulation of MetC is essential for D-alanine-independent growth of an *alr/dadX*-deficient *Escherichia coli* strain. *J. Bacteriol.* **193**, 1098–1106

39. Kato, S. and Oikawa, T. (2018) A novel bifunctional amino acid racemase with multiple substrate specificity, MalY from *Lactobacillus sakei* LT-13: genome-based identification and enzymological characterization. *Front. Microbiol.* **9**, 403

40. Dupourque, D., Newton, W.A., and Snell, E.E. (1966) Purification and properties of D-serine dehydrase from *Escherichia coli*. *J. Biol. Chem.* **241**, 1233–1238

41. Connolly, J.P., Goldstone, R.J., Burgess, K., Cogdell, R.J., Beatson, S.A., Vollmer, W., Smith, D.G., and Roe, A.J. (2015) The host metabolite D-serine contributes to bacterial niche specificity through gene selection. *ISME J.* **9**, 1039–1051

42. Kimura, K., Tran, L.P., and Itoh, Y. (2004) Roles and regulation of the glutamate racemase isogenes, *racE* and *yrpC*, in *Bacillus subtilis*. *Microbiology* **150**, 2911–2920

43. Kada, S., Nanamiya, H., Kawamura, F., and Horinouchi, S. (2004) Glr, a glutamate racemase, supplies D-glutamate to both peptidoglycan synthesis and poly-gamma-glutamate production in gamma-PGA-producing *Bacillus subtilis*. *FEMS Microbiol. Lett.* **236**, 13–20

44. Tanizawa, K., Masu, Y., Asano, S., Tanaka, H., and Soda, K. (1989) Thermostable D-amino acid aminotransferase from a thermophilic *Bacillus* species. Purification, characterization, and active site sequence determination. *J. Biol. Chem.* **264**, 2445–2449

45. Yonaha, K., Misono, H., Yamamoto, T., and Soda, K. (1975) D-Amino acid aminotransferase of *Bacillus sphaericus*. Enzymologic and spectrometric properties. *J. Biol. Chem.* **250**, 6983–6989

46. Espaillat, A., Carrasco-López, C., Bernardo-García, N., Pietrosemoli, N., Otero, L.H., Álvarez, L., de Pedro, M.A., Pazos, F., Davis, B.M., Waldor, M.K., Hermoso, J.A., and Cava, F. (2014) Structural basis for the broad specificity of a new family of amino-acid racemases. *Acta Crystallogr. D Biol. Crystallogr.* **70**, 79–90

47. Alvarez, L., Aliashkevich, A., de Pedro, M.A., and Cava, F. (2018) Bacterial secretion of D-arginine controls environmental microbial biodiversity. *ISME J.* **12**, 438–450

48. Radkov, A.D. and Moe, L.A. (2018) A broad spectrum racemase in *Pseudomonas putida* KT2440 plays a key role in amino acid catabolism. *Front. Microbiol.* **9**, 1343

49. Radkov, A.D. and Moe, L.A. (2013) Amino acid racemization in *Pseudomonas putida* KT2440. *J. Bacteriol.* **195**, 5016–5024

50. Li, C. and Lu, C.D. (2009) Arginine racemization by coupled catabolic and anabolic dehydrogenases. *Proc. Natl. Acad. Sci. USA* 106, 906–911

51. Oshima, T. and Imahori, K. (1974) Description of *Thermus thermophilus* (Yoshida and Oshima) comb. nov., a nonsporulating thermophilic bacterium from a Japanese thermal spa. *Int. J. Syst. Evol. Microbiol.* 24, 102–112

52. Quintela, J.C., Pittenauer, E., Allmaier, G., Arán, V., and de Pedro, M.A. (1995) Structure of peptidoglycan from *Thermus thermophilus* HB8. *J. Bacteriol.* 177, 4947–4962

53. Seow, T.K., Inagaki, K., Nakamura, T., Maeda, R., Tamura, T., and Tanaka, H. (2000) Purification and some characteristics of a monomeric alanine racemase from an extreme thermophile. *Thermus thermophilus*. *J. Biosci. Bioeng.* 90, 344–346

54. Miyamoto, T., Moriya, T., Homma, H., and Oshima, T. (2020) Enzymatic properties and physiological function of glutamate racemase from *Thermus thermophilus*. *Biochim. Biophys. Acta Proteins Proteom.* 1868, 140461

55. Huber, R., Langworthy, T.A., Kjønig, H., Thomm, M., Woese, C.R., Sleytr, U.B., and Stetter, K.O. (1986) *Thermotoga maritima* sp. nov. represents a new genus of unique extremely thermophilic eubacteria growing up to 90°C. *Arch. Microbiol.* 144, 324–333

56. Boniface, A., Parquet, C., Arthur, M., Mengin-Lecreulx, D., and Blanot, D. (2009) The elucidation of the structure of *Thermotoga maritima* peptidoglycan reveals two novel types of cross-link. *J. Biol. Chem.* 284, 21856–21862

57. Guinand, M., Ghuysen, J.M., Schleifer, K.H., and Kandler, O. (1969) The peptidoglycan in walls of *Butyribacterium rettgeri*. *Biochemistry* 8, 200–207

58. Miyamoto, T., Katane, M., Saitoh, Y., Sekine, M., and Homma, H. (2019) Elucidation of the D-lysine

biosynthetic pathway in the hyperthermophile *Thermotoga maritima*. *FEBS J.* 286, 601–614

59. Liechti, G., Singh, R., Rossi, P.L., Gray, M.D., Adams, N.E., and Maurelli, A.T. (2018) *Chlamydia trachomatis dapF* encodes a bifunctional enzyme capable of both D-glutamate racemase and diaminopimelate epimerase activities. *MBio* 9, e00204-18

60. Ferla, M.P., Brewster, J.L., Hall, K.R., Evans, G.B., and Patrick, W.M. (2017) Primordial-like enzymes from bacteria with reduced genomes. *Mol. Microbiol.* 105, 508–524

61. McKevitt, M.T., Bryant, K.M., Shakir, S.M., Larabee, J.L., Blanke, S.R., Lovchik, J., Lyons, C.R., and Ballard, J.D. (2007) Effects of endogenous D-alanine synthesis and autoinhibition of *Bacillus anthracis* germination on in vitro and in vivo infections. *Infect. Immun.* 75, 5726–5734

62. Kepert, I., Fonseca, J., Müller, C., Milger, K., Hochwind, K., Kostric, M., Fedoseeva, M., Ohnmacht, C., Dehmel, S., Nathan, P., Bartel, S., Eckelberg, O., Schlöter, M., Hartmann, A., Schmitt-Kopplin, P., and Krauss-Etschmann, S. (2017) D-Tryptophan from probiotic bacteria influences the gut microbiome and allergic airway disease. *J. Allergy Clin. Immunol.* 139, 1525–1535

63. Nakade, Y., Iwata, Y., Furuichi, K., Mita, M., Hamase, K., Konno, R., Miyake, T., Sakai, N., Kitajima, S., Toyama, T., Shinozaki, Y., Sagara, A., Miyagawa, T., Hara, A., Shimizu, M., Kamikawa, Y., Sato, K., Oshima, M., Yoneda-Nakagawa, S., Yamamura, Y., Kaneko, S., Miyamoto, T., Katane, M., Homma, H., Morita, H., Suda, W., Hattori, M., and Wada, T. (2018) Gut microbiota-derived D-serine protects against acute kidney injury. *JCI Insight* 3, e97957
