
Oncogene (2021) 40:3637–3654  
https://doi.org/10.1038/s41388-021-01781-x  

---

### REVIEW ARTICLE

#### Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy

**Marek Wanior**<sup>1,2</sup> • **Andreas Krämer**<sup>1,2,3</sup> • **Stefan Knapp**<sup>1,2,3,4</sup> • **Andreas C. Joerger**<sup>1,2,4</sup>

Received: 23 November 2020 / Revised: 15 March 2021 / Accepted: 6 April 2021 / Published online: 3 May 2021  

© The Author(s) 2021. This article is published with open access.

---

### Abstract

Multi-subunit ATPase-dependent chromatin remodelling complexes SWI/SNF (switch/sucrose non-fermentable) are fundamental epigenetic regulators of gene transcription. Functional genomic studies revealed a remarkable mutation prevalence of SWI/SNF-encoding genes in 20–25% of all human cancers, frequently driving oncogenic programmes. Some SWI/SNF-mutant cancers are hypersensitive to perturbations in other SWI/SNF subunits, regulatory proteins and distinct biological pathways, often resulting in sustained anticancer effects and synthetic lethal interactions. Exploiting these vulnerabilities is a promising therapeutic strategy. Here, we review the importance of SWI/SNF chromatin remodellers in gene regulation as well as mechanisms leading to assembly defects and their role in cancer development. We will focus in particular on emerging strategies for the targeted therapy of SWI/SNF-deficient cancers using chemical probes, including proteolysis targeting chimeras, to induce synthetic lethality.

---

### Introduction: SWI/SNF chromatin remodelling complexes as potential targets for cancer therapy

For many decades, oncologists have mostly used non- or poorly specific treatment options such as chemotherapy, causing considerable harm to non-cancerous tissues. Recently, new approaches, including immunotherapy and targeted therapy, have increased the number of therapy options for the treatment of some cancers, resulting in a considerable improvement in treatment efficacy and a reduction of side effects. The discovery of new cancer-specific genetic or epigenetic liabilities will help to develop next-generation anti-cancer drugs and personalised medicines.

Epigenetic mechanisms, including chromatin remodelling, control access to a specific genetic locus on chromatin. Epigenetic alterations are involved in the development of many diseases, in particular in tumourigenesis, where dysregulated epigenetic modulators often constitute strong oncogenic drivers [1]. The role of chromatin remodellers in human carcinogenesis has recently been demonstrated in a number of genome-wide and exome-wide sequencing studies, revealing that genes encoding components of the SWI/SNF complex are mutated in 20–25% of human cancers [2]. This high mutation rate indicates tumour suppressive roles, which leads to the question of how perturbations of SWI/SNF chromatin remodellers contribute to tumourigenesis, how they promote tumour growth and, more importantly, whether SWI/SNF complexes can be targeted therapeutically. Targeting chromatin remodelling complexes appears challenging at first glance because many SWI/SNF-associated cancers exhibit loss-of-function mutations due to missense or nonsense mutations, or the complete loss of individual SWI/SNF-subunits [3, 4]. But, functional genomic studies have identified vulnerabilities with other genes causing sustained synthetic lethal interactions [5–7]. Synthetic lethality occurs when the combination of deficiencies in the expression or function of two genes leads to cell death, whereas a deficiency in only one of them results in a viable phenotype [8]. Cancers with

---

**Stefan Knapp**  
knapp@pharmchem.uni-frankfurt.de  

**Andreas C. Joerger**  
joerger@pharmchem.uni-frankfurt.de  

1. Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Frankfurt am Main, Germany  
2. Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Frankfurt am Main, Germany  
3. Frankfurt Cancer Institute (FCI), Frankfurt am Main, Germany  
4. German Translational Cancer Network (DKTK) site Frankfurt/Mainz, Frankfurt am Main, Germany  

ATPase domain DNA-binding domain Bromodomain Chromodomain PHD-finger domain

canonical BAF PBAF non-canonical BAF (ncBAF/GBAF)

Fig. 1 Subunit composition and assembly of mammalian SWI/SNF chromatin remodelling complexes BAF, PBAF and ncBAF. Illustration based on the cryo EM structures of nucleosome-bound BAF and the latest data on SWI/SNF complex assembly [16–18].

Nucleosomes are bound via DNA- and histone-binding domains. Shared SWI/SNF subunits are coloured in grey, BAF subunits not found in ncBAF in orange, PBAF-specific subunits in green and ncBAF-specific subunits in blue.

SMARCA4 deficiency, a genetic alteration frequently observed in malignant solid tumours or lung adenocarcinomas, for example, show a dependency on the paralogue SMARCA2 [6]. They are also sensitive to perturbations in the SWI/SNF interactome, including protein kinases such as Aurora A [9] or cyclin-dependent kinase 4/6 (CDK4/6) [10]. In this review, we discuss the recent progress in targeting SWI/SNF-mutant cancers with highly specific inhibitors, also called “chemical probes”, as well as FDA-approved drugs that may pave the way for new therapeutic concepts to treat cancer patients.

### Structural assembly and function of SWI/SNF complexes

The human genome consists of three billion base pairs, which corresponds to an approximately two-metre-long DNA molecule that has to be condensed into a tight chromatin structure in order to fit into the nucleus of a cell with a diameter of only 5 μm [11]. The basic units of chromatin are nucleosomes, consisting of 147 base pairs of DNA wrapped around an octamer of four core histones (H2A, H2B, H3 and H4), resulting in a compact chromatin structure. Dense chromatin DNA, termed heterochromatin, is inaccessible to protein binding, with the result that gene expression is largely inactivated. Whenever the chromatin structure is altered to an open euchromatin state through epigenetic mechanisms, such as DNA demethylation or histone acetylation, and the action of chromatin remodellers, transcription factors and associated proteins can access their target genes and initiate gene expression, thereby switching on signalling cascades and biological pathways [11].

Four evolutionarily conserved classes of ATP-dependent chromatin remodelling complexes have been discovered in mammals: ISWI, CHD, INO80 and SWI/SNF [12]. These multiprotein complexes (up to 2 MDa in size) all share a central Snf2-like ATPase domain, but they differ in the composition of interacting subunits and recruited proteins, which determines their biological roles (Fig. 1, Table 1) [12, 13]. SWI/SNF complexes, for example, consist of a combinatorial product of at least 29 proteins. The discovery of SWI/SNF chromatin remodellers and their evolutionary history have been described in detail elsewhere [13]. SWI/SNF complexes are essential for cell proliferation and differentiation, but the exact mechanisms of how these remodellers carry out their role are currently poorly understood. They contain several nucleosome-binding domains, including DNA-binding domains, such as AT-rich inter-active domains (ARIDs), zinc-finger domains or high-mobility group box domains (HMGs), and histone-binding domains, such as bromodomains (BRDs), plant homeodomains (PHDs) and chromodomains, which are found in most SWI/SNF subunits (Fig. 1, Table 1) [12, 14].

Three mammalian SWI/SNF complexes have been identified: BAF (BRG1/BRM-associated factor), PBAF (polybromo-associated BAF) and, very recently, non-canonical BAF (ncBAF or GBAF) [15, 16]. They all contain a mutually exclusive catalytic ATPase subunit, SMARCA2 or SMARCA4, and share a number of associated proteins (Fig. 1, Table 1). BAF and PBAF, for example, both contain an ARID and a SMARCB1 subunit. The recently published cryo electron microscopy (EM) structures of recombinant and endogenous human BAF complexes have provided exciting new insights into the overall assembly of the complex and the mechanisms of how nucleosomes interact with BAF (Fig. 2) [17, 18]. These

Table 1 Binding interactions/functions of SWI/SNF subunits and their role in cancer.

| Complex | SWI/SNF subunits (genes) | Domains$^a$ | Chromatin interaction | SWI/SNF-associated cancer |
| --- | --- | --- | --- | --- |
| BAF/PBAF/ncBAF | BRG1 (SMARCA4) | ATPase, helicase, bromodomain BRK, DNA binding, histone binding | Breast cancer, cutaneous cancer, B-cell lymphoma, bladder carcinoma, non-small cell lung cancer, melanoma, glioma carcinoma, hepatocellular carcinoma |  |
|  | BRM (SMARCA2) | HSA, QLQ, SnAC [130] | Ovarian small cell carcinoma, sarcoma, lung adenocarcinoma, hepatocellular carcinoma |  |
|  | BAF53A (ACTL6A) | ATPase, helicase, bromodomain, BRK, DNA binding, histone binding | Squamous cell carcinoma, rhabdomyosarcoma |  |
|  | SS18* (SS18) | Actin-like | Synovial sarcoma, Ewing sarcoma |  |
|  | β-actin (ACTB) | No domain annotation | Abnormal expression and polymerisation of ACTB in most cancers leading to changes of the cytoskeleton associated with invasiveness and metastasis [133]. |  |
|  | BCL7 (A, B or C) | Armadillo repeats | Rhabdoid cancer, small cell lung cancer |  |
|  | BAF155 (SMARCC1) | Chromodomain, SWIRM, SANT | Histone binding | Gastric cancer, colorectal cancer, pancreatic cancer |  |
|  | BAF170$^s$ (SMARCC2) | Chromodomain, SWIRM, SANT | Histone binding | Multiple myeloma, non-Hodgkin lymphoma |  |
|  | BAF60 (A, B$^s$ or C$^s$) (SMARCD1, 2 or 3) | Conserved N-terminal domain | Not reported | Lung cancer, gastric cancer |  |
|  | BAF57$^s$ (SMARCE1) | SWIB | Breast cancer, endometrial carcinoma, ovarian carcinoma, meningioma |  |
|  | BAF47$^s$ (SMARCB1) | HMG box | Malignant rhabdoid tumour, prostate cancer, renal medullary carcinoma, epithelioid sarcoma, familial schwannomatosis |  |
| BAF | BAF250 (A or B) (ARIDIA or B) | Winged helix DNA-binding domain [34, 35] | DNA binding acidic patch binding | Ovarian clear cell carcinoma, uterine endometrial cancer, neuroblastoma, colorectal cancer |  |
|  | BAF45 (B, C or D) (DPFL, 2 Zinc finger/PHD finger or 3) | ARID (AT-rich interactive domain) | DNA binding | Colorectal cancer |  |
| PBAF | BAF200 (ARID2) | Bromodomain, DUF3512 | Histone binding | Melanoma, non-small cell lung cancer, hepatocellular carcinoma, colorectal carcinoma |  |
|  | BRD7 (BRD7) | ARID (AT-rich interactive domain) | DNA binding, histone binding | Hepatocellular carcinoma, nasopharyngeal carcinoma, lung cancer, ovarian carcinoma |  |
|  | BAF180 (PBRM1) | Bromodomain, BAH domain, HMG box | Histone binding | Clear cell renal cell carcinoma |  |
|  | BAF45A (PHF10) | PHD finger | Histone binding | Gastric cancer |  |
| ncBAF | BRD9 (BRD9) | Bromodomain, DUF3512 | Histone binding | Nut midline carcinoma, rhabdoid cancer |  |
|  | GLTSCR1 (BICRA) | No annotation | Not reported, BRD4 interactor | Glioma tumour, prostate cancer |  |
|  | GLTSCR1L (BICRAL) | No annotation | Not reported, BRD4 interactor | Prostate cancer |  |

$^a$Selected domains annotated in UniProt or the indicated reference. *Found only in BAF and ncBAF. $^s$Found only in BAF and PBAF.

Fig. 2 Cryo-EM structure of human BAF in complex with nucleosome core particle. Individual subunits are shown in different colours (PDB entry 6LTJ) [17]. ARID1A serves as the core of the base module stabilising the entire complex, while SMARCA4 and SMARCB1 bind the nucleosome. The insert shows the ATPase domain of SMARCA4 in complex with an allosteric ATPase inhibitor (PDB entry 6EG3) [74].

|  |  |
| --- | --- |
| SMARCA4 | H2B |
| ARID1A | DPF2 |
| H2A | H3 |
| SMARCD1 | H4 |
| SMARCE1 | ACTL6A |
| SMARCB1 | ACTB |
| SMARCC2b | DNA |
| SMARCC2a |  |

DNA Entry  
ATPase Lobe 2  
Histone-DNA Complex  

ATPase Lobe 1  

Head  

DNA Exit  

Thumb  

Compound 15  

Palm  

Fingers  

BAF Complex  

structures revealed a modular organisation of the complex that is consistent with earlier data on the complex assembly pathway [16], and also provided the first mechanistic insights into ATP-driven rearrangements causing histone eviction. ARID1A forms the structural core of the base module, with SMARCC1/2 serving as scaffolding proteins and SMARCE1/D1 supporting complex formation. The nucleosome is sandwiched between SMARCB1 and the ATPase module SMARCA4, which both interact with nucleosome H2A/H2B acidic patch regions. ATP-driven engagement of SMARCA4 with the nucleosome eventually leads to nucleosome eviction, creating an elongated stretch of nucleosome-free DNA that is accessible for transcription factors.

PBAF, the second main SWI/SNF chromatin remodeller in humans, shares high homology with BAF, but there are also significant differences in its subunit composition [16]. ARID1A, the structural core of the base module of the BAF complex is replaced in PBAF by the paralogue ARID2. In addition, PBAF contains the bromodomain-containing protein BRD7. A particularly interesting feature that is unique to PBAF and hence gives it its name: the subunit PBRM1 (polybromo-1), which contains six tandem-acting bromodomains (PB1(1-6)). In the smaller ncBAF, the central ARID1A subunit of BAF is replaced by the glioma tumour suppressor candidate region gene 1 (GLTSCR1) subunit [15]. ncBAF further lacks other base module BAF subunits, including SMARCC2, SMARCE1 and the nucleosome-recognition unit SMARCB1, but features the bromodomain-containing BRD9 subunit, which is not present in the other

two complexes, according to the most recent systematic study of SWI/SNF complex assembly [16].

Due to their combinatorial and hence structural diversity, the three human SWI/SNF chromatin remodellers interact specifically with various enhancers and promoters in a cell-type-specific manner. BAF complex subunits also undergo distinct switches during development, resulting in diverse transcriptional signatures [19]. BAF is generally associated with binding to enhancers, whereas PBAF or ncBAF are frequently enriched at promoter regions [20, 21]. SMARCA4, for example, is known to co-localise with H3K27ac at enhancers regulating lineage-specific developmental programmes [22]. Two recent publications have provided intriguing insights into how expression levels of a key subunit, in this particular case SMARCB1, which is unique to BAF and PBAF, can shift the balance between canonical and non-canonical SWI/SNF complexes and hence alter transcriptional signatures. SMARCB1 loss results in a widespread impairment of typical enhancer activity but not super-enhancer activity [23]. Conversely, SMARCB1 negatively regulates super-enhancers in any cell type, suggesting that high levels of SMARCB1 prevent the formation of ncBAF [24].

Role of SWI/SNF complexes in cancer development

SWI/SNF complexes play a fundamental role in maintaining and regulating the access of transcription factors, and,

therefore, they also exert considerable tumour-suppressive activities. Consequently, SWI/SNF perturbations can trigger reprogramming of cellular processes and drive oncogenic programmes. Loss-of-function mutations in genes encoding SWI/SNF subunits are found in >20% of human cancers, with point mutations occurring about twice as often as deletions [25]. The recently published cryo EM structures of the human BAF complex revealed that a large fraction of oncogenic mutations maps to intra-complex subunit–subunit interfaces, exposed surfaces that may interact with regulatory proteins or interaction sites with the nucleosome, thereby altering the chromatin remodelling activity of the complex [18]. SWI/SNF genes are also amplified in many cancers. The prevalence of amplifications strongly depends on the type of cancer and occurs most commonly in lung squamous cell carcinoma, ovarian cancer and sarcoma [25]. Especially BRD9 and ACTL6A show a high amplification frequency across multiple cancers, highlighting their oncogenic potential [25].

SWI/SNF chromatin remodellers were first linked to cancer development more than two decades ago when it was discovered that biallelic loss-of-function mutations of the SMARCB1 gene drive tumorigenesis in malignant rhabdoid tumours (MRTs) [26]. The main reason for developing MRT is that SMARCB1 loss prevents interaction of BAF with typical enhancers, whereas binding to super-enhancers is barely affected [23]. These observations led to the assumption that the remaining SWI/SNF subunits induce gene expression mainly at super-enhancers but not at typical enhancer sites, thereby maintaining aberrant cell identity and enabling MRT survival [23]. Newer studies have now provided a more detailed picture, showing that gene expression in MRT is driven by the BRD9-containing ncBAF complex, which does not require a SMARCB1 subunit [18, 21, 27, 28]. In addition, SMARCB1 rescue experiments in MRTs restored enhancer-based activation and demonstrated that wild-type BAF is able to antagonise polycomb repressive complex mediated gene repression [20]. These findings are consistent with an earlier study showing that SMARCB1 loss leads to an increased expression of the polycomb subunit EZH2 (enhancer of zeste 2) and to elevated levels of H3K27me3 gene silencing marks during MRT transformation [29].

Loss of SMARCB1 activity is also a major driver for developing aggressive synovial sarcoma (SS). In the development of this tumour type, a chromosomal translocation involving BAF subunit SS18 leads to the formation of an SS18-SSX fusion protein [30] that displaces SMARCB1 in the BAF complex. As a result, SS18-SSX-containing BAF is retargeted from specific enhancer regions, inducing the activation of a unique transcriptional signature in SS [31]. SMARCB1 loss has been linked to the activation of several oncogenic signalling pathways (e.g., hedgehog or Wnt) and the development of malignant tumours such as renal medullary carcinoma [32, 33]. Mutations in the N-terminal winged-helix DNA-binding domain of SMARCB1 have been associated with schwannomatosis, a syndrome predisposing to mostly benign tumours of the central nervous system [34, 35].

SMARCA4 is the most frequently mutated Snf2-like ATPase in humans [36], and it has been found to be inactivated or disrupted in many cancers, including breast cancer [37], lung cancer [38] and colorectal cancer [39]. Although the ARID1 subunit of BAF and the PBRM1 subunit of PBAF exhibit the highest mutation prevalence, SMARCA4-mutant cancers are typically more aggressive and are associated with a poorer prognosis [36]. The ATPase subdomain of SMARCA4 is a mutational hotspot. Mutations of this subdomain not only result in impaired ATPase activity and nucleosome remodelling [36] but also impair BAF competition with polycomb repressive complexes [40]. In contrast, the SMARCA4 parologue SMARCA2 is less frequently mutated in cancer. It is, however, frequently epigenetically silenced, which has been demonstrated, for example, by restoring SMARCA2 function in SMARCA2-deficient cancers using histone deacetylase (HDAC) inhibitors [41].

The BAF subunit ARID1A binds DNA non-specifically via its AT-rich interactive DNA-binding domain and has the highest mutation frequency among BAF subunits in cancer [42]. ARID1A acts as a tumour suppressor, and loss of function has been associated with tumour development in ovarian carcinoma [43], pancreatic cancer [44] and cervical cancer [45]. Interestingly, the cancer mutations are spread across almost all regions of the ARID1A gene and do not cluster in the DNA-binding domain as seen in transcription factors such as p53 [42, 46]. One reason for this is that ARID1A is involved in many subunit–subunit contacts stabilising the base module of the complex that are susceptible to inactivating mutations. Loss of ARID1A tumour suppressive functions triggers cancer development through perturbations of DNA-damage response and cell-cycle pathways [42]. Furthermore, several studies have also demonstrated that an ARID1A loss is associated with activation of phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) and a concurrent loss of PTEN expression, which both activate the PI3K/AKT/mTOR cell-cycle pathway [47–50].

ARID1B-related cancers are less common. Mutations of this gene, however, have been associated with developmental disabilities such as the Coffin-Siris syndrome [51, 52]. There is compelling evidence, though, that its parologue in PBAF, ARID2, acts as a tumour suppressor. ARID2 is frequently mutated in non-small cell lung carcinoma [53] and hepatocellular carcinoma [54]. Mechanistic studies on the role of ARID2 in hepatocellular carcinoma and lung cancer have shown that an ARID2 loss drives

Table 2 Synthetic lethal interactions and chemical tools for the targeted therapy of SWI/SNF-mutant cancers.

| SWI/SNF-   | Synthetic lethal interaction/epigenetic antagonism | Inhibitors                                                                 | Ref.         |
|------------|--------------------------------------------------|--------------------------------------------------------------------------|--------------|
| deficiency |                                                                  |                                                                          |              |
| SMARCA4    | SMARCA2                                                         | PFI-3#, ACBI1#, ATPase-14#                                             | [6, 65, 66]  |
|            | ACTB                                                            | n/a                                                                      | [5]          |
|            | ARID2                                                           | n/a                                                                      | [5]          |
|            | CDK4/6                                                          | Palbociclib§, Abemaciclib§, Ribociclib§                                 | [10, 111]    |
|            | Aurora A                                                       | Tozasertib*(VX-680)                                                      | [9]          |
|            | OxPhos                                                         | IACS-010759*                                                            | [134]        |
|            | BET                                                            | (+)-JQ1#, OTX-015*                                                      | [135]        |
|            | EZH2                                                           | Tazemetostat§, GSK126*, CPI-169#                                       | [136–138]    |
| ARID1A     | ARID1B                                                         | n/a                                                                      | [7, 64]      |
|            | EZH2                                                           | GSK126*                                                                  | [136, 139]   |
|            | PARP                                                           | Talazoparib§, Olaparib§, Rucaparib§, Veliparib§                         | [119, 120]   |
|            | PI3K/AKT                                                       | MK-2206*, Perifosine*, Buparlisib*                                      | [49]         |
|            | ATR                                                            | VE-821#, VX-970(M6620)*                                                 | [98]         |
|            | Abl, Src, c-KIT                                                | Dasatinib§                                                               | [110]        |
|            | GCLC                                                           | Buthionine sulfoximine*                                                 | [128]        |
|            | Aurora A                                                       | TCS-7010#                                                                | [108]        |
|            | HDAC6                                                          | Ricolinostat(ACY-1215) *                                                | [114]        |
| SMARCC1    | SMARCC2                                                        | n/a                                                                      | [5]          |
| SMARCA2    | SMARCA4                                                        | dBRD9#                                                                   | [140]        |
| PBRM1      | EZH2                                                           | GSK126*                                                                  | [136, 141]   |
| ARID2      | PARP                                                           | Veliparib§                                                               | [56]         |
| SMARCB1    | EZH2                                                           | Tazemetostat§                                                            | [29]         |
|            | HDAC                                                           | Panobinostat§                                                            | [112]        |
|            | BRD9                                                           | BI-7273#/9564#, dBRD9#, VZ185#                                         | [21, 27]     |
|            | MDM2/4                                                         | Idasanutlin*                                                             | [121]        |
|            | UBE2C                                                          | Ixazomib§, bortezomib§                                                  | [123]        |
| SS18-SSX   | BRD9                                                           | dBRD9#                                                                   | [27, 28]     |
|            | KDM2B                                                          | n/a                                                                      | [124]        |
|            | ATR                                                            | VX-970*, AZD6738*                                                       | [142]        |

*Chemical probe.*Clinical probe.§FDA-approved drug.

identified a number of promising genes for targeted therapy
of SWI/SNF-mutant cancers (Table 2) [5–7, 64–67]. One
therapeutic strategy targets the epigenetic antagonism
between SWI/SNF complexes and the polycomb repressive
complexes, which has been excellently reviewed in detail
elsewhere [68]. Another approach exploits synthetic lethal
interactions, where a concurrent loss of two genes leads to
cell death, while a single perturbation of one of these genes
is viable.

A number of intra- and inter-complex SWI/SNF vulnerabilities have been identified that are associated with a
synthetic lethal phenotype in SWI/SNF-deficient cancers
(Table 2). In some cases, synthetic lethality is caused by the
concomitant knockout of two mutually exclusive paralogues [69]: Tumours with a SMARCA4 deficiency are
sensitive to SMARCA2 depletion [65], and ARID1B is
required for survival of ARID1A-mutant cancers [7]. In
addition, BRD9 is a specific vulnerability in SMARCB1-
deficient malignant rhabdoid tumours and in synovial sarcoma (SS18-SSX) cells [21, 27, 28]. This particular synthetic lethality can be rationalised based on the composition
and structure of the three types of SWI/SNF complexes
outlined above (Fig. 1). The assembly and structural
integrity of canonical BAF and PBAF complexes depend
on SMARCB1, whereas assembly and function of ncBAF
do not require SMARCB1. Correct assembly and integrity
of ncBAF, which drives oncogenic signalling in MRT,
however, depends on the scaffolding function of BRD9
[16]. So, the additional knockdown of BRD9 in
SMARCB1-deficient cells also abrogates the function of
the remaining non-canonical SWI/SNF chromatin remodelling complexes and is therefore lethal. Additional vulnerabilities have been recently identified in a large genetic
screen, revealing intra-complex synthetic lethal interactions
between SMARCC1/SMARCC2, SMARCA4/ACTB and
SMARCA4/ARID2 [5].

Vulnerabilities of SWI/SNF-mutant cancers have also
been found in the interactome of SWI/SNF chromatin
remodellers, including protein kinases involved in cell-
cycle control or DNA-damage repair (Table 2). Although
not every protein kinase has been targeted to date, there is
a large diversity of inhibitors available for every family of
the kinome that could provide excellent starting points for
targeted therapy. Targeting specific SWI/SNF vulnerabilities with small molecules is therefore a promising
strategy for this highly complex system. However, this
approach requires the identification and validation of
druggable targets and the development of high-quality
chemical probes to elucidate the biological role of a target
of interest [70]. Such probes must fulfil a number of specific criteria, including chemical stability, selectivity and
potency, and their mode of action must be sufficiently well
characterised [71, 72].

Direct targeting of SWI/SNF components for
inducing synthetic lethality

The development of effective chemical probes for the direct
targeting of components of SWI/SNF complexes to exploit
synthetic lethal interactions relies on the presence of unique
and well-defined druggable pockets. Targeted SWI/SNF
drug discovery has therefore mainly focused on the ATPase
and bromodomains. SMARCA2/4 contains a bromodomain
and an ATPase domain, BRD7 and BRD9 contain a bromodomain, and PBRM1 has six tandem-acting PB1 bromodomains. Targeting these domains may be desirable in
the context of a synthetic lethal approach to inactivate
specific SWI/SNF subunits but is of course also a promising
strategy to neutralise an over-expression and/or aberrant
oncogenic function of SWI/SNF subunits. A prime example
is BRD9, which is often amplified in cancers [25] but is also
a specific vulnerability in SMARCB1-deficient cancers
[21, 27, 28]. In the following, we will first describe the
structural features and druggability of bromodomain and
ATPase modules and then discuss the current state of chemical probe development and its implications.

Structure and druggability of SWI/SNF ATPase and
bromodomains

The ATPase domain of human SWI/SNF complexes is an
Snf2-like ATPase that uses the energy from ATP hydrolysis
to remodel the chromatin structure. Until very recently,
most structural information was based on Snf2 homologues
from non-human species, though [73]. The cryo-EM
structures of human BAF have now provided exciting
mechanistic insights into BAF assembly and potential
mechanisms for nucleosome/DNA unwinding [17, 18]. Due
to the high flexibility of the SMARCA4 subunit within the
BAF complex, the resolution of this particular region of the
complex was relatively poor, which prevented a detailed
view of the structure of the ATPase domain in the context of
the overall complex [17]. It was, however, possible to
determine the crystal structure of the N-RecA lobe of the
human SMARCA2 ATPase domain, which revealed a
druggable allosteric pocket in the proximity of the ATP-
binding site (Fig. 2) [74].

Bromodomains are highly conserved protein-protein
interaction modules that recognise acetylated lysines on
histone tails. The human genome encodes 61 BRDs that are
part of 46 larger multi-domain proteins. They can be subdivided into eight families, which are mainly associated
with DNA accessibility and the control of gene regulation
[75]. Bromodomain-mediated transcriptional activation is
often enhanced by the concurrent interaction with additional
chromatin-modifying enzymes to ensure accurate chromatin
engagement and gene expression. All BRDs comprise ~120

Fig. 3 Druggable binding pockets of SWI/SNF chromatin remodelling complex bromodomains with selected chemical probes.

A, B Comparison of BRD4 in complex with (+)-JQ1 (PDB entry 3MXF) [82] and BRD9 in complex with BI-7273 (PDB entry 5EU1) [80] reveals a different shape of the binding pocket. The residue numbering in the BRD9 structure is based on the canonical isoform in UniProt entry Q9H8M2, which differs from the numbering in the original PDB entry that is based on an isoform lacking the first 116 residues. C–E Crystal structures of BRD subfamily VIII members with bound inhibitors (PDB entries 5DKD, 6ZS4, 6ZNV) reveal a similar overall inhibitor binding mode [90, 93]. F Crystal structure of the ternary complex of SMARCA4 and the E3 ubiquitin ligase VHL bound to PROTAC ACBI1, showing that the complex is stabilised by additional protein–protein interactions (PDB entry 6HR2) [104].

amino acids and fold into a characteristic structure that contains a bundle of four α-helices (αA, αB, αC, αZ) with a hydrophobic cavity formed by two loops (ZA, BC). In canonical BRDs, acetylated lysines (Kac) are anchored via a highly conserved asparagine and a network of water molecules [75]. The biological substrate specificity is mainly determined by the electrostatic potential of the BRD surface. The bromodomains BRD7/9, SMARCA2/4 and PB1(2-5) read histone acetylation marks such as H3K14ac [63], H3K27ac [22] or H3K9ac [76], thereby recruiting SWI/SNF chromatin remodelling complexes to chromatin. The members of BRD subfamily IV, BRD7 and BRD9, share a high sequence identity and form a highly conserved overall structure, especially in the Kac binding site, but some notable minor differences are found in the vicinity of the binding pocket [77]. As a consequence, the biological roles of BRD7 and BRD9 are remarkably different, which may explain their mutual exclusivity in either PBAF or ncBAF, respectively [78, 79]. This phenomenon of different substrate specificities of BRD family members is exemplified by comparing BRD9 with BRD4, a member of the bromodomain and extra-terminal domain (BET) family. BRD9 has a different shape of the binding pocket because the gatekeeper residue Tyr222 on helix αC blocks the entrance to a subpocket, which is accessible in BRD4 (WPF shelf region) due to a smaller hydrophobic residue at this position (Ile146) (Fig. 3a, b) [80]. In members of BRD subfamily VIII, SMARCA2/4 and PB1(1-6), the binding pocket is smaller and lacks well-defined hydrophobic grooves. They also have a shorter ZA loop than BRD4, somewhat reducing potential ligand interactions with this region (Fig. 3A, C–E) [81]. SMARCA2/4 and PB1(2-5) are canonical BRDs, whereas the first and sixth BRD of PBRM1 have either an occluded

binding site or lack the conserved asparagine. The PB1(2-4) BRDs additionally have a specific tyrosine in the ZA loop that acts as a lid at the entrance of the binding pocket [81].

### Chemical probes for targeting individual SWI/SNF domains

Early bromodomain drug discovery focused on the development of chemical probes targeting the BET bromodomains (bromodomain subfamily II) [82–84]. More recently, an increasing number of new bromodomain inhibitors targeting other bromodomain families have been reported, especially chemical probes targeting the bromodomains of SWI/SNF complexes [85]. For example, many BRD7/9 inhibitors are now available to study the biological roles of these essential epigenetic modulators (Fig. 4A). However, due to the high sequence conservation of the binding pockets of BRD7/9, most of the available chemical probes exhibit dual BRD7/9 activity [77, 80, 86–88]. The first published selective dual BRD7/9 inhibitor, LP99, was developed from a quinolone-fused lactam in an SAR study [86]. Shortly after, a selective BRD9 inhibitor, I-BRD9, was reported with a 200-fold selectivity over BRD7 [89]. The selectivity was attributed to small differences in the GFF shelf region (GFF vs. AFF in BRD7) and the ZA channel of the two bromodomains.

The first published SMARCA2/4 and PB1(5) bromodomain inhibitor, PFI-3, was developed in an SAR study using an unusual salicylic acid fragment as an acetyl-lysine mimetic moiety (Figs. 3c and 4b) [90]. Structural studies showed that PFI-3 penetrated deeper into the binding pocket compared with the canonical binding mode of earlier BRD inhibitors by displacing conserved structural water molecules, resulting in an excellent selectivity profile within all BRD families. Due to the lack of chemical stability of PFI-3 in cellular systems over longer time periods, several other SMARCA2/4 BRD inhibitors were developed [91, 92]. However, only one of them, SGC-SMARCA-BRDVIII (compound 22) [93], which has been recently developed based on a patent [94], met chemical probe criteria in terms of potency (Figs. 3d and 4b). The PB1 BRDs have not been considered as drug targets so far. Due to the high sequence conservation in the acetyl-lysine binding site of BRD subfamily VIII, most of the PB1-BRD inhibitors show selectivity for the fifth PB1 bromodomain and for SMARCA2/4 BRDs; the recently published compound 32, however, also showed activity on the second and third bromodomain of PB1 (Figs. 3e and 4b) [90, 91, 93, 94]. Only one selective ligand with low affinity for the fifth PB1-BRD has been developed so far [95].

Besides targeting the BRDs, an allosteric inhibitor of the ATPase domain of SMARCA2/4 was developed (Fig. 4C) [74]. High throughput screening methods and hit validation techniques identified a dipyridyl urea derivative as initial lead compound, which was subsequently developed into several inhibitors (Fig. 4C). This inhibitor class binds to an allosteric pocket in the vicinity of the ATP binding site, with the two NH groups of the urea moiety interacting with the carboxylate group of the catalytic residue Glu 852 [74].

Other than that, no additional direct-acting inhibitors for other subunits of SWI/SNF complexes have been developed so far because of a lack of known binding pockets. However, a non-toxic 12-membered macrolactam, BRDK98645985, was recently identified in a high-throughput screen for inhibitors of BAF-mediated transcription in cells, which seems to preferentially bind to ARID1A-containing BAF complexes and relieves transcriptional repression of HIV-I [96]. This compound was further found to synergize with ATR (ataxia-telangiectasia mutated (ATM) and Rad3-related protein kinase) inhibitor VE-821 and induce a synthetic lethal interaction in cancer cells [97], which had been described before in RNA interference screens [98]. The binding site or the specific target protein of BRDK98645985 remains elusive.

#### Phenotypic response of direct SWI/SNF domain inhibition and the emergence of PROTACs as a game-changing technology

The excitement that specific bromodomain inhibitors are available to potentially exploit synthetic lethal interactions in SWI/SNF mutant cancers was initially somewhat dampened by their failure to induce synthetic lethality. The first example to illustrate this issue are the SMARCB1-deficient cancers that are sensitive to depletion of the ncBAF subunit BRD9. Initial biological assays with inhibitor I-BRD9 indicated that BRD9 bromodomain function is not required for cell proliferation but regulates gene expression associated with oncogenic and immunoresponsive pathways [89]. The chemical probes I-BRD9 and BI-7273 both showed some weak cytotoxic effect in SMARCB1-deficient MRT and synovial sarcoma but, importantly, only at a relatively high compound concentration of about 10 μM or higher, hinting at off-target activity of these inhibitors at high concentration [99]. Overall, targeting BRD7/9 with conventional BRD inhibitors failed to fully recapitulate the phenotypic response of genetic BRD9 knockout studies in SMARCB1-deficient cancers. Experiments with diverse BRD9 truncation mutants revealed that its bromodomain is dispensable for the integrity of ncBAF, and that the yet undruggable, poorly characterized DUF3512 domain has an essential scaffolding function and constitutes the actual vulnerability in MRT [21]. The initial disappointment, however, soon disappeared with the emergence of the first bromodomain-specific degraders [100]. Proteolysis-targeting chimeras, PROTACs, may represent a game-

A

I-BRD9  
BRD9: $IC_{50} = 50$ nM  

LP99  
BRD9: $K_{D} = 99$ nM  
BRD7: $K_{D} = 0.91$ μM  

28  
BRD9: $K_{D} = 68$ nM  
BRD7: $K_{D} = 0.37$ μM  

GNE-375  
BRD9: $K_{D} = 5.0$ nM  

TP-472  
BRD9: $K_{D} = 33$ nM  
BRD7: $K_{D} = 0.34$ μM  

BI-9564 (R₁=OMe, R₂=H)  
BRD9: $K_{D} = 14$ nM  
BRD7: $K_{D} = 0.24$ μM  

BI-7273 (R₂=OMe, R₁=H)  
BRD9: $IC_{50} = 19$ nM  
BRD7: $IC_{50} = 0.12$ μM  

dBRD9  
BRD9: $DC_{50} = 104$ nM  

VZ185  
BRD9: $DC_{50} = 1.8$ nM  
BRD7: $DC_{50} = 4.5$ nM  

B

PFI-3  
SMARCA2: $K_{D} = 81$ nM  
SMARCA4: $K_{D} = 89$ nM  
PB1(5): $K_{D} = 48$ nM  

26  
SMARCA2: $K_{D} = 0.26$ μM  
SMARCA4: $K_{D} = 0.42$ μM  
PB1(5): $K_{D} = 0.12$ μM  

12  
PB1(5): $K_{D} = 3.3$ μM  

DCSM06-05  
SMARCA2: $IC_{50} = 9.0$ μM  

GNE/CPI-SMARCA  
(lead-structure)  

SGC-SMARCA-BRDVIII  
(R₁=Boc-piperazine)  
SMARCA2: $K_{D} = 35$ nM  
SMARCA4: $K_{D} = 36$ nM  
PB1(5): $K_{D} = 13$ nM  

32 (R₁ = 4-OH-piperidine)  
SMARCA2: $K_{D} = 69$ nM  
SMARCA4: $K_{D} = 61$ nM  
PB1(2): $K_{D} = 0.46$ μM  
PB1(3): $K_{D} = 0.45$ μM  
PB1(4): $K_{D} = 4.16$ μM  
PB1(5): $K_{D} = 64$ nM  

ACBI1  
SMARCA2: $DC_{50} = 6$ nM  
SMARCA4: $DC_{50} = 11$ nM  
PB1(5): $DC_{50} = 32$ nM  

ATPase-14  
SMARCA2: $IC_{50} < 0.005$ μM  
SMARCA4: $IC_{50} < 0.005$ μM  

C

Fig. 4 Selection of small molecules targeting the SWI/SNF bromodomains and the ATPase domain. A Inhibitors and PROTACs targeting bromodomains BRD7/9 [80, 86–89, 102, 103]. B Inhibitors and PROTACs targeting the bromodomains of SMARCA2/4 and PBRM1 [90–95, 104]. C Allosteric inhibitor of the ATPase domain of SMARCA2/4 [74].

changing technology, as this new class of small-molecule modulators initiate the degradation of a protein of interest rather than merely inhibiting the function of a specific domain [101]. By linking a conventional inhibitor to an E3 ligase recruiting moiety (e.g. thalidomide or VHL ligand 1), PROTACs induce ubiquitin transfer onto target proteins, thereby marking them for proteasomal degradation. This technology was applied to target BRD9, and the developed PROTACs dBRD9 [102] and VZ185 [103] (Fig. 4A) were much more effective in treating SMARCB1-mutant cancers than their parent inhibitor BI-7273. dBRD9 is a selective cereblon-based degrader of BRD9, whereas VZ185 targets both BRD7 and BRD9 for proteasomal degradation mediated by the VHL E3-ubiquitin ligase. Both dBRD9 and VZ185 showed increased cytotoxicity in acute myeloid leukaemia and MRT cells that were sensitive to BRD9 depletion. dBRD9 also altered transcriptional programmes essential for SMARCB1 loss-of-function driven tumours, thereby blocking proliferation of synovial sarcoma and MRT cells [27, 28].

The recent progress in the field of targeted degradation also led to the development of ACBI1, a VHL-dependent degrader of SMARCA2/4 and PBRM1 based on the bromodomain inhibitor GNE/CPI-SMARCA (Fig. 4B) [104]. A sensitivity to SMARCA2 loss by RNA interference knockdown was observed in SMARCA4-deficient lung cancers, which triggered a phenotypic lethal response [105]. Inhibition of the SMARCA2 BRD with the chemical probe PFI-3 [90], however, was unable to phenocopy this synthetic lethal interaction [105], but it had pronounced effects on embryonic and trophoblastic stem cell differentiation [106]. The lack of efficacy of SMARCA2/4 BRD inhibitors in SMARCA4-mutated cancers suggested the ATPase domain rather than the BRD as the therapeutically relevant target in oncology, which was confirmed by genetic complementation studies [105]. Consistent with these complementation studies, expression of SMARCA2-associated genes was downregulated and cell growth of SMARCA4-mutant lung cancers was impaired upon inhibition of the ATPase domain with the SMARCA2/4 ATPase inhibitor 14 [74]. The PROTAC ACBI1 has now led to a renaissance of bromodomains as drug targets in SMARCA4-deficient cancers. This compound induced anti-proliferative effects and apoptosis in SMARCA4-mutant cells by triggering proteasomal degradation of the full-length SMARCA2 protein, thereby also knocking out the SMARCA2 ATPase domain [104]. This study further provided insights into the structural basis of protein degradation by elucidating the structural details of the ternary complex between PROTAC-bound SMARCA2 and VHL (Fig. 3F). More recently, an additional PROTAC based on this scaffold targeting SMARCA2/4 was reported [107]. The potential of altering selectivity profiles is another exciting aspect of PROTACs

development because efficient PROTAC-mediated degradation depends on the exact geometry and spatial arrangement of the ternary complex with the E3 ligase, which is influenced by regions outside the canonical inhibitor binding site. It may therefore be possible to convert dual or pan-selective inhibitors into selective degraders that exclusively target one subunit type or a particular paralogue, e.g. SMARCA2 or 4, for efficient degradation. At the moment, however, such strategies often rely on serendipity rather than on actual structure-based design.

The above examples highlight the enormous potential of PROTACs for targeting undruggable components of SWI/SNF complexes. Future strategies may aim to develop PROTACs that target non-essential but druggable components of SWI/SNF complexes to eliminate ‘undruggable activities’ in a specific disease context, either to induce synthetic lethality or to counteract the effect of gene amplification.

### Small-molecule modulators for targeting the SWI/SNF interactome

Over the past years, great progress has been made in the identification of synthetic lethal targets that interact with or regulate SWI/SNF chromatin remodelling complexes (Table 2 and Fig. 5). Although not all of these targets have been drugged yet, there is a large collection of inhibitors targeting these proteins described in the literature that may be used to recapitulate the effects of genetic knockdown studies. One exploitable class of proteins are protein kinases, which are involved in many signalling pathways, including cell-cycle control and DNA-damage repair, and for which a myriad of different inhibitors are available. A second targetable protein class are the HDACs, which alter the histone code that is read by chromatin-interacting proteins such as the SWI/SNF bromodomains. A third target group are the poly ADP ribose polymerase (PARP) enzymes, which play a role in DNA-damage repair by modulating chromatin structure through ADP ribosylation. Last but not least, the ubiquitin-ligase/proteasome system as well as the p53/MYC-signalling pathway can be targeted with inhibitors of the p53-MDM2 interaction or the proteasome.

#### Protein kinases

Protein kinases are exploitable vulnerabilities in SMARCA4-and ARID1A-mutant cancers. Non-small cell lung cancer (NSCLC) cells carry a SMARCA4 loss-of-function mutation, which makes them highly sensitive to inhibition or deletion of Aurora A, a kinase essential for cell division during mitosis and for the regulation of chromosome segregation. Aurora A inhibition in NSCLC cells with tozasertib, for example,

Tazemetostat

PFI-3, ACBI1,
SGC-SMARCA-BRDVIII,
ATPase-14

n/a

Palbociclib,
Abemaciclib,
Ribociclib

Tozasertib

IACS-010759

(+)-JQ1, OTX-015

Talazoparib,
Olaparib,
Rucaparib

Perifosine,
Buparlisib,
MK-2206

VE-821,
VX-970

Dasatinib

Buthionine
sulfoximine

PCR2
EZH2

SMARCA2

ACTB

CDK4/6

Aurora A

OxPhos

BET

PARP

PI3K/AKT

ATR

Abl, Src,
c-KIT

GCLC

Aurora A

SWI/SNF
mutated
cancer

SS18

SMARCA4

ARID1A

SMARCB1

ARID1B

HDAC6

n/a

Ricolinostat

TCS-7010

KDM2B

ATR

BRD9

dBRD9

VX-970,
AZD6738

UBE2C

Ixazomib,
Bortezomib

MDM2/4

Idasanutlin

BI-7273/9562,
dBRD9, VZ185

HDAC

Panobinostat

Fig. 5 Overview of synthetic lethal interactions in SWI/SNF-
mutant cancers with chemical tools for targeted cancer therapy.
Known synthetic lethal interactions in SWI/SNF-mutant cancers are shown in matching colours. In addition, the SWI/SNF-polycomb
(PCR2/EZH2) epigenetic antagonism pathway is highlighted.

resulted in a synthetic lethal response by reprogramming
centrosome pathways during mitotic spindle formation [9]. A
recent publication also suggested Aurora A as a potential
therapeutic target in ARID1A-deficient colorectal cancer cells
[108]. Wild-type ARID1A was shown to downregulate the
expression of Aurora A, while ARID1A loss led to increased
expression levels of Aurora A and sustained activation levels
of cell division cycle 25 C (CDC25C). Inhibition of Aurora A
with a pan-selective kinase inhibitor, TCS-7010, in ARID1A-
mutant colorectal cancer cells induced a G2/M phase arrest,
followed by apoptosis. A very recent study reported the
development of an Aurora A PROTAC, JB170, showing that
depletion of Aurora A results in strong S-phase arrest, which
was not seen for the parent inhibitor TCS-7010, suggesting a
non-catalytic scaffolding function of Aurora A during DNA
replication [109]. JB170 induced a strong apoptotic phenotype
in several cancer cell lines, which may open a new therapeutic
window for the treatment of SWI/SNF-deficient cancers that
depend on Aurora A [109].

Available chemical tools for each target involved in a synthetic lethal
interaction in SWI/SNF-mutant cancers are listed. The red arrows
highlight synthetic lethal interactions with other SWI/SNF chromatin
remodelling complex subunits, whereas the black arrows mark
reported vulnerabilities with other genes.

The broad-spectrum tyrosine kinase inhibitor dasatinib
was found to arrest the cell cycle during G1 and S phases
through increased p21 and retinoblastoma protein expression,
which resulted in a robust tumour reduction in
ARID1A-mutated ovarian cancer [110]. This observation
was confirmed in a second study, showing that small-
molecule inhibition of the upstream regulators of p21, PI3K
and AKT, also induced cell-cycle arrest and a lethal phenotype in ARID1A-deficient cell lines [49].
A quite intriguing synthetic lethal interaction was found
for SMARCA4 and CDK4/6, suggesting that it may also be
possible to exploit critically low levels of an oncogene for
therapy. NSCLS and SCCOHT (small cell carcinoma of the
ovary hypercalcemic type) cells are more sensitive to
CDK4/6 inhibition with ribociclib or palbociclib than
SMARCA4-proficient controls. This effect can be attributed
—at least in part—to lower cyclin D expression levels in
SMARCA4-deficient cancers, making them more susceptible to CDK4/6 inhibitors because their activity is already

compromised [10, 111]. Interestingly, the low cyclin D levels in NSCLS seem to result from a combination of restricted chromatin accessibility of the CCND1 locus and that of its transcriptional activator JUN [10, 111].

### Histone deacetylases

HDACs are frequently overexpressed in SMARCB1-mutant tumours and in ARID1A-deficient cancers, making them sensitive to HDAC inhibitors. The pan-HDAC inhibitor panobinostat was able to induce MRT growth arrest, promote cellular senescence and inhibit self-renewal at low doses [112]. A second study showed that HDAC inhibition is more effective in combination with radiation when treating atypical teratoid/rhabdoid tumours [113]. ARID1A-deficient tumours depend on the activity of HDAC6, which erases the pro-apoptotic Lys120 acetylation mark in the DNA-binding domain of p53 [114]. Mechanistically, acetylation of Lys120 alters p53 DNA-binding specificity, favouring binding to response elements of pro-apoptotic target genes [115]. In addition, acetylation of several lysines in the C-terminal regulatory domain of p53 is crucial for p53 activation in general, by blocking ubiquitination sites and reducing non-specific DNA binding [46]. Accordingly, the selective HDAC6 inhibitor ricolinostat promoted an apoptotic response through p53 hyperactivation and thus improved the survival of mice bearing ARID1A-mutated ovarian tumours [114]. Another study on ARID1A-mutant cancer showed that HDAC2 and PRC2/EZH2 co-repressed the expression of apoptosis-promoting protein PIK3IP1 [116]. HDAC2 inhibition with SAHA re-localised HDAC2 to the cytoplasm and induced expression of PIK3IP1, which correlated with an improved prognosis for mice bearing ARID1A-mutated tumours [116].

### Poly ADP ribose polymerases

PARP enzymes involved in synthetic lethal interactions with SWI/SNF components are well-characterised drug targets and have been targeted with small molecules such as the highly potent PARP1 inhibitors talazoparib or veliparib, which block the NAD⁺ binding site [117]. In response to DNA damage, BRIT1 (BRCT-repeat inhibitor of hTERT expression) regulates the recruitment of SWI/SNF to DNA lesions [118]. This localisation is enhanced through phosphorylation of SMARCC2 by ATM and ATR kinases [118], and is supported by ARID1A, which interacts with double-stranded DNA breaks (DSBs) and regulates the DNA-damage checkpoint [119]. However, loss of ARID1A prevented ATR activation and DSB repair, generating a sensitivity to PARP inhibitors [119] as well as to ATR kinase inhibitors [98]. Talazoparib suppressed tumour growth of ARID1A-mutant cancers in a xenograft model by reducing expression levels of checkpoint kinase 1 and increasing levels of apoptosis marker caspase 3 [119]. In addition, ARID1A-mutant ovarian cancers are sensitive to a combined therapy of exogenous irradiation and PARP inhibition because of a reduced efficiency of the non-homologous end-joining (NHEJ) machinery to repair DNA lesions [120]. Lung cancer patients with ARID2 deficiency may also benefit from PARP inhibition, as PARP inhibitor veliparib has been shown to reduce cell survival of certain lung cancer cell lines [56].

### Ubiquitin-ligase/proteasome system and beyond

In MRT and renal medullary carcinoma cancer cells, an interesting link between SMARCB1 loss and the ubiquitin-ligase/proteasome system has been observed. MRT cells are sensitive to inhibition of the E3 ubiquitin ligase MDM2 (murine double minute 2) and its homologue MDM4/X. The MDM2 inhibitor idasanutlin, which blocks the MDM2-p53 interaction and hence proteasomal degradation of the latter, reduced tumour growth in an MRT xenograft model via upregulation of p53 [121]. These findings are consistent with the observation that MRT cells are highly sensitive to perturbations of the proteostasis programme and the p53/MYC-axis as shown by treatment with proteasome inhibitor ixazomib or the autophagy/lysosomal inhibitor chloroquine [122]. Moreover, it has been shown that SMARCB1-mutated renal medullary carcinoma are also dependent on the ubiquitin/proteasome system [123]. Proteasome inhibitors MLN2238 and bortezomib led to an accumulation of cyclin B1 as a result of an inactive E2 ubiquitin-conjugating enzyme, which triggered G2/M cell-cycle arrest [123].

Recent years have seen a surge of publications on targeting synthetic lethal interactions within the SWI/SNF interactome, not just involving the protein classes described in detail above. In addition, there are, for example, synthetic lethal interactions with the lysine demethylase of polycomb repressive complex 2 [124], downstream targets of c-MYC [125], components of the respiratory chain [126, 127] as well as with enzymes regulating the critical balance between glutathione and toxic reactive oxygen species [128] (see Table 2 and Fig. 5).

### Concluding remarks and outlook

The enormous impact of SWI/SNF chromatin remodelling complexes on chromatin biology, and thus on the regulation of almost all cellular pathways, has been gradually revealed over the last years. Ground-breaking discoveries ranged from how remodellers assemble their structures and organise chromatin accessibility, repair DNA defects or control gene expression, to distinct mechanisms that control cellular processes such as
differentiation and proliferation. Perturbations of these fragile liaisons drive oncogenesis and promote tumour development. Mutations of SWI/SNF complexes occur in 20–25% of all human cancers, making them prime targets for therapeutic intervention [13]. Because every tumour differs in its genetic or epigenetic profile, there is a great need for new medicines and personalised therapies to successfully treat cancer patients. Choosing the right drug is thus highly context-dependent, and therapeutic concepts must be assessed on an individual basis.

At first glance, SWI/SNF inhibition seems counterintuitive, given its tumour suppressor functions. Indeed, inactivation of certain SWI/SNF subunits may actually promote cancer development. A very recent paper provided intriguing insights into how SWI/SNF cancers may be successfully targeted [129]. This study on the nematode *Caenorhabditis elegans* suggested that it is all about the right balance—as with so many things in life. Cancer cells depend on an incomplete inactivation of SWI/SNF genes to maintain their full oncogenic potential. A partial loss-of-function eliminates the intrinsic tumour suppressor properties of SWI/SNF complexes, leading to aberrant proliferation and differentiation, whereas the complete loss of SWI/SNF activity results in cell division arrest during cell development in the *C. elegans* model [129]. The ATPase subunit is a good example to illustrate this phenomenon. High functional levels of the ATPase subunit are required to maintain its tumour suppressor activity. Mutation of this gene can result in lower functional levels, but, importantly, the residual SWI/SNF components are able to sustain cell proliferation [129]. Similarly, in malignant rhabdoid tumours, SMARCB1 loss reduces the ability to bind to typical enhancers, but not to super-enhancer sites, indicating that the residual SWI/SNF subunits are sufficient for tumour progression and survival through formation of ncBAF complexes [21, 23].

In recent years, immense progress has been made in the identification and validation of targets to treat SWI/SNF-mutant cancers (Fig. 5) as well as in the development of selective and potent bromodomain inhibitors for directly targeting SWI/SNF function (Fig. 4) [80, 89, 90, 93]. It came somewhat as a surprise, though, that these bromodomain inhibitors failed to display significant antiproliferative effects. Again, it seems that the non-targeted SWI/SNF domains are sufficient to ensure sustained cell survival. But, with the advent of the PROTACs technology, which enabled the development of these inhibitors into specific degraders of entire SWI/SNF subunits, several proof-of-concept studies have now shown that it is indeed possible to induce synthetic lethality with chemical probes targeting bromodomains [100, 101]. The beauty of this technology is that it offers new targeting opportunities that are still in their infancy. The most intriguing aspect is undoubtedly that it is now, in principle, possible to target ‘undruggable’ SWI/SNF components, for example, specific scaffolding modules, by targeting associated

domains for which inhibitors exist. Many SWI/SNF complex subunits contain interaction modules of yet unknown function. Recent structural studies have, for example, identified a binding groove in the BRK domain of SMARCA4 that may be druggable [130], and more potential targets may emerge in the future.

Targeting the SWI/SNF regulatory network may also be a therapeutic option to induce synthetic lethality, but this requires a more detailed understanding of the intricate mechanistic details of this network, which is currently still lacking. Several studies showed (de)phosphorylation mechanisms in the DNA-damage response, indicating that multiple protein kinases are involved in SWI/SNF regulation [131, 132]. There is a plethora of available kinase inhibitors that could be repurposed for investigating the phosphorylation state of SWI/SNF and its functional consequences. Again, the development of PROTACs, as in the case of the recently developed Aurora A degrader [109], seems to be the way forward to achieve improved therapeutic effects. Another potential future strategy may involve targeting protein–protein interactions with allosteric modulators in order to induce conformational changes within specific SWI/SNF subunits and/or alter the overall assembly of the complex and thereby change the biological binding profile.

The combined efforts of cell biologists, medicinal chemists and structural biologists will hopefully see some of those strategies come to fruition in the near future and provide tailor-made compounds for targeting SWI/SNF complexes in different types of cancer.

Acknowledgements We thank Susanne Müller-Knapp and John Spencer for helpful discussions and for their critical reading of the paper. We are grateful for support by the SGC, a registered charity (no: 1097737) that receives funds from; AbbVie, Bayer AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genentech, Genome Canada through Ontario Genomics Institute [OGI-196], EU/EFPIA/OICR/McGill/KTH/Diamond, Innovative Medicines Initiative 2 Joint Undertaking [EUbOPEN grant 875510], Janssen, Merck KGaA (aka EMD in Canada and US), Merck & Co (aka MSD outside Canada and US), Pfizer, São Paulo Research Foundation-FAPESP, Takeda and Wellcome [106169/ZZ14/Z]. MW is also grateful for support by the Research Training Group Translational Research Innovation—Pharma (TRIP), supported by the Else Kröner-Fresenius Foundation (EKFS). SK, ACJ and AK are thankful for support by the German cancer network DKTK and the FCI. ACJ is supported by German Research Foundation (DFG) grant JO 1473/1-3.

Funding Open Access funding enabled and organized by Projekt DEAL.

## Compliance with ethical standards

Conflict of interest The authors declare no competing interests.

Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.
Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

### References

1. Dawson MA, Kouzarides T. Cancer epigenetics: from mechanism to therapy. Cell. 2012;150:12–27.
2. Kadoch C, Hargreaves DC, Hodges C, Elias L, Ho L, Ranish J, et al. Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy. Nat Genet. 2013;45:592–601.
3. Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA, et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. 2012;2:401–4.
4. Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci Signal. 2013;6:pl1.
5. Schick S, Rendeiro AF, Runggatscher K, Ringler A, Boidol B, Hinkel M, et al. Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalties in human cancers. Nat Genet. 2019;51:1399–410.
6. Wilson BG, Helming KC, Wang X, Kim Y, Vazquez F, Jagani Z, et al. Residual complexes containing SMARCA2 (BRM) underlie the oncogenic drive of SMARCA4 (BRG1) mutation. Mol Cell Biol. 2014;34:1136–44.
7. Helming KC, Wang X, Wilson BG, Vazquez F, Haswell JR, Manchester HE, et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. Nat Med. 2014;20:251–4.
8. Nijman SMB. Synthetic lethality: general principles, utility and detection using genetic screens in human cells. FEBS Lett. 2011;585:1–6.
9. Tagal V, Wei S, Zhang W, Brekken RA, Posner BA, Peyton M, et al. SMARCA4-inactivating mutations increase sensitivity to Aurora kinase A inhibitor VX-680 in non-small cell lung cancers. Nat Commun. 2017;8:14098.
10. Xue Y, Meehan B, Fu Z, Wang XQD, Fiset PO, Rieker R, et al. SMARCA4 loss is synthetic lethal with CDK4/6 inhibition in non-small cell lung cancer. Nat Commun. 2019;10:557.
11. Kouzarides T. Chromatin modifications and their function. Cell. 2007;128:693–705.
12. Clapier CR, Iwasa J, Cairns BR, Peterson CL. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. Nat Rev Mol Cell Biol. 2017;18:407–22.
13. Kadoch C, Crabtree GR. Mammalian SWI/SNF chromatin remodeling complexes and cancer: mechanistic insights gained from human genomics. Sci Adv. 2015;1:e1500447.
14. Wang W, Chi T, Xue Y, Zhou S, Kuo A, Crabtree GR. Architectural DNA binding by a high-mobility-group/kinesin-like subunit in mammalian SWI/SNF-related complexes. Proc Natl Acad Sci U.S.A. 1998;95:492–8.
15. Alpsoy A, Dykhuizen EC. Glioma tumor suppressor candidate region gene 1 (GLTSCR1) and its paralog GLTSCR1-like form

SWI/SNF chromatin remodeling subcomplexes. J Biol Chem. 2018;293:3892–903.
16. Mashtalir N, D'Avino AR, Michel BC, Luo J, Pan J, Otto JE, et al. Modular organization and assembly of SWI/SNF family chromatin remodeling complexes. Cell. 2018;175:1272–1288.e20.
17. He S, Wu Z, Tian Y, Yu Z, Yu J, Wang X, et al. Structure of nucleosome-bound human BAF complex. Science. 2020;367:875–81.
18. Mashtalir N, Suzuki H, Farrell DP, Sankar A, Luo J, Filipovski M, et al. A structural model of the endogenous human BAF complex informs disease mechanisms. Cell. 2020;183:802–817.e24.
19. Alfert A, Moreno N, Kerl K. The BAF complex in development and disease. Epigenetics Chromatin. 2019;12:19.
20. Nakayama RT, Pulice JL, Valencia AM, McBride MJ, McKenzie ZM, Gillespie MA, et al. SMARCB1 is required for widespread BAF complex-mediated activation of enhancers and bivalent promoters. Nat Genet. 2017;49:1613–23.
21. Wang X, Wang S, Troisi EC, Howard TP, Haswell JR, Wolf BK, et al. BRD9 defines a SWI/SNF sub-complex and constitutes a specific vulnerability in malignant rhabdoid tumors. Nat Commun. 2019;10:1881.
22. Alexander JM, Hota SK, He D, Thomas S, Ho L, Pennacchio LA, et al. Brgl modulates enhancer activation in mesoderm lineage commitment. Development. 2015;142:1418–30.
23. Wang X, Lee RS, Alver BH, Haswell JR, Wang S, Mieczkowski J, et al. SMARCB1-mediated SWI/SNF complex function is essential for enhancer regulation. Nat Genet. 2017;49:289–95.
24. Langer LF, Ward JM, Archer TK. Tumor suppressor SMARCB1 suppresses super-enhancers to govern hESC lineage determination. eLife. 2019;8:e45672.
25. Sima X, He J, Peng J, Xu Y, Zhang F, Deng L. The genetic alteration spectrum of the SWI/SNF complex: the oncogenic roles of BRD9 and ACTL6A. PLoS ONE. 2019;14:e0222305.
26. Versteegge I, Sévenet N, Lange J, Rousseau-Merck MF, Ambros P, Handgretinger R, et al. Truncating mutations of hSNF5/INI1 in aggressive paediatric cancer. Nature. 1998;394:203–6.
27. Michel BC, D'Avino AR, Cassel SH, Mashtalir N, McKenzie ZM, McBride MJ, et al. A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation. Nat Cell Biol. 2018;20:1410–20.
28. Brien GL, Remillard D, Shi J, Hemming ML, Chabon J, Wynne K, et al. Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma. eLife. 2018;7:3892.
29. Wilson BG, Wang X, Shen X, McKenna ES, Lemieux ME, Cho Y-J, et al. Epigenetic antagonism between polycomb and SWI/SNF complexes during oncogenic transformation. Cancer Cell. 2010;18:316–28.
30. Clark J, Rocques PJ, Crew AJ, Gill S, Shipley J, Chan AM, et al. Identification of novel genes, SYT and SSX, involved in the t(X;18)(p11.2;q11.2) translocation found in human synovial sarcoma. Nat Genet. 1994;7:502–8.
31. McBride MJ, Pulice JL, Beird HC, Ingram DR, D'Avino AR, Shern JF, et al. The SS18-SSX fusion oncoprotein hijacks BAF complex targeting and function to drive synovial sarcoma. Cancer Cell. 2018;33:1128–41.e7.
32. Cheng JX, Tretiakova M, Gong C, Mandal S, Krausz T, Taxy JB. Renal medullary carcinoma: rhabdoid features and the absence of INI1 expression as markers of aggressive behavior. Mod Pathol. 2008;21:647–52.
33. Kohashi K, Oda Y. Oncogenic roles of SMARCB1/INI1 and its deficient tumors. Cancer Sci. 2017;108:547–52.
34. Allen MD, Freund SM, Zinzalla G, Bycroft M. The SWI/SNF subunit INI1 contains an N-terminal winged helix DNA binding domain that is a target for mutations in schwannomatosis. Structure. 2015;23:1344–9.

35. Smith MJ, Wallace AJ, Bowers NL, Eaton H, Evans DGR. SMARCB1 mutations in schwannomatosis and genotype correlations with rhabdoid tumors. Cancer Genet. 2014;207:373–8.
36. Hodges C, Kirkland JG, Crabtree GR. The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer. Cold Spring Harb Perspect Med. 2016;6:a026930.
37. Bai J, Mei P, Zhang C, Chen F, Li C, Pan Z, et al. BRG1 is a prognostic marker and potential therapeutic target in human breast cancer. PLoS One. 2013;8:e59772.
38. Reisman DN, Sciarrotta J, Wang W, Funkhouser WK, Weissman BE. Loss of BRG1/BRM in human lung cancer cell lines and primary lung cancers: correlation with poor prognosis. Cancer Res. 2003;63:560–6.
39. Pyo J-S, Son BK, Oh D, Kim EK. BRG1 is correlated with poor prognosis in colorectal cancer. Hum Pathol. 2018;73:66–73.
40. Stanton BZ, Hodges C, Calarco JP, Braun SMG, Ku WL, Kadoch C, et al. Smarca4 ATPase mutations disrupt direct eviction of PRC1 from chromatin. Nat Genet. 2017;49:282–8.
41. Glaros S, Cirrincione GM, Muchardt C, Kleer CG, Michael CW, Reisman D. The reversible epigenetic silencing of BRM: implications for clinical targeted therapy. Oncogene. 2007;26:7058–66.
42. Pavlidou E, Balis V. Diagnostic significance and prognostic role of the ARID1A gene in cancer outcomes. World Acad Sci J. 2020;2:49–64.
43. Wiegand KC, Shah SP, Al-Agha OM, Zhao Y, Tse K, Zeng T, et al. ARID1A mutations in endometriosis-associated ovarian carcinomas. N Engl J Med. 2010;363:1532–43.
44. Zhang L, Wang C, Yu S, Jia C, Yan J, Lu Z, et al. Loss of ARID1A expression correlates with tumor differentiation and tumor progression stage in pancreatic ductal adenocarcinoma. Technol Cancer Res Treat. 2018;17:1533034618754475.
45. Cho H, Kim JS-Y, Chung H, Perry C, Lee H, Kim J-H. Loss of ARID1A/BAF250a expression is linked to tumor progression and adverse prognosis in cervical cancer. Hum Pathol. 2013;44:1365–74.
46. Joerger AC, Fersht AR. The p53 pathway: origins, inactivation in cancer, and emerging therapeutic approaches. Annu Rev Biochem. 2016;85:375–404.
47. Bosse T, ter Haar NT, Seeber LM, v Diest PJ, Hes FJ, HFA Vasen, et al. Loss of ARID1A expression and its relationship with PI3K-Akt pathway alterations, TP53 and microsatellite instability in endometrial cancer. Mod Pathol. 2013;26:1525–35.
48. Lee D, Yu EJ, Ham I-H, Hur H, Kim Y-S. AKT inhibition is an effective treatment strategy in ARID1A-deficient gastric cancer cells. Onco Targets Ther. 2017;10:4153–9.
49. Samartzis EP, Gutsche K, Dedes KJ, Fink D, Stucki M, Imesch P. Loss of ARID1A expression sensitizes cancer cells to PI3K-and AKT-inhibition. Oncotarget. 2014;5:5295–303.
50. Chandler RL, Damrauer JS, Raab JR, Schisler JC, Wilkerson MD, Didion JP, et al. Coexistent ARID1A-PIK3CA mutations promote ovarian clear-cell tumorigenesis through pro-tumorigenic inflammatory cytokine signalling. Nat Commun. 2015;6:6118.
51. Hoyer J, Ekici AB, Endele S, Popp B, Zweier C, Wiesener A, et al. Haploinsufficiency of ARID1B, a member of the SWI/SNF-a chromatin-remodeling complex, is a frequent cause of intellectual disability. Am J Hum Genet. 2012;90:565–72.
52. Santen GWE, Aten E, Sun Y, Almomani R, Gilissen C, Nielsen M, et al. Mutations in SWI/SNF chromatin remodeling complex gene ARID1B cause Coffin-Siris syndrome. Nat Genet. 2012;44:379–80.
53. Manceau G, Letouzé E, Guichard C, Didelot A, Cazes A, Corté H, et al. Recurrent inactivating mutations of ARID2 in non-small cell lung carcinoma. Int J Cancer. 2013;132:2217–21.

54. Li M, Zhao H, Zhang X, Wood LD, Anders RA, Choti MA, et al. Inactivating mutations of the chromatin remodeling gene ARID2 in hepatocellular carcinoma. Nat Genet. 2011;43:828–9.
55. Oba A, Shimada S, Akiyama Y, Nishikawaji T, Mogushi K, Ito H, et al. ARID2 modulates DNA damage response in human hepatocellular carcinoma cells. J Hepatol. 2017;66:942–51.
56. Moreno T, González-Silva L, Monterde B, Betancor-Fernández I, Revilla C, Agraz-Doblas A, et al. ARID2 deficiency promotes tumor progression and is associated with higher sensitivity to PARP inhibition in lung cancer. bioRxiv. 2020: https://doi.org/10.1101/2020.01.10.898726.
57. Varela I, Tarpey P, Raine K, Huang D, Ong CK, Stephens P, et al. Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma. Nature. 2011;469:539–42.
58. Nargund AM, Pham CG, Dong Y, Wang PI, Osmangoyoglu HU, Xie Y, et al. The SWI/SNF protein PBRM1 restrains VHL-loss-driven clear cell renal cell carcinoma. Cell Rep. 2017;18:2893–906.
59. Chandrasekaran R, Thompson M. Polybromo-1-bromodomains bind histone H3 at specific acetyl-lysine positions. Biochem Biophys Res Commun. 2007;355:661–6.
60. Charlop-Powers Z, Zeng L, Zhang Q, Zhou M-M. Structural insights into selective histone H3 recognition by the human Polybromo bromodomain 2. Cell Res. 2010;20:529–38.
61. Porter EG, Dykhuizen EC. Individual bromodomains of polybromo-1 contribute to chromatin association and tumor suppression in clear cell renal carcinoma. J Biol Chem. 2017;292:2601–10.
62. Slaughter MJ, Shanle EK, McFadden AW, Hollis ES, Suttle LE, Strahl BD, et al. PBRM1 bromodomains variably influence nucleosome interactions and cellular function. J Biol Chem. 2018;293:13592–603.
63. Liao L, Alicea-Velázquez NL, Langbein L, Niu X, Cai W, Cho E-A, et al. High affinity binding of H3K14ac through collaboration of bromodomains 2, 4 and 5 is critical for the molecular and tumor suppressor functions of PBRM1. Mol Oncol. 2019;13:811–28.
64. Kelso TWR, Porter DK, Amaral ML, Shokhirev MN, Benner C, Hargreaves DC. Chromatin accessibility underlies synthetic lethality of SWI/SNF subunits in ARID1A-mutant cancers. eLife. 2017;6:14648.
65. Hoffman GR, Rahal R, Buxton F, Xiang K, McAllister G, Frias E, et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers: PNAS. Proc Natl Acad Sci USA. 2014;111:3128–33.
66. Oike T, Ogiwara H, Tominaga Y, Ito K, Ando O, Tsuta K, et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 2013;73:5508–18.
67. Mathur R, Alver BH, San Roman AK, Wilson BG, Wang X, Agoston AT, et al. ARID1A loss impairs enhancer-mediated gene regulation and drives colon cancer in mice. Nat Genet. 2017;49:296–302.
68. Bracken AP, Brien GL, Verrijzer CP. Dangerous liaisons: interplay between SWI/SNF, NuRD, and Polycomb in chromatin regulation and cancer. Genes Dev. 2019;33:936–59.
69. Kaelin WG. Synthetic lethality: a framework for the development of wiser cancer therapeutics. Genome Med. 2009;1:99.
70. Frye SV. The art of the chemical probe. Nat Chem Biol. 2010;6:159–61.
71. Arrowsmith CH, Audia JE, Austin C, Baell J, Bennett J, Blagg J, et al. The promise and peril of chemical probes. Nat Chem Biol. 2015;11:536–41.
72. Müller S, Ackloo S, Arrowsmith CH, Bauser M, Baryza JL, Blagg J, et al. Donated chemical probes for open science. eLife. 2018;7:e34311.

73. Xia X, Liu X, Li T, Fang X, Chen Z. Structure of chromatin remodeler Swi2/Snf2 in the resting state. Nat Struct Mol Biol. 2016;23:722–9.
74. Papillon JPN, Nakajima K, Adair CD, Hempel J, Jouk AO, Karki RG, et al. Discovery of orally active inhibitors of brahma homolog (BRM)/SMARCA2 ATPase activity for the treatment of brahma related gene 1 (BRG1)/SMARCA4-mutant cancers. J Med Chem. 2018;61:10155–72.
75. Filippakopoulos P, Picaud S, Mangos M, Keates T, Lambert J-P, Barsyte-Lovejoy D, et al. Histone recognition and large-scale structural analysis of the human bromodomain family. Cell. 2012;149:214–31.
76. Sun H, Liu J, Zhang J, Shen W, Huang H, Xu C, et al. Solution structure of BRD7 bromodomain and its interaction with acetylated peptides from histone H3 and H4. Biochem Biophys Res Commun. 2007;358:435–41.
77. Karim RM, Chan A, Zhu J-Y, Schönbrunn E. Structural basis of inhibitor selectivity in the BRD7/9 subfamily of bromodomains. J Med Chem. 2020;63:3227–37.
78. Hohmann AF, Martin LJ, Minder JL, Roe J-S, Shi J, Steurer S, et al. Sensitivity and engineered resistance of myeloid leukemia cells to BRD9 inhibition. Nat Chem Biol. 2016;12:672–9.
79. Drost J, Mantovani F, Tocco F, Elkon R, Comel A, Holstege H, et al. BRD7 is a candidate tumour suppressor gene required for p53 function. Nat Cell Biol. 2010;12:380–9.
80. Martin LJ, Koegl M, Bader G, Cockcroft X-L, Fedorov O, Fiegen D, et al. Structure-based design of an in vivo active selective BRD9 inhibitor. J Med Chem. 2016;59:4462–75.
81. Vidler LR, Brown N, Knapp S, Hoelder S. Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites. J Med Chem. 2012;55:7346–59.
82. Filippakopoulos P, Qi J, Picaud S, Shen Y, Smith WB, Fedorov O, et al. Selective inhibition of BET bromodomains. Nature. 2010;468:1067–73.
83. Nicodeme E, Jeffrey KL, Schaefer U, Beinke S, Dewell S, Chung C, et al. Suppression of inflammation by a synthetic histone mimic. Nature. 2010;468:1119–23.
84. Wu Q, Heidenreich D, Zhou S, Ackloo S, Krämer A, Nakka K, et al. A chemical toolbox for the study of bromodomains and epigenetic signaling. Nat Commun. 2019;10:1915.
85. Cochran AG, Conery AR, Sims RJ. Bromodomains: a new target class for drug development. Nat Rev Drug Discov. 2019;18:609–28.
86. Clark PGK, Vieira LCC, Tallant C, Fedorov O, Singleton DC, Rogers CM, et al. LP99: discovery and synthesis of the first selective BRD7/9 bromodomain inhibitor. Angew Chem Int Ed Engl. 2015;54:6217–21.
87. Crawford TD, Vartanian S, Côté A, Bellon S, Duplessis M, Flynn EM, et al. Inhibition of bromodomain-containing protein 9 for the prevention of epigenetically-defined drug resistance. Bioorg Med Chem Lett. 2017;27:3534–41.
88. Hay DA, Rogers CM, Fedorov O, Tallant C, Martin S, Monteiro OP, et al. Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains. Med Chem Commun. 2015;6:1381–6.
89. Theodoulou NH, Bamborough P, Bannister AJ, Becher I, Bit RA, Che KH, et al. Discovery of I-BRD9, a selective cell active chemical probe for bromodomain containing protein 9 inhibition. J Med Chem. 2016;59:1425–39.
90. Gerstenberger BS, Trzupek JD, Tallant C, Fedorov O, Filipakopoulos P, Brennan PE, et al. Identification of a chemical probe for family VIII bromodomains through optimization of a fragment hit. J Med Chem. 2016;59:4800–11.
91. Sutherell CL, Tallant C, Monteiro OP, Yapp C, Fuchs JE, Fedorov O, et al. Identification and development of 2,3-dihydropyrrolo[1,2-a]quinazolin-5(1H)-one inhibitors targeting bromodomains within the switch/sucrose nonfermenting complex. J Med Chem. 2016;59:5095–101.
92. Lu T, Hu J-C, Lu W-C, Han J, Ding H, Jiang H, et al. Identification of small molecule inhibitors targeting the SMARCA2 bromodomain from a high-throughput screening assay. Acta Pharm Sin. 2018;39:1544–52.
93. Wanior M, Preuss F, Ni X, Krämer A, Mathea S, Göbel T, et al. Pan-SMARCA/PB1 bromodomain inhibitors and their role in regulating adipogenesis. J Med Chem. 2020;63:14680–99.
94. Albrecht BK, Cote A, Crawford T, Duplessis M, Good AC, Leblanc Y, et al. Therapeutic pyridazine compounds and uses thereof; U.S. Patent WO2016138114 A1, 2016.
95. Myrianthopoulos V, Gaboriaud-Kolar N, Tallant C, Hall M-L, Grigoriou S, Brownlee PM, et al. Discovery and optimization of a selective ligand for the switch/sucrose nonfermenting-related bromodomains of polybromo protein-1 by the use of virtual screening and hydration analysis. J Med Chem. 2016;59:8787–803.
96. Marian CA, Stoszko M, Wang L, Leighty MW, Crignis E, de Maschinot CA, et al. Small molecule targeting of specific BAF (mSWI/SNF) complexes for HIV latency reversal. Cell Chem Biol. 2018;25:1443–55.e14.
97. Chory EJ, Kirkland JG, Chang C-Y, D’Andrea VD, Gourisankar S, Dykhuizen EC, et al. Chemical inhibitors of a selective SWI/SNF function synergize with ATR inhibition in cancer cell killing. ACS Chem Biol. 2020;15:1685–96.
98. Williamson CT, Miller R, Pemberton HN, Jones SE, Campbell J, Konde A, et al. ATR inhibitors as a synthetic lethal therapy for tumours deficient in ARID1A. Nat Commun. 2016;7:13837.
99. Krämer KF, Moreno N, Frühwald MC, Kerl K. BRD9 inhibition, alone or in combination with cytostatic compounds as a therapeutic approach in rhabdoid tumors. Int J Mol Sci. 2017;18:1537.
100. Pettersson M, Crews CM. Proteolysis Targeting Chimeras (PROTACs)—Past, present and future. Drug Disco Today Technol. 2019;31:15–27.
101. Nalawansha DA, Crews CM. PROTACs: an emerging therapeutic modality in precision medicine. Cell Chem Biol. 2020;27:998–1014.
102. Remillard D, Buckley DL, Paulk J, Brien GL, Sonnett M, Seo H-S, et al. Degradation of the BAF complex factor BRD9 by heterobifunctional ligands. Angew Chem Int Ed Engl. 2017;56:5738–43.
103. Zoppi V, Hughes SJ, Maniaci C, Testa A, Gmaschitz T, Wieshofer C, et al. Iterative design and optimization of initially inactive proteolysis targeting chimeras (PROTACs) identify VZ185 as a potent, fast, and selective von Hippel-Lindau (VHL) based dual degrader probe of BRD9 and BRD7. J Med Chem. 2019;62:699–726.
104. Farnaby W, Koegl M, Roy MJ, Whitworth C, Diers E, Trainor N, et al. BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design. Nat Chem Biol. 2019;15:672–80.
105. Vangamudi B, Paul TA, Shah PK, Kost-Alimova M, Nottebaum L, Shi X, et al. The SMARCA2/4 ATPase domain surpasses the bromodomain as a drug target in SWI/SNF-mutant cancers: Insights from cDNA rescue and PFI-3 inhibitor studies. Cancer Res. 2015;75:3865–78.
106. Fedorov O, Castex J, Tallant C, Owen DR, Martin S, Aldeghi M, et al. Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance. Sci Adv. 2015;1:e1500723.
107. Crew AP, Wang J, Berlin M, Dragovich P, Chen H, Staben L. BRM targeting compounds and associated methods of use.; U.S. Patent WO20190300521, 2019.

108. Wu C, Lyu J, Yang EJ, Liu Y, Zhang B, Shim JS. Targeting AURKA-CDC25C axis to induce synthetic lethality in ARID1A-deficient colorectal cancer cells. Nat Commun. 2018;9:3212.
109. Adhikari B, Bozilovic J, Diebold M, Schwarz JD, Hofstetter J, Schröder M, et al. PROTAC-mediated degradation reveals a non-catalytic function of AURORA-A kinase. Nat Chem Biol. 2020;16:1179–88.
110. Miller RE, Brough R, Bajrami I, Williamson CT, McDade S, Campbell J, et al. Synthetic lethal targeting of ARID1A-mutant ovarian clear cell tumors with dasatinib. Mol Cancer Ther. 2016;15:1472–84.
111. Xue Y, Meehan B, Macdonald E, Venneti S, Wang XQD, Witkowski L, et al. CDK4/6 inhibitors target SMARCA4-determined cyclin D1 deficiency in hypercalcemic small cell carcinoma of the ovary. Nat Commun. 2019;10:558.
112. Muscat A, Popovski D, Jayasekara WSN, Rossello FJ, Ferguson M, Marini KD, et al. Low-dose histone deacetylase inhibitor treatment leads to tumor growth arrest and multi-lineage differentiation of malignant rhabdoid tumors. Clin Cancer Res. 2016;22:3560–70.
113. Knipstein JA, Birks DK, Donson AM, Alimova I, Foreman NK, Vibhakar R. Histone deacetylase inhibition decreases proliferation and potentiates the effect of ionizing radiation in atypical teratoid/rhabdoid tumor cells. Neuro-Oncol. 2012;14:175–83.
114. Bitler BG, Wu S, Park PH, Hai Y, Aird KM, Wang Y, et al. ARID1A-mutated ovarian cancers depend on HDAC 6 activity. Nat Cell Biol. 2017;19:962–73.
115. Arbely E, Natan E, Brandt T, Allen MD, Veprintsev DB, Robinson CV, et al. Acetylation of lysine 120 of p53 endows DNA-binding specificity at effective physiological salt concentration. Proc Natl Acad Sci U.S.A. 2011;108:8251–6.
116. Fukumoto T, Park PH, Wu S, Fatkhutdinov N, Karakashev S, Nacarelli T, et al. Repurposing pan-HDAC inhibitors for ARID1A-mutated ovarian cancer. Cell Rep. 2018;22:3393–400.
117. Slade D. PARP and PARG inhibitors in cancer treatment. Genes Dev. 2020;34:360–94.
118. Peng G, Yim E-K, Dai H, Jackson AP, van der Burgt I, Pan M-R, et al. BRIT1/MCPH1 links chromatin remodelling to DNA damage response. Nat Cell Biol. 2009;11:865–72.
119. Shen J, Peng Y, Wei L, Zhang W, Yang L, Lan L, et al. ARID1A deficiency impairs the DNA damage checkpoint and sensitizes cells to PARP inhibitors. Cancer Discov. 2015;5:752–67.
120. Park Y, Chui MH, Suryo Rahmanto Y, Yu Z-C, Shamanna RA, Bellani MA, et al. Loss of ARID1A in tumor cells renders selective vulnerability to combined ionizing radiation and PARP inhibitor therapy. Clin Cancer Res. 2019;25:5584–94.
121. Howard TP, Arnoff TE, Song MR, Giacomelli AO, Wang X, Hong AL, et al. MDM2 and MDM4 Are therapeutic vulnerabilities in malignant rhabdoid tumors. Cancer Res. 2019;79:2404–14.
122. Carugo A, Minelli R, Sapio L, Soeung M, Carbone F, Robinson FS, et al. p53 Is a master regulator of proteostasis in SMARCB1-deficient malignant rhabdoid tumors. Cancer Cell. 2019;35:204–220.
123. Hong AL, Tseng Y-Y, Wala JA, Kim W-J, Kynnap BD, Doshi MB, et al. Renal medullary carcinomas depend upon SMARCB1 loss and are sensitive to proteasome inhibition. eLife. 2019;8:818.
124. Banito A, Li X, Laporte AN, Roe J-S, Sanchez-Vega F, Huang C-H, et al. The SS18-SSX oncoprotein hijacks KDM2B-PRC1.1 to drive synovial sarcoma. Cancer Cell. 2018;34:346–8.
125. Alimova I, Pierce A, Danis E, Donson A, Birks DK, Griesinger A, et al. Inhibition of MYC attenuates tumor cell self-renewal and promotes senescence in SMARCB1-deficient group 2

126. Deribe YL, Sun Y, Terranova C, Khan F, Martinez-Ledesma J, Gay J, et al. Mutations in the SWI/SNF complex induce a targetable dependence on oxidative phosphorylation in lung cancer. Nat Med. 2018;24:1627.
127. Molina JR, Sun Y, Protopopova M, Gera S, Bandi M, Bristow C, et al. An inhibitor of oxidative phosphorylation exploits cancer vulnerability. Nat Med. 2018;24:1036–46.
128. Ogiwara H, Takahashi K, Sasaki M, Kuroda T, Yoshida H, Watanabe R, et al. Targeting the vulnerability of glutathione metabolism in ARID1A-deficient cancers. Cancer Cell. 2019;35:177–190.e8.
129. van der Vaart A, Godfrey M, Portegijs V, van den Heuvel S. Dose-dependent functions of SWI/SNF BAF in permitting and inhibiting cell proliferation in vivo. Sci Adv. 2020;6:eaay3823.
130. Allen MD, Bycroft M, Zinzalla G. Structure of the BRK domain of the SWI/SNF chromatin remodeling complex subunit BRG1 reveals a potential role in protein-protein interactions. Protein Sci. 2020;29:1047–53.
131. Bennetzen MV, Larsen DH, Bunkenborg J, Bartek J, Lukas J, Andersen JS. Site-specific phosphorylation dynamics of the nuclear proteome during the DNA damage response. Mol Cell Proteom. 2010;9:1314–23.
132. Ochoa D, Jarnuczak AF, Viéitez C, Gehre M, Soucheray M, Mateus A, et al. The functional landscape of the human phosphoproteome. Nat Biotechnol. 2020;38:365–73.
133. Guo C, Liu S, Wang J, Sun M-Z, Greenaway FT. ACTB in cancer. Clin Chim Acta. 2013;417:39–44.
134. Lissanu Deribe Y, Sun Y, Terranova C, Khan F, Martinez-Ledesma J, Gay J, et al. Mutations in the SWI/SNF complex induce a targetable dependence on oxidative phosphorylation in lung cancer. Nat Med. 2018;24:1047–57.
135. Shorstova T, Marques M, Su J, Johnston J, Kleinman CL, Hamel N, et al. SWI/SNF-compromised cancers are susceptible to bromodomain inhibitors. Cancer Res. 2019;79:2761–74.
136. Kim KH, Kim W, Howard TP, Vazquez F, Tsherniak A, Wu JN, et al. SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2. Nat Med. 2015;21:1491–6.
137. Chan-Penebre E, Armstrong K, Drew A, Grassian AR, Feldman I, Knutson SK, et al. Selective killing of SMARCA2- and SMARCA4-deficient small cell carcinoma of the ovary, hypercalcemic type cells by inhibition of EZH2: in vitro and in vivo preclinical models. Mol Cancer Ther. 2017;16:850–60.
138. Januario T, Ye X, Bainer R, Alicke B, Smith T, Haley B, et al. PRC2-mediated repression of SMARCA2 predicts EZH2 inhibitor activity in SWI/SNF mutant tumors. Proc Natl Acad Sci U.S.A. 2017;114:12249–54.
139. Bitler BG, Aird KM, Garipov A, Li H, Amatangelo M, Kos senkov AV, et al. Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers. Nat Med. 2015;21:231–8.
140. Ehrenhofer-Wölfer K, Puchner T, Schwarz C, Rippka J, Blaha- Ostermann S, Strobl U, et al. SMARCA2-deficiency confers sensitivity to targeted inhibition of SMARCA4 in esophageal squamous cell carcinoma cell lines. Sci Rep. 2019;9:11661.
141. Huang K, Sun R, Chen J, Yang Q, Wang Y, Zhang Y, et al. A novel EZH2 inhibitor induces synthetic lethality and apoptosis in PBRM1-deficient cancer cells. Cell Cycle. 2020;19:758–771.
142. Jones SE, Fleuren EDG, Frankum J, Konde A, Williamson CT, Krastev DB, et al. ATR is a therapeutic target in synovial sarcoma. Cancer Res. 2017;77:7014–26.
