
Structure

Structural basis of bacteriophage lambda capsid maturation

Graphical abstract

Authors
Chang Wang, Jianwei Zeng, Jiawei Wang

Correspondence
zengjw8890@163.com (J.Z.), jwwang@tsinghua.edu.cn (J.W.)

In brief
Bacteriophage lambda serves as a model system for studying genetic, biochemical, and structural features during virus development. Wang et al. present high-resolution cryo-electron microscopy (cryo-EM) structures of the classical bacteriophage lambda procapsid and mature capsid at 3.88 Å and 3.76 Å resolution, respectively, and decipher the mechanisms of capsid maturation and stability.

Highlights
- Solution of the 3.88 Å cryo-EM structure of the bacteriophage lambda procapsid
- Determination of the 3.76 Å cryo-EM structure of the capsid
- Conformational changes of major capsid protein gpE occur during capsid maturation
- Decoration protein gpD plays an auxiliary role during capsid maturation

Wang et al., 2022, Structure 30, 637–645
April 7, 2022 © 2021 Elsevier Ltd.
https://doi.org/10.1016/j.str.2021.12.009

Structure

Article

**Structural basis of bacteriophage lambda capsid maturation**

Chang Wang,$^{1}$ Jianwei Zeng,$^{1,*}$ and Jiawei Wang$^{1,2,*}$

$^{1}$State Key Laboratory of Membrane Biology, Beijing Advanced Innovation Center for Structural Biology, School of Life Sciences, Tsinghua University, Beijing 100084, P. R. China

$^{2}$Lead contact

*Correspondence: zengjw8890@163.com (J.Z.), jwwang@tsinghua.edu.cn (J.W.)

https://doi.org/10.1016/j.str.2021.12.009

---

### SUMMARY

Bacteriophage lambda is an excellent model system for studying capsid assembly of double-stranded DNA (dsDNA) bacteriophages, some dsDNA archaeal viruses, and herpesviruses. HK97 fold coat proteins initially assemble into a precursor capsid (procapsid) and subsequent genome packaging triggers morphological expansion of the shell. An auxiliary protein is required to stabilize the expanded capsid structure. To investigate the capsid maturation mechanism, we determined the cryo-electron microscopy structures of the bacteriophage lambda procapsid and mature capsid at 3.88 Å and 3.76 Å resolution, respectively. Besides primarily rigid body movements of common features of the major capsid protein gpE, large-scale structural rearrangements of other domains occur simultaneously. Assembly of intercapsomers within the procapsid is facilitated by layer-stacking effects at 3-fold vertices. Upon conformational expansion of the capsid shell, the missing top layer is fulfilled by cementing the gpD protein against the internal pressure of DNA packaging. Our structures illuminate the assembly mechanisms of dsDNA viruses.

---

### INTRODUCTION

DNA packaging and capsid expansion is a buildup of internal pressure within the shell (Fuller et al., 2007). To strengthen the capsid shell against the internal pressure of DNA packaging, a decoration protein (gpD) attaches to the quasi- and icosahedral-3-fold vertices to stabilize the expanded mature capsid shell. Finally, tails are added to the DNA-filled capsid via a connector composed of gpW and gpFII proteins (Cardarelli et al., 2010), to complete the virus particle (Campbell et al., 2020; Gaussier et al., 2006).

Structural characterization of lambda capsid assembly and maturation has been performed using cryo-electron microscopy (cryo-EM) (Dokland and Murialdo, 1993), while individual components of lambda capsid or their homologues were studied by macromolecular crystallography and nuclear magnetic resonance spectroscopy (NMR). Highlights include the crystal structure of the HK97 fold-like gpE homologue CFT073 (PDB: 3BQW), the trimeric crystal structure of gpD in which the N-terminal residues required for capsid binding are disordered (PDB: 1C5E and 1TCZ) (Chang et al., 2004; Yang et al., 2000), and the full-length NMR structure of the monomeric gpD (PDB: 1VD0) (Iwai et al., 2005). Moreover, the cryo-EM structures of a 14.5 Å empty procapsid shell and a 6.8 Å DNA-filled mature capsid have been determined (Lander et al., 2008). The pseudoatomic models of the procapsid and capsid obtained through docking the HK97 and gpD crystal structures into the cryo-EM maps, building the missing N-terminal 14 residues of gpD proteins, and further adjustment through a combination of chemical crosslinking/mass spectrometry and computational modeling (Lambert

Structure 30, 637–645, April 7, 2022 © 2021 Elsevier Ltd.

CellPress

Structure
Article

A  Procapsid
B  T = 7 icosahedral symmetry
C  Capsid

D  Procapsid ASU
E  Mature capsid ASU

Figure 1. cryo-EM reconstruction of bacteriophage lambda procapsid and mature capsid

(A) Radially colored cryo-EM map of procapsid from the phage center (red to blue), overlaid with a black mesh of hexagons and pentagons covering an icosahedron.

(B) Central slices of the lambda procapsid and capsid reconstructions.

(C) Cryo-EM map of mature capsid.

(D and E) Local reconstructions of the asymmetric units (ASUs) of procapsid (D) and capsid (E). The color gradient represents the local resolution of the maps. The overall resolutions of (A), (C), (D), and (E) are 5.0, 5.6, 3.88, and 3.76 Å, respectively.

et al., 2017; Singh et al., 2013) have greatly improved our understanding of the capsid maturation mechanism. However, the high-resolution experimental structures of the lambda major capsid protein (MCP) gpE in both procapsid and capsid states remain unreported. A better understanding of gpE could facilitate further research on capsid assembly, maturation, and stability. To this end, we herein sought to determine high-resolution procapsid and capsid structures.

RESULTS

cryo-EM reconstruction of bacteriophage lambda procapsid and capsid

Phage lambda capsid-related particles were purified from a lysate of phage-infected cells and imaged under cryogenic conditions using a transmission electron microscope (Figure S1). Two kinds of particles were observed in the sample: a round, thick-shelled procapsid and an angular, thin-shelled, mature capsid. Procapsid and capsid particles were picked separately and reconstructed by applying 60-fold icosahedral symmetry in order to maximize the signal-to-noise ratio of the raw data (Table S1). The resulting icosahedrally averaged lambda procapsid

and capsid cryo-EM maps at resolutions of 5.03 Å and 5.6 Å, respectively, are of good quality, revealing distinct secondary structures, including the spine helix of the MCP gpE. Procapsids are round, lumpy, and thick-shelled (Figure 1A), with a diameter of 58 nm (Figure 1B), while mature capsids are angular, thin-shelled, and larger, with a diameter of 68 nm (Figures 1B and 1C). The protruding densities correspond to the decoration protein gpD at the icosahedral and quasi-3-fold axis (Figure 1C). The gpD trimer located at quasi-3-fold vertices is farther from the center than that at the icosahedral vertex.

To further improve the local resolution of cryo-EM maps, the data used for icosahedral reconstruction were expanded to generate all symmetry-related orientations of individual particles (Scheres, 2016). After the icosahedral asymmetric unit (ASU) of the procapsid and capsid were masked out, local refinement was performed to permit ASU subregions of the viral procapsid/capsid to make small deviations from strict icosahedral symmetry (Goetschius et al., 2019). Focused refinement resulted in the procapsid ASU at 3.88 Å resolution (Figure 1D) and the capsid ASU at 3.76 Å resolution (Figure 1E), from which the atomic models of the ASUs of the procapsid and capsid were built (Table S1).

638 Structure 30, 637–645, April 7, 2022

Structure
Article

Figure 2. Components and features of MCP gpE

(A) The HK97 fold with important common features labeled and color-coded as indicated in the figure. The inset in (A) shows specific interactions between the E-loop tip and the neighboring gpE.

(B) Besides the common parts of gpE in (A), the unique components in procapsid gpE are indicated.

(C) Additional components in capsid penton gpE.

(D) Additional components in capsid hexon gpE.

Organization and classification of lambda major capsid protein gpE

The typical HK97-like capsid protein consists of various conserved elements (Figure 2A), including (1) the A domain (axial domain), with a triangular shape comprising a β-sheet-rich core surrounded by prominent α-helices along the edges that interact with adjacent A domains; (2) the P domain (peripheral domain), consisting of a long α-helix (backbone or spine helix) nestled against a multistranded β-sheet; (3) the E-loop (extended loop), which protrudes from the P domain contacting neighboring subunits; and (4) the N arm (N-terminal arm) protruding from the subunit body. Besides the common HK97 components, the lambda gpE monomer includes several insertion domains (Figure 2A): the la loop (insertion into the A domain), the ls loop (inserted into the spine helix, also called the “G loop”), and the lp loop (inserted into the P domain).

Comparison of 7 gpEs in the procapsid (Figure S2A), as well as 7 gpEs in the capsid (Figure S2B), revealed major rearrangements involving the E loop, the A domain (especially the A loop), and the N arm, and minor rearrangements involving the P loop and S loop (Figure S2C). However, the E loop tip from the neighboring gpE molecule is specifically clamped between the la loop, ls loop, and P domain (insets in Figures 2A and S2C), and this arrangement is conserved in the flagellotropic bacteriophage YSD1 (Hardy et al., 2020). The E loop stem is the only variable part, stretching into a β-sheet during capsid maturation. The N arm refolds from N arm 1 to N arm 2 and adds to the third strand of a β-sheet formed by a neighboring E loop stem during transition to the mature capsid (Figures 2B–2D). The A loops form intracapsomer interactions at the center of capsomers and refold from A loop 1 to A loop 2 in the hexon capsomer during maturation. Based on the features of the A loop and N arm, gpE can be classified into three states: procapsid gpE (Figure 2B), capsid penton gpE (Figure 2C), and capsid hexon gpE (Figure 2D).

Overall capsomer shape changes during conversion of procapsid to mature capsid

All capsomers in procapsids adopt domelike shapes that give the procapsids a lumpy appearance, with heights of 69 Å and 53 Å for pentons and hexons, respectively (Figure 3A). The domed hexons in procapsids are asymmetric (Figure 3B). To establish a pseudoatomic model for the lambda procapsid shell, Singh et al. (2013) identified chemical crosslinks within the procapsid by mass spectrometry. Most of the chemical crosslinks can be interpreted reasonably using the current models of procapsid penton and hexon capsomers, except the 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide HCl (EDC) crosslinks between K79 and E158, which are ~34 Å in length, considerably longer than the 10 Å expected for EDC. In mature capsids, the hexons morph into almost flat, regular hexagons with a height of only 36 Å, while pentons remain pyramidal in shape, with a

height of 47 Å at vertices (Figure 3C). Upon the reorganization of the procapsid into a mature capsid, each subunit gpE of the precursor alters both its conformation and its relationship with its neighbors. The gpEs in procapsid pentons rotate ~30° and translate ~21 Å in order to transition to capsid pentons (Figure 3A). Simultaneously, procapsid penton gpEs alter their conformation to form capsid penton gpEs. The procapsid hexon gpEs rotate more than 38° and translate less than 9 Å during maturation, and thereby they transition to capsid hexon gpEs. The angle between penton and hexon capsomers remains similar in the procapsid and capsid (Figure 3A).

Upon maturation, conformational changes of the capsid structure expose the binding sites for the attachment of gpD at the quasi- and icosahedral-3-fold vertices. The gpD protein has a tulip fold and trimeric organization (Chang et al., 2004; Yang et al., 2000). The N terminus of the gpD protein extends out of the beta-tulip domain along the icosahedral or quasi-2-fold axis (Figure 3D). Although gpD trimers at quasi- and icosahedral-3-fold-vertices are located slightly away from the center of the virus, both have an identical conformation by themselves (Figure S2D).

### Intracapsomer interactions

Within the capsomers, the A domain fills the centers of the pentons and hexons, while the P domain forms the periphery. E loops make important intracapsomer contacts, in which the E-loop tip is specifically clamped by the P domain, Ia loop, and Is loop of the adjacent subunit in the same capsomer (Figure 2A), as in YSD1 (Hardy et al., 2020) and P23-45 (Bayfield et al., 2019). In the procapsid, P domains are separated by shorter distances, and N arms are hidden inside the capsomers (Figures 4A and 4B). The E-loop stems form a twisted or curved shape. During maturation, MCPs rotate like “flying swings” (Figure 4A). As a result, the P domains become more separated and the E-loop stems are stretched taut to form a two-stranded β-sheet (Figures 4C and 4D). The N arms from adjacent subunits swing toward the crevice (N arm 2) and complement the E-loop stem β-sheet. During capsid hexon maturation, besides the flying swing rotation, other changes occur involving the pointed tip and A loop of the A domain (Figure 4E). The A loop usually contains an α-helix, as in the procapsid and capsid penton (A loop 1). During capsid hexon maturation, A loop 1 unfurls from its location in the interior (Figure 4E) to attain a fully extended and stable flap secondary structure that fills the centers of the hexons (Figure 4D). The conformational rearrangement of A loop 1 into A loop 2 induces extensive interactions with the A loops of neighboring subunits (Figure 4D). A previous study showed that either R222C or S226F mutations in the A loop of the lambda phage produce empty, expanded heads that cannot attach to tails (Katsura, 1989). The importance of the A loop was also reported for other phages: in the phage P22 coat protein, the A loop could tolerate the addition of three but not four alanines (Morris and Prevelige, 2014); and two intragenic global suppressor substitutions

(C) Mature capsid capsomers. Pentagons and hexagons linking the neighboring Gly312 residues are shown for easier visualization of the capsomers.

(D) Orientations of decoration protein gpD.

Structure
Article

A Flying-swing-like rotation B
Procapsid penton gpE Procapsid hexon gpE

C D
Capsid penton gpE Capsid hexon gpE

E
Putative hexon intermediate
Flying-swing-like rotation
A-loop conformational changes

Figure 4. Intracapsomer interactions
(A) Procapsid penton.
(B) Procapsid hexon.
(C) Capsid penton.
(D) Capsid hexon.
(E) Putative capsid hexon with A loop 1 conformations.

(D163G and T166I) within the A loop of P22 rescued the folding of several temperature-sensitive folding-defective coat proteins (Parent et al., 2007).

Intercapsomer interactions
The intercapsomer interactions are mediated at quasi-/icosahedral- 2- and 3-fold contact points (Figures 5A and 5B). Quasi-icosahedral and icosahedral contact points present similar structures and environments, and quasi-interfaces are discussed hereafter. The buried surface area (BSA) between capsomers increased extensively during the maturation process, from 2,848 Å² to 4,584 Å². No quaternary contact with neighboring capsomers along the 2-fold axis was observed for procapsids (Figure 5C). The quaternary rearrangement of the N arm and S loop is associated with maturation and undergoes an increase in quaternary contacts (Figure 5D). However, a previous study with HK97 maturation showed that 3-fold interactions are crucial for maintaining intercapsomer linkages (Gertsman et al., 2010). At the 3-fold axes, W308 mediates interactions between P loops from three neighboring capsomers to form a stable interaction triangle (Figure 5E). The W308A mu-

tation within the P loop prevents the assembly of gpEs into a procapsid shell, although gpE retains the ability to bind the scaffolding protein, the shell assembly chaperone protein (Lambert et al., 2017). P loop contacts are protected through layer-stacking interactions involving the Ip loop from the bottom and the E loop from the top, which is bound by the la loop and the Is loop (Figure 5F). While individual subunits undergo bending and refolding motions during capsid expansion, P loop sites are exposed upon particle expansion (Figure 5G), which are previously covered by E-loop tips in procapsids. For stabilization, other decoration proteins are required to fulfill the role of the E-loop tip at the top of P loops (Figure 5H).

Decoration role of cementing protein gpD
To strengthen the capsid shell against the internal pressure of DNA packaging, a cementing protein (gpD) attaches to the quasi-icosahedral and icosahedral 3-fold vertices during expansion, stabilizing the mature phage capsid shell (Figure 6A). The gpD monomer has an N-terminal region and a β-tulip domain. The β-tulip domains mediate the trimerization of gpD proteins (Yang et al., 2000). Trimers of the auxiliary protein are arranged

Structure 30, 637–645, April 7, 2022 642

Figure 5. Intercapsomer interactions

(A) Schematic diagram of procapsid capsomer interactions.

(B) Capsomer interactions in capsids. The view is from the exterior of the capsid. The buried surface area between neighboring capsomers is measured in Å².

(C and D) Intercapsomer interfaces along the quasi-2-fold axis in procapsids and capsids.

(E and F) Interfaces on the quasi-3-fold apex in procapsids.

(G and H) Interfaces on the quasi-3-fold apex in capsids. The cryo-EM maps around the P loop trimers were also shown.

and S3). Upon maturation, conformational changes expose the P loops of the capsid structure, which are previously covered by E-loop tips in procapsids. The spike base of the gpD trimer then interacts with the capsid surface, especially the P loop (Figure 6D). The role of the gpD trimer is similar to that of the crosslinks in HK97 (Gertsman et al., 2010).

## DISCUSSION

The MCP gpE shares the common HK97 fold (Suhannovsky and Teschke, 2015) with other capsid proteins from herpesviruses, phages, and some archaeal viruses. HK97 fold proteins are subdivided into the A and P domains. Besides the core domains of the HK97 fold, extra insertion domains are present at distinct positions in many viruses (e.g., Figure 2A). The A domain is triangular and forms the central, wedge-shaped sectors of capsomers, hexons, and pentons, while the P domain forms the periphery of capsomers (Figures 3B and 3C). The relative orientation between the A and P domains of gpE remains invariant during bacteriophage lambda capsid maturation (Figure S2C). During capsid hexon maturation, A loops unfurl from their location in the interior to attain a fully formed and stable flap secondary structure that fills the centers of the hexons (Figures 4B and 4D). N arms from the adjacent neighboring subunit swing into the crevice and complement the E-loop stem β-sheet. The two endpoint high-resolution procapsid and capsid structures determined herein allowed us to propose a model for the transition during capsid maturation, which in vivo is mediated by DNA packaging. The capsid maturation model obtained with the bacteriophage lambda could be applied to other HK97-like phages as well (e.g., thermophages P23-45 or P74-26 and HK97 itself).

The unexpanded procapsid and the expanded, empty, mature capsid of a thermobacteriophage, P23-45 of Thermus thermophilus, were determined to have resolutions of 4.4 Å and 3.7 Å, respectively, by cryo-EM single-particle analysis (Bayfield

at each 3-fold or quasi-3-fold position (Figure 6B). The N-terminal extension of gpD complements the three-stranded β-sheets of the E-loop stem and neighboring N arm features (Figures 6C

Structure
Article

A
![Image Description](image1)

B
![Image Description](image2)
Is-loop  
Ia-loop  
E-loop tip  
P-loop  
Ip-loop  

C
![Image Description](image3)

D
![Image Description](image4)
90°  
gpD trimer  

Figure 6. Decoration roles of gpD proteins
(A) Schematic diagram of gpD proteins.
(B) Top view of interactions.
(C) The stabilizing four-stranded β-sheet.
(D) Side view of (B).

et al., 2019), as well as the DNA-filled capsid of a thermos- stable phage P74-26 at 2.8 Å resolution (Stone et al., 2019). Although the genomes of two thermophages, P23- 45 and P74-26, are unexpectedly twice as large as those of known T = 7 phages, their capsids still have T = 7 quasi-symmetry. Extra-huge DNA packaging triggered a super- sized capsid of thermophages (Figure S4A). In the pro- capsid of thermophage P23-45, the A and P domains have the same relative orientations as those of the bacteriophage lambda, even for the binding site of the neighboring E-loop tip (Figure S4B). However, the A loop in the MCP of the thermophage swings to the interior of the capsomer (named “A loop 3” hereafter). As a result of the A loop 3’s occu- pancy in the capsomer interior, the core domains of the HK97 fold, including the A and P domains, were pushed out of the center, which resulted in a larger procapsid of the thermophage (cyan and red central slices in Figure S4A).

Interestingly, during thermophage maturation, the A loops of the MCPs penton and hexon keep the same conformation, the A loop 3, as that in the procapsid (Figure S4C), as well as for the A domain (Figure S4D). However, the P domain rotated about 16° away rela- tive to the A domain (Figure S4C). The neighboring E-loop tip was accompanied by the P domain (Figure S4E). The A loop 3’s involvement enlarged the edge of the procapsid hexon from 65.2 Å (lambda, Figure 3B) to 76.5 Å (P23-45, Figure S5A). If the A and P domains rotated as a rigid body as in the bacteriophage lambda, the edge of the hexon can merely reach 91.4 Å, taking the effect of the A loop 3 into account (Figure S5B). The P domain’s rotation farther from the A domain in the thermophage MCP contributed to another 7.3 Å at the edge of the capsid hexon (Fig- ure S5C), which resulted in a supersized T-7 capsid. Other capsid maturation features learned from the bacteriophage lambda were also conserved in the thermophages, such as the P loop’s protec- tion by layer stacking involving the Ip loop from the bottom and E

Structure 30, 637–645, April 7, 2022 643

loop from the top, the N arm’s conformational changes during maturation (Figure S5D), and the cementing roles played by the auxiliary protein (gpD or gp88) (Bayfield et al., 2019). The canonical HK97 virus has the smallest capsid volume compared with the bacteriophage lambda and the two thermophages (Figures S4A and S6A) (Helgstrand et al., 2003). The A domains between HK97 and the phage lambda were superimposed reasonably well based on each individual MCP or the hexametric MCPs in the hexon (Figures S6B and S6C). However, the P domain in HK97 rotates approximately 12.5° closer to the A domain (Figure S6C), resulting in a smaller diameter of the capsomers and a smaller overall capsid volume as well. Both A loop 3 insertion into capsomers and P domains leaving away from A domain together contributed to the supersized capsid (Figure S6D).

### Conclusions

Bacteriophage lambda, a prototypic complex dsDNA virus, has served as a model system for interrogation of genetic, biochemical, and structural features during virus development. High-resolution cryo-EM structures of both the procapsid and capsid presented here reveal major conformational changes associated with capsid maturation in a dsDNA bacteriophage. During the maturation process, capsomer MCPs rotate like flying swings. As a result, P domains become more separated and stretch the E-loop stems tautly to form a two-stranded β-sheet. N arms from the adjacent neighboring subunit swing into the crevice and complement the E-loop stem β-sheet. Although the P loop acts as a staple during maturation, the top cover moves away to expose the main structure. A cementing protein attaches to the quasi-icosahedral and icosahedral 3-fold vertices on the top of the P loop to fill in the missing previously covered region. To fasten the cementing protein at the exterior surface of the capsid shell, its N-terminal extension complements the three-stranded β-sheet formed by the E-loop stem and the neighboring N arm. The high-resolution procapsid and capsid structures determined herein allowed us to propose a mechanism for lambda capsid maturation and provide atomic details about how the lambda cementing protein contributes to capsid stability. The procapsid structure also provides a framework for understanding its assembly.

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- **KEY RESOURCES TABLE**
- **RESOURCE AVAILABILITY**
  - Lead contact
  - Materials availability
  - Data and code availability
- **EXPERIMENTAL MODEL AND SUBJECT DETAILS**
- **METHOD DETAILS**
  - Production and purification of lambda virions
  - Cryo-EM data acquisition
  - Cryo-EM image processing
  - Model building and structure refinement
- **QUANTIFICATION AND STATISTICAL ANALYSIS**

---

### SUPPLEMENTAL INFORMATION

Supplemental information can be found online at https://doi.org/10.1016/j.str.2021.12.009.

### ACKNOWLEDGMENTS

We thank Dr. Xiaochun Li (UT Southwestern) and all the reviewers for their insightful comments on this manuscript, Tsinghua University Branch of China National Center for Protein Sciences (Beijing) for cryo-EM facility and computational facility access on the cluster of Bio-Computing Platforms for this study. We also thank J. Lei, X. Li, F. Yang, and J. Wen for their technical assistance. This work was supported by grants from the National Natural Science Foundation of China (grant number 32171190) and the Ministry of Science and Technology of China (grant number 2016YFA0501103).

### AUTHOR CONTRIBUTIONS

C.W., J.Z., and J.W. conceived the project. C.W. and J.Z. designed and performed experiments. All authors analyzed the data and contributed to manuscript preparation. J.W. wrote the manuscript.

### DECLARATION OF INTERESTS

The authors declare no competing interests.

Received: August 20, 2021  
Revised: October 12, 2021  
Accepted: December 15, 2021  
Published: April 7, 2022  

### REFERENCES

Ackermann, H.W. (2007). 5500 Phages examined in the electron microscope. Arch. Virol. 152, 227–243. https://doi.org/10.1007/s00705-006-0849-1.

Afonine, P.V., Poon, B.K., Read, R.J., Sobolev, O.V., Terwilliger, T.C., Urzhumtsev, A., and Adams, P.D. (2018). Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr. D Struct. Biol. 74, 531–544. https://doi.org/10.1107/s2059798318006551.

Bayfield, O.W., Klimuk, E., Winkler, D.C., Hesketh, E.L., Chechik, M., Cheng, N., Dykeman, E.C., Minakhin, L., Ranson, N.A., Severinov, K., et al. (2019). Cryo-EM structure and in vitro DNA packaging of a thermophilic virus with supersized T=7 capsids. Proc. Natl. Acad. Sci. USA 116, 3556–3561. https://doi.org/10.1073/pnas.1813204116.

Campbell, P.L., Duda, R.L., Nassur, J., Conway, J.F., and Huet, A. (2020). Mobile loops and electrostatic interactions maintain the flexible tail tube of bacteriophage lambda. J. Mol. Biol. 432, 384–395.

Cardarelli, L., Pell, L.G., Neudecker, P., Pirani, N., Liu, A., Baker, L.A., Rubinstein, J.L., Maxwell, K.L., and Davidson, A.R. (2010). Phages have adapted the same protein fold to fulfill multiple functions in virion assembly. Proc. Natl. Acad. Sci. USA 107, 14384–14389. https://doi.org/10.1073/pnas.1005822107.

Casjens, S.R. (2011). The DNA-packaging nanomotor of tailed bacteriophages. Nat. Rev. Microbiol. 9, 647–657. https://doi.org/10.1038/nrmicro2632.

Casjens, S.R., and Hendrix, R.W. (2015). Bacteriophage lambda: early pioneer and still relevant. Virology 479-480, 310–330. https://doi.org/10.1016/j.virol.2015.02.010.

Chang, C., Plückthun, A., and Wlodawer, A. (2004). Crystal structure of a truncated version of the phage lambda protein gpD. Proteins 57, 866–868. https://doi.org/10.1002/prot.20254.

Chen, S., McMullan, G., Faruqi, A.R., Murshudov, G.N., Short, J.M., Scheres, S.H., and Henderson, R. (2013). High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. Ultramicroscopy 135, 24–35. https://doi.org/10.1016/j.ultramic.2013.06.004.

DeLano, W.L. (2002). The PyMOL molecular graphics system. http://www.pymol.org.

Dokland, T., and Murialdo, H. (1993). Structural transitions during maturation of bacteriophage lambda capsids. J. Mol. Biol. 233, 682–694. https://doi.org/10.1006/jmbi.1993.1545.

Duda, R.L., and Teschke, C.M. (2019). The amazing HK97 fold: versatile results of modest differences. Curr. Opin. Virol. 36, 9–16. https://doi.org/10.1016/j.coviro.2019.02.001.

Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). Features and development of coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486–501. https://doi.org/10.1107/S0907444910007493.

Fane, B.A., and Prevelige, P.E., Jr. (2003). Mechanism of scaffolding-assisted viral assembly. Adv. Protein Chem. 64, 259–299. https://doi.org/10.1016/s0065-3233(03)01007-6.

Fuller, D.N., Raymer, D.M., Rickgauer, J.P., Robertson, R.M., Catalano, C.E., Anderson, D.L., Grimes, S., and Smith, D.E. (2007). Measurements of single DNA molecule packaging dynamics in bacteriophage lambda reveal high forces, high motor processivity, and capsid transformations. J. Mol. Biol. 373, 1113–1122. https://doi.org/10.1016/j.jmb.2007.09.011.

Gaussier, H., Yang, Q., and Catalano, C.E. (2006). Building a virus from scratch: assembly of an infectious virus using purified components in a rigorously defined biochemical assay system. J. Mol. Biol. 357, 1154–1166. https://doi.org/10.1016/j.jmb.2006.01.013.

Gertsman, I., Komives, E.A., and Johnson, J.E. (2010). HK97 maturation studied by crystallography and H/2H exchange reveals the structural basis for exothermic particle transitions. J. Mol. Biol. 397, 560–574. https://doi.org/10.1016/j.jmb.2010.01.016.

Goetschius, D.J., Lee, H., and Hafenstein, S. (2019). CryoEM reconstruction approaches to resolve asymmetric features. Adv. Virus Res. 105, 73–91. https://doi.org/10.1016/bs.aivir.2019.07.007.

Grant, T., and Grigorieff, N. (2015). Measuring the optimal exposure for single particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6. Elife 4, e06980. https://doi.org/10.7554/eLife.06980.

Hardy, J.M., Dunstan, R.A., Grinter, R., Belousoff, M.J., Wang, J., Pickard, D., Venugopal, H., Dougan, G., Lithgow, T., and Coulibaly, F. (2020). The architecture and stabilisation of flagellotropic tailed bacteriophages. Nat. Commun. 11, 3748. https://doi.org/10.1038/s41467-020-17505-w.

Helgstrand, C., Wikoff, W.R., Duda, R.L., Hendrix, R.W., Johnson, J.E., and Liljas, L. (2003). The refined structure of a protein catenane: the HK97 bacteriophage capsid at 3.44 Å resolution. J. Mol. Biol. 334, 885–899. https://doi.org/10.1016/j.jmb.2003.09.035.

Iwai, H., Forrer, P., Plückthun, A., and Güntert, P. (2005). NMR solution structure of the monomeric form of the bacteriophage lambda capsid stabilizing protein gpD. J. Biomol. NMR 31, 351–356. https://doi.org/10.1007/s10858-005-0945-7.

Katsura, I. (1989). Structure and inherent properties of the bacteriophage lambda head shell. VI. DNA-packaging-defective mutants in the major capsid protein. J. Mol. Biol. 205, 397–405. https://doi.org/10.1016/0022-2836(89)90350-1.

Kucukelbir, A., Sigworth, F.J., and Tagare, H.D. (2014). Quantifying the local resolution of cryo-EM density maps. Nat. Methods 11, 63–65. https://doi.org/10.1038/nmeth.2727.

Lambert, S., Yang, Q., De Angeles, R., Chang, J.R., Ortega, M., Davis, C., and Catalano, C.E. (2017). Molecular dissection of the forces responsible for viral capsid assembly and stabilization by decoration proteins. Biochemistry 56, 767–778. https://doi.org/10.1021/acs.biochem.6b00705.

Lander, G.C., Evilevitch, A., Jeembaeva, M., Potter, C.S., Carragher, B., and Johnson, J.E. (2008). Bacteriophage lambda stabilization by auxiliary protein gpD: timing, location, and mechanism of attachment determined by cryo-EM. Structure 16, 1399–1406.

Medina, E., Wieczorek, D., Medina, E.M., Yang, Q., Feiss, M., and Catalano, C.E. (2010). Assembly and maturation of the bacteriophage lambda procapsid:

gpC is the viral protease. J. Mol. Biol. 401, 813–830. https://doi.org/10.1016/j.jmb.2010.06.060.

Morris, D.S., and Prevelige, P.E. (2014). The role of the coat protein A-domain in p22 bacteriophage maturation. Viruses 6, 2708–2722. https://doi.org/10.3390/v6072708.

Parent, K.N., Suhanovsky, M.M., and Teschke, C.M. (2007). Polyhead formation in phage P22 pinpoints a region in coat protein required for conformational switching. Mol. Microbiol. 65, 1300–1310. https://doi.org/10.1111/j.1365-2958.2007.05868.x.

Parvez, M.K. (2020). Geometric architecture of viruses. World J. Virol. 9, 5–18. https://doi.org/10.5501/wjv.v9.i2.5.

Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612. https://doi.org/10.1002/jcc.20084.

Rosenthal, P.B., and Henderson, R. (2003). Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. J. Mol. Biol. 333, 721–745. https://doi.org/10.1016/j.jmb.2003.07.013.

Rossmann, M.G., Bernal, R., and Pletnev, S.V. (2001). Combining electron microscopic with x-ray crystallographic structures. J. Struct. Biol. 136, 190–200. https://doi.org/10.1006/jsbi.2002.4435.

Sambrook, J., Russell, D.W., Sambrook, J., and Russell, D.W. (2001). Molecular Cloning: A Laboratory Manual (Cold Spring Harbor Laboratory Press).

Scheres, S.H. (2016). Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579, 125–157. https://doi.org/10.1016/bs.mie.2016.04.012.

Singh, P., Nakatani, E., Goodlett, D.R., and Catalano, C.E. (2013). A pseudo-atomic model for the capsid shell of bacteriophage lambda using chemical cross-linking/mass spectrometry and molecular modeling. J. Mol. Biol. 425, 3378–3388. https://doi.org/10.1016/j.jmb.2013.06.026.

Stone, N.P., Demo, G., Agnello, E., and Kelch, B.A. (2019). Principles for enhancing virus capsid capacity and stability from a thermophilic virus capsid structure. Nat. Commun. 10, 4471. https://doi.org/10.1038/s41467-019-12341-z.

Su, R., Zhang, S.Y., and Wang, J.W. (2019). Automatically generating mask for mapping out the density map in cryoEM single-particle reconstruction. Prog. Biophys. 46, 1020–1030. https://doi.org/10.16476/j.pibb.2019.0102.

Suhanovsky, M.M., and Teschke, C.M. (2015). Nature’s favorite building block: deciphering folding and capsid assembly of proteins with the HK97-fold. Virology 479–480, 487–497. https://doi.org/10.1016/j.virol.2015.02.055.

Yang, F., Forrer, P., Dauter, Z., Conway, J.F., Cheng, N., Cerritelli, M.E., Steven, A.C., Plückthun, A., and Wlodawer, A. (2000). Novel fold and capsid-binding properties of the lambda-phage display platform protein gpD. Nat. Struct. Biol. 7, 230–237. https://doi.org/10.1038/73347.

Zhang, K. (2016). Gctf: real-time CTF determination and correction. J. Struct. Biol. 193, 1–12. https://doi.org/10.1016/j.jsb.2015.11.003.

Zheng, S.Q., Palovcak, E., Armache, J.P., Verba, K.A., Cheng, Y., and Agard, D.A. (2017). MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14, 331–332. https://doi.org/10.1038/nmeth.4193.

Zhou, N., Wang, H., and Wang, J. (2017). EMBuilder: a template matching-based automatic model-building program for high-resolution cryo-electron microscopy maps. Sci. Rep. 7, 2664. https://doi.org/10.1038/s41598-017-02725-w.

Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. Elife 7. https://doi.org/10.7554/eLife.42166.

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| Bacterial and virus strains |  |  |
| Bacteriophage lambda | ATCC | https://www.atcc.org/products/97537 |
| E.coli C600 | ATCC | N/A |
| Chemicals, peptides, and recombinant proteins |  |  |
| Luria Bertani (LB) medium | BD | Cat# 292438 |
| CsCl | Sigma-Aldrich | Cat# 7641-17-8 |
| Poly-L-lysine | Sigma-Aldrich | Lot# SLCC 5591 |
| Deposited data |  |  |
| Atomic coordinates of icosahedral procapsid | This paper | PDB: 7VI9 |
| Cryo-EM map of icosahedral procapsid | This paper | EMD-32004 |
| Atomic coordinates of procapsid ASU | This paper | PDB: 7VIA |
| Cryo-EM map of procapsid ASU | This paper | EMD-32005 |
| Atomic coordinates of icosahedral capsid | This paper | PDB: 7VII |
| Cryo-EM map of icosahedral capsid | This paper | EMD-32010 |
| Atomic coordinates of capsid ASU | This paper | PDB: 7VIK |
| Cryo-EM map of capsid ASU | This paper | EMD-32011 |
| Software and algorithms |  |  |
| MotionCorr 2 | (Zheng et al., 2017) | https://emcore.ucsf.edu/ucsf-software |
| Gctf | (Zhang, 2016) | https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/zhang-software/ |
| Gautomatch | N/A | https://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/ |
| Relion-3.0 | (Zivanov et al., 2018) | http://www2.mrc-lmb.cam.ac.uk/relion; RRID: SCR_016274 |
| EM AutoMask | (Su et al., 2019) | https://bitbucket.org/Rui-Su/relion-1.4-research/src/master/src/ |
| ResMap | (Kucukelbir et al., 2014) | https://github.com/akucukelbir/resmap |
| EMBuilder | (Zhou et al., 2017) | https://github.com/NiyunZhou/EMBuilder |
| Coot | (Emsley et al., 2010) | https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/; RRID: SCR_014222 |
| Phenix | (Afonine et al., 2018) | https://www.phenix-online.org; RRID: SCR_014224 |
| EMfit | (Rossmann et al., 2001) | https://bilbo.bio.purdue.edu/~viruswww/Rossmann_home |
| PyMol | (DeLano, 2002) | http://www.pymol.org; RRID: SCR_000305 |
| UCSF Chimera | (Pettersen et al., 2004) | https://www.cgl.ucsf.edu/chimera; RRID: SCR_004097 |
| Other |  |  |
| R 1.2/1.3 plus C2 on 200 copper mesh grids | Quantifoil | X-101-Cu200C2 |

RESOURCE AVAILABILITY

Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Jiawei Wang ([jwwang@tsinghua.edu.cn](http://jwwang@tsinghua.edu.cn)).

e1 Structure 30, 637–645.e1–e3, April 7, 2022

Structure
Article

**Materials availability**

All unique reagents generated in this study are available from the Lead Contact.

**Data and code availability**

- Data have been deposited at PDB and EMDB and are publicly available as of the date of publication. Accession numbers are listed in the key resources table.
- This paper does not report original code.
- Any additional information required to reanalyse the data reported in this paper is available from the lead contact upon request.

**EXPERIMENTAL MODEL AND SUBJECT DETAILS**

The bacteriophage lambda viral particle infection was performed in *Escherichia coli* c600, maintained in LB medium at 37°C.

**METHOD DETAILS**

**Production and purification of lambda virions**

We isolated and purified bacteriophage lambda from an infected culture of *Escherichia coli* c600. A 200 ml volume of Luria Bertani (LB) medium was inoculated with 6 ml of an overnight (~15 h) culture of *Escherichia coli* c600 and grown for approximately 1.5 h at 37°C and 220 rpm, until the OD600 of 0.55 had been reached. The culture was infected with high concentration bacteriophage lambda and the culture had been lysed after 3 h. Non-lysed cells and cell debris were removed by centrifugation at 4°C (12,000 g, 10 min). The clarified supernatant was filtered through a 0.22 μM microfiltration membrane. Lambda phage particles were harvested by centrifugation at 4°C (150,000 g, 2 h), gently resuspended in 2 ml SM buffer (10 mM NaCl/10 mM MgSO₄/50 mM Tris-HCl, pH 7.4) and incubated at 4°C overnight. The buffer with phage particles were added 1 g CsCl and were loaded onto a CsCl gradient at densities of 1.45, 1.5, 1.7 g/ml in SM buffer. Gradients were centrifuged with SW41 rotor at 4°C at (180,000 g, 16 h) ([Sambrook et al., 2001](https://doi.org/10.1038/nmeth.1659)). Two phage bands could be obtained. One was at the interface of cesium chloride ρ = 1.15 g/ml and ρ = 1.45 g/ml which contained procapsid and mature capsid. Another was at the interface of cesium chloride ρ = 1.45 g/ml and ρ = 1.5 g/ml which contained mostly mature capsid with DNA. We carefully removed the phage band at the interface of caesium chloride ρ = 1.15 g/ml and ρ = 1.45 g/ml to obtain procapsid and mature capsid, then dialysed the phage particles into SM buffer.

**Cryo-EM data acquisition**

Aliquots (4 μL) of concentrated phage particles were loaded onto holey carbon grids (Quantifoil Cu R1.2/1.3, 200 mesh) pretreated with polylysine, blotted for 5–10 s, and plunge-frozen in liquid ethane cooled by liquid nitrogen using a Vitrobot Mark IV instrument (Thermo Fisher) at 8°C and with 100% humidity. Grid sample quality was verified with a Tecnai Arctica 200-kV electron microscope equipped with a Falcon III camera (Thermo Fisher). Verified grids with optimal ice thickness and particle density were transferred to a Titan Krios electron microscope (Thermo Fisher) operating at 300 kV. Micrographs were recorded using a K2 Summit camera (Gatan Company) in super-resolution mode with a nominal magnification of 105,000×, resulting in a calibrated pixel size of 0.65327 Å. Each stack contains 32 frames, and received a total accumulated dose of 50 e⁻ / Å². All 32 frames in each stack were aligned and summed using the motion correction program MotionCor2 ([Zheng et al., 2017](https://doi.org/10.1038/s41592-017-0021-5)), binned to a pixel size of 1.30654 Å, and dose weighting was also performed ([Grant and Grigorieff, 2015](https://doi.org/10.1038/s41592-017-0021-5)). The defocus values were set from –1.0 to –2.0 μm and were estimated by Gctf ([Zhang, 2016](https://doi.org/10.1038/s41592-017-0021-5)).

**Cryo-EM image processing**

All particles were automatically picked using Gautomatch ([https://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/](https://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/) and REION-3.0 ([Zivanov et al., 2018](https://doi.org/10.1038/s41592-017-0021-5)) from 7,766 micrographs, followed by manual checking. All subsequent 2D and 3D classifications and refinements were performed using RELION-3.0 and EMAutoMask ([Su et al., 2019](https://doi.org/10.1038/s41592-017-0021-5)). Multiple rounds of reference-free 2D classification were performed to remove ice spots, contaminants and aggregates, yielding 182,568 and 95,685 good particles for procapsid and capsid, respectively. The particles were re-extracted with a bin2 box size of 360, and processed through 3D classification and 3D auto-refinement procedures. A total of 174,346 and 88,783 good particles for procapsid and capsid were re-extracted with a bin1 box size of 720, followed by local contrast transfer function (CTF) refinement with Gctf ([Zhang, 2016](https://doi.org/10.1038/s41592-017-0021-5)), which yielded reconstructions with I2 symmetry at resolutions of 5.03 Å and 5.60 Å for procapsid and capsid, respectively.

To improve the resolution of the final reconstructions, both particle datasets were expanded with I2 symmetry, which resulted in 10,460,760 and 5,326,980 particles for the procapsid ASU and capsid asymmetric unit (ASU), respectively. Local masks of one icosahedral ASU were applied during multiple rounds of 3D local classification which resulted in 1,310,146 and 835,912 particles for the procapsid ASU and capsid ASU, respectively, yielding final ASU reconstructions at improved resolutions of 3.88 Å and 3.76 Å.

Reported resolutions were calculated on the basis of the FSC 0.143 criterion ([Rosenthal and Henderson, 2003](https://doi.org/10.1038/s41592-017-0021-5)) through a high-resolution noise substitution method ([Chen et al., 2013](https://doi.org/10.1038/s41592-017-0021-5)). Prior to visualisation, all density maps were corrected for the modulation transfer function (MTF) of the detector and sharpened by applying a negative B-factor that was estimated using automated procedures. Local resolution variations were estimated using ResMap ([Kucukelbir et al., 2014](https://doi.org/10.1038/s41592-017-0021-5)).

Model building and structure refinement
The high-resolution ASU cryo-EM maps of procapsid and capsid were used for *de novo* model building in EMBuilder ([Zhou et al., 2017](https://doi.org/10.1038/s41592-017-0006-0)) and adjusted manually in COOT ([Emsley et al., 2010](https://doi.org/10.1107/S090744491000949X)). Structure refinements were carried out by PHENIX in real space with secondary structure and geometry restraints to prevent structure overfitting ([Afonine et al., 2018](https://doi.org/10.1107/S2059798318004603)). The structures of whole phage lambda procapsid and capsid were generated with EMfit ([Rossmann et al., 2001](https://doi.org/10.1016/S0022-2836(01)94140-9)) by applying icosahedral symmetry. Statistics for 3D reconstruction and model refinement can be found in [Table S1](#table-s1). All structure figures were prepared using PyMol ([DeLano, 2002](https://www.pymol.org/)) and UCSF Chimera ([Pettersen et al., 2004](https://doi.org/10.1002/jcc.20084)).

QUANTIFICATION AND STATISTICAL ANALYSIS

No statistical analysis was performed in this work.
