
Biochem. J. (1995) 309, 999–1007 (Printed in Great Britain)

# The biosynthesis of threonine by mammalian cells: expression of a complete bacterial biosynthetic pathway in an animal cell

William D. REES and Susan M. HAY  
The Rowett Research Institute, Greenburn Road, Bucksburn, Aberdeen AB2 9SB, Scotland, U.K.

The coding regions for the *Escherichia coli* gene for aspartokinase I/homoserine dehydrogenase I (*thrA*) and the *Corynebacterium glutamicum* gene for aspartic semialdehyde dehydrogenase (*asd*) have been subcloned into a Simian Virus 40 (SV40)-based mammalian expression vector. Both enzyme activities are expressed in mouse 3T3 cells after transfer of the corresponding chimaeric gene. The kinetic parameters are similar to those of the native bacterial enzymes, and aspartokinase I/homoserine dehydrogenase I retains its allosteric regulation by threonine. An extract of the cells expressing aspartokinase I/homoserine dehydrogenase I, mixed with one from cells expressing aspartic semialdehyde dehydrogenase, produced homoserine when the mixture was incubated with aspartic acid, ATP and NADPH. The *thrA* and *asd* expression cassettes were combined into a single plasmid which, when transfected into 3T3 cells, enabled them to produce homoserine from aspartic acid. Homoserine-producing 3T3 cells were transfected with the plasmid pSVthrB/C (homoserine kinase and threonine synthase) and selected for growth on homoserine. Cell lines isolated from these cells expressed the complete bacterial threonine pathway, were independent of threonine for growth and could be maintained in medium which contained no free threonine. The threonine in the proteins of these cells became enriched in ${}^{15}\mathrm{N}$ when the culture medium contained [${}^{15}\mathrm{N}$]aspartic acid. The production of homoserine and the growth of cells was at a maximum when there was more than 2.5 mM aspartate in the medium. Below this concentration the high $K_{\mathrm{m}}$ of aspartokinase limited the flux through the pathway. In the presence of additional aspartic acid the new pathway could sustain a cell cycle time close to that of the same cells cultured in threonine-containing medium.

## INTRODUCTION

Higher animals lack the enzymes required for the biosynthesis of the essential amino acids. The nutrition of non-ruminants fed diets based on cereal proteins can be significantly improved by feeding rations supplemented with synthetic amino acids, particularly lysine and threonine. The introduction of pathways for the biosynthesis of these limiting amino acids into animal cells would eliminate the need for supplementation, and we have previously discussed the possibility of producing transgenic animals capable of synthesizing these amino acids [1].

In bacteria, lysine, threonine and methionine are produced from aspartic acid. The five-step pathway for threonine synthesis is shown in Scheme 1. Aspartic acid is phosphorylated by aspartokinase (EC 2.7.2.4) to produce L-aspartyl $\beta$-phosphate, which is reduced to L-aspartic $\beta$-semialdehyde by aspartic semialdehyde dehydrogenase (EC 1.2.1.11). The semialdehyde is then further reduced by homoserine dehydrogenase (EC 1.1.1.3) to the non-protein amino acid homoserine. Homoserine kinase (EC 2.7.1.39) phosphorylates homoserine to produce homoserine O-phosphate, which is converted to threonine by threonine synthase (EC 4.2.99.2). Lysine and methionine are produced from branches in this pathway using aspartic semialdehyde and homoserine as the respective precursors.

By producing chimaeric genes with a suitable eukaryotic promoter attached to the coding region of a bacterial gene, it is possible to express bacterial genes in animal cells. We have used this approach to introduce the *Escherichia coli thrB* (homoserine kinase) and *thrC* (threonine synthase) genes into mouse 3T3 cells [2,3]. The bacterial genes were subcloned into a Simian Virus 40 (SV40)-based expression vector and then transfected into mouse 3T3 cells. Cells which had been transfected with both genes and selected for growth in threonine-free homoserine-containing medium were able to produce their own supply of threonine from homoserine.

To make animal cells truly threonine-independent requires the three remaining enzyme activities which synthesize homoserine from aspartic acid, and in this paper we report the expression of the *E. coli thrA* and *Corynebacterium glutamicum asd* genes in mouse 3T3 cells. *thrA* codes for one of the three aspartokinase isoforms found in *E. coli*, each of which is feedback-inhibited by one of the amino acid products. Aspartokinase I is sensitive to threonine, aspartokinase II to methionine and aspartokinase III to lysine. Aspartokinase is also unusual in another respect, as isoforms I and II are bifunctional enzymes which can also catalyse the third reaction of the pathway, the reduction of aspartic semialdehyde to homoserine (homoserine dehydrogenase). The gene coding for the threonine-sensitive aspartokinase I/homoserine dehydrogenase I enzyme, *thrA*, is the first gene of the *E. coli* threonine operon [4,5] and has a 2.3 kbp open reading frame [6]. The bifunctional 360 kDa protein comprises four identical subunits of 86 kDa [7].

The other enzyme required for homoserine synthesis from aspartic acid is aspartic semialdehyde dehydrogenase. The reaction catalysed by this enzyme is common to the pathways of all of the amino acids produced from aspartic acid and the gene is located separately in the *E. coli* chromosome. This is not the case in Gram-positive bacteria, where the gene for aspartic semialdehyde dehydrogenase (*asd*) is found with the lysine pathway genes in the chromosome of *C. glutamicum* [8,9]. The enzyme is

Abbreviations used: DMEM, Dulbecco’s modified Eagle’s medium; SV40, Simian Virus 40; G-418, Geneticin [O-2-amino-2,7-dideoxy-$\alpha$-D-glucopyranosyl(1-4)-O-3-deoxy-$C_4$-methyl-3-(methylamino)-$\beta$-L-arabinopyranosyl-D-streptamine]; APE, atom percent excess.

Aspartic acid

ATP
↓
Aspartokinase I
(thrA)
↓
ADP

Aspartyl phosphate

NADPH
↓
Aspartic semialdehyde
dehydrogenase
(asd)
↓
NADP⁺ + Pᵢ

Aspartic β-semialdehyde

NADPH
↓
Homoserine
dehydrogenase
(thrA)
↓
NADP⁺

Homoserine

ATP
↓
Homoserine kinase
(thrB)
↓
ADP

Homoserine O-phosphate

Pi
↓
Threonine synthase
(thrC)
↓

Threonine

---

**Scheme 1 The E. coli threonine pathway**

a dimer composed of two subunits with a molecular mass of 38 kDa and catalyses the synthesis of aspartic β-semialdehyde by the reductive dephosphorylation of aspartyl β-phosphate using NADPH.

In this paper we describe the engineering of a mouse cell which can synthesize the amino acid threonine and is thus independent of what was formerly an essential nutrient. Cells have been transfected with chimaeric thrA and asd genes and a line which is able to produce homoserine from aspartic acid has been isolated. Further transfection of this line with similar constructs expressing the thrB and thrC genes has completed the metabolic pathway enabling the cells to convert homoserine into threonine.

---

**MATERIALS AND METHODS**

**General**

The plasmid pIP3 [5] was a gift from Dr. I. Saint-Girons, Institut Pasteur, Paris, France. The plasmid pSVpoly [10] was a gift from Dr. A. Stacey, Colorado State University, Fort Collins, CO, U.S.A. pCS2 was a gift from Professor A. Puhler, Universitat Bielefeld, Germany. Recombinant DNA methods were as described by Maniatis et al. [11]. Mouse 3T3 cells (ICN-Flow) were cultured in Dulbecco’s modified Eagle’s medium (DMEM) (Gibco) containing 12 % (v/v) newborn calf serum (ICN-Flow) in a 5 % CO₂ atmosphere. Cells were transfected by the calcium phosphate method described previously [3]. The preparation of RNA and DNA for Northern and Southern analysis have been described previously [12].

**Amplification of DNA by PCR**

The primers were, for the positive strand, 5′-GGTACCATGCGAGTGTTCGAAGTTCCGGCGGTAC-3′, and for the negative strand 5′-ATACGGCGGGGTGGACTCGGCAA-3′. The PCR reaction was carried out in a 100 μl volume of buffer containing 0.2 μmol of primers, 0.2 mM dNTPs, 100 ng of template DNA and 1 unit of Taq DNA polymerase (Boehringer). Amplification was carried out for 25 cycles comprising annealing at 55 °C for 3 min, extension at 72 °C for 2 min 30 s and denaturation at 94 °C for 1 min 30 s. After subcloning, the product was checked by sequencing both strands with an ABI 373A automatic DNA sequencer using the Taq-Dye-deoxy terminator kit (ABI).

**Preparation of cell extracts for enzyme assays**

Mouse 3T3 cells were grown to confluence in 90 mm Petri dishes. The cells were washed with PBS, scraped into 0.5 ml of PBS with a rubber policeman and harvested by centrifugation at 10000 g for 1 min. The PBS was removed and the cells resuspended in assay buffer. For the estimation of aspartokinase this comprised 100 mM KCl, 20 mM Tes (pH 7.4) and 0.1 mM EDTA. For the estimation of dehydrogenase activity the buffer contained 50 mM Tris/Mops (pH 9.0) and 200 mM KCl. The cells were lysed by sonication for 15 s at 60 W with a 0.1 cm probe. The extract was then centrifuged at 100000 g for 15 min and the supernatant was used for the assay.

**Aspartokinase assay**

Aspartokinase was measured by the method of Black and Wright [13] as modified by Shaul and Galili [14]. The assay was carried out in 0.25 ml containing about 400 μg of cell protein, 100 mM Tris/HCl (pH 8.0), 10 mM ATP, 0.5 mM MgCl₂, 1 M hydroxylamine (adjusted to pH 7.0) and 30 mM aspartic acid. After incubation at 37 °C for 1 h the reaction was terminated by the addition of 0.25 ml of a 1:1:1 mixture of 10 % FeCl₃ in 0.1 M HCl, 3 M HCl and 12 % (w/v) trichloroacetic acid (1:1:1, by vol.). The mixture was placed on ice for 5 min and then centrifuged at 10000 g for 5 min. The absorbance of the supernatant was read at 490 nm. Blanks were estimated by omitting the aspartic acid. The progress curve for the reaction remained linear for 2 h. The molar absorption coefficient for the hydroxamate was 6.3 × 10⁵ M⁻¹·cm⁻¹ [15].

**Homoserine dehydrogenase assay**

Homoserine dehydrogenase was assayed in the reverse direction by following the reduction of NADP⁺ by homoserine and estimating the amount of NADPH produced using fluorimetry. The reaction was carried out in a 30 μl volume containing 50 mM Mops/Tris (pH 7.5), 200 mM KCl, 25 μl of extract, 2.5 μl of 2.5 mM NADP⁺ and 2.5 μl of 120 mM homoserine [16]. The reaction was incubated at 37 °C for 30 min and then stopped by the addition of 3 ml of cold 10 mM sodium phosphate buffer (pH 7.4). The fluorescence of the sample was read at 460 nm with an excitation wavelength of 340 nm. The fluorimeter was calibrated with a standard NADPH solution. The progress curve remained linear for 60 min, and there was a linear relationship between the enzyme activity and the amount of extract added in the range 20–100 μg of protein.

**Aspartic semialdehyde dehydrogenase assay**

Aspartic semialdehyde dehydrogenase activity was measured by following the L-aspartic β-semialdehyde-dependent reduction of NADP⁺ by fluorimetry. L-Aspartic β-semialdehyde was prepared by the ozonolysis of L-allylglycine (Sigma) [17]. For the assay, 2.5 μl of 2.5 mM NADP⁺ and 5 μl of sodium arsenate (400 mM, pH 9.0) was added to 25 μl of cell extract. The reaction was

Figure 1 Preparation of the plasmids pSVthrA and pSVasd

A 3.8 kbp fragment containing the *E. coli thrA* and *thrB* genes was isolated from pLP3, partially digested with *Aval* and subcloned into pSVpoly. A 0.65 kbp portion of the 5′ end of the *thrA* coding region was synthesized by PCR and used to replace an *Xbal/PvuI* fragment of the 5′ end of pSVthrA1. A 1.03 kbp *DraI/XhoI* fragment containing the *C. glutamicum asd* gene was isolated from pCS2 and subcloned into pSVpoly to produce pSVasd. The SV40/*thrA* expression cassette was removed from pSVthrA2 by digestion with *SalI* and *NotI*, attached to a *NotI/SalI* linker (sequence 5′-TCGACGGCCGCG-3′) and subcloned into the *NotI* site of pSVasd. Abbreviations: SV40, SV40 early promoter; P.A., SV40 polyadenylation site; *amp*, ampicillin-resistance gene; ori, origin of replication. The diagrams are not drawn to scale.

started by the addition of 2.5 μl of 30 mM L-aspartic β-semialdehyde (neutralized immediately before use). The reaction was incubated at 37 °C for 30 min and stopped by the addition of 3 ml of cold 10 mM sodium phosphate buffer (pH 7.4). The fluorescence of the sample was read at 460 nm following excitation at 340 nm. The background fluorescence was determined in samples from which aspartic semialdehyde had been omitted. The fluorimeter was calibrated with a standard NADPH solution.

The progress curve remained for 60 min, and there was a linear relationship between the enzyme activity and the amount of extract added in the rate 20–100 μg of protein.

### Amino acid analysis

Cell extracts were prepared in the same way as described for the enzyme assays. The cell extracts were incubated overnight at

1002 W. D. Rees and S. M. Hay

![Figure 2](#fig2)

**Figure 2** Kinetics of aspartokinase I/homoserine dehydrogenase I in 3T3 A-neo-15 cells

(a) Relationship between aspartokinase I rate and aspartic acid concentration. The calculated \( K_m \) is 1.7 mM and the \( V_{\text{max}} \) is 2.55 nmol/min per mg of protein. (b) Allosteric inhibition of aspartokinase I with threonine. The aspartic acid concentration was 30 mM and the uninhibited reaction rate was 2.96 nmol/min per mg of protein. (c) Relationship between homoserine concentration and rate for homoserine dehydrogenase. The calculated \( K_m \) is 10.5 mM and the \( V_{\text{max}} \) is 0.265 nmol/min per mg of protein. (d) Allosteric inhibition of homoserine dehydrogenase I with threonine. The homoserine concentration was 10 mM and the uninhibited rate was 0.249 nmol/min per mg of protein. All of the data are the averages of three experiments with duplicate estimations for each point. The lines are fitted by linear regression and the error bars in (a) and (c) represent the S.E.M.

---

**GC–MS analysis**

Cells were cultured in threonine-free medium containing L-[¹⁵N]aspartic acid (Sigma; 98 % enrichment) diluted to give a final enrichment of 10 % atom percent excess (APE) and a final concentration of 25 mM. After 4 days in culture, the cells were washed with PBS and the proteins harvested in 0.3 M NaOH. The cell protein was hydrolysed by heating in 6 M HCl at 100 °C overnight. The samples were freeze-dried before preparing the n-butyl heptafluorobutyryl derivatives of the amino acids. The derivatized amino acids were analysed on a Hewlett Packard 5989A GC–MS instrument (Hewlett Packard, Manchester, U.K.) and the ¹⁵N enrichment was calculated from data generated under selective ion recording conditions.

---

**Growth curves**

Between 0.8 × 10³ and 1.2 × 10³ cells were plated into each well of a 96-well plate and cultured in 50 μl of medium which was changed every 2 days. At appropriate times, plates were harvested by adding 1 % glutaraldehyde in PBS, incubating at room temperature for 20 min and then washing twice with 0.2 ml of PBS [20]. The cells were stained with 0.1 % Crystal Violet, washed extensively and the \( A_{620} \) of each well was read using a multiwell plate reader.

---

**RESULTS**

**Plasmid preparation**

Subcloning of the *thrA* gene was complicated by the lack of suitable restriction sites immediately adjacent to the coding region. Initially a 3.8 kbp fragment containing the *E. coli thrA*

thrO gene, which in the bacterium controls the expression of the other genes in the polycistronic mRNA by forming a series of hairpin loops which can prevent ribosomal initiation at the thrA gene [21]. One of these loops would have remained in the 5' end of the chimaeric RNA produced in mammalian cells and may have interfered with the initiation of the mammalian ribosome. In order to delete the remains of thrO, a 0.65 kbp portion of the 5' end of the coding region was synthesized by PCR using a 5' primer modified to include an XbaI site immediately upstream of the start codon and a 3' primer corresponding to a region just downstream of a PvuI site within the coding region. An XbaI/PvuI fragment was removed from pSVthrA1 and replaced with the PCR product. One of the resulting plasmids, pSVthrA2, was sequenced in both strands to confirm that the anticipated modification had been carried out. The sequence of the segment of the coding region which had been replaced matched the sequence of the template exactly and confirmed the deletion of the potential hairpin loop.

pSVasd was produced by subcloning the C. glutamicum asd gene isolated from pCS2 [8] into the vector pSVpoly. A 1.03 kbp fragment which contained the complete asd gene coding region was isolated by digestion of pCS2 with DraI and XhoI, end-filled and ligated into the SmaI site of pSVpoly. The recombinant plasmids were recovered from E. coli DH5α and screened by restriction mapping to identify one with the insert in the correct orientation relative to the SV40 promoter.

The two chimaeric genes were combined into a single construct by excising the SV40/thrA expression cassette from pSVthrA2 by digestion with SalI and NotI. The purified fragment was ligated to a NotI/SalI linker, digested with NotI and then ligated into a NotI site in pSVasd. Restriction analysis of the resulting recombinant plasmids showed that approximately half of them had the anticipated restriction map with the SV40/thrA and SV40/asd genes in the orientation shown in Figure 1. There were an equal number of plasmids which were slightly shorter than the original pSVasd, and we believe that these were produced by the SV40/thrA cassette inserting in the opposite orientation to produce an unstable construct which subsequently rearranged.

### Production of stable cell lines

Stable 3T3 cell clones which had incorporated the chimaeric plasmids were produced by co-transfection of the plasmid along with a plasmid coding for neomycin resistance (pMC1neoPA) and selection for resistance to the antibiotic G-418 (Geneticin). Representative resistant colonies were isolated and screened for the presence of mRNAs corresponding to either thrA or asd. Cell lines A-neo-15 (pSVthrA2), asd-neo-6 (pSVasd) and A/asd-neo-13 (pSVthrA/asd) were chosen, as each expressed high levels of the appropriate mRNA.

### Characterization of aspartokinase I/homoserine dehydrogenase I and aspartic semialdehyde kinetics

When extracts of 3T3 A-neo-15 cells were assayed for aspartokinase activity a red colour with the characteristic absorption at 490 nm was produced by extracts of the transfected cells. The reaction was time- and concentration-dependent, and Figure 2(a) shows a secondary plot of the rate of aspartokinase activity with increasing concentrations of aspartic acid. The calculated K_m for aspartic acid from these experiments is 1.77 mM, close to the value of 1.5 mM for the enzyme purified from E. coli [22]. The allosteric regulation of aspartokinase I by threonine is also retained, and high concentrations of threonine decrease the maximum reaction rate to about 20% of the uninhibited rate, with a K_i of approx. 0.5 mM (Figure 2b).

and thrB genes was isolated from the plasmid pIP3 by digestion with EcoRI. This fragment was purified by agarose gel electrophoresis and then subjected to a partial digestion with AvaI. The products of this digest were then separated on another agarose gel and a fragment of 2.65 kbp was isolated. This product was end-filled with dNTPs, ligated into the EcoRV site of the mammalian expression vector pSVpoly [10] and the ligation mixture was used to transform E. coli DH5α. The resulting plasmids were analysed by restriction mapping in order to isolate pSVthrA1, which has the insert in the correct orientation relative to the SV40 promoter. This removed all but 65 bp of the thrB gene at the 3' end of the coding sequence. However, the non-coding DNA at the 5' end of this construct contained part of the

---

**Figure 3** Production of homoserine by cell-free extracts of 3T3 A/asd-neo-13 cells

Elution profiles from HPLC chromatography of the phenyl isothiocyanate derivatives of the free amino acids. (a) Extract of control 3T3 neo cells; (b) the same extract with 200 pmol of homoserine added prior to derivatization; (c) extract of 3T3 A/asd-neo-13 cells.

---

**Table 1** Intracellular content of aspartic acid and homoserine in 3T3 cell lines cultured in medium containing different concentrations of aspartic acid

The cells were cultured for 3 days in DMEM supplemented with aspartic acid. The results are the averages of two experiments with duplicate observations for each experiment.

| Cell line       | Aspartic acid in medium (mM) | Intracellular amino acid content (pmol/mg of protein) |
|-----------------|-------------------------------|-----------------------------------------------------|
|                 |                               | Aspartic acid                                      | Homoserine                                      |
| 3T3 neo        | 0.4                           | 0.29                                               | 0.64                                            |
|                 | 5.0                           | 1.23                                               | 0.30                                            |
|                 | 25.0                          | 2.60                                               | 0.37                                            |
| 3T3 A/asd-neo-13 | 0.4                           | 0.14                                               | 0.43                                            |
|                 | 5.0                           | 0.60                                               | 0.82                                            |
|                 | 25.0                          | 2.92                                               | 1.58                                            |

---

1004 W. D. Rees and S. M. Hay

(a)

![Graph](https://i.imgur.com/1234567.png)

(b)

![Graph](https://i.imgur.com/8901234.png)

Figure 4 (a) Growth curves for A/asd-neo-13 and A/asd-B/C cell lines grown in threonine-free medium and (b) growth of A/asd-B/C clone 6 on threonine-deficient medium containing aspartic acid

(a) Cells were grown in threonine-free medium containing 25 mM aspartic acid (●) or in threonine-free medium with 2 mM threonine added (○). (b) The exponential rate constant (*k*) was obtained by fitting an exponential curve to growth data (six time points). The error bars represent the S.E.M.

3T3 A-neo-15 cells also expressed the other function of the *thrA* gene product, catalysing the homoserine-dependent reduction of NADP⁺. The *K*ₘ for homoserine in the reverse reaction (Figure 2c) was 10.5 mM and is very similar to the value obtained with purified *E. coli* enzyme [23]. The homoserine dehydrogenase activity is also subject to allosteric inhibition by threonine (Figure 2d).

Extracts of the cell lines expressing pSVasd were assayed for aspartic semialdehyde dehydrogenase activity by measuring the aspartic semialdehyde-dependent reduction of NADP⁺. The reaction was time- and concentration-dependent, and kinetic analysis showed that the *K*ₘ for aspartic semialdehyde was 0.35 mM. This is very similar to the value of 0.2 mM for the enzyme purified from *E. coli* [24].

### Synthesis of homoserine by cell extracts

The production of homoserine by cell extracts was measured by HPLC of the phenyl isothiocyanate derivative. An authentic homoserine standard (200 pmol) emerged from the HPLC column at 13.7 min (Figure 3b) and corresponded with a peak of homoserine (397 pmol) found after extracts of 3T3 A/asd-neo-13 cells had been incubated with aspartic acid, ATP and NADPH (Figure 3c). A mixture of extracts from 3T3 A-neo-15 and 3T3 asd-neo-6 cells produced a similar result (results not shown). There was a small amount of homoserine (about 50 pmol) in the control 3T3 neo extract (Figure 3a), but since this was present before and after incubation we believe it to be a contaminant, possibly originating from traces of homoserine adsorbed on to glassware and plastics [25]. Homoserine concentrations remained

at this background level when extracts of 3T3 A-neo-15 or 3T3 asd-neo-65 cells were used and the production of homoserine was dependent on the presence of ATP, NADPH and aspartic acid in the reaction mixture (results not shown).

Homoserine was also detected in intact cells (Table 1). The medium itself contained a significant amount of homoserine (15.7 nmol/ml), possibly derived from contamination of the synthetic amino acids used to prepare the medium or homoserine adsorbed on to glass or plastic. This background of homoserine remained constant in the 3T3 neo control cell extracts and did not increase as the concentration of aspartic acid in the medium was increased. In the 3T3 A/asd-neo-13 cells the intracellular concentration of homoserine was dependent on the aspartic acid concentration in the medium, and at the highest aspartic acid concentrations it was significantly higher than in the control cells. Because some of the homoserine produced may be lost into the medium it is difficult to quantitatively estimate the rate of homoserine production from this experiment.

### Isolation of cell lines which synthesize threonine

Initial attempts to select 3T3 cells which synthesized threonine by selecting cell transfected with equal amounts of pSVthrA/asd and pSVthrB/C for growth on a threonine-free medium were unsuccessful. In an alternative approach, A/asd-neo-13 cells which produced homoserine were transfected with pSVthrB/C. After transfection, cells were selected by culture in either threonine-free medium containing additional aspartic acid or threonine-free medium containing 5 mM homoserine. Cells cultured in the threonine-free medium did not survive; however, a number of discrete colonies formed in the cells cultured in medium containing homoserine. The homoserine-resistant cell lines were isolated and transferred to threonine-free medium containing aspartic acid, where a number of lines continued to grow.

The removal of threonine from the medium did not affect the growth rates of the A/asd-B/C cell lines, which grew at similar rates in the presence and absence of threonine (Figure 4a). Neither A/asd-neo-13 cells nor the original 3T3 cell line was able to grow in the threonine-free medium. The A/asd-B/C cells were unable to grow in a lysine-free medium, showing that they still required other essential amino acids. The growth rate of A/asd-B/C cell lines was dependent on the concentration of aspartic acid in the medium. A/asd-B/C clone 6 cells grown in threonine-free medium containing no additional aspartic acid had a doubling time of about 59 h, and this fell to about 17 h when the aspartic acid concentration was raised above 2.5 mM (Figure 4b). The original 3T3 cell line had a doubling time of approx. 17 h when the cells were cultured in complete medium. Despite the fact that the synthesis of homoserine requires a high concentration of aspartic acid in the medium (Table 1), the endogenous aspartate pool can support some threonine synthesis as the cells can grow, albeit at a reduced rate, in the threonine-free medium.

To confirm that aspartic acid was the source of threonine which had been synthesized by A/asd-B/C cells, the mass ratios of aspartic acid and threonine were determined by GC–MS after the cells were cultured in medium containing L-[¹⁵N]aspartic acid. Threonine isolated from A/asd-B/C 3T3 cells was considerably enriched in ¹⁵N, with values ranging from 1.63 to 1.83 APE.

### Characteristics of threonine-independent 3T3 cells

All of the cells contained mRNA which hybridized to probes for thrA, thrB and asd (Figure 5) and had copies of both the

---

**Figure 5 Northern blots of cell lines transfected with pSVthrA/asd and pSVthrB/C**

Total cell RNA (25 μg/lane) from the lines was probed with a probe corresponding to the coding region of the appropriate plasmid. The arrows indicate the positions of the 18 and 28 S eukaryotic ribosomal RNAs and the 16 and 23 S prokaryotic ribosomal RNAs used as standards. (a) RNA probed for *thrA*. Lane 1, 3T3 neo; lane 2, A/asd-neo-13; lane 3, A/asd-B/C clone 1; lane 4, A/asd-B/C clone 3; lane 5, A/asd-B/C clone 6. (b) RNA probed for *thrB*. Lane 1, 3T3 neo; lane 2, A/asd-neo-13; lane 3, A/asd-B/C clone 1; lane 4, A/asd-B/C clone 3; lane 5, A/asd-B/C clone 6. (c) RNA probed for *asd*. Lanes 1, 2 and 3, RNA derived from clones 1, 3 and 6 at passage number 10; lanes 4 and 5, RNA derived from clones 3 and 6 at passage number 25.

pSVthrA/asd and pSVthrB/thrC plasmids incorporated into their chromosomal DNA (Figure 6). Both genes produce major transcripts of the anticipated sizes; however, there are additional larger transcripts present. The reason for this is not clear. It would appear to be an extension from the 5′ end of the gene, as there is no difference between the pSVasd and pSVthrA/asd constructs (results not shown) despite the insertion of the SV40/thrA cassette 3′ to the *asd* gene. If the mRNA was extended at the 3′ end beyond the polyadenylation site this would then contain part of the *thrA* gene which should be detected by its probe.

Table 2 shows the aspartokinase and homoserine kinase activities of three of the A/asd-B/C 3T3 cell lines. Cells containing pSVthrB/thrC had a capacity to synthesize threonine from homoserine (Table 2) which was similar to that reported previously [12]. pSVthrA/asd had not been lost during the selection, but the copy number in clones 3 and 6 appeared to be

passages after isolation of the clones). Cells cultured for a further 15 passages in the selective medium were compared with cells which had been stored in liquid nitrogen. mRNAs which hybridized to both thrA (results not shown) and asd (Figure 5c) were present at similar levels, and there was no evidence for increased enzyme activity or amplification of the thrA/asd genes.

## DISCUSSION

This paper describes the transfer of an entire multi-step biosynthetic pathway across species boundaries. The expression of bacterial synthetic genes has made mouse cells independent of a supply of threonine and allowed them to grow in the absence of an essential amino acid.

Aspartokinase I/homoserine dehydrogenase I and aspartic semialdehyde dehydrogenase expressed in mouse cells exhibit very similar kinetic parameters to the wild-type bacterial enzymes, although the relative expression of the two enzymes is different to that in *E. coli*. Three of the four enzymes of the threonine pathway are co-ordinately regulated in *E. coli*, as they are produced from one polycistronic message. *asd*, which is involved in more than one pathway, is not included in the *thr* operon and is expressed constitutively at a high level without regulation. The activity of aspartic semialdehyde dehydrogenase is considerably higher than that of aspartokinase, with a ratio of aspartokinase to aspartic semialdehyde dehydrogenase of 0.14:1 in a derepressed mutant of *E. coli* [26]. In mouse cells each gene is expressed from a separate promoter with different efficiencies and it is not possible to balance expression in the same way, with the result that 3T3 A/asd-neo-13 cells have a ratio of aspartokinase to aspartic semialdehyde dehydrogenase of 9.5:1. There is also an imbalance in the second part of the pathway, where the ratio of homoserine kinase to threonine synthase is 2.1, compared with 0.4 in *E. coli* [3]. Ideally the first step should have the lowest activity in order to prevent the accumulation of intermediates, and indeed in *E. coli* aspartokinase activity is the lowest in the pathway. A ratio of 0.65:1 of aspartokinase to homoserine kinase ensures that the rate of homoserine production cannot exceed the rate of homoserine conversion to threonine. In the 3T3 cell lines the ratio of the two activities is between 12:1 and 14:1, but despite this there appears to be no adverse effect on the cells.

In 3T3 cells transfected with pSVthrB/C, homoserine kinase activities as low as 0.05 nmol/min per mg of protein can supply about 60% of the threonine requirement when homoserine is provided. The higher aspartokinase activity would be expected to provide a more than adequate supply of homoserine if the saturating concentrations of both substrates and cofactors found in the enzyme assays were available in the intact cell. In the current experiments it is only with high levels of aspartic acid in the medium that a significant pool of homoserine accumulates in the cell and the cell growth rates approach those of the threonine-supplemented cells. This suggests that the rate of threonine production is not limited by the availability of cofactors, but by the size of the aspartic acid pool. It is only with very high concentrations of aspartic acid that the rate of aspartyl phosphate production approaches those of the other steps in the pathway, and this probably explains why the excess of aspartokinase activity does not have serious consequences for the cell.

While there are clearly significant kinetic restraints on threonine production in animal cells, it is possible to select cell lines which produce sufficient threonine for limited growth in threonine-free medium containing normal concentrations of aspartic acid. It is difficult to extrapolate these results to the whole animal, but these experiments suggest that it may be

**Figure 6 Southern blots of cell lines transfected with pSVthrA/asd and pSVthrB/C**

Samples (10 μg) of chromosomal DNA from 3T3 neo cells and A/asd-B/C clones 1, 3 and 6 were digested with HindIII and BamHI and the blot was probed for either *thrA* (a) or *thrB* (b). On the right hand side of each gel is a serial dilution of the plasmid pSVthrA/asd or pSVthrB/C which was digested with the same enzymes.

**Table 2 Characteristics of threonine-synthesizing 3T3 cell lines**

The measurements of homoserine (HS)-derived threonine, aspartokinase I (AK) activity and homoserine kinase (HSK) activity are all the averages of three separate experiments. Where given, the errors are ± S.D. The copy numbers are derived from four separate determinations of pSVthrA/asd and one determination of pSVthrB/thrC.

| Cell line | AK activity (nmol/min per mg) | HSK activity (nmol/min per mg) | HS-derived Thr (nmol/mg) | Copy no. |
|-----------|-------------------------------|--------------------------------|--------------------------|----------|
| B/C*      | –                             | 0.24 ± 0.05                    | 28.5                     | –        |
| A/asd-neo 13 | 19.25                        | –                              | –                        | 10–15    |
| Clone 1   | 42.9 ± 16.6                   | 3.38 ± 0.98                    | 28.6                     | 6–10     |
| Clone 3   | 43.9 ± 19.3                   | 1.62 ± 0.61                    | 28.3                     | 4–6      |
| Clone 6   | 25.2 ± 12.4                   | 1.77 ± 0.58                    | 35.8                     | 3–6      |

* Mixed cell lines isolated from medium containing 5 mM homoserine.

lower than that of the parent line. The enzyme activities (Table 2) show that there has been a slight increase in the aspartokinase activity of the clones, and this may reflect the small changes seen in the Southern blot. The mRNA levels also indicate that there may have been a small increase in expression. It is not clear whether this represents some rearrangement of the thrA/asd plasmids during the second transformation or is due simply to errors in the estimations. These measurements were made on cells of approximately the same passage number (about 7–10

possible to engineer cells and provide a significant supplement to dietary essential amino acids from *de novo* synthesis.

We thank Dr. I. Saint-Girons for providing the plasmid pIP3, Professor A. Puhler for pCS2 and Dr. A. Stacey for pSVpoly. We also thank Mr. D. S. Brown and Ms. A. Newman for the DNA synthesis and operating the DNA sequencer, and Mrs. V. Buchan and Mr. A. G. Calder for the amino acid analysis. This work was supported by the Scottish Office Agriculture and Fisheries Department.

**REFERENCES**

1. Rees, W. D., Fuller, M. F. and Flint, H. J. (1990) Bio/Technology 8, 629–633
2. Rees, W. D., Hay, S. M. and Flint, H. J. (1992) Biochem. J. 281, 865–870
3. Rees, W. D. and Hay, S. M. (1993) Biochem. J. 291, 315–322
4. Theze, J., Kleidman, L. and Saint-Girons, I. (1974) J. Bacteriol. 118, 577–581
5. Parsot, C., Cosart, P., Saint-Girons, I. and Cohen, G. N. (1983) Nucleic Acids Res. 11, 7331–7345
6. Katinka, M. and Yaniv, M. (1981) Nucleic Acids Res. 9, 339–347
7. Falcoz-Kelly, F., Janin, J., Saari, J. C., Veron, M., Truffa-Bachi, P. and Cohen, G. N. (1972) Eur. J. Biochem. 28, 507–519
8. Kalinowski, J., Bachmann, B., Thierbach, G. and Puhler, A. (1990) Mol. Gen. Genet. 224, 317–324

9. Kalinowski, J., Cremer, J., Bachmann, B., Eggeling, L., Sahm, H. and Puhler, A. (1991) Mol. Microbiol. 5, 1197–1204
10. Stacey, A. and Schnieke, A. (1990) Nucleic Acids Res. 18, 2829
11. Maniatis, T., Fritsch, E. F. and Sambrook, J. (1982) Molecular Cloning: A Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
12. Rees, W. D., Grant, S. D., Hay, S. M. and Saqib, K. M. (1994) Biochem. J. 299, 637–644
13. Black, S. and Wright, N. G. (1955) J. Biol. Chem. 213, 27–38
14. Shaul, O. and Galili, G. (1992) Plant Physiol. 100, 1157–1163
15. Wampler, D. E. and Westhead, E. W. (1968) Biochemistry 7, 1661–1671
16. Hama, H., Kayahara, T., Tsuda, M. and Tsuchiya, T. (1991) J. Biochem. (Tokyo) 109, 604–608
17. Black, S. and Wright, N. G. (1955) J. Biol. Chem. 213, 39–60
18. Heinrikson, R. L. and Meredith, S. C. (1984) Anal. Biochem. 136, 65–74
19. Cohen, S. A., Meys, M. and Tarvin, T. L. (1989) The Pico-Tag Method: A Manual of Advanced Techniques for Amino Acid Analysis, Millipore Corporation, Bedford, MA
20. Gilles, R. J., Didier, N. and Denton, M. (1986) Anal. Biochem. 159, 109–113
21. Gardner, J. F. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 1706–1710
22. Truffa-Bachi, P. and Cohen, G. N. (1970) Methods Enzymol. 17A, 694–699
23. Wedler, F. C. and Ley, B. W. (1993) Arch. Biochem. Biophys. 301, 416–423
24. Hegeman, G. D., Cohen, G. N. and Morgan, R. (1970) Methods Enzymol 17A, 708–713
25. Leaf, G. (1968) Shandon Instrument Applications 26, 30–37
26. Szczesiul, M. and Wampler, D. E. (1976) Biochemistry 15, 2236–2244

Received 8 February 1995/23 March 1995; accepted 3 April 1995
