
JOURNAL OF BACTERIOLOGY, June 1995, p. 3095–3103

0021-9193/95/$04.00+0

Copyright © 1995, American Society for Microbiology

# Molecular and Biochemical Characterization of Two meta-Cleavage Dioxygenases Involved in Biphenyl and m-Xylene Degradation by Beijerinckia sp. Strain B1

EUNGBIN KIM AND GERBEN J. ZYLSTRA*

Center for Agricultural Molecular Biology, Cook College, Rutgers University, New Brunswick, New Jersey 08903-0231

Received 19 January 1995/Accepted 30 March 1995

Beijerinckia sp. strain B1 is able to grow on either biphenyl or *m*-xylene as the sole source of carbon and is capable of cooxidizing many polycyclic aromatic hydrocarbons. The catabolic pathways for biphenyl and *m*-xylene degradation are coinduced and share common downstream enzymatic reactions. The catabolic pathway for biphenyl degradation involves two *meta*-cleavage steps, one for 2,3-dihydroxybiphenyl and a second for catechol. The catabolic pathway for *m*-xylene involves one *m*-cleavage step for 3-methylcatechol. The genes for two *meta*-cleavage dioxygenases were cloned from Beijerinckia sp. strain B1 on a single fragment of genomic DNA. The two genes are located approximately 5.5 kb away from one another. Expression of each gene separately in Escherichia coli and analysis of the *meta*-cleavage dioxygenase produced showed that one enzyme was more specific for 2,3-dihydroxybiphenyl while the second was more specific for catechol. The genes for the two *meta*-cleavage enzymes were thus labeled *bphC* and *xylE* for 2,3-dihydroxybiphenyl 1,2-dioxygenase and catechol 2,3-dioxygenase, respectively. Nondenaturing polyacrylamide gel electrophoresis followed by enzyme activity staining showed that the two *meta*-cleavage dioxygenases could be easily separated from each other. Similar analyses of Beijerinckia sp. strain B1 grown on succinate, biphenyl, or *m*-xylene indicate that both *meta*-cleavage enzymes are induced when cells are grown on either biphenyl or *m*-xylene. The nucleotide sequence was determined for both *bphC* and *xylE*. The two genes are transcribed in opposite directions, demonstrating that at least two operons must be involved in biphenyl degradation by Beijerinckia sp. strain B1. Analysis of the deduced amino acid sequence indicates that 2,3-dihydroxybiphenyl 1,2-dioxygenase (BphC) falls into the class of *meta*-cleavage dioxygenases acting on dihydroxylated polycyclic aromatic hydrocarbons and is somewhat distinct from the main group of *meta*-cleavage dioxygenases acting on 2,3-dihydroxybiphenyl. Catechol 2,3-dioxygenase (XylE) falls into the class of *meta*-cleavage enzymes acting on dihydroxylated monocyclic aromatic hydrocarbons but shows little similarity to the canonical TOL plasmid-encoded catechol 2,3-dioxygenase.

Beijerinckia sp. strain B1 was originally isolated for the ability to grow on biphenyl as the sole source of carbon and energy for growth (19). Beijerinckia sp. strain B1 degrades biphenyl (Fig. 1) through an initial dioxygenase attack at the 2 and 3 positions on the aromatic ring to form (+)-cis-(1S,2R)-dihydroxy-3-phenylcyclohexa-3,5-diene (*cis*-biphenyl dihydrol [58]). A subsequent dehydrogenation by *cis*-biphenyl dihydrodiol dehydrogenase results in the formation of 2,3-dihydroxybiphenyl. The latter compound undergoes ring cleavage by 2,3-dihydroxybiphenyl 1,2-dioxygenase to form the yellow *meta*-cleavage compound 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoic acid (19). A hydrolase cleaves the latter compound to produce benzoic acid and 2-oxo-penta-4-enoate. Benzoate is further degraded by a *meta*-cleavage pathway (35). Beijerinckia sp. strain B1 is also able to grow on *m*-xylene as the sole carbon source through a TOL plasmid-type metabolic pathway involving successive oxidation of the methyl group to form toluic acid (42). Toluate is further metabolized through a *meta*-cleavage pathway. Degradation of biphenyl and *m*-xylene by Beijerinckia sp. strain B1 thus proceeds by separate “upper” metabolic pathways that intersect at the “lower” pathway for benzoate (or toluate) degradation. The two upper metabolic pathways for the degradation of biphenyl and *m*-xylene are induced after growth on either biphenyl or *m*-xylene, suggesting a common regulatory element (42). The biphenyl degradative pathway of Beijerinckia sp. strain B1 has been implicated in the oxidation of polycyclic aromatic hydrocarbons such as phenanthrene (27, 36), anthracene (27), dibenzothiophene (38), acenaphthene (49), acenaphthylene (49), carbazole (45), dibenzo-*p*-dioxin (35), and benz[*a*]anthracene (28, 41). The wide range of substrates oxidized by Beijerinckia sp. strain B1 may be attributed to the relaxed specificity of the initial enzymes involved in biphenyl degradation.

In contrast to the biochemical studies, there have been very few genetic studies of the ability of Beijerinckia sp. strain B1 to degrade aromatic compounds. A small 20.8-MDa plasmid, pKG2, was implicated in the degradation of biphenyl by Beijerinckia sp. strain B1 (34). Subsequent studies (including those presented here) demonstrated that the structural genes for biphenyl degradation are not present on pKG2 (61). The metabolic pathways for biphenyl and *m*-xylene degradation by Beijerinckia sp. strain B1 are coordinately induced, and constitutive mutant strains express the enzymes for both *m*-xylene and biphenyl degradation at high levels (42). This suggests that a single regulatory gene may be involved and/or that the genes for the degradation of *m*-xylene and biphenyl are in the same operon(s). In order to investigate the ability of Beijerinckia sp.

3096 KIM AND ZYLSTRA

![Diagram](attachment:diagram.png)

FIG. 1. Pathway for degradation of biphenyl and $m$-xylene by *Beijerinckia* sp. strain B1 (19, 35, 42). The genes coding for the enzymes responsible for each step are indicated as follows: $bphA$, biphenyl dioxygenase; $bphB$, cis-biphenyl dihydrodiol dehydrogenase; $bphC$, 2,3-dihydroxybiphenyl 1,2-dioxygenase; $bphD$, 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate hydrolase; $xylAM$, xylene oxygenase; $xylB$, benzylalcohol dehydrogenase; $xylC$, benzaldehyde dehydrogenase; $xylD$ ($xylXYZ$), toluate dioxygenase; $xylL$, 1,2-dihydroxycyclohexa-3,5-diene-carboxylate dehydrogenase; and $xylE$, catechol 2,3-dioxygenase. In the case of biphenyl, $-R$ is a hydrogen, and in the case of $m$-xylene, $-R$ is a methyl group. TCA, tricarboxylic acid.

strain B1 to degrade $m$-xylene and biphenyl in more detail at the molecular level, the present work was initiated.

(A preliminary report of this work has been presented elsewhere [31, 62].)

# MATERIALS AND METHODS

**Bacterial strains, plasmids, and media.** *Beijerinckia* sp. strain B1 is the wild-type strain capable of growth on both biphenyl and $m$-xylene (19). Strain B8/36, derived from *Beijerinckia* sp. strain B1, grows on $m$-xylene and accumulates cis-biphenyl dihydrodiol when grown on succinate in the presence of biphenyl (19). *Beijerinckia* sp. strains EK3 and EK4 were derived in the present work by transposon mutagenesis of *Beijerinckia* sp. strain B1 and do not grow on either $m$-xylene or biphenyl. *Escherichia coli* S17-1 ($thi\ pro\ hsdR\ hsdM^{+}\ recA$ [50]) was used as the recipient strain in the cosmid cloning experiments, and *E. coli* DH5$\alpha$ [$F^{-}\ \phi80dlcZ\Delta M15\ \Delta(lacZY-argF)U169\ deoR\ recA1\ endA1\ hsdR17\ (r_{K}^{-}\ m_{K}^{+})\ supE44\ thi-1\ gyrA96\ relA1$; Gibco-BRL, Gaithersburg, Md.] was used as the recipient strain in all other cloning experiments. The cosmid cloning vector pHC79 (25) was obtained from Gibco-BRL, and the pGEM series of cloning vectors were obtained from Promega (Madison, Wis.). Plasmids constructed in the present study are shown in Fig. 2 and 3. Mineral salts basal medium (MSB) (51) was used for carbon source and metabolite accumulation studies. MSB was supplemented with 20 mM sodium succinate when needed. Biphenyl was provided as crystals in the petri dish lids or as crystals in liquid medium. $m$-Xylene was provided in the vapor phase in cotton-stoppered glass vials for solid media or in a glass bulb for liquid culture. MSB-TTC indicator medium containing 1 mM sodium succinate and 2,3,5-triphenyl-2H-tetrazolium chloride for mutant screening was prepared as described previously (5, 14). L broth (40) was used as complete medium. Solid media contained 2% agar. Ampicillin and kanamycin were added to the medium when needed at 100 and 50 $\mu$g/ml, respectively. *Beijerinckia* sp. strain B1 and *Beijerinckia* mutant strains were grown at 30°C, and *E. coli* strains were grown at 37°C.

**Generation of transposon mutants.** Transposon mutagenesis of *Beijerinckia* sp. strain B1 was performed according to the method of de Lorenzo et al. (9). *Beijerinckia* sp. strain B1 and *E. coli* S17-1 $\lambda\ pir$ harboring the pUT derivative with mini-Tn5$Km1$ were grown overnight in 5 ml of L broth. Kanamycin was added to the *E. coli* culture. Samples (100 $\mu$l) of each culture were mixed in 5 ml of 10 mM MgSO$_{4}$ and filtered through a 13-mm-diameter cellulose filter (Millipore Corporation, Bedford, Mass.). The filter was placed on the surface of an L agar plate and further incubated for 8 to 10 h at 30°C. The cells grown on the filter surface were resuspended in 5 ml of sterile 10 mM MgSO$_{4}$, and 0.1 ml of the suspension was plated out on MSB-TTC plates containing 50 $\mu$g of kanamycin per ml. The plates were incubated in the presence of biphenyl, and mutants were screened as described previously (14). High-performance liquid chromatography (HPLC) analysis of culture supernatants for accumulating metabolites was performed with a Beckman System Gold HPLC (Fullerton, Calif.) fitted with a reverse-phase 5-$\mu$m C$_{18}$ column (4.6 mm by 25 cm) with diode array spectral detection with a gradient of 50 to 100% methanol in water.

**meta-Cleavage dioxygenase assays.** A spray plate method was used to screen for colonies showing meta-cleavage dioxygenase activity on plates. An ether solution of catechol (0.1%) was sprayed onto colonies on plates and screened for yellow color formation as a result of meta cleavage of catechol by a meta-cleavage dioxygenase. A test tube assay was also employed for detection of low levels of meta-cleavage dioxygenase activity. One loopful of cells grown overnight on plates was resuspended in 1 ml of 50 mM phosphate buffer (pH 7.5). Catechol (20 $\mu$l of a 20 mM stock solution in methanol) was added, and the formation of a yellow color was monitored visually over time. In order to do a quantitative measurement of meta-cleavage dioxygenase activity, cell-free enzyme assays were performed according to previously described procedures (15). For these assays, *Beijerinckia* sp. strain B1 was grown in 100 ml of MSB containing succinate and either biphenyl or $m$-xylene. *E. coli* DH5$\alpha$ strains containing recombinant plasmids were grown in 100 ml of L broth containing ampicillin. When the optical density of the culture reached a value of 1.0 at 600 nm, the cells were harvested, washed in a half volume of 50 mM phosphate buffer (pH 7.5), suspended in 5 ml of the same buffer containing 10% acetone (PA buffer), and disrupted by sonication. Unbroken cells and cell debris were removed by centrifugation at 28,000

× g for 30 min. Cold (−20°C) acetone (0.8 volume) was added to the supernatant, and the resulting precipitate was removed by centrifugation at 10,000 × g for 10 min. Another 0.5 volume of cold acetone was added to the supernatant to give a final concentration of 63% acetone. The resulting precipitate was collected by centrifugation at 10,000 × g for 5 min, resuspended in 0.5 ml of PA buffer, and used as the enzyme solution. *meta*-Cleavage dioxygenase activity was assayed spectrophotometrically by measuring the increase in the absorbance at the corresponding wavelength of each *meta*-cleavage product formed from the following substrates: 2,3-dihydroxybiphenyl, λ<sub>max</sub> = 434 nm and ε = 22,000 cm<sup>−1</sup> M<sup>−1</sup> (17); catechol, λ<sub>max</sub> = 375 nm and ε = 33,400 cm<sup>−1</sup> M<sup>−1</sup> (3); 3-methylcatechol, λ<sub>max</sub> = 388 nm and ε = 13,800 cm<sup>−1</sup> M<sup>−1</sup> (3); 4-methylcatechol, λ<sub>max</sub> = 382 nm and ε = 28,100 cm<sup>−1</sup> M<sup>−1</sup> (3); and 4-chlorocatechol, λ<sub>max</sub> = 379 and ε = 40,000 cm<sup>−1</sup> M<sup>−1</sup> (46). The reaction mixture contained 50 mM phosphate buffer (pH 7.5) and an appropriate volume of enzyme solution. The reaction was initiated with the addition of the appropriate substrate at a final concentration of 0.4 mM. Protein content was determined by the method of Bradford (6) with bovine gamma globulin as the standard. One unit of enzyme activity is defined as the formation of 1.0 μmol of *meta*-cleavage product per min at 25°C. Nondenaturing polyacrylamide gel electrophoresis (7.5% acrylamide) was performed as described previously (39). Protein samples for gel electrophoresis (20 μl) were prepared by acetone fractionation of the crude cell extract after sonication of the cells as described for the enzyme assays. After electrophoresis, the nondenaturing gel was equilibrated in PA buffer for 10 min, transferred to PA buffer containing 0.4 mM catechol, and incubated for 10 min at room temperature to allow the formation of the yellow *meta*-cleavage compound 2-hydroxymuconic semialdehyde due to the presence of a *meta*-cleavage dioxygenase.

Molecular techniques. Total genomic DNA from *Beijerinckia* sp. strain B1 was prepared by the method of Olsen et al. (44). Plasmid DNA was isolated by the alkaline-sodium dodecyl sulfate procedure of Birnboim and Doly (4, 26) or purified by the QIAprep spin column procedure (Qiagen, Inc., Chatsworth, Calif.). DNA was digested with restriction enzymes as recommended by the supplier (Gibco-BRL). Agarose gel electrophoresis was performed in 40 mM Tris–20 mM acetate–2 mM EDTA (TAE) buffer. Transfer of DNA from agarose gels to Zeta-probe nylon membranes (Bio-Rad Laboratories, Rockville Center, N.Y.) was carried out with an LKB Vacugene apparatus as recommended by the supplier (LKB Instruments, Piscataway, N.J.). DNA restriction fragments to be used as probes in Southern blotting experiments were separated by gel electrophoresis and eluted from gel fragments by the procedure of Vogelstein and Gillespie (55). DNA fragments were labeled by the random priming method of Feinberg and Vogelstein (13). Southern hybridizations were performed as recommended by the nylon membrane supplier (Bio-Rad Laboratories). A chromosomal library was constructed in *E. coli* S17-1 (50) with the cosmid pHC79 (25). Partially *MboI*-digested total DNA of *Beijerinckia* sp. strain B1 was size fractionated by 10 to 40% sucrose gradient centrifugation (47). DNA fragments in the 30- to 40-kb size range were pooled and ligated to *BamHI*-digested pHC79 as described previously (25). Packaging of the ligated DNA and transfection into *E. coli* S17-1 were carried out by the procedure recommended by the packaging supplier (Promega). Colony blot experiments were performed against the cosmid library as described previously (47). Plasmid DNA was transformed into competent *E. coli* DH5α by the procedure of Hanahan (20). The chain termination procedure of Sanger et al. (48) was used for nucleotide sequencing employing the T7 or SP6 primer on pGEM vectors with T7 polymerase (52). Sequence analysis, database searches, and sequence comparisons were performed with the Genetics Computer Group software package (11). Homology searches were carried out with the GenBank database.

Materials. (+)-*cis*-(1S,2R)-Dihydroxy-3-phenylcyclohexa-3,5-diene (*cis*-biphenyl dihydrodiol) was prepared from biphenyl with *Beijerinckia* sp. strain B8/36 according to published procedures (19). 2,3-Dihydroxybiphenyl was prepared from biphenyl with *E. coli* JM109(pDTG602), which harbors the genes for toluene dioxygenase and *cis*-toluene dihydrodiol dehydrogenase from *Pseudomonas putida* F1 (59, 60). 3-Chlorocatechol and 4-chlorocatechol were obtained from Helix Biotech Corp. (Richmond, British Columbia, Canada). Biphenyl, *m*-xylene, catechol, 3-methylcatechol, and 4-methylcatechol were obtained from Aldrich Chemical Company, Inc. (Milwaukee, Wis.). All chemicals were of the highest purity obtainable.

Nucleotide sequence accession numbers. The nucleotide sequences for *bphC* and *xylE* from *Beijerinckia* sp. strain B1 have been deposited in the GenBank database under the accession numbers U23374 and U23375, respectively.

**RESULTS**

Mutant construction. *Beijerinckia* sp. strain B1 was mutagenized with a conjugatable suicide delivery plasmid carrying the transposable element mini-Tn5Km1 (9). *Beijerinckia* sp. strain B1 mutants blocked in the degradation of biphenyl were initially detected with MSB-TTC plates as described in Materials and Methods. The mutant strains were characterized by HPLC analysis of culture supernatants after growth on MSB with succinate in the presence of biphenyl. Two mutant strains

obtained from separate transposon mutagenesis experiments accumulate *cis*-biphenyl dihydrodiol, as determined by comparison with the authentic compound. These mutants, designated EK3 and EK4, were also unable to grow on *m*-xylene or *m*-toluate. Southern blotting experiments with total genomic DNA from EK3 and EK4 digested with *NotI* or *EcoRI* demonstrated that a single restriction fragment hybridized with the kanamycin resistance gene from mini-Tn5Km1 used as a probe. (*NotI* and *EcoRI* cut mini-Tn5Km1 adjacent to the kanamycin resistance gene.) This means that EK3 and EK4 have only one copy of the transposable element in the genome, ruling out the possibility of multiple insertions. Spontaneous reversion experiments with the mutant strains EK3 and EK4 showed that revertants selected for growth either on biphenyl or on *m*-xylene recover the ability to grow on both substrates. These data suggest that the genes for biphenyl and *m*-xylene degradation may be physically linked in *Beijerinckia* sp. strain B1.

Cloning of the genes for two *meta*-cleavage dioxygenases from *Beijerinckia* sp. strain B1. A cosmid library of *Beijerinckia* sp. strain B1 was constructed with partially *MboI*-digested, size-fractionated total genomic DNA and the cosmid pHC79. Each cosmid clone should contain approximately 30 kb of the *Beijerinckia* sp. strain B1 genome. The cosmid library (1,200 clones) was screened for the ability to produce indigo from indole, indicative of dioxygenase activity (12). Colonies were also screened for the formation of the yellow compound 2-hydroxymuconic semialdehyde from catechol through the action of a *meta*-cleavage dioxygenase by being sprayed with an ether solution of catechol. No cosmid clones that had the ability to catalyze either reaction were detected. Because activity assays were not successful in locating the desired cosmid clone, a more molecular approach was taken to locate a clone containing the genes for biphenyl and *m*-xylene degradation from *Beijerinckia* sp. strain B1.

The mini-Tn5Km1 transposon insertion in the mutant strain EK3 should be in an operon containing the genes for *m*-xylene and biphenyl degradation. The Southern blotting data indicated that a 6.0-kb *NotI* fragment of genomic DNA in EK3 contains the kanamycin resistance gene of mini-Tn5Km1. DNA fragments 6.0 kb in size were excised from a gel of an EK3 genomic *NotI* digest and cloned into pGEM5Zf(−) with selection for kanamycin resistance. The resultant clone, pGJZ1501 (Fig. 2), demonstrated *meta*-cleavage dioxygenase activity, indicating that the gene for this enzyme was adjacent to the transposon insertion. In order to determine the exact location for the *meta*-cleavage dioxygenase gene, subclones of pGJZ1501 were constructed and assayed for *meta*-cleavage dioxygenase activity. The results (Fig. 2) indicate that the *meta*-cleavage dioxygenase gene can be localized to a 2.1-kb *ApaI*-to-*ClaI* fragment of DNA. The restriction enzymes *EcoRV* and *BamHI* must cleave the gene for the *meta*-cleavage dioxygenase, because subclones interrupted at either of those two sites (pGJZ1505 through pGJZ1507) do not express *meta*-cleavage dioxygenase activity. The clones pGJZ1502 and pGJZ1503 represent the 3.0-kb *ClaI* fragment cloned into the vector pGEM7Zf(−) in opposite orientations, with only pGJZ1503 showing *meta*-cleavage dioxygenase activity. If gene expression is driven only from the *lac* promoter on the vector, then it is probable that the *meta*-cleavage dioxygenase gene is transcribed from left to right on the map in Fig. 2.

The cloned *meta*-cleavage dioxygenase gene was utilized as a probe to locate a cosmid clone in the genomic library. First, however, the number of copies of the gene (and cross-hybridizing genes) in the *Beijerinckia* sp. strain B1 genome was determined. Southern blots were performed with total genomic DNA prepared from *Beijerinckia* sp. strain B1 cut with

3098 KIM AND ZYLSTRA

![Diagram](attachment:diagram.png)

FIG. 2. Restriction map of the cloned region in pGJZ1501 and localization of the *meta*-cleavage dioxygenase gene. The boxes indicate the deduced positions of the genes for catechol 2,3-dioxygenase and kanamycin (Km) resistance (from mini-Tn5KmI). The activity column designates whether the clone expresses *meta*-cleavage dioxygenase activity. The arrow indicates the direction of transcription from the *lac* promoter on the cloning vector [pGEM5Zf(−) for pGJZ1501 and pGJZ1505 and pGEM7Zf(−) for the other plasmids]. Abbreviations: A, *Apa*I; B, *Bam*HI; C, *Cla*I; Ev, *Eco*RV; and N, *Not*I.

the restriction enzymes *Not*I, *Eco*RI, *Hind*III, *Bam*HI, and *Eco*RV. The 3.0-kb *Cla*I fragment from pGJZ1503 containing the *meta*-cleavage dioxygenase gene was used as a probe under low-stringency conditions. In the cases of *Not*I, *Eco*RI, and *Hind*III, a single genomic restriction fragment hybridized to the probe, while with *Bam*HI and *Eco*RV, two restriction fragments hybridized with the probe. These data indicate that only one homologous fragment of DNA was present and that no related genes would cross-hybridize with the probe. The 3.0-kb *Cla*I fragment of pGJZ1503 was then utilized in colony blot experiments with the cosmid library as the target. Initially, 700 cosmid clones were screened and one colony showed a strong hybridization signal. This clone (designated pGJZ1510) also had weak *meta*-cleavage dioxygenase activity, as demonstrated with the concentrated whole-cell test tube assay. Southern blots with pGJZ1510 digested with *Cla*I and with the 3.0-kb *Cla*I fragment from pGJZ1503 as a probe showed that a 3.0-kb *Cla*I fragment hybridized.

A restriction map of the cosmid clone pGJZ1510 is shown in Fig. 3. Two *meta*-cleavage dioxygenases are postulated to be involved in biphenyl degradation by *Beijerinckia* sp. strain B1 (Fig. 1), and it is possible that the genes for these two enzymes are physically linked. This being the case, several subclones of the cosmid clone pGJZ1510 were constructed and screened for *meta*-cleavage dioxygenase activity (Fig. 3). *Meta*-Cleavage dioxygenase activity was detected from a subclone containing either a 3.0-kb *Cla*I fragment (pGJZ1521) or a 2.1-kb *Cla*I-to-*Apa*I fragment (pGJZ1522) but not from a 2.0-kb *Not*I-to-*Eco*RV fragment (pGJZ1523). This was expected on the basis of the data presented above, because pGJZ1510 represents the same region of the genome as pGJZ1501, except that it was cloned from the wild-type *Beijerinckia* sp. strain, B1, rather than from the transposon mutant strain EK3. A second gene for a *meta*-cleavage dioxygenase was detected first on a 7.2-kb *Hind*III-to-*Xba*I subclone (pGJZ1511) and localized further to a 2.3-kb *Sst*I-to-*Kpn*I fragment (pGJZ1513) (Fig. 3). Because

the oppositely oriented subclones pGJZ1512 and pGJZ1513 both show *meta*-cleavage dioxygenase activity, no inferences could be made regarding transcription direction. A subclone containing a 2.3-kb *Hind*III-to-*Eco*RI fragment (pGJZ1514) does not show *meta*-cleavage dioxygenase activity, indicating that *Eco*RI cleaves the gene for the *meta*-cleavage dioxygenase. Thus, two genes that code for two different *meta*-cleavage dioxygenases were found. The two genes are located approximately 5.5 kb away from each other on the 34-kb genomic region cloned in pGJZ1510.

Characteristics of the two *meta*-cleavage dioxygenases. The two *meta*-cleavage dioxygenase genes found may be specific for two different steps in the biphenyl catabolic pathway (Fig. 1). In order to determine if this is actually the case, two of the subclones (pGJZ1513 and pGJZ1522) expressing *meta*-cleavage dioxygenase activity were analyzed for their ability to catalyze ring cleavage of several different dihydroxylated aromatic compounds. Enzyme assays for *meta*-cleavage dioxygenase activity were performed with acetone-fractionated crude cell extracts of *E. coli* DH5α(pGJZ1513) or DH5α(pGJZ1522) as described in Materials and Methods. The dihydroxylated compounds 2,3-dihydroxybiphenyl, catechol, 3-methylcatechol, 4-methylcatechol, 3-chlorocatechol, and 4-chlorocatechol were used as substrates, and the formation of the *meta*-cleavage product was monitored. The results are summarized in Table 1. Neither enzyme is able to cleave 3-chlorocatechol, and both enzymes are able to cleave 4-chlorocatechol, albeit at a very low rate. The catechol dioxygenase encoded by pGJZ1522 has maximal activity against catechol, with significant activity against 3-methylcatechol and 4-methylcatechol (62.1 and 30.3% of the activity against catechol, respectively). However, the enzyme encoded by pGJZ1522 has very little activity (4.9% of that against catechol) with 2,3-dihydroxybiphenyl as the substrate. In contrast, the enzyme encoded by pGJZ1513 shows maximal activity against 2,3-dihydroxybiphenyl and is able to cleave catechol at less than one-fourth the rate seen

VOL. 177, 1995 BEIJERINCKIA SP. STRAIN B1 meta-CLEAVAGE DIOXYGENASES 3099

| Activity | Plasmid |
| --- | --- |
| - | pGJZ1523 |
| + | pGJZ1522 |
| + | pGJZ1521 |

![Diagram](#)

FIG. 3. Restriction map of pGJZ1510 and localization of the two *meta*-cleavage dioxygenase genes. The activity column designates whether the clone expresses *meta*-cleavage dioxygenase activity. The solid arrows indicate the direction of transcription from the *lac* promoter on the cloning vector [pGEM3Z for pGJZ1511, pGJZ1512, and pGJZ1514; pGEM5Zf(−) for pGJZ1513 and pGJZ1523; and pGEM7Zf(−) for pGJZ1521 and pGJZ1522]. The dashed arrows indicate the direction of transcription as determined by the nucleotide sequence. The *StuI*-to-*ApaI* fragment marked *xylE* and the two adjacent *NcoI* fragments marked *bphC* are the regions for which the nucleotide sequences were determined (Fig. 5 and 6). The double lines on the pGJZ1510 map indicate the pHc79 cosmid cloning vector (the *HindIII* site on the right is in pHc79). Abbreviations: A, *ApaI*; C, *ClaI*; E, *EcoRI*; Ev, *EcoRV*; H, *HindIII*; K, *KpnI*; N, *NotI*; Nc, *NcoI*; S, *SstI*; and St, *StuI*.

with 2,3-dihydroxybiphenyl as the substrate. The ability of the *meta*-cleavage dioxygenase encoded by pGJZ1513 to cleave 3-methylcatechol and 4-methylcatechol is between the ability to cleave catechol and the ability to cleave 2,3-dihydroxybiphenyl. These data suggest that pGJZ1513 contains the *bphC* gene, coding for 2,3-dihydroxybiphenyl 1,2-dioxygenase, while

pGJZ1522 contains the *xylE* gene, coding for catechol 2,3-dioxygenase (Fig. 1).

The two *meta*-cleavage dioxygenases do not share significant genetic homology, as determined by the Southern blotting data described above. The catalytic functions of the two enzymes are similar, and thus the presence of either enzyme is difficult to detect in the presence of the other. However, noticeably different migration distances were seen for the two *meta*-cleavage dioxygenases when samples were run on a nondenaturing polyacrylamide gel (Fig. 4). Acetone-fractionated crude cell extract samples were run on a 7.5% nondenaturing polyacrylamide gel and then exposed to catechol for activity staining. *Beijerinckia* sp. strain B1 when grown on either biphenyl or *m*-xylene (Fig. 4, lanes 2 and 3, respectively) shows two activity-staining bands that are not present when the strain is grown on succinate (lane 1). The cosmid clone pGJZ1510, containing both *meta*-cleavage dioxygenase genes, also shows two activity-staining bands (Fig. 4, lane 6). The subclone pGJZ1522, containing the catechol 2,3-dioxygenase coded by the *xylE* gene, has only the lower activity-staining band (Fig. 4, lane 4). The subclone pGJZ1512, containing the 2,3-dihydroxybiphenyl 1,2-

TABLE 1. Ring cleavage dioxygenase enzyme activities in acetone-fractionated cell extracts from *E. coli* DH5α containing pGJZ1513 or pGJZ1522 with different dihydroxylated substrates

| Substrate | Enzyme activity (U/mg of protein)<sup>a</sup> |
| --- | --- |
|  | DH5α(pGJZ1513) | DH5α(pGJZ1522) |
| 2,3-Dihydroxybiphenyl | 518.6 ± 31.4 (100.0) | 215.4 ± 18.7 (4.9) |
| 3-Methylcatechol | 465.0 ± 45.7 (89.7) | 2704.4 ± 355.5 (62.1) |
| 4-Methylcatechol | 127.6 ± 4.4 (24.6) | 1319.4 ± 31.9 (30.3) |
| 4-Chlorocatechol | 36.8 ± 3.9 (7.1) | 413.6 ± 7.4 (9.5) |
| Catechol | 103.6 ± 2.8 (20.0) | 4353.9 ± 345.6 (100.0) |

<sup>a</sup> Enzyme activities are the averages from at least three independent experiments. Values in parentheses are relative activities.

3100 KIM AND ZYLSTRA

FIG. 4. Nondenaturing polyacrylamide gel electrophoresis separation of the two *meta*-cleavage dioxygenases. The *meta*-cleavage dioxygenases were visualized after activity staining with catechol as the substrate. Samples of acetone-fractionated crude cell extracts were added to each well as follows: lane 1, *Beijerinckia* sp. strain B1 grown on succinate; lane 2, *Beijerinckia* sp. strain B1 grown on biphenyl; lane 3, *Beijerinckia* sp. strain B1 grown on *m*-xylene; lane 4, *E. coli* DH5α(pGJZ1522); lane 5, *E. coli* DH5α(pGJZ1512); lane 6, *E. coli* DH5α(pGJZ1510).

dioxgenase coded for by the *bphC* gene, has only the upper activity-staining band (Fig. 4, lane 5). The data indicate that the two catechol dioxygenases can be separated from one another by native gel electrophoresis. The data also indicate that both the *bphC* and the *xylE* genes are induced when *Beijerinckia* sp. strain B1 is grown on either biphenyl or *m*-xylene. The latter conclusion could not be drawn from enzyme assays alone because of the ability of both enzymes to act on the same substrate.

Nucleotide sequences of the *bphC* and *xylE* genes. In order to characterize the two *meta*-cleavage dioxygenases in more detail at the molecular level, the nucleotide sequences of *bphC* and *xylE* were determined. The nucleotide sequences of two adjacent *NcoI* fragments (857 and 342 bp) containing the *bphC* gene were determined and are shown in Fig. 5. The nucleotide sequence of a 1,278-bp *StuI*-to-*ApaI* fragment of DNA containing the *xylE* gene was determined and is shown in Fig. 6. Both strands of the DNA were sequenced in their entirety. A single open reading frame can be found in each sequenced region (Fig. 5 and 6) that corresponds to either the *bphC* gene or the *xylE* gene. A putative ribosome binding site was found in front of each gene. The locations of the sequenced regions on the cosmid clone pGJZ1510 are shown in Fig. 3. It is interesting that the two genes are transcribed in opposite directions.

DISCUSSION

*Beijerinckia* sp. strain B1 degrades biphenyl and *m*-xylene by the pathway shown in Fig. 1 (19, 35, 42). Two *meta*-cleavage dioxygenases are involved in the biodegradative pathway for biphenyl, with one enzyme responsible for cleavage of 2,3-dihydroxybiphenyl and the other enzyme responsible for cleavage of catechol. In the case of *m*-xylene degradation, only one *meta*-cleavage dioxygenase is necessary for the cleavage of 3-methylcatechol. The genes for two *meta*-cleavage dioxygenases were cloned from *Beijerinckia* sp. strain B1 on a single fragment of genomic DNA. Subcloning experiments localized the position of each *meta*-cleavage dioxygenase gene on the cosmid clone, and expression of each gene in *E. coli* provided a functional analysis of the physiological role of each enzyme in the cell. The data on substrate specificity (Table 1) suggest that the *meta*-cleavage dioxygenase encoded by pGJZ1513 is responsible for cleaving 2,3-dihydroxybiphenyl in the biphenyl degradation pathway. The data also suggest that the *meta*-cleavage dioxygenase encoded by pGJZ1522 is responsible for cleaving catechol in the biphenyl degradation pathway. These data complement those from previous work on substrate specificity with partially purified enzyme preparations of 2,3-dihydroxybiphenyl 1,2-dioxygenase and catechol 2,3-dioxygenase (35). The ability to cleave 3-methylcatechol in the *m*-xylene degradation pathway may be shared by both enzymes. Both 2,3-dihydroxybiphenyl 1,2-dioxygenase and catechol 2,3-dioxygenase are induced when *Beijerinckia* sp. strain B1 is grown on either *m*-xylene or biphenyl, as evidenced through detection of each enzyme by nondenaturing gel electrophoresis (Fig. 4). These data prove that *Beijerinckia* sp. strain B1 degrades biphenyl through a *meta*-cleavage pathway for benzoate degradation. This provides physical proof that both an upper (benzoate) and lower (benzoate degradation) pathway for biphenyl degradation are induced in *Beijerinckia*

FIG. 5. Nucleotide sequences of two adjacent *NcoI* fragments containing *bphC* coding for 2,3-dihydroxybiphenyl 1,2-dioxygenase. A putative ribosome binding site is underlined. An asterisk indicates the stop codon.

pounds. An exception to this rule is TodE (59), which is responsible for cleavage of 3-methylcatechol in the toluene degradation pathway of *P. putida* F1. This position in the dendrogram may be due to the fact that the initial dioxygenases in the biphenyl (54) and toluene (59) degradation pathways share significant homology, and thus the genetically linked meta-cleavage dioxygenases would also be closely related. In addition, four meta-cleavage dioxygenases (1, 29, 30) do not fall into either of the two classes and are not shown on the dendrogram in Fig. 7. The *Beijerinckia* sp. strain B1 catechol 2,3-dioxygenase (XylE) is most closely related (89% identity) to CmpE, a catechol 2,3-dioxygenase from *Pseudomonas* sp. strain HV3 (57). These two enzymes (*Beijerinckia* sp. strain B1 XylE and *Pseudomonas* sp. CmpE) constitute a subclass distinct from the mainstream XylE class personified by XylE from *P. putida* mt-2 (42), with less than 50% homology between *Beijerinckia* sp. strain B1 XylE and *P. putida* mt-2 XylE. On the other hand, the nucleotide sequence of *bphC* from *Beijerinckia* sp. strain B1 is identical to that for *bphC* from *Pseudomonas paucimobilis* Q1 (53) except for three bases outside of the open reading frame. The biphenyl degradation pathway in *P. paucimobilis* Q1 is induced by both *m*-xylene and biphenyl (17), just as is the biphenyl degradation pathway in *Beijerinckia* sp. strain B1 (42). The identical sequence data for *bphC* and the common induction data for the biphenyl degradation pathway suggest that the rest of the genes for biphenyl degradation in *Beijerinckia* sp. strain B1 and *P. paucimobilis* Q1 are similar. As illustrated by the dendrogram in Fig. 7, *Beijerinckia* sp. strain B1 BphC is more closely related (60% homology) to NahC from *P. putida* PpG7 (22) than to BphCs involved in biphenyl degradation. NahC is involved in cleavage of 1,2-dihydroxynaphthalene in the NAH plasmid naphthalene degradative pathway (56). Indeed, Kuhm et al. have shown that BphC from *P. paucimobilis* Q1 has a much higher level of enzymatic activity with 1,2-dihydroxynaphthalene as the substrate than with 2,3-dihydroxybiphenyl as the substrate (37). This observation may explain the fact that *Beijerinckia* sp. strain B1 is able to degrade a variety of polycyclic aromatic hydrocarbons through ring cleavage (19, 38, 41) upon induction of the biphenyl degradative pathway.

*Beijerinckia* sp. strain B1 is able to degrade biphenyl and *m*-xylene by mutually inducible pathways that share in common many downstream enzymatic reactions (Fig. 1). The genes for the two pathways must be physically linked, because the independently derived transposon mutants EK3 and EK4 are simultaneously blocked in the degradation of both *m*-xylene and biphenyl. Both EK3 and EK4 accumulate *cis*-biphenyl dihydrodiol, indicating that each mutant strain lacks *cis*-dihydrodiol dehydrogenase enzymatic activity (the product of the *bphB* gene). These two strains are also unable to grow on *m*-xylene or *m*-toluic acid, indicating that both of the transposon mutant strains EK3 and EK4 must also be blocked in the downstream common region of both the biphenyl and *m*-xylene catabolic pathways. Cloning of the genomic region adjacent to the transposon insertion in the mutant strain EK3 demonstrated that *xylE* is located upstream from the insertion point (Fig. 2 and 3). Insertion of a transposon in an operon may cause polar effects on expression of genes downstream of the insertion point in the same operon. The present data thus suggest that *bphB* is not only located in the same operon as *xylE* but is also positioned downstream of *xylE*. This hypothesis is supported by the fact that insertional inactivation of the cloned *xylE* gene with a kanamycin resistance cassette and reinsertion into the *Beijerinckia* sp. strain B1 genome by homologous site-specific recombination also results in a mutant strain that displays a *bphB* phenotype (32). Such a combination

FIG. 7. Dendrogram showing the levels of homology between the amino acid sequences of different meta-cleavage dioxygenases. The dendrogram was constructed with the Pileup program of the Genetics Computer Group sequence analysis package. The sequences are as follows: C23OII, *P. putida* MT15 catechol 2,3-dioxygenase (unpublished [GenBank accession number U01826]); TdnC, *P. putida* UCC2 3-methylcatechol 2,3-dioxygenase (unpublished [GenBank accession number X59790]); CmpE, *Pseudomonas* sp. strain HV3 catechol 2,3-dioxygenase (57); XylE, *P. putida* mt-2 catechol 2,3-dioxygenase (43); XylE, *Bacillus subtilis* (unpublished [patent EP0086139-A1]); BztE, *Pseudomonas aeruginosa* JI104 catechol 2,3-dioxygenase (unpublished [GenBank accession number X60740]); BphE, *Pseudomonas* sp. strain IC catechol 2,3-dioxygenase (8); PhhB, *P. putida* P35X catechol 2,3-dioxygenase (unpublished [GenBank accession number X77856]); DmpB, *P. putida* CF600 catechol 2,3-dioxygenase (2); NahH, *P. putida* PpG7 catechol 2,3-dioxygenase (18); Cdo, *Rhodococcus rhodochrous* CTM catechol 2,3-dioxygenase (7); PheB, *Bacillus stearothermophilus* FDTP-3 catechol 2,3-dioxygenase (unpublished [GenBank accession number X67860]); BphC, *P. paucimobilis* Q1 2,3-dihydroxybiphenyl 1,2-dioxygenase (53); NahC, *P. putida* PpG7 1,2-dihydroxynaphthalene dioxygenase (22); DoxG, *Pseudomonas* sp. 1,2-dihydroxynaphthalene dioxygenase (10); BphC, *Pseudomonas* sp. strain LB400 2,3-dihydroxybiphenyl 1,2-dioxygenase (24); BphC, *Pseudomonas pseudoalcaligenes* KF707 2,3-dihydroxybiphenyl 1,2-dioxygenase (16); BphC, *P. pseudoalcaligenes* KF715 2,3-dihydroxybiphenyl 1,2-dioxygenase (23); BphC, *Pseudomonas* sp. strain KKS102 2,3-dihydroxybiphenyl 1,2-dioxygenase (33); BphC1, *Rhodococcus globerulus* P6 2,3-dihydroxybiphenyl 1,2-dioxygenase (1); TodE, *P. putida* F1 3-methylcatechol 2,3-dioxygenase (58); and DbfB, *Sphingomonas* sp. strain RW1 2,2′,3-trihydroxybiphenyl dioxygenase (21).

sp. strain B1 after growth on either biphenyl or *m*-xylene, confirming earlier observations (35, 42).

The nucleotide sequences of the genes for 2,3-dihydroxybiphenyl 1,2-dioxygenase and catechol 2,3-dioxygenase (*bphC* and *xylE*, respectively) were determined (Fig. 5 and 6). Comparison of the deduced amino acid sequences of both meta-cleavage dioxygenases with other published meta-cleavage dioxygenase sequences confirms the designations of *bphC* and *xylE* for the respective genes. A dendrogram showing the relationships between the amino acid sequences of different meta-cleavage dioxygenases is shown in Fig. 7. The dendrogram confirms the observation of Harayama and Rekik (22) that most meta-cleavage dioxygenases can be clustered into two general families: dioxygenases responsible for extradiol cleavage of monocyclic aromatic compounds and dioxygenases responsible for extradiol cleavage of bicyclic aromatic com-

of genes in a single operon would explain why *m*-xylene is capable of inducing both the *m*-xylene degradative pathway and the biphenyl degradative pathway in *Beijerinckia* sp. strain B1.

For many dioxygenase-mediated degradative pathways, a *cis*-dihydrodiol dehydrogenase gene (*nahB*, *bphB*, or *todD*) is found nearby and upstream of a *meta*-cleavage dioxygenase gene (*nahC*, *bphC*, or *todE*) (10, 54, 59). However, this is not true in *Beijerinckia* sp. strain B1. As discussed in the previous paragraph, the gene for *cis*-biphenyl dihydrodiol dehydrogenase (*bphB*) must be located downstream from the gene for catechol 2,3-dioxygenase (*xylE*). The gene for 2,3-dihydroxybiphenyl 1,2-dioxygenase (*bphC*) is located 5.5 kb upstream from *xylE*. In addition, the nucleotide sequence data indicate that *xylE* and *bphC* are transcribed in opposite directions. This suggests that at least two different operons are involved in biphenyl degradation by *Beijerinckia* sp. strain B1. Both 2,3-dihydroxybiphenyl 1,2-dioxygenase and catechol 2,3-dioxygenase are produced when *Beijerinckia* sp. strain B1 is grown on either biphenyl or *m*-xylene (Fig. 4), indicating that both of these operons are induced after growth on either substrate.

### ACKNOWLEDGMENTS

This work was supported by a grant from the Electric Power Research Institute. G.J.Z. is the recipient of a Young Investigator Award from the National Science Foundation.

We thank A. Goyal for assistance in the nucleotide sequence analysis and P. Day and A. Goyal for reviewing the manuscript prior to submission.

### REFERENCES

1. Asturias, J. A., L. D. Eltis, M. Prucha, and K. N. Timmis. 1994. Analysis of three 2,3-dihydroxybiphenyl 1,2-dioxygenases found in *Rhodococcus globulus* P6: identification of a new family of extradiol dioxygenases. J. Biol. Chem. 269:7807–7815.
2. Bartilson, M., and V. Shingler. 1989. Nucleotide sequence and expression of the catechol 2,3-dioxygenase-encoding gene of phenol-catabolizing *Pseudomonas* CF600. Gene 85:233–238.
3. Bayly, R. C., S. Dagley, and D. T. Gibson. 1966. The metabolism of cresols by species of *Pseudomonas*. Biochem. J. 101:293–301.
4. Birnboim, H. C., and J. Doly. 1979. A rapid alkaline extraction procedure for screening recombinant plasmid DNA. Nucleic Acids Res. 7:1513–1523.
5. Bochner, B. R., and M. A. Savageau. 1977. Generalized indicator plates for genetic, metabolic, and taxonomic studies with microorganisms. Appl. Environ. Microbiol. 33:434–444.
6. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72:248–254.
7. Candidus, S., K.-H. van Pée, and F. Lingens. 1994. The catechol 2,3-dioxygenase gene of *Rhodococcus rhodochrous* CTM: nucleotide sequence, comparison with isofunctional dioxygenases and evidence for an active-site histidine. Microbiology 140:321–330.
8. Carrington, B., A. Lowe, L. E. Shaw, and P. A. Williams. 1994. The lower pathway operon for benzoate catabolism in biphenyl-utilizing *Pseudomonas* sp. strain IC. Microbiology 140:499–508.
9. de Lorenzo, V., M. Herrero, U. Jakubzik, and K. N. Timmis. 1990. Mini-Tn5 transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in gram-negative eubacteria. J. Bacteriol. 172:6568–6572.
10. Denome, S. A., D. C. Stanley, E. S. Olson, and K. D. Young. 1993. Metabolism of dibenzothiophene and naphthalene in *Pseudomonas* strains: complete DNA sequence of an upper naphthalene catabolic pathway. J. Bacteriol. 175:6890–6901.
11. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12:387–395.
12. Ensley, B. D., B. J. Ratzkin, T. D. Osslund, M. J. Simon, L. P. Wackett, and D. T. Gibson. 1983. Expression of naphthalene oxidation genes in *Escherichia coli* results in the biosynthesis of indigo. Science 222:167–169.
13. Feinberg, A. P., and B. Vogelstein. 1983. A technique for radiolabeling DNA restriction endonuclease fragments to high specific activity. Anal. Biochem. 132:6–13.
14. Finette, B. A., V. Subramanian, and D. T. Gibson. 1984. Isolation and characterization of *Pseudomonas putida* PpF1 mutants defective in the toluene dioxygenase enzyme system. J. Bacteriol. 160:1003–1009.
15. Furukawa, K., and N. Arimura. 1987. Purification and properties of 2,3-dihydroxybiphenyl dioxygenase from polychlorinated biphenyl-degrading *Pseudomonas pseudoalcaligenes* and *Pseudomonas aeruginosa* carrying the cloned *bph* gene. J. Bacteriol. 169:924–927.
16. Furukawa, K., N. Arimura, and T. Miyazaki. 1987. Nucleotide sequence of the 2,3-dihydroxybiphenyl dioxygenase gene of *Pseudomonas pseudoalcaligenes*. J. Bacteriol. 169:427–429.
17. Furukawa, K., J. R. Simon, and A. M. Chakrabarty. 1983. Common induction and regulation of biphenyl, xylene/toluene, and salicylate catabolism in *Pseudomonas paucimobilis*. J. Bacteriol. 154:1356–1362.
18. Ghosal, D., I.-S. You, and I. C. Gunsalus. 1987. Nucleotide sequence and expression of gene *nahH* of plasmid NAH7 and homology with gene *xylE* of TOL pWWO. Gene 55:19–28.
19. Gibson, D. T., R. L. Roberts, M. C. Wells, and V. M. Kobal. 1973. Oxidation of biphenyl by a *Beijerinckia* species. Biochem. Biophys. Res. Commun. 50:211–219.
20. Hanahan, D. 1983. Studies on transformation of *Escherichia coli* with plasmids. J. Mol. Biol. 166:557–580.
21. Happe, B., L. D. Eltis, H. Poth, R. Hedderich, and K. N. Timmis. 1993. Characterization of 2,2′,3-trihydroxybiphenyl dioxygenase, an extradiol dioxygenase from the dibenzofuran- and dibenzo-*p*-dioxin-degrading bacterium *Sphingomonas* sp. strain RW1. J. Bacteriol. 175:7313–7320.
22. Harayama, S., and M. Rekik. 1989. Bacterial aromatic ring-cleavage enzymes are classified into two different gene families. J. Biol. Chem. 264:15328–15333.
23. Hayase, N., K. Taira, and K. Furukawa. 1990. *Pseudomonas putida* KF715 *bphABCD* operon encoding biphenyl and polychlorinated biphenyl degradation: cloning, analysis, and expression in soil bacteria. J. Bacteriol. 172:1160–1164.
24. Hofer, B., L. D. Eltis, D. N. Dowling, and K. N. Timmis. 1993. Genetic analysis of a *Pseudomonas* locus encoding a pathway for biphenyl/polychlorinated biphenyl degradation. Gene 130:47–55.
25. Hohn, B., and J. Collins. 1980. A small cosmid for efficient cloning of large DNA fragments. Gene 11:291–298.
26. Ish-Horowitz, D., and J. F. Burke. 1981. Rapid and efficient cosmid cloning. Nucleic Acids Res. 9:2989–2998.
27. Jerina, D. M., H. Selander, H. Yagi, M. C. Wells, J. F. Davey, V. Mahadevan, and D. T. Gibson. 1976. Dihydrodiols from anthracene and phenanthrene. J. Am. Chem. Soc. 98:5988–5996.
28. Jerina, D. M., P. J. van Bladeren, H. Yagi, D. T. Gibson, V. Mahadevan, A. S. Neese, M. Koreeda, N. D. Sharma, and D. R. Boyd. 1984. Synthesis and absolute configuration of the bacterial *cis*-1,2-, *cis*-8,9-, and *cis*-10,11-dihydrodiol metabolites of benz[a]anthracene formed by a strain of *Beijerinckia*. J. Org. Chem. 49:3621–3628.
29. Kabisch, M., and P. Fortnagel. 1990. Nucleotide sequence of metapyrocatechase I (catechol 2,3-oxygenase I) gene *mpcI* from *Alcaligenes eutrophus* JMP222. Nucleic Acids Res. 18:3405–3406.
30. Kabisch, M., and P. Fortnagel. 1990. Nucleotide sequence of the metapyrocatechase II (catechol 2,3-oxygenase II) gene *mpcII* from *Alcaligenes eutrophus* JMP222. Nucleic Acids Res. 18:5543.
31. Kim, E., and G. J. Zylstra. 1994. Cloning and nucleotide sequence of two catechol 2,3-dioxygenase genes from *Beijerinckia* sp. strain B1, abstr. Q-398, p. 458. *In* Abstracts of the 94th General Meeting of the American Society for Microbiology 1994. American Society for Microbiology, Washington, D.C.
32. Kim, E., and G. J. Zylstra. Unpublished data.
33. Kimbara, K., T. Hashimoto, M. Fukuda, T. Koana, M. Takagi, M. Oishi, and K. Yano. 1989. Cloning and sequencing of two tandem genes involved in degradation of 2,3-dihydroxybiphenyl to benzoic acid in the polychlorinated biphenyl-degrading soil bacterium *Pseudomonas* sp. strain KKS102. J. Bacteriol. 171:2740–2747.
34. Kiyohara, H., M. Sugiyama, F. J. Mondello, D. T. Gibson, and K. Yano. 1983. Plasmid involvement in the degradation of polycyclic aromatic hydrocarbons by a *Beijerinckia* species. Biochem. Biophys. Res. Commun. 111:939–945.
35. Klečka, G. M., and D. T. Gibson. 1980. Metabolism of dibenzo-*p*-dioxin and chlorinated dibenzo-*p*-dioxins by a *Beijerinckia* species. Appl. Environ. Microbiol. 39:288–296.
36. Koreeda, M., M. N. Akhtar, D. R. Boyd, J. D. Neill, D. T. Gibson, and D. M. Jerina. 1978. Absolute stereochemistry of *cis*-1,2-, *trans*-1,2-, and *cis*-3,4-dihydrodiol metabolites of phenanthrene. J. Org. Chem. 43:1023–1027.
37. Kuhm, A. E., A. Stolz, and H.-J. Knackmuss. 1991. Metabolism of naphthalene by the biphenyl-degrading bacterium *Pseudomonas paucimobilis* Q1. Biodegradation 2:115–120.
38. Laborde, A. L., and D. T. Gibson. 1977. Metabolism of dibenzothiophene by a *Beijerinckia* species. Appl. Environ. Microbiol. 34:783–790.
39. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) 227:680–685.
40. Lennox, E. S. 1955. Transduction of linked genetic characters of the host by bacteriophage P1. Virology 1:190–206.
41. Mahaffey, W. R., D. T. Gibson, and C. E. Cerniglia. 1988. Bacterial oxidation of chemical carcinogens: formation of polycyclic aromatic acids from benz[a]anthracene. Appl. Environ. Microbiol. 54:2415–2423.
42. Mondello, F. J. 1985. Regulation of aromatic hydrocarbon metabolism in a

Beijerinckia species, abstr. K 167, p. 199. *In* Abstracts of the 85th Annual Meeting of the American Society for Microbiology 1985. American Society for Microbiology, Washington, D.C.

43. Nakai, C., H. Kagamiyama, M. Nozaki, T. Nakazawa, S. Inouye, Y. Ebina, and A. Nakazawa. 1983. Complete nucleotide sequence of the metapyrocatechase gene on the TOL plasmid of *Pseudomonas putida* mt-2. J. Biol. Chem. 258:2923–2928.

44. Olsen, R. H., G. DeBusscher, and W. R. McCombie. 1982. Development of broad-host-range vectors and gene banks: self-cloning of the *Pseudomonas aeruginosa* PAO chromosome. J. Bacteriol. 150:60–69.

45. Resnick, S. M., D. S. Torok, and D. T. Gibson. 1993. Oxidation of carbazole to 3-hydroxycarbazole by naphthalene 1,2-dioxygenase and biphenyl 2,3-dioxygenase. FEMS Microbiol. Lett. 113:297–302.

46. Sala-Trepat, J. M., and W. C. Evans. 1971. The *meta* cleavage of catechol by *Azotobacter* sp.: 4-oxalocrotonate pathway. Eur. J. Biochem. 20:400–413.

47. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

48. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

49. Schocken, M. J., and D. T. Gibson. 1984. Bacterial oxidation of the polycyclic aromatic hydrocarbons acenaphthene and acenaphthylene. Appl. Environ. Microbiol. 48:10–16.

50. Simon, R., U. Priefer, and A. Pühler. 1983. A broad host range mobilization system for *in vivo* genetic engineering: transposon mutagenesis in gram-negative bacteria. Bio/Technology 1:784–791.

51. Stanier, R. Y., N. J. Palleroni, and M. Duodoroff. 1966. The aerobic pseudomonads: a taxonomic study. J. Gen. Microbiol. 43:159–271.

52. Tabor, S., and C. C. Richardson. 1987. DNA sequence analysis with a modified bacteriophage T7 DNA polymerase. Proc. Natl. Acad. Sci. USA 84:4767–4771.

53. Taira, K., N. Hayase, N. Arimura, S. Yamashita, T. Miyazaki, and K. Furukawa. 1988. Cloning and nucleotide sequence of the 2,3-dihydroxybiphenyl dioxygenase gene from the PCB-degrading strain of *Pseudomonas paucimobilis* Q1. Biochemistry 27:3990–3996.

54. Taira, K., J. Hirose, S. Hayashida, and K. Furukawa. 1992. Analysis of *bph* operon from the polychlorinated biphenyl-degrading strain of *Pseudomonas pseudoalcaligenes* KF707. J. Biol. Chem. 267:4844–4853.

55. Vogelstein, B., and D. Gillespie. 1979. Preparative and analytical purification of DNA from agarose. Proc. Natl. Acad. Sci. USA 76:615–619.

56. Yen, K.-M., and I. C. Gunsalus. 1982. Plasmid gene organization: naphthalene/salicylate oxidation. Proc. Natl. Acad. Sci. USA 79:874–878.

57. Yrjälä, K., P. Lars, S. Kilpi, and M. Romantschuk. 1994. Cloning of *cmpE*, a plasmid-borne catechol 2,3-dioxygenase-encoding gene from the aromatic and chloroaromatic-degrading *Pseudomonas* sp. HV3. Gene 138:119–121.

58. Ziffer, H., K. Kabuto, D. T. Gibson, V. M. Kobal, and D. M. Jerina. 1977. The absolute stereochemistry of several *cis*-dihydrodiols microbially produced from substituted benzenes. Tetrahedron 33:2491–2496.

59. Zylstra, G. J., and D. T. Gibson. 1989. Toluene degradation by *Pseudomonas putida* F1: nucleotide sequence of the *todC1C2BADE* genes and their expression in *Escherichia coli*. J. Biol. Chem. 264:14940–14946.

60. Zylstra, G. J., and D. T. Gibson. 1991. Aromatic hydrocarbon degradation: a molecular approach, p. 183–203. *In* J. K. Setlow (ed.), Genetic engineering: principles and methods. Plenum Press, New York.

61. Zylstra, G. J., and D. T. Gibson. Unpublished data.

62. Zylstra, G. J., X. P. Wang, E. Kim, and V. A. Didolkar. 1994. Cloning and analysis of the genes for polycyclic aromatic hydrocarbon degradation. Ann. N. Y. Acad. Sci. 721:386–398.
