
ARTICLES

# A genome-wide search for human type 1 diabetes susceptibility genes

June L. Davies, Yoshihiko Kawaguchi, Simon T. Bennett,  
James B. Copeman, Heather J. Cordell, Lynn E. Pritchard, Peter W. Reed,  
Stephen C. L. Gough, Suzanne C. Jenkins, Sheila M. Palmer,  
Karen M. Balfour, Beth R. Rowe*, Martin Farrell†, Anthony H. Barnett*,  
Stephen C. Bain* & John A. Todd

Nuffield Department of Surgery, University of Oxford, John Radcliffe Hospital, Headington, Oxford OX3 9DU, and The Wellcome Trust Centre for Human Genetics, University of Oxford, Windmill Road, Oxford OX3 7BN, UK  
*Department of Medicine/Diabetes/Endocrinology, University of Birmingham, Birmingham Heartlands Hospital, Bordesley Green East, Birmingham B9 5SS, UK  
†MRC Clinical Research Centre, Hammersmith Hospital, Du Cane Road, London W12 ONN, and The Wellcome Trust Centre for Human Genetics, University of Oxford, Windmill Road, Oxford OX3 7BN, UK

We have searched the human genome for genes that predispose to type 1 (insulin-dependent) diabetes mellitus using semi-automated fluorescence-based technology and linkage analysis. In addition to *IDDM1* (in the major histocompatibility complex on chromosome 6p21) and *IDDM2* (in the insulin gene region on chromosome 11p15), eighteen different chromosome regions showed some positive evidence of linkage to disease. Linkages to chromosomes 11q (*IDDM4*) and 6q (*IDDM5*) were confirmed by replication, and chromosome 18 may encode a fifth disease locus. There are probably no genes with large effects aside from *IDDM1*. Therefore polygenic inheritance is indicated, with a major locus at the major histocompatibility complex.

MANY common diseases such as diabetes, ischaemic heart disease, asthma and schizophrenia are likely to be aetiologically and genetically heterogeneous, where the same clinical disorder is caused by different susceptibility genes and environmental factors. The autoimmune disease type 1 diabetes has been shown to be a polygenic trait in mice¹⁻³, with the major locus encoded by the major histocompatibility complex (MHC) and at least nine other loci contributing to disease development. In humans, two chromosome regions show association with, and linkage to, type 1 diabetes: the MHC HLA region on chromosome 6p21 (*IDDM1*)⁴⁻⁵ and the insulin gene region (*INS*)⁶⁻¹¹ on chromosome 11pl5 (*IDDM2*); the *CD3E* locus on chromosome 11q23 was previously designated *IDDM2* (ref. 12) but we have shown that this gene is not significantly associated with, or linked to disease (L.E.P. and J.A.T., manuscript in preparation)). The weight of evidence for *IDDM2* has come from case-control⁶,⁹,¹⁰ and intrafamilial association⁷,⁸,¹¹ studies, but linkage has also been demonstrated⁹ and replicated¹⁰.

The degree of familial clustering of a disease (*λs*) can be estimated from the ratio of the risk for siblings of patients and the population prevalence¹³. For type 1 diabetes, *λs* is about 15 (6%/0.4%), but *IDDM1* and *IDDM2* alone do not account for the observed degree of familial clustering¹³. The *λs* for a particular disease locus can be estimated by calculating the ratio of the expected proportion of affected sibpairs sharing zero alleles identical-by-descent (IBD), which is 0.25, and the observed proportion¹³. In type 1 diabetic multiplex families (data from this study), *λs* for *IDDM1* is 3.1 (*z₀*=0.08; *z₂*, *z₁* and *z₀* are the probabilities for sharing 2, 1 or 0 alleles IBD, respectively) and the *λs* for *IDDM2* is 1.3 (*z₀*=0.19). From these *λs* values it can be calculated¹³ that *IDDM1* and *IDDM2* contribute 42% and 10%, respectively, to the familial clustering of the disease.

A significant excess of alleles shared IBD in affected sibpairs is taken as evidence of genetic linkage. This approach does not require prior knowledge of disease gene frequencies or mode of inheritance¹⁴. The efficiency of the affected sibpair method can be increased by using a maximum likelihood ratio test to obtain data from families that are not fully informative¹⁵, and by restricting the sharing probabilities to *z₀*≥0, *z₁*≤0.5 and *z₁*≥2*z₀* (ref. 16). Under these restrictions, a maximum lod score (MLS)≥2.3 (*P*≤0.001) is taken to indicate significant evidence of linkage¹⁶.

So far, successful dissection of complex traits by genome-wide linkage mapping has been restricted to experimental organisms such as tomatoes¹⁷ and mice¹,¹⁸. The characterization of microsatellite marker loci by polymerase chain reaction (PCR)¹⁹, construction of high-resolution human genetic linkage maps²⁰, and the application of fluorescence-based, automated DNA fragment-sizing technology²¹⁻²³ have now greatly facilitated human genome-wide exclusion mapping, as demonstrated here.

## Families and mapping strategy

Presence of insulitis in human diabetic pancreata²⁴, the level and frequency of autoantibodies, and the strength of the HLA association are correlated with the age-of-onset of type 1 diabetes²⁵⁻²⁷. Furthermore, the persistence of plasma C-peptide, an indicator of residual pancreatic β-cell function, is more common in cases diagnosed over the age of 16 years²⁸. To reduce the level of aetiological heterogeneity and thus increase the power of our study, recruitment of families in the United Kingdom was restricted to nuclear, multiplex families in which at least one sibling was diagnosed under age 17 years and the other under age 29 years²⁸. Ethnic origin of the families was also an important criterion²⁹. The grandparents of all UK families were Caucasian and born in the UK³⁰.

A two-stage approach³¹ was employed to map genes for type 1 diabetes²⁹. First, a total genome screen by analysis of 289 microsatellite marker loci in 96 UK sibpair families (UK 96) was carried out. Then, two additional data sets comprising 102 UK sibpair families (UK 102) and 84 USA sibpair families (USA 84) were used for further analysis of marker loci with some positive evidence of linkage to type 1 diabetes (MLS≥1,

$P<0.05)$. The USA sibpairs were selected according to our age-of-onset criterion from the Human Biological Data Interchange (HBDI) repository of type 1 diabetic multiplex families. These families were all Caucasian but of diverse ethnic origin.

As $IDDM1$ is a major locus, sibpairs were subdivided into two groups by IBD status at HLA: those sharing two HLA alleles or haplotypes IBD (that is, families showing stronger evidence of linkage to HLA) and those sharing either one or zero alleles. Sibpairs can also be subdivided according to their HLA type. This strategy is computationally simple, and is, in principle, a stepwise version of simultaneous search, which was devised to facilitate detection of genes in genetically heterogeneous traits.

### Stage 1: Analysis of 96 UK families

The microsatellite marker locus $TNFa$, located in the HLA class III region of the MHC and coincident with $IDDM1$, showed striking evidence of linkage to type 1 diabetes in the UK 96 families ($MLS=7.3; \lambda_s=2.5$; Fig. 1). MLS values approaching the magnitude of those obtained on chromosome 6p were not observed for any other chromosome (Fig. 1). Marker loci located up to 20 centimorgans (cM) away from the $TNFa$ locus were also seen to have MLS values exceeding 1.0 (Fig. 1; MLS value equal to 1.0 was chosen as an experimental threshold). Therefore, by taking a 20 cM radius on each side of every marker locus, we can exclude empirically at least 97% of the genome for genes with effects equivalent to the HLA-encoded $IDDM1$ locus. $IDDM2$ on chromosome 11p15 also showed evidence of linkage to type 1 diabetes ($TH/INS; MLS=2.1; \lambda_s=1.7$; Fig. 1). This MLS value of 2.1 in the UK 96 families is the strongest evidence to date for linkage of $IDDM2$ to type 1 diabetes, without having to rely on intrafamilial association analyses or on excluding families with parents homozygous for the $INS$ minisatellite locus.

Eighteen other chromosome regions showed evidence of linkage to type 1 diabetes in the UK 96 families, with $MLS\geq1.0$ ($P<0.05$): chromosomes 2q, 3q, 6q (three distinct regions), 7q, 8q, 10cen, 10q, 11q, 13q, 14q, 16q, 17p, 18q, 19q and X (two regions) (Fig. 1). Ten of these exceeded MLS value of 1.7 ($P<0.005$): chromosomes 3q ($D3S1303$), 6q ($ESR$ and $D6S264$), 7q ($CFTR$), 8q ($D8S556$), 10cen ($D10S193$), 11q ($FGF3$), 13q ($D13S158$) and X ($DXS991$ and $DXS999$). Chromosomes 6q and 11q were analysed further and are

---

**TABLE 1 Mapping of IDDM4 and IDDM5**

| Chr | Families | Locus | $\theta(\%)$ | Total | MLS | HLA 2 | MLS | HLA 1,0 | MLS |
|-----|----------|-------|--------------|-------|------|-------|------|---------|------|
|     |          |       |              | 1     | 0    | 1     | 0    | 1       | 0    |
| 6p  | UK 96    | $TNFa$ |              | 97    | 35   | 7.3   |      |         |      |
|     | UK 102   |       |              | 87    | 36   | 5.5   |      |         |      |
|     | USA 84   |       |              | 82    | 27   | 6.7   |      |         |      |
|     | Total    |       |              | 268   | 98   | 19.3  |      |         |      |
| 11p15 | UK 96   | $TH/INS$ |            | 107   | 67   | 2.1   | 63   | 40      | 1.3  | 44   | 27   | 1.0  |
|      | UK 102  |        |              | 103   | 89   | 0.4   | 56   | 45      | 0.3  | 47   | 40   | 0.2  |
|      | USA 84  |        |              | 77    | 71   | 0.1   | 50   | 38      | 0.6  | 26   | 32   | 0.0  |
|      | Total   |        |              | 287   | 227  | 1.6   | 169  | 123     | 1.9  | 117  | 99   | 0.7  |
| 11q  | UK 96   | $D11S903$ |            | 59    | 66   | 0.0   | 33   | 36      | 0.0  | 25   | 29   | 0.0  |
|      |         | $FGF3$ | 18           | 71    | 60   | 0.2   | 33   | 37      | 0.0  | 38   | 21   | 1.8  |
|      |         | $D11S916$ | 4           | 65    | 78   | 0.0   | 30   | 47      | 0.0  | 34   | 27   | 0.4  |
|      | UK 102  | $D11S903$ |            | 80    | 55   | 1.0   | 37   | 24      | 0.6  | 38   | 25   | 0.6  |
|      |         | $FGF3$ | 18           | 70    | 64   | 0.2   | 28   | 30      | 0.0  | 37   | 28   | 1.0  |
|      |         | $D11S916$ | 4           | 77    | 76   | 0.0   | 34   | 34      | 0.0  | 40   | 31   | 0.3  |
|      | USA 84  | $D11S903$ |            | 60    | 48   | 0.3   | 29   | 31      | 0.0  | 26   | 17   | 0.3  |
|      |         | $FGF3$ | 18           | 67    | 51   | 1.1   | 37   | 28      | 1.0  | 23   | 18   | 0.2  |
|      |         | $D11S916$ | 4           | 78    | 57   | 1.1   | 40   | 30      | 0.9  | 29   | 23   | 0.2  |
|      | Total   | $D11S903$ |            | 199   | 169  | 0.6   | 99   | 91      | 0.1  | 89   | 71   | 0.4  |
|      |         | $FGF3$ | 18           | 208   | 175  | 1.3   | 98   | 95      | 0.0  | 98   | 67   | 2.8  |
|      |         | $D11S916$ | 4           | 220   | 211  | 0.2   | 104  | 111     | 0.0  | 103  | 81   | 0.9  |
| 6q   | UK 96   | $ESR$ |              | 95    | 59   | 1.8   | 61   | 30      | 2.5  | 34   | 29   | 0.1  |
|      |         | $D6S290^*$ | 3          | 102   | 69   | 1.4   | 63   | 36      | 1.4  | 39   | 33   | 0.1  |
|      |         | $D6S305$ | 11          | 59    | 40   | 0.8   | 33   | 20      | 0.8  | 26   | 20   | 0.3  |
|      | UK 102  | $ESR$ |              | 65    | 73   | 0.0   | 30   | 29      | 0.0  | 27   | 37   | 0.0  |
|      |         | $D6S290^*$ | 3          | 72    | 76   | 0.0   | 31   | 33      | 0.0  | 30   | 34   | 0.0  |
|      |         | $D6S305$ | 11          | 77    | 62   | 0.4   | 35   | 31      | 0.1  | 30   | 28   | 0.2  |
|      | USA 84  | $ESR$ |              | 67    | 48   | 1.0   | 32   | 27      | 0.3  | 27   | 17   | 0.8  |
|      |         | $D6S290^*$ | 3          | 75    | 54   | 1.5   | 41   | 30      | 1.2  | 28   | 20   | 0.7  |
|      |         | $D6S305$ | 11          | 45    | 58   | 0.0   | 23   | 30      | 0.0  | 20   | 22   | 0.2  |
|      | Total   | $ESR$ |              | 227   | 180  | 1.5   | 123  | 86      | 1.6  | 88   | 83   | 0.1  |
|      |         | $D6S290^*$ | 3          | 249   | 199  | 1.5   | 135  | 99      | 2.0  | 97   | 87   | 0.2  |
|      |         | $D6S305$ | 11          | 181   | 160  | 0.8   | 91   | 81      | 0.2  | 76   | 70   | 0.7  |

Linkage of marker loci on chromosomes 6p ($TNFa$), 11p15 ($TH/INS$), 11q ($FGF3$ region) and 6q ($ESR$ region) to type 1 diabetes in 96 UK families, 102 additional UK families and 84 USA families. Sibpairs were subdivided into those sharing two alleles IBD at HLA ($HLA 2$) and those sharing 1 or 0 alleles IBD at HLA ($HLA 1,0$). Values of $\theta$ were calculated using LINKAGE on the data obtained from these families. $D6S290$ was made more informative by haplotyping (denoted by an asterisk) with the flanking marker loci $ESR$ and $D6S305$. It has been suggested that the critical level of significance should be increased by $\log_{10}(t)$, where $t$ is the number of tests (for example, the critical MLS value for stage 1 would be $2.3 + \log_{10}(2) = 2.6$). But, in the case for results which are positively correlated, as for our data, this adjustment may be too conservative, and a number smaller than $\log_{10}(t)$, which remains to be determined, should be added. In the absence of clear theoretical guidelines, we have chosen an unadjusted threshold value for significant evidence of linkage of $MLS\geq2.3$ (ref. 16), in addition to requiring that the initial linkage was replicated in at least one data set. DNA samples from UK and USA families are available from the BDA and the HBDI, respectively.

ARTICLES

Chromosome 1

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D1S228 | 62 57 | 41 37 | 21 19 |  |  |
| D1S234 | 73 69 | 42 41 | 30 27 |  |  |
| D1S255 | 43 61 | 27 37 | 15 23 |  |  |
| D1S197 | 71 68 | 38 39 | 33 29 |  |  |
| D1S209 | 65 74 | 43 42 | 26 32 |  |  |
| D1S216 | 76 73 | 47 48 | 33 34 |  |  |
| D1S207 | 81 82 | 39 34 | 26 33 |  |  |
| D1S252 | 67 67 | 43 42 | 18 23 |  |  |
| D1S196 | 45 59 | 26 35 | 31 29 |  |  |
| D1S238 | 75 72 | 34 35 | 27 22 |  |  |
| D1S229 | 61 57 | 26 45 | 31 19 |  |  |
| D1S103 | 58 65 |  |  |  |  |

Chromosome 2

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D2S131 | 67 71 | 45 40 | 22 29 |  |  |
| D2S165 | 65 72 | 37 40 | 28 30 |  |  |
| D2S177 | 71 87 | 41 49 | 29 38 |  |  |
| D2S134 | 70 58 | 31 50 | 28 25 |  |  |
| D2S139 | 76 63 | 37 31 | 32 27 |  |  |
| D2S160 | 69 74 | 41 38 | 33 25 |  |  |
| D2S114 | 63 56 | 34 46 | 26 21 |  |  |
| D2S142 | 73 67 | 35 35 | 37 20 |  |  |
| D2S326 | 77 67 | 39 40 | 38 26 |  |  |
| D2S152 | 66 66 | 33 39 | 32 26 |  |  |
| D2S72 | 73 78 | 38 45 | 34 32 |  |  |
| D2S157 | 64 67 | 36 40 | 27 26 |  |  |
| D2S126 | 64 55 | 38 31 | 24 24 |  |  |
| D2S206 | 68 79 | 44 40 | 24 37 |  |  |

Chromosome 3

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D3S1297 | 60 65 | 39 41 | 18 22 |  |  |
| D3S1263 | 71 55 | 34 29 | 35 22 |  |  |
| D3S11 | 74 61 | 37 34 | 35 25 |  |  |
| D3S1300 | 69 58 | 38 36 | 27 18 |  |  |
| D3S1261 | 72 60 | 43 33 | 25 26 |  |  |
| D3S1271 | 48 50 | 26 33 | 21 16 |  |  |
| D3S1303 | 57 47 | 35 21 | 1.9 25 |  |  |
| D3S1279 | 69 66 | 37 38 | 28 26 |  |  |
| D3S1282 | 47 53 | 22 36 | 18 20 |  |  |
| D3S1262 | 42 65 | 35 37 | 18 27 |  |  |
| D3S1265 | 70 69 |  | 30 30 |  |  |

Chromosome 4

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D4S412 | 66 59 | 29 37 | 35 22 |  |  |
| D4S403 | 53 68 | 27 40 | 25 28 |  |  |
| D4S418 | 68 68 | 40 36 | 26 32 |  |  |
| D4S405 | 62 54 | 29 34 | 33 20 |  |  |
| D4S398 | 65 58 | 41 35 | 28 30 |  |  |
| D4S231 | 59 69 | 32 37 | 22 31 |  |  |
| D4S250 | 53 94 | 28 50 | 24 44 |  |  |
| D4S406 | 47 100 | 25 58 | 20 42 |  |  |
| D4S430 | 69 84 | 39 45 | 29 39 |  |  |
| D4S413 | 57 76 | 39 44 | 18 32 |  |  |
| D4S415 | 51 69 | 31 33 | 19 35 |  |  |
| D4S408 | 61 48 | 32 23 | 27 25 |  |  |

Chromosome 5

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D5S405 | 46 44 | 29 23 | 17 21 |  |  |
| D5S268 | 59 62 | 33 36 | 25 26 |  |  |
| D5S416 | 56 61 | 32 37 | 23 23 |  |  |
| D5S419 | 76 71 | 41 41 | 33 30 |  |  |
| D5S426 | 60 51 | 30 32 | 28 19 |  |  |
| D5S407 | 66 86 | 39 48 | 26 37 |  |  |
| D5S253 | 57 55 | 35 32 | 22 22 |  |  |
| D5S428 | 62 54 | 40 32 | 21 21 |  |  |
| D5S409 | 35 47 | 20 29 | 14 18 |  |  |
| D5S421 | 63 51 | 37 29 | 24 22 |  |  |
| D5S210 | 59 54 | 38 28 | 20 26 |  |  |
| D5S410 | 50 56 | 26 30 | 23 25 |  |  |
| D5S400 | 72 64 | 43 34 | 28 29 |  |  |
| D5S408 | 67 63 |  | 27 27 |  |  |

Chromosome 6

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D6S470 | 61 58 | 36 26 | 25 32 |  |  |
| D6S259 | 71 45 | 51 20 | 20 24 |  |  |
| D6S260 | 99 53 | 3.9 | 28 31 |  |  |
| D6S276 | 95 43 | 71 6 | 24 37 |  |  |
| D6S258 | 90 27 | 8 | 23 27 |  |  |
| D6S273 | 97 35 | 7.6 | 20 35 |  |  |
| TNFα | 69 31 | 3.7 | 22 27 |  |  |
| D6S291 | 82 32 | 45 4 | 21 23 |  |  |
| D6S426 | 84 52 | 5.1 | 31 27 |  |  |
| D6S294 | 72 58 | 1.7 | 34 26 |  |  |
| D6S286 | 56 55 | 38 32 | 22 27 |  |  |
| D6S300 | 50 59 | 34 28 | 18 25 |  |  |
| D6S267 | 69 68 | 31 34 | 25 30 |  |  |
| D6S287 | 49 51 | 42 38 | 19 24 |  |  |
| D6S407 | 64 58 | 28 27 | 27 21 |  |  |
| D6S262 | 44 53 | 27 32 | 23 15 |  |  |
| D6S270 | 50 43 | 20 28 | 28 17 |  |  |
| D6S308 | 49 45 | 28 35 | 33 20 |  |  |
| D6S314 | 61 55 | 47 31 | 32 29 |  |  |
| D6S310 | 81 60 | 61 30 | 34 29 |  |  |
| D6S311 | 95 59 | 1.8 | 56 35 |  |  |
| ESR | 90 60 | 1.5 | 29 20 |  |  |
| D6S441 | 53 40 | 42 33 | 30 27 |  |  |
| D6S290 | 74 60 | 33 20 | 26 20 |  |  |
| D6S415 | 59 40 | 1.0 | 22 18 |  |  |
| D6S305 | 65 39 | 1.2 | 22 21 |  |  |
| D6S264 | 54 48 | 32 27 |  |  |  |

Chromosome 7

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D7S531 | 61 62 | 35 33 | 26 28 |  |  |
| D7S513 | 62 80 | 35 44 | 25 36 |  |  |
| D7S503 | 71 67 | 41 35 | 29 32 |  |  |
| D7S493 | 84 75 | 48 46 | 35 29 |  |  |
| D7S629 | 58 54 | 36 31 | 21 23 |  |  |
| D7S484 | 64 66 | 35 44 | 29 20 |  |  |
| D7S519 | 61 78 | 31 51 | 30 27 |  |  |
| D7S502 | 73 78 | 35 49 | 37 28 |  |  |
| D7S669 | 64 69 | 36 44 | 28 25 |  |  |
| D7S524 | 70 65 | 38 38 | 30 27 |  |  |
| D7S2527 | 51 58 | 22 39 | 28 18 |  |  |
| D7S486 | 54 45 | 26 33 | 27 12 |  |  |
| D7S530 | 56 44 | 28 30 | 27 14 |  |  |
| D7S684 | 68 72 | 34 45 | 33 26 |  |  |
| D7S483 | 67 73 | 38 45 | 29 27 |  |  |
| D7S550 | 67 61 | 31 32 | 36 29 |  |  |

Chromosome 8

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D8S504 | 36 53 | 23 31 | 13 21 |  |  |
| D8S552 | 58 71 | 32 45 | 26 26 |  |  |
| D8S261 | 48 57 | 27 41 | 20 15 |  |  |
| D8S258 | 32 58 | 16 31 | 15 26 |  |  |
| D8S283 | 67 64 | 33 38 | 32 26 |  |  |
| D8S285 | 43 44 | 23 26 | 18 18 |  |  |
| D8S260 | 55 56 | 28 32 | 25 24 |  |  |
| D8S286 | 69 59 | 38 30 | 30 29 |  |  |
| D8S273 | 75 59 | 1.2 | 1.3 25 |  |  |
| D8S88 | 82 64 | 50 30 | 30 34 |  |  |
| D8S257 | 69 51 | 1.2 | 40 27 |  |  |
| D8S556 | 71 38 | 2.6 | 27 19 |  |  |
| D8S281 | 62 52 | 39 28 | 21 24 |  |  |
| D8S198 | 64 50 | 1.0 | 17 21 |  |  |
| D8S284 | 67 69 | 37 37 | 29 32 |  |  |
| D8S272 | 56 52 | 34 25 | 21 26 |  |  |

Chromosome 9

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D9S144 | 61 62 | 36 38 | 25 21 |  |  |
| D9S168 | 69 68 | 36 39 | 32 29 |  |  |
| D9S156 | 67 61 | 45 32 | 22 28 |  |  |
| D9S171 | 65 69 | 40 37 | 24 32 |  |  |
| D9S147E | 60 68 | 38 37 | 21 30 |  |  |
| D9S43 | 70 57 | 42 30 | 27 26 |  |  |
| D9S15 | 50 54 | 26 36 | 23 18 |  |  |
| D9S175 | 61 78 | 31 44 | 28 34 |  |  |
| D9S257 | 73 85 | 47 44 | 25 40 |  |  |
| D9S176 | 75 68 | 43 40 | 32 27 |  |  |
| D9S155 | 51 39 | 26 25 | 24 13 |  |  |
| D9S154 | 51 50 | 30 29 | 19 17 |  |  |
| D9S120 | 51 49 | 29 37 | 21 11 |  |  |
| D9S118 | 68 70 | 34 44 | 33 25 |  |  |
| D9S159 | 63 76 | 37 34 | 27 31 |  |  |
| D9S158 | 65 55 | 33 34 | 25 26 |  |  |

Chromosome 10

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D10S189 | 38 50 | 24 22 | 13 28 |  |  |
| D10S191 | 76 58 | 46 32 | 30 26 |  |  |
| D10S197 | 63 61 | 28 41 | 35 19 |  |  |
| D10S193 | 71 48 | 37 33 | 34 14 |  |  |
| D10S220 | 82 63 | 1.3 | 42 38 |  |  |
| D10S539 | 44 57 | 27 37 | 35 37 |  |  |
| D10S201 | 65 76 | 46 35 | 33 34 |  |  |
| D10S192 | 68 66 | 33 34 | 28 39 |  |  |
| D10S190 | 67 57 | 24 32 | 33 18 |  |  |
| D10S217 | 62 57 |  | 18 20 |  |  |
| D10S212 | 42 53 |  |  |  |  |

Chromosome 11

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  |  |  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D11S922 | 101 72 | 58 40 | 40 30 |  |  |
| TH | 77 47 | 47 27 | 29 19 |  |  |
| TH/INS | 107 67 | 53 40 | 44 27 |  |  |
| D11S569 | 50 41 | 32 25 | 16 14 |  |  |
| D11S899 | 55 57 | 29 36 | 23 21 |  |  |
| D11S904 | 72 81 | 42 46 | 27 33 |  |  |
| D11S907 | 58 65 | 34 31 | 24 32 |  |  |
| D11S903 | 59 66 | 33 36 | 25 29 |  |  |
| FGF3 | 71 60 | 33 37 | 38 21 |  |  |
| D11S916 | 65 78 | 30 47 | 34 27 |  |  |
| D11S901 | 62 76 | 36 44 | 24 29 |  |  |
| D11S35 | 42 50 | 25 33 | 16 15 |  |  |
| D11S927 | 72 84 | 46 43 | 26 36 |  |  |
| D11S925 | 69 69 | 39 37 | 22 30 |  |  |
| D11S934 | 70 61 | 30 21 | 20 24 |  |  |
| D11S910 | 51 61 | 28 36 | 20 24 |  |  |
| D11S
Chromosome 12

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D12S99 | 52 76 | 29 47 | 23 27 |  |
| D12S77 | 68 75 | 40 46 | 28 28 |  |
| D12S358 | 57 76 | 32 44 | 25 30 |  |
| D12S364 | 66 74 | 42 44 | 24 30 |  |
| D12S62 | 71 71 | 47 44 | 21 23 |  |
| D12S87 | 53 49 | 32 26 | 27 24 |  |
| D12S368 | 70 56 | 43 32 | 15 25 |  |
| D12S83 | 48 61 | 32 35 | 20 27 |  |
| D12S43 | 60 66 | 39 39 | 21 29 |  |
| D12S92 | 51 67 | 29 38 | 24 38 |  |
| D12S95 | 59 76 | 33 38 | 22 24 |  |
| D12S79 | 49 63 | 27 39 | 24 21 |  |
| D12S366 | 68 58 | 43 36 | 27 31 |  |
| D12S86 | 67 56 | 39 53 | 24 19 |  |
| D12S342 | 62 56 | 34 36 | 14 19 |  |
| D12S97 | 39 52 | 24 33 |  |  |

Chromosome 13

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D13S192 | 95 74 | 58 44 | 37 30 |  |
| D13S120 | 78 59 | 49 36 | 29 23 |  |
| D13S153 | 87 77 | 50 47 | 37 30 |  |
| D13S170 | 85 78 | 47 52 | 38 26 |  |
| D13S122 | 71 55 | 36 37 | 35 18 | 1.2 |
| D13S158 | 84 54 | 46 37 | 38 17 | 2.4 |
| D13S173 | 55 55 | 32 40 | 23 15 |  |

Chromosome 14

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D14S72 | 54 62 | 32 36 | 22 26 |  |
| D14S50 | 50 55 | 30 33 | 18 22 |  |
| D14S80 | 69 64 | 38 36 | 29 28 |  |
| D14S49 | 81 72 | 45 42 | 36 30 | 1.3 |
| D14S70 | 64 43 | 31 28 | 22 25 |  |
| D14S75 | 71 57 | 47 32 | 17 15 |  |
| D14S63 | 45 44 | 27 29 | 25 27 |  |
| D14S74 | 59 72 | 33 44 | 29 26 |  |
| D14S68 | 66 58 | 36 32 | 25 32 |  |
| D14S51 | 70 79 | 43 47 | 29 34 |  |
| D14S267 | 71 77 |  |  |  |

Chromosome 15

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D15S128 | 55 57 | 36 34 | 18 23 |  |
| D15S118 | 64 64 | 33 36 | 30 26 |  |
| CYP19 | 60 47 | 39 27 | 21 20 |  |
| D15S117 | 58 55 | 34 33 | 22 22 |  |
| D15S114 | 48 45 | 29 27 | 18 17 |  |
| D15S127 | 55 77 | 32 45 | 21 32 |  |

Chromosome 16

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D16S423 | 63 60 | 36 36 | 27 23 |  |
| D16S407 | 76 80 | 38 50 | 37 29 |  |
| D16S287 | 65 56 | 31 36 | 33 19 |  |
| D16S420 | 67 68 | 32 42 | 34 26 |  |
| D16S261 | 60 61 | 28 34 | 31 26 |  |
| D16S411 | 64 64 | 23 23 | 26 17 |  |
| D16S415 | 65 49 | 31 40 | 32 23 |  |
| D16S320 | 77 73 | 31 31 | 33 17 | 1.4 |
| D16S265 | 62 81 | 42 44 | 34 28 |  |
| D16S503 | 69 79 | 31 47 | 31 25 |  |
| D16S515 | 48 54 | 39 41 | 29 38 |  |
| D16S516 | 51 58 | 27 29 | 20 24 |  |
| D16S289 | 66 68 | 28 32 | 22 25 |  |
| D16S422 | 67 49 | 34 26 | 28 32 |  |
| D16S520 |  | 32 22 | 32 22 |  |

Chromosome X

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| DXS996 | 43 31 | 23 16 | 18 14 |  |
| DXS999 | 33 26 | 24 10 | 9 15 |  |
| DXS451 | 34 32 | 19 17 | 14 13 |  |
| DMD | 38 35 | 20 18 | 15 17 |  |
| DXS538 | 40 35 | 23 17 | 14 18 |  |
| DXS1068 | 32 24 | 13 15 | 17 9 |  |
| DXS993 | 30 32 | 16 18 | 12 14 |  |
| MAOB | 42 32 | 20 20 | 20 4 | 3.2 |
| DXS991 | 38 18 | 1.4 | 16 14 | 20 9 | 1.2 |
| AR | 36 28 | 13 19 | 15 18 |  |
| DXS990 | 26 35 | 11 15 | 11 18 |  |
| DXS984 | 24 43 | 13 23 | 11 18 |  |
| DXS998 | 20 15 | 12 8 | 7 6 |  |

Chromosome 17

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D17S926 | 53 37 | 28 20 | 24 16 | 1.1 |
| D17S513 | 60 41 | 36 19 | 23 21 |  |
| D17S786 | 59 51 | 40 28 | 19 23 |  |
| D17S799 | 55 60 | 34 37 | 20 22 |  |
| D17S953 | 33 49 | 26 26 | 7 23 |  |
| D17S798 | 47 68 | 30 32 | 17 34 |  |
| D17S250 | 71 68 | 40 43 | 30 25 |  |
| D17S934 | 61 58 | 40 34 | 21 24 |  |
| HOX2B | 36 36 | 21 21 | 15 15 |  |
| D17S787 | 70 69 | 39 39 | 30 29 |  |
| D17S807 | 66 67 | 41 37 | 24 29 |  |
| D17S789 | 59 55 | 37 31 | 21 23 |  |
| D17S939 | 70 59 | 43 38 | 26 21 |  |
| D17S784 | 66 45 | 29 18 |  |  |

Chromosome 18

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D18S59 | 78 63 | 48 33 | 29 29 |  |
| D18S52 | 56 49 | 31 27 | 25 22 |  |
| D18S62 | 54 49 | 33 28 | 20 20 |  |
| D18S53 | 60 58 | 30 27 | 30 29 |  |
| D18S71 | 84 74 | 49 40 | 35 32 |  |
| D18S57 | 69 66 | 37 37 | 31 28 |  |
| D18S64 | 72 55 | 33 34 | 38 21 | 1.5 |
| D18S61 | 73 88 | 38 52 | 35 35 |  |

Chromosome 19

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D19S221 | 74 60 | 42 35 | 32 25 |  |
| D19S49 | 70 64 | 44 35 | 26 27 |  |
| D19S225 | 57 61 | 31 38 | 26 23 |  |
| D19S220 | 69 65 | 42 41 | 27 24 |  |
| D19S217 | 48 44 | 29 24 | 18 19 |  |
| APOC2 | 51 47 | 31 28 | 20 19 |  |
| D19S210 | 48 44 | 27 25 | 21 19 |  |

Chromosome 20

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D20S186 | 62 66 | 39 35 | 23 31 |  |
| D20S112 | 71 70 | 39 38 | 31 31 |  |
| D20S200 | 54 58 | 37 29 | 17 28 |  |
| D20S106 | 57 56 | 37 31 | 20 25 |  |
| D20S107 | 51 55 | 31 33 | 20 22 |  |
| D20S120 | 60 71 | 37 44 | 23 27 |  |
| D20S171 | 48 58 | 34 36 | 13 21 |  |
| D20S93 | 72 71 | 48 38 | 23 32 |  |

Chromosome 21

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D21S258 | 63 68 | 33 43 | 30 25 |  |
| D21S265 | 55 60 | 30 31 | 25 27 |  |
| D21S219 | 48 62 | 20 41 | 28 21 |  |
| D21S65 | 68 69 | 32 47 | 36 22 |  |
| D21S167 | 61 53 | 33 36 | 27 17 |  |
| D21S270 | 65 69 | 29 43 | 35 25 |  |
| D21S268 | 61 72 | 29 45 | 31 27 |  |
| D21S198 | 30 30 | 16 15 | 14 15 |  |

Chromosome 22

| MARKER LOCUS | TOTAL DATA | HLA SPLIT |
| --- | --- | --- |
|  | 1 0 IBD | MLS | 2 SHARE | 1/0 SHARE |
|  | 1 0 IBD | MLS | 1 0 IBD | MLS |
| D22S264 | 64 65 | 40 38 | 22 27 |  |
| D22S156 | 59 47 | 38 25 | 20 22 |  |
| D22S277 | 30 34 | 23 22 | 7 12 |  |
| CYP2D | 54 45 | 32 28 | 22 16 |  |
| D22S274 | 68 63 | 46 36 | 21 26 |  |

FIG. 1 Genome-wide search for type 1 diabetes susceptibility genes by linkage analysis of 290 marker loci in 96 UK affected sibpair families. The data for sharing 1 or 0 alleles IBD, summed over maternal and paternal meioses, are shown at each microsatellite locus; families were subdivided according to their IBD status at HLA (sibpairs sharing two alleles at HLA IBD:HLA SPLIT 2 share, and sibpairs sharing 1 or 0 alleles at HLA IBD:HLA SPLIT 1/0 share). MLS values were based on the 2,1,0 sharing of alleles IBD at each marker and are shown for values ≥1.0 (P<0.05). As described previously¹⁵, families in which the IBD status could not be determined unambiguously (scored as 0 or 1 sharers, 1 or 2 sharers, and 0 or 2 sharers) also contributed to the MLS values. MLS values for the X chromosome were calculated according to the sex of the sibpair and exact significance levels for these remain to be calculated by simulation studies. The linkage map was constructed using genotyping data from the UK 96 families and fluorescence-based chromosome-specific microsatellite sets²³. Interlocus distances are shown as sex-averaged, % recombination fractions (θ) (except for θ≤0.01) when the evidence for linkage, by two-point analysis using LINKAGE, exceeded Lod 3. For the locus-proterminus regions and where the evidence for linkage between two markers was less than Lod 3, the value of θ (denoted by an asterisk) was taken from other maps²⁰,⁴⁶⁻⁴⁸. The most likely position for D3S11 at 3p14-p21 (ref. 49) is between D3S1263 and D3S1300. The marker locus TH/INS was a combination of the marker TH, a tetranucleotide microsatellite in the gene encoding tyrosine hydroxylase⁵⁰, 10 kb from the INS structural gene, and the INS minisatellite locus⁶, which was typed using standard blotting techniques⁶. Results files for all families showing double recombinations in regions of θ≤0.3 were re-checked. The accuracy of fluorescence-based genotyping²² was confirmed by comparison with genotypes generated using conventional methods at HLA and at INS, by typing the FGF3 microsatellite locus twice using two different sets of primers (two errors in 384 genotypes), and by the fact that our linkage map agrees closely with that of Généthon.

reported here. The others are currently being followed up by analysis of additional families and markers.

### Stage 2: Mapping of IDDM4 and IDDM5

The IDDM1-linked TNFa locus showed strong evidence of linkage to type 1 diabetes in all three data sets (UK 96, UK 102 and USA 84; Table 1). However, IDDM2 (TH/INS locus) did not show significant linkage to disease in either the UK 102 families (MLS = 0.4; λs = 1.0) or in USA 84 families (MLS = 0.1; λs = 1.2; Table 1).

In all families combined (UK 96, UK 102 and USA 84), FGF3 on chromosome 11q was linked to disease (MLS = 1.3; Table 1). However, the most significant evidence of linkage was obtained in the subset of families sharing 1 or 0 alleles IBD at HLA (MLS = 2.8; P < 0.001; λs = 1.2; Table 1). Six additional polymorphic microsatellite marker loci in the FGF3 region were analysed, giving a peak MLS of 3.4 at FGF3 (P < 0.0001; λs = 1.3; Fig. 2, and data not shown). A disease locus, designated IDDM4, may lie in a 14-cM region, between D11S1253 and D11S1314 (Fig. 2).

In the UK 96 families, the MLS for the chromosome 6q marker locus ESR was 1.8 (P < 0.005), and MLS = 2.5 (P < 0.001) was obtained in the families sharing two alleles IBD at HLA (Fig. 1). Linkage of chromosome 6q to disease was also obtained in the USA families (MLS values for ESR and D6S290 were 1.0 and 1.5, respectively; Table 1), thereby replicating the initial finding. There was no support, however, for linkage in the UK 102 families (MLS = 0; Table 1). The combined results from all data sets provided strong, albeit not significant support for a disease locus in this region (D6S290 MLS = 2.0; P < 0.005 in the category sharing two HLA alleles; Table 1). Nevertheless, given that we did obtain an MLS ≥ 2.3 in the initial data set of 96 UK families, and then obtained further support for linkage in the independent data set of 84 USA families, the results strongly implicate the involvement of this region of chromosome 6q in type 1 diabetes. The locus has been designated IDDM5.

---

**TABLE 2** Linkage of IDDM4 and IDDM5 by HLA-DR type

| Families | Locus | DR3/4 | MLS | DR3 | MLS |
|----------|-------|-------|-----|-----|-----|
|          |       | 1     | 0   | 1   | 0   |
| UK 96    | FGF3  | 30    | 34  | 15  | 0.0 |
| UK 102   |       | 38    | 40  | 22  | 0.0 |
| USA 84   |       | 38    | 16  | 13  | 2.2 |
| Total    |       | 106   | 90  | 50  | 0.6 |
| UK 96    | ESR   | 39    | 17  | 16  | 1.8 |
| UK 102   |       | 25    | 28  | 13  | 0.0 |
| USA 84   |       | 25    | 14  | 13  | 0.7 |
| Total    |       | 89    | 59  | 42  | 1.3 |

| Families | Locus | DR4 | MLS | Others | MLS |
|----------|-------|-----|-----|--------|-----|
|          |       | 1   | 0   | 1      | 0   |
| UK 96    | FGF3  | 35  | 34  | 10     | 0.1 |
| UK 102   |       | 33  | 29  | 7      | 0.4 |
| UK 84    |       | 30  | 36  | 8      | 0.0 |
| Total    |       | 98  | 99  | 25     | 0.2 |
| UK 96    | ESR   | 33  | 28  | 7      | 0.1 |
| UK 102   |       | 21  | 27  | 4      | 0.0 |
| UK 84    |       | 25  | 28  | 4      | 0.0 |
| Total    |       | 79  | 83  | 15     | 0.0 |

The 1,0 IBD sharing data at the IDDM4-linked locus FGF3 and the IDDM5-linked locus ESR in affected sibpairs according to HLA-DR type are shown. In the HLA-DR3/4 category both siblings were DR3/4 (identical-by-state). In the DR3 category both siblings were DR3-positive; sibpairs in which both siblings were DR3/4 heterozygotes, and sibpairs in which both siblings were DR4-positive were not included in this category. Similarly, the DR4 category contained only those sibpairs in which both siblings were DR4-positive but not DR3/4. The Others category contained the remaining sibpairs. MLS values were calculated as described (ref. 15, Fig. 1 legend).

Linkage of IDDM4 and of IDDM5 to disease was analysed according to HLA-DR types of the sibpairs (Table 2). In all three family data sets combined, there was no significant evidence of heterogeneity (P > 0.05; evaluated by χ² contingency table analysis) in the linkage of FGF3 or ESR to disease by HLA type. However, there was heterogeneity at IDDM4-linked locus FGF3 by HLA type in the USA families (χ² = 8.17; 3 d.f.; P = 0.04), with FGF3 showing the strongest evidence of linkage in the sibpairs who were both HLA-DR3/4 (MLS = 2.2; P < 0.005). These data indicate that the linkage of chromosome 11q to type 1 diabetes differs between USA and UK families in the HLA-DR3/4 category (χ² = 8.0; 2 d.f.; P = 0.02), and, at least in USA sibpairs, according to their HLA-DR status. At the IDDM5-linked locus ESR, there was also evidence for heterogeneity between family data sets in the HLA-DR3/4 category (χ² = 6.08; 2 d.f.; P = 0.05), and the USA families according to HLA type (χ² = 8.43; 3 d.f.; P = 0.04), with the strongest evidence of linkage in the sibpairs who were both HLA-DR3 positive (MLS = 2.1; P < 0.005).

### Discussion

We have searched the human genome for type 1 diabetes susceptibility genes using 96 affected sibpairs and a linkage map of 290 marker loci (Fig. 1). The fluorescence-based genome linkage map had an average spacing of 11 cM (35 additional microsatellites were added to the index map described previously²³) and the marker loci had a mean heterozygosity of 0.8. Although gaps exist in the map, particularly at proterminal regions, only about 3% of the genome remains to be covered at a 20-cM resolution. Simulation analyses revealed that there is a 99.9% probability of obtaining MLS ≥ 2.3 for a locus with λs = 2.5 in 96 affected sibpair families using these markers. It seems unlikely, therefore, that another locus with an effect equivalent to IDDM1 exists. For a locus with an effect equivalent to IDDM2 (λs = 1.7 in UK 96), there is a 46% probability of obtaining MLS ≥ 1.0, and a 12% probability of obtaining MLS ≥ 2.3. Thus, we have prob-

FIG. 2 Location of IDDM4 on chromosome 11q. MLS values were obtained by analysis of all families (UK 96, UK 102 and USA 84) with both siblings diagnosed under the age of 17 years and sharing HLA 1,0 alleles IBD (96 sibpairs). 88%, 75% and 87% of the UK 96, UK 102 and USA 84 families, respectively, had both siblings diagnosed under age 17 years. Dotted lines indicate the one MLS unit support interval for IDDM4. The informativeness of the marker loci was increased by haplotyping, which made FCER1B fully informative in 70% of the families, D11S1253 88%, D11S480 76%, D11S987 84%, FGF3 87%, D11S1314 86%, and D11S916 88%. The flanking markers D11S903 and D11S534 were fully informative in 68 and 79% of families, respectively. MLS values were calculated from the haplotyped data. This approach is simple and in fully genotyped families provides at least as much power as multipoint methods based on joint likelihood calculations. Values of θ were calculated based on genotyping data from all the diabetic families using the LINKAGE program.

ably not detected all of the disease loci in the \(\lambda_{s}=1.2-1.8\) range. It is also noted that a marker locus with heterozygosity of 0.8 situated only 3 cM from \(IDDM2\) would not have shown MLS \(\geqslant 1.0\) in the UK 96 families. Therefore, to identify all the major genes for type 1 diabetes by linkage analysis, a map with markers at 3-cM intervals will be required, as well as several hundred sibpairs. These considerations highlight the advantages of incorporating intrafamilial association studies\(^{11}\) (or well designed case-control studies\(^{31}\)) and candidate gene analyses into searches for genes influencing complex traits.

Owing to the failure to replicate linkage data in psychiatric diseases\(^{31,34,35}\), confirmation of the initial findings is essential as it is extremely unlikely that false-positive results would be replicated in additional data sets. In the present study, therefore, three different sets of sibpair families, one of which, from the USA, was collected independently, have been analysed to verify linkages on chromosomes 6q and 11q. Support for linkage of \(FGF3\) and flanking markers on chromosome 11q to type 1 diabetes was found in the UK 96, UK 102 and USA 84 families (MLS = 3.4; \(P < 0.0001\), \(IDDM4\); Table 1 and Fig. 2). Evidence of linkage to chromosome 6q near the marker loci \(ESR\) and \(D6S290\) was found in the UK 96 families and also in the USA 84 families (MLS = 2.0; \(P < 0.005\), \(IDDM5\); Table 1). We did not find linkage of disease to \(IDDM5\) in the UK 102 families, nor was \(IDDM2\) linked to disease in the UK 102 and USA 84 families, even though the mapping of \(IDDM2\) in type 1 diabetes is well established by both intrafamilial association studies\(^{7,8,11}\) and linkage analyses\(^{9,10}\). Problems with ascertainment, diagnostic criteria, aetiological heterogeneity and failure to correct for the testing of multiple genetic models, have all been blamed for failure to replicate linkages in other family data sets\(^{34,35}\). It has been shown\(^{34}\) by computer simulation that if a complex trait is caused by, for example, six loci with equal effects, then the number of families required to achieve replication is approximately five times the number of families required for the first detection (\(N - 1\), where \(N\) is the number of loci). This may help explain why we have failed to replicate the results for \(IDDM2\) and \(IDDM5\) in all three data sets.

\(IDDM4\) on chromosome 11q would not have been detected if we had not evaluated linkage of markers to disease simultaneously with linkage to \(IDDM1\). Linkage of chromosome 11q to type 1 diabetes was probably not detected previously\(^{36}\) because this approach was not taken. More detailed localization of \(IDDM4\) within the 14-cM region of chromosome 11q (Fig. 2) will require the analysis of additional microsatellite loci and other polymorphisms by testing for associations of marker alleles with disease in families. This should be feasible, given the current and improving status of the genetic\(^{20}\) and physical\(^{37}\) maps, and the advantages of the fluorescence-based technology\(^{23}\).

How many of the 18 MLS values \(\geqslant 1.0\) represent true effects (ignoring the additional data for \(ESR\) and \(FGF3\)) and how many are false positives due to chance (ref. 33)? The proportion of potential false positives can be estimated from our own data set. During stage 1 (Fig. 1) it was noted that at nine unlinked marker loci more sibpairs shared zero alleles IBD than one allele (\(P < 0.05\), or negative quasi-MLS \(\geqslant 1.0\); Fig. 3). As these negative distortions have no plausible biological meaning they must represent random events. Only one negative sharing locus, \(D4S430\), exceeded a quasi-MLS value of 1.8 (\(P < 0.005\)) in the UK 96 families. This, as expected, was not replicated in the UK 102 or USA 84 families (Fig. 3 legend). In contrast, there were seven loci with positive MLS values > 1.8. These data suggest that many of the loci with positive MLS values less than 1.8 may be false, whereas the majority of loci with MLS values > 1.8 are probably indicative of genuine susceptibility genes.

Some of the linked marker loci occur in regions that contain candidate genes. Notably, \(D2S326\) and \(D10S193\) are near the \(GAD1\) and \(GAD2\) genes, respectively, which encode the enzyme glutamic acid decarboxylase, a pancreatic \(\beta\)-cell autoantigen in type 1 diabetes. \(ESR\) on chromosome 6q maps close to the gene

FIG. 3 Distributions of MLS values for marker loci showing positive (filled bars) and negative (hatched bars) quasi-MLS values \(\geqslant 1.0\) in the UK 96 families (data from Fig. 1). The negative quasi-MLS values for the nine marker loci showing more 0 sharing than 1 sharing IBD, were estimated by dividing the \(\chi^2\) values by \(2 \times \ln 10\). \(\chi^2\) values were calculated from the 1,0 distributions given in Fig. 1 against an expected distribution of 1:1. The nine marker loci were \(D1S103\), \(D3S1262\), \(D4S408\), \(D4S430\), \(D7S486\), \(D7S519\), \(D8S258\), \(D10S189\) and \(D21S219\). In Fig. 1 MLS values for these markers were 0.0 because they exceeded the boundaries of the possible sharing probabilities. \(D4S430\), which had a negative quasi-MLS of 4.1 in the UK 96 families, was analysed in the UK 102 and the USA 84 families, and, as predicted for a chance event, no distortions of the sharing of alleles IBD were observed (the 1,0 sharing distributions for the UK 102 and USA 84 families were, 83,79, and 67,70, respectively).

\(SOD2\), encoding superoxide dismutase, which may relate to the sensitivity of \(\beta\)-cells to oxygen free radical damage. A restriction-fragment length polymorphism of the \(SOD2\) gene has been associated with type 1 diabetes\(^{38}\). Two regions that showed linkage to disease, on chromosomes 2q and 17p, possibly contain the homologues of the mouse insulin-dependent diabetes genes, \(Idd5\) and \(Idd4\), respectively\(^{1,2,39}\). The microsatellite locus \(D18S64\), which in the UK 96 families showed evidence of linkage to type 1 diabetes (Fig. 1; MLS = 1.1), is in the same region of chromosome 18 as the Kidd blood group locus\(^{40}\). Linkage to Kidd blood group was obtained over 10 years ago (initially a Lod score of 2.8 was obtained using the LIPID program under several different genetic models)\(^{41,42}\), and, although a second independent study failed to replicate the linkage\(^{43}\), a population-based association between Kidd blood group and type 1 diabetes was reported\(^{44}\). Taken together with the linkage we have observed on chromosome 18, the data suggest that a susceptibility gene lies very close to or at the Kidd blood group locus.

The transmission of human type 1 diabetes is clearly complex. Indeed it appears from this study that a proportion of the familial clustering unaccounted for by \(IDDM1\) and \(IDDM2\) is caused by at least three other genes.

Received 26 May; accepted 12 July 1994.

1. Todd, J. A. et al. Nature **351**, 542–547 (1991).
2. Ghosh, S. et al. Nature Genet. **4**, 404–409 (1993).
3. Risch, N., Ghosh, S. & Todd, J. A. Am. J. hum. Genet. **53**, 702–714 (1993).
4. Morton, N. E. et al. Am. J. hum. Genet. **35**, 201–213 (1983).
5. Todd, J. A. Diabetic Med. **11**, 6–16 (1994).
6. Bell, G. I., Horita, S. & Karam, J. H. Diabetes **33**, 176–183 (1984).
7. Thomson, G., Robinson, W. P., Kuhner, M. K., Joe, S. & Klitz, W. Genet. Epidemiol. **6**, 155–160 (1989).
8. Owerbach, D., Gunn, S. & Gabbay, K. H. Diabetes **39**, 1504–1509 (1990).
9. Julier, C. et al. Nature **354**, 155–159 (1991).
10. Bain, S. C. et al. Nature Genet. **2**, 212–215 (1992).

LETTERS TO NATURE

11. Spielman, R. S., McGinnis, R. E. & Ewens, W. J. Am. J. hum. Genet. 52, 506–516 (1993).
12. Junien, C. & van Heyningen, V. Cytogenet. cell. Genet. 58, 459–554 (1991).
13. Risch, N. Am. J. hum. Genet. 40, 1–14 (1987).
14. Penrose, L. Ann. Eugenics 18, 120–124 (1953).
15. Risch, N. Am. J. hum. Genet. 46, 242–253 (1990).
16. Holmans, P. Am. J. hum. Genet. 52, 362–374 (1993).
17. Paterson, A. H. et al. Nature 335, 721–726 (1988).
18. Rise, M. L., Frankel, W. N., Coffin, J. M. & Seyfried, T. N. Science 253, 669–673 (1991).
19. Weber, J. L. & May, P. E. Am. J. hum. Genet. 44, 388–396 (1989).
20. Gyapay, G. et al. Nature Genet. 7, 246–339 (1994).
21. Ziegler, J. S. et al. Genomics 14, 1026–1031 (1992).
22. Schwengel, D. A., Jedlicka, A. E., Nanthakumara, E. J., Weber, J. L. & Levitt, R. C. Genomics 22, 46–54 (1994).
23. Reed, P. W. et al. Nature Genet. 7, 390–395 (1994).
24. Foulis, A. K., Liddle, C. N., Farquharson, M. A., Richmond, J. A. & Weir, R. S. Diabetologia 29, 267–274 (1986).
25. Caillat-Zucman, S. et al. J. clin. Invest. 90, 2242–2250 (1992).
26. Vandewalle, C. L. et al. Diabetologia 36, 1155–1162 (1993).
27. Karjalainen, J., Salmela, P., Ilonen, J., Surcel, H. M. & Knip, M. New Engl. J. Med. 320, 881–86 (1989).
28. Eff, C., Faber, O. & Deckert, T. Diabetologia 15, 169–172 (1978).
29. Todd, J. A. & Bain, S. C. Diabetes 41, 1029–1034 (1992).
30. Bain, S. C., Todd, J. A. & Barnett, A. H. Autoimmunity 7, 83–85 (1990).
31. Elston, R. C. in Genetic Approaches to Mental Disorders (eds Gershon, E. S. & Cloninger, C. R.) 3–21 (American Psychiatric Press, Washington DC, 1994).
32. Lernmark, A. et al. Am. J. hum. Genet. 47, 1028–1030 (1990).
33. Lander, E. & Botstein, D. Genetics 121, 185–199 (1989).

34. Suarez, B. K., Hampe, C. L. & Van Eerdewegh, P. in Genetic Approaches to Mental Disorders (eds Gershon, E. S. & Cloninger, C. R.) 23–46 (American Psychiatric Press, Washington DC, 1994).
35. Risch, N. Genet. Epidemiol. 7, 3–16 (1990).
36. Hyer, R. N. et al. Am. J. hum. Genet. 48, 243–257 (1991).
37. Cohen, D., Chumakov, I. & Weissenbach, J. Nature 366, 698–701 (1993).
38. Pociot, F., Johansen, J. & Nerup, J. Autoimmunity (in the press).
39. Cornell, R. J. et al. Nature 353, 262–265 (1991).
40. Le Beau, M. & Geurts van Kessel, A. Cytogenet. cell. Genet. 58, 739–750 (1991).
41. Hodge, S. E. et al. Lancet II, 893–895 (1981).
42. Hodge, S. E., Anderson, C. E., Neiswanger, K., Sparkes, R. S. & Rimoin, D. L. Am. J. hum. Genet. 35, 1139–1155 (1983).
43. Dunsworth, T. S., Rich, S. S., Swanson, J. & Barbosa, J. Diabetes 31, 991–993 (1982).
44. Barbosa, J., Rich, S. S., Dunsworth, T. S. & Swanson, J. J. clin. Endocrin. Metab. 55, 193–195 (1982).
45. Risch, N. Am. J. hum. Genet. 48, 1058–1064 (1991).
46. NIH/CEPH Collaborative Group Science 258, 67–86 (1992).
47. Matise, T. C., Perlin, M. & Chakravarti, A. Nature Genet. 6, 384–390 (1994).
48. Buetow, K. H. et al. Nature Genet. 6, 391–393 (1994).
49. Brett, P. M., Melmer, G. & Gurling, H. M. D. Nucleic Acids Res. 19, 6978 (1991).
50. Hearne, C. M., Ghosh, S. & Todd, J. A. Trends Genet. 8, 288–294 (1992).

ACKNOWLEDGEMENTS. J.L.D. and Y.K. contributed equally to this work. We thank the Wellcome Trust, the Medical Research Council, the British Diabetic Association, the Juvenile Diabetes Foundation International, and Lilly Industries UK for financial support; M. Murphy, S. Howell, J. Carr-Smith and M. Pizzey for their contributions to the collection of UK families; J. Sowinski, L. Ducat and the HBDI for the USA families; and N. Risch for advice and encouragement. Y.K. is a Wellcome Trust Overseas Postdoctoral Fellow and J.A.T. is a Wellcome Trust Principal Research Fellow.

---

LETTERS TO NATURE

**Carbon and the formation of reduced chondrules**

Harold C. Connolly Jr*, Roger H. Hewins*, Richard D. Ash†, Brigitte Zanda‡, Gary E. Lofgren§ & Michèle Bourot-Denise‡

* Department of Geological Sciences, Rutgers University, New Brunswick, New Jersey 08903, USA  
† Department of Geology, University of Manchester, Manchester M13 9PL, UK  
‡ Laboratoire de Minéralogie, Muséum National d'Histoire Naturelle, 61 Rue Buffon, 75005 Paris and Institut d'Astrophysique Spatiale, Orsay, France  
§ SN-4, NASA-JSC, Houston, Texas 77058, USA  

CHONDRULES are millimetre-sized spheroidal bodies composed mainly of olivine and orthopyroxene, which comprise the dominant fraction of most chondritic meteorites. They are the products of partial melting of aggregates of fine-grained silicates with minor contributions from metals, sulphides and oxides. Although the formation conditions of chondrules are not well understood, these are thought to involve a transient melting event in the solar nebula. The ubiquity of reduced carbon in interstellar clouds and primitive meteorites implies that it was also present in the early solar nebula, and may thus have been a potential constituent of chondrule precursor material. We describe here experiments in which carbon and magnesian silicate precursor material of primitive chondrule composition are ‘flash-heated’ together and then crystallized. The resulting material shows many mineralogical features characteristic of natural chondrules, which are not produced in the absence of carbon. Our results suggest not only that carbon was present in the solar nebula, but also that it played a key role in chondrule formation by creating within the melt a reducing environment that was decoupled from the nebula gas.

Previous dynamic crystallization (nonequilibrium) experiments that have studied chondrule formation assumed that chondrule precursors were only silicates. However, natural chondrule precursors almost certainly were a complex, heterogeneous mixture of silicates and non-silicates. Amongst these non-silicates, reduced carbon in the form of organic or graphitic material was probably among the most abundant and chemically important. It has been suggested that chondrule precursors may have even been held together with some form of organic 'glue'. The occurrence of carbon in chondrites has been described. Furthermore, the amount of carbon in ‘pristine’ early Solar System materials such as CI chondrites and the matrix of type 3 ordinary chondrites is up to 5% (ref. 18) and it may be that this is a reasonable amount to expect in the chondrule precursor.

Experiments (Table 1) were performed at 1 atm in an Astro furnace. Two starting materials of the same composition with initial grain sizes of 23–45 µm and 125–250 µm were prepared from ground silicate minerals (70% San Carlos olivine, 10% orthopyroxene, 10% diopside and 10% albite). Graphite (grain size <1 µm to 200 µm) and diamond (grain size 0.5 µm) were added to our silicate starting materials (from 0 to 10% wt%) for a combined pellet weight of 25 mg. Our experiments used graphite and diamond because the highly volatile, explosive nature of many forms of organic carbon prevents its use in performing experiments in our furnace. We are confident the redox properties of carbon in this form will match that of organic material closely enough for relevant conclusions to be drawn concerning the effects of carbon upon mineral chemistry and texture. The oxygen fugacity \(f_{\mathrm{O}_{2}}\) was maintained at IW-1.5 with a mixture of CO and \(\mathrm{CO}_{2}\). Thermal conditions imposed on pellets follow techniques described elsewhere.

Experiments Ast 32 and Ast 33 were carbon free and are our baseline (control) charges. Charge Ast 32 has a texture similar to type IA chondrules (Fig. 1a) whereas charge Ast 33 has a porphyritic texture with numerous relict (unmelted) olivines. The compositions of the olivines in these charges are listed in Table 1. These charges also contain rare, inclusion-free, Fe-rich, metal grains <10 µm in size.

Porphyritic textures similar to Ast 32 and Ast 33 were produced from charges with added carbon (Fig. 1b). However, the addition of 1–5% carbon in the starting material produces olivine phenocrysts with increasing MgO (termed forsterite, Fo) contents (Figs 1b, 2a, b). Adding >5% carbon to the starting material results in little further change in melt-grown olivine composition.

Relict olivines in charges with added carbon contain numerous metal grains, on average <1 µm in diameter (Fig. 1c). Microprobe analysis of the largest metal grains in olivines determined that they are kamacite (Fe-Ni with variable, low concentration of Ni) metal with variable amounts of Ni. Other relict olivines have veins of Fe metal and veins of more MgO-rich olivine (Fig. 1d). These olivines are similar to natural dusty olivines that have been interpreted as surviving chondrule precursor sol-

NATURE · VOL 371 · 8 SEPTEMBER 1994
