
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1996 by The American Society for Biochemistry and Molecular Biology, Inc.

**Purification and Characterization of Two Isoenzymes of DL-Glycerol-3-phosphatase from Saccharomyces cerevisiae**

IDENTIFICATION OF THE CORRESPONDING GPP1 AND GPP2 GENES AND EVIDENCE FOR OSMOTIC REGULATION OF Gpp2p EXPRESSION BY THE OSMOSENSING MITOGEN-ACTIVATED PROTEIN KINASE SIGNAL TRANSDUCTION PATHWAY*

(Received for publication, January 25, 1996, and in revised form, March 7, 1996)

Joakim Norbeck, Anna-Karin Pålman, Noreen Akhtar, Anders Blomberg, and Lennart Adler†

From the Department of General and Marine Microbiology, Lundberg Laboratory, Göteborg University, Medicinaregatan 9C, S-41390 Göteborg, Sweden

The existence of specific DL-glycerol-3-phosphatase (EC 3.1.3.21) activity in extracts of Saccharomyces cerevisiae was confirmed by examining strains lacking nonspecific acid and alkaline phosphatase activities. During purification of the glycerol-3-phosphatase, two isozymes having very similar molecular weights were isolated by gel filtration and anion exchange chromatography. By microsequencing of trypsin-generated peptides the corresponding genes were identified as previously sequenced open reading frames of unknown function. The two genes, GPP1 (YIL053W) and GPP2 (YER062C) encode proteins that show 95% amino acid identity and have molecular masses of 30.4 and 27.8 kDa, respectively. The intracellular concentration of Gpp2p increases in cells subjected to osmotic stress, while the production of Gpp1p is unaffected by changes of external osmolarity. Both isoforms have a high specificity for DL-glycerol-3-phosphate, pH optima at 6.5, and \(K_{\mathrm{m}}^{\mathrm{G} 3 \mathrm{P}}\) in the range of 3–4 mM. The osmotic induction of Gpp2p is blocked in cells that are defective in the HOG-mitogen-activated protein kinase pathway, indicating that GPP2 is a target gene for this osmosensing signal transduction pathway. Together with DOG1 and DOG2, encoding two highly homologous enzymes that dephosphorylate 2-deoxyglucose-6-phosphate, GPP1 and GPP2 constitute a new family of genes for low molecular weight phosphatases.

zymes are a NAD⁺-dependent glycerol-3-phosphate dehydrogenase (GPD)¹ and a poorly characterized glycerol-3-phosphatase. Transfer of cells to hyperosmolar conditions leads to increased production of GPD (3) due to transcriptional activation of one of the isogenes (GPD1) for the enzyme (4–6). In addition, membrane permeability to glycerol is diminished (7), presumably mainly due to inhibition of the facilitator that transports glycerol across the membrane (8). Under strongly hyperosmotic conditions, S. cerevisiae can accumulate glycerol up to molar concentrations (9). The osmotic induction of GPD1 appears to be controlled via an osmosensing pathway (6), involving cell surface osmosensors encoded by SHO1 (10) and a two-component system encoded by SLN1/SSK1 (11, 12). These detection systems are linked to a cascade of protein kinases encoded by SSK2, SSK22, PBS2, and HOG1, forming a mitogen-activated protein (MAP) kinase signaling pathway (10, 12, 13).

Osmotic control of glycerol production might also be exerted at the level of dephosphorylation of glycerol 3-phosphate. The early work by Tsuboi et al. (14, 15) indicated the existence of a phosphatase in S. cerevisiae having a specific role in glycerol production. These researchers described a partially purified phosphomonoesterase from bakers’ yeast that displayed a high specificity toward glycerol 3-phosphate at pH 6.5. Using conditions that repress the ortho-phosphate repressible phosphatases and strains lacking various nonspecific phosphatases, we here confirm the existence of a phosphatase activity, specific for DL-glycerol 3-phosphate in extracts of S. cerevisiae. To functionally characterize this enzyme and gain further insight into its role in cellular osmoregulation we took on purification of the glycerol-3-phosphatase and unexpectedly detected two molecular forms of the enzyme, one of which shows constitutive expression, the other being induced by increased extracellular osmolarity. By partial amino acid sequencing of tryptic digests of the two molecular forms, the corresponding genes, previously reported as open reading frames without function, were identified. We propose different roles for the two glycerol-3-phosphatase isoenzymes that belong to a previously unrecognized family of low molecular weight phosphomonoesterases.

EXPERIMENTAL PROCEDURES

Strains, Media, and Growth Conditions—The genotypes of the S. cerevisiae strains used are shown in Table I. Strains were routinely grown in a medium composed of 1% yeast extract, 2% Bacto peptone, and 2% glucose (YPD) or in a synthetic yeast nitrogen base (YNB)

* This work was supported by grants from the Swedish National Board for Natural Sciences (NFR), the Swedish National Board for Technical Development (TFR and NUTEK), and the Bank of Sweden Tercentenary Foundation (RJ). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
† To whom correspondence should be addressed. Tel.: 46 31 773 25 72; Fax: 46 31 7732599; E-mail: adler@gmm.gu.se.

1 The abbreviations used are: GPD, glycerol-3-phosphate dehydrogenase; MAP, mitogen-activated protein; PAGE, polyacrylamide gel electrophoresis; HPLC, high pressure liquid chromatography; HOG, high osmolarity glycerol.

13876 Two $DL$-Glycerol-3-phosphatases from $S$. cerevisiae

**TABLE I**

| Name     | Genotype                                                                 | Source or reference |
|----------|-------------------------------------------------------------------------|---------------------|
| YS18     | $MAT\alpha his3 leu2 ura3-\Delta 5 can^R$                                | A. Hinnen           |
| YMR4     | $MAT\alpha his3 leu2 ura3-\Delta 5 can^R pho5,3::ura3\Delta 1$          | A. Hinnen           |
| SKQ2n    | $MAT\alpha/ade1/+ ade2/+ his1/+$                                        | (38)                |
| YPH102a  | $MATa ura3 leu2\Delta his3\Delta ade2 lys2$                             | (36)                |
| JBY10    | $MATa ura3 leu2\Delta his3\Delta ade2 lys2 trp1 hog1-\Delta 1::TRP1$    | (13)                |
| JBY43    | $MATa ura3 leu2\Delta his3\Delta ade2 lys2 trp1 pbs2-\Delta 1::URA3$    | (13)                |
| SH611    | $SUC2 mal mel gal2 CUP1$                                                | S. Harashima        |
| SH3544   | $MATa pho8::URA3 ura3 his3-\Delta 300 trp1-\Delta 901$                  | S. Harashima        |
| SH3598   | $MATa pho3-1 pho13::URA3 ura3-1,2 leu2-3,112 lys1$                      | S. Harashima        |

medium (Difco), supplemented with 2% glucose and other necessary growth requirements (16) at 120 mg/liter. When appropriate, 10 mM inorganic phosphate (pH 5.5) was added to the YNB or YPD medium used (then named YNBP or YPDP medium) to ascertain repression of repressible phosphatases.

Solid media contained 2% agar. Responses to osmotic stress were assessed in media adjusted with NaCl or sorbitol. Liquid cultures were incubated with rotary agitation at 30 °C. Growth was monitored by measuring optical density at 610 nm ($A_{610}$). For preparation of cell extracts, cultures were harvested by centrifugation at $A_{610} = 0.5-1$.

Preparation of Cell-free Extracts—Cells were washed twice in ice-cold TRED buffer (10 mM triethanolamine, 1 mM EDTA, 1 mM dithiothreitol, pH 7.5) and resuspended in the same buffer plus 0.5 $\mu$l/ml of a protease inhibitor mix containing 70 mg/ml phenylmethylsulfonyl fluoride, 1.4 mg/ml leupeptin, and 1 mg/ml pepstatin. Extracts were prepared by disruption of cells using acid-washed glass beads, and the obtained extract was cleared by centrifugation (14,000 × g for 15 min). Extracts used to measure phosphatase activity were desalted by gel filtration according to procedures described previously (17). All operations were performed at 0–4 °C.

Protein Determination—Protein concentration was routinely determined using the method of Bradford (Bio-Rad). Protein concentrations of purified protein fractions were estimated by determining the absorbance at 280 nm of concentrated samples in a 100-$\mu$l microcuvette. Values were compared with those of a bovine serum albumin standard and translated to protein concentration after correction for the difference in tryptophan content of the two proteins.

Enzyme Assay—Glycerol-3-phosphatase was assayed essentially as described by Sussman and Avron (18). Cell-free extracts were routinely incubated in 20 mM Tricine-HCl (pH 7.0), 5 mM MgCl$_2$, and 10 mM DL-glycerol 3-phosphate in a total volume of 1.0 ml. For determination of kinetic parameters, the DL-glycerol 3-phosphate concentration was varied within the concentration range of 0.5–32 mM. After starting the reaction, samples of 90 $\mu$l were withdrawn at time intervals and immediately added to 10 $\mu$l 50% HClO$_4$ to stop the enzyme reaction. Substrates other than glycerol-3-phosphate were added to a final concentration of 10 mM, and for these slower reactions, samples were withdrawn at 60-s intervals for 6 min. Released inorganic phosphate was analyzed according to Ames (19), and the reaction rate was calculated from the slope of a linear plot of released phosphate versus time. To detect enzyme activity during the enzyme purification procedure, samples of 5 $\mu$l were incubated in 95 $\mu$l of assay buffer for 15 min, followed by phosphate analysis as described above.

PhastGel Electrophoresis—Native molecular weight estimations were performed using PhastGel 8–25 gradient gels (Pharmacia LKB) for native PAGE. Strain YMR4 was cultured in YPDP and homogenized as described above, and 4.5 $\mu$l of extract was applied to each lane. The molecular weight standards used were those of the Pharmacia electrophoresis calibration kit. Two parallel gels were run in a PhastSystem (Pharmacia) as recommended by the manufacturer. One gel was stained for phosphatase using 10 mM DL-glycerol 3-phosphate in assay buffer containing 15 mM CaCl$_2$. Released phosphate was in situ precipitated as white calcium phosphate. The other gel was stained for proteins using the recommended Coomassie staining program.

Protein Purification—Protein purification was carried out at 4 °C, and whenever possible, samples were kept on ice. All chromatographic steps were performed on an fast protein liquid chromatography system (Pharmacia). Cell extracts (5–15 mg/ml protein) were loaded onto a 120-ml Superdex 200 column that was equilibrated with 10 mM TRED buffer. The column was eluted with the same buffer at a flow rate of 1 ml/min, and 1-ml fractions were collected. In this and subsequent chromatographic steps, protein concentration was monitored by continuous UV absorbance at 280 nm, and the peak enzyme activity was localized by phosphatase assays. The most active fractions were pooled and applied to a 1-ml Mono-Q column equilibrated with 10 mM triethanolamine (pH 7.5) containing 1 mM dithiothreitol (TRD buffer). The column was washed with 45 ml of TRD buffer and eluted with a 30-ml linear gradient of 0–0.15 M NaCl in the same buffer followed by 5 ml of buffer containing 1 M NaCl. The fractions active for glycerol-3-phosphatase were reapplied to the same Mono-Q column and subjected to a consecutive round of ion exchange chromatography. This cycle was initiated by washing with 15 ml of TRD buffer adjusted to pH 7.0, followed by elution with 20 ml of a combined ion strength-pH gradient generated by linearly increasing the proportion of 10 mM Bis-Tris buffer (pH 5.5) containing 0.25 M KCl from 0 to 30% in the basal TRD buffer (pH 7.0), and finally terminated by elution with 5 ml of 100% Bis-Tris buffer. Purified enzyme fractions of the two isoforms of the phosphatase were taken from the peak fractions of first cycle of Mono Q chromatography, where the enzymes eluted at 63 mM NaCl (Gpp1p, see Fig. 4C) and at 35 and 57 mM NaCl (Gpp2 and Gpp1, respectively, Fig. 4D).

SDS-Polyacrylamide Gel Electrophoresis—To examine the homogeneity of the purified enzyme fractions, samples were analyzed by SDS, 15% polyacrylamide gel electrophoresis, and proteins were visualized by silver staining (20). For microsequencing of the purified phosphatases, the stained band was excised with a scalpel and prepared for trypsin digestion as described previously (20).

Two-dimensional PAGE—Two-dimensional PAGE was run on a Millipore Investigator system as described previously (20). Dried gels were exposed to image plates and subsequently scanned in a PhosphorImager (Molecular Dynamics) at a resolution of 176 × 176 $\mu$m. Images were automatically processed and quantified by the two-dimensional software PDQuest (PDI) as described by Blomberg et al. (21). Individual quantification of resolved proteins were normalized to the total amount of radioactivity in all quantified spots of the gel and expressed as parts per million.

Generation and Sequencing of Peptides—In gel trypsin digestion of purified phosphatase and separation of the generated peptides on reversed phase-HPLC (Smart system, Pharmacia), were performed as described elsewhere (20). N-terminal sequence analysis was acquired using a pulsed liquid sequenator 473A (Applied Biosystems) according to procedures recommended by the manufacturer. Fractions were applied to BioBrene conditioned filters in 20-$\mu$l portions that were dried under argon between applications.

---

**RESULTS**

Evidence for Specific Glycerol-3-phosphatase Activity in Crude Extract—To facilitate the search for a specific glycerol-3-phosphatase in crude extracts of $S$. cerevisiae, efforts were made to minimize the presence of various nonspecific alkaline and acid phosphatases. The acid phosphatases of $S$. cerevisiae encoded by the $PHO5$, $PHO10$, and $PHO11$ genes as well as the alkaline phosphatase encoded by the $PHO8$ gene are all repressed at the transcriptional level by inorganic phosphate (22, 23). To curb formation of these enzymes, cell cultures were grown with 10 mM inorganic phosphate. Enzyme activity measurements in extracts from such cultures indicated negligible contribution to the observed glycerol-3-phosphatase activity from residual activity of the repressible acid (Pho5p) and alkaline (Pho8p) phosphatases, since mutants lacking the corresponding genes (strain YMR4 and SH3544, respectively) demonstrated activities similar to those of the wild type strains (Table II). Neither was the analyzed glycerol-3-phosphatase activity notably influenced by the absence of the nonrepressible

Two $DL$-Glycerol-3-phosphatases from $S$. cerevisiae

**TABLE II**

Glycerol-3-phosphatase activity in crude extracts of various phosphatase-defective mutants of $S$. cerevisiae grown in YNB medium

Values are means of three separate experiments ± S.D.

| Strain | Relevant genotype | Glycerol-3-phosphatase activity |
|--------|-------------------|----------------------------------|
|        |                   | $nmol \, min^{-1} \, mg \, protein^{-1}$ |
| YS 18  | Parent strain     | $68.0 \pm 3.8$                  |
| YMR 4  | $pho5, 3::ura3\Delta 1$ | $70.2 \pm 3.9$                  |
| SH 611 | Parent strain     | $73.1 \pm 0.5$                  |
| SH 3544 | $pho8::URA3$      | $67.6 \pm 5.0$                  |
| SH 3598 | $pho3-1 pho13::URA3$ | $62.5 \pm 0.3$                  |

**FIG. 1.** Native molecular weight of glycerol-3-phosphatase as estimated by gradient polyacrylamide electrophoresis of standard proteins and crude extract of $S$. cerevisiae strain YMR4. Samples of standard solutions and cell extracts were electrophoresed on two parallel gels that were stained for proteins and glycerol-3-phosphatase, respectively. The arrow marks the migrating position of the glycerol-3-phosphatase activity.

**TABLE III**

Phosphatase activity of crude extract and purified Gpp1p and Gpp2p on various organic phosphomonoesters

Gpp1p and Gpp2p were purified by gel filtration and Mono-Q chromatography. For each preparation the enzyme activity with DL-glycerol 3-phosphate was set to 100. The corresponding activities were as follows (in $nmol \, min^{-1} \, mg \, protein^{-1}$): crude extract (68.0), Gpp1p (16170), and Gpp2p (2050).

| Substrate               | Relative enzyme activity |
|-------------------------|--------------------------|
|                         | Crude extract | Gpp1p | Gpp2p |
| DL-glycerol 3-phosphate | 100            | 100   | 100   |
| Glycerol 2-phosphate    | 8.6            | 2.7   | 0.8   |
| Glycerate 3-phosphate   | 3.0            | 0.01  | 0.2   |
| Glycerate 2,3-diphosphate | 2.8            | 0.4   | 1.0   |
| Glucose 6-phosphate     | 3.7            | 0.5   | 0.5   |
| Fructose 6-phosphate    | 3.7            | 0.1   | 1.0   |
| 2-Deoxyglucose 6-phosphate | ND$^a$       | 0.2   | 0.0   |

$a$ ND, not determined.

acid phosphatase encoded by $PHO3$ or the constitutive alkaline phosphatase coded for by $PHO13$ (Table II, strain SH3598).

Examination of the substrate specificity of the detected enzyme activity in crude extract of wild type cells (Table III) showed preferential activity on glycerol 3-phosphate; less than 9% of that activity was exerted on glycerol 2-phosphate, glycerate 3-phosphate, glycerate 2,3-diphosphate, glucose 6-phosphate, and fructose-6 phosphate. This substrate profile was very similar with extracts of mutants lacking one ($pho3$, YMR4) or both ($pho3 pho13$, SH3598) of the constitutive acid and alkaline phosphatases (data not shown). From these results we conclude that it is possible to assay a specific glycerol-3-phosphatase activity in crude extracts of $S$. cerevisiae under the pH-neutral conditions used and that this activity is clearly distinct from that of previously described acid and alkaline phosphatases.

To further characterize the glycerol-3-phosphatase, the molecular weight was estimated from the mobility of the enzyme on gel electrophoresis in a native gradient 5–25% polyacrylamide gel. Staining of gels for released phosphate from glycerol 3-phosphate revealed one band with a migrating position corresponding to a molecular mass of 30 kDa (Fig. 1).

**Effect of Extracellular Osmolarity on Glycerol-3-phosphatase Activity—** Culturing cells with various concentrations of NaCl demonstrated that the total cellular activity of glycerol-3-phosphatase increased with increased extracellular salinity (Fig. 2). Substituting NaCl by osmotically equivalent concentrations of sorbitol gave a comparable increase, consistent with the supposition that the cells adjust their specific activity or enzyme concentration in response to changes of the surrounding osmolarity. The observed increase in activity was specific for glycerol 3-phosphate; no increased hydrolysis of the other organic phosphate monoesters used for the substrate specificity test was observed for cells grown at $0.7 \, M$ NaCl (data not shown), indicating that the demonstrated regulation is specifically ex-

**FIG. 2.** Activity of glycerol 3-phosphatase in cell extracts of $S$. cerevisiae strain YMR4 grown cultured in YPDP medium ($-0.4$ MPa) adjusted to $-1.9$ MPa by the addition of $0.35 \, M$ NaCl or $0.56 \, M$ sorbitol or to $-3.3$ MPa by the addition of $0.7 \, M$ NaCl or $1 \, M$ sorbitol. Activities are means of two different experiments. Differences between samples were $<15\%$.

**FIG. 3.** Activity of glycerol-3-phosphatase in cell extracts of $S$. cerevisiae strain YPH 102a and a congenic strain having a deleted $PBS2$ (J BY43) or a deleted $HOG1$ (J BY10) gene. Cells were cultured in YNBP medium containing no added NaCl, $0.35 \, M$ NaCl, or $0.7 \, M$ NaCl, as indicated.

ered on the enzyme activity showing specificity for glycerol 3-phosphate.

To examine whether protein synthesis is required to achieve this increase in activity, cells were shifted from basal medium to medium containing $0.7 \, M$ NaCl in the presence or absence of cycloheximide. In the absence of inhibitor, enzyme activity increased to levels 1.5 and 1.7 times the original value after incubation for 60 and 110 min, respectively. This increase was completely prevented by the presence $100 \, \mu g/ml$ of cycloheximide (data not shown), indicating that the increase in glycerol-3-phosphatase activity requires protein synthesis.

Osmotic stress-induced glycerol production in $S$. cerevisiae requires the $HOG1$ and $PBS2$ genes, encoding MAP kinase homologues of the osmosensing and signaling HOG pathway

13878 Two $D_{L}$-Glycerol-3-phosphatases from S. cerevisiae

FIG. 4. Purification by Superdex 200 gel filtration (A and B) and Mono-Q ion chromatography (C and D) of glycerol 3-phosphatase from S. cerevisiae strain SKQ grown in YNB medium (A and C) and YNB medium + 1.4 M NaCl (B and D). The protein elution (solid line, $A_{280}$) and the phosphatase activity (○—○, arbitrary units) profiles are shown.

(13). Examination of the specific activity of glycerol-3-phosphatase in extracts of $pbs2\Delta$ and $hog1\Delta$ mutants cultured at elevated salinities demonstrated that either defect in the signaling pathway caused a complete inhibition of the osmotic induction of the glycerol 3-phosphatase activity (Fig. 3).

Purification of Glycerol-3-phosphatase—To purify the glycerol-3-phosphatase we employed gel filtration and anion exchange chromatography. As a first step, extracts of cells grown in YNB medium and the same medium containing 1.4 M NaCl were subjected to Superdex 200 gel filtration (Fig. 4, A and B). The presence of a shoulder in the activity profile of eluate from extracts of cells grown at high salinity suggested two forms of the enzyme in salt-grown cells. This assumption was reinforced by the subsequent Mono-Q ion chromatography step (Fig. 4, C and D) in which the proteins were eluted by a linear 0–0.15 M NaCl gradient. The phosphatase activity emerged as a single peak (form I) with extracts of nonstressed cells, whereas with extracts of cells grown at high salinity, an additional form (form II) was eluted at slightly lower salt concentration. The active Mono-Q fractions from cells cultured in 1.4 M NaCl media were further resolved by rechromatography on the same column, this time eluted with a combined pH and KCl gradient (see “Experimental Procedures” and Fig. 5A). By this step form I, hereafter called Gpp1p, was purified to electrophoretic homogeneity as revealed by silver-stained SDS gel electrophoresis (Fig. 5B), while the recovery of activity of form II by this procedure was insignificant. Enzyme activity data demonstrated that Gpp1p was not appreciably affected by changed salinity, while the second molecular form, hereafter called Gpp2p, increased strongly with salinity from an activity below detection level in nonstressed cells (Fig. 4, C and D). A quantitative summary of the purification of the two isoforms as performed with extract of cells grown at 1.4 M NaCl, is shown in Table IV.

N-terminal Sequence Analysis of Gpp1p and Gpp2p—Gpp1p was excised from SDS gels and cleaved by in-gel trypsinization. The generated peptides were separated on reverse phase HPLC, and selected fractions were subjected to N-terminal sequencing. The eight peptides examined revealed the sequences shown in Table V. When these peptides were compared with sequence data of the Swiss Protein Data Bank, full identity was established with an open reading frame recognized by the systematic sequencing of chromosome IX. The corresponding gene (YIL053W) which was designated GPP1, encodes a

FIG. 5. A, rechromatography on Mono-Q of pooled Gpp1p fractions from the first Mono-Q separation step. Symbols are as in Fig. 4. B, SDS-polyacrylamide gel electrophoresis of protein from the peak glycerol-3-phosphatase fraction of the Gpp1p rechromatography. An aliquot of 0.1 μg of protein was subjected to electrophoresis through 15% polyacrylamide gel. Proteins were visualized by silver staining.

protein of 271 amino acids having a molecular mass of 30.4 kDa.

SDS-polyacrylamide gel electrophoresis and tryptic digestion of Gpp2p, using the most active fractions from the Mono-Q chromatography (Fig. 4D), yielded seven sequences that showed striking similarity to Gpp1p, although some of the sequences displayed a slight but distinct sequence deviation from the corresponding regions of Gpp1p (Table V, unique amino acids underlined). Database search revealed full identity with the primary structure of a gene (YER062C) identified by sequencing of chromosome V. This gene encodes a protein of 250 amino acids having a molecular mass of 27.8 kDa, which is slightly lower than for Gpp1p. This gene was designated GPP2.

Comparison of GPP1 and GPP2—Sequence comparisons between the GPP1 and GPP2 using the Gap and BestFit program from the Genetics Computer Group demonstrated strong homology, the inferred amino acid sequences showing 95% identity (Fig. 6). The only identified proteins to which Gpp1p and Gpp2p show significant identity (30–35%) are two phosphomonoesterases encoded by DOG1 and DOG2 (24, 25).

Two-dimensional PAGE Analysis of Gpp1p and Gpp2p—From the expression pattern and the theoretical $M_{r}$, the positions of Gpp1p and Gpp2p were tentatively localized on analytical two-dimensional PAGE gels. To confirm the identity, the located protein spots were excised from preparative gels, and internal peptides were generated and sequenced. The obtained amino acid sequences (Table V) unequivocally identified the candidate spots as the GPP1 and GPP2 encoded proteins. The

Two DL-Glycerol-3-phosphatases from S. cerevisiae

TABLE IV  
Purification of Gpp1p and Gpp2p from extracts of S. cerevisiae SKQ grown at 1.4 M NaCl  

| Fraction         | Volume (ml) | Protein (µg) | Specific activity (nmol min⁻¹ mg protein⁻¹) | Total Activity (nmol min⁻¹) | Purification (-fold) | Yield (%) |
|------------------|-------------|--------------|--------------------------------------------|----------------------------|----------------------|-----------|
| Cell-free extract | 2           | 27520        | 108                                        | 2973                       | 1                    | 100       |
| Superdex 200     | 11          | 181.50       | 7063                                       | 1282                       | 65                   | 43        |
| Mono-Q I         |             |              |                                            |                            |                      |           |
| GPP1             | 1           | 25.72        | 24029                                      | 618                        | 222                  | 21        |
| GPP2             | 1           | 8.55         | 9400                                       | 80                         | 87                   | 2.7       |
| Mono-Q II        |             |              |                                            |                            |                      |           |
| GPP1             | 1           | 4.97         | 12062                                      | 60                         | 112                  | 2         |
| GPP2$^a$         |             |              |                                            |                            |                      |           |

$a$ Activity not detectable.

---

TABLE V  
Amino acid sequences of peptides generated from the two isoforms of glycerol-3-phosphatase, resolved by protein chromatography or two-dimensional PAGE  

Numbers refer to the amino acid positions in the predicted protein sequences of the YIL053W (GPP1) and YER062C (GPP2) genes. Underlined residues mark positions of unique amino acids.

| Constitutive form (GPP1) | Salt-induced form (GPP2) |
|--------------------------|--------------------------|
| Purified by protein      |                          |
| chromatography           |                          |
| INAALFDV(V) GXI          | DXPXXD                   |
| FAPDADEEYVN              | TFDAIAK                  |
| WAVATXG                  | FAPDXXN                  |
| WFDILK                   | (W)AVA(T) XG             |
| XXRPEYFITA               | YFITAND                  |
| QGKP(H) PEPYLK           | QG(K) P(H) PEPYLK        |
| IVGIATTFDLD              | VGGY                     |
| VGEYNAEXD                | 223-226                  |
| Extracted from preparative |                          |
| two-dimensional PAGE    |                          |
| FAPDADEEYV               | VNAALFDVDGTI             |
| YGEHXIEVP                | FAPDFAN                  |
| VVVFEDAPAGIAAG           | WFEH                     |
| XFITAND                  | 136-142$^a$              |
| VVVFEDAPA                | 174-182                  |
| VGGYNAETDEV              | 223-233                  |

$a$ Sequence is preceded by E in YIL053W and would therefore not provide a site for trypsin.

position of the two isoforms are indicated by arrows in Fig. 7, A-C, which shows relevant portions of gels run with L-[S³⁵] methionine-labeled extracts of cells cultured in medium containing 0, 0.7, and 1.4 M NaCl. Computerized quantitation (Fig. 7, D and E) of the two forms confirmed previous observations; Gpp1p remains abundant independent of external salinity. This form comprises 0.2–0.3% of totally detected proteins as determined from the methionine incorporation. Normalizing obtained parts per million values for differences in methionine content, thus enabling stoichiometric comparisons, we found that Gpp1p was approximately equimolar to actin (Act1p) in cells grown in basal medium. Gpp2p increases, on the other hand, from a low level to become about as abundant as Gpp1p at the highest salinity (1.4 M NaCl).

Properties of Gpp1p and Gpp2p—The purified isoforms showed insignificant activity on phosphomonoesters other than glycerol 3-phosphate (Table III). The stereospecificity of the enzyme for the D- and L-forms of glycerol 3-phosphate was examined by comparison of the enzyme activity obtained for the racemic substrate (DL-form) with that observed for the pure L-form (sn-glycerol 3-phosphate), which was anticipated to be the natural substrate for the enzyme. Surprisingly, both forms of the enzyme show essentially similar activities with DL- and L-glycerol 3-phosphate (data not shown), indicating no obvious stereoselectivity for either enantiomer. Hence, the enzymes should be named DL-glycerol-3-phosphatases. Using various concentrations of the DL-form, as the substrate, Eadie-Hofstee plots yielded apparent $K_m$ of 3.1 mM for Gpp1p and of 3.9 mM for Gpp2p. The corresponding $V_{max}$ values were 49 and 46 µmol/min/mg protein. The pH dependence profile showed an optimum pH 6.5 for both enzymes (Fig. 8).

The HOG Pathway Controls Osmotic Induction of GPP2—To further examine the effect of the HOG pathway on osmotic induction of the DL-glycerol-3-phosphatase activity, enzymes were purified from extracts of a hog1Δ and a parent strain that had been both incubated for 1 h at 0.7 M NaCl. The parent strain displayed clear Gpp2p and Gpp1p peaks after Mono-Q chromatography, while the hog1Δ mutant showed roughly the same amount of Gpp1p as the parent but insignificant Gpp2p enzyme (Fig. 9).

DISCUSSION

This work reports the presence of two isoforms of DL-glycerol 3-phosphatase (EC 3.1.3.21) in S. cerevisiae. Peptide sequences obtained from purified isoforms demonstrated that the enzymes responsible for this activity are encoded by two distinct but highly identical genes, here named GPP1 and GPP2. These genes were previously sequenced as part of the systematic sequencing of the yeast genome but were reported as open reading frames without known function. The predicted amino acid sequences of GPP1 and GPP2 are 95% identical and show about 35% identity with that of DOG1, which was isolated by its ability to confer resistance to 2-deoxyglucose (24). This nonmetabolizable glucose analogue elicits glucose repression, and this effect is counteracted by overexpression of the DOG1 gene product which has a 2-deoxyglucose-6-phosphatase activity. Interestingly, Dog1p has a “twin” homologue encoded by the recently identified DOG2 gene (25), showing 92% amino acid identity to that of DOG1. The high similarity within each of the two gene pairs points to an origin by gene duplication and dispersal. The true physiological function of Dog1p and Dog2p is not yet established (24, 25). However, they clearly appear to have a role different from Gpp1p and Gpp2p, since these enzymes do not dephosphorylate 2-deoxyglucose 6-phosphate (Table IV) and Dog1p and Dog2p show insignificant activity on glycerol 3-phosphate (25, 26). Together these four

13880 Two DL-Glycerol-3-phosphatases from S. cerevisiae


1 MGLTTKPLSLKVNAALFDVDGTIIISQPA 29
| ||||||||:|||||||||||||||||||
1 MKRFNVVLKYIRTTKANIQTIAMPLTTKPLSLKINAALFDVDGTIIISQPA 50
30 IAAFWRDFGKDKPYFDAEHVIQVSHGWRTFDAIAKFAPDFAEEYVNKLE 79
| ||||||||| ||||||||||||:||||||:|||||||||||||||
51 IAAFWRDFGKDKPYFDAEHVIISHGWRTYDAIAKFAPDFADEEYVNKLE 100
80 AEIPVKYGEKSIEVPGAVKLCNALNALKPEKWAVATSGTRDMAQKWFEHL 129
:||| |||| |||||||||||||||||||||||||||||||||||||:|
101 GEIPEKYGEHSIEVPGAVKLCNALNALKPEKWAVATSGTRDMAKKWFDIL 150
130 GIRRPKYFITANDVKQGKPHPEPYLKGR.GLGYPINEQDPSKSKVVVFED 178
|:|| ||||||||||||||||||||||||||||:|||||||||||||||||||
151 KIKRPEYFITANDVKQGKPHPEPYLKGRNGLGFINEQDPSKSKVVVFED 200
179 APAGIAAGKAAGCKIIGIATTFDLDFLKEKGCDIIVKNHESTIRVGGYNAE 228
||||||||||||||||||||||||||||||||||||||||||||||||||||
201 APAGIAAGKAAGCKIVGIATTFDLDFLKEKGCDIIVKNHESIRVGEYNAE 250
229 TDEVEIFDDYLYAKDDLLKW 249
||||||:||||||||||||||||
251 TDEVELIFDDYLYAKDDLLKW 271

FIG. 6. Alignment of deduced amino acid sequences of GPP1 (YIL053W, lower sequence) and GPP2 (YER062C, upper sequence). The sequences were aligned to each other using the Gap and BestFit program of the Genetics Computer Group sequence analysis package. Straight lines indicate amino acid identity; two dots indicate conservative substitutions.

![Figure 7](https://via.placeholder.com/300x200?text=Fig.+7)

FIG. 7. Autoradiograms of portions of two-dimensional PAGE gels containing proteins synthesized by S. cerevisiae strain SKQ cultured to \(A_{610} \approx 0.5\) in YNB medium containing 0 M NaCl (A), 0.7 M NaCl (B), and (C) 1.4 M NaCl. Thirty minutes prior to harvest, cells were labeled by the addition \({ }^{[35]}\) methionine to a specific activity of 15 \(\mu\)Ci/ml. The positions of Gpp1p and Gpp2p are indicated by arrows. The relative abundance (parts per million, PPM) of Gpp1p (D) and Gpp2p (E) in cells grown at different salinities was determined by computerized spot quantitation.

enzymes constitute a previously unrecognized family of low molecular weight phosphomonoesterases, having molecular masses of approximately 30 kDa. Gel filtration and gradient gel electrophoresis under nondenaturing conditions indicate that Gpp1 and Gpp2 are catalytically active as monomers. Hence, the glycerol-3-phosphatase isozymes are clearly different from the nonspecific alkaline and acid phosphatases, which have subunit sizes of 57–66 kDa as determined by DNA sequencing and SDS gel electrophoresis (27–31), and which are generally active as oligomers (32).

The glycerol 3-phosphatase activity that was originally described in partially purified extracts of bakers’ yeast (14) is by all likelihood identical to that of Gpp1p, since this enzyme is the predominant form in cells cultured at low osmolarity (Figs. 4 and 7). As indicated from subcellular fractionation of lysed spheroplasts, this activity has its main location in the cytosol (33, 34). The narrow substrate specificity of the DL-glycerol-3-phosphatase conforms with its location in the cytosol, where a strict specificity would be essential to avoid interference with

![Figure 8](https://via.placeholder.com/300x200?text=Fig.+8)

FIG. 8. Influence of pH on the glycerol-3-phosphatase activity of Gpp1p and Gpp2p. Symbols: (○—○) Gpp1p, (●—●) Gpp2p.

![Figure 9](https://via.placeholder.com/300x200?text=Fig.+9)

FIG. 9. Phosphatase activity profile (arbitrary units) after Mono-Q column chromatography of glycerol 3-phosphatase extracted from S. cerevisiae strain YPH 102a (○—○) and a congenic strain having a deleted HOG1 gene (J BY10) (●—●), grown in YNB medium to \(A_{610} \approx 1\) and thereafter incubated for 1 h in the same medium containing 0.7 M NaCl.

the plethora of phosphorylated metabolites in the metabolic network. In contrast, the nonspecific phosphatases of acid and alkaline preferences are primarily confined to the vacuole or the periplasm of the cell (32).

Glycerol has dual functions in S. cerevisiae; it serves as a nontoxic redox sink during fermentation (35)² and as an osmoregulator during hyperosmotic stress (2). Osmoregulation in S. cerevisiae is achieved by varying the internal glycerol concentration to adjust the intracellular osmolarity in response to changes of the extracellular water potential. Exposed to hyperosmotic stress, the cells increase glycerol production and glycerol accumulation. A contribution from Gpp2p to this response is indicated by the finding that this isoform, which is the clearly minor form under nonstressed conditions, increases as cells are cultured at elevated osmolarity (Figs. 4 and 7). A strict requirement for a separate osmoregulated form of DL-glycerol-3-phosphatase in the osmoadaptation of S. cerevisiae remains, however, to be established, since the other GPP1 encoded isoform is abundantly present, constituting about 0.25% of total cellular protein, independent of external osmolarity (Fig. 7D).

Recent evidence indicates that the signaling mechanisms behind the glycerol response are dependent on a MAP kinase module involving the PBS2 and HOG1 gene products (13). These kinases appear to be activated via membrane-localized osmosensors (10, 12). According to this model, changes in external osmolarity generate an internal signal that is transmitted to the MAP kinase cascade. One of the targets of this signaling is GPD1, encoding glycerol-3-phosphate dehydrogenase, the enzyme constituting the first step in the glycerol biosynthetic pathway (5, 6). The rapid induction of GPD1 expression after a step increase in salinity is blocked in pbs2Δ³ or hog1Δ (6) signaling-defective mutants. It would seem logical that the osmoregulated Gpp2p enzyme would be controlled

² R. Ansell, S. Hohmann, J. Thevelein, and L. Adler, submitted for publication.
³ P. Eriksson and A-K. Pahlman, unpublished results.

coordinate with Gpd1p via the HOG-MAP kinase pathway. Indeed, such a coordinated control appears to be exerted, as evidenced by the finding that the Gpp2p activity does increase insignificantly or only slightly in an osmosignaling-defective hog1Δ mutant that is cultured at elevated salinities (Fig. 9). In analogy with the osmostimulated expression of the *GPD1* one would expect that the osmotic control is due to transcriptional induction of the *GPP2* gene. Hirayama et al. (37) very recently reported the cloning of seven hyperosmolarity-responsive (*HOR*) genes from *S. cerevisiae* by a differential screening method. Two of these genes, *HOR2* and *HOR7*, were of unknown function. When inspecting the sequences of these genes, we observed that the *HOR2* gene is identical to *GPP2*. Interestingly, Northern blot analysis showed that the expression of *HOR2/GPP2* is induced by increased osmolarity and that this induction is inhibited in a *hog1Δ* strain. Hirayama et al. (37) also noted that *HOR2* has a close homologue called *RHR2*, which is not subject to osmotic regulation. Since *RHR2* is identical to *GPP1*, the results of these authors confirm on the transcriptional level, the observed regulatory pattern of *GPP1* and *GPP2*.

**Acknowledgments—** Sincere thanks are due to Drs. M. Gustin, S. Harashima, A. Hinnen, and T. Hottiger for generously providing yeast strains. We are also grateful to Dr. S. Hohmann for constructive criticism of the manuscript.

**REFERENCES**

1. Brown, A. D. (1978) *Adv. Microb. Physiol.* **17**, 181–242
2. Blomberg, A., and Adler, L. (1992) *Adv. Microb. Physiol.* **33**, 145–212
3. André, L., Hemming, A., and Adler, L. (1991) *FEBS Lett.* **286**, 13–17
4. Eriksson, P., Andre, L., Ansell, R., Blomberg, A., and Adler, L. (1995) *Mol. Microbiol.* **17**, 95–107
5. Larsson, K., Eriksson, P., Ansell, R., and Adler, L. (1993) *Mol. Microbiol.* **10**, 1101–1111
6. Albertyn, J., Hohmann, S., Thevelein, J. M., and Prior, B. A. (1994) *Mol. Cell. Biol.* **14**, 4135–4144
7. Blomberg, A., and Adler, L. (1989) *J. Bacteriol.* **171**, 1087–1092
8. Luyten, K., Albertyn, J., Skibbe, W. F., Prior, B. A., Ramos, J., Thevelein, J., and Hohmann, S. (1995) *EMBO J.* **14**, 1360–1371

9. Ölz, R., Larsson, K., Adler, L., and Gustafsson, L. (1993) *J. Bacteriol.* **175**, 2205–2213
10. Maeda, T., Takekawa, M., and Saito, H. (1995) *Science* **269**, 554–558
11. Ota, I. M., and Varshavsky, A. (1993) *Science* **262**, 566–569
12. Maeda, T., Wurgler-Murphy, S. M., and Saito, H. (1994) *Nature* **369**, 242–245
13. Brewster, J. L., de Valoir, T., Dwyer, N. D., Winter, E., and Gustin, M. C. (1993) *Science* **259**, 1760–1763
14. Tsuboi, K. K., Wiener, G., and Hudson, P. B. (1957) *J. Biol. Chem.* **224**, 621–635
15. Tsuboi, K. K., and Hudson, P. B. (1956) *Arch. Biochem. Biophys.* **61**, 197–210
16. Sherman, F., Fink, G. R., and Lawrence, C. W. (1979) *Methods in Yeast Genetics: A Laboratory Manual*, pp. 61–64, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
17. André, L., Nilsson, A., and Adler, L. (1988) *J. Gen. Microbiol.* **134**, 669–677
18. Sussman, I., and Avron, M. (1981) *Biochim. Biophys. Acta* **661**, 199–204
19. Ames, B. N. (1966) *Methods Enzymol.* **8**, 115–117
20. Norbeck, J., and Blomberg, A. (1995) *Electrophoresis* **16**, 149–156
21. Blomberg, A., Blomberg, L., Norbeck, J., Fey, S., Mose Larsen, P., Roepstorff, P., Degand, H., Boutry, M., Posch, A., and Görg, A. (1995) *Electrophoresis* **16**, 1935–1945
22. Lemire, J. M., Willcock, T., Halvorson, H. O., and Bostian, K. A. (1985) *Mol. Cell. Biol.* **5**, 2131–2141
23. Kramer, R. A., and Andersen, N. (1980) *Proc. Natl. Acad. Sci. U.S.A.* **77**, 6541–6545
24. Sanz, P., Randez-Gil, F., and Prieto, J. A. (1994) *Yeast* **10**, 1195–1202
25. Randez-Gil, F., Blasco, A., Prieto, J. A., and Sanz, P. (1995) *Yeast* **11**, 1233–1240
26. Martin, M., and Heredia, C. F. (1977) *FEBS Lett.* **83**, 245–248
27. Kaneko, Y., Toh-e, A., Banno, I., and Oshima, Y. (1989) *Mol. & Gen. Genet.* **220**, 133–139
28. Plankert, U., Purwin, C., and Holzer, H. (1991) *Eur. J. Biochem.* **196**, 191–196
29. Venter, U., and Hörz, W. (1989) *Nucleic Acids Res.* **17**, 1353–1369
30. Bajwa, W., Meyhack, B., Rudolph, H., Schweingruber, A.-M., and Hinnen, A. (1984) *Nucleic Acids Res.* **12**, 7721–7739
31. Nosaka, K. (1990) *Biochim. Biophys. Acta* **1037**, 147–154
32. Johnston, M., and Carlson, M. (1992) in *The Molecular and Cellular Biology of the Yeast Saccharomyces: Gene Expression* (Jones, E. W., Pringle, J. R., and Broach, J. R., eds) Vol. II, pp. 193–281, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
33. Tonino, G. J. M., and Steyn-Parve, E. P. (1963) *Biochim. Biophys. Acta* **67**, 453–469
34. Wiemken, A., Schellenberg, M., and Urech, K. (1979) *Arch. Microbiol.* **123**, 23–35
35. Nordström, K. (1968) *J. Inst. Brew.* **74**, 429–432
36. Sikorski, R. S., and Hieter, P. (1989) *Genetics* **122**, 19–27
37. Hirayama, T., Maeda, T., Saito, H., and Shinozaki, K. (1995) *Mol. & Gen. Genet.* **249**, 127–138
38. Bataillé, N., Thoraval, D., and Boucherie, H. (1988) *Electrophoresis* **9**, 774–780

Purification and Characterization of Two Isoenzymes of DL-Glycerol-3-phosphatase  
from *Saccharomyces cerevisiae*: IDENTIFICATION OF THE CORRESPONDING  
GPP1 AND GPP2 GENES AND EVIDENCE FOR OSMOTIC REGULATION OF  
Gpp2p EXPRESSION BY THE OSMOSENSING MITOGEN-ACTIVATED  
PROTEIN KINASE SIGNAL TRANSDUCTION PATHWAY  

Joakim Norbeck, Anna-Karin Pahlman, Noreen Akhtar, Anders Blomberg and Lennart Adler  

J. Biol. Chem. 1996, 271:13875-13881.

---

Access the most updated version of this article at [http://www.jbc.org/content/271/23/13875](http://www.jbc.org/content/271/23/13875)

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 36 references, 12 of which can be accessed free at  
[http://www.jbc.org/content/271/23/13875.full.html#ref-list-1](http://www.jbc.org/content/271/23/13875.full.html#ref-list-1)
