
JOURNAL OF BACTERIOLOGY, Feb. 1997, p. 919–927
0021-9193/97/$04.00+0
Copyright © 1997, American Society for Microbiology

# Characterization of the *p*-Toluenesulfonate Operon *tsaMBCD* and *tsaR* in *Comamonas testosteroni* T-2

FRANK JUNKER,$^{1}$ ROLAND KIEWITZ,$^{2}$ AND ALASDAIR M. COOK$^{1,2\ast}$

*Institute of Microbiology, Swiss Federal Institute of Technology, ETH-Zentrum, CH-8092 Zürich, Switzerland,*$^1$  
*and Faculty of Biology, The University, D-78434 Konstanz, Germany*$^2$

Received 3 September 1996/Accepted 18 November 1996

*Comamonas testosteroni* T-2 uses a standard, if seldom examined, attack on an aromatic compound and oxygenates the side chain of *p*-toluenesulfonate (TS) (or *p*-toluencarboxylate) to *p*-sulfobenzoate (or terephthalate) prior to complete oxidation. The expression of the first three catabolic enzymes in the pathway, the TS methyl-monooxygenase system (comprising reductase B and oxygenase M; TsaMB), *p*-sulfobenzyl alcohol dehydrogenase (TsaC), and *p*-sulfobenzaldehyde dehydrogenase (TsaD), is coregulated as regulatory unit R1 (H. R. Schläfli Oppenberg, G. Chen, T. Leisinger, and A. M. Cook, Microbiology [Reading] 141:1891–1899, 1995). The components of the oxygenase system were repurified, and the N-terminal amino acid sequences were confirmed and extended. An internal sequence of TsaM was obtained, and the identity of the [2Fe-2S] Rieske center was confirmed by electron paramagnetic resonance spectroscopy. We purified both dehydrogenases (TsaC and TsaD) and determined their molecular weights and N-terminal amino acid sequences. Oligonucleotides derived from the partial sequences of TsaM were used to identify cloned DNA from strain T-2, and about 6 kb of contiguous cloned DNA was sequenced. Regulatory unit R1 was presumed to represent a four-gene operon (*tsaMBCD*) which was regulated by the LysR-type regulator, TsaR, encoded by a deduced one-gene transcriptional unit. The genes for the inducible TS transport system were not at this locus. The oxygenase system was confirmed to be a class IA mononuclear iron oxygenase, and class IA can now be seen to have two evolutionary groups, the monooxygenases and the dioxygenases, though the divergence is limited to the oxygenase components. The alcohol dehydrogenase TsaC was confirmed to belong to the short-chain, zinc-independent dehydrogenases, and the aldehyde dehydrogenase TsaD was found to resemble several other aldehyde dehydrogenases. The operon and its putative regulator are compared with units of the TOL plasmid.

Aromatic sulfonates are produced in the multimillion tonne range annually and dispersed in the environment as detergents, dyestuffs, and additives to products as diverse as ink and engine oil (24). These charged compounds are now readily found in the environment (see, e.g., reference 38); defined, naturally occurring sulfonates, usually aliphatic compounds, are also known (64; see also reference 76). At least five different mechanisms for the desulfonation of the aromatic compounds in aerobes are known (17, 21, 34, 39, 84; see also reference 16), and different organisms can have different strategies to attack, e.g., *p*-toluenesulfonate as a carbon source (21, 34, 61). There is no information on the genetics of these pathways, and in only one case is an extensive understanding of the enzymology and physiological controls available, the degradation of *p*-toluenesulfonate by *Comamonas testosteroni* T-2 (35, 42, 61).

*C. testosteroni* T-2, grown with toluenesulfonate as the sole source of carbon and energy, synthesizes an inducible, specific, secondary proton symport system (42) and inducible enzymes that convert toluenesulfonate (or toluencarboxylate) to sulfo-benzoate (or terephthalate) in three steps (Fig. 1) (41). These reactions require four enzymes, now termed TsaMB (*p*-toluenesulfonate methyl-monooxygenase system, comprising reductase B and oxygenase M), TsaC (*p*-sulfobenzyl alcohol dehydrogenase) and TsaD (*p*-sulfobenzaldehyde dehydrogenase), which are synthesized in one regulatory unit, termed R1 (61).

The pathway (Fig. 1) bears superficial comparison with the upper pathway of the TOL plasmid (see, e.g., reference 3) and is thus representative of a major mode of attack on aromatic compounds: degradation initiated at the alkyl side chain (41). The superficiality is immediately visible at the enzyme level. Oxygenase TsaMB (strain T-2) has been attributed to the class IA multicomponent, mononuclear iron oxygenases (5, 44, 51), whereas oxygenase XylMA (TOL plasmid) belongs to a class of multicomponent, di-iron monooxygenases (65, 70); the alcohol dehydrogenases seem to belong to different classes (41, 66), and the same was believed to be true for the aldehyde dehydrogenases (41).

Whereas it is reasonable to hypothesize that regulatory unit R1 (Fig. 1) (61) represents an operon (see, e.g., references 74 and 83), it is impossible to anticipate the type of regulation involved, e.g., LysR-, XylS/AraC-, or XylR-type (33, 53, 59). In addition, the (presumptive) genes encoding the inducible transport system for toluenesulfonate may be an integral part of an operon encoding degradative enzymes (49, 80) located close to the degradative operon (52) or found at another locus (48). The transport system for toluenesulfonate does not transport toluencarboxylate (42).

We now report more information, including two enzyme purifications (TsaC and TsaD), on the four enzymes of regulatory unit R1 in *C. testosteroni* T-2 (Fig. 1) (61), and we report that R1 is indeed an operon comprising *tsaMBCD*, which is presumably under the control of a putative transcriptional LysR-type regulator, TsaR. The genes encoding the transport system(s) are distant from this structure. This is the first report characterizing both the enzymes and the nucleotide sequence encoding the attack on the side chain of an aromatic compound.

*Corresponding author. Mailing address: Faculty of Biology, The University, Universitätsstr. 10, D-78434 Konstanz, Germany. Phone: 7531 88 4247. Fax: 7531 88 2966. E-mail: Alasdair.Cook@uni-konstanz.de.*

JUNKER ET AL.

![Diagram](https://i.imgur.com/1234567.png)

FIG. 1. The reactions catalyzed by the monooxygenase system TsaMB, the alcohol dehydrogenase TsaC, and the aldehyde dehydrogenase TsaD of regulatory unit R1 (61) of *C. testosteroni* T-2.

MATERIALS AND METHODS

**Bacteria, growth conditions and plasmids.** *C. testosteroni* T-2 (DSM 6577) was grown in batch culture at 30°C in mineral salts medium, pH 7.2, containing 6 mM toluenesulfonate (40). Most cells for enzyme purifications were grown in a 12.5-liter fermentor with an 8- to 10-liter working volume (Biostat V; Braun, Melsungen, Germany), harvested early (60 mg of protein/liter, to avoid problems with foaming and clumping) in a Pallikon cassette filtration system (Millipore, Neu-Isenburg, Germany), washed by centrifugation, and stored frozen, as in our earlier work (39). When vanillate or *p*-methoxybenzoate (6 mM) was used as the carbon source, the pH was adjusted to 7.0.

*Escherichia coli* DH5α was used as the host for pBluescript-KSII constructs and was routinely grown in Luria broth containing 100 μg of ampicillin per ml at 37°C.

**Preparation of cell extracts, enzyme assays, and enzyme purifications.** Cells were disrupted in a French pressure cell, and the crude extract, in 20 mM phosphate buffer (pH 7.0) with or without 1 mM dithiothreitol, was prepared as described elsewhere (39). Reductase TsaB was assayed at 25°C by the reduction of cytochrome *c* (39). Monooxygenase TsaM was assayed by oxygen uptake in a coupled assay with the reductase (40). Each component (TsaB and TsaM) was purified in two chromatographic steps, anion exchange and hydrophobic interaction (40).

Cell extract for the purification of alcohol dehydrogenase TsaC and aldehyde dehydrogenase TsaD was prepared without dithiothreitol. Activities were assayed at room temperature (21°C) by oxidation (TsaD) or reduction (TsaC) of *p*-carboxybenzaldehyde as described elsewhere (41). TsaC was purified in two chromatographic steps (anion exchange and hydrophobic interaction), essentially as described elsewhere (41). A new purification scheme for TsaD was developed, involving an anion exchange and subsequent gel filtration. The anion-exchange column (Mono Q HR 10/10; Pharmacia, Uppsala, Sweden) was equilibrated with 20 mM phosphate buffer, pH 6.8 (buffer A), and proteins were eluted with an increasing gradient of sodium sulfate (to 1 M, buffer B) at a flow rate of 4 ml/min as follows: from 0 to 40 min, 100% buffer A; from 40 to 68 min, 0 to 3% buffer B; from 68 to 130 min, 3 to 6% buffer B; from 130 to 200 min, 6 to 20% buffer B. Four-milliliter fractions were collected, and fraction 17 was worked up. In the second purification step the gel filtration column (Superose 12 HR 10/30; Pharmacia) was equilibrated with 150 mM sodium sulfate in 50 mM phosphate buffer, pH 6.8, at a flow rate of 0.5 ml/min and a sample volume of 300 μl was used.

Only the fraction of TsaC or TsaD with the highest specific activity was used for further work. Purifications were monitored by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) with Coomassie staining (37, 40). Purified protein bands were blotted to a polyvinylidene fluoride membrane (Immobilon-P; Millipore, Bedford, Mass.) and subjected to Edman degradation (39, 40).

Protein samples were concentrated by membrane filtration (Centricon-10; Amicon, Danvers, Mass.). The hydrophobic TsaD bound extensively to the filter unit unless the unit was first incubated overnight in 5% aqueous Tween 20 solution and then rinsed thoroughly with water.

Determinations of the molecular weights under native conditions were done with the gel filtration column. The sample volume for molecular weight markers (40) and samples was 300 μl.

**EPR.** Electron paramagnetic resonance (EPR) spectra of monooxygenase TsaM (about 2 mg in 0.3 ml) in the X band were recorded with an EPR 300 spectrometer (Bruker, Rheinstetten, Germany) at 10 K as described elsewhere (55).

**Cyanogen bromide cleavage.** Purified TsaM was separated on an SDS–12% PAGE gel, visualized by the Zn²⁺ reverse-staining method (18), and cut from the gel. The gel chip was minced, suspended in 500 μl of 0.1% CNBr, flushed with nitrogen, and allowed to react in the dark for 20 h. The reaction mixture was incubated for 2 h at 37°C with extraction solution (50% formic acid, 25% acetonitrile, 15% propanol, 10% H₂O) and then centrifuged (12,000 × g, 10 min). The supernatant fluid was taken to dryness in a vacuum centrifuge. The digest was dissolved in water, and protein fragments were separated by reversed-phase chromatography (ProRPC HR 10/10; Pharmacia). The column was equilibrated with 0.1% trifluoroacetic acid (0.5 ml/min), and the sample was eluted with an increasing gradient of methanol (0 to 100% in 50 min). The fragments eluted in two major peaks. All protein fragments of peak two were pooled and dried in a vacuum centrifuge. The pellet was dissolved in loading buffer and separated by SDS-PAGE in a 15% gel. Peptides were then stained with Coomassie brilliant blue. Three bands (14, 11.5, and 9.0 kDa) were identified, blotted onto a polyvinylidene fluoride membrane, and subjected to Edman degradation (see above). The N-terminal amino acids of the 9.0-kDa fragment were used to deduce an oligonucleotide probe.

**Enzymes and reagents.** All restriction enzymes, Klenow fragments, T4 DNA ligases, and polynucleotide kinases were obtained from Fermentas, Vilnius, Lithuania. [α-³⁵S]dATP for sequencing and [γ-³²P]dATP for oligonucleotide labeling were obtained from Dupont (Regensdorf, Switzerland).

All reagents for growth media and protein purification were purchased from Fluka (Buchs, Switzerland).

**Preparation of total DNA and plasmid DNA.** Total DNA from a 100-ml batch culture (200 mg of protein/liter of culture) of toluenesulfonate-grown *C. testosteroni* T-2 was prepared by the cetyltrimethylammonium bromide precipitation method (4). Plasmid DNA from *E. coli* was isolated with Nucleobond AX-100 columns (Macherey-Nagel, Oensingen, Switzerland).

**DNA cloning, screening, sequencing, and analysis.** Total DNA from strain T-2 was digested with restriction enzymes and separated in a 1% agarose gel. The DNA was blotted onto a Hybond-N nylon membrane (Amersham International, Amersham, United Kingdom). Degenerate oligonucleotide probes were 5′ labeled with [γ-³²P]dATP, and DNA fragments were labeled with [α-³⁵S]dATP by random oligonucleotide primers (4). Both probes were used for screening. DNA fragments were cut from the agarose gel and eluted by phenol-chloroform extraction (4). The eluted fragments were ligated into pBluescript-KSII and introduced via electroporation (4) into *E. coli* DH5α. All cloning was done with pBluescriptII.

Nucleotide sequencing of both strands was performed by the modified dideoxy chain termination method (57, 63) from double-stranded DNA templates with Sequenase (U.S. Biochemical Corp., Cleveland, Ohio), and PCR-based sequencing was performed with fluorescence-labeled nucleotides (Microsynth, Balgach, Switzerland). Nucleic acid and amino acid sequences were analyzed by using the

![Figure 2](https://i.imgur.com/8901234.png)

FIG. 2. EPR spectrum of the oxygenase component TsaM of monooxygenase system TsaMB from *C. testosteroni* T-2.

![Figure 3](https://i.imgur.com/5678901.png)

FIG. 3. SDS-PAGE, with Coomassie blue staining, of the products of purification of *p*-sulfo benzaldehyde dehydrogenase (TsaD) from *C. testosteroni* T-2. Lane A, molecular weight markers; lane B, crude extract (40 μg of protein); lane C, fraction from the anion exchanger (7 μg of protein); lane D, fraction from the gel filtration column (1 μg of protein).

VOL. 179, 1997

TABLE 1. Purification of *p*-sulfobenzyl alcohol dehydrogenase TsaC from *C. testosteroni* T-2

| Purification step | Vol (ml) | Protein (µg) | Activity (nkat) | Sp act (kat/kg) | Yield (%) | Purification (fold) |
|-------------------|----------|--------------|-----------------|-----------------|-----------|---------------------|
| Crude extract     | 2        | 45,000       | 6.5             | 0.15            | 100       | 1                   |
| Anion exchanger   | 4        | 1,300        | 1.8             | 1.4             | 27        | 9.8                 |
| Hydrophobic interaction | 0.5 | 20           | 0.26            | 13.3            | 4         | 92                  |

$p$-TOLUENESULFONATE DEGRADATIVE OPERON 921

TABLE 2. Purification of *p*-sulfobenzaldehyde dehydrogenase TsaD from *C. testosteroni* T-2

| Purification step | Vol (ml) | Protein (µg) | Activity (µkat) | Sp act (mkat/kg) | Yield (%) | Purification (fold) |
|-------------------|----------|--------------|-----------------|------------------|-----------|---------------------|
| Crude extract     | 2        | 70,700       | 140             | 2                | 100       | 1                   |
| Anion exchanger   | 4        | 648          | 124             | 192              | 88        | 96                  |
| Gel filtration    | 0.5      | 12.5         | 6               | 501              | 4         | 250                 |

Genetics Computer Group program package (University of Wisconsin, Madison, Wis.). The evolutionary tree was generated with the CLUSTAL W program (71). Nucleotide sequence accession number. The sequence data are available in the NCBI GenBank library under accession number U32622.

RESULTS

Biochemical characterizations and development of oligonucleotide probes for *tsaMB*. Both components of oxygenase TsaMB were repurified, and the molecular weights of the denatured proteins (43,000 and 36,000) were confirmed (40). The N-terminal amino acid sequences were confirmed and extended as follows: TsaM, MFIRNXWYVAAWDTEIPAEGLFHR; TsaB, ADVPTVAAVRARDVLALERHANGQ. The sequence of TsaM contained a suitable sequence of amino acids with low degeneracy of coding. The sequence of TsaB did not, and the amino acid composition (39), largely amino acids with high degeneracy of coding, implied that no such sequence could be expected. We therefore sequenced an internal fragment of TsaM to be able to derive a second oligonucleotide probe: TsaM<sub>int</sub>, KPGYIHYQANYKLIVDNLLDFTHLAXV HPT.

The flavin content of the reductase component, TsaB, was consistent with the yellow color of the protein, and the ability of the protein to reduce cytochrome *c* indicated its [2Fe-2S] ferredoxin component (see reference 39). The nature of the [2Fe-2S] center in monooxygenase TsaM was presumed from its UV spectrum to be a Rieske center. We have now used EPR spectroscopy (Fig. 2; Table 4) to confirm the presence of the Rieske [2Fe-2S] center (see reference 40).

Monooxygenase TsaMB has a broad substrate range which includes *p*-methoxybenzoate (40). The monooxygenase systems vanillate-demethylase (VanAB) and *p*-methoxybenzoate-demethylase (MbdAB) reported in the literature (6, 9) are closely related to TsaMB (see below), and *C. testosteroni* T-2 utilized both *p*-methoxybenzoate and vanillate as sole sources of carbon for growth. Extracts of toluenesulfonate-grown cells of strain T-2, however, could not oxygenate vanillate, so monooxygenase system TsaMB is not identical with the vanillate-demethylase synthesized by this organism. Similarly, extracts of *p*-methoxybenzoate-grown cells of strain T-2 could not convert toluenesulfonate to sulfobenzoate, so, because TsaMB is expressed concomitantly with the dehydrogenases yielding the latter compound (61), monooxygenase system TsaMB is not identical with the *p*-methoxybenzoate-demethylase in this organism.

Purification of alcohol dehydrogenase TsaC to give identification tags. TsaC was purified to apparent homogeneity in a two-step procedure which gave a 92-fold purification (Table 1). The N terminus was unambiguously determined to be MN LNKQVAIV. SDS gel electrophoresis and gel filtration chromatography confirmed earlier data indicative of a dimeric native protein with a subunit molecular weight of 29,000 (see reference 41).

Purification of aldehyde dehydrogenase TsaD to give identification tags. TsaD was extensively purified (Fig. 3) in a two-step procedure which gave a 250-fold purification (Table 2). The keys to this purification were treatment of the concentration membranes to eliminate extensive protein losses due to binding (see Materials and Methods), the avoidance of pooled samples which introduced massive impurities, and a brisk routine to obtain data before the enzyme lost activity. We thus avoided pitfalls which led to separation of the wrong protein (41).

The subunit molecular weight determined by SDS-gel electrophoresis was 56,000. The protein smeared on gel filtration chromatography, presumably due to interaction of the extremely hydrophobic protein with the stationary phase, and we assume a homodimeric native protein (Table 2). The N-terminal amino acids were determined to be XSTVLYRXPEL.

Cloning, sequencing, and identification of the *tsaMBCD* and *tsaR* genes. We made a restriction digest map of the *tsaM* locus (Fig. 4) with the aid of an oligonucleotide probe representing the N terminus of TsaM (as indicated in Fig. 5) and confirmed the map with a probe representing the internal amino acid sequence (see above). Total DNA was subjected to digestion with *SalI* and *BglII*, and fragments in the 4-kb region, localized in a Southern blot with a probe for *tsaM*, were ligated into pBluescript-KSII. Clones (250) were screened with the probe for *tsaM*, and a single clone was identified. Both DNA strands of the 4-kb *SalI-BglII* insert were partially sequenced. One complete (*tsaR*, see below) and one incomplete open reading frame (ORF) were detected. The incomplete ORF included the N-terminal and internal sequences from the monooxygenase component, TsaM. To complete this sequence, a 6-kb *KpnI* fragment (Fig. 4), which overlaps the 4-kb *SalI-BglII* fragment, was cloned and partially sequenced. The 6-kb fragment was identified with a 0.3-kb *KpnI-BglII* probe derived from the 4-kb *SalI-BglII* clone. Downstream of *tsaM* we located a complete ORF, whose deduced sequence includes a section identical to the N-terminal sequence of reductase TsaB. In total, 6,040 bp were sequenced and five ORFs were found and analyzed (Fig. 4 and 5).

Sequence and characterization of monooxygenase TsaMB. The start codon (ATG) of monooxygenase *tsaM* was found at position 1477 (Fig. 5); the stop codon (TGA) was found at position 2518. We located a putative Shine-Dalgarno sequence (AAGGAG, position 1464) upstream of this gene (68). The start codon of reductase *tsaB* was found at position 2517, the stop codon was found at position 3468, and a putative Shine-Dalgarno sequence (CGGGA) was found at nucleotide 2505. The stop codon of *tsaM* (TGA) and the start codon of *tsaB* (ATG) overlap (Fig. 5). Gene *tsaM* has a coding capacity for 347 amino acids, giving a calculated molecular weight ($M_{r,\text{calc}}$) of 39,557, and gene *tsaB* has a coding capacity for 317 amino acids ($M_{r,\text{calc}} = 34,186$); these molecular weights corresponded to the values (43,000 and 36,000, respectively) obtained with SDS-PAGE.

A sequence analysis of TsaM confirmed the presence of motifs attributed to a Rieske [2Fe-2S] center (e.g., Cys-48,

The region downstream of $tsaMBCD$. No ORF was found downstream (370 bp) of $tsaMBCD$. A putative transcription stop signal [stem-loop and poly(T) sequence] in the region from 5688 to 5718 and the functionality of the upstream region (see below) indicate that this gene cluster represents an operon.

Sequencing of the putative LysR-type transcriptional regulator, $tsaR$. At 114 bp upstream of $tsaM$ an ORF, termed $tsaR$, was found in the reverse orientation to $tsaMBCD$. The putative start codon (GTG) of $tsaR$ was attributed to position 1362, the stop codon (TGA) was attributed to position 468, and a potential Shine-Dalgarno sequence (CGGGA) was attributed to position 1374. A sequence analysis showed that $tsaR$ would encode a protein containing 298 amino acids ($M_{r,\text{calc}} = 32,630$) with similarities, largely at the N terminus (where the helix-turn-helix motif for DNA binding was located), to LysR-type transcriptional regulators (Table 3; Fig. 5 [where only highly conserved residues of the $\alpha\beta\alpha$ structure are marked]) (59).

The region downstream of $tsaR$. No ORF was found downstream (465 bp in the direction of transcription) of $tsaR$. A putative transcription stop signal [stem-loop and poly(T) sequence] in the region from 387 to 407 and the location of the $tsaMBCD$ operon indicate that this single gene constitutes a separate transcriptional unit.

Structure of the region between $tsaMBCD$ and $tsaR$. The intergenic region between $tsaR$ and $tsaM$, 114 bp, was found to have a GC content of 48%, much lower than the 70% observed in the coding regions. A low GC content of the intergenic region has been observed previously between a LysR-type regulator gene and the neighboring gene(s) it regulates (77). One putative recognition site for a LysR-type regulator was found, position 1376 (TTC-8 bp-GAA), which corresponds to Schell's model (59). A putative sigma ${}^{70}$ "housekeeping" promoter of operon $tsaMBCD$ is located upstream from $tsaM$, though the Pribnow box TAGCAT at position 1442 and the $-35$ region (1418; TTGTTG) show only weak homologies with the consensus sequences (45). Another potential promoter, which could be recognized by a sigma ${}^{70}$ factor, lies upstream of $tsaR$: the Pribnow box (position 1401; TATAA; reverse strand) is identical to the consensus, whereas the $-35$ region (position 1424; ACAACA; reverse strand) has poor homology with the consensus sequence.

DISCUSSION

Each of the genes in the cluster $tsaMBCD$ has been identified from the deduced N-terminal amino acid sequence (Fig. 5) and by its size. This cluster thus encodes the inducible enzymes which catalyze the reactions (Fig. 1) of the physiologically defined regulatory unit R1 involved in the degradation of toluenesulfonate in *C. testosteroni* T-2 (61). The evidence that the cluster represents a four-gene operon lies in the absence of neighboring genes (in the direction of transcription (Fig. 5), in the putative stop signal (Fig. 5), and in the detection of a 4.5-kb mRNA (33a), i.e., a polycistronic messenger of the correct length.

We presume that the putative LysR-type regulatory gene, $tsaR$, is expressed as a one-gene transcriptional unit and that its product is involved in both autoregulation and regulation of expression of the neighboring $tsa$ catabolic operon. $tsaR$ is typical of the supergene family, both from its structure and from the nature of the intergenic region with which it would have to interact (59), but only experiments will confirm this hypothesis. The induction of operons controlled by LysR-type regulators takes place through the binding to that protein of a substrate or an intermediate of the pathway, so toluenesulfon-

VOL. 179, 1997

$p$-TOLUENESULFONATE DEGRADATIVE OPERON

PstI
1 GACGTGCTCCCGTTACTGGTACCCCGGCCACCGCCTTGTA GTGGAGGTCGTGCGGGTG 60
61 GTCCGGGACGGCCCCACCACGAGTGGCCGGCCGCACGGAGACCGGCA C 120
121 CGGCTCCCACGGGTGGGTGGTCCACTACATGGCCGGTCTTCTGCGCCGGTCG 180
181 GAACAGCTCGACGGGGTAGCGGTGTCGGCTAGTTCCGGGGTGGACGGCCCGTC 240
241 CTACCGGGGCTACCGGCGGTGCGTGAAACCCGCTCAAGTCGGGCGCACGCTTAGCT 300
301 CGGCGCGTGCAGGTGCTCCTAGTGGTCGGTGGGAACCGGTCGGGCTAGGTCGGTC 360
Stem Loop Stem
361 GTCGCGTGCGGGTGGTCCGGCTCGGTTTTGGCCACTACGGGTAGTGGGTATCCGGCGG 420
PstI
421 TCCTCGTGCGACCGCCCCATGTCGGTAAGAACGGTCGGGTACGTGGGCGACGTC 480
End G T Q L
481 CGGCACCACGACTTAGGTCGCTAGTCGGCCGGCGCCCGGAGTGGCCGGTCACGAC 540
A H H Q I W R I L G A A A P T V P L D H
PstI
541 GGCCGCGTGTGCATTACCACCCCAAACCGTCCCGCAGAACGTGCCCCTACGAGTCTC 600
R R L V Y I T P N P L A D Q L L P I S C L
601 GACCAAGAAACTTCCGCAAACGGAGCATCTCCCACGGCCGCGTACACCACTCGTCCAGCCT 660
Q D K F A N R E Y L T R P M T T L L D S
661 TACCCGGTGGTGGCCCGTCCGGTCTTGTCAAGCGTGGTCCGGGTCGAACCCGAG 720
H A V V G P L A L F S E C V L G L K P E
721 CCCGTCCGGCATCGCCCGCTTCCGCAACGCTTACTACCGCGGCCCCCGCGCGCGCGA 780
P L G Y R A F A N R I I A G P G R P A S
781 CCTCTTTGGGTGGCGTAGAACGTGAGCCGGTCCGCGACCCGAAACGGTACCCAC 840
S F A W R C E Q L E A L R T A N A M P H
841 GACCGCGACCGGCTGCTACTGGTGCAGCGAGTCATGTCGCCACCCGAAAGTCCAGCCA 900
Q R Q G V I V D S V Y L P Q A E L D T
901 CAGCTACACGGCACGAAACCCCGCCGCACTCCGGCTCGGTCGACGCGGCGGTTGAC 960
D I D H K H A A T L A F D L T G D R L Q
961 CCCGCTGTGGCGCCCACATGTAGCGCGTGCAAGTGCCACTGCGACCCCCTTTGAGCGC 1020
P S V A P Y M G D R V N T V D P F E R
1021 CCGTTTTCGACCCTGGCGGTCGGCCGGGCGTCCACGGTTCACCCCGGCTCCGGCGCTT 1080
A F S A L A L P L A A I A P S A A F
1081 CCACACTACCGGGGAGGGTGCGGGGCGCTTCGACCCGCTAAAGGAAACGCGCCGGCGGA 1140
T I H G* E W R G R L Q G I E E Q A R R S
1141 GAGCCAGTGCTAGTGGCGCGCACGAAGTACTCCCGACGGGCTTCTCGACTCGAGTG 1200
E T V I L R A H K M F A Q G F S T* L* S V
1201 CGGCGGAAACCACGGGTGTCGGCGCCCGGAAAGTCGAGCAAGGTCAACGACTACC G 1260
G R* K T R* V L L* P A K L E D E* L* Q Q I A
1261 GCGGCTGTCCCGCCCAACCGCTCTCCACGTGTGACGGCGGGCGCGTCCACGGGTTG 1320
A S L* A P* Q* S L H* L L Q A* A* A R L S G* V
PstI
TTCCTCAATGCAGATGAGTGCCCTGCAGGCTCTGGAGCTTCACGGGCAATGTCCGGTTCA G 1380
AAGGAGTTACGTCTACTCACGGAGCTGCAGACCTCGAAGTGCCGTTACAGGGCCAAGTC
E E I C I L A Q L T Q L K V←tsaR *****
TGATTTGAAAAAACGATTATATCGCCGAAAAAAAAAGATTGTGTGTTTCAGTGCTCGCTCA
1381 ----------------+----------------+----------------+ 1440
ACTAACCTTTTGGCTAATATAFGCCCTTTTTGCTAAAACACAAGTCACGAGCGAGT
******
CTAGCATCCCCAAAACCCCAACAGGAGACACTGCATGTTCACTCCGCAATTGCTGGTAC 1450
GATCGTAGGGGGTTTTGGGGTTTCTCTGTGACGTTCAAGTGGCTTAACGACCATG 1500
tsaM → M F I R N C W Y
1501 GTGGCGGCTGGGACACC GAAATTCGGCCGAAAGGCTGTTCACCGGACCCCTGCTCAAC 1550
V A A W D T E I P A E G L F H R T L L N
1561 GAGCCCGTTCTGCTACCGGCAACCCAGGGCGGTGGGCGCTGAAACCCGCTGC 1600
E P V L L Y R D T Q G R V V A L E N R C*
1621 TGCCACCGCTCGCCACCGCTGCACATTGGCCGCAAGGGCACTTGGCTGCGCTGCTG 1650
C H* R S A P L H I G R Q E G D C V R C* L
1681 TACCA CGG CCTGAAGTTCAACCCCAACGGCCCTGCGCTCGAGAATTCGGGGCAGAAACAG 1730
Y H* G L K F N P S G A C V E I P G Q E Q


1741 ATTCGCCCCAAGACCTGCATCAAAGACTACCCGGTGGTAGCGCAAACGGCCTGGTCTGG 1800
I P P K T C I K S Y P V V E R N R L V W
1801 ATCTGGATGGGCGACCCGGCGCGCCAAACCGACAGCATCGTGGATTACTTCTGGGAC 1860
I W M G D P A R A N P D D I V D Y F W H
1861 GACTCGCCGAGTGGCGATGACCCGGGCTACATCCACTACCGGCAAACATCAAGCTG 1920
D S P E W R M K P G Y I H Y O A N Y K L
1981 ATCGTGGACAACCTGCTGGACTTCCACCCTGGGCTGGTGAACCCCAACAGCTGGGC 1980
I V D N L L D* F T H* L A W V H* P T T L G
1981 ACCGACAGCGGCCCCTCGCTCAAGCCCGTATCGAGCGGCAACCACTGGCACTGGCAAG 2040
T D S A A S L K P V I E R D T T G T G K
KpnI
2041 CTCACCATCACCCGCTGGTACCTCAACGAGCACATGTCACCAACCTGCACAAGGGCGTGGCC 2100
L T I T R W Y L N D D M S N L H K G V A
2101 AAGTTCGAGGGCAAGGGACCCGCTGGCAGATTTACCAATGGCCCGCCCCCGCTGCTG 2160
K F E G K A D R W Q I Y Q W S P P A L L
2161 CGCATGGACACC GGGCTCGGCGCCCACCGGGCAGGGCCGCGGTGCCC 2220
R M D T G S A P T G T G A P E G R R V P
2221 GAGGCGGTGCAGTTCGCCACACCTCGATCCAGACCCCCGAGACCGAGACCCACAGCCAC 2280
E A V Q F R H T S I Q T P E T E T T S H
BglII
2281 TACTGGTTCTGCCAGGCGGCAA CTTGCGACCTGGAGACGGGCCCCTCAGCCGAGAAGATC 2340
Y W F C Q A R N F D L D D E A L T E K I
2341 TACCAAGCGTGTTGGTGGCCTTTGAAGAAGACCCGACCATGATCGGGCGCACAAAAA 2400
Y Q G V V V A F E E D R T M I E A H E K
2401 ATCTTGAGCCAGTGGCCGATGGCCCATGGTGCCCGACGGCGGGGCTGAAC 2460
I L S Q V P D R P M V P I A A D A G L N
******
2461 CAGGGCCGCTGGCTGGACC GCGCTGCTCAAGGCCGAGAACGGCGGACC CGGCATGA 2520
Q G R W L L D R L L K A E N G G T A P End
tsaB → M S
2521 GCGCCGATGTGCCCGTGACCGTGGCCCGGTGGCCCGGACGGGTGCCACATGG 2580
A D V P V T V A A V R A V D V L A L
2581 TGGAACTGCGCCACGGCAA CGGCCAGCCCCCTGCCCCGGCCAGCGGCGGTGCCACAT TG 2640
E L R H A N G Q P L P G A S A G A H I D
2641 ACCTGCCCCCTGCCAAATGGCCTGGTGCCGAGGACTACTCGCTGGTGAAAGCCACCGGCCAGG 2700
L A L P N G L V R* Q Y* S* L* V N A A T G Q A
2701 CCACCATGGACTGCTACCAGGTGGCCCTGGGAGACGCCAACAGCCGTGGCCGGCTCGG 2760
T M D C Y Q V A V G W D A N S R G G S V
2761 TGTGGAATTCACGAAAAGCTCAAGTGGCCACCGCTGGCGTGACGACCCGGCGACCT 2820
W I H E K L K V G Q A L R V T T H R A T C
2821 GTTCCGAGATGGCGCCCAGGACACCCGCGTGCTGCTGGCCGGTGCAATTGGCGTCA 2880
S E M A P E H R V V L L A G* G* I* G* V T*
2881 CGCCCACTATGCCCATGGCCACGGCTTGGCGCAGAAAGCGTGGAGTGGCTGGGG 2940
P* I Y A M* A Q A C A Q Q G V D V E L W A
2941 CCAGCGCCGCTCGGCCCCGGCCTACTGGGAAAGAACTCAAGGCACTGCTGGGGCC 3000
S A R* S A P R L A Y L E E L K A L L G Q
3001 AGCCCTCCACGGCTACGGCAGACCGAGCCCGCCCCCATGAGAACCTCACGGGCC 3060
R L H L H A D D E Q G G P M N L T E R L
3061 TGCCCACCCAGCGCTGGGACGGCTTTAGCCCTGGCCCGCCCATGCTGGAGCGGC 3120
A T Q R W D A V Y A C G P* A P M L D A L
3121 TCACCGGCCACCCGGCTAGGTTCGGTGCTCGCCCGGAAGACCGCGCCGAACAGCGGGAATGA 3180
T A A T A H W A P G S V R M E R F K G A
PstI
3181 CGGAGCAGCCCGCCAGCGAGCGCCAGCCCTTTGAGCTGGTGCTGCGACGCGCGCCGGCTGA 3240
E Q P A S E R Q P F* E L V L Q R A G L S
3241 GCACCACGGTGGACGGCCAGAGGTGCTCGAGCGGCACTGGAGCGCGGTGGTGGA CT 3300
T T V D A H E S V L D A M E R V G V D F
3301 TCCCTTGGTCTGGCGGAAGGCATTTGCGGCACCTTGCGAAAGCGCCGGTGCTCGAAAGCG 3360
P W S C* R E G I C* E A P V L E G E
3361 AGGTGCAGCACCTCGATTACGTGGCTCTCGCCCGGAAGACCGCGCCGAACAGCGGGGAATGA 3420
V Q H L D Y V L S P E E R A E Q R R M M


3421 TGGTCTGCGGTGTCGCGTTGTGGCGGGCCGGGCTGGTCTGGGACATCTGAACAACCAACC 3480
V C* V S R C G G R L V L D I End
3481 ACCATGAAACCTGAAACAAAACAGTGGCCCATCGTGGACGGTGGCCGGCCTTTGGCGC 3540
tsaC → M N L N K O V A I V T*G* G A S G* F G* A
3541 GCCATCGCCCGCCGACTCAGCCAGGCCGGCGGGCTGGTGGGCCACCTCAAAGCC 3600
A I A R R L S Q A G A A V L V A D L N A
3601 GAGGGCGCGCAGCGCATGGCAACGAACTCAACGCGCTGGCCGGCCCTGGGCATG 3660
E G A Q R M A T E L N A A G G R A L G M
3661 GCCTGCGAGTGTGCGAAAAGGCCGACTACCGCGGGTGTGCGATGCCCGCATCGCCCAG 3720
A C D* V S K E A D Y R A V V D A A I A Q
3721 CTGGCGGCTGTCACATCGTGGTAAACACGGCCGACACCGCACCACCAAGCCGGCG 3780
L G G L H I V V N N N A G T T H R N K P A
AccI
3781 CTGGCCGTGACCGAAAGCGAGTTGCACCGCGTCTACCGCGTGAACCTGAAAAGCGTGTAC 3840
L A V T E D E F D R V Y R V N L K S V Y
3841 TGCTCTGCCCACTGCGCTTCCCGCACTTGCCCAACAGGCGCATGGGGTGATGGTCAAC 3900
W S A Q C A L P H F A Q Q G H G V M V N
SmaI
3901 GTGGCCTCCACCAACGGTGCGCCCCGGCCGGCTGACCTGGTACAGGGCGAGCAAG 3960
V A S* T T G V R P G G L T W Y* S G S K*
3961 GCCGCATGTATCAACCTCACCAAAGGCTGGCGCTGGAGTTGCCCCCTCGGGCGTGGCG 4020
A A M I N L T K G L A L E F A R S G V R
4021 ATCAACGGGTGAAACCATGTATGGCGGAGACCCCATGATGGCCGACTTCATGGGCATG 4080
I N A V N P M I G E T P M M A D F M G M
4081 GAAGACACCCCGCCAAACCGGAGGGCTCTCCTCGCGCATTCGGCCCTTTCACC 4140
E D T P A N R E R F L S R I P L G R F T
4141 CGCCCGGACAGTGGCCTCGGCGTTTCTCGGCTGATGACGCCAGTTCTCTC 4200
R P D D V A S A V A F L A S D D A S F L
*****
4201 ACCGGCGTGTG CCTGGACGTGGACGGGGGCGCAACATATGACACC GGTCTCTACCGCT 4260
T G V C L D V D G G R N I End
tsaD → M S T V L Y R C
4261 GCCCCGAGCTGCTGATCGGGGGGCGAATGGGGCGGCTGGCCACAGGCAACGCGCTGGTG 4320
P E L L I G G E W R P G R H E Q R L V V
4321 TGCGCAA CCCGCCACCCGGGAGCGGCGCTGGAGCTGGCCCACTGCGCACGAGCACC 4380
R N P A T G E P L D E L R L A S A D D L
4381 TGCAACTGCGCTTGC AAAACCCAGCGACAGACCTTTTGAGCACTGGCGCCAGTTGCCCGC 4440
Q L A L Q T T Q A F E H W R Q V P A H
4441 AGAGCGCTGCGCCC GCGCTTGGAGCGGGGCTGCGCGGAGAACACCGAGGAGCA 4500
E R C A R L E R G V A R L R E N T E R I
4501 TGCCCA CCTGCTCAGCGTGGAGCAGGGCAAGACCTGGGCGAGGCCCGCATGGAATGGG 4560
A H L L T L E Q G K T L A E A R M E C A
4561 CCATGGCGGCGACCTGCTA CAAGTGTAGCCGGAAGAAGCGGCGCGCTATGGCCCGG 4620
M A A D L I K W Y A E E A R R V Y G R V
4621 TGATCCCCGGCCCCCTGCCC AAACGGCGATGGAGTTGTTCAAGTTCCCGGTGGCCCGG 4680
I P A R L P N S R M E V F K F P V G P V
4681 TGCGGCGGTCTCGGCCCCTT GGAACCTTCCCTGGTCTCGGCGCGCAAGCTGGGCGG 4740
A A F S P W N F P V L V S A R K L G G A
4741 CCATGCGGCGGGTTGTTC CAC TGTCGTCAGGGCGGCGAGGAAACCCCGGCGCAGTG G 4800
I A A G C S I V L K A A E E T P A S V A
4801 CGGCATGGTGGGACTCTTAA CCAGGAGCTGCCCCCGGCGTGTGCAACTGCTCTACAG 4860
A M V D C L N Q E L P P G V V Q L L Y G
4861 GG TGC CCGCAGG TGTGC GCA GGC GCT ATGC G CCT GGC CGT GTT GCG CA AAG GT CAC COT 4920
V P A E V S Q A L I A S P V V R K V T F
4921 TCACCGGCTCGGTGCGCCCGCCACCTTGGCGAGCTGTCGGCCCGCCACTCAAGC 4980
T G* S V P V G* R H L A E L S A R H L K R
4981 GCATCACGGCTGGGAGCTGGGCGCCACGGCGGTGATCGTGGCGGACGCCCACATTG 5040
I T L E* L G G H A P V I V C G D A D I A
5041 CGCGCACCGTCAACCTGATGGTGCAGCAAAAGTTCCGCAAACGGCCGGCAGGCTGCGG 5100
R T V N L M V Q H K F R N A G Q A C* L A
5101 GGCCCACCCGCTTCTTCTG TGGACGGCACTTTCGTGGAGCGCCTTCCGGCG 5160
P T R F F V D R R I Y G D F V D A F G A
5161 CCACCCAGGCCC TGC GGTGGGGCGGGCATGGGCGCCGAGACCCAGATGGGCCGGTG G 5220
T Q A L R V G A M A A E T Q M G P V A
5221 CGAGCGGCGCCCAAGCGGCTGCAAGACCTCATGTCGCTCGGCTGGGGCGTGTG 5280
S A R R Q A A V Q D L I A R S V A A G A
5281 CGGCGCCGTGGCCAGCGGCTGTGGCAGGCTGTTTACCTCGTGGCGCCACCTGCTGG 5340
R P V A S A V P E A G Y F V A P T L L A
5341 CGGATGTGCGGCTGAGTGGACAGGACCCCTCGGCCCCGTGGCTCGG 5400
D V P L D A P V M S E E P F G P V A C A
5401 CGGTGCCCTTCACTGCTGGACCAAGGACCATCGCCCAACACCAACCGTATAGGCC 5460
V P F D S L D Q A I A Q A N H N P Y G L
5461 TGGCGGGCTACCTGTTCACCGGATTCGGCAAGGCCAATCCTGGGTGTCGAGCGGCTGG 5520
A G Y L F T D S A K A I L A V S E R L E
5521 AGTGGGCGAGCCTGGCGTGAACGGCATGGCGGTGCG
TABLE 3. Comparison of TsaMBCD and TsaR with related proteins

| Protein | $M_{r, \text{calc}}$ | Structure | Position | GC content (%) | Related proteins$^b$ |
| --- | --- | --- | --- | --- | --- |
|  |  | SD$^c$ | Gene |  | Name | % Identity | $M_{r, \text{calc}}$ | Structure | GC content (%) | Accession no. |
| TsaR | 32,630 (298) | ND$^d$ | 1374 | 1362–469 | 70 | Nac | 27.8 | 32,754 (305) | ND | 62 | Q08597 |
|  |  |  |  |  | ClcR | 27.2 | 32,527 (294) | ND | 56 | L06464 |
|  |  |  |  |  | SdsB | 27.1 | 32,955 (306) | ND | 74 | M86744 |
|  |  |  |  |  | CynR | 26.9 | 32,961 (299) | ND | 55 | P27111 |
|  |  |  |  |  | TfdR | 25.9 | 32,070 (295) | ND | 67 | P10086 |
| TsaM | 39,557 (347) | Tetramer | 1464 | 1477–2517 | 66 | VanA | 32.7 | 36,578 (329) | ND | 69 | P12609 |
|  |  |  |  |  | Pht3 | 23.8 | 49,297 (439) | Tetramer | 59 | D13229 |
|  |  |  |  |  | CbaA | 23.7 | 48,927 (432) | ND | 59 | U00692 |
|  |  |  |  |  | PobA | 21.0 | 46,258 (409) | ND | 67 | X78823 |
| TsaB | 34,186 (317) | Monomer | 2505 | 2517–3467 | 71 | VanB | 41.2 | 33,706 (314) | ND | 73 | P12580 |
|  |  |  |  |  | PobB | 38.9 | 33,558 (319) | ND | 73 | X78823 |
|  |  |  |  |  | Pht2 | 34.1 | 35,999 (324) | Monomer | 57 | D13229 |
|  |  |  |  |  | CbaB | 30.9 | 31,699 (288) | ND | 59 | U00692 |
| TsaC | 26,438 (252) | Dimer | 3473 | 3484–4239 | 69 | Xan | 38.9 | 26,143 (250) | ND | 69 | S47054 |
|  |  |  |  |  | BudC | 37.4 | 25,313 (241) | Tetramer | 63 | Q04520 |
|  |  |  |  |  | 3Bhd | 37.4 | 26,848 (253) | ND | 59 | P19871 |
|  |  |  |  |  | LinX | 36.8 | 25,492 (250) | ND | 59 | D23722 |
|  |  |  |  |  | Dhk1 | 36.1 | 28,393 (272) | ND | 72 | P16542 |
| TsaD | 51,010 (476) | Dimer? | 4223 | 4239–5666 | 71 | GabD | 38.7 | 51,719 (482) | ND | 58 | P25526 |
|  |  |  |  |  | AldH | 34.7 | 54,158 (497) | ND | 54 | X95396 |
|  |  |  |  |  | Ald | 34.0 | 52,142 (478) | Tetramer | 51 | P25553 |
|  |  |  |  |  | DhaC | 33.7 | 54,743 (500) | Dimer | ND | P15437 |
|  |  |  |  |  | AldH6 | 33.3 | 56,009 (512) | ND | 55 | A55684 |

$a$ $M_{r, \text{calc}}$, molecular weight calculated from amino acid composition. The total number of amino acids is in parentheses.

$b$ LysR-type regulators: Nac, nitrogen assimilation control regulator; CynR, cyanate operon regulator; SdsB, sodium dodecyl sulfate regulator; ClcR, chlorocatechol operon regulator; TfdR, 2,4-dichlorophenoxyacetate operon regulator. Oxygenases and reductases: VanAB, Pht23, CbaAB, and PobAB (see legend for Fig. 6). Group II dehydrogenases (DH): Xan, hypothetical protein; BudC, acetoin (diacetyl) DH; 3Bhd, 3-β-hydroxysteroid DH; LinX, γ-hexachlorocyclohexane dehydrochlorinase; Dhk1, putative ketoacyl reductase (granaticin polyketide synthase). Aldehyde DHs: GabD, succinate-semialdehyde DH; AldH, aldehyde DH; Ald, lactaldehyde DH; DhaC, horse cytosol aldehyde DH; AldH6, human cytosol aldehyde DH.

$c$ SD, Shine-Dalgarno sequence.

$d$ ND, not determined.

dehydrogenases (e.g., XylL) (47), which represent a very different reaction type (regeneration of the aromatic ring structure) from that of TsaC (Fig. 1). Sequence comparisons similar to those in Table 3 show that the diene-diol dehydrogenases have identities with TsaC lower than 32% (Table 3).

As anticipated from biochemical data (44), monooxygenase system TsaMB has a high level of homology to class IA oxygenase systems, the reductases showing a higher level of identity than the oxygenases (Table 3). We generated an evolutionary tree for oxygenase components of oxygenase systems with an arbitrarily set root (Fig. 6). It was apparent that, despite the similarities in class IA, there are two separate evolutionary lines in this group of oxygenase components. The subdivision of the class IA oxygenase components into monooxygenases and dioxygenases, considered with the similarity of the Rieske centers (Table 4) (27), presumably indicates differences due to the binding site for the aromatic substrate and oxygen. In contrast, no subgroups were apparent in trees obtained from alignments of sequences from reductase components (not shown). It is well known that the domain structure of the reductases in class IA differs from those in other classes. Thus, we rationalize the homogeneity in the class IA reductases in the common property of supplying electrons to the oxygenase components.

Whereas there is considerable understanding of the function of the reductases of the mononuclear iron oxygenase systems (14, 22, 44) and a steady increase in our understanding of the [2Fe-2S] Rieske center in the oxygenase (27, 28, 44), there is still very little information on the mononuclear iron itself. The long-standing data from Bernhardt’s group (7), from which no sequence data are available, are only now being expanded (72). The close biochemical similarity of monooxygenase TsaMB and *p*-methoxybenzoate demethylase (*M*<sub>r</sub>, redox centers, amino acid composition, and overlapping substrate ranges [e.g., 4-methoxybenzoate and 4-methylbenzoate]) (40) serves to link the biophysical work (7) with the biochemical and/or sequencing work done with several dioxygenase systems (Table 3).

The catalytic enzymes (TsaM, TsaC, and TsaD) encoded by the *tsa* operon do not discriminate between the sulfonate and the carboxyl substituents, whereas the subsequent enzymes do (61). Despite the similarities of both aromatic substituents, it seems that their different physical and chemical properties sometimes require specialized enzymes. Each class IA oxygenase requires an aromatic carboxyl or, as in this case, an aromatic sulfonate substituent in its substrate. As monooxygenase TsaM can accommodate a sulfonate substituent, the question of whether other class IA oxygenases can tolerate a sulfonate substituent and still oxygenate the compound arises.

The high degree of similarity of monooxygenase and dioxygenase systems seen in Fig. 6 (51), in the face of old structures

FIG. 6. Rooted evolutionary tree of class IA multicomponent oxygenases with representatives from classes IB, IIB, and III based on the amino acid composition of the oxygenases. VanA, vanillate demethylase (9); CbaA, 3-chlorobenzoate 2,3-dioxygenase (46); Pht3, phthalate 4,5-dioxygenase (49); PobA, phenoxybenzoate dioxygenase (15); BenA, benzoate dioxygenase (accession number [AC.] P07767); BnzA, benzene dioxygenase (AC. P08084); NdoB, naphthalene dioxygenase (AC. P23094). The N-terminal amino acid sequence of *p*-sulfobenzoate dioxygenase PsbA (another dioxygenase system in the degradative pathway of toluenesulfonate) (39) displays a high sequence identity value of 48% with CbaA over the available 35-amino-acid sequence, indicating that the oxygenases are closely related (46) and that PsbAC is another class IA dioxygenase system. *p*-Methoxybenzoate demethylase system MbdAB (no sequence data available) apparently shares substrates with vanillate-demethylase system VanAB (3-methoxybenzoate and 3,4-dimethoxybenzoate) and monooxygenase system TsaMB (4-methylbenzoate and 4-methoxybenzoate), whereas TsaMB does not share any common substrate with VanAB. We expect high levels of sequence similarity between monooxygenase MbdAB and the other two mono-oxygenases.

than the *tsa* catabolic operon and includes a transport function (XylN); different regulatory systems are used (sigma<sup>54</sup> promoter, integration host factor, and the NtrC-like regulator XylR in the TOL system contrasted with putative sigma<sup>70</sup> promoters and LysR-type regulation) and different types of mono-oxygenases and different gene families of alcohol dehydrogenases (class I in TOL [67] and class II in *tsa* [Table 3]) are involved. Only the aldehyde dehydrogenases (Table 3) (see the introduction) belong to the same family, with 29% sequence identity between TsaD and XylC being determined. When two types of attack on a methyl side chain differ so markedly (see also reference 31), we may expect yet more diversity when further pathways of oxidation of aromatic methyl groups are examined.

The short-chain alcohol dehydrogenases have been found frequently in aromatic metabolism, but usually as diene-diol

TABLE 4. Data from EPR spectra of the [2Fe-2S] centers in the oxygenase components of some multicomponent oxygenase systems

| Enzyme                          | Class | $g_{z}$ | $g_{y}$ | $g_{x}$ | Center | Source or reference |
|----------------------------------|-------|---------|---------|---------|--------|---------------------|
| 4-Toluenesulfonate methyl-monooxygenase | IA    | 2.019   | 1.918   | 1.790   | Rieske | This paper         |
| 4-Sulfobenzoate DOS<sup>a</sup>  | IA    | 2.025   | 1.921   | 1.745   | Rieske | 35                  |
| 4-Methoxybenzoate demethylase    | IA    | 2.008   | 1.913   | 1.72    | Rieske | 73                  |
| Phthalate DOS                    | IA    | 2.016   | 1.914   | 1.763   | Rieske | 12                  |
| 4-Chlorophenylacetate DOS        | IA    | 2.021   | 1.922   | 1.737   | Rieske | 43                  |
| 2-Halobenzoate DOS               | IB    | 2.025   | 1.912   | 1.788   | Rieske | 19                  |
| Pyrazon DOS                      | IIA   | 2.02    | 1.91    | 1.79    | Rieske | 58                  |
| Benzene DOS                      | IIB   | 2.018   | 1.917   | 1.754   | Rieske | 23                  |
| Naphthalene DOS                  | III   | 2.01    | 1.91    | 1.80    | Rieske | 69                  |

<sup>a</sup> DOS, dioxygenase system.

from the Enzyme Commission (81), is not restricted to sequence comparisons. Not only can dioxygenase systems mono- oxygenate some substrates (26, 78), but a monooxygenase system (MbdAB) can dioxygenate at least one substrate (82). When promulgated, the wording of the Enzyme Commission rule was quite correct (29). It would seem appropriate to adjust the rules of nomenclature to the biochemical and molecular biological facts, especially as a monooxygenase system in class IB has also been detected (56).

# ACKNOWLEDGMENTS

We are grateful to T. Leisinger for his ready advice and for making available laboratory space and apparatus. P. M. H. Kroneck kindly collected the EPR spectra. We are grateful for discussions with P. M. H. Kroneck, S. Vuilleumier, A. Riedel, V. Weiss-Rudolf, M. Schmid-Appert, and A. Wasserfallen and to S. Fetzner, who kindly made preprints available.

The work was supported by a grant from the Priority Programme Environment of the Swiss National Science Foundation and by funds made available by the University of Konstanz.

# REFERENCES

1. Abalain, J. H., S. Di Stefano, Y. Amet, E. Quemener, M. L. Abalain-Colloc, and H. H. Floch. 1993. Cloning, DNA sequencing and expression of (3-17) beta hydroxysteroid dehydrogenase from *Pseudomonas testosteroni*. J. Steroid Biochem. Mol. Biol. **44**:133–139.
2. Ahmad, D., J. Fraser, M. Sylvestre, A. Larose, A. Khan, J. Bergeron, M. J. Juteau, and M. Sondossi. 1995. Sequence of the *bphD* gene encoding 2-hydroxy-6-oxo-(phenyl/chlorophenyl)hexa-2,4-dienoic acid (HOP/cPDA) hydrolase involved in the biphenyl/polychlorinated biphenyl degradation pathway in *Comamonas testosteroni*: evidence suggesting involvement of Ser<sup>112</sup> in catalytic activity. Gene **156**:69–74.
3. Assinder, S. J., and P. A. Williams. 1990. The TOL plasmids: determinants of the catabolism of toluene and the xylenes. Adv. Microb. Physiol. **31**:1–69.
4. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1987. Current protocols in molecular biology. John Wiley & Sons, Inc., New York.
5. Batie, C. J., D. P. Ballou, and C. C. Correll. 1991. Phthalate dioxygenase reductase and related flavin-iron-sulfur containing electron transferases, p. 543–556. In F. Müller (ed.), Chemistry and biochemistry of flavoenzymes, vol. 3. CRC Press, Boca Raton, Fla.
6. Bernhardt, F.-H., E. Bill, A. X. Trautwein, and H. Twilfer. 1988. 4-Methoxybenzoate monooxygenase from *Pseudomonas putida*: isolation, biochemical properties, substrate specificity, and reaction mechanisms of the enzyme components. Methods Enzymol. **161**:281–294.
7. Bernhardt, F.-H., K. Gersonde, H. Twilfer, P. Wende, E. Bill, A. X. Trautwein, and K. Pfleger. 1982. Dioxygen activation by putidamonooxin: substrate-modulated reaction of activated dioxygen, p. 63–77. In M. Nozaki, S. Yamamoto, Y. Ishimura, M. J. Coon, L. Ernster, and R. R. Estabrook (ed.), Oxygenases and oxygen metabolism. Academic Press, New York.
8. Bocco, J. L., G. Panzetta, L. A. Actis, and S. Genti-Raimondi. 1993. *Pseudomonas testosteroni* beta-hydroxysteroid dehydrogenase gene, and translated products. EMBL data library accession number L08971.
9. Brunel, F., and J. Davison. 1988. Cloning and sequencing of *Pseudomonas* genes encoding vanillate demethylase. J. Bacteriol. **170**:4924–4930.
10. Busse, H.-J., T. El-Banna, H. Oyaizu, and G. Auling. 1992. Identification of xenobiotic-degrading isolates from the beta subclass of the *Proteobacteria* by a polyphasic approach including 16S rRNA partial sequencing. Int. J. Syst. Bacteriol. **42**:19–26.
11. Choi, K. Y., and W. F. Benisek. 1988. Nucleotide sequence of the gene for the Δ<sup>5</sup>-3-ketosteroid isomerase of *Pseudomonas testosteroni*. Gene **69**:121–129.
12. Cline, J. F., B. M. Hoffman, W. B. Mims, E. LaHaie, D. P. Ballou, and J. A. Fee. 1985. Evidence for N coordination to Fe in the [2Fe-2S] clusters of Thermes Rieske protein and phthalate dioxygenase from *Pseudomonas*. J. Biol. Chem. **260**:3251–3254.
13. Coco, W. M., R. K. Rothmel, S. Henikoff, and A. M. Chakrabarty. 1993. Nucleotide sequence and initial functional characterization of the *clcR* gene encoding a LysR family activator of the *clcABD* chlorocatechol operon in *Pseudomonas putida*. J. Bacteriol. **175**:417–427.
14. Correll, C. C., C. J. Batie, D. P. Ballou, and M. L. Ludwig. 1992. Phthalate dioxygenase reductase: a modular structure for electron transfer from pyridine nucleotides to [2Fe-2S]. Science **258**:1604–1610.
15. Dehmel, U., K.-H. Engesser, K. N. Timmis, and D. F. Dwyer. 1995. Cloning, nucleotide sequence, and expression of the gene encoding a novel dioxygenase involved in metabolism of carboxydiphenyl ethers in *Pseudomonas pseudoalcaligenes* POB310. Arch. Microbiol. **163**:35–41.
16. Denger, K., M. A. Kertesz, E. H. Vock, R. Schön, A. Mägli, and A. M. Cook. 1996. Anaerobic desulfonation of 4-tolylsulfonate and 2-(4-sulfohenyl)butyrate by a *Clostridium* sp. Appl. Environ. Microbiol. **62**:1526–1530.
17. Feigel, B. J., and H.-J. Knackmuss. 1993. Syntrophic interactions during degradation of 4-aminobenzenesulfonic acid by a two species bacterial culture. Arch. Microbiol. **159**:124–130.
18. Ferreras, M., J. G. Gavilanes, and J. M. García-Segura. 1993. A permanent Zn<sup>2+</sup> reverse staining method for the detection and quantification of proteins in polyacrylamide gels. Anal. Biochem. **213**:206–212.
19. Fetzner, S., R. Müller, and F. Lingens. 1992. Purification and some properties of 2-halobenzoate 1,2-dioxygenase, a two-component enzyme system from *Pseudomonas cepacia* 2CBS. J. Bacteriol. **174**:279–290.
20. Florin, C., T. Köhler, M. Grandguilliot, and P. Plesiat. 1996. *Comamonas testosteroni* 3-ketosteroid-Δ<sup>4</sup>(5α)-dehydrogenase: gene and protein characterization. J. Bacteriol. **178**:3322–3330.
21. Focht, D. D., and F. D. Williams. 1970. The degradation of *p*-toluenesulphonate by a *Pseudomonas*. Can. J. Microbiol. **16**:309–316.
22. Gassner, G. T., M. L. Ludwig, D. L. Gatti, C. C. Correll, and D. P. Ballou. 1995. Structure and mechanism of the iron-sulfur flavoprotein phthalate dioxygenase reductase. FASEB J. **9**:1411–1418.
23. Geary, P. J., F. Saboowalla, D. Patil, and R. Cammack. 1984. An investigation of the iron-sulphur proteins of benzene dioxygenase from *Pseudomonas putida* by electron-spin-resonance spectroscopy. Biochem. J. **217**:667–673.
24. Gerhartz, W. (ed.). 1995. Ullmann’s encyclopedia of industrial chemistry, 5th ed. VCH Verlagsgesellschaft, Weinheim, Germany.
25. Ghosh, D., C. M. Weeks, P. Grochulski, W. L. Duax, M. Erman, R. L. Rimsay, and J. C. Orr. 1991. Three-dimensional structure of holo 3α,20β-hydroxysteroiddehydrogenase: a member of short-chain dehydrogenase family. Proc. Natl. Acad. Sci. USA **88**:10064–10068.
26. Gibson, D. T., S. M. Resnick, K. Lee, J. M. Brand, D. S. Torok, L. P. Wackett, M. J. Schocken, and B. E. Haigler. 1995. Desaturation, dioxygenation, and monooxygenation reactions catalyzed by naphthalene dioxygenase from *Pseudomonas* sp. strain 9816-4. J. Bacteriol. **177**:2615–2621.
27. Gurbiel, R. J., P. E. Doan, G. T. Gassner, T. J. Macke, D. A. Case, T. Ohnishi, J. A. Fee, D. P. Ballou, and B. M. Hoffman. 1996. Active site structure of Rieske-type proteins: electron nuclear double resonance studies of isotopically labeled phthalate dioxygenase from *Pseudomonas cepacia* and Rieske protein from *Rhodobacter capsulatus* and molecular modeling studies of a Rieske center. Biochemistry **35**:7834–7845.
27a.Harayama, S. Lecture at ETH. Zurich, Switzerland.
28. Harayama, S., M. Kok, and E. L. Neidle. 1992. Functional and evolutionary relationships among diverse oxygenases. Annu. Rev. Microbiol. **46**:565–601.
29. Hayaishi, O. 1974. General properties and biological function of oxygenases, p. 1–28. In O. Hayaishi (ed.), Molecular mechanisms of oxygen activation. Academic Press, New York.
30. Hidalgo, E., Y. M. Chen, E. C. C. Lin, and J. Aguilar. 1991. Molecular cloning and DNA sequencing of the *Escherichia coli* K-12 *ald* gene encoding aldehyde dehydrogenase. J. Bacteriol. **173**:6118–6123.
31. Hopper, D. J. 1988. Properties of *p*-cresol methylhydroxylases, p. 247–258. In S. R. Hagedorn, R. S. Hanson, and D. A. Kunz (ed.), Microbial catabolism and the carbon cycle. Harwood Academic Publishers, New York.
32. Horn, J. M., S. Harayama, and K. N. Timmis. 1991. DNA sequence determination of the TOL plasmid (pWWO) *xylGFJ* genes of *Pseudomonas putida*: implications for the evolution of aromatic catabolism. Mol. Microbiol. **5**:2459–2474.
33. Inouye, S., A. Nakazawa, and T. Nakazawa. 1988. Nucleotide sequence of the regulatory gene *xylR* of the TOL plasmid from *Pseudomonas putida*. Gene **66**:301–306.
33a.Junker, F. Unpublished data.
33b.Junker, F., and A. M. Cook. Submitted for publication.
34. Junker, F., T. Leisinger, and A. M. Cook. 1994. 3-Sulphocatechol 2,3-dioxygenase and other dioxygenases (EC 1.13.11.2 and EC 1.14.12.-) in the degenerative pathways of 2-aminobenzenesulphonic, benzenesulphonic and 4-toluenesulphonic acids in *Alcaligenes* sp. strain O-1. Microbiology (Reading) **140**:1713–1722.
35. Junker, F., E. Saller, H. R. Schläfli Oppenberg, P. M. H. Kroneck, T. Leisinger, and A. M. Cook. 1996. Degradative pathways for *p*-toluenecarboxylate and *p*-toluenesulfonate and their multicomponent oxygenases in *Comamonas testosteroni* strains PSB-4 and T-2. Microbiology (Reading) **142**:2419–2427.
36. Kuliopulos, A., D. Shortle, and P. Talalay. 1987. Isolation and sequencing of the gene encoding Δ<sup>5</sup>-3-ketosteroid isomerase of *Pseudomonas testosteroni*: overexpression of the protein. Proc. Natl. Acad. Sci. USA **84**:8893–8897.
37. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) **227**:680–685.
38. Lange, F. T., M. Wenz, and H.-J. Brauch. 1995. Trace-level determination of aromatic sulfonates in water by on-line ion-pair extraction/ion-pair chromatography and their behaviour in the aquatic environment. J. High Resolut. Chromatogr. **18**:243–252.
39. Locher, H. H., T. Leisinger, and A. M. Cook. 1991. 4-Sulphobenzoate 3,4-dioxygenase: purification and properties of a desulphonative two-component

enzyme system from *Comamonas testosteroni* T-2. Biochem. J. 274:833–842.

40. Locher, H. H., T. Leisinger, and A. M. Cook. 1991. 4-Toluene sulfonate methyl-monooxygenase from *Comamonas testosteroni* T-2: purification and some properties of the oxygenase component. J. Bacteriol. 173:3741–3748.

41. Locher, H. H., C. Malli, S. Hooper, T. Vorherr, T. Leisinger, and A. M. Cook. 1991. Degradation of *p*-toluic acid (*p*-toluenecarboxylic acid) and *p*-toluene sulphonic acid via oxygenation of the methyl sidechain is initiated by the same set of enzymes in *Comamonas testosteroni* T-2. J. Gen. Microbiol. 137:2201–2208.

42. Locher, H. H., B. Poolman, A. M. Cook, and W. N. Konings. 1993. Uptake of 4-toluene sulfonate by *Comamonas testosteroni* T-2. J. Bacteriol. 175:1075–1080.

43. Markus, A., D. Krekel, and F. Lingens. 1986. Purification and some properties of component A of the 4-chlorophenylacetate 3,4-dioxygenase from *Pseudomonas* species strain CBS. J. Biol. Chem. 261:12883–12888.

44. Mason, J. R., and R. Cammack. 1992. The electron-transport proteins of hydroxylating bacterial dioxygenases. Annu. Rev. Microbiol. 46:277–305.

45. Moran, C. P. J., N. Lang, S. F. J. Legrice, G. Lee, M. Stephens, A. L. Sonenshein, J. Pero, and R. Losick. 1982. Nucleotide sequences that signal the initiation of transcription and translation in *Bacillus subtilis*. Mol. Gen. Genet. 186:339–346.

46. Nakatsu, C. H., N. A. Straus, and R. C. Wyndham. 1995. The nucleotide sequence of the Tn5271 3-chlorobenzoate 3,4-dioxygenase genes (*cbaAB*) unites the class IA oxygenases in a single lineage. Microbiology (Reading) 141:485–495.

47. Neidle, E. L., C. Hartnett, L. N. Ornston, A. Bairoch, M. Rekik, and S. Harayama. 1992. *Cis*-diol-dehydrogenase encoded by the TOL pWW0 plasmid *xylL* gene and the *Acinobacter calcoaceticus* chromosomal *benD* gene are members of the short chain alcohol dehydrogenase superfamily. Eur. J. Biochem. 204:113–120.

48. Nichols, N. N., and C. S. Harwood. 1995. Repression of 4-hydroxybenzoate transport and degradation by benzoate: a new layer of regulatory control in the *Pseudomonas putida β*-ketoadipate pathway. J. Bacteriol. 177:7033–7040.

49. Nomura, Y., M. Nakagawa, N. Ogawa, S. Harashima, and Y. Oshima. 1992. Genes in PHT encoding the initial degradation pathway of phthalate in *Pseudomonas putida*. J. Ferment. Bioeng. 74:333–344.

50. Plesiat, P., M. Grandguillot, S. Harayama, S. Vragar, and Y. Michel-Briand. 1991. Cloning, sequencing, and expression of the *Pseudomonas testosteroni* gene encoding 3-oxosteroid Δ¹-dehydrogenase. J. Bacteriol. 173:7219–7227.

51. Powloksi, P., and V. Shingler. 1994. Genetics and biochemistry of phenol degradation by *Pseudomonas* sp. CF600. Biodegradation 5:219–236.

52. Prieto, M. A., E. Díaz, and J. L. García. 1996. Molecular characterization of the 4-hydroxyphenylacetate catabolic pathway of *Escherichia coli* W: engineering a mobile aromatic degradative cluster. J. Bacteriol. 178:111–120.

53. Ramos, J. L., F. Rojo, L. Zhou, and K. N. Timmis. 1990. A family of positive regulators related to the *Pseudomonas putida* TOL plasmid XylS and the *Escherichia coli* AraC activators. Nucleic Acids Res. 18:2149–2152.

54. Reid, M. F., and C. A. Fewson. 1994. Molecular characterization of microbial alcohol dehydrogenases. Crit. Rev. Microbiol. 20:13–56.

55. Riester, J., W. G. Zumft, and P. M. H. Kroneck. 1989. Nitrous oxide reductase from *Pseudomonas stutzeri*: redox properties and spectroscopic characterization of different forms of the multicopper enzyme. Eur. J. Biochem. 178:751–762.

56. Rosche, B., B. Tshisuaka, S. Fetzner, and F. Lingens. 1995. 2-Oxo-1,2-dihydroquinoline-8-monooxygenase, a two-component enzyme system from *Pseudomonas putida* 86. J. Biol. Chem. 270:17836–17842.

57. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

58. Sauber, K., C. Fröhner, G. Rosenberg, J. Eberspächer, and F. Lingens. 1977. Purification and properties of pyrazon dioxygenase from pyrazon-degrading bacteria. Eur. J. Biochem. 74:89–97.

59. Schell, M. A. 1993. Molecular biology of the LysR family of transcriptional regulators. Annu. Rev. Microbiol. 47:597–626.

60. Schläffli, H. R., D. P. Baker, T. Leisinger, and A. M. Cook. 1995. Stereospecificity of hydride removal from NADH by reductases of multicomponent nonheme iron oxygenase systems. J. Bacteriol. 177:831–834.

61. Schläffli Oppenberg, H. R., G. Chen, T. Leisinger, and A. M. Cook. 1995. Regulation of the degradative pathways from 4-toluenesulfonate and 4-toluene carboxylate to protocatechuate in *Comamonas testosteroni* T-2. Microbiology (Reading) 141:1891–1899.

62. Schlaman, H. R. M., R. J. H. Okker, and B. J. J. Lugtenberg. 1992. Regulation of nodulation gene expression by NodD in rhizobia. J. Bacteriol. 174:5177–5182.

63. Schuurman, R., and W. Keulen. 1991. Modified protocol for DNA sequence analysis using sequenase 2.0. BioTechniques 10:185.

64. Seitz, A. P., and E. R. Leadbetter. 1995. Microbial assimilation and dissimilation of sulfonate sulfur. ACS Symp. Ser. 612:365–375.

65. Shanklin, J., E. Whittle, and B. G. Fox. 1994. Eight histidine residues are catalytically essential in a membrane-associated iron enzyme, stearoyl-CoA desaturase, and are conserved in alkane hydroxylase and xylene monooxygenase. Biochemistry 33:12787–12794.

66. Shaw, J. P., and S. Harayama. 1990. Purification and characterization of TOL plasmid-encoded benzyl alcohol dehydrogenase and benzaldehyde dehydrogenase of *Pseudomonas putida*. Eur. J. Biochem. 191:705–714.

67. Shaw, J. P., F. Schwager, and S. Harayama. 1992. Substrate specificity of benzyl alcohol and benzaldehyde dehydrogenases encoded by TOL plasmid pWWO. Biochem. J. 273:789–794.

68. Shine, J., and L. Dalgarno. 1975. Determinant of cistron specificity in bacterial ribosomes. Nature (London) 254:34–38.

69. Suen, W.-C., and D. T. Gibson. 1993. Isolation and preliminary characterization of the subunits of the terminal component of naphthalene dioxygenase from *Pseudomonas putida* NCIB 9816-4. J. Bacteriol. 175:5877–5881.

70. Suzuki, M., T. Hayakawa, P. J. Shaw, M. Rekik, and S. Harayama. 1991. Primary structure of xylene monooxygenase: similarities to and differences from the alkane hydroxylation system. J. Bacteriol. 173:1690–1695.

71. Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progressive multiple alignment through sequence weighting, position specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673–4680.

72. Tsang, H.-T., C. J. Batie, D. P. Ballou, and J. E. Penner-Hahn. 1996. Structural characterization of the mononuclear iron site in *Pseudomonas cepacia* phthalate DBO1 dioxygenase using X-ray absorption spectroscopy. J. Biol. Inorg. Chem. 1:24–33.

73. Twilfer, H., F.-H. Bernhardt, and K. Gersonde. 1981. An electron-spin-resonance study on the redox-active centers of the 4-methoxybenzoate monooxygenase from *Pseudomonas putida*. Eur. J. Biochem. 119:595–602.

74. van der Meer, J. R., W. M. de Vos, S. Harayama, and A. J. B. Zehnder. 1992. Molecular mechanisms of genetic adaptation to xenobiotic compounds. Microbiol. Rev. 56:677–694.

75. van der Meer, J. R., A. C. J. Frijters, J. H. J. Leveau, R. I. L. Eggen, A. J. B. Zehnder, and W. M. de Vos. 1991. Characterization of the *Pseudomonas* sp. strain P51 gene *tcbR*, a LysR-type transcriptional activator of the *tcbCDEF* chlorocatechol oxidative operon, and analysis of the regulatory region. J. Bacteriol. 173:3700–3708.

76. van Loon, W. M. G. M., J. J. Boon, and B. de Groot. 1993. Quantitative analysis of sulfonic acid groups in macromolecular lignosulfonic acids and aquatic humic substances by temperature-resolved pyrolysis-mass spectrometry. Environ. Sci. Technol. 27:2387–2396.

77. Viale, A. M., H. Kobayashi, T. Akazawa, and S. Henikoff. 1991. *rbcR*, a gene coding for a member of the LysR family of transcriptional regulators, is located upstream of the expressed set of ribulose 1,5-bisphosphate carboxylase/oxygenase genes in the photosynthetic bacterium *Chromatium vinosum*. J. Bacteriol. 173:5224–5229.

78. Wackett, L. P., L. D. Kwart, and D. T. Gibson. 1988. Benzyllic monooxygenation catalyzed by toluene dioxygenase from *Pseudomonas putida*. Biochemistry 27:1360–1367.

79. Wang, Y., M. Rawlings, D. T. Gibson, D. Labbé, H. Bergeron, R. Brousseau, and P. C. K. Lau. 1995. Identification of a membrane protein and a truncated LysR-type regulator associated with the toluene degradation pathway in *Pseudomonas putida* F1. Mol. Gen. Genet. 246:570–579.

80. Wang, Y. Z., Y. Zhou, and G. J. Zylstra. 1995. Molecular analysis of isophthalate and terephthalate degradation by *Comamonas testosteroni* YZW-D. Environ. Health Perspect. 130:9–12.

81. Webb, E. C. 1992. Enzyme nomenclature. Academic Press, San Diego, Calif.

82. Wende, P., F.-H. Bernhardt, and K. Pfleger. 1989. Substrate-modulated reactions of putidamonooxin: the nature of the active oxygen species found and its reaction mechanism. Eur. J. Biochem. 181:189–197.

83. Wyndham, R. C., A. E. Cashore, C. H. Nakatsu, and M. C. Peel. 1994. Catabolic transposons. Biodegradation 5:323–342.

84. Zürrer, D., A. M. Cook, and T. Leisinger. 1987. Microbial desulfonation of substituted naphthalenesulfonic acids and benzenesulfonic acids. Appl. Environ. Microbiol. 53:1459–1463.
