
CARBOHYDRATES, LIPIDS, AND  
OTHER NATURAL PRODUCTS:  
MNN6, a Member of the KRE2/MNT1  
Family, Is the Gene for Mannosylphosphate  
Transfer in *Saccharomyces cerevisiae*

Xiao-Hui Wang, Ken-ichi Nakayama,  
Yoh-ichi Shimma, Atsushi Tanaka and  
Yoshifumi Jigami  
J. Biol. Chem. 1997, 272:18117-18124.  
doi: 10.1074/jbc.272.29.18117  

Access the most updated version of this article at [http://www.jbc.org/content/272/29/18117](http://www.jbc.org/content/272/29/18117)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

This article cites 42 references, 25 of which can be accessed free at [http://www.jbc.org/content/272/29/18117.full.html#ref-list-1](http://www.jbc.org/content/272/29/18117.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1997 by The American Society for Biochemistry and Molecular Biology, Inc.

**MNN6, a Member of the KRE2/MNT1 Family, Is the Gene for Mannosylphosphate Transfer in Saccharomyces cerevisiae**

(Received for publication, December 23, 1996, and in revised form, May 13, 1997)

Xiao-Hui Wang‡, Ken-ichi Nakayama‡, Yoh-ichi Shimma‡, Atsushi Tanaka§,  
and Yoshifumi Jigami¶

From the ‡National Institute of Bioscience and Human Technology, Tsukuba, Ibaraki 305, Japan  
and the §Asahi Chemical Industry Co., Ltd., Fuji, Shizuoka 416, Japan

In yeast *Saccharomyces cerevisiae* the N-linked sugar chain is modified at different positions by the addition of mannosylphosphate. The *mnn6* mutant is deficient in the mannosylphosphate transferase activity toward mannotetraose (Karson, E. M., and Ballou, C. E. (1978) J. Biol. Chem. 253, 6484–6492). We have cloned the *MNN6* gene by complementation. It has encoded a 446-amino acid polypeptide with the characteristics of type II membrane protein. The deduced Mnn6p showed a significant similarity to Kre2p/Mnt1p, a Golgi α-1,2-mannosyltransferase involved in O-glycosylation. The null mutant of *MNN6* showed a normal cell growth, less binding to Alcian blue, hypersensitivity to Calcoflour White and hygromycin B, and diminished mannosylphosphate transferase activity toward the endoplasmic reticulum core oligosaccharide acceptors (Man8GlcNAc2-PA and Man5GlcNAc2-PA) *in vitro*, suggesting the involvement of the *MNN6* gene in the endoplasmic reticulum core oligosaccharide phosphorylation. However, no differences were observed in N-linked mannoprotein oligosaccharides between Δoch1Δmnn1 cells and Δoch1Δmnn1Δmnn6 cells, indicating the existence of redundant genes required for the core oligosaccharide phosphorylation. Based on a dramatic decrease in polymanose outer chain phosphorylation by *MNN6* gene disruption and a determination of the mannosylphosphorylation site in the acceptor, it is postulated that the *MNN6* gene may be a structural gene encoding a mannosylphosphate transferase, which recognizes any oligosaccharides with at least one α-1,2-linked mannobiose unit.

outer chain attached to the core-like oligosaccharide contains an α-1,6-linked polymannose backbone with branches of α-1,2-linked mannobiose capped with terminal α-1,3-linked mannose residues (1–5). In yeast, N-linked sugar chains are also modified at different positions by the addition of mannosylphosphate (6–9). Although this oligosaccharide modification significantly contributes to a major negative charge of the cell wall (8–10), less is known about its biosynthesis and function in yeast.

Ballou and co-workers (11) have isolated *mnn* mutants that are blocked at various stages of outer chain elongation. The *mnn1* mutant lacks α-1,3-mannosyltransferase activity and is defective in adding terminal α-1,3-linked mannose to both N-linked and O-linked oligosaccharides (11, 12). The α-1,3-mannosyltransferase has a property that competes with mannosylphosphate transferase (10, 14, 15). The *mnn4* and *mnn6* mutants are known to produce phosphate-deficient mannan relative to wild type cells, presenting a phenotype of less binding to the phthalocyanin dye, Alcian blue (10, 15). The *MNN4* gene has been cloned and predicted to encode a large protein containing 1,178 amino acids functioning as a positive regulator for mannosylphosphate transferase (16). The *mnn6* is a recessive mutation, indicating a lack of mannosylphosphate in the branches of the mannose outer chain *in vivo*, and is deficient in the mannosylphosphate transferase activity toward mannotetraose *in vitro* (15). Although mannosylphosphate transferase activity was decreased in the *mnn6* mutant, it was still uncertain whether the *mnn6* mutation affected the N-linked core oligosaccharide phosphorylation.

In this work, we have cloned the *MNN6* gene by functional complementation and analyzed the effects of the *mnn6* null mutation on enzymatic activities toward the core oligosaccharide acceptors. The Δ*mnn6* mutation does not affect the apparent profiles of N-linked core oligosaccharide phosphorylation *in vivo*, suggesting the presence of redundant functional genes. Characterization of enzymatic reaction product suggested that the *MNN6* gene may encode a mannosylphosphate transferase, which recognizes any oligosaccharides with at least one α-1,2-linked mannobiose unit.

---

In yeast *Saccharomyces cerevisiae* the biosynthesis of N-linked oligosaccharides has been studied in detail. The core oligosaccharide (Man8GlcNAc2) synthesized in the endoplasmic reticulum (ER)¹ is identical in yeast and mammals. The

---

*This work was supported in part by a grant-in-aid from the Research and Development Project of Basic Technologies for Future Industries from the Ministry of International Trade and Industry, Japan. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.*

The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) U43922.

† To whom correspondence should be addressed: National Institute of Bioscience and Human Technology, 1-1 Higashi, Tsukuba, Ibaraki 305, Japan. Tel.: 81-298-54-6224; Fax: 81-298-54-6220.

¹ The abbreviations used are: ER, endoplasmic reticulum; HPLC, high performance liquid chromatography; MALDI-TOF, matrix-assisted laser desorption ionization time-of-flight; CFW, Calcoflour White; PA, pyridylamine; M8-PA, Man8GlcNAc2-PA; ManP-M8-PA, monomannosylphosphorylated M8-PA; M5-PA, Man5GlcNAc2-PA; ManP-M5-PA, monomannosylphosphorylated M5-PA; ManP-M6-PA, monomannosylphosphorylated Man6GlcNAc2-PA.

---

EXPERIMENTAL PROCEDURES

Yeast Strains and Media—Strain TO3-6D, used for the cloning of *MNN6*, was a meiotic segregant from a cross of LB1425-1B, kindly provided by C. E. Ballou (University of California, Berkeley) and a strain of LB1-10B, purchased from the American Type Culture Collection (ATCC). KK4 and its isogenic *mnn6* disruptant strain XW44 were used for the Calcoflour White (CFW; Sigma) and hygromycin B (Sigma) sensitivity test. Strains YS125-15B, XW27, YS131-30A, and YS131-30D were used for microsomal membrane preparation. YS126-47D and its isogenic *mnn6* disruptant strain XW43 were used for oligosaccharide analysis. Yeast strains used in this study are summarized in Table I.

18118

**MNN6 Is the Gene for Mannosylphosphate Transfer**

Yeast strains with multigene disruptions were constructed by standard genetic methods (17, 18). Yeast strains were grown either in YPD (2% Bacto-peptone, 1% yeast extract, and 2% glucose) or in a complete minimal medium containing 4% glucose, 0.67% Bacto-yeast nitrogen base without amino acids (Difco), 0.3 M sorbitol and supplemented with the appropriate auxotrophic requirements (18). CFW and hygromycin B were separately added to autoclaved YPD/agar to a final concentration of 50 μg/ml just prior to pouring plates.

**Cloning of MNN6 by Modified Alcian Blue Staining—** Yeast genomic DNA library in YCp50, “CEN BANK” A and B, were purchased from ATCC. Transformation of yeast cells was carried out by the lithium acetate procedure (19). Transformants were selected and maintained on an SD-ura plate. The colonies from master plates were transferred to nitrocellulose filter and incubated for one more day at 30 °C. Those colonies on the filter were fixed by autoclaving at 120 °C for 1 h and stained by immersion into 0.1% Alcian blue solution until the blue color was developed on the wild type (*MNN6*) colonies at room temperature (20–30 min). Positive clones were screened as those providing a blue stain (wild type phenotype). The putative clones were further reassayed and confirmed individually by conventional Alcian blue assay (11).

**DNA Sequencing—** Bacterial strain JM109 was used for the preparation of plasmids. Restriction fragments containing a portion of the *MNN6* gene were subcloned into pRS316 vector (20). Sequencing was performed by the dideoxy chain termination method with dye primers (21) and done with the SequiTherm™ Long-Read Cycle Sequencing Kit-LC by the LI-COR model 4000L Automated Sequencer. Sequence comparisons against the GenBank or GenPept sequence data bases were performed using the FASTA (22) and BLAST (23) programs. The hydrophobicity plot was generated by the method of Kyte and Doolittle (24).

**Gene Disruption—** Disruption of *MNN6* gene was made by the single-step gene replacement procedure (25). The 4.3-kilobase pair *HpaI-SacI* fragment containing *MNN6* from pSA9-7 was digested with *BglII* and *BclI* restriction endonucleases. The *BglII* site is located 333 base pairs upstream from the ATG, and the *BclI* site is found 558 base pairs upstream from the stop codon (see Fig. 4A). This digestion removed a 1116-base pair fragment encompassing 261 amino acids of the *MNN6* sequence and further replaced it with a *BglII* fragment containing the complete *ADE2* gene from pASZ11 (26). Haploid yeast strains were used to transform with the linearized *mnn6::ADE2* DNA fragments (Fig. 4A). The disruption of *MNN6* was confirmed by Southern hybridization (data not shown). Selection of *mnn6* disruptants (*Δmnn6*) was done as follows: for strains carrying the *ade2* mutation, *Δmnn6* was selected on an SD-ade plate; for strains carrying no *ade2* mutation (like KK4), *Δmnn6* was selected by QAE-Sepharose adsorption according to the method described by Ballou (11).

**Construction of High Copy Plasmid Carrying MNN6—** A complementing fragment (*HpaI/NruI* fragment) containing the entire *MNN6* gene was excised by digestion with *KpnI/SacI*, whose sites are located in multicloning sites of pRS316-based plasmid pRSMNN6 and then inserted into the multicopy vector of pET351 and named pETMNN6. The pET351 high copy vector was constructed based on YEp351 (27), in which a *BamHI/HpaI* fragment containing the *LEU2* gene was replaced with a *BamHI/PvuII* fragment carrying *TRP1* gene from pJJ246 vector (28).

**Mannosylphosphate Transferase Assay—** The microsomal membrane proteins containing mannosylphosphate transferase activity were prepared according to the previous method (29), except that the cell pellets were frozen at –20 °C for 1 h before the cells were destroyed by glass beads using a B. Braun homogenizer. The enzyme assay was carried out by using 400 μg of protein of the high speed pellet (centrifugation at 100,000 × g for 60 min) in 50 μl of 50 mM Tris-HCl (pH 6.0), 10 mM MnCl₂, 25 or 50 pmol of acceptor (depending on the acceptor used), 0.6% Triton X-100, 1 mM GDP-mannose, and 0.5 mM 1-deoxy-mannojirimycin as an inhibitor of α-mannosidase in yeast (30) at 30 °C for 60 min. The enzyme reaction was terminated by boiling for 5 min, and the reaction mixture was ultrafiltered with ultrafree C3LGC (Millipore). The filtered solution was lyophilized and used for high performance liquid chromatography (HPLC) analysis. Man8GlcNAc2-PA, purchased from Takara Shuzo Co. (Kyoto, Japan), was used as the acceptor (50 pmol) in the enzyme assay. Man5GlcNAc2-PA acceptor (25 pmol) prepared from strain YS133-1D mannoproteins was used for the enzyme assay.

**Isolation of N-Linked Oligosaccharides from Yeast Mannoproteins—** Preparation of oligosaccharides was the same as described previously (31). In brief, yeast cells were grown in YPAD medium at 25 °C and harvested at stationary phase. Mannoproteins were hot citrate buffer-extracted, ethanol-precipitated, and further purified by concanavalin A-Sepharose. The N-linked oligosaccharides were liberated from the bulk yeast mannoproteins using glycopeptidase A (Seikagaku Kogyo Co., Tokyo, Japan), an enzyme specific to release N-linked oligosaccharides from glycoprotein or glycopeptide. Pyridylation of the oligosaccharides was performed using a commercial reagent kit (Takara Shuzo Co., Kyoto, Japan). The PA-oligosaccharides were obtained by gel filtration of pyridylaminated products on a Toyopearl HW-40F column (1.0 × 40 cm) and used for detection by fluorescence (excitation = 310 nm; emission = 380 nm).

**HPLC Analysis—** The separation of PA-oligosaccharides was carried out by HPLC using a Tosoh CCPM-II pump, a Tosoh PX-8020 controller, and a Shimadzu spectrofluorometric detector, RF-550. Phosphorylated oligosaccharides were fractionated by their size and polarity with Asahipak NH₂P-50 (0.46 × 25 cm) (Asahi Chemical Co., Tokyo, Japan) at a flow rate of 1 ml/min. The retention time of oligosaccharide largely depends on the number of sugar residues in the amine-modified column chromatography. Samples were resuspended in buffer A and injected in up to 20-μl aliquots. For analysis of mannosylphosphorylated Man8GlcNAc2-PA, the ratio of 200 mM acetic acid adjusted with triethylamine (pH 7.3) to acetonitrile was 30:70 (v/v) for buffer A and 70:30 (v/v) for buffer B. Initial solvent composition was 80% buffer A with 20% buffer B, and a linear gradient was run over 40 min, in which the percentage of buffer B increased from 20 to 100% with a flow rate of 1 ml/min. For analysis of mannosylphosphorylated Man5GlcNAc2-PA, buffer A contained a 10:90 (v/v) ratio of 200 mM acetic acid adjusted with triethylamine (pH 7.3) to acetonitrile, and buffer B was 100% 200 mM acetic acid adjusted with triethylamine (pH 7.3). The initial solvent of 100% buffer A (0% buffer B) was run for 5 min, and then the percentage of buffer B was linearly increased from 0 to 50% within 30 min; finally, the percentage of buffer B was linearly increased from 50 to 100% for another 15 min with a flow rate of 1 ml/min.

**α-1,2-Mannosidase Digestion—** Samples (enzymatic reaction product or *in vivo* acidic oligosaccharide product prepared from Δoch1Δmnn1 cells) were dissolved in 8 μl of 0.1 M sodium acetate buffer (pH 5.0). One microunit/2 μl of α-1,2-mannosidase solution (from *Aspergillus saitoi*; Oxford Glycosystems, Inc.) was added and then incubated for 15 h at 37 °C.

**Mass Spectrometry—** Matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) mass spectrometry was performed in the negative ion mode by using α-cyano-4-hydroxy cinnamic acid as a matrix. The mass spectrometer used in this work was a Finnigan Lasermat (Finnigan MAT Ltd., Hempstead, United Kingdom). Samples (100–1000 pmol) were desalted by HPLC using a Tosoh TSK-GEL Carbon-500 column (0.46 × 10 cm). Two solvents, A and B, were used. Solvent A was water containing 0.1% trifluoroacetic acid. The column was equilibrated with solvent A. After the sample injection, the proportion of solvent B was increased linearly up to 100% over 60 min. PA-oligosaccharides were detected by fluorescence (excitation = 320 nm; emission = 400 nm).

**¹H NMR—** ¹H NMR spectra of PA-oligosaccharide were measured on a JNM-A500 (JEOL Co.) at 50 °C. Samples (~10 nmol) were dissolved in 99.96% D₂O and lyophilized. After three repetitions of the above procedure, the samples were finally dissolved in 700 μl of 99.996% D₂O. The chemical shifts (δ) are expressed in ppm downfield from internal sodium 4,4-dimethyl-4-silapentane-1-sulfonate, but they were actually measured by reference to internal acetone (δ = 2.217 ppm).

---

**RESULTS**

**Cloning of the MNN6 Gene—** The original *mnn6* mutant exhibited a reduced amount of phosphomannan on the cell wall and showed less binding to Alcian blue, a dye that binds to a phosphate moiety of cell surface mannoproteins (11). The wild type cells exhibited a blue color after staining with Alcian blue, while the *mnn6* mutant showed as white. This characteristic was used to clone the wild type *MNN6* gene by complementation. Strain TO3-6D (*mnn1 mnn6*) was transformed with a yeast genomic DNA library constructed in the centromeric vector YCp50, which carries the *URA3* as a selective marker. Transformants were selected and maintained on an SD-ura plate. About 1.2 × 10⁴ transformants were screened, and two positive clones (blue) were selected from among a majority of colorless white transformants.

Restriction maps of the insert DNA on the plasmids from two positive clones were identical. A ~10-kb DNA fragment that can complement the *mnn6* mutation was isolated. To identify the smallest complementing region, further subcloning into

MNN6 Is the Gene for Mannosylphosphate Transfer

FIG. 1. Complementation analysis of isolated DNA. The restriction map of the MNN6 clone is shown. Open areas and straight lines indicate cloned yeast DNA and vector DNA, respectively. The ability of DNA fragments (from pSA9-1 to pRS-MNN6) to complement the mnn6 mutant was assayed by Alcian blue dye binding (indicated by + or -). The DNA fragment of pRSMNN6 was sequenced; the open reading frame (ORF) of MNN6 is indicated by the shaded box, and the direction of transcription is denoted by the arrow.

pRS316 vector was carried out. Plasmid pA9 (Fig. 1) contained a ~10-kb Sau3AI fragment insert at the BamHI site on YCp50 vector. pSA series plasmids ranging from pSA9-1 to pSA9-7 and pRSMNN6 (Fig. 1) were the subclones derived from plasmid pA9 and inserted into single-copy yeast vector pRS316. These plasmids were introduced into strain TO3-6D, and the complementation analysis was carried out by Alcian blue staining. The complementing region was assigned into a 2.7-kb HpaI-NruI fragment (pRSMNN6) (Fig. 1). To exclude the possibility of the suppressor gene cloning of mnn6 mutation, strain XW13 (Δmnn6::ADE2) (Table I and Fig. 4A) was crossed with strain TO3-6D (mnn6), and after sporulation, tetrads were analyzed by Alcian blue staining. All of the segregants derived from 20 tetrads showed the mutant phenotype, which was not able to bind to Alcian blue, demonstrating the gene disruption at the original mnn6 locus and confirming the cloning of the MNN6 gene.

MNN6 Is a Member of the KRE2/MNT1 Mannosyltransferase Gene Family—Sequence analysis of a 2.7-kb fragment revealed one open reading frame with 1338 base pairs, which was translated to a protein of 446 amino acids (MNN6 accession number U43922). From the GenBank™ data base, the MNN6 gene was identical with the KTR6 gene, which was reported by the genome sequencing as a family of killer toxin related genes (accession number U39205). Two potential N-glycosylation sites were found in the Mnn6p sequence (Fig. 2A). Kyte-Doolittle hydrophobicity analysis showed a potential membrane-spanning region near the N-terminus suggesting a type II membrane protein (Fig. 2B).

In addition, a homology search of the MNN6 sequence revealed that Mnn6p shares 38% identity and 79% similarity with Kre2p/Mnt1p, an α-1,2-mannosyltransferase responsible for O-linked glycosylation in yeast (32, 33). Sequence alignment of the Mnn6p and Kre2 protein families (Ktr1p, Ktr2p, Ktr3p, and Yur1p) (34–36) is shown in Fig. 3. Six cysteine residue positions in the latter half of Mnn6p were identical to those of the other proteins, suggesting a similarity of three-dimensional structures. Interestingly, Mnn6p has an additional cysteine at the 120th residue, which shares an identical position with Kre2p, but lacks one cysteine at the 229th residue, which is commonly located in the other five proteins.

Disruption of the MNN6 Gene Results in Calcoflour White and Hygromycin B Sensitivities—To study the function of Mnn6p, the MNN6 gene was disrupted by inserting the ADE2 gene (Fig. 4). MNN6 gene disruption did not affect the cell morphology and the rate of cell growth, indicating a nonessential gene for normal cell growth. The mnn6 null (Δmnn6) mutant displayed the same phenotype as the original mnn6 mutant, which provides a prominent loss of Alcian blue binding ability, while the isogenic wild type cells strongly bound the dye. A single copy plasmid containing the MNN6 gene restored the Alcian blue binding, which was lost in the Δmnn6 mutant (data not shown), suggesting the involvement of MNN6 in oligosaccharide phosphorylation. A sensitivity to a negatively charged fluorescent dye CFW and an aminoglycoside antibiotic hygromycin B was examined. As shown in Fig. 4B, the Δmnn6 mutant was sensitive to CFW and hygromycin B, while the isogenic wild type was not affected. A single copy of the MNN6 gene recovered the growth defect of the Δmnn6 mutant by CFW and hygromycin B, respectively. Since CFW binds to nascent chains of chitin and prevents both microfibril formation and cell wall assembly (37), the result may suggest a lesser charge repulsion between CFW and the cell surface in Δmnn6 mutant.

Alcian Blue Staining of Various Strains—The original mnn6 mutant was deficient in the mannosylphosphate addition to the mannose outer chain (15). This is supported by the Alcian blue staining of the isogenic pairs with or without the MNN6 gene (Δmnn1 cells and Δmnn1Δmnn6 cells in Table II, G-1). To further examine whether the MNN6 gene may affect phosphorylation of the oligosaccharide lacking a mannose outer chain, other isogenic pairs of double and triple mutant cells (Δoch1Δmnn1 and Δoch1Δmnn1Δmnn6) were constructed. The Δoch1Δmnn1 cells showed a significant dye binding, while the Δoch1Δmnn1Δmnn6 cells failed to bind the dye (Table II, G-2). The introduction of a multicopy plasmid containing the MNN6 gene (pETMNN6) into Δoch1Δmnn1Δmnn6 cells recovered the

18120 MNN6 Is the Gene for Mannosylphosphate Transfer

TABLE I  
Strains used in this study  

| Strain       | Genotype                                                                                          | Source     |
|--------------|---------------------------------------------------------------------------------------------------|------------|
| LB1425-1B    | MATα mnn6                                                                                        | C.E. Ballou |
| LB1-10B      | MATα suc2 mal gal2 cup1 mnn1                                                                      | ATCC       |
| TO3-6D       | MATα leu2 ura3 his1 or his3 mnn1 mnn6                                                             | This study |
| XHW13        | MATα leu2 ura3 trp1 his3 ade2 lys2 mnn6::ADE2                                                     | This study |
| YS125-15B    | MATα leu2 ura3 trp1 his3 ade2 lys2 och1::LEU2 mnn1::URA3                                            | This study |
| XW27         | MATα leu2 ura3 trp1 his3 ade2 lys2 och1::LEU2 mnn1::URA3 mnn6::ADE2                                  | This study |
| YS126-47D    | MATα leu2 ura3 trp1 his3 ade2 lys2 och1::LEU2 mnn1::URA3 kre2::TRP1                                  | This study |
| XW43         | MATα leu2 ura3 trp1 his3 ade2 lys2 och1::LEU2 mnn1::URA3 kre2::TRP1 mnn6::ADE2                        | This study |
| YS131-30A    | MATα leu2 ura3 trp1 his3 ade2 lys2 mnn1::URA3 kre2::TRP1                                            | This study |
| YS131-30D    | MATα leu2 ura3 trp1 his3 ade2 lys2 mnn1::URA3 kre2::TRP1 mnn6::ADE2                                  | This study |
| KK4          | MATα ura3 his1 or his3 trp1 leu2 gal80                                                            | Y. Nogi    |
| XW44         | MATα ura3 his1 or his3 trp1 leu2 gal80 mnn6::ADE2                                                  | This study |
| YS133-1D     | MATα leu2 ura3 trp1 his3 ade2 lys2 och1::LEU2 mnn1::URA3 kre2::TRP1 mnn6::ADE2 mnn4::lys2 alg3::His3   | This study |

A  
MHVLLSKKIARFLLISFVFVLALMVTINHPKTKQMSEQYVTPYLPKSLQPIAKISAEEQR  
RIQSEQEEAELKQSLEGEAIRNATVNAIKEKIKSYGGNETTLGFMVPSYINHRGSPPKAC  
FVSLITERDSMTQILQSIDEVQVKFNKFAYPWVFISQGELDGMKQEMIRQAITDSMNGD  
PELINIKFAEIPADEWVYPEWIDENKAAESLISLANVPDGDSRAVRYQARYFAGFFWRHP  
VLDEFDWYWRVDPGIKLYCDIDHDLFRWMQDEGKVFGFTLSMSEAKEANEKIWDVTKKFA  
KDFPKFISENNFKSFITKKDSEDFNCEFTSNFEIGNLFYRSPAYRKFFNYIDEEGGIF  
YWKWSDSIIHTIGLSMLLPKDKIHFFENIGFHYDKYNNCPLNDDIWNYNCNCDQGNDFT  
FRSGSCGGHYFDIMKKDKPEGWDRLP  

FIG. 2. The predicted Mnn6p sequence showed a feature of type II membrane protein. A, the deduced amino acid sequence of the MNN6 gene. The putative membrane spanning region is underlined. Two potential N-glycosylation sites are indicated by boldface type. B, hydropathy profile of Mnn6p according to Kyte and Doolittle (24) using a window of 16 amino acids. The hydrophobic region is indicated by positive values, and the hydrophilic portion is shown by negative values.

B  
![Hydropathy Profile](https://example.com/hydropathy_profile_image.png)

dye binding ability, but the effect of multicopy gene dosage was not observed on Alcian blue staining (Table II, G-2). These results suggest that the MNN6 gene may be involved in the phosphorylation in vivo not only at the outer chain portion but also the N-linked core and/or O-linked oligosaccharides. In contrast, the Alcian blue staining was not changed by the introduction of the Δmnn6 mutation into Δoch1Δmnn1Δkre2 cells, which produces the N-linked core oligosaccharide (Man8GlcNAc2) (31) and truncated O-linked chains (Man2) (32), suggesting the possibility of no apparent effect of the MNN6 gene on N-linked core oligosaccharide phosphorylation in vivo.

Reduction of Mannosylphosphate Transferase Activity toward N-Linked Core Oligosaccharide Acceptors in amnn6 Null Mutant—The assay conditions for the mannosylphosphate transferase were established by using 1 mM GDP-mannose as a donor and 50 pmol of pyridylaminated core oligosaccharide Man8GlcNAc2-PA (M8-PA; see structure shown in Fig. 8, A-1) as an acceptor in 50 μl of reaction mixture (see “Experimental Procedures”). Under these assay conditions, microsomal membranes from MNN6 wild type cells (Δoch1Δmnn1, strain YS125-15B) showed two reaction products (peaks 1 and 2) (Fig. 5, A-1), which were already identified as a monomannosylphosphorylated Man8GlcNAc2-PA (ManP-M8-PA) (16). In contrast, microsomal membranes from isogenic Δmnn6 cells (Δoch1Δmnn1Δmnn6, strain XW27) diminished corresponding peaks (Fig. 5, A-2), and the enzyme activity was restored after the introduction of the MNN6 gene into Δmnn6 cells (Fig. 5, A-3), indicating more directly the involvement of the MNN6 gene in the mannosylphosphate transferase activity toward Man8GlcNAc2-PA in vitro. However, introduction of MNN6 in a multicopy plasmid did not produce any higher enzymatic activities in wild type cells (Fig. 5, A-4), suggesting the presence of some limiting factors for this enzyme reaction.

Furthermore, the enzyme activity toward the N-linked core-like oligosaccharide derivative Man5GlcNAc2-PA (prepared from Δoch1Δmnn1Δalg3 cells, 25 pmol) (see structure shown in Fig. 8, A-2), was also compared by using microsomal membranes prepared from isogenic pairs of Δmnn1 cells (strain YS131-30A) and Δmnn1Δmnn6 cells (strain YS131-30D). The MNN6 cell exhibited the enzyme activity toward the Man5GlcNAc2-PA (M5-PA) acceptor and gave the reaction

MNN6 Is the Gene for Mannosylphosphate Transfer 18121

Mnn6p MHVLLSKKIARFLLISFVVLALMVTINHPKTKQ-MSEQYVTPY-LPK-SLQPIAK-ISA 56
Kre2P MALFLSKRLLRFTVIAGAVIVLLLTLNSNSTRQQYIPSSISAADFTSGSISPEQQVISE 60
Ktr1p ------------------M-A-----KIMIPAS-KQPV---YKK-L-GLLLVA V 24
Ktr2p ------------------M-QICKVFLTQVKKLLEVSLLFCLIAOTCWLA LV 33
Ktr3p ------------------M-SVHHKKKLMPKS-ALLIRKYQKGIRSSFGLLI 32
Yur1p ------------------MAKGGSLYIVGI-ELPIWTEMITYIFGKELFL-IR 32

Mnn6p E-EQRRIQSEQEEAELKQSLEGEAIRNATVNAIKEKIKSYGGNETTLGFMVP SYINHRGS 115
Kre2P ENDAAKLEQSALNSEASEDSEAMDEESKALKAAAEKADAPIDTKTTMDYITPSF-ANKAG 119
Ktr1p FTW---YVF---F-HGAQYARGS----APSPKYSTVLSSGSGYKYS----KVE---L-PKYTGP 69
Ktr2p PYQRQLSDSYSFFRRSREVSSRYDFTRRHMNQTTLKLSSNTYNDEPLN---KTKGIKN-Q- 89
Ktr3p TVLS-ELF---FMSGSRSP EVP I---AQGTSVSRVASKDYLMPFTD---KSQGVIH PVDDGK 84
Yur1p KYQK-ID-STYTALSQR-VKEQYDTSRRRNYFPKVKLSRNSYDDYTILNYTRQNDSDSFHL 89

Mnn6p PPKAFVSLITERDSMTQILQSIDEVQVKFNKNFAYPWFISQGELDG MKQEMIRQAIID 175
Kre2P KPKAYVTIVRNKE-LKGLLSSI KYVENKIKKFPYPWVEL-N-D-EPFTEE-FKEAVTK 174
Ktr1p REKATFVTLVRNRDL YSLAESIKSVEDRFNSKFNDWV--------LNDEEFTDEFKNVRS 123
Ktr2p RENATLLMLVRNWELSGALRS MRSLED RFNKNYQYDMTF-------LNDVPFDQEFIEATT 143
Ktr3p KEKGVMVTLARNSDLWNLVKSIRHVEDRFNNRYHDWVF------LNDQPF SDEFKRVES 138
Yur1p RENATILLIVRNSELEGALDSMRSLED RFNKYHYDWTF-------LNDVPFDQDFIEATT 143

Mnn6p SMNGDPELINIKFAEI PADEWVYPEWI-DENKAAE-SLISLANVPD GDSRAVRYQARYFA 233
Kre2P AV-S-SE----VKFGILPKEHW SYPEWT-NQTKAAEIRADAATKYIYG GSESYRHMGRYQS 228
Ktr1p ALVSG----TTKYGVIPKEHWSFPEWIDEEKAAQVRKEMGEKR IIYGDSISYRHMG RFES 179
Ktr2p AMASG----RTQYALIP AEDWNRPSWINETLFEERALQLMEEKNILYCGSKSYRNM CRFNS 199
Ktr3p ALVSG----KAKYGTI PKDHWSIPSWDTEKFDEKRLAMGKLDIPYCSSVPYRHMC RFQS 194
Yur1p SMASG----KTQYALIPPEDWNRPQWINDTLFEERLRVMEDEGVLYCGSKSYRNM CRFNS 199

Mnn6p GEFWRHPVLDEFDWYWRVDPGIKLYCBDIDHDLFRWMQDEGKVFGFTLSMSBEAKEANEKI W 293
Kre2P GEFWRHELLEEYDWYWRVEPDIKL YCBJINYDVFKWMQENEKVYGFTVSIHEYEV TIP TLW 288
Ktr1p GEFYRHPLMDDYDWYWRVEPDIKLHCBDIDYDVFKFMKDNKKKYAFAISIKEYEATIP TLW 239
Ktr2p GEFFRQKILDPYDFYFRVEPDVEYFCDFPYDPFKVMRQNNKKYG FVITMYBYEDTIPS LW 259
Ktr3p GFIWRHPLLEEYEWFWRVDTDITLFCDIQYDIKFLKVNNKKYG FILSVSEYERTIP TLW 254
Yur1p GEFFRQSIL DNYDYFYRVEPNVKYYCDFPYDPFRVMRLKGKKYG FVISLYEYEETIP TLW 259

Mnn6p -DVTKKF-AKDFPKFISENNFKSF------------------ITKKDSEDFNNGEFTSNFEIGNLNF 340
Kre2P -QTSMDF-IKKNPEYLDENNLMSE------------------LSNDNGKTYNLCHWSNEFTANLNL 335
Ktr1p ETTRKF-MEAHPELIHEN-NMLDE------------------LSYNLCHEWSNEFTAAALDL 286
Ktr2p EAVEEYLEETESADIDMESNAFGEVSNDFIGKSFGVIDSNSGYNLCHFWTNEFTGD LNFE 319
Ktr3p ETTKKF-IKKNPKFLHKN-NLMKE--ISNDDG-----------DTYNMCHFWTNEFTGSL DF 301
Yur1p DAVEEYLVA SEETILRKEDSAYABLTD SGLVGKHYPVVEANSDYNLCH EWSNEFTGD LNFE 319

Mnn6p YRSPA YRKFFNYIDEEEGLFYWKWSDSIIERTIGLSMLLPKDKIHFFENIGFH YDKYNNCP 400
Kre2P WRSPA YREYDTLDHQGCGFFYERWGDA PVHSIAAALFLPKDKIHYESDIGYH HPYPDNCP 395
Ktr1p WRSPA YSAYFDYLDERGCGFFYERWGDA PVHSEIGAALFDRSEHHFGDIGYY HVPFHSCP 346
Ktr2p FRSEKYIRFFEYLDSKCGGFY YERWGDA PVHTAASLLLKKDELIHFDELGY KHMPFGTCP 379
Ktr3p FRSDAYREYFDYLDSSCCGFFYERWGDA PVHTAASLFLDKSEHFEDGLGFH HPDFTS CP 361
Yur1p FRSD EYKHFEETLD AKCGFYERWGDA PVHSTGV SLLLLRPDELIHFDELGY FHSPFGTCP 379

Mnn6p LNDDIWNQYNCNE--DQGNDFTFRSGSCGGHYFDIMKKDKPEGWDR LP
Kre2P LDKEVYNSNNEC--DQGNDFTFQGYS CGKEYYDAQGLVKPKNWKK FRE 442
Ktr1p IDTSIRLANKCDDPSKDFTW--HSYSCTTK-FYNINKLPKPAGWQNH IG 393
Ktr2p SAYYLRLQQRCLDSNHPDNIDLNVISCLRRWWKDGSGKYFLK HD S 425
Ktr3p IEQKIRLQNKCIEPSKDV IW-TPDYFGTRK-YFSAGNYKLPP GI 404
Yur1p ASYAVRLDQRCRCKSDDESVIDITPHSCLMRWWKNGSGKYFLKEEQ DEI 428

product of peak 3 (Fig. 5, B-1). This reaction product was identified as a monomannosylphosphorylated Man5GlcNAc2-PA (ManP-M5-PA) by alkaline phosphatase digestion and subsequent mild acid treatment (data not shown). Disruption of the MNN6 gene diminished mannosylphosphate transferase activity toward M5-PA (Fig. 5, B-2), indicating the involvement of MNN6 in mannosylphosphorylation of core-like oligosaccharide Man5GlcNAc2.

Taken together, these data strongly suggest that the MNN6 gene may encode a mannosylphosphate transferase, which is involved in the oligosaccharide phosphorylation not only of the mannose outer chain but also of the N-linked core portion.

Characterization of Enzymatic Reaction Products—To determine the structure of reaction products, peaks 1 and 2 in Fig. 5 were analyzed by MALDI-TOF mass spectrometry. The molecular ion peaks were observed at m/z 2048.9 for peak 1 and 2042.5 for peak 2, respectively. These mass values were nearly identical to the molecular mass of ManP-M8-PA (calculated \(M_r\) 2041.8). The \({}^1\mathrm{H}\) NMR spectra of peak 2 show the mannosylphosphate signal at δ 5.44 (Fig. 6B). The intensity of this signal indicates the presence of one mannosylphosphate group in peak 2. Measurement of \({}^1\mathrm{H}\) NMR spectra of peak 1 was not successful due to the loss of material during the purification process.

Since the core-like oligosaccharide has two mannosylphosphorylation sites (38), two structures of ManP-M8-PA are possible. One is the structure in which mannosylphosphate attaches to the side of the α-1,6-branch of core Man8GlcNAc2 (structure I); the other is the structure in which mannosylphosphate attaches to the side of α-1,3-branch of the same Man8GlcNAc2 (structure II). By α-1,2-mannosidase digestion, structure I releases three mannoses and yields ManP-M5-PA, but structure II releases two mannoses and yields ManP-M6-PA. The α-1,2-mannosidase digestion products of peaks 1 and 2 were analyzed by MALDI-TOF mass spectrometry. The molecular ion peak of each product was detected at \(m/z\) 1724.7 (ManP-M6-PA, calculated \(M_r\) 1717.5) for peak 1 and 1558.9 (ManP-M5-PA, calculated \(M_r\) 1555.4) for peak 2, respectively. These results revealed the site of monomannosylphosphorylation at Man8GlcNAc2. Peak 1 product was mannosylphosphorylated at the α-1,3-branch of core Man8GlcNAc2 (structure II) and peak 2 product was mannosylphosphorylated at the α-1,6-branch of core Man8GlcNAc2 (structure I).

The N-Linked Core Oligosaccharide Profiles Were Not Changed by MNN6 Gene Disruption in Vivo—Further to examine the effect of MNN6 gene disruption on the mannosylphosphate addition in the N-linked core portion, we analyzed the oligosaccharides of mannoproteins prepared from Δoch1Δmnn1

18122

**MNN6 Is the Gene for Mannosylphosphate Transfer**

A

FIG. 4. Disruption of the MNN6 gene. A, construction of the *mnn6::ADE2*-disrupted DNA fragment. A 1116-base pair DNA fragment (*Bgl*II/*Bcl*I fragment), which was composed of a 333-base pair upstream region from the ATG codon and 783 base pairs of the MNN6 open reading frame, was deleted. B, hypersensitivity to Calcoflour White and hygromycin B of wild type and *mnn6* null mutant. Cell cultures of isogenic strains KK4 (MNN6), XW44 (∆*mnn6*), and XW44 transformed with pRSMNN6 were spotted on YPD plates, YPD with 50 μg/ml CFW, and YPD with 50 μg/ml hygromycin B, respectively, and incubated for 2 days at 30°C.

B

| MNN6 | ∆*mnn6* | ∆*mnn6* + pRSMNN6 |
| --- | --- | --- |
| MNN6 | ∆*mnn6* | ∆*mnn6* + pRSMNN6 |
| MNN6 | ∆*mnn6* | ∆*mnn6* + pRSMNN6 |

YPD          YPD/CFW         YPD/Hygromycin B

TABLE II  
Alcian blue staining of various strains

| Group | Yeast strain           | Relative characteristics                     | Alcian blue staining$^a$ |
|-------|------------------------|----------------------------------------------|---------------------------|
| G-1   | YS131-30A              | ∆*mnn1*                                      | ++++                      |
|       | YS131-30D              | ∆*mnn1*∆*mnn6*                               | -                         |
| G-2   | YS125-15B              | ∆*och1*∆*mnn1*                              | +++                       |
|       | XW27                   | ∆*och1*∆*mnn1*∆*mnn6*                        | -                         |
|       | XW27(pET-351)          | ∆*och1*∆*mnn1*∆*mnn6*(pET-351)               | -                         |
|       | XW27(pET-MNN6)         | ∆*och1*∆*mnn1*∆*mnn6*(pET-MNN6)              | ++                        |
| G-3   | YS126-47D              | ∆*och1*∆*mnn1*∆*kre2*                       | ++                        |
|       | XW43                   | ∆*och1*∆*mnn1*∆*kre2*∆*mnn6*                 | ++                        |

$^a$ The colors developed on the cell pellet by the Alcian blue assay were classified as follows: -, white; ++, light blue; ++++, blue; ++++, dark blue.

(strain YS126-47D) and ∆*och1*∆*mnn1*∆*mnn6* (strain XW43) cells. In HPLC analysis, a peak of neutral M8-PA, two peaks of ManP-M8-PA (peaks 1 and 2), and unknown peak $x$ were observed in ∆*och1*∆*mnn1* cells (Fig. 7). Unexpectedly, the same oligosaccharide pattern was observed in ∆*och1*∆*mnn1*∆*mnn6* cells (data not shown). Especially, the ratio of peak area corresponding to ManP-M8-PA to neutral M8-PA was not changed in both cells. These results suggested the presence of functionally redundant gene(s), which may interfere the appearance of mutant phenotype due to *MNN6* gene disruption *in vivo*.

We also analyzed the structure of peak $x$, which was eluted at 39.5 min in Fig. 7. The molecular ion peaks were observed at $m/z$ 2294.7, which was nearly identical to the molecular mass of dimannosylphosphorylated Man8GlcNAc-PA (calculated $M_r$ 2283.9). ${}^1$H NMR spectra of peak $x$ showed a mannosylphosphate signal at δ 5.44, and the intensity of this signal indicated the presence of two mannosylphosphate groups (Fig. 6C). To determine the mannosylphosphorylation sites, the α-1,2-mannosidase digestion product of peak $x$ was first subjected to time-of-flight mass spectrometry analysis. However, the molecular ion peak was not observed, presumably due to the increase of negative charge in the molecule. Then the above digestion product was subjected to mild acid treatment followed by alka-

line phosphatase to convert to neutral PA-oligosaccharide. This product showed a retention time of 9.5 min in amino column HPLC (Asahipack NH$_2$P-50), which was identical to the authentic Man6GlcNAc2-PA (Takara PA-sugar chain 018) (data not shown). Therefore, the sites of two mannosylphosphates in peak $x$ were confirmed as shown by the star in Fig. 8, A-1.

DISCUSSION

We have reported the cloning and analysis of the *MNN6* gene. For the cloning, the original Alcian blue dye binding assay in a test tube was not appropriate for the colony screening due to the laborious work. To solve this problem, a modified procedure for Alcian blue staining was developed on plates. The method established in this work should be applicable to clone other yeast genes, especially genes related to the biosynthesis of cell wall components.

Sequence analysis of the *MNN6* gene predicted a type II membrane protein with 446 amino acids, which is highly homologous to α-1,2 mannosyltransferase, Kre2p/Mnt1p, involved in O-glycosylation in *S. cerevisiae* (32, 33). A significant homology between Mnn6p and *KRE2/MNT1* gene family proteins suggests the presence of similar domain involved in the recognition of a common structure of the acceptor (α-1,2-linked

MNN6 Is the Gene for Mannosylphosphate Transfer

FIG. 5. Assay of mannosylphosphate transferase activity in vitro. A, mannosylphosphate transferase activities toward the M8-PA acceptor. The positions of M8-PA and ManP-M8-PA are indicated. The yeast membranes were prepared from wild type MNN6 strain YS125-15B (Δoch1 Δmnn1) transformed with pET351 vector (A-1), isogenic Δmnn6 strain XW27 transformed with pET351 vector (A-2), strain XW27 transformed with pETMNN6 plasmid (A-3), and strain YS125-15B transformed with plasmid pETMNN6 (A-4). Cells were cultured at 25°C in SD-trp medium containing 20 μg/ml adenine. B, mannosylphosphate transferase activity toward M5-PA acceptor. The positions of M5-PA and ManP-M5-PA are indicated. Yeast membranes were prepared from wild type MNN6 strain YS131-30A (Δmnn1, B-1) and Δmnn6 strain YS131-30D (Δmnn1 Δmnn6, B-2). Cells were cultured at 30°C in synthetic complete medium containing 20 μg/ml adenine.

mannobiose) recognized by all of these enzymes. The functional relations between Mnn6p and Kre2p/Mnt1p will be investigated in future work.

Disruption of *MNN6* caused a hypersensitivity to CFW and hygromycin B (Fig. 4). The former phenotype is caused by the loss of charge repulsion between the cell surface and drug, leading to the penetration of drug through the outermost mannoprotein portion in the cell wall. It is noteworthy that hygromycin B-sensitive mutants involve not only the defects in sugar chain length, as reported (39, 40), but also in oligosaccharide phosphorylation, although its mechanism is still unclear.

We have shown that *MNN6* is involved in core oligosaccharide phosphorylation by demonstrating the loss of mannosylphosphate transferase activity in vitro toward Man8GlcNAc2 and Man5GlcNAc2 in Δmnn6 cells. Two reaction products (peaks 1 and 2) corresponding to ManP-M8-PA were identified when M8-PA was used as an acceptor. The mannosylphosphorylation site was determined by time-of-flight mass spectrometry after the α-1,2-mannosidase treatment. These sites were identical to the phosphorylation sites observed in dimannosylphosphorylated oligosaccharide in vivo (peak x compound in Fig. 7) described in this paper and to those reported for the N-linked core-like Man10GlcNAc2 oligosaccharide from carboxypeptidase Y and *mnn1 mnn9* strain mannoproteins (38, 41). When M5-PA was used for acceptor substrate, only one peak corresponding to monomannosylphosphorylated

FIG. 6. <sup>1</sup>H NMR spectra of enzymatic reaction product from Man8GlcNAc2-PA and of dimannosylphosphorylated oligosaccharide released from cell surface mannoproteins prepared from Δoch1Δmnn1 cells. A, Man8GlcNAc2-PA; B, peak 2 in Fig. 5; C, peak x in Fig. 7. Structural assignments for each H-1 signal are indicated.

FIG. 7. Analysis of N-linked oligosaccharides from mannoproteins accumulated in wild type strain YS126-47D (Δoch1Δmnn1). The positions of M8-PA and ManP-M8-PA, which are identical with standard samples reported previously (10, 16, 29), are indicated. Peak x is also analyzed and identified as dimannosylphosphorylated Man8GlcNAc2-PA, as described under “Results.” The sites of mannosylphosphorylation are shown in Fig. 8, A-1.

product was observed (Fig. 5, peak 3, B-1). Although the phosphorylation site could not be determined by α-1,2-mannosidase treatment due to the limited amount of purified material, based on combined results on both the phosphorylation sites determined for ManP-M8-PA in vitro and the structure of dimannosylphosphorylated oligosaccharide determined in vivo, the most reasonable phosphorylation site in ManP-M5-PA is shown in Fig. 8 (A-2).

Apparently, α-1,2-linked mannotriose (Manα1,2Manα1,2Man) (mannose residue for the phosphorylation is shown in boldface type) is a common structure for the phosphorylation of Man8GlcNAc2, Man5GlcNAc2, and mannose outer chain branch. Consistent with the previous result (15), we found that Δmnn6 mutant diminished the enzyme activity toward the α-1,2-linked mannotriose (Manα1,2Manα1,2Man), which mimics

18124

**MNN6 Is the Gene for Mannosylphosphate Transfer**

A-1

M₁—₂M* (site of α-1,6-branch)
     |
     6 M₃
     |   \
     M₁   M—GN—GN
     |
     2 M*
     |
     M


A-2

M₁
    \
     6 M
     |   \
     M₁   M—GN—GN
     |
     2 M*
     |
     M


of mannose 1-phosphate and galactose from GDP-mannose and UDP-galactose, respectively (13). Since these reactions involve transfer of mannose 1-phosphate from GDP-mannose, this study may also contribute to the understanding of other mannosylphosphate transferases, such as those in *Leishmania*.

**Acknowledgments**—We are grateful to T. Odani for the construction of strain TO3-6D, K. Ichikawa for the establishment of enzyme assay conditions, and Dr. T. Nishida for the construction of pET351 plasmid vector. X.-H. Wang thanks Mr. T. Odani and Dr. J. L. Horecka for helpful suggestions and discussion.

---

**REFERENCES**

1. Nakajima, T., and Ballou, C. E. (1974) *J. Biol. Chem.* **249**, 7685–7694
2. Kukuruzinska, M. A., Bergh, M. L. E., and Jackson, B. J. (1987) *Annu. Rev. Biochem.* **56**, 915–944
3. Herscovics, A., and Orlean, P. (1993) *FASEB J.* **7**, 540–550
4. Lehle, L., and Tanner, W. (1995) in *Protein Glycosylation in Yeast*, in *Glycoproteins* (Montreuil, J., Schachter, H., and Vliegenthart, J. F. G., eds) pp. 475–509, Elsevier, Amsterdam
5. Trimble, R. B., and Verostek, M. F. (1995) *Trends Glycosci. Glycotech.* **7**, 1–30
6. Mill, P. J. (1966) *J. Gen. Microbiol.* **44**, 329–341
7. Cawly, T. N., and Ballou, C. E. (1972) *J. Bacteriol.* **111**, 690–695
8. Thieme, T. R., and Ballou, C. E. (1971) *Biochemistry* **10**, 4121–4129
9. Raschke, W. C., and Ballou, C. E. (1971) *Biochemistry* **10**, 4130–4135
10. Ballou, C. E., Kern, K. A., and Raschke, W. C. (1973) *J. Biol. Chem.* **248**, 4667–4673
11. Ballou, C. E. (1990) *Methods Enzymol.* **185**, 440–470
12. Nakajima, T., and Ballou, C. E. (1975) *Proc. Natl. Acad. Sci. U. S. A.* **72**, 3912–3916
13. Carver, M. A., and Turco, S. J. (1992) *Arch. Biochem. Biophys.* **295**, 309–317
14. Rosenfeld, L., and Ballou, C. E. (1974) *J. Biol. Chem.* **249**, 2319–2321
15. Karson, E. M. and Ballou, C. E. (1978) *J. Biol. Chem.* **253**, 6484–6492
16. Odani, T., Shimma, Y., Tanaka, A., and Jigami, Y. (1996) *Glycobiology* **6**, 805–810
17. Sherman, F., and Hicks, J. (1991) *Methods Enzymol.* **194**, 21–37
18. Sherman, F., Fink, G. R., and Hicks, J. B. (1986) *Methods in Yeast Genetics*, pp. 163–167, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
19. Ito, H., Fukuda, Y., Murata, K., and Kimura, A. (1983) *J. Bacteriol.* **153**, 163–168
20. Sikorski, R. S., and Hieter, P. (1989) *Genetics* **122**, 19–27
21. Prober, J. M., Trainor, G. J., Dam, R. J., Hobbs, F. W., Robertson, C. W., Zagursky, R. J., Cocuzza, A. J., Jensen, M. A., and Baumeister, K. (1987) *Science* **238**, 336–341
22. Pearson, W. R., and Lipman, D. J. (1988) *Proc. Natl. Acad. Sci. U. S. A.* **85**, 2444–2448
23. Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) *J. Mol. Biol.* **215**, 403–410
24. Kyte, J., and Doolittle, R. F. (1982) *J. Mol. Biol.* **157**, 105–132
25. Rothstein, R. (1991) *Methods Enzymol.* **194**, 281–301
26. Stotz, A., and Linder, P. (1990) *Gene (Amst.)* **95**, 91–98
27. Hill, J. E., Myers, A. M., Koerner, T. J., and Tzagoloff, A. (1993) *Yeast* **2**, 163–167
28. Jones, J. S., and Prakash, L. (1990) *Yeast* **6**, 363–366
29. Nakayama, K., Nagasu, T., Shimma, Y., Kuromitsu, J., and Jigami, Y. (1992) *EMBO J.* **11**, 2511–2519
30. Jelinek-Kelly, S., Akiyama, T., Saunier, B., Tkacz, J. S., and Herscovics, A. (1985) *J. Biol. Chem.* **260**, 2253–2257
31. Nakanishi-Shindo, Y., Nakayama, K., Tanaka, A., Toda, Y., and Jigami, Y. (1993) *J. Biol. Chem.* **268**, 26338–26345
32. Häusler, A., Ballou, L., Ballou, C. E., and Robbins, P. W. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 6846–6850
33. Hill, K., Boone, C., Goebl, M., Puccia, R., Sdicu, A.-M., and Bussey, H. (1992) *Genetics* **130**, 273–283
34. Lussier, M., Sdicu, A.-M., Camirand, A., and Bussey, H. (1996) *J. Biol. Chem.* **271**, 11001–11008
35. Lussier, M., Camirand, A., Sdicu, A.-M., and Bussey, H. (1993) *Yeast* **9**, 1057–1063
36. Mallet, L., Bussereau, F., and Jacquet, M. (1994) *Yeast* **10**, 819–831
37. Ram, F. J., Wolters, A., Hoopen, R. T., and Klis, F. M. (1994) *Yeast* **10**, 1019–1030
38. Hernandez, L. M., Ballou, L., Alvarado, E., Tsai, P.-K., and Ballou, C. E. (1989) *J. Biol. Chem.* **264**, 13648–13659
39. Ballou, L., Hitzeman, R. A., Lewis, M. S., and Ballou, C. E. (1991) *Proc. Natl. Acad. Sci. U. S. A.* **88**, 3209–3212
40. Dean, N. (1995) *Proc. Natl. Acad. Sci. U. S. A.* **92**, 1287–1291
41. Hashimoto, C., Cohen, R. E., Zhang, W. J., and Ballou, C. E. (1981) *Proc. Natl. Acad. Sci. U. S. A.* **78**, 2244–2248
42. Couso, R., Lang, L., Roberts, R. M., and Kornfeld, S. (1986) *J. Biol. Chem.* **261**, 6326–6331
43. Jars, M. U., Osborn, S., Forstrom, J., and MacKay, V. L. (1995) *J. Biol. Chem.* **270**, 24810–24817
44. Descoteaux, A., Luo, Y., Turco, S. J., and Beverly, S. M. (1995) *Science* **269**, 1869–1872

---

**FIG. 8. Mannosylphosphorylation sites at the N-linked core oligosaccharides and its enzymatic reaction.** $A$, structures for ManP-Man8GlcNAc2 and ManP-Man5GlcNAc2. The mannosylphosphorylation sites are indicated by asterisks. $B$, schematic presentation for the enzymatic reaction by Mnn6p. $M$ and $R$ represent mannose and any type of sugar residues, respectively. At least an $\alpha$-1,2-linked mannobiose structure in mannotriose is required for mannosylphosphate addition by Mnn6p using GDP-mannose as a donor and producing GMP as the other reaction product.
