
Arch Microbiol (1997) 168:102–113

© Springer-Verlag 1997

---

**ORIGINAL PAPER**

Joachim Ahlert · Jürgen Distler · Kambiz Mansouri  
Wolfgang Piepersberg  

**Identification of stsC, the gene encoding the L-glutamine:scyllo-inosose aminotransferase from streptomycin-producing Streptomyces**

Received: 7 January 1997 / Accepted: 24 March 1997

---

**Abstract** Eight new genes, *strO-stsABCDEFG*, were identified by sequencing DNA in the gene cluster that encodes proteins for streptomycin production of *Streptomyces griseus* N2-3-11. The StsA (calculated molecular mass 43.5 kDa) and StsC (45.5 kDa) proteins — together with another gene product, StrS (39.8 kDa), encoded in another operon of the same gene cluster — show significant sequence identity and are members of a new class of pyridoxal-phosphate-dependent aminotransferases that have been observed mainly in the biosynthetic pathways for secondary metabolites. The aminotransferase activity was demonstrated for the first time by identification of the overproduced and purified StsC protein as the L-glutamine:scyllo-inosose aminotransferase, which catalyzes the first amino transfer in the biosynthesis of the streptidine subunit of streptomycin. The *stsC* and *stsA* genes each hybridized specifically to distinct fragments in the genomic DNA of most actinomycetes tested that produce diaminocyclitolaminoglycosides. In contrast, only *stsC*, but not *stsA*, hybridized to the DNA of *Streptomyces hygroscopicus* ssp. *glebosus*, which produces the monoaminocyclitol antibiotic bluensomycin; this suggests that both genes are specifically used in the first and second steps of the cyclitol transamination reactions. Sequence comparison studies performed with the deduced polypeptides of the genes adjacent to *stsC* suggest that the enzymes encoded by some of these genes [*strO* (putative phosphatase gene), *stsB* (putative oxidoreductase gene), and *stsE* (putative phototransferase gene)] also could be involved in (di-)aminocyclitol synthesis.

**Key words** Streptomycin biosynthesis · Aminocyclitol · Aminotransferase · *Streptomyces griseus*

---

J. Ahlert · J. Distler · K. Mansouri · W. Piepersberg (✉)  
Bergische Universität GH, FB 9,  
Department of Chemical Microbiology, Gauss-Str. 20,  
D-42097 Wuppertal, Germany  
Tel. +49-202-439-2521; Fax +49-202-439-2698  
e-mail: piepersb@uni-wuppertal.de

---

**Introduction**

Most of the complex pathways leading to the formation of antibiotics and similar secondary metabolites in actinomycetes are still incompletely analyzed both enzymatically and genetically. This is particularly true for the extended group of (di-)aminocyclitol-containing aminoglycosides (Piepersberg 1995; Piepersberg and Distler 1997). The biosynthetic pathways of only two model compounds, streptomycins and fortimicins, have been studied more extensively by genetic and biochemical approaches (Ohta et al. 1993; Piepersberg 1995). The proposed pathway leading to the diaminoctiol moiety of streptomyacin, designated streptidine, starting from D-myo-inositol-3-phosphate, consists of a twice-repeated series of five enzyme-catalyzed steps: dephosphorylation, oxidation of hydroxyl groups, transamination, phototransfer, and transamidination (Walker 1975). Interestingly, in the streptomycete producers of a very similar compound, bluensomycin, the second set of enzyme reactions is replaced by a two-step process that introduces a carbamoyl group and results in another aminocyclitol, bluensidine (Walker 1990). In comparison to the pathway for streptidine, this phenomenon has been interpreted as being a likely ancestral pathway. The first step L-glutamine:scyllo-inosose aminotransferase seems to play a key role in aminocyclitol formation (Fig. 1). Such an L-glutamine-dependent enzyme occurs in all the aminocyclitol-producing actinomycetes tested so far, even when a 2-deoxy-scyllo-inosose is used as the acceptor substrate and the diaminoctiol 2-deoxystreptamine is formed (Walker and Walker 1969; Lucher et al. 1989; Walker 1995).

The genetics and biochemistry of the production of streptomycin-like aminoglycosides have not yet been completely worked out. In the past we have described more than 25 genes that are either assumed or known to be involved in the biosynthesis of streptomycin from two strains, *Streptomyces griseus* and *Streptomyces glaucescens* [for reviews, see Piepersberg (1995) and Piepersberg and Distler (1997)]. The genes for the aminotransferases pos-

E. coli BL21 was cultured in liquid LB at various temperatures. The thiostrepton-inducible expression of cloned genes in S. lividans 1326 was carried out according to Takano et al. (1995) with the exception that 10 μg thiostrepton per ml of YEME liquid medium (Hopwood et al. 1985) was used and that the incubation time after induction was prolonged up to 36 h. Preparation, transformation, and regeneration of competent E. coli cells (Sambrook et al. 1989) or S. lividans 1326 protoplasts (Hopwood et al. 1985; Babcock and Kendrick 1988) were carried out according to published procedures.

General DNA manipulation techniques and DNA sequencing

Streptomyces genomic DNA was extracted according to method B as described elsewhere (Mehling et al. 1995). DNA manipulation, cloning, and hybridization procedures were performed according to standard methods (Hopwood et al. 1985; Sambrook et al. 1989). The genomic DNA was immobilized on a Hybond N+ nylon membrane (Amersham Buchler, Braunschweig, Germany) and subjected to Southern hybridization. Hybridization was performed at 55–65°C overnight using nick-translated [³²P]dCTP (Amersham Buchler)-labeled DNA fragments as shown in Fig. 2. Stringency washes were done with 2–0.5 × SSC at 55–68°C depending on the genomic DNAs of the different strains. The DNA sequence was determined by the dideoxynucleotide chain-termination method (Sanger et al. 1977) using the AutoRead Sequencing Kit (Pharmacia, Freiburg, Germany) and an A.L.F. DNA sequencer (Pharmacia). The entire sequences of both strands were determined from double-stranded plasmid DNAs prepared by the alkaline lysis method (Sambrook et al. 1989). The sequences of the strO-stsABCDEFG genes are available from the EMBL data library under accession no. Y08763.

Amplification and expression of the aminotransferase genes

The putative aminotransferase genes strS, stsA, and stsC were amplified by PCR using genomic DNA as template, and the primers are listed in Table 2. Primers JA 6, JA 14, and JA 16 were designed for the introduction of an NdeI site in place of the natural start codon and for the ability to create start codon fusions of these genes into the promoter/ribosome-binding-site cassettes of expression vectors. Amplifications were performed in a Biometra Personal Cycler (Göttingen, Germany) using Vent DNA polymerase (New England Biolabs, Schwalbach, Germany) or Taq DNA polymerase (Gibco, Eggenstein, Germany). Each assay (100 μl vol.) contained 200 ng chromosomal DNA, 20 pmol of each appropriate primer, 0.2 mM dNTPs (Boehringer, Mannheim, Germany), incubation buffer, 10% dimethylsulfoxide, and 1.5 mM MgCl₂ (for Taq DNA polymerase only). The following conditions for the assays were used: 98°C for 5 min, then 30 cycles [95°C for 1 min, 54°C for 45 s (57°C for stsA amplification), 72°C for 1 min], and 72°C for 5 min (ramping rate 1°C/s). The PCR products were cloned into pUC18 HincII, resulting in pJAW74, pJAW75, and pJAW76. From these, the inserts were cleaved with NdeI/BamHI and ligated into pET11a, pET16b, and pIJ6021 NdeI/BamHI, respectively; this resulted in the derivatives listed in Table 1, which were transformed either into E. coli BL21 or into S. lividans 1326. The isopropylthiogalactoside(IPTG)-inducible expression of genes cloned in E. coli BL21 (DE3) (pLysS/pET11a) or (pLysS/pET16b) was carried out according to the manufacturer’s instructions (Novagene, Heidelberg, Germany). Cells were harvested by centrifugation, washed twice in ice-cold Tris/HCl (25 mM, pH 7.5), and suspended in 2 ml of “crack buffer” [25 mM Tris/HCl (pH 7.5), 1 mM dithiothreitol, and 0.2 mM phenylmethylsulfonyl fluoride, with or without 50 mM pyridoxal phosphate] per g wet wt. of the cells.

Cell disruption and aminotransferase assay

Cells were disrupted by sonication in the above buffer system in the presence or absence of pyridoxal phosphate, and the resulting

Materials and methods

Bacteria, plasmids, and culture conditions

The bacterial strains, λ phages, and plasmids used in this study are listed in Table 1. Streptomyces lividans 1326 and the other actinomycete strains were grown in tryptic soy broth (TSB; Hopwood et al. 1985) as a liquid medium or on soy meal/mannitol agar (SMA; Distler et al. 1985) at 28°C. Escherichia coli strains were grown on Luria Bertani (LB) agar at 37°C. For overproduction of proteins,

| Strain or plasmid                                                                 | Properties or production of                                                                                          | Source or reference                     |
|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|----------------------------------------|
| Bacterial strains                                                              |                                                                                                                  |                                        |
| Escherichia coli BL21 (DE3) (pLysS)                                              | ompT, hsdSB(rB–mB–), λ prophage with T7-pol gene, pLysS, cat                                                    | Studier et al. (1990)                  |
| Escherichia coli DH5α                                                          | Δ80dlacZΔM15, endA1, recA1, gyrA96, relA1, hsdR17, (rk–mk–), supE44, thi-1, λ, Δ(lacZYA-argF) U169                | Hanahan (1983)                        |
| Streptomyces lividans 66 strain 1326                                            | spc-1                                                                                                           | Hopwood et al. (1985)                 |
| Streptomyces lividans 66 strain TK23                                            | Streptomycin                                                                                                    | Hopwood et al. (1985)                 |
| Streptomyces griseus DSM 40236                                                  | Streptomycin                                                                                                    | DSM                                   |
| Streptomyces griseus N2-3-11                                                    | Streptomycin                                                                                                    | Kaken Chemical Company, Tokyo         |
| Streptomyces griseus M66                                                       | Streptomycin-negative mutant of N2-3-11                                                                         | Distler et al. (1985)                 |
| Streptomyces griseus M73                                                       | Streptomycin-negative mutant of N2-3-11                                                                         | Distler et al. (1985)                 |
| Streptomyces griseus M181                                                      | Streptomycin-negative mutant of N2-3-11                                                                         | Distler et al. (1985)                 |
| Streptomyces bikiniensis NRRL1049b                                              | Streptomycin                                                                                                    | NRRL                                  |
| Streptomyces galbus DSM 40480                                                   | Streptomycin                                                                                                    | DSM                                   |
| Streptomyces ornatus DSM 40307                                                  | Streptomycin                                                                                                    | DSM                                   |
| Streptoverticillium mashuense DSM 40221                                         | Streptomycin                                                                                                    | DSM                                   |
| Streptoverticillium griseocarnus DSM 40004                                      | 5′-Hydroxy-streptomycin                                                                                        | DSM                                   |
| Amycolatopsis orientalis ssp. lurida DSM 43187                                  | 5′-Hydroxy-streptomycin                                                                                        | DSM                                   |
| Streptomyces subrutilis DSM 40445                                               | 5′-Hydroxy-streptomycin                                                                                        | DSM                                   |
| Streptomyces glaucescens GLA.0ETH 22794                                         | 5′-Hydroxy-streptomycin                                                                                        | NRRL                                  |
| Streptomyces hygroscopicus ssp. glebosus DSM 40823                              | Bluensomycin                                                                                                   | DSM                                   |
| Streptomyces fradiae ATCC 10745                                                 | Neomycin                                                                                                       | A. Böck, Munich                       |
| Streptomyces rimosus ssp. paromomycinus NRRL 2455                               | Paromomycin                                                                                                    | A. Jimenez, Madrid                    |
| Streptoalloteichus hindustanus NRRLB-11280                                      | Tobramycin                                                                                                     | NRRL                                  |
| Streptomyces atrofaciens DSM 40475                                              | Hygromycin B                                                                                                   | DSM                                   |
| Streptomyces kanamyceticus DSM 40500                                            | Kanamycin                                                                                                      | DSM                                   |
| Micromonospora sioyaensis NRRLB-5408                                            | Sisomicin                                                                                                      | NRRL                                  |
| Streptomyces ribosidificus ATCC 21294                                           | Ribostamycin                                                                                                   | ATCC                                  |
| Plasmids and phages                                                            |                                                                                                                  |                                        |
| λ-EMBL3N3                                                                      |                                                                                                                  | Mansouri and Piepersberg (1991)       |
| pKMD66                                                                        | 7.4-kb BamHI/XhoI from N3inpUC18                                                                                | This work                             |
| pUC18                                                                         | bla, lacZ-α                                                                                                    | Yanisch-Perron et al. (1985)         |
| pET11                                                                         | bla, T7-lac promoter                                                                                           | Studier et al. (1990)                 |
| pET16b                                                                        | bla, T7-lac promoter, His-tag                                                                                   | Novagene, Heidelberg                  |
| pIJ4123                                                                       | kan, tsr, redD, His-tag                                                                                        | Takano et al. (1995)                  |
| pIJ6021                                                                       | kan, tsr                                                                                                       | Takano et al. (1995)                  |
| pJAW74                                                                        | 1.2-kb PCR fragment (contains strS) in pUC18HincII                                                             | This work                             |
| pJAW74-2                                                                      | strS, tsr, 1.2-kb NdeI/BamHI fragment of pJAW74inpIJ6021                                                       | This work                             |
| pJAW74-11                                                                     | 1.2-kb NdeI/BamHI fragment of pJAW74inpET11a                                                                   | This work                             |
| pJAW75                                                                        | 1.2-kb PCR fragment (contains strA) in pUC18 HincII                                                             | This work                             |
| pJAW75-2                                                                      | stsA, tsr, 1.2-kb NdeI/BamHI fragment of pJAW75 inpIJ6021                                                      | This work                             |
| pJAW75-11                                                                     | 1.2-kb NdeI/BamHI fragment of pJAW75inpET11a                                                                   | This work                             |
| pJAW76                                                                        | 1.3-kb PCR fragment (contains stsC) in pUC18HincII                                                              | This work                             |
| pJAW76-2                                                                      | stsC, tsr, 1.3-kb NdeI/BamHI fragment of pJAW76 inpIJ6021                                                      | This work                             |
| pJAW76-6                                                                      | stsC/His-tag, tsr, 1.3-kb NdeI/EcoRI fragment of pJAW76inpIJ4123                                                | This work                             |
| pJAW76-11                                                                     | 1.3-kb NdeI/BamHI fragment of pJAW76inpET11a                                                                   | This work                             |
| pJAW76-16                                                                     | 1.3-kb NdeI/BamHI fragment of pJAW76inpET16b                                                                   | This work                             |

2. The radioactive assay described by Walker (1975; test system B) was more sensitive for assaying L-glutamine:scyllo-inosose amino-transferase: \[^{14}C\]-labeled scyllo-inosamine was used as a substrate for the reverse reaction of the physiological transamination in an exchange reaction between labeled scyllo-inosamine and nonlabeled scyllo-inosose. In this assay, Tris was used instead of phosphate buffer.

3. As a further assay for activity, the forward direction was measured radioactively (test system C). The \[^{14}C\]-labeled scyllo-inosose deamination product was isolated from the reaction mixture of cold scyllo-inosose with \[^{14}C\]-labeled scyllo-inosamine (see above) by cation-exchange chromatography on a Dowex 50W-X8 column (1.2-ml bed volume) with water; then it was lyophilized and used for the physiological forward reaction. The complete enzyme assay contained: 15 μl 10 mM \[^{14}C\]-scyllo-inosose (approximately 1,000 cpm); 15 μl 100 mM L-glutamine, 5 μl incubation buffer [150 mM Tris/HCl (pH 7.5) and 1.5 mM pyridoxal phosphate], and 50 μl cell-free extract (2 mg protein per ml). After incubation at 33°C for various times (0.5–2 h), the \[^{14}C\]-labeled substrate (scyllo-inosose) and product (scyllo-inosamine) were separated by cation-exchange chromatography (see above). Scyllo-inosamine was eluted with 0.5 M HCl and was counted in a liquid scintillation counter.

Purification of StsC

All enzyme purification steps were carried out with an FPLC set (Pharmacia) at 6°C. Cell-free extract (65 ml) of a 600-ml thiostrepton-induced culture of S. lividans pJAW76-2 was applied to a DE-52 cellulose (Whatman, Maidstone, England) column (bed size 5 × 5.5 cm). After the column was washed with buffer A [25 mM Tris/HCl (pH 7.5), 1 mM dithiothreitol, and 0.1 mM phenylmethylsulfonyl fluoride], StsC was eluted with 900 ml buffer A containing NaCl in a linear gradient from 0 to 1 M at a flow rate of 2 ml/min. The fractions with the highest levels of activity were pooled (64 ml total), concentrated with an ultrafiltration cell (Amicon, Lexington, Mass., USA), and desalted by adding an equal volume of buffer A followed by a further ultrafiltration step. To the resulting 18 ml of protein solution, (NH₄)₂SO₄ and EDTA were added to final concentrations of 1 M and 0.25 mM, respectively, in a final volume of 24 ml, and the mixture was loaded onto a phenyl-Sepharose CL-4B (Sigma, Deisenhofen, Germany) column (1.6 × 9 cm). After the column was washed with buffer A containing 1 M (NH₄)₂SO₄, StsC was eluted with 150 ml of buffer A in a linear gradient of (NH₄)₂SO₄ from 1 to 0 M at a flow rate of 2 ml/min. The fractions with the highest levels of activity were pooled (37 ml) and loaded onto a Bio-Gel HT hydroxylapatite (BioRad, Munich, Germany) column (2.5 × 6.5 cm). StsC was eluted with 400 ml of a linear gradient from 10 to 100 mM sodium phosphate (pH 7.5) in buffer A at a flow rate of 2 ml/min. The fractions containing StsC were pooled (95 ml) and concentrated by ultrafiltration, desalted with buffer A containing 50 μM pyridoxal phosphate, and concentrated again to a final volume of 1.5 ml. This solution was applied to a Mono Q HR5/5 column (Pharmacia; 0.5 × 5 cm). After washing with buffer A, StsC was eluted with 30 ml of a linear gradient from 0 to 100 mM NaCl in buffer A at a flow rate of 1 ml/min. Two fractions (2 ml total) containing the highest activity of L-glutamine:scyllo-inosose aminotransferase were pooled and

Table 2 Primers used in this study

| Primer | Sequence | PCR amplification of |
|--------|----------|----------------------|
| JA 6   | CGACATATGAGCTCTTTCCAGGAATTGCC | strS |
| JA 7   | TCAGCTCGCGACGGC | strS |
| JA 14  | TGGAGGAACCCATATGCCGGGACCGGG | stsA |
| JA 15  | CTCTCAGGACACC GCCCTCG | stsA |
| JA 16  | CGAGGGAACATATGGATAGTTCACTGGCAA | stsC |
| JA 18  | CACCTCAGCGGGGAAACCT | stsC |

StsA MPGPGYRFFGDEERANVMAVLDRWAE--SR--EAFDDP-GEGEWVRR-FEQ----------------AA-ADRFGAAHCLGVNSGTSALVAALVGLGIGPGDEVIVPGYMFAVASIAAVL
StsC MDSSLA-ISGGPRLSNREWP--RWPQPGDRALKSLEDVLTSGRWTISCAYQGRDSYERQFASAFADYCGSAMCVPISTGTASLAIALEACGVGAGDEVIVPGLSWVASASAVL
StrS M-SSFQEL---P---------------RWPQLTDDDIEAAVALRSNRLV----GQGN-STVEEFEAALAAGQGVEHAVAVSTGTAAVHLALHALDVGPGDEVIVPTHTFIGSASPVT
cons M ss g p n RWpq dr ea d l sgrwv Qg s f aA Ad G ahcv vstGtaal AL a gvGpGDEVIVPg vaSasaVl

StsA HCGADVVLAEVD-DSLTLDPADVRARITPRTRAVMPVHMLGAPADMTALRAVADEHGLHLLLEDCAQSAGGSYRGRPLGTLGSAGTFSLNHYKMITSL-QGGFVLMDDPLVFQR
StsC GINAVPVLVDVDPAITYCLDPAATEAITERTRAITVVHAYSAAVADLDALLDIARRHGLPLIEDCAHAHGAGFRGRPVGHAAGVFSMQGSKLLTC-GEGGALVTDDADVALR
StrS YLGARPVFADVTPDTHCLDPDSVKSLIGERTKAIVVVHINGIAADMAALTAVAAEAGVPVIEDAAQALGTEIGGRPIGGFGDLACVSLFEQKVITSGGEGGAVLTDPVYAER
cons gA pVladVdpdt cLDPa v a IteRTrAi vVH ga ADm AL avA ehGlpliEDcAqa G rGRP G G ag fSl K iTs geGGavltDdp va R *

StsA AYSFHDQGWFPYRQDRGEDPPLLGMNLGLGEL-NAAVALAQLGKDLLILDRIRGVK-HRLVAA-------IGELPGVRRRTLHDAEGECGTVAVY------VFEDAAHALDVAR
StsC AEHLRADGRVVRREPVGVGEMELE-ETGRMMGSNACLSEFHAAVLLDQLELLDGQNARRTRAADHLTDRSEL-GM----TA-QATAP-GTTARAYRYLVRLPDEVLAVAPVE
StrS VRRLRSHGEGPVSGSPGMIWA--H-EVG-----YNYRLTSVQAAVGLAQHKRLGDLEARRRNAA-YLSERLAGVEGL---EL-PVEPP-GTT-HAYWKYAVRVVPGDGRRSAAD
cons a lr G r G g l e G Na l qaavll ql l g arR AA l rl el G l ae p GT a aY y vr d a a

StsA RLGTRVLLDSPT-HYYGGLPALAAAFGRGDRSTVPFRAPGG-RPSASFEPGTLPRTDDVLGRSIALATGVSDDYLGPGFGVHADSSAAEIDRAADAFRRAVKGEAVS 392 aa
StsC RFAHALTAELGFAVTQTHRPLNDNPLNRPSRRRFATDARYLERVDPERSFDLPAAKRA-HESVVSFSHEVLLAPLDAIDDIAFRFKVLDN-VREVSR 425 aa
StrS IAAHLRSRGVPVLLRYPY-PLHKQPAFAE-YH--------G----------SLPVAERLSQELLALLPSHPGLVEGH-LDHAVEEVRKAVA---S 377 aa
cons r ah p y Pl p s f g rv lp S a s l ld av evrrav s


Fig. 3 Alignment of the three putative aminotransferase proteins StsA, StsC, and StrS. A consensus sequence (cons) is given below with residues conserved in all three sequences given in capital letters. The single conserved lysine residue is marked by an asterisk.

Results

Cloning and analysis of additional operons in the str/sts gene cluster of *S.* griseus N2-3-11

The 7.4-kb *Bam*HI/*Xho*I fragment from the genomic DNA of *S.* griseus N2-3-11 cloned in λ phage EMBL3 N3 (Mansouri and Piepersberg 1991) was subcloned (pKMD66) and analyzed by DNA sequencing (cf. Fig. 2). This segment covered the stretch of genomic DNA upstream of the *strN* gene of *S.* griseus (Pissowatzki et al. 1991) and contained eight additional reading frames called *strO* (partial), *stsA*, *stsB*, *stsC*, *stsD*, *stsE*, *stsF*, and *stsG*.

When compared to each other, the putative proteins encoded by the genes *stsA* (410 amino acids, 43.5 kDa per monomer) and *stsC* (424 amino acids, 45.5 kDa) showed considerable sequence similarity (36.4% identity in a 302-amino-acid overlap). A similar degree of sequence relatedness was found to the protein encoded by a third gene, *strS* (378 amino acids, 39.8 kDa; StsA and StrS proteins show 34.1% identity in a 378-amino-acid overlap, and StsC and StrS proteins show 39.5% identity in a 306-amino-acid overlap), which had already been identified earlier downstream of the *strK* gene of *S.* griseus (cf. Fig. 2) (EMBL accession no. Y00459; Mansouri and Piepersberg 1991). An alignment of the three protein sequences (Fig. 3) revealed conserved sequence motifs that distantly resemble those in the family of aspartate and aromatic amino acid aminotransferases with a single conserved lysine residue in the central part of the polypeptide chains. Therefore, the three genes were taken as candidate genes for the aminotransferases postulated to be involved in the biosynthesis of streptomycins (Walker 1975; Piepersberg 1995). However, the most highly conserved peptide sequence motif in this new class (represented by the three proteins StrS, StsA, and StsC), G-D-E-x<sub>77</sub>-E-D-x<sub>10</sub>-G-x<sub>3</sub>-G-x<sub>8</sub>-S-x<sub>4</sub>-K-x<sub>5-6</sub>-(E,Q)-G-G (cf. Fig. 3), does not correlate with the sequences identified in the five aminotransferase classes listed in the PROSITE data base.

Data bank searches revealed that the other genes located on the insert of plasmid pKMD66 putatively encode proteins related to the following enzyme families: the

stored at 6°C. The aminotransferase StsC thus obtained was electrophoretically pure and stable for several weeks. A 0.5-ml sample of purified StsC (approximately 50 μg protein) was applied onto a Superose 12 HR10/30 column (Pharmacia; 1.0 × 30 cm) previously equilibrated with buffer A containing 0.1 M NaCl and eluted with the same buffer at a flow rate of 0.5 ml/min. The differential elution of the standard proteins β-amylase (potato), yeast alcohol dehydrogenase, and bovine chymotrypsinogen A (all from Sigma) was determined and compared to that of StsC. The optical properties of the purified enzyme were measured by monitoring the absorption spectrum in the range of 300–700 nm of samples containing purified StsC protein in the above-mentioned dialysis buffer; addition of amino acids L-glutamine and L-alanine occurred stepwise in the range of 1.5–9 mM.

Protein gel electrophoresis

StrS, StsA, and StsC production and the results of the purification steps were analyzed by SDS-PAGE (Laemmli 1970). Protein-containing extracts were lysed by boiling in sample buffer (Laemmli 1970) and separated on SDS-polyacrylamide gels containing 12.5% polyacrylamide.

Preparation of substrates

The [¹⁴C]-labeled scyllo-inosamine, kindly provided by J. Distler, was prepared from mutant strain *S.* griseus M126 (a derivative of strain N2-3–11 blocked in a step of streptidine biosynthesis after the first aminotransferase reaction) after feeding 2.5 μCi [¹⁴C]-labeled myo-inositol (Amersham-Buchler, Braunschweig, Germany) with 2–5% yield according to a protocol published earlier (Distler et al. 1985). Nonlabeled scyllo-inosamine was prepared chemically and kindly provided by M. Roggel and H.-J. Altenbach (Bergische Universität GH, Wuppertal, Germany).

Computer analysis of protein and DNA sequences

The software packages or algorithms for the analysis of DNA and protein sequences were DNA-Strider 1.2 (Marck 1988), FASTA (Pearson and Lipman 1988), and PAUP 3.1 (from D.L. Swofford, Center of Biodiversity, Champaign, Ill., USA).

gene product (259 amino acids) of strO is related to myo-inositolmonophosphate phosphatases, e.g., the E. coli protein SuhB (Gene Bank accession no. M34828) or the human myo-inositol-1(or 4)-monophosphatase protein (EMBL accession no. X66922). The gene stsB encodes a protein (489 amino acids) that is related to the large family of dinucleotide-dependent dehydrogenases. It contains a peptide motif, V(#)₂GgGxaGx₂Ax₂Lxex₃exV#L#EAg (# represent hydrophobic residues, capital letters indicate invariant residues), resembling the dinucleotide-(FAD-)binding folds at the N-terminus of several oxidoreductases. The StsE protein (312 amino acids) deduced from the DNA sequence shows distant sequence similarity to antibiotic kinases (e.g., the ribostamycin 3′-phosphotransferase from Streptomyces ribosidificus, EMBL accession no. M22126) and contains the characteristic peptide signatures common to the large family of antibiotic and protein kinases that phosphorylate hydroxyl residues (Piepersberg et al. 1988; Heinzel et al. 1988). Especially the characteristic amino acid residues in the catalytic domain of this class of phototransferases, hgDₓ₄N(#)₃x₇(#)₂Dfe (# indicates hydrophobic residues, capital letters indicate invariant residues) are conserved. StsG (253 amino acids) shows significant similarity to a wide variety of methyltransferases and exhibits all three conserved peptide motifs found in members of this family of enzymes, which use S-adenosylmethionine as a coenzyme (Kagan and Clarke 1994). The gene products of the two genes stsD (214 amino acids) and stsF (229 amino acids) did not show any significant similarity to known protein families.

Overproduction of StrS, StsA, and StsC and identification of the L-glutamine:scyllo-inosose aminotransferase

The genes for the three putative aminotransferases StrS, StsA, and StsC were cloned by PCR into expression vectors of E. coli (pET11a and pET16b) and of S. lividans (pIJ6021) and were transformed into respective host strains according to the protocols given in Materials and methods (cf. Table 1). The expression in E. coli BL21 derivatives resulted in the overproduction of large amounts of the StrS and StsC proteins, which, however, formed insoluble inclusion bodies under all conditions tested; only in the case of StsA did overproduction occur at a lower level and in a soluble form (not shown). The production of a lesser amount of soluble protein was also visible as extra bands in SDS gels in S. lividans 1326 containing plasmids pJAW74-2 (StrS), pJAW75-2 (StsA), and pJAW76-2 or pJAW76-6 (StsC) (Fig. 4A). Crude, ribosome-free extracts from the E. coli and S. lividans derivatives were tested for L-glutamine:scyllo-inosose aminotransferase activity by use of the scyllo-inosamine/scyllo-inosose exchange reaction (test system B) or the standard TLC assay (test system A). The results are listed in Table 3. The reverse reaction tested by a simple amino group exchange reaction (cf. Fig. 2) turned out to give highest activity values but also exhibited a strong background activity in the control extracts from E. coli and S. lividans

**Fig. 4** SDS-PAGE separations of the overproduced putative aminotransferases StrS, StsA, and StsC (A) and of the fractions during purification of StsC (B). A The lanes were loaded with 5 μg of protein each of cell-free extracts from Streptomyces lividans 1326 containing the following plasmids: pIJ6021 (lanes 1 and 2), pJAW74-2 (lanes 4 and 5), pJAW75-2 (lanes 6–9), pJAW76-2 (lanes 10 and 11), and pJAW76-6 (lanes 12 and 13). Protein extracts were either from uninduced cells (lanes 1, 4, 6, 8, 10, and 12) or induced cells (lanes 2, 5, 7, 9, 11, and 13). The successful overproduction of StrS (lane 5), StsA (lanes 7 and 9), and StsC (lanes 11 and 13) is indicated by arrows. The molecular masses (kDa) of the marker proteins (lane 3, 10 μg) were: albumin, bovine (66); ovalbumin, chicken (45); glyceraldehyde-3-phosphate dehydrogenase, rabbit (36); carbonic anhydrase, bovine (29); trypsinogen, bovine (24); and trypsin inhibitor, soybean (20). B Purification of cloned StsC from S. lividans 1326 pJAW76-2 after induction with thiostrepton. The lanes contained cell-free extracts from S. lividans 1326 pIJ6021 (lane 2, 32 μg protein), cell-free extract of S. lividans 1326 pJAW76-2 (lane 3, 32 μg protein), and the pooled active fractions from the separations on DE-52 (lane 4, 23 μg protein), Phenyl Sepharose (lane 5, 15 μg), hydroxylapatite (lane 6, 12 μg), and Mono Q (lane 7, 1.5 μg), respectively. Lanes 1 and 8 contained protein marker (10 μg)

strains. Only the extracts from cells containing the StsC protein exhibited a positive reaction in these assay systems. When the StsC enzyme was tested from crude extracts of S. lividans 1326 (pJAW76-6) produced in the absence of pyridoxal phosphate from the extraction and dialysis buffers, a remarkable dependence of activity on the addition of pyridoxal phosphate to the test reaction (test system A) was observed: a very low residual activity — which could be restored to full activity on addition of pyridoxal phosphate (1.2 mM) — was detected in the absence of the coenzyme. In contrast, when pyridoxal phosphate had already been added to the extraction buffer (50 mM), its further addition to the test reaction was not essential for full activity. This indicated a lack of sufficient pyridoxal phosphate in the coenzyme pool of the StsC-overproducing strain of S. lividans 1326, dependence of StsC on pyridoxal phosphate, and strong binding of the coenzyme to the enzyme protein in absence of an amino donor substrate.

The StsC protein could also be formed as a protein N-terminally modified by a His-tag extension in S. lividans 1326 (pJAW76-6) and was also fully active in this form (cf. Table 3). When the ribosome-free extract of E. coli

Table 3 Activity of L-glutamine:scyllo-inosose amino-transferase in various strains

| Enzyme source$^a$ | Gene product | Non-labeled TLC assay$^b$ | Labeled reaction [pkat/mg]$^b$ |
|--------------------|--------------|--------------------------|--------------------------------|
|                    |              | Reverse                  | Forward                       |
| Streptomyces griseus N 2-3-11 | StrS, StsA, StsC | – | 58.3 | 0.0 |
| Escherichia coli BL 21 (pET16b) | – | – | 8.7 | n.d. |
| Escherichia coli BL 21 (pJAW 76-11) | StsC | – | 106.7 | 0.0 |
| Escherichia coli BL 21 (pJAW 76-16) | His•tag-StsC | – | 5.3 | n.d. |
| Streptomyces lividans 1326 (pIJ 6021) | – | – | 3.2 | 0.0 |
| Streptomyces lividans 1326 (pJAW 74-2) | StrS | – | 4.0 | n.d. |
| Streptomyces lividans 1326 (pJAW 75-2) | StsA | – | 8.7 | n.d. |
| Streptomyces lividans 1326 (pJAW 76-2) | StsC | + | 850.0 | 177.0 |
| Streptomyces lividans 1326 (pJAW 76-6) | His•tag-StsC | + | 728.0 | 153.3 |

$a$: Crude cell extracts if not otherwise stated (cf. Materials and methods)

$b$: Test systems are described in Materials and methods; n.d., not determined

Table 4 Hybridization of stsA-, stsC-, and strS-specific probes against DNA from different aminoglycoside antibiotic-producing actinomycetes$^a$

| Strain | Produced aminoglycoside antibiotic | Size of hybridizing DNA fragments (kb) |
|--------|-------------------------------------|---------------------------------------|
|        |                                     | stsA                                  | stsC                                  | strS                                 |
| Streptomyces griseus DSM 40236 | Streptomycin | 14.5B                                 | 14.5B                                 | 20.0B                                |
|        |                                   | 8.3B/N                                | 8.3B/N                                | 3.5B/N                               |
| Streptomyces griseus N 2-3-11 | Streptomycin | 14.5B                                 | 14.5B                                 | 20.0B                                |
|        |                                   | 8.3B/N                                | 8.3B/N                                | 3.5B/N                               |
| Streptomyces bikiniensis NRRL 1049b | Streptomycin | 5.0B/N                               | 7.0B/N                                | 2.3B/N                               |
| Streptomyces galbus DSM 40480 | Streptomycin | 10.0B/N                               | 10.0B/N                               | 10.0B/N                              |
| Streptomyces ornatus DSM 40307 | Streptomycin | 9.4B/N                                | 9.4B/N                                | 3.6B/N                               |
| Streptoverticillium mashuense DSM 40221 | Streptomycin | 2.5B/N                              | 4.2B/N                                | 1.8B/N                               |
| Streptoverticillium griseocarnus DSM 40004 | Hydroxystreptomycin | 0.8B/N                              | 2.1B/N                                | 2.7B/N                               |
| Amycolatopsis orientalis ssp. lurida DSM 43187 | Hydroxystreptomycin | 3.2B/N                              | 4.0B/N                                | 0.5B/N                               |
| Streptomyces subrutilus DSM 40445 | Hydroxystreptomycin | 1.9B/N                              | 0.6B/N                                | 5.0B/N                               |
| Streptomyces glaucescens GLA.0ETH 22794 | Hydroxystreptomycin | 4.8B                                | 4.8B                                  | 2.1B/N                               |
|        |                                   | 2.4B/N                                | 1.5B/N                                |                                      |
| Streptomyces hygroscopicus ssp. glebosus DSM 40823 | Bluensomycin | –                                    | 1.8B/N                                | 0.9B/N                               |
| Streptomyces fradiae ATCC 10745 | Neomycin | >20.0B                                | >20.0B                                | >20.0B                               |
| Streptomyces rimosus ssp. paromomycinus NRRL 2455 | Paromomycin | >20.0B                                | >20.0B                                | >20.0B                               |
| Streptoalloteichus hindustanus NRRL B-11280 | Tobramycin | >20.0B                                | >20.0B                                | >20.0B                               |
| Streptomyces atrofaciens DSM 40475 | Hygromycin B | 9.4B                                  | 9.4B                                  | 3.0B                                 |
|        |                                   | 4.8B                                  | 4.8B                                  |                                      |
|        |                                   | 3.0B                                  | 3.5B                                  |                                      |
| Streptomyces kanamyceticus DSM 40500 | Kanamycin | 15.0B                                 | 15.0B                                 | 15.0B                                |
| Micromonospora sioyaensis NRRL B-5408 | Sisomicin | –                                    | –                                     | 4.2B                                 |
| S. ribosidificus ATCC 21294 | Ribostamycin | –                                    | –                                     | 6.0B                                 |
|        |                                   |                                       |                                       | 3.0B                                 |

$a$: Abbreviations: B, BamHI; N, NcoI; –, no hybridizing signal

BL 21 containing the plasmid pJAW 76-11 was examined for aminotransferase activity using the radioactive enzyme assay (test B), it also showed a weak positive reaction in comparison to the controls, indicating that a minor amount of StsC protein was produced in a soluble and active form. Enzyme activities could not be measured in crude extracts of S. griseus N 2-3-11 and E. coli BL 21 (pJAW 76-11) by use of both test systems for the forward reaction (test systems A and C), probably because of enzyme concentrations that were too low. In the exchange

reaction measured in test system B, the activity of StsC was at least 12-fold higher in *S. lividans* 1326 (pJAW76-2) than in streptomycin-producing *S. griseus* N2-3-11 (cf. Table 3). By use of the scyllo-inosamine/scyllo-inosose exchange reaction (test system B), no activity could be measured in the extracts from *S. griseus* mutants M66 and M73, whereas a very low activity was observed in mutant strain M181 (cf. Table 1).

### Purification of StsC

The StsC protein was purified from *S. lividans* 1326 (pJAW76-2) cells induced by thiostrepton. StsC was almost pure after the chromatography step on Mono Q (Fig. 4B). The apparent molecular mass of 95–110 kDa, measured by elution from the gel filtration on Superose 12, suggested that the native protein is a dimer since the molecular mass as determined on SDS-polyacrylamide gels (cf. Fig. 4B) and as calculated from the deduced protein sequence is 45.5 kDa. The enzyme thus obtained did not accept L-alanine, which is reported to be the cosubstrate in the second transamination reaction involved in streptidine biosynthesis (Walker 1975), as an alternative amino donor substrate. The purified enzyme protein showed an absorption peak around 415 nm, which is similar to that of the equivalent enzyme from *Micromonospora purpurea* and which is typically found with pyridoxal-phosphate-containing enzymes (Lucher et al. 1989). Furthermore, this absorption could be significantly quenched by the addition of L-glutamine (1.5–3.0 mM), but not by the addition of L-alanine. The lack of sufficiently labeled substrate prevented more extensive and reproducible measurement of StsC activity [e.g., the determination of kinetic constants for substrate (scyllo-inosose) and cosubstrate (L-glutamine)].

### Distribution of *stsA*- and *stsC*-related genes in other aminocyclitol producers

Hybridization assays using the fragments indicated in Fig. 1 as reading-frame-specific probes for the *strS*, *stsA*, and *stsC* genes were performed on *BamHI-* or *BamHI/NcoI*-cleaved genomic DNAs from a variety of actinomycete genera that produce aminocyclitol aminoglycoside antibiotics (Table 4). The DNA of most tested aminoglycoside producers hybridized to the *stsC*, *stsA*, and *strS* probes. Exceptions were the genomic DNAs of the bluensomycin-producing strain *S. hygroscopicus* ssp. *glebosus*, which did not hybridize to the *stsA* probe, and of the strains *S. ribosidificus* and *Micromonospora sioyaensis*, which hybridized only to *strS* (cf. Table 4). No specific hybridization signals were detected in the DNA of *S. lividans* 66, which does not produce any known aminoglycosides. The two DNA fragments that hybridized to the *stsA* and *stsC* probes in the 5′-hydroxystreptomycin producer *S. glaucescens* GLA.0 were localized on two adjacent segments cloned on cosmid p25C9 (Mayer et al. 1988). We had already analyzed the DNA sequences directly neighboring this area (EMBL accession nos. X78975 and Y08764) and covering the *stsNB2M* and *strO* operons in this strain. These genes were in the same order as in *S. griseus* (cf. Fig. 2).

---

### Discussion

We showed here that the StsC protein, encoded by one of the three related putative aminotransferase genes (*strS*, *stsA*, and *stsC*) of the streptomycin gene cluster of *S. griseus*, is the L-glutamine:scyllo-inosose aminotransferase. The enzyme activity of StsC was clearly identified by the following criteria: (1) substrate specificities, (2) cosubstrate specificities, (3) the coenzyme requirement, and (4) absorption properties of the purified enzyme. On the grounds of its enzymatic activity, this enzyme has been described earlier by Walker and coworkers as occurring in all tested aminocyclitol-producing actinomycetes (Walker and Walker 1969; Lucher et al. 1989; Walker 1995). It has also been demonstrated recently that these three genes are only transiently expressed in *S. griseus* N2-3-11 before the onset of streptomycin production during a second vegetative growth phase called the “execution phase” (Neumann et al. 1996). This productive phase occurs after an initial “decision phase” and a separating lag phase that seem to be characteristic for most streptomycetes growing in liquid culture.

We had earlier described a class of mutants of *S. griseus* N2-3-11 (e.g., strains M66, M73, and M181) as assumed to be blocked in the first transamination step because they produced streptomycin only when supplemented with scyllo-inosamine, and not with scyllo-inosose or myo-inositol (Distler et al. 1985). We also showed here that these mutants either have largely decreased L-glutamine:scyllo-inosose aminotransferase activity (strain M181) or fail to exhibit this activity at all (strains M66 and M73), which provides evidence that at least the latter two mutant strains are blocked in the formation of a functional StsC enzyme. Since all attempts failed either to induce gene-specific knock-out mutants in *S. griseus* or to complement mutants M66 and M73 by cloned DNA (pJAW76-2 and pJAW76-6; data not shown), probably due to high restriction barriers or to the fact that the *tipAp* promoter was not active in *S. griseus*, we had to use the approach of identifying and actively expressing the candidate aminotransferase genes in a host organism free of this activity as described herein. The host strains used, *E. coli* and *S. lividans*, were free of this activity. Also, total genomic DNAs of other producers of streptomycin-related aminoglycosides (e.g., *S. hygroscopicus* ssp. *glebosus*, but not the host strain *S. lividans*) hybridized to various gene probes derived from the *str/sts* gene cluster of *S. griseus*, including the *strS-*, *stsA-*, and *stsC*-specific probes (cf. Table 4 and unpublished results). Because genes related to *stsA* and *stsC* also occur in actinomycete producers of other diaminocyclitol aminoglycosides such as neomycins, kanamycins, and hygromycin B, the two proteins StsC and StsA are likely to represent aminoglycoside-specific aminotransferases. Therefore, the *stsA* and *stsC* genes could be

A

| StsCSgr | HGLPLIEDCAAHGAG--FRG--RPVGAHGAAGV--FSMQGSKLLTC-GEGGALVTDDADVALRAEHLRADGRV-----------------V----------------RREEGDPLLGMNLGLGELNAAVALAQLGKLD |
| StsASgr | HGLHLLLEDCAQSAGGS--YRG--RPLGTLGSAGT--FSLNHYKMITSL-QGGFVLMDTPLVFQRAYSFHDQGWF-----------------PYRQDRGPV-GVGEMELEETGRMMGSNACLSEFHAAVLLDQLELLD |
| StrSSgr | AGVPVIEDAAQALGTE--IGG--RPIGGFGDLAC--VSLFEQKVITSGGEGGAVLTDPVYAERVRRLRSHGEG-----------------PVSGSPGMIW-AHE--VGYNYRLTSVQAAVGLAQHKRLG |
| LmbSSli | HGLTLLLEDAAH-TLAR---DGD-AVAGRAGDASA--FSFFATKPITTA-EGGMLCTDTARVADEARRWSLHGLS-----------------RGAVVNRYRPGHKYNMSD-LG-R-RPGPAQLAKAGR-L-HAR |
| DnrJSpe | HDLKVLEDCAQAHGAR--RHG--RLVGTQGHAAA--FSFYPTKVLGAYGDGGAVVTPDAEVDRLLRRLRYYGMG-----------------ERYY-VVDTPGHNSRLDEVQAEILRRKLRRILD |
| TylBSfr | HGLAVVEDAAQATARY---RG--RRIGSGHRTA---FSFYPGKNL GALGDGGA VVTSDPELADRIRLLRNYGAR-----------------EKYRHEER-GTNSRLDELQAAARVRKLPRNL |
| EryC1Se | HGLALVEDVAQAVGAR--HRG--HRVGAGSNAAA--FSFYPGKNL GALGDGGA VVTTDPALAERIRLLRNYGSK-----------------QKYVHEVR-GTNARLDELQAAVLRVKLRHLD |
| Prg1Sal | YGLRVLEDCSHAHGSR--YKG--KPVGTFGDAAV--FSLQANKAVYA-GEGGILVTDD-ALVQDRATLLL---GHY-----------------RDRSRDSVIDEELRSHWVTG---FGLKLRMSPFNAIVARHSLAAF |
| DegTBst | HGLVVIEDAAQAIGAK---YNG--KCVGELGTAAT--YSFFPTKNL GAYGDGGMI ITNDDDELAEKCRVIRVHGSK-----------------PKYYHH-V-LGYNNSRLDEMQAAILSVKFPHLD |
| SpsCBsu | HGLFVLEDAAHAVYTT--YKQ--RMIGSIGDATA--FSFYATKNLAT-EGGMLTTDDEELADKIRVLSLHGMS-----------------KAAWNRYSSNGSWYYEVESPGYKMNMFDLQAALGLHQLKRLD |
| SMAT2Bp | HGLKVIEDAAHALPAS--WQG--RRIGSLESDLTV-YSFYATKTIA T-EGGGMVVT RDPALAKRCVMRLHGID-----------------RDAFDRFTSKKPAWYEIVAPGFKYNMDTDAAAMGRVQLQRVQ |
| SMAT1Bp | HGLPVIEDAAQSFGAT--YKG--RKSCNLSTIGC--TSFFPSKPLGCYGDGGALFTNDDDELAQAMREIRVHGQS-----------------GRYYHARIGVGGRMDTLQCAVVLGKLERF-DWEIAQRIKIGARYQ |
| MosBRme | QGIAVIEDCAQAHGAL--YKG--QHVGLLSDIGC--FSMQKSKHL-SAGDGGFMVTRDPTLAQKMRDICNFGLPTP-----------------KANYRFDEVVRDGYAVFRECEQIGGMFLQPMSAALVMHQLEHLR |
| YifIEco | HNLFVVEDAAQGVMST--YKG--RALGTIGHIGC--FSFHETKNTAGGEGGATLINDKALIERAEIIREKGTN-----------------RSQFFRGQVDKYTWR-DIGSSYLMSDLQAYLWAQLEAAD |
| PerSVch | HGLYLIEDCAEAIGTA---VNG--KKVGTFGDVST--FSFFGNKTITS-EGGGMVVNSDIIIDKCLRLKNQGVV-----------------AGKRYWH-DLVAYNYRMTNLCAATIGVAQLERVD |
| FlaACcr | SGAVLVEDAPHALGSTAFDGVAHPVGDAYSSFASF SFHPVKTLAT-EGGMLTTNDPALAAKARILLRSHGMV-----------------RQPGGDPWWYEMP-ELGFNYRIPDVL CALGLSQLAKLD |
| RifDAme | TGVPLLQDAAAHHGAR--WQG--KRVGELDSIATE--SFQNGKLMTA-EGGAVVFPDGTEKEYETAFLRHSCGRP-----------------RDDRRYFH-KIAGSNMRLNEFSASVLR AQLARLD |
| AHBSAct | TGVPVWQDAAAHGAH---WRG--KRVGELGSIA T--FSFQNGKLMTA-EGGAVLFADQOQWEKAFVLHSCGRPKG-----------------DRGYFH-LTSGSNFRMNEFSAAVLRAQLGR LD |
| AHBS Sco | SGVPLIQDAAHAQGAT--WNG--RRLGELGSVAA--FSFQNGKLMTA-EGGAVLFFTAEMAEHAFLRHSCGRPRN-----------------DRGYFH-RTSGSNFRLNEFSASVLR AQLARLD |
| RfbHEco | RDIILLEDNCESMGAV--FQN--KQAGTFGVMGTF S-SFYSHHIATM--EGGCVVTTDDEELYHVLCLRAHG WTRN-LP----------KENMVTGTKSDDIFEESFK-VLPGYNVRPLEMSG AIGIEQLKKLP |
| SnoSSno | HDLFLIEDNCDAV GSL--YDG--KLTGTFGDMTTV--SFYPAHHLT M-EGGCVLTSNLALARIVESLRDWGRDCWCEPGENDRCLKRFKYQM GTLPAGYD-KYIF-SHVGYLNKATDIQAALGLTQLAKLD |
| AscCYps | YNLWLEDCCD AIGST--YDG--KMAGTFGDIGTV--SFYPAHHITM-EGGAVFTQSAELKSILESFRDWGRDCYCAPGCDNTCKKRFGQQQL GSLPFGYDHKYTY-SHLGYLNKITDMQAACGLAQLEPIE |
| RfbHSen | YNLWLIEDCCDALGTT--YEG--QMVGTFGDIGTV--SFYPAHHITM-EGGAVFTKSGELKKIIESFRDWGRDCYCAPGCDNTCGKRFGQQQL GSLPQGYDHKYTY-SHLGYLNKITDMQAACGLAQLERVE |

B


the histidine residue instead of the lysine residue and the four conserved cysteines in the dehydrase subfamily. B A phylogenetic tree (phylogram) of the conserved central domain of the SMAT super-family of enzymes was calculated by the PAUP program on the basis of those portions of all amino acid sequences that corresponded and could be aligned to the residues (79–315) of the StsC sequence; the four dehydrase sequences (see below) were used as an outgroup. The numbers denote the calculated lengths of branches; the numbers in parentheses give the percent score of nodes found in bootstrapping (500 times). The following sequences were used and are identified (in parentheses) by their SWISS-PROT (P) or NCBI-PID (g) accession numbers: (1) known or postulated keto-cyclitol or keto-sugar aminotransferases: StsC-Sgr, StsASgr, and StrSSgr (this paper); LmbSSli from Streptomyces lincolnensis (g 581695); DnrJSpe from Streptomyces peucetius (P 25048); TylBSfr from Streptomyces fradiae (g 473598); EryC1Se from Saccharopolyspora erythrea (P 14290); Prg1Sal from Streptomyces alboniger (g 1045229); SpsCBsu from Bacillus subtilis (P 39623); MosBRme from Rhizobium meliloti (g 310304); YifIEco from Escherichia coli (P 27833); PerSVch from Vibrio cholerae (g 280157); (2) putative aminotransferases of unknown function: DegTBst from Bacillus stearothermophilus (P 15263); SMAT1Bp from Bordetella pertussis (g 992973); SMAT2Bp from Bordetella pertussis (g 992976); FlaACcr from Caulobacter crescentus (g 903372); (3) 3-amino-5-hydroxybenzoic acid synthases: RifDAme from Amycolatopsis mediterranei (g 974607); AHBSAct from Actinosynnema pretiosum auranticum (g 974609); AHBS Sco from Streptomyces collinus (g 995684); (4) known or postulated keto-sugar dehydrases: RfbHEco from Escherichia coli (g 1155021); SnoSSno from Streptomyces nodosus (S. Verseck, W. Piepersberg, unpublished data); AscCYps from Yersinia pseudotuberculosis (g 609643); and RfbHSen from Salmonella enterica B (P 26398)


Fig. 5A, B A conserved superfamily of enzymes related to the StsC, StsA, and StrS proteins encoded by the streptomycin gene cluster. A Alignment of the conserved central domain of the secondary metabolic aminotransferases (SMATs), 3-amino-5-hydroxybenzoate synthases, and NDP-hexose dehydrases. The arrow above denotes the conserved lysine residue in the putatively pyridoxal-phosphate-dependent subfamily. The arrows below indicate

used to design pathway-specific gene probes such as the strD, strE, and strM genes for the dTDP-6-deoxyhexose pathways (Stockmann and Piepersberg 1992).

Given that all components are made de novo from D-glucose-6-phosphate, there are two further steps of possible amino transfer in the biosynthesis of streptomycin: (1) introduction of the second amino group into the diaminocyclitol streptidine, and (2) introduction of amino-N during the biosynthesis of the unusual aminohexose N-methyl-L-glucosamine (Walker 1975; Piepersberg 1995). Our current suggestion is that StsA might be the enzyme for the second transamination step during streptidine biosynthesis since its sequence is more closely related to that of StsC and its gene is obviously linked to the preceding

one, stsB (cf. Fig. 1), which could encode the second oxidase involved in keto group formation during streptidine biosynthesis (J. Ahlert and W. Piepersberg, unpublished results). This is also underlined by the fact that the stsA-and stsC-specific gene probes hybridized to distinct genomic DNA fragments in most of the producers of diaminocyclitol-containing aminoglycosides. Often both of these hybridizing genes seemed to be located on equal-sized fragments, indicating an adjacent location in a common gene cluster similar to that in S. griseus (cf. Table 4). An exception is the streptomycin-related bluensomycin molecule, which contains only a monoaminocyclitol (Piepersberg 1995). Consistent with this, the stsA probe did not show any hybridization in the DNA of a bluensomycin
producer, *S. hygroscopicus* ssp. *glebosus*. The failure to detect *stsC-* and *stsA*-specific signals in some of the strains tested [such as in the genomic DNA from strains *S. ribosidificus* and *M. sioyaensis*, which produce 2-deoxy-streptamine-containing aminoglycosides (Piepersberg and Distler 1997)] could indicate either that the sequence similarity between the homologous genes is too low or that another type of aminotransferase is used in those strains.

The three proteins StrS, StsA, and StsC are members of a protein family that at first was suggested to comprise regulatory proteins (Stutzman-Engwall et al. 1992). However, our data and the demonstration of 3-amino-5-hydroxy-benzoic acid synthase activity for another member, RifD from *Amycolatopsis mediterranei* (Kim et al. 1992; cf. Gene Bank accession no. U33061), clearly proves that all these proteins represent a new class of pyridoxal/pyridoxamine-phosphate-dependent aminotransferases or aromatizing aminocyclitol dehydratases, as suggested earlier (Retzlaff et al. 1993; Liu and Thorson 1994; Pascarella and Bossa 1994; Piepersberg 1994). The most highly conserved peptide sequence motif in this new class of aminotransferases is G-(D,V,T)-(E,V,Q)-x<sub>76–84</sub>-(E,Q,R)-D-x<sub>10–12</sub>-(G,D,Q)-x<sub>3–5</sub>-G-x<sub>8–10</sub>-S-x<sub>4</sub>-K-x<sub>4–5</sub>-(G,A,L,F)-(E,D,Q)-G-G (Fig. 5; cf. Fig. 3). In the StsC protein, the highly conserved amino acid residues D-173 (D-140 in StsA, D-157 in StrS) and K-202 (K-169 in StsA, K-186 in StrS) are likely to be involved in interacting with the N1 of the coenzyme pyridoxal phosphate and in forming the Schiff base to the coenzyme, respectively, by comparison to the class I aminotransferases (Pascarella and Bossa 1994). These seem to occur preferentially in highly variable and periferal (i.e., strain-specific rather than species-specific) pathways such as in the biosynthesis of secondary metabolites and extracellular polysaccharides or in the metabolism of rhizopines (cf. Fig. 5; Piepersberg 1994). Together with the sugar dehydrases represented by the AscC or RfbH proteins from *Yersinia* sp. and *Salmonella* sp. (Liu and Thorson 1994), respectively, these secondary metabolic aminotransferases (SMATs) form a protein superfamily. Structurally, the subgroups of the SMAT superfamily differ mainly in the two features outlined in Fig. 5 A, namely the presence or absence of a conserved lysine residue (replaced by a histidine in the dehydrases) and of a cysteine-rich insert (absent from the aminotransferases). For the latter feature, the RfbH-like *E. coli* protein represents an exception because it lacks the cysteine-rich peptide insertion; otherwise, it clearly belongs to the dehydrase group (cf. Fig. 5 A). Therefore, since the cysteines are likely to be involved in binding an iron–sulfur cluster (Liu and Thorson 1994) for electron transfer, this *E. coli* protein of unknown function could be a dehydrase that uses neither an oxidoreduction mechanism nor another electron donor and transfer system. Interestingly, the two different subgroups of proteins in this family also differ in their coenzyme requirement: pyridoxal phosphate in the aminotransferase group and pyridoxamine-phosphate in the dehydrase subfamily (Thorson et al. 1993; this work).

Thus, the evolution in the SMAT family of proteins seems to have created a remarkably broad set of enzymes

that recognize a large variety of substrates and cosubstrates and that use completely different enzyme mechanisms. This is also reflected in the deeply rooted phylogenetic relationships among these proteins; these relationships seem to be distributed nontaxonomically (cf. Fig. 5B; Table 4). For example, the occurrence of certain members of both dehydrases and aminotransferases in certain strains of gram-negative and gram-positive bacterial genera, but not in close relatives of the relevant species, underlines this suggestion (Piepersberg 1992; Reeves 1993; unpublished results). A phylogenetic tree of some of the members of the SMAT family of proteins (cf. Fig. 5B) reveals further surprising facts:

1. The StrS, StsA, and StsC proteins are more distant from each other than from other members of the family involved in other pathways. For example, the StsC protein is more closely related to the Prg1 gene product encoded by the biosynthetic gene cluster for puromycin, which putatively represents a ketopentose aminotransferase (Ter-cero et al. 1996).

2. StrS is more similar to the LmbS protein (putative ke-tooctose aminotransferase; Peschke et al. 1995; unpublished observations) from a lincomycin-producing streptomyce and, therefore, could also be a keto-sugar aminotransferase involved, for example, in the synthesis of the *N*-methyl-L-glucosamine moiety of the streptomycin-related aminoglycosides.

3. Although the presumed dTDP-3-keto-6-deoxy-D-hexose aminotransferases TylB (Merson-Davies and Cund-liffe 1994) and EryCl (Dhillon et al. 1989) should catalyze very similar reactions in the biosynthesis of almost identical macrolide sugars in the pathways for tylosin and erythromycin, respectively, they are more distant from each other than TylB is from the DnrJ protein (Stutzman-Engwall et al. 1992). DnrJ is now presumed to be involved in the synthesis of the 3-amino-6-deoxy-L-hexose daunosamine in a daunorubicin producer (Liu and Thorson 1994).

The identification of additional genes adjacent to *stsC* and *stsA* in the *str/sts* gene cluster of *S.* *griseus* N2-3-11 revealed the existence of at least three other genes that, according to the proposed pathway leading to streptidine formation (Walker 1975), could be involved in aminocyclitol production: (1) *stsB*, putatively encoding a dehydrogenase that could be involved in keto-aminocyclitol formation (see above) since the product of the *strI* gene (Mansouri et al. 1991) has been recently shown to be the first step of ketocyclitol-forming oxidase that converts myo-inositol to scyllo-inosose (J. Ahlert and W. Piepersberg, unpublished results); (2) *stsE*, putatively encoding an aminocyclitol kinase; and (3) *strO*, possibly encoding an (amino-)cyclitol-phosphate phosphatase. StrO is the only protein related to known cytoplasmic phosphatases identified among the *str/sts*-gene products so far. Therefore, it could be involved in one or both of the two dephosphorylation steps postulated to occur during the biosynthesis of streptidine-6-phosphate (Walker 1975).In conclusion, we have identified another subset of streptomycin production genes, *strO-stsABCDEFG*, which at least in part encode crucial functions of the streptidine subpathway of streptomycin biosynthesis. As an enzyme of central importance in all diaminocyclitol producers, the enzyme for the first, L-glutamine-dependent, transamination step was identified to be StsC. Further characterization of its enzymatic properties, such as substrate specificity and kinetic data, is now possible and in progress.

Acknowledgements The fruitful comments of our colleagues are thankfully acknowledged. We are grateful to N. Cremer and S. Kuber ski for expert technical assistance and DNA sequence analyses. We thank M. Roggel and H.-J. Altenbach for providing scyllo-inosamine. This work was supported by grants from the Deutsche Forschungsgemeinschaft and the German Ministry of Science and Education (BMBF; grant no. BEO22-0310185).

---

### References

- Babcock MJ, Kendrick KE (1988) Cloning of DNA involved in sporulation of *Streptomyces griseus*. J Bacteriol 170:2802–2808
- Beyer S, Distler J, Piepersberg W (1996) New operons in the *str* gene cluster for the biosynthesis of 5′-hydroxystreptomycin in *Streptomyces glaucescens* GLA.0 (ETH 22794) and evidence for pathway-specific regulation by StrR. Mol Gen Genet 250:775–784
- Cooper AJL (1978) Spot test for the determination of α-ketoglutaric acid in human cerebrospinal fluid. Anal Biochem 90:444–446
- Dhillon N, Hale RS, Cortes J, Leadlay PF (1989) Molecular characterization of a gene from *Saccharopolyspora erythraea* (*Streptomyces erythraeus*) which is involved in erythromycin biosynthesis. Mol Microbiol 3:1405–1414
- Distler J, Klier K, Piendl W, Werbitzky O, Böck A, Kresze G, Piepersberg W (1985) Streptomycin biosynthesis in *Streptomyces griseus*. 1. Characterization of streptomycin-idiotrophic mutants. FEMS Microbiol Lett 30:145–150
- Hanahan D (1983) Studies on transformation of *E. coli* with plasmids. J Mol Biol 166:557–580
- Heinzel P, Werbitzky O, Distler J, Piepersberg W (1988) A second streptomycin resistance gene from *Streptomyces griseus* codes for streptomycin-3″-phosphotransferase. Relationship between antibiotic and protein kinases. Arch Microbiol 150:184–192
- Hopwood DA, Bibb MJ, Chater KF, Kieser T, Bruton CJ, Kieser HM, Lydiate DJ, Smith CP, Ward JM, Schrempf H (1985) Genetic manipulation of *Streptomyces*: a laboratory manual. The John Innes Foundation, Norwich
- Kagan RM, Clarke S (1994) Widespread occurrence of three sequence motifs in diverse S-adenosylmethionine-dependent methyltransferases suggest a common structure for these enzymes. Arch Biochem Biophys 310:417–427
- Kim CG, Kirsching A, Bergon P, Ahn Y, Wang JJ, Shibuya M, Floss HG (1992) Formation of 3-amino-5-hydroxy-benzoic acid, the precursor of mC7N units in ansamycin antibiotics, by a new variant of the shikimate pathway. J Am Chem Soc 114:4941–4943
- Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227:680–685
- Liu H-W, Thorson JS (1994) Pathways and mechanisms in the biosynthesis of novel deoxysugars by bacteria. Annu Rev Microbiol 48:223–256
- Lucher LA, Chen Y-M, Walker JB (1989) Reactions catalysed by purified L-glutamine:keto-scyllo-inositol aminotransferase, an enzyme required for biosynthesis of aminocyclitol antibiotics. Antimicrob Agents Chemother 33:452–459

- Mansouri K, Piepersberg W (1991) Genetics of streptomycin production in *Streptomyces griseus*: nucleotide sequence of five genes, *strFGHIK*, including a phosphatase gene. Mol Gen Genet 228:459–469
- Marck C (1988) “DNA-Strider”: a “C” programm for the fast analysis of DNA and protein sequences on the Macintosh family of computers. Nucleic Acids Res 16:1829–1836
- Mayer G, Vögtli M, Pissowatzki K, Hütter R, Piepersberg W (1988) Colinearity of streptomycin production genes in two species of *Streptomyces*. Evidence for occurrence of a second amidinotransferase gene. Mol Gen (Life Sci Adv) 7:83–87
- Mehling A, Wehmeier UF, Piepersberg W (1995) Application of random amplified polymorphic DNA (RAPD) assays in identifying conserved regions of actinomycete genomes. FEMS Microbiol Lett 128:119–126
- Merson-Davies L, Cundliffe E (1994) Analysis of five tylosin biosynthetic genes from the *tylIBA* region of the *Streptomyces fradiae* genome. Mol Microbiol 13:349–355
- Neumann T, Piepersberg W, Distler J (1996) Decision phase regulation of streptomycin production in *Streptomyces griseus*. Microbiology 142:1953–1963
- Ohta T, Dairi T, Hashimoto E, Hasegawa M (1993) Use of a heterologous gene for molecular breeding of actinomycetes producing structurally related antibiotics: self-defense genes of producers of fortimicin-A (astromicin)-group antibiotics. Actinomycetologia 7:145–155
- Pasarella S, Bossa F (1994) Similarity between pyridoxal/pyridoxamine phosphate-dependent enzymes involved in dideoxy and deoxyaminosugar biosynthesis and other pyridoxal phosphate enzymes. Protein Sci 3:701–705
- Pearson WR, Lipman DJ (1988) Improved tools for biological sequence analysis. Proc Natl Acad Sci USA 85:2444–2448
- Peschke U, Schmidt H, Zhang H-Z, Piepersberg W (1995) Molecular characterization of the lincomycin production gene cluster of *Streptomyces lincolnensis* 78-11. Mol Microbiol 16:1137–1156
- Piepersberg W (1992) Metabolism and cell individualization. In: Chadwick D, Whelan J (eds) Secondary metabolites: their function and evolution, Ciba Foundation Symposium 171. Wiley, Chichester, pp 294–304
- Piepersberg, W (1994) Pathway engineering in secondary metabolite-producing actinomycetes. Crit Rev Biotechnol 14:251–285
- Piepersberg W (1995) Streptomycin and related aminoglycosides. In: Vining L, Stuttard C (eds) Biochemistry and genetics of antibiotic biosynthesis. Butterworth-Heinemann, Boston, pp 531–570
- Piepersberg W, Distler J (1997) Aminoglycosides and sugar components in other secondary metabolites. In: Rehm H-J, Reed G, Pühler A, Stadler P (eds) Biotechnology, 2nd edn, vol 7. VCH, Weinheim, pp 397–488
- Piepersberg W, Distler J, Heinzel P, Perez-Gonzalez J-A (1988) Antibiotic resistance by modification: many resistance genes could be derived from cellular control genes in actinomycetes—a hypothesis. Actinomycetology 2:83–98
- Pissowatzki K, Mansouri K, Piepersberg W (1991) Genetics of streptomycin production in *Streptomyces griseus*. Molecular structure and putative function of genes *strELMB2N*. Mol Gen Genet 231:113–123
- Reeves P (1993) Evolution of *Salmonella O* antigen variation by interspecific gene transfer on a large scale. Trends Genet 9:17–22
- Retzlaff L, Mayer G, Beyer S, Ahlert J, Verseck S, Distler J, Piepersberg W (1993) Streptomycin production in *Streptomyces*: a progress report. In: Baltz RH, Hegemann GD, Skatrud PL (eds) Industrial microorganisms: basic and applied molecular genetics. American Society for Microbiology, Washington, DC, pp 183–194
- Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning. A laboratory manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
- Sanger F, Nicklan S, Coulson AR (1977) DNA sequencing with chain-determination inhibitors. Proc Natl Acad Sci USA 74:5463–5467
Stockmann M, Piepersberg W (1992) Gene probes for the detection of 6-deoxyhexose metabolism in secondary metabolite-producing streptomycetes. FEMS Microbiol Lett 90:185–190

Studier WF, Rosenberg AH, Dunn JJ, Dubendorff JW (1990) Use of the T7 RNA polymerase to direct expression of cloned genes. Methods Enzymol 185:60–89

Stutzman-Engwall K, Otten S, Hutchinson CR (1992) Regulation of secondary metabolism in *Streptomyces* sp. and overproduction of daunorubicin in *Streptomyces peucetius*. J Bacteriol 174: 144–154

Takano E, White J, Thompson CJ, Bibb MJ (1995) Construction of thiotrepton-inducible, high-copy-number expression vectors for use in *Streptomyces* sp. Gene 166:133–137

Tercero JA, Espinosa JC, Lacalle RA, Jimenez A (1996) The biosynthetic pathway of the aminonucleoside antibiotic puromycin, as deduced from the molecular analysis of the *pur* cluster of *Streptomyces alboniger*. J Biol Chem 271:1579–1590

Thorson JS, Lo SF, Liu H-W, Hutchinson CR (1993) Biosynthesis of 3,6-dideoxyhexoses: new mechanistic reflections upon 2,6-dideoxy, 4,6-dideoxy, and amino sugar construction. J Am Chem Soc 115:6993–6994

Walker JB (1975) Pathways of biosynthesis of the guanidinated inositol moieties of streptomycin and bluensomycin. Methods Enzymol 43:429–470

Walker JB (1990) Possible evolutionary relationships between streptomycin and bluensomycin biosynthetic pathways: detection of novel inositol kinase and O-carbamoyltransferase activities. J Bacteriol 172:5844–5851

Walker JB (1995) Enzymatic synthesis of aminocyclitol moieties of aminoglycoside antibiotics from inositol by *Streptomyces* sp.: detection of glutamine-aminocyclitol aminotransferase and diaminocyclitol aminotransferase activities in a spectinomycin producer. J Bacteriol 177:818–822

Walker JB, Walker MS (1969) Streptomycin biosynthesis. Transamination reactions involving inosamines and inosadiamines. Biochemistry 8:763–770

Yanisch-Perron C, Vieira J, Messing J (1985) Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119
