
JOURNAL OF BACTERIOLOGY, Aug. 1997, p. 4841–4849

0021-9193/97/$04.00+0

Copyright © 1997, American Society for Microbiology

# Cloning of Genes Involved in Carbazole Degradation of *Pseudomonas* sp. Strain CA10: Nucleotide Sequences of Genes and Characterization of meta-Cleavage Enzymes and Hydrolase

SHIN-ICHI SATO,$^{1}$ NAOKI OUCHIYAMA,$^{2}$ TOSHIAKI KIMURA,$^{1\dagger}$ HIDEAKI NOJIRI,$^{1}$ HISAKAZU YAMANE,$^{1}$ AND TOSHIO OMORI$^{1*}$

Biotechnology Research Center, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113,$^{1}$ and Kurume Research Laboratories, Chemicals Inspection & Testing Institute, Kurume Fukuoka 830,$^{2}$ Japan

Received 13 January 1997/Accepted 29 April 1997

The DNA fragment encoding *meta*-cleavage enzymes and the *meta*-cleavage compound hydrolase, involved in carbazole degradation, was cloned from the carbazole-utilizing bacterium *Pseudomonas* sp. strain CA10. DNA sequence analysis of this 2.6-kb *Sma*I-*Sph*I fragment revealed that there were three open reading frames (ORF1, ORF2, and ORF3, in this gene order). ORF1 and ORF2 were indispensable for *meta*-cleavage activity for 2′-aminobiphenyl-2,3-diol and its easily available analog, 2,3-dihydroxybiphenyl, and were designated *carBa* and *carBb*, respectively. The alignment of CarBb with other *meta*-cleavage enzymes indicated that CarBb may have a non-heme iron cofactor coordinating site. On the basis of the phylogenetic tree, CarBb was classified as a member of the protocatechuate 4,5-dioxygenase family. This unique extradiol dioxygenase, CarB, had significantly higher affinity and about 20-times-higher *meta*-cleavage activity for 2,3-dihydroxybiphenyl than for catechol derivatives. The putative polypeptide encoded by ORF3 was homologous with *meta*-cleavage compound hydrolases in other bacteria, and ORF3 was designated *carC*. The hydrolase activity of CarC for 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoic acid, the *meta*-cleavage compound of 2,3-dihydroxybiphenyl, was 40 times higher than that for 2-hydroxy-6-oxohepta-2,4-dienoic acid, the *meta*-cleavage compound of 3-methylcatechol. Alignment analysis and the phylogenetic tree indicate that CarC has greatest homologies with hydrolases involved in the monoaromatic compound degradation pathway. These results suggest the possibility that CarC is a novel type of hydrolase.

Microorganisms which degrade xenobiotic compounds such as polycyclic aromatic hydrocarbons and heterocyclic aromatic compounds are widely spread throughout the environment and have been isolated on the basis of their ability to assimilate these compounds. In the general bacterial degradation system, the aromatic compounds are converted to the corresponding cis-dihydrodiol compounds and then to the corresponding catechol-type derivatives, after which they are cleaved into the *meta*- or *ortho*-ring fission compounds (10, 54). The degradation pathways and degradative genes of polycyclic aromatic hydrocarbons and biphenyl derivatives have been well studied (5, 14, 22, 26, 30, 34, 35, 51). On the other hand, the catabolic pathways of heterocyclic aromatic compounds such as dibenzothiophene are established (29, 40), but there is little information on the degradative genes of these compounds (11).

Carbazole (CAR) is a heterocyclic aromatic compound containing a dibenzopyrrole system, being derived from coal tar and shale oil (37). As CAR is known to possess mutagenic and toxic activities (3) and also to be a recalcitrant molecule (9), serious problems arise if it is released into the environment. Although several CAR-utilizing bacteria have been isolated (17, 18, 25, 32, 41), almost no information has been obtained on the genes involved in the degradation of CAR. The conversion of CAR by bacterial multicomponent dioxygenases such as naphthalene 1,2-dioxygenase or biphenyl 2,3-dioxygenase was examined by Resnick et al. (42), but these dioxygenases could not attack the angular position adjacent to the nitrogen atom.

Previously, we isolated *Pseudomonas* sp. strain CA10 from a sample of activated sludge as a microorganism with the ability to utilize CAR as a sole source of carbon, nitrogen, and energy (41). As shown in Fig. 1, the degradation pathway of CAR in strain CA10 was proposed on the basis of identification of metabolites during growth and comparison with other aromatic compound degradative pathways such as biphenyl and naphthalene. The initial step of degradation of CAR is considered to be dioxygenation at the angular position adjacent to the nitrogen atom to give the dihydroxylated intermediate, which is considered to be spontaneously converted to 2′-aminobiphenyl-2,3-diol. Then extradiol dioxygenase attacks the hydroxylated ring at the *meta* position. Hydrolysis of the *meta*-cleavage compound yields anthranilic acid, which is further converted to catechol. Catechol is considered to be metabolized through the β-ketoadipate pathway.

In a previous study (31), we isolated and characterized a transposon Tn5 mutant of strain CA10 which is deficient in CAR catabolism, and we cloned the Tn5 and its flanking sequences. We also reported that the cloned DNA fragment contained the *meta*-cleavage enzyme and that the genes involved in degradation of CAR were clustered. Recently, we succeeded in cloning the genes encoding the enzymes involved

* Corresponding author. Mailing address: Biotechnology Research Center, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113, Japan. Phone: 81(3)3812-2111, ext. 3067. Fax: 81(3)5802-3326. E-mail: aseigyo@hongo.ecc.u-tokyo.ac.jp.
† Present address: Bio Research Laboratory, Toyota Motor Corporation, Aichi 471-71, Japan.

4842 SATO ET AL. J. BACTERIOL.

![image](https://i.imgur.com/1234567.png)

FIG. 1. Degradation pathway of carbazole by *Pseudomonas* sp. strain CA10. Gene designations: *carA*, carbazole 1,9a-dioxygenase; *carB*, 2′-aminobiphenyl-2,3-diol 1,2-dioxygenase; *carC*, 2-hydroxy-6-oxo-6-(2′-aminophenyl)-hexa-2,4-dienoic acid (*meta*-cleavage compound) hydrolase. Compounds: I, carbazole; II, 2′-aminobiphenyl-2,3-diol; III, 2-hydroxy-6-oxo-6-(2′-aminophenyl)-hexa-2,4-dienoic acid; IV, anthranilic acid.

in the carbazole degradation pathway from the wild-type strain CA10, and *meta*-cleavage enzyme and hydrolase were found to be contained in the DNA fragment (6.9-kb EcoRI fragment) corresponding to the above-mentioned DNA sequences with the Tn5 insertion. We also found that the 6.9-kb EcoRI fragment contained the genes encoding carbazole 1,9a-dioxygenase (CARDO).

We herein describe the details of cloning and characterization of the genes encoding *meta*-cleavage enzyme and hydrolase involved in CAR degradation. The cloning and characterization of CARDO are reported in the accompanying report (46).

## MATERIALS AND METHODS

### Bacterial strains, plasmids, media, and culture conditions.
Bacterial strains and plasmids used in this study are listed in Table 1. Strain CA10 was grown as described previously (41). *Escherichia coli* JM109 (57) was used as the host strain for plasmids pUC118 and pUC119 (*lacZ Ap*, 3,162 bp [55]), pSTV29 (*lacZ Cm*, 2,999 bp [Takara Shuzo Co., Ltd., Kyoto, Japan]), and their derivatives. *E. coli* strains were grown on 2× YT medium (43) at 37°C. Ampicillin and chloramphenicol (final concentrations, 50 and 30 μg/ml, respectively) were used for liquid cultures and selection on agar plates. For plate cultures, the above-described media solidified with 1.6% (wt/vol) agar were used.

### DNA manipulation.
Plasmid DNA was prepared from the *E. coli* host strain by the alkaline lysis method (7). Total DNA of strain CA10 was prepared from cells grown on 2× YT medium at 30°C according to a protocol using hexadecyltrimethyl ammonium bromide (6). Restriction endonuclease, T4 DNA ligase, and a DNA blunting kit (Takara Shuzo) were used according to the manufacturer's instructions. DNA fragments were extracted by the glass powder method (Geneclean II kit; Bio101 Inc., La Jolla, Calif.) as instructed by the manufacturer. Other DNA manipulations were performed according to standard methods (43).

### Cloning of the genes encoding *meta*-cleavage enzymes.
Total DNA of strain CA10 was digested with restriction endonuclease EcoRI or SphI and ligated to pUC119 vector digested with EcoRI or SphI, respectively. Transformation of *E. coli* JM109 was done as described by Hanahan (21), and ampicillin-resistant transformants were selected on 2× YT agar plates containing ampicillin. The activity of *meta*-cleavage enzymes was assayed by observing whether the color of the transformant colonies changed to yellow after the plates were sprayed with ethereal 2,3-dihydroxybiphenyl (0.5 mg/ml) (16). The solution of 2,3-dihydroxybiphenyl was prepared fresh daily.

### Hybridization experiments.
Southern blotting (50) was performed by using a Hybond-N+ nylon membrane (Amersham Life Science, Tokyo, Japan) as recommended by the manufacturer. A nonradioactive digoxigenin DNA labeling and detection kit (Boehringer GmbH, Mannheim, Germany) was used according to the manufacturer’s instructions. Hybridizations were carried out at 68°C in the presence of 2× SSC (1× SSC is 0.15 M NaCl plus 0.015 M sodium citrate). After the hybridization, the membranes were washed with 0.1× SSC containing 0.1% sodium dodecyl sulfate (SDS) at 68°C for 30 min.

### Resting cell reaction.
*E. coli* JM109 cells harboring recombinant plasmids were cultivated in 100 ml of 2× YT medium at 37°C. When the optical density at 550 nm reached 0.8, isopropyl-β-D-thiogalactopyranoside (IPTG) was added to a final concentration of 0.5 mM, and the cells were incubated for another 5 h. Then the cells were harvested by centrifugation and washed twice with ice-cold 50 mM sodium phosphate buffer (pH 7.5), and the resultant cells were resuspended in 10 ml of the same buffer. A solution of 2,3-dihydroxybiphenyl (100 mg/ml) was prepared in ethanol, and then a 100-μl aliquot of this solution was added to each batch of resting cells. The resultant reaction mixture was incubated on a reciprocal shaker (300 strokes/min) at 30°C for 1 h. Metabolites were extracted with ethyl acetate after acidification to pH 3.0 with 1 N HCl. The ethyl acetate layer was dried with anhydrous sodium sulfate and then evaporated to dryness in vacuo at below 40°C. Each extract was methylated with ethereal diazomethane and subjected to gas chromatography-mass spectrometry (GC-MS).

### GC-MS.
For GC-MS, a model JMS-Automass 150 gas GC-MS system (JEOL, Ltd., Tokyo, Japan) fitted with a fused silica chemically bonded capillary column (DB-5; 0.25 mm [inside diameter] by 15 m, 0.25-μm film thickness; J & W Scientific Inc., Folsom, Calif.) was used. Each sample was injected onto the

## TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant characteristics | Source or reference |
|-------------------|--------------------------|----------------------|
| **Bacterial strains** | | |
| *Pseudomonas* sp. strain CA10 | Car<sup>+</sup> | 41 |
| *E. coli* JM109 | *recA1 endA1 gyrA96 thi hsdR17 supE44 relA1 Δ(lac-proAB)/F'*[traD36, proAB<sup>+</sup>, *lacI<sup>q</sup> lacZΔM15*] | 57 |
| **Plasmids** | | |
| pUC118 | *Ap<sup>r</sup>* | 55 |
| pUC119 | *Ap<sup>r</sup>* | 55 |
| pSTV29 | *Cm<sup>r</sup>* | Takara Shuzo |
| pTD21 | *Ap<sup>r</sup>*, pUC119 with EcoRI insert of *Pseudomonas* sp. strain TD2 DNA | 31 |
| pUCA1 | *Ap<sup>r</sup>*, pUC119 with EcoRI insert of *Pseudomonas* sp. strain CA10 DNA | This study |
| pUCA121 | *Ap<sup>r</sup>*, pUC118 with SmaI-BglII insert from pUCA1 insert | This study |
| pUCA122 | *Ap<sup>r</sup>*, pUC118 with SmaI-SphI insert from pUCA1 insert | This study |
| pUCA15 | *Ap<sup>r</sup>*, pUC119 with EcoRI-XhoI insert from pUCA1 insert | This study |
| pSCA16 | *Cm<sup>r</sup>*, pUC119 with XhoI-BglII insert from pUCA1 insert | This study |
| pUCA17 | *Ap<sup>r</sup>*, pUC119 with EcoRI-SacII insert from pUCA1 insert | This study |
| pUCA18 | *Ap<sup>r</sup>*, pUC119 with EcoRI-BglII insert from pUCA1 insert | This study |
| pUCA123 | *Ap<sup>r</sup>*, pSTV29 with SmaI-KpnI insert from pUCA1 insert | This study |
| pUCA191 | *Ap<sup>r</sup>*, pUC119 with HindIII-SphI insert from pUCA1 insert | This study |
| pYN105 | *Ap<sup>r</sup>*, pUC119 with SphI insert of *Pseudomonas* sp. strain CA10 DNA | This study |

<sup>a</sup> Car<sup>+</sup> indicates the ability to grow on CAR as a sole source of carbon, nitrogen, and energy.

VOL. 179, 1997                                                                 CLONING OF PSEUDOMONAS SP. STRAIN CA10 car GENES 4843

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 2. (A) Restriction map of the 6.9-kb EcoRI insert of pUCA1 and relationship of the deleted plasmids. Only restriction sites of relevant enzymes are shown. The location and orientation of the lac promoter of the pUC118 and pUC119 vectors are indicated by arrows. The presence (+) or absence (-) of meta-cleavage activity, which was assayed by observing the change of color of the transformant colonies after being sprayed with ethereal 2,3-dihydroxybiphenyl, is shown at the right. Restriction site designations: E, EcoRI; Sm, SmaI; Xh, XhoI; Bg, BglII; Sp, SphI; K, KpnI; H, HindIII; P, PstI; Sc, SacII.

column at 60°C in the splitless mode. After a 2-min isothermal hold at 60°C, the column temperature was increased at 4°C/min to 72°C and then increased at 16°C/min to 280°C. The head pressure of helium carrier gas was 65 kPa.

Nucleotide sequence determination and phylogenetic analyses. Unidirectional deletion mutants were constructed by using a DNA deletion kit (Takara Shuzo) (24). Nucleotide sequence determination was carried out by the chain termination method of Sanger et al. (44), using an Applied Biosystems 373A DNA sequencer (Perkin-Elmer Japan Co. Ltd., Chiba, Japan), and the obtained nucleotide sequences were analyzed with DNASIS-Mac software (version 3.6; Hitachi Software Engineering Co. Ltd., Yokohama, Japan). Alignment of the CarBb, CarB2, and CarC sequences was performed with the CLUSTAL W package (53). Programs used to infer phylogenetic trees are contained in the PHYLIP package (version 3.5c; obtained from J. Felsenstein, University of Washington). PROT DIST with the Dayhoff PAM matrix option was used to calculate evolutionary distances. Phylogenetic trees were constructed from evolutionary distance data by the neighbor-joining method (45), implemented through the program NEIGHBOR. A total of 100 bootstrapped replicate data sets for PROTDIST were generated with the program SEQBOOT, to provide confidence estimates for tree topologies (15).

Preparation of cell extract. *E. coli* harboring recombinant plasmid pUCA122, pYN105, or pUCA191 was grown to an optical density of 0.8 at 550 nm. After further incubation in the presence of 0.5 mM IPTG for 5 h at 37°C, the cells were harvested, washed twice with 50 mM sodium phosphate buffer (pH 7.5) containing 10% acetone, and resuspended in the same buffer. The resultant cell suspensions were sonicated and centrifuged at 17,000 × g at 4°C for 60 min, and their supernatants were used as cell extracts.

Measurement of enzymatic activities. The meta-cleavage activities for 2,3-dihydroxybiphenyl in the cell extracts were assayed by measuring the increase in \(A_{434}\) for 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienic acid (HOPDA; the meta-cleavage compound of biphenyl) with a Beckman DU-7400 spectrophotometer equipped with a thermostatted cuvette holder and a TAI TE C circulating water bath. The assay was performed at 25°C by using 50 mM sodium phosphate buffer (pH 7.5) containing 10% acetone. The substrate was dissolved in ethanol, and the resultant stock solution was added to the reaction buffer at the desired concentrations. The reaction was initiated by the addition of 5 μl of appropriately diluted cell extracts to the reaction buffer. One unit of activity was defined as the amount of the enzyme required to form 1 μmol of meta-cleavage compound per min at 25°C. The molar extinction coefficient of HOPDA under this assay condition was taken to be 13,200 cm\(^{-1}\) M\(^{-1}\) (13).

The relative ring cleavage activities were determined from the extinction coefficients of the ring fission products formed from the following substrates: catechol (\(\lambda_{\text{max}}\), 375 nm; \(E\), 36,000 cm\(^{-1}\) M\(^{-1}\)), 3-methylcatechol (\(\lambda_{\text{max}}\), 388 nm; \(E\), 32,000 cm\(^{-1}\) M\(^{-1}\)), and 4-methylcatechol (\(\lambda_{\text{max}}\), 382 nm; \(E\), 17,000 cm\(^{-1}\) M\(^{-1}\)) (5). Specific activity was defined as the number of enzyme units per milligram of protein. 2′-Aminobiphenyl-2,3-diol was prepared from CAR by use of the resting cells of *E. coli* harboring pUCARA (46) and extracted as described for the resting cell reaction. The ethyl acetate extract from 5 ml of reaction mixture, which was redissolved in 500 μl of ethanol after the evaporation of ethyl acetate, was used as the substrate stock solution. The meta-cleavage activity for 2′-aminobiphenyl-2,3-diol was determined by monitoring the oxygen consumption during the dioxygenase reaction with a Clark-type oxygen electrode (Iijima Electronics Co., Aichi, Japan); 2 ml of 50 mM sodium phosphate buffer (pH 7.5) containing 10% acetone and supplemented with 200 μl of cell extract was used as the reaction mixture. Reactions were initiated by the addition of 20 μl of the substrate stock solution.

Protocatechuate 4,5-dioxygenase activity was determined by measuring the \(A_{410}\) of the reaction mixture, using the cell extract of *E. coli* harboring pUCA122, as described by Noda et al. (38).

The hydrolase activity of the cell extracts was assayed by monitoring the decrease in \(A_{434}\) and \(A_{388}\) for HOPDA and for 2-hydroxy-6-oxohepta-2,4-dienic acid (HOHDA; meta-cleavage compound of toluene), respectively. HOPDA and HOHDA were prepared from 2,3-dihydroxybiphenyl and 3-methylcatechol by the resting cell reaction of *E. coli* harboring pUCA122 and pIP103 (2), respectively, as described above. After the conversion to meta-cleavage compounds, 5 ml of each reaction mixture was centrifuged to remove the cells, and the yellow supernatant was extracted with ethyl acetate. The ethyl acetate layer was evaporated to dryness at 25°C in vacuo. The residue was redissolved in 100 μl of ethyl acetate and used as the substrate for CarC. The assay was performed at 25°C by using 50 mM sodium phosphate buffer (pH 7.5) containing 10% acetone. The \(A_{434}\) and \(A_{388}\) of the reaction mixtures were adjusted to about 1.0 to 1.2 with ethyl acetate solutions of HOPDA and HOHDA, respectively. The reaction was initiated by the addition of 5 μl of appropriately diluted cell extracts to the reaction mixture. One unit of activity was defined as the amount of the enzyme required to degrade 1 μmol of meta-cleavage compound per min at 25°C.

The hydrolase activity for 2-hydroxy-6-oxo-6-(2′-aminophenyl)hexa-2,4-dienic acid (HOADA; the meta-cleavage compound of 2′-aminobiphenyl-2,3-diol) was determined by observing the disappearance of the yellow meta-cleavage compound. Using the resting cells of *E. coli* harboring both pUCA18 and pSCA14 (46), we prepared HOADA from CAR as described above. The assay was performed under the conditions described above.

Determination of protein concentrations. The protein concentration was determined by the method of Bradford (8), with bovine serum albumin as a standard, using Protein Assay Kit II (Bio-Rad Laboratories, Richmond, Calif.).

SDS-PAGE. SDS-polyacrylamide gel electrophoresis (SDS-PAGE) was performed by the method of Laemmli (33), using 15% polyacrylamide gels. Protein staining of the gels was performed with Coomassie brilliant blue R-250.

Chemicals. CAR was purchased from Katayama Chemical Industries Co. Ltd., Osaka, Japan. Although this CAR was of the highest purity commercially available, recrystallization from ethanol was done for further purification. 2,3-Dihydroxybiphenyl was purchased from Wako Pure Chemical Industries, Ltd., Osaka, Japan, for use as an analogous compound of 2′-aminobiphenyl-2,3-diol. All other chemicals were of the highest purity commercially available.

Nucleotide sequence accession number. The nucleotide sequence data for the insert DNA fragments of pUCA1 and pYN105 are available from the DDBJ, EMBL, and GenBank nucleotide sequence databases under accession no. D89064 and D89065, respectively.

---

**RESULTS**

Cloning of the genes encoding meta-cleavage enzymes. *E. coli* JM109 was transformed with the genomic library, and the

ampicillin-resistant colonies were selected and subsequently screened for production of the yellow compound after being sprayed with ethereal 2,3-dihydroxybiphenyl, which is an analog of 2′-aminobiphenyl-2,3-diol, a catabolic intermediate of CAR degradation. As a result of this screening, we obtained some clones which represented the *meta*-cleavage activity for 2,3-dihydroxybiphenyl, and plasmid DNAs were prepared from these clones. On the basis of restriction mapping analyses, these clones were classified into two types of plasmids, designated pUCA1 (6.9-kb EcoRI insert) and pYN105 (1.4-kb SphI insert).

**Hybridization analysis.** To examine which of the cloned genes encoding the *meta*-cleavage enzymes were derived from strain CA10 and involved in the degradation of CAR, Southern blot analysis was carried out. We had previously obtained a Tn5 mutant, strain TD2, which was deficient in the conversion from CAR to anthranilic acid, and cloned the Tn5-flanking region, the insert of pTD21, from strain TD2 (31). Therefore, pTD21 and the EcoRI-digested total DNA of strain CA10 were electrophoresed and transferred onto a nylon membrane, and then the membrane was hybridized with the insert DNA fragment of pUCA1 or pYN105. The labeled insert of pUCA1 hybridized to CA10 genomic DNA fragments and pTD21, while the labeled insert DNA of pYN105 hybridized to the CA10 genomic DNA fragment but not to pTD21 (data not shown). This result indicated that the insert DNA fragments of both pUCA1 and pYN105 were derived from strain CA10 and that the insert DNA fragment of pUCA1 encodes the gene for a *meta*-cleavage enzyme involved in the degradation of CAR.

**Mapping of the gene encoding the *meta*-cleavage enzyme.** Figure 2 shows the restriction map of the pUCA1 insert; the region constituting the structural gene for the *meta*-cleavage enzyme was identified by examination of the activities of a series of deletion mutants. Of the mutants carrying the deleted plasmids constructed from pUCA1, the transformant carrying pUCA122, which contained a 1.9-kb SmaI-BglII fragment, showed *meta*-cleavage activity. This result showed that the structural gene encoding the *meta*-cleavage enzyme is located in the 1.9-kb SmaI-BglII fragment. Nevertheless, no *meta*-cleavage activity was observed in the plate assay of the transformant carrying pUCA121, which contains the 1.9-kb SmaI-BglII fragment and its adjacent 0.7-kb BglII-SphI fragment. This result implies the possibilities that the SmaI-SphI fragment also encodes the *meta*-cleavage compound hydrolase and that the yellow *meta*-cleavage compound, which was produced by the *meta*-cleavage enzyme, was quickly converted to benzoic acid as in the study by Kimbara et al. (30).

**Resting cell reactions.** On the basis of the consideration noted above, 2,3-dihydroxybiphenyl was subjected to reaction with resting cells of *E. coli* JM109 harboring pUCA121, pUCA122, or pUC119 (as a control), and the metabolites were analyzed by GC-MS. In the strain carrying either pUC121 or pUCA122, the color of the reaction mixture turned yellow immediately after the addition of 2,3-dihydroxybiphenyl. However, benzoic acid was detected only in the reaction mixture with resting cells of *E. coli* JM109 harboring pUCA121 (data not shown). In the strain carrying pUC119, the color of the reaction mixture did not change, and benzoic acid was not detected as a metabolite. These results clearly indicated that pUCA121 insert DNA encodes not only the *meta*-cleavage enzyme but also the *meta*-cleavage compound hydrolase.

**Nucleotide sequence analysis.** After determining the nucleotide sequence of the 2,614-bp insert of pUCA121, we identified by computer analysis three open reading frames (ORF1, ORF2, and ORF3), which were linked tandemly and considered to be translated to the same direction, as shown in Fig. 3.

VOL. 179, 1997                                                                 CLONING OF PSEUDOMONAS SP. STRAIN CA10 car GENES 4845

**TABLE 2. Substrate specificities of two meta-cleavage enzymes**

| Substrate                | concn (µM) | Activity with enzyme<sup>a</sup> |
|--------------------------|------------|---------------------------------|
|                          |            | JM109 (pUCA 122)               |
|                          |            | JM109 (pYN 105)               |
| 2,3-Dihydroxybiphenyl    | 100        | 100                           |
| Catechol                 | 300        | 1.2                           |
|                          |            | 0.5                           |
| 3-Methylcatechol         | 300        | 4.4                           |
|                          |            | 5.5                           |
| 4-Methylcatechol         | 300        | 5.9                           |
|                          |            | 2.3                           |
| 2′-Aminobiphenyl-2,3-diol| +          | −                             |

<sup>a</sup> Activities measured in cell extracts of *E. coli* recombinant strains carrying plasmids pUCA 122 and pYN 105 for detection of CarB and CarB2 activities, respectively, are expressed as percentages of that measured with 2,3-dihydroxybiphenyl as a substrate, taken as 100%. The change of the absorbance of each product was monitored at the corresponding wavelength. Activity of 2′-aminobiphenyl-2,3-diol was determined with an oxygen electrode. +, rapid consumption of dioxygen and simultaneous production of yellow meta-cleavage compounds were observed; –, dioxygen consumption was almost same as for the negative control, using the cell extract of JM109(pUC119), and a meta-cleavage compound was not detected.

**amino acid sequence of ORF2 showed 31.0% identity with LigB of protocatechuate 4,5-dioxygenase, which consists of two subunits, in *Pseudomonas paucimobilis* SYK6 (38). The deduced amino acid sequence of ORF3 showed 30.3, 31.3, and 31.8% identities with TodF (36), XylF (27), and DmpD (39), respectively, which are meta-cleavage compound hydrolases in other bacteria. The amino acid sequence Gly-Xaa-Ser-Xaa-Gly (Fig. 3), which was highly conserved in the hydrolases (1, 19, 26), was also observed in ORF3. These results indicate that ORF3 encodes the meta-cleavage compound hydrolase; thus, ORF3 was designated carC.**

We also sequenced the about 1,351-bp insert of pYN 105


1   GCATGCAGCGCACCGTAGGTATGGGTTGAGCGGAAGCGGATACCCATGCGGTGCGGTATTGAT     60
61  GGGTATCGCAAAGCTCAACCATCCTACGAAAAGCCGACCGTGCCGCGCGTCCCCTCGCA        120
121 ATGTGCCCGCCACCGCCACAACCCCCGATTTCCGCGCCTTTCGCGCTAGTCCCATTTGA        180
181 CGATGCCTCCCCCTCCGGCTCGCCGATAAATCCTCCCCATGGAGCCCCGCGTACGACT        240
241 GCGCGGGGCGGATGCAATGAGGAGAACCGCATGGATATCGGTGGCCTGGGTTACGTCAC        300
1   carB2 M D I R G L G Y V T                                      10
301 CGTAGCCTCGACAGACCTGACGCGCTGGAGCGACTACGCCACCGGGGTGCTGGCATGAT        360
10  V A S T D L T R W S D Y A T G V L G M M                         30
361 GGTTGATGCGGCGCCCATGAGCGGCTCTACCTGAAAGATGGAGCGGCCCCCTATCGCAT        420
30  V D A G A H E R L Y L K M D E R P Y R I                         50
421 CCTGGTGGAGCGCGGACCGGACGGCTATGGCGGCTGGGGAAAGTGGCGGGCAA             480
50  L V E R A D R D G Y G A C G W E V A G K                         70
481 GGCGGCTTCCGAGCAGGCCATCGCCGAACTGCAGCAGGCCGATGTCGAGGTTCCGCGTGG       540
70  A A F E Q A I A E L Q Q A D V E V R R G                         90
541 CAGCGCCAGCGACGCGCCTCGCGCAAAGTGCGAGGACTGGCCCTGTTCGCGGACCCGGA        600
90  S A S D A A S R K V Q E L A L F A D P D                        110
601 CGGCAACCGTCATGAGCTGTTCTGGGGCCCAGCTGCAGGATTTTCGCGCCATTCATTTCCGC      660
110 G N R H E L F W G P L Q D F A P F I S P                        130
661 GGCGGCGTGTCGGTTTCGTCACCAAGCGGCTGGGCCATGGGGTCTCTGCGGC              720
130 A G V S G F V T S A L G M G H A V L P A                        150
721 GCCGTCCCTCGACCGCTGGGCTTCTACCAGGGGTCATGGGCTTCCGGCCCTCTCCGA         780
150 P S F D R C L D F Y Q R V M G F G L S D                        170
781 CCTGATGAAAGTGCGCTTCACCCCCGACCCCCGAGCCCCGAAAAAACGCATCCACTTCCCT      840
170 L M K V R F T P D P A E P E K R I H F L                        190
841 CCATTGCAAACAACGGCCCGCCACCCTCCCTGGCGTTGTCGAATGCCGATCCCTCCGG        900
190 H C N N A R H H F P G V V R M P D P S G                        210
901 TTGCGTGCA CCTGATGGTGGAA GTGCCGAGCTGGACAGCTGGCCCGCCTTGGACCG        960
210 C V H L M V E V R E L D D V G R A L D R                        230
961 CATGACGGCCACGGCGTGAAGCTCCTCGCCACCTTCGGCCGCCACCAACGAGACAT         1020
230 M T A H G V K L S A T F G R H T N D D                          250
1021 GACCAGCTTCTACATGCAGACCCCCGGCGGCTTCGACCTGGAGTACGGCTGGGCGGCAA       1080
250 T S F Y M Q T P G G F D L E Y G C G G K                        270
1081 GGTCATGGGACTGGGAGGTTCCACCCCCCTTCGAAAGCACGGTGGTCAGCCACTGGGGCCA     1140
270 V M D W E V H T P F E S T V V S H W G H                        290
1141 TGACTTCAGCGTTGGCCGCGGCTAAGGAGCAGCACCATGGATAAACGCATGACCGCGCC       1200
290 D F S V G R R *                                              299
1201 GACCTGGTGGCCAGCTGGCGGATGGCATGACCATCGGGTGGGCTGGGGCCCAGG           1260
1261 CGCAAGCCGATGGGCCCTGGGTACGGGAGATACTCCGCTCCGACCTCAAGGACCTCACC GTG   1320
1321 GTCGCCTATGGCGGTGCCGATGTCGGCATGC                                 1351


FIG. 4. Nucleotide and deduced amino acid sequences of the *SphI* fragment of pYN 105. The sense DNA strand is shown. The putative coding region spans nucleotides 272 to 1165. The putative SD sequence is underlined. Asterisks indicate stop codons.

**TABLE 3. Kinetic parameters of two meta-cleavage enzymes cloned from strain CA10<sup>a</sup>**

| Enzyme | Substrate | $K_m$ (µM) | $V_{\text{max}}$ (nmol/min/mg) |
|--------|-----------|-------------|-------------------------------|
| CarB   | 2,3-DHBP  | 1.2 (0.1<sup>b</sup>) | 7,435                     |
|        | 3-MC      | 1,590.7 (171.4) | 2,016                      |
| CarB2  | 2,3-DHBP  | 16.3 (1.2) | 809                          |
|        | 3-MC      | 703.4 (93.2) | 179                          |

<sup>a</sup> Activities were measured in cell extracts of *E. coli* recombinant strains carrying plasmids pUCA 122 and pYN 105 for detection of CarB and CarB2 activities, respectively. 2,3-DHBP, 2,3-dihydroxybiphenyl; 3-MC, 3-methylcatechol.

<sup>b</sup> Standard error of the mean ($n = 3$).

(Fig. 4). Computer analysis revealed that there was one ORF, and its deduced amino acid sequence had 40.2 and 37.0% identities with 2,3-dihydroxybiphenyl dioxygenases (BphCs) of *P. pseudoalcaligenes* KF 707 (51) and *Pseudomonas* sp. strain KKS 102 (30), respectively. These results indicate that this ORF encodes the meta-cleavage enzyme; thus, we designated it carB2.

Deletion and complementation analysis of ORF1 and ORF2. The transformant harboring pUCA 122 containing the *SmaI*-BglII fragment encoding complete ORF1 and ORF2 showed the meta-cleavage activity for 2,3-dihydroxybiphenyl (Fig. 2). This result indicated that either ORF1 or ORF2 or both are responsible for the meta-cleavage activity. To examine whether both ORFs are necessary for the meta-cleavage activity, we constructed additional deletion mutants and examined their meta-cleavage activities for 2,3-dihydroxybiphenyl and 2′-aminobiphenyl-2,3-diol by detection of the yellow meta-cleavage compound (16).

ORF1 and ORF2 were subcloned into vectors pUC119 and pSTV 29, respectively, which could exist compatibly in *E. coli* cells, and the resultant plasmids were designated pUCA 123 and pSCA 16, respectively (Fig. 2). Although no yellow colony was observed in *E. coli* cells harboring either pUCA 123 or pSCA 16, the yellow pigmentation of colonies of transformant harboring both plasmids was observed after plates were sprayed with ethereal 2,3-dihydroxybiphenyl as well as 2′-aminobiphenyl-2,3-diol. Since these results strongly indicated that ORF1 and ORF2 are indispensable for the meta-cleavage activities for 2,3-dihydroxybiphenyl and 2′-aminobiphenyl-2,3-diol, ORF1 and ORF2 are designated carBa and carBb, respectively.

Enzymatic activities of two meta-cleavage enzymes. Both CarB and CarB2 showed meta-cleavage activity for 2,3-dihydroxybiphenyl, and their enzymatic activities for 2,3-dihydroxybiphenyl were 1.4 and 0.2 U/mg of protein in crude extracts from *E. coli* transformants harboring pUCA 122 and pYN 105, respectively. No activity was observed with the extract from JM109(pUC119). The activities of catechol, 3-methylcatechol and 4-methylcatechol were almost negligible (Table 2). The kinetic parameters of the two meta-cleavage enzymes for 2,3-dihydroxybiphenyl and 3-methylcatechol were calculated from Lineweaver-Burk plots (Table 3). CarB had the higher affinity for 2,3-dihydroxybiphenyl; its $K_m$ value was about 14 times lower than that of CarB2. The maximum initial velocity ($V_{\text{max}}$) of CarB was two times higher than that of CarB2.

The meta-cleavage activities of these two enzymes for 2′-aminobiphenyl-2,3-diol, which is the intermediate of CAR degradation pathway, were analyzed by dioxygen consumption measured with an oxygen electrode. A rapid consumption of dioxygen and simultaneous production of yellow meta-cleavage compounds were clearly observed in a reaction mixture sup-

were detected in the reaction mixture prepared with the cell  
extract of *E. coli* harboring pUCA122 (CarB) or pYN105  
(CarB2) (data not shown).  

Enzymatic activity of CarC. We constructed pUCA191,  
which contains the HindIII-SphI fragment encoding only *carC*,  
and analyzed the enzymatic activity of CarC in the cell extract.  
The enzymatic activities of the extract from JM109(pUCA191)  
for HOPDA and HOHDA were 1.20 and 0.03 U/mg, respectively. No hydrolase activity was observed with the extract from  
*E. coli* JM109 carrying pUC119. On the other hand, we detected rapid disappearance of the yellow color of HOADA in  
the reaction mixture (data not shown). These results indicated  
that the hydrolase activity of CarC for HOPDA was significantly higher than that for HOHDA and that CarC has hydro-  
lase activity for HOADA.  

Detection of *carBa*, *carBb*, *carC*, and *carB2* gene products.  
To examine if the observed ORFs could be translated to proteins of the predicted sizes in *E. coli* cells, cellular proteins  
were analyzed by SDS-PAGE (Fig. 5). Expression of genes  
contained in the SmaI-BglII insert of pUCA122 yielded two  
peptides with molecular masses of 29 and 10 kDa, which correspond to the predicted molecular masses of the *carBb* and  
*carBa* gene products, respectively (lane 2). Similar analysis of  
the HindIII-SphI insert of pUCA191 and the SphI insert of  
pYN105, containing the *carC* and *carB2* genes, respectively,  
revealed that these inserts encode proteins of 32 and 34 kDa,  
respectively (lane 3 and 4). These estimated molecular masses  
also correspond to the predicted molecular masses of the *carC*  
and *carB2* gene products, respectively. Expression of genes in  

CarBb    MKIVAAAGTSKILMSPKGCEESAA---------RVVNGIAELGRRILKEAR--PDVLVIITSD  
LigB     MARVTTGITSSHIPALGAAIQTGTSDNDYWGPVKGYQPIRDWIKQPGNMPDVVILVYND  
MpcI     --MPIQLECLSBTPLHGYVDPAF----EVVAEVERVQAAARDVRRAFD--PELVVVFAPD  
MhpB     --MHAYLHCLSISPLVGYVDPAQ----EVLDEVNGVIASARERIAAFS--PELVVLFAPD  
**                                                                  
CarBb    HMFININLSMQPRFVVGIADSYTPMGMDIPR--DLVPGSREVGRAIALQADEDGF DLCQA  
LigB     HASAFDMNIIPTFAGCAETFKADEGWGPRVPDVKGHPDLAWHTIAQSLLIDEFDMTIM  
MpcI     HFNGFFYDVMPPFCIGAAA--TAIGDFKSLA--GKLVPADLALS LAESVMAADIDVALS  
MhpB     HYNGFFYDVMPPFCLGVGA--TAIGDFGSAA--GELPVPVELAEACAHVMKSGIDLAVS  
*                                                                  
CarBb    EEYSLDGIMI PILFMG----MKEIPVVPVIVNINTDPIPSARRCVLAESIRQAI EKRT  
LigB     NQMDVDHGCTVPLSMIFGEEPWPCKVIPFVNVVTYP PSGKRCFALGDSIRAAVES-F  
MpcI     HRMQVDRGCADALAALTGS--LHRYPVIPVFINSVAPPMATLRRARLLLGDAVGRFLSR--  
MhpB     YCMQVDRGFAQPLEFLLGG--LDKVPVLPVFNKAVATPLPGFQRTHMLGETIGRFTST--  
***                                                                  
CarBb    PDGCRVA VVGAGGLSHWL CVP-------------------R----------------HGEVSEKF  
LigB     PEDLNHVHWGTGGMSSHQLQGP--------------------------RAGLINKEF  
MpcI     -AGKRVLVVGSGGISHEPPVPELAGASEEEVAERLIAGR N--PSPESAARQARTVAAAKSF  
MhpB     -LNKRVLFLGSGGLSHQPPVPELAKADAHM DRLLLGSGKDLPASERELRQQRVISAAEKF  
* ** * ** *                                                                  
CarBb    D-----------MVMD ELVRGNAEKLVAMGNEAIID-QGGNAGVEILTWIMA AVA  
LigB     DLNFIDKLISDPELSKM PHIQLRESGSEGVELVMW LIMRGALPEKV RDLTYFYHIPAS  
MpcI     VAGDSHLHPLNPE-WNRAFLSLLASGELTAVDGMTNDAITR-DGGKSAHEIRTWVAAFGA  
MhpB     VEDQRTLHPLNP I-WDNQFMTLLEQGRIQELDAVSNEELSA-IAGKSTHEIKTWVAAFAA  
*                                                                  
CarBb    SEASS---GEKV FYEAMTQWFTGIGGMEFH VK--  
LigB     NTALGAMILQPEETAGTPLEPRKVMSGHSLAQ A-  
MpcI     LAAYG PYRASLDFYRAIPEWIAGFATMHA EPA AV  
MhpB     ISAFGNWRSEGRYYRP IPEWIAGFGSLSARTEN-  

FIG. 6. Multiple amino acid sequence alignment of CarBb and other extradiol dioxygenases from *P. paucimobilis* SYK6 (LigB) (38), *A. eutrophus* JMP222 (MpcI)  
(28), and *E. coli* (MhpB) (49). Amino acid residues identical among all proteins are indicated by asterisks, and positions that are well conserved are indicated by dots.  
Histidine residues that are considered to form essential active sites are indicated by shaded boxes.

the SmaI-SphI insert of pUCA121 yielded three proteins, CarBa, CarBb, and CarC (lane 5). As shown in lane 5, CarC was expressed more strongly than CarBa and CarBb. This result is in accordance with the observation that no yellow meta-cleavage product was detected from 2,3-dihydroxybiphenyl on the plate with E. coli harboring pUCA121 and that benzoic acid was identified by GC-MS as a metabolite of 2,3-dihydroxybiphenyl in the reaction mixture prepared with resting cells of JM109 harboring pUCA121.

### DISCUSSION

In this study, we successfully cloned two different genes encoding meta-cleavage enzymes from the CAR-degrading bacterium *Pseudomonas* sp. strain CA10 into pUC119, and the resultant plasmids were named pUCA1 and pYN105. Nucleotide sequence analysis followed by assays of enzymatic activities of the gene products indicated that CarB and CarC encoded by pUCA121 were involved in the degradation of CAR. Many genes encoding meta-cleavage enzymes have been cloned from both gram-negative (2, 54) and gram-positive (4, 34, 35) bacteria, using color detection of meta-cleavage compounds as in our study. Although the protein sequence of CarBb could not be aligned to those shown in Fig. 7A (data not shown), alignment with the other meta-cleavage enzymes of the protocatechuate 4,5-dioxygenase family, protocatechuate 4,5-dioxygenase of *P. paucimobilis* SYK6 (LigB) (38), catechol 2,3-dioxygenase I of *Alcaligenes eutrophus* JMP222 (MpcI) (28), and 2,3-dihydroxyphenylpropionate 1,2-dioxygenase of *E. coli* (MhpB) (49), was successfully done (Fig. 6). The phylogenetic trees of these four meta-cleavage enzymes and other meta-cleavage enzymes were individually constructed (Fig. 7). As shown in Fig. 7, all meta-cleavage enzymes were classified to three families. The catechol 2,3-dioxygenase family constitutes most of the meta-cleavage enzymes involved in the degradation of several aromatic compounds. CarB2 is a member of this family. The meta-cleavage enzymes obtained from rhodococci (4, 5) and *Sphingomonas* sp. strain BN6 (23) consist of a short-chain dioxygenase family. CarBb was classified as a member of the protocatechuate 4,5-dioxygenase family by this phylogenetic tree.

The alignment of 12 meta-cleavage enzymes performed by Hofer et al. revealed that 18 amino acids were highly conserved (26). Recently, the three-dimensional structure of meta-cleavage enzymes (BphCs) from polychlorinated biphenyl-degrading pseudomonads were determined, and the locations of the active site, ferrous iron coordination system, and oxygen binding site were elucidated (20, 47). With respect to the protocatechuate 4,5-dioxygenase family, MpcI and MhpB show activities in a single protein (28, 49), and CarB and protocatechuate 4,5-dioxygenase (38) need the two subunits for meta-cleavage activity. As a result of the alignment of CarBb, LigB, MpcI, and MhpB (Fig. 6), the histidine residues, which may constitute a non-heme iron cofactor coordinating site (49), were found to be conserved (Fig. 6). The conserved His-12 of CarBb corresponds in position to His-146 of BphC in LB400, which is one of the iron(II) ligands (20). His-53 and His-117 of CarBb correspond in position to His-195 and His-241 of the LB400 enzymes, which are positioned close to the iron(II) cofactor (20). On the other hand, the substrate specificities of CarB and protocatechuate 4,5-dioxygenase are quite different. Protocatechuate 4,5-dioxygenase has no meta-cleavage activities for catechol, 4-methylcatechol, and so on, although the meta-cleavage activity for 2,3-dihydroxybiphenyl was not examined (58). However, CarB has meta-cleavage activities

FIG. 7. Phylogenetic trees of meta-cleavage enzymes in the catechol-2,3-dioxygenase family or short-chain dioxygenase family (A) and those in the protocatechuate 4,5-dioxygenase family (B). Names of bacterial strains (in parentheses) and GenBank database accession numbers are indicated after the enzyme names. The meta-cleavage enzymes in the short-chain dioxygenase family and MpcII were used as outgroups in the phylogenetic trees in panels A and B, respectively. The scale bar denotes 0.5 substitution per site. The trees were constructed by the neighbor-joining method, and bootstrap values above 50 from 100 resamplings are shown for each node. Groups: I, catechol 2,3-dioxygenase family; II, short-chain dioxygenase family; III, protocatechuate 4,5-dioxygenase family.

for 2,3-dihydroxybiphenyl and 4-methylcatechol (Table 2) but not protocatechuate 4,5-dioxygenase (data not shown). Given these observations, it is quite possible that the active site of CarB is located in the CarBb subunit. However, because there is little information on meta-cleavage enzymes of the protocatechuate 4,5-dioxygenase family, locations of the active site, oxygen binding site, and substrate recognition site of each of these meta-cleavage enzymes are still undetermined. The accumulation of information on meta-cleavage enzymes classified into this family and the determination of three-dimensional structures will help us to understand what structure of CarB is involved in the recognition of substrates and what amino acid residues are located in the active site of CarB.

The nucleotide sequence of *carB2* showed 50.6, 53.2, and 53.8% homologies at the DNA level with *nahC* of the NAH7 plasmid (22), *bphC1* of *Rhodococcus globerulus* P6 (4), and *bphC* of *Pseudomonas* sp. strain KKS102 (30), respectively. Because the genes encoding these previously cloned meta-cleavage enzymes are known to be clustered with the other degradative genes, we carried out sequence analysis of the upstream and downstream flanking regions of *carB2*; however, we could find no other genes exhibiting homologies with the other degradative genes. This result indicates the possibility that *carB2* does not form a cluster as is the case with *bphC2* and *bphC3* of *R. globerulus* P6 (4). Multiple genes encoding differ-

dioxigenase (14) and cumene dioxygenase (2, 12), suggesting that the pUCA1 insert contains genes encoding CARDO. Further sequence analysis of the upstream and downstream regions of carBC will be done to obtain information on the genes encoding CARDO.

---

**REFERENCES**

1. Ahmad, D., J. Fraser, M. Sylvestre, A. Larose, A. Khan, J. Bergeron, J. M. Juteau, and M. Sondossi. 1995. Sequence of the *bphD* gene encoding 2-hydroxy-6-oxo-(phenyl/chlorophenyl)hexa-2,4-dienoic acid (HOP/cPDA) hydrolase involved in the biphenyl/polychlorinated biphenyl degradation pathway in *Comamonas testosteroni*: evidence suggesting involvement of Ser<sup>112</sup> in catalytic activity. Gene **56**:69–74.
2. Aoki, H., T. Kimura, H. Habe, H. Yamane, T. Kodama, and T. Omori. 1996. Cloning, nucleotide sequence, and characterization of the genes encoding enzymes involved in the degradation of cumene to 2-hydroxy-6-oxo-7-methylocta-2,4-dienoic acid in *Pseudomonas fluorescens* IP01. J. Ferment. Bioeng. **81**:187–196.
3. Arcos, J. S., and M. F. Argus. 1968. Molecular geometry and carcinogenic activity of aromatic compounds. New perspectives. Adv. Cancer Res. **11**:305–471.
4. Asturias, J. A., L. D. Eltis, M. Prucha, and K. N. Timmis. 1994. Analysis of three 2,3-dihydroxybiphenyl 1,2-dioxygenases found in *Rhodococcus globerulus* P6. J. Biol. Chem. **269**:7807–7815.
5. Asturias, J. A., and K. N. Timmis. 1993. Three different 2,3-dihydroxybiphenyl-1,2-dioxygenase genes in the gram-positive polychlorobiphenyl-degrading bacterium *Rhodococcus globerulus* P6. J. Bacteriol. **175**:4631–4640.
6. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1990. Current protocols in molecular biology. John Wiley & Sons, Inc., New York, N.Y.
7. Birnboim, H. C., and J. Doly. 1979. A rapid alkaline extraction procedure for screening recombinant plasmid DNA. Nucleic Acids Res. **7**:1513–1523.
8. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. **72**:248–254.
9. Chemicals Inspection and Testing Institute Japan. 1992. Biodegradation and bioaccumulation: data of existing chemicals based on the CSCL, Japan. Japan Chemical Industry Ecology-Toxicology and Information Center, Tokyo, Japan.
10. Dagley, S. 1986. Biochemistry of aromatic hydrocarbon degradation in pseudomonads, p. 527–555. In J. R. Sokatch (ed.), The bacteria, vol. 10. The biology of *Pseudomonas*. Academic Press Inc., Orlando, Fla.
11. Denome, S. A., C. Oldfield, L. J. Nash, and K. D. Young. 1994. Characterization of the desulfurization genes from *Rhodococcus* sp. strain IGTS8. J. Bacteriol. **176**:6707–6716.
12. Eaton, R. W., and K. N. Timmis. 1986. Characterization of a plasmid-specific pathway for catabolism of isopropylbenzene in *Pseudomonas putida* RE204. J. Bacteriol. **168**:123–131.
13. Eltis, L. D., B. Hofmann, H.-J. Hecht, H. Lünsdorf, and K. N. Timmis. 1993. Purification and crystallization of 2,3-dihydroxybiphenyl 1,2-dioxygenase. J. Biol. Chem. **268**:2727–2732.
14. Ensley, B. D., B. J. Ratzkin, T. D. Osslund, M. J. Simon, L. P. Wackett, and D. T. Gibson. 1983. Expression of naphthalene oxidation genes in *Escherichia coli* results in the biosynthesis of indigo. Science **222**:167–169.
15. Felsenstein, J. 1985. Confidence limits on phylogenies: an approach using the bootstrap. Evolution **39**:783–791.
16. Furukawa, K., and T. Miyazaki. 1986. Cloning of a gene cluster encoding biphenyl and chlorobiphenyl degradation in *Pseudomonas pseudoalcaligenes*. J. Bacteriol. **166**:392–398.
17. Gieg, L. M., A. Otter, and P. M. Fedorak. 1996. Carbazole degradation by *Pseudomonas* sp. LD2: metabolic characteristics and the identification of some metabolites. Environ. Sci. Technol. **30**:575–585.
18. Grosser, R. J., D. Warshawsky, and J. R. Vestal. 1991. Indigenous and enhanced mineralization of pyrene, benzo[α]pyrene, and carbazole in soils. Appl. Environ. Microbiol. **57**:3462–3469.
19. Habe, H., K. Kasuga, H. Nojiri, H. Yamane, and T. Omori. 1996. Analysis of cumene (isopropylbenzene) degradation genes from *Pseudomonas fluorescens* IP01. Appl. Environ. Microbiol. **62**:4471–4477.
20. Han, S., L. D. Eltis, K. N. Timmis, S. W. Muchmore, and J. T. Bolin. 1995. Crystal structure of the biphenyl-cleaving extradiol dioxygenase from a PCB-degrading pseudomonad. Science **270**:976–980.
21. Hanahan, D. 1983. Studies on transformation of *Escherichia coli* with plasmids. J. Mol. Biol. **166**:557–580.
22. Harayama, S., and M. Rekik. 1989. Bacterial aromatic ring-cleavage enzymes are classified into two different gene families. J. Biol. Chem. **264**:15328–15333.
23. Heiss, G., A. Stolz, A. E. Kuhm, C. Müller, J. Klein, J. Altenbuchner, and H.-J. Knackmuss. 1995. Characterization of a 2,3-dihydroxybiphenyl dioxygenase from the naphthalenesulfonate-degrading bacterium strain BN6. J. Bacteriol. **177**:5865–5871.
24. Henikoff, S. 1984. Unidirectional digestion with exonuclease III creates tar-


The OCR process has been completed, and the content has been converted to Markdown format as requested. Let me know if you need further assistance!
VOL. 179, 1997                                                                 CLONING OF PSEUDOMONAS SP. STRAIN CA10 car GENES 4849

geted breakpoints for DNA sequencing. Gene 28:351–359.

25. Hisatsuka, K., and M. Sato. 1994. Microbiol transformation of carbazole to anthranilic acid by *Pseudomonas stutzeri*. Biosci. Biotechnol. Biochem. 58: 213–214.

26. Hofer, B., L. D. Eltis, D. N. Dowling, and K. N. Timmis. 1993. Genetic analysis of a *Pseudomonas* locus encoding a pathway for biphenyl/polychlorinated biphenyl degradation. Gene 130:47–55.

27. Horn, J., S. Harayama, and K. N. Timmis. 1991. DNA sequence determination of the TOL plasmid *xylGFJ* genes of *Pseudomonas putida*: implications for the evolution of aromatic metabolism. Mol. Microbiol. 5:2459–2474.

28. Kabisch, M., and P. Fortnagel. 1990. Nucleotide sequence of metapyrocatechase I (catechol 2,3-dioxygenase I) gene *mpcI* from *Alcaligenes eutrophus* JMP222. Nucleic Acids Res. 18:3405–3406.

29. Kayser, K. J., B. A. Bielaga-Jones, K. Jackowski, O. Odusan, and J. J. Kilbane II. 1993. Utilization of organosulphur compounds by axenic and mixed cultures of *Rhodococcus rhodochrous* IGTS8. J. Gen. Microbiol. 139: 3123–3129.

30. Kimbara, K., T. Hashimoto, M. Fukuda, T. Koana, M. Takagi, M. Oishi, and K. Yano. 1989. Cloning and sequencing of two tandem genes involved in degradation of 2,3-dihydroxybiphenyl to benzoic acid in the polychlorinated biphenyl-degrading soil bacterium *Pseudomonas* sp. strain KKS102. J. Bacteriol. 171:2740–2747.

31. Kimura, T., Y. Zhang, T. Kodama, and T. Omori. 1996. Isolation and characterization of Tn5-induced mutants deficient in carbazole catabolism. FEMS Microbiol. Lett. 135:65–70.

32. Kobayashi, T., R. Kurane, K. Nakajima, Y. Nakamura, K. Kirimura, and S. Usami. 1995. Isolation of bacteria degrading carbazole under microaerobic condition, *i.e.* nitrogen gas substituted conditions. Biosci. Biotechnol. Biochem. 59:932–933.

33. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) 227:680–685.

34. Maeda, M., S.-Y. Chung, E. Song, and T. Kudo. 1995. Multiple genes encoding 2,3-dihydroxybiphenyl 1,2-dioxygenase in the gram-positive polychlorinated biphenyl-degrading bacterium *Rhodococcus erythropolis* TA421, isolated from a termite ecosystem. Appl. Environ. Microbiol. 61:549–555.

35. Masai, E., A. Yamada, J. M. Healy, T. Hatta, K. Kimbara, M. Fukuda, and K. Yano. 1995. Characterization of biphenyl catabolic genes of gram-positive polychlorinated biphenyl degrader *Rhodococcus* sp. strain RHA1. Appl. Environ. Microbiol. 61:2079–2085.

36. Menn, F.-M., G. J. Zylstra, and D. T. Gibson. 1991. Location and sequence of the *todF* gene encoding 2-hydroxy-6-oxohepta-2,4-dienoate hydrolase in *Pseudomonas putida* F1. Gene 104:91–94.

37. Nestler, F. H. M. 1974. Characterization of wood-preserving coal-tar creosote by gas-liquid chromatography. Anal. Chem. 46:46–53.

38. Noda, Y., S. Nishikawa, K. Shiozuka, H. Kadokura, H. Nakajima, K. Yoda, Y. Katayama, N. Morohoshi, T. Haraguchi, and M. Yamasaki. 1990. Molecular cloning of the protocatechuate 4,5-dioxygenase genes of *Pseudomonas paucimobilis*. J. Bacteriol. 172:2704–2709.

39. Nordlund, I., and V. Shingler. 1990. Nucleotide sequences of the meta-cleavage pathway enzymes 2-hydroxymuconic semialdehyde dehydrogenase and 2-hydroxymuconic semialdehyde hydrolase from *Pseudomonas* CF600. Biochim. Biophys. Acta 1049:227–230.

40. Omori, T., L. Monna, Y. Saiki, and T. Kodama. 1992. Desulfurization of dibenzothiophene by *Corynebacterium* sp. strain SY1. Appl. Environ. Microbiol. 58:911–915.

41. Ouchiyama, N., Y. Zhang, T. Omori, and T. Kodama. 1993. Biodegradation

of carbazole by *Pseudomonas* spp. CA06 and CA10. Biosci. Biotechnol. Biochem. 57:455–460.

42. Resnick, S. M., D. S. Torok, and D. T. Gibson. 1993. Oxidation of carbazole to 3-hydroxycarbazole by naphthalene 1,2-dioxygenase and biphenyl 2,3-dioxygenase. FEMS Microbiol. Lett. 113:297–302.

43. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

44. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

45. Saitou, N., and M. Nei. 1987. The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol. Biol. Evol. 4:406–425.

46. Sato, S. I., J.-W. Nam, K. Kasuga, H. Nojiri, H. Yamane, and T. Omori. 1997. Identification and characterization of genes encoding carbazole 1,9a-dioxygenase in *Pseudomonas* sp. strain CA10. J. Bacteriol. 179:4850–4858.

47. Senda, T., K. Sugiyama, H. Narita, T. Yamamoto, K. Kimbara, M. Fukuda, M. Sato, K. Yano, and Y. Mitsui. 1996. Three-dimensional structures of free form and two substrate complexes of an extradiol ring-cleavage type dioxygenase, the BphC enzyme from *Pseudomonas* sp. strain KKS102. J. Mol. Biol. 255:735–752.

48. Shine, J., and L. Dalgarno. 1975. Determination of cistron specificity in bacterial ribosomes. Nature 254:34–38.

49. Spence, E. L., M. Kawamura, J. Sanvoisin, H. Braven, and T. D. H. Bugg. 1996. Catechol dioxygenase from *Escherichia coli* (MhpB) and *Alcaligenes eutrophus* (MpcI): sequence analysis and biochemical properties of a third family of extradiol dioxygenase. J. Bacteriol. 178:5249–5256.

50. Southern, E. M. 1975. Detection of specific sequences among DNA fragments separated by gel electrophoresis. J. Mol. Biol. 98:503–517.

51. Taira, K., J. Hirose, S. Harayama, and K. Furukawa. 1992. Analysis of *bph* operon from the polychlorinated biphenyl-degrading strain of *Pseudomonas pseudoalcaligenes* KF707. J. Biol. Chem. 267:4844–4853.

52. Tan, H.-M., H.-Y. Tang, C. L. Joannou, N. H. Abdel-Wahab, and J. R. Mason. 1993. The *Pseudomonas putida* ML2 plasmid-encoded genes for benzene dioxygenase are unusual in codon usage and low in G+C content. Gene 130:33–39.

53. Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673–4680.

54. van der Meer, J. R., W. M. de Vos, S. Harayama, and A. J. B. Zehnder. 1992. Molecular mechanisms of genetic adaptation to xenobiotic compounds. Microbiol. Rev. 56:677–694.

55. Vieira, J., and J. Messing. 1987. Production of single-stranded plasmid DNA. Methods Enzymol. 153:3–11.

56. Werlen, C., H. E. Kohler, and J. R. van der Meer. 1996. The broad substrate chlorobenzene dioxygenase and *cis*-chlorobenzene dihydrodiol dehydrogenase of *Pseudomonas* sp. strain P51 are linked evolutionarily to the enzymes for benzene and toluene degradation. J. Biol. Chem. 271:4009–4016.

57. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

58. Zabinski, R., E. Münnck, P. M. Champion, and J. M. Wood. 1972. Kinetic and Mössbauer studies on the mechanism of protocatechuic acid 4,5-oxygenase. Biochemistry 11:3212–3219.

59. Zylstra, G. J., and D. T. Gibson. 1989. Toluene degradation by *Pseudomonas putida* F1: nucleotide sequence of the *todC1C2BADE* genes and their expression in *Escherichia coli*. J. Biol. Chem. 264:14940–14946.
