
CARBOHYDRATES, LIPIDS, AND  
OTHER NATURAL PRODUCTS:  
Identification of the MNN2 and MNN5  
Mannosyltransferases Required for  
Forming and Extending the Mannose  
Branches of the Outer Chain Mannans of  
*Saccharomyces cerevisiae*

Julian C. Rayner and Sean Munro  
J. Biol. Chem. 1998, 273:26836-26843.  
doi: 10.1074/jbc.273.41.26836

Access the most updated version of this article at [http://www.jbc.org/content/273/41/26836](http://www.jbc.org/content/273/41/26836)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 56 references, 35 of which can be accessed free at [http://www.jbc.org/content/273/41/26836.full.html#ref-list-1](http://www.jbc.org/content/273/41/26836.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1998 by The American Society for Biochemistry and Molecular Biology, Inc.

**Identification of the MNN2 and MNN5 Mannosyltransferases Required for Forming and Extending the Mannose Branches of Saccharomyces cerevisiae**

(Received for publication, July 10, 1998)

Julian C. Rayner and Sean Munro‡

*From the Medical Research Council Laboratory of Molecular Biology, Hills Road, Cambridge CB2 2QH, United Kingdom*

The mannan structure found on the N-linked glycans of the yeast *Saccharomyces cerevisiae* is composed of a long backbone of α-1,6-linked mannose to which are attached branches consisting of two α-1,2-linked mannoses followed by an α-1,3-linked mannose. In the mutants *mnn2* and *mnn5*, the addition of the first and second of these two mannoses, respectively, is defective. In this paper, we report the identification of the genes corresponding to these mutations. The two genes encode closely related proteins with distant homology to the known Mnn1p α-1,3-mannosyltransferase. We show that these proteins are localized in an early compartment of the yeast Golgi and that they are not physically associated with each other or with the two protein complexes known to be involved in synthesizing the α-1,6-linked backbone. The identification of Mnn2p and Mnn5p allows us to assign Golgi proteins to all of the catalytic steps in *S. cerevisiae* mannan synthesis.

yeast and fungi (9–13). The backbone is usually α-1,6-linked mannose, and the first branch is usually an α-1,2-linked mannose, but the rest of the side chain varies greatly between species. Mannoses of varying number can be present in a variety of linkages, and other sugars can be incorporated including N-acetylglucosamine and galactose. This variation of the side chain may reflect evolutionary pressures to evade cell wall-digesting hydrolases, immune responses, and sugar-binding killer toxins from other yeasts.

As well as having its own biological relevance, the synthesis of yeast mannan provides as a model system to study how cells make the large and complex carbohydrate structures found in many other species such as the glycosaminylglycans of animals and the hemicelluloses and pectins of plants. Like yeast mannans, these molecules have a long backbone carrying diverse modifications and are synthesized in the secretory pathway rather than being directly extruded through the plasma membrane. Understanding of the mechanism of *S. cerevisiae* mannan synthesis has come from both genetic and biochemical studies. Conditional mutations that block the yeast secretory pathway have allowed the delineation of the processing steps into the different compartments of the pathway (14, 15). In addition, several genetic screens have identified yeast mutations that cause defects in the synthesis of mannan including the *mnn* (mannan), *och* (outer chain), *ngd* (*N*-glycosylation defective), and *ldb* (low dye binding) sets of mutants (16–19). Finally, it has been possible to identify mannosyltransferase activities in yeast extracts and, in some cases, correlate these to the products of genes identified in the above screens (20, 21). From these studies, a basic pathway has emerged. The core structure of the N-linked glycan is attached to nascent proteins in the ER¹ and, as in mammals, is trimmed to Man₈GlcNAc₂ before the proteins leave the ER (22). Upon arrival in the Golgi, a single α-1,6-linked mannose is initially added to all of the core structures by the Och1p transferase (17, 23). Then, on a subset of proteins, this mannose is extended to a long α-1,6-mannose backbone by two complexes of proteins. The first contains Mnn9p and Van1p, and the second contains Mnn9p, Anplp, Hoc1p, and Mnn11p (24). At present, it is not clear if all these proteins are α-1,6-mannosyltransferases, but both complexes possess such activity. What causes the Mnn9p-Van1p complex to only initiate this backbone on a subset of yeast glycoproteins is not understood, but it presumably reflects recognition of some feature of the proteins themselves.

Of the enzymes involved in synthesizing the mannose branches, that responsible for adding the final α-1,3-linked mannose has been identified as the product of the *MNN1* gene (20, 25, 26). Some of the branches also receive phosphomannan

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ To whom correspondence should be addressed. Tel.: 44-1223-402236; Fax: 44-1223-412142; E-mail: sean@mrc-lmb.cam.ac.uk.

1 The abbreviations used are: ER, endoplasmic reticulum; ORF, open reading frame; HA, hemagglutinin; TEV, tobacco etch virus; Endo H, endoglycosidase H; PKC, protein kinase C.

This paper is available online at http://www.jbc.org

groups, and there is evidence that the *MNN6* gene product is responsible for this addition (27). In contrast, the enzymes responsible for attaching the two first α-1,2-linked mannoses of the branches are at present unknown. However, two of the *mnn* mutations specifically affect these steps. Thus, the *mnn2* mutant is defective in the addition of the first α-1,2-linked mannose as it has no branches on the backbone (Fig. 1A). Likewise, the *mnn5* mutant is defective in the addition of the second mannose as the branches on its mannan consist of a single α-1,2-linked mannose. In this paper, we report the identification of the *MNN2* and *MNN5* genes and show that they encode proteins that are related to the Mnn1p mannosyltransferase and that are located in the early Golgi of yeast.

### EXPERIMENTAL PROCEDURES

#### Yeast Strains and Plasmids—Standard protocols were used for the growth and mating of yeast strains, transformation by the lithium acetate method, and preparation of genomic DNA and alcian blue staining (16, 28). The wild-type strain used was SEY6210 (*Matα ura3–52 his3Δ200 leu2–3, 112 trp–Δ901 lys2–801 suc2–Δ9*) (29). *Mnn* strains LB1–3B (*MATα mnn2–1*) and LB65–5D (*MATα, mnn5*) and parental strain X2180 were obtained from the ATCC (30, 31). YBR015c and YJL186w were either disrupted by replacing the entire ORF, or epitope tagged precisely at the C terminus using homologous recombination with appropriate polymerase chain reaction products containing the *Schizosaccharomyces pombe HIS5* or *Kluyveromyces lactis URA3* genes; polymerase chain reaction was used to check the integrants (32, 33). Epitope-tagging integrations added the amino acid sequence GAGAG followed by either three copies of the hemagglutinin (HA) tag or nine copies of the Myc tag. For COOH-terminal protein A fusions, polymerase chain reaction products were amplified from plasmid pZZ-HIS5, which contains the *HIS5* marker and encodes a linker sequence starting GAGAGASENLYFQGAHDEAVDNK…, where SENLYFQ/G is the TEV protease cleavage site (34) and VDNK is the start of the first of two synthetic protein A “Z” domains (35). Myc-tagged invertase, Mnn1p, Mnt1p were expressed in strains using an integration plasmid or CEN plasmids, respectively (26, 36).

#### Mannosyltransferase Assays—Mannosyltransferase assays on beads were essentially a modified version of that used by Nakajima and Ballou, as described previously (24, 40). Briefly, protein A fusions precipitated on IgG-Sepharose beads were washed once with T/D/I buffer (4 °C, 15 min) and once briefly with buffer II (50 mM HEPES, pH 7.2, 0.4% digitonin, 10 mM MnCl₂). Then, 50 μl of buffer II was added to the beads, supplemented by 0.6 mM GDP-β-mannose, 10 mM α-1,6-D-mannobiose (Dextra Labs), 62 nCi GDP-β-[U-¹⁴C]mannose (Amersham Pharmacia Biotech), and the mixture was gently agitated for 2 h at 30 °C. For reactions lacking Mn²⁺, MnCl₂ was also omitted from the buffer II wash. Reactions were terminated by applying the supernatant of the mixture to a 0.5 × 5-cm Dowex 1-X8 column, eluting the neutral reaction products with 800 μl of water, and their radioactivity was quantified by liquid scintillation counting.

#### Indirect Immunofluorescence and Immunoblotting—The Myc epitope tag was detected with mouse monoclonal 9E10; the HA tag was detected with mouse monoclonal 12CA5 or rat monoclonal 3F10 (Boehringer Mannheim). In addition, rabbit antisera against both tags (Santa Cruz), Anp1p (24) and α-1,6 mannose (37), were used for protein blotting and immunofluorescence. Total yeast proteins were prepared by resuspending cells at 1 A₆₀₀ unit per 20 μl of SDS sample buffer, vortexing with glass beads (425–600 μm, Sigma) for 5 min at 4 °C, and denaturing for 5 min at 65 °C. For immunoblotting, proteins were separated by SDS-polyacrylamide gel electrophoresis and then electroblotted onto nitrocellulose (0.45 μm, Schleicher & Schuell) and probed with antibodies in phosphate-buffered saline, 2% dried milk. Peroxidase-conjugated anti-rabbit and anti-mouse secondary antibodies were detected by chemiluminescence (ECL, Amersham Pharmacia Biotech). Indirect immunofluorescence was performed as described previously (38) using fluorescein isothiocyanate-conjugated donkey anti-rat Ig (Southern Biotech., 1/50) and Cy3-conjugated goat anti-rabbit IgG (Amersham Pharmacia Biotech, 1/700) and a Bio-Rad MRC-600 confocal laser-scanning microscope.

#### Immunoprecipitation and Protease Release—Yeast cell growth was stopped in log phase by addition of 10 mM sodium azide, and 50 A₆₀₀ units of cells were resuspended in 5 ml of 100 mM Tris-HCl, 10 mM dithiothreitol, pH 9.4, 10 mM sodium azide and incubated at 30 °C for 10 min. Cells were then resuspended in 1 ml of spheroplast solution (1 M sorbitol, 50 mM Tris-HCl, pH 7.4, 2 mM MgCl₂, 10 mM dithiothreitol, 10 mM sodium azide, 100 units of zymolyase (20T, ICN)) and incubated at 30 °C for 30 min, or until spheroplasting was complete. Spheroplasts were then washed in spheroplast solution without zymolyase at 4 °C and lysed at 4 °C for 30 min by gentle rolling in 1 ml of T/D/I buffer (10 mM triethanolamine, pH 7.5, 150 mM NaCl, 1% digitonin, 2 mM EDTA supplemented by protease inhibitors). Cell debris was removed by centrifugation at 4 °C (10 min at 14,000 × g), and the supernatant was precleared by incubation at 4 °C for 1 h with 10 μl of protein A-Sepharose (Amersham Pharmacia Biotech) for 12CA5 immunoprecipitations or protein G-Sepharose (Sigma) for 9E10. Antibody was then added (12CA5, 3.8 μg/ml or 9E10 2.7 μg/ml) followed after 2–4 h by 12.5 μl of protein A- or protein G-Sepharose; complexes were precipitated by gentle agitation at 4 °C overnight. The beads were then washed in T/D/I buffer and either eluted with SDS sample buffer at 65 °C for 5 min or used directly for mannosyltransferase assays.

For preparative scale isolation of protein A fusions, spheroplasting and lysis was carried out as above using 1000 A₆₀₀ units of cells with the volumes of buffers scaled up accordingly. After lysis and subsequent pelleting of cell debris at 14,000 × g, the supernatant was removed, and 25 μl of IgG-Sepharose Fast Flow (Amersham Pharmacia Biotech) was added. After binding overnight at 4 °C with gentle rocking, the beads were washed at 4 °C (four times in 1 h in 1 ml of T/D/I buffer, with gentle agitation). The beads were then resuspended in 50 μl of T/D/I buffer containing 10 units of TEV protease (Life Technologies, Inc.) and rocked overnight at 4 °C. Cleaved product was separated from the beads by pelleting the beads in a microfuge and removing the supernatant. This step was repeated several times to ensure no beads were carried over. For Endo H treatment, the samples were diluted to 1 × SDS sample buffer and heated at 65 °C for 5 min. The denatured samples were then diluted 5-fold and made 50 mM phosphate-citrate (pH 5.9), 0.1% Nonidet P-40. 500 units of Endo H (New England Biolabs) was added, and the mixture was incubated at 37 °C for 1 h. Protein was then precipitated by methanol/chloroform extraction, resuspended in SDS buffer by sonication, and separated by SDS-polyacrylamide gel electrophoresis. After Coomassie staining, gel slices were excised and digested with trypsin, and the tryptic fragment masses were determined by matrix-assisted laser desorption ionization mass spectrometry using a Perceptive Voyager mass spectrometer (39).

### RESULTS

#### Identification of Candidate Genes for *MNN2* and *MNN5*—Neither the *MNN2* gene nor the *MNN5* gene has been correlated to an ORF in the sequence of the complete yeast genome. However, the *mnn2* locus was mapped to chromosome II, between the centromere and *gal1* (41). The nucleotide sequence of this section of chromosome II reveals 19 ORFs, and we examined them for homology to known glycosyltransferases. Only ORF YBR015c has such a homology, a distant relationship to Mnn1p, an α-1,3-mannosyltransferase of the yeast Golgi apparatus (Fig. 1B). Like Mnn1p, the protein encoded by YBR015c has a single amino-terminal transmembrane domain and a predicted type II orientation, as seen for all known Golgi glycosyltransferases in both mammals and yeast. Indeed, when this ORF was sequenced as part of the genome project, it was also given the description *TTP1* for type II protein (42). Mnn1p has three close homologues in the yeast genome of unknown function, and searching the data bases with the sequence of YBR015c revealed homology to these, but also to a second ORF, YJL186w, that was much closer to YBR015c than the Mnn1p family. In addition, there is an ORF in *Pichia pastoris* that is closer to YBR015c and YJL186w than to *MNN1* (Fig. 1B). Finally, it has been observed recently that the activity of Mnn1p requires an aspartate-containing motif (26). This motif is found to be conserved in many families of glycosyltransferases, and it is also conserved in YBR015c and YJL186w (Fig. 1B). These observations suggested that YBR015c was a good candidate to be *MNN2*, and as *mnn2* and *mnn5* both affect the addition of an α-1,2-linked mannose, it seemed possible that the closely related ORF YJL186w might be *MNN5*.

#### Deletion of YBR015c and YJL186w Results in Defects in Glycosylation—To examine the possibility that the ORFs YBR015c and YJL186w correspond to *MNN2* and *MNN5*, we

26838 Components of Outer Chain Branching in Yeast

A

M →² M  
↓⁶  
M →² M  
↓⁶  
M →³ M →² M  
↓⁶  
M →³ M →² M →² M  
↓⁶  
M →³ M → (P)  

wild-type  

M →² M  
↓⁶  
M →² M →² M  
↓⁶  
M →² M →² M  
↓⁶  
M →² M →² M  
↓⁶  
M →² M → (P)  

mnn1  

M →² M  
↓⁶  
M →² M  
↓⁶  
M →² M  
↓⁶  
M →² M  
↓⁶  
M →² M  

mnn5  

M  
↓⁶  
M  
↓⁶  
M  
↓⁶  
M  
↓⁶  
M  

mnn2  

B

Mnn1p  
YBR015  
YJL186  
Pp ORF  

ML LTKRFSKLFKLTFIVLI LCGLFVITNKYMDENTSVKEYKE  
ML IRLKKRKILQVIVSAVVLL-LFFCS--VHNDVSSSWLYGK  
MFNSLAPMRLKKLLKVFCASVVLAAATSVVLLFFHFGGQIIPIPE  

YDRCFMCANPVN----------IQEIFPKSDKMSKERVQSKLIKTLLNA  
YLDRYVQS-YSNKYSSSSDAASADDSTPLRDNEAGNEKLKSFYNNVNFN  
KLRLLPVLTRSN----------LKNN----------FYTTLVQA  
RTVTLSPT-PAN----------DTWQFQQ-------FFNGYLDA  

227 TFPNYDPDNFKKYDQFEFHKMFPFI NNFTTETF-HEMVP--KITSPFGK  
92 LMVD-SPKGST---AKQYN-EACLLKGDIGDRPD-HYKDLYKLSAKELSK  
63 IVEN-KPADSSFDLSKLHGAEBCSFANNVAAHDSGHSDS---LSYESLSK  
72 LLEN-NLSYPIP-ERWNHEVTNVRFFNRIGELLS----------ESRLQE  

274 VLEQGFLPKFDHKTGKVQEYFKYEYDPSKTFWANWRDMSAKVAGR-GIVL  
136 CLE--LSPDEVASLTKSHKDYVEHIA---T--LVSPK--GTYKGS-GIAT  
109 CYN--LNKTVQESLREVKSKFTDTLSGKLN--FSIPQREALFSGSEGIYT  
110 LIH--FSPEFIEDTSDKFDNIVEQIP-----AKWPY-ENMYRGD-GYVI  

323 SLGSNQFPLAVKFIASLRFEGNTLPICVVYR-GDELSQLVDKLIYAARS  
176 VGGGKFSLMALILIKTIRNMGTTLTPVEVLIPPGDEGETEFCNKIL----  
155 IGGGKYSVALAYTMKKLRDTGTTLPIEVIIIPPODEGEDDFCKNW------  
150 VGGGRHTFLALLNLNALRRAGNKLPEEVVLPYDDYEEEDFOENHF------  

372 PDFKPVENNYDNSTNPQEIWFLDVSNTHPKWRGDGSKSKWLVVLLN  
221 ----PKYN--------SKCIVYSDILLP-RETIEKFVFKGYQFKSLALIAS  
200 ----PKFN--------GKCIVFSDIVP--SKPLSDLKLTHFQLKVFGLIIS  
195 ----PLLN--------ARGVILEERFG-DQVYPRLLQLGGYQFKIFAIAAS  

422 LLQEFVFLDIDAI SYEKIDNYFKTTEYQKTGTVFYRERALRENVNERCI A  
258 SFENLLL LDADNFPIKPLDNI FNEEPYVSTGLVMWPDFWARRTTTH---PL  
237 SFKRIIF LDADNYAVKNLDLAFNTTSFNDTGLILWPDFWRRVTP---PA  
232 SFKNCFL LDSDNIPLRKMDKI FSSELYKNKTMITWPDFWLRSTS---PH  

472 RYETLLPRNLESKNFQN-SLLIDP------DHA------LNECDNTLT-TEE  
304 YYDIAGIAVDKKKRVRNSRDDITP------PAV------YTKDLKD--LSD  
283 FYNIIGSSI NIKRVRFVSDDISPVSRYDPFVSNsndYT PKERQEHFLKH  
278 YYHNITKTPIGDKRVRYNDFYTN------PNE------YYYYDED P...  

511 YIFKAFFHHRRQHOLEAGLFAVDKSKHTI PLVLAAMIHLAKNTAHCTHGD  
341 VPLSDLDGTTIPDVSTESGQLMNKTKHLATALLSLFYNVNGPTWYYPIFS  
333 VPLHDLDGTMPDLSESSEQOMVIDKIRHFNTLLLLALYYNVYGPTWYYKMI S  

560 KENFWLGFLAAGHTYALQGVYSGAIGDYVKKTDLNG-KRQEAAVEICSGQ  
391 QKAAGEGDKET--FIAAANFYGLSFYQVRTRTGVEG-YHDEDGFGHVAM  
383 QGTAGEGDKDT--FFAAAHALNMPYYQVRTNFEFDGFFYQKDDYKGLA-L  

609 IAHMSTDKKTLLLWVNGGGTFCRHDN-AAKDDWKKDGFKKFKDQFKTFEE  
437 LQHDFVQDYGRYLNAMESIGNKYGGTKSADAIFKFDKNYSLEK-YTEEFFD  
430 LQHDFEQDYKQYQKAQQKV---KAN---IEEFSKLDPDYTLDNGFLKTLMV  

658 MEKYYYITPISSKYVILPDPKSDDWHRASAG---ACGG---YIWCATHKT  
486 NEDLNANKNHVDVMFIHSNFPKFDPYDLSKSN-FLT TNGKPARSYTA LKKV  
475 NDDG---SDLDIMFIHASFYKADPWTLYHENRFIGPNGEQVRGF RKP HRY  

702 L LKPYSYNHRTTHGELITLDEE------QRLHI DAVN---TVWSHANKDNT  
535 KNYDI EL ENFKVLNEYVCVNK----PFKYLDDL LGQDKTEWKRVCGYIT  
522 G-MDFELFLFNDMRGSFCTTPKSQVIKFKYFTDKVN---TPEWDAMECYLT  

744 RSFTEEEIKELNSRHEQS*  
581 DRLAFLESTHDKAIAGK*  
569 NHVNYLESTHK EAMGEKN*  

FIG. 1. Open reading frames of unknown function related to *MNN1* potentially correspond to the *MNN2* and *MNN5* genes. A, structures of the mannans from wild-type, *mnn1*, *mnn2*, and *mnn5* yeast strains. The backbone extends for 30–50 residues, and representative side chains are shown; the exact order and frequency of these in wild type is unknown. B, the protein encoded by ORF YBR015c from the *mnn2* region is related to the Mnn1p mannosyltransferase. The predicted amino acid sequence of YBR015c was aligned with that of *MNN1* using CLUSTAL. Also shown are the proteins encoded by YJL186w (an ORF closely related to YBR015c) and by a partial ORF from *P. pastoris* (Pp ORF, Swissprot 92264). Residues identical (*black*) or related (*gray*) in at least two of the sequences are shaded. The putative TMDs of the proteins are underlined (that of Mnn1p is within the first 174 residues,

A

B

WT  
ΔYBR015  
ΔYJL186  

X2180  
mnn2/WT  
mnn2/ΔYBR015  
mnn2/ΔYJL186  

α-1,6-mannose invertase  

C

haploids diploids  

WT  
mnn2/WT  

ΔYBR015  
mnn2/ΔYBR015  

ΔYJL186  
mnn2/ΔYJL186  

X2180  
mnn5/WT  

mnn2  
mnn5/ΔYBR015  

mnn5  
mnn5/ΔYJL186  

FIG. 2. Deletion of ORFs YBR015c and YJL186w results in phenotypes similar to *mnn2* and *mnn5* and failure to complement these mutations. A, immunoblots of total proteins from either 6210 (WT) or from derivatives in which either YBR015c or YJL186w was deleted, were probed with antibodies to α-1,6-mannose or to the Myc-epitope to detect a Myc-tagged version of invertase. B, anti α-1,6-mannose immunoblots of total proteins from yeast strains *mnn2*, its parent X2180, or diploids from matings between *mnn2* and the 6210-derived haploid strains in A. C, alcian blue-stained whole cells from the strains in A and B and also the analogous *mnn5* haploids and diploids. Equal numbers of stained cells were filtered onto nitrocellulose, washed, and imaged.

made disruptants of the genes in a haploid background and examined the resulting phenotypes. Both deletion strains were viable with no obvious growth defect. One characteristic of the *mnn2*, but not the *mnn5*, mutation is that the loss of branches on the α-1,6-linked backbone results in greatly increased binding by anti-α-1,6-mannose antisera (31). Fig. 2A shows that when total proteins from the stains are separated on a protein gel, blotted, and probed with an anti-α-1,6 antisera, greatly increased antibody binding is seen with the ΔYBR015c strain. We also examined the mobility of the protein invertase in the deletion strains. The N-linked glycans of invertase are modified with hypermannose structures, and the invertase in the ΔYBR015c strain showed a slightly increased mobility on an SDS gel, consistent with a defect in the N-linked mannan (Fig. 2A). Both *mnn2* and *mnn5* strains are known to show loss of binding of the cationic dye alcian blue (31, 43). This is due to an absence of phosphate in the mannose structure, apparently which is not shown). The conserved DXD motif found in many families of glycosyltransferases is marked with black circles (26).

caused by the branches being too short for recognition by the phosphomannose-transferase. Fig. 2C shows that the staining of cells by alcian blue is greatly reduced by deletion of either YBR015c or YJL186w. Thus, deletion of YBR015c results in an exposed α-1,6 backbone and a reduction in cell wall phosphate, whereas deletion of YJL186w causes only the latter defect, the same phenotypes seen with *mnn2* and *mnn5*, respectively.

YBR015c and YJL186w Are the Genes MNN2 and MNN5, Respectively—To demonstrate that YBR015c and YJL186w correspond to MNN2 and MNN5, complementation tests were carried out. The deletion strains were mated with *mnn2* and *mnn5* strains, and the resulting diploids were examined for glycosylation defects. Fig. 2 shows that mating the mutant strains to wild type results in restoration of normal glycosylation as expected from the recessive nature of the *mnn2* and *mnn5* mutations. In contrast, diploids from the mating of *mnn2* with ΔYBR015c have not recovered the low levels of anti-α-1,6 binding or high levels of alcian blue binding, indicating a failure of the YBR015c to complement the *mnn2* mutation (Fig. 2, B and C). Similarly, diploids from the mating of *mnn5* with ΔYJL186c do not recover high levels of alcian blue binding (Fig. 2C). These specific losses of complementation indicate that YBR015c is allelic with MNN2 and YJL186w with MNN5. From now on, we will refer to these ORFs by their original MNN gene names.

The Gene Products of MNN2 and MNN5 Are Glycoproteins of the Golgi Apparatus—To further characterize the proteins encoded by MNN2 and MNN5, triple HA epitope tags were directly inserted at the COOH terminus of the genes in the genome by homologous recombination. Immunoblots of total cellular proteins revealed that the two proteins were expressed at comparable levels (Fig. 3A). Mnn2p ran as two closely migrating bands, the upper of which disappeared upon treatment with Endo H, indicating partial use of one of three potential sites for N-linked glycosylation. Mnn5p ran as a single band that shifted c6kD upon Endo H treatment, suggesting use of at least three of the four potential glycosylation sites. After Endo H treatment, a minor slower migrating band was also visible, suggesting a small degree of modification by O-linked glycosylation. The apparent molecular weights of deglycosylated Mnn2p-HA and Mnn5p-HA (72 and 69 kDa) correspond closely to their predicted sizes (71.8 and 71.3 kDa, respectively).

Examination of the intracellular localization of Mnn2p and Mnn5p by immunofluorescence showed that both proteins were distributed in a punctate pattern characteristic of yeast Golgi (Fig. 3B). Double label immunofluorescence showed that for both proteins, these spots showed substantial colocalization with the cis Golgi protein Anp1p, but not with the medial Golgi proteins Mnn1p and Mnt1p (Fig. 3B and data not shown). The distribution of Mnn2p and Mnn5p were compared by tagging them with different epitope tags, and a high degree of colocalization was observed. The distribution of Anp1p and of Mnn2p or Mnn5p was never absolutely identical, with occasional spots being positive for only one or the other marker. A similar situation has been observed for other yeast Golgi proteins in both the cis and medial compartments, and this presumably reflects the rapid recycling and movement of resident proteins within the Golgi stack (24, 44–46).

Investigation of Possible Physical Interactions between Mnn2p and Mnn5p and Other Golgi Proteins—The Golgi proteins involved in synthesizing the α-1,6-linked mannan backbone are found in two multi-protein complexes, which share Mnn9p as a common component (24). Since Mnn2p and Mnn5p are apparently required for the next steps in mannan synthesis, we were interested to determine if these proteins are associated with either of these Mnn9p-containing complexes or,

FIG. 3. Mnn2p and Mnn5p are glycoproteins of the cis Golgi of yeast. A, immunoblot of total protein from 6210 (-) or from strains in which the endogenous MNN2 or MNN5 genes were tagged at the COOH terminus with three copies of the HA-epitope, proteins being treated with or without Endo H as indicated. B, double label immunofluorescent confocal micrographs of yeast expressing triple HA-tagged Mnn2p, stained for HA and either endogenous Anp1p, or the tag on Mnn1p-myc or on Mnn5p-myc as indicated. For Mnn2p versus Anp1p and for Mnn2p versus Mnn5p, the majority of Golgi structures were positive for both proteins, although a few spots were positive for only one, as indicated by arrow heads. Comparison of Mnn5p with Anp1p or Mnn1p gave results indistinguishable from those seen with Mnn2p (data not shown).

indeed, with any other Golgi proteins. Thus, the epitope-tagged Mnn2p and Mnn5p were immunoprecipitated from cells lysed with the mild detergent digitonin, and the immunoprecipitates were blotted for the presence of known Golgi enzymes. Fig. 4A shows that precipitation of Mnn2p or of Mnn5p did not result in the coprecipitation of either Van1p or Anp1p, components of the two complexes that synthesize the α-1,6-linked backbone. Likewise, neither Mnn2p nor Mnn5p showed association with either Mnn1p, which adds the α-1,3-linked mannose to the α-1,2-linked mannose that is missing in the *mnn5* mutant, or with Mnt1p, an α-1,2-mannosyltransferase that modifies O-

26840 Components of Outer Chain Branching in Yeast

A
Anti-Van1p
WT Mnn9p-HA Mnn2p-HA Mnn5p-HA
-97
-70
-44

Anti-Anp1p
WT Mnn11p-HA Mnn2p-HA Mnn5p-HA
-70
-44

B
ip ab H H H M M H M
H-tag 2 2 5 5 2 2 2
M-tag 1 T 1 T 1 T 5 5

Anti-M-tag

Anti-H-tag

FIG. 4. Mnn2p or Mnn5p do not interact with known Golgi enzymes involved in mannan synthesis. A, immunoblots of proteins isolated by anti-HA immunoprecipitation from strains in which the triple HA epitope was inserted at the COOH terminus of the ORFs for the indicated proteins. The blots were probed with antisera against Van1p or Anp1p as shown. B, immunoblots of proteins isolated by anti-epitope tag immunoprecipitation from strains coexpressing two proteins tagged with the Myc (M) or the HA (H) epitopes. The tagged proteins were Mnn2p (2), Mnn5p (5), Mnn1p (1), and Mnt1p (T), the precipitations were performed with mouse monoclonals, and then the blots were probed with rabbit anti-tag sera as shown.

linked glycans. Finally, examination of a strain expressing epitope-tagged forms of both Mnn2p and Mnn5p showed that when one is precipitated, the other does not coprecipitate (Fig. 4B). Thus, it seems that neither Mnn2p nor Mnn5p associates with other Golgi enzymes involved in mannan synthesis.

To investigate whether these proteins have any binding partners, a different system for precipitation was used that gives low background even with mild detergent conditions. To achieve this, a protein A tag was inserted at the COOH terminus of both MNN2 and MNN5 (Fig. 5A). The linker between the ORF and the protein A contains a cleavage site for the sequence-specific protease from TEV (34). Thus, the protein A fusions can be precipitated from digitonin-solubilized cells on IgG-Sepharose, washed, and then eluted with TEV protease under non-denaturing conditions. This elution method releases the tagged protein and any binding partners while leaving behind the proteins stuck nonspecifically to the beads, thereby minimizing the background. Fig. 5B shows the results of such a precipitation performed with protein A-tagged Mnn2p and Mnn5p. When the eluted proteins were separated by SDS-

A
N protein-A
cleavage repeats
site (TEV)
protein
Z Z

B
control
Mnn2p-ZZ
Mnn5p-ZZ
97-
70-
44-
28-

FIG. 5. Mnn2p and Mnn5p do not form a stoichiometric association with any other protein. A, structure of the protein A fusion chimeras. The open reading frame is extended by a linker containing the cleavage site for TEV protease and then two copies of a synthetic "Z" domain from protein A. B, Coomassie-stained gel of the proteins eluted by TEV protease from IgG-Sepharose beads incubated in digitonin lysates of the indicated strains ("control" is 6210 containing no fusion protein). The proteins were digested with Endo H following elution, and each lane represents the product of 1000 A<sub>600</sub> units of cells. The 30-kDa band in all lanes is the protease, and the 90-kDa band from wild-type 6210 is a sporadically observed background band.

polyacrylamide gel electrophoresis and stained with Coomassie Blue, a single predominant band was observed in both cases, suggesting that neither protein is stoichiometrically associated with a second component. For both proteins, an additional faint band could be seen that was specific to the precipitated protein (Fig. 5B). To identify these bands and to confirm the identity of the major band, they were excised from the gel and digested in the gel slice with trypsin, and the tryptic peptides were analyzed by mass spectrometry. The masses of the tryptic fragments confirmed that the major bands were indeed Mnn2p and Mnn5p. In addition, the minor band migrating ahead of Mnn2p produced tryptic peptides characteristic of Mnn2p, suggesting that it is a proteolytic product from Mnn2p that has probably arisen during precipitation, as it is not seen in the blot of total protein from yeast expressing tagged Mnn2p (Fig. 4A). The minor band migrating behind Mnn5p also produced fragments characteristic of Mnn5p, and this presumably corresponds to the minor band seen in this position in the total protein blot in Fig. 4A.

Components of Outer Chain Branching in Yeast

plest hypothesis for the function of Mnn2p and Mnn5p is that they are the two α-1,2-mannosyltransferases that act sequentially to form, and then extend, the branches on the mannan backbone.

S. cerevisiae contains a second family of well characterized α-1,2-mannosyltransferases, the closely related Golgi proteins Mnt1p, Ktr1p, and Ktr3p that are required for attaching the second and third mannoses to O-linked glycans (47). Although the residues they add are both α-1,2-linked mannoses, these enzymes show no sequence homology to Mnn2p or Mnn5p. It has been reported that when *MNT1*, *KTR1*, and *KTR3* are all deleted, not only is extension beyond the first O-linked mannose absent, but in addition a small increase in the mobility of invertase is observed, suggesting that N-linked structures are also affected (48). It is possible that extension of the mannan branches can also be performed to some extent by the Mnt1p family, either contributing to the addition of the second mannose or possibly adding a third α-1,2-linked mannose, as has been reported for branches in other yeasts (9). Alternatively, the effect could be indirect if some of the mannan-synthesizing enzymes require full-length O-linked chains to be fully effective in the Golgi. α-1,2-Linked mannose is also found on GPI anchors, but this is unaffected in *mnn2* and *mnn5* mutants (49).

The identification of the Mnn2p and Mnn5p mannosyltransferases means that it is now possible to assign Golgi enzymes to all of the catalytic steps in the formation of the S. cerevisiae mannan structure (Fig. 7). The initial extension and subsequent elongation of the backbone are catalyzed by two multiprotein complexes, but these complexes do not contain the enzyme acting before this step, Och1p, nor, as shown here, the subsequent transferases Mnn2p and Mnn5p. Likewise, Mnn2p and Mnn5p do not seem to associate with each other or with the Mnn1p enzyme that catalyzes the next step in elaboration of the mannan branch, acting on the two α-1,2-linked mannoses whose addition requires first *MNN2* and then *MNN5*. Thus, it seems that the enzymes involved in branch formation act separately and not as a complex. It is perhaps noteworthy that the three proteins that are required to make the basic branch, Mnn2p, Mnn5p, and Mnn1p, share sequence homology. This suggests that these enzymes may have evolved from a common ancestor that made a simple branch structure, and subsequent evolutionary selection for branch variation has acted on duplicated genes to produce a diversity of transferases. A requirement for continuous variation of the branch-forming enzymes during evolution may have prevented them forming a larger complex of the type seen for the more conserved α-1,6-linked backbone.

There are several reasons why these branch-forming enzymes in S. cerevisiae may be of wider interest. First, the mannan branches are the structures that are recognized during immune responses against pathogenic yeast. This, combined with the possible involvement in adhesion during infection, means that these enzymes are potential targets for antifungal drugs (8). Second, flocculation of S. cerevisiae is mediated through lectin-like recognition of the mannan branches and so is dependent on the activity of *MNN2* (7). Thus, regulation of the expression of *MNN2* might provide a means of controlling flocculation, a critical process in fermentation (50). Third, the fact that two related proteins, Mnn2p and Mnn1p, are located in different cisternae of the Golgi may provide a system to investigate the general question of how the enzymes of the Golgi are restricted to particular subcompartments of this structure (51). Finally, there is evidence that the mannan-synthesizing enzymes interact with signal transduction pathways. Several enzymes in Fig. 7 show genetic interactions with the signaling pathways that are involved in mon-

FIG. 6. Mnn2p and Mnn5p have mannosyltransferase activity in vitro. Mannosyltransferase activity from GDP-mannose to α-1,6-mannobiose was assayed in vitro on IgG-Sepharose beads precipitated from strains expressing the indicated protein A fusions. The assay was performed as described under “Experimental Procedures” (complete) and also with either the acceptor or the manganese, omitted as indicated. The results are means of triplicates with error bars showing the standard deviation.

Thus, it seems that neither Mnn2p nor Mnn5p associates stoichiometrically with any other proteins in the same mild detergent conditions that have allowed the isolation of complexes of membrane proteins from both ER and Golgi. Such a protein A fusion has been used successfully to isolate the large complex of Golgi proteins containing Anp1p.² This suggests that Mnn2p and Mnn5p perform their functions separately and not as part of a larger complex.

Mnn2p and Mnn5p Have Mannosyltransferase Activity—The homology of the sequences of Mnn2p and Mnn5p with the known mannosyltransferase Mnn1p, and the loss of a specific linkage in the *mnn2* and *mnn5* mutants, strongly suggests that the proteins are themselves mannosyltransferases. To investigate this, the protein A chimeras described above were precipitated, and mannosyltransferase activity on the beads was assayed using GDP-mannose and α-1,6-mannobiose as an acceptor. Fig. 6 shows that Mnn2p-protein A has mannosyltransferase activity in this in vitro assay, and this was inhibited by removing the divalent cation manganese. The Mnn5p-protein A fusion showed a smaller amount of transferase activity in the same assay, which presumably reflects the fact that its normal substrate would be a branched version of the α-1,6 backbone. Thus, both the Mnn2p and Mnn5p proteins have mannosyltransferase activity in vitro.

DISCUSSION

In this paper, we have identified the yeast genes corresponding to the mannan synthesis mutants *mnn2* and *mnn5*. The proteins encoded by these genes are both predicted to be type II membrane proteins, and both are primarily localized to the cis Golgi. Our immunoprecipitation experiments demonstrate that Mnn2p and Mnn5p exist as discrete proteins rather than associating with any of the other enzymes involved in synthesizing the mannan structure. Previous characterization of *mnn2* and *mnn5* strains has shown that in each case a specific linkage in the N-linked mannan structure is missing, strongly suggesting that the *MNN2* and *MNN5* gene products are mannosyltransferases (30, 31). Consistent with this, the gene products show homology to the Golgi α-1,3-mannosyltransferase Mnn1p. This homology includes an aspartate-containing motif that we have recently shown to be essential for the catalytic activity of Mnn1p and that is conserved in a diverse range of both prokaryotic and eukaryotic glycosyltransferases (26). Moreover, mannosyltransferase activity was detectable in vitro with precipitated fusion proteins of Mnn2p and Mnn5p. Thus, the sim-

² J. C. Rayner and S. Munro, unpublished data.

26842 Components of Outer Chain Branching in Yeast

$\alpha$-1,6-mannosyl $\circ$

$\alpha$-1,2-mannosyl O

$\alpha$-1,3-mannosyl $\bullet$

![Diagram](attachment:diagram.png)

FIG. 7. A model for the synthesis of the mannan structure of *S. cerevisiae*. Glycoproteins arriving in the cis Golgi from the ER have N-linked Man₈GlcNAc₂ structures to which Och1p adds a single α-1,6-linked mannose. The next step is for the Mnn9p-Van1p complex to synthesize a short α-1,6-linked mannose polymer, which is then elongated to a length of 40–60 mannoses by the Mnn9p-Anp1p complex. There is evidence that the extension of the backbone is terminated by an α-1,2-linked mannose, and the Anp1p-Mnn9p complex has α-1,2-mannosyltransferase activity in vitro. However, the rest of the backbone is then branched with α-1,2-linked mannoses by Mnn2p, and these are then extended by Mnn5p. After this, the chains terminated by an α-1,3-linked mannose added by Mnn1p, and phosphomannose is added by Mnn6p. The question marks indicate the possibility that the Anp1p-containing complex contains further enzymes in addition to the proteins described here.

itoring cell wall integrity or in regulating the cell cycle. Thus, HOC1 is a multicopy suppressor of mutations in protein kinase C (PKC), and loss of any of ANP1, MNN9, and OCH1 produces synthetic lethality with overexpression of a cyclin that bypasses a cell-cycle checkpoint at budding (52, 53). Similar genetic interactions have been described for the enzymes that are involved in the synthesis of the glucan and chitin components of the cell wall, such as Kre6p and Chs3p (54, 55). It seems that when one component of the cell wall is defective, signal transduction pathways are required to induce compensatory changes in the synthesis or targeting of other cell wall components (56). Thus, these interactions of the mannan-synthesizing enzymes presumably reflect the importance of the mannan structure for cell wall integrity. Interestingly, there is evidence that loss of activity of MNN2 is also sensed by one of the cell wall-sensitive signaling pathways. A genetic screen was recently reported identifying mutations with a Ca²⁺-resistant and vanadate-sensitive (*crv*) phenotype (57). Four such *crv* mutations were obtained, one of which, *crv4*, was identified as being YBR015c, which we have now shown corresponds to MNN2. It was previously known that loss of the Ca²⁺-regulated protein phosphatase calcineurin in yeast results in a *crv* phenotype, and *crv1* is allelic to *cnb1*, the regulatory subunit of calcineurin. Interestingly, *CRV2* and *CRV3* were shown to be *BCK1* and *SLT2*, components of the mitogen-activated protein kinase pathway that is regulated by PKC and that senses cell wall integrity (58). Moreover, a double mutation of *crv4* (*i.e.* *mnn2*) and *slt2* showed a more severe osmotic sensitivity than either mutation alone, suggesting that unbranched mannan expressed in an *mnn2* strain results in an altered cell wall and that signaling via the PKC pathway is required to compensate for this defect. Evidence is beginning to emerge for the downstream targets of these signal transduction pathways that allow compensation for cell wall defects. Thus, it seems that the transcription of *MNN1* and *KRE6* is regulated in part by PKC (55), but it is also possible that there is post-transcriptional control of the activity or stability of the Golgi enzymes. Now that most of the enzymes required for mannan synthesis in the yeast Golgi have been identified, it should be possible to understand more of how they are organized within the Golgi and to determine how their activity is regulated by growth state, cell cycle, and cell wall integrity.

Acknowledgments—We thank Joern Jungmann for invaluable help in the initial stages of this project, Neta Dean for providing plasmids, Sew Yeu Peak-Chew for mass spectrometric analysis, Clint Ballou for helpful advice on *mnn* mutants, Matthias Mann for the invaluable recommendation of TEV protease and protein A, and Hugh Pelham for critical reading of the manuscript and helpful discussions.

REFERENCES

1. Herscovics, A., and Orlean, P. (1993) FASEB J. 7, 540–550
2. Orlean, P. (1997) in *The Molecular and Cellular Biology of the Yeast Saccharomyces Cerevisiae* (Pringle, J. R., Broach, J. R., and Jones, E. W., eds) Vol. 3, pp. 229–362, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
3. Ballou, L., Hernandez, L. M., Alvarado, E., and Ballou, C. E. (1990) Proc. Natl. Acad. Sci. U. S. A. 87, 3368–3372
4. Trimble, R. B., and Atkinson, P. H. (1986) J. Biol. Chem. 261, 9815–9824
5. Peat, S., Turvey, J. R., and Doyle, D. (1961) J. Chem. Soc. 3918–3923
6. Klis, F. M. (1994) Yeast 10, 851–869
7. Stratford, M. (1992) Yeast 8, 635–645
8. Calderone, R. A. (1993) Trends Microbiol. 1, 55–58
9. Shibata, N., Ikuta, K., Imai, T., Satoh, Y., Satoh, R., Suzuki, A., Kojima, C., Kobayashi, H., Hisamichi, K., and Suzuki, S. (1995) J. Biol. Chem. 270, 1113–1122
10. Ballou, L., and Ballou, C. (1995) Proc. Natl. Acad. Sci. U. S. A. 92, 2790–2794
11. Jimenez-Barbero, J., Prieto, A., Gomez-Miranda, B., Leal, J. A., and Bernabe, M. (1995) Carbohydr. Res. 272, 121–128
12. Nakajima, T., and Ichishima, E. (1994) J. Ferment. Bioeng. 78, 472–475
13. Iawahara, S., Mishima, T., Ramli, N., and Tekagawa, K. (1996) Biosci. Biotech. Biochem. 60, 957–961
14. Novick, P., Ferro, S., and Schekman, R. (1981) Cell 25, 461–469
15. Graham, T. R., and Emr, S. D. (1991) J. Cell Biol. 114, 207–218
16. Ballou, C. E. (1990) Methods Enzymol. 185, 440–470
17. Lehle, L., Eiden, A., Lehnert, K., Haselbeck, A., and Kopetzki, E. (1995) FEBS Lett. 370, 41–45
18. Nagasu, T., Shimma, Y. I., Nakanishi, Y., Kuromitsu, J., Iwama, K., Nakayama, K. I., Suzuki, K., and Jigami, Y. (1992) Yeast 8, 535–547
19. Manas, P., Olivero, I., Avalos, M., and Hernandez, L. M. (1997) Glycobiology 7, 487–497
20. Verostek, M. F., and Trimble, R. B. (1995) Glycobiology 5, 671–681
21. Häusler, A., Ballou, L., Ballou, C. E., and Robbins, P. W. (1992) Proc. Natl. Acad. Sci. U. S. A. 89, 6846–6850
22. Esmon, B., Esmon, P. C., and Schekman, R. (1984) J. Biol. Chem. 259, 10322–10327
23. Nakayama, K., Nagasu, T., Shimma, Y., Kuromitsu, J., and Jigami, Y. (1992) EMBO J. 11, 2511–2519
24. Jungmann, J., and Munro, S. (1998) EMBO J. 17, 423–434
25. Yip, C. L., Welch, S. K., Klebl, F., Gilbert, T., Seidel, P., Grant, F. J., O'Hara, P. J., and MacKay, V. L. (1994) Proc. Natl. Acad. Sci. U. S. A. 91, 2723–2727
26. Wiggins, C. A. W., and Munro, S. (1998) Proc. Natl. Acad. Sci. U. S. A. 95, 7945–7950
27. Wang, X.-H., Nakayama, K., Shimma, Y., Tanaka, A., and Jigami, Y. (1997) J. Biol. Chem. 272, 18117–18124
28. Guthrie, C., and Fink, G. R. (1991) *Guide to Yeast Genetics and Molecular Biology*, Academic Press, San Diego
29. Robinson, J. S., Klionsky, D. J., Banta, L. M., and Emr, S. D. (1988) Mol. Biol. Biol. 8, 4936–4948
30. Raschke, W. C., Kern, K. A., Antalis, C., and Ballou, C. E. (1973) J. Biol. Chem. 248, 4660–4666
31. Cohen, R. E., Ballou, L., and Ballou, C. E. (1980) J. Biol. Chem. 255, 7700–7707
32. Baudin, A., Ozierkalogeropoulos, O., Denouel, A., Lacroute, F., and Cullin, C. (1993) Nucleic Acids Res. 21, 3329–3330
33. Wach, A., Brachat, A., AlbertiSegui, C., Rebischung, C., and Philippsen, P. (1997) Yeast 13, 1065–1075
34. Parks, T. D., Leuther, K. K., Howard, E. D., Johnston, S. A., and Dougherty, W. G. (1994) Anal. Biochem. 216, 413–417
35. Nilsson, B., Moks, T., Jansson, B., Abrahmsen, L., Elmblad, A., Holmgren, E., Henrichson, C., Jones, T. A., and Uhlen, M. (1987) Protein Eng. 1, 107–113
36. Dean, N., and Poster, J. B. (1996) Glycobiology 6, 73–81
37. Dean, N., and Pelham, H. R. B. (1990) J. Cell Biol. 111, 369–377

38. Hardwick, K. G., and Pelham, H. R. B. (1992) *J. Cell Biol.* 119, 513–521
39. Shevchenko, A., Jensen, O. N., Podtelejnikov, A. V., Sagliocco, F., Wilm, M., Vorm, O., Mortensen, P., Shevchenko, A., Boucherie, H., and Mann, M. (1996) *Proc. Natl. Acad. Sci. U. S. A.* 93, 14440–14445
40. Nakajima, T., and Ballou, C. (1975) *Proc. Natl. Acad. Sci. U. S. A.* 72, 3912–3916
41. Ballou, D. L. (1975) *J. Bacteriol.* 123, 616–619
42. Romero, P. A., Athanasiadis, A., Lussier, M., and Herscovics, A. (1994) *Yeast* 10, 1111–1115
43. Ballou, C. E., Kern, K. A., and Raschke, W. C. (1973) *J. Biol. Chem.* 248, 4667–4673
44. Lussier, M., Sdicu, A. M., Ketela, T., and Bussey, H. (1995) *J. Cell Biol.* 131, 913–927
45. Schwappach, B., Stobrawa, S., Hechenberger, M., Steinmeyer, K., and Jentsch, T. J. (1998) *J. Biol. Chem.* 273, 15110–15118
46. Harris, S. L., and Waters, M. G. (1996) *J. Cell Biol.* 132, 985–998
47. Lussier, M., Sdicu, A.-M., Winnett, E., Vo, D. H., Sheraton, J., Düsterhöft, A., Storms, R. K., and Bussey, H. (1997) *Yeast* 13, 267–274
48. Lussier, M., Sdicu, A.-M., Bussereau, F., Jacquet, M., and Bussey, H. (1997)
49. Sipos, G., Puoti, A., and Conzelmann, A. (1995) *J. Biol. Chem.* 270, 19709–19715
50. Straver, M. H., Kijne, J. W., and Smit, G. (1993) *Trends Biotech.* 11, 228–232
51. Munro, S. (1998) *Trends Cell Biol.* 8, 11–15
52. Neiman, A. M., Mhaiskar, V., Manus, V., Galibert, F., and Dean, N. (1997) *Genetics* 145, 637–645
53. Mondesert, G., Clarke, D. J., and Reed, S. I. (1997) *Genetics* 147, 421–434
54. Roemer, T., Paravicini, G., Payton, M. A., and Bussey, H. (1994) *J. Cell Biol.* 127, 567–579
55. Igual, J. C., Johnson, A. L., and Johnston, L. H. (1996) *EMBO J.* 15, 5001–5013
56. Ram, A. F. J., Kapteyn, J. C., Montijn, R. C., Caro, L. H. P., Douwes, J. E., Baginsky, W., Mazur, P., VandenEnde, H., and Klis, F. M. (1998) *J. Bacteriol.* 180, 1418–1424
57. Nakamura, T., Ohmoto, T., Hirata, D., Tsuchiya, E., and Miyakawa, T. (1997) *Mol. Gen. Genet.* 256, 481–487
58. Nakamura, T., Ohmoto, T., Hirata, D., Tsuchiya, E., and Miyakawa, T. (1996) *Mol. Gen. Genet.* 251, 211–219
