
P1: DPI/plb
P2: NBL/dat
QC: NBL
April 29, 1998
13:49
Annual Reviews
AR057-23

Annu. Rev. Biochem. 1998.67:721–51
Copyright © 1998 by Annual Reviews. All rights reserved

# THE DNA REPLICATION FORK
## IN EUKARYOTIC CELLS

### Shou Waga and Bruce Stillman
Cold Spring Harbor Laboratory, P.O. Box 100, Cold Spring Harbor, New York 11724

#### KEYWORDS:
polymerase switching, Simian virus 40, cell cycle, Okazaki fragments, replisome proteins

---

### ABSTRACT
Replication of the two template strands at eukaryotic cell DNA replication forks is a highly coordinated process that ensures accurate and efficient genome duplication. Biochemical studies, principally of plasmid DNAs containing the Simian Virus 40 origin of DNA replication, and yeast genetic studies have uncovered the fundamental mechanisms of replication fork progression. At least two different DNA polymerases, a single-stranded DNA-binding protein, a clamp-loading complex, and a polymerase clamp combine to replicate DNA. Okazaki fragment synthesis involves a DNA polymerase-switching mechanism, and maturation occurs by the recruitment of specific nucleases, a helicase, and a ligase. The process of DNA replication is also coupled to cell-cycle progression and to DNA repair to maintain genome integrity.

---

### CONTENTS
- INTRODUCTION ..... 722
- CELLULAR REPLICATION FORK PROTEINS ..... 722
  - DNA Polymerase α/Primase Complex ..... 723
  - Replication Protein A (RPA) ..... 724
  - Replication Factor C (RFC) ..... 726
  - Proliferating Cell Nuclear Antigen (PCNA) ..... 728
  - DNA Polymerases δ and ε ..... 731
  - FEN1 and RNase HI ..... 732
  - DNA Helicases ..... 733
  - Molecular Links Among FEN1, Dna2 Helicase, and DNA Polymerase α/Primase ..... 735
- MECHANISMS OF DNA SYNTHESIS AT A REPLICATION FORK ..... 736
  - Primosome Assembly ..... 736
  - Polymerase Switching ..... 738
  - Maturation of Okazaki Fragments ..... 739

0066-4154/98/0701-0721$08.00

722 WAGA & STILLMAN

REPLICATION FORK PROTEINS AND CELL-CYCLE CONTROL ..... 741
S-Phase Checkpoint Control ..... 741
PCNA-p21 Interaction ..... 742
Regulation of Telomere Length ..... 743
CONCLUDING REMARKS ..... 744

# INTRODUCTION

In a proliferating eukaryotic cell, the duplication of the genetic complement occurs every S phase of the cell cycle and must occur with high accuracy only once per cell cycle. Furthermore, DNA replication in a single cell must be coordinated with other cell-cycle processes such as mitosis and cytokinesis and with DNA replication in the surrounding cells in tissue. Much of this regulation occurs at the level of initiation of DNA replication via interaction with the pathways that control cell cycle progression. Although we still do not fully understand how initiation of DNA replication occurs in eukaryotes, rapid progress is being made and has been reviewed elsewhere (1, 2). Once initiation occurs, the replication apparatus copies each replicon in a highly efficient process. The fundamental mechanisms that operate at the eukaryotic DNA replication fork are now quite well known and are discussed here. Because the replication of DNA in eukaryotic cells must be coupled to DNA repair and assembly of the DNA into chromatin, the replication fork proteins play prominent roles in maintaining the fidelity of DNA replication, in coordinating replication with cell-cycle progression, and in the inheritance of chromatin complexes.

# CELLULAR REPLICATION FORK PROTEINS

Most of what is known about DNA replication in eukaryotes comes from extensive studies performed using cell extracts from mammalian cells that support the complete replication of plasmid DNAs containing the Simian Virus 40 DNA replication origin (SV40 ori) (3–6). SV40 DNA replication requires a single virus-encoded protein, the SV40 large tumor antigen (T antigen), which functions both as an initiator protein by binding to the SV40 ori and as a DNA helicase at the replication fork; thus the replication is achieved primarily by cellular proteins that also function to duplicate cellular DNA. The purification and reconstitution of DNA replication with purified proteins has yielded great insight into the mechanism of DNA replication as well as other aspects of DNA metabolism such as DNA repair and recombination (7–10).

Investigation of more specific enzymatic reactions and yeast genetic studies have uncovered several proteins thought to be involved directly in DNA synthesis at the replication fork. This review briefly outlines the current understanding of the eukaryotic fork proteins (Table 1) and the reactions in which they

DNA REPLICATION IN EUKARYOTES

Table 1 Functions of DNA replication fork proteins

| Proteins       | Functions                                                                 |
|----------------|--------------------------------------------------------------------------|
| RPA            | Single-stranded DNA binding; stimulates DNA polymerases; facilitates helicase loading |
| PCNA           | Stimulates DNA polymerases and RFC ATPase                                |
| RFC            | DNA-dependent ATPase; primer-template DNA binding; stimulates DNA polymerases; PCNA loading |
| Pol α/primase  | RNA-DNA primer synthesis                                                |
| Pol δ/ε<sup>a</sup> | DNA polymerase; 3′–5′ exonuclease                                    |
| FEN1           | Nuclease for removal of RNA primers                                     |
| RNase HI       | Nuclease for removal of RNA primers                                     |
| DNA ligase I   | Ligation of DNA                                                        |
| T antigen<sup>b</sup> | DNA helicase; primosome assembly                                   |

<sup>a</sup>A specific function of DNA polymerase ε in replication has not been assigned, although it is known to be essential for S-phase progression in *S. cerevisiae* (196, 197).

<sup>b</sup>T antigen is required for the replication of SV40 DNA. Its functional equivalent in eukaryotic cells has not been identified.

participate. DNA ligases (11, 12) and topoisomerases I and II (13) are also required for replication, but since the function of these enzymes has been reviewed elsewhere, they are not covered extensively here.

### DNA Polymerase α/Primase Complex

The DNA polymerase α/primase complex (pol α/primase) is the only enzyme capable of initiating DNA synthesis de novo by first synthesizing an RNA primer and then extending the primer by polymerization to produce a short DNA extension (RNA-DNA primer) (14). Analyses of SV40 ori–dependent replication in vivo and in vitro has demonstrated that pol α/primase can synthesize an RNA-DNA primer of approximately 40 nucleotides (nt) in length, including about 10 nt of RNA primer (15–20). The short RNA-DNA then serves as a primer for extension by another polymerase for DNA synthesis on either the leading (continuously synthesized) strand or for each Okazaki fragment on the lagging (discontinuously synthesized) strand (21–26). This process involves a polymerase switch from pol α/primase to either DNA polymerase δ or ε (pol δ and ε) (see below). This switch occurs because, unlike other more complex polymerases, pol α/primase is not capable of processive DNA synthesis and dissociates from the template DNA following primer synthesis (27).

The human cell pol α/primase consists of four subunits (p180, p70, p58, and p48), and similar subunits are found in all eukaryotes examined (for review see 14, 28). cDNAs encoding all four subunits of human, mouse, or yeast (*Saccharomyces cerevisiae* and *Schizosaccharomyces pombe*) pol α/primase have been cloned, and active complexes have been reconstituted from recombinant

proteins expressed from baculovirus vectors in infected insect cells (14, 29, 30). The p180 and p48 polypeptides harbor the DNA polymerase and primase catalytic activities, respectively. Extensive mutational analyses of the genes encoding both polymerase and primase catalytic subunits show that they function in DNA replication in vivo but also suggest a regulatory role for the primase subunit (see below; 14, 31–33). The p58 subunit is necessary for the stability and activity of the primase p48 subunit (29, 34, 35). Although no enzymatic activity has been associated with p70 (also known as B subunit or p86 in *S. cerevisiae*) (36, 37), biochemical studies have shown that p70 plays an important role in the assembly of the primosome (see below). In a complementary approach, genetic experiments with temperature-sensitive mutants of *S. cerevisiae* (*pol12-T*) revealed an essential function for the equivalent subunit (p86) in the initiation of DNA replication (37).

The phosphorylation of pol α/primase has been reported in human and *S. cerevisiae* cells (38–40); both human p180 and p70 were phosphorylated predominantly during late G2 and M phases of the cell cycle (38). Yeast p86 (B subunit) is phosphorylated in a manner that depends on the stage of the cell cycle (39), but most phosphorylation occurs late in the cell cycle, suggesting that it might play a role either in coordinating replication and mitosis or in resetting the replication apparatus for the next S phase. Analysis of the initiation of SV40 DNA replication showed that the primase activity during initiation could be suppressed when pol α/primase was phosphorylated by cyclin A-CDK2, but not by cyclin B-CDK2 or cyclin E-CDK2, whereas DNA polymerase or primase activities with synthetic templates were hardly affected by this phosphorylation (41). This observation suggests that the phosphorylation of pol α/primase may play a regulatory role in the initiation of replication, but the in vivo significance of the phosphorylation is unknown.

### Replication Protein A (RPA)

Replication protein A (RPA; also reported previously as RFA or HSSB) is a single-stranded DNA-binding protein that exists as a heterotrimeric complex consisting of subunits with apparent masses of approximately 70, 34, and 11 kDa in all eukaryotic cells examined (for review see 42, 43). The trimeric protein was initially identified as a factor essential for SV40 DNA replication in vitro (44–46), but subsequently it was shown to be involved in DNA recombination and repair (43, 47).

RPA promotes extensive unwinding of duplex DNA containing the SV40 ori by SV40 T antigen, which in addition to recognizing the *ori* in a sequence-specific manner, is an RPA-stimulated DNA helicase (48–52). RPA also stimulates pol α/primase activity under certain conditions and is required for replication factor C (RFC)– and proliferating cell nuclear antigen (PCNA)–dependent

DNA synthesis by DNA polymerase δ (25, 53–57). In support of these biochemical studies, some temperature-sensitive mutations in the gene encoding the large subunit of *S. cerevisiae* RPA showed synthetic lethality, a form of genetic interaction, with mutations in genes encoding pol α, primase, or pol δ (58). Binding assays with purified proteins demonstrated that p70 binds to the primase subunits of pol α/primase and that the RPA heterotrimeric complex, but not p70 alone, binds to SV40 T antigen (59, 60). These interactions are thought to be required for the assembly of the primosome (see below).

cDNAs encoding each of the individual subunits of RPA have been cloned from a variety of species (43, 61). Furthermore, RPA has been produced in bacteria, and the recombinant human trimeric protein can support SV40 DNA replication in vitro (62, 63). Functional RPA has also been produced by infection of recombinant baculoviruses into insect cells (64). Although the human p70 subunit alone can bind single-stranded DNA, it cannot support DNA replication in vitro (55, 65, 66). Mutational studies to probe the structure of p70 have located the DNA-binding region to the N-terminal two-thirds of the subunit; the C-terminal third containing a putative zinc-finger was required for interactions with the other two subunits (65–67).

More recent structural analysis of yeast RPA suggests that it contains a total of four potential single-stranded DNA-binding domains that are distantly related to each other (68). The domains are called SBDs and are made up of about 120 amino acids each; two are in the large subunit and one each is in the middle and the small subunits, although the evidence for the DNA-binding domain in the small subunit is not as convincing as for the large subunit domains. A crystal structure of the two DNA-binding domains derived from the human large subunit (p70) revealed two structurally related subdomains, each corresponding to an SBD (69). It has been suggested that these SBDs, which contain clusters of aromatic amino acids that are similar to structures within the DNA-binding domain of the *Escherichia coli* single-stranded DNA-binding protein (SSB), are responsible for a higher-order assembly of the RPA-DNA complex, where RPA might be wrapped with DNA (68). Indeed DNA-binding studies with human RPA have demonstrated the existence of at least two distinct DNA-binding modes of the protein: one involving 8–10 nucleotides and another involving 30 nucleotides (70, 71). There may well be higher-order interactions between RPA and longer stretches of single-stranded DNA, as occurs for *E. coli* SSB (72).

Both the large (p70) and the middle (p34) subunits of human and yeast RPA are phosphorylated in a cell-cycle-dependent manner (73). Increased levels of similar phosphorylated forms of RPA are also seen in response to DNA damage (74, 75). The phosphorylation of the p34 subunit has been characterized extensively; it is phosphorylated in the S and G2 phases of the cell cycle, and cyclin-dependent and DNA-dependent kinases have been identified as enzymes

726 WAGA & STILLMAN

capable of phosphorylating p34 (76–84). The phosphorylation of RPA is delayed in cells from patients who have ataxia telangiectasia (AT), a cancer-prone disease resulting from loss of the DNA damaging surveillance ATM gene product; and the phosphorylation of RPA is compromised in yeast lacking the *MECl* gene, a gene with similarities to ATM (74, 85, 86). This is particularly interesting because phosphorylation of RPA seems to decrease the interaction between this protein and the cellular tumor suppressor protein, p53 (87). However, the role of phosphorylation on the function of RPA in either SV40 DNA replication or nucleotide-excision repair is not clear (88–90), and even the link between phosphorylation of RPA and S-phase checkpoint controls has been questioned (91). Thus the functional significance of the RPA phosphorylation remains to be determined.

A little understood aspect of RPA in cellular DNA metabolism is its ability to assemble into discrete foci (pre-replication center) on postmitotic, decondensing chromosomes, which afterward serve as replication foci following the assembly of a nuclear membrane (92). A protein called FFA-1, which is required for the assembly of the RPA foci, has been identified in *Xenopus* egg extracts (93), but formation of the RPA foci required neither subunits of the cellular initiator protein ORC (origin recognition complex), nor the Cdc6 protein (94), both of which are essential for initiation of DNA replication in *Xenopus* extracts and in yeast (2). It is therefore not clear what relationship these replication foci have to ORC-dependent DNA replication or if they form in a normal cell cycle. It is possible that during chromosome decondensation, single-stranded DNA regions created by torsional strain might be bound by RPA, providing local assembly sites for pol α/primase, which binds to RPA. The structure and function of these foci and their role in normal DNA replication in cells remain to be elucidated.

### Replication Factor C (RFC)

One of the key proteins involved in loading the replicative polymerases to create the replication fork is replication factor C (RFC), a complex of five subunits (p140, p40, p38, p37, and p36) that is conserved in all eukaryotes (for review see 42). Functional homologs exist in bacteria, some bacteriophages, and Archaea (95). The cDNAs encoding the individual subunits of human and the yeast *S. cerevisiae* RFC have been cloned, and all five yeast genes are essential for cell viability (96–104). Sequence comparisons show a high degree of similarity among all five subunits, and based on these similarities and the conserved sequences found among species, short stretches of amino acid sequences called RFC boxes I–VIII have been defined (104). Box I is unique to the larger p140 subunit, is related to sequences in prokaryote DNA ligases, and is distantly related to the BRCT motif present in many proteins that respond to DNA damage

in cells (98, 105); boxes III and V are characteristic of sequence motifs present in many ATP- and GTP-binding proteins. The other RFC sequence motifs, particularly RFC box VIII, are found in a number of other replication proteins and proteins of unknown function, in addition to being highly conserved among the RFC subunits (104, 106).

RFC was first identified because it is an essential factor for SV40 DNA replication in vitro (107). It preferentially binds to a primer-template junction created by the annealing of an oligonucleotide to single-stranded DNA, or by synthesis of a DNA primer on a single-stranded DNA template. RFC can also bind to a nick in duplex DNA. Binding requires ATP and upon binding to DNA, RFC functions as a DNA-dependent ATPase, an activity stimulated further by PCNA (56, 108–112). One study suggests that ATP hydrolysis is required for the stable loading of PCNA (113), but other studies suggest that this is not the case (56); thus the precise role of ATP in this process remains to be determined. The main role for RFC is to load the trimeric, ring-like structure of PCNA onto DNA at a primer-template junction or to load it onto a nicked site in duplex DNA (56, 108, 109, 114). It has been reported that RFC can load PCNA onto completely duplex DNA, but compared to the interactions with the above-mentioned DNAs, this interaction is very sensitive to physiologic salt concentrations and probably does not represent a reaction that occurs during DNA replication in cells (115). RFC-catalyzed PCNA loading is a prerequisite for assembly of pol δ onto the template DNA to form a processive holoenzyme (25, 108–113, 116), which then functions during synthesis of both the leading and lagging strands at a DNA replication fork (see below).

A functional human RFC complex has been reconstituted using proteins expressed in recombinant baculovirus-infected insect cells (114, 117, 118), and the yeast RFC has been overexpressed in yeast cells (119). Several mutational analyses of human RFC show that distinct regions of the p140 subunit are responsible for DNA and PCNA binding (120, 121). The PCNA-binding domain from the p140 subunit inhibits DNA replication in mammalian cells (120), supporting a role for RFC in DNA replication in vivo. While the N-terminal region that includes RFC box I in p140 has a DNA-binding activity, an RFC complex lacking this region exhibits enhanced activity in a reconstituted SV40 DNA replication reaction as well as enhanced PCNA loading activity (122). This observation suggests a regulatory role for this protein domain that includes the similarity to the BRCT motif. The large subunit of RFC is a target for caspases, the proteases activated during apoptosis or programmed cell death (123, 124), perhaps because it is a significant ATP-regulated enzyme involved in DNA metabolism.

Limited structure-function analysis of the small subunits has been performed, but the C-terminal sequences of each of the small subunits are required for

formation of the RFC complex (122, 125). Three small subunits (p40, p37, and p36) form a stable, core complex that has some DNA-dependent ATPase activity, but without the large p140 subunit, this ATPase is no longer stimulated by PCNA (117, 118, 126, 127). The p38 small subunit seems to provide a link between the p140 and the core complex (118, 126).

A cold-sensitive *S. cerevisiae* mutant in the gene encoding the RFC large subunit (*cdc44*) has been isolated and characterized (100). This mutation has an altered DNA metabolism at the nonpermissive temperature that is consistent with a role for RFC in DNA replication or DNA repair. These defects were suppressed by mutations in the gene encoding PCNA (*pol30*), supporting the biochemical interaction between these two proteins (128). Interestingly, a mutation in the *RFC5* gene encoding the *S. cerevisiae* p38 subunit homolog causes a defect in a DNA-damage checkpoint signal that transmits to the Rad53 protein and the Tel1 protein, a yeast protein similar to the ataxia telangiectasia gene product ATM (129). This observation suggests that RFC might function in monitoring DNA damage at the replication fork.

Of all the DNA polymerase accessory proteins, the subunits of RFC show both striking sequence and functional similarities to replication fork proteins present in *E. coli* and bacteriophage T4 (*γ*-complex and gp44/gp62, respectively; 95, 106). These proteins load a ring-like DNA polymerase clamp (e.g., PCNA in eukaryotes and the *E. coli β*-subunit of DNA polymerase III) onto the template DNA and, therefore, are known as clamp-loading proteins. The recent determination of the crystal structure of a member of the *γ*-complex, the *δ'* subunit, shows it to have a provocative C-like shape that shows striking structural overlap with the ring-like shape of the *E. coli β*-subunit and PCNA, suggesting how the clamp-loader might open the ring-like clamp and load it onto to DNA (Figure 1; 130). The clamp-loading function of RFC may be one of the key regulated events in DNA metabolism.

# Proliferating Cell Nuclear Antigen (PCNA)

Perhaps one of the most intensely studied proteins is proliferating cell nuclear antigen (PCNA), the DNA polymerase clamp. Not only does this protein play a central role in DNA metabolism, but it has become a significant clinical

---

**Figure 1** Structures of a polymerase clamp and a clamp-loading protein subunit. *Top:* stereo view of a ribbon diagram of the structure of the clamp-loader, *δ'*, a part of the *E. coli* DNA polymerase III *γ*-complex and a functional homolog of the RFC subunits (130). *Bottom:* super-position of the *δ'* structure and a ribbon diagram of the structure of the *E. coli* DNA polymerase III *β*-subunit (133), a functional and structural homolog of PCNA. Note the similar shape of the clamp ring surrounding the dimer-dimer interface of the two *β*-subunits and the clamp-loader C-shape.

P1: DPI/plb     P2: NBL/dat     QC: NBL
April 29, 1998     13:49     Annual Reviews     AR057-23

DNA REPLICATION IN EUKARYOTES     729

$\alpha_{13}$  
$\alpha_{12}$  
$\alpha_{11}$  
$\alpha_{15}$  
$\alpha_{10}$  
$\alpha_{14}$  
$\alpha_{8}$  
$\alpha_{9}$  
Sensor-1 Motif  
$\beta_{5}$  
$\beta_{1}$  
$\beta_{4}$  
$\alpha_{6}$  
$\beta_{3}$  
$\beta_{2}$  
$\alpha_{1}$  
$\alpha_{2}$  
$\alpha_{4}$  
$\alpha_{5}$  
$\alpha_{3}$  
$Zn^{2+}$  

$\alpha_{13}$  
$\alpha_{12}$  
$\alpha_{11}$  
$\alpha_{15}$  
$\alpha_{10}$  
$\alpha_{14}$  
$\alpha_{8}$  
$\alpha_{9}$  
P-loop  
Sensor-2 Motif  
Sensor-1 Motif  
$\beta_{5}$  
$\beta_{1}$  
$\beta_{4}$  
$\alpha_{6}$  
$\beta_{3}$  
$\beta_{2}$  
$\alpha_{1}$  
$\alpha_{2}$  
$\alpha_{4}$  
$\alpha_{5}$  
$\alpha_{3}$  
$Zn^{2+}$  

DExx  
Motif  

DExx  
Motif  

$\beta$-subunit ring  

Dimer  
Interface  

diagnostic marker for proliferating cells. A protein with an apparent mass on polyacrylamide gels of 36 kDa, PCNA forms a homotrimeric complex and functions as a DNA polymerase accessory factor (for reviews see 42, 131, 132). The primary amino acid sequence of PCNA is not highly conserved among species, but yeast and human PCNA nevertheless have an almost identical three-dimensional shape that is also very similar to the structures of the functional homologs of PCNA present in *E. coli* and bacteriophage T4 (i.e. the β-subunit of DNA polymerase III in *E. coli* and the gp45 protein of T4 phage) (95, 132–135). These proteins exist as stable trimers that form a closed ring with a hole in the center that encircles duplex DNA (Figure 1). Each protein monomer of PCNA consists of two structurally similar domains that are linked via an interdomain connecting loop on the surface of the protein (134). The internal surface of the trimer contains six α helices, and each helix is present in a structural repeat in each PCNA monomer. Since RFC loads the PCNA trimer onto DNA, it topologically links the PCNA trimer to the DNA, allowing PCNA to track along the DNA (136). Most probably, RFC also functions to unload the PCNA when DNA synthesis is complete, much like the bacterial and phage T4 counterparts (137–139).

PCNA functions as a processivity factor for pol δ during DNA replication (140–144). Stimulation of pol ε by PCNA has also been detected but only under limited conditions (145–147). PCNA itself does not have DNA-binding activity, but it can be loaded onto the DNA by RFC in an ATP-dependent manner (56, 114). PCNA then associates with pol δ/ε at the primer-template junction (56, 113, 116). Mutational analyses of PCNA have shown that distinct regions of the trimeric ring are required for the stimulation of pol δ or RFC ATPase (148–151); PCNA mutants that alter certain amino acids on the internal surface of the ring failed to stimulate pol δ (149). In contrast, regions on the outer surface, including both the N- and C-termini and the interdomain connecting loop, are necessary for the interaction with pol δ and RFC (149, 151).

PCNA also is capable of binding to other proteins, including the FEN1/Rad27/MF1 nuclease (152, 153) (see below); DNA ligase 1 (154); the p53-inducible cyclin-dependent kinase (CDK) inhibitor protein p21 (WAF1, CIP1, sdil) (155, 156); the p53-inducible GADD45 protein (157, 158); the nucleotide excision repair protein XPG (159); DNA-(cytosine-5) methyltransferase (160); the mismatch repair proteins MLH1 and MSH2 (161); and cyclin D (162, 163). These varied interactions with DNA metabolism proteins imply that PCNA is a central factor for the coordination of DNA replication, DNA repair, epigenetic inheritance, and cell-cycle control.

The crystal structure of PCNA bound to a peptide derived from p21 shows that the previously unstructured p21 peptide inserts itself into a cleft in the interdomain connecting loop surface of PCNA (164). This very stable binding

causes the inhibition of PCNA stimulation of pol δ activity and competition with the binding of PCNA to DNA ligase, DNA-(cytosine-5) methyltransferase, XPG, and FEN1 (see below). Moreover, a monoclonal antibody that specifically binds to this loop inhibits PCNA-activated DNA synthesis by pol δ (165). Thus the loop serves as an interface for interactions with other cellular proteins.

Immunofluorescent staining of cells with anti-PCNA antibodies has shown that PCNA co-localizes with sites of DNA synthesis in nuclei (replication foci) (166, 167). The PCNA staining pattern varies with the stage in the S phase, corresponding with the region of the genome being replicated. In addition, PCNA becomes resistant to extraction from nuclei with detergents such as Triton X-100, only when cells enter the S phase of the cell cycle (166, 168). PCNA resistance to Triton-extraction also occurs in DNA-damaged cells, even in the G1 phase of the cell cycle, indicating that PCNA localizes to sites of DNA repair (168–172). The variable, detergent-resistant PCNA in nuclei is thought to reflect its topologically closed interaction with DNA at sites of DNA metabolism.

# DNA Polymerases δ and ε

Two DNA polymerases function during the S phase of the cell cycle in eukaryotic cells (14). DNA polymerase δ (pol δ) is a heterodimer composed of p125 and p50 subunits (for review see 14), although we consider it likely that the native enzyme in mammalian cells and *S. cerevisiae* contains an additional subunit of approximate mass of 50 kDa. Recent biochemical and genetic evidence shows that the *S. pombe* pol δ contains five subunits, three of which are essential (173–175). p125 is the catalytic subunit for both DNA polymerase activity and a proofreading, 3′-5′ exonuclease activity. cDNAs for each subunit have been cloned (175–183). Moreover, an active human dimeric complex has been reconstituted using proteins expressed in recombinant baculovirus-infected insect cells (184). The N-terminal region of p125 interacts with PCNA (185, 186), and although one report suggests that the polymerase activity of the large subunit alone can be stimulated by PCNA in *S. cerevisiae* (185), a recent study of mammalian pol δ shows that the p50 subunit is required for PCNA stimulation (184). To complicate matters further, a putative PCNA-independent pol δ was also isolated from mouse (187) and *Drosophila* (188). Thus further characterization of the structure of DNA polymerase δ in mammalian cells and *S. cerevisiae* is necessary.

Pol δ is required for DNA synthesis of both the leading and lagging strands during SV40 DNA replication in vitro (22–24, 26, 189). Many studies using a variety of DNA templates support a role for pol δ in RFC- and PCNA-dependent DNA replication (24–26, 56, 109, 113, 146, 184, 189–194). Crosslinking of DNA polymerases to replicating SV40 DNA in vivo also supports the

involvement of both pol δ and pol α in SV40 DNA replication (195). Although some reports demonstrate a role for pol ε in cellular DNA replication (113, 146, 196–200), the in vivo crosslinking experiments indicate that pol ε is not essential for SV40 DNA replication (195). This study, however, found that pol ε did crosslink to replicating cell chromosomal DNA, consistent with the essential function of S. cerevisiae pol ε gene (POL2) in cellular DNA replication (196, 197). The precise role of polymerase ε in cellular DNA replication remains to be determined.

# FEN1 and RNaseHI

FEN1, a 46-kDa single polypeptide in human and mouse, is a 5′-3′ exo/endonuclease that is required for Okazaki fragment maturation (for reviews see 10, 201). The primary sequence of the protein shows similarity to the repair protein Rad2/XPG and other nucleases (202, 203). FEN1 was identified through purification of the enzyme that specifically cleaves a flap-structure DNA substrate, hence the name flap endonuclease (204). Other researchers independently identified the same protein in different contexts, as follows: MF1 (192), 5′-3′ exonuclease (205), cca/exo (206), DNase IV (207), pol ε-associated nuclease (208), human homolog of S. pombe Rad2 (203), and factor pL (209). The homologs from S. cerevisiae and S. pombe were identified as Rad27 (210, 211) and Rad2 (203), respectively, indicating a role for the protein in DNA repair. FEN1 is required for SV40 DNA replication in vitro (192, 205).

Several biochemical studies have shown that FEN1 functions specifically to remove the RNA primer attached to the 5′-end of each Okazaki fragment (26, 200, 212–215). Extensive studies show that removal of the RNA primer involves other proteins including an RNA-DNA junction endonuclease, PCNA, and Dna2 helicase (10). The mechanism for maturation of Okazaki fragments is described below, but removal of the RNA primer from the 5′-end of the penultimate Okazaki fragment prior to joining to the newly synthesized Okazaki fragment requires strand displacement synthesis to displace the RNA primer, creating a flap structure with a 5′ unpaired RNA-DNA strand.

The enzymatic properties of FEN1 have been examined in some detail. When provided a flap structure containing a 5′-segment of DNA (or RNA) that is not paired to a template DNA, FEN1 efficiently cleaves at the branch point, releasing the unpaired segment (202, 204, 207, 215, 216). However, if the 5′-segment of DNA (or RNA) is completely paired, FEN1 can degrade DNA (or RNA) only exonucleolytically from the 5′-terminus (192, 200, 202, 204, 214). A substrate that contains a 5′-triphosphorylated ribonucleotide that is annealed to DNA, such as an Okazaki fragment that is completely annealed to a single-stranded DNA, cannot be degraded by FEN1 (200). In such a situation, removal of the RNA requires a ribonuclease in addition to FEN1 (see below).

PCNA and RPA stimulate yeast FEN1 activity under certain conditions (152, 217). These interactions may be important for the maturation of Okazaki fragments during lagging strand replication, as described below. The cyclin-CDK inhibitor p21 disrupts the FEN1-PCNA interaction, suggesting that the step might be regulated (153).

While both *S. cerevisiae* RAD27 and *S. pombe rad2+* are not essential for cell viability, both null mutants exhibit elevated chromosome loss rates and increased UV sensitivity (203, 210, 218). Furthermore, a *rad27* null mutant shows temperature-sensitive lethality (210, 211). Interestingly, a novel type of mutation can be generated in cells lacking FEN1 activity. Sequences of 5–108 nucleotide pairs that are flanked by direct repeats of 3–12 nucleotide pairs become duplicated at high frequency in *rad27* null mutants (219). This unique mutation event is thought to result from a defect in lagging-strand DNA synthesis, causing an increased recombination rate due to induced double-strand-break repair of these lesions (220, 221).

RNase H activities in eukaryotic cells have been grouped into two classes, I and II, based on molecular mass of the enzyme and requirements for a cofactor (222). Among them, RNase HI is thought to be involved in the removal of RNA primers during Okazaki fragment synthesis (200, 206, 223–225). While the enzymatic activity of RNase HI has been known for a long time, its precise molecular weight and subunit structure are still unclear. Nevertheless, biochemical characterizations of RNase HI have revealed a unique substrate specificity; it could endonucleolytically cleave RNA that is attached to the 5′-end of a DNA strand, such as in an Okazaki fragment, leaving a single ribonucleotide on the 5′-end of the DNA strand (200, 226–228).

## DNA Helicases

DNA helicases are enzymes that promote the processive unwinding of duplex DNA, such as occurs at the DNA replication fork to create templates for the polymerases. During SV40 DNA replication, the virus-encoded T antigen functions as the replicative DNA helicase, but for cell chromosome replication, the nature of the replicative helicase remains unclear. Most DNA helicases required for viral or prokaryotic DNA replication form a homomultimeric complex (hexamer in most of the cases) (for review see 229). For example, T antigen functions as a hexamer and assembles at the SV40 ori as a do-decamer (two hexamers; 48, 49, 229). As DNA replication proceeds, the two hexamers at the divergent replication forks probably stay connected, forming part of a so-called replisome that the DNA passes through (230). It is highly likely, however, that several distinct helicases would function in cell chromosomal replication and also that the role of an individual helicase might be specialized for certain steps in the replication process. Many eukaryotic cell DNA helicases have been identified,

734 WAGA & STILLMAN

including the helicases associated with pol δ/ε or RFC (see 229), but only a few have been implicated in DNA replication.

DNA2 HELICASE Yeast Dna2 helicase was identified by screening for replication-defective mutants using a DNA replication assay in permeabilized cells (231, 232). The DNA2 gene encodes a 172-kDa protein that is essential for cell viability, and the purified protein shows both DNA-dependent ATPase and 3′-to-5′ DNA helicase activities (232, 233). Analysis of ³H-labeled replication intermediates from wild-type and dna2 temperature-sensitive mutant cells showed that low-molecular-weight intermediates accumulate in the dna2 mutants but not in wild-type cells, indicating that the defect is at the elongation stage of DNA replication (232).

MOUSE HELICASE B This mammalian helicase was identified through studies of a temperature-sensitive mutant mouse cell line, tsFT848, which was shown to be defective in DNA replication (234–236). DNA synthesis, but not RNA and protein syntheses, in these mutant cells decreased at the nonpermissive temperature. Comparative analyses of DNA-dependent ATPase activities in fractionated extracts from wild-type and mutant cells showed that one of the major ATPase activities, now designated helicase B, is decreased in the mutant cells. Furthermore, this helicase activity from mutant cells showed heat sensitivity. The DNA chain-elongation rate in the mutant cells, when analyzed by fiber autoradiography, was the same in both wild-type and mutant cells, suggesting that helicase B might be involved in a process that does not determine the elongation rate of the fork.

MINI-CHROMOSOME MAINTENANCE PROTEINS The mini-chromosome maintenance (MCM) proteins were first identified by yeast genetic studies as proteins required for replication of plasmid DNAs containing cellular origins of DNA replication. Researchers have since determined that these proteins are essential components of the pre-replication complex established prior to the S phase at origins of DNA replication (reviewed in 237, 238). Six MCM proteins have been found in all eukaryotic cells examined to date, and they share a similar amino acid sequence motif called the MCM box, part of which contains a putative ATPase motif (239). Genes encoding MCM proteins have also been found in recent sequences from Archaea species, suggesting that they are ancient replication proteins. The MCM protein complex appears to be a hexamer containing equal amounts of each of the six proteins (239a). Two recent observations, when combined, suggest, but do not prove, that the MCM proteins function as a replicative DNA helicase at the cellular replication fork (240, 241). A complex of three MCM proteins (Mcm4, 6, and 7) is capable of displacing a short oligonucleotide from a larger single-stranded DNA in an ATP-dependent manner, suggesting

that it contains DNA helicase activity (241). In addition, some MCM proteins appear to be bound to different regions of a replicon in the yeast genome at different times throughout the S phase of the cell cycle, beginning with the origins of DNA replication where they are assembled in an ORC- and Cdc6-dependent manner prior to S-phase entry (2, 240). This suggests that the MCM proteins might track along the DNA with the DNA replication fork, perhaps acting to unwind the DNA or performing another essential function (240).

**Molecular Links Among FEN1, Dna2 Helicase, and DNA Polymerase α/Primase**

Accumulating evidence indicates that a multi-enzyme complex exists in cells that contains many of the activities discussed above (193, 242). In addition, biochemical and genetic studies of yeast DNA replication have made connections linking many replication proteins into a multi-protein complex that may function as a so-called replisome. Physical and genetic interactions between the Dna2 helicase and FEN1 have been demonstrated; both proteins co-purified and co-immunoprecipitated, and the overexpression of FEN1 suppressed the temperature-sensitive growth of a *dna2-1* mutant (243). Conversely, overexpression of *DNA2* suppressed the temperature-sensitive lethality of a *rad27* null mutant (defective in FEN1; 243). This FEN1-Dna2 helicase interaction may play an important role in maturation of the lagging strand.

In an independent study, another allele of *dna2* (*dna2-2*) was isolated in a genetic screen for mutants that show synthetic lethality with the *ctf4-Δ4* mutant (244). Ctf4 protein (Ctf4p), which is identical to Pob1p and Chl15p, was identified as a protein that bound to a pol α catalytic subunit-affinity column (245–247). The *ctf4* null mutant is viable but exhibits elevated chromosome loss, implying a function in some process of DNA metabolism (245, 246).

Cdc68p and Pob3p have also been identified as pol α-binding proteins (244). These proteins seem to compete with Ctf4p for binding to pol α. Biochemical and genetic interactions between pol α and these pol α-binding proteins [Ctf4p, Cdc68p (248–250) and Pob3p] have also been demonstrated (244). Furthermore, the *cdc68-1* mutation was also synthetic lethal with the *dna2-2* mutation (244). These genetic and biochemical data suggest that the synthesis of an RNA-DNA primer to start an Okazaki fragment and maturation of the Okazaki fragment might be coordinately carried out during lagging strand synthesis by a multi-protein complex containing pol α/primase, Ctf4p, Cdc68p, Pob3p, Dna2p helicase, and FEN1. Because FEN1 can bind directly to PCNA, this complex might also contain RFC and pol δ, consistent with biochemical observations (193, 217, 242).

Another allele of *RAD27* (*erc11-2*) was unexpectedly isolated in a genetic screen that sought to identify proteins that interacted with the G1 cyclins Cln1p

and Cln2p [erc (elevated requirement for CLN function) mutations; 251]. The rad27/erc11-2, cln1cln2 mutant strain arrested in the S phase at nonpermissive temperature and gradually lost viability. The temperature-sensitive lethality could be rescued by expression of CLN1 or CLN2 but not the other G1 cyclin, CLN3. Moreover, overexpression of DNA2 (referred to as SEL1 in the original paper) and CDC9 (DNA ligase) also rescue the temperature-sensitive lethality (251). Although it is unclear how Cln1p/2p can rescue the defect in replication, it is intriguing that both proteins that rescued the defect when overexpressed could interact with PCNA. These studies suggest that the G1 cyclins Cln1p and Cln2p might affect functions at the DNA replication fork.

# MECHANISMS OF DNA SYNTHESIS AT A REPLICATION FORK

The above-described studies on the replication proteins in eukaryotic cells have identified their biochemical functions and several specific interactions among these proteins. These interactions underlie the mechanism of DNA synthesis of both the leading and lagging strands at the DNA replication fork. The following sections describe our current understanding of how these proteins cooperate to replicate DNA.

## Primosome Assembly

One of the first steps after recognition of the origin of DNA replication and local unwinding of the DNA is to load the pol α/primase complex onto the DNA, a step called primosome assembly. Details about the role of T antigen in origin recognition and local unwinding of the ori have been reviewed elsewhere (229, 252). Primosome assembly normally involves a DNA helicase interacting with the pol α/primase, but a cellular helicase that functions in cellular DNA replication in the same way that T antigen functions during SV40 DNA replication has not been identified to date.

T antigen, pol α/primase, and RPA interact with each other and cooperate to initiate DNA synthesis at the SV40 ori (16, 27, 36, 60, 254). The protein-protein interactions that have been demonstrated are T antigen-pol α/primase (p70 and/or p180) (36, 255–258), RPA p70-primase (p48 and p58) (59), and RPA-T antigen (59, 60). In addition, the bovine papillomavirus E1 initiator and helicase protein also binds to pol α/primase in a manner analogous to T antigen (259). Primase assays with single-stranded template DNAs have shown that although RPA from either humans or yeast represses primase activity, T antigen can reverse the inhibition only when human RPA, but not *S. cerevisiae* RPA, is coating the DNA (60, 260). In addition, only human pol α/primase, but not calf thymus or mouse pol α/primase, can support primer synthesis in the presence

P1: DPI/plb P2: NBL/dat QC: NBL
April 29, 1998 13:49 Annual Reviews AR057-23

DNA REPLICATION IN EUKARYOTES 737

|  |  |
| --- | --- |
| 1 | RPA RNA primer previous Okazaki fragment Lagging strand template |
| 2 | pol α/primase Priming and DNA synthesis by pol α/primase |
| 3 | RFC RFC binding Displacement of pol α/primase |
| 4 | PCNA pol δ Assembly of pol δ complex |
| 5 | RNase HI/FEN1 Elongation Removal of RNA primer by RNase HI and FEN1 |
| 6 | Gap-filling |
| 7 | DNA ligase I Joining of Okazaki fragments |

Figure 2 Polymerase switching and maturation of Okazaki fragments on a lagging-strand DNA template. See text for details. Adapted from Reference 313.

of SV40 T antigen and RPA (261), indicating the existence of a species-specific protein-protein interaction during primosome assembly (30, 261, 262). These interactions that promote primosome assembly occur not only during initiation of DNA replication at the ori but also for the synthesis of each Okazaki fragment (Figure 2; 10, 26, 56, 200, 212, 260).

Mapping the start sites for RNA-DNA primer synthesis around the SV40 ori sequence shows some preference for primer-site selection in vivo and in the crude SV40 replication system (263–265; for review see 20). It has not been determined whether initiation with the purified pol α/primase, RPA, and T antigen occurs at preferred sites for primer synthesis. A number of cellular proteins that associate or cooperate with pol α/primase, such as AAF (266, 267), Ctf4 (Pob1) (245–247), Cdc68 (248–250), and Pob3 (244), might modulate the interaction of the primase with the DNA and affect primer-site selection. In addition, transcription factor activator domains can bind to RPA and stimulate

replication in vitro (268, 269), but the precise mechanism of this activation is unclear. In the context of cellular chromatin, these site-specific DNA-binding proteins may aide in recruiting RPA and hence pol α/primase to the DNA.

# Polymerase Switching

Biochemical studies using the SV40 DNA replication system reconstituted with purified proteins (21, 24, 26, 54, 190, 191) have shown that two different polymerases, pol α/primase and pol δ, are involved in DNA synthesis and that pol δ is involved in the synthesis of both the leading and lagging strands. The switching from pol α/primase to pol δ occurs during priming of the leading strand (24) and during synthesis of every Okazaki fragment (26). The involvement of two different DNA polymerases in lagging-strand DNA synthesis was suggested by the analysis of SV40 DNA replication in vivo with DNA polymerase inhibitors (15, 17). The mechanisms of initiation of leading strand synthesis and initiation of each Okazaki fragment are apparently very similar: An RNA-DNA primer is produced by pol α/primase, and the 3′-terminus of the initiator DNA (iDNA) is recognized by RFC and PCNA to expel the pol α/primase and load pol δ (Figure 2; 24–26).

As suggested by the model in Figure 2, pol α/primase starts the synthesis of an RNA-DNA primer on an RPA-coated, single-stranded DNA template, perhaps assisted by a putative cellular loading activity (such as T antigen), to yield a short 30-nt primer RNA-DNA (18, 264, 270). Once the RNA-DNA primer is synthesized, RFC binds to the 3′-end of the iDNA, displacing pol α/primase. The turnover of pol α/primase most likely occurs by the inherent nonprocessive nature of the pol α catalytic activity and the tight binding of RFC to the primer-template junction, since both RPA and RFC decrease the length of the iDNA (25). RFC binding triggers the assembly of the primer recognition complex, which is accomplished through the loading of PCNA and subsequent association of PCNA with pol δ. Then the relatively processive pol δ holoenzyme extends the DNA strand (16, 23–25, 191). For the initiation of leading-strand DNA replication, the synthesis by the pol δ/PCNA complex is then processive and continuous, at least for 5–10 kb of DNA. For synthesis on the lagging strand, DNA synthesis of the Okazaki fragment continues until the polymerase encounters the previously synthesized Okazaki fragment. The RNA primer from the preceding Okazaki fragment is then removed by a complex processing reaction described below, and the remaining nick is sealed by DNA ligase I.

Biochemical studies in vitro (described in this review) and studies in vivo (195) show that pol ε is not required for SV40 DNA replication. But given the enzymatic similarities between pol δ and pol ε, such as high processivity, and the observation that polymerase ε is essential for DNA replication (196, 197), pol ε

almost certainly participates in cellular chromosomal replication. A difference between these two systems is the mechanism of initiation, because many cellular proteins are required to achieve what is achieved by T antigen. Therefore, pol ε might be involved in the initiation of DNA replication. Alternatively, each polymerase might be involved specifically in a separate process during DNA replication. A genetic study utilizing *S. cerevisiae* strains that contain mutations in the proofreading exonuclease domains of either pol δ or pol ε suggests that each polymerase is involved in replicating different strands of the DNA (271).

Polymerase ε is also a unique polymerase in that it is involved in cell-cycle checkpoint control and may therefore function at the DNA replication fork to ensure accurate DNA synthesis, perhaps by a postreplicative repair mechanism or another, as yet unrecognized, mechanism (272, 273). It is also possible that pol ε functions in replicating large chromosomes as a specialized enzyme for initiation of DNA replication at sites where DNA synthesis has halted temporarily because the replication fork has to cope with the complex topology of the cell's chromosomes. Clearly, more studies on the role of pol ε are required.

A key protein in the polymerase switching appears to be RFC, and the loading of PCNA by RFC is an essential event for the transition from a priming mode to an extension mode of DNA synthesis. Given the functional similarities between RFC and the *E. coli* pol III γ-complex (95), it is possible that RFC coordinates the synthesis of both the lagging and leading strands. Further investigation of the mechanism for coordinating DNA synthesis of both strands at the eukaryotic cell DNA replication fork is necessary.

## Maturation of Okazaki Fragments

In maturation of Okazaki fragment synthesis, the short Okazaki fragments synthesized discontinuously on the lagging strand template are converted into long, ungapped DNA products. This involves several distinct steps including removal of the RNA primer, DNA gap synthesis, and sealing together of the two DNA. Recent studies, and the observation that many of the proteins involved in this process bind to PCNA, suggest that these steps may be regulated coordinately with each other.

The analysis of lagging-strand DNA synthesis using highly purified proteins has revealed the basis of the maturation mechanism. Two different nucleases, RNase HI and FEN1, are involved in the complete removal of the RNA primer (192, 200, 205, 206, 212–216). These nucleases are required for the complete replication of SV40 DNA and for reconstitution of lagging strand synthesis on artificial templates (26, 192, 200). An in vitro assay with a model substrate showed that PCNA binds to FEN1 and stimulates FEN1 activity (152). This observation suggests that the removal of RNA (or RNA-DNA) might be triggered either by the upstream DNA polymerase complex or by the newly synthesized

DNA, creating a duplex DNA region upstream of the RNA at the 5′-end of the Okazaki fragment. Consistent with the latter possibility, an assay for FEN1 using a synthetic oligonucleotide substrate showed that an upstream DNA can influence the cleavage of a downstream flap substrate (204, 213–215).

Okazaki fragment processing and genome integrity during cell chromosome replication might require additional proteins, such as the Dna2 helicase, to complete the same process. Dna2 helicase has been suggested to be involved in removal of RNA primers because of its biochemical and genetic interactions with FEN1/Rad27 nuclease (243; also see above). Dna2 helicase, in conjunction with the polymerase complex that synthesizes the Okazaki fragments, might displace the RNA primer from the template DNA, thereby creating a flap-like substrate for FEN1 endonuclease (see FEN1 and RNase HI section, Figure 3, and Reference 10). A more interesting possibility is that in addition to the RNA at the 5′-end of an Okazaki fragment, the DNA beyond the RNA-DNA

![Diagram](attachment:diagram.png)

Figure 3 Two mechanisms for the removal of RNA primers. (A) RNase HI cleaves the RNA segment attached to the 5′-end of the Okazaki fragment, leaving a single ribonucleotide adjacent to the RNA-DNA junction. FEN1 then removes the remaining ribonucleotide. (B) Dna2 helicase displaces the RNA segment (or RNA-DNA). FEN1 then cleaves endonucleolytically the branch point, releasing the displaced RNA (or RNA-DNA). Although the Dna2/FEN1-dependent mechanism has not been proved, recent biochemical and genetic studies strongly support this model (see 10).

junction that was synthesized by DNA pol α might also be displaced by Dna2 helicase and cleaved off by FEN1 (10). If this were the case, then the iDNA that was made by pol α would be removed by this Okazaki maturation process, and the gap created would be filled in by either pol δ or pol ε. This would be a significant advantage for cells for maintaining genome integrity because pol α/primase does not have a proofreading activity and thus could not remove any errors that it inserted in the iDNA, whereas if this region of the template DNA were “re-replicated” by pol δ or pol ε, then the proofreading exonucleases from these enzymes would ensure increased accuracy during replication. This mechanism for Okazaki fragment maturation might also prevent inappropriate DNA replication of repeated regions in the genome, such as mini-satellite repeated sequences, a common occurrence in human cancer cells (219, 220).

# REPLICATION FORK PROTEINS AND CELL-CYCLE CONTROL

## S-Phase Checkpoint Control

When the genome is subjected to excessive DNA damage, or progression of DNA replication forks is blocked, cells arrest progression of the cell cycle at either the G1-S phase transition, the G2-M phase transition, or they slow down S-phase progression. This activates transcription of genes that encode proteins required for the repair of DNA and facilitates the repair process itself (reviewed in 274). The signal transduction mechanisms that detect aberrant replication or DNA damage in the S phase, and then block cell-cycle progression are called S-phase checkpoints. Previous genetic studies in *S. cerevisiae* and *S. pombe* suggest that several replication fork proteins might be involved in these S-phase checkpoints.

In *S. cerevisiae*, mutant cells that have a defect in pol ε (*pol2*) fail to activate the damage-inducible transcription of certain damage responsive genes in response to DNA damage by methyl methane sulfonate (MMS), or nucleotide depletion induced by hydroxyurea during the S phase. The mutant cells enter into the M phase without correctly completing DNA replication (272). Thus *POL2* might be involved in an S-phase checkpoint signaling mechanism. The domain in pol ε that is responsible for its checkpoint function is separable from its DNA polymerase catalytic domain (272). Recent studies suggest that the Rad53p protein kinase is required for DNA-damage checkpoint signaling by a pol ε- and Rfc5p-dependent mechanism (129, 273, 275).

*DPB11*, which was isolated as a multicopy suppressor of mutations in genes encoding subunits of *S. cerevisiae* pol ε (*pol2* and *dpb2*), has a checkpoint function (276). A temperature-sensitive mutant in *DPB11* underwent cell division

742 WAGA & STILLMAN

without completion of replication at the nonpermissive temperature, and the mutant cells were sensitive to hydroxyurea, MMS, and UV irradiation. A similar defect was demonstrated in mutants in the cut5+ gene in *S. pombe*, although these mutants were sensitive only to DNA damage and not to nucleotide depletion (277, 278). A recent report showed that the Cut5 protein, which is similar in sequence to Dpb11p from *S. cerevisiae*, binds to the Chkl protein kinase that is known to control the cyclin-dependent Cdc2 kinase, the principal regulator for M-phase entry (279).

Other replication proteins have been implicated in checkpoint controls. Yeast mutants with defects in genes encoding RPA (*S. pombe rad11*) (280), primase (*S. cerevisiae pril*) (281), *S. pombe pol α* and *pol δ* (282–284), and an RFC small subunit (*S. cerevisiae rfc5*) (275, 285) have also been shown to exhibit checkpoint defects. Some of these defects may occur because a replication fork is not established after commitment to cell division, and others may be the result of a real defect in signaling in response to DNA damage or a replication fork block. The primase mutations are particularly interesting because they override a mechanism that slows down S-phase progression in the presence of continuous low doses of a DNA alkylating agent (281). Interestingly, loss of the *MECl* gene, which is related to the ATM gene that is defective in human ataxia telangiectasia, also causes the same effect (286), suggesting that primase might be part of a signaling mechanism to the Mec1p checkpoint pathway.

### PCNA-p21 Interaction

The involvement of replication fork proteins in checkpoint signaling in metazoan species is likely, but additional controls may be imposed on DNA replication in these cells. The identification of proteins that interact specifically with PCNA has led to the emergence of PCNA as a key protein required for the coordinated regulation of replication and other events that take place at the replication fork, such as DNA methylation and DNA repair. In addition, overexpression of PCNA in *S. pombe* results in the delay of entry into the M phase (287), perhaps because PCNA is binding a protein that is essential for this process.

One of the well-characterized PCNA-interacting proteins is the p21 protein, an inhibitor for cyclin-dependent kinase (CDK) (p21 has alternate names of CIP1, WAF1, Sdi1; for review see 288, 289). p21 is induced during mitogenic stimulation of mammalian cells and in response to DNA damage by the tumor suppression protein p53, as well as many other stimuli. The protein binds to and inhibits cyclin-dependent kinase (CDK) activities that are required for G1/S progression, leading to cell-cycle arrest. Thus p21 functions as part of a DNA-damage checkpoint mechanism.

DNA REPLICATION IN EUKARYOTES 743

Interest in p21 with respect to the DNA replication fork was triggered by the observation that p21 can form a quaternary complex with a CDK, cyclin, and PCNA in normal cells but not in many transformed cells (162, 290), creating a possible link between cell-cycle progression and DNA replication. p21 inhibits SV40 DNA replication in vitro and in frog cell extracts through its direct binding to PCNA (155, 156, 291, 292). Even though PCNA is essential for nucleotide-excision repair (293, 294), p21 did not inhibit the repair in vitro (295, 296) and in vivo (168). These results suggest that through its binding to both CDK kinase and PCNA, p21 might function to coordinately regulate DNA replication, repair, and cell-cycle progression in response to DNA damage, perhaps inducing a switch from replication to repair.

The biological significance of this interaction is still unclear, even though the p21-PCNA interaction has been characterized extensively, including mapping of the binding domain in p21 (297–303), determination of a crystal structure of PCNA complexed with a p21-derived peptide (164), and investigation of the effect of p21 on DNA synthesis and the loading of PCNA onto DNA (115, 304). A recent observation that p21 modulates the interactions between PCNA and FEN1 endonuclease, DNA (5-cytosine) methyltransferase, and DNA ligase suggests that the levels of p21 in the cell might control a switch from one PCNA-dependent function to another at the DNA replication fork (154, 160, 305). More evidence that the p21-PCNA interaction might be relevant in vivo is provided by the recent observations that the E7 oncoprotein from human papillomavirus type 16 can abrogate a DNA damage-induced cell-cycle arrest by binding to p21 (306, 307). These studies show that the p21-E7 interaction results in both the reversal of p21 inhibition of CDK kinase activity and the p21 inhibition of PCNA function (306).

### Regulation of Telomere Length

The appropriate regulation of telomere length and replication is also important for maintaining the integrity of the genome. In addition, the length of telomeres correlates with the replicative potential of cells (for review see 308). Recent studies have indicated that in addition to telomerase, replication fork proteins may directly regulate the length of telomeres. The mutations in the *S. cerevisiae POLI/CDC17* gene encoding polymerase α, as well as the gene encoding the large subunit of RFC (*CDC44* in *S. cerevisiae*), cause elongation of the telomeres (309, 310). Furthermore, pharmacological inhibitors of cellular DNA polymerases disrupt coordinated DNA replication of the G-rich strand that is synthesized by telomerase and the C-rich lagging strand (311). Given that this telomere elongation requires telomerase activity and that both pol α/primase and RFC are involved in lagging strand synthesis, both telomere extension and lagging strand synthesis might be regulated coordinately, in

744 WAGA & STILLMAN

conjunction with the function of CDC13, a telomere-sequence-specific, single-stranded DNA-binding protein that is required for telomere maintenance (312). Further biochemical investigation of the mechanism for telomere replication is needed to understand how both strands of the telomere are replicated.

# CONCLUDING REMARKS

Modern research on the replication of DNA in eukaryotes can be grouped into three related categories. One active area of research is to characterize the DNA sequences that are important for replicator function and determine where origins of DNA replication occur in chromosomes. Another area is to identify the proteins involved in the mechanism and control of the initiation of DNA replication in eukaryotes. This area is the most active area at the present time, providing interesting links to cell-cycle research, developmental biology, control of gene expression, and chromosome structure and function. A third area is to extend the already remarkable progress in understanding the events at the DNA replication fork, outlined above, to identify other proteins that may cooperate with the known replication fork proteins to ensure accurate and efficient DNA replication. Connections need to be made between the initiation proteins that establish the replication fork and those proteins discussed above that function to replicate DNA after initiation. Proteins that function at the replication fork will likely play significant roles in inheritance of chromosome structures by interacting with other proteins involved in chromatin assembly and epigenetically inherited protein complexes. Clearly much more work needs to be done.

## ACKNOWLEDGMENTS

We thank John Kuryan for Figure 1, Dr. Robert Bambara for discussions, and colleagues who provided preprints of research papers. Our work was supported by a grant from the National Cancer Institute (CA13106).

Visit the Annual Reviews home page at  
http://www.AnnualReviews.org.

## Literature Cited

1. Diffley JFX. 1996. *Genes Dev.* 10:2819–30
2. Stillman B. 1996. *Science* 274:1659–64
3. Li JJ, Kelly TJ. 1984. *Proc. Natl. Acad. Sci.* USA 81:6973–77
4. Li JJ, Kelly TJ. 1985. *Mol. Cell. Biol.* 5:1238–46
5. Stillman BW, Gluzman Y. 1985. *Mol. Cell. Biol.* 5:2051–60
6. Wobbe CR, Dean F, Weissbach L, Hurewitz J. 1985. *Proc. Natl. Acad. Sci. USA* 82:5710–14
7. Challberg MD, Kelly TJ. 1989. *Annu. Rev. Biochem.* 58:671–717

8. Stillman B. 1989. *Annu. Rev. Cell. Biol.* 5:197–245
9. Bambara RA, Huang L. 1995. *Prog. Nucleic Acid Res. Mol. Biol.* 51:93–122
10. Bambara RA, Murante RS, Henricksen LA. 1997. *J. Biol. Chem.* 272:4647–50
11. Lindahl T, Barnes DE. 1992. *Annu. Rev. Biochem.* 61:251–81
12. Nash R, Lindahl T. 1996. See Ref. 314, pp. 575–86
13. Andersen AH, Bendixen C, Westergaard O. 1996. See Ref. 314, pp. 587–617
14. Wang TS-F. 1996. See Ref. 314, pp. 461–93
15. Nethanel T, Reisfeld S, Dinter-Gottlieb G, Kaufmann G. 1988. *J. Virol.* 6:2867–73
16. Matsumoto T, Eki T, Hurwitz J. 1990. *Proc. Natl. Acad. Sci. USA* 87:9712–16
17. Nethanel T, Kaufmann G. 1990. *J. Virol.* 64:5912–18
18. Bullock PA, Seo YS, Hurwitz J. 1991. *Mol. Cell. Biol.* 11:2350–61
19. Murakami Y, Eki T, Hurwitz J. 1992. *Proc. Natl. Acad. Sci. USA* 89:952–56
20. Salas M, Miller JT, Leis J, DePamphilis ML. 1996. See Ref. 314, pp. 131–76
21. Prelich G, Stillman B. 1988. *Cell* 53: 117–26
22. Lee S-H, Eki T, Hurwitz J. 1989. *Proc. Natl. Acad. Sci. USA* 86:7361–65
23. Weinberg DH, Kelly TJ. 1989. *Proc. Natl. Acad. Sci. USA* 86:9742–46
24. Tsurimoto T, Melendy T, Stillman B. 1990. *Nature* 346:534–39
25. Tsurimoto T, Stillman B. 1991. *J. Biol. Chem.* 266:1961–68
26. Waga S, Stillman B. 1994. *Nature* 369: 207–12
27. Murakami Y, Hurwitz J. 1993. *J. Biol. Chem.* 268:11008–17
28. Wang TS-F. 1991. *Annu. Rev. Biochem.* 60:513–52
29. Stadlbauer F, Brueckner A, Rehfuess C, Eckerskorn C, Lottspeich F, et al. 1994. *Eur. J. Biochem.* 222:781–93
30. Brückner A, Stadlbauer F, Guarino LA, Brunahl A, Schneider C, et al. 1995. *Mol. Cell. Biol.* 15:1716–24
31. Longhese MP, Jovine L, Plevani P, Lucchini G. 1993. *Genetics* 133:183–91
32. Copeland WC, Tan X. 1995. *J. Biol. Chem.* 270:3905–13
33. Longhese MP, Fraaschini R, Plevani P, Lucchini G. 1996. *Mol. Cell. Biol.* 16:3235–44
34. Santocanale C, Foiani M, Lucchini G, Plevani P. 1993. *J. Biol. Chem.* 268: 1343–48

35. Bakkenist CJ, Cotterill S. 1994. *J. Biol. Chem.* 269:26759–66
36. Collins KL, Russo AAR, Tseng BY, Kelly TJ. 1993. *EMBO J.* 12:4555–66
37. Foiani M, Marini F, Gamba D, Lucchini G, Plevani P. 1994. *Mol. Cell. Biol.* 14:923–33
38. Nasheuer H-P, Moore A, Wahl AF, Wang TS-F. 1991. *J. Biol. Chem.* 266:7893–903
39. Foiani M, Liberi G, Lucchini G, Plevani P. 1995. *Mol. Cell. Biol.* 15:883–91
40. Ferrari M, Lucchini G, Plevani P, Foiani M. 1996. *J. Biol. Chem.* 271:8661–66
41. Voitenleitner C, Fanning E, Nasheuer H-P. 1997. *Oncogene* 14:1611–15
42. Hübscher U, Maga G, Podust VN. 1996. See Ref. 314, pp. 525–43
43. Wold MS. 1997. *Annu. Rev. Biochem.* 66:61–92
44. Wobbe CR, Weissbach L, Borowiec JA, Dean FB, Murakami Y, et al. 1987. *Proc. Natl. Acad. Sci. USA* 84:1834–38
45. Fairman MP, Stillman B. 1988. *EMBO J.* 7:1211–18
46. Wold MS, Kelly T. 1988. *Proc. Natl. Acad. Sci. USA* 85:2523–27
47. Sancar A. 1996. *Annu. Rev. Biochem.* 65:43–81
48. Stahl H, Droge P, Knippers R. 1986. *EMBO J.* 5:1939–44
49. Dean FB, Bullock P, Murakami Y, Wobbe CR, Weissbach L, Hurwitz J. 1987. *Proc. Natl. Acad. Sci. USA* 84:16–20
50. Dodson M, Dean FB, Bullock P, Echols H, Hurwitz J. 1987. *Science* 238:964–67
51. Wold MS, Li JJ, Kelly TJ. 1987. *Proc. Natl. Acad. Sci. USA* 84:3643–47
52. Brill SJ, Stillman B. 1989. *Nature* 342: 92–95
53. Kenny MK, Lee S-H, Hurwitz J. 1989. *Proc. Natl. Acad. Sci. USA* 86:9757–61
54. Tsurimoto T, Stillman B. 1989. *EMBO J.* 8:3883–89
55. Erdile LF, Heyers W-D, Kolodner R, Kelly TJ. 1991. *J. Biol. Chem.* 266: 12090–98
56. Tsurimoto T, Stillman B. 1991. *J. Biol. Chem.* 266:1950–60
57. Braun KA, Lao Y, He Z, Ingles CJ, Wold MS. 1997. *Biochemistry* 36:8443–54
58. Longhese MP, Plevani P, Lucchini G. 1994. *Mol. Cell. Biol.* 14:7884–90
59. Dornreiter I, Erdile LF, Gilbert IU, von Winkler D, Kelly TJ, Fanning E. 1992. *EMBO J.* 11:769–76
60. Melendy T, Stillman B. 1993. *J. Biol. Chem.* 268: 3389–95

746 WAGA & STILLMAN

61. Ishiai M, Sanchez JP, Amin AA, Murakami Y, Hurwitz J. 1996. *J. Biol. Chem.* 271:20868–78  
62. Henricksen LA, Umbricht CB, Wold MS. 1994. *J. Biol. Chem.* 269:11121–32  
63. He Z, Wong JMS, Maniar HS, Brill SJ, Ingles CJ. 1996. *J. Biol. Chem.* 271:28243–49  
64. Stigger E, Dean FB, Hurwitz J, Lee S-H. 1994. *Proc. Natl. Acad. Sci. USA* 91:579–83  
65. Gomes XV, Wold MS. 1995. *J. Biol. Chem.* 270:4534–43  
66. Kim DK, Stigger E, Lee S-H. 1996. *J. Biol. Chem.* 271:15124–29  
67. Lin Y-L, Chen C, Keshav KF, Winchester E, Dutta A. 1996. *J. Biol. Chem.* 271:17190–98  
68. Philipova D, Mullen JR, Maniar HS, Lu JA, Gu CY, Brill SJ. 1996. *Genes Dev.* 10:2222–33  
69. Bochkarev A, Pfuetzner RA, Edwards AM, Frappier L. 1997. *Nature* 385:176–81  
70. Blackwell LJ, Borowiec JA. 1994. *Mol. Cell. Biol.* 14:3993–4001  
71. Kim C, Paulus BF, Wold MS. 1994. *Biochemistry* 33:14197–206  
72. Mitas M, Chock JY, Christy M. 1997. *Biochem. J.* 324:957–61  
73. Din S, Brill S, Fairman MP, Stillman B. 1990. *Genes Dev.* 4:968–77  
74. Liu VF, Weaver DT. 1993. *Mol. Cell. Biol.* 13:7222–31  
75. Carty MP, Zernik-Kobak M, McGrath S, Dixon K. 1994. *EMBO J.* 13:2114–23  
76. Dutta A, Stillman B. 1992. *EMBO J.* 11:2189–99  
77. Fotedar R, Roberts JM. 1992. *EMBO J.* 11:2177–87  
78. Brush GS, Anderson CW, Kelly TJ. 1994. *Proc. Natl. Acad. Sci. USA* 91:12520–24  
79. Pan Z-Q, Amin AA, Gibbs E, Niu H, Hurwitz J. 1994. *Proc. Natl. Acad. Sci. USA* 91:8343–47  
80. Boubnov NV, Weaver DT. 1995. *Mol. Cell. Biol.* 15:5700–6  
81. Fried LM, Koumenis C, Peterson SR, Green SL, van Zijl P, et al. 1996. *Proc. Natl. Acad. Sci. USA* 93:13825–30  
82. Gibbs E, Pan Z-Q, Niu H, Hurwitz J. 1996. *J. Biol. Chem.* 271:22847–54  
83. Niu H, Erdjument-Bromage H, Pan Z-Q, Lee S-H, Tempst P, Hurwitz J. 1997. *J. Biol. Chem.* 272:12634–41  
84. Zernik-Kobak M, Vasunia K, Connelly M, Anderson CW, Dixon K. 1997. *J. Biol. Chem.* 272:23896–904  

85. Brush GS, Morrow DM, Hieter P, Kelly TJ. 1996. *Proc. Natl. Acad. Sci. USA* 93:15075–80  
86. Cheng XB, Cheong N, Wang Y, Ilakis G. 1996. *Radiother. Oncol.* 39:43–52  
87. Abramova NA, Russell J, Botchan M, Li R. 1997. *Proc. Natl. Acad. Sci. USA* 94:7186–91  
88. Henricksen LA, Wold MS. 1994. *J. Biol. Chem.* 269:24203–8  
89. Pan Z-Q, Park CH, Amin AA, Hurwitz J, Sancar A. 1995. *Proc. Natl. Acad. Sci. USA* 92:4636–40  
90. Henricksen LA, Carter T, Dutta A, Wold MS. 1996. *Nucleic Acids Res.* 24:3107–12  
91. Morgan SE, Kastan MB. 1997. *Cancer Res.* 57:3386–89  
92. Adachi Y, Laemmli U. 1992. *J. Cell Biol.* 119:1–16  
93. Yan H, Newport J. 1995. *Science* 269:1883–85  
94. Coleman TR, Carpenter PB, Dunphy WG. 1996. *Cell* 87:53–63  
95. Stillman B. 1996. See Ref. 314, pp. 435–60  
96. Chen M, Pan Z-Q, Hurwitz J. 1992. *Proc. Natl. Acad. Sci. USA* 89:5211–15  
97. Chen M, Pan Z-Q, Hurwitz J. 1992. *Proc. Natl. Acad. Sci. USA* 89:2516–20  
98. Bunz F, Kobayashi R, Stillman B. 1993. *Proc. Natl. Acad. Sci. USA* 90:11014–18  
99. Burbelo PD, Utani A, Pan Z-Q, Yamada Y. 1993. *Proc. Natl. Acad. Sci. USA* 90:11543–47  
100. Howell EA, McAlear MA, Rose D, Holm C. 1994. *Mol. Cell. Biol.* 14:255–67  
101. Li XY, Burgers PMJ. 1994. *Proc. Natl. Acad. Sci. USA* 91:868–72  
102. Li XY, Burgers PMJ. 1994. *J. Biol. Chem.* 269:21880–84  
103. Noskov V, Maki S, Kawasaki Y, Leem S-H, Ono B-I, et al. 1994. *Nucleic Acids Res.* 22:1527–35  
104. Cullmann G, Fien K, Kobayashi R, Stillman B. 1995. *Mol. Cell. Biol.* 15:4661–71  
105. Bork P, Hofmann K, Bucher P, Neuwald AF, Altschul SF, Koonin EV. 1997. *FASEB J.* 11:68–76  
106. O'Donnell M, Onrust R, Dean FB, Chen M, Hurwitz J. 1993. *Nucleic Acids Res.* 21:1–3  
107. Tsurimoto T, Stillman B. 1989. *Mol. Cell. Biol.* 9:609–19  
108. Tsurimoto T, Stillman B. 1990. *Proc. Natl. Acad. Sci. USA* 87:1023–27

109. Lee S-H, Kwong AD, Pan Z-Q, Hurwitz J. 1991. *J. Biol. Chem.* 266:594–602  
110. Yoder BL, Burgers PMJ. 1991. *J. Biol. Chem.* 266:22689–97  
111. Fien K, Stillman B. 1992. *Mol. Cell. Biol.* 12:155–63  
112. Podust VN, Georgaki A, Strack B, Hubscher U. 1992. *Nucleic Acids Res.* 20:4159–65  
113. Burgers PMJ. 1991. *J. Biol. Chem.* 266:22698–706  
114. Cai JS, Uhlmann F, Gibbs E, Flores-Rozas H, Lee C-G, et al. 1996. *Proc. Natl. Acad. Sci. USA* 93:12896–901  
115. Podust VN, Podust LM, Goubin F, Ducommun B, Hubscher U. 1995. *Biochemistry* 34:8869–75  
116. Lee S-H, Hurwitz J. 1990. *Proc. Natl. Acad. Sci. USA* 87:5672–76  
117. Podust VN, Fanning E. 1997. *J. Biol. Chem.* 272:6303–10  
118. Ellison V, Stillman B. 1998. *J. Biol. Chem.* 273:5979–87  
119. Gerik KJ, Gary SL, Burgers PMJ. 1997. *J. Biol. Chem.* 272:1256–62  
120. Fotedar R, Mossi R, Fitzgerald P, Roussel T, Maga G, et al. 1996. *EMBO J.* 15:4423–33  
121. Mossi R, Jonsson Z, Allen BL, Hardin SH, Hubscher U. 1997. *J. Biol. Chem.* 272:1769–76  
122. Uhlmann F, Cai JS, Gibbs E, O'Donnell M, Hurwitz J. 1997. *J. Biol. Chem.* 272:10058–64  
123. Rheaume E, Cohen LY, Uhlmann F, Lazure C, Alam A, et al. 1997. *EMBO J.* 16:6346–54  
124. Song QZ, Lu H, Zhang N, Luckow B, Shah G, et al. 1997. *Biochem. Biophys. Res. Commun.* 233:343–48  
125. Uhlmann F, Gibbs E, Cai JS, O'Donnell M, Hurwitz J. 1997. *J. Biol. Chem.* 272:10065–71  
126. Uhlmann F, Cai JS, Flores-Rozas H, Dean FB, Finkelstein J, et al. 1996. *Proc. Natl. Acad. Sci. USA* 93:6521–26  
127. Cai JS, Gibbs E, Uhlmann F, Phillips B, Yao N, et al. 1997. *J. Biol. Chem.* 272:18974–81  
128. McAlear MA, Howell EA, Espenshade KK, Holm C. 1994. *Mol. Cell. Biol.* 14:4390–97  
129. Sugimoto K, Ando S, Shimomura T, Matsumoto K. 1997. *Mol. Cell. Biol.* 17:5905–14  
130. Guenther B, Onrust R, Sali A, O'Donnell M, Kuriyan J. 1997. *Cell* 91:335–45  
131. Jonsson ZO, Hubscher U. 1997. *BioEssays* 19:967–75  

132. Kelman Z. 1997. *Oncogene* 14:629–40  
133. Kong X-P, Onrust R, O'Donnell M, Kuriyan J. 1992. *Cell* 69:425–37  
134. Krishna TSR, Kong X-P, Gary S, Burgers PM, Kuriyan J. 1994. *Cell* 79:1233–43  
135. Kelman Z, O'Donnell M. 1995. *Annu. Rev. Biochem.* 64:171–200  
136. Tinker RL, Kassavetis GA, Geiduschek EP. 1994. *EMBO J.* 13:5330–37  
137. Hacker KJ, Alberts BM. 1994. *J. Biol. Chem.* 269:24209–20  
138. Hacker KJ, Alberts BM. 1994. *J. Biol. Chem.* 269:24221–28  
139. Stukenberg PT, Turner J, O'Donnell M. 1994. *Cell* 78:877–87  
140. Tan CK, Castillo C, So AG, Downey KM. 1986. *J. Biol. Chem.* 261:12310–16  
141. Bravo R, Frank R, Blundell PA, MacDonald-Bravo H. 1987. *Nature* 326:515–17  
142. Prelich G, Kostura M, Marshak DR, Mathews MB, Stillman B. 1987. *Nature* 326:471–75  
143. Prelich G, Tan CK, Kostura M, Matthews MB, So AG, et al. 1987. *Nature* 326:517–20  
144. Bauer GA, Burgers PMJ. 1988. *Proc. Natl. Acad. Sci. USA* 85:7506–10  
145. Hamatake RK, Hasegawa H, Clark AB, Bebenek K, Kunkel TA, Sugino A. 1990. *J. Biol. Chem.* 265:4072–83  
146. Lee S-H, Zhen-Qiang P, Kwong AD, Burgers PMJ, Hurwitz J. 1991. *J. Biol. Chem.* 266:22707–17  
147. Chui G, Linn S. 1995. *J. Biol. Chem.* 270:7799–808  
148. Ayyagari R, Impellizzeri KJ, Yoder BL, Gary SL, Burgers PMJ. 1995. *Mol. Cell. Biol.* 15:4420–29  
149. Fukuda K, Morioka H, Imajou S, Ikeda S, Ohtsuka E, Tsurimoto T. 1995. *J. Biol. Chem.* 270:22527–34  
150. Arroyo MP, Downey KM, So AG, Wang TS-F. 1996. *J. Biol. Chem.* 271:15971–80  
151. Eissenberg JC, Ayyagari R, Gomes XV, Burgers PM. 1997. *Mol. Cell. Biol.* 17:6367–78  
152. Li XY, Li J, Harrington J, Lieber MR, Burgers PMJ. 1995. *J. Biol. Chem.* 270:22109–12  
153. Chen JJ, Chen S, Saha P, Dutta A. 1996. *Proc. Natl. Acad. Sci. USA* 93:11597–602  
154. Levin DS, Bai W, Yao N, O'Donnell M, Tomkinson AE. 1997. *Proc. Natl. Acad. Sci. USA* 94:12863–68  
155. Flores-Rozas H, Kelman Z, Dean FB,

748 WAGA & STILLMAN

156. Pan Z-Q, Harper JW, et al. 1994. *Proc. Natl. Acad. Sci. USA* 91:8655–59  
157. Waga S, Hannon GJ, Beach D, Stillman B. 1994. *Nature* 369:574–78  
158. Smith ML, Chen I-T, Zhan QM, Bae IS, Chen C-Y, et al. 1994. *Science* 266: 1376–80  
159. Hall PA, Kearsley JM, Coates PJ, Norman DG, Warbrick E, Cox LS. 1995. *Oncogene* 10:2427–33  
160. Gary R, Ludwig DL, Cornelius HL, MacInnes MA, Park MS. 1997. *J. Biol. Chem.* 272:24522–29  
161. Chuang LS-H, Ian H-I, Koh T-W, Ng H-H, Xu GL, Li BFL. 1997. *Science* 277: 1996–2000  
162. Umar A, Buermeyer AB, Simon JA, Thomas DC, Clark AB, et al. 1996. *Cell* 87:65–73  
163. Xiong Y, Zhang H, Beach D. 1992. *Cell* 71:505–14  
164. Matsuoka S, Yamaguchi M, Matsukage A. 1994. *J. Biol. Chem.* 269:11030–36  
165. Gulbis JM, Kelman Z, Hurwitz J, O'Donnell M, Kuriyan J. 1996. *Cell* 87: 297–306  
166. Roos G, Jiang Y, Landberg G, Nielsen NH, Zhang P, Lee MYWT. 1996. *Exp. Cell Res.* 226:208–13  
167. Bravo R, MacDonald-Bravo H. 1987. *J. Cell Biol.* 105:1549–54  
168. Humbert C, Santisteban MS, Usson Y, Robert-Nicoud M. 1992. *J. Cell Sci.* 103:97–103  
169. Li R, Hannon GJ, Beach D, Stillman B. 1996. *Curr. Biol.* 6:189–99  
170. Celis JE, Madsen P. 1986. *FEBS Lett.* 209:277–83  
171. Toschi L, Bravo R. 1988. *J. Cell Biol.* 107:1623–28  
172. Miura M, Domon M, Sasaki T, Kondo S, Takasaki Y. 1992. *Exp. Cell Res.* 201: 541–44  
173. Pagano M, Theodoras AM, Tam SW, Draetta GF. 1994. *Genes Dev.* 8:1627– 39  
174. Hughes DA, MacNeill SA, Fantes PA. 1992. *Mol. Gen. Genet.* 231:401–10  
175. MacNeill SA, Moreno S, Reynolds N, Nurse P, Fantes PA. 1996. *EMBO J.* 15: 4613–28  
176. Zuo S, Gibbs E, Kelman Z, Wang TS-F, O'Donnell M, et al. 1997. *Proc. Natl. Acad. Sci. USA* 94:11244–49  
177. Boulet A, Simon M, Faye G, Bauer GA, Burgers PMJ. 1989. *EMBO J.* 8:1849– 54  
178. Pignede G, Bouvier D, Recondo A-M, Baldacci G. 1991. *J. Mol. Biol.* 222:209– 18  
179. Zhang J, Chung DW, Tan C-K, Downey KM, Davie EW, So AG. 1991. *Biochemistry* 30:11742–50  
180. Yang C-L, Chang L-S, Zhang P, Hao H, Zhu L, et al. 1992. *Nucleic Acids Res.* 20:735–45  
181. Cullmann G, Hindges R, Berchtold MW, Hübscher U. 1993. *Gene* 134:191– 200  
182. Park H, Francesconi S, Wang TS-F. 1993. *Mol. Biol. Cell* 4:145–57  
183. Sugimoto K, Sakamoto Y, Takahashi O, Matsumoto K. 1995. *Nucleic Acids Res.* 23:3493–500  
184. Zhang J, Tan C-K, McMullen B, Downey KM, So AG. 1995. *Genomics* 29:179– 86  
185. Zhou J-Q, He H, Tan C-K, Downey KM, So AG. 1997. *Nucleic Acids Res.* 25:1094–99  
186. Brown WC, Campbell JL. 1993. *J. Biol. Chem.* 268:21706–10  
187. Zhang S-J, Zeng X-R, Zhang P, Toomey NL, Chuang R-Y, et al. 1995. *J. Biol. Chem.* 270:7988–92  
188. Goulian M, Herrmann SM, Sackett JW, Grimm SL. 1990. *J. Biol. Chem.* 265: 16402–11  
189. Chiang C-S, Mitsis PG, Lehman IR. 1993. *Proc. Natl. Acad. Sci. USA* 90: 9105–9  
190. Melendy T, Stillman B. 1991. *J. Biol. Chem.* 266:1942–49  
191. Weinberg DH, Collins KL, Simancek P, Russo A, Old MS, et al. 1990. *Proc. Natl. Acad. Sci. USA* 87:8692–96  
192. Eki T, Matsumoto T, Murakami Y, Hurwitz J. 1992. *J. Biol. Chem.* 267:7284– 94  
193. Waga S, Bauer G, Stillman B. 1994. *J. Biol. Chem.* 269:10923–34  
194. Maga G, Hübscher U. 1996. *Biochemistry* 35:5764–77  
195. McConnell M, Miller H, Mozzherin DJ, Quamina A, Tan C-K, et al. 1996. *Biochemistry* 35:8268–74  
196. Zlotkin T, Kaufmann G, Jiang Y, Lee MYWT, Uitto L, et al. 1996. *EMBO J.* 15:2298–305  
197. Morrison A, Araki H, Clark B, Hamatake RK, Sugino A. 1990. *Cell* 62:1143– 51  
198. Araki H, Ropp PA, Johnson AL, Johnston LH, Morrison A, Sugino A. 1992. *EMBO J.* 11:733–40  
199. Budd ME, Campbell JL. 1993. *Mol. Cell. Biol.* 13:496–505  
200. Podust VN, Hübscher U. 1993. *Nucleic Acids Res.* 21:841–46  
Turchi JJ, Huang L, Murante RS, Kim Y, Bambara RA. 1994. *Proc. Natl. Acad. Sci.* USA 91:9803–7

201. Lieber MR. 1997. *BioEssays* 19:233–40
202. Harrington JJ, Lieber MR. 1994. *Genes Dev.* 8:1344–55
203. Murray JM, Tavassoli M, Al-Harithy R, Sheldrick KS, Lehmann AR, et al. 1994. *Mol. Cell. Biol.* 14:4878–88
204. Harrington JJ, Lieber MR. 1994. *EMBO J.* 13:1235–46
205. Ishimi Y, Claude A, Bullock P, Hurwitz J. 1988. *J. Biol. Chem.* 263:19723–33
206. Goulian M, Richards SH, Heard CJ, Bigsby BM. 1990. *J. Biol. Chem.* 265: 18461–71
207. Robins P, Pappin DJC, Wood RD, Lindahl T. 1994. *J. Biol. Chem.* 269:28535–38
208. Siegel G, Turchi J, Myers T, Bambara R. 1992. *Proc. Natl. Acad. Sci. USA* 89: 9377–81
209. Kenny MK, Balogh LA, Hurwitz J. 1988. *J. Biol. Chem.* 263:9801–8
210. Reagan MS, Pittenger C, Siede W, Friedberg EC. 1995. *J. Bacteriol.* 177:364–71
211. Sommers CH, Miller EJ, Dujon B, Prakash S, Prakash L. 1995. *J. Biol. Chem.* 270:4193–96
212. Turchi JJ, Bambara RA. 1993. *J. Biol. Chem.* 268:15136–41
213. Murante RS, Huang L, Turchi JJ, Bambara RA. 1994. *J. Biol. Chem.* 269:1191–96
214. Huang L, Rumbaugh JA, Murante RS, Lin RJR, Rust L, Bambara RA. 1996. *Biochemistry* 35:9266–77
215. Murante RS, Rumbaugh JA, Barnes CJ, Norton JR, Bambara RA. 1996. *J. Biol. Chem.* 271:25888–97
216. Murante RS, Rust L, Bambara RA. 1995. *J. Biol. Chem.* 270:30377–83
217. Biswas EE, Zhu FZ, Biswas SB. 1997. *Biochemistry* 36:5955–62
218. Johnson RE, Kovvali GK, Prakash L, Prakash S. 1995. *Science* 269:238–40
219. Tishkoff DX, Filosi N, Gaida GM, Kolodner RD. 1997. *Cell* 88:253–63
220. Gordenin DA, Kunkel TA, Resnick MA. 1997. *Nat. Genet.* 16:116–18
221. Kunkel TA, Resnick MA, Gordenin DA. 1997. *Cell* 88:155–58
222. Cathala G, Rech J, Huet J, Jeanteur P. 1979. *J. Biol. Chem.* 254:7353–61
223. Büsen W. 1980. *J. Biol. Chem.* 255: 9434–43
224. DiFrancesco RA, Lehman IR. 1985. *J. Biol. Chem.* 260:14764–70
225. Hagemeier A, Grosse F. 1989. *Eur. J. Biochem.* 185:621–28

226. Eder PS, Walder JA. 1991. *J. Biol. Chem.* 266:6472–79
227. Huang L, Kim Y, Turchi JJ, Bambara RA. 1994. *J. Biol. Chem.* 269:25922–27
228. Rumbaugh JA, Murante RS, Shi S, Bambara RA. 1997. *J. Biol. Chem.* 272: 22591–99
229. Borowiec JA. 1996. See Ref. 314, pp. 545–74
230. Wessel R, Schweizer J, Stahl H. 1992. *J. Virol.* 66:804–15
231. Kuo C-L, Huang C-H, Campbell JL. 1983. *Proc. Natl. Acad. Sci. USA* 80: 6465–69
232. Budd ME, Campbell JL. 1995. *Proc. Natl. Acad. Sci. USA* 92:7642–46
233. Budd ME, Choe W-C, Campbell JL. 1995. *J. Biol. Chem.* 270:26766–69
234. Matsumoto K, Seki M, Masutani C, Tada S, Enomoto T, Ishimi Y. 1995. *Biochemistry* 34:7913–22
235. Saitoh A, Tada S, Katada T, Enomoto T. 1995. *Nucleic Acids Res.* 23:2014–18
236. Seki M, Kohda T, Yano T, Tada S, Yanagisawa J, et al. 1995. *Mol. Cell. Biol.* 15:165–72
237. Chong JPJ, Thommes P, Blow JJ. 1996. *Trends Biochem. Sci.* 21:102–6
238. Kearsey SE, Maiorano D, Holmes EC, Todorov IT. 1996. *BioEssays* 18:183–90
239. Koonin EV. 1993. *Nucleic Acids Res.* 21:2541–47
239a. Adachi Y, Usukura J, Yanagida M. 1997. *Genes Cells* 2:467–79
240. Aparicio OM, Weinstein DM, Bell SP. 1997. *Cell* 91:59–69
241. Ishimi Y. 1997. *J. Biol. Chem.* 272: 24508–13
242. Tom TD, Malkas LH, Hickey RJ. 1996. *J. Cell. Biochem.* 63:259–67
243. Budd ME, Campbell JL. 1997. *Mol. Cell. Biol.* 17:2136–42
244. Wittmeyer J, Formosa T. 1997. *Mol. Cell. Biol.* 17:4178–90
245. Kouprina N, Kroll E, Bannikov V, Bliskovsky V, Gizatullin R, et al. 1992. *Mol. Cell. Biol.* 12:5736–47
246. Miles J, Formosa T. 1992. *Mol. Cell. Biol.* 12:5724–35
247. Miles J, Formosa T. 1992. *Proc. Natl. Acad. Sci. USA* 89:1276–80
248. Prendergast JA, Murray LE, Rowley A, Carruthers DR, Singer RA, Johnston GC. 1990. *Genetics* 124:81–90
249. Malone EA, Clark CD, Chiang A, Winston F. 1991. *Mol. Cell. Biol.* 11:5710–17
250. Rowley A, Singer RA, Johnston GC. 1991. *Mol. Cell. Biol.* 11:5718–26

750 WAGA & STILLMAN

251. Vallen EA, Cross FR. 1995. *Mol. Cell. Biol.* 15:4291–302  
252. Fanning E, Knippers R. 1992. *Annu. Rev. Biochem.* 61:55–85  
253. Deleted in proof  
254. Murakami Y, Hurwitz J. 1993. *J. Biol. Chem.* 268:11018–27  
255. Smale ST, Tjian R. 1986. *Mol. Cell. Biol.* 6:4077–87  
256. Gannon JV, Lane DP. 1987. *Nature* 329:456–58  
257. Dornreiter I, Höss A, Arthur AK, Fanning E. 1990. *EMBO J.* 9:3329–36  
258. Dornreiter I, Copeland WC, Wang TS-F. 1993. *Mol. Cell. Biol.* 13:809–20  
259. Park P, Copeland W, Yang L, Wang T, Botchan MR, Mohr II. 1994. *Proc. Natl. Acad. Sci. USA* 91:8700–4  
260. Collins KL, Kelly TJ. 1991. *Mol. Cell. Biol.* 11:2108–15  
261. Schneider C, Weibhart K, Guarino LA, Dornreiter I, Fanning E. 1994. *Mol. Cell. Biol.* 14:3176–85  
262. Stadlbauer F, Voitenleitner C, Bruckner A, Fanning E, Nasheuer H-P. 1996. *Mol. Cell. Biol.* 16:94–104  
263. Denis D, Bullock PA. 1993. *Mol. Cell. Biol.* 13:2882–90  
264. Bullock PA, Tevosian S, Jones C, Denis D. 1994. *Mol. Cell. Biol.* 14:5043–55  
265. Bullock PA, Denis D. 1995. *Mol. Cell. Biol.* 15:173–78  
266. Goulian M, Heard CJ. 1990. *J. Biol. Chem.* 265:13231–39  
267. Goulian M, Heard CJ, Grimm SL. 1990. *J. Biol. Chem.* 265:13221–30  
268. He ZG, Brinton BT, Greenblatt J, Has- sell JA, Ingles CJ. 1993. *Cell* 73:1223–32  
269. Li R, Botchan MR. 1993. *Cell* 73:1207–21  
270. Eliasson R, Reichard P. 1978. *J. Biol. Chem.* 253:7469–75  
271. Shcherbakova PV, Pavlov YI. 1996. *Genetics* 142:717–26  
272. Navas TA, Zhou Z, Elledge SJ. 1995. *Cell* 80:29–39  
273. Navas TA, Sanchez Y, Elledge SJ. 1996. *Genes Dev.* 10:2632–43  
274. Elledge SJ. 1996. *Science* 274:1664–72  
275. Sugimoto K, Shimomura T, Hashimoto K, Araki H, Sugino A, Matsumoto K. 1996. *Proc. Natl. Acad. Sci. USA* 93:7048–52  
276. Araki H, Leem S-H, Phongdara A, Sugino A. 1995. *Proc. Natl. Acad. Sci. USA* 92:11791–95  
277. Saka Y, Yanagida M. 1993. *Cell* 74:383–93  
278. Saka Y, Fantes P, Sutani T, McLernery C,  

279. Creanor J, Yanagida M. 1994. *EMBO J.* 13:5319–29  
280. Saka Y, Esashi F, Matsusaka T, Mochida S, Yanagida M. 1997. *Genes Dev.* 11:3387–400  
281. Parker AE, Clyne RK, Carr AM, Kelly TJ. 1997. *Mol. Cell. Biol.* 17:2381–90  
282. Marini F, Pellicioli A, Paciotti V, Luc- chini G, Plevani P, et al. 1997. *EMBO J.* 16:639–50  
283. D'Urso G, Grallert B, Nurse P. 1995. *J. Cell Sci.* 108:3109–18  
284. Francesconi S, De Recondo A-M, Bal- dacci G. 1995. *Mol. Gen. Genet.* 246:561–69  
285. Murakami H, Okayama H. 1995. *Nature* 374:817–19  
286. Sugimoto K, Ando S, Shimomura T, Matsumoto K. 1997. *Mol. Cell. Biol.* 17:5905–14  
287. Paulovich AG, Hartwell LH. 1995. *Cell* 82:841–47  
288. Waseem NH, Labib K, Nurse P, Lane DP. 1992. *EMBO J.* 11:5111–20  
289. Sherr CJ, Roberts JM. 1995. *Genes Dev.* 9:1149–63  
290. Sherr CJ. 1996. *Science* 274:1672–77  
291. Xiong Y, Zhang H, Beach D. 1993. *Genes Dev.* 7:1572–83  
292. Strausfeld UP, Howell M, Rempel R, Maller JL, Hunt T, Blow JJ. 1994. *Curr. Biol.* 4:876–83  
293. Jackson PK, Chevalier S, Philippe M, Kirschner MW. 1995. *J. Cell Biol.* 130:755–69  
294. Nichols AF, Sancar A. 1992. *Nucleic Acids Res.* 20:2441–46  
295. Shivji MKK, Kenny MK, Wood RD. 1992. *Cell* 69:367–74  
296. Li R, Waga S, Hannon GJ, Beach D, Stillman B. 1994. *Nature* 371:534–37  
297. Shivji MKK, Grey SJ, Strausfeld UP, Wood RD, Blow JJ. 1994. *Curr. Biol.* 4:1062–68  
298. Chen J, Jackson PK, Kirschner MW, Dutta A. 1995. *Nature* 374:386–88  
299. Goubin F, Ducommun B. 1995. *Oncogene* 10:2281–87  
300. Luo Y, Hurwitz J, Massague J. 1995. *Nature* 375:159–61  
301. Nakanishi M, Robetorye RS, Pereira- Smith OM, Smith JR. 1995. *J. Biol. Chem.* 270:17060–63  
302. Warbrick E, Lane DP, Glover DM, Cox LS. 1995. *Curr. Biol.* 5:275–82  
303. Chen I-T, Akamatsu M, Smith ML, Lung F-DT, Duba D, et al. 1996. *Oncogene* 12:595–607  
304. Chen J, Saha P, Kornbluth S, Dynlacht BD, Dutta A. 1996. *Mol. Cell. Biol.* 16:4673–82

DNA REPLICATION IN EUKARYOTES 751

304. Gibbs E, Kelman Z, Gulbis JM, O'Donnell M, Kuriyan J, et al. 1997. *J. Biol. Chem.* 272:2373–81  
305. Warbrick E, Lane DP, Glover DM, Cox LS. 1997. *Oncogene* 14:2313–21  
306. Funk JO, Waga S, Harry JB, Espling E, Stillman B, Galloway DA. 1997. *Genes Dev.* 11:2090–100  
307. Jones DL, Alani RM, Münger K. 1997. *Genes Dev.* 11:2101–11  
308. Greider CW, Collins K, Autexier C. 1996. See Ref. 314, pp. 619–38  

309. Carson MJ, Hartwell L. 1985. *Cell* 42: 249–57  
310. Adams AK, Holm C. 1996. *Mol. Cell. Biol.* 16:4614–20  
311. Fan X, Price CM. 1997. *Mol. Biol. Cell* 8:2145–55  
312. Nugent CI, Hughes TR, Lue NF, Lundblad V. 1996. *Science* 274:249–52  
313. Stillman B. 1994. *Cell* 78:725–28  
314. DePamphilis ML, ed. 1996. *DNA Replication in Eukaryotic Cells*. Cold Spring Harbor, NY: Cold Spring Harbor Lab

Annual Review of Biochemistry  
Volume 67, 1998  

CONTENTS  

An Accidental Biochemist, Edwin G. Krebs  
0  

HIV-1: Fifteen Proteins and an RNA, Alan D. Frankel and John A. T. Young  
1  

Sphingolipid Functions in Saccharomyces Cerevisiae: Comparison to Mammals, Robert C. Dickson  
27  

Transporters of Nucleotide Sugars, ATP, and Nucleotide Sulfate in the Endoplasmic Reticulum and Golgi Apparatus, Carlos B. Hirschberg, Phillips W. Robbins, and Claudia Abeijon  
49  

Ribonucleotide Reductases, A. Jordan and P. Reichard  
71  

Modified Oligonucleotides: Synthesis and Strategy for Users, Sandeep Verma and Fritz Eckstein  
99  

The Molecular Control of Circadian Behavioral Rhythms and Their Entrainment in Drosophila, Michael W. Young  
135  

Ribonuclease P: Unity and Diversity in a tRNA Processing Ribozyme, Daniel N. Frank and Norman R. Pace  
153  

Base Flipping, Richard J. Roberts and Xiaodong Cheng  
181  

The Caveolae Membrane System, Richard G. W. Anderson  
199  

How Cells Respond to Interferons, George R. Stark, Ian M. Kerr, Bryan R. G. Williams, Robert H. Silverman, and Robert D. Schreiber  
227  

Nucleocytoplasmic Transport: The Soluble Phase, Iain W. Mattaj and Ludwig Englmeier  
265  

Role of Small G Proteins in Yeast Cell Polarization and Wall Biosynthesis, Enrico Cabib, Jana Drgonová, and Tomás Drgon  
307  

RNA Localization in Development, Arash Bashirullah, Ramona L. Cooperstock, and Howard D. Lipshitz  
335  

Biochemistry and Genetics of von Willebrand Factor, J. Evan Sadler  
395  

The Ubiquitin System, Avram Hershko and Aaron Ciechanover  
425  

Phosphoinositide Kinases, David A. Fruman, Rachel E. Meyers, and Lewis C. Cantley  
481  

The Green Fluorescent Protein, Roger Y. Tsien  
509  

Alteration of Nucleosome Structure as a Mechanism of Transcriptional Regulation, J. L. Workman, and R. E. Kingston  
545  

Structure and Function in GroEL-Mediated Protein Folding, Paul B. Sigler, Zhaohui Xu, Hays S. Rye, Steven G. Burston, Wayne A. Fenton, and Arthur L. Horwich  
581  

Matrix Proteoglycans: From Molecular Design to Cellular Function, Renato V. Iozzo  
609  

G Protein Coupled Receptor Kinases, Julie A. Pitcher, Neil J. Freedman, and Robert J. Lefkowitz  
653  

Enzymatic Transition States and Transition State Analog Design, Vern L. Schramm  
693  

The DNA Replication Fork in Eukaryotic Cells, Shou Waga and Bruce Stillman 721  
TGF-beta Signal Transduction, J. Massagué 753  
Pathologic Conformations of Prion Proteins, Fred E. Cohen and Stanley B. Prusiner 793  
The AMP-Activated/SNF1 Protein Kinase Subfamily: Metabolic Sensors of the Eukaryotic Cell?, D. Grahame Hardie, David Carling, and Marian Carlson 821
