
MOLECULAR AND CELLULAR BIOLOGY, Jan. 1999, p. 471–483
0270-7306/99/$04.00+0
Copyright © 1999, American Society for Microbiology. All Rights Reserved.

The Yeast RER2 Gene, Identified by Endoplasmic Reticulum Protein Localization Mutations, Encodes *cis*-Prenyltransferase, a Key Enzyme in Dolichol Synthesis

MIYUKI SATO,$^{1}$ KEN SATO,$^{1}$ SHUH-ICHI NISHIKAWA,$^{2}$ AIKO HIRATA,$^{3}$ JUN-ICHI KATO,$^{4}$ AND AKIHIKO NAKANO$^{1*}$

Molecular Membrane Biology Laboratory, RIKEN, Wako, Saitama 351-0198,$^{1}$ Department of Chemistry, Graduate School of Science, Nagoya University, Chikusa-ku, Nagoya 464-8602,$^{2}$ Institute of Molecular and Cellular Biosciences, University of Tokyo, Yayoi, Bunkyo-ku, Tokyo 113-0032,$^{3}$ and Department of Molecular Biology, Institute of Medical Science, University of Tokyo, Tokyo 108-8639,$^{4}$ Japan

Received 27 July 1998/Returned for modification 11 September 1998/Accepted 16 September 1998

As an approach to understand the molecular mechanisms of endoplasmic reticulum (ER) protein sorting, we have isolated yeast *rer* mutants that mislocalize a Sec12-Mfa1p fusion protein from the ER to later compartments of the secretory pathway (S. Nishikawa and A. Nakano, Proc. Natl. Acad. Sci. USA 90:8179–8183, 1993). The temperature-sensitive *rer2* mutant mislocalizes different types of ER membrane proteins, suggesting that *RER2* is involved in correct localization of ER proteins in general. The *rer2* mutant shows several other characteristic phenotypes: slow growth, defects in N and O glycosylation, sensitivity to hygromycin B, and abnormal accumulation of membranes, including the ER and the Golgi membranes. *RER2* and *SRT1*, a gene whose overexpression suppresses *rer2*, encode novel proteins similar to each other, and their double disruption is lethal. *RER2* homologues are found not only in eukaryotes but also in many prokaryote species and thus constitute a large gene family which has been well conserved during evolution. Taking a hint from the phenotype of newly established mutants of the Rer2p homologue of *Escherichia coli*, we discovered that the *rer2* mutant is deficient in the activity of *cis*-prenyltransferase, a key enzyme of dolichol synthesis. This and other lines of evidence let us conclude that members of the *RER2* family of genes encode *cis*-prenyltransferase itself. The difference in phenotypes between the *rer2* mutant and previously obtained glycosylation mutants suggests a novel, as-yet-unknown role of dolichol.

In the secretory pathway, the endoplasmic reticulum (ER) is the compartment from which the biosynthetic membrane flow begins. Newly synthesized proteins are folded and modified in the lumen of the ER and then transported to their destinations by vesicular processes. On the other hand, a set of proteins is sorted from these proteins and retained in the ER to carry out their functions. Such ER localization is known to be fulfilled by the recognition of signals that are present in the ER resident proteins. Well-known examples of the ER localization signals include the C-terminal H(K)DEL and KKXX sequences, which are involved in the retrieval of proteins from the Golgi apparatus to the ER (19, 25, 26, 33, 46).

Sec12p is a type II transmembrane glycoprotein of the yeast *Saccharomyces cerevisiae* and is essential for the formation of COPII vesicles from the ER (4, 5, 10, 27, 32). Although Sec12p has neither HDEL nor KKXX signals, most Sec12p is localized to the ER in the steady state and is not detected on the purified transport vesicles (4, 27, 29). However, a significant portion of Sec12p receives *cis*-Golgi-specific modification on its N-linked oligosaccharide chains (27, 29). From these observations, we postulated that the ER localization of Sec12p involves two different mechanisms: static retention in the ER and dynamic retrieval from the early Golgi (29, 42, 44). To identify factors participating in these sorting events, we isolated two mutants, the *rer1* and *rer2* (for return to the ER or retention in the ER)

mutants, which mislocalize a Sec12-Mfa1 fusion protein beyond the early Golgi (29).

The *RER1* gene has been extensively studied by ourselves and others; it encodes a protein possessing four transmembrane domains which is located in the early Golgi compartment (6, 42). We have also demonstrated that Sec12p contains two signals for ER localization: an Rer1p-dependent retrieval signal in the transmembrane domain and an Rer1p-independent retention signal in the cytoplasmic domain (44). Furthermore, we have recently shown that Rer1p is required for the retrieval of not only Sec12p but also a variety of ER membrane proteins (43). From these studies, we propose that Rer1p is a component of the machinery required for the Golgi-to-ER retrograde traffic, which constitutes a major retrieval pathway in addition to the KDEL- and KKXX-dependent mechanisms (43).

In this paper, we report the characterization of the *rer2* mutant. The *rer2* mutant shows quite pleiotropic defects in the normal endomembrane system, suggesting that *RER2* is very important for maintaining the integrity of organelles. Cloning by complementation has revealed the presence of a suppressor gene, *SRT1*, in addition to the authentic *RER2* gene. *RER2* and *SRT1* are similar to each other, and their products belong to a new protein family that is well conserved in many organisms. Finally, we present evidence that *RER2* encodes a key enzyme of dolichol synthesis. The analysis of the *rer2* mutant provides new insight into the physiological roles of dolichol.

*MATERIALS AND METHODS*

Strains and culture conditions. The *S. cerevisiae* strains used in this study are listed in Table 1. Yeast cells were grown in YPD (1% [wt/vol] Bacto yeast extract

TABLE 1. Strains used in this study

| Strain    | Genotype                                                                                   | Reference or source |
|-----------|-------------------------------------------------------------------------------------------|---------------------|
| YPH500    | MATαade2trp1his3leu2ura3lys2                                                               | 48                  |
| YPH501    | MATa/MATαade2/ade2trp1/trp1his3/his3leu2/leu2ura3/ura3lys2/lys2                              | 48                  |
| ANY200    | MATa/MATαura3/ura3leu2/leu2trp1/trp1his3/his3his4/his4gal2/gal2suc/suc                       | 42                  |
| ANY21     | MATaura3leu2trp1his3his4gal2suc                                                            | 42                  |
| SNY9      | MATαmfα1::ADE2mfα2::TRP1bar1::HIS3ade2trp1his3leu2ura3lys2                                  | 29                  |
| SKY1      | MATαmfα1::ADE2mfα2::LEU2bar1::HIS3ade2trp1his3leu2ura3lys2                                  | 42                  |
| SNH23-7A  | MATαrer2-2mfα1::ADE2mfα2::TRP1bar1::HIS3ade2trp1his3leu2ura3lys2                            | This study          |
| SNH23-7D  | MATαrer2-2mfα1::ADE2mfα2::TRP1bar1::HIS3ade2trp1his3leu2ura3lys2                            | This study          |
| SNH23-10A | MATαrer2-2mfα1::ADE2mfα2::TRP1bar1::HIS3ade2trp1his3leu2ura3lys2                            | 29                  |
| HS23-3BA  | MATαrer2-2mfα1::ADE2mfα2::TRP1bar1::HIS3suc2::LEU2ade2trp1his3leu2ura3lys2                   | This study          |
| SMY2      | MATαRER2::URA3::RER2ade2trp1his3leu2ura3lys2                                               | This study          |
| SMY41     | MATαΔrer2::LEU2mfα1::ADE2mfα2::TRP1bar1::HIS3ade2trp1his3leu2ura3lys2                       | This study          |
| SMY13     | MATαΔsrt1::TRP1mfα1::ADE2mfα2::LEU2bar1::HIS3ade2trp1his3leu2ura3lys2                       | This study          |
| SMY5      | MATa/MATαΔrer2::LEU2/RER2Δsrt1::TRP1/SRT1ura3/ura3leu2/leu2trp1/trp1his3/his3his4/gal2/gal2suc/suc | This study          |
| SMY20     | MATαΔrer2::HIS3ade2trp1his3leu2ura3lys2                                                   | This study          |
| BC180     | MATasst2-Δ2ura3leu2his3ade2                                                                | 29                  |
| SF604-9C  | MATαsec59-1ura3his4suc2                                                                   | R. Schekman         |

[Difco Laboratories, Detroit, Mich.], 2% [wt/vol] polypeptone [Nihon Seiyaku, Tokyo, Japan], and 2% [wt/vol] glucose) or in MVD (0.67% yeast nitrogen base without amino acids [Difco Laboratories] and 2% glucose) supplemented appropriately. MCD medium is MVD containing 0.5% Casamino Acids (Difco Laboratories). Hygromycin B (Wako Junyaku Kogyo, Osaka, Japan) and sodium orthovanadate (Sigma-Aldrich Japan, Tokyo, Japan) were added to YPD medium to give final concentrations of 50 μg/ml and 4 mM, respectively.

Antibodies. Rabbit anti-Kar2p, anti-Ypt1p, and anti-Gas1p polyclonal antibodies were generous gifts from M. Rose of the Massachusetts Institute of Technology; D. Gallwitz of the Max Planck Institute of Biophysical Chemistry, Göttingen, Germany; and H. Riezman of the University of Basel, respectively. Affinity-purified rabbit anti-Dap2p polyclonal antibody was kindly provided by Y. Amaya of Niigata University and Y. Wada of Osaka University. Rabbit anti-Sec12p and anti-carboxypeptidase Y (anti-CPY) polyclonal antibodies were prepared as described previously (27, 49). The 16B12 mouse monoclonal anti-hemagglutinin (anti-HA) antibody was purchased from Berkeley Antibody Company (Richmond, Calif.). Rabbit anti-Kex2p and anti-Pgk1p polyclonal antibodies were prepared through the support of Suntory Limited, Osaka, Japan.

Cloning of RER2 and SRT1. To clone RER2, hygromycin B sensitivity was used. SNH23-7D (rer2-2) was transformed with a yeast genomic library on YEp13 (56), and the transformants were replicated onto YPD plates containing 50 μg of hygromycin B per ml. Plasmids were recovered from the colonies that grew on the hygromycin B plates and subcloned into pRS316 (48). The 1.6-kb SpeI-NdeI fragment (pR7) was capable of complementing both the hygromycin B and temperature sensitivities of rer2 cells. To test the linkage between this clone and the RER2 locus, the 5.2-kb SalI (in vector)-BglII fragment from the original clone was inserted into the BamHI-SalI sites of YIp5, an integration vector with a URA3 marker (50). This plasmid was digested with SacI and integrated in the chromosome of a wild-type strain (YPH500) by homologous recombination (to give strain SMY2). SMY2 was crossed with HS23-3BA (rer2-2) and sporulated. Tetrad analysis showed a tight linkage between the URA3 marker and the rer2 locus (11 parental ditypes of 13 tetrads), indicating that this clone contained the authentic RER2 gene. We also tried to clone the RER2 gene by complementation of the temperature sensitivity of rer2 cells. The same yeast genomic library was introduced into SNH23-10A (rer2-2), and the transformants that grew at 37°C were selected. Plasmids recovered from these colonies contained two overlapping clones which were different from RER2. They were subcloned into pRS316 and pQR326 (35), and the 2.4-kb XhoI-BglII fragment was the smallest subclone that suppressed rer2-2. We named the gene in this subclone SRT1.

Disruption of RER2 and SRT1. The 0.7-kb AfII-SplI region of RER2 was replaced by the LEU2 gene in pYl, which contained the 5.5-kb SalI (in vector of the original clone)-NheI fragment in YEp352 (18). The disrupted copy of RER2 was excised by HindIII digestion and introduced into a diploid strain constructed by crossing SNY9 and an isogenic MATa strain. A Leu+ haploid segregant (SMY41) was obtained by tetrad dissection. The 2.4-kb XhoI-BglII fragment of SRT1 was subcloned into pBluescript SK(+) (Stratagene Cloning Systems, La Jolla, Calif.) to give pRER206. The SRT1 gene was disrupted by replacing the 0.6-kb EcoO109I-BamHI region with TRPI in pRER206. This plasmid was digested with XhoI and PvuII and introduced into diploid (YPH501) and haploid (SKY1) strains. A viable haploid strain whose chromosomal SRT1 was disrupted (SMY13) was obtained.

To construct a Δrer2Δsrt1 double mutant, one of the RER2 loci was disrupted with LEU2 by the same strategy as described above in a diploid strain, ANY200, to give SMY1. One of the SRT1 loci was further disrupted with TRPI to give

recently demonstrated that both Mfα1-Sec71p and Mfα1-
Sec63p are mislocalized in Δrer1, indicating that Rer1p is a
component of the general machinery involved in the ER local-
ization of these proteins (43). In the present study, we intro-
duced these fusions into the rer2 mutant on a single-copy
plasmid and performed the halo assay at 23°C. As shown in
Fig. 1A, both fusions produced larger halos in rer2 cells than in
the wild-type cells, indicating that the rer2 mutant is constitu-
tively defective in the correct localization of Sec71p and
Sec63p. It is also noteworthy that the rer2 mutant missecretes
a significant amount of immunoglobulin heavy-chain binding
protein (BiP), a soluble ER protein possessing the HDEL

FIG. 1. (A) Secretion of mature α-factor by cells producing Mfα1-Sec71p
and Mfα1-Sec63p. Wild-type (WT) (SNY9), Δrer1 (SKY7), and rer2-2 (SNH23-
7D) cells expressing Mfα1-Sec71p or Mfα1-Sec63p on a single-copy vector were
examined by the halo assay at 23°C. (B) The rer2 mutants missecrete BiP. Wild-
type (SNY9), rer2-2 (SNH23-16C), Δrer2 (SMY41), and Δsrt1 (SNY25) cells were
grown in YPD medium at 23°C to the early logarithmic phase. Proteins from the
medium (equivalent to 1.5 × 10⁶ cells) were analyzed by immunoblotting with
the anti-BiP antibody.

matography on a Silica Gel G-60 plate (Merck) with the solvent system of
benzene-ethyl acetate (95:5) to confirm that the products were mostly dolichol
and dehydrololichol compounds. Another aliquot of this chloroform suspension
was subjected to liquid scintillation counting to measure the total radioactivity
incorporated into the dolichol fraction. The total cell lysates prepared as de-
scribed above were also incubated under the same assay conditions, and the
products were similarly analyzed by thin-layer chromatography. The reference
molecules, i.e., dolichol from porcine liver, ficaprenol (polyprenol from *Ficus*
*elastica*), solanesol (all-trans-nonaprenpol), and squalene, were purchased from
Sigma.

Other methods. Halo assays were performed on MCD plates with a tester
MATa sst2 strain as described previously (29). For electron microscopy, thin
sections of yeast cells were prepared by the freeze-substitution fixation method
(51). Yeast transformation was carried out by the lithium thiocyanate method
(22) or by electroporation (16). DNA manipulations were performed by standard
techniques (41). DNA sequences of *RER2*, *SRT1*, and their derivatives were de-
termined by the dideoxy method with a DNA sequencer (model 373A; Applied
Biosystems, Tokyo, Japan).

RESULTS

The rer2 mutant mislocalizes a variety of ER proteins. To
monitor the localization of Sec12p within a cell, we have de-
vised an in vivo method utilizing the fusion protein with a yeast
α-mating-factor precursor (Mfα1p) as a reporter. If this fusion
protein (Sec12-Mfα1p) is transported to the late Golgi, the
Mfα1p moiety is processed, leading to the secretion of mature
α-factor. The secreted α-factor can be detected by the halo
assay (Halo⁺). By this method, rer mutants were isolated as
cells defective in correct localization of Sec12-Mfα1p (29). The
rer1 mutant cells show no obvious growth defect at any tem-
peratures in spite of a clear Halo⁺ phenotype. In contrast, the
rer2 mutant exhibits severe growth inhibition even at 23°C and
no longer grows at 37°C (29). The halo formed around the rer2
cells is visible but small, which may be due to the slow growth
of the cells.

The halo assay method can be applied to monitor the local-
ization of Sec71p and Sec63p, which are also membrane pro-
teins localized to the ER in the wild-type cells. Sec71p is a type
III transmembrane protein (topology opposite to that of
Sec12p), and Sec63p spans the membrane three times. These
proteins form a multimeric complex required for the ER trans-
location of newly synthesized proteins (11, 12, 23, 39, 40). We
constructed fusion proteins of these proteins and Mfα1p and

FIG. 2. (A) Underglycosylation phenotype of rer2 mutants. Wild-type (WT)
RER2 (SNY9) and rer2-2 (SNH23-7D) cells were grown in YPD medium to 10⁷
cells/ml at 23°C and then shifted to 37°C for the indicated times. Total cell
extracts were prepared, and proteins (70 μg) were analyzed by immunoblotting
with the anti-Sec12p antibody. (B) Biosynthesis of CPY in rer2 mutants. The
rer2-2 mutant (SNH23-7A) and wild-type (SNY9) cells were preincubated for 1 h
at 23 or 37°C and then labeled for 4 min with Tran³⁵S-label and chased for the
indicated times at the same temperature. CPY was immunoprecipitated from cell
extracts and treated with or without endo H. Samples were analyzed by SDS-
PAGE and fluorography. p1, ER form; p2, Golgi form; m, mature vacuolar form;
*, unglycosylated pro-CPY. (C) Biosynthesis of a GPI anchor protein, Gas1p, in
rer2 mutants. After preincubation at 37°C for 1 h, rer2-2 (SNH23-7A) and wild-
type (SNY9) cells were labeled for 4 min with Tran³⁵S-label and chased at 37°C.
Gas1p was immunoprecipitated and analyzed by SDS-PAGE and fluorography.

The mobility of this band was the same as that of endo H-treated pro-CPY, suggesting again that the rer2 mutant has a defect in N glycosylation. The endo H treatment also revealed that about 30% of the CPY molecules had experienced the proteolytic processing in 30 min, indicating that the vesicular trafficking itself was not blocked in the mutant. The deficiency of glycosylation was more severe at 37°C (Fig. 2B, lower panels).

A pulse-chase experiment was also performed for Gas1p, a glycosylphosphatidylinositol (GPI)-anchored protein which is also modified by N- and O-linked glycosylation (53). As shown in Fig. 2C, the major band migrated with an apparent molecular mass of 70 kDa in the rer2 mutant, which coincided with the size of the polypeptide moiety (14). Taken together, the data indicate that the rer2 mutant apparently possesses a profound temperature-sensitive defect in both N and O glycosylation.

In liquid culture, rer2 cells tend to form aggregates, which is typical for glycosylation mutants. This could be due to alterations in the cell wall structure resulting from the glycosylation defect. The rer2 mutant also shows sensitivity to hygromycin B and resistance to sodium orthovanadate. Such drug sensitivity and tolerance are generally observed in various mutants that have defects in glycosylation in either the ER or the Golgi (3, 9).

Abnormal membrane structures accumulate in the rer2 mutants. The intracellular membrane structures of the rer2 mutant were observed by indirect immunofluorescence microscopy with antibodies against organelle-specific marker proteins. As shown by staining with the anti-BiP antibody (Fig. 3), the rer2 mutant showed accumulation of the ER membranes, which developed in the cytoplasm. The morphology of the Golgi apparatus also looked abnormal in the rer2 mutant (Fig.

FIG. 3. Immunofluorescence localization of an ER marker protein, BiP. Wild-type (SNY9) (A and B) and rer2-2 (SNH23-7D) (C and D) cells growing at 23°C were subjected to indirect immunofluorescence microscopy with the anti-BiP antibody. (A and C) Fluorescence images with the antibody. (B and D) DNA staining with DAPI.

signal (Fig. 1B) (29, 42). Because the intracellular level of BiP was not appreciably different in the wild-type and the rer2 cells (data not shown), the missecretion is not simply due to the overflow of the Erd2p-dependent retrieval system. The rer2 mutant appears to have a lesion that disturbs the localization of various ER proteins.

The rer2 mutant shows glycosylation defects. To examine the level of expression of Sec12p in rer2 cells, immunoblotting was performed with an anti-Sec12p antibody (Fig. 2A). Sec12p acquires both N- and O-linked oligosaccharide modifications and is usually detected as a fuzzy band around 70 kDa (27). The steady-state amount of Sec12p in rer2 cells was comparable to that in the wild type. In rer2 cells, however, a ladder of smaller species was detected in addition to the fuzzy band corresponding to the normal Sec12p in the wild-type cells. After endoglycosidase H (endo H) treatment, the smaller species of Sec12p in rer2 cells were still present (data not shown). These results suggest that the rer2 mutant is defective in both N- and O-linked glycosylation in the ER.

We also performed a pulse-chase experiment with CPY, a vacuolar enzyme (49), to investigate the vesicular biosynthetic pathway of the mutant (Fig. 2B). In the wild-type cells, the conversion from the ER form through the Golgi form to the mature vacuolar form of CPY takes place in about 30 min. The same conversion of CPY species with a similar time course was also observed with the rer2 mutant at 23°C. However, a ladder of bands was detected throughout the chase time, the major species of which migrated even faster than the mature form.

FIG. 4. Immunofluorescence localization of a cis-Golgi marker protein, Ypt1p. Wild-type (SNY9) (A and B) and rer2-2 (SNH23-7D) (C and D) cells were incubated at 37°C for 4 h and subjected to indirect immunofluorescence microscopy with the anti-Ypt1p antibody. (A and C) Fluorescence images with the antibody. (B and D) DNA staining with DAPI.

FIG. 5. Electron micrographs of rer2 cells. Wild-type (SNY9) (A) and rer2-2 (SNH23-10D) (B to E) cells were incubated at 23°C (A, B, and D) or shifted to 37°C for 2 h (C and E) and then subjected to freeze-substitution fixation and electron microscopic observation. Panels D and E are enlargements of panels B and C, respectively.

476 SATO ET AL.

4). The antibody against Ypt1p, which is localized mostly to the early Golgi (34), stained enlarged structures which often had a ring-like shape. These abnormal membrane structures were seen even at 23°C but were recognized more clearly at 37°C. We further observed the organellar morphology by electron microscopy by the freeze-substitution fixation method. As shown in Fig. 5, membrane structures were dramatically altered in the rer2 mutant. At 23°C (Fig. 5B and D), ER-like membranes piled up in the cytoplasm and the nuclear envelope, a part of the ER, looked twisted, as often seen for some early sec mutants (31). In addition, ring- and cup-shaped membranes 150 to 200 nm in diameter and numerous small vesicles accumulated throughout the cytoplasm. The ring- and cup-shaped structures are reminiscent of Berkeley bodies, which are formed in sec7 and sec14 mutants, resulting from the deformation of accumulating Golgi membranes (31), and presumably correspond to the structures visualized by immunofluorescence of Ypt1p. Apparently, multiple secretory organelles accumulated in the rer2 mutant even at 23°C.

When the mutant cells were incubated at 37°C for 2 h, the intracellular membranes became more aberrant (Fig. 5C and E). The ER-like membrane further developed and expanded in the cytoplasm. The boundary of the membranes became unclear compared to the normal ER membrane. In addition, irregularly shaped unidentifiable structures and many fragmented vacuoles were observed. Thus, RER2 appears to be required to maintain the normal functions and structures of the whole central vacuolar system.

Cloning of the RER2 and SRT1 genes. To obtain the wild-type RER2 gene, we first tried to isolate a clone that complements the temperature-sensitive growth of the rer2 mutant. The rer2-2 mutant (SNH23-10A) was transformed with a YEp13-based yeast genomic library (56), and the transformants that grew at 37°C were selected. We obtained two clones which overlap in the DNA inserts and identified the common fragment (the 2.4-kb XhoI-BglII fragment) that complements rer2 temperature sensitivity. This DNA fragment contained a single open reading frame (YMR101c) encoding a protein of 343 amino acids and was able to complement the temperature-sensitive growth of the rer2 mutant on a single-copy plasmid. However, the integration analysis indicated that this clone did not map at the rer2 locus. We concluded that this clone is a suppressor of rer2 and named it SRT1 for suppressor of rer two.

The authentic RER2 gene was isolated by complementation of the hygromycin B sensitivity of the rer2 mutant. rer2-2 cells (SNH23-7D) were transformed with the same yeast genomic library, and transformants were replicated onto YPD plates containing 50 μg of hygromycin B per ml. Four independent clones that complement rer2 were identified. They had an overlapping genomic fragment, and the 1.6-kb SpeI-NdeI fragment completely complemented the hygromycin B sensitivity of the rer2-2 mutant on a single-copy plasmid. This subclone also complemented the temperature sensitivity of growth and the Halo⁺ phenotype of the rer2 mutant on a single-copy plasmid. We sequenced this DNA fragment and revealed a single open reading frame (YBR002c) that encodes a protein of 286 residues. By integration mapping, this gene was confirmed to be the bona fide RER2 (see Materials and Methods).

Figure 6 shows the growth phenotypes of the original rer2 mutant, the rer2 mutant suppressed by SRT1, and the rer2 mutant complemented by RER2. The SRT1 gene on a single-copy plasmid (CEN) suppresses the temperature-sensitive growth of the rer2 mutant like the authentic RER2 gene but does not suppress the hygromycin B sensitivity very well. On a multicopy plasmid (2μm), it suppresses both phenotypes. It should be noted that the single-copy SRT1 gene also sup-

![Figure 6](#fig6)

FIG. 6. Growth phenotypes of the rer2 mutant and complementation by RER2 and SRT1. rer2-2 cells (SNH23-7D) were transformed with RER2 or SRT1 on a single-copy (CEN) or multicopy (2μm) plasmid. The transformants were streaked on MCD plates and incubated at 23 or 37°C for 4 days. The same cells were also streaked on a YPD plate containing 50 μg of hygromycin B per ml and incubated at 23°C for 4 days.

pressed the Halo⁺ phenotype of the rer2 mutant (data not shown).

Proteins encoded by RER2 and SRT1 (Rer2p and Srt1p, respectively) show high sequence homology (30% identical) over the whole polypeptide (Fig. 7A). Hydropathy analysis (24) indicated no hydrophobic region in either Rer2p or Srt1p that is capable of acting as a signal sequence or transmembrane domain. Database searches revealed many homologues of Rer2p in various organisms, as shown in Fig. 7B. Srt1p is the only S. cerevisiae homologue of Rer2p. High degrees of identity to the Schizosaccharomyces pombe SPAC4D7.04c and the Caenorhabditis elegans T01G1.b gene products (40 and 31%, respectively), products of open reading frames identified by sequencing projects, were found. A higher plant, Arabidopsis thaliana, also seems to have a homologue of Rer2p. Interestingly, similar proteins can be found in many prokaryote species, including Haemophilus influenzae, Escherichia coli, and Bacillus subtilis. These proteins show ~30% identity to Rer2p throughout the molecule. Thus, Rer2p seems to belong to a family that contains well-conserved members from prokaryotes to eukaryotic multicellular organisms. However, these entries in the database are all hypothetical proteins. It should be also noted here that Mycoplasma, a microbe lacking a cell wall, is the only organism that does not possess any Rer2p homologue among those whose genomes were completely sequenced.

We determined the mutation points of the rer2 mutants. The genomic DNA fragments containing the rer2 locus were recovered by PCR and sequenced. The rer2-1 and rer2-2 alleles harbored missense mutations that led to G164-to-D and S209-to-N amino acid replacements, respectively. These residues are well conserved in the RER2 gene family from prokaryotes to higher eukaryotes (Fig. 7B). The two mutants were very similar in every phenotype we examined.

VOL. 19, 1999                                                                 Rer2p IN DOLICHOL SYNTHESIS 477

**A**

| Rer2p | METDSQTPGH -----------------SFVLK------------------K--NI[FSR]---TIR[A]-----SNC[VP][HV] GFIMDGNRRF ARKKEMDVKE | 55 |
| Srt1p | MKMPSIQIQ FVALKRLLLVE TKEQMCAVK SIFQRVFANW MSLSLFSWFY VNQNILIKA LRVGPPPEHY SFIMDGNRRY AKSRRLPYKK | 90 |
| Rer2p | GHEAGFVMS RIELYEAG VDTATVFAFS TENFKRSSRE VB[SLMT]ARE RIRQITERGE --LACKYGVR TK[IGDJSLI DKSLLEDYRV | 143 |
| Srt1p | GHEAGGLTL TL[YICKRLG VKCVSAYAFS TEN[NRPKEE VDI[MNFTV KLDEFAKRAK DYKDPLYGSK IR[MGDQSLL SPEMRKKKKK | 180 |
| Rer2p | AVETTKNNKR A[TLNTCFPYT] GREELHMK ETIVQH--KK GAADESTLE SHLYTAGVPP L-DLLTRTSG VSRLSDFLIN QASSKGVRTF | 230 |
| Srt1p | VEELITQDGDD F[ILFICFPYT] SRNDMLHTR DSVEDELNK SPRINIRKFT NKM[MGFHSN KC[LIRTSG HRR[LSDYMLW QVHENAT-TE | 269 |
| Rer2p | LLDCWNPEFG PIRMAWIL[K FSF---HKSF LNKE---Y-R LE[--GDYD EETNG------DPIDLKE KKLN | 286 |
| Srt1p | FSDT[SRPNES FFAMYLMI[K WSFSTIQKY NEKHSLFEK IH[SVPSIFK KKKTAMSLYN FPNP[TSSV TGDE | 343 |

**B**

| Rer2p | ME----TD5GI-----------------PGHSFVLK-------------WTKNIFSR-----TLR ---ASNCVPRH[GFIMDGNR RFARKKEMD-VKEGHEAGFV | 62 |
| S. pombe | MG----NLSGW---------------FIHCPPLO-------------WTLDQLET-----MMI NTIKRKVPO[HAFVMGNR RWARQRRMETI-E[GHSSGFE | 65 |
| C. elegans | MGTEPTNFCSRGERVHHSTT EAMLSPFHHCAVMAANLTAD EEDGWFAQQEQPWWQWLLR RFIASGPIPRH[AFVMGNR RFAXTKHLGNVIK[HEKGGFT | 100 |
| E. coli | --------MMI--------------SA-----------------------TQPLS EKLPAHGC-RHVAIIMDNGN RWAKKQGKIRA-FGHKAGAK | 48 |
| H. influenzae | -------MI-------------EL------------------------DQ--- TNIP------KHVAIIMDNGN RWAKQKNMRI-FGHTNGVT | 40 |
| B. subtilis | M----LNILKNWKNQTTAAS NL-------------------------ERYTK EDILKGEIPEH[AIIMDNGN RWAKKRSLPRI-A[GHHEGMK | 62 |
| Synecochystis | M------TIKP-VILETLPP DL-----------------------D---- QORL----PQH[AVIMDNGN RWAKQRGMPRI-MGHQRGV[D | 51 |
| M. jannaschii | M----GILGKIKNKLKSIGK RVIIDF------------YRFLDN SGVL----------KIYEKILE EAIDKDNLPK[HVAIIMDGNR RAAEIYGKDRY-YGHYLGAE | 79 |
| Rer2p | SMSRILELCYEAGVDTATVF AFSENFKRSSREVESMTL ARERIRQITERGELACKYGVR RIKIIGDLSLLDKSLLLEDVR VAETTKNNKRATLNICFPY | 162 |
| S. pombe | ALKSV----CLKLGYKEVSAF TFSIENFKRSKYEDMLEI AKNSLTQITAHGDLVDQYGI RIRIVGDLSDLQPDVLETAL KAVEITKHNTKATLNVCFPY | 162 |
| C. elegans | QLAKILDWCNRFGIREITVY AFSENFKRSEEEVSLMRL AEEKFQKLLNDSKLDEKRI CFRFYGNRSLLSSRLQKLMS DIEHR[ENFDGGRLNVCMPY | 200 |
| E. coli | SVRAVSFAANN GIEALTLY AESSENWRPAQEVSALMEL FVWAL--DSEVKS[LH-RHNV RLRIIGDTSRFNSRLQERJR KSEALTAGNTGLTLNIAANY | 145 |
| H. influenzae | AVRKAVAYARQIGVEVLTLY AFSSENWSRPEQETSALMSL FMQAL--DREVKKLH-KNNI CLKITGDVSRFSETLQEKJK KAENLTEKNTALTLNIAANY | 137 |
| B. subtilis | VVKRTTKLANEGVKVLTLY AFSTENWKRPKMEVDFIMKL PEEFL--NTYLPELV-EENV QVRIIGDETALPAHLRAJE KAVQDTAQNDGMILNFALNY | 159 |
| Synecochystis | SLKNLLRCCDDWGIAALTAY AFSTENWGRPLEEVEMTL FERVL--RRELKEMM-AENV RIRFVGNLTSIPRSLQDETA RSMEOTKNNNTGIRFTVATNY | 148 |
| M. jannaschii | KVREVLRWARDLGINVVTLY AFSTENFRPKEEVDKLMEL FEKKFYEIADDEEIH-RYEY RVRAIGRINLLPKNVQKATK YAEERTKNYNKFFVNIAIAY | 178 |
| Rer2p | TGREETLHAMKETIVQHKKG ----AAIDESTLESHLTYA- ----GVPPLDLLJRTSGVSR JSDFLJWOASSKGVRIELLD CLWPBFGPIRMAWILLKFSF | 253 |
| S. pombe | TSRHEIATSIVSQIVKMAEDG TITPEDDEDIFEKNLLIK- ----DSLPLDLLJRTSGVER JSDFMLWQ-CHKNT[EIKFID FYWPDF-------SI------ | 244 |
| C. elegans | TSRDEIARSFETIRKHVKDG KVNVDEINESMIDAC LDSG- ----CGGTSPDLFJRISGEHR JSDFLMWQ--ASETHVYFDD VLWPEFGYFNLCKAILNYQY | 294 |
| E. coli | GGRWDI[QGVROLAEKVQQG] ----NLQPDQIDEELMNQH VCMHELAPVQLMIRTGGEHR JSNFLJWQ--IAYAELYFTD VLWPDFDEQDFEGALNAFAN | 238 |
| H. influenzae | GG[CWDIQQAAKQTAEKVKE] ----EMSVDINNSTFQHH LATQNAPPVQ[IJRTSGEQR JSNFLJWQ--IAYAELYFSD VLWPDFNQLEFNRAIASYQQ | 230 |
| B. subtilis | GGRTEI[VSAAKSLAEKVKEG] ----SINIEDIDESFSTY LMTESLQDPE[IJRTSGEIR JSNFLJWQ--VAYSEFVFTD VLIIPDFKEDHFLQALGEFQQ | 252 |
| Synecochystis | GGRQEI[VHYVCQAIAREVQS] ----KLDPETIDEGVVEKY LYTEGLSHPD[IJRTSGEER JSNFLJWQ--MAYAEIYVTI TLWPDFDRQAFHLALERDYQQ | 241 |
| M. jannaschii | GGQOETIIDAVKKIAEKVKRG EIEPEDIDKELIDKHLYTAN L---PFPNPDU[IJRTSGEER JSNFLJWQ--SSYSELYFCD IYWPERRVDFLRARVDYQR | 273 |
| Rer2p | HKSFLNK-------------E YRLEEGDYD-EETNGDPIDL KEKKLN | 286 |
| S. pombe | ----------WKF-FP----MLIQYQ-LQN QPA--- | 261 |
| C. elegans | YRTTVTKMTSSKVSDDNATS WKMNFSGNDRDLISVKSRP QGNREK | 340 |
| E. coli | RERRFGGTEPGDETA---------- | 253 |
| H. influenzae | RHRRFGGTE---------- | 239 |
| B. subtilis | RGRRFGGI---------- | 260 |
| Synecochystis | RHRRFGKV---------- | 249 |
| M. jannaschii | RQRRFGK---------- | 280 |

FIG. 7. Sequence comparison of Rer2p with its homologues. The boxes show identical amino acid residues, and the borderless shading indicates similar residues. (A) Sequences of yeast Rer2p and Srt1p. (B) Rer2p homologues in various organisms, including both eukaryotes and prokaryotes. They are all hypothetical proteins found by the genome projects. The proteins are *S. pombe* SPAC4D7.04c, *C. elegans* T01G1.1, *E. coli* o253, *H. influenzae* HI0920, *B. subtilis* yluA, *Synecochystis* strain PCC6803 SLL0506, and *Methanococcus jannaschii* MJ1372. Asterisks indicate the mutation points found in the *rer2-1* (G164D) and *rer2-2* (S209N) alleles.

The Rer2p family is essential for vegetative growth. We constructed *RER2* and *SRT1* null mutants. In a diploid strain (SMY3), one *RER2* locus was disrupted by replacing the *AflII-SplI* fragment of *RER2* with a *LEU2* fragment (Fig. 8A). The disruption was confirmed by Southern blotting (data not shown). Tetrad dissection identified Leu⁺ haploid cells (∆rer2), which were viable but grew very slowly even at 23°C (Fig. 8B). This growth defect was more severe than that of the original *rer2* mutants. The ∆rer2 strain showed phenotypes similar to those of the *rer2* mutants: temperature-sensitive growth at 37°C, missecretion of BiP, hygromycin B sensitivity, and vanadate resistance. These observations suggest that *rer2-1* and *rer2-2* were both decrease-of-function type mutations, although not null. We could not observe the mislocalization of Sec12-Mfα1p, be-

FIG. 8. Disruption of the *RER2* gene. (A) The *AflII-SplI* region of *RER2* was replaced by the *LEU2* fragment. The disrupted copy of *RER2* was excised with HindIII and introduced into a diploid strain. (B) Tetrad analysis of the resulting strain (*RER2/∆rer2::LEU2*) at 23°C. All small colonies were Leu⁺, indicating that the disruption of *RER2* causes a severe growth defect.

cause the Δrer2 mutant grew too slowly to be examined by the halo assay. SRT1 was also able to suppress the temperature-sensitive and slow-growth phenotypes of the Δrer2 mutant (data not shown).

The disruption of SRT1 was performed by replacement of the EcoO109I-BamHI fragment with the TRP1 gene. We could obtain a haploid strain (SMY13) whose chromosomal SRT1 gene was disrupted, indicating that the SRT1 gene is not essential for growth. The correct disruption was confirmed by Southern blotting (data not shown). The Δsrt1 strain grew normally at any temperature examined and did not show a significant difference from the wild type in hygromycin B sensitivity. The localizations of Sec12-Mfa1p (data not shown) and BiP (Fig. 1B) were also normal in the Δsrt1 mutant.

Double disruption of the RER2 and SRT1 genes was performed with LEU2 and TRP1, respectively, in a diploid strain (SMY5) (see Materials and Methods). In dissection of 16 tetrads after sporulation, no Leu⁺ Trp⁺ segregants were able to grow, suggesting that the double disruption of RER2 and SRT1 was lethal. To confirm this, the RER2 gene was introduced into SMY5 on the pRS316 plasmid, and tetrad dissection was carried out again. We could obtain five Leu⁺ Trp⁺ Ura⁺ segregants, but none of them grew on a plate containing fluoroorotic acid, a reagent that forces elimination of the URA3 plasmid, even at 23°C. Thus, RER2 and SRT1 constitute a family that has an essential function for cell viability.

Subcellular localization of Rer2p. To analyze the biochemical characteristics of the RER2 gene product, the 3HA tag was introduced in the N-terminal part of Rer2p. This 3HA-Rer2p was completely functional, because the 3HA-RER2 gene was able to complement both the temperature and hygromycin B sensitivities of the Δrer2 mutant on a single-copy plasmid (data not shown). Total cell lysates were prepared from Δrer2 cells

FIG. 9. (A) Subcellular fractionation of 3HA-Rer2p. Δrer2 cells expressing 3HA-Rer2p on a single-copy vector were spheroplasted, homogenized, and subjected to a series of centrifugations: 300 × g for 5 min, 13,000 × g for 15 min, and 100,000 × g for 45 min. Aliquots were taken from the pellet of the 13,000 × g centrifugation (P13) and the pellet (P100) and supernatant (S100) fractions of the 100,000 × g centrifugation and analyzed by immunoblotting. (B) Extraction of 3HA-Rer2p. The total homogenate was treated with the reagents indicated and centrifuged at 436,000 × g for 1 h. The pellets and supernatants were analyzed by Western blotting with the anti-HA antibody. TX-100, Triton X-100.

FIG. 10. Subcellular localization of 3HA-Rer2p. Δrer2 cells (SMY41) harboring RER2 (A and B) or 3HA-RER2 (C and D) on a single-copy plasmid were grown at 30°C and prepared for indirect immunofluorescence microscopy with the anti-HA antibody (16B12). (A and C) Fluorescence images with the antibody. (B and D) DNA staining with DAPI.

FIG. 11. Double immunofluorescence staining of BiP and 3HA-Rer2p. Δrer2 cells (SMY41) harboring 3HA-RER2 on a single-copy plasmid were grown at 30°C and prepared for indirect immunofluorescence microscopy. (A) Rhodamine fluorescence corresponding to the anti-BiP antibody. (B) Fluorescein fluorescence corresponding to the anti-HA antibody (16B12). (C) DNA staining with DAPI.

harboring 3HA-RER2 on a single-copy or multicopy plasmid, and immunoblotting was performed with an anti-HA monoclonal antibody (16B12). A single band migrating at around 39 kDa was detected in a dose-dependent manner; this mass was slightly larger than the predicted molecular mass of 36 kDa (data not shown).

To assess the localization of 3HA-Rer2p, subcellular frac-

TABLE 2. Activity of *cis*-prenyltransferase in wild-type and *rer2* cells$^a$

| Cells           | *cis*-Prenyltransferase activity (cpm/min/mg of protein) | % of wild type |
|-----------------|---------------------------------------------------------|----------------|
| RER2            | 6,800                                                   | 100            |
| Δrer2           | 180                                                     | 2.7            |
| Δrer2/RER2 (CEN)| 6,900                                                   | 101            |
| RER2/RER2 (2μm)| 11,100                                                  | 163            |

$^a$ The activity of *cis*-prenyltransferase was measured at 30°C as described in Materials and Methods.

FIG. 12. Isoprenoid biosynthesis activities in *rer2* mutants. Total lysates were prepared from *RER2* (SNY9) (lanes 1 and 3), *rer2-2* (SNH23-7D) (lanes 2 and 4), and Δ*rer2* (SMY20) (lane 5), Δ*rer2* (SMY20) cells harboring *RER2* on a single-copy plasmid (lane 6), and *RER2* (SNY9) cells harboring *RER2* on a multicopy plasmid (lane 7) and incubated with \[^{1-14}C\]IPP and FPP at 20°C (lanes 1 and 2) or 30°C (lanes 3 to 7). The lipdic products were extracted with chloroform-methanol, spotted on a thin-layer chromatography plate, and developed with the benzene-ethyl acetate (95:5) solvent system (see Materials and Methods). Dolichol from porcine liver, ficaprenol (polyprenol from *F. elaeica*), solanesol (all-trans-nonaprenpol), and squalene were used as standards.

tionation was performed. A total cell lysate was prepared from the Δ*rer2* strain expressing 3HA-Rer2p on a single-copy plasmid and subjected to differential centrifugation at 13,000 × g and 100,000 × g (Fig. 9A). Although Rer2p has no hydrophobic region, most of the 3HA-Rer2p was fractionated in the P13 fraction, suggesting that 3HA-Rer2p is peripherally associated with membranes. This fractionation pattern coincided with that of an ER membrane protein, Sec12p, and a vacuolar enzyme, Dap2p, but not with that of a cytosolic enzyme, phosphoglycerate kinase, or a late-Golgi marker, Kex2p, which were detected mostly in the S100 and P100 fractions, respectively (15, 37, 38, 42). We further characterized the nature of the membrane association of 3HA-Rer2p by using several chemical reagents. The total homogenate was treated with reagents as indicated in Fig. 9B and centrifuged at 436,000 × g for 30 min. 3HA-Rer2p was not extracted with 1 M NaCl or 2 M urea but was partially solubilized with 0.1 M sodium carbonate (pH 11.5) and 1% (wt/vol) Triton X-100. This result indicates that 3HA-Rer2p is tightly associated with membranes.

Indirect immunofluorescence microscopy was also performed for the Δ*rer2* strain expressing 3HA-Rer2p on a single-copy plasmid. The cells were fixed and prepared for immunofluorescence with the anti-HA monoclonal antibody (16B12) (Fig. 10). We expected that 3HA-Rer2p would be localized to the ER from the result of the fractionation experiment, but the staining pattern of 3HA-Rer2p was quite different from the typ-

ical ER pattern. As shown in Fig. 10C, the anti-HA antibody stained punctate structures. These structures did not look like vacuoles either. We realized that these punctate structures were frequently observed alongside the nuclei (compare with the DAPI [4′,6-diamidino-2-phenylindole] staining in Fig. 10D) and near the cell surface. Since these regions are where ER membranes often exist, we carried out double-staining immunofluorescence with anti-HA and anti-BiP antibodies. As shown in Fig. 11, the staining of 3HA-Rer2p (Fig. 11B) overlaps at least partly with the staining of the ER protein BiP, suggesting that 3HA-Rer2p is localized in a subregion of the ER.

The *rer2* mutants are deficient in the *cis*-prenyltransferase activity required for dolichol synthesis. As described above, the *rer2* mutants showed deficiencies in early glycosylation steps. Hygromycin B sensitivity, vanadate resistance, and cell aggregation are also phenotypes often observed for glycosylation mutants. However, we hesitated to consider a direct role of Rer2p in protein glycosylation at this point, because the Rer2p family is conserved not only in eukaryotes but also in a variety of prokaryotic species. An important clue to reveal their function was obtained from our study of the o253 gene, the sole *E. coli* homologue of *RER2*. The o253 gene was cloned by PCR and found to be an essential gene in *E. coli*. Temperature-sensitive mutants, which were constructed by random mutagenesis on the plasmid, showed abnormal swollen cell shapes at the nonpermissive temperature, and many cells appeared to eventually die due to cell bursting (21a). This suggested that o253 plays an important role in the cell wall synthesis. The fact that *Mycoplasma*, a parasitic eubacterium lacking a cell wall, does not possess any *RER2* homologue in the genome supported this possibility.

The major constituent of the bacterial cell wall is peptidoglycan. By looking into the metabolic map, we realized that peptidoglycan synthesis in bacteria and glycosylation in eukaryotes employ very similar reactions. Both pathways utilize a specific lipid as the carrier of donor sugars: undecaprenyl phosphate (UP-P) for peptidoglycan and dolichyl phosphate (Dol-P) for oligosaccharides. Dol-P is made from dehydrolischyl diphosphate (Dedol-PP) by reduction, dephosphorylation, and phosphorylation. Dehydrolischol (C₆₅ to C₁₀₀) and undecaprenol (C₅₅) are both long-chain polyprenols and are synthesized by condensation of IPP units onto FPP with a *cis* configuration (*Z*-type), the reaction catalyzed by a single enzyme, *cis*-prenyltransferase (see Fig. 14). The genes coding for *cis*-prenyltransferase were not known for either prokaryotes or eukaryotes.

The possibility that the *RER2* gene encodes the *cis*-prenyltransferase to synthesize Dedol-PP was tested as follows. Membrane fractions were prepared from wild-type and Δ*rer2* cells and assayed for *cis*-prenyltransferase activity by measuring synthesis of polyprenol from IPP and FPP. The results are shown in Table 2. In Δ*rer2* cells, the activity of this enzyme was remarkably decreased. The residual activity, 2.7% of the wild type, may be due to the *SRT1* gene. The deficiency of *cis*-prenyltransferase was efficiently complemented by the *RER2* gene on a single-copy plasmid. Moreover, the wild-type cells overexpressing Rer2p on a multicopy plasmid showed 1.6-fold higher activity of this enzyme. To rule out the possibility that the decrease of the activity was due to the mislocalization of the enzyme, the whole activities of the isoprenoid biosynthesis utilizing FPP and IPP were investigated with total cell lysates (Fig. 12). The *rer2-2* and Δ*rer2* mutants synthesized compounds that comigrate with squalene (an intermediate of ergosterol synthesis) and all-*trans*-polyprenol (comigrating with solanesol) in amounts comparable (or even better in the case of squalene) to those for the wild-type cells. However, the syn-

FIG. 13. Overexpression of *RER2* suppresses the temperature-sensitive growth of *sec59* cells. The *sec59* mutant (SF604-9C) was transformed with *RER2* on a multicopy vector, *SRT1* on a multicopy vector, or a vector only. These and the control wild-type strain (ANY21) harboring an empty vector were incubated on an MCD plate at 35°C for 3 days.

thesis of dolichol compounds was quite deficient in the mutants. This deficiency was observed at either 20 or 30°C. Thus, the activity for synthesis of dolichol was almost lost in the *rer2* mutant cells. These observations let us conclude that the *RER2* gene is in fact responsible for the *cis*-prenyltransferase activity in yeast.

SEC59 has been known to encode dolichol kinase, an enzyme required to complete the Dol-P synthesis pathway (see Fig. 14) (17). *RER2* on a multicopy plasmid partially suppressed the temperature-sensitive growth of the *sec59* mutant at 35°C (Fig. 13). This observation strengthens our conclusions described above. Perhaps the elevated amount of de novo-synthesized dolichol from the overexpression of *RER2* suppresses the partial defect of dolichol kinase.

## DISCUSSION

In this paper, we have presented genetic and biochemical evidence that the yeast *RER2* gene encodes the *cis*-prenyltransferase enzyme Dedol-PP synthase. The in vivo function of dolichol and its derivatives, which was never assessed before, is discussed below.

Dolichol synthesis and *rer2* mutant phenotypes. Dolichol, a polyisoprenoid derivative, has been known to play essential roles in protein glycosylation (8). In N-linked glycosylation, the precursor oligosaccharide is preassembled on Dol-P and then transferred to proteins. Dol-P is also needed for O-linked glycosylation and GPI precursor synthesis in yeast. However, its biosynthesis has remained one of the least understood reactions in the metabolic pathways. Basically all isoprenoid compounds of eukaryotes, including sterols, ubiquinones, and dolichol, are synthesized from mevalonate. The first branch in this pathway to the synthesis of dolichol is the formation of long-chain (C₆₅ to C₁₀₀) polyprenyl diphosphate (Dedol-PP) by addition of isoprenyl units from IPP to FPP. This reaction is catalyzed by a single enzyme, *cis*-prenyltransferase (Fig. 14). Although the activity of this enzyme can be detected in crude extracts and membrane fractions (1, 2, 55), the gene encoding this enzyme was never identified. The details of the later steps

Vol. 19, 1999                                                                 Rer2p IN DOLICHOL SYNTHESIS 481

A

![Diagram A]

B

![Diagram B]

FIG. 14. (A) Role of carrier lipids in two systems. The unit of peptidoglycan is preassembled on UP-P in prokaryotes, whereas the core oligosaccharide complex in N glycosylation is synthesized on Dol-P in eukaryotes. Both lipids are derivatives of long-chain polyprenol. (B) Biosynthetic pathway of dolichol.

of dolichol synthesis, including α-saturation, dephosphorylation, and phosphorylation, are not well established either.

rer2 mutants, yeast mutants which we isolated as defective in the correct localization of Sec12p, show defects in early steps of N- and O-linked glycosylation. However, their other phenotypes, including mislocalization of multiple ER proteins and accumulation of aberrant ER and Golgi, were not previously reported for many known glycosylation mutants. Moreover, Rer2p homologues are found not only in eukaryotes but also in prokaryotes. Thus, we were interested in determining the general function of this protein family.

The Rer2p family is cis-prenyltransferase, an enzyme conserved from prokaryotes to eukaryotes. The link between the Rer2p family and cis-prenyltransferase was revealed by our study of the E. coli RER2 homologue, the o253 gene. This gene turned out to be essential for growth in *E. coli*, and its temperature-sensitive, conditional-lethal mutants showed a defect in cell wall synthesis. The synthesis of bacterial peptidoglycan, the major constituent of the cell wall, also utilizes a polyprenol compound, UP-P, as the sugar carrier. The unit of peptidoglycan (a disaccharide with a peptide side chain) is preassembled on UP-P, a reaction similar to the oligosaccharide synthesis on Dol-P (Fig. 14). The synthesis of UP-P is also catalyzed by a cis-prenyltransferase enzyme, undecaprenyl diphosphate (UP-PP) synthase, and the only difference from Dedol PP synthase is the chain length of the product. We measured the activity of UP-PP synthase for the o253 temperature-sensitive mutants and found that it was in fact markedly decreased in the mutants (21a). Furthermore, while these studies were in progress, we learned that Koyama's group at Tohoku University

such a phenotype has not been performed for other glycosylation mutants and is now under way in our laboratory.

The difference between the *rer2* and *sec59* mutants is particularly intriguing, because both are defective in the supply of the final product of the lipid carrier, Dol-P, and the greatest difference is that the *rer2* mutant lacks all of the derivatives of polyprenol but the *sec59* mutant can make dolichol. If they indeed differ in phenotypes, this will provide a good clue to understand a novel, as-yet-unknown physiological function of dolichol. There are some reports that the presence of dolichol or Dol-P in the lipid bilayer alters the membrane properties of model liposomes (8, 20, 45, 54). It is possible that dolichol and its derivatives have a structural role in membranes. The regulation of the dolichol level and its phosphorylation may be important to maintain the organellar membrane integrity. An interesting observation along this line is that ergosterol synthesis seems to be activated in *rer2* mutants (Fig. 12), suggesting that a mechanism to balance dolichol and sterol exists. Alternatively, dolichol might positively participate in various cellular functions, such as protein sorting in the secretory organelles. *rer2* and *srt1* mutants will provide important tools to reveal the physiological roles of this least-studied lipid in eukaryotes, and we will continue these efforts in our future projects.

### Subcellular localization of Rer2p

It has been reported that the *cis*-prenyltransferase activity is associated with microsomal membranes (2, 55). This is consistent with the results of our fractionation experiment that 3HA-Rer2p is a peripheral but tightly associated membrane protein and colocalizes with an ER marker. However, immunofluorescence microscopy has revealed that 3HA-Rer2p is not continuously present in the ER but rather shows a quite restricted localization. A fascinating explanation for this may be that Rer2p is localized to a subregion of the ER membrane that is specialized for dolichol synthesis. In the yeast *Saccharomyces carlsbergensis*, the formation of dolichol was observed when an in vitro assay was performed with the intact membrane, but Dedol-PP was not efficiently converted to dolichol when the enzyme was solubilized (7). The enzymes in dolichol synthesis might form a complex or be concentrated in the ER subdomain to increase the efficiency of the sequence of reactions.

### Physiological function of dolichol

Do the defects in N and O glycosylation as well as in GPI anchor synthesis due to the deficient dolichol synthesis explain all of the phenotypes of the *rer2* mutants? Formally, glycosylation mutants can have quite pleiotropic phenotypes, because they are deficient in a variety of glycoproteins. However, among a large number of yeast glycosylation mutants so far isolated, several phenotypes may be unique to the *rer2* mutants. For example, intracellular membrane structures are quite abnormal in the *rer2* mutants. The ER membranes are extensively elongated and accumulated, and the Golgi membranes form ring-like structures. Although information on the intracellular morphology in glycosylation mutants is very limited, the *sec53* mutant, a phosphomannomutase mutant, was reported to show no appreciable lesion in the membrane structures (13). The *sec59* mutant, a dolichol kinase mutant, was reported to exhibit some extension and fragmentation of the ER membrane, but this was not drastic like that of the *rer2* mutants (13). Tunicamycin treatment, which shuts off N-linked glycosylation, does not seem to affect organellar morphologies so dramatically. We would argue that the accumulation of abnormal membranes in the *rer2* mutants is not a general outcome of glycosylation deficiency. The most important phenotype of the *rer2* mutants from the viewpoint of membrane trafficking is that they mislocalize various ER proteins (Sec12-Mfa1p, Mfa1-Sec63p, Mfa1-Sec71p, and BiP) to later compartments of the secretory pathway. Examination of

### ACKNOWLEDGMENTS

We are grateful to M. Rose of the Massachusetts Institute of Technology; D. Gallwitz of the Max Planck Institute of Biophysical Chemistry, Göttingen, Germany; H. Riezman of the University of Basel; Y. Amaya of Niigata University; Y. Wada of Osaka University; and Suntory Limited, Osaka, Japan, for antibodies. We also thank A. Ohta of the University of Tokyo, K. Hosaka of Gunma University, M. Wachi of Tokyo Institute of Technology, H. Hara of Saitama University, and S. Fujisaki of Toho University for helpful discussions on the functions of the Rer2p family. We are particularly indebted to S. Fujisaki for valuable advice on the assay of *cis*-prenyltransferase. Thanks are also due to T. Koyama of Tohoku University for providing information prior to publication. Finally, we appreciate valuable discussions with the members of the Nakano laboratory.

This work was supported by grants-in-aid from the Ministry of Education, Science, Sports and Culture of Japan, by a research grant from the Human Frontier Science Program, and by funds from the Inamori Foundation and from the Biodesign Project of RIKEN. M. Sato and K. Sato are recipients of the Research Fellowship for Young Scientists from the Japan Society for the Promotion of Science.

### REFERENCES

1. Adair, W. L. J., and N. Cafmeyer. 1987. Characterization of the *Saccharomyces cerevisiae cis*-prenyltransferase required for dolichyl phosphate biosynthesis. Arch. Biochem. Biophys. **259**:589–596.
2. Adair, W. L. J., and R. K. Keller. 1982. Dolichol metabolism in rat liver. Determination of the subcellular distribution of dolichyl phosphate and its site and rate of *de novo* biosynthesis. J. Biol. Chem. **257**:8990–8996.
3. Ballou, L., R. A. Hitzeman, M. S. Lewis, and C. E. Ballou. 1991. Vanadate-resistant yeast mutants are defective in protein glycosylation. Proc. Natl. Acad. Sci. USA **88**:3209–3212.
4. Barlowe, C., L. Orci, M. H. Yeung, S. Hamamoto, N. Salama, M. F. Rexach, M. Ravazzola, M. Amherdt, and R. Schekman. 1994. COPII: a membrane coat formed by Sec proteins that drive vesicles budding from the endoplasmic reticulum. Cell **77**:895–907.
5. Barlowe, C., and R. Schekman. 1993. *SEC12* encodes a guanine nucleotide exchange factor essential for transport vesicle budding from the ER. Nature (London) **365**:347–349.
6. Boehm, J., H. D. Ulrich, R. Ossig, and H. D. Schmitt. 1994. Kex2-dependent invertase secretion as a tool to study the targeting of transmembrane proteins which are involved in ER-Golgi transport in yeast. EMBO J. **13**:3696–3710.
7. Bukhtiyarov, Y. E., Y. A. Shabalin, and I. S. Kulaev. 1993. Solubilization and characterization of dehydrolidichyl diphosphate synthase from yeast *Saccharomyces carlsbergensis*. J. Biochem. **113**:721–728.
8. Chojnacki, T., and G. Dallner. 1988. The biological role of dolichol. Biochem. J. **251**:1–9.

9. Dean, N. 1995. Yeast glycosylation mutants are sensitive to aminoglycosides. Proc. Natl. Acad. Sci. USA 92:1287–1291.

10. d'Enfert, C., C. Barlowe, S. Nishikawa, A. Nakano, and R. Schekman. 1991. Structural and functional dissection of a membrane glycoprotein required for vesicle budding from the endoplasmic reticulum. Mol. Cell. Biol. 11:5727–5734.

11. Deshaies, R. J., S. L. Sanders, D. A. Feldheim, and R. Schekman. 1991. Assembly of yeast Sec proteins involved in translocation into the endoplasmic reticulum into a membrane-bound multisubunit complex. Nature (London) 349:806–808.

12. Feldheim, D., K. Yoshimura, A. Admon, and R. Schekman. 1993. Structural and functional characterization of Sec66p, a new subunit of the polypeptide translocation apparatus in the yeast endoplasmic reticulum. Mol. Biol. Cell 4:931–939.

13. Ferro-Novick, S., P. Novick, C. Field, and R. Schekman. 1984. Yeast secretory mutants that block the formation of active cell surface enzymes. J. Cell Biol. 98:35–43.

14. Gatti, E., L. Popolo, M. Vai, N. Rota, and L. Alberghina. 1994. O-linked oligosaccharides in yeast glycosyl phosphatidylinositol-anchored protein gp115 are clustered in serine-rich region not essential for its function. J. Biol. Chem. 269:19695–19700.

15. Gaynor, E. C., S. te Heesen, T. R. Graham, M. Aebi, and S. D. Emr. 1994. Signal-mediated retrieval of a membrane protein from the Golgi to the ER in yeast. J. Cell Biol. 127:653–665.

16. Guthrie, C., and G. Fink. 1991. Guide to yeast genetics and molecular biology. Methods Enzymol. 194:182–187.

17. Heller, L., P. Orlean, and J. W. L. Adair. 1992. Saccharomyces cerevisiae sec59 cells are deficient in dolichol kinase activity. Proc. Natl. Acad. Sci. USA 89:7013–7016.

18. Hill, J. E., A. M. Myers, T. J. Koerner, and A. Tzagoloff. 1986. Yeast/E. coli shuttle vectors with multiple unique restriction sites. Yeast 2:163–167.

19. Jackson, M. R., T. Nilsson, and P. A. Peterson. 1990. Identification of a consensus motif for retention of transmembrane proteins in the endoplasmic reticulum. EMBO J. 9:3153–3162.

20. Janas, T., and H. T. Tien. 1988. Influence of dolichyl phosphate on permeability and stability of bilayer lipid membranes. Biochim. Biophys. Acta 939:624–628.

21. Jones, J. S., and L. Prakash. 1990. Yeast Saccharomyces cerevisiae selectable markers in pUC18 polylinkers. Yeast 6:363–366.

21a. Kato, J., S. Fujisaki, K. Nakajima, Y. Nishimura, M. Sato, and A. Nakano. Submitted for publication.

22. Keszenman-Pereyra, D., and K. Hieda. 1988. A colony procedure for transformation of Saccharomyces cerevisiae. Curr. Genet. 13:21–23.

23. Kurihara, T., and P. Silver. 1993. Suppression of a sec63 mutation identifies a novel component of the yeast endoplasmic reticulum translocation apparatus. Mol. Biol. Cell 4:919–930.

24. Kyte, P., and R. F. Doolittle. 1982. A simple method for displaying the hydrophobic character of a protein. J. Mol. Biol. 157:105–132.

25. Letourneur, F., E. C. Gaynor, S. Hennecke, C. Démollière, R. Duden, S. D. Emr, H. Riezman, and P. Cosson. 1994. Coatomer is essential for retrieval of dilysine-tagged proteins to the endoplasmic reticulum. Cell 79:1199–1207.

26. Munro, S., and H. R. B. Pelham. 1987. A C-terminal signal prevents secretion of luminal ER proteins. Cell 48:899–907.

27. Nakano, A., D. Brada, and R. Schekman. 1988. A membrane glycoprotein, Sec12p, required for protein transport from the endoplasmic reticulum to the Golgi apparatus in yeast. J. Cell Biol. 107:851–863.

28. Nishikawa, S., and A. Nakano. 1991. The GTP-binding Sar1 protein is localized to the early compartment of the yeast secretory pathway. Biochim. Biophys. Acta 103:135–143.

29. Nishikawa, S., and A. Nakano. 1993. Identification of a gene required for membrane protein retention in the early secretory pathway. Proc. Natl. Acad. Sci. USA 90:8179–8183.

30. Nishikawa, S., N. Umemoto, Y. Ohsumi, A. Nakano, and Y. Anraku. 1990. Biogenesis of vacuolar membrane glycoproteins of yeast Saccharomyces cerevisiae. J. Biol. Chem. 265:7440–7448.

31. Novick, P., C. Field, and R. Schekman. 1980. Identification of 23 complementation groups required for post-translational events in the yeast secretory pathway. Cell 21:205–215.

32. Oka, T., and A. Nakano. 1994. Inhibition of GTP hydrolysis by Sar1p causes accumulation of vesicles that are a functional intermediate of the ER-to-Golgi transport in yeast. J. Cell Biol. 124:425–434.

33. Pelham, H. R. B., K. G. Hardwick, and M. J. Lewis. 1988. Sorting of soluble ER proteins in yeast. EMBO J. 7:1757–1762.

34. Preuss, D., J. Mulholland, A. Franzusoff, N. Segev, and D. Botstein. 1992.

Characterization of the Saccharomyces Golgi complex through the cell cycle by immunoelectron microscopy. Mol. Biol. Cell 3:789–803.

35. Qadota, H., I. Ishii, A. Fujiyama, Y. Ohya, and Y. Anraku. 1992. RHO gene products, putative small GTP-binding proteins, are important for activation of the CAL1/CDC43 gene product, a protein geranylgeranyltransferase in Saccharomyces cerevisiae. Yeast 8:735–741.

36. Quellhorst, G. J. J., C. W. Hall, A. R. Robbins, and S. S. Krag. 1997. Synthesis of dolichol in a polyisoprenoid reductase mutant is restored by elevation of cis-prenyl transferase activity. Arch. Biochem. Biophys. 343:19–26.

37. Redding, K., C. Holcomb, and R. S. Fuller. 1991. Immunolocalization of Kex2 protease identifies a putative late Golgi compartment in yeast Saccharomyces cerevisiae. J. Cell Biol. 113:527–538.

38. Roberts, C. J., G. Pohling, J. H. Rothman, and T. H. Stevens. 1989. Structure, biosynthesis, and localization of dipeptidyl aminopeptidase B, an integral membrane glycoprotein of the yeast vacuole. J. Cell Biol. 108:1363–1373.

39. Rothblatt, J. A., R. J. Deshaies, S. L. Sanders, G. Daum, and R. Schekman. 1989. Multiple genes are required for proper insertion of secretory proteins into the endoplasmic reticulum in yeast. J. Cell Biol. 109:2641–2652.

40. Sadler, I., A. Chiang, T. Kurihara, J. Rothblatt, J. Way, and P. Silver. 1989. A yeast gene important for protein assembly into the endoplasmic reticulum and the nucleus has homology to DnaJ, an Escherichia coli heat shock protein. J. Cell Biol. 109:2665–2675.

41. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

42. Sato, K., S. Nishikawa, and A. Nakano. 1995. Membrane protein retrieval from the Golgi apparatus to the endoplasmic reticulum (ER): characterization of the RERI gene products as a component involved in ER localization of Sec12p. Mol. Biol. Cell 6:1459–1477.

43. Sato, K., M. Sato, and A. Nakano. 1997. Rer1p as common machinery for the endoplasmic reticulum localization of membrane proteins. Proc. Natl. Acad. Sci. USA 94:9693–9698.

44. Sato, M., K. Sato, and A. Nakano. 1996. Endoplasmic reticulum localization of Sec12p is achieved by two mechanisms: Rer1p-dependent retrieval that requires the transmembrane domain and Rer1p-independent retention that involves the cytoplasmic domain. J. Cell Biol. 134:279–293.

45. Schutzbach, J. S., and J. W. Jensen. 1989. Bilayer membrane destabilization induced by dolichylphosphate. Chem. Phys. Lipids 51:213–218.

46. Semenza, J. C., K. G. Hardwick, N. Dean, and H. R. B. Pelham. 1990. ERD2, a yeast gene required for the receptor-mediated retrieval of luminal ER proteins from the secretory pathway. Cell 61:1349–1357.

47. Shimizu, N., T. Koyama, and K. Ogura. 1998. Molecular cloning, expression and purification of undecaprenyl diphosphate synthase—no sequence similarity between E- and Z-prenyl diphosphate synthases. J. Biol. Chem. 273:19476–19481.

48. Sikorski, R. S., and P. Hieter. 1989. A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae. Genetics 122:19–27.

49. Stevens, T., B. Esmon, and R. Schekman. 1982. Early stages in the yeast secretory pathway are required for transport of carboxypeptidase Y to the vacuole. Cell 30:439–448.

50. Struhl, K., D. T. Stinchcomb, S. Scherer, and R. W. Davis. 1979. High-frequency transformation of yeast: autonomous replication of hybrid DNA molecules. Proc. Natl. Acad. Sci. USA 76:1035–1039.

51. Sun, G.-H., A. Hirata, Y. Ohya, and Y. Anraku. 1992. Mutations in yeast calmodulin cause defects in spindle pole body functions and nuclear integrity. J. Cell Biol. 119:1625–1639.

52. Takita, Y., Y. Ohya, and Y. Anraku. 1995. The CLS2 gene encodes a protein with multiple membrane-spanning domains that is important for Ca²⁺ tolerance in yeast. Mol. Gen. Genet. 246:269–281.

53. Vai, M., E. Gatti, E. Lacana, L. Popolo, and L. Alberghina. 1991. Isolation and deduced amino acid sequence of the gene encoding gp115, a yeast glycosylphosphatidylinositol-anchored protein containing a serine-rich region. J. Biol. Chem. 266:12242–12248.

54. van Duijn, G., C. Valtersson, T. Chojnacki, A. J. Verkleij, G. Dallner, and B. de Kruijff. 1986. Dolichyl phosphate induces non-bilayer structures, vesicle fusion and transbilayer movement of lipids: a model membrane study. Biochim. Biophys. Acta 861:211–223.

55. Wong, T. K., and W. J. Lennarz. 1982. The site of biosynthesis and intracellular deposition of dolichol in rat liver. J. Biol. Chem. 257:6619–6624.

56. Yoshihisa, T., and Y. Anraku. 1989. Nucleotide sequence of AMS1, the structure gene of vacuolar α-mannosidase of Saccharomyces cerevisiae. Biochem. Biophys. Res. Commun. 163:908–915.
