
# Molecular and Biochemical Characterization of the Protein Template Controlling Biosynthesis of the Lipopeptide Lichenysin

**DIRK KONZ, SASCHA DOEKEL, AND MOHAMED A. MARAHIEL***

Philipps-Universität Marburg, Fachbereich Chemie/Biochemie, 35032 Marburg, Germany

Received 26 May 1998/Accepted 30 September 1998

Lichenysins are surface-active lipopeptides with antibiotic properties produced nonribosomally by several strains of *Bacillus licheniformis*. Here, we report the cloning and sequencing of an entire 26.6-kb lichenysin biosynthesis operon from *B. licheniformis* ATCC 10716. Three large open reading frames coding for peptide synthetases, designated *licA*, *licB* (three modules each), and *licC* (one module), could be detected, followed by a gene, *licTE*, coding for a thioesterase-like protein. The domain structure of the seven identified modules, which resembles that of the surfactin synthetases SrfA-A to -C, showed two epimerization domains attached to the third and sixth modules. The substrate specificity of the first, fifth, and seventh recombinant adenylate domains of LicA to -C (cloned and expressed in *Escherichia coli*) was determined to be Gln, Asp, and Ile (with minor Val and Leu substitutions), respectively. Therefore, we suppose that the identified biosynthesis operon is responsible for the production of a lichenysin variant with the primary amino acid sequence L-Gln-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile, with minor Leu and Val substitutions at the seventh position.

Many strains of *Bacillus* are known to produce lipopeptides with remarkable surface-active properties (11). The most prominent of these powerful lipopeptides is surfactin from *Bacillus subtilis* (1). Surfactin is an acylated cyclic heptapeptide that reduces the surface tension of water from 72 to 27 mN m⁻¹ even in a concentration below 0.05% and shows some antibacterial and antifungal activities (1). Some *B. subtilis* strains are also known to produce other, structurally related lipoheptapeptides (Table 1), like iturin (32, 34) and bacillomycin (3, 27, 30), or the lipodecapeptides fengycin (50) and plipastatin (29).

In addition to *B. subtilis*, several strains of *Bacillus licheniformis* have been described as producing the lipopeptide lichenysin (14, 17, 23, 26, 51). Lichenysins can be grouped under the general sequence L-Glx-L-Leu-D-Leu-L-Val-L-Asx-D-Leu-L-Ile/Leu/Val (Table 1). The first amino acid is connected to a β-hydroxyl fatty acid, and the carboxy-terminal amino acid forms a lactone ring to the β-OH group of the lipophilic part of the molecule. In contrast to the lipopeptide surfactin, lichenysins seem to be synthesized during growth under aerobic and anaerobic conditions (16, 51). The isolation of lichenysins from cells growing on liquid mineral salt medium on glucose or sucrose basic has been studied intensively. Antimicrobial properties and the ability to reduce the surface tension of water have also been described (14, 17, 26, 51). The structural elucidation of the compounds revealed slight differences, depending on the producer strain. Various distributions of branched and linear fatty acid moieties of diverse lengths and amino acid variations in three defined positions have been identified (Table 1).

In contrast to the well-defined methods for isolation and structural characterization of lichenysins, little is known about the biosynthetic mechanisms of lichenysin production. The structural similarity of lichenysins and surfactin suggests that the peptide moiety is produced nonribosomally by multifunctional peptide synthetases (7, 13, 25, 49, 53). Peptide synthetases from bacterial and fungal sources describe an alternative route in peptide bond formation in addition to the ubiquitous ribosomal pathway. Here, large multienzyme complexes affect the ordered recognition, activation, and linking of amino acids by utilizing the thiotemplate mechanism (19, 24, 25). According to this model, peptide synthetases activate their substrate amino acids as aminoacyl adenylates by ATP hydrolysis. These unstable intermediates are subsequently transferred to a covalently enzyme-bound 4′-phosphopantetheinyl cofactor as thioesters. The thioesterified amino acids are then integrated into the peptide product through a stepwise elongation by a series of transpeptidations directed from the amino terminals to the carboxy terminals. Peptide synthetases have not only awakened interest because of their mechanistic features; many of the nonribosomally processed peptide products also possess important biological and medical properties.

In this report we describe the identification and characterization of a putative lichenysin biosynthesis operon from *B. licheniformis* ATCC 10716. Cloning and sequencing of the entire *lic* operon (26.6 kb) revealed three genes, *licA*, *licB*, and *licC*, with structural patterns common to peptide synthetases and a gene designated *licTE*, which codes for a putative thioesterase. The modular organization of the sequenced genes resembles the requirements for the biosynthesis of the heptapeptide lichenysin. Based on the arrangement of the seven identified modules and the tested substrate specificities, we propose that the identified genes are involved in the nonribosomal synthesis of the portion of the lichenysin peptide with the primary sequence L-Gln-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile (with minor Val and Leu substitutions).

---

## MATERIALS AND METHODS

### Bacterial strains and growth conditions.
*B. licheniformis* ATCC 10716 was maintained on agar plates of sporulation medium DSM (Difco, Detroit, Mich.) and grown in rich medium 2xYT (37) at 37°C for DNA isolation procedures. *Escherichia coli* XL1 Blue (Stratagene, Heidelberg, Germany) was used for the preparation of recombinant plasmids. *E. coli* XL1 Blue MRA(P2) was used as the host for the λ-EMBL3 genomic library and the preparation of λ phages.

*Corresponding author. Mailing address: Philipps-Universität Marburg, Fachbereich Chemie/Biochemie, Hans-Meerwein Str., 35032 Marburg, Germany. Phone: (0) 6421-285722. Fax: (0) 6421-282191. E-mail: Marahiel@ps1515.chemie.uni-marburg.de.*
### TABLE 1. Lipoheptapeptide antibiotics of *Bacillus* spp.

| Lipopeptide       | Organism          | Structure                                                                 | Reference |
|-------------------|-------------------|-------------------------------------------------------------------------|-----------|
| Lichenysin A      | *B. licheniformis* | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asn-D-Leu-L-Ile                          | 51, 52    |
| Lichenysin B      |                   | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu                          | 23, 26    |
| Lichenysin C      |                   | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile                          | 17        |
| Lichenysin D      |                   | $FA^a$-L-Gln-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile                          | This work |
| Surfactant 86     | *B. licheniformis* | $FA^a$-L-$Glx^d$-L-Leu-D-Leu-L-Val-L-$Asx^d$-D-Leu-$L$-Ile$^e$           | 14, 15    |
|                   |                   |                                                                         |           |
|                   |                   | L-Val                                                                   |           |
| Surfactin         | *B. subtilis*     | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu                          | 1, 7, 49  |
| Esperin           | *B. subtilis*     | $FA^b$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu$^e$                      | 45        |
|                   |                   | L-Val                                                                   |           |
| Iturin A          | *B. subtilis*     | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn-L-Ser                         | 32        |
| Iturin C          |                   | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn$^e$-L-Asn$^e$                | 34        |
|                   |                   | D-Ser-L-Thr                                                             |           |
| Bacillomycin L    | *B. subtilis*     | $FA^c$-L-Asp-D-Tyr-D-Asn-L-Ser-L-Gln-D-Pro$^e$-L-Thr                     | 3         |
|                   |                   | D-Ser-                                                                  |           |
| Bacillomycin D    |                   | $FA^c$-L-Asp-D-Tyr-D-Asn-L-Pro-L-Glu-D-Ser-L-Thr                         | 30, 31    |
| Bacillomycin F    |                   | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn-L-Thr                         | 27        |

#### Footnotes:
- $^a$ FA, β-hydroxy fatty acid. The β-hydroxy group forms an ester bond with the carboxy group of the C-terminal amino acid.
- $^b$ FA, β-hydroxy fatty acid. The β-hydroxy group forms an ester bond with the carboxy group of Asp5.
- $^c$ FA, β-amino fatty acid. The β-amino group forms a peptide bond with the carboxy group of the C-terminal amino acid.
- $^d$ Only the following combinations of amino acid 1 and 5 are allowed: Gln-Asp or Glu-Asn.
- $^e$ Where an alternative amino acid may be present in a structure, the alternative is also presented.

---

### Text

**Overexpression of recombinant proteins was carried out in *E. coli* M15 (Qiagen, Hilden, Germany).**

**Transformation of *E. coli* and DNA manipulations.** Standard genetic techniques for in vitro DNA manipulations, cloning, and transformation of competent *E. coli* cells were used (37). Total DNA from *B. licheniformis* was obtained by lysozyme treatment and phenol-chloroform extraction (8). For the construction of a λ-EMBL3 genomic library (Stratagene), chromosomal DNA of *B. licheniformis* ATCC 10716 was partially digested with Sau3AI and size fractionated in 15 to 40% (wt/vol) NaCl gradients. The fractions containing DNA fragments of 13 to 22 kb were pooled and ligated to λ-EMBL3 arms digested with BamHI. In vitro packaging was performed with Gigapack III Gold (Stratagene). For Southern blotting and plaque filter hybridization, the ECL random prime labeling and detection system (Amersham/Buchler, Braunschweig, Germany) and positively charged nylon membranes (Amersham/Buchler) were used according to the manufacturer’s protocol.

**Identification, cloning, and sequencing of the *lic* operon.** For the identification of the lichenysin synthetase genes a previously described PCR method (48) with degenerate primers derived from core sequences T and A7 of peptide synthetases (25) was performed. The sequences of the degenerate oligonucleotides used were as follows (the nucleotides in parentheses are degenerate): oligo-TGD, 5′-T(AT)(CT) CGI ACI GGI GA(TC) (CT)(TG)I G(TG)I CG-3′, and oligo-LGG, 5′-A(TA)I GA(GA) (TG)(CG)I CCI CCI (GA)(GA)(GC) I(AC)(AG) AA(GA) AA-3′. (All primers used in this study were synthesized by MWG Biotech [Ebersberg, Germany].) Variation of the annealing temperature in a range between 40 and 60°C yielded a set of different fragments, which were isolated and sequenced. The derived amino acid sequence of the DNA fragment PCR01 (Fig.

---

### FIGURE 1

#### (A) Predicted primary structure of lichenysin D. The boxes represent the residues incorporated by the three lichenysin synthetases, LicA, LicB, and LicC.

#### (B) Code for the patterns used for different domains in the remainder of the figure.

#### (C) Physical organization of the *lic* operon and additional identified ORFs. Three genes, *licA*, -B, and -C, coding for the lichenysin synthetases LicA to -C were identified. The domain organization of the peptide synthetases is illustrated within the genes. Abbreviations: E, EcoRI; S, SalI.

#### (D) PCR01, used for the screening of the λ-EMBL3 genomic library and λ clones isolated in this work. The probe used for chromosomal walking is marked.

#### (E) Gene fragments corresponding to A domains that were overexpressed and biochemically analyzed.

---

### Figure Components

#### (A)
- **C<sub>12</sub>-C<sub>17</sub>-beta-hydroxy acid**
- **LicA**: L-Gln → L-Leu → d-Leu
- **LicB**: L-Asp ← L-Val
- **LicC**: d-Leu ← L-Ile

#### (B)
- **condensation (C)**
- **epimerisation (E)** (optional)
- **amino acyladenylate formation (A)**
- **thioester formation (T)**

#### (C)
- **Genetic Map**
  - **0 to 30 kb**
  - **EcoRI (E) and SalI (S) sites**
  - **Genes: licA, licB, licC, licTE, ORF6, ORF5**

#### (D)
- **PCR01**
- **Probe locations**

#### (E)
- **Overexpressed gene fragments**
  - pQE-LicA1-A
  - pQE-LicB2-A
  - pQE-LicC-A

---

### TABLE 1. Lipoheptapeptide antibiotics of *Bacillus* spp.

| Lipopeptide       | Organism          | Structure                                                                 | Reference |
|-------------------|-------------------|-------------------------------------------------------------------------|-----------|
| Lichenysin A      | *B. licheniformis* | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asn-D-Leu-L-Ile                          | 51, 52    |
| Lichenysin B      |                   | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu                          | 23, 26    |
| Lichenysin C      |                   | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile                          | 17        |
| Lichenysin D      |                   | $FA^a$-L-Gln-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile                          | This work |
| Surfactant 86     | *B. licheniformis* | $FA^a$-L-$Glx^d$-L-Leu-D-Leu-L-Val-L-$Asx^d$-D-Leu-$L$-Ile$^e$           | 14, 15    |
|                   |                   |                                                                         |           |
|                   |                   | L-Val                                                                   |           |
| Surfactin         | *B. subtilis*     | $FA^a$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu                          | 1, 7, 49  |
| Esperin           | *B. subtilis*     | $FA^b$-L-Glu-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Leu$^e$                      | 45        |
|                   |                   | L-Val                                                                   |           |
| Iturin A          | *B. subtilis*     | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn-L-Ser                         | 32        |
| Iturin C          |                   | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn$^e$-L-Asn$^e$                | 34        |
|                   |                   | D-Ser-L-Thr                                                             |           |
| Bacillomycin L    | *B. subtilis*     | $FA^c$-L-Asp-D-Tyr-D-Asn-L-Ser-L-Gln-D-Pro$^e$-L-Thr                     | 3         |
|                   |                   | D-Ser-                                                                  |           |
| Bacillomycin D    |                   | $FA^c$-L-Asp-D-Tyr-D-Asn-L-Pro-L-Glu-D-Ser-L-Thr                         | 30, 31    |
| Bacillomycin F    |                   | $FA^c$-L-Asn-D-Tyr-D-Asn-L-Gln-L-Pro-D-Asn-L-Thr                         | 27        |

#### Footnotes:
- $^a$ FA, β-hydroxy fatty acid. The β-hydroxy group forms an ester bond with the carboxy group of the C-terminal amino acid.
- $^b$ FA, β-hydroxy fatty acid. The β-hydroxy group forms an ester bond with the carboxy group of Asp5.
- $^c$ FA, β-amino fatty acid. The β-amino group forms a peptide bond with the carboxy group of the C-terminal amino acid.
- $^d$ Only the following combinations of amino acid 1 and 5 are allowed: Gln-Asp or Glu-Asn.
- $^e$ Where an alternative amino acid may be present in a structure, the alternative is also presented.

---

### Text

**Overexpression of recombinant proteins was carried out in *E. coli* M15 (Qiagen, Hilden, Germany).**

**Transformation of *E. coli* and DNA manipulations.** Standard genetic techniques for in vitro DNA manipulations, cloning, and transformation of competent *E. coli* cells were used (37). Total DNA from *B. licheniformis* was obtained by lysozyme treatment and phenol-chloroform extraction (8). For the construction of a λ-EMBL3 genomic library (Stratagene), chromosomal DNA of *B. licheniformis* ATCC 10716 was partially digested with Sau3AI and size fractionated in 15 to 40% (wt/vol) NaCl gradients. The fractions containing DNA fragments of 13 to 22 kb were pooled and ligated to λ-EMBL3 arms digested with BamHI. In vitro packaging was performed with Gigapack III Gold (Stratagene). For Southern blotting and plaque filter hybridization, the ECL random prime labeling and detection system (Amersham/Buchler, Braunschweig, Germany) and positively charged nylon membranes (Amersham/Buchler) were used according to the manufacturer’s protocol.

**Identification, cloning, and sequencing of the *lic* operon.** For the identification of the lichenysin synthetase genes a previously described PCR method (48) with degenerate primers derived from core sequences T and A7 of peptide synthetases (25) was performed. The sequences of the degenerate oligonucleotides used were as follows (the nucleotides in parentheses are degenerate): oligo-TGD, 5′-T(AT)(CT) CGI ACI GGI GA(TC) (CT)(TG)I G(TG)I CG-3′, and oligo-LGG, 5′-A(TA)I GA(GA) (TG)(CG)I CCI CCI (GA)(GA)(GC) I(AC)(AG) AA(GA) AA-3′. (All primers used in this study were synthesized by MWG Biotech [Ebersberg, Germany].) Variation of the annealing temperature in a range between 40 and 60°C yielded a set of different fragments, which were isolated and sequenced. The derived amino acid sequence of the DNA fragment PCR01 (Fig.

---

### FIGURE 1

#### (A) Predicted primary structure of lichenysin D. The boxes represent the residues incorporated by the three lichenysin synthetases, LicA, LicB, and LicC.

#### (B) Code for the patterns used for different domains in the remainder of the figure.

#### (C) Physical organization of the *lic* operon and additional identified ORFs. Three genes, *licA*, -B, and -C, coding for the lichenysin synthetases LicA to -C were identified. The domain organization of the peptide synthetases is illustrated within the genes. Abbreviations: E, EcoRI; S, SalI.

#### (D) PCR01, used for the screening of the λ-EMBL3 genomic library and λ clones isolated in this work. The probe used for chromosomal walking is marked.

#### (E) Gene fragments corresponding to A domains that were overexpressed and biochemically analyzed.

---

### Figure Components

#### (A)
- **C<sub>12</sub>-C<sub>17</sub>-beta-hydroxy acid**
- **LicA**: L-Gln → L-Leu → d-Leu
- **LicB**: L-Asp ← L-Val
- **LicC**: d-Leu ← L-Ile

#### (B)
- **condensation (C)**
- **epimerisation (E)** (optional)
- **amino acyladenylate formation (A)**
- **thioester formation (T)**

#### (C)
- **Genetic Map**
  - **0 to 30 kb**
  - **EcoRI (E) and SalI (S) sites**
  - **Genes: licA, licB, licC, licTE, ORF6, ORF5**

#### (D)
- **PCR01**
- **Probe locations**

#### (E)
- **Overexpressed gene fragments**
  - pQE-LicA1-A
  - pQE-LicB2-A
  - pQE-LicC-A


1D) showed high homology with other peptide synthetases and was used as a probe in the subsequent screening of the genomic λ-EMBL3 library of *B. licheniformis*.

A total of ca. 8,000 plaques were screened, with approximately 500 plaques per agar plate (8.5-cm diameter). Positive phage clone λ-A11.1 (Fig. 1) was isolated, and its DNA insert was mapped and subcloned in pBluescript SK(−) by using *SalI* and *EcoRI*. Sequencing of the double-stranded pBluescript plasmids was performed with standard universal SK primers and walking primers derived from the determined sequences. Small gaps between the subclones were filled by PCR amplification and sequencing of the product or by sequencing of the λ DNA template. In a second round of screening, using a 5′-terminal fragment of the λ-A11.1 insert, among others, as a probe the phage clone λ-C8.1 was isolated. The insert of this phage was also subcloned and sequenced in the same way as λ-A11.1.

Sequencing reactions were carried out by the chain termination method (38) with dye-labeled dideoxy terminators from the PRISM Ready Reaction DyeDeoxy Terminator cycle sequencing kit with AmpliTaq FS polymerase (Applied Biosystems) according to the manufacturer’s protocol and were analyzed on the ABI 310 genetic analyzer.

**Amplification and cloning of gene regions corresponding to adenylation domains.** PCR techniques with the Expand long-range PCR system (Boehringer, Mannheim, Germany) were used for amplifying the first (*LicA1-A*), fifth (*LicB2-A*), and seventh (*LicC-A*) adenylation domains (25) of the *lic* operon from *B. licheniformis* DNA. We used 5′-modified primers to generate terminal restriction sites for subsequent cloning into the pQE60/70 His₆-tagged cloning vector (Qiagen). The sequences of the oligonucleotides used were as follows (modified restriction sites are italicized; restriction sites are in boldface): *licA1-5′NcoI*, **5′-ATT CCA TGG AGA TTG TTC CCG CTT TT-3′**; *licA1-3′BamHI*, **5′-TAT GGA TCC ATA TTC ATT TCT CGG TGC-3′**; *licB2-5′SphI*, **5′-ATT GCA TGC TTT CAG TCA AAG AGC G-3′**; *licB2-3′BamHI*, **5′-ATT GGA TCC GCA GAG GGC TTT TTC-3′**; *licC-5′SphI*, **5′-TAA GCA TGC AGC CGC TTG ACG ACA T-3′**; and *licC-3′BamHI*, **5′-TAA GGA TCC GAG AAC CTC AGA CCA A-3′**. For verifying the correct insertion of the DNA fragments into previously digested pQE70 (*SphI/BamHI*) and pQE60 (*NcoI/BamHI*), the fusion sites between the vector and the inserts were examined by sequencing with the following primers: 5′ promoter, **5′-GGC GTA TCA CGA GGC CC-3′**, and 3′ terminator, **5′-ACG CCC GGC GGC AAC CG-3′**.

**Expression and purification of the three His₆-tagged adenylation domains.** The recombinant pQE derivatives designated pQE-LicA1-A, pQE-LicB2-A, and pQE-LicC-A were transformed in *E. coli* M15(pREP4) and expressed as previously described (42), except that the cells were induced with 0.5 mM IPTG (isopropyl-β-d-thiogalactopyranoside) at an *A*₆₀₀ of 0.7 and allowed to grow for an additional 1.5 h before being harvested. Purification of the His₆-tagged proteins LicA1-A, LicB2-A, and LicC-A was performed after breaking the cells by three passages through a French press with Ni²⁺ affinity chromatography as described previously (42). The expression and purification of the recombinant proteins were checked by Coomassie brilliant blue-stained sodium dodecyl sulfate (SDS)-polyacrylamide gels. The concentrations of the purified proteins were determined by a Bradford test (4).

**ATP-PPᵢ exchange.** In order to determine the substrate specificities of the purified proteins, amino acid-specific ATP-PPᵢ exchanges were assayed as previously described (42), with minor modifications. The assay mixture contained 0.4 mM amino acid, 4 mM ATP, and 200 nM enzyme in buffer A (2 mM MgCl₂, 2 mM dithiothreitol, and 50 mM sodium phosphate, pH 8.0). Exchange was initiated by the addition of 0.15 μCi of sodium [³²P]pyrophosphate (to 1 mM) in a total volume of 0.1 ml. For kinetic experiments, the concentration of ATP (0.2 to 4 mM) or amino acid (0.02 to 6 mM) was varied. After incubation at 37°C for 10 min, the reaction was stopped by the addition of 0.5 ml of termination mix (100 mM sodium pyrophosphate, 560 mM HClO₄, and 1.2% [wt/vol] activated charcoal). The charcoal was pelleted by centrifugation, washed once with 1 ml of H₂O, resuspended in 0.5 ml of H₂O, and added to a scintillation vial containing 4.0 ml of scintillation fluid, and the bound radioactivity was determined by liquid scintillation counting.

**Nucleotide sequence accession number.** The nucleotide sequences (28,798 bp) from *B. licheniformis* ATCC 10716 described in this paper have been submitted to GenBank under accession no. U95370.

---

### RESULTS

**Cloning and sequencing of the *lic* operon.** A recently reported PCR method for the identification of peptide synthetase genes (48) was performed. Using degenerate oligonucleotide primers derived from the core sequences A7 (TGD) and T (LGGxSI) of peptide synthetase LGGxSI (25), a DNA fragment, designated PCR01, was PCR amplified from the chromosome of *B. licheniformis* ATCC 10716. The sequencing of this fragment resulted in the identification of the internal core sequences A8 to A10 (25), which are characteristic of peptide synthetases and thus indicate that PCR01 represents part of a peptide synthetase gene. In order to clone the complete identified gene, we constructed a λ-EMBL3 genomic library of *B. licheniformis* ATCC 10716 and screened it with the PCR01 fragment as a probe. Eleven λ phage clones hybridizing with the probe were isolated. One of them, designated λ-A11.1 (Fig. 1D), was further investigated. The 20.2-kb insert of λ-A11.1 was mapped and subcloned into pBluescript SK(−) with the restriction enzymes *EcoRI* and *SalI*. By terminal sequencing of the generated subclones, it was shown that the entire 5′ part of the λ-A11.1 insert DNA has homology with peptide synthetase genes whereas the 3′ part is not homologous. To identify the remaining part of the peptide synthetase gene, an additional step of chromosomal walking was performed. A 5′-terminal restriction fragment of the λ-A11.1 insert was used as a probe for the screening of the λ-EMBL3 genomic library (Fig. 1D). Twenty-one phages reacting with this probe were isolated. The inserts of the isolated phages were mapped and analyzed by Southern hybridization. One clone named λ-C8.1 (13.8-kb insert), which showed only a short terminal overlap with λ-A11.1, was subcloned into pBluescript SK(−) by using *EcoRI* and *SalI*. The terminal sequencing of the subclones showed that this phage contained a 5′-terminal region lacking homology with peptide synthetase genes, thus indicating that the entire peptide synthetase gene was cloned. Altogether, the two λ phages A11.1 and C8.1 span a continuous region of 28.8 kb. The complete nucleotide sequences of the generated subclones were determined. Remaining gaps between the subclones were closed by PCR amplification from chromosomal DNA. In all, 28,798 bp of *B. licheniformis* ATCC 10716 were sequenced, showing a typical GC content of about 50%.

The *lic* operon includes three peptide synthetase genes, *licA*, *licB*, and *licC*. Within the investigated sequence three large open reading frames (ORFs) with the same transcriptional direction were identified. The first ORF, designated *licA* and 10,746 bp in length, codes for a protein of 3,582 amino acids with a predicted mass of 402,623 Da. The second ORF, *licB*, with 10,764 bp, codes for a protein of approximately the same size (3,588 amino acid; 403,872 Da). The third large ORF (3,864 bp), named *licC*, codes for a protein of 1,288 amino acids with a predicted mass of 144,919 Da. These three ORFs span a continuous region 25,439 bp in length. We suggest that the translation of *licA* starts with the ATG codon at nucleotide 462, which is located 9 bp downstream of a putative ribosomal binding site (AGGAGG). Between nucleotides 315 to 323 and 335 to 340, we identified the putative promoter regions −35 (TCTTTTCTT) and −10 (TATAAT). Based on our knowledge of other genes coding for peptide synthetases, it is likely that these genes are organized in an operon (25): all three genes are in the same transcriptional direction, and the distances between the translational stops and starts for *licA-licB* and *licB-licC* are obviously not more than 19 and 38 bp, respectively. Thus, we designated a site 6 bp downstream of a putative ribosomal binding site (ACGAGG) at nucleotide 11,230 as the translational start of *licB* and a site 7 bp downstream of an AGCGGG sequence at nucleotide 22,035 as the start of *licC*.

By detailed sequence analysis of the deduced amino acid sequence, three modules typical of peptide synthetases were identified within *licA*. The modules, each containing an adenylation (A) domain and a thioester-binding (T) domain, are linked by two condensation (C) domains (25). Additionally, a condensation domain in front of the first module and an epimerization (E) domain attached to the C-terminal end of the third module were detected. The analysis of *licB* showed that it has exactly the same domain organization as *licA*. The predicted translational product of *licC* is composed of one AT

**TABLE 2. Homology of identified gene products with other lichenysin and surfactin synthetases**

| Proteins aligned          | % Identity$^a$ |
|---------------------------|----------------|
| LicA and LchAA            | 97             |
| LicA and SrfA-A           | 60             |
| LchAA and SrfA-A          | 60             |
| LicB and SrfA-B           | 61             |
| LicC and SrfA-C           | 57             |
| LicTE and SrfA-T          | 58             |

$^a$ Aligned with the ClustalW program.

---

domain and a N-terminal C domain. Sequence motifs of approximately 250 amino acid residues typical of thioesterase-like proteins could be detected at the C-terminal end of this protein, indicating that it may serve as a terminator in nonribosomal peptide synthesis (9, 25, 39). Taken together, the three genes *licA* to *-C* represent a seven-module peptide synthetase system that presumably will incorporate D-amino acids in the third and sixth positions of the product heptapeptide. These findings are in perfect agreement with the primary structure of lichenysins (Table 1). Also, the existence of a further C domain in front of the first module, possibly serving as an acceptor for a fatty acid, as in the case of surfactin, supports the point of view that the identified genes are coding for the lichenysin synthetases.

Homology searches by databank alignment revealed that the DNA sequence of *licA* is about 98% identical to a sequence submitted to the GenBank database under accession no. Y10550 and, in part, X94148 (Table 2). This sequence was obtained from the lichenysin A-producing strain *B. licheniformis* BNP29 (53) and is described as a putative lichenysin synthetase subunit I gene, *lchAA*. This high percentage of identity clearly shows that the genes *licA* and *lchAA*, with minor strain-specific alterations, are identical. The second-highest overall quota of homology (Table 2) for *licA* to *-C* was found with the surfactin synthetase genes *srfA-A* to *-C* of *B. subtilis* ATCC 21332 (GenBank accession no. X70356) (7). Surfactin of *B. subtilis* and lichenysins of *B. licheniformis* contain a very homologous (in the case of lichenysin B, an identical) peptide part (Table 1). Therefore, it can be expected that the biosyntheses of both substances follow similar pathways. The modular organization found in the surfactin synthetases SrfA-A to -C (7) is completely identical with the synthetases LicA to -C reported here. Indeed, not only do the number of modules and the organization of the domains reflect those of the surfactin synthetases, even the single domains of *lic* show the highest homology to the corresponding domains of SrfA (~60% identity). This similarity reflects the homology previously found among some domains of the *grs* and *tyc* operons (28). Among themselves and in comparison to peptide synthetase domains of other origin (i.e., gramicidin, tyrocidine, and bacitracin synthetases), the domains show an identity of approximately 50% (data not shown). Particularly striking is a more than 93% identity of the third AT domain of the first synthetase (*licA3*) to the third AT domain of the second synthetase (*licB3*). Again, these findings resemble the analogous 96% identity found between *srfA3* and *srfB3*. These data strongly support the predictions that (i) the genes *licA* to *-C* are involved in the nonribosomal biosynthesis of lichenysin and (ii) the genes for lichenysin synthetases of *B. licheniformis* and surfactin synthetases of *B. subtilis* may have the same evolutionary origin.

Additional ORFs identified in the 3′ region of *licC*. Within a downstream region of *licC* comprising 2,897 bp we detected two additional genes and the 5′ region of a third ORF. Only 4 bp downstream of the translational stop codon of *licC*, a putative start codon of a 771-bp ORF was identified. The predicted gene product encodes a protein of 257 amino acid residues that has a mass of 28.7 kDa. Database alignments of this gene product showed 71% identity to the external thioesterase-like protein found in the corresponding region of the *srfA* operon of *B. subtilis* (SrfTE) (7) and 40 to 45% identity to analogous proteins found in connection with other peptide synthetases (GrsT, TycF, etc.) (21, 28). Therefore, we designated this gene *licTE* (ORF4).

At 958 bp downstream of the translational stop of *licTE* and in the opposite transcriptional direction, a putative start codon of the second ORF (ORF5) was identified. This gene of 1,032 bp, which overlaps in part with *licTE*, codes for a protein of 344 amino acid residues (37.8 kDa) and shows the highest homology to ORF7 in the *srfA4-sfp* intergenic region of *B. subtilis* (7). The third ORF (ORF6) starts 107 bp upstream of ORF5 and shows no translational stop codon in the remaining 1,055 bp. The derived protein sequence shares the highest homology with ORF8 in the *srfA4-sfp* intergenic region (7).

Even though the last two genes were also found within the 3′ region of the *srfA* operon and show the highest similarity to each other, the order of the genes in *lic* and *srfA* 3′ regions is different. No analogues to the *srfA4-sfp* intergenic region’s ORF6 and -7 could be detected. The lack of an ORF7 analogue in particular might be surprising, because it has been suggested that the *srfA4-sfp* intergenic region ORF6 and ORF7 form an unusual transcriptional response regulator pair (7, 35).

Amplification, expression, and substrate specificity of three A domains from *licA*, *licB*, and *licC*. Via PCR techniques we amplified DNA fragments from chromosomal DNA of *B. licheniformis* ATCC 10716 corresponding to the adenylation domains of LicA1, LicB2, and LicC (Fig. 1E). The fragments, of approximately 1,500 bp, were cloned into pQE expression vectors (see Materials and Methods). Almost completely soluble protein fragments were obtained, using the defined adenylation domain borders first described by Mootz and Marahiel (28). For this approach, the N-terminal ends of LicA1-A, LicB2-A, and LicC-A domain fragments 97, 94, and 102 amino acids in front of core sequence A2 (L/MKA/SGG), respectively, were chosen (25). We chose the C-terminal ends (in the same order) 30, 24, and 18 amino acids in front of core sequence T (I/LGGHS) (25). After expression in *E. coli*, the proteins LicA1-A, LicB2-A, and LicC-A were visualized by SDS-polyacrylamide gel electrophoresis by comparison of whole *E. coli* protein extracts before and 1.5 h after induction with IPTG (Fig. 2). Purification on Ni²⁺-chelate affinity chromatography confirmed the presence of the recombinant His₆-tagged proteins, whose observed mass (60 to 70 kDa) is in good agreement with the calculated masses. The purified protein fractions were applied to amino acid-dependent ATP-PPᵢ exchange assays with all 20 proteinogenic amino acids plus L-Orn, D-Asp, and one control without amino acids. The results of these experiments as presented in Fig. 3 clearly revealed that LicA1-A activates L-Gln, LicB2-A activates L-Asp, and LicC-A activates L-Ile. If the highest activation for each domain protein was defined as 100%, the measured background level of the control reaction without amino acids was found to be 0.1%. With LicC-A a comparatively high side preference for L-Val (30%) and L-Leu (30%) could be detected. LicA1-A and LicB2-A showed only small side preferences (1% of the specific amino acid) for L-Ala, L-Met, and L-Trp and L-Asn, L-Ile, and L-Val, respectively.

The quantitative affinities of the proteins to their substrate ATP and the cognate amino acid (for LicC-A, also L-Leu and L-Val) were determined by measurement of the *Km* values (Table 3). The values range from 0.3 to 3 mM for the amino

FIG. 2. Coomassie brilliant blue-stained SDS-polyacrylamide gels of *E. coli*-overexpressed Lic domains LicA1-A (a), LicB2-A (b), and LicC-A (c). Lanes: (1, 10-kDa protein ladder; 2, whole-cell extract prior to induction; 3, whole-cell extract after 1.5-h induction with IPTG; 4, protein purified by Ni²⁺-chelate affinity chromatography.

acid and from 0.1 to 6 mM for ATP. For LicC-A, the higher \( K_m \) values obtained for L-Leu and L-Val were in good agreement with the quantitative results. With the exception of the observed ATP \( K_m \) value for LicA1-A, which seems surprisingly high, the other data are in the same range as already reported for other wild-type peptide synthetases as well as recombinant peptide synthetase fragments (10, 18, 28, 36, 42).

Based on the already demonstrated functional integrity of recombinant peptide synthetase domains (28, 42), the determined substrate specificity for LicA1-A, LicB2-A, and LicC-A let us conclude that the first, fifth, and seventh positions within the product peptide processed by the seven-module peptide synthetases LicA, LicB, and LicC are L-Gln(1), L-Asp(5), and L-Ile(7), with minor L-Leu and L-Val substitutions in the C-terminal positions.

## DISCUSSION

We have cloned and characterized a lichenysin (*lic*) biosynthesis operon of *B. licheniformis* ATCC 10716 and a 2.1-kb region downstream of this operon in which a minimum of two ORFs can be detected. The *lic* operon is composed of three peptide synthetase genes, *licA* to -C, and a gene, *licTE*, coding for a protein homologous to SrfA-TE (7), GrsT (21), and TycF (28). We therefore anticipate that *licTE* codes for a thioesterase of uncertain function that was found to be associated with several bacterial operons encoding peptide synthetases (25, 39). Downstream of the *lic* operon and in the opposite transcriptional direction we identified an ORF, designated ORF5, coding for a protein which shows a significant homology to transmembrane proteins. An analogous gene, whose product is thought to act as a sensor protein in a two-component regulatory system (7), was found in the 3′ region of the *srfA* operon (*srfA*-ORF7). A second ORF (ORF6), upstream of ORF5 and with the same direction of transcription as the *lic* operon, shows homology to proteins of the helix-turn-helix type and is most similar to the GNTR family of transcriptional regulators (43).

By thorough sequence analysis, seven modules typical of peptide synthetases (25) could be identified within the characterized genes *licA* to -C. Attached to the last module, a thioesterase (TE) domain was detected, which in all bacterial peptide synthetase systems identified so far is found exclusively associated with the module responsible for the activation and incorporation of the last amino acid into the product peptide (25). Furthermore, we found an E domain attached to the third and sixth modules, which presumably catalyze the incorporation of a D-amino acid into the peptide product. In front of the first module of LicA, an additional C domain is located, indicating that the first amino acid could be acylated. In more detailed sequence alignments it can be shown that this first C domain of LicA has only around 20% identity with the remaining six C domains of LicA, -B, and -C, which show approximately 40% identity to each other. Similar conditions can be found in related systems, like surfactin (SrfA-A to -C) (7) or fengycin (Pps1 to -5 and FenA to -D) (46), in which the first amino acid of the product peptide is acylated with a fatty acid. Together, these findings reflect the fact that the first C domain of each of these systems is responsible for the linking of a fatty acid instead of a peptidyl moiety to an amino acid.

These data are in perfect agreement with the primary structure of lichenysins, a group of homologous lipoheptapeptides with alterations of the amino acid composition in defined positions: 1, Glx; 5, Asx; and 7, Ile/Leu/Val (Table 1). For other peptide synthetase systems, a strong colinearity between the amino acid sequence of the product peptide and the order of amino acid-incorporating modules within the synthetases was demonstrated (20, 25, 28). Therefore we were particularly interested in analyzing the substrate specificities of recombinant A domains corresponding to the first (LicA1-A), fifth (LicB2-A), and seventh (LicC) positions. In relation to the obtained amino acid-dependent activation patterns of the three recombinant A domains, we propose that the lichenysin synthetases (LicA to -C) reported here are responsible for the production of a lipoheptapeptide variant of the following sequence: L-Gln-L-Leu-D-Leu-L-Val-L-Asp-D-Leu-L-Ile/Val/Leu. A lichenysin with this specific amino acid sequence has not yet been described, although the predicted structure most closely resembles that of surfactant 86 (15). However, in the case of surfactant 86 it is still unclear which amino acids are incorporated in the first and fifth positions (Gln/Asp or Glu/Asn). Thus, we designated the predicted product of the lichenysin synthetase LicA as lichenysin D (Table 1).

Variations at defined positions involving hydrophobic amino acids (Ile/Leu/Val), like those reported for lichenysins, were also described for many other nonribosomally synthesized peptides, like surfactin (2), esperin (45), bacitracin (44), tyrocidine (12), cyclosporine (22), and enniatin (33). Previously, it has been shown that nutrient conditions can affect the composition of the product peptide, either in vivo by adding certain amino acids to the medium or in vitro in cell-free systems with arbitrarily chosen substrate concentrations. Indeed, most nonribosomal...

#### Table 3: Determination of \( K_m \) values of cognate amino acid substrates and ATP of internal adenylation domains LicA1-A, LicB2-A, and LicC-A

| Tested domain | Substrate       | \( K_m \) (mmol·liter\(^{-1}\)) ± SD |
|---------------|-----------------|-------------------------------------|
|               | Amino acid      |                                     |
| LicA1-A       | L-Gln           | 1.4 ± 0.5                          |
| LicB2-A       | L-Asp           | 0.3 ± 0.15                         |
| LicC-A        | L-Ile           | 0.4 ± 0.15                         |
| LicC-A        | L-Leu           | 3 ± 1                              |
| LicC-A        | L-Val           | 2.5 ± 1                            |
|               | ATP\(^a\)       |                                     |
| LicA1-A       |                 | 5 ± 3                              |
| LicB2-A       |                 | 1 ± 0.3                            |
| LicC-A        |                 | 0.1 ± 0.02                         |
| LicC-A        |                 | 5 ± 2                              |
| LicC-A        |                 | 0.6 ± 0.2                          |

**Footnote:**  
\(^a\) Experiments were performed in the presence of the specific substrate amino acid.

---

### Text Content

In addition to alterations of hydrophobic amino acids, exchanges of Gln and Glu or Asn and Asp were described for numerous nonribosomally synthesized peptides, such as plipastatin and fengycin (29, 50), iturin A and C (32, 34), and bacillomycin L, D, and F (Table 1) (3, 27, 31). By determining the substrate specificity of LicA1-A and LicB2-A, we were able to show that these domains are highly specific and have no side specificity towards Glu and Asn, respectively. Analogous results have already been reported for the recombinant A domains of TycC1 (Asn) and TycC2 (Gln) (28). Also, sequence comparisons reveal (at least for several Gln- and Glu-specific somally synthesized peptides represent mixtures of several compounds. In the case of lichenysins, the diversity in the last amino acid position can presumably be related to the observed relaxed substrate specificity of the corresponding A domain (LicC-A), which, besides Ile, also activates Leu and Val to a lesser extent. Therefore, it is reasonable that single mutations during evolutionary processes within these domains slightly shifted their amino acid preferences, resulting in the observed strain-specific alterations of Ile, Leu, and Val at the seventh position of lichenysins. Indeed, several A domains most related to the sequence of LicC (Fig. 4) show similar relaxed specificity towards Ile, Val, Leu, or Ala (2).

---

### Figures

#### Figure 3: Relative substrate specificities of the purified proteins LicA1-A (a), LicB2-A (b), and LicC-A (c) obtained from the results of amino acid-dependent ATP-PPi exchange experiments. In addition to the 20 proteinogenic amino acids, L-Orn and D-Asp were also tested. Activities are represented by the bars, and the highest activity was defined as 100%. Control experiments performed by omitting all amino acids showed activities below 0.1%. LicA1-A and LicB2-A showed highly specific substrate recognition towards L-Gln and L-Asp, respectively, whereas the specificity of LicC-A was not fully restricted to L-Ile. Also, L-Leu and L-Val showed some activities.

#### Figure 4: Phylogenetic tree obtained by an alignment of the amino acid sequences of 58 peptide synthetase modules of *Bacillus* origin. Approximately 180 amino-acid regions between the core sequences A3 and A6, which are thought to represent the substrate specificity-determining segment of peptide synthetase modules (6), were aligned with the ClustalW program. The scale represents the percentage of homology. Amino acid specificity is represented as a three-letter amino acid code. Abbreviations of the protein names are as follows: Grs, gramicidin synthetases (47); Tyc, tyrocidine synthetases (28); SrfA, surfactin synthetases (7); Fen, fengycin synthetases (5); Pps, plipastatin synthetases (46); Bac, bacitracin synthetases (20); Lic, lichenysin synthetases; and Pks, polyketide or polypeptide synth(et)ase of unknown function from *B. subtilis* (40). The letters following the protein names represent synthetase subunits; the numbers represent the module within the subunit. Clustered groups of sequences obtained from adenylation domain segments with the same or homologous specificity are indicated by shaded boxes.

---

# Table 3: Determination of \( K_m \) values of cognate amino acid substrates and ATP of internal adenylation domains LicA1-A, LicB2-A, and LicC-A

| Tested domain | Substrate       | \( K_m \) (mmol·liter\(^{-1}\)) ± SD |
|---------------|-----------------|-------------------------------------|
|               | Amino acid      |                                     |
| LicA1-A       | L-Gln           | 1.4 ± 0.5                          |
| LicB2-A       | L-Asp           | 0.3 ± 0.15                         |
| LicC-A        | L-Ile           | 0.4 ± 0.15                         |
| LicC-A        | L-Leu           | 3 ± 1                              |
| LicC-A        | L-Val           | 2.5 ± 1                            |
|               | ATP\(^a\)       |                                     |
| LicA1-A       |                 | 5 ± 3                              |
| LicB2-A       |                 | 1 ± 0.3                            |
| LicC-A        |                 | 0.1 ± 0.02                         |
| LicC-A        |                 | 5 ± 2                              |
| LicC-A        |                 | 0.6 ± 0.2                          |

**Footnote:**  
\(^a\) Experiments were performed in the presence of the specific substrate amino acid.

---

### Text Content

In addition to alterations of hydrophobic amino acids, exchanges of Gln and Glu or Asn and Asp were described for numerous nonribosomally synthesized peptides, such as plipastatin and fengycin (29, 50), iturin A and C (32, 34), and bacillomycin L, D, and F (Table 1) (3, 27, 31). By determining the substrate specificity of LicA1-A and LicB2-A, we were able to show that these domains are highly specific and have no side specificity towards Glu and Asn, respectively. Analogous results have already been reported for the recombinant A domains of TycC1 (Asn) and TycC2 (Gln) (28). Also, sequence comparisons reveal (at least for several Gln- and Glu-specific somally synthesized peptides represent mixtures of several compounds. In the case of lichenysins, the diversity in the last amino acid position can presumably be related to the observed relaxed substrate specificity of the corresponding A domain (LicC-A), which, besides Ile, also activates Leu and Val to a lesser extent. Therefore, it is reasonable that single mutations during evolutionary processes within these domains slightly shifted their amino acid preferences, resulting in the observed strain-specific alterations of Ile, Leu, and Val at the seventh position of lichenysins. Indeed, several A domains most related to the sequence of LicC (Fig. 4) show similar relaxed specificity towards Ile, Val, Leu, or Ala (2).

---

### Figures

#### Figure 3: Relative substrate specificities of the purified proteins LicA1-A (a), LicB2-A (b), and LicC-A (c) obtained from the results of amino acid-dependent ATP-PPi exchange experiments. In addition to the 20 proteinogenic amino acids, L-Orn and D-Asp were also tested. Activities are represented by the bars, and the highest activity was defined as 100%. Control experiments performed by omitting all amino acids showed activities below 0.1%. LicA1-A and LicB2-A showed highly specific substrate recognition towards L-Gln and L-Asp, respectively, whereas the specificity of LicC-A was not fully restricted to L-Ile. Also, L-Leu and L-Val showed some activities.

#### Figure 4: Phylogenetic tree obtained by an alignment of the amino acid sequences of 58 peptide synthetase modules of *Bacillus* origin. Approximately 180 amino-acid regions between the core sequences A3 and A6, which are thought to represent the substrate specificity-determining segment of peptide synthetase modules (6), were aligned with the ClustalW program. The scale represents the percentage of homology. Amino acid specificity is represented as a three-letter amino acid code. Abbreviations of the protein names are as follows: Grs, gramicidin synthetases (47); Tyc, tyrocidine synthetases (28); SrfA, surfactin synthetases (7); Fen, fengycin synthetases (5); Pps, plipastatin synthetases (46); Bac, bacitracin synthetases (20); Lic, lichenysin synthetases; and Pks, polyketide or polypeptide synth(et)ase of unknown function from *B. subtilis* (40). The letters following the protein names represent synthetase subunits; the numbers represent the module within the subunit. Clustered groups of sequences obtained from adenylation domain segments with the same or homologous specificity are indicated by shaded boxes.


A domains) remarkable identities of up to 80% between PpsD2 and FenA2 and 65% between SrfA-A1 and LicA1 (Fig. 4). Recently, the crystal structure of the GrsA adenylation domain, with its substrates L-Phe and AMP, was solved at a resolution of 1.9 Å, defining the residues involved in substrate recognition and the locations of the highly conserved core sequences in the superfamily of adenylate-forming enzymes (6). Due to the high conservation of A domains, we conclude that the residues of the binding pocket attached to the substrate amino acid as shown for GrsA-Phe are located within an approximately 180-amino-acid region located between the core sequences A3 and A6 (25). This fairly heterologous region within otherwise highly conserved adenylation domains seems to be the substrate-determining segment.

For the Asp-specific A domain of SrfA-B2, it was shown that a single mutation (His738Glu) in this region is sufficient to alter the specificity to Asn completely (41). We therefore suppose that only a few amino acid substitutions within Glu- and Asp-specific domains are responsible for the conversion of specificity to Gln and Asn, respectively, even if the Asn- and Asp-specific A domains described so far may have evolved independently (Fig. 4). In conclusion, it is reasonable that lichenysin biosynthesis operons for all lichenysin isoforms have emerged from the same origin. Alterations in the primary peptide sequence may be due to slight evolution-dependent strain-specific mutations.

In this context, it is remarkable that LicA and LchAA, although they are up to 97% identical, seem to be responsible for the incorporation of different amino acids in the first position: LicA specifically activates Gln, whereas LchAA is thought to incorporate Glu into its product peptide (51). If the corresponding substrate-determining regions between the core sequences A3 and A6 (25) of LicA1 and LchAA1 are aligned with each other, only two amino acid substitutions can be detected (Gly773 and Ala773 and Asn795 and Gly795 in LicA1 and LchAA1, respectively). On the basis of these two varying residues, it is not possible to formulate an obvious and conclusive rule which could explain how the two enzymes distinguish between Glu and Gln. Therefore, it remains unknown if additional mechanisms in vivo could be responsible for the conversion of Gln to Glu in the peptide product or if Gln and Asp are eventually incorporated in lichenysin A rather than Glu and Asn.

### ACKNOWLEDGMENTS

D. Konz and S. Doekel contributed equally to this work. We thank Inge Schüler for excellent technical assistance and Henning Mootz for helpful discussion. The work was supported by the Deutsche Forschungsgemeinschaft, EG Project Cell Factories, and the Fonds der Chemischen Industrie.

### REFERENCES

1. Arima, K., A. Kakinuma, and G. Tamura. 1968. Surfactin, a crystalline peptidelipid surfactant produced by *Bacillus subtilis*: isolation, characterization and its inhibition of fibrin clot formation. Biochem. Biophys. Res. Commun. **31**:488–494.
2. Baumgart, F., B. Kluge, C. Ullrich, J. Vater, and D. Ziessow. 1991. Identification of amino acid substitutions in the lipopeptide surfactin using 2D NMR spectroscopy. Biochem. Biophys. Res. Commun. **177**:998–1005.
3. Besson, F., F. Peypoux, G. Michel, and L. Delcambe. 1977. The structure of bacillomycin L, an antibiotic from *Bacillus subtilis*. Eur. J. Biochem. **77**:61–67. (In French.)
4. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. **72**:248–254.
5. Chen, C. L., L. K. Chang, Y. S. Chang, S. T. Liu, and J. S. Tschen. 1995. Transposon mutagenesis and cloning of the genes encoding the enzymes of fengycin biosynthesis in *Bacillus subtilis*. Mol. Gen. Genet. **248**:121–125.
6. Conti, E., T. Stachelhaus, M. A. Marahiel, and P. Brick. 1997. Structural basis for the activation of phenylalanine in the non-ribosomal biosynthesis of gramicidin S. EMBO J. **16**:4174–4183.
7. Cosmina, P., F. Rodriguez, F. de Ferra, G. Grandi, M. Perego, G. Venema, and D. van Sinderen. 1993. Sequence and analysis of the genetic locus responsible for surfactin synthesis in *Bacillus subtilis*. Mol. Microbiol. **8**:821–831.
8. Cutting, S. M., and P. B. Vander Horn. 1990. Genetic analysis, p. 27–74. In C. R. Harwood and S. M. Cutting (ed.), Molecular biological methods for *Bacillus*. John Wiley & Sons, New York, N.Y.
9. de Ferra, F., F. Rodriguez, O. Tortora, C. Tosi, and G. Grandi. 1997. Engineering of peptide synthetases. Key role of the thioesterase-like domain for efficient production of recombinant peptides. J. Biol. Chem. **272**:25304–25309.
10. Elsner, A., H. Engert, W. Saenger, L. Hamoen, G. Venema, and F. Bernhard. 1997. Substrate specificity of hybrid modules from peptide synthetases. J. Biol. Chem. **272**:4814–4819.
11. Fiechter, A. 1992. Biosurfactants: moving towards industrial application. Trends Biotechnol. **10**:208–217.
12. Fujikawa, K., Y. Sakamoto, T. Suzuki, and K. Kurahashi. 1968. Biosynthesis of tyrocidine by a cell-free enzyme system of *Bacillus brevis* ATCC 8185. II. Amino acid substitution in tyrocidine. Biochim. Biophys. Acta **169**:520–533.
13. Fuma, S., Y. Fujishima, N. Corbell, D. Souza C., M. M. Nakano, P. Zuber, and K. Yamane. 1993. Nucleotide sequence of 5′ portion of *srfA* that contains the region required for competence establishment in *Bacillus subtilis*. Nucleic Acids Res. **21**:93–97.
14. Horowitz, S., J. N. Gilbert, and W. M. Griffin. 1990. Isolation and characterization of a surfactant produced by *Bacillus licheniformis* 86. J. Ind. Microbiol. **6**:243–248.
15. Horowitz, S., and W. M. Griffin. 1991. Structural analysis of *Bacillus licheniformis* 86 surfactant. J. Ind. Microbiol. **7**:45–52.
16. Javaheri, M., G. E. Jennemann, M. J. McInerney, and R. M. Knapp. 1985. Anaerobic production of a biosurfactant by *Bacillus licheniformis* JF-2. Appl. Environ. Microbiol. **50**:698–700.
17. Jenny, K., O. Käppeli, and A. Fiechter. 1991. Biosurfactants from *Bacillus licheniformis*: structural analysis and characterization. Appl. Microbiol. Biotechnol. **36**:5–13.
18. Kittelberger, R., M. Altmann, and H. von Döhren. 1982. Kinetics of amino acid activation in gramicidin S synthesis, p. 209–218. In H. Kleinkauf and H. von Döhren (ed.), Peptide antibiotics, biosynthesis and functions. Walter de Gruyter, Berlin, Germany.
19. Kleinkauf, H., and H. von Döhren. 1996. A nonribosomal system of peptide biosynthesis. Eur. J. Biochem. **236**:335–351.
20. Konz, D., A. Klens, K. Schörgendorfer, and M. A. Marahiel. 1997. The bacitracin biosynthesis operon of *Bacillus licheniformis* ATCC 10716: molecular characterization of three multi-modular peptide synthetases. Chem. Biol. **4**:927–937.
21. Krätzscharmar, J., M. Krause, and M. A. Marahiel. 1989. Gramicidin S biosynthesis operon containing the structural genes *grsA* and *grsB* has an open reading frame encoding a protein homologous to fatty acid thioesterases. J. Bacteriol. **171**:5422–5429.
22. Lawen, A., and R. Traber. 1993. Substrate specificities of cyclosporin synthetase and peptolidine SDZ 214-103 synthetase. Comparison of the substrate specificities of the related multifunctional polypeptides. J. Biol. Chem. **268**:20452–20465.
23. Lin, S. C., M. A. Minton, M. M. Sharma, and G. Georgiou. 1994. Structural and immunological characterization of a biosurfactant produced by *Bacillus licheniformis* JF-2. Appl. Environ. Microbiol. **60**:31–38.
24. Lipmann, F. 1980. Bacterial production of antibiotic polypeptides by thiol-linked synthesis on protein templates. Adv. Microb. Physiol. **21**:227–266.
25. Marahiel, M. A., T. Stachelhaus, and H. D. Mootz. 1997. Modular peptide synthetases involved in non-ribosomal peptide synthesis. Chem. Rev. **97**:2651–2673.
26. McInerney, M. J., M. Javaheri, and D. P. Nagle, Jr. 1990. Properties of the biosurfactant produced by *Bacillus licheniformis* strain JF-2. J. Ind. Microbiol. **5**:95–101.
27. Mhammedi, A., F. Peypoux, F. Besson, and G. Michel. 1982. Bacillomycin F, a new antibiotic of iturin group: isolation and characterization. J. Antibiot. (Tokyo) **35**:306–311.
28. Mootz, H. D., and M. A. Marahiel. 1997. The tyrocidine biosynthesis operon of *Bacillus brevis*: complete nucleotide sequence and biochemical characterization of functional internal adenylation domains. J. Bacteriol. **179**:6843–6850.
29. Nishikiori, T., H. Naganawa, Y. Muraoka, T. Aoyagi, and H. Umezawa. 1986. Plipastatins: new inhibitors of phospholipase A2, produced by *Bacillus cereus* BMG302-fF67. III. Structural elucidations of plipastatins. J. Antibiot. (Tokyo) **39**:755–761.
30. Peypoux, F., F. Besson, G. Michel, and L. Delcambe. 1981. Structure of bacillomycin D, a new antibiotic of the iturin group. Eur. J. Biochem. **118**:323–327.
31. Peypoux, F., F. Besson, G. Michel, C. Lenzen, L. Dierickx, and L. Delcambe. 1980. Characterization of a new antibiotic of the iturin group: bacillomycin D. J. Antibiot. (Tokyo) **33**:1146–1149.

32. Peypoux, F., M. Guinand, G. Michel, L. Delcambe, C. Das, and E. Lederer. 1978. Structure of iturin A, a peptidelipid antibiotic from *Bacillus subtilis*. Biochemistry 17:3992–3996.

33. Pieper, R., H. Kleinkauf, and R. Zocher. 1992. Enniatin synthetases from different *fusaria* exhibiting distinct amino acid specificities. J. Antibiot. (Tokyo) 45:1273–1277.

34. Quentin, M. J., F. Besson, F. Peypoux, and G. Michel. 1982. Action of peptidolipidic antibiotics of the iturin group on erythrocytes. Effect of some lipids on hemolysis. Biochim. Biophys. Acta 684:207–211.

35. Reverchon, S., W. Nasser, and J. Robert-Baudouy. 1994. *pecS*: a locus controlling pectinase, cellulase and blue pigment production in *Erwinia chrysanthemi*. Mol. Microbiol. 11:1127–1139.

36. Riederer, B., M. Han, and U. Keller. 1996. D-Lyseryl peptide synthetase from the ergot fungus *Claviceps purpurea*. J. Biol. Chem. 271:27524–27530.

37. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

38. Sanger, F., S. Niklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

39. Schneider, A., and M. A. Marahiel. 1998. Genetic evidence for a role of thioesterase domains, integrated in or associated with peptide synthetases, in non-ribosomal peptide biosynthesis in *Bacillus subtilis*. Arch. Microbiol. 169: 404–410.

40. Scotti, C., M. Piatti, A. Cuzzoni, P. Perani, A. Tognoni, G. Grandi, A. Galizzi, and A. M. Albertini. 1993. A *Bacillus subtilis* large ORF coding for a polypeptide highly similar to polyketide synthases. Gene 130:65–71.

41. Stachelhaus, T., and M. A. Marahiel. 1997. Unpublished results.

42. Stachelhaus, T., and M. A. Marahiel. 1995. Modular structure of peptide synthetases revealed by dissection of the multifunctional enzyme GrsA. J. Biol. Chem. 270:6163–6169.

43. Stock, J. B., A. J. Ninfa, and A. M. Stock. 1989. Protein phosphorylation and the regulation of adaptive responses in bacteria. Microbiol. Rev. 53:450–490.

44. Tetler, L. W., S. N. Davey, and M. Morris. 1993. Analysis of bacitracin B using fast atom bombardment and tandem mass spectrometry. Biol. Mass Spectrom. 22:712–720.

45. Thomas, D. W., and T. Ito. 1969. The revised structure of the peptide antibiotic esperin, established by mass spectrometry. Tetrahedron 25:1985–1990.

46. Tognoni, A., E. Franchi, C. Magistrelli, E. Colombo, P. Cosmina, and G. Grandi. 1995. A putative new peptide synthase operon in *Bacillus subtilis*: partial characterization. Microbiology 141:645–648.

47. Turgay, K., M. Krause, and M. A. Marahiel. 1992. Four homologous domains in the primary structure of GrsB are related to domains in a superfamily of adenylate-forming enzymes. Mol. Microbiol. 6:529–546.

48. Turgay, K., and M. A. Marahiel. 1994. A general approach for identifying and cloning peptide synthetase genes. Pept. Res. 7:238–241.

49. Ullrich, C., B. Kluge, Z. Palacz, and J. Vater. 1991. Cell-free biosynthesis of surfactin, a cyclic lipopeptide produced by *Bacillus subtilis*. Biochemistry 30: 6503–6508.

50. Vanittanakom, N., and W. Loeffler. 1986. Fengycin—a novel antifungal lipopeptide antibiotic produced by *Bacillus subtilis* F29-3. J. Antibiot. (Tokyo) 39:888–901.

51. Yakimov, M. M., K. N. Timmis, V. Wray, and H. L. Fredrickson. 1995. Characterization of a new lipopeptide surfactant produced by thermotolerant and halotolerant subsurface *Bacillus licheniformis* BAS50. Appl. Environ. Microbiol. 61:1706–1713.

52. Yakimov, M. M., H. L. Fredrickson, and K. N. Timmis. 1996. Effect of heterogeneity of hydrophobic moieties on surface activity of lichenysin A, a lipopeptide biosurfactant from *Bacillus licheniformis* BAS50. Biotechnol. Appl. Biochem. 23:13–18.

53. Yakimov, M. M., and P. N. Golyshin. 1997. ComA-dependent transcriptional activation of lichenysin A synthetase promoter in *Bacillus subtilis* cells. Biotechnol. Prog. 13:757–761.
